<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-29 09:39:09 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>79</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, , Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC), accounting for 80–90% of renal malignancies, is frequently driven by VHL inactivation—either through mutation or promoter hypermethylation—resulting in constitutive HIF2α activation and pseudohypoxic signaling. VHL gene inactivation is a hallmark of von Hippel–Lindau syndrome, a hereditary disorder predisposing patients to ccRCC and other tumors, underscoring its central role in disease pathogenesis. While VHL dysfunction promotes aggressive tumor phenotypes, the therapeutic potential of VHL restoration remains underexplored. Here, using the Cas9 induced VHL-mutation in the Caki-1 cell line model, we demonstrate that VHL inactivation augments hypoxia-like pathways and enhances anaerobic glycolysis. Rescue of functional VHL reversed these activation patterns and modulated the expression of genes associated with angiogenesis. Using single cell transcriptomics, we show that the VHL-positive and -negative Caki-1 cells are characterized with different proportions of benign and aggressive cells as seen by analysis of specific gene expression. Furthermore, the identified angiogenesis-related genes were linked to affect clinical outcomes in ccRCC patients, suggesting that VHL restoration may mitigate high-risk molecular features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3312bb9b7da4f1993f3b32599331d1d37aad30b7" target='_blank'>
              Manipulation with Mutational Status of VHL Regulates Hypoxic Metabolism and Pro-Angiogenic Phenotypes in ccRCC Caki-1 Cells
              </a>
            </td>
          <td>
            P. Abramov, Alexander Mazur, Aleksey Starshin, S. Zhenilo, Egor Prokhortchouk
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8adbd0387ac6e2aa073dbe75080e4f98049b516f" target='_blank'>
              Acute Myeloid Leukemia Relapse after Bromodomain Inhibitor Treatment or Chemotherapy is Characterized by Myc-Ras Transcriptional Remodeling
              </a>
            </td>
          <td>
            Benjamin J. Huang, Jeevitha D’Souza, Alexa Rane Batingana, M. Harris, Xinyue Wang, E. Hwang, Anica M Wandler, Michael R. Burgess, Qing Li, S. Meshinchi, Gideon Bollag, Kevin Shannon
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="


 Despite the remarkable success of targeted therapy, the emergence of acquired resistance remains a major obstacle to cure acute myeloid leukemia (AML) patients. While existing studies have predominantly focused on genetic heterogeneity as a driver for therapy resistance, recent evidence suggests that non-genetic persister state is equally important, but such studies are lacking in AML. Here, we define the temporal transcriptome and chromatin landscape along with lineage trajectory of individual genetically identical AML persister clones to existing anti-AML therapeutics.
 To model persister state in AML, we utilized watermelon library (Oren et al., 2021), that expresses lineage barcode with inducible Histone-2B-mCherry to trace clonal evolution and cell division. We first evaluated whether slow-cycling, quiescent persisters emerge in AML lines after exposure to standard therapy. Our data showed the emergence of slow cycling persisters with cytarabine (AraC) in all 4 cell lines, and gilteritinib (GLT) in MOLM-13 and MV4-11 cells while BH3 mimetics (venetoclax (VEN), S63845 and navitoclax) had fast-cycling persisters. Integrating proliferative-index profiles with cell cycle profiles depicted extensive heterogeneity in AML persisters imposed by drug target rather than genetic state. Drug holiday experiments exhibited a reversible drug tolerance state in both slow and fast-cycling persisters. Longitudinal drug sensitivity and BH3 profiling assays showed that persisters emerged with adaptation to reduced mitochondrial priming that reversed after drug holiday. However, transitioning from persister to stable resistant cells resulted in a sustained loss in apoptotic priming, mirroring observations made in PDX models and primary tumors.
 To understand trajectory of persister emergence and underlying mechanisms, we performed single-cell multiomics on drug-naïve and persister populations (day 9) in MV4-11 (GLT), and OCI-AML2 (VEN and AraC). We expanded 10,000 unique barcoded cells and administered the drug treatment to derive persisters. Using lineage barcode analysis, we report significant reduction in the dominant lineage of day 0 and survival of multiple small lineages indicating an acquired state of persistence than selection of pre-existing resistant clone in both the lines. Further trajectory analysis defined three fates of the lineages as: expanding (log2FC>1), shrinking (log2FC <1) and stable. AraC persisters showed higher expanding and stable lineages, while VEN and GLT persisters resulted in higher shrinking and stable clones, revealing heterogeneity in lineage expansion. We next asked whether persisters resulting from different therapies arised from the same parental lineage. To test this, we performed lineage tracing on VEN and AraC persisters in OCI-AML2 and compared surviving lineages at Day 9 vs Day 0 cells. This analysis identified 52 parental lineages that were able to produce both VEN and AraC persisters in AML2, providing evidence of branched evolution of persisters from the same parental clone.
 We finally defined transcriptional and epigenetic heterogeneity in baseline and persister cells by performing UMAP on single-cell RNAseq and ATACseq data. We identified persisters to all 3 drugs clusters distinctly from their drug naïve counterparts at both transcriptome and chromatin accessibility. Next, we investigated differentially expressed pathways in persisters and observed higher inflammatory response pathways such as TNFα signaling in GLT, enrichment of MYC targets and oxidative phosphorylation in VEN and high mTORC1 signalling in AraC persisters. We observed higher chromatin accessibility in GLT and VEN whereas condensed chromatin in AraC persisters. We next asked which motifs are highly enriched in each persister and observed the enrichment of various KLF and SP family of motifs in each persister driving multiple gene signatures in each persister type. We also observed enrichment of LSC-primed and cDC like cells in GLT persisters but with AraC and VEN persisters a uniform pattern of all states was observed, indicated stemness may not be common mechanism to persister evolution.
 Collectively, our integrated single-cell multiomics and lineage tracing approach reveals that AML persisters emerge through stochastic drug-specific, non-genetic adaptive mechanisms characterized by distinct transcriptional and chromatin remodeling programs rather than deterministic Darwinian selection of pre-existing resistant cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b912c83b14f52bda36c8d81fafb30ac8e86a906c" target='_blank'>
              Dissection of non-genetic resistance mechanisms by lineage tracing and single cell multiomics reveals drug-specific persister programs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Bhatia, Xiao Xian Lin, Ankit Jana, Emanuelle I. Grody, Ryan Jeremy Lim Jing Wei, Shawn Koh Jia Jun, Elisha Teh Jia Yu, Cin thet Kyi, Prasad Iyer, Greg Kellogg, Yogesh Goyal, Lawrence Boise, Yaara Oren, Shruti Bhatt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4820b1bf2279dace4e93e3451d6b890388f9b9" target='_blank'>
              Computational Inference of Metabolic Programs: A Case Study Analyzing the Effect of BRCA1 Loss
              </a>
            </td>
          <td>
            M. Masid, Ioanna A. Rota, David Barras, Flavia De Carlo, P. Ginefra, Mathieu Desbuisson, Y. Ortiz-Miranda, Dmitriy Zamarin, Sohrab P. Shah, N. Vannini, G. Coukos, Denarda Dangaj-Laniti, V. Hatzimanikatis
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acquired resistance to immune checkpoint inhibitors (ICIs) remains a significant challenge in the treatment of metastatic melanoma. Phenotypic plasticity, such as dedifferentiation and transdifferentiation, is an increasingly recognized mechanism for treatment resistance. We present a case of a man in his 70s with metastatic melanoma who experienced progression through sequential treatments including pembrolizumab in combination with the HDAC inhibitor entinostat, and ipilimumab. During treatment a histologically distinct pleomorphic rhabdomyosarcoma (RMS) emerged at metastatic sites. Longitudinally acquired tumor samples representing both phenotypes were analyzed using whole- exome sequencing (WES), RNA sequencing (RNA-seq) and high-plex tissue imaging (spatial proteomics). WES revealed driver mutations (e.g. NRAS, NF1) and loss-of-heterozygosity (LOH) shared between phenotypes indicating a common ancestral clone. Phylogenetic analysis demonstrated an early divergence of the phenotypes, with each later acquiring unique mutations. RNA-seq showed mutually exclusive expression of lineage-specific markers as well as epithelial-mesenchymal transition and myogenic gene set enrichment in the RMS samples. High-plex imaging identified distinct tumor microenvironments, with RMS lesions enriched in CD163+ macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, P. Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e46bc8969d218f90b6869d6ad86ad7ec80c28324" target='_blank'>
              Loss of ESE3/EHF is sufficient to promote cell plasticity, transformation and androgen-independent status in the early stage of prostate carcinogenesis
              </a>
            </td>
          <td>
            D. Albino, G. Sandrini, C. Musumeci, Elisa Storelli, Giovanni Papa, Daniela Impellizzieri, A. Cacciatore, G. Cassanmagnago, Atik Balla, Simone Mosole, G. Civenni, Andrea Rinaldi, Carmen Jeronimo, Rui M. Henrique, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Despite the clinical success of the BCL2 inhibitor venetoclax in acute myeloid leukemia (AML), treatment resistance frequently emerges, leading to relapse. Mounting evidence suggests that metabolic reprogramming contributes to venetoclax resistance, yet the underlying mechanisms remain poorly defined. This study aimed to develop and characterize venetoclax-resistant AML models and explore metabolic vulnerabilities that could be exploited therapeutically.
 For this purpose, we established two venetoclax-resistant AML cell lines (MV4-11VR and MOLM-13VR) through repeated intermittent exposure to increasing doses of venetoclax. Resistance was assessed via cell viability assays, annexin V-based apoptosis analysis, mitochondrial membrane potential, cell cycle profiling, and colony formation assays. Metabolic reprogramming was evaluated using measurements of oxygen consumption rate (OCR) and extracellular acidification rate (ECAR). Molecular and genomic alterations were explored through transcriptomic profiling, global proteomics, and targeted sequencing. In addition, we tested the therapeutic potential of combining venetoclax with metabolic inhibitors, metformin, a mitochondrial complex I inhibitor, and KPT-9274, a NAMPT inhibitor, by calculating synergy scores and quantifying apoptotic responses.
 Venetoclax-resistant MV4-11VR and MOLM-13VR cells exhibited markedly elevated IC50 values (more than 200-fold compared to parental cells), impaired apoptosis induction, preserved proliferative capacity, and enhanced colony formation under venetoclax exposure. Notably, resistant cells showed distinct metabolic rewiring: MV4-11VR cells relied on enhanced glycolysis, whereas MOLM-13VR cells displayed increased oxidative phosphorylation (all p < 0.05). These findings were supported by functional assays and proteomic signatures, which showed enrichment of carbohydrate metabolism pathways in MV4-11VR and aerobic energy production in MOLM-13VR (all FDR < 0.05).
 At the molecular level, venetoclax resistance was accompanied by cell line-specific modulation of BCL2-family gene expression. MV4-11VR cells exhibited downregulation of multiple pro-apoptotic genes, including BAX, BCL2L11, BBC3, BIK, and BNIP3, while MOLM-13VR cells showed increased expression of MCL1 and BAD, along with reduced expression of BID, PMAIP1, and BMF. At the protein level, both models demonstrated reduced PARP1 cleavage and lower γH2AX levels upon venetoclax exposure, indicating suppression of apoptosis and DNA damage signaling. Furthermore, total BCL2 levels remained unchanged, while BCL-XL expression was stable in resistant cells but downregulated in parental cells upon drug exposure. A shared hallmark in both models was the hyperactivation of the PI3K/AKT/mTOR pathway, as evidenced by increased phosphorylation of the downstream effector RPS6. Additionally, MV4-11VR cells showed reduced ERK1/2 phosphorylation, whereas MOLM-13VR cells exhibited increased ERK1/2 activation.
 Genomic profiling revealed additional mechanisms contributing to resistance. MV4-11VR cells acquired a novel TP53 mutation (R158H) and showed clonal expansion of a pre-existing TP53 R248W mutation, alongside a mosaic deletion of the TP53 locus. In MOLM-13VR cells, sequencing revealed the emergence of a truncating mutation in the SPEN gene (I1326*) and acquisition of trisomy 5, further supporting distinct genetic trajectories of resistance.
 Therapeutically, venetoclax-resistant cells showed partial sensitivity to the metabolic inhibitors metformin and KPT-9274. Importantly, combination therapy with venetoclax significantly increased apoptosis in resistant cells. The venetoclax + KPT-9274 combination yielded strong synergy (ZIP scores up to 13.86) and eliminated over 95% of resistant cells, demonstrating the feasibility of targeting metabolic dependencies to restore drug sensitivity.
 Acquired venetoclax resistance in AML arises through heterogeneous yet targetable metabolic reprogramming. Distinct metabolic and molecular adaptations in resistant MV4-11 and MOLM-13 cells point to the complexity of resistance mechanisms. Nonetheless, the shared activation of survival pathways and metabolic plasticity creates vulnerabilities that can be exploited using mitochondrial or NAD+ biosynthesis inhibitors. These findings support a combinatorial therapeutic strategy targeting both apoptotic and metabolic pathways to overcome acquired venetoclax resistance in AML. Supported by FAPESP, CAPES, and CNPq.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98bf25f48d9830834e2992fc3cf2049960a971f" target='_blank'>
              Metabolic reprogramming drives acquired venetoclax resistance in Acute Myeloid Leukemia and reveals targetable vulnerabilities
              </a>
            </td>
          <td>
            K. Lima, Gustavo Nery de Queiroz, Marcella Cipelli, Victória Tomaz, L. G. Cortes, Marina Basto Silva, R. Guedes, P. Campregher, Niels Câmara, L. Costa-Lotufo, E. Rego, J. Machado-Neto
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Relapse remains the leading cause of mortality in pediatric acute myeloid leukemia (AML), yet the genetic changes contributing to relapse remain incompletely defined. To address this gap, we performed whole-genome sequencing and targeted-capture sequencing on 39 diagnosis-relapse and 2 relapse-relapse pairs of pediatric AML. Mutational burden increased at relapse, largely reflecting spontaneous mutagenesis, whereas therapy-related signatures were rarely observed and only occasionally associated with pathogenic mutations. Although recurrently enriched mutations at relapse included those in FLT3, WT1, and TP53, relapse-fated subclones were frequently marked only by non-pathogenic or non-coding variants. Longitudinal deep sequencing in eight patients showed rapid depletion of major clones after induction therapy, whereas subclones often displayed variable chemosensitivity. Relapse-specific mutations emerged only late or remained undetectable during remission, suggesting that clonal selection of pre-existing clones is the predominant mechanism of relapse. Transcriptome analysis of paired RNA sequencing data revealed no differentially expressed genes, but gene set enrichment analysis and CIBERSORT deconvolution in each pair uncovered heterogeneous trajectories to relapse. Although relapse is often attributed to the emergence of stem-like phenotypes, our data demonstrate that transcriptional evolution is more diverse: some cases acquired stem-like features, whereas others showed partial differentiation, which was confirmed by re-analysis of a public single cell RNA sequence dataset. These changes were largely constrained by baseline differentiation states at diagnosis. Together, our data indicate that pediatric AML relapse arises through selection of pre-existing clones with diverse trajectories, underscoring the need to target both stem-like and differentiated populations to achieve durable cures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b28a4cea8f32f994db559bdedf94de2b973cd5" target='_blank'>
              Clonal Evolution of Pediatric Acute Myeloid Leukemia and Its Contribution to Disease Relapse
              </a>
            </td>
          <td>
            Masayuki Umeda, P. Kolekar, A. Huskey, S. Suryaprakash, Jing Ma, Hanxia Li, Tamara Westover, Michael P Walsh, Guangchun Song, Yanling Liu, Q. Tran, V. Balagopal, Lu Wang, Samuel W Brady, Tanja A. Gruber, S. Pounds, Paul E. Mead, Seth E. Karol, Hiroto Inaba, J. Rubnitz, Raul C. Ribeiro, Xiaotu Ma, J. Klco
          </td>
          <td>2025-11-06</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Drug tolerant persister cells (DTPs) refer to a transient drug-tolerance sub-population of cancer cells characteristics of phenotype plasticity and heterogeneity. This adaptive cell state is a critical transitional phase, standing on the crossroad that cancer cells reacquire drug sensitivity or enter into the permanent drug resistance. Emerging evidences indicate the epitranscriptomic regulations, particularly RNA methylations are the important mechanism underline post-transcriptional regulations of genes expression across all RNA species. RNA is integral to gene expression as messenger RNA (mRNA), transfer RNA (tRNA) and ribosomal RNA (rRNA), which play roles in transmitting information from DNA to the synthesis of functional proteins. Methylation modifications on these RNAs are prevalent and represent a well-recognized non-genetic mechanism, exerting multifaceted regulatory effects on nucleic acid metabolism, such as nucleotide precursor availability, RNA processing dynamics, sub-cellular localization, transcript stability and translational fidelity/ efficiency. This review systematically sorts out the relevant references, demonstrating recent advances on the knowledge of the patterns of methylation modifications on mRNA, tRNA and rRNA, and how these modifications drive the generation and development of DTPs, which hallmarks of epithelial-mesenchymal transition, metabolism shift and immune escape. And then clinical strategies are delineated, leveraging pharmacological modulators of RNA-modifying enzymes alongside non-pharmaceutical lifestyle advice, for the development of therapy strategies preventing DTPs-rooted tumor relapse in this anti-tumor armamentarium with cytotoxic reagents, targeted therapies and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098e2b3a70bd6f86f19a2d17ea69012683c3e930" target='_blank'>
              Therapy-induced mRNA, rRNA and tRNA methylation alterations confer tolerance phenotype in tumor cells: mechanism and implications.
              </a>
            </td>
          <td>
            Anfeng Jiang, Shujie Liu, Zhiyuan Li, Xiongzhou Zhang, Minghao Duan, Bin Li
          </td>
          <td>2026-01-01</td>
          <td>International journal of biological sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer recurrence remains a major clinical challenge, often associated with therapy resistance and altered metabolic states. To define metabolic vulnerabilities of recurrent disease, we performed a CRISPR knockout screen targeting 421 metabolic genes in paired primary and recurrent HER2-driven breast cancer cell lines. While both primary and recurrent tumors shared dependencies on core metabolic pathways, recurrent tumors exhibited selective essentiality for the de novo pyrimidine synthesis pathway, including Cad, Dhodh, and Ctps. Pharmacologic inhibition of the rate-limiting enzyme DHODH with BAY-2402234 selectively impaired the growth of recurrent tumor cells, while primary tumor cells were relatively resistant. BAY treatment robustly inhibited pyrimidine synthesis in all lines, but only recurrent cells underwent iron-dependent lipid peroxidation and ferroptotic cell death. Lipidomic profiling revealed enrichment of polyunsaturated ether phospholipids in recurrent cells, which may predispose them to ferroptosis. A sensitizer CRISPR screen in primary cells further identified nucleotide salvage and lipid metabolic pathways as modifiers of DHODH inhibitor sensitivity. Stable isotope tracing and nutrient depletion experiments showed that primary cells can compensate for DHODH inhibition through nucleotide salvage, whereas recurrent cells exhibit impaired salvage capacity, likely due to reduced expression of Slc28/Slc29 nucleoside transporters. Together, these findings reveal that breast cancer recurrence is associated with increased dependence on de novo pyrimidine synthesis to suppress ferroptosis, highlighting a therapeutically actionable metabolic vulnerability in recurrent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83ca3ef9a21a542254ed4d0723e6bc92c26d2a66" target='_blank'>
              Recurrent Breast Cancer Cells Depend on De novo Pyrimidine Biosynthesis to Suppress Ferroptosis
              </a>
            </td>
          <td>
            Kelley R. McCutcheon, Josh Wu, Yasemin Ceyhan Ozdemir, Brock J McKinney, Sharan Srinivasan, Ayaha Itokawa, Oliver J. Newsom, Anna-Lena B G Vigil, Douglas B. Fox, Lucas B. Sullivan, James V. Alvarez
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/768d3fe55602ad3e6e64d51b9bc66ae2727e39e3" target='_blank'>
              TP53 and RB1 are predictive genetic biomarkers for sensitivity to cytarabine in gliomas
              </a>
            </td>
          <td>
            L. Laaniste, Joachim Luginbuehl, Mi Trinh, Rachel Watkins, Gregor Lueg, Grant Neilson, Marjolein Burgers, Kasia Kania, Regina Reynolds, Zoe Smith, Julia Kleniuk, Lauren Gould, Dorota Pregowska, Sophie Mathiesen, Michael Johnson, Louwai Muhammed
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="SF3B1, a component of the U2 snRNP pre-mRNA splicing factor, plays a critical role in splicing and is frequently mutated in cancer, particularly hematologic malignancies. We investigated the effects of the most common SF3B1 mutation, heterozygous substitution of Lysine 700 to Glutamate (K700E), in human embryonic stem cells (hESC), using CRISPR-Cas9 to generate heterozygous SF3B1K700E clones. Interestingly, we observed the upregulation of several key transcription regulators associated with hematopoiesis and a broad range of immune genes in SF3B1K700E hESCs. Despite differences in the transcriptional and splicing profiles between hESC and myelodysplastic syndrome (MDS) cells harboring the SF3B1K700E mutation, several common immune gene programs were identified in both cell types. To elucidate the molecular mechanisms underlying dysregulated gene expression in SF3B1K700E hESCs, we mapped actively engaged RNA polymerase II (RNA Pol II) using Precision Run-On sequencing (PRO-seq). These analyses revealed that the SF3B1K700E mutation alters RNA Pol II elongation properties. Specifically, we observed a general increase in pause release in SF3B1K700E hESCs, consistent with recent work in leukemia cells suggesting that the SF3B1K700E mutation affects early transcription elongation. Taken together, our study identifies several candidate genes that could contribute to the SF3B1 mutated phenotype and clarifies the role for the U2 snRNP and pre-spliceosome assembly on transcription by RNA Pol II. Further, our data suggest that mutations of SF3B1 impact immune gene expression independent of cell type, providing new insights into the role of SF3B1K700E in hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21f566d363570cd1232df0a75a294a1715f4579d" target='_blank'>
              SF3B1K700E mutation in human embryonic stem cells causes aberrant expression of immune-related genes
              </a>
            </td>
          <td>
            M. Dastpak, Claudia A. Mimoso, Moein Farshchian, Christina L. Paraggio, Claudia E. Leonard, Shanye Yin, Binkai Chi, Kelsey W. Nassar, R. G. Rowe, Jiuchun Zhang, Zhonggang Hou, George Q. Daley, Karen Adelman, Robin Reed
          </td>
          <td>2025-12-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d53c86924fae7e6d4164c1b616a447b8ef85b5" target='_blank'>
              Comprehensive and accelerated mapping of driver mutations through single-nucleotide random mutagenesis of target genes
              </a>
            </td>
          <td>
            Keisuke Yoshida, Elena Solovieva, N. Miura, Shigeyuki Shichino, Michiaki Hamada, M. Muratani, Takanori Amano, Kazufumi Honda
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia is a genetically and cellularly heterogeneous disease. We characterize 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identifies four main clusters, thereby extending current genomic classification of AML. Notably, NPM1-mutated AML can be stratified into two clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation. NPM1class II is instead associated with resistance to allogeneic T cells in an ex vivo co-culture assay, and importantly, dismal survival following hematopoietic stem cell transplantation. These findings provide insights into the cellular state space of AML, define diagnostic entities, and highlight potential therapeutic intervention points. The clinical outcomes and treatment responses of acute myeloid leukemia (AML) patients are heterogeneous. Here, the authors use bulk and single-cell sequencing approaches and identify two transcriptomic subtypes within NPM1-mutated AML with distinct immune evasion properties and responses to hematopoietic stem cell transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0674b44d7df57e733fca27fc81a60b0eeaf92970" target='_blank'>
              The AML cellular state space unveils NPM1 immune evasion subtypes with distinct clinical outcomes
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Babesiosis is an emerging and re-emerging zoonosis that is prevalent worldwide, caused by over 100 Babesia species. These intracellular parasites efficiently invade host red bloods cells, replicate rapidly, and exit the infected cells to cause clinical symptoms. However, the underlying mechanisms of regulating this asexual stage is largely unknown. Here, we generated a chromosome-level reference assembly of a novel Babesia species, Babesia xinjiangensis. Using single-copy orthologous genes, we confirmed its phylogenetic relationships with other apicomplexan parasites and estimated its speciation time. We identified species-specific gene families and core gene families that could be responsible for species speciation and immune evasion. Furthermore, we also used a single-cell RNA-sequencing (scRNA-seq) protocol to uncover hidden transcriptional variations in the asexual stages of this unicellular Babesia parasite and its cell-to-cell heterogeneity. We inferred the replication cycle and performed a pseudotime analysis to speculate the gene expression profiles. Although the peak expression times of most epigenetic markers and transcription factors were confined to specific phases, BxAP2-M2 (GWHPERCV002055) is constantly expressed during asexual development progression. Genetic analyses revealed that BxAP2-M2 directly or indirectly regulates the expression of rhoptry proteins and membrane proteins, which may play critical roles in the parasite’s invasion of red blood cells and the merozoite morphology. Our findings provide valuable markers of asexual replication, including some that are specific to Babesia gametocytes, and regulators specific to distinct cell-cycle phases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcc905e6b3b252b6e7bea19a4cf0e4af2d5a472" target='_blank'>
              Comparative genomics, integrated with single-cell sequencing and genetic analyses, reveal roles of transcription factor AP2-M2 in asexual replication of Babesia parasite
              </a>
            </td>
          <td>
            Jinming Wang, Yijun Chai, Yuxin Ye, Jifei Yang, Xinxin Zhang, Hong Yin, Guiquan Guan
          </td>
          <td>2025-11-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Berberine (BBR), a natural compound with diverse anticancer properties, exerts potent inhibitory effects on lung cancer cell proliferation. However, its particular molecular targets remain unknown. The present study aimed to identify the key genetic determinants that mediate the cellular response to BBR in lung cancer. To achieve this, genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) knockout screening was conducted in A549 cells, followed by functional validation and mechanistic assays. In the study, the genome-wide CRISPR/Cas9 knockout screening in A549 lung cancer cells identified claudin-1 (CLDN1) as a critical modulator of BBR sensitivity. Notably, whilst CLDN1 was enriched in the positive selection screen, its knockout markedly increased the sensitivity of A549 cells to BBR, thus leading to enhanced G1-phase arrest and reduced proliferation. These findings suggest that CLDN1 could serve a dual role, promoting cellular resistance under selective pressure, while simultaneously demonstrating a therapeutic vulnerability when directly inhibited. Overall, the present study identified CLDN1 as a key mediator of the anticancer effects of BBR, thus providing a foundation for its potential development as a therapeutic target to optimize lung cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4f6034fc6b67d610759292e57a0e8b62f65752" target='_blank'>
              Genome-wide CRISPR screening identifies CLDN1 as a central node in the anticancer mechanisms of berberine and as a therapeutic target in lung cancer
              </a>
            </td>
          <td>
            Xiaogang Wang, Benzhong Mi, Jinfeng Lv, Xin Tang, Jingyu Huang, Fei Liu, Yajing Xing, Xingjian Wen, Yongde Wang, Yadong Zhong
          </td>
          <td>2025-12-01</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy arising from abnormal T-cell differentiation. Conventional treatment strategies remain suboptimal, and no effective targeted therapy is currently available. Here, we found that ASS1 is significantly upregulated in T-ALL and is associated with poor prognosis. Inhibition of ASS1 expression markedly suppresses T-ALL progression both in vitro and in vivo. Mechanistically, RNA-seq revealed that ASS1 depletion led to a significant downregulation of one-carbon metabolism and ribose synthesis pathways, and the mTORC1/c-Myc signaling pathway. Experimental validation confirmed that mTORC1/c-Myc signaling was significantly downregulated upon ASS1 blockage. Furthermore, the intracellular arginine level was significantly reduced following ASS1 knockdown, while supplementation with arginine enhanced mTORC1/c-Myc pathway activity. Notably, restoration of c-Myc effectively rescued T-ALL cells from the suppressive effects induced by ASS1 depletion. Collectively, our findings demonstrate that ASS1 supports mTORC1/c-Myc pathway activity by regulating arginine availability, thereby promoting the survival of T-ALL cells and leukemia progression. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-27576-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/444836a8f61f21800c05518b5f04592c6f4c2173" target='_blank'>
              ASS1 facilitates T-ALL progression via the arginine-mediated mTORC1/c-Myc signaling pathway
              </a>
            </td>
          <td>
            Zijian Zhang, Xiyu Liu, Qifang Wu, Jianwei Liang, Yuan Zhao, Jiaqi Deng, Yuqing Wang, Qingrun Yu, Zishan Ye, Haichuan Zhu, Jian Lin
          </td>
          <td>2025-12-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Therapeutic failure in cancer often arises from the ability of tumor cells to adapt to external stressors such as drug exposure, metabolic and oxidative stress, and immune surveillance within the tumor microenvironment (TME). This adaptive capacity is a significant factor contributing to poor clinical outcomes and the lethality associated with many aggressive cancer types. During tumor progression, a subpopulation of cells may acquire stem-like characteristics, leading to the development of cancer stem cells (CSCs), which play a pivotal role in resistance and disease recurrence. A key driver of this process is epigenetic reprogramming, which alters histone modifications and chromatin architecture in response to environmental stimuli. These modifications activate genes associated with stemness by creating transcriptionally permissive chromatin regions, while simultaneously silencing genes involved in differentiation, thereby maintaining the tumor cells’ undifferentiated state. This cellular plasticity enables cancer cells to undergo transdifferentiation or dedifferentiation, enhancing their capacity for survival and adaptation. Importantly, distinct transcriptional states are often regulated by specific epigenetic signatures, some of which are valuable for tumor subtype classification, prognosis assessment, and treatment response prediction. Furthermore, certain epigenetic biomarkers show considerable promise for detecting tumor recurrence and monitoring minimal residual disease with high sensitivity. In this review, we examine how epigenetic reprogramming influences chromatin plasticity to promote stemness and therapeutic resistance. Additionally, we discuss ongoing translational research aimed at leveraging these mechanisms to advance precision oncology, with the ultimate goal of improving diagnostic accuracy and therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) persists as a leading contributor to global cancer-associated mortality. Identifying key oncogenic drivers is crucial for improving therapeutic strategies. This study aimed to investigate the role of TICRR in LUAD progression and its potential as a therapeutic target. Methods Hub genes were screened through integration of DepMap CRISPR-Cas9 data, TCGA expression profiles, and survival analysis. TICRR expressions was assessed in LUAD tissues, adjacent controls, and cell lines by RT-qPCR and immunohistochemistry. Functional roles were examined using MTT, colony formation, transwell, wound healing, and EdU assays in vitro. Bioinformatics analyses, including GSEA, somatic mutation profiling, immune correlation, CMap drug sensitivity, and CT-based radiomics, were performed to explore mechanisms, therapeutic potential and clinical correlation. Results TICRR expression was observed to be significantly elevated in LUAD tissues and cell lines, and its higher levels correlated with unfavorable patient outcomes and enrichment of malignant pathways, including EMT, E2F targets, and PI3K/AKT/mTOR signaling. High TICRR expression correlated with distinct somatic mutation patterns, increased tumor mutation burden, and elevated immune checkpoint expression. In vitro, TICRR knockdown suppressed Lung cancer progression. CMap analysis identified KU0063794 and CDK inhibitors as potential therapeutic agents targeting TICRR. Using a CT-based radiomics approach, the predictive model revealed a positive correlation between TICRR infiltration and radiological features in LUAD patients. Conclusion TICRR functions as a critical oncogenic driver in LUAD, promoting proliferation, invasion, and immune evasion. Targeting TICRR may represent a novel strategy for personalized treatment of LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8879ea7af18a8373ac313167b49d629d7e64cc7" target='_blank'>
              Multiomics analysis demonstrated that TOPBP1 interacting checkpoint and Replication Regulator may serve as an immune-related biomarker indicative of poor prognosis in lung adenocarcinoma
              </a>
            </td>
          <td>
            Chengbin Lin, Ziyi Chen, Wenbin Song, Long He, Hongyan Yu, Gaofeng Liang, Keyun Zhu, Jinxian He, Cangchang Shi
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Comparative genetic studies of primates have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short (≤31 base pairs) hDels has been assessed using reporter assays. However, how structural variant-sized (≥50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 20 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the brain. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d81c8c2f5bb5b9ea0191569860cee91545d0b60" target='_blank'>
              Mapping cis- and trans-regulatory target genes of human-specific deletions
              </a>
            </td>
          <td>
            Tyler Fair, Bryan J. Pavlovic, Dani Swope, Octavio E. Castillo, Nathan K. Schaefer, Alex A. Pollen
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The androgen receptor (AR) is a critical driver of prostate cancer (PCa). Here, to study regulators of AR protein levels and oncogenic activity, we developed a live-cell quantitative endogenous AR fluorescent reporter. Leveraging this AR reporter, we performed genome-scale CRISPRi flow cytometry sorting screens to systematically identify genes that modulate AR protein levels. We identified and validated known AR protein regulators, including HOXB13 and GATA2, and also unexpected top hits including PTGES3—a poorly characterized gene in PCa. PTGES3 repression resulted in loss of AR protein, cell-cycle arrest and cell death in AR-driven PCa models. Clinically, analysis of PCa data demonstrates that PTGES3 expression is associated with AR-directed therapy resistance. Mechanistically, we show PTGES3 binds directly to AR, regulates AR protein stability and is necessary for AR function in the nucleus at AR target genes. PTGES3 represents a potential therapeutic target for overcoming known mechanisms of resistance to existing AR-directed therapies in PCa. Genome-wide CRISPRi screens for modulators of androgen receptor (AR) protein levels using live-cell quantitative endogenous AR fluorescence reporters identify PTGES3 as a new regulator of AR stability and function in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b624f946f0cc87b60d21d7777716b1e092cc49" target='_blank'>
              Genome-scale CRISPR screens identify PTGES3 as a direct modulator of androgen receptor function in advanced prostate cancer
              </a>
            </td>
          <td>
            Haolong Li, James E. Melnyk, Becky Xu Hua Fu, Raunak Shrestha, Meng Zhang, M. Sjöström, S. Feng, Jasmine A Anderson, Wanting Han, Lisa N Chesner, Hyun Jin Shin, Tatyanah Farsh, Humberto J. Suarez, Seema Nath, Jonathan Chou, Rajdeep Das, Emily A Egusa, Marsha Calvert, Audrey Kishishita, Abhilash Barpanda, Jun Zhu, Ashutosh Maheshwari, William S. Chen, M. Alshalalfa, Aidan Winters, Junjie T Hua, Tianyi Liu, E. Davicioni, Arun P. Wiita, B. Stohr, Javed Siddiqui, Bo Huang, Eric J. Small, K. Shokat, Peter S. Nelson, David A. Quigley, Elizabeth V. Wasmuth, Luke A. Gilbert, Felix Y. Feng
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e91ffccc5754debd09b64e59ef74200817ed97" target='_blank'>
              Transcriptomic and epigenetic profiling of neuroblastoma states in response to RBM39 degrader.
              </a>
            </td>
          <td>
            Hongjian Jin, Jie Fang, Jason Myers, Shivendra Singh, Jun Yang
          </td>
          <td>2025-12-09</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background and Objective: Acute myeloid leukemia (AML) is a genetically heterogeneous disease, but even isogenic leukemic populations exhibit phenotypic variability. To determine how non-genetic, biological heterogeneity influences growth, therapy response, and adaptation to metabolic stress, we generated and deeply characterized isogenic human and mouse AML clones. By longitudinally tracking single-cell clones, we found that metabolic plasticity, not newly acquired mutations or changes in transcriptional programs, dictates cell growth and therapeutic response to venetoclax. These data highlight the central importance of biological heterogeneity in AML that is not dictated by genetic changes.
 Methods: We isolated 29 single-cell clones from the MLL-rearranged AML cell line MOLM13 and assessed longitudinal differences in baseline growth, adaptation to nutrient stress, and drug response. Clones were tested under glutamine or serum restriction and treated with cytarabine, daunorubicin, venetoclax, and azacitidine. Mitochondrial respiration was measured using Seahorse assays. To explore the genetic basis for phenotypic divergence, we performed RNA-seq and whole-exome sequencing on the most venetoclax-sensitive and -resistant clones. We validated findings in a primary mouse MLL-AF9 AML using both a barcoding approach to tracking clonal dynamics by NGS and with individually isolated and cultured clones.
 Results: Even without exogenous stress, baseline growth rates varied >3-fold among MOLM13 clones and remained stable over time. Nutrient stress further amplified heterogeneity: glutamine deprivation suppressed growth by 40–70% across clones, whereas 0.1% FBS had a more uniform suppressive effect. Glutamine restriction reduced basal oxygen consumption rates (OCR) in all clones but preserved maximal respiration after FCCP uncoupling, indicating metabolic adaptation via alternative substrates. Faster-growing clones displayed higher basal OCR and relative resistance to venetoclax, while slower-growing clones were more venetoclax-sensitive. Cytarabine responses showed an opposite trend, suggesting that drug sensitivity is growth rate-dependent. The IC50 of venetoclax varied by ~12-fold, compared to ~2-fold for daunorubicin, indicating a context-dependent influence of clonal heterogeneity on therapy response. Next, we compared the genetic and transcriptional profile of clones that displayed the most sensitivity and resistance to venetoclax. RNA-seq showed no differential expression of BCL2-family genes between sensitive and resistant clones, and whole-exome sequencing revealed hundreds of unique, non-overlapping non-synonymous variants per clone, but no recurrent driver mutations were linked to the phenotype. These results suggest non-genetic mechanisms underlie observed phenotypic diversity. Finally, we extended our findings using the primary mouse MLL-AF9 AML. Using a lentiviral barcoding approach, we again observed interclonal heterogeneity and changes in clonal representation under glutamine restriction and venetoclax exposure. We then isolated and grew individual MLL-AF9 clones, following a similar approach to our work with the MOLM13 cells, and observed interclonal variability in OCR and drug sensitivity, confirming our findings in an independent, primary AML.
 Conclusions: Our study reveals substantial biological heterogeneity among isogenic AML clones, with stable differences in growth, metabolic plasticity, and drug response. These phenotypes were not attributable to known driver mutations, which indicate that non-genetic factors may be involved in clonal adaptation. Clonal traits were context-dependent: venetoclax and glutamine restriction revealed the most heterogeneity, suggesting that selective pressures shape phenotypic diversity. These findings highlight the need to consider isogenic clonal variability when interpreting experimental results or designing therapies and highlight that functional heterogeneity may be an underappreciated driver of therapeutic response in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffb7687231df66f3626536a54d60f11393391a61" target='_blank'>
              Clonal heterogeneity of acute myeloid leukemia under nutrient and chemotherapy stress
              </a>
            </td>
          <td>
            Christina Mayerhofer, Eve Crompton, Ernst Mayerhofer, Trine A. Kristiansen, Tsuyoshi Fukushima, Sam Keyes, A. Sharda, Dan Li, Carmen Da Silva, Deumaya Shrestha, Anna Kiem, T. Feuchtinger, David T. Scadden, P. Miller
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c23f53baca0dc3b384bd4e216c9b8503ff2be41" target='_blank'>
              Pooled single-cell CRISPRa/i screens for functional genomics in bacteria at scale
              </a>
            </td>
          <td>
            Jacob R. Brandner, Quoc Tran, Yujia Huang, Dmitry Sutormin, Karl D. Gaisser, Georg Seelig, J. Zalatan, James M. Carothers, A. Kuchina
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Motivation Understanding double-strand break (DSB) repair and its disruption is key to decipher genomic instability driving diseases such as cancer and reveal therapeutic avenues. Numerous genes have been linked to DSB repair for the first time in recent genome-wide perturbation studies assessing effects on mutational outcomes. However, the functional roles of most such genes remain poorly understood. Evidence from other studies shows that related genes similarly modulate the frequency of specific mutational outcomes following DSB repair, but analysis has largely ignored the multiplicity of gene functions and cross-talk between pathways. Here, we infer functional roles for candidate genes based on knockout mutational spectra by identifying mutational signatures shared with known genes and then mapping them to DSB repair functions. Signatures are identified using non-negative matrix factorization (NMF) to reflect functional multiplicity and cross-talk. Results We generated mutational spectra for mouse embryonic stem (mES) cells at three target sites across CRISPR knockouts of 307 known and 459 candidate DSB repair genes. Analysis using NMF revealed four mutational signatures associated with homology-directed repair (HDR), microhomology-mediated end-joining (MMEJ), and the initiation and ligation components of non-homologous end-joining (NHEJ). Signatures suggested that candidate genes Dbr1 and Hnrnpk could influence MMEJ and Fanconi anaemia (FA), and that loss of core NHEJ components (e.g. MRN complex or Ku proteins) could shift repair preference towards Ku-independent NHEJ. These findings demonstrate the utility of NMF for characterizing the contribution of genes to repair pathways and provide a foundation to discover new gene functions in DSB repair. Availability github.com/joanagoncalveslab/MuSpectraNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Introduction: Extensive characterization of the genetic landscape in AML has revealed a set of driver mutations sufficient to initiate leukemic transformation. However, some cases do not contain any of these known drivers and thus pose a clinical challenge for prognostication and targeted therapy. To address this, we employed a novel, proximity ligation based, linked-read whole genome NGS assay to identify single nucleotide variants (SNVs), short Insertions/Deletions (InDels), copy number variation (CNV), structural variants (SVs) and 3D genome contact maps in AML. By employing this approach, we investigate whether comprehensive detection of genetic alterations in conjunction with insights into the epigenetic state of leukemic cells can reveal important biological information in cases lacking canonical driver mutations.
 Methods: We curated a cohort of patient samples where concomitant targeted sequencing by NGS and conventional karyotyping showed no detectable driver mutations. Further, we included control specimens with either known AML drivers as well as remission specimens after allogeneic transplantation. We utilized a multi-purpose NGS assay (Dovetail® LinkPrep™) to detect SNVs, InDels, CNVs, SVs and 3D genome contact maps. Peripheral blood or bone marrow cells were thawed and immediately fixed, subjected to crosslinking and in situ tagmentation, followed by library prep. Libraries were sequenced on Illumina sequencers with a target of 400M reads per library. Data analysis was performed using both published and custom tools to call SNVs/InDels (DeepVariant), Copy Number estimates (Purple), Structural Variants (HiC-Breakfinder, DTG-SELVA) and 3D contact maps (Juicebox).
 Results: We sequenced whole genome libraries to achieve an average coverage of 50.4x (range 41.1x - 58.1x) showing a long-range linkage percentage of 26.2% (range 17.9% - 31.8%). SNVs and short InDels, such as likely pre-leukemic mutations in DNMT3A, TET2, and mutations in GATA2 were detected reliably. Of note, the linked-read nature of this assay allowed us to perform phasing in a case with two TET2 mutations and confirm the compound heterozygous nature of these two TET2 mutations at a distance of 24kb. Further, known AML drivers in control specimens in genes such as NPM1, FLT3 and KRAS were reliably detected with variant allele frequencies as detected by targeted sequencing. Surprisingly, however, we found new structural variants in half of our cohort, including the diagnostic specimens with no detectable driver mutations, despite all specimens having no chromosomal abnormalities by conventional karyotyping. These new SVs included large intrachromosomal inversions and unbalanced translocations. We validated the presence of all SVs in sorted leukemic blasts and absence in sorted T cell fractions by PCR and Sanger sequencing. We subsequently examined the 3D genome contact maps and performed RNA-Sequencing to determine the epigenetic state and potential biological consequences across these novel breakpoints. We found several neoloops connecting enhancers with promoters across these breakpoints in 3D contact maps, and gene expression dysregulation proximal to the SVs by RNA-Seq. When we evaluated the expression of these putative driver genes across a cohort of healthy donors and AML patients, we found lack of expression in healthy cells but significantly increased expression in 9-10% of patients with AML (n=777 patients). Finally, we also identified an undetected structural variant in engrafted donor cells from a patient with myelodysplastic neoplasms (MDS) in remission after allogeneic transplantation, highlighting the potential to assay SVs in the context of non-leukemic hematopoiesis as well.
 Conclusion: In this exploratory cohort, we find several otherwise undetected structural variants in AML cases without known driver lesions using a proximity ligation based, linked-read whole genome NGS assay while maintaining our ability to detect recurrent, known driver lesions. This approach allowed us to interrogate the biological consequences of these genomic alterations by leveraging the ability to not only detect genomic variants but also analyze the epigenetic state through 3D genome contact maps. Our results imply that our current understanding of the genomic landscape in AML might be underestimating a sizable number of structural variants that are difficult to detect using standard assays, implying putative novel driver genes in AML pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f033c80701d9a0710865b0dce2b79cfc9a9b774" target='_blank'>
              Sensitive detection of novel structural variants and 3D chromosome conformation reveals likely novel drivers including enhancer hijacking in AML
              </a>
            </td>
          <td>
            Thomas Koehnke, A. Ediriwickrema, Mital S. Bhakta, Alexander Fortuna, Charu Tiwari, Zackary Sanborn, Tian Zhang, Lisa Munding, Ravi Majeti
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81b381c14e5b1230a77d230dfeaa5698f06f2cc" target='_blank'>
              Characterizing resistance in prostate cancer at a single cell level with hormonal treatment and epigenetic inhibitors.
              </a>
            </td>
          <td>
            Morgan Howells, Priyadarsini Gangadharannambiar, Calum McMullen, Francesco Crea, W. Bacon, Colette Christiansen
          </td>
          <td>2025-11-03</td>
          <td>The Journal of pharmacy and pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) is the most common adult acute leukemia. The traditional “3+7” induction regimen with anthracycline and cytarabine remains a standard therapy, but 30%-40% of patients develop drug resistance to anthracyclines, leading to relapse. Refractory/relapsed AML has dismal survival rates, posing a major challenge. Elucidating mechanisms of therapy resistance and relapse is crucial for improving outcomes in high-risk AML.
 Methods ATAC-seq analyzed chromatin accessibility of anthracycline-sensitive and -resistant AML cells, focusing on METTL3 locus. RNA-seq, qRT-PCR and Western blot measured METTL3 expression in cell lines and primary cells. METTL3 was regulated by small-molecule inhibition, shRNA knockdown and overexpression to study its effect on m6A modification and phenotypes in resistant cells. MeRIP-seq and RNA-seq identified METTL3 target genes in shMETTL3-transfected resistant cells. MeRIP-qPCR validated m6A-modified mRNAs and actinomycin D assays tested RNA stability. BMI1 and METTL3 inhibitors were used to explore their functional relationship in drug resistance.
 Results
 In anthracycline-sensitive AML patients, over 50% of the global chromatin accessible regions are enriched in promoter regions. Conversely, in anthracycline-resistant AML patients, promoter-accessible regions account for less than 25% of the total. Treatment with anthracyclines induces changes in promoter chromatin accessibility in only approximately 5% of the genome in sensitive patients, whereas this proportion reaches approximately 25% in resistant patients. Notably, METTL3 exhibits significantly elevated chromatin accessibility in anthracycline-resistant AML cells. Following anthracycline exposure, METTL3 accessibility is further enhanced in resistant cells but decreased in sensitive cells. m6A modification was significantly upregulated in AML drug-resistant cells, accompanied by consistently higher METTL3 expression. Integrative multi-omics analyses identified HOXA9 as a key transcriptional regulator of METTL3, with HOXA9 positively modulating METTL3 expression. Clinical datasets (TCGA and GSE6891) revealed that elevated HOXA9 expression was significantly associated with poor prognosis in AML patients. HOXA9 levels were markedly higher in drug-resistant and relapsed/refractory (R/R) AML cells. HOXA9 knockdown in resistant cells suppressed proliferation and induced apoptosis. Dual-luciferase reporter assays confirmed that HOXA9 specifically bound to the “TAACGTG” motif within CpG island 3 of the METTL3 promoter.
 Mechanistically, reducing METTL3 expression in drug-resistant cells decreased m6A modification, inhibited cell proliferation, and triggered apoptosis. Conversely, METTL3 overexpression in sensitive cells enhanced m6A modification, promoted proliferation, suppressed apoptosis, and augmented doxorubicin resistance. METTL3 promoted m6A modification of BMI1 mRNA, stabilizing BMI1 transcripts and elevating BMI1 expression in anthracycline-resistant cells.
 BMI1 was overexpressed in anthracycline-resistant cells and correlated with poor prognosis in AML patients. Inhibiting BMI1 in resistant cells induced dose-dependent proliferation arrest and apoptosis, while restoring sensitivity to anthracyclines.
 Conclusion Our study demonstrates that chromatin accessibility in the promoter regions of anthracycline-resistant AML cells is significantly decreased compared to anthracycline-sensitive counterparts. Anthracycline elicits more pronounced changes in promoter chromatin accessibility in resistant cells than in sensitive cells, underscoring a heightened involvement of promoter-level epigenetic regulation in anthracycline-resistant AML. METTL3 exhibits both upregulated expression and enhanced chromatin accessibility in anthracycline-resistant AML cells relative to sensitive cells. Furthermore, anthracycline exposure leads to a further increase in chromatin accessibility at the METTL3 promoter in resistant cells, suggesting dynamic epigenetic modulation of this key m6A regulator during drug resistance. Our study demonstrated that The HOXA9/METTL3/BMI1 axis promotes anthracycline resistance in acute myeloid leukemia through mRNA m6A modification and suggested the therapeutic potential of targeting the METTL3/BMI1 axis in the treatment of anthracycline-resistant AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266257a13a97162936fed07815814a1d0db44fc4" target='_blank'>
              HOXA9/METTL3/BMI1 axis is required for promoting anthracycline resistance in acute myeloid leukemia cells by modulating BMI1 expression through  mRNA m6A modification
              </a>
            </td>
          <td>
            Hao Wang, Daihong Liu, Liping Dou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Refractory cancers may arise either through the acquisition of resistance mechanisms or represent distinct disease states. The origin of childhood T-cell acute lymphoblastic leukaemia (T-ALL) that does not respond to initial treatment, i.e. refractory disease, is unknown. Refractory T-ALL carries a poor prognosis and cannot be predicted at diagnosis. Here, we perform single cell mRNA sequencing of T-ALL from 58 children (84 samples) who did, or did not respond to initial treatment. We identify a transcriptionally distinctive blast population, exhibiting features of innate-like lymphocytes, as the major source of refractory disease. Evidence of such blasts at diagnosis heralds refractory disease across independent datasets and is associated with survival in a large, contemporary trial cohort. Our findings portray refractory T-ALL as a distinct disease with the potential for immediate clinical utility. T-cell acute lymphoblastic leukemia is a highly aggressive disease with varying recurrence rates. Here, the authors build a single cell transcriptomic atlas of childhood T-cell acute lymphoblastic leukaemia (T-ALL). They identified a distinctive cancer cell state that correlates with high risk, treatment refractory T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb3ad5f4d75526c87385fa8cfc2e02d2569e276" target='_blank'>
              A non-canonical lymphoblast in refractory childhood T-cell leukaemia
              </a>
            </td>
          <td>
            Bram S. J. Lim, Holly J. Whitfield, Mi K. Trinh, Gianna Bloye, Rebecca Thomas, Nathaniel D. Anderson, Anna Wenger, A. Hodder, T. D. Treger, H. Lee-Six, Tim H. H. Coorens, Conor Parks, Toochi Ogbonnah, P. Pölönen, C. Mullighan, D. Teachey, Jason Xu, Kai Tan, Melanie M. Hagleitner, L. Kester, F. V. van Leeuwen, Gordon Beattie, Marc R. Mansour, Owen Williams, Jack Bartram, S. Adams, Laura Jardine, Sam Behjati, D. O'connor
          </td>
          <td>2025-11-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="


 INTRODUCTION Multiple Myeloma (MM) is a genetically complex blood cancer marked by diverse mutations, structural rearrangements, and chromosomal abnormalities that drive disease progression and treatment resistance. Traditional genomic approaches often miss critical alterations due to technical limitations in resolving complex regions. Oxford Nanopore long-read sequencing offers unprecedented insight by generating ultra-long reads capable of spanning difficult genomic landscapes. ONT's innovative Adaptive Sampling technology enables high-resolution detection of clinically relevant mutations and structural variants without the need for laborious capture or amplification steps, opening new avenues for comprehensive and rapid MM genomic profiling.
 METHODS We designed a custom adaptive sampling panel covering ∼6% of the genome, targeting key genes, structural hotspots, and translocation-prone regions relevant to cancer. Genomic DNA from CD138+ cells of 17 MM patients was processed using ONT protocols and sequenced on the PromethION (P2i) platform with Adaptive Sampling enabled. We assessed enrichment efficiency, coverage, and variant detection. All patients had been previously characterized using gold standard methods, including FISH for del(17p), del(1p), amp(1q), t(4;14), t(14;16), and a targeted 38-gene NGS panel for multiple myeloma.
 RESULTS This technology enabled molecular karyotyping of all 17 patients through whole-genome characterization. Using adaptive sampling, we performed low-coverage whole-genome sequencing, which facilitated the detection of large-scale structural variants and copy number alterations across the entire genome. Among the 17 patients, 2 showed no major chromosomal alterations; 4 were classified as hyperdiploid, each presenting trisomy's in various odd-numbered chromosomes. One patient exhibited an isolated del(1p), while another showed an isolated amp(1q). The remaining 7 patients displayed diverse combinations of genomic abnormalities, including del(1p), amp(1q), monosomy 8, monosomy 13, del(17p), and hyperdiploidy. Across 51 comparable probes (3 CNV targets per patient), 94.1% (48/51) showed concordant results, while the 3 discordant cases corresponded to small deletions in the short arm of chromosome 1 (1p), likely undetected by conventional FISH probes.
 We initially focused on rearrangements involving the IGH locus on chromosome 14, given their relevance in multiple myeloma. All patients had previously been evaluated by FISH for t(4;14) and t(14;16) as stablished prognostic markers and were negative for both. However, long-read sequencing identified IGH rearrangements in two distinct patients: one involving t(14;20) and the other t(11;14), both of which have been previously reported in MM. In addition, 88% [15/17] of the patients exhibited other chromosomal translocations that warrant further characterization to assess their potential biological and clinical significance in the disease.
 Regarding SNVs, the short-read targeted NGS panel did not detect variants in 5 out of 17 patients. In contrast, long-read sequencing identified pathogenic variants with variant allele frequencies (VAFs) above 5% in all 17 patients, suggesting that exome sequencing with long-read technology enhances the detection of clinically relevant alterations. Furthermore, the use of a clinical exome enables the identification of variants in genes associated with other diseases or with potential pharmacogenomic implications, as it is not limited to a predefined set of targets.
 CONCLUSIONS Preliminary results indicate that the proposed technology enables the simultaneous detection of the main genomic alterations required for a comprehensive diagnostic workup in patients with multiple myeloma (MM), using a single, unified assay. Its agnostic design, based on broad genomic coverage without pre-selection of specific regions, allows for an unbiased analysis of the tumor genome. This approach not only facilitates the integration of multiple diagnostic markers into a single workflow but also supports potential scalability to other hematologic malignancies or solid tumors, as it is not restricted to a predefined set of genomic loci. The flexibility, robustness of the method and the short turnaround time for comprehensive characterization of MM patients position it as a promising tool for precision oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87400f9201c3814073c1f811caea785b333c8c8a" target='_blank'>
              Targeted long-read sequencing with adaptive sampling in multiple myeloma: A pilot study toward a unified diagnostic platform
              </a>
            </td>
          <td>
            B. Ariceta, P. Aguirre-Ruiz, Andrea Castro, Patxi San Martín-Úriz, Saioa Vicente, M. Larráyoz, L. Tamariz-Amador, A. Alfonso-Piérola, Jesús F San-Miguel, M. Calasanz, P. Rodríguez-Otero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Interleukin-1β (IL-1β) is dysregulated in chronic inflammatory diseases, yet the genetic factors influencing IL-1β production remain largely unknown. Myeloid-derived cells are the primary producers of IL-1β, which prompted a genome-wide CRISPR knockout screen in the human myeloid-derived U937 cells treated with lipopolysaccharide (LPS) to mimic inflammatory conditions and sorted for high and low intracellular IL-1β levels. A total of 295 genes are identified as regulators of IL-1β production, with 57 overlapping loci associated with inflammatory diseases, including the TNRC18 gene locus associated with multiple diseases in the Finnish population. U937 cells engineered with the Finnish-enriched rs748670681 risk allele demonstrate decreased expression of TNRC18 and an adjacent gene WIPI2, reduction in LPS-dependent gene activation and cytokine production, but elevation of interferon-responsive gene programs. Transcriptomic profiles for individual knockouts of TNRC18 and WIPI2 attribute the loss of LPS-dependent signaling primarily to TNRC18, which occurs through the modulation of H3K27 acetylation around inflammatory regulatory regions via TNRC18 and its protein interaction network. In contrast, the loss of WIPI2 is characterized by an exacerbation of interferon signaling. These findings delineate the global regulatory mechanisms of IL-1β production and provide molecular insights to the role of the rs748670681 variant in inflammatory diseases. Here the authors perform a gene knockout screen in myeloid cells, identifying 295 genes regulating interleukin-1β production, of which 57 lie in regions associated with inflammatory disease risk. The study sheds light on genetic control of interleukin-1β in inflammation, beyond previously known factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6bb7d17fa85a51ef67d08915963f06f1fb79c7" target='_blank'>
              A genome-wide CRISPR screen identifies the TNRC18 gene locus as a regulator of inflammatory signaling
              </a>
            </td>
          <td>
            F. Rahimov, Sujana Ghosh, Sakina M. Petiwala, Mary Schmidt, Eugene Nyamugenda, Manman Shi, Jason Tam, Daniel Verduzco, Sanjana Singh, Victor Avram, Apexa Modi, Celso A. Espinoza, Charles Lu, Jing Wang, Ashleigh Keller, Michael Macoritto, N. Mahi, Tifani Anton, Namjin Chung, Michael J. Flister, Kanstantsin V. Katlinski, Amlan Biswas, Anneke I. den Hollander, Jeffrey F. Waring, Joshua D. Stender
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="CRISPR-based gene activation (CRISPRa) has emerged as a promising therapeutic approach for neurodevelopmental disorders (NDD) caused by haploinsufficiency. However, scaling this cis-regulatory therapy (CRT) paradigm requires pinpointing which candidate cis-regulatory elements (cCREs) are active in human neurons, and which can be targeted with CRISPRa to yield specific and therapeutic levels of target gene upregulation. Here, we combine Massively Parallel Reporter Assays (MPRAs) and a multiplex single cell CRISPRa screen to discover functional human neural enhancers whose CRISPRa targeting yields specific upregulation of NDD risk genes. First, we tested 5,425 candidate neuronal enhancers with MPRA, identifying 2,422 that are active in human neurons. Selected cCREs also displayed specific, autonomous in vivo activity in the developing mouse central nervous system. Next, we applied multiplex single-cell CRISPRa screening with 15,643 gRNAs to test all MPRA-prioritized cCREs and 761 promoters of NDD genes in their endogenous genomic contexts. We identified hundreds of promoter- and enhancer-targeting CRISPRa gRNAs that upregulated 200 of the 337 NDD genes in human neurons, including 91 novel enhancer-gene pairs. Finally, we confirmed that several of the CRISPRa gRNAs identified here demonstrated selective and therapeutically relevant upregulation of SCN2A, CHD8, CTNND2 and TCF4 when delivered virally to patient cell lines, human cerebral organoids, and a humanized mouse model of hTcf4. Our results provide a comprehensive resource of active, target-linked human neural enhancers for NDD genes and corresponding gRNA reagents for CRT development. More broadly, this work advances understanding of neural gene regulation and establishes a generalizable strategy for discovering CRT gRNA candidates across cell types and haploinsufficient disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c894b7f9e1d5db5f69bf147d03e982a27eebec4" target='_blank'>
              Large-scale discovery of neural enhancers for cis-regulation therapies
              </a>
            </td>
          <td>
            Troy A. McDiarmid, Nicholas F. Page, Florence M. Chardon, R. Daza, George T. Chen, Michael Kosicki, Lucas M. James, Hannah C. Nourie, Dianne Laboy-Cintrón, Arthur S. Lee, , Diego Calderon, Jean‐Benoît Lalanne, Beth K. Martin, Kyle D. Fink, Michael Talkowski, A. Muotri, Benjamin D. Philpot, Len A. Pennacchio, D. Geschwind, Stephan J. Sanders, N. Ahituv, Jay Shendure
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The mitochondrial glutamine transporter SLC1A5_var plays a central role in the metabolic reprogramming of cancer cells by facilitating glutamine import into mitochondria for energy production and redox homeostasis. Despite its critical function, the development of effective and selective inhibitors targeting SLC1A5_var has remained a significant challenge. Here, we introduce iMQT_020, a selective allosteric inhibitor identified through structure-based screening. iMQT_020 disrupts the trimeric assembly of SLC1A5_var, causing metabolic crisis in cancer cells and selectively suppressing their growth. Mechanistically, iMQT_020 reduces glutamine anaplerosis and oxidative phosphorylation, resulting in a broad disruption of cancer metabolism. Additionally, iMQT_020 treatment epigenetically upregulates PD-L1 expression, enhancing the efficacy of combination therapies with anti-PD-L1 immune checkpoint inhibitors. These findings highlight the therapeutic potential of targeting SLC1A5_var as a critical metabolic vulnerability in cancer and demonstrate that targeting allosteric interprotomer interactions is a novel and promising therapeutic strategy for cancer treatment. Glutamine addiction is a hallmark of many cancers. iMQT_020, a first-in-class allosteric inhibitor of the mitochondrial glutamine transporter SLC1A5_var, disrupts glutamine-dependent mitochondrial metabolism, selectively killing cancer cells and enhancing immune checkpoint inhibitor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6282212ca0c4f23038a97cd9bbfe2c84422eef1" target='_blank'>
              Targeting cancer glutamine dependency with a first-in-class inhibitor of the mitochondrial glutamine transporter SLC1A5_var
              </a>
            </td>
          <td>
            Yulseung Sung, Ya Chun Yu, Mirim Lee, Seonghun Lim, Yechan Lee, Mincheol Kang, D. Kwon, Apeksha Parajulee, Junjeong Choi, Do Sik Min, Kuglae Kim, W. Namkung, Yun-Hee Kim, Sang Myung Woo, Nam Doo Kim, H. Yoo, Jung Min Han
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by complex genomic background and treatment resistance. We first defined “Titin (TTN) inactivation”, a state of TTN expression deficiency or mutation, affecting tumor progression. However, how TTN inactivation regulates immune escape and affects therapeutic resistance remains unclear. Using whole-exome sequencing, single-cell transcriptome sequencing, and spatial transcriptome sequencing, we screened the clinical features of TNBC patients with TTN inactivation who accepted neoadjuvant therapy. Meanwhile, we used CRISPR-Cas9 technology to construct various mutant TNBC cell lines. Lentiviral vector carrying TTN and delta-like ligand 4 (DLL4) was validated in vivo to verify potential mechanism. Myeloid-derived suppressor cells (MDSCs) metabolic function was measured using glycolysis-related molecular experiments. Immunotherapeutic agents against TNBC with TTN inactivation were explored in the orthotopic MCT4fl/fl genetically modified mouse model. DLL4-regulated ecological niche was established in TNBC with TTN inactivation. Mechanistically, TTN deficiency and mutation led to DLL4 secretion in TNBC. DLL4 enhanced MCT4-mediated glycolysis via MDSCs-derived NOTCH2 signaling pathway, driving the malignant function and lactate acid excretion of MDSCs. DLL4-derived MDSCs promoted stemness-mediated drug resistance by inducing histone lactate modification in TNBC, suppressing the anti-tumor activities of CD8+T cells. Blocking the DLL4-MCT4 axis stimulated anti-tumor immunity and synergized with anti-PD-1, improving response rates for first-line neoadjuvant therapy in TNBC. Our study revealed intrinsic mechanism by which TTN regulates the tumor immune microenvironment and provided a potential target for immunotherapy in TNBC with TTN inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf824ac3335bab02348fceec44a41f866d2add7b" target='_blank'>
              Delta-like ligand 4 mediated myeloid-derived suppressor cell metabolic reprogramming promotes neoadjuvant therapy resistance in titin-inactivated triple-negative breast cancer
              </a>
            </td>
          <td>
            Yanfang Yang, Ziyun Liu, Parhat Kaysar, Yuxi Han, Bo Ni, Linwei Li, Lina Zhang, Xiaobin Shang, Yaoyao Zhou, Yongjie Xie, Zhansheng Jiang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f697ecd66287673460b53084d2f61f279143459" target='_blank'>
              FOXA1 mutations co-opt nascent transcription factor networks in partnership with androgen receptor to enhance prostate tumorigenicity
              </a>
            </td>
          <td>
            Erik Ladewig, Abbas Nazir, Tyler Park, Vinson B. Fan, Zhendong Cao, Jacob Hawk, Lauren Kelly, Robert Tjian, Christina S. Leslie, Charles L. Sawyers
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df1c06700a8d3d29d2328428496f19af36bbcc45" target='_blank'>
              A comprehensive view of somatic mosaicism by single-cell DNA analysis
              </a>
            </td>
          <td>
            Lovelace J. Luquette, Tim H. H. Coorens, Abhiram Natu, M. Šuvakov, Ann Caplin, Mee Sook Jun, Alisa Mo, Joe Pelt, Lisa Anderson, Michele Berselli, Sravya V Bhamidipati, Thomas G Blanchard, Joseph A. Brew, Hye-Jung E. Chun, Hyunbin Chun, Mrunal K. Dehankar, William C. Feng, Rob Furatero, Christopher M. Grochowski, Eric Ho, Yeongjun Jang, Kavya Kottapalli, M. K. Leonard, Nam Seop Lim, Tina Lindsay, Sarah Nicholson, Ivan Raimondi, Alexi Runnels, Constantijn Scharlee, JaeEun Shin, Alexander D. Veit, M. VonDran, Yifan Wang, Dennis J. Yuan, Yifan Zhao, SMaHT Single Cell, Focus Group, Thomas J. Bell, K. Ardlie, Harsha Doddapaneni, Robert S. Fulton, S. Germer, Dan Landau, Jihyeon Oh, Peter J. Park, F. Vaccarino, Christopher A. Walsh, A. Abyzov
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>61</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains clinically challenging due to its molecular heterogeneity and poor outcomes, highlighting the urgent need for novel biomarkers and therapeutic targets. Purpose This study aims to identify and characterize the role of leucine-rich α-2-glycoprotein 1 (LRG1) in AML, evaluating its potential as both a prognostic biomarker and a therapeutic target. Methods We conducted an integrated basic and clinical investigation of LRG1 in AML. Methods included analysis of LRG1 expression in patient samples versus controls and pre- versus post-treatment, assessment of its clinical correlations with mutations and subtypes, and evaluation of its prognostic impact. Functional validation was performed using LRG1 knockdown models to assess effects on colony formation, apoptosis, and differentiation. Single-cell RNA profiling was utilized to identify LRG1-enriched cell populations and explore its role in microenvironmental crosstalk. Results Integrated analysis revealed significantly elevated LRG1 expression in AML patients compared to controls (P<0.001), with levels decreasing post-treatment (P<0.001). High LRG1 expression correlated with FLT3 mutations (P<0.01), M3-M5 AML subtypes (M0&M1&M2 VS M3, P<0.001; M0&M1&M2 VS M4, P<0.01; M0&M1&M2 VS M5, P<0.001; M3 VS M5, P<0.05; M4 VS M5, P<0.05), and worse survival (P<0.01). Functionally, LRG1 knockdown impaired colony formation (P<0.001), increased apoptosis (P<0.001), and disrupted differentiation (P<0.01). Single-cell profiling identified LRG1 enrichment in hematopoietic stem and progenitor cells (HSPCs) and myeloid progenitors, where it facilitated microenvironmental crosstalk via Macrophage Migration Inhibitory Factor (MIF), Galactoside-binding lectin (GALECTIN), and Cyclophilin A (CypA) signals. Conclusion Our findings establish LRG1 as a robust prognostic biomarker and a key functional regulator of AML maintenance through myeloid progenitor dysregulation, presenting it as a promising target for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c85b059f315aeac16b816e67ba97682c83e790" target='_blank'>
              LRG1 Drives AML Progression by Disrupting Myeloid Progenitor Regulation
              </a>
            </td>
          <td>
            Rongxia Guo, Peng Wu, Shuxin Yao, Fumei Liu
          </td>
          <td>2025-12-01</td>
          <td>Blood and Lymphatic Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8838d113f21c285c0455db7c93dfda5adc298f" target='_blank'>
              A switch from histone methyltransferase-EZH2 to demethylase KDM6A activity marks reinitiation of proliferation of drug treated colorectal cancer cells
              </a>
            </td>
          <td>
            Subhashree Chatterjee, Ritika Jaiswal, Aniruddha Roy, S. Chowdhury, Sudeshna Mukherjee, Rajdeep Chowdhury
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10582cdaf7507df9370133688fbcd15ae5208a8b" target='_blank'>
              Multiomic characterization, early detection, and therapeutic targeting of myeloid sarcoma
              </a>
            </td>
          <td>
            B. Nadorp, Audrey Lasry, S. Loghavi, Ravi Patel, Hager Mansour, Benjamin J. Kelly, Christopher J. Walker, Jill Buss, Isaiah Boateng, Wafa Al-Santli, Zoe Ciantra, Rebecca Austin, Adrienne Heyrosa, Helee Desai, Andrea Laganson, Hasan Abaza, Linda Procell, Tejas Patel, Benjamin Kaffenberger, Saranga Wijeratne, Maria Guillamot, Maria Velegraki, Luis Chiriboga, Zihai Li, L. Abruzzo, Daniel A. Pollyea, Christine M McMahon, A. Shana'ah, John C. Byrd, Alan H Shih, R. L. Levine, E. Papapetrou, A. Tsirigos, Elaine R. Mardis, Alice S. Mims, I. Aifantis, Ann-Kathrin Eisfeld
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Ferroptosis has emerged as a promising therapeutic vulnerability of diverse malignancies, yet the regulatory circuits adopted by each in cholangiocarcinoma (CCA) subtypes remain incompletely understood. We integrated the genome-wide CRISPR–Cas9 loss-of-function screens and transcriptomic profiles of the Cancer Dependency Map and then systematically assessed the essentiality of ferroptosis suppressor genes (FSGs) in the intrahepatic (iCCA) and extrahepatic (eCCA) subtypes. Nineteen and 16 essential FSGs were identified in iCCA and eCCA, respectively, among which GPX4 exhibited a significantly higher dependency in iCCA. Pharmacological inhibition of GPX4 with RSL3 markedly reduced cell viability and induced lipid peroxidation in iCCA cell lines, whereas eCCA cell lines displayed pronounced resistance associated with elevated GPX4 expression. A transcriptomic comparison revealed enrichment of WNT signaling in eCCA. Co-treatment with the tankyrase inhibitor XAV-939 and RSL3 enhanced growth inhibition of eCCA cells, indicating that WNT signaling contributed to ferroptosis resistance. These findings indicate that iCCA exhibits a preferential dependency on GPX4, whereas WNT–β-catenin signaling mediates resistance in eCCA. Collectively, the results clarify the molecular basis of subtype-specific ferroptosis vulnerability and offer a rationale for combinatorial therapeutic strategies that integrate GPX4 and WNT pathway inhibition when treating refractory eCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27285ae27dba13909a85b18f9e67566514450b07" target='_blank'>
              Characterization of Differential GPX4 Essentiality Between Intrahepatic and Extrahepatic Cholangiocarcinoma via Leveraging of a Large-Scale Functional Genomic Screen
              </a>
            </td>
          <td>
            Ye Rim Lee, Chaeyoung Seo, Mohamed M. Abdullah, Su Hyun Baek, Seung Jin Lee
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Background: Fusion transcripts are formed by combining exons from two different genes, often due to structural rearrangements such as deletions, inversions or translocations (genomic rearrangement-dependent, GRD) or through aberrant splicing (genomic rearrangement-independent, GRI). In hematological malignancies, many fusion transcripts act as driver events, playing crucial roles in leukemogenesis and serving as diagnostic markers, such as BCR::ABL1. However, the role and diagnostic significance of other fusion transcripts, like P2RY8::CD99—which has been found in both healthy samples and B-ALL patients—are less clear. Similarly, the clinical relevance of other GRI fusion transcripts, such as SEMA6A::FEM1C, often remains uncertain. Single-cell analysis is a promising approach to detect these fusion transcripts within individual cells and cell populations, shedding light on the subclonal architecture and cells of origin.
 Aim: Evaluating the feasibility of integrating long-read sequencing with single-cell library preparation to detect and characterize GRD and GRI fusion transcripts.
 Patients and Methods: Our cohort consisted of 10 samples diagnosed with B-ALL, featuring various fusion transcripts: BCR::ABL1 (n = 6, GRD), SEMA6A::FEM1C (n = 10, GRI), EBF1::PDGFRB (n = 1, GRD), and P2RY8::CD99 (n = 1, GRI), as identified through bulk whole transcriptome analysis. Cryopreserved cells were processed using the GEM-X Universal 3' Expression Library Prep Kit (10x Genomics) and GEM-X cDNAs with cell-specific labels served as input for the Kinnex single-cell RNA kit (PacBio). GEM-X libraries were sequenced on the NovaSeqX instrument (Illumina) with a median depth of 21,565 reads per cell. Kinnex libraries on the Revio (PacBio) with 3,767,066 mean HiFi reads per sample and a mean HiFi read length of 14.2kb. GEM-X libraries were analyzed using cellranger (v9.0.1) and seurat (v5.2.1). Kinnex libraries were preprocessed with the Iso-Seq workflow (PacBio) and fusion transcripts were called with pbfusion.
 Results: Cells from different samples were merged and clustered based on their gene expression profiles from GEM-X libraries. Cell clusters were annotated as B-cells based on CD10, CD79A, and PAX5 expression; T-cells based on CD3 expression; hematopoietic stem cells as CD34+; and myeloid cells by the absence of lymphatic markers and the presence of CD14, FCER1G, and CEBPD. Within the B-cell population, three distinct subpopulations corresponding to various B-cell developmental states were identified: pro-B, pre-proB, and pro-B VDJ. The pro-B VDJ state, characterized by heavy chain rearrangement, showed increased CD20 expression. Furthermore, cell cycle analysis using 100 genes associated with the S-phase/G2M-phase revealed a subpopulation of cycling pro-B cells marked by high levels of MKI67 (G2M-phase) and MCM4 (S-phase). Our fusion calling pipeline successfully detected BCR::ABL1, SEMA6A::FEM1C, EBF1::PDGFRB, and P2RY8::CD99 fusion transcripts within the long-read dataset, without any false positive calls at the sample level. These fusion transcripts were found exclusively in B precursor cells when mapped to the cellular landscape. Notably, cells in the pro-B VDJ state did not harbour any fusion transcripts. Interestingly, P2RY8::CD99 was identified only in B-cells, not in T-cells or myeloid progenitors, suggesting its association with pathogenic cells. Similarly, SEMA6A::FEM1C, a GDI fusion transcript, was confined to B precursor cells, indicating an association with the B-ALL clone.
 Conclusions: We have demonstrated the feasibility and utility of integrating long-read isoform sequencing with single-cell library preparation for detecting fusion transcripts of diverse origins. This transcriptome-wide approach enables disease-agnostic detection and characterization of fusion transcripts, not only those arising from chromosomal aberrations (GRD) but also those resulting from aberrant splicing (GRI) at the individual cell level. Moreover, this method facilitates a comprehensive analysis of cell-specific gene expression profiles and fusion transcripts and could be expanded to include the detection of single-nucleotide variants and copy number changes, thereby completing the molecular profile. While currently not designed for integration into routine workflows, this method represents a valuable tool for research projects aimed at elucidating the molecular mechanisms underlying various diseases, with the goal to enhance patient care.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae73b6fed764877551a830ff73b15a040a45186" target='_blank'>
              Long-read single-cell isoform sequencing for cell type-specific detection of genomic rearrangement-dependent and -independent fusion transcripts
              </a>
            </td>
          <td>
            W. Walter, W. Kern, A. Stengel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330bae0129f86adaee3ff8e6b953e0c449188da" target='_blank'>
              Virus-like particles enable targeted gene engineering and pooled CRISPR screening in primary human myeloid cells
              </a>
            </td>
          <td>
            Hyuncheol Jung, Pascal Devant, Carter Ching, Mineto Ota, Jennifer Hamilton, Zachary Steinhart, Wayne Ngo, Luis R. Sandoval, Jae Hyung Jung, Da Xu, Meirui An, Esha Urs, P. Chen, Vincent Allain, Takuya Tada, James K. Nuñez, Nathaniel R Landau, David R. Liu, Justin Eyquem, Jennifer A. Doudna, Alex Marson, J. Carnevale
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Fusion transcription factors (TF) are hallmark oncogenic drivers in pediatric B cell acute lymphoblastic leukemia (B-ALL), contributing to leukemia initiation and transformation. Despite advances in risk-adapted therapies guided by clinical and molecular features, genetic heterogeneity and variable outcomes underscore the need to uncover driver-specific vulnerabilities to expand personalized treatment strategies, particularly in the relapsed/refractory context. Here, we apply a systematic approach to identify rational combination strategies informed by integrating identification of driver-specific downstream targets of the fusion TF TCF3::PBX1 – a common driver in pediatric B-ALL – with an unbiased high-throughput drug combination screen.
 To dissect fusion-specific druggable genes we knocked down (KD) TCF3::PBX1 by short hairpin RNA (shRNA) in the 697 cell line and filtered for transcriptional targets with available approved and clinical-phase candidate drugs. Among top fusion-associated genes we identified ornithine decarboxylase (ODC1), the rate-limiting enzyme of synthesizing polyamines, small polycations with high affinity for nucleic acids controlling key cellular processes like transcription, translation, DNA replication and cell cycle progression. Public datasets confirmed highest mRNA expression of ODC1 in TCF3::PBX1+ patient samples across B-ALL subtypes (St. Jude) and selective dependency on ODC1 in TCF3::PBX1+ cell lines (DepMap) reinforcing its role as a specific fusion-driven vulnerability. This subtype-specific overexpression of ODC1 led us to investigate its regulation, and public ChIP-seq data revealed binding peaks of TCF3::PBX1 at the ODC1 promoter supporting its role as a novel regulator alongside MYC. Genetic interference by shRNA-mediated ODC1 KD and pharmacological inhibition by Difluoromethylornithine (DFMO) reduced intracellular polyamine levels of putrescine, spermidine and spermine and demonstrated cytostatic effects with decreased proliferation with a reduction of cells in S-Phase in TCF3::PBX1+ cell lines. In patient-derived xenograft (PDX) models, ex vivo sensitivity to DFMO was highest in TCF3::PBX1+ PDX compared to other subtypes. Notably, PDXs from TCF3::PBX1+ relapses exhibited sensitivity levels comparable to those observed in TCF3::PBX1+ PDX with favorable outcomes. Transcriptomic and proteomic analysis of DFMO-treated 697 cells uncovered changes in gene and protein expression associated with amino acid and polyamine transport, mitochondrial metabolism, global translation and cell cycle. Strikingly, treatment with DFMO induced regulation of key B cell identity TF and B cell receptor signaling components, alongside an enrichment of transcriptional signatures indicative of more mature B cell states.
 To uncover synthetic lethal vulnerabilities of this induced rewired cell state we performed an unbiased high-throughput combination screen (n = 155 drugs) in 697 cells with or without ODC1 KD. Among known synergistic targets like AMXT-1501, a polyamine transport inhibitor, BCL-XL inhibitors emerged as potent synergistic partners to genetic and pharmacological ODC1 inhibition, further validated in PDX models ex vivo. To evaluate this combination in vivo, we employed a dual-targeting strategy against polyamine metabolism using DFMO to inhibit synthesis and AMXT-1501 to block uptake, as ex vivo data revealed pronounced metabolic plasticity and compensatory upregulation of polyamine transporters following DFMO monotherapy. In vivo combination of DFMO + polyamine-transport inhibitor AMXT-1501 and BCL-XL PROTAC DT2216 in TCF3::PBX1+ PDX models demonstrated promising antileukemic efficacy seen by reduced leukemic burden in spleen and bone marrow at the end of treatment.
 Collectively, our findings identify the fusion TF TCF3::PBX1 as a previously unrecognized positive regulator of polyamine synthesis, acting via ODC1 upregulation, positioning ODC1 as a promising target in this subtype. We further demonstrated that targeting polyamine synthesis induces transcriptional changes in essential B-cell developmental and metabolic programs, uncovering opportunities to target synthetic lethal principles through BCL-XL. The recent approval of DFMO for neuroblastoma maintenance therapy highlights its translational potential not only in TCF3::PBX1+ B-ALL but warrants exploration of this rational combination in other MYC-driven and/or polyamine-addicted B-ALL, especially in the relapsed/refractory context.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ff0281ccf3599cc3a7d33b49b4b28f1ec98b96" target='_blank'>
              Fusion-specific regulation of polyamine synthesis sensitizes to BCL-XL inhibition in TCF3::PBX1+ B-ALL
              </a>
            </td>
          <td>
            James Kim, Sarah Cherkaoui, Simona Ulrich, Bartimée Galvan, Chengkang Li, Caroline Mussak, Samanta Carbajal, Shannon Mezger, B. Bornhauser, Raphael J. Morscher, Jean-Pierre Bourquin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/301f009007e6fbe5ab075165ab5186ae985d6123" target='_blank'>
              Senescent cells exhibit features of developmental signaling centres
              </a>
            </td>
          <td>
            Daniel Sampaio Gonçalves, Ludivine Dulac, Birgit Ritschka, Muriel Rhinn, Annabelle Klein, C. D. Wyatt, Manuel Irimia, W. M. Keyes
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Engineering CRISPR-Cas systems for improved or altered function is central to both research and therapeutic applications. Unfortunately most optimization, especially directed evolution in bacterial hosts, fails to capture the functional requirements of the complex mammalian cellular milieu, where activity is usually required. Robust strategies to enable continuous directed evolution of genome-targeting agents directly in human cells remain lacking. Here, we introduce CRISPR-MACE (Mammalian cell-enabled Adenovirus-assisted Continuous Evolution) as a foundational technology to address this need. CRISPR-MACE integrates virus-based continuous evolution with anti-CRISPR-based tunable selection to generate novel Streptococcus pyogenes Cas9 variants with both increased and decreased DNA binding capacity and nearly 1000-fold–enhanced resistance to AcrIIA4, the strongest known inhibitor of SpCas9. Notably, across independent evolution campaigns the same Cas9 gatekeeper mutation reproducibly emerged first, enabling subsequent adaptive steps along two interdependent axes of Cas9 function. In addition to advancing CRISPR technologies, this work establishes key principles and synthetic circuits for continuously evolving CRISPR-Cas systems directly in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/111c1092bd21d86e47943d56bffc7aeb5164a270" target='_blank'>
              Anti-CRISPR-mediated continuous directed evolution of CRISPR-Cas9 in human cells
              </a>
            </td>
          <td>
            Andrew L. Sabol, Amanuella A. Mengiste, Vedagopuram Sreekanth, Prashant Singh, Samuel J. Hendel, Minh Thuan Nguyen Tran, Anton M. Barybin, Santosh K. Chaudhary, Ra'Mal M Harris, Kristi E. Liivak, Zachary C Severance, Cale M. Locicero, Karishma Kailass, Chaiheon Lee, Lucy Qinghua Xu, V. Butty, Amit Choudhary, M. Shoulders
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Background Acute myeloid leukaemia (AML) is a malignancy of the haematopoietic system with extremely high mortality. Current AML therapies, such as chemotherapy, hematopoietic stem cell transplantation, and targeted therapy, have greatly improved the prognosis of AML patients. However, there are still a number of AML patients who develop drug resistance after pharmacotherapy, leading to poor prognosis and relapse. Currently, the global 5-year survival rate for adults with AML remains below 50%. Therefore, it is urgent to identify the aetiology for AML relapse and drug resistance.
 Methods Bone marrow mononuclear cells were isolated from bone marrow blood samples from 4 patients with AML relapse, 3 patients with AML remission and 5 control donors by density gradient centrifugation using Lymphocyte separation medium. After RNA extraction and sequencing, differentially expressed genes (DEGs) analysis, overlapping analysis, WGCNA and co-expression analysis, alternative splicing analysis and functional enrichment analysis were further performed.
 Results RNA sequencing andDEGs analysis demonstrated that a total of 593 overlapped up-regulated genes and 999 overlapped down-regulated genes were discovered in the reAML group in comparison to the Healthy and AML groups. Of particular, further overlapping analysis revealed that 33 RBP genes were overlapped up-regulated and 30 RBP genes were overlapped down-regulated in the reAML group. Further WGCNA and alternative splicing analyses highlighted that significant changes were found in AS in the reAML group compared with the AML and Healthy groups, and multiple differential regulatory alternative splicing genes (RASGs) and regulatory alternative splicing events (RASEs) were also identified.
 Conclusions The aberrance of regulatory alternative splicing (RAS) and differential expression of RNA-binding protein (RBP) genes are highly associated with AML relapse and drug resistance in AML patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2939a3706c9e8e829fd5d4dfeb6fd0b9e552e7a" target='_blank'>
              Genome-wide investigation of differentially expressed alternative splicing and RNA-binding protein genes association with AML drug resistance
              </a>
            </td>
          <td>
            Jingyi Wang, Yu-Mei Zhang, Xueyan Dong, Wei Zheng, Jie Xu, Zhenzhen Wang, Shumin Ding, Yingying Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell state plasticity drives metastasis and therapy resistance in cancers. In melanoma, these behaviors map onto a melanocytic-to-mesenchymal-like continuum regulated by AP-1 transcription factors. However, how the AP-1 network encodes a limited set of discrete states, why their distributions vary across tumors, and what drives phenotypically consequential AP-1 state transitions remain unclear. We develop a mechanistic ODE model of the AP-1 network capturing their dimerization-controlled, co-regulated, competitive interactions. Calibrated to heterogeneous single-cell data across genetically diverse melanoma populations and combined with statistical learning, the model reveals network features explaining population-specific AP-1 state distributions. These features correlate with MAPK activity across tumor lines and with variability within clones, linking MAPK signaling to AP-1 states. The model predicts and experiments validate adaptive AP-1 reconfiguration under MAPK inhibition, inducing a dedifferentiated, therapy-resistant state that can be blocked by model-guided AP-1 perturbations. These results establish AP-1 as a configurable network and provide a computational framework for predicting and modulating AP-1 driven cell state plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80053bfbfe501e1acc3e16cb2b130e2f5342e9b1" target='_blank'>
              Cell state plasticity emerging from co-regulated, competitive, and configurable interactions within the AP-1 network
              </a>
            </td>
          <td>
            Yonatan N. Degefu, Magda Bujnowska, Douglas G. Baumann, Mohammad Fallahi-Sichani
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Epigenetic mutations (e.g. TETE2, ASXL1, DNMT3A) are a frequently occurring class of mutations in Acute Myeloid Leukemias (AMLs). One of the commonly used therapies in AML is hypomethylating agent (HMA) with venetoclax. Despite complete remissions as high as 60%, it is not a curative regimen with resistance and relapse developing in nearly all patients. HMAs act as epigenetic agents demethylating DNA but how this contributes to synergy with venetoclax is not well understood. We hypothesize HMAs induce an epigenetic-mediated differentiation effect that promotes venetoclax efficacy. Additional epigenetic agents and cell death inducers may be modeled after the HMA/Ven combination and have therapeutic potential in cases of HMA failure. We have previously developed a Tet2 Flt3 – mutant model of leukemia that demonstrates in vivo sensitivity to HMA and utilize it to study HMA resistance and additional epigenetic vulnerabilities.
 Methods To understand HMA's activity, we developed a resistance model of leukemia through repeated treatment cycles. We characterized these cells through RNA-seq and ATAC-seq. We also screened leukemia cell lines and primary mouse cells for potential epigenetic targets using CRISPR. We then tested in vitro anti-leukemic activity of epigenetic agents.
 Results We treated a Tet2 Flt3 leukemia mouse model through several rounds of HMA therapy and identified a resistant clone. In HMA-sensitive models platelets increased two weeks after HMA therapy (mean, 441 k/µl to 781 k/µl) whereas in the resistant clone platelet counts were lower to start and recovered less (mean, 180 k/µl to 269 k/µl). Similarly, more significant anemia and lower recovery (Sens, 46% to 41%; Res, 34% to 17%), but the white blood cell count still responded and declined following HMA (Sens, 18 k/µl to 8.4 k/µl; Res, 43 k/µl to 11.4 k/µl). Resistant cells had altered gene-expression in stem-progenitor cells by RNA-seq. Using GSEA analysis, we identified signatures enriched in hedgehog signaling (NES 2.05, adj p 0.005) and depleted in hematopoietic cell lineage (NES -2.24, adj p 0.001). The altered gene expression was also enriched in signatures with high-CpG density promoters bearing specific histone methylation marks (NES 2.05, adj p 0.002).
 We identified potential pathologic mutations in signaling, epigenetic, and transcription factors including (Rock2, Ptprn, Ube4b, Kdm2, Aebp1, Ints1). ATAC-seq results identified differential chromatin states with binding motifs in the accessible chromatin regions suggesting hematopoietic transcription factors are altered at their binding sites (e.g. PU.1, IRF8). We demonstrate that Kdm2 loss in leukemia cell line K562 cells could blunt the expression of CD41 upon PMA induced megakaryocytic differentiation. This suggests genes regulating differentiation may mediate HMA resistance.
 We screened a CRISPR epigenetic library of target genes that would inhibit growth in our Tet2 Flt3 leukemia model. We used an endothelial co-culture system to select target genes (guides that become depleted over a week in culture). We identified Brd2, Setdb1, Setd4, Kdm4a, Jmjd1c, Atad2b, and Chd6 as potential target genes. In vivo, we show that targeting of Kdm4 in Tet2 Flt3-mutant leukemia cells leads to reduced engraftment of leukemic cells compared to targeting a control locus or Ep300 (a positive selection gene) (mean engraftment, Ctrl:Ep300:Kdm4a 48%:54%:35%, p<.05). We tested pharmacologic inhibition of KDM4A and BRD2. We demonstrate that these agents could inhibit growth of leukemia cells including resistant clones and have combinatorial activity with venetoclax.
 Conclusion Tet2 Flt3 leukemia demonstrates a differentiation response to HMA therapy. Activation of this process may be important in the combination effect with venetoclax. When deriving resistant HMA clones, we identified genes in signaling, epigenetics, and transcriptional regulation that become mutated. These mutations affected the maturation of megakaryocyte-erythroid progenitors. Altered gene regulation at CpG islands may mediate these effects. CRISPR screening identified epigenetic targets (Brd2, Kdm4a)that have therapeutic potential in myeloidleukemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d00e507ec2c9e6d03c66935e5456471e634f6dd1" target='_blank'>
              Identifying epigenetic vulnerabilities for treatment of HMA refractory myeloid leukemia
              </a>
            </td>
          <td>
            Alan H Shih, R. Koche, Michael R Waarts, Brianna Kelly, Claudia Brady
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 Background Drug-tolerant persister (DTP) cells are increasingly recognized as a key contributor to therapy resistance in cancer patients. In mantle cell lymphoma (MCL), the mechanisms by which DTP cells adapt to treatments and develop resistance remain poorly understood. This study investigates the presence and development of pirtobrutinib-tolerant persister cells in MCL and aims to elucidate their underlying mechanisms of resistance. By identifying these mechanisms, we seek to provide insights into novel therapeutic strategies for targeting DTP cells in MCL patients.
 Methods A non-stochastic drug-tolerant persister (DTP) cell line model was developed using mantle cell lymphoma (MCL) cells with acquired resistance to therapies. This model includes Mino cells resistant to venetoclax (Mino-VEN-R) and JeKo-1 cells resistant to ibrutinib (JeKo-1-IBN-R). Both bulk and single-cell RNA sequencing were employed to characterize upregulated and downregulated pathways. Metabolite profiling was conducted using ultra-high-resolution mass spectrometry (HRMS). Oxygen consumption was measured with the Seahorse XF96 analyzer (Agilent Technologies). Anti-CD19 CAR T-cells were generated from human primary pan-T cells isolated from healthy donors, transduced with lentivirus expressing a CD19-directed CAR (FMC63 scFv, 4-1BB costimulatory domain, and CD3ζ stimulatory domain). The presence of pirtobrutinib-tolerant persister cells in MCL was validated using patient-derived organoid and xenograft models. Transcriptome profiles from 62 primary MCL patient samples were analyzed via RNA sequencing.
 Results MCL cells with abnormal morphology have been recognized as an indicator of aggressive diseases with poor response to therapies. However, less was known about their origin and mechanism. Here, we observed that pirtobrutinib treatment induces anabolic growth rather than proliferation in Mino-VEN-R and JeKo-1-IBN-R mantle cell lymphoma (MCL) cells, resulting in enlarged cells with pleomorphism, designated as “Giant cells.” Upon pirtobrutinib withdrawal, these cells resume proliferation, indicating a reversible phenotypic transition characteristic of a drug-tolerant persister (DTP) state, with Giant cells serving as a transitional state for pirtobrutinib-tolerant persister cells. RNA sequencing revealed that Giant cells exhibit a dedifferentiated state under pirtobrutinib treatment, marked by significant loss of B-cell markers, such as CD19, conferring resistance to anti-CD19 CAR T-cell therapy in NSG mice. Mechanistically, an anabolic tricarboxylic acid (TCA) cycle fuels DTP cell development, driven by elevated acetyl-CoA levels mediated by increased ATP-citrate lyase (ACL) activity, which profoundly alters gene expression profiles. GOT2, a key enzyme, supports biosynthesis, including nucleotide production, while ribosome biogenesis is a hallmark of the Giant cell state. Notably, the EMT modulator SNAI1 is a key transcription factor potentially driving dedifferentiation. Upon drug withdrawal, the TCA cycle shifts to a catabolic mode, as validated in patient-derived organoid (PDO) cultures. Additionally, DTP cells in the Giant cell state were identified in patient-derived xenograft (PDX) tumor samples from an MCL patient resistant to pirtobrutinib and CAR T-cell therapy. More evidence from primary MCL patients with resistance to pirtobrutinib and CAR T-cell therapies showed the potential presence of DTP cells, including results from IHC imaging and RNA sequencing. These indicate that DTP cells can be more prevalent in therapy-resistant MCL patients compared to therapy-sensitive patients.
 Conclusions We conclude that non-genetic mechanisms, particularly metabolic reprogramming, drive the fate of drug-tolerant persister (DTP) cells in mantle cell lymphoma (MCL). The tricarboxylic acid (TCA) cycle dynamically switches between anabolic and catabolic modes, enabling tumor cells to adapt and evade therapeutic stress. Acetyl-CoA is a critical metabolite that reshapes the epigenetic landscape of tumor cells. This phenotypic plasticity of DTP cells contributes to tumor heterogeneity following prolonged therapies. DTP cells are prevalent in MCL patients with therapy resistance. Targeting the dedifferentiated state of DTP cells, rather than oncogenic lesions, offers a novel strategy to overcome stable therapy resistance in MCL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89437df3f4f40ad405722c379a0b55564e1288a0" target='_blank'>
              The switch of TCA cycle mode determines the fate of pirtobrutinib-tolerant persister cells
              </a>
            </td>
          <td>
            Michael Wang, Yang Liu, Qingsong Cai, L. Nie, H. Lee, F. Yan, Yue Fei, Yijing Li, Lin Tan, Philip Lorenzi, Chengtai Yu, Wei Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease that arises from dysregulated myeloid proliferation. We performed a high-throughput CRISPR interference (CRISPRi) screen in the THP-1 monocytic cancer cell line to identify long noncoding RNAs (lncRNAs) that play a role in contributing to cell proliferation. Our screen identified INSTAR (Intergenic Nuclear Suppressor lncRNA Targeting Adjacent Regulator SFMBT2) as a top candidate. RNA-seq on INSTAR deficient THP-1 cells revealed transcriptional changes in genes involved in cell proliferation as well as other cellular processes. Loss of INSTAR selectively reduced expression of its neighboring gene, SFMBT2. Functional assays confirmed that both genes suppress cell growth, revealing a cis-regulatory mechanism in which INSTAR regulates SFMBT2 expression to control monocyte proliferation. Here, we leverage high-throughput screening to rapidly pinpoint functional lncRNAs providing novel insights into a key regulatory locus consisting of INSTAR and SFMBT2 which could be critical for better understanding dysregulation contributing to acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR, in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Background: Acute myeloid leukemia (AML) often acquires drug resistance and relapses after standard apoptosis-based therapy. Ferroptosis is an iron-dependent non-apoptotic form of cell death, with lipid peroxidation as a hallmark. Accumulating evidence has been proving the potential of ferroptosis for AML therapy that replenishes current therapeutic strategies and overcomes acquired resistance. Glutathione Peroxidase 4 (GPX4) is an antioxidant protein, reduces lipid peroxidation in cell membranes and functions as the major negative regulator of ferroptosis. Our data has shown that the inhibition of GPX4 with a specific inhibitor, ML210, induces mitochondria-dependent ferroptosis even in venetoclax-resistant AML cells. However, the prolonged survival of AML xenograft mice by GPX4 knockdown in vivo is limited. These findings underscore the need to further optimize ferroptosis-based approaches to improve anti-leukemic efficacy.
 Methods: To identify novel therapeutic targets that may enhance ferroptosis-based AML treatment, we performed a genome-wide CRISPR knockout screen in MOLM13 cells in the context of ferroptosis induction to identify AML-specific ferroptosis regulators. To minimize compound off-target and dosage effects, in this study we induced ferroptosis by genetic GPX4 knockdown with a previously verified doxycycline-inducible short-hairpin RNA (shRNA). We engineered MOLM13 cells with stable Cas9 expression, and the verified cell line was transduced with the lentiviral Brunello library targeting 19,114 genes. After 4 days of puromycin selection, the cells were treated with 1 µg/ml doxycycline for GPX4 knockdown. Cells were passaged every 3 days with the single guide RNA (sgRNA) coverage in each passage at about 500-fold. Ferroptotic cell death was monitored every 3 days during passaging. Genomic DNA was prepared from the cells collected after 9 days of doxycycline treatment and used as the template for amplifying the sgRNA library with a modified two-round of PCR. After high-throughput sequencing, the sgRNA enrichment was analyzed by MAGeCK with comparison to the non-doxycycline control.
 Results: Our results successfully identified several known pro-ferroptosis regulators, such as ACSL4, DHCR7, and LPCAT3 from positive selection. We also observed multiple established ferroptosis suppressors, such as SLC7A11 and GPX4-independent ferroptosis suppressors, such as DHODH, AIFM2, and GCH1 from the negative selection. Interestingly, we identified the ferroptosis suppressors, LRP8 and STARD7, which have been recently investigated in solid tumors, but not in AML. These results indicate the robustness of our screening dataset. Interestingly, OPA1 is identified as a potential ferroptosis suppressor in the present AML study, opposite to the pro-ferroptosis role reported in mouse embryonic fibroblasts and human U2OS osteosarcoma cells, suggesting the specific role of OPA1 in AML. Further enrichment analysis showed that multiple mitochondrion-related pathways were top-ranked, including mitochondrial RNA/protein degradation, mitochondrial gene expression, mitochondrion organization and complex I biogenesis. Notably, we found a novel potential ferroptosis suppressor, MAFG, a basic leucine zipper transcription factor, ranked as a top hit. Based on re-analyses of publicly available databases, the expression of MAFG is positively correlated with GPX4 expression in AML and with the resistance to several ferroptosis inducers, including ML162, ML210, and RSL3. ChIP-seq data of MAFG in K562 cells indicates SLC7A11 is one of its targets. Therefore, we hypothesize that MAFG is the upstream transcriptional regulator of SLC7A11, modulating ferroptosis in a GPX4-dependent manner.
 Conclusion: We performed a genome-wide CRISPR knockout screen using a GPX4 genetic knockdown model and identified multiple known and novel ferroptosis regulators in AML cells. Specifically, our data suggests MAFG as a novel potential ferroptosis suppressor in AML. Further mechanistic studies are ongoing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53c257f712692018b0fd18b8b546fb2bb351fc0" target='_blank'>
              A genome-wide CRISPR screen identifies conventional and novel ferroptosis regulators in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Liang Zhang, Ran Zhao, Kazuharu Kamachi, Saurabh Kumar Gupta, Hiroki Akiyama, M. Andreeff, Jo Ishizawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8def14fbea3eb04059fd733743c146dfc02c3bcc" target='_blank'>
              Mammalian longevity is associated with restricted epithelial plasticity
              </a>
            </td>
          <td>
            Zhizhong Zheng
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 MAPK pathway alterations occur in 10-20% of diffuse midline glioma, H3K27-altered (DMG-H3K27), resulting in a proposed novel subtype of the disease with distinct clinical features. In a previous co-clinical trial platform, we showed an increased sensitivity to MEK inhibitors in DMG-MAPK patient-derived models, though resistance occurred in vitro, in vivo and clinically. To understand the nature of these resistance mechanisms, we used sophisticated coupling of retrievable high-complexity barcoding and single cell RNA sequencing in DMG-MAPK cells. Lineage tracing conducted in several technical replicate experiments showed a subset of clones is consistently enriched across replicates when treated with a GI90 concentration of trametinib for 11 weeks. Interrogation of clonal dynamics using mathematical modelling showed barcodes commonly enriched across replicates exhibit significantly higher clonal fitness compared to replicate-specific enriched barcodes, suggesting a deterministic pattern of resistance. Serial single cell RNA sequencing analysis at weeks 5, 8 and 11 post-treatment initiation showed some resistant H3.3K27M/BRAF-mutant cells adopted astrocyte-like and neural progenitor-like metaprograms in which barcodes were commonly enriched across two cell states, suggesting a subset of clones circumvent trametinib treatment by phenotypic switching; simultaneously, a late-emerging resistant population of cells activate a distinct mesenchymal program. Furthermore, we determine the previously identified MAP2K1 (MEK1) mutations to arise specifically from a small population of radial glia-like stem cells that appear acutely predisposed to the acquisition of these mutations. Together, our data suggest two clonally and mechanistically distinct resistance mechanisms can be active in DMG-MAPK in response to MEK inhibition, involving both transcriptional plasticity and a progenitor-like state primed for the acquisition of genetic determinants which circumvent drug response. With trametinib-resistant cells being highly sensitive to selective SRC inhibition, evolutionary-guided combinatorial treatment schedules may provide for strategies to significantly lengthen time taken for resistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c54926c3fb10216fcf5c70e35738abc3a296f32" target='_blank'>
              DDDR-39. Untangling MEK inhibitor resistance in DMG-MAPK using coupled lineage tracing and single cell RNA sequencing
              </a>
            </td>
          <td>
            Molina Das, Y. Grabovska, Haider Tari, A. Mackay, Rebecca F Rogers, Jared Collins, Jiin Song, Laura Bevington, A. Burford, Drenusha Sejdiu, R. Beroukhim, P. Bandopadhayay, Chris Jones
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ce4a4012ffb45da6b02201b0801273b5519b7e" target='_blank'>
              Integrative single-cell and spatial mapping of oxidative stress response uncovers GCLC+ mesenchymal tumor cell state linked with favorable outcomes in triple negative breast cancer
              </a>
            </td>
          <td>
            Nomeda Girnius, Tuulia Vallius, Wenqing Chen, I.-M. Launonen, S. Palomino-Echeverría, Jia-Ren Lin, Caitlin E. Mills, Silja Kauppila, Pauliina Kronqvist, A. Ellonen, Merja Perala, Eloise Withnell, Yu-An Chen, M. Secrier, S. Santagata, P. Sorger, Anniina Farkkila
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Objectives Immunotherapies, including chimeric antigen receptor T-cell (CAR-T) therapy, represent a pivotal approach in the treatment of multiple myeloma (MM). However, the complex immunosuppressive tumor microenvironment (TME) poses significant challenges to their efficacy. Among the immunosuppressive cells in the MM TME, granulocytic myeloid-derived suppressor cells (G-MDSCs) are predominant, yet their functions remain incompletely understood. This study aimed to comprehensively analyze the role of G-MDSCs in MM patients undergoing CAR-T therapy. Methods Single-cell transcriptomic data from seven MM patients before and after CAR-T therapy were analyzed to characterize G-MDSCs. Functional enrichment and gene set enrichment analysis (GSEA) were performed to identify signaling pathways. A risk prediction model was constructed using RNA-seq and survival data from 713 MM patients, and validated by Kaplan–Meier analysis. In vitro experiments were conducted to assess the immunosuppressive functions of G-MDSCs. Results The findings, though exploratory due to limited sample size and inter-patient heterogeneity, suggested pathological activation and immunosuppressive roles of G-MDSCs potentially linked to patient prognosis. G-MDSCs were identified as key modulators of immune responses within the TME, with GSEA indicating regulation via IFN-α/γ signaling. They may facilitate immune evasion of MM cells by promoting proliferation through the IGF1–IGF1R axis and inhibiting T cells and other immune cells via the SIRPA–CD47 pathway. The risk prediction model based on G-MDSC gene signatures demonstrated high prognostic accuracy (AUC = 0.94). PTGS1 was identified as a key marker associated with high-risk groups, and functional assays confirmed its role in mediating G-MDSC immunosuppressive activity. Conclusions This study provides preliminary insights into the functional role of G-MDSCs in the MM TME and highlights their impact on CAR-T therapy outcomes. The findings suggest potential therapeutic strategies, including targeting PTGS1, to enhance CAR-T efficacy. Larger and more diverse cohorts are required to substantiate these observations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8ff33894cad405d661ae879b8aae448cec342e" target='_blank'>
              Functional role of granulocytic myeloid-derived suppressor cells in CAR-T therapy: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Chaoyu Zhang, Fang An
          </td>
          <td>2025-11-02</td>
          <td>Immunopharmacology and Immunotoxicology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b81384b510909f679a401421e801231c6a36b6" target='_blank'>
              A tailored in vivo CRISPR screen identifies BAP1 as a potent tumor suppressor of soft tissue sarcoma
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Zhaoyu Sun, Melissa J. Kasiewicz, Yaping Wu, Brian D. Piening, John T. Welle, Wesley K. Rosales, V. Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, Rutulkumar Patel, Ziqiang Zhang, B. Bernard, William L Redmond, Walter J. Urba, R. B. Bell, D. Kirsch
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Adoptive cell therapy (ACT) has shown remarkable success in the treatment of some malignancies, particularly leukemia. However, there are multiple factors that limit the durability of ACT in solid tumors, including dose-limiting toxicities, the immunosuppressive tumor microenvironment, and T-cell exhaustion. As the manufacture and preparation of adoptive T-cell therapies allows time and adequate conditions for ex vivo T-cell engineering, forward genetic screens can identify novel genetic targets that could improve their effectiveness. CRISPR is a commonly used functional genomics tool that has been successfully used to both enhance our understanding of mechanisms of resistance and to discover potential genetic edits to improve ACT. A complementary approach, Sleeping Beauty transposon mutagenesis provides additional opportunities to identify novel genetic edits without being constrained by the annotated human genome. Here, we summarize forward genetic screens and their tools to uncover strategies to enhance ACT. Complementary approaches can be combined and improved on to identify translatable genetic editing strategies through studies that accurately recapitulate disease-specific challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba1ded609a6b6846507e3268782cac85e220153" target='_blank'>
              Functional genomics for improving adoptive T-cell transfer therapies
              </a>
            </td>
          <td>
            Joseph G. Skeate, Chang-Jung Lee, C. Stewart, Mathew J Fischbach, Bibek Kar, Alexander K. Tsai, S. Kenderian, Ingunn M. Stromnes, D. A. Largaespada, B. Moriarity, Laura M. Rogers
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Deuterium, a stable isotope of hydrogen present in natural water at ~150 ppm, has been implicated in modulating cellular metabolism and tumor progression. While deuterium-depleted water (DDW) has shown anti-cancer effects in preclinical and clinical studies, the underlying transcriptional mechanisms remain incompletely defined. Here, we profiled gene expression in A549 lung adenocarcinoma cells cultured for 72 h in media containing four graded deuterium concentrations (40, 80, 150, and 300 ppm) using a targeted NanoString panel of 236 cancer-related genes. After stringent quality filtering, 87 genes were retained and classified into nine distinct expression patterns based on fold-change trends relative to the 150 ppm control. High deuterium (300 ppm) induced strong upregulation (up to 2.1-fold) of oncogenic and survival-related genes (e.g., EGFR, CTNNB1, STAT3, CD44), while DDW (40–80 ppm) led to selective downregulation (down to 0.58-fold) of oncogenes (e.g., MYCN, ETS2, IRF1) and drug-resistance genes (e.g., ABCB1). Se-veral genes involved in DNA repair, apoptosis, and extracellular matrix remodeling exhibited dose-dependent responses, suggesting coordinated regulation by deuterium abundance. These findings demonstrate that deuterium concentration functions as a biologically active variable capable of modulating cancer-relevant gene networks. This exploratory dataset refines mechanistic models of DDW action and provides a foundation for future studies incorporating biological replication, functional assays, and in vivo validation. Significance: Deuterium concentration modulation alters oncogenic, apoptotic, and drug-resistance gene networks in lung adenocarcinoma cells, refining prior models of deuterium-depleted water effects. These findings identify deuterium concentration as a biologically active variable warranting further mechanistic and translational investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e32eb13240959435681e9160aca37dba43a34498" target='_blank'>
              Gene Expression Patterns in Lung Adenocarcinoma Cells in Response to Changes in Deuterium Concentration
              </a>
            </td>
          <td>
            Gabor I. Csonka, András Papp, I. Somlyai, G. Somlyai
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Over 75% of PTCL-NOS patients fail to achieve long-term control with current therapies, highlighting the need for novel therapeutic targets. Genetic and non-genetic drivers in cancer cells require external cues from other cancer cells and from the lymphoma microenvironment (LME), which enable cancer cell adaptation to metabolic stress, immune mediated stress and treatment exposure. The resulting signaling crosstalk propels the co-evolution of cancer and LME cells with emergence of diverse cancer-supporting cell states. Identifying these and their maintenance mechanisms may therefore reveal novel therapeutic targets.
 We characterized the LME of PTCL using transcriptional signature deconvolution and multiparametric imaging in 845 patient samples. We identified four biologically and clinically distinct categories named Depleted (DP), B-cell rich (BR), Mesenchymal (MS) and Inflammatory (IN) based on the proportion of different LME cells. RNA-seq analysis showed that SMA+ cancer associated fibroblasts (CAF) are present in DP, MS and IN LMEs each with distinct phenotypes. To functionally analyze the contribution of CAFs to PTCL maintenance, we established multiple patient-derived tumor xenografts (PDXs) that retain the same LME categories of primary tumors. We then tested lymphoma cell survival in co-cultures of CAFs and PTCL cells from matched (e.g., IN-CAF + IN-PTCL) and unmatched PDXs (e.g., IN-CAF + DP-PTCL). Matched co-cultures sustained lymphoma cell proliferation to a higher extent than unmatched, independently of LME subtype. This suggests that CAF are distinctly reprogrammed within each LME.
 We hypothesize that pro-tumoral CAF phenotypes require adaptive gene expression in coordination with lymphoma cells' phenotypes. RNA-Seq of matched IN-CAF co-cultured with IN-PTCL revealed activation of pathways like “Transcription”, “DNA replication” and “Translation” including the upregulation of CDK7. Although CDK7 plays a minor role in cell cycle progression, its phosphorylation of the RNA polymerase II as part of the TFIIH complex is critical in activating and coordinating “cell state”-defining transcription.
 To assess CDK7's role in establishing lymphoma-supportive CAF states, IN-PTCL PDX mice were treated with the covalent selective CDK7 inhibitor YKL-5-124 for 12 h followed by transcriptomic and proteomic profiling of IN-PTCL and IN-CAFs. In IN-CAFs, CDK7 inhibition altered gene and protein expression of several collagens and cytokines, including COL1A1, COL1A2, MMP8, and CXCL12. Ligand-receptor analysis identified CXCL12-CXCR4 as the key CDK7-dependent interaction between IN-CAFs and IN-PTCL cells. IN-PTCL cells showed activation of DNA damage repair pathways, including upregulation of ALYREF and EIF4E, likely as compensatory mechanisms for decreased CDK7-dependent transcription. Since these may represent acquired therapeutic vulnerabilities, we conducted viability screenings in co-cultures of IN-PTCL and IN-CAF isolated from YKL-5-124 (and vehicle) tretated PDX mice. Cells were subsequently exposed for 48 h, individually and in co-cultures, to a drug library containing 40 clinical-phase compounds. This approach led to the identification of 14 compounds with synergistic anti-lymphoma effects. Among these, ten drugs including the XPO1 inhibitor Selinexor, the HSP90 inhibitor PU-H71 and the translation inhibitor Omacetaxine exhibited enhanced anti-lymphoma activity specifically in the co-culture setting, whereas four compounds showed reduced efficacy. This was not seen in mismatched co-cultures and indicates a potential protective role of CAFs and, importantly, that this mechanism is CDK7 dependent.
 To address the limitations of PDX models established in immunodeficient mice with limited ability to capture the immune-mediated effects of drug treatments, we developed an IN-PTCL syngeneic model by intrasplenic implantation of the murine T-cell lymphoma cell line EL4. Treatment with YKL-5-124 significantly reduced the tumor burden compared to vehicles. Cytokine profiling (immunoblotting array) and ELISA of culture medium of YKL-5-124 treated EL4-CAFs showed reduced levels of pro-inflammatory cytokines such as CXCL12, IL6, CCL2, ICAM-1, and CXCL1 indicating that CDK7 is necessary in establishing pro-lymphomagenic IN-CAF phenotypes.
 In conclusion, CDK7 activity is required to sustain specific pro-tumoral crosstalk interactions of matched CAFs and PTCL cells, which can be exploited therapeutically.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386e7a67f6fb5e2e6ca873d1aaf4d401efdcd89c" target='_blank'>
              CDK7 enables adaptive gene transcription required by stromal cells to support distinct T-cell lymphoma phenotypes and represents a novel microenvironment-directed therapeutic target
              </a>
            </td>
          <td>
            K. Kouidri, D. Fiore, M. Revuelta, N. Di Siervi, Federico Ruiz Moreno, Subhashini Joshi, C. Kayembe, Abigail Taylor, G. Inghirami, L. Cerchietti, N. Zamponi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is an aggressive and biologically heterogeneous hematologic malignancy. It is characterized by the clonal expansion of aberrant myeloid progenitor cells and predominantly affects older adults. As many elderly patients are not suitable for intensive chemotherapy or allogeneic hematopoietic stem cell transplantation, the five-year overall survival rate remains below 10%.
 The emergence of venetoclax, a novel, orally bioavailable selective BCL-2 inhibitor, provides a promising therapeutic option for elderly AML patient population. However, resistance to venetoclax develops in approximately 30% of patients, posing a major clinical challenge. Although accumulating evidence implies metabolic reprogramming in venetoclax resistance, effective strategies to reverse this process remain elusive.
 To recapitulate the stepwise acquisition of venetoclax resistance, we first generated MOLM-13 sublines exhibiting intermediate and complete resistance. Whole-exome sequencing of the derived lines identified two BCL-2 mutations. Interestingly, the progressively increasing variant allele frequencies correlate with the resistance severity, implicate these mutations as potential molecular drivers of venetoclax resistance. Remarkably, a relapsed AML patient after allogeneic HSC transplantation, went under venetoclax treatment for the relapse, became venetoclax-resistant while developed the same two BCL-2 mutations in patient serial leukemic cell samples, underscoring the clinical relevance of these findings.
 To elucidate the molecular alterations associated with venetoclax resistance, we firstly performed transcriptomic and proteomic profiling on the resistant MOLM-13 sublines. These analyses revealed a marked upregulation of lipid metabolism pathways in resistant cells compared to wild-type. This finding prompted us to further investigate whether metabolic reprogramming functionally contributes to resistance. To this end, we conducted a metabolism-focused CRISPR-Cas9 screen in wild-type MOLM-13 cells under venetoclax pressure. The screen identified 175 and 191 negatively selected genes at day 8 and 16, respectively, with 30 genes overlapping at both time points. Pathway enrichment analysis highlighted lipid metabolism as a key pathway, with PTEN prominently enriched, suggesting a potential role in mediating venetoclax resistance via metabolic regulation.
 We next sought to validate the functional role of PTEN in modulating venetoclax response. In vitro, genetic ablation of PTEN using CRISPR/Cas9 or shRNA significantly increased venetoclax-induced apoptosis in both parental and resistant AML cell lines. Pharmacological inhibition of PTEN with small-molecule inhibitors (SF1670 or bpV(HOpic)) produced similar sensitizing effects. Importantly, these findings were corroborated in primary AML samples, where PTEN suppression enhanced venetoclax responsiveness. In vivo, PTEN inhibition in xenograft models led to increased apoptotic response upon venetoclax treatment, further confirming its critical regulatory role in venetoclax response.
 Mechanistically, PTEN loss led to pronounced lipid droplet accumulation, as confirmed by MS/MS-based lipidomic profiling showing elevated fatty acid levels. 13C-labeled palmitate tracing showed reduced incorporation of fatty acids into the TCA cycle, indicating impaired β-oxidation. Concomitantly, upregulation of FASN and SREBP suggested increased lipid synthesis. This dual effect of reduced lipid degradation and increased synthesis resulted in intracellular lipid accumulation, which caused mitochondrial damage, as evidenced by elevated ROS levels. This mitochondrial stress may contribute to reduced resistance to venetoclax by promoting apoptosis, providing a mechanistic link between PTEN loss, lipid metabolism reprogramming, and enhanced drug sensitivity.
 In conclusion, we identify PTEN as a key lipid metabolic regulator of venetoclax sensitivity in AML. By linking lipid metabolic reprogramming to apoptotic vulnerability, we uncover a mechanistic basis for venetoclax sensitization. Our findings highlight the potential of targeting lipid metabolic pathways as a strategy to overcome venetoclax resistance in AML, particularly in the context of BCL-2 mutations.
 MT.C, LZ.C and YJ.J contributed equally to this work.
 Corresponding authors: WP.Y, EL.J and YJ.C.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622b6cf0575b845ded4dfa8ce291137a9afa72e" target='_blank'>
              PTEN loss reprograms lipid metabolism to modulate venetoclax resistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            Mutian Cao, Lizhen Cong, Y. Jia, Yifei He, Mingyue Hao, Yuanyi Ling, Wanting Li, Yajing Chu, E. Jiang, Weiping Yuan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, Jangkeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cells rapidly induce PD-L1 expression in response to inflammatory cytokines such as IFNγ from cytotoxic T cells. Increased surface PD-L1 is a primary mechanism of cancer cells evading cytotoxic T-cell-mediated immune clearance. Identifying how cancer cells increase PD-L1 expression may yield clinically relevant immune checkpoint regulators. However, the key regulators and molecular mechanisms mediating rapid PD-L1 induction are yet to be understood entirely. To identify targetable mechanisms controlling cytokine-induced PD-L1 expression, we performed functional CRISPR gene KO screening with a custom-designed sgRNA library that targets “druggable” genes. We performed the screening in 6 different cancer lines: 3 ovarian (OVCAR4, CaOV3, and SKOV3) and three pancreatic cancer (MiaPaca2, ASPC1 and KP4) cell lines. The screening recovered the known regulators of PD-L1 expression and uncovered several novel regulators of PD-L1 that control its expression in all cell lines or in a cancer-type-specific fashion. For example, while genetic or pharmacological depletion of CSNK1A1 results in reduced PD-L1 expression in ovarian cancer cells, CDK1 depletion modulates PD-L1 in pancreatic cancer cell lines. Significantly, we discovered that KEAP1 depletion or pharmacological inhibition diminishes PD-L1 in all cell lines tested (n = 6). Mechanistically, KEAP1 depletion-mediated reduced PD-L1 is due to transcriptional repression of the PD-L1 gene by NRF2 activation. As such, depletion of NRF2 restores PD-L1 expression, while its overexpression leads to diminished PD-L1 expression. Supporting this, pharmacological NRF2 activation resulted in significant antitumor immunity with increased cytotoxic effector T cell infiltration and reduced exhausted T cells, resulting in smaller xenografted tumors. These findings establish the KEAP1/NRF2 axis as a novel and potentially druggable mechanism of IFNγ-meditated PD-L1 expression in cancer cells. CRISPR screening of druggable genes identified KEAP1/NRF2 axis as a novel regulator of PD-L1 expression. KEAP1 depletion or pharmacological inhibition reduces PD-L1 transcription via NRF2 activation, enhancing antitumor immunity in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d42b79a60457a15da5b80564f464751ddc6948" target='_blank'>
              Druggable genome CRISPR screening identifies the KEAP1/NRF2 axis as a mediator of PD-L1 expression
              </a>
            </td>
          <td>
            Fidan Seker-Polat, Magdalena Rogozinska, Y. Ban, Fatih Abdula, Kadir Buyukcelebi, Ping Xie, Jie Fan, Neda Abbaszadeh, Yasemin Kingham, Seyedehzahra Paylakhi, Bin Zhang, M. Adli
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e643a6feb31e1193c4fe51251e45e7c7b63296ed" target='_blank'>
              Cross-Species Transcriptomic Integration Reveals a Conserved, MIRO1-Mediated Macrophage-to-T Cell Signaling Axis Driving Immunosuppression in Glioma
              </a>
            </td>
          <td>
            Zehui Du, Menghan Li, Brandon Bergsneider, Andy P. Tsai, Kwang Bog Cho, Lily H Kim, John Choi, Gordon Li, T. Wyss-Coray, Michael Lim, Xinnan Wang
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary The von Hippel–Lindau Syndrome, an inherited condition that predisposes to clear cell renal cell carcinoma (ccRCC), affects developmental pathways of the kidney. Studying these early stages could help develop customized, targeted treatments. One approach is to reverse-engineer cancer cells into a stem cell-like state and then grow them into organ models. We attempted this with clear cell renal cell carcinoma (ccRCC) cells and adjacent normal (AN) healthy kidney cells using cell reprogramming. While the adjacent normal cells successfully reprogrammed to induce pluripotent stem cells (iPSCs), the cancer cells did not. We then restored the VHL gene—often missing in ccRCC. This reintroduced normal cell characteristics, but the cells upon reprogramming could not maintain a stable stem cell state. Our findings suggest that while restoring VHL is necessary for reprogramming, other genetic issues block ccRCC cells from fully returning to a healthy, stem-like state, indicating the importance of other, yet to be determined factors. Similarities between cancer cells and stem cells have been recognized, as common genes can influence both outcomes. In this study, we demonstrate that the VHL gene can be added to this group, as it is involved in both cancer development and stem cell regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e849d414c2da1724a3bcbca925e3a62cc9f656a" target='_blank'>
              VHL Gene Restoration Supports RCC Reprogramming to iPSCs but Does Not Ensure Line Stability
              </a>
            </td>
          <td>
            Zsuzsanna Lichner, Y. Shamshirgaran, Katarzyna Pieczonka, A. Jonebring, M. Kibschull, O. Shynlova, Jalna Meens, R. Kim, Laurie Ailles, B. Bilican, Ryan Hicks, Ian M. Rogers
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC) is the second most common form of non-melanoma skin cancer (NMSC), with a steadily increasing global incidence, especially in populations with prolonged ultraviolet (UV) exposure. Advanced or metastatic cSCC carries a poor prognosis, while the application of targeted therapies and immunotherapies remains in the exploratory stage, due to limited understanding of the underlying mechanisms of cSCC occurrence and progression. Epigenetic dysregulation plays important roles in the progression of cSCC. However, how multi-dimensional regulatory networks reshape the epigenetic landscape and contribute to dysregulated gene expression in cSCC remains unclear. In this study, we performed parallel RNA m6A sequencing, 850K DNA methylation arrays, whole transcriptome sequencing, and ATAC-seq chromatin accessibility profiling on samples from normal skin, actinic keratosis (AK), and cSCC. We analyzed the regulatory networks and pathways of epigenetic modifications on gene expression. We further explored the crosstalk of epigenetic regulatory networks by correlation analysis. By integrating single-cell RNA-seq data, we identified epigenetically upregulated candidate genes and confirmed their expression and functions with experimental methods. Our integrated multi-omics analysis provides a comprehensive and dynamic epigenetic map of cSCC progression. Further analysis revealed that DNA methylation and m6A modification jointly regulate gene expression through independent and synergistic ways. The identified epigenetically upregulated candidate genes IDO1, IFI6, and OAS2 were validated to be overexpressed in cSCC tissues and cell lines, and functional assays confirmed their potential key roles in regulating the processes of cell proliferation, migration and invasion in cSCC. By integrating multi-omics data, this study systematically highlights the multi-layered epigenetic alterations and regulatory mechanisms involved in cSCC development. This multi-stage, multi-omics, and multi-resolution integrated analysis provides a theoretical basis and new insights for future personalized treatment strategies for cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdeb7e8c78575b1d7e37855326ebd04eb68da2c1" target='_blank'>
              Multi-omics analysis reveals the crosstalk of epigenetic regulatory networks in cutaneous squamous cell carcinoma progression
              </a>
            </td>
          <td>
            Ya-Zhou Sun, Dan-Dan Zou, Xin-Jie Li, Xiao-Li Wang, Jia-Hao Feng, Jia-Ying Liu, Ke He, Yu-Tong He, Xian-Xin Lai, Xin Li, Xuan Jiang, Tian-Shun Gao, Li He
          </td>
          <td>2025-11-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan C Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Introduction: Acute Myeloid Leukemia (AML) is an aggressive hematologic malignancy characterized by the clonal expansion of undifferentiated myeloid precursors in the bone marrow and peripheral blood. This uncontrolled proliferation is primarily driven by a range of genetic mutations that disrupt normal hematopoiesis and impair the function of healthy leukocytes, erythrocytes, and platelets. Among emerging regulatory mechanisms, N6-methyladenosine (m6A) RNA modification plays a critical role in post-transcriptional gene regulation and cancer progression. The m6A methyltransferase complex (MAC), which catalyzes the methylation of adenosine residues in mRNA, is primarily composed of METTL3 and METTL14. METTL3 serves as the catalytic subunit responsible for methyltransferase activity, while METTL14 provides structural support and enhances substrate recognition and binding. Together, this heterodimeric complex efficiently installs m6A marks on mRNA, thereby influencing RNA stability, translation, and splicing. Dysregulation of m6A modification has been implicated in the pathogenesis of multiple cancers, including AML. This study investigated the regulatory role of METTL3 in AML by reanalyzing publicly available RNA-seq datasets from AML MOLM13 cells treated with ZW27941, a potent small-molecule degrader of METTL3.
 Methods: Human AML MOLM13 cells were treated with either DMSO (control) or ZW27941 for 24 hours. Following treatment, total RNA was extracted, and RNA-seq was performed. Transcriptome data (GSE273361) were reanalyzed to identify differentially expressed genes (DEGs) using R packages (false discovery rate < 0.05 and |log₂ fold-change| > 0.2). Gene ontology (GO) enrichment analysis was conducted to identify biological processes significantly altered upon METTL3 degradation.
 Results: ZW27941 treatment induced substantial transcriptomic changes, with 534 genes upregulated and 641 genes downregulated. The suppressed genes included key drivers of AML progression (e.g., TREM1, S100A8/A9, TLR4, CD84, CXCR3, ITPK1, PECAM1, and ADGRE1), indicating pathways related to innate immune signaling, cytokine-mediated activation, and inflammatory cell adhesion. Downregulation of genes involved in lipid metabolism (e.g., PPARGC1A, ELOVL6, CYP27A1/B1/W1, ABCA1/2/9, and SLC-family transporters) suggests a disruption in fatty acid synthesis and cholesterol homeostasis, pathways known to support leukemic cell survival and drug resistance. Additionally, the downregulation of transcription factors and developmental regulators (e.g., RUNX1, TNR, SOX18, NR2F2, and FOXM1) indicates interference with lineage identity, cell proliferation, and AML subtype specification. Conversely, genes and pathways involved in apoptosis, DNA damage repair, cell cycle arrest, and immune activation (e.g., IL-2, IL-6, TNFα, NF-κB, and JAK-STAT3 signaling), were significantly upregulated in response to METTL3 degradation.
 Conclusion: Our study demonstrates that METTL3 plays a pivotal role in sustaining AML cell survival by regulating genes associated with inflammation, lipid metabolism, proliferation, and immune evasion. Pharmacological degradation of METTL3 by ZW27941 not only suppresses oncogenic transcriptional programs but also activates apoptotic and immune responses. These findings demonstrate METTL3 as a promising therapeutic target in AML and support further exploration of m6A modulators in anti-leukemic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ab99e6d63a6b8558aeee4c091c181cedae6e6c" target='_blank'>
              Targeting METTL3 disrupts oncogenic transcriptional programs and activates immune and apoptotic pathways in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Harshita Ravi, Qian Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Introduction: Philadelphia-negative myeloproliferative neoplasms (MPNs) are a group of diseases, including primary myelofibrosis (PMF), essential thrombocythemia (ET), and polycythemia vera (PV). Ruxolitinib, as a JAK1/2 inhibitor, can reduce splenomegaly and overproduction of blood cells. However, prolonged ruxolitinib treatment often leads to loss of therapeutic efficacy or intolerable side effects, ultimately resulting in treatment discontinuation in many patients. Genome-scale CRISPR-Cas9 screen has emerged as a robust, unbiased tool for discovering novel cancer vulnerabilities and therapeutic targets. In this study, we performed CRISPR-Cas9 knockout screening in vitro to identify genes whose loss was involved in resistance to ruxolitinib.
 Methods: We employed the genome-scale CRISPR-Cas9 knockout (GeCKO) library and utilized the SET-2 cell line as a myeloproliferative neoplasm (MPN) model. The CRISPR/Cas9 library screening process involved several key steps. The first was pre-experiments, including GFP control virus infection and drug concentration tests. Next, stable Cas9-expressing SET-2 cells were generated using lentiviral vectors carrying Cas9 and selected with Blasticidin. For the main screening, the gRNA library was introduced into stable Cas9-expressing SET-2 cells, followed by puromycin selection to enrich successfully infected cells, named as Human GeCKO SET-2. Based on the results of the pre-experiment, Human GeCKO SET-2 cells were treated with 150 nM ruxolitinib for twenty-one days. Finally, genomic DNA was extracted from surviving cells, and the gRNA sequences were amplified and analyzed via next-generation sequencing (NGS). Bioinformatic tools were used to identify enriched or depleted gRNAs, linking gene knockouts to phenotypic changes.
 Results: Raw sequencing data were subjected to stringent quality checks, including assessment of base quality (Q30 > 80%), error rates (<0.5%), and GC content uniformity. AGeCK software quantified gRNA and gene-level read counts, normalized for comparative analysis. Essential genes (e.g., KAT7, MRPL34) were identified via negative selection (depleted gRNAs), while resistance-associated genes (e.g., FAM71C, GPRIN2) emerged from positive selection (enriched gRNAs). We obtained 3069 negatively selected genes (FDR < 0.25) and 1913 positively selected candidates through Robust Rank Aggregation (RRA) analysis. Top hits included KAT7 (neg-score = 7.32e-06) and FAM71C (pos-score = 2.75e-06). Volcano plots highlighted genes with |log2FC| > 2 and FDR < 0.05, such as DHODH (negative) and CDC34 (positive). Gene Ontology (GO) and KEGG pathway analyses were conducted to elucidate the biological roles of the identified resistance-associated genes.
 The top three enriched GO pathway were Cytoplasmic translation, Mitochondrial translation and Exonucleolytic nuclear-transcribed mrna catabolic process. The top three enriched KEGG pathway were Aminoacyl-trna biosynthesis, Proteasome and Rna polymerase. The top ten resistance-associated genes that we were interested in, were linked to cell cycle regulation and proliferation (2 genes), metabolic detoxification mechanisms (2 genes), transcriptional and translational control (2 genes), oxidative stress response (1 gene), epigenetic modulation and signal integration (1 gene), and unexpected involvement of neural-related functions (2 genes). They synergistically promote tumor cell survival under chemotherapeutic pressure by sustaining cell cycle progression, enhancing drug metabolism and clearance, stabilizing pro-survival gene expression, resisting oxidative damage, remodeling epigenetic states, and potentially co-opting neurodevelopmental mechanisms to reinforce adaptive resistance. This multifunctional interplay suggests that overcoming drug resistance requires a combinatorial therapeutic approach targeting metabolic, epigenetic, and stress-defense pathways. Next, we will conduct experimental studies to further validate these genes.
 Conclusion: We performed the genome-scale CRISPR-Cas9 knockout screening to identify genetic drivers of ruxolitinib resistance, providing actionable targets for further validation.
 Acknowledgement: This research was funded by Zhejiang Provincial Health High-level Innovative Talent Project (2022-2026).
 *Correspondence to: Jian Huang, M.D., Ph.D., Department of Hematology, The First Affiliated Hospital of Zhejiang University School of Medicine, Hangzhou, 310003, Zhejiang, China. E-mail:househuang@zju.edu.cn.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a3d397ca550b7b1037bff3dac7e18a28aa1fbd" target='_blank'>
              Genome-wide CRISPR-Cas9 screen reveals novel ruxolitinib-resistance targets in myeloproliferative neoplasms
              </a>
            </td>
          <td>
            Qingmei Han, Rumeng Li, Bingbing Wen, Tian Zeng, Zhikang Zheng, Xiaojing Bao, Xiudi Yang, Jian Huang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Importance of Study Protein phosphatase magnesium-dependent 1D (PPM1D) is frequently mutated in diffuse midline gliomas (DMGs). DMGs are rare pediatric brain tumors with limited treatment options. Due to the cancer’s rapid progression, patients usually survive 12-24 months after diagnosis. This underscores the critical need to better understand the molecular mechanisms driving DMGs. This study describes a novel mouse model that provides a powerful platform to investigate PPM1D-driven tumor biology and offers mechanistic insights into disease development and progression. Furthermore, it serves as a valuable preclinical system for evaluating therapeutic strategies and identifying translational opportunities to target Ppm1d-mutant tumors. Background Diffuse midline gliomas (DMGs) are incurable brain tumors with limited treatment options. Approximately 20% of DMGs harbor truncating mutations in exon 6 of phosphatase PPM1D, which stabilize the protein and deregulate p53 signaling. However, the consequences of these mutations for tumor initiation, progression, and therapy remain unclear. Methods We developed a conditional Ppm1d-loxP-exon6-loxP-exon6-E518X-tag mouse allele (Ppm1d-flex-6) that enables lineage-, spatial-, and temporal-specific expression of a DMG-derived truncated Ppm1d protein from its endogenous locus in the presence of Cre-recombinase. Ubiquitous activation of mutant Ppm1d was modeled using the Meox2-Cre driver, and primary gliomas were modeled using the RCAS/tv-a system to introduce Cre and PDGFB co-drivers into Nestin-positive neural stem cells. Complementary studies were performed in mouse embryonic fibroblasts (MEFs) expressing truncated Ppm1d following Cre recombination. Results While Meox2-Cre-driven ubiquitous recombination of Ppm1d-flex-6 produced muted phenotypes, Ppm1d-flex-6 recombination in Nestin+ neural stem cells accelerated gliomagenesis. Its oncogenic effect was weaker than complete p53 loss, and it did not accelerate tumorigenesis further in p53-null tumors. Single-cell RNA-sequencing revealed that Ppm1d-flex-6 gliomas adopt more progenitor-like transcriptional states and upregulate p53- and cell cycle associated pathways. In MEFs, Ppm1d-flex-6 enhanced proliferation and shifted transcriptomic programs toward MAPK and PI3K-Akt signaling, while impairing DNA damage responses, including reduced γ-H2AX induction after irradiation. These defects sensitized cells to radiation and decreased clonogenic survival after ionizing radiation and PARP inhibition. Conclusions Ppm1d mutations confer intermediate suppression of the p53 pathway, consistent with the clinical features of PPM1D-mutant DMGs and are associated with radiosensitivity and PARP inhibitor vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b526ced19406db51fd6c39018ffced7037a7861b" target='_blank'>
              Oncogenic Ppm1d mutations deregulate the p53 pathway in primary mouse gliomas
              </a>
            </td>
          <td>
            Vennesa Valentine, Abigail J. Groth, Joshua Tolliver, Avneesh Saravanapavan, Spencer M. Maingi, Loren B. Weidenhammer, Nerissa T. Williams, Lixia Luo, Matthew S. Waitkus, Sheeba Jacob, Zachary J. Reitman
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Background: Chronic myelomonocytic leukemia (CMML) is a clonal hematopoietic stem cell disorder that combines clinical and biological characteristics of myelodysplastic neoplasms and myeloproliferative neoplasms. Current treatment choices are limited by incomplete understanding of its transcriptional programs and clonal evolution. Clinical heterogeneity necessitates single-cell resolution to resolve subclonal diversity and identify actionable nodes. We applied single-cell transcriptome sequencing to delineate cellular and molecular heterogeneity and to uncover clonal evolution features that may inform actionable targets.
 Methods: Bone marrow aspirates from 5 newly diagnosed CMML patients and 4 healthy donors underwent single-cell transcriptome sequencing. Comprehensive analysis included dimensionality reduction and clustering, gene ontology (GO) and KEGG pathway enrichment of significantly different genes (p<0.05 and fold changes > 1.5), intercellular communication inference (CellChat), regulatory network reconstruction (SCENIC), gene set variation analysis (GSVA), consensus non-negative matrix factorization, and construction of explainable gene ontology fingerprints.
 Results: 1. We identified 13 distinct cellular clusters with significant compositional shifts in CMML versus healthy controls; clusters 2, 5, 7, 9, and 10–13 showed the most pronounced differences. We also identified marker genes for each of the 13 cell clusters, and the top 10 marker genes in each cluster were identified based on log fold change and adjusted P-value, including ALPL, MAL, KLRF1, SASH1, SLC2A5, CRISP3, XCL2, E2F7, PAX5, ITGB3, and P2RY6. 2. A total of 1,866 genes were differentially expressed(1,584 upregulated, 282 downregulated) in CMML patients compared to healthy volunteers. 3. GO enrichment analysis was performed separately for biological process (BP), molecular function (MF), and cellular component (CC). In BP, upregulated genes were significantly enriched for neutrophil degranulation, whereas downregulated genes were overrepresented in SRP-dependent cotranslational protein targeting to the membrane. In CC, upregulated genes localized predominantly to the cytosol, while downregulated genes were concentrated in ribosomal structures. In MF, upregulated genes were chiefly associated with protein binding, whereas downregulated genes were enriched for functions related to the structural constituent of the ribosome. 4. KEGG enrichment analysis revealed significant enrichment in pathways such as B cell receptor signaling, microRNAs in cancer, ribosomes, and antigen processing and presentation. 5. Protein interaction analysis using the STRING database generated a PPI network of the top 25 differentially expressed genes, highlighting a central hub that includes NAMPT, SOD2, CXCL8, LYN, LTF, VMP1, PDE4D, FNDC3B, DEFA3, and ACSL1.
 Conclusion: This single-cell transcriptome sequencing study delineated disease-specific transcriptional and clonal evolution features in CMML versus healthy controls, providing mechanistic insight into CMML pathogenesis and identifying candidate molecular targets for therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d3add7a95a8334b2ea20e007ff803fc2eda72c" target='_blank'>
              Exploring molecular heterogeneity and clonal evolution in chronic myelomonocytic leukemia via single-cell transcriptome sequencing
              </a>
            </td>
          <td>
            Jie Zhou, Yuyuan Huang, Hao Wu, Bing Li, Shaoguang Li, Aibin Liang, Jianfei Fu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer, ranking among the most prevalent malignancies in males worldwide, is undergoing a significant evolution in therapeutic paradigms from conventional approaches to precision medicine, with recent advances in targeted therapies offering novel strategic insights. This review delineates the molecular foundations of prostate carcinogenesis, elucidating pivotal domains including genetic mutations, hormonal regulation, tumor microenvironment dynamics, cell cycle dysregulation, epigenetic modifications, and tumor heterogeneity. Furthermore, we evaluate the clinical translation of targeted strategies such as AR signaling axis inhibition, PI3K/AKT/mTOR pathway modulation, DNA damage repair machinery exploitation, prostate-specific membrane antigen -directed interventions, and combinatorial immunotherapy. Concurrent challenges—AR-driven heterogeneity, adaptive drug resistance mechanisms, spliceosomal vulnerabilities, and scarcity of selective molecular targets—are critically analyzed. Notwithstanding these obstacles, targeted therapies exhibit considerable potential to enhance therapeutic efficacy while mitigating systemic toxicities, paving the way for more personalized and precision-oriented oncologic care. By underscoring the imperative to decode prostate cancer’s molecular architecture, this work outlines future research priorities and advances a robust scientific framework for innovation in therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6680208f5ea18e2a4fa84ced337f25d04b9c91c6" target='_blank'>
              Unravelling targeted therapy in prostate cancer: from molecular mechanisms to translational opportunities
              </a>
            </td>
          <td>
            Litong Wu, Junfeng Qiu, Zhi-ming Hong, Quan Wang, Qixin Li, Wenbin Zhou
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Background: Acute myeloid leukemia (AML) is a complex disease marked by a multitude of genetic mutations and dysregulated gene expression profiles stemming from genetic and epigenetic alterations. The disease is marked by significant biological heterogeneity, which complicates effective treatment and contributes to the emergence of drug resistance. The molecular profiles of gene mutations, chromosomal anomalies within AML considerably impacts disease progression, therapeutic response, and survival prognosis. Several predictive and prognostic models are used but present inaccuracy at individual level. Integrating multiple layers of biological data, including transcriptomic analyses (gene expression, mutational profiling, gene fusion identification) and population-level characterization through deconvolution and multiparametric flow cytometry, offers a powerful approach to better understand the complexity of AML and improve prognostic prediction and guide treatment strategies.
 Methods: We used a multi-omics data integration approach, including bulk RNAseq-based gene expression (n=45), single nucleotide variant and gene fusion data from bulk RNA sequencing (n=45), deconvolution of tumor microenvironment immune subtypes (n=45), immune subtype identification and blast characterization using multiparametric flow cytometry (MFC) (n=36), in vitro drug response (Venetoclax, n=23; 5-Azacytidine, n=23; Midostaurine, n=21, Gilteritinib, n=22), and relevant clinical metadata. This allowed us to thoroughly characterize tumor and its tumor microenvironment.
 Results: Applying Multiomics Factor Analysis (MOFA) identified 9 principal factors capturing the diversity of the disease. Factor 1 was significantly associated with Venetoclax response (p<0.05). Factors 2 and 4 showed significant associations with NPM1 mutations and DNMT3A mutations, while Factors 6 and 9 were significantly associated with 5-Azacytidine response (p<0.05). Factor 9 was also significantly associated with KIT mutations (p<0.05). The correlation analysis revealed strong associations between factors and key clinical covariates including timepoint and drug response groups.
 Next, we performed unsupervised clustering using the latent MOFA factors and identified four distinct clusters. Cluster 1 (C1, n=14) predominantly consisted of diagnostic samples (79%) with balanced immune infiltration. Cluster 2 (C2, n=13) showed moderate monocyte enrichment. Cluster 3 (C3, n=10) was characterized by significantly elevated regulatory T cell infiltration (p=0.017 vs C4, p=0.023 vs C1) and higher myeloid dendritic cell populations (p = 0.023 vs C2, p = 0.0051 vs C4), confirmed by both deconvolution (p=0.023 vs C2, p=0.0051 vs C4) and multiparametric flow cytometry analysis, consistent with an immunosuppressive microenvironment in this relapse-enriched cluster (60% relapsed samples). Cluster 4 was significantly enriched in monocytes (p < 0.001 vs C1 and C3).
 Finally, therapeutic response profiling across molecular clusters revealed distinct patterns of drug sensitivity. C1 demonstrated favorable responses to Venetoclax, Midostaurine, and Gilteritinib. C2 exhibited poor responses to all four tested drugs, indicating a globally resistant phenotype. C3 showed resistance specifically to Midostaurine and Gilteritinib, while remaining potentially responsive to other agents. Interestingly, C4 was characterized by poor sensitivity to Venetoclax and 5-Azacytidine but responded well to Midostaurine and Gilteritinib, highlighting subtype-specific vulnerabilities that could inform personalized therapeutic strategies. Interestingly, Gilteritinib demonstrated significant activity in specific molecular subgroups (C1 and C4) independently of FLT3-ITD status, suggesting broader therapeutic potential beyond canonical FLT3-driven AML. This unexpected sensitivity, possibly linked to off-target effects such as AXL inhibition, highlights novel vulnerabilities revealed by our integrative multi-omics framework.
 Conclusion: Our comprehensive multi-omics MOFA framework identified four clinically distinct AML subtypes with distinct therapeutic vulnerabilities, immune microenvironment characteristics, mutational profiles, and blast immunophenotypes. Our systematic approach demonstrates the potential of multi-omics integration for advancing personalized medicine in AML and provides a framework that will benefit from validation with larger cohorts to improve risk stratification personalized treatment approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9ac3a924304ce3dd2cf299a9f56500c5d7d76a" target='_blank'>
              Multi-omics integration reveals therapeutic vulnerabilities and immune microenvironment subtypes in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            E. Alaterre, H. de Boussac, A. Kassambara, A. Steer, Amélie Machura, Julien Aityaya, Ouissem Karmous Gadacha, M. Abouladzé, C. Bret, Gaspar Aspas Requena, Ludovic Gabellier, C. Herbaux, A. Bruyer, Jérôme Moreaux
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Metastatic melanoma (MM) displays remarkable phenotypic plasticity, allowing tumor cells to transition reversibly between proliferative and mesenchymal (MES)-like states. This dynamic switching is strongly associated with therapeutic resistance and poor prognosis. Although transcriptional and epigenetic mechanisms driving these transitions have been extensively studied, the role of post-translational regulation, particularly the ubiquitin–proteasome system, remains poorly understood. Here, we identify the ubiquitin E3 ligase RNF 123 (KPC1) as a key post-translational suppressor of MES reprogramming in MM. Integrative analyses of bulk and single-cell transcriptomic datasets revealed that KPC1 expression is inversely correlated with the expression of core mesenchymal markers such as ZEB1, CDH2, and AXL, and positively associated with epithelial and melanocytic lineage genes, including CDH1 and MITF. Deconvolution of TCGA-SKCM RNA-seq data confirmed that this inverse correlation is specific to malignant melanoma cells and strongest in tumors enriched for mesenchymal gene signatures. Single-cell trajectory and enrichment analyses further demonstrated that decreasing KPC1 expression accompanies MES-like switch. Mechanistically, KPC1 binds and promotes the ubiquitination and proteasomal-mediated degradation of ZEB1, thereby suppressing cadherin switching and cell motility. Loss of KPC1 in melanoma cells prevented ZEB1 proteasomal-mediated degradation, increased expression of mesenchymal markers, and enhanced MM cells migration. Clinically, low KPC1 protein levels were associated with increased expression of ZEB1 and CDH2 and poorer overall survival. Furthermore, combined assessment of KPC1, ZEB1, and CDH2 expression improved patient stratification, suggesting the potential utility of multi-marker signatures for prognostic modeling. These findings establish KPC1 as a central post-translational regulator of melanoma cell state plasticity through targeted degradation of ZEB1. This study highlights a novel mechanism regulating MES-like transition and highlights KPC1 as a potential theragnostic target in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e81a070c1d8deb27cd4afec735c4116591dd61f" target='_blank'>
              Ubiquitin E3 ligase KPC1 governs mesenchymal metastatic melanoma reprogramming via proteasomal degradation of ZEB1
              </a>
            </td>
          <td>
            Yusuke Nakano, Matias A. Bustos, Kelly K. Chong, Yoshinori Hayashi, Aaron Ciechanover, Dave S.B. Hoon
          </td>
          <td>2025-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Therapy resistance in acute myeloid leukemia (AML) remains a major clinical obstacle, particularly due to the persistence of leukemia stem cells (LSCs) capable of metabolic adaptation. While venetoclax (Ven) inhibits oxidative phosphorylation (OXPHOS), we found that Ven-resistant LSCs undergo glycolytic reprogramming to bypass OXPHOS inhibition. This metabolic shift is supported by enhanced ribosome biogenesis, sustained by upregulated de novo guanine nucleotide biosynthesis. Abundant guanine nucleotides suppress the impaired ribosome biogenesis checkpoint (IRBC), leading to TP53 destabilization and persistent MYC expression. Inhibition of inosine monophosphate dehydrogenases (IMPDH1/2) depletes guanine nucleotides, activates IRBC, stabilizes TP53, represses MYC, and impairs the metabolic shift to glycolysis. This metabolic rewiring disrupts LSC stemness and suppresses the reconstitution of human AML cells in xenotransplantation experiments. Notably, the suppression of LSC stemness was observed regardless of Ven resistance or the TP53 mutational status of AML cells. These findings reveal that mutation-independent TP53 inactivation is involved in resistant AML and suggest that targeting guanine nucleotide biosynthesis may offer a clinically actionable strategy to eradicate therapy-resistant LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6613ff1ea81832476d4f80a2812043adb8207057" target='_blank'>
              Guanine nucleotides drive ribosome biogenesis and glycolytic reprogramming in acute myeloid leukemia stem cells.
              </a>
            </td>
          <td>
            Gentaro Kawano, Riichiro Ikeda, Daisuke Ishihara, T. Shima, Teppei Sakoda, Shunsuke Yamamoto, Yuichi Kochi, Yuichiro Semba, Sanae Ashitani, Y. Mori, Koji Kato, Takahiro Maeda, Toshihiro Miyamoto, Tomoyoshi Soga, Koichi Akashi, Y. Kikushige
          </td>
          <td>2025-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Intracellular parasites like Toxoplasma gondii scavenge host nutrients, particularly lipids, to support their growth and survival. Although Toxoplasma is known to adjust its metabolism based on nutrient availability, the mechanisms that mediate lipid sensing and metabolic adaptation remain poorly understood. Here, we perform a genome-wide CRISPR screen under lipid-rich (10% Fetal Bovine Serum (FBS)) and lipid-limited (1% FBS) conditions to identify genes critical for lipid-responsive fitness. We identify the Toxoplasma protein GRA38 as a lipid-dependent regulator of parasite fitness. GRA38 exhibits phosphatidic acid (PA) phosphatase (PAP) activity in vitro, which is significantly reduced by mutation of its conserved DxDxT/V catalytic motif. Disruption of GRA38 leads to the accumulation of PA species and widespread alterations in lipid composition, consistent with impaired PAP activity. These lipid imbalances correlate with reduced parasite virulence in mice. Our findings identify GRA38 as a metabolic regulator important for maintaining lipid homeostasis and pathogenesis in Toxoplasma gondii.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527f4bbd127d6492e648846eacca387bafbea4ec" target='_blank'>
              A genome-wide CRISPR screen identifies GRA38 as a key regulator of lipid homeostasis during Toxoplasma gondii adaptation to lipid-rich conditions
              </a>
            </td>
          <td>
            Mebratu A Bitew, T. Paredes-Santos, Parag Maru, Shruthi Krishnamurthy, Yifan Wang, L. O. Sangaré, S. Duley, Yoshiki Yamaryo-Botté, Cyrille Y. Botté, Jeroen P. J. Saeij
          </td>
          <td>2025-12-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 T-cell acute lymphoblastic leukemia (T-ALL) is a highly aggressive hematologic malignancy driven by uncontrolled proliferation of immature T-cell precursors. Despite recent therapeutic advances, relapse remains frequent, underscoring the need for novel targeted strategies. Our previous work revealed that T-ALL cells exhibit a profound reliance on oxidative phosphorylation (OXPHOS), which supports energy production necessary for rapid proliferation and confers resistance to conventional therapies (Baran et al., Nat Commun 2022). While blockade of OXPHOS via inhibition of mitochondrial Complex I shows initial efficacy, T-ALL adapts by upregulating glutaminolysis and glycolysis, exporting lactate via monocarboxylate transporters (MCTs), and acidifying the microenvironment, thereby limiting durable responses. Understanding these intricate metabolic dependencies is crucial for identifying new intervention points aimed at disrupting energy pathways and overcoming therapy resistance.
 To systematically uncover these adaptive vulnerabilities, we performed genome-wide CRISPR-Cas9-based synthetic lethality screens in PF382 T-ALL cells treated with OXPHOS and MCT1 inhibitors. These screens particularly following MCT1 inhibition, identified key mitochondrial dependencies, including electron transport chain components (NDUF, UQCRC, COX), mitochondrial ribosomal proteins, mitochondrial translation factors, TCA cycle enzymes, mitochondrial genome regulators, and cofactors critical for mitochondrial function. Additionally, significant hits involved stress response pathways: sensors of apoptosis, chromatin- and nuclear membrane regulators, as well as lipid metabolism, lipid biosynthesis, and membrane trafficking genes. These findings suggest that dual OXPHOS/MCT1 inhibition triggers extensive metabolic reprogramming involving mitochondrial dysfunction, oxidative stress, and chromatin remodeling, which collectively enable cell survival.
 Combined OXPHOS/MCT1 targeting resulted in potent synthetic lethality (SL) by disrupting critical mitochondrial energy generation and lactate export. Consistent with these findings,
 Seahorse and GEA analyses indicated that MCT1 blockade increases OXPHOS activity, unveiling SL relationships involving mitochondrial biosynthesis and bioenergetics pathways, highlighting OXPHOS inhibition or downstream targeting as a promising potent therapeutic approach. We further validated these mechanisms utilizing multi-omics (GEA, targeted and untargeted metabolomics, in-silico METAFlux), functional assays (Seahorse, flow cytometry (FL), western blotting (WB)), and advanced imaging (confocal-, electron (EM)-, high-resolution (HRM)- microscopy in-vitro, hyperpolarized MRI in-vitro and in-vivo).
 In vitro, MCT1/OXPHOS inhibition caused irreversible mitochondrial damage, disrupted fusion/fission dynamics (EM, HRM), impaired enzymatic activity of mitochondrial complexes, perturbed transmembrane traffic of metabolites (Mass spectrometry, METAFlux), perturbed oxidative and anaerobic respiration (Seahorse), depleted ATP, disrupted redox homeostasis (Mass spectrometry), elevated ROS leading to DNA damage, and induced apoptosis (FL, WB), while sparing healthy hematopoietic cells. MCT1/OXPHOS blockade, in line with results of our screen, induced intracellular acidification and triggered lipophagy, rendering cells additionally vulnerable to inhibitors of lipid metabolism, as indicated in our in vitro screen.
 In vivo, hyperpolarized MRI in T-ALL PDX models, supported by an ex-vivo metabolites analysis (HPLC), confirmed the therapeutic effect, demonstrated by on-target reduced pyruvate-to-lactate ratios and increased lactate trapping post-MCT1 and MCT1/OXPHOS inhibitors treatment, with the latter leading to disease eradication and significantly prolonged overall survival.
 In summary, our CRISPR-Cas9 screens reveal critical mitochondrial dependencies and adaptive metabolic pathways in T-ALL. Targeting OXPHOS and MCT1, or their downstream signaling, simultaneously induces synthetic lethality toward T-ALL cells, offering a promising therapeutic strategy to eradicate T-ALL cells, providing therapeutic window to spare healthy hematopoietic cells, and ultimately warranting further in vitro and in vivo investigations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5294decf637f17e1b03096897a9ba0d5e8eb488" target='_blank'>
              CRISPR-Cas9 screen-driven identification of metabolic vulnerabilities reveals synthetic lethality via synergistic targeting of mitochondrial energy pathways and lactate export in oxphos-dependent T-ALL
              </a>
            </td>
          <td>
            Natalia Baran, Alessia Lodi, Shraddha Patel, Jose Enriquez Ortiz, Meghan Collins, Priyanka Sharma, Vakul Mohanty, Merve Dede, Ivo Veletic, Cassandra L Ramage, Jun Hyoung Park, Yogesh Dhungana, Anna Skwarska, Yu Zhou, Zhihong Zeng, Connie C. Weng, Kala Hayes, Gheath Alatrash, K. Dunner, J. Marszalek, G. Borthakur, Jiyang Yu, Palaniraja Thandapani, R. Davis, B. Kaipparettu, Ken Chen, Pratip K Bhattacharya, S. Tiziani, Marina Konopleva
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Erythropoiesis is a complex, tightly regulated process involving sequential changes in epigenetics, metabolism, and proliferation. While conventional immunophenotyping combined with transcriptomic and epigenomic profiling has provided valuable insights, these approaches lack the resolution to fully capture dynamic transitions. A comprehensive understanding of how these programs are coordinated remains incomplete.
 To address this, we integrated single-cell RNA sequencing (scRNA-seq), cellular indexing of transcriptomes and epitopes (CITE-seq), histone modification profiling, and chromatin accessibility analysis to dissect murine erythropoiesis at high resolution. scRNA-seq identified nine transcriptional clusters (C1–C9) along a continuous trajectory. Conventional R3 cells (CD71⁺Ter119high) were further resolved into three subclusters (C5–C7), exhibiting heterogeneity in ribosome biogenesis, heme metabolism, hemoglobin gene expression, chromatin condensation, and enucleation priming. Notably, C4 displayed the highest unique molecular identifier (UMI) counts and total RNA content, co-expressing markers from neighboring clusters alongside its own signature genes.
 Differential expression analysis of C4 versus neighboring clusters revealed three coordinated changes: (1) initiation of heme biosynthesis and hemoglobin gene expression, (2) a decline in ribosome biogenesis compared to C3, and (3) a metabolic switch from glycolysis to oxidative phosphorylation (OXPHOS), marked by a 98% reduction in Pkm (pyruvate kinase M) expression. To probe this metabolic shift, we FACS-sorted R2 (CD71⁺Ter119⁻), R2–3 (CD71⁺Ter119med, corresponding to C4), and R3 populations to assess intracellular pyruvate and ATP levels and chromatin accessibility. Consistent with reduced Pkm, R2–3 cells showed a sharp decline in pyruvate and diminished ATAC-seq signals at the Pkm locus, suggesting epigenetic repression. CUT&Tag data revealed increased H3K27me3 at Pkm in R2–3, implicating Polycomb Repressive Complex 2 (PRC2) in the regulation of this metabolic switch.
 To test PRC2's functional role, we generated Ezh2 conditional knockout mice using Vav-Cre. These mice exhibited defective erythropoiesis, with R2–3 cell accumulation and greatly reduced downstream R3/R4 populations, resulting in anemia. scRNA-seq of Ezh2 KO bone marrow precursors revealed premature chromatin opening at hemoglobin and heme synthesis gene loci (e.g., Alas2, Fech), even though these genes were not direct PRC2 targets in wild-type cells. Hb gene expression in KO R4 cells remained comparable to wild-type, but heme synthesis gene expression was suboptimal, suggesting impaired heme availability.
 Strikingly, Pkm expression increased ~160-fold in KO C4 cells, and glycolysis gene scores were elevated across KO clusters, indicating enhanced pyruvate-generating capacity. FACS-sorted KO R2–3 and R3 cells confirmed pyruvate accumulation, exhibited reduced ATP levels, and showed impaired proliferation and increased apoptosis.
 To test the effect of pyruvate accumulation directly, we treated G1ER cells (arrested at the R2–R2–3 transition) with increasing pyruvate (10–50 mM). Intracellular pyruvate increased with dose, while 40–50 mM pyruvate led to ATP reduction, impaired proliferation, and increased cell death. Similarly, in vitro–differentiated Lin⁻ bone marrow cells treated with 50 mM pyruvate exhibited reduced frequencies of R2–3 and R3 cells and elevated cell death. In contrast, Lin⁺ cells cultured under the same conditions were less affected, suggesting lineage-stage-specific sensitivity to pyruvate overload.
 In summary, we identify a metabolically distinct transitional population (C4) in which erythroid precursors undergo a PRC2-regulated shift from glycolysis to OXPHOS, coinciding with the onset of heme and hemoglobin production. PRC2 suppresses Pkm expression via H3K27me3 deposition, thereby restraining pyruvate production and coordinating energy metabolism with terminal differentiation. Loss of PRC2 disrupts this metabolic control, leading to pyruvate accumulation, ATP deficiency, and defective erythropoiesis culminating in anemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a64b2b511f581be6d16c4c74b3ebadad0ee9f0b5" target='_blank'>
              PRC2-regulated glycolysis-to-oxphos metabolic switch coordinates the onset of terminal erythropoiesis
              </a>
            </td>
          <td>
            Zhiling Shi, Jiazhuo Li, Panpan Zhu, Stuart H Orkin, Huafeng Xie
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction: The t(4;14) translocation subtype of multiple myeloma (MM) is characterized by overexpression of NSD2, a histone methyltransferase which catalyzes di-methylation of histone H3 lysine 36 (H3K36me2). About 12% of MM patients have t(4;14) and this is associated with poor outcome. If and how NSD2 drives unique epigenetic vulnerabilities in this subtype of MM is not well understood.
 Methods: Primary MM samples from the MMRF CoMMpass trial (NCT01454296) were analyzed by Whole Genome Bisulfite Sequencing (WGBS). CRISPR/Cas9 was used to genetically ablate the NSD2 translocated allele in KMS11 and KMS18 (hereafter referred to as NSD2-high and NSD2-low). Multiple single cell clones were isolated and confirmed by immunoblot analysis and sequencing. KMS11 NTKO and TKO models (Kuo et al. Molecular Cell 2011) were also used for CUT&Tag and DNAm analysis. Cell responses to decitabine (DAC) and GSK-3685032 were evaluated by flow cytometry for Annexin V and live/dead staining on day 4. Dynamic BH3 profiling (DBP) was performed similar to previously described (Matulis, ASH 2024). Gene expression was assessed using RNAseq with ribosomal RNA depletion (Kapa).
 Results: Analysis of CoMMpass samples showed higher levels of DNAm in the t(4;14) subtype compared to other subtypes (p=1.17x10-12). Consistent with DNAm being connected to NSD2 overexpression, KMS11 NSD2-high cells had higher DNAm than isogenic NSD2-low cells (p<1x10-9). Elevated DNAm in NSD2-high cells co-occurred in genomic regions with higher levels of H3K36me2 (OR=1.85, p<1x10-15). Treating KMS11 and KMS18 NSD2 isogenic cell lines with the DNA hypomethylating agent DAC showed NSD2-high cells are significantly more sensitive to DNAm inhibition (p<0.01 at ≥100 nM for KMS11 and p<0.001 at ≥500 nM for KMS18). This sensitivity to DNAm inhibition was seen again using the DNMT1 enzymatic inhibitor GSK-3685032 (p<0.001 at ≥100 nM). DBP of KMS18 showed NSD2-high cells increased mitochondrial priming when treated with DAC (p<0.001) whereas NSD2-low cells did not significantly change, consistent with the increased apoptosis observed in NSD2-high cells.
 RNAseq of KMS11 NSD2-high and NSD2-low clones treated with 0, 100, or 500 nM DAC showed 3,599 differentially expressed genes (FDR<0.01, fold-change>2). Genes downregulated with DAC treatment were common between NSD2-high and NSD2-low cells and included genes involved in cell cycle and MYC targets as indicated by Gene Set Enrichment Analysis (GSEA). Genes upregulated with DAC-treatment were more distinct between NSD2-high and NSD2-low cells, with GSEA indicating interferon response and TNFa signaling via NFKB more significantly induced in NSD2-high cells (FDR<0.0023) than in the NSD2-low cells (FDR>0.13).
 Further investigation of DAC-induced RNAseq shows an increase in transcription of endogenous retroviruses (ERVs), specifically in the NSD2-high KMS11 cells. This suggests that hypomethylating agents induce transcription of these ERVs, which then induce an intracellular interferon response, and this is specific to NSD2-high MM. Consistent with this, these ERVs had higher levels of DNAm in isogenic KMS11 NSD2-high cells as compared to NSD2-low cells (FDR<0.01). This was also observed in CoMMpass samples where higher DNA methylation was found at these ERVs in t(4;14) MM.
 Conclusions: NSD2-mediated increases in H3K36me2 in t(4;14) MM correspond with higher DNAm as compared to other MM subtypes. This is consistent with data from other cell types suggesting H3K36me2 is recognized by the PWWP domains of the DNMT3 DNA methyltransferasaes. Multiple NSD2 isogenic models suggest this creates a preferential sensitivity to DNA hypomethylating agents in NSD2-high MM. This phenomenon was shown both with DAC, a cytosine analog that incorporates into DNA, as well as with GSK-3685032, a DNMT1 enzymatic inhibitor. Both Annexin V staining and DBP indicated increased apoptosis, specific to NSD2-high cells. In addition, RNAseq data suggests this increased apoptosis is due to re-expression of ERVs that are triggering a viral defense response and interferon signaling. Consistent with this, t(4;14) cell line models and patient samples had higher levels of DNAm at ERVs, suggesting these are silenced through DNAm in t(4;14), but by other mechanisms in non-t(4;14) MM. Together, these data indicate NSD2 remodels the epigenome of t(4;14) MM creating unique epigenetic dependencies that can be exploited with readily available DNA hypomethylating agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c372d03852ab31bfaf4fcce61c8d2085a69b5cdf" target='_blank'>
              NSD2 expression remodels DNA methylation, creating a targetable epigenetic dependency in t(4;14) multiple myeloma
              </a>
            </td>
          <td>
            Anna Baj, Jonathan C Patton, Robert M Chavez, Shannon M Matulis, Doris R. Powell, Mala Shanmugam, Arun Wiita, Constantine S Mitsiades, J. Keats, Jonathan Licht, S. Lonial, Lawrence Boise, Benjamin G. Barwick
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="


 Background: Despite recent advancements in treatment for multiple myeloma (MM), most patients eventually relapse due to the emergence of resistant tumor subclones. Bulk DNA sequencing has identified recurrent resistance-associated mutations and can infer subclonal architecture, but it lacks the resolution to determine whether multiple mutations co-exist within the same malignant cell or arise in separate subclones. This distinction is critical, as multiple somatic variants within individual cells may confer resistance that does not arise from any single alteration alone. We hypothesized that targeted single-cell DNA sequencing of myeloma tumor cells would reveal the co-occurrence of mutations within individual cells that are missed by conventional bulk sequencing approaches.
 Methods: Bone marrow aspirates were collected from 46 patients, including 9 with newly diagnosed and 37 with relapsed or refractory MM. Two patients were sampled at multiple time points, yielding a total of 48 samples. CD138⁺ plasma cells were enriched using magnetic-activated cell sorting and subjected to single-cell encapsulation for simultaneous genomic and proteomic profiling. Libraries were prepared using the Mission Bio Tapestri Single-Cell Multiple Myeloma Multiomics Assay, which includes amplicons targeting single-nucleotide variants, insertions and deletions, and focal copy number aberrations commonly altered in the disease. In parallel, immunoglobulin heavy- and light-chain V(D)J clonotypes were identified, and cells were labeled with a 20-plex oligo-conjugated antibody panel to quantify surface protein markers associated with cell lineage and immunotherapy targets. Tumor cells were defined as those harboring the dominant immunoglobulin V(D)J sequence within each sample. Reference cells were identified as VDJ-negative and expressing high levels of non–plasma cell markers (CD3, CD19, or CD33). Somatic mutations were identified by comparing variant allele frequencies between tumor and reference cell populations.
 Results: Across all 48 samples, 146,946 cells were sequenced, with 68,052 of those being tumor cells. Hallmark MM copy number alterations, such as 17p deletion, 1q amplification, and 13q deletion, were identified using the copy number assay. The most frequent mutations identified were in KRAS, TP53, TENT5C, and CRBN. The two patients with CRBN mutations underwent 27 and 34 prior cycles of immunomodulatory (IMiD) therapy, respectively, compared to a median of 8.0 cycles in patients without CRBN mutations, raising the possibility of treatment-driven selection. In another case, we identified three mutually exclusive KRAS mutations (Q61H, G12V, and G12D) in distinct subclones within a single patient, consistent with convergent evolution toward activation of the MAPK pathway. We also resolved a true “double hit” in another patient, characterized by 58% of tumor cells carrying a deletion of one allele and a somatic mutation in the other, consistent with biallelic inactivation. Finally, we observed CDKN2A more frequently co-mutated with TP53, PTPRT, and TENT5C than as isolated events, suggesting multi-hit inactivation of key cell-cycle checkpoints and DNA-damage response pathways that may drive selective growth advantage or therapeutic resistance. CDKN2A and TP53 are canonical tumor suppressors that regulate G1/S transition and apoptosis, respectively, while PTPRT encodes a phosphatase that negatively regulates STAT3 signaling, and TENT5C is a tumor suppressor involved in mRNA stability and plasma cell homeostasis. These recurrent co-mutations may act synergistically by simultaneously disrupting transcriptional and signaling regulation, thereby conferring a selective tumor cell growth advantage under therapeutic pressure.
 Conclusions: Targeted single-cell DNA sequencing reveals co-occurring mutations within individual myeloma cells that are obscured by bulk approaches, providing critical insight into the cooperative and competitive genomic events driving disease progression and resistance. These findings support a model in which myeloma subclones accumulate synergistic alterations within the same cell. Work is ongoing to provide functional validation of key mutation pairs and their integration with transcriptomic data to refine the mechanistic understanding and identify candidates for combination therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98368199e493cbd0fcaadb2df0e1e6f8e200c1a5" target='_blank'>
              Single-cell DNA sequencing uncovers synergistic co-mutations in multiple myeloma
              </a>
            </td>
          <td>
            Ava Amidei, B. Diamond, Elizabeth Lyubchenko, Stephanie Fernandes, Baharak G. Fard, L. Nousheen, Marc Arribas-Layton, Adam Sciambi, D. Kazandjian, James Hoffman, Abhishek Pandey, Marcella Kaddoura, Brian Walker, O. Landgren, D. Coffey
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Multiple myeloma (MM) is a genetically heterogeneous malignancy marked by complex clonal dynamics and a range of cytogenetic alterations that impact prognosis and treatment response. While fluorescence in situ hybridization (FISH) remains a standard diagnostic tool, it is limited to known abnormalities and requires enriched plasma cells. Next-generation sequencing (NGS) offers comprehensive profiling in a single assay but is not yet widely adopted in routine clinical care.
 To evaluate the performance and clinical utility of a custom-designed, dual-input (DNA/RNA) targeted sequencing panel across multiple platforms (Ion Torrent and Illumina) for molecular karyotyping, clonality assessment, and transcriptomic profiling in newly diagnosed MM.
 Bone marrow samples from 85 patients enrolled in GEM clinical trials with high-risk FISH-defined cytogenetics were analyzed. DNA and RNA were extracted from CD138⁺ plasma cells and CD138⁻ germline controls. Targeted capture libraries were sequenced on Illumina and Ion Torrent platforms using a panel covering 2.735 Mb and 107 genes involved in MM pathogenesis. Bioinformatic analysis included variant calling (SNVs, indels), structural variant and CNV detection, and V(D)J rearrangement analysis via MiXCR. IG/TR clonality was calculated using normalized Shannon entropy. RNA-seq data were used for differential expression analysis.
 The sequencing panel achieved high coverage (median depth: 602× on Illumina; 291× on Ion Torrent) and uniformity. Molecular karyotyping revealed: mutations in 145 variants affecting 59 genes, with frequent alterations in KRAS, NRAS, ZFHX4, and TP53; translocations t(4;14), t(11;14), and t(14;16) were detected with 99% accuracy compared to FISH; and CNVs including del(1p), gain/amplification(1q), and del(17p) were accurately distinguished (PPV: 95%, sensitivity: 92%). Notably, NGS identified del(1p) events undetectable by CDKN2C FISH probes in 7% of cases and discriminated 1q gain vs amplification, potentially refining risk stratification.
 In parallel, RNA-seq revealed distinct expression clusters, including CCND1 and FGFR3-driven subgroups, and differential expression of SLAMF7, NF1, and ABCB4 associated with survival outcomes. IG/TR clonality, especially RNA-based IG clonality, was significantly associated with both overall and disease-free survival (p < 0.05).
 This dual-platform, dual-input targeted sequencing strategy enables a comprehensive molecular characterization of MM from limited material. It surpasses conventional techniques in detecting structural and numerical chromosomal abnormalities and provides actionable insights into clonal architecture and gene expression with prognostic significance. These findings support the integration of DNA/RNA-based NGS panels into routine MM diagnostic workflows to guide risk-adapted treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67577b2c15159cac0263f5ee5152e9e90515eba0" target='_blank'>
              Comprehensive DNA/RNA-based targeted sequencing for molecular karyotyping and clonality assessment in newly diagnosed multiple myeloma patients
              </a>
            </td>
          <td>
            J. M. Rosa-Rosa, N. Buenache, I. Vazquez, R. Sánchez, C. Jiménez, L. Rosiñol, Norma C. Gutierrez, A. Oriol, Manuela Fernández, M. Blanchard, R. Ayala Diaz, R. Ríos, A. Sureda, M. H. Hernández García, J. de la Rubia, Joan Batista Blade Creixenti, M. Mateos, J. L. Lahuerta Palacios, J. San-Miguel, M. Calasanz, Ramón García-Sanz, J. Martínez-López
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3160e385fe96505bcb09a45df54fdae63937a125" target='_blank'>
              NONO as a Sensor of Intracellular Oxidation: Relevance to Neuroblastoma Cell Death
              </a>
            </td>
          <td>
            S. S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="The most prevalent kind of acute leukemia in adults is acute myeloid leukemia (AML). While some individuals have had better effectiveness due to advancements in targeted medications, recurrence after remission and inadequate treatment specificity continue to be significant therapeutic problems. By controlling essential metabolic pathways and metabolites, metabolic reprogramming, a crucial strategy for cellular adaptability to energy needs, modifies cellular metabolic rhythms. In addition to being involved in immune cell proliferation, differentiation, and effector function, this pathway is also essential for leukemogenesis and survival signaling in AML. By altering the expression of immune molecules, the release of certain metabolites (such as lactate, ROS, glutamine, etc.) has a significant impact on the immune response to tumors. It is noteworthy that the metabolic interactions between immune cells and AML cells form a distinct pattern of energy competition in the tumor microenvironment. This study examined the new approach of targeting metabolic pathways to improve immunotherapy, systematically clarified the regulatory mechanism of metabolic reprogramming between AML cells and immune cells to counteract tumor immunity, and concentrated on the synergistic effect of current therapies and metabolic interventions. These findings offered a fresh perspective on how to fully realize the potential of metabolic therapy for AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1000f5360b85e79cb0e223ba72fdd74742ac3a53" target='_blank'>
              Metabolic reprogramming and immune regulation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Xiaogang Hao, Chenyang Fan, Lixiang Yan, Reaila Jianati, Yifei Guo, Xinli Zhou, Gengda Zhu, Yucheng Zhang, Chengyulong Zheng, Ying Zhang, Zhexin Shi
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is now recognized as pivotal to cancer progression and treatment outcomes, alongside cancer cell-intrinsic factors. Among TME constitute, cancer-associated fibroblasts (CAFs) are abundant and actively shape tumour biology. They are heterogenous population arising from multiple origins, and engage in dynamic crosstalk with cancer cells and immune infiltrates to promote virtually all the hallmarks of cancer. By influencing and reinforcement of cancer hallmark traits, CAFs create a pro-tumorigenic niche that also protects cancer cells from therapy, in this way, conferring resistance to conventional therapy. Consequently, CAFs have emerged as central players linking cancer hallmarks to therapeutic resistance. This review discusses the origins and phenotypic diversity of CAFs, the mechanisms of their communication with tumour and immune cells, and how these interactions enforce hallmark capabilities of cancer that underlie treatment resistance. We also highlight preclinical and clinical evidence for CAF-driven drug resistance and examine current strategies to target CAFs or their effects. A deeper understanding of CAF heterogeneity and function could pave the way for novel combination therapies that dismantle the tumour-CAF alliance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10e61bcec209fefda855e7181e3c9e5807e7aba" target='_blank'>
              Cancer-associated fibroblasts: central players in cancer hallmarks and therapeutic resistance
              </a>
            </td>
          <td>
            Yuanjun Lu, Ya Wu, Xiaoyu Xu, Yibin Feng, Ning Wang
          </td>
          <td>2025-12-24</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Highlights What are the main findings? CRISPR models reveal gene-specific contributions to ovarian and metabolic PCOS phenotypes, including hyperandrogenism and insulin resistance. Single-gene models show species-specific effects, highlighting the need for dual-gene and integrated models. What are the implications of the main findings? CRISPR studies identify potential diagnostic markers and clarify gene-hormone interactions in PCOS. Combined gene editing and hormonal models provide a platform for studying the polygenic, multifactorial nature of PCOS. Abstract Polycystic ovary syndrome (PCOS) is a complex disorder characterized by reproductive abnormalities such as hyperandrogenism, ovulatory dysfunction, and polycystic ovarian morphology, and is frequently accompanied by metabolic disturbances such as insulin resistance, obesity and dyslipidemia. Genome-wide association studies (GWASs) have identified several susceptibility loci, yet little is known about their functional implications. Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (CRISPR/Cas9) has emerged as a powerful gene editing tool in bridging this gap by allowing researchers to directly target candidate genes in ovarian and metabolic pathways. For instance, experimental models have highlighted the role of CYP17A1 and DENND1A.V2 in androgen excess, anti-Müllerian hormone (AMH) in follicular arrest, and insulin receptor substrate 1 (IRS1) and PPARγ in insulin signaling and adipogenesis. To highlight the multifactorial nature of PCOS, animal models, including zebrafish and rodents, have been used to reveal interactions between reproductive and metabolic phenotypes. Nevertheless, most studies remain restricted to single-gene models, and dual-gene models or combined gene editing and hormonal induction models remain underexplored. Future research integrating precision editing, multi-omic platforms, and patient-derived organoids may provide more accurate disease models and novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3a8b22bc194b7aa0b8690d31a434228fd726258" target='_blank'>
              CRISPR as a Tool to Uncover Gene Function in Polycystic Ovary Syndrome: A Literature Review of Experimental Models Targeting Ovarian and Metabolic Genes
              </a>
            </td>
          <td>
            Shahd Bucheeri, Yasmine Alcibahy, Yara Bucheeri, Sarah Bucheeri, Abrar Alhermi, Alexandra E. Butler
          </td>
          <td>2025-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Diffuse midline gliomas (DMGs) have uniformly poor survival outcomes despite initial responses to radiation therapy (RT). Functional genomic and high throughput drug screens have identified promising therapeutic targets for DMGs, but these efforts have largely been performed in cell cultures, thereby missing targets that are selectively vulnerable within in vivo tumor environments. Here, we develop a genome-wide CRISPR interference (CRISPRi) platform to screen essential and radiation sensitizing genetic targets in DMG xenografts. We demonstrated the feasibility of genome-wide in vivo screens by introducing a lentiviral library of 22,580 inert barcodes prior to orthotopic transplantation of SJ-DIPG7 cells. Individual tumors yielded only 55-77% of this library after 3-4 weeks of intracranial growth. However, in silico pooling of two or more tumors enabled >88% genome-wide coverage. We then performed in vivo CRISPRi screens using a dual-sgRNA genome-wide library comprising 21,554 gene-targeting vectors and 1,026 non-targeting controls. SJ-DIPG7 stably expressing dCas9-Zim3 were infected with CRISPRi libraries and intracranially transplanted to immunocompromised mice. Animals were treated with control or RT (2 Gy x 5 fractions), a dose that extended survival from SJ-DIPG7 xenografts but not sufficient for cure. After in silico pooling of 2-3 mice, 93% of barcoded sgRNAs were recovered and we identified 340 genes that diminished in vivo growth upon knockdown (mean log2 fold-change -5.58;FDR 2%). These genes included 145 known essential genes from DepMap, supporting the validity of our method, and were enriched for proteasome components. We also identified 108 radiation sensitizing targets (mean log fold-change -6.02;FDR 7%) including DNA repair (TOPBP1, PPP4C), metabolism (CNDP1, LDHA, OGDH), and neuronal function (LSAMP, EFNA1). We validated the essential phenotype of the proteasome component PSMA3. Our work establishes the feasibility of genome-wide in vivo functional genomics for systematically identifying therapeutic vulnerabilities of DMGs, enabling the discovery of combination treatments for patients living with DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c58f1e437f00caa915d46128ea9259f60b4c0f9" target='_blank'>
              DDDR-47. Development of in vivo CRISPR interference screens in diffuse midline glioma xenografts to identify genetic vulnerabilities controlling responses to radiation therapy
              </a>
            </td>
          <td>
            I-Ching Wang, C. Zou, Akane Hubbard, Sasha Sengelmann, T. Ozawa, D. Raleigh, Christopher L. Tinkle, A. Shelat, S. J. Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Introduction: Multiple myeloma (MM) is an incurable plasma cell (PC) malignancy, in which nearly all patients are expected to relapse and undergo multiple lines of treatment over the course of their disease. By the time patients receive CAR-T cell therapy, they have typically been exposed to numerous prior therapies, including high-dose melphalan (HDM). Chemotherapeutic agents that cause genotoxic stress, such as HDM, can induce cellular senescence in normal cells. More recently, it has been demonstrated that tumor cells can also exhibit features of therapy-induced senescence (TIS). TIS in tumor cells is associated with distinct gene expression patterns that have been implicated in resistance to therapy and relapse. To investigate whether TIS is activated in MM cells following exposure to chemotherapies, particularly in the context of late-stage disease, we performed an integrated analysis of single-cell RNA sequencing (scRNA-seq) datasets.
 Methods: We integrated scRNA-seq data from bone marrow plasma cells and MM cells using the following three sources: 1) Boiarsky, et al. (GSE193531, dbGaP: phs001323.v3.p1), which includes healthy donors, MGUS, SMM, and newly diagnosed MM (NDMM) patients; 2) Dhodapkar, et al. (GSE210079), which includes late-stage MM patients pre- and post-CAR-T cell therapy; and 3) newly generated scRNA-seq data from healthy donor (N=3) and late-stage MM patients pre- (N=10) and post- (N=2) CAR-T cell therapy. Non-plasma cells were excluded based on canonical plasma cell markers.
 We used Seurat v4.3 for data integration, employing canonical correlation analysis (CCA) to define anchors using the FindIntegrationAnchors and IntegrateData functions. Integration quality across datasets was validated by the consistent clustering of healthy plasma cells and expected expression patterns of chromosomal translocation markers CCND1 in t(11;14) cases and FGFR3 in t(4;14) cases, as previously described (PMID: 36396631).
 To evaluate senescence, we applied single-cell gene set enrichment analysis (scGSEA) using previously published gene sets (PMID: 40164720): SenUp (Senescence Upregulated), SenGA (Senescence Growth Arrest), and SCAPs (Senescent Cell Anti-apoptosis Pathways). We also curated a Plasma Cell Senescence (PCSen) gene set, composed of senescence-related genes that were differentially upregulated in pre-malignant plasma cells across two independent datasets (GSE5900, GSE47552).A normal plasma cell gene set was also generated, comprising genes differentially expressed between healthy and abnormal plasma cells in GSE193531.
 Results:In the integrated dataset, scGSEA revealed significantly higher enrichment of PCSen, SenUp, SenGA, and SCAPs in late-stage MM cells (both pre- and post-CAR-T cell therapy) compared to NDMM and healthy donors (p<0.05, Kruskal-Wallis with Dunn's posttest). Conversely, the Normal gene set was significantly enriched in healthy plasma cells relative to MM (p<0.05). Expression of CDKN2A, which encodes the senescence marker p16, was also increased in late-stage MM relative to NDMM or healthy plasma cells. Notably, in our newly generated scRNA-seq samples collected within 6 months before CAR-T therapy, we observed a significant inverse correlation between senescence marker expression and progression free survival (PFS) post-CAR-T, suggesting that MM cells exhibiting TIS may be more resistant to T cell mediated cytotoxicity. Supporting this, preliminary in vitro experiments showed that MM cells treated with HDM displayed reduced sensitivity to T cell-mediated cytotoxicity, consistent with a senescence-associated resistant phenotype.
 Conclusion: Our findings demonstrate that late-stage MM cells exhibit transcriptional signatures consistent with therapy-induced senescence, potentially driven by prior exposure to chemotherapy. This senescent phenotype may contribute to resistance to immunotherapies, including CAR-T cells. These data highlight senescence as a mechanism of resistance and a potential biomarker for predicting CAR-T outcomes. Further studies are needed to explore senescence-targeted strategies to enhance immunotherapeutic efficacy in MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2f51f1fce50aa3b8937f9efdeb797d0f3453ef" target='_blank'>
              Plasma cell senescence in relapsed multiple myeloma revealed by integrated single-cell RNA-seq analysis
              </a>
            </td>
          <td>
            Gabriel Alvares Borges, A. Guilatco, Sofia Arbelaez, Chen Wu, Neal I. Sannuli, Ying Li, T. Kourelis, W. Gonsalves, Yi Lin, Megan M. Weivoda
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29ad767de6b1b60629bc324b66e333d5c7c4a2b" target='_blank'>
              A Human Genetics Framework for De-risking Gene Editing Targets for Hematopoietic Cell and Gene Therapy
              </a>
            </td>
          <td>
            A. Dhall, Alyssa Pyclik, Matthew H. Ung, Julia Etchin, Y. Keschner, Huanying Gary Ge, Tirtha Chakraborty, J. Lydeard
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioblastomas (GBM) are aggressive brain tumors characterized by significant heterogeneity and notable resistance to treatment, particularly to temozolomide (TMZ), a standard chemotherapeutic agent utilized in clinical practice. A key mechanism underlying this resistance is the upregulation of the MGMT (O6-methylguanine-DNA methyltransferase) gene. Prior studies have shown that alterations in MGMT promoter methylation status influence MGMT expression levels. However, the regulatory mechanisms governing this process remain unclear. To identify the molecular changes associated with TMZ resistance, we established eight TMZ-resistant cell lines derived from patient-derived glioma stem cells (PDGs). Quantitative PCR (qPCR) and Western blot (WB) analyses confirmed elevated MGMT expression in these resistant cells. RNA sequencing of both treatment-naïve and resistant cells revealed significant alterations in key epigenetic enzymes, including DNA methyltransferases (DNMTs) DNMT1 and DNMT3A, as well as various DNA demethylation enzymes and methyl-binding domains, such as TETs and MBDs. qPCR further validated the downregulation of UHRF1 and DNMT1 mRNA in resistant cells. Besides, treatment-naïve cells exhibiting high methyltransferase expression demonstrated increased susceptibility to 5-Azacytidine (5-AZA), a pan DNMT inhibitor, compared to their resistant counterparts, which displayed reduced methylation capacity. Moreover, 5-AZA-mediated epigenetic reprogramming enhanced the resistance of treatment-naïve cells by inhibiting cell proliferation and upregulating MGMT expression. Notably, withdrawal of 5-AZA reverted MGMT expression levels to those observed in untreated cells. Retrospective analysis of sequencing data from paired initial and recurrent tumors in adult diffuse glioma patients revealed a consistent negative correlation between DNMTs expression and MGMT expression, with DNMT1 emerging as the most significant factor. Finally, specific inhibition of DNMT1 resulted in a marked upregulation of MGMT expression. In summary, our work highlights the role of MGMT in glioma TMZ resistance and identifies the downregulation of DNMTs as a critical mechanism facilitating MGMT upregulation, thereby contributing to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/184ff52797748eb27e56749cfa70525ad6478ba8" target='_blank'>
              DDDR-32. DNMTs epigenetically regulate MGMT expression and contribute to TMZ resistance in glioblastoma
              </a>
            </td>
          <td>
            Mengli Yan, Dong Song, Qi An, R. Chai, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Introduction: TP53 mutated myeloid neoplasms (MNs) are associated with aberrant copy number variations (CNVs), complex karyotype (CK) and dismal outcomes. Recent studies showed that TP53 mutated clonal hematopoiesis (CH) can also harbor subclonal populations with CNVs; and is linked with higher risk of transformation into therapy-related myeloid neoplasm (tMN) following chemotherapy. To track the evolution of CK myeloid entities and the role TP53 mutations play in disease development, we integrated single-cell whole genome sequencing (scWGS-seq) by direct library preparation (DLP, Laks et al 2019) and transcriptome sequencing (scRNA-seq) to track the clonal dynamics of CH, MN and tMN.
 Methods: 28 samples from 20 patients harboring TP53 mutations were ascertained by MSK-IMPACT panel (VAF >2%): 5 samples from 5 CH patients, 2 from 1 CCUS patient, 12 from 8 tMN patients, and 9 from 6 MN patients. scWGS-seq was performed using DLP, which offers single cell detection of CNV, single nucleotide variants (SNV), mutational signatures, and structural variants (SV). scRNA-seq was done using the 10x Genomics Chromium Single Cell 5' system; CNVs were inferred using InferCNV and Numbat packages. Cells with ≥3 CNVs were classified as CK.
 Results: We generated scRNA and scWGS data for 218280 cells from 28 samples and 8154 cells from 8 samples, respectively. In CH/CCUS, no CNVs were identified by scRNA (Total cells: 60253) or DLP (Total cells: 1524), despite high TP53 VAF (median of 24%, range: 11%-49%) and detection of TP53 mutations by both methods. In contrast, CNVs were detected in all 21 of MN/tMN samples by scRNA (median TP53 VAF=43% with range 4%-91%), and in 19/21 samples (13/14 patients) these CNVs presented as CK. In 7 CK samples with both scRNA and DLP, the latter provided higher resolution and revealed subclonal branching.
 In all 13 patients with CK clones, we noted 3 distinct modes of clonal evolution; single dominant clone (n=6), linear (n=4) and parallel (n=3) multi-clonal progressions, in line with Leppä at el 2024. Bi-allelic TP53 hits were detected in all CK clones. Importantly, in 3/5 patients with DLP, TP53-mutant cells lacking CK were identified, representing the most recent common ancestor predating CNV (del5q/del7q) acquisitions. In one patient with a history of exposure to platinum, platinum-associated mutational signatures (SBS31) was only observed in the CK branches of the phylogenetic tree, indicating that CNV and CK clones developed from a TP53 mutated CH clone already presented at the time of platinum exposure. Together with the absence of CNV in CH/CCUS, these findings showed that TP53 mutations always precede CK acquisitions (del5q/del7q) in myeloid pathogenesis.
 Thanks to DLP resolution, we identified partially overlapping chromothripsis in 4 tMN patients, resulting in focal hyperamplification on chr19p. This stratified CK into clones with chr19p hypergain (CK-19p) and without (CK). To validate this finding, we examined 88 TP53-mutant CK MN and tMN patients in the MSK-IMPACT database, and identified 9 (10%) with chr19p hypergain events. In all 9 cases, CK-19p clones appeared only after therapy, suggesting that chr19p hypergain was a late, therapy selected event.
 Transcriptomic profiling of clones' phenotype with scRNA revealed a bias toward progenitor cells in CK clones (42% vs 2% in non-CK clones) and MkEP-Erythroblasts in CK-19p clones (96% vs 4% in non-CK clones). Differential gene expression analysis showed that 1) CK progenitor clones were less proliferative with LSC quiescence-like phenotype (Zeng et al. 2022) and lower MYBL2 expression vs non-CK clones; 2) CK-19p MkEP-Erythroblasts clones were more proliferative than CK progenitor clones. These findings suggest that LSC quiescence-like phenotype is a hallmark of CK clones; and chr19p hypergain drives proliferation, resistance and a potentially distinct entity.
 Conclusions: Using scRNA and scWGS-seq, we showed that biallelic TP53 inactivation is the founding event driving diverse CK clones in MN and tMN patients. In contrast, in tMN, chr19 hypergain emerged as a late event promoting a proliferative, MkEP-Erythroid phenotype, potentially as a novel mechanism for resistance and disease evolution. Finally, we identified LSC quiescence-like phenotype and lower MYBL2 expression as the hallmarks of CK clones, enabling survival and evolution under cytotoxic pressure.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/291c6d225943d2d699ae779922d9b182fb7a229e" target='_blank'>
              Integrating single-cell whole genome and transcriptome sequencing to track the clonal evolution of TP53-mutated myeloid neoplasms
              </a>
            </td>
          <td>
            Sarun Sereewattanawoot, M. Sirenko, Georgios Asimomitis, Sheng F. Cai, Shoron Mowla, Kaitlyn Ko, Francesco Maura, Ross L Levine, Sohrab P. Shah, Andrew McPherson, G. Gundem, E. Papaemmanuil
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is an aggressive hematological malignancy with 5-year survival rate of less than 32%, mainly due to treatment failure and disease relapse. Thus, a systematic framework that allows us to monitor treatment response is urgently needed. Previously, we applied a state-transition modeling framework to represent AML disease evolution as messenger RNA (mRNA) and microRNA (miRNA) transcriptome trajectories in their respective state-spaces, partitioning samples into health, transition, and leukemic states based on mathematically defined critical points. These state-spaces were constructed using time-series RNA-seq data collected from a murine model of AML which mimics the fusion geneCBFB::MYH11 (CM) created by chromosome 16 inversion.
 To test the hypothesis that response to chemotherapy could be predicted by mRNA and miRNA transcriptome dynamics using a state-transition framework, we performed an experiment and mathematical analysis of longitudinal sample collections and multiomic profiling of peripheral blood from CM knock-in mouse model (Cbfb56M/+/Mx1-Cre) before, during, and after chemotherapy, as follows. After detection of overt AML (circulating cKit+ > 20%), CM mice (n=9) were treated with a “5+3” combination of cytarabine (50mg/kg/day, 5 days) and daunorubicin (1.5mg/kg/day, 3 days), modeling the standard “7+3” chemotherapy. Peripheral blood samples were collected weekly and were subjected to bulk RNA-seq and miRNA-seq. We used PCA to project the samples into a reference state-space constructed using untreated CM mice and C57BL/6 WT controls. The mRNA and miRNA transcriptome trajectories were first analyzed independently and then together to form a 2D multiomic state-space.
 After chemotherapy, we observed movement of both mRNA and miRNA transcriptomes from a leukemia state back towards a healthy state, before relapsing back to overt AML. Interestingly, we observed a desynchronization of mRNA and miRNA transcriptome dynamics post-chemotherapy. Specifically, the mRNA transcriptome immediately moved from a leukemia state to a health state, with an average peak response at week 4 (n=7; before week 4, n=2), however, a delayed response of 2 or more weeks was observed in the miRNA transcriptome trajectories, which continued to move towards a disease state post-treatment before moving towards a health state and reached a maximum response at later time-points (4 weeks, n=3; 5 weeks, n=2; 6 weeks and after, n=4). These desynchronized dynamics post-chemotherapy were enabled by the state-transition critical points and appeared more evident when visualized in a 2D multiomic state-space.
 To identify genes, miRNAs, and pathways altered by chemotherapy, we performed differential gene expression and gene set enrichment analyses. These analyses revealed that metabolic pathways such as OXPHOS, fatty acid metabolism and glycolysis are downregulated after chemotherapy and upregulated during relapse (p<0.05).
 We also performed a weighted gene co-expression network analysis on mRNAs and miRNAs to identify features with similar expression dynamics. We found a set of 34 miRNAs which were significantly upregulated and responsible for the delayed response of the miRNA transcriptome dynamics post-chemotherapy. Out of these 34 miRNAs, 23 are found in the DLK1-DIO3 imprinted region, located on chromosomes 14q32 in humans and 12qF1 in mice, which has been associated with stress response, pathogenesis of APL, solid tumors, and type 2 diabetes. These miRNAs have not previously been associated with effects of standard-of-care chemotherapy in AML. The co-localization of these miRNAs to a region of the genome suggests a potential explanation for genome instability and chemotherapy resistance.
 In summary, this study provides a longitudinal multiomic characterization of genomic instability and desynchronization of patterns of mRNA and miRNA expression induced by standard-of-care “7+3” therapy in AML, guided by mathematical analysis and critical points.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d8bb9687ac3e42f4c8f48f1349be6b29eb33446" target='_blank'>
              From theory to therapy: Predicting response to chemotherapy with multiomic longitudinal profiling of peripheral blood in AML
              </a>
            </td>
          <td>
            Jennifer Rangel Ambriz, Ziang Chen, Yu-Hsuan Fu, D. Frankhouser, Denis O’Meally, S. Branciamore, Jihyun Irizarry, Bin Zhang, Lianjun Zhang, Ying-Chieh Chen, Guido Marcucci, Russell C. Rockne, Ya-Huei Kuo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) and diffuse midline glioma (DMG) are two of the most aggressive brain tumors, marked by significant intratumoral heterogeneity and transcriptional plasticity, which drive resistance to therapy. These tumors share similar transcriptional programs, though the specific genes associated with these programs differ. Transitions between transcriptional states, particularly to the mesenchymal state, which is associated with greater resistance to treatment, complicate treatment strategies. A significant challenge in glioma research is the inability to dynamically track these state transitions in vitro and in vivo. To address this challenge, we introduce CRISPR-TAG reverse genetics, a novel approach that enables real-time monitoring of key transcriptional state markers in glioblastoma and diffuse midline glioma cell lines. Using CRISPR-TAG, we insert a fluorescent reporter sequence at the 3’ end of target genes, linking the fluorescent signal to endogenous gene expression. This allows us to dynamically track changes in gene expression as cells transition between states. We have successfully tagged 30 key cell state markers in patient-derived GBM and DMG cell lines, providing a robust model to investigate the upstream regulators of cell state transitions. By integrating CRISPR-TAG with global CRISPR screens and high-content pharmacological screens, we uncover genetic dependencies that underlie glioma plasticity. For example, we identify modulators of the mesenchymal cell state, including ANXA1 expression, which serves as a critical marker of this resistant state. Additionally, we demonstrate how our method allows in vivo tracking and modulation of cell state transitions in explant cultures, with AI-assisted analysis providing detailed insights into these transitions. In conclusion, CRISPR-TAG reverse genetics lays a critical foundation for building a high-resolution map of glioma state regulators, paving the way for targeted therapeutic strategies. By enabling the precise modulation of cell states, this method offers the potential to complement and enhance existing treatment modalities for GBM and DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eaf38521e14ac09d468437f1dbddc6b4207f994" target='_blank'>
              CNSC-29. RESOLVING THE REGULATION OF GLIOBLASTOMA CELL STATES BY CRISPR-TAG REVERSE GENETICS
              </a>
            </td>
          <td>
            Josephine Heinold, Madeleine Skeppås, Filippa Kruse, Clara Nordquist, Irem Uppman, Sven Nelander
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Despite significant therapeutic advancements in multiple myeloma (MM) treatment, which have improved patient outcomes and extended survival, most patients experience relapse or are non-responsive to therapy, leading to relapsed/refractory multiple myeloma (RRMM). The disease's complex progression, driven by intricate molecular mechanisms and genetic alterations, contributes to its resistance phenotype, underscoring an urgent need for innovative strategies to improve patient outcomes.
 Myeloma cells undergo extensive metabolic reprogramming to sustain rapid proliferation and survival. A key feature of this reprogramming is altered lipid metabolism, particularly the dysregulation of de novo fatty acid synthesis. Recent studies have identified de novo lipogenesis (DNL) as a critical metabolic vulnerability in MM. In our study, we characterized a novel de novo lipogenesis inhibitor (DNL-I) designed to target lipid metabolism in MM. DNL-I disrupts lipid homeostasis by selectively inhibiting de novo lipogenesis, the biosynthetic pathway responsible for endogenous fatty acid production.
 Our findings demonstrate that DNL-I potently induces apoptosis in both MM cells sensitive to standard therapies and those resistant to proteasome inhibitors (PI). The apoptotic effect was dose-dependent, with increasing concentrations of DNL-I correlating with enhanced cell death across multiple MM cell lines. Mechanistically, we found that DNL-I downregulates the expression of key enzymes involved in fatty acid synthesis, such as acetyl-CoA carboxylase (ACC) and fatty acid synthase (FASN), without altering the expression or activity of enzymes associated with fatty acid β-oxidation, such as carnitine palmitoyltransferase 1 (CPT1). This selective inhibition confirms that DNL-I specifically targets de novo lipogenesis, a pathway increasingly recognized as a vulnerability in cancer cells reliant on lipid availability for membrane biogenesis and signalling. Particularly, we conducted rescue experiments to validate the role of lipid depletion in DNL-I's mechanism of action. Supplementing MM cells with an exogenous lipid mixture containing arachidonic, linoleic, linolenic, myristic, oleic, palmitic, and stearic acids fully reversed DNL-I-induced apoptosis in both sensitive and PI-resistant MM cell lines. This reversal suggests that the cytotoxic effects of DNL-I stem directly from its disruption of lipid availability, which is essential for MM cell survival and proliferation.
 Notably, untargeted metabolomic analyses of DNL-I-treated MM cells revealed significant alterations in amino acid metabolism, the tricarboxylic acid (TCA) cycle, and sphingolipid metabolism, all of which are essential for MM cell survival. These findings highlight the therapeutic potential of targeting de novo lipogenesis in MM, particularly for overcoming resistance to proteasome inhibitors, and underscore the importance of lipid metabolism as a novel therapeutic target in this disease.
 Funding: The authors acknowledge the research funding from Blood Cancer UK (17014 to CB and SP), the Key Kendall Leukaemia Fund (KKL1361 to CB and SP) and Leukaemia & Myeloma Research UK (refs. 122498 to SP and 20700 to CB). FNK is sponsored by the Republic of Turkey Ministry of National Education Scholarship (MEB1416). AM is sponsored by Majlis Amanah Rakyat Malaysia.
 Authors note: CB and SP are co-senior authors and correspondence should be addressed to: s.papa@leeds.ac.uk or concetta.bubici@brunel.ac.uk
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce91d49ca87e228162d91203f9bfb93ecd66d9e4" target='_blank'>
              Rewiring lipid metabolism: A novel approach to overcome drug resistance in multiple myeloma
              </a>
            </td>
          <td>
            Abdullah AlMarzouq, F. Kacı, Alessio Lepore, Aqeelah Mohammad, Xiujie Zhang, A. Besse, Christoph Driessen, Lenka Besse, Salvatore Papa, C. Bubici
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents, including doxorubicin, are common. Therefore, this project aims to characterize the evolution of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from the ATRT cell line, BT16, and RMC cell line, PEDS005T, utilizing a dose escalation approach. This resulted in DR cell lines that are 10-to-30-fold resistant compared to the parental cell line. Furthermore, we generated two independent DR cell lines from each parental cell line, enabling the capture of distinct evolutionary trajectories leading to resistance. We profiled the transcriptomes by bulk RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1, in 3 of 4 models. To assess whether ABCB1 upregulation alone was sufficient for DR, we treated them with an ABCB1 pharmacological inhibitor and found that ABCB1-upregulated DR cells were re-sensitized to doxorubicin, whereas an ABCB1-negative BT16 DR cell line remained resistant. To determine a shared mechanism of resistance between these models, we profiled the methylomes by Oxford Nanopore Technologies long-read sequencing. We observed global hypermethylation when compared to the parental counterparts. Treatment with a hypomethylating agent reduced cell viability up to 75%. Future studies will focus on identifying and therapeutically targeting these commonalities to overcome resistance regardless of evolutionary trajectory. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there are different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, reduced cell viability resulting from treatment with a hypomethylating agent suggests there are common vulnerabilities despite differing evolutionary trajectories.



 Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong. The evolution of doxorubicin resistance in SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebc129836d1140b7ad59722551b94ec70c22ad5" target='_blank'>
              Abstract B017: The evolution of doxorubicin resistance in
 SMARCB1
 -deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="KRAS and BRAF mutations are critical oncogenic drivers in colorectal cancer (CRC), with distinct molecular and clinical implications. To uncover genes differentially expressed between these mutations, we compared KRAS G12D- and BRAF V600E-mutated CRC cell lines. Protein-protein interaction (PPI) network construction, pathway enrichment, survival analysis, and drug target screening were performed to identify therapeutic opportunities. Ten hub genes—TNF, IL1B, FN1, EGF, IFI44L, EPSTI1, AHR, COL20A1, CDH1, and SOX9—were identified as critical in KRAS-driven CRC. Enrichment analysis highlighted immune and inflammatory pathways, including “SARS-CoV-2 Signaling” and “Macrophage Stimulating Protein Signaling,” as well as cellular processes like “Positive Regulation of PI3K Signaling”. IL1B was the only hub gene significantly associated with overall survival, suggesting its role as a favorable prognostic marker. Drug screening identified selective inhibitors such as Canakinumab and Rilonacept targeting IL1B, with docking studies revealing the strongest interaction for Omeprazole with AHR, followed by Tapinarof with AHR and Donepezil with IL1B. This study sheds light on the molecular mechanisms of KRAS-mutated CRC, emphasizing IL1B as a prognostic marker. Among the identified therapeutic agents, Omeprazole demonstrated the strongest interaction with AHR, suggesting a potential for repurposing in CRC treatment, while IL1B-targeting inhibitors such as Canakinumab emerged as selective candidates for modulating tumor-promoting inflammation. Novel genes such as EPSTI1, COL20A1, CDH1, and SOX9 warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347d7ffedc4eee87a046791d3b66c644b633f639" target='_blank'>
              A systems biology approach to identify key targets in KRAS/BRAF-mutated colorectal cancer
              </a>
            </td>
          <td>
            Shiva Shiravi, Negar Mottaghi-Dastjerdi, Behzad Shahbazi, Nahid Ahmadi, Mohammad Soltany-Rezaee-Rad, Abozar Ghorbani, Hamed Montazeri
          </td>
          <td>2025-11-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 BackgroundThird-generation tyrosine kinase inhibitors (3G-TKIs), olverembatinib and ponatinib, improved outcomes for patients with resistant chronic myeloid leukemia (CML). However, nearly half still developed resistance or progression. The longitudinal molecular evolution during therapy, at both genomic and transcriptomic features, remains incompletely understood. We investigated the dynamic clonal architecture and transcriptional reprogramming during 3G-TKI therapy to elucidate mechanisms of therapeutic response and resistance.
 MethodsWe analyzed 202 CML patients treated with 3G-TKIs. Baseline and serial bone marrow or blood samples were collected for targeted DNA sequencing and RNA sequencing. Variant allele frequency (VAF) trajectories were used to classify mutational dynamics. Gene expression profiles (GEPs) were analyzed by multi-machine learning and unsupervised clustering. Pathway enrichment, immune deconvolution (xCell algorithm) and T cell state analysis (TcellSI algorithm) characterized biological transitions during therapy.
 ResultsDynamic mutational profiling revealed divergent clonal evolution. In responders (n = 49), the frequency and number of ABL1and other somatic mutations significantly declined (p < 0.05). In contrast, primary resistance cases (n = 62) showed persistent or newly emergent high-VAF mutation clones, particularly ASXL1G646Wfs*12, RUNX1, and PHF6. Unsupervised clustering identified four prognostically distinct mutational patterns: (1) persistent clones; (2) emergent/expanding clones; and (3 - 4) contracting or eradicated clones. Patterns 1 and 2 were significantly correlated with therapy resistance, whereas Patterns 3 and 4 were correlated with favorable responses.
 Transcriptomic remodeling during 3G-TKI therapy provided novel insights into the molecular basis of response and resistance. In responders, leukemic stem cells (LSCs), E2F, MYC and DNA replication related pathways were markedly down-regulated (FDR < 0.001) during therapy, while immune and apoptotic pathways (TNF-α and IFN-α) were significantly activated (FDR < 0.001 - 0.04). Immune infiltration, including T cells, NK cells and dendritic cells, significantly increased (p < 0.001). In contrast, resistant patients retained or more-activated LSCs and metabolic related pathways with minimal immune induction, reflecting an immune-cold, transcriptionally static state.
 Using multi-machine learning and unsupervised clustering, 3 molecular subtypes were defined: GEPC1 (immune-active, n = 42), GEPC2 (stromal-remodeling, n = 86) and GEPC3 (stemness-high and immune-cold, n = 21) with distinct therapy responses and outcomes (p < 0.001 - 0.006). During therapy, patients in GEPC1 showed transcriptomic features characterized by down-regulation of LSCs and cell cycle related pathways (FDR < 0.001), along with strong up-regulation of immune-related pathways, such as TNF-ɑ, interferon-ɑ pathways (FDR < 0.001). Immune infiltration, such as T cells, B cells and NK cells, increased significantly during therapy. GEPC2 exhibited a moderate but partial transition, with suppression of proliferation signatures and activation of extracellular matrix and stromal pathways, and enrichment of fibroblasts and M2 macrophages. Immune activation was present but attenuated. In contrast, GEPC3 showed only limited transcriptional remodeling during therapy. Although LSCs and proliferative signatures declined slightly, they remained markedly higher compared to GEPC1/2, and immune activation was absent. xCell and TcellSI showed no increase in effector T cells, dendritic cells or cytotoxic function, consistent with an immune-cold micro-environment. Importantly, these patients often experienced disease progression without significant new mutations, suggesting transcription mediated resistance independent of clonal evolution.
 Conclusions Our integrative longitudinal analyses revealed that 3G-TKI therapy induced divergent molecular trajectories in CML: durable responses were defined by mutation clonal clearance, LSCs suppression and immune activation, whereas resistance was driven by persistent or expanding mutation clones (e.g., ASXL1G646Wfs*12, RUNX1and PHF6), sustained stemness, and immune-cold states. Transcriptomic remodeling provided a powerful early indicator of therapeutic failure and supports dynamic molecular monitoring to guide personalized therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/280ba844f9e6179dd23e23035a9361badf93983f" target='_blank'>
              Dynamic mutational evolution and transcriptomic remodeling on 3rd-generation TKI therapy in TKI-resistant patients with chronic myeloid leukemia
              </a>
            </td>
          <td>
            Xiao-shuai Zhang, Lu Yu, Zongru Li, Ya-Zhen Qin, Qian Jiang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Radiation therapy is a crucial component of the standard of care for newly diagnosed glioblastoma patients. However, the majority of tumors inevitably recur, most within the prior radiation field. Despite its broad clinical application, the molecular mechanisms that mediate response to radiation therapy in glioblastoma remain poorly understood. To address this knowledge gap, we systematically profiled the radiation sensitivity of four glioblastoma cell lines (T98G, SF295, LN18, DBTRG) and four patient-derived neurospheres (BT112, BT145, BT359, CPDM0095). The neurospheres were characterized at the genomic, transcriptomic and clinical levels. These models exhibited a spectrum of responses to radiation, with LD50s ranging from 1.5 to 8 Gy. To identify molecular factors that sensitize glioblastoma cells to radiation, we conducted a genome-wide CRISPR inactivation screen in T98G cells treated with or without a sub-lethal dose of radiation. Cells were transduced with the Inzolia multiplex CRISPR/Cas12a knockout library, targeting 19,687 single genes, 4,435 paralog pairs, 376 paralog triples, and 100 paralog quads. Cells were then divided into a control condition and an irradiated condition (3 Gy). After 18 days in culture, genomic DNA was harvested, and the relative abundance of each guide RNA array was compared between the two conditions. Targets preferentially depleted in the irradiated condition included known radiation sensitizers such as ATM and PRKDC, validating the screen’s performance. Gene set enrichment analysis (KEGG) of the top 200 hits revealed enrichment of DNA repair pathways, including homologous recombination and non-homologous end joining, as well as pathways related to the spliceosome, cell cycle, endocytosis, ubiquitin-mediated proteolysis, and pyrimidine metabolism. Notably, three components of the translesion synthesis pathway (REV1, REV3L, MAD2L2), not previously implicated in radiation response, also emerged as top candidates. A mini-pool library of the top 300 hits was subsequently generated and is being used for validation in additional glioblastoma cell lines and neurospheres. These findings provide a foundation for uncovering novel mechanisms of radiation response and nominate new targets for potential radiosensitization strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15dea4ab418adfb3586fa12497a41869a83e604c" target='_blank'>
              DNAR-13. Uncovering novel mediators of radiation sensitivity in glioblastoma using functional genomic screening
              </a>
            </td>
          <td>
            E. Aquilanti, Vincent Bozinov, Matthew L. Meyerson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The treatment of cancer has taken a remarkable new direction with the advent of innovative scientific and technological methods. Immunotherapy has revolutionized oncology by harnessing the body's immune system to identify and destroy cancerous cells, leading to long-term remissions in many cancer types. Precision medicine enhances treatment effectiveness by tailoring therapeutic interventions to individual patients based on genetic, environmental, and lifestyle factors, thereby reducing both efficacy and adverse effects. The development of liquid biopsy methods enables non-invasive, real-time cancer detection and monitoring through blood samples (circulating tumor DNA and other biomarkers), allowing for treatment adjustments based on early cancer diagnosis. Recent advancements in CRISPR-Cas9 gene editing have opened new avenues for correcting oncogenic mutations and developing gene-based treatments. The use of artificial intelligence (AI) in cancer diagnostics, prognostics, and drug discovery is becoming increasingly common, enhancing prediction accuracy and optimizing predictive treatment planning. Combination therapies have shown synergistic potential to boost therapeutic effects and overcome drug resistance mechanisms. Meanwhile, cancer metabolism studies aim to exploit the metabolic changes in cancerous cells for therapeutic benefit. Epigenetic therapeutics seek to reverse abnormal genomic developmental control by restoring normal cellular epigenetic patterns and functions. Additionally, research on microbiomes is uncovering the crucial role of gut and tissue microbiota in cancer initiation, progression, and therapy response, suggesting potential new prevention and treatment strategies. These interdisciplinary innovations represent a paradigm shift towards more effective, personalized, and sustainable cancer care, offering new hope for improved patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52ad1f09c9728280095d0d63e09d01947a45160" target='_blank'>
              A Review: Recent Advancements in treatment of Cancer Research
              </a>
            </td>
          <td>
            Rahul Kumar, Khalid Raza, Kamna Gupta, Komal Gupta, Lukman Ansari, Sneha Kumari, Md. Danish Shahid Raja, Manjar Ansari, Faizal Nawaz, Raja, Articles Info, IJPHI26102810292025
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Pharmaceutical and Healthcare Innovation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Whilst the broad clonal architecture of naturally progressing chronic lymphocytic leukemia (CLL) has been described, a comprehensive picture of how chemotherapy and targeted agents reshape that landscape is lacking. Here we integrate clone-specific growth kinetics with mutational profiles, transcriptomic subtypes, and CpG-methylation-based epigenetic classes to capture the multidimensional evolutionary responses of CLL under native conditions and during treatment.



 We previously reported a systematic whole-exome sequencing (WES) analysis of 417 leukemia–germline pairs from 169 treatment-naïve patients ≥ 65 y (Karisani et al., Blood 2024). The treatment arm comprised 83 patients who received ibrutinib and 39 who received chlorambucil in the RESONATE-2 trial (NCT01722487). Forty-seven age-matched “watch-and-wait” patients from the CLL Research Consortium served as untreated controls. Peripheral-blood samples were collected at baseline and again ~300 d and ~600 d after therapy initiation (or diagnosis in the watch-and-wait cohort), enabling longitudinal clonal tracking.
 WES data were processed with PhylogicNDT to reconstruct clonal architectures, define phylogenies, and identify significant clonal shifts.  MutSig tools identified mutations enriched in expanding or contracting clones. Pretreatment RNA-sequencing profiles were grouped into transcriptional subtypes by consensus clustering based on the Louvain algorithm. Differentially methylated regions plus epitypes were called from reduced representation bisulfite sequencing data.



 Across the 169 patients with WES, we resolved 579 subclonal clusters. Significant clonal shifts (defined as a distribution shift of cancer cell fraction in a subclone of >95% between two timepoints) were uncommon during watch-and-wait (36%) but markedly higher after therapy — chlorambucil  67% (q = 0.01), and ibrutinib 77% (q < 10-4). In ibrutinib-treated CLL, regressing subclones were enriched for KRAS (q<10-3) and SF3B1 mutations (q<10-3), whereas expanding subclones preferentially harboured mutations in BIRC3 (q<10-4), NOTCH1 (q <10-3),  TP53 (q=0.02), and POT1 (q=0.098). In chlorambucil-treated patients, no specific mutations were found to be enriched in expanding or regressing subclones.
 Using the recently defined CLL expression clusters (ECs) in patients treated with chemoimmunotherapy as defined by Knisbacher et al. (Nat Genet 2022), we were unable to identify any relationship between ECs and outcomes. We therefore sought to identify de novo ECs with prognostic relevance in the context of BTK-inhibition (BTKi-ECs). We identified a transcriptionally defined patient subgroup (C4, n=20) associated with inferior progression-free survival (median PFS = 63 months, log-rank q=0.05), independent of known prognostic clinical and molecular features, including epitypes, in multivariate analysis (Cox HR=3.7, 95% CI: 1.6-8.6, q=0.02). Gene-set enrichment in this cluster highlighted TNF-ɑ signalling via NF-κB (REL, TNFAIP3, NFKB2) and G2M checkpoint pathways (SMC4, EZH2). We also identified a distinct transcriptionally defined patient subgroup (C3, n=18) that was enriched for patients harboring at least one contracting subclone and no expanding subclones (q=0.005). This cluster showed a relative depletion of mutations within the chromatin modification pathway (6% vs 50%, q<0.01). Gene-set enrichment in this cluster highlighted a downregulation of genes in the TNF-ɑ signalling via NF-κB pathway (SOCS3, ICOSLG, NFKB2).



 BTK inhibition and chemotherapy drive distinct evolutionary trajectories in CLL. Multi-omics profiling reveals mutation-defined subclones with divergent sensitivity to targeted therapy and delineates two informative expression states: a high-risk NF-κB/inflammatory cluster (C4; poor PFS) and a cluster enriched with contracting subclones with depleted TNF-ɑ signalling via NF-κB (C3). Integrating clonal growth kinetics with genomic, transcriptomic, and methylation data can potentially identify patients likely to progress on BTK inhibitors and identify actionable pathways for adaptive or combination therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856e14a8f04e6b7cc7673d0e466f8ba7e3d9cc" target='_blank'>
              Systematic molecular profiling to identify determinants of response to ibrutinib
              </a>
            </td>
          <td>
            Negin Karisani, Amit Sud, Liang Li, Satyen H Gohil, Kristy L. Schlueter-Kuck, Zhihan Zhu, L. Rassenti, Tuan Tran, L. Cheung, I. Szafer-Glusman, James P. Dean, Guang Chen, Junko Tsuji, M. Palma, Robert Månsson, Neil E. Kay, Kanti Rai, Jennifer R. Brown, John Gribben, John C Byrd, D. Neuberg, C. Stewart, Ivana Bozic, H. Kretzmer, T. Kipps, G. Getz, Catherine J. Wu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) progresses through the sequential accumulation of somatic mutations, with signaling transducer mutation typically preceding the acquisition of epigenetic alterations. However, recent single-cell DNA sequencing (scDNA-seq) studies have identified cases that do not follow this canonical mutational hierarchy. The prevalence of these non-classical mutation orders, and the biological mechanisms by which mutation timing drives phenotypic heterogeneity remain unclear.
 To address this, we analyzed publicly available scDNA-seq data from >150 newly diagnosed normal karyotype AML patients to reconstruct their evolutionary trajectories (Morita et al., 2020; Miles et al., 2020). We observed that 78% of the signaling mutation events—particularly FLT3-ITD, NRAS, and PTPN11—were terminal subclonal events in branch evolution. Surprisingly, 22% of the signaling mutations serve as initiating events, following a linear evolutionary trajectory and patients harboring these initiating signaling mutations exhibited significantly higher white blood cell and blast counts. These data suggest that the timing of signaling mutation acquisition is strongly linked to the resulting clonal evolution pattern, with FLT3-ITD emerging as the most prominent driver among all signaling events.
 To investigate the pathogenic mechanisms of initiating FLT3-ITD and the characteristics of the resultant AML, we therefore performed multimodal profiling (scDNA+protein-seq, scRNA-seq and patient-derived xenograft models) on 16 FLT3-ITD positive AML cases. Eight cases had FLT3-ITD as the initiating mutation (Initiating FLT3-ITD AML, IFA), and eight as subclonal events (Subclonal FLT3-ITD AML, SFA).
 Comparing tumor cell proportions identified by scRNA-seq with the mutant clone fractions from scDNA-seq in the same patient, we confirmed that leukemic transformation preceded acquisition of FLT3-ITD in SFA. Integrated genotype-immunophenotype analysis of paired samples—adjusted for co-mutations—revealed that initiating FLT3-ITD drove an expansion of hematopoietic stem and progenitor cells, whereas subclonal FLT3-ITD did not. Bayesian modeling further showed that initiating FLT3-ITD selectively upregulated leukemia stem cell (LSC) markers CD69 and CD123, resulting in a significantly larger CD123+CD69+ population (74% vs 35%, P=0.04). Consistently, in PDX models, IFA samples exhibited higher engraftment rates (5/8 vs 3/8). ScRNA-seq further revealed expansion of multipotent progenitor (MPP) and lymphoid-primed multipotent progenitor (LMPP) populations (29% vs 13%, P=0.04) and significant enrichment of LSC transcriptional signatures in IFA compared to SFA. Unsupervised clustering identified progenitor-enriched clusters in IFA characterized by downregulation of G2M checkpoint and E2F target pathways, indicative of a quiescent, stem-like state. Unexpectedly, canonical FLT3-ITD downstream pathways showed no significant differences between IFA, SFA, and FLT3-WT AML patients. Instead, in IFA, we observed significant upregulation of Hedgehog pathway genes (GLI2, ALDH1A1), known regulators of cell cycle and LSC quiescence.
 We observed frequent emergence of chr13q (FLT3 locus) uniparental disomy (UPD) in Initiating FLT3-ITD AML patients (15/23). Notably, we found the acquisition or expansion of UPD clones in IFA PDX models. Inferred from scRNA-seq, these UPD clones displayed increased proliferative capacity without losing LSC traits, potentially via RB1 inactivation. Drug sensitivity testing in PDX models revealed that IFA samples responded to chemotherapy and gilteritinib with distinct patterns, with gilteritinib selectively reducing leukemic blasts and inducing differentiation. However, single-cell genotyping of residual cells showed a marked rise in UPD clone post-gilteritinib. Longitudinal follow-up of these models confirmed that relapse was driven by the expansion of UPD clones. These data suggested chr13q is a required evolutionary event in the development and drug resistant progression of IFA.
 Our study demonstrates that initiating FLT3-ITD mutations reprograms leukemic cells to a quiescent, stem-like state, enforcing a linear evolutionary pattern, compared to subclonal FLT3-ITD. Clonal dominance is ensured by non-canonical FLT3 signaling and late-emerging chr13q UPD. These findings underscore the critical role of clonal timing in shaping AML biology and highlight new therapeutic vulnerabilities in initiating FLT3-ITD-driven AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c74975f6b634c7f7ca88b37dc7ce75122c37169" target='_blank'>
              Clonal timing of FLT3-ITD dictates evolutionary trajectory and phenotypic heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Xu-Dong He, Mengfang Xia, Ruoyang Shao, Qifa Liu, Qian-Fei Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) poses a significant global health challenge, yet immune checkpoint blockade (ICB) therapy benefits only a small subset of patients with mismatch repair-deficient (dMMR) or microsatellite instability-high (MSI-H) tumours. Through analyses of public single-cell and spatial transcriptomic datasets, primary mouse cell sorting and adoptive transfer experiments, flow cytometry, multiplex immunofluorescence, immunohistochemistry, and coimmunoprecipitation, we revealed that sentrin-specific protease 7 (SENP7) promotes regulatory B-cell (Breg) differentiation and inhibits senescence by activating the expression of the NAD-dependent protein deacetylase sirtuin-1 (SIRT1) via deSUMOylation, thereby enhancing the expression of genes such as interleukin-10 (IL-10). Notably, targeting SENP7 in B cells improved the antitumour efficacy of anti-PD-1 therapy. These findings suggest that inhibiting SENP7 may offer a promising strategy to sensitize immunologically "cold" tumours to immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddf8df97840e103aaf23b0df7a5e6b5ffd3a725b" target='_blank'>
              A SENP7-SIRT1-IL-10 Axis Driven by DeSUMOylation Promotes Breg Differentiation and Immune Evasion in Colorectal Cancer.
              </a>
            </td>
          <td>
            Yuhan Liao, Xinghua Zhuo, Yuan Huang, Huimeng Xu, Zhe Hao, Lanhui Huang, Haoxuan Zheng, Jun Zhou
          </td>
          <td>2026-01-01</td>
          <td>International journal of biological sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="


 Background:
 Chronic lymphocytic leukemia (CLL) is a heterogeneous B-cell malignancy characterized by clonal expansion and immune dysfunction. Although peripheral blood (PB) is frequently used for genomic and transcriptomic profiling, it may not fully capture the biological activity of the disease. The lymph node (LN) microenvironment plays a central role in promoting malignant cell survival, proliferation, and immune evasion. Genomic alterations such as TP53, NOTCH1, and SF3B1 mutations are associated with adverse outcomes, but it remains unclear how these mutations interact with the local immune contexts across compartments. We investigated how clonal architecture and immune composition differ between PB and LN, and how these features relate to evolutionary dynamics.
 Methods:
 We performed whole exome sequencing (WES) and bulk RNA sequencing on paired LN biopsies and CD19⁺-purified PB samples from 26 patients with CLL. We selected samples from patients with known CLL driver mutations. Somatic variants, copy number alterations, and clonal architecture were inferred using SuperFreq, and cancer cell fraction (CCF) was used to define clonal distribution. Variants were filtered to include only those with >3 alternate reads, >20 total reads, a variant allele frequency (VAF) >0.02, and classified as non-synonymous mutations. CCF was considered equivalent to VAF if >0.6, otherwise CCF was calculated as VAF * 2. CCF was adjusted by dividing CCF by the ancestral or founding clone CCF as a proxy for tumor cell fraction. Clones were considered compartmentalized if the absolute CCF difference between LN and PB exceeded 0.25. Exome alteration was calculated by summing the length of exonic targets falling in copy-number segments with an absolute log2 copy-ratio >0.2 and dividing by the total captured exome length. LN immune composition was estimated from bulk RNA-seq data using CIBERSORTx.
 Results:
 Of the 26 patients, 12 were treatment-naive and 18 were IGHV unmutated. In the PB, mutational frequency was as follows: 46.1% NOTCH1, 42.3% TP53, and 26.9% SF3B1. Oncoplot analysis revealed the presence of multi-hit alterations and a predominance of missense and frameshift variants.
 Analysis of paired PB and LN samples revealed notable heterogeneity in clonal architecture. Using SuperFreq, we identified between 2-6 clones per patient (median: 4). The median number of subclones did not differ between LN and PB (4 vs. 4), treatment naive vs. relapsed/refractory status (4 vs. 4) or between IGHV unmutated vs mutated (4.5 vs. 4). We evaluated genomic instability and found significantly higher exome alteration in patients who were IGHV unmutated (P = .03). Clonal compartmentalization (CCF difference of |.25|) was observed in 18 of 26 (69%) patients, with 5 patients showing clonal evolution in the LN (28%), 6 in the PB (33%), and 7 with distinct clones evolving in both LN and PB (39%).
 SF3B1 mutations were common in clones with CCF trending higher in the LN compared to PB (median ΔCCF 0.16, P = .11). Interestingly, CCF of NOTCH1 mutations was balanced across compartments (median ΔCCF = -0.02; P = .36), and CCFof TP53 mutations was numerically slightly lower in LN than PB (median ΔCCF –0.05; P = .97).
 LN samples with stable clonal populations had higher proportions of CD4⁺ naïve T cells compared to those with evolving clones (P = .02), but no other significant changes in T cell populations were noted. Patients with NOTCH1 or SF3B1 mutations had reduced dendritic cell abundance in LN (P = .04 and P = .06, respectively) compared to those with TP53 mutations or no known driver mutations.
 Conclusions:
 This study demonstrates distinct clonal and immune features between PB and LN compartments in CLL, a heterogeneous clonal architecture across compartments. Differences in clonal composition across LN sites within individual patients suggest that localized immune contexts play a role in clonal selection. Evolving LN clones were more common in LNs with relative decreases in CD4⁺ T cell subsets, and the presence of common driver mutations was associated with reductions in dendritic cell content. Together these observations suggest an interplay of supportive vs restrictive roles of the LN TME in CLL progression. These findings highlight the added biological insight provided by LN sampling and suggest that PB alone may not fully capture the clonal heterogeneity of CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121dea9d74142004efb29c8ec3326a61ca23f8a3" target='_blank'>
              Role of the immune microenvironment in the clonal evolution of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Victoria Rogness, Bharath Dileep Kumar, Chingiz Underbayev, Pia Nierman, Clare Sun, Christine Gruessner, A. Wiestner
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="


 Epigenetic dysregulation, particularly changes in DNA methylation, are major contributors to the onset and progression of Large Granular Lymphocytic Leukemia (LGLL). We previously identified hypermethylation of tumor suppressor genes as a factor in LGLL pathogenesis; however, the full landscape of epigenetic dysregulation and causality versus correlation to transcriptomic dysregulation remains unknown.
 In this study, we performed reduced representation bisulfite sequencing to profile the methylome, alongside RNA sequencing, on flow cytometry sorted CD3⁺CD8⁺CD5⁻/dim peripheral blood cells isolated from treatment-naïve LGLL patients and healthy donors. The analysis of differentially methylated regions (DMR) revealed increased hypermethylation in patients relative to healthy donors, spanning exons, introns, promoters, and transcription start sites. While promoter methylation usually suppresses gene expression, some genes showed increased transcription despite being hypermethylated. A notable example is IL-15, whose expression was upregulated following promoter methylation, due to silencing of a repressor of IL-15. To investigate the role of methylation in this atypical gene activation, we treated MOTN1 cells (human LGLL cell line) with the demethylating agent 5-aza-2′-deoxycytidine (5-aza). This intervention reversed the methylation status, resulting in reduced expression of IL-15, thereby confirming that its transcriptional upregulation was driven by promoter methylation (p<0.001). Further supporting the role of IL-15 in disease pathogenesis, transplanting MOTN1 cells revealed a markedly increased LGLL burden and reduced overall survival in NSG-Tg(Hu-IL15) transgenic mice compared to NSG control mice (n=5 each, p<0.05). To assess whether treatment 5-aza and reversal of the associated methylation signature influenced LGLL cell viability and proliferation, MOTN1 cells labeled with CellTrace dye were incubated with 5-aza for 48 hours in an in vitro assay. Cell viability was measured using Zombie Aqua staining, while proliferation via dilution of the CellTrace dye. The results demonstrated a dose-dependent increase in LGLL cell death and a corresponding reduction in LGLL proliferation. Parallel experiments conducted on primary patient samples (N=5) revealed similar alterations in viability and proliferative capacity following 5-aza treatment in vitro. In vivo, NSG huIL-15 mice engrafted with LGLL cells and treated with 5-aza (3 mg/kg) showed significant LGLL reduction via IVIS imaging and improved overall survival (n=5 each, p<0.001), supporting the potential of epigenetic therapy in LGLL therapy.
 Given the strong preclinical impact of 5-aza on T-LGLL cells, we initiated a multi-center Phase I/II trial using oral 5-aza (CC-486) in relapsed/refractory T-LGLL patients (NCT05141682). Here we report results from the Phase 1 dose-escalation trial, where patients received 3 escalating doses of oral 5-aza (100, 200, 300 mg) on days 1-14 of a 28 day cycle. Dose-limiting toxicities (DLT) were evaluated during the first two cycles to establish the recommended phase II dosage (RP2D) using a Baysian Optimal Interval Design (BOIN). Secondary endpoints included overall partial and complete response rates per ECOG5998 criteria. Nine patients were enrolled and evaluated for DLT and response. The median age was 60 (range: 32-80), and 7 (78%) were female. 3 (33%) patients were treated for transfusion/symptomatic anemia, 6 (67%) patients were treated for neutropenia (ANC<500 or ANC <1500 with frequent infections) and the median number of prior therapies was 4 (range: 2-5). Three patients each received 100, 200, or 300mg doses. No dose-limiting toxicities were observed and the RP2D dose was determined to be 300mg. Per ECOG 5998, one patient per dose level attained a partial response (ORR 33%), with two patients attaining resolution of transfusion-dependent anemia, and one patient achieving resolution of long-standing severe neutropenia. All three patients remain on therapy. LGLL counts showed a marked decline with 5-aza in patients, underscoring the role of hypermethylation in T-LGLL.
 In conclusion, methylation is a therapeutic target in LGLL. In vitro, in vivo, and Phase I trial data show that epigenomic targeting with 5-aza yields clinical responses across doses without DLTs. The Phase II component of the oral 5-aza clinical trial is currently ongoing, reinforcing the critical role of the methylome in T-LGLL pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/569d4467ac93ab464c0a30aa3e60ad9de97e539e" target='_blank'>
              Targeting the methylome in T-LGL leukemia: Pre-clinical analysis and Results of a Phase I trial with oral 5-azacytidine
              </a>
            </td>
          <td>
            Jonathan E Brammer, Zachary Braunstein, Patrick Stevens, Zachary Hutchins, Giulia Calabretto, Soprina Guarneri, Eric McLaughlin, Amy E Boles, John Reneau, W. Hanel, Enrica Marchi, Maciej Pietrzak, Anjali Mishra
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a lethal cancer driven by a fusion between tumor suppressor Capicua (CIC) and pioneer transcription factor double homeobox 4 (DUX4). To develop an immunocompetent pre-clinical model of CDS, we previously generated three genetically engineered mouse models (GEMMs) of CDS with CIC::DUX4 regulated by loxP-STOP-loxP cassettes. However, all three models developed spontaneous tumors without Cre recombinase. Here, we established an innovative GEMM of CDS (dFLEx CDS) that employs a dual recombinase (Cre + FLPE) FLEx-switch design to activate CIC::DUX4 expression and initiate sarcomagenesis in a spatially and temporally-controlled manner. Because CIC::DUX4 drives sarcoma development by activating a distinct oncogenic transcriptional program, we performed a drug screen on human-derived CDS cell lines using a library of compounds that modulate transcriptional regulation. This screen identified Minnelide, an inhibitor of RNA polymerase II-mediated transcription, as a selective inhibitor of CDS. Mechanistically, Minnelide acted through xeroderma pigmentosum type B to alter phosphorylation of RPB1, the largest subunit of RNA polymerase II. Subsequently, RPB1 underwent degradation leading to apoptosis of CDS cells. Minnelide demonstrated in vivo efficacy in autochthonous dFLEx CDS GEMMs and in human CDS xenografts. As Minnelide has already been demonstrated to be safe in clinical trials with activity for adult cancers, these findings nominate Minnelide as a novel therapeutic option to test in CDS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170765ba25b86abafc2b59067c8d9ac6a11c944c" target='_blank'>
              Efficacy of Minnelide in a Next-Generation Dual-Recombinase Regulated Genetically Engineered Mouse Model of CIC::DUX4 Sarcoma
              </a>
            </td>
          <td>
            MaKenna R. Browne, Axel V. Silver, Risha Banerjee, Benigno Aquino, K. Oristian, Jonathon E. Himes, P. Hendrickson, D. Kirsch
          </td>
          <td>2025-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Age-related clonal hematopoiesis (CH) and clonal cytopenia of undetermined significance (CCUS) represent pre-leukemic conditions associated with predisposition to hematological malignancies, particularly myeloid neoplasms. Frequently associated with mutations in epigenetic regulators, such as DNMT3A, TET2, and ASXL1, deregulated epigenomes are implicated in the aberrant gene expression involved in the pathogenesis of CH and CCUS, whose molecular mechanism, however, has not been investigated. The major difficulty lies in the technical limitation to simultaneously analyze gene mutation, transcriptome, and epigenetics in a small fraction of mutant cells in comparison with wild-type cells within the same patients, particularly in those cases where mutant cell fractions are very small.
 Aim/Methods In this study, we developed a novel platform that enabled simultaneous measurements of gene mutations and chromatin accessibility at the single-cell level based on the modified Mission Bio Tapestri system, which was applied to the analysis of 11 samples with TET2-mutated (TET2-MUT) CH and CCUS to investigate the impact of TET2 mutations on chromatin accessibility. In a subset of the samples (n = 3), we performed 10x Chromium single-cell ATAC+RNA-seq (Multiome) experiments and then predicted single-cell TET2 genotypes by employing ATAC peak accessibility-based logistic regression models trained on the paired Tapestri multimodal data, enabling the analysis of effects of TET2 mutation on gene regulation and transcriptional outputs at single-cell resolution.
 Results First, our platform was validated using a mixed cell line sample (HL-60 and SKM-1). ATAC data alone clearly separated the two cell lines, and genotyping accuracy for cell line-discriminative SNPs reached 94-99%, confirming robust integration of ATAC-seq and genotyping at the single-cell level.
 Next, to investigate the consequences of TET2 mutation on hematopoiesis and chromatin accessibility, we applied the platform to a total of 37,190 cells from TET2-MUT CH/CCUS patients, achieving successful genotyping in 83% (74–92%) of the cells. TET2-MUT cells exhibited a differentiation bias toward myeloid and plasmacytoid dendritic cell (pDC) lineages, with reduced mature lymphoid output. In line with this skewed differentiation, TET2-MUT hematopoietic stem/progenitor cells (HSPCs) showed an increased accessibility to the motifs of transcription factors (TFs) associated with myeloid/pDC differentiation (e.g., CEBPE, RUNX1, and IKZF1), compared to wild-type (TET2-WT) HSPCs, whereas TET2-MUT B cells had a reduced accessibility to the motifs of B cell development-associated TFs (e.g., BACH2, IRF4, and IRF8), compared with TET2-WT counterparts. In erythroblasts, TET2-MUT cells exhibited a decreased accessibility to erythroid maturation-related genes and an increased accessibility to genes involved in cell proliferation, including E2F and MYC targets, compared with TET2-WT cells, suggesting dysregulated maturation of TET2-MUT erythroblasts. Collectively, these results revealed TET2 mutation-driven alterations in chromatin accessibility affecting lineage maturation.
 Next, to investigate combined effects of TET2 mutation on TF-mediated gene regulation and transcriptional programs, we analyzed 10x Multiome data with predicted single-cell TET2 genotypes. We first analyzed TET2-MUT cell fractions within monocyte subsets and observed a depletion of mutants in the CD16(+) subset, suggesting impaired maturation of TET2-MUT cells. To uncover the underlying mechanisms, we then compared TF activity between genotypes, using SCENIC+, which integrates paired single-cell ATAC-seq and RNA-seq data to infer TF activity based on both accessibility at TF-binding enhancers and expression of their target genes. For both accessibility- and expression-based TF activities, TET2-MUT monocytes exhibited up-regulation of cell cycle-regulating TFs (e.g., TFDP1, TFDP2, and MYBL2) and down-regulation of key monocyte maturation-related TFs (e.g., CEBPB, CEBPD, and SPI1). These results suggest that TET2 mutation may impair monocyte development by epigenetically and transcriptionally suppressing maturation-related programs and sustaining proliferative states.
 ConclusionOur innovative single-cell multi-omics approaches revealed that TET2 mutations rewire chromatin accessibility and TF regulatory networks, leading to dysregulated lineage maturation and aberrant cell cycle control in CH/CCUS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8043e74c1da9433c5daf135387ded1955d981b6" target='_blank'>
              Single-cell multiomic profiling of gene mutation, chromatin accessibility, and gene expression in TET2-mutant clonal hematopoiesis
              </a>
            </td>
          <td>
            Masanori Motomura, Y. Ochi, Y. Nannya, N. Kakiuchi, N. Sezaki, S. Kasahara, Tomoe Nakagawa, Lanying Zhao, Shuichi Matsuda, M. Nakagawa, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa756c05989530f16b2f4c4778c0bfc833ec875f" target='_blank'>
              Deep Learning links TP53 genotype to expression-defined transcriptional program in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            M. Tsesmelis, Rebecca Andersson, Alba Ferrer Perez, Sara Montserrat Vazquez, M. Carolina Florian, M. Mulaw
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer drug resistance, whether intrinsic or acquired, underlies most relapses and treatment failures. Reliable preclinical models are crucial to define resistance mechanisms and test counterstrategies. This overview article concisely compares three model classes: (1) clinical, patient‑derived xenografts that retain tumor heterogeneity and can mirror patient resistance; (2) induced‑resistance models produced by prolonged drug selection in vivo or in vitro that recapitulate tumor evolution under therapy; and (3) engineered isogenic cell lines that isolate specific resistance drivers. We summarize some key resistance mechanisms revealed and potential therapeutic approaches informed by these models, including rational combinations, mutation‑targeted inhibitors/degraders, efflux/epigenetic modulators, and immune-related combinations. Each model has trade‑offs, but integrating them accelerates mechanistic insight and translational drug development. This overview guides selection of preclinical models and design of strategies to overcome cancer drug resistance. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/329bb2e5a0a76225e84bcf3ca5e48cf0783cbcff" target='_blank'>
              Overview of Preclinical Drug-Resistant Tumor Models and Their Application in Drug Discovery.
              </a>
            </td>
          <td>
            Bin Li, Ting Ni, Dong Wang, Gaoxiang Liu, Wenting Shi, Zhixiang Zhang, Qingyang Gu, Jian Xiang
          </td>
          <td>2025-12-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 While venetoclax (VEN) combined with hypomethylating agents (HMA) has greatly advanced outcomes for acute myeloid leukemia (AML), resistance remains a major clinical challenge. Prior studies revealed several resistance mechanisms, including monocytic differentiation and RAS/TP53 mutations, but the full spectrum of mechanisms is still elusive.
 To assess the cellular mechanisms underlying VEN resistance, we performed an integrative analysis using bulk RNA sequencing (RNA-seq) and single-cell multi-omics in samples collected from the AML patients treated with 10-day decitabine and venetoclax (DAC10+VEN) trial (NCT03404193). First, we performed RNA-seq on 81 baseline and 10 relapse samples. K-means clustering of baseline samples identified a cluster enriched with non-responders, marked by a higher frequency of KRAS mutations (31%, p=0.012) and the absence of NPM1 and IDH1/2 mutations. Using LASSO logistic regression, we developed a 25-gene VEN-resistance (“VEN-R”) score comprising genes associated with MAPK, NF-κB, and apoptosis pathways. The VEN-R score significantly distinguished treatment response to DAC10+VEN (p<0.0001) and was also elevated in relapse samples (p=0.00018), supporting its association with resistance. We further validated VEN-R in two external, ex vivo datasets, finding significantly higher VEN-R scores in VEN-resistant samples compared to VEN-sensitive samples (BeatAML: p<0.0001 and Hashimoto et al. (Nature Cancer 2021): p=0.0002).
 To study the association between cell composition and DAC10+VEN response, we performed computational deconvolution of the RNA-seq data using CIBERSORTx. Baseline samples had higher fractions of erythroid (p=0.026) and CD8 effector (p=0.034) cells in nonresponders, while hematopoietic stem/progenitor (p=0.0006) and late erythroid (p=0.032) cells were more abundant in responders. Furthermore, compared with baseline samples, relapse samples had decreased hematopoietic stem (p=0.018), lymphoid primed multipotent progenitor (p=0.028), and naïve B (p=0.013) cells, and increased early erythroid (p=0.048), late erythroid (p=0.020), and granulocyte monocyte progenitor (p=0.036) cells.
 To assess the relationship between cell composition and VEN resistance at the cellular level, we performed single-cell RNA sequencing (scRNA-seq) on 22 samples (117,540 cells; 6 baseline, 16 relapse) treated with DAC10+VEN. scRNA-seq results corroborated the enrichment of mature cells at relapse; specifically, we identified the enrichment of erythroid, monocytic, and cDC (“EMD”) populations and the loss of progenitor cells. BCL2-expressing cells were depleted at relapse, whereas cells expressing MCL1, BCL2L1, or BCL2A1 were enriched in monocytes, erythroid cells, and cDCs, respectively. The EMD cells also had one of the highest expression levels of the VEN-R score at the single-cell level.
 To examine whether phenotypic remodeling is driven by genetic changes, we then performed single-cell DNA-antibody sequencing (DAb-seq) on 75 longitudinal samples, including six baseline-relapse pairs. DAb-seq analysis substantiated the findings from scRNA-seq that relapse is marked by the loss of stem/progenitor cells and the enrichment of erythroid/monocytic cells. Phenotypic remodeling occurred in five of the six pairs. One monocytic shift was accompanied by the de novo acquisition of FLT3 and KRAS mutations, one erythroid shift by expansion of an NRAS-mutant subclone, and a second erythroid shift by loss of a GATA2-mutant subclone. In the remaining two cases, remodeling occurred without any detectable genetic alterations. Integrating scRNA-seq and DAb-seq from 22 relapse samples revealed four phenotypic patterns: erythroid-predominant (6 cases), monocytic-predominant (4), mixed (erythroid±monocytic±cDC; 8), and inconclusive (4).
 Finally, we examined whether baseline phenotypic diversity predicts relapse after DAC10+VEN. In the DAb-seq cohort, a higher Shannon diversity index was associated with a higher risk of not only relapse (p=0.043) but also poor response (p=0.057).
 In summary, DAC10+VEN resistance is frequently observed with the enrichment of mature AML cells. To our knowledge, this is the first study to implicate erythroid cells and cDCs as major contributors to VEN resistance in patient samples, extending prior ex vivo studies (Kuusanmäki et al., Blood 2023) to a clinical setting. Monitoring these phenotypes and combining VEN with agents that target both immature and mature clones may improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9212f63e36e106f299197c47a52ccce0eb0f7523" target='_blank'>
              Phenotypic heterogeneity and remodeling drive resistance to venetoclax treatment in acute myeloid leukemia
              </a>
            </td>
          <td>
            Y. J. Kim, Ken Furudate, C. Dinardo, A. Maiti, M. Swaminathan, Zongrui Li, Eva Ordoñez, O. Nwogbo, Minggao Liang, Yoriko Saito, Akiko Kaneko, Kiyomi Morita, Jairo Matthews, Hongli Tang, Erika J. Thompson, Tomoyuki Tanaka, H. Uryu, N. Daver, N. Pemmaraju, T. Kadia, F. Ravandi, N. Short, M. Ohanian, G. Borthakur, Nitin Jain, G. Issa, L. Masarova, S. Kornblau, W. Wierda, S. Loghavi, Sa A. Wang, G. Garcia-Manero, Hagop Kantarjian, E. Jabbour, C. Bueso-Ramos, Shanshan Pei, C. Jordan, A. Futreal, Fumihiko Ishikawa, M. Konopleva, Koichi Takahashi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>141</td>
        </tr>

        <tr id="Cancer treatment is a core challenge in clinical oncology. Despite the continuous advancement of chemotherapy, targeted therapy, and immunotherapy, the emergence of tumor drug resistance remains a key factor leading to treatment failure and worsening patient prognosis. Clinical data indicate that approximately 90% of treatment failures in patients with advanced cancer are directly related to drug resistance. Therefore, systematically analyzing the mechanisms of drug resistance and developing targeted strategies to address them are crucial for improving the efficacy of cancer treatment. This article systematically expounds on the main mechanisms of tumor drug resistance from the perspectives of tumor cell biological characteristics, tumor microenvironment regulation, and epigenetic modifications. These include reduced intracellular drug accumulation, mutations or aberrant expression of drug targets, inactivation of apoptotic pathways, cancer stem cell (CSC)-mediated resistance, and the regulatory role of immunosuppressive cells and cytokines in the tumor microenvironment. Furthermore, based on clinical practice, this article summarizes currently used strategies, such as drug development targeting resistance-associated molecules, chemotherapy regimen optimization and combination therapy, synergistic application of immunotherapy and conventional therapies, personalized treatment plan development, and the clinical translation of resistance monitoring technologies. Research has shown that the development of tumor drug resistance is the result of multiple factors and pathways, and a single strategy is unlikely to effectively reverse drug resistance. However, multi-target, multi-modal combination therapy based on resistance mechanisms, combined with dynamic resistance monitoring technology, can significantly improve treatment outcomes for patients with drug-resistant tumors. This article aims to provide a theoretical basis for clinicians to formulate tumor treatment plans and to guide future research related to drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f859514bae2da8d810219f47530fe8e0b51bb170" target='_blank'>
              A Review of the Mechanisms of Tumor Drug Resistance and Clinical Strategies
              </a>
            </td>
          <td>
            Wenqing Tie
          </td>
          <td>2025-11-21</td>
          <td>International Journal of Public Health and Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) with tandem duplications in the upstream binding transcription factor gene (UBTF-TD) has recently been described as a new AML entity in younger adults. The UBTF gene encodes the UBF protein, which facilitates transcription of 18S, 5.8S and 28S ribosomal RNAs (rRNAs) by forming a pre-initiation complex with RNA polymerase I and SL1, a selectivity factor required for rRNA transcription. In AML patients with UBTF-TD, these alterations frequently co-occur with WT1 mutations, internal tandem duplications of the FLT3 gene, and trisomy 8, but are mutually exclusive with other AML subtype-defining alterations such as NPM1 mutations and recurrent translocations. UBTF-TDs appear to be early clonal events associated with poor response to conventional chemotherapy, high variant allele frequencies, and stability throughout disease progression.
 As an emerging AML entity, the molecular mechanisms by which the UBTF-TD promotes leukemogenesis remain incompletely understood. HOXA and HOXB gene overexpression has been identified as a predictive biomarker of sensitivity to BCL-2 inhibition, while UBTF-TD AMLs have recently been shown to respond to menin inhibition. In addition, as the UBTF gene encodes a nucleolar protein and has a well-established key role in rRNA biogenesis and nucleoli formation, RNA polymerase I inhibition might be beneficial in treating AML with UBTF-TD. CX-5461 is an anticancer drug that selectively inhibits RNA polymerase I-mediated transcription by disrupting rRNA synthesis initiation, and has received Fast Track Designation from the FDA due to its promising therapeutic potential.
 In this study, we established xenograft models by injecting primary AML patient samples with UBTF-TD or UBTF wild-type into immunodeficient NSG mice. Following engraftment, mice were treated with vehicle, CX-5461, venetoclax plus azacitidine (VEN-AZA), or the combination of CX-5461 and VEN-AZA. The tumor burden was significantly reduced when CX-5461 was added to VEN-AZA. Flow cytometry revealed that CX-5461, alone or in combination with VEN-AZA, induced a near-complete shift toward a primitive CD34+ immunophenotype specifically in UBTF-TD PDX models, suggesting enhanced cellular plasticity. Single-cell sequencing (CITE-seq) of primary AML samples, including pre-treatment and relapse samples with UBTF-TD or UBTF wild-type controls, identified consistent transcriptional and surface marker dysregulation in UBTF-TD samples. This included downregulation of key cell cycle regulators (e.g., E2F targets and G2M checkpoint genes), consistent with reduced proliferation and a quiescent-like phenotype, alongside reduced TNF-α signaling. Analysis of paired diagnosis-relapse samples showed an increased stem-like cell population at relapse, mirroring the treatment-induced shift toward a primitive, stem-like state observed in PDX models. Further analyses to investigate the functional consequences of this pronounced primitive shift upon CX-5461 treatment are currently underway.
 In conclusion, our findings demonstrate that VEN-AZA in combination with CX-5461 reduces tumor burden in preclinical models and induces a pronounced shift toward a primitive, quiescent, stem-like phenotype in UBTF-TD AML, reflecting significant cellular plasticity. These data support combined targeting of BCL-2 and RNA polymerase I as a promising therapeutic strategy for AML patients harboring UBTF-TD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e603acb46cf19f2c9f2bb06e27afcbf13614df" target='_blank'>
              Acute myeloid leukemias with UBTF tandem duplications respond to combined BCL-2 and RNA polymerase I inhibition with increased stemness and a quiescent-like state
              </a>
            </td>
          <td>
            Patrick Stelmach, Nesrine Aroua, Dawn S Lin, E. Donato, Ann-Kathrin Eisfeld, C. Röllig, Christian Thiede, Andreas Trumpp
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Background - Methods: Hundreds of cell lines have been examined by genome-sale CRISPR knockout (KO) studies to identify functional drivers of tumors, but only few lines have been systematically examined with CRISPR activation (CRISPRa). We reasoned that CRISPRa studies can identify genes whose activation may suppress tumor cell fitness and serve as negative regulators of tumor cell survival/proliferation (NRSPs). To this end, we performed genome-scale CRISPRa studies (dCas9-P65-HSF transcriptional activation system; Calabrese sgRNA library, typically 6 sgRNAs/gene) in 10 genotypically diverse multiple myeloma (MM) cell lines at multiple timepoints and compared results with similar publicly available or in-house CRISPRa studies in other cancers (n=17; including lymphoma, leukemia, NSCLC, melanoma, ovarian, prostate cancer; with up to 3 lines per tumor type).
 Results: Based on multiple quantitative metrics, hundreds of genes were recurrently identified as NRSPs (e.g. 500 NRSPs in 5 or more lines). Integrated analyses of our CRISPRa data with molecular annotation for the respective MM lines did not identify any obvious bias in the CRISPRa-based detection of NRSP that relates to e.g., DNA copy number status; chromatin accessibility; or baseline levels of transcript expression. Top recurrent and pronounced NRSPs included several known proapoptotic genes (e.g., death receptors; caspases; BH3 only Bcl-2 family members) or other tumor suppressive molecules (e.g. CDK inhibitors), providing reassuring positive controls. Moreover, many previously unrecognized NRSPs were identified including transcription factors (TFs)/cofactors; chromatin remodeling genes and epigenetic regulators; signaling molecules regulating survival/proliferation; RNA binding proteins; nucleic acid-sensing and antiviral restriction factors; metabolic regulators; E3 ligases and other proteostatic regulators; DNA repair genes; solute carrier transporters, among others. No correlation is currently observed between the pattern of NRSPs for individual MM cell lines and their respective molecular subtype (e.g., t(4;14), t(14;16), t(11;14), etc). Some clusters of NRSPs are shared between MM and non-MM lines; and others are more prominent in MM, e.g. certain plasma cell-related TFs or endoplasmic reticulum regulators. We validated the functional relevance of several NRSPs with individual sgRNAs for CRISPRa; using focused sgRNA libraries for CRISPRa in MM cells in vitro and in NSG mice in vivo. We also performed single cell RNA-seq of MM cells transduced with pools of sgRNAs (CROP-seq) or bulk RNA-seq of MM cells transduced with individual sgRNAs for select NRSPs. Many NRSPs have undetectable/low transcript levels, without recurrent deletions/mutations, in MM cell lines or patient-derived MM cells but are expressed in healthy and malignant cells from different non-hematopoietic tissues. Thus, the NRSP effect of these genes in MM cells may involve lineage-inappropriate activation of molecular programs that are critical for, or at least tolerated in, other lineages but suppress MM cell fitness. Among the top 200 NRSPs with recurrent expression in MM cells, analyses of MMRF CoMMpass data revealed downregulation of ≥1 NRSPs in ~50% of paired MM samples after relapse vs. before treatment, suggesting that suppression of such NRSP genes can contribute, alone or in concert, to enhanced MM cell fitness in clinical relapses. Some NRSPs defined by CRISPRa are potent dependencies by CRISPR KO, including the MM-preferential, lineage-defining, TF PRDM1. CROP-Seq and bulk RNA-seq of MM cells with CRISPRa of PRDM1 (vs. other NRSPs or control sgRNAs) revealed that PRDM1 overexpression is toxic to MM cells by suppressing other dependencies, including IRF4, another lineage-defining TF and master MM dependency.
 Conclusions: Collectively,our studies have identified a large collection of NRSP genes whose overexpression suppresses tumor cell fitness in MM, with several of them also being relevant for other hematologic neoplasias. Several canonical tumor suppressor genes are also identified by CRISPRa, but most NRSPs have not been previously examined for their potential to suppress tumor cell fitness in MM or other neoplasias. Our study highlights the concept that engineered overexpression of NRSPs has intriguing potential for therapeutic applications in MM and other hematologic neoplasias.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28bf8e777a2bb3a092a1281fdd85652dfe41d721" target='_blank'>
              Negative regulators of survival and proliferation (NRSPs) identified by CRISPR activation studies in multiple myeloma (MM) vs. other neoplasias: Therapeutic implications
              </a>
            </td>
          <td>
            O. Dashevsky, Ricardo de Matos Simoes, Huihui Tang, Torsten Steinbrunn, S. Yamano, Eleni Anastasakou, Benjamin G. Barwick, S. Gandolfi, Ryosuke Shirasaki, Vikas A. Gupta, Michal Sheffer, Jonathan Licht, Lawrence Boise, Matthew Oser, Constantine S Mitsiades
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Therapy of AML patients remains challenging due to high rates of relapses even after intensive treatment. Relapses are mostly caused by residual chemoresistant leukemic stem cells (LSC). Long non-coding RNAs (lncRNA) have been shown to be important regulators of normal hematopoiesis and pathogenesis of leukemias. LSCs show a pattern of highly enriched lncRNAs. One of those highly abundant lncRNAs is WT1-AS, which was described as a tumor suppressive as well as oncogenic lncRNA in different solid malignancies. The high expression of WT1-AS in LSCs suggests an oncogenic role in AML. However, the function of WT1-AS in AML has not been elucidated yet.
 Since dysregulated growth is one key feature of malignant cells, the impact of WT1-AS on cell growth of AML cells was evaluated. Therefore, an shRNA-based WT1-AS knockdown (WT1-ASkd) was established in representative AML cell lines, and growth competition assays using RFP-labelling as readout were performed. Here, WT1-ASkd cells consistently exhibited a significant growth disadvantage compared to WT1-ASwildtype cells. To further investigate the cause of the observed growth disadvantage, cell cycle properties and apoptosis were analyzed. The knockdown of WT1-AS not only increased the proportion of apoptotic cells but also induced a cell cycle arrest in the G0/G1 phase of the cell cycle.
 To gain first mechanistic insight into the pathways underlying the induction of apoptosis and cell cycle arrest upon WT1-AS knockdown, comprehensive transcriptomic analyses were performed. WT1-ASkd cells exhibited upregulated expression of myeloid differentiation gene signatures alongside a downregulation of stemness-associated genes. Since terminal differentiation is accompanied by cell cycle exit and ultimately apoptosis, these findings provide a plausible explanation for the observed growth arrest and increased cell death.
 To recapitulate stemness and differentiation in AML, an in vivo competition assay using an AML patient-derived xenograft (PDX) mouse model was performed assessing the influence of WT1-AS knockdown on AML progression. PDX cells were transduced either with an shRNA eithertargeting WT1-AS or a scramble control, and subsequently transplanted in a ~1:1 RFP+/RFP- ratio into sublethally-conditioned NOD scid gamma (NSG) mice. After AML onset, mice were sacrificed and the percentage of human CD45+/RFP+ was quantified in the peripheral blood (PB), the bone marrow (BM), and the spleen, and compared between the WT1-ASkd and the scramble control group. The percentage of RFP+ cells significantly decreased within all three compartments in the WT1-ASkd group, whereas it remained relatively stable in the control cohort (PB 1% vs 37%, p<0.0001; BM 2% vs 31%, p=0.0053; spleen 8% vs 38%, p=0.009), implicating that WT1-AS knockdown reduces AML engraftment in this model. Notably, the remaining human CD45+/RFP+ cells in the WT-AS1kd PDX model predominantly expressed the myeloid differentiation marker CD11b, while the large proportion of human CD45+/RFP-cells were mostly CD11b-negative (68% vs 18% CD11b+; p=0.001) indicating a reintroduction of myeloid differentiation upon WT1-AS knockdown in vivo.
 To explore the clinical relevance of our findings, we investigated the impact of WT1-AS expression in the BEAT AML patient cohort focusing on patients with a normal karyotype. Here, the quartiles with the highest WT1-AS (WT1-AShigh) and lowest WT1-AS (WT1-ASlow) mRNA expression were compared. Consistent with our in vitro findings using the WT1-ASkd models, the WT1-ASlow group associates with several myeloid differentiation gene signatures (NES<-1.7, p<0.05). Furthermore, the LSC17 score, which reflects the presence of LSCs, is higher in the WT1-AShigh cohort underlining the more stemness-like phenotype of WT1-AShigh AML. Comparing the mutational profile of both groups, the WT1-AShigh AML patients exhibit mutations in NPM1 and WT1 more frequently, whereas the WT1-ASlow patients show significantly more secondary-type mutations (SRSF2, STAG2, ASXL1).
 In summary, our data identify WT1-AS as a key regulator of leukemic stemness and a suppressor of myeloid differentiation in AML. WT1-AS knockdown leads to reintroduction of myeloid differentiation in vitro and in vivo, which ultimately leads to cell cycle arrest and apoptosis. Our data further highlight the decisive impact of lncRNAs on the fate of AML cells and their potential as targets in AML treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1527c5a2ed835dfeda263a8a0b1cc1a81025c" target='_blank'>
              The long non-coding RNA WT1-AS controls differentiation programs and leukemic cell fate in AML
              </a>
            </td>
          <td>
            Silvia Schäfer, Pascal Fichtel, Silke Brilloff, Sandra Küchler, Elahe Rahimian, Vida Kufrin, Ekaterina Longvinova, C. Ball, H. Glimm, Denis M. Schewe, M. von Bonin, M. Bornhäuser, A. Wurm, Marius Bill
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract The clonal evolution of breast cancer involves a complex dialogue between tumor cells and their environment. In this process, epigenetic mechanisms play a crucial role in regulating the cellular transcriptome without altering the underlying DNA sequence. Here, we provide an updated summary of three main epigenetic mechanisms: histone modifications, long non-coding RNAs (lncRNAs), and higher-order chromatin structures. Post-translational modifications of DNA or histones influence gene expression by altering chromatin accessibility and/or recruiting regulatory protein complexes. This process is dynamically regulated by enzymes that add or remove these marks, as well as by reader proteins that recognize them. Dysregulated expression or malfunction of these regulators creates an aberrant epigenetic landscape and gene expression profile, contributing to breast cancer initiation, metastasis, and drug resistance. Notably, the donor molecules for chromatin modifications are largely derived from intermediate metabolites shaped by environmental cues, highlighting the intricate crosstalk between epigenetic regulation and both cellular and systemic metabolic states. DNA and histone modifications are further interrelated with lncRNAs and higher-order chromatin architectures, which have been actively investigated in breast carcinogenesis. We also briefly introduce the role of epigenetics in other chromatin-associated events such as replication initiation. Aberrant replication initiation can drive gene duplication and genomic alterations resembling those observed in clinical breast cancer, endowing tumor cells with growth advantages and therapeutic resistance. Finally, we summarize emerging therapeutic strategies that target epigenetic vulnerabilities in breast cancer and discuss their current limitations and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502ad7aff3505dc7bf1c94f78c3fe440208426d" target='_blank'>
              Advances in the epigenetic regulation of breast cancer
              </a>
            </td>
          <td>
            Xiao Liu, Xin Wu, Jianan Ding, Xiaoying Qiao, Jing Liang
          </td>
          <td>2025-11-21</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bladder cancer is a highly heterogeneous malignant tumor of the urinary system with high recurrence rates, posing significant challenges in its diagnosis and treatment. Advances in multi-omics research have elucidated the molecular mechanisms underlying the pathogenesis and progression of bladder cancer, including driver gene mutations (e.g., FGFR3, TP53/RB1), dysregulation of signaling pathways (such as PI3K/AKT/mTOR and RAS-MAPK), epigenetic alterations, non-coding RNA networks, tumor microenvironment remodeling, and metabolic reprogramming. This review systematically summarizes recent progress in translational research bridging molecular mechanisms to breakthroughs in precision therapy, covering the clinical applications and challenges of FGFR inhibitors, immune checkpoint inhibitors, antibody-drug conjugates, and gene therapies. Translational efforts are increasingly relying on molecular subtyping to develop subtype-specific treatment strategies. Although significant advances have been made in precision therapy for bladder cancer, critical research gaps remain, including tumor heterogeneity, therapy resistance, and insufficient validation of biomarkers. Future research directions emphasize the potential of liquid biopsy for non-invasive diagnosis and dynamic monitoring, rational combination therapies, multi-omics data integration, and artificial intelligence in advancing personalized treatment, providing a systematic and forward-looking perspective on precision medicine in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14a6737ae3e58aea00cf12ac1c7b98be1125652" target='_blank'>
              Molecular mechanisms and translational advances in bladder cancer: from driver genes to precision therapy
              </a>
            </td>
          <td>
            Jinjie Xiang, Yuhui Luo, Baiyu Zhang, Kunbin Ke, Hao Li
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Despite improved targeted therapies and chemotherapy regimens for acute myeloid leukemia (AML), durable remissions remain elusive for most patients. A major barrier to curative treatment is therapy-persistent leukemia cells (residual disease) that evade initial treatment and drive relapse. We hypothesized that these persistent cells harbor distinct biological features that enable them to survive initial therapy. To identify selective vulnerabilities, we conducted a genome-wide CRISPR interference (CRISPRi) screen in AML cells that survived treatment. In parallel, we established an in vivo patient-derived xenograft (PDX) model of residual disease that models clinical disease scenarios and enables transcriptomic profiling at key treatment stages.
 We first performed CRISPRi screening in CAL-1 cells, derived from BPDCN, a representative high-risk AML subtype that responds to most initial therapies but rapidly relapses. Two regimens were tested, venetoclax plus azacitidine (VEN/AZA) and tagraxofusp, each at its IC90 dose, to explore shared targets in residual disease. Our screen identified several expected pathways — pro-apoptotic genes such as BAX and NOXA as mediators of sensitivity to VEN/AZA, and CD123 and diphthamide biosynthesis genes as sensitizers to tagraxofusp. Notably, KH-type splicing regulatory protein (KHSRP) was the top gene promoting therapy persistence across both regimens. In validation experiments, KHSRP depletion substantially reduced residual leukemia viability across several regimens in multiple AML cell lines derived from other high-risk disease subtypes, achieving up to 10 times more residual cell elimination after initial treatment vs. controls.
 Using the in vivo PDX model, we defined three time points: pre-treatment, residual disease after VEN/AZA treatment, and relapse. Pre-treatment and relapse had >80% marrow burden, whereas residual disease was <20%. RNA-seq revealed that residual cells were transcriptionally distinct from both primary and relapsed leukemia. We then integrated PDX expression analysis with gene expression in AML cell lines after KHSRP depletion. This turned our focus to SLC23A1, which encodes the sodium-dependent vitamin C transporter 1, as it was the only gene significantly downregulated in residual disease and upregulated by KHSRP depletion. Mechanistically, we found that KHSRP promotes SLC23A1 mRNA decay via recruitment of the NEXT nuclear RNA degradation complex. We thus hypothesized that residual leukemia cells reduce SLC23A1 to resist chemotherapy, and that KHSRP depletion restores its expression, sensitizing cells to treatment. This model was supported by rescue experiments in which SLC23A1 knockdown abolished the therapy-enhancing effects of KHSRP depletion.
 Vitamin C has been explored as an anti-leukemic agent via promoting the conversion of 5-methylcytosine (5-mC) to 5-hydroxymethylcytosine (5-hmC), facilitating DNA demethylation, particularly in TET2 mutant cells. Given that the active, reduced form of vitamin C is unstable in standard culture media, we tested the role of uric acid, the other molecule transported by SLC23A1 and that is present in the media at stable physiologic concentrations via FBS supplementation. Similar to vitamin C, we found that soluble uric acid also enhanced 5-mC to 5-hmC conversion in AML cells. Furthermore, exogenous vitamin C and uric acid were significantly synergistic in reducing viability of KHSRP-deficient residual cells after VEN/AZA treatment. Moreover, TET2 re-expression in TET2-null AML cells mimicked the effects of KHSRP depletion in eliminating the residual population, with no additional effect observed upon further KHSRP knockdown. Thus, uric acid and vitamin C may serve as alternative TET enzyme cofactors that mediate the therapeutic effects of KHSRP depletion.
 Importantly, in the Cancer Dependency Map, only a few solid tumor cell models are strongly dependent on KHSRP, and our data showed that KHSRP depletion had minimal effects on untreated leukemia cells. In contrast, the impact of KHSRP depletion on eliminating therapy-persistent leukemia cells was chemotherapy dose-dependent, indicating a selective sensitivity in residual disease. We are currently evaluating the effects of KHSRP depletion in prolonging survival of two independent AML in vivo models treated with VEN/AZA. These data nominate KHSRP as a selective vulnerability in therapy-persistent AML and warrant the development of KHSRP inhibitors to target residual disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afceff0e3f46ce9daa03fb1c28ee8b20052c2b60" target='_blank'>
              Khsrp depletion selectively eliminates AML cells that persist after venetoclax/azacitidine therapy via modulating SLC23A1-transported TET enzyme cofactors
              </a>
            </td>
          <td>
            Qingyu Luo, Karley Whalen, Xiaowei Wu, Amanda L Fortune, Lin Zhang, Weiye Qian, Evangeline G. Raulston, Y. Nan, Christopher A G Booth, Kezhi Yan, David Root, John Doench, Andrew Lane
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Introduction: Subclonal diversity in multiple myeloma (MM)—defined by genetically distinct populations of malignant plasma cells—plays a pivotal role in disease progression, therapeutic resistance, and relapse. Characterizing subclone-specific transcriptomic profiles, copy number alterations, somatic variants, and their interactions with the immune microenvironment is critical for understanding tumor heterogeneity and identifying novel therapeutic targets. In this study, we analyzed smoldering multiple myeloma (SMM) and newly diagnosed multiple myeloma (NDMM) samples to explore the genomic landscape of subclones and their crosstalk with immune cells.
 Methods: CD138+ scRNA-seq and scATAC-seq multiomic data from SMM patients (n=10) and NDMM patients (n=21) were obtained from the published study dbGAP phs003220 (PMC11099140). Matched CD138- scRNA-seq data of immune cells were sourced from Sudha et al., IMS 2023. Subclones for each tumor sample were identified using inferCNV (v1.8.1) and a custom-built integration pipeline that utilized both scRNA-seq and scATAC-seq data. Cell-cell communications between subclones and immune cell types were identified using CellChat (v1.6.0) using the ligand-receptor interactions from CellChatDB. Copy Number Abnormality (CNA) profiles of subclones from single cell were compared with whole genome sequencing (WGS) data (phs003220) and similarity scores were computed. Due to differing technologies (RNA-seq vs WGS) and sequencing depth (single cell vs. bulk), this score reflects how well single-cell multiomics captures major chromosomal changes seen in WGS. Subclone-to-immune cell interactions were quantified to identify statistically significant chromosomal abnormalities that change their prevalence. Survival analysis was performed on the most common MM abnormalities to identify if subclonal co-occurrence influences prognosis. For each subclone, BAM files were generated, and somatic variants (SNVs + Indels) were called using Strelka2 with WGS control samples from the same patient.
 Results: We identified 23 subclones in SMM with a mean of 2.30 (min= 1; max=4); and 79 subclones in NDMM with a mean of 3.59 (min=1; max=8). Subclone and WGS CNAs were comparable to one another with mean Jaccard index (JI) of 0.57 for SMM and 0.69 for NDMM. Samples with low JI tended to have more subclones and more complex CNAs as measured through ploidy (correlation coefficient (r) = -0.069 in SMM and -0.177 in NDMM). NDMM subclones tended to have more CNAs than in SMM. From the IU cohort, Del13 was observed in 60% of the SMM and 45% of the NDMM patients. Gain1q was identified in 50% of the SMM and NDMM patients. Patients with co-occurring Gain1q and Del13q in the same subclone had significantly shorter progression-free survival (PFS) than patients with at least one of these alterations (median PFS 285 days vs 1194 days; P = 0.004). Subclones with Gain1q had a lower average percentage of interactions with CD8T cells (Gain1q=25.435, Norm1q=33.26; P=0.005). In NDMM subclones with Gain1q, there was a lower percentage of MHC-I pathway interactions with CD8T cells (P=0.001) and MHC-II pathway interactions with CD4T cells (P=0.001). Subclones with Del13q showed a higher percentage of MIF pathway (MIF-CD74/CXCR4) interactions in NDMM. SMM subclones with Del13p tended to have a lower percentage of MHC-I interactions in CD8T, CD4T and NK cells. Gain6p subclones had a higher percentage of MK pathway (MDK-NCL/ITGA4+ITGB1) interactions in SMM subclones. Somatic variants were identified for each subclone from the scATAC-seq. When compared with WGS somatic variants in NDMM, a median of 253 variants was found both subclones and WGS while 2752 were unique to subclones. Numerous pathogenic somatic variants were identified in the myeloma driver genes from subclones.Conclusions: The identification of subclone-specific immune interaction networks provides new opportunities for targeted therapeutic intervention. These results establish subclonal CNA profiling as both a prognostic tool and guide for precision immunotherapy in multiple myeloma, particularly for high-risk disease characterized by specific genomic alterations co-occurring in subclones. This study underscores the importance of single-cell analysis to fully capture the clinically relevant genomic heterogeneity that drives disease progression and treatment resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4387fcdd106b6a1f94375e3c3e641c2dbb1f9a" target='_blank'>
              Subclonal genomic heterogeneity shapes immune crosstalk and clinical outcomes in multiple myeloma
              </a>
            </td>
          <td>
            Parvathi Sudha, Sylvia Robertson, Sihong Li, Sean D. McCabe, Kelvin Lee, R. Abonour, Brian Walker, A. Suvannasankha, Travis S. Johnson
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Older patients with acute myeloid leukemia (AML) have significantly worse response rates and overall survival compared to younger patients, even when matched for cytogenetic risk. This discrepancy has previously been explained by decreased tolerance of therapy and higher incidence of adverse risk disease. However, when these factors are taken into consideration there remains a higher incidence of primary refractory disease and relapse in older individuals, suggesting an underlying non-mutational resistance to therapy. Recent studies have implicated epigenomic disorganization from DNA damage repair as an important driver of cellular aging. The biological basis for age-related resistance and whether epigenomic disorganization contributes remains unclear.
 To investigate the impact of age on AML biology, we generated murine AML models by transducing hematopoietic stem and progenitor cells (HSPCs) from fetal (E13.5), young adult (8-10 weeks), and aged (>24 months) mice with retroviruses expressing MLL-ENL and NRasG12D. All models developed monocytic AML, confirmed by morphology and immunophenotyping, and were lethal upon transplantation into syngeneic recipients. Additionally, gene expression profile differences between young and old murine models were consistent with differences seen between younger and older AML patient samples (FDR q < 0.001) supporting their relevance to the human disease. We assessed in vitro sensitivity to menin inhibition, azacitidine, venetoclax, doxorubicin, and cytarabine. Whole exome sequencing (WES), bulk RNA-seq, single-cell RNA-seq (scRNA-seq), and proteomic profiling were performed to identify molecular correlates of resistance.
 AML cells derived from aged mice HSPCs were significantly more resistant to all tested therapies compared to fetal and young derived AML cells with marked resistance to cytarabine, doxorubicin and menin inhibition (p<0.001). WES did not reveal any recurrent mutations to explain this resistance. Transcriptomic and proteomic data were analyzed by KEGG pathway analysis revealing significant down regulation of several DNA damage repair pathways in older AML cells including base excision repair, homologous recombination and mismatch repair. In contrast, older AML cells had significantly increased expression of anti-apoptotic genes including multiple BCL2 family members. RNA and proteomics data demonstrated that younger AML cells had significantly increased expression of genes/proteins involved in chromatin and epigenome maintenance as identified by Gene Ontology terms (FDR q < 0.01) suggesting older AML cells may have a more disordered epigenome. Consistent with this, scRNA-seq revealed significantly increased transcriptional heterogeneity in aged AML cells compared to young and fetal models. This finding was corroborated in primary human AML samples when compared by age (p=0.0004). This suggests that AML cells derived from older HSPCs have a more disordered epigenome likely as the result of repeated rounds of DNA damage repair displacing epigenetic marks. To directly test if repeated rounds of DNA damage repair from a non-carcinogenic source would result in AML therapy resistance, we constitutively expressed the restriction endonuclease PPOI or GFP in fetal derived AML cells. After 12 weeks PPOI expressing fetal AML cells were significantly more resistant to cytarabine (p<0.0001) and doxorubicin (p<0.0001) compared to GFP expressing cells, mimicking the aged phenotype. They also appeared more aggressive when injected into syngeneic recipients.
 These findings support a unifying model of epigenetic based therapy resistance in AML, wherein aged HSPCs after multiple years of DNA damage repair, have a disordered epigenome and increased transcriptional heterogeneity. This heterogeneity is passed on to the resultant AML and enables survival of resistant subclones under therapeutic pressure, contributing to poor outcomes in older adults. This would also explain why therapy related and relapsed AML, where repeated DNA damage repair cycles have taken place, also have a worse outcome within the same cytogenetic risk category. Finally, it identifies a potential source of confounding in preclinical models that utilize enzymes that repeatedly induce DNA strand breaks like CAS9. Most importantly, it suggests that clinical trials data should be interpreted with extreme caution when considering patients outside the age range of those enrolled in the study.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399dae455f6a2d1caf558c73e5130583d602ef95" target='_blank'>
              Age-associated non-mutational resistance in acute myeloid leukemia is driven by DNA damage-induced transcriptional heterogeneity
              </a>
            </td>
          <td>
            Jordon Burton, K. Pladna, Lance Miller, Birgit Schilling, Timothy Pardee
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Objectives: Acute myeloid leukemia (AML) is mostly driven by specific mutations that affect transcription factors and epigenetic regulation. While targeted therapies are being developed for few AML subtypes, most patients still rely on systemic genotoxic treatments to induce complete remission and/or to bridge to bone marrow transplantation. Hence, deeper understanding of underlying disease and therapeutic mechanisms is urgently needed. In recent years, agents targeting the DNA damage response - such as PARP inhibitors (PARPi) already exploited in solid tumours, have emerged as promising therapeutic agents also in AML. PARPi alone exhibits high efficacy for a subset of AML, such as those driven by AML1-ETO and RARα fusions, but is rather ineffective forMixed-Lineage Leukemia gene-rearranged (MLLr) AML. A major function of these genotoxic agents is to interfere with DNA replication resulting in replication fork collapse and excessive DNA damage. However, cancer cells may adapt damage tolerance mechanisms, including fork reversal and repriming, to potentially bypass the genotoxic insults. We recently reported that, upon stimuli-induced proliferation, hematopoietic stem and progenitor cells (HSPCs) experience drastic changes in DNA replication dynamics and use specific fork protection mechanisms to promote their repopulation potential. Hence, we hypothesized that AML derived from deregulated hyperproliferating HSPCs may also adapt their DNA replication process, which may offer novel vulnerabilities with therapeutic potential.
 Methods: We investigated DNA replication dynamics and replication-associated DNA damage in established human and mouse AML models, that express different oncofusion proteins and reportedly display differential sensitivity to PARPi. To this aim, we used single-cell and single-molecule assays (e.g. comet assays, quantitative image-based cytometry, DNA fiber spreading) under baseline conditions and upon treatment with PARPi or standard-of-care chemotherapeutics. We also combined these functional assays with inactivation or overexpression of key replication fork plasticity factors, and with cell survival, apoptosis and differentiation assays, to evaluate how specific mechanisms of fork plasticity contribute to replication dynamics and therapy response in different AML subtypes.
 Results: We found that all human and mouse AML lines exhibit remarkably fast replication fork progression, compared to control HSPCs. However, AML lines previously reported as PARPi-sensitive – i.e. NB-4 (PML-RARα) and Kasumi-1 (AML1-ETO) – rapidly respond to PARPi by further accelerating fork progression, and eventually experiencing fork breakage and S-phase specific DNA damage signaling. Remarkably, inactivation of the specialized fork restart helicase RECQ1 suppresses these phenotypes, linking PARPi response to replication fork reversal and restart. Conversely, we found that PrimPol-mediated DNA synthesis – an alternative DNA damage tolerance mechanism that competes with fork remodelling – is predominant in MLL-rearranged leukemia, i.e. an aggressive AML subtype associated with poor prognosis. Notably, overexpression of Primpol in PARPi-sensitive AML is sufficient to prevent fork breakage and PARPi-sensitivity in these cells, while Primpol downregulation sensitizes MLL-rearranged AML lines (e.g. MOLM-13, THP-1), inducing fork collapse and replication-associated breaks. Importantly, similar observations were collected upon AML standard-of-care treatments (e.g. AraC), suggesting that mechanisms underlying replication fork plasticity play a central role in governing AML response to genotoxic treatments.
 Conclusions: These findings shed light on key mechanisms of the replication stress response in AML and underscore their potential for clinical translation. In light of our data, fork plasticity factors may represent novel therapeutic targets and predictive biomarkers for AML patient stratification. This evidence encourages the development of inhibitory compounds to target specific replication fork transactions and thereby improve therapeutic outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee77d6b18642e1d784e7f8b45c03785c72394af4" target='_blank'>
              Replication fork remodeling is a therapeutic vulnerability in acute myeloid leukemia
              </a>
            </td>
          <td>
            Cyril Dördelmann, T. Fung, Tristan Gasparetto, Daniel González‐Acosta, Larissa Bomfim, Chi Wai Eric So, Massimo Lopes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00ed13bb7acb39c5e98a4fa14592f332d5a91492" target='_blank'>
              A narrative review of therapy-induced senescence in cancer: mechanisms, immune interplay, and therapeutic opportunities
              </a>
            </td>
          <td>
            Henry Sutanto, Alfan Ahkami, D. Fetarayani, P. Z. Romadhon
          </td>
          <td>2025-12-01</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 BackgroundAcute myeloid leukemia (AML) remains a lethal hematologic malignancy characterized by clonal expansion of immature myeloid blasts and dismal long-term survival. Despite advances in genomics-guided therapy, relapse and chemoresistance are common, especially in older or refractory patients. Emerging evidence implicates metabolic reprogramming and bone marrow (BM) hypoxia as key drivers of leukemic stem cell (LSC) persistence and drug resistance. We previously identified FGF11—a member of the intracellular FGF subfamily—as markedly upregulated in primary AML blasts and cell lines. However, its mechanistic role in AML pathogenesis and therapeutic vulnerability remains undefined.MethodsWe analyzed FGF11 and HIF-1α expression in 173 de novo AML patient samples (TCGA-LAML and GTEx), 15 paired diagnosis/remission BM aspirates, and two AML cell lines (HL-60, HEL) using RNA-seq, qPCR, and immunohistochemistry. Subcellular localization and co-localization of FGF11 and HIF-1α were assessed by immunofluorescence and proximity ligation assays. Functional studies employed siRNA-mediated knockdown, and rhFGF11 supplementation. Seahorse metabolic flux analysis quantified glycolytic and oxidative phosphorylation (OXPHOS) rates. ResultsCompared with normal bone marrow, FGF11 mRNA was 4.3-fold higher in primary AML blasts (p<0.01) and predicted shorter event-free survival (EFS; HR=1.54, p=0.0028). Bioinformatic interrogation of the FGF11 promoter revealed hypomethylation at the CpG site cg00795277 that was tightly linked to adverse clinical outcomes, suggesting epigenetic activation of the locus.In HL-60 cells, immunofluorescence showed nuclear co-localization of FGF11 with HIF-1α, providing a spatial basis for functional interaction. Functional assays demonstrated that siRNA-mediated knockdown of FGF11 attenuated proliferation, enhanced apoptosis, and curtailed migration and invasion. Transcriptomic profiling of these cells disclosed down-regulation of HIF-1α signaling, alongside re-activation of mitochondrial and oxidative-phosphorylation programs, implying that FGF11 represses oxidative metabolism.To explore the reciprocal relationship, we exposed AML cells to CoCl2-induced hypoxia and observed a time-dependent increase in both FGF11 and HIF-1α mRNA and protein, confirming that HIF-1α can transcriptionally up-regulate FGF11. Proteomic analysis of hypoxic versus normoxic cells, annotated by COG classification, further enriched energy-metabolism pathway.Finally, exogenous rhFGF11 not only recapitulated the aggressive phenotype—enhancing migration and invasion—but also triggered a metabolic shift from oxidative phosphorylation to glycolysis. ConclusionsOur data uncover a novel FGF11–HIF-1α epigenetic-metabolic axis that orchestrates AML progression and chemoresistance through glycolytic reprogramming. Targeting this loop—either by disrupting FGF11/HIF-1α interaction or reversing metabolic addiction—represents a promising therapeutic strategy to overcome residual disease and improve outcomes in high-risk AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a40abdc907bf009e1fd38544864df761a97e31a" target='_blank'>
              FGF11–HIF-1α glycolytic epigenetic axis drives AML progression and chemoresistance via metabolic reprogramming
              </a>
            </td>
          <td>
            Hang Lv, Jiayao He, Zhuang Xin, Songfu Jiang, Xiaoying Man, Ruochen Wang, Rong Yang, Xiaoyu Zeng, Honglan Qian, Yifen Shi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, Cheng-Che Lee, George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, S. Yi, Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 IDH-mutant gliomas are typically slow growing on initial presentation, however selective pressure during conventional upfront treatment with genotoxic radiation and chemotherapy can fuel the evolution of sub-populations of tumor cells. These subclones promote recurrence of IDH-mutant gliomas that are less likely to benefit from standard treatments. Therefore, novel tumor-specific therapeutic strategies with non-overlapping mechanisms are needed for durable tumor control. IDH mutant cancer cells are known to reprogram their metabolism to sustain rapid proliferation, which generate potentially targetable vulnerabilities in canonical metabolic pathways. Using the patient-derived, WHO grade 3 oligodendroglioma line TS603, we screened a small molecule library of 240 metabolism-related compounds, including both FDA-approved and non- approved drugs, over a range of 10 concentrations (1nM-20µM). Drug efficacy was assessed after 5-day incubation via cell viability assay (Cell-Titer Glo). We identified 53 compounds with IC50<2µM, including drugs that target previously reported IDH-mutant glioma targets, like NAMPT and DHODH. Notable target pathways include folate metabolism and de novo thymidylate synthesis (methotrexate, raltitrexed, 5-FU), p97 AAA ATPase/VCP chaperone (CB-5083, ML240, eeyarestatin I), mevalonate pathway (Ro 48-8071 fumarate, atorvastatin calcium), and the BCL-2 family (Navitoclax, ABT-737). We are validating these drugs across a panel of patient-derived IDH-mutant gliomasphere models, and examining the combinatorial effects of these agents with vorasidenib and CDK4/6 inhibitors. Our results provide insight into important metabolic pathway dependencies in recurrent IDH1-R132H mutant gliomas that have targeting potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99af98af377416756584e34e3b8b2129cefb73a2" target='_blank'>
              DDDR-40. Identifying metabolic vulnerabilities of recurrent IDH1-R132H mutant glioma by high throughput screening
              </a>
            </td>
          <td>
            Yutong Wang, Sayali Daware, D. Cahill, Julie Miller
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background and significance. Stemness in acute myeloid leukemia (AML) is determined by a clonal hierarchy with rare, phenotypically defined leukemia-initiating cell (LIC) at its apex (Lapidot 1994, Bonnet 1997). Given that LICs in AML are drug-resistant and initiate relapse of refractory disease, the identification of an LIC population in B-ALL would be consequential. However, unlike AML, B-ALL does not evolve from a clonal hierarchy and LIC populations have remained elusive in B-ALL (Kelly 2007, Le Viseur 2008, Rehe 2013).
 Results. Time-lapse studies of single patient-derived B-ALL cells revealed that most B-ALL cells were continuously proliferating (steady), while subpopulations underwent periodic transitions between quiescent and proliferative cell states (alternating). Gene expression studies of 'steady’ and 'alternating’ cells revealed a dominant and reciprocally repressive transcriptional program driven by MYC in 'steady’ and BCL6 in 'alternating’ B-ALL cells. To dissect how MYC and BCL6 dynamics govern transitions between proliferative and quiescent cell states in B-ALL, we engineered patient-derived B-ALL (PDX) by CRISPR and HDRT with dual-reporter knockin alleles expressing C-terminal MYC–mNeonGreen and BCL6–mScarletI fusion proteins. Using time-lapse imaging, we tracked MYC and BCL6 protein expression in single B-ALL cells over 20 hours. Matching their proliferative characteristics, 'steady’ B-ALL cells expressed MYC and no detectable BCL6. However, B-ALL PDX also included 'alternating’ cells that underwent multiple transitions between MYC+ BCL6- and MYC- BCL6+ cell states. PDX from BCR-ABL1 B-ALL included approximately 30% 'alternating’ cells, whereas 'alternating’ cells exceeded 50% in B-ALL PDX with RAS-pathway lesions. Among 'alternating’ cells, transitions occurred independently from cell divisions (every 36 hours) with shorter MYC-phases of three hours and longer BCL6-phases of six hours.
 Integrated ChIP-seq, RNA-seq, and untargeted metabolomics revealed that MYC-high cells are much larger, activated glycolysis and protein-synthesis pathways, whereas BCL6-high cells are smaller and enriched for phosphatidylethanolamine (PtdEtn) synthesis pathways. Consistent with the essential role of PtdEtn in autophagosome biogenesis, BCL6+ B-ALL cells exhibited substantially increased autophagy (LC3B flux). To directly compare MYC and BCL6 dynamics with cell growth trajectories, we combined quantitative phase microscopy and time-lapse fluorescence imaging to measure single-cell dry mass over 8 hours. Cells in the MYC-high state accumulated biomass at twice the rate of BCL6-high cells, producing a step-by-step progression of fast (MYC) and slow (BCL6) cell mass accumulation that aligned with each MYC/BCL6 state transition. Finally, multiplex immunofluorescence of FFPE biopsies from patient samples revealed spatially segregated niches of MYC-high and BCL6-high leukemia cells reminiscent of partitioning of MYC+ centrocyte and BCL6+ centroblasts in germinal centers.
 To experimentally induce cell state-transitions, we engineered patient-derived B-ALL cells by HDRT with dual-reporter knockin alleles expressing C-terminal MYC–dTAG fusion proteins. In addition to the MYC-dTAG degron system, we used the BCL6 PROTAC degrader ARV-393 for acute ablation of either MYC or BCL6, which was achieved within 90 minutes. Acute dTAG-mediated degradation of MYC induced cell shrinkage and decreased cellular dry mass, whereas BCL6 degradation via PROTAC decreased autophagic flux and increased dry mass, confirming opposing roles in anabolic versus catabolic metabolism.
 To functionally study 'steady’ (MYC-only) and 'alternating’ (MYC/BCL6) B-ALL populations, we developed a cell-sorting strategy that enriched each of the two populations to a purity of 85% as confirmed by subsequent time-lapse imaging. Extreme limiting dilution and serial transplant experiments revealed that 'steady’ (MYC-only) largely lacked LIC-capacity. In contrast, 'alternating’ (MYC/BCL6) B-ALL cells were highly enriched for LIC and initiated fatal leukemia after short latency.
 Conclusion: This study uncovers previously unrecognized cell state transitions in B-ALL that are controlled by MYC- and BCL6-dependent transcriptional programs. Alternating cycles of quiescent and proliferative cell states balance anabolic and catabolic metabolism, promote drug resistance, and enhance leukemia-initiating potential.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21ac412fe67d4caf3c95440feaedce483b7297e3" target='_blank'>
              Alternating cycles of quiescent and proliferative cell states determine stemness and leukemia-initiation capacity in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhangliang Cheng, Kohei Kume, Ruoyi Shi, Mark E. Robinson, Kadriye N Cosgun, Yujin Bao, Siyi Chen, Pranav Mishra, Joerg Bewersdorf, Mina Xu, Markus Müschen
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Myeloproliferative neoplasms (MPNs), including chronic myeloid leukemia (CML), are oncogene-driven blood cancers arising from hematopoietic stem cells (HSCs) that often progress from a ‘smouldering’ chronic phase to a drug-resistant, pre-terminal blast phase. In CML, recent studies have shown that stem and progenitor cells (SPCs) from chronic phase (CP) patients at high risk of blast crisis (BC) transformation exhibit a convergent gene expression signature that mirrors that of overt BC, independent of underlying somatic mutations (Ko et al., Blood 2020; Krishnan et al., Blood 2023). Collectively, these findings suggest the existence of a common drug-resistant SPC state. Identifying the therapeutic vulnerabilities of such states holds promise for transforming the management of high-risk MPNs.
 To identify these resistant states, we performed single-cell (SC) multi-omic profiling (scRNA-seq, scATAC-seq, and targeted mutational analysis) on pre-treatment bone marrow (BM) mononuclear cells from (n=24, CML) three clinical groups (optimal imatinib [IM] responders, suboptimal IM responders, IM failure with BC progression; n=5/5/6), normal BM (n=4), and BC samples (myeloid n=2, lymphoid n=2). scATAC-seq data annotated 28 cell types from 240,456 cells; we identified DARs with DESeq2, inferred TF activity with ChromVAR, integrated gene-to-peak and enrichment analyses, mapped epigenetic remodeller occupancy with HOMER, and built Gene Regulatory Networks (GRNs) using SCENIC+ algorithm.
 Across all prognostic groups, analysis of LSCs revealed a shared, CML-specific epigenetic state distinct from normal BM HSCs, characterized by 29,317 differentially accessible regions (DARs) among 421,985 peaks (6.95%), with ~75% located in intronic and intergenic regions. Consistent with the drug-resistant phenotype of LSCs, functional annotation of these DARs highlighted hallmark features of resistance, including increased inflammatory and proliferative signaling and diminished stemness. Importantly, these LSC states were closely linked to canonical BCR::ABL1 signaling pathways, with enrichment of AP-1, STAT1/3/5A/5B, and NF-κB TF binding activity, implicating these pathways in the establishment of the LSC state. Strikingly, among the top 10 CML-specific transcription factors driving the LSC program, only STAT binding activity correlated with increasing clinical resistance, specifically through enhanced IL2-STAT5 and TNFα signaling.
 Given the relative stability of the LSC state across prognostic groups, we next examined the progenitor population for cell state changes. Unlike LSCs, nearly 32% of the progenitor epigenome showed altered chromatin accessibility with BC progression more extensive in lymphoid BC (LBC) (94,245 DARs) than myeloid BC (MBC) (43,378 DARs). In CP patients who developed LBC, 29,167 LBC-specific DARs were enriched in lymphoid progenitors at diagnosis. Whereas MBC-associated DARs (1,278–5,569) were weakly detected across myeloid subsets in CP patients who developed MBC. These findings support the notion of a common epigenetically-primed state in diverse progenitor compartments, predisposing them to subsequent transformation. Subtype-specific TF analysis showed AP-1/CEBP enrichment in MBC, and IRF8/9 in LBC. GRN reconstruction from scRNA-seq showed these TFs drive resistance associated programs characteristic of full-blown BC (Ko et al. Blood 2020). Ongoing GWAS-linked analysis seeks germline or acquired variants tied to these primed states.
 Lastly, we identified shared upregulation of STAT and IRF pathways in both BC subtypes, TFs downstream of JAK1/2/3, indicating convergent regulatory programs. This led us to testt, an FDA-approved JAK inhibitor, to dismantle these cell states. Although tofacitinib alone showed minimal cytotoxicity, it markedly sensitized BC cells (n=4) to TKIs (~90% killing) while sparing normal progenitors (~25% killing). Thus, mapping the CML epigenetic landscape identified core transformation regulators and revealed actionable therapeutic opportunities.
 This study presents the first scATAC atlas of CML resistance and progression, revealing that: (1) Epigenetic reprogramming in CP precedes BC transformation; (2) Distinct TF networks drive BC subtypes; and (3) JAK inhibition is a novel strategy to dismantle JAK/STAT-IRF activation in high-risk SPCs. Our findings highlight the power of SC- epigenomics to identify actionable vulnerabilities and guide pre-emptive therapies for high-risk MPN patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e0b2fbf0807bd6dc58e419db5b166831585fd5e" target='_blank'>
              Single-cell chromatin profiling reveals early epigenetic reprogramming, subtype-specific regulatory networks, and JAK/STAT–IRF vulnerabilities in high-risk chronic myeloid leukemia
              </a>
            </td>
          <td>
            S. T. Ong, Pavithra Shyamsunder, Beijun Chen, Zahid Nawaz, P. Venkatesh, Xinyi Yang, Y. Lim, Charles Chuah, Shyam Prabhakar, Vaidehi Krishnan, John Ouyang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard J Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, , Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Acidosis in tumors arises from reprogrammed metabolism and compromised vasculature, creating a harsh, acidic microenvironment that drives the evolutionary selection of acid-resistant cell phenotypes. A mathematical model is proposed to integrate phenotypic evolution, microenvironmental acidification, and tumor density dynamics. Three key mechanisms are incorporated in it: frequency-dependent selection favoring acid-resistant cells below a critical pH, stress-induced phenotypic switching, and a positive feedback loop where resistant cells produce excess acid that intensifies selection pressure. The well-posedness of the model is established. Through numerical simulations across biologically relevant parameter regimes, we identify two therapeutically targetable parameters: baseline acid clearance rate (a proxy for vascular perfusion) and a protection factor (representing acid-resistance"machinery"effectiveness) as critical bifurcation parameters for resistance evolution. The model exhibits qualitatively distinct dynamics depending on phenotypic plasticity levels. In low-plasticity tumors, both parameters exhibit sharp bifurcations with strong parameter interactions: clearance and protection effects are context-dependent, with therapeutic interventions effective only within specific parameter ranges. In high-plasticity tumors, both parameters produce continuous, monotonic responses with independent, additive effects. These regime-dependent dynamics suggest that treatment strategies should adapt to tumor plasticity: in the formar, targeting perfusion alone is typically sufficient, though sequential therapy may be required if the perfusion rate approaches or exceeds the bifurcation threshold, whereas in the latter treatment might benefit from combination therapies addressing both parameters simultaneously.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0887c6d3e30f61231afcb895923c66fd10d672" target='_blank'>
              Evolutionary Dynamics of Acid Resistance in Tumors: A Mathematical Model
              </a>
            </td>
          <td>
            Prithvi Anickode, Fabio Milner
          </td>
          <td>2025-10-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly specialized intratumoral compartment responsible for tumor initiation, metastatic dissemination, therapeutic resistance, and disease recurrence. A central conceptual challenge in CSC biology is their capacity to oscillate between a quiescent G0 state and a proliferative, stem-like phenotype, reflecting a high degree of phenotypic plasticity. Although dysregulation of the G1/S checkpoint is a hallmark of malignant transformation, its mechanistic contribution to CSC identity and plastic behavior remains poorly defined.This review outlines a conceptual model that integrates aberrant G1/S control with CSC state transitions. We propose that defective checkpoint regulation accelerates CSC proliferation, leading to the progressive intracellular accumulation of Cyclin D, which in turn drives a self-reinforcing, rapid G1 progression through phosphorylation-dependent pathways that operate independently of the slower, transcription-driven Cyclin D-Rb-E2F regulatory axis. With continued cycling, depletion of key E2F-regulated DNA replication factors ensues, eventually forcing CSCs into a quiescent, biosynthetic restoration phase. During this interval, essential genomic replication and cell cycle machinery are replenished until microenvironmental or intracellular cues trigger reentry into the proliferative cycle, giving rise to another burst of accelerated division.Through these cyclical perturbations in the Cyclin D/E2F balance, CSCs undergo temporally governed shifts between quiescent and proliferative states, thereby sustaining plasticity, intratumoral heterogeneity, and treatment-resistant phenotypes. This model also identifies potential therapeutic strategies, such as leveraging stimuli-responsive delivery systems that exploit cyclic CSC vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53543eb088f0d567f8bdf7288c39cb199fa8756" target='_blank'>
              Dysregulation of the Cyclin D/E2F activity as a core mechanism driving cancer stem cell plasticity and cell cycle dynamics.
              </a>
            </td>
          <td>
            Yasin Ahmadi, Trefa Mohammed, Sakhavat Abolhasani, Shwan Abdullah Hamad, Tahran Faiq
          </td>
          <td>2025-12-14</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing.
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f73f73f3e37aa93d53554d82182a30e140df7f94" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identifies CLK1 inhibition as a strategy to restore PARP inhibitor sensitivity via ERCC1 isoform switching.
              </a>
            </td>
          <td>
            Chaohua Liu, Xu Fei, Yutuan Wu, Li Jiana, Mengdong Ni, Xia Siyu, Chen Lihua, Haiyun Zhao, Yu Min, Zhou Yuqi, Meiqin Zhang, Jiajia Li, Xiaohua Wu, Huang Yan, Zhu Tao, Xiaojun Chen
          </td>
          <td>2025-11-04</td>
          <td>Protein & cell</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer therapy resistance emerges from highly integrated molecular systems that enable tumor cells to evade cell death and survive cytotoxic therapeutic stress. High Mobility Group Box 1 (HMGB1) is increasingly gaining recognition as a central coordinator of these resistance programs. This review delineates how HMGB1 functions as a molecular switch that dynamically redistributes between cellular compartments in response to stress, with each localization enabling a distinct layer of resistance. In the nucleus, HMGB1 enhances chromatin accessibility and facilitates the recruitment of DNA repair machinery, strengthening resistance to radio- and chemotherapeutic damage. Cytosolic HMGB1 drives pro-survival autophagy, maintains redox stability, and modulates multiple regulated cell death pathways, including apoptosis, ferroptosis, and necroptosis, thereby predominantly shifting cell-fate decisions toward survival under therapeutic pressure. Once released into the extracellular space, HMGB1 acts as a damage-associated molecular pattern (DAMP) that activates key pro-survival and inflammatory signaling pathways, establishing microenvironmental circuits that reinforce malignant progression and therapy escape. HMGB1 further intensifies resistance through upregulation of multidrug resistance transporters, amplifying drug efflux. Together, these compartmentalized functions position HMGB1 as a central node in the networks of cancer therapy resistance. Emerging HMGB1-targeted agents, ranging from peptides and small molecules to receptor antagonists and nanoformulations, show promise in reversing resistance, but clinical translation will require precise, context- and redox-informed HMGB1 targeting to overcome multifactorial resistance program in refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9023ae13f0952b85c73d6ba319988bfa2cf22" target='_blank'>
              HMGB1: A Central Node in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Bashar A. Alhasan, Boris A. Margulis, Irina V. Guzhova
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) remains a challenge in treatment and mechanisms understanding in disease progression and therapeutic resistance. Through large-scale transcriptomic profiling integrated with 10 machine learning algorithms, we identified CPNE8, a calcium-dependent phospholipid-binding protein which is involved in signal transduction, is significantly upregulated in HOXA-driven AML subtypes, whose function and regulatory mechanisms in leukemia warrants further investigation.
 Methods We integrated clinical and transcriptomic profiles from a cohort of 655 AML patients to develop a machine learning–based prognostic model, with external validation performed in the TCGA-LAML (n = 151) and GSE16432 (n = 436) datasets. CPNE8 expression across hematopoietic and immune cell compartments was profiled using single-cell RNA sequencing (GSE116256). Functional assays following CPNE8 knockdown were performed to evaluate cell viability, apoptosis, cell cycle dynamics, and leukemogenic potential in vivo. CPNE8-interacting proteins were identified via Co-immunoprecipitation (Co-IP) and Mass Spectrum (MS), with downstream signaling examined by immunoblotting. CUT&Tag profiling and dual-luciferase reporter assays were used to elucidate transcriptional regulation. OCR and ECAR were measured to assess cellular mitochondrial respiration and glycolytic capacity. Chemoresistance was assessed through compound screening and CellTiter-Glo assays, and the therapeutic efficacy of Menin inhibition was tested both in vitro and in vivo using a CPNE8-high OCI-AML3 xenograft model.
 Results CPNE8 was markedly upregulated in HOXA9-driven AML and associated with poor prognosis and chemoresistance. Patients with high CPNE8 expression exhibited significantly lower complete remission (CR) rates following two cycles of standard induction chemotherapy compared to CPNE8-low patients (p < 0.001), with pronounced resistance to conventional “3+7” anthracycline-based regimens. CPNE8 knockdown inhibited leukemic cell proliferation, induced apoptosis, and caused G0/G1 cell cycle arrest. Mechanically, CPNE8 physically interacted with SmgGDS, as identified by Co-IP–mass spectrometry and further validated by Co-IP assays and immunofluorescence co-localization，thereby activating the Rap1 GTPase module and driving downstream PI3K-AKT and MAPK (JNK/p38) signaling. To further assess the role of CPNE8 in drug resistance, ex vivo drug sensitivity assays were conducted using PBMCs from 107 AML patients. CPNE8-high PBMCs showed significantly elevated IC50 values across a range of clinically relevant chemotherapeutics, including idarubicin, homoharringtonine, fludarabine, azacitidine, venetoclax, aclarubicin, all-trans retinoic acid (ATRA), and mitoxantrone. Additionally, CPNE8 depletion elevated intracellular and mitochondrial ROS, reduced mitochondrial membrane potential, and impaired both glycolysis and oxidative phosphorylation, thereby sensitizing AML cells to chemotherapy and enhancing their vulnerability to T cell–mediated cytotoxicity. To delineate the upstream regulatory network, CUT&Tag profiling identified HOXA9 as a direct transcriptional activator of CPNE8, positioning CPNE8 as a critical effector of HOXA9-driven leukemogenesis. To therapeutically target the HOXA9–CPNE8 axis in CPNE8-high AML, we evaluated Menin inhibition as a strategy to disrupt the HOXA9 transcriptional program. Treatment with a Menin inhibitor significantly reduced CPNE8 expression in vitro. Importantly, both Menin inhibition and CPNE8 knockdown prolonged survival in CPNE8-high AML xenograft models, underscoring their therapeutic potential.
 Conclusion CPNE8 functions as a critical downstream effector of HOXA9, driving AML progression and therapy resistance through activation of Rap1 signaling, reprogramming of mitochondrial metabolism, and reshaping of the immune microenvironment. Blocking the HOXA9–CPNE8–Rap1 pathway, especially with Menin inhibitors, provides a rationale for stratified treatment strategies in CPNE8-high AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7845759371b121eb83b0e1b1c2020de00b0cd246" target='_blank'>
              Targeting CPNE8 suppresses HOXA9-dependent AML progression and overcomes chemotherapy resistance
              </a>
            </td>
          <td>
            Wenjing Shu, Yuncan Cao, Xiaojing Lin, Xin-jie Chen, Wen-Yan Cheng, Yang Shen
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Somatic mutations in SET binding protein 1 (SETBP1), a putative chromatin regulator, occur in ~30% of high-risk myeloid malignancies, including myelodysplastic syndromes (MDS), chronic myelomonocytic leukemia (CMML), and secondary acute myeloid leukemia (AML), and strongly correlate with adverse clinical outcomes. Despite this significance, the precise roles of SETBP1 mutations in blood cancer remain enigmatic, primarily due to the lack of physiologically relevant cell lines or animal models that recapitulate the clonal evolution. Indeed, our 23 attempts to establish SETBP1-mutated PDX models failed, as the clones did not show advantageous effects. While conventional wisdom suggests that SETBP1 mutations escape proteasomal degradation to drive oncogenesis, our Setbp1 knockout mice revealed that SETBP1 is dispensable for normal and malignant hematopoiesis (Tanaka et al. Leukemia 2023), challenging prevailing paradigms.
 To resolve these contradictions, we developed a blood-specific Setbp1 mutant locus knock-in (KI) mouse model using Vav1-iCre, corresponding to the most prevalent D868N mutation. Unlike retroviral overexpression that immortalizes murine hematopoietic stem and progenitor cells (HSPCs), our KI model did not perturb HSPC function or transcriptome in vivo, indicating that SETBP1 acts as a cooperative rather than an initiating driver. As ASXL1 mutations are the most frequent founder mutations in SETBP1-mutated MDS/AML, we created double KI mice with Asxl1/Setbp1 physiologic mutations for transplant studies. These mice exhibited striking monocytosis (~40% of peripheral white blood cells) and HSPC expansion, rescuing stemness defects imposed by ASXL1 mutations. scRNA-seq/ATAC-seq revealed unique transcriptional clusters enriched for Hoxa9/10, Eya1, Myb, and Myc pathway alongside increased chromatin accessibility, suggesting that both mutants cooperate to remodel chromatin and amplify oncogenic transcriptional programs. Over time, the transplant model with double KI developed a leukemia phenotype, including serially transplantable AML and CMML-like diseases, all spontaneously acquired RAS pathway-activating mutations in Nras, Flt3, and Ptpn11. This reflects findings from over 7,000 patient samples, showing a significant correlation between SETBP1, ASXL1, and RAS pathway mutations. Indeed, the presence of RAS mutations predicts poor outcomes in SETBP1-mutated cases. Functional validation confirmed that all three alterations are required for complete transformation toward AML, establishing a three-step model: ASXL1mutations initiate clonal evolution, SETBP1 mutations enhance oncogenic programs, and RAS pathway activation completes the transformation.
 Leveraging these unprecedented in vivo models and cell lines with physiologically relevant SETBP1 mutations, we conducted comprehensive therapeutic screens using whole-genome CRISPR/sgRNA and pan-cancer drug libraries. The integrated results revealed a critical dependency on the druggable Exportin 1 (XPO1), shown by a 20.8-fold reduction (p = 0.00004) in the 7-day CRISPR screen and a 0.0003-fold growth under Leptomycin B treatment compared to DMSO at day 4. This unique dependency was robustly validated through genetic and pharmacological approaches using shXpo1, Selinexor, and Leptomycin B, resulting in monocytic differentiation and apoptosis in Setbp1-mutated AML cells. Notably, Selinexor monotherapy achieved sustained remissions exceeding 110 days in vivo, profoundly extending survival compared to controls, which succumbed to leukemia with a median survival of 59.5 days (p = 0.0004), demonstrating its potential to eradicate SETBP1-mutated leukemia. Mechanistically, XPO1 physically interacts with SETBP1 through the nuclear export signal groove and acts as a chromatin regulator beyond its canonical nuclear-cytoplasmic transport role. XPO1 inhibition suppressed the transcription of leukemogenic genes, including Hoxacluster genes, Myb, Eya1, and, critically, Myc pathway, with altered XPO1-mediated chromatin accessibility, as shown by integrated ChIP/RNA/ATAC-seq. These results uncover a critical epigenetic vulnerability that represents a promising target for therapeutic intervention in SETBP1-mutated leukemia.
 In summary, our study elucidates the cooperative leukemogenic roles of SETBP1, ASXL1, and RAS pathway mutations and identifies XPO1 as a targetable epigenetic vulnerability. These findings support precision treatment strategies for aggressive myeloid malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6566631ba342b06e47eaa7af05b86f27fd8cd1" target='_blank'>
              Clonal evolution and therapeutic vulnerabilities in SETBP1-mutated leukemia: Insights from physiological mouse model and integrative functional screens
              </a>
            </td>
          <td>
            Wataru Saika, Koutarou Nishimura, Atsushi Tanaka, J. Takeda, A. Kon, A. Yoda, Kazunari Aoki, Masaki Nomura, Yui Koike, Hiromi Yamazaki, Seigi Oshima, Shinri Okada, Daichi Morii, Weijia Zang, Takaya Yamasaki, Hiromi Ito, Manabu Nakayama, Hiromi Tatsumi, Masashi Matsuda, Chien-Chin Lin, Kosuke Yusa, Haruhiko Koseki, Makoto Murata, Seishi Ogawa, Daichi Inoue
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Hematopoietic stem and progenitor cells (HSPCs) are central to therapeutic strategies such as bone marrow transplantation and gene therapy/editing for a wide range of hematologic and genetic disorders. Their long-term clinical efficacy, however, hinges on successful engraftment and sustained hematopoietic reconstitution. However, due to the heterogeneity of HSPCs, accurate identification of long-term hematopoietic stem cells (LT-HSCs), possessing robust self-renewal and multilineage potential, remains essential for enhancing therapeutic outcomes.
 In this work, we aimed at defining the transcriptomic and epigenetic signatures that enable the robust identification of different human HSPC subpopulations, with a special emphasis on primitive HSCs. This molecular characterization would enable not only the improved gene targeting of specific cell populations to design more effective therapeutic strategies, but also the prediction of functional attributes and efficacy of engineered HSCs, thereby reducing the need for animal use in the evaluation of the safety and efficacy of HSC gene therapies.
 In these experiments we performed single-cell multiome profiling (scRNA-seq + scATAC-seq) on CD34⁺ bone marrow (BM) cells from five healthy donors, capturing both gene expression and chromatin accessibility at single-cell resolution. Given the importance of precise cell annotation, we implemented a two-step strategy: automated annotation using SingleR, followed by manual refinement based on well-established HSPC markers. Within BM CD34⁺ cells, we identified a transcriptionally defined HSC compartment. We applied clustering analysis alongside the StemPublic stemness score to resolve three distinct HSC subtypes: (i) quiescent LT-HSCs (qLT-HSCs), enriched for quiescence-related genes such as EMCN and TEK; (ii) activated LT-HSCs (aLT-HSCs), expressing MECOM and RUNX1; and (iii) short-term HSCs (ST-HSCs), marked by elevated CDK6 expression. Our results enabled the definition of a novel transcriptional fingerprint (StemFingerprint) that offers a novel and robust molecular tool for accurate identification of LT-HSCs and distinction from closely related progenitors.
 Given the widespread clinical use of mobilized peripheral blood (mPB) as an HSC source in gene therapy appraoches, we also performed single-cell multiome profiling of CD34⁺ cells from mPB of three healthy donors. Applying StemFingerprint, we successfully annotated LT-HSCs and ST-HSCs, but, consistent with prior reports, no qLT-HSCs were detected, likely due to their rapid exit from quiescence upon mobilization and BM egress. A comparative analysis between LT-HSCs and ST-HSCs across BM and mPB samples revealed a conserved expression of stemness-associated genes between the different HSC sources. Notably, we identified novel LT-HSC-associated genes, including FTH1 (iron homeostasis), RYR3 (calcium signaling), and SRGN (niche maintenance), potential regulators of human HSC function. Interestingly, our analysis also identified novel surface markers (undisclosed) which are currently being validated for the selective isolation of LT-HSCs and assessment of their engraftment potential, a critical feature for the development of improved gene therapy/editing strategies.
 To further dissect the regulatory landscape of HSC subtypes, we analyzed transcription factor (TF) motif accessibility and inferred gene regulatory networks. We observed that ETS1 showed the highest centrality in qLT-HSCs, consistent with its role in maintaining quiescence. In contrast, aLT-HSCs exhibited central roles for MYC and KLF6, consistent with their activation and proliferation signatures. Notably, MYC, a pivotal regulator of LT-HSC activation, was co-expressed with TFEB and TFR1, recapitulating the regulatory axis governing the quiescence-to-activation transition. Altogether, our data highlight differential dynamics of transcriptional and regulatory programs across HSC states.
 Overall, our study presents a comprehensive molecular map of human HSPCs in both BM and mPB, integrating transcriptomic and epigenetic features with functional potential. By defining key molecular and regulatory signatures, particularly those defining quiescent and active LT-HSC states, we provide tools for improved HSC identification, manipulation, and clinical application. These findings pave the way for more precise, effective, and ethically responsible strategies in stem cell and gene therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb437fcb0b891b6649d7bd44b9a114cde1a97175" target='_blank'>
              Single-cell multiomic dissection of human hematopoietic stem cells reveals novel transcriptional fingerprints and regulatory programs in bone marrow and mobilized peripheral blood
              </a>
            </td>
          <td>
            Daniel Mouzo, A. Ortega-Legarreta, Ana Rosa López-Pérez, Núria Planell, Patxi San Martín-Úriz, A. Vilas-Zornoza, P. Aguirre-Ruiz, Patricia Jauregui, Rebeca Martinez-Turrillas, J. Rifon, V. Lagani, Robert Lehman, Jesper N. Tegnér, Marta Perez-Rodriguez, Susana Navarro, J. Bueren, Paula Río, J. R. Rodríguez-Madoz, D. Gómez-Cabrero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Medulloblastoma is molecularly characterized into 4 main subgroups consisting of WNT, SHH, Grp 3 and Grp 4, which are further divided into 13 subtypes. Medulloblastoma manifests as a clinically heterogeneous tumor with varying prognoses depending on the molecular subgroup. Treatments for high-risk patients include craniospinal irradiation (36Gy) after surgical resection and chemotherapy. Group 3 tumors with upfront radiation therapy relapse in a shorter amount of time at 1 year. Currently, there are limited therapeutic strategies that address medulloblastoma tumor recurrence which has detrimental effects on patient outcome with <5% long term survival. Subsequently, there is a critical need to investigate the molecular mechanisms underlying therapeutic resistance in medulloblastoma and identify the best strategies to combat it. To understand what causes radiation resistance in medulloblastoma, we analyzed matched primary and relapsed patient tumors by single cell ATAC seq + gene expression. We further evaluated radiation resistance in an orthotopic mouse model and analyzed gene expression changes by single-cell RNA sequencing. We identify a resistant cell population that exhibits a distinctive gene signature comprising stem and metabolic gene transcription programs. We observe elevated expression of metabolic marker genes such as wtIDH1 and stem cell markers, TEAD1 and SOX6, within the recurrent cell population. We speculate that radiation resistance in group 3 medulloblastoma is facilitated by wtIDH1 metabolic reprogramming, leading to epigenetic adaptation and cell persistence. Thus, we evaluate how IDH1 changes the epigenetic landscape of radiation-resistant and sensitive cells by examining histone modifications (H3K27ac, H3K27me3, H3K4me3) and the broader chromatin architecture in response to IDH1 inhibition by Cut and Run and ATAC-sequencing. We observe that the manipulation of wtIDH1 reduces MB cell growth, stemness characteristics, and changes the chromatin landscape. Further, we find inhibition of wtIDH1 enhances susceptibility to re-irradiation and provides a survival benefit. Our results highlight evidence of the development of radiation resistance and provide a potential therapeutic alternative for treatment of relapsed MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d6602a0e48e41dffae7101ad80db0948939a5f" target='_blank'>
              DDDR-30. Targeting wtIDH1 in radiation-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, Rajeev Vibhakar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 CRISPR/Cas9 screening is a powerful tool used to delineate genes critical for AML progression and drug resistance. However, most of the studies have been performed in cell lines and PDX models. While cell lines fail to represent the genetic, epigenetic, and phenotypic heterogeneity of primary leukemic cells, PDX models are limited by variable engraftment efficiency, time-consuming processes, and high costs. To overcome these challenges, we pioneered an integrated approach combining bioinformatics and CRISPR/Cas9 screening to uncover genetic drivers and vulnerabilities in AML using patient-derived primary cells in culture. Insights from our study could inform the development of targeted therapies and improve patient outcomes.



 Custom CRISPR/Cas9 screen targeting 2,440 genes associated with AML, leukemic stemness, and the druggable genome was performed using mononuclear cells (MNCs) isolated from peripheral blood or bone marrow from 9 AML patients. Briefly, the cells were transduced with custom library and cultured for 24-30 days. After an initial few days in suspension, a subset of MNCs from 5 of 9 patients transitioned to an adherent state 5-7 days post-transduction. Both adherent and suspension cells were collected at different time points, sgRNA was quantified and sequenced using the Illumina NovaSeq X Plus. sgRNA abundance was analyzed using the RRA algorithm of the MAGeCK computational tool and genes with significant negative or positive enrichment in each patient was identified by comparing early versus late time points and adherent versus suspension cell fractions.



 In a subset of patient samples, AML cells transitioned from suspension to an adherent phenotype during long-term culture, a state previously associated with increased chemotherapy resistance due to interactions with endothelial niches. Our CRISPR/Cas9 screen revealed that this adherent cell population harbored distinct genetic dependencies, including consistent negative enrichment of EDNRA and SOD1.These genes were selectively essential in the adherent fraction but not in suspension cells, suggesting context-specific vulnerabilities. The identification of these targets offers new opportunities to therapeutically disrupt chemo resistant AML niches, potentially enhancing the efficacy of existing treatments and reducing relapse risk. EDNRA has been implicated in many cancers and in context of AML, it is a downstream target of HOXA9 and MEIS1. Our results indicate that EDNRA knockout inhibits AML cell proliferation and growth, highlighting its potential as a therapeutic target. Furthermore, EDNRA is overexpressed in AML compared to normal cells, and since FDA approved EDNRA inhibitors such as Macitentan, Bosentan, Ambrisentan, and Clazosentan are already being used to treat pulmonary arterial hypertension, this presents a promising opportunity to evaluate these inhibitors as potential chemotherapy-sensitizing agents for high-risk AML.SOD1 plays a critical role in eliminating toxic radicals generated within biological systems and has been associated with poor outcomes in AML. SOD1 has been identified as a synthetic lethal target in PPM1D-mutant leukemia cells, corroborating our findings. TRAPPC9, SUZ12, and ERG were significant genes with negative enrichment when comparing adherent and suspension fractions across 5 patient samples. These genes have a higher expression in primary AML cells compared to normal controls, implying potential roles in disease progression. SDHA was identified as a significant negatively enriched gene in the suspension cell fraction in 4 of 9 patients. It exhibited elevated expression in AML cells compared to controls, and its deficiency has been linked to the development of drug resistance in AML, emphasizing its role in both drug resistance and disease progression.



 This study demonstrates the establishment of a scalable platform for CRISPR screening in primary AML cells for identification of more conserved vulnerabilities that may be exploited therapeutically, with EDNRA as a potential target. Ongoing work includes validating key targets with the goal of advancing therapeutic strategies. Future studies will expand CRISPR screening to a broader set of patient samples with diverse cytogenetic profiles and incorporate CROP-seq technology to perform single-cell CRISPR screens, providing deeper insights into the molecular mechanisms underlying AML progression and relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46eb706147f8e498d47ce576631d70d699b36b33" target='_blank'>
              A patient-derived CRISPR platform reveals selective dependencies in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Aneesha Nath, Nam Nguyen, A. Tagmount, Nathan D. Seligson, Christopher D. Vulpe, Vivek Shastri, Dina Khalaf, Christopher Cogle, S. Pounds, J. Lamba
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 CML progression is defined by increased disease burden and potential for transformation to blastic phase (BP), yet the mechanisms and ability to predict this evolution remain elusive. We applied elements of state-transition theory to model transcriptional dynamics underlying leukemic progression, where individual cells' gene expression profiles represent microstates, and aggregated population-level gene expression profiles define macrostates that govern phenotypic transitions from health to disease initiation to overt leukemia. Previously, we demonstrated through bulk transcriptomic analyses that CML progression and therapeutic responses can be accurately predicted by modeling a leukemic potential landscape derived from transcriptional state-space geometry.
 To extend this framework to single-cell (sc) resolution and capture more granular information at the microstate level, we collected weekly peripheral blood samples from inducible chronic phase (CP) and BP CML mouse models to generate time-series scRNA-seq data tracking the transition from health to overt CP or BP. After performing quality control and cell labeling, we used the first (healthy) time point before the induction of BCR::ABL (T0) and the final leukemia time point (Tf), in addition to other intermediate time points, to assess changes in cell populations and gene expression. All analyses were first performed in CP mice, validated in BC mice, and corroborated using two independent human CML scRNA-seq datasets.
 By comparing sequential time points, we observed that progression to CP CML was associated with a significant decrease in B cells (p<0.01) and increases in both myeloid (p<0.01) and stem cell (p<0.001) populations. Differential gene expression (DEG) analysis revealed marked transcriptional changes across all major lineages (DEGs: B cells = 781; T cells = 2,149; Myeloid = 1,999; Stem cells = 194). Despite these changes, no single-cell transcriptional state at any time point was uniquely associated with health or disease, as transcriptional profiles of leukemic cells largely overlapped with those from healthy cells. However, when scRNA-seq data were computationally aggregated into pseudobulk (PsB) samples, mimicking bulk RNA-seq, a distinct CML state-space emerged, revealing a clear disease trajectory defined by three stable macrostates representing early, transitional, and late leukemia. Cell type–specific PsB analyses uncovered independent state-transition dynamics in B cell, T cell, myeloid, and stem cell compartments, highlighting previously unrecognized complexity in disease evolution involving multiple lineages. We further showed that the transition from health to leukemia at the PsB level could be reconstructed as a linear combination of transitions within individual cell subpopulations.
 To this end, we quantified each lineage's contribution to disease progression by performing computational simulations that subtracted the influence of each cell type in turn. Interestingly, B and myeloid populations contributed most and comparably to the global disease trajectory—despite B cells decreasing and myeloid cells expanding over time. This counterintuitive finding suggests that leukemic information is encoded not simply by abundance but by dynamic transcriptional shifts across compartments. These findings were validated in a BP mouse model and in two human CML datasets.
 In summary, we introduce a conceptual framework that distinguishes between microstates (individual cells) and macrostates (population-level transcriptional ensembles), showing that disease progression is not encoded at the single-cell level but instead emerges only when cells are aggregated. While this may appear intuitive, we were surprised to find that higher resolution sc data did not provide clearer insight than pseudobulk data. We speculate that this may reflect greater Shannon information entropy at the single-cell level, which introduces variability that obscures coherent state transitions. To our knowledge, this framework is novel and may inform future approaches to interpreting scRNA-seq data in leukemia and other diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef9c6ca07080f16386ecab84fd1ec67de21d807" target='_blank'>
              Chronic myeloid leukemia (CML) evolution is not encoded in single-cell microstates, but emerges from population-level macrostates
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, Yu-Hsuan Fu, Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Jeffrey Trent, Stephen J Forman, Ya-Huei Kuo, Adam MacLean, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pediatric acute myeloid leukemias (AML) are genetically and molecularly diverse. Pediatric AML carry a range of different driver mutations, often including translocations that produce fusion oncoproteins. Furthermore, they arise from diverse progenitor cell populations and at different stages of fetal and childhood development. This molecular and contextual diversity complicates efforts to identify effective targeted therapies, in part because modeling strategies do not capture the genetic heterogeneity observed among patients.
 We have developed a multipronged strategy to model complex genetic and contextual features of pediatric AML in mice, with the overarching goal of identifying malignant self-renewal programs that afford targetable vulnerabilities. We first used Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) and paired functional assays to identify functionally distinct subpopulations of NUP98-rearranged (NUP98r) AML using a previously described NUP98::HOXD13/Flt3ITD transgenic model. Our approach identified a leukemia stem cell (LSC) subpopulation and gene expression signature that was conserved in human NUP98r AML. The LSC signature was also enriched in human UBTF-tandem duplication (UBTF-TD) and NPM1 mutated (NPM1c) AML, but not KMT2A-rearranged AML, suggesting mutation class-specific self-renewal mechanisms.
 To compare LSC self-renewal mechanisms across molecular subclasses of AML, we developed a novel strategy to generate inducible, genetically diverse AML models. Specifically, we generated induced pluripotent stem cells (iPSCs) and chimeric mice to express NUP98::HOXA9, NUP98::KDM5A, NUP98::NSD1, RUNX1::RUNX1T1, MNX1, or KMT2A::MLLT1 oncoproteins specifically in blood cells at defined stages of ontogeny. We used CRISPR/Cas9 gene editing to introduce cooperating mutations at endogenous loci, including Flt3 internal tandem duplication (FLT3ITD), Nf1 loss of function (NF1LOF), and NrasG12D. With this approach, we could induce expression of any desired oncoprotein at any desired stage of development along with relevant cooperating mutation, all while bypassing complex mouse breeding. We also used gene editing strategies to generate NPM1c/FLT3ITD and UBTF-TD/ FLT3ITD AML, again with mutations at endogenous loci. The resulting AML models are transplantable, genetically manipulable and in some cases gave rise to immortalized in vitro cell lines.
 To credential the models and identify fusion/cooperating mutation-specific vulnerabilities, we performed CITE-seq focusing on NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF,RUNX1::RUNX1T1/NF1LOF, KMT2A::MLLT1/NRASG12D, UBTF-TD/ FLT3ITD, and NPM1c/FLT3ITD models. Cross species comparisons to human AML revealed faithful representation of heterogeneity and gene expression within the murine AML. Each genetic subclass exhibited distinct but reproducible differentiation trajectories and transcriptional profiles. NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF, NPM1c/FLT3ITD, UBTF-TD/ FLT3ITD, and KMT2A::MLLT1/NRASG12DAML all displayed intraleukemic heterogeneity and had putative LSC populations that expressed known self-renewal genes. RUNX1::RUNX1T1/NF1LOF AML was homogeneous and lacked a discernible LSC population. Presumptive LSC populations from NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF, NPM1c/FLT3ITD, and UBTF-TD/ FLT3ITD were all enriched for a shared gene expression signature. These highly tractable models thus capture key aspects of human pediatric AML biology.We sought to identify actionable, mutation class-specific vulnerabilities within putative LSC populations, focusing primarily on the NUP98r models. We found that genes encoding sphingosine transporters SPNS2 and SPNS3were highly expressed in NUP98r LSCs relative to non-LSCs, identifying them as potential therapeutic targets. SPNS2/3 were also expressed, to a lesser degree, in NPM1c/FLT3ITD and UBTF-TD/ FLT3ITD AML. Pharmacological SPNS2/3 inhibition severely impaired NUP98rAML growth in vitro, and it significantly extended survival in vivo. In some cases, mice with NUP98r AMLwere cured with SPNS2/3-directed monotherapy. Other genetic subclasses were less sensitive to SPNS2/3 inhibitors, as were normal human and murine hematopoietic progenitors. Our data indicate a therapeutic window for targeting SPNS2/3 in NUP98r AML. They raise the possibility of identifying other mutation class-specific vulnerabilities through the use of well-credentialed, iPSC-derived chimeric AML models.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c884502e32c16eaf731d0261d136ef166db839" target='_blank'>
              NUP98-rearranged pediatric AML is sustained by a conserved leukemia stem cell program with SPNS2/3 as a targetable vulnerability
              </a>
            </td>
          <td>
            Jihye Yoon, Elisabeth D Denby, Wei Yang, Run Zhang, Masayuki Umeda, Tingting Hu, Jonny Mendoza-Castrejon, Yanan Li, Emily B. Casey, Riddhi M. Patel, J. M. White, J. Klco, Jeffrey Magee
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Resistance to single-agent therapy in acute myeloid leukemia (AML) often arises from compensatory epigenetic circuits. We present a causal inference framework that treats therapy as an intervention (do-operator) and defines resistance as the deviation of dual perturbations from the additive expectation of monotherapies. By modeling the causal structure of the leukemia stem-cell epigenetic landscape that maintains identity and fitness, the framework shows that synergistic and antagonistic interactions can be inferred directly from single perturbations, suggesting interaction inference may be simpler than previously assumed. We applied this model to Perturb-seq of 16 epigenetic regulators in KMT2A-rearranged AML (>31,000 single-cell transcriptomes). From single-knockout profiles, the framework correctly predicted synergistic pairs (Menin+KAT6A, Menin+DOT1L) that enhanced differentiation and induced cell death, as well as an antagonistic pair (DOT1L + PCGF1) that conferred resistance. Predictions were validated by pharmacologic inhibition and bulk RNA-seq, confirming the model’s predictive accuracy. This work presents a mechanism-guided approach for rational prioritization of epigenetic drug combinations in AML and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057014eca0b4705ea6ee698244a083b962878940" target='_blank'>
              Causal interaction inference of compensatory structures from single-cell perturb-seq in AML
              </a>
            </td>
          <td>
            Changde Cheng, Sajesan Aryal, Brittany Curtiss, Xinyue Zhou, Rui Lu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Introduction: Acute Myeloid Leukemia (AML) is a heterogeneous malignancy characterized by the clonal expansion of immature myeloid precursor cells in the bone marrow and peripheral blood, leading to disrupted hematopoiesis and rapid disease progression. The incidence of AML increases with age, with a median age at diagnosis of approximately 68 years. Prognosis remains poor, particularly among older adults and those with adverse-risk genetic mutations, with 5-year overall survival rates often below 30%, even with intensive chemotherapy and hematopoietic stem cell transplantation (Ferrara et al., 2024; Ofran et al., 2023). With these clinical limitations, there remains a critical need for a better understanding of leukemogenic mechanisms and therapeutic resistance to guide novel and effective targeted treatments. In AML, long non-coding RNAs (lncRNAs) are now recognized as key players in leukemogenesis, disease progression, and therapeutic response. In particular, MALNC is a novel lncRNA recurrently overexpressed in AML subtypes harboring NPM1, IDH2 mutations, or the PML-RARA fusion gene, but its functional significance remains unclear (Esteller 2022). This study examined the role of MALNC in regulating transcriptomic response to all-trans retinoic acid (ATRA)–induced differentiation in acute promyelocytic leukemia (APL).
 Methods: A publicly available RNA-seq dataset GSE299161, originally designed to investigate the independent and ATRA-interactive effects of MALNC deletion on global gene expression, was reanalyzed. The dataset included RNA-seq data from two CRISPR-Cas9 knockout conditions, namely KOA (targeting MALNC exons 1.1 and 1.2), and KOB (extended deletion of exon 1.0) as well as untreated wild-type (WT) controls. Additionally, KOB and WT NB4 APL cells were treated with 1 µM ATRA for 72 hours. Differential expression was assessed with R packages (FDR < 0.05; |log₂FC| > 0.2).
 Results: No significant differentially expressed genes (DEGs) were identified between WT and KOA or between WT and KOB under untreated conditions, suggesting that deletion of MALNC exons 1.0, 1.1, and 1.2 does not independently perturb baseline gene expression in NB4 APL cells. By contrast, 72-hour ATRA exposure reprogrammed the transcriptome, generating 5,992 DEGs (2,973 up- and 3,019 downregulated genes) in WT and 5,202 ((2,670 up- and 2,732 downregulated)) in KOB cells. Among these, 2,907 transcripts (1,371 up- and 1,536 downregulated) showed significantly different log₂ fold changes between KOB and WT, demonstrating that MALNC modulates the magnitude of response to ATRA for a substantial subset of genes. A KOB-versus-WT comparison under ATRA uncovered 2,857 DEGs (1,463 up- and 1,394 downregulated); within this set, 237 genes (113 up-and 124 downregulated) were both ATRA-regulated and MALNC-dependent. This MALNC-sensitive cohort was enriched for canonical differentiation drivers (PML, ITGAM, CSF3R, ALDH1A2, SPI1, IRF1, GDF15, and TGM2, all markers of myeloid differentiation that are consistently induced during ATRA-mediated maturation of APL cells), while oncogenic or pro-survival transcripts (MYC, NPM1, BCL2, and POU2F2) were repressed, suggesting a MALNC-mediated amplification of pro-differentiation and anti-leukemic programs.
 Conclusions: MALNC deletion alone does not alter gene expression under steady-state conditions, indicating that it is dispensable for basal transcription, However, MALNC significantly influences the gene expression dynamics during ATRA-induced differentiation. These findings suggest that MALNC is a key modulator of retinoid-responsive gene networks and may serve as a potential therapeutic target or biomarker for treatment stratification in AML, particularly in subtypes exhibiting sensitivity to the retinoic acid pathway. Further functional studies are warranted to elucidate the molecular mechanisms by which MALNC interacts with ATRA signaling and its implications for overcoming therapeutic resistance in leukemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51b6e8550aab8996596e0130d08fe2b3a8f6f2ab" target='_blank'>
              Lncrna malnc modulates ATRA-induced transcriptional reprogramming in acute promyelocytic leukemia cells
              </a>
            </td>
          <td>
            Qian Wang, Kuan-Ying Yu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric cancer, with Hispanic/Latino (H/L) children in the United States exhibiting the highest incidence rates. Despite this, the genetic basis of this susceptibility remains poorly understood. Genome-wide association studies identified several well-replicated single-nucleotide polymorphisms (SNPs) associated with childhood B-ALL risk. Recent analyses discovered a novel childhood B-ALL risk signal near the IKZF1 gene that is positively selected for in H/L populations. The putatively causal variant, rs1451367 C>T, resides downstream of IKZF1 in a 3'cis-regulatory element (3'CRE) defined by histone marks and chromatin accessibility. In addition, an independent B-ALL risk variant, rs17133807 G>A, found across ancestries lies 26 bp away in the same regulatory element. However, how this regulatory element mechanistically impacts IKZF1 expression remains unknown.
 To unravel the molecular role of the 3'CRE, we integrated chromatin accessibility data, 3D chromatin interaction mapping, computational transcription factor (TF)-binding predictions, and functional genomic assays. ATAC-seq profiling from 156 B-ALL patients revealed a distinct and variable accessible chromatin landscape at this 3'CRE in B cell progenitors and precursors, suggestive of highly-regulated and stage-specific activity. Of note, in the presence of the risk variants, allele-specific chromatin accessibility was reduced, implicating decreased enhancer activity. Further, Promoter Capture Hi-C data from human lymphoblastoid cells revealed strong interaction with the IKZF1 promoter, supporting the CRE's role as a bona fide IKZF1 enhancer. Having established its accessibility and promoter interaction in early B cells, we next sought to identify the TFs occupying the 3'CRE and determine how their binding is altered by the risk allele.
 Employing motifbreakR and delta SVM modeling, we predicted that rs1451367 would disrupt binding of several Erythroblast Transformation Specific family members, including ERG, ETV6, ELK3, in addition to altering a potential IKZF1 motif. Prioritizing these TFs with a negative differential binding score, we then performed chromatin immunoprecipitation (ChIP) coupled with quantitative real-time PCR (qPCR) to examine differential TF binding. To model the risk variants at this locus, we used CRISPR/Cas9 genome editing to introduce a 60 bp microdeletion spanning the 3'CRE variants in SEM cells, a human B-ALL cell line with a KMT2A-AFF1 translocation, in which we confirmed the CRE harbors accessible chromatin. ChIP qPCR revealed strong ERG occupancy at the 3'CRE, which was abolished in the presence of microdeletions of the risk variants. These findings implicate ERG as a key participant in orchestrating IKZF1 expression and demonstrate that the risk variant disrupts this interaction. Intriguingly, Nalm-6 cells, which harbor an IGH-DUX4 rearrangement, displayed a strikingly different pattern, with an absence of ERG binding at the 3'CRE. Western blotting detected an additional ERG isoform in these cells, which we identified as ERGalt – a dominant-negative isoform generated through alternative transcription driven by IGH-DUX4. ChIP across multiple B-ALL lines confirmed distinct binding profiles, underscoring the context-dependent regulation of this enhancer by ERG across leukemic subtypes. Notably, as IKZF1 itself is known to regulate ERG, our findings suggest a previously unrecognized feedback mechanism that might occur among these TFs. Also, we found the adjacent multi-ancestry risk variant to disrupt a TCF3 motif, with ongoing studies assessing its impact on IKZF1 transcription. Thus, to more finely dissect precise regulation at this element, we are conducting systematic mutagenesis studies at this CRE.
 Collectively, this work identifies a mechanistic model in which the Hispanic-specific risk variant disrupts ERG-mediated regulation of IKZF1, stalling B-cell differentiation at an early stage and priming cells for leukemic transformation. By identifying ERG as a novel transcriptional regulator at the IKZF1 3'CRE, our work reveals a previously unrecognized regulatory intersection between two key B-cell transcription factors. These insights not only elucidate how population-specific noncoding variation contributes to B-ALL susceptibility, but also highlight a regulatory axis that could be leveraged for enhancer-targeted or TF-modulating therapeutic interventions in the future.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/606e694bb6a2d590187674c0419d978017077562" target='_blank'>
              Dissecting mechanisms at an IKZF1 cis-regulatory element harboring ancestry-specific and cross-ancestry B-cell acute lymphoblastic leukemia risk variation
              </a>
            </td>
          <td>
            L. Messa, Lara Wahlster, Andrew Lee, Pardiss Mehrzad, Mateusz Antoszewski, Anna-Lena Neehus, , Alexis Caulier, Gaurav Agarwal, A. D. de Smith, Jonathan Paolino, Yana Pikman, Katia Georgopoulos, V. Sankaran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) exhibits extensive cell state plasticity that constitutes a primary mechanism of therapeutic resistance and disease progression. Despite recognition that epigenetic regulators drive this plasticity, the critical molecular mediators remain inadequately characterized, limiting the development of targeted therapeutic interventions. This study employed GBM patient-derived xenograft cell lines, CRISPR-mediated genetic modification, RNA sequencing, and comprehensive in vitro and in vivo modeling to elucidate the role of BRD2 in regulating NF-κB-mediated mesenchymal transition. Our investigation revealed that PTEN loss induces RelA chromatin localization and acetylation, subsequently recruiting BRD2 to mesenchymal gene promoters where it functions as an essential regulator of gene expression and phenotype maintenance. Genetic deletion or pharmacological inhibition of BRD2 bromodomains effectively reversed mesenchymal transition, enhanced radiation sensitivity, and significantly improved survival outcomes in preclinical models. Treatment with a brain-penetrant BD2-selective inhibitor demonstrated suppression of mesenchymal gene expression, increased radiosensitivity of glioblastoma stem cells, reduced invasion capacity, and decreased tumor-associated macrophage infiltration in vivo. These findings establish BRD2 as a central mediator of mesenchymal transition in GBM, with its bromodomains serving as critical drivers of cell state plasticity through chromatin regulation. The therapeutic targeting of BRD2 with BD2-selective inhibitors represents a promising strategy to overcome treatment resistance and improve patient outcomes. This work provides mechanistic insight into the epigenetic control of GBM plasticity and identifies a clinically actionable target for combination with standard therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f387cb1ff663645d0f0a134f2781d6e0bf20ba" target='_blank'>
              EXTH-08. BRD2 bromodomain-mediated regulation of cell state plasticity modulates therapy response in glioblastoma
              </a>
            </td>
          <td>
            Raghavendra Vadla, Brett Taylor, Shunichiro Miki, Yohei Miyake, Benjamin Lin, Daisuke Kawauchi, Brandon Jones, D. Nathanson, C. Miller, Frank B Furnari
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Cellular senescence is a dynamic cancer cell condition with both tumor-suppressive and relapse-promoting features. While terminating cell proliferation, it may sustain disease persistence through its senescence-associated secretory phenotype (SASP), immune alteration, and stem-like reprogramming. Here, we identify therapy-induced senescence (TIS) as a unifying, plastic state across newly diagnosed (nd), genetically diverse non-M3 acute myeloid leukemias (AML) that reprograms blasts towards an acute promyelocytic leukemia (APL)-like phenotype with distinct plasticity-related therapeutic vulnerabilities.
 We established a short-term ex vivo chemotherapy assay to quantify the patient-individual TIS capacity of primary AML blasts by fluorescent senescence-associated β-galactosidase activity, further characterized by p16INK4a expression, Ki67 loss, and H3K9me3 heterochromatin remodeling. Like daunorubicin or cytarabin, hypomethylating agents, hydroxyurea, and anti-CD33 antibody-drug conjugates also triggered TIS to varying extents, showing that diverse agents converge on senescence and prime AML cells for potential senolytic elimination.
 Probing baseline transcriptomes of nd AML samples by an AML TIS-high signature of the top-100 differentially expressed genes failed to stratify outcomes across TCGA (n=172), OHSU Beat AML (n=405), and MLL Munich Leukemia Laboratory (n=433) cohorts. Unexpectedly, a 13-gene subclassifier, determined by unsupervised cluster analysis of the top-100 genes and unrelated to the M3-typical t(15;17) PML::RARA translocation, identified 65 of 67 genetically defined APL cases across these nearly 1,000 all-subtype AML transcriptomes, suggesting that TIS rendered non-M3 AML APL-like. Multi-omic analyses including bulk and single-cell RNA sequencing plus PRC2/SUZ12 and H3K27me3 ChIP-seq unveiled TIS-related loss of polycomb repression, epigenetic remodeling, and transcriptional reprogramming, yet independent of PML::RARA, as the underlying molecular mechanism. Functionally, TIS-associated APL-like plasticity conferred sensitivity to differentiation therapies such as all-trans retinoic acid (ATRA) and histone deacetylase inhibitors, and to Bcl2-targeting agents with senolytic activity. Patient-derived xenograft models validated these vulnerabilities as sequential TIS followed by senescence targeting achieved durable control exclusively in models able to mount a senescence response.
 Given the biologically unifying and ATRA-sensitizing role of TIS-associated M3-like plasticity, we assessed its clinical relevance in three independent cohorts totaling 92 newly diagnosed patients. TIS capacity at diagnosis robustly stratified outcomes, predicted superior disease-free and overall survival, and was associated with favorable ELN classification, normal cytogenetics, and NPM1 or DNMT3A mutations, while being lowest in adverse-risk SRSF2-mutant AML.
 Our findings uncover TIS-related M3-like plasticity as a novel state exploitable by a two-punch strategy of senesence induction first followed by targeted interventions through pro-differentiation or senolytic approaches, establishing a new therapeutic paradigm with immediate clinical potential and broader relevance to other malignancies where therapy-induced plasticity dictates outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/530f7791e9ce429389bda61c51512e84812d995e" target='_blank'>
              Therapy-induced senescence reprograms non-M3 AML into a drug-exploitable APL-like state
              </a>
            </td>
          <td>
            Martin Schönlein, W. Walter, Paulina Richter-Pechanska, Katharina Barbara Rösel, Diego Gilioli, Zhiting Shao, Jana Ihlow, Karsten Spiekermann, Katharina S. S. Götze, Soyoung Lee, Yong Yu, T. Haferlach, Hugues De Thé, R. Di Micco, Clemens A. Schmitt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The abnormal expression of the HOXA genes is intricately associated with the pathogenesis and progression of acute myeloid leukemia (AML), as these genes are crucial in regulating cellular differentiation and proliferation during normal hematopoiesis. In normal hematopoiesis, the expression of HOXA genes is meticulously regulated by the interplay between activation by the KMT2A complex or CDX and repression by the PRC2 complex. AML disrupts this balance, resulting in the persistent activation of HOXA. They perpetuate a leukemic state by directly activating a network of downstream target genes, which facilitate proliferation, hinder differentiation, and inhibit apoptosis. Genetically, this dysregulation can be classified as KMT2A-dependent, as shown in KMT2A rearrangements, NPM1 mutations, NUP98 rearrangements and other type of acute myeloid leukemia. Menin inhibitors may be employed for the dysregulation of HOXA genes reliant on them. Although menin inhibitors have considerable therapeutic efficacy, issues such as drug resistance and particular toxicity still require resolution. Diverse targeted therapeutics may be accessible for the dysregulation of non-KMT2A-dependent HOXA genes, contingent upon the specific genetic alterations. Therefore, a comprehensive understanding of the HOXA signaling network is vital for advancing targeted AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4881e7925db0d903cefbc766f04b1e7269c07b" target='_blank'>
              Dysregulation of HOXA genes in acute myeloid leukemia and targeted therapy
              </a>
            </td>
          <td>
            Wusixian Huang, Fenghong Zhang, Zhiyu Zhang, Qinrong Wang, Suning Chen
          </td>
          <td>2025-12-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality among women, with hormone receptor-positive (HR + ) disease representing the majority of cases. Despite advances in targeted therapies and antibody-drug conjugates, resistance to treatment remains a significant challenge, necessitating innovative therapeutic strategies. Epigenetic dysregulation, particularly involving histone acetylation, plays a crucial role in tumorigenesis and therapy resistance. Lysine acetyltransferase 6 (KAT6A/B), key regulators of chromatin structure and gene expression, have emerged as promising therapeutic targets in HR+ breast cancer. This review highlights the molecular mechanisms by which KAT6A/B modulate histone acetylation and gene regulation, emphasizing their roles in oncogenic pathways and cellular processes such as transcriptional activation, chromatin remodeling, and regulation of cellular senescence. We discuss preclinical evidence supporting the therapeutic potential of KAT6 inhibition and the challenges faced in epigenetic drug development, including the need for robust biomarker-driven approaches. Finally, we explore the prospects for integrating KAT6 inhibitors with existing therapies to overcome resistance and improve patient outcomes in breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882962d718736529869b6bb59db5c43e6a4e7b49" target='_blank'>
              Exploring KAT6 as a therapeutic target in breast cancer: epigenetic approaches for precision medicine
              </a>
            </td>
          <td>
            Cyril Roussel-Simonin, Aranzazu Fernandez-Martinez, S. Postel-Vinay, F. Mosele, Alessandro A. Viansone, C. Massard, Adrien Mouren, Julien Vibert, B. Pistilli
          </td>
          <td>2025-12-19</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gastrointestinal (GI) cancers pose a significant global health burden, driven by complex molecular alterations and microenvironmental interactions. Advances in molecular pathogenesis have elucidated recurrent driver gene mutations, such as KRAS, TP53, and APC, alongside dysregulated signaling pathways including Wnt, RAS-MAPK, and PI3K-AKT, which collectively underpin tumor initiation and progression. Complementing genetic changes, epigenetic alterations—such as DNA hypermethylation, histone modifications, and regulatory non-coding RNAs—further contribute to malignant evolution by reshaping chromatin architecture and gene expression. These mechanisms not only promote uncontrolled proliferation but also reinforce therapeutic resistance by dynamically modifying the tumor microenvironment (TME). Molecular subtyping efforts, including The Cancer Genome Atlas (TCGA) classification for gastric cancer (GC) and the Consensus Molecular Subtypes (CMS) for colorectal cancer (CRC), have delineated disease heterogeneity, revealing distinct pathogenic pathways and enabling refined prognostic stratification. Such insights provide the biological rationale for diagnostic techniques and targeted interventions. For instance, anti-EGFR and anti-VEGF monoclonal antibodies disrupt oncogenic signaling and tumor angiogenesis, respectively, and have demonstrated substantial clinical efficacy in selected patient populations. In parallel, immunotherapy has emerged as a transformative modality in oncology. Immune checkpoint inhibitors targeting PD-1/PD-L1 and CTLA-4 reinvigorate antitumor immunity and have reshaped standard-of-care protocols for several GI malignancies. Beyond conventional immunotherapies, innovative strategies such as CAR-T cell therapy and neoantigen-based vaccines are being actively investigated. These approaches aim to overcome immune evasion mechanisms and enhance tumor-specific targeting, offering promise for patients with resistant or advanced disease. This review comprehensively analyzes the evolving molecular landscape of GI cancers and the corresponding development of targeted and immunotherapeutic agents. It highlights a balanced integration of mechanistic discovery and clinical translation, underscoring their synergistic roles in advancing precision oncology and improving survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5602f60be23d19b8d126a3c7ae3c86407dda38e" target='_blank'>
              Gastrointestinal cancer: molecular pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Yang Jin, Xiaobo He, Yanfeng Wu
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. Although immune checkpoint inhibitors (ICIs) have achieved striking clinical efficacy in the subset of CRCs with mismatch repair deficiency/high microsatellite instability (dMMR/MSI-H), the vast majority of patients—those with proficient mismatch repair/microsatellite-stable (pMMR/MSS) tumors—derive little benefit from current immunotherapies. Ferroptosis, an iron-dependent form of regulated cell death driven by lethal accumulation of lipid peroxides, has emerged as a promising antitumor mechanism that can interact with and modulate antitumor immunity. Concurrently, the gut microbiota exerts powerful control over host metabolism and immune tone through microbial community structure and metabolite production; accumulating evidence indicates that microbiota-derived factors can either sensitize tumors to ferroptosis (for example, via short-chain fatty acids) or confer resistance (for example, indole-3-acrylic acid produced by Peptostreptococcus anaerobius acting through the AHR→ALDH1A3→FSP1/CoQ axis). In this review we synthesize mechanistic data linking microbial ecology, iron and lipid metabolism, and immune regulation to ferroptotic vulnerability in CRC. We discuss translational strategies to exploit this “microbiota–ferroptosis” axis—including precision microbiome modulation, dietary interventions, pharmacologic ferroptosis inducers, and tumor-targeted delivery systems—and we outline biomarker frameworks and trial designs to evaluate combinations with ICIs. We also highlight major challenges, such as interindividual microbiome variability, potential collateral harm to ferroptosis-sensitive immune cells, adaptive antioxidant compensation (e.g., NRF2/FSP1 activation), and safety/regulatory issues for live biotherapeutics. In summary, this review highlights that targeting the microbiota-ferroptosis axis may represent a rational and potentially transformative approach to reprogramming the tumor microenvironment and overcoming immune checkpoint resistance in pMMR/MSS colorectal cancer; however, further research is essential to validate this concept and address existing challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192d7ffc58922891ac356bc9874a4e2b421c66ae" target='_blank'>
              Gut Microbiota and Ferroptosis in Colorectal Cancer: A Comprehensive Review of Mechanisms and Therapeutic Strategies to Overcome Immune Checkpoint Resistance
              </a>
            </td>
          <td>
            Yingchang Cai, Feng Zhao, Xiaofei Cheng
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Malignant tumors developing in the conditions of chronic hypoxia, unlike normal tissues, quickly acquire resistance to decreased oxygen level. Hypoxia-resistant cells in malignant disease obtain various features promoting their survival: they alter metabolism to aerobic glycolysis, activate angiogenesis-stimulating programs, and rearrange signaling cascades to adapt to hypoxia. At the same time, tumor cells become resistant to chemotherapy and radiation therapy and effectively colonize metastatic niches. 
This review analyzes signaling pathways which activate in the conditions of chronic hypoxia and underlie tumor adaptation to insufficient oxygen. Apart from the main pathway associated with activation of hypoxia-inducible factor 1α (HIF-1α), such cascades as STAT3, Snail cascade associated with epithelial-mesenchymal transition, NRF2 factor responsible for adaptation to reactive oxygen species, and stemness factors OCT4, SOX2 and NANOG are also considered. Effective killing of tumor cells requires simultaneous inhibition of several parts of signaling pathways because disruption of just one will lead to switching of signaling proteins to bypass the blocked one. The article lists various inhibitors of signaling pathways patricipating in tumor cell adaptation to chronic hypoxia and describes their mechanisms of action and targets. Development of successful strategies for treatment of hypoxia-resistant tumors requires identification of an effective combination of the above-mentioned and other inhibitors of the key signaling cascades participating in adaptation to hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21710e96836bbca03c72f6e2b505d714f5b359ca" target='_blank'>
              Rearrangement of signaling pathways and adaptation of tumor cells to hypoxia
              </a>
            </td>
          <td>
            O. E. Andreeva, E. I. Mikhaevich, M. A. Krasil’nikov
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Osteosarcoma arises within heterogeneous tumor–immune ecosystems in which impaired antigen visibility—shaped by chromatin programs—limits immune surveillance and blunts responses to immunotherapy. Beyond structural defects in the antigen-processing pathway, Polycomb-mediated repression, DNA hypermethylation, and state-specific enhancer closure converge on the HLA class I/NLRC5/interferon axis to diminish peptide display. These constraints are context dependent, varying across malignant clones, differentiation states, and myeloid and T-cell niches. Traditional bulk assays obscure this complexity; single-cell ATAC-seq, integrated with single-cell and spatial transcriptomics, now resolves promoter–enhancer accessibility at HLA, NLRC5, and antigen-processing genes, distinguishes reversible repression from fixed lesions, and links microenvironmental stress to interferon competence. Translationally, epigenetic reprogramming—targeting Polycomb repressive complex 2 (PRC2), DNA methyltransferases (DNMTs), and complementary regulators (for example, LSD1, BET, CDK4/6, YAP/TEAD)—offers biomarker-guided avenues to restore antigen presentation, provided ecosystem-aware pharmacodynamic readouts track chromatin opening and antigen-presentation recovery across compartments. Despite encouraging preclinical evidence, efficacy will depend on clone selection, scheduling that preserves interferon signaling, and rational combinations with innate agonists and checkpoint blockade. This mini-review synthesizes epigenetic mechanisms of antigen-presentation failure in osteosarcoma and outlines how single-cell chromatin profiling can guide strategies to reinstate tumor antigen visibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51ccd00bee57595f98289280452a3ec7548b15" target='_blank'>
              Epigenetic control of antigen presentation failure in osteosarcoma: from single-cell chromatin maps to therapeutic strategies
              </a>
            </td>
          <td>
            Yan He, Heng Wu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Mature T-cell lymphomas (TCL) are a heterogeneous group of non-Hodgkin lymphomas arising from the malignant transformation of post-thymic T and NK cells. These lymphomas are characterized by poor response to chemotherapy, high relapse rates, and dismal survival outcomes. Genomic analyses of TCL have revealed recurrent alterations affecting intracellular signaling pathways and epigenetic regulators. The frequent involvement of epigenetic modifiers suggests that epigenetic dysregulation is a critical step in TCL pathogenesis.
 To identify the epigenetic changes driving TCL transformation, we analyzed chromatin modifications and gene expression profiles in murine TCL models that closely recapitulate human disease. Our data revealed that Id2, a repressor of basic helix-loop-helix proteins, is consistently overexpressed at both RNA and protein levels across multiple murine lymphoma models. Supporting this, analysis of the Cancer Cell Encyclopedia identified TCL as one of the tumor types with the highest ID2 expression, which we confirmed by Western blot in human TCL-derived cell lines. Furthermore, analysis of single-nuclei RNAseq data from a cohort of primary angioimmunoblastic T-cell lymphoma (AITL) and peripheral T-cell lymphoma not otherwise specified (PTCL, NOS) patient samples generated in our laboratory, demonstrated significant ID2 overexpression in a subset of cases (12/27; 44%) compared to tonsil and lymph node controls. Interestingly, all relapsed TCL cases in our cohort showed significant overexpression of ID2. Based on our data, we hypothesize that ID2 is epigenetically regulated during transformation and contributes to T-cell lymphomagenesis.
 To define the mechanisms leading to ID2 overexpression in TCL, we performed H3K27ac and H3K4me1 ChIP-seq, ATAC-seq, reverse-ChIP coupled with mass spectrometry, and dual-luciferase reporter assays. These analyses identified a conserved region near the ID2 locus enriched for H3K27ac and H3K4me1 marks that co-localizes with ATAC-seq peaks and is ranked as a super-enhancer by the ROSE algorithm. This region is specifically activated in mature TCL cells compared to control CD4⁺ T cells. Reporter assays using the most conserved segment within the enhancer region showed more than twofold increased luciferase activity relative to a minimal promoter, confirming its enhancer function. Interestingly, reverse-ChIP identified CTCF and ASCL1 as candidate regulators of enhancer looping and transcriptional activation, respectively. To characterize the proteins mediating the epigenetic function of ID2 in TCL, we employed a biotin ligase proximity labeling assay (BioID) approach coupled with mass spectrometry. Our results identified known ID2 partners, including the E-proteins TCF3, TCF4 and TCF12, key regulators of T-cell differentiation, as well as novel partners including elements of the SWI/SNF chromatin-remodeling complex, suggesting that ID2 overexpression indirectly regulates transcriptional programs critical for lymphomagenesis.
 To determine the functional role of ID2 in TCL, we characterized the effects of both genetic and pharmacological inhibition of ID2, using shRNA and CRISPR-Cas9 mediated knockout and novel pan-ID inhibitors AGX51 and AGXA, respectively. Loss of ID2 leads to decreased cell viability and increased apoptosis in in vitro TCL models. Interestingly, RNA-seq analysis revealed enrichment in signatures associated with glucose metabolism, suggesting a novel role for ID2 in tumor metabolic regulation. Finally, in vivo treatment with the pan-ID inhibitor AGX51 in a murine lymphoma model induced a modest but consistent reduction in tumor burden, spleen weight, and activated CD69⁺ T cells population in AGX51-treated mice compared to vehicle-treated controls.
 In summary, we have identified a novel TCL-specific super-enhancer that drives ID2 overexpression in TCL and demonstrated an essential role for ID2 in supporting lymphoma proliferation and survival. More importantly, pharmacological targeting of ID2 reduces tumor burden in vitro and in vivo, highlighting its potential as a therapeutic target in mature T-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31f2dc0fbeb09da6f80dbe4eff065e3af3490a0" target='_blank'>
              Inhibitor of DNA binding 2 (ID2) is epigenetically regulated during lymphomagenesis and constitutes a novel therapeutic target in mature T-cell lymphoma
              </a>
            </td>
          <td>
            Anouchka P. Laurent, Hannah Miller, Ryan D. Najac, R. Albero, Jose R. Cortés, Cindy Ma, W. Lin, Anqi Wang, Ruth Alonso Alonso, L. Quevedo, Rajesh Soni, O. Ouerfelli, Raul Rabadan, R. Benezra, Teresa Palomero
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a heterogeneous tumor, ranging from cases with spontaneous regression (MS stage) to high risk (HR) tumors. Resistance to therapy presents a major challenge in the treatment and management of HR-NB, contributing to a poor prognosis. In these patients, the resistance to conventional treatment is also related to non-genetic features, such as the cellular plasticity. Novel studies have demonstrated the presence of two distinct cell phenotypes: adrenergic (ADRN) and mesenchymal (MES), which reflect the heterogeneity of NB. MicroRNAs (miRNAs) are small, endogenous and non-coding RNAs with the ability to regulate gene expression and may have a crucial role in controlling cell plasticity. However, the role of miRNAs in NB plasticity has not been investigated yet. We investigated miRNA signature in NB cells subtypes (MES and ADRN) and in the extracellular vesicles (EVs) released by them, to identify potential MES related biomarkers. Differentially expressed miRNAs were identified by RT-qPCR and subjected to gene ontology, KEGG pathway, and protein-protein interaction network analyses. Candidate miRNAs were validated in plasma-derived EVs from NB patients. We identified miR-199a-3p as strongly upregulated in the MES cells subtype. Moreover, its expression levels were significantly higher in primary cell lines derived from HR patients compared to low-risk (LR) ones. This was confirmed by a bioinformatics analysis in patient tissue obtained from TARGET NB dataset. Protein-protein interaction analysis uncovered a complex network, with FN1, CD44, and YAP1, identified as key genes upregulated by miR-199a-3p, all of which are closely associated with the MES phenotype. Among the miRNAs significantly upregulated in EVs derived from MES cell lines, miR-584a-5p was significantly higher in EVs isolated from plasma of HR and L/Intermediate(I)R patients compared to MS. MiR-584-5p is typically considered a tumor suppressor; to support this role miR-584a-3p resulted significantly upregulated in L/IR tumor tissue, both in the INSS and COG classifications (TARGET NB database). Our findings identified specific miRNAs as MES phenotype related biomarkers. Further studies should investigate the potential impact of miRNAs on plasticity-related pathways in order to open new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b8406e9d606c161b955f6866c60bbec146aba97" target='_blank'>
              Identification of a miRNAs signature as potential biomarker of mesenchymal phenotype in neuroblastoma patients
              </a>
            </td>
          <td>
            Silvia Lampis, Alessandro Paolini, Virginia Di Paolo, A. Galardi, Salvatore Raieli, E. Miele, Lauriane Lemelle, F. Fabozzi, A. Serra, A. Mastronuzzi, M. A. De Ioris, Andrea Masotti, Franco Locatelli, A. Di Giannatale
          </td>
          <td>2025-11-26</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Glucose metabolic reprogramming is a hallmark of cancer, best exemplified by the Warburg effect. It plays a central role in driving tumour growth, metastasis and resistance to therapy.



 This review systematically delineates the molecular underpinnings of this metabolic shift, encompassing the roles of key glycolytic enzymes, transporters, oncogenic signalling pathways and multilayered epigenetic regulation. We further propose a novel framework that conceptualizes glucose metabolism as an integrated system for signal transduction and niche engineering, which remodels the tumour microenvironment to promote immunosuppression.



 Finally, we highlight the translational applications of these insights, including metabolic imaging for diagnosis and prognosis and therapeutic strategies targeting glycolytic pathways, applied both as monotherapies and in rational combinations with conventional and emerging treatments.



 Targeting glucose metabolic reprogramming offers a promising perspective and novel strategies for cancer diagnosis and therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a515096593d2a091667e6c53ad2d9541a5e5359" target='_blank'>
              Mechanisms, impacts and therapeutic strategies of glucose metabolic reprogramming in tumours
              </a>
            </td>
          <td>
            Li Ma, Ming-Yang Wu, Cui-Cui Tian, Hao-Ze Shi, Dan Song
          </td>
          <td>2025-12-01</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-throughput sequencing (HTS) has revolutionized tumor immunology by enabling precise dissection of tumor-immune interactions, directly informing the development of precision immunotherapies. This review highlights key advances in HTS technologies—including whole genome sequencing (WGS), RNA sequencing, assay for transposase-accessible chromatin using sequencing, and single-cell immunogenomics (scTCR-seq/scBCR-seq)—and their clinical translation in personalized cancer vaccines, engineered T-cell therapies, and combination regimens. We discuss how these tools decode tumor-specific mutations, immune evasion mechanisms, and therapeutic targets, while addressing challenges in data standardization, sample processing, and computational integration. Emerging breakthroughs such as spatial multiomics, real-time monitoring, and artificial intelligence-driven discovery are transforming the field by enabling dynamic, personalized treatment strategies. Finally, we outline future directions to overcome current barriers and expand equitable access to HTS-driven precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1666fd68dbba4d54d4ecfda7e481cfc47c3e80e1" target='_blank'>
              High-throughput sequencing unveils tumor-immune interactions: From genomic alterations to clinical translation
              </a>
            </td>
          <td>
            Ling Yin
          </td>
          <td>2025-11-07</td>
          <td>Global Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Despite major advances in molecularly targeted therapies for hematologic malignancies such as CML and malignant lymphoma, effective treatment options for acute myeloid leukemia (AML) remain limited. Drugs such as FLT3 inhibitors, DNA methyltransferase inhibitors, and BCL2 inhibitors have been introduced, yet relapse remains a major challenge. To improve outcomes, identifying novel molecular targets and developing corresponding therapies is crucial. Emerging evidence highlights the critical role of “leukemia stem cells” (LSCs), a small but resilient subpopulation, in disease relapse. Our previous studies and others have shown that AML LSCs are heterogeneous and dynamic (Stem Cells, 2021). This heterogeneity likely underpins their survival advantage under chemotherapeutic stress. A comprehensive understanding of LSC biology is therefore essential for developing targeted therapies.
 In this study, we aimed to identify biomarkers that reflect the biological status of AML LSCs, and to explore strategies for modulating their activity. Using in-house screening methods, we identified multiple molecules implicated in hematopoietic regulation. One such molecule is endothelial cell-selective adhesion molecule (ESAM), a membrane protein primarily associated with the vascular endothelium. We previously reported that ESAM is expressed on normal hematopoietic stem cells (HSCs) (Blood 2009, Exp Hematol 2016) and serves as a marker of HSC activation in response to chemotherapeutic stress (J Immunol 2012, PLoS One 2016, J Exp Med 2021). Notably, ESAM is expressed in AML but not in lymphoblastic leukemia, and its expression differentiates proliferative from quiescent AML clones (Stem Cells 2022). These findings suggest that ESAM may serve as a functional biomarker of AML LSCs and a potential therapeutic target.
 To investigate the role of ESAM in AML LSCs, we generated human AML cell lines expressing ESAM. The construct encoded ESAM fused at the C-terminus with AirID, a proximity biotinylation enzyme, enabling downstream interactome analysis. Among the transduced lines, we selected MOLM-14 cells, originally derived from relapsed AML with FLT3-ITD and lacking endogenous ESAM expression, for further study. Under normal culture conditions, ESAM expression did not alter proliferation compared to parental or AirID-only cells. However, upon treatment with FLT3 inhibitors (CEP701 and quizartinib), ESAM-expressing cells retained heigh drug sensitivity. Then, we performed RNA sequencing on MOLM-14 ESAM-AirID versus AirID-only cells. This analysis revealed marked transcriptomic changes: 680 genes were upregulated and 622 genes were downregulated (>3-fold change). Upregulated genes included PPP1R10, ZNF236, and P2RY6, all of which are associated with cell growth and stress responses. Downregulated genes included HSD17B8 and NTAN1, which are involved in steroid metabolism and protein degradation, respectively. Pathway analysis indicated that ESAM expression enhances signaling through G protein-coupled receptors and suppresses apoptosis-related pathways. These data support the hypothesis that ESAM promotes cellular activity of cancer stem-like cels in AML. Of interest, ESAM expression significantly upregulated MS4A3, recently identified as a regulator of LSC reactivation and drug sensitivity (Blood 2022). This further reinforces the functional relevance of ESAM in AML biology.
 We next performed proximity-dependent biotinylation using the AirID system. Mass spectrometry identified 1,511 biotinylated proteins, including tight junction proteins ZO-1 and ZO-2 (encoded by TJP1 and TJP2), both of which contain PDZ domains that likely interact with the PDZ-binding motif of ESAM. Additionally, membrane-associated proteins such as ZDHHC17, CNST, RASA3, and ATIC were detected in high abundance. These proteins are known to participate in diverse signaling and transport functions, suggesting that ESAM may influence multiple biological pathways through its intracellular interactions.
 In summary, our findings demonstrate that ESAM expression alters the transcriptomic and functional landscape of AML cells. ESAM enhances biological activity of AML LSCs, likely through modulation of signaling pathways and interactions with junctional and membrane-associated proteins. These findings identify ESAM as a key regulator of LSC behavior and a potential therapeutic target in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4989b20e5c19ac4bbabf5c0f3775725851f38c0" target='_blank'>
              ESAM is functionally involved in the activity of cancer stem-like cells in human acute myeloid leukemia
              </a>
            </td>
          <td>
            Henyun Shi, Yosuke Matsuoka, Jun Nakayama, Yusuke Imagawa, Toru Hiratsuka, Hidenori Kasahara, Shigeki Higashiyama, Takafumi Yokota
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Epigenetic deregulation is a major driver for acute myeloid leukemia (AML) including those with Mixed-Lineage Leukemia gene-rearrangement (MLLr). Over the past decades, significant efforts have been made by others and us to identify and target key components associated with oncogenic transcriptional complexes for development of more effective treatments. Among the inhibitors targeting components of MLL complexes, MENIN and DOT1L have successfully made to the clinics. Although promising results from initial clinical trials demonstrate their safeness and potential efficacy, the complete remission rates remained low (<10% for DOT1Li and 33% for MENINi) and short-lived with only 25% of AML patients treated with MENINi being MRD free in the 2 years follow up, urging a better understanding of the underlying mechanisms and the development of effective combination therapies. Strikingly, while suppression of key MLL fusion targets such as Hoxa and Meis1 by these epigenetic inhibitors could be detected in most patients, their downregulation did not strictly correlate with or predict treatment response, suggesting that these inhibitors may also act by different mechanisms. To this end, our study aims to better understand the leukemia suppressive functions of DOT1L/MENIN inhibitors and thereby facilitate the rational development of more effective treatments.
 To identify key pathways commonly affected by inhibitors targeting DOT1L/MENIN complexes, we performed RNA-seq on both primary MLL-AF9 AML derived from mouse hematopoietic stem/progenitor cells (HSPCs) and human MLL-r AML cell line MV4;11 with or without DOT1Li (EPZ004777) or MENINi (MI-503). In addition to the classical Hoxa/Meis1 pathways, both DOT1L and MENIN inhibitions exhibited significant suppression of pathways associated with DNA replication, closely linked to DNA damage and leukemia cell survival. Upon replication stress, two major pathways, namely fork reversal and repriming, can be used to potentially overcome obstacles to DNA synthesis. Fork reversal is associated with slower fork speed and involves key DNA replication/repair proteins such as PARP to stabilize reversed forks during pausing. Conversely repriming requires Primpol to bypass obstacles without slowing fork progression. To assess the impact of DOT1Li and MENINi on DNA replication fork stability and dynamics, DNA fibre assays showed that MLL fusion transformed cells heavily rely on repriming for overcoming replication stress. Strikingly, upon DOT1Li/MENINi treatment, MLL fusion transformed cells were forced to switch to fork reversal resulting in slower fork speed, which may in part account for the inhibitory effects. To further confirm this, we generated a novel MLL-AF9 leukemia model using mouse HPSCs, where Primpol can be inducibly inactivated upon 4-OHT treatment. As a result, we show that inactivation of Primpol phenocopies DOT1Li/MENINi and could drive MLLr-AML to adapt fork reversal as a coping mechanism. Together these results reveal a significant impact of these epigenetic inhibitors on DNA replication fork dynamics and repair mechanisms.
 Switching from fork repriming to reversal by the epigenetic drugs can lead to a reliance on PARP proteins that stabilize reversed forks during the arrest and protect them from premature restart, thereby creating a novel vulnerability to PARP inhibition. To test this hypothesis, PARPi was used in combination with DOT1Li/MENINi in MLLr-AML. Strikingly, while PARPi alone had no significant impact on fork dynamics, it triggered premature fork restart in DOT1Li treated cells, leading to increase fork speed and subsequent fork collapse, revealing a potential vulnerability created by DOT1Li. Consistently, combination treatment significantly suppressed cell growth of MLLr-AML, which displayed significantly shorter fork length and excessive DNA damage as compared to control and single treatments. Finally, we also tested the efficacy of these novel regimens in vivo. While single treatments with PARPi, DOT1Li, or MENINi exhibited no or moderate effects on disease progression, combo treatments in particular MENINi+PARPi significantly prolonged the disease latency with all animals remaining disease-free at the experimental endpoint. Taken together, these data reveal that DOT1L/MENIN inhibition alter replication fork plasticity and induce a novel vulnerability that can be therapeutically targeted by PARP inhibition for effective leukemia treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/757b773a243504fb282db04d14daf965acdd6ee2" target='_blank'>
              DOT1L/menin inhibitors target replication fork plasticity and create novel vulnerability to PARP inhibitor in MLL-rearranged AML
              </a>
            </td>
          <td>
            T. Fung, Cyril Dördelmann, Danyue Peng, Ali Akin, Jennifer Lynch, Nga Ieng Priscilla Lau, B. Zeisig, Larissa Bomfim, Lu Zhao, Polina Maximchik, Massimo Lopes, Chi Wai Eric So
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Background: Multiple Myeloma (MM) is a plasma cell malignancy characterized by genomic instability, often manifesting as chromothripsis. Although CNVs in DNA repair genes are recurrent in MM, the combined effect of germline defects and somatic alterations in driving aggressive, therapy-resistant disease remains incompletely understood.
 Methods: We performed whole-genome sequencing (WGS) on tumor DNA from a patient with newly diagnosed (NDMM), progressive, treatment-resistant multiple myeloma and CNV gains across multiple chromosomes. This patient was from the original cohort of 17 patients we reported where we detected a possible correlation of copy number gains in DDR genes and aggressive myeloma.1 Analysis included detection of SNVs, indels, CNVs, structural variants (SVs), HRD scoring, mutational signatures, and pathway enrichment.
 Results: Germline mutations were identified in key DDR genes (RAD50, BRCA1, CHD1, SEMA4D), resulting in an HR-deficient phenotype (high HRD score/ high Genomic Instability). Variants in CYP2B6, CYP2D6, CYP3A5 indicated impaired drug metabolism. Somatic mutations in TRAF3andARID5B affected immune signaling and transcription. The genome showed near-triploidy (ploidy score: 2.98), high SV burden, and dominant CNV signatures, associated with whole-genome duplication and chromothripsis (or, chromoplexy).
 Mutational signatures included SBS1, SBS5, and DBS17, indicating age-related processes, DNA repair defects, and oxidative stress. A high MATH score along with local hypermutation events reflected tumor heterogeneity. SV analysis revealed extensive deletions, duplications, and complex rearrangements, especially on chromosomes 1 and 11. Widespread Alu element duplications and Alu-mediated rearrangements suggest non-allelic homologous recombination and L1 retrotransposition events.
 Pathway analysis revealed:DNA repair: HRR and NHEJ dysfunction; BRCAness without BRCA1/2 biallelic inactivation.Cell cycle: Impaired G2/M checkpoints.Immune signaling: TRAF3 mutation disrupted cytokine and interferon pathways.Gene transcription: TP53 pathway dysregulation.Metabolism: CYP variants affected drug/xenobiotic metabolism.Signal transduction: CHD1 mutation impacted nuclear receptor signaling.Developmental biology: SEMA4D and BRCA1 affected migration and differentiation.Hemostasis: F5 mutation altered fibrin clot formation, suggesting thrombotic risk, which is a known feature of myeloma.
 Discussion: Thibaud et al.2 found pathogenic germline variants in BRCA1/2 contributing to hereditary risk in 10 percent myeloma cases. Nazaryan-Petersen et al.3 showed germline chromothripsis driven by L1-Mediated Retrotransposition and Alu/Alu Homologous Recombination. Aoki et al.4 found hypomethylation of LINE elements correlated with progression and poorer prognosis of myeloma. They detected an association between common breakpoint loci at commonly lost regions with increased LINE-1 densities. Our findings of widespread Alu duplications in this case suggests the possibility that chromoplexy in myeloma may be induced by retrotransposon activity.
 This case illustrates a “two-hit” MM pathogenesis model: germline mediated HRD creates a permissive genomic background for catastrophic somatic events, including chromothripsis (or, chromoplexy) and immune gene disruption. Similar “two-hit” model involving tumor suppressor genes and oncogenes has been reported by Rodrigues et al.5 These findings emphasize the value of integrated germline-somatic multimodal analysis to identify high-risk disease biology and guide therapeutic strategies. Pawlyn et al.6 have suggested using PARP inhibitors in HRD-LOH MM, which correlated with high-risk disease in the relapsed MM (RRMM) setting.
 Our case suggests that such high-risk presentation can occur in newly diagnosed myeloma (NDMM) too and begs the question: should PARP inhibitors be evaluated for upfront induction treatment of such cases?
 References: Raychaudhuri J et al. Abstract 507/Transplantation and Cellular Therapy 30 2S (2024) S370-S398 Thibaud et al. Blood Cancer Discovery (2024);5: 428–41 Nazaryan-Petersen L et al. Hum Mutat. (2016) Apr;37(4):385-95 Aoki et al. Genome Medicine (2012), 4:101 Rodrigues et al. iScience, (2025) 28, 111620, January 17, Pawlyn C et al. Leukemia (2018) 32:1561–1566
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faab97a25ec8eddcce44ce6eb8cdc61a206c2acd" target='_blank'>
              Germline mediated HR deficiency, alu element duplication, and immune pathway alterations define aggressive, drug-resistant multiple myeloma
              </a>
            </td>
          <td>
            Anindyajit Banerjee, Sombuddharoy Bhowmick, S. Vasudevan, Anurag Mehta, N. Agrawal, V. Sheth, D. Bhurani, Neeraj Arora, Jyotishankar Raychaudhuri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7ea627495d26a3f7b3d82b351b84bb5f1b7889" target='_blank'>
              Genomics of Acute Myeloid Leukemia at Diagnosis and Remission
              </a>
            </td>
          <td>
            Kai Yu, Laura W. Dillon, Jesse M Tettero, Gege Gui, Rasha W Al-Ali, Michael R Grunwald, Elizabeth F Krakow, Elizabeth A. Griffiths, Alexandra Gomez-Arteaga, R. Vedula, M. Solh, A. Salhotra, N. Bejanyan, Lori S. Muffly, A. Jimenez-Jimenez, M. Drazer, Yi-Bin Chen, Aaron C. Logan, Reena V. Jayani-Kosarzycki, Sophia R Balderman, J. Blachly, Brian C Shaffer, L. Druhan, C. Yeung, Vanessa Kennedy, Amir T Fathi, H. Carraway, Sandeep Gurbuxani, M. Tjota, Farah Sahoo, Max Smith, Dylan Barfield, Jim Guo, James Han, Jason Hu, Heejon Jo, Vidya Kudlingar, Wilfred Li, Yutong Qiu, P. Sathyan, Sean Truong, S. Catreux, Sam Ng, Khai Luong, Yunjiao Zhu, Reem Bahr, Jamie Diemer, C. Ferrone, Allison J Getker, Jackie Bjerke, Stephanie Bo-Subait, Steven M. Devine, Bergetta Dietel, Gabrielle Giammarino, Emily Heying Chihak, Jianqun Kou, Emily Kolb, Danielle O'Donnell Vitali, Stephen R. Spellman, Brenna K Tesch, Jenny Vogel, Stephanie Waldvogel, Syreeta Weatherspoon, Jeffery J. Auletta, Christopher S. Hourigan
          </td>
          <td>2025-12-07</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Epigenetic regulation is a fundamental mechanism controlling gene expression and cellular function, primarily mediated through reversible modifications such as DNA methylation, histone acetylation, and chromatin remodeling. Dysregulation of critical epigenetic enzymes, including histone deacetylases (HDACs), DNA methyltransferases (DNMTs), and bromodomain and extraterminal domain (BET) proteins, has been closely associated with tumor initiation, progression, metastasis, immune evasion, and resistance to conventional therapies. Targeting these epigenetic regulators with small-molecule inhibitors or degraders has emerged as a promising therapeutic strategy, capable of reprogramming aberrant transcriptional networks and reshaping the tumor microenvironment. Beyond direct cytotoxic effects, epigenetic drugs have demonstrated the ability to enhance antitumor immunity by restoring antigen presentation, promoting immunogenic cell death, modulating cytokine profiles, and reversing local immune suppression. Recent preclinical and clinical studies have highlighted the potential of combining epigenetic therapies with immune checkpoint inhibitors to achieve synergistic antitumor responses and overcome resistance mechanisms. This review provides a comprehensive summary of the mechanisms of action, pharmacological characteristics, and clinical applications of epigenetic drugs, with a focus on innovative combination strategies and ongoing translational advancements. We also discuss future directions, emphasizing the need to improve drug specificity, minimize off-target effects, integrate personalized immunotherapeutic approaches, and identify predictive biomarkers to optimize patient selection and clinical outcomes. Overall, epigenetic therapy represents a versatile and evolving avenue for precision oncology with broad implications for tumor control and immunomodulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1142f91bb95f6983a1d418601e7e6fb2c4af66" target='_blank'>
              Epigenetic drugs in cancer therapy: mechanisms, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Chen Ma, Junkai Cheng, Jian Gu, Qin Wang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Cancer stem cells (CSCs), first identified in acute myeloid leukemia with a CD34+/CD38- phenotype, are hypothesized to sustain clonal survival through self-renewal and resistance to chemotherapy. In B-cell acute lymphoblastic leukemia (B-ALL), stem-like properties can be reacquired through plastic transitions among stem-like, hematogone-like, and naïve B-cell–like states. This plasticity provides an evolutionary route by which chemotherapy-resistant subpopulations re-establish stem-like compartments, contributing to disease relapse under therapy. Such fluidity complicates efforts to target the stem-like compartment directly and underscores the need to understand how chemotherapy reshapes these transitions to drive resistance. We analyzed flow cytometry data from bone marrow (N=63) and peripheral blood (N=46) samples of B-ALL patients at diagnosis, post-induction, and in remission (N=38). Four major cell states are defined from stem-like, progenitor, and differentiated: 1) CD34+/CD38-, 2) CD34+/CD38+, 3) CD34-/CD38+, and 4) CD34-/CD38-. Patient-specific, continuous-time Markov chain modeling quantified transition rates, Qij between state i and j. In remission, differentiation (Q31) consistently exceeded dedifferentiation (Q13), reflecting normal lineage progression. In relapse, however, Q31 – Q13 was often negative with greater variance, indicating that resistant clones rebuilt stem-like compartments through dedifferentiation. This heterogeneity underscores distinct evolutionary mechanisms of resistance across patients. We next explore evolution-based interventions based on our model. Promoting differentiation (increasing Q31) transiently reduced stem-like fractions but failed to prevent rebound due to ongoing dedifferentiation. Inhibiting dedifferentiation (reducing Q13) produced sustained depletion of CD34+/CD38- cells and more effectively shifted relapse dynamics toward remission-like equilibria. Thus, targeting the dedifferentiation fraction emerges as a promising therapeutic strategy. To link transition dynamics with clinical outcomes, we applied mechanistic learning by integrating Markov-derived features with machine learning. Linear discriminant analysis (LDA) achieved high predictive accuracy for BCR::ABL1 status (80%), disease status (84% marrow, 89% blood), and minimal residual disease (71%). Principal component analysis highlighted the balance of differentiation and dedifferentiation as the most informative predictor. These results show that resistance evolves not only from persistence of stem-like cells but also from reciprocal transitions that replenish this compartment. Our findings suggest that therapy pressure influences cell transition landscapes in B-ALL, with relapse often associated with dedifferentiation-driven restoration of stem-like states. Mechanistic learning highlights stem-state reciprocity (in/out rate balance) as a potential predictive biomarker, while modeling indicates that inhibiting dedifferentiation may be effective to reduce resistant stem-cell like clones.



 Sadegh Marzban, Curtis Gravenmier, Ling Zhang, Jeffrey West. Cell state transitions drive the evolution of disease progression in B-lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37463b573a65ae5e34a361dd2f42a6408948633c" target='_blank'>
              Abstract B035: Cell state transitions drive the evolution of disease progression in B-lymphoblastic leukemia
              </a>
            </td>
          <td>
            S. Marzban, Curtis Gravenmier, Ling Zhang, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Early T-cell precursor (ETP) acute lymphoblastic leukemia/lymphoma (ALL) is an aggressive hematological malignancy characterized by the uncontrolled expansion of T-cell progenitors. Loss-of-function mutations in genes encoding epigenetic regulators, including core components of the Polycomb Repressive Complex 2 (PRC2), such as the EZH2 histone methyltransferase, are frequently observed in ETP-ALL patients.To investigate the role of EZH2 inactivation in human early T-cell leukemogenesis, we optimized a CRISPR/Cas9-based strategy to disrupt EZH2 gene expression by integrating a DNA cassette containing the constitutive human PGK-1 promoter followed by the GFP reporter. Specifically, human CD34⁺ hematopoietic stem/progenitor cells (HSPCs) derived from cord blood were first edited to knock out (KO) EZH2, then transduced with lentiviruses encoding different T-ALL oncogenes or an empty vector control, and maintained in serum-free medium supplemented to support T-cell progenitor expansion. We found that only HOXA9- and LYL1-transduced EZH2KO HSPCs expanded in vitro for up to 30 days, and were enriched for early T-cell subsets, as identified by single-cell RNA sequencing (scRNA-seq) and a 26-color flow cytometry assay. Notably, these same subsets were capable of initiating T-cell leukemia in immunocompromised (NSG) mice following transplantation, suggesting that EZH2 inactivation cooperates with HOXA9 or LYL1 overexpression to drive human T-cell transformation. To assess the relevance of these findings in primary human T-ALLs, we analyzed whether loss-of-function mutations in PRC2 genes (EZH2, EED, SUZ12) were enriched in specific T-ALL subgroups, particularly ETP-ALL, in a cohort of 1,335 patients from the Children's Oncology Group (COG) study. Interestingly, PRC2-mutated T-ALLs were significantly associated with the HOXA subgroup and the ETP-ALL phenotype, and exhibited higher levels of minimal residual disease (MRD) 30 days after therapy compared to other T-ALL cases. To further dissect the molecular effects of EZH2 loss and HOXA9 overexpression in ETP-like T-ALL, we knocked out EZH2 in the human Loucy and PEER cell lines using CRISPR/Cas9. These EZH2KOcells were then transduced with HOXA9-encoding lentiviruses or empty vectors and analyzed by RNA sequencing. In parallel, ChIP-seq analyses were performed to map HOXA9 binding sites and assess histone modifications affected by EZH2 loss. Integrated analysis of RNA-seq and ChIP-seq data revealed that HOXA9 expression in EZH2-deficient ETP-like cells led to the upregulation of a stem-like/immature T-cell gene signature enriched in transcriptional activators and chromatin remodelers (e.g., RUNX2, IRX3, SATB1, ARID5B, CHD1, INO80D).Furthermore, scRNA-seq profiling of primary PRC2-mutated T-ALL samples revealed distinct leukemic subsets with high HOXA9 expression and strong enrichment for the same gene signature, as determined by Gene Set Variation Analysis (GSVA). Importantly, this signature stratified PRC2-mutated T-ALLs and identified a HOXA positive subgroup associated with poor prognosis.
 Collectively, these findings uncover a critical epigenetic circuit involving EZH2 loss and HOXA9-driven transcriptional reprogramming that promotes stem-like, therapy-resistant T-ALL subsets and highlights novel vulnerabilities that could be therapeutically exploited to eliminate high-risk disease at its origin.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d423907fad1e15df50c67eaeeaa6ff45c0caebf" target='_blank'>
              EZH2 inactivation cooperates with HOXA9 to induce leukemic transformation of human t-cell progenitors and sustain high-risk subsets of early t-cell precursor acute lymphoblastic leukemia (ETP-ALL)
              </a>
            </td>
          <td>
            M. Colucci, Costanzo Padovano, Gaja Bruno, E. De Santis, Serena di Iasio, Chiara Di Nunzio, Andrew Weng, Vincenzo Giambra
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric hematological malignancy. While advances in modern medicine have significantly improved survival – especially in children– treatment resistance, relapse, and refractory disease remain major challenges, highlighting the complexities in treating B-ALL and the need for ongoing research and development of novel therapeutic strategies.
 Our lab previously demonstrated that the SWI/SNF chromatin remodeling complex subunit SMARCD1 is essential for the development of the lymphoid lineage, as its inactivation in the adult hematopoietic system leads to lymphopenia while sparing the myeloid and erythroid lineages (Priam et al., Developmental Cell, 2024). In this study, we identify SMARCD1 as an essential gene in B-ALL. Specifically, SMARCD1 inactivation in B-ALL cell lines and patient-derived xenograft (PDX) cells results in profound defects in cellular proliferation and survival. However, the molecular mechanisms underlying its oncogenic role in this context remain to be elucidated.
 The SMARCD1 subunit does not directly bind DNA or chromatin; instead, it is recruited to specific genomic loci through interactions with various co-factors, including the tumor suppressor protein p53. Interestingly, although p53 is somatically mutated in over half of human cancers, it remains wild-type (WTp53) in approximately 75% of newly diagnosed adult ALL cases and 97–98% of pediatric ALL cases at diagnosis (Stengel et al., Blood, 2014, Leukemia, 2017). Given that genetic instability is a hallmark of cancer, understanding why some malignancies retain WTp53 despite its tumor-suppressive functions is a key question with important therapeutic implications. Emerging evidence suggests WTp53 may promote oncogenic functions in specific cancer contexts by regulating metabolic plasticity and pro-survival signaling pathways.
 In B-ALL cell lines, we confirmed a biochemical interaction between SMARCD1 and WTp53 using coimmunoprecipitation (Co-IP) and bimolecular fluorescence complementation (BiFC). We further demonstrated that the SMARCD1-dependent B-ALL cell lines are also reliant on WTp53, as p53 inactivation impairs proliferation and survival to a similar extent as SMARCD1 depletion. Notably, co-depletion of both proteins did not rescue the proliferative defect, suggesting a cooperative role in driving leukemia progression. Importantly, this dependency was not observed in p53-proficient non-hematopoietic cell lines, supporting a context-specific oncogenic role for this SMARCD1-WTp53 axis in B-ALL.
 To explore how the SMARCD1-WTp53 axis drives B-ALL progression, we performed bulk RNA sequencing of B-ALL cells depleted of either SMARCD1 or TP53. Differential expression and gene set enrichment analyses (GSEA) revealed overlapping downregulation of pathways linked to proliferation and metabolism, including hypoxia, fatty acid metabolism, glycolysis, and mTORC1 signaling. Notably, key metabolic enzymes and regulators - such as GLUT4/SLC2A4, HK2, and SDHA- were consistently downregulated upon loss of either SMARCD1 or TP53. These transcriptomic changes were supported by Seahorse XF metabolic flux analyses, which confirmed reduced metabolic activity in SMARCD1- and TP53-depleted B-ALL cells. Cleavage Under Targets and Tagmentation (CUT&Tag) analyses further showed that SMARCD1 and WTp53 predominantly co-occupy the promoter regions of common target genes. Ongoing studies aim to elucidate their cooperative roles in modulating chromatin accessibility and promoter/enhancer function at these loci.
 Collectively, our findings identify the SMARCD1-WTp53 axis as a key regulator of metabolic programming in B-ALL, supporting a high metabolic state and rapid proliferation through control of a distinct oncogenic transcriptional program. This work has important therapeutic implications, highlighting the potential of targeting chromatin-metabolism crosstalk in B-ALL via disruption of the SMARCD1-WTp53 axis. Furthermore, this study reveals a context-dependent oncogenic role for WTp53, emphasizing the need for precision therapies tailored to p53 status - whether by activating WTp53 or restoring its function in p53-mutant cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969dc91cb92875d724cae066a989cc5ef6e8a223" target='_blank'>
              SMARCD1 subunit of SWI/SNF chromatin remodeling complexes collaborates with p53 to exert an oncogenic role in B-ALL by maintaining high metabolic activity
              </a>
            </td>
          <td>
            Laurence Côté, Pierre Priam, Alexandre Polsinelli, Hugo Philippeau, Gabriel Alzial, Geneviève Deblois, Julie A. Lessard
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Metastasis is the primary cause of cancer-related mortality worldwide. This narrative review integrates recent advances in the molecular circuits orchestrating metastatic progression, encompassing epithelial–mesenchymal transition (EMT), organotropism, extracellular matrix remodeling, angiogenesis, hypoxia-inducible signaling, tumor-cell migration modes, and tumor–immune interactions through expert-guided literature selection. We examined therapeutic innovations that disrupt these pathways, including EMT modulators, matrix metalloproteinase inhibitors, VEGF/VEGFR-targeted regimens, hypoxia-activated prodrugs, and next-generation immunotherapies such as immune checkpoint blockade and chimeric antigen receptor T cells. Additionally, we discuss established nanotechnology-based delivery systems, advancing multi-omics integration, evolving single-cell analyses, and emerging CRISPR-Cas9 gene-editing applications as tools for improving metastasis detection, monitoring, and treatment. Despite this progress, translational obstacles persist, particularly regarding intratumoral heterogeneity, adaptive resistance, and limited preclinical model fidelity. Addressing these challenges requires biomarker-guided, multi-target therapeutic combinations, interdisciplinary collaboration, and globally inclusive clinical trials. This evidence underscores the importance of integrated strategies that simultaneously target intrinsic tumor plasticity and microenvironmental support to transform metastatic cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7d3de1a04e69b49fb547b3a441611b4041aca5" target='_blank'>
              Deciphering molecular pathways driving cancer invasion and metastasis: advances and therapeutic prospects
              </a>
            </td>
          <td>
            Aziza Alshahrani, Arwa Alsubait, Z. Asiri, Sahar Alghamdi, Sarah Bin Saqyah, Tariq Alqahtani, Rawan Fitaihi, Njoud Altuwaijri, Yahya F. Jamous
          </td>
          <td>2025-12-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 INTRODUCTION Monoclonal gammopathies –including MGUS, smoldering multiple myeloma (SMM), and multiple myeloma (MM)– are driven by clonal expansion of malignant plasma cells. Understanding their clonal architecture and the genomic landscape at single-cell resolution is essential to unravel disease progression and guide treatment strategies, particularly in the context of novel immunotherapies. In this study, single-cell DNA sequencing, was combined with surface protein profiling, to comprehensively dissect somatic mutations, copy number variations (CNVs), clonotypes, and immunophenotypes across disease stages and treatment responses. This novel single cell multi-omics approach provides a powerful perspective on the genomic and functional heterogeneity of plasma cell disorders, offering key insights for precision medicine and therapeutic stratification.
 METHODS CD138+ were isolated from bone marrow aspirates of 11 patients (21 samples) with MGUS, SMM, or MM using AutoMACS sorting. Patient samples included diagnostic, progression, relapse, and treatment-resistant stages. Single-cell DNA sequencing was performed using a panel of 839 amplicons, combined with surface protein profiling via a 20-antibody cocktail. Clonal architecture was inferred by integrating somatic mutations, clonotyping and CNVs. Surface protein expression was used to correlate genotypic features with targetable phenotypes at single cell resolution.
 RESULTS Clonal complexity and genomic architecture showed substantial heterogeneity across disease stages and therapeutic contexts. In early disease, such as MGUS and SMM, some cases already exhibited complex CNV profiles or early driver mutations, indicating subclinical clonal evolution. Several patients displayed stable CNV landscapes over time, while mutational profiles fluctuated, suggesting a selection or suppression of cell-specific subclones rather than wholesale genomic shifts.
 In relapsed/refractory multiple myeloma, branched clonal evolution was frequently observed, with sequential acquisition of mutations affecting genes such as TP53, FRG1, and TRAF2 in specific cells. In some cases, dual-clone coexistence with distinct genomic and immunoglobulin rearrangement profiles was evident, reflecting convergent tumor evolution. Furthermore, certain patients exhibited a distinct pattern of surface antibody expression, indicating the presence of plasma cells co-expressing both major therapeutic targets in MM, GPRC5D and BCMA, while other plasma cell populations expressed only one or neither of these targets.
 A key finding emerged when integrating genomic data with clinical outcomes: patients harboring monoallelic deletions and/or mutations in GPRC5D or BCMA (TNFRSF17) consistently showed limited or no clinical response to corresponding targeted therapies. As an example, one patient with biallelic inactivation of GPRC5D in near 90% of the cells prior to TALQUETAMAB treatment, failed to respond, correlating with absent membrane expression of the target. Conversely, a separate patient who progressed on anti-GPRC5D therapy demonstrated a favorable clinical response to BCMA-directed CAR-T cells, consistent with preserved BCMA expression and copy number in all the cells.
 These findings indicate that the genomic integrity and surface expression of immunotherapeutic targets strongly influence treatment efficacy. Importantly, discrepancies between gene copy number and surface protein levels — such as cases with BCMA amplification but heterogeneous expression — highlight post-transcriptional regulation as an additional layer of resistance.
 CONCLUSIONS Single-cell DNA and protein co-profiling in monoclonal gammopathies provides critical insights into clonal evolution, therapeutic resistance, and the dynamic expression of immunotherapeutic targets. Specifically, GPRC5D genomic alterations and reduced protein expression may compromise response to targeted therapies, highlighting the need for functional biomarker screening. Our data support the clinical relevance of integrating single-cell genomics in the management of plasma cell dyscrasias to tailor therapy and anticipate resistance. Our results suggest that performingsingle-cell genomic and proteomic profiling prior to treatment initiation may improve therapeutic selection and predict response, particularly in the era of targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b66dd32e338c623da193862d4d414630dc6f82e" target='_blank'>
              Single-cell DNA profiling unveils clonal evolution and identifies mechanisms of therapeutic resistance and progression in monoclonal gammopathies
              </a>
            </td>
          <td>
            B. Ariceta, P. Aguirre-Ruiz, Andrea Castro, D. Alignani, Saioa Vicente, A. Lopez, A. Alfonso-Piérola, L. Tamariz-Amador, Jesús F San-Miguel, P. Rodríguez-Otero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the most common childhood cancer, characterized by a multitude of molecular subtypes with vastly different prognoses. However, the biological basis of inter-patient variability in treatment response remains poorly understood. In particular, the exact impact of ALL genomics on leukemia sensitivity to individual chemotherapeutic agents are largely unclear. Consequently, frontline ALL treatment protocols worldwide continue to employ a largely uniform set of anti-leukemic drugs with limited variation in their application. In the era of personalized medicine, it is now imperative to comprehensively map “gene-drug” relationships in ALL and use it to guide the development of the next generation of clinical trials for this disease.
 To this end, two recent studies from our group have reported foundational insights into the pharmacogenomic landscape of ALL, with adults and children profiled for both drug sensitivity and leukemia genomics (Lee et al., Nat Med 2023; Yoshimura, et al., J Clin Oncol 2024). Nonetheless, much of these rich datasets remain underutilized in their potential for discovery research. Therefore, we sought to develop an open-source platform for exploring ALL pharmacogenomics, with a series of innovative data visualization and analysis tools. This web-based portal, the ALL-Pharmacotyping Data Portal (APDP), is a freely accessible resource encompassing drug sensitivity and genomic data from over 1,000 pediatric and adult ALL cases, i.e., up to 21 drugs per case with a total of 7,975 datapoints. The portal facilitates identification of drug sensitivity patterns across molecular subtypes of ALL, clinical and demographic features, and their correlations with transcriptomes and treatment response. Examples of these analyses are provided below:Compare ALL subtype vs. drug sensitivity: Users can examine how LC50 values vary across 23 different molecular subtypes of ALL using interactive boxplots, violin plots, and statistical tests. This enables quick identification of subtypes with distinct sensitivity patterns to specific drugs.On-the-fly differential expression (DE) analysis of sensitive vs. resistant ALL samples: For any drug, the portal allows users to classify ALL samples as sensitive or resistant based on LC50 thresholds and run DE analysis between the two groups. Ranked DE genes can be exported or used for hierarchical clustering, heatmap visualization, and gene set enrichment analysis which can highlight enriched biological pathways and molecular processes potentially underlying drug sensitivity or resistance.Visualize drug sensitivity on transcriptome UMAP: This enables intuitive visual correlation between drug response and transcriptomic clustering patterns. From any region of the UMAP, users can select clusters of interest and launch DE analysis to further explore differences between sensitive and resistant samples.Multivariable regression analysis: This allows users to confirm the statistical correlation of drug sensitivities with clinical or gene expression variables by adjusting for co-variates including patient demographics such as sex, genetic ancestry, and age at diagnosis.
 This integrated framework allows investigators to explore pharmacogenomic patterns in ALL from multiple angles, combining statistical rigor, interactive visualization, and intuitive navigation to accelerate discovery of novel insights into drug response biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909273b30933075b8473d932a219a58389f7d3c7" target='_blank'>
              ALL-pharmacotyping data portal: An open-source platform for exploring genomics of leukemia drug susceptibility and resistance
              </a>
            </td>
          <td>
            Karishma Gangwani, S. Yoshimura, Zhenhua Li, Robin Paul, Wenjian Yang, Congyu Lu, Jian Wang, Andrew Willems, Gavriel Y Matt, Edgar Sioson, Colleen Reilly, Airen Zaldívar Peraza, Aleksandar Acić, Shawn Lee, Jun J. Yang, Xin Zhou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Despite recent advances in targeted chemotherapies for acute myeloid leukemia (AML), treatment of relapsed and refractory AML remains a significant challenge. Overexpression or rearrangement of the Myelodysplasia Syndrome (MDS) and Ecotropic Virus Integration Site 1 (EVI1) Complex locus (MECOM) is associated with particularly poor prognoses. High MECOM expression establishes a population of leukemia stem cells (LSCs) that have increased resistance to chemotherapy. Prior data from our group revealed a MECOM associated gene regulatory network that is essential in hematopoietic stem cell self-renewal and function, which may represent a therapeutic vulnerability in high-risk AML. Here, we investigated the association of the MECOM transcriptional network with chemoresistance in multiple AML models and found that perturbation of MECOM increased sensitivity to standard chemotherapies independent of cell state changes.



 To probe the interaction of MECOM with chemosensitivity, we engineered multiple high MECOM expressing human AML cell lines by targeting the endogenous MECOM locus with a FKBP12F36V degron that enabled rapid and complete MECOM protein degradation with exposure to the small molecule dTAG. MUTZ-3, an inv(3) AML cell line, was the primary system used as it can expand indefinitely and has a subpopulation of CD34+ progenitors that either self-renew or differentiate into CD14+ monocytes. Findings were later verified in the UCSD-AML-1 and HNT-34 cell lines. Using these models, we determined differential chemosensitivity of AML cells following MECOM protein degradation compared to control cells by characterizing phenotypic properties and differential gene expression profiles with etoposide and doxorubicin treatment, common backbones in current AML treatment regimens.
 In MUTZ-3 cells, etoposide treatment increased the CD34+ stem cell progenitor population from 59.5% to 67% compared to the control (p<0.05, n=3). CD34+ MUTZ-3 LSCs are less likely to undergo apoptosis following etoposide treatment (5.5%, n=3) compared to more differentiated CD14+ cells (60.6%, n=3), demonstrating that etoposide selects for LSCs. Hypothesizing that some of this chemoresistance of LSCs was driven by MECOM, we determined IC50 values of etoposide and doxorubicin after MECOM degradation. MECOM degradation led to a 2.2-fold, 1.6-fold, and 1.7-fold decrease in etoposide IC50 respectively in MUTZ-3, UCSD-AML-1, and HNT-34 cells (p<0.05, n=3). Similarly, MECOM degradation led to a 1.5-fold, 1.8-fold, and 1.5-fold decrease in doxorubicin IC50 in MUTZ-3, UCSD-AML-1, and HNT-34 cell lines (p<0.05, n=3), which indicates that MECOM degradation sensitizes AML cells to etoposide and doxorubicin.
 To examine the transcriptional signatures associated with chemotherapy treatment and MECOM degradation, we performed bulk RNA sequencing with the CD34+ subpopulation of MUTZ-3 cells. We observed significant enrichment of the MECOM gene network following treatment with etoposide (NES =1.81, p <0.001) and doxorubicin (NES=1.91, p <0.001) that is reversed with degradation of MECOM with both etoposide (NES = -2.610, p<0.001) and doxorubicin treatment (NES= -2.634, p<0.001).
 To determine the relevance of this MECOM gene set in primary AMLs, we evaluated RNA sequencing data from adult patients in the BEAT-AML data set. We compared samples at diagnosis (n=450) and relapse (n=37) and found enrichment of the MECOM gene network in relapsed samples compared to diagnostic samples (NES = 1.9, p <0.001). We additionally examined diagnostic samples from patients that were later found to have a complete (n=220) or refractory (n=103) response to first-line chemotherapy and found an enrichment of the MECOM gene network in patients with a refractory response (NES = 1.7, p <0.002).



 In summary, the MECOM gene network is associated with etoposide and doxorubicin resistance in AML human cell models, suggesting that chemotherapy resistance is not only mediated by cell state changes, but is also modified by transcriptional changes in LSCs. Additionally, this gene set is enriched in relapsed and refractory primary AML patient samples. Further study of the mechanisms underlying the observed chemotherapy resistance and ways to overcome it are desperately needed for the development of newer and more effective therapies for high-risk AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfccb6c8ddff1c95d1884bc510e150bea0e7adef" target='_blank'>
              Mecom gene network drives chemotherapy resistance in high-risk acute myeloid leukemia
              </a>
            </td>
          <td>
            Sanjana B Shah, Travis J. Fleming, Richard Voit, V. Sankaran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Long-lived plasma cells (LLPCs) sustain lifelong antibody secretion, forming the foundation of durable immune memory. However, the molecular mechanisms underpinning LLPC survival within the bone marrow (BM) remain incompletely understood. Using single-nucleus RNA-seq and ATAC-seq (10X Multiome) from matched blood and BM antibody-secreting cells (ASCs) of healthy adults, we profiled 8,059 nuclei, generating an integrated transcriptional and epigenetic map of ASCs across their developmental continuum with unparalleled resolution. In the BM LLPC cluster, Gene Set Enrichment Analysis (GSEA) validated the enrichment of the TNF/NF-κB signaling pathway and TLR cascades, emphasizing their anti-apoptotic roles. These clusters also demonstrate increased chromatin accessibility at key loci such as RELA, TLR1, TLR6, TLR10, NFKBIA, and TAB1, underscoring the central roles of TLR and TNF-α signaling pathways in activating the NF-κB pathway. Upregulated genes such as BCL2, NFKBIZ, TNFAIP3, and BIRC3 highlighted key survival mechanisms, while motif enrichment analysis revealed significant enrichment for REL and NF-κB binding motifs, supporting chromatin-level regulation of LLPC longevity. Together, these findings advance our understanding of LLPC survival and resilience and provide a framework for improving vaccine strategies and targeted immunotherapies.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86cf15dec942c9c651d1f78f13be792e4e03baf" target='_blank'>
              Dissecting the transcriptional and epigenetic landscape of human antibody-secreting cells in blood and bone marrow long-lived plasma cells 4589
              </a>
            </td>
          <td>
            Hira Anis, Monica Cabrera-Mora, Christopher Scharer, Ignacio Sanz, Greg Gibson, F. Eun‐Hyung Lee
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a5fb32298b06f7826592aac1dafed7fb246a78" target='_blank'>
              Genotype-phenotype correlations and de novo induction of cancer stem cells in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M. Linssen, J. Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer, recognized as one of the most prevalent malignancies with the highest rates of incidence and mortality globally, presents a substantial challenge on a worldwide scale. This challenge is exacerbated by the disease’s difficulty in early detection, a pronounced rate of metastasis, and resistance to treatment, all of which contribute to elevated mortality rates. The tumor microenvironment (TME) plays a critical role in the sustenance and advancement of various solid tumors, including lung cancer. The intricate composition of the TME facilitates tumor proliferation, metastatic spread, and therapeutic resistance by supplying metabolic resources, fostering angiogenesis, and enabling immune evasion. Nonetheless, the regulatory frameworks operating within the TME remain poorly understood. An increasing body of evidence suggests that epigenetic regulation—encompassing mechanisms such as DNA methylation, histone modification, and the action of non-coding RNAs—is pivotal in the initiation and progression of lung cancer. Furthermore, epigenetic modifications significantly influence the functional dynamics of the tumor microenvironment, thereby impacting intercellular interactions and cellular behaviors within the TME, which in turn affects the trajectory of disease progression. This article aims to present the most recent advancements in research concerning the epigenetic regulation of tumor cell interactions with the TME in the context of lung cancer biology. Additionally, it examines the current implications of epigenetic regulation within the tumor microenvironment and its influence on lung cancer behavior. We also investigate the potential relevance and emerging therapeutic avenues presented by epigenetic regulation in the clinical diagnosis and treatment of lung cancer, aspiring to propose novel strategies to address existing treatment challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f9bbf3d9c9172ba516c36021afce36965e86d5" target='_blank'>
              Epigenetic regulation of the tumor microenvironment in lung cancer: mechanism insights and therapeutic prospects
              </a>
            </td>
          <td>
            Maoqin Yang, Xudong Lei, Dexiang Ren, Dakai Qin, Xiaojun Xia
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Introduction Acute myeloid leukemia(AML) is a heterogeneous malignancy marked by impaired myeloid differentiation and resistance to apoptosis. Venetoclax, a selective BCL-2 inhibitor that restores apoptotic signaling, has shown improved outcomes when combined with hypomethylating agents in older or unfit patients; however, many remain refractory or experience relapse. We used single-cell RNA sequencing(scRNA-seq) to compare bone marrow samples from patients with AML before and during venetoclax-based therapy with those from healthy controls, focusing on cellular type diversity and gene expression dynamics.
 Methods We analyzed 10 bone marrow samples from patients with AML treated with venetoclax plus azacitidine or decitabine, as well as from healthy donors at the University of Iowa. These included samples from 3 healthy controls, 4 responders (1 diagnostic[RD1] and 4 during treatment[RT1-RT4]), and 2 non-responders(1 diagnostic[NRD1] and 1 during treatment[NRT1]). scRNA-seq was performed using 10X Genomics 3' v3 chemistry, sequenced on an Illumina NovaSeq 6000, and processed with Cell Ranger v8.0.0. We excluded cells with <100 or >7,500 genes or >75% mitochondrial reads. Seurat (v5.0.1) was used for normalization, scaling, integration, clustering(resolution = 0.1), and UMAP reduction using the top 30 principal components. Cell type annotation used SCINA with references from Human Cell Atlas ((https://doi.org/10.1016/j.exphem.2018.09.004)). We focused on CD34+ cells to study stem cells and early hematopoietic progenitors.
 Results We identified various hematopoietic populations, including hematopoietic stem cells(HSCs), multilineage-primed(Multi-Lin) cells, monocyte-dendritic progenitors(MDP-1 and MDP-2), megakaryocyte-erythroid progenitors(MEPs), and other early progenitors, as well as mature blood cells.
 Control samples showed consistent distributions with expected proportions of early progenitors, while AML samples displayed distinct cellular profiles. RD1 sample was dominated by a high percentage of blasts(47%) and extensive expansion of MDP-2 cells(~90%). Refractory samples(NRD1 and NRT1) exhibited elevated blasts(60% and 12%) and significant expansion of immature T progenitors(pre-T cells, 33–49%). RT1 had residual blasts(7%) and showed increased numbers of Multi-Lin cells(34%) and MEPs(7%), which may indicate the persistence of immature leukemic clones. RT2, RT3, and RT4 were negative for blasts, with recovery of diverse hematopoietic populations, notably an expansion of MDP-2(13–70%) and lymphoid cells, consistent with marrow recovery.
 In differential gene expression analysis, a consistent finding across multiple cell types was the downregulation of FKBP5, a glucocorticoid-responsive gene involved in stress signaling and resistance to apoptosis. In HSCs, we observed BCL2 downregulation along with upregulation of pro-apoptotic genes(BAX, BAK, CASP7, CASP9). There was suppression of STAT5(growth-promoting) and SMAD3(growth-inhibitory) signals in HSCs, which may indicate disrupted stem cell signaling and early commitment or exhaustion. Downregulation of PTGER4, IL3RA, and IL12RB2 suggested decreased signaling through pro-survival and proliferative cytokine pathways. Additionally, ribosomal protein downregulation was observed, a phenomenon commonly associated with apoptosis, quiescence, or differentiation, where protein synthesis was actively suppressed. CD14 was upregulated in HSCs and MDP cell types, indicating monocytic skewing. In MDP-1 cells, heat shock proteins(HSPA1A and HSPA1B) were downregulated, and DUSP7(a MAPK phosphatase) was suppressed, pointing to impaired stress-response signaling. Changes in the expression of RASA1, RASA2, and MAP3K family genes suggest disrupted Ras-MAPK pathway activity. In MDP-2 cells, decreased CD1B coupled with increased CD14 expression marked a shift away from dendritic differentiation toward an aberrant monocytic identity.
 Conclusion Venetoclax-based therapy is associated with transcriptional remodeling across different cell types in bone marrow. Downregulation of pro-survival and stress-related genes and upregulation of differentiation and apoptotic programs were observed. Distinct transcriptional programs in MDP-1 vs. MDP-2 suggest functional remodeling in different lineages during therapy. These data provide insight into how venetoclax can reshape the bone marrow microenvironment in AML and may help identify early markers of treatment response.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589edb4fcbc10647fadb024701f20f9d7fed3689" target='_blank'>
              Single-cell insights into acute myeloid leukemia treated with venetoclax-based therapy
              </a>
            </td>
          <td>
            Prajwal Dhakal, Michael Chimenti, Benjamin Darbro, Michael Tomasson, M. Bates
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities.
              </a>
            </td>
          <td>
            Md Shiblee Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains one of the leading causes of cancer-related morbidity and mortality in men worldwide, and therapeutic resistance—particularly to immunotherapy—continues to limit clinical efficacy. Mounting evidence has positioned the immunosuppressive tumor microenvironment (TME) as a core driver of disease progression and a formidable barrier to effective immune-based interventions. In this review, we present a comprehensive and multi-dimensional analysis of the cellular, stromal, and molecular constituents of the immunosuppressive TME in PCa, highlighting its significant heterogeneity and context-dependent functions. We emphasize recent breakthrough insights obtained through single-cell RNA sequencing (scRNA-seq), spatial multi-omics, and high-dimensional imaging technologies, which are redefining our understanding of tumor–immune–stromal interactions. Based on these mechanistic findings, we examine precision therapeutic strategies aimed at remodeling the TME, including combinatorial immune checkpoint blockade, metabolic reprogramming, cytokine network regulation, and advanced nanomedicine-based delivery systems. Finally, we discuss translational opportunities and future research directions, underscoring the necessity of integrating advanced omics technologies with biomarker-driven clinical trial design to enable individualized, precision interventions and improve survival outcomes for patients with PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities
              </a>
            </td>
          <td>
            Mingfeng Li, L. Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Loss-of-function studies (LOF) have historically been the main functional approach to identify and study genes which drive the biology of Multiple Myeloma (MM) or other neoplasias. We hypothesized that substantial complementary data can be derived from systematically examining the impact of transcriptional activation of endogenous loci of different genes. We thus performed genome-scale CRISPR gene activation (CRISPRa) screens in 10 genotypically diverse MM cell lines (dCas9-P65-HSF transcriptional activation system; Calabrese genome-scale sgRNA library) to identify genes with significant sgRNA enrichment reflecting enhanced tumor cell fitness. These experiments identified that MM driver genes are highly cell line-dependent, with no universal “hit” but with many recurrent (e.g. in ≥4/10 lines) drivers, including key transcription factors/cofactors (e.g. POU2F2, POU2AF1, IRF4, MYC, HOXA3); growth factor signaling mediators, (e.g. IGF1R, IRS1); Ras family members (e.g. KRAS in MM1S cells); NFκB signaling (e.g. RELA, TIFA), PKC pathway (PRKCE), cell surface receptors (e.g. CD48, HTR1D, IGF1R, UPK3B), solute carrier transporter genes (e.g. SLC38A1). No correlation is currently observed between the pattern of driver genes for the individual MM cell lines and their respective molecular subtype (e.g., t(4;14), t(14;16), t(11;14), etc). Some MM-preferential dependencies identified by CRISPR knockout (KO) studies (de Matos Simoes et al Nat Cancer 2023) were identified by CRISPRa as recurrent MM drivers (e.g. POU2AF1, IRF4) or drivers with limited role in individual line(s) (e.g., IKZF3 and IKZF1 in MM1S cells). Notably, many top MM drivers are not essential for MM cells based on CRISPR KO studies. Most top MM drivers were not identified in similar CRISPRa studies as recurrent drivers for 17 non-MM cell lines from other hematologic neoplasias or solid tumors. With some notable exceptions (e.g., IRF4, POU2AF1), most other MM drivers are not identified in MM patient samples among the molecules with most frequent mutations, gene amplification or overexpression (e.g. vs normal plasma cells) at transcript or protein level. However, transcript upregulation (log2FC≥1.0) of at least one MM driver gene is observed in ~85% of paired MM samples after relapse vs. before treatment (MMRF CoMMpass study), suggesting that many CRISPRa-driver genes can contribute, alone or in concert, to enhanced MM cell fitness in clinical relapses. We validated the functional relevance of several MM driver genes (e.g. POU2F2, EGLN1, SLC38A1) with individual sgRNAs for CRISPR activation and/or cDNA overexpression (vs. isogenic controls) in competition experiments. We further probed the mechanistic basis for the role of POU2F2 as one of the top positive regulators of MM cell fitness and performed RNA sequencing analyses of MM1S cells harboring CRISPRa-based upregulation of POU2F2 expression. We observed upregulation of a distinct cluster of genes (including HOXA3, USP32, HTR1D) that are identified in our genome-scale CRISPRa studies as driver genes specifically in MM1S cells or ≥4 MM lines of our analyses, as well as several genes that are essential for MM survival (e.g., U2AF1, ZNF492, HYPK, PAM16, H2BC11). These results indicate that genome-scale CRISPRa studies provide data which are complementary to those derived from LOF studies. Indeed, our CRISPRa studies validated that MM cell fitness can be enhanced by upregulation of some genes that are prominently essential for MM cells in LOF studies (e.g. IRF4, POU2AF1) but also identified many other promising, previously understudied, regulators of MM cell biology which are not essential for baseline survival/growth of MM cells but can induce growth when further activated. Upregulation of one or more MM driver genes is recurrently observed in MM clinical relapses, which raises intriguing hypotheses about how potential cooperative interactions between these genes contribute to the risk of MM relapses. More broadly, the results indicate that CRISPRa studies can provide novel insights into the biology of MM cells and identify previously understudied genes with therapeutic implications towards suppressing the transition of MM cells to states of advanced biological aggressiveness.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02015e88839b8028448eeed27364c1dfd71a4d2b" target='_blank'>
              Identification of novel “driver” genes of multiple myeloma through genome-scale CRISPR activation studies
              </a>
            </td>
          <td>
            Huihui Tang, Ricardo de Matos Simoes, Torsten Steinbrunn, Ryosuke Shirasaki, O. Dashevsky, Eleni Anastasakou, Oliver Bohorquez, S. Yamano, E. Papadopoulos, Lisa B Leypoldt, Arun Wiita, Olwyn Mahon, Vikas A. Gupta, Jonathan Licht, Benjamin G. Barwick, Lawrence Boise, Constantine S Mitsiades
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Breast cancer (BC) remains a major contributor to cancer-related morbidity and mortality among women globally. While systemic therapies have significantly advanced, the emergence of drug resistance continues to hinder durable clinical benefit, leading to treatment failure and disease relapse. Long non-coding RNAs (lncRNAs) have gained recognition as key regulators in BC biology and therapeutic response, with growing evidence implicating them in resistance to chemotherapy, endocrine therapy, targeted agents, and immunotherapy. This review provides a detailed overview of the molecular mechanisms through which specific lncRNAs promote resistance, emphasizing their roles in modulating processes such as programmed cell death, epithelial-mesenchymal transition, and remodeling of the tumor microenvironment. We further highlight common regulatory axes - such as competing endogenous RNA networks and the phosphatidylinositol 3-kinase/protein kinase B pathway - that serve as converging points across distinct resistance mechanisms. In addition, we discuss recent progress in harnessing lncRNAs as liquid biopsy biomarkers and examine the key challenges and opportunities in translating lncRNA-based targets into therapeutic strategies. Together, these insights offer a foundation for future research aimed at overcoming drug resistance in BC through novel lncRNA-centered interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfac8b955e147276484ab116353ca6e5f90be4a2" target='_blank'>
              The role of long non-coding RNAs in breast cancer drug resistance: a comprehensive review
              </a>
            </td>
          <td>
            Yingkun Xu, Jingyu Ma, Changyuan Ding, Weili Liang, Q. Zeng, Bin Lv
          </td>
          <td>2025-12-10</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d2645c176f5e0510e7c9fe1ad9d0e8b2a80929" target='_blank'>
              Stroma-driven horizontal transfer of TCA-related proteins mediates metabolic plasticity and imatinib resistance in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            P. Chrościcki, Nikodem Kasak, D. Dymkowska, L. Turos-Korgul, Dominik Cysewski, Vira Chumak, Dawid Stępnik, Monika Kusio-Kobiałka, Agata Kominek, M. Lebiedzińska-Arciszewska, Alicja Krop, Joanna Szczepanowska, Mariusz Wieckowski, Tomasz Stoklosa, Krzysztof Zabłocki, K. Piwocka
          </td>
          <td>2025-12-02</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31ee50ee35e90e081933ec18f7fbe53abb71750" target='_blank'>
              Proteostasis Stress Drives Stem Cell Aging, Clonal Hematopoiesis and Leukemia
              </a>
            </td>
          <td>
            F. Zhou, Michelle K. Le, Helen C. Wang, Wei Yang, Huan-You Wang, Arukshita Tiwari, Xinjian Cen, M. Sunshine, Jeffrey A. Magee, R. Signer
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="


 Long non-coding RNAs (lncRNAs) represent an important layer of gene regulation and have emerged as significant contributors to multiple myeloma (MM) progression (Malard, F., Neri, P., Bahlis, N.J. et al.,2024). The lncRNA NEAT1 is over-expressed in MM patients, supporting malignant plasma cells proliferation and promoting therapy resistance (Taiana E.et al., 2020). Besides its conventional function in the assembly of paraspeckles (PSs), we recently demonstrated that NEAT1 is directly involved in transcriptional control (Puccio N. et al., 2024). In this study, we deepen our understanding of NEAT1's function in shaping the activity of transcriptional bodies, providing the rationale for targeted therapeutic intervention.
 Starting from a transcriptomic approach in NEAT1-silenced (KD) and CRISPRa NEAT1-overexpressing MM cells, we derived a list of 378 targets, primarily involved in mitosis. 52% of these present the cell cycle homology region motif (CHR) in their promoter region, which synchronizes their activation during late cell cycle phases. We confirmed the clinical relevance of NEAT1's transcriptional program in the coMMpass cohort of almost 900 MM patients. Unsupervised clustering analysis, based on the expression of 378 genes NEAT1's target genes, identified two different groups of patients characterized by high or low NEAT1's expression program and different clinical outcomes. Notably, the cluster with the high NEAT1-CHR program showed reduced overall survival. To explore the mechanistic contribution of NEAT1 to mitotic gene expression, we performed computational analysis through the binding model of analysis of transcription (BART) and query of the ENCODE database. These analyses revealed that FOXM1 and CDK9 as upstream regulators of NEAT1 program. Notably, FOXM1 resulted to be highly expressed in R2-ISS stages II-III of myeloma disease as compared to stage I, and in secondary plasma cell leukemia (sPCL) of myeloma origin, based on coMMpass and GSE66293 datasets analysis. We supported these observations by performing in vitro assays. We described a co-localization of NEAT1 and FOXM1 condensates in MM cells, through combined RNA-FISH/immunofluorescence, and direct binding of NEAT1-FOXM1/NEAT1-CDK9 by targeted RNA-immunoprecipitation (RIP). Additionally, with Chromatin immunoprecipitation experiments (ChIP), we demonstrated that NEAT1 KD results in a reduced occupancy of FOXM1 at the promoters of essential CHR genes. Finally, by performing high-throughput drug screening, we identified CDK9 inhibitors to have a strong synergistic effect with NEAT1 KD, consistent with a synthetic lethality effect. These results were confirmed by live cell imaging approach, with two selective CDK9 inhibitors (CDK9i) on a panel of MM cell lines. Consistently, NEAT1 overexpression confers an increased resistance to CDK9i, confirming the participation of NEAT1 to the transcriptional complex that synchronizes mitotic genes and executes the cell division program.
 These data are in line with our recent findings that demonstrate that NEAT1 controls and cooperates with Aurora kinase A (AURKA) in ensuring proficient chromosomal segregation(Puccio N. et al., 2024).
 In conclusion, our data demonstrated that NEAT1 contributes to overt phases of MM and to the clinical aggressive behaviour of the disease, by orchestrating a mitotic transcriptional program. Furthermore, through the application of a multi-dimensional approach, we revealed that the dual inhibition of NEAT1 and transcriptional regulators as CDK9 may offer a promising strategy for novel combinatorial anti-MM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0423164211e0008033b3494329d936fcc2b1eb0a" target='_blank'>
              Disruption of NEAT1-transcriptional network identifies targetable non-genetic dependencies in multiple myeloma
              </a>
            </td>
          <td>
            N. Puccio, Christian Boni, Federica Torricelli, Emanuele Vitale, E. Taiana, D. Ronchetti, Valentina Fragliasso, Selene Mallia, Giulia Gambarelli, Roberto Piva, N. Bolli, S. Luminari, A. Ciarrocchi, A. Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Epitranscriptomics, the study of dynamic chemical modifications on RNA mediated by “writers,” “erasers,” and “readers,” has emerged as a pivotal discipline in elucidating the intricate interplay between cancer and immune regulation. These reversible modifications (e.g. m6A, m5C, Ψ) govern RNA metabolism, stability, and translation, thereby exerting spatiotemporal control over immune cell differentiation, activation, and function. Dysregulation of RNA-modifying proteins disrupts immune surveillance, enhances tumor cell survival under stress, and promotes chemoradiotherapy resistance by altering RNA splicing, translation, and stress adaptation pathways. This review summarized the recent progress in the regulatory mechanisms profoundly influencing the tumor microenvironment (TME), modulating immune checkpoints, antigen presentation pathways, and the activity of immune cells. Furthermore, we discussed the therapeutic strategies and challenges in targeting epitranscriptomic regulators and epitranscriptomic editing technologies to enhance anti-tumor immune responses and overcome therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c0776f1a44885afebca84621fac676e8beb717" target='_blank'>
              Epitranscriptomic control of cancer immunity and therapy resistance
              </a>
            </td>
          <td>
            Xingsen Zhao, Suhua Guan
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma multiforme (GBM) is a highly aggressive and treatment-resistant brain tumour. ATRX mutations are frequently observed in both adult and paediatric GBM, including many single-residue missense mutations. These remain underexplored due to technical limitations in modelling, as conventional knockout models do not fully reflect their biology. This study aimed to establish a Prime Editing-based platform to precisely model clinically recurrent ATRX mutations that have been previously intractable with existing tools and to investigate their functional and therapeutic implications. We analysed the COSMIC database and identified 186 ATRX single-residue variants in glioma. Nine recurrent mutations were shortlisted, and three were selected based on feasibility for Prime Editing—a precise DNA engineering method in which a Cas9-nicked site primes synthesis of replacement DNA via reverse transcription from a guide RNA-encoded template. Editing efficiency was enhanced using our artificial intelligence (AI)-designed MLH1 small binder (PE7-SB system). For each target site, we designed more than eight Prime Editing schemes—combinations of specialized RNA sequences—and evaluated editing outcomes in human cancer cell lines. Editing attempts in multiple GBM cell lines consistently resulted in lethality, suggesting these ATRX mutations may be incompatible with GBM cell survival without additional adaptations. In contrast, sixteen viable mutant clones were generated in HeLa and HCT116 cells. These clones demonstrated increased sensitivity to a compound in clinical trials for ATRX-deficient tumours, enabling functional and therapeutic exploration of recurrent ATRX mutations. Our Prime Editing platform, enhanced by AI-guided design, enables precise and efficient modelling of clinically observed ATRX mutations. This approach uncovers novel biological constraints and lethal dependencies in glioblastoma. By enabling residue-level interrogation of tumour-associated gene function, this toolkit supports the broader neuro-oncology research community in dissecting key molecular alterations underlying glioma biology and lays the groundwork for future functional studies and therapeutic exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62c24696781616c181ce28d7b176b521566ae98a" target='_blank'>
              TMOD-01. Prime Editing of ATRX point mutations in glioblastoma reveals lethal dependencies and targetable vulnerabilities
              </a>
            </td>
          <td>
            Kanggeon Kim, Yeji Lee, E. Outwin, Timothy Humphrey, S. Bae
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Patients with Multiple Myeloma (MM) receiving adaptive T cell therapies such as chimeric antigen receptor (CAR) T cells and T cell engagers (TCEs) targeting BCMA have shown encouraging responses. However, despite initial response, most patients experience relapse. We have recently observed an enrichment of NF-kB activation mutations, including deletion of TRAF3 and CYLD or amplification of MAP3K14, also known as NF-kB Inducing Kinase (NIK), in patients with BCMA acquired resistance. Furthermore, in a genome-wide CRISPR screen performed using the YUSA library containing 90,709 unique sgRNA targeting 18,010 genes in OPM2 cells, sgRNAs targeting TRAF3 were the top enriched gene in MM cells surviving multiple exposures to an anti-BCMA TCE. Additional enriched hits include sgRNAs targeting genes involved in apoptotic regulation, such as CASP8, and key regulators of the NF-kB pathway, including BIRC3, NFKBIA and NFKBIB. Therefore, we aimed to determine the impact of NF-kB alterations (TRAF3, CYLD and NIK) on the development of resistance to anti-BCMA therapies in MM.
 Through CRISPR Cas9 gene editing, we have generated stable TRAF3 and CYLD knockout (KO) clones in OPM2 and ARP1 MM cells. Moreover, we have cloned NIK in a pLEX-307 vector to overexpress (OE) the NIK protein in both OPM2 and ARP1 cells. As controls, we generated a non-targeting Cas9 and pLEX-307 empty vector cells. The protein changes were confirmed by western blot and the activation of the non-canonical NF-kB pathway was validated by ELISA assay.
 By using flow cytometry-based cytotoxicity assays, we have next demonstrated that the KO and OE cells co-cultured with healthy donors' PBMCs exhibited resistance to anti-BCMA therapies, including CAR-T cells and TCEs. Of interest, similar resistance was observed in the presence of anti-GPRC5D targeting CAR-T cells and TCE, suggesting that their resistance is rather intrinsic and target antigen agnostic.
 To elucidate the molecular signature associated with the development of this acquired intrinsic resistance, we have next performed transcriptome profiling (scRNA-seq) on the OPM2 KO and OE clones and their isogenic controls.
 Several differentially activating pathways, including upregulation of anti-apoptotic signaling and an enrichment of the interferon alpha (IFN) and TNF-a signaling, were transcriptionally enriched in these TCE/CAR T resistance cells, with several targets confirmed at the protein level. Of note, this transcriptionally enriched IFN response was also functionally validated in vitro by demonstrating the lack of infection and replication of the VSV-ΔM51-GFP virus in the TRAF3KO and CYLDKO MM cells relative to their isogenic controls. In addition, multiplex cytokine assay performed on the supernatant of these TRAF3/CYLD KO and NIK OE TCE/CAR T resistant cells revealed elevated levels of multiple factors and cytokines with known inhibitory effect on T cell function, such as sFas, sFasL, TGF-b3, TNF-a, IL-10 and IP-10 when compared to the control isogenic cells. Importantly, we demonstrated that IP-10 (encoded by CXCL10) addition to healthy donor PBMCs co-cultures with MM cell lines was able to attenuate T-cell mediated killing in the presence of anti-BCMA CAR-T cells or TCEs. Lastly, by flow cytometry, we have observed upregulation of Fas expression in the KO cells as the likely source for the increased levels of soluble Fas (sFas). We suggest that sFas may interfere with T cell function by acting as a decoy receptor to immune cells FAS ligand (FASL) blocking its binding to FAS on MM cells.In summary, we have identified a novel NF-kB dependent tumor intrinsic mechanism of resistance to T cell-based therapies mediated by TRAF3/CYLD deletion and NIK amplification that promotes an inflammatory and immunosuppressive microenvironment suppressing T cell activity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c08a4a12b3d82a261da5787f2a12e375667b1e3" target='_blank'>
              Deletion of TRAF3 and CYLD and NIK amplification activates NF-kb signalling and drives resistance to T cell based therapies in multiple myeloma by suppressing T cell activity
              </a>
            </td>
          <td>
            David Jung, S. Benaoudia, Holly Lee, Sungwoo Ahn, Sejal Chikhale, Mansour Poorebrahim, Noémie Leblay, N. Bahlis, Paola Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 The clinical outcome of acute myeloid leukemia (AML) is determined by therapy resistance and relapse. AML-induced immunosuppression has emerged as one of many cofactors and seems to be caused by distinct mechanisms, e.g. immune checkpoint interactions, altered antigen presentation and dysregulation of humoral factors. With chemotherapy, these alterations are expected to be reversible, but the exact mechanisms remain unclear. Adressing this issue, we performed single-cell RNAseq and spatial assessment to identify therapy-induced perturbations of the bone marrow (BM) microenvironment in newly diagnosed AML patients prior to and after induction therapy.



 43,000 single-cell RNA sequencing profiles were generated from 12 paired samples in 6 AML patients using 10x Genomics 5'-scRNA/CITE-seq (140 Total-seq C antibodies) at initial diagnosis (T1) and after induction chemotherapy with either “7+3” or CPX-351 (T2). Analyses were performed using R v4.1.1 and Seurat v4.3.0. Signaling pathway activity was calculated using PROGENy. Stemness was estimated using CytoTrace. Potential cell-cell interactions were scored using CellChat. Spatial correlation was performed by histopathology and immunohistochemistry in matched trephine biopsies using the following stains: H&E, Giemsa, PAS, Gomori silver, Prussian Blue, Glycophorin C, MPO, CD61, CD34, CD117, CD14, CD68, CD20, CD3, CD138, VISTA. All slides were microscopically assessed by a trained hematopathologist.



 Pre-treatmentBM (T1) was characterized by reduction of normal hematopoiesis (p=0.009), in particular classical dendritic cells type 1 (cDC1) (p=0.066) and monocytes (p=0.034). High levels of AML cells were associated with significantly higher levels of mature naïve B cells (p=0.002), higher levels of class-switched B cells (p=0.030) and enrichment of inflammatory monocyte-like cells with MDSC phenotype. Regarding AML cells, 8 different transcriptional subtypes were identified across patients: 4 clusters with highly immature properties (undifferentiated and LMPP-like AML cells) and 4 clusters with signs of maturation (aberrant erythroid-like, EMP-like, promyelocyte-like and monocyte-like AML cells). The immature AML clusters showed high similarities in their transcriptional and signaling patterns, high stemness, quiescence, strong expression of homing factors (CD44, ITGA4), DNA instability and signs of chemoresistance such as upregulation of beta-Catenin, Trail and TGFb pathways. Furthermore, an upregulation of NFkB, TNFa and JAK/STAT signaling was observed in these AML cell clusters, indicating the formation of a highly inflammatory and dysfunctional immune microenvironment. This environment was characterized by a decrease of cDC1, presence of IFIT-high monocyte-like MDSCs, clonal expansion of senescent T cells as well as functional disturbation of antigen-presenting cells, T and NK cells with high expression of TGFb. Interestingly, each AML cell cluster showed a distinct expression profile of immune checkpoint markers, with most prominent expression of VISTA in monocyte-like AML cells. Spatially, VISTA-positive monocyte-like AML cells surrounded other AML cell populations in the BM niche. In addition, monocyte-like AML cells showed high expression of hypoxia- and metabolism-associated signatures (e.g. OXPHOS, fatty acid/cholesterol homeostasis), which might have supported the dysfunctional microenvironment. These microenvironmental changes were largely reversible after chemotherapy (T2).



 Our study reveals AML-induced dysregulation of the BM immune microenvironment, which seems reversible upon AML treatment with standard chemotherapy. Our data suggest that immature AML cells sustain the leukemic stem cell pool, whereas more mature EMP- and promyelocyte-like AML cells contribute to the bulk of AML. Furthermore, monocyte-like leukemic cells may shape the microenvironment to safeguard AML, possibly by VISTA-induced modulation of the immune system. This might explain limited therapeutic efficacy with currently available checkpoint inhibitors in AML. In the long run, identification of specific immune signatures in AML may improve our understanding of AML biology and inform therapeutic algorithms with regard to immunotherapeutic strategies in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da4a552fa669ffb6977b0dc73a6ab66c9e44bb5" target='_blank'>
              AML maturation stages associate with disease-permissive and immunosuppressive effects in newly diagnosed AML patients: An analysis at the single-cell level
              </a>
            </td>
          <td>
            Jana Ihlow, J. Schrezenmeier, A. Trinks, M. Milek, Seunghoon Kim, Julia Jesse, Eva Tranter, Christian Schneider, Denise Bachorska, Dieter Beule, L. Penter, Sandrine Sander, Simon Haas, M. Morkel, C. Sers, Nils Bluethgen, David Horst, Lars Bullinger, P. Bischoff, J. Westermann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="While FLT3 inhibitors have significantly improved the treatment of aggressive FLT3-mutated acute myeloid leukemia (AML), the emergence of resistance remains as a major challenge. Here, we applied our recently developed single-cell lineage tracing method ReSisTrace to identify cells that are intrinsically resistant or sensitive to the FLT3 inhibitors midostaurin and quizartinib in AML with FLT3-ITD mutations. Comparison of the gene expression profiles of these cells revealed transcriptional resistance signatures, including upregulation of GSPT1. Depletion of GSPT1 with CRISPR-Cas9-mediated knockout resulted in increased sensitivity of AML cells to quizartinib treatment. Further, targeting GSPT1 with the small molecule CC-90009 exhibited strong synergistic effects when combined with FLT3 inhibitors in the FLT3-ITD cell lines and primary AML patient samples. In addition, in an FLT3-ITD-positive AML patient-derived xenograft (PDX) mouse model, the CC-90009 and quizartinib combination showed significantly higher anti-tumor efficacy and prolonged overall survival compared to either treatment alone. Furthermore, compounds that induced transcriptomic changes opposite to the resistance signatures prompted cells to acquire FLT3 inhibitor-sensitive states. Vistusertib (mTOR inhibitor), linsitinib (IGF1R and insulin receptor inhibitor), and meisoindigo (IGF1R and Src family kinase inhibitor), all inhibiting pathways parallel to or downstream of oncogenic FLT3 signaling, were predicted and validated to sensitize FLT3-mutated cell lines and primary cells to FLT3 inhibitors. Collectively, these findings demonstrate the ability of ReSisTrace to unveil pre-existing transcriptional features of treatment vulnerability in hematological cancers and elucidate strategies for enhancing FLT3 inhibitor treatment efficacy in FLT3-ITD-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84189684c5cf7142729adb65c133047ed0e40a88" target='_blank'>
              Single-Cell Lineage Tracing Uncovers Resistance Signatures and Sensitizing Strategies to FLT3 Inhibitors in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Johanna Eriksson, Shuyu Zheng, M. Popa, Jie Bao, Jun Dai, Wenyu Wang, Emmet McCormack, Anna Vähärautio, Jing Tang
          </td>
          <td>2025-11-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 We previously described three subtypes of Waldenstrom's Macroglobulinemia (WM) using a multi-omic data set derived from 249 untreated patients with WM who had detectable MYD88 mutations. The B-cell like (BCL) and Plasma cell like (PCL) subtypes have distinct cell of origin but evolve from a shared transcriptional subtype (Early WM) found at early stages of disease development (Hunter et al, Manuscript in Review. Research Square, preprint 2025). We also described the WM evolutionary score (EScore) that corresponds to the progression from smoldering WM to symptomatic disease correlating with time to first therapy and WM bone marrow involvement. The transcription factor (TF) network differences that underlie these findings remain poorly understood. In this study, we examined changes in implied gene regulatory networks between WM and healthy donor memory B-cells (HDMB; CD19+CD27+), with EScore, and between WM subtypes. Findings were further analyzed for expression, alternative splicing, and TF motif enrichment in differentially regulated ATAC peaks.



 Sample specific gene regulatory networks were modeled using PANDA (Glass et al. Plos One, 2013) followed by LIONESS (Kuijer et al. iScience, 2019). Together, these calculate an interaction score for each TF/gene pair for per sample. These scores were summed up by TF and by gene to model to net regulatory connectivity known as the target score (TS). LIONESS and TS data were analyzed using limma in R and top networks were derived from the results. Top TF and gene hits were analyzed further for changes in expression and isoform composition based on Gencode augmented by novel isoforms discovered in our IsoSeq long read analysis of HDMB and WM samples (Hunter et al, ASH 2024). TF motif enrichment from differentially regulated ATACSeq peaks were calculated using MEME Suite's SEA.



 Network analysis of EScore TF enrichment revealed AHR and TFAP4 as top drivers. AHR is a regulator of xenobiotic metabolism which was already known to be negatively associated with EScore from our previous Hallmark gene set enrichment analysis of the expression data. Expression of AHR was negatively correlated with EScore (r = -0.73; p<0.0001) with the median expression falling by a factor of 12 from the earliest to latest EScore levels. AHR has been reported to inhibit B-cell class switching and plasma cell differentiation while upregulating the pro-inflammatory PTGS2 (cyclooxygenase-2) and anti-apoptotic BCL2L1 (BCL-XL) genes. The correlation with PTGS2 (r=0.641; p<0.0001) and BCL2L1 (r=0.266; p<0.0001) was observed in this study. Notably BCL2 was negatively correlated (r=-0.552; p<0.0001) consistent with a change in BCL2 family dependency during progression to symptomatic disease. Unlike AHR, TFAP4 was both transcriptionally upregulated and showed increased TS with EScore, consistent with its reported role as an oncogene. Most of the AP-1 transcription factor family also appeared in the top EScore network results including JUNB, FOS, FOSL1, FOSL2, ATF3, and MAF. Moreover, JUNB, FOS, FOSL2, and FOSB were all top enriched motifs in differentially closed chromatin between early and late EScore (p<0.0001 for all). Transcriptional levels of FOS and FOSL2 decreased >15x with EScore (p<0.0001 for both) while FOSB and JUNB levels were largely steady despite notable TS enrichment. Isoform analysis further revealed that while FOSB expression in WM is over 2x higher than HDMB expression, HDMB expression is primarily intron retained (ENST00000587358) while WM primarily expressed the full-length transcript (ENST00000353609). Comparing HDMB to WM Subtypes revealed strong differences in TF and gene TS with corresponding isoform dysregulation including PRDM1, PRDM4, IL15, RCAN3, CRIP2, and CD19. Despite increased plasmacytic differentiation, CD19 was expressed at only slightly lower levels than HDMB. However, HDMB primarily expressed a novel isoform skipping exon 2 while WM expressed the canonical transcript (p <0.0001). Given the important role of CD19 in BCR and TLR signaling, our findings indicate a critical dependency for CD19 in WM.



 Our extensive regulatory network analysisutilizing a large multi-omic data set identified several novel therapeutic targets including TFAP4 and CD19, as well as the AHR, TFAP4, and AP-1 transcription factors as key drivers of inflammatory signaling, BCL2 family dysregulation and malignant cell growth for symptomatic disease progression in WM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b66707f752fe87e364445e1c73a482a07260f60" target='_blank'>
              Regulatory network analysis identifies changes in ahr, TFAP4, and AP-1 transcription factors as top drivers of symptomatic disease progression in Waldenstrom's macroglobulinemia.
              </a>
            </td>
          <td>
            Z. Hunter, Derrick DeConti, Brian Lawney, N. Tsakmaklis, M. L. Guerrera, A. Kofides, Xia Liu, Shirong Liu, Hao Sun, Dominic Pizzarella, Abigail Peachey, Alberto Guijosa, J. Hatcher, K. Meid, Christopher Patterson, Torsten Steinbrunn, Constantine S Mitsiades, John Quackenbush, A. Branagan, S. Sarosiek, J. Castillo, S. Treon
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) is a clinically and genetically heterogeneous disease. While traditional classification systems such as FAB relied on morphology and immunophenotype, recent classifications (WHO, ICC) emphasize genetic alterations for diagnosis and therapy. However, more directly involved in the phenotypic cellular inheritance, the epigenomic profile may offer an additional dimension for understanding AML heterogeneity. Previously, we performed ATAC-seq of 1,536 AML samples and identified 16 epigenetically defined AML subgroups (subgroups A-P), which were associated with distinct clinical, genetic, and transcriptional features (Ochi et al., ASH 2024). In the present study, we extend these findings by using single-cell RNA and ATAC-seq (scRNA/ATAC-seq) profiling to explore intra- and inter-tumor epigenetic heterogeneity, transcriptional regulation, and hierarchical differentiation trajectories across AML subgroups.
 Methods We performed multi-platform scRNA/ATAC-seq on 36 AML samples including all the 16 epigenomic subgroups and 4 remission samples (as controls), profiling a total of 281,167 mononuclear cells. Computational analyses included cell clustering, projection onto normal hematopoiesis reference maps, pseudotime inference, transcription factor (TF) activity analysis using SCENIC+, and leukemic stem cell (LSC) signature scoring.
 Results scATAC-based clustering showed that leukemic cells from individual AML samples formed distinct clusters largely separated from normal cells, typically co-clustered by subgroup, regardless of their differentiation status, indicating that leukemic cells within each subgroup share a distinct intrinsic epigenetic program. Projection onto a normal hematopoiesis reference map revealed divergent differentiation arrest among subgroups, even within genetically similar cases. For instance, all NPM1-mutant subgroups (D–F) were HOX-related but differed in differentiation states: subgroup E was arrested at the HSC stage, D at the GMP stage, and F contained both progenitors and mature cells. Single-cell analysis also refined differentiation states of subgroups F-H: while bulk ATAC-based deconvolution suggested monocyte enrichment in these subgroups, single-cell analysis revealed mature monocytes predominated in H, whereas F and G were enriched for immature promonocytes.
 Combined with LSC signature analysis based on gene expression, pseudotime analysis demonstrated that cells showing high LSC scores consistently mapped to early stages of the differentiation trajectory across all subgroups. These results indicate the presence of a conserved leukemic hierarchy, with stem-like cells positioned at the apex, irrespective of epigenetic subgroup, thereby underpinning AML pathogenesis. We further analyzed TF activity using the SCENIC+ program based on scRNA/ATAC-seq data, which identified key TFs enriched in each subgroup, many of which overlapped with those found in bulk RNA/ATAC-seq analysis. Pseudotime analysis further showed that TFs exhibited subgroup-specific activation dynamics. For example, HOXA9 was activated throughout the myeloid differentiation trajectory in HOX-related subgroups but peaked at different stages according to subgroups. In the RUNX1-enriched subgroup, IRF8 and BCL11A were both active but peaked at mature and immature stages, respectively. These findings indicate that key TFs exhibit subgroup-specific activation patterns at distinct stages along the myeloid differentiation trajectory.
 Conclusion Single-cell multi-omics analysis reveals that AML epigenetic subgroups exhibit distinct but hierarchically organized differentiation trajectories, driven by dynamically regulated TF programs. These findings refine our understanding of AML diversity and may inform more precise, differentiation-stage–specific therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92f306fb3417df7e3eff8b67fe8a062b478725" target='_blank'>
              Single-cell multi-omics uncovers differentiation-dependent transcriptional programs shaping epigenetic subgroups in AML
              </a>
            </td>
          <td>
            Y. Ochi, Markus Liew-Littorin, Y. Nannya, S. Bengtzén, Bénédicte Piauger, Stefan Deneberg, M. Jadersten, Vladimir Lazarevic, Jörg Cammenga, Anna Robelius, L. Wennström, Emma Olander, S. Kasahara, N. Hiramoto, N. Kanemura, N. Sezaki, M. Sakurada, M. Iwasaki, J. Kanda, Yasunori Ueda, Satoshi Yoshihara, Tom Erkers, Nona Struyf, Yu Watanabe, Masanori Motomura, M. Nakagawa, R. Saiki, Hidehito Fukushima, Koji Okazaki, S. Morimoto, A. Yoda, R. Okuda, Shintaro Komatsu, Guoxiang Xie, Albin Osterroos, A. Kon, Lanying Zhao, Yuichi Shiraishi, Takayuki Ishikawa, Satoru Miyano, Shuichi Matsuda, A. Takaori-Kondo, Hiroyuki Aburatani, Hiroshi Suzuki, Olli-P. Kallioniemi, Gunnar Juliusson, Martin Höglund, Sören Lehmann, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="


 Background: Extranodal NK/T-cell lymphoma (NKTCL) is an aggressive non-Hodgkin lymphoma subtype. Despite improved outcomes with immunotherapy, persistent resistance in a subset of patients underscores an unmet need to delineate underlying molecular drivers. Lipid metabolic reprogramming provides energy and biosynthetic precursors for tumor growth while actively remodeling the tumor microenvironment to suppress immunity. Thus, cotargeting immune checkpoints and lipid metabolism represents a promising strategy to overcome resistance; however, its mechanistic basis and clinical applicability in NKTCL remain elusive.
 Methods: Using complementary multiomics profiling—single-cell RNA sequencing (scRNA-seq; n = 13 patients), bulk RNA sequencing (RNA-seq; n = 24 patients), and metabolomics(n = 22 patients)—in the prospective SPIRIT cohort (NCT04127227), we identified reprogrammed lipid metabolism as a hallmark of immunotherapy resistance in NKTCL. To mechanistically dissect this, we deployed dual parallel systems: (1) therapy-resistant models and (2) CRISPR-based gene-editing platforms across in vitro settings and humanized mouse models. Functional validation was performed by leveraging these systems with multiomics analyses, including bulk RNA-seq, scRNA-seq, RNA immunoprecipitation sequencing (RIP-seq), methylated RNA immunoprecipitation sequencing (MeRIP-seq), and metabolomics.
 Results: YTHDF3 deficiency stabilizes PI3K-activating transcripts (IRS1, IGF1R, PIK3R6), driving constitutive PI3K signaling and lipid metabolic reprogramming. This hyperactivated lipid metabolism induces endoplasmic reticulum stress (ERS), promoting ERS-mediated MHC-I downregulation to confer immunotherapy resistance. Translating these findings, we designed a dual-targeting strategy synergizing PI3K inhibition (to reverse metabolic immunosuppression) and anti-PD-1 therapy (to block immune checkpoint evasion), which demonstrated efficacy in murine models. This therapeutic strategy is now being evaluated clinically in a phase Ib trial (NCT06793956).
 Conclusion: Our study delineates the YTHDF3–PI3K–lipid metabolism axis as a central driver of immunotherapy resistance in NKTCL. This mechanism-guided framework, with dual targeting of PI3K and PD-1, bridges preclinical validation to clinical translation (NCT06793956).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41871b050607d2beba386ede97a14d32b3a24966" target='_blank'>
              Lipid metabolic reprogramming mediated by YTHDF3 deficiency underlies immunotherapy resistance in extranodal NK/T-cell lymphoma
              </a>
            </td>
          <td>
            Jinni Wang, Q. Cai, Wenlong Zhong, Yuchen Zhang, Danling Dai, Yi Cao, Shuai Liang, Jun Cai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains one of the deadliest brain tumors, with minimal and short-lived response to temozolomide (TMZ), the frontline chemotherapeutic. To uncover the genetic circuitry driving this resistance, we conducted a large-scale in vivoCRISPR-Cas9 loss-of-function screen targeting 2,500 genes involved in transcriptional regulation, RNA splicing, and epigenetic modification. Using 8 sgRNAs per gene across multiple collaborative laboratories, our unbiased screen revealed a striking spectrum of genes whose disruption either conferred resistance or heightened sensitivity to TMZ. Among resistance-associated hits were RNF10, WHSC1, SMAD2, GFI1, ZNF551, TRIM33, MNDA, HNF1A, POGZ, EID2B, and ARID1A—with ARID1A emerging as a top candidate regulating epigenetic plasticity and therapy evasion. Conversely, genes such as ASCL1, SNAI2, MED12, ATF7, VRK1, ZNF26, HOXB8, SEBOX, and ZSCAN4 sensitized tumors to TMZ. For functional validation, sgRNAs from GeCKO and Brunello libraries were transfected into U251 GBM cells, with knockout efficiency confirmed by Western blot. TMZ response was quantified using CellTiter-Glo viability assays across drug gradients. Strikingly, loss of resistance-driving genes sensitized GBM cells to TMZ, while knockout of sensitizer genes reversed this effect—confirming their functional roles in chemoresistance. To dissect spatial dynamics, we employed high-resolution spatial transcriptomics, mapping the expression patterns of resistance- and sensitivity-associated genes across the tumor microenvironment. This integration revealed spatially defined gene programs contributing to therapy failure or response. Together, our work decodes key regulators of chemoresistance in GBM and nominates a set of high-priority, druggable targets—such as ARID1A—for therapeutic development. These insights open new avenues for overcoming treatment resistance and improving outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dfba0793caecc975a3f7afe28cb4d10dfae7bd7" target='_blank'>
              DDDR-70. Unlocking the Genetic Code of Chemotherapy Response in Glioblastoma
              </a>
            </td>
          <td>
            Lara Koutah, Shengin Liu, Cheol Park, J. Duffy, Umme H Faisal, Rafał Chojak, S. Budhiraja, C. Dmello, Atique U. Ahmed
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20ec8c67f8ff833438b56a407614424d68047c71" target='_blank'>
              Single-cell transcriptome analysis defines novel molecular subtypes and reveals therapeutic implications of T/myeloid mixed-phenotype acute leukemia.
              </a>
            </td>
          <td>
            Bin Huang, Wenjie Liu, Yuxin Du, Ping Liu, Zixing Lu, Shiyang Zhong, Xingfei Hu, Wanting Zhou, Yuzhu Shi, Runheng Huang, Xian Zhang, Jinning Shi, Chuanyang Lu, Chunling Wang, Liang Yu, Lingxiang Wu, Wei Wu, Peng Xia, Qian Sun, Liwei Zhu, Zhi Wang, Ruohan Zhang, Xinrui Lin, Sali Lv, Qianghu Wang, Sixuan Qian, Kening Li, Mingyao Hong
          </td>
          <td>2025-11-29</td>
          <td>Genome medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Introduction Treatment-free remission (TFR) remains unpredictable in chronic myeloid leukemia (CML) management, with half of patients relapsing after tyrosine kinase inhibitor (TKI) cessation due to persistent, difficult-to-isolate leukemic stem cells (LSCs). Current predictors, such as TKI therapy duration, deep molecular response (DMR) length, or BCR::ABL1 halving kinetics, are insufficient to precisely identify which patients harbor relapse-fated clones at diagnosis. Critically, no tools currently allow for relapse risk stratification at diagnosis, thereby limiting opportunities for early pre-emptive intervention. While previous studies have suggested that relapse biology may be preconfigured in diagnostic LSCs, specific molecular signatures predictive of future relapse have not been clearly defined.
 Methodology To address this gap, we performed single-cell RNA sequencing (scRNA-seq) on a rare, clinically well-annotated cohort (n=52) of diagnostic bone marrow (BM) [Australia, n=20 and Singapore, n=4)] samples and peripheral blood (PB) [Australia, n=18 and Japan, n=10)] from CML patients stratified by TFR outcome. This approach enabled retrospective linkage of stem cell transcriptional states to long-term clinical fate. Such diagnostic CML samples were compared to 7 healthy donors. We profiled 149,000 BM cells, including 83,686 CD34⁺ hematopoietic progenitors, and applied integrative analyses including gene set enrichment analysis (GSEA), gene set variation analysis (GSVA), differential gene expression testing, and CellChat-based intercellular communication analysis to characterize LSC states, pathway activity, and microenvironmental interactions. In parallel, we profiled 35,525 CD34⁺ cells from the PB to determine whether diagnostic biomarkers identified in the BM compartment are preserved in the peripheral circulation.
 Results We compared diagnostic LSCs (2692 BCR::ABL+ and 3031 BCR::ABL1- stem cells) from patients who maintained remission versus those who relapsed following TKI cessation. Gene Set Variation Analysis (GSVA) demonstrated stark transcriptional differences at diagnosis between these groups. LSCs from relapsing patients showed pronounced enrichment of cell cycle progression programs, including E2F targets and DNA replication, and modest enrichments in lipid storage and immune signalling. Intriguingly, a closer examination resolved two molecularly distinct, relapse-fated LSC subtypes at diagnosis: a proliferative subtype (Relapse-P; n = 6) defined by E2F-driven transcriptional programs (DNA replication), metabolic rewiring, and oncogenic signalling; and an immune subtype (Relapse-I; n = 8) characterized by inflammatory signalling, cell adhesion processes, and fatty acid metabolism. Both subtypes exhibited a Short-Term Hematopoietic Stem Cell (ST-HSC)-like state of activated quiescence, marked by CD71 expression. Critically, quiescent Relapse-P LSCs demonstrated a druggable, non-canonical PRC1.1 (Polycomb Repressive Complex) activation, revealing a targetable vulnerability absent in remission cells. Cell-cell communication analysis uncovered self-reinforcing autocrine circuits specifically enriched in relapse LSCs, notably sterol metabolism (DHEA-PPARA/ESR1) and adhesion signalling (NAMPT-ITGA5/ITGB1). Common to both the Relapse-P and Relapse-I subgroups, we identified the cancer testis antigens, SPAG6 and PRSS21 as BM-specific candidate biomarkers. However, their expression was markedly diminished in the Relapse-PB cohort, with only SPAG6 retained at reduced sensitivity. These observations underscore the need for BM-centric biomarker development at the point of CML diagnosis.
 ConclusionOur study provides compelling evidence that relapse fate in CML is molecularly pre-encoded within diagnostic LSCs, years before therapy cessation and enables early risk stratification via SPAG6, PRSS21 and CD71 as markers of BM-derived LSCs. This atlas provides a valuable resource for dissecting LSC-intrinsic programs and immune–LSC interactions, while also enabling two key clinical applications: (1) development of early biomarkers based on BM-derived LSC signatures, and (2) rational design of combination therapies (e.g., TKI + epigenetic drugs) to improve curative outcomes in CML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d83ada8785fcbbe1669148755f0161eae9c08c" target='_blank'>
              A single-cell atlas of diagnostic bone marrow to uncover the origins of CML relapse following therapy cessation
              </a>
            </td>
          <td>
            Vaidehi Krishnan, Xi Ren, K. Lee, Radhika Sonthalia, P. Venkatesh, Mengge Yu, Pavithra Shyamsunder, John Ouyang, Xinyi Yang, Florian Schmidt, V. Saunders, Naranie Shanmuganathan, Charles Chuah, Naoto Takahashi, S. Branford, Timothy Hughes, Shyam Prabhakar, S. T. Ong
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged (KMT2A-r) acute myeloid leukemia (AML) is characterized by epigenetic dependencies, yet it often remains clinically resistant to therapies targeting individual regulators. This suggests that resistance is not attributable to any single regulator but is an emergent property of a compensatory epigenetic network. A systematic map of this network's architecture is essential for developing rational and durable combination therapies.
 Methods: We performed a high-throughput Perturb-seq screen targeting 16 key epigenetic regulators in the KMT2A-r AML cell line MOLM-13. Using computational modeling, we identified co-regulated transcriptional modules and inferred functional interactions. Key findings were validated through pharmacological inhibition assays (in vitro), bulk RNA-sequencing, and clinical correlation analyses using two large, independent patient cohorts (TCGA, n=151; Beat AML, n=672).
 Results: Our analysis uncovered a core transcriptional module, which we termed the “Myeloid Program”, that was silenced by a compensatory epigenetic repression circuit. This circuit is maintained by a synergistic hub where KAT6A, Menin, and DOT1L converge to maintain leukemic identity. While individual perturbations of these hub components only partially derepressed the Myeloid Program, their simultaneous pharmacological inhibition collapsed the circuit's buffering capacity, leading to robust Myeloid Program reactivation and potent synergistic anti-leukemic activity (Menin/KAT6A ZIP score: 20.19; DOT1L/KAT6A ZIP score: 25.94). Bulk RNA-seq analysis confirmed that dual Menin/KAT6A inhibition induced a unique transcriptional state, synergistically and robustly upregulating the Myeloid Program (NES = 1.77, p-adj = 2.13e-08) more effectively than either single agent. Clinically, high Myeloid Program activity served as a powerful independent prognostic biomarker for improved overall survival (TCGA HR = 0.91, p = 0.0017; Beat AML HR = 0.29, p = 0.046). Our model also predicts potential drug interaction antagonisms. For example, we showed that disruption of an antagonistic regulator, the PRC1.1 component PCGF1, confers strong resistance to DOT1L inhibition (IC50 shift >50-fold). Finally, we established the Myeloid Program as a predictive biomarker whose low activity score identifies a vulnerable, undifferentiated state with unique sensitivity to PI3K/AKT/mTOR pathway inhibitors (e.g., Selumetinib, r = -0.37, p = 2.2e-16; MK-2206, r = -0.31, p = 1.2e-11; Rapamycin (r = -0.34, p = 8.44e-14). Consequently, combining a differentiation-inducing Menin inhibitor with agents targeting this pathway (e.g., MK-2206, ZIP: 21.738, Rapamycin, ZIP: 16.224) was highly synergistic.
 Conclusion: Our work establishes circuit-level epigenetic compensation as a core mechanism of resistance in KMT2A-r AML. We identify the Myeloid Program as a central regulatory node, a dual-purpose biomarker, and a key therapeutic objective. These findings provide a data-driven rationale for two distinct precision strategies: 1) collapsing the compensatory repressive circuit with synergistic co-inhibition of its core components, or 2) exploiting the emergent vulnerability of the undifferentiated state by combining differentiation agents with PI3K/mTOR pathway inhibitors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4815f0ba4a8b914671857f7c82e068e94df379f" target='_blank'>
              A perturb-seq map of a differentiation hub reveals synergistic vulnerabilities in KMT2A-rearranged AML
              </a>
            </td>
          <td>
            Sajesan Aryal, Brittany Curtiss, Xinyue Zhou, Rui Lu, Changde Cheng
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosome instability (CIN) remains among the most important problems in modern cancer research. In this study, we conducted a genome-wide RNAi screen to identify genes that contribute to CIN. To achieve this, we used a human artificial chromosome in a novel sensitized screen to measure CIN. We screened 18,658 genes for their roles in maintaining chromosomal stability and identified 834 candidates as potential CIN genes. A secondary RNAi screen identified 44 genes with the most pronounced CIN phenotypes. In guilt-by-association analysis using a published set of 8,498 proteins across a panel of 949 cancer cell lines, this cohort of 44 genes displayed a striking correlation with mitotic regulators. Furthermore, altered expression of these proteins was associated with a poor prognosis across multiple cancer types. Specifically, downregulation of AMY2B, ALAD, PDGFRA, PPIE, VEZ1, and TTC19 is associated with poor survival in two or more of the following: small cell lung cancer, lung adenocarcinoma, adrenocortical carcinoma, ovarian cancer, and breast cancer. The genes identified in this screen hold potential as prognostic markers for patient survival across several cancer types and could potentially serve as targets for the development of new therapeutic approaches aimed at mitigating CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48402b9da158c79dbc9aec979b9f8a854d30f0b3" target='_blank'>
              A genome-wide RNAi screen for novel CIN genes using human artificial chromosome.
              </a>
            </td>
          <td>
            Mikhail Liskovykh, N. Y. Kochanova, Chih-Yuan Chiang, A. Dhall, V. Aksenova, Yu-Chi Chen, William C Reinhold, M. Dasso, Anish Thomas, , Y. Pommier, William C. Earnshaw, Vladimir Larionov, N. Kouprina
          </td>
          <td>2025-12-01</td>
          <td>PNAS nexus</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Large parallel genetic screens have been used to identified targets and regulators that enhance T cell antitumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducted a pathway enrichment analysis. A T cell screen perturbation score (TPS) metric was employed to quantify the impact of a gene perturbation on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease cohort and T cell-derived cancer patients were incorporated to gain further insight on gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 35 T cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is accessible through our webserver (http://tcpgdb.sidichenlab.org/) and allows users to interactively explore the impact of query genes on T cell function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39e4489bcd5040b80166991d7202bfd2ac7d128" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators.
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2025-11-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="As a novel lactate-derived post-translational modification, histone lactylation links metabolic reprogramming and epigenetic regulation in cancer. Histone lactylation, particularly at histone H3 lysine 18 lactylation (H3K18la), has been implicated in tumor initiation, progression, metastasis, immune evasion and therapy resistance. It modulates oncogenic pathways (such as PI3K/Akt/mTOR, NF-κB, JAK/STAT) and metabolic pathways (such as glycolysis enhancement, fatty acid synthesis via stearoyl-CoA desaturase and glutamine metabolism) and by altering chromatin structure and gene transcription. In the tumor microenvironment, lactate-induced H3K18la polarizes macrophages toward an M2 phenotype, upregulates immune checkpoints and induces CD8+ T cells dysfunction, which promotes immunosuppression. However, CD8+ T cell-intrinsic lactylation may enhance antitumor immunity during checkpoint blockade. Histone lactylation also induces chemoresistance via autophagy activation, DNA repair and ferroptosis suppression. Therapeutic strategies targeting lactylation include inhibiting lactate transporters, glycolysis or regulation enzymes (such as E1A-binding protein, lysine acetyltransferase 2A and brahma-related gene 1). Furthermore, the clinical potential is emerging, with H3K18la and H4K5la serving as prognostic biomarkers in multiple types of cancer. However, key questions regarding the non-enzymatic modification mechanisms, identification of histone lactation regulatory enzymes and pan-cancer functional heterogeneity are yet to be elucidated. Future research should prioritize translational validation of lactylation-targeted therapies and their integration with existing regimens to overcome resistance and improve immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acaf3fa0db490490209755ee44e28fd45be81701" target='_blank'>
              From mechanism to targeted therapy: Advances in histone lactylation-driven cancer progression (Review)
              </a>
            </td>
          <td>
            Zhe Jia, Shan Lu, Zhen-chuan Wang, Pengfei Ge
          </td>
          <td>2025-11-11</td>
          <td>Oncology Letters</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa98043059e18d9fd50446cb2c494c87e5e2054" target='_blank'>
              Dynamics of clonal hematopoiesis and cellular responses to stress-induced toxicity in autologous stem cell transplantation.
              </a>
            </td>
          <td>
            Catarina M Stein, Raphael Hablesreiter, F. Christen, P. Löwe, Coral Fustero-Torre, Klara Kopp, Benjamin N. Locher, Lena Nitsch, Robert Altwasser, Johanna Franziska Kerschbaum, L. Bullinger, Leif S. Ludwig, Paulina M Strzelecka, Frederik Damm
          </td>
          <td>2025-12-19</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="CRISPR screening has revolutionized cancer research by enabling systematic, genome-wide interrogation of gene
function to identify therapeutic vulnerabilities, resistance mechanisms, and immunotherapy determinants. This review
examines methodological approaches for CRISPR screenings using the Cas9 systems, evaluating how system and methodology selection
profoundly influence which biological mechanisms are revealed across therapeutic resistance, metastasis regulation, and immunotherapy
response. Cas9-based screens-from simple one-cell-type (1CT) in vitro approaches to complex autochthonous in vivo systems-excel at
identifying cell-autonomous DNA-level dependencies. Two-cell-type (2CT) co-culture approaches validate immune-dependent mechanisms,
while in vivo screens capture physiological complexity and have identified clinically relevant targets including PTPN2, now in Phase 1 trials.
Critical challenges include incomplete gene editing, false-positive and false-negative hits, context-dependent essentiality, and system-specific
technical limitations. Emerging technologies integrating artificial intelligence, prime editing, single-cell profiling, and patient-derived models
promise to address limitations while enabling functional precision medicine. This review provides a framework for selecting appropriate
systems, interpreting results within methodological constraints, and integrating complementary approaches to maximize biological insight and
translational impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6e21f4ff0a121855b3f14e7fcc069197b5d3507" target='_blank'>
              CRISPR- CAS SCREENING IN CANCER RESEARCH: METHODOLOGIES, APPLICATIONS, AND TRANSLATIONAL CHALLENGES
              </a>
            </td>
          <td>
            Ahana Patel
          </td>
          <td>2025-11-01</td>
          <td>INDIAN JOURNAL OF APPLIED RESEARCH</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 β-hemoglobinopathies, the most common monogenetic blood disorders, result from defective function or inadequate production of β-globin. Unfortunately, accessible and efficient treatments remain limited. Previous studies demonstrated that β-hemoglobinopathies patients with increased naturally occurring β-like subunit, γ-globin, a component of fetal hemoglobin (HbF), exhibit milder disease severity. Investigating novel factors that upregulate γ-globin expression could provide new therapeutic avenues for patients affected with β-hemoglobinopathies.
 In this study, we performed an unbiased genome-wide pooled CRISPR activation (CRISPRa) screen to identify novel γ-globin regulators. First, we generated a clonal HUDEP-2 cell line stably expressing the MPH activation complex (MS2-P65-HSF1). This cell line was then transduced with a lentiviral library expressing a VP64 activation element fused to a catalytically dead Cas9, a Blasticidin resistant gene, and one of 3 sgRNAs targeting virtually every gene in the human genome. On day 8 of differentiation, intracellular HbF staining was performed, and the 10% highest and lowest HbF-expressing cells (F-Cells) were sorted. sgRNAs were deep sequenced and compared between high and low γ-globin expressing cells. Expectedly, sgRNAs targeting the previously reported γ-globin repressor, BCL11A, were enriched in low γ-globin expressing cells, while sgRNAs targeting the γ-globin promotor were enriched in the high γ-globin expressing cells. Interestingly, this screen also identified several previously unknown putative γ-globin activators.
 To validate our screen results, we first chose the top 25 candidate genes for which sgRNAs were enriched in high HbF-expressing cells. For each candidate gene, we generated lentiviral particles expressing an sgRNA that targets it, VP64 fused to a catalytically dead Cas9, and a Blasticidin resistant cassette and then transduced into HUDEP-2 cells expressing MPH. Differentiated HUDEP-2-MPH cells were analyzed for percent F-Cells by flow cytometry and HBG1/2 (which encode γ-globin) mRNA levels by RT-qPCR. Compared to control cells transduced with non-targeting sgRNAs, activation of HBG2, HIC1, CHCHD7, FXR2, ANKH and SKIL resulted in >2-fold increase in %F-Cells and HBG1/2 mRNA levels. Since HIC1 is paralogous to HIC2, we confirmed that the HIC1-activating sgRNA does not additionally activate HIC2. Furthermore, we showed that HIC1 activation results in reduced BCL11A mRNA, suggesting that HIC1 may induce HbF production by suppressing BCL11A, a hypothesis we are currently testing.
 Next, we sought to validate our findings in primary human erythroid cells. To do so, we co-transduced primary human hematopoietic stem and progenitor cells (HSPCs) undergoing erythroid differentiation with two lentiviral constructs, one expressing MPH and a hygromycin resistant cassette and another expressing an sgRNA targeting a gene of interest, VP64-dCas9, and a Blasticidin resistance cassette. We found that, compared to control cells transduced with non-targeting sgRNAs, activation of HBG2, HIC1, CHCHD7, FXR2, ANKH and SKIL led to >3-fold increase in γ-globin mRNA levels, validating our HUDEP-2 findings described above. In additional preliminary results, we found that HIC1 and CHCHD7 cDNA overexpression in HSPCs resulted in increased %F-Cells and γ-globin mRNA. Further studies aimed at defining the mechanisms by which activation of the genes identified by our screen result in HbF induction are ongoing.
 In summary, we identified novel regulators of γ-globin expression. Our findings have important implications for furthering our understanding of β-globin switching, with important implications for β-hemoglobinopathies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b5f338b0fc7d497ea0e52566fa71e75ee9058a9" target='_blank'>
              A genome-wide CRISPR activation screen identifies novel γ-globin regulators
              </a>
            </td>
          <td>
            Carea Mullin, Ginette Balbin-Cuesta, Greggory Myers, Claire Drysdale, Zesen Lin, Lei Yu, Beth McGee, Ann Friedman, R. Khoriaty
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous loci linked to late-onset Alzheimer’s disease (LOAD), but the pan-brain regional effects of these loci remain largely uncharacterized. To address this, we systematically analyzed all LOAD-associated regions reported by Bellenguez et al. using the FILER functional genomics catalog across 174 datasets, including enhancers, transcription factors, and quantitative trait loci. We identified 41 candidate causal variant-effector gene pairs and assessed their impact using enhancer–promoter interaction data, variant annotations, and brain cell-type-specific gene expression. Notably, the LOAD risk allele of rs74504435 at the SEC61G locus was computationally predicted to increase EGFR expression in LOAD-related cell types: microglia, astrocytes, and neurons. Functional validation using promoter-focused Capture C, ATAC-seq, and CRISPR interference in the HMC3 human microglia cell line confirmed this regulatory relationship. Our findings reveal a microglial enhancer regulating EGFR in LOAD, suggesting EGFR inhibitors as a potential therapeutic avenue for the disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d07f394fd7eb383df0b9014ab8e1eef1a6183ec" target='_blank'>
              Integrated genomic analysis and CRISPRi implicates EGFR in Alzheimer’s disease risk
              </a>
            </td>
          <td>
            Y. Leung, Pavel P. Kuksa, Luke Carter, Jeffrey Cifello, Emily Greenfest-Allen, O. Valladares, Louisa Boateng, Shannon Laub, Natalia Tulina, Sofia Moura, Aura Maria Ramirez, K. Celis, Fulai Jin, R. Feng, Gao Wang, Phil De Jager, Jeff Vance, Liyong Wang, Struan F. A. Grant, Gerard D. Schellenberg, A. Chesi, Li-San Wang
          </td>
          <td>2025-12-16</td>
          <td>Npj Dementia</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Introduction: ASXL1-mutant (mt) Chronic Myelomonocytic Leukemia (CMML) is an aggressive myeloid neoplasm characterized by increased proliferation, resistance to epigenetic therapies, and poor survival. Epigenetic remodeling and up-regulation of leukemogenic driver genes are hallmarks of ASXL1mt CMML. Truncating mutations in ASXL1 activate enhancers that drive the expression of key leukemogenic driver genes such as HOXA9 and its co-factor MEIS1. Here, we investigate the chromatin accessibility of these leukemogenic enhancers by ASXL1 genotype at single-cell resolution (GoT-ChA-seq).
 Methods: After IRB approval, we performed bulk RNA-seq and ChIP-seq (H3K4me1, H3K4me3, H3K27ac, H3K27me3, H2AK119ub, H2BK20ac) on bone marrow (BM) mononuclear cells from 40 patients with CMML (19 ASXL1mt, 21 -wildtype/wt). After quality control, differential analyses of gene expression and histone modification occupancy between ASXL1mt and wt patients were conducted. A consensus peak calling framework was used to define genomic regions with ASXL1mt-specific histone modifications. Canonical enhancers were defined as genomic regions with a co-occupancy of H3K4me1 and H3K27ac. We then added regions marked by H2BK20ac to capture regulatory regions associated with p300 (Narita et al. Nat Genet 2023). We used an in silico prediction strategy to associate regulatory regions marked by strong histone modification occupancy with highly up-regulated leukemogenic driver genes (HOXA6-10, MEIS1). The ReMap database (Hammal et al. Nucleic Acids Res 2022) was queried for transcription factor binding in the genomic regions of interest. We then performed GoT-ChA-seq (Izzo et al. Nature 2024) to investigate chromatin accessibility for ASXL1mt (n=3) and wildtype BM single cells from CMML patients. In vitro colony forming assays were conducted on primary CMML BM cells (11- ASXL1mt and 9 wt CMML), assessing the efficacy of EP31670, a novel, oral, dual inhibitor of p300/BRD4.
 Results: This study included 40 patients with CMML, 19 ASXL1mt and 21 wt. The variant allele frequencies for ASXL1 were compatible with heterozygosity (median 0.41) and TET2 mutations were balanced between the two groups. We identified 149 candidate regulatory regions correlating with the expression of up-regulated target genes of interest (HOXA6-10, MEIS1) by bulk ChIP- and RNA-seq. Publicly available ChIP-seq data from myeloid cell lines demonstrated enrichment of p300 (in THP-1 cells, enrichment effect size +2.58, p=0.002) and BRD4 (in K-562 cells, enrichment effect size +1.58, p=0.045) in these candidate regulatory regions. Next, we interrogated the chromatin accessibility of ASXL1mt (n=13620) and ASXL1wt (n=24269) single cells from 3 CMML patients using GoT-ChA-seq. Differential accessibility analysis revealed 144 genomic regions, all of which were preferentially accessible in ASXL1mt cells compared to ASXL1wt cells (p<0.05 for all regions). Four of these preferentially accessible genomic regions (p<0.001 for all regions) overlapped with the previously identified candidate regulatory regions for MEIS1. The H3K27ac and H2BK20ac occupancy levels in these regions was strongly associated with the expression of MEIS1 (median Pearson r=0.73, p=1.16x10-7). These candidate regulatory regions were annotated in ENCODE as distal and proximal enhancers. Targeting this biology with the dual p300/BRD4 inhibitor EP31670 in progenitor colony forming assays led to a preferential therapeutic effect in ASXL1mt BM samples (median IC50 = 25nM) compared to in ASXL1wt samples (median IC50 = 74nM, p=0.03). Treatment with EP31670 was also associated with increased differentiation (increased blast forming units-erythroid; BFU-E) and organization of colonies in ASXL1mt vs wt CMML.
 Conclusions: ASXL1mt CMML is characterized by the up-regulation of several leukemogenic driver genes including HOXA9 and its co-factor MEIS1, secondary to the activation of genotype-specific enhancers. The preferential accessibility of these enhancers can be defined between ASXL1mt and ASXL1wt cells within patients. The enhancers of interest interact with p300 and BRD4 in myeloid cell lines and targeting them with the novel dual p300/BRD4 inhibitor, EP31670, induces therapeutic responses at low nanomolar drug concentrations in vitro, preferentially in ASXL1-mutant cells. These data solidify the biological rationale for targeting p300/BRD4 in ASXL1mt chronic myeloid neoplasms for therapeutic benefit (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d58c09cc34747d2000f4bf32710a735358c8669" target='_blank'>
              Clone-specific epigenetic regulatory mechanisms in ASXL1-mutant chronic myelomonocytic leukemia
              </a>
            </td>
          <td>
            Moritz Binder, T. Lasho, Jenna A. Fernandez, Wazim Mohammed Ismail, Amelia Mazzone, C. Finke, Pankaj Pradeep, A. Mangaonkar, Nickolas Steinauer, Huihuang Yan, Liguo Wang, Alexandre Gaspar-Maia, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Glioblastoma is the most lethal primary brain tumor in adults with over 75% of all patients succumbing to the disease within two years after diagnosis. Extensive research of the past decades has shed light on the potential causes of this dismal prognosis including the roles of glioma stem-like cells, inter-patient and intratumor heterogeneity and the tumor microenvironment in this tumor’s strong resistance to standard-of-care treatment (surgery and chemoradiotherapy). Yet, standard-of-care has remained the same for over 20 years and nearly all clinical trials have resulted in failure to substantially improve patient outcome. In this study, we put cellular plasticity forward as one of the main drivers of treatment resistance as it allows the glioma stem-like cells to survive and adapt to imposed stress conditions. Based on advanced single-cell RNA sequencing upon chemoradiation in a large selection of patient-derived glioma stem-like cells (PDGSCLs), we identified several treatment-induced metaprograms and shifts in gene regulatory networks that are potential drivers of plasticity. To investigate their functional role in these plastic cell state shifts, we used pooled CRISPR screening with single-cell transcriptome resolution (CROPseq) for a selection of 250 candidate targets in three heterogeneous PDGSCLs treated with (chemo)radiation. Several potential targets were identified to modulate chemoradiation-induced plasticity as their knockout locked cells in their current state, even after treatment. Response to chemoradiation was further investigated in vitro and in silico after (pharmacological) inhibition of these targets. Together, our findings indicate new vulnerabilities of glioma stem-like cells, thereby proposing promising new targets to combat acquired treatment resistance in glioblastoma. This study is funded by FWO (11B2325N) and the Foundation against Cancer (F/2022/2026).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d401c7f25060494f52e0c8bac105b6167947e171" target='_blank'>
              STEM-14. SINGLE-CELL CRISPR PROFILING UNMASKS TARGETABLE CHEMORADIATION-INDUCED PLASTICITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            L. Van Eupen, C. González-Blas, Julie Morscio, Mirte De Temmerman, Y. De Visser, Paulien Van Minsel, A. Claeys, M. Derweduwe, Bernard Thienpont, Paul Clement, S. De Vleeschouwer, R. Sciot, Sandra Jacobs, I. Primac, V. Vermeirssen, R. Quintens, F. De Smet
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Gliomas carry a high disease burden, particularly the aggressive IDH-wildtype subtype. However, some IDH-mutant gliomas can transition from a less aggressive G-CIMP-High (GCH) state to a more malignant G-CIMP-Low (GCL) phenotype, mimicking IDH-wildtype tumors. Understanding the cellular and molecular mechanisms behind this progression is vital for developing effective therapies.



 We propose that single-nuclei resolution analysis can reveal distinct neoplastic cell subtypes driving the GCH-to-GCL transition through activation of stemness- and proliferation-related gene programs.



 We analyzed 22 glioma specimens from 15 patients (including matched primary and recurrent tumors) using single-nuclei RNA- and ATAC-sequencing. After quality control and neoplastic cell identification via somatic CNVs, we applied non-negative matrix factorization (NMF) to identify cell subclusters based on gene expression patterns. We further characterized these clusters using gene set enrichment, protein-protein interaction networks, developmental trajectory analysis, and integration with external RNA-seq datasets. Chromatin accessibility was assessed via snATAC-seq to identify regulatory elements and transcription factor activity.



 We identified six major neoplastic cell subtypes: Astrocytic-Support, Pro-neural Housekeepers, Proteo-synthetic Cells, Mitotic Proliferators, Neural Guidance & Growth Cells, and Mesenchymal-mutant Cells. Each showed distinct gene expression signatures, pathway enrichments, and survival associations. Two subtypes—Mitotic Proliferators and Mesenchymal-mutant Cells—were linked to high-grade tumors and poor patient outcomes. Chromatin accessibility patterns corroborated transcriptional findings, highlighting differential TF regulation in aggressive subtypes.



 This study reveals two aggressive GCL-related neoplastic cell subtypes with distinct stem-like and proliferative features. These findings provide insight into glioma progression mechanisms and uncover potential molecular targets for therapeutic intervention in IDH-mutant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd05421cc91e3cb0411e5625733c706631851341" target='_blank'>
              EPCO-21. SINGLE-NUCLEI ANALYSIS IDENTIFIES NOVEL NEOPLASTIC IDH MUTANT ASTROCYTOMA PROLIFERATIVE AND STEMNESS PHENOTYPES DRIVING POOR PATIENT OUTCOMES
              </a>
            </td>
          <td>
            H. Noushmehr, G. Herrgott, L. Garofano, Ian Lee, J. Snyder, T. Walbert, A. Castro, A. Iavarone
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="


 Background: Chimeric Antigen Receptor (CAR) T-cell therapy targeting CD19 has transformed outcomes for patients with relapsed and refractory B-cell malignancies. Despite its success, more than half of treated patients ultimately experience disease recurrence, underscoring the need to better understand treatment failure and to enhance CAR T-cell effectiveness and persistence. In a prior study (Fraietta et al., Nature, 2018), our group described an exceptional clinical benefit in a patient with chronic lymphocytic leukemia (CLL), driven by the clonal expansion of a CD8+ CAR T-cell harboring a lentiviral integration in the TET2 gene. This observation led us to hypothesize that certain gene disruptions resulting from vector integration may contribute to improved therapeutic efficacy. We analyzed lentiviral vector integration sites (LVIS) from CAR T-cells across 40 patients with acute lymphoblastic leukemia (ALL) and CLL (Nobles et al., JCI, 2020). Several genes were found to be recurrently targeted and clonally dominant across patients with similar outcomes.
 Methods: To mechanistically interrogate LVIS-based disruptions associated with long-term responders, we designed a focused CRISPR-Cas9 library targeting 180 genes, using 4-8 single guide RNAs (sgRNA) per gene. Our one-shot lentiviral vector was engineered to drive the expression of both the anti-CD19 CAR and gRNAs under the promoters EF1-alpha and human U6, respectively. To generate gene-modified CAR T cells, healthy donor T cells were transduced at a low multiplicity of infection to ensure single-gene edits per cell. Three days post-transduction, CAR-positive cells were purified, electroporated with the Cas9 protein and subsequently expanded in culture for an additional 10 days. As a negative control, CAR T-cells containing the sgRNA library were mock-electroporated without Cas9 and expanded under identical conditions. The impact of the gene edits on CAR T-cell fitness was assessed in an in vitro chronic stimulation assay using CD19-expressing leukemia cells, and in our well-established in vivo immunodeficient xenograft model engrafted with the aggressive leukemia cell line NALM6. Positive hits from this screen were further validated via single gene edits using individual sgRNA-loaded Cas9 ribonucleoproteins (RNPs). Each gene was targeted using 2 sgRNAs to ensure >80% knockout efficiency. The anti-leukemia efficacy, proliferative response, cytokine polyfunctionality, CAR T cell effector and memory differentiation, and global gene expression changes were measured for the top candidates.
 Results: We identified an overlapping set of highly ranked hits (Log2FC ≥ 0.25, FDR ≤ 0.1) in our in vitro and in vivo CRISPR-Cas9 screens that significantly enhanced proliferation and effector function upon repeated tumor challenges when compared to the control groups. These genes were mostly associated with MAPK signaling, DNA damage, apoptotic pathways, cell migration, and cell cycle. Importantly, during our in vivo screen, although the edited CAR T-cells (+Cas9) presented a delayed anti-tumor response compared to unedited CAR T-cells (-Cas9), mice from the +Cas9 group survived longer with a lower tumor burden when compared to mice from the -Cas9 group. Results from our validation using Cas9-RNPs for our top-ranked genes showed that several genes exhibited increased proliferation and anti-tumor efficacy compared to unedited cells and our safe-harbor control AAVS1. Mice treated with single-edited CAR T-cells, particularly with CARD8 and SRCAP disruptions, displayed improved survival and tumor clearance compared to the control group. Phenotyping of the cells as well as RNA sequencing are currently ongoing and will be presented at ASH 2025.
 Conclusions: Our CRISPR-library screens validated the functional relevance of LVIS-based disruptions identified in CAR T-cell responders (Nobles et al., JCI, 2020). Our in vitro and in vivo pipelines successfully recapitulate what has been observed in patients through the enrichment of genes implicated with T-cell proliferation and survival. Preliminary assessment of several genes highlights promising candidates which can be modulated to enhance CAR T-cell efficacy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8034fdb34e3f095bd608b52e18a48b50ea75dfb" target='_blank'>
              Patient-informed CRISPR screening reveals gene targets to improve CAR T-cell efficacy
              </a>
            </td>
          <td>
            Beatriz Coutinho de Oliveira, Meraj Khan, Cassandra Gilmour, Saaurav Bari, Jessica Kerr, Yee Peng Phoon, J. Fraietta, Frederic Bushman, J. Melenhorst
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has a high incidence and mortality rate globally, with approximately 40% of patients harboring Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations. Patients with KRAS mutations exhibit poorer prognoses compared to those with KRAS wild-type tumors, often showing resistance to targeted therapies and chemotherapy. There is an acute need for innovative therapeutic strategies, particularly as KRASG12C inhibitors have been approved by the U.S. Food and Drug Administration (FDA) for second-line therapy in treating non-small cell lung cancer (NSCLC). Yet, their efficacy in CRC remains suboptimal due to the emergence of drug resistance. Current research on KRAS inhibitor resistance in CRC focuses mainly on single biological processes, leaving the complex interplay between cellular systems underexplored. This review synthesizes evidence across four key dimensions - metabolism, epigenetics, immunology, and proteostasis - to reveal multidimensional mechanisms of resistance. How these interconnected pathways work in synergy with signaling aberrations, genetic alterations, and translational modifications to reinforce resistance was investigated. The research also integrates recent advances in multi-targeted strategies, including combinations of KRAS inhibitors with metabolic or epigenetic modulators or immune checkpoint blockade. These findings provide potential strategies for overcoming resistance in KRAS-mutant CRC, addressing a critical gap in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de09a3f40e67f360dfd11b639fa71c10cdd13842" target='_blank'>
              Unravelling Resistance: Integrating Metabolism, Epigenetics, Immunology, and Proteostasis in Strategies against Kirsten Rat Sarcoma Viral Oncogene Homolog-mutant Colorectal Cancer.
              </a>
            </td>
          <td>
            Jingyi Li, Na Song, Rui Ma, Xiujuan Qu
          </td>
          <td>2026-01-01</td>
          <td>International journal of biological sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pathogenic germline variants in mismatch repair (MMR) genes, or Lynch syndrome (LS), increases patients’ risk of developing colorectal cancer (CRC). We previously identified shared, frameshift (fs)-neoantigens in MMR deficient (MMRd) cancers. We hypothesized that T cell surveillance occurs early in CRC development in LS but T cell activity is abrogated by immune regulatory programs during tumor progression. Blood and normal, precancerous (adenoma), and CRC tissues were analyzed from a cohort of 92 LS patients. We leveraged a custom neoantigen-discovery pipeline, functional antigen recognition assays, T cell receptor sequencing, and single cell and spatial transcriptomics. This identified for the first time T cells recognizing shared fs-neoantigens expressed in normal mucosa, adenomas, and tumors of LS patients. Immune editing was evidenced by i) distinct neoantigen repertoires in precancerous tissues compared to tumors and ii) recurrence of low affinity (but not high affinity) neoantigens in later lesions captured from the same patients. Transcriptomic and ex vivo functional analysis revealed that T cells in tumors, relative to precancerous tissues, had reduced functional capacity associated with an infiltration of immunoregulatory myeloid cells (tumor-associated neutrophils and TREM1+ macrophages) which can be therapeutically targeted in future clinical studies. Finally, these shared, precancerous fs-neoantigens can serve as vaccine targets to prevent MMRd cancer in LS.



 Supported by NIH UG3CA290517 and the Parker Institute for Cancer Immunotherapy



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bfb57527590807c825fa581d5e3eb9f0eacef25" target='_blank'>
              Identifying targets of precancerous neoantigen-specific T cell surveillance in patients with Lynch syndrome and immune suppression programs supporting colorectal cancer progression 3671
              </a>
            </td>
          <td>
            Matthew Brown, L. Velazquez, Joyce Serebrenik, Storey Harbison, S. Holtzman, Mesude Bicak, Alexandros Polydorides, Cansu Cimen Bozkus, Aimee L Lucas, R. Samstein, Nina Bhardwaj
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Ovarian cancer is the gynecologic malignancy that bears the highest mortality rate in the Western world. This is attributed to late diagnosis and limited therapeutic progress. Recent advances in molecular oncology have highlighted the pivotal role of epigenetic modifications—including DNA methylation, histone modifications, non-coding RNAs, chromatin remodeling, and RNA methylation—in ovarian cancer development, progression, and treatment resistance. DNA methylation patterns affect key tumor suppressors and oncogenes, while histone modifications alter chromatin accessibility, influencing gene expression. Chromatin remodeling complexes, particularly the SWI/SNF complex, are frequently mutated in specific ovarian cancer subtypes, which is central in shaping their biological behavior. Non-coding RNAs, including microRNAs and long non-coding RNAs, further regulate tumor cell behavior and the immunosuppressive tumor microenvironment. Epigenetic profiles vary among histological subtypes and hold promise for biomarker development, early detection, prognosis, and therapeutic monitoring. Liquid biopsy approaches leveraging circulating tumor DNA methylation show diagnostic potential superior to conventional markers. Moreover, targeting epigenetic regulators—such as DNMT and HDAC inhibitors, EZH2 antagonists, and RNA-modifying enzymes—offers novel avenues for treatment, particularly in reversing chemoresistance and sensitizing tumors to immunotherapy. While promising, these strategies require further validation through clinical research to translate into effective clinical interventions. This review aims to summarize the current literature and highlights potential applications of epigenetic manipulation in day-to-day practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e682da74ce847ab8714dc4327d4b867b8d126a5" target='_blank'>
              Epigenetics in Ovarian Cancer: A Review of Current Knowledge and Future Perspectives
              </a>
            </td>
          <td>
            Nikolaos Dedes, Michael Liontos, Dimitrios Haidopoulos, Flora Zagouri, K. Angelou, A. Svarna, A. Michas, Aikaterini Aravantinou Fatorou, Angeliki Andrikopoulou, Meletios A. Dimopoulos
          </td>
          <td>2025-11-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Introduction:Plasma cell leukemia (PCL) is a rare yet highly aggressive form of plasma cell dyscrasia, defined by high tumor burden, frequent extramedullary dissemination, and rapid clinical progression. Compared with multiple myeloma (MM), PCL exhibits greater genomic instability, immune escape, and a more hostile bone marrow microenvironment, resulting in poor therapeutic responses and short survival. While prior bulk sequencing and cytogenetic studies have revealed select features of PCL biology, a high-resolution, single-cell dissection of its malignant and immune landscape remains lacking. In particular, how PCL reshapes its tumor microenvironment (TME) to support immune evasion and systemic dissemination has not been fully elucidated.
 Aim: This study aims to characterize the transcriptional, genomic, and immunologic features of PCL at single-cell resolution. We sought to identify PCL-specific oncogenic drivers, immune evasion mechanisms, and intercellular communication circuits that differentiate PCL from MM and healthy bone marrow, with a particular focus on the ERBB signaling axis and T/NK cell dysfunction.
 Methods:Bone marrow samples were collected from three newly diagnosed primary PCL patients and one MM patient from our center, These specimens were subjected to 10x Genomics 3' single-cell sequencing. Additionally, three normal bone marrow and three MM bone marrow samples retrieved from the Gene Expression Omnibus (GEO) database were incorporated for integrative analysis. Comparative analyses were conducted to assess clonal architecture, transcriptional programs, and immune landscape. Key findings were validated using qPCR and publicly available clinical and transcriptomic datasets.
 Results:After quality control filtering, we obtained single-cell transcriptome data from 49,059 high-quality mononuclear cells. Based on canonical marker gene expression, we identified eight major cell populations: plasma cells, myeloid cells, T cells, NK cells, NKT cells , B cells , dendritic cells, and megakaryocytes. This included 11,485 PCs, 3,466 B cells, and 33,603 other immune cells.
 PCL plasma cells demonstrated significantly higher CNV burden than MM, with recurrent gains on chr3q, chr9q, chr15q, and chr18q. Amplification of oncogenes including BCL2, MCL1, PIK3CA, KRAS, and BRAF suggested enhanced apoptotic resistance, proliferative signaling, and immune evasion.
 Transcriptomic profiling revealed PCL-specific upregulation of the ERBB signaling axis, including ERBB4, EPHA6, and downstream effectors KRAS, SOS1/2, and MAPK1/8. ERBB pathway enrichment was confirmed by GSEA and pseudotime analysis revealed persistent upregulation along the malignant differentiation trajectory. High expression of ERBB pathway genes correlated with significantly worse overall survival in the CoMMpass cohort. qRT-PCR validation in patient samples confirmed significant overexpression of ERBB-related genes in PCL.
 Analysis of over 23,000 T/NK lineage cells identified 21 subtypes. In PCL, cytotoxic subsets such as CD8⁺ naive, CD56dim NK, and NKT cells were notably reduced or exhibited elevated exhaustion scores. NKT cells showed elevated IFN-response genes but downregulation of cytotoxic effectors, indicating a paradoxical “hyperactivated-yet-dysfunctional” state.
 CellChat analysis revealed extensive reprogramming of ligand-receptor interactions in PCL. Compared to ND and MM, PCL plasma cells engaged in enriched immunosuppressive and adhesive signaling via MIF, TGF-β, CD86, NOTCH, and APRIL axes. Enhanced communication with CD8⁺ T and NKT cells may promote chronic stimulation and immune exhaustion, supporting tumor immune escape and dissemination.
 Conclusion: This study presents the first high-resolution single-cell atlas of primary PCL, revealing that enhanced ERBB signaling, elevated CNV burden, and profound immune dysfunction collectively drive its aggressive clinical behavior. Malignant plasma cells in PCL not only acquire stem-like features and oncogenic pathway activation but also actively remodel the immune microenvironment via suppressive signaling, leading to T/NK cell exhaustion and impaired surveillance. These findings uncover actionable targets such as ERBB4/PI3K/MAPK and immune exhaustion pathways, offering a molecular framework for future precision therapy in PCL. Further functional validation and clinical translation of these discoveries are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c5eed7a02f279167bbebc35b4990c64fcc8584" target='_blank'>
              Single-cell transcriptomics reveals erbb-driven clonal evolution and immune evasion in plasma cell leukemia
              </a>
            </td>
          <td>
            Tingting Lei, Luyao Shi, Xinli Han, Chi Li, Zongyang Li, Xiaotong Zhang, Kairan Li, Saisai Sun, Lu Yang, Chong Wang, Zhilei Bian, Shujuan Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Brain tumors such as glioblastoma remain among the most lethal and immunologically resistant cancers, in large part due to epigenetic programs that sculpt the tumor–immune microenvironment. DNA methylation, histone modifications, and chromatin remodeling do not merely drive tumor-intrinsic changes; they also profoundly reprogram immune responses, shaping antigen presentation, cytokine signaling, and immune cell recruitment. At the center of this regulation are T cells, whose effector functions are suppressed through promoter hypermethylation of antigen-processing genes, silencing of interferon pathways, and the establishment of exhaustion-specific chromatin states. Mutations such as IDH1/2 and H3K27M further reinforce these epigenetic barriers, fostering immune-cold microenvironments that disable cytotoxic T-cell activity. Emerging evidence highlights both CNS-specific adaptations, including microglial and astrocytic epigenetic programs that reinforce immune privilege, and conserved features of T cell exhaustion that mirror those in peripheral cancers. This duality underscores the need for therapeutic strategies that dismantle CNS-specific barriers while leveraging shared exhaustion programs across tumor types. Epigenetic drugs, ranging from DNA methyltransferase and EZH2 inhibitors to BET degraders and CRISPR-based epigenome editors, are beginning to restore antigenicity, reverse T cell dysfunction, and sensitize tumors to checkpoint blockade. Yet these approaches carry the paradoxical risk of disrupting CNS immune tolerance, potentially triggering harmful neuroinflammation or autoimmunity. To our knowledge, this is among the first comprehensive reviews to integrate CNS-specific immune privilege mechanisms with peripheral exhaustion pathways, providing a unified perspective on how epigenetic regulation orchestrates immune dysfunction across central and peripheral contexts. By mapping the continuum between immune evasion and global immunosuppression, we propose a conceptual framework for tailoring epigenetic-immunotherapy combinations to achieve durable antitumor immunity in the CNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7202ed1c4ffc9d0ee982c0f1ad42403d845276" target='_blank'>
              Beyond the genome: epigenetic regulation of immune responses and T cells in brain tumors
              </a>
            </td>
          <td>
            Shuo Sun, Yu Han, Haiying Li, Chengyan Wang, Shu Zhou, Xiaowei Zhang, Shuhong Dai, Yao Peng, Zhuoqun Wang
          </td>
          <td>2025-11-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lineage plasticity is a cancer hallmark that drives disease progression and treatment resistance1,2. Plasticity is often mediated by epigenetic mechanisms that may be reversible; however, there are few examples of such reversibility. In castration-resistant prostate cancer (CRPC), plasticity mediates resistance to androgen receptor (AR) inhibitors and progression from adenocarcinoma to aggressive subtypes, including neuroendocrine prostate cancer (CRPC-NE)3–5. Here we show that plasticity-associated treatment resistance in CRPC can be reversed through the inhibition of NSD2, a histone methyltransferase6. NSD2 upregulation in CRPC-NE correlates with poor survival outcomes, and NSD2-mediated H3K36 dimethylation regulates enhancers of genes associated with neuroendocrine differentiation. In prostate tumour organoids established from genetically engineered mice7 that recapitulate the transdifferentiation to neuroendocrine states, and in human CRPC-NE organoids, CRISPR-mediated targeting of NSD2 reverts CRPC-NE to adenocarcinoma phenotypes. Moreover, a canonical AR program is upregulated and responses to the AR inhibitor enzalutamide are restored. Pharmacological inhibition of NSD2 with a first-in-class small molecule reverses plasticity and synergizes with enzalutamide to suppress growth and promote cell death in human patient-derived organoids of multiple CRPC subtypes in culture and in xenografts. Co-targeting of NSD2 and AR may represent a new therapeutic strategy for lethal forms of CRPC that are currently recalcitrant to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31a72e85b6954f413f71412e13d4f1a6449106e" target='_blank'>
              NSD2 targeting reverses plasticity and drug resistance in prostate cancer
              </a>
            </td>
          <td>
            Jia J Li, A. Vasciaveo, Dimitris Karagiannis, Zhen Sun, Kristjan H. Gretarsson, Xiao Chen, O. Ouerfelli, F. Socciarelli, Ziv Frankenstein, Hanyang Dong, Min Zou, W. Yuan, Guangli Yang, Gabriel M Aizenman, Tania Pannellini, Xinjing Xu, H. Beltran, Yu Chen, Kevin L Gardner, Brian D. Robinson, J. D. De Bono, O. Gozani, Cory Abate-Shen, Mark A. Rubin, M. Loda, Charles L. Sawyers, Andrea Califano, Chao Lu, Michael M. Shen
          </td>
          <td>2025-11-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="


 Introduction CAR-T cell therapy efficacy in B-cell lymphoma remains limited by treatment resistance. To address this fundamental challenge and elucidate underlying mechanisms, we undertook a comprehensive single-cell analysis of the CAR-T-tumor microenvironment interface. By leveraging five public scRNA-seq datasets spanning 1.48 million cells from CD19-targeted CAR-T patients, this study establishes an advanced computational framework to decipher the cellular ecosystem governing therapeutic resistance.
 Methods Methodologically, we developed three integrated innovations: (1) A biologically informed hierarchical graph convolutional network (GCN) modeling patient-specific tumor microenvironments as cell-cell correlation graphs for response prediction; (2) A novel Grad-CAM extension enabling quantitative interpretation of GCN outputs at single-cell resolution to establish gene-outcome relationships and resistance biomarkers; (3) GeneTransformer—a hybrid transformer architecture predicting transcriptional responses to genetic perturbations while resolving cross-platform heterogeneity via stochastic triple-masking during finetuning.
 Results Analysis revealed system-wide transcriptional dysregulation and extensive cell-subset-specific alterations distinguishing non-responders. Functional integration identified four core resistance modules: pro-survival signaling (BCL2/XIAP), metabolic rewiring (AMPK exhaustion), immune dysfunction (PD-1/TIGIT), and homeostatic collapse (Hippo/YAP-mediated stemness). Crucially, we exposed mechanistic crosstalk revealing actionable combinatorial targets—notably PI3K inhibition synergizing with anti-PD-1 blockade. Cross-population validation confirmed model robustness, with 84.65% (p<0.05) of reverse-engineered resistance genes exhibiting differential expression. The GCN achieved superior predictive performance (AUC=0.900).
 Conclusion This study establishes a transformative computational framework that deciphers the single-cell architecture of CAR-T resistance, providing a mechanism-guided blueprint for designing combinatorial therapeutic strategies. The GeneTransformer platform further enables optimization of individualized therapies through perturbation response prediction. Our findings illuminate fundamental resistance pathways while delivering clinically actionable targets for overcoming treatment failure in B-cell malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8a729b90dea4325ceebae0fe0354baffbb43f1" target='_blank'>
              Deep learning-based tumor microenvironment profiling predicts CAR-t efficacy and guides combinatorial immunotherapy in B-cell lymphoma
              </a>
            </td>
          <td>
            Shaojie Wu, Jiatong Wang, Jiaxiu Li
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Lineage infidelity, manifested as either lineage drift or lineage switch, has emerged as a mechanism of resistance to B-cell–targeted therapies. While lineage switch has been most extensively documented in KMT2A-rearranged (KMT2A-r) B-cell acute lymphoblastic leukemia (B-ALL), lineage switches have also been reported in non-KMT2A-r subtypes under the pressure of B-cell-targeted therapy. We anticipate that altered expression of B-cell antigens (including full lineage switches) will become more common as CD19 targeted T-cell engagers such as blinatumomab are broadly incorporated into frontline therapies. We and others have previously demonstrated the presence of small subclones with stem cell/myeloid transcriptional signature in B-ALL at initial diagnosis using single cell multi-omics, suggesting a pre-existing reservoir of cells that are not committed to a B-cell fate.
 To explore the presence and functional significance of lineage infidelity in B-ALL, we performed comprehensive multiomic profiling—including single-cell RNA sequencing (scRNA-seq), single-cell ATAC-seq (scATAC-seq), single-cell DNA-seq (scDNA-seq), CITE-seq, and high-dimensional spectral flow cytometry on a cohort of standard- and high-risk paired pediatric B-ALL samples collected at diagnosis and at the end of induction therapy. Our integrated analysis uncovered the following key insights:
 Evidence of CD19 surface protein negative stem cell/myeloid leukemogenic subclones: Prior studies identified stem cell/myeloid subclones using single cell transcriptomic methods, which are inferior in assessing surface protein expression of several antigens, including CD19, due to high drop-out. In this study we validated the existence of this stem cell/myeloid-like population by detecting known leukemia-defining mutations such as KRAS, PAX5, and CREBBP within this population, confirming their clonal relationship to the bulk leukemia, and we confirm their surface protein marker profile using CITE-Seq and spectral flowcytometry.
 The stem/myeloid-like population showed reduced or absent expression of canonical B-cell markers such as CD19, CD22, and CD79a, both at the transcript and protein levels. Unexpectedly, this phenotype was not restricted to KMT2A-r ALL but was also observed in other subtypes such as Ph-like, Ph+, and iAMP21. To assess the functional potential of these lineage-infidel subclones, we sorted CD19⁻ blasts and transplanted them into immunodeficient NSG mice. These cells were successfully engrafted and gave rise to leukemia. This is the first study to provide evidence of lineage plasticity by integrating surface protein marker detection with transcriptional profiling and functional assays documenting the leukemogenicity of these clones.
 Stem cell/myeloid-like blasts enriched at minimal residual disease (MRD): Analysis of paired initial diagnosis and MRD samples documented enrichment of stem cell / myeloid clones at MRD. Again, this was not confined to KMT2A-r samples but also seen in several patients with other high-risk genotypes. Integration with drug profiling suggests increase resistance of the myeloid subclones to standard ALL chemotherapy agents. These findings suggests that in a subset of patients ALL Induction chemotherapy selects for a residual CD19-negative blast population that may jeopardize the efficacy of subsequent B-cell directed immunotherapy such as blinatumomab.
 Collectively, our findings have two key implications: (1) our data challenges current risk stratifications that rely on B-lineage flowcytometry to detect residual blasts at the end of Induction. Patients with a shift in lineage phenotype may potentially be misclassified when using standard flow cytometry based on lymphoid markers only. Lineage infidelity may also explain the occasionally observed discordant MRD assessments when using both flowcytometry and molecular methods. (2) Our findings have implications for treatment. Flowcytometry for stem cell / myeloid markers at MRD may identify a subgroup of patients that benefits less from blinatumomab. In addition, we identified surface protein markers associated with the CD19-negative population that may serve as alternative targets for novel immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f768f774edfacf769eb24ded149337c40bdec0" target='_blank'>
              Chemotherapy selects for myeloid subclones in multiple subtypes of B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Fatemeh Alikarami, Rushabh Mehta, Changya Chen, L. Brodersen, D. Teachey, Kai Tan, K. Bernt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Background Mosaic loss of the Y chromosome (LOY) is a somatic, age‐related phenomenon associated with an increased risk of developing Alzheimer's disease (AD) by our group and others. While the underlying biological mechanism driving this risk remains unknown, genome wide association studies (GWAS) of LOY outcomes identified ∼150 loci involved in cell cycle regulation, mitotic pathways, and DNA damage response. These variants also predict X chromosome loss in women, suggesting that LOY is a marker of genomic instability in males that precedes the AD clinical phenotype. Method To understand the mechanism by which LOY may contribute to AD risk, we sought to identify both affected and neurologically normal individuals with LOY from genotyping data using the MADloy program. We reprogrammed peripheral blood mononuclear cells (PBMCs) from three of these individuals and used the CytoTune iPS 2.0 Sendai Reprogramming Kit to create induced pluripotent stem cells (iPSCs) lines. Result 1,068 males with genotyping data were screened for loss of the Y chromosome; 41 (3.83%) had detectable LOY in the blood. As expected, LOY was positively correlated with the age of the individual at the time of sample collection. Three LOY samples (2 cognitively normal individuals and 1 individual with Alzheimer's disease) were chosen for iPSC reprogramming. Samples were independently shown to have LOY through either CNV TaqMan assays or whole genome sequencing data analyzed via the Control‐FREEC software. iPSC reprogramming was performed using Sendai virus. Clones were identified from each sample that either carried or were missing the Y chromosome, thereby generating isogenic pairs of cell lines. Conclusion We have generated paired iPSC cell lines derived from AD and cognitively normal individuals that allow for explicit examination of the impact of LOY on AD‐relevant cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fdfe3fdcfff07d9fa3235623a05b3a308822589" target='_blank'>
              Generation of Isogenic Induced Pluripotent Stem Cell Lines Derived From Individuals Mosaic for Loss of the Y Chromosome
              </a>
            </td>
          <td>
            H. N. Cukier, Yeunjoo E. Song, Lauren E. Coombs, Eugene Tang, Tianjie Gu, Brooke A. DeRosa, Derek M. Dykxhoorn, Jonathan L. Haines, M. Pericak-Vance, William S Bush, Anthony J Griswold
          </td>
          <td>2025-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Background
 Cytosine arabinoside (ARA-C) is a major active drug against AML cells, and is widely used also in this era of targeted treatments. Resistance to ARA-C remains a significant obstacle to achieving CR and long-term remissions. We previously reported an association between CNN-LOH in chromosome 1p that was more common in samples from older AML patients compared to younger patients (Koren-Michowitz M et al. Leuk Res. 2012 Apr;36(4):467-73). Since AML in older patients often exhibit primary therapy resistance, we hypothesized that genes in the commonly deleted chromosome 1p region could be associated with chemotherapy resistance.
 Methods
 We performed a CRISPR-Cas9 pooled loss-of-function screen using a library of sgRNAs targeting all the coding genes located on the commonly deleted chromosome 1p region to identify candidate genes whose disruption confers resistance to ARA-C in AML cells. The CRISPR-Cas9 pooled screen was designed using 1,000 non-targeting sgRNAs and 10 sgRNAs per target gene, along with three positive controls (DCK, p53, and SLC29A1). OCI-AML2-Cas9 stable cells were transduced with the pooled sgRNA library and treated with 1 μM ARA-C for 3 days. MAGeCK computational analysis was performed to identify genes whose knockout altered sensitivity to ARA-C. Genes were considered positively selected if p ≤ 0.008 and FDR ≤ 0.4 and negatively selected if p ≤ 0.05 and FDR ≤ 0.002.
 Results
 Analysis of the pooled screen revealed 13 negatively selected and 8 positively selected candidate genes. Gene ontology (GO) functional enrichment analysis revealed statistically significant enrichment (p < 0.05) among negatively selected targets for RNA binding and DNA-binding transcription factor activity. No significant enrichment was observed among the positively selected genes. Expression of the negatively selected genes in AML samples was retrieved from the TARGET initiative (phs000465). SRSF10 showed the highest differential expression, with a 52-fold increase (p = 1.07 × 10⁻73) in AML samples compared to normal CD34+ cells.
 SRSF10 CRISPR-Cas9 single knockout (KO) clones of OCI-AML2 and MOLM13 cell lines exhibited increased sensitivity to ARA-C compared to non-targeting sgRNA controls (IC₅₀ 97 nM and 138 nM for OCI-AML2 cells, 53 nM and 82 nM MOLM13 cells, respectively). This was accompanied with an increase in the proportion of early apoptotic cells and accumulation of cells in the G2/M checkpoint in the KO cells compared to control cells. Co-culture of AML cell lines with HS5 bone marrow stromal cells protected AML cells against ARA-C cytotoxicity, as was previously shown with other anti-AML drugs. Interestingly, SRSF10 KO cells were more sensitive to ARA-C also in this setting.
 Conclusions
 SRSF10 could be a target for enhancing cytotoxicity of ARA-C in AML. Studies in primary AML cells are being conducted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb9e519fc9544e99958424d8bf194b60501e3f6" target='_blank'>
              Knockdown of SRSF10 enhances the sensitivity of AML cells to cytosine arabinoside treatment
              </a>
            </td>
          <td>
            Maya Koren-Michowitz, Yoav Haviv, Zehavit Rogov Levi, Roni Golan-Lavi, A. Apel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer reversion therapy represents a paradigm shift in oncology, focusing on reprogramming malignant cells to a non-malignant state rather than destroying them. This narrative review synthesizes current evidence, emerging technologies, and future directions in this promising field. Cancer reversion is founded on key biological observations: somatic cell reprogramming, spontaneous cancer regression, and microenvironmental influences on malignant behavior. Current approaches include epigenetic reprogramming using HDAC inhibitors and DNA methyltransferase inhibitors; microenvironmental modulation through extracellular matrix manipulation and vascular normalization; differentiation therapy exemplified by all-trans retinoic acid in acute promyelocytic leukemia; and targeting oncogene addiction as demonstrated in BCR-ABL-driven leukemias. Emerging technologies accelerating progress include single-cell analyses that reveal cancer heterogeneity and cellular state transitions; CRISPR-based approaches enabling precise genetic and epigenetic manipulation; patient-derived organoids that model tumor complexity; and artificial intelligence applications that identify novel reversion-inducing agents. Critical evaluation reveals that many reported “reversion” phenomena represent stimulus-dependent plasticity or transient growth arrest rather than stable phenotypic normalization. True cancer reversion requires durable, heritable phenotypic changes that persist after treatment withdrawal, with evidence of epigenetic consolidation and functional restoration. Despite promising advances, significant challenges remain: cancer cell plasticity facilitating therapeutic escape, difficulties in establishing stable reversion states, delivery challenges for solid tumors, and the need for combination approaches to address tumor heterogeneity. Future directions include integrated multi-omics analyses to comprehensively map cellular state transitions, studies of natural regression phenomena to identify reversion mechanisms, advanced nanodelivery systems for targeted therapy, and synthetic biology approaches creating intelligent therapeutic systems. By redirecting rather than destroying cancer cells, reversion therapy offers the potential for reduced toxicity and resistance, potentially transforming cancer from a deadly disease to a manageable condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ef16378f672c1c6324bdae132a2624ad963a92" target='_blank'>
              Cancer Reversion Therapy: Prospects, Progress and Future Directions
              </a>
            </td>
          <td>
            Emmanuel O. Oisakede, D. Olawade, Oluwakemi Jumoke Bello, Claret Chinenyenwa Analikwu, Eghosasere Egbon, Oluwaseun Fapohunda, S. Boussios
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="
 Cancer is a major threat to human health, with genomic instability being a primary cause of cancer, and replication stress serving as a crucial factor in genomic instability. Modern anticancer drugs often use the induction of high levels of replication stress in cancer cells to kill them. Therefore, understanding how cancer cells respond to replication stress is important. Studies have shown that PAX3 and PARP1 are highly expressed in glioma cells. We have preliminary discovered that PAX3, through PARP1, localizes to replication forks, reducing replication stress induced by anticancer drugs and contributing to drug resistance. Experiments have also revealed that PARP inhibitors (PARPi) prevent PAX3 from localizing to replication forks during replication, and PARPi are considered to have therapeutic potential. These results indicate that PARP1 plays an important role in PAX3’s ability to stabilize the genome under replication stress. Therefore, we hypothesize that PARP1 may recruit PAX3 to replication forks in glioma cells, reducing replication stress and promoting cancer cell survival. To confirm this hypothesis, we are collaborating with Taichung Veterans General Hospital (TVGH) to investigate the roles of PAX3 and PARP1 in regulating genomic stability and drug resistance in glioma cells. This study will analyze the relationship between PAX3 and PARP1 and the effects of PARP inhibitors on glioma cells. We expect that inhibiting PARP1 expression will affect PAX3’s ability to localize to replication forks, thereby impacting its function in suppressing replication stress, leading to increased genomic instability markers, and ultimately resulting in cancer cell death. The findings of this research will contribute to a deeper understanding of how glioma cells cope with replication stress to develop drug resistance and provide a theoretical basis for developing new therapeutic strategies based on PARP inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1461be0df678bc8ada1f5943d4ffca888b6f273a" target='_blank'>
              TMIC-11. Study of PAX3 and PARP1 for genomic stability and drug resistance in glioblastoma
              </a>
            </td>
          <td>
            Chiung-Chyi Shen, Wen-Ming Yang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Myeloid/lymphoid neoplasms with eosinophilia and tyrosine kinase gene fusions (MLN-TK) are rare hematologic malignancies characterized by constitutive activation of tyrosine kinase (TK) signaling. Among them, platelet-derived growth factor receptor β (PDGFRB) fusion-positive MLN (MLN-PDGFRB) typically responds well to tyrosine kinase inhibitors (TKIs) in the chronic phase (CP); however, treatment options for the blast phase (BP) remain limited, and overcoming TKI resistance in BP requires identification of other key genetic drivers. To identify novel therapeutic vulnerabilities in BP MLN-PDGFRB, we performed genome-wide CRISPR-Cas9 knockout screening using a murine Ba/F3 cell model expressing EBF1-PDGFRB.
 Ba/F3 cells stably expressing EBF1-PDGFRB and Cas9 were generated and confirmed to proliferate in an IL-3 independent manner. Genome-wide CRISPR-Cas9 screening was performed using the GeCKO library, with cells split into IL-3(+) and IL-3(−) conditions for 14 days. sgRNAs depleted under IL-3(−) but not IL-3(+) conditions were prioritized as candidate dependencies specific to EBF1-PDGFRB-driven signaling. These candidate genes were further validated via targeted knockout and functional assays. Pathway enrichment analysis was also performed using MAGeCKFlute and MAGeCK pipelines.
 Among the top-ranked hits under IL-3(−) condition, RIC8 guanine nucleotide exchange factor A (Ric8a) and phosducin-like (Pdcl), both of which were positive regulators of G-protein-coupled receptor (GPCR) signaling, were identified as essential genes for the survival of Ba/F3-EBF1-PDGFRB cells. Targeted knockout of either Ric8a or Pdcl significantly impaired cell proliferation only in IL-3(−) conditions. Moreover, knockout of either gene individually led to a synergistic enhancement of TKI-mediated growth inhibition. Notably, pathway enrichment analysis suggested partial reliance on the mechanistic target of rapamycin (mTOR) pathway; however, pharmacologic inhibitors targeting mTOR failed to show selective efficacy. These results suggest that multiple signaling pathways, including GPCR and mTOR, may cooperatively sustain PDGFRB-mediated oncogenesis in a complementary manner.
 Importantly, neither RIC8A nor PDCL have been previously implicated in MLN-PDGFRB pathobiology, and their identification was unexpected given the upstream location of PDGFRB within RTK signaling. Functional crosstalk between RTKs and GPCRs may underlie this dependency. These results are consistent with previous reports in EGFR-mutated lung cancer, where RIC8A knockout enhances TKI sensitivity through modulation of the Hippo-YAP pathway. While Ric8a or Pdcl knockout in Ba/F3 cells expressing other RTK fusion oncogenes such as FGFR1 or PDGFRA did not result in comparable effects on cell proliferation or TKI sensitivity, similar growth-inhibitory effects were observed in Ba/F3 cells expressing another PDGFRB fusion, AGGF1-PDGFRB. These findings suggest that the observed dependency may be shared across multiple PDGFRB fusion variants, but not across other types of RTK-driven MLN-TK.
 This study demonstrates the feasibility and utility of genome-wide CRISPR-Cas9 screening in Ba/F3-MLN model. We identified RIC8A and PDCL as novel therapeutic targets in MLN-PDGFRB, potentially mediating crosstalk between RTK and GPCR signaling. Further in vivo validation and extension to other PDGFRB fusion partners are warranted to assess their translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b34afb6e6a3dc5be8b9ebc5f3ee4f6f127ebbf" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identified GPCR regulators RIC8A and PDCL as therapeutic targets in MLN with PDGFRB fusions
              </a>
            </td>
          <td>
            Michiaki Sato, Toshiya Hino, T. Ishida, Hiroki Hayashida, Yosuke Masamoto, Mineo Kurokawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains uniformly fatal because the standard temozolomide (TMZ) + radiotherapy regimen is quickly thwarted by epigenetic adaptations that reactivate proliferation and blunt DNA-damage responses; mounting evidence implicates simultaneous up-regulation of lysine-specific demethylase 1 (KDM1A/LSD1) and histone deacetylase 2 (HDAC2) as a central driver of this resistance, yet no brain-penetrant agent has been available to co-target both enzymes. Here we show that MPT0G521, a dual KDM1A/HDAC inhibitor able to cross the blood–brain barrier, reverses this adaptive program. Integrated analyses of TMZ- and radiation-resistant GBM lines, public bulk datasets and 66 000 single cells revealed concerted over-expression of KDM1A and HDAC2, tight co-regulation with 50 core cell-cycle genes, and enrichment of LSD1-high NPC/OPC-like populations at invasive margins; high expression of either gene predicted significantly worse overall survival in CGGA cohorts (log-rank p < 0.01). In both parental and TMZ-resistant A172 and patient-derived Pt#3 cells, MPT0G521 produced dose- and time-dependent growth inhibition, suppressed clonogenicity, induced G2/M arrest, and triggered caspase-3/PARP-mediated apoptosis. Mechanistically, treatment raised H3K4me1/2 and H3K9/K14ac without altering core histone levels, confirming effective blockade of KDM1A demethylase and HDAC deacetylase activities. These findings identify KDM1A/HDAC2 over-expression as a tractable epigenetic vulnerability in therapy-refractory GBM and position MPT0G521 as a promising lead compound for overcoming TMZ resistance and improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a64cf703dedf5e47c6085e0296a117bfd8d31b01" target='_blank'>
              DDDR-13. A Brain-Penetrant Epigenetic Strategy to Overcome Therapy-Refractory Glioblastoma
              </a>
            </td>
          <td>
            W. Lo, An-Chih Wu, Pin-Yuan Chen, J. Chuang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Introduction: With the advent of modern multi-drug chemotherapy regimens, several aggressive hematologic malignancies now have excellent outcomes. However, long-term follow-up studies show increased incidences of secondary malignancies, lifestyle diseases, and other features of accelerated aging.
 Aging is a complex, pervasive, and progressive phenomenon typified by characteristic molecular and cellular changes. Over the past two decades, a new unifying theory of aging has been shown to best explain the aging phenotype - the “Information Theory of Aging” proposes that aging in eukaryotes is due to loss of transcriptional networks and epigenetic information over time. Human DNA methylation studies demonstrate this accrual of epigenetic dysregulation over time which strongly corroborates with parameters of aging, while genomic sanctity is broadly maintained by high-fidelity, endogenous repair mechanisms.
 This raises the concern that standard chemotherapeutic drugs, despite their life-saving action, may affect the epigenetic landscape via DNA methylation modification, resulting in an accelerated aging phenotype in keeping with the prevailing information theory of aging.
 Our objective was to study these methylation changes in survivors of hematologic malignancies.
 Methods: In this observational cohort study, we conducted a comprehensive analysis of blood methylation profiles in survivors of Hodgkin's Lymphoma (HL), Acute Promyelocytic Leukemia (APL), and Acute Myeloid Leukemia (AML); all 30-50 years of age, disease-free for >5 years from treatment completion (n=10 in each group). Age-matched normal individuals were tested as controls. Whole-genome methylation patterns were assessed using Illumina's Infinium MethylationEPIC v2.0 BeadChips and represent current methylation profiles of nucleated myeloid/lymphoid cells and their progenitors. A comparison of methylation profiles against controls hypothetically represents epigeno-toxicity of three different treatment regimens, differing in drugs, dosages, and durations.
 Results: Methylation profiles in the cancer survivor cohorts were found to be different from age-matched controls. We demonstrated that levels of aberrant DNA methylation were higher among cancer survivors; observing an average of 2.34%, 6.93%, and 10.68% higher methylation in the lowest quantile of the 935K CpG sites on the BeadChip array in HL, APL, and AML survivors, respectively. To understand if these methylation changes were associated with epigenetic age acceleration (EAA), we used the PCGrimAge and DunedinPACE epigenetic clock tools. Indeed, when applying the DunedinPACE tool, EAA corresponded to increased methylation noted above. When normalized against healthy controls, HL, APL, and AML survivors showed an age increase of 13.8, 23.8 and 28.8 months on PCGrimAge, respectively.
 Of the differentially methylated CpGs (DMCs) identified, the HL, APL, and AML cohorts had 211, 2085, and 4248 DMCs of significance (p<0.0001), respectively. Principal Component Analyses of the top DMCs showed a clear separation between these cohorts. A gene ontology pathway analysis of the most involved DMCs in the AML survivor cohort compared to normal controls showed enrichment in embryonic/tissue morphogenesis, neuronal migration and stem cell proliferation - processes potentially pointing to cellular de-differentiation. All these findings taken together demonstrate a signal towards significant, dose-dependent methylation change in these individuals, in keeping with accelerated epigenetic aging.
 Conclusions: Our findings suggest that different hematological malignancies and their chemotherapies are associated with varying long-term epigenetic changes and EAA in normal hematopoietic progenitors. The implications underscore an unmet need for further research and potential changes in treatment algorithms on this basis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cbdfe7f7a2e9334033268408aae575e18bf5a0d" target='_blank'>
              Impact of cancer and therapy on the epigenetic landscapes of normal cells: Describing a novel biomarker in survivorship
              </a>
            </td>
          <td>
            S. Selvarajan, U. Kulkarni, R. Shaji, A. Abraham, A. Korula, S. Lionel, Ganesh Jammula, Biju George, Saurabh Agarwal, V. Mathews
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Deletion of the long arm of chromosome 9, del(9q), is a recurrent cytogenetic abnormality in AML and encompasses the HNRNPK locus at 9q21.32. hnRNP K is a multifunctional RNA- and DNA-binding protein involved in chromatin remodeling, transcriptional regulation, and RNA metabolism. Monoallelic deletions or loss-of-function mutations in HNRNPK, collectively observed in ~5–6% of AML cases, directly contribute to leukemogenesis.
 We previously showed that Hnrnpk haploinsufficiency disrupts p53 and myeloid lineage-specifying pathways (e.g., C/EBPα and PU.1), impairing differentiation and expanding myeloid progenitors—recapitulating del(9q) AML features. While elucidating these pathways has aided our understanding of AML biology, they remain largely undruggable. Therefore, identifying alternative, targetable vulnerabilities in HNRNPK-deficient AML is of significant clinical interest. To this end, our current studies reveal that HNRNPK-deficient cells exhibit marked sensitivity to translational inhibition, indicating a synthetic lethal interaction driven by impaired protein synthesis.
 To delineate the molecular basis of this vulnerability, we used RIP-seq, proteomics (RPPA, IP-MS), and translational profiling. Here, we identified that HNRNPK deficiency caused dysregulated mTOR signaling, aberrant ribosomal protein S6 phosphorylation, and altered expression of key translation initiation/elongation factors. Critically, rRNA processing was impaired, disrupting 47S precursor maturation and leading to defective 40S and 60S subunit assembly and reduced active polysomes—suggesting that del(9q) AML may represent a novel ribosomopathy.
 Importantly, complete HNRNPK loss is embryonically lethal, and large-scale CRISPR datasets identify it as essential, underscoring its dosage sensitivity and potential for synthetic lethal therapeutic strategies. To uncover synthetic lethal interactions, we performed a genome-wide CRISPR-Cas9 dropout screen in HNRNPK-deficient cells, identifying dependencies in ribosome biogenesis, translation initiation, and metabolite synthesis as potential vulnerabilities. We are currently using primary cells harboring HNRNPK deletions and genetically engineered models with reduced hnRNP K expression to identify therapies that induce cell death in an hnRNP K-dependent manner.
 Collectively, our findings establish hnRNP K as a dosage-sensitive tumor suppressor and suggest del(9q) AML as a novel ribosomopathy. Moreover, the synthetic vulnerabilities identified in our study provide a strong rationale for developing targeted therapies that exploit HNRNPK haploinsufficiency and lay the foundation for the clinical development of agents that induce synthetic lethality in this subset of patients with AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a1f4b3ab63f8b9803758eab2816344752deed3f" target='_blank'>
              Hnrnpk haploinsufficiency in del(9q) Acute Myeloid Leukemia reveals targetable vulnerabilities
              </a>
            </td>
          <td>
            S. Post, Oscar Benitez, Ashley G Rodriguez, Marisa Aitken, P. Malaney, Miguel Gallardo, Hussein A. Abbas
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 INTRODUCTION: Anemia is the main clinical manifestation of low-risk myelodysplastic syndromes (MDS), reflecting ineffective erythropoiesis. Although therapies such as erythropoietin and luspatercept are available, their effects are often transient, highlighting the need for new therapeutic agents. In the past years, long non-coding RNAs (lncRNAs) have emerged as critical regulators of gene expression in hematopoiesis and cancer, yet their role in ineffective erythropoiesis is poorly defined. The aim of this study was to identify lncRNAs contributing to erythroid dysregulation using del(5q) MDS as a model.



 Single-cell RNA-seq (10x Genomics) was performed on CD34+ bone marrow cells from 4 MDS-del(5q) patients and 3 age-matched healthy donors. Preprocessing was done using CellRanger and Kallisto (hg38, GENCODE v37), followed by quality filtering via emptyDrops and scDblFinder. Cell types were annotated based on canonical marker expression.
 To identify erythroid-relevant lncRNAs, ELATUS pipeline using recommended thresholds (ratio >3, CR >150, SI >0.1) was applied across the erythroid stages – megakaryocyte-erythroid progenitors (MEP), early (EE) and late erythroid (LE).
 Differential expression was assessed on pseudobulk data via DESeq2 (log2FC >1, padj <0.05) Pseudotime trajectories were inferred with Monocle3, and lncRNA expression dynamics were modeled applying generalized additive models (GAM), using binarized cell expression (>0 counts) as input.
 The significance of dynamic differences between conditions was tested using ANOVA by comparing full (condition-interacting) vs null (no condition-interacting) GAM. Candidate lncRNAs were defined as those meeting all three criteria: ELATUS selection, differential expression, and altered dynamics. Gene ranking was performed using a Z-score combining scaled log2FC and -log10(padj-GAM) and expression dynamics of axis-related genes were examined.



 Among the 2,100 biologically relevant lncRNAs identified, 46 were differentially expressed between MDS and controls. Of these, 43 also displayed significantly altered expression dynamics, fulfilling all selection criteria. Literature review revealed that 10 of the 43 lncRNAs have cis-targets implicated in erythropoiesis. Among them, ENSG00000288934, an antisense (AS) transcript of PPP1R15B emerged as top candidate based on the previous prioritization score and dynamic profile.
 In healthy controls, PPP1R15B-AS showed a sharp, stage-specific pulse of expression during erythroid maturation, while PPP1R15B was concomitantly activated in a mirror-like pattern. In MDS, this lncRNA was downregulated, and PPP1R15B expression became flattened and temporally uncoupled from erythroid progression.
 In parallel, axis-related genes such as ATF4 showed a progressive increase during pseudotime, while pro-apoptotic effector CHOP (DDIT3) exhibited a marked late-stage peak coinciding with terminal maturation. This contrasts with the more restricted and temporally coordinated expression observed in healthy samples.



 In our dataset, downregulation of PPP1R15B-AS coincided with a temporal shift and loss of coordination in PPP1R15B expression, suggesting a role as repressor in fine-tuning gene activation during specific erythroid stages. While PPP1R15B is known to resolve integrated stress response by promoting eIF2a dephosphorylation and repressing effectors such as ATF4 and CHOP, we observed that both transcription factors remained overexpressed despite having increased levels of PPP1R15B. These findings point toward a dysregulation of the PPP1R15B-eiF2a axis in MDS, in which PPP1R15B-AS may contribute to fine-timing of stress deactivation.
 Altogether, these observations support a model in which ineffective erythropoiesis may not only result from aberrant expressions, but also from disruption of transcriptional dynamics mediated by lncRNAs. This altered expression of PPP1R15B-AS, despite its genomic location outside 5q deletion, may reflect broader downstream remodeling. Ongoing in vivo studies are developing to validate these findings.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b69695216aa01aa2d8a0ecf9cc6bc511d5b9f0" target='_blank'>
              Single-cell transcriptomics reveal long non-coding RNA dysregulation in PPP1R15B-EIF2A axis impairing erythroid maturation in myelodysplastic syndromes
              </a>
            </td>
          <td>
            F. Beas, Enrique Goni, Aintzane Díaz-Mazkiaran, T. Ezponda, A. Alfonso-Piérola, A. Jerez, Felipe Prósper, David Valcárcel, Mikel Hernáez Arrazola
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Drug-tolerant persister (DTP) cells drive therapeutic resistance in EGFR-mutant lung adenocarcinoma. Using single-cell RNA sequencing, we identified a clinically significant RGS5+MYL9+ cancer-associated fibroblast (CAF) population that was associated with EGFR tyrosine kinase inhibitor (TKI) resistance and poor prognosis. These CAFs were recruited to DTP niches via CCL11 signaling, and they formed tunneling nanotubes through Miro1/RhoA activation induced by TKI-generated mitochondrial reactive oxygen species (ROS). Remarkably, RGS5+MYL9+ CAFs functioned as "metabolic sinks" by accepting tumor-derived damaged mitochondria, thereby promoting DTP survival. Treatment with fasudil, a Rho kinase inhibitor, effectively blocked mitochondrial transfer and restored sensitivity to the EGFR-TKI osimertinib in vivo. Together, this work reveals targetable stromal-tumor crosstalk that sustains DTP populations, proposing a combination therapy for overcoming EGFR-TKI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b95827d896bc8971d8b880f2e978477fdef6355" target='_blank'>
              Transfer of Damaged Mitochondria from Cancer Cells to Cancer-Associated Fibroblasts Promotes Tyrosine Kinase Inhibitor Tolerance in EGFR-Mutant Lung Cancer.
              </a>
            </td>
          <td>
            Tongyan Liu, Shiyi Huang, Zhitong Li, Jianyu Li, Qinhong Sun, Qinglin Wang, Ziyang Shen, Zhijun Xia, Fanchen Meng, Youtao Xu, Wenjia Xia, Jing Tan, Rong Yin
          </td>
          <td>2025-12-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background: Neuroblastoma (NB) is the most prevalent extracranial solid tumor in children. Therefore, urgent exploration of novel therapeutic targets and more effective approaches is imperative to enhance the prognosis of these children. Methods: NB datasets were obtained from OpenGWAS—ieu-a-816 (1,627 cases; 3,254 controls; European ancestry) and prot-a-2003 (N=3,301; European ancestry)—and were combined by inverse-variance meta-analysis. Adrenal expression quantitative trait loci (eQTL) data were sourced from GTEx v8 (n=233). Bidirectional Mendelian randomization (MR) and summary-based MR analyses were conducted to infer the causal relationship between adrenal-related genes and NB, with validation by Steiger directionality testing and phenotype scanning. Blood cis-eQTL datasets were obtained from eQTLGen (n=31,684 across 37 cohorts), and immune-cell data were retrieved from OpenGWAS (731 traits; IDs ebi-a-GCST90001391–GCST90002121). Using immune-cell traits as intermediate variables, bidirectional MR assessed the causal relationship between blood-related genes and NB, and colocalization analysis was performed for blood-related genes and immune cells. Results: Using two-sample bidirectional MR, we identified eight adrenal genes associated with NB risk. Upregulated expression of CPNE1 (OR 0.94, 95% CI [0.93–0.95]; P=1.69×10-31), ZNF559 (OR 0.93, 95% CI [0.92–0.95]; P=5.96×10-11), TTC18 (OR 0.86, 95% CI [0.85–0.87]; P=1.04×10-110), DNAJC9 (OR 0.85, 95% CI [0.84–0.86]; P=1.62×10-109), and EP400NL (OR 0.84, 95% CI [0.83–0.84]; P<1×10-300) was associated with lower NB risk, whereas upregulation of BTNL2 (OR 1.07, 95% CI [1.07–1.07]; P<1×10-300), FAM182B (OR 1.07, 95% CI [1.06–1.08]; P=3.94×10-23), and NBPF3 (OR 1.13, 95% CI [1.12–1.13]; P=6.49×10-5) was associated with higher risk. In the blood, 43 genes influence NB by affecting the expression of 54 immune cell phenotypes. Among these genes and immune cells, eight exhibited colocalization effects with seven immune cells, indicating their potential as therapeutic targets. Conclusion: This study revealed that certain genes in the adrenal glands and blood affect the occurrence of NB, with immune cells playing a crucial role in the process influenced by blood genes. MR and colocalization prioritize candidate genes/putative therapeutic targets for NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9cfa8536a863b787c9dc268dbef80c2fd91c" target='_blank'>
              Potential Therapeutic Targets for Neuroblastoma Screened through Mendelian Randomization Analysis
              </a>
            </td>
          <td>
            Zhenge Yang, Yunlong Zhang, Shan Wang
          </td>
          <td>2025-11-01</td>
          <td>Archives of Iranian Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Most genetic variants that confer risk of complex immune-mediated diseases (IMD) affect gene regulation in specific cell types. Their target genes and focus cell types are often unknown, partially because some effects are hidden in untested cell states. B cells are drivers of IMDs such as lupus and multiple sclerosis. Yet, B cell activation states are underrepresented in functional genomics studies. In this study, we obtained B cells from a total of 23 healthy female donors. We stimulated B cells in vitro by targeting key pathways in six activation conditions such as the CD40, B cell receptor, TLR7, TLR9, and a cocktail that promotes differentiation into double negative 2 (DN2) B cells. We profiled B cell activation at different time points using RNA-seq, single-cell RNA-seq coupled with surface protein markers (CITE-seq), and ATAC-seq. We provide an in-depth characterization of how IMD-associated genes respond to stimuli and group into modules with distinct functions. Using single-cell data, we describe pathway-dependent B cell fates. Chromatin data reveal the transcription factors involved in pathway-dependent B cell activation. These open chromatin regions capture a significant proportion of the heritability of IMDs and elucidate IMD-associated variants with previously unknow function. These data are shared via an interactive browser that can be used to query the dynamics of gene regulation and B cell differentiation during activation by different stimuli.



 Supported by Lupus Research Alliance (LRA); Arthritis National Research Foundation (ANRF); Gilead; Join Biology Consortium (JBC)



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bfee869441bc2a7a90854f0e5e1d453ccb1fe55" target='_blank'>
              A multiomics resource of B cell activation helps decipher disease pathogenesis 4355
              </a>
            </td>
          <td>
            Vitor R.C. Aguiar, Marcella E. Franco, Daniela Fernandez-Salinas, Marcos Chiñas, Qian Xiao, Nicolaj S. Hackert, Mariasilvia Colantuoni, Nada Abdel Aziz, M. Todd, Brian Wauford, Alexandra Wactor, Vaishali Prahalad, Raquel Laza Briviesca, Roxane Darbousset, Qiang Wang, Emily Banasiak, Scott A Jenks, K. Cashman, Esther C. Zumaquero-Martinez, Zhu Zhu, Junning Case, Paloma Cejas, Miguel Gomez, M. Benamar, Pui Lee, Lauren A. Henderson, Margaret H Chang, Kevin Wei, Henry Long, Carl D. Langefeld, Ignacio Sanz, Jeffrey Sparks, E. Meidan, P. Nigrovic, María Gutiérrez-Arcelus
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 TET2 mutations cause clonal expansion of hematopoietic stem and progenitor cells (HSPCs) and chimeric antigen receptor (CAR)-expressing T cells, driving clonal hematopoiesis (CH) and CAR+ T cell lymphomas, respectively. TET2-mutated CH has been linked to hyperinflammation, but the mechanisms remain poorly understood.
 We developed human isogenic induced pluripotent stem cell (iPSC)-based models of the 3 main CH mutations – DNMT3A R882H, TET2 haploinsufficiency and ASXL1 truncations –and performed integrated transcriptome, chromatin accessibility and DNA methylation (whole genome bisulfite sequencing) analyses in CD34+ HSPCs. Expectedly, DNMT3A- and TET2-mutant HSPCs showed a global decrease and increase, respectively, in DNA methylation, compared to isogenic wild-type (WT) controls. Single-cell RNA-seq analyses and immunoblotting experiments, unexpectedly, showed cell-autonomous activation of the type I interferon (IFN) induction pathway and secretion of bioactive type I IFN by TET2-mutated HSPCs. Furthermore, genomic and functional experiments in iPSC- and cord blood (CB)- HSPCs with TET2 shRNA knockdown or CRISPR/Cas9-mediated knockout demonstrated that, while type I IFN suppresses the expansion of WT CD34+/CD38- cells in a dose-dependent manner, TET2-mutant HSPCs downregulate their downstream IFN response and are thus protected. Importantly, while TET2-deficient CD34+/CD38- cells outgrew co-cultured WT cells, blockade of type I IFN signaling by a IFN alpha and beta receptor subunit 1 (IFNAR1) antibody mitigated their selective advantage.
 To understand the mechanism, we searched for hypermethylated genes with concomitant loss of chromatin accessibility and gene expression specifically in TET2-mutant HSPCs. These analyses, followed by knockdown and knockout (KO) validation experiments, identified silencing of the E3 ubiquitin ligase tripartite motif containing 4 (TRIM4) gene as a putative effector of TET2 deficiency. Corroborating findings from scATAC-seq and TARGET-seq+ datasets of HSPCs from individuals with TET2-mutated CH, clonal cytopenia of undetermined significance (CCUS) and healthy donors, revealed decreased promoter accessibility and expression of TRIM4 in primary TET2-mutant HSPCs. Importantly, engineering TRIM4 deficiency by CRISPR in iPSC- and CB- HSPCs phenocopied TET2 deficiency phenotypes, including monocytic bias and cell-autonomous type I IFN induction. In addition, TRIM4-deficient iPSC- and CB- HSPCs outcompeted WT HSPCs in liquid culture and serial replating assays in semisolid media, recapitulating the clonal advantage of TET2-deficient HSPCs. Furthermore, ectopic TRIM4 re-expression in TET2-deficient HSPCs, rescued their clonal expansion both in vitro and in competitive repopulation assays in NSGS mice.
 While TRIM proteins have known pleiotropic roles in immune signaling, TRIM4 is not well-characterized and lacks a mouse homolog. To identify substrates of TRIM4 we first performed integrated quantitative proteomics and ubiquitinome profiling in TET2+/-, TRIM4 KO, and isogenic WT iPSC-HSPCs. These analyses nominated 190 proteins with decreased ubiquitination and concomitant increased abundance by both TET2 and TRIM4 deficiency. Knockout of key components of the 3 main innate sensing pathways – TRIF/MYD88 (Toll-like receptor signaling pathway), MAVS (cytosolic RNA sensing) and STING (cytosolic DNA sensing) – identified MAVS as the dominant pathway. DExH-box helicase 58(DHX58), one of the 190 proteins identified by our proteomics analyses and a positive regulator of the MAVS pathway, was confirmed by knockout experiments to mediate the aberrant IFN production by TET2-deficient HSPCs. These results support a model whereby TET2 deficiency and consequent TRIM4 silencing activate type I IFN production through decrease in proteasome-mediated degradation of DHX58. Finally, we show that IFNAR1 blockade by the FDA-approved monoclonal antibody anifrolumab alleviates the clonal advantage of TET2-deficient T cells expressing a CD19-targeted CAR.
 In summary, we report secretion of type I IFN by TET2-mutant HSPCs as the cause of a pro-inflammatory local microenvironment that favors the expansion of mutant cells; we discover TRIM4 silencing as a critical effector of TET2 deficiency; we characterize TRIM4 as a novel negative regulator of type I IFN induction through degradation of DHX58; and we propose a new pharmacologic approach to mitigate the adverse clinical consequences of TET2 mutations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3c2c311180626235836ae13cb5857af7df9374" target='_blank'>
              TET2 mutations drive cell-autonomous type I interferon production and selective advantage through TRIM4 silencing
              </a>
            </td>
          <td>
            Baosen Jia, Igor Fijalkowski, Manon Jaud, Minh Nguyen, Lewis Tomalin, Emmanuel Olivier, Chih-wei Fan, V. Jethalia, N. Thongon, Max Dougherty, Saul Carcamo, Franco Izzo, S. Colla, Maria Themeli, Elvin Wagenblast, D. Hasson, Minji Byun, Panagiotis Ntziachristos, E. Papapetrou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroblastoma is an embryonal tumour that arises from the malignant transformation of neural crest cells and remains one of the deadliest malignancies in children under five. Neural crest development is regulated by dynamic switches in transcriptional programmes, guided by a variety of growth factors. Due to its developmental origin, neuroblastoma is unique in that these tumours often retain overactivation of growth factor signalling, which can be targeted by receptor tyrosine kinase (RTK) inhibitors. However, mutations in kinases, except for ALK, are extremely rare in neuroblastoma. Furthermore, the high degree of intratumoural heterogeneity often renders RTK inhibition ineffective as a monotherapy. For high-risk tumours, which lack effective treatment options, there remains an unmet need for targeted therapies. This review summarises the roles of growth factor receptors in neural crest and neuroblastoma development in light of recent single-cell studies. We provide a systematic overview of RTK inhibitors that can target growth factor signalling in neuroblastoma and detail their current status in clinical development. We also explore the role of intratumoural heterogeneity in resistance to RTK inhibitors, focusing on the adrenergic-to-mesenchymal transition, which drives a switch in growth factor receptor expression. Finally, we discuss strategies to overcome RTK inhibitor resistance by targeting neuroblastoma cell plasticity, disrupting downstream signalling pathways, or inhibiting escape mechanisms from cell death. This review provides a theoretical basis for developing novel combination therapies incorporating RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251f60bb5eddba4ba1d386fbe214011bbf9166f4" target='_blank'>
              Advances in Targeting Growth Factor Signalling in Neuroblastoma and Overcoming Drug Resistance
              </a>
            </td>
          <td>
            K. Ivanenko, Ruslan Shaymardanov, Vladimir S Prassolov, Timofey Lebedev
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Recent studies have increasingly highlighted the pivotal role of fatty acid metabolism in regulating hematopoietic stem cells (HSCs) fate and its influence on tumorigenesis, metastasis, and the development of drug resistance. Carnitine palmitoyltransferase 1A (CPT1A), the rate-limiting enzyme of fatty acid β-oxidation, is essential for the maintenance of normal HSCs function. However, its role in aberrant hematopoiesis, particularly in aberrant stem cells, remains largely unexplored. In this study, we tried to address this issue and provided novel insights into the therapeutic strategy for the eradication of aberrant stem cells in AML.



 The expression and regulation of CPT1A by the AML1-ETO fusion protein were assessed using public ChIP-seq and Cut&Run data, and verified in a doxycycline-inducible U937 cell model. Metabolomic profiling, pharmacological inhibition were performed in Runx1Runx1t1/+; Mx1-Cre mouse (AML1/ETO mouse), and CRISPR-Cas9-mediated knockout of Cpt1a was performed in AML1/ETO; R26-CAG-Cas9 mice. Flow cytometry was used to evaluate metabolic activity, HSPCs populations, and mature hematopoietic lineages. Colony formation was performed to evaluate the function of HSPCs. The downstream functional patterns were studied following RNA-seq in c-kit+ cells and RT-qPCR was used to verify molecular changes.



 In this study, we found that CPT1A was highly expressed in leukemic stem and progenitor cells, and elevated CPT1A levels were significantly associated with poor clinical prognosis. Further analysis demonstrated that the common leukemia fusion protein, AML1-ETO oncoprotein directly regulated the expression of CPT1A. To further investigate the functional role of CPT1A, we employed AML/ETO mouse, which is characterized by pathological expansion of aberrant HSCs and HPCs populations and blockade in myeloid differentiation. RNA sequencing revealed a marked enrichment of Cpt1a expression in the aberrant stem/progenitor compartments of AML/ETO mice.
 Given the central role of CPT1A in cellular metabolism, we next examined metabolic alterations in AML1/ETO aberrant HSCs compared to wild-type counterparts. We observed that carnitine metabolism was selectively upregulated in the context of globally suppressed metabolic activity, which corresponded with elevated Cpt1a expression. To determine whether CPT1A represents a functional vulnerability, we assessed the effects of pharmacologic and genetic Cpt1a inhibition in the AML/ETO model. Cpt1a suppression induced extensive metabolic reprogramming, notably enhancing glycolysis–OXPHOS coupling, which led to partial resolution of the differentiation blockade and restoration of the myeloid developmental hierarchy.
 Furthermore, Cpt1a knockout activated the Myc/Bcl-2 signaling axis, promoting cell cycle progression and facilitating myeloid maturation. These findings indicate that CPT1A maintains aberrant stem cell quiescence and contributes to differentiation arrest in AML1-ETO related leukemogenesis.



 In summary, our study demonstrates that targeting CPT1A induces a critical state transition in aberrant stem cells, converting them from a quiescent, primitive phenotype to a proliferative and differentiation-competent state. This metabolic intervention effectively alleviates AML1-ETO mediated differentiation block and restores myeloid lineage commitment. Our findings position CPT1A as a metabolic checkpoint sustaining LSC stemness and resistance, and suggest that CPT1A inhibition may provide a promising therapeutic strategy for the eradication of chemoresistant leukemic stem cells in t(8;21) AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a022f5e1d53c49eea5e6983998b3b987f9441f" target='_blank'>
              Inhibition of CPT1A reverses AML1-ETO-induced differentiation blockage by activating BCL-2/MYC signaling and metabolic reprogramming
              </a>
            </td>
          <td>
            Shaowei Qiu, Fan Ding, Yu Liu, Haoren Wang, Qing Rao, R. Gu, Min Wang, Jianxiang Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 BackgroundGene expression is regulated by post-translational modification of chromatin structures by a network of epigenetic regulators. EP300 and CREBBP (CBP) are paralogous histone acetyltransferases that regulate enhancer activity and transcription mainly by histone H3 lysine 27 acetylation (H3K27ac). Although these co-transcription factors display overlapping regulatory roles, their functional overlap varies across cancers. Multiple myeloma (MM) is a hematological malignancy reliant on enhancer-driven transcription factors such as the IRF4-MYC axis. However, the specific contribution of EP300 remains to be explored. Here, we investigated the role and selective dependency of EP300 in MM, its functional non-redundancy relative to CBP, and its potential as a therapeutic target.MethodsTo investigate the role of EP300, we used shRNA and PROTAC JQAD1 to selectively degrade EP300 in MM cell lines. We evaluated the functional changes by performing cell viability, apoptosis, migration, and invasion assays. Transcriptomic changes following EP300 loss were assessed via RNA sequencing. Chromatin remodeling and enhancer function were profiled using ATAC-seq and ChIP-seq for EP300, CBP, and H3K27ac. EP300 and CBP-associated protein complexes were identified via immunoprecipitation followed by mass spectrometry (IP–MS). To examine EP300 dependency in MM, CUT&RUN was performed on CD138⁺ plasma cells isolated from newly diagnosed and relapsed/refractory MM patients. Simultaneously, we employed public databases such as DepMap to determine EP300 dependency in MM cell lines and its correlation with high-risk cytogenic factors such as t(4;14), del(17p), 1q gain, and RAS mutations. ResultsWe observed decreased cell viability, increased apoptosis, reduced cell migration, and invasion potential following EP300 degradation across MM cell lines. Reduction of EP300 resulted in a wide array of transcriptomic changes tied to the enhancer network across different cell lines. Epigenomic profiling revealed global enhancer disruption and loss of H3H27ac at key regulatory regions and IRF4-MYC super enhancers. IP-Mass spectroscopy revealed unique interacting partners with EP300, which did not interact with paralogous CBP. Furthermore, CUT&RUN sequencing from patient samples confirmed EP300 binding with the lineage-defining enhancers.DepMap analysis supported a consistent pattern of EP300 dependency across MM lines, especially in the context of IRF4-high expression, regardless of genetic subtype.ConclusionAlthough EP300 and CBP share 90% structural similarity, EP300 uniquely regulates the enhancer activity required for transcriptional identity and survival of MM. Importantly, selective degradation of EP300, as opposed to dual EP300/CBP targeting, leads to decreased thrombocytopenia in pre-clinical models. In addition, combining EP300 inhibition with other treatment modalities exhibit enhanced therapeutic efficacy. Overall, our findings endorse further development of EP300-targeted aproaches for patients with high-risk and relapsed multiple myeloma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d0c70f9f236ac123f12e972b00418aeb77647e" target='_blank'>
              Deciphering the role of EP300 in maintaining the enhancer landscape and lineage-specific epigenetic vulnerability in multiple myeloma
              </a>
            </td>
          <td>
            Rajib Shome, Mehmet K Samur, S. Talluri, Rie Matsubara, Advait Joshi, Farah Rexha, Diana D Cirstea, Andrew J. Yee, Nikhil C Munshi, N. Raje
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [15, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>