<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-31 09:28:51 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>29</td>
          <td>42</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Objective Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance. Methods Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation. Results We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation. Conclusion This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4cbce0b7fb1d47370357ef7f4f5d368e5b4543" target='_blank'>
              Genetic Profiling of Acute and Chronic Leukemia via Next-Generation Sequencing: Current Insights and Future Perspectives
              </a>
            </td>
          <td>
            Laras Pratiwi, Fawzia Hanum Mashudi, Mukti Citra Ningtyas, H. Sutanto, P. Z. Romadhon
          </td>
          <td>2025-03-28</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Transposable elements (TEs), which constitute nearly half of the human genome, have long been regarded as genomic "dark matter". However, their reactivation in tumor cells, resulting in the production of TE-chimeric transcripts (TCTs), has emerged as a potential driver of cancer progression. The complexity and full extent of these transcripts remain elusive, largely due to the limitations of short-read next-generation sequencing technologies. These methods have struggled to comprehensively capture the diversity and structure of TCTs, particularly those involving short interspersed nuclear elements (SINEs) or closely co-transcribed TEs. Methods Leveraging full-length cDNA sequencing technology based on nanopore sequencing platform, we developed a customized pipeline for identifying and quantifying TCTs in 19 lung adenocarcinoma (LUAD) cell lines. The short-read RNA-seq dataset from a LUAD corhort (~ 200 tumor samples) was employed to validate the identified TCTs and explore their association with tumor progression. To assess the functional roles of a specific TCTs, cell migration and cell proliferation assays were performed. Results We uncovered 208 unique TCT candidates in the LUAD cell lines. Our approach allowed for the identification of cryptic promoters and terminators within non-transposing TEs. Notably, we identified a chimeric transcript involving MIR_HKDC1, which appears to play a significant role in the progression of LUAD. Furthermore, the expression of these TCTs were associated with poor clinical outcomes in a cohort of LUAD patients, suggesting their potential as novel biomarkers for both LUAD progression and prognosis. Conclusions Our study underscores the application of long-read sequencing to unravel the complex landscape of TCTs in LUAD. We provide a comprehensive characterization of TCTs in LUAD, exploring their potential regulatory roles in cancer progression. These findings contribute to a deeper understanding of the genomic intricacies underlying cancer, and offer new directions for the development of targeted therapies and personalized treatment strategies for LUAD. This research highlights the potential of TCTs as both biomarkers and therapeutic targets in the oncogenesis, offering new insights into the interplay between transposable elements and gene regulation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13888-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1fcc8488427a1fecd1258b2070d8c49656eecc" target='_blank'>
              Long-read RNA sequencing enables full-length chimeric transcript annotation of transposable elements in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yang Li, Yahui Liu, Yingxin Xie, Yaxuan Wang, Jing Wang, Huan Wang, Lin Xia, Dan Xie
          </td>
          <td>2025-03-15</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398c45e50f2a25bf4d4f179535f4ed528664ab8c" target='_blank'>
              Genomic profiling reveals molecular heterogeneity in patients with Richter transformation (RT) and chronic lymphocytic leukemia (CLL)
              </a>
            </td>
          <td>
            Shulan Tian, Hanyin Wang, S. Parikh, Yuanhang Liu, Helen Jin-4 Lee, Erik Jessen, Eric W. Klee, Yucai Wang, Fan Leng, Min Shi, Cinthya J Zepeda-Mendoza, Rong He, Saad J. Kenderian, Linda B. Baughn, D. L. Dyke, P. Hampel, Neil E. Kay, Esteban Braggio, S. Slager, Huihuang Yan, Wei Ding
          </td>
          <td>2025-03-18</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Cancer progression relies on the ability of cells to adapt to challenging environments overcoming stresses and growth constraints. Such adaptation is a multifactorial process that depends on the rapid reorganization of many basic cellular mechanisms. Protein synthesis is often dysregulated in cancer, and translational reprogramming is emerging as a driving force of cancer adaptive plasticity. Long non-coding RNAs (lncRNAs) represent the main product of genome transcription. They outnumber mRNAs by an order of magnitude and their expression is regulated in an extremely specific manner depending on context, space and time. This heterogeneity is functional and allows lncRNAs to act as context-specific, fine-tuning controllers of gene expression. Multiple recent evidence underlines how, besides their consolidated role in transcription, lncRNAs are major players in translation control. Their capacity to establish multiple and highly dynamic interactions with proteins and other transcripts makes these molecules able to play a central role across all phases of protein synthesis. Even if through a myriad of different mechanisms, the action of these transcripts is dual. On one hand, by modulating the overall translation speed, lncRNAs participate in the process of metabolic adaptation of cancer cells under stress conditions. On the other hand, by prioritizing the synthesis of specific transcripts they help cancer cells to maintain high levels of essential oncogenes. In this review, we aim to discuss the most relevant evidence regarding the involvement of lncRNAs in translation regulation and to discuss how this specific function may affect cancer plasticity and resistance to stress. We also expect to provide one of the first collective perspectives on the way these transcripts modulate gene expression beyond transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, M. K. Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background: Breast cancer is the most prevalent malignancy among females worldwide. Extensive research has highlighted cancer stem cells (CSCs) as critical drivers of tumor initiation, progression, recurrence, and therapy resistance. However, the heterogeneity of breast cancer stem cells (BCSCs) and their dynamic roles within the tumor microenvironment remain inadequately understood. Methods: This study utilized the single-cell RNA sequencing dataset to categorize BCSCs into two subgroups within the breast cancer microenvironment and investigate their pseudo-time developmental dynamics. Bulk transcriptomic data from TCGA-BRCA were integrated to assess the prognostic significance and infiltration abundance of the BCSCs-2 subgroup. Functional enrichment, co-expression network analysis, and somatic mutation profiling were performed to elucidate key biological pathways and genetic features. Additionally, drug sensitivity analyses were conducted using the Connectivity Map database to identify potential therapeutic strategies. Results: A total of 459 BCSCs were identified and further classified into two distinct subpopulations: BCSCs-1 and BCSCs-2. High infiltration of BCSCs-2 was associated with poor prognosis and an immunosuppressive tumor microenvironment. Co-expression network analysis identified 16 key genes linked to BCSCs-2, while somatic mutation analysis revealed distinct mutation patterns associated with its infiltration. Drug sensitivity analysis suggested that patients with high BCSCs-2 infiltration could benefit from classical chemotherapy agents, such as Cisplatin, and other novel therapeutic compounds. Conclusions: This study offers novel insights into the heterogeneity and functional roles of BCSCs in breast cancer. The findings highlight the prognostic and therapeutic importance of the BCSCs-2 subgroup, providing potential biomarkers and therapeutic targets for precision medicine in breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/491a724da75389432860fa6c4bc953c58c7363a6" target='_blank'>
              Multi-Omics Profiling Identifies a High-Risk Subgroup of Breast Cancer Stem Cells for Prognostic Stratification and Personalized Treatment
              </a>
            </td>
          <td>
            Guixin Wang, Ziyi Chen, Yao Tian, Yuxin Zhu, Shuo Wang, Wenbin Song, Xin Wang, Yingxi Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd49383fc8e846f3a7bdc91c9cc99940e4850c1" target='_blank'>
              Innovative logic "AND" gate gene circuits for bladder cancer treatment: preclinical study.
              </a>
            </td>
          <td>
            Chaojie Xu, Ying Dong, Dongchen Pei, Xintao Zhang, Xiaohong Han, Congcong Cao, Baorui Wu, Changning Lv, Zhengjun Kang, Liqun Zhou, Yuchen Liu, Lin Yao
          </td>
          <td>2025-02-04</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d77495b125c5aaab4dcf465d06c71c35fe30e9b" target='_blank'>
              SNHG10: A Novel Long Non-coding RNA with Multifaceted Roles in Human Cancers.
              </a>
            </td>
          <td>
            Haodong He, Jingjie Yang, Yan Zhou, Lihan Chen, Chuyuan Liao, Ting Gong, Tongtong Li, Haoran Liu, Yunfei Pan, Pengbo Zhang, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2025-03-05</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Targeting cancer stem cells (CSCs) is crucial for effective cancer treatment, yet resistance mechanisms to LGR5+ CSC depletion in WNT-driven colorectal cancer (CRC) remain elusive. In the present study, we revealed that mutant intestinal stem cells (SCs) depart from their canonical identity, traversing a dynamic phenotypic spectrum. This enhanced plasticity is initiated by oncofetal (OnF) reprogramming, driven by YAP and AP-1, with subsequent AP-1 hyperactivation promoting lineage infidelity. The retinoid X receptor serves as a gatekeeper of OnF reprogramming and its deregulation after adenomatous polyposis coli (APC) loss of function establishes an OnF ‘memory’ sustained by YAP and AP-1. Notably, the clinical significance of OnF and LGR5+ states in isolation is constrained by their functional redundancy. Although the canonical LGR5+ state is sensitive to the FOLFIRI regimen, an active OnF program correlates with resistance, supporting its role in driving drug-tolerant states. Targeting this program in combination with the current standard of care is pivotal for achieving effective and durable CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0203e20c6bc4f26349dbeefa5213e0b374637b7e" target='_blank'>
              Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer
              </a>
            </td>
          <td>
            Slim Mzoughi, Megan Schwarz, Xuedi Wang, D. Demircioglu, Gulay Ulukaya, Kevin Mohammed, Habiba Zorgati, Denis Torre, Lewis E. Tomalin, Federico Di Tullio, Carlos Company, Yuliia Dramaretska, M. Leushacke, Bruno Giotti, T. Lannagan, D. Lozano-Ojalvo, P. Karras, Peter B Vermeulen, Dan Hasson, Robert P Sebra, A. Tsankov, Owen J. Sansom, J. Marine, Nick Barker, G. Gargiulo, Ernesto Guccione
          </td>
          <td>2025-02-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Multiple myeloma (MM) is a blood cancer of bone marrow that is difficult to diagnose and treat because it changes over time and has complex interactions with the body’s immune system. This review explores how single-cell technologies help scientists understand multiple myeloma more comprehensively by identifying hidden tumor clones, tracking how the disease evolves, and discerning why some treatments stop working. Researchers have found that certain genetic changes, like mutations or deletion of TP53, loss of chromosome 13, and gain of chromosome 1, make the cancer more resistant to treatment, and that the immune system often fails to fight off MM because there is a suppressive environment around the tumor. New treatments, like Chimeric Antigen Receptor T cell (CAR-T) cell therapy and Bispecific T cell engagers (BiTE), take advantage of the immune microenvironment and are showing promise in targeting myeloma cell directly. However, they are not always effective and can have adverse effects. By studying single-cell resolution data to identify new vulnerabilities in myeloma cells and pathways related to resistance that can be different within the cells in a single patient, new treatment regimens can be identified and tailored for individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, Travis S. Johnson
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Transcriptional dysregulation is a hallmark of cancer initiation and progression, driven by genetic and epigenetic alterations. Enhancer reprogramming has emerged as a pivotal driver of carcinogenesis, with cancer cells often relying on aberrant transcriptional programs. The advent of high-throughput sequencing technologies has provided critical insights into enhancer reprogramming events and their role in malignancy. While targeting enhancers presents a promising therapeutic strategy, significant challenges remain. These include the off-target effects of enhancer-targeting technologies, the complexity and redundancy of enhancer networks, and the dynamic nature of enhancer reprogramming, which may contribute to therapeutic resistance. This review comprehensively encapsulates the structural attributes of enhancers, delineates the mechanisms underlying their dysregulation in malignant transformation, and evaluates the therapeutic opportunities and limitations associated with targeting enhancers in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e943d1571a44f68ed1e894e56544a330cb5757e" target='_blank'>
              Enhancer reprogramming: critical roles in cancer and promising therapeutic strategies
              </a>
            </td>
          <td>
            Jinshou Yang, Feihan Zhou, Xiyuan Luo, Yuan Fang, Xing Wang, Xiaohong Liu, Ruiling Xiao, Decheng Jiang, Yuemeng Tang, Gang Yang, Lei You, Yupei Zhao
          </td>
          <td>2025-03-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic malignancy characterized by poor clinical outcomes, frequently exacerbated by mutations in the FMS-like tyrosine kinase 3 (FLT3) gene. Although FLT3 inhibitors (FLT3i) have emerged as promising therapeutic agents, the absence of molecular biomarkers to predict FLT3i response remains a critical limitation in clinical practice. In this study, we performed a comprehensive multi-omics analysis integrating transcriptomic, proteomic, and pharmacogenomic data from the Beat AML cohort, the Cancer Cell Line Encyclopedia (CCLE), and the PXD023201 repository to elucidate the molecular consequences of FLT3 mutations in AML. Our analysis revealed significant differences in RNA and protein expression profiles between FLT3-mutant and wild-type AML cases, with a particularly striking association between FLT3 mutations and immune suppression. We further evaluated the drug sensitivity of FLT3-mutant patients to 3 FDA-approved FLT3i, gilteritinib, midostaurin, and quizartinib, and observed heightened sensitivity in FLT3-mutant cohorts, accompanied by the activation of immune-related pathways in treatment-responsive groups. These findings suggest a potential synergy between FLT3i efficacy and immune activation. Through rigorous bioinformatic analysis, we identified 3 candidate biomarkers: CD36, SASH1, and NIBAN2, associated with FLT3i sensitivity. These biomarkers were consistently upregulated in favorable prognostic subgroups and demonstrated strong correlations with immune activation pathways. The identification of CD36, SASH1, and NIBAN2 as predictive biomarkers offers a novel toolset for stratifying FLT3i response and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db31fdfefb748c6a8ae42e65130f654333e405d0" target='_blank'>
              Multi-omics integration reveals immune hallmarks and biomarkers associated with FLT3 inhibitor sensitivity in FLT3-mutated AML
              </a>
            </td>
          <td>
            Mingming Niu, Ning Wang, Dandan Yang, Lixia Fu, Yang Yang, Long Shen, Hong Wang, Xianfeng Shao
          </td>
          <td>2025-03-19</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="The classical paradigm of drug screening often faces significant limitations due to the challenges associated with identifying molecular or cellular read-outs that are relevant to specific diseases. This issue is particularly pronounced for the thousands of diseases of genetic origin, given the abundance of databases listing disease-associated changes in gene expression and various transcripts, indicating that potential read-outs may be concealed within these resources. To remedy this, an alternative approach was tested: compounds were evaluated for their effects on gene expression and alternative splicing in a healthy cell model, and the resulting data were matched to molecular signatures of diseases. A subset of 50 FDA-approved drugs was tested on mesenchymal stem cells derived from a human pluripotent stem cell line. Over half of the compounds altered gene expression, many affecting pathways linked to monogenic diseases. One hit, increased SQSTM1 expression induced by prazosin, was further validated in ALS models caused by SQSTM1 haploinsufficiency, including patient-derived fibroblasts, SQSTM1-depleted hiPSC-derived motor neurons, and a zebrafish model. Extending this paradigm could involve testing diverse cell types and larger drug libraries. One Sentence Summary A novel drug screening approach for monogenic diseases integrating human pluripotent stem cell derivatives and RNA sequencing to profile gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2756ee22739ee10642a0c818c71c07402f6f6f2e" target='_blank'>
              Integrative Analysis of Drug-Gene Expression Signatures in Human Pluripotent Stem Cells Identifies Novel Drug Candidates for ALS and Monogenic Diseases
              </a>
            </td>
          <td>
            Florine Roussange, Jacqueline Gide, Johana Tournois, Michel Cailleret, A. Boland, Christophe Battail, J. Deleuze, Hélène Polvèche, Didier Auboeuf, Knut Brockmann, E. Kabashi, A. Marian, Lina El Kassar, Sophie Blondel, François Salachas, G. Bruneteau, Marc Peschanski, C. Martinat, Sandrine Baghdoyan
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Molecular profiles of mesenchymal stem cells (MSCs) are needed to standardize the composition and effectiveness of MSC therapeutics. This study employs RNA sequencing to identify genes to be used in concert with CD264 as a molecular profile of aging MSCs at a clinically relevant culture passage. CD264− and CD264+ populations were isolated by fluorescence-activated cell sorting from passage 4 MSC cultures. CD264+ MSCs exhibited an aging phenotype relative to their CD264− counterpart. Donor-matched CD264−/+ mRNA samples from 5 donors were subjected to pair-ended, next-generation sequencing. An independent set of 5 donor MSCs was used to validate differential expression of select genes with quantitative reverse transcription PCR. Pairwise differential expression analysis identified 2,322 downregulated genes and 2,695 upregulated genes in CD264+ MSCs relative to donor-matched CD264− MSCs with a Benjamini–Hochberg adjusted p-value (BH padj) < 0.1. Nearly 25% of these genes were unique to CD264−/+ MSCs and not differentially expressed at a significance level of BH padj < 0.1 in previous RNA sequencing studies of early- vs. late-passage MSCs. Least Absolute Shrinkage and Selection Operator regression identified microtubule-associated protein 1A (MAP1A) and pituitary tumor-transforming gene 1 (PTTG1) as predictive genes of CD264+ MSCs. Combined MAP1A and PTTG1 expression correctly classified CD264 status of MSC samples with an accuracy of 100%. Differential expression and predictive ability of MAP1A and PTTG1 compared favorably with that of existing senescence markers expressed in early passage CD264−/+ MSCs. This study provides the first linkage of MAP1A to CD264, aging and senescence. Our findings have application as quality metrics to standardize the composition of MSC therapies and as molecular targets to slow/reverse cellular aging. Supplementary Information The online version contains supplementary material available at 10.1007/s10616-025-00724-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df97a5ece885215c844356881d36483043d3e173" target='_blank'>
              RNA sequencing identifies MAP1A and PTTG1 as predictive genes of aging CD264+ human mesenchymal stem cells at an early passage
              </a>
            </td>
          <td>
            Margaret K. Giler, H. A. Tucker, Amanda K. Foote, Avery G. Francis, Sean D. Madsen, Yao-Zhong Liu, Kim C. O’Connor
          </td>
          <td>2025-02-19</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="InfoScan is a novel bioinformatics tool designed for the comprehensive analysis of full-length single-cell RNA sequencing (scRNA-seq) data. It enables the identification of unannotated transcripts and rare cell populations, providing a powerful platform for transcriptome characterization. In this study, InfoScan was applied to glioblastoma multiforme (GBM), identifying a rare “neoplastic-stemness” subpopulation exhibiting cancer stem cell-like features. Functional analyses suggested that tumor-associated macrophages (TAMs) secrete SPP1, which binds to CD44 on neoplastic-stemness cells, activating the PI3K/AKT pathway and driving lncRNA transcription to promote metastasis. Integration of TCGA and CGGA datasets further supported these findings, highlighting key mutations associated with the neoplastic-stemness subpopulation. Drug sensitivity assays indicated that neoplastic-stemness cells might be sensitive to omipalisib, a PI3K inhibitor, pointing to a potential therapeutic target. InfoScan offers a robust framework for exploring complex transcriptomic landscapes and characterizing rare cell populations, providing valuable insights into GBM biology and advancing precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d19fc728a932329a1c6b81702c24f21513cb2f5" target='_blank'>
              InfoScan: A New Transcript Identification Tool Based on scRNA-Seq and Its Application in Glioblastoma
              </a>
            </td>
          <td>
            Shiqiang Mei, Jinjin Huang, Zhen Zhang, Haotian Lei, Qiaojuan Huang, Lianghu Qu, Lingling Zheng
          </td>
          <td>2025-02-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, Daniel Ejim Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Methylation processes in different molecular contexts (DNA, RNA, and histones) are controlled by different regulatory factors and serve as critical determinants in cancer development. However, the mechanistic links between these epigenetic modifications during malignant transformation, metastasis, disease relapse, and therapeutic resistance remain incompletely understood. In this research, we investigated the transcriptional and genetic alterations of regulators associated with 3 major types of methylation modifications in clear cell renal cell carcinoma. Utilizing ChIP/MeRIP-seq and 450K methylation array data, we identified genes regulated by multiple methylation modifications and constructed a scoring model to quantify the methylation patterns for each patient. Our findings indicate that patients with a low score may be more likely to respond to immunotherapy, whereas patients with a high score may be more sensitive to targeted therapy, such as RITA, Pazopanib, Irlotinib, SU-11274, BRD-K16762525, and FCCP. In conclusion, the score model can serve as a valuable biomarker to guide clinical selection of immunotherapy and targeted drugs and help to improve personalized clear cell renal cell carcinoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4691220e407b79a42a6b06dcf7ad89ec71062f" target='_blank'>
              Investigating the efficacy of immune checkpoint inhibitors in clear cell renal cell carcinoma based on methylation cross talk scoring
              </a>
            </td>
          <td>
            Qinglong Du, Qiyuan Wang, Chen Yang, Yiping Wang, Huiyang Yuan, Bing Zhang, Hong Ji, Shuai Fu, Chunlei Xue
          </td>
          <td>2025-03-14</td>
          <td>Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b08a5b6171c80002da0a1517c54be1999c70c6" target='_blank'>
              MED12 dysregulation: insights into cancer and therapeutic resistance.
              </a>
            </td>
          <td>
            R. Bhole, Jagruti Shinkar, Sonali Labhade, Pawan N Karwa, Harshad S. Kapare
          </td>
          <td>2025-03-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer exhibits substantial inter- and intra-tumor heterogeneity, with features that present significant challenges in advancing biomarker discovery and the development of targeted therapeutics. To fill this gap, we employed single-cell RNA sequencing (scRNA-seq) and advanced bioinformatics tools to evaluate the transcriptomic heterogeneity of immortalized, non-transformed (BEAS2B) and transformed (H460) lung epithelial cell lines and their responses to carcinogen challenge. Gene expression profiles resolved four primary clusters further discretized into unique subclusters based on genetic signatures and phenotypic profiles. Profiles of long non-coding RNAs (lncRNAs) identified microRNA host genes, antisense RNA genes, divergent transcript, and long intergenic non-coding RNAs as contributors to cellular heterogeneity. These findings indicate that distinct patterns of gene expression, remarkably in lncRNAs, define cellular heterogeneity in non-transformed versus transformed cells. These features can be exploited for the development of therapies directed at specific cell subpopulations in precancerous lesions and within lung tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/233c01bff81dc65b1be1e7c0ebbee44d4a90ef36" target='_blank'>
              Single-Cell RNA Sequencing Reveals Extensive Heterogeneity and Unique Gene Trajectories in Non-Transformed and Transformed Human Lung Epithelial Cells: Insights into the Role of LncRNAs in Tumor Heterogeneity
              </a>
            </td>
          <td>
            Sokviseth Moeng, Andres D. Chamorro-Parejo, Minsun S. Jeon, James J. Cai, Kenneth S Ramos
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aneuploidy is prevalent in cancer and associates with fitness advantage and poor patient prognosis. Yet, experimentally induced aneuploidy initially leads to adverse effects and impaired proliferation, suggesting that cancer cells must adapt to aneuploidy. We performed in vitro evolution of cells with extra chromosomes and obtained cell lines with improved proliferation and gene expression changes congruent with changes in aneuploid cancers. Integrated analysis of cancer multi-omics data and model cells revealed increased expression of DNA replicative and repair factors, reduced genomic instability, and reduced lysosomal degradation. We identified E2F4 and FOXM1 as transcription factors required for adaptation to aneuploidy in vitro and in cancers and validated this finding. The adaptation to aneuploidy also coincided with specific copy number aberrations that correlate with poor patient prognosis. Chromosomal engineering mimicking these aberrations improved aneuploid cell proliferation, while loss of previously present extra chromosome impaired it. The identified common adaptation strategies suggest replication stress, genomic instability, and lysosomal stress as common liabilities of aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2024-03-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ce7ec0649fbc5d3e5e12bff74054318a2f6bce" target='_blank'>
              MEDUSA for Identifying Death Regulatory Genes in Chemo-genetic Profiling Data.
              </a>
            </td>
          <td>
            Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-02-07</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Chromosomal instability (CIN) is involved in about 70% of colorectal cancers (CRCs) and is associated with poor prognosis and drug resistance. From a clinical perspective, a better knowledge of these tumour’s biology will help to guide therapeutic strategies more effectively. Methods We used high-density chromosomal microarray analysis to evaluate CIN level of patient-derived organoids (PDOs) and their original mCRC tissues. We integrated the RNA-seq and mass spectrometry-based proteomics data from PDOs in a functional interaction network to identify the significantly dysregulated processes in CIN. This was followed by a proteome-wGII Pearson correlation analysis and an in silico validation of main findings using functional genomic databases and patient-tissues datasets to prioritize the high-confidence CIN features. Results By applying the weighted Genome Instability Index (wGII) to identify CIN, we classified PDOs and demonstrated a good correlation with tissues. Multi-omics analysis showed that our organoids recapitulated genomic, transcriptomic and proteomic CIN features of independent tissues cohorts. Thanks to proteotranscriptomics, we uncovered significant associations between mitochondrial metabolism and epithelial-mesenchymal transition in CIN CRC PDOs. Correlating PDOs wGII with protein abundance, we identified a subset of proteins significantly correlated with CIN. Co-localisation analysis in PDOs strengthened the putative role of IPO7 and YAP, and, through in silico analysis, we found that some of the targets give significant dependencies in cell lines with CIN compatible status. Conclusions We first demonstrated that PDO models are a faithful reflection of CIN tissues at the genetic and phenotypic level. Our new findings prioritize a subset of genes and molecular processes putatively required to cope with the burden on cellular fitness imposed by CIN and associated with disease aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03308-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae0df005b2d8a0f47444591e98d3dc4bb550830" target='_blank'>
              Epidermal Growth Factor Receptor Aberrations Identified by Next-generation Sequencing in Patients with Metastatic Cancers.
              </a>
            </td>
          <td>
            Minkyue Shin, Dae-Ho Choi, Jaeyun Jung, Deokgeun Kim, Sung Hee Lim, Seung-Tae Kim, Jung Yong Hong, Se Hoon Park, Joon-Oh Park, Kyoung-Mee Kim, Jeeyun Lee
          </td>
          <td>2025-02-21</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by expanding blood cell clones carrying somatic mutations in healthy aged individuals and is associated with various age-related diseases and all-cause mortality. While CH mutations affect diverse genes associated with myeloid malignancies, their mechanisms of expansion and disease associations remain poorly understood. We investigate the relationship between clonal fitness and clinical outcomes by integrating data from three longitudinal aging cohorts (n=713, observations=2,341). We demonstrate pathway-specific fitness advantage and clonal composition influence clonal dynamics. Further, the timing of mutation acquisition is necessary to determine the extent of clonal expansion reached during the host individual’s lifetime. We introduce MACS120, a metric combining mutation context, timing, and variant fitness to predict future clonal growth, outperforming traditional variant allele frequency measurements in predicting clinical outcomes. Our unified analytical framework enables standardized clonal dynamics inference across cohorts, advancing our ability to predict and potentially intervene in CH-related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df776ea20c82ef08bfb6dc73e82dc94bd703a7e7" target='_blank'>
              Clinical progression of clonal hematopoiesis is determined by a combination of mutation timing, fitness, and clonal structure
              </a>
            </td>
          <td>
            Eric Latorre-Crespo, Neil A. Robertson, E. Kosebent, Louise Macgillivray, Lee Murphy, M. Uddin, Eric Whitsel, Michael C. Honigberg, Alexander G. Bick, A. Reiner, Valeria Orru, Michele Marongiu, Francesco Cucca, E. Fiorillo, I. Deary, Sarah Harris, Simon Cox, Riccardo Marioni, Linus Schumacher, Tamir Chandra, Kristina Kirschner
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genomic and epigenomic instability are defining features of cancer, driving tumor progression, heterogeneity, and therapeutic resistance. Central to this process are epigenetic echoes, persistent and dynamic modifications in DNA methylation, histone modifications, non-coding RNA regulation, and chromatin remodeling that mirror underlying genomic chaos and actively influence cancer cell behavior. This review delves into the complex relationship between genomic instability and these epigenetic echoes, illustrating how they collectively shape the cancer genome, affect DNA repair mechanisms, and contribute to tumor evolution. However, the dynamic, context-dependent nature of epigenetic changes presents scientific and ethical challenges, particularly concerning privacy and clinical applicability. Focusing on lung cancer, we examine how specific epigenetic patterns function as biomarkers for distinguishing cancer subtypes and monitoring disease progression and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3b401f96a57f38873c48e3928dd0d7e44a2cbef" target='_blank'>
              Epigenomic Echoes—Decoding Genomic and Epigenetic Instability to Distinguish Lung Cancer Types and Predict Relapse
              </a>
            </td>
          <td>
            Alexandra A Baumann, Zholdas Buribayev, Olaf Wolkenhauer, Amankeldi A. Salybekov, M. Wolfien
          </td>
          <td>2025-02-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) has the highest mortality rate of all breast cancer subtypes and currently lacks effective targeted therapies. LARP6 is an RNA-binding protein associated with cancer promotion, but its mechanism of action in TNBC remains unclear. We conducted RNA sequencing (RNA-seq) and improved RNA immunoprecipitation and sequencing (iRIP-seq) to identify the differentially expressed genes (DEGs) and alternative splice sites bound and regulated by LARP6 in MDA-MB-231 cells. Finally, both RT-qPCR and RIP-qPCR were employed for verification. Our study revealed that LARP6 overexpression altered the expression levels of 171 genes and that the number of regulated alternative splicing events (RASEs) exceeded 1000. The regulated alternative splicing genes (RASGs) corresponding to RASEs were enriched in biological processes such as DNA repair, the cell cycle, and the cellular response to DNA damage stimulus. In addition, we found that LARP6 tends to bind the CGACGAG motif. The intersection of peak-related genes with RASGs suggested that LARP6 can bind to 16 genes and regulate their alternative splicing (AS), thus playing an important role in TNBC progression. Our research indicated that LARP6 may promote the proliferation and invasion of TNBC cells by directly regulating the AS of related genes, providing new clues for targeted therapy for TNBC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92351-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb087ff58a5d637f585f09e70365a21011c3f17" target='_blank'>
              The RNA-binding protein LARP6 regulates the alternative splicing of related genes in MDA-MB-231 cells
              </a>
            </td>
          <td>
            Li Guo, Yaobang Liu, Shuxun Yan, Hong Li, Kai Zhang, Jinping Li
          </td>
          <td>2025-03-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f68c71be9a5f4d5ac85f15c79176d7904112ca" target='_blank'>
              Long-read single-cell RNA sequencing enables the study of cancer subclone-specific genotypes and phenotypes in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            Gage S. Black, Xiaomeng Huang, Yi Qiao, Philip Moos, Deepa Sampath, Deborah M. Stephens, J. Woyach, Gabor T. Marth
          </td>
          <td>2025-02-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583a65cabb3a9c9f197a9eab6fd088b3c233c72e" target='_blank'>
              Naturally occurring variation in gene-associated transposable elements impacts gene expression and phenotypic diversity in woodland strawberry
              </a>
            </td>
          <td>
            Ileana Tossolini, Rocio Tolley, Julia Llinares-Gómez, Camila Zlauvinen, Tuomas Toivainen, T. Hytönen, David Posé, Carmen Martín-Pizarro, P. Manavella
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Simple Summary Proto-oncogenes and tumor-suppressor genes play important roles in preventing the development of tumors. The former supports healthy cell growth and division, while the latter regulates cell division. Mutations in proto-oncogenes, converting them into oncogenes, and mutations in tumor-suppressor genes can lead to uncontrolled cell proliferation and cancer development. This review explores key molecular pathways associated with these genes, highlighting their influence on cancer progression and treatment resistance, which are crucial for creating effective, personalized, and targeted treatments for patients. This comprehensive review also discusses the recent advancements and emerging technologies that enhance the analysis, diagnosis, prevention, and treatment of cancer at the genomic level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4acb0abcdb89e4becb0be9ae1ca6990d3d6cdf49" target='_blank'>
              Translational Advances in Oncogene and Tumor-Suppressor Gene Research
              </a>
            </td>
          <td>
            R. Stojchevski, Edward Agus Sutanto, Rinni Sutanto, N. Hadzi-Petrushev, M. Mladenov, Sajal Raj Singh, J. K. Sinha, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, Rakesh Bhaskar, Dimiter B Avtanski
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is the most common type of acute leukemia, accounting for 20% of cases in children and adolescents. Genome-wide studies have identified genes that are commonly mutated in AML, including many epigenetic regulators involved in either DNA methylation (DNMT3A, TET2, IDH1/2) or histone post-translational modifications (ASXL1, EZH2, MLL1). Several cell lines derived from AML patients are widely used in cancer research. Whether important differences in these cell lines exist remains poorly characterised. Results Here, we used RNA sequencing (RNA-Seq) to contrast the transcriptome of four commonly used AML-derived cell lines: THP-1, NOMO-1, MOLM-13 bearing the common initiating t(9;11) translocation, and MV4.11 bearing the t(4;11) translocation. Gene set enrichment analyses and comparison of key transcription and epigenetic regulator genes revealed important differences in the transcriptome, distinguishing these AML models. Among these, we found striking differences in the expression of clusters of genes located on chromosome 19 encoding Zinc Finger (ZNF) transcriptional repressors. Low expression of many ZNF genes within these clusters is associated with poor survival in AML patients. Conclusion The present study offers a valuable resource by providing a detailed comparative characterisation of the transcriptome of cell lines within the same AML subtype used as models for leukemia research. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11415-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a934cae686db7528797f8dea84b54367fe76eeb" target='_blank'>
              Transcriptomic characterisation of acute myeloid leukemia cell lines bearing the same t(9;11) driver mutation reveals different molecular signatures
              </a>
            </td>
          <td>
            Elise Georges, William Ho, Miren Urrutia Iturritza, L. Eory, Kamila Malysz, U. Sobhiafshar, Alan L. Archibald, Daniel J Macqueen, Barbara Shih, David Garrick, Douglas Vernimmen
          </td>
          <td>2025-03-25</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac8334d0db490037d00a83f0b8a5cb44474c730" target='_blank'>
              Single-cell analysis reveals cellular heterogeneity, gene expression profiles, and pathway dynamics in acne vulgaris.
              </a>
            </td>
          <td>
            Xiaoyi Yang, Xiaoyan Wang, Jiankang Yang
          </td>
          <td>2025-02-07</td>
          <td>Archives of dermatological research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance.
 Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features.
 Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations.
 Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF -mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572fd6e1dc6d845d6923c7249ae200e2136415b1" target='_blank'>
              Genome-wide transcriptomic response of whole blood to radiation
              </a>
            </td>
          <td>
            Ahmed Salah, D. Wollschläger, Maurizio Callari, H. Schmidberger, Federico Marini, S. Zahnreich
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The ability to identify gene functions and interactions in specific cellular contexts has been greatly enabled by functional genomics technologies. CRISPR-based genetic screens have proven invaluable in elucidating gene function in mammalian cells. Single-cell functional genomics methods, such as Perturb-seq and Spear-ATAC, have made it possible to achieve high-throughput mapping of the functional effects of gene perturbations by profiling transcriptomes and DNA accessibility, respectively. Combining single-cell chromatin accessibility and transcriptomic data via multiomic approaches has facilitated the discovery of novel cis and gene regulatory interactions. However, pseudobulk readouts from cell populations can often cloud the interpretation of results due to a heterogeneous response from cells receiving the same genetic perturbation, which could be mitigated by using transcriptional profiles of single cells to subset the ATAC-seq data. Existing methods to capture CRISPR guide RNAs to simultaneously assess the impact of genetic perturbations on RNA and ATAC profiles require either cloning of gRNA libraries in specialized vectors or implementing complex protocols with multiple rounds of barcoding. Here, we introduce CAT-ATAC, a technique that adds CRISPR gRNA capture to the existing 10X Genomics Multiome assay, generating paired transcriptome, chromatin accessibility and perturbation identity data from the same individual cells. We demonstrate up to 77% guide capture efficiency for both arrayed and pooled delivery of lentiviral gRNAs in induced pluripotent stem cells (iPSCs) and cancer cell lines. This capability allows us to construct gene regulatory networks (GRNs) in cells under drug and genetic perturbations. By applying CAT-ATAC, we were able to identify a GRN associated with dasatinib resistance, indirectly activated by the HIC2 gene. Using loss of function experiments, we further validated that the gene, ZFPM2, a component of the predicted GRN, also contributes to dasatinib resistance. CAT-ATAC can thus be used to generate high-content multidimensional genotype-phenotype maps to reveal novel gene and cellular interactions and functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5994854c014d82e61919b765b3c023100d1d1160" target='_blank'>
              Simultaneous single-cell CRISPR, RNA, and ATAC-seq enables multiomic CRISPR screens to identify gene regulatory relationships
              </a>
            </td>
          <td>
            Kaivalya Shevade, Yeqing Angela Yang, Kevin Feng, Karl Mader, V. Sevim, Jacob Parsons, Gunisha Arora, Hasnaa Elfawy, Rachel Mace, S. Federman, Rustam Esanov, Shawn Shafer, Eric D Chow, Laralynne Przybyla
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortalities, characterized by substantial genetic heterogeneity that challenges a comprehensive understanding of its progression. This study employs next-generation sequencing data analysis to transform our comprehension of LUAD pathogenesis. Integrating epigenetic and transcriptomic data of LUAD patients, this approach assessed the critical regulatory occurrences, identified therapeutic targets, and offered profound insights into cancer molecular foundations. We employed the DNA methylation data to identify differentially methylated CpG sites and explored the transcriptome profiles of their adjacent genes. An intersectional analysis of gene expression profiles uncovered 419 differentially expressed genes (DEGs) influenced by smoke-induced differential DNA methylation, among which hub genes, including mitochondrial ribosomal proteins (MRPs), and ribosomal proteins (RPs) such as MRPS15, MRPS5, MRPL33, RPL24, RPL7L1, MRPL15, TUFM, MRPL22, and RSL1D1, were identified using a network-based approach. These hub genes were overexpressed and enriched to RNA processing, ribosome biogenesis, and mitochondrial translation, which is critical in LUAD progression. Enhancer Linking Methylation/Expression Relationship (ELMER) analysis revealed transcription factor (TF) binding motifs, such as JUN, NKX23, FOSB, RUNX3, and FOSL1, which regulated these hub genes through methylation-dependent enhancer dynamics. Predominant hypomethylation of MRPs and RPs disrupted mitochondrial function, contributed to oxidative phosphorylation (OXPHOS) and metabolic reprogramming, favoring cancer cell survival. The survival analysis validated the clinical relevance of these hub genes, with high-expression cohorts exhibiting poor overall survival (OS) outcomes enlightened their relevance in LUAD pathogenesis and presented the potential for developing novel targeted therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93769-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f19b44fa19b901c527db0c43e3ead70766aaf9b" target='_blank'>
              Unraveling the intricate molecular landscape and potential biomarkers in lung adenocarcinoma through integrative epigenomic and transcriptomic profiling
              </a>
            </td>
          <td>
            Arnab Mukherjee, Manon Boonbangyang, Mukunthan K.S.
          </td>
          <td>2025-03-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is driven by a complex network of resistance mechanisms against standard-of-care therapies, resulting in poor long-term outcomes. This review offers a uniquely comprehensive and integrative perspective on these resistance pathways, systematically examining both androgen receptor (AR)-dependent factors (including AR overexpression, point mutations, glucocorticoid receptor signaling, splice variants, post-translational modifications, altered coregulators, and intratumoral hormone biosynthesis) and AR-independent pathways (such as neuroendocrine differentiation, lineage plasticity, and alternative growth factor signaling). We also highlight resistance mechanisms influencing immunotherapy, chemotherapy, radiopharmaceutical therapy and targeted therapy. By synthesizing emerging insights across these domains, this review not only clarifies the underlying biology of mCRPC resistance but also identifies key leverage points for more effective interventions. Building on this foundation, we propose a forward-looking framework for overcoming mCRPC drug resistance, emphasizing the importance of biomarker-guided patient selection, combination strategies that simultaneously target multiple resistance mechanisms, and novel therapies under investigation. These recommendations are intended to guide future clinical trial designs and research priorities that move beyond incremental improvements. Ultimately, this comprehensive synthesis aims to serve as a resource for clinicians and researchers to accelerate the development of durable, precision-based treatment strategies in mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b50b02beb4327daa61573fe429543e569947b38" target='_blank'>
              Overcoming drug resistance in castrate-resistant prostate cancer: current mechanisms and emerging therapeutic approaches
              </a>
            </td>
          <td>
            Adam Khorasanchi, Feng Hong, Yuanquan Yang, Eric A. Singer, Peng Wang, Mingjia Li, Linghua Zheng, P. Monk, Amir Mortazavi, Lingbin Meng
          </td>
          <td>2025-02-19</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, Carola Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Purpose Endometriosis is a common chronic neuroinflammatory disease with a poorly understood pathogenesis. Molecular changes and specific immune cell infiltration in the eutopic endometrium are critical to disease progression. This study aims to explore immune mechanisms and molecular differences in the proliferative eutopic endometrium of endometriosis by integrating bulk RNA-seq and single-cell RNA sequencing (scRNA-seq) data, and to develop diagnostic and predictive models for the disease. Methods Gene expression profiles from the proliferative endometrium of endometriosis patients and healthy controls were obtained from the Gene Expression Omnibus. Single-cell RNA-seq data were processed using R packages, and cell clusters’ contributions to endometriosis were calculated. Differentially expressed genes (DEGs) from bulk RNA-seq were intersected with significant mesenchymal cell genes from scRNA-seq, and a predictive model was constructed using LASSO analysis. Key gene mechanisms were explored through Gene Set Enrichment and Variation Analyses. miRNA networks and transcriptional regulation analyses were conducted, and potential drugs were predicted using the Connectivity Map database. RT-qPCR validated key gene expression. Results Mesenchymal cells in the proliferative eutopic endometrium were identified as major contributors to endometriosis pathogenesis. LASSO regression identified eight key genes: SYNE2, TXN, NUPR1, CTSK, GSN, MGP, IER2, and CXCL12. The predictive model based on these genes achieved AUC values of 1.00 and 0.8125 in training and validation cohorts. Immune infiltration analysis showed increased CD8+ T cells and monocytes in the eutopic endometrium of endometriosis patients. Drug target prediction indicated that drugs like Retinol, Orantinib, Piperacillin, and NECA were negatively correlated with the expression profiles of endometriosis. RT-qPCR validated gene expression in patients aligned with bioinformatics analysis. Conclusion Significant transcriptomic changes and altered immune cell infiltration in the proliferative eutopic endometrium potentially contribute to endometriosis pathogenesis. Our predictive model based on the key genes demonstrates high diagnostic accuracy, offering insights for diagnosis and potential treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a24e34bf48eebfe993286a3f22d60c5eec8cac" target='_blank'>
              Integration of Single Cell and Bulk RNA-Sequencing Reveals Key Genes and Immune Cell Infiltration to Construct a Predictive Model and Identify Drug Targets in Endometriosis
              </a>
            </td>
          <td>
            Hanke Zhang, Yu-Qing Fang, Dan Luo, Yan-Hui Li
          </td>
          <td>2025-02-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The use of AAV vectors for in vivo gene therapy has demonstrated the potential to permanently correct genetic diseases by delivering functional gene copies to the nuclei of affected tissues. AAV vectors, as tools for in vivo gene delivery, are particularly appealing and have shown safety and long-term efficacy in numerous organ-targeted experiments. Nevertheless, employing AAV vectors for gene therapy in the brain faces a notable hurdle in the shape of immune responses, chiefly instigated by the brain’s resident immune cells, microglia. Additionally, lower levels of AAV vector-neutralizing antibodies have been detected in the cerebrospinal fluid compared to the circulatory system. This research, leveraging transcriptomic and single-cell RNA sequencing (scRNA-seq) data in conjunction with Mendelian randomization analysis, has identified the potential role of the XBP1 protein in mediating B-cell immunosuppression in the brain via the MDK-NCL ligand-receptor pair and associated genes. Furthermore, it paves the way for further investigation into the regulatory factors and pathways within the immune modulation network, as well as their prospective beneficial implications in immunotherapeutic treatments. By employing various innovative approaches, the study seeks to recreate the immune environment generated by AAV in the brain and preliminarily explore the immune suppression mechanisms induced by AAV vectors in the brain. Supplementary Information The online version contains supplementary material available at 10.1007/s12026-025-09609-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fbe09900693cf8126b4ed7a1f742723962eeea9" target='_blank'>
              A comprehensive analysis of scRNA-Seq and RNA-Seq unveils B cell immune suppression in the AAV-loaded brain
              </a>
            </td>
          <td>
            Shunyu Wu, Lu Xue, Xiang Li, Yaoxuan Wang, Yuting Zhu, Yuanbo Luo, Jiayu Sun, Tingting Jin, Wenying Shu, Zhaoyan Wang
          </td>
          <td>2025-03-05</td>
          <td>Immunologic Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9-mediated homology-directed repair (HDR) allows precise gene editing, but its efficiency remains low for certain cell types, such as human induced pluripotent stem cells (hiPSCs). In this study, we aimed to introduce the CTNNA1: c.2023C>T (p.Q675*) genetic alteration, which is associated with Hereditary Diffuse Gastric Cancer, into hiPSCs using CRISPR/Cas9. We designed a single-guide RNA targeting the alteration site and a single-stranded oligonucleotide donor DNA template for HDR-based repair. Herein, we report the successful introduction of the CTNNA1: c.2023C>T homozygous alteration in one hiPSC line, which resulted in severe phenotypic changes, including impaired colony formation and cell proliferation. Additionally, we established a straightforward protocol to assess hiPSCs karyotype integrity, ensuring the chromosomal stability required for the gene-editing process. This protocol involves routine G-banding analysis that is required for regular quality controls during handling of hiPSCs. This study demonstrates an efficient approach to precisely edit hiPSCs by CRISPR/Cas9 and highlights the essential role of CTNNA1 expression in maintaining hiPSC viability. Our methodology provides a valuable framework for modeling disease-associated alterations in human-derived cellular models that can be reproduced for other genes and other types of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b6de2d94791247728d12901bea9b29daa88c32" target='_blank'>
              A protocol for karyotyping and genetic editing of induced pluripotent stem cells with homology-directed repair mediated CRISPR/Cas9
              </a>
            </td>
          <td>
            Silvana Lobo, Rita Barbosa-Matos, Sofia Dória, A. Pedro, Ana Brito, D. Ferreira, Carla Oliveira
          </td>
          <td>2025-02-28</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mapping genetic interactions (GIs) is crucial for understanding genetic network complexity. In this study, we investigate the utility of Cas13d, a CRISPR system targeting RNA, for GI mapping and compare it to Cas9 and Cas12a, two DNA nucleases commonly used for GI mapping. We find that Cas13d induces faster target gene perturbation and generates more uniform cell populations with double perturbations than Cas9 or Cas12a. We then encounter Cas13d gRNA-gRNA interference when concatenating gRNAs targeting different genes into one gRNA array, which we overcome by a dual promoter gRNA expression strategy. Moreover, by concatenating three gRNAs targeting the same gene into one array, we are able to maximize the Cas13d-mediated knockdown effects. Combining these strategies enhances proliferation phenotypes while reducing library size and facilitates reproducible quantification of GIs in oncogenic signaling pathways. Our study highlights the potential of Cas13d for GI mapping, promising advancements in understanding therapeutically relevant drug response pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Lung cancer is one of the most commonly diagnosed cancers. N6-methyladenosine (m6A) modification has a profound impact on RNA translation, splicing, transportation, and stability. Aims This research aimed to identify and verify m6A-modified signatures for Lung adenocarcinoma (LUAD) tumorigenesis. Objective Our previous mRNA-seq and m6A-seq data from 26 pairs of LUAD samples and tumor-adjacent normal tissues are used. Methods Univariate Cox regression analysis and the least absolute shrinkage and selection operator (LASSO) analysis were used to estimate the significance of 37 collected m6A regulators. WGCNA was constructed to identify the genes correlated with LUAD tumorigenesis. Pearson correlation analysis between mRNA-seq and m6A-seq data was used to identify the m6A-correlation genes. Results LASSO-Cox analysis identified 18 m6A significant regulators. The top 3 regulators, including METTL16, FTO, and SRSF10, and their downstream genes which were reported in the literature were analysed to confirm their role in LUAD tumorigenesis. Blue and brown coexpression modules were chosen as key modules for LUAD tumorigenesis. At last, we intersected Lasso-downstream genes, m6A-correlation genes, with blue or brown module genes. As a result, 56 m6A-modified gene signatures were obtained. Among them, AKAP9, PLXNB2, BRPF3, HPS4, EXOC7, and KLF6 have an inconsistent expression in protein and mRNA levels, probably due to m6A modification. In addition, these genes may be involved in regulating drug resistance. Conclusions 56 m6A-modified gene signatures for LUAD tumorigenesis were obtained from Pearson correlation analysis between mRNA-seq and m6A-seq data, along with LASSO and WGCNA analysis. Among them, AKAP9, PLXNB2, BRPF3, HPS4, EXOC7 and KLF6 play a crucial role in LUAD tumorigenesis in an m6A modification-dependent manner. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02106-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a1734ebf3b7cc16ea04d4fbcf6ec4bc799a0c4" target='_blank'>
              Identification of m6A-modified gene signatures in lung adenocarcinoma tumorigenesis and their potential role in drug resistance
              </a>
            </td>
          <td>
            Xiaomin Han, Qiang Ma, Ruyi Chang, Siyuan Xin, Guojun Zhang, Ruilong Wang, Yukun Wang
          </td>
          <td>2025-03-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654c576507b39c0969a5f79ea0daae0b983731a4" target='_blank'>
              Riding the Wave of Progress: Examining the Current Landscape and Future Potential of MicroRNAs in Cancer Gene Therapy.
              </a>
            </td>
          <td>
            Saleem Ahmad, Hanif Khan, Njoka Irene Muthoni, Mohammed Alissa, Awaji Y. Safhi, F. Y. Sabei, Osama Abdulaziz, Khadijah Hassan Khan, Essam H Ibrahim, S. Khan
          </td>
          <td>2025-03-24</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Studying the functional consequences of structural variants (SVs) in mammalian genomes is challenging because (i) SVs arise much less commonly than single-nucleotide variants or small indels and (ii) methods to generate, map, and characterize SVs in model systems are underdeveloped. To address these challenges, we developed Genome-Shuffle-seq, a method that enables the multiplex generation and mapping of thousands of SVs (deletions, inversions, translocations, and extrachromosomal circles) throughout mammalian genomes. We also demonstrate the co-capture of SV identity with single-cell transcriptomes, facilitating the measurement of SV impact on gene expression. We anticipate that Genome-Shuffle-seq will be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and three-dimensional nuclear architecture, while also initiating a path toward a minimal mammalian genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd57d0a97983513ebb6d083a8255399d3304d8d" target='_blank'>
              Multiplex generation and single-cell analysis of structural variants in mammalian genomes
              </a>
            </td>
          <td>
            Sudarshan Pinglay, Jean-Benoît Lalanne, R. Daza, Sanjay Kottapalli, Faaiz Quaisar, Jonas Koeppel, Riddhiman K. Garge, Xiaoyi Li, David S Lee, Jay Shendure
          </td>
          <td>2025-01-31</td>
          <td>Science (New York, N.Y.)</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Synthetic lethality exploits the genetic vulnerabilities of cancer cells to enable a targeted, precision approach to treat cancer1. Over the past 15 years, synthetic lethal cancer target discovery approaches have led to clinical successes of PARP inhibitors2 and ushered several next-generation therapeutic targets such as WRN3, USP14, PKMYT15, POLQ6 and PRMT57 into the clinic. Here we identify, in human cancer, a novel synthetic lethal interaction between the PELO–HBS1L and SKI complexes of the mRNA quality control pathway. In distinct genetic contexts, including 9p21.3-deleted and high microsatellite instability (MSI-H) tumours, we found that phenotypically destabilized SKI complex leads to dependence on the PELO–HBS1L ribosomal rescue complex. PELO–HBS1L and SKI complex synthetic lethality alters the normal cell cycle and drives the unfolded protein response through the activation of IRE1, as well as robust tumour growth inhibition. Our results indicate that PELO and HBS1L represent novel therapeutic targets whose dependence converges upon SKI complex destabilization, a common phenotypic biomarker in diverse genetic contexts representing a significant population of patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3833ef2c0218061364fcad4c0598f346a7f4c6cb" target='_blank'>
              Synthetic lethality of mRNA quality control complexes in cancer
              </a>
            </td>
          <td>
            Vivian Prindle, Adam E Richardson, Kimberly R Sher, Sarah Kongpachith, Kaitlin Kentala, Sakina M. Petiwala, Dong Cheng, Deborah Widomski, Phuong Le, Maricel Torrent, Anlu Chen, Stephen Walker, Marianne B. Palczewski, Diya Mitra, V. Manaves, Xu Shi, Charles Lu, Stephanie Sandoval, Zoltan Dezso, F. Buchanan, Daniel Verduzco, Brian Bierie, Jonathan A Meulbroek, W. Pappano, Joshua P Plotnik
          </td>
          <td>2025-02-05</td>
          <td>Nature</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the considerable effort to characterize the genomic landscape of chronic lymphocytic leukemia (CLL), published data have been almost exclusively derived from patients of European Ancestry (EA), with significant underrepresentation of minorities, including patients of African Ancestry (AA). To begin to address this gap, we evaluated whether differences exist in the genetic and transcriptomic features of 157 AA and 440 EA individuals diagnosed with CLL. We sequenced 59 putative driver genes and found an increased frequency of high-impact mutations in AA CLL, including genes of the DNA damage repair (DDR) pathway. Telomere erosion was also increased in AA CLL, amplifying the notion of increased genomic instability in AA CLL. Furthermore, we found transcription enrichment of the Tumor Necrosis Factor-alpha (TNFα) Signaling via NF-κB pathway in AA CLL compared to EA CLL, suggesting that tumor promoting inflammation plays an important role in AA CLL. In summary, these results suggest that genomic instability and NF-kB activation is more prevalent in AA CLL than EA CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c673a6c02d35f0c2838ab5179fa8927b41b41fc" target='_blank'>
              Genomic characterization of chronic lymphocytic leukemia in patients of African ancestry
              </a>
            </td>
          <td>
            Cecilia Bonolo de Campos, Chantal E McCabe, L. Bruins, Daniel R. O’Brien, Sochilt Brown, R. Tschumper, C. Allmer, Y. Zhu, K. Rabe, S. Parikh, Neil E. Kay, Huihuang Yan, J. Cerhan, John N Allan, Richard R Furman, J. B. Weinberg, Danielle M Brander, D. Jelinek, M. Chesi, S. Slager, Esteban Braggio
          </td>
          <td>2025-02-06</td>
          <td>Blood Cancer Journal</td>
          <td>1</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006d2b04fc1884e4bfd784cf684ee4d760f680ad" target='_blank'>
              Exploring the tumor suppressor role of RIN1 in familial thyroid carcinoma.
              </a>
            </td>
          <td>
            Luna Picello, Mattia Dalle Nogare, Daniele Puggina, Cecilia Salvoro, G. Pennelli, G. Gaudenzi, S. Carra, Monica Oldani, Davide Gentilini, Laura Fugazzola, A. Faggiano, Giovanni Vitale, Gianluca Occhi, Giovanni Vazza
          </td>
          <td>2025-03-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematologic malignancy characterized by the proliferation of abnormal clonal plasma cells in the bone marrow. The heterogeneity in Chinese MM populations remains underexplored. Methods We conducted whole-exome sequencing (WES) on 241 tumor samples, complemented by RNA sequencing (RNA-seq) on 131 samples from 212 Chinese MM patients. Results We identified a novel mutational signature and analyzed molecular differences between newly diagnosed MM (NDMM) and relapsed/refractory MM (RRMM) patients. NFKBIA mutations were notably more frequent in NDMM patients compared to the MMRF-COMMPASS cohort (4/50 vs 22/937, p = 0.048), with additional recurrent mutations in several genes like TTN, IGLL5 and SYNE1. In RRMM patients, UBR5 mutations were more prevalent (4/24 vs 0/50, p = 0.01), alongside frequent mutations in OBSCN, CACNA1H, and HSPG2. Clonal evolution was assessed through multiple time points and locations, identifying genes potentially linked to circulating plasma cell formation. Cox regression analysis revealed that age and mutations in OBSCN and RB1 were significant predictors of progression-free survival (PFS) in NDMM patients. Additionally, albumin, β2-microglobulin, and RB1 mutations were correlated with overall survival (OS). Conclusions In summary, we characterized the genomic landscape of MM in diverse Chinese populations, confirmed clonal evolution, and identified prognostic genes. Supplementary Information The online version contains supplementary material available at 10.1186/s12920-025-02116-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a64b8c5384425f3201b477d667c659c3322888" target='_blank'>
              Genomic characteristics and prognostic correlations in Chinese multiple myeloma patients
              </a>
            </td>
          <td>
            Xi Chen, Tiancheng Luo, Wenhui Zhang, Sheng Wang, Mengxuan Zhu, Haiyan He, Jin Liu, Jing Lu, W. Qiang, Yanchun Jia, Nan Hou, Xuenan Zhao, Shan Zhang, Jing Li, Juan Du
          </td>
          <td>2025-03-14</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a common malignancy with high mortality. Despite advancements in understanding its molecular causes and improved drug therapies, patient survival rates remain low. The main reasons for the high mortality rate are cancer metastasis and the emergence of drug-resistant cancer cell populations. While genetic changes are recognized as the main driver of CRC occurrence and progression, recent studies suggest that epigenetic regulation is a crucial marker in cancer, influencing the interplay between genetics and the environment. Research has shown the significant regulatory roles of non-coding RNAs (ncRNAs) in CRC development. This review explores epigenetically regulated ncRNAs and their functions, aiming to understand key regulatory mechanisms that impact CRC development. Additionally, it discusses the potential use of these ncRNAs in CRC diagnosis, prognosis, and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c88c2a819f1e9676a6d273b84bfd94da9274e4c" target='_blank'>
              Epigenetically Regulating Non-coding RNAs in Colorectal Cancer: Promises and Potentials
              </a>
            </td>
          <td>
            Zahra Taheri, Majid Zaki-Dizaji
          </td>
          <td>2025-01-31</td>
          <td>Middle East Journal of Digestive Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Acute lymphoblastic leukemia (ALL) is the most common childhood cancer, comprising approximately 25% of pediatric malignancies. Notably, chromosomal aberrations and genetic alterations play a central role in the pathogenesis of ALL, serving as critical diagnostic and prognostic markers. In this study, we use array-based comparative genomic hybridization (array-CGH) to explore the landscape of copy number variations (CNVs) and variants of uncertain significance (VUS) in 67 Malaysian childhood ALL patients with normal karyotype. Results A total of 36 pathogenic CNVs (26 gains, 10 losses) were identified in 19 (28.4%) patients which harbor genes related to the development of ALL. The genes include the MLLT3 (9p21.3), ETV6 (12p13.2), RUNX1 (21q22.12), ERG (21q22.2) and DMD (Xp21.1). On the other hand, a total of 46 variants of uncertain significance (VUS) was observed in 34 (50.7%) patients. Conclusions Our study indicates that array-CGH is able to identify and characterize the CNVs responsible for the pathogenesis of childhood ALL. However, further studies are required to determine the pathogenic implications of VUS in the development of childhood ALL. Supplementary Information The online version contains supplementary material available at 10.1186/s13039-025-00709-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4d7cd5e5ce30655ab47d932b22f9c51cbf343c" target='_blank'>
              Genomic landscape of childhood acute lymphoblastic leukemia in Malaysia: insights from array-CGH
              </a>
            </td>
          <td>
            Azli Ismail, Fadly Ahid, Wong Nyuk Moi, N. Kamaluddin, E. Esa, Y. M. Yusoff, Z. A. Seman, Muhammad Asyraff Mohammed, Elizabeth George, Asmida Isa, Z. Zakaria
          </td>
          <td>2025-03-28</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Pediatric leukemia is one of the most prevalent childhood malignancies, and its pathogenesis involves complex molecular and genetic mechanisms. Among the critical players in leukemogenesis, the Janus kinase 2 (JAK2) gene has garnered significant attention due to its role in aberrant signal transduction and hematopoietic dysregulation. Mutations such as JAK2 V617F, while more commonly associated with myeloproliferative neoplasms, have also been implicated in pediatric leukemia subtypes, including acute lymphoblastic leukemia (ALL) and acute myeloid leukemia (AML). These mutations result in constitutive activation of the JAK-STAT pathway, promoting unchecked cell proliferation, survival, and resistance to apoptosis. The dysregulation of the JAK2 signaling pathway not only drives malignant transformation but also facilitates interactions with the bone marrow microenvironment, creating a niche that supports leukemia cell survival and immune evasion. Therapeutic advancements have focused on JAK2 inhibitors such as ruxolitinib and fedratinib, which show promise in preclinical and early clinical settings. However, challenges such as drug resistance and off-target effects limit their efficacy, necessitating the exploration of combination therapies and novel drug formulations. Current strategies include combining JAK2 inhibitors with chemotherapy, immune checkpoint inhibitors, or epigenetic modulators to achieve synergistic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8c5bd5328b2c81b8552a0d00878e9d6a8e64d6" target='_blank'>
              JAK2 in pediatric leukemia: mechanisms of pathogenesis and drug development- a narrative review
              </a>
            </td>
          <td>
            E. Obeagu
          </td>
          <td>2025-03-18</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ab64f4c6c0ffff56fec877ba1f8433465976" target='_blank'>
              Single-cell RNA sequencing revealed changes in the tumor microenvironment induced by radiotherapy for cervical cancer and the molecular mechanism of mast cells in immunosuppression.
              </a>
            </td>
          <td>
            L. Xue, Linzhi Gao, Shimin Zhou, Chaofan Yan, Xian Zhang, Wei Lin, Hu Li, Yuan Shen, Xiaoyu Wang
          </td>
          <td>2025-03-14</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231338d317a31d1b141aef89e6f810d3087be9ba" target='_blank'>
              Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation
              </a>
            </td>
          <td>
            Gaurav Agarwal, Mateusz Antoszewski, Xueqin Xie, Y. Pershad, U. Arora, Chi-Lam Poon, Peng Lyu, Andrew J. Lee, , Tianyi Ye, Laila Barakat Norford, Anna-Lena Neehus, Lucrezia della Volpe, Lara Wahlster, Diyanath Ranasinghe, Tzu-Chieh Ho, T. Barlowe, Arthur Chow, A. Schurer, James Taggart, Benjamin H. Durham, Omar Abdel-Wahab, Kathy L. McGraw, , Ruslan Soldatov, Alexander G. Bick, M. Kharas, V. G. Sankaran
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transposable elements (TEs) are reactivated in tumors and serve as significant contributors to tumor transcriptomic complexity and heterogeneity. However, their repetitive nature and diverse transcriptional forms have hindered efforts to characterize TE-derived transcripts independent of host gene contexts. Here, using our self-developed splicing-junction analysis tool ASJA, we systematically interrogated TE transcription across 32 cancer types, identifying transcripts autonomously initiated from and exclusively spliced among TEs with high TE content, termed TEtrans. Our analysis revealed 5,361 TEtrans exhibiting pan-cancer prevalence, tumor-specific expression and features of mature transcripts, including canonical splicing, 5’ caps and polyA tails. Mostly unannotated and enriched in intergenic regions, TEtrans are predominantly derived from primate-specific classes with conserved splice pattern. TEtrans burden demonstrates heterogeneous associations with prognosis and immune activity across cancers, while their expression can be epigenetically dual-regulated by stemness- and inflammation-associated transcription factors. Functional studies uncovered that TEtrans could act as oncogenes, exemplified by tsTE1, a HERVH-derived transcript that promotes colorectal cancer proliferation by enhancing TOP1-mediated DNA supercoil relaxation. Remarkably, ∼7.2% of TEtrans are identified to encode tumor neoantigens, including viral proteins and unannotated peptides, which are shared among patients and validated by proteogenomic analysis. These TEtrans-derived neoantigens are immunogenic both in vitro and in vivo, and exceed neoepitopes from genomic alterations in abundance per tumor, particularly in cancers with low mutational burden. Collectively, as a gene-independent form of TE-derived transcripts, TEtrans represents a unique source of oncogenes and tumor neoantigens, expanding the functional repertoire of TEs in cancer biology and offering new avenues for therapeutic targeting. Highlights TEtrans are tumor-specific transcripts with conserved splicing patterns, predominantly derived from primate TEs in intergenic regions. TEtrans expression is epigenetically activated and dually regulated by stemness/inflammation transcription factors. TEtrans function as oncogenes, exemplified by tsTE1 enhancing TOP1-mediated DNA relaxation. TEtrans encode tumor neoantigens that elicit CD8+ T cell responses, providing targets for immunotherapy, especially in low-mutation tumors. Abstract figure">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90153db4362d7e931500fbc74364483bb05472a" target='_blank'>
              Transcriptomic splicing analysis reveals a gene-independent form of TE reactivation in tumors representing unrecognized source of oncogenes and neoantigens
              </a>
            </td>
          <td>
            Ziteng Li, Yichao Bao, Qin Li, Hongwu Yu, He-Yun Zhang, Yifan Wen, Yu Yang, Jingjing Zhao, Peng Lin, Yan Li, Zhixiang Hu, Xichun Hu, Xiaodong Zhu, Shenglin Huang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Colibactin is a genotoxic secondary metabolite produced by certain Enterobacteriaceae strains that populate the intestine and produces a specific mutational signature in human colonocytes. However, the host pathways involved in colibactin response remain unclear. To address this gap, we performed genome-wide CRISPR/Cas9 knockout screens and RNA sequencing utilizing live pks+ bacteria and a synthetic colibactin analog. We identified 20 enriched genes with a MAGeCK score of >2.0 in both screens, including proteasomal subunits (e.g., PSMG4 and PSMD4), RNA processing factors (e.g., SF1 and PRPF8), and RNA polymerase III (e.g., CRCP), and validated the role of PSMD4 in colibactin sensitization. PSMD4 knockout in HEK293T and HT-29 cells promoted cell viability and ameliorated G2-M cell cycle arrest but did not affect the amount of phosphorylated H2AX foci after exposure to synthetic colibactin 742. Consistent with these observations, PSMD4−/− cells had a significantly higher colony formation rate and bigger colony size than control cells after 742 exposure. These findings suggest that PSMD4 regulates cell cycle arrest following colibactin-induced DNA damage and that cells with PSMD4 deficiency may continue to replicate despite DNA damage, potentially increasing the risk of malignant transformation. IMPORTANCE Colibactin has been implicated as a causative agent of colorectal cancer. However, colibactin-producing bacteria are also present in many healthy individuals, leading to the hypothesis that some aspects of colibactin regulation or host response dictate the molecule’s carcinogenic potential. Elucidating the host-response pathways involved in dictating cell fate after colibactin intoxication has been difficult, partially due to an inability to isolate the molecule. This study provides the first high-throughput CRISPR/Cas9 screening to identify genes conferring colibactin sensitivity. Here, we utilize both bacterial infection and a synthetic colibactin analog to identify genes directly involved in colibactin response. These findings provide insight into how differences in gene expression may render certain individuals more vulnerable to colibactin-initiated tumor formation after DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c10ad4bc58eb16a2a76999a8e88cc374bfe8d0" target='_blank'>
              Genome-scale CRISPR/Cas9 screening reveals the role of PSMD4 in colibactin-mediated cell cycle arrest
              </a>
            </td>
          <td>
            Michael W. Dougherty, Ryan M. Hoffmann, Maria C. Hernandez, Yougant Airan, R. Gharaibeh, Seth B. Herzon, Ye Yang, Christian Jobin
          </td>
          <td>2025-02-07</td>
          <td>mSphere</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2 + tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One sentence summary: Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00341-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279ce0f97c7d66cc42d068594f84880c4fe69c7b" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2025-02-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Simple Summary Nilotinib and other tyrosine kinase inhibitors (TKIs) target the BCR-ABL1 oncoprotein in Chronic Myeloid Leukemia (CML), reducing the growth of leukemic stem cells (LSCs) and promoting their death. However, some LSCs persist in the bone marrow (BM), which may lead to relapse and disease progression. We analyzed the gene expression profiling of BM CD34+/lin− cells from 79 Chronic-Phase CML patients at diagnosis and after 12 months of nilotinib treatment, comparing them to the healthy cells from 10 donors (CTRLs). Our study identified key genes involved in eight crucial biological pathways: CML, cell cycle, JAK-STAT, PI3K-Akt, MAPK, Ras, NF-kB, and ABC transporters. We observed a progressive down-regulation of the expression levels of several genes across these pathways from diagnosis to 12 months of nilotinib treatment and to CTRLs. This could suggest that nilotinib effects create selective pressure, potentially supporting the survival and self-renewal of LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d3a05e9350f9c4633e6a38d029b612b50597a4" target='_blank'>
              Bone Marrow CD34+/lin− Cells of Patients with Chronic-Phase Chronic Myeloid Leukemia (CP-CML) After 12 Months of Nilotinib Treatment Exhibit a Different Gene Expression Signature Compared to the Diagnosis and the Corresponding Cells from Healthy Subjects
              </a>
            </td>
          <td>
            Alessandra Trojani, E. Pungolino, Barbara Di Camillo, L. E. Bossi, Cassandra Palumbo, M. D’adda, A. Perego, Mauro Turrini, C. Elena, L. Borin, A. Iurlo, Simona Malato, Francesco Spina, M. L. Latargia, P. Spedini, Salvatore Artale, M. Anghilieri, M. Carraro, C. Bucelli, A. Beghini, R. Cairoli
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent malignant tumor characterized by high global incidence and mortality rates. Furthermore, it is imperative to comprehend the molecular mechanisms underlying its development and to identify effective prognostic markers. These efforts are crucial for pinpointing potential therapeutic targets and enhancing patient survival rates. Therefore, we develop a novel prognostic model aimed at providing new theoretical support for clinical prognosis evaluation and treatment. We downloaded data from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases. Subsequently, we performed single-cell analysis and developed a prognostic model associated with colorectal cancer. We divided the scRNA-seq dataset (GSE221575) into 19 cell clusters and classified these clusters into 11 distinct cell types using marker genes. Using univariate Cox regression and LASSO (Least Absolute Shrinkage and Selection Operator) analyses, we developed a prognostic model consisting of 9 genes. Based on our 9-gene model, we divided patients into high-risk and low-risk groups using the median risk score. The high-risk group demonstrated significant positive correlations with M0 macrophages, CD8+ T cells, and M2 macrophages. The enrichment analyses indicate significant enrichment of immune-related pathways in the high-risk group, including HEDGEHOG_SIGNALING, Wnt signaling pathway, and cell adhesion molecules. Drug sensitivity analysis revealed that the low-risk group was sensitive to 5 chemotherapeutic drugs, while the high-risk group was sensitive to only 1. Additionally, we developed a highly reliable nomogram for clinical application. This suggests that the risk score derived from our modeling analysis is highly effective for stratifying colorectal cancer samples. This study comprehensively applied bioinformatics methods to construct a risk score model. The model showed good predictive performance, offering potential guidance for individualized treatment of colorectal cancer patients. Furthermore, it may provide valuable insights into the disease’s pathogenesis and identify potential therapeutic targets for further research. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-91761-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ee3ba4be7a7e3b5e63b304d989c55900905b7" target='_blank'>
              Integrated analysis of single-cell and bulk RNA-sequencing to predict prognosis and therapeutic response for colorectal cancer
              </a>
            </td>
          <td>
            Liyang Cai, Xin Guo, Yucheng Zhang, Huajie Xie, Yongfeng Liu, Jianlong Zhou, Huolun Feng, Jiabin Zheng, Yong Li
          </td>
          <td>2025-03-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive and heterogeneous subtype, representing 15% of lung cancer cases. Although SCLC initially responds to etoposide and platinum (EP) chemotherapy, nearly all patients relapse with resistant tumors. While recent advances in immunotherapy have shown promise, only 10–20% of patients benefit, and effective stratification methods are lacking. The mechanisms of resistance to both therapeutics remain obscure. In this study, we aimed to gain insights into those leveraging a recent surge in the field of SCLC genomics. We constructed a regulatory network for SCLC and identified granulin precursor (GRN) as a hub of EP response associated genes. GRN-low patients showed improved survival with chemotherapy, while GRN-high patients exhibited resistance. GRN overexpression in SCLC cells conferred resistance to EP treatment and suppressed neuroendocrine features. GRN and its associated genes were linked to cancer cell intrinsic immunogenicity, and single-cell RNA-seq data revealed that GRN expression is particularly high in subsets of tumor-associated macrophages. In concordance with these findings, GRN-low tumors showed significantly better survival with chemo-immunotherapy, while GRN-high tumors did not benefit from additional immunotherapy. GRN-high tumors, associated with non-neuroendocrine (non-NE) subtypes, had a higher level of macrophage infiltration, potentially contributing to immunotherapy resistance. These results highlight GRN as a critical regulator of chemo-resistance and a potential biomarker for immunotherapy resistance in SCLC. Targeted therapeutic strategies for GRN-low patients could improve outcomes, while new approaches are needed for GRN-high patients. Overall, our findings implicate GRN as a bridge between chemotherapy and immunotherapy resistance through GRN-mediated mechanisms. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01667-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b664fbb9ca2f0d479f54b9221d29d13ae0d1a6" target='_blank'>
              Transcriptional regulatory network analysis identifies GRN as a key regulator bridging chemotherapy and immunotherapy response in small cell lung cancer
              </a>
            </td>
          <td>
            S. Yoo, Ayushi S. Patel, S. Karam, Yi Zhong, Li Wang, Feng Jiang, Ranran Kong, Sharon Bikvan, Wenhui Wang, A. Sinha, Charles A. Powell, Jun Zhu, Hideo Watanabe
          </td>
          <td>2025-02-05</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Migrasomes, recently discovered cellular substructures, may play a crucial role in cancer progression, treatment response, and prognosis. However, the prognostic value of migrasome-associated long non-coding RNAs (lncRNAs) in colon adenocarcinoma (COAD) remains unexplored. Methods RNA-seq data from 459 COAD patients, including clinical characteristics and outcome information, were obtained from The Cancer Genome Atlas. A risk model was constructed through co-expression analysis of migrasome genes and lncRNAs, followed by Cox regression and least absolute shrinkage and selection operator analysis to identify prognostic lncRNAs. Functional enrichment analyses were performed to elucidate underlying biological mechanisms. Immune landscape characterization utilized ESTIMATE, CIBERSORT, Tumor Immune Estimation Resource (TIME), and single-sample Gene Set Enrichment Analysis (ssGSEA). Drug sensitivity analysis was conducted for select therapeutic agents. Results Nine prognostic lncRNAs (AC010463.3, AL590483.4, AP005264.1, ZEB1-AS1, AC104088.1, PRKAR1B-AS2, AC009315.1, SUCLG2-AS1, and AC006111.2) were identified and incorporated into a risk model. Low-risk patients demonstrated significantly improved survival outcomes. The model exhibited independent prognostic capability, with AUCs of 0.783, 0.749, and 0.713 for one-, three-, and five-year survival, respectively, in the training cohort. High-risk patients displayed reduced overall survival and elevated tumor mutation burden. Additionally, these patients showed decreased sensitivity to therapeutic agents, including Oxaliplatin, Irinotecan, and 5-Fluorouracil. Conclusion Our novel migrasome-associated lncRNA signature demonstrates robust predictive capacity for both prognosis and chemotherapeutic sensitivity in COAD, potentially facilitating personalized treatment strategies and improved patient management. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02197-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081cc01a6847cf137157802eb2a50ffa335c8848" target='_blank'>
              Comprehensive identification of a migrasomes-associated long non-coding RNA signature to predict the prognosis and treatment options in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhen Zheng, Hui Liu, Quan Xu, Wei Cui, Kaitai Liu
          </td>
          <td>2025-03-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6ca066f0de020212e2771b80c63997af778abe" target='_blank'>
              Abstract A044: Immune profiling tools for adult and pediatric cancer: From whole genome and transcriptome sequencing to immunotherapy clinical trial design
              </a>
            </td>
          <td>
            Yaoqing Shen, Chelsea Mayoh, Kathleen Wee, E. Pleasance, E. Titmuss, R. Corbett, L. Williamson, Zakhar Krekhno, A. Nabbi, Raoul Santiago, Stéphanie Bianco, Scott Davidson, A. Shlien, K. Yuki, Denise Connolly, Daniel A. Morgenstern, M. Bonakdar, Greg Taylor, V. Csizmok, C. Grisdale, Melissa McConechy, Jing Xu, John H. Dupuis, K. Mungall, E. Chuah, A. Mungall, J. Nelson, Stephen Yip, Sophie Sun, Howard Lim, D. Renouf, J. Laskin, S. Cohen-Gogo, S. Rassekh, R. Deyell, M. Marra, P. Ekert, S. Jones
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The second and third most frequently diagnosed cancers worldwide are breast (2.3 million new cases) and colorectal (1.9 million new cases), respectively. Although advances in cancer therapies and early detection have improved the overall survival of patients, patients still develop resistance or cancer recurrence. Thus, the development of novel therapies that can affect multiple mechanisms of drug resistance and cell survival is ideal for the treatment of advanced and metastatic cancers. CHD1L is a novel oncogenic protein involved in regulating chromatin remodeling, DNA damage repair, epithelial-mesenchymal transition (EMT), and programmed cell death via PARthanatos. Herein, we assess in real-time how the CHD1L inhibitor (CHD1Li) OTI-611 modulates cell cycle progression in Colo678, SUM149PT, and SW620 cell lines. By utilizing a cell cycle reporter, we tracked the real-time cell cycle progression of cancer cells treated with OTI-611 alone and in combination with standard-of-care (SOC) therapies. Our results indicate that OTI-611 causes G1 phase cell cycle arrest through a CHD1L-mediated mechanism that regulates Cyclin D1 expression and localization. As a result of this mechanism, OTI-611 can reprogram the cell cycle effects of other antitumor agents to modulate and arrest cells in G1 when used in combination, including agents commonly known to arrest cells in the G2/M phase. Therefore, we conclude that OTI-611-induced G1 arrest represents a critical component of its unique mechanism of action, contributing significantly to its anticancer activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775d92956dd4575de627e0b27e219c4a04b6d817" target='_blank'>
              Targeted Inhibition of CHD1L by OTI-611 Reprograms Chemotherapy and Targeted Therapy-Induced Cell Cycle Arrest and Suppresses Proliferation to Produce Synergistic Antitumor Effects in Breast and Colorectal Cancer
              </a>
            </td>
          <td>
            Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-02-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Although GBM’s immunosuppressive environment is well known, the tumor’s resistance to CD8+ T cell killing is not fully understood. Our previous study identified Checkpoint Kinase 2 (Chek2) as the key driver of CD8+ T cell resistance in mouse glioma through an in vivo CRISPR screen and demonstrated that Chk2 inhibition, combined with PD-1/PD-L1 blockade, significantly enhanced CD8+ T cell-mediated tumor killing and improved survival in preclinical model. Here, we aimed to elucidate the immunosuppressive function of Chek2. Immunoprecipitation (IP) followed by mass spectrometry (MS) and phosphoproteomics identified an association between Chek2 with the DNA/RNA-binding proteins YBX1 and YBX3 that are implicated in transcriptional repression of pro-inflammatory genes. Single-gene knock-out and overexpression studies of CHEK2, YBX1, and YBX3 in multiple glioma cell lines revealed that these proteins positively regulate each other’s expression. RNA sequencing coupled with chromatin immunoprecipitation-sequencing (ChIP-seq) analysis demonstrated common inflammatory genes repressed by CHK2-YBX1&YBX3 hub. Targeting one of the hub proteins, YBX1, with the YBX1 inhibitor SU056 led to degradation of CHK2-YBX1&YBX3 hub. Targeting of this hub by SU056 led to enhanced antigen presentation and antigen specific CD8+ T cell proliferation. Further, combination of SU056 with ICB significantly improved survival in multiple glioma models. Collectively, these findings reveal an immunosuppressive mechanism mediated by the CHK2-YBX1&YBX3 hub proteins. Therefore, CHK2-YBX1&YBX3 hub targeting in combination with immune checkpoint blockade therapies in gliomas is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5eee0adfec60774c8a1b6ada43414f89d5d5752" target='_blank'>
              Targeting CHEK2-YBX1&YBX3 regulatory hub to potentiate immune checkpoint blockade response in gliomas
              </a>
            </td>
          <td>
            Heba Ali, Ningjia Zhou, Li Chen, Levi van Hijfte, Vivekanudeep Karri, Yalu Zhou, K. Habashy, Victor A. Arrieta, Kwang-Soo Kim, J. Duffy, Ragini Yeeravalli, D. Tiek, Xiao Song, Snehasis Mishra, Catalina Lee-Chang, Atique U. Ahmed, Dieter H. Heiland, A. Sonabend, C. Dmello
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Background In B‐cell precursor acute lymphoblastic leukemia (BCP‐ALL), chromosomal translocations are strongly associated with prognoses. RNA sequencing (RNA‐seq) is a powerful technology that reveals a close correlation between types of translocation and patterns of gene expression in clinical samples of BCP‐ALL. Cancer cell lines are powerful research tools, and thus, we built a larger series of BCP‐ALL cell lines and performed RNA‐seq analysis to confirm their utility as a model system. Methods We performed RNA‐seq in a total of 94 BCP‐ALL cell lines, including 80 cell lines with 8 representative types of translocations. Results In the UMAP visualization, a close association was confirmed between the types of fusion genes and patterns of gene expression. In the cluster analysis of the gene expression profile, each type of fusion gene showed a clear association with the expression profile in the top 51 variable genes. Of clinical importance, the majority of the top variable genes in the BCP‐ALL cell lines also showed a significant association with the types of fusion genes in the clinical samples. When an association of 125 cell cycle‐related genes with the percentage of S and G2/M phases in 67 cell lines was evaluated, a significant positive correlation with cell cycle progression was confirmed in 10 cell cycle‐related genes (HDAC2, CDC23, YWHAG, MAD2L1, CCNH, ANAPC7, CDC6, ANAPC5, ORC3, andRBX1). Moreover, significant upregulation and downregulation of 40 and 10 genes, respectively, were observed in the cell lines established at relapse compared with those established at diagnosis. Four (SP6, CCNE1, HIST1H2BH, and DECR2) and two (EVI2B and SYN1) of these genes were also significantly higher and lower, respectively, in the clinical samples at relapse than in those at diagnosis. Conclusion Large series of BCP‐ALL cell lines is a powerful research tool for studying the mechanisms of leukemogenesis and the disease progression of BCP‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c14bf4771e63afa160dc9961de41a9da67d0a47d" target='_blank'>
              Utility of a Large Series of B‐Cell Precursor Acute Lymphoblastic Leukemia Cell Lines as a Model System
              </a>
            </td>
          <td>
            Minori Tamai, Chiaki Komatsu, K. Kagami, S. Kasai, K. Akahane, K. Goi, Kanji Sugita, C. Tomoyasu, Toshihiko Imamura, Hiroaki Goto, Takeshi Inukai
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="Background Established genetic biomarkers in chronic lymphocytic leukemia (CLL) have been useful in predicting response to chemoimmunotherapy but are less predictive of response to targeted therapies. With several such targeted therapies now approved for CLL, identifying novel, non-genetic predictive biomarkers of response may help to select the optimal therapy for individual patients. Methods We coupled data from a functional precision medicine technique called BH3-profiling, which assesses cellular cytochrome c loss levels as indicators for survival dependence on anti-apoptotic proteins, with multi-omics data consisting of targeted and whole-exome sequencing, genome-wide DNA methylation profiles, RNA-sequencing, protein and functional analyses, to identify biomarkers for treatment response in CLL patients. Results We initially studied 73 CLL patients from a discovery cohort. We found that greater dependence on the anti-apoptotic BCL-2 protein was associated with prognostically favorable genetic biomarkers. Furthermore, BCL-2 dependence was strongly associated with gene expression patterns and signaling pathways that suggest a more targeted drug-sensitive milieu and was predictive of drug responses. We subsequently demonstrated that these associations were causal in cell lines and additional CLL patient samples. To validate the findings from our discovery cohort and in vitro studies, we utilized primary CLL cells from 54 additional patients treated on a prospective, phase-2 clinical trial of the BTK inhibitor ibrutinib given in combination with chemoimmunotherapy (fludarabine, cyclophosphamide, rituximab) and confirmed in this independent dataset that higher BCL-2 dependence predicted favorable clinical response, independent of the genetic background of the CLL cells. Conclusion We comprehensively defined BCL-2 dependence as a potential functional and predictive biomarker of treatment response in CLL, underscoring the importance of characterizing apoptotic signaling in CLL to stratify patients beyond genetic markers and identifying novel combinations to exploit BCL-2 dependence therapeutically. Our approach has the potential to help optimize targeted therapy combinations for CLL patients. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02260-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f993cc37705c650add4d10eacea4b3e32cdfd64" target='_blank'>
              BCL-2 dependence is a favorable predictive marker of response to therapy for chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            S. Chong, Junyan Lu, Rebecca Valentin, Timothy Lehmberg, J. Eu, Jing Wang, Fengfang Zhu, L. Kong, Stacey M. Fernandes, Jeremy Zhang, Charles Herbaux, Boon Cher Goh, Jennifer R. Brown, Carsten U Niemann, W. Huber, Thorsten Zenz, M. Davids
          </td>
          <td>2025-03-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Objectives/Goals: This study aims to explore transcriptional adaptation, where mutations in one gene trigger compensatory changes in related genes, and how this affects the variability in clinical manifestations of ALS. Our findings will provide insights for therapeutic strategies, while we also use gene editing to investigate correcting variants in ALS patients. Methods/Study Population: The presence of a premature termination codon triggers transcriptional activation. Therefore, we utilized CRISPR-Cas9 tool to generate a premature termination codon in CHCHD10 gene in multiple types of cells, including induced pluripotent stem cells derived from patient samples with known CHCHD10 mutations causative for Amyotrophic lateral sclerosis. CRISPR-Cas9 tool was delivered via ribonucleoprotein electroporation and transfected cell’s DNA was sequenced to validate gene editing. To confirm transcriptional adaption, changes in levels of protein and gene expression will be measured via immunoblot and quantification of CHCHD10 and CHCHCD2 from whole cell lysates of the edited cells. Results/Anticipated Results: Utilizing CRISPR-Cas9 tools, we anticipate that CHCHD2 gene can functionally compensate for the loss of function in the CHCHD10 locus through transcriptional adaptation. Additionally, employing single-stranded oligodeoxynucleotides (ssODNs) we aim to accurately correct the genetic aberrations in ALS patient cells, and study the pathomechanisms of aberrant CHCHD10. Discussion/Significance of Impact: The significance of this research lies in its potential to uncover transcriptional adaptation in humans, which could explain why patients with the same genetic variant experience different symptoms. By understanding this mechanism, we could pave the way for novel therapies, especially for CHCHD10-associated ALS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e325e0bdd2a54149390c949c2bbd9ef99b9c77" target='_blank'>
              482 Translational gene editing strategies for understanding and treating CHCHD10-associated amyotrophic lateral sclerosis (ALS)
              </a>
            </td>
          <td>
            A. Gomez, Nathan P. Staff, S. Ekker
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background SETD2 is the sole epigenetic factor responsible for catalyzing histone 3, lysine 36, tri-methylation (H3K36me3) in mammals. Its role in regulating cellular processes such as RNA splicing, DNA repair, and spurious transcription initiation underlies its broader tumor suppressor function. SETD2 mutation promotes the epithelial-mesenchymal transition and is clinically associated with adverse outcomes highlighting a therapeutic need to develop targeted therapies against this dangerous mutation. Results We employ an unbiased genome-wide synthetic lethal screen, which identifies another H3K36me writer, NSD1, as a synthetic lethal modifier in SETD2-mutant cells. Confirmation of this synthetic lethal interaction is performed in isogenic clear cell renal cell carcinoma and immortalized renal epithelial cell lines, in mouse and human backgrounds. Depletion of NSD1 using a CRISPRi targeting approach promotes the loss of SETD2-mutant cells coincident with elevated levels of DNA damage and apoptosis. Surprisingly, only suppression of NSD1, but not related H3K36-methyltransferases, promotes synthetic lethality in these models. Mapping of genomic H3K36me2 targeting by NSD1 and NSD2 individually highlights the independent functions of these epigenetic writers. Furthermore, as a proof-of-principle, we demonstrate the therapeutic feasibility of targeting this synthetic lethal interaction by recapitulating the phenotype using BT5, a first-in-class pharmacologic inhibitor against NSD1. Conclusions These findings unify genome-wide screening approaches with the latest genetic and pharmacologic modeling methodologies to reveal an entirely novel epigenetic approach to individualize therapies against a challenging loss-of-function SETD2 mutation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03483-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f74f88772da2d32d539504f99797675a6a54aee" target='_blank'>
              JMJD4 is a potential diagnostic and immunological biomarker in pan-cancer
              </a>
            </td>
          <td>
            Qinyi Li
          </td>
          <td>2025-03-03</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The use of CRISPR to knockdown or knockout genes is a powerful tool for understanding the specific role of a gene in disease development. However, it can cause many unanticipated changes to the transcriptome that are not detected by DNA amplification and Sanger sequencing of the target site. Various RNA-sequencing techniques can be used to identify these changes and effectively gauge the full impact of the CRISPR knockout, thereby providing a means of selecting appropriate clones for further experimentation. Background/Objectives: RNA-seq data from 4 CRISPR knockout experiments were analyzed and techniques developed to both confirm the success of the CRISPR modifications and identify potential issues. Methods: A broad-based analysis of RNA-sequencing data identified many CRISPR-based changes not identified by PCR amplification of DNA around the CRISPR target site. These changes included an inter-chromosomal fusion event, exon skipping, chromosomal truncation, and the unintentional transcriptional modification and amplification of a neighboring gene. Conclusions: The inadvertent modifications identified by the evaluation of 4 CRISPR experiments highlight the value of using RNA-seq to identify transcriptional changes to cells altered by CRISPR, many of which cannot be recognized by evaluating DNA alone. Specific guidelines are presented for designing and analyzing CRISPR experiments using RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/220a9a7ebf2665e1a3e4bd65d2ca79919ea760ce" target='_blank'>
              Techniques for Validating CRISPR Changes Using RNA-Sequencing Data
              </a>
            </td>
          <td>
            S. Rathe, Tracy A. Marko, Elizabeth N. Edwards, Paige Hazelton Ridder, Jyotika Varshney, Kyle B. Williams, James E. Johnson, B. Moriarity, D. A. Largaespada
          </td>
          <td>2025-03-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e707dd4d2590e8deca1a1e653d9b70d94049240c" target='_blank'>
              Suppress the proliferation process of glioblastoma by breaking off the tumor - microenvironment interaction by CRIPSR -mediated technology
              </a>
            </td>
          <td>
            Wei Wang
          </td>
          <td>2025-02-27</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC) is a highly aggressive malignancy characterized by rapid proliferation, early metastasis, and frequent recurrence, which contribute to a poor prognosis. SCLC is defined by the near-universal inactivation of key tumor suppressor genes, notably TP53 and RB1, which play central roles in its pathogenesis and resistance to therapy. The p53 family of proteins, including p53, p63, and p73, is essential to maintaining cellular homeostasis and tumor suppression. TP53 mutations are almost ubiquitous in SCLC, leading to dysregulated apoptosis and cell cycle control. Moreover, p73 shows potential as a compensatory mechanism for p53 loss, while p63 has a minimal role in this cancer type. In this review, we explore the molecular and functional interplay of the p53 family in SCLC, emphasizing its members’ distinct yet interconnected roles in tumor suppression, immune modulation, and therapy resistance. We highlight emerging therapeutic strategies targeting these pathways, including reactivating mutant p53, exploiting synthetic lethality, and addressing immune evasion mechanisms. Furthermore, this review underscores the urgent need for novel, isoform-specific interventions to enhance treatment efficacy and improve patient outcomes in this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc096d40431e9f2c1e7043ab1a608082051ec709" target='_blank'>
              Recent Research on Role of p53 Family in Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Minho Jeong, Kee-Beom Kim
          </td>
          <td>2025-03-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Genes and genetic variants associated with human disease are continually being discovered, but validating their causative roles and mechanisms remains a significant challenge. CRISPR/Cas9 genome editing in model organisms like zebrafish can enable phenotypic characterization of founder generation (F0) knockouts (Crispants), but existing approaches are not amenable to high-throughput genetic screening due to high variability, cost, and low phenotype penetrance. To overcome these challenges, here we provide guide RNA (gRNA) selection rules that enable high phenotypic penetrance of up to three simultaneous knockouts in F0 animals following injection of 1–2 gRNAs per gene. We demonstrate a strong transcriptomic overlap in our F0 knockouts and stable knockout lines that take several months to generate. We systematically evaluated this approach across 324 gRNAs targeting 125 genes and demonstrated its utility in studying epistasis, characterizing paralogous genes, and validating human disease gene phenotypes across multiple tissues. Applying our approach in a high-throughput manner, we screened and identified 10 novel neurodevelopmental disorders and 50 hearing genes not previously studied in zebrafish. Altogether, our approach achieves high phenotypic penetrance using low numbers of gRNAs per gene in F0 zebrafish, offering a robust pipeline for rapidly characterizing candidate human disease genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d442e32931385bdcd94b5a33eaea8e6b64c86c8" target='_blank'>
              Optimizing gRNA selection for high-penetrance F0 CRISPR screening for interrogating disease gene function
              </a>
            </td>
          <td>
            Sheng-Jia Lin, Kevin Huang, C. Petree, Wei Qin, Pratishtha Varshney, Gaurav K Varshney
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Phenotypic outcomes can be heavily affected by environmental factors. In this study, we exploited the previously observed nutrient-dependency of cell biological phenotypic features, captured by a cross-condition image-based profiling assay of Escherichia coli deletion strains, to examine this in more detail. We identified several general principles, including the existence of a spectrum of deviating phenotypes across nutrient conditions (i.e., from nutrient- or feature-specific to pleiotropic phenotypic deviations), limited conservation of phenotypic deviations across nutrient conditions (i.e., limited phenotypic robustness), and a subset of nutrient-independent phenotypic deviations (indicative of consistent genetic determinants of specific phenotypic features). In a subsequent step, we used this cross-condition dataset to identify five genes of unknown function of which the deletion displayed either nutrient-independent phenotypic deviations or phenotypic similarities to genes of known function: yibN, yaaY, yfaQ, ybiJ, and yijD. These genes showed different levels of phylogenetic conservation, ranging from conserved across the tree of life (yibN) to only present in some genera of the Enterobacterales (yaaY). Analysis of the structural properties of the proteins encoded by these y-genes, identification of structural similarities to other proteins, and the examination of their subcellular localization yielded new insights into their contribution to E. coli cell morphogenesis, cell cycle progression and cell growth. Together, our approach showcases how bacterial image-based profiling assays and datasets can serve as a gateway to reveal the function of uncharacterized proteins. Importance Despite unprecedented access to genomic information, predicting phenotypes based on genotypes remains notoriously difficult. One major confounding factor is the environment and its ability to modulate phenotypic outcomes. Another is the fact that a large fraction of protein-coding genes in bacterial genomes remains uncharacterized and have no known function. In this work, we use a large-scale cross-condition image-based profiling dataset to characterize nutrient-dependent phenotypic variability of E. coli deletion strains and exploit it to provide insight into the cellular role of genes of unknown function. Through our analysis, we identified five genes of unknown function that we subsequently further characterized by examining their phylogenetic conservation, predicted structural properties and similarities, and their intracellular localization. Combined, this approach highlights the potential of cross-condition image-based profiling, which extracts many cell biological phenotypic readouts across multiple conditions, to better understand nutrient-dependent phenotypic variability and uncover protein function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e64fdb79587408307645dda0d8efcc1410cda8f" target='_blank'>
              Complex interplay between gene deletions and the environment uncovers cellular roles for genes of unknown function in Escherichia coli
              </a>
            </td>
          <td>
            Kaat Sondervorst, Kristina Nesporova, Matthew Herdman, Bart Steemans, Joëlle Rosseels, Sander K. Govers
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95b9504bba9cb27a525f77f6f19beab513280064" target='_blank'>
              Ribosome Profiling Reveals Translational Reprogramming via mTOR Activation in Omacetaxine Resistant Multiple Myeloma.
              </a>
            </td>
          <td>
            Z. Walker, Katherine F. Vaeth, Amber Baldwin, Denis J Ohlstrom, Lauren T. Reiman, Kady A Dennis, Kate Matlin, Beau M. Idler, Brett M Stevens, Neelanjan Mukherjee, Daniel W. Sherbenou
          </td>
          <td>2025-03-06</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="PURPOSE It is a clinical challenge to select patients for BRAF-targeted therapy because of the lack of predictive biomarkers besides the BRAF V600E mutation. By analyzing the genome, transcriptome, and proteome, this study investigated the association between baseline molecular alterations and outcomes of BRAF-targeted therapy. PATIENTS AND METHODS Fresh tumor tissue from patients enrolled in the Copenhagen Prospective Personalized Oncology study was collected and underwent comprehensive molecular profiling. RESULTS TP53 comutations were most frequently detected. Patients with a TP53 wild-type tumor had a significantly longer median progression-free survival than those with TP53 comutations (hazard ratio, 2.8 [95% CI, 1.13 to 7.08]; P = .02). RNAseq revealed a distinct gene expression signature for patients with long-term disease control (LDC), including hallmarks of cell cycle arrest and proliferation in the p53 pathway. The protein analysis demonstrated that ubiquitin-conjugating enzyme EK2 was significantly downregulated in patients with LDC. CONCLUSION Using a multiomic approach, we identified molecular alterations associated with treatment outcomes. The potential of analyzing multiomic data is promising and should be prioritized in translational cancer research to uncover the full potential within precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d33023bdab6e4dfcabecc3f50f8af28550770e9" target='_blank'>
              Multiomics Identifies Potential Molecular Profiles Associated With Outcomes After BRAF-Targeted Therapy in Patients With BRAF V600E-Mutated Advanced Solid Tumors
              </a>
            </td>
          <td>
            M. Eriksen, Anne M Hansen, Annelaura B Nielsen, F. Mundt, Matthias Mann, U. Lassen, L. B. Ahlborn, M. Højgaard, I. Spanggaard, Camilla Qvortrup, C. Yde, K. Rohrberg
          </td>
          <td>2025-03-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15bb80576bbd8d5340c20144f6b3a46d4763d6f" target='_blank'>
              Clofazimine treatment modulates key non-coding RNAs associated with tumor progression and drug resistance in lethal Prostate Cancer
              </a>
            </td>
          <td>
            Sarah Batten, Harish Kumar, Suman Mazumder, T. Ghosh, Amit Kumar
          </td>
          <td>2025-02-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) is the most prevalent malignancy among women, characterized by extensive heterogeneity stemming from molecular and genetic alterations. This review explores the intricate epigenetic landscape of BC, highlighting the significant role of epigenetic modifications—particularly DNA methylation, histone modifications, and the influence of non-coding RNAs—in the initiation, progression, and prognosis of the disease. Epigenetic alterations drive crucial processes, including gene expression regulation, cell differentiation, and tumor microenvironment interactions, contributing to tumorigenesis and metastatic potential. Notably, aberrations in DNA methylation patterns, including global hypomethylation and hypermethylation of CpG islands, have been associated with distinct BC subtypes, with implications for early detection and risk assessment. Furthermore, histone modifications, such as acetylation and methylation, affect cancer cell plasticity and aggressiveness by profoundly influencing chromatin dynamics and gene transcription. Finally, non-coding RNAs contribute by modulating epigenetic machinery and gene expression. Despite advances in our knowledge, clinical application of epigenetic therapies in BC is still challenging, often yielding limited efficacy when used alone. However, combining epi-drugs with established treatments shows promise for enhancing therapeutic outcomes. This review underscores the importance of integrating epigenetic insights into personalized BC treatment strategies, emphasizing the potential of epigenetic biomarkers for improving diagnosis, prognosis, and therapeutic response in affected patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a596476c40bed6ddb55424b08c446ae8d78faf" target='_blank'>
              Decoding the Epigenome of Breast Cancer
              </a>
            </td>
          <td>
            Elisa Cortellesi, Isabella Savini, Matteo Veneziano, Alessandra Gambacurta, M. Catani, V. Gasperi
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64abcfa81e97e4ccd6aa92a436d6301cbd6a137c" target='_blank'>
              CRISPR-Cas9: Transforming Functional Genomics, Precision Medicine, and Drug Development - Opportunities, Challenges, and Future Directions.
              </a>
            </td>
          <td>
            Zuber Khan, Mumtaz, Sumedha Gupta, Sidharth Mehan, Tarun Sharma, Manjeet Kumar, Pankaj Kumar Maurya, Arun Kumar Sharma, Ghanshyam Das Gupta, A. Narula
          </td>
          <td>2025-03-21</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4625a1647e0225fc85616b697ca8ae26c76a3c30" target='_blank'>
              Transformative CRISPR-Cas9 Technologies: A Review of Molecular Mechanisms, Precision Editing Techniques, and Clinical Progress in Sickle Cell Disease.
              </a>
            </td>
          <td>
            Komal, Prabhjot Kaur, Nidhi Arora, Jyotiram A. Sawale, Amandeep Singh
          </td>
          <td>2025-03-05</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Purpose The complexity and heterogeneity of the tumor immune microenvironment (TIME) are linked to the development and poor prognosis of hepatocellular carcinoma (HCC). However, the cell type within the TIME that is most closely associated with HCC development remains unclear. Herein, we aimed to identify cell clusters that significantly contribute to HCC development and their underlying mechanisms. Method and Results Using single-cell RNA sequencing (scRNA-seq), we analyzed changes in the TIME of normal and tumor tissues, identifying plasma cells as the key cluster in HCC development. Based on plasma cell-related genes (PCRGs), we constructed and validated an eight-gene prognostic model (ST6GALNAC4, SEC61A1, SSR3, RPN2, PRDX4, TRAM1, SPCS2, CD79A) using internal and external datasets and a nomogram. Functional enrichment, miRNA network construction, and transcriptional regulation analyses were performed to explore underlying mechanisms. TIDE scores and the GDSC database were used to predict immunotherapy and chemotherapy sensitivity in different risk groups. Finally, SSR3’s biological function was validated in vitro in HCC cell lines. Conclusion Plasma cells are key clusters in HCC development. A prognostic model based on the PCRGs can accurately predict the prognosis of patients with HCC and guide clinical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/789d040bed058d7536fc99da4ae75b9692f60293" target='_blank'>
              Integrative Analysis of scRNA-Seq and Bulk RNA-Seq Identifies Plasma Cell Related Genes and Constructs a Prognostic Model for Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Mingyang Tang, Yuyan Xu, Mingxin Pan
          </td>
          <td>2025-02-01</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/995da50c8649d8341a6a265386cac02363ea1e3b" target='_blank'>
              Performing an In Vitro Genome-Wide CRISPR Knockout Screen in Chimeric Antigen Receptor T Cells.
              </a>
            </td>
          <td>
            Carli M. Stewart, Claudia Manriquez Roman, S. Kenderian
          </td>
          <td>2025-01-31</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immortalized cell lines are commonly used as proxies for primary cells in human biology research. For example, Jurkat leukemic T cells fundamentally contributed to uncovering T cell signaling, activation, and immune responses. However, the immortalization process can alter key cellular properties, and researchers widely believe this process could significantly change RNA modification machinery and modification sites. In this study, we focus on pseudouridine (Ψ), one of the most abundant mRNA modifications, and compare Ψ profiles in mRNA from primary and immortalized T cells using direct RNA sequencing (DRS). Surprisingly, 87% of Ψ-sites were shared between the two cell types, primarily in transcripts encoding proteins involved in essential cellular processes, including RNA-modification regulation. Furthermore, the analysis of the 13% of sites unique to each cell type reveals that Jurkat cells contained transcripts linked to immune activation and oncogenesis, while primary T cells contained transcripts associated with calcium signaling and intracellular trafficking. We provide a list of these genes, which should be considered when using immortalized cells to study RNA modifications in immunology contexts. Most differences were driven by whether the mRNA was present or absent in the immortalized or primary cell type. Interestingly, RNA-modification enzyme expression levels were highly conserved in both cell types. This suggests that site-specific differences in Ψ levels arise from regulatory processes acting in trans rather than differences in modification enzyme levels. Significance Statement It is widely believed that RNA modification machinery and modification sites could be significantly altered in immortalized cells, yet this has never been tested. Focusing on pseudouridine (Ψ), we map Ψ in the transcriptomes of primary and immortalized human T cells. Surprisingly, most sites are conserved in the primary and immortalized T cells, with several important examples of cases with cell type specificity and should be considered on a case-by-case basis. Furthermore, we evaluated RNA-modification machinery levels in primary and immortalized T cells, finding high conservation across the cell lines. Our findings demonstrate that RNA modifications are largely conserved between primary and immortalized cells, and the edge cases can be considered individually.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b46d555b004bfc6033b490f3c93648df1820f4" target='_blank'>
              Direct RNA sequencing of primary human T cells reveals the impact of immortalization on mRNA pseudouridine modifications
              </a>
            </td>
          <td>
            Oleksandra Fanari, Dylan Bloch, Yuchen Qiu, Michele Meseonznik, Amr Makhamreh, M. Wanunu, S. Rouhanifard
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b02a5edf5833a9413d878e11e7ef959bc982d0" target='_blank'>
              Embracing the Circular Journey of RNA in Colorectal Cancer.
              </a>
            </td>
          <td>
            A. Barjasteh, Ayat Heidar Abdolamir, Abdulridha Mohammed Al-Asady, Hanieh Latifi, Amir Avan, Majid Khazaei, Mikhail Ryzhikov, S. M. Hassanian
          </td>
          <td>2025-03-03</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) is a fatal malignant tumor of the digestive system, and immunotherapy has currently emerged as a key therapeutic approach for treating PAAD, with its efficacy closely linked to T-cell subsets and the tumor immune microenvironment. However, reliable predictive markers to guide clinical immunotherapy for PAAD are not available. We analyzed the single-cell RNA sequencing (scRNA-seq) data focused on PAAD from the GeneExpressionOmnibus (GEO) database. Then, the information from the Cancer Genome Atlas (TCGA) database was integrated to develop and validate a prognostic risk model derived from T-cell marker genes. Subsequently, the correlation between these risk models and the effectiveness of immunotherapy was explored. Analysis of scRNA-seq data uncovered six T-cell subtypes and 1837 T-cell differentially expressed genes (DEGs). Combining these data with the TCGA dataset, we constructed a T-cell prognostic risk model containing 16 DEGs, which can effectively predict patient survival and immunotherapy outcomes. We have found that patients in the low-risk group had better prognostic outcomes, increased immune cell infiltration, and signs of immune activation compared to those in the high-risk group. Additionally, analysis of tumor mutation burden showed higher mutation rates in patients with PAAD in the high-risk group. Risk scores with immune checkpoint gene expression and drug sensitivity analysis provide patients with multiple therapeutic targets and drug options. Our study constructed a prognostic risk model for PAAD patients based on T-cell marker genes, providing valuable insights into predicting patient prognosis and the effectiveness of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2eccd9b99a66967879131e3edf663927ebf7fa1" target='_blank'>
              Construction of T-Cell-Related Prognostic Risk Models and Prediction of Tumor Immune Microenvironment Regulation in Pancreatic Adenocarcinoma via Integrated Analysis of Single-Cell RNA-Seq and Bulk RNA-Seq
              </a>
            </td>
          <td>
            Ding-ya Sun, Yijie Hu, Jun Peng, Shan Wang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is known for frequent copy number alterations (CNAs) and metabolic reprogramming. However, the mechanism by which CNAs of metabolic genes drive distinct metabolic reprogramming and affect disease progression remains unclear. Through an integrated analysis of our TNBC multiomic dataset (n = 465) and subsequent experimental validation, we identify copy number amplification of the metabolic gene flavin-adenine dinucleotide synthetase 1 (FLAD1) as a crucial genetic event that drives TNBC progression. Mechanistically, FLAD1, but not its enzymatically inactive mutant, upregulates the enzymatic activity of FAD-dependent lysine-specific demethylase 1 (LSD1). LSD1 subsequently promotes the expression of sterol regulatory element-binding protein 1 (SREBP1) by demethylating dimethyl histone H3 lysine 9 (H3K9me2). The upregulation of SREBP1 enhances the expression of lipid biosynthesis genes, ultimately facilitating the progression of TNBC. Clinically, pharmacological inhibition of the FLAD1/LSD1/SREBP1 axis effectively suppresses FLAD1-induced tumor progression. Moreover, LSD1 inhibitor enhances the therapeutic effect of doxorubicin and sacituzumab govitecan (SG). In conclusion, our findings reveal the CNA-derived oncogenic signalling axis of FLAD1/LSD1/SREBP1 and present a promising treatment strategy for TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05abf56ac7febcd59ec911620962ed978222a49b" target='_blank'>
              Copy number amplification of FLAD1 promotes the progression of triple-negative breast cancer through lipid metabolism
              </a>
            </td>
          <td>
            Xiao-Qing Song, Tian-Jian Yu, Ou-Yang Yang, Jianlong Ding, Yizhou Jiang, Zhi-Ming Shao, Yi Xiao
          </td>
          <td>2025-02-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Malignant mesothelioma is a highly aggressive cancer with a poor prognosis and limited therapeutic options. The tumor microenvironment (TME) plays a pivotal role in driving tumor progression, with immune cells influencing disease outcomes. However, the molecular mechanisms underpinning mesothelioma’s progression remain insufficiently understood. HLA-C, a class I major histocompatibility complex (MHC) molecule, has been implicated in immune modulation and cancer progression, but its specific role in mesothelioma has yet to be thoroughly investigated. Methods This study employed a comprehensive multi-omics approach, integrating single-cell RNA sequencing, expression quantitative trait loci (eQTL) analysis, and Mendelian randomization (MR), to elucidate the role of HLA-C in mesothelioma progression. We first analyzed HLA-C expression within the TME, with particular focus on immune cells, especially macrophages. Survival analysis was conducted using data from the TCGA mesothelioma cohort to assess the clinical relevance of HLA-C expression. We utilized mediated MR analysis to investigate the impact of DNA methylation on HLA-C expression, identifying key mediators such as inflammatory cytokines, immune cell populations, blood cell types, and metabolites that could potentially influence patient prognosis. Results HLA-C was predominantly expressed in macrophages, T cells, and NK cells within the TME, and higher expression levels were associated with improved patient survival. MR analysis revealed that DNA methylation regulates HLA-C expression, which in turn impacts mesothelioma outcomes. Mediated MR analysis, encompassing 91 inflammatory cytokines, 731 immune cell populations, 91 blood cell types, and 1400 metabolites, highlighted several critical mediators of HLA-C’s effect on prognosis, including IL-10, CD33 expression on CD33dim HLA DR- myeloid cells, the reticulocyte perturbation response, and the ADP-to-citrate ratio. Gene set enrichment analysis (GSEA) showed significant enrichment of immune-related and inflammatory pathways in patients with high HLA-C expression. Conclusion HLA-C, regulated by DNA methylation, plays a central role in mesothelioma prognosis by modulating immune responses, inflammatory cytokines, blood cell populations, and metabolic processes within the TME. Our findings suggest that HLA-C could serve as both a prognostic biomarker and a potential therapeutic target for mesothelioma, offering new insights into the molecular mechanisms driving this aggressive cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04012-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4904fecc6fa927af49272155c376129c6ffe55e9" target='_blank'>
              DNA methylation-regulated HLA-C expression modulates immune responses and metabolic alterations to influence prognosis in mesothelioma
              </a>
            </td>
          <td>
            Huilin Zhang, Luanxue Yu, Yunqing Guo, Jiawei Ming, Zhenying Guo
          </td>
          <td>2025-03-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is complex because of the critical imbalance in genetic regulation as characterized by both the overexpression of oncogenes (OGs), mainly through mutations, amplifications, and translocations, and the inactivation of tumor-suppressor genes (TSGs), which entail the preservation of genomic integrity by inducing apoptosis to counter the malignant growth. Reviewing the intricate molecular interplay between OGs and TSGs draws attention to their cell cycle, apoptosis, and cancer metabolism regulation. In the present review, we discuss seminal discoveries, such as Knudson’s two-hit hypothesis, which framed the field’s understanding of cancer genetics, leading to the next breakthroughs with next-generation sequencing and epigenetic profiling, revealing novel insights into OG and TSG dysregulation with opportunities for targeted therapy. The key pathways, such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, are presented in the context of tumor progression. Importantly, we further highlighted the advances in therapeutic strategies, including inhibitors of KRAS and MYC and restoration of TSG function, despite which mechanisms of resistance and tumor heterogeneity pose daunting challenges. A high-level understanding of interactions between OG-TSGs forms the basis for effective, personalized cancer treatment—something to strive for in better clinical outcomes. This synthesis should integrate foundational biology with translation and, in this case, contribute to the ongoing effort against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b3a2a7fdf1a317ed911bfc6a066f5d8abc14563" target='_blank'>
              Exploring the Function of METTL3 in Lung Cancer: Understanding Drug Resistance and Exploring Innovative Therapeutic Approaches
              </a>
            </td>
          <td>
            Shiqing Huang, Ye Li, F. Liew
          </td>
          <td>2025-03-12</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20366f8a8e956d20a8164d2739ca733da77de50e" target='_blank'>
              RUNX1::RUNX1T1 Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. Daniel, Mata Casimiro, Luca van den Brink, Anita T. van Oort, Minoo Ashtiani, C. Zwaan, Anja Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, H. J. Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Clear cell renal cell carcinoma (ccRCC) is a prevalent and aggressive subtype of kidney cancer, often associated with metastasis and recurrence. Identifying key genes involved in ccRCC progression is critical for improving treatment strategies and patient outcomes. Methods: We performed a large-scale genome-wide CRISPR screening to identify genes crucial to ccRCC progression using the DepMap database. For discovery and validation, we integrated multi-omics data from The Cancer Genome Atlas (TCGA), GEO, and the NJMU-ccRCC clinical cohort. Bioinformatics analyses, including differential expression, pathway enrichment, and protein-protein interaction network analysis, were conducted to elucidate the biological functions. To validate our findings, we employed immunohistochemistry, qRT-PCR, and various cellular assays to investigate the role of PRC1 in ccRCC. Results: CRISPR screening identified PRC1 as a key gene significantly overexpressed in ccRCC tissues from the DepMap database. Elevated PRC1 expression was associated with poor overall survival, disease-specific survival, and progression-free interval. Silencing PRC1 in ccRCC cell lines inhibited cell proliferation, migration, and colony formation. Functional enrichment analyses revealed that PRC1 is involved in essential processes such as cell cycle regulation, mitosis, and cytokinesis. Additionally, PRC1 expression was correlated with the activation of the Wnt/β-catenin pathway, suggesting that PRC1 plays a pivotal role in tumor progression. Conclusion: PRC1 emerges as a promising biomarker and therapeutic target for ccRCC. Elevated PRC1 expression is associated with poor prognosis, and its inhibition suppresses ccRCC cell proliferation and migration. Our findings underscore the crucial role of PRC1 in ccRCC progression and highlight the need for further investigation into its molecular mechanisms and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f7e34d29eb95f0f25970817abd06150852dbc62" target='_blank'>
              Large-scale genomic-wide CRISPR screening revealed PRC1 as a tumor essential candidate in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Baochao Li, Yongsheng Pan, Jiajin Wu, Chenkui Miao, Zengjun Wang
          </td>
          <td>2025-03-03</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f86d44b9e09159a099ee094747765812cf22c69" target='_blank'>
              CRISPR-dCas9-Mediated PTEN Activation via Tumor Cell Membrane-Coated Nanoplatform Enhances Sensitivity to Tyrosine Kinase Inhibitors in Nonsmall Cell Lung Cancer.
              </a>
            </td>
          <td>
            Kaiyong Yang, Chunli Zhang, Zeyu Wang, Qiqing Huang, Jing Qian, Gaoyu Shi, Wenwen Sun, Jinqiu Wang, Yu Ji, Zhaorui Sun, Yanni Song, Xin Han
          </td>
          <td>2025-02-20</td>
          <td>ACS applied materials & interfaces</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human T-cell lymphotropic virus type 1 (HTLV-1) infection causes the uncommon and deadly cancer known as adult T-cell leukemia/lymphoma (ATLL), which affects mature T cells. Its clinical appearance is varied, and its prognosis is often miserable. Drug resistance to conventional therapies confers significant therapeutic challenges in the management of ATLL. This review discusses the emerging role of epigenetic medical advances in the treatment of ATLL, focusing on DNA methyltransferase inhibitors, histone deacetylase inhibitors, histone methyltransferase inhibitors, and BET inhibitors. Indeed, several classes of epigenetic therapies currently exhibit trailed efficacy in preclinical and clinical studies: DNA methyltransferase inhibitors like azacitidine and decitabine reexpression of silenced tumor suppressors; histone deacetylase inhibitors like vorinostat and romidepsin induce cell cycle arrest and apoptosis; bromodomain and extra-terminal inhibitors like JQ1 disrupt oncogenic signaling pathways. Whereas preclinical and early clinical data indicate modest to good efficacy for such treatments, significant challenges remain. Here, we discuss the current state of understanding of epigenetic dysregulation in ATLL and appraise the evidence supporting the use of these epi-drugs. However, despite the opened doors of epigenetic treatment, much more research is required with regard to showing the best combinations of drugs and their resistance mechanisms, the minimization of adverse effects, and how this hope will eventually be translated into benefit for the patient with ATLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Background Investigating the pivotal role of CTHRC1 in the tumor microenvironment of triple-negative breast cancer (TNBC). Method The RNA transcriptomic data obtained from the Cancer Genome Atlas and single-cell sequencing data from TNBC in Gene Expression Omnibus (GEO) were acquired and subjected to analysis. A comprehensive investigation was conducted with a specific focus on characterizing CTHRC1 in TNBC and its correlation with invasive genes. Furthermore, additional analyses were performed to explore the relationship between CTHRC1, tumor immune cell infiltration, and immunotherapy in TNBC. The expression of CTHRC1 in the tumor microenvironment, cellular differentiation, and cellular communication was systematically analyzed using single-cell data from TNBC. Result The expression of CTHRC1 in patients with TNBC gradually increases concomitantly with the progression of tumor T-stage and N-stage. Simultaneously, there is a concurrent increase in the expression of most invasive gene sets. Furthermore, there is a significant augmentation in both infiltration abundance and activity of M2-type macrophages associated with elevated levels of CTHRC1 expression. Single-cell data reveal an upregulated expression of the invasive gene set in CTHRC1-positive cancer associated fibroblasts (CAFs), thereby modulating their interaction with M2-type macrophages. Multiple immunofluorescence analyses confirmed that CTHRC1 modulates immune cell infiltration and tumor cell invasion through the mediation of CAFs. Conclusion CTHRC1 was a molecule that exhibits characteristic expression in TNBC. CTHRC1 positive CAFs exert regulatory effects within the immunosuppressive microenvironment of TNBC by modulating M2-type macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4be4e81b525de2c62cbaf620445907b577884546" target='_blank'>
              Single-cell and transcriptome analyses revealed CTHRC1 a potential therapeutic target mediating invasion and tumor microenvironment in TNBC: experimental validation
              </a>
            </td>
          <td>
            Hong Wan, Zichen Ling, Yuwei Xie, Han Jiang, Zhifan Ruan, Dashuai Yang, Xiaowei Yang, Jing Pei
          </td>
          <td>2025-03-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401aff8c0f114a1e1c91ffcb779a01c90497fc5c" target='_blank'>
              Molecular insights and treatment innovations: Advancing outcomes in acute myeloid leukemia with myelodysplasia‑related changes (Review).
              </a>
            </td>
          <td>
            Hong Qiu, Chao-Wei Zhang, Xiaochen Ma, Ying Li
          </td>
          <td>2025-03-19</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Aberrant hypermethylation in the promoter regions of tumor suppressor genes facilitates the pathogenesis and progression of cancer. Therefore, inhibitors targeting DNA methyltransferase (DNMT) have been tested in clinical studies. However, the current monotherapy of DNMT inhibitors shows limited efficacy. Furthermore, the mechanism of action of DNMT inhibitors is DNA replication-dependent. To address these limitations, we developed a novel core–shell-type “epigenetics control (EpC) nanocarrier” that encapsulated decitabine (5-aza-dC) in the PLGA core nanoparticle and hybridized TET1 gene-encoding pDNA on the lipid shell surface. This study aimed to evaluate whether the dual delivery of DNMT inhibitors and pDNA of TET1 could synergistically enhance tumor suppressor gene expression and induce cell cycle arrest and/or apoptosis in cancer cells. Herein, we demonstrate the potential of the EpC carrier in HCT116 human colon cancer cells to upregulate tumor suppressor gene expression and rapidly achieve cell cycle arrest. Methods: PLGA core nanoparticles were prepared by the W/O/W double emulsion method. The formation of core–shell nanoparticles and complexation with pDNA were investigated and optimized by dynamic light scattering, zeta potential measurement, and agarose gel electrophoresis. The cellular uptake and transfection efficiency were measured by confocal laser scanning microscopy and a luciferase assay, respectively. The expression of p53 protein was detected by Western blotting. The anti-tumor effects of the EpC nanocarrier were evaluated by cell cycle analysis and an apoptosis assay. Results: The EpC nanocarrier delivered the DNMT inhibitor and TET gene-encoding pDNA into HCT116 cells. It promoted the expression of the tumor suppressor protein p53 and induced rapid cell cycle arrest in the G2/M phase in HCT116 cells. Conclusions: Our findings suggest that the dual-targeting of DNMT and TET enzymes effectively repairs aberrant DNA methylation and induces growth arrest in cancer cells, and the dual-targeting strategy may contribute to the advancement of epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310794284715bd67edbc662582ab25897ba7a464" target='_blank'>
              Novel Epigenetics Control (EpC) Nanocarrier for Cancer Therapy Through Dual-Targeting Approach to DNA Methyltransferase and Ten-Eleven Translocation Enzymes
              </a>
            </td>
          <td>
            Risa Mitsuhashi, Kiyoshi Sato, Hiroyoshi Kawakami
          </td>
          <td>2025-02-11</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Monoallelic deletion of the short arm of chromosome 17 (del17p) is a recurrent abnormality in cancers with poor outcomes. Best studied in relation to haematological malignancies, associated functional outcomes are attributed mainly to loss and/or dysfunction of TP53, which is located at 17p13.1, but the wider impact of deletion of other genes located on 17p is poorly understood. 17p is one of the most gene-dense regions of the genome and includes tumour suppressor genes additional to TP53, genes essential for cell survival and proliferation, as well as small and long non-coding RNAs. In this review we utilise a data-driven approach to demarcate the extent of 17p deletion in multiple cancers and identify a common loss-of-function gene signature. We discuss how the resultant loss of heterozygosity (LOH) and haploinsufficiency may influence cell behaviour but also identify vulnerabilities that can potentially be exploited therapeutically. Finally, we highlight how emerging animal and isogenic cell line models of del17p can provide critical biological insights for cancer cell behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Background Cell-free DNA (cfDNA) in plasma carries epigenetic signatures specific to tissue or cell of origin. Aberrant methylation patterns in circulating cfDNA have emerged as valuable tools for noninvasive cancer detection, prenatal diagnostics, and organ transplant assessment. Such epigenetic changes also hold significant promise for the diagnosis of neurodegenerative diseases, which often progresses slowly and has a lengthy asymptomatic period. However, genome-wide cfDNA methylation changes in neurodegenerative diseases remain poorly understood. Results We used whole-genome bisulfite sequencing (WGBS) to profile age-dependent and ALS-associated methylation signatures in cfDNA from 30 individuals, including young and middle-aged controls, as well as ALS patients with matched controls. We identified 5,223 age-related differentially methylated loci (DMLs) (FDR < 0.05), with 51.6% showing hypomethylation in older individuals. Our results significantly overlapped with age-associated CpGs identified in a large blood-based epigenome-wide association study (EWAS). Comparing ALS patients to controls, we detected 1,045 differentially methylated regions (DMRs) in gene bodies, promoters, and intergenic regions. Notably, these DMRs were linked to key ALS-associated pathways, including endocytosis and cell adhesion. Integration with spinal cord transcriptomics revealed that 31% of DMR-associated genes exhibited differential expression in ALS patients compared to controls, with over 20 genes significantly correlating with disease duration. Furthermore, comparison with published single-nucleus RNA sequencing (snRNA-Seq) data of ALS demonstrated that cfDNA methylation changes reflects cell-type-specific gene dysregulation in the brain of ALS patients, particularly in excitatory neurons and astrocytes. Deconvolution of cfDNA methylation profiles suggested altered proportions of immune and liver-derived cfDNA in ALS patients. Conclusions cfDNA methylation is a powerful tool for assessing age-related changes and ALS-specific molecular dysregulation by revealing perturbed locus, genes, and the proportional contributions of different tissues/cells to the plasma. This technique holds promise for clinical application in biomarker discovery across a broad spectrum of neurodegenerative disorders. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01366-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f50dea3e9f7300d7800974584eb194f23dc33f9" target='_blank'>
              Whole-genome bisulfite sequencing of cell-free DNA unveils age-dependent and ALS-associated methylation alterations
              </a>
            </td>
          <td>
            Yulin Jin, Karen N Conneely, Wenjing Ma, Robert K Naviaux, Teepu Siddique, Emily G. Allen, Sandra Guingrich, Robert M Pascuzzi, Peng Jin
          </td>
          <td>2025-02-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb21c2801e782dbb110eb435c1cbbe7b47f6623" target='_blank'>
              Identifying gene expression and cellular pathways involved in glomerular AL-amyloidosis and correlation with experimental data: seeking novel therapeutic interventions.
              </a>
            </td>
          <td>
            Guillermo A. Herrera, J. Teng, Chun Zeng, L. Del Pozo-Yauner, Bing Liu, E. Turbat-Herrera
          </td>
          <td>2025-02-21</td>
          <td>Ultrastructural pathology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pluripotent stem cells (PSCs) possess the extraordinary capability to differentiate into a variety of cell types. This capability is tightly regulated by epigenetic mechanisms, particularly histone modifications. Moreover, the reprogramming of somatic or fate-committed cells into induced pluripotent stem cells (iPSCs) largely relies on these modifications, such as histone methylation and acetylation of histones. While extensive research has been conducted utilizing mouse models, the significance of histone modifications in human iPSCs is gaining increasing recognition. Recent studies underscore the importance of epigenetic regulators in both the reprogramming process and the regulation of cancer stem cells (CSCs), which are pivotal in tumor initiation and the development of treatment resistance. This review elucidates the dynamic alterations in histone modifications that impact reprogramming and emphasizes the necessity for a balance between activating and repressive marks. These epigenetic marks are influenced by enzymes such as DNA methyltransferases (DNMTs) and histone deacetylases (HDACs). Furthermore, this review explores therapeutic strategies aimed at targeting these epigenetic modifications to enhance treatment efficacy in cancer while advancing the understanding of pluripotency and reprogramming. Despite promising developments in the creation of inhibitors for histone-modifying enzymes, challenges such as selectivity and therapy resistance continue to pose significant hurdles. Therefore, future endeavors must prioritize biomarker-driven approaches and gene-editing technologies to optimize the efficacy of epigenetic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6f9fda52e0528a1ec82918f3f164c54bd8aa69" target='_blank'>
              Epigenetic regulation of reprogramming and pluripotency: insights from histone modifications and their implications for cancer stem cell therapies
              </a>
            </td>
          <td>
            Woori Bae, Eun A. Ra, Myon Hee Lee
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47287c3d47ac736b24fe418c263277c4788b373b" target='_blank'>
              Domains rearranged methyltransferases (DRMs)-mediated DNA methylation plays key roles in modulating gene expression and maintaining transposable element silencing in soybean.
              </a>
            </td>
          <td>
            Hongwei Xun, Lijie Lian, Jing Yuan, Jianhui Hong, Shanmeng Hao, Haonan Zhao, Shuhan Liu, Wanjie Feng, Huanran Yin, Bao Liu, Xutong Wang
          </td>
          <td>2025-03-07</td>
          <td>Journal of integrative plant biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="T cell-based immunotherapies hold promise in treating cancer by leveraging the immune system’s recognition of cancer-specific antigens1. However, their efficacy is limited in tumours with few somatic mutations and substantial intratumoural heterogeneity2–4. Here we introduce a previously uncharacterized class of tumour-wide public neoantigens originating from RNA splicing aberrations in diverse cancer types. We identified T cell receptor clones capable of recognizing and targeting neoantigens derived from aberrant splicing in GNAS and RPL22. In cases with multi-site biopsies, we detected the tumour-wide expression of the GNAS neojunction in glioma, mesothelioma, prostate cancer and liver cancer. These neoantigens are endogenously generated and presented by tumour cells under physiologic conditions and are sufficient to trigger cancer cell eradication by neoantigen-specific CD8+ T cells. Moreover, our study highlights a role for dysregulated splicing factor expression in specific cancer types, leading to recurrent patterns of neojunction upregulation. These findings establish a molecular basis for T cell-based immunotherapies addressing the challenges of intratumoural heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6192f93a04789d7b2aabf4cf16ca2d7d42091ce4" target='_blank'>
              Innovative CRISPR-Cas9 enhances cell rejuvenation, addressing disease
              </a>
            </td>
          <td>
            Linfeng Zhang
          </td>
          <td>2025-03-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, with its incidence and mortality rates projected to rise due to population aging. In this study, we utilized PCa transcriptome data from public databases and applied bioinformatics methods to identify three prognostic genes (CDC20, RAD51, and TTK) related to centrosome duplication in PCa. CDC20 is involved in cell cycle regulation, RAD51 in deoxyribonucleic acid double-strand break repair, and TTK in spindle assembly checkpoint function and cell proliferation. We constructed a risk model and a nomogram model, both demonstrating moderate to good predictive performance with area under the curve values ranging from 0.611 to 0.765 at different time points. Gene set enrichment analysis revealed that these genes were enriched in 64 pathways, including the cell cycle pathway, which is dysregulated in cancer. Furthermore, we analyzed the immune microenvironment and identified 13 differential immune cells and 13 differential immune checkpoints between high- and low-risk groups, providing insights into potential immunotherapy targets for PCa. In conclusion, this study contributes to a deeper understanding of PCa pathogenesis and lays important theoretical and experimental foundations for developing new diagnostic markers and treatment strategies. Future research requires more clinical samples and continued monitoring of the mechanism of these genes in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8581bf4b2f1cfe894fadfd64d88a1382b8cc282" target='_blank'>
              Prognostic Value of Centrosome Replication-Related Genes in Prostate Cancer Based on Transcriptomic and Mendelian Randomization
              </a>
            </td>
          <td>
            Qizhong Lu, Yufan Wu, Qiwei Yu, Jun Ouyang
          </td>
          <td>2025-03-01</td>
          <td>American Journal of Men's Health</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) play pivotal roles in innate immunity and contribute to the advancement of lung cancer. We aimed to identify novel TAM-related biomarkers and significance of macrophage infiltration in lung adenocarcinoma (LUAD) through an integrative analysis of single-cell RNA-sequencing (scRNA-seq) data. To describe the cell atlas and construct a novel prognostic signature in LUAD. Methods The gene signature linked to TAMs was identified utilizing Scanpy from the scRNA-seq dataset GSE131907. Subsequent analysis involved evaluating the expression levels of these genes, their potential molecular mechanisms, and prognostic significance in LUAD using data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases. We also constructed a risk score models through LASSO Cox regression for these genes. The underlying mechanism was further elucidated through the application of GSEA, ESTIMATE, TIDE, and other bioinformatic algorithms. Results Single-cell atlas was described by analyze 29 scRNA-seq samples from 19 LUAD patients. The TAMs-related gene signature (TGS) was identified as an independent prognostic factor by LASSO Cox regression analysis using differential expression genes (DEGs) derived from pro- and anti-inflammatory macrophage cells. Risk score model including nine TAMs-related genes (FOSL1, ZNF697, ADM, UBE2S, TICAM1, S100P, BIRC3, TLE1, and DEFB1) were obtained for prognosis construction. Moreover, the risk model underwent additional validation in four external GEO cohorts: GSE31210, GSE72094, GSE26939, and GSE30219. Interestingly, TGS-high tumors revealed enrichments in TGF-β signaling and hypoxia pathways, which shown low immune infiltration and immunosuppression by ESTIMATE and TIDE algorithm. The TGS-high risk group exhibited lower richness and diversity in the T-cell receptor (TCR) repertoire. Conclusion This study introduces a novel TGS score developed through LASSO Cox regression analysis, utilizing DEGs in pro- and anti-inflammatory macrophage cells. High TGS tumors exhibited enrichment in TGF-β signaling and hypoxia pathways, suggesting their potential utility in predicting prognosis and immune responses in patients with LUAD. These results offer promising implications for the development of therapeutic strategies for LUAD. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-01834-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab542107a71d04ec4e797429aa65c1f0ffcb1f72" target='_blank'>
              Single cell transcriptomic analysis reveals tumor immune infiltration by macrophage cells gene signature in lung adenocarcinoma
              </a>
            </td>
          <td>
            Xiaotong Guo, Youjun Deng, Wenjun Jiang, Heng Li, Yisheng Luo, Huachuan Zhang, Hao Wu
          </td>
          <td>2025-03-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Precision medicine approaches to cancer treatment aim to exploit genomic alterations that are specific to individual patients to tailor therapeutic strategies. Yet, some targetable genes and pathways are essential for tumor cell viability even in the absence of direct genomic alterations. In underrepresented populations, the mutational landscape and determinants of response to existing therapies are poorly characterized because of limited inclusion in clinical trials and studies. One way to reveal tumor essential genes is with genetic screens. Most screens are conducted on cell lines that bear little resemblance to patient tumors, after years of culture under nonphysiologic conditions. To address this problem, we aimed to develop a CRISPR screening pipeline in three-dimensionally grown patient-derived tumor organoid (PDTO) models. A breast cancer PDTO biobank that focused on underrepresented populations, including West African patients, was established and used to conduct a negative-selection kinome-focused CRISPR screen to identify kinases essential for organoid growth and potential targets for combination therapy with EGFR or MEK inhibitors. The screen identified several previously unidentified kinase targets, and the combination of FGFR1 and EGFR inhibitors synergized to block organoid proliferation. Together, these data demonstrate the feasibility of CRISPR-based genetic screens in patient-derived tumor models, including PDTOs from underrepresented patients with cancer, and identify targets for cancer therapy. Significance: Generation of a breast cancer patient-derived tumor organoid biobank focused on underrepresented populations enabled kinome-focused CRISPR screening that identified essential kinases and potential targets for combination therapy with EGFR or MEK inhibitors. See related commentary by Trembath and Spanheimer, p. 407.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde16e03b04b6ab8e7612d1796bad37a03db0723" target='_blank'>
              Kinome-Focused CRISPR-Cas9 Screens in African Ancestry Patient-Derived Breast Cancer Organoids Identify Essential Kinases and Synergy of EGFR and FGFR1 Inhibition.
              </a>
            </td>
          <td>
            Florencia P. Madorsky Rowdo, Rachel Martini, Sarah Ackermann, Colin P Tang, Marvel Tranquille, Adriana Irizarry, Ilkay Us, Omar Alawa, Jenna E Moyer, M. Sigouros, John Nguyen, Majd Al Assaad, Esther Cheng, Paula S Ginter, J. Manohar, Brian Stonaker, Richard Boateng, J. Oppong, E. Adjei, B. Awuah, I. Kyei, F. Aitpillah, M. Adinku, K. Ankomah, E. Osei-Bonsu, K. Gyan, Syed A Hoda, Lisa A Newman, Juan Miguel Mosquera, A. Sboner, O. Elemento, Lukas E. Dow, Melissa B. Davis, M. L. Martin
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e66bd557c2872856f8b9b782a05344a5f43e969c" target='_blank'>
              Preventing large deletions and chromosome loss in engineered human primary T cells by CasPlus with optimized guide RNAs
              </a>
            </td>
          <td>
            Qiaoyan Yang, Jonathan S. Abebe, Michelle Mai, Colin T. Konishi, Orrin Devinsky, Chengzu Long
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eccae0a3a741529fc24b2b02ce5646b06a22df12" target='_blank'>
              Engineering Gizmos for Short Cancer Genetic Fragments Discrimination.
              </a>
            </td>
          <td>
            Jisoo Yuk, Jeonghun Kim, Sunghwan Jung, Soong Ho Um
          </td>
          <td>2025-02-05</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis. Both breakage fusion bridge (BFB) cycles and extrachromosomal DNA (ecDNA) can lead to high oncogene copy numbers, but the impact of BFB amplifications on intratumoral heterogeneity, treatment response, and patient survival remains poorly understood due to detection challenges with DNA sequencing. We introduce an algorithm, OM2BFB, designed to detect and reconstruct BFB amplifications using optical genome mapping (OGM). OM2BFB demonstrates high precision (>93%) and recall (92%) in identifying BFB amplifications across cancer cell lines, patient-derived xenograft models, and primary tumors. Comparisons using OGM reveal that BFB detection with our AmpliconSuite toolkit for short-read sequencing also achieves high precision, though with reduced sensitivity. We identify 371 BFB events through whole genome sequencing of 2557 primary tumors and cancer cell lines. BFB amplifications are prevalent in cervical, head and neck, lung, and esophageal cancers, but rare in brain cancers. Genes amplified through BFB exhibit lower expression variance, with limited potential for regulatory adaptation compared to ecDNA-amplified genes. Tumors with BFB amplifications (BFB(+)) show reduced structural heterogeneity in amplicons and delayed resistance onset relative to ecDNA(+) tumors. These findings highlight ecDNA and BFB amplifications as distinct oncogene amplification mechanisms with differing biological characteristics, suggesting distinct avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf4afc79561ecf01107d9cb1fb29509ee0c751e0" target='_blank'>
              Harnessing ALDH1A2 vulnerability in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Andrieu, L. Courtois, Mickael Bonnet, Camille Gillet, Mélodie Queri, M. Féroul, L. Lhermitte, P. Villarese, Aurélie Cabanes-Hamy, Phillippe Rousselot, Mileydis Perez, I. Ceylan, Salvatore Spicuglia, Vahid Asnafi
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer chemoresistance presents a challenge in oncology, often leading to treatment failure and disease progression. CD44, a multifunctional cell surface glycoprotein, has garnered attention for its involvement in various aspects of cancer biology. Through alternative splicing, CD44 can form isoforms with the inclusion of only standard exons, typical for normal tissue, or with the addition of variant exons, frequently expressed in cancer tissue and associated with chemoresistance. The functions of CD44 involved in regulation of cancer signaling pathways are being actively studied, and the significance of specific variant exons in modulating cell death pathways, central to the response of cancer cells to chemotherapy, begins to become apparent. This review provides a comprehensive analysis of the association of CD44 variant exons/total CD44 with clinical outcomes of patients undergoing chemotherapy. The role of CD44 variant exons v6, v9 and others with a significant effect on patient chemotherapy outcomes by means of key cellular death pathways such as apoptosis, ferroptosis and autophagy modulation is further identified, and their impact on drug resistance is highlighted. An overview of clinical trials aimed at targeting variant exon-containing isoforms is provided, and possible directions for further development of CD44-targeted therapeutic strategies are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c896fa2073872384fe9590cc27feab2b38f88e9" target='_blank'>
              CD44 variant exons induce chemoresistance by modulating cell death pathways
              </a>
            </td>
          <td>
            Maria Yanova, Evgeniya Stepanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="One-carbon metabolism (1CM) has been reported to promote cancer progression across various malignancies. While 1CM is critical for cell proliferation by enhancing nucleotide synthesis, its physiological roles within different cell types in the tumor immune microenvironment (TIME) still remain unclear. In this study, we analyzed bulk-RNA sequencing and single-cell RNA sequencing (scRNA-seq) data from lung adenocarcinoma (LUAD) patients to elucidate the functional roles of 1CM within the TIME. Moreover, we examined scRNA-seq data from patients treated with immunotherapy across various cancers, including LUAD, glioblastoma, renal cell carcinoma, colorectal cancer, and triple-negative breast cancer. Compared to other cell types, 1CM gene profiles are significantly enriched in a specific subset of T cells. Intriguingly, these high-1CM T cells are identified as proliferative intermediate exhausted T cells (Texint). Furthermore, these proliferative Texint received the most robust CD137 signaling. Consistently, analysis of scRNA-seq data from LUAD patients undergoing anti-PD1 immunotherapy demonstrated that proliferative Texint exhibited higher 1CM scores and increased CD137 signaling. This observation was particularly pronounced in non-responders to immunotherapy, where the Texint population was significantly expanded. We further established that 1CM is a prominent signaling pathway in proliferative Texint in patients resistant to immunotherapy across multiple cancer types. Collectively, we identify CD137 signaling as a distinctive pathway in proliferative Texint of LUAD patients who do not respond to immunotherapy. These findings propose that targeting 1CM may represent a novel therapeutic strategy to enhance the efficacy of immunotherapy by mitigating Texint proliferation in diverse cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a799bb85faaaac3fa04136a525f7e0a65fbc4560" target='_blank'>
              One-carbon metabolism is distinct metabolic signature for proliferative intermediate exhausted T cells of ICB-resistant cancer patients
              </a>
            </td>
          <td>
            Ye-Chan Park, Yesueng Hwang, Jae Woong Jeong, Chae Min Lee, Minki Kim, Sugyeong Jo, Seyeon Joo, Nahee Hwang, Sungsoon Fang
          </td>
          <td>2025-02-14</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The tumor suppressor p53 (Trp53), also known as p53, is the most commonly mutated gene in cancer. Canonical p53 DNA damage response pathways are well characterized and classically thought to underlie the tumor suppressive effect of p53. Challenging this dogma, mouse models have revealed that p53-driven apoptosis and cell cycle arrest are dispensable for tumor suppression. Here, we investigated the inverse context of a p53 mutation predicted to drive the expression of canonical targets but is detected in human cancer. Methods We established a novel mouse model with a single base pair mutation (GAG>GAT, p53E221D) in the DNA-Binding domain that has wild-type function in screening assays, but is paradoxically found in human cancer in Li-Fraumeni syndrome. Using mouse p53E221D and the analogous human p53E224D mutants, we evaluated expression, transcriptional activation, and tumor suppression in vitro and in vivo. Results Expression of human p53E224D from cDNA translated to a fully functional p53 protein. However, p53E221D/E221D RNA transcribed from the endogenous locus is mis-spliced resulting in nonsense-mediated decay. Moreover, fibroblasts derived from p53E221D/E221D mice do not express a detectable protein product. Mice homozygous for p53E221D exhibited increased tumor penetrance and decreased life expectancy compared to p53WT/WT animals. Conclusions Mouse p53E221D and human p53E224D mutations lead to splice variation and a biologically relevant p53 loss of function in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a5ab888592bac1b40fc9b15ac14955958bd526" target='_blank'>
              Mis-splicing drives loss of function of p53E224D point mutation
              </a>
            </td>
          <td>
            Ian C. Lock, Nathan H Leisenring, Warren Floyd, Eric S. Xu, Lixia Luo, Yan Ma, Erin C. Mansell, Diana M. Cardona, Chang-Lung Lee, D. Kirsch
          </td>
          <td>2025-03-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH), characterized by the expansion of hematopoietic stem and progenitor cells harboring somatic mutations, has emerged as a significant age-related phenomenon with profound implications for human health. While initially recognized in the 1960s, recent technological advances have revealed its complex nature and widespread prevalence, affecting up to 84% of individuals aged ≥ 70 years. The clinical significance of CH extends beyond its well-established role as a precursor to hematological malignancies, encompassing its association with cardiovascular diseases, chronic kidney disease, and other non-malignant disorders. This comprehensive review synthesizes the current understanding of CH, focusing on recent advances in genetic and molecular mechanisms, particularly the roles of commonly mutated genes such as DNMT3A, TET2, and ASXL1. We address the emerging distinction between myeloid and lymphoid CH, their differential impacts on disease progression, and the complex interplay between CH and inflammation. Special attention is given to newly identified genetic determinants of clonal expansion rates and their implications for disease progression. The review also examines the revolutionary concept of passenger-approximated clonal expansion rate and its utility in understanding CH dynamics. Furthermore, we discuss therapeutic strategies targeting inflammatory pathways and their potential in mitigating CH-associated complications. By integrating recent findings from genetic, molecular, and clinical studies, this review provides a framework for understanding CH as a systemic condition and highlights promising directions for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7375f0846811e9250a784d2b67a30ac0f6b0da37" target='_blank'>
              Clonal hematopoiesis: elements associated with clonal expansion and diseases
              </a>
            </td>
          <td>
            Gangpyo Ryu, Youngil Koh, S. Jaiswal, S. Yoon
          </td>
          <td>2025-03-13</td>
          <td>Blood Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Significance Estrogen receptor (ER)-positive breast cancer often develops resistance to endocrine therapies like tamoxifen. Understanding the molecular mechanisms underlying both carcinogenesis and drug resistance is critical for improving therapeutic strategies. This study identifies circFOXK2 as a key regulator of CCND1 in ER-positive breast cancer. By stabilizing CCND1 mRNA through RNA–RNA pairing and recruitment of ELAVL1/HuR, circFOXK2 enhances the CCND1–CDK4/6–p-RB–E2F axis, promoting cell proliferation and therapy resistance. Genetic silencing and pharmacological inhibition of circFOXK2 suppress tumor growth and sensitize or restore tamoxifen responsiveness in ER-positive breast cancer. These findings highlight circFOXK2 as a potential therapeutic target for overcoming resistance and improving outcomes in ER-positive breast cancer patients, offering avenues for clinical intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a572f11006892aa8b4e142fead4939f8a023665b" target='_blank'>
              A circRNA–mRNA pairing mechanism regulates tumor growth and endocrine therapy resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Jia Yi, Jiao Du, Xue Chen, Rui-chao Nie, Guo-Sheng Hu, Lei Wang, Yue-ying Zhang, Shang Chen, Xiao-sha Wen, Di-xian Luo, Hua He, Wen Liu
          </td>
          <td>2025-02-18</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recombinant adeno-associated viral vectors (rAAV) are a powerful tool for gene delivery but have a limited DNA carrying capacity. Efforts to expand this genetic payload have focused on engineering the vector components, such as dual trans-splicing vectors which double the delivery size by exploiting the natural concatenation of rAAV genomes in host nuclei. We hypothesized that inefficient dual vector transduction could be improved by modulating host factors which affect concatenation. Since factors mediating concatenation are not well defined, we performed a genome-wide screen to identify host cell regulators. We discover that Homologous Recombination (HR) is inhibitory to dual vector transduction. We demonstrate that depletion or inhibition of HR factors BRCA1 and Rad51 significantly increase reconstitution of a large split transgene by increasing both concatenation and expression from rAAVs. Our results define roles for DNA damage repair in rAAV transduction and highlight the potential for pharmacological intervention to increase genetic payload of rAAV vectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de6693e1cc9fcaaa6e44ac852bdcdb47fc66775d" target='_blank'>
              Double-strand break repair pathways differentially affect processing and transduction by dual AAV vectors
              </a>
            </td>
          <td>
            Anna C Maurer, Brian Benyamini, O. Whitney, Vinson B. Fan, Claudia Cattoglio, Djem U. Kissiov, G. Dailey, X. Darzacq, Matthew D Weitzman, Robert Tjian
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glucose metabolism reprogramming provides significant insights into the development and progression of malignant tumors. This study aims to explore the temporal-spatial evolution of the glucose metabolism in HCC using single-cell sequencing and spatial transcriptomics (ST), and validates G6PD as a potential therapeutic target for HCC.We collected single-cell sequencing data from 7 HCC and adjacent non-cancerous tissues from the GSE149614 database, and ST data from 4 HCC tissues from the HRA000437 database. Pseudotime analysis was performed on the single-cell data, while ST data was used to analyze spatial metabolic activity. High-throughput sequencing and experiments, including wound healing, CCK-8, and transwell assays, were conducted to validate the role and regulatory mechanisms of G6PD in HCC.Our study identified a progressive upregulation of PPP-related genes during tumorigenesis. ST analysis revealed elevated PPP metabolic scores in the central and intermediate tumor regions compared to the peripheral zones. High-throughput sequencing and experimental validation further suggested that G6PD-mediated regulation of HCC cell proliferation, migration, and invasion is likely associated with glutathione metabolism and ROS production. Finally, Cox regression analysis cofirmed G6PD as an independent prognostic factor for overall survival in HCC patients.Our study provides novel insights into the changes in glucose metabolism in HCC from both temporal and spatial perspectives. We experimentally demonstrated that G6PD regulates proliferation, migration, and invasion in HCC and propose G6PD as a prognostic marker and therapeutic metabolic target for the HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d4060fcfc333045f5d44b00ea2e74dde3d1885" target='_blank'>
              Single-cell sequencing and spatial transcriptomics reveal the evolution of glucose metabolism in hepatocellular carcinoma and identify G6PD as a potential therapeutic target
              </a>
            </td>
          <td>
            Deyang Xi, Yinshuang Yang, Jiayi Guo, Mengjiao Wang, Xuebing Yan, Chunyang Li
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc5699f8d775d7c0b21dfb5b4e13161b3a6de2f" target='_blank'>
              Appraisal of CRISPR Technology as an Innovative Screening to Therapeutic Toolkit for Genetic Disorders.
              </a>
            </td>
          <td>
            A. Shahid, Ambreen Zahra, S. Aslam, Amen Shamim, Waqas Rafique Ali, Bilal Aslam, Sultan Habibullah Khan, Muhammad Imran Arshad
          </td>
          <td>2025-02-02</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055214dbdb29cbe5585cc333e89737a58f2ba504" target='_blank'>
              The Epigenetic Landscape: From Molecular Mechanisms to Biological Aging.
              </a>
            </td>
          <td>
            Rachel Evangelina, Subhashree Ganesan, Melvin George
          </td>
          <td>2025-03-17</td>
          <td>Rejuvenation research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5392c951f2de30ee4a1335350f2dfe3bff927f11" target='_blank'>
              Circular RNAs: driving forces behind chemoresistance and immune evasion in bladder cancer.
              </a>
            </td>
          <td>
            M. Saadh, Wael Sheet Hussein, Ali Fawzi Al-Hussainy, A. K. Bishoyi, M. M. Rekha, Mayank Kundlas, V. Kavitha, Zafar Aminov, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq
          </td>
          <td>2025-03-25</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Overexpression of the H3K36 histone methyltransferase NSD2 in t(4;14) multiple myeloma (MM) is an early, oncogenic event, and understanding its impact on genomic organisation and expression is relevant to understanding MM biology. We performed epigenetic, transcriptional and phenotypic profiling of the t(4;14) KMS11 myeloma cell line and its isogenic translocation knock out (TKO) to characterise the sequelae of NSD2 overexpression. We found a marked global impact of NSD2 on gene expression and DNA organisation implicating cell identity genes; notably the early lymphocyte regulator, LAIR1 and MM cell surface markers, including CD38, a classical marker of plasma cells which was reduced in TKO cells. Plasma cell transcription factors such as PRDM1, IRF4 and XBP1 were unaffected, suggesting a downstream direct gene effect of NSD2 on cell identity. Changes in cell surface markers suggest an altered surface immunophenotype. Our findings suggest a role for NSD2 in maintaining MM cell identity, with potential implications for future therapeutic strategies based on targeting of NSD2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3900ea300ad687a680fc22372f0a01df92719adc" target='_blank'>
              NSD2-epigenomic reprogramming and maintenance of plasma cell phenotype in t(4;14) myeloma
              </a>
            </td>
          <td>
            Andrea Gunnell, Scott T. Kimber, R. Houlston, Martin Kaiser
          </td>
          <td>2025-03-21</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e607ac4077bd0584584c78f861a3a32f882b386" target='_blank'>
              Chemical Strategies to Modulate and Manipulate RNA Epigenetic Modifications.
              </a>
            </td>
          <td>
            Liang Cheng
          </td>
          <td>2025-03-18</td>
          <td>Accounts of chemical research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a heterogeneous group of tumor cells that play a significant role in tumorigenesis, therapeutic resistance, and recurrence in liver hepatocellular carcinoma (LIHC). This study combines clinical data sets from The Cancer Genome Atlas (TCGA) and the International Cancer Genome Consortium (ICGC) with bulk RNA sequencing data. This study also features the GSE156625 single-cell RNA sequencing (scRNA) data set from the GEO to explore the prognostic significance of CSC biomarkers (BCSCs) in LIHC. In this research, we introduce a developed prognostic risk model that relies on nine specific BCSCs, including ADM, CCL5, CD274, DLGAP5, HOXD9, IGF1, S100A9, SOCS2, and TNFRSF11B. It was found that high-risk patients experience shorter overall survival rates when compared to low-risk patients. Additionally, the study characterized the composition of immune cells within the tumor microenvironment (TME) and revealed significant variations in gene-expression levels and mutation rates between different risk groups. The model suggests that liver cancer progression might be driven by immune evasion independent of PD-L1 and highlights the potential of the low-risk BCSC group being sensitive to various treatments. Our findings offer a promising foundation for personalized LIHC therapy and highlight the need for further experimental validation of the roles of these CSCs in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccc91ecd266e7bcb8a4e400c1045bb16c558c70" target='_blank'>
              Identification of a Cancer Stem Cell-Related Gene Signature in Hepatocellular Carcinoma Based on Single-Cell RNA-Seq and Bulk RNA-Seq Analysis
              </a>
            </td>
          <td>
            Jing Wu, Xu Liu, Sheng Huang, Wei Liu
          </td>
          <td>2025-03-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer genome alterations often lead to vulnerabilities that can be used to selectively target cancer cells. Various inhibitors of such synthetic lethal targets have been approved by the FDA or are in clinical trials, highlighting the potential of this approach1–3. Here we analysed large-scale CRISPR knockout screening data from the Cancer Dependency Map and identified a new synthetic lethal target, PELO, for two independent molecular subtypes of cancer: biallelic deletion of chromosomal region 9p21.3 or microsatellite instability-high (MSI-H). In 9p21.3-deleted cancers, PELO dependency emerges from biallelic deletion of the 9p21.3 gene FOCAD, a stabilizer of the superkiller complex (SKIc). In MSI-H cancers, PELO is required owing to MSI-H-associated mutations in TTC37 (also known as SKIC3), a critical component of the SKIc. We show that both cancer subtypes converge to destabilize the SKIc, which extracts mRNA from stalled ribosomes. In SKIc-deficient cells, PELO depletion induces the unfolded protein response, a stress response to accumulation of misfolded or unfolded nascent polypeptides. Together, our findings indicate PELO as a promising therapeutic target for a large patient population with cancers characterized as MSI-H with deleterious TTC37 mutations or with biallelic 9p21.3 deletions involving FOCAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/196e4f5c9c9b8a27230e96850b6632f862c0dcae" target='_blank'>
              SKI complex loss renders 9p21.3-deleted or MSI-H cancers dependent on PELO
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony C Lau, Lucy Parker-Burns, Daniel A Lubicki, Tianxia Li, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Tessla Chan, Riya V Kishen, Anisah Adeagbo, Srivatsan Raghavan, Elisa Aquilanti, John R. Prensner, J. Krill-Burger, Todd R. Golub, Catarina D. Campbell, Joshua M Dempster, Edmond M Chan, Francisca Vazquez
          </td>
          <td>2025-02-05</td>
          <td>Nature</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3095fb742b0c6045977fafeab8ed1d06964edc6e" target='_blank'>
              FXYD5 regulates gastric cancer cell metastasis and drug resistance by EMT modulation.
              </a>
            </td>
          <td>
            Yuning Mao, Yaohua Hu, Han Meng, Jing Qin, Qingling An, Caiqin Zhang, Chenbo Guo, Yong Zhao, Dengxu Tan, Xu Ge, Changhong Shi
          </td>
          <td>2025-02-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab9164c2b57024d5a00b47fc6698b2e2fcdfba04" target='_blank'>
              Mapping the Future: A Content Analysis of the Evolution of Gene Therapy in Urological Cancer
              </a>
            </td>
          <td>
            Muhammad Sidharta Krisna, Muhammad Alfi Reza, Bobby Aksanda Putra, Bukhari Muslim Siregar, Muhammad Alif Adhani
          </td>
          <td>2025-02-14</td>
          <td>International Journal of Health and Pharmaceutical (IJHP)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Melanoma is the deadliest form of skin cancer, and its treatment poses significant challenges due to its aggressive nature and resistance to conventional therapies. Long non-coding RNAs (lncRNAs) represent a new frontier in the search for suitable targets to control melanoma progression and invasiveness. Indeed, lncRNAs exploit a wide range of regulatory functions along chromatin remodeling, gene transcription, post-transcription, transduction, and post-transduction to ultimately tune multiple cellular processes. The understanding of this intricate and flexible regulatory network orchestrated by lncRNAs in pathological conditions can strategically support the rational identification of promising targets, ultimately speeding up the setup of new therapeutics to integrate the currently available approaches. Here, the most recent findings on lncRNAs involved in melanoma will be analyzed. In particular, the functional links between their mechanisms of action and some frequently underestimated features, like their different subcellular localizations, will be highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08119f9fccb5c4f5a79dd1ca1781581cce224f1" target='_blank'>
              Unravelling the Regulatory Roles of lncRNAs in Melanoma: From Mechanistic Insights to Target Selection
              </a>
            </td>
          <td>
            Beatrice Moras, C. Sissi
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most common cancer and the second leading cause of cancer-related deaths worldwide. Despite treatment advancements in the last decades, CRC remains heterogeneous with significant clinical and genetic diversity. Human epidermal growth factor receptor 2 (HER2) proto-oncogene plays a critical role, as its amplification or overexpression leading to abnormal cell proliferation and tumorigenesis. HER2 overexpression or amplification is identified in 2-4% of metastatic CRCs (mCRC) patients, representing a potential therapeutic target. It is also associated with resistance against epidermal growth factor receptor (EGFR)-targeted therapies like cetuximab and panitumumab, for treatment of RAS wild-type mCRC. Although HER2-positive mCRC is rare, assessing HER2 levels is important. Furthermore, anti-HER2 therapies exhibited non-toxic profile and high efficacy in chemorefractory cases. This review delves into modern management of anti-HER2 therapies in CRC with a particular focus on recent advances and current knowledge about the prognostic and predictive value of HER2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a366c7d16dafc61bb9a48ebda7fc255bd8a1ee2d" target='_blank'>
              HER2-targeted therapy in colorectal cancer: a comprehensive review.
              </a>
            </td>
          <td>
            Yeliz Benli, Helin Arıkan, Özge Akbulut-Çalışkan
          </td>
          <td>2025-03-14</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c5574ba347be751b5260edd71a2fcb0e3909c08" target='_blank'>
              Integrative single-cell and bulk RNA-seq analysis identifies lactylation-related signature in osteosarcoma.
              </a>
            </td>
          <td>
            Zhou Xie, Xiao Qu, Jun Zhang, Yanran Huang, Runhan Zhao, Dagang Tang, Ningdao Li, Zhule Wang, Xiaoji Luo
          </td>
          <td>2025-03-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f316939d905dbc2373c0385e4fa8591bd033217e" target='_blank'>
              Current efficacy of immune checkpoint inhibitors in microsatellite unstable colorectal cancer and potential biomarkers
              </a>
            </td>
          <td>
            Mariam Rojas, Clara Rodrigo, Reinaldo Moreno, Marta Cascante, Joan Maurel
          </td>
          <td>2025-02-18</td>
          <td>Exploration of Digestive Diseases</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background High transcription levels are essential for cancer cells to maintain their malignant phenotype. While RNA polymerases (POLRs) have been implicated in various transcriptional mechanisms, their impact on the tumor microenvironment (TME) remains poorly understood. Methods We analyzed publicly available pan-cancer cohorts to evaluate the expression and genomic alterations of POLRs. Focusing on head and neck squamous cell carcinoma (HNSC), we integrated bulk RNA sequencing, single-cell, and spatial transcriptome data to identify POLR2C expression patterns and its potential regulation by Yin Yang 1 (YY1). In vitro and in vivo experiments were conducted to validate the functional role of the YY1-POLR2C axis in cancer proliferation and immune modulation. Results POLRs were found to be aberrantly expressed in cancers and associated with genomic alterations. In HNSC, POLR up-regulation was linked to poor prognostic features. POLR2C was significantly up-regulated in malignant cells, and its expression appeared to be transcriptionally regulated by YY1. Functional studies demonstrated that the YY1-POLR2C axis drives cell-cycle dysregulation and malignant proliferation in HNSC. Additionally, high POLR expression negatively correlated with immune cell infiltration and facilitated immune evasion. Mechanistically, POLRs mediated frequent interactions between malignant and immune cells, potentially contributing to resistance to immunotherapy. Conclusion This study highlights the dual role of POLRs in promoting malignant proliferation and shaping an immunosuppressive TME. POLR2C, regulated by YY1, emerges as a critical mediator in HNSC and a promising target for precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5caf5c8af345e8c1d8ae660db4ba00bbab580435" target='_blank'>
              Comprehensive Analysis of Multi-Omics Data on RNA Polymerase as an Adverse Factor in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Yue Gu, Jie Zhang, Yuan Liu, Qian Zhang, Qi-Feng Geng
          </td>
          <td>2025-03-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Pseudogene‐derived lncRNAs are widely dysregulated in cancer. Technological advancements have facilitated the functional characterization of increasing pseudogenes in cancer progression. However, the association between pseudogenes and RNA N6‐methyladenosine (m6A) modification in cancer, as well as the underlying mechanisms, remains largely unexplored. Methods We analyzed the expression of 12 146 pseudogenes and comprehensively examined the m6A modification of RNAs derived from them and their paralogs. Through integrative analysis of multi‐omics data, we explored the associations between pseudogene dysregulation and m6A, identifying critical pseudogenes involved in HGSOC progression. Tumour promotion role of RPS15AP12 and its cognate parent gene was characterized by cell proliferation, transwell assays, and scratch assays in ovarian cells and xenograft nude mice. RNA decay assays were used to reveal the participation of m6A in decreasement of RPS15AP12 lncRNA stability. Luciferase reporter assays were performed to verify that RPS15AP12 enhances RPS15A expression by competitively binding to miR‐96‐3p. Western blot and phosphorylation assays were performed to investigate the impairment of RPS15AP12 towards the sensors of MAVS (RIG‐I and MDA5), and downstream p‐TBK1 and p‐IRF3. Finally, ELISA assays were performed to explore the regulatory role of RPS15AP12 in IFN‐β expression. Results M6A is distributed across over a thousand pseudogenes, and hypomethylation leads to their upregulation in HGSOC. We identified a processed pseudogene, RPS15AP12, upregulated by FTO‐mediated m6A demethylation. RPS15AP12 enhances the growth ability and metastatic capabilities of ovarian cancer (OC) cells via functioning as a competitive endogenous RNA (ceRNA) for its host gene, RPS15A, through the sequestration of miR‐96‐3p. Importantly, the deletion of RPS15AP12 diminishes the expression of RPS15A, leading to the upregulation of anti‐tumour immune responses by activating RIG‐I and MDA5 and downstream p‐TBK1 and p‐IRF3 as well as IFN‐β levels. Conclusion Our findings expand the understanding of m6A‐modulated pseudogenes in tumour growth and anti‐tumour innate immunity in OC. Key Points Genome‐wide profiling reveals the redistribution of m6A modification on pseudogene‐derived lncRNAs and m6A redistribution‐relevant dysregulation of pseudogenes in HGSOC. RPS15AP12, as a representative processed pseudogene, is up‐regulated by FTO‐mediated demethylation and acts as a miRNA sponge to promote RPS15A expression via competitively binding to miR‐96‐3p. RPS15AP12/RPS15A axis inhibits MAVS sensors (RIG‐I and MDA5) and downstream IFN‐β levels in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f32a6bc1ac1ae8503c9d4c32bcdde0ae2bb380" target='_blank'>
              Genome‐wide profiling of N6‐methyladenosine‐modified pseudogene‐derived long noncoding RNAs reveals the tumour‐promoting and innate immune‐restraining function of RPS15AP12 in ovarian cancer
              </a>
            </td>
          <td>
            Jie Xu, Yifei Ren, Jiayi Lu, Fengjiang Qin, Dan Yang, Chunyan Tang, Yu Yang, Jing Xu, Tao Liu, Ping Yi
          </td>
          <td>2025-02-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Activating PIK3CA mutations, present in up to 40% of hormone receptor-positive (HR+), human epidermal growth factor receptor 2-negative (Her2−) breast cancer (BC) patients, can be effectively targeted with the alpha isoform-specific PI3K inhibitor Alpelisib. This treatment significantly improves outcomes for HR+, Her2−, and PIK3CA-mutated metastatic BC patients. However, acquired resistance, often due to aberrant activation of the mTOR complex 1 (mTORC1) pathway, remains a significant clinical challenge. Our study, using in vitro and orthotopic xenograft mouse models, demonstrates that constitutively active mTORC1 signaling renders PI3K inhibitor-resistant BC exquisitely sensitive to various drugs targeting cancer metabolism. Mechanistically, mTORC1 suppresses the induction of autophagy during metabolic perturbation, leading to energy stress, a critical depletion of aspartate, and ultimately cell death. Supporting this mechanism, BC cells with CRISPR/Cas9-engineered knockouts of canonical autophagy genes showed similar vulnerability to metabolically active drugs. In BC patients, high mTORC1 activity, indicated by 4E-BP1T37/46 phosphorylation, correlated with p62 accumulation, a sign of impaired autophagy. Together, these markers predicted poor overall survival in multiple BC subgroups. Our findings reveal that aberrant mTORC1 signaling, a common cause of PI3K inhibitor resistance in BC, creates a druggable metabolic vulnerability by suppressing autophagy. Additionally, the combination of 4E-BP1T37/46 phosphorylation and p62 accumulation serves as a biomarker for poor overall survival, suggesting their potential utility in identifying BC patients who may benefit from metabolic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69e4dbaba941d1cd07858705c4708f9bc19fb8f" target='_blank'>
              Targeting PI3K inhibitor resistance in breast cancer with metabolic drugs
              </a>
            </td>
          <td>
            N. Gremke, I. Besong, Alina Stroh, Luise von Wichert, Marie Witt, S. Elmshäuser, M. Wanzel, Martin F Fromm, R. V. Taudte, S. Schmatloch, T. Karn, M. Reinisch, N. Hirmas, S. Loibl, Thomas Wündisch, Anne-Sophie Litmeyer, Paul Jank, Carsten Denkert, Sebastian Griewing, U. Wagner, Thorsten Stiewe
          </td>
          <td>2025-03-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background In non-small cell lung cancer (NSCLC), the rapid advancement of predictive genetic testing of tumors by identifying specific pathogenic driver variants has significantly improved treatment guidance. However, immune checkpoint blockade (ICB) is typically administered to patients with tumors in the absence of such driver variants. Since only about 30% of patients will respond to ICB treatment, identifying novel genetic biomarkers of clinical response is crucial and will improve treatment decisions. This prospective clinical study aims to combine molecular biology, advanced bioinformatics and clinical data on response to treatment with ICB from a prospective cohort of NSCLC patients to identify single or combination of genetic variants in the tumor that can serve as predictive biomarkers of clinical response. Methods In this prospective bi-center clinical study, we performed next-generation sequencing (NGS) of 597 cancer-associated genes in a prospective cohort of 49 patients as the final cohort analyzed, with stage III or IV NSCLC, followed by establishment of an in-house developed bioinformatics-based molecular classification method that integrates, interprets and evaluates data from multiple databases and variant prediction tools. Overall survival (OS) and progression-free survival (PFS) were analyzed for selected candidate genes and variants identified using our novel methodology including molecular tools, databases and clinical information. Results Our novel molecular interpretation and classification method identified high impact variants in frequently altered genes KRAS, LRP1B, and TP53. Analysis of these genes as single predictive biomarkers in ICB-treated patients revealed that the presence of likely pathogenic variants and variants of unclear significance in LRP1B was associated with improved OS (p = 0.041). Importantly, further analysis of variant combinations in the tumor showed that co-occurrence of KRAS and LRP1B variants significantly improved OS (p = 0.003) and merged PFS (p = 0.008). Notably, the triple combination of variants in KRAS, LRP1B, and TP53 positively impacted both OS (p = 0.026) and merged PFS (p = 0.003). Conclusions This study suggests that combination of the LRP1B and KRAS variants identified through our novel molecular classification scheme leads to better outcomes following ICB treatment in NSCLC. The addition of TP53 improves the outcome even further. To our knowledge, this is the first report indicating that harboring a combination of KRAS, LRP1B, and TP53 variants can significantly enhance the response to ICB, suggesting a novel predictive biomarker combination for NSCLC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03342-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2008a33bf46a0804b91c89a973e1bb2840eaa1e" target='_blank'>
              Comprehensive genetic variant analysis reveals combination of KRAS and LRP1B as a predictive biomarker of response to immunotherapy in patients with non-small cell lung cancer
              </a>
            </td>
          <td>
            E. Eklund, Johanna Svensson, Louise Stauber Näslund, Maria Yhr, S. Sayin, C. Wiel, Levent M Akyürek, Per Torstensson, V. Sayin, A. Hallqvist, Sukanya Raghavan, A. Rohlin
          </td>
          <td>2025-02-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rationale: Non-small cell lung cancer (NSCLC) is a predominant cause of cancer-related mortality, with its progression and treatment resistance significantly influenced by cancer stem cells (CSCs) and their complex intercellular communication mechanisms. Small extracellular vesicles (sEVs) have emerged as pivotal mediators of intercellular signaling, affecting tumor microenvironment modulation and therapeutic resistance. This study investigates the role of CSC-derived sEVs in transmitting stemness traits through the selective sorting of pyruvate kinase M2 phosphorylated at the Y105 site (pY105-PKM2), mediated by the adaptor protein IQGAP1, which supports CSC maintenance and drug resistance in NSCLC. Methods: In vitro and in vivo experiments, including proteomic and transcriptomic analyses, were conducted to identify key regulators of sEV-mediated signaling. Immunoprecipitation, proximity ligation assays, and immunofluorescence were used to examine the role of IQGAP1 in the sorting of pY105-PKM2 into sEVs. Functional assays, including sphere formation, chemoresistance tests, metabolic assessments, and cell cycle analysis, were conducted to evaluate the effects of sEV-mediated delivery of pY105-PKM2 on recipient cells. Additionally, immunohistochemistry and survival analysis were performed on tumor samples from NSCLC patients to establish clinical correlations. Results: We unveiled a novel mechanism by which CSC-derived sEVs transmit stemness traits to replenish the CSC pool in NSCLC. CSC-derived sEVs were enriched with pY105-PKM2, correlating with enhanced stemness, chemoresistance, and poor clinical outcomes. Mechanistically, IQGAP1 was identified as an adaptor facilitating the selective sorting of pY105-PKM2 into sEVs through interactions with the ESCRT component TSG101. Recipient cells treated with CSC-derived sEVs exhibited metabolic reprogramming, slower cell cycle progression, and enhanced chemoresistance. The synergistic role of IQGAP1 and pY105-PKM2 was confirmed, highlighting their critical contributions to CSC maintenance and malignant progression. Conclusion: This study highlights the critical role of CSC-derived sEVs in NSCLC progression and therapy resistance through the IQGAP1-mediated selective sorting of pY105-PKM2. By uncovering this novel pathway, our findings provide valuable insights into CSC pool replenishment and therapeutic resistance mechanisms in NSCLC, identifying IQGAP1 and pY105-PKM2 as promising therapeutic targets for mitigating CSC-driven malignancy and enhancing treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8844821c21b603caec632dd4e93148e05d6265" target='_blank'>
              A novel pathway for stemness propagation and chemoresistance in non-small cell lung cancer via phosphorylated PKM2-loaded small extracellular vesicles
              </a>
            </td>
          <td>
            Jingyi Wang, Liu Liu, Xinyu Gao, Xiyu Liu, Yitian Dai, Zijun Mao, Shengzhe Huang, Junjian Li, Dongliang Wang, Yu Qi, Yingwen Han, Yunjing Xu, C. Y. Chua, A. Grattoni, Wenhui Xie, Hao Yang, Gang Huang
          </td>
          <td>2025-02-24</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Bacteriophages constitute one of the largest reservoirs of genes of unknown function in the biosphere. Even in well-characterized phages, the functions of most genes remain unknown. Experimental approaches to study phage gene fitness and function at genome scale are lacking, partly because phages subvert many modern functional genomics tools. Here we leverage RNA-targeting dCas13d to selectively interfere with protein translation and to measure phage gene fitness at a transcriptome-wide scale. We find CRISPR Interference through Antisense RNA-Targeting (CRISPRi-ART) to be effective across phage phylogeny, from model ssRNA, ssDNA and dsDNA phages to nucleus-forming jumbo phages. Using CRISPRi-ART, we determine a conserved role of diverse rII homologues in subverting phage Lambda RexAB-mediated immunity to superinfection and identify genes critical for phage fitness. CRISPRi-ART establishes a broad-spectrum phage functional genomics platform, revealing more than 90 previously unknown genes important for phage fitness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d283ad1ac40041b683846c272f3ed7e85e462ab" target='_blank'>
              CRISPRi-ART enables functional genomics of diverse bacteriophages using RNA-binding dCas13d
              </a>
            </td>
          <td>
            Benjamin A. Adler, Muntathar J. Al-Shimary, Jaymin R. Patel, Emily G. Armbruster, David Colognori, Emeric Charles, Kate V Miller, Arushi Lahiri, Michael Cui, Agnès Oromí-Bosch, Angela Voelker, M. Trinidad, Jina Lee, Sebastien Beurnier, Ron S. Boger, Jason Nomburg, R. Barrangou, Vivek K. Mutalik, J. Schoeniger, Joseph A. Pogliano, David F. Savage, Jennifer A. Doudna, Brady F. Cress
          </td>
          <td>2025-02-26</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing.
              </a>
            </td>
          <td>
            Qiuhui Li, Ayse G Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb803b3e058115ac9f37e42f7b7a8e8748c0c02" target='_blank'>
              Next-Generation Sequencing-Based In Silico Transgene Integration Profiling Tool Accelerates Cell Line Genetic Characterizations.
              </a>
            </td>
          <td>
            Zhenqiu Huang, Hyo-Young Jeong, Luke Nelson, Caitlyn Devine, Guanghua Benson Li, Lin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Biotechnology journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain- and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Comprehensive and systematic proteome-wide experiments require financial and technical resources unavailable to most researchers. Here we describe a scalable CRISPR/Cas9-non-homologous end joining (NHEJ) based method for Pooled Recombinant Integration of Seamless Markers (PRISM) into protein coding genes. We created two gRNA libraries for 5’- and 3’-tagging of 18,804 human protein-coding genes. Selection for in-frame integration of the donor cassette can be guaranteed by fusing it to an antibiotic resistance enzyme (ARE) and P2A self-cleaving peptide, resulting in tagged proteins and free ARE to select for clones with inserted donor cassettes. We achieved a library integration rate of 19.75% and tagging of ∼80% for genes expressed in Hek293T cells and notably, 89.7% of essential genes, with donor DNA. Our strategy is scalable, specific, and selective, paving the way for genome-scale construction of human cell lines tagged with different types of reporter genes for protein functional characterization. Significance statement Here we report the first practical method to achieve near complete 5’- or 3’-end integration of reporter protein-coding sequences to express seamless fusions of human protein and reporter proteins that we call Pooled Recombinant Integration of Seamless Markers (PRISM). PRISM is independent of homology templates, yet it works with high efficiency and precision. Using gRNA-directed CRISPR/Cas9 genome editing and mini-plasmid donor cassettes we were able to tag about 80% of the protein-coding sequences of human genes with a green florescent protein at an integration efficiency of about 20% and most notably, of 90% of essential genes in the human cell line HEK 293T. This is important because the characterization of gene functions in a particular cell type usually begins with essential genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834136786fe808caa704ff04cd6e336c885cd2d" target='_blank'>
              A strategy for genome-wide seamless tagging of human protein-coding genes
              </a>
            </td>
          <td>
            Minoo Karimi, Masih M. Saber, Nicolas Stifani, Louis Gauthier, Adrian Serohijos, S. Michnick
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Lentiviral vector-transduced T cells were approved by the FDA as gene therapy anti-cancer medications. Little is known about the effects of host genetic variation on the safety and efficacy of the lentiviral vector gene delivery system. To narrow this knowledge gap, we characterized hepatic gene delivery by lentiviral vectors across the Collaborative Cross (CC) mouse genetic reference population. For 24 weeks, we periodically measured hepatic luciferase expression from lentiviral vectors in 41 CC mouse strains. Hepatic and splenic vector copy numbers were determined. We report that the CC mouse strains showed highly diverse outcomes following lentiviral gene delivery. For the first time, a moderate correlation between mouse-strain-specific sleeping patterns and transduction efficiency was observed. We associated two quantitative trait loci (QTLs) with intrastrain variations in transduction phenotypes, which mechanistically relates to the phenomenon of metastable epialleles. An additional QTL was associated with the kinetics of hepatic transgene expression. Genes found in the above QTLs are potential targets for personalized gene therapy protocols. Importantly, we identified two mouse strains that open new directions for characterizing continuous viral vector silencing and HIV latency. Our findings suggest that wide-range patient-specific outcomes of viral vector-based gene therapy should be expected. Thus, novel clinical protocols should be considered for non-fatal diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4970be4c0655da468396fb187c5303cf10c27ee8" target='_blank'>
              Analysis of Hepatic Lentiviral Vector Transduction: Implications for Preclinical Studies and Clinical Gene Therapy Protocols
              </a>
            </td>
          <td>
            Peirong Hu, Yajing Hao, Wei Tang, Graham H. Diering, Fei Zou, T. Kafri
          </td>
          <td>2025-02-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Epithelial Ovarian Cancer (EOC) is the most lethal gynecologic cancer and those affected are in urgent need of new therapeutic strategies. Standard treatment is surgery followed by taxane- and platinum-based chemotherapy, but the rate of relapse is high and the 5-year survival is only 45%. Oncolytic viruses (OV) are a promising approach to EOC therapy through remodeling the immune composition of the tumor microenvironment (TME). Treatment response in EOC tumors can differ based on the presence of key tumorigenic mutations. This study evaluated the impact of specific tumor mutations on the response to the current standard of care carboplatin, two promising OV candidates VSV∆M51 and MG1, an infected cell vaccine (ICV-MG1) regimen, and the anti-angiogenic drug Fc3TSR. Mice with tumors harboring constitutive K-Ras activation showed an enhanced response to carboplatin and VSV∆M51 treatment. Additionally, VSV∆M51 treatment prolonged survival of syngeneic mice bearing tumors with mutations in Pten and Kras, Pten and Trp53, or Trp53 and Brca2 with increased activation of CD4+ and CD8+ T lymphocytes in the peritoneal TME. To enhance OV potency, an MG1-based infected cell vaccine inducing expression of IL-21 or IL15+21 was developed and found to enable strong and long-lasting antitumoral immunity in two carboplatin refractory syngeneic models, ID8 Trp53-/- and STOSE. VSV∆M51 combined with the anti-angiogenic Fc3TSR enhanced efficacy in the ID8 model. In summary, OV-based immunotherapy has shown promise in diverse murine models of EOC-bearing clinically relevant mutations, thus laying the foundation for developing new OV-based strategies to target a large spectrum of EOC genotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0be8c226447376238aa375a0e7bc3031f54f154" target='_blank'>
              Specific Genetic Mutations Impact Chemotherapy Resistance and Therapeutic Efficacy of Oncolytic Viruses in Ovarian Cancer.
              </a>
            </td>
          <td>
            Alison O Cudmore, G. Rodriguez, Vincent Maranda, Salar Farokhi Boroujeni, Humaira Murshed, Elizabeth A Macdonald, Melanie Grondin, K. Garson, K. Matuszewska, J. Diallo, James J Petrik, B. Vanderhyden
          </td>
          <td>2025-03-10</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065f2fe139f12cdb2cce92471f4bfcdb7c6fdb7" target='_blank'>
              Transgenesis in Drug Discovery: Enhancing Target Identification and Validation.
              </a>
            </td>
          <td>
            Rumela Mete, Sourav Das, Arindam Saha, Sukanta Roy, Smritilekha Mondal, A. Bose, Biswajit Basu, Gehan M. Elossaily, B. Prajapati
          </td>
          <td>2025-03-27</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff7398d28009d21e0ca4b313f04557fbbf4b820" target='_blank'>
              High Tumor Mutation Burden (TMB) and a Novel Somatic Mutation in the TREX1 Gene in a Patient with Aggressive and Refractory High-Grade B-Cell Lymphoma: A Case Report
              </a>
            </td>
          <td>
            Mariia S. Gusakova, F. Sharko, Eugenia Boulygina, N. Slobodova, M. Gladysheva-Azgari, Darima Badmazhapova, Artem Bullikh, Marina Khestanova, N. Gabeeva, T. Obukhova, Eugene Zvonkov, Svetlana Tsygankova
          </td>
          <td>2025-03-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise approximately half of eukaryotic genomes and significantly contribute to genome plasticity. In this study, we focused on a specific TE, MERVL, which exhibits particular expression during the 2-cell stage and commonly serves as an indicator of embryonic totipotency. However, its precise role in embryo development remains mysterious. We utilized DRUG-seq to investigate the effects of oxidative damage on genes and TEs expression. Our findings revealed that exposure to hydrogen peroxide (H2O2) could induce DNA damage, apoptosis, and incomplete DNA demethylation in embryos, which were potentially associated with MERVL expression. To further explore its function, antisense nucleotides (ASO) targeting MERVL were constructed to knockdown the expression in early embryos. Notably, this knockdown led to the occurrence of DNA damage and apoptosis as early as the 2-cell stage, consequently reducing the number of embryos that could progress to the blastocyst stage. Moreover, we discovered that MERVL exerted an influence on the reprogramming of embryonic DNA methylation. In MERVL-deficient embryos, the activity of the DNA demethylase ten-eleven translocation 3 (TET3) was suppressed, resulting in impaired demethylation when compared to normal development. This impairment might underpin the mechanism that impacts embryonic development. Collectively, our study not only verified the crucial role of MERVL in embryonic development but also probed its regulatory function in DNA methylation reprogramming, thereby laying a solid foundation for further investigations into MERVL's role. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01143-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ed27bb6383161df0bd1281e88568674842389e" target='_blank'>
              Transposition element MERVL regulates DNA demethylation through TET3 in oxidative-damaged mouse preimplantation embryos
              </a>
            </td>
          <td>
            Lihong Liu, Siyao Ha, Dan Cao, MingQing Li, Zhiling Li
          </td>
          <td>2025-03-12</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="T-cell large granular lymphocytic leukemia (T-LGLL) is a rare hematologic neoplasm characterized by clonal expansion of CD3 + cytotoxic T lymphocytes and a highly heterogeneous clinical course. Conventional therapy primarily includes immunosuppressive regimen. However, optimal front-line approaches still need to be defined and refractory disease remains a clinical challenge. Thus, we here aimed to explore functional dependencies of T-LGLL as a basis for personalized therapeutic strategies. We performed functional apoptosis profiling and ex vivo drug treatment in a series of 8 clinically and genetically characterized T-LGLL patients from two German University hospitals. Our series of patients underscored the clinical and genetic heterogeneity of the disease. Genetically, only 2 patients harbored a STAT3 mutation. To identify targetable anti-apoptotic mechanisms, we performed selective functional BH3 profiling on the patients’ CD8 + T-cells harboring the malignant T-LGLL cells versus the same patients’ normal CD4 + T-cells. CD8 + cells in 50% of the patients (4/8) demonstrated a dominant functional dependence on MCL-1 as compared to the same patients’ normal T-cells. Accordingly, CD8 + T-LGLL cells from patients with enhanced MCL1 dependence significantly responded to AZD-5991 ex vivo while no response was observed in the remaining samples lacking enhanced MCL-1 dependence. Across clinically and genetically heterogeneous cases of T-LGLL, functional apoptosis profiling identified patients with CD8 + T-LGLL cells harboring a dominant dependence on MCL-1 as a potential therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1007/s00277-025-06230-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f9719dd384f2308e83ceab545b5fdb3fcb41a6" target='_blank'>
              Functional apoptosis profiling reveals vulnerabilities in T-cell large granular lymphocytic leukemia
              </a>
            </td>
          <td>
            Evgenii Shumilov, P. Mazzeo, M. Trautmann, Lena Levien, Kerstin Menck, Katharina Richter, Katharina Markus, Lena Ries, Detlef Haase, Elena Oberle, Philipp Berning, Wolfgang Hartmann, Philipp Ströbel, Andrea Kerkhoff, Georg Lenz, Gerald Wulf, Raphael Koch
          </td>
          <td>2025-02-06</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739d0a74d9a81f832abc4486254c5ebe8688f593" target='_blank'>
              Abstract A018: Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain
              </a>
            </td>
          <td>
            N. Dasyam, Yasmin Nouri, J. Halpin, A. McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, M. Templin, Qin Le, Peng Li, I. Hermans, Rachel Perret, R. Weinkove
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b18df3c001a51eac902f935e05595c844edc904" target='_blank'>
              Characterization of DNA methylation profile of the entire CpG island spanning the 5' untranslated region to intron 1 of the Oct4/POU5F1 gene in bovine gametes, embryos, and somatic cells.
              </a>
            </td>
          <td>
            Amanda Oliveira Moura, Thainara Christie Ferreira Silva, Alexandre Rodrigues Caetano, N. R. Kussano, M. Dode, Maurício Machaim Franco
          </td>
          <td>2025-02-01</td>
          <td>Animal genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Electroporation is an efficient method for nucleotide and protein transfer, and is used for clustered regularly interspaced short palindromic repeat (CRISPR)-associated protein 9 (Cas9)-mediated genome editing. In this study, we investigated the effects of electroporation on platelet-derived growth factor receptor alpha (PDGFRA) and receptor tyrosine kinase (RTK) expression in U-251 and U-87 MG cells. PDGFRA mRNA and protein expression decreased 2 days after electroporation in both cell lines, with recovery observed after 13 days in U-87 MG cells. However, in U-251 MG cells, PDGFRα expression remained suppressed, despite mRNA recovery after 13 days. Similar expression profiles were observed for lipofection in the U-251 MG cells. Comprehensive RNA sequencing confirmed electroporation-induced up- and down-regulation of RTK mRNA in U-251 MG cells 2 days post-electroporation. In contrast, recombinant adeno-associated virus (rAAV) transfected with mNeonGreen fluorescent protein or Cas9 did not affect PDGFRA, RTKs, or inflammatory cytokine expression, suggesting fewer adverse effects of rAAV on U-251 MG cells. These findings emphasize the need for adequate recovery periods following electroporation or the adoption of alternative methods, such as rAAV transfection, to ensure the accurate assessment of CRISPR-mediated gene editing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a20134832925c18b077d30903fdc286075431d4" target='_blank'>
              Electroporation Induces Unexpected Alterations in Gene Expression: A Tip for Selection of Optimal Transfection Method
              </a>
            </td>
          <td>
            Taiji Hamada, Seiya Yokoyama, T. Akahane, Kei Matsuo, I. Kitazono, Tatsuhiko Furukawa, Akihide Tanimoto
          </td>
          <td>2025-01-31</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e8bbeec84df302b9adf6fafec7b5584f0a9749" target='_blank'>
              Advancement of Gene Therapy in Cancer
              </a>
            </td>
          <td>
            Kahkashan Sultana, Ayesha Fatima
          </td>
          <td>2025-03-15</td>
          <td>Journal of Advanced Scientific Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185bf82df0621d29d2214d3aa74691b2937269d4" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The ataxia-telangiectasia mutated (ATM) protein kinase plays a critical role in activating the cellular response to DNA double-strand breaks and promoting homology-directed repair. ATM is frequently mutated in cancer, contributing to an accumulation of DNA damage that drives genomic instability. To exploit cancer cells’ inherent vulnerability to DNA damage, various small molecule inhibitors have been developed that target ATM. ATM inhibitors have shown great versatility in preclinical studies and increasing use in the clinic. Here, we review the development of ATM inhibitors and their role in cancer therapy. We describe their limitations and the advances that have led to increases in both the number and diversity of active clinical trials targeting ATM. We also discuss ATM’s role in personalized medicine and the current challenges to more widespread use of ATM inhibitors in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1ae4df6ed1675fe0ee6d9bf7493ed3c92eb628" target='_blank'>
              The Development of ATM Inhibitors in Cancer Therapy
              </a>
            </td>
          <td>
            Elizabeth A. Ampolini, Judit Jimenez-Sainz, David T Long
          </td>
          <td>2025-03-01</td>
          <td>Targeted Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objectives/Goals: Investigating the B-cell acute lymphoblastic leukemia (B-ALL)-associated germline SNP rs7090445, located in intron 3 of ARID5B, which is more frequently observed in individuals of Hispanic/Latino descent. Investigating the mechanisms behind this inherited single nucleotide polymorphisms (SNP) that may contribute to the higher incidence of B-ALL in this population. Methods/Study Population: Specific Aim 1: We hypothesize ARID5B SNP rs7090445 disrupts intrinsic enhancer function. Identification of critical DNA looping events impacted by ARID5B variants using Capture C. Affinity purification-mass spectrometry to identify potential ARID5B transcription mediators. Specific Aim 2: We hypothesize the B-ALL-associated SNP leads to a partial human B-cell differentiation block. Utilize Cas9-mediated homology-directed repair to create ARID5B SNP in primary human hematopoietic stem cells. Gene-edited HSCs will be differentiated into B cells using an ex vivo system. Fluorescence-activated cell sorting to sort our pool of cells into stages of B-cell development spectrum. Amplicon sequencing and variant allele frequency of rs7090445 SNP to evaluate its impact on B-cell development. Results/Anticipated Results: This proposal is conceptually innovative as it seeks to understand the mechanism by which the B-ALL-associated SNP rs7090445 in intron 3 of ARID5B disrupts enhancer function and investigates its impact on human B-cell development. Future research will investigate a tumor-suppressive role of ARID5B and whether it constitutes a “first-hit” of leukemogenesis. Discussion/Significance of Impact: Successful completion of this research will elucidate the critical role of the B-ALL-associated ARID5B SNP rs7090445 in human B-cell development and leukemogenesis. As this SNP is more prevalent in Hispanic/Latino populations, it will also provide crucial insights into the genetic factors behind the elevated incidence of B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4345d4a2e4bf5156f2152b887906b55727b5a5a1" target='_blank'>
              71 Investigating gene regulatory mechanisms associated with B-cell acute lymphoblastic leukemia incidence in Hispanic/Latino populations
              </a>
            </td>
          <td>
            Clarissa Garcia, Julian Grandvallet, Illaria Iacobucci, Jun Yang, C. Mullighan, Tzu Phang, Matthew T. Witkowski
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Genetic profiling and molecular biology methods have made it possible to study the etiology of the end-stage organ disease that led to transplantation, the genetic factors of compatibility and tolerance of the transplant, and the pharmacogenetics of immunosuppressive drugs and allowed for the development of monitoring methods for the early assessment of allograft rejection. This study aims to report the design and baseline characteristics of an integrated personalized genetic approach in solid organ transplantation, including whole-exome sequencing (WES) and the monitoring of dd-cfDNA by dPCR. Preliminary results reported female recipients with male donors undergoing two pediatric and five adult kidney and three heart transplantations. WES revealed a pathogenic mutation in RBM20 and VUS in TTN and PKP2 in heart recipients, while kidney donors presented mutations in UMOD and APOL1 associated with autosomal-dominant kidney diseases, highlighting the risks requiring the long-term monitoring of recipients, donors, and their family members. %dd-cfDNA levels were generally stable but elevated in cadaveric kidney recipient and one pediatric patient with infectious complications and genetic variants in the ABCB1 and ABCC2 genes. These findings highlight the potential of combining genetic and molecular biomarker-based approaches to improve donor–recipient matching, predict complications, and personalize post-transplant care, paving the way for precision medicine in transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666450e0c505e01dbd341948e6058bf80c44e749" target='_blank'>
              Whole-Exome Sequencing Followed by dPCR-Based Personalized Genetic Approach in Solid Organ Transplantation: A Study Protocol and Preliminary Results
              </a>
            </td>
          <td>
            M. Bayanova, Aidos K. Bolatov, Dias Malik, Aida Zhenissova, A. Abdikadirova, Malika Sapargaliyeva, Lyazzat Nazarova, G. Myrzakhmetova, S. Novikova, Aida Turganbekova, Y. Pya
          </td>
          <td>2025-03-04</td>
          <td>Methods and Protocols</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74796a287444dba7d01b2945d7a36e2b35ccf96" target='_blank'>
              THE FUTURE OF MEDICINE: ADVANCING GENE THERAPY WITH CRISPR-CAS9’S EXACT PRECISION IN PEDIATRIC MALES
              </a>
            </td>
          <td>
            Peace Chinonyerem Ike, ONYEBUKWA ANTHONY KELECHI, Victor Ifechukwude Agboli, SHOKOYA TUMININU, JUSTINA CHINAZAEKPERE IKE
          </td>
          <td>2025-01-31</td>
          <td>Journal of Health Systems Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0687d954e3a7ce3f1c8da610c5b175b62bd0c491" target='_blank'>
              Single-cell multiomics reveals a gene regulatory circuit driving leukemia cell differentiation.
              </a>
            </td>
          <td>
            Xin Tian, Liuqingqing Zhang, Guiqiyang Xiang, Yijia Tang, Ping Zhu, Shuting Yu, Fangying Jiang, Shuai Wang, Jinzeng Wang, Yao Dai, Desheng Zheng, Jianbiao Wang, Xiangqin Weng, Shengyue Wang, Yun Tan, Feng Liu
          </td>
          <td>2025-02-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background TP53 mutations are linked to aggressive progression and chemoresistance in gastric cancer (GC). Frameshift mutation is the second most common mutation type of TP53. However, the consequences of this mutation type in GC were not well understood, and targeted therapies for cancer patients harboring frameshift mutations were also not established. Histone methylation significantly influences tumorigenesis in TP53-mutated cancers, and related inhibitors are emerging as specific therapeutic strategies. Methods and results By treating GC cell lines harboring various TP53 mutation types with a library of histone demethylase inhibitors, we identified that GSK690, a reversible inhibitor of lysine-specific demethylase 1 (LSD1), selectively inhibits GC cells harboring TP53 frameshift mutations without nuclear localization sequence (NLS) (termed TP53 Frameshift NLS), which accounts for 89% TP53 frameshift mutations in GC patients. GSK690 showed significant specific inhibition in vitro and in vivo against this subtype by inducing G1/S cell cycle arrest via the LSD1-CCNA2 axis. Importantly, dual-luciferase assays and ChIP-qPCR confirmed that the loss of transcriptional repression activities of p53 in drives LSD1 upregulation in TP53 Frameshift NLS cancer cells. Conclusions In summary, our results indicate that the nuclear localization deficiency of p53 accounts for increased expression of LSD1 in TP53 Frameshift NLS GCs. GSK690 inhibits cell cycle progression and tumor growth by suppressing aberrantly activated LSD1-CCNA2 signaling in this GC subtype, counteracting malignant proliferation and thereby providing a precise therapeutic strategy for GC patients with TP53 Frameshift NLS. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01829-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Rationale: Ovarian cancer is a highly lethal gynecological malignancy with common platinum resistance. Lactylation is involved in multiple biological processes. Thus, we explored the role of histone and non-histone lactylation in platinum resistance, providing a potential therapeutic target to overcome platinum resistance in ovarian cancer. Methods: We utilized gene set enrichment analysis to investigate lactylation-related pathway alterations between platinum-resistant and platinum-sensitive patients from the TCGA cohort. Differential expression of H3K9la was demonstrated using Western blotting and immunohistochemistry. Progression-free and overall survival were determined using a log-rank test. Drug response to cisplatin was evaluated by CCK8, apoptosis flow cytometry, and clonogenic assays in vitro. ChIP-seq and ChIP-qPCR assays were performed to identify downstream targets of H3K9la, which was further confirmed by qRT-PCR. LC-MS/MS was conducted to identify specific lactylation sites for RAD51. Co-IP was used to reveal the interaction between GCN5 and H3K9la or RAD51la. Cell line-derived and patient-derived xenograft (PDX) models of ovarian cancer were constructed for the in vivo experiments. Results: Our study showed elevated histone lactylation, especially of H3K9la, in platinum-resistant ovarian cancer. Moreover, high H3K9la indicated platinum resistance and poor prognosis of ovarian cancer. Impairing H3K9la enhanced response to cisplatin. Mechanistically, H3K9la directly activated RAD51 and BRCA2 expression to facilitate homologous recombination (HR) repair. Furthermore, RAD51K73la enhanced HR repair and subsequently conferred cisplatin resistance. H3K9la and RAD51K73la shared the same upstream regulator, GCN5. Notably, a GCN5 inhibitor remarkably improved the tumor-killing ability of cisplatin in PDX models of ovarian cancer. Conclusions: Our study demonstrated the essential role of histone and RAD51 lactylation in HR repair and platinum resistance. It also identified a potential therapeutic strategy to overcome platinum resistance and improve prognosis in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5ab75585545ad6fb4a52167fc59fff4dafb2d9e" target='_blank'>
              Targeting platinum-resistant ovarian cancer by disrupting histone and RAD51 lactylation
              </a>
            </td>
          <td>
            Chenggong Sun, Xiao Li, Qiuli Teng, Xihan Liu, Li Song, H. Schiöth, Huan Wu, Xinyu Ma, Zhaoyang Zhang, Changjian Qi, Haocheng Zhang, Kun Song, Qing Zhang, Beihua Kong
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The acquisition of drug resistance poses a significant challenge for successful cancer treatment. Understanding the mechanisms by which cancer evades drug action is crucial for developing effective therapies. PARP inhibitors (PARPi) induce synthetic lethality in BRCA-deficient cancer cells; however, these cells eventually develop PARPi resistance. Here, we demonstrate that prolonged exposure of BRCA-deficient cancer cells to PARPi or Cisplatin leads to alterations in YAP1 pre-mRNA splicing patterns, resulting in selective upregulation of a minor YAP1 isoform, YAP1-2α. Elevated YAP1-2α specifically heterodimerizes with TAZ, triggering liquid-liquid phase separation (LLPS). This process generates nuclear condensates that activate super-enhancers containing the YAP1-2α/TAZ heterodimer, TEAD, and BRD4. These super-enhancers (here referred to as YAP1-2α/TAZ/TEAD super-enhancers) are also activated in CD44(variant)high cancer stem-like populations in parental cells. In both PARPi-resistant and CD44(variant) high cancer cell populations, treatments that disrupt YAP1-2α/TAZ/TEAD super-enhancers abolish cancer stem-like properties. Thus, the activation of YAP1-2α/TAZ/TEAD super-enhancers reinforces cancer cell stemness, which confers resistance to adverse conditions, such as those caused by anticancer drug treatment, including PARPi and Cisplatin. Our investigation reveals that YAP1-2α/TAZ/TEAD super-enhancers transcriptionally facilitate the malignant grade of cancer cells. Disrupting these super-enhancers represents a promising strategy to hinder neoplastic progression and prevent multidrug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a34b15f56fa0ab0ce3e33d624b269f9e4ee39eca" target='_blank'>
              YAP1-2α/TAZ heterodimerization drives phase separation, activating TEAD-positive super-enhancers that enhance cancer stemness and multidrug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4036111a96308ac464169ce403c17e9659fbafc5" target='_blank'>
              De-novo DNA Methylation of Bivalent Promoters Induces Gene Activation through PRC2 Displacement
              </a>
            </td>
          <td>
            H. O’Geen, Alina Mihalovits, Benjamin D. Brophy, He Yang, Maxwell W. Miller, Chelsey J. Lee, David J. Segal, M. Tomková
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Accumulated evidence supported the crucial role of a tiny population of cells within the tumor called cancer stem cells (CSCs) in cancer origination, and proliferation. Additionally, these cells are distinguished by their self-renewal, differentiation, and therapeutic resistance capabilities. Interestingly, many studies recorded dysregulation of different types of noncoding RNAs, such as microRNA (miRNA) and long non-coding RNA (LncRNA), in cancer cells as well as CSCs. Moreover, several studies also supported the regulation of the transcription factors and signaling pathways required for CSC progression by these noncoding RNAs. However, the exact biological functions of all these noncoding RNAs are not well understood yet. These findings are of great interest, implying usage of noncoding RNA as therapeutic tool to target these cells. In this review, we provide an insight into how noncoding RNAs regulate CSCs and how this correlation is manipulated to develop new therapies to eradicate cancer cells successfully.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71380c52d563b7fa516173a081f75f827548dbc2" target='_blank'>
              Role of noncoding RNA as a pacemaker in cancer stem cell regulation: a review article.
              </a>
            </td>
          <td>
            Yasmin M. Attia, Samer A Tadros, S. A. Fahim, Doaa M Badr
          </td>
          <td>2025-03-24</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc949647e3b9b0b845365c842f739d457eb9e866" target='_blank'>
              Functional profiling of p53 and RB cell cycle regulatory proficiency suggests mechanism-driven molecular stratification in endometrial carcinoma.
              </a>
            </td>
          <td>
            Zelei Yang, S. Mogre, Hyeji Jun, Ruiyang He, Sneha Ghosh Chaudhary, U. R. Bhattarai, Shannan J. Ho Sui, Ursula A Matulonis, Suzan Lazo, Aniket Shetty, A. Cameron, Quang-De Nguyen, Sarah J Hill
          </td>
          <td>2025-02-11</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89e38976a47d12226a011902bfacb13e2fb36e59" target='_blank'>
              Genome-wide CRISPR screen in human T cells reveals regulators of FOXP3.
              </a>
            </td>
          <td>
            Kelvin Y. Chen, Tatsuya Kibayashi, Ambre Giguelay, Mayu Hata, Shunsuke Nakajima, N. Mikami, Yusuke Takeshima, Kenji Ichiyama, Ryusuke Omiya, Leif S. Ludwig, Kunihiro Hattori, Shimon Sakaguchi
          </td>
          <td>2025-03-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f17ec0f574b24441e84e586e6063343737a2a9" target='_blank'>
              Massively parallel in vivo Perturb-seq screening.
              </a>
            </td>
          <td>
            Xinhe Zheng, Patrick C Thompson, Cassandra M. White, Xin Jin
          </td>
          <td>2025-02-12</td>
          <td>Nature protocols</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Background. Mutations in homologous recombination repair (HRR) genes (BRCA2, ATM, CHEK2, NBN, etc.) are found in 20–25 % of patients with metastatic prostate cancer (PC) and are an indication for prescription of PARP inhibitors. Sensitivity to these drugs results from homologous recombination deficiency (HRD) in the tumor. It is currently not fully elucidated which HRR genes besides BRCA1/2 cause HRD. The aim of the study was to evaluate the presence of homologous recombination deficiency in pc associated with mutations in different HRR genes. Material and methods. Paired tumor and normal DNA samples from 272 pc patients were examined using the HiSNP Ultra Panel v1.0 NGS panel (Nanodigmbio). Tumor copy number variation profiles were used to obtain the HRD score, which was determined as the unweighted sum of three characteristics: the number of chromosomal regions with loss of heterozygosity (LOH), large scale state transitions (LST), and telomeric allelic imbalances (TAI). HRD scores in different pc categories were compared using the Mann–Whitney test. Results. The studied case series included 58 pcs with pathogenic/ likely pathogenic mutations in at least one of 34 HRR genes, and 214 pcs without HRR mutations. The median HRD scores in the groups with BRCA2, ATM, CHEK2, other HRR mutations and without HRR mutations were 41, 22.5, 7.5, 7.5, 14 and 9, respectively. The HRD score was significantly higher in BRCA2-associated tumors than in other PC categories (p<0.01 for all comparisons), except in cases with atm mutations, where the difference did not reach formal significance (p=0.051). Homologous recombination deficiency, defined as HRD score ≥25, was observed in 19/58 (32.8 %) PCs with HRR mutations and 40/214 (18.7 %) tumors without HRR mutations (p=0.03). In tumors without HRR mutations, the HRD score was significantly higher in the presence of somatic TP53 mutations (p<0.0001). Conclusion. In contrast to BRCA2-associated PCs, most tumors with mutations in the CHEK2, NBN, BLM, FANCM, BRCA1 genes are not characterized by homologous recombination deficiency. in the case of ATM gene lesions, approximately half of pcs have a high HRD score. Testing for HRD score allows identification of a significant proportion (5–19 %, depending on the threshold chosen) of tumors with HRD among pcs that do not contain mutations in HRR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/475a2bac32f3b3fba84f14504ff2c301042d61c8" target='_blank'>
              Analysis of homologous recombination deficiency in prostate cancer
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, N. S. Morozova, A. D. Shestakova, E. Kuligina, E. Imyanitov
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a fundamental mechanism in placental mammals that compensates for gene dosage differences between sexes. Using methylation levels of genes under XCI, we establish defective levels of XCI as a new source of interindividual variation among cancer types in females, characterized by a significant and consistent lowering of XIST expression and enrichment of differentially expressed genes under XCI. We show that defective XCI is an additive factor to the cancer risk of XCI escape deregulation in women. Defective XCI of more than 10% has an attributable risk of 40% among 12 different cancers from The Cancer Genome Atlas. Validations between independent studies of breast cancer samples show that defective XCI increases triple-negative subtype frequency, decreases survival rates, and is reduced by chemotherapy treatment. Mechanistically, it is associated with somatic mutations at TP53 and top MYC gains. In independent studies, defective XCI is detectable in blood and increases with aging, menopause, and cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303ef18264231a4b1a297a874c3c158036c417de" target='_blank'>
              Defective X-chromosome inactivation and cancer risk in women
              </a>
            </td>
          <td>
            A. Cáceres, Luis A. Pérez-Jurado, Albert Alegret-García, Varun B. Dwaraka, Ryan Smith, Juan R. González
          </td>
          <td>2025-02-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="As research into tumour immunotherapy continues to accelerate, new frontiers are being revealed in the field of cancer treatment. A significant focus has been drawn to neoantigen-based personalised tumour vaccines, a pioneering immunotherapy. This approach involves the use of genetic mutations that are unique to tumor cells to custom-design personalized tumor vaccines. These vaccines elicit an immune response that is specifically directed at targeting and eliminating cancer cells. The incorporation of neoantigens, arising from mutations within tumor cells, confers a distinct advantage to personalized tumor vaccines in terms of precision and the mitigation of adverse effects. However, the intricate pathways from antigen presentation to the activation of tumor immunogenicity remain to be elucidated. This paper primarily delves into the origins and characteristics of neoantigens, and also neoantigen prediction, highlights existing screening methods, and addresses the limitations of current approaches. It is hoped that this review will act as a catalyst, accelerating the understanding of relevant knowledge and illuminating research hotspots for scientists poised to venture into neoantigen research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea61347bed9e76289661e280b6f539cf9500a162" target='_blank'>
              Neoantigens: new hope for cancer therapy
              </a>
            </td>
          <td>
            Yitong Hu, Tengda Zhou, Ping Cai, Zihao He
          </td>
          <td>2025-03-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Germinal centres (GCs) are specialized structures where B cells undergo iterative steps of B-cell receptor (BCR) somatic hypermutation and selection of best antigen binders in a darwinian-like fashion. The accelerated evolutionary process leads to the production of high-affinity antibodies that are crucial for robust and long-term humoral immunity. Within this frame, single-cell BCR sequencing analysis is a method of choice to track GC B cell dynamics as somatic mutations can be utilised as an in vivo molecular tracer. Herein, we present SeQuoIA, a start-to-finish pipeline for the analysis of BCR repertoire sequencing data at the single-cell level, including improved clonotype assignment and phylogeny reconstruction. Most importantly, we introduce a new method for the inference of BCR-driven selection pressure based on somatic mutation patterns, that was validated with biological data. With this pipeline, we explored public datasets and proposed new selection mechanisms in GCs. Significance Our pipeline should contribute to a better understanding of the basic biology of GC dynamics, and potentially help in laboratory animal usage reduction. Clinical applications could include assessment of vaccine efficacy, monitoring of B cell anti-tumoral responses, and identification of BCR-mediated processes in B cell lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1235a7befe3184ce5a84373898145bc5761dacf1" target='_blank'>
              SeQuoIA: a single-cell BCR sequencing analysis pipeline for tracking selection mechanisms in germinal centres
              </a>
            </td>
          <td>
            Eglantine Hector, P. Milpied
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c19557776012bc520b8b5ecf4614f5b4aaa6f641" target='_blank'>
              The transcription factor SRF regulates MERVL retrotransposons and gene expression during zygotic genome activation.
              </a>
            </td>
          <td>
            C. Hermant, Carlos Michel Mourra-Díaz, Marlies E. Oomen, Luis Altamirano-Pacheco, Mrinmoy Pal, Tsunetoshi Nakatani, M. Torres-Padilla
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Alterations in the tumor suppressor gene TP53 are common in human cancers and are associated with an aggressive nature. Approximately 8%-12% of myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) harbor TP53 mutations (TP53mut) and present immense challenges due to inherent chemoresistance and poor outcomes. As TP53mut are more common in older individuals and those with secondary/therapy-related myeloid neoplasms (MN), their incidence is expected to increase with an aging population and rising proportion of cancer survivors. Treatments used for other MN-intensive chemotherapy, hypomethylating agents, and the BCL-2 inhibitor venetoclax-do not improve the survival of TP53mut MN patients meaningfully. Additionally, further development of many promising agents has been discontinued, highlighting the challenges. Widespread acknowledgment of these problems led to the recognition of TP53mut MN as a distinct entity in the 5th edition of the World Health Organization and International Consensus Classifications. However, critical discrepancies between the two classifications may lead to under- or overestimation of the prognostic risk. Here, we review recent advances in the biology, diagnosis, and treatment of TP53mut MN. The development of TP53mut MN is positioned at the intersection of age, hereditary predisposition, and anti-cancer therapies. Precursor TP53mut clones can be detected years prior to the eventual leukemic transformation-raising the possibility of early intervention. We discuss the two classification systems and the bearing of the discrepancies between the two on timely and effective management. We provide novel evidence in the areas of discrepancies. Finally, we review the current therapeutic landscape and the obvious limitations of the currently used therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b892f35891a00a215db6f4653f26ae1d8881890" target='_blank'>
              TP53-Mutated Myeloid Neoplasms: 2024 Update on Diagnosis, Risk-Stratification, and Management.
              </a>
            </td>
          <td>
            M. Shah, Daniel A Arber, D. Hiwase
          </td>
          <td>2025-03-11</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Chemoresistance remains a major hurdle in ovarian cancer (OC) treatment, as many patients eventually develop resistance to platinum‐based chemotherapy and/or PARP inhibitors (PARPi). Methods We performed transcriptome‐wide analysis by RNA sequencing (RNA‐seq) data of platinum‐resistant and ‐sensitive OC tissues. We demonstrated the role of LINC02776 in platinum resistance in OC cells, mice models and patient‐derived organoid (PDO) models. Results We identify the long noncoding RNA LINC02776 as a critical factor of platinum resistance. Elevated expression of LINC02776 is observed in platinum‐resistant OC and serves as an independent prognostic factor for OC patients. Functionally, silencing LINC02776 reduces proliferation and DNA damage repair in OC cells, thereby enhancing sensitivity to platinum and PARPi in both xenograft mouse models and patient‐derived organoid (PDO) models with acquired chemoresistance. Mechanistically, LINC02776 binds to the catalytic domain of poly (ADP‐ribose) polymerase 1 (PARP1), promoting PARP1‐dependent polyADP‐ribosylation (PARylation) and facilitating homologous recombination (HR) restoration. Additionally, high HIF‐1α expression in platinum‐resistant tissues further stimulates LINC02776 transcription. Conclusions Our findings suggest that targeting LINC02776 represents a promising therapeutic strategy for OC patients who have developed resistance to platinum or PARPi. Key points LINC02776 promotes OC cell proliferation by regulating DNA damage and apoptosis signaling pathways. LINC02776 binds PARP1 to promote DNA damage‐triggered PARylation in OC cells. LINC02776 mediates cisplatin and olaparib resistance in OC cells by enhancing PARP1‐mediated PARylation activity and regulating the PARP1‐mediated HR pathway. The high expression of LINC02776 is induced by HIF‐1α in platinum‐resistant OC cells and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f154627a6e9fbb35a9c2baccde3a2a62d9723cc8" target='_blank'>
              HIF‐1α‐induced long noncoding RNA LINC02776 promotes drug resistance of ovarian cancer by increasing polyADP‐ribosylation
              </a>
            </td>
          <td>
            Yangjun Wu, Yu Zeng, Yong Wu, Xinyu Ha, Zheng Feng, Chaohua Liu, Ziqi Liu, Jiajia Wang, X. Ju, Shenglin Huang, Linhui Liang, Bin Zheng, Lulu Yang, Jun Wang, Xiaohua Wu, Shengli Li, H. Wen
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12858fb8729b079d5d5eaad7f8943da255c87e81" target='_blank'>
              Transcription Factor FOSL1 Promotes Cisplatin Resistance in Non-Small Cell Lung Cancer Cells by Modulating the Wnt3a/β-Catenin Signaling through Upregulation of PLIN3 Expression.
              </a>
            </td>
          <td>
            Wanning Tong, Jianjun Sun, Bingxiao Shen, Yaohua Hu, Chenxing Wang, Min Rao, Jin Li, Delin Xia, Jiagui Dong, Hong Wang, Dongmei Zhu, Haibo Wu, Zhigang Cai
          </td>
          <td>2025-03-19</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f7401161a6069ba8245d4505d68dfc98e30045" target='_blank'>
              The Feasibility of Using CRISPR-Cas9 to Knock Out IL-6 Receptor in CAR-T Cells to Mitigate Cytokine Release Syndrome in Cancer Therapy
              </a>
            </td>
          <td>
            Zhiyuan Wang
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175bd3895700f513d14429236b145a33de18651c" target='_blank'>
              MicroRNA-371-373 cluster and methylome analysis suggests that a subset of 'somatic-type' malignancies arising in germ cell tumors may originate in yolk sac tumor components.
              </a>
            </td>
          <td>
            João Lobo, Nuno Tiago Tavares, Diana Fonseca, Carmen Jerónimo, Rui Henrique, Nicolas Wyvekens, Yiying Yang, Matija Snuderl, Fiona Maclean, Jennifer B. Gordetsky, C. Fletcher, Michelle S Hirsch, Jason L. Hornick, M. T. Idrees, Katrina Collins, L. Warmke, Thomas Ulbright, Andres M Acosta
          </td>
          <td>2025-03-28</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9782e1bf6046bb6e08da1b33f9dcb44ab84e474" target='_blank'>
              A pivotal contribution of proteostasis failure and mitochondrial dysfunction to chromosomal instability-induced microcephaly
              </a>
            </td>
          <td>
            Amanda González-Blanco, Adrián Acuña-Higaki, David Boettger, Marco Milán
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) are a class of non-coding RNA molecules with transcripts longer than 200 bp, which were initially thought to be noise from genomic transcription without biological function. However, since the discovery of H19 in 1980 and Xist in 1990, increasing evidence has shown that lncRNAs regulate gene expression at epigenetic, transcriptional, and post-transcriptional levels through specific regulatory actions and are involved in the development of cancer and other diseases. Despite many lncRNAs being expressed at lower levels than those of protein-coding genes with less sequence conservation across species, lncRNAs have become an intense area of RNA research. They exert diverse biological functions such as inducing chromatin remodeling, recruiting transcriptional machinery, acting as competitive endogenous RNAs for microRNAs, and modulating protein–protein interactions. Epithelial–mesenchymal transition (EMT) is a developmental process, associated with embryonic development, wound healing, and cancer progression. In the context of oncogenesis, the EMT program is transiently activated and confers migratory/invasive and cancer stem cell (CSC) properties to tumor cells, which are crucial for malignant progression, metastasis, and therapeutic resistance. Accumulating evidence has revealed that lncRNAs play crucial roles in the regulation of tumor epithelial/mesenchymal plasticity (EMP) and cancer stemness. Here, we summarize the emerging roles and molecular mechanisms of lncRNAs in regulating tumor cell EMP and their effects on tumor initiation and progression through regulation of CSCs. We also discuss the potential of lncRNAs as diagnostic and prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46ca0f5ca7874e4c69f36777bb099d0d32ae920f" target='_blank'>
              Long Non-Coding RNAs: Key Regulators of Tumor Epithelial/Mesenchymal Plasticity and Cancer Stemness
              </a>
            </td>
          <td>
            Yuan Yuan, Yun Tang, Zeng Fang, Jian Wen, Max S. Wicha, Ming Luo
          </td>
          <td>2025-02-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objectives/Goals: The never in mitosis kinase (NEK) family regulates vital processes, namely cell cycle progression, but their potential as therapeutic targets in TNBC has not been fully explored. Our studies aim to develop a toolkit to investigate the functional roles of NEKs in pathologies including carcinogenesis. Methods/Study Population: To assess differential NEK expression in normal and tumor tissues and correlation of gene expression with patient survival, we used Gene Expression Profiling Interactive Analysis (GEPIA) and Kaplan–Meier Plotter (KMPlot) pan-cancer analysis, respectively. Basal NEK protein levels were determined by immunoblot across a panel of cell lines, including breast cancer, osteosarcoma, hepatocellular carcinoma, and non-cancerous cells, to identify appropriate systems for evaluation of NEK function. Doxycycline-inducible cell lines were generated by transduction with lentiviral stocks of NEK shRNA and overexpression constructs and antibiotic selection. Expression was analyzed by qPCR and immunoblot. Results/Anticipated Results: Expression of NEK2, 4, 5, 6, 8, and 11 was higher in breast tumors compared to normal tissue by GEPIA analysis. Further examination using KMPlot showed a correlation between elevated NEK6 expression and decreased overall survival in patients with aggressive cancers. As an initial proof-of-concept study, we analyzed NEK6 protein expression in breast cancer cells. Levels of NEK6 were elevated in TNBC cells (MDA-MB-231) compared to hormone receptor positive (HR+) breast cancer cells (MCF7). Using complementary approaches to investigate the functional role of NEK6 in breast cancer, we depleted NEK6 expression using shRNAs in TNBC cells and expressed NEK6 in HR+ cells Discussion/Significance of Impact: Because kinase dysregulation promotes oncogenesis and metastasis, targeting kinases is a key strategy in therapeutic development. A NEK-specific molecular toolkit allows researchers to elucidate NEK functions and contributions to carcinogenesis, promoting advancement of novel therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/818cfe4068f4242b8099633fcb05e557e255d8de" target='_blank'>
              426 Developing a Molecular Toolkit to define NEK functions in triple-negative breast cancer (TNBC) biology
              </a>
            </td>
          <td>
            Julia Boehling, Jack R. Elliot, Elizabeth C. Martin, B. Collins-Burow, D. Drewry, V. Hoang, Sean Lee, M. Burow
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Pathogenic gene dysregulation can be attributed to chromatin state change that pre-transcriptionally regulates expression. Recent breakthroughs elucidating the rules governing this DNA control layer, an epigenetic code, unlock a modality in precision medicine to target gene dysregulation across myriad diseases. Here we present a modular platform to design programmable mRNA therapeutics, Epigenomic Controllers (EC), that control gene expression through directed epigenetic change. By leveraging natural mechanisms, ECs tune expression levels of one or multiple genes with durable effect of weeks-to-months in female mice following a single dose. We design and characterize ECs to multiple target genes and identify an EC that effectively inhibits the cancer- and inflammatory-disorder-associated multi-gene cluster CXCL1-8. With precision targeting of NF-kB signaling and identification of homologous murine surrogates, ECs significantly reduce neutrophil migration in vivo during acute lung inflammation in female mice. A platform approach to EC design for epigenomic modulation expands treatment frontiers for diverse gene targets, including those considered “undruggable.”">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c55ad387d82ce60d43ef750c8b35debeea4164" target='_blank'>
              Programmable mRNA therapeutics for controlled epigenomic modulation of single and multiplexed gene expression in diverse diseases
              </a>
            </td>
          <td>
            Charles W O'Donnell, J. Farelli, Houda Belaghzal, Justin Chen, Lauren Beech, James Sullivan, Chevaun D. Morrison-Smith, Stephen Siecinski, Adam Katz, Samuel Mildrum, Mayur Gurnani, Prachi Dhanania, Caitlyn R Webb, Giuliana Castello Coatti, Pranjali Rumale, Daniel F G Costa, Marcus I Gibson, Yaoyu E. Wang, J. Newman, T. McCauley
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da248ee7a941cb4fe66c4c17c284b6ea6d864ede" target='_blank'>
              Different In Vitro Models of Chronic Myeloid Leukemia Show Different Characteristics: Biological Replicates Are Not Biologically Equivalent.
              </a>
            </td>
          <td>
            Alessia Cavalleri, Besjana Xhahysa, Silvia Mutti, R. Ferraro, E. L. Mazzoldi, M. Farina, A. Leoni, Luca Garuffo, Federica Trenta, Federica Re, V. Radici, Eugenia Accorsi Buttini, E. Morello, Gabriele Magliano, V. Cancelli, S. Giliani, Michele Malagola, Domenico Russo, S. Bernardi
          </td>
          <td>2025-03-01</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Engineering T cell specificity and function at multiple loci can generate more effective cellular therapies, but current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here we develop a one-step process to enrich unlabeled cells containing knock-ins at multiple target loci using a family of repair templates named synthetic exon expression disruptors (SEEDs). SEEDs associate transgene integration with the disruption of a paired target endogenous surface protein while preserving target expression in nonmodified and partially edited cells to enable their removal (SEED-Selection). We design SEEDs to modify three critical loci encoding T cell specificity, coreceptor expression and major histocompatibility complex expression. The results demonstrate up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). This method is compatible with existing clinical manufacturing workflows and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857a45248fbd262ae3ffedba1c112578d12a79d8" target='_blank'>
              SEED-Selection enables high-efficiency enrichment of primary T cells edited at multiple loci.
              </a>
            </td>
          <td>
            Christopher R. Chang, Vivasvan S. Vykunta, Jae Hyun J Lee, Ke Li, Clara Kochendoerfer, J. J. Muldoon, Charlotte H. Wang, Thomas Mazumder, Yang Sun, Daniel Goodman, William A. Nyberg, Chang Liu, Vincent Allain, A. Rothrock, C. Ye, Alex Marson, B. Shy, Justin Eyquem
          </td>
          <td>2025-02-05</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f886ffbc920327f15f89859ed3277c346fdf7187" target='_blank'>
              Heterogeneous and novel transcript expression in single cells of patient-derived clear cell renal cell carcinoma organoids.
              </a>
            </td>
          <td>
            Tülay Karakulak, Natalia Zajac, H. A. Bolck, A. Bratus-Neuenschwander, Qin Zhang, Weihong Qi, Debleena Basu, Tamara Carrasco Oltra, H. Rehrauer, C. von Mering, Holger Moch, Abdullah Kahraman
          </td>
          <td>2025-03-19</td>
          <td>Genome research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Leukemia is characterized by multiple rearrangements of signal transduction genes and overexpression of nonmutated genes, such as transcription factors (TFs) genes. Super-enhancers (SEs) are prevalent in human cancers and are associated with the accumulation of numerous core TFs. SEs drive the expression of core TF genes by delivering robust transcriptional activation signals. Additionally, core TFs sustain the stability and activity of SEs through mutual auto-regulation loops, creating a positive feedback loop known as the Core Transcriptional Regulation Circuit (CRC). Using ChIP-seq data, we identified the involvement of the SE-related gene ZNF217 in acute myeloid leukemia (AML), in which its functional role and underlying mechanism remain unclear. We demonstrated that ZNF217, ELF1, MEF2D, RUNX2, and FOXP1 are likely integral components of the AML CRC through various experimental techniques, including CUT&Tag, short hairpin RNA (shRNA) transduction, and Luciferase reporter assays. Notably, ZNF217 was determined to be indispensable for the proliferation and viability of AML cells both in vitro and in vivo. Subsequent analysis of RNA-seq and CUT&Tag results identified MYB as a key direct target of ZNF217. Overall, our research highlights ZNF217 as a critical oncogene in AML and offers new insights into the transcriptional regulatory mechanisms at play in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/388657aaeaf8efe74b215605c5baaa2e335b1840" target='_blank'>
              Core transcriptional regulatory circuitry molecule ZNF217 promotes AML cell proliferation by up-regulating MYB
              </a>
            </td>
          <td>
            Bi Zhou, Fang Fang, Yongping Zhang, Zhiheng Li, Yixin Hu, Yan Li, Wanyan Jiao, Yumeng Wu, X. Wan, Ying Yang, Fenli Zhang, Ling Xu, Tongting Ji, Jian Pan, Shaoyan Hu
          </td>
          <td>2025-02-18</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Introduction Acute myeloid leukemia (AML) is a leading cause of mortality among Indian children and adults, driven by diverse genetic and epigenetic abnormalities. Limited access to genomic sequencing in India due to resource constraints has hindered a comprehensive understanding of prognostic factors specific to this population.
 Objectives This study aims to analyze publicly available genomic data using statistical and bioinformatics tools to identify key prognostic markers relevant to Indian AML patients.
 Materials and Methods The study utilized tumor/normal pair data from 200 adult de novo AML patients, obtained from The Cancer Genome Atlas (TCGA) and the Genotype-Tissue Expression database, analyzed using cBioPortal. Statistical and bioinformatics tools were employed to assess the impact of existing prognostic targets on disease response and to identify variables with clinical relevance and practical testing feasibility.
 Results Analysis of the TCGA-AML data set identified high-frequency gene mutations (≥ 10%) and well-defined cytogenetic subtypes, including t(8;21)(q22;q22), NPM1 mutations, and CEBPA mutations as key factors for future prognostic evaluation. These findings will contribute to the development of a prognostic scoring system using R programming in future.
 Conclusion This study offers insights into the cytogenetic and mutational landscape of AML in the Indian population, identifying critical genetic and cytogenetic markers with the potential to enhance prognostication, guide treatment strategies, and inform transplant decisions. Using R tools like limma and edgeR, differential expression analysis identified five key genes—NPM1, FLT3, IDH2, RUNX1, and STAG2—as significantly upregulated in AML. Notably, STAG2 emerges as a novel marker with potential prognostic significance, warranting validation in larger Indian cohorts. These findings may help uncover novel therapeutic targets not currently recognized in Western populations, paving the way for a more tailored and personalized approach to treatment in India. By leveraging global genomic databases, this research addresses regional gaps in knowledge. Future work should focus on validating these findings through large-scale studies in Indian cohorts to ensure their broader applicability and impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a9d42cdf5ba552b6ee663d661dbe857122a4db" target='_blank'>
              Analysis of TCGA (The Cancer Genome Atlas) Data for Prognosis, Risk Categorization, and Survival of AML Patients Using Bioinformatics
              </a>
            </td>
          <td>
            Bhavika Rishi, Shamsuz Zaman, Fouzia Siraj, Sree Vishmaya, Himanshu Dhanda, Manpreet Kaur, S. Swain, Baseer Noor, S. Chaudhry, Ankur Jain, Prashant Prabhakar, Amitabh Singh, Aroonima Misra
          </td>
          <td>2025-02-24</td>
          <td>Indian Journal of Medical and Paediatric Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The homeobox (HOX) genes especially for HOXA cluster play crucial roles in leukemogenesis. HOXA overexpression caused by genetic alterations, such as KMT2A rearrangements, NUP98- fusions and FLT3-ITD mutations, is frequently identified in AML. However, very few studies determined the DNA methylation-mediated epigenetic regulation of the HOXA cluster genes in AML. Methods We systematically first screened the prognostic value of HOXA cluster genes methylation in AML from The Cancer Genome Atlas (TCGA) datasets. Afterwards, the candidate prognosis-related gene HOXA9 were selected for clinical relevance analysis and were further validated in another independent cohort from our research center. Results The methylation of HOXA9, among HOXA cluster genes, negatively correlated with adverse prognosis and expression were screened and identified in AML among TCGA datasets. Clinically, HOXA9 hypomethylation was positively correlated with specific subtypes of AML, such as French-American-British (FAB)-M5/M7, normal karyotype and FLT3, NPM1 and DNMT3A mutation, whereas negatively associated with FAB-M3, t(15;17), t(8;21) and t(16;16). Importantly, AML patients with HOXA9 hypomethylation may profit from transplantation, whereas AML patients with HOXA9 hypermethylation could not, suggesting that HOXA9 methylation may be used to guide therapeutic selection between transplantation and chemotherapy. Bioinformatics analysis demonstrated the association of HOXA9 expression with diverse leukemia-related genes (HOXAs, SOSTDC1, MEG3, miR-10a, miR-381 and miR-193b) and signaling pathways (PI3K-Akt signaling) in AML. Subsequently, we further validate the hypomethylation pattern of HOXA9 in AML patients and the epigenetic regulation of HOXA9 methylation in AML cell-lines. Conclusions HOXA9 methylation linked to HOXA9 expression correlates with diverse genetic abnormalities of AML, such as normal karyotype, t(15;17), t(8;21), t(16;16) and FLT3, NPM1 and DNMT3A mutations. Moreover, HOXA9 hypomethylation may be associated with adverse prognosis, and may guide treatment choice in AML. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13633-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3871f7d01e14055294cf2e4bed721b5625d759b" target='_blank'>
              Identification of HOXA9 methylation as an epigenetic biomarker predicting prognosis and guiding treatment choice in acute myeloid leukemia
              </a>
            </td>
          <td>
            Fei Xie, Ting-Juan Zhang, Xin-long Zhang, Zi-jun Xu, Liang Qiao, Yun Wang, Yangli Zhao, Jun Qian, Jing-dong Zhou
          </td>
          <td>2025-02-07</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eed7b2fdc5b17bdbf958e634178449e843736cc" target='_blank'>
              Genome-Wide CRISPR Screening Identifies Cellular Factors Controlling Nonviral Genome Editing Efficiency
              </a>
            </td>
          <td>
            Shivani Saxena, Meha Kabra, Amr Abdeen, Divya Sinha, Min Zhu, Ruosen Xie, Giovanni Hanstad, Maria A. Fernandez Zepeda, David M. Gamm, B. Pattnaik, Shaoqin Gong, Krishanu Saha
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epidemiological studies have revealed significant sex differences in the incidence of tumors unrelated to reproductive functions, with females demonstrating a lesser risk and a better response to therapy than males. However, the reasons for these disparities are still unknown and cancer therapies are generally sex-unbiased. The tumor-suppressor protein p53 is a transcription factor that can activate the expression of multiple target genes mainly involved in the maintenance of genome stability and tumor prevention. It is encoded by TP53, which is the most-frequently mutated gene in human cancers and therefore constitutes an attractive target for therapy. Recently, evidence of sex differences has emerged in both p53 regulations and functions, possibly providing novel opportunities for personalized cancer medicine. Here, we will review and discuss current knowledge about sexual disparities in p53 pathways, their role in tumorigenesis and cancer progression, and their importance in the therapy choice process, finally highlighting the importance of considering sex contribution in both basic research and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a84bb2275916188060a8234c6ff4a6afe8ab155d" target='_blank'>
              Sex Disparities in P53 Regulation and Functions: Novel Insights for Personalized Cancer Therapies
              </a>
            </td>
          <td>
            Miriana Cardano, G. Buscemi, L. Zannini
          </td>
          <td>2025-02-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7249f3e8ac6eca3f70d08c8938c34e4b84a4f3" target='_blank'>
              In Vivo Time-Resolved Single-Cell RNA-Seq Reveals Chemotherapy-Induced Transcriptional Dynamics in Tumor Infiltrating Lymphocytes.
              </a>
            </td>
          <td>
            Wenqi Zhu, Xiangyu Wu, Zihan Zhao, Meng Zhou, Tianyang Zhang, Derek Zhao, Ning Jiang, Xin Yang, Hongqian Guo, Nan Fang, Jie P. Li, Rong Yang, Tian Tian
          </td>
          <td>2025-02-05</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Targeted therapy is an effective strategy for the treatment of advanced and metastatic pancreatic cancer, one of the leading causes for cancer-related death worldwide. To address the limitations of existing targeted drugs, there is an urgently need to find novel targets and therapeutic strategies. Transcription factor FOS like 1 (FOSL1) is a potential therapeutic target for challenging pancreatic cancer, which contributes to the malignant progression and poor gnosis of pancreatic cancer. High mobility group A1 (HMGA1) is a nonhistone chromatin structural protein that contributes to malignant progression and poor prognosis of cancer. Methods Human FOSL1 complete RNA, shRNA against FOSL1 and shRNA against HMGA1 lentiviral recombination vectors were used to overexpress FOSL1 and knock down FOSL1 and HMGA1. RNA sequencing, Q-PCR and Western blots were used to investigate the mechanism of FOSL1 in regulating the proliferation of pancreatic cancer cells. The relationship between FOSL1 and HMGA1 were analyzed by co-immunoprecipitation Mass spectrometry, Q-PCR of chromatin immunoprecipitation and Western blots. The regulation of FOSL1 and HMGA1 in the invasion and migration, stemness, and multidrug efflux system were determined by transwell assay, sphere formation assay, immunofluorescence, Q-PCR and Western blots. Results We found that FOSL1 promoted the proliferation and progression of pancreatic cancer by trigging stemness, invasion and metastasis, and drug resistance. HMGA1 was a key downstream target regulated by FOSL1 at the transcriptional level and directly interacted with FOSL1. Knockdown of HMGA1 inhibited the proliferation of pancreatic cancer cells by regulating the expression of genes related to stemness, epithelial-mesenchymal transition and multidrug efflux system. Targeted inhibition of FOSL1 and HMGA1 expression significantly inhibited the proliferation of pancreatic cancer cells. Conclusion FOSL1 promote the malignant progression of pancreatic cancer by promoting HMGA1 expression. Targeting FOSL1 and HMGA1 in monotherapy or combination therapy is a promising strategy for the treatment of advanced and metastasis pancreatic cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06304-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb9672d2fcbfed129bd061551ff24b1fe5d4981" target='_blank'>
              FOSL1 drives the malignant progression of pancreatic cancer cells by regulating cell stemness, metastasis and multidrug efflux system
              </a>
            </td>
          <td>
            Xiaolong Liu, Xueyan Zhang, Tingyu Zeng, Yali Chen, Liu Ye, Shuping Wang, Yulan Li
          </td>
          <td>2025-03-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c637d49373bdbc8ffeb399dfd912513e88f9743f" target='_blank'>
              A discrete region of the D4Z4 is sufficient to initiate epigenetic silencing
              </a>
            </td>
          <td>
            Ellen M. Paatela, Faith G. St. Amant, Danielle C Hamm, Sean R. Bennett, Taranjit S. Gujral, Silvère M. van der Maarel, Stephen J. Tapscott
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs), such as enhancers and promoters, are fundamental regulators of gene expression and, across different cell types, the MYC locus utilizes a diverse regulatory architecture driven by multiple CREs. To better understand differences in CRE function, we perform pooled CRISPR inhibition (CRISPRi) screens to comprehensively probe the 2.8 Mb topologically-associated domain containing MYC in 6 human cancer cell lines with nucleotide resolution. We map 32 CREs where inhibition leads to changes in cell growth, including 8 that overlap previously identified enhancers. Targeting specific CREs decreases MYC expression by as much as 60%, and cell growth by as much as 50%. Using 3-D enhancer contact mapping, we find that these CREs almost always contact MYC but less than 10% of total MYC contacts impact growth when silenced, highlighting the utility of our approach to identify phenotypically-relevant CREs. We also detect an enrichment of lineage-specific transcription factors (TFs) at MYC CREs and, for some of these TFs, find a strong, tumor-specific correlation between TF and MYC expression not found in normal tissue. Taken together, these CREs represent systematically identified, functional regulatory regions and demonstrate how the same region of the human genome can give rise to complex, tissue-specific gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/effa3cdf47f124f2fc25cc824cb0bcd26047dfd8" target='_blank'>
              Comprehensive dissection of cis-regulatory elements in a 2.8 Mb topologically associated domain in six human cancers
              </a>
            </td>
          <td>
            Christina M. Caragine, Victoria T Le, Meer Mustafa, Bianca Jay Diaz, John A Morris, Simon Müller, A. Méndez-Mancilla, Evan Geller, Noa Liscovitch-Brauer, Neville E. Sanjana
          </td>
          <td>2025-02-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Regulating the occurrence and progression of tumor cells at the epigenetic level is a new insight of clear cell renal cell carcinoma (ccRCC). Chromatin accessibility is an important pathway of epigenetic regulation, which may explain the mystery of tumor occurrence. Assay for transposase-accessible chromatin sequencing (ATAC-seq) provides insight into the epigenetic regulatory features of ccRCC, especially at the single-cell level. In this study, we performed scATAC-seq of 3 ccRCC samples and captured a total of 18,703 high-quality cell nuclei and 104,818 unique peaks. Our protocol for nuclear extraction was reliable and stable, which can be used to deal with fresh and frozen single-cell suspensions. We presented basic methods for scATAC-seq data analysis, such as cell clustering, gene activity scoring, cell subtype specific peaks, transcription factors, motif and motif footprinting analysis. Taken together, our data indicated the valuable epigenetic features of ccRCC, which will provide more references for the study of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f2a7adbad7b64718a49a79d5c94f7b319bf5e0" target='_blank'>
              Single-cell assay for transposase-accessible chromatin sequencing of human clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Wenhao Lu, Yixuan Liu, Jiaping Li, Shengzhu Huang, Zheng Wen, Cheng Su, Zheng Lu, Zengnan Mo, Zhenyuan Yu
          </td>
          <td>2025-02-25</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fee21f91c15389f14f1f653f6c03f8be3762274" target='_blank'>
              TP53 gene mutations and cytogenetic aberrations in tumor cells of patients with primary mediastinal B-cell large cell lymphoma
              </a>
            </td>
          <td>
            Y. Mangasarova, B. Biderman, T. Obukhova, R. R. Abdurashidova, T. Abramova, A. Magomedova, Yuliya Chabaeva, N. Gabeeva, E. Zvonkov, А. Sudarikov
          </td>
          <td>2025-03-13</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic defects caused by truncating mutations or deletions in the Retinoblastoma tumor suppressor gene (RB1) are frequently observed in many cancer types leading to dysregulation of the RB pathway. Here, we propose an integrative proteogenomic approach that predicts cancers with dysregulation in the RB pathway. A subset of these cancers, which we term as “RBness,” lack RB1 genomic defects and yet phenocopy the transcriptional profile of RB1-defective cancers. We report RBness as a pan-cancer phenomenon, associated with patient outcome and chemotherapy response in multiple cancer types, and predictive of CDK4/6 inhibitor response in estrogen-positive breast cancer. Using RNA interference and a CRISPR-Cas9 screen in isogenic models, we find that RBness cancers also phenocopy synthetic lethal vulnerabilities of cells with RB1 genomic defects. In summary, our findings suggest that dysregulation of the RB pathway in cancers lacking RB1 genomic defects provides a molecular rationale for how these cancers could be treated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b769a127908e214629b60c835bc205b258e11d" target='_blank'>
              Proteogenomic discovery of RB1-defective phenocopy in cancer predicts disease outcome, response to treatment, and therapeutic targets
              </a>
            </td>
          <td>
            Jacopo Iacovacci, R. Brough, Fatemeh Ahmadi Moughari, J. Alexander, H. Kemp, Andrew N J Tutt, R. Natrajan, C. J. Lord, Syed Haider
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a517176c60f9a80290d19966f93c06f80af4cac7" target='_blank'>
              Emerging technologies of single-cell multi-omics.
              </a>
            </td>
          <td>
            Yi June Kim, Koichi Takahashi
          </td>
          <td>2025-02-06</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8904feaf495af004b999f1283404430b8205d7d8" target='_blank'>
              p53-deficient cancer cells hyperactivate DNA double-strand break repair pathways to overcome chemotherapeutic damage and augment survival.
              </a>
            </td>
          <td>
            Rebecca Dsouza, Meghna Jain, Ekta Khattar
          </td>
          <td>2025-03-22</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f07daeb74e80fef9e4142d5ed3323341effa067" target='_blank'>
              Abstract B090: ATRA reprograms neutrophils in the tumor microenvironment and overcomes resistance to immune checkpoint inhibition in murine models of LKB1-deficient non-small cell lung cancer
              </a>
            </td>
          <td>
            William P Crosson, Rui Li, R. Salehi-Rad, Bitta Kahangi, Samantha Chin, R. Lim, J. Abascal, E. P. Reyes, M. Oh, C. Dumitras, Nico Edgar, Ryan Chew, Rashel Jacobo, Adam Tfayli, K. Krysan, L. Tran, Steven M Dubinett, Bin Liu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease.Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy.Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction.Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d49890aa6fe1c33df234142445b8239a8f8a36" target='_blank'>
              Genome-Wide Silencer Screening Reveals Key Silencer Modulating Reprogramming Efficiency in Mouse Induced Pluripotent Stem Cells.
              </a>
            </td>
          <td>
            Xiusheng Zhu, Lei Huang, Guoli Li, Biao Deng, Xiaoxiao Wang, Hu Yang, Yuanyuan Zhang, Qiuhan Wen, Chao Wang, Jingshu Zhang, Yunxiang Zhao, Kui Li, Yuwen Liu
          </td>
          <td>2025-03-20</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Lysosomal-associated protein transmembrane-4 beta (LAPTM4B) protein expression was increased in solid tumors, whereas few studies were performed in hematologic malignancies. We aimed to study the effect of the LAPTM4B gene in pan-cancer and Philadelphia chromosome-positive acute B cell lymphoblastic leukemia (Ph+ B-ALL). Methods The differential expression, diagnosis, prognosis, genetic and epigenetic alterations, tumor microenvironment, stemness, immune infiltration cells, function enrichment, single-cell analysis, and drug response across cancers were conducted based on multiple computational tools. Additionally, Ph+ B-ALL transgenic mouse model with Laptm4b knockout was used to analyze the function of LAPTM4B in vivo. BrdU incorporation method, flow cytometry, and Witte-lock Witte culture were used to evaluate the roles of LAPTM4B in vitro. Results We identified that LAPTM4B expression was increased in various cancers, with significant associations with clinical outcomes. LAPTM4B expression correlated with DNA and RNA methylation patterns and was associated with drug resistance. It also influenced the tumor immune microenvironment, with implications for immunotherapy response. In leukemia, LAPTM4B was expressed in stem cells and associated with specific subtypes. Knockout of LAPTM4B impeded B-ALL progression in mice and reduced cell proliferation and caused G0/G1 arrest in vitro. Discussion Our study elucidated the role LAPTM4B that promoted the development and progression in Ph+ B-ALL. Furthermore, LAPTM4B played a diagnostic, prognostic, and immunological factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec638e3775596d8397e609fbd999f736d99d3a7b" target='_blank'>
              A comprehensive prognostic and immune analysis of LAPTM4B in pan-cancer and Philadelphia chromosome-positive acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Hui Zhou, Yuyao Yi, Wei He, Li Zheng, Yiguo Hu, Ting Niu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4766c6ad595902085d51a487e2edbf0d4dfa6d" target='_blank'>
              Abstract A014: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Progeria syndromes are very rare, incurable premature aging conditions recapitulating most aging features. Here, we report a whole genome, multiparametric CRISPR screen, identifying 43 genes that can rescue multiple cellular phenotypes associated with progeria. We implement the screen in fibroblasts from Néstor-Guillermo Progeria Syndrome male patients, carrying a homozygous A12T mutation in BAF. The hits are enriched for genes involved in protein synthesis, protein and RNA transport and osteoclast formation and are validated in a whole-organism Caenorhabditis elegans model. We further confirm that BAF A12T can disrupt protein synthesis rate and fidelity, which could contribute to premature aging in patients. This work highlights the power of multiparametric genome-wide suppressor screens to identify genes enhancing cellular resilience in premature aging and provide insights into the biology underlying progeria-associated cellular dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bae74b424ea31be0a19ad896060dac632aa75c9" target='_blank'>
              A multiparametric anti-aging CRISPR screen uncovers a role for BAF in protein synthesis regulation
              </a>
            </td>
          <td>
            Sophia Y Breusegem, Jack Houghton, Raquel Romero-Bueno, Adrián Fragoso-Luna, K. Kentistou, Ken K Ong, Anne F J Janssen, Nicholas A Bright, Christian G Riedel, J. Perry, P. Askjaer, Delphine Larrieu
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db0bd1b39e444d04b310b8aafda9975f33581381" target='_blank'>
              Advancements in CRISPR Technologies and Treatment of Genetic Disorders
              </a>
            </td>
          <td>
            Youxi Chen
          </td>
          <td>2025-03-13</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of healthy adults - a phenomenon termed clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. We assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis, remission and/or relapse (total n=182 patients). FLT3 and NPM1 mutations were generally eliminated at complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, CH-associated mutations that likely precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0b511bf0bfbb32bc9fd1061ccf533beadb0c77" target='_blank'>
              Persistent post-remission clonal hematopoiesis shapes the relapse trajectories of acute myeloid leukemia.
              </a>
            </td>
          <td>
            Ryan D Chow, Priya Velu, Safoora Deihimi, Jonathan P Belman, Angela Youn, Nisargbhai Shah, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Robert L. Bowman
          </td>
          <td>2025-02-12</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e491b785ffb0bdd2ceea7ba005ed8f5742762" target='_blank'>
              Relapse-specific genetic patterns in the exomic mutational landscape in acute myeloid leukemia
              </a>
            </td>
          <td>
            Wardah Qureshi, Aisha Jamal, Atia Gohar, Marium Imran Mansuri, Muhammad Irfan, Ishtiaq Ahmad Khan, Muhammad Shakeel
          </td>
          <td>2025-03-25</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6495d97605d89361fd6a142107147bd6e190f07a" target='_blank'>
              MorPhiC Consortium: towards functional characterization of all human genes.
              </a>
            </td>
          <td>
            M. Adli, Laralynne Przybyla, Tony Burdett, P. Burridge, P. Cacheiro, Howard Y. Chang, Jesse M. Engreitz, Luke A. Gilbert, William J. Greenleaf, L. Hsu, Danwei Huangfu, Ling-Hong Hung, A. Kundaje, Shengyu Li, Helen E. Parkinson, Xiaojie Qiu, Paul Robson, Stephan C Schürer, Ali Shojaie, William Skarnes, D. Smedley, L. Studer, Wei Sun, D. Vidović, Thomas Vierbuchen, Brian S. White, Kayee Yeung, Feng Yue, Ting Zhou
          </td>
          <td>2025-02-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710d5afc8c953bcefef21a775add425b3e4ac994" target='_blank'>
              The histone modifier KAT2A presents a selective target in a subset of well-differentiated microsatellite-stable colorectal cancers.
              </a>
            </td>
          <td>
            Vida Kufrin, Annika Seiler, Silke Brilloff, Helen Rothfuß, Sandra Küchler, Silvia Schäfer, Elahe Rahimian, Jonas Baumgarten, Li Ding, Frank Buchholz, C. Ball, Martin Bornhäuser, H. Glimm, Marius Bill, A. A. Wurm
          </td>
          <td>2025-03-27</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd3b0816d9ce9321d656adc7fa4e3bc0da80c79" target='_blank'>
              Patterns and variations of copy number alterations in acute myeloid leukemia: insights from the LeukAtlas database.
              </a>
            </td>
          <td>
            Yanxun Su, Zhenxian Han, Yutong Ji, Anqi Liu, D. Zou, Lina Yan, Dan Liu, Zhang Zhang, Qian-Fei Wang
          </td>
          <td>2025-02-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17cde34fb09a63a05abe76e3aee95cc3a79629d0" target='_blank'>
              A narrative review of epigenetic marker in H3K27ac and its emerging potential as a therapeutic target in cancer.
              </a>
            </td>
          <td>
            Meizi Zhu, Xuejin Lu, Danhong Wang, Jinhu Ma, Yi Wang, Rui Wang, Hongye Wang, Wenhui Cheng, Yaling Zhu
          </td>
          <td>2025-02-21</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/622934218386f9dc97c13a0c545169d234cb7502" target='_blank'>
              Abstract A019: CRISPR-based investigation of CHIP mutations reveals their role in CAR-T therapy-related inflammatory toxicities
              </a>
            </td>
          <td>
            Xiuli Wang, S. Goldsmith, J. Rhee, Stephen J Forman, Chunyan Zhang, S. Armenian
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>