<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-24 09:34:13 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>69</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1830846c3b9289add1524e2178f4fc69c53b21d" target='_blank'>
              Deciphering the transcriptomic landscape of pyrazinamide resistance in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Ananthi Rajendran, Ahmed Kabir Refaya, K. Palaniyandi
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Radiation resistance presents a significant challenge in the treatment of triple-negative breast cancer (TNBC). To investigate the molecular adaptations associated with radiation therapy resistance, MDA-MB-231 cells were subjected to a repeated radiation (RR) regimen totaling 57 Gy over 11 weeks, followed by clonal selection. The resulting radiation-adapted cells (MDA-MB-231RR) were analyzed using whole-transcriptome RNA sequencing, revealing substantial dysregulation of pathways related to cell adhesion, mitochondrial function, and epithelial–mesenchymal transition (EMT). These transcriptional changes were corroborated by functional assays. MDA-MB-231RR cells exhibited reduced expression of adhesion receptors (ITGB1, ITGA2, ITGA6) and extracellular matrix proteins (fibronectin, collagen, laminins), accompanied by significantly impaired cell adhesion to fibronectin, collagen, and laminin substrates. Mitochondrial dysfunction was supported by downregulation of oxidative phosphorylation genes (MTCO1, MTND1) and confirmed by JC-1 dye assays demonstrating a marked reduction in mitochondrial membrane potential. EMT-associated changes included increased mesenchymal markers and loss of epithelial markers (CTNNB1, SNAI2, CK19), consistent with enhanced migratory potential. Taken together, this study delineates key molecular features of radiation adaptation in TNBC, providing a foundation for the development of targeted therapies to overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cab99416741bb4565c7ddddd5532aa9428991da" target='_blank'>
              Molecular Adaptations to Repeated Radiation Exposure in Triple-Negative Breast Cancer: Dysregulation of Cell Adhesion, Mitochondrial Function, and Epithelial–Mesenchymal Transition
              </a>
            </td>
          <td>
            Noah Dickinson, Alyssa Murray, Megan Davis, Kaitlyn Marshall-Bergeron, Jessica Dougherty, W. Al-Khayyat, Ramya Narendrula, Maggie Lavoie, Emma Mageau, Ronan Derbowka, A. T. Kovala, D. Boreham, Natalie Lefort, Christopher Thome, T. C. Tai, S. Tharmalingam, Upregulated DEGs, Downregulated DEGs
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Despite the outstanding achievements of precision medicine in hematology, many targeted therapies eventually fail due to the emergence of resistance mechanisms. Traditionally, a genocentric approach has been adopted to uncover the molecular underpinnings of treatment resistance. This has contributed to identifying resistance gene mutations and designing novel therapeutic molecules with increased potency for the mutant target. However, over the last five years, additional non-genetic adaptations have become increasingly recognized as crucial promoters of treatment resistance. In parallel, emerging works in the field of evolutionary biology suggest that advantageous phenotypic traits appear most often due to cell-intrinsic phenotypic plasticity and can arise independently of gene mutations. In selected cases, single genetic abnormalities such as those involving TP53 can prime human cells for plasticity and facilitate phenotypic variability. In this narrative review, we retrace the resistance mechanisms to targeted therapies in the framework of these novel evolutionary concepts. We highlight the dichotomy between genes-first and phenotypes-first pathways of treatment adaptation, with the former being driven by traditional single-point mutations and the latter initiated by the phenotypic diversity and the high-level plasticity of cancer cells. Focusing on resistance mechanisms to kinase inhibitors and BH3 mimetics in leukemias and lymphomas, we describe how each drug can trigger both escape routes, which may even coexist within the tumor bulk of individual patients. Lastly, we provide a three-step translational perspective on how to counteract phenotypes-first resistance mechanisms, with the aim of prolonging disease control in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/326e4b504dccc956ee299598bcd494d4433bc387" target='_blank'>
              Genes-first and phenotypes-first paths to treatment resistance in hematological malignancies
              </a>
            </td>
          <td>
            Edoardo Tamellini, Cristina Frusteri, Isacco Ferrarini
          </td>
          <td>2025-11-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Emergence of drug resistance is the main cause of therapeutic failure in patients with high-grade serous ovarian cancer (HGSOC)1. To study drug resistance in patients, we developed CloneSeq-SV, which combines single-cell whole-genome sequencing2 with targeted deep sequencing of clone-specific genomic structural variants in time-series cell-free DNA. CloneSeq-SV exploits tumour clone-specific structural variants as highly sensitive endogenous cell-free DNA markers, enabling the relative abundance measurements and evolutionary analysis of co-existing clonal populations over the therapeutic time course. Here, using this approach, we studied 18 patients with HGSOC over a multi-year period from diagnosis to recurrence and showed that drug resistance typically arose from selective expansion of a single or small subset of clones present at diagnosis. Drug-resistant clones frequently showed interpretable and distinctive genomic features, including chromothripsis, whole-genome doubling, and high-level amplifications of oncogenes such as CCNE1, RAB25, MYC and NOTCH3. Phenotypic analysis of matched single-cell RNA sequencing data3 indicated pre-existing and clone-specific transcriptional states such as upregulation of epithelial-to-mesenchymal transition and VEGF pathways, linked to drug resistance. In one notable case, clone-specific ERBB2 amplification affected the efficacy of a secondary targeted therapy with a positive patient outcome. Together, our findings indicate that drug-resistant states in HGSOC pre-exist at diagnosis, leading to positive selection and reduced clonal complexity at relapse. We suggest these findings motivate investigation of evolution-informed adaptive treatment regimens to ablate drug resistance in future HGSOC studies. A combined sequencing technique assesses 18 patients with high-grade serous ovarian cancer over a multi-year period from diagnosis to recurrence and shows drug resistance typically arises from selective expansion of one or a few clones present at diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, C. Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acquired resistance to immune checkpoint inhibitors (ICIs) remains a significant challenge in the treatment of metastatic melanoma. Phenotypic plasticity, such as dedifferentiation and transdifferentiation, is an increasingly recognized mechanism for treatment resistance. We present a case of a man in his 70s with metastatic melanoma who experienced progression through sequential treatments including pembrolizumab in combination with the HDAC inhibitor entinostat, and ipilimumab. During treatment a histologically distinct pleomorphic rhabdomyosarcoma (RMS) emerged at metastatic sites. Longitudinally acquired tumor samples representing both phenotypes were analyzed using whole- exome sequencing (WES), RNA sequencing (RNA-seq) and high-plex tissue imaging (spatial proteomics). WES revealed driver mutations (e.g. NRAS, NF1) and loss-of-heterozygosity (LOH) shared between phenotypes indicating a common ancestral clone. Phylogenetic analysis demonstrated an early divergence of the phenotypes, with each later acquiring unique mutations. RNA-seq showed mutually exclusive expression of lineage-specific markers as well as epithelial-mesenchymal transition and myogenic gene set enrichment in the RMS samples. High-plex imaging identified distinct tumor microenvironments, with RMS lesions enriched in CD163+ macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181bda14a56f6f64f5ac4fffa1ec57650a41f5a9" target='_blank'>
              Cooperative and antagonistic interactions between sub-clones favour the co-existence of multiple resistance mechanisms in melanoma
              </a>
            </td>
          <td>
            Karin Schlegelmilch, Viola Hollek, S. Hooper, Giovanni Giangreco, Sasha Bailey, Sarah C. Macfarlane, Alexandrine Carminati, Amy Bowes, S. Strohbuecker, Xiao Fu, E. Sahai
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, Fangfang Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, Preetesh Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd8a87b52ce435726d0713cb35d9987d7862c73" target='_blank'>
              The two sides of resistance: aggressiveness and mitotic instability as the Achilles’ heel of Osimertinib-resistant NSCLC
              </a>
            </td>
          <td>
            P. Armanetti, E. Cabrera San Millán, R. Mercatelli, Chiara Cavallini, Alice Chiodi, E. Mosca, Arianna Consiglio, Nicola Losito, Luca Menichetti, E. Bramanti, Azhar Ali, Elena Levantini, Giorgia Maroni
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Hypoxia is a well-recognized clinical feature of solid tumors, including lung cancer, and is associated with poor prognosis due to its role in promoting resistance to chemotherapy and radiotherapy. To investigate the cellular consequences of hypoxia, we cultured A549 lung adenocarcinoma cells under 1% O2 and examined their growth, cell cycle distribution, and redox status. Hypoxia significantly reduces cell proliferation and induced G1 phase cell cycle arrest, suggesting a cytostatic effect. Biochemical analysis showed a 2.64- and 2.04-fold increase in total and mitochondrial reactive oxygen species (ROS) levels, respectively, along with an elevated total thiol levels under hypoxic conditions compared to normoxia. To assess the reversibility of the hypoxic response, we performed a reciprocal oxygen exposure experiment where cells initially grown under hypoxia were re-exposed to normoxia, and vice versa. To explore the underlying molecular mechanism, we analyzed transcriptomic datasets (GEO accession: GSE48134 and GSE42416) which revealed that hypoxia downregulated key genes involved in energy metabolism (e.g., PDK4, G6PD), cell cycle progression (e.g., CCND1, CDK2), and redox regulation (e.g., GCLM, TXNRD1, NQO1, GCLC). Further, few of redox-related genes were validated by RT-PCR in A549 cells cultured under hypoxia and normoxia for 24 h. Importantly, cyclic hypoxia (intermittent hypoxia-reoxygenation) conditions showed partial restoration of some of these transcripts, supporting the transient nature of hypoxic stress, consistent with our in vitro observations. Furthermore, transcriptome profiles from adenocarcinoma patients (GEO accession: GSE30979) also match our cell line observations. Thus, our results clearly show that hypoxia causes a temporary cell cycle arrest in lung cancer cells, which is reversible when oxygen is restored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77bbcdb23f2ddac965c8e06176945c4fad76ded" target='_blank'>
              Hypoxia-drives reversible cell cycle arrest in lung cancer cells via modulation of cellular redox and gene expression
              </a>
            </td>
          <td>
            D. Maurya, Varshita Mehta, Babita Singh
          </td>
          <td>2025-09-29</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb2093f1cec488c24db178be2a0d7e4f15fc0ea7" target='_blank'>
              Leveraging death of drug-sensitive cancer cells to promote immune-mediated bystander killing of subclones of drug-resistant tumour cells
              </a>
            </td>
          <td>
            M. Tomaschko, KangBo Ng, C. Moore, Claire E. Pillsbury, S. Rana, James Campbell, Saptaparna Mukherjee, Ania Mikolajczak, P. Anastasiou, Andrea de Castro, Alicia Alonso de la Vega, Sophie de Carné Trécesson, Nate Goehring, M. Molina-Arcas, J. Downward
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and aggressive malignancy, characterized by a lack of targeted therapies and limited clinical benefits. Here, we conducted an optimized whole-genome CRISPR screen across five HNSCC cell lines aimed at identifying actionable genetic vulnerabilities for rapid preclinical evaluation as novel targeted therapies. Given their critical role in cancer, cyclin-dependent kinases (CDKs) were prioritized for further investigation. Among these, CDK7 was identified as an essential and targetable gene across all five cell lines, prompting its selection for in-depth functional and molecular characterization. Genetic and pharmacological inhibition of CDK7 significantly and consistently reduced tumor cell proliferation due to generalized cell cycle arrest and apoptosis induction. Additionally, CDK7 knockout (KO) and selective inhibitors (YKL-5-124 and samuraciclib) demonstrated potent antitumor activity, effectively suppressing tumor growth in HNSCC patient-derived organoids (PDOs), as well as in both cell line- and patient-derived xenograft (PDX) mouse models with minimal toxicity. Mechanistically, CDK7 inhibition led to a broad downregulation of gene sets related to cell cycle progression and DNA repair, and significantly reduced the transcription of essential genes and untargetable vulnerabilities identified by our CRISPR screen. These findings highlight CDK7 as a promising therapeutic target for HNSCC. Our study provides strong evidence of the robust antitumor activity of CDK7-selective inhibition in disease-relevant preclinical models, strongly supporting its progression to clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4085230d5db63fd39c16115dd67e53fe6113079" target='_blank'>
              CDK7-targeted therapy effectively disrupts cell cycle progression and oncogenic signaling in head and neck cancer
              </a>
            </td>
          <td>
            María Otero-Rosales, M. Álvarez-González, Irene Pazos, Beatriz de Luxán-Delgado, Sonia Del Marro, Esperanza Pozo-Agundo, Mar Rodríguez-Santamaría, Ana López-Fernández, D. Corte-Torres, R. Granda-Díaz, S. Álvarez-Teijeiro, Iván Fernández-Vega, Corina Lorz, Ramón García-Escudero, J. P. Rodrigo, Konstantinos Tzelepis, G. Vassiliou, Irene Ferrer, Mónica Álvarez-Fernández, J. García-Pedrero, F. Hermida-Prado
          </td>
          <td>2025-11-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Oncogenesis involves widespread genetic and epigenetic alterations, yet the full spectrum of somatic variation genome-wide remains unresolved. We generated a near-telomere-to-telomere (T2T) diploid assembly of a donor paired with deep short- and long-read sequencing of their melanoma. This revealed that 16% of somatic variants occur in sequences absent from GRCh38, with satellite repeats acting as hotspots for UV-induced damage due to sequence-intrinsic mutability and inefficient repair. Centromere kinetochore domains emerged as focal sites of structural, genetic, and epigenetic variation, leading to remodeling of centromere kinetochore binding domains during tumor evolution. Single-molecule telomere reconstructions uncovered cycles of attrition, deletion, and telomerase-mediated extension that shape cancer telomeres. Finally, diploid chromatin maps exposed that copy number alterations and epimutations, rather than point mutations, predominate in rewiring cancer regulatory programs. These findings define the full landscape of a cancer’s somatic variation and their functional impact, establishing a blueprint for T2T studies of mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, A. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, A. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Melanoma plasticity drives immune evasion and therapy resistance through dynamic cell-state transitions beyond genetic alterations. Epigenetic remodeling critically influences such processes, yet its role in reshaping the tumor ecosystem under therapeutic pressure remains unresolved. Here, we profiled longitudinal biopsies from melanoma patients treated in the phase Ib NIBIT-M4 epi-immunotherapy clinical trial (NCT02608437), testing the combination of a DNMT1 inhibitor with anti-CTLA4 using single-cell multiome and high-resolution spatial transcriptomics. Integrated analyses resolved seven malignant meta-programs, including a rare Wnt/β-catenin–driven melanocytic state and a de-differentiated neural crest–like state enriched in non-responders. Spatial modeling revealed that homotypic clustering stabilizes resistant programs, with neural crest–like cells forming compact, centrally localized niches, whereas Wnt/β-catenin subpopulations displayed a bimodal architecture, either cohesive clusters sustained by adhesion or dispersed, transcriptionally plastic cells. Responders exhibited progressive enrichment of an antigen presentation/interferon program and coordinated remodeling of the tumor microenvironment with T and B cell expansion, whereas tumors from non-responder patients maintained stable composition of neural crest–like clusters. Epigenetic therapy reactivated transposable elements, providing both regulatory signals that prime innate immunity within microenvironment and generating antigens that drive immunoediting and immunogenicity of Antigen presentation/interferon cell states in responders. Finally, NFATC2 emerged as a master regulator of neural crests–like transcriptional phenotypes and promoter of resistance to therapeutic interventions in melanoma patients. NFATC2 perturbation was able to shift tumor cells towards more differentiated and immunogenic states. These findings reveal how epigenetic-based immunotherapy reshapes melanoma ecosystems, provide mechanistic insights into how multiple transcriptional programs promote tumor plasticity and resistance to both combinatorial therapies and immune checkpoint blockade, identify spatial clustering as a principle stabilizing resistant niches, and highlight β-catenin and NFATC2 as actionable vulnerabilities to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Aim: This study aimed to elucidate the role of regulator of chromosome condensation 1 (RCC1) in colorectal cancer (CRC) progression, as well as its involvement in chemoresistance. We specifically examined how RCC1 knockdown modulates cellular responses, including cell cycle, apoptosis, and senescence induced by 5-fluorouracil (5-FU) or doxorubicin (Doxo) in both parental and drug-resistant CRC cell lines. Additionally, we assessed the potential of RCC1 inhibition as an adjuvant therapeutic strategy to enhance the efficacy of chemoradiotherapy in CRC. Methods: The expression of RCC1 in colon cancer tissues and corresponding adjacent non-cancerous tissues was evaluated through tissue microarrays, and its correlation with characteristics and patient prognosis was also examined. Subsequently, a series of in vivo and in vitro experiments based on parental and drug-resistant CRC cell lines were conducted to assess the impact of RCC1 knockdown on sensitivity to 5-FU or Doxo. Finally, transcriptomic analysis and subsequent validation assays were performed to explore the underlying molecular mechanisms. Results: RCC1 knockdown significantly enhanced the antitumor efficacy of 5-FU and Doxo in both CRC and drug-resistant CRC cells. In xenograft models, RCC1 knockdown in combination with 5-FU or Doxo suppressed tumor growth with no evident systemic toxicity observed. Transcriptomic profiling and experimental verification revealed that RCC1 knockdown may impair DNA repair by downregulating key repair proteins, thereby leading to more severe and sustained DNA damage. Conclusion: Our results indicate that RCC1 downregulation enhances the responsiveness of both parental and drug-resistant CRC cells to 5-FU and Doxo, highlighting its potential as a therapeutic adjunct to improve clinical outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8afbd5b9e059c22aa8009a12c6871018f16b21" target='_blank'>
              RCC1 knockdown sensitizes drug-resistant colorectal cancer to 5-fluorouracil or doxorubicin by impairing DNA repair
              </a>
            </td>
          <td>
            Jing Li, Ya Meng, Xumei Ouyang, Xiaowen Lin, Yangzhe Wu, Hang Fai Kwok
          </td>
          <td>2025-11-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, Wei Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d57534bc0d9d0ab660dd7d7cddbf910c13f4aa" target='_blank'>
              Strain-resolved CRISPRi-Seq reveals conserved antibiotic vulnerabilities in Staphylococcus aureus
              </a>
            </td>
          <td>
            Maria-Vittoria Mazzuoli, Jessica Burnier, Albane Schmid, Louise Martin, Axel B. Janssen, Clément Gallay, A. Zinkernagel, J. Veening
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d87abfb015ce97c63d2404657d94409eac97df" target='_blank'>
              A Sequential Triple-Drug Strategy for Selective Targeting of p53-Mutant Cancers
              </a>
            </td>
          <td>
            M. Alruwaili, Yanqi Guo, Justin Zonneville, T. Melendy, R. Straubinger, Barbara A. Foster, Priya T. Rajan, Henry G. Withers, Sarah N. Chatley, Renuka Iyer, C. Fountzilas, A. Bakin
          </td>
          <td>2025-11-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) presents significant therapeutic challenges, particularly due to the development of resistance to standard chemotherapeutic agents such as irinotecan. In this study, we aimed to investigate the molecular and phenotypic mechanisms underlying irinotecan resistance in CRC using the LIM1215 cell line model. Transcriptomic analysis demonstrated that drug withdrawal induced major transcriptional reprogramming, characterized by downregulation of ABC transporters (ABCB1 and ABCG2), extracellular matrix-related genes, and markers of epithelial-to-mesenchymal transition (EMT), alongside reactivation of cell cycle pathways. Drug screening further indicated that resistant cells maintained under irinotecan pressure exhibited a multidrug-resistant phenotype, while withdrawn cells regained sensitivity, particularly to tyrosine kinase inhibitors. Supplementation with the efflux inhibitor Elacridar partially restored drug sensitivity in resistant cells, emphasizing the role of transporter-mediated efflux in maintaining resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3002080effb079c5d4239d8ceb95fa35b9194a90" target='_blank'>
              Phenotypic plasticity including drug efflux drives reversible irinotecan resistance in LIM1215 colorectal cancer cells.
              </a>
            </td>
          <td>
            Tugce Di̇lber, Gizem Damla Yalcin, Kübra Yılmaz, Ahmet Acar
          </td>
          <td>2025-10-28</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer cells frequently reprogramme their metabolism to support growth and survival, making metabolic pathways attractive targets for therapy. In this study, we investigated the metabolic effects of three kinase inhibitors and their synergistic combinations in the gastric cancer cell line AGS using genome-scale metabolic models and transcriptomic profiling. We applied the tasks inferred from the differential expression (TIDE) algorithm to infer pathway activity changes in the different conditions. We also explored a variant of TIDE that uses task-essential genes to infer metabolic task changes, providing a complementary perspective to the original algorithm. Our results revealed widespread down-regulation of biosynthetic pathways, particularly in amino acid and nucleotide metabolism. Combinatorial treatments induced condition-specific metabolic alterations, including strong synergistic effects in the PI3Ki–MEKi condition affecting ornithine and polyamine biosynthesis. These metabolic shifts provide insight into drug synergy mechanisms and highlight potential therapeutic vulnerabilities. To support reproducibility, we developed an open-source Python package, MTEApy, implementing both TIDE frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3017a5eb28e4bc7906f6c3abd507b3d2f862ad" target='_blank'>
              Constraint based modeling of drug induced metabolic changes in a cancer cell line
              </a>
            </td>
          <td>
            Xavier Benedicto, Åsmund Flobak, M. Ponce-de-León, Alfonso Valencia
          </td>
          <td>2025-10-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aneuploidy causes genome plasticity and enables adaptive responses that confer drug resistance in eukaryotes ranging from fungal pathogens to human cancer cells. Aspergillus fumigatus is a soil-resident fungus and the most common cause of invasive fungal infections globally. Aspergillosis has an alarmingly high mortality rate due in part to treatment failure resulting from widespread antimicrobial resistance and an ability of the fungus to dynamically adapt to changing environmental and host conditions. Our understanding of these treatment failures is hindered by gaps in knowledge about how this organism adapts to resist antimicrobial stress. We find that exposure to FK506, an antifungal and immunosuppressive compound produced by Streptomyces, selects for whole-chromosome aneuploidy in this fungus. Growth without FK506 results in rapid aneuploid chromosome loss and reversion to the euploid, drug-susceptible state. These chromosome duplications also reduce susceptibility to clinically-used azole antifungals, underscoring the utility of aneuploidy as a dynamic stress adaptation strategy. Transcriptomic analysis revealed that Chr7 disomy leads to induction of the normally-silent neosartoricin biosynthetic gene cluster in response to FK506 exposure, and we demonstrate that constitutive genetic induction of this cluster recapitulates in the euploid background the response to FK506 in the aneuploid state. We further show that this response is independent from the production of neosartoricin, suggesting a role for cross-pathway metabolic changes resulting from activation of this gene cluster or from the aneuploid state. This study represents a major advance in understanding how aneuploidy leads to transient adaptation and antimicrobial drug resistance in a global human fungal pathogen. Highlights ‒ fumigatus rapidly gains and loses chromosomes in response to antimicrobial stress. ‒ Aneuploidy leads to large-scale transcriptional and metabolic changes. ‒ Overexpression of nscR is sufficient to confer FK506 tolerance in the euploid. ‒ Aneuploidy reduces susceptibility to both FK506 and azole antifungals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c573e7d40354c3158261048e704da74d1dc94a5" target='_blank'>
              Whole-chromosome duplications drive antimicrobial resistance in Aspergillus fumigatus
              </a>
            </td>
          <td>
            Anna E. Lehmann, Enrique Aguilar Ramírez, Nancy P. Keller, Joseph Heitman
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, Jonas Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, O. Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, Gitta Boons, Junbin Qian, Stein Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oxaliplatin resistance remains a critical barrier to effective colorectal cancer treatment. The molecular mechanisms underlying this resistance are not fully understood, highlighting the need to define the transcriptional alterations that contribute to therapeutic failure. Accordingly, a comparative transcriptome analysis was performed on oxaliplatin-resistant colorectal cancer cells (HCT-116-ROx) and their parental counterparts (HCT-116) using RNA sequencing in this study. Differentially expressed gene (DEG) analysis was conducted using a quasi-likelihood negative binomial model. Gene Ontology (GO) enrichment and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment were carried out using the topGO and clusterProfiler packages, respectively. To confirm the robustness of the transcriptomic data, the genes with the most significant expression changes, based on false discovery rate-adjusted P value less than 0.05 and a |logFC| > 2 thresholds, were selected for validation by quantitative real-time PCR (qRT-PCR). A total of 313 DEGs were identified, including ALDH3A1 and TACSTD2 (upregulated) and IFITM1 (downregulated); these three genes were chosen for validation by qRT-PCR. Gene Ontology enrichment revealed significant changes in cell motility, redox regulation, and extracellular matrix remodeling. KEGG analysis indicated upregulation of ferroptosis, glutathione metabolism, and lysosome-related pathways, and downregulation of p53 signaling, oxidative phosphorylation, and cancer-specific pathways. Oxaliplatin-resistant colorectal cancer cells undergo multifaceted transcriptional reprogramming that promotes redox homeostasis, metabolic adaptation, and structural plasticity while suppressing apoptotic and mitochondrial functions. These changes support chemoresistance and may represent potential therapeutic targets to restore drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5ab501b314c92e8ff167ffcb8251950e164cb1" target='_blank'>
              Multifaceted transcriptional reprogramming supports oxaliplatin chemoresistance in colorectal cancer cells.
              </a>
            </td>
          <td>
            Gizem Calibasi-Kocal, Hasan Kurter, Zerrin Isik
          </td>
          <td>2025-10-09</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT MYC is among the most frequently dysregulated oncogenes in human cancer, yet its direct targeting remains a significant challenge. Here, we present an in‐silico integrative screening approach to identify compounds and combinations that can block MYC's oncogenic function by specifically disrupting its transcriptional regulatory function. Using a doxycycline (DOX)‐inducible model, we established a MYC loss‐of‐function (LOF) gene signature that specifically captures the molecular consequences corresponding to the loss of MYC's ability in transcriptional regulation. By integrating large‐scale post‐perturbation transcriptomic profiling from the CMAP database, we screened over 8300 drug‐induced profiles and identified 70 recurrent compounds that are predicted to antagonize MYC's transcriptional programs. To further enhance their therapeutic potential, we also developed an orthogonality analysis to pinpoint synergistic drug combinations that suppress MYC activity more effectively than single agents. Our scalable framework enables a rational and systematic identification of compounds with potential to antagonize MYC's oncogenic function by disrupting its transcriptional regulatory ability without necessarily decreasing its abundance. Our approach provides new insights on utilizing existing anticancer drugs to indirectly target MYC in MYC‐driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f19d2dd4a2e3e24bf8c740f6f2a3bb3a7026742" target='_blank'>
              An Integrative Drug‐Induced Transcriptomic Analysis Identifies Novel MYC Antagonists and Potential Synergistic Drug Combinations
              </a>
            </td>
          <td>
            Anthony Aceto, Yue Wang, Da Yang
          </td>
          <td>2025-09-30</td>
          <td>Molecular Carcinogenesis</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancer (OC) is lethal due to near universal development of resistance to platinum-based chemotherapy. Metabolic adaptations can play a pivotal role in therapy resistance. Here, we aimed to identify key metabolic pathways that regulate platinum response and represent potential therapeutic targets. Transcriptomic and metabolomic analyses in cisplatin sensitive and resistant ovarian cancer cells identified enrichment of pyrimidine metabolism related to upregulated de novo pyrimidine synthesis. 15N-glutamine flux analysis confirmed increased de novo pyrimidine synthesis in cisplatin resistant cells. Targeting this pathway using brequinar (BRQ), an inhibitor of the key enzyme dihydroorotate dehydrogenase (DHODH), decreased cell viability, delayed G2/M cell cycle progression, and altered expression of genes related to mitochondrial electron transport in resistant cells. Under basal conditions, cisplatin resistant cells had a lower oxygen consumption rate (OCR) and spare respiratory capacity (SRC) than sensitive cells. BRQ suppressed OCR in both sensitive and resistant but only inhibited SRC in resistant cells. In cell line-derived and patient-derived xenograft models, BRQ attenuated the growth of cisplatin resistant ovarian tumors and enhanced the inhibitory effects of carboplatin. Together, these results identify metabolic reprogramming in cisplatin resistant ovarian cancer that induces an acquired dependency on de novo pyrimidine synthesis, which can be targeted to sensitize tumors to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d708b6c5cad44de1ded3e0fd0aa39875db711a4c" target='_blank'>
              Metabolic Dependency on De Novo Pyrimidine Synthesis is a Targetable Vulnerability in Platinum Resistant Ovarian Cancer.
              </a>
            </td>
          <td>
            H. Cardenas, Yinu Wang, Guangyuan Zhao, Delan Xingyue Hao, A. Isac, Vanessa Hernandez, Ujin Kim, Wenan Qiang, Hao F. Zhang, Daniela Matei
          </td>
          <td>2025-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The vast majority of recurrent somatic mutations arising in tumors affect protein-coding genes in the nuclear genome. Here, through population-scale analysis of 14,106 whole tumor genomes, we report the discovery of highly recurrent mutations affecting both the small (12S, MT-RNR1) and large (16S, MT-RNR2) mitochondrial RNA subunits of the mitochondrial ribosome encoded within mitochondrial DNA (mtDNA). Compared to non-hotspot positions, mitochondrial rRNA hotspots preferentially affected positions under purifying selection in the germline and demonstrated structural clustering within the mitoribosome at mRNA and tRNA interacting positions. Using precision mtDNA base editing, we engineered models of an exemplar MT-RNR1 hotspot mutation, m.1227G>A. Multimodal profiling revealed a heteroplasmy-dependent decrease in mitochondrial function and loss of respiratory chain subunits from a heteroplasmic dosage of ~10%. Mutation of conserved positions in ribosomal RNA that disrupt mitochondrial translation therefore represent a class of functionally dominant, pathogenic mtDNA mutations that are under positive selection in cancer genomes. Analysis of 14,106 tumor genomes highlights recurrent mutations in mitochondrial ribosomal RNA encoded within the mitochondrial genome. Mutations occur at hotspot positions and are under strong purifying selection in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eda054c049bfbd95fe327306b7323841c22683" target='_blank'>
              Functionally dominant hotspot mutations of mitochondrial ribosomal RNA genes in cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Jacqueline Tait-Mulder, Minsoo Kim, Cerise Tang, Tricia Park, Flora McNulty, S. Lilla, S. Zanivan, Alejandro Huerta-Uribe, Benan N. Nalbant, Mark Zucker, D. Sumpton, Geoffray Monteuuis, Christopher B Jackson, Wei Wei, P. Chinnery, R. Chaligné, Caleb A. Lareau, Ed Reznik, P. Gammage
          </td>
          <td>2025-11-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573448152ca34657743158301dc6d5b323113cd1" target='_blank'>
              TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition
              </a>
            </td>
          <td>
            Jean-François Spinella, J. Chagraoui, I. Boivin, Guillaume Richard Carpentier, Céline Moison, N. Mayotte, François Béliveau, Vincent-Philippe Lavallée, J. Hébert, Guy Sauvageau
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Multidrug resistance is a significant barrier in cancer therapy largely due to poorly understood regulatory mechanisms. Here we reveal that certain anticancer drugs can bind to newly synthesized proteins prior to reaching their canonical targets, resulting in various forms of protein damage. This binding disrupts protein functions, particularly those of mitochondrial proteins, resulting in substantial cytotoxicity. The protein damage is further exacerbated by mitochondrial reactive oxygen species generated as a consequence of the initial damage, creating a positive feedback loop. In response, cancer cells rapidly initiate a chain of events, which we term the Protein Damage Response (PDR). This includes damage recognition primarily mediated by protein ubiquitination and subsequent damage clearance via the proteasome system. Notably, patients with advanced, drug-resistant metastatic breast or colon cancers exhibit elevated proteasome activity. In an effort to predict drug resistance, we developed a sensitive kit for detecting proteasome levels, enabling the identification and subtyping of patients with high proteasome activity to support tailored therapeutic strategies. Using a three-dimensional tumor slice culture-based drug sensitivity assay and an investigator-initiated clinical trial, we demonstrate that three clinically approved proteasome inhibitors effectively overcome multidrug resistance in colon and breast cancer patients with elevated proteasome activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1785188ad7f7984f485f66a5779a54c95bdf57ed" target='_blank'>
              Suppressing protein damage response to overcome multidrug resistance in cancer therapy
              </a>
            </td>
          <td>
            Fangyuan Shao, Zongjie Li, Hao Xiao, Yujun Chen, Yuheng Zhang, Ling Li, Yuzhong Peng, Xinyi Li, Yuxing Hou, Bo Li, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Xi Han, Jiaxin Yao, Cuiting Zhang, Lijian Wang, Haifeng Li, N. Shao, Kai Miao, Xiaoling Xu, Yanxia Shi, Changhua Zhang, Jun Yan, Ying Lin, Chu-Xia Deng
          </td>
          <td>2025-09-30</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81b381c14e5b1230a77d230dfeaa5698f06f2cc" target='_blank'>
              Characterizing resistance in prostate cancer at a single cell level with hormonal treatment and epigenetic inhibitors.
              </a>
            </td>
          <td>
            Morgan Howells, Priyadarsini Gangadharannambiar, Calum McMullen, Francesco Crea, W. Bacon, Colette Christiansen
          </td>
          <td>2025-11-03</td>
          <td>The Journal of pharmacy and pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Refractory cancers may arise either through the acquisition of resistance mechanisms or represent distinct disease states. The origin of childhood T-cell acute lymphoblastic leukaemia (T-ALL) that does not respond to initial treatment, i.e. refractory disease, is unknown. Refractory T-ALL carries a poor prognosis and cannot be predicted at diagnosis. Here, we perform single cell mRNA sequencing of T-ALL from 58 children (84 samples) who did, or did not respond to initial treatment. We identify a transcriptionally distinctive blast population, exhibiting features of innate-like lymphocytes, as the major source of refractory disease. Evidence of such blasts at diagnosis heralds refractory disease across independent datasets and is associated with survival in a large, contemporary trial cohort. Our findings portray refractory T-ALL as a distinct disease with the potential for immediate clinical utility. T-cell acute lymphoblastic leukemia is a highly aggressive disease with varying recurrence rates. Here, the authors build a single cell transcriptomic atlas of childhood T-cell acute lymphoblastic leukaemia (T-ALL). They identified a distinctive cancer cell state that correlates with high risk, treatment refractory T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb3ad5f4d75526c87385fa8cfc2e02d2569e276" target='_blank'>
              A non-canonical lymphoblast in refractory childhood T-cell leukaemia
              </a>
            </td>
          <td>
            Bram S. J. Lim, Holly J. Whitfield, Mi K. Trinh, Gianna Bloye, Rebecca Thomas, Nathaniel D. Anderson, Anna Wenger, A. Hodder, T. D. Treger, H. Lee-Six, Tim H. H. Coorens, Conor Parks, Toochi Ogbonnah, P. Pölönen, C. Mullighan, D. Teachey, Jason Xu, Kai Tan, Melanie M. Hagleitner, Lennart A. Kester, F. V. van Leeuwen, Gordon Beattie, Marc R. Mansour, Owen Williams, Jack Bartram, S. Adams, Laura Jardine, Sam Behjati, D. O'connor
          </td>
          <td>2025-11-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation. Radiation-induced malignantly transformed cells undergo multi-scale 3D genome reorganization. TP63 acts as a key mediator, creating accessible chromatin sites that form tumour-specific loops activating the MYC oncogene during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive neuroendocrine malignancy characterized by rapid onset of chemoresistance and poor clinical outcomes. Transcriptional heterogeneity among treatment-naïve SCLC tumors underlies four transcriptional subtypes, each with distinct clinical vulnerabilities. Though previously hypothesized to delineate a distinct subtype, expression of YAP1 is largely absent from treatment-naïve, pure SCLC. To characterize relapsed SCLC, circulating tumor DNA, circulating tumor cells, and core needle biopsies from SCLC patients and preclinical models following resistance to standard-of-care therapies were analyzed. In contrast to treatment-naïve SCLC, these analyses reveal an emergent YAP1-positive cell population that coincides with treatment resistance. These YAP1-positive cells exhibit characteristics of drug tolerant persister cells, including senescence, stemness, and plasticity, as YAP1 positive cells largely abandon features characteristic of SCLC to adopt those of large-cell neuroendocrine carcinoma (LCNEC). As a result of this SCLC-like to LCNEC-like evolution, YAP1-positive cells lack several clinically relevant SCLC surface targets (i.e., DLL3, SEZ6), but are enriched for others (i.e., B7-H3, TROP2). We propose a model where YAP1 expressing cells emerge with SCLC treatment resistance and characterize a tenacious subpopulation capable of diverging from the treatment naïve lineage and adopting features to evade therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, A. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, M. Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20106bc7c27e870a9fd1562421c545c6ddaef656" target='_blank'>
              Multi-omics-driven kinetic modeling reveals metabolic vulnerabilities and differential drug-response dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Ilias Toumpe, M. Masid, V. Hatzimanikatis, L. Miskovic
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The regulatory mechanisms governing transcriptional programs in the cancer genome remain elusive, particularly those concerning cell-type specificity. We carefully curated single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) and single-cell RNA sequencing (scRNA-seq) data from eight distinct carcinoma tissues, including breast, skin, colon, endometrium, lung, ovary, liver, and kidney. Using single-cell multi-omics analysis, we identified extensive open chromatin regions and constructed peak-gene link networks, which can reveal distinct cancer gene regulation and genetic risks. We further explored conserved epigenetic regulation across cell types within cancer and elucidated their functional implications. Moreover, we identified cell-type-associated transcription factors (TFs) that regulate key cellular functions, such as the TEAD family of TFs, which widely control cancer-related signaling pathways in tumor cells. In colon cancer, we further identified tumor-specific TFs that are more highly activated in tumor cells than in normal epithelial cells, including CEBPG, LEF1, SOX4, TCF7, and TEAD4, which are pivotal in driving malignant transcriptional programs and represent potential therapeutic targets, as corroborated by single-cell sequencing data from multiple sources and in vitro experiments. Our findings provide a comprehensive understanding of the regulatory dynamics underlying carcinomas and offer valuable insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdeeefa6fa9c95e73d3a569b4614cf1e9a4ee17" target='_blank'>
              Single-cell multi-omics analysis reveals cancer regulatory elements of transcriptional programs and clinical implications
              </a>
            </td>
          <td>
            Xiaowei Tang, Qiaoling Zhang, Zichu Shen, Jian Xiao, Minghao Li, Xiangyan Meng, Chenyi Wang, Guangze Zhang, Anhang Liu, Yuxin Yin
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="HPV-negative (−) head and neck squamous cell carcinoma (HNSCC) is a highly heterogeneous cancer characterized by high mutational burden, an immunosuppressive microenvironment, and poor response to standard therapies. These features highlight the urgent need for novel and more effective treatment strategies. Drug sensitivity prediction was performed using integrated datasets from TCGA, GDSC, and CCLE. To assess the therapeutic potential and underlying mechanisms of combining the CDK inhibitor AZD5438 with the MEK1/2 inhibitor PD0325901, we employed a comprehensive panel of HNSCC models, including established cell lines, orthotopic mouse tumor models, and patient-derived organoids (PDOs). Lipid nanoparticles (LNPs) were engineered to co-deliver both agents into the same cancer cell populations. The tumor secretome was profiled using biotinylation coupled with liquid chromatography-mass spectrometry (LC-MS). Molecular alterations were examined by immunofluorescence, immunohistochemistry, ELISA, flow cytometry, and Western blot. Our bioinformatics analysis identified AZD5438 and PD0325901 as two of thirteen candidate drugs whose sensitivity is consistently associated with the five most frequently mutated genes in HPV (−) HNSCC. Notably, among these candidates, AZD5438 and PD0325901 exhibited the lowest correlation in their sensitivity profiles, suggesting complementary mechanisms of action. In experimental models, the combination of AZD5438 and PD0325901 not only outperformed either monotherapy in suppressing tumor growth but also augmented CD8⁺ T cell-mediated antitumor immunity by promoting caspase-8/gasdermin E-dependent pyroptosis. Furthermore, in both orthotopic tumor-bearing mice and PDOs, the LNP-encapsulated drug combination produced significantly greater therapeutic efficacy compared with the free drug formulation. Our findings indicate that the combination of AZD5438 and PD0325901 holds therapeutic potential for the treatment of HPV (−) HNSCC, particularly in tumors with a high mutational burden. By targeting complementary pathways, this combination may improve treatment outcomes in this aggressive cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c2bb6bc0e733d8972e63698f07f227976cd16a" target='_blank'>
              Dual CDK and MEK Inhibition potentiates CD8+ T cell-mediated antitumor immunity by inducing pyroptotic cell death in high-mutational head and neck cancer
              </a>
            </td>
          <td>
            Fanghui Chen, Fan Yang, David O. Popoola, Jianqiang Yang, Chris Tang, Alexis Payne, Lynn Zhang, N. C. Schmitt, Jin Xie, Nabil F. Saba, Yamin Li, Yong Teng
          </td>
          <td>2025-11-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite significant progress, the treatment of estrogen receptorpositive (ER+) breast cancer remains clinically challenging due to reversible drug resistance and immune evasion. Drug resistance often arises as cells undergo a dynamic epithelial-to-mesenchymal transition (EMT), while elevated PD-L1 levels contribute to immune escape. While these phenotypic features can variably co-occur, the impact of co-occurrence on the availability of synergistic treatment strategies remains unknown. To investigate their interplay, we constructed an ER-EMT-PD-L1 gene regulatory network and simulated these networks as coupled ordinary differential equations with biologically informed parameters, to generate steady-state expression profiles. Our study revealed that the relevant overarching network generated antagonistic epithelial and mesenchymal modules, capable of producing monostable, bistable, and tristable dynamics. We further examined the link between phenotypes and immune evasion by quantifying average PD-L1 expression, and found that epithelial-sensitive states consistently exhibited low PD-L1. In contrast, hybrid- and mesenchymal-resistant states were associated with high PD-L1, highlighting a strong coupling between EMT, resistance, and immune evasion. Extending on these network-level insights, we further used a spatially explicit agent-based model seeded with GRN-derived phenotypes to probe tumor behavior under therapeutic pressure. Simulations revealed that tumor escape required co-occurrence of therapy resistance, motility, and immune suppression, with plasticity and multistability further promoting adaptive persistence. Lastly, we identified combination therapies predicted to constrain malignant diversification and enhanced immune accessibility. Taken together, our modeling work links regulatory dynamics with tumor-level adaptation and underscores potential strategies to therapeutically reprogram cell states toward sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery.
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37222bf4d60567842a4aa73e1f34998e136ade83" target='_blank'>
              Translatome and translation dynamics analysis of a RiboCancer cell line panel reveals that leukemia-associated Rps15 mutations rewire translation through codon-specific tRNA accommodation defects
              </a>
            </td>
          <td>
            Anaïs Astier, Marino Caruso, Stijn Vereecke, Paulo E. Santo, Coralie Capron, C. Froment, Dana Rinaldi, David Cabrerizo Granados, Marine Leclercq, Jonathan Royaert, J. Verbeeck, Naomy Pasau, L. Plassart, Daniele Pepe, Steven Verbruggen, Hermes Paraqindes, Simon Lebaron, V. Marcel, Sébastien Durand, Clément Chapat, G. Menschaert, Pierre Close, F. Rapino, Frédéric Catez, J. Marcoux, Célia Plisson-Chastang, Kim De Keersmaecker
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The treatment of infections by Acinetobacter baumannii, a clinically significant nosocomial gram-negative bacterial pathogen, is hampered by antibiotic resistance, which is exacerbated by its exceptional genetic plasticity. Knowledge of the dynamics and mechanisms underlying the acquisition of antibiotic resistance is essential for the proper stewardship of their utilization. Here, we used a continuous culture device (morbidostat) to characterize the evolutionary trajectories of two A. baumannii strains in response to the increasing pressure of last-resort drugs, tigecycline and colistin. This approach allows us to confidently and comprehensively map resistance-driving mutations while circumventing both the “driver vs passenger” uncertainty and “selection bottleneck” limitations characteristic of clinical isolate analysis and conventional laboratory evolution, respectively. Tigecycline resistance predominantly occurred through the combination of missense mutations in the adeSR two-component system and disruptive events in the S-adenosyl methionine (SAM)-dependent methyltransferase, trm, while colistin resistance predominantly occurred through missense mutations in the gene cluster responsible for lipid A phosphoethanolamine modification, pmrCAB. Mapping of these mutational events over numerous publicly available A. baumannii genomes identified a relatively low prevalence of resistance to these two drugs. This work represents an initial step toward predictive resistomics of A. baumannii, leveraging gene-level genomic variations in addition to the conventional approaches based on the presence or absence of antibiotic resistance genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6157696474e55a2be83838d634d994f26454b5" target='_blank'>
              Mutations in two-component signaling systems drive experimental evolution of tigecycline and colistin resistance in Acinetobacter baumannii
              </a>
            </td>
          <td>
            J. E. Kent, M. Elane, S. Leyn, J. Zlamal, N. Wong, M. Aizin, C. Zampaloni, S. Louvel, A. Haldimann, M. Vercruysse, A. Osterman
          </td>
          <td>2025-10-29</td>
          <td>Antimicrobial Agents and Chemotherapy</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8838d113f21c285c0455db7c93dfda5adc298f" target='_blank'>
              A switch from histone methyltransferase-EZH2 to demethylase KDM6A activity marks reinitiation of proliferation of drug treated colorectal cancer cells
              </a>
            </td>
          <td>
            Subhashree Chatterjee, Ritika Jaiswal, Aniruddha Roy, S. Chowdhury, Sudeshna Mukherjee, Rajdeep Chowdhury
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            S. Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Gemcitabine is the primary chemotherapeutic agent used as a first-line treatment for pancreatic cancer (PC), underscoring its pivotal role in cancer treatment. This drug induces apoptosis through multiple mechanisms, with the disruption of cellular pyrimidine metabolism being a crucial mode of action. However, gemcitabine resistance significantly limits the long-term efficacy of PC treatments. Our study aimed to identify potential targets for gemcitabine sensitization. We identified a gene set associated with pyrimidine metabolism through cluster analysis of The Cancer Genome Atlas Pancreatic Adenocarcinoma database. Intersecting this set of differentially upregulated genes in PC with normal tissues revealed inhibin βA (INHBA) as a gene related to pyrimidine metabolism. Quantitative reverse transcription-polymerase chain reaction, western blotting, and immunohistochemistry analyses confirmed INHBA overexpression in PC cells. INHBA’s effects of INHBA on cell proliferation and chemoresistance in PC cells were validated in vitro using a colony counting kit-8 assay, drug sensitivity analysis, EdU incorporation, flow cytometry, and colony formation assay. The proliferation and chemoresistance promotion by INHBA in PC cells was validated in vivo using a mouse xenograft model. Downstream targets of INHBA were identified using immunoprecipitation mass spectrometry. INHBA, a pyrimidine metabolism-associated gene, was highly expressed in PC, and its expression correlated with gemcitabine resistance. INHBA interacts with cytidine triphosphate synthase 1 (CTPS1) and competitively inhibits SMAD Specific E3 Ubiquitin Protein Ligase 1 mediated ubiquitination, enhancing CTPS1 stability and promoting pyrimidine metabolism. INHBA promotes gemcitabine resistance in PC by stabilizing CTPS1 and enhancing pyrimidine metabolism, making it a potential target for chemosensitization in PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aea2b2ecd7e98f320aad3799b76a99f906f124b0" target='_blank'>
              INHBA promotes chemoresistance in pancreatic cancer by enhancing CTPS1 stability and mediating pyrimidine metabolism
              </a>
            </td>
          <td>
            Xinghao Sun, Zikai Wei, Xingkun Liao, Dijie Zheng, Futang Li, Liwen Chen, Yanyu Gong, Suye Ran, Xiaozhao Han, Zhiwei He, Chao Yu
          </td>
          <td>2025-11-14</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df66d9d10b79290100e939f48a6a4eda6aef615a" target='_blank'>
              NR0B1 alters the 9-cis-retinoic acid response in Ewing Sarcoma cells
              </a>
            </td>
          <td>
            Benjamin D. Sunkel, Meng Wang, Rachel D Dreher, M. Sammons, Lindsay Ryan, A. Gross, Scott Friedland, Ryan D. Roberts, Emily R. Theisen, Elaine R. Mardis, Richard K. Wilson
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The human somatic genome evolves throughout our lifespan, producing mosaic individuals comprising clones harboring different mutations across tissues. While clonal expansions in the hematopoietic system have been extensively characterized and reported to be nearly ubiquitous, clonal mosaicism (CM) has more recently also been described across multiple solid tissues. However, outstanding questions remain about the parameters and processes of human somatic evolution in non-cancerous solid human tissues, including when clones arise, how they evolve over time, and what mechanisms lead to their expansion. Questions of timing and clonal dynamics can be addressed through phylogenetic reconstruction, which serves as a ‘temporal microscope’, while uncovering the mechanisms of expansion necessitates simultaneous phenotypic profiling. To address this gap, here we develop Single-cell Miniaturized Automated Reverse Transcription and Primary Template-directed Amplification (SMART-PTA) for joint single-cell whole-genome and whole-transcriptome sequencing for large scale and cost efficient interrogation of solid tissue CM. We established a workflow that generates hundreds of matched single-cell whole genome and transcriptome libraries within a week. We profiled phenotypically normal esophagus tissue from four aged donors and used somatic variants to build high-resolution single-cell lineages from >2,700 cells with accompanying transcriptomic information, reconstructing >70 years of somatic evolution. T cell expansions identified from T cell receptor (TCR) sequences validated the clonal structure of the single-nucleotide variant (SNV)-based phylogenies and phylogenetic cross-correlation analysis showed that epithelial cells had higher degrees of shared ancestry by spatial location compared to immune cells. Mapping mutation signatures to the phylogenetic tree revealed the emergence of tobacco/alcohol exposure-related signatures later in life, consistent with the donors’ exposure histories. We identified variants in driver genes that were previously reported in the phenotypically normal esophagus, detecting clonal expansions harboring mutations in genes including TP53 and FAT1. We mapped the evolution of clones with both monoallelic as well as biallelic TP53 loss, including a clone associated with high expression of cell cycling genes and higher chromosome instability. Leveraging the matched transcriptome data, we uncovered cell type biases in mutant clones, with a higher proportion of TP53 or FAT1-mutant cells in an earlier basal epithelial cell state compared to wild-type cells. We further observed copy-neutral loss of heterozygosity (CNLOH) events on chromosome 9q that spanned the NOTCH1 locus in up to ∼35% of epithelial cells. Mapping CNLOH events to the phylogenetic tree revealed a striking pattern in which CNLOH was separately acquired many times, reflecting convergent evolution. Cells with CNLOH events were biased towards the earlier basal epithelial state, suggestive of a selective advantage that leads to prevalent recurrence of chr9q CNLOH. Together, we demonstrate that SMART-PTA is an efficient, scalable approach for single-cell whole-genome and whole-transcriptome profiling to build phenotypically annotated single-cell phylogenies with enough throughput and power for application to normal tissue somatic evolution. Moreover, we reconstruct the evolutionary history of the esophageal epithelium at high scale and resolution, providing a window into the dynamics and processes that shape clonal expansions in phenotypically normal tissues throughout a lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a93b5a70ed710c9aacd9cdfba3788cae5ad699" target='_blank'>
              Large-scale single-cell phylogenetic mapping of clonal evolution in the human aging esophagus
              </a>
            </td>
          <td>
            Tamara Prieto, Dennis J. Yuan, John Zinno, Clayton Hughes, Nicholas Midler, Sheng Kao, Jani Huuhtanen, Ramya Raviram, Foteini Fotopoulou, Neil Ruthen, Srinivas Rajagopalan, Joshua S. Schiffman, Andrew R. D’Avino, Sang-Ho Yoon, Jesus Sotelo, Nathaniel D. Omans, Noelle Wheeler, Alejandro Garces, Barun Pradhan, A. P. Cheng, N. Robine, Catherine Potenski, Katharine Godfrey, N. Kakiuchi, Akira Yokoyama, Seishi Ogawa, Julian Abrams, Ivan Raimondi, D. Landau
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cells to transition to an alternative phenotype as a means for adaptation, is an increasingly recognized mechanism of tumor evolution and a driver of resistance to anticancer therapies. The most extensively described clinical settings impacted by such molecular phenomena include neuroendocrine transformation in androgen receptor-dependent prostate adenocarcinoma, and adenocarcinoma-to-neuroendocrine and adenocarcinoma-to-squamous transdifferentiation in epidermal growth factor receptor-driven lung adenocarcinoma, affecting 10%–20% of patients treated with targeted therapy. Recent analyses of human tumor samples and in vivo models of histological transformation have led to insights into the biology of lineage plasticity, including biomarkers predictive of high risk of transformation. However, no clinically available therapies aimed to prevent or revert plasticity are currently available. In the present review, we will provide a biological and therapeutic overview of the current understanding of common and divergent molecular drivers of neuroendocrine and squamous transdifferentiation in tumors from different origins, including descriptive analysis of previously known and recently described molecular events associated with histological transformation, and propose evidence-based alternative models of transdifferentiation. A clear definition of the commonalities and differences of transforming tumors in different organs and to different histological fates will be important to translate molecular findings to the clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell Research</td>
          <td>2</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Shillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Elucidating mechanisms of drug resistance is key for overcoming resistance, guiding drug design, and enabling accurate resistance prediction. Recently, disease metabolic reprogramming has emerged as a novel mechanism of resistance, which enables disease cells to adapt to therapeutic resistance by altering energy production pathways, cellular signaling, and biosynthesis processes. Moreover, protein structure alterations also play a pivotal role in resistance study, facilitating mechanistic understanding, and structure-based target discovery. In other words, integrating these recently accumulated critical data is essential for enriching the landscape of drug resistance data. Therefore, in this study, DRESIS was a significant update by providing (i) 236 molecules that drive metabolic reprogramming and confer resistance to 168 drugs, together with a detailed mechanism, (ii) 2228 protein structural variants implicated in resistance to 671 drugs across 238 diseases, and (iii) greatly expanded landscapes of drug resistance information, now featuring 398 newly added key drug-resistant molecules, 356 drugs with the latest published resistance mechanisms, and 81 new drug-resistant disease categories. All in all, DRESIS 2.0 is expected to serve as a valuable resource for the scientific community and provide important support in tackling the global challenge of drug resistance, which is now publicly accessible at https://idrblab.org/dresis/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafc45a66260eff1536a05a8101c61e45dec038b" target='_blank'>
              DRESIS 2.0: the comprehensive landscape of drug resistance information.
              </a>
            </td>
          <td>
            Xiuna Sun, Zhangle Wei, Xinyuan Yu, Kaixuan Liu, Shun Yao, Zheng Ni, Hanbing Wang, Ziming Zhao, Yinpeng Zhang, Yuxuan Liu, Hanlu Ding, Yintao Zhang, Zhiguo Liu, Mang Xiao, Feng Zhu
          </td>
          <td>2025-11-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Triple-negative breast cancer (TNBC) is a major cause of cancer mortality, with distant metastases presenting a significant clinical challenge. While epigenetic mechanisms like DNA methylation are known to influence TNBC progression, their specific role in driving metastasis remains underexplored. To address this gap, we performed comprehensive multi-omics analysis, integrating epigenomic and transcriptomic profiles from TNBC xenograft models and patient cohorts. Genome-wide DNA methylation profiling of primary tumors, lymph nodes, and lung metastases from xenografts revealed pronounced global hypomethylation in lung lesions, consistent with findings from the clinical cohort. Promoter-methylation changes were enriched in pathways linked to invasion and proliferation, and transcriptomic integration identified 22 epigenetically regulated genes. Among these, elevated AK1 , SLC2A5 , TPI1 , and ZBTB17 expression correlated with a higher risk of lung dissemination. These findings highlight altered DNA methylation as a driver of TNBC lung colonization and identify candidate prognostic markers, emphasizing the importance of epigenetic reprogramming in organ-specific lung metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcba6d1c1c788aab2b1bb08a4a08046f1d4d0352" target='_blank'>
              Epigenetic programs shaping lung metastasis in triple-negative breast cancer
              </a>
            </td>
          <td>
            D. Marzese, Andrés F. Bedoya-López, Javier I. J. Orozco, Miquel Ensenyat-Mendez, Betsy J. Valdez, Sandra Íñiguez-Muñoz, Alexander D. Boiko, Huiwen Xie, Maggie L. DiNome, P. Llinàs-Arias
          </td>
          <td>2025-11-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa96a0707b7c6c78fe5bd4b2a45727093019bee" target='_blank'>
              Targeting the crosstalk of metabolism reprogramming and replication stress: novel strategy to combat cancer
              </a>
            </td>
          <td>
            Wenbo Liu, Xiangyan Jiang, Yong Ma, Huiguo Qing, Yingcun Bao, Zuoyi Jiao
          </td>
          <td>2025-09-26</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Chemoresistance remains a major barrier to improving outcomes in pancreatic ductal adenocarcinoma (PDAC), particularly in response to frontline gemcitabine and nab-paclitaxel (GnP) therapy. We aimed to identify transcriptional drivers of acquired GnP resistance and evaluate their therapeutic potential in PDAC. Although GnP is a standard-of-care regimen, most preclinical models rely on monotherapy-based systems, limiting insight into combination-specific resistance mechanisms. To address this, we developed GnP-resistant (GPR) cell lines by continuously culturing six syngeneic mouse PDAC lines that span the epithelial-to-mesenchymal transition (EMT) spectrum using a clinically relevant 1:8 paclitaxel-to-gemcitabine ratio. Two lines served as EMT controls with EMT-loss via Snail1/Twist1 knockout or EMT-gain via E-cadherin knockout. The remaining four unmodified KPC lines exhibited stable, spontaneous EMT phenotypes. Parental GPR cell lines >10- to 30-fold increase in IC50 as compared to their parental counterparts. Stable resistance was established through serial passaging under continuous exposure to consistent drug concentrations of gemcitabine paclitaxel. To assess whether resistance persisted in vivo, GPR and parental cells were orthotopically implanted into the pancreas of immunocompetent C57BL/6 mice. Upon palpable tumor formation, mice were treated with gemcitabine and nab-paclitaxel twice weekly. GnP significantly extended survival in mice bearing parental tumors (median >60 days), whereas those with GPR-derived tumors showed reduced survival (25–30 days), like vehicle-treated controls, demonstrating that acquired resistance negated GnP efficacy in vivo. To uncover molecular mechanisms of resistance, we performed bulk RNA sequencing on all six matched parental-GPR pairs. Transcriptomic analysis revealed shared changes in gene expression across GPR lines, including activation of proliferative pathways and factors involved in transcriptional regulation and chromatin remodeling. These resistance-associated programs were further validated in GnP-treated tumors by immunohistochemistry and RNA sequencing. Analysis of patient datasets showed that elevated expression of chromatin remodeling factors was associated with poor prognosis in PDAC, underscoring their potential clinical relevance. Future studies will focus on defining chromatin-based mechanisms by which transcriptional reprogramming supports resistance to combination chemotherapy and identifying targetable epigenetic vulnerabilities in treatment-refractory PDAC.



 Meet Patel, Richard Laube, Ayushi Mandloi, Jace Baines, Robert S. Welner, Julienne L. Carstens. Transcriptional and Chromatin Remodeling Programs Underlie Resistance to Gemcitabine and Nab-Paclitaxel in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A062.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35935c1852c231b8da0c50cb3825fb6d98d5f58" target='_blank'>
              Abstract A062: Transcriptional and Chromatin Remodeling Programs Underlie Resistance to Gemcitabine and Nab-Paclitaxel in Pancreatic Cancer
              </a>
            </td>
          <td>
            Meet Patel, Richard Laube, Ayushi Mandloi, Jace Baines, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint blockade (ICB) therapy, which has revolutionized cancer treatment, has been approved for the treatment of triple-negative breast cancer (TNBC). Unfortunately, most patients with TNBC are either not eligible for treatment or exhibit resistance, resulting in limited overall survival benefits. There is an urgent need to elucidate the mechanisms of resistance and enhance therapeutic efficacy. Here, via CRISPR activation (CRISPRa) screening, we identified family with sequence similarity 114 member A1 (FAM114A1) as a key mediator of immune evasion and ICB resistance in TNBC. Mechanistically, FAM114A1 binds p85α to disrupt the p85α/p110α protein complex, thus activating the PI3K/AKT pathway and simultaneously preventing condensate formation of E2F Transcription Factor 4 (E2F4) to promote E2F4-driven Metadherin (MTDH) transcription. Upregulation of these FAM114A1-mediated pathways suppresses tumor antigen presentation and consequently attenuates antitumor immunity in TNBC. Moreover, targeting FAM114A1 improves the therapeutic effectiveness of anti-PD-1 therapy in mouse models, and a FAM114A1-based signature shows strong predictive performance for identifying patients with TNBC who may benefit from ICB. Collectively, our findings not only reveal that FAM114A1 is an immune evasion driver but also highlight it as a promising biomarker and therapeutic target. Our study provides new insights into TNBC immune evasion and outlines a potential avenue to improve the effectiveness of ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6cebf1ccb54b2528f79a6a1b67182cda5a3f55" target='_blank'>
              Family with sequence similarity 114 member A1 orchestrates immune evasion in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wenhao Zhang, Yanzhi Gai, Mengxue Qiao, Michelle Rowicki, Yong Wei, Xiang Hang, Zhengkai Wei, He Yang, Xifu Ye, Hang Ju, Yi Lu, Yibin Kang, Minhong Shen
          </td>
          <td>2025-11-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has shown remarkable success in hematologic malignancies but faces significant challenges in solid tumors due to the immunosuppressive tumor microenvironment (TME). Among these, hypoxia plays a vital role, yet the molecular mediators that link hypoxia to CAR-T dysfunction remain incompletely understood. Anti-mesothelin (MSLN) CAR-T cells were cultured under normoxic (21% O2) and hypoxic (1% O2) conditions for six days. We assessed cell expansion, phenotypes, cytotoxicity, and metabolic features. RNA sequencing was conducted to identify key gene expression changes induced by hypoxia. Findings were further validated using anti-CD70 CAR-T cells. Hypoxia reduced CAR-T proliferation, increased apoptosis, lowered memory phenotypes, raised exhaustion, and weakened cytotoxicity in short-term and long-term assays. Transcriptomic and metabolic analyses showed metabolic reprogramming with increased glycolysis and reduced oxidative phosphorylation. Among the dysregulated genes, the serine/threonine-protein kinase PIM3 emerged as a previously underexplored mediator of hypoxia-driven dysfunction. Genetic or pharmacologic inhibition of PIM3 counteracted hypoxia-induced impairment, enhancing memory phenotypes of CAR-T cells, and improving their anti-tumor activity both in vitro and in vivo. This work identifies PIM3 as a previously underexplored target that links hypoxia to CAR-T cell dysfunction and demonstrates that PIM3 inhibition can reverse these effects. These findings provide a mechanistic rationale for incorporating PIM3 inhibition into CAR-T cell manufacturing or engineering to improve their therapeutic potential in hypoxic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adcb89e4cbae164c14c49d77ccdb838d38a89aa" target='_blank'>
              Metabolic reprogramming through PIM3 inhibition reverses hypoxia-induced CAR-T cell dysfunction in solid tumors
              </a>
            </td>
          <td>
            Muya Zhou, Luxia Xu, Jinhua Hu, Wenwen Chen, Jialu Hong, Mufeng Wang, Zhi-Cheng Guo
          </td>
          <td>2025-11-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) is traditionally discussed in the context of horizontally acquired resistance genes and point mutations at target loci. However, this gene-centred model fails to account for a large number of clinically important modalities of resistance. There is now substantial evidence implicating bacteria in the ability to escape the effects of antibiotics in a variety of non-canonical ways, which are not considered in traditional diagnostic and surveillance pipelines. Among these factors, we can list those arising from global regulatory networks, phase variability, epigenetic tuning, small RNAs, genome structural variability, and phenotypic states like tolerance and persistence. This review will blend the current knowledge on these alternative pathways of resistance and underscore how they intersect with canonical genetic determinants. We will highlight cases where resistance emerges in the absence of known resistance genes, analyse the role of regulatory plasticity in efflux pump expression and membrane remodelling, and examine the contributions of bacterial stress responses and post-transcriptional control. Additionally, we will address methodological gaps in the detection of these mechanisms and their implications for clinical treatment failure, resistance surveillance, and drug development. By integrating insights from molecular microbiology, systems biology, and genomics, this review aims to offer a framework for understanding AMR as a multifaceted, context-dependent phenotype, not merely a genotype. We conclude by identifying knowledge gaps and suggesting priorities for research and diagnostic innovation in this evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cfbc4cd9dd576d2ed3bfaa746258988e167067" target='_blank'>
              Beyond Genes: Non-Canonical Mechanisms Driving Antimicrobial Resistance in Bacteria
              </a>
            </td>
          <td>
            Leonard Koolman, C. Emenike, Debasis Mitra, S. Chattaraj
          </td>
          <td>2025-10-01</td>
          <td>Bacteria</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Glioma, the most common primary brain tumor, shows poor prognosis and resistance to first-line therapies, like temozolomide (TMZ), lomustine (LOM) and panobinostat-marizomib (PM), complicating treatment and diminishing disease-free survival. It is critical to investigate the molecular mechanisms that underlie chemoresistance to develop effective therapies. This study examines common transcriptional networks that facilitate chemoresistance to identify novel therapeutic targets that may overcome resistance.



 The pediatric high-grade glioma KNS42 cell line was chronically treated with increasing doses of TMZ, LOM, or PM to induce drug resistance (RES); control cells received vehicle (DMSO) treatment. Bulk-RNA sequencing was performed followed by differential gene expression analyses (PartekFlow-ANOVA/DESeq2) followed by pathway and comparison analysis (QIAGEN IPA, |FDR|<0.05).



 RNA-sequencing analysis followed by comparison-analysis revealed differential gene expression in treatment-resistant glioma cells as compared to control cells, including 3,839 differentially expressed genes (DEGs) in LOM RES, 3,641 in TMZ RES, and 3,445 in PM RES groups compared to control group. Among these, 374, 417, and 360 DEGs were downregulated in LOM, TMZ, and PM RES cells, respectively, while 265 (LOM and TMZ RES) and 216 (PM RES) DEGs were upregulated (log2FC > 2; p < 0.05). Interestingly, when the top 10 upregulated genes from each resistant group were compared, six genes (VCAN, FOXA1, SOX2, PDGFRA, ERBB4, and COL4A) were found to be commonly upregulated in all 3 resistant groups. We further confirmed that SOX2, PDGFRA, and ERBB4 are upregulated at the protein level. Ongoing studies will test whether these 6 DEGs are required for mediating resistance against diverse drugs.



 RNA-sequencing identified common genes linked to TMZ, LOM, and PM chemoresistance, revealing key genetic networks involved in acquiring and maintaining drug resistance. Targeting these pathways, along with epigenetic silencing and/or combinatorial therapies, may resensitize chemoresistant cells and allow for effective treatment of glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5824551e4b1f9c0b4ac4e374b69dbac7316d823" target='_blank'>
              DDDR-78. Identifying common transcriptional mechanisms in treatment-resistant glioma
              </a>
            </td>
          <td>
            N. B. Rao, Aaron R Johnson, Subrahmanya D. Vallabhapurapu, Tracy A Miller, Ian F Pollack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Squamous cell cancers are responsible for 1 in 5 cancer deaths and survival improvements lag behind those seen in adenocarcinomas. This disparity is in large part due to the limited impact of immunotherapy due to therapeutic resistance, where less than ten percent of patients respond in early stage squamous cell carcinomas of the Head and Neck. Mechanisms that govern intrinsic resistance remain poorly understood and likely arise during the premalignant or dysplastic state. Here, we generated a dataset of murine and human oral squamous epithelia spanning the earliest premalignant stages through invasive carcinoma. Integrative analysis of single-cell and spatial transcriptomics data across the dysplasia to carcinoma continuum reveals early and sustained shifts in epithelial transcriptomes. Spatially informed cell-cell interaction analysis reveals dysplasia-specific upregulation of wound healing and immune remodeling programs in severe dysplasia and invasive carcinoma in a human sample of oral dysplasia. The presence of these transcriptomic programs within early dysplasia may account for the aggressive clinical presentations of squamous cancers including intrinsic therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A. Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA), a probable human carcinogen, induces toxic and mutagenic O6-methylguanine lesions that are repaired by the O6-methylguanine methyltransferase (MGMT). To elucidate mechanisms of NDMA-induced liver cancer progression, we performed longitudinal analyses of phenomic, transcriptomic, and phosphoproteomic changes in wild-type and MGMT-deficient mice, observing amplified responses in the deficient genotype. Early molecular rewiring indicative of a DNA damage response was detected by phosphoproteomic and transcriptomic profiling within days post-exposure. Transcriptomic analyses identified a persistent and robust interferon response as the dominant activated pathway. This chronic interferon signaling, which remained unresolved, correlated with extensive clonal expansion, an early hallmark of oncogenesis. Spatial transcriptomics further revealed pathway alterations favoring tumorigenesis within clonally expanded cells. These findings delineate the cascade of molecular events triggered by acute early-life NDMA exposure, culminating in cancer development months later. Our study unveils potential predictive biomarkers and strategies for disease mitigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily M. Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408e4356836091a48672426b49d0e709f36de3ec" target='_blank'>
              The transcriptional regulatory circuit as a driver and therapeutic target in CML blast crisis
              </a>
            </td>
          <td>
            Liling Jiang, Enzhe Lou, Jinxin Fang, Lizhen Jiang, Guanjie Peng, Jinghong Chen, Bingyuan Liu, Bo Lu, Yi Meng, Haichuan Zhang, Aochu Liu, Qiong Mao, Peilong Lai, Yueyuan Zheng, Jinbao Liu, Xianping Shi
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer. Prostate cancer (PCa) evaluation remains challenging due to its heterogenous and multifocal nature. Here, the authors reconstruct the nuclear chromatin compartmentalization of PCa patients-derived biopsies distinguishing two epigenetic subtypes and deriving an 18-gene signature with prognostic value on TCGA samples that is further validated with multiple independent PCa cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transcriptional regulatory elements (TREs) orchestrate gene expression programs fundamental to cellular identity and transitions between physiological and pathological states. Decoding the regulatory logic of human biology requires resolving where, when, and how these elements are transcriptionally engaged. Here, we profiled the active transcriptional regulatory landscape across all major organ systems and a broad spectrum of developmental and disease states using PRO-cap, a high-resolution method that captures nascent transcription start sites with unprecedented sensitivity and specificity. This atlas of active TREs highlights elements shaped by their cellular contexts and evolutionary constraints, sheds light on the genetic architecture of human traits and diseases, and reveals how patterns of transcription initiation and pausing encode regulatory logic. In cancer, nascent transcription enables the delineation of lineage-specific regulatory states, metastatic adaptations, and the co-option of pre-existing programs. Together, these findings establish nascent transcription as a core dimension of gene regulation, illuminating principles that govern development, physiology, and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa390b89a37848a8f83c759acd4c4c93e92f26e" target='_blank'>
              The regulatory landscape of nascent transcription in human health and disease
              </a>
            </td>
          <td>
            Sagar R. Shah, You Chen, A. K. Leung, Priscila V. Castilho Navarro, Mauricio I. Paramo, Juhi Gupta, Aishwarya Gurumurthy, Rebuma Firdessa Fite, Annika K. Weimer, Qian Du, Ahmed M. Mohyeldin, Dieter Egli, Remi J. Creusot, Russell J H Ryan, Michael P. Snyder, Andrew G. Clark, John T. Lis, Haiyuan Yu
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b535784b00dc4f23ce87d1f10a7f009abae40d" target='_blank'>
              Extensive and differential platinum chemotherapy mutagenesis in children
              </a>
            </td>
          <td>
            Anna Wenger, H. Lee-Six, Manas Dave, M. Layeghifard, A. Lawson, Federico Abascal, Pantelis A. Nicola, T. D. Treger, Toochi Ogbonnah, Conor Parks, Thomas R. W. Oliver, Jonathan Kennedy, A. Hodder, Nathaniel D. Anderson, Felipe Luz Torres Silva, Mi K. Trinh, Thomas Dowe, Marwo Habarwaa, James J. Sun, Sergio Assia-Zamora, M. Cortes-Cerisuelo, W. Jassem, Charlotte Town, Anil Dhawan, Vandana Jain, Karin Straathof, M. Deheragoda, I. Martincorena, Liina Palm, J. C. Hutchinson, Tim H H Coorens, Claire Trayers, Nigel Heaton, A. Shlien, Yoh Zen, Foad J. Rouhani, Sam Behjati
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="


 CDK4/6 inhibitors exhibit therapeutic potential in several tumor entities. Their clinical efficacy in glioblastoma might be limited by intrinsic and acquired resistance mechanisms. Understanding underlying genetic and molecular factors of this resistance might inform the design of effective therapeutic strategies. Genome-wide CRISPR-Cas9 screens offer a powerful approach to systematically identify resistance drivers and synthetic lethal interactions, paving the way for rational combination therapies.



 We conducted genome-wide CRISPR-Cas9 knockout and activation screens in human glioma cell lines LN229 and GS9 under CDK4/6 inhibition using Brunello and Calabrese libraries. Top hits were validated using functional assays. Furthermore, we conducted bulk RNA sequencing of treated and untreated cells for transcriptomic insights into drug response modifiers and molecular pathways involved.



 Functional genomics identified Ambra1 and CCNE1 as key drivers of resistance. Both Ambra1 knockout and CCNE1 overexpression, conferred resistance across both long-term and stem-like glioma models. Additionally, knockout of CHEK1 and FAM122A uncovered synthetic lethal interactions with CDK4/6 inhibition. Functional validation confirmed that glioma cells rely on CHEK1 or FAM122A for cell cycle progression, with cell line specific sensitivity. Transcriptomic analyses consistently highlighted alterations in pathways associated with cell cycle regulation, DNA replication, and the Fanconi anemia pathway. Of note, several differentially expressed genes overlapped with screen hits, supporting the robustness of our findings. Notably, pharmacological inhibition of CHEK1, both in vitro and ex vivo, enhanced sensitivity to CDK4/6 inhibition in a sequence-dependent manner, indicating a critical role of the sequence of drug administration.



 Our functional genomics approach uncovers mediators of acquired resistance and synthetic lethality in the context to CDK4/6 inhibitors in glioma. Co-targeting of CHEK1 or FAM122A with CDK4/6 inhibitors represents a promising therapeutic strategy to overcome resistance and improve the therapeutic efficacy. Ongoing in vivo studies using xenograft and syngeneic glioma models aim to validate the survival benefit of these combinations and to elucidate their impact on the tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaeb6b136b57493744dd9b571df497dca29c55b" target='_blank'>
              OS05.8.A DISSECTING RESISTANCE TO CDK4/6 INHIBITION IN EXPERIMENTAL GLIOMA VIA GENOME-WIDE CRISPR-CAS9 SCREENING
              </a>
            </td>
          <td>
            S. Surender, L. Haeusser, L. Kuhlburger, F. Tsiami, L. Maise, S. Nahnsen, S. Beck, D. Merk, G. Tabatabai
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains difficult to cure despite recent advances. Off-target side effects of drugs currently used lead to significant morbidity and mortality. There is recognition that in AML, there is an increased dependence on OXPHOS metabolism, especially in the leukemia stem cell compartment (AML-LSC). It is also recognized that there is potential to exploit this vulnerability to treat AML. Drug re-purposing screens have suggested the potential use of artesunate (ART) to inhibit mitochondrial respiration. We have explored the potential role of ART as an additive agent in treating AML in combination with conventional therapy. Through in-vitro and in-vivo mouse model studies, we demonstrate the mechanism and efficacy of these combinations and their potential to overcome venetoclax resistance. We further demonstrate the specificity of these combinations with minimal off-target effects on normal hematopoietic stem cells (HSC). These observations warrant exploration of the additive role of ART in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f30bb0a803c2c360d413ebfd59a4fc9ab44ee4" target='_blank'>
              A novel strategy to target metabolic dependencies in acute myeloid leukemia
              </a>
            </td>
          <td>
            Nithya Balasundaram, Hamenth Kumar Palani, Arvind Venkatraman, Yolanda Augustin, Shruthi Pichandi, Clement Regnault, Majeela Solomon, Abirami Rajasekaran, M. Yasar, Swathy Palani Kumar, Reeshma Nair Radhakrishnan, A. Korula, U. Kulkarni, E. Edison, P. Balasubramanian, Biju George, A. Abraham, Sanjeev Krishna, Vikram Mathews
          </td>
          <td>2025-11-04</td>
          <td>Cell Death & Disease</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="The advent of poly (ADP-ribose) polymerase inhibitors (PARPi) over the past decade has significantly altered the management of epithelial ovarian cancer (EOC). We proposed that the etiology of homologous recombination deficiency (HRD) might underlie the variable responses to PARPi observed across patient populations. As part of the phase 2 study of the Chinese HRD Harmonization Project, we developed a genomic methylation sequencing (GM-seq) pipeline facilitated by the TET enzyme for the simultaneous identification of methylated modifications and genetic variations in EOC tumor samples, and compared with established DNA sequencing-based HRD assays. Somatic mutation and HRD scores were confounded by low tumor purity in our cohort of 98 locally advanced/advanced EOC patients. In samples with tumor purity ≥ 30% (n = 45), the GM-seq pipeline showed high consistency with DNA sequencing-based HRD assay, identifying genetic variations in homologous recombination repair (HRR) genes and HRD score with 92.6% (25/27) and 97.1% (33/34) consistency respectively, in addition to conducting methylation profiling. Moreover, different underlying mechanisms of HRD were associated with varying degrees of PARPi efficacy, with BRCA1/2 LOH group having the best efficacy (median PFS, undefined), followed by BRCA1 methylation group (median PFS, 23.4 months), and those with unknown etiology of HRD having the worst efficacy (median PFS, 8.8 months, p < 0.001). Our findings underscore the importance of considering HRD etiology when evaluating PARPi efficacy in EOC patients. The GM-seq pipeline, represents a significant advancement in HRD detection, enabling more accurate predictions of PARPi response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5213220b5e2f3e35a5d8562e34edf897da4714a0" target='_blank'>
              Comprehensive assessment of homologous recombination deficiency via simultaneous methylation and mutation analysis in epithelial ovarian cancer: implications for PARP inhibitors efficacy
              </a>
            </td>
          <td>
            Lin Dong, Huanwen Wu, Ning Li, Wenbin Li, Yan Song, Yu Xiong, Huan Yin, Huan Fang, Rongrong Chen, Xin Yi, Jie Huang, Jianming Ying
          </td>
          <td>2025-10-10</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Non-small cell lung cancer (NSCLC) harboring activating mutations in the Epidermal Growth Factor Receptor (EGFR) has been effectively treated with EGFR tyrosine kinase inhibitors (TKIs). However, the clinical efficacy of these targeted therapies is invariably limited by the development of acquired resistance. While secondary mutations like T790M and bypass pathway activation are well-documented mechanisms, there is a growing appreciation for the profound role of epigenetic regulators, particularly microRNAs (miRNAs), in orchestrating the resistant phenotype. This review provides a comprehensive and detailed analysis of the multifaceted roles of miRNAs in the emergence and maintenance of EGFR-TKI resistance in NSCLC, including their regulation of alternative receptor tyrosine kinase signaling pathways, driving phenotypic plasticity, specifically the epithelial-mesenchymal transition (EMT) and the acquisition of cancer stem cell (CSC) characteristics, as well as dysregulating core cellular processes, such as apoptosis. We further examine the complex interplay within competing endogenous RNA (ceRNA) networks, where long non-coding RNAs and circular RNAs sequester miRNAs, thereby modulating the expression of resistance-associated genes. Finally, the potential of specific miRNAs as circulating biomarkers for monitoring treatment response and as therapeutic targets to overcome resistance is discussed. This review underscores the central role of miRNA-mediated gene regulation as a critical layer of complexity in EGFR-TKI resistance, highlighting a sophisticated network that governs the fate of cancer cells under therapeutic pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec44c69790c0243dc2e3a1211fa70c429a0a3017" target='_blank'>
              The Intricate Web of MicroRNAs in Modulating EGFR-TKI Resistance in Non-Small Cell Lung Cancer: A Comprehensive Review
              </a>
            </td>
          <td>
            Ying Xu, Jie Huang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, D. D. Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c9859a1f25cc1a4fd7861658d95ffc68bbecfd" target='_blank'>
              Molecular mechanisms underlying tamoxifen resistance in breast cancer MCF-7 cell line using transcriptomic and bioinformatics approaches
              </a>
            </td>
          <td>
            M. Alamholo, Alireza Tarinejad
          </td>
          <td>2025-10-01</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genome instability, tumour-promoting inflammation, and immune escape are three distinct hallmarks of cancer. However, accumulating scientific and clinical evidence over the past decade have uncovered a multifaceted interplay of complex dynamic network of interactions between genome instability, the DNA damage response (DDR), and tumour immunogenicity. Fuelled by the clinical successes of immune checkpoint blockers (ICB), growing interest for immuno-oncology and recent cancer biology discoveries have allowed a better understanding of the underlying biology and clinical opportunities brought by this interplay—which is yet, still only in its infancy. The cooperative nature of tumour cell-intrinsic and -extrinsic mechanisms involved suggests that harnessing genomic instability in cancer does not only hamper cancer cells fitness but also stimulate the anti-tumour immune response, thereby paving the way to the development of DDR-based immunomodulatory therapeutic strategies applicable to a variety of molecular and histological cancer types. Here, we review the various aspects of this crosstalk between genome instability and tumour immunogenicity, including feedforward and feedback mechanisms affecting either side of this interplay, as well as the specific consequences of chromosomal instability. We further discuss emerging DDR-based predictive biomarkers of response to ICB therapies, and finally examine the latest clinical developments of therapeutic combinations that exploit the DDR–immunity interplay in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd59247f293c0351f5d4a4afa84e86e0d39c9e9" target='_blank'>
              Genome instability and crosstalk with the immune response
              </a>
            </td>
          <td>
            Roman M. Chabanon, F. Danlos, Kaïssa Ouali, Sophie Postel-Vinay
          </td>
          <td>2025-11-04</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Most mutant adenomatous polyposis coli (APC) gene produced truncated APC protein (Trunc-APC), which has been shown to function as an oncogene in colorectal cancer (CRC) pathogenesis; however, its role in modulating innate immune responses within tumor cells remains unexplored. We utilized CRISPR-Cas9 to knockout mutant APC and performed transcriptome sequencing across multiple CRC cell lines to investigate the immunomodulatory function of Trunc-APC. Subcellular fractionation, proteinase K protection assays, and immunofluorescence were employed to determine Trunc-APC subcellular localization. Protein interaction studies, ubiquitination assays, and aggregation analyses were conducted to elucidate Trunc-APC binding to MAVS and its impact on MAVS ubiquitination and RIG-I association. The effects of Trunc-APC deletion, alone or in combination with 5-azacytidine and trichostatin A, were evaluated on type I interferon activation, apoptosis, and tumor growth both in vitro and in vivo. We found that Trunc-APC partially localizes to the mitochondrial outer membrane and attenuates type I interferon signaling by binding to MAVS, suppressing its K63-linked polyubiquitination, and disrupting MAVS-RIG-I interactions. Deletion of Trunc-APC, particularly when combined with 5-azacytidine and trichostatin A, enhanced innate immune activation, promoted tumor cell apoptosis, and significantly inhibited CRC tumor growth both in vitro and in vivo. Our study reveals a previously unrecognized role of Trunc-APC in dampening tumor-intrinsic innate immunity and suggests that co-targeting Trunc-APC with epigenetic therapy may offer a promising strategy to enhance anti-tumor immune responses in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0e1308090a0906847a31cc6ff0f2170860b1e8" target='_blank'>
              Truncated APC impairs innate immune response by targeting MAVS on mitochondria in colorectal cancer
              </a>
            </td>
          <td>
            Si Li, Xin-Yi Wang, Jing Wang, Jinghua Cao, Yong-rui Lv, Dan Xie, Fengwei Wang
          </td>
          <td>2025-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Melanoma shows heterogeneity across body sites like skin, acral skin, and the uvea, driven by molecular characteristics and genetic variations. However, comparative studies exploring the heterogeneity of melanoma across different anatomical sites remain limited, hindering a comprehensive understanding of its underlying biology. We proposed a research framework through bioinformatics to analyze the tumor ecosystems of cutaneous, acral, and uveal melanoma, from molecular characteristics to genetic variations at single-cell resolution. We found that oxidative phosphorylation (OXPHOS) is a critical driver of tumor cell evolution, with abnormal ribosomal gene and tumor suppressor expression observed in uveal melanoma (UM). Additionally, we screened for potential drug targets and drugs against tumor cells. In the immune microenvironment, acral melanoma (AM) and UM exhibit stronger immunosuppressive characteristics compared to cutaneous melanoma (CM). OXPHOS contributes to T cell cytotoxicity dysregulation in CM and AM, while interferon-γ is crucial in UM. Tumor cells may also induce T cell dysfunction through biological signals such as MIF-CD74 and HLA-E-NKG2A. This study offers valuable insights into melanoma heterogeneity, providing a comprehensive research framework for understanding the distinct molecular and immune characteristics of CM, AM, and UM, and potentially guiding the development of therapeutic strategies tailored to each melanoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da658ce35ad37afa8afa0fc4adb1c52869ddb7f4" target='_blank'>
              Dissecting Melanoma Ecosystem Heterogeneity from Molecular Characteristics to Genetic Variation at Single-Cell Resolution
              </a>
            </td>
          <td>
            Congxue Hu, Liyuan Li, Tengyue Li, Baobin Qi, Wanqi Mi, He Yu, Kaiyue Yang, Qi Ou, Xia Li, Yunpeng Zhang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), the most aggressive and prevalent type of brain cancer, presents a formidable therapeutic challenge due to its high degree of drug resistance. Despite advances in chemotherapy, targeted therapy and immunotherapy, patient outcomes remain dismal, with a 5-year survival rate of only 7%. Resistance mechanisms are multifactorial, including the restrictive nature of the blood-brain barrier (BBB), tumor heterogeneity and adaptive responses within the tumor microenvironment (TME). The BBB limits drug delivery, while efflux transporters further reduce therapeutic efficacy. Additionally, the vast molecular genetic and cellular heterogeneity of a glioblastoma enables the survival of resistant subpopulations, such as glioblastoma stem cells, that evade treatment. Chemotherapy resistance in GBM, particularly to temozolomide, is driven by factors such as O6-methylguanine-DNA methyltransferase upregulation, defective mismatch repair, hypoxia-induced gene expression and activation of several signaling pathways, such as the NF-κB, Hippo and Wnt pathways. Targeted therapies have shown limited success due to activation of compensatory pathways and tumor plasticity, while immunotherapeutic approaches are hindered by an immunosuppressive TME. Recently identified resistance mechanisms, including exosomal transfer of noncoding RNAs and metabolic reprogramming, further complicate treatment. Future directions should emphasize overcoming these challenges through combination therapies, enhanced drug delivery systems and precision medicine approaches. Emerging strategies include targeting persister cells, leveraging metabolic vulnerabilities and integrating AI-driven drug discovery approaches and nanotechnology. Robust patient stratification and biomarker-driven interventions are critical for tailoring therapies and improving outcomes. The present review highlights the urgent need for innovative, multidisciplinary approaches to address the complexity of GBM resistance and advance therapeutic strategies for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7873b8c2ed7832370f4bef19a80cb17ac426667" target='_blank'>
              Drug resistance in glioblastoma: Challenges, mechanisms and therapeutic strategies (Review)
              </a>
            </td>
          <td>
            Yash Durga, Raju Pusapati
          </td>
          <td>2025-11-06</td>
          <td>Molecular and Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Cancer develops through a complex series of genetic and molecular changes. Modern sequencing technologies now allow scientists and physicians to study these changes in great detail, helping them understand why each person’s cancer behaves differently. By examining the entire genome, exome, or transcriptome of a tumor, researchers can uncover genetic alterations that guide personalized treatment decisions. This article explains how these sequencing methods, together with data analysis tools and artificial intelligence, help identify the best targeted and immune-based therapies for different cancer types. By linking genetic findings to approved drugs, the study aims to make precision cancer care more effective and accessible for patients while also guiding future research directions in oncology. Abstract The rapid evolution of sequencing technologies has profoundly advanced precision oncology. Whole-exome sequencing (WES), whole-genome sequencing (WGS), and whole-transcriptome sequencing (RNA-Seq) enable comprehensive characterization of tumor biology by detecting actionable mutations, gene fusions, splice variants, copy number alterations, and pathway dysregulation. These approaches also provide critical insights into biomarkers such as homologous recombination deficiency (HRD), tumor mutational burden (TMB), and microsatellite instability (MSI), which are increasingly essential for guiding therapeutic decisions. Importantly, comprehensive genomic profiling not only refines patient stratification for targeted therapies but also sheds light on tumor–immune interactions and the tumor microenvironment, paving the way for more effective immunotherapeutic combinations. WGS is considered the gold standard for detecting germline mutations and complex structural variants, while WES remains central for detecting somatic driver mutations that guide targeted therapies. RNA-Seq complements these methods by capturing gene expression dynamics, identifying clinically relevant fusions, and revealing mechanisms of resistance. Together with advances in bioinformatics and artificial intelligence, these tools translate molecular data into actionable strategies for patient care. This review integrates insights from WGS, WES, and RNA-Seq with an overview of FDA- and EMA-approved targeted therapies, organized by tumor type, and highlights the molecular signaling pathways that drive cancer development and treatment. By bridging genomic profiling with regulatory-approved therapies, we outline current advances and future perspectives in delivering personalized cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dfbd87c701593d0e68f1b2d6aa4002ef2eff85" target='_blank'>
              Advances in Precision Oncology: From Molecular Profiling to Regulatory-Approved Targeted Therapies
              </a>
            </td>
          <td>
            P. Brlek, V. Škaro, Nenad Hrvatin, Luka Bulić, Ana Petrović, Petar Projić, Martina Smolić, Parth Shah, D. Primorac
          </td>
          <td>2025-10-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Alterations in chromatin-associated proteins contribute to the rewiring of transcriptional networks, promoting lymphomagenesis, disease progression, and resistance to therapies. Thus, epigenetic modulators offer a promising therapeutic avenue. Marginal zone lymphoma (MZL) is characterized by methylation changes, recurrent mutations in genes encoding chromatin remodelers and transcription regulators. Epigenetic reprogramming can sustain resistance to targeted agents. To identify novel therapeutic targets in MZL, we took advantage of the Structural Genomics Consortium (SGC) collection of chemical probes that inhibit or antagonize proteins involved in epigenetic signaling (www.thesgc.org). We exposed MZL cell lines, including models of secondary resistance to BTK, BCL2 and PI3K inhibitors, to 46 epigenetic compounds, spanning molecules targeting bromodomain (BRD), methyltransferases (MT), lysine demethylases (KDM) and histone acetyltransferases (HAT).



 Cells were exposed to the SGC library compounds. Cell viability was assessed after 72h. Compounds giving >40% reduction in cell proliferation versus control were considered active. Selected hits were validated in full dose-response assays.



 Two MZL cell lines (VL51, Karpas1718) and four derivatives with secondary resistance to BTK, PI3K, and BCL2 inhibitors (VL51-Ide, VL51-Copa, VL51-Ibru, Karpas1718-Ibru) were exposed to the SGC library compounds at 100 nM and 5 μM. Resistant derivatives were generally more sensitive to various epigenetic compounds than their parental cells. Only the BET inhibitor (i) JQ1 and the DOT1L-i SGC 0946 showed activity in the parental cells (both 100 nM and 5 μM, both models). Conversely, six compounds were active in resistant models at 100 nM: JQ1 and SGC 0946 (4/4), the PRMT5-i LLY 283 (2/4), and the BRPF2/TAF1-i BAY 299 (1/4). At 5 μM, 13 compounds were active: JQ1 and SGC 0946 in 4/4, 8 compounds in at least 2 models (targeting PRMT5, BRPF2/TAF1, G9a/GLP, JMJD3/UTX, BRD9, p300/CBP, EZH2) and 3 compounds in at least 1 model (PRMT7, p300/CBP, PRMT1/3/4/6/8). We then performed validation MTT experiments for 8 compounds (BAY 299, BI 9564, GSK 2801, I-BRD9, LLY 238, SGC 0946, SGC 3027, UNC 0642) using 6 concentrations of each molecule in all models. Higher activity in resistant derivatives was largely confirmed for the Karpas1718 model, and not in VL51, in which similar activity was seen in parental and resistant cells. The most active compounds were BAY 299 (BRPF2/TAF1; median IC50 6 nM), SGC 3027 (PRMT7; 229 nM) and SGC 0946 (DOT1L, 380 nM), exhibiting dose-dependent cytotoxicity across parental and resistant MZL models, followed by the others, all with median IC50s lower than or equal to 2 μM.



 Our findings reveal novel potential epigenetic vulnerabilities (PRMT7, DOT1L, and BRPF2/TAF1) in lymphoma models, also with acquired resistance to targeted therapies, besides confirming PRMT5 and BET bromodomains as valid targets. Compounds targeting these proteins should be further explored as single agents and in combinations.



 Alberto Jesus. Arribas, Luca Licenziato, Eleonora Cannas, Luca Aresu, Afua Adjeiwaa. Mensah, Francesco Bertoni. Epigenetic vulnerabilities in marginal zone lymphoma [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbf4328ab2c7b5c9c31fe8bd744d96696423fe" target='_blank'>
              Abstract C027: Epigenetic vulnerabilities in marginal zone lymphoma
              </a>
            </td>
          <td>
            A. Arribas, L. Licenziato, Eleonora Cannas, Luca Aresu, A. Mensah, Francesco Bertoni
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) remains the most aggressive primary brain tumour in adults, marked by pronounced cellular heterogeneity, diffuse infiltration, and resistance to conventional treatment. In recent years, transcriptomic profiling has provided valuable insights into the molecular mechanisms that govern the progression of glioblastoma. This systematic review aims to synthesise the current literature on dysregulated gene expression in GBM, focusing on gene signatures associated with stemness, immune modulation, extracellular matrix remodelling, metabolic adaptation, and therapeutic resistance. Methods We conducted a systematic search of PubMed, The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and the GlioVis portal for studies published between January 2005 and April 2025, limited to English-language reports. Studies were eligible if they included adult glioblastoma tissue or patient-derived datasets and reported gene-level expression or clinical associations. Reviews, commentaries, and studies on non-GBM gliomas were excluded. Screening followed the PRISMA 2020 checklist, with 410 records initially identified, 90 duplicates removed, and 125 studies retained after full-text review. Data were synthesised descriptively, and findings were validated against TCGA/CGGA expression datasets to ensure consistency across cohorts. Results We categorised recurrently dysregulated genes by their biological function, including transcription factors (SOX2, ZEB2), growth factor receptors (EGFR, PDGFRA), immune-related markers (PD-L1, TAP1, B2M), extracellular matrix regulators (MMP2, LAMC1, HAS2), and metabolic genes (SLC7A11, PRMT5, NRF2). For each group, we examine the functional consequences of transcriptional alterations and their role in driving key glioblastoma phenotypes, including angiogenesis, immunosuppression, invasiveness, and recurrence. Conclusion We further discuss the prognostic implications of these gene signatures and evaluate their potential utility in precision medicine, including current clinical trials that target molecular pathways identified through transcriptomic data. This review highlights the power of gene expression profiling to stratify glioblastoma subtypes and improve personalised therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f178b970f37a5487e44c1e368727e8a6a513f43" target='_blank'>
              RNA Expression Signatures in Glioblastoma: A Systematic Review of Tumour Biology and Therapeutic Targets
              </a>
            </td>
          <td>
            Amber Hassan, Badr Hafiz, Taghreed A. Alsinani, Rakan Bokhari, D. Mirdad, Awab T. Tayyib, Alaa Alkhotani, Ahmad Fallata, Iman Mirza, Eyad Faizo, Saleh Baeesa, Huda Alghefari, Maher Kurdi
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="This study investigates the expression and functional significance of mitochondrial ribosomal protein S12 (MRPS12) within NSCLC pathogenesis. The Cancer Genome Atlas dataset reveals a significant association between MRPS12 overexpression and unfavorable clinical outcomes in NSCLC patients. Single-cell RNA sequencing data unequivocally confirm MRPS12 overexpression across diverse NSCLC cancer cell populations. MRPS12 is also upregulated in locally-treated NSCLC tissues. shRNA-mediated gene silencing or CRISPR-Cas9-mediated gene knockout (KO) of MRPS12 impaired mitochondrial function, resulting in a reduction in oxygen consumption rate, marked declines of complex I activity and ATP levels, mitochondrial membrane depolarization, and an increase in reactive oxygen species production in NSCLC cells. Moreover, MRPS12 silencing or knockout attenuated cell viability, proliferation, and migratory capacity, while inducing apoptotic cell death in various NSCLC cell types. Conversely, ectopic MRPS12 overexpression conferred a pro-tumorigenic phenotype to NSCLC cells. In vivo studies demonstrated that MRPS12 silencing markedly inhibited the growth of subcutaneous xenografts derived from primary NSCLC cells in nude mice. The MRPS12-silenced NSCLC xenografts exhibited decreased ATP levels, elevated oxidative injury, diminished proliferation, inhibited Akt-mTOR activation and an increase in apoptosis. These findings support the important role of MRPS12 in supporting mitochondrial function and driving the malignant progression of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83f9722c929e54c00c45abc4f6d59820f25b7d72" target='_blank'>
              Exploring mitochondrial ribosomal protein S12 as a novel target for non-small cell lung cancer
              </a>
            </td>
          <td>
            Zihao Chen, Yu Jin, Shiyou Wei, Rongqiang Wei, Xinyu Ding
          </td>
          <td>2025-11-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Leukemia and lymphoma represent a significant burden in Nigeria, characterized by distinct molecular profiles and treatment responses compared to global patterns. This uniqueness can be attributed to several factors including genetic diversity within the Nigerian population, environmental exposures specific to the region, and healthcare access challenges that affect early diagnosis and treatment adherence. The comprehensive understanding of these hematological malignancies has been revolutionized by the advent of single-cell technologies, which have uncovered unprecedented resolution into tumor heterogeneity, evolution, and microenvironmental interactions. Unlike traditional bulk sequencing methods that average signals across thousands of cells, single-cell approaches allow researchers to examine individual cells, revealing rare but critical subpopulations that drive disease progression and therapy resistance.
This review synthesizes findings from Nigerian studies integrated with global advances in single-cell research, highlighting how clonal architecture, cellular ecosystems, and molecular signatures contribute to pathogenesis and treatment outcomes. We explore how single-cell multi-omics approaches—including transcriptomics, genomics, and proteomics—have identified novel therapeutic targets and resistance mechanisms in leukemia and lymphoma. Additionally, we discuss the potential for implementing these technologies in Nigeria to advance precision medicine, considering both the opportunities and challenges specific to the Nigerian healthcare context. The integration of single-cell analyses into clinical practice promises to improve risk stratification, therapy selection, and overall management of these malignancies in resource-limited settings through more targeted and effective treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca65a47e4dd6298d29b7f5a8f9777fdb3fa1474c" target='_blank'>
              Unraveling Clonal Architecture and Microenvironmental Complexity of Leukemia and Lymphoma Using Singlecell Multi-Omics in Nigeria
              </a>
            </td>
          <td>
            Leon Oluci Kochukgu
          </td>
          <td>2025-11-06</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, Céline Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Cancer cells exhibit reprogrammed metabolic pathways to sustain aggressive phenotypes, including continuous cell division, stemness, invasion, and metastasis. Emerging evidence suggests that these metabolic adaptations profoundly impact DNA repair pathways, which contribute to the responses to therapy and influence overall outcomes. Metabolic processes such as the Warburg effect, nicotinamide adenine dinucleotide (NAD) metabolism, glutamine metabolism, and one-carbon metabolism support DNA repair by expanding the metabolite pool and facilitating post-translational modifications. Conversely, oncometabolites impair DNA repair pathways through epigenetic reprogramming, thereby promoting genomic instability. This review highlights recent discoveries that elucidate the intricate connections between metabolic hallmarks in cancer cells and DNA repair mechanisms, offering insights into potential therapeutic targets for future cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0683240e558e0cfd022d73899ecd3cc85f65ee47" target='_blank'>
              Cancer Metabolism Meets DNA Repair: The Hidden Link to Therapy Resistance.
              </a>
            </td>
          <td>
            Chunzhang Yang
          </td>
          <td>2025-09-28</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is extensively studied, yet its origins and progression remain unclear. A key question is why tumors of the same type vary in aggressiveness and treatment response. In colorectal cancer (CRC), the third most common cancer, this variability led to the identification of molecular subtypes (CMS). However, the tumor microenvironment remains poorly understood and may be crucial for understanding carcinogenesis and drug resistance. A promising approach is analyzing cell interactions through ligand-receptor expression. This study used bioinformatics to examine CRC in different anatomical locations, identify diagnostic and prognostic biomarkers, and propose targeted drugs. Expression data were obtained from the TCGA-COAD database. All samples were filtered based on the tumor’s region of origin and purity. RNA-seq expression analysis was then conducted to assess molecular differences according to tumor location and purity, identifying region-specific ligands and receptors using the secretome list as a reference. Once these differences were identified, an interactome was constructed to depict cell interactions within the tumor microenvironment. The most relevant genes were then evaluated for their prognostic potential through survival analysis, and their susceptibility to pharmacological modulation was assessed to identify potential new drug candidates for CRC treatment. The integration of secretome data and the construction of the interactome proved to be a valuable approach for detecting novel biomarkers specific to right- and left-sided CRC. Through this approach, FGFR4, FLT1, and WNT5A were identified as key biomarkers involved in tumor carcinogenesis, modulating distinct processes in each region, such as fibroblast recruitment and cell division. Based on these biomarkers, Dovitinib and Nintedanib were predicted as potential therapeutic agents, as they target multiple identified markers. This study highlights FGFR4, FLT1, and WNT5A as key diagnostic and therapeutic biomarkers for CRC, with their relevance varying based on the tumor’s site of origin. Leveraging these findings, we propose Dovitinib and Nintedanib as promising targeted therapies for CRC. These insights can enhance current treatment strategies and pave the way for future in vivo and in vitro studies, driving progress in CRC research and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86a3d9f8b9f730ccd2b60a1bbaaeddf8c7f0fad" target='_blank'>
              Integrating the secretome and interactome to identify novel biomarkers and therapeutic targets in colorectal cancer
              </a>
            </td>
          <td>
            Gabriel Henrique Caxali, Mirian Carolini Esgoti Aal, Catherine Wesselka Garcia Osvaldo, Jakeline Santos Oliveira, L. Bidinotto, Robson Francisco Carvalho, F. K. Delella
          </td>
          <td>2025-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction Colon cancer is a common disease, treated with few chemotherapeutic agents with similar treatment sequencing despite its heterogeneity. A significant proportion of patients are diagnosed with metastasis, and resistance to antineoplastic drugs is associated with disease progression and therapeutic failure. It is known that the tumor microenvironment plays an essential role in cancer progression, contributing to processes that may be associated with therapeutic resistance mechanisms in colon cancer. In this study, we aim to identify a gene expression signature and its relationship with immune cell infiltration in colon cancer, contributing to the identification of potential resistance biomarkers. Methods An in silico study was conducted using RNA-seq data from The Cancer Genome Atlas Program (TCGA) samples, subdivided into two groups (treatment-resistant and non-resistant), taking into account the molecular subgroups (CMS1, CMS2, CMS3, and CMS4). The following algorithms were used: i. Limma was applied to identify differentially expressed genes; ii. WGCNA was applied to construct co-expression networks; iii. CIBERSORT was applied to estimate the proportion of infiltrating immune cells; and iv. TIMER was applied to explore the relationship between core genes and immune cell content. Results Twenty differentially expressed genes (DEGs) were found, with 18 related to the group considered resistant to oncologic treatment and presenting poorer overall survival. T CD4 memory resting cells and M0 and M2 macrophages were found in more significant proportions in the analyzed samples and more infiltrated in the tumor microenvironment, the higher the expression of some of these resistance DEGs. Additionally, these genes correlate with biological aspects of neuronal differentiation, axogenesis, and synaptic transmission. Conclusion The gene expression signature suggests the presence of differentially expressed synaptic membrane genes, which may be involved in neuronal pathways that influence the tumor microenvironment, potentially serving as future biomarkers. Furthermore, the presence of M0 and M2 macrophages and T CD4 memory resting cells suggests a potential interaction that may play a role in therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d04147fc5526a1e2f859a73c8cdc7c99f7a9f" target='_blank'>
              Gene expression profile in colon cancer therapeutic resistance and its relationship with the tumor microenvironment
              </a>
            </td>
          <td>
            P. Doria, G. Rocha, Vanessa Dybal Bertoni, Roberto de Souza Batista dos Santos, Mariana Araújo-Pereira, Clarissa A Gurgel
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancers are molecularly heterogeneous, with subtype-specific differences in transcriptional programs, chromatin architecture, and therapeutic responses. While PARP1 has been extensively studied in the context of DNA repair, emerging evidence implicates its catalytic activity in a broader set of cellular processes, including the regulation of gene expression. Here, we employed an NAD analog-sensitive PARP1 (asPARP1) chemical genetics approach combined with mass spectrometry to map the ADP-ribosylated proteome across six human breast cancer cell lines representing luminal and basal/triple negative subtypes. We identified thousands of PARP1 substrates and hundreds of Glu/Asp ADPRylation sites, revealing both shared and subtype-specific modifications in cell lines maintained under basal growth conditions. Luminal-specific substrates were enriched in chromatin and transcriptional regulators, whereas basal-specific substrates were preferentially linked to translation and RNA processing, highlighting lineage-dependent PARP1 activity. Transcription factors emerged as major substrates, with TFAP2A serving as a proof-of-concept; it is selectively ADPRylated in luminal cells and inhibition of PARP1-mediated ADPRylation modulates its promoter occupancy in a subtype-specific manner. Our data provide a new resource for studying subtype-specific PARP1-mediated ADPRylation in breast cancer cells. Collectively, our findings expand the conceptual framework for PARP1 function beyond DNA repair, offering mechanistic insights into subtype-specific gene regulation and potential determinants of PARP inhibitor sensitivity in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4df105f2ecd1754e819d2661c1298b7b3761196" target='_blank'>
              Mapping the Subtype-Specific PARP1 ADP-ribosylated Proteome in Breast Cancer Cells
              </a>
            </td>
          <td>
            Sneh Koul, Minjung Kwon, Poulami Tapadar, Yangyang Dai, T. Nandu, Dan Huang, C. Camacho, W. Kraus
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) activates pro-inflammatory cGAS/STING signaling, positioning it as a biomarker for combining immune checkpoint blockade (ICB) and PARP inhibition (PARPi). However, the consequences of HRD on the immune landscape across cancers remain unclear. Here, we applied a pan-cancer HRD classifier to >10,000 tumors from The Cancer Genome Atlas and uncovered striking heterogeneity in immune activity. Compared to HR-proficient tumors, HRD tumors showed elevated inflammation in breast, ovarian, and endometrial cancers. These tumors exhibited robust activation of innate and adaptive immune pathways (IFN, NF-κB) and transcriptional hallmarks of senescence, angiogenesis, and adenosine signaling. In contrast, lung, head and neck, and melanoma HRD tumors displayed suppressed inflammation and evidence of immune escape through large-scale loss-of-heterozygosity (LOH) at IFNA/B, STING, and other loci. These tumors also frequently presented HLA LOH and oncogene amplifications, suggesting selection under immune pressure and replication stress. Together, our study resolves HRD tumors into two immune archetypes, immune-inflamed and immune-evasive, linked to chromosomal instability and lineage, informing biomarker-driven evaluation of immune checkpoint blockade/PARPi combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b68316c5391afa0d45aa42e226e37099bf84d45" target='_blank'>
              Tissue specificity and chromosomal alterations shape divergent immune programs in HRD tumors
              </a>
            </td>
          <td>
            D. Gulhan, David Barras, Marco Mina, Eleonora Ghisoni, Yoo-Na Kim, V. Viswanadham, Hu Jin, Florian Huber, K. Homicsko, M. Bassani-Sternberg, G. Ciriello, Peter J. Park, Denarda Dangaj Laniti
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Next-generation sequencing of canine spontaneous cancer is a powerful approach in both comparative oncology and veterinary genomics. We encountered a unique case with concurrent mammary tumors. Using whole-genome sequencing (WGS), we profiled the tumor-specific landscape of somatic mutations across multiple tumor subtypes, providing unprecedented evidence within an identical genetic background. Of the seven mammary gland tumors (MGTs) isolated, two were diagnosed as benign and five as malignant. High-quality WGS (34.5X average sequencing depth, ≥ 20X coverage across 94.9% of the genome) on tumors and a blood sample revealed missense mutations in human breast cancer-related genes such as BRCA2 and TP53. Furthermore, we found evidence of canine-specific somatic mutations depending on the tumor subtypes, including HECTD4 in malignant tumors and NIPBL in epithelial-derived malignant tumors. This unique case study provides novel insights into the genomic heterogeneity, clonal evolution, and subtype-specific pathogenesis of naturally occurring canine MGTs. Despite some inherent limitations and potential for individual-specific variation, our results emphasize and guide future large-scale, longitudinal studies to further elucidate the clinical and biological significance of key somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa805022d59e7e8ebaa159b214972178e6ca2e8d" target='_blank'>
              Comparative genomic analysis of multiple mammary tumors from a single dog: whole-genome sequencing study
              </a>
            </td>
          <td>
            Keon Kim, Tae-Hoon Shin, Sin-Wook Park, Sang-Ik Park, Yoon-Jung Do, Woong-Bin Ro, Chang-Min Lee
          </td>
          <td>2025-10-30</td>
          <td>Irish Veterinary Journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Thyroid carcinomas are phenotypically heterogeneous malignancies. Advances in molecular and cellular technologies have revealed genetic, epigenetic, and nongenetic factors underlying this heterogeneity. Our study aimed to assess the impact of single and combined treatments with anticancer agents (Carboplatin, Doxorubicin, Paclitaxel, Avastin), natural compounds (Quercetin), and epigenetic modulators (suberoylanilide hydroxamic acid and 5-Azacytidine) on the expression of long noncoding RNAs, methylation regulators, and functional features in the human thyroid cancer cell line K1. Methods: Treated and untreated K1 cells were used throughout experiments to evaluate the drug-induced cytotoxicity, apoptosis, cell cycle distribution, cytokine release, gene expression, and global DNA methylation levels. Results: Some single- and combined-drug treatments modulated both cell cycle progression and apoptotic events, demonstrating anti-tumor activity of the tested compounds. Gene expression analysis showed treatment-specific regulation of target genes and lncRNAs, including both upregulation and downregulation across different drug combinations. All treatments resulted in increased global DNA methylation levels compared to the untreated controls. Several combinations significantly upregulated DNMT1 and DNMT3B, while concomitantly decreased EZH2 levels. Conclusions: These coordinated epigenetic changes highlight the therapeutic potential of combining epigenetic modulators with chemotherapeutic agents, suggesting a strategy to prevent or reverse treatment resistance and improve outcomes in thyroid cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1eba8f910f8930ce4647663ca6adba36af18" target='_blank'>
              Epigenetic Remodeling in Thyroid Cancer: New Dimensions of Targeted Therapy Through lncRNA Modulation
              </a>
            </td>
          <td>
            A. Albulescu, A. Fudulu, M. Mihăilă, I. Iancu, Adriana Pleșa, M. Bostan, A. Botezatu, L. Brasoveanu, C. Hotnog
          </td>
          <td>2025-10-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d6f72ebd8a3733c60e8ee75cfbcd9b5da5d9c6" target='_blank'>
              Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes
              </a>
            </td>
          <td>
            Zhe Liu, Yihang Bao, An Gu, Weichen Song, Guan Ning Lin
          </td>
          <td>2025-09-26</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="L1 comprises 17% of the human genome, with 50-150 full-length sequences capable of retrotransposition. Although largely inactive in normal somatic tissues, tumorigenesis leads to L1 derepression and overexpression of its RNA binding chaperone protein, ORF1p. A potential cancer biomarker, ORF1p expression is a hallmark of multiple cancers and an early event in precursor lesions. Our study provides a comprehensive pan-cancer analysis of ORF1p using CPTAC proteogenomic data, revealing a dichotomous role in modulating immune responses. Integrated analyses and supervised learning models divide ORF1p-high tumors into two groups: (1) high ORF1p associates with immunosuppression, reduced interferon signaling and diminished immune cell infiltration (HNSCC and LSCC), and (2) ORF1p-high tumors associate with immune activation (UCEC). Linear regression models reveal that cancer-specific aneuploidies may underlie this immune dichotomy, highlighting the prognostic significance of ORF1p and informing new strategies to leverage detection of plasma-circulating ORF1p to enhance immunotherapeutic efficacy in different tumor contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3642f23461379e5a0adfe4119834fff5e5e8de7" target='_blank'>
              Pan-cancer analysis reveals context-dependent roles of LINE-1 ORF1p in immune regulation and copy number alterations
              </a>
            </td>
          <td>
            E. Kaparos, Wenjing Zhang, Ryusei Miyanaga, Wilson H. McKerrow, J. Boeke, Teresa Davoli, P. Mita, Kelly Ruggles, David Fenyö
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="


 Glioblastoma remains one of the most treatment-refractory solid tumors due to rapid adaptive resistance across multiple convergent molecular pathways. In particular, persistent activation of the PI3K-AKT axis combined with mesenchymal transition (EMT) reprogramming contributes to rapid failure of standard therapeutic regimens, including temozolomide, radiation, and immunotherapeutics. To address this clinical gap, we developed a synthetic RNA-guided dual-targeting system designed to simultaneously suppress both canonical PI3K-AKT activation and mesenchymal escape signatures, using strand-engineered molecules optimized for combinatorial silencing. Using patient-derived glioblastoma stem cell lines and isogenic resistance models, we performed transcriptomic profiling to identify co-expressed resistance-driving nodes. Computational integration of CRISPR perturbation datasets, RNA interference screens, and synthetic lethality maps enabled selection of targetable PI3K-AKT intermediates and EMT regulators including SOX2, TWIST1, and ZEB1. Synthetic RNA molecules were designed to co-suppress both vertical (PI3K-AKT) and horizontal (EMT plasticity) resistance axes, with in vitro validation demonstrating >70% knockdown efficiency and significant synergistic reduction in proliferation (p<0.001) across diverse resistance phenotypes. Mechanistic assays revealed reversal of mesenchymal signatures with restored epithelial markers (E-cadherin upregulation; vimentin downregulation) alongside AKT dephosphorylation. Combination treatment with low-dose temozolomide further amplified cell death, suggesting sensitization to standard chemotherapy. In vivo murine xenograft models exhibited >60% reduction in tumor volume compared to controls (p<0.01), without evidence of off-target cytotoxicity or systemic toxicity. This study demonstrates the feasibility of multi-axis synthetic RNA-guided combinatorial targeting to suppress glioblastoma therapeutic resistance. These findings support future clinical translation of modular RNA-based therapeutics capable of overcoming the heterogeneity and plasticity that characterize high-grade gliomas.



 Shivi Kumar. Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746db514f096cab5d3002088013773d14b8c8b84" target='_blank'>
              Abstract A036: Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways
              </a>
            </td>
          <td>
            Shivi Kumar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ee3d709c3a53cc71a54980118c28a4596ab59b" target='_blank'>
              HyperdCas12a-Based Multiplexed Genetic Regulation in Candida albicans
              </a>
            </td>
          <td>
            Nicholas C. Gervais, R. K. Rogers, Madeleine R. Robin, Rebecca S. Shapiro
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 DHX9, a multi-functional RNA helicase, plays important roles in maintaining genome stability by unwinding various nucleic acid secondary structures that may lead to replication stress and DNA damage if left unresolved. DHX9 is overexpressed in multiple cancers, with selective vulnerability to DHX9 inhibition noted in tumors characterized by genomic instability and high replicative stress. ATX-559 is an orally bioavailable, small-molecule inhibitor of DHX9 helicase activity currently in Phase 1 clinical development. ATX-559 induces robust anti-tumor activity in a variety of preclinical models with genomic instability, including those with microsatellite instability-high (MSI-H) and/or deficient mismatch repair (dMMR), BRCA alterations, and homologous recombination repair deficiency (HRD).



 ATX-559 anti-proliferative activity was tested in a multi-indication cell panel screen. Triple negative breast cancer (TNBC) and colorectal cancer (CRC) cells were used to evaluate ATX-559 mechanistic activity using immunofluorescence imaging, dot blot, and western blot assays. ATX-559 was dosed orally BID to evaluate its anti-tumor activity in CRC and TNBC cell line-derived xenograft (CDX) models, as well as a panel of 64 patient-derived xenograft (PDX) models including TNBC, Ovarian, CRC, endometrial, and gastric cancers. qPCR was used to evaluate the target engagement PD biomarker, circBRIP1.



 ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress, subsequent DNA damage, and cell death selectively in cancer models with genomic instability and replicative stress. ATX-559 sensitivity was enriched in dMMR/MSI-H cell lines including CRC. Sensitivity was also enriched in BRCA altered TNBC and HRD ovarian cancer cell lines. ATX-559 is well tolerated in vivo, resulting in robust dose-dependent tumor growth inhibition and regression selectively observed in CRC dMMR/MSI-H and BRCA altered TNBC CDX models, along with a well correlated PK/PD/efficacy relationship. Additionally, ATX-559 demonstrated enriched sensitivity in a large panel of BRCA altered TNBC (75%), dMMR/MSI-H (68%), and HRD ovarian (55%) PDX models.



 Together, these findings represent preclinical validation of DHX9 as a selective therapeutic in cancers with genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1/2, open-label, dose-escalation and expansion study to evaluate the safety profile of ATX-559 and to determine the recommended phase 2 dose (RP2D) and preliminary antitumor activity in patients with locally advanced or metastatic solid tumors, and molecularly defined cancers (NCT06625515).



 Sunaina Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver. ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2a8e9dfeed1b9e094182590e0697b62580f888" target='_blank'>
              Abstract B045: ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            S. Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen M. Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Mitochondria are essential organelles involved in energy production, cellular metabolism, and signal transduction. They have important impacts on tumorigenesis and cancer progression. Nevertheless, the associations between mitochondrial metabolic processes and chemotherapy resistance in colorectal cancer (CRC) are not well understood.



 We generated a chemotherapy-resistant colorectal cancer cell line, HCT-15/DOX, via doxorubicin (DOX) induction. We then performed proteomic and metabolomic analyses via LC-MS/MS technology on both the parental and the DOX-resistant cell lines. Additionally, transmission electron microscopy was used to examine changes in mitochondrial morphology between the two cell lines.



 The results revealed significant dysregulation of 185 proteins and 1099 metabolites in HCT-15/DOX cells relative to parental cells, highlighting the impact of chemotherapy resistance on cellular processes. The key functional proteins that were identified included upregulated SDHA, BCKDHB, CRYZ, NUDT6, CPT1A, and POLG, and downregulated CRAT, FDPS, SFXN1, and ATAD3B. Additionally, through combined multiomics pathway enrichment analysis, pyrimidine metabolism, purine metabolism, ascorbate and aldarate metabolism, propanoate metabolism, and the citrate cycle (TCA cycle) were identified as important metabolic processes associated with CRC chemotherapy resistance. Transmission electron microscopy analysis revealed that HCT-15/DOX cells had increased mitochondrial number, length, and area.



 This research highlights notable differences in mitochondrial morphology and diverse mitochondrial metabolic functions between parental and DOX-resistant HCT-15 CRC cells. The findings of the present study provide insights into the mitochondrial metabolic changes associated with CRC chemotherapy resistance, offering valuable insights into the mechanisms underlying these changes and identifying potential therapeutic targets for addressing CRC chemotherapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47edb9f4c564452e7f562c5c1875b89a970a0bb4" target='_blank'>
              Multiomics analyses of human colorectal cancer reveal changes in mitochondrial metabolism associated with chemotherapy resistance
              </a>
            </td>
          <td>
            Shiyi Chen, Qian Li, Wei Zheng
          </td>
          <td>2025-11-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite the initial responses to the tyrosine kinase inhibitor (TKI) for cancer therapy, many patients often relapse with no curative regimens available. Further, the ability to target therapeutic agents to cancer cells with appropriate doses remains challenging in the clinic, especially for leukemia. Here, we show that naïve CML cells are dynamically heterogeneous in colony formation. Larger clones expand while smaller ones diminish and eventually disappear. Compared to resistant cells, parental populations, including CD44+ stem cells, form a greater number of larger, solid spheroids. Upregulation of fat mass and obesity associated protein (FTO), an RNA N6-methyladenosine demethylase, and stem cell markers (e.g., CD44, CD133, CD25) is more obvious in resistant cells compared to parental cells. FTO inhibitors (e.g., CS1, FB23-2) appreciably impair the growth of resistant cells either alone or in combination with nilotinib. FTO protein expression is unexpectedly upregulated by CS1 or FB23-2 treatment in multiple leukemia cell lines. We then constructed RNA nanoparticles encapsulating FTO siRNAs and conjugated with anti-CD133 RNA aptamers. We showed that, compared to negative control, these nanoparticles were taken up much more efficiently by resistant cells that highly express CD133. Treatment with the CD133-guided FTO siRNA nanoparticles efficiently silenced FTO expression in resistant cells, which leads to a significant reduction in their colony and spheroid formation. These findings offer new insights into cancer drug resistance and advance the application of RNA nanotechnology for treating leukemia. The research provides a foundation for developing novel, targeted therapies for resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610e20fd12fcf52179a62018bf638a0c7e862c87" target='_blank'>
              CD133-Guided RNA Nanoparticle Delivery of FTO siRNA Impairs Leukemia Resistance to Tyrosine Kinase Inhibitor Therapy
              </a>
            </td>
          <td>
            Huiqin Bian, Changli Zhou, Hiroaki Koyama, Wen Gao, Sicheng Bian, Tao Cheng, Xiaonan Han, William Tse, Shujun Liu, Alexander Miron
          </td>
          <td>2025-10-15</td>
          <td>RNA NanoMed</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Disclosure: S. Gadad: None. M. Sedano: None. Triple-negative breast cancer (TNBC) accounts for approximately 12-20% of all new breast cancer cases diagnosed annually and is known to be associated with poorer clinical outcomes compared to other molecular subtypes of breast cancer. Patients with TNBC typically present with larger tumors and higher tumor grades, resulting in decreased overall and recurrence-free survival rates compared to individuals with non-TNBC tumors. The strategies involved in treating TNBC include chemotherapy agents such as doxorubicin, an anthracycline chemotherapeutic agent; however, its efficacy may vary due to the development of resistance. The specific mechanisms behind the constantly evolving resistance of TNBC to chemotherapy remain unclear, emphasizing the need for further investigation. In our study, we utilized two TNBC cell lines engineered with gradually increasing concentrations of doxorubicin to develop resistance. We conducted integrative genomic analyses to uncover the cellular mechanisms involved, identified altered molecular pathways relevant to cancer, and ascertained the factors contributing to the phenotypic characteristics of doxorubicin-resistant TNBC. Our findings indicate that many evolutionarily young transcripts with coding or limited coding potential are differentially expressed compared to non-resistant TNBC cells. Furthermore, analyses of adjacent genomic regions containing aberrantly expressed genes revealed that these genes are located in areas implicated in cellular plasticity, allowing cancer cells to adapt to the presence of doxorubicin, potentially through transcriptional reprogramming. We are also investigating the genetic and epigenetic alterations that may have facilitated the evolution of TNBC cells into doxorubicin-resistant variants. Additionally, we are assessing the clinical implications of the identified factors by studying them in vivo using mouse models with genetically engineered chemo-sensitive and -resistant cells, linking the outcomes to changes observed in patient samples associated with clinical prognosis. By elucidating the specific mechanisms contributing to the development of doxorubicin resistance in TNBC, we aim to uncover the underlying processes driving resistance and recurrence, potentially paving the way for novel therapeutic strategies. S.S.G. is a CPRIT Scholar in Cancer Research. S.S.G. was supported by a First-time faculty recruitment award from the Cancer Prevention and Research Institute of Texas (CPRIT; RR170020). S.S.G. is supported by the The American Cancer Society (RSG-22-170-01-RMC), NIH 1R16GM149497 grants and CPRIT-TREC (RP230420). Presentation: Sunday, July 13, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b4373af8394b05d1e7e23058cecf5a8e12738f" target='_blank'>
              SUN-210 The role of transcriptional reprogramming and genomic alterations in the transition from chemo-sensitive to chemoresistant triple-negative breast cancer
              </a>
            </td>
          <td>
            Shrikanth S. Gadad, Melina J. Sedano
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Sarcomas are a heterogeneous group of mesenchymal malignancies with poor prognosis and limited response to standard therapies, including immune checkpoint inhibitors (ICIs). Tumour‐intrinsic factors—such as telomere maintenance mechanisms (TMMs) and metabolic reprogramming—play central roles in driving immune evasion and therapeutic resistance. Telomerase activation and alternative lengthening of telomeres sustain replicative immortality while influencing the tumour immune microenvironment. In parallel, metabolic adaptations, including glutamine dependency and arginine auxotrophy, further suppress antitumour immunity. Together, TMMs and metabolism form an integrated axis that shapes immune modulation and treatment outcomes. Recent advances—ranging from telomerase‐based vaccines and TMM‐targeted immunotherapies to metabolic modulators combined with ICIs—demonstrate the translational promise of targeting this axis. This review synthesises current knowledge on telomere‒metabolism crosstalk in sarcomas, highlights its impact on immunotherapy response, and outlines future directions for biomarker‐driven, combinatorial strategies to overcome resistance and improve patient outcomes. Key points Telomere maintenance mechanisms (telomerase reverse transcriptase and alternative lengthening of telomeres) reprogram metabolism and dampen innate immune sensing in sarcomas. Metabolic rewiring (glutamine addiction, glycolysis and fatty acid oxidation) fosters T‐cell dysfunction and myeloid‐derived suppressor cell accumulation. Targeting the telomere‒metabolism‒immunity axis offers strategies to overcome immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6055e4c1c995f49293e41de8f62c97d2eae45e89" target='_blank'>
              Telomere–metabolism–immunity axis in sarcoma: Immune evasion mechanisms and therapeutic strategies
              </a>
            </td>
          <td>
            Ji-Yong Sung, June Hyuk Kim
          </td>
          <td>2025-10-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="This article provides a comprehensive overview of targeted and epigenetic therapies for acute myeloid leukemia (AML), highlighting their role in advancing treatment strategies. Recognizing that AML arises from complex genetic and epigenetic alterations, a deeper understanding of its pathogenesis has paved the way for precision medicine. The article reviews key targeted therapies, including IDH inhibitors (e.g., ivosidenib, enasidenib) for IDH mutations, FLT3 inhibitors (e.g., midostaurin, gilteritinib) for FLT3 alterations, BCL2 inhibitors (e.g., venetoclax) impacting apoptosis, and emerging Menin inhibitors for specific subtypes like KMT2A-rearranged or NPM1-mutant AML. It also discusses epigenetic therapies that target reversible changes, such as DNMT inhibitors (e.g., azacitidine, decitabine) for DNA methylation and HDAC inhibitors for histone modifications, along with other promising agents under development targeting epigenetic regulators. Furthermore, the article explores gene therapy as a significant future direction, detailing advancements in gene editing (CRISPR/Cas9) and strategies designed to shield healthy hematopoietic cells from targeted toxicity (e.g., CD33, CD123 editing) or to enhance the anti-leukemic function of immune cells (e.g., CAR-T, TCR therapy). The future perspective emphasizes the importance of developing synergistic combination therapies and leveraging molecular insights for truly personalized treatment approaches to overcome therapeutic challenges and improve long-term outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f1a005625d793e87f3fac27421531cd23f4756" target='_blank'>
              Targeted and epigenetic therapies for acute myeloid leukemia treatment
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Mobin Ghaderi, Milad Mokarami, Homaira Arian, Sara Mohammadnabi, Arastoo Farokhi
          </td>
          <td>2025-09-26</td>
          <td>Discover Oncology</td>
          <td>2</td>
          <td>3</td>
        </tr>

        <tr id="Skin cutaneous melanoma (SKCM) is a highly aggressive malignancy that arises from melanocytes and has been associated with a notable increase in incidence and mortality rates across the globe. The current treatment modalities for SKCM, which encompass surgical interventions, radiotherapy, chemotherapy, and immunotherapy, have notably enhanced survival outcomes for certain patient subsets. Nonetheless, these approaches are frequently limited by factors such as variable efficacy, adverse side effects, and the emergence of resistance mechanisms. In this context, there is a pressing need to explore novel therapeutic targets that may enhance treatment effectiveness. This study examines the Dedicator of cytokinesis 2 (DOCK2) gene, which has been implicated in immune regulation, yet its functional roles within oncogenesis, particularly in melanoma, remain inadequately characterized. We employed an integrated bioinformatics approach utilizing clinical datasets from The Cancer Genome Atlas (TCGA), the Genotype-Tissue Expression project (GTEx), and the Human Protein Atlas (HPA) to evaluate DOCK2 expression across various malignancies and to assess its prognostic significance in SKCM. Our analyses revealed that DOCK2 expression is differentially modulated across several cancer types and serves as a favorable prognostic marker specifically in SKCM. Notably, elevated DOCK2 levels were significantly correlated with improved therapeutic responses to immune checkpoint inhibitors (ICIs), suggesting that DOCK2 may play a pivotal role in modulating the tumor immune microenvironment. Furthermore, functional enrichment analyses indicated that DOCK2-associated genes are critically involved in vital immune pathways, encompassing leukocyte-mediated immunity and immune receptor activity. Increased expression of DOCK2 was also associated with heightened T cell infiltration in SKCM tumor tissues. Collectively, these findings highlight the promising prognostic and therapeutic implications of DOCK2 in melanoma, emphasizing its potential as a biomarker for assessing immunotherapy efficacy. Our research advocates for further investigation into DOCK2 as a viable target for precision immunotherapy, which could ultimately lead to improved treatment strategies for patients battling SKCM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05412ddf5533ddc2c8b5c7fa382b55c55f2fdd2a" target='_blank'>
              DOCK2 modulates immune checkpoint inhibitor responsiveness and prognosis in cutaneous melanoma through multi-omics and single-cell T cell dynamics profiling
              </a>
            </td>
          <td>
            Huicong Wang, Chao Feng, Chengjun Lao, Yitong Liu, Min Yan, Jiliang Wang, Zhenyan Li, Juan Li
          </td>
          <td>2025-11-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="The mesenchymal–epithelial transition/plasticity (MET/EMP) axis is a key regulator of tumor development, cancer progression, and resistance to therapy, making it an attractive target for intervention. This review highlights strategies to modulate MET/EMP using three representative agents—capmatinib, bemcentinib, and galunisertib—each acting on distinct signaling pathways. Capmatinib is a selective MET tyrosine kinase inhibitor with notable efficacy in non-small cell lung cancer harboring MET exon 14 skipping mutations. Bemcentinib blocks AXL receptor tyrosine kinase, interfering with AXL/GAS6 signaling that promotes tumor survival, metastasis, and therapeutic resistance. Galunisertib inhibits TGF-β signaling, reducing epithelial–mesenchymal transition (EMT), immune evasion, and metastatic potential. We discuss their mechanisms of action, therapeutic applications, and current clinical progress. Although these targeted therapies show potential to overcome resistance and improve patient outcomes, challenges remain due to the complex regulation of EMP. Future directions focus on refining combination strategies and advancing personalized approaches to enhance efficacy across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c6070980fd2c90add504e29dabd025c6ba8b50" target='_blank'>
              Targeted Therapies Modulating Mesenchymal–Epithelial Transition-Linked Oncogenic Signaling in the Tumor Microenvironment: Comparative Profiling of Capmatinib, Bemcentinib, and Galunisertib
              </a>
            </td>
          <td>
            Piotr Kawczak, I. Feszak, Tomasz Bączek
          </td>
          <td>2025-09-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, A. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c038a3eb980cd4c3376d3d5fe3b978d2d8815e95" target='_blank'>
              A Machine Learning Approach Reveals CRISPR-Cas I-F as a Genomic Marker of Antibiotic Susceptibility in Uropathogenic E. coli
              </a>
            </td>
          <td>
            Alexandra M Young, P. Humburg, Fang Liu, Michael C. Wehrhahn, A. Tay, Stephen M. Riordan, Li Zhang
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="RNA repair is critical for cellular function. The Rtc system maintains RNA integrity within the translational machinery of bacteria. In E. coli, Rtc expression enables cells to rescue growth and survive treatment by conferring transient resistance to ribosome-targeting antibiotics, yet the mechanisms underpinning this resistance remain obscure. Here, we present a computational model of Rtc-regulated repair of translational RNAs. Integrating model predictions with experimental validations, we uncover notable cell-to-cell heterogeneity in rtc expression that impacts on translational capacity, indicating that rtc may induce a form of heteroresistance. We moreover identify Rtc targets that may reduce the translational capacity of cells and so potentiate antibiotic effects. Our findings elucidate a complex response underpinning resistance conferred by Rtc, offering alternate routes for addressing resistance in E. coli and other relevant pathogens. RNA repair helps bacteria survive antibiotic stress. Here, authors show that Rtc-driven repair causes cell-to-cell variation in resistance levels, revealing a potential form of heteroresistance, and identify key Rtc targets to enhance antibiotic effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd19ca4b27ee23689628c6e23e1bb197f23a5f1c" target='_blank'>
              Heterogeneity in responses to ribosome-targeting antibiotics mediated by bacterial RNA repair
              </a>
            </td>
          <td>
            Hollie J Hindley, Zechuan Gong, S. Moradian, M. G. Giuliano, Andrei V. Sapelkin, I. Kotta-Loizou, Martin Buck, C. Engl, Andrea Y Weiße
          </td>
          <td>2025-11-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 IDH-mutant gliomas are diffusely infiltrating, slow-growing brain tumors that inevitably undergo malignant progression to grade IV astrocytoma - a highly aggressive and uniformly fatal disease with dismal prognosis. Despite extensive research, the molecular mechanisms underlying this transformation remain poorly understood. While alterations in canonical driver genes such as CDKN2A, PTEN, and PDGFRA have been implicated, the broader landscape of genetic drivers involved in IDH-mutant glioma progression is largely uncharacterized. In contrast to many other malignancies, IDH-mutant gliomas acquire relatively few somatic point mutations during progression - typically in the range of 30 to 40 - suggesting that alternative mechanisms may be driving tumor evolution. Notably, these tumors exhibit widespread chromosomal instability, resulting in a heterogeneous and highly prevalent pattern of copy number alterations (CNAs). This observation raises the possibility that gene dosage changes due to CNAs play a critical role in tumor progression.



 To investigate this hypothesis, we performed integrative analysis of IDH-mutant glioma cohorts from the Mayo Clinic and TCGA datasets (~1,500 patients), identifying twelve highly recurrent CNAs significantly associated with patient outcomes. We refer to these as malignant IDH-associated CNAs (mIDH CNAs). We hypothesized that these recurrent CNAs harbor driver genes that confer selective growth advantages, thereby contributing to their recurrence and prognostic relevance. To systematically assess the functional impact of genes within these regions, we developed a novel in vivo CRISPR-based screening platform, termed CRISPR-Knock Out and Activation Linked Assay (CRISPR-KOALA). This approach utilizes stereotaxic delivery of lentiviral gRNA libraries - targeting mouse orthologs of ~1,800 genes within recurrently deleted regions and ~1,200 genes within amplified regions - into the brains of a genetically engineered mouse model of IDH-mutant glioma (rs557mut;IDHmut;P53KO;AtrxKO;Cas9GFP: Yanchus et. al Science 2022).



 Mice injected with mIDH CNA-targeting libraries developed tumors significantly more rapidly than control cohorts. Deconvolution of the CRISPR-KOALA screen identified several putative driver genes, including regulators of the NOTCH signaling pathway, lipid metabolism modulators such as THEM6, and the NMDA receptor modulator GRINA - implicating dysregulation of metabolism, immune response, and glutamatergic signaling in the malignant transformation of IDH-mutant gliomas.



 Together, these findings provide critical insights into the genetic mechanisms driving the progression of IDH-mutant gliomas and identify new candidate genes and pathways for potential therapeutic targeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174bd23b8ba2dba117306ce08fc0c554d76de475" target='_blank'>
              OS09.5.A RECURRENT COPY NUMBER ALTERATIONS DRIVE IDH-MUTANT ASTROCYTOMA DEVELOPMENT AND PROGRESSION
              </a>
            </td>
          <td>
            S. Haider, K. Drucker, R. Tsai, K. Al-Zahrani, C. Chiu, T. Kollmeyer, G. Zadeh, R. Jenkins, D. Schramek
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is among the deadliest human cancers, characterized by high heterogeneity that renders standardized treatments largely ineffective. The identification of biomarkers predictive of therapy response, and elucidating their underlying mechanisms, could unveil novel therapeutic opportunities and advance precision medicine strategies. We recently demonstrated that WNT6, a key WNT signaling activator, acts as an oncogenic driver and has prognostic value in GBM, but the molecular mechanisms by which WNT6 influences GBM aggressiveness and its impact on therapy response remain elusive.



 RNA sequencing of two matched GBM in vitro models with differential WNT6 expression provided a comprehensive transcriptomic profile. Bioinformatics analyses, including functional annotation, gene-set enrichment, and ingenuity pathways analysis, identified WNT6-driven biological processes. Functional and molecular assays, along with an in vivo orthotopic experiment, evaluated WNT6’s impact on temozolomide (TMZ) sensitivity, the standard GBM chemotherapy. Clinical data analyses assessed the prognostic significance of WNT6 in TMZ-treated GBM patients.



 The WNT6 transcriptome was enriched in processes and pathways that suggest a potential role for WNT6 in therapy resistance. In vitro, WNT6 expression reduced TMZ efficacy across multiple oncogenic features, which was further supported in vivo, where WNT6-high tumors showed decreased responsiveness to TMZ in comparison to WNT6-low tumors. WNT6 expression was also associated with the differential activation of DNA repair pathways critical for TMZ response, including base excision repair, mismatch repair, and MGMT-mediated repair. Among TMZ-treated GBM patients, WNT6 was found to be an independent predictor of reduced survival.



 This study suggests that WNT6 is predictive of TMZ response in GBM, and sheds light on the molecular mechanisms by which WNT6 may be operating. Collectively, these findings position WNT6 as a promising candidate for future therapeutic strategies in precision medicine, including combination therapies targeting WNT6 and its associated resistance pathways to improve TMZ efficacy.



 Financial support was provided by grants and projects from FCT (https: //doi.org/10.54499/2022.04859.PTDC to B.M.C. and E.P.M.; https: //doi.org/10.54499/2021.02600.CEECIND/CP1664/CT0015 to C.S.G; and 2024.00785.BD to J.M.F.) and by National funds https: //doi.org/10.54499/UIDB/50026/2020, and https: //doi.org/10.54499/UIDP/50026/2020, https: //doi.org/10.54499/LA/P/0050/2020, and by the project NORTE-01-0145-FEDER-000055, supported by Norte Portugal Regional Operational Programme (NORTE 2020), under the PORTUGAL 2020 Partnership Agreement, through the European Regional Development Fund (ERDF).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93dd14a38dce213f951800a3a65a7c23b2f067e1" target='_blank'>
              P12.08.A DECIPHERING WNT6-DRIVEN MECHANISMS OF AGGRESSIVENESS AND RESISTANCE TO CHEMOTHERAPY IN HUMAN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. M. Ferreira, C. Gonçalves, E. Martins, B. Costa
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b270922941e97bc91f75cdb1ab9aa350f3916bd" target='_blank'>
              Mitotic bypass and endocycling promote cancer cell survival after genotoxic chemotherapy
              </a>
            </td>
          <td>
            Kevin Truskowski, Louis T A Rolle, George Butler, Margaret E Yang, Kenneth J. Pienta, S. Amend
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ovarian cancer is a highly lethal malignancy, with treatment efficacy frequently compromised by the development of cisplatin resistance. This study investigates the molecular mechanisms driving cisplatin resistance, focusing on interferon-induced protein with tetratricopeptide repeats 1 (IFIT1), a key interferon-stimulated gene. Through comprehensive transcriptome profiling, we systematically compared gene expression patterns between cisplatin-sensitive and cisplatin-resistant ovarian cancer cells, identifying a consistent upregulation of IFIT1 in resistant cells. Functional assays provided evidence that IFIT1 plays a central role in mediating cisplatin resistance. Mechanistically, IFIT1 was found to be associated with activation of inflammatory signaling pathways, suggesting its involvement in resistance-related immune regulation. Clinically, elevated IFIT1 expression significantly correlated with an unfavorable prognosis in ovarian cancer patients. Moreover, we demonstrated that hypoxia-inducible factor-1 alpha transcriptionally regulates IFIT1, linking its expression to hypoxia response. Overall, these findings identify IFIT1 as a key driver of cisplatin resistance in ovarian cancer, shedding light on its clinical significance and the regulatory networks underpinning therapy resistance. IFIT1 holds promise both as a prognostic indicator and as a target for therapeutic intervention. However, its clinical translation will depend on rigorous validation and in-depth exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3e993b5a5faff584d9a192dda2fe082dc88120" target='_blank'>
              Overcoming cisplatin resistance in ovarian cancer by targeting IFIT1-mediated inflammation responses
              </a>
            </td>
          <td>
            Min Hu, Xuejiao Zhao, Xiuming Li, Bin Liu, Chunluan Yuan
          </td>
          <td>2025-11-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/487c4f78dbf061ef150408c6cab4f54351fa39a1" target='_blank'>
              Forecasting multi-trait resistance evolution under antibiotic stress
              </a>
            </td>
          <td>
            Suvam Roy, E. Libby, Peter Lind
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background Since its inception, RNA sequencing has been pivotal in studying differential gene expression. Despite its extensive results in large-scale oncological studies, differential expression predominantly reflects a response to cancer. Therefore, we introduce differential allelic dispersion (AD) as a more effective measure. AD highlights consistent differences in expression between the 2 alleles of a gene that is, unlike cis-expression quantitative trait loci, independent of normal genetic variation. Such differences can, for example, arise from prevalent copy number alterations or epimutations occurring in the original cancer cell, which are mitotically expanded during cancer growth, making increased AD a marker for allele-specific dysregulation in early carcinogenesis. Findings We present the maelstRom R/C++ software package that enables (differential) AD analysis solely requiring large-scale RNA sequencing data. Using the The Cancer Genome Atlas renal clear cell carcinoma cohort as a case study, we successfully benchmark maelstRom’s AD modeling using known copy number alterations. We also detect increased AD for loci featuring normal random monoallelic expression, including the X chromosome, but demonstrate minimal interference with cancer-specific AD detection. Finally, we identify early dysregulated genes (e.g., FBP1, CCDC8, ECHS1, CLDN7) and pathways in renal cancer, often related to metabolism (e.g., pentose phosphate pathway). Strikingly, many of these genes are known causal contributors to renal carcinogenesis. Conclusions Differential AD clearly indicates early dysregulation in renal cancer, complementing basic differential expression analysis in cancer transcriptomics. AD is also relevant to study random monoallelic expression and may equally detect allele-specific (dys)regulation during early development or in noncancer diseases. maelstRom is available as an open-source software package at github.com/Biobix/maelstRom.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/457e775646c46e5a03c9c6239d832825d07a63f0" target='_blank'>
              Population-level allelic dispersion modeling by maelstRom yields genome-wide maps of allele-specific dysregulation during early carcinogenesis
              </a>
            </td>
          <td>
            Cedric Stroobandt, Louis Coussement, T. Goovaerts, Femke De Graeve, Jeroen Galle, Wim Van Criekinge, Tim De Meyer
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive and lethal form of primary brain tumor, often characterized by resistance to conventional therapies and a poor clinical prognosis. Tumor Necrosis Factor-Related Apoptosis-Inducing Ligand (TRAIL) selectively induces apoptosis in cancer cells; however, TRAIL resistance remains a significant obstacle in GBM treatment. SETD5, a histone methyl transferase with emerging roles in chromatin remodeling and gene regulation, has been implicated in neurodevelopmental disorders and cancer, but remains poorly characterized in glioma biology. This study investigates the role of SETD5 in modulating TRAIL resistance and explores its potential as a therapeutic target in GBM. Publicly available datasets (TCGA, GTEx, GEPIA2, UALCAN, Gliovis, and GSCA) were used to assess SETD5 expression, mutation, and promoter methylation in GBM. Associations with clinical parameters, molecular subtypes, immune infiltration, and functional states were analyzed using tools like cBioPortal, CancerSEA, MEXPRESS, and STRING. siRNA-mediated knockdown of SETD5 was performed in U87 and LN229 GBM cell lines, followed by TRAIL treatment. Western blotting assessed changes in apoptotic and survival pathways, while wound healing assays evaluated cell migration. Expression of deubiquitinases (USP5, USP8, and USP10) was also measured. SETD5 was significantly upregulated in GBM compared to normal brain tissues across multiple databases and was especially elevated in the Mesenchymal and Classical subtypes. CNV analyses revealed alterations in SETD5 that correlated with immunosuppressive cell infiltration. SETD5 expression was positively correlated with survival pathways and negatively correlated with cytotoxic immune infiltration. TRAIL treatment induced SETD5 expression in a dose-dependent manner. Knockdown of SETD5 enhanced TRAIL-mediated apoptosis, reduced expression of survival markers (p-AKT, P-ERK, NF-κB), and increased pro-apoptotic proteins (Cytochrome c, SMAC). SETD5 depletion also downregulated USP5, USP8, and USP10, suggesting its role in stabilizing oncogenic proteins via deubiquitination. SETD5 plays a critical role in promoting TRAIL resistance and GBM survival by regulating apoptotic pathways, immune evasion, and deubiquitinase expression. Moreover knockdown of SETD along with TRAIL treatment leads to down regulation NFκB which a major promoter for cell survival. Its inhibition sensitizes glioma cells to TRAIL-induced apoptosis, identifying SETD5 as a potential therapeutic target. Targeting SETD5 could represent a novel strategy to overcome TRAIL resistance and enhance the efficacy of GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3902ac90e3299e5f399e27e82b6fce1864afc3" target='_blank'>
              SETD5 in glioma cells conferred TRAIL resistance induction
              </a>
            </td>
          <td>
            Lakshay Taneja, Sachin Bhardwaj, Ajay Kumar Yadav
          </td>
          <td>2025-09-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The integration of chimeric antigen receptor (CAR) therapies with precision medicine holds potential to impact the treatment landscape for acute myeloid leukemia (AML). Genetic mutations play a role in the efficacy of CAR-T and CAR-NK cells, influencing their crucial role in determining the effectiveness of these cells, as well as their proliferation, persistence, resistance, and safety. This review examines how mutations in FLT3, DNMT3A, NPM1, TP53, TET2, gene fusions involving RUNX1 and KMT2A and other key genes modulate CAR-based immunotherapies, highlighting both vulnerabilities and resistance mechanisms. Recent findings demonstrate that mutations in genes such as DNMT3A and NPM1 enhance antigen expression, thereby improving CAR targeting. In contrast, mutations in TP53 drive immune escape and resistance to therapy. Understanding these mutation-specific effects is essential for tailoring CAR therapies to individual patients, optimizing efficacy while minimizing toxicity. By leveraging genomic profiling and personalized engineering approaches, CAR therapies can be refined to overcome resistance and enhance precision in AML treatment. Future research should focus on integrating multiomic data to develop mutation-adapted CAR strategies, ensuring that patients receive the most effective and personalized immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fed72fbe58eba50628755381ec963c392e48c54" target='_blank'>
              Precision medicine with car cells in acute myeloid leukemia: where are we?
              </a>
            </td>
          <td>
            Larissa C. Zanetti, Victória Tomaz, Ingrid Ferreira de Souza, P. Campregher, Nelson Hamerschlak, L. Kerbauy
          </td>
          <td>2025-11-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma (GBM), as one of the most common and aggressive primary brain tumors, pose significant challenges in diagnosis and treatment, highlighting the urgent need for the identification of novel biomarkers, NFKBIE, which may play a critical role in tumor progression, immune modulation, and therapeutic response. We investigated the role of NFKBIE in GBM through both bioinformatics analysis and experimental validation. Initially, bioinformatics analysis based on public databases revealed significant differential expression of NFKBIE across various cancer types, particularly in GBM, where elevated expression was correlated with poor patient prognosis, including overall survival, disease-specific survival, and progression-free survival. Furthermore, genetic alterations in NFKBIE, such as copy number variations (CNVs) and tumor mutational burden (TMB), were significantly associated with tumor progression. In the experimental validation phase, we utilized small interfering RNA (siRNA) technology to silence NFKBIE expression in GBM cell lines, including U87 and T98G. Functional assays were then performed to assess the impact of NFKBIE knockdown. Cell proliferation assays, including CCK-8 and clonogenic assays, demonstrated that silencing NFKBIE significantly inhibited the proliferation of GBM cells. Migration and invasion assays, including wound healing and Transwell assays, further confirmed that NFKBIE knockdown markedly suppressed the migratory and invasive abilities of GBM cells. Additionally, apoptosis assays, such as TUNEL staining, revealed that NFKBIE silencing significantly promoted apoptosis in GBM cells. Western blot analysis confirmed the changes in the expression of key markers associated with proliferation, stemness, migration, invasion, and apoptosis. Furthermore, in a mouse xenograft model, NFKBIE knockdown significantly slowed tumor growth. Histological analysis of the excised tumors confirmed that reduced NFKBIE expression was closely associated with the inhibition of tumor growth. Notably, NFKBIE silencing also led to the downregulation of components of the Hedgehog signaling pathway, suggesting that NFKBIE may regulate tumor progression through modulation of this critical pathway. In conclusion, high NFKBIE expression is strongly associated with GBM progression and poor prognosis, and targeting its expression may offer a promising therapeutic strategy for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567a01090acb470e17dfe52d5931751c0e81fc63" target='_blank'>
              NFKBIE as a prognostic biomarker and therapeutic target for GBM: role in Hedgehog signaling activation
              </a>
            </td>
          <td>
            Feng Mo, Yang Li, Ce Dong, Lijun Yang, Xiaoli Ma, Baogen Pan, Z. Fan
          </td>
          <td>2025-10-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bfbe39384668fd86cf5e7bd9896fdd269b9c106" target='_blank'>
              Peptide-Based Therapeutic Platforms for Addressing Drug Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Shreyasi Meur, Dipanjan Karati
          </td>
          <td>2025-10-23</td>
          <td>International Journal of Peptide Research and Therapeutics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Recurrent GBM poses significant treatment challenges due to limited therapeutic options and reliance on primary tumor molecular profiles, often overlooking treatment-induced molecular and phenotypic changes in recurrent tumors. This study investigates genomic, DNA methylation, and transcriptomic differences between a primary GBM (HF3016) and its matched recurrent tumor post-temozolomide and radiotherapy (HF3016R), using neurosphere cultures and patient-derived xenografts (PDXs) to assess radiotherapy (RT) response. Genomic DNA was analyzed via whole-genome sequencing, whole-exome sequencing, and EPIC DNA methylation arrays. Bulk RNA sequencing was performed on HF3016 and HF3016R cancer stem cell (CSC) cultures (n=4) and orthotopic PDX models (n=3). Differential gene expression was analyzed using NOISeq (q=0.95, fold change >2) on TMM-normalized data, with pathway enrichment via Metascape 3.1. Tumor growth and RT sensitivity were evaluated in NCRNU athymic nude mice implanted intracranially with HF3016 or HF3016R cells (3x10^5 cells/mouse, N=9-10 per group), receiving 5 Gy RT daily for 3 days or no treatment. Survival was assessed using Kaplan-Meier curves and Log-rank tests. No genomic driver alterations (CDK4, EGFR, MYC ecDNA amplification, PTEN, p53 mutations) or DNA methylation mesenchymal subgroup changes were observed between primary and recurrent models. However, a transcriptional shift from proneural to mesenchymal subtype occurred in recurrent models, with enriched epithelial-mesenchymal transition and interferon response pathways in HF3016R, while nervous system development pathways were enriched in HF3016. HF3016R PDXs exhibited faster growth (p=0.002) and greater RT resistance (p=0.0021 for HF3016, p=0.061 for HF3016R) compared to HF3016. These findings highlight the importance of considering treatment-induced molecular changes in recurrent GBM for optimizing therapeutic strategies. This study reveals that while genomic and DNA methylation profiles remain stable between primary and recurrent GBM models, significant transcriptional reprogramming occurs. The enrichment of epithelial-mesenchymal transition and interferon response pathways in recurrent models is associated with their aggressive growth and increased radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec6b57a9c052c7471268a4341bec91fcb4af93" target='_blank'>
              EXTH-73. Identifying therapeutic targets for recurrent glioblastoma using patient-derived longitudinal models
              </a>
            </td>
          <td>
            Indrani Datta, O. Nuga, A. Berezovsky, Laila M. Poisson, Ana deCarvalho
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Radiation therapy is a crucial component of the standard of care for newly diagnosed glioblastoma patients. However, the majority of tumors inevitably recur, most within the prior radiation field. Despite its broad clinical application, the molecular mechanisms that mediate response to radiation therapy in glioblastoma remain poorly understood. To address this knowledge gap, we systematically profiled the radiation sensitivity of four glioblastoma cell lines (T98G, SF295, LN18, DBTRG) and four patient-derived neurospheres (BT112, BT145, BT359, CPDM0095). The neurospheres were characterized at the genomic, transcriptomic and clinical levels. These models exhibited a spectrum of responses to radiation, with LD50s ranging from 1.5 to 8 Gy. To identify molecular factors that sensitize glioblastoma cells to radiation, we conducted a genome-wide CRISPR inactivation screen in T98G cells treated with or without a sub-lethal dose of radiation. Cells were transduced with the Inzolia multiplex CRISPR/Cas12a knockout library, targeting 19,687 single genes, 4,435 paralog pairs, 376 paralog triples, and 100 paralog quads. Cells were then divided into a control condition and an irradiated condition (3 Gy). After 18 days in culture, genomic DNA was harvested, and the relative abundance of each guide RNA array was compared between the two conditions. Targets preferentially depleted in the irradiated condition included known radiation sensitizers such as ATM and PRKDC, validating the screen’s performance. Gene set enrichment analysis (KEGG) of the top 200 hits revealed enrichment of DNA repair pathways, including homologous recombination and non-homologous end joining, as well as pathways related to the spliceosome, cell cycle, endocytosis, ubiquitin-mediated proteolysis, and pyrimidine metabolism. Notably, three components of the translesion synthesis pathway (REV1, REV3L, MAD2L2), not previously implicated in radiation response, also emerged as top candidates. A mini-pool library of the top 300 hits was subsequently generated and is being used for validation in additional glioblastoma cell lines and neurospheres. These findings provide a foundation for uncovering novel mechanisms of radiation response and nominate new targets for potential radiosensitization strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15dea4ab418adfb3586fa12497a41869a83e604c" target='_blank'>
              DNAR-13. Uncovering novel mediators of radiation sensitivity in glioblastoma using functional genomic screening
              </a>
            </td>
          <td>
            E. Aquilanti, Vincent Bozinov, Matthew L. Meyerson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pathways of intrinsic resistance in bacteria are promising targets for novel antibiotics and resistance breakers. Here, we used a genome-wide screen to identify single gene knockouts of Escherichia coli that were hypersusceptible to trimethoprim and chloramphenicol, two chemically diverse broad-spectrum antibiotics. Among the hits from our screen, knockouts of acrB, an efflux pump, and rfaG or lpxM, both involved in cell envelope biogenesis, were hypersensitive to multiple antimicrobials and could sensitize genetically resistant E. coli strains to antibiotics. Using experimental evolution under trimethoprim pressure, we show that high drug selection regimes drove these knockouts to extinction more frequently than wild type. Among them, ΔacrB was most compromised in its ability to evolve resistance, establishing it as a promising target for “resistance proofing.” At a sub-inhibitory trimethoprim concentration, however, all three knockouts adapted to the antibiotic and consequently recovered from hypersensitivity, albeit to different extents. This recovery was driven by mutations in drug-specific resistance pathways, rather than compensatory evolution, frequently involving upregulation of the drug target. Notably, resistance-conferring mutations could by-pass defects in cell wall biosynthesis more effectively than efflux even though resistant mutations did not directly engage either pathway. Since inhibiting drug-efflux emerged as a better strategy, we tested the ability of chlorpromazine, an efflux pump inhibitor (EPI), to resistance proof E. coli against trimethoprim. While qualitatively similar in the short term, genetic and pharmacological inhibition differed dramatically on an evolutionary time scale due to evolution of resistance to the EPI. Further, adaptation to the EPI-antibiotic pair also led to multidrug adaptation. The lack of concordance between genetic and pharmacological inhibition revealed a crucial lacuna in our understanding of the mutational repertoires that facilitate adaptation to antibiotics in bacteria. We propose that while intrinsic resistance mechanisms are effective targets for antibiotic sensitization, rapid evolutionary recovery may significantly limit their utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfa86a0d6ec827f3a247e4e17cf602f5ec115f1d" target='_blank'>
              Impeding pathways of intrinsic resistance in Escherichia coli confers antibiotic sensitization and resistance proofing
              </a>
            </td>
          <td>
            Manasvi Balachandran, Rohini Chatterjee, Ishaan Chaudhary, Chinmaya Jena, N. Matange
          </td>
          <td>2025-10-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background: The elucidation of the human genome has fundamentally transformed our understanding of disease etiology, positioning gene mutations as central players in susceptibility, pathogenesis, and progression across a vast spectrum of human disorders. The translation of this genetic knowledge into targeted therapeutic strategies represents the cornerstone of precision medicine, heralding a new era in clinical management.
Objective: This narrative review aims to synthesize the current landscape of how specific gene mutations influence disease development and progression, and to explore how these discoveries are shaping the development and application of novel, targeted therapeutic approaches.
Main Discussion Points: The review thematically explores the paradigm of oncogenic mutations in driving targeted cancer therapies, such as tyrosine kinase inhibitors and PARP inhibitors, while also examining the role of germline mutations in hereditary cancer syndromes. It further expands into non-oncological domains, including cardiology and neurology, highlighting the development of treatments like PCSK9 inhibitors and antisense oligonucleotides. The discussion also covers groundbreaking advanced therapies, such as gene replacement and gene editing, using examples from spinal muscular atrophy and sickle cell disease. Critical analysis is provided on the challenges of therapeutic resistance, variants of uncertain significance (VUS), and issues of health equity.
Conclusion: The collective evidence firmly establishes that targeting specific gene mutations is a powerful and transformative therapeutic strategy. However, realizing the full potential of precision medicine requires overcoming significant hurdles, including resistance mechanisms, the high cost of therapies, and a lack of diversity in genetic research. Future efforts must focus on innovative trial designs, long-term safety monitoring, and equitable implementation to ensure these advances benefit all patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382be60380435d45208a6a43c1baabce83169b3e" target='_blank'>
              NARRATIVE REVIEW ON THE IMPACT OF GENE MUTATIONS IN DISEASE SUSCEPTIBILITY, PROGRESSION, AND TARGETED THERAPEUTIC APPROACHES: A NARRATIVE REVIEW.NARRATIVE REVIEW ON THE IMPACT OF GENE MUTATIONS IN DISEASE SUSCEPTIBILITY, PROGRESSION, AND TARGETED THERAPEUTIC APPROACHES
              </a>
            </td>
          <td>
            Noor Ul, Ain Khaliq¹, Muhammad Yasir Yasin², Talha Waheed³, Atif Maqsood⁴, Ayesha Maalik⁵, Musa Khan⁶
          </td>
          <td>2025-10-04</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ovarian cancer is a lethal disease with high mortality due to late diagnosis, recurrence, and chemotherapy resistance. There is an urgent need for better diagnostic and therapeutic strategies. Recently, protein post-translational modifications (PTMs) have gained significant attention for their role in the onset, progression, and treatment of ovarian cancer. PTMs (including phosphorylation, ubiquitination, methylation, ADP-ribosylation, and others) significantly influence tumor cell proliferation, metastasis, and drug resistance by modulating cellular signal transduction, DNA repair mechanisms, and metabolic processes. PARP inhibitors block the active site of PARP1 in BRCA-mutant ovarian cancer, disrupting ADP-ribosylation, inducing apoptosis in cancer cells, and extending progression-free survival. However, the emergence of drug resistance, such as BRCA reversal mutations, and the insufficient specificity of targets remain significant limitations to therapeutic efficacy. Strategies targeting other modification pathways, such as ubiquitination and methylation, provide promising options for expanding treatments. Combination therapies, like integrating PARP inhibitors with chemotherapy or immunotherapy, and precision medicine also show potential to overcome therapeutic challenges. This article systematically examines the pivotal role of PTMs in ovarian cancer pathogenesis, outlines therapeutic strategies and associated challenges in targeting PTMs, and offers a comprehensive overview with strategic guidance for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7787d1c8ae8d6c5881a1febad8059703b970c76a" target='_blank'>
              Therapeutic targeting of post-translational modifications in ovarian cancer: mechanisms and clinical applications
              </a>
            </td>
          <td>
            Ling Ke, Ying Zhang
          </td>
          <td>2025-11-11</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a52f82067df3b6fe8b772cfcc0c82d13aa9a9b1" target='_blank'>
              Integrative transcriptomic analysis of non-small cell lung cancer: Microenvironment, circulation, and metastasis
              </a>
            </td>
          <td>
            Adnan Khosravi, Sharareh Seifi, Babak Salimi, Maryam Mabani, Parsa Rostami, Masoumeh Nomani
          </td>
          <td>2025-11-11</td>
          <td>Network Modeling Analysis in Health Informatics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7ab6bec4cbb03839af3fabebb27d4a6903a92d9" target='_blank'>
              Genome-wide characterization of clonal hematopoiesis reveals extensive non-coding putative driver mutations
              </a>
            </td>
          <td>
            J. Weinstock, K. Conneely, J. Woo, M. Arvanitis, M. Machiela, C. Russell
          </td>
          <td>2025-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768ee203fcd258706f2498c5fcf4ab4120a2eda" target='_blank'>
              Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages
              </a>
            </td>
          <td>
            Moein Rajaei, Alexander Yang, Kira A. Glasmacher, Christopher N. Cross, Nic Fisk, Elizabeth B. Perry, Jeff Mandell, S. Gaffney, Takafumi N. Yamaguchi, J. Livingstone, Jose Costa, Peter Humphrey, Vincent L. Cannataro, P. Boutros, Jeffrey P. Townsend
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Lower-grade glioma (LGG) is a common primary brain tumor with potential progression to glioblastoma. Its development is closely linked to DNA damage repair. DNA polymerase delta subunit 3 (POLD3), a subunit of the DNA polymerase δ complex, participates in DNA replication and repair. POLD3 may act as a therapeutic target and prognostic biomarker in LGG. We collected primary data from The Cancer Genome Atlas (TCGA), the Chinese Glioma Genome Atlas (CGGA), the Tumor Immune Estimation Resource (TIMER), and Gene Set Enrichment Analysis (GSEA). The CGGA dataset included messenger RNA (mRNA) sequencing and clinical data from 403 LGG cases; its microarray subset contained 142 additional cases. The TCGA dataset provided clinical data for 503 LGG cases. Employing bioinformatics tools, we investigated the correlation of POLD3 expression with the clinical characteristics, pathological features, survival outcomes, and tumor immune microenvironment in LGG. Subsequent functional validation of POLD3’s impact in LGG was performed using both cellular and animal models. Integrative analysis combining bioinformatics and experimental approaches revealed significant POLD3 overexpression in LGG. Elevated POLD3 levels exhibited significant associations with distinct clinicopathological features, altered immune cell infiltration profiles, and adverse clinical outcomes. Functionally, POLD3 was demonstrated to drive LGG cell proliferation, invasion, and modulate the tumor microenvironment. This study establishes POLD3 as a critical molecular determinant in LGG pathogenesis. Our findings position POLD3 as a promising multifaceted target, with significant implications for advancing LGG diagnostics, therapeutic development, immune modulation strategies, and prognostic assessment, thereby providing a fresh conceptual framework for understanding and managing this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b766711209d5df5ec31c2b94f1204021afe9d8" target='_blank'>
              POLD3 knockdown effects on low-grade glioma: insights from bioinformatics and experimental validation
              </a>
            </td>
          <td>
            Zhihui Yang, Zhonghao Dou, Hao Wang, Zhaojiang Hu, Yanbo Gao, Yilei Bai, Junfeng Liu, Rongjun Qian
          </td>
          <td>2025-10-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, , Maria Namwanje, Runjun D Kumar, Jiaming Li, Mark R. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer (CC) remains among the top causes of death for women worldwide, especially in low-income countries, where screening strategies are less widespread. Treatment strategies are mainly based on DNA-damaging agents, though resistance mechanisms still pose a substantial challenge. Among the cellular components that mediate treatment resistance, long non-coding RNAs (lncRNAs) stand out because of their broad regulatory effects. They are involved in virtually all drug resistance mechanisms, such as drug efflux, DNA repair, evasion of cell death, and aberrant epigenetic modifications. Although resistance mechanisms are fundamentally similar in most cancers, the underlying regulatory networks vary substantially. Here, we review the literature for lncRNAs involved in treatment resistance mechanisms in general, and then focus on lncRNAs that mediate resistance in CC. We found a broad area of opportunity in lncRNA research in resistant CC, as the lncRNAs involved are still to be described. These master regulators are promising candidates for response markers and therapeutic targets. May this compilation serve as the basis for further descriptions of the regulatory roles of lncRNA in CC treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38374134a894f6b5db043d7b741d396bd485cc0d" target='_blank'>
              Long noncoding RNAs involved in therapeutic response: implications for cervical cancer drug resistance
              </a>
            </td>
          <td>
            Samuel Trujano-Camacho, Carlos Contreras-Romero, Verónica García-Castillo, David Sánchez-Marín, Mercedes Olvera-Valencia, M. Rodríguez-Dorantes, O. Peralta-Zaragoza, David Cantú- De León, E. López-Urrutia, C. Pérez-Plasencia
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="SUMMARY Bacteroides fragilis occupies a dynamic position within the human gut. Though it comprises a relatively minor fraction of the gut microbiota, it is disproportionately enriched at extraintestinal sites of infection. This ability to survive in contrasting host environments pivots on a regulatory framework that is both modular and highly plastic. Rather than deploying a suite of hierarchical global regulators, B. fragilis employs numerous operon-embedded transcriptional switches, including site-specific DNA inversions, phase-variable epigenetic systems, extracytoplasmic function sigma/anti-sigma factor pairs, and hybrid two-component systems. These networks are further complemented by cis-regulatory elongation checkpoints and post-transcriptional control by small RNAs. This review explores the full spectrum of these regulatory mechanisms, highlighting how they facilitate niche adaptation, surface variation, immune evasion, and metabolic prioritization. It also explores intraspecies variation focusing on glycan metabolism, antibiotic resistance, and virulence. Additionally, it outlines recombination-driven regulation, alongside extracytoplasmic function sigma factor diversification, flexible promoter architecture, and elongation checkpoints, each contributing to the evolution of transcriptional control in B. fragilis. Finally, it outlines unanswered questions, including the largely unexplored sRNA regulon, the coordination of DNA inversions, elongation control, and phase-variable methylation, and proposes experimental strategies to investigate the integration of these regulatory systems during environmental transitions. Taken together, B. fragilis emerges as a model bacterium for studying decentralized gene regulation in complex microbial ecosystems, with implications for both microbial ecology and therapeutic targeting of the gut microbiota.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f20bfd3b213b02ddffd61d220bca702ce67a3e4a" target='_blank'>
              Gene regulation in Bacteroides fragilis: adaptive control in a dynamic host environment
              </a>
            </td>
          <td>
            Daniel Ryan
          </td>
          <td>2025-10-17</td>
          <td>Microbiology and Molecular Biology Reviews</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a primary driver of cervical cancer. Integration of HPV into the human genome causes persistent expression of viral oncogenes E6 and E7, which promote carcinogenesis and disrupt host genomic function. However, the impact of integration on host gene expression remains incompletely understood. We used multimodal RNA sequencing, combining total RNA-seq and Cap Analysis of Gene Expression (CAGE), to clarify virus–host interactions after HPV integration. HPV-derived transcripts were detected in 17 of 20 clinical samples. In most specimens, transcriptional start sites (TSSs) showed predominant early promoter usage, and transcript patterns differed with detectable E4 RNA region. Notably, the high RNA expressions of E4 region and viral-human chimeric RNAs were mutually exclusive. Chimeric RNAs were identified in 13 of 17 samples, revealing 16 viral integration sites (ISs). CAGE data revealed two patterns of TSS upregulation centered on the ISs: a two-sided pattern (43.8%) and a one-sided pattern (31.3%). Total RNA-seq showed upregulation of 12 putative cancer-related genes near ISs, including MAGI1-AS1, HAS3, CASC8, BIRC2, and MMP12. These findings indicate that HPV integration drives transcriptional activation near ISs, enhancing expression of adjacent oncogenes. Our study deepens understanding of HPV-induced carcinogenesis and informs precision medicine strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48263c0732371718c3701a40772227c4abc1885c" target='_blank'>
              Elucidating Alterations in Viral and Human Gene Expression Due to Human Papillomavirus Integration by Using Multimodal RNA Sequencing
              </a>
            </td>
          <td>
            Kana Tamai, S. Kinjo, Ayumi Taguchi, K. Nagasaka, Daisuke Yoshimoto, Anh Quynh Duong, Yoko Yamamoto, Hitoshi Iuchi, M. Mori, K. Sone, Michiaki Hamada, Kei Kawana, Kazuho Ikeo, Yasushi Hirota, Yutaka Osuga
          </td>
          <td>2025-10-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Acute leukemias are highly aggressive hematologic malignancies that demand intensive chemotherapy regimens. However, drug toxicity remains a major barrier to treatment success and patient survival. In this context, pharmacogenomics offers a promising strategy by identifying single-nucleotide variants (SNVs) that influence drug metabolism, efficacy, and toxicity, ultimately impacting treatment outcomes. This study analyzed data from the ClinPGx/PharmGKB database to identify clinically annotated variants related to chemotherapy response in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL). A total of 24 variants were curated for AML and 57 for ALL. Among these, nonsynonymous variants were most frequent in ALL (31.6%), while synonymous variants predominated in AML (33.3%). Although traditionally considered neutral, synonymous and intronic variants may influence gene expression through regulatory or splicing mechanisms. The analysis revealed clinically significant variants associated with chemotherapy response, particularly in the ABCB1 gene, observed in 12.5% of AML and 10.5% of ALL cases. Several variants, particularly TPMT, NUDT15, ABCC1, SLC28A3, and RARG, were associated with severe adverse effects such as myelotoxicity, mucositis, cardiotoxicity, and hepatotoxicity. This study reinforces the importance of genetic variants in modulating the therapeutic response and toxicity to chemotherapy drugs in acute leukemias. Analysis of ClinPGx/PharmGKB data emphasizes ABCB1 as a potential resistance marker and supports pre-treatment genotyping of genes like TPMT and NUDT15 to prevent severe toxicities. Future advances should include the expansion of pharmacogenetic studies in underrepresented populations and the clinical validation of new markers in prospective trials, aiming to consolidate precision medicine as a routine part of the therapeutic management of acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d398f2f78483d0f180028688c26b519950cb78" target='_blank'>
              Pharmacogenomics Applied to Acute Leukemias: Identifying Clinically Relevant Genetic Variants
              </a>
            </td>
          <td>
            Flávia Melo Cunha de Pinho Pessoa, Isabelle Magalhães Farias, B. Nogueira, C. B. Machado, Igor Valentim Barreto, Anna Karolyna da Costa Machado, Guilherme Passos de Morais, Leidivan Sousa da Cunha, Deivide de Sousa Oliveira, André Pontes Thé, Rodrigo Monteiro Ribeiro, Patrícia Maria Pontes Thé, Manoel Odorico de Moraes Filho, Maria Elisabete Amaral de Moraes, Caroline Aquino Moreira-Nunes
          </td>
          <td>2025-10-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cellular plasticity enables cells to dynamically adapt their phenotype in response to environmental cues, a process central to development, tissue repair, and disease. Among the most studied plasticity programs is epithelial–mesenchymal transition (EMT), a transcriptionally controlled process by which epithelial cells acquire mesenchymal traits. Originally described in embryogenesis, EMT is now recognized as a key driver in both tumor progression and fibrotic remodeling. In cancer, EMT and hybrid epithelial/mesenchymal (E/M) states promote invasion, metastasis, stemness, therapy resistance, and immune evasion. In fibrotic diseases, partial EMT (pEMT) contributes to fibroblast activation and excessive extracellular matrix deposition, sustaining organ dysfunction mainly in the kidney, liver, lung, and heart. This review integrates recent findings on the molecular regulation of EMT, including signaling pathways (TGF-β, WNT, NOTCH, HIPPO), transcription factors (SNAIL, ZEB, TWIST), and regulatory layers involving microRNAs and epigenetic modifications. Moreover, we discuss the emergence of pEMT states as drivers of phenotypic plasticity, functional heterogeneity, and poor prognosis. By comparing EMT in cancer and fibrosis, we reveal shared mechanisms and disease-specific features, emphasizing the translational relevance of targeting EMT plasticity. Finally, we explore how cutting-edge technologies, such as single-cell transcriptomics and lineage tracing, are reshaping our understanding of EMT across pathological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bed4863500de9abc33bfc60581f8d09b96d454d" target='_blank'>
              Shared and Context-Specific Mechanisms of EMT and Cellular Plasticity in Cancer and Fibrotic Diseases
              </a>
            </td>
          <td>
            V. A. F. Bastos, A. G. Souza, Virginia C. Silvestrini Guedes, T. Cunha
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This comprehensive review investigates the complex interactions between oncogenesis and stem cells, underscoring the crucial importance of cancer stem cells (CSCs) in the mechanisms of tumorigenesis, metastatic dissemination, and resistance to therapeutic interventions. CSCs constitute a distinct subset within neoplasms, defined by unique characteristics such as the ability to self-renew, the potential to differentiate into various cellular phenotypes, and an inherent resistance to conventional treatment strategies. It is hypothesized that these properties are regulated by a multitude of signaling pathways, including Wnt/β-catenin, Notch, and Hedgehog signaling cascades. The tumor microenvironment (TME) significantly influences CSC dynamics by providing a nurturing niche that enhances CSC survival, enables immune evasion, and contributes to resistance against therapies. Key components of the TME, such as cancer-associated fibroblasts and immune-modulating exosomes, play a crucial role in augmenting CSC plasticity and metastatic capabilities. Furthermore, the hypoxic environment prevalent within the TME enhances CSC stemness through both metabolic and epigenetic alterations. Despite the existence of promising CSC-targeted therapeutic strategies, including immunotherapy, small-molecule inhibitors, and combinatorial approaches, notable challenges persist. These obstacles include CSC plasticity and heterogeneity, as well as the potential for immune rejection of CSC-targeted therapies. Advancements in precision medicine, genomics, and novel therapeutic interventions instill hope for the improvement of CSC-targeted strategies. A comprehensive understanding of the fluid interactions between CSCs, standard stem cells, and the TME is vital for advancing personalized cancer treatments that enhance patient success and reduce recurrence rates. This review highlights the need for continued research to bridge existing knowledge gaps and facilitate the translation of foundational scientific discoveries into clinical practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65aea70f8623678ce094b97b023361a5efc723c" target='_blank'>
              Biology of stem cell paradox: a double-edged sword-implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Yitayew Demessie, M. Kinde, Kalkidan Getnet, A. Bitew, Kassahun Berrie, Wagaw Sendeku, Asnakew Mulaw Berihun, A. Abat, Atsede solomon Mebratu, Melaku Getahun Feleke, Nesibu Tilahun Yesist, M. M. Ferede, Melkie Dagnaw Fenta
          </td>
          <td>2025-10-28</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background/Aim: Cancer remains a leading cause of mortality globally, driven by complex molecular mechanisms and characterized by significant biological heterogeneity across cancer types. We aimed to discover mitogen-activated protein kinases (MAPKs) family members as both biomarkers and therapeutic targets in different cancer types. MAPKs are key signaling molecules regulating cell proliferation, differentiation, stress response, and apoptosis. Dysregulation of MAPK pathways has been implicated in the onset and progression of multiple cancers, contributing to tumor growth, metastasis, and therapeutic resistance. Given their diverse roles across cancer types, systematic analysis of MAPK gene expression, mutations, and interactions with tumor microenvironment is essential. Materials and Methods: The present study undertakes a comprehensive transcriptomic analysis of MAPKs in nine major cancer types using RNA-Seq datasets from The Cancer Genome Atlas (TCGA). RNA-Seq data were analyzed to identify differentially expressed MAPKs across nine cancer types using DESeq2, NOIseq, and limma. Significant genes (adjusted p<0.05) were subjected to GO and KEGG enrichment (EnrichR), mutational profiling (cBioPortal), and Kaplan-Meier survival analysis. Tumor immune infiltration was further assessed using TIMER2.0 to explore immune–gene interactions. Results: Through rigorous differential expression analysis, we identified key MAPKs that are significantly altered in liver hepatocellular carcinoma (LIHC) and lung adenocarcinoma (LUAD). Specifically, six MAPKs (MAPK3, MAPK7, MAPK9, MAPK10, MAPK12, and MAPK13) were found to be differentially expressed in LIHC, while MAPK6 emerged as the sole significant candidate in LUAD. Functional enrichment and pathway analysis revealed that these MAPKs are involved in critical oncogenic pathways, including MAPK-mediated transcriptional regulation and stress-activated signaling cascades. Mutational profiling and survival analysis further validated the prognostic significance of these genes, with several MAPKs showing strong associations with reduced patient survival. Tumor immune infiltration analysis indicated potential roles of these MAPKs in modulating immune responses within the tumor microenvironment. Also, MAPKs identified in this study are structurally related, suggesting that targeting them collectively may enhance therapeutic efficacy and overcome resistance mechanisms. Conclusion: Our integrated approach underscores the value of MAPK family members as both biomarkers and therapeutic targets in LIHC and LUAD. This study contributes important insights into MAPK-related oncogenic processes and supports the development of targeted therapies under the framework of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b14065bbb6005ef268d03b15d61fb336481cae" target='_blank'>
              Investigating the Roles of MAPKs as Precision Oncology Targets: A Multi-cancer Expression and Survival Study
              </a>
            </td>
          <td>
            Md Nayab Sulaimani, Prithvi Singh, Mohammad Umar Saeed, Sobia Zaidi, Afzal Hussain, M. Alajmi, Anas Shamsi, Ravins Dohare, M. Hassan
          </td>
          <td>2025-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 IDH-mutant gliomas are typically slow growing on initial presentation, however selective pressure during conventional upfront treatment with genotoxic radiation and chemotherapy can fuel the evolution of sub-populations of tumor cells. These subclones promote recurrence of IDH-mutant gliomas that are less likely to benefit from standard treatments. Therefore, novel tumor-specific therapeutic strategies with non-overlapping mechanisms are needed for durable tumor control. IDH mutant cancer cells are known to reprogram their metabolism to sustain rapid proliferation, which generate potentially targetable vulnerabilities in canonical metabolic pathways. Using the patient-derived, WHO grade 3 oligodendroglioma line TS603, we screened a small molecule library of 240 metabolism-related compounds, including both FDA-approved and non- approved drugs, over a range of 10 concentrations (1nM-20µM). Drug efficacy was assessed after 5-day incubation via cell viability assay (Cell-Titer Glo). We identified 53 compounds with IC50<2µM, including drugs that target previously reported IDH-mutant glioma targets, like NAMPT and DHODH. Notable target pathways include folate metabolism and de novo thymidylate synthesis (methotrexate, raltitrexed, 5-FU), p97 AAA ATPase/VCP chaperone (CB-5083, ML240, eeyarestatin I), mevalonate pathway (Ro 48-8071 fumarate, atorvastatin calcium), and the BCL-2 family (Navitoclax, ABT-737). We are validating these drugs across a panel of patient-derived IDH-mutant gliomasphere models, and examining the combinatorial effects of these agents with vorasidenib and CDK4/6 inhibitors. Our results provide insight into important metabolic pathway dependencies in recurrent IDH1-R132H mutant gliomas that have targeting potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99af98af377416756584e34e3b8b2129cefb73a2" target='_blank'>
              DDDR-40. Identifying metabolic vulnerabilities of recurrent IDH1-R132H mutant glioma by high throughput screening
              </a>
            </td>
          <td>
            Yutong Wang, Sayali Daware, D. Cahill, Julie Miller
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f73f73f3e37aa93d53554d82182a30e140df7f94" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identifies CLK1 inhibition as a strategy to restore PARP inhibitor sensitivity via ERCC1 isoform switching.
              </a>
            </td>
          <td>
            Chaohua Liu, Xu Fei, Yutuan Wu, Li Jiana, Mengdong Ni, Xia Siyu, Chen Lihua, Haiyun Zhao, Yu Min, Zhou Yuqi, Meiqin Zhang, Jiajia Li, Xiaohua Wu, Huang Yan, Zhu Tao, Xiaojun Chen
          </td>
          <td>2025-11-04</td>
          <td>Protein & cell</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ca5c42693a477fde2aa581c6c760a192f8faa2" target='_blank'>
              The cumulative impact of passenger mutations on cancer development
              </a>
            </td>
          <td>
            Akshatha Nayak, Candace S. Y. Chan, Ioannis Mouratidis, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Antibiotic resistance remains one of the most formidable challenges to modern medicine, threatening to outpace therapeutic innovation and undermine decades of clinical progress. While resistance was once viewed narrowly as a clinical phenomenon, it is now understood as the outcome of complex ecological and molecular interactions that span soil, water, agriculture, animals, and humans. Environmental reservoirs act as silent incubators of resistance genes, with horizontal gene transfer and stress-induced mutagenesis fueling their evolution and dissemination. At the molecular level, advances in genomics, structural biology, and systems microbiology have revealed intricate networks involving plasmid-mediated resistance, efflux pump regulation, integron dynamics, and CRISPR-Cas interactions, providing new insights into the adaptability of pathogens. Simultaneously, the environmental dimensions of resistance, from wastewater treatment plants and aquaculture to airborne dissemination, highlight the urgency of adopting a One Health framework. Yet, alongside this growing threat, novel therapeutic avenues are emerging. Innovative β-lactamase inhibitors, bacteriophage-based therapies, engineered lysins, antimicrobial peptides, and CRISPR-driven antimicrobials are redefining what constitutes an “antibiotic” in the twenty-first century. Furthermore, artificial intelligence and machine learning now accelerate drug discovery and resistance prediction, raising the possibility of precision-guided antimicrobial stewardship. This review synthesizes molecular insights, environmental drivers, and therapeutic innovations to present a comprehensive landscape of antibiotic resistance. By bridging ecological microbiology, molecular biology, and translational medicine, it outlines a roadmap for surveillance, prevention, and drug development while emphasizing the need for integrative policies to safeguard global health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2665fac6c17a4051c07ada66840ae2833282cd2b" target='_blank'>
              Beyond the Resistome: Molecular Insights, Emerging Therapies, and Environmental Drivers of Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-10-01</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancers exhibit genetic diversity between individual cancer cells. Previous work shows greater diversity than heretofore expected (1) and that also increases more quickly during a patient’s clinical course than previously thought (1, 2). Rare subclones will harbor pre-existing resistance to any single agent and may cause medium to late term relapse (1), which may evolve further variants that have simultaneous resistance to non-cross resistant therapies (3, 4). We now present a probability distribution function (PDF) of the variant allele fraction (VAF) or prevalence of a rare subclone, derived from previous evolutionary theory (1, 2). We show that current clinical sequencing protocols fail to detect the vast majority of rare subclones. By the time of detection, simultaneous multiple resistance may evolve. We then apply the PDF to simulation of dynamic precision medicine (DPM) (3), an evolutionary guided precision medicine paradigm that attempts to proactively eliminate singly-resistant subclones before they evolve multiple resistance, with significant potential to extend survival. We show that the simulated benefit of DPM with perfect information is degraded by inability to detect rare subclones if they are assumed to be absent when undetectable. But this benefit is restored if the PDF is used to calculate the likelihood of the subclone being present below the level of detection and incorporated into the DPM simulation and therapy recommendations in a probabilistic fashion. Moreover, other common statistical distributions are less effective. This theoretical advance facilitates DPM and potentially other evolutionary guided approaches to precision cancer medicine in spite of the limitations of clinical sequencing. Significance Statement Cancers contain many cells, each genetically unique. These variations can include pre-existing resistance to therapy, enabling relapse. DNA sequencing cannot detect minority cellular populations (subclones) below a certain size, by which time they may have evolved simultaneous resistance to multiple therapies. We present a mathematical approach that enables assigning a risk that a subclone is present, and at what prevalence in the cellular population, even when undetected. We simulate applying this approach to dynamic precision medicine (DPM), which attempts to proactively eliminate singly resistant cells before they become resistant to multiple therapies. Using this probability distribution, we can retain the benefit of DPM even when most rare subclones are undetectable, in contrast to just assuming undetected subclones are absent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary This review highlights the role of lncRNAs in tumor progression and therapy resistance, focusing particularly on their ability to drive the transition from reversible epigenetic adaptations to stable genetic changes in resistant cancer cells. lncRNAs not only modulate gene expression and signaling pathways through interacting with miRNAs, DNA, RNA, and proteins, but also actively contribute to the reprogramming of the cellular environment. This action facilitates the shift from early adaptive, often transient, epigenetic mechanisms of resistance to more permanent genetic alterations that support tumor evolution and sustained therapy resistance. Their widespread expression and deregulation in tumors suggest their strong potential as prognostic biomarkers and as therapeutic targets, especially in combination with traditional or next-generation treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f629bb465bf4a59c23101191cc50b7fe057523" target='_blank'>
              The Role of LncRNAs in Radio- and Chemoresistance of Glioblastoma: Prognostic or Therapeutic?
              </a>
            </td>
          <td>
            Elisa Tremante, Ana Belén Díaz Méndez, Maria Giulia Rizzo
          </td>
          <td>2025-09-27</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epidermal growth factor receptor exon 20 insertion mutations (EGFRex20ins) are a unique molecular subtype of non-small cell lung cancer (NSCLC) linked to resistance to EGFR tyrosine kinase inhibitors of the first and second generations. Until recently, these patients had few and frequently ineffective treatment options. Amivantamab, a bispecific antibody targeting both EGFR and mesenchymal–epithelial transition factor, has become a novel therapeutic strategy for this population. This review explores the mechanism of action of Amivantamab and its clinical efficacy and safety as demonstrated in clinical trials. Additionally, the clinical development of the subcutaneous formulation of amivan-tamab, real-world evidence and its regulatory status were evaluated. Lastly, we contex-tualise Amivantamab in the current treatment landscape by contrasting it with mobocer-tinib and highlighting current studies that aim to improve central nervous system activity and overcome resistance mechanisms. This review highlights the therapeutic benefit of Amivantamab in EGFRex20ins-mutated NSCLC and offers guidance for future research in this quickly developing area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac73581dcae1cceea8c975ae407a4672d857585f" target='_blank'>
              Evolving treatment strategies for EGFRex20ins-mutated NSCLC: a comprehensive review of Amivantamab’s role and future directions
              </a>
            </td>
          <td>
            Rafael Alvim Pereira, M. Tumelero, W. Schwengber, G. Lenz
          </td>
          <td>2025-11-17</td>
          <td>ecancermedicalscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) remains one of the most lethal malignancies, primarily due to its robust resistance to genotoxic therapies, such as chemotherapy and radiotherapy. Understanding the mechanisms underlying this resistance is essential to improve clinical outcomes. Here, we identified trimethylguanosine synthase 1 (TGS1), previously known for its role in RNA modification, as a critical mediator of homologous recombination (HR) repair that specifically contributes to resistance in PAAD. TGS1 was significantly overexpressed in PAAD tissues, correlating strongly with advanced disease stages, therapy resistance, and poor patient prognosis. Following DNA damage, ATM kinase phosphorylated TGS1 at serine residues S389 and S531, which mediated its direct interaction with BRCA1 and subsequent recruitment of BRCA1 to DNA damage sites. The phosphorylation-dependent interaction enhanced HR repair efficiency, enabling cancer cells to survive genotoxic stress. Depletion or pharmacological inhibition of TGS1 induced HR deficiency and markedly enhanced sensitivity to DNA-damaging agents, particularly PARP inhibitors, in PAAD cell lines in vitro and in both cell line-derived and patient-derived xenograft models in vivo. Collectively, these findings uncover an ATM-TGS1-BRCA1 signaling axis that promotes DNA repair and resistance to genotoxic therapies in pancreatic cancer, positioning TGS1 as a promising predictive biomarker and therapeutic target to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abe0d36d758071ca95a0b7efd96da1ea51c63685" target='_blank'>
              Targeting the ATM-TGS1-BRCA1 Axis Overcomes Genotoxic Therapy Resistance in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Changying Li, Xuran Zhao, Xinyan Li, Chunyan Wang, Zhenyu Huo, Xin Xu, Wenjuan Kang, S. Nowsheen, Khaled Aziz, Guangyi Sun, Zhihua Liu, Zhenkun Lou, Min Deng
          </td>
          <td>2025-11-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Hypoxia is a recognized driver of transcriptional heterogeneity in high-grade glioblastoma (GBM), yet its functional significance in low-grade gliomas (LGGs) remains poorly understood. We hypothesized that hypoxia orchestrates a reversible quiescence-activation axis in LGGs via EGFR-dependent transcriptional reprogramming, thereby influencing tumor progression.



 We conducted integrative analyses of single-cell RNA sequencing (n=7 patients) and spatial transcriptomics (n=8 samples) across molecularly defined IDH-mutant LGGs to delineate hypoxia-associated transcriptional states. EGFR mutation, and clinical correlations with overall survival, were assessed in The Cancer Genome Atlas (TCGA) bulk RNA-seq cohorts. To functionally validate microenvironmental contributions, we established innovative co-culture models of LGG cells with hypoxia-conditioned primary microglia and astrocytes.



 We identified a hypoxia-associated, LGG-specific ribosomal-enriched (RE) transcriptional program characterizing a non-cycling subpopulation enriched in quiescence and activation stages. Quiescent RE cells exhibited glioma stem cell (GSC)-like features, with EGFR signaling dependency for transition to an activated phenotype. Spatial transcriptomics revealed distinct co-localization of hypoxia, RE program expression, and quiescent states, whereas EGFR-activated cells formed discrete micro-niches. Hypoxia-conditioned microglia significantly enhanced RE program activation and EGFR signaling, establishing a functional microenvironmental driver of phenotypic plasticity.



 Our findings delineate a novel hypoxia-EGFR signaling axis sustaining a quiescent yet activation-capable subpopulation in IDH-mutant LGGs. These insights nominate hypoxia-modulated EGFR pathways as potential therapeutic targets to disrupt tumor cell plasticity. Future studies will interrogate vulnerabilities of RE-programmed LGG cells under hypoxic conditions to inform next-generation therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748507ebfe3412d3babe3bb777f9dae226192bc1" target='_blank'>
              P02.22.A HYPOXIA-DRIVEN EGFR SIGNALING PROMOTES A QUIESCENT-ACTIVATION AXIS IN IDH-MUTANT LOW-GRADE GLIOMAS
              </a>
            </td>
          <td>
            D. Das, V. Jariwala, A. Pandele, A. Mohammed, T. Das, N. Ghariwala, C. Krolla, S. Arosh, N. Hossain, I. Hosen, S. Smith, J. Beck, J. Grauvogel, I. Vasilikos, R. Roelz, R. Rahman, M. Börries, K. Joseph, V. Ravi, G. Andrieux, S. Chakraborty
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary malignant tumor of bone and occurs predominantly in children and adolescents during periods of accelerated skeletal growth. Within the tumor microenvironment (TME), vascular smooth muscle cells (VSMCs) may acquire osteogenic characteristics. Nuclear factor of activated T cells c1 (NFATc1) is central to skeletal biology, yet its behavior in VSMCs in osteosarcoma remains insufficiently defined. We profiled fresh osteosarcoma tissues by single-cell RNA sequencing on the 10x Genomics Chromium platform. Mechanical and enzymatic dissociation yielded single-cell suspensions; viable cells (> 85%) were enriched; libraries were prepared with Single Cell 3′ v3 reagents and sequenced on NovaSeq 6000. Reads were processed with Cell Ranger (GRCh38). Following log-normalization, 2,000 highly variable genes were retained. Harmony integration reduced batch structure. We computed 50 principal components, generated UMAP/t-SNE embeddings, and applied k-nearest neighbor/Louvain clustering (k = 20; resolution = 0.5). Cell types were assigned from differentially expressed markers cross-checked with CellMarker and PanglaoDB. NFATc1 distribution and co-expression patterns were assessed by Pearson correlation. Nine major cell classes were identified, including VSMCs, fibroblasts, macrophages, T/NK subsets, B cells, neutrophils, mast cells, and inflammatory myeloid cells. NFATc1 expression was most prominent in VSMCs, with a bimodal distribution within this lineage. In VSMCs, NFATc1 co-varied with SERGEF, APOE, and CCS, pointing to coordinated cytoskeletal remodeling, lipid handling, and copper homeostasis during osteogenic reprogramming. At single-cell resolution, NFATc1 marks a VSMC fraction with osteogenic features in osteosarcoma. The associated network suggests testable links to metabolism and matrix remodeling; functional perturbation will be required to establish causality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3638bf4253781f5bd9cf49aea68da6335438a6" target='_blank'>
              Single-cell RNA sequencing reveals NFATc1-mediated regulatory mechanisms of vascular smooth muscle cell osteogenic transformation in the osteosarcoma microenvironment
              </a>
            </td>
          <td>
            Xuejiao Yi, Shilin Fu, Chunliang Zhang, Tianle Zhao, Huan Feng, Dehui Qian
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms. Cancer cell plasticity is often linked to treatment failure. Here, the authors discuss the role of epigenome in DNA repair and, how DNA repair-induced chromatin changes influence cancer cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31e0a8462023f5f44b70b1be6da7b2a7f15e9f9" target='_blank'>
              Imp, a key regulator of transposable elements, cell growth, and differentiation genes during embryogenesis
              </a>
            </td>
          <td>
            P. Vazquez-Pianzola, D. Beuchle, P. M’Angale, Gimena Alegre, Greco Hernández, S. Bullock, Travis Thomson, B. Suter
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
This review explores the relationship between genotypes and resistant phenotypes in Gram-negative pathogens. We analyse to what extent conventional phenotypic methods predict genetic mechanisms of resistance, the reliability of genotypic approaches, and how integrated strategies may improve diagnostic accuracy and clinical utility.


RECENT FINDINGS
Traditional AST remains the clinical reference standard due to its correlation with therapeutic outcomes, yet it often fails to identify the molecular basis of resistance. Molecular methods such as PCR, microarrays, and targeted sequencing allow rapid detection of known genes but cannot reliably predict expression or functionality. Whole-genome sequencing provides the most comprehensive overview, capturing both known and novel resistance determinants as well as mobile genetic elements. Nevertheless, genotype-phenotype discordance persists, driven by regulatory mutations, inducible expression, or synergistic mechanisms. Emerging technologies - including real-time sequencing, metagenomics, and machine learning-based predictive models - are enhancing our ability to infer phenotypes from genomic data. Still, these approaches face challenges of standardization, validation, and integration into clinical workflows.


SUMMARY
Linking genotypes to resistant phenotypes in Gram-negative pathogens remains complex. While phenotypic AST ensures reliability for therapy, genotypic methods provide unprecedented insight into resistance mechanisms and epidemiology. Discrepancies between the two highlight the need for integrated diagnostic platforms that combine functional and genomic perspectives. Artificial intelligence-driven predictive models and curated resistance databases hold promise for improving accuracy, but widespread adoption requires robust datasets, clinical validation, and harmonized interpretative frameworks. Ultimately, integrating phenotypic and genotypic data represents the most effective strategy to provide mechanism-informed, clinically actionable diagnostics for antimicrobial resistance management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f34c2b73b9d73ae3113ba2142013f1f7e8cf759a" target='_blank'>
              Is it possible to identify genotypes underlying resistant phenotypes in Gram-negative pathogens?
              </a>
            </td>
          <td>
            A. Burillo, Julia Serrano-Lobo, Emilio Bouza, Patricia Muñoz
          </td>
          <td>2025-10-14</td>
          <td>Current opinion in infectious diseases</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chemotherapy resistance is a critical barrier to effective cancer treatment, contributing to the high number of cancers related death in patients. This review comprehensively examines the molecular mechanisms driving resistance, including innate and acquired resistance, tumor heterogeneity, and tumor microenvironment (TME) interactions. Key mechanisms include enhanced drug efflux via ABC transporters, increased DNA repair, altered drug targets, drug detoxification, and evasion of apoptosis. Tumor heterogeneity fosters resistant subclones, while TME factors like hypoxia and cancer-associated fibroblasts (CAFs) exacerbate resistance. Crosstalk between signaling pathways, such as PI3K/AKT and MAPK, fuels resistance but offers hope for combination therapies. Insights into these mechanisms are vital for developing targeted therapies to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215edb205618059ddf2fe5de4347e899f8539201" target='_blank'>
              Mechanism of Chemotherapy Resistance: a Comprehensive Review
              </a>
            </td>
          <td>
            Subash Paudel, Laxmi Parajuli, Santosh Shrestha, Indira Rai
          </td>
          <td>2025-09-30</td>
          <td>Nepalese Journal of Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The endosomal pathway plays a crucial role in eukaryotes, and its conservation across evolution underscores its importance in fundamental processes, such as endocytosis, cytokinesis, and autophagy. Dysregulation of the Notch signaling system, essential for growth and development, contributes to cancers like leukemia. In Drosophila melanogaster (DM), specific endosomal genes function as tumor suppressors, regulating Notch signaling, either through promoting Notch degradation or directing its recycling. Nevertheless, it is still unanswered how endosomal genes specifically, lethal giant discs (lgd/CC2D1A), affect Notch-1-mediated transduction in human malignancies, such as T-cell acute lymphoblastic (T-ALL). This study aimed to comprehensively characterize the (lgd/CC2D1A) CC2D1A/lgd in human T-ALL by performing mutational analysis and sequencing and evaluating its gene and protein expression levels. These analyses were conducted to elucidate better the role of (lgd/CC2D1A)CC2D1A in the pathophysiology of T-ALL and to explore its potential association with the Notch-1 signaling pathway. In this study, we sequenced the lgd and investigated its expression in peripheral blood samples of human T-ALL samples. We used molecular approaches to analyse the Lgd/CC2D1A gene for mutations and characterize its gene and protein expression across genomic and proteomic datasets. Sequencing data revealed the presence of the lgd variant (CC2D1A) in T-ALL patients and healthy individuals. Lgd and Notch-1 expression were significantly upregulated in T-ALL patients, demonstrating a strong positive correlation at both the genomic and proteomic levels. Through this analysis, we reported the presence of Lgd with an altered nucleotide sequence in T-ALL patients. We also found its elevated expression compared with healthy controls, which might affect disease progression. Our investigation results also lay the foundation for the possible development of Lgd as a predictive indication for the early identification of T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b8e3e49ce5c7f6dd3ebfcedabaf637bb4da449" target='_blank'>
              Characterization, expression, and mutational analysis of endosomal protein lethal giant disc (Lgd/CC2D1A) as a biomarker in patients with T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sandeep Pandey, Ranjana Singh, Nimra Habib, Rashmi Kushwaha, A. A. Mahdi
          </td>
          <td>2025-11-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mismatch repair (MMR) system alterations can trigger transient hypermutation, promoting adaptive mutations under stress, such as antibiotic exposure. While most organisms use MutS and MutL protein families for MMR, many archaea and actinobacteria, including the major human pathogen Mycobacterium tuberculosis, lack these components and instead rely on NucS, a structurally distinct enzyme driving a non-canonical MMR pathway. Given the role of MMR in mutation control, understanding how nucS expression is regulated could be essential for uncovering the molecular basis of antibiotic resistance development in mycobacteria. In this study, we characterized the nucS promoter and transcription start site in Mycobacterium smegmatis. We found that nucS expression declines during the stationary phase in both M. smegmatis and M. tuberculosis, paralleling replication activity and canonical MMR downregulation. Our data suggest that the alternative sigma factor σB may negatively regulate nucS expression during this phase. Additionally, we identified candidate compounds that may modulate nucS expression, underscoring its responsiveness to environmental cues. These findings enhance our understanding of mycobacterial stress responses and lay the groundwork for exploring antibiotic resistance mechanisms. Strikingly, our work reveals a case of double convergent evolution: both canonical (MutS/MutL) and non-canonical (NucS) pathways have independently evolved not only the same DNA repair function, but also similar regulatory frameworks for genome integrity preservation under stress conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7d5cf01753d5a2e85e0726bcedfc7ae6cce138b" target='_blank'>
              Regulation of the Expression of nucS, a Key Component of the Mismatch Repair System in Mycobacteria
              </a>
            </td>
          <td>
            E. Cebrián-Sastre, Ángel Ruiz-Enamorado, A. Castañeda-García, Susanne Gola, Pablo García-Bravo, Leonor Kremer, Jesús Blázquez
          </td>
          <td>2025-10-24</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90dbf82e0a483cd704102a3b6a055d355e22f224" target='_blank'>
              A mathematical model of drug-resistant cancer with cell cycle structure: stability, optimal control, and cost-effectiveness analysis
              </a>
            </td>
          <td>
            Tawfik Jaber, O. Balatif
          </td>
          <td>2025-09-29</td>
          <td>International Journal of Dynamics and Control</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b73fae020e6752c501ebd1de56b61a2010abe7" target='_blank'>
              Cancer metabolism: bridging tumorigenesis mechanisms to treatment susceptibility.
              </a>
            </td>
          <td>
            Jian Liu, Dan Shan, Zhaokai Zhou, Xutao Wen, Rao Fu, Bo Xu, Peng Luo, Zhengrui Li, Ling Zhang
          </td>
          <td>2025-09-28</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Highlights What are the main findings? Emerging data from multidimensional and high-resolution phenotyping of disease-relevant tissues indicates that osteoarthritis (OA) is a polygenic disease, where genetic factors disrupt the chromatin landscape in disease-relevant cells leading to aberrant expression of effector genes that drive OA pathogenesis. According to the concept of a developmental origin for OA, the functional cooperation between chromatin dynamics and transcription factors (TFs) regulates the unfolding of a development-specific gene expression program, defines the outcome of skeletogenesis, and ultimately determines the articular biomechanics and the risk of OA. What is the implication of the main findings? Detailed mapping and functional characterization of the OA-associated chromatin conformation and epigenetic disruptions may accelerate the discovery of disease-modifying drugs for OA. Novel technologies pave the way for precise epigenetic editing at the desired genomic regions and may allow a targeted transcriptional regulation of OA-relevant genes in disease-relevant cells. Abstract Recent technological breakthroughs have enabled multidimensional phenotyping, with unprecedented single-cell resolution and genome-wide coverage, across multiple osteoarthritis (OA)-relevant tissues, such as articular cartilage, synovium, infrapatellar fat pad, and subchondral bone. The majority of the single nucleotide variations (SNVs) that have been associated with OA are located in non-protein coding regions and confer risk for disease by altering the expression level, instead of the amino acid sequence of the gene product. These data have shaped the concept of OA as a polygenic disease, where genetic factors disrupt the chromatin landscape in disease-relevant cells, leading to aberrant expression of effector genes. Pharmacologic manipulation of the OA-driving epigenetic landscape has recently emerged as an attractive path for the development of disease-modifying drugs. Novel clustered regulatory interspaced short palindromic repeats (CRISPR)-based technologies provide opportunities for precise epigenetic editing at the desired genomic regions and may allow a targeted transcriptional regulation of disease-relevant genes in disease-relevant cells. The aim of the present narrative review is to summarize the emerging data on the role of epigenetic factors and chromatin structure as calibrators of the risk for developing OA and to discuss the opportunities and challenges arising from the use of chromatin landscape to guide drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb518b2e515941622a9daa713244c8885ed8ed56" target='_blank'>
              The Articular Chromatin Landscape in Osteoarthritis
              </a>
            </td>
          <td>
            G. Kalliolias, E. Basdra, Athanasios G. Papavassiliou
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background/Aim: Lineage plasticity enables prostate cancer cells to bypass androgen receptor (AR) dependence, contributing to metastasis, treatment resistance, and lethality. The developmental transcription factor PROX1 was recently identified as an early driver of lineage plasticity. The TMPRSS2-ERG fusion, a common genomic event in prostate cancer, induces ERG overexpression and aggressive tumor behavior. We investigated the relationship between PROX1 expression and ERG expression in the context of TMPRSS2-ERG fusion in prostate cancer. Materials and Methods: We analyzed data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) dataset (n=492 samples). The UCSC Xena Browser was used to visualize and integrate gene expression, copy number alterations, and fusion status. Complementary analyses were performed using cBioPortal. Genomic instability was assessed using both fraction of the genome altered (FGA) and mutation count. The correlation between PROX1 and ERG expression was evaluated in fusion-positive and fusion-negative tumors. Results: PROX1 expression showed a strong positive correlation with ERG expression in TMPRSS2-ERG fusion-positive tumors (r=0.4, p=3.2×10−16). Fusion-positive samples exhibited elevated PROX1 expression, compared to fusion-negative samples. This pattern suggests that PROX1 induction is associated with ERG-driven transcriptional reprogramming linked to the TMPRSS2-ERG fusion. Conclusion: PROX1 expression is significantly associated with ERG expression and TMPRSS2-ERG fusion status in prostate cancer. These findings reinforce the role of PROX1 as an early marker and potential mediator of lineage plasticity. Targeting PROX1 or its regulatory pathways may offer a novel therapeutic strategy to mitigate plasticity-driven progression and treatment resistance in TMPRSS2-ERG fusion-positive prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c9168d60bbc1c5a4a0d510943f10c27a506bc7" target='_blank'>
              PROX1 Expression Is Significantly Associated With ERG Expression and the TMPRSS2-ERG Fusion Gene in Prostate Cancer
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2025-09-26</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) remains a leading cause of cancer mortality, with therapeutic resistance posing the primary barrier to durable outcomes. Beyond genetic and epigenetic alterations, amino acid transporter-driven metabolic reprogramming—mediated by LAT1 (SLC7A5), ASCT2 (SLC1A5), and xCT (SLC7A11)—supports tumor proliferation, redox homeostasis, and immune escape. Their preferential expression in NSCLC highlights their potential as therapeutic targets and predictive biomarkers. In parallel, α-particle therapy has gained attention for its capacity to eradicate resistant clones through densely clustered, irreparable DNA double-strand breaks. Astatine-211 (211At) combines a clinically relevant half-life, high linear energy transfer, and predictable decay scheme, positioning it as a unique candidate among α-emitters. Preclinical studies of 211At-labeled transporter ligands, particularly LAT1-targeted conjugates, demonstrate potent tumor suppression and synergy with targeted therapy, chemotherapy, radiotherapy, immunotherapy, and ferroptosis inducers. Advances in radiochemistry, delivery systems (antibodies, peptides, and nanocarriers), and PET tracers such as [18F]FAMT and [18F]FSPG collectively support a theranostic framework for patient stratification and adaptive dosing. By linking transporter biology with α-particle delivery, 211At-based theranostics offer a mechanistically orthogonal strategy to overcome resistance and heterogeneity in NSCLC. Successful translation will depend on precise dosimetry, scaffold stabilization, and biomarker-guided trial design, enabling progression toward first-in-human studies and future integration into multimodal NSCLC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebf01ec54df66cef316dd1652f94338f18b56da1" target='_blank'>
              Astatine-211-Labeled Therapy Targeting Amino Acid Transporters: Overcoming Drug Resistance in Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Sifan Feng, Kentaro Hisada, Haruna Yorifuji, Y. Shirakami, K. Kaneda-Nakashima
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The biological mechanisms driving the long survival in glioblastoma (GBM). Five‐year long‐term survival (LTS) and 10‐year survival very long‐term survival (VLTS) remain significantly understudied. Here we molecularly detailed two cases. AR10‐046 (VLTS) was affected by a giant cell‐GBM, classified as the pedHGG_RTK1a subtype according to the v12.5 Heidelberg brain tumor methylation classifier. Somatic and germline MSH6 mutations, typically in Lynch syndrome, and high tumour mutational burden were detected. The copy number variation plots showed chromosome 1q gain and chromosome 13 loss with no other typical GBM alterations. AR10‐037 (LTS) suffered from a classical GBM, identified as pedHGG_MYCN subclass. Apart from the canonical chromosome 7 gain and chromosome 10q loss, we observed MDM2 gene amplification and possible rearrangements on chromosome 12 and 18 with the typical aspect of chromothripsis, harbouring two putative new gene fusions: CPSF6::CPM and PTPRR::RAB3IP. We described two patients with peculiar tumour molecular profile, widening the scenario of clinical and molecular variability in such patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8551ad58743a80e7a359a8ec1bcd9af76ef1c7" target='_blank'>
              Unique Genetic and Epigenetic Alterations in Glioblastoma Long‐Term Survivors: Insights From Two Clinical Cases
              </a>
            </td>
          <td>
            E. Anghileri, E. Miele, S. Patrizi, S. Barresi, E. Lazzarini, L. Maddaloni, M. Patanè, L. Pedace, R. Paterra, A. Silvani, Franco Locatelli, S. Indraccolo, B. Pollo
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cancers are complex diseases characterized by dynamic perturbations of regulatory networks across multiple hierarchical levels, which cannot be fully captured by alterations in a small number of genes. To this end, based on the concept of Hallmarks of Cancer, a whole genomic data-driven approach is proposed to capture the dynamic variation from normal to cancerous cells. This framework focuses on the characteristic functional modules of cancer via hallmarks of cancer by constructing a coarse-grained gene regulatory network of hallmarks. Through this framework, with stochastic differential equations, macroscopic dynamic changes in tumorigenesis are simulated and further explored. The analysis results reveal that network topology undergoes significant reconfiguration before shifts in hallmark levels, serving as an early indicator of malignancy. A pan-cancer examination across 15 cancer types uncovers universal patterns, for example, the “Tissue Invasion and Metastasis” hallmark exhibits the most significant difference between normal and cancer states, while “Reprogramming Energy Metabolism” shows the least pronounced differences. These findings reinforce the systemic nature of cancer evolution, highlighting the potential of network-based systems biology methods for understanding critical transitions in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6814d7f9a3ff58aa852df9a43fe33cc0022a2e6" target='_blank'>
              Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-11-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Posttranslational modifications (PTMs) on histones play critical roles in cellular processes, including gene expression and tumorigenesis. However, the regulatory mechanisms and functional consequences of newly identified lysine acylation modifications in cancer therapy remain to be elucidated. Here, we established diverse preclinical tumor models resistant to epithelial growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) and focused on histone lysine crotonylation (Kcr), as it exhibited more pronounced alterations compared to other acylations. Next, we identified acyl-CoA synthetase short-chain family member 2 as a key regulator responsible for the resistance-associated decrease in crotonylation levels. Furthermore, integrated crotonylomic, transcriptomic, and epigenomic profiling, supplemented by gene manipulation studies, revealed that EGFR-TKI resistance resulted from transcriptional suppression of HNF1A and activation of the PI3K/AKT signaling pathway, which were regulated by reduced histone H3 lysine 56 crotonylation. Importantly, through pharmacological screening, we identified a histone decrotonylase inhibitor that enhanced EGFR-TKI sensitivity by activating epigenetic reprogramming through the selective upregulation of histone Kcr levels across multiple models in vitro and in vivo. Collectively, our findings uncover a previously unrecognized epigenetic mechanism driven by crotonylation that contributes to EGFR-TKI resistance, highlighting the potential of modulating crotonylation as a novel therapeutic strategy to enhance the efficacy of EGFR-TKIs in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9119d6e3343091d67a219621bccb08e3e1171344" target='_blank'>
              Activation of epigenetic reprogramming via crotonylation overcomes resistance to EGFR-TKI therapy in lung cancer.
              </a>
            </td>
          <td>
            Sihong Chen, Mengyan Zhong, Xiaoyuan Wang, Youwei Su, Yingxi Zhao, Simeng Wang, Shougeng Liu, Yongshan Zhao, Yang Zhang, Xinyu Fan, Zongang Liu, Lina Jia, Xuefei Bao, Wei Cui, Jingyu Yang, Chun-Hui Wu, Guoliang Chen, Lihui Wang
          </td>
          <td>2025-09-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Despite significant advances in cancer research, the complexity of signaling pathways remains a major challenge in precision oncology. Tumors harbor a diverse array of genetic alterations; however, these often converge onto a limited set of core signaling pathways, notably RAS/RAF/MEK/ERK (MAPK), PI3K/AKT/mTOR (PAM), and Wnt/β-catenin. Concurrently, these pathways diverge extensively downstream, driving therapeutic resistance through mechanisms such as epithelial-mesenchymal transition, immune evasion, and metabolic reprogramming. Unlike previous literature that largely provides descriptive accounts of pathway alterations, this review uniquely synthesizes convergent and divergent signaling into a clinically actionable diagnostic and therapeutic framework. It critically assesses current precision oncology strategies, identifies gaps, and proposes a dual-axis model that integrates static genomic profiling with dynamic signaling evolution to inform precision therapy. By highlighting opportunities for combination therapy informed by pathway interdependencies and adaptive resistance mechanisms, this perspective provides novel clinical insights and tangible directions for future research in overcoming resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f096cbac949c4b15346a048f653260da1f8e6" target='_blank'>
              Convergent and Divergent Signaling Pathways in Cancer: A Dual-Axis Model for Adaptive Precision Oncology.
              </a>
            </td>
          <td>
            Aliasgar Shahiwala PhD
          </td>
          <td>2025-10-08</td>
          <td>Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precision therapy for glioma remains a major challenge due to tumor heterogeneity. The Origin Recognition Complex Subunit 6 (ORC6) is a crucial regulator of DNA replication initiation. This study aims to investigate the expression of ORC6 in gliomas and its relationship with survival rates and malignancy, while screening potential drugs targeting its functional network. By integrating multiple bioinformatics approaches with structure-based virtual screening, retrospective RNA sequencing data analysis was performed using patients from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. A protein-protein interaction (PPI) network was constructed from ORC6-coexpressed genes to identify core hubs. Molecular docking was employed to screen a library of natural compounds and known drugs against these hub targets. Research has revealed that ORC6 is significantly upregulated in high-grade gliomas, with its elevated expression associated with poor survival outcomes and immune inflammatory responses. Network analysis identified five core hub genes (ORC1, ORC2, MCM2, MCM6, CDC45) central to DNA replication. Molecular docking revealed that several compounds, including the natural flavonoid Baicalein and the FDA-approved drug Palbociclib, exhibited high binding affinity to these hub targets. ORC6 represents a highly promising novel target for precision therapy in glioma. Potential approaches to target this pathway include disrupting the ORC6-replication axis using existing drugs (such as palbociclib) or natural products (such as baicalin).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324196dc2699eff3ae974af550ed77475be4c5ee" target='_blank'>
              High ORC6 expression is a prognostic indicator of poor survival in glioma patients
              </a>
            </td>
          <td>
            Mengjie Wang, Song Feng, Chen Zhang, Feng Jin
          </td>
          <td>2025-11-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) is a reversible growth arrest induced by anticancer treatments, which may contribute to the development of long-term therapy resistance in tumor cells. Senotherapeutics, agents targeting senescent cells, are being tested in clinical trials to improve patient outcomes. Due to the transient nature of TIS, we hypothesized that senolytics would be most effective when administered at the appropriate time. We created a reliable TIS cell line model in triple-negative breast cancer (TNBC) using experimental drug YM155. We observed that a single dose of YM155 triggers a brief senescence, leading to a persistent drug-tolerant state that cannot be reversed by redosing. This reversibility is not limited to cancer cells. It extends to noncancerous human cells and live zebrafish larvae, suggesting a rapid adaptation mechanism against xenobiotics. We identified transforming growth factor-β (TGF-β), a cytokine linked to TNBC chemoresistance, as being expressed alongside the emergence of drug tolerance. We inhibited TGF-β signaling to eliminate the tolerant phenotype and promote the clearance of cancer cells by immune cells. However, this was most effective within a specific time window after TIS induction. We suggest that the timely use of senotherapeutics could improve the effectiveness of anticancer drugs in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ec0acb6fede0ff85bcba84d195e45333ad41d7" target='_blank'>
              Mapping the Progression of Therapy-Induced Senescence to Therapy Tolerance: An Evolutionarily Conserved Mechanism for Optimizing Cancer Treatment with Senotherapeutics.
              </a>
            </td>
          <td>
            Gargi Mukherjee, Neha Dutta, Muthumeena Ramanathan, Kankonika Bhattacharyya, Alabhya Das, Shaista Haider, Goutam Chowdhury, Anindita Chakrabarty
          </td>
          <td>2025-10-10</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is notorious for its rapid progression, tendency to metastasize, high recurrence rates, dismal outcomes, and limited treatment options, underscoring the urgent need to uncover new biomarkers and molecular pathways to enhance diagnosis, prognosis, and therapeutic strategies. Metabolic reprogramming continues to play a role throughout the life cycle of cancer, evolving and adapting. In this study, we aimed to identify specific genes associated with metabolic reprogramming in TNBC, which can potentially become unique biomarkers of this cancer. TNBC datasets retrieved from the Gene Expression Omnibus were employed to pinpoint genes exhibiting altered expression linked to tumor metabolic reprogramming. Key genes were accurately screened through machine learning algorithms, and then externally verified using the TBNC dataset based on the Cancer Genome Atlas database. Finally, immunohistochemical methods were used to clinically confirm the differential expression and trends of these key genes. Our analysis accurately identified four genes—CLEC7A, IRS1, RSPO3, and ALB—that are closely correlated with the metabolic reprogramming characteristics of cancer, and could be regarded as innovative biomarkers for TNBC. This opens a new avenue for further investigation into the mechanisms of metabolic reprogramming in TNBC and new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe8f189447c321586a776e941cd069962ea01d63" target='_blank'>
              Identification and analysis of metabolic reprogramming-related genes in triple-negative breast cancer
              </a>
            </td>
          <td>
            Husheng Kong, Mengyuan Pan, Li Sun
          </td>
          <td>2025-10-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Mutational activation of the KRAS oncogene in pancreatic ductal adenocarcinoma (PDAC) plays a crucial role in tumor initiation, progression, and alterations of cellular programs that support PDAC growth. Mutant KRAS alters the metabolic state of PDAC and has been linked to upregulation of glycolysis and alterations in the nutrient scavenging pathways autophagy and macropinocytosis. Our lab determined that genetic suppression or inhibition of the RAS pathway unexpectedly resulted in upregulation of autophagy. Additionally, concurrent inhibition of the RAS-MAPK pathway and autophagy synergistically improved anti-tumor efficacy in both PDAC cell lines, organoids, and PDX tumor models. To begin to address the temporal regulation of metabolic signaling following RAS inhibition, we examined the sustainability of upregulated autophagic flux following prolonged (168 h) RAS inhibition. Preliminary data indicates that autophagy upregulation following KRAS inhibition is transient and returns to basal levels following prolonged inhibitor treatment. Given the limited durability of autophagic induction upon RAS inhibition, we sought to determine whether other nutrient scavenging pathways that culminate in the lysosome were altered following prolonged RAS inhibitor treatment. Macropinocytosis is upregulated in PDAC and is RAS-dependent. We monitored macropinocytic uptake following direct inhibition of RAS, and found that following an initial decrease in macropinocytosis, PDAC cells exhibited a compensatory increase in macropinocytotic uptake and utilization at later timepoints. Based on this data, we have begun to interrogate the metabolic phenotypes associated with cells subjected to acute (24-72 h) and prolonged (168 h) RAS inhibitor treatment. Furthermore, we have also derived PDAC cell lines that are resistant to RAS inhibition via prolonged inhibitor dose escalation. We found that RAS inhibitor resistant cell lines have heterogenous capacities for autophagic flux and retain similar sensitivity to autophagy inhibitors. Conversely, resistant cells display persistently increased macropinocytosis, suggesting that that scavenging pathway compensates for metabolic requirements of growth under sustained RAS inhibition. Future studies are aimed at investigating the molecular underpinnings driving upregulated macropinocytosis and more broadly profiling the metabolic landscape of RAS-inhibitor resistant PDAC models.



 Sarah E. Ackermann, Ryan Robb, Khalilah E. Taylor, Runying Yang, Kirsten L. Bryant. Assessing metabolic requirements for RAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f9b16a108ea5a5b93f12330dee22b394bce19" target='_blank'>
              Abstract A006: Assessing metabolic requirements for RAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Sarah E Ackermann, Ryan Robb, Khalilah E Taylor, Run-Wei Yang, Kirsten L. Bryant
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf829c80ee16d8593d7d0383b2985d0175ddbfa6" target='_blank'>
              Cell passage number drives transcriptomic drift as an overlooked factor in experimental reproducibility.
              </a>
            </td>
          <td>
            Shuai Liu, Hailun Peng, Junyan Meng, Liyuan Zhuo, Shaoping Fu, Wei Yang
          </td>
          <td>2025-11-21</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-10</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related mortality among women worldwide, with treatment resistance posing a significant clinical challenge. Circular RNAs (circRNAs), a class of non-coding RNAs, have gained increasing attention as key regulators of gene expression, influencing BC pathogenesis, progression, and therapeutic response. This review explores the mechanistic insights into circRNA functions in BC, focusing on their involvement in tumor proliferation, metabolic reprogramming, epithelial-mesenchymal transition (EMT), angiogenesis, metastasis, and apoptosis. Additionally, we highlight the crosstalk between circRNAs and microRNAs, emphasizing their potential as diagnostic and prognostic biomarkers. Beyond their roles in tumor biology, circRNAs are implicated in drug resistance, modulating responses to chemotherapy, targeted therapy, and endocrine treatment. Despite their promising applications, challenges remain, including the complexity of circRNA interactions, and the development of robust preclinical models. Addressing these challenges through interdisciplinary research integrating genomics, transcriptomics, and functional studies will pave the way for circRNA-based therapeutic strategies and personalized medicine approaches in BC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3e2c40f929a3a40c435801c1f0dae7da3b0326" target='_blank'>
              Circular RNAs orchestrating breast cancer hallmarks: bridging tumor biology and therapy resistance
              </a>
            </td>
          <td>
            Abdelhamid M Abdelhamid, Shafei Ahmed Mai, Mariam Gamaleldin, Alaa Bassam Heikal, Wesam H Khidr, Nabil Hazza Al-Saadi, Ganna Magdy Kandil, Mahmoud M Omarn, Maha Alhelf
          </td>
          <td>2025-10-13</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematopoietic stem cell (HSC) therapy remains essential in treating blood disorders, autoimmune diseases, neurodegenerative conditions, and cancers. Despite its potential, challenges arise from the inherent heterogeneity of HSCs and the complexity of their regulatory niche. Recent advancements in single-cell RNA sequencing and chromatin accessibility sequencing have provided deeper insights into HSC markers and chromatin dynamics, highlighting the intricate balance between intrinsic and extrinsic regulatory mechanisms. Zebrafish models have emerged as valuable tools in HSC research, particularly through live imaging and cellular barcoding techniques. These models have allowed us to describe critical interactions between HSCs and embryonic macrophages, involving reactive oxygen species and calreticulin signaling. These are essential for ensuring HSC quality and proper differentiation, with implications for improving HSC transplant outcomes. Furthermore, the review examines clonal hematopoiesis, with a focus on mutations in epigenetic regulators such as DNMT3A, TET2, and ASXL1, which elevate the risk of myelodysplastic syndromes and acute myeloid leukemia. Emerging technologies, including in vivo cellular barcoding and CRISPR-Cas9 gene editing, are being investigated to enhance clonal diversity and target specific mutations, offering potential strategies to mitigate these risks. Additionally, macrophages play a pivotal role in maintaining HSC clonality and ensuring niche localization. Interactions mediated by factors such as VCAM-1 and CXCL12/CXCR4 signaling are crucial for HSC homing and the stress response, opening new therapeutic avenues for enhancing HSC transplantation success and addressing clonal hematopoiesis. This review synthesizes findings from zebrafish models, cutting-edge sequencing technologies, and novel therapeutic strategies, offering a comprehensive framework for advancing HSC biology and improving clinical outcomes in stem cell therapy and the treatment of hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6985cdf565a94b5e4ce65dadd2a9ba4efa0f62f7" target='_blank'>
              Advancements in Hematopoietic Stem Cell Therapy: From Biological Pathways to Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Viviana Cortiana, H. Chorya, R. H. Abbas, Jade Gambill, Adhith Theyver, Chandler Park, Yan Leyfman
          </td>
          <td>2025-10-15</td>
          <td>Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has revolutionized oncology by achieving durable remissions in refractory hematologic malignancies. However, emerging reports link this therapy to second primary malignancies, including CAR+ lymphomas and leukemias, driven by insertional mutagenesis from semi-random viral vector integration near oncogenes or tumor suppressor loci. These rare but serious complications underscore the dual challenge of eradicating primary tumors while mitigating delayed genotoxic risks. Conventional CAR-T cell manufacturing, reliant on gamma-retroviral or lentiviral vectors, introduces genomic instability through integration into fragile sites or transcriptionally active regions. CRISPR/Cas9-mediated genome editing further amplifies risks via off-target double-strand breaks and chromosomal rearrangements. This review evaluates genomic safe harbors (GSHs)—such as AAVS1, TRAC, CCR5, ROSA26 and CLYBL—as loci validated for stable, high-level CAR transgene expression without oncogenic disruption. GSHs meet stringent criteria: distal from cancer-related genes, resistant to epigenetic silencing, and transcriptionally permissive. Preclinical studies demonstrate that site-directed CAR integration into GSHs preserves antitumor efficacy while eliminating malignant transformation risks. Challenges persist in optimizing homology-directed repair efficiency, mitigating residual dsDNA toxicity, and standardizing regulatory frameworks for long-term genomic surveillance. Emerging technologies—base/prime editing, hybrid nucleases, and rigorous monitoring—promise enhanced precision and safety. By reconciling therapeutic innovation with genomic integrity, GSH-engineered CAR-T cells herald a paradigm shift toward precision immunotherapies, offering curative potential while preempting secondary oncogenesis. Collaborative efforts to refine manufacturing, harmonize global standards, and prioritize patient-specific risk stratification will be critical to advancing this transformative approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a919a0552fea16eb7ba7478bb3fc8327302c77cd" target='_blank'>
              Preventing secondary primary malignancies (SPMs) in CAR-T cell therapy through site-specific transgene integration into genomic safe harbors (GSHs)
              </a>
            </td>
          <td>
            Pooria Safarzadeh Kozani, Pouya Safarzadeh Kozani
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most common malignant primary brain tumor in adults. Cadherin-3 (or P-cadherin) is a cell adhesion molecule encoded by the CDH3 gene, which we previously identified as a key contributor to GBM aggressiveness. Despite its functional effects, the underlying mechanisms through which Cadherin-3 affects GBM remain unclear. This study aims to elucidate the molecular, cellular, and metabolic processes mediated by CDH3 that drive GBM.



 RNA sequencing was performed in two GBM models, including a patient-derived culture and a cell line, genetically manipulated to express differential levels of CDH3. Metabolic profiling was conducted via Seahorse analyses and extracellular glucose and lactate quantifications by HPLC. Lactate levels were further measured in the interstitial fluid of subcutaneous xenograft tumors. Protein expression of metabolism-associated markers was evaluated through western blot and immunohistochemistry in GBM cells and in in vivo tumors. The Cancer Genome Atlas (TCGA) data was used to study correlations between CDH3 and metabolic gene signatures, and associations with patient survival.



 Transcriptomic analyses revealed that CDH3 expression is associated with multiple oncogenic pathways, including those related to metabolism. Findings from independent GBM cell models indicate that Cadherin-3 influences energy metabolism in a cell context-dependent manner, affecting the balance between glycolysis and oxidative phosphorylation. In GBM patient samples, CDH3 expression is positively correlated with glycolytic/plurimetabolic (GPM)-related genes, and inversely correlated with mitochondrial (MTC) signature genes. Notably, high CDH3 expression combined with the GPM signature defines a subgroup of patients with particularly poor survival outcomes.



 These findings suggest Cadherin-3 as a key regulator of GBM metabolism, shaping distinct energetic profiles depending on the cellular context. Furthermore, its association with the GPM signature identifies a subset of GBM patients with worse prognosis, underscoring the potential for CDH3 as both a biomarker for patient stratification and a possible therapeutic target in GBM.



 Financial support was provided by FCT - Fundação para a Ciência e a Tecnologia - https: //doi.org/10.54499/2022.04859.PTDC, PD/BDE/143154/2019, COVID/BDE/153298/2023, co-funded by MCTES, FSE, UE, and NORTE 2020; and by National funds through FCT - https: //doi.org/10.54499/UIDB/50026/2020, https: //doi.org/10.54499/UIDP/50026/2020, and https: //doi.org/10.54499/LA/P/0050/2020, and by the projects NORTE-01-0145-FEDER-000039, and NORTE-01-0145-FEDER-000055, supported by NORTE 2020, under the PORTUGAL 2020 Partnership Agreement through the European Regional Development Fund (ERDF).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86af8fd649dc07baf379d8b4dbce9b39b94fac35" target='_blank'>
              P01.23.A CADHERIN-3-MEDIATED METABOLIC REPROGRAMMING AND PROGNOSTIC STRATIFICATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            E. Martins, C. Gonçalves, J. Peixoto, J. M. Ferreira, S. Queirós, A. Longatto-Filho, P. M. Costa, N. S. Osório, S. Granja, B. Costa
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Non-small cell lung cancer (NSCLC) is the most common form of lung cancer and a leading cause of cancer-related death. Despite therapeutic advances, long-term survival in stage IV disease is uncommon. Tumor analyses that combine genomic and functional platforms may provide the opportunity to monitor clonal dynamics and guide therapy selection. Case Presentation: We report a 67-year-old woman with metastatic poorly differentiated lung adenocarcinoma, who achieved four durable remissions and survived nearly 12 years. Serial studies using ex vivo analysis of programmed cell death (EVA/PCD) functional-profiling-guided therapeutic choices were correlated with next-generation sequencing (NGS). Molecular events included the emergence of a BRAF V600E mutation responsive to dabrafenib plus trametinib and the acquisition of an EGFR exon 19 deletion responsive to Osimertinib. EVA/PCD identified activity for targeted agents and revealed synergy for vinorelbine plus Osimertinib not predicted by genomic profiling, which provided additional response. Discussion: This case highlights clonal evolution in NSCLC and illustrates how serial tissue analyses correlating phenotypic and genomic events can offer therapeutic interventions to provide long-term survival. Conclusions: The integration of functional and genomic profiling may improve personalized treatment in NSCLC by interrogating tumor heterogeneity and clonal evolution to inform rational therapeutic selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb66a2f50798ff6be9a24ac5af17d9ad46fa3951" target='_blank'>
              Serial Functional and Genomic Analyses Illuminate Clonal Evolution in Metastatic NSCLC with 12-Year Survival
              </a>
            </td>
          <td>
            Vikrant S. Bakaya, Sabina A. Schneider, Tracy Nguyen, Derrick Phu, Lucas A. Alvarez, S. S. Evans, P. Bernard, F. Francisco, Adam J. Nagourney, Luisa Torres, John Henry, P. D’Amora, R. Nagourney
          </td>
          <td>2025-11-19</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Melanoma is a highly heterogeneous disease, with unique genetic subtypes that influence clinical behavior and treatment response. While targeted therapies and immunotherapy have transformed care for more common types of melanoma, optimal strategies for KIT-mutant melanoma are less well-defined. In this review, we summarize the associations between KIT mutations and specific clinico-pathologic patterns, including their enrichment in acral, mucosal, and chronically sun-damaged melanomas. We detail the spectrum of KIT mutations across relevant exons, noting how mutation subtype influences sensitivity to targeted therapies. We will then analyze several therapeutic trials and case reports that describe the use of c-KIT inhibitors as well as immune checkpoint inhibitors in melanoma treatment. We also discuss early findings supporting combination strategies that may enhance therapeutic outcomes. Finally, we identify critical gaps in understanding the mechanisms of resistance, CNS progression, and the immunogenic landscape of KIT-driven melanoma. Deeper insight into the function of KIT and its interaction with therapeutic pathways is essential to optimizing treatment sequencing and tailoring personalized strategies that improve outcomes in this patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a3b1783130151a4e725cc98816d0f672f99e5" target='_blank'>
              KIT-Mutant Melanoma: Understanding the Pathway to Personalized Therapy
              </a>
            </td>
          <td>
            Aditi Kaveti, Ryan J. Sullivan, Hensin Tsao
          </td>
          <td>2025-11-13</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Persister cells in Escherichia coli (E. coli) represent a transient, non-mutational form of antibiotic tolerance that contributes to treatment failure and chronic infections. This study investigated the genetic and epigenetic responses of E. coli persister cells exposed to ciprofloxacin-induced stress, with a focus on the role of nucleoid-associated proteins (NAPs) in promoting DNA protection and persistence. Fifty clinical E. coli isolates were screened for persister cells using the Rapid Killing and Replica Plating Tolerance Isolation System (REPTIS). The expression of selected NAP genes (hns, hupA, hupB, and dps) was analyzed using real-time PCR (RT-qPCR), while hns gene methylation was assessed via qMethyl PCR. DNA integrity was evaluated by the neutral comet assay, and Dps protein expression was measured by Western blotting. Persister cells were detected in 4% of isolates, with isolate E3 confirmed by REPTIS. RT-qPCR showed significant upregulation of NAP genes in persisters, with hupB showing the highest fold increase (685-fold). qMethyl PCR revealed elevated hns methylation levels (61.55%), suggesting epigenetic regulation. The comet assay showed reduced DNA damage in persister cells, while Western blotting indicated a 1.2-fold increase in Dps expression under ciprofloxacin stress. These findings suggest that E. coli persistence involves a multifaceted mechanism, including transcriptional activation, epigenetic modifications, and protein-level regulation of NAPs. The enhanced expression of Dps under antibiotic stress highlights its protective role and supports the potential of targeting NAPs in developing novel therapeutic strategies against persistent and recurrent E. coli infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77531baaae0bf08af5e5d25a617bafc8cffc2938" target='_blank'>
              EPIGENETIC AND GENETIC RESPONSES OF ESCHERICHIA COLI PERSISTER CELLS TO CIPROFLOXACIN STRESS: INSIGHTS INTO THE ROLE OF NUCLEOID-ASSOCIATED PROTEINS
              </a>
            </td>
          <td>
            Ahmed Abdulwahhab, Mohammed Al Marjani, Zayneb Abdulghany
          </td>
          <td>2025-09-29</td>
          <td>Journal of microbiology, biotechnology and food sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CRISPR screening has revolutionized cancer research by enabling systematic, genome-wide interrogation of gene
function to identify therapeutic vulnerabilities, resistance mechanisms, and immunotherapy determinants. This review
examines methodological approaches for CRISPR screenings using the Cas9 systems, evaluating how system and methodology selection
profoundly influence which biological mechanisms are revealed across therapeutic resistance, metastasis regulation, and immunotherapy
response. Cas9-based screens-from simple one-cell-type (1CT) in vitro approaches to complex autochthonous in vivo systems-excel at
identifying cell-autonomous DNA-level dependencies. Two-cell-type (2CT) co-culture approaches validate immune-dependent mechanisms,
while in vivo screens capture physiological complexity and have identified clinically relevant targets including PTPN2, now in Phase 1 trials.
Critical challenges include incomplete gene editing, false-positive and false-negative hits, context-dependent essentiality, and system-specific
technical limitations. Emerging technologies integrating artificial intelligence, prime editing, single-cell profiling, and patient-derived models
promise to address limitations while enabling functional precision medicine. This review provides a framework for selecting appropriate
systems, interpreting results within methodological constraints, and integrating complementary approaches to maximize biological insight and
translational impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6e21f4ff0a121855b3f14e7fcc069197b5d3507" target='_blank'>
              CRISPR- CAS SCREENING IN CANCER RESEARCH: METHODOLOGIES, APPLICATIONS, AND TRANSLATIONAL CHALLENGES
              </a>
            </td>
          <td>
            Ahana Patel
          </td>
          <td>2025-11-01</td>
          <td>INDIAN JOURNAL OF APPLIED RESEARCH</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbb0b650183e1ab55b2143b3779619fab2e24dd" target='_blank'>
              aiAtlas: High-Fidelity Cell Simulations of Genetic Perturbations in Rare Diseases and Cancers
              </a>
            </td>
          <td>
            WR Danter
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized cancer treatment, offering durable responses in a subset of patients. Despite this progress, cancer cells have developed sophisticated mechanisms to evade immune surveillance, including genetic mutations, activation of immune escape pathways, alterations in the tumor microenvironment, and metabolic reprogramming. Additionally, the immune system’s ability to effectively target cancer cells can be compromised by immune tolerance and functional exhaustion. There is growing evidence that abnormal activation of cancer signaling pathways, such as the Wnt/β-catenin, Notch, JAK/STAT, p53, and PTEN pathways, acts independently or through crosstalk to form complex regulatory networks that influence tumor immune escape, the therapeutic response, and immune resistance. Consequently, elucidating the mechanisms by which these signaling pathways contribute to tumor immune escape is essential for optimizing current immunotherapy regimens and developing novel therapeutic strategies. Targeting these pathways in combination with immunotherapy represents a promising approach to overcome immune resistance and enhance treatment efficacy. The integration of pathway modulation with immunotherapeutic interventions may reveal new avenues for more effective and durable cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9142ff6a5cec86bc497f410ddb74aab33a5d7d49" target='_blank'>
              From intracellular drivers to immune modulators: emerging paradigms in oncogenic pathway-directed immunotherapy optimization
              </a>
            </td>
          <td>
            Dean Rao, Tiantian Wang, Chenan Fu, Yiming Luo, Junli Lu, Zhoubin Sun, Wei Yan, Huifang Liang, Limin Xia, Wenjie Huang
          </td>
          <td>2025-10-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Many chemotherapeutic agents exert their cytotoxic effects primarily by inducing DNA damage. In response to DNA damage, cells activate a signaling cascade known as DNA damage response (DDR) to repair the damage and promote cell survival. Accordingly, the capacity of the DDR in cancer cells is a critical factor that influences their sensitivity to chemotherapy. Here, we identified a role for interferon γ-inducible protein 16 (IFI16) in modulating the DDR and chemosensitivity of breast cancer cells. Depletion of IFI16 in MDA-MB-231 cells conferred resistance to the DNA-damaging agents doxorubicin and 5-fluorouracil, as evidenced by increased cell viability and reduced caspase-3 cleavage compared to control cells. Mechanistically, IFI16 interacted with the MRE11-RAD50-NBS1 complex and disrupted the interaction between NBS1 and ataxia telangiectasia mutated (ATM), a critical step for ATM activation. In vivo, xenograft tumors derived from IFI16 knockout cells exhibited diminished responses to doxorubicin treatment, characterized by decreased apoptotic cell death and reduced expression of DSB marker proteins, such as γH2AX and 53BP1. Furthermore, analysis of breast cancer patient datasets revealed that high IFI16 expression correlated with an improved pathological complete response rate following chemotherapy. Our findings suggest that IFI16 could serve as both a predictive biomarker for chemotherapy response and a potential therapeutic target for enhancing the efficacy of DNA-damaging agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97552038179200f37fe515ddaf27aa8bae8377e" target='_blank'>
              IFI16 Enhances Chemosensitivity of Breast Cancer Cells by Inhibiting DNA Damage Response
              </a>
            </td>
          <td>
            Na-Lee Ka, G. Lim, Seung-Su Kim, Mi-Ock Lee
          </td>
          <td>2025-10-02</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 BCOR and its homologue BCORL1 are core components of the non-canonical polycomb repressor complex (PRC) 1.1, playing a central role in epigenetic gene silencing and transcriptional repression. Genomic alterations involving BCOR/L1, primarily BCOR internal tandem duplications (ITD) or BCOR/L1 fusions, occur in central nervous system (CNS) tumors with distinct molecular characteristics and poor clinical outcome. Thus far, the impact of these BCOR/L1-alterations on PRC1.1 functionality in CNS tumors remains elusive. Therefore, a better understanding of the underlying biology is essential to develop improved therapies



 By analyzing bulk, single-cell RNA and chromatin immunoprecipitation (ChIP) sequencing data in tissue samples, we explored whether BCOR-ITD or BCOR/L1 fusions share molecular mechanisms and target gene activation patterns in brain tumors. The functionality and targetable vulnerabilities of identified BCOR/L1 targets were confirmed in-vitro and in-vivo using patient-derived and transgenic cell as well as xenograft models.



 BCOR-altered CNS tumors harbor a unique transcriptomic gene signature, indicative of impaired PRC1.1 repressive functions. This dysregulation leads to activation of distinct BCOR-mediated (FGF/FGFR, PDGFR) and PRC1.1 target genes (GATA6). ChIP-Sequencing analyses confirmed recruitment of activating histone marks (H3K27ac), despite binding of KDM2B and mutant BCOR/L1 to the respective target loci. In parallel, PRC1/2-mediated repressive histone modifications, H3K27me3 and H2AK119ub, were absent. Single-cell RNA sequencing further uncovered a distinct tumor cell population enriched for GATA6 and FGF/FGFR expression. Finally, the dependency on receptor tyrosine kinase signaling activity, identified in BCOR/L1-altered tumors, was confirmed by drug screening, revealing FGFR/PDGFR inhibitors as highly active compounds. The multikinase inhibitor nintedanib not only showed anti-cancer activity in a BCORL1-fused PDX model, but also clinical activity in two patients affected by progressive BCOR/L1-driven CNS tumors.



 We deciphered key oncogenic pathways driven by BCOR/BCORL1 alterations in CNS tumors and identified potential targeted therapeutics, which may be leveraged for improved therapy of patients suffering from these aggressive malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75b0af93bf5bc077fbcbc7bfdc9baedb941faee0" target='_blank'>
              P10.23.B UNCOVERING THERAPEUTIC VULNERABILITIES IN PEDIATRIC BCOR/L1-DRIVEN CNS TUMORS THROUGH MULTI-OMICS PROFILING
              </a>
            </td>
          <td>
            D. Lötsch-Gojo, L. Gabler-Pamer, K. Bruckner, D. Kirchhofer, J. LaBelle, A. Beck, S. Wang, D. Al Tukmachi, J. Quack, S. Madlener, D. Senfter, A. Federico, C. Dorfer, T. Eder, F. Grebien, A. Tauziède-Espariat, J. Pahnke, J. Bunt, V. Hovestadt, J. Schueler, C. Haberler, M. Filbin, M. Kool, W. Berger, J. Gojo
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="OBJECTIVES
Antimicrobial resistance (AMR) is an escalating global health concern, driven by multifactorial biological processes not yet fully understood. This study employed network diffusion analysis to dissect the molecular mechanisms driving AMR in Escherichia coli, aiming to identify novel potential drug targets for therapeutic development.


METHODS
A systems biology approach was used to identify genes and biological pathways associated with AMR, by mapping known AMR-related genes from the Comprehensive Antibiotic Resistance Database (CARD) and PointFinder database into the E. coli protein interactome. Through a network diffusion algorithm, several network modules were identified, i.e. genes and pathways, in part already known to be involved in AMR mechanisms. We selected gene candidates for performing an in vitro susceptibility validation test, consisting of 13 knockout mutants against nine different antibiotics.


RESULTS
Compared with the WT E. coli BW25113, the AMR of some mutants showed significant shifts of biological relevance: ΔuhpB (S/I) and ΔmdaB (S/R) against ampicillin, ΔrpmG (I/S) and ΔrplA (I/S) against ciprofloxacin, and ΔrplA (I/S) against streptomycin (S, susceptible; I, intermediate; R, resistant). In other cases, only a significant change in inhibition disc diameter was observed, probably deserving further studies.


CONCLUSIONS
Network diffusion is an effective tool to infer relevant biological insights related to AMR from microbial biological networks. Our results contribute to a better understanding and characterization of AMR in E. coli. Furthermore, the in vitro validated genes could be considered as new putative drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1855f0a6e4ce4deeef7bbc3d3b9fa04f99bcd8" target='_blank'>
              Identification of antimicrobial resistance genes in Escherichia coli through network diffusion.
              </a>
            </td>
          <td>
            Anis Mansouri, F. Durazzi, Muhammad Ahmed Ihsan, Sholeem Griffin, G. Manfreda, V. Valdramidis, F. Pasquali, D. Remondini
          </td>
          <td>2025-11-11</td>
          <td>The Journal of antimicrobial chemotherapy</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dbb38faf877fdc4bb2c148091abcd837c4061f" target='_blank'>
              Pharmacological targeting of HDAC/BET pathway enhances 5-FU efficacy in esophageal squamous cancer cells.
              </a>
            </td>
          <td>
            Xiaqing Xu, Qian Liu, Feifei Yang, Yiqing Zhang, Wanruo Yuan, Wenfang Gao, LiYing Ma, Qi Zhang
          </td>
          <td>2025-11-19</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Breast cancer remains a major global health challenge because of limitations in early detection and therapeutic outcomes. This study employed bulk and single-cell RNA sequencing to investigate SUMOylation-associated molecular networks, aiming to identify prognostic biomarkers and potential therapeutic applications.




 Transcriptomic profiling was performed on 1,445 breast cancer and 113 normal samples to identify differentially expressed genes. Four hub genes,
 NR3C2
 ,
 CDCA8
 ,
 AURKA
 , and
 PLK1
 , were prioritized using machine learning. Consensus clustering stratified patients into molecular subtypes based on the hub gene expression patterns. Differential immune infiltration analysis was used to evaluate 28 immune cell populations between the subtypes. Hub gene-immune cell interactions were visualized using bubble diagrams. Pharmacogenomic sensitivity profiling was performed using subtype-specific drug response data. Single-cell sequencing identified epithelial subclusters enriched for hub genes, and transcription factor networks were analyzed using SCENIC. Pan-cancer validation was performed to assess the oncogenic role of hub genes in 21 malignancies. Statistical significance was determined using the Student’s
 t
 -test (
 p
 < 0.0001).





 Tumor tissues exhibited significant upregulation of
 CDCA8
 ,
 AURKA
 , and
 PLK1
 , whereas
 NR3C2
 was notably downregulated (
 p
 < 0.0001). Consensus clustering identified two distinct molecular subtypes: Subtype1, characterized by
 NR3C2
 upregulation and poorer prognosis, and Subtype2, distinguished by enhanced expression of
 CDCA8
 ,
 AURKA
 , and
 PLK1
 , correlating with favorable outcomes. Notably, PIK3CA mutations were prevalent in Subtype1, whereas
 TP53
 mutations dominated Subtype2. Immune infiltration profiles differed significantly between the two subtypes for most immune cell types. Pharmacogenomic assessments revealed distinct drug sensitivity profiles for each subtype in response to various therapeutic agents. A pan-cancer analysis of the four hub genes demonstrated consistent expression patterns, immune correlations, and prognostic associations across malignancies.




 Our findings reveal that SUMOylation subtypes in breast cancer exhibit distinct prognostic, immunological and pharmacogenomic profiles. These insights may provide candidate biomarkers for future personalized treatment strategies for breast cancer and potentially for other malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b694520cfc56236ad597a9d4d5eb2e2e73faafb" target='_blank'>
              Significance of SUMOylation in breast cancer progression: a comprehensive investigation using single-cell analysis and bioinformatics
              </a>
            </td>
          <td>
            Wenxing He, Zhengkui Sun, Dongmei Li, Tenghua Yu
          </td>
          <td>2025-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Acute myeloid leukemia (AML) is paradigmatic for therapeutic resistance driven by genetic heterogeneity, epigenetic plasticity and microenvironmental protection. Over the past decade, six targeted or pathway-directed small molecules—midostaurin, gilteritinib, quizartinib, ivosidenib, enasidenib, venetoclax and glasdegib—have changed frontline and relapsed/refractory (R/R) practice in genomically defined subgroups or in patients unfit for intensive chemotherapy. Yet primary refractoriness and early relapse remain common, frequently
 via
 adaptive rewiring of apoptotic dependencies, clonal evolution and differentiation resistance. Here we integrate mechanistic insights with clinical evidence to: (i) map resistance biology onto targetable nodes (apoptosis control; signalling kinases; chromatin/lineage programmes; RNA splicing; DNA-damage response; nuclear export; niche adhesion and innate immune evasion); (ii) summarise the clinical trajectory and current limits of approved and emerging small molecules (including menin and LSD1 inhibitors); (iii) propose rules for rational doublets and triplets that are biologically orthogonal yet clinically tolerable; (iv) outline a regulatory timeline for key AML small molecules; and (v) prioritise where drug development should go next, including next-generation BH3 toolkits, clonal-pressure-aware designs, minimal residual disease (MRD)–adapted trials and therapy guided by dynamic functional profiling. The review closes with cross-platform challenges—myelosuppression, infectious risk, resistance monitoring and trial design—and a pragmatic framework for moving beyond incrementalism toward durable control and cure.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f83b4532e166032da2b89eea2b4afa25aa7b8b" target='_blank'>
              Acute myeloid leukemia drug resistance: targetable nodes and the clinical trajectory of small-molecule inhibitors
              </a>
            </td>
          <td>
            Xixi Zhang, Yao Peng, Yina Tian, Shichao Chen, Yijiu Jia, Mengtong Liu, Li Zhang
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Conventional antibiotics are becoming less effective as the global health crisis of multidrug-resistant (MDR) bacterial infections worsens. The function of bacterial DNA repair systems in promoting MDR is a quickly developing paradigm, despite the well-established nature of conventional resistance mechanisms such as efflux pumps and drug-inactivating enzymes. The present understanding of how DNA repair pathways, which are necessary for the stability of the genome, paradoxically promote adaptive mutagenesis and horizontal gene transfer under antibiotic stress is summarized in this review. In important MDR pathogens, we investigate the complex interactions among repair mechanisms, stress reactions, and resistance evolution. The regulatory crosstalk with other bacterial systems and the potential of DNA repair inhibitors as novel therapeutic adjuvants are two examples of significant knowledge gaps that are highlighted. We wrap up by going over potential future directions for focusing on DNA repair to re-sensitize MDR bacteria and prolong the effectiveness of current antibiotics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eab813759b15d787a511efda36aa4a661434f2" target='_blank'>
              DNA Repair Mechanisms in Multi Drug Resistant Bacteria: Impact on Genome Stability and Antibiotic Resistance
              </a>
            </td>
          <td>
            Faiqa Shakeel, Iman Mustafa, Ruba Shahid, Amna Noor, Mishaal Fatima, Kiran Ghafoor, AminaTariq, Samra Zulfiqar
          </td>
          <td>2025-09-30</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Anaplastic lymphoma kinase (ALK) gene fusions are potent oncogenic drivers across diverse malignancies, including non-small cell lung cancer (NSCLC), anaplastic large cell lymphoma (ALCL), breast cancer, and glioma. These fusions lead to constitutive activation of the ALK tyrosine kinase domain (TKD), rendering tumors initially responsive to ALK tyrosine kinase inhibitors (TKIs)—primarily in NSCLC, but also in rare cases such as infant-type hemispheric glioma. However, resistance inevitably emerges, frequently due to acquired single nucleotide variants (SNVs) within the TKD. A significant proportion of these SNVs are classified as variants of uncertain significance (VUSs), highlighting the need for systematic functional annotation to inform precision therapy.



 We employed prime editing to introduce all possible SNVs across the ALK TKD (exons 20-28). High-throughput drug resistance screening was conducted against three generations of ALK TKIs: alectinib (second-generation), lorlatinib (third-generation), and zotizalkib (fourth-generation, previously evaluated in clinical trials). Resistance classification data were further assessed using Kaplan-Meier survival analyses to explore potential prognostic implications.



 A total of 2,953 SNVs (99.2% of all possible variants in exons 20-28) were successfully introduced and profiled. Resistance phenotypes were comprehensively mapped for each variant across the three TKIs. Canonical resistance mutations such as G1202R and L1196M were accurately recapitulated, validating our screening platform. Moreover, several previously uncharacterized VUSs were identified as conferring differential resistance profiles. Structural modeling revealed mechanistic insights into altered TKI binding. These findings aligned with existing clinical and preclinical datasets and expanded the known spectrum of clinically actionable ALK resistance mutations.



 This study presents the most comprehensive functional landscape to date of ALK TKD variants in the context of TKI resistance. Our data provide an invaluable resource for interpreting VUSs, optimizing TKI selection, and advancing personalized treatment in ALK fusion-positive cancers. Furthermore, the approach and insights are broadly applicable to other ALK-rearranged malignancies, including glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6b913fdc6e574ca69abb2b1d6922fc64f2b203" target='_blank'>
              JS04.5.A HIGH RESOLUTION DRUG-RESISTANT MAPPING OF ALK FUSION VARIANTS USING PRIME EDITING
              </a>
            </td>
          <td>
            H. Oh
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Tumor cells often display aberrant chromosome counts, or aneuploidy, which can be exacerbated by increased chromosome missegregation rates in a condition called chromosomal instability (CIN). Despite the prevalence of CIN in solid tumors and its correlation to poor patient prognosis, understanding how CIN status affects cancer aggression remains a contested issue. In this work, we present a novel mathematical model linking missegregation levels and degree of aneuploidy to the underlying fitness landscape, and examine how these fitness landscapes can affect drug efficacy.



 To model how chromosome counts can shift over time, we use a partial differential equation that contains a diffusion term corresponding to chromosome missegregation and a growth term proportional to the relative fitness of a particular chromosomal distribution. Using this framework, a fitness landscape that connects chromosome count to relative fitness can be inferred from a steady-state cellular population. Fluorescence in situ hybridization (FISH) was used to track chromosome counts in both CIN and non-CIN cell lines over the course of 2 months, with data collected for chromosome 10 individually and for the group of chromosomes 1, 5, and 19. Experimental results were then analyzed using our mathematical framework to determine fitness landscapes for both CIN and non-CIN cells.



 A partial differential equation was developed to model how chromosome counts change over time based on a diffusion-like term corresponding to lagging chromosome rate and a growth rate based on a chromosome count’s contribution to overall cellular fitness. The CIN lines exhibited chromosome count distributions that were statistically closer to normal distributions than non-CIN lines. When connected to our mathematical model, these distributions imply that the CIN lines have a flatter fitness landscape than their non-CIN counterparts. This finding reinforces the notion that CIN lines are tolerant of highly varying degrees of aneuploidy.



 Our work suggests an underlying mechanism for the clinical observation that CIN tumors are more resilient to treatment with taxanes and other microtubule or mitotic targeting drugs. The flatter fitness landscapes associated with CIN provide an explanation for why these drugs are less effective, since increasing or decreasing the number of chromosomes in a CIN line does not lead to a relatively large change in fitness.



 Kelly Brock, Marianna Kleyman, Madison Stoddard, T. Ryan Gregory, Andrew Chen, Arijit Chakravarty. Fitness landscape modeling indicates decreased effectiveness of microtubule assembly targeting drugs in tumors demonstrating chromosomal instability (CIN) [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1879dcd4be6c24e75a32134afbe36fcff096e0" target='_blank'>
              Abstract C006: Fitness landscape modeling indicates decreased effectiveness of microtubule assembly targeting drugs in tumors demonstrating chromosomal instability (CIN)
              </a>
            </td>
          <td>
            Kelly Brock, Marianna Kleyman, M. Stoddard, T. R. Gregory, Andrew Chen, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Lactylation, a recently identified post‐translational modification that utilizes lactic acidas a substrate, has emerged as an important regulator of gene expression andprotein function. Since its discovery in 2019, lactylation has beenincreasingly recognized for its roles in cancer biology and treatment response. Main text Lactylationis strongly associated with tumor progression and malignancy, underscoring itspotential as a therapeutic target. Recent studies also link lactylation tocancer treatment resistance, suggesting that modulating this modification couldenhance therapeutic efficacy. As treatment resistance remains a major clinicalchallenge in oncology, accumulating evidence indicates that dysregulatedlactylation contributes to resistance across chemotherapy, immunotherapy, targeted therapy, and radiotherapy. Preclinical and clinical research has begunto delineate the molecular pathways through which lactylation shapes theseresistance processes, and experimental approaches targeting lactylation arebeing explored to restore therapeutic sensitivity. Conclusion This review systematically summarizes the mechanisms of lactylation and its roles intreatment resistance, highlighting the interplay between lactylation andtherapeutic response. We discuss current and emerging strategies that targetlactylation, providing a foundation for future therapeutic development aimed atovercoming resistance and improving cancer treatment outcomes. Key points Lactylation links glycolysis to tumor progression and therapeutic response. Modulating lactylation writers and erasers restores treatment sensitivity. Aberrant lactylation drives resistance tomultiple cancer therapies. Crosstalk with other post‐translational modifications suggests novel combination strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bade220278a5c32855cdc72f3721e07a31a4d8d" target='_blank'>
              Lactylation in cancer: Advances and opportunities for treatment resistance
              </a>
            </td>
          <td>
            Keke Xu, Yiyi Shou, Ruiqi Liu, Hao Xiong, Xiaomeng Dai, Xuanwen Bao, Xiaoyan Chen, Luanluan Huang, Hailong Sheng, Haibo Zhang, Yanwei Lu
          </td>
          <td>2025-09-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer genome sequencing efforts have revealed hundreds of genes under positive selection, many of which are now being developed as therapeutic targets. However, positively selected mutations also populate our aging tissues in the absence of cancer. For most mutations, it is currently unknown whether they are recurrently found in cancer genomes because they cause cancer or because they expand during normal tissue evolution and are passively inherited. Here, we develop a mathematical and statistical framework that distinguishes these two factors. We discover – across thousands of cancer and normal tissue genomes – that mutations that most strongly increase cancer risk are enriched in younger patients’ cancers, whereas mutations that are positively selected in normal tissue without causing cancer are enriched in older patients. Focusing on a particularly data-rich cancer type, acute myeloid leukemia, we show that genetic differences between young- and adult-onset cancers can largely be explained by the cumulative effects of normal tissue evolution, contradicting the long-standing notion that childhood cancers require a distinct set of causal mutations. Our framework establishes patient age as a powerful resource for clarifying whether positively selected mutations in cancer genomes are truly disease-promoting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56a3d728c459f75967661fea1d9baa92b4ef2796" target='_blank'>
              Age distinguishes selection from causation in cancer genomes
              </a>
            </td>
          <td>
            David Cheek, Martin Blohmer, Martin A. Nowak, Tibor Antal, Kamila Naxerova
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The estimation of the in vitro potency of anticancer agents against tumor cell lines in tissue culture is known to be fraught with a high degree of variability. While quality control of experimental conditions has been demonstrated by others to reduce this variability, other sources of variability remain even after standardization of experimental conditions. In this poster, we assess two potential mechanisms for this residual variability: phenotypic stochasticity in response and heritable resistance to treatment.



 We developed three agent-based mathematical models representing different potential scenarios for cellular responses to anticancer therapy: (1) all cells have an equal probability of survival under treatment (phenotypic stochasticity) and future division; (2) sensitive cells deterministically die under treatment while resistant cells survive and may divide to produce resistant offspring (heritable resistance); (3) sensitive subclones have a greater probability of death under treatment compared to resistant subclones (both mechanisms). We implemented these models in MATLAB, running simulated in vitro experiments consisting of large (106) or small (500 or 5000) populations of virtual cells under treatment. The final size of these virtual cell populations serves as the readout of the virtual experiments. To assess the effect of phenotypic stochasticity and heritable resistance on variation in experimental outcomes, we simulated these experiments 10,000 times and report the coefficient of variation (CV).



 Variability in response to treatment was low under most simulated conditions. In the first model simulating phenotypic stochasticity alone, high coefficients of variation in assay outcome were observed only when the probability of survival under treatment was less than 10%. In simulations of deterministic heritable survival under treatment, the coefficient of variation was generally low. However, simulations incorporating both mechanisms exhibited substantial variability in experimental outcome, consistent with the observed phenomenon.



 Stochastic cellular response to treatment or heritable binary resistance alone are insufficient explanations for the variation in in vitro experimental results. On the other hand, the high degree of variation in experimental outcomes is consistent with a combination of heritable resistance and phenotypic stochasticity. Our findings are consistent with a role for Darwinian evolution in the response to anticancer treatment for cancer cells in tissue culture.



 Alexander L. Young, Liam Noronha, Madison Stoddard, T Ryan. Gregory, Arijit Chakravarty. Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of in vitro potency of anticancer agents [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C073.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a885f4ecedc46db5c7868acd715fccb8dde4f83" target='_blank'>
              Abstract C073: Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of
 in vitro
 potency of anticancer agents
              </a>
            </td>
          <td>
            Alexander L. Young, Liam Noronha, M. Stoddard, T. R. Gregory, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly aggressive malignancy with poor prognosis and significant molecular heterogeneity. This study investigates the role of tumor-specific enhancers in metabolic reprogramming, focusing on glucose-6-phosphate dehydrogenase (G6PD) and the pentose phosphate pathway (PPP). Using native elongating transcript-cap analysis of gene expression and single-cell RNA sequencing, tumor-specific enhancers driving G6PD overexpression were identified in ICC tumor epithelial cells. Functional assays demonstrated that G6PD promotes tumor proliferation by enhancing PPP activity and maintaining redox homeostasis, which provides NADPH to counter oxidative stress. Enhancer knockdown disrupted G6PD expression and PPP activity, increasing reactive oxygen species levels and reducing the NADPH/NADP+ ratio. These metabolic changes impaired tumor cell proliferation and sensitized ICC cells to cisplatin, emphasizing the dual therapeutic potential of targeting G6PD to inhibit tumor growth and overcome chemoresistance. Survival analyses showed that high G6PD expression correlates strongly with poor overall survival in ICC patients. While previous studies have recognized the roles of G6PD and PPP in cancer metabolism, this study uniquely links enhancer-mediated regulation to these processes in ICC, offering novel insights into epigenetic mechanisms driving metabolic reprogramming. Moreover, the findings highlight tumor-specific enhancers as critical epigenetic drivers of ICC progression, with potential as therapeutic targets. Future research should explore the integration of enhancer profiling into precision medicine frameworks and the development of novel enhancer-targeting strategies. These efforts could uncover additional metabolic vulnerabilities and provide effective treatments for this highly aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f09ce5004e00e224dd82b9d53133eeef55290e" target='_blank'>
              Epigenetic Regulation of G6PD Drives Metabolic Reprogramming in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Yusuke Nakano, Miwa Tanaka, Takeharu Sakamoto, M. Hashimoto, Taro Tobo, Hideyuki Saito, Tadashi Abe, Tomohiko Ikehara, Takashi Ofuchi, Koto Kawata, Takaaki Masuda, T. Ogino, Mamoru Uemura, H. Eguchi, Y. Doki, K. Mimori
          </td>
          <td>2025-10-07</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Highlights What are the main findings? ALT functions as a context-dependent double-edged sword: it drives aggressiveness in rapidly proliferating cancers but imposes replicative and metabolic stress in slow-growing, immune-cold tumors such as glioblastoma and chondrosarcoma. Loss of ATRX/DAXX and ALT activation creates unique DNA repair vulnerabilities, including hypersensitivity to ATR and PARP inhibition, opening avenues for targeted therapy. What is the implication of the main finding? Refines prognostic interpretation of ALT positivity—transforming it from a uniformly negative marker into a tumor-type-specific indicator of clinical outcome. Positions ALT as a precision oncology target, where integrated biomarkers (C-circle as say, APB detection, ATRX/DAXX mutation profiling) enable patient stratification and therapeutic decision-making. Abstract Telomere maintenance enables unlimited cell proliferation by counteracting telomere erosion. While the majority of tumors activate telomerase, a significant subset—approximately 10–15%—utilizes alternative lengthening of telomeres (ALT), a recombination-based mechanism. ALT-positive cancers are classically associated with genomic instability, anaphase bridges, chromosomal rearrangements, and resistance to DNA-damaging therapies. This process is closely associated with genetic instability, which contributes to chromosomal rearrangements and tumor evolution. Consequently, ALT has traditionally been considered an adverse prognostic marker in aggressive malignancies such as osteosarcoma, pancreatic neuroendocrine tumors, and high-grade sarcomas. Paradoxically, recent evidence demonstrates that ALT positivity correlates with improved survival in glioblastoma (GBM) and chondrosarcoma, two tumor types that have historically been regarded as immune-cold and therapeutically intractable. This favorable outcome likely reflects a convergence of factors, including replication stress and DNA damage that impose a fitness cost in slow-growing or metabolically constrained tumors. Loss of ATRX/DAXX, while enabling ALT, further amplifies chromatin fragility, and ALT-mediated instability may paradoxically enhance immunogenicity within immune-quiescent microenvironments. Moreover, ALT-positive cells exhibit unique therapeutic vulnerabilities, particularly to ATR and PARP inhibitors. Together, these observations support a context-dependent model in which ALT functions as a double-edged sword, acting as a driver of malignant aggressiveness in rapidly proliferating cancers while serving as a relative liability in slower-growing, immune-cold tumors. Understanding this duality not only refines prognostic stratification but also opens opportunities for precision oncology. By integrating ALT-specific biomarkers into clinical workflows and exploiting ALT-related DNA repair dependencies, clinicians may transform a once uniformly negative prognostic factor into an actionable therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6ed85bfa1a7b22ce5afc4a049e72d82de6bf4" target='_blank'>
              Alternative Lengthening of Telomeres: A Prognostic Paradox in Cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Advanced prostate cancer (PrCa) remains a leading cause of cancer-related death among men due to disease progression in nearly all patients on standard of care therapy targeting the androgen receptor. An important mechanism driving therapeutic resistance is lineage plasticity, which enables PrCa cells to reprogram into lineage variants no longer dependent on androgen receptor signaling. As inhibitors of the histone methyltransferase EZH2 are being evaluated clinically for the treatment of advanced PrCa, we investigated here how EZH2 affects PrCa lineage plasticity. Data from genetically engineered mice and human clinical samples demonstrated that genetic or pharmacological suppression of EZH2 altered chromatin to expand active transcription factor programs. These changes in gene expression during PrCa progression increased the diversity of PrCa lineage variants that arose. EZH2 suppression did not inhibit disease progression or therapeutic resistance in this context. These findings advance the current understanding of PrCa lineage plasticity and suggest EZH2 inhibitors may be less effective in treating PrCa prone to lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee66b00f883047cd2ed8c292d9f0f040e78b6ffe" target='_blank'>
              EZH2 Suppression Diversifies Prostate Cancer Lineage Variant Evolution and Lacks Efficacy in Inhibiting Disease Progression.
              </a>
            </td>
          <td>
            Justine J Jacobi, K. Wadosky, Neha Jaiswal, Xiaojing Zhang, Yanqing Wang, Prashant K Singh, Jie Wang, Eduardo Cortes Gomez, Jianmin Wang, Bo Xu, Mark D Long, Paloma Cejas, Shweta Kukreja, H. Long, David W. Goodrich
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 High-grade gliomas (HGGs) are among the leading causes of cancer-related deaths due to a lack of effective treatments. Common genetic mutations observed in these tumors include alterations in receptor tyrosine kinases (RTKs), such as amplification, mutation, and gene fusions. However, directly targeting RTK variants with tyrosine kinase inhibitors yields only partial responses in patients, due in part to the molecular and cellular heterogeneity within these tumors. To address this challenge, we developed experimental model systems that recapitulate inter- and intra-tumoral heterogeneity to facilitate therapeutic discovery. For this purpose, we created human brain organoid-tumor models of HGG. These organoid models provide an improved ex vivo system for examining how the neural microenvironment shapes cellular and molecular heterogeneity among tumor cells over time. We tested these models for responses to small molecule inhibitors and targeted therapies, including approaches aimed at effector pathways that act downstream or in parallel to RTKs. Candidate targets included the YAP and TAZ transcription factors, which cooperate with TEAD co-factors to promote target gene expression, orchestrate stem cell fate decisions, and drive proliferation during neural development. We examined responses in both tumor and non-tumor cells in the neural microenvironment following pharmacological and genetic inhibition of YAP/TAZ-TEAD, using single-cell RNA sequencing and immunofluorescence to visualize cell-specific responses. We found that YAP/TAZ-TEAD function is required for the growth and progression of HGGs with oncogenic RTK variants. Treatment with small molecule inhibitors targeting YAP/TAZ and TEAD reduced proliferation, inhibited HGG cell self-renewal, and induced both differentiation and apoptosis, mimicking the effects of genetic inhibition, while also reducing intra-tumoral transcriptional heterogeneity. Multi-omic profiling of tumor subpopulations resistant to YAP/TAZ-TEAD inhibitors identified additional targets for combination therapies. Preclinical studies in primary HGG cultures and mouse genetic models corroborated these findings and support our ongoing clinical trial of the YAP/TAZ-TEAD inhibitor verteporfin. Together, our results demonstrate that YAP/TAZ-TEAD inhibition represents a promising therapeutic strategy for HGGs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd6bbc21728d2802bb2ef20d58a583437af6bb7" target='_blank'>
              DDDR-84. The YAP/TAZ-TEAD axis promotes tumor stem cell functionality and cellular heterogeneity in high-grade gliomas
              </a>
            </td>
          <td>
            R. Read, J. L. J. Macias, Mohamed Ishan, Se-yeong Oh, Zhengang Hu, K. Yin, Anson Sing, Steven A Sloan
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cellular senescence and immunosenescence encompass critical molecular pathways that govern aging and age-related pathologies. Central to cellular senescence are DNA damage response activation, telomere attrition, chromatin remodeling, metabolic reprogramming, and cytoplasmic DNA sensing via cGAS-STING signaling, which collectively drive cell cycle arrest and the pro-inflammatory senescence-associated secretory phenotype (SASP). Immunosenescence involves progressive deterioration of immune cell function characterized by depleted naive lymphocytes, accumulation of dysfunctional senescent immune cells, and chronic inflammation (inflammaging), creating a feedback loop that exacerbates tissue degeneration and systemic aging. Model organisms such as mice and killifish have been indispensable for unraveling these mechanisms, enabling genetic and functional studies that illuminate senescence dynamics and immune clearance processes. Future research, empowered by multi-omics, single cell sequencing, and artificial intelligence, promises deeper dissection of senescence heterogeneity and tissue-specific pathways, offering biomarkers and therapeutic targets with unprecedented precision. Therapeutic strategies aiming to selectively eliminate or modulate senescent cells through senolytics, senomorphics, and immunomodulatory approaches hold promise to extend health span and ameliorate chronic diseases. However, challenges including senescent cell heterogeneity, context-dependent functions, and biomarker limitations necessitate individualized and careful translation of findings into clinical therapies. Continued interdisciplinary efforts integrating molecular biology, systems medicine, and clinical research will be pivotal in harnessing the full potential of senescence targeting for healthy aging and transformative disease management. This review was conducted to comprehensively compile and discuss the intricate molecular mechanisms underlying cellular senescence and immunosenescence, which are critical processes involved in aging and age-related diseases. The aim of this review article is to comprehensively elucidate the molecular mechanisms underlying cellular senescence and immunosenescence, integrating insights gained from model organism research and emerging signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9da5a504721107d0fa7edd2d6a3796a679c3a349" target='_blank'>
              Molecular Mechanisms of Cellular Senescence and Immuno-Senescence: Insights from Model Organisms and Emerging Pathways
              </a>
            </td>
          <td>
            Alebachew Molla
          </td>
          <td>2025-10-30</td>
          <td>American Journal of Biomedical and Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase inhibitors (PARPis) have improved the prognosis of patients with high-grade serous ovarian carcinoma (HGSOC). However, PARPis are not effective for all HGSOC patients. Spatial transcriptomics is a powerful tool for characterizing the tumor microenvironment. We used Visium to analyze eight tumor samples from HGSOC patients with clinical information on PARPi sensitivity. Two complementary analyses were performed: an integrated analysis across all samples, without considering spatial information, and an analysis with spatial information within each sample. Both approaches indicated that midkine (MDK) signaling is involved in PARPi resistance. Furthermore, we identified receptors that enhance MDK signaling in cancer cells. To assess the generalizability of this finding, we deconvolved bulk RNA-sequencing data using single-cell RNA-sequencing data as a reference to examine the relationship between receptor expression levels and overall survival (OS). This analysis revealed that high SDC4 expression in cancer cells is associated with poor OS in HGSOC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6e6aeb1b5b6668556eb27188c5496dc383c124" target='_blank'>
              Exploring drug resistance via intercellular crosstalk using spatial transcriptomics in high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Hironori Suzuki, Kosuke Yoshida, Akira Yokoi, Kazuhiro Suzuki, Yuta Hirano, Masami Kitagawa, Eri Asano-Inami, M. Yoshihara, Yukari Nagao, S. Tamauchi, N. Yoshikawa, Hiroaki Kajiyama, Yusuke Yamamoto
          </td>
          <td>2025-11-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Polycomb repressive complex 2 (PRC2) silences transcription by tri-methylating histone H3 at lysine 27, to regulate cell growth and differentiation. We demonstrated that PRC2 and androgen receptor (AR) inhibitors promote durable antitumor activity in prostate cancer models by suppressing the evolution of lineage escape pathways, driving cell differentiation, leading to reprogrammed cells that depend on oncogenic AR signaling. Recent work has further implicated PRC2 in enabling cell plasticity in other cancers. For example, in triple-negative breast cancer, combining EZH2 with AKT inhibitors induced differentiation, triggering cell death through mechanisms akin to mammary gland involution. Similarly, in KRAS-mutant colorectal cancer (CRC) models, concurrent PRC2 and MAPK pathway inhibition induced cell differentiation and impaired tumor growth. KRAS inhibitor resistance can arise by non-genetic reprogramming and transcriptional plasticity. Given that PRC2 inhibition, as shown with ORIC-944 in prostate cancer models, may overcome treatment resistance by restricting access to plasticity program genes, we investigated the impact of combined KRAS and PRC2 inhibition in KRAS-mutant CRC and non-small cell lung cancer (NSCLC).



 ORIC-944 is a next-generation, potent, highly selective, orally bioavailable small molecule inhibitor of PRC2 that allosterically targets the EED subunit, with potential best-in-class drug properties including superior half-life and reduced drug-drug interaction liabilities. In vitro and in vivo studies across KRAS-mutant NSCLC and CRC models were used to evaluate ORIC-944 and KRAS inhibition. The mechanism of PRC2 inhibition was profiled using RNA- and ATAC-sequencing.



 Evaluation of combined KRAS inhibition and PRC2 inhibition with ORIC-944 was performed in KRAS-mutant cell line xenograft models in vivo. The combination of adagrasib, a KRAS G12C inhibitor, and ORIC-944 resulted in greater antitumor efficacy, more tumor regressions, and significantly smaller tumors, in both CRC and NSCLC models. Moreover, combining ORIC-944 with KRAS inhibition delayed KRAS inhibitor relapse and prolonged progression-free survival (PFS) in both models. In vivo NSCLC findings confirmed in vitro, where ORIC-944 combined with KRAS inhibition further reduced proliferation and viability effects vs. single agent KRAS inhibition. Gene expression analysis confirmed strong PRC2 inhibition with ORIC-944 across xenograft studies and reduced cell cycle and G2M signatures were the most prominent common feature of the combination. In the CRC model, ORIC-944 induced enrichment of secretory lineage conversion, which was sustained with combination treatment.



 Combining ORIC-944 with KRAS inhibition significantly improved efficacy and PFS in NSCLC and CRC models, demonstrating that PRC2 inhibition can deepen and extend KRAS inhibitor responses to prevent or delay resistance to KRAS inhibition. ORIC-944 is under clinical evaluation in a global Phase 1b study (NCT05413421).



 Stephanie W. Ni, Natalie Yuen, Livia Ulicna, Denise F. Reyes, Michal Pawlak, Frank L. Duong, Jason E. Long, Aleksandr Pankov, Lori S. Friedman, Anneleen Daemen, Melissa R. Junttila. PRC2 inhibition enhances KRAS inhibitor response to delay treatment relapse in KRAS-mutant preclinical lung and colorectal cancer models [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/508771855cf3f0d4a98ae63f8edf452520c62271" target='_blank'>
              Abstract A048: PRC2 inhibition enhances KRAS inhibitor response to delay treatment relapse in KRAS-mutant preclinical lung and colorectal cancer models
              </a>
            </td>
          <td>
            Stephanie W Ni, Natalie Yuen, Livia Ulicna, Denise F. Reyes, Michal Pawlak, Frank L. Duong, Jason E Long, A. Pankov, Lori S Friedman, Anneleen Daemen, M. Junttila
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 To identify novel drivers of pancreatic ductal adenocarcinoma (PDAC) malignancy, we employed a systems biology approach called regulatory network analysis, which accurately infers the activity of transcription factors and other regulatory proteins based on the integrated expression of their positive and negative target genes. This highly validated approach enables the identification of the most hyper-activated and hyper-repressed regulatory proteins (“master regulators”) that drive phenotypic distinctions. We applied this technique to a set of 200 laser capture microdissected human PDAC samples as well as 45 low-grade precursors for which we had matched histopathological, clinical, and epidemiological annotation. We identified the master regulators associated with 4 malignancy phenotypes: tumor initiation, tumor progression, survival post resection, and association with KRAS activity. Integrating across these phenotypes, the top master regulator was BMAL2, a member of the PAS family. Although BMAL2 is classically associated to the circadian rhythm, gene set enrichment analysis highlighted a potential role in hypoxia response, a key phenotype in PDAC evolution. We previously demonstrated that PDAC in humans and in the genetically engineered “KPC” mouse model is hypovascularized and hypoperfused, and we have since shown that KPC tumors are profoundly hypoxic. Sequence conservation analysis of BMAL2 demonstrated close analogy to hypoxia-related genes, its activity is induced in response to it and inhibited following treatment with multiple RAF, MEK, and ERK inhibitors, validating its computationally inferred association with RAS activity. Strikingly, knockout of BMAL2 in human PDAC cells led to defects in viability and invasion in the setting of hypoxia. Metabolically, loss of BMAL2 impairs the ability to induce glycolysis - key metabolic pathway unregulated during hypoxia response - upon exposure to severe hypoxia. Surprisingly, knockout of BMAL2 in multiple PDAC cell lines led to a complete loss of HIF1A stabilization in response to hypoxia. By contrast, HIF2A was further upregulated under hypoxia in the setting BMAL2 loss. Additionally, metabolomic studies showed an overrepresentation of glycolysis metabolites and related metabolic pathways in hypoxic control conditions but to a lesser degree in BMAL2 knockout cells. Finally, preliminary in vivo experiments in cell line derived xenograft models displayed lower tumor growth rates in BMAL2 knock out cells. Together, our data indicates that BMAL2 loss blunts hypoxia response providing also a potential answer to the long-standing question of how these proteins are differentially regulated. Certainly, a connection between RAS signaling and glycolysis has long been apparent, but our findings provide a novel mechanistic understanding of the relationship between RAS and hypoxic metabolism.



 Alvaro Curiel Garcia, Carlo H. Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Steven A. Sastra, Anthony Andren, Li Zhang, Tessa Le Large, Irina Sagalovsky, Daniel R. Ross, Vilma Rosario, Kate Lu, Ethan Ferraiuolo, Nicholas Spinosa, Winston Wong, Kaitlin Shaw, John A. Chabot, Jeanine Genkinger, Hanina Hibshoosh, Gulam A. Manji, Alina C. Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, Costas A. Lyssiotis, Kenneth P. Olive. Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b200421d854e96ea48e923482c3749c3e10676" target='_blank'>
              Abstract A047: Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer
              </a>
            </td>
          <td>
            Á. C. García, Carlo Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Stephen A. Sastra, A. Andren, Li Zhang, Tessa Y. Le Large, Irina Sagalovsky, Daniel R. Ross, V. Rosario, K. Lu, Ethan Ferraiuolo, Nicholas Spinosa, W. Wong, K. Shaw, John A. Chabot, Jeanine Genkinger, H. Hibshoosh, G. Manji, Alina C Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, C. Lyssiotis, K. Olive
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) remains a leading cause of cancer mortality, and therapeutic failure is often driven by resistance to chemotherapy. Cdc42-a is a small Rho GTPase that regulates cell survival, cytoskeletal dynamics, and stress responses and has emerged as a plausible target for reversing chemoresistance in CRC. 
Materials and methods: Non-resistant (NR) and doxorubicin-resistant (DOX-R) LoVo cells were generated by stepwise doxorubicin escalation and validated by IC50 determination (NR: 0.1398 µM; DOX-R: 0.5281 µM). Cells were treated for 48 h with CASIN (a Cdc42 inhibitor; 5 µM), doxorubicin (at each line’s IC50), or the combination. Cell viability was measured by Alamar Blue. Cdc42 activity was quantified using a G-LISA assay, and intracellular doxorubicin was assessed fluorometrically. 
Results: CASIN alone did not significantly alter viability in either NR or DOX-R cells. In contrast, CASIN plus doxorubicin markedly reduced viability in both models, lowering DOX-R cell survival to <10% of control-consistent with restored drug sensitivity. Basal Cdc42 activity was comparable between NR and DOX-R cells, indicating resistance was not driven by elevated Cdc42 activation. CASIN did not increase intracellular doxorubicin, suggesting the chemosensitizing effect occurs independently of altered drug uptake or efflux. 
Conclusion: Pharmacologic Cdc42 inhibition with CASIN substantially restores doxorubicin responsiveness in resistant CRC cells through mechanisms unrelated to drug transport. These findings support Cdc42 targeting as a promising approach to overcome chemoresistance in CRC. 
Acknowledgements: This research has been funded by the Science Committee of the Ministry of Science and Higher Education of the Republic of Kazakhstan (Grant No. AP26100973) and Nazarbayev University, Collaborative Research Project (CRP) Grant No. 211123CRP1611. 
Key words: colorectal cancer, chemoresistance, cdc42, CASIN, Doxorubicin, IC50 
References: 

Zhylkaidarova, A.; Kaidarova, D.; Batyrbekov, K.; Shatkovskaya, O.; Begimbetova, D. Trends of Colorectal Cancer Prevalence in Kazakhstan Related to Screening. Clin Endosc 2021, 54, 32-37, doi:10.5946/ce.2019.198. 
Sakamori, R. et al. (2014) ‘CDC42 inhibition suppresses progression of incipient intestinal tumors’, Cancer Research, 74(19), pp. 5480–5492. doi:10.1158/0008- 5472.can-14-0267. 
Tacar, O., Sriamornsak, P., & Dass, C. R. (2013). Doxorubicin: an update on anticancer molecular action, toxicity and novel drug delivery systems. Journal of Pharmacy and Pharmacology, 65(2), 157-170. doi:10.1111/j.2042-7158.2012.01567.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0ae92ee96281181ad1927ebdc0effaf0a1160ad" target='_blank'>
              Restoring drug sensitivity in colorectal cancer through Cdc42 inhibition
              </a>
            </td>
          <td>
            Aislu Yermekova, Aibanu Sailauova, B. Umbayev, Altyn Zhuraliyeva
          </td>
          <td>2025-11-14</td>
          <td>Eurasian Journal of Applied Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Targeted therapy has been established as a therapeutic option for the treatment of metastatic melanoma. Despite initially being very efficient, many tumors develop resistance to targeted therapy, leading to its failure. We previously demonstrated that the neural crest (NC)-associated gene ERRFI1 is highly expressed in metastatic melanoma and correlates with a bad prognosis. Here, we show that the expression of ERRFI1 was upregulated in melanoma and negatively correlated with the expression of melanocytic differentiation markers, such as MITF and TYR. Downregulation of ERRFI1 with the help of siRNA increased the susceptibility of melanoma cells toward BRAF inhibition (BRAFi) and resensitized BRAFi-resistant melanoma cells to BRAFi. Mass spectrometry-based proteomic analysis revealed that ERRFI1 silencing diminished the activation of the mitogen-activated protein kinase (MAPK) and AKT signaling pathways, which usually contribute to drug resistance. Furthermore, we show that miR-200c targeted the 3'UTR of ERRFI1 and reduced its expression, resulting in the resensitization of BRAFi-resistant melanoma cells to BRAFi. Our study results suggest that ERRFI1 could be a potential therapeutic target for the treatment of metastatic melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a645774843921b1750028307066a9681c4de577" target='_blank'>
              The neural crest-associated gene ERRFI1 is involved in melanoma progression and resistance toward targeted therapy.
              </a>
            </td>
          <td>
            Nina Wang, Qian Sun, Daniel Novák, Lei Zhu, Juliane Poelchen, Tamara Steinfass, Yiman Wang, V. Umansky, Jochen Utikal
          </td>
          <td>2025-10-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Distinguishing the rare"driver"mutations that fuel cancer progression from the vast background of"passenger"mutations in the non-coding genome is a fundamental challenge in cancer biology. A primary mechanism that non-coding driver mutations contribute to cancer is by affecting gene expression, potentially from millions of nucleotides away. However, existing predictors of gene expression from mutations are unable to simultaneously handle interactions spanning millions of base pairs, the extreme sparsity of somatic mutations, and generalize to unseen genes. To overcome these limitations, we introduce GenVarFormer (GVF), a novel transformer-based architecture designed to learn mutation representations and their impact on gene expression. GVF efficiently predicts the effect of mutations up to 8 million base pairs away from a gene by only considering mutations and their local DNA context, while omitting the vast intermediate sequence. Using data from 864 breast cancer samples from The Cancer Genome Atlas, we demonstrate that GVF predicts gene expression with 26-fold higher correlation across samples than current models. In addition, GVF is the first model of its kind to generalize to unseen genes and samples simultaneously. Finally, we find that GVF patient embeddings are more informative than ground-truth gene expression for predicting overall patient survival in the most prevalent breast cancer subtype, luminal A. GVF embeddings and gene expression yielded concordance indices of $0.706^{\pm0.136}$ and $0.573^{\pm0.234}$, respectively. Our work establishes a new state-of-the-art for modeling the functional impact of non-coding mutations in cancer and provides a powerful new tool for identifying potential driver events and prognostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b9d5175cec408294f8a32b0ff261d25b5b9f7e" target='_blank'>
              GenVarFormer: Predicting gene expression from long-range mutations in cancer
              </a>
            </td>
          <td>
            David Laub, Ethan Armand, Arda Pekis, Zekai Chen, Irsyad Adam, Shaun Porwal, Bing Ren, Kevin Brown, Hannah Carter
          </td>
          <td>2025-09-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, defined by the absence of estrogen receptor (ER), progesterone receptor (PR), and HER2 amplification. This molecular profile limits the use of targeted endocrine or HER2-directed therapies, leaving patients reliant on conventional chemotherapeutics such as doxorubicin. However, the frequent development of chemoresistance underscores the urgent need for new therapeutic strategies. Long noncoding RNAs (lncRNAs), a class of regulatory transcripts with emerging roles in cancer biology, remain largely unexplored in the context of TNBC pathogenesis and drug resistance. This study aimed to systematically identify lncRNAs that drive tumor survival and chemoresistance to uncover novel therapeutic targets and enhance the efficacy of doxorubicin-based treatment. We employed a high-throughput, dual-excision CRISPR-Cas9 screening system targeting the transcription start sites (TSSs) of approximately 1,000 lncRNAs. Functional screens were performed in both 2D monolayer and 3D organoid culture systems, with and without doxorubicin exposure, to closely mimic the tumor microenvironment and treatment stress. Integrative analysis of the screen data identified 22 high-confidence oncogenic lncRNAs essential for TNBC cell survival and resistance to doxorubicin, particularly under 3D conditions that recapitulate in vivo extracellular matrix architecture. Two of these lncRNAs were significantly overexpressed in patient-derived TNBC datasets, highlighting their clinical relevance. Functional validation using antisense oligonucleotides (ASOs) prioritized two top candidates—designated Candidate 100 and Candidate 305—whose knockdown resulted in marked inhibition of TNBC cell growth and viability in both 2D and 3D models. These effects were supported by impaired colony formation, reduced organoid growth, and AO/EtBr live-dead staining assays. This study presents the first comprehensive lncRNA essentiality map in TNBC under 3D culture conditions and reveals the context-dependent roles of lncRNAs in chemoresistance. Our findings establish Candidates 100 and 305 as actionable therapeutic targets, emphasizing the potential of RNA-based precision medicine approaches to overcome doxorubicin resistance in TNBC.



 Vishnubalaji Radhakrishnan, Ramesh Elango, Nehad M. Alajez. Systematic functional profiling of long noncoding RNAs identifies therapeutic vulnerabilities in doxorubicin-resistant triple-negative breast cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C101.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b88e1c44c29ba0bde4cc2f646df4f72b05fb33" target='_blank'>
              Abstract C101: Systematic functional profiling of long noncoding RNAs identifies therapeutic vulnerabilities in doxorubicin-resistant triple-negative breast cancer
              </a>
            </td>
          <td>
            Vishnubalaji Radhakrishnan, R. Elango, Nehad M. Alajez
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chromosome instability (CIN) and Genetic instability are hallmark features of malignant tumor cells, contributing to tumour progression, drug resistance and metastasis. KIF18A, a member of the kinesin superfamily of motor proteins, is essential for preserving the structural integrity of the bipolar spindle apparatus during cell division. Recent studies have demonstrated that genetic depletion of KIF18A impacts the chromosome alignment, elongation of mitotic spindles, and activates spindle assembly checkpoint in chromosomal instable tumour cells ultimately leading to cell death, while having negligible impact towards the survival of normal cells. These findings suggest that targeting KIF18A represents a promising therapeutic approach for selectively inhibiting the growth of chromosomally instable tumours, while sparing the normal division of euploid cells. We have designed novel macrocyclic KIF18A inhibitors based on our structural understanding from Cryo-EM structures of initial acyclic binders. Structure guided SAR led to the identification of highly potent and selective compounds with optimized properties. Lead compounds exhibited significant inhibition of KIF18A ATPase activity and demonstrated robust cellular anti-proliferative potency in chromosomally instable (CIN-high) high-grade serous ovarian carcinoma cells (HGSOC, OVCAR-3) while retaining a good selectivity over the CIN-low MCF7 cells. This in vitro potency was supported by a significant and dose-dependent upregulation of phospho-Histone H3 and gamma-H2AX, consistent with the induction of mitotic arrest and DNA damage. Further, the lead compounds exhibited encouraging drug-like properties, including acceptable in vitro ADMET profile and excellent in vivo exposure with low clearance resulting in good oral bioavailability in rodents. Additionally, these compounds showed significant anti-tumor efficacy that was associated with robust induction of phospho-histone H3 in the OVCAR3 CDX model. The macrocyclic lead compound is being extensively evaluated in advanced in vitro and in vivo assays to support further development.



 Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh Vishwanath. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, Samiulla D S, Amith A, Aravind A B, Girish C. Daginakatte, Kavitha Nellore, Kishore Narayanan, Chandrasekhar Abbineni, Murali Ramachandra, Susanta Samajdar. Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab5f82e03ba5520dcf239a8a55ec749b43cc168" target='_blank'>
              Abstract A030: Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors
              </a>
            </td>
          <td>
            Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh V. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, S. S, A. A., A. B, G. Daginakatte, K. Nellore, Kishore Narayanan, C. Abbineni, Murali Ramachandra, S. Samajdar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eacc8b7b25e7e9f3513371fcb6ce3caa5b129f" target='_blank'>
              Navigating the landscape of EGFR TKI resistance in EGFR-mutant NSCLC - mechanisms and evolving treatment approaches.
              </a>
            </td>
          <td>
            Jing Zhao, Wanting Xu, Fei Zhou, Xiangyu Zhang, Mo Zhou, Da Miao, Lan Yu, Yongchang Zhang, Junqiang Fan, Caicun Zhou, Wen Li, Tony S K Mok, Xiuning Le, M. Li, Yang Xia
          </td>
          <td>2025-11-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA damage and repair are central to the onset of cancer, aging, and aging-related diseases. Rare genetic defects in the nucleotide excision repair pathway, such as those causing the cancer-prone disorder xeroderma pigmentosum (XP) or the progeroid condition Cockayne syndrome, highlight the dramatic consequences of unrepaired DNA lesions. In this issue of the JCI, two related papers from Ogi and coworkers — Fassihi et al. and Nakazawa et al. — describe a new XP clinical entity, XP-J, linked to a pathogenic variant in the p52 subunit of the transcription-repair complex TFIIH. The studies’ characterization of XP-J and the p52ΔC variant opened unexpected possibilities to ameliorate the molecular defect in another subunit of TFIIH that causes a different, more severe repair syndrome: trichothiodystrophy. This commentary provides a broader historical, medical, and molecular context for the intricate genotype-phenotype relationship between compromised repair and its clinical consequences and discusses next steps for the advances reported.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47309874361d974d2143fc65fd785375287e5c39" target='_blank'>
              Expanding the landscape of nucleotide excision repair disorders: from discovery to therapy
              </a>
            </td>
          <td>
            A. Theil, J. H. J. Hoeijmakers
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acdfefa7d1e9916c129010f57a43699739821df" target='_blank'>
              Cancer gene therapy: historical perspectives, current applications, and future directions
              </a>
            </td>
          <td>
            Jingyi Zeng, Jingwen Luo, Yingchun Zeng
          </td>
          <td>2025-09-29</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>