<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-26 09:52:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>37</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4abf519afad0178ba59d0cd17f3f428071bf92cb" target='_blank'>
              Abstract 2674: Cellular plasticity facilitates therapy resistance through enhancing adaptability
              </a>
            </td>
          <td>
            Alicia Bjornberg, Xierali Aobuli, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Jacob G. Scott, Berkley Gryder, David Bassanta, Alexander Anderson, Virginia Turati, A. Marusyk
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, M. Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bca1af35819fcfb6f9d7674ff4443096f1ec742" target='_blank'>
              Single-Cell Pharmacogenomic Landscapes of Epigenetic Therapy in Gastric Cancer
              </a>
            </td>
          <td>
            Chang Xu, Haoran Ma, T. Sheng, M. Lee, S. A. A. Abdul Ghani, Laura Perlaza-Jiménez, K. Huang, Shen Kiat Lim, Supriya Srivastava, Xuewen Ong, S. Tay, S. Ho, Angie Tan, Feng Zhu, Vincenzo Nasca, G. Randon, Hassan Ashktorab, A. S. Cheng, A. Jeyasekharan, Shang Li, Ming Teh, Raghav Sundar, David R. Powell, Joseph Rosenbluh, F. Pietrantonio, W. Yong, J. So, Patrick Tan
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43efa757ce011d3db682c5cad92c0307f3b656c7" target='_blank'>
              The Role of Phenotypic Plasticity in Adaptation to Treatment and Prospective Plasticity Drivers in Hepatoblastoma
              </a>
            </td>
          <td>
            Emilia Chen, Christie English, Alejandro Allo Anido, Siân Hamer, Stefano Cairo, Alejandra Bruna
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecdb6f080369d4f144052961e7b7ab0cd6c99ec" target='_blank'>
              Abstract 539: Secretory EMT fuels ERBB3 axis activation in resistance to EGFR inhibitors
              </a>
            </td>
          <td>
            A. Morselli, Donatella Romaniello, Chiara Miroglio, Martina Mazzeschi, M. Sgarzi, F. Pagano, C. Girone, Francesca Ambrosi, W. Kothalawala, Gyorffy Balazs, A. De Giglio, Michelangelo Fiorentino, Andrea Ardizzoni, Yosef Yarden, M. Lauriola
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer remains a significant global health burden due to its high morbidity and mortality. Oncogene-targeted therapy and immunotherapy have markedly improved the 5-year survival rate in the patients with advanced or metastatic tumors compared to outcomes in the era of chemotherapy/radiation. Nevertheless, the majority of patients remain incurable. Initial therapies eliminate the bulk of tumor cells, yet residual populations termed drug-tolerant persister cells (DTPs) survive, regenerate tumor and even drive distant metastases. Notably, DTPs frequently render tumor cross-resistance, a detrimental phenomenon observed in the patients with suboptimal responses to subsequent therapies. Analogous to species evolution, DTPs emerge as adaptative products at the cellular level, instigated by integrated intracellular stress responses to therapeutic pressures. These cells exhibit profound heterogeneity and adaptability shaped by the intricate feedforward loops among tumor cells, surrounding microenvironments and host ecology, which vary across tumor types and therapeutic regimens. In this review, we revisit the concept of DTPs, with a focus on their generation process upon targeted therapy or immunotherapy. We dissect the critical phenotypes and molecule mechanisms underlying DTPs to therapy from multiple aspects, including intracellular events, intercellular crosstalk and the distant ecologic pre-metastatic niches. We further spotlight therapeutic strategies to target DTP vulnerabilities, including synthetic lethality approaches, adaptive dosing regimens informed by mathematical modeling, and immune-mediated eradication. Additionally, we highlight synergistic interventions such as lifestyle modifications (e.g., exercise, stress reduction) to suppress pro-tumorigenic inflammation. By integrating mechanistic insights with translational perspectives, this work bridges the gap between DTP biology and clinical strategies, aiming for optimal efficacy and preventing relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a66d9e3af1fc508d0ed83941eccffc587ef2ea" target='_blank'>
              Drug-induced tolerant persisters in tumor: mechanism, vulnerability and perspective implication for clinical treatment
              </a>
            </td>
          <td>
            Shujie Liu, Anfeng Jiang, Faqing Tang, Minghao Duan, Bin Li
          </td>
          <td>2025-05-24</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cancer has consistently posed a significant challenge on a global scale, with rising incidence and mortality. Despite the tremendous advancements in cancer therapeutics throughout the years, chemotherapy remains the current standard treatment in cancer management. As a result, chemotherapy resistance among cancer patients has emerged as a major clinical challenge, causing unsatisfactory responses to the treatment. Chemotherapy resistance is closely related to the dysregulation of a normal cellular process called ‘pre-mRNA Alternative Splicing’, where one gene can generate various protein forms with different structures and functions. A comprehensive understanding of dysregulated alternative splicing mechanisms is essential to propose novel strategies in cancer research aimed at identifying new and improved methods to maximise treatment efficacy and overcome chemotherapy resistance. Therefore, this review highlights the complexity of chemotherapy resistance and the potential roles of alternative splicing in overcoming resistance to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c315ff50467ff3617156ad8cf2b1b2d1ebcf6f" target='_blank'>
              Aberrant Splicing as a Mechanism for Resistance to Cancer Therapies
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Copy number variations (CNVs), as an important structural variant in genomes, are widely present in plants, affecting their phenotype and adaptability. In recent years, CNV research has not only focused on changes in gene copy numbers but has also been linked to complex mechanisms such as genome rearrangements, transposon activity, and environmental adaptation. The advancement in sequencing technologies has made the detection and analysis of CNVs more efficient, not only revealing their crucial roles in plant disease resistance, adaptability, and growth development, but also demonstrating broad application potential in crop improvement, particularly in selective breeding and genomic selection. By studying CNV changes during the domestication process, researchers have gradually recognized the important role of CNVs in plant domestication and evolution. This article reviews the formation mechanisms of CNVs in plants, methods for their detection, their relationship with plant traits, and their applications in crop improvement. It emphasizes future research directions involving the integration of multi-omics to provide new perspectives on the structure and function of plant genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0396c8ecd922291ec579ba57375e978767ea2182" target='_blank'>
              A Systematic Review of the Advances and New Insights into Copy Number Variations in Plant Genomes
              </a>
            </td>
          <td>
            Saimire Silaiyiman, Jiaxuan Liu, Jiaxin Wu, Lejun Ouyang, Zheng Cao, Chao Shen
          </td>
          <td>2025-05-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df82979a5771df69f4d6ba63039b4660f6ec17f" target='_blank'>
              Abstract 5065: Single-cell transcriptomic profiling of longitudinal patient tumors and PDX models reveals resistance mechanisms in ALK- and EGFR-mutant LUAD
              </a>
            </td>
          <td>
            Naomi Mfonfu, F. Braye, Galina Boldina, Matteo Cesaroni, Luc Fibroulet, Sergey I. Nikolaev, Andrey A. Yurchenko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e09a8dc09cd22d667885f6e94da508cf98cf5f03" target='_blank'>
              Abstract 5364: Multi-omic analyses reveal epigenetic and transcriptomic reprogramming behind ibrutinib-venetoclax resistance development in mantle cell lymphoma
              </a>
            </td>
          <td>
            Kevin Qiu, Jing Gao, Qing Yin, Allen Hu, Zhenjia Wang, Loryn L Blackburn, Bijal Shah, Jeffrey W. Craig, C. Portell, Michael E Williams, Chongzhi Zang, Jianguo Tao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/221d45fdda028b3fdec5da9d6b92cc4847a8b483" target='_blank'>
              Understanding the Role of Toggle Genes in Chronic Lymphocytic Leukemia Proliferation
              </a>
            </td>
          <td>
            Olga Sirbu, Gunjan Agarwal, Alessandro Giuliani, Kumar Selvarajoo
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7191d1be8112623ab8fc75b326b24bf447563f16" target='_blank'>
              Abstract 1257: Response to primary therapy predicts tumor evolution dynamics at relapse in ovarian cancer
              </a>
            </td>
          <td>
            J. Oikkonen, Giulia Micoli, Susanna Holmström, Kari Lavikka, M. T. Nguyen, Yilin Li, G. Marchi, A. Rajavuori, Heidi Koskela, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Summary We demonstrate that carcinogenesis and multi-therapy resistance in multiple myeloma (MM)—a treatable yet incurable plasma cell malignancy—are driven by epigenetic dysregulation. In this new paradigm, genomic and cytogenetic events unlock epigenetic plasticity, reshaping MM cell biology to evade tumor microenvironment constraints and therapeutic pressure. These conclusions are derived from a newly assembled cohort of nearly 1,000 patients, spanning premalignant to late-stage refractory MM, comprehensively characterized at molecular and clinical levels. Our findings provide a unifying framework to explain inter-patient genomic heterogeneity and the emergence of therapy resistance in sequential samples without new genomic alterations. In conclusion, we propose targeting epigenetic plasticity-mediated plasma cell evasion as a promising therapeutic strategy in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19a7c4379c96f4e14d05eb168255be930f2fe17c" target='_blank'>
              Epigenetic Plasticity Drives Carcinogenesis and Multi-Therapy Resistance in Multiple Myeloma
              </a>
            </td>
          <td>
            A. Silva, Rafael R. Canevarolo, P. Sudalagunta, M. Meads, Maria Silva, Xiaohong Zhao, Dario Magaletti, R. Alugubelli, Gabriel De Avila, Erez Persi, F. Maura, Elissa Bell, Ryan T Bishop, Christopher L Cubitt, S. Sansil, Wei Zhang, Jamie K Teer, Mingxiang Teng, Sean J Yoder, E. Siegel, Bijal Shah, T. Nishihori, Lori Hazlehurst, Conor C Lynch, O. Landgren, Oliver Hampton, Robert Gatenby, Daniel Sullivan, J. Brayer, William S. Dalton, John L Cleveland, Melissa Alsina, Rachid C Baz, K. Shain
          </td>
          <td>2025-04-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b60f1dd8ce9984467648ff9cf2c70d381e498a6" target='_blank'>
              Abstract 7502: Radiation-induced transcriptomic reprogramming of mesenchymal glioblastoma stem cells reveals key pathways and potential targets
              </a>
            </td>
          <td>
            Grace M. Rosner, Abigail Goen, Godwin Peasah-Darkwah, Jonathan Rodgers Gochicoa, Alos B. Diallo, Gokul Srinivasan, Elisa Bu Sha, Joshua Levy, Kelli B. Pointer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217101cddf06b5ca15708fdd50f15de9f63e9eb9" target='_blank'>
              Abstract 5092: Characterization of mismatch repair status in colorectal cancer using single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Dimitri Joseph, Alexandra Guillaume
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colon cancer (CC) is one of the most common and lethal cancers worldwide, with rising incidence rates in both developed and developing countries. Although advances in treatments such as surgery, chemotherapy, and targeted therapies have been made, prognosis for advanced colon cancer, particularly with metastasis, remains poor. Recent studies highlight the significant role of post-transcriptional modifications like acetylation in cancer biology, affecting processes like gene transcription, metabolism, and tumor progression. Methods This study applied multi-omics analyses, including single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and Mendelian randomization. Data were obtained from public datasets like GSE132465, UCSC Xena, and GeneCards. We focused on acetylation-related genes, specifically NAT10 and GNE, using scoring methods, cell–cell interaction models, and survival analyses to investigate their role in colon cancer development, metastasis, and immune evasion. Results This study identifies that NAT10 is highly expressed in epithelial cells of colorectal cancer (CC) and is closely associated with tumor progression and metastasis. Single-cell RNA sequencing analysis revealed that NAT10-positive epithelial cells exhibited strong interactions with myeloid cells and T cells, with significant differences in cell–cell communication (p < 0.05). Based-on-summary-data Mendelian randomization (SMR) analysis further supports a causal relationship between NAT10 and colorectal cancer. In the MR analysis, a significant positive correlation was observed between NAT10 and colorectal cancer risk using summary data from genome-wide association studies (GWAS) and expression quantitative trait loci (eQTL) studies (β_SMR = 0.004, p_SMR = 0.041, p_HEIDI = 0.737). These findings suggest that NAT10 may serve as a pathogenic factor in colorectal cancer development, providing additional genetic evidence that links this acetylation-related gene to colorectal cancer. Survival analysis further demonstrated that NAT10-positive epithelial cells are associated with poorer prognosis. In the TCGA dataset, patients with NAT10-positive epithelial cells exhibited a significantly shorter disease-free survival (DFS) (p = 0.012). Unlike GNE-positive cells, NAT10-positive epithelial cells exhibited immune escape characteristics, and TIDE analysis indicated that NAT10-positive epithelial cells were associated with a lower response to immune checkpoint blockade therapy (p = 1.3e−5), suggesting that they may impair the efficacy of immunotherapy by promoting immune evasion. In contrast, GNE was also significantly expressed in epithelial cells of colorectal cancer, but its role differs from that of NAT10. GNE-positive epithelial cells demonstrated strong communication with immune cells, particularly in interactions between myeloid cells and T cells through receptor-ligand pairs. Despite the important role of GNE-positive epithelial cells in the tumor microenvironment, their association with immune escape is weaker compared to NAT10. Survival analysis revealed that GNE-positive epithelial cells were associated with a better prognosis (p = 0.015). In the TCGA dataset, patients with GNE-positive epithelial cells displayed longer disease-free survival (DFS), contrary to the results from the SMR analysis. Conclusions Leveraging SMR and multi-omics analysis, this study highlights the significant role of acetylation-related genes, particularly NAT10, in colon cancer. The findings suggest that acetylation modifications in epithelial cells contribute to immune evasion and cancer progression. NAT10 could serve as a promising biomarker and therapeutic target for early diagnosis and targeted therapy, offering new avenues for improving colon cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37348c834755949c2631ebe0e9fc050634d0c0ce" target='_blank'>
              Deciphering the role of acetylation-related gene NAT10 in colon cancer progression and immune evasion: implications for overcoming drug resistance
              </a>
            </td>
          <td>
            Xuancheng Zhou, Xun Sang, Lai Jiang, Shengke Zhang, Chenglu Jiang, Yuheng Gu, Yiping Fu, Guanhu Yang, Jieying Zhang, Hao Chi, Binbin Wang, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumor subclones refer to distinct cell populations within the same tumor that possess different genetic characteristics. They play a crucial role in understanding tumor heterogeneity, evolution, and therapeutic resistance. The formation of tumor subclones is driven by several key mechanisms, including the inherent genetic instability of tumor cells, which facilitates the accumulation of novel mutations; selective pressures from the tumor microenvironment and therapeutic interventions, which promote the expansion of certain subclones; and epigenetic modifications, such as DNA methylation and histone modifications, which alter gene expression patterns. Major methodologies for studying tumor subclones include single-cell sequencing, liquid biopsy, and spatial transcriptomics, which provide insights into clonal architecture and dynamic evolution. Beyond their direct involvement in tumor growth and invasion, subclones significantly contribute to tumor heterogeneity, immune evasion, and treatment resistance. Thus, an in-depth investigation of tumor subclones not only aids in guiding personalized precision therapy, overcoming drug resistance, and identifying novel therapeutic targets, but also enhances our ability to predict recurrence and metastasis risks while elucidating the mechanisms underlying tumor heterogeneity. The integration of artificial intelligence, big data analytics, and multi-omics technologies is expected to further advance research in tumor subclones, paving the way for novel strategies in cancer diagnosis and treatment. This review aims to provide a comprehensive overview of tumor subclone formation mechanisms, evolutionary models, analytical methods, and clinical implications, offering insights into precision oncology and future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d4c21265c412fc5d4c0775f9aa0ecd36b6c8eb" target='_blank'>
              Advances in tumor subclone formation and mechanisms of growth and invasion
              </a>
            </td>
          <td>
            Yuhong Zhang, Weidong Wang
          </td>
          <td>2025-04-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R. Chu, William F Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, I. Hajirasouliha, Marcin Imielinski, D. Nanus, S. Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) has emerged as a pivotal technology in the field of oncology, transforming the approach to cancer diagnosis and treatment. This paper provides a comprehensive overview of the integration of NGS into clinical settings, emphasizing its significant contributions to precision medicine. NGS enables detailed genomic profiling of tumors, identifying genetic alterations that drive cancer progression and facilitating personalized treatment plans targeting specific mutations, thereby improving patient outcomes. This capability facilitates the development of personalized treatment plans targeting specific mutations, leading to improved patient outcomes and the potential for better prognosis. The application of NGS extends beyond identifying actionable mutations; it is instrumental in detecting hereditary cancer syndromes, thus aiding in early diagnosis and preventive strategies. Furthermore, NGS plays a crucial role in monitoring minimal residual disease, offering a sensitive method to detect cancer recurrence at an early stage. Its use in guiding immunotherapy by identifying biomarkers that predict response to treatment is also highlighted. Ethical issues related to genetic testing, such as concerns around patient consent and data privacy, are also important considerations that need to be addressed for the broader implementation of NGS. These include the complexities of data interpretation, the need for robust bioinformatics support, cost considerations, and ethical issues related to genetic testing. Addressing these challenges is essential for the widespread adoption of NGS. Looking forward, advancements such as single-cell sequencing and liquid biopsies promise to further enhance the precision of cancer diagnostics and treatment. This review emphasizes the transformative impact of NGS in oncology and advocates for its incorporation into routine clinical practice to promote molecularly driven cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f589e1477d90e5debe146a1fa230847b1a203d" target='_blank'>
              Next-generation sequencing in cancer diagnosis and treatment: clinical applications and future directions
              </a>
            </td>
          <td>
            Nima Ghoreyshi, Reza Heidari, A. Farhadi, Mohsen Chamanara, Nastaran Farahani, Mahmood Vahidi, Javad Behroozi
          </td>
          <td>2025-04-20</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a04310544d53b394a40ce917fc33a338d122cf3" target='_blank'>
              Abstract SY34-01: Developmental dependency and metabolic vulnerabilities in NUP98-rearranged pediatric leukemia
              </a>
            </td>
          <td>
            Elvin Wagenblast
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c9f0f37888749fdd1b5acd90887f2484e915ec3" target='_blank'>
              Abstract 5553: Acquisition of adaptive and reversible drug-tolerance state to cabozantinib in renal cell carcinoma cell lines
              </a>
            </td>
          <td>
            A. Zaccagnino, Hiresh Ayoubian, Kerstin Junker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a3bde6f2ebc33e7239f55d9b0ddb6732733f31" target='_blank'>
              Abstract 4190: CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells
              </a>
            </td>
          <td>
            Martina Miotto, S. Lamba, G. Grasso, G. Corti, Julie Bonetto, A. Sogari, A. Bardelli, Mariangela Russo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6992085eb1aec63c000f8c350d79dda9f01e87" target='_blank'>
              A computational strategy to uncover fusion genes in prostate cancer cell lines
              </a>
            </td>
          <td>
            Rebecca A. Morgan, Gary Hardiman
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076e8ebfb600d70faa36a964146ab32ad3cd766a" target='_blank'>
              Screening and application of unstable genetically resistant strains in fission yeast.
              </a>
            </td>
          <td>
            Lu-Feng Dan, Yi-Wen Chu, Xin-Rong Wang, Xiang-Wei He
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5713e3952828abd074f75de5f64c2e281aa291d9" target='_blank'>
              Abstract 4408: CRISPR/Cas9 knockout and AI-based drug screening for novel gene targets in ovarian cancer chemoresistance
              </a>
            </td>
          <td>
            Jinjiang Wang, Jiaxi Li, Jiao Liu, Kit-Ying Chan, Ho-Sze Lee, Zhewei Zhang, Joseph Kwong, T. Lau, C. Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbfea92c86debe9c5d855419c64075ab48f3f011" target='_blank'>
              Single cell transcriptional evolution of myeloid leukaemia of Down syndrome
              </a>
            </td>
          <td>
            Mi K Trinh, Konstantin Schuschel, Hasan Issa, Rebecca Thomas, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, L. Mamanova, E. Prigmore, Emilia Robertson, A. Hodder, Anna Wenger, Nathaniel D Anderson, Holly J. Whitfield, T. Treger, J. Gonçalves-Dias, Karin C Straathof, David O’Connor, Matthew D. Young, Laura Jardine, Stuart Adams, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea7a11f970c3515d6da1945980a0668b04d9e68" target='_blank'>
              Abstract 4189: CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors
              </a>
            </td>
          <td>
            Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, F‐J Hao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd0035313cd95f13c0ad0b1db10655d699eb842" target='_blank'>
              Abstract 2991: Systematic discovery of regulators of adoptive T-cell therapeutic resistance using in vivo CRISPR screening
              </a>
            </td>
          <td>
            Chenlin Zhao, Sharif U. Ahmed, Xiyue Hu, R. S. Atwal, Zongjie Wang, Sophie Xiao, S. Kelley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a1ac1da292bdfa7cb083f4691cc613492b11ea" target='_blank'>
              Pan-cancer Analysis Identified Ectopic RUNX1T1 Associated with Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, I. Kim, Su Deng, Ping Mu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Development of next generation sequencing technologies allows to identify a large number of genetic landscape types in various cancers including breast cancer. Frequent genetic abnormalities identified using whole genome sequencing are point mutations (missense, nonsense mutations), deletions, insertions, which usually lead to activation of protooncogenes and inactivation of tumor suppressor genes. Genome sequencing of malignant tumors allowed, on one hand, to identify driver mutations in carcinogenic genes in different organs, and on the other – to use mutated genes for targeted therapy. Study of biological functions of these genes from the point of view of their contribution to carcinogenesis allows to better understand its mechanism. In this review, signaling cascades of breast cancer with identified mutated genes – targets for therapy – are analyzed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf51075c6ced2d719b16f81522dd0e475d5f9454" target='_blank'>
              Abstract 4605: Exploring resistance mechanisms in cancer using patient-derived organoids reveals replication stress and EMT as potential biomarkers
              </a>
            </td>
          <td>
            Ruben M. Drews, Finnian Firth, Paul R. Barber, Anna Pasto, Tony T. Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6afa5982640193cfc266d2068be1bacf5082da2a" target='_blank'>
              Abstract 2648: ZFP36L2 orchestrates stress adaptive plasticity during injury repair and metastasis
              </a>
            </td>
          <td>
            Qingwen Jiang, Manisha Raghavan, Britney Forsyth, Aileen Rodriguez, Cyrus Tam, Sasha Balkaran, S. Hartner, Morgan Lallo, Ahmed Mahmoud, Andrew Moorman, Jura Pintar, D. Pe’er, R. Koche, Quaid Morris, Joseph Chan, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27130e90001be9912edb71735648fc5052748bf" target='_blank'>
              Abstract 7494: An integrative systems-level framework reveals SMARCD3 resistance regulon signatures vulnerable to MEK1/2 inhibitors in ER+BC cells resistant to everolimus
              </a>
            </td>
          <td>
            Eric F. Medina, Elena Farmaki, Jason I. Griffiths, V. Grolmusz, Andrea H Bild, Aritro Nath
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract APOBEC mutagenesis is one of the most common endogenous sources of mutations in human cancer and is a major source of genetic intratumor heterogeneity. High levels of APOBEC mutagenesis are associated with poor prognosis and aggressive disease across diverse cancers, but the mechanistic and functional impacts of APOBEC mutagenesis on tumor evolution and therapy resistance remain relatively unexplored. To address this, we investigated the contribution of APOBEC mutagenesis to acquired therapy resistance in a model of EGFR-mutant non–small cell lung cancer. We find that inhibition of EGFR in lung cancer cells leads to a rapid and pronounced induction of APOBEC3 expression and activity. Functionally, APOBEC expression promotes the survival of drug-tolerant persister cells (DTP) following EGFR inhibition. Constitutive expression of APOBEC3B alters the evolutionary trajectory of acquired resistance to the EGFR inhibitor gefitinib, making it more likely that resistance arises through de novo acquisition of the T790M gatekeeper mutation and squamous transdifferentiation during the DTP state. APOBEC3B expression is associated with increased expression of the squamous cell transcription factor ΔNp63 and squamous cell transdifferentiation in gefitinib-resistant cells. Knockout of p63 in gefitinib-resistant cells reduces the expression of the ΔNp63 target genes IL-1α/β and sensitizes these cells to the third-generation EGFR inhibitor osimertinib. These results suggest that APOBEC activity promotes acquired resistance by facilitating evolution and transdifferentiation in DTPs and that approaches to target ΔNp63 in gefitinib-resistant lung cancers may have therapeutic benefit. Significance: APOBEC mutagenesis is a common source of genetic heterogeneity in cancer, and APOBEC mutational signatures are enriched in metastatic and drug-resistant tumors. However, the mechanisms through which APOBEC3 enzymes promote tumor evolution remain unknown. In this study, we show that APOBEC3 activity contributes to the development of therapy-resistant cancer cells by promoting evolution of DTP cells. These findings offer insights into the role of APOBEC mutagenesis in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7a11c8b2dc4731f11b9133f359e7ac89041b4a" target='_blank'>
              APOBEC3 Activity Promotes the Survival and Evolution of Drug-Tolerant Persister Cells during EGFR Inhibitor Resistance in Lung Cancer
              </a>
            </td>
          <td>
            N. M. G. Garcia, Jessica N Becerra, Sharan Srinivasan, Brock J McKinney, Ashley V DiMarco, Feinan Wu, Matthew Fitzgibbon, James V. Alvarez
          </td>
          <td>2025-05-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Primary mouse renal proximal tubule epithelial cells (moRPTECs) were immortalized by lentivirus transduction to create hTERT or SV40LgT (LgT) cell lines. Prior work showed a more pronounced injury and repair response in LgT versus hTERT cells after chemical challenge. We hypothesized that unique genomic changes occurred after immortalization, altering critical genes and pathways. RNA-seq profiling and whole-genome sequencing (WGS) of parent, hTERT, and LgT cells showed that 92.5% of the annotated transcripts were shared, suggesting a conserved proximal tubule expression pattern. However, the cell lines exhibited unique transcriptomic and genomic profiles different from the parent cells. Three transcript classes were quite relevant for chemical challenge response—Cyps, ion channels, and metabolic transporters—each important for renal function. A pathway analysis of the hTERT cells suggested alterations in intermediary and energy metabolism. LgT cells exhibited pathway activation in cell cycle and DNA repair that was consistent with replication stress. Genomic karyotyping by combining WGS and RNA-seq data showed increased gene copy numbers in chromosome 5 for LgT cells, while hTERT cells displayed gene copy losses in chromosomes 4 and 9. These data suggest that the exaggerated transcriptional responses of LgT cells versus hTERT cells result from differences in gene copy numbers, replication stress, and the unique selection processes underlying LgT or hTERT immortalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5fc2423810ee9671b81c235e2952419eb5bbc3d" target='_blank'>
              hTERT and SV40LgT Renal Cell Lines Adjust Their Transcriptional Responses After Copy Number Changes from the Parent Proximal Tubule Cells
              </a>
            </td>
          <td>
            B. A. Merrick, Ashley M Brooks, Julie F. Foley, N. Martin, Rick D. Fannin, Wesley Gladwell, Kevin E. Gerrish
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d383f031fb716745aefe8cfd14a2a089d4db51c" target='_blank'>
              Quantifying replication stress in cancer without proliferation confounding
              </a>
            </td>
          <td>
            Philipp Jungk, Maik Kschischo
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca6f30d3b9e64a4de714f44e69384d7201b4a8d" target='_blank'>
              Abstract 440: Dual epigenetic targeting of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kyounghyun Kim, Ponmari Guruvaiya
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d1fce0d3ccba183762c44ebcd950f1a93cc11" target='_blank'>
              Abstract 6934: Acetate-induced transcriptomic alterations reveal potential therapeutic targets in triple negative breast cancer
              </a>
            </td>
          <td>
            Gabriel Borges-Vélez, Diego N. Bracero-Sánchez, Liah M. Román-Calderón, Oscar A. Loperena-González, Ariana S. García-Lopez, Angel J. Amaral-Maisonet, Jeannette L. Salgado-Montilla, Esther A. Peterson-Peguero, Josué Pérez-Santiago
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc2b803c7ac7b99f1e6df1e4aac1c92682211a5" target='_blank'>
              Abstract 5557: Deciphering the interplay of β-catenin/YAP signaling and alveolar lineage plasticity during targeted therapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yu-Ting Chou, Wei Wu, D. Kerr, V. Olivas, Macey Slota, T. Bivona
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The integrity of the genome is maintained by mismatch repair (MMR) proteins that recognize and repair base mismatches and insertion/deletion errors generated during DNA replication and recombination. A defective MMR system results in genome-wide instability and the progressive accumulation of mutations. Tumors exhibiting deficient MMR (dMMR) and/or high levels of microsatellite instability (termed “microsatellite instability high”, or MSI-H) have been shown to possess fundamental differences in clinical, pathological, and molecular characteristics, distinguishing them from their “microsatellite stable” (MSS) counterparts. Molecularly, they are defined by a high mutational burden, genetic instability, and a distinctive immune profile. Their distinct genetic and immunological profiles have made dMMR/MSI-H tumors particularly amenable to treatment with immune checkpoint inhibitors (ICIs). The ongoing development of biomarker-driven therapies and the evaluation of novel combinations of immune-based therapies, with or without the use of conventional cytotoxic treatment regimens, continue to refine treatment strategies with the goals of maximizing therapeutic efficacy and survival outcomes in this distinct patient population. Moreover, the resultant knowledge of the mechanisms by which these features are suspected to render these tumors more responsive, overall, to immunotherapy may provide information regarding the potential optimization of this therapeutic approach in tumors with proficient MMR (pMMR)/MSS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0727d40a1aa873459400ba4134581be209441f5d" target='_blank'>
              Deficient Mismatch Repair and Microsatellite Instability in Solid Tumors
              </a>
            </td>
          <td>
            Joy A. Awosika, James L. Gulley, Danielle M. Pastor
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2254442f541b7850b7eb9c6ffd437b3ddb785eca" target='_blank'>
              Abstract 4834: Genomic and metabolic effects of KDM6A loss regulates differential responses to immune checkpoint therapy and chemotherapy in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, D. Raychaudhuri, Bidisha Chakraborty, Aminah J. Tannir, Swadhin Meher, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, P. Pilié, Sangeeta Goswami
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Background Recent genomic analyses of poor prognostic and relapsed leukemia have revealed the involvement of diverse gene mutations in treatment resistance. These gene mutations are classified into two groups: mutations involving resistance to specific agents such as the BCR::ABL1 fusion gene mutations (typically T315I mutation) in tyrosine kinase inhibitor (TKI) resistance and those involving the resistance to diverse therapeutic modalities such as the TP53 gene mutations. In the latter type, although their associations with drug resistance have been clinically demonstrated, the direct association with resistance to each therapeutic modality remains to be fully elucidated. To overcome treatment resistance induced by these gene mutations, appropriate leukemic cellular models are urgently required. Using the cytidine base editing (CBE) system, we introduced two types of mutations through C-to-T transition into human leukemia cell lines and evaluated their significance in the drug sensitivities. Methods We applied the CBE system to introduce the T315I (ACT to ATT) mutation of the BCR::ABL1 fusion gene in a human Philadelphia chromosome-positive leukemia cell line and to introduce the T125M (ACG to ATG) mutation of the TP53 gene in a human B-cell precursor acute lymphoblastic leukemia (BCP-ALL) cell line. Results We first confirmed an introduction of the T315I mutation in one of four BCR::ABL1 alleles as a result of base editing in the obtained TKI-resistant subline. We also identified the additional C-to-T transition at adjacent codon 314 (ATC), which resulted in a silent mutation, in the same allele. We next confirmed that the obtained subline acquired the T125M mutation of the TP53 gene without additional C-to-T transition. In the T125M subline, transcriptional activities of the p53 protein were disrupted and the sensitivities to diverse chemotherapeutic drugs and irradiation were reduced. Conclusion Our observations demonstrated the utility of the CBE system for introducing specific nucleotide transitions into human leukemia cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ade1d81a455d4e9f820780a2721c7705b4dbb" target='_blank'>
              Utility of the Base Editing System for Introducing Drug-Resistant Gene Mutations Into Human Leukemia Cellular Models
              </a>
            </td>
          <td>
            Thao Nguyen, Minori Tamai, Shinichi Fujisawa, Akiko Nagamachi, K. Kagami, Chiaki Komatsu, S. Iwamoto, Toshiya Inaba, Takanori Teshima, Takeshi Inukai
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dec962af7d1ee05246dee3d378f9152a2d6c3e" target='_blank'>
              Genetic variants in NUDT15 gene their clinical implications in cancer therapy
              </a>
            </td>
          <td>
            Y. Jarrar, Maria Ghishan, Fatima Khirfan, Nancy Hakooz
          </td>
          <td>2025-04-14</td>
          <td>Drug Metabolism and Personalized Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematological malignancy. Patients with wild-type FLT3 relapsed or refractory (R/R) AML face significant therapeutic challenges due to the persistent lack of effective treatments. A comprehensive understanding of the mechanisms underlying chemotherapy resistance is needed to the development of effective treatment strategies. Therefore, we investigated the molecular mechanisms underlying cytarabine (Ara-C) resistance and daunorubicin (DNR) tolerance in Ara-C-resistant RHI-1 cells derived from the wild-type FLT3 AML cell line SHI-1. Quantitative analysis of intracellular drug concentrations, proteomics, and phosphoproteomics showed that DNR resistance in Ara-C-resistant RHI-1 cells is driven by metabolic remodeling toward mitochondrial metabolism, upregulation of DNA repair pathways, and enhanced reactive oxygen species (ROS) detoxification rather than reduced drug uptake. Moreover, targeting these compensatory mechanisms, particularly the OXPHOS complex I proteins, significantly improved the efficacy of both Ara-C and DNR. Conclusively, these findings highlight mitochondrial metabolism and DNA repair as critical factors in chemotherapy resistance and offer valuable insights into potential therapeutic targets for enhancing treatment outcomes in patients with wild-type FLT3 R/R AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49a8e37a3ebb622ad11d194102e12d1292163e3" target='_blank'>
              Mechanisms of chemotherapy failure in refractory/relapsed acute myeloid leukemia: the role of cytarabine resistance and mitochondrial metabolism
              </a>
            </td>
          <td>
            Soo Yeon Chae, Se-Young Jang, Jinhui Kim, Sehyun Hwang, D. Malani, O. Kallioniemi, S. Yun, Jong-Seo Kim, Hugh I Kim
          </td>
          <td>2025-04-23</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc2cff3f4e6a1f191f248963c0061d80205862fb" target='_blank'>
              A whole genomic CRISPR-Cas9 screen identifies the amino acid transporter SLC43A1 (LAT3) as a major determinant of oxaliplatin sensitivity in colorectal cancer cells
              </a>
            </td>
          <td>
            NR Pawar, HM Wade, Z. Jackson, N. Poungpeth, AV Mitchell, CP Jewell, D. Chan, RW Robey, PJ Batista, LM Jenkins, MM Gottesman
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ead68569a8b2470e49064bdb4fedf8ee91a2841" target='_blank'>
              Differential PARP inhibitor responses in BRCA1-deficient and resistant cells in competitive co-culture
              </a>
            </td>
          <td>
            Shiella A Soetomo, Michael F Sharp, W. Crismani
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648d311b893d674bef8fa9845b19283b6062d3e3" target='_blank'>
              Abstract 1477: Do MSH3 mutations really affect colorectal cells neoplastic transformation: an evolution of three African Americans novel mutations within the MSH3 ATPase domains
              </a>
            </td>
          <td>
            Mudasir Rashid, H. Brim, Katherine M Casazza, Jennifer A. Surtees, Shaolei Teng, Adebiyi Sobitan, A. Laiyemo, Z. Sherif, Koi Minoru, John Carethers, H. Ashktorab
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cbcd78ee21d56ae8a1d6d546cf74947ee0c43c0" target='_blank'>
              Abstract 561: Targeting mTOR/S6 signaling reverses acquired resistance to MTRX849 in KRASG12C-mutant lung adenocarcinoma cell lines
              </a>
            </td>
          <td>
            Athena M Madrid, Gabriel Choi, Jenny J. Hong, Sahel Hazeghsa, Jenny Z. Zhang, Kenny W. Castro, Neil A. O'Brien, Dennis J. Slamon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1582331698256f81dcbd70eb9ab64e969d89900e" target='_blank'>
              Abstract 5664: Identifying novel vulnerabilities in clear cell sarcoma through functional and genomic screening
              </a>
            </td>
          <td>
            Laura Leuenberger, Larissa Isenegger, Yanjiang Chen, Miles Burri, Sarah Kollar, Lara Planas-Paz, Peter Karl Bode, A. Wozniak, S. Bauer, Patrick Schöffski, Chantal Pauli, Christian Britschgi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Chimeric antigen receptor (CAR) T-cell therapy has transformed cancer management over the past decades, offering new hope to many patients. However, its effectiveness is often limited due to cytokine release syndrome (CRS), a life-threatening inflammatory response. Despite its clinical relevance, the molecular mechanisms underlying CRS, specifically in CAR T-cell products, remain poorly understood. This study aims to identify hub genes and pathways in preinfusion CAR T-cell products associated with CRS development and evaluate their potential as therapeutic targets through drug-gene interaction analysis and immune cell correlation profiling. Methods: We examined gene expression data from 43 preinfusion clusters of differentiation 22 of CAR T-cell samples (CD22+), sourced from the Gene Expression Omnibus dataset GSE200296. Using the linear models for microarray data package, we identified differences in gene expression and conducted enrichment analyses to explore relevant biological pathways, including Kyoto Encyclopedia of Genes and Genomes and Gene Ontology terms. We built protein-protein interaction networks using the Search Tool for Retrieval of Interacting Genes/Proteins database to understand how these genes interact and pinpointed central "hub genes" with Cytoscape and the cytoHubba plugin. Our findings were validated using the GeneCards database (Weizmann Institute of Science, Israel) and an independent CRS-related dataset (GSE164805). Additionally, we analyzed immune cell populations and explored potential drug-gene interactions. Results: Our study identified 24 genes with changed expression levels: 16 were downregulated and eight were upregulated. We identified five hub genes, interleukin (IL)1B, IL15, CD276, NCR2, and CCL17, as key contributors in CRS, which were primarily implicated in immune-related pathways, including cytokine-cytokine receptor interactions, IL17 signaling, and TNF signaling. These genes were especially expressed in monocytes, macrophages, and dendritic cells, confirming that those immune cell types play a critical role in CRS development. Through drug-gene interaction analysis, we found prospective therapies, such as enoblituzumab (targeting CD276) and canakinumab (targeting IL1B), which might assist in reducing CRS severity. Conclusion: The study highlights IL1B, IL15, CD276, NCR2, and CCL17 as key CRS genes in preinfusion CAR T-cell products. Their dysregulation activity may contribute to the increased inflammation noted in CRS, pointing to a loss of regulatory control. Bringing us closer to better patient outcomes, these findings not only suggest that these genes could serve as valuable biomarkers for predicting CRS but also open the way for the development of more precise treatments such as combining drugs such as enoblituzumab and canakinumab, which might assist in reducing CRS severity and making CAR T-cell therapy safer and more effective, ultimately improving patient lives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd6910e44d2ba1a3106170ae84bee8bcf217bd" target='_blank'>
              Identification of Hub Genes and Pathways in Preinfusion Chimeric Antigen Receptor (CAR) T-cell Products Associated With Cytokine Release Syndrome
              </a>
            </td>
          <td>
            Somia A Khalafallah, Anas Mohammed Elzein, Israa M Faris, Gamila A Attaelmanan, Marwa F. Alamin, Alaa Almleeh, Mohammed Bader, Mohamed Alfaki
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11b087a0a9fc0566d0ad6c1ba6f886b3c8f91ce" target='_blank'>
              Abstract 6385: Heterogeneity in sensitivity to second-line inhibitors of therapy-resistant melanoma
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Pavithran T. Ravindran, Arjun Raj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2d4cdd26b00d23c9f2af55b08e343fcfbd17a4" target='_blank'>
              Abstract 1469: Deciphering mechanisms of intrinsic radioresistance in breast cancer: Insights from transcriptomics and CRISPRi screens
              </a>
            </td>
          <td>
            B. McBean, Priyanka S. Rana, Benjamin Hauk, A. Michmerhuizen, S. Lichtman-Mikol, Reine Abou Zeidane, Vesna Markovic, K. Wilder-Romans, Mingfang Tao, Alan P. Boyle, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acaaf06b5fb2fb1345caa571c60786539fe457bf" target='_blank'>
              Somatic TP53 Mutations Drive T and NK Cell Dysfunction in AML and Can be Rescued by Reactivating Wild Type p53
              </a>
            </td>
          <td>
            Li Li, M. Muftuoglu, Edward Ayoub, Jiangxing Lv, Mahesh Basyal, A. Bidikian, Ran Zhao, Menon Prashant, Poo Yee Mak, Rupesh K. Kesharwani, Yuki Nishida, G. Issa, N. Varadarajan, B. Carter, Michael Andreeff
          </td>
          <td>2025-04-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Chemotherapeutic resistance is a major obstacle to chemotherapeutic failure. Cancer cell resistance involves several mechanisms, including epithelial-to-mesenchymal transition (EMT), signaling pathway bypass, drug efflux activation, and impairment of drug entry. P-glycoproteins (P-gp) are an efflux transporter that pumps chemotherapeutic drugs out of cancer cells, resulting in chemotherapeutic resistance. Several types of long noncoding RNA (lncRNAs) have been identified in resistant cancer cells, including ODRUL, MALAT1, and ANRIL. The high expression level of ODRUL is related to the induction of ATP-binding cassette (ABC) gene expression, resulting in the emergence of doxorubicin resistance in osteosarcoma. lncRNAs are observed to be regulators of drug transporters in cancer cells such as MALAT1 and ANRIL. Targeting P-gp expression using natural products is a new strategy to overcome cancer cell resistance and improve the sensitivity of resistant cells toward chemotherapies. This review validates the inhibitory effects of natural products on P-gp expression and activity using in silico molecular docking. In silico analysis showed that Delphinidin and Asparagoside-f are the most significant natural product inhibitors of p-glycoprotein-1. These inhibitors can reverse multi-drug resistance and induce the sensitivity of resistant cancer cells toward chemotherapy based on in silico molecular docking. It is important to validate that pre-elementary docking can be confirmed using in vitro and in vivo experimental data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcff0bc9887f04614dc5ee21157374eef0c56686" target='_blank'>
              Discovery of drug transporter inhibitors tied to long noncoding RNA in resistant cancer cells; a computational model -in silico- study
              </a>
            </td>
          <td>
            Mohanad Diab, Amel Hamdi, Feras N. Al-Obeidat, Wael Hafez, Ivan Cherrez-Ojeda, Muneir Gador, Gowhar Rashid, S. F. Elkhazin, Mahmad Anwar Ibrahim, Tarek Farag Ismail, S. S. Alkafaas
          </td>
          <td>2025-04-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf13a264e1cd78c0d16de17ca799be490bf7e832" target='_blank'>
              Challenges and Pathways in Regulating Next-Gen Biological Therapies.
              </a>
            </td>
          <td>
            Surendra Agrawal, Sunita Vaidya, Jitendra Patel, P. Jirvankar, Pravina Gurjar
          </td>
          <td>2025-04-23</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6350a9e7edaa4d3aae5b8855bca2a2b934a43915" target='_blank'>
              P53 deficiency triggers hypertranscription, inducing nucleotide insufficiency causing replication stress, and genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, Tchelet Goldberg, A. Eliassaf, Karen Grimes, M. Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8de13c6fd974bf26d03a422a6ee56b54180c3" target='_blank'>
              Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, F. Cidre-Aranaz, A. Wenninger-Weinzierl, M. Zylka, Karla Queiroz, D. Kurek, A. Sastre, Javier Alonso, Martin Distel, Anna Obenauf, T. G. Grünewald, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is a significant global health challenge, with 2% of cases fuelled by RET rearrangements. RET inhibitors (RETi) have revolutionized treatment for these patients, but resistance remains an important clinical challenge limiting therapy effectiveness. This study investigated the mechanisms underlying resistance to RETi. Methods NSCLC cells were exposed to increasing doses of RETi (pralsetinib/BLU-667 and selpercatinib/LOXO-292) to generate resistant cells. RNA-Sequencing analysis identified differentially expressed genes in resistant versus sensitive cells, followed by in vitro and in vivo functional assays to explore novel therapeutic strategies. Additionally, tumor biopsies from RET-rearranged NSCLC patients who exhibited cancer progression on RET inhibitor therapy were analyzed. Results RNA-sequencing analysis revealed the upregulation of the EGFR signaling pathway and hyperactivation of AP1 complex members in resistant cells compared to sensitive cells. Silencing of EGFR and AP1 complex members significantly reversed drug resistance, whereas EGFR overexpression reduced the sensitivity of parental Lc2/AD cells to RET inhibitors. Furthermore, the combination of RET and EGFR inhibitors showed synergistic antitumor activity in vitro and hindered tumor growth in mouse models with resistant cell xenografts. Notably, we observed a significant increase in EGFR expression in tumor biopsies from NSCLC patients treated with RET inhibitors who experienced disease progression, further validating the clinical relevance of our findings. Conclusions This study elucidates EGFR's role in mediating resistance to RET inhibitors in NSCLC patients. These findings offer insights into therapeutic adaptation and explore personalized combinations of RET and EGFR inhibitors for improved clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03392-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a7ddba9885e41061d39a28d4851cc532e72ca2d" target='_blank'>
              AP1-mediated reprogramming of EGFR expression triggers resistance to BLU-667 and LOXO-292 in RET-rearranged tumors
              </a>
            </td>
          <td>
            Daniela Esposito, C. M. Ascione, S. Belli, F. Napolitano, A. Servetto, Felice Pepe, U. Malapelle, A. Iaccarino, G. Troncone, Diletta Barone, Emilio Bria, Roberto Ferrara, Daniele Lorenzini, G. Lo Russo, Maria-Rosa Ghigna, A. Marinello, M. Aldea, B. Besse, Luigi Formisano, Roberto Bianco
          </td>
          <td>2025-05-22</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect—a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ec8a3709640e7f5608e28fbcbdbf6376d5a9c2" target='_blank'>
              A redefined InDel taxonomy provides insights into mutational signatures
              </a>
            </td>
          <td>
            G. Koh, A. S. Nanda, Giuseppe Rinaldi, Soraya Boushaki, A. Degasperi, Cherif Badja, A. Pregnall, S. Zhao, Lucia Chmelova, Daniella Black, Laura Heskin, João M. L. Dias, Jamie Young, Yasin Memari, Scott Shooter, Jan Czarnecki, Matthew A Brown, Helen R Davies, Xueqing Zou, S. Nik-Zainal
          </td>
          <td>2025-04-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0da8bbe26d7b85bc1bd170b1562ddebfb5b49ba9" target='_blank'>
              Abstract 6382: DNA damage repair pathways enable a drug-tolerant persister state in RET-fusion NSCLC and precedes TKI resistance
              </a>
            </td>
          <td>
            F. Uddin, Samuel Tischfield, R. Somwar, Alexander Lim, Hyung Jun Woo, Christina J Falcon, Barbara P. Mello, Dennis Kinyua, Harsha Sridhar, P. Manoj, Hong Zhong, Juan Qiu, E. de Stanchina, M. Donoghue, Marc Ladanyi, E. Redin, Charles M. Rudin, Á. Quintanal-Villalonga, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58e72b48369a6d6279bf45ff8835cc869b1ce39" target='_blank'>
              CRISPR screens in the context of immune selection identify CHD1 and MAP3K7 as mediators of cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Alex Watterson, G. Picco, Vivien Veninga, Youhani Samarakoon, C. M. Cattaneo, Sara F Vieira, Emre Karakoc, S. Bhosle, Thomas W Battaglia, Sarah Consonni, Timotheus Y. F. Halim, E. Voest, Mathew J. Garnett, Matthew A. Coelho
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d1574fe2e02ef3f4ee02ae939fc0c878b2510b" target='_blank'>
              Abstract 2392: The use of transcriptomic analysis to identify novel drug targets for DSA drug combination studies in acute myeloid leukemia
              </a>
            </td>
          <td>
            William A Chen, Valery Filippov, Jie Fang, Pedro Ochoa, C. Casiano, Kristopher E Boyle, S. Dovat, Hongjian Jin, Jun Yang, Olivia L Francis-Boyle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza M. Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dead0a4e6712a90fb575b926eb0a4af6b736d577" target='_blank'>
              Abstract 1259: Studying the role of chromatin accessibility in lineage plasticity of Rb1-deficient EGFR-mutant lung adenocarcinoma
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Lynch syndrome (LS), caused by germline pathogenic variants in the mismatch repair genes, leads to high rates of frameshift-derived neopeptide (FSDN) expression due to microsatellite instability (MSI). While colorectal cancer (CRC) prevention is effective, most LS-related tumors lack such strategies. Cancer vaccines targeting FSDNs offer a promising approach for immune interception in LS. This study aimed to identify and validate LS-related FSDNs to develop vaccines for cancer prevention. Methods We identified LS-related coding MS mutations and predicted FSDN with high coverage on common Human Leukocyte Antigen (HLA)-I and II alleles. We validated FSDN-associated mutations in colorectal adenomas (CrAD), endometrial cancers (EC), and CRC samples from patients with LS, non-LS tumors, and cell lines. Immunogenicity was assessed through interferon (IFN)-γ enzyme-linked immunospot and flow cytometry analysis of tissue-infiltrating lymphocytes (TILs) from LS carriers. Results We prioritized 53 HLA-I and 45 HLA-II FSDNs in MSI tumors using in silico predictions. Validation revealed 86.7% of FSDN-associated mutations present in LS-CRC samples, with a median of 7.67 (6.5–9) mutations in CrADs and 6.02 (2–10) in CRCs. Sequencing of CrAD and EC samples showed 95% and 77.5% of predicted FSDN-associated mutations, respectively. MSI cancer cell lines transcribed 69.8% of FSDNs. Immunogenicity assays showed that 71% of potential FSDNs elicited IFN-γ responses, with a median of 7.37 (1–10.75) HLA-I and 6 (2–5.75) HLA-II FSDNs per patient. After prioritizing 24 FSDN, in a cohort of 19 LS-derived samples (4 CrAD and 15 normal mucosa), 52% (10/19) demonstrated T-cell reactivity to an HLA-I neoantigen pool. CD8+CD137+ activation markers increased significantly (p=0.037) over time and peptide-specific cells were detected by pentamer staining. Conclusions Our predicted FSDN set has optimal coverage among LS carriers and can induce IFN-γ inflammatory responses in LS-derived TILs, offering an opportunity for vaccine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4e379b9e444406c0c4ab08355c1d7dbd836a26" target='_blank'>
              Discovery and validation of frameshift-derived neopeptides in Lynch syndrome: paving the way for novel cancer prevention strategies
              </a>
            </td>
          <td>
            Cristina Bayó, Giancarlo Castellano, Fátima Marín, J. Castillo-Iturra, T. Ocaña, Hardeep Kumari, M. Pellisé, Leticia Moreira, L. Rivero, M. Daca-Alvarez, Oswaldo Ortíz, S. Carballal, Rebeca Moreira, Júlia Canet-Hermida, Marta Pineda, Capella Gabriel, Georgina Flórez-Grau, Manel Juan, D. Benitez-Ribas, Francesc Balaguer
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43978659ce38ef19fbe0b48f8cd99b848f7e9b3" target='_blank'>
              Abstract 7490: Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b24fd08026b81ce4d7ce4581da39b975952905" target='_blank'>
              Abstract 5108: Decoding tumor intrinsic and stroma-related drug resistance in EGFR mutant NSCLC
              </a>
            </td>
          <td>
            Radhika Koranne, Aimee Gleason, Emily Hensley, Xiaodong Wang, Alba Font-Tello, Brian Rabe, Fayaz Seifuddin, Eugene Drokhlyansky, Kartik Subramanian, Martino Colombo, Jason R. Dobson, Mavrakis J. Konstantinos, Leila Dardaei, Aaron N. Hata
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b28cf36bb8a53d3cb31ec9abfc975b9a190e19" target='_blank'>
              CRISPR/Cas9 Screening Highlights PFKFB3 Gene as a Major Contributor to 5-Fluorouracil Resistance in Esophageal Cancer
              </a>
            </td>
          <td>
            Feng Xue, Hai Yang, Pengyan Xu, Shuman Zhang, N. Britzen-Laurent, Li-Li Bao, R. Grützmann, C. Krautz, Christian Pilarsky
          </td>
          <td>2025-05-12</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a21c97e108e69ab358170dee175b96d9645c472" target='_blank'>
              Cancer Dependency Map: Past and Present
              </a>
            </td>
          <td>
            Rand Arafeh, T. Shibue, Francisca Vazquez, William C Hahn
          </td>
          <td>2025-04-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="INTRODUCTION
Retinoblastoma (Rb) is a rare cancer, yet it is the most common eye tumor in children. It can occur in either a familial or sporadic form, with the sporadic variant being more prevalent, though its downstream effects on epigenetic markers remain largely unclear. Currently, the treatment for retinoblastoma typically involves aggressive chemotherapy and surgical resection. The identification of specific epigenetic characteristics of non-hereditary (sporadic) Rb has led to the development of advanced, high-throughput methods to explore its epigenetic profile. Our previous research demonstrated that treatment with the demethylating agent 5-Aza-2'-deoxycytidine (decitabine; DAC) induced cell cycle arrest and apoptosis in a well-characterized retinoblastoma model (WERI-Rb-1). Our analysis of time-dependent gene expression in WERI-Rb-1 cells following DAC exposure has led to the development of testable hypotheses to further investigate the epigenetic impact on the initiation and progression of retinoblastoma tumors.


METHODS
Gene expression analysis of publicly available datasets from patients' primary tumors and normal retina have been compared with those found in WERI-Rb-1 cells to assess the relevance of DAC-driven genes as markers of primary retinoblastoma tumors. The effect of DAC treatment has been evaluated in vivo, both in subcutaneous xenografts and in orthotopic models. qPCR analysis of gene expression and Methylation-Specific PCR (MSP) was performed.


RESULTS
Our analysis of network maps for differentially expressed genes in primary tumors compared to DAC-driven genes identified 15 hub/driver genes that may play a pivotal role in the genesis and progression of retinoblastoma. DAC treatment induced significant tumor growth arrest in vivo in both subcutaneous and orthotopic xenograft retinoblastoma models. This was associated with changes in gene expression, either through the direct switching-on of epigenetically locked genes or through the indirect regulation of linked genes, suggesting the potential use of DAC as an epigenetic anti-cancer drug for the treatment of retinoblastoma patients.


CONCLUSION
There is a pressing need to develop innovative treatments for retinoblastoma. Our research revealed that DAC can effectively suppress the growth and progression of retinoblastoma in in vivo models, offering a potential new therapeutic approach to battle this destructive disease. This discovery highlights the impact of this epigenetic therapy in reprogramming tumor dynamics, and thus its potential to preserve both the vision and lives of affected children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1396b8218887b23122c0972059e183badc67f6a1" target='_blank'>
              Epigenetic Reprogramming by Decitabine in Retinoblastoma.
              </a>
            </td>
          <td>
            Lisa Gherardini, Ankush Sharma, M. Taranta, C. Cinti
          </td>
          <td>2025-04-25</td>
          <td>Frontiers in bioscience</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="The effect of ubiquitous environmental conditions on mutational mechanisms, particularly loss of heterozygosity (LOH) remains poorly understood. Environment induced LOH can rapidly alter the genome and promote disease progression. Using mutation accumulation (MA) lines, we analysed the effect of ubiquitous environmental conditions on mutational mechanisms in a diploid hybrid (S288c/YJM789) baker’s yeast strain. These included blue light, low glucose (calorie restriction), oxidative stress (H2O2), high temperature (37°C), ethanol, and salt (NaCl). The frequency of LOH increased significantly in all environments including calorie restriction relative to the control (YPD). Interestingly, the percentage of the genome covered by LOH varied significantly depending on the condition. For example, the LOH tracts seen in calorie restriction conditions were significantly shorter than those observed in blue light exposure that rapidly homozygotized the genome. We also report a unique mutational signature of blue light exposure comprising LOH, small indels, large deletions and transversion mutations (G:C > T:A; G:C > C:G), with the latter likely to result from the photooxidation of guanine bases. Our results suggest ubiquitous environmental conditions cause LOH but result in distinct mutational signatures due to the type of damage induced and the pathways used to repair them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5a621f590a5a906aa35d1bcbbd1e7434261e118" target='_blank'>
              Loss of Heterozygosity associated with ubiquitous environments in yeast
              </a>
            </td>
          <td>
            Nikilesh Vijayan, Sameer Joshi, Praseetha Sarath, K. T. Nishant
          </td>
          <td>2025-05-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Acquired drug resistance is a main factor contributing to cancer therapy failure and high cancer mortality, highlighting the necessity to develop novel intervention targets. Circular RNAs (circRNAs), an abundant class of RNA molecules with a closed loop structure, possess characteristics including high stability, which provide unique advantages in clinical application. Growing evidence indicates that aberrantly expressed circRNAs are associated with resistance against various cancer treatments, including targeted therapy, chemotherapy, radiotherapy, and immunotherapy. Therefore, targeting these aberrant circRNAs may offer a strategy to improve the efficiency of cancer therapy. Herein, we present a summary of the most recently studied circRNAs and their regulatory roles on cancer drug resistance. With the advances in artificial intelligence (AI)-based bioinformatics algorithms, circRNAs could emerge as promising biomarkers and intervention targets in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7979b6f763a0f2eb2d4ff99ed47af16e914c8d95" target='_blank'>
              Circular RNAs modulate cancer drug resistance: advances and challenges
              </a>
            </td>
          <td>
            Jinghan Hua, Zhe Wang, Xiaoxun Cheng, Jiaojiao Dai, Ping Zhao
          </td>
          <td>2025-03-28</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary HNC cell lines exposed to nicotine for three months exhibited increased invasiveness and 1223 dysregulated genes. Integration with TCGA-HNSC data identified a Nic-HNC gene set of 168 genes (149 oncogenes, 19 tumor suppressors) as potential biomarkers, including 36 oncogenes overexpressed in heavy smokers. Pathway analysis revealed the upregulation of oncogenic signaling pathways (such as PI3K-AKT) and the suppression of immune responses (such as NF-κB signaling), contributing to tumor aggressiveness. Drug repurposing analysis across GDSC, CTRP, and PRISM databases identified five candidate compounds, with AZD1332 and JAK-8517 showing strong potential to counteract nicotine-driven oncogenes. These findings provide novel insights into nicotine-induced oncogenesis, propose potential therapeutic strategies, and support precision medicine approaches for tobacco-associated HNC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b967173ed137358de2e21ba69ea5bc0ababc5073" target='_blank'>
              Deciphering Nicotine-Driven Oncogenesis in Head and Neck Cancer: Integrative Transcriptomics and Drug Repurposing Insights
              </a>
            </td>
          <td>
            Guo-Rung You, Daniel Yu Chang, Hung-Han Huang, Yin-Ju Chen, J. T. Chang, Ann-Joy Cheng
          </td>
          <td>2025-04-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874bff63660c812dc6306123212ce70b86ccda05" target='_blank'>
              Abstract 6789: Identification of DGKA as a regulatory factor for paclitaxel resistance in lung cancer based on CRISPR screen and transcriptome sequencing
              </a>
            </td>
          <td>
            Jiang Luo, Mini Zhang, Jiawen Tang, Qingwei Wang, Yu Yuan, Min Xia, Yufei Yan, Yushuang Tan, Conghua Xie, Yan Gong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background About 40% of relapsed or non-responder tumors exhibit therapeutic resistance in the absence of a clear genetic cause, suggesting a pivotal role of intracellular communication. A deeper understanding of signaling pathways rewiring occurring in resistant cells is crucial to propose alternative effective strategies for cancer patients. Methods To achieve this goal, we developed a novel multi-step strategy, which integrates high sensitive mass spectrometry-based phosphoproteomics with network-based analysis. This strategy builds context-specific networks recapitulating the signaling rewiring upon drug treatment in therapy-resistant and sensitive cells. Results We applied this strategy to elucidate the BCR::ABL1-independent mechanisms that drive relapse upon therapy discontinuation in chronic myeloid leukemia (CML) patients. We built a signaling map, detailing - from receptor to key phenotypes - the molecular mechanisms implicated in the control of proliferation, DNA damage response and inflammation of therapy-resistant cells. In-depth analysis of this map uncovered novel therapeutic vulnerabilities. Functional validation in patient-derived leukemic stem cells revealed a crucial role of acquired FLT3-dependency and its underlying molecular mechanism. Conclusions In conclusion, our study presents a novel generally applicable strategy and the reposition of FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for CML relapsed patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02185-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff332b1ad425972fa586920b8d83716a55d5548" target='_blank'>
              A network-based approach to overcome BCR::ABL1-independent resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, F. Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, Massimo Breccia, Dimitrios Mougiakakos, Martin Böttcher, Thomas Fischer, Livia Perfetto, Francesca Sacco
          </td>
          <td>2025-04-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="The DNA damage response (DDR) is a multifaceted network of pathways that preserves genome stability1,2. Unravelling the complementary interplay between these pathways remains a challenge3,4. Here we used CRISPR interference (CRISPRi) screening to comprehensively map the genetic interactions required for survival during normal human cell homeostasis across all core DDR genes. We captured known interactions and discovered myriad new connections that are available online. We defined the molecular mechanism of two of the strongest interactions. First, we found that WDR48 works with USP1 to restrain PCNA degradation in FEN1/LIG1-deficient cells. Second, we found that SMARCAL1 and FANCM directly unwind TA-rich DNA cruciforms, preventing catastrophic chromosome breakage by the ERCC1–ERCC4 complex. Our data yield fundamental insights into genome maintenance, provide a springboard for mechanistic investigations into new connections between DDR factors and pinpoint synthetic vulnerabilities that could be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Constitutional mismatch repair deficiency (CMMRD) is a rare and highly aggressive cancer predisposition syndrome caused by biallelic germline mutations in mismatch repair genes. This condition is characterized by early-onset malignancies across multiple organ systems, including central nervous system tumors, hematological cancers, and gastrointestinal malignancies. CMMRD-associated tumors exhibit hypermutation and microsatellite instability, resulting in a high tumor mutation burden and rendering these malignancies responsive to immune checkpoint inhibitors (ICIs). ICIs targeting programmed cell death protein-1 and programmed cell death ligand 1 have demonstrated remarkable efficacy, particularly in hypermutated tumors, providing durable responses and improving survival outcomes. Advances in genetic and molecular diagnostics have enhanced the ability to identify CMMRD early, allowing for the implementation of comprehensive surveillance programs and improved management strategies. A multidisciplinary and individualized approach is essential for managing CMMRD patients. This review underscores the importance of early diagnosis, surveillance, and emerging therapeutic approaches to improve outcomes and quality of life for individuals and families affected by this devastating syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf20191865645f39f87803bf02b422fa40cafb91" target='_blank'>
              Constitutional Mismatch Repair Deficiency, the Most Aggressive Cancer Predisposition Syndrome : Clinical Presentation, Surveillance, and Management
              </a>
            </td>
          <td>
            Eungu Kang, Jin Kyung Suh, Sang-Dae Kim
          </td>
          <td>2025-04-29</td>
          <td>Journal of Korean Neurosurgical Society</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a1965c162640161a9b607e38387b4fb02d284d" target='_blank'>
              Abstract 2104: Epigenetic, transcriptomic and genetic analyses of HR+/HER2- primary and metastatic breast cancer results in cohort stratification by aggressiveness and reveals heterogeneous mechanisms of endocrine therapy resistance
              </a>
            </td>
          <td>
            Jomar Sangalang, Tony Sourisseau, V. Scott, Diep Tran, Nusaibah Ibrahimi, A. Tran-Dien, Hakim Benkirane, N. Servant, Fabrice André, Sergey I. Nikolaev
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daad54b0fcb7c5c704adea069e134ecf7c7a856b" target='_blank'>
              Abstract 1252: Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy
              </a>
            </td>
          <td>
            Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, C. Lamy, Fabrice Lécuru, Christophe Le Tourneau, Céline Vallot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) harboring BCR/ABL-T315I mutation has been a challenging obstacle for targeted therapy due to the acquired resistance to tyrosine kinase inhibitor (TKI)-based therapy. Thus, it is especially urgent to investigate more effective therapeutic targets to overcome T315I-induced resistance. Here, we reported that BCR/ABL-T315I mutant CML cells possessed a long-term proliferative capacity and tolerance to metabolic stress. Importantly, we also found that selenoamino acid metabolism was increased in the bone marrows of BCR/ABL-T315I patients compared with non-T315I patients by GSEA from RNA-Seq data. Indeed, GPX1 was highly expressed in T315I mutant cells, while knockout of GPX1 significantly suppressed cell proliferation and triggered apoptosis under glucose-deprived condition. GPX1 knockout showed decreased cell metabolism signaling as well as mitochondrial gene expression by RNA-Seq. Mechanistically, GPX1 maintained mitochondrial activity and oxygen consumption rate (OCR), retaining mitochondrial redox homeostasis and oxidative phosphorylation (OXPHOS). Additionally, mercaptosuccinic acid (MSA), a GPX inhibitor, inhibited CML colony formation and induced cell apoptosis under glucose-free condition. Therefore, GPX1 is a promising therapeutic target to overcome drug resistance induced by the T315I mutation, which provides a novel approach for BCR/ABL-T315I CML treatment by disturbing mitochondrial OXPHOS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9003b9be433ceb088a877bba6ada889eb7e9e1" target='_blank'>
              GPX1 confers resistance to metabolic stress in BCR/ABL-T315I mutant chronic myeloid leukemia cells
              </a>
            </td>
          <td>
            Jun-Dan Wang, Jin-Xing Wang, Zhi-Li Lin, Na Xu, Ling Zhang, Jia-Jun Liu, Rui Gao, Z. Long
          </td>
          <td>2025-05-09</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Lynch Syndrome (LS) is an autosomal dominant disease characterized by germline heterozygous mutations in DNA mismatch repair (MMR) genes. High-risk LS patients may proceed to colorectal cancer (CRC). However, the drivers or biomarkers of LS benign colon tissue approaching malignant CRC are not completely understood. The similarity and difference between LS-related and nonLS-related CRC are also not well interrogated (LS-CRC vs. nonLS-CRC). This study aimed to understand the cellular changes during malignant transition in LS. Methods Single-cell RNA sequencing (scRNA-seq) was used to analyze paired biopsy samples from 3 patients with LS (cancer tissues vs. adjacent normal tissues, n=3). Single-nuclear RNA sequencing (snRNA-seq) was used to analyze a frozen biopsy sample from a patient with LS (n=1). scRNA-seq or snRNA-seq datasets from CRC were downloaded from the open source. Integrative computational analysis was performed to conclude the distinct pattern in the single-cell atlas. Immuno-histo-fluorescence staining (IHF) were also performed for three key markers. Results In the single-cell atlas, we observed an increase in primitive cancer stem-cell-like cells with high expression of a general cancer biomarker CarcinoEmbryonic Antigen-related Cell Adhesion Molecule 5 (CEACAM5) in the epithelium of the LS. Infiltration of immune cells and DNA repair biological activity are dramatically increased in LS carcinoma. The burden in LS is fundamentally elevated compared to that in CRC or healthy donors when the mutations are in the coding-sequence-wide range. Furthermore, T cell and macrophage-related tumor immunity is readily mobilized in carcinomas compared to paraCArcinomas. Conclusions This study provides single-cell transcriptomic resource using affected tissues from patients with Lynch Syndrome and describes an integrative profile covering the alterations (cancer stem cell markers, mutation burden, and tumor immunity) during the malignant transition from healthy to Lynch Syndrome and to colorectal cancer at the single-cell level. Highlights and Figure/Table Index (Take-Home Messages) Colon tissues from HD (n=4), patients with LS (n=6) and with CRC (n=3) were collected for snRNA-seq or scRNA-seq analysis; Diagnosis of the 6 LS patients are clearly supported by our pedigree documentations (Fig. 1 and Supp. Fig. 1) and briefed in Table 1; Among the 6 patients with LS, 1 for snRNA-seq analysis (a carcinoma tissue only), 3 for scRNA-seq analysis (3 paired para-carcinoma and carcinoma tissues) and 3 for experimental validations; (Fig. 1) Following studies are focused on the comparisons between HD, LS and CRC; In addition, comparisons between paired carcinoma and para-carcinoma tissues from patients with LS were also performed; (Fig. 2-7) When HD and LS are compared, a cancer stem cell marker CEACAM5 is readily detected by snRNA-seq analysis; (Fig. 5) When carcinoma and para-carcinoma from LS patients are compared, increased immune cell infiltration of and enhanced DNA repair activity in the malignant tissues is readily detected by scRNA-seq analysis; in addition, three upregulated markers in LS carcinoma (BACE2, GRPC5A and OLFM4) were identified and validated by immunohistoinfloresnce; (Fig. 3 and Fig. 4) SComatic algorithm-based mutation calling analysis suggests a comparable mutation burden between carcinoma and para-carcinoma from LS patients, but such mutation burdens are grossly greater than that from HD or that from patients with CRC; (Fig. 6)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c44a229dbcaa552459faeccde9844a06701be" target='_blank'>
              Tracing the stemness and malignant transition in a heritable colorectal cancer Lynch Syndrome by single-cell RNA-seq analysis
              </a>
            </td>
          <td>
            Junfeng Xu, Yuhang Li, Zhiqin Wang, Qianru Li, Aijun Liu, Jianqiu Sheng, Ge Dong, Lang Yang, Zhigang Cai
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Germline variants in the HSD3B1 gene influence response to hormone therapies. This study examined the “adrenal-permissive” allele (c.1100 C) and its impact on various somatic features in prostate tumors. Upon inferring HSD3B1 variant status from somatic DNA sequencing data in a cohort of 6550 primary and metastatic prostate tumors, we found that tumors harboring the adrenal-permissive CC genotype, as compared to the adrenal-restrictive AA genotype, had limited somatic changes in their exomes or transcriptomes. However, significant differences were observed when we examined immune regulatory pathways and immune cell populations. This was accompanied by differential gene expression of immunoglobulins, MHC class I and II molecules, and druggable cell surface genes. These features appeared to deviate across different metastatic biopsy sites. Altogether, while tumor-intrinsic features are not regulated by germline HSD3B1 c.1100 alleles, these genotypes instead may impact cell–cell interactions and the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5953257502a6a5fc55756fab20f962c1af1666a8" target='_blank'>
              The Influence of the Germline HSD3B1 Adrenal-Permissive Allele (c.1100 C) on the Somatic Alteration Landscape, the Transcriptome, and Immune Cell Infiltration in Prostate Cancer
              </a>
            </td>
          <td>
            Samuel Kellen, Allison Makovec, Carly D Miller, Shayan Nazari, Andrew Elliott, Aiden Deacon, Emily John, Nikitha Vobugari, N. Agarwal, Rana R McKay, Pedro C. Barata, Charles J. Ryan, Nima Sharifi, Justin H. Hwang, E. S. Antonarakis
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16cb06677804f48c2ac85f36bdf345c2b4df6264" target='_blank'>
              Abstract 431: Novel WRN drug resistant CDX models
              </a>
            </td>
          <td>
            Haiting Dai, Tiejun Bing, Huayi Qu, Huawei Pan, Wen-Jen Yu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe1c8ab3660cf8e9ea6efb51c5232619240b0082" target='_blank'>
              The G protein-coupled receptor GPR89A is a novel potential therapeutic target to overcome cisplatin resistance in NSCLC Calu1 cells.
              </a>
            </td>
          <td>
            Hale Guler Kara, Eda Doğan, V. Bozok, Ç. Aktan, E. Cakiroglu, Z. Eroglu, Duygu Dondu Aygunes Jafarı, Kemal Sami Korkmaz, Serif Senturk, B. Kosova
          </td>
          <td>2025-04-17</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5a495cc4677d82bf911302787442a5796efaf4" target='_blank'>
              Abstract 1818: Enhancing double-strand break induction in hypoxic glioblastoma cells through dNTP modulation and its potential therapeutic implications
              </a>
            </td>
          <td>
            E. Usoro, Dominique Monroe, Kristen Carver, Emily Steinbeck, Arilyn Williams, Waaqo Daddacha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Adenosine deaminases acting on RNA (ADAR) enzymes constitute a natural cellular mechanism that induces A-to-I(G) editing, introducing genetic changes at the RNA level. Recently, interest in the endogenous-ADAR editor has emerged for correcting genetic mutations, consisting of a programmed oligonucleotide that attracts the native ADAR, thereby offering opportunities for medical therapy. Here, we systematically chart the scope of cancer mutations that endogenous-ADAR can correct. First, analyzing germline single nucleotide variants in cancer predisposition genes, we find that endogenous-ADAR can revert a fifth of them, reducing the risk of cancer development later in life. Second, examining somatic mutations across various cancer types, we find that it has the potential to correct at least one driver mutation in over a third of the samples, suggesting a promising future treatment strategy. We also highlight key driver mutations that are amenable to endogenous-ADAR, and are thus of special clinical interest. As using endogenous-ADAR entails delivering relatively small payloads, the prospects of delivering endogenous-ADAR to various cancers seem promising. We expect that the large scope of correctable mutations that are systematically charted here for the first time will pave the way for a new era of cancer treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a024daf32faffd9eaf4adfd22fdca5d40c18746f" target='_blank'>
              A systematic evaluation of the therapeutic potential of endogenous-ADAR editors in cancer prevention and treatment
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Ariel Dadush, Sumeet Patiyal, P. Rajagopal, Gulzar Daya, Shay Ben-Aroya, A. Schäffer, Eli Eisenberg, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aba9f5d2cf8bd1341acacd8ce88421b1a228c70" target='_blank'>
              Abstract 5547: High resolution target identification by unbiased forward genetics
              </a>
            </td>
          <td>
            Moritz Horn, Georg Michlits, Andreia Rocha, Christoph Stelzer, Josef M. Penninger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J. Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335901b6f2e363bcd4c769050615cefdadb15cb6" target='_blank'>
              Mechanistic Insights into Proteomic Mutation-Phenotype Linkages from Tiling Mutagenesis Screens
              </a>
            </td>
          <td>
            Wei He, Jen-Wei Huang, Yalong Wang, Samuel B. Hayward, G. Leuzzi, Rongjie Fu, Shuyue Wang, Alina Vaitsiankova, Mark T. Bedford, Raphaël Guérois, Alberto Ciccia, Han Xu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Radiotherapy is a cornerstone of colorectal cancer (CRC) treatment; however, its therapeutic efficacy is often compromised by both intrinsic and acquired resistance in CRC cells. This study employed small interfering RNA (siRNA) technology to elucidate the functional roles of BAMBI, GADD34, NFKBIA, and NFKBID in CRC cell lines SW480 and HCT116. We assessed their impact on key cellular processes and radiation sensitivity. Gene silencing of all four target genes significantly suppressed CRC cell proliferation, migration, and invasion. Moreover, siRNA-mediated knockdown enhanced radiation sensitivity, as evidenced by a substantial increase in apoptosis and a marked reduction in cell viability compared with controls. These findings suggest that BAMBI, GADD34, NFKBIA, and NFKBID serve as critical regulators of CRC progression and radiation resistance. Overall, this study provides a mechanistic foundation for further exploration into the pathways underlying radiation resistance and underscores the potential for developing personalized radiotherapy strategies guided by molecular profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b02786eb68733b852aeed62e7695e4393e02cbb" target='_blank'>
              Mechanistic Insights into Radiation Resistance in Colorectal Cancer: Gene Exploration Study
              </a>
            </td>
          <td>
            Beamjun Park, Soohyeon Lee, Inyoung Jo, Donghyun Kang, Taewan Kim, Jaesung Ryu, Hyejeong Kong, Moojun Baek, Taesung Ahn
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background Resistance to radiation therapy is a common challenge in the field of oncology. Cancer cells with an increased ability to effectively repair DNA or cells with higher levels of antioxidants are more resistant to radiation. As cancer cells rely on these traits for survival, they may offer vulnerabilities that could be exploited. Methods In the current study, rectal cancer organoids that showed different responses to radiation treatment were identified. RNA sequencing was used to compare radioresistant and radiosensitive organoids. In vitro combination drug screens were performed. The selection of drugs was guided by the RNA sequencing results. Results Radioresistant organoids exhibited superior transcriptional adaptability and activated more DNA repair pathways when irradiated. Additionally, radioresistant organoids displayed enhanced antioxidant metabolism, including pathways related to the detoxification of reactive oxygen species and the synthesis of glutathione. Combinatorial drug screens identified the combination of RRx-001 (an inducer of oxidative stress) with GCLC inhibitor BSO as a highly effective and synergistic drug combination in killing radioresistant organoids. CRISPR-CAS-mediated knockout of GCLC sensitised organoids to RRx-001. Conclusion Combining RRx-001 with the inhibition of GCLC may be a promising alternative treatment strategy in radioresistant rectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc4a68bb9eb32114e3b611e03801c7a052fee55" target='_blank'>
              Rational design of alternative treatment options for radioresistant rectal cancer using patient-derived organoids
              </a>
            </td>
          <td>
            D. Andel, A. J. Nouwens, S. Klaassen, J. Laoukili, B. Viergever, A. Verheem, M. P. W. Intven, M. Zandvliet, J. Hagendoorn, I. Borel Rinkes, O. Kranenburg
          </td>
          <td>2025-04-10</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/176561a82959305826a2da9c2604661ae413f694" target='_blank'>
              Abstract 2900: Musashi-1 regulated oncogenic switch enhances therapy resistance to DNA damage in glioblastoma
              </a>
            </td>
          <td>
            Xian Liu, Tsoi Pok Kong, Man Ding Wong, Liang Ting Lin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c1d77f1a7adf30fbae49c91b8912428daef56f" target='_blank'>
              CircRNA_103809/miR-516a/FBXL18 contributes to stemness and Gemcitabine Resistance of bladder cancer cells
              </a>
            </td>
          <td>
            Yu Yang, Feng Wang, Weiqiang Ning, Yi Ding, Wei Chen, Zhixian Yu
          </td>
          <td>2025-04-20</td>
          <td>Archives of Medical Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9a2666f037bf98ff78850cacfb5af10e4de653" target='_blank'>
              Abstract 1338: System biological approach identifies heterochromatin modification as an important mechanism of acquired cancer radioresistance
              </a>
            </td>
          <td>
            Chaw Yee Beh, Sharon Pei Yi Chan, Celestia Pei Xuan Yeo, Boon Hao Hong, Evelyn Mui Cheng Tan, Dewi Susanti, Pei Ling Tai, Eugenia Li Ling Yeo, Khee Chee Soo, Edward Kai-Hua Chow, Melvin Lee Kiang Chua
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cf5adc7adc0a7d23f75264732a2a440d4be0c25" target='_blank'>
              Single-Cell eQTL Mapping Reveals Cell Type-Specific Genetic Regulation in Lung Cancer
              </a>
            </td>
          <td>
            Yating Fu, Yi Wang, Chang Zhang, Chenying Jin, Jiaying Cai, Linnan Gong, Chenying Jin, Chen Ji, Yuanlin Mou, Caochen Zhang, Shihao Wu, Xinyuan Ge, Yahui Dai, Sunan Miao, Huimin Ma, Xiaoyang Ma, Mengping Wang, Lijun Bian, Erbao Zhang, J. Dai, Zhibin Hu, Guangfu Jin, Meng Zhu, Hongbing Shen, Hongxia Ma
          </td>
          <td>2025-04-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3847b0cc02af732174f2aba89f84c63f9e5ea11" target='_blank'>
              Proof-of-Concept of A DNA-Based Recording System for High-Throughput Functional Gene Screening
              </a>
            </td>
          <td>
            Shoichi Kato, Atsushi Ikemoto, Jun Isayama, Tetsuya Takimoto, Hideyuki Saya, Ken-ichi Hamada
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b82c7744ef91d882615ad02c557cbabf06a4d73" target='_blank'>
              Updates on DNA Repair Gene Deficiency in Colorectal Cancer (dMMR)
              </a>
            </td>
          <td>
            Gustavo Sevá-Pereira, C. S. R. Coy, Carlos Augusto Real Martinez
          </td>
          <td>2024-11-01</td>
          <td>Journal of Coloproctology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1079d87207d8519780da663c4b3230cdeb5ebf5" target='_blank'>
              Abstract 6785: MicroRNA-lincrna cross-talk in BRCA1-mutated breast cancer and its therapeutic implications
              </a>
            </td>
          <td>
            O. Fadebi, Zodwa Dlamini, R. Marima
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b575dc8351428b4d22b9c3382531e97fc82f1bcf" target='_blank'>
              Abstract 545: RFX5 knockdown restores sensitivity to cellular cyclophosphamide in triple-negative breast cancer cell lines
              </a>
            </td>
          <td>
            Elizabeth K Molchan, Kathleen Streeks, Alyssa Ohaegbulam, Mohammed Gbadamosi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In cancer, mutational processes act in concert with epigenetic reprogramming to endow malignant cells with hallmark properties that underpin tumorigenesis. Compared with the relatively rigid and slow processes of genetic evolution, the plastic nature of chromatin enables cells to adapt to a changing environment more rapidly. Multiple myeloma is characterised by high levels of inter- and intra-patient heterogeneity at both the genetic and epigenetic levels. Understanding the many layers of genetic and non-genetic evolution and their interplay is crucial to improve patient outcomes. In this short review, we discuss the most common and extensively characterised epigenetic alterations that occur during myeloma development. We also touch on emerging approaches to reverse the aberrant epigenome of myeloma cells as a treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb9ecdf7e50a09e0efbdf19061d3f2e07fadfcb" target='_blank'>
              Epigenetic reprogramming in multiple myeloma-Challenges and opportunities.
              </a>
            </td>
          <td>
            Subhasree Kumar, Lev M. Kats, Emily Gruber
          </td>
          <td>2025-04-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16855fcdba1a65e13a1196ac0779031e3a7e82b5" target='_blank'>
              DNA repair under heat: DNA Polymerase λ modulates heat stress-induced mutagenesis in plants
              </a>
            </td>
          <td>
            Clair M. Wootan, John Lutterman, Nathan Springer, Xiaosa Xu, Feng Zhang
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Homologous recombination between short repeated sequences, such as Alu sequences, can generate pathogenic chromosomal rearrangements. We used budding yeast to measure homologous recombination between short repeated his3 sequences located on non-homologous chromosomes to identify pathways that suppress spontaneous and radiation-associated translocations. Previous published data demonstrated that genes that participate in RAD9-mediated G2 arrest, the S phase checkpoint, and recombinational repair of double-strand breaks (DSBs) suppressed ectopic recombination between small repeats. We determined whether these pathways are independent in suppressing recombination by measuring frequencies of spontaneous recombination in single and double mutants. In the wild-type diploid, the rate of spontaneous recombination was (3 ± 1.2) × 10−8. This rate was increased 10–30-fold in the rad51, rad55, rad57, mre11, rad50, and xrs2 mutants, seven-fold in the rad9 checkpoint mutant, and 23-fold in the mec1-21 S phase checkpoint mutant. Double mutants defective in both RAD9 and in either RAD51, RAD55, or RAD57 increased spontaneous recombination rates by ∼40 fold, while double mutants defective in both the MEC1 (ATR/ATM ortholog) and RAD51 genes increased rates ∼100 fold. Compared to frequencies of radiation-associated translocations in wild type, radiation-associated frequencies increased in mre11, rad50, xrs2, rad51, rad55 and rad9 rad51 diploid mutants; an increase in radiation-associated frequencies was detected in the rad9 rad51 diploid after exposure to 100 rads X rays. These data indicate that the S phase and G2 checkpoint pathways are independent from the recombinational repair pathway in suppressing homology-directed translocations in yeast.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c442344c7555fd746bb68962a7be775830f91d7" target='_blank'>
              Checkpoint and recombination pathways independently suppress rates of spontaneous homology-directed chromosomal translocations in budding yeast
              </a>
            </td>
          <td>
            Li Zeng, Mingzeng Sun, M. Fasullo
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abnormal cellular respiration plays a critical role in carcinogenesis. However, the molecular mechanisms underlying dysregulation of respiratory gene expression across different cancer types remain unclear. Here, we developed a computational framework that provides an analytical approach for exploring the molecular alterations and clinical relevance of respiratory genes in pan-cancer. We identified a total of 53 gene signatures in the three stages of respiration (including glycolysis, tricarboxylic acid cycle, and oxidative phosphorylation) through this framework and found that they were broadly differentially expressed and genetically altered across 33 cancer types. Pathway analysis manifested that the expression levels of almost all respiratory gene signatures were remarkably associated with the activation or inhibition of numerous oncogenic pathways, such as metabolism, angiogenesis, cell proliferation, and apoptosis. Survival analysis highlighted the oncogenic or tumor suppressor potential of the respiratory gene signatures. In particular, VCAN has shown significant oncogenic features in multiple cancer types. Finally, we identified a number of respiratory gene signatures that could be potential therapeutic targets, including VCAN. We also predicted small-molecule compounds targeting respiratory gene signatures or components of pathways regulated by them. Overall, our comprehensive analysis has greatly enhanced the understanding of molecular alterations of respiratory genes in tumorigenesis and progression, and provided insights into developing new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e93642cceee2b8d761d0dc72e72474a146b2e" target='_blank'>
              Comprehensive characterization of respiratory genes based on a computational framework in pan-cancer to develop stratified treatment strategies
              </a>
            </td>
          <td>
            Caiyu Zhang, Chenyu Liu, Zhuoru Wang, Di Wang, Wenli Chen, Jian Li, Qianyi Lu, Jiajun Zhou, Yetong Chang, Peng Wang, Yue Gao, Junwei Wang, Hui Zhi, Shangwei Ning
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10fbe2acee93fcc04d4cf719859f00b1fc38a341" target='_blank'>
              Megabase-scale human genome rearrangement with programmable bridge recombinases
              </a>
            </td>
          <td>
            Nicholas T. Perry, Liam J. Bartie, Dhruva Katrekar, Gabriel A. Gonzalez, Matthew G. Durrant, James J. Pai, Alison Fanton, Masahiro Hiraizumi, Chiara Ricci-Tam, Hiroshi Nishimasu, Silvana Konermann, Patrick D. Hsu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919d6be56b49f799484f0a056c28802836f4c288" target='_blank'>
              Abstract 2771: CRISPR/Cas9 essentiality screens identify stand-alone and IRF4-linked genetic vulnerabilities in melanoma cells
              </a>
            </td>
          <td>
            U. Sobhiafshar, N. T. Emre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c23f23708c35f99383a4f766adfd798e0bd378" target='_blank'>
              Abstract 6636: Multi-site evaluation of expanded solid tumor, pan-cancer reference material
              </a>
            </td>
          <td>
            Dana J. Ruminski Lowe, Patricia Helsel, Praveena Kamineni, Edward S. Davis, Andrew Anfora, Yves Konigshofer, Catherine Huang, R. Garlick, B. Anekella
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b05e86f87ff9d548ca0d97272bc2384f88e53a2" target='_blank'>
              Abstract 3643: AI driven discovery of novel synthetic lethal gene pairs for targeted cancer therapy
              </a>
            </td>
          <td>
            Amalia Soenens, Daniel S. Miller, Oliver Vipond, Florencia Skuras, Alfie Brennan, Anna Hercot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ab08c733369ba98ccceec430e609487bfe090d5" target='_blank'>
              Abstract 1247: Characterizing the evolution of osimertinib resistance in EGFR mutated non-small cell lung cancer
              </a>
            </td>
          <td>
            Mina N. Dinh, Jinling Wu, Masahiro Hitomi, Maximilian Strobl, Jacob G. Scott
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892d32d1ea696e84aa4f250bf0edeb5b5584501c" target='_blank'>
              CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes
              </a>
            </td>
          <td>
            Sam Ghatan, Jasper Panten, Winona Oliveros, Neville E. Sanjana, John A Morris, Tuuli Lappalainen
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05df4b99ec2d4ca5afa53e36017d7bbb5db0a668" target='_blank'>
              Functional Genomic Screens Reveal RBBP4 as a Key Regulator of Cell Cycle Progression in TMZ-Resistant Glioblastoma
              </a>
            </td>
          <td>
            Ezgi Yagmur Kala, Filiz Senbabaoglu Aksu, Erdem Ercan, Altar Ozbıyık, Ahmet Cingöz, Ozlem Yedier-Bayram, A. C. Aksu, Ayse Derya Cavga, E. Yilmaz, I. Kok, A. Humeyra, Dur Karasayar, Ibrahim Kulac, Hamzah Syed, M. Philpott, AP Cribbs, T. Bagci-Onder
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532359eb783a00c7e5a0af7f6a8c8009cbbb7c55" target='_blank'>
              Abstract 2758: Methylome plasticity as a biomarker of treatment response in small cell lung cancer
              </a>
            </td>
          <td>
            D. Sacdalan, Sami Ul Haq, L. Zhan, J. J. Li, V. Philip, Scott V. Bratman, Geoffrey Liu, Benjamin H. Lok
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421aa7dc86b3e3526576ee75dc0634ab5c74725" target='_blank'>
              Harnessing the Microbiome: CRISPR-Based Gene Editing and Antimicrobial Peptides in Combating Antibiotic Resistance and Cancer.
              </a>
            </td>
          <td>
            R. A. Amen, Yaser M Hassan, R. A. Essmat, Rana H Ahmed, Marwan M Azab, Nadia R Shehata, Mariam M Elgazzar, Wael M. El-Sayed
          </td>
          <td>2025-05-16</td>
          <td>Probiotics and antimicrobial proteins</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most prevalent malignancies worldwide. With the increasing incidence of CRC, there is an urgent need for effective strategies for early diagnosis and treatment. Circadian rhythm, a natural biological clock, regulates various physiological processes, and its disruption has been implicated in the onset and progression of cancer. However, the specific roles of circadian rhythm-related genes (CRDGs) in CRC remain unclear. Methods In this study, we analyzed the expression patterns of CRDGs in CRC using single-cell RNA sequencing (scRNA-seq) data and bulk RNA sequencing data from the GSE178318 dataset. We constructed a CRC prognostic model based on CRD scores. Additionally, we explored the potential mechanisms of CRDGs in tumor progression through weighted gene co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA), and assessed their impact on the response to immune checkpoint inhibitors. Results The analysis revealed that CRDGs were significantly upregulated in liver metastasis samples compared to primary CRC samples and were closely associated with several metabolic and immune-related pathways. The prognostic model based on CRD scores indicated that higher CRD scores were associated with poorer outcomes in immunotherapy. These findings were further validated in multiple datasets, underscoring the potential of CRDGs as prognostic indicators in CRC. Conclusion This study systematically reveals, for the first time, the expression characteristics of CRDGs in CRC and their relationship with tumor progression and response to immunotherapy. CRDGs may serve as effective prognostic biomarkers and therapeutic targets, offering new strategies for the personalized treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b7699b45efdb0dd0f3e56012bf04dabfd7ec73" target='_blank'>
              Integration of scRNA-Seq and Bulk RNA-Seq Identifies Circadian Rhythm Disruption-Related Genes Associated with Prognosis and Drug Resistance in Colorectal Cancer Patients
              </a>
            </td>
          <td>
            Yong Tao, Jun Li, Jianhui Pan, Qing Wang, Ru Ke, D. Yuan, Hongbiao Wu, Yuepeng Cao, Lei Zhao
          </td>
          <td>2025-04-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844d851fd9820543dfb4e679c6a5eddb01a41ffd" target='_blank'>
              Abstract 3311: Uncovering methylation signatures and pathways associated with resistance to anti-EGFR therapy in colorectal cancer
              </a>
            </td>
          <td>
            Ana Regina de Abreu, Yann Heylen, J. Ibrahim, Marc Peeters, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a986ae0bdffed64d9a4eefeb701fd4d7309816c" target='_blank'>
              Abstract SY14-02: Mapping synthetic lethal interactions at scale: A functional genomics approach to cancer therapeutic discovery
              </a>
            </td>
          <td>
            Francisca Vazquez
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c037285ee33cfd3dd5fa1bd1a95f79d26559f4b5" target='_blank'>
              Abstract 813: In vitro and in vivo modelling of drug tolerance and minimal residual disease to discover effective therapeutic combination strategies for EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Kristyna A. Gorospe, Ming Li, A. Arivajiagane, Nhu-An Pham, R. Navab, Kelsie L. Thu, M S Tsao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae249d8a2eee39a7ac3a222a7e9c11de20842930" target='_blank'>
              Abstract 505: Differences in gene expression across races in pediatric T-cell acute lymphoblastic leukemia (T-ALL)
              </a>
            </td>
          <td>
            Rishabh Gaur, I. Pushel, M. Farooqi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08272a17f03be6497869a3622bf7e66968e9d5bd" target='_blank'>
              Abstract 219: Uncovering synergistic and antagonistic interactions in epigenetic therapeutics using perturb-seq
              </a>
            </td>
          <td>
            Changde Cheng, Rui Lu, Sajesan Aryal, Doug A. Welsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55dbe0088f239e1966f0ecbd4169616a69ee9add" target='_blank'>
              Abstract 5673: Identification of potential therapeutic targets by in vivo CRISPR screening on xenograft models
              </a>
            </td>
          <td>
            Ce Zhong, Lu Bai, Tongtong Wang, Yiying Shao, Lingyun Zhang, Lu Zhang, Jingqi Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent research has identified a potential protective effect of higher numbers of circulating lymphocytes on colorectal cancer (CRC) development. However, the importance of different lymphocyte subtypes and activation states in CRC development and the biological pathways driving this relationship remain poorly understood and warrant further investigation. Specifically, CD4+ T cells -- a highly dynamic lymphocyte subtype -- undergo remodelling upon activation to induce the expression of genes critical for their effector function. Previous studies investigating their role in CRC risk have used bulk tissue, limiting our current understanding of the role of these cells to static, non-dynamic relationships only. Methods Here, we combined two genetic epidemiological methods -- Mendelian randomisation (MR) and genetic colocalisation -- to evaluate evidence for causal relationships of gene expression on CRC risk across multiple CD4+ T cell subtypes and activation stage. Genetic proxies were obtained from single-cell transcriptomic data, allowing us to investigate the causal effect of expression of 1,805 genes across five CD4+ T cell activation states on CRC risk (78,473 cases; 107,143 controls). We repeated analyses stratified by CRC anatomical subsites and sex, and performed a sensitivity analysis to evaluate whether the observed effect estimates were likely to be CD4+ T cell-specific. Results We identified six genes with evidence (FDR-P<0.05 in MR analyses and H4>0.8 in genetic colocalisation analyses) for a causal role of CD4+ T cell expression in CRC development -- FADS2, FHL3, HLA-DRB1, HLA-DRB5, RPL28, and TMEM258 . We observed differences in causal estimates of gene expression on CRC risk across different CD4+ T cell subtypes and activation timepoints, as well as CRC anatomical subsites and sex. However, our sensitivity analysis revealed that the genetic proxies used to instrument gene expression in CD4+ T cells also act as eQTLs in other tissues, highlighting the challenges of using genetic proxies to instrument tissue-specific expression changes. Conclusions Our study demonstrates the importance of capturing the dynamic nature of CD4+ T cells in understanding disease risk, and prioritises genes for further investigation in cancer prevention research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3854c3b38125ddae1dd0d632aa93ab78f073f33f" target='_blank'>
              Transcriptome-wide Mendelian randomisation exploring dynamic CD4+ T cell gene expression in colorectal cancer development
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A. Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, A. Lindblom, Shuji Ogino, Veronika Vymetálková, A. Wolk, Anna H. Wu, J. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E. Thomas, Rish K Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-04-17</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-17</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701bc66d5a290edadb747fc58b7aca8807e79e86" target='_blank'>
              Mutational signature catalogue (MUSIC) of DNA double-strand break repair
              </a>
            </td>
          <td>
            Marco Barazas, R. van Schendel, M. Tijsterman
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1834a1d0667a17db6a7427294528dfb9ae37c" target='_blank'>
              Abstract 4358: Single nucleotide polymorphisms located in PK/PD pathway genes of standard AML chemotherapy regimen predicts outcome in pediatric AML
              </a>
            </td>
          <td>
            Aneesha Nath, Richard J Marrero, Xueyuan Cao, J. Rubnitz, S. Pounds, J. Lamba
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ddc9edbf2bf1f968d70d06263a337e58c74158" target='_blank'>
              Abstract 4498: Drug screens identify new therapeutic targets that synergize with EZH1/2 inhibition in adult T-cell leukemia/lymphoma
              </a>
            </td>
          <td>
            Noor B Ghanem, C. Ip, Kit I Tong, Mehran Bakhtiari, Michael Y. He, A. Schimmer, R. Kridel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="How do cancer cells respond to their environment, and what are the key regulators behind their behaviour? While immune cell reprogramming in the tumour microenvironment (TME) has been extensively studied, the dynamic regulatory changes within cancer cells in response to interactions with immune cells remain poorly understood. In Chronic Lymphocytic Leukaemia (CLL), this knowledge gap limits our ability to fully grasp the disease progression and to design effective, personalised interventions. To tackle this, we combine time-series transcriptomics with data-driven gene regulatory network (GRN) inference to uncover the temporal regulatory mechanisms driving CLL cell behaviour within a reconstituted in vitro TME. Using cultures of peripheral blood from CLL patients or of purified patient-derived CLL cells, we profile gene expression across five time points spanning 14 days under these experimental conditions. By inferring GRNs from transcription factor activity, we capture patient-specific and temporally resolved regulatory interactions that highlight how immune signals shape cancer cell phenotypic changes. Our network analysis reveals distinct gene modules associated with critical processes such as cytokine signalling, metabolic reprogramming and differentiation, hallmarks of immune-cancer cell interaction. Intriguingly, we found that while the presence of immune cells in the environment significantly alters CLL cell activation, their survival trajectories are predominantly governed by intrinsic features. This study not only offers mechanistic insights into how immune cell presence influences CLL cell fate but also presents a robust computational framework for integrating time-series transcriptomics with GRN inference, which can then be used to study the long-term behaviour of the CLL cells through dynamical modelling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46422bf8177dcfdbb03604f9382eef5f720cbd11" target='_blank'>
              Time-series RNA-Seq and data-driven network inference unveil dynamics of cell activation, survival and crosstalk in Chronic Lymphocytic Leukaemia in vitro models
              </a>
            </td>
          <td>
            Malvina Marku, Hugo Chenel, J. Bordenave, Marcelo Hurtado, Marcin Domagala, Flavien Raynal, M. Poupot, Loïc Ysebaert, Andrei Zinovyev, Vera Pancaldi
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4063f1f916968534cd4c71fd92b3b7e211345625" target='_blank'>
              Abstract 4404: BRPF1 inhibition restores chemosensitivity in triple negative breast cancer via ribosomal pathway modulation
              </a>
            </td>
          <td>
            Ozlem Yedier Bayram, Ahmet Cingöz, E. Yilmaz, A. C. Aksu, B. Esin, N. Pınarbaşı-Değirmenci, Ayse Derya Cavga, Beyza Dedeoğlu, B. Cevatemre, Hamzah Syed, M. Philpott, A. Cribbs, Udo Opperman, Nathan A. Lack, Ceyda Açılan Ayhan, T. Onder, Tugba Bagci Onder
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631764abf56c15adfa48bcb7f3a761557f103393" target='_blank'>
              Abstract 444: ASXL1 mutations in AML are associated with a distinct epigenetic state that results in vulnerabilities to epigenetic-targeted agents
              </a>
            </td>
          <td>
            Christopher P. Mill, W. Fiskus, Christine E. Birdwell, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, K. Sasaki, S. Loghavi, T. Kadia, Naval G Daver, C. Dinardo, Lu Wang, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8bc96152bbbb1bdeac5241e7bd6de510c434bdc" target='_blank'>
              Abstract 454: Leveraging cell cycle to enhance differentiation therapy in acute myeloid leukemia
              </a>
            </td>
          <td>
            Riccardo Valzelli, Kourosh Hayatigolkhatmi, Amir Hosseini, E. Ceccacci, Giorgia Parodi, A. Bachi, S. Minucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5520257da55eb20975e2b76db4a10f023d4be1" target='_blank'>
              Abstract 5543: High resolution lineage tracing for the study of cancer drug persistence
              </a>
            </td>
          <td>
            Binyanim Zhitomirsky, Jideofor Ezike, B. Danysh, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1950d0d983eeb2c05d4af1f02b47efd0ad374c" target='_blank'>
              Theoretical investigation of AKT1 mutations in breast cancer: a computational approach to structural and functional insights.
              </a>
            </td>
          <td>
            Balu Kamaraj, G. C
          </td>
          <td>2025-05-09</td>
          <td>Journal of computer-aided molecular design</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The CRISPR system has transformed many research areas, including cancer and immunology, by providing a simple yet effective genome editing system. Its simplicity has facilitated large-scale experiments to assess gene functionality across diverse biological contexts, generating extensive datasets that boosted the development of computational methods and machine learning/artificial intelligence applications. Integrating CRISPR with single-cell technologies has further advanced our understanding of genome function and its role in many biological processes, providing unprecedented insights into human biology and disease mechanisms. This powerful combination has accelerated AI-driven analyses, enhancing disease diagnostics, risk prediction, and therapeutic innovations. This review provides a comprehensive overview of CRISPR-based genome editing systems, highlighting their advancements, current progress, challenges, and future opportunities, especially in cancer and immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c507290af0e2ba4309109a058ca1b72e212ae70d" target='_blank'>
              Unraveling the future of genomics: CRISPR, single-cell omics, and the applications in cancer and immunology
              </a>
            </td>
          <td>
            A. Menon, Bicna Song, Lumen Chao, Diksha Sriram, Pamela Chansky, Ishnoor Bakshi, Jane Ulianova, Wei Li, Xiaoyu Zhou, Zexiang Chen
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent approvals of PARP inhibitors (PARPi) for BRCA-mutant metastatic castration resistant prostate cancer (mCRPC) necessitate an understanding of the factors that shape sensitivity and resistance. Reversion mutations that restore homologous recombination (HR) repair are detected in ∼50-80% of BRCA-mutant patients who respond but subsequently relapse, but there is currently little insight into why only ∼50% of BRCA-mutant patients display upfront resistance. To address this question, we performed a genome-wide CRISPR screen to identify genomic determinants of PARPi resistance in murine Brca2Δ/Δ prostate organoids genetically engineered in a manner that precludes the development of reversion mutations. Remarkably, we recovered multiple independent sgRNAs targeting three different members (Cdt1, Cdc6, Dbf4) of the DNA pre-replication complex (pre-RC), each of which independently conferred resistance to olaparib and the next generation PARP-1 selective inhibitor AZD5305. Moreover, sensitivity to PARP inhibition was restored in Brca2Δ/Δ, Cdc6-depleted prostate cells by knockdown of geminin, a negative regulator of Cdt1, further implicating the critical role of a functional pre-RC complex in PARPi sensitivity. Furthermore, ∼50% of CRPC tumors have copy number loss of pre-RC complex genes, particularly CDT1. Mechanistically, prostate cells with impaired pre-RC activity displayed rapid resolution of olaparib-induced DNA damage as well as protection from replication fork degradation caused by Brca2 loss, providing insight into how Brca2-mutant cancer cells can escape cell death from replication stress induced by PARP inhibition in the absence of HR repair. Of note, a pharmacologic inhibitor that targets the CDT1/geminin complex (AF615) restored sensitivity to AZD5305, providing a potential translational avenue to enhance sensitivity to PARP inhibition in BRCA-mutant cancers. Significance Here, we address a major limitation in the effectiveness of PARP inhibitors in BRCA-mutant prostate cancer treatment: only ∼50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARP inhibitor resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss of function mutations in the DNA pre-replication complex confer PARP inhibitor resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-replication complex, can restore PARP inhibitor sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7234a92766796291cf8fafb5a6fc78a58509b240" target='_blank'>
              BRCA2 reversion mutation-independent resistance to PARP inhibition in prostate cancer through loss of function perturbations in the DNA pre-replication complex
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, N. Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles Sawyers
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="Background Trastuzumab emtansine (T-DM1) is an antibody–drug conjugate that is an effective therapy for HER2-positive breast cancer; however, its efficacy is limited by drug resistance. While multiple mechanisms of resistance have been proposed, these are not yet well understood. Greater understanding of T-DM1 sensitivity and resistance could provide new combination strategies to overcome resistance or predictive biomarkers to guide therapy. Methods We have conducted CRISPR/Cas9 functional genomics modifier screens in HER2-positive breast cancer cell lines to allow for unbiased discovery of T-DM1 sensitivity and resistance genes. Whole-genome knockout screens were carried out in MDA-MB-361 and MDA-MB-453 cells treated with T-DM1 and its payload cytotoxin DM1. Hits were validated in secondary T-DM1 screens using a focused single-guide RNA (sgRNA) library and subsequently by individual gene knockout. Results The whole-genome CRISPR screens with T-DM1 and DM1 identified 599 genes as potential modifiers of T-DM1 sensitivity and resistance. Of these, 17 genes were significantly enriched and 3 genes depleted at P < 0.001 in either or both MDA-MB-361 and MDA-MB-453 libraries in the secondary screens. Among the top hits, were known T-DM1 sensitivity genes ERBB2 and SLC46A3, in addition to negative regulators of mTOR complex 1: TSC1 and TSC2. MDA-MB-453 clones with knockout of TSC1 or partial knockout of TSC2 were more resistant to T-DM1 than wild type cells in competition growth assays and to T-DM1 and other HER2 targeting therapies (T-DXd, lapatinib and neratinib) in growth inhibition assays, and had increased internalisation of T-DM1 at 6 h. T-DM1 and the mTOR inhibitor everolimus demonstrated synergistic activity at inhibiting cell proliferation at multiple T-DM1 concentrations across four HER2-positive breast cancer cell lines. Conclusions Our CRISPR screening approach with T-DM1 in HER2-positive breast cancer cell lines identified genes not previously implicated in T-DM1 sensitivity or resistance, including TSC1 and TSC2. These genes may inform new strategies to enhance T-DM1 therapy in the clinic. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02000-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee2394ef1c6b351b41594d320afdc6e8fce82eb" target='_blank'>
              CRISPR screens with trastuzumab emtansine in HER2-positive breast cancer cell lines reveal new insights into drug resistance
              </a>
            </td>
          <td>
            Barbara A Lipert, K. Siemens, Aziza Khan, R. Airey, Gech Heng Dam, Man Lu, Marcella Flinterman, Queenie Yong, T. Lee, Francis W Hunter, Stephen M F Jamieson
          </td>
          <td>2025-03-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758af464be2f5fe707681ad668cca7d960076b75" target='_blank'>
              Targetable BIRC5 dependency in therapy-resistant TP53 mutated acute myeloid leukemia
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, Fang Qi Lim, Yang Mi, E. Olesinski, Cheryl Gek Teng Chan, S. Jasdanwala, Xiao Xian Lin, Yichen Wang, Jordan Yong Ming Tan, K. Bhatia, Valeriia Sapozhnikova, Chuqi Wang, Aarthi Nivasini Mahesh, Daniel Tan En Liang, Nishtha Chitkara, P. Mertins, L. Hogdal, , Torsten Haferlach, Camille Lobry, Coleman Lindsley, Alexandre Puissant, Han Kiat Ho, Sanjukta Das, A. Letai, S. Kornblau, J. Krönke, Edward Ayoub, Koji Itahana, Michael Andreeff, Shruti Bhatt
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575ec445e704ca863d5507eaa78e755caa175d07" target='_blank'>
              The Use of Mutational Signatures to Decipher the Inter-Relationship of Reactive Oxygen Species with Homologous Recombination and Non-Homologous End-Joining Deficiencies as Well as Their Effects on APOBEC Mutagenesis in Breast Cancer
              </a>
            </td>
          <td>
            Amir Farmanbar, Robert Kneller, Sanaz Firouzi
          </td>
          <td>2025-05-12</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb676fe8f34e39303ea5d65e9d531a1c170ae75" target='_blank'>
              Abstract 4677: Combating radioresistance in lung cancer: S6K1 signaling as a therapeutic target through metabolic modulation
              </a>
            </td>
          <td>
            Ali Jose Calderon-Aparicio, Nicole L. Simone
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a highly aggressive cancer with a 5-year survival rate of less than 35%. It is characterized by significant drug resistance and abnormal energy metabolism. Mitochondrial dynamics and metabolism are crucial for AML cell survival. Mitochondrial fusion protein optic atrophy (OPA)1 is upregulated in AML patients with adverse mutations and correlates with poor prognosis. Method This study investigated targeting OPA1 with TMQ0153, a tetrahydrobenzimidazole derivative, to disrupt mitochondrial metabolism and dynamics as a novel therapeutic approach to overcome treatment resistance. Effects of TMQ0153 treatment on OPA1 and mitofusin (MFN)2 protein levels, mitochondrial morphology, and function in AML cells. In this study, we examined reactive oxygen species (ROS) production, oxidative phosphorylation (OXPHOS) inhibition, mitochondrial membrane potential (MMP) depolarization, and apoptosis. Additionally, metabolic profiling was conducted to analyze changes in metabolic pathways. Results TMQ0153 treatment significantly reduced OPA1 and mitofusin (MFN)2 protein levels and disrupted the mitochondrial morphology and function in AML cells. This increases ROS production and inhibits OXPHOS, MMP depolarization, and caspase-dependent apoptosis. Metabolic reprogramming was observed, shifting from mitochondrial respiration to glycolysis and impaired respiratory chain activity. Profiling revealed reduced overall metabolism along with changes in the glutathione (GSH)/oxidized glutathione (GSSG) and NAD⁺/NADH redox ratios. TMQ0153 treatment reduces tumor volume and weight in MV4-11 xenografts in vivo. Combination therapies with TMQ0153 and other AML drugs significantly reduced the leukemic burden and prolonged survival in NOD scid gamma (NSG) mice xenografted with U937-luc and MOLM-14-luc cells. Conclusion TMQ0153 targets mitochondrial dynamics by inhibiting OPA1, inducing metabolic reprogramming, and triggering apoptosis in AML cells. It enhances the efficacy of existing AML therapies and provides a promising combination treatment approach that exploits mitochondrial vulnerability and metabolic reprogramming to improve treatment outcomes in AML. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03372-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/568e976bde37882483f1e97ff3463f331304f49c" target='_blank'>
              Tetrahydrobenzimidazole TMQ0153 targets OPA1 and restores drug sensitivity in AML via ROS-induced mitochondrial metabolic reprogramming
              </a>
            </td>
          <td>
            Su Jung Park, C. Cerella, Jin Mo Kang, Jinyoung Byun, David Kum, Barbora Orlikova-Boyer, Anne Lorant, M. Schnekenburger, Ali Al-Mourabit, C. Christov, Juyong Lee, Byung Woo Han, Marc Diederich
          </td>
          <td>2025-04-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b02574f80309e49880132063bfa30fa50f5b" target='_blank'>
              Abstract 6343: Frameshift neoantigen-based vaccine testing in an intra-cecal implantation mouse model of Lynch syndrome
              </a>
            </td>
          <td>
            Yurong Song, Brandon Somerville, Shaneen S Baxter, Jason D. Marshall, Lei Wei, Chelsea Sanders, Qiang Hu, Song Liu, Alan Hutson, Baktiar Karim, Simone Difilippantonio, Ligia A. Pinto, Shizuko Sei, Robert H. Shoemaker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31571cc4d771e5d8cbcfb30a223ba711a12ce0b" target='_blank'>
              Resistance to the KRASG12D Inhibitor MRTX1133 is Associated with Increased Sensitivity to BET Inhibition
              </a>
            </td>
          <td>
            Daniel R. Principe, Jeffrey H Becker, Anastasia E. Metropulos, Alejandra M Marinelarena, Thao D Pham, A. Aissa, H. Munshi
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0460f2bfed152b0841d1d7ccdc52c08315edba3" target='_blank'>
              In vivo CRISPR screening reveals epigenetic regulators of hepatobiliary plasticity.
              </a>
            </td>
          <td>
            Jonathan H. Sussman, Hector Cure, Salina Yuan, Kenji Ito, Irfan A Asangani, Benjamin A Garcia, Ben Z. Stanger, Takeshi Katsuda
          </td>
          <td>2025-04-01</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5923c5ce45c39d1a157879ac9124f4aecb6425a" target='_blank'>
              Abstract 1468: Chemotherapy-resistant bladder cancer: Involvement of homologous recombination repair
              </a>
            </td>
          <td>
            Nils C H van Creij, Francisco J Roa, F. Handle, F. Santer, P. Holm, Zoran Culig, Renate Pichler
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c746df0c60c5ade414ef49a06f3bb771699caf55" target='_blank'>
              Abstract 427: Genome-wide CRISPR knockout screen identifies novel determinant of PARP inhibitor resistance in small-cell lung cancer
              </a>
            </td>
          <td>
            Jiaqi Xiong, M. Aparnathi, Tony Yu, Lifang Song, Jonathan R St-Germain, B. Raught, Troy Ketela, Sami Ul Haq, V. Philip, Benjamin H. Lok
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d3beaaac78a42fdc6dae5c6e9c46a2476c5705" target='_blank'>
              Abstract 6709: NF1 mutations in lung adenocarcinoma preclinical models and potential targeted therapies: The crucial role of the RAS-MAPK pathway
              </a>
            </td>
          <td>
            J. Giraud, Doriane Gorret, Manuela Ye, Dominique Lallemand, William C. Reinhold, Didier Decaudin, Ingrid Laurendeau, E. Pasmant, Camille Tlemsani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aff6e4c1130ad77a048fffd67082d1b6b987410" target='_blank'>
              Abstract 1480: Unlocking DNA repair mechanisms with dynamic single-molecule analysis
              </a>
            </td>
          <td>
            Paul Pease, Jack O'Sullivan, Trey Simpson, Josephine Yeh, A. Candelli
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d7e0fc018510f5e46705cec074e3eec19e6f09" target='_blank'>
              SOX17: a new therapeutic target for immune evasion of colorectal cancer
              </a>
            </td>
          <td>
            Avash Das, Omer H Yilmaz, Osman H Yilmaz, Vikram Deshpande
          </td>
          <td>2025-05-10</td>
          <td>Journal of Clinical Pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43e1e965f05b28c3994c4e8f0fb6a1f9448be497" target='_blank'>
              Abstract 2856: Pks+ E. coli trigger intestinal stem cell plasticity and early onset colorectal cancer
              </a>
            </td>
          <td>
            Stefanie Gerstberger, Melissa Lumish, S. Hartner, Farheen Shah, Seongmin Choi, Krystal Lum, Christopher Cowley, A. Luthra, Qingwen Jiang, Hyung Jun Woo, Ahmed Mahmoud, H. Walch, Asha Saxena, Tavis J. Reed, A. Cercek, R. Yaeger, Ileana Cristea, Andrew McPherson, F. Sánchez-Vega, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37654945a8b87aa1dc552938a4ce7cfc7067cb7a" target='_blank'>
              Abstract 3806: Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML
              </a>
            </td>
          <td>
            Andy G. X. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, C. Mullighan, J. Dick
          </td>
          <td>2025-05-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4631b336ea353f2fab308e1332f9b36313977696" target='_blank'>
              Abstract 4200: Elevated dNTP pools impair homologous recombination-mediated repair, promoting apoptosis in therapy-resistant glioblastoma
              </a>
            </td>
          <td>
            Dominique Monroe, Mercy Kehinde-Ige, Apeksha Anand, Arilyn Williams, E. Usoro, Matthew Kededa, Aman Kalsi, Christian Fahmy, Ramsha Khanam, Huidong Shi, Waaqo Daddacha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7430000b0cae106ff398286a8c011c0d35dbd7" target='_blank'>
              Loss-of-Function genetic Screen Unveils Synergistic Efficacy of PARG Inhibition with Combined 5-Fluorouracil and Irinotecan Treatment in Colorectal Cancer
              </a>
            </td>
          <td>
            Cristina Queralt, Cristina Moreta-Moraleda, Marta Costa, Ferran Grau-Leal, J. Diesch, Carla Vendrell-Ayats, Eva Musulén, C. Bugés, José Luis Manzano, Sara Cabrero, Johannes Zuber, M. Buschbeck, Sonia Forcales, E. Martínez-Balibrea
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f09c779b15647c7ef62bbebbbbd96b9880a1d08" target='_blank'>
              PROX1: a key regulator of hepatocyte identity and tumorigenesis.
              </a>
            </td>
          <td>
            T. K. Lee, Stephianie Ma
          </td>
          <td>2025-03-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The diploid genome of the fungal pathogen Candida albicans is highly heterozygous, with most allele pairs diverging at either the coding or regulatory level. When faced with selection pressure like antifungal exposure, this hidden genetic diversity can provide a reservoir of adaptive mutations through loss of heterozygosity (LOH) events. Validating the potential phenotypic impact of LOH events observed in clinical or experimentally evolved strains can be difficult due to the challenge of precisely targeting one allele over the other. Here, we show that a CRISPR-Cas9 system can be used to overcome this challenge. By designing allele-specific guide RNA sequences, we can induce targeted, directed LOH events, which we validate by whole-genome long-read sequencing. Using this approach, we efficiently recapitulate a recently described LOH event that increases resistance to the antifungal fluconazole. Additionally, we find that the recombination tracts of these induced LOH events have similar lengths to those observed naturally. To facilitate future use of this method, we provide a database of allele-specific sgRNA sequences for Cas9 that provide near genome-wide coverage of heterozygous sites through either direct or indirect targeting. This approach will be useful in probing the adaptive role of LOH events in this important human pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/161077a2ca34ace846023a0e504cab47b241a6a0" target='_blank'>
              Targeted loss of heterozygosity in Candida albicans using CRISPR-Cas9
              </a>
            </td>
          <td>
            Philippe C. Després, Nicholas C. Gervais, Meea Fogal, Ruby KJ Rogers, Christina A. Cuomo, Rebecca S. Shapiro
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The clustered regularly interspaced short palindromic repeats (CRISPR)/Cas system is a powerful genome editing tool that has revolutionized research. Single nucleotide polymorphisms (SNPs) are the most common form of genetic variation in humans. Only a subset of these SNPs has been shown to be linked to genetic diseases, while the biological relevance of the majority remains unclear. Investigating these variants of unknown significance could provide valuable insights into their roles in biological processes, disease susceptibility, and treatment responses. While CRISPR/Cas has emerged as a transformative technology, its ability to make single nucleotide substitutions remains a significant limitation. Other techniques in single nucleotide editing, such as base editing and prime editing, offer promising possibilities to complement CRISPR/Cas systems, though they also have their own limitations. Hence, alternative approaches are necessary to overcome the limitations of CRISPR. Here, to improve the feasibility of generating single base edits in the genome, we provide a protocol that introduces a multiple expression and dual selection (MEDS) system, which, alongside CRISPR, utilizes the opposing roles of cytosine deaminase/uracil phosphoribosyltransferase (CD/UPRT) for negative selection and neomycin phosphotransferase II (NPT II) for positive selection. As a proof of concept and to demonstrate feasibility of the method, we used MEDS, along with traditional CRISPR-Cas9, to generate sickle hemoglobin by introducing a point mutation (A → T) in the sixth codon of the hemoglobin beta gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea49db880f24cd1e00dae98c95c6cafbd8936b9" target='_blank'>
              CRISPR genome editing using a combined positive and negative selection system
              </a>
            </td>
          <td>
            I. Sharma, Kerisa Hall, Shannon Moonah
          </td>
          <td>2025-05-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers that arise from germline mutations of breast cancer associated gene 1 (BRCA1), which is a crucial player in homologous recombination (HR) DNA repair, are vulnerable to DNA-damaging agents such as platinum and PARP inhibitors (PARPis). Increasing evidence suggests that BRCA1 is an essential driver of all phases of the cell cycle, thereby maintaining orderly steps during cell cycle progression. Specifically, loss of BRCA1 activity causes the S-phase, G2/M, spindle checkpoints, and centrosome duplication to be dysregulated, thereby blocking cell proliferation and inducing apoptosis. In vertebrates, loss of HR genes such as BRCA1 and/or BRCA2 is lethal, since HR is a prerequisite for genome integrity. Thus, cancer cells utilize alternative DNA repair pathways such as non-homologous end joining (NHEJ) to cope with the loss of BRCA1 function. In this review, we attempt to update and discuss how these novel components are crucial for regulating DNA damage repair (DDR) in BRCA1-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca96851a56316ea96ea1fc73475ff478d721c03" target='_blank'>
              Overview of Roles of Novel Components in the Regulation of DNA Damage Repair in BRCA1-Deficient Cancers: An Update
              </a>
            </td>
          <td>
            Nhat Nguyen, Dominic Arris, M. T. Tran
          </td>
          <td>2025-04-01</td>
          <td>DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29e60f63dffafbff9d001a2d3645ac87336a0cf" target='_blank'>
              G6PC3 promotes genome maintenance and is a candidate mammary tumor suppressor.
              </a>
            </td>
          <td>
            Xin Li, Maria Rossing, Ana Moisés da Silva, Muthiah Bose, Thorkell Gudjónsson, Jan Benada, J. Thatte, Jens Vilstrup Johansen, J. Börcsök, H. van der Gulden, Ji-Ying Song, Renée Menezes, Asma Tajik, Lucía Sena, Zoltan Szallasi, Morten Frödin, J. Jonkers, F. C. Nielsen, C. S. Sørensen
          </td>
          <td>2025-04-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Chemotherapeutic drugs, like cisplatin, function by damaging genomic DNA, thus inducing cell apoptosis. Cancer cells can enhance their DNA repair capacity, leading to chemotherapeutic resistance. Nucleotide excision repair (NER) involves repairing DNA adducts and crosslinks caused by chemotherapeutic agents. Transforming growth factor-beta (TGF-β) pathway contributes to carcinogenesis, DNA repair alteration, and chemoresistance. However, the connection between TGF-β pathway, NER function alteration, and resistance to cisplatin therapy remains elusive. Therefore, the objective of current study was to fill this gap by assessing the impact of TGF-β inhibition and activation on cisplatin-induced antiproliferation, apoptosis, and DNA damage using the MTT assay, flow cytometry analysis, and COMET assay, respectively. Four NER genes, XPA, XPB, XPC, and XPF, were measured using Real-time Polymerase Chain Reaction (qPCR). MDA-MB-231 cell line was utilized as a model of breast cancer. Blockade of the TGF-β pathway strengthened cisplatin cytotoxicity, whereas induction of the TGF-β pathway suppressed cisplatin cytotoxicity. In cisplatin-treated breast cancer cells, DNA damage significantly increased upon the TGF-β pathway inhibition. Conversely, cisplatin-induced DNA damage decreased significantly upon TGF-β pathway stimulation. Finally, cisplatin caused an overexpression of the four NER genes which was curtailed and augmented by TGF-β inhibition and stimulation, respectively. Overall, this study presented evidence of the impact exerted by TGF-β pathway on NER and cisplatin sensitivity of breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da79b3598334d2968d0a956c3775a3a49b9ba178" target='_blank'>
              Modulation of Chemotherapy Sensitivity of Breast Cancer Cells through Transforming Growth Factor-beta Pathway-mediated Alterations in DNA Damage Response
              </a>
            </td>
          <td>
            Abdullah S. Alhamed, Mohammad S El-Wetidy, Mervat M. Abdelwahed, Sabry M. Attia, Abdulrahman M. Alabkka, Saleh A. Alaraj, Khalid Alhazzani, Ahmed Z Alanazi, Faris Almutairi, Ibrahem A. Alotibi, Mohammed Alqinyah
          </td>
          <td>2025-03-31</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ab372dd4cae4e9e87fbdd2a61f62f73ea8595e" target='_blank'>
              Abstract 2973: Discovery of novel synthetic lethal targets for effective and safe colorectal cancer therapies
              </a>
            </td>
          <td>
            Andrzej Mazan, M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Izabella Wieckowska, M. Chmiel, A. Kurzejamski, B. Lipert, Katarzyna Sarad, M. Mikula, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5271ff004a44b818284ad63ea30a279c93609a" target='_blank'>
              Abstract 3882: Patterns of genomic patient-model conservation and evolution in the Human Cancer Models Initiative (HCMI) next-generation cancer model resource
              </a>
            </td>
          <td>
            Andrew McPherson, Seongmin Choi, William F Hooper, Jennifer Shelton, Timothy R. Chu, Dina Elharouni, Mushriq Al-Jazrawe, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Mathew J. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2db16c30d79726e181227a7646a42de7a4c218" target='_blank'>
              Abstract 829: ITGAV is a targetable vulnerability in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Gilbert K. Salloum
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/642babaec15e8d22494c611094dc188788c6a876" target='_blank'>
              Abstract 3807: Validation of a functional genomics screen based on lentiviral integration site analysis reveals single-gene disruptions enhancing CAR19 T-cell effectiveness in preclinical models
              </a>
            </td>
          <td>
            Beatriz Coutinho de Oliveira, Saaurav Bari, Jessica Kerr, Victoria Putzbach, Y. Phoon, J. Fraietta, Wei-Ting Hwang, Frederic D. Bushman, J. Melenhorst
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe56ec63c27c1afc432bcd0d7820f4cd76de13e" target='_blank'>
              Abstract 2733: Super-enhancers in oncogene upregulation in high-grade ovarian serous carcinoma proliferation and platinum-resistance
              </a>
            </td>
          <td>
            Zhongmou Jin, Kristin Adams, Kendra Wendt, Josie Wood, Sydney Olson, Jessica D. Lang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe96f8609d981f1d51f682dd9dde5af13de525a" target='_blank'>
              Abstract 5533: Drug tolerance mechanisms in patient-derived xenograft models with epidermal growth factor receptor mutated tumors treated with tyrosine kinase inhibitors
              </a>
            </td>
          <td>
            A. Arivajiagane, Q. Li, Nhu-An Pham, Ming Li, Katrina Hueniken, Takamasa Koga, N. Radulovich, M S Tsao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2840ab52aa192e80f9f34d1b245eef91e6a39eea" target='_blank'>
              Abstract 7183: METTL1-mediated m7G modification promotes 5-FU resistance in colorectal carcinoma
              </a>
            </td>
          <td>
            Silei Sui, Yuan Li, Rui Su, Man Li, A. Goel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Osteosarcoma (OS) presents significant treatment challenges due to chemoresistance. This study explores the molecular mechanisms underlying chemoresistance in OS, focusing on the novel fusion gene RPS27-RPS24. Using single-cell multi-omics techniques, we identified a significant upregulation of RPS27-RPS24 in chemoresistant OS cells. Our analyses revealed that RPS27-RPS24 enhances glutaminase (GLS)-mediated glutamine metabolism and inhibits copper-induced cell death, thereby promoting chemoresistance. In vitro experiments with adriamycin-resistant (ADMR) OS cells confirmed that overexpression of RPS27-RPS24 leads to increased cell viability and proliferation under chemotherapy. In vivo studies further validated these findings, demonstrating that targeting glutamine metabolism can reverse chemoresistance. Our results suggest that the RPS27-RPS24 fusion gene plays a critical role in OS chemoresistance through metabolic reprogramming, providing a potential therapeutic target for improving OS treatment outcomes. The application of multiple analytical techniques in this study (as shown in the upper image) and the hypothesized mechanism (as shown in the lower image).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe8559842f0f521ca0cecc77e49311c45066c19f" target='_blank'>
              Single-cell multi-omics elucidates the role of RPS27-RPS24 fusion gene in osteosarcoma chemoresistance and metabolic regulation
              </a>
            </td>
          <td>
            Zhiwei Tao, Pingan Zou, Zhengxu Yang, Tao Xiong, Zhi Deng, Qincan Chen
          </td>
          <td>2025-04-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f9135b22221853a957c30c494e32f6d68e606" target='_blank'>
              Abstract LB016: Targeting ATM modulates oncogenic pathways and amplifies chemotherapy efficacy in small cell lung cancer
              </a>
            </td>
          <td>
            Debdatta Halder, Utsav Sen, V. Jethalia, Subhamoy Chakraborty, Yosuke Dotsu, Andrew Elliot, A. Vanderwalde, D. Demircioglu, D. Hasson, Triparna Sen
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1d772bceb37d658c289fadfd2de9d117e82a8a" target='_blank'>
              Abstract 4103: MLL4/COMPASS dysfunction in cancer and treatment
              </a>
            </td>
          <td>
            Zibo Zhao, Sarah Gold, Yuki Aoi, K. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, I. Ben-Sahra, Rintaro Hashizume, D. Mahalingam, Joshua Meeks, A. Shilatifard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Telomere biology disorders (TBDs) are genetic diseases caused by defective telomere maintenance. TBD patients often develop bone marrow failure and have an increased risk of myeloid neoplasms. To better understand the factors underlying hematopoietic outcomes in TBD, we comprehensively evaluated acquired genetic alterations in hematopoietic cells from 166 pediatric and adult TBD patients. Of these patients, 47.6% (28.8% of children, 56.1% of adults) had clonal hematopoiesis. Recurrent somatic alterations involved telomere maintenance genes (7.6%), spliceosome genes (10.4%, mainly U2AF1 p.S34), and chromosomal alterations (20.2%), including 1q gain (5.9%). Somatic variants affecting the DNA damage response (DDR) were identified in 21.5% of patients, including 20 presumed loss-of-function variants in ataxia-telangiectasia mutated (ATM). Using multimodal approaches, including single-cell sequencing, assays of ATM activation, telomere dysfunction-induced foci analysis, and cell-growth assays, we demonstrate telomere dysfunction–induced activation of the ATM-dependent DDR pathway with increased senescence and apoptosis in TBD patient cells. Pharmacologic ATM inhibition, modeling the effects of somatic ATM variants, selectively improved TBD cell fitness by allowing cells to bypass DDR-mediated senescence without detectably inducing chromosomal instability. Our results indicate that ATM-dependent DDR induced by telomere dysfunction is a key contributor to TBD pathogenesis and suggest dampening hyperactive ATM-dependent DDR as a potential therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7e03f128958621aaeeb6a79d8a716a78521617" target='_blank'>
              ATM-dependent DNA damage response constrains cell growth and drives clonal hematopoiesis in telomere biology disorders
              </a>
            </td>
          <td>
            Christopher M Sande, Stone Chen, Dana V Mitchell, Ping Lin, Diana M Abraham, Jessie M. Cheng, Talia Gebhard, Rujul Deolikar, Colby Freeman, Mary Zhou, Sushant Kumar, Michael Bowman, Robert L Bowman, Shannon Zheng, Bolormaa Munkhbileg, Qijun Chen, Natasha L. Stanley, Kathy Guo, Ajibike Lapite, Ryan Hausler, Deanne M. Taylor, James Corines, Jennifer J D Morrissette, David B Lieberman, Guang Yang, O. Shestova, Saar Gill, Jiayin Zheng, Kelcy Smith-Simmer, L. Banaszak, Kyle Shoger, Erica F Reinig, Madilynn Peterson, Peter Nicholas, A. Walne, I. Dokal, Justin P. Rosenheck, K. Oetjen, Daniel C. Link, A. Gelman, Christopher R Reilly, Ritika Dutta, R. C. Lindsley, K. Brundige, Suneet Agarwal, Alison A. Bertuch, Jane E. Churpek, L. Tague, F. B. Johnson, Timothy Olson, D. Babushok
          </td>
          <td>2025-04-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b855a13f4583cca6b74a017150a15f6334f81973" target='_blank'>
              Abstract 2773: The Gundersen Precision Oncology Cohort: a biospecimen resource for mutation characterization and target discovery
              </a>
            </td>
          <td>
            Peter Feiszt, Paraic A. Kenny
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9cc09a30e02fe7681c9c9c4146bae9713fa5e17" target='_blank'>
              Integrative CRISPR Screens and RNA-Omics Discover an Essential Role for PUF60-3’ Splice Site Interactions in Cancer Progression
              </a>
            </td>
          <td>
            Alexandra T. Tankka, J. Einstein, Cathy Zhou, Vivian N. Pham, Yuhan Zhang, Jack T. Naritomi, Grady G. Nguyen, Orel Mizrahi, M. Perelis, Joseph Sarsam, Frederick E. Tan, D. Kaufman, Corina E. Antal, Gene W Yeo
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5750dddf572726400a2fad0203c2ec7fc9717955" target='_blank'>
              Abstract 478: Engineering viruses for the next generation targeted cancer therapy
              </a>
            </td>
          <td>
            Ikeda Trashi, Jeremiah Gassensmith
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic‐type malignancies (SMs) arising in germ cell tumors (GCTs) are aggressive neoplasms resistant to systemic treatment. Most are diagnosed in metastatic sites after chemotherapy; however, they have also been well‐documented in primary testicular GCTs. Historically, SMs were thought to originate in components of teratoma that acquire molecular alterations equivalent to those that characterize their true somatic counterparts. However, recent studies have shown that SMs typically lack the hallmark molecular alterations seen in similar somatic tumors. Additionally, clinicopathologic and molecular data suggest that a subset may derive from yolk sac tumor (YST) rather than teratoma. In this study, we evaluated the relationship between conventional histological types of GCTs and SMs by comparing expression of microRNA (miR)‐371–373 and genomic methylation profiles. A total of 96 samples (including multiple paired conventional GCT–SM samples from individual tumors) were assessed for miR‐371–373 expression by RT‐qPCR and genomic DNA methylation using a clinically validated assay. Expression of miR‐371–373 was higher in conventional GCTs than in SMs (considered as a single category encompassing all histological subtypes). However, miR‐371–373 expression was heterogeneous among SMs, with significantly higher levels in sarcomatoid YST (SYST) and glandular neoplasms than in other SMs. Genomic DNA methylation analysis showed that SMs (considered as a single category) did not form a distinct cluster. Instead, they grouped into multiple clusters that did not show perfect correspondence with histology and often included conventional GCTs. Genome‐wide methylation assessment showed a higher abundance of hypermethylated regions in SMs than in conventional GCTs. Analysis of paired conventional GCT and ‘somatic‐type’ components that did not meet size criteria for SMs dissected from individual tumors demonstrated separation according to histology, suggesting that epigenetic processes play a role in the transition from conventional GCT to ‘somatic‐type’ phenotypes. Gene‐level and pathway‐level analyses identified MAPK/RAS signaling, mitosis/proliferation, differentiation towards neural tissue/neuroectoderm, epithelial‐to‐mesenchymal transition, and DNA repair as key differentially regulated processes in components with somatic‐type histology, suggesting mechanisms of progression from conventional to ‘somatic’ phenotypes in GCT. These results support the hypothesis that a subset of SMs derive from YST and suggest that some subtypes (such as SYST) may represent ‘intermediate’ phenotypes. Additionally, analysis of differentially methylated promoter regions in SM identified genes and biologic processess that may underlie 'somatic tranformation' in GCTs. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175bd3895700f513d14429236b145a33de18651c" target='_blank'>
              MicroRNA‐371–373 cluster and methylome analysis suggests that a subset of ‘somatic‐type’ malignancies arising in germ cell tumors may originate in yolk sac tumor components
              </a>
            </td>
          <td>
            João Lobo, Nuno Tiago Tavares, Diana Fonseca, Carmen Jerónimo, Rui Henrique, Nicolas Wyvekens, Yiying Yang, Matija Snuderl, Fiona Maclean, Jennifer B. Gordetsky, C. Fletcher, Michelle S Hirsch, Jason L. Hornick, M. T. Idrees, Katrina Collins, L. Warmke, Thomas Ulbright, Andres M Acosta
          </td>
          <td>2025-03-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca665dc53860039a134586f0a4361eb2c643769" target='_blank'>
              Abstract 5645: Targeting transcription-replication conflict for selective chemotherapy in glioblastoma
              </a>
            </td>
          <td>
            Long Gu, Robert J. Hickey, L. Malkas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2116c40f11199361c9cb93d0a62e0a019f2b44da" target='_blank'>
              Abstract 4681: Multiomic insights into mechanisms of androgen receptor-driven radioresistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Benjamin Hauk, B. McBean, A. Michmerhuizen, Douglas Gurdak, Savannah Tidmore, Michelle Paulsen, L. Lerner, C. Ward, Kassidy M. Jungles, Daniel Spratt, Lori J. Pierce, Mats Ljungman, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801d1a28b5036fffab863a5f513c202209cf4c12" target='_blank'>
              Abstract 4722: Crucial prognostic biomarkers for resistance to KRAS-G12C inhibitors
              </a>
            </td>
          <td>
            Yan Wang, Linyan Tian, Chengming Liu, Sufei Zheng, Huiyang Shi, Fang Wei, Wenxin Jiang, Yucheng Dong, Haiyan Xu, Enzhi Yin, Nan Sun, Jie He
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f17818e89e418eef53189bf0e536334d7fee6f" target='_blank'>
              Abstract 5042: The Paipu pipeline for pan-mammalian cancer genomics
              </a>
            </td>
          <td>
            Bria S. Smith, Geesa Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ca3a1f08d426b3a13684391aa11a6fb1c8556a" target='_blank'>
              Abstract LB443: Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting
              </a>
            </td>
          <td>
            Alexander Schreiner, A. Foitzik, J. Levenga, Karin Boettcher, Barbara Sonnenberg, Z. Strezoska, Joe Trask
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a9e6faf45fc758543cd3dfb6d7c7bf6cfe5d5f" target='_blank'>
              Abstract 6892: Ribonucleotide reductase is a potential therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Federica Invrea, Consalvo Petti, C. Isella, E. Médico
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f032fd3c43cdde86d384472ad3a3c6263ba1644" target='_blank'>
              Abstract 2746: 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer
              </a>
            </td>
          <td>
            Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genotoxic stress-induced stem cell maldifferentiation (GSMD) integrates DNA damage responses with loss of stemness and lineage-specific differentiation to prevent damaged stem cell propagation. However, molecular mechanisms governing GSMD remain unclear. Here, we identify the p53-induced long non-coding RNA LOC644656 as a key regulator of GSMD in human embryonic stem cells. LOC644656 accumulates in the nucleus upon DNA damage, disrupting pluripotency by interacting directly with POU5F1 and KDM1A/LSD1-NuRD complexes, repressing stemness genes, and activating TGF-β signaling. Additionally, LOC644656 mitigates DNA damage by binding DNA-PKcs and modulating the DNA damage response. In cancer, elevated LOC644656 correlates with poor patient survival and enhanced chemoresistance. Our findings demonstrate that LOC644656 mediates stemness suppression and resistance to genotoxic stress by coordinating DNA damage signaling and differentiation pathways. Thus, LOC644656 represents a potential therapeutic target for overcoming chemoresistance and advancing stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a2bd396273c74fb23a4471cceb78f9c794e8d" target='_blank'>
              p53-inducible lncRNA LOC644656 causes genotoxic stress-induced stem cell maldifferentiation and cancer chemoresistance
              </a>
            </td>
          <td>
            Ai Tamura, Kazuyuki Yamagata, Takashi Kono, Masanori Fujimoto, Takahiro Fuchigami, Motoi Nishimura, Masataka Yokoyama, Akitoshi Nakayama, N. Hashimoto, I. Sakuma, Nobuyuki Mitsukawa, Yusuke Kawashima, O. Ohara, S. Motohashi, Eiryo Kawakami, Takashi Miki, Atsushi Onodera, Tomoaki Tanaka
          </td>
          <td>2025-05-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb98d4aed3d99f6dd44032425162abd6d9f52343" target='_blank'>
              Abstract 4753: ACSS2 inhibition alters epigenetic landscape to improve endocrine therapy outcome in ER+ metastatic breast cancer cells
              </a>
            </td>
          <td>
            A. Mogol, Jin Young Yoo, David J. Dutton, Zeynep Madak-Erdogan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are short non-coding RNAs, which perform a key role in cellular differentiation and development. Most human diseases, particularly cancer, are linked to miRNA functional dysregulation implicated in the expression of tumor-suppressive or oncogenic targets. Cancer hallmarks such as continued proliferative signaling, dodging growth suppressors, invasion and metastasis, triggering angiogenesis, and avoiding cell death have all been demonstrated to be affected by dysregulated miRNAs. Thus, for the treatment of different cancer types, the detection and quantification of this type of RNA is significant. The classical and current methods of RNA detection, including northern blotting, reverse transcription-quantitative PCR, rolling circle amplification and next-generation sequencing, may be effective but differ in efficiency and accuracy. Furthermore, these approaches are expensive, and require special instrumentation and expertise. Thus, researchers are constantly looking for more innovative approaches for miRNA detection, which can be advantageous in all aspects. In this regard, an RNA manipulation tool known as the CRISPR and CRISPR-associated sequence 13 (CRISPR/Cas13) system has been found to be more advantageous in miRNA detection. The Cas13-based miRNA detection approach is cost effective and requires no special instrumentation or expertise. However, more research and validation are required to confirm the growing body of CRISPR/Cas13-based research that has identified miRNAs as possible cancer biomarkers for diagnosis and prognosis, and as targets for treatment. In the present review, current updates regarding miRNA biogenesis, structural and functional aspects, and miRNA dysregulation during cancer are described. In addition, novel approaches using the CRISPR/Cas13 system as a next-generation tool for miRNA detection are discussed. Furthermore, challenges and prospects of CRISPR/Cas13-based miRNA detection approaches are described.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7cad934b860ca66c2574dc37104aa0e67005c2" target='_blank'>
              Current updates regarding biogenesis, functions and dysregulation of microRNAs in cancer: Innovative approaches for detection using CRISPR/Cas13-based platforms (Review)
              </a>
            </td>
          <td>
            Abdulaziz A. Aloliqi, Abdullah M. Alnuqaydan, Aqel Albutti, Basmah F. Alharbi, A. Rahmani, Amjad Ali Khan
          </td>
          <td>2025-04-11</td>
          <td>International Journal of Molecular Medicine</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c350dc98b4d3786f2ffe26744dbe7c5702a670" target='_blank'>
              Abstract 5246: Chromosomal instability promotes colorectal cancer progression through manipulation of gamma delta T cells
              </a>
            </td>
          <td>
            Paulino Tallón de Lara, Aviad Ben-Shmuel, Wen-Hao Hsu, Chaohao Li, Kyle Labella, Chi Wut Wong, Yonghong Liu, Shan Jiang, Xiaoying Shang, S. Kopetz, Ronald A. DePinho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b9bda18d8cb73c7c5646b13e67e456908ec19a2" target='_blank'>
              Single-cell and bulk transcriptome analyses reveal elevated amino acid metabolism promoting tumor-directed immune evasion in colorectal cancer
              </a>
            </td>
          <td>
            Tianyue Sun, Yan Chen, Ying Chen
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The dual genetic control of mitochondrial respiratory function, combined with the high mutation rate of the mitochondrial genome (mtDNA), makes mitochondrial diseases among the most frequent genetic diseases in humans (1 in 5,000 in adults). With no effective treatments available, gene therapy approaches have been proposed. Notably, several studies have demonstrated the potential for nuclear expression of a healthy copy of a dysfunctional mitochondrial gene, referred to as allotopic expression, to help recover respiratory function. However, allotopic expression conditions require significant optimization. We harnessed engineering biology tools to improve the allotopic expression of the COX2-W56R gene in the budding yeast Saccharomyces cerevisiae. Through conducting random mutagenesis and screening of the impact of vector copy number, promoter, and mitochondrial targeting sequence, we substantially increased the mitochondrial incorporation of the allotopic protein and significantly increased recovery of mitochondrial respiration. Moreover, CN-PAGE analyses revealed that our optimized allotopic protein does not impact cytochrome c oxidase assembly, or the biogenesis of respiratory chain supercomplexes. Importantly, the most beneficial amino acid substitutions found in the second transmembrane helix (L93S and I102K) are conserved residues in the corresponding positions of human MT-CO2 (L73 and L75), and we propose that mirroring these changes could potentially help improve allotopic Cox2p expression in human cells. To conclude, this study demonstrates the effectiveness of using engineering biology approaches to optimise allotopic expression of mitochondrial genes in the baker's yeast.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b719879ddbf80db7c9d88a52dbf9b4791e0038" target='_blank'>
              A new set of mutations in the second transmembrane helix of the Cox2p-W56R substantially improves its allotopic expression in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Kewin Gombeau, Stefan A Hoffmann, Yizhi Cai
          </td>
          <td>2025-04-01</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d431bff55b178eb615284d91d43a85ede147802" target='_blank'>
              Abstract 5488: Identifying novel Ewing sarcoma therapeutics using a high-throughput chromatin accessibility assay
              </a>
            </td>
          <td>
            Darian Williams, Andrew McFadden, Austin J. Hepperla, Min Shen, Tino W Sanchez, Gordon M. Stott, M. Henderson, Ian Davis, Samantha Pattenden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434c5eb06ef2c6750531e761d704cb04ab799fe6" target='_blank'>
              Management of T-cell malignancies: Bench-to-bedside targeting of epigenetic biology.
              </a>
            </td>
          <td>
            Ariana Sabzevari, J. Ung, Jeffrey W. Craig, K. Jayappa, Ipsita Pal, D. Feith, Thomas P Loughran, Owen A O'Connor
          </td>
          <td>2025-04-15</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b4e4b812f53d84fe2b91abe7b8dd8fa8cfa234" target='_blank'>
              Abstract 1514: CRISPRi screening reveals RNA editing as a major determinant of platinum sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07b2694a7cc900c5a2210ecef718b0d5e786e3ea" target='_blank'>
              MBNL2 enhances cisplatin resistance by regulating apoptosis in ovarian cancer cells.
              </a>
            </td>
          <td>
            H. Sung, Jihye Han, Woong Ju, Jihee Lee Kang, Aekyung Park, Jung-Hyuck Ahn
          </td>
          <td>2025-04-03</td>
          <td>BMB reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18bd2d7b496fa24120af2bd1f113301f206be6f5" target='_blank'>
              Abstract 1139: Massive scale phenotypic screening using generative chemogenomics repurposes safe drugs and discovers new drugs for intercepting early-stage lung adenocarcinoma progression
              </a>
            </td>
          <td>
            Abhilasha Sinha, John Hawley, Nanzeeba Faraby, Tanya Cashorali, Thomas A. Neyarapally, William Oh, Paul D. McDonagh, Charles A. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction CRISPR/Cas9-edited induced pluripotent stem cells (iPSCs) are valuable research models for mechanistic studies. However, gene conversion between a gene-pseudogene pair that share high sequence identity and form direct repeats in proximity on the same chromosome can interfere with the precision of gene editing. Mutations in the human beta-glucocerebrosidase gene (GBA1) are associated with Gaucher disease, Parkinson’s disease, and Lewy body dementia. During the creation of a GBA1 KO iPSC line, we detected about 70% gene conversion from its pseudogene GBAP1. These events maintained the reading frame and resulted from GBA1-specific cleavage by CRISPR/Cas9, without disrupting the GBA1 gene. Method To increase the percentage of alleles with out-of-frame indels for triggering nonsense-mediated decay of the GBA1 mRNA, we supplied the cells with two single-stranded oligodeoxynucleotide (ssODN) donors as homology-directed repair (HDR) templates. Results We demonstrate that HDR using the ssODN templates effectively competes with gene conversion and enabled biallelic KO clone isolation, whereas the nonallelic homologous recombination (NAHR)-based deletion rate remained the same. Discussion Here, we report a generalizable method to direct cellular DNA repair of double strand breaks at a target gene towards the HDR pathway using exogenous ssODN templates, allowing specific editing of one gene in a gene-pseudogene pair without disturbing the other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522033033b99ee07b46d656d628a06d2bb8f2da5" target='_blank'>
              Efficient GBA1 editing via HDR with ssODNs by outcompeting pseudogene-mediated gene conversion upon CRISPR/Cas9 cleavage
              </a>
            </td>
          <td>
            Joseph S. Lagas, M. Sentmanat, Xiaoxia Cui
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This study presents a comprehensive genetic characterization of the H9c2 cell line, a widely used model for cardiac myoblast research. We established a short tandem repeat (STR) profile for H9c2 that is useful to confirm the identity and stability of the cell line. Additionally, we prepared H9c2 metaphase chromosomes and performed karyotyping and molecular cytogenetics to further investigate chromosomal characteristics. The genetic analysis showed that H9c2 cells exhibit chromosomal instability, which may impact experimental reproducibility and data interpretation. Next-generation sequencing (NGS) was performed to analyze the transcriptome, revealing gene expression patterns relevant to cardiac biology. Western blot analysis further validated the expression levels of selected cardiac genes identified through NGS. Additionally, Phalloidin staining was used to visualize cytoskeletal organization, highlighting the morphological features of these cardiac myoblasts. Our findings collectively support that H9c2 cells are a reliable model for studying cardiac myoblast biology, despite some genetic alterations identified resembling sarcoma cells. The list of genes identified through NGS analysis, coupled with our comprehensive genetic analysis, will serve as a valuable resource for future studies utilizing this cell line in cardiovascular medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1647a813c18457ce232ec6ab89eefeaf1dce3b9a" target='_blank'>
              Genetic and Molecular Characterization of H9c2 Rat Myoblast Cell Line
              </a>
            </td>
          <td>
            Thomas Liehr, Stefanie Kankel, Katharina S. Hardt, E. Buhl, H. Noels, Diandra T. Keller, Sarah K. Schröder-Lange, Ralf Weiskirchen
          </td>
          <td>2025-03-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973aff9217cd2f6bd7d72f2139353fd552c0fa4e" target='_blank'>
              SARE: an optimized method for detection functional regulatory elements in genome
              </a>
            </td>
          <td>
            Mohammadjavad Hosseinpoor
          </td>
          <td>2025-04-16</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The leading cause of human pregnancy loss is aneuploidy, often tracing to errors in chromosome segregation during female meiosis. While abnormal crossover recombination is known to confer risk for aneuploidy, limited data have hindered understanding of the potential shared genetic basis of these key molecular phenotypes. To address this gap, we performed retrospective analysis of preimplantation genetic testing data from 139,416 in vitro fertilized embryos from 22,850 sets of biological parents. By tracing transmission of haplotypes, we identified 3,656,198 crossovers, as well as 92,485 aneuploid chromosomes. Counts of crossovers were lower in aneuploid versus euploid embryos, consistent with their role in chromosome pairing and segregation. Our analyses further revealed that a common haplotype spanning the meiotic cohesin SMC1B is significantly associated with both crossover count and maternal meiotic aneuploidy, with evidence supporting a non-coding cis-regulatory mechanism. Transcriptome- and phenome-wide association tests also implicated variation in the synaptonemal complex component C14orf39 and crossover-regulating ubiquitin ligases CCNB1IP1 and RNF212 in meiotic aneuploidy risk. More broadly, recombination and aneuploidy possess a partially shared genetic basis that also overlaps with reproductive aging traits. Our findings highlight the dual role of recombination in generating genetic diversity, while ensuring meiotic fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed8e001dc646667003aedc35613a33f0517e9a4" target='_blank'>
              Common variation in meiosis genes shapes human recombination phenotypes and aneuploidy risk
              </a>
            </td>
          <td>
            S. A. Carioscia, Arjun Biddanda, Margaret R. Starostik, Xiaona Tang, E. Hoffmann, Zachary Demko, R. McCoy
          </td>
          <td>2025-04-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths globally, with radiotherapy as a key treatment modality for inoperable cases. Lactate, once considered a by-product of anaerobic cellular metabolism, is now considered critical for cancer progression. Lactate dehydrogenase B (LDHB) converts lactate to pyruvate and supports mitochondrial metabolism. In this study, a re-analysis of our previous transcriptomic data revealed that LDHB silencing in the NSCLC cell lines A549 and H358 dysregulated 1789 genes, including gene sets associated with cell cycle and DNA repair pathways. LDHB silencing increased H2AX phosphorylation, a surrogate marker of DNA damage, and induced cell cycle arrest at the G1/S or G2/M checkpoint depending on the p53 status. Long-term LDHB silencing sensitized A549 cells to radiotherapy, resulting in increased DNA damage and genomic instability as evidenced by increased H2AX phosphorylation levels and micronuclei accumulation, respectively. The combination of LDHB silencing and radiotherapy increased protein levels of the senescence marker p21, accompanied by increased phosphorylation of Chk2, suggesting persistent DNA damage. Metabolomics analysis revealed that LDHB silencing decreased nucleotide metabolism, particularly purine and pyrimidine biosynthesis, in tumor xenografts. Nucleotide supplementation partially attenuated DNA damage caused by combined LDHB silencing and radiotherapy. These findings suggest that LDHB supports metabolic homeostasis and DNA damage repair in NSCLC, while its silencing enhances the effects of radiotherapy by impairing nucleotide metabolism and promoting persistent DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fecf04a0b52ec43bfbb157471e371b12fd93b5" target='_blank'>
              LDHB silencing enhances the effects of radiotherapy by impairing nucleotide metabolism and promoting persistent DNA damage
              </a>
            </td>
          <td>
            Haibin Deng, Fatlind Malsiu, Huixiang Ge, T. Losmanovà, Michaela Medová, N. Zamboni, Wenxiang Wang, Ren-Wang Peng, Jinming Tang, Patrick Dorn, T. M. Marti
          </td>
          <td>2025-03-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Rare monogenic disorders are caused by mutations in single genes and have an incidence rate of less than 0.5%. Due to their low prevalence, these diseases often attract limited research and commercial interest, leading to significant unmet medical needs. In a therapeutic landscape where treatments are targeted to manage symptoms, gene editing therapy emerges as a promising approach to craft curative and lasting treatments for these patients, often referred to as “one-and-done” therapeutics. CRISPR-dependent base editing enables the precise correction of genetic mutations by direct modification of DNA bases without creating potentially deleterious DNA double-strand breaks. Base editors combine a nickase version of Cas9 with cytosine or adenine deaminases to convert C·G to T·A and A·T to G·C, respectively. Together, cytosine (CBE) and adenine (ABE) base editors can theoretically correct ∼95% of pathogenic transition mutations cataloged in ClinVar. This mini-review explores the application of base editing as a therapeutic approach for rare monogenic disorders. It provides an overview of the state of gene therapies and a comprehensive compilation of preclinical studies using base editing to treat rare monogenic disorders. Key considerations for designing base editing-driven therapeutics are summarized in a user-friendly guide for researchers interested in applying this technology to a specific rare monogenic disorder. Finally, we discuss the prospects and challenges for bench-to-bedside translation of base editing therapies for rare monogenic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50941b3a230c6aba4e286e74fa37535f7cdb8846" target='_blank'>
              CRISPR-dependent base editing as a therapeutic strategy for rare monogenic disorders
              </a>
            </td>
          <td>
            Júlia-Jié Cabré-Romans, Raquel Cuella-Martin
          </td>
          <td>2025-04-02</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8df54e46e7183e7945f80187fc6745c6cb3b628" target='_blank'>
              Abstract 4632: Molecular determinants to neoadjuvant chemotherapy resistance in breast cancer patients: Insights from a Latin-American cohort
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos A. Orozco, R. Parra-Medina, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Combita
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            V. Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe0540c5e4b9f9aeac13f8f767746a6b0c968412" target='_blank'>
              Clinical Impact of Somatic Genomic Variants of Oncogenes and Tumor Suppressor Genes in Previously Treated Advanced Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            D. Zhou, Janene Dalrymple, Danielle Klingberg, Frank Lin, S. J. Lord, Wendy A. Cooper, Milita Zaheed, R. J. Simes, Thomas John, C. Lee
          </td>
          <td>2025-04-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a636c2401d6f90b475b0150ef9a2c6977b69c4" target='_blank'>
              Abstract 2894: NTX-452: a non-covalent, potent, selective and highly efficacious WRN inhibitor with best-in-class potential for the treatment of MSI-H tumors
              </a>
            </td>
          <td>
            R. Svensson, Scott Boiko, B. Browning, S. Carreiro, Derun Li, Angela V Toms, J. Caravella, S. Leit, S. Jacques-O'Hagan, Hyesun Oh, E. Florine, Zhenhong Li, A. Phadke, Y.M. Khong, A. Scheuber, P. Tummino
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
 Colorectal cancer (CRC) is the second most prevalent cause of cancer-related mortality and is increasing in younger individuals. Chemotherapy, a crucial adjuvant systemic therapy for CRC management, often leads to resistance through poorly characterized underlying molecular mechanisms. The long noncoding RNA SNHG5 is highly expressed in CRC and promotes tumor proliferation and invasion, prompting us to hypothesize that SNHG5 may play a crucial role in the chemotherapeutic agent 5-fluorouracil (5-Fu) resistance in CRC.
 AIM
 To identify the function and mechanism of SNHG5 in 5-Fu resistance in CRC.
 METHODS
 Quantitative real-time polymerase chain reaction was performed to examine the expression of SNHG5 in CRC tissues from 22 5-Fu-sensitive patients and 14 5-Fu-resistant patients and in CRC cells and 5-Fu-resistant CRC cells. Cell viability and apoptosis were assessed in SNHG5 -overexpressing CRC cells and SNHG5 -knockdown 5-Fu-resistant CRC cells. SNHG5 function in 5-Fu resistance in CRC was further analyzed using a xenograft mouse model. SNHG5 interactions with microRNAs were predicted by bioinformatics analysis. Luciferase reporter and RNA immunoprecipitation assays were performed to verify the binding between SNHG5 and miR-26b. Rescue experiments were performed to validate the functional interaction between SNHG5 and the miR-26b/p-glycoprotein (Pgp) axis.
 RESULTS
 SNHG5 expression was upregulated in 5-Fu-resistant CRC tissues and 5-Fu-resistant CRC cells. In vitro functional experiments demonstrated that SNHG5 overexpression significantly reduced cell apoptosis and enhanced cell viability, whereas SNHG5 knockdown in 5-Fu-resistant CRC cells increased cell apoptosis and decreased cell viability upon 5-Fu treatment. In a xenograft mouse model, we confirmed that SNHG5 overexpression led to a reduction in 5-Fu sensitivity in CRC in vivo . Mechanistically, SNHG5 acted as a molecular sponge for miR-26b. Rescue experiments validated that SNHG5 conferred 5-Fu resistance in CRC by regulating the miR-26b/Pgp axis.
 CONCLUSION
 SNHG5 /miR-26b/Pgp regulates CRC chemosensitivity, providing potential therapeutic targets for the treatment of 5-Fu-resistant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ba246ad7d5aba9cd9fac4d6db284ed3bfeef7e" target='_blank'>
              Long noncoding RNA SNHG5 promotes 5-fluorouracil resistance in colorectal cancer by regulating miR-26b/p-glycoprotein axis
              </a>
            </td>
          <td>
            Bin Wang, Qian Zhou, Cui-E Cheng, Yi-Jie Gu, , Jia-Ming Qiu, Gui-Ning Wei, Ya-Dong Feng, , Rui-Hua Shi
          </td>
          <td>2025-05-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7d798ffee04bc73cc38a17396f4a99f40e9d7c9" target='_blank'>
              Dysfunctional mismatch repair in patients with early triple-negative breast cancer.
              </a>
            </td>
          <td>
            María Muñiz-Castrillo, Noel Blaya Boluda, E. García-Torralba, P. Jiménez-Fonseca, Carmen González Del Rey, Milagros Balbín, Ginés Luengo-Gil, F. Ayala de la Peña, Emilio Esteban González, A. Carmona-Bayonas
          </td>
          <td>2025-05-04</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1269bfebfd62817cb97479986e42e37f874297f4" target='_blank'>
              Abstract 4371: PHI-501 as a potent pan-RAF/DDRs inhibitor suppresses lung cancer cell proliferation and overcomes KRAS G12C inhibitor resistance
              </a>
            </td>
          <td>
            An-Na Moon, Ji Young Kim, Hee-sun Hwang, Gi-Jun Sung, Jinsun Kwon, Ky-Youb Nam, Kyu-Tae Kim, J. Han, Jeong-Hyeok Yoon, Ki-Young Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) typically develops in the context of chronic liver disease, where prolonged hepatocyte exposure to inflammation drives the synergistic accumulation of genetic and epigenetic alterations. Epigenetic regulation encompasses multiple mechanisms that govern the transcription machinery accessibility to DNA. This process is regulated by the addition and removal of covalent marks on chromatin, which can either affect DNA-histone interactions or serve as scaffolds for other proteins, among other mechanisms. Recent research has revealed that epigenetic alterations can disrupt chromatin homeostasis, redirecting transcriptional regulation to favour cancer-promoting states. Consequently, these alterations play a pivotal role in the acquisition of cancer hallmarks and provide insights into several biological processes involved in hepatocarcinogenesis. This review highlights the key epigenetic mechanisms underlying the development, progression and dissemination of HCC, with a particular focus on DNA methylation and histone post-translational modifications. This knowledge is relevant for guiding the development of innovative therapeutic approaches based on epigenetic modulators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1d7979671681364e95ca7a0ffe9920c08634f1" target='_blank'>
              Epigenetic mechanisms involved in hepatocellular carcinoma development and progression
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite Garcia Fernandez de Barrena, Matias Antonio Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-05-01</td>
          <td>eGastroenterology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract The SETD2 tumour suppressor encodes a histone methyltransferase that specifically trimethylates histone H3 on lysine 36 (H3K36me3), a key histone mark implicated in the maintenance of genomic integrity among other functions. We found that SETD2 protein deficiency, mirrored by H3K36me3 deficiency, is a nearly universal event in advanced‐phase chronic myeloid leukemia (CML) patients. Similarly, K562 and KCL22 cell lines exhibited markedly reduced or undetectable SETD2/H3K36me3 levels, respectively. This resulted from altered SETD2 protein turnover rather than mutations or transcriptional downregulation, and proteasome inhibition led to the accumulation of hyper‐ubiquitinated SETD2 and to H3K36me3 rescue suggesting that a functional SETD2 protein is produced but abnormally degraded. We demonstrated that phosphorylation by Aurora‐A kinase and ubiquitination by MDM2 plays a key role in the proteasome‐mediated degradation of SETD2. Moreover, we found that SETD2 and H3K36me3 loss impinges on the activation and proficiency of homologous recombination and mismatch repair. Finally, we showed that proteasome and Aurora‐A kinase inhibitors, acting via SETD2/H3K36me3 rescue, are effective in inducing apoptosis and reducing clonogenic growth in cell lines and primary cells from advanced‐phase patients. Taken together, our results point to SETD2/H3K36me3 deficiency as a mechanism, already identified by our group in systemic mastocytosis, that is reversible, druggable, and BCR::ABL1‐independent, able to cooperate with BCR::ABL1 in driving genetic instability in CML. Key Points Virtually all CML patients in blast crisis display SETD2 loss of function. SETD2 loss seems to be accomplished at the posttranslational level rather than being the result of genetic/genomic hits or transcriptional repression. Phosphorylation by Aurora kinase A and ubiquitination by MDM2 contribute to SETD2 proteasome‐mediated degradation in blast crisis CML patients. Loss of SETD2 results in increased DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58c369299bf404553dfac73a905cd3d8e7f9f95" target='_blank'>
              SETD2 loss of function is a recurrent event in advanced‐phase chronic myeloid leukemia and contributes to genomic instability
              </a>
            </td>
          <td>
            M. Mancini, S. De Santis, C. Monaldi, F. Castagnetti, M. Iezza, A. Iurlo, D. Cattaneo, Sara Galimberti, M. Cerrano, I. Capodanno, Massimiliano Bonifacio, Maura Rossi, Claudio Agostinelli, M. Meggendorfer, Torsten Haferlach, M. Cavo, G. Gugliotta, Simona Soverini
          </td>
          <td>2025-04-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0431d2b0b8dd854cb5fd756e0cd0875bd2691218" target='_blank'>
              CRISPR/Cas9-mediated generation of two isogenic CEP290-mutated iPSC lines
              </a>
            </td>
          <td>
            Joana Figueiro-Silva, Melanie Eschment, Michelle Mennel, Affef Abidi, B. Oneda, Anita Rauch, R. Bachmann-Gagescu
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3454a8d3322c6500bdc824693cdc309f89f675f" target='_blank'>
              Abstract 1393: Functional analysis of mitochondria-targeting Wnt-inhibitors on metabolically distinct breast cancer cell types
              </a>
            </td>
          <td>
            Y. Kam, Lisa Winer, N. Romero
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d8faa11ff0e47828029568767559cbff4cf0c6" target='_blank'>
              Abstract CT194: First-in-human, phase 1/2 study of GSK4418959, an oral Werner DNA helicase inhibitor, alone or combined with other anticancer agents in adults with advanced mismatch repair-deficient or microsatellite instability-high solid tumors (SYLVER)
              </a>
            </td>
          <td>
            Vivek Samnotra, Natalia Ramos-Hernandez, Thomas Faitg, Meenakshi Srinivasan, Tania Roy, Andrew Liu, Tomasz Zaremba, Clare Holford, Rachel Smith, Jason T Henry, Jeanne Tie, N. Yamamoto, Gul Y. Serbest
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfdfff54622d4227a75b1cb40c4cc408256a9de" target='_blank'>
              DNA Damage Response and Repair Genes and Anthracycline-Induced Cardiomyopathy in Childhood Cancer Survivors: A Report From the Children’s Oncology Group and the Childhood Cancer Survivor Study
              </a>
            </td>
          <td>
            Xuexia Wang, Purnima Singh, Romina B Cejas, Liting Zhou, Noha Sharafeldin, P. Trainor, Wendy Landier, Changde Cheng, L. Hageman, Fan Wang, Y. Sapkota, Yutaka Yasui, M. Hudson, Eric J Chow, S. Armenian, Joseph P. Neglia, D. S. Hawkins, JillP Ginsberg, P. Burridge, Gregory T Armstrong, Smita Bhatia
          </td>
          <td>2025-03-28</td>
          <td>Circulation: Genomic and Precision Medicine</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy.
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb67a4446cf1d38e1dfaa6a69287fa69917e1c7" target='_blank'>
              Abstract 2735: The role of ca2 in regulating 3d chromatin structure in tamoxifen resistant breast cancer
              </a>
            </td>
          <td>
            Lavanya Choppavarapu, Kun Fang, Tianxiang Liu, A. Ohihoin, Victor X. Jin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739935357776c471010c005564687898973a29a6" target='_blank'>
              Epidemiology, pathogenesis, biology and evolving management of MSI-H/dMMR cancers.
              </a>
            </td>
          <td>
            M. Ambrosini, P. Manca, Vincenzo Nasca, C. Sciortino, F. Ghelardi, Jenny F Seligmann, J. Taieb, F. Pietrantonio
          </td>
          <td>2025-04-03</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is crucial for maintaining genomic stability by repairing DNA damage. Despite its importance, HRR's role in cancer progression is not fully elucidated. Here, this work shows that nuclear-localized branched-chain α-ketoacid dehydrogenase kinase (BCKDK) acts as a modulator of HRR, promoting cell resistance against DNA damage-inducing therapy in breast cancer. Mechanistically, this work demonstrates that BCKDK is localized in the nucleus and phosphorylates RNF8 at Ser157, preventing the ubiquitin-mediated degradation of RAD51, thereby facilitating HRR-mediated DNA repair under replication stress. Notably, aberrant expression of the BCKDK/p-RNF8/RAD51 axis correlates with breast cancer progression and poor patient survival. Furthermore, this work identifies a small molecule inhibitor of BCKDK, GSK180736A, that disrupts its HRR function and exhibits strong tumor suppression when combined with DNA damage-inducing drugs. Collectively, this study reveals a new role of BCKDK in regulating HRR, independent of its metabolic function, presenting it as a potential therapeutic target and predictive biomarker in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac4a9c9be6494a377793b137fd2fa59c3a2efc2e" target='_blank'>
              Nuclear-Localized BCKDK Facilitates Homologous Recombination Repair to Support Breast Cancer Progression and Therapy Resistance.
              </a>
            </td>
          <td>
            Haiying Liu, Jiaqian Feng, Tingting Pan, Pinggen Zhang, Ling Ye, Zetan Jiang, Zilong Zhou, Qiankun Mao, Jian Li, Xinyi Yang, P. Gao, De Huang, Huafeng Zhang
          </td>
          <td>2025-04-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca4cdf73484b348317c78d3c6466b5fa0ebab2b" target='_blank'>
              Abstract 7053: Proteomics and integrative pathway analysis to understand the mechanism of action of mithramycin and etoposide combination induced cell death in Ewing sarcoma cells
              </a>
            </td>
          <td>
            C. Lambring, K. Zaman, Elin Stone, Natalie Gierat, Ameya Bhargava, Laszlo Prokai, Riyaz Basha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 To date, there is an unmet need for novel approaches that address haploinsufficiency disorders in a targeted and robust manner. Here, we developed a novel, modular tool for transcriptional regulation of genes involved in neurodevelopmental disease in the absence of creating double-stranded breaks. We utilized this novel platform technology to identify sgRNAs that efficiently regulate expression of the intellectual disability gene FOXG1 that utilizes an optimized protein intein system. Using a modular assembly of CRISPR trans-activators, including single and dual VP64 dCas9 fusion proteins, sgRNA-mediated MS2 p65 HSF (MPH) recruitment or the addition of the VPR tripartite effector (VP64, p65, Rta) can induce dynamic gene regulation. We further increased mRNA levels of FOXG1 using duplexed sgRNAs paired with different combinations of transcriptional activators. We demonstrate that FOXG1 gene upregulation results in a significant increase in FoxG1 protein translation. Following treatment of mutant FOXG1 neural stem cells with the CRISPR activators, we observed a significant increase in FOXG1 expression with no-off targets as determined by in silico sgRNA binding prediction paired with RNA-sequencing. This approach holds great promise as a modular platform technology that is readily adapted for gene therapy delivery for those affected by rare genetic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc7fd173fd5669ad14660b53ec6364396f64ec51" target='_blank'>
              A modular CRISPRa system for molecular therapy of FOXG1 syndrome
              </a>
            </td>
          <td>
            J. Halmai, Casiana E. Gonzalez, P. Deng, Jasmine L Carter, J. Waldo, D. Cameron, Angels Almenar-Queralt, Lauro Thiago Turaça, A. Muotri, Kyle D. Fink
          </td>
          <td>2025-04-17</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Missense mutations in the BCR::ABL1 kinase domain are found in approximately 12–80% of patients with chronic myeloid leukemia (CML). Clinically significant mutations include T315I, M244V, Y253H/F, E255K/V, V299L, and F359V. The aim of this study was to create a diagnostic system for rapid and inexpensive detection of the above mutations. We used genomic DNA and RNA from peripheral blood and bone marrow cells of 57 patients with a Ph-positive CML diagnosis established in the chronic phase. We have developed a method to detect mutations in the BCR::ABL1 gene based on allele-specific real-time polymerase chain reaction (AS-PCR). In parallel, we analyzed the RNA sequence of the protein kinase domain of the same samples by next-generation sequencing (NGS) covering the points of putative mutations. In this work, we compared the results obtained by both methods for mutation detection and variant allele frequency (VAF) estimation of mutated vs. normal alleles. The sensitivity and specificity of our diagnostic system were also evaluated. It was found that AS-PCR gives reliable results at VAF up to 0.01%. AS-PCR has high sensitivity and may serve as an alternative for the more time-consuming NGS in some cases, as well as for monitoring CML treatment and for analyzing archival material.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc35b97b7ff1497c87471f2609685fa153e61f6d" target='_blank'>
              Allele-Specific PCR for Detection of Missense Mutations in the Chimeric BCR::ABL1 Gene Causing Failure of Tyrosine Kinase Inhibitor Therapy in CML Patients
              </a>
            </td>
          <td>
            Anastasia Skripkina, I. Fevraleva, E. Kuzmina, B. Biderman, E. Stepanova, E. Chelysheva, Anna Turkina, A. Sudarikov
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea10556f7e26587dbcdc0c9ad631944d5c7f3c43" target='_blank'>
              Abstract 3824: Investigating the role of allelic imbalance in pancreatic cancer aggressiveness, survival, and therapeutic resistance
              </a>
            </td>
          <td>
            Enrico Gurreri, Li Zhang, Hania Khan, Lisa Maria Mustachio, Ningping Feng, T. Heffernan, Joseph R Marszalek, Basturk Olca, David Klimstra, Stefano Sioletic, A. Sgambato, G. Tortora, Giulio F. Draetta, Luigi Perelli, G. Genovese
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba26ff57f58d982ea6db2c2b425446ece055593" target='_blank'>
              Exploring the Prognostic and Predictive Impact of Genomic Loss of Heterozygosity and Homologous Recombination Deficiency Alterations in Patients With Metastatic Colorectal Cancer
              </a>
            </td>
          <td>
            R. Moretto, M. Germani, M. Carullo, V. Conca, A. Minelli, M. Giordano, Rossella Bruno, D. Rossini, Eleonora Gusmaroli, M. D. De Grandis, C. Antoniotti, L. Salvatore, A. Passardi, S. Tamberi, M. Scartozzi, F. Pietrantonio, S. Lonardi, Clara Ugolini, G. Masi, C. Cremolini
          </td>
          <td>2025-04-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Protein evolution has emerged as a crucial tool for generating proteins with novel characteristics. A key step in protein evolution is the mutagenesis of protein-coding DNA. Error-prone PCR (epPCR) is a frequently used technique, but its low mutation efficiency often requires multiple rounds of mutagenesis, which can be time-consuming. To address this, we developed a novel DNA mutagenesis strategy termed deaminase-driven random mutation (DRM). DRM utilizes the engineered cytidine deaminase A3A-RL and the engineered adenosine deaminase ABE8e to introduce a broad spectrum of mutations, including C-to-T, G-to-A, A-to-G, and T-to-C, in both the protein-coding strand and the complementary strand. This approach enables the generation of a multitude of DNA mutation types within a single round of mutagenesis, resulting in a higher DNA mutagenic capability than epPCR. The results show that the DRM strategy exhibits a 14.6-fold higher DNA mutation frequency and produces a 27.7-fold greater diversity of mutation types compared to epPCR, enabling a more comprehensive exploration of the genetic landscape. This enhanced mutagenic capability increases the chances of discovering novel and useful mutants. With its ability to produce high-quality DNA products and the superior protein mutant generation capacity, DRM is an attractive tool for researchers seeking to engineer new proteins or improve existing ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0303a56f35114863026aa1824085dd5f1e4dcf02" target='_blank'>
              Deaminase-driven random mutation enables efficient DNA mutagenesis for protein evolution
              </a>
            </td>
          <td>
            Ying Hao, T. Ji, Shu-Yi Gu, Shan Zhang, Yao-Hua Gu, Xia Guo, Li Zeng, Fang-Yin Gang, Jun Xiong, Yu-Qi Feng, Neng-Bin Xie, Bifeng Yuan
          </td>
          <td>2025-04-15</td>
          <td>Chemical Science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed706044b2c945f86bdba6c75c9b7f8653518a59" target='_blank'>
              Abstract 3068: The clonal nature of FGFR3-TACC3 in glioma: Mechanism discovery and preclinical study
              </a>
            </td>
          <td>
            Yiming Li, Tao Li, Xiude Ren, Jianshen Liang, Jikang Fan, Xisen Wang, Sheng-ping Yu, Wei Zhang, Xuejun Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f7c0a88fb3654ac3464cd672c3b86897a3782a" target='_blank'>
              KPT-330, A New Candidate Drug for Targeting NOTCH1 Overexpression in T-cell Acute Lymphoblastic Leukemia, An In Vitro and Silico Study.
              </a>
            </td>
          <td>
            Fariha Naz, Saima Ejaz, Tayyaba Wali, Atiqa Nudrat, Mian Abdur Rehaman Arif, Muhammad Qandeel Waheed, Fatma Hussain, Saadiya Zia
          </td>
          <td>2025-04-11</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c687713915048ef2b6e851768a7161399616a610" target='_blank'>
              Abstract 6416: Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            David J. Falvo, A. Grimont, R. Chandwani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3217dae134c4119cc94589ae992593aeb93f60c2" target='_blank'>
              KRAS mutations in colorectal cancer: impacts on tumor microenvironment and therapeutic implications.
              </a>
            </td>
          <td>
            Anita Emami, Pouya Mahdavi Sharif, Nima Rezaei
          </td>
          <td>2025-05-04</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17eeae7cebe3a01a8802644a89b867255b13f516" target='_blank'>
              Abstract 7043: HMGA1 Epigenetic Regulators Drive Relapse in Pediatric B-Cell Leukemia By Amplifying ETV5 and Stemness Networks
              </a>
            </td>
          <td>
            Linda S. Resar, Jung-Hyun Kim, Andrei Thomas-Tikonenko, S. Tasian
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Chromosomal alterations are frequent events in lung cancer progression. Although gains and losses of chromosomal position have been reported, the association between copy number alteration and lung cancer patient survival has not been extensively investigated. In this study, we performed a meta-analysis of public cBioPortal datasets spanning 25 lung cancer studies to identify putative cancer driver genes with copy number alterations associated with overall patient survival. Ten copy-number altered genes enriched in deceased lung cancer patients were identified. Seven of these putative driver genes were located in the 7p11.2 chromosomal location, and two were in the 9p21.3 cytoband. Among these genes, the mitochondrial ribosomal protein S17 (MRPS17) amplification was significantly associated with a lower patient survival rate (P = 1.47e-7). To investigate the functional role of MRPS17, small interfering RNA-mediated knockdown was performed in two non-small cell lung cancer cell lines, A549 and NCI-H460. MRPS17 knockdown significantly reduced cell proliferation, migration, invasion, and anchorage-independent growth in both cell lines. Furthermore, knockdown of MRPS17 decreased the activation of the phosphatidylinositol 3-kinase/protein kinase B signaling pathway, suggesting its role in driving lung cancer progression through this critical oncogenic pathway. Our findings highlight MRPS17 as a potential cancer therapy target and a prognostic biomarker that may improve the survival rates of lung cancer patients. Future studies should explore its inhibition as a therapeutic strategy as well as elucidate its molecular mechanisms in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1c0b37271452f619e3684f3550b14101ddebda" target='_blank'>
              Mitochondrial Ribosomal Protein S17 Silencing Inhibits Proliferation and Invasiveness of Lung Cancer Cells
              </a>
            </td>
          <td>
            Woo Rin Lee, Kook Sun Ha
          </td>
          <td>2025-03-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa91244e52369ba86bd81040f0cf520d88eb9d9f" target='_blank'>
              Expansion of a bacterial operon during cancer treatment ameliorates fluoropyrimidine toxicity.
              </a>
            </td>
          <td>
            Kai R. Trepka, Wesley A Kidder, Than S. Kyaw, Taylor Halsey, Christine A Olson, Edwin F Ortega, Cecilia Noecker, Vaibhav Upadhyay, Dalila Stanfield, Paige Steiding, Benjamin G. H. Guthrie, Peter Spanogiannopoulos, Darren Dumlao, Jessie A. Turnbaugh, Matthew D. Stachler, Erin L. Van Blarigan, A. Venook, C. Atreya, P. Turnbaugh
          </td>
          <td>2025-04-16</td>
          <td>Science translational medicine</td>
          <td>1</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b17c8f4c66aab3898b741804dfbd546dbdd92394" target='_blank'>
              Abstract 6825: Novel therapeutic strategies targeting MECOM rearranged AMLs discovered by unbiased drug screen
              </a>
            </td>
          <td>
            Christine E. Birdwell, W. Fiskus, Christopher P. Mill, T. Kadia, Naval G Daver, C. Dinardo, K. Sasaki, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, R. K. Shamanna, Michele Ceribelli, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867439245770812a0b6fc8a701b4e620584bcc27" target='_blank'>
              A genome-wide association study using HapMap cell lines reveals modulators of cellular response to cyclophosphamide.
              </a>
            </td>
          <td>
            Mohammed O Gbadamosi, N. Bhise, T. Ghosh, Elizabeth K Molchan, Kathleen Streeks, Jason Puglise, Alyssa Ohaegbulam, Mariana Makarem, Oluwaseyi Olabige, Changlin Yang, Luisel J Ricks-Santi, Duane A Mitchell, Brooke L Fridley, J. Lamba
          </td>
          <td>2025-05-13</td>
          <td>Future oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0e44d4f49dc3c34ac8ec8c2f4c7c94f3fd34e" target='_blank'>
              Abstract 2854: The role of NONO in genome stability: Implications for RNA-DNA hybrid accumulation, genomic instability, and inflammatory signaling
              </a>
            </td>
          <td>
            Vikas Gaur, Sara Knowles, Weihang Chai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>