<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-12 09:41:44 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>81</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Oncogene-targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance. Therapeutic approaches to treat tumours that have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms. Rather than treating resistance after it emerges, it may be possible to prevent it by inhibiting the adaptive processes that initiate resistance, but these are poorly understood. Here we report that residual cancer persister cells that survive oncogene-targeted therapy are growth arrested by drug stress-induced intrinsic type I interferon signalling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA fragmentation factor B (also known as caspase-activated DNase), which induces DNA damage, mutagenesis and stress response factor activating transcription factor 3 (ATF3). ATF3 limits activator protein 1-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DNA fragmentation factor B or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes the regrowth of residual cancer cells that survive drug treatment. Williams et al. report a growth arrest mechanism in residual cancer persister cells through targeted therapy-induced upregulation of type I interferon signalling, which is negatively regulated by apoptotic DNA endonuclease DFFB to allow tumour relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0eb23b0262f48e292a21b5682ac6f6faf0a4d24" target='_blank'>
              DNA fragmentation factor B suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F. Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X Wang, B.E. Mauch, Rhea Plawat, Ariel H. Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J. Hangauer
          </td>
          <td>2025-11-17</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8adbd0387ac6e2aa073dbe75080e4f98049b516f" target='_blank'>
              Acute Myeloid Leukemia Relapse after Bromodomain Inhibitor Treatment or Chemotherapy is Characterized by Myc-Ras Transcriptional Remodeling
              </a>
            </td>
          <td>
            Benjamin J. Huang, Jeevitha D’Souza, Alexa Rane Batingana, M. Harris, Xinyue Wang, E. Hwang, Anica M Wandler, Michael R. Burgess, Qing Li, S. Meshinchi, Gideon Bollag, Kevin Shannon
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="High-risk neuroblastoma is a pediatric tumor that originates from immature sympathetic neuroblasts and is characterized by a low mutational burden. Nevertheless, it frequently presents with highly recurrent chromosomal imbalances, including MYCN amplification and gain of chromosome 17q. Highly proliferative cancers, such as neuroblastoma, exhibit significant DNA replication stress, rendering tumor cells dependent on ATR-CHK1 signalling and DNA damage repair pathways. We previously hypothesized that gene dosage effects resulting from 17q copy number alterations of non-mutated genes involved in replication stress resistance could offer novel therapeutic opportunities. To identify critical candidate genes and pathways driving high-risk neuroblastoma, we performed an integrated bioinformatics analysis, and identified the BRIP1 gene, encoding the FANCJ protein, as top-ranked 17q candidate. FANCJ is involved in multiple processes that alleviate replication stress. In the absence of specific FANCJ-targeting compounds, we evaluated the phenotypic and molecular effects of pharmacological inhibition of the MUS81 endonuclease, which functions downstream of FANCJ in restarting stalled replication forks. When combined with CHK1 inhibition, we observed synergistic effects on neuroblastoma cell growth and survival, supporting further development of on-target MUS81 inhibitors for in vivo preclinical testing and future clinical trials aimed at overcoming replication stress resistance in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, P. Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e46bc8969d218f90b6869d6ad86ad7ec80c28324" target='_blank'>
              Loss of ESE3/EHF is sufficient to promote cell plasticity, transformation and androgen-independent status in the early stage of prostate carcinogenesis
              </a>
            </td>
          <td>
            D. Albino, G. Sandrini, C. Musumeci, Elisa Storelli, Giovanni Papa, Daniela Impellizzieri, A. Cacciatore, G. Cassanmagnago, Atik Balla, Simone Mosole, G. Civenni, Andrea Rinaldi, Carmen Jeronimo, Rui M. Henrique, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Background Undifferentiated pleomorphic sarcoma (UPS) is a rare and aggressive soft tissue sarcoma with limited treatment options and a poor prognosis. As a complex karyotype tumor, UPS lacks recurrent targetable mutations, and response rates to standard first-line doxorubicin therapy are low. Phenotypic drug screening offers an alternative approach to identify new therapeutic targets without requiring prior knowledge of molecular mechanisms. Methods A library of FDA-approved compounds and a custom histone deacetylase (HDAC) inhibitor library were screened using well-annotated patient-derived cell lines. Hit compounds were further characterized using apoptosis assays and in vivo xenograft studies. Biomarkers of activity were evaluated using gene expression and western blot analyses. Synergy with doxorubicin was evaluated in combination assays. Results HDAC inhibitors emerged as a promising therapeutic class, demonstrating low IC50 values across cell lines (14.8−26.89 nM), with quisinostat taken forward for further evaluation. Gene expression changes in EPAS1, FOXO1, AMOT, and FOSL1 were observed as potential biomarkers of activity. Combination assays revealed synergy between quisinostat and doxorubicin (average ZIP score: 1.02−15.65; ZIPmax: 3.98−33.71), increasing apoptotic cell death in vitro. In vivo, quisinostat alone and in combination with doxorubicin significantly reduced the tumor volume (vehicle 160.0 ± 63.2 mm3, doxorubicin 78.0 ± 35.2 mm3, quisinostat 84.3 ± 13.1 mm3, and combination 49.2 ± 10.2 mm3). Quisinostat also showed potent activity in leiomyosarcoma (LMS) cell lines (5.82−31.32 nM), which represent an additional complex karyotype soft tissue sarcoma. Conclusions Quisinostat demonstrated strong preclinical activity and synergy with standard-of-care doxorubicin in models of UPS and LMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/027cd9df5df6321a1733155d1a6f75c50785f55b" target='_blank'>
              High-throughput screening identifies the activity of histone deacetylase inhibitors in patient-derived models of soft tissue sarcoma
              </a>
            </td>
          <td>
            Molly R. Danks, Piotr Manasterski, H. Beetham, John C. Dawson, R. Elliott, Jayne Culley, Rashi Krishna, Morwenna Muir, John P Thomson, A. Oswald, Ailith Ewing, William Kerrison, Paul H Huang, Ioanna Nixon, Neil O. Carragher, V. Brunton
          </td>
          <td>2025-11-30</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, C. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Drug tolerant persister cells (DTPs) refer to a transient drug-tolerance sub-population of cancer cells characteristics of phenotype plasticity and heterogeneity. This adaptive cell state is a critical transitional phase, standing on the crossroad that cancer cells reacquire drug sensitivity or enter into the permanent drug resistance. Emerging evidences indicate the epitranscriptomic regulations, particularly RNA methylations are the important mechanism underline post-transcriptional regulations of genes expression across all RNA species. RNA is integral to gene expression as messenger RNA (mRNA), transfer RNA (tRNA) and ribosomal RNA (rRNA), which play roles in transmitting information from DNA to the synthesis of functional proteins. Methylation modifications on these RNAs are prevalent and represent a well-recognized non-genetic mechanism, exerting multifaceted regulatory effects on nucleic acid metabolism, such as nucleotide precursor availability, RNA processing dynamics, sub-cellular localization, transcript stability and translational fidelity/ efficiency. This review systematically sorts out the relevant references, demonstrating recent advances on the knowledge of the patterns of methylation modifications on mRNA, tRNA and rRNA, and how these modifications drive the generation and development of DTPs, which hallmarks of epithelial-mesenchymal transition, metabolism shift and immune escape. And then clinical strategies are delineated, leveraging pharmacological modulators of RNA-modifying enzymes alongside non-pharmaceutical lifestyle advice, for the development of therapy strategies preventing DTPs-rooted tumor relapse in this anti-tumor armamentarium with cytotoxic reagents, targeted therapies and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098e2b3a70bd6f86f19a2d17ea69012683c3e930" target='_blank'>
              Therapy-induced mRNA, rRNA and tRNA methylation alterations confer tolerance phenotype in tumor cells: mechanism and implications
              </a>
            </td>
          <td>
            Anfeng Jiang, Shujie Liu, Zhiyuan Li, Xiongzhou Zhang, Minghao Duan, Bin Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, often arising from mutations to DNA damage repair and response (DDR) genes. Classical genetic, biochemical, and structural approaches elucidated the foundational mechanisms of DDR pathways and provided a scientific understanding of their involvement in repair of lesions induced by broad classes of DNA-damaging agents (DDAs). However, given the chemical diversity of DDAs and resultant DNA lesions, along with the multitude of interconnected DDR factors, the chemogenomic landscape of DDA–DDR interactions remains incompletely mapped. To this end, we developed a DDR-targeted, CRISPR knockout screening approach and assessed relationships amongst 353 DNA repair genes and 15 DDAs in LN229 glioma cells. Within this dataset of 5295 DDR-related chemogenomic interactions, we identified many established interactions and discovered novel ones. For example, we observed a specific role of transcription-coupled nucleotide excision repair in the repair of adducts generated by monofunctional alkylating agents, a role for the Fanconi anemia pathway in addressing methyl lesions, overt differences in DSB repair following treatment with topoisomerase I versus II poisons, and repair dependencies associated with the imidazotetrazines temozolomide, mitozolomide, and KL-50. Future directions will continue to investigate the mechanisms of novel chemogenomic interactions that we have uncovered as well as work to identify chemogenomic interactions amenable to clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018770119a2496ee321bc85a8d0504634bbf54c9" target='_blank'>
              Targeted CRISPR knockout screening identifies known and novel chemogenomic interactions between DNA damaging agents and DNA repair genes
              </a>
            </td>
          <td>
            Collin D. Heer, James L. Elia, Vijay Menon, Spenser S Johnson, Sofia R. Arbelaez, Sam Friedman, Francesc Lopez-Giraldez, R. Sundaram, Seth B Herzon, R. Bindra, Susan E. Gueble
          </td>
          <td>2026-01-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Resistance to androgen receptor (AR)-targeted therapies such as enzalutamide in castration-resistant prostate cancer (CRPC) often arises through lineage plasticity, yet the molecular mechanisms that define this process remain incompletely understood. While previous studies reported that Notch1 and Notch2 exert distinct and sometimes opposing effects in prostate cancer differentiation, the integrated role of Notch pathway activity has not been systematically explored. Here, we identify Notch signaling as a graded Rheostat that governs prostate cancer cell fate transitions. Integrative transcriptomic and functional analyses revealed that intermediate Notch activity maintains a stem-like progenitor state, whereas reduced or elevated signaling drives divergent differentiation trajectories toward luminal or neuroendocrine lineages, respectively. During CRPC progression and enzalutamide resistance, Notch signaling becomes dynamically rewired, peaking in progenitor-like populations that sustain plasticity and survival. Both CRISPR-mediated knockout and pharmacologic inhibition of Notch signaling depleted these progenitor cells and restored enzalutamide sensitivity. These findings demonstrate that the level, rather than the binary presence, of Notch signaling dictates lineage directionality and therapeutic response in CRPC, establishing it as a tunable and actionable driver of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09956ef93f0e7d80206dd41714c0c97806e5efbf" target='_blank'>
              Graded Notch Signaling Functions as a Rheostat of Lineage Plasticity and Therapy Resistance in Prostate Cancer
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Longjun Li, Melanie Fraidenburg, Choushi Wang, I. Y. Kim, Ping Mu
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Loss-of-function mutations in the TP53 gene are the most frequent genetic events in human malignancies. However, the extent to which missense TP53 mutations confer oncogenic gain-of-function (GOF) properties remains incompletely understood. Using genetically defined pancreatic cancer organoid models, we demonstrated that mutant p53 provides a selective growth advantage under acute environmental stress and drives molecular transition toward a basal-like state. Transcriptomic analyses showed that under stress conditions, organoids expressing mutant p53 adopted a basal-like gene expression program, a characteristic of a poor prognosis subtype of pancreatic cancer. Furthermore, the molecular subtype was reversed to the classical subtype once it returned to normal growth conditions, suggesting that transcriptional plasticity arises from acute changes in the tissue microenvironment. Given these context-dependent oncogenic properties, our findings suggest that targeting mutant p53 may be a promising therapeutic strategy for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f58263187bbd03e5cadcf959b6bde70a5832e40c" target='_blank'>
              Mutant p53 regulates reversible subtype plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            June-Ha Shin, Hwa-Ryeon Kim, Jae-Seok Roe
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The reproducibility of scientific research has been increasingly challenged in recent years. While extrinsic factors (e.g., cross-contamination, mycoplasma infection, serum variability) are well-studied, the role of intrinsic attributes like cell passage number remains underexplored. Using two tumor cell lines (ACHN and Renca), we employed RNA sequencing to analyze transcriptomic dynamics across passages (P3 to P39). Results revealed a nonlinear transcriptomic shift: mid-passage cells (P10/P11, P17) showed heightened activity in cell cycle, metabolism, and stress response, whereas low (P3) and high passages (P24/P39) exhibited stable and similar profiles. KEGG analysis indicated significant alterations in signaling, immune, and metabolic pathways during passaging. This study demonstrates that passage number shapes transcriptional landscapes and impacts experimental reproducibility. We advocate for strict passage control and explicit reporting of passage information to enhance reliability. These findings underscore the need for standardized cell culture practices to improve research reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf829c80ee16d8593d7d0383b2985d0175ddbfa6" target='_blank'>
              Cell passage number drives transcriptomic drift as an overlooked factor in experimental reproducibility
              </a>
            </td>
          <td>
            Shuai Liu, Hai-juan Peng, Junyan Meng, Liyuan Zhuo, Shaoping Fu, Wei Yang
          </td>
          <td>2025-11-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer recurrence remains a major clinical challenge, often associated with therapy resistance and altered metabolic states. To define metabolic vulnerabilities of recurrent disease, we performed a CRISPR knockout screen targeting 421 metabolic genes in paired primary and recurrent HER2-driven breast cancer cell lines. While both primary and recurrent tumors shared dependencies on core metabolic pathways, recurrent tumors exhibited selective essentiality for the de novo pyrimidine synthesis pathway, including Cad, Dhodh, and Ctps. Pharmacologic inhibition of the rate-limiting enzyme DHODH with BAY-2402234 selectively impaired the growth of recurrent tumor cells, while primary tumor cells were relatively resistant. BAY treatment robustly inhibited pyrimidine synthesis in all lines, but only recurrent cells underwent iron-dependent lipid peroxidation and ferroptotic cell death. Lipidomic profiling revealed enrichment of polyunsaturated ether phospholipids in recurrent cells, which may predispose them to ferroptosis. A sensitizer CRISPR screen in primary cells further identified nucleotide salvage and lipid metabolic pathways as modifiers of DHODH inhibitor sensitivity. Stable isotope tracing and nutrient depletion experiments showed that primary cells can compensate for DHODH inhibition through nucleotide salvage, whereas recurrent cells exhibit impaired salvage capacity, likely due to reduced expression of Slc28/Slc29 nucleoside transporters. Together, these findings reveal that breast cancer recurrence is associated with increased dependence on de novo pyrimidine synthesis to suppress ferroptosis, highlighting a therapeutically actionable metabolic vulnerability in recurrent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83ca3ef9a21a542254ed4d0723e6bc92c26d2a66" target='_blank'>
              Recurrent Breast Cancer Cells Depend on De novo Pyrimidine Biosynthesis to Suppress Ferroptosis
              </a>
            </td>
          <td>
            Kelley R. McCutcheon, Josh Wu, Yasemin Ceyhan Ozdemir, Brock J McKinney, Sharan Srinivasan, Ayaha Itokawa, Oliver J. Newsom, Anna-Lena B G Vigil, Douglas B. Fox, Lucas B. Sullivan, James V. Alvarez
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/768d3fe55602ad3e6e64d51b9bc66ae2727e39e3" target='_blank'>
              TP53 and RB1 are predictive genetic biomarkers for sensitivity to cytarabine in gliomas
              </a>
            </td>
          <td>
            L. Laaniste, Joachim Luginbuehl, Mi Trinh, Rachel Watkins, Gregor Lueg, Grant Neilson, Marjolein Burgers, Kasia Kania, Regina Reynolds, Zoe Smith, Julia Kleniuk, Lauren Gould, Dorota Pregowska, Sophie Mathiesen, Michael Johnson, Louwai Muhammed
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df86684e553e2308bc95a51da59d93a391350fdb" target='_blank'>
              Human CSB-deficient iPSCs exhibit impaired DNA damage repair and stress responses following BPDE exposure in an early developmental model
              </a>
            </td>
          <td>
            Alessia Lofrano, Wasco Wruck, Nina Graffmann, James Adjaye
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, , S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Abstract Acquired resistance presents a major challenge for targeted therapies, with initially responsive tumors eventually reverting underlying vulnerabilities. Our group recently reported that cancers harboring isocitrate dehydrogenase 1/2 (IDH1/2) mutations have defective recruitment of homology-directed repair (HDR) factors to sites of DNA damage and consequent sensitivity to poly(ADP-ribose) polymerase inhibitors (PARPi), a vulnerability that is being tested in clinical trials. To probe potential mechanisms by which resistance to PARPi might arise in this setting, we modeled PARPi resistance in IDH-mutant tumors via serial transplantation of patient-derived xenografts in mice treated with PARPi. An analysis of candidate DNA repair factors in these resistant tumor populations identified downregulation of two end protection factors that are negative regulators of HDR, 53BP1, and REV7. Knockout of these factors by CRISPR–Cas9 in IDH1-mutant cancer cells conferred robust resistance to PARPi and restored HDR capacity. To overcome this resistance, we found that treatment with the receptor tyrosine kinase inhibitor, cediranib, previously reported to suppress expression of downstream HDR factors, resensitizes 53BP1 and REV7-knockout cells to PARPi treatment. Our findings identify key pathways driving PARPi resistance in IDH1-mutant cancers and highlight potential therapeutic strategies to overcome this resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef230260548104420dc4aabb1c7dae9c604e7081" target='_blank'>
              PARP inhibitor resistance in IDH1-mutant cancers due to loss of end protection factors, 53BP1 and REV7
              </a>
            </td>
          <td>
            Daniel A. Colon-Rios, Jonathan Dow, Adam Krysztofiak, Yanfeng Liu, Faye A. Rogers, Peter M. Glazer
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4fd78fdc6e7168376ba52f3c22a1f28b8371667" target='_blank'>
              Transcriptomic profiling of epigenetic regulators and metabolic reprogramming in human cholangiocarcinoma
              </a>
            </td>
          <td>
            Amaya López-Pascual, Jasmin Elurbide, Emiliana Valbuena-Goiricelaya, M. Latasa, Elena Anaya, Elena Adán-Villaescusa, Borja Castelló-Uribe, L. A. Martínez-Pérez, I. Uriarte, M. Arechederra, Sergio Ciordia, Fernando Corrales, Sona Frankova, Eva Sticová, Ondřej Fabián, L. Colyn, Patricia Inacio, Robert Arnes-Benito, J. Bayo, Meritxell Huch, C. Berasain, M. G. Fernández-Barrena, Matías A. Avila
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Transcriptional dysregulation has emerged as a critical driver of melanoma progression, yet the molecular mechanisms governing this process and their potential as therapeutic targets remain inadequately characterized. Here, we identify FRA1 as a potent and actionable driver of melanoma metastasis. FRA1 enhanced both the initial seeding and subsequent outgrowth of metastatic lesions. Comprehensive multi-omics integration revealed transcriptional target genes of FRA1, with AXL, CDK6, and FSCN1 exhibiting increased expression in melanoma metastasis and a significant correlation with poor patient outcomes. Silencing AXL, CDK6, or FSCN1 abrogated FRA1-mediated invasion in vitro and reduced metastatic colonization. Furthermore, pharmacological inhibition of CDK6 and FSCN1, and to a lesser extent AXL, suppressed melanoma metastasis and prolonged overall survival. The expression of FRA1 and its target genes correlates with shortened survival across multiple cancer types, highlighting the broader clinical relevance of this pathway. This study unveils an actionable FRA1-mediated transcriptional network that drives cancer progression and metastasis, offering potential avenues for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7942610022450bea58a8d74747c2996140dfd5be" target='_blank'>
              FRA1 drives melanoma metastasis through an actionable transcriptional network
              </a>
            </td>
          <td>
            Xiaonan Xu, Vinesh Jarajapu, Manon Chadourne, Zulaida Soto-Vargas, Shaaron L. Ochoa-Rios, Neel Jasani, Kaizhen Wang, Nicol Mecozzi, Jiqiang Yao, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-11-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Therapeutic failure in cancer often arises from the ability of tumor cells to adapt to external stressors such as drug exposure, metabolic and oxidative stress, and immune surveillance within the tumor microenvironment (TME). This adaptive capacity is a significant factor contributing to poor clinical outcomes and the lethality associated with many aggressive cancer types. During tumor progression, a subpopulation of cells may acquire stem-like characteristics, leading to the development of cancer stem cells (CSCs), which play a pivotal role in resistance and disease recurrence. A key driver of this process is epigenetic reprogramming, which alters histone modifications and chromatin architecture in response to environmental stimuli. These modifications activate genes associated with stemness by creating transcriptionally permissive chromatin regions, while simultaneously silencing genes involved in differentiation, thereby maintaining the tumor cells’ undifferentiated state. This cellular plasticity enables cancer cells to undergo transdifferentiation or dedifferentiation, enhancing their capacity for survival and adaptation. Importantly, distinct transcriptional states are often regulated by specific epigenetic signatures, some of which are valuable for tumor subtype classification, prognosis assessment, and treatment response prediction. Furthermore, certain epigenetic biomarkers show considerable promise for detecting tumor recurrence and monitoring minimal residual disease with high sensitivity. In this review, we examine how epigenetic reprogramming influences chromatin plasticity to promote stemness and therapeutic resistance. Additionally, we discuss ongoing translational research aimed at leveraging these mechanisms to advance precision oncology, with the ultimate goal of improving diagnostic accuracy and therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management. The impact of tumour heterogeneity on metastatic potential in prostate cancer remains poorly understood. Here, the analysis of single nuclei RNA sequencing and whole-genome sequencing from samples from five patients suggests an interplay between clonal evolution and cellular plasticity driving metastatic seeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Cancer drug resistance, whether intrinsic or acquired, underlies most relapses and treatment failures. Reliable preclinical models are crucial to define resistance mechanisms and test counterstrategies. This overview article concisely compares three model classes: (1) clinical, patient‑derived xenografts that retain tumor heterogeneity and can mirror patient resistance; (2) induced‑resistance models produced by prolonged drug selection in vivo or in vitro that recapitulate tumor evolution under therapy; and (3) engineered isogenic cell lines that isolate specific resistance drivers. We summarize some key resistance mechanisms revealed and potential therapeutic approaches informed by these models, including rational combinations, mutation‑targeted inhibitors/degraders, efflux/epigenetic modulators, and immune‐related combinations. Each model has trade‑offs, but integrating them accelerates mechanistic insight and translational drug development. This overview guides selection of preclinical models and design of strategies to overcome cancer drug resistance. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/329bb2e5a0a76225e84bcf3ca5e48cf0783cbcff" target='_blank'>
              Overview of Preclinical Drug‐Resistant Tumor Models and Their Application in Drug Discovery
              </a>
            </td>
          <td>
            Bin Li, Ting Ni, Dong Wang, Gaoxiang Liu, Wenting Shi, Zhixiang Zhang, Qingyang Gu, Jian Xiang
          </td>
          <td>2025-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29acdf562d4b29c5b781b23b586561ef03a93de" target='_blank'>
              Pan-cancer analysis reveals genomic fidelity and evolution of patient-derived organoids
              </a>
            </td>
          <td>
            Linoy Raz, Tal Ben-Yishay, Haia Khoury, Ron Saad, Amjad Zoabi, Giulia Orlandi, S. Donzelli, Giovanni Blandino, Uri Ben-David
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The rapid emergence of multidrug-resistant Mycobacterium tuberculosis (Mtb) threatens global TB control, yet the mechanisms enabling rapid evolution of drug resistance in Mtb remain poorly understood. Here we reveal that pre-existing mutations in oxidative stress response genes create permissive genomic backgrounds that accelerate high-level isoniazid resistance (INHR) without fitness costs, challenging the paradigm that resistance mutations always precede their fitness compensatory adaptations. Using M. smegmatis mc2155 (Msm) as a model, we show that brief exposure to sublethal INH (2× IC50) enriches for “low-level resistance and tolerance” (LLRT) mutants in a single step. These LLRT mutants, particularly those with ohrR loss-of-function mutations, acquire high-level resistance (> 500× IC50) at 6-fold higher rates than wildtype, primarily through otherwise-deleterious mycothiol biosynthesis mutations that become tolerable in the oxidative stress-buffered background. Crucially, we demonstrate that sublethal oxidative stress alone, mimicking host immune pressure, nearly tripled the rate of INH resistance evolution in Msm. Bayesian analysis of 1,578 clinical Mtb isolates from Vietnam confirmed that mutations in oxidative stress response genes were significantly associated with the emergence of INHR strains (p-value = 1.09×10−7). Independently, reanalysis of genome-wide CRISPRi screens revealed that the OSR network and high Bayes probability genes are functionally associated with treatment escape and survival with multiple antibiotics, including isoniazid, rifampicin, ethambutol, bedaquiline, vancomycin, clarithromycin, linezolid, and streptomycin. Our findings that host-imposed oxidative stress and inadequate drug penetration may synergistically prime Mtb populations for rapid resistance evolution suggest that targeting pre-resistance mechanisms, such as oxidative stress defenses, could help slow the emergence of antibiotic resistance in tuberculosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97167fa83fb26f5ed168ddabdf26926250881a65" target='_blank'>
              Host oxidative stress primes mycobacteria for rapid antibiotic resistance evolution
              </a>
            </td>
          <td>
            N. Baliga, Evan Pepper-Tunick, Vivek Srinivas, Fred D. Mast, Song Li, Sagan Russ, Weston Hanson, Amy Zamora, Wei-Ju Wu, Matthew Silcocks, Dang T.M. Ha, S. Dunstan, Nguyen Thuong, S. Turkarslan, J. Aitchison, Mario L. Arrieta-Ortiz
          </td>
          <td>2025-12-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7247a45454e42ff687acafe0ee75660b9dd942d" target='_blank'>
              A modular lentiviral system for multiplexed gene perturbation and functional analysis reveals interdependence of hormone receptors in breast cancer growth in vivo
              </a>
            </td>
          <td>
            Seher Alam, Daria Matvienko, Johan Boström, Maria J. Pires, Aikaterini Papadimitropoulou, Saeed Eshtad, Kumar Sanjiv, Hong Qian, Nicholas C. K. Valerie, C. Brisken, Mikael Altun
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Comparative genetic studies of primates have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short (≤31 base pairs) hDels has been assessed using reporter assays. However, how structural variant-sized (≥50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 20 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the brain. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants. Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Here the authors use CRISPR-based perturbations and epigenetic profiling in chimpanzee cells to identify gene regulatory roles for genomic sequence lost in the human lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d81c8c2f5bb5b9ea0191569860cee91545d0b60" target='_blank'>
              Mapping cis- and trans-regulatory target genes of human-specific deletions
              </a>
            </td>
          <td>
            Tyler Fair, Bryan J. Pavlovic, Dani Swope, Octavio E. Castillo, Nathan K. Schaefer, Alex A. Pollen
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e91ffccc5754debd09b64e59ef74200817ed97" target='_blank'>
              Transcriptomic and epigenetic profiling of neuroblastoma states in response to RBM39 degrader.
              </a>
            </td>
          <td>
            Hongjian Jin, Jie Fang, Jason Myers, Shivendra Singh, Jun Yang
          </td>
          <td>2025-12-09</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84948bcfa1aec61d5686e9537d4f4d0902383f" target='_blank'>
              ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation
              </a>
            </td>
          <td>
            A. Nakayashiki, Arata Nagai, Yuko Iwasaki, Abdulrahman Mousa, Moe Kumai, Munehiro Kimura, Shadi Al-Mesitef, Hiroshi Kanno, S. Yamashita, Daisuke Ando, Yoshiteru Shimoda, Masayuki Kanamori, Keisuke Goda, Hidenori Endo, Sherif Rashad, Kuniyasu Niizuma
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Motivation Understanding double-strand break (DSB) repair and its disruption is key to decipher genomic instability driving diseases such as cancer and reveal therapeutic avenues. Numerous genes have been linked to DSB repair for the first time in recent genome-wide perturbation studies assessing effects on mutational outcomes. However, the functional roles of most such genes remain poorly understood. Evidence from other studies shows that related genes similarly modulate the frequency of specific mutational outcomes following DSB repair, but analysis has largely ignored the multiplicity of gene functions and cross-talk between pathways. Here, we infer functional roles for candidate genes based on knockout mutational spectra by identifying mutational signatures shared with known genes and then mapping them to DSB repair functions. Signatures are identified using non-negative matrix factorization (NMF) to reflect functional multiplicity and cross-talk. Results We generated mutational spectra for mouse embryonic stem (mES) cells at three target sites across CRISPR knockouts of 307 known and 459 candidate DSB repair genes. Analysis using NMF revealed four mutational signatures associated with homology-directed repair (HDR), microhomology-mediated end-joining (MMEJ), and the initiation and ligation components of non-homologous end-joining (NHEJ). Signatures suggested that candidate genes Dbr1 and Hnrnpk could influence MMEJ and Fanconi anaemia (FA), and that loss of core NHEJ components (e.g. MRN complex or Ku proteins) could shift repair preference towards Ku-independent NHEJ. These findings demonstrate the utility of NMF for characterizing the contribution of genes to repair pathways and provide a foundation to discover new gene functions in DSB repair. Availability github.com/joanagoncalveslab/MuSpectraNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c23f53baca0dc3b384bd4e216c9b8503ff2be41" target='_blank'>
              Pooled single-cell CRISPRa/i screens for functional genomics in bacteria at scale
              </a>
            </td>
          <td>
            Jacob R. Brandner, Quoc Tran, Yujia Huang, Dmitry Sutormin, Karl D. Gaisser, Georg Seelig, J. Zalatan, James M. Carothers, A. Kuchina
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Replicative immortality is a hallmark of cancer, driven by the activation of telomere maintenance mechanisms, that is yet to be therapeutically exploited. To expedite discoveries that will enable the development of therapeutics that target telomere maintenance mechanisms, this study provides a resource of telomere biology metrics for a pan-cancer panel of 976 cell lines. We generate proteomic data from data-independent-acquisition mass spectrometry for most of these cell lines and integrate pre-existing multi-omic, drug sensitivity, and molecular dependency data from CRISPR/Cas9 knock-out screens. The data illustrate a broad range and heterogeneity in telomere biology, including states that diverge from the binary model of telomere maintenance activation involving either telomerase or the Alternative Lengthening of Telomeres mechanism. Using the telomere biology metrics and multi-omic data, we derive proteomic and transcriptomic predictors of Alternative Lengthening of Telomeres and telomerase activity levels. Our investigations also reveal molecular vulnerabilities associated with the Alternative Lengthening of Telomeres mechanism and drug sensitivity correlating with telomerase activity levels. These findings illustrate opportunities for leveraging this resource to realize the potential for telomere biology-directed cancer therapeutics and companion diagnostics. The therapeutic relevance of telomere maintenance mechanisms in cancer, remains to be explored. Here, the authors integrate multi-omic data and functional readouts, generate a resource of telomere biology metrics and identify potential molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc8e08e7ae63f123dc686a0aa51e8c03fa2c2c5" target='_blank'>
              Large-scale drug sensitivity, gene dependency, and proteogenomic analyses of telomere maintenance mechanisms in cancer cells
              </a>
            </td>
          <td>
            Yangxiu Wu, Zhaoxiang Cai, Dale Cross, Jane R. Noble, Kelsy Prest, Jamie Littleboy, Scott B. Cohen, Baylee Edlundh, Jennifer M. S. Koh, Ran Xu, Zainab Noor, Milad Bastami, Sara Valentini, Laura Richardson, S. Barthorpe, Nader Arymanesh, Phillip J. Robinson, P. Hains, Mathew J. Garnett, Qing Zhong, R. Reddel, Karen L MacKenzie
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer. Krepelova and colleagues identify an aging- and colon cancer-associated DNA methylation drift in healthy and cancerous human colon samples, echoed in the mouse intestinal epithelium. They show it expands via intestinal cell clonality and is driven by dysregulated iron metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539b2ee1d2d15c9295867b71d2600ec2c2b281c5" target='_blank'>
              MELK controls tumor metabolism to promote resistance to melanoma therapy
              </a>
            </td>
          <td>
            A. C. B. Sant’Anna-Silva, P. Abbe, Sophie Pagnotta, N. Amoedo, Michael D. Torno, Fanny Bonnard, T. Botton, Frédéric Larbret, Cyril Ronco, H. Montaudié, I. Ben-Sahra, C. Robert, Thierry Passeron, Rodrigue Rossignol, Michaël Cerezo, Stéphane Rocchi
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Melanoma brain metastasis (MBM) remains lethal with limited treatment efficacy. Meanwhile, the cellular origins and drivers of brain metastasis from melanoma have yet to be defined. Through integrated single-cell/single-nucleus RNA sequencing of 26 melanoma samples, we identified a pre-brain-metastatic tumor subpopulation (MBMATCs, MBM-associated tumor cells) within a conserved malignant cell trajectory (Mela3). MBMATCs exhibited activated pro-metastatic pathways and upregulated neural/adhesion genes (NRG3, NCAM1), suggesting a cellular origin for brain tropism. MBM ecosystems showed T cell exhaustion (elevated PD-1, HAVCR2, LAG3) and Treg enrichment. An MBM-Index derived from bulk RNA-seq accurately quantifies MBMATCs abundance, independently predicting poor overall survival in both TCGA-SKCM and validation cohorts. Furthermore, we assessed the clinical relevance of the MBM-Index and uncovered five candidate drugs with potential activity against MBMATCs. This study identifies MBMATCs as brain metastasis associated tumor cells and positions the MBM-Index as a biomarker for early stratification of melanoma patients at high risk of brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0332483b66f858c33c245348a73898413f2eeff5" target='_blank'>
              Single-cell transcriptomics profiling reveals cellular origins and molecular drivers underlying melanoma brain metastasis
              </a>
            </td>
          <td>
            Jingxin Zeng, Ling Lin, Yangyang Ma, Wenzhe Huang, Quan Luo, Ju Wen
          </td>
          <td>2025-11-24</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edbcbe66a598eaa27dab20a758b907c70d1442c" target='_blank'>
              Adaptive Responses Directed by CREB Control Epithelial-Mesenchymal Plasticity in Cancer
              </a>
            </td>
          <td>
            Kshitij Parag-Sharma, Harish Bharambe, John J. Powers, W. Gong, Daniel J. Purcell, Caleb Mensah, Chloe Twomey, Jay H. Mehta, Samy Lamouille, Antonio L. Amelio
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains clinically challenging due to its molecular heterogeneity and poor outcomes, highlighting the urgent need for novel biomarkers and therapeutic targets. Purpose This study aims to identify and characterize the role of leucine-rich α-2-glycoprotein 1 (LRG1) in AML, evaluating its potential as both a prognostic biomarker and a therapeutic target. Methods We conducted an integrated basic and clinical investigation of LRG1 in AML. Methods included analysis of LRG1 expression in patient samples versus controls and pre- versus post-treatment, assessment of its clinical correlations with mutations and subtypes, and evaluation of its prognostic impact. Functional validation was performed using LRG1 knockdown models to assess effects on colony formation, apoptosis, and differentiation. Single-cell RNA profiling was utilized to identify LRG1-enriched cell populations and explore its role in microenvironmental crosstalk. Results Integrated analysis revealed significantly elevated LRG1 expression in AML patients compared to controls (P<0.001), with levels decreasing post-treatment (P<0.001). High LRG1 expression correlated with FLT3 mutations (P<0.01), M3-M5 AML subtypes (M0&M1&M2 VS M3, P<0.001; M0&M1&M2 VS M4, P<0.01; M0&M1&M2 VS M5, P<0.001; M3 VS M5, P<0.05; M4 VS M5, P<0.05), and worse survival (P<0.01). Functionally, LRG1 knockdown impaired colony formation (P<0.001), increased apoptosis (P<0.001), and disrupted differentiation (P<0.01). Single-cell profiling identified LRG1 enrichment in hematopoietic stem and progenitor cells (HSPCs) and myeloid progenitors, where it facilitated microenvironmental crosstalk via Macrophage Migration Inhibitory Factor (MIF), Galactoside-binding lectin (GALECTIN), and Cyclophilin A (CypA) signals. Conclusion Our findings establish LRG1 as a robust prognostic biomarker and a key functional regulator of AML maintenance through myeloid progenitor dysregulation, presenting it as a promising target for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c85b059f315aeac16b816e67ba97682c83e790" target='_blank'>
              LRG1 Drives AML Progression by Disrupting Myeloid Progenitor Regulation
              </a>
            </td>
          <td>
            Rongxia Guo, Peng Wu, Shuxin Yao, Fumei Liu
          </td>
          <td>2025-12-01</td>
          <td>Blood and Lymphatic Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As a highly fatal and heterogeneous disease, ovarian cancer (OC) lacks sufficient treatment choices. Over the past decade, RNA sequencing (RNA-seq) has profoundly changed research methods in molecular biology, enabling comprehensive analysis of transcriptomes, somatic mutations, and tumour heterogeneity. In OC research, the application of RNA-seq technologies has significantly enhanced scientists’ understanding of the biological mechanisms underlying the disease, providing deeper insights into the intrinsic mechanisms of key processes such as tumour initiation, metastasis progression, drug resistance formation, and immune evasion. This breakthrough not only expands our understanding of OC but also holds promise for the discovery of novel diagnostic biomarkers, the development of innovative therapies, and the establishment of precise prognostic assessment systems. This review summarizes RNA-seq technologies and their applications in OC research, while also discussing the challenges and opportunities for their broader implementation in cancer studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38a5fc4d3383fd0f8d4c8ba650bad14685267c27" target='_blank'>
              RNA sequencing in ovarian cancer research: a comprehensive review
              </a>
            </td>
          <td>
            Fang Shen, Chan Li, Yue Li, Ying Shen, Fengjuan Han
          </td>
          <td>2025-11-24</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d6f15706cb0580ba50eb7f865a8457afcd0809" target='_blank'>
              The aging transcriptome: a condition-dependent response to a deteriorating soma
              </a>
            </td>
          <td>
            Z. A. Syed, Martin I. Brengdahl, V. Shenoi, Avani Mital, Christopher M. Kimber, Helen Ekman, Claudio Mirabello, Malin Larsson, David Berger, Urban Friberg
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT DNA surveillance mechanisms are crucial for maintaining genome stability and minimizing mutation rates. Mismatch repair (MMR) corrects replication errors that escape DNA-polymerase proofreading. In most organisms, MMR is orchestrated by MutS and MutL proteins. However, certain Archaea and Actinobacteria, including the major human pathogen Mycobacterium tuberculosis, lack these components. Instead, they appear to rely on the nuclease EndoMS/NucS, a structurally distinct enzyme governing a non-canonical MMR pathway. Since M. tuberculosis acquires drug resistance exclusively through chromosomal mutations, understanding its mutation rate regulation is critical. Nevertheless, the role of NucS in drug resistance evolution remains largely unexplored. We investigated NucS function in M. tuberculosis and uncovered a unique resistance dynamic distinct from other Actinobacteria. Deleting nucS altered the mutational spectrum but had minimal impact on the emergence of antibiotic-resistant mutants, contrasting sharply with other Actinobacteria where nucS loss dramatically increases mutation rates. This atypical behavior cannot be attributed solely to the R144S NucS polymorphism present in the H37Rv reference strain. Introducing the consensus NucS sequence in H37Rv produced only subtle changes in mutational spectrum without major effects on mutation rates. Analysis of 44,921 M. tuberculosis genomes revealed that most R144S-containing strains belong to the Euro-American sub-lineage 4.9, with no significant association with antibiotic resistance. However, nucS is under strong purifying selection, and R144S changes arose independently during M. tuberculosis evolution (homoplasy). Overall, our findings challenge the view that nucS serves as a genome stability guardian in M. tuberculosis and suggest additional mismatch repair mechanism(s) beyond NucS in this pathogen. IMPORTANCE DNA repair systems are crucial for maintaining the integrity of genetic information by scanning and correcting errors that arise during DNA replication. Most organisms use well-characterized proteins (MutS and MutL) for this task, but some bacteria and archaea, including most Actinobacteria, lack these conventional components. Instead, they employ an alternative enzyme, NucS, to fix replicative DNA errors. This alternative pathway is thought to limit harmful mutations and, in turn, reduce the likelihood of drug resistance development. In our study of Mycobacterium tuberculosis, the pathogen responsible for tuberculosis, we found that NucS plays only a minor role in preventing resistance-associated mutations. This unexpected result challenges current assumptions about DNA repair in this pathogen and points to the existence of other, yet unidentified mechanisms that safeguard its genome. Understanding these processes could open new avenues for therapeutic strategies against tuberculosis, a disease that remains a major global health threat. DNA repair systems are crucial for maintaining the integrity of genetic information by scanning and correcting errors that arise during DNA replication. Most organisms use well-characterized proteins (MutS and MutL) for this task, but some bacteria and archaea, including most Actinobacteria, lack these conventional components. Instead, they employ an alternative enzyme, NucS, to fix replicative DNA errors. This alternative pathway is thought to limit harmful mutations and, in turn, reduce the likelihood of drug resistance development. In our study of Mycobacterium tuberculosis, the pathogen responsible for tuberculosis, we found that NucS plays only a minor role in preventing resistance-associated mutations. This unexpected result challenges current assumptions about DNA repair in this pathogen and points to the existence of other, yet unidentified mechanisms that safeguard its genome. Understanding these processes could open new avenues for therapeutic strategies against tuberculosis, a disease that remains a major global health threat.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d734f5ee057101186e729eca6288974decb87e6d" target='_blank'>
              The unique role of nucS-mediated noncanonical mismatch repair in Mycobacterium tuberculosis resistance evolution
              </a>
            </td>
          <td>
            I. Martín-Blecua, Jorge Sastre-Domínguez, J. R. Valverde, Pablo García-Bravo, Ángel Ruiz-Enamorado, Rafael Prados-Rosales, Lahari Das, William R. Jacobs, Á. San Millán, Jesús Blázquez, Sonia Gullón
          </td>
          <td>2025-12-22</td>
          <td>mBio</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Bladder cancer (BCa) is a prevalent malignancy and major cause of cancer-related mortality in men, with clinical outcomes still varying despite advances in personalized treatments. High inter- and intra-tumor heterogeneity significantly contributes to this variability. While traditional high-throughput sequencing has provided insights into BCa mechanisms, driver genes, and clinical strategies, it falls short in completely elucidating cellular heterogeneity. Recently, single-cell sequencing (SCS) technologies have substantially enhanced the detection of tumor heterogeneity by improving sensitivity, accuracy, and efficiency. Single-cell transcriptome sequencing offers unbiased, high-resolution analysis of gene expression patterns at the single-cell level, offering essential insights into BCa pathogenesis. This article reviews advancements in SCS technology and its applications in evaluating tumor heterogeneity, the tumor microenvironment, metastasis, and treatment resistance, offering new perspectives for future BCa research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B. Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Cervical cancer, as one of the most common types of malignant tumors in women, poses a significant threat to the health and survival of women worldwide. Tumor drug resistance has emerged as a critical bottleneck severely limiting the efficacy of current cervical cancer treatment strategies, making it imperative to delve deeper into the underlying mechanisms for improved therapeutic outcomes. Spatial transcriptomics and single-cell sequencing technologies were employed. Single-cell sequencing was used to uncover the heterogeneity in cervical cancer development. Pseudotime analysis, exploration of cell communication pathways, and spatial transcriptomics were further applied to investigate the roles of relevant molecules. This study reveals an abundance of PI3 + neoplastic (NEO) in tumors and SLC40A1 + NEO cells in HSIL samples. These cell populations may play crucial roles in the development of drug resistance, as they could potentially alter the tumor microenvironment and interfere with the efficacy of therapeutic agents. Pseudotime analysis, cell communication pathways, and spatial transcriptomics highlight the pivotal role of LGALS9 and its ligands, offering new insights for early diagnosis, molecular typing, prognosis evaluation, and personalized treatment of cervical cancer. Our findings reveal that high expression of LGALS9 suppresses T cell function, fosters a strongly immunosuppressive tumor microenvironment, and is associated with chemotherapy resistance, ineffective immunotherapy, and poor prognosis. LGALS9 may serve as a biomarker for predicting immunotherapy response. This knowledge has the potential to facilitate early diagnosis, precise molecular typing, accurate prognosis evaluation, and the development of personalized treatment strategies that can overcome tumor drug resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b81384b510909f679a401421e801231c6a36b6" target='_blank'>
              A tailored in vivo CRISPR screen identifies BAP1 as a potent tumor suppressor of soft tissue sarcoma
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Zhaoyu Sun, Melissa J. Kasiewicz, Yaping Wu, Brian D. Piening, John T. Welle, Wesley K. Rosales, V. Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, Rutulkumar Patel, Ziqiang Zhang, B. Bernard, William L Redmond, Walter J. Urba, R. B. Bell, D. Kirsch
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a364e2a10884750437094b545faf2c3a6d27319" target='_blank'>
              DNA Methylation Dynamics Reveal Unique Plant Responses and Transcriptional Reprogramming to Combined Heat and Phosphate Deficiency Stress
              </a>
            </td>
          <td>
            Alberto Lozano-Enguita, Victoria Baca-González, Adrián Morillas-Montáez, Jesús Pascual, Luis Valledor, J. C. del Pozo, Elena Caro
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab23a3ffe076670b85b6131789fd95f8b29a8b8c" target='_blank'>
              Valproate Releases the Latent Differentiation Capacity of Neuroblastoma by Targeting SP1/CDK4 Signaling
              </a>
            </td>
          <td>
            Mingwei Su, Yuhan Ma, Xiaoshan Liu, Yu Wu, Xiaomin Peng, Xilin Xiong, Ke Huang, Shouhua Zhang, Yang Li
          </td>
          <td>2026-01-03</td>
          <td>Neurochemical Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes. Class switch recombination (CSR) is a process contributing to antibody diversity and generating characteristic genomic junctions that reflect DNA double-strand break repair efficiency. Here authors present a platform to profile the repertoire of CSR junction patterns; long-read-sequenced PCR products from blood-derived lymphocytes are analysed by a dedicated bioinformatic pipeline to accurately detect immunodeficiencies and DNA repair defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a387e7c68c6390622f2c2d1c86f9db7afbe8e8ce" target='_blank'>
              miRISC inhibition causes mitotic defects and synergizes with genotoxic agents in cancers
              </a>
            </td>
          <td>
            Minsi Zhang, Ziqi Jiao, Ylenia Cendon Florez, Ivana Lessel, Xiaoyi Li, Melissa A. Yao, Moritz Weigl, Mateusz M. Ozimek, Turgut Dogruluk, Kevin Chen, Keith Conrad Fernandez, Vinagolu K. Rajasekhar, Olesja Popow, João A. Paulo, Davide Pradella, Tanmay Mishra, C. Mastroleo, Juliana I. Delgado, Francesco Enrico D’Amico, Vincenzo Cavalieri, Carol D. Morris, M. B. Tsou, Kevin M. Haigis, Jayanta Chaudhuri, R. Benezra, Joana A. Vidigal, Davor Lessel, Andrea Ventura, Gaspare La Rocca
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Amino acid metabolism plays a critical role in tumor growth and immune regulation, yet its comprehensive function in breast cancer remains underexplored. We developed an amino acid metabolism–related gene signature (AAMRGS) to predict prognosis and therapeutic response in breast cancer. The AAMRGS was constructed using a machine-learning framework integrating ten algorithms and validated across multiple independent cohorts. It served as an independent prognostic factor and outperformed existing amino acid metabolism–related signatures and clinical variables. Moreover, the prognostic utility of AAMRGS was further validated across pan-cancer datasets, and an AAMRGS-based nomogram was constructed to facilitate clinical application. Functional enrichment and protein–protein interaction analyses revealed that AAMRGS genes were primarily involved in metabolic reprogramming and cell proliferation. Experimental validation confirmed the downregulation of key genes such as SAV1 and IGF2R in breast cancer cells. Integrative analyses revealed that the high-AAMRGS subgroup exhibited a greater copy number variation burden, higher tumor mutation burden (TMB), enrichment of immunosuppressive cell populations, and increased sensitivity to most chemotherapeutic drugs. In contrast, the low-AAMRGS subgroup displayed higher immune scores, stronger immune activation, enrichment of anti-tumor immune cells, and greater responsiveness to immunotherapy. Collectively, our findings establish AAMRGS as a reliable prognostic signature and a potential tool to guide individualized therapeutic strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/012e3e3011711fffcec4b5947c341bef4c09870d" target='_blank'>
              Establishment of an amino acid metabolism related signature for prognostic and therapeutic sensitivity prediction in breast cancer by machine learning
              </a>
            </td>
          <td>
            Xinrui Zhao, , Nan Hu, Xiaoming Wu, Junbo Duan
          </td>
          <td>2026-01-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeted therapy has revolutionized the treatment landscape of hepatocellular carcinoma (HCC), offering unprecedented hope to patients. However, despite its promise, significant challenges arise in the form of drug resistance. Only a fraction of HCC patients respond to targeted therapy, and even those who respond often develop resistance over time. Sorafenib and lenvatinib, the sole first-line targeted therapeutic drugs for HCC, face severe clinical limitations due to drug resistance. Understanding the mechanisms underlying sorafenib/lenvatinib resistance is crucial for maximizing treatment efficacy. Recent studies have highlighted the pivotal role of epigenetic regulation in drug resistance. Cancer is recognized as both a genetic and epigenetic disease, with epigenetic factors influencing various aspects of tumor cell biology, especially drug resistance. This review systematically summarizes the mechanisms of epigenetic-mediated sorafenib/lenvatinib resistance, encompassing non-coding RNA (ncRNA) regulation, DNA methylation, RNA methylation, and histone modification. These abnormal epigenetic processes typically influence biological activities, including escaped programmed cell death, tumor metabolic reprogramming, formation and maintenance of drug-resistant cells, uncontrolled cell proliferation signaling pathways, and abnormal transport processes, ultimately culminating in profound drug resistance. By comprehensively summarizing the latest discoveries in epigenetic regulation mechanisms, this review highlights potential strategies to overcome drug resistance, paving the way for future advancements in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8f7f2274db91d56455b33e92f98f5f8697e415" target='_blank'>
              Resistance of first-line targeted drugs in hepatocellular carcinoma: the epigenetic regulation mechanisms
              </a>
            </td>
          <td>
            Ding Qi, Yongqing Qin, Haidong Zhu, Yong Li, Shisong Han
          </td>
          <td>2025-11-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, limiting the efficiency and precision of genome editing in many clinically relevant tissues. Here, we address this barrier by using induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. CRISPR editing outcomes differ dramatically in neurons compared to genetically identical dividing cells: neurons take longer to fully resolve this damage, and upregulate non-canonical DNA repair factors in the process. Manipulating this response with chemical or genetic perturbations allows us to direct DNA repair toward desired editing outcomes in nondividing human neurons, cardiomyocytes, and primary T cells. By studying DNA repair in clinically relevant cells, we reveal unforeseen challenges and opportunities for precise therapeutic editing. Genome editing tools can precisely introduce a specified lesion into the DNA, but ultimately rely on the cell’s DNA repair machinery to determine the editing outcome. Here, authors demonstrate how neurons’ unique DNA repair pathways impact the safety, efficiency, and precision of CRISPR edits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3be0abd5edf19f4fcddf4c37efa7ec4d61c80" target='_blank'>
              Characterizing and controlling CRISPR repair outcomes in nondividing human cells
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J. Namaganda, Manasi M. Kumar, J. Hamilton, Rohit Sharma, Karena G. Chow, Luke A. Workley, Bria L. Macklin, Mengyuan Sun, Alvin Ha, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Philip H. Dierks, Rudra Bose, Julianne Jin, Barbara S. Perez, Cindy R. Sandoval Espinoza, Madeline P. Matia, Serena H. Lu, Luke M. Judge, Brian R. Shy, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2025-11-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) undergoes profound metabolic reprogramming to fuel its aggressive progression and metastatic dissemination. While transketolase (TKT), a central metabolic enzyme, has been shown to exert dichotomous roles as either oncogenic or tumor-suppressive factors across different malignancies, its functional significance in RCC pathogenesis remains inadequately defined. In this study, we demonstrate that TKT promotes glucose metabolism in RCC by enhancing glycolysis, thereby supporting tumor progression. TKT expression is significantly elevated in RCC tissues and correlates with poor patient prognosis. Mechanistically, we uncovered a novel functional axis between TKT and the glycolytic gatekeeper pyruvate kinase M2 (PKM2), where their coordinated action drives metastatic progression and metabolic adaptation in RCC. Knockdown of PKM2 significantly impaired the TKT-mediated increases in glycolysis, cell proliferation, and invasive potential. Taken together, our findings highlight TKT as a pivotal regulator of metabolic reprogramming in RCC and suggest its potential as a therapeutic target for the treatment of this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78dbaa7d0abd5e148ab4d82b25d037d1f670c014" target='_blank'>
              TKT drives renal cell carcinoma progression through metabolic reprogramming and synergistic interaction with PKM2
              </a>
            </td>
          <td>
            Qianqing Wang, Anqun Tang, Qingxin Zhuang, Haifeng Xu, Shiping Xu, Jing Zhang, Yao Wang, Liantao Li, Sufang Chu, Yan Wang, Jin Bai, Minle Li, Rui Zhang
          </td>
          <td>2025-11-18</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/420f96e213065a56fe1298cd2a47d84efe8c11e0" target='_blank'>
              Pooled CRISPRi screening reveals fungal-specific vulnerabilities across environments and genetic backgrounds
              </a>
            </td>
          <td>
            Lauren F Wensing, Philippe C. Després, Desiree Francis, Meea Fogal, Anthony Hendriks, Nicholas C. Gervais, Clara Fikry, Abdul-Rahman Adamu Bukari, A. Gerstein, Christina A. Cuomo, Rebecca S. Shapiro
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, Fang Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8a70e1ae8c9ca8d6322c082b4e20481405aa734" target='_blank'>
              Genome-wide CRISPR screen reveals PEX11B as a host restriction factor against ORFV through membrane fluidity regulation
              </a>
            </td>
          <td>
            Xiaoran Gao, Jinyang Hao, Sha Lu, Shida Wang, Yu Sun, Xin Ke, Xiaoran Gao, Yuan Su, Yuze Sun, Yu Tian, Wenyu Yan, Jinliang Wang, Rong Hai, Qianyi Zhang, Jingfei Wang, Wei Hu, Guojun Wang
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Importance of Study Protein phosphatase magnesium-dependent 1D (PPM1D) is frequently mutated in diffuse midline gliomas (DMGs). DMGs are rare pediatric brain tumors with limited treatment options. Due to the cancer’s rapid progression, patients usually survive 12-24 months after diagnosis. This underscores the critical need to better understand the molecular mechanisms driving DMGs. This study describes a novel mouse model that provides a powerful platform to investigate PPM1D-driven tumor biology and offers mechanistic insights into disease development and progression. Furthermore, it serves as a valuable preclinical system for evaluating therapeutic strategies and identifying translational opportunities to target Ppm1d-mutant tumors. Background Diffuse midline gliomas (DMGs) are incurable brain tumors with limited treatment options. Approximately 20% of DMGs harbor truncating mutations in exon 6 of phosphatase PPM1D, which stabilize the protein and deregulate p53 signaling. However, the consequences of these mutations for tumor initiation, progression, and therapy remain unclear. Methods We developed a conditional Ppm1d-loxP-exon6-loxP-exon6-E518X-tag mouse allele (Ppm1d-flex-6) that enables lineage-, spatial-, and temporal-specific expression of a DMG-derived truncated Ppm1d protein from its endogenous locus in the presence of Cre-recombinase. Ubiquitous activation of mutant Ppm1d was modeled using the Meox2-Cre driver, and primary gliomas were modeled using the RCAS/tv-a system to introduce Cre and PDGFB co-drivers into Nestin-positive neural stem cells. Complementary studies were performed in mouse embryonic fibroblasts (MEFs) expressing truncated Ppm1d following Cre recombination. Results While Meox2-Cre-driven ubiquitous recombination of Ppm1d-flex-6 produced muted phenotypes, Ppm1d-flex-6 recombination in Nestin+ neural stem cells accelerated gliomagenesis. Its oncogenic effect was weaker than complete p53 loss, and it did not accelerate tumorigenesis further in p53-null tumors. Single-cell RNA-sequencing revealed that Ppm1d-flex-6 gliomas adopt more progenitor-like transcriptional states and upregulate p53- and cell cycle associated pathways. In MEFs, Ppm1d-flex-6 enhanced proliferation and shifted transcriptomic programs toward MAPK and PI3K-Akt signaling, while impairing DNA damage responses, including reduced γ-H2AX induction after irradiation. These defects sensitized cells to radiation and decreased clonogenic survival after ionizing radiation and PARP inhibition. Conclusions Ppm1d mutations confer intermediate suppression of the p53 pathway, consistent with the clinical features of PPM1D-mutant DMGs and are associated with radiosensitivity and PARP inhibitor vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b526ced19406db51fd6c39018ffced7037a7861b" target='_blank'>
              Oncogenic Ppm1d mutations deregulate the p53 pathway in primary mouse gliomas
              </a>
            </td>
          <td>
            Vennesa Valentine, Abigail J. Groth, Joshua Tolliver, Avneesh Saravanapavan, Spencer M. Maingi, Loren B. Weidenhammer, Nerissa T. Williams, Lixia Luo, Matthew S. Waitkus, Sheeba Jacob, Zachary J. Reitman
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer, ranking among the most prevalent malignancies in males worldwide, is undergoing a significant evolution in therapeutic paradigms from conventional approaches to precision medicine, with recent advances in targeted therapies offering novel strategic insights. This review delineates the molecular foundations of prostate carcinogenesis, elucidating pivotal domains including genetic mutations, hormonal regulation, tumor microenvironment dynamics, cell cycle dysregulation, epigenetic modifications, and tumor heterogeneity. Furthermore, we evaluate the clinical translation of targeted strategies such as AR signaling axis inhibition, PI3K/AKT/mTOR pathway modulation, DNA damage repair machinery exploitation, prostate-specific membrane antigen -directed interventions, and combinatorial immunotherapy. Concurrent challenges—AR-driven heterogeneity, adaptive drug resistance mechanisms, spliceosomal vulnerabilities, and scarcity of selective molecular targets—are critically analyzed. Notwithstanding these obstacles, targeted therapies exhibit considerable potential to enhance therapeutic efficacy while mitigating systemic toxicities, paving the way for more personalized and precision-oriented oncologic care. By underscoring the imperative to decode prostate cancer’s molecular architecture, this work outlines future research priorities and advances a robust scientific framework for innovation in therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6680208f5ea18e2a4fa84ced337f25d04b9c91c6" target='_blank'>
              Unravelling targeted therapy in prostate cancer: from molecular mechanisms to translational opportunities
              </a>
            </td>
          <td>
            Litong Wu, Junfeng Qiu, Zhi-ming Hong, Quan Wang, Qixin Li, Wenbin Zhou
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24dfa7ddd758cdfb84fb3b4e9f07b2ebd47b469" target='_blank'>
              Hidden-driver inference reveals synergistic brain-penetrant therapies for medulloblastoma
              </a>
            </td>
          <td>
            Jingjing Liu, Xu Yang, Mingrui Zhu, Xinran Dong, Honglei Zhou, B. Bianski, Barbara Jonchere, Wenwei Lin, Xiang Fu, Lei Yang, Burgess B. Freeman, Abigail S. Wang, Ruilin Jiang, Taosheng Chen, G. Robinson, M. Roussel, T. E. Merchant, Amar Gajjar, Jiyang Yu
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cervical cancer is among the prevalent gynecological malignancies affecting women worldwide. The diverse cellular populations and their intricate communication patterns within the tumor environment significantly influence cancer development and metastasis. Our research focused on elucidating the cellular diversity, intercellular signaling networks, and underlying molecular processes in cervical cancer tissues via single-cell transcriptomic approaches. Publicly available single-cell RNA sequencing (scRNA-seq) data for cervical cancer (GSE236738) were re-analysed to delineate the cellular composition and transcriptional programs of the tumour microenvironment. We applied UMAP and t-SNE algorithms for dimensionality reduction, alongside intercellular communication mapping, MIF pathway analysis, temporal trajectory modeling, and pathway enrichment studies, to comprehensively characterize the cellular architecture and molecular profiles within cervical cancer tissues. In addition, paired tumour and adjacent non-tumour cervical tissues from 20 patients were subjected to quantitative real-time PCR and enzyme-linked immunosorbent assay to confirm the transcriptional and protein-level expression of key genes identified by scRNA-seq. Analysis revealed substantial cellular and molecular diversity within cervical cancer specimens. Ten distinct cellular populations were characterized, including NK lymphocytes, epithelial components, and macrophage subsets, indicating pronounced cellular heterogeneity. Intercellular communication mapping revealed that stromal and epithelial populations are the primary coordinators of cell-to-cell signaling. MIF pathway activity peaked in cancer-associated fibroblasts, which exhibited distinct cell type distributions. Temporal analysis revealed dynamic cellular states and pinpointed critical regulatory elements, including ISG15. Pathway analysis emphasized the importance of neutrophil activity, mitochondrial respiration, and cell adhesion mechanisms in cancer progression. qPCR confirmed that ISG15 and TNFRSF18 were significantly upregulated in tumor tissues, while SDF4 was downregulated and NOC2L showed a moderate increase. ELISA results were consistent with transcript-level findings, demonstrating significant protein overexpression of ISG15 and TNFRSF18 in tumor tissues compared with controls. Our investigation comprehensively characterized the cellular diversity and molecular features of the cervical cancer microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7329b5a877ad7165acf8e0004372f128c5c7552" target='_blank'>
              Mapping cellular interactions and communication landscapes in cervical cancer via single-cell transcriptomics
              </a>
            </td>
          <td>
            Shaoli Zhuang, Xiaoxiao Deng, Furong Fu, Zhang Zhang
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Gene amplification is thought to be common in bacterial populations, providing a rapid and reversible mode of adaptation to diverse stresses, including the acquisition of antibiotic resistance. We previously showed that the opportunistic pathogen Staphylococcus aureus evolves resistance to the dual-targeting fluoroquinolone delafloxacin (DLX) that inhibits both the DNA gyrase and DNA topoisomerase IV via amplification of an efflux pump encoding gene sdrM. However, the pathways that control gene amplification, and consequently adaptive trajectories, remain understudied, especially in gram-positive bacteria like S. aureus. Here, we show that specific DNA repair and chromosomal separation proteins alter the frequency of gene amplification and selection of amplified regions in S. aureus. Through a screen of 40 mutants deficient in various DNA processes, we determined that while sdrM amplification was still the almost universal path to DLX resistance, other mutations that increased sdrM expression reduced the selection frequency of sdrM amplification, demonstrating the critical role of sdrM in DLX resistance. We found that similar to other bacteria, both sdrM amplification and loss of amplified gene copies required a functional RecA recombinase, but multiple other mutants in pathways required for amplification in other species still exhibited frequent sdrM amplification, suggesting that S. aureus may have alternate routes of gene amplification. Finally, loss of function mutants of the tyrosine recombinase XerC, that is known to play a role in chromosomal separation, were deficient for sdrM amplification, indicating that XerC is a novel modulator of gene amplification, or the maintenance or selection of amplified gene copies. Thus, our work sheds light on genetic factors that alter gene amplification-mediated evolutionary trajectories to antibiotic resistance in S. aureus and can potentially unlock mechanisms by which such evolution of resistance can be inhibited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875b466f28280308897b4f66650c929c02cc31e9" target='_blank'>
              Modulators of gene amplification alter evolution of antibiotic resistance in Staphylococcus aureus.
              </a>
            </td>
          <td>
            Kalinga Pavan T Silva, Anthony M Martini, Anupama Khare
          </td>
          <td>2025-12-31</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="MYC-driven medulloblastoma (MB) is a highly aggressive brain tumor with poor prognosis and limited treatment options. Through CRISPR-Cas9 screening, we identify the Mediator-associated kinase CDK8 as a critical regulator of MYC-driven MB. Both genetic loss and pharmacological inhibition of CDK8 impair MB tumor growth. Moreover, we find that CDK8 cooperates with MYC to sustain the MYC-mediated translational program, as CDK8 depletion induces pronounced transcriptional changes in translation-associated gene sets, reduces ribosome biogenesis, and impairs protein synthesis. Mechanistically, CDK8 regulates the occupancy of RNA polymerase II at specific chromatin loci, facilitating epigenetic alterations that promote the transcription of ribosomal genes. Furthermore, combined inhibition of CDK8 and mTOR synergistically enhances therapeutic efficacy in vivo, leading to more pronounced tumor growth suppression. Overall, our findings establish a functional link between CDK8-mediated transcriptional regulation and mRNA translation, suggesting a promising therapeutic approach targeting protein synthesis for MYC-driven MB. MYC-driven medulloblastoma is an aggressive pediatric tumor with limited treatment options. Here, the authors show that CDK8 regulates ribosome biogenesis and that combined inhibition of CDK8 and mTOR demonstrates therapeutic efficacy in mouse models of this cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57987b24c602508744e2198d54727be7f386ebb5" target='_blank'>
              Transcriptional regulation of protein synthesis by mediator kinase represents a therapeutic vulnerability in MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Dong Wang, Caitlin Ritz, Yuhuan Luo, Ammu Suresh, Angela M Pierce, Bethany L. Veo, Breauna Brunt, Nathan A. Dahl, Natalie J Serkova, S. Venkataraman, Etienne P Danis, Kamil Kuś, Milena Mazan, T. Rzymski, Rajeev Vibhakar
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The DNA Damage Repair (DDR) signaling pathway serves as a crucial molecular hub that regulates chemotherapy efficacy, offering significant translational value in the field of precision oncology. This review systematically analyzes the molecular mechanisms of five core DDR pathways (Homologous Recombination Repair, Non‐Homologous End Joining, Base Excision Repair, Nucleotide Excision Repair, and Mismatch Repair) in mediating chemotherapy resistance in tumors, and thoroughly elucidates the correlation between key molecular events—such as BRCA1/2 deficiency, MMR functional abnormalities, and Ataxia Telangiectasia Mutated/Ataxia Telangiectasia and Rad3‐related (ATM/ATR) signaling pathway dysregulation—and chemotherapy sensitivity. The DDR deficiency biomarker system established through the integration of multi‐omics data provides molecular classification tools for predicting the efficacy of platinum‐based drugs. This study focuses on the mechanism by which Poly ADP‐Ribose Polymerase inhibitors reverse Homologous Recombination‐Deficient tumor resistance through “synthetic lethality” effects while also revealing the synergistic anti‐tumor effects of ATM/ATR inhibitors in combination with chemotherapeutic agents. The research presents an innovative molecular synergy model between DDR regulation and Immune Checkpoint Blockade, confirming that tumor neoantigen release induced by DDR deficiency can enhance immunotherapy responses. This article also provides perspectives on multidimensional intervention strategies based on the DDR network, including the development of inhibitors targeting novel DDR targets, the establishment of DDR pathway functional assessment systems based on multidimensional biomarkers, and the investigation of synergistic paradigms between DDR and novel therapeutic modalities. Additionally, we explore the dynamic evolution mechanisms of DDR‐mediated chemotherapy resistance by analyzing the interactions between DDR and metabolic reprogramming, as well as other related processes. These breakthrough advances provide theoretical foundations and innovative directions for overcoming chemotherapy resistance and advancing personalized treatment, marking a new era in cancer therapy characterized by precision targeting of DDR pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/944e141ebbf24071dde5281d6930372251bae23d" target='_blank'>
              Translational Aspects of DNA Damage Repair in Optimizing Cancer Chemotherapy
              </a>
            </td>
          <td>
            Anqi Lin, Jinyue He, Aimin Jiang, Jian Zhang, Quan Cheng, Hengguo Zhang, Wenjun Mao, Peng Luo
          </td>
          <td>2025-12-01</td>
          <td>Advanced Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are driven by complex molecular interactions between genomic instability and immune modulation. Defects in the DNA damage response (DDR) not only promote tumor heterogeneity but also shape the tumor immune landscape through the generation of neoantigens, activation of the cGAS–STING pathway, and modulation of immune checkpoints. This review provides an integrative overview of the molecular mechanisms linking DDR dysfunction to immune crosstalk, emphasizing how these processes influence tumor evolution and response to therapy. We discuss emerging therapeutic strategies that exploit DDR–immune interactions, including PARP and ATR inhibitors, synthetic lethality approaches, and combination regimens with immune checkpoint blockade. Understanding the bidirectional connection between DNA repair pathways and immune signaling unveils new translational opportunities for precision oncology and offers a framework for developing combinatorial therapies capable of overcoming resistance and improving long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855e668433efed3e8846b48896d51f3b8c724981" target='_blank'>
              Targeting DNA Damage Response and Immune Crosstalk in Cancer: Mechanistic Insights and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Lavinia Marcut, Roxana Brata, A. Barb, Alexia Manole, Dan Gabriel Stef, C. Dumitru, Flavia Zară, Raul Pătrașcu
          </td>
          <td>2025-11-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer therapy remains an active field of investigation, particularly in understanding and overcoming therapy resistance. Small non-coding RNAs, such as microRNAs (miRNAs), are emerging as key regulators of cancer survival, progression, proliferation, invasion, migration, and metastasis. Although many studies have linked miRNAs to cancer therapy outcomes, significant questions remain regarding their precise molecular and cellular roles in therapy resistance. Increasing evidence shows that miRNAs influence critical pathways such as apoptosis, immune evasion, and other signaling cascades. However, there have been many setbacks because of the limitations in knowledge of each specific miRNA’s function. A deeper understanding of miRNA expression and function may enhance the development of more effective cancer therapeutics and improve overall survival of patients. This review explores the role of miRNA expression as a key regulator of therapeutic resistance in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2eecd6c06795aaec7cf5be8c8739a720372740" target='_blank'>
              MicroRNAs as key regulators of cancer drug resistance: insights and future directions in chemotherapy, targeted-therapy, radiotherapy, and immunotherapy
              </a>
            </td>
          <td>
            Michael A Attathikhun, A. Jurj, George A. Calin
          </td>
          <td>2025-12-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/022b91f34e43e78a47866226e6f399f4d37ba4bc" target='_blank'>
              Lifespan-extending downregulation of insulin signalling reduces germline mutation load
              </a>
            </td>
          <td>
            Elizabeth M. L. Duxbury, A. Godden, Jean-Charles de Coriolis, Hanne Carlsson, Simone Immler, Alexei A. Maklakov
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality globally, with poor survival outcomes in advanced stages due to therapy resistance and metastasis. While long non-coding RNAs (lncRNAs) are emerging as key regulators of CRC progression, their functional interplay and metabolic implications remain poorly understood. Here, we identified LINC02878 as a novel oncogenic lncRNA significantly upregulated in CRC and correlated with poor prognosis. Mechanistically, LINC02878 binds to the ZNF282 to activate PYCR2 expression, thereby enhancing proline biosynthesis and driving tumor aggressiveness. In vivo, LINC02878 knockdown suppressed subcutaneous tumor growth, reduced lung metastasis, and improved survival in xenograft models. Our study unveils the LINC02878/ZNF282/PYCR2/proline axis as a critical metabolic vulnerability in CRC, offering potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e023958dff991207f5ee1dae585ccd9ef299ed60" target='_blank'>
              LINC02878/ZNF282/PYCR2 axis promotes proline synthesis and tumor progression in colorectal cancer
              </a>
            </td>
          <td>
            Jun-Ping Lei, Lin Wang, Tianyang Wang, Bing-Hu Lin, Yan Xu, Song Shang, Tian Xia, Min-Ming Gong, Yu Guo, Jia Fu, Yuan-Lin Li
          </td>
          <td>2025-12-02</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Understanding DNA double-strand break (DSB) repair is crucial for the development of targeted anticancer therapies, yet the roles of many genes remain unclear. Recent studies show that disruption of known DSB repair genes can alter the sequence-specific distribution of mutations arising after DSB repair, suggesting that genome-wide perturbation screens could be leveraged to identify new DSB genes leading to distinct deviations from the expected wild-type distribution. Given the challenges of designing controls for a genome-wide screen, we explore the high gene throughput to forgo the use of traditional controls by reframing the analysis as an outlier detection problem, assuming that most genes have minimal influence on DSB repair outcomes. We propose MUSICiAn (Mutational Signature Catalogue Analysis), a compositional data analysis method that ranks gene perturbation impact on mutational spectra without controls by measuring deviations from the central tendency considering the distribution of all spectra. We show that MUSICiAn effectively estimates pseudo-controls for the Repair-seq screen, yielding 476 genes and 60 nontargeting controls. We further apply MUSICiAn to the first genome-wide screen of 18 406 genes with mutational spectra readout, MUSIC, reporting that MUSICiAn successfully recovers known DSB repair genes, highlights the spliceosome as a lesser-appreciated player, and reveals candidates for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a1d3962960ca885e8c3efafcd749c2130bf3c7" target='_blank'>
              MUSICiAn: genome-wide identification of genes involved in DNA repair via control-free mutational spectra analysis
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, 
          </td>
          <td>2026-01-08</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, Laura Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Understanding how chemotherapy perturbs cell cycle dynamics is critical for advancing cancer treatment. We develop a probabilistic, multi-generational framework based on a Bellman–Harris branching process to quantify treatment-induced shifts in tumor cell dynamics. The model incorporates key drug-responsive behaviors, including checkpoint activation, apoptosis, and checkpoint adaptation that propagates inherited DNA damage, enabling the characterization of heterogeneous survival outcomes after treatment. Biological parameters map directly onto DNA repair fidelity and cell-fate decisions, providing mechanistic insights beyond what is accessible from experiments alone. Dose-dependent extensions further allow exploration of treatment-induced perturbations. Model parameters were calibrated to empirical cell cycle measurements using the robust adaptive Metropolis algorithm. Global sensitivity analysis shows that scale parameters governing unfaithful DNA repair under G2/M- and S phase–specific agents exert major influence on model predictions, particularly at later time points. Across three chemotherapies, the framework reveals consistent dose-dependent alterations in cell cycle dynamics, with higher doses driving pronounced disruptions. Together, these results demonstrate how model-informed analyses can provide quantitative insight into treatment-induced cell cycle perturbations and support the refinement of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130ec8b5cedf3ce34952bca8649e510976ff8a38" target='_blank'>
              Probabilistic modeling of cell cycle dynamics in response to cell cycle targeting chemotherapy drugs to guide treatment strategies
              </a>
            </td>
          <td>
            Chenhui Ma, Evren Gurkan-Cavusoglu
          </td>
          <td>2025-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Senescence is a pleiotropic phenotype that alternatively suppresses or promotes cancer. The tumor suppressive roles are linked to clearance of damaged cells by the immune system, while cells with tumor promoting functions resist apoptosis, evade immune clearance and either persist and support the tumor microenvironment, or escape and proliferate. What generates these diverse populations is unclear. We investigated this in preneoplastic zebrafish livers where the epigenetic regulator, UHRF1, is overexpressed in hepatocytes. Double strand breaks, DNA methylation repatterning, retrotransposon expression, cell cycle withdrawal and activation of Atm and Tp53 dependent senescence were early responses to UHRF1 overexpression. This evolved to generate diverse populations of senescent cells, some which expressed immune and senescence signatures plus anti-apoptotic markers, and others co-expressed proliferative genes. The fate of these populations was dictated by UHRF1 levels and Tp53, as Tp53 loss enabled proliferation of cells with reduced UHRF1 expression but not in cells expressing high UHRF1. The senolytic Navitoclax targeted only a subset of senescent cells. Thus, the diversity of senescent cells driven by epigenetic changes can generate divergent outcomes. Short summary UHRF1 overexpression in zebrafish hepatocytes induces DNA damage leading to Atm–tp53–dependent senescence with UHRF1 levels dictating whether hepatocytes remain senescent, escape, or undergo senolytic elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b990cf440846339c231eccc76316041249b0f0d" target='_blank'>
              UHRF1 Overexpression Generates Distinct Senescent States with Different Tp53 Dependencies
              </a>
            </td>
          <td>
            Elena Magnani, Charlene Chen, F. Macchi, Bhavani P Madakashira, Tijana Randic, Ian McBain, Kirsten C. Sadler
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Neuroblastoma is an embryonal tumour that arises from the malignant transformation of neural crest cells and remains one of the deadliest malignancies in children under five. Neural crest development is regulated by dynamic switches in transcriptional programmes, guided by a variety of growth factors. Due to its developmental origin, neuroblastoma is unique in that these tumours often retain overactivation of growth factor signalling, which can be targeted by receptor tyrosine kinase (RTK) inhibitors. However, mutations in kinases, except for ALK, are extremely rare in neuroblastoma. Furthermore, the high degree of intratumoural heterogeneity often renders RTK inhibition ineffective as a monotherapy. For high-risk tumours, which lack effective treatment options, there remains an unmet need for targeted therapies. This review summarises the roles of growth factor receptors in neural crest and neuroblastoma development in light of recent single-cell studies. We provide a systematic overview of RTK inhibitors that can target growth factor signalling in neuroblastoma and detail their current status in clinical development. We also explore the role of intratumoural heterogeneity in resistance to RTK inhibitors, focusing on the adrenergic-to-mesenchymal transition, which drives a switch in growth factor receptor expression. Finally, we discuss strategies to overcome RTK inhibitor resistance by targeting neuroblastoma cell plasticity, disrupting downstream signalling pathways, or inhibiting escape mechanisms from cell death. This review provides a theoretical basis for developing novel combination therapies incorporating RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251f60bb5eddba4ba1d386fbe214011bbf9166f4" target='_blank'>
              Advances in Targeting Growth Factor Signalling in Neuroblastoma and Overcoming Drug Resistance
              </a>
            </td>
          <td>
            K. Ivanenko, Ruslan Shaymardanov, Vladimir S. Prassolov, Timofey Lebedev
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is now recognized as pivotal to cancer progression and treatment outcomes, alongside cancer cell-intrinsic factors. Among TME constitute, cancer-associated fibroblasts (CAFs) are abundant and actively shape tumour biology. They are heterogenous population arising from multiple origins, and engage in dynamic crosstalk with cancer cells and immune infiltrates to promote virtually all the hallmarks of cancer. By influencing and reinforcement of cancer hallmark traits, CAFs create a pro-tumorigenic niche that also protects cancer cells from therapy, in this way, conferring resistance to conventional therapy. Consequently, CAFs have emerged as central players linking cancer hallmarks to therapeutic resistance. This review discusses the origins and phenotypic diversity of CAFs, the mechanisms of their communication with tumour and immune cells, and how these interactions enforce hallmark capabilities of cancer that underlie treatment resistance. We also highlight preclinical and clinical evidence for CAF-driven drug resistance and examine current strategies to target CAFs or their effects. A deeper understanding of CAF heterogeneity and function could pave the way for novel combination therapies that dismantle the tumour-CAF alliance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10e61bcec209fefda855e7181e3c9e5807e7aba" target='_blank'>
              Cancer-associated fibroblasts: central players in cancer hallmarks and therapeutic resistance
              </a>
            </td>
          <td>
            Yuanjun Lu, Ya Wu, Xiaoyu Xu, Yibin Feng, Ning Wang
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="


 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents, including doxorubicin, are common. Therefore, this project aims to characterize the evolution of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from the ATRT cell line, BT16, and RMC cell line, PEDS005T, utilizing a dose escalation approach. This resulted in DR cell lines that are 10-to-30-fold resistant compared to the parental cell line. Furthermore, we generated two independent DR cell lines from each parental cell line, enabling the capture of distinct evolutionary trajectories leading to resistance. We profiled the transcriptomes by bulk RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1, in 3 of 4 models. To assess whether ABCB1 upregulation alone was sufficient for DR, we treated them with an ABCB1 pharmacological inhibitor and found that ABCB1-upregulated DR cells were re-sensitized to doxorubicin, whereas an ABCB1-negative BT16 DR cell line remained resistant. To determine a shared mechanism of resistance between these models, we profiled the methylomes by Oxford Nanopore Technologies long-read sequencing. We observed global hypermethylation when compared to the parental counterparts. Treatment with a hypomethylating agent reduced cell viability up to 75%. Future studies will focus on identifying and therapeutically targeting these commonalities to overcome resistance regardless of evolutionary trajectory. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there are different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, reduced cell viability resulting from treatment with a hypomethylating agent suggests there are common vulnerabilities despite differing evolutionary trajectories.



 Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong. The evolution of doxorubicin resistance in SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebc129836d1140b7ad59722551b94ec70c22ad5" target='_blank'>
              Abstract B017: The evolution of doxorubicin resistance in
 SMARCB1
 -deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite substantial progress in the field of bladder cancer management, the disease continues to represent an important health issue characterized by increased recurrence and progression rates. This is largely attributed to cancer stem cells (CSCs), a unique cell subpopulation capable of self-renewal, differentiation and resistance to conventional anti-cancer therapies. At the same time, our understanding of cancer biology has been revolutionized by the identification of non-coding RNAs (ncRNAs), a heterogeneous group of RNA molecules that do not translate into proteins yet function as pivotal regulators of gene expression. Emerging evidence demonstrates that ncRNAs modulate key hallmarks of CSCs, including self-renewal, epithelial–mesenchymal transition and drug resistance. This review investigates the intricate interplay between ncRNAs and the core signaling pathways that underlie bladder CSC biology. Unravelling the nexus between CSCs and ncRNAs is crucial for developing novel diagnostic biomarkers, better prognostic tools and innovative therapeutic strategies for patients with bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13091e7103acf49fcdc024b48e3b7a3f0f4f481d" target='_blank'>
              The Crossroads of Cancer Regulation: Discussing the Role of Non-Coding RNAs in Bladder Cancer Stem Cells
              </a>
            </td>
          <td>
            Alexandros Georgiou, Dimitrios Triantis, M. Goulielmaki, V. Zoumpourlis
          </td>
          <td>2025-12-11</td>
          <td>Uro</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metabolic reprogramming is a fundamental hallmark and a key driver of malignant tumors. By reshaping glucose, lipid, and amino acid metabolism, as well as mitochondrial function, it sustains the abnormal proliferation and survival of tumor cells, making it a crucial target for anti-tumor therapy. Curcumin, a natural multi-target compound, exhibits unique advantages in intervening in tumor metabolic reprogramming due to its low toxicity and broad-spectrum regulatory properties. In various tumor models, it can directly modulate the activity of key glycolytic enzymes, such as hexokinase 2, lactate dehydrogenase A, and pyruvate kinase M2, as well as transporters like glucose transporter 1. Furthermore, it inhibits the expression of proteins related to lipid metabolism, including fatty acid synthase and stearoyl-CoA desaturase 1, while also intervening in amino acid metabolic networks, such as glutaminase and branched-chain amino acid transaminase. Additionally, curcumin targets mitochondrial function and reactive oxygen species balance, creating multi-dimensional intervention effects through various pathways, including the induction of ferroptosis by regulating the SLC7A11/GPX4 axis and modulating gut microbiota metabolism. Its mechanism of action involves the synergistic regulation of key signaling pathways, including phosphoinositide 3-kinase/Akt, NF-κB, AMP-activated protein kinase, and hypoxia-inducible factor-1alpha. Furthermore, its specific effect profile demonstrates significant dependency on cell type and tumor model. This article systematically reviews the regulatory effects of curcumin on these critical metabolic processes and pathways in tumor metabolic reprogramming, revealing its molecular mechanisms in disrupting tumor growth and progression by targeting energy and biosynthetic metabolism. These findings provide a significant theoretical foundation and a preclinical research perspective for the development of natural antitumor drugs based on metabolic regulation, as well as for optimizing combination therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ef3c2431a487af7bdd9c9e3b429614616570fa" target='_blank'>
              Curcumin Rewires the Tumor Metabolic Landscape: Mechanisms and Clinical Prospects
              </a>
            </td>
          <td>
            Ding-ya Sun, Dun Hu, Jialu Wang, Xin Li, Jun Peng, Shan Wang
          </td>
          <td>2025-12-23</td>
          <td>Nutrients</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Structural variants (SVs) comprise an axis of genetic diversity with strong consequences for phenotype and fitness, making them a potentially important target for conservation genomics. Here we review how and why SVs can play a role in in conservation genomics; the different types of SVs and how they can affect phenotype; and how pangenomes and long-read sequencing are illuminating their evolution in populations, including small populations and those of conservation concern. SVs comprise multinucleotide mutations including insertions, deletions, transpositions, inversions, and other multinucleotide mutations, often overlapping genes and other functional genome regions. As a result, SVs often play important roles in phenotypic evolution and local adaptation and can contribute substantially to genetic load in inbred populations. However, our understanding of the factors influencing SV diversity in populations is still in its infancy and is complicated by the vast range of sizes, effects, and mechanisms of formation of these mutations. We argue that SVs are an important axis of genetic diversity that should be characterized alongside more traditional metrics of genetic diversity in conservation contexts. There are a number of analytical challenges to detecting and studying SVs, but analyses aimed at understanding the role of SVs in inbreeding load and population health are rapidly becoming realizable goals, accelerated by new technologies and analytical approaches. New tools, including population-scale long-read sequencing and pangenome approaches, are beginning to make SVs accessible in ways that can be readily applied in conservation settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9bdbc3c19da02b9ba08777de00a5f85af83fcc" target='_blank'>
              The value of structural variants to conservation genomics in the pangenome era.
              </a>
            </td>
          <td>
            Teresa M Pegan, Ashwin Sivakumar, Christian Burns, Kelsie A Lopez, Bohao Fang, Nicole M Melendez, Lauren Bartel, Scott V. Edwards
          </td>
          <td>2025-11-24</td>
          <td>The Journal of heredity</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Globally, breast cancer (BC) is the most prevalent cancer in women, and drug resistance is a foremost challenge in BC management that contributes to 90% of BC-related deaths. Drug resistance arises as cellular adaptation during treatment, but the underlying mechanisms remain complex and are not yet fully understood. Research studies have shown that most anticancer drugs are effective against the mass of tumor cells rather than Cancer Stem Cells (CSCs). They have the ability to become dormant upon exposure to anticancer drugs and acquire drug resistance over a period of time after silently modifying themselves to resist the drug environment. The maintenance of the stem cell phenotype is mostly driven by stem cell transcription factors (e.g., KLFs, Myc, ZEB2, RUNX1, TWIST1, OCT4, NANOG, FOXO3, SOX2, E2Fs, etc.), which are mediated by epigenetic changes in tumor cells at many levels. These recognized factors/pathways have been demonstrated to be promising therapeutic targets. Therefore, knowledge of epigenetics in regulating cancer stem cells via stem cell transcription factors may be a promising solution for the reversal of therapy-resistant BC. In this review, we emphasize stem cell transcription factors, cancer stem cells, and epigenetic regulation as critical drivers of drug resistance in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bb0cff2c052008da468be3376723f26d8ad1c7" target='_blank'>
              Interplay between Epigenetic and Transcription Factors Mediating Drug Resistance via Stem Cells in Breast Cancer
              </a>
            </td>
          <td>
            Komal Barhela, A. Chaudhary, Richa Mishra, Nupur Nupur, Sri Krishna Jayadev Magani, H. Rudayni, Sanjay Kumar, Binayak Kumar
          </td>
          <td>2025-12-09</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="One of the most important changes in the transformation of normal cells into tumor cells is metabolism. In order to satisfy the more active proliferation, migration and metastasis of cancer cells, abnormal changes occur in various pathways and molecules involved in metabolism, which eventually lead to metabolic reprogramming of tumor cells. This process involves the uptake of nutrients and changes in major metabolic forms. As an important part of post-transcriptional epigenetics, RNA methylation modifications can regulate RNA processing and metabolism, while dynamically and reversibly influencing the expression of specific molecules, thereby ultimately affecting diverse biological processes and cellular phenotypes. In this review, various types of RNA methylation modifications involved in cancer are summarized. Subsequently, we systematically elucidate the mechanism of RNA modification for metabolic reprogramming in cancer, including glucose, lipid, amino acid and mitochondrial metabolism. Most importantly, we discuss in depth the clinical significance of RNA modification in metabolic targeted therapy and immunotherapy from mechanism to therapeutic application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e3fd9d374b6d8826edc1b140acc672b5b30712" target='_blank'>
              RNA Methylation in Cancer Metabolism: from Mechanisms to Therapeutic Opportunities
              </a>
            </td>
          <td>
            Zeyu Wu, Yuncan Xing, S. Mei, T. Xiao, Fangze Wei, Qian Liu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified over 60 autosomal risk loci associated with clear cell renal cell carcinoma (ccRCC), yet the functional mechanisms underlying these associations remain largely unclear. To establish connections between risk variants and their target genes, we applied the activity-by-contact (ABC) model, which integrates epigenomic data and Micro-C interactions, complemented with renal-specific quantitative trait loci, to predict enhancer-gene relationships. Our analyses implicate variation in hypoxia sensing, cell cycle regulation, and telomerase maintenance pathways as central mediators of ccRCC risk. These findings provide new insights into the molecular basis of ccRCC susceptibility and highlight potential therapeutic avenues for prevention and treatment. Integration of genome-wide association data with epigenomic and chromatin interaction data identifies target genes mediating clear cell renal cell carcinoma risk, implicating disrupted hypoxia signalling, cell cycle, and telomerase regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26c65e87cfb98c7e3f6c3d6a8a7f0754996d3cf" target='_blank'>
              Deciphering genetic susceptibility to clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Maria Mandelia, Philip J. Law, Charlie Mills, M. Went, J. Vijayakrishnan, R. Houlston
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c863981147c795d5bd3bdf0f0373efc730fa086e" target='_blank'>
              Genetic identification of the RAS proteostatic machinery and its failure to regulate oncogenic variants
              </a>
            </td>
          <td>
            J. Bigenzahn, F. Kartnig, Manuela Vollert, V. Sedlyarov, Giulio Superti-Furga
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Homologous recombination (HR) and non-homologous end joining (NHEJ) are two main repair pathways of DNA double-strand breaks (DSBs). The regulation of repair pathway choice is crucial for maintaining genomic stability and preventing carcinogenesis. Consequently, there is increasing interest in elucidating the molecular mechanisms that govern DSB repair pathway selection. Here, we identify chromosome 8 open reading frame 33 (C8orf33) as a novel regulator of DSB repair choice. We show that C8orf33 is a nuclear protein localized predominantly to the nucleolus and is recruited to DSB sites within both nuclear and nucleolar regions. We demonstrate that C8orf33 promotes the recruitment of 53BP1, thus channeling DSB repair toward NHEJ. Consequently, C8orf33 inhibits DNA end resection and counteracts the recruitment of HR factors, BRCA1 and RAD51, to DSB sites. Mechanistically, chromatin profiling analysis reveals that C8orf33 antagonizes the chromatin association of KAT8 acetyltransferase at DSB sites leading to reduced histone 4 lysine 16 acetylation (H4K16ac) levels. Accordingly, the loss of C8orf33 enhances KAT8 chromatin binding that increases H4K16ac levels. This promotes the recruitment of HR factors while suppressing the accumulation of NHEJ factors at DSB sites, thereby favoring HR over NHEJ. Additionally, we demonstrate that the elevated HR activity in C8orf33-deficient cells causes genomic instability, as evidenced by accelerated loss of ribosomal DNA repeats and increases cell death. Collectively, our findings establish C8orf33 as a critical regulator of DSB repair pathway choice, safeguarding genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac1d5cfef9dd20c22a9cd88671f578821b21baa" target='_blank'>
              C8orf33 dictates DNA double-strand break repair choice by modulating KAT8-mediated H4K16 acetylation
              </a>
            </td>
          <td>
            Laila A. Bishara, Enas R. Abu-Zhayia, Marian Nicola, N. Ayoub
          </td>
          <td>2025-11-17</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The adaptive response of cancer cells to hypoxia, a key microenvironmental factor in solid tumors, is orchestrated by Hypoxia-inducible factor 1 (HIF-1). Recent evidence indicate that oxygen tension in tumor is dynamic, with hypoxia being frequently unstable, or cycling. Cycling hypoxia is associated with specific phenotypic outcomes for the cancers. Transcriptomic analysis shows that for most genes, expression changes in cycling hypoxia lie expectedly in between the change caused by stable hypoxia, suggesting multi-cycle averaging of dosage in the oxygen tension, and likely HIF-1 induced transcription. However, a small subset of genes show an oscillation/cycling hypoxia specific response, suggesting that the transcriptional machinery of these genes may interpret cycling HIF-1 activity differently from stably high HIF-1 activity. Here, we model a gene regulatory circuit, the incoherent feed-forward loops (IFFLs) to illustrate that there are parameter regimes in such genetic circuits where oscillatory specific transcription is plausible. In these IFFL models, HIF-1 regulates gene transcription of a target gene directly, as well indirectly via another transcription factor with an opposite effect on gene transcription. This IFFL circuit is able to generate gene expression of certain target genes that is more extreme than either normoxia or stable hypoxia, and this nonlinear IFFL behavior can result from either the dynamic nature or even the intermediate, time averaged hypoxic signal Supplementary Information 1 (Steady state analysis of IFFL circuits). This gene circuit also allows us to search for plausible signaling intermediaries involved in the IFFL mediated cycling hypoxic response. Finally, we present experimental evidence suggesting that HIF-1 can form IFFLs with two key transcription factors p53, and Notch1, resulting in cycling hypoxia-specific gene expression linked to breast cancer progression and poor prognosis. Our work aims to draw attention to genetic circuits as plausible mechanisms where temporal fluctuations in the tumor microenvironment may directly inform downstream transcription. These ideas could identify hitherto unknown HIF-1 driven mechanism of cancer progression contributing to emergent tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6a647a093466b30d57061112c2acfa8e72ce4e" target='_blank'>
              Hif-1 responsive IFFLs to explain specific transcriptional responses to cycling hypoxia in cancers
              </a>
            </td>
          <td>
            Xihua Qiu, Yamin Liu, Paola Vera-Licona, E. Agmon, Kshitiz, Yasir Suhail
          </td>
          <td>2025-11-24</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma (NB) is a childhood cancer that arises when certain nerve-related cells fail to mature properly and become malignant. Our study focuses on a gene called INSM1, which is normally active during early cell development but is abnormally elevated in NB tumors. We aimed to understand how INSM1 contributes to cancer’s aggressiveness and poor differentiation. We discovered that INSM1 keeps tumor cells in an immature, stem-like state and is closely linked to a chemical pathway called the methionine cycle, which affects gene regulation through epigenetic changes. Disrupting this cycle alters INSM1 activity and may help push cancer cells toward normal development. We also found that retinoic acid, a compound known to promote cell differentiation, reduces INSM1 and another cancer-driving gene, N-Myc. These findings suggest that targeting INSM1 and its metabolic regulation could be a promising strategy to treat NB by encouraging tumor cells to mature and lose their malignant properties. This research provides new insights into the biology of NB and may lead to more effective therapies for children affected by this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb4304aef088d414eab3a9ced76da697e7e9f2" target='_blank'>
              Regulation of INSM1 Gene Expression and Neuroendocrine Differentiation in High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Chiachen Chen, Siyuan Cheng, Xiuping Yu, Yisheng Lee, Michael S. Lan
          </td>
          <td>2025-12-22</td>
          <td>Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer treatment is a core challenge in clinical oncology. Despite the continuous advancement of chemotherapy, targeted therapy, and immunotherapy, the emergence of tumor drug resistance remains a key factor leading to treatment failure and worsening patient prognosis. Clinical data indicate that approximately 90% of treatment failures in patients with advanced cancer are directly related to drug resistance. Therefore, systematically analyzing the mechanisms of drug resistance and developing targeted strategies to address them are crucial for improving the efficacy of cancer treatment. This article systematically expounds on the main mechanisms of tumor drug resistance from the perspectives of tumor cell biological characteristics, tumor microenvironment regulation, and epigenetic modifications. These include reduced intracellular drug accumulation, mutations or aberrant expression of drug targets, inactivation of apoptotic pathways, cancer stem cell (CSC)-mediated resistance, and the regulatory role of immunosuppressive cells and cytokines in the tumor microenvironment. Furthermore, based on clinical practice, this article summarizes currently used strategies, such as drug development targeting resistance-associated molecules, chemotherapy regimen optimization and combination therapy, synergistic application of immunotherapy and conventional therapies, personalized treatment plan development, and the clinical translation of resistance monitoring technologies. Research has shown that the development of tumor drug resistance is the result of multiple factors and pathways, and a single strategy is unlikely to effectively reverse drug resistance. However, multi-target, multi-modal combination therapy based on resistance mechanisms, combined with dynamic resistance monitoring technology, can significantly improve treatment outcomes for patients with drug-resistant tumors. This article aims to provide a theoretical basis for clinicians to formulate tumor treatment plans and to guide future research related to drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f859514bae2da8d810219f47530fe8e0b51bb170" target='_blank'>
              A Review of the Mechanisms of Tumor Drug Resistance and Clinical Strategies
              </a>
            </td>
          <td>
            Wenqing Tie
          </td>
          <td>2025-11-21</td>
          <td>International Journal of Public Health and Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The clonal evolution of breast cancer involves a complex dialogue between tumor cells and their environment. In this process, epigenetic mechanisms play a crucial role in regulating the cellular transcriptome without altering the underlying DNA sequence. Here, we provide an updated summary of three main epigenetic mechanisms: histone modifications, long non-coding RNAs (lncRNAs), and higher-order chromatin structures. Post-translational modifications of DNA or histones influence gene expression by altering chromatin accessibility and/or recruiting regulatory protein complexes. This process is dynamically regulated by enzymes that add or remove these marks, as well as by reader proteins that recognize them. Dysregulated expression or malfunction of these regulators creates an aberrant epigenetic landscape and gene expression profile, contributing to breast cancer initiation, metastasis, and drug resistance. Notably, the donor molecules for chromatin modifications are largely derived from intermediate metabolites shaped by environmental cues, highlighting the intricate crosstalk between epigenetic regulation and both cellular and systemic metabolic states. DNA and histone modifications are further interrelated with lncRNAs and higher-order chromatin architectures, which have been actively investigated in breast carcinogenesis. We also briefly introduce the role of epigenetics in other chromatin-associated events such as replication initiation. Aberrant replication initiation can drive gene duplication and genomic alterations resembling those observed in clinical breast cancer, endowing tumor cells with growth advantages and therapeutic resistance. Finally, we summarize emerging therapeutic strategies that target epigenetic vulnerabilities in breast cancer and discuss their current limitations and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502ad7aff3505dc7bf1c94f78c3fe440208426d" target='_blank'>
              Advances in the epigenetic regulation of breast cancer
              </a>
            </td>
          <td>
            Xiao Liu, Xin Wu, Jianan Ding, Xiaoying Qiao, Jing Liang
          </td>
          <td>2025-11-21</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6b2575bc4527f97594e91475e68fff42631efd" target='_blank'>
              On the role of dcm and vsr in Escherichia coli adaptive mutation
              </a>
            </td>
          <td>
            R. Minda, Jyoti Ramchandani, G. Bindal, D. Rath, Prashant Kodgire, R. D. Makde, Swapan Bhattacharjee
          </td>
          <td>2025-12-12</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            , Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Bladder cancer is a highly heterogeneous malignant tumor of the urinary system with high recurrence rates, posing significant challenges in its diagnosis and treatment. Advances in multi-omics research have elucidated the molecular mechanisms underlying the pathogenesis and progression of bladder cancer, including driver gene mutations (e.g., FGFR3, TP53/RB1), dysregulation of signaling pathways (such as PI3K/AKT/mTOR and RAS-MAPK), epigenetic alterations, non-coding RNA networks, tumor microenvironment remodeling, and metabolic reprogramming. This review systematically summarizes recent progress in translational research bridging molecular mechanisms to breakthroughs in precision therapy, covering the clinical applications and challenges of FGFR inhibitors, immune checkpoint inhibitors, antibody-drug conjugates, and gene therapies. Translational efforts are increasingly relying on molecular subtyping to develop subtype-specific treatment strategies. Although significant advances have been made in precision therapy for bladder cancer, critical research gaps remain, including tumor heterogeneity, therapy resistance, and insufficient validation of biomarkers. Future research directions emphasize the potential of liquid biopsy for non-invasive diagnosis and dynamic monitoring, rational combination therapies, multi-omics data integration, and artificial intelligence in advancing personalized treatment, providing a systematic and forward-looking perspective on precision medicine in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14a6737ae3e58aea00cf12ac1c7b98be1125652" target='_blank'>
              Molecular mechanisms and translational advances in bladder cancer: from driver genes to precision therapy
              </a>
            </td>
          <td>
            Jinjie Xiang, Yuhui Luo, Baiyu Zhang, Kunbin Ke, Hao Li
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Resistance to genotoxic therapies remains a major contributor to tumor recurrence and treatment failure, yet the mechanisms by which cancer cells escape these therapies through DNA damage response (DDR) activation are not fully understood. Here, we identify a DDR regulatory pathway in which glycogen synthase kinase 3 β (GSK3B), a multifunctional serine/threonine kinase, governs DNA double-strand break (DSB) repair pathway choice by phosphorylating 53BP1 at threonine 334 (T334) — a site distinct from canonical ATM targets. This phosphorylation event disrupts 53BP1’s interaction with nonhomologous end joining (NHEJ) effectors PTIP and RIF1, promoting their dissociation from DSBs and inhibiting 53BP1-driven NHEJ. Simultaneously, T334 phosphorylation facilitates the recruitment of CtIP and RPA32 for DNA end resection and promotes homologous recombination (HR) by enabling BRCA1 and RAD51 loading. Notably, the phospho-deficient T334A mutant of 53BP1, unlike 53BP1 loss, accumulates aberrantly at DSBs along with PTIP/RIF1, impairs end resection, and suppresses HR activity. Importantly, both genetic and pharmacologic disruption of the GSK3B–53BP1 axis sensitizes tumors to PARP inhibitors (PARPi) independently of BRCA1 status. Together, these findings reveal a GSK3B-dependent mechanism that regulates DSB repair pathway choice and provide a rationale for targeting this axis to enhance PARPi efficacy in solid tumors regardless of BRCA1 status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7947f267430312493c565ecaa4ae8dfb1f98a221" target='_blank'>
              GSK3B directs DNA repair choice and determines tumor response to PARP1 inhibition independent of BRCA1
              </a>
            </td>
          <td>
            H. Allam, Scarlett Acklin-Wehnert, R. Sadhukhan, M. Patra, F. Xia
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00ed13bb7acb39c5e98a4fa14592f332d5a91492" target='_blank'>
              A narrative review of therapy-induced senescence in cancer: mechanisms, immune interplay, and therapeutic opportunities
              </a>
            </td>
          <td>
            Henry Sutanto, Alfan Ahkami, D. Fetarayani, Pradana Zaky Romadhon
          </td>
          <td>2025-12-01</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Epigenetics encompasses heritable but reversible modifications of gene expression that occur without changes in the DNA sequence and involve mechanisms such as DNA and RNA methylation and histone modifications. These mechanisms modulate chromatin architecture, genome stability, and cellular responses to environmental stressors, and their dysregulation contributes to oncogenesis and cancer progression. In parallel, radiotherapy remains a cornerstone of cancer treatment; furthermore, ionizing radiation induces epigenetic modifications alongside direct DNA double-strand breaks and oxidative damage. Radiation-induced epigenetic changes, including global or locus-specific DNA methylation shifts (e.g., genes promoter CpG islets), histone acetylation and methylation imbalances, are increasingly recognized as key contributors to molecular radioresistance. These adaptive responses may enhance tumor cell survival, affect therapeutic efficacy, and promote metastasis. Understanding the interplay between radiation exposure and epigenetic remodeling opens new perspectives for precision oncology and diagnostics. Epigenetic biomarkers hold potential for predicting treatment response and prognosis, while epigenetic modifiers may sensitize tumors to radiation. This review summarizes current evidence on radiation-induced epigenetic mechanisms and evaluates their diagnostic, prognostic, and therapeutic implications in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18126342fcb0010fdf4cce9fa40fdeeb69a2ac9b" target='_blank'>
              A New Look at the Role of Radiation-Related Epigenetic Mechanisms in Diagnosis and Anticancer Therapies
              </a>
            </td>
          <td>
            A. Olichwier, Magdalena Bruzgo-Grzybko, Izabela Suwda Kalita, Natalia Bielicka, Ewa Chabielska, A. Gromotowicz-Poplawska
          </td>
          <td>2025-11-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, Cheng-Che Lee, George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, S. Yi, Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly specialized intratumoral compartment responsible for tumor initiation, metastatic dissemination, therapeutic resistance, and disease recurrence. A central conceptual challenge in CSC biology is their capacity to oscillate between a quiescent G0 state and a proliferative, stem-like phenotype, reflecting a high degree of phenotypic plasticity. Although dysregulation of the G1/S checkpoint is a hallmark of malignant transformation, its mechanistic contribution to CSC identity and plastic behavior remains poorly defined.This review outlines a conceptual model that integrates aberrant G1/S control with CSC state transitions. We propose that defective checkpoint regulation accelerates CSC proliferation, leading to the progressive intracellular accumulation of Cyclin D, which in turn drives a self-reinforcing, rapid G1 progression through phosphorylation-dependent pathways that operate independently of the slower, transcription-driven Cyclin D-Rb-E2F regulatory axis. With continued cycling, depletion of key E2F-regulated DNA replication factors ensues, eventually forcing CSCs into a quiescent, biosynthetic restoration phase. During this interval, essential genomic replication and cell cycle machinery are replenished until microenvironmental or intracellular cues trigger reentry into the proliferative cycle, giving rise to another burst of accelerated division.Through these cyclical perturbations in the Cyclin D/E2F balance, CSCs undergo temporally governed shifts between quiescent and proliferative states, thereby sustaining plasticity, intratumoral heterogeneity, and treatment-resistant phenotypes. This model also identifies potential therapeutic strategies, such as leveraging stimuli-responsive delivery systems that exploit cyclic CSC vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53543eb088f0d567f8bdf7288c39cb199fa8756" target='_blank'>
              Dysregulation of the Cyclin D/E2F activity as a core mechanism driving cancer stem cell plasticity and cell cycle dynamics.
              </a>
            </td>
          <td>
            Yasin Ahmadi, Trefa Mohammed, Sakhavat Abolhasani, Shwan Abdullah Hamad, Tahran Faiq
          </td>
          <td>2025-12-14</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard J Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, Ana P. Gomes, Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ef5b0807e681dc02a817ada526d9c5fc0ddd59" target='_blank'>
              RegEvol: detection of directional selection in regulatory sequences through phenotypic predictions and phenotype-to-fitness functions
              </a>
            </td>
          <td>
            Alexandre Laverré, T. Latrille, Marc Robinson-Rechavi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3de429a7817141d17bab7346b3911bc733e9f80d" target='_blank'>
              Integrated methylome analysis identifies an epigenetically silenced tumor suppressor RASA4 in small cell lung cancer.
              </a>
            </td>
          <td>
            Meng Fu, Qizhi Zhu, Jian Qi, Xiaopeng Zhang, Zhipeng Wang, Jialiang Wang, Xiang Huang, Zongtao Hu, Wei Wang, Bo Hong, Hongzhi Wang
          </td>
          <td>2025-12-31</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Targeted in vivo hypermutation drives rapid sequence diversification within specific DNA regions of living cells, enabling continuous evolution of biomolecules for enhanced functionality. In living organisms, this is prominently illustrated by the preferential somatic hypermutation of complementarity-determining regions during antibody affinity maturation. This particular targeting strategy employs focused mutagenesis in predefined, functionally important intragenic regions and should ideally allow for target range adjustment to support broader applications. However, molecular tools incorporating these features remain limited. Here, we present RESPECTevo, a system for focused hypermutation of endogenous and heterologous regions up to ~200 bp with predefined boundaries. By leveraging specific targeting capability, spacer length flexibility, and mismatch tolerance of the Escherichia coli Cascade complex, RESPECTevo enables highly specific targeted hypermutation, target range modulation, and continuous hypermutation across dual regions. Collectively, RESPECTevo serves as an efficient platform for hypermutating user-defined target regions in living cells, facilitating continuous protein evolution for biotechnological applications. Antibody somatic hypermutation is a remarkable example of targeted hypermutation in living organisms, yet synthetic tools replicating its precision remain limited. Here, authors introduce a new method that installs mutations within a defined, tunable window of up to ~200 bp in target DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e935feb30964de2ec6e75f007501e092c4470483" target='_blank'>
              Continuous targeted hypermutation with a tunable mutation window
              </a>
            </td>
          <td>
            Chanwoo Lee, Seokhee Kim
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer therapy resistance emerges from highly integrated molecular systems that enable tumor cells to evade cell death and survive cytotoxic therapeutic stress. High Mobility Group Box 1 (HMGB1) is increasingly gaining recognition as a central coordinator of these resistance programs. This review delineates how HMGB1 functions as a molecular switch that dynamically redistributes between cellular compartments in response to stress, with each localization enabling a distinct layer of resistance. In the nucleus, HMGB1 enhances chromatin accessibility and facilitates the recruitment of DNA repair machinery, strengthening resistance to radio- and chemotherapeutic damage. Cytosolic HMGB1 drives pro-survival autophagy, maintains redox stability, and modulates multiple regulated cell death pathways, including apoptosis, ferroptosis, and necroptosis, thereby predominantly shifting cell-fate decisions toward survival under therapeutic pressure. Once released into the extracellular space, HMGB1 acts as a damage-associated molecular pattern (DAMP) that activates key pro-survival and inflammatory signaling pathways, establishing microenvironmental circuits that reinforce malignant progression and therapy escape. HMGB1 further intensifies resistance through upregulation of multidrug resistance transporters, amplifying drug efflux. Together, these compartmentalized functions position HMGB1 as a central node in the networks of cancer therapy resistance. Emerging HMGB1-targeted agents, ranging from peptides and small molecules to receptor antagonists and nanoformulations, show promise in reversing resistance, but clinical translation will require precise, context- and redox-informed HMGB1 targeting to overcome multifactorial resistance program in refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9023ae13f0952b85c73d6ba319988bfa2cf22" target='_blank'>
              HMGB1: A Central Node in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Bashar A. Alhasan, Boris A. Margulis, Irina V. Guzhova
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ca5c42693a477fde2aa581c6c760a192f8faa2" target='_blank'>
              The cumulative impact of passenger mutations on cancer development
              </a>
            </td>
          <td>
            Akshatha Nayak, Candace S. Y. Chan, Ioannis Mouratidis, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The transcription factor GATA-binding protein 3 (GATA-3) regulates oncogenic transcriptional programs across diverse T-cell lymphomas, including subsets of both peripheral and primary cutaneous T-cell lymphomas. These GATA-3 dependent transcriptional programs, in collaboration with the genetic landscape, promote cell growth and survival, and confer resistance to conventional chemotherapeutic agents. We observed that transcriptional cyclin dependent kinase 9 (CDK9) activation regulates diverse oncogenic transcriptional programs in these aggressive T-cell lymphomas and is thus a novel therapeutic vulnerability. Using complementary and orthogonal approaches, we identified multiple independent mechanisms by which CDK9 promotes T-cell lymphomagenesis, including a mechanism by which GATA-3 promotes CDK9 activation at GATA-3 dependent loci. We also identify novel mechanisms by which GATA-3 and CDK9 regulate rRNA transcription and processing, respectively, collaboratively promoting ribosome biogenesis. Therefore, CDK9 is a therapeutic vulnerability across genetically and transcriptionally diverse T-cell lymphomas, including those for which GATA-3 is oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce1855a49cf59262cae5bab6172d5360a704a63" target='_blank'>
              CDK9 is a dependency in GATA-3 driven and MCL-1 independent T-cell Lymphomas
              </a>
            </td>
          <td>
            Chenguang Wang, Suhaib Abdelrahman, Xiangrong Geng, Alyssa Burgess, Ying S Hu, Mohd Ahmar Rauf, N. Kady, Yao Fu, Tara A Reilly, Ira P Maine, Phillip Boonstra, Kirill Sabitov, Carlos A. Murga-Zamalloa, Ryan A Wilcox
          </td>
          <td>2025-11-27</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Summary SISTEM is a software package and mathematical framework for simulating tumor evolution and cell migrations at single-cell resolution. Unlike existing frameworks which simulate cancer cell populations under the neutral coalescent or using simple birth–death models, SISTEM simulates tumor populations under somatic clonal selection using an agent-based framework. SISTEM can generate mutation profiles, read counts, and DNA sequencing reads along with ground truth cell lineages and migration graphs under a number of easily customizable mutation and selection models. For improved realism, SISTEM allows for cell fitness to be driven by genomic events of various scales including single nucleotide variants, segmental gains and losses, whole-chromosomal and chromosome-arm aberrations, and whole-genome duplications. SISTEM also includes numerous migration models to simulate metastatic cancers, facilitating the exploration and evaluation of diverse migration patterns. Availability and implementation SISTEM is written in Python and is freely available open-source under GNU GPLv3 from: https://github.com/samsonweiner/sistem">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6ea45c6bd4d361ab9e67581ccc6b02aecffc37" target='_blank'>
              SISTEM: simulation of tumor evolution, metastasis, and DNA-seq data under genotype-driven selection
              </a>
            </td>
          <td>
            Samson Weiner, M. Bansal
          </td>
          <td>2025-11-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background: Among primary malignant brain tumors in adults, glioblastoma is the most common and the most aggressive, characterized by profound metabolic reprogramming. This metabolic shift is essential for sustaining relentless proliferation and adapting to the challenging tumor microenvironment (TME). Central to this adaptation in glioma is the Warburg effect, which leads to excessive lactate production and accumulation, even in the presence of oxygen. This metabolic divergence significantly impacts the tumor immune microenvironment, promoting the recruitment of immunosuppressive cells and weakening the anti-tumor immune response. Core Content: This review provides a comprehensive analysis of the multifaceted roles of lactate in IDHwt glioma pathogenesis. It explores how lactate serves as a critical nexus connecting aberrant metabolism, epigenetic reprogramming (notably via histone lactylation), and immune evasion. The review delves into the molecular mechanisms by which lactate, particularly through the post-translational modification known as lactylation, directly modulates the epigenetic landscape to promote oncogene expression. Furthermore, it examines lactate’s role in acidifying the TME, promoting the immunosuppressive M2 polarization of glioma-associated macrophages (GAMs), and inhibiting the cytotoxic activity of T lymphocytes. Conclusions: This “lactate-centric” framework provides a unifying model that links metabolic dysregulation directly to malignant progression and therapeutic resistance (e.g., to TMZ). By elucidating this metabolic–epigenetic–immune axis, the review highlights a critical dependency that fuels glioma aggression. Finally, it discusses emerging therapeutic strategies aimed at targeting lactate production (LDHAi), transport (MCTi), and downstream epigenetic signaling (HDACi/p300i), offering novel avenues for integrative immunometabolic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b50b3cdb0eb39f78be330d84ff6e266ce7350786" target='_blank'>
              Lactate-Mediated Epigenetic and Immunometabolic Reprogramming in Glioma: An Emerging Axis Linking Metabolism to Tumor Progression
              </a>
            </td>
          <td>
            Xinyi Xie, Wenjing Zhou, Yin Ku, Shasha Li, Yunhao Yang, Xiaohu Hao, Yaohui Chen
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pediatric brain tumors (PBTs) are the leading cause of cancer-related mortality in children. Despite advances in next-generation sequencing (NGS) deepening our understanding of the molecular features of PBT, the lack of preclinical models that capture their complexity and diversity remains a significant barrier to developing less toxic and more effective treatments. We have established 20 ex-vivo patient-derived tumoroid (PDTs) cultures from fresh surgical material of a wide range of PBTs, including low- and high-grade gliomas, medulloblastomas, and even rarer tumor entities such as a CNS tumor with BCOR alteration. Immunofluorescence, NGS analysis, and DNA methylation profiling revealed that the PDTs faithfully mirrored the cellular nature, the genetic and epigenetic landscape of their matched primary tumors. Our study shows the feasibility of generating PDTs, even from rarer entities, that recapitulate the genetic and epigenetic features of primary tumors, highlighting their potential as models for tumor biology studies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ce2dc294b160225733b6cd33eb603575d30e64" target='_blank'>
              Patient-derived tumoroids recapitulate the morphologic and molecular features of pediatric brain tumors
              </a>
            </td>
          <td>
            Bárbara Soares-Ferreira, Joana Peixoto, Anabela Ferro, Beatriz Esteves, Jorge Pinheiro, Roberto Silva, Josué Pereira, Joana Oliveira, Arnaud Da Cruz Paula, Yingjie Zhu, Maria Teresa Azevedo, Luísa Fonseca, Lara Coutinho, André Filipe Maia, Paula Soares, S. Nunes, Ana Paula Fernandes, MJ Gil Da Costa, Jorge Lima
          </td>
          <td>2025-11-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous malignancy encompassing distinct molecular subtypes such as Luminal A & B, HER2 enriched and triple negative breast cancer, each characterized by unique genomic, transcriptomic and proteomic landscapes that dictate differential prognosis and therapeutic response. This review delineates the complex signalling pathways underlying breast cancer including estrogen receptor, progesterone receptor and HER2 pathways and their downstream effectors such as PI3K/AKT/mTOR, RAS/MAPK, Notch, Wnt/β-catenin and JAK/STAT cascades. The interconnectivity of these pathways not only governs tumor cell proliferation, survival and metastasis but also contributes to intrinsic and acquired resistance mechanisms against standard-of-care therapies including endocrine agents, HER2-targeted antibodies and chemotherapy. Emerging therapeutic strategies such as next-generation selective estrogen receptor degraders, proteolysis-targeting chimeras, antibody-drug conjugates, immune checkpoint inhibitors and pathway-specific small-molecule inhibitors highlights their mechanistic rationale, clinical evidence and biomarker-guided applicability. Resistance mechanisms involving ESR1 mutations, PTEN loss, p95HER2 expression, BRCA reversions and tumor microenvironment-mediated immunosuppression contributes to therapy resistance. The review explores molecular subtypes, oncogenic signalling pathways, treatment approaches, resistance mechanisms including acquired resistance & crosstalk, metabolic reprogramming, epigenetic modifications, immune modulations and tumor microenvironment, spatial transcriptomics insights into immune cell subsets & treatment response biomarkers and the role of artificial intelligence in breast cancer treatment. The integration of artificial intelligence-based predictive modelling and rational drug combinations offers a promising paradigm for overcoming resistance and tailoring precision oncology approaches. By comprehensively mapping these oncogenic factors and therapeutic intersections, this review provides a strategic framework for optimizing breast cancer management in the era of molecular medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3630728d4d7c062ce8b3bea5630e479dd9f7514d" target='_blank'>
              Decoding breast cancer: insights into molecular pathways & therapeutic approaches
              </a>
            </td>
          <td>
            Tanjima Tarique Laskar, Hasina Mehnaj Laskar, Jiaul Ahmed Mazumder, Rupajit Bhattacharjee, Md Ikbal Husain, Bandana Das, Panchami Das, Paromita Dutta Choudhury, Monica Arora, Sudarshana Borah, D. Chakraborty, Prithviraj Chakraborty, Amlan Das
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Resistance to single-agent therapy in acute myeloid leukemia (AML) often arises from compensatory epigenetic circuits. We present a causal inference framework that treats therapy as an intervention (do-operator) and defines resistance as the deviation of dual perturbations from the additive expectation of monotherapies. By modeling the causal structure of the leukemia stem-cell epigenetic landscape that maintains identity and fitness, the framework shows that synergistic and antagonistic interactions can be inferred directly from single perturbations, suggesting interaction inference may be simpler than previously assumed. We applied this model to Perturb-seq of 16 epigenetic regulators in KMT2A-rearranged AML (>31,000 single-cell transcriptomes). From single-knockout profiles, the framework correctly predicted synergistic pairs (Menin+KAT6A, Menin+DOT1L) that enhanced differentiation and induced cell death, as well as an antagonistic pair (DOT1L + PCGF1) that conferred resistance. Predictions were validated by pharmacologic inhibition and bulk RNA-seq, confirming the model’s predictive accuracy. This work presents a mechanism-guided approach for rational prioritization of epigenetic drug combinations in AML and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057014eca0b4705ea6ee698244a083b962878940" target='_blank'>
              Causal interaction inference of compensatory structures from single-cell perturb-seq in AML
              </a>
            </td>
          <td>
            Changde Cheng, Sajesan Aryal, Brittany Curtiss, Xinyue Zhou, Rui Lu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Malignant tumors developing in the conditions of chronic hypoxia, unlike normal tissues, quickly acquire resistance to decreased oxygen level. Hypoxia-resistant cells in malignant disease obtain various features promoting their survival: they alter metabolism to aerobic glycolysis, activate angiogenesis-stimulating programs, and rearrange signaling cascades to adapt to hypoxia. At the same time, tumor cells become resistant to chemotherapy and radiation therapy and effectively colonize metastatic niches. 
This review analyzes signaling pathways which activate in the conditions of chronic hypoxia and underlie tumor adaptation to insufficient oxygen. Apart from the main pathway associated with activation of hypoxia-inducible factor 1α (HIF-1α), such cascades as STAT3, Snail cascade associated with epithelial-mesenchymal transition, NRF2 factor responsible for adaptation to reactive oxygen species, and stemness factors OCT4, SOX2 and NANOG are also considered. Effective killing of tumor cells requires simultaneous inhibition of several parts of signaling pathways because disruption of just one will lead to switching of signaling proteins to bypass the blocked one. The article lists various inhibitors of signaling pathways patricipating in tumor cell adaptation to chronic hypoxia and describes their mechanisms of action and targets. Development of successful strategies for treatment of hypoxia-resistant tumors requires identification of an effective combination of the above-mentioned and other inhibitors of the key signaling cascades participating in adaptation to hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21710e96836bbca03c72f6e2b505d714f5b359ca" target='_blank'>
              Rearrangement of signaling pathways and adaptation of tumor cells to hypoxia
              </a>
            </td>
          <td>
            O. E. Andreeva, E. I. Mikhaevich, M. A. Krasil’nikov
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gastrointestinal (GI) cancers pose a significant global health burden, driven by complex molecular alterations and microenvironmental interactions. Advances in molecular pathogenesis have elucidated recurrent driver gene mutations, such as KRAS, TP53, and APC, alongside dysregulated signaling pathways including Wnt, RAS-MAPK, and PI3K-AKT, which collectively underpin tumor initiation and progression. Complementing genetic changes, epigenetic alterations—such as DNA hypermethylation, histone modifications, and regulatory non-coding RNAs—further contribute to malignant evolution by reshaping chromatin architecture and gene expression. These mechanisms not only promote uncontrolled proliferation but also reinforce therapeutic resistance by dynamically modifying the tumor microenvironment (TME). Molecular subtyping efforts, including The Cancer Genome Atlas (TCGA) classification for gastric cancer (GC) and the Consensus Molecular Subtypes (CMS) for colorectal cancer (CRC), have delineated disease heterogeneity, revealing distinct pathogenic pathways and enabling refined prognostic stratification. Such insights provide the biological rationale for diagnostic techniques and targeted interventions. For instance, anti-EGFR and anti-VEGF monoclonal antibodies disrupt oncogenic signaling and tumor angiogenesis, respectively, and have demonstrated substantial clinical efficacy in selected patient populations. In parallel, immunotherapy has emerged as a transformative modality in oncology. Immune checkpoint inhibitors targeting PD-1/PD-L1 and CTLA-4 reinvigorate antitumor immunity and have reshaped standard-of-care protocols for several GI malignancies. Beyond conventional immunotherapies, innovative strategies such as CAR-T cell therapy and neoantigen-based vaccines are being actively investigated. These approaches aim to overcome immune evasion mechanisms and enhance tumor-specific targeting, offering promise for patients with resistant or advanced disease. This review comprehensively analyzes the evolving molecular landscape of GI cancers and the corresponding development of targeted and immunotherapeutic agents. It highlights a balanced integration of mechanistic discovery and clinical translation, underscoring their synergistic roles in advancing precision oncology and improving survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5602f60be23d19b8d126a3c7ae3c86407dda38e" target='_blank'>
              Gastrointestinal cancer: molecular pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Yang Jin, Xiaobo He, Yanfeng Wu
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality among women, with hormone receptor-positive (HR + ) disease representing the majority of cases. Despite advances in targeted therapies and antibody-drug conjugates, resistance to treatment remains a significant challenge, necessitating innovative therapeutic strategies. Epigenetic dysregulation, particularly involving histone acetylation, plays a crucial role in tumorigenesis and therapy resistance. Lysine acetyltransferase 6 (KAT6A/B), key regulators of chromatin structure and gene expression, have emerged as promising therapeutic targets in HR+ breast cancer. This review highlights the molecular mechanisms by which KAT6A/B modulate histone acetylation and gene regulation, emphasizing their roles in oncogenic pathways and cellular processes such as transcriptional activation, chromatin remodeling, and regulation of cellular senescence. We discuss preclinical evidence supporting the therapeutic potential of KAT6 inhibition and the challenges faced in epigenetic drug development, including the need for robust biomarker-driven approaches. Finally, we explore the prospects for integrating KAT6 inhibitors with existing therapies to overcome resistance and improve patient outcomes in breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882962d718736529869b6bb59db5c43e6a4e7b49" target='_blank'>
              Exploring KAT6 as a therapeutic target in breast cancer: epigenetic approaches for precision medicine
              </a>
            </td>
          <td>
            Cyril Roussel-Simonin, Aranzazu Fernandez-Martinez, S. Postel-Vinay, F. Mosele, Alessandro Viansone, C. Massard, Adrien Mouren, Julien Vibert, B. Pistilli
          </td>
          <td>2025-12-01</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The abnormal expression of the HOXA genes is intricately associated with the pathogenesis and progression of acute myeloid leukemia (AML), as these genes are crucial in regulating cellular differentiation and proliferation during normal hematopoiesis. In normal hematopoiesis, the expression of HOXA genes is meticulously regulated by the interplay between activation by the KMT2A complex or CDX and repression by the PRC2 complex. AML disrupts this balance, resulting in the persistent activation of HOXA. They perpetuate a leukemic state by directly activating a network of downstream target genes, which facilitate proliferation, hinder differentiation, and inhibit apoptosis. Genetically, this dysregulation can be classified as KMT2A-dependent, as shown in KMT2A rearrangements, NPM1 mutations, NUP98 rearrangements and other type of acute myeloid leukemia. Menin inhibitors may be employed for the dysregulation of HOXA genes reliant on them. Although menin inhibitors have considerable therapeutic efficacy, issues such as drug resistance and particular toxicity still require resolution. Diverse targeted therapeutics may be accessible for the dysregulation of non-KMT2A-dependent HOXA genes, contingent upon the specific genetic alterations. Therefore, a comprehensive understanding of the HOXA signaling network is vital for advancing targeted AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4881e7925db0d903cefbc766f04b1e7269c07b" target='_blank'>
              Dysregulation of HOXA genes in acute myeloid leukemia and targeted therapy
              </a>
            </td>
          <td>
            Wusixian Huang, Fenghong Zhang, Zhiyu Zhang, Qinrong Wang, Suning Chen
          </td>
          <td>2025-12-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer.
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72271f0d34d2f8f24354ade5fddf8b10f1a093fc" target='_blank'>
              Whole-genome landscapes of 1,364 breast cancers.
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, Seokhwi Kim, Jeong Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, P. Paik, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyungjung Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeong Seok Lee, Woo-Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park
          </td>
          <td>2025-12-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human cells possess a complex network of regulatory systems——the DNA damage repair pathway (DDR), to deal with various DNA damage to prevent the inheritance of deleterious mutations. DDR perform its functions via cell cycle checkpoint activation. Mutations in the DDR gene are also the basis for tumor progression, metastasis, and therapeutic effect to treatments that cause DNA damage. Consequently, DDR has received high attention for its role in the therapy against cancer, and the related experiments have been carried out to further explore. Up to now, PARP inhibitors such as Olaparib, Rucaparib, Niraparib, Talazoparib, etc. have been approved for marketing and clinical use, other DDR inhibitors (DDRi) such as ATMi/ATRi/DNA-PKi, Chk1/2i, and WEE1i have partially entered clinical trials. Synthetic lethality has led to the exploration of combination of DDRi with other anti-tumor therapies, in particular, the combination of DDRi with immunotherapies such as chemoradiotherapy and immune checkpoint inhibitors (ICIs) has achieved potential efficacy. This article reviewed the DDR inhibitors that are currently in development or in clinical therapy, analyzed the limitations and future development prospects, and stated the mechanisms and clinical trials in antitumor and immunotherapy, to provide ideas for the subsequent development of DDRi and enhance the efficacy of antitumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9a6e3b57be514835a193a70ee6587199732e497" target='_blank'>
              Research progress of DNA damage repair (DDR) and DDR inhibitors in tumor immunotherapy
              </a>
            </td>
          <td>
            Qi Liu, Chunmei Zhang, Yixuan Gao, Dongmei Feng, Duo Deng, Yun Pan
          </td>
          <td>2025-12-01</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma arises within heterogeneous tumor–immune ecosystems in which impaired antigen visibility—shaped by chromatin programs—limits immune surveillance and blunts responses to immunotherapy. Beyond structural defects in the antigen-processing pathway, Polycomb-mediated repression, DNA hypermethylation, and state-specific enhancer closure converge on the HLA class I/NLRC5/interferon axis to diminish peptide display. These constraints are context dependent, varying across malignant clones, differentiation states, and myeloid and T-cell niches. Traditional bulk assays obscure this complexity; single-cell ATAC-seq, integrated with single-cell and spatial transcriptomics, now resolves promoter–enhancer accessibility at HLA, NLRC5, and antigen-processing genes, distinguishes reversible repression from fixed lesions, and links microenvironmental stress to interferon competence. Translationally, epigenetic reprogramming—targeting Polycomb repressive complex 2 (PRC2), DNA methyltransferases (DNMTs), and complementary regulators (for example, LSD1, BET, CDK4/6, YAP/TEAD)—offers biomarker-guided avenues to restore antigen presentation, provided ecosystem-aware pharmacodynamic readouts track chromatin opening and antigen-presentation recovery across compartments. Despite encouraging preclinical evidence, efficacy will depend on clone selection, scheduling that preserves interferon signaling, and rational combinations with innate agonists and checkpoint blockade. This mini-review synthesizes epigenetic mechanisms of antigen-presentation failure in osteosarcoma and outlines how single-cell chromatin profiling can guide strategies to reinstate tumor antigen visibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51ccd00bee57595f98289280452a3ec7548b15" target='_blank'>
              Epigenetic control of antigen presentation failure in osteosarcoma: from single-cell chromatin maps to therapeutic strategies
              </a>
            </td>
          <td>
            Yan He, Heng Wu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a heterogeneous tumor, ranging from cases with spontaneous regression (MS stage) to high risk (HR) tumors. Resistance to therapy presents a major challenge in the treatment and management of HR-NB, contributing to a poor prognosis. In these patients, the resistance to conventional treatment is also related to non-genetic features, such as the cellular plasticity. Novel studies have demonstrated the presence of two distinct cell phenotypes: adrenergic (ADRN) and mesenchymal (MES), which reflect the heterogeneity of NB. MicroRNAs (miRNAs) are small, endogenous and non-coding RNAs with the ability to regulate gene expression and may have a crucial role in controlling cell plasticity. However, the role of miRNAs in NB plasticity has not been investigated yet. We investigated miRNA signature in NB cells subtypes (MES and ADRN) and in the extracellular vesicles (EVs) released by them, to identify potential MES related biomarkers. Differentially expressed miRNAs were identified by RT-qPCR and subjected to gene ontology, KEGG pathway, and protein-protein interaction network analyses. Candidate miRNAs were validated in plasma-derived EVs from NB patients. We identified miR-199a-3p as strongly upregulated in the MES cells subtype. Moreover, its expression levels were significantly higher in primary cell lines derived from HR patients compared to low-risk (LR) ones. This was confirmed by a bioinformatics analysis in patient tissue obtained from TARGET NB dataset. Protein-protein interaction analysis uncovered a complex network, with FN1, CD44, and YAP1, identified as key genes upregulated by miR-199a-3p, all of which are closely associated with the MES phenotype. Among the miRNAs significantly upregulated in EVs derived from MES cell lines, miR-584a-5p was significantly higher in EVs isolated from plasma of HR and L/Intermediate(I)R patients compared to MS. MiR-584-5p is typically considered a tumor suppressor; to support this role miR-584a-3p resulted significantly upregulated in L/IR tumor tissue, both in the INSS and COG classifications (TARGET NB database). Our findings identified specific miRNAs as MES phenotype related biomarkers. Further studies should investigate the potential impact of miRNAs on plasticity-related pathways in order to open new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b8406e9d606c161b955f6866c60bbec146aba97" target='_blank'>
              Identification of a miRNAs signature as potential biomarker of mesenchymal phenotype in neuroblastoma patients
              </a>
            </td>
          <td>
            Silvia Lampis, Alessandro Paolini, Virginia Di Paolo, A. Galardi, Salvatore Raieli, E. Miele, Lauriane Lemelle, F. Fabozzi, A. Serra, A. Mastronuzzi, M. A. De Ioris, Andrea Masotti, Franco Locatelli, A. Di Giannatale
          </td>
          <td>2025-11-26</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Epigenetic regulation is a fundamental mechanism controlling gene expression and cellular function, primarily mediated through reversible modifications such as DNA methylation, histone acetylation, and chromatin remodeling. Dysregulation of critical epigenetic enzymes, including histone deacetylases (HDACs), DNA methyltransferases (DNMTs), and bromodomain and extraterminal domain (BET) proteins, has been closely associated with tumor initiation, progression, metastasis, immune evasion, and resistance to conventional therapies. Targeting these epigenetic regulators with small-molecule inhibitors or degraders has emerged as a promising therapeutic strategy, capable of reprogramming aberrant transcriptional networks and reshaping the tumor microenvironment. Beyond direct cytotoxic effects, epigenetic drugs have demonstrated the ability to enhance antitumor immunity by restoring antigen presentation, promoting immunogenic cell death, modulating cytokine profiles, and reversing local immune suppression. Recent preclinical and clinical studies have highlighted the potential of combining epigenetic therapies with immune checkpoint inhibitors to achieve synergistic antitumor responses and overcome resistance mechanisms. This review provides a comprehensive summary of the mechanisms of action, pharmacological characteristics, and clinical applications of epigenetic drugs, with a focus on innovative combination strategies and ongoing translational advancements. We also discuss future directions, emphasizing the need to improve drug specificity, minimize off-target effects, integrate personalized immunotherapeutic approaches, and identify predictive biomarkers to optimize patient selection and clinical outcomes. Overall, epigenetic therapy represents a versatile and evolving avenue for precision oncology with broad implications for tumor control and immunomodulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1142f91bb95f6983a1d418601e7e6fb2c4af66" target='_blank'>
              Epigenetic drugs in cancer therapy: mechanisms, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Chen Ma, Junkai Cheng, Jian Gu, Qin Wang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0127956ee657e81733c7622d2b48c0407a2cab1" target='_blank'>
              Cell heterogeneity contributes to the variable response of HIV-1 to latency reversing agents
              </a>
            </td>
          <td>
            Rachel Topno, Hussein Karaki, Flavia Mazzarda, Oriane Pourcelot, Manon Philippe, K. Zibara, Ovidiu Radulescu, Edouard Bertrand
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4010e585ccb0e081982a4a6be2071e2416b2dd17" target='_blank'>
              Inference and validation of a dynamic gene regulatory network influencing cell fate during Duchenne muscular dystrophy initiation
              </a>
            </td>
          <td>
            Elsa G. Guillot, Quentin G Fort, Elise Mozin, Bérangère Robert, Matteo Bouvier, Cynthia Fourgeux, Paul Simon, Sara Munoz, M. Childers, Jérémie Poschmann, O. Adjali, David L. Mack, Arnaud Bonnaffoux, J. Dupont
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad062077323547175d6b721e78e8771a9325ce15" target='_blank'>
              Phenotypic Plasticity and Competition Shape Therapy Sequencing in HER2+/HER2− Breast Cancer: A Mathematical Framework
              </a>
            </td>
          <td>
            Aleksandra Gavrilova, Trachette L. Jackson, Nizhum Rahman
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) imposes a significant global cancer mortality burden, with conventional therapies (surgery, ablation, chemotherapy, radiotherapy) and newer modalities (targeted agents, immune checkpoint inhibitors) limited by therapeutic resistance. Notably, liver cancer stem cells (Liver-CSCs)—defined by their self-renewal and unlimited proliferative capacity—drive tumor initiation, metastasis, heterogeneity, and therapy resistance. This review synthesizes current knowledge on Liver-CSCs, focusing on their distinctive features, supporting microenvironments, signaling pathways, and therapy resistance mechanisms. We also examine novel therapeutic strategies targeting these cells. Clinically, we evaluate recent research, identify knowledge gaps, and suggest potential directions for advancing HCC therapies. Finally, we discuss how these insights may inform development of more effective treatments to improve clinical HCC management. Understanding Liver-CSC biology and treatment resistance mechanisms will enable better-tailored therapies to overcome these challenges and enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be853183c1eda31eb58c99c01ef49e15a7eeaf74" target='_blank'>
              Cancer stem cells in hepatocellular carcinoma: therapy resistance and emerging treatments
              </a>
            </td>
          <td>
            Dipesh Kumar Yadav, Rajesh Kumar Yadav, Alina Singh, Yi Huang, Dandan Bao, Zhangwei Yang, Hanzhang Huang, Yin Jiang, Pengwei Wang, Sisi Lin, Yongfei Hua, Yiren Hu
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="SMARCA4 deficiency plays a critical role in the oncogenesis of various aggressive tumors that exhibit resistance to conventional chemotherapy and radiotherapy, posing significant challenges to clinical management. The pivotal role of SMARCA4 deficiency in tumorigenesis suggests the need for a paradigm shift from traditional tumor origin-based approaches to novel tumor-agnostic strategies focused on molecular alterations associated with SMARCA4 deficiency. This review explores potential targetable molecular changes and emerging therapeutic strategies for SMARCA4-deficient tumors. Molecular alterations related to SMARCA4 deficiency involve impaired genomic stability, defects in DNA mismatch repair, and elevated tumor mutation burdens, all of which suggest potential sensitivity to immune checkpoint inhibitors (ICIs). Recent studies indicate that combining ICIs with chemotherapy or anti-angiogenic agents as first-line treatments may offer clinical benefits for SMARCA4-deficient tumors. Furthermore, SMARCA4 deficiency epigenetically affects chromatin accessibility, alters the distribution of Polycomb group proteins on chromatin, and modulates histone acetylation, highlighting the potential efficacy of epigenetic regulators such as EZH2 and HDAC inhibitors. In addition, synthetic lethality strategies targeting vulnerabilities in SMARCA4-deficient tumors are promising therapeutic approaches, including inhibitors of SMARCA2, CDK4/6, ATR, CHK1, PARP, and the oxidative phosphorylation pathway. Based on current clinical evidence, ICI-based combination therapies represent the most promising first-line regimens for SMARCA4-deficient tumors. Although a theoretical basis supports the potential of tumor-agnostic therapy as a promising strategy for these tumors, several challenges remain in clinical practice. These include heterogeneous therapeutic responses across tumor types, safety concerns associated with synthetic lethality-based agents, and the absence of any histology-agnostic approved therapy for SMARCA4-deficient tumors. The continued development of novel therapeutics and further large-scale clinical evaluations are essential to overcoming these barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff8adc9ee03e886c2da7337f12fb44eb799835" target='_blank'>
              Tumor-agnostic therapy: a potential therapeutic approach for SMARCA4-deficient malignancies
              </a>
            </td>
          <td>
            Yang Liu, Zhi-Hui Liu, Qiong Zhang, Yin-Miao Bai, Xiao-Bo Guo, Hong-Chen Ji, Hong-Mei Zhang
          </td>
          <td>2025-12-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a70cf29a8102e7c49d2b80d42e52230a68a64c" target='_blank'>
              Integrative multi-omics reveals PARP14 as a key IFNγ-regulated mediator of metastatic progression in Ewing sarcoma
              </a>
            </td>
          <td>
            S. Sánchez-Serra, M. Chicón-Bosch, Núria Martorell-Villanueva, Eduardo Candeal, Ignasi Jarne-Sanz, J. Bech-Serra, D. Giménez-Llorente, Manuel Gris-Lorente, P. Monaco, L. Santana-Viera, Silvia Mateo-Lozano, Jaume Mora, Mario Niepel, Carolina de la Torre Gomez, Rebeca Sanz-Pamplona, Ana Cuadrado, Ana Losada, R. López-Alemany, Ò. M. Tirado
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Metastasis is the primary cause of cancer-related mortality worldwide. This narrative review integrates recent advances in the molecular circuits orchestrating metastatic progression, encompassing epithelial–mesenchymal transition (EMT), organotropism, extracellular matrix remodeling, angiogenesis, hypoxia-inducible signaling, tumor-cell migration modes, and tumor–immune interactions through expert-guided literature selection. We examined therapeutic innovations that disrupt these pathways, including EMT modulators, matrix metalloproteinase inhibitors, VEGF/VEGFR-targeted regimens, hypoxia-activated prodrugs, and next-generation immunotherapies such as immune checkpoint blockade and chimeric antigen receptor T cells. Additionally, we discuss established nanotechnology-based delivery systems, advancing multi-omics integration, evolving single-cell analyses, and emerging CRISPR-Cas9 gene-editing applications as tools for improving metastasis detection, monitoring, and treatment. Despite this progress, translational obstacles persist, particularly regarding intratumoral heterogeneity, adaptive resistance, and limited preclinical model fidelity. Addressing these challenges requires biomarker-guided, multi-target therapeutic combinations, interdisciplinary collaboration, and globally inclusive clinical trials. This evidence underscores the importance of integrated strategies that simultaneously target intrinsic tumor plasticity and microenvironmental support to transform metastatic cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7d3de1a04e69b49fb547b3a441611b4041aca5" target='_blank'>
              Deciphering molecular pathways driving cancer invasion and metastasis: advances and therapeutic prospects
              </a>
            </td>
          <td>
            Aziza Alshahrani, Arwa Alsubait, Z. Asiri, Sahar Alghamdi, Sarah Bin Saqyah, Tariq Alqahtani, Rawan Fitaihi, Njoud Altuwaijri, Yahya F. Jamous
          </td>
          <td>2025-12-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Colorectal cancer (CRC) persists as a significant global health challenge, distinguished by intricate molecular modifications and a notable propensity for resistance to standard therapeutic interventions. Among the regulatory factors contributing to CRC pathogenesis, microRNAs (miRNAs) have emerged as pivotal regulators of gene expression, presenting innovative prospects for diagnostic and therapeutic advancements. Notably, microRNA-143 (miR-143) has attracted considerable attention as a tumor-suppressive miRNA, exhibiting diverse functions in the development, progression, and therapeutic response of CRC. This review delineates an exhaustive examination of the molecular mechanisms by which miR-143 modulates critical oncogenic pathways, encompassing KRAS signaling, epithelial-mesenchymal transition, and metabolic reprogramming. We underscore recent progress in the molecular biosciences that position miR-143 as a promising biomarker for the early detection and prognosis of CRC. Furthermore, we investigate its emergent function in the modulation of sensitivity to chemotherapeutic and targeted therapeutic agents, emphasizing its potential utility in predicting and mitigating drug resistance in CRC cells. By synthesizing contemporary findings within the domains of molecular diagnostics and therapeutic interventions, this review accentuates the clinical potential of targeting miR-143 in the personalized management of CRC and the prediction of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8c35da017b5ecf9023804e3cb13c799f4993a0" target='_blank'>
              Targeting microRNA-143 in colorectal cancer: advances in molecular biosciences for biomarker-based diagnostics, therapeutic strategies, and drug resistance prediction
              </a>
            </td>
          <td>
            Xue Wang, Lei Wang, Long Ke
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, Mohammad Alinoor Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15c2c938fad8c796a1fdbc96dde9e4476c7e8f1" target='_blank'>
              Distinct mechanisms of cell in breast cancer and their clinical implication
              </a>
            </td>
          <td>
            Aodi Li, Sishi Liu, Ying Xuan, Ying Liang, Lei Wang, Lei Zhao
          </td>
          <td>2025-12-09</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lung cancer, recognized as one of the most prevalent malignancies with the highest rates of incidence and mortality globally, presents a substantial challenge on a worldwide scale. This challenge is exacerbated by the disease’s difficulty in early detection, a pronounced rate of metastasis, and resistance to treatment, all of which contribute to elevated mortality rates. The tumor microenvironment (TME) plays a critical role in the sustenance and advancement of various solid tumors, including lung cancer. The intricate composition of the TME facilitates tumor proliferation, metastatic spread, and therapeutic resistance by supplying metabolic resources, fostering angiogenesis, and enabling immune evasion. Nonetheless, the regulatory frameworks operating within the TME remain poorly understood. An increasing body of evidence suggests that epigenetic regulation—encompassing mechanisms such as DNA methylation, histone modification, and the action of non-coding RNAs—is pivotal in the initiation and progression of lung cancer. Furthermore, epigenetic modifications significantly influence the functional dynamics of the tumor microenvironment, thereby impacting intercellular interactions and cellular behaviors within the TME, which in turn affects the trajectory of disease progression. This article aims to present the most recent advancements in research concerning the epigenetic regulation of tumor cell interactions with the TME in the context of lung cancer biology. Additionally, it examines the current implications of epigenetic regulation within the tumor microenvironment and its influence on lung cancer behavior. We also investigate the potential relevance and emerging therapeutic avenues presented by epigenetic regulation in the clinical diagnosis and treatment of lung cancer, aspiring to propose novel strategies to address existing treatment challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f9bbf3d9c9172ba516c36021afce36965e86d5" target='_blank'>
              Epigenetic regulation of the tumor microenvironment in lung cancer: mechanism insights and therapeutic prospects
              </a>
            </td>
          <td>
            Maoqin Yang, Xudong Lei, Dexiang Ren, Dakai Qin, Xiaojun Xia
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumorigenesis is closely related to defects in DNA damage response (DDR), which plays an important role in maintaining genomic integrity. The DDR process includes DNA damage recognition, cell cycle checkpoint pathway activation, and DNA damage repair. On the one hand, defects in DDR increase the instability of the cellular genome, which ultimately leads to tumorigenesis. On the other hand, tumor therapy can take advantage of DDR defects to induce apoptosis by causing damage to tumor cells, thus achieving therapeutic goals. Tumor immunotherapy is an emerging method of tumor treatment that attacks and kills cancer cells by inhibiting negative regulatory factors, enhancing the ability of immune cells to recognize antigens on the surface of tumor cells and activating the patient's own immune system. Recent research evidence suggests that DDR inhibition enhances antitumor immune responses and acts in synergy with immunotherapy. This review describes the different categories of DNA damage, their corresponding DNA damage repair pathways, the antitumor mechanisms of action targeting DDR inhibition, and the prominent contributions of these mechanisms to the immunotherapeutic response. By exploring recent advances in the interactions between targeted DDR and immunotherapy, this review provides novel therapeutic insights into the treatment of DNA damage-associated cancers as well as the development of combination immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ffe870d92fa7406cd9d4f5f65954a04fd0351a" target='_blank'>
              Implications of DNA damage response and immunotherapy in tumor therapy
              </a>
            </td>
          <td>
            Wenqin Li, Binghang Li, Chang Cai, Jie Chen, Yiqiu Li, Rongrui Liang, Jing Zhao
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) remains a major contributor to cancer-related morbidity and mortality among women globally. While systemic therapies have significantly advanced, the emergence of drug resistance continues to hinder durable clinical benefit, leading to treatment failure and disease relapse. Long non-coding RNAs (lncRNAs) have gained recognition as key regulators in BC biology and therapeutic response, with growing evidence implicating them in resistance to chemotherapy, endocrine therapy, targeted agents, and immunotherapy. This review provides a detailed overview of the molecular mechanisms through which specific lncRNAs promote resistance, emphasizing their roles in modulating processes such as programmed cell death, epithelial-mesenchymal transition, and remodeling of the tumor microenvironment. We further highlight common regulatory axes - such as competing endogenous RNA networks and the phosphatidylinositol 3-kinase/protein kinase B pathway - that serve as converging points across distinct resistance mechanisms. In addition, we discuss recent progress in harnessing lncRNAs as liquid biopsy biomarkers and examine the key challenges and opportunities in translating lncRNA-based targets into therapeutic strategies. Together, these insights offer a foundation for future research aimed at overcoming drug resistance in BC through novel lncRNA-centered interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfac8b955e147276484ab116353ca6e5f90be4a2" target='_blank'>
              The role of long non-coding RNAs in breast cancer drug resistance: a comprehensive review
              </a>
            </td>
          <td>
            Yingkun Xu, Jingyu Ma, Changyuan Ding, Weili Liang, Q. Zeng, Bin Lv
          </td>
          <td>2025-12-10</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains one of the leading causes of cancer-related morbidity and mortality in men worldwide, and therapeutic resistance—particularly to immunotherapy—continues to limit clinical efficacy. Mounting evidence has positioned the immunosuppressive tumor microenvironment (TME) as a core driver of disease progression and a formidable barrier to effective immune-based interventions. In this review, we present a comprehensive and multi-dimensional analysis of the cellular, stromal, and molecular constituents of the immunosuppressive TME in PCa, highlighting its significant heterogeneity and context-dependent functions. We emphasize recent breakthrough insights obtained through single-cell RNA sequencing (scRNA-seq), spatial multi-omics, and high-dimensional imaging technologies, which are redefining our understanding of tumor–immune–stromal interactions. Based on these mechanistic findings, we examine precision therapeutic strategies aimed at remodeling the TME, including combinatorial immune checkpoint blockade, metabolic reprogramming, cytokine network regulation, and advanced nanomedicine-based delivery systems. Finally, we discuss translational opportunities and future research directions, underscoring the necessity of integrating advanced omics technologies with biomarker-driven clinical trial design to enable individualized, precision interventions and improve survival outcomes for patients with PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities
              </a>
            </td>
          <td>
            Mingfeng Li, L. Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="The convergence of genomics and epigenetics has revolutionized biomedical science by illuminating the molecular underpinnings of health, disease, and individual variability. While the sequencing of the human genome has provided insight into genetic predispositions, the emergence of epigenetic science has revealed how environmental and lifestyle factors dynamically influence gene expression without altering DNA sequences. This review comprehensively explores the interplay between genomic architecture and epigenetic regulation, elucidating their combined roles in disease manifestation, progression, and therapeutic response. We further discuss how personalized health, driven by genomic and epigenetic profiling, is reshaping the landscape of preventive and precision medicine. Emphasis is placed on recent advances in epigenomic technologies, the Lifestylopathy approach, and the ethical implications of genomic personalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3474cc81df5564773d87c15552102429c9aa1d4c" target='_blank'>
              Epigenetics and Genomics in Personalized Health: Unravelling Mechanisms of Human Diseases
              </a>
            </td>
          <td>
            Dr Neha Singh, Dr. R C Gupta, Dr. Indra Pratap Singh
          </td>
          <td>2025-11-20</td>
          <td>Advanced International Journal for Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c1df38e22e1ee22880d25154078c1bc6f87527f" target='_blank'>
              EP300 deficiency leads to chronic replication stress mediated by defective replication fork protection.
              </a>
            </td>
          <td>
            Angelica Barreto-Galvez, Mrunmai Niljikar, J. Gagliardi, Carolina Plasencia Guzman, Ranran Zhang, Vasudha Kumar, Aastha Juwarwala, Archana Pradeep, Ankit Saxena, Cristina Montagna, Priya Mittal, Jeannine Gerhardt, Bing Xia, Jian Cao, Keisuke Kataoka, A. Durbin, Jun Qi, B. H. Ye, Advaitha Madireddy
          </td>
          <td>2025-12-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/534987841cc5b057714d290f608f6acdcd5ef2ac" target='_blank'>
              Iron and metabolic rewiring in cancer.
              </a>
            </td>
          <td>
            Marina Ciscar, César Rodríguez-Santana, Naiara Santana-Codina
          </td>
          <td>2026-01-09</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db19d348df7947bfaa58517d856ab10f519b71ee" target='_blank'>
              Epigenomic landscape of nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Saleh Suleiman Silmi Almohammadin, R. B. S. M. N. Mydin, Rosle Bahar, Denisha Sivasamy, Muhamad Yusri Musa
          </td>
          <td>2025-12-26</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancers are complex diseases characterized by dynamic perturbations of regulatory networks across multiple hierarchical levels, which cannot be fully captured by alterations in a small number of genes. To this end, based on the concept of Hallmarks of Cancer, a whole genomic data-driven approach is proposed to capture the dynamic variation from normal to cancerous cells. This framework focuses on the characteristic functional modules of cancer via hallmarks of cancer by constructing a coarse-grained gene regulatory network of hallmarks. Through this framework, with stochastic differential equations, macroscopic dynamic changes in tumorigenesis are simulated and further explored. The analysis results reveal that network topology undergoes significant reconfiguration before shifts in hallmark levels, serving as an early indicator of malignancy. A pan-cancer examination across 15 cancer types uncovers universal patterns, for example, the “Tissue Invasion and Metastasis” hallmark exhibits the most significant difference between normal and cancer states, while “Reprogramming Energy Metabolism” shows the least pronounced differences. These findings reinforce the systemic nature of cancer evolution, highlighting the potential of network-based systems biology methods for understanding critical transitions in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6814d7f9a3ff58aa852df9a43fe33cc0022a2e6" target='_blank'>
              Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-11-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a5fb32298b06f7826592aac1dafed7fb246a78" target='_blank'>
              Genotype-phenotype correlations and de novo induction of cancer stem cells in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M. Linssen, J. Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gliomas are among the most malignant and aggressive tumors of the central nervous system, characterized by the absence of early diagnostic markers, poor prognosis, and a lack of effective treatments. Advances in high‐throughput technologies have facilitated a refined molecular classification of gliomas, incorporating genetic features. However, diagnosis and clinical management based on isolated genetic data often fail to capture the full histological and molecular complexity of these tumors, posing significant challenges. In the era of computational methodologies and artificial intelligence, the integration of multiple omics layers—genomics, transcriptomics (including sex‐dependent differential expression patterns), epigenomics, proteomics, metabolomics, radiomics, single‐cell analysis, and spatial omics—into a comprehensive framework holds the potential to deepen our understanding of glioma biology and enhance diagnostic precision, prognostic accuracy, and treatment efficacy. Herein, we provide a comprehensive overview of multi‐omics strategies used to decipher the adult‐type diffuse glioma molecular taxonomy and describe how the integration of multilayer data combined with machine‐learning‐based algorithms is paving the way for advancements in patient prognosis and the development of personalized, targeted therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2325480a513f4711617466fc9b8bcf51e07f5072" target='_blank'>
              Multi‐Omics Integration for Advancing Glioma Precision Medicine
              </a>
            </td>
          <td>
            M. Guarnaccia, Valentina La Cognata, G. Gentile, Giovanna Morello, S. Cavallaro
          </td>
          <td>2025-11-17</td>
          <td>Annals of Clinical and Translational Neurology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c4664b6580edc429e59cb7fc3bb0cee1ddce94" target='_blank'>
              LISA: A Comprehensive R Package for Lentiviral Integration Site Analysis in Gene Therapy Safety Assessment
              </a>
            </td>
          <td>
            Shuai Ni, Ning Wu, Yun Kang, Fengjiao Zhu, Jueyu Wang, Lina Wang, Wei Wang, Heng Mei, Kejia Kan
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="RNA modification, a prominent epigenetic mechanism, has been implicated in regulating RNA function, stability, processing, and interactions, including pseudouridylation, acetylation, and methylation. Recent evidence highlights that 5-methylcytosine (m5C) influences key cellular processes such as proliferation, differentiation, apoptosis, and stress responses by modulating RNA stability, translation, transcription, nuclear export, and cleavage. This review consolidates current insights into the role and mechanisms of m5C methylation across various tumor types, underscoring its pivotal involvement in post-transcriptional regulation and its profound effects on gene expression, cellular dynamics, and tumor biology. The mechanisms through which m5C methylation impacts tumor progression, including modulation of glucose and iron metabolism, as well as resistance to therapeutic agents, are also discussed. Finally, the review identifies critical future research avenues, focusing on elucidating the underlying mechanisms, developing targeted therapies, and advancing personalized medicine approaches to leverage m5C methylation in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/201f634747623dbdceebac9d2c8e4b2347b7edde" target='_blank'>
              RNA m5C methylation in cancer: mechanisms and biological impact
              </a>
            </td>
          <td>
            Zhenyu Guan, Wendong Li, Yuting He, Wenzhi Guo
          </td>
          <td>2025-11-21</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b17689bb0c61711df8a9782fea6ac34ddce8c95c" target='_blank'>
              FOXO1 links KRAS G12D and G12V alleles to glutamine and nitrogen metabolism in colorectal cancer.
              </a>
            </td>
          <td>
            Suzan Ber, Ming Yang, M. Sciacovelli, S. Samarajiwa, Khushali Patel, Efterpi Nikitopoulou, Annie Howitt, Simon J. Cook, Ashok R Venkitaraman, C. Frezza, Alessandro Esposito
          </td>
          <td>2025-11-20</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Lineage plasticity is a cancer hallmark that drives disease progression and treatment resistance1,2. Plasticity is often mediated by epigenetic mechanisms that may be reversible; however, there are few examples of such reversibility. In castration-resistant prostate cancer (CRPC), plasticity mediates resistance to androgen receptor (AR) inhibitors and progression from adenocarcinoma to aggressive subtypes, including neuroendocrine prostate cancer (CRPC-NE)3, 4–5. Here we show that plasticity-associated treatment resistance in CRPC can be reversed through the inhibition of NSD2, a histone methyltransferase6. NSD2 upregulation in CRPC-NE correlates with poor survival outcomes, and NSD2-mediated H3K36 dimethylation regulates enhancers of genes associated with neuroendocrine differentiation. In prostate tumour organoids established from genetically engineered mice7 that recapitulate the transdifferentiation to neuroendocrine states, and in human CRPC-NE organoids, CRISPR-mediated targeting of NSD2 reverts CRPC-NE to adenocarcinoma phenotypes. Moreover, a canonical AR program is upregulated and responses to the AR inhibitor enzalutamide are restored. Pharmacological inhibition of NSD2 with a first-in-class small molecule reverses plasticity and synergizes with enzalutamide to suppress growth and promote cell death in human patient-derived organoids of multiple CRPC subtypes in culture and in xenografts. Co-targeting of NSD2 and AR may represent a new therapeutic strategy for lethal forms of CRPC that are currently recalcitrant to treatment. Inhibition of the histone methyltransferase NSD2 and the androgen receptor in preclinical models can reverse lineage plasticity to suppress tumour growth and promote cell death in multiple subtypes of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31a72e85b6954f413f71412e13d4f1a6449106e" target='_blank'>
              NSD2 targeting reverses plasticity and drug resistance in prostate cancer
              </a>
            </td>
          <td>
            Jia J Li, A. Vasciaveo, Dimitris Karagiannis, Zhen Sun, Kristjan H. Gretarsson, Xiao Chen, O. Ouerfelli, F. Socciarelli, Ziv Frankenstein, Hanyang Dong, Min Zou, W. Yuan, Guangli Yang, Gabriel M Aizenman, Tania Pannellini, Xinjing Xu, H. Beltran, Yu Chen, Kevin L Gardner, Brian D. Robinson, J. D. De Bono, O. Gozani, Cory Abate-Shen, Mark A. Rubin, M. Loda, Charles L. Sawyers, Andrea Califano, Chao Lu, Michael M. Shen
          </td>
          <td>2025-11-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da30b80b8917619722f2ccc295095fa65d05c44e" target='_blank'>
              Gene functions determine stochastic or adaptive futile transcription in cancers through deregulated start site deployments
              </a>
            </td>
          <td>
            Aditi Lakshmi Satish, Praveen Kumar, Ishani Morbia, Umashankar Singh
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome instability (CIN) remains among the most important problems in modern cancer research. In this study, we conducted a genome-wide RNAi screen to identify genes that contribute to CIN. To achieve this, we used a human artificial chromosome in a novel sensitized screen to measure CIN. We screened 18,658 genes for their roles in maintaining chromosomal stability and identified 834 candidates as potential CIN genes. A secondary RNAi screen identified 44 genes with the most pronounced CIN phenotypes. In guilt-by-association analysis using a published set of 8,498 proteins across a panel of 949 cancer cell lines, this cohort of 44 genes displayed a striking correlation with mitotic regulators. Furthermore, altered expression of these proteins was associated with a poor prognosis across multiple cancer types. Specifically, downregulation of AMY2B, ALAD, PDGFRA, PPIE, VEZ1, and TTC19 is associated with poor survival in two or more of the following: small cell lung cancer, lung adenocarcinoma, adrenocortical carcinoma, ovarian cancer, and breast cancer. The genes identified in this screen hold potential as prognostic markers for patient survival across several cancer types and could potentially serve as targets for the development of new therapeutic approaches aimed at mitigating CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48402b9da158c79dbc9aec979b9f8a854d30f0b3" target='_blank'>
              A genome-wide RNAi screen for novel CIN genes using human artificial chromosome.
              </a>
            </td>
          <td>
            Mikhail Liskovykh, N. Y. Kochanova, Chih-Yuan Chiang, A. Dhall, V. Aksenova, Yu-Chi Chen, William C Reinhold, M. Dasso, Anish Thomas, K. Cheng, Y. Pommier, William C. Earnshaw, Vladimir Larionov, N. Kouprina
          </td>
          <td>2025-12-01</td>
          <td>PNAS nexus</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Lung cancer persists as the leading cause of cancer-related mortality globally, largely due to late detection, genomic complexity, and limited durability of existing therapeutic interventions. The integration of CRISPR/Cas9 gene-editing systems with biomarker-driven therapeutic strategies represents a transformative advance in precision oncology. Biomarkers—including genetic mutations, epigenetic alterations, protein signatures, and circulating analytes—enable early detection, patient stratification, and dynamic monitoring of therapeutic response. CRISPR/Cas9 offers a unique opportunity to directly reprogram these biomarkers or the pathways regulating them, enhancing tumor immunogenicity, reversing immune evasion mechanisms, and strengthening anti-tumor immune responses. Preclinical models demonstrate that CRISPR-mediated biomarker editing can restore antigen presentation, augment T-cell cytotoxicity, sensitize resistant tumors to immunotherapy, and improve tumor regression. Early clinical trials further validate the feasibility and safety of CRISPR-engineered immune cells in patients. 
However, major challenges persist, including off-target editing, inefficient delivery to solid tumors, tumor microenvironment–mediated suppression, and ethical considerations linked to genome manipulation. Rapid advancements in editing fidelity, lipid nanoparticle systems, viral vectors, engineered vesicles, high-throughput biomarker discovery, and artificial intelligence–assisted CRISPR design are expected to accelerate clinical translation. This review synthesizes the current landscape, mechanistic underpinnings, emerging applications, and future directions of CRISPR/Cas9-enabled biomarker engineering for lung cancer immunotherapy, positioning this technology as a cornerstone of next-generation personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80a98d71ca868cfed559df1a6401f5e2e8b0702" target='_blank'>
              Precision Reprogramming of Lung Cancer Biomarkers via CRISPR/Cas9: A Paradigm Shift in Personalized Immunotherapy
              </a>
            </td>
          <td>
            M. R. Jameel, Amar Arora, Prem Shankar Mishra, Hina Jameel, M. Makhdoomi, Abid Ali Sheikh
          </td>
          <td>2025-12-30</td>
          <td>Journal of Analytical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b02f1b354e2c642c1280c585ed8336d631e18" target='_blank'>
              ZMYM3 S464: a potential phospho-regulatory hub in epigenetic remodeling and oncogenesis
              </a>
            </td>
          <td>
            Apoorva Pai, Althaf Mahin, Samseera Ummar, Athira Perunelly Gopalakrishnan, Prathik Basthikoppa Shivamurthy, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-06</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma (NB) is the most common extracranial solid tumor in children and accounts for 15% of childhood cancer death. The nucleosome remodeling and deacetylase (NuRD) complex is a major chromatin remodeling complex that regulates chromatin accessibility and gene transcription. However, its role in the pathogenesis of neuroblastoma remains poorly understood.


METHODS
The genetic dependency and clinical significance of MBD3 in neuroblastoma was evaluated by analysis of public datasets. The function of MBD3 in neuroblastoma cell growth was evaluated by shRNA knockdown experiment. Cleavage under targets and tagmentation sequencing (CUT&Tag-seq), coupled with RNA-sequencing, was employed to explore the mechanisms involved in the epigenetic regulation executed by NuRD decommissioning following MBD3 deficiency.


RESULTS
Here we find that MBD3 is the most lineage-selective dependency among the non-enzymatic subunits of the NuRD complex in neuroblastoma. Knockdown of MBD3 induces cell cycle arrest and apoptosis, and inhibits neuroblastoma growth in vivo. Mechanistically, MBD3 deficiency leads to decommissioning of the NuRD complex and dissociation of the EZH2-PRC2 complex from chromatin, thereby orchestrating the epigenetic regulation of gene expression by modulating the balance between histone acetylation and methylation. NuRD decommissioning upon MBD3 deficiency selectively downregulates the expression of core regulatory transcription factors and upregulates a tumor suppressor SRCIN1, collectively suppressing neuroblastoma progression.


CONCLUSIONS
Our data identify MBD3 and the NuRD complex as potential therapeutic targets in neuroblastoma, highlighting the critical role of epigenetic regulation in tumor maintenance. Targeting this pathway may offer a novel strategy to selectively impair neuroblastoma cell survival and improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9509051ef6556e77505cd01c5bcf3d3b1c22a6b6" target='_blank'>
              MBD3 deficiency decommissions the NuRD complex and orchestrates the epigenetic regulation of gene expression to suppress neuroblastoma progression.
              </a>
            </td>
          <td>
            Chen Chen, Bize Guo, Siyan Wu, Qinfang Zhu, Hui Shi, Jinhu Wang, Qiang Shu, Ting Tao
          </td>
          <td>2026-01-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction DNA double-strand breaks (DSBs) are repaired via homologous recombination (HR) or the more error-prone non-homologous end joining (NHEJ). breast cancer gene 1 (BRCA1) and breast cancer gene 2 (BRCA2) are key genes in HR, and their mutations are associated with aggressive prostate cancer (PCa). While PARP inhibitors (PARPi) improve survival in BRCA-mutated PCa, their efficacy in late-stage disease is limited and often accompanied by serious side effects. This study aims to develop an in vitro model of BRCA-mutated PCa and evaluate the therapeutic potential of DNA-dependent protein kinase (DNA-PK) inhibitors that target the NHEJ pathway. Methods The genes BRCA1 and BRCA2 were targeted for knockout (KO) in lymphnode cancer of the prostate (cell line) [LNCaP] using clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 technology. The KO were assessed via Western blot analysis. Scramble LNCaP, BRCA1 KO, and BRCA2 KO cells were treated with the PARPi talazoparib, the DNA-PK inhibitor nedisertib and their combination. The impact on cell proliferation was evaluated using the CellTiter-Glo assay and synergy of the treatments was analyzed with SynergyFinder. Cytotoxic effects were measured by flow cytometry using an Annexin V-fluorescein isothiocyanate (FITC) apoptosis detection kit. The presence of DSB was quantified using immunofluorescence. Results BRCA2 and BRCA1 protein expression were successfully downregulated in the knockout (KO) cell lines. After two days of treatment with talazoparib and/or nedisertib, a significant decrease in cell proliferation was observed. Additive effects of the combination treatment were detected exclusively in the BRCA KO cells. These cells also exhibited significantly higher rates of necrosis after treatment compared to scramble cells and, DNA DSB were significantly more prevalent in the BRCA KO cells. Additionally, BRCA1/2 loss is inversely correlated with DNA-PK, with knockout leading to increased DNA-PK expression to support NHEJ. Conclusion BRCA knockout LNCaP models were established, exhibiting increased DNA-PK activity and indicating greater susceptibility to DNA-PK inhibition. Therapeutically targeting NHEJ presents a promising approach in treating BRCA-mutated PCa. Further in vivo investigations are required to assess the tolerability of this drug combination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a995f399e397271dd4a847fb2e46b35d96ad921b" target='_blank'>
              Combined DNA-PK and PARP Inhibition as a Therapeutic Strategy in BRCA-Mutated Prostate Cancer: An in Vitro Pilot Study
              </a>
            </td>
          <td>
            T. Scherer, S. Salemi, Valentin Baumgartner, Dominik Enderlin, Alekhya Mazumdar, Daniel Eberli
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9014a6b11423460ea6b5d8b30bbc873c704d2d2a" target='_blank'>
              A key regulator of missing-self innate immunity is polymorphic and under diversifying selection
              </a>
            </td>
          <td>
            Rocco F. Notarnicola, M. Herdegen-Radwan, Joanna Różańska-Wróbel, M. Konczal, Karolina Przesmycka, P. Kotlík, W. Babik, Jacek Radwan
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T-cell therapy constitutes a paradigm-shifting advancement for treating refractory malignancies, demonstrating unique therapeutic capabilities beyond conventional cancer treatments, which has brought new hope to many patients with advanced cancer. This cellular immunotherapy has achieved unprecedented success in hematologic cancers, yet its extension to solid tumors confronts fundamental biological constraints. Lifethreatening toxicities such as cytokine release syndrome and immune effector cell-associated neurotoxicity originate from dysregulated immune activation cascades, while T cell exhaustion—characterized by hierarchical epigenetic reprogramming and progressive metabolic failure—severely undermines long-term therapeutic efficacy. These dual limitations demand innovative bioengineering solutions to augment both safety profiles and functional persistence. The present review analyzes core biological mechanisms restricting CAR-T efficacy and surveys cutting-edge approaches designed to circumvent these barriers through integrated genetic reprogramming, metabolic optimization, and microenvironment-responsive receptor engineering. Subsequent discussion focuses on how such multidisciplinary strategies could potentiate cellular immunotherapy’s applicability across diverse oncological contexts, ultimately transforming cancer management paradigms.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87009d7a43106a2ecb2497d6da83253988dae72a" target='_blank'>
              Current Clinical Challenges and Strategic Advancements in CAR-T Cell Therapy
              </a>
            </td>
          <td>
            Ziling Shao
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in DNA mismatch repair (MMR) pathway genes (MSH2, MSH6, MLH1, and PMS2) are linked to acquired resistance to temozolomide (TMZ) and high tumor mutation burden (TMB) in high-grade gliomas (HGG), including glioblastoma (GBM). However, the specific roles of individual MMR genes in the initiation, progression, TMB, microsatellite instability (MSI), and resistance to TMZ in glioma remain unclear. Here, we developed de novo mouse models of germline and somatic MMR-deficient (MMRd) HGG. Surprisingly, loss of Msh2 or Msh6 does not lead to high TMB, MSI, nor confer response to anti-PD-1 in GBM. Similarly, human GBM shows discordance between MMR gene mutations and TMB/MSI.Germline MMRd leads to promoted progression from low-grade to HGG and reduced survival compared to MMR-proficient (MMRp) tumor-bearing mice. This effect is not tumor cell intrinsic but is associated with MMRd in the tumor immune microenvironment, driving immunosuppressive myeloid programs, reduced lymphoid infiltration, and CD8+ T cell exhaustion. Both MMR-reduced (MMRr) and MMRd GBM are resistant to temozolomide (TMZ), unlike MMRp tumors. Our study shows that KL-50, a imidazotetrazine-based DNA targeting agent inducing MMR-independent cross-link-mediated cytotoxicity, was effective against germline and somatic MMRr/MMRd GBM, offering a potential therapy for TMZ-resistant HGG with MMR alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59eb07f5af2f7c1bb36a92e8c3ca00e2e992ebc" target='_blank'>
              Mismatch repair deficiency drives malignant progression and alters the tumor immune microenvironment in glioblastoma models.
              </a>
            </td>
          <td>
            Montse Puigdelloses Vallcorba, N. Soni, Seung-won Choi, Kavita Rawat, T. Joshi, Sam Friedman, Alice Buonfiglioli, Angelo Angione, Zhihong Chen, Gonzalo Piñero, Gabrielle Price, Mehek Dedhia, Raina Roche, E. Radkevich, Anne M. Bowcock, Deepti Bhatt, Winfried Edelmann, R. Samstein, Timothy E. Richardson, N. Tsankova, A. Tsankov, R. Bindra, Raul Rabadan, Juan C Vasquez, D. Hambardzumyan
          </td>
          <td>2025-12-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Ovarian cancer is the gynecologic malignancy that bears the highest mortality rate in the Western world. This is attributed to late diagnosis and limited therapeutic progress. Recent advances in molecular oncology have highlighted the pivotal role of epigenetic modifications—including DNA methylation, histone modifications, non-coding RNAs, chromatin remodeling, and RNA methylation—in ovarian cancer development, progression, and treatment resistance. DNA methylation patterns affect key tumor suppressors and oncogenes, while histone modifications alter chromatin accessibility, influencing gene expression. Chromatin remodeling complexes, particularly the SWI/SNF complex, are frequently mutated in specific ovarian cancer subtypes, which is central in shaping their biological behavior. Non-coding RNAs, including microRNAs and long non-coding RNAs, further regulate tumor cell behavior and the immunosuppressive tumor microenvironment. Epigenetic profiles vary among histological subtypes and hold promise for biomarker development, early detection, prognosis, and therapeutic monitoring. Liquid biopsy approaches leveraging circulating tumor DNA methylation show diagnostic potential superior to conventional markers. Moreover, targeting epigenetic regulators—such as DNMT and HDAC inhibitors, EZH2 antagonists, and RNA-modifying enzymes—offers novel avenues for treatment, particularly in reversing chemoresistance and sensitizing tumors to immunotherapy. While promising, these strategies require further validation through clinical research to translate into effective clinical interventions. This review aims to summarize the current literature and highlights potential applications of epigenetic manipulation in day-to-day practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e682da74ce847ab8714dc4327d4b867b8d126a5" target='_blank'>
              Epigenetics in Ovarian Cancer: A Review of Current Knowledge and Future Perspectives
              </a>
            </td>
          <td>
            Nikolaos Dedes, Michael Liontos, Dimitrios Haidopoulos, Flora Zagouri, K. Angelou, A. Svarna, A. Michas, Aikaterini Aravantinou Fatorou, Angeliki Andrikopoulou, M. Dimopoulos
          </td>
          <td>2025-11-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has a high incidence and mortality rate globally, with approximately 40% of patients harboring Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations. Patients with KRAS mutations exhibit poorer prognoses compared to those with KRAS wild-type tumors, often showing resistance to targeted therapies and chemotherapy. There is an acute need for innovative therapeutic strategies, particularly as KRASG12C inhibitors have been approved by the U.S. Food and Drug Administration (FDA) for second-line therapy in treating non-small cell lung cancer (NSCLC). Yet, their efficacy in CRC remains suboptimal due to the emergence of drug resistance. Current research on KRAS inhibitor resistance in CRC focuses mainly on single biological processes, leaving the complex interplay between cellular systems underexplored. This review synthesizes evidence across four key dimensions — metabolism, epigenetics, immunology, and proteostasis — to reveal multidimensional mechanisms of resistance. How these interconnected pathways work in synergy with signaling aberrations, genetic alterations, and translational modifications to reinforce resistance was investigated. The research also integrates recent advances in multi-targeted strategies, including combinations of KRAS inhibitors with metabolic or epigenetic modulators or immune checkpoint blockade. These findings provide potential strategies for overcoming resistance in KRAS-mutant CRC, addressing a critical gap in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de09a3f40e67f360dfd11b639fa71c10cdd13842" target='_blank'>
              Unravelling Resistance: Integrating Metabolism, Epigenetics, Immunology, and Proteostasis in Strategies against Kirsten Rat Sarcoma Viral Oncogene Homolog-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            Jingyi Li, Na Song, Rui Ma, Xiujuan Qu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Brain tumors such as glioblastoma remain among the most lethal and immunologically resistant cancers, in large part due to epigenetic programs that sculpt the tumor–immune microenvironment. DNA methylation, histone modifications, and chromatin remodeling do not merely drive tumor-intrinsic changes; they also profoundly reprogram immune responses, shaping antigen presentation, cytokine signaling, and immune cell recruitment. At the center of this regulation are T cells, whose effector functions are suppressed through promoter hypermethylation of antigen-processing genes, silencing of interferon pathways, and the establishment of exhaustion-specific chromatin states. Mutations such as IDH1/2 and H3K27M further reinforce these epigenetic barriers, fostering immune-cold microenvironments that disable cytotoxic T-cell activity. Emerging evidence highlights both CNS-specific adaptations, including microglial and astrocytic epigenetic programs that reinforce immune privilege, and conserved features of T cell exhaustion that mirror those in peripheral cancers. This duality underscores the need for therapeutic strategies that dismantle CNS-specific barriers while leveraging shared exhaustion programs across tumor types. Epigenetic drugs, ranging from DNA methyltransferase and EZH2 inhibitors to BET degraders and CRISPR-based epigenome editors, are beginning to restore antigenicity, reverse T cell dysfunction, and sensitize tumors to checkpoint blockade. Yet these approaches carry the paradoxical risk of disrupting CNS immune tolerance, potentially triggering harmful neuroinflammation or autoimmunity. To our knowledge, this is among the first comprehensive reviews to integrate CNS-specific immune privilege mechanisms with peripheral exhaustion pathways, providing a unified perspective on how epigenetic regulation orchestrates immune dysfunction across central and peripheral contexts. By mapping the continuum between immune evasion and global immunosuppression, we propose a conceptual framework for tailoring epigenetic-immunotherapy combinations to achieve durable antitumor immunity in the CNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7202ed1c4ffc9d0ee982c0f1ad42403d845276" target='_blank'>
              Beyond the genome: epigenetic regulation of immune responses and T cells in brain tumors
              </a>
            </td>
          <td>
            Shuo Sun, Yu Han, Haiying Li, Chengyan Wang, Shu Zhou, Xiaowei Zhang, Shuhong Dai, Yao Peng, Zhuoqun Wang
          </td>
          <td>2025-11-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposable elements (TEs), commonly referred to as “jumping genes,” are discrete DNA fragments capable of mobilization within the genome, often creating new copies during transposition. Once considered non-functional “junk DNA,” TEs are now recognized as essential regulators of genome evolution, gene networks, and epigenetic processes. They contribute to genome innovation through gene duplication, exon shuffling, regulatory rewiring, and stress-responsive plasticity. Conversely, their uncontrolled activation can disrupt genetic integrity. Such aberrant TE activity has been associated with immune dysfunction, cancer, neurodegeneration, and reproductive disorders. 
In livestock, recent studies have identified TE-derived regulatory elements that influence growth, reproduction, immunity, and adaptation. Notably, TEs constitute approximately 45–60% of the genomes of major livestock species, including pigs, sheep, and cattle, highlighting their extensive regulatory impact. Advances in genome editing and artificial intelligence have facilitated their functional characterization. This allows a sharper focus on TE-driven regulation in animal genomes without repeatedly specifying the livestock context. 
Emerging technologies such as CRISPR–transposon systems, programmable epigenome editing, and AI-driven TE mapping offer new opportunities for precise genome engineering, functional genomics, and livestock breeding. This review consolidates current knowledge on TE classification, molecular mechanisms, evolutionary roles, disease associations, and biotechnological  applications, with emphasis on livestock genomics. It also outlines existing challenges, biosafety considerations, and future directions toward harnessing TEs for precision animal breeding and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434f9971fc3a63cead1542771f75e88db7b4209d" target='_blank'>
              Jumping Genes (Transposable Elements): A Comprehensive Review of Their Molecular Mechanisms, Evolutionary Impact and Applications in Livestock and Biomedicine
              </a>
            </td>
          <td>
            Dhruv Bhatia, Prakashji Thakor, Pankti J Naik, Brijal Barot
          </td>
          <td>2025-12-22</td>
          <td>Journal of Experimental Agriculture International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e47a668958565050b674aeca2b9f77b67dc49b4" target='_blank'>
              Investigation of a global mouse methylome atlas reveals subtype-specific copy number alterations in pediatric cancer models.
              </a>
            </td>
          <td>
            Melanie Schoof, T. Zheng, M. Sill, R. Imle, Alessia Cais, L. Altendorf, Alicia Fürst, Nina Hofmann, K. Ernst, Dominik Vonficht, Kenneth Chun-Ho Chan, T. Holland-Letz, Andreas Postlmayr, Ryo Shiraishi, Wanchen Wang, A. Morcavallo, Michael Spohn, C. Göbel, J. Niesen, Levke-Sophie Peter, Franck Bourdeaut, Zhi-Yan Han, Yanxin Pei, Najiba Murad, F. Swartling, Jessica Taylor, Monika Yadav, Garrett R Gibson, Richard J Gilbertson, M. Dottermusch, Rajanya Roy, K. Kerl, R. Glass, Jiying Cheng, M. Horstmann, G. Wolters-Eisfeld, Haotian Zhao, D. Sturm, Viveka Nand Yadav, L. Chesler, Simon Haas, William A. Weiss, P. Northcott, L. Kutscher, A. G. Guerreiro Stucklin, O. Ayrault, J. Neumann, Daisuke Kawauchi, David T. W. Jones, K. Pajtler, A. Banito, S. Pfister, U. Schüller, Marc Zuckermann
          </td>
          <td>2025-12-11</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="This paper delves into the role of epigenetics in complementing and challenging Darwinian evolution theory within the field of evolution. Epigenetics has revealed new sources of genetic variation, with epigenetic variations such as DNA methylation and histone modifications capable of influencing gene expression without altering DNA sequences. These variations can spread and accumulate within populations, exhibiting reversibility and dynamism, enabling species to rapidly adapt to environmental changes. Additionally, epigenetics expands the scope of natural selection, revises the patterns of genetic information flow, and highlights the need for updated evolutionary predictive models. Researchers have proposed theoretical integration pathways such as the Extended Evolutionary Synthesis framework, multilevel selection theory, and gene-epigenome coevolution models, offering new perspectives and directions for evolutionary biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b730d3e4497777825de2252ebd00338b352de17" target='_blank'>
              Epigenetics Complements and Challenges Darwinian Evolution: From Gene Regulation to the Study of Species Adaptation
              </a>
            </td>
          <td>
            Meichen Zhou
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="TP53 mutation-driven gene expression programs define oncogenic phenotypes. While extensive studies have concentrated on the transcriptome and proteome, post-transcriptional processes, particularly translational variation, remain underexplored. This study presents a comprehensive analysis of the transcriptomics, translatiomics, and proteomics dynamics in the ovarian cancer cell line SKOV3, with a focus on the effects of p53 missense mutations (R175H, R273H, and Y220C) on gene dosage fluctuations. Despite clear transcriptional differences between wild-type and mutant p53, we find that extensive translational and post-translational buffering processes attenuate these discrepancies, yielding comparatively stable protein abundances. Moreover, we delineate that the relative contributions of transcription output, translation engagement, and protein stability collectively shape the final protein abundance in the context of p53 mutations. Clinical proteomic analysis of platinum-resistant ovarian cancer tissues reveals tumor-specific factors and acquired resistance pathways linked to p53 mutations. Our findings elucidate the multilayered regulatory landscape of p53 mutations and identify potential risk factors for platinum resistance associated with these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765c815e9b25f95d1edceba753e2ba1a94641e62" target='_blank'>
              Multilayered Regulatory Dynamics of p53 Mutations and Platinum Resistance in Ovarian Cancer.
              </a>
            </td>
          <td>
            Liling Hu, Hanchen Zou, LvYing Peng, Fan Li, Danya Liu, Jiangli Lu, Yuying Li, Chris Zhiyi Zhang, Qiu-Hong Tian
          </td>
          <td>2025-12-16</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477ee36d35bf0a27b85e21d8ea8f32e1953a4901" target='_blank'>
              Interrupting glioblastoma malignant circuitry: plasticity traps, synaptic blockade, and myeloid reprogramming.
              </a>
            </td>
          <td>
            Hong Sheng Cheng, Carol Tang, Nguan Soon Tan
          </td>
          <td>2026-01-08</td>
          <td>Current opinion in clinical nutrition and metabolic care</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bac6ffe176a143cf7d0ecc53b36ca203893c495" target='_blank'>
              Precision Nanotechnology: Revolutionizing Therapeutic Strategies Against Drug-Resistant Breast Cancer.
              </a>
            </td>
          <td>
            ZahraSadat Razavi, Arefeh Mottaghi, Lyudmila Dmitrieva, Shiva Fallahianshafiei, Erfan Olfat, Nahid Ahmadi
          </td>
          <td>2026-01-07</td>
          <td>Annals of biomedical engineering</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b000d16168ddb5cfe7cae83d67af9ed383742f" target='_blank'>
              Cancer-associated fibroblasts at the crossroads of tumor progression and therapy resistance: from heterogeneity to precision reprogramming
              </a>
            </td>
          <td>
            Ravi Adusumalli, R. Banala
          </td>
          <td>2025-12-01</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary ADP-ribosylation is a post-translational modification that plays a crucial role in DNA repair, transcription, replication, and overall cell fitness. The ADP-ribose cycle represents one of the most tightly regulated biochemical pathways in the cell, as precise control of mono- and poly-ADP-ribose levels, together with the coordinated activity of the enzymes that synthesize and recycle ADP-ribose to NAD+, is essential for cellular homeostasis. Moving beyond the traditional cancer framework focused exclusively on oncogenes and tumor suppressor genes, ADP-ribose can be viewed as a central signaling molecule that regulates proteins involved in tumor progression and adaptation to stress conditions within the tumor microenvironment. In this context, proteins whose roles in ADP-ribosylation are relatively less characterized, such as mono-ADP-ribosyltransferases (MARTs) and poly(ADP-ribose) glycohydrolases (PARGs), are emerging as important regulators of these processes. As their functions and biological implications continue to be elucidated, these enzymes should be considered promising prognostic markers and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90553f7630f85c06dd88b57a27b79fc887c602e0" target='_blank'>
              Targeting MARylation and DePARylation in Cancer Therapy: New Promising Therapeutic Opportunities
              </a>
            </td>
          <td>
            Vanesa Cabeza-Fernández, Francisco Javier Ríos-Sola, D. Martín-Oliva, Jerónimo Borrego-Pérez, F. J. Oliver, J. Yélamos, J. Rodríguez-Vargas
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c25f4ef53a5a25e93945a09a8adf53121cb1fe0" target='_blank'>
              Tumor microenvironment-driven drug resistance in urologic cancers: mechanisms and therapeutic targets.
              </a>
            </td>
          <td>
            Gi-Eun Yang, Seo-Yeong Yoon, Ju-Seog Lee, S. Leem, Y. Choi
          </td>
          <td>2025-11-26</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b41a50acd50b88a1b8f2629c5c9719051133237" target='_blank'>
              The inhibition of de novo purine synthesis increases LAMP2 expression to preserve cell viability.
              </a>
            </td>
          <td>
            Angela De Cristofaro, Serena Castelli, Federica Felice, M. Ciriolo, Enrico Desideri
          </td>
          <td>2025-12-11</td>
          <td>Cell death discovery</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a major global public health challenge, driven by a multifactorial interplay of genetic, epigenetic, hormonal and environmental determinants. In recent years, the human microbiome has emerged as a critical and previously underappreciated contributor to PCa initiation, progression, and therapeutic response. Emerging high-resolution multi-omics studies have demonstrated that microbial communities across the gut, urinary tract and prostate form a functional axis that shapes immune surveillance, hormonal metabolism, inflammatory tone and epigenetic regulation. Dysbiosis in these compartments promotes chronic inflammation, modulates androgen receptor signaling, and produces bioactive metabolites, including short-chain fatty acids, that activate oncogenic IGF-1/MAPK/PI3K and NF-κB/JAK/STAT pathways. Cross-compartmental trafficking of bacterial taxa and metabolites reinforces tumor-promoting circuits, while specific commensals such as Akkermansia muciniphila enhance antitumor immunity and improve responses to androgen deprivation therapy. Importantly, microbiota-derived factors also modulate microRNA (miRNAs) expression and epigenetic signatures, thereby affecting tumor plasticity and resistance to therapy. These mechanistic insights have catalyzed interest in microbiome-based therapeutic approaches, including probiotics, prebiotics, fecal microbiota transplantation, dietary modulation and bacteriophage therapy, which hold promise for restoring eubiosis and enhancing treatment efficacy. Nevertheless, clinical translation remains limited by inter-individual variability and the need for well-designed, longitudinal studies integrating shotgun metagenomics, metabolomics and host-microbe interactomics. Overall, the prostate, urinary and gut microbiomes represent interconnected targets that may inform precision diagnostics and novel therapeutic strategies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6f8dcf9e9833df8dbea7a3387a16114216be92" target='_blank'>
              Reprogramming prostate cancer through the microbiome
              </a>
            </td>
          <td>
            Jhommara Bautista, Walter D. Cardona-Maya, Kelly Gancino-Guevara, Andrés López-Cortés
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The Methyl CpG Binding Protein 2 (MECP2) gene has copy number gain in a number of human cancers and functions as an oncogene through an unusual epigenetic mechanism. We explored the possibility that MECP2 might be a therapeutic target, and whether its epigenetic mode of action is amenable to specific therapy. Constitutively expressed or inducible lentiviral short hairpin RNA (shRNA) directed at MECP2 in human triple negative breast cancer (TNBC) cell lines with high or low level of MECP2 protein were grown as xenografts to assess oncogene addiction. We next evaluated the effect of DNA methylation inhibitors or histone deacetylase inhibitors on monolayer or soft agar growth of isogenic human mammary epithelial cells with or without MECP2 overexpression, and xenograft growth of MECP2-dependent or MECP2-independent human TNBC or lung cancer cell lines. We then investigated the mechanism of MECP2-induced activation of MAPK pathway and assessed the effect of drug treatment. Human cancer cell lines with MECP2 overexpression show MECP2 dependence, and epigenetic drugs are effective in these models. Activated RAS and other activators of the MAPK pathway caused resistance to these therapies, giving insight into their novel mode of action and demonstrating specificity. The kinase PAK3 is important for MECP2-mediated induction of MAPK pathway, is modulated by epigenetic drugs affecting MECP2 action as expected and may itself be a therapeutic target for MECP2-driven cancers. These preclinical studies show that tumors overexpressing MECP2 might benefit from epigenetic therapy targeting MECP2 function and demonstrate a novel mechanism of action for these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4802b6674f8b3e0a4e440436dd1fc69dc98c09c" target='_blank'>
              MECP2-Dependent Cancers Can Be Targeted by Epigenetic Drugs: A New Role for Epigenetic Cancer Therapy.
              </a>
            </td>
          <td>
            Manish Neupane, Mukesh Kumar, Malela A Mwamufiya, Colleen M Donnelly, Alsu Ibragimova, Brian Curcio, Inna Chervoneva, Irina Vasilevskaya, Daniel P Silver
          </td>
          <td>2025-12-11</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            , , M. L. Abel, D. S. Vargas Solivan, K. Bhadresha, S. Kartal, S. Nichols, K. Parmar, J. Twohig, T. M. Truong, C. H. Chau, K. Kelly, W. Figg, A. Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>None</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Persistent fetal gene expression in childhood neoplasms is usually explained by a maturation block originating in the prenatal phase. In contrast, reactivation of fetal genes in adult malignancies is considered a consequence of oncofetal reprogramming (OFR) and is associated with aggressive disease. By reconstructing epigenetic ontogeny in juvenile myelomonocytic leukemia (JMML), we identified a postnatal maturation state of JMML stem cells with high transcriptional plasticity indicative of OFR in high-risk disease. Similarly, post-natal activation of oncogenic signaling by inducible Ptpn11E76K mutation in mice, triggered molecular plasticity and reactivation of fetal gene expression. Integrative multi-omics analysis revealed aberrant CD52 expression as a feature of high-risk JMML stem cells. Anti-CD52 treatment depleted JMML stem cells and blocked disease propagation in xenograft models. Our results challenge the prevailing maturation-block model of pediatric leukemogenesis and establish RAS-associated stem-cell plasticity as a determinant of OFR and potential therapeutic vulnerabilities in high-risk JMML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c603c5b4c0adc417c6958428e24696c22fabe88d" target='_blank'>
              Molecular Plasticity Results in Oncofetal Reprogramming and Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia.
              </a>
            </td>
          <td>
            M. Hartmann, M. Schönung, J. Rajak, Valentin J. Maurer, Ling Hai, K. Bauer, M. Hakobyan, Sina Stäble, Jens Langstein, Laura Jardine, R. Roelz, S. Bohler, Eleonora Khabirova, Abdul-Habib Maag, Dominik Vonficht, D. Lebrecht, K. Bernt, Kai Tan, Changya Chen, Fatemeh Alikarami, Julia A. Meyer, Jun Wang, Tobias Boch, Viktoria Flore, P. Lutsik, M. Milsom, S. Raffel, Christian Buske, Simon Haas, M. Haniffa, Jan-Philipp Mallm, S. Behjati, M. Bonder, Stefan Frohling, Elliot Stieglitz, Charlotte M. Niemeyer, Joschka Hey, C. Flotho, C. Plass, M. Erlacher, Matthias Schlesner, D. Lipka
          </td>
          <td>2025-12-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Cancer reversion therapy represents a paradigm shift in oncology, focusing on reprogramming malignant cells to a non-malignant state rather than destroying them. This narrative review synthesizes current evidence, emerging technologies, and future directions in this promising field. Cancer reversion is founded on key biological observations: somatic cell reprogramming, spontaneous cancer regression, and microenvironmental influences on malignant behavior. Current approaches include epigenetic reprogramming using HDAC inhibitors and DNA methyltransferase inhibitors; microenvironmental modulation through extracellular matrix manipulation and vascular normalization; differentiation therapy exemplified by all-trans retinoic acid in acute promyelocytic leukemia; and targeting oncogene addiction as demonstrated in BCR-ABL-driven leukemias. Emerging technologies accelerating progress include single-cell analyses that reveal cancer heterogeneity and cellular state transitions; CRISPR-based approaches enabling precise genetic and epigenetic manipulation; patient-derived organoids that model tumor complexity; and artificial intelligence applications that identify novel reversion-inducing agents. Critical evaluation reveals that many reported “reversion” phenomena represent stimulus-dependent plasticity or transient growth arrest rather than stable phenotypic normalization. True cancer reversion requires durable, heritable phenotypic changes that persist after treatment withdrawal, with evidence of epigenetic consolidation and functional restoration. Despite promising advances, significant challenges remain: cancer cell plasticity facilitating therapeutic escape, difficulties in establishing stable reversion states, delivery challenges for solid tumors, and the need for combination approaches to address tumor heterogeneity. Future directions include integrated multi-omics analyses to comprehensively map cellular state transitions, studies of natural regression phenomena to identify reversion mechanisms, advanced nanodelivery systems for targeted therapy, and synthetic biology approaches creating intelligent therapeutic systems. By redirecting rather than destroying cancer cells, reversion therapy offers the potential for reduced toxicity and resistance, potentially transforming cancer from a deadly disease to a manageable condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ef16378f672c1c6324bdae132a2624ad963a92" target='_blank'>
              Cancer Reversion Therapy: Prospects, Progress and Future Directions
              </a>
            </td>
          <td>
            Emmanuel O. Oisakede, D. Olawade, Oluwakemi Jumoke Bello, Claret Chinenyenwa Analikwu, Eghosasere Egbon, Oluwaseun Fapohunda, S. Boussios
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="ABSTRACT Growing knowledge around disease states has led to opportunities within research to make designer molecules with improved specificity and broader efficacy. These next-generation molecules frequently take advantage of multispecific targeting and controlled mechanisms of action by utilizing four unique peptide chains as seen in many bispecific or trispecific antibodies. However, with all the opportunities these multispecifics offer, their increased biological complexities come with increased challenges during expression and purification to produce high-quality material. Lower yields accompanied with a high degree of mispairing after the initial capture purification step are often limiting factors. Developing new methods for stable pool expression can offer a strong advantage for progressing these molecules through research toward development. Here, we implemented optimized stable cell pools using targeted dual selection (TDS), a novel approach that combines specified selective pressure with transposon-guided semi-targeted gene integration. By utilizing key analytical data obtained during early-stage high-throughput transient productions, we can predict improved vector configurations for the generation of optimized TDS stable pools. We demonstrate that this design can improve molecule quality at the initial capture purification step in two Y-shaped bispecific molecules and two cross-over dual variable trispecific molecules by achieving up to four-fold increase in protein of interest yields while maintaining product quality. Use of this strategy in research can enable simplified purification strategies as well as increased production yields required for successful and timely project progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4ead5e389c831e279ea452ea421a912ad9538e" target='_blank'>
              Targeted dual selection to optimize transposon stable pool generation of multispecifics
              </a>
            </td>
          <td>
            Julie Johnston, Sonja Tierson, Yuyan Xu, Kalie Mix, Yj Jane Guo, Serhan Zenger, David Reczek, Dietmar Hoffmann, Brian Hall, Virginia Brophy
          </td>
          <td>2025-12-14</td>
          <td>mAbs</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J. K. Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d9b30b9c8b778788d45f8a145085491f7761e84" target='_blank'>
              CAR-adapted PIK3CD base editing enhances T cell anti-tumor potency.
              </a>
            </td>
          <td>
            Philip Bucher, Nadine Brückner, Jule Kortendieck, Melanie Grimm, Jan T. Schleicher, Karlotta Bartels, Steffen Hardy, Martina Rausch, Hannah Wurzer, Meike Thiemann, Celina May, Mara Mitstorfer, Dennis Letzgus, Julia Quach, Carolin Schneider, Denis A Ispan, I. González-Menéndez, Nayan Jain, Yu-jui Ho, Jiangqing Chen, Francisco J. Sánchez-Rivera, Jie Sun, Leticia Quintanilla-Martinez, Christoph Trautwein, Bettina Weigelin, M. Claassen, Michel Sadelain, Judith Feucht, Josef Leibold
          </td>
          <td>2026-01-06</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lung cancer remains a leading cause of cancer-related death. Despite advances in targeted therapies and immunotherapy, treatment outcomes remain suboptimal due to tumor heterogeneity and therapeutic resistance. The tumor microenvironment (TME), a dynamic ecosystem comprising immune cells, stromal components, extracellular matrix and bioactive molecules, serves a critical role in promoting tumor progression and resistance. The present review comprehensively analyzes the molecular mechanisms underlying TME-mediated immune evasion, and resistance to chemotherapy, radiotherapy and immunotherapy. In addition, emerging therapeutic strategies targeting the TME are highlighted, such as immune microenvironment modulation, metabolic and epigenetic interventions, and nanotechnology-based drug delivery systems. By integrating multi-omics datasets and spatial transcriptomics, TME-directed interventions are moving toward biomarker-guided, personalized regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fae8277ad66f4ef1a792064e768c58164735e5e" target='_blank'>
              The tumor microenvironment in lung cancer: Heterogeneity, therapeutic resistance and emerging treatment strategies (Review)
              </a>
            </td>
          <td>
            Li Liu, Li Yang, Hong-Mei Li, Tianlu Shang, Lihan Liu
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df76d59f3a1b2aa9161ce2bf7becbe9cb09edf1" target='_blank'>
              Uncovering the role of LINE-1 in the evolution of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, Jian Sang, John P McElderry, Michelle Antony, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J Colón-Matos, M. Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, N. E. Caporaso, M. Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. S. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, B. G. Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, R. Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin M. Brown, Nathaniel Rothman, S. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, D. Wedge, M. Landi
          </td>
          <td>2025-12-10</td>
          <td>Nature</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Drug-tolerant persister (DTP) cells drive therapeutic resistance in EGFR-mutant lung adenocarcinoma. Using single-cell RNA sequencing, we identified a clinically significant RGS5+MYL9+ cancer-associated fibroblast (CAF) population that was associated with EGFR tyrosine kinase inhibitor (TKI) resistance and poor prognosis. These CAFs were recruited to DTP niches via CCL11 signaling, and they formed tunneling nanotubes through Miro1/RhoA activation induced by TKI-generated mitochondrial reactive oxygen species (ROS). Remarkably, RGS5+MYL9+ CAFs functioned as "metabolic sinks" by accepting tumor-derived damaged mitochondria, thereby promoting DTP survival. Treatment with fasudil, a Rho kinase inhibitor, effectively blocked mitochondrial transfer and restored sensitivity to the EGFR-TKI osimertinib in vivo. Together, this work reveals targetable stromal-tumor crosstalk that sustains DTP populations, proposing a combination therapy for overcoming EGFR-TKI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b95827d896bc8971d8b880f2e978477fdef6355" target='_blank'>
              Transfer of Damaged Mitochondria from Cancer Cells to Cancer-Associated Fibroblasts Promotes Tyrosine Kinase Inhibitor Tolerance in EGFR-Mutant Lung Cancer.
              </a>
            </td>
          <td>
            Tongyan Liu, Shiyi Huang, Zhitong Li, Jianyu Li, Qinhong Sun, Qinglin Wang, Ziyang Shen, Zhijun Xia, Fanchen Meng, Youtao Xu, Wenjia Xia, Jing Tan, Rong Yin
          </td>
          <td>2025-12-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="During a lifetime, normal cells accumulate thousands of changes in their genome sequence. These changes, termed somatic mutations, have mostly been studied in the context of cancer, but their presence in normal tissues is ubiquitous and widespread. Somatic mutation accompanies the aging process and is influenced by genetic and environmental factors. Differently from gene expression or imaging data, which fluctuate over time, somatic variants are non-reversible marks in the genome and accumulate over time. This property can be exploited to track the history of a cell, from conception to old age, providing information that cannot be acquired via classical histological tissue inspection nor other types of omics data. Mutations can track embryonic development, measure how clones compete in a tissue over time, or report the mutational processes active in cells and tissues throughout life. We discuss selected examples and emphasize how somatic mutation analysis can enable expanding applications at the service of physiology and cell biology, as well as a deeper understanding of the aging process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e1754e49deb7eef5ac28d0fb31ba15c272ffd93" target='_blank'>
              Reconstructing the lifelong history of cells and tissues via somatic mutation analysis
              </a>
            </td>
          <td>
            Sipontina Faienza, J. P. Margaria, Irene Franco
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA damage and repair are central to the onset of cancer, aging, and aging-related diseases. Rare genetic defects in the nucleotide excision repair pathway, such as those causing the cancer-prone disorder xeroderma pigmentosum (XP) or the progeroid condition Cockayne syndrome, highlight the dramatic consequences of unrepaired DNA lesions. In this issue of the JCI, two related papers from Ogi and coworkers — Fassihi et al. and Nakazawa et al. — describe a new XP clinical entity, XP-J, linked to a pathogenic variant in the p52 subunit of the transcription-repair complex TFIIH. The studies’ characterization of XP-J and the p52ΔC variant opened unexpected possibilities to ameliorate the molecular defect in another subunit of TFIIH that causes a different, more severe repair syndrome: trichothiodystrophy. This commentary provides a broader historical, medical, and molecular context for the intricate genotype-phenotype relationship between compromised repair and its clinical consequences and discusses next steps for the advances reported.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47309874361d974d2143fc65fd785375287e5c39" target='_blank'>
              Expanding the landscape of nucleotide excision repair disorders: from discovery to therapy
              </a>
            </td>
          <td>
            A. Theil, J. H. J. Hoeijmakers
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: Genomic integrity is crucial to the cellular life cycle, which involves a tightly regulated process where cells progress through specific phases to ensure that fully replicated, undamaged DNA is inherited by daughter cells. Any dysfunction in this process or unrepaired DNA damage leads to cell cycle arrest and programmed cell death. Cancer cells are known to exploit these mechanisms to continue dividing. Usually, DNA damage arrests replication, allowing the DNA Damage Response (DDR) pathway to activate, which repairs the DNA or bypasses the damage to support cell survival and preserve genome integrity. For DNA damage bypass or translesion synthesis (TLS), a group of low-fidelity polymerases perform error-prone DNA synthesis opposite damaged bases, where REV1 functions as the main scaffolding protein. Previously, we reported non-TLS functions of REV1, including its role in triggering DNA damage-dependent specific DNA metabolic processes. Methods and Results: In this study, we demonstrate that REV1 plays a significant role in cell cycle progression and that its loss causes arrest at the G2/M phase in flow cytometry analysis. This unexpected phenotype includes dysregulation of G2/M regulators, such as Cyclin B1 and tubulins, in REV1-deficient cells compared to controls, as quantified by Western blot. Additionally, phosphorylation of histone H3 at serine 28 was significantly reduced in these REV1-deficient cells. These G2/M arrest features were even more pronounced in REV1-deficient cells treated with the tubulin inhibitor colchicine. Conclusions: Overall, this study reveals a previously unrecognized link between REV1 TLS polymerase inhibition and the G2/M cell cycle arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/349f4407e91b1d8708aa93b1aa069a7dcc5ae83b" target='_blank'>
              REV1 Loss Triggers a G2/M Cell-Cycle Arrest Through Dysregulation of Mitotic Regulators
              </a>
            </td>
          <td>
            Brailey Buntin, Madison Guyette, Vihit Gupta, Kanayo Ikeh, Sombodhi Bhattacharya, Erica N. Lamkin, Allison Lafuze, Roxana del Rio-Guerra, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-12-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bladder cancer is a prevalent and lethal malignancy worldwide, with treatment options often limited by the development of chemoresistance, especially to cisplatin-based regimens. In this study, we identified ubiquitin-specific protease 54 (USP54) as a novel regulator of bladder cancer cell proliferation, stemness, and cisplatin resistance. USP54 expression was consistently downregulated in both clinical bladder cancer specimens and cell lines. Functional experiments demonstrated that USP54 overexpression inhibited cancer cell growth, attenuated CSC-like properties, and restored cisplatin sensitivity in resistant cells by promoting drug-induced apoptosis. Mechanistically, USP54 directly interacted with SIRT6 and facilitated its deubiquitination, leading to increased SIRT6 protein stability. This post-translational stabilization of SIRT6 was shown to mediate the tumor-suppressive functions of USP54 in bladder cancer progression and chemoresistance. Together, these results uncover the USP54-SIRT6 axis as a previously unrecognized regulatory pathway controlling stemness and cisplatin response in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94fe31451f9b92eb7a9c410e61c50d70da9b9403" target='_blank'>
              USP54 suppresses bladder cancer proliferation, stemness, and cisplatin resistance via deubiquitination-mediated stabilization of SIRT6.
              </a>
            </td>
          <td>
            Yi-nan Han, Shengxing Wang, Rurui Xia, Bang-fen Zhou, Shigong Huang, Wengui Xie, Shijie Guo
          </td>
          <td>2025-11-26</td>
          <td>Journal of chemotherapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Antibiotic resistance is a widespread problem that reduces the effectiveness of antibiotics, leading the rise of multidrug resistance (MDR). Efflux pump, biofilm formation and plasmid based gene transfer together form the basis of co-adaptive mechanisms providing fleeting tolerance resistant overtime. Genes encoding efflux systems are alternative efflux-pumping systems that remove some antimicrobial agents, so that over extremely low intracellular concentrations are required for the bacterium to survive in the biofilm matrix.These, in turn, contribute to the structural and functional coherence of biofilms, which serve as protected niches favoring horizontal gene transfer and the propagation of plasmid-mediated resistance genes. In this review, we combine recent data from molecular and omics-based studies to demonstrate how these systems are co-regulated to control resistance in bacteria, with an emphasis on cross-talk between regulatory pathways including quorum-sensing, stress-response signalling and global transcriptional regulators. Insights into this interplay can pave the way for innovative therapeutic strategies, such as efflux pump inhibitors, biofilm dispersal agents and plasmid-curing strategies. Comprehensive knowledge of these interrelated processes may influence the development of future antimicrobial agents and their use, and may reduce the emergence of pan-resistant species of bacterial pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f46ff4958d5baeef3aee465cd5d3e79a160508" target='_blank'>
              An integrated view of bacterial resistance in the antibiotic era: efflux pumps biofilms and plasmids
              </a>
            </td>
          <td>
            Zahraa Mohammed
          </td>
          <td>2025-12-31</td>
          <td>Journal of Biomedicine and Biochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Methyl-donor nutrients, including folate, vitamin B12, vitamin B6, choline, betaine, and methionine, play indispensable roles in one-carbon metabolism and govern key processes such as DNA methylation, nucleotide synthesis, and genomic maintenance. Yet despite decades of research, their relationship with cancer remains paradoxical and frequently misunderstood. Much of the confusion arises from an overreliance on epidemiological studies that use cancer incidence as a late-stage endpoint, thereby obscuring how the biological actions of methyl donors differ fundamentally across the continuum from precancerous lesions to established tumors. By synthesizing evidence from mechanistic studies, precancerous lesion research, and early-stage carcinogenic models, this review suggests that adequate methyl-donor availability may be protective during the earliest phases of cancer development. However, these same nutrients may later become substrates hijacked by neoplastic cells to fuel rapid proliferation, maintain oncogenic methylation programs, and enhance tumor progression in established malignancies and high-risk populations. Therefore, this review proposes a reframing that methyl donors may not be evaluated merely as protective or harmful, but rather as context-dependent modifiers whose influence is shaped by timing, metabolic status, and the underlying biology of the target tissue. Such a shift is promising for advancing precision nutrition and the prevention or targeted suppression of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf401262f6b61b60e458c86d532862cdfdc9412" target='_blank'>
              The Double-Edged Nature of Methyl Donors in Cancer Development from Prevention to Progression
              </a>
            </td>
          <td>
            Da Pan, Shaokang Wang, Guiju Sun
          </td>
          <td>2025-12-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetics has emerged as a transformative field in understanding the molecular basis of evolutionary adaptations in animals. Unlike genetic mutations that alter DNA sequences, epigenetic mechanisms such as DNA methylation, histone modification, non-coding RNAs, and chromatin remodelling regulate gene expression without changing the underlying genome. These modifications are responsive to environmental signals and can be stably inherited across cell divisions and, in some cases, across generations. This review synthesises current research on the role of epigenetics in shaping evolutionary outcomes in zoology, emphasising its impact on phenotypic plasticity, stress responses, development, and behaviour across invertebrates and vertebrates. Epigenetic regulation plays a critical  role in processes such as caste differentiation in insects, temperature-dependent sex determination in reptiles, and stress adaptation in mammals. The integration of epigenetics into the evolutionary framework through concepts like soft inheritance, niche construction, and extended evolutionary synthesis offers a more dynamic and flexible understanding of   how organisms adapt to environmental changes. Advances in high-throughput sequencing, CRISPR-based epigenome editing, and single-cell epigenomics have greatly expanded our ability to map, manipulate, and interpret epigenetic landscapes across species. Despite these advances, challenges remain in distinguishing causation from correlation, assessing the stability of epigenetic marks across generations, and integrating multi-omic data in natural populations.  Ethical considerations also arise when applying epigenetic knowledge in conservation and animal breeding. This review identifies critical gaps in long-term and field-based studies and advocates for interdisciplinary approaches that unite molecular biology, ecology, and evolutionary theory. As the field matures, epigenetics promises to deepen our understanding of biological complexity and provide new perspectives on the mechanisms driving evolutionary change in animals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97cc13f9d04e0cb3dafa0eb95e818fa5f3850c69" target='_blank'>
              The Epigenetic Dimension of Evolution: Integrating Molecular and Ecological Perspectives in Zoology
              </a>
            </td>
          <td>
            Rahul Jagannath Patil, G. S. Kumar, Shruti R Hiremath, Farindra Tiwari, V. Sreedevi, Thangam Sathish. A, Dinesh Digambar Wankhede
          </td>
          <td>2025-12-16</td>
          <td>UTTAR PRADESH JOURNAL OF ZOOLOGY</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c2666cb6814b50c610dbbea4b7e0fe4f5eecb71" target='_blank'>
              Rapamycin Differentially Impacts Germline Stem Cell Quiescence Across Diverse Genetic Backgrounds of Drosophila Melanogaster
              </a>
            </td>
          <td>
            Sahiti Peddibhotla, Miriam Gonzaga, Tricia Zhang, Yasha Goel, Jun Sun, Benjamin R. Harrison, Daniel E. L. Promislow, H. Ruohola-Baker
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cell states and biological processes are defined by their epigenetic profiles, distinctive composites of DNA‐ and histone‐based chromatin components. However, the specific histone posttranslational modifications that distinguish cellular senescence and the impact of their distribution on transcription, especially with regard to gene length, have not been fully elucidated. Here, we show that promoter loss of symmetric dimethylated H4R3 (H4R3me(2s)) and spreading of trimethylated H3K79 (H3K79me3) across gene bodies are functional features of replicative senescence associated with gene upregulation. We report that highly upregulated genes in replicative senescence exhibit enrichment of H3K79me3 and, in contrast to the characteristic trend of aging cells and tissues, are substantially longer than those that are significantly downregulated. Furthermore, by assessing all expressed genes, we demonstrate that gene body accumulation of H3K79me3 during the transition to replicative senescence constitutes a broader phenomenon that is positively correlated with gene length and expression level genome‐wide. Consistently, pharmacological inhibition of H3K79me3 deposition attenuates gene upregulation in replicative senescence. We also document a striking increase in the levels of H3K79me3 as well as a robust H4R3me(2s) to asymmetric dimethylated H4R3 (H4R3me(2as)) epigenetic switch that manifest globally in late‐passage cells, suggesting that these histone modifications might represent novel molecular biomarkers of replicative senescence. Finally, we implicate the associated epigenetic regulators, including DOT1L, PRMT1, PRMT5, and JMJD6, as modifiers of cellular lifespan, potentially disclosing unappreciated therapeutic targets for interventions in normal and pathological aging. Collectively, our findings provide novel insights into the histone code that mediates altered transcriptional regulation in replicative senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d5d203d4f53578d323266e38c25fe56a1f8279c" target='_blank'>
              A Histone Code Functionally Linked to Replicative Senescence
              </a>
            </td>
          <td>
            Thomas Suter, Meyer J. Friedman, Cagdas Tazearslan, Amir Gamliel, Daria Merkurjev, K. Ohgi, Zhongjun Zhou, Michael G Rosenfeld, Yousin Suh
          </td>
          <td>2026-01-01</td>
          <td>Aging Cell</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Conventional cancer therapies, including radiation therapy and chemotherapy, rely on inflicting DNA damage, yet they inevitably affect normal cells, leading to severe adverse effects. The advent of precision chemotherapy exploiting tumor-specific DNA repair defects has validated the effectiveness of this approach. The first successful example is PARP inhibitors, which selectively kill homologous recombination (HR) defective cancers, such as familial breast cancer possessing HR deficiency due to BRCA gene mutations. However, the broader landscape of DNA maintenance-including DNA replication, repair, and checkpoint pathways-harbors numerous mutations in tumors that remain untargeted. Here, we propose repurposing chain-terminating nucleoside analogs (CTNAs) to target such cancers' vulnerabilities. CTNAs, long utilized as anti-cancers and anti-viral drugs, inhibit replication and thereby suppress growth, but their activity has never been systematically aligned with specific cancer mutations associated with DNA maintenance defects. Based on our recent studies, we demonstrate that CTNAs elicit synthetic lethality in cells deficient for distinct DNA maintenance systems, amplifying replication stress, leading to cell death. We highlight the spectrum of CTNA-induced lesions and repair pathways required for cellular tolerance. This framework presents a versatile "repair-defect-guided" chemotherapy that expands the clinical utility of CTNAs and improves therapeutic effect by reducing side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e63be07db0a569b7816efc4ad2f977abe094978" target='_blank'>
              Targeting Genome Maintenance Defects of Cancers Using Chain-Terminating Nucleoside Analogs.
              </a>
            </td>
          <td>
            Ryotaro Kawasumi, Rubaiat E Tabassum, Kouji Hirota
          </td>
          <td>2025-12-10</td>
          <td>Cancer science</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations—including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells—as well as non-cellular elements such as the extracellular matrix and extracellular vesicles. Objective A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms. Discussion and conclusion Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bf0d027edeb1c0ea3dba2055e8f587e64fbdfe5" target='_blank'>
              Antiplasmid systems: a novel strategy to combat antibiotic resistance.
              </a>
            </td>
          <td>
            Mohd Asim, Syed Ahmed Rizvi, Qazi Mohd Rizwanul Haq
          </td>
          <td>2025-12-19</td>
          <td>The Journal of antimicrobial chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction. Bacteria are capable of integrating a portion of the genetic material of an invading infectious agent as a spacer into the CRISPR array. This process is called CRISPR adaptation and is necessary for bacteria to adequately respond to the same infectious agent in the future. The molecular mechanisms of CRISPR adaptation remain unclear, particularly the role of extra-CRISPR-Cas components in bacteria during prespacer generation and spacer acquisition. In this study, we developed an approach to identifying nucleases responsible for the formation of prespacer 3’-ends in vivo. Materials and methods. We generated a genetic construct based on the pBR322 plasmid vector with gRNAs targeting the genes encoding cellular exonucleases with 3’-5’ activity in E. coli. This genetic construct and the pdCas9 plasmid were transformed into KD403 cells. The level of primed CRISPR adaptation was assessed using PCR and sequencing. The structure of the 3' ends of the prespacers was assessed using the FragSeq short-fragment sequencing method using the Accel-NGS® 1S Plus DNA Library Kit, Swift biosciences (xGen™ ssDNA & Low-Input DNA Library Preparation Kit, IDT). Results. In the KD403 ΔrecB strain with suppressed nucleases, a reduction in primed adaptation was detected, but sequencing revealed no disruption of the 3’ ends of the spacer precursors. Conclusion. Our results contribute to the understanding of the molecular mechanisms of CRISPR adaptation, which may aid in the development of CRISPR-based approaches in medicine and agriculture in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c67f3795fd0e777318561920146c27024f0382d" target='_blank'>
              Studying the mechanisms of prespacer 3’-end formation during CRISPR adaptation via simultaneous gene repression using dCas9
              </a>
            </td>
          <td>
            I.A. Balazs, O. Musharova, Yunna Petrusenko, S. Vinogradova, K. Severinov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma.
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah B Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ebbac96e406721d5a187e13aac4ead4e46cec50" target='_blank'>
              Restoration of PKM1 improves functional maturation of human stem-cell derived-β cell by regulating PEP metabolism.
              </a>
            </td>
          <td>
            Haopeng Lin, Xin Liu, Feng Zhang, Deqi Chen, Xiaoxiao Xie, Qifei Dong, Jiawei Yan, Jiaxiang Yin, Zirong Bi, Zhihua Dou, Yingling Jiang, Kuo Jiang, Tongran Zhang, Peng Xue, Wei Peng, Lihua Chen, Tao Xu, Zonghong Li, Yanying Guo, Huisheng Liu
          </td>
          <td>2025-12-21</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Widespread insecticide use has led to the evolution of resistance in numerous insect pests. Sublethal exposures play a significant, yet often underestimated, role in driving adaptive responses by inducing a wide range of physiological and behavioral alterations. This review integrates current insights into how sublethal and low-dose insecticide exposures induce epigenetic modifications in insects and examines the functional implications of these changes. Emerging evidence suggests that such exposures induce a spectrum of epigenetic alterations, including shifts in DNA methylation patterns; for example, foundational work in Myzus persicae Sulzer demonstrated that demethylating an amplified esterase gene was directly linked to lost insecticide resistance. Exposures also trigger various post-translational histone modifications, as seen in pollinators like Apis mellifera Linnaeus (Hymenoptera: Apidae), where histone acetylation changes following neonicotinoid exposure modulate the expression of critical detoxification and immune genes. Moreover, insecticides influence the regulatory activity of non-coding RNAs (ncRNAs), with specific microRNAs shown to repress the translation of key metabolic resistance genes. These molecular changes serve as a dynamic regulatory layer modulating the expression of genes involved in detoxification pathways, target site sensitivity, and broader stress resilience, thereby contributing to hormesis and the emergence of insecticide tolerance. Furthermore, these insecticide-induced epigenetic imprints can be heritable, providing a potent mechanism for transgenerational inheritance and highlighting the potential for epigenetic priming to accelerate resistance within pest populations. A deeper understanding of these non-Mendelian inheritance processes is thus indispensable for developing sustainable insecticide resistance management (IRM) strategies and for optimizing ecological risk assessments that more accurately reflect the subtle yet consequential effects on both target and non-target insect species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/794445df8f23fd6f505e654033aab2e7c5320ce6" target='_blank'>
              Epigenetic Effects of Sublethal Insecticide Exposure in Insects
              </a>
            </td>
          <td>
            Hira Aslam, M. Yalçın
          </td>
          <td>2025-12-28</td>
          <td>Turkish Journal of Agriculture - Food Science and Technology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The convergence of artificial intelligence and synthetic biology is innovating and accelerating the design of novel viral genomes, expanding both therapeutic opportunities and dual-use risk. This review articulates a countermeasure strategy for emerging and engineered viruses leveraging the programmable CRISPR modality. Building on mounting in vitro and in vivo evidence that Cas9 degrades DNA viruses (e.g., Orthopoxviruses, HSV-1, ASFV), while Cas13 targets RNA viral genomes (e.g., Influenza A, Dengue, RSV), both leading to reduced viremia, diminished disease burden, and alleviated symptoms. Here, we outline a rapid-response pipeline to position CRISPR-based countermeasures in translational and pandemic-response frameworks, linking real-time sequencing to AI-assisted gRNA selection and multiplexed cassette design to achieve viral targeting efficacy. To minimize resistance and off-target risk, we emphasize multi-gRNA cocktails, continuous genomic surveillance, and adaptive gRNA rotation. We also propose governance mechanisms, such as pre-cleared gRNA repositories, transparent design logs, standardized off-target/safety screening, and alignment with evolving nucleic-acid-synthesis screening frameworks to enable emergency deployment while preserving security. Furthermore, compressing the time from sequence to treatment and complementary to vaccines and small-molecule antivirals, CRISPR represents a technologically agile and strategically essential capability to combat both natural outbreaks and AI-enabled biothreats. Collectively, programmable CRISPR antivirals represent an auditable, rapidly adaptable foundation for next-generation biodefense preparedness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bcf647c14a396b63e1f9c3b5f4e131bd7a256c7" target='_blank'>
              CRISPR Treatments for AI-Designed Synthetic Viruses: Rapid Programmable Countermeasures for Emerging and Engineered Viruses
              </a>
            </td>
          <td>
            D. Gladue, Alison O’Mahony
          </td>
          <td>2025-12-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 A crucial adaptability trait of melanoma cells that persist under all selection pressures, including therapies, is their ability to efficiently switch from proliferation to quiescence. With the goal of improving therapeutic strategies for preventing melanoma recurrence, herein we describe an approach to modeling this trait in cell culture. Body imposes a variety of selection pressures and bottlenecks ranging from different components of immunity to metabolic challenges to enforce quiescence and influence cancer evolution. The rationale for the choice of model as cell culture is that it is ideal for modeling the realistic depth of quiescence and for evaluating therapeutic strategies that may inhibit highly abnormal but adaptable cancer cells. As it is not possible to replicate all such bottlenecks in cell culture, our approach is to apply as severe a bottleneck as possible, thus eliminating all proliferative (non-adaptable) cells, leaving behind rare cells surviving in deep quiescence. We hypothesize that if the quiescent cell state being modeled can survive a severe bottleneck, it can also survive other bottlenecks of similar or lower severity. A lack of glutamine proved to be a strong and prolonged selection pressure for the highly metastatic human melanoma cell line A375SM, killing more than 99% of cells and selecting rare cells based on their ability to survive in deep quiescence. After 4 weeks, cells gradually exited quiescence and proliferated indefinitely. Furthermore, we modeled even deeper quiescence lasting longer than 4 weeks by increasing the severity of the selection pressure using dialyzed serum in medium. The cells selected in this manner were much more resistant to paclitaxel than was their parental cell line. We obtained similar results with the highly metastatic mouse melanoma cell line B16-BL6. We conclude that our approach is suitable for modeling abnormal deep quiescence in melanoma that drives the disease, including recurrence and metastasis. The main strength of the approach is that, besides modeling genetic alterations, it incorporates suitable selection pressures and the time element that are important in cancer evolution. Although there is a limitation of lack of body-like microenvironments in cell culture, we believe that abnormal quiescent cancer cell state being modeled here is capable of overcoming restraints imposed by the body. We will also discuss our published and new observations with rare cells (0.01% in population) surviving in quiescence for several weeks in aggressive triple-negative breast cancer cell lines and their functional characteristics. Our results obtained with adaptable cells from SUM149 TNBC cell line, modeled as above for melanoma cells, suggest that they are stem-like quiescent cells. We will also discuss how we choose and evaluate therapies that would likely inhibit quiescent and slow-growing cancer cells prior to recurrence. This approach may significantly improve how we investigate deep intrinsic resistance and strategies to overcome it.



 Balraj Singh, Vanessa N. Sarli, Nikil Erry, Anthony Lucci. A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4de6eb92fbb6a26e62be45c5389b5a53529d5" target='_blank'>
              Abstract A010: A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis
              </a>
            </td>
          <td>
            Balraj Singh, V. Sarli, Nikil Erry, Anthony Lucci
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that influences the regulation of gene expression, genome stability, and plant adaptation to the environment. In plants, the most common forms of methylation occur in cytosines (5mC) and, more recently, also in adenines (6mA), both playing specific and complementary roles in the control of genes, transposable elements, and plant development. The mechanisms that control the addition, removal, and reading of these marks include methyltransferase proteins, glycosylases, and methylation recognition domains, which interact with transcriptional factors and histone modifiers. Methylation is involved in various biological processes such as cell differentiation, flowering, embryogenesis, seed dormancy, and responses to biotic and abiotic stresses. The discovery of heritable epialleles and their application in plant breeding opens new possibilities for selection based on epigenetic traits. Epigenetic editing with CRISPR/Cas9 has emerged as a tool for inducing targeted modifications without altering the DNA sequence, offering alternatives for non-transgenic plant biotechnology. Furthermore, methylation contributes to the phenomenon of heterosis, revealing a functional role in hybrid superiority. However, challenges persist regarding the stability of these marks and the legal regulation of epigenetically edited cultivars. This work reviews recent advances, highlighting the integration of plant epigenetics with genetic, precision editing, and adaptive responses in cultivated plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0115e8ca9083e3fba7bd10aaa44af2c0bf0d7bc" target='_blank'>
              DNA methylation: regulation of gene expression and applications in plant breeding
              </a>
            </td>
          <td>
            Helio Souza Junior, Adriana Gonela
          </td>
          <td>2025-12-03</td>
          <td>Agronomy Science and Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3677c50969edbedb78929193a3490430e71897" target='_blank'>
              Advances in ovarian cancer: biological insights, therapeutic innovations, and future perspectives.
              </a>
            </td>
          <td>
            Xiao Ma, Linzhuo Qu, Qunyin Wu, Qingqing Yu, Zhenhua Lin, Yang Yang, Xianchun Zhou
          </td>
          <td>2025-12-07</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer.
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2702367051da70ddb6fa2941183d106513bc" target='_blank'>
              The expanding roles of homologous recombination proteins in genome stability.
              </a>
            </td>
          <td>
            Lorenzo Sassi, Andrea Martinez Marroquin, Salli Waked, Alessandra Ardizzoia, Vincenzo Costanzo
          </td>
          <td>2026-01-03</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Urothelial carcinoma of the bladder (UBC) is a heterogeneous disease from the clinicopathologic and molecular standpoint and presents clinical management problems for its range in progression, frequent recurrence and complicated genetic/epigenetic architecture. Not only traditional genetic changes, but also epigenetic dysregulation (e.g., alterations of chromatin remodeling, histone modification and DNA methylation) has been increasingly recognized to play a pivotal role in the process of tumor initiation, progression, metastasis and resistance to therapy. It is a dissapointing result that mutations of the chromatin remodeler such as ARID1A, KMT2C, KMT2D, and histone modifying enzymes (EP300, CREBBP and EZH2) not only cause loss-of-nucleosome positioning in addition lead to lack of enhancer function and perturbation ofT program contributing to shaping up intratumoral heterogeneity as well as transcriptional plasticity resulting in failure in immunoevasion. These changes sculpt the tumor microenvironment and impact the response to chemotherapy, immunotherapy, and targeted therapy.Epigenetic biomarkers based on mutation profiles, chromatin accessibility and DNA methylation signatures present new non-invasive strategies that can be followed for early detection, monitoring of disease and stratification of patients usage; which are all available in urine or plasma. Therapeutically, inhibitors of EZH2, HDACs, BET proteins, DNMTs and certain chromatin remodeling factors are currently being heavily investigated in preclinical studies or clinical trials as monotherapies or rational combinations with ICIs, chemotherapy or targeted therapies.Advances in tech nologies, such as single- and spatial genomics combined with AI-facilitated multi-omics integration make it possible to realize high-resolution maps of tumor heterogeneity, enhancer d namics, and epigenetic vulnerabilities. This review aims to present the current data on UBC epigenetic modifications and their mechanistic, biomarker and therapeutic implications. By combining mechanistic understanding with translational perspectives, we outline future directions to move precision oncology forward, optimize our treatment approach and improve clinical outcomes for UBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c58b19d1ac81843818571ae3d302c08f8f24348" target='_blank'>
              Epigenetic Alterations and Chromatin Landscape Remodeling in Urothelial Bladder Cancer: Molecular Drivers, Biomarker Potential, and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Amr Abd-Elraheem Abdo, Ahmed Mohamed Kamel
          </td>
          <td>2025-12-10</td>
          <td>Annals of urologic oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over the past decade, non-viral DNA template delivery has been used with engineered nucleases to target single-stranded DNA sequences in hematopoietic stem and progenitor cells. While effective for gene therapy, this method is limited to short DNA donor templates, restricting its applications to gene corrections. To expand its scope, we developed an editing process using kilobase-long circular single-stranded DNA donor templates and TALEN technology. Our results show that the CssDNA editing process achieves high gene insertion frequency in HSPCs. Compared to AAV-edited HSPCs, CssDNA-edited HSPCs show a higher propensity to engraft and maintain gene edits in a female NCG murine model. This positive outcome is partly due to higher levels of primitive edited HSPCs, a more quiescent metabolic state, and elevated expression of bone marrow niche adhesion markers. Our findings highlight the strong potential of CssDNA as a universal, scalable and efficient non-viral DNA template for gene therapy applications. Letort et al. demonstrate that Circular single-stranded DNA enables efficient and precise gene insertion in hematopoietic stem cells, outperforming other DNA delivery formats and supporting progress toward scalable next-generation gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8334d5fc2c8d3325ef3a32ecf5b25cddb9e9e5c9" target='_blank'>
              Circular single stranded DNA potentiates non-viral gene insertion in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, Diane Le Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54d931016d849a45a82192eedf06a276e9b6b73" target='_blank'>
              Rescuing the Function of Missense-Mutated Tumor Suppressor VHL using Stabilizing Small Molecules
              </a>
            </td>
          <td>
            Mariam Ahmed Fouad, Christopher S. Parry, Sven A. Miller, Shipra Malhotra, Glenn A. Doyle, Ezra Shimabenga, Mashhura Nurilloeva, Elena Bondarenko, Grigorii V. Andrianov, Wayne Childers, Benjamin E. Blass, Petr Makhov, Johnathan R. Whetstine, M. Clapper, E. Golemis, John Karanicolas
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84c3793ebdb7f2226cb0fc2c197d56bcba3c5730" target='_blank'>
              Philadelphia chromosome-positive acute lymphoblastic leukemia: exploring microRNA-based strategies to improve outcomes
              </a>
            </td>
          <td>
            Setare Kheyrandish, Niloofar Pilehvari, Amirhossein Rastgar, Mehdi Allah bakhshian-Farsani, Bahram Chahardouli, Zahra Nazari, S. Rostami, Saeed Mohammadi, Ahmad Gharehbaghian
          </td>
          <td>2025-11-18</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6139c8ae9b321fbea21e5f26cd01d386682251c" target='_blank'>
              Familial colorectal cancer: search for novel predisposition genes.
              </a>
            </td>
          <td>
            A. Försti, Beiping Miao, Abhishek Kumar, Dagmara Dymerska Zaremba, Magdalena Marciniak, J. Lubiński, Kari Hemminki
          </td>
          <td>2025-12-30</td>
          <td>Human genomics</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Both metabolic dysregulation and the immunosuppressive tumor microenvironment of pancreatic ductal adenocarcinoma (PDAC) contribute to the recalcitrance of this lethal disease to treatment. Accordingly, we aimed to identify and characterize a target that elicits an anti-cancer response through both disrupting cancer cell redox homeostasis and increasing the immunogenicity of PDAC. First, mitochondrial metabolic dependencies in PDAC were identified by using a CRISPR-Cas9 screening system with a custom sgRNA library. Functional validation analyses revealed GFER, a mitochondrial FAD-dependent sulfhydryl oxidase, as an essential regulator of tumor growth. In vitro and in vivo methodologies demonstrated that GFER depletion perturbed redox homeostasis and stimulated tumor immunogenicity, including sensitization to immune checkpoint blockade. In patient-derived xenograft models of PDAC, the growth-inhibitory response induced by GFER depletion was mediated by an altered oxidative balance that released damaged mitochondrial DNA into the cytoplasm of tumor cells, leading to the activation of the cGAS-STING pathway and expression of type I interferons. This effect was recapitulated in a mouse immunocompetent syngeneic PDAC model, where GFER depletion suppressed tumor growth and promoted T cell infiltration to enhance tumor-killing effects. Consequently, GFER depletion significantly increased the anti-tumor efficacy of immune checkpoint blockade. Overall, these findings identify GFER as a critical node for both mitochondrial redox homeostasis and immunomodulation in PDAC and reveal a therapeutic opportunity for sensitizing PDAC to immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949aadb7dd1799a37a0edb47d973f3dddb1cc24a" target='_blank'>
              GFER Represents a Target for Dual Disruption of Redox Homeostasis and Reactivation of the Immune Response in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Ziheng Chen, I. Ho, Jintan Liu, Melinda Soeung, Er-Yen Yen, Sara Sainani, Charles A Dyke, Rutvi Shah, Hyuk Jean Kwon, Guangchun Han, Xinmiao Yan, Shaojun Zhang, Sanjana Srinivasan, Johnathon L. Rose, F. Citron, Shan Jiang, Qing Chang, Ningping Feng, G. Genovese, Sisi Gao, W. Yao, Linghua Wang, Huamin Wang, H. Ying, A. Viale, Giulio Draetta
          </td>
          <td>2025-12-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="CRISPR-Cas systems provide bacteria with adaptive immunity against bacteriophages and mobile genetic elements, driving an evolutionary arms race in which phages deploy anti-CRISPR (Acr) proteins. Acr proteins are often co-encoded in operons with anti-CRISPR-associated (Aca) proteins, which coordinate the regulation of acr gene expression. Here, we reveal the molecular basis of DNA binding that mediates transcriptional repression by two distinct Aca family members: Aca7 and Aca11. Crystal structures of Aca7 and Aca11 highlight conserved helix-turn-helix (HTH) motifs within α-helix bundles, providing a universal DNA-binding platform. Aca7 forms a symmetrical dimer to recognize a 19-bp inverted repeat (IR) within the acrIF11-aca7 operon. Strikingly, Aca11 binds 22-bp IRs in two distinct promoters, suggesting that Aca proteins can control multiple target operons. Mutagenesis and electrophoretic mobility shift assays (EMSAs) confirm that dimerization and sequence-specific IR recognition are essential for DNA binding. Despite mechanistic similarities, these and other Aca proteins exhibit notable differences. Structural comparisons across Aca families reveal that while monomer structures are generally similar with conserved HTH motifs, the structures of their dimeric functional units vary significantly. These structural differences might be essential for Aca proteins to bind to various promoters and regulate the expression of different Acr proteins. The crystal structures of two distinct phage anti-CRISPR (Acr) proteins bound to their cognate DNA binding sites reveal important structural differences that likely govern transcriptional regulation by Acr proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ec734bb7c57d7ce2029cf319816b1aa27a505e" target='_blank'>
              Regulation of anti-CRISPR operons by structurally distinct families of Aca proteins
              </a>
            </td>
          <td>
            So Yeon Lee, N. Birkholz, Jun Hyuck Lee, Peter C. Fineran, Hyun Ho Park
          </td>
          <td>2025-11-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="INTRODUCTION
Mismatch-repair deficiency (MMRd) is a pan-cancer mechanism resulting in high tumor mutation and microsatellite burden, leading to remarkable therapeutic success using immune checkpoint inhibition (ICI). However, efficacy remains variable. Our goal is to review the existing and emerging literature that provide mechanistic insights on how MMRd diversely impacts the current landscape of cancer therapeutics.


AREAS COVERED
We summarize the biological impact of MMRd, highlight how variable etiologies and different MMRd cancer-types have significant diversity in genomic and immune profiles, how this drives primary and acquired ICI-resistance, and highlight the clinical trial landscape and preclinical efforts to harness these distinct mechanistic vulnerabilities into new therapeutic strategies.


EXPERT OPINION
MMRd is currently recognized a tumor-agnostic biomarker for ICI treatment. However, differences in the MMRd etio-biology impact the immune microenvironment, contributing to significant response heterogeneity. Some patients show dramatic response to neoadjuvant ICI monotherapy and don't even need surgery, while most will need biologically rational combination approaches. Additionally, MMRd-driven genomic instability can make this a 'moving target,' mandating an ongoing effort to track and adapt to MMRd-driven cancer and microenvironment evolution. The next decade will focus on how best to effectively tailor immune-directed therapies for different MMRd cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3e3a37df3f165aafcd442ce28ce82c79d222ae3" target='_blank'>
              Harnessing mismatch repair deficiency for therapeutic targeting in cancers.
              </a>
            </td>
          <td>
            Irem Yenidogan, U. Tabori, Anirban Das
          </td>
          <td>2025-12-19</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>77</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [14, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>