<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-22 09:31:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>56</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The development of drug-resistant cell lines is essential for understanding the mechanisms of drug resistance and identifying strategies to overcome treatment failure in cancer therapy. Resistance models enable preclinical evaluation of novel compounds, repurposed drugs, and combination therapies. To generate resistant cells, parental cancer cell lines are repeatedly exposed to incrementally increasing concentrations of the target drug over several weeks. Cells that survive and proliferate at each stage are selected, expanded, and exposed to higher drug doses. The development of resistance is confirmed by quantifying and comparing the half-maximal inhibitory concentration (IC50) values between parental and resistant cells using cell viability assays and nonlinear regression analysis. Significantly increased IC50 values indicate successful adaptation to drug pressure and the development of resistance. These drug-resistant cell lines are available for comprehensive analysis, such as microarray and single-cell sequencing, as well as various in vitro or in vivo experiments. These models provide valuable tools for investigating potential therapeutic strategies to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb7439fafb8c53daf4f72d4753cdd3f7c5b1b9f" target='_blank'>
              Development of Drug-resistant Cell Lines for Experimental Procedures.
              </a>
            </td>
          <td>
            S. Kadomoto, G. Shelley, Atsushi Mizokami, Evan T. Keller
          </td>
          <td>2025-08-12</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="KRAS is mutationally activated in 45–50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRASG12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, J. Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="
 Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species.

 Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures.

 Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign.

 Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Platinum-based chemotherapy resistance remains a critical barrier in colorectal cancer (CRC) treatment. In this study, cytogenetic karyotyping was combined with transcriptomic profiling (RNA-seq) to elucidate resistance mechanisms by comparing two CRC cell lines: oxaliplatin-sensitive HCT116 and its resistant derivative HCT116 oxpl-R. Karyotyping unveiled tetraploidization and extensive genomic rearrangements in resistant cells, accompanied by prominent transcriptomic changes: 1807 differentially expressed genes (1216 upregulated and 519 downregulated). Pathway enrichment highlighted altered redox homeostasis and metabolic adaptation. Specifically, HCT116 oxpl-R cells exhibited elevated reactive oxygen species (ROS) production and enhanced energy metabolism. These findings establish a direct association between structural genomic alterations, transcriptional rewiring, and functional phenotypes in platinum resistance, providing a framework for targeting metabolic vulnerabilities in refractory CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1c5397ea3bf691ad1c8a6e4e4cf327eb00a1d" target='_blank'>
              Transcriptomic and Cytogenetic Analysis of Oxaliplatin-Resistant Colorectal Adenocarcinoma HCT116 Cells to Identify Markers Associated with Platinum Resistance
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, Ksenia Fedotova, Natalija Yartseva, Nikolay Aksenov, M. Igotti
          </td>
          <td>2025-09-11</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a genetically and clinically heterogeneous malignancy marked by poor prognosis and limited therapeutic options, especially in older patients. While conventional treatments such as the “7 + 3” chemotherapy regimen and allogeneic stem cell transplantation remain standard care options, the advent of next-generation sequencing (NGS) has transformed our understanding of AML’s molecular complexity. Among the emerging hallmarks of AML, metabolic reprogramming has gained increasing attention for its role in supporting leukemic cell proliferation, survival, and therapy resistance. Distinct AML subtypes—shaped by specific genetic alterations, including FLT3, NPM1, and IDH mutations—exhibit unique metabolic phenotypes that reflect their underlying molecular landscapes. Notably, FLT3-ITD mutations are associated with enhanced reactive oxygen species (ROS) production and altered energy metabolism, contributing to disease aggressiveness and poor clinical outcomes. This review highlights the interplay between metabolic plasticity and genetic heterogeneity in AML, with a particular focus on FLT3-driven metabolic rewiring. We discuss recent insights into how these metabolic dependencies may be exploited therapeutically, offering a rationale for the development of metabolism-targeted strategies in the treatment of FLT3-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdd39e68513bba88417b983cb0b3114bb567964" target='_blank'>
              Metabolic Signature of FLT3-Mutated AML: Clinical and Therapeutic Implications
              </a>
            </td>
          <td>
            C. Banella, Gianfranco Catalano, Maura Calvani, Eleonora Candi, N. Noguera, S. Travaglini
          </td>
          <td>2025-09-08</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Cancer genomic profiling has revolutionized cancer research and clinical practice by facilitating personalized medicine approaches. Next-generation sequencing technologies have played a pivotal role in expanding the possibilities of cancer genomics, providing a comprehensive view of the genomic landscape of tumors and identifying potential therapeutic targets. In Japan, the approval of several cancer-related gene panel tests has accelerated the implementation of cancer genomic medicine. However, there remains a need for evidence-based selection criteria for the different panel tests available. Among these tests, ACTOnco+® stands out as a comprehensive cancer genomic profiling tool. In this study, we compared 110 samples from 29 different tumor types using FoundationOne® CDx (324 genes for DNA analysis) and ACTOnco+® (440 genes for DNA and 31 genes for RNA analysis). Overall, the mutation profiles between the 2 assays exhibited 82.8% positive agreement in reported sequence alterations in clinically actionable genes, including single nucleotide variants and insertions–deletions. Copy number gains showed concordance of 76.9%, and copy number losses in 66.7%. In the case of KIAA1549-BRAF fusion-positive astrocytoma, the fusion event was not detected by DNA only test, but RNA sequencing identified the rearrangement. The patient exhibited clinical benefit from MEK inhibitor treatment. Tumor mutational burden and microsatellite instability (MSI) demonstrated high concordance across various cancer types. ACTOnco+® identified a total of 329 heterozygous deletions, which were subsequently validated for reliability using FISH analysis. Gene profiling with ACTOnco+® exhibited comparable results to FoundationOne® CDx, thereby contributing to future personalized medicine endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584447fccf78026ffcd29a7adc016edfe5f11d6e" target='_blank'>
              Characterization of a novel comprehensive genomic profiling test with better detection of heterozygous deletions and RNA-based gene fusion analysis
              </a>
            </td>
          <td>
            Ryouta Kakuta, Akira Takemoto, Kousuke Tanimoto, Rika Noji, Ryo Kudo, Yoshihito Kano, Yayoi Ando, Tatsunori Simoi, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-08-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Multiple Myeloma (MM) is a malignant plasma cell dyscrasia that progresses through the consecutive asymptomatic, often undiagnosed, precancerous stages of Monoclonal Gammopathy of Undetermined Significance (MGUS) and Asymptomatic Multiple Myeloma (SMM). MM is characterized by low survival rates, severe complications and drug resistance; therefore, understanding the molecular mechanisms of progression is crucial. This study aims to detect genetic mutations, both germline and somatic, that contribute to disease progression and drive tumorigenesis at the final stage of MM, using samples from patients presenting MGUS or SMM, and newly diagnosed MM patients. Methods: Mutations were identified through a fully computational pipeline, implemented in a Linux and RStudio environment, applied to each patient sequence, obtained through single-cell RNA-sequencing (scRNA-seq), separately. Structural and functional mutation types were identified by stage, along with the affected genes. The analysis included quality control, removal of the Unique Molecular Identifiers (UMIs), trimming, genome mapping and result visualization. Results: The findings revealed frequent germline and somatic mutations, with distinct structural and functional patterns across disease stages. Mutations in key genes were identified, pointing to molecules that may play a central role in carcinogenesis and disease progression. Notable examples include the HLA-A, HLA-B and HLA-C genes, as well as the KIF, EP400 and KDM gene families, with the first four already confirmed. Comparative analysis between the stages highlighted molecular transition events from one stage to another. Emphasis was given to novel genes discovered in newly diagnosed MM patients, that might contribute to the tumorigenesis that takes place. Conclusions: This study contributes to the understanding of the genetic basis of plasma cell dyscrasias and the transition events between the stages, offering insights that could aid in early detection and diagnosis, guide the development of personalized therapeutic strategies, and improve the understanding of mechanisms responsible for resistance to existing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8c2ca600e5f8d213e747b7e35b473639690b95" target='_blank'>
              Genomic Insights into Tumorigenesis in Newly Diagnosed Multiple Myeloma
              </a>
            </td>
          <td>
            Marina Kyriakou, C. Papaloukas
          </td>
          <td>2025-08-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, Mario Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Ujvari, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations.To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation.Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies.This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="KRAS mutations are among the most prevalent oncogenic drivers in non-small cell lung cancer (NSCLC), yet the mechanisms of therapeutic resistance to KRAS inhibitors in these cancers remains poorly understood. Here, we deploy high-throughput CRISPR base editing screens to systematically map resistance mutations to three mechanistically distinct KRAS-targeted therapies, including KRAS-G12C(OFF) inhibitor (adagrasib), RAS(ON) G12C-selective tri-complex inhibitor (RMC-4998), and RAS(ON) multi-selective tri-complex inhibitor (RMC-7977). Using both a saturation Kras tiling approach and cancer-associated mutation library, we identify common and compound-selective second-site resistance mutations in Kras, as well as gain-of-function and loss-of-function variants across cancer-associated genes that rewire signaling networks in a context-dependent manner. Notably, we identify a recurrent missense mutation in capicua (Cic), that promotes resistance to RMC-7977 in vitro and in vivo. Moreover, we show that targeting NFκB signaling in CIC-mutant cells can resensitize them to RAS pathway inhibition and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597a0301e9cbbccb635fb6b4893ce93089ff3713" target='_blank'>
              Genetic mechanisms of resistance to targeted KRAS inhibition
              </a>
            </td>
          <td>
            Bianca J. Diaz, Max Kops, Sara Bernardo, Henri Schmidt, Elizabeth Grankowsky, Adrian Vega, Chen Zhang, Matthew Bott, Maria Skamagki, Aidan C A Tomlinson, Nicole A Vita, Alyna Katti, Mark Labrecque, Ida Aronchik, Mallika Singh, Lukas E. Dow
          </td>
          <td>2025-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chemotherapy is the most widely used and effective treatment method in today's cancer treatment. However, the rise of cancer drug resistance has limited chemotherapy effectiveness. The molecular mechanisms and related genes by which many cancer cells acquire resistance to chemotherapy drugs remain to be identified. Recent advancements of CRISPR-Cas9 genome-editing techniques have drawn rigorous exploration of precision oncology applications. CRISPR-Cas9 technology can knock out, inhibit or activate the expression of specific genes. By observing the phenotype of gene-edited cells, drug-resistant genes in carcinoma cells can be identified. This paper will decipher the molecular basis of CRISPR-Cas9 technology in targeted gene modification and give examples to illustrate its application in the screening of drug resistance genes in three types of human cancers. Additionally, the application limitations and emerging horizons of CRISPR-Cas9 gene modification technique in strategies for overcoming anticancer drug resistance are discussed. The continuous development and optimization of this technology are expected to provide strong support for breaking through the bottlenecks in tumor treatment and pushing precision cancer medicine to new heights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0492def30740e765a8ca58409631a4cc538a87f" target='_blank'>
              CRISPR-Cas9 Gene-Editing Technology for the Screening of Drug Resistance Genes in Tumor Cells
              </a>
            </td>
          <td>
            Zisheng Zeng
          </td>
          <td>2025-09-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51250a57c51180b7973c2f5e6b3bbb56fc3acbc6" target='_blank'>
              Personalizing cancer therapy: the role of pharmacogenetics in overcoming drug resistance and toxicity.
              </a>
            </td>
          <td>
            Hossein Mardani -Nafchi, Hormoz Solatzadeh, Elnaz Hajimaghsoudi, Emad Babakhanzadeh
          </td>
          <td>2025-08-02</td>
          <td>Molecular biology reports</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive blood cancer in which disease initiation and relapse are driven by leukemic cells with stem-like properties, known as leukemic stem cells (LSCs). The LSC compartment is highly heterogenous and this contributes to differences in therapy response. This heterogeneity is determined by genetic and nongenetic factors including somatic mutations, the cell of origin, transcriptional and epigenetic states as well as phenotypic plasticity. While this complicates the identification and eradication of LSCs, it also presents an opportunity to tailor therapeutic strategies to the phenotypic and functional states of LSCs present in a patient, exploiting their specific vulnerabilities. The emergence of single-cell multiomics technologies has transformed our ability to dissect cellular heterogeneity in AML, enabling simultaneous interrogation of genomic, transcriptomic, epigenomic and proteomic layers and providing high-resolution molecular snapshots of individual cells. In this review, we discuss causes and consequences of LSC heterogeneity, highlight advances in single-cell multiomics technologies to resolve it and outline how they can address shortcomings in our understanding of LSC heterogeneity and plasticity to revolutionize diagnostics and disease monitoring of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07654c74db7467e7588089114a2ee5e54a7d1bf9" target='_blank'>
              Resolving leukemic stem cell heterogeneity and plasticity with single-cell multiomics.
              </a>
            </td>
          <td>
            Frank Y Huang, Andreas Trumpp, Patrick Stelmach
          </td>
          <td>2025-08-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Aim: Urinary bladder cancer (UBC) often develops chemoresistance, reducing treatment effectiveness. This study aimed to investigate diverse molecular mechanisms underlying acquired resistance by establishing and characterizing a comprehensive panel of UBC cell lines resistant to common chemotherapeutics.

 Methods: Fifteen UBC cell lines were examined: three parental lines (RT-112, TCC-SUP, UMUC-3) and twelve derived sublines adapted to cisplatin, vinblastine, or gemcitabine. Drug sensitivity was assessed using the SRB assay. Resistance mechanisms were explored via quantitative real-time PCR (targeting genes including ABCB1, dCK, hENT1, ECHDC1, TUBB3), Western blotting (assessing proteins such as p21, Cyclin B, and Mcl-1), and biochemical assessment of glutathione levels and redox state.

 Results: The adapted sublines exhibited distinct resistance profiles and cross-resistance patterns. Gene expression and protein analyses revealed drug- and lineage-specific alterations, involving factors such as p21, Cyclin B, and Mcl-1. Changes in glutathione metabolism were also associated with resistance. Notably, no single, universal mechanism accounted for resistance across the entire panel.

 Conclusion: UBC cells develop diverse, context-dependent adaptive strategies to resist cisplatin, vinblastine, and gemcitabine. These findings highlight the complexity of chemoresistance mechanisms. The characterized cell line panel represents a valuable resource for future studies aimed at understanding and overcoming drug resistance in bladder cancer, suggesting that personalized therapeutic approaches may be necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fbd01421411fd7eb4380d44e0e070090cf7648" target='_blank'>
              Distinct adaptive strategies to cisplatin, vinblastine and gemcitabine in a panel of chemoresistant bladder cancer cell lines
              </a>
            </td>
          <td>
            Monika Cuprych-Belter, Agnieszka Łupicka-Słowik, Artur Anisiewicz, Martin Michaelis, J. Cinatl jr., Mateusz Psurski
          </td>
          <td>2025-09-12</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318599f6810efcf5bf3014e92955379d25d16eda" target='_blank'>
              Synthetic lethality in cancer drug discovery: challenges and opportunities.
              </a>
            </td>
          <td>
            Emanuel Gonçalves, Colm J. Ryan, David J. Adams
          </td>
          <td>2025-09-11</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="With advancements in technology, single-cell RNA sequencing has emerged as a power-ful tool in cancer drug discovery. This technique enables the construction of gene expression profiles at the individual cell level, offering detailed insights into cellular heterogeneity and molecular path-ways involved in tumor development. It enables researchers to gain a deeper understanding of tumor heterogeneity. Researchers can study cell subpopulations and gene expression patterns. This under-standing helps in identifying potential drug targets. Additionally, it aids in predicting therapeutic responses. This high-resolution gene expression analysis provides a new perspective and oppor-tunity for cancer drug development, which is expected to accelerate the discovery and development process of new anti-cancer drugs. This article provides a comprehensive overview of the basic pro-cesses and developmental trajectory of single-cell RNA sequencing technology, with a particular emphasis on its applications in various aspects of cancer drug discovery. It also addresses the chal-lenges faced by single-cell RNA sequencing and potential future directions. This review aims to enhance readers' understanding of single-cell sequencing, inspire new ideas for oncology drug de-velopment, and support the translation of clinical research into practice, ultimately enabling physi-cians to design more precise and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f74b7c7821b0c57a9649d29962bfb630eb54595" target='_blank'>
              Single-Cell Transcriptomics: Technical Advances, Applications and Challenges in Cancer Drug Discovery.
              </a>
            </td>
          <td>
            Yueying Yang, Lingyu Meng, Teng Zhang, Jianjun Tan
          </td>
          <td>2025-07-30</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ~19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorectal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, Da-Eun Kim, Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, Hyobin Julianne Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, Kyle K.-H. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, Dae-Kyum Kim
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The addition of CDK4/6 inhibitors to endocrine therapy has significantly improved outcomes in HR+/HER2-breast cancer. However, variable patient responses and acquired resistance remain a clinical challenge. We therefore defined the comprehensive molecular response to palbociclib, the most clinically used CDK4/6 inhibitor. Global analyses of gene expression, protein abundance, splicing, and chromatin accessibility revealed broad patterns and specific changes that result from CDK4/6-inhibition in breast cancer cells. We uncovered unexpected feedback between CDK4/6 and estrogen-response signaling, which has clear clinical implications. We also revealed a widespread alternative splicing program that partially overlapped with genes whose expression is regulated, and which is expected to impact protein function. These molecular changes nominated combination therapies that interfere with the activation of CDKs or ERα. Accordingly, co-targeting CDK7, which regulates CDK2, CDK4/6 and ERα, additively impacted cell fitness. Collectively, these data reveal a complex, multi-tiered response to CDK4/6 inhibition, with implications for therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceb1151f3964ee516b8b4bec2c0e19c51711b54" target='_blank'>
              Charting the Multi-level Molecular Response to Palbociclib in ER-Positive Breast Cancer
              </a>
            </td>
          <td>
            Archishma Kavalipati, Amy Aponte, Michael E Sullivan, Sarah L Whittington, José C Martínez, Grant A. Goda, Maria M. Aleman, Michael J Emanuele, Daniel Dominguez
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Melanoma, the deadliest human skin cancer, frequently harbors activating BRAF mutations, with V600E being the most prevalent. These alterations drive constitutive activation of the MAPK pathway, promoting uncontrolled cell proliferation, survival, and dissemination. The advent of BRAFi and MEKi has significantly improved outcomes in BRAF V600-mutant melanoma. However, therapeutic resistance remains a major clinical barrier. Methods: This review integrates recent findings from preclinical and clinical studies to delineate resistance mechanisms to BRAF-targeted therapy. It categorizes resistance into primary (intrinsic), adaptive, and acquired forms, and analyzes their molecular underpinnings, including genetic and epigenetic alterations, pathway reactivation, and microenvironmental interactions. Results: Primary resistance is linked to pre-existing genetic and epigenetic changes that activate alternative signaling pathways, such as PI3K-AKT. Adaptive and acquired resistance includes secondary BRAF mutations, pathway redundancy, phenotype switching, and immune and stromal interactions. High-throughput sequencing has revealed novel mutations, including NRAS, NF1, and PTEN alterations, that contribute to resistance. Discussion: Understanding the multifaceted nature of resistance is critical to improving outcomes in advanced melanoma. This review highlights emerging strategies to overcome resistance, including combinatorial therapies, metabolic targeting, and biomarker-driven approaches, aiming to inform future therapeutic development and precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fe52d09f6a6cd013c45f83f652a44de19330f7" target='_blank'>
              Molecular Basis of BRAF Inhibitor Resistance in Melanoma: A Systematic Review
              </a>
            </td>
          <td>
            Ilaria Cosci, V. Salizzato, P. Del Fiore, J. Pigozzo, Valentina Guarneri, Simone Mocellin, A. Ferlin, Sara Mathlouthi, L. Piccin, Mariangela Garofalo
          </td>
          <td>2025-08-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a9aba01e0cb2e8c3213b471e7d8f34414b59cc" target='_blank'>
              Functional phenotyping of genomic variants using joint multiomic single-cell DNA-RNA sequencing.
              </a>
            </td>
          <td>
            Dominik Lindenhofer, Julia R. Bauman, John A. Hawkins, Donnacha Fitzgerald, Umut Yildiz, Haeyeon Jung, Anastasiia Korosteleva, Mikael Marttinen, M. Kueblbeck, Judith B. Zaugg, K. Noh, Sascha Dietrich, W. Huber, O. Stegle, Lars M. Steinmetz
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-09</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome instability in induced pluripotent stem cells (IPSC) poses a significant challenge for their use in research and medicine. Cataloging and precisely describing all the identified aberrations that arise during cell reprogramming, expansion, and differentiation is essential for improving approaches to instability prevention and ensuring genetic quality control. We report the karyotypic analysis of 65 cell lines derived from skin fibroblasts, urinal sediment, and peripheral blood mononuclear cells of 33 individuals, 82% of whom suffer from monogenic genetic disorders not associated with genetic instability. Trisomy of chromosomes 20 and 8 was revealed recurrently, while the 1q arm was the most frequently affected region involved in interstitial duplications and unbalanced translocations with chromosomes 15 and 18. The localization of rearrangement breakpoints identified by SNP arrays within the large DCC gene and histone gene clusters links genetic instability in IPSCs to replication-stress-induced chromosome breakage at common and early replicating fragile sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4802c78d378e6001adc2d0cfa2a7308e860155" target='_blank'>
              Chromosomal Aberrations in Induced Pluripotent Stem Cells: Identification of Breakpoints in the Large DCC Gene and HIST2 Histone Gene Cluster
              </a>
            </td>
          <td>
            D. Zheglo, V. Pozhitnova, A. V. Kislova, Zh.G. Markova, Danila Kiselev, P.S. Sviridov, Valeria Sviridova, Lyajsan I. Gumerova, S. Smirnikhina, Almaqdad Alsalloum, S. V. Pylina, S. Kutsev, Ekaterina Sergeevna Voronina
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The prognosis of low-grade glioma (LGG) exhibits significant heterogeneity, and the optimal management strategy remains controversial. Therefore, identifying biomarkers associated with glioma prognosis is necessary. Methods Hub genes were identified and a prognostic risk signature was constructed in the TCGA cohort using LASSO Cox regression. The predictive significance of this model was assessed and confirmed in the independent CGGA cohort. Single-cell transcriptional profiling characterized the hub gene expression patterns across cell types. Results We screened four hub genes (SERPINE1, KIF2C, SLC16A1, FABP5). Patients were stratified into high- and low-risk groups using this model. The high-risk group had significantly lower survival rates in both TCGA and CGGA cohorts. The model demonstrated strong predictive power (1/2/3-year AUCs: 0.85/0.85/0.83 in TCGA; 0.73/0.71 for 2/3 years in CGGA). Functionally, the high-risk group exhibited activation of pro-tumorigenic pathways, showed higher levels of immune infiltration across multiple lymphocyte and myeloid subsets, and demonstrated higher sensitivity to drugs such as AZD5582 and AZD8055, evidenced by significantly lower IC50 values. At the single-cell level, FABP5 was highly expressed in myeloid cells, and FABP5 + tumor-associated macrophages (TAMs) were enriched in lipid metabolism and antigen presentation pathways. Pseudotime analysis suggested increasing FABP5 expression during TAMs differentiation. Conclusions The constructed 4-gene hypoxia-lactylation prognostic model can effectively predict survival risk in LGG patients. The high-risk group is characterized by activation of pro-tumorigenic pathways, high immune infiltration, and sensitivity to specific targeted drugs. FABP5 + TAMs participate in LGG progression by regulating lipid metabolism and inflammatory responses, representing a potential therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03459-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/193ed959679c670c14a31c6b0e0120de559bd31d" target='_blank'>
              Integrated single-cell and bulk RNA-sequencing data reveal prognosis and therapeutic response in low-grade glioma based on hypoxia-lactylation related genes
              </a>
            </td>
          <td>
            Hongyu Tang, Ruijie Zhang, Ying Xu, Xu Wang, Jingyan Gu, Lianping Gu, Yao Liu
          </td>
          <td>2025-08-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

Cancer remains one of the most complex diseases globally, driven by genetic and epigenetic
alterations that disrupt normal cellular processes. This complexity is further amplified by
mechanisms such as uncontrolled cell proliferation, inhibition of apoptosis, angiogenesis, immune
evasion, and metabolic reprogramming, all of which contribute to tumor growth and metastasis.
While advancements in diagnostic technologies, including next-generation sequencing and liquid
biopsies, have facilitated early detection and the development of personalized therapeutic strategies,
traditional treatments such as chemotherapy, radiotherapy, and immunotherapy continue to
face significant limitations, including drug resistance, adverse side effects, and tumor heterogeneity.
Recent studies have emphasized the potential of bacteriophages in cancer treatment. As viruses
that specifically target bacteria, phages offer unique advantages due to their genetic modifiability,
roles in immunotherapy, and capabilities in targeted drug delivery. They can reshape the tumor microenvironment,
enhance immune responses, and deliver therapeutic agents directly to cancer
cells. Moreover, phages hold promise in addressing antibiotic resistance in cancer patients and improving
the efficacy of traditional therapies through combination strategies. However, challenges
remain regarding the optimization of phage stability, bioavailability, and tumor-specific targeting.
Research and clinical trials on the therapeutic application of phages are critical to unlocking their
full potential as innovative approaches to overcome the limitations of current cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f176f476b0e17d22a339146ab7610aed3ab80193" target='_blank'>
              Bacteriophages in Cancer Treatment
              </a>
            </td>
          <td>
            Mustafa Ustundag, Berrin Ustundag
          </td>
          <td>2025-08-22</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Pancreatic ductal adenocarcinoma (PDAC) is a highly fatal disease with limited treatment options. PARP inhibitors (PARPi) have shown promise in treating PDAC with homologous recombination deficiency (HRD), but rapid acquisition of resistance limits their efficacy. Our objective is to investigate mechanisms of resistance to PARPi in BRCA2‐mutant PDAC cells and identify potential therapeutic targets to modulate this resistance. We developed olaparib‐ and talazoparib‐resistant Capan‐1 cell lines and characterised their resistance profiles using viability assays, RNA sequencing and metabolomic profiling. We also developed a cisplatin‐resistant Capan‐1 cell line to compare resistance mechanisms between PARPi and platinum agents. Both olaparib‐ and talazoparib‐resistant cells showed cross‐resistance to other PARPi and oxaliplatin, but not to gemcitabine or 5‐FU. Talazoparib‐resistant cells exhibited a similar resistance profile to cisplatin‐resistant cells, including decreased PARP1 expression and altered metabolomic profiles. RNA sequencing and metabolomic profiling revealed significant enrichment of metabolic pathways, including oxidative phosphorylation and glycolysis, in resistant cells. Our study highlights the complexity of resistance mechanisms to PARPi in PDAC and identifies potential therapeutic targets in metabolism. The differences in the resistance profiles between olaparib and talazoparib suggest that PARP‐trapping potency may play a role in resistance development. Further research is needed to validate these findings and explore novel therapeutic strategies to overcome resistance to PARPi in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba75e6971e2007287691a51829a55e0eef1065d" target='_blank'>
              Mechanisms of Resistance to PARPi in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Jojanneke Stoof, Charlotte Andrieu, Fiona O’Connell, Jacintha O'Sullivan, M. Lowery, Naomi Walsh
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive malignancy of the bile ducts, and resistance to gemcitabine, a first-line chemotherapy, significantly complicates treatment. Despite extensive research, the molecular mechanisms underlying gemcitabine resistance in iCCA are not fully understood. This study aims to identify key genes associated with gemcitabine resistance in iCCA, investigate the role of lactylation, and propose potential therapeutic targets. Methods A comprehensive bioinformatics analysis was conducted using publicly available transcriptomic data from gemcitabine-resistant iCCA cell lines and patient samples. Differential expression analysis was performed to identify upregulated and downregulated genes. GSEA were used to explore relevant molecular pathways. Immune landscape analysis was carried out using CIBERSORT to assess immune cell infiltration in the tumor microenvironment. Key resistance-related genes were identified through Lasso, RF, and SVM-REF analyses. ITGB4 function was further validated in vitro by siRNA knockdown in HUCCT1 and RBE cells, followed by cell viability and apoptosis assays with or without gemcitabine treatment. Results Pathway analysis revealed the involvement of cell cycle regulation, DNA replication, and p53 signaling in gemcitabine resistance. The high group associated with resistance showed significantly worse survival outcomes, with a positive correlation between resistance and lactylation levels. Immune landscape analysis indicated altered immune cell infiltration, including increased M2 macrophages and decreased CD8+ T cells in the high group. Key resistance-related genes, including CDC20, H2AX, HK2, and ITGB4, were identified as critical in drug resistance. Experimentally, ITGB4 knockdown markedly enhanced gemcitabine’s antiproliferative and pro-apoptotic effects on cholangiocarcinoma cells, supporting its role in mediating resistance. Molecular docking revealed Dioscin and Deacetyllanatoside C as potential ITGB4-interacting compounds. Conclusion This study sheds light on the molecular mechanisms of gemcitabine resistance in iCCA, emphasizing lactylation’s role and the significance of immune modulation. ITGB4 is identified as a promising therapeutic target, and the findings suggest that targeting these genes could help overcome resistance in iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea10bda53b7cbe7c506a45067aed62ce5a1a0fe" target='_blank'>
              Integrated transcriptomic analysis identifies lactylation-linked gemcitabine resistance and therapeutic targets in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Wenwei Xie, Jialiang Hu, Hanmei Xu
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Psoriasis is a chronic inflammatory skin disease involving dysregulated immune responses and complex genetic factors. This study combines single-cell RNA sequencing (scRNA-seq), gene expression profiling, and genetic analysis to explore cellular and molecular contributors to psoriasis. Methods Single-cell RNA-seq data (n = 3 psoriasis, n = 2 control; GSE228421) were used for cell-type annotation and functional characterization. T cell subsets were analyzed for differentiation trajectories and cell-cell communication. Differentially expressed genes in mucosal-associated invariant T (MAIT) cells were evaluated by enrichment analysis. Candidate gene causality was tested via eQTL-based Mendelian randomization (MR) and supported by bulk RNA-seq validation. Results MAIT cells were enriched in psoriatic lesions and exhibited strong intercellular interactions. Functional analyses revealed activation of IL6-JAK-STAT3 signaling, TNF-NFκB pathway, and glycolysis in MAIT cells. MR identified RPS20 as a protective factor (OR = 0.5994, p = 0.011) and PFN1 as a potential risk gene (OR = 1.7229, p = 0.037), with PFN1 highly expressed in MAIT cells. Colocalization analysis showed no significant genetic overlap between PFN1 expression and psoriasis risk. Metabolic profiling revealed differential pathway involvement in PFN1+ and PFN1− MAIT cells. Conclusion Our integrative analysis highlights MAIT cells and PFN1 as likely contributors to psoriasis pathogenesis. These findings offer insights into immune and metabolic alterations, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48055881b868991adfc41d78b968062c6c8aae6e" target='_blank'>
              MAIT Cells with High PFN1 Expression Mediate Immune Activation and Metabolic Reprogramming in Psoriasis
              </a>
            </td>
          <td>
            Man-Ning Wu, , Xiang-Nan Zhou, Sangwon Hong, Lei Wang, Yan-Ping Bai
          </td>
          <td>2025-09-09</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer research has traditionally focused on identifying driver genes, those with mutations that initiate tumorigenesis. The Cancer Driver Gene (CDG) paradigm, further supported by the observation of oncogene addiction in tumors, has successfully guided the development of targeted therapies. However, the limitations of this driver-centric view, highlighted by the broad emergence of frequent therapeutic resistance, the presence of driver mutations in healthy tissues or individuals, and the lack of identifiable drivers in many tumors, call for a shift in perspective and clinical practice. The latest network controllability perspective on cancer cells introduced the concept of Cancer Keeper Genes (CKGs) and a CKG-based paradigm for cancer therapeutics. The new concept encompasses the concept of non-oncogene addiction, emphasizing reliance on non-mutated pathways crucial for maintaining oncogenic cellular states. Here, we explore the transition towards a system-level understanding of cancer based on the CKG paradigm, emphasizing the essential role of genes required for tumor maintenance, irrespective of their initiating function or mutational capacity. We discuss clinical implications of this paradigm shift, highlighting the progress made so far and potential of targeting non-driver CKGs, genes involved in processes like DNA damage response, proteostasis, and metabolism, as a promising strategy to overcome therapeutic challenges and achieve more durable cancer control. Targeting these maintenance vulnerabilities represents a critical evolution in precision oncology, moving towards therapies designed to dismantle the networks sustaining malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e80123ffc1524bf35e125c1b2645e60b9bc7703" target='_blank'>
              From Cancer Drivers to Cancer Keepers: Paradigm Shift and Clinical Implications
              </a>
            </td>
          <td>
            Xizhe Zhang, Weixiong Zhang
          </td>
          <td>2025-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Targeted therapies have significantly improved the prognosis and productivity of non-small-cell lung cancer (NSCLC) patients carrying driver mutations, but drug resistance is inevitable. Histological transformation, an important resistance mechanism, is often manifested as transformation into small-cell lung cancer, large-cell neuroendocrine carcinoma, squamous cell carcinoma, and sarcomatoid carcinoma. The mechanisms involved are complex, including RB1/TP53 inactivation, epithelial–mesenchymal transition, and microenvironmental changes. Post-transformation tumors are often more aggressive and drug-resistant, with limited therapeutic options and poorer prognosis. In this paper, we systematically review the histological transformation types, molecular mechanisms, and therapeutic strategies of NSCLC after resistance to targeted therapy, with the aim of providing a reference for clinical decision-making and promoting the development of individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71911ca0beebb701fd3deee5a9b4ea23c86ca3e7" target='_blank'>
              Histologic transformation after targeted therapy resistance in driver gene-positive NSCLC: mechanisms and therapeutic challenges
              </a>
            </td>
          <td>
            Xinyue Li, K. Ding, Dujiang Liu, Yuxia Wang, Yanjun Xu
          </td>
          <td>2025-08-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Heat stroke, caused by excessive heat production or impaired dissipation, often results from prolonged heat exposure or strenuous activity. Ferroptosis, a novel form of programmed cell death, has been implicated in its pathogenesis, though its mechanisms remain unclear. Objective This study investigates the molecular mechanisms linking heat stroke and ferroptosis using single-cell and transcriptomic analyses to identify diagnostic and therapeutic targets. Methods Peripheral blood samples from 29 heat stroke patients, recruited at an early stage of the condition, underwent transcriptome sequencing, and single-cell RNA sequencing was conducted for two representative cases. Ferroptosis-related genes were identified by integrating the FerrDB database, followed by weighted gene co-expression network analysis (WGCNA) and differential gene expression analysis to pinpoint ferroptosis-related genes most characteristic of heat stroke. Functional enrichment analyses, including GO and KEGG pathways, were performed. Single-cell RNA sequencing revealed immune microenvironment alterations and marker genes linked to heat stroke pathogenesis. Receiver operating characteristic (ROC) analysis evaluated the diagnostic potential of these genes. Additionally, pseudotime analysis elucidated cell differentiation trajectories and the roles of key genes. Results In Dataset 1, 630 differentially expressed genes (546 up-regulated, 84 down-regulated) and 1,979 heat stroke-related genes were identified. Among them, 14 intersected with 1,001 ferroptosis-related genes and were enriched in pathways like fatty acid metabolism, inflammation, and immune regulation. Single-cell sequencing showed increased monocytes and macrophages in heat stroke patients. Five core genes (ACSL1, MAPK14, ALOX5AP, PROK2, and DUSP1) were validated in Dataset 2 with high AUC values (1.0, 1.0, 0.952, 0.976, and 0.881, respectively). These genes were highly expressed in neutrophils, dendritic cells, and monocytes. Pseudotime analysis confirmed their roles in cell differentiation and disease progression. Conclusion ACSL1, MAPK14, ALOX5AP, PROK2, and DUSP1 were identified as novel biomarkers for diagnosing heat stroke. ROC validation confirmed their strong association with the disease, highlighting their potential as diagnostic targets. Pseudotime analysis revealed their consistency in cellular differentiation trajectories. Supplementary Information The online version contains supplementary material available at 10.1186/s12920-025-02188-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/add94abc3cfc92cd3ce2bbe6f418f6aab6e136f8" target='_blank'>
              Single-cell sequencing-based study of ferroptosis mechanisms in heat stroke: identification of key biomarkers and dynamic analysis of the immune microenvironment
              </a>
            </td>
          <td>
            Defeng Yin, Qin Guo, Hao Jiang, Yonglan Hu, Lu Liu, Xiang Li, Chenglin Wang, Shilin Li, Kaiyu Jin, Yingchun Hu
          </td>
          <td>2025-07-31</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6adfd64895599aa6bae6a1f8896e93b0d56bc5ac" target='_blank'>
              ADAR1-mediated RNA editing in breast cancer: molecular mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Si-Yu Chen, Si-Ying Chen, Shu Yang, Yi Li, Shengyong Yang
          </td>
          <td>2025-08-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59d7e7dcc2e2b5046587aac4504172a2317b36b" target='_blank'>
              Coordinated gene expression within sustained STAT3-associated chromatin conformations contributes to hepatocellular carcinoma progression.
              </a>
            </td>
          <td>
            Sunyoung Jang, Sumin Yoon, Hyeokjun Yang, Nayun Choi, Su-hyang Han, Lark Kyun Kim, Hyoung-Pyo Kim, Jong Hoon Park, Daeyoup Lee, Kyung Hyun Yoo
          </td>
          <td>2025-07-26</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of >1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485aaa6c6ae7c10944eff4bdd9fd536b93adfcd9" target='_blank'>
              Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution
              </a>
            </td>
          <td>
            R. Edahiro, Go Sato, Tatsuhiko Naito, Y. Shirai, R. Saiki, K. Sonehara, Yoshihiko Tomofuji, Kenichi Yamamoto, S. Namba, Noa Sasa, Genta Nagao, Qingbo S. Wang, Yugo Takahashi, Takanori Hasegawa, T. Kishikawa, Ken Suzuki, Yu-Chen Liu, D. Motooka, Ayako Takuwa, Hiromu Tanaka, S. Azekawa, H. Namkoong, R. Koike, Akinori Kimura, S. Imoto, Satoru Miyano, Takanori Kanai, K. Fukunaga, Mamoru Uemura, T. Morita, Yasuhiro Kato, H. Hirata, Y. Takeda, Y. Doki, H. Eguchi, Daisuke Okuzaki, Shuhei Sakakibara, Seishi Ogawa, Atsushi Kumanogoh, Yukinori Okada
          </td>
          <td>2025-07-28</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>55</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) poses a considerable global health challenge, notably due to the resistance to sorafenib therapy, which significantly impedes effective treatment strategies. This study aimed to identify potential resistance-associated genes and develop a robust prognostic model to predict patient outcomes. Methods We utilized transcriptomic data from the gene expression omnibus (GEO) database, focusing on sorafenib-resistant Huh7 and MHCC97H cell lines (GSE94550 and GSE176151), and integrated expression, mutation, and clinical data from the cancer genome atlas (TCGA) and international cancer genome consortium (ICGC) databases. Single-cell RNA sequencing data (GSE149614) were processed with the Seurat and Harmony R packages for quality control and integration. Differential gene expression analysis, consensus clustering, and principal component analysis were performed to identify significant genes and stratify patients based on prognostic outcomes. Results The analysis revealed 305 potential resistance-associated genes, with a seven-gene (ANAPC13, NCAPD2, KIF2C, CDK5RAP2, MANBA, PPAT, and LPCAT1) risk model demonstrating significant prognostic capability, indicated by area under curve values of 0.824, 0.746, and 0.717 for 1, 3, and 5-year survival predictions, respectively. Notably, immune cell infiltration analyses highlighted significant correlations between risk scores and specific immune cell types, suggesting potential therapeutic targets. Drug sensitivity analysis further identified various compounds with lower IC50 values in high-risk groups. To facilitate clinical application, a nomogram plot was designed. Conclusion This study provides a comprehensive framework for understanding sorafenib resistance in HCC, alongside a validated prognostic model that holds potential for clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6acc2fd61529281885a6db1b562f4fbba92f2fe" target='_blank'>
              Identifying genes associated with Sorafenib resistance in hepatocellular carcinoma to develop risk model
              </a>
            </td>
          <td>
            Kuan He, Hongyan Hui, Fang Zhou, Kai Liu, Xi Zhao, Jun Liu
          </td>
          <td>2025-08-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common malignancies worldwide. Its complex molecular mechanisms make traditional treatments ineffective in some patients. In recent years, advances in genetic engineering technologies have opened up new possibilities for precision medicine in BC. Researchers have attempted to intervene in cancer cell proliferation and induce apoptosis through various genetic strategies. Preclinical studies are increasing, and therapeutic targets and delivery technologies are expanding. However, current research has largely focused on specific cases or single technologies, lacking a systematic integration and analysis of various genetic strategies from a mechanistic perspective. This paper reviews three major genetic engineering strategies for BC based on their therapeutic mechanisms: RNA interference, gene compensatory expression, and gene repair and deletion. Representative studies, key technologies, challenges, and development trends are analyzed for each. Furthermore, this paper summarizes common challenges across these different mechanisms regarding delivery efficiency, stability, and targeting, and proposes potential directions for future synergistic strategies. This paper helps clarify current research approaches in BC gene therapy from a mechanistic perspective, providing a reference for the design of subsequent cross-mechanism integration and precision intervention strategies. However, issues such as mechanism overlap and diversification of delivery platforms still need further in-depth exploration. Future research can focus more on multi-mechanism combination strategies, individualized treatment pathways, and feasibility assessment of clinical transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38bb75ae7dd66792d49d2225441735e8c43eeb18" target='_blank'>
              Applications of Three Gene Engineering Mechanisms in Breast Cancer Therapy
              </a>
            </td>
          <td>
            Yifan Tian
          </td>
          <td>2025-09-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b8053e41e58296473fbe96b5f5b8ab91cf715" target='_blank'>
              Reversible therapeutic resistance in EGFR mutant lung cancer caused by RB1 loss induced lineage plasticity.
              </a>
            </td>
          <td>
            Letian Zhang, M. Chinnam, Bradley Balk, Xiaojing Zhang, Yanqing Wang, Tatiana Shaurova, Shamshad Alam, Saraswati Pokharel, Prashant Singh, Justine Jacobi, Jianming Wang, Eduardo Cortes Gomez, Michael Habitzruther, Mukund Seshadri, B. Atanassov, Wei Wu, E. Benevolenskaya, C. Blakely, Pamela Hershberger, Mark Long, T. Bivona, David W. Goodrich
          </td>
          <td>2025-08-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Thyroid cancer pathogenesis involves complex interactions between genetic predisposition and alterations in the tumor microenvironment. The causal relationships between inflammatory gene variants and thyroid cancer risk remain poorly understood, as does the cellular heterogeneity within the tumor ecosystem. This study aimed to investigate the causal associations between inflammatory protein genes and thyroid cancer risk, and to characterize the cellular composition and differentiation trajectories within the thyroid cancer microenvironment. Methods We employed a two-pronged approach combining Mendelian randomization (MR) analysis and single-cell RNA sequencing (scRNA-seq). MR analyses were conducted using genetic variants associated with the expression of inflammatory proteins (4EBP1_EIF4EBP1, ADA_ADA, ARTN_ARTN, AXIN1_AXIN1, and Beta-NGF_NGF) as instrumental variables to assess their causal effects on thyroid cancer risk. Multiple MR methods (MR Egger, weighted median, inverse variance weighted, simple mode, and weighted mode) were used to enhance robustness. For the cellular characterization, scRNA-seq was performed on thyroid cancer samples, followed by dimensionality reduction, clustering analysis, cell type annotation, and pseudotime trajectory inference. Results MR analyses revealed a significant positive causal association between AXIN1_AXIN1 expression and thyroid cancer risk (weighted median: OR = 1.396, p < 0.05; inverse variance weighted: OR = 1.291, p < 0.05), while ADA_ADA showed protective effects (simple mode: OR = 0.731, p < 0.05). The scRNA-seq analysis identified six major cell populations within the thyroid cancer microenvironment: epithelial cells, T cells, natural killer cells, fibroblasts, stromal cells, and macrophages. Pseudotime analysis revealed distinct differentiation trajectories with natural killer cells and macrophages appearing in early pseudotime, while epithelial cells and fibroblasts demonstrated multiple developmental states. Gene expression profiling identified four distinct cellular states with unique molecular signatures, including immune/inflammatory, stromal, and vascular components. Conclusion Our findings suggest that inflammatory protein genes, particularly AXIN1, have causal effects on thyroid cancer risk, providing potential targets for risk prediction and intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbfdc7a38a023cb6812db2d719e9e3e9bf3c6626" target='_blank'>
              The role of cellular heterogeneity in thyroid cancer: a multi-modal analysis approach with Mendelian randomization
              </a>
            </td>
          <td>
            Yichun Qian, Wei Chen, Xinyuan Chen, Shuai Cheng, Fangzhou Liu
          </td>
          <td>2025-08-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for elucidating tumor heterogeneity and clonal evolution. However, comprehensive scDNA-seq studies and tailored mutation panels for CSCC are lacking. Methods We analyzed the genomic landscape of Chinese CSCC patients via a Multi-Patient-Targeted (MPT) scDNA-seq approach. This method combined bulk exome sequencing with Tapestri scDNA-seq. Mutations identified through bulk sequencing were used to design a targeted panel for scDNA-seq. Comparative analysis was conducted to explore the associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis was performed to understand the evolutionary trajectories of the tumors. Results Bulk sequencing revealed a diverse spectrum of somatic mutations in CSCC tumors, with missense mutations being predominant. The top tumor mutations, such as those in NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A, presented similar frequencies to those reported in studies in Korean and Caucasian populations. However, the mutation frequencies of HRAS, TTN, MUC16 and MUC4 were significantly different from the Korean and Caucasian populations. Comparative analysis revealed associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis via scDNA-seq revealed distinct evolutionary trajectories and their potential correlation with tumor development and patient prognosis. Furthermore, scDNA-seq identified two low-frequency mutation clones, NLRP5 and HMMR, which play important roles in the clonal evolution of CSCC. Conclusions Our study introduced a novel MPT sequencing approach for CSCC, providing insights into tumor heterogeneity and clonal evolution. We identified novel mutations and their potential associations with patient survival and tumor characteristics. Overall, our study layed the groundwork for personalized treatment strategies and provides a reference for future applications of the MPT panel in CSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14585-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667c56c3222da6ac371627cb03ea1bf3e1f96087" target='_blank'>
              Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing
              </a>
            </td>
          <td>
            Wen Chen, Jiawei Xu, Chengdong Yu, Meng Zhou, Yong Ai, Wenbing Rao, Qingchuan Wang, Siyi Xu, Lei Tang, Si Gong, Jun Rao
          </td>
          <td>2025-08-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Late diagnosis and chemotherapy resistance, particularly to 5-fluorouracil (5-FU), contribute to the low survival rate in cholangiocarcinoma (CCA) patients. Identifying relevant genes and pathways, as well as novel targeted molecules, is crucial to overcoming 5-FU resistance and improving treatment outcomes for CCA patients.


OBJECTIVES
This study aimed to determine the potential molecules associated with 5-FU resistance in CCA cells.


MATERIAL AND METHODS
Transcriptomic datasets from 4 stable 5-FU-resistant cell lines and their corresponding parental lines were retrieved from the Gene Expression Omnibus. A series of bioinformatics analyses were conducted to identify key genes upregulated in 5-FU-resistant cells compared to their parental counterparts. The expression levels of candidate genes identified through bioinformatics analysis were validated in CCA tissues and cell lines.


RESULTS
Differential gene expression, protein-protein interaction, and Hub genes analysis revealed 8 genes that were significantly upregulated in 5-FU resistance cells compared to their parental cells. Six of the 8 genes, including TCP1, RPS6, RPS29, HSPA5, RPS15A, and NOTCH1, were upregulated in patient CCA tissues. Using real-time PCR, only the expression levels of NOTCH1 and TCP1 were significantly higher in the 5-FU insensitive CCA cell lines, KKU-213A and KKU-213B, than that of the 5-FU sensitive CCA cell line, KKU-055. A similar result was observed in stable 5-FU-resistant cell lines (KKU-213A-FR and KKU-213B-FR) compared to their parental cells.


CONCLUSIONS
The bioinformatic analysis and PCR results revealed that NOTCH1 and TCP1 might be associated with 5-FU resistance and serve as potential molecular targets to enhance 5-FU sensitivity in CCA cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8104f35b6140638792fb57197dadae1949d4723" target='_blank'>
              Bioinformatics analysis identified TCP1 and NOTCH1 as potential target molecules to overcome 5-fluorouracil resistance in cholangiocarcinoma.
              </a>
            </td>
          <td>
            Sonexai Kidoikhammouan, Nopkamol Kanchanangkul, Worachart Lert-itthiporn, Raksawan Deenonpoe, Charupong Saengboonmee, S. Obchoei, S. Wongkham, Wunchana Seubwai
          </td>
          <td>2025-09-04</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Pancreatic cancer is marked by a poor prognosis and an exceptionally high mortality rate, with its aggressive nature contributing to its classification as a highly malignant disease. For effective therapeutic strategies, the development of sophisticated and regulated DNA manipulation methods is essential. Originally part of the prokaryotic immune system, CRISPR/Cas has emerged as a pivotal genome-editing tool with promising applications in pancreatic cancer research and therapy. This gene editing method is known for simplicity, rapid advancement, and superior precision compared to earlier techniques. Its adaptability allows precise gene editing for therapeutic purposes, including oncogene silencing and correction of pathogenic mutations. Additionally, CRISPR-driven gene editing has facilitated the development of pancreatic cancer models, which serve as valuable platforms for drug discovery and personalized treatment strategies, offering deeper insights into the genetic landscape of pancreatic tumors. This article provides an overview of the current applications of CRISPR technology in gene therapy and cancer research, particularly in the context of pancreatic cancer, and lays the foundation for future studies. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab39b4071496f7653becd6d74d5e06cbbcdd5a1" target='_blank'>
              CRISPR/Cas technologies in pancreatic cancer research and therapeutics: recent advances and future outlook
              </a>
            </td>
          <td>
            Hanieh Noormohamadi, Hamed Soleimani Samarkhazan, Maryam Kargar, F. Maroufi, Nazli Servatian, Fatemeh Davami
          </td>
          <td>2025-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Circulating tumour DNA (ctDNA) in liquid biopsies has emerged as a powerful biomarker in cancer patients. Its relative abundance in cell-free DNA serves as a proxy for the overall tumour burden. Here we present GeneBits, a method for cancer therapy monitoring and relapse detection. GeneBits employs tumour-informed enrichment panels targeting 20–100 somatic single-nucleotide variants (SNVs) in plasma-derived DNA, combined with ultra-deep sequencing and unique molecular barcoding. In conjunction with the newly developed computational method umiVar, GeneBits enables accurate detection of molecular residual disease and early relapse identification. Results To assess the performance of GeneBits and umiVar, we conducted benchmarking experiments using three different commercial cell-free DNA reference standards. These standards were tested with targeted next-generation sequencing (NGS) workflows from both IDT and Twist, allowing us to evaluate the consistency and accuracy of our approach across different oligo-enrichment strategies. GeneBits achieved comparable depth of coverage across all target sites, demonstrating robust performance independent of the enrichment kit used. For duplex reads with ≥ 4x UMI-family size, umiVar achieved exceptionally low error rates, ranging from 7.4×10-7 to 7.5×10-5. Even when including mixed consensus reads (duplex & simplex), error rates remained low, between 6.1×10-6 and 9×10-5. Furthermore, umiVar enabled variant detection at a limit of detection as low as 0.0017%, with no false positive calls in mutation-free reference samples. In a reanalysed melanoma cohort, variant allele frequency kinetics closely mirrored imaging results, confirming the clinical relevance of our method. Conclusion GeneBits and umiVar enable highly accurate therapy and relapse monitoring in plasma as well as identification of molecular residual disease within four weeks of tumour surgery or biopsy. By leveraging small, tumour-informed sequencing panels, GeneBits provides a targeted, cost-effective, and scalable approach for ctDNA-based cancer monitoring. The benchmarking experiments using multiple commercial cell-free DNA reference standards confirmed the high sensitivity and specificity of GeneBits and umiVar, making them valuable tools for precision oncology. UmiVar is available at https://github.com/imgag/umiVar. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06993-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599400583a25eed13254e7af73c49e834e021490" target='_blank'>
              GeneBits: ultra-sensitive tumour-informed ctDNA monitoring of treatment response and relapse in cancer patients
              </a>
            </td>
          <td>
            Julian Broche, Olga Kelemen, Aishwarya Sekar, Leon Schütz, Francesc Muyas, A. Forschner, Christopher Schroeder, Stephan Ossowski
          </td>
          <td>2025-08-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of both variants of each pair were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a prophage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variants
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Immune-related genes (IRGs) play a pivotal role in the tumor microenvironment of triple-negative breast cancer (TNBC). This study aimed to uncover novel prognostic markers for TNBC based on IRGs. Differential expression analysis, univariate Cox regression, and machine learning algorithms identified BPI, GPHA2, KIR3DL3, MASP1, RASGRP1, and SEMA4A as potential prognostic genes. A random survival forest (RSF) model was developed and subsequently validated, alongside the creation of a prognostic nomogram to assess gene diagnostic performance. Further, epithelial cells, fibroblasts, and T cells were recognized as the primary cell types through which RASGRP1 exerts its effects. Finally, an analysis of the immune microenvironment and drug predictions provided fresh insights for the treatment of TNBC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15157-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5a164db4384d30a7ce7d87a791eadd2b7becb9" target='_blank'>
              Immune microenvironment and prognostic genes of triple-negative breast cancer in the context of transcriptome and single-cell sequencing
              </a>
            </td>
          <td>
            Xiangqing Huang, Wei Chen, Zhiwu Lin, Hui Zhang, Ruijuan Wang, Cailing Lin
          </td>
          <td>2025-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Trachea squamous cell carcinoma (TSCC) is a subtype of lung cancer. A thorough investigation of the tumor microenvironment of TSCC is crucial for the development of cancer therapeutics and predicting clinical responses. In this study, we utilized single-cell RNA sequencing to analyze seven TSCC samples (including five malignant and two non-malignant samples) and obtained 70,682 high-quality cells. Based on the expression levels of marker genes, we identified 7 major cell types within the samples. By comparing malignant samples that received chemotherapy with those that did not, we identified critical transcriptional regulators responsible for T cell state transition in response to chemotherapy. Additionally, we found specific transcriptional regulators and differentially expressed genes between malignant and non-malignant groups. We identified more particularly abundant specifical intercellular communication in the malignant sample group and that may significantly influence the progression and spread of cancerous cells. Overall, our study provides the first single-cell atlas that comprehensively explains TSCC development and chemotherapy effects, thereby laying a new molecular foundation for therapeutic research in TSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f4d5fafc26e824adc29e95477c2a4bfd6ebf074" target='_blank'>
              Comprehensive single-cell sequencing reveals the tumor microenvironment and tumor-specific characteristics in trachea squamous cell carcinoma
              </a>
            </td>
          <td>
            HongWu Wang, Hongli Li, Heng Zou, Han Meng, Yan Liu, Chengjun Ban, Weixia Yu, Miao Cheng, Jun Teng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a type of hematological cancer that arises from uncontrolled proliferation of plasma cells. In addition to frequent genetic mutations, malignant plasma cells are characterized by alterations to the epigenome. Myeloma cells display a genome-wide loss of DNA methylation and a corresponding increase in ‘active’ chromatin modifications. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements, which can impact genome regulation. Through paired epigenome and transcriptome profiling of patient derived MM samples, we have found that loss of DNA methylation in MM genomes results in the formation of partially methylated domains that are variable across patients. This loss of DNA methylation coincides with the expression of hundreds of transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. MM samples can be stratified based on L1 activity with distinct gene expression signatures. The high L1 samples are characterized by a more proliferative, less differentiated state as well as inhibition of interferon and genome defense pathways. Several L1 promoters generate chimeric transcripts with adjacent oncogenes. We further find that KRAB-zinc finger proteins (KZFPs) that are responsible for the epigenetic silencing of L1s have abnormally low abundance in MM samples with high L1 activity. These results indicate that cell proliferation in MM is associated with a loss of KZFP expression and activation of L1 elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8c94d95f4ddee7048f28a27ac43f9a3d704355" target='_blank'>
              Methylation variability and LINE-1 activation in multiple myeloma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Y. Song, Dustin Schones
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Long-read sequencing provides a more complete view of the genome than short-read sequencing, with advantages in the detection of structural variants, tandem repeats, and small variants (single nucleotide variants and insertions and deletions) in difficult-to-map regions. One limitation of long-read sequencing has been high input DNA requirements, with several micrograms required per sample. Here, we evaluated two methods of amplification-based long-read, whole-genome sequencing: Ultra-Low Input HiFi (ULI-HiFi) sequencing and droplet multiple displacement amplification (dMDA) sequencing. When the accuracy of these methods were benchmarked with a reference set of variant calls from the Genome in a Bottle consortium (NA24385), we observed high precision and recall of single nucleotide variants (SNVs) with ULI-HiFi compared to the dMDA-amplified samples (F1 scores for SNVs of 99.82% for ULI-HiFi compared to 89.46% for dMDA). Across a catalog of >1.6 million tandem repeats (TRs), ULI-HiFi achieved 90.4% perfect concordance and 98.9% accuracy when allowing for single motif differences. ULI-HiFi also successfully illuminated several medically-important genes that were poorly mapped with short-read DNA sequencing. Because ULI-HiFi only requires 10-20 nanograms of DNA, we extended ULI-HiFi to analyze a normal, polyp, and adenocarcinoma sample from a patient with familial adenomatous polyposis (FAP), a hereditary form of colorectal cancer. We identified a TR that progressively expanded in length from normal to polyp to adenocarcinoma. This repeat is located in the 5' UTR of LIMD1, a reported tumor suppressor. We conclude that ULI-HiFi improves the characterization of genetic variants in dark regions of genomes from patient samples, enabling a better understanding of human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f143c1d79384cc975b586b0a244b0df19c4bdd67" target='_blank'>
              Whole-genome variant detection in long-read sequencing data from ultra-low input patient samples
              </a>
            </td>
          <td>
            Katherine Wang, Hayan Lee, Cera J Aex, Lucas Finot, Kevin Zhu, Julianna R Chang, Aaron M Horning, W. Rowell, Philip Li, S. Kingan, M. Snyder, G. Erwin
          </td>
          <td>2025-07-27</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c0a59124a03b9df5aeccdd7b10806e57fa0f866" target='_blank'>
              In vivo screening reveals cell-intrinsic mediators of solid tumor resistance to CAR T cell-therapy.
              </a>
            </td>
          <td>
            Julia Fröse, Charles A Whittaker, Paul Leclerc, Evelyn Chen, Sean Doherty, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, Tuomas Tammela, Michael T. Hemann
          </td>
          <td>2025-07-29</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background/Objectives: Medulloblastoma is the most common malignant brain tumor in children and comprises four molecular subtypes—WNT, SHH, Group 3, and Group 4—each with distinct genetic, epigenetic, and metabolic features. Increasing evidence highlights the critical role of metabolic reprogramming and epigenetic alterations in driving tumor progression, therapy resistance, and clinical outcomes. This review aims to explore the interplay between metabolic and epigenetic mechanisms in medulloblastoma, with a focus on their functional roles and therapeutic implications. Methods: A comprehensive literature review was conducted using PubMed and relevant databases, focusing on recent studies examining metabolic pathways and epigenetic regulation in medulloblastoma subtypes. Particular attention was given to experimental findings from in vitro and in vivo models, as well as emerging preclinical therapeutic strategies targeting these pathways. Results: Medulloblastoma exhibits metabolic adaptations such as increased glycolysis, lipid biosynthesis, and altered amino acid metabolism. These changes support rapid cell proliferation and interact with the tumor microenvironment. Concurrently, epigenetic mechanisms—including DNA methylation, histone modification, chromatin remodeling, and non-coding RNA regulation—contribute to tumor aggressiveness and treatment resistance. Notably, metabolic intermediates often serve as cofactors for epigenetic enzymes, creating feedback loops that reinforce oncogenic states. Preclinical studies suggest that targeting metabolic vulnerabilities or epigenetic regulators—and particularly their combination—can suppress tumor growth and overcome resistance mechanisms. Conclusions: The metabolic–epigenetic crosstalk in medulloblastoma represents a promising area for therapeutic innovation. Understanding subtype-specific dependencies and integrating biomarkers for patient stratification could facilitate the development of precision medicine approaches that improve outcomes and reduce long-term treatment-related toxicity in pediatric patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733445d34f0ee82899b45eeeaeabd1e9ff5c7ba8" target='_blank'>
              Deciphering Medulloblastoma: Epigenetic and Metabolic Changes Driving Tumorigenesis and Treatment Outcomes
              </a>
            </td>
          <td>
            Jenny Bonifacio-Mundaca, Sandro Casavilca-Zambrano, Christophe Desterke, Íñigo Casafont, Jorge Mata-Garrido
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is a major challenge to public health in China. Its incidence and mortality rates are increasing year by year, and traditional treatment methods are facing limitations such as drug resistance and side effects. This year's research has found that circRNA plays an important role in tumorigenesis, metastasis and drug resistance by regulating mechanisms such as miRNA interaction networks, gene expression and cell cycles. Its stability and specificity make it a novel biomarker and therapeutic target. However, the standardization of circRNA detection techniques is insufficient, and its role in epigenetic regulation and the tumor microenvironment remains unclear. This article systematically analyzes the generation mechanism, functional diversity (such as molecular sponges, protein interactions, and encoded peptides) of circRNAs, as well as their application strategies in cancer treatment (such as exosome detection) and therapy (such as mesenchymal stem cell delivery), revealing the dynamic network by which they regulate tumor drug resistance and cellular metabolism. The research results provide a theoretical basis for the development of non-invasive diagnostic tools and targeted therapies. In the future, it is necessary to combine multi-group masking techniques and engineered delivery systems to break through the bottleneck of detection technology and explore the synergistic mechanism between circRNA and other ncRNAs, so as to promote the clinical transformation of precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cfef4283107d45fd6a939a00a655bc04b6cb7fb" target='_blank'>
              Evaluation of the Regulatory Role of Circular RNA in Tumors and Its Application Value
              </a>
            </td>
          <td>
            Yingjie Xin
          </td>
          <td>2025-08-26</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Simple Summary Breast cancer (BC) remains a significant global health challenge, partly due to its ability to develop resistance to treatment. BC resistance arises from various biological mechanisms, including cancer cells actively expelling drugs, altering their internal metabolic and signaling pathways, evading the immune system, or persisting as drug-tolerant cancer stem cells. These changes occur in the context of a tumor microenvironment that further shields the cancer, limiting treatment access. As the disease advances, particularly when it spreads to other body parts, effective treatment becomes increasingly difficult. Recent advancements in artificial intelligence (AI) offer promising avenues for improving early detection, refining diagnoses, enhancing clinical decision-making and thereby personalizing treatment strategies. This review examines the biological mechanisms that cause BC resistance and its treatment. It further explores AI’s potential to address critical challenges, transforming BC therapy to provide improved survival rates and enhanced quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb5357bf49f1fab994fbcce36bcc1738760cec3" target='_blank'>
              The Underlying Mechanisms and Emerging Strategies to Overcome Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Krishnaswamy Kannan, Alagarsamy Srinivasan, Aarthi Kannan, Nawab Ali
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hormone receptor-positive (HR + )/HER2−negative (HER2 − ) breast cancer is the most common subtype, with biomarker-driven therapies improving outcomes. Circulating tumor DNA (ctDNA) analysis enables minimally invasive assessment of somatic alterations to guide therapy. However, assay choice impacts clinical utility, and access remains inconsistent. This study compares single-gene and panel-based sequencing for assessing PIK3CA mutations and broader genomic profiling. We conducted a prospective, multicenter study analyzing 161 plasma samples from 146 patients before initiating a new line of palliative therapy using the SiMSen-Seq (SSS) assay for PIK3CA hotspot mutations, the AVENIO ctDNA Expanded assay (77 genes) and mFAST-SeqS for tumor fraction estimation. High concordance (92.6%) was observed between SSS and AVENIO for PIK3CA mutations. AVENIO identified additional actionable alterations, including ESR1 (17.5%) and PI3K pathway alterations (40.6%), and together with tumor fraction estimation, improved interpretation of negative liquid biopsy findings. These findings support broader ctDNA profiling in clinical practice while highlighting accessibility challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e5152da00e15846285751e550fbaaef5313e05" target='_blank'>
              Clinical impact of single-gene vs. panel sequencing in advanced HR + /HER2− breast cancer: insights and implications
              </a>
            </td>
          <td>
            E. Klocker, N. Dobrić, Ricarda Graf, Christine Beichler, D. Hlauschek, C. Suppan, L. Pancheri, D. Egle, Carmen Albertini, R. Bartsch, A. Starzer, P. Jost, G. Rinnerthaler, Ellen Heitzer, N. Dandachi, M. Balic
          </td>
          <td>2025-08-07</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Introduction Next‐generation sequencing‐based comprehensive cancer genomic profiling (CGP) tests are beneficial for refining diagnosis and personalized treatment of various cancers. However, the clinical impact of CGP, as covered by public health insurance in the management of sarcomas, remains unknown. Especially, the data on the utility of the newly emerging dual DNA–RNA panel compared to the conventional DNA‐only panel in clinical settings is lacking. Therefore, we evaluated the utility of CGP in routine clinical practice for sarcoma treatment. Patients and Methods In this study, three types of DNA panel and one DNA–RNA panel, reimbursed by Japanese public health insurance, were utilized. We detected oncogenic and druggable gene mutations and genotype‐matched therapies. Results One hundred and thirty‐six patients were included in this study. Based on the detection of highly histology‐specific translocations in the sequencing results, 2.2% of patients were re‐classified. In patients with translocation‐related sarcomas, a DNA–RNA panel identified more histology‐specific fusion genes than DNA panels (p = 0.0035). Specifically, 86.8% and 39.0% of patients had oncogenic and druggable genomic alterations, respectively. Of these, 9.6% underwent genotype‐matched therapy, with a 36.3% response rate and an 81.8% disease control rate. Patients who were administered genomically matched therapy had better overall survival (OS) than those who did not in patients with metastatic or advanced sarcoma with no prior chemotherapy (3‐year OS: 83.3% vs. 48.0%, p = 0.42). Patients with TP53 and RB1 mutations had worse OS than those without. Germline findings were detected in 11.0% of the patients, one of whom had a truly germline origin. Conclusions This study suggests that publicly reimbursed CGP tests, particularly the dual DNA–RNA panel, could be beneficial for refining diagnostic precision in selected sarcoma subtypes, treatment decisions, detecting the germline findings, and prognosis prediction in routine clinical settings for sarcoma. The implementation of genotype‐matched therapies showed favorable clinical outcomes and improved the prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f63a84e1907839e26eef4d1f8d8db48869e5a3" target='_blank'>
              Real‐World Data of Comprehensive Cancer Genomic Profiling Tests Performed in the Routine Clinical Setting in Sarcoma
              </a>
            </td>
          <td>
            E. Nakata, D. Ennishi, Tatsunori Osone, K. Ninomiya, Shuta Tomida, Takuto Itano, T. Fujiwara, T. Kunisada, Naoyuki Ida, Hideki Yamamoto, Mashu Futagawa, T. Shimoi, Hiroyuki Yanai, Akira Hirasawa, S. Toyooka, Masahiro Tabata, Toshifumi Ozaki
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e7464bc295a39dadfcf5089a53f0d09a8ca29a" target='_blank'>
              High-throughput profiling of chemical-induced gene expression across 93,644 perturbations.
              </a>
            </td>
          <td>
            Lei Xiang, Yumei Wang, Wei Shao, Qingzhou Li, Xiankuo Yu, Mingming Wei, Yu Gui, Shengrong Li, Pan Qin, Chao Hu, Guochen Zhang, Xianwen Zhang, Jiawen Wang, Yingying Li, Jun An, Yan Luo, Yile Liao, Jinghong Deng, Xinran Tai, Richard Y Xu, Lijun Huang, Dale Guo, Guanbin Zhang, Zhi Xie, Yun Deng, Junquan Xu, Dong Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96cb29c7c4ee18f843780f85e8c5d424d7abe77" target='_blank'>
              Epigenetic modulation in cancer drug discovery: promising targets and clinical applications.
              </a>
            </td>
          <td>
            S. Vickram, Shofia Saghya Infant, M. Sivasubramanian, Saravanan Anbalagan, Mathan Muthu Chinnakannu Marimuthu, Hitesh Chopra
          </td>
          <td>2025-09-02</td>
          <td>Pharmacological reports : PR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor that requires challenging treatments. Tumor Treating Fields (TTFields), an FDA-approved therapy for glioblastoma (GBM), pleural mesothelioma, and platinum-refractory metastatic nonsmall cell lung cancer (in combination with PD-1/PD-L1 inhibitors or docetaxel), employs specific frequency electric fields to disrupt cell division and enhance treatment efficacy. However, their molecular mechanisms remain unclear. This study aimed to elucidate these mechanisms and optimize the therapeutic potential of TTFields through quantitative proteomics, phosphoproteomics, and glycoproteomics. Pathway analysis of the proteomics revealed that TTFields impact the cell cycle, DNA repair, autophagy, and DNA replication. Phosphoproteomic studies further demonstrated a marked decline in the activity of key kinases ABL1 and PDK1, while glycoproteomics highlighted disruptions in cell adhesion and ECM-receptor interactions. Notably, proteomic analysis identified an upregulation of PARP1 and BRD4 protein levels, suggesting a previously unrecognized resistance mechanism. Consistently, combining TTFields with inhibitors targeting these proteins significantly enhanced the treatment efficacy in U87 cells. Thus, this study uncovers comprehensive molecular mechanisms underlying TTFields' effects on GBM cells and supports the development of concomitant therapies to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23e36a333192c5464239bf92b14bd9554e97f70e" target='_blank'>
              Multiomics Insights into the Mechanism and Enhanced Efficacy of Tumor Treating Fields (TTFields) Therapy in Glioblastoma.
              </a>
            </td>
          <td>
            Yanmei Gao, Chenxu Wang, Guobin Liu, Ruilin Zhang, Xuelian Ren, Guangyuan Hu, Qi Mei, He Huang
          </td>
          <td>2025-09-01</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573d73f70f9bc8245cb6e46c84579fa69539af01" target='_blank'>
              Role of tumor microenvironment in cancer promotion, development of drug resistance and cancer treatment.
              </a>
            </td>
          <td>
            Duaa e Fathah, S. Ejaz
          </td>
          <td>2025-09-15</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The serrated pathway represents a significant route to colorectal cancer (CRC), accounting for approximately 15–30% of cases, yet the specific epithelial cell subpopulations driving this pathway remain poorly understood. This study explores the causal relationship between serrated epithelial cells and CRC risk using single-cell transcriptomics and Mendelian randomization (MR). Publicly available single-cell RNA sequencing data were utilized to analyze epithelial cell subpopulations in CRC, focusing on specific serrated cells (SSCs). By integrating genome-wide association study data, MR was employed to assess the causal relationship between gene expression patterns and CRC risk. The study found that an increase in SSCs is closely associated with CRC progression. MR analysis revealed a significant correlation between expression changes in specific genes, such as IER3 in SSCs, and CRC risk (p < 0.05). Functional analyses indicated that IER3 may promote malignancy by regulating cell proliferation, adhesion, and immune evasion. Several genetic loci related to SSC gene expression were identified and validated for CRC risk association. This study demonstrates the significant role of serrated epithelial cell subpopulations in CRC development, particularly through key genes such as IER3, providing new perspectives for understanding CRC pathogenesis and future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd087daa1599bd00592ced3c9e805ceb0377196a" target='_blank'>
              Single-Cell Dissection of the Serrated Pathway: Cellular Heterogeneity and Genetic Causality in Colorectal Cancer
              </a>
            </td>
          <td>
            Ming Cen, Yunhan Wen, Zhijun Feng, Yahai Shu, Chuanxia Hu
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome drives cancer metabolic reprogramming.
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7263a903e4588231be1bcb20d71d5c5ee5801de5" target='_blank'>
              Transcriptome-wide Mendelian randomization exploring dynamic CD4+ T cell gene expression in colorectal cancer development.
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, Annika Lindblom, S. Ogino, Veronika Vymetálková, A. Wolk, Anna H Wu, J. R. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E Thomas, Rish K. Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-09-17</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ABSTRACT Glioblastoma is a highly aggressive and devastating brain malignancy with dismal prognosis and extremely limited therapeutic options. Identification of prognostic biomarkers and therapeutic targets from multi‐omics data is critical for improving patient outcomes. In this study, we investigated the clinical significance of cellular heterogeneity and super‐enhancer‐driven regulatory networks, which are critically implicated in glioblastoma progression and treatment resistance. We first performed scRNA‐seq to dissect tumour microenvironment heterogeneity, identifying 16 distinct cell clusters, including astrocytes, macrophages, and CD8+ T cells. CellChat analysis revealed key intercellular signalling pathways, with astrocytes and macrophages acting as central communication hubs. To integrate bulk RNA sequencing data, we applied the Scissor algorithm to identify survival‐associated cell states. By combining single‐cell and bulk transcriptomic data, we uncovered 642 survival‐related genes, including QKI and RBM47, which robustly predicted patient survival and immunotherapy response. Furthermore, WGCNA analysis identified seven co‐expression modules and super enhancer‐regulated networks orchestrated by transcription factors (RFX2, RFX4) and hub genes (NEAT1, CFLAR). These networks stratified patients into high‐ and low‐risk groups with significant survival differences. Collectively, our findings elucidate the intricate interplay between cellular heterogeneity and super enhancer‐driven gene regulation in glioblastoma, providing a translational framework for targeting oncogenic hubs and modulating microenvironment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7616587b7549f42937faeb12de6e3dcf343225e7" target='_blank'>
              Integration of Single‐Cell RNA and Bulk RNA Sequencing Reveals Cellular Heterogeneity and Identifies Survival‐Associated Regulatory Networks in Glioblastoma
              </a>
            </td>
          <td>
            Zijun Xu, Bohan Xi, Jiaming Huang, Liqiang Zhang, Sifu Cui, Xianwei Wang, Dong Chen, Shupeng Li
          </td>
          <td>2025-08-13</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311b7e5ef3cdd7a5e15487d2af0221a920cc2b9a" target='_blank'>
              Integrated Analysis of Bulk and Single-Cell Transcriptomics Identifies Prognostic Biomarkers in Breast Cancer Metastasis.
              </a>
            </td>
          <td>
            Qi-Qiao Wu, Kun Liu, Jian-Fang Xu, Yi Zhang, Jun-Rong Jiang, Hui-Lin Wang, Lin-Feng Wang, Jia-Na Zhou, Juan Liu, Xin Lin, Huan Chen, Ying-Ying Guan, Ping Yang, Jing Sun, Wei Wu
          </td>
          <td>2025-08-19</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction RNA sequencing (RNA-seq) can measure whole transcriptome gene expression from tissues or even individual cells, providing a powerful tool to study the immune response. Analysis of RNA-seq data involves mapping relatively short sequence reads to a reference genome, and quantifying genes based on the position of alignments relative to annotated genes. While this is usually robust, genetic polymorphism or genome/annotation inaccuracies result in genes with systematically missing or inaccurate data. These issues are frequently hidden or ignored, yet are highly relevant to immunologic data, where balancing selection has generated many polygenic gene families not accurately represented in a ‘one-size-fits-all’ reference genome. Methods Here we present nimble, a tool to supplement standard RNA-seq pipelines. Nimble uses a previously developed pseudoaligner to process either bulk- or single-cell RNA-seq data using custom gene spaces. Importantly, nimble can apply customizable scoring criteria to each gene set, tailored to the biology of those genes. Results We demonstrate that nimble recovers data in diverse contexts, ranging from simple cases (e.g., incorrect gene annotation or viral RNA), to complex immune genotyping (e.g., major histocompatibility or killer-immunoglobulin-like receptors). We use this enhanced capability to identify killer-immunoglobulin-like receptor expression specific to tissue-resident memory T cells and demonstrate allele-specific regulation of MHC alleles after Mycobacterium tuberculosis stimulation. Discussion Combining nimble data with standard pipelines enhances the fidelity and accuracy of experiments, maximizing the value of expensive datasets, and identifying cellular subsets not possible with standard tools alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1616a4aa1b827317ffb6664ba13b3dacd9a3e2" target='_blank'>
              An immune-focused supplemental alignment pipeline captures information missed from dominant single-cell RNA-seq analyses, including allele-specific MHC-I regulation
              </a>
            </td>
          <td>
            Sebastian Benjamin, G. McElfresh, Maanasa Kaza, Gregory J. Boggy, B. Varco-Merth, Sohita Ojha, Shana Feltham, William Goodwin, Candice Nkoy, D. Duell, Andrea N. Selseth, Tyler Bennett, Aaron M. Barber-Axthelm, Nicole N Haese, Helen Wu, Courtney Waytashek, Carla Boyle, Jeremy Smedley, C. Labriola, Michael Axthelm, R. K. Reeves, D. Streblow, J. Sacha, A. Okoye, Scott G. Hansen, Louis J. Picker, B. Bimber
          </td>
          <td>2025-08-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin Wlodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A. Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
In the era of precision medicine, molecular biomarker testing is increasingly becoming standard of care for Non-Small Cell Lung Cancer (NSCLC) patients. Tissue and liquid biopsy-based Next-Generation Sequencing (NGS) is now highly recommended.


AREAS COVERED
Different NGS platforms emerged as a cost-effective strategy to perform a massive and parallel sequencing performing higher technical sensitivity than old generation technologies in detecting low abundant alterations in challenging diagnostic samples. NGS systems can detect single nucleotide variants (SNV), small insertions and deletions (indels), copy number alterations (CNAs) and structural variants (SVs) or gene fusions across selected druggable genes optimizing clinical administration of NSCLC patients. The diagnostic implementation of the most adequate NGS panel depending on several factors that could impact on the clinical utility of testing assay.


EXPERT OPINION
Promising advanced technologies are emerging as potentially integrative tools in personalized medicine. In this context, multi-omic evaluation including genomic, transcriptomic, fragmentomic and epigenomic signatures are under investigation to significantly modify clinical algorithm of NSCLC patients. On this basis, sequencing strategies may play a pivotal role in the implementation of a new predictive model for cancer diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95aabf8289adb3992031b4fb1935f0065d48a17f" target='_blank'>
              Innovation in next-generation sequencing in non-Small cell lung cancer diagnostics.
              </a>
            </td>
          <td>
            C. Scimone, Lucia Palumbo, Roberto Borea, Claudia Sarracino, Ilaria Tomaiuolo, Domenica di Giovanni, Sabrina Alfano, Mariantonia Nacchio, G. Russo, A. Russo, F. Pepe, G. Troncone, U. Malapelle
          </td>
          <td>2025-08-18</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="ABSTRACT Multiple myeloma (MM) remains incurable; gaps in our understanding of MM molecular pathogenesis and drugs’ resistance mechanisms are involved in the failure of therapies. This study aims to identify proteins significantly impacting MM patients’ response to commonly used therapeutic regimens. Bone marrow CD138+ selected plasma cells were isolated from patients who had achieved Response (Responders, R) and those who were Non‐Responders (NR) to their primary MM therapy. We used LC‐MS/MS to investigate the proteomic profile of MM samples, followed by bioinformatics analysis. We identified 1190 proteins, of which 230 showed a statistically significant difference between R and NR, with 27 proteins being upregulated and 203 downregulated in R compared to NR. Pathway enrichment analysis identified pathways related to the immune response and protein synthesis regulation, closely associated with MM progression and response to therapy. The results were validated through individual RNA dataset analysis, corroborating the differential expression of several proteins, including proteins associated with MM (e.g., MIF, ILF3) as well as novel findings (e.g., DCPS and SET). Collectively, proteomics data obtained from R and NR to MM therapy displayed significant changes in the immune system and protein synthesis regulation, supporting their potential role in progression and therapeutic response of MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da55e0e2199913a64ea869848da54bd159ba5889" target='_blank'>
              Proteomic Analysis of Bone Marrow CD138+ Cells to Identify Proteins Associated With the Response of Multiple Myeloma Patients to Commonly Used Therapeutic Regimens
              </a>
            </td>
          <td>
            Foteini Paradeisi, A. Tserga, V. Lygirou, M. Makridakis, R. Stroggilos, Grigoris Georgiou, George M. Spyrou, Ioannis V. Kostopoulos, C. Liacos, Aikaterini Termentzi, M. Dimopoulos, O. Tsitsilonis, A. Vlahou, E. Kastritis, J. Zoidakis
          </td>
          <td>2025-08-01</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b08abb53b00ad74c46d78193611643fc952b7c2" target='_blank'>
              Organoid-evaluable clinical biomarkers predict drug responses and guide new breast cancer therapies.
              </a>
            </td>
          <td>
            T. B. Bui, Denise M Wolf, Michael C Bruck, Kaitlin Moore, Jessica Lien, Sarah D W Choi, Shruti Warhadpande, Amirabbas Parizadeh, Deborah Dillon, B. Overmoyer, F. Lynce, Isaac J Nijman, Boudewijn M T Burgering, Isaac S Harris, Laura J. Esserman, Laura J van 't Veer, Jennifer M. Rosenbluth
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Introduction Lynch Syndrome accounts for 1–7% of all colorectal cancers and is caused by germline mutations in DNA mismatch repair (MMR) genes. Timely molecular diagnosis is crucial for effective genetic counseling and management. Among understudied Latin American populations, Uruguay’s genetic admixture provides an opportunity to identify novel Lynch Syndrome related variants. Methods This study analyzed 70 unrelated Uruguayan colorectal cancer patients meeting Lynch Syndrome clinical criteria to identify carriers of pathogenic variants. A customized Next-Generation Sequencing (NGS) panel was developed and sequenced on the Ion Torrent platform to analyze nine genes: MLH1, MSH2, MSH6, EPCAM, FAN1, MUTYH, PMS1, PMS2, and APC. Copy number variations and large EPCAM deletions are not detected by the assay. Gene variants were prioritized based on allelic frequency, in silico predictions, pathogenicity records, and ACMG guidelines. The performance of this custom NGS panel was evaluated for in-house applications, and its limitations were thoroughly assessed. Results and discussion The custom NGS panel demonstrated effectiveness for scalable in-house testing despite minor disclosed sequence coverage limitations. Pathogenic and likely pathogenic variants were identified in 25 patients, including four novel Lynch Syndrome-associated variants. In four patients, a rare ambiguously classified gene variant co-occurs with a known pathogenic variant in another gene. The mutation profile correlated with clinical parameters such as age of diagnosis, diagnosis criteria, tumor location, and microsatellite instability (MSI). Conclusion This is the most comprehensive genetic study to date on a Uruguayan Lynch syndrome cohort. The mutational landscape aligns with findings in other populations while highlighting novel variants of clinical relevance. These findings highlight the value of customized panels for improving genetic screening in small-scale healthcare facilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a87e92229b92284618b34a18aaa8e6552d753f6" target='_blank'>
              Genetic characterization of Lynch syndrome germline variants in a LATAM cohort using a customized NGS gene panel
              </a>
            </td>
          <td>
            Cecilia Mathó, Santiago Chávez, R. Fort, A. Della Valle, F. Neffa, J. Sotelo-Silveira, Nora Artagaveytia, M. A. Duhagon
          </td>
          <td>2025-08-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The fusion of nanotechnology with gene editing promises a revolutionary strategy in combating cancer, providing the possibility of precise and focused treatments. This review examines the synergistic integration of these two potent technologies, specifically emphasising their combined effectiveness in oncological therapies. Nanotechnology offers a flexible framework for administering gene-editing tools, improving their accuracy, and reducing unintended side effects, all of which are significant obstacles in existing cancer treatments. Nanoparticles can improve the effectiveness of therapies, lower the risk of systemic toxicity, and allow the simultaneous manipulation of many genetic pathways involved in cancer growth by delivering CRISPR-Cas9 and other gene-editing systems directly to tumour sites. We conduct a thorough analysis of recent progress in this burgeoning field, emphasising significant advancements in the design of nanoparticles and gene-editing techniques that propel the development of next-generation cancer medicines. In addition, we address the present obstacles and constraints, such as the effectiveness of delivery, apprehensions over safety, and regulatory obstacles, while suggesting potential areas of future research to surmount these barriers. This study thoroughly examines the promise of nano-precision gene editing as a transformative approach to cancer treatment by incorporating findings from recent clinical trials and case studies. By highlighting recent clinical advancements and emerging innovations, this review underscores the potential of nano-precision gene editing as a groundbreaking approach in next-generation cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a93a0064afa2aa4000c185a6eb3a8730afcf15" target='_blank'>
              Harnessing Nanotechnology and Gene Editing for Cancer Therapy: A Synergistic Approach to Precision Medicine.
              </a>
            </td>
          <td>
            A. Goel, Istuti Saraswat, Sahaj Sharma, Rijul Joshi
          </td>
          <td>2025-08-01</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epigenetic changes, such as DNA methylation, chromatin remodeling, and histone modifications, regulate gene expression without altering the DNA sequence. This review systematically analyzed over 500 studies including human cell line experiments (n>200), animal models (n>50), clinical cohort studies (n>100), and bioinformatics analyses retrieved from PubMed, Web of Science, and TCGA (The Cancer Genome Atlas). Studies increasingly show that genes involved in glucose and lipid metabolism, energy production, and modulation of metabolic hormones are regulated through epigenetic mechanisms. On the other hand, various metabolites participate in epigenetic modifications as coenzymes or substrates. Therefore, a greater understanding of the crosstalk between metabolism and epigenetics in cancer-related pathways could lead to the identification of key signaling molecules for targeted therapies, and raise the possibility of using dietary interventions to modulate epigenetic markers for individualized treatment. In this review, we have summarized the metabolic and epigenetic regulatory networks in cancer development, including glycolipid metabolic reprograming, the role of metabolites produced by the glut flora and tumor microenvironment, and key epigenetic drivers such as non-coding RNAs (ncRNAs). Data were curated from peer-reviewed articles, grounded in mechanistic studies using cell lines (SW480, MCF7 (Michigan cancer foundation-7)) and animal models (APC-mutant mice), with a focus on mechanistic studies, omics analyses, and translational research. Furthermore, we have discussed the potential of therapeutically targeting these pathways, along with the current challenges and future research directions, and a new strategy for reversing therapeutic drug resistance based on metabolism and epigenetic interaction was systematically explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447a75110b1468dd9ba2d78ee2627a7a828fee6" target='_blank'>
              Metabolism and epigenetics in cancer: toward personalized treatment
              </a>
            </td>
          <td>
            Xiaoman Zhang, Dequan Liu, Sulan Yin, Yaru Gao, Xiaorui Li, Guangzhen Wu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Targeted DNA-based comprehensive genomic profiling (CGP) to detect clinically significant alterations is increasingly becoming standard for patients with advanced or recurrent cancer. RNA-based sequencing, however, may improve performance of fusion detection. We developed a robust targeted RNA sequencing assay (FoundationOne®RNA) and evaluated its analytic performance. FoundationOne®RNA is a hybrid-capture based targeted RNA sequencing test designed to optimally detect fusions (318 genes) and measure gene expression (1521 genes). Analytical validation studies were performed in College of American Pathologists (CAP)-accredited and Clinical Laboratory Improvement Amendments (CLIA)-certified lab to assess fusion call accuracy, assay reproducibility, limit of detection (LoD) and gene expression in 189 clinical solid tumor specimens. In the accuracy study, 160 out of 189 biopsy samples which were previously profiled using large-panel DNA- or RNA-based next-generation sequencing (NGS) passed quality control metrics and were studied using the FoundationOne®RNA assay. Analysis of all diagnostic fusions showed a positive percent agreement (PPA) of 98.28%, as well as a negative percent agreement (NPA) of 99.89% when compared to orthogonal assays. The FoundationOne®RNA assay was able to identify a low level BRAF fusion missed by orthogonal whole transcriptome RNA sequencing and was confirmed by fluorescence in situ hybridization (FISH). The range for the minimum RNA input and LoD was determined based on dilutions from 5 fusion-positive cell lines. It spans from 1.5ng (0.5% input) to 30ng (10% input) for RNA input and from 21 to 85 supporting reads for LoD. In the precision study, 10 out of 10 pre-defined target fusions had 100% reproducibility. In our gene expression analysis, multiple gene expression signatures were detected in fusion positive samples. FoundationOne®RNA assay successfully detected oncogenic fusions with high concordance to orthogonal NGS based tests, high reproducibility, and low limit of detection. This study demonstrated the robustness of FoundationOne®RNA and supports its use as a supplement to tissue DNA comprehensive genomic profiling (CGP) in routine clinical practice. Additional work is required to clarify optimal clinical scenarios for fusion detection and enable gene expression biomarkers for clinical use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0548e407f5f836fa742cb029c76e09f54d80d2da" target='_blank'>
              Analytical validation (accuracy, reproducibility, limit of detection) and gene expression analysis of FoundationOneRNA assay for fusion detection in 189 clinical tumor specimens
              </a>
            </td>
          <td>
            Daokun Sun, Richard S P Huang, Michelle Green, Chenming Cui, S. Sisoudiya, Andrej Savol, Chang Xu, Cui Guo, J. Skoletsky, Justin M Allen, Khaled Tolba, Varun Pattani, Alyssa Tarzia, Jennifer Whiting, Yanhua Tang, L. Albacker, C. Vietz, Michael Datto, Jie He, W. R. Jeck
          </td>
          <td>2025-09-12</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, Hafiz Muhammad Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="T cell receptors (TCRs) orchestrate adaptive immunity, yet the complex, repetitive architecture of the TCR loci has impeded systematic characterization of human genetic variation in the genes encoding the TCR. Using public long-read sequencing data from 2,668 donors, we build a near-complete map of common alleles in TCR V, D, and J genes, revealing amino acid variation at almost every position within V genes. We discover pervasive evidence of natural selection on TCR genes, including balancing selection on a TRAJ gene recognizing an immunodominant influenza epitope and positive selection on a TRAV gene. We find TCR allelic polymorphism alters core functional properties of T cells, including thymic fate commitment, phenotypes in diseased tissues, and cell-surface receptor abundance. Collectively, our findings position inherited variation in TCR genes as a key axis of immunological diversity that may shape interindividual differences in immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d398484bfbdbd87bb2aee53bb4ed8e33eec07466" target='_blank'>
              TCR germline diversity reveals evidence of natural selection on variable and joining alpha chain genes
              </a>
            </td>
          <td>
            S. Mantena, Ali Akbari, Soumya Raychaudhuri
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Background Cervical cancer remains a global public health challenge, particularly in regions with limited access to screening and vaccination. While high‐risk HPV infection is the primary cause, the genetic and molecular mechanisms driving cervical cancer progression are not fully understood. Objective This study integrates Mendelian randomization (MR) and single‐cell RNA sequencing (scRNA‐seq) to identify causal eQTL‐related genes and explore their roles in tumorigenesis. Functional experiments were conducted to validate key findings. Methods MR analysis identified eQTL‐related genes with significant causal associations with cervical cancer. Functional enrichment and Gene Set Variation Analysis (GSVA) revealed their involvement in key pathways. scRNA‐seq explored cell‐specific expression patterns and immune cell infiltration in the tumor microenvironment (TME). In vitro experiments, including qRT‐PCR, siRNA knockdown, migration, proliferation, and colony formation assays, validated the biological roles of pivotal genes. Results A total of 307 eQTL‐related genes were identified, enriched in pathways such as Th17 cell differentiation, TNF, and IL‐17 signaling. scRNA‐seq revealed cell‐specific expression of key genes, including FUOM, which was elevated in cervical cancer cells. FUOM knockdown significantly reduced cell proliferation (by 37%, p < 0.001), migration (by 43%, p < 0.001), and colony formation (by 62%, p < 0.001). Regulatory analysis identified miRNAs as upstream modulators of these genes. Conclusion This study identifies FUOM as a novel driver gene in cervical cancer progression and highlights its role in tumorigenesis and immune modulation. These findings provide insights into potential biomarkers and therapeutic targets, offering a foundation for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f2af3648a1778b7bc1979bb8e9f7cee9b957bcb" target='_blank'>
              Integrative Genomic and Functional Approaches Identify FUOM as a Key Driver and Therapeutic Target in Cervical Cancer
              </a>
            </td>
          <td>
            Wenzhi Jiao, Shanshan Liu, Jianwei Shi, Minmin Yu
          </td>
          <td>2025-08-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose: Although the combination of androgen deprivation therapy with docetaxel or abiraterone acetate and prednisone has become a standard treatment for metastatic hormone-sensitive prostate cancer (mHSPC) and has shown improved overall survival, a subset of patients still progress to castration-resistant disease. However, the underlying molecular features in these patients remain poorly understood.Materials and Methods: We performed single-cell RNA sequencing (scRNA-seq) on 12 tissue samples, including 2 mHSPC samples, 7 primary prostate cancer (PCa) samples, and 3 matched normal samples from 7 patients. Raw sequencing data was processed by Cell Ranger software (10X Genomics) and aligned to the human reference genome (GRCh38). A comprehensive analysis of samples from patients with mHSPC was also conducted and validated using a cohort of 52 patients with mHSPC.Results: Our results identified distinct subpopulations within luminal and mononuclear phagocyte (MNP) clusters characterized by proliferative activation associated with unfavorable clinical outcomes. Furthermore, we observed that the MNP cluster exhibited significant proliferation activity. To understand the underlying mechanisms associated with aggressiveness, we conducted cell-cell interaction, copy number variation and pseudotime analysis. Utilizing 87 network genes from scRNA-seq, we classified 52 mHSPC patients into 2 molecular subtypes and demonstrated their correlation with distinct transcriptomic profiles and survival outcomes. Importantly, a 14-gene signature derived from 3 distinct gene sets exhibited a strong association with patient survival and drug response. The prognostic value of our findings was further validated in large-scale cohorts comprising localized PCa, metastatic PCa, mHSPC, and metastatic castration-resistant PCa patients.Conclusion: This study provides valuable insights into the identification of high-risk patients, novel biomarkers, and potential therapeutic targets for individuals with mHSPC. Furthermore, the results in this study can serve as a basis for future investigations aimed at refining prognostic strategies and developing targeted therapies for patients with mHSPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450db38b74f006113af6f667e3cedb5cf553cec7" target='_blank'>
              Determinant of Aggressive Phenotype in Metastatic Hormone-Sensitive Prostate Cancer Depends on an Intrinsic, Highly Aggressive Cell Cluster: Integrated Single-Cell RNA and Whole Transcriptomic Sequencing Analyses
              </a>
            </td>
          <td>
            Minyong Kang, B. Jee, Jiwoong Yu, Soohyun Hwang, Kyunghee Park, Kyung Yeon Han, Wan Song, H. Sung, H. Jeon, Byong Change Jeong, S. Seo, Se Hoon Park, Woong-Yang Park, S. Jeon
          </td>
          <td>2025-07-31</td>
          <td>Journal of Urologic Oncology</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Lenvatinib, a multiple-receptor tyrosine kinase inhibitor, has gained recent approval for its use as a first-line treatment of hepatocellular carcinoma (HCC). While lenvatinib demonstrates notable therapeutic efficacy, the drug resistance undermines its sustained tumor control potential. The restricted clinical utility of lenvatinib underscores the imperative necessity to elucidate the mechanisms underpinning drug resistance. We established lenvatinib-resistant cell lines and investigated the changes in their biological characteristics. Next-generation sequencing was performed to identify genes associated with lenvatinib resistance. Western blots were utilized to confirm the involvement of these genes. Using lentiviral technology, we generated cell lines with lowered nuclear receptor coactivator 5 (NCOA5), a pivotal drug resistance-related gene, to explore the underlying resistance mechanism. Moreover, we developed a subcutaneous HCC xenograft tumor model to explore strategies for reversing drug resistance. Our study showed that HCC cells acquire resistance to lenvatinib through the activation of NCOA5, thereby stimulating the NCOA5-Protein Kinase B-mammalian target of rapamycin (AKT-mTOR) axis. Furthermore, the clinical evaluation of HCC specimens established a correlation between the activation of the NCOA5 pathway and the response to lenvatinib treatment. Everolimus, an mTOR inhibitor, in combination with lenvatinib and everolimus, exerted significant synergistic effects against HCC in vivo and in vitro. HCC cells develop resistance to lenvatinib by activating the NCOA5-AKT-mTOR pathway. The combination therapy of lenvatinib with everolimus is a promising strategy to overcome acquired resistance, thereby enhancing the clinical efficacy of lenvatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4645ca240d89c81caab05ab94ee824d557e8c782" target='_blank'>
              Inhibition of nuclear receptor coactivator 5 overcomes acquired lenvatinib resistance driven by protein kinase B-mammalian target of rapamycin signaling in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Hongyuan Zhou, Qin Zhang, Lu Yang, Zhao-Long Pan, Haijing Zheng, Zewu Zhang, Dongyang Li, Guangtao Li, Xiaomeng Liu, Xu Bao, Chen Liu, Wei Zhang
          </td>
          <td>2025-07-31</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is still a huge problem because of the complication of genetic heterogeneity and emergence of drug resistance which leads to poor therapeutic responses. Conventional drug development is usually restricted in their capacity to discover the exact molecular target and anticipate the drug responses. This is because the integration of CRISPR-based genome editing technologies into cancer drug screening does two main services: it offers a new direction of discovering gene-drug interactions and targeting against drugs. The study has used A549, MCF-7, and HCT116 cancer cell lines to examine how the consequences of knocking out and activating certain genes (KRAS, TP53, MYC, PTEN, and EGFR) affect a cell in terms of response to chemotherapeutic agents ( d o x rubicin, paclitaxel and c is platinum ). The design of the experiment was the inclusion of drug treatments at different concentrations, cell viability assays, measurement of gene expression and quantifying the protein. The statistical tests, such as ANOVA and post-hoc testing, were used in order to determine the statistical significance of the changes. The data showed that KRAS knockout sensitized the A549 cells to doxorubicin (IC50 = 0.35 10 9 ) and TP53 knockout rendered the cells resistant (IC50 = 1.50 10 9 ). MYC CRISPR interference decreased sensitivity to paclitaxel whereas MYC activation increased it. PTEN knockout (in MCF- 7) rendered cells more susceptible to cisplatin and activation induced resistance. Pearson correlation analysis demonstrated that there were significant correlations between the change in expression of the genes and cell viability indicating the functional relevance of the CRISPR alterations. The presented study also notes that CRISPR-based functional screening can be a promising field of precision oncology and can help to understand the molecular underpinning of drug resistance and discover new targets of therapeutics against cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6621891a1e50692e473d10afd771d57415d90fde" target='_blank'>
              CRISPR-BASED GENOME EDITING TECHNOLOGIES IN CANCER RESEARCH AND THEIR APPLICATIONS IN FUNCTIONAL DRUG SCREENING
              </a>
            </td>
          <td>
            Aiman Nishat, Ahmad Ashraf, Tahir Hasan, Abrar Hussain, Muhammad Afaq, Haseeb Khaliq, Muhammad Kamran Tahir, Abul Hassan Khan, Gul Sika
          </td>
          <td>2025-08-14</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding how genes influence drug responses is critical for advancing personalized cancer treatments. However, identifying these gene-drug interactions in a physiologically relevant human system remains a challenge, as it requires a model that reflects the complexity and heterogeneity among individuals. Here we show that large-scale CRISPR-based genetic screens, including knockout, interference (CRISPRi), activation (CRISPRa), and single-cell approaches, can be applied in primary human 3D gastric organoids to systematically identify genes that affect sensitivity to cisplatin. Our screens uncover genes that modulate cisplatin response. By combining CRISPR perturbations with single-cell transcriptomics, we resolve how genetic alterations interact with cisplatin at the level of individual cells and uncover an unexpected link between fucosylation and cisplatin sensitivity. We identify TAF6L as a regulator of cell recovery from cisplatin-induced cytotoxicity. These results highlight the utility of human organoid models for dissecting gene-drug interactions and offer insights into therapeutic vulnerabilities in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/306fe56d4d5b9a4de12be73a14dcdbee9c2f3137" target='_blank'>
              Large-scale CRISPR screening in primary human 3D gastric organoids enables comprehensive dissection of gene-drug interactions
              </a>
            </td>
          <td>
            Yuan-Hung Lo, Hudson T Horn, Mo-Fan Huang, Wei-Chieh Yu, Chia-Mei Young, Qing Liu, Madeline Tomaske, Martina Towers, Antonia Dominguez, M. Bassik, Dung-Fang Lee, L. S. Qi, Jonathan S. Weissman, Jin Chen, Calvin J. Kuo
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="Background and Objective Parental chromosomal structural variations (SVs) represent a primary genetic factor contributing to recurrent spontaneous abortion (RSA). Individuals carrying SVs with complex chromosomal rearrangements (CCRs) typically exhibit a normal phenotype but are at an increased risk of miscarriage. Current standard clinical detection methods are insufficient for the identification and interpretation of all SV types, particularly complex and occult SVs, thereby presenting a significant challenge for clinical genetic counseling. Leveraging the high-resolution capabilities of optical genome mapping (OGM) technology, this study aims to rapidly and accurately identify complex SVs in RSA couples. Furthermore, it seeks to conduct an in-depth analysis of the genetic information within the breakpoint regions, thereby providing a more comprehensive scientific foundation for genetic counseling of RSA couples at both the cellular and genetic levels. Material and Methods This study involved the selection of nine subjects from two families who underwent genetic counseling at our hospital. Family 1 comprised a couple with the wife as a SVs carrier, and both her parents and brother were simultaneously analyzed for chromosomal karyotype. Family 2 included a couple with the husband as the SVs carrier, with his parents also undergoing chromosomal karyotype analysis. For SVs carriers whose karyotype analysis did not elucidate the recombination pattern, optical genome mapping (OGM) technology was utilized for further investigation, followed by Sanger sequencing to validate the OGM findings. Results In Family 1, only the wife was identified as an SVs carrier. Initial chromosomal karyotype analysis suggested a karyotype of 46,XX,t (5; 6;8; 13; 15) (?). However, OGM analysis ultimately confirmed the karyotype as 46,XY,der (5)t (5; 13) (q35.2; q21.32), der (6)t (6; 8) (q25.3; q13.1)ins (6; 13) (q25.3; q21.32q21.33),der (8)t (6; 8) (q26; q13.1)ins (8; 13) (q13.1; q21.33q22.1),der (13)t (13; 15) (q21.32; q26.1)ins (13; 6) (q21.32; q25.3q26), der (15)t (5; 15) (q35.2; q26.1). Furthermore, OGM identified a novel translocation variant of the KIF7 gene that is associated with recurrent miscarriage. In Family 2, both the husband and his maternal parent were identified as SVs carriers. Nuclear type analysis revealed a karyotype of 46,XY,?t (1; 6) (q42; p21) (husband) and 46,XX,?t (1; 2) (p31.1; q24.1),?t (1; 6) (q42; p21) (mother). Through OGM detection and analysis, the final karyotype was determined to be 46,XY,ins (1; 6) (q42.2; p22.3p11.3) (husband) and 46,XX,der (1)t (1; 2) (p31.1; q24.1)ins (1; 6) (q42.2; p22.3p11.3), der (2) t (1; 2), der (6)ins (1; 6) (mother). Conclusion OGM technology facilitates the rapid and precise identification of complex chromosomal structural variations, effectively overcoming the limitations associated with traditional karyotype G-banding techniques in detecting intricate and cryptic SVs. This advancement substantially enhances the diagnostic rates of genetic etiology in patients experiencing RSA. The present study elucidates the specific manifestations of complex SVs using OGM technology, accurately pinpointing breakpoints and interpreting affected gene information. This provides novel reference approaches and evidence for disease assessment and genetic counseling in RSA patients. However, it is important to acknowledge certain limitations of this research: the study’s inclusion of only two RSA family cohorts (comprising nine participants) may limit the generalizability of its conclusions due to the small sample size, necessitating further validation through large-scale studies. Additionally, the causal relationship between KIF7 gene dysfunction and recurrent miscarriage remains to be experimentally verified in subsequent research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c823d1db805757e9a370ed45922b739e08e31bb6" target='_blank'>
              Analysis of complex chromosomal structural variants through optical genome mapping integrated with karyotyping
              </a>
            </td>
          <td>
            Xiaoxi Zhu, Huiling Zheng, Xue Wan, Hang Duan, Ying Qi, Weijia Tang, Fan Yang, Limei Yu
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) critically regulate tumor progression, angiogenesis, metastasis, and therapeutic resistance. This study investigated the characteristics of CAFs in glioblastoma (GBM) and developed a CAF-based risk signature to predict patient prognosis. The single-cell RNA sequencing (scRNA-seq) data were sourced from the Gene Expression Omnibus (GEO) database, whereas the bulk RNA-seq datasets were retrieved from The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA), respectively. The Seurat R package processed scRNA-seq data to identify CAF clusters using established markers. Prognostic genes were screened through univariate Cox regression, with Lasso regression constructing the final risk model. A nomogram incorporating clinical parameters was subsequently developed. Immunohistochemical validation was performed using the Human Protein Atlas (HPA) for the signature genes. The qRT-PCR validation was conducted in U251 and HA cells. ScRNA-seq analysis revealed five CAF clusters in GBM, including three prognostically relevant subtypes. Three key genes were refined to construct a risk signature functionally enriched in the the IL6_JAK_STAT3 signaling, P53 pathway, and inflammatory response. The signature correlated strongly with stromal and immune scores. Multivariate analysis confirmed risk signature independent prognostic value (P < 0.0001), followed by age (P = 0.005). The CAF-derived nomogram demonstrated superior predictive accuracy for 1-/2-year survival compared to clinical factors alone. The signature genes were validated in the HPA database and experimental validation. This study proposes CAF-derived molecular signatures as potential predictors of glioblastoma prognosis worthy of clinical validation. Systematic characterization of CAF heterogeneity may offer insights for interpreting GBM immunotherapy responses, providing a foundation for future exploration of stroma-targeted therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00548-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d7a7b7ca4435c9538e14b7aed765c936957688" target='_blank'>
              Integrated single-cell and bulk transcriptomic profiling reveals cancer-associated fibroblast heterogeneity in glioblastoma and establishes a clinically actionable prognostic model and preliminary experimental validation
              </a>
            </td>
          <td>
            Wenhua Zhang, Yaxiong Li, Conghui Li, Qian Huang
          </td>
          <td>2025-08-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cellular senescence is a double-edged sword in cancer biology, functioning as both a tumor-suppressive mechanism and a driver of malignancy. Initially, senescence acts as a protective barrier by arresting the proliferation of damaged or oncogene-expressing cells via pathways such as oncogene-induced senescence and the DNA damage response. However, persistent senescence-associated secretory phenotype and metabolic reprogramming in senescent cells create a pro-inflammatory, immunosuppressive tumor microenvironment, fueling cancer progression, therapy resistance, and metastasis. This comprehensive review systematically examines the molecular mechanisms of senescence across diverse cancers, spanning digestive, reproductive, urinary, respiratory, nervous, hematologic, endocrine, and integumentary systems, and elucidates its context-dependent roles in tumor suppression and promotion. We highlight groundbreaking therapeutic innovations, including precision senolytics, senomorphics, and combinatorial strategies integrating immunotherapy, metabolic interventions, and epigenetic modulators. The review also addresses microenvironment remodeling and cutting-edge technologies for dissecting senescence heterogeneity, epigenetic clocks for biological age prediction, and microbiome engineering to modulate senescence. Despite their promise, challenges such as off-target effects, biomarker limitations, and cellular heterogeneity underscore the need for precision medicine approaches. Finally, we propose future directions to harness senescence as a dynamic therapeutic target, offering transformative potential for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9e8f75a567edbd4415256ca44b46f90ddbb5c2" target='_blank'>
              Cellular senescence in cancer: from mechanism paradoxes to precision therapeutics
              </a>
            </td>
          <td>
            Tiejun Feng, Fuda Xie, Leo M.Y. Lee, Zhiqiang Lin, Yifan Tu, Yang Lyu, Peiyao Yu, Jialin Wu, Bonan Chen, Ge Zhang, Gary M K Tse, K. To, Wei Kang
          </td>
          <td>2025-08-08</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background While the RNA-binding protein HuR is implicated in individual cancers, its comprehensive diagnostic, prognostic, and immunological roles across diverse cancer types remain unexplored. Methods We performed an integrated pan-cancer analysis of HuR using public datasets. This encompassed expression profiling, survival analysis, diagnostic accuracy assessment, immune microenvironment characterization, and drug sensitivity prediction. We investigated HuR’s regulatory mechanisms through pathway correlation and differential gene expression analyses. Results HuR expression was consistently elevated across multiple cancers and correlated with poor patient prognosis. It demonstrated high diagnostic accuracy (>85%) via TMB/PD-L1 biomarkers. High HuR expression was associated with an immunosuppressive tumor microenvironment and reduced efficacy of immune checkpoint inhibitors, establishing it as a key immunoregulatory biomarker. HuR also predicted sensitivity to cell cycle inhibitors and other pathway-targeted drugs. Mechanistically, HuR drives malignancy by dysregulating core processes: cell cycle progression, immune evasion, and cellular metabolism. Conclusions Our pan-cancer analysis establishes HuR as a consistently upregulated oncogenic driver across malignancies, functioning as a potential universal biomarker for prognosis and diagnosis. Its critical roles in modulating the immune response and predicting therapeutic sensitivity highlight its importance for personalized cancer treatment strategies. HuR orchestrates tumorigenesis and malignant progression by integrally regulating vital cellular processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee36dd4154d3b1fc037a47fa32d256e8c8ba2af" target='_blank'>
              Integrated pan-cancer analysis of RNA binding protein HuR investigates its biomarker potential in prognosis, immunotherapy, and drug sensitivity
              </a>
            </td>
          <td>
            Jian Peng, Jichuan Quan, Xiaoru Wang
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="AIDS progression varies widely among individuals, influenced by genetic and molecular factors affecting immune responses and viral replication. Identifying genetic variants associated with disease progression can reveal host-pathogen interactions and potential therapeutic targets. This study analyzes genome-wide association study (GWAS) data and performs functional enrichment analyses to explore key genetic determinants of AIDS progression. GWAS data were analyzed to identify single nucleotide polymorphisms (SNPs), significantly associated with AIDS progression. Functional enrichment through Gene Ontology (GO), KEGG pathways and CellMarker databases classified genes by biological functions, immune cell associations and metabolic pathways. Unsupervised machine learning algorithms, including hierarchical clustering and principal component analysis (PCA) were applied to validate patterns.

Significant SNPs were identified in genes such as PARD3B, LTF, CCRL2, NBPF14, DGKI, RPH3AL and H2AFY, linked to immune regulation, inflammation and viral replication control. PARD3B was associated with slower disease progression while DGKI highlighted metabolic involvement through phosphatidylinositol signaling. Hierarchical clustering revealed natural SNP groupings and PCA confirmed distinct genomic patterns influencing AIDS progression. This study highlights genetic variants influencing AIDS progression and identifies potential biomarkers for future therapeutic strategies. Validation and further exploration of personalized treatment approaches are recommended.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34bf558f9d01070512eaeba6ab1b07218bbc9e77" target='_blank'>
              Genetic and Molecular Insights into AIDS Progression: A Genome-Wide Association Data-Driven Analysis
              </a>
            </td>
          <td>
            Deepthi C. Sravana, Eddula Venkata Ravikanth, U. Adiga, S. Vasishta, B. S. Sindhu
          </td>
          <td>2025-07-31</td>
          <td>Research Journal of Biotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Emerging RAS inhibitors show promise in treating KRAS-mutated malignancies, but resistance mechanisms limit their clinical efficacy. Given recent clinical findings associating KRAS mutations with reduced response to neoadjuvant therapy in rectal cancer (RC), we aimed to investigate their impact on treatment outcomes and explore potential therapeutic strategies.


METHODS
We conducted a retrospective analysis of 390 rectal cancer patients to evaluate the association of KRAS mutations with disease-free survival (DFS) and response to therapy. We assessed the efficacy of KRAS inhibitors in rectal cancer cell lines, patient-derived organoids (PDOs), and patient-derived cell lines (PDCLs), and explored adaptive resistance mechanisms through transcriptomic profiling and unbiased drug screening experiments.


RESULTS
Mutant KRAS was associated with a reduced DFS and RCs harboring G12C and G12V mutations had less complete pathological responses to neo-adjuvant therapies. KRAS-mutated RC cells demonstrated adaptive resistance to KRAS inhibitors, characterized by transcriptomic restoration of oncogenic pathways, including MYC and E2F, and upregulation of ERBB2/3 expression. Consistently, drug screening identified EGFR family inhibitors as potent combinatorial partners, effectively overcoming KRAS inhibitor tolerance by inducing apoptosis. In patient-derived models, the pan-RAS inhibitor RMC-6236 combined with EGFR inhibitors demonstrated significant synergistic effects and prevented long-term tumor cell outgrowth.


CONCLUSION
Our findings point to the negative impact of KRAS mutations, particularly G12C and G12V, on RC treatment outcomes. Adaptive resistance by upregulation of ERBB genes limits the efficacy of KRAS inhibitors. Combining these with pan-ERBB inhibitors enhances anti-tumor effects in patient-derived cellular RC models, showing its potential as an alternative to the combination with anti-EGFR antibodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9be2135f6b5cf4f9e01a3b8ac31ac51a76a3401" target='_blank'>
              Pan-ERBB Inhibitors Synergize With KRAS Inhibitors in Rectal Cancer.
              </a>
            </td>
          <td>
            Jonas Buchloh, M. Spitzner, Hauke Zimmermann, Xin Fang, Constanza Tapia Contreras, Carolin Schneider, Tiago de Oliveria, Stefan Küffer, Michael Linnebacher, Felix Rühlmann, Lena-Christin Conradi, Matthias Wirth, Michael Ghadimi, Marian Grade, Jochen Gaedcke, Günter Schneider
          </td>
          <td>2025-09-16</td>
          <td>United European gastroenterology journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Vascular diseases such as atherosclerosis, aneurysms, and peripheral arterial disease remain leading causes of morbidity and mortality, with current treatments primarily managing symptoms rather than addressing underlying molecular drivers. Gene therapy offers a promising avenue for targeted intervention, and recent advances in multi-omics approaches-including genomics, transcriptomics, and epigenetics-are enhancing the precision and efficacy of these therapies. High-throughput sequencing and integrative omics analyses have facilitated the identification of causal genes, non-coding RNAs, and epigenetic regulators involved in vascular pathology. This review examines how multi-omics frameworks inform gene therapy design, from genomic editing of cardiovascular disease loci to transcriptome-guided RNA therapies and epigenetic modulation of disease states. We highlight emerging applications such as CRISPR-based interventions, RNA therapeutics, and individualized precision medicine strategies. Additionally, we address analytical challenges, implementation hurdles, and ethical considerations in translating multi-omics-driven gene therapies into clinical practice. By integrating systems biology and advanced computational methods, the convergence of multi-omics and gene therapy holds transformative potential for vascular medicine, offering new avenues for disease modification and patient-specific therapeutic solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a11c68f0e7401720c5e3271796816c820e6e436" target='_blank'>
              Multi-Omics Approaches in Gene Therapy for Vascular Diseases: Bridging Genomics, Transcriptomics, and Epigenetics.
              </a>
            </td>
          <td>
            Jianxin Dong, Ming Sun, Yanmei Li, Zhilei Xie
          </td>
          <td>2025-08-05</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common primary bone cancer of mesenchymal origin in children and young adolescents, remains a challenge due to metastasis and resistance to chemotherapy. It displays severe aneuploidy and a high mutation frequency which drive tumor initiation and progression; however, recent studies have highlighted the role of epigenetic modifications as a key driver of OS pathogenesis, independent of genetic mutations. DNA and RNA methylation, histone modifications and non-coding RNAs are among the major epigenetic modifications which can modulate the expression of oncogenes. Abnormal activity of these mechanisms contributes to gene dysregulation, metastasis and immune evasion. Therapeutic targeting against these epigenetic mechanisms, including inhibitors of DNA and RNA methylation as well as regulators of RNA modifications, can enhance tumor suppressor gene activity. In this review, we examine recent studies elucidating the role of epigenetic regulation in OS pathogenesis and discuss emerging drugs or interventions with potential clinical utility. Understanding of tumor- specific epigenetic alterations, coupled with innovative therapeutic strategies and AI-driven biomarker discovery, could pave the way for personalized therapies based on the molecular profile of each tumor and improve the management of patients with OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69115f288109b08c5dc3c9431da8115c0747b13f" target='_blank'>
              Epigenetic Modifications in Osteosarcoma: Mechanisms and Therapeutic Strategies
              </a>
            </td>
          <td>
            Maria A. Katsianou, Dimitrios Andreou, Penelope Korkolopoulou, E. Vetsika, C. Piperi
          </td>
          <td>2025-07-28</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Treatment resistance is a major hurdle in diffuse large B-cell lymphoma (DLBCL) therapy. Here, we assessed the relevance of the Musashi (MSI) RNA-binding protein family for DLBCL treatment efficacy. As important gene expression regulators, these proteins have previously been associated with tumorigenesis, treatment failure, and reduced survival in other malignancies, making them promising candidates for assessment in the context of DLBCL outcome and therapy resistance. Methods We first leveraged publicly available gene expression studies to determine expression and prognostic relevance of MSI1 and MSI2 in DLBCL. We then characterized MSI2 co-expressed therapy-relevant signaling. After performing MSI2 knockdown experiments we investigated subsequent effects on DLBCL gene expression in vitro using qPCR, Western blot, protein arrays, and flow cytometry. Finally, cell viability assays and clonogenic assessments were used to assess resistance to radiation, vincristine, and doxorubicin chemotherapy. Results MSI2 was overexpressed and prognostically unfavorable in univariable and multivariable analyses in DLBCL while MSI1 showed very low expression. High MSI2 expression was associated with increased stemness and DNA repair signaling. MSI2 knockdown led to a loss of stemness-associated markers and compromised DNA repair protein activation while increasing radiation-induced DNA double-strand break levels. Cell survival after either radiotherapy, vincristine or doxorubicin chemotherapy was impaired after MSI2 knockdown in follow-up analyses, suggesting a radio- and chemosensitizing effect. Discussion We propose that MSI2, a prognostic marker, may modulate the susceptibility of DLBCL towards genotoxic therapy. Suppressing MSI2 may hold promise to sensitize DLBCL to DNA-targeted treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a873df352503c3224c8f6ec89d594d3543c9ab8" target='_blank'>
              Prognostic marker Musashi-2 modulates DNA damage response and radioresistance in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Timo Habig, Lasse Reichstein, Kathrin A. Brücksken, M. Sicking, Jan Labisch, M. Oertel, Eberhard Korsching, Georg Lenz, Stephan Hailfinger, Burkhard Greve, Fabian M Troschel, H. Eich
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a major cause of cancer-related deaths worldwide, often diagnosed at advanced stages with limited treatment options. Despite advancements in systemic therapies, including the use of lenvatinib, the survival rate for advanced HCC remains low due to drug resistance and tumor heterogeneity. Method This study employed single-cell sequencing and spatial transcriptomics to investigate intra-tumor heterogeneity and identify subpopulations of malignant cells with inherent drug resistance. Tumor samples were obtained from patients undergoing lenvatinib, and differential analysis was performed to identify biomarkers associated with drug resistance. Results Analysis revealed significant differences in treatment response between different HCC nodules within the same patient, indicating intra-tumor heterogeneity. Single-cell sequencing and spatial transcriptomics identified distinct cell clusters within tumors, with specific subpopulations showing inherent resistance to lenvatinib. Differential analysis identified 17 common upregulated genes, including PCK1 and ALDH1A1, which were significantly associated with drug resistance. Pseudotime analysis further supported the role of PCK1 and ALDH1A1 as potential biomarkers of inherent drug resistance in HCC. This study highlights the importance of tumor heterogeneity and identifying biomarkers associated with drug resistance in HCC. Conclusions The findings suggest that PCK1 and ALDH1A1 may serve as potential biomarkers for predicting treatment response and guiding therapeutic strategies in HCC patients. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03481-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5d2cb3017ef7eb96af09bc83b104b7725e1bc2c" target='_blank'>
              PCK1 and ALDH1A1 are identified as biomarkers for inherent drug resistance in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wei Chen, Cheng Zhou, Gao Liu, Xinyu Wang, Yong Yi, Yang Xu, Shuangjian Qiu
          </td>
          <td>2025-08-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b37ee620d5697800e8d511cd10bb157702e6caa" target='_blank'>
              Clinical and analytical validation of MI Cancer Seek®, a companion diagnostic whole exome and whole transcriptome sequencing-based comprehensive molecular profiling assay.
              </a>
            </td>
          <td>
            V. Domenyuk, Kasey Benson, Peggy Carter, Dan Magee, Jian Zhang, Nitin Bhardwaj, Hongseok Tae, James Wacker, Foram Rathi, Siobhan Miick, Aastha Kohli, Joshua Carroll, Lori Cuyugan, Evelyn Perez, Wayeesha Zhang, John Collins, Patrick Kennedy, Jeremy Ellis, A. Stark, Andrey Loskutov, Brittany Cuttone, Blake Taylor, Rebecca A Feldman, Jeff Swenson, David Bryant, R. Hahn-Lowry, Raunaq Kaushal, Jennifer R. Ribeiro, J. Abraham, M. Radovich, George W Sledge, M. Oberley, D. Spetzler
          </td>
          <td>2025-08-13</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Neuroblastoma (NB) is the most common extracranial solid tumor in early childhood and frequently presents with bone marrow (BM) metastasis, particularly in high-risk cases. Metastatic NB cells residing in the BM exhibit distinct biological behaviors and are closely associated with treatment resistance and poor prognosis. Emerging evidence suggests that metabolic reprogramming is a hallmark of NB progression; however, its regulatory landscape within the bone marrow microenvironment remains poorly understood. This study aimed to systematically elucidate the molecular mechanisms underlying metabolic reprogramming in bone marrow–infiltrating metastatic NB cells by integrating single-cell transcriptomic data, bulk RNA-sequencing profiles, and in vitro functional validation. Single-cell RNA-sequencing data from 17 bone marrow aspirates of NB patients—with and without marrow infiltration—were retrieved from the GEO database, while bulk RNA-seq data from 155 NB tumor samples were obtained from the TARGET database. A suite of bioinformatics tools, including Seurat, Harmony, CellChat, and SCENIC, was employed to perform multi-dimensional analyses such as cell subtype annotation, intercellular communication mapping, and transcription factor regulon construction. AUCell scoring, differential gene expression analysis, random survival forest modeling, and Kaplan–Meier survival analysis were conducted to identify key genes involved in metabolic reprogramming and to characterize their immune infiltration features. Gene Set Enrichment Analysis (GSEA), Gene Set Variation Analysis (GSVA), and single-cell pathway scoring were applied to predict the metabolic and immune pathways associated with these candidate genes. Furthermore, co-expression network analysis was used to evaluate the relationships between candidate genes and known NB regulatory factors. Finally, in vitro experiments were carried out to validate the roles of selected genes in regulating NB cell proliferation, migration, invasion, and mitochondrial function. Single-cell transcriptomic analysis of bone marrow–derived NB samples identified nine distinct cell subpopulations with diverse intercellular communication networks. Among these, the ligand–receptor pair MDK–NCL emerged as a key mediator of cell–cell signaling. Regulatory network analysis revealed five critical regulons—JUND, JUNB, FOS, E2F1, and KLF16—closely associated with metabolic reprogramming in NB. The “Neuroblastoma cell” cluster displayed markedly elevated metabolic activity. Through integrated analyses, five core metabolic reprogramming genes (MRPL21, NHP2, RPL13, RPL18A, and RPL27A) were identified and shown to be significantly associated with poor prognosis. High expression of these genes correlated with an immunosuppressive tumor microenvironment, characterized by reduced infiltration of monocytes, M1 macrophages, and T cells. Functional enrichment analysis revealed that these genes were primarily enriched in oxidative phosphorylation, MYC targets, PI3K-Akt, and p53 signaling pathways. Co-expression network analysis further demonstrated that MRPL21 and NHP2 positively correlated with known NB regulatory genes TP53, NRAS, and NKIF1B. In vitro assays confirmed that MRPL21 knockdown significantly impaired NB cell proliferation, migration, invasion, and mitochondrial oxidative phosphorylation. This study identified five key genes involved in metabolic reprogramming of bone marrow–infiltrating NB cells, which are closely associated with immunosuppressive microenvironment formation and enrichment in tumor-associated metabolic pathways. Among these, MRPL21 plays a pivotal role in regulating NB cell proliferation and mitochondrial function, underscoring its potential as a promising therapeutic target. These findings suggest that integrated strategies targeting both tumor metabolism and the immune microenvironment may provide new avenues for the treatment of high-risk, metastatic NB. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13626-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b395447d21c8867ac393546695c70f8056f2bca" target='_blank'>
              Integrated single-cell and transcriptomic analysis of bone marrow-derived metastatic neuroblastoma reveals molecular mechanisms of metabolic reprogramming
              </a>
            </td>
          <td>
            Jing Chu, Rong Qin, Shu-Jing Wang, Qiang Wang, Qiang Wu
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastatic cancer is the final frontier in disease progression with few avenues of treatment open to patients. The failure of therapeutic options at the end-stage disease is confounded by the presence of diverse clones in the metastatic tumor which contributes to intratumoral heterogeneity (ITH), resulting in intrinsic and acquired drug resistance. We aim to elucidate the contribution of ITH to the development of metastasis with a focus on the origins and molecular mechanisms driving ITH, as well as the clinical and technical challenges of acquiring and studying metastatic cohorts in which we can investigate this phenomenon. Bioinformatic approaches that could help in silico analysis will be discussed, which could shed light on how to design new therapeutic strategies for metastatic cancer and give hope to patients living under the shadow of the sword of Damocles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ee12f1d193c46bad8ef5d6e4f9b07aba3289ca" target='_blank'>
              Role of intratumoral heterogeneity in metastatic progression and drug resistance
              </a>
            </td>
          <td>
            Neeha Sinai Borker, Jyothilakshmi Sajimon, Seemadri Subhadarshini, Rifat Aara, M. Jolly, Jyothy S Prabhu, Radhika Nair
          </td>
          <td>2025-09-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="ABSTRACT Background The influence of immune cell traits (ICTs) on the onset of multiple brain diseases has been previously investigated; however, it is limited by the sample size or colocalization evidence. Besides, the impact remains inconclusive. Methods We performed a Mendelian randomization (MR) study to elucidate the causal correlation between significant ICTs and diverse brain disorders and explored the biomarkers linked to glioblastoma (GBM), a form of solid tumor, by integrating expression quantitative trait locus (eQTL) and single‐cell RNA sequencing (scRNA‐seq) analyses. The nonnegative matrix factorization (NMF) method was utilized to reclassify malignant cells into distinct cell states. Related functional analyses at the scRNA‐seq level were also performed. Results We examined 731 ICTs across 13 brain disorders; impacts from these ICTs varied a lot across different brain diseases. Such ICTs mainly involved T/natural killer (NK) cell activation, B cell differentiation, and myeloid cell suppression or activation. Pleiotropy or heterogeneity in current results has been checked and excluded via sensitivity analyses. Specifically, colocalization analyses demonstrated protective roles of distinct ICTs in T/B/NK cell panels for amyotrophic lateral sclerosis (ALS) and GBM, while myeloid and human leukocyte antigen (HLA)‐associated traits were associated with increased risk of Alzheimer's disease (AD), and then two memory cell traits were linked to the increased risk of major depressive disease (MDD). By NMF, we identified six distinct cell states within GBM cells. Furthermore, we established an eight‐marker glioblastoma risk signature (GBRS) using scRNA‐seq and eQTL data, with higher GBRS scores observed in the NFkB cluster and EGFR cluster, indicating their highlighted aggression among malignant cells. Epigallocatechin gallate could be an effective treatment candidate targeting the EGFR cluster via markers of SQLE and VCP. Conclusion Our findings identified causal effects of distinct ICTs on both malignant and nonmalignant brain diseases and underscored the pivotal role of neuroinflammation in their etiology. With combined evidence from eQTL and scRNA‐seq, GBM could be better characterized and managed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91ab5455b35326f6300f48d861ff5796be4eae6a" target='_blank'>
              Causal Association Between Immune Cell Traits and Risk of Multiple Malignant and Nonmalignant CNS Diseases: A Mendelian Randomization and Single‐Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            Shanbao Ke, Junya Yan, Baiyu Li, Xiao Feng
          </td>
          <td>2025-08-01</td>
          <td>Brain and Behavior</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Meningiomas are common brain tumors, but a subset behaves aggressively and resists current treatments. This review proposes a developmentally informed framework showing that where a meningioma arises in the skull mirrors its embryologic origin (neural crest vs. mesoderm) and imprints distinct genetic, epigenetic, and immune programs that shape tumor behavior and treatment response. This lineage “stamp” helps explain why convexity and skull-base tumors differ and highlights actionable circuits, NF2/YAP–TAZ, PI3K/AKT, TRAF7-linked pathways. We integrate spatial genomics and single-cell data to map tumor location and molecular class to rational therapies, including CDK4/6, PI3K/AKT, YAP/TEAD, and epigenetic inhibitors, with emerging strategies such as synthetic-lethal combinations, CRISPR-guided target discovery, and degrader (PROTAC) approaches for high-grade and recurrent disease. Our goal is a practical roadmap that links anatomy and biology to precision treatment and trial design, improving outcomes by matching the right therapy to the right meningioma at the right site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3274f2ff22ffc0fdf55125037e12d854830b1a18" target='_blank'>
              Spatially Encoded Oncogenesis and Transcriptional Plasticity in Meningioma: Drivers of Therapeutic Resistance and Opportunities for Targeted Intervention
              </a>
            </td>
          <td>
            Matthew A Abikenari, Amit Regev, Brandon Bergsneider, Vratko Himič, Shreyas Annagiri, Lily H Kim, Ravi Medikonda, John Choi, S. Jeyaretna, Daniel M. Fountain, Michael Lim
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences that constitute a significant portion of mammalian genomes. While typically silenced by epigenetic mechanisms, mounting evidence indicates TEs can regulate gene expression and chromatin architecture. However, their regulatory roles under various environmental exposures remain largely unexplored. In this study, we investigate the regulatory functions of TEs in mouse liver tissue following early-life exposure to environmental toxicants, including arsenic (As), lead (Pb), bisphenol A (BPA), tributyltin (TBT), di-2-ethylhexyl phthalate (DEHP), tetrachlorodibenzo-p-dioxin (TCDD), and particulate matter less than 2.5 micrometers (PM2.5). These toxicants are linked to various health issues, including neurodevelopmental deficits, metabolic and immune dysfunction, and increased cancer risks. Integrative analysis of 351 multi-omics datasets from liver tissues of 5-month-old mice indicated that early-life environmental exposures significantly altered chromatin accessibility and expression of TEs in later life stage, revealing distinct exposure-specific signatures and sex-dependent responses. 6,699 TEs were identified with altered chromatin accessibility, mostly in non-coding regions, suggesting potential impact on gene regulation. Within these TEs, LINE elements were enriched in genes involved in metabolic pathways, while LTR elements, particularly the ORR1E subfamily, were predominantly associated with immune-related genes. Additionally, we identified 140 TE-gene chimeric transcripts with TE-derived novel transcription start sites, highlighting TE-contributed transcriptional plasticity. Our findings depict a comprehensive landscape of TE regulation under early-life toxicant exposures, offering insights into TEs biology and their impact on health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5f0711b9c755ac942e42700386af14ea62ffc2" target='_blank'>
              The impact of environmental exposures on the epigenomic and transcriptomic landscape of transposable elements
              </a>
            </td>
          <td>
            Benpeng Miao, Yu Zhang, Wanqing Shao, Shuhua Fu, Marisa S. Bartolomei, Cheryl Walker, Cristian Coarfa, Bo A Zhang, Ting Wang
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite the high sensitivity of estrogen receptor positive (ER+) breast cancer to endocrine therapy, many patients have primary resistance or develop resistance to endocrine therapies. Acquired resistance to endocrine therapy is a great challenge in the treatment of ER+ breast cancer patient. Here we showed that Response Gene to Complement (RGC)-32 expression is higher in breast cancer than paired normal tissues, which was a poor predictive factor. RGC-32 overexpression resulted in tamoxifen resistance, whereas knockdown of RGC-32 in tamoxifen-resistant cells restored tamoxifen sensitivity. Tamoxifen resistance mediated by RGC-32 was shown to be partially dependent on FoxM1 expression. Mechanistically, RGC-32 could activated PI3K signaling pathway, and then enhanced estrogen receptor alpha (ERα) activity. ERα activation is essential for RGC-32-mediated the expression of FoxM1. These data support that targeting RGC-32 could effectively mitigate cancer progression and tamoxifen resistance, offering a complementary therapeutic approach to reduce acquired endocrine resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd8ff0c9ab0a8e724cafedf2beedbe10119c0d4" target='_blank'>
              Response Gene to Complement 32 promotes cell proliferation and tamoxifen resistance in breast cancer via elevated FoxM1 expression
              </a>
            </td>
          <td>
            Xinlei Li, Yan Liu, Zhiqian Wang, Xiaocui Bu, Yu Wang, Wei Zhang, Peng Zhao
          </td>
          <td>2025-07-28</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861fd3ffe0c8c1bb6ed63c62ce2660ccaa3e238b" target='_blank'>
              miRNA-mediated resistance mechanisms in prostate cancer: implications for targeted therapy and metastatic progression.
              </a>
            </td>
          <td>
            Mostafa M. Mostafa, M. K. A. El-Aziz, D. Ellakwa
          </td>
          <td>2025-08-29</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cardiac myxoma (CM), a rare primary cardiac tumor, poses significant life-threatening risks. Current CM research has remained largely limited to clinical case observations and pathological analyses, thus restricting its clinical therapeutic impact. Fundamental research should be urgently strengthened to better support future CM treatment strategies. In this work, single-cell sequencing is used to elucidated the intricate cellular composition of the CM microenvironments. The mechanisms of heart myxoma cell growth are investigated via proteomics and organoid models, while our western blot analysis reveals cardiac myxoma’s immune evasion strategies. This study successfully characterizes diverse cell types within the CM microenvironment. Notably, ap-CAF cells are found to effectively recruit immune cells via chemokine secretion, fostering immune microenvironment formation. The work’s pseudotime trajectory analysis also demonstrates that CM tumor cells derive from mesenchymal stem cells. Additionally, this work demonstrated that the glycolysis pathway is significantly activated and fuels CM cell growth. Tumor cells exploit the SIRPα-CD47 immune checkpoint to evade the immune system by inhibiting antigen-presenting cell phagocytosis. Tumor-associated macrophages (TAMs) concurrently assume M2 polarization and suppress autoimmune activity through IL-10. This research comprehensively examines CM’s microenvironmental cellular architecture, metabolic features, and immune escape mechanisms. These work’s findings not only deepen the current understanding of CM’s biological nature but also offer vital theoretical foundations for developing safer, more effective CM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaa9c5f44a7fada1677fd86632e043c456f245b" target='_blank'>
              Single-cell RNA sequencing and proteomics uncover SIRPα-CD47 immune checkpoint and glycolysis-driven immune evasion in cardiac myxoma
              </a>
            </td>
          <td>
            Xinyuan Zhu, Qingle Gao, Xintong Dai, Baofa Sun, Yanping Li, Hongyan Zhai, Changliang Shan
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54525feaa0ad7b16a4eefd8a46c2d6fd3ccc01e5" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of Single-Nucleotide Variant and Copy Number Aberration Evolution from Single-Cell DNA Sequencing of Tumors.
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2025-09-15</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d108c58970b821d1c2170e04d8c4196841969e5f" target='_blank'>
              Phage therapy and its role in cancer treatment and control.
              </a>
            </td>
          <td>
            H. A. Abd El-Moaty, A. Doghish, Hebatallah Ahmed Mohamed Moustafa, Wagiha S. Elkalla, Ghadir A Sayed, Nourhan H Elshami, S. S. A. Mageed, Osama A Mohammed, Mahmoud A. Elrebehy, Kareem AlFarsi, Amr M. Abdelfatah, Aly M Waseem, Ahmed O Abdulkader, Rabab S. Hamad, Akram N Salah
          </td>
          <td>2025-09-17</td>
          <td>Folia microbiologica</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background
Copy number variations (CNVs) play an important role in human genetic disorders. Detection of exon-level CNVs is crucial for accurate clinical diagnosis. The CytoScan XON Array, a high-resolution microarray, was recently developed to detect exonic CNVs of various genes.


Methods
We evaluated the clinical performance of the CytoScan XON Array using 59 patient samples with previously identified CNVs, confirmed via methods including multiple ligation-dependent probe amplification (MLPA), gene-dose PCR, and mRNA assay. Concordance between CytoScan XON and orthogonal methods was evaluated in target regions, and diagnostic utility was compared with that of genome sequencing (GS)-based CNV calling tools through analysis of false-positive CNVs in non-target genomic regions.


Results
For target regions, the CytoScan XON Array achieved concordance rates of 89.8% and 92.5% at the exon and gene levels, respectively, for all CNV calls. Concordance was higher for multi-exon CNVs (100%) than that for single-exon CNVs (82.6%, P =0.03). For non-target regions, false-positive CNV calls were reduced to fewer than 0.01 per gene per person through filtering strategies. The array exhibited false-positive detection rates within dosage-sensitive genes comparable with those of GS-based tools.


Conclusions
The CytoScan XON Array, a reliable tool for detecting exon-level CNVs in target regions, can serve as a complementary approach to GS-based CNV calling tools for genome-wide CNV screening with high resolution. However, its performance for single-exon CNVs requires further optimization. Cross-validation with GS-based CNV calling tools is recommended to improve diagnostic accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a08a61a6f8683a7068bd9bbb76ec77849b8894" target='_blank'>
              High-resolution Chromosomal Microarray with Diagnostic Potential for Detecting Exon-level Copy Number Variations Using Targeted and Non-targeted Approaches.
              </a>
            </td>
          <td>
            Yeseul Kim, Jee-Soo Lee, Boram Kim, M. Kim, Sung Im Cho, Seung-Won Chae, H. Shin, Hoyeon Lee, Ji Yeon Kim, Moon-Woo Seong
          </td>
          <td>2025-07-30</td>
          <td>Annals of laboratory medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most common and aggressive brain derived tumor. It often shows genetic alterations in kinase signaling pathways, such as the Pi3K/mTOR and RAS/MAPK pathways, that frequently converge onto oncogenic processes. However, it is unknown to what extend co-vulnerabilities exist within this network and which kinase drug targets are promising for GB treatment. We investigated the drug sensitivity of GB cell line models to monotherapy and synergy effects in dual combination therapy to targeting components of Pi3K/mTOR and RAS/MAPK pathways. In addition, we examined cell line drug sensitivities in relation to their individual genetic tumor driving lesions (i.e. NF1 alterations as well as transcriptomic defined GB subtypes). Synergy levels were correlated to in-lab generated phospoproteomic data. Lastly, serial or simultaneous addition of MEK and mTOR inhibitors were investigated in longitudinal experiments. Dual inhibition of MEK and mTOR resulted in synergistic effects, which associated with NF1-deficiency. Strong synergy effects was also associated with the mesenchymal subtype. Dual inhibition of MEK and mTOR led to prolonged growth inhibition in GB spheroids. In addition, sequential drug treatment resulted in similar growth inhibitory effects compared to simultaneous combination therapies. Our findings highlight the potential of dual inhibition strategies targeting multiple kinases for the treatment of GB, particularly in NF1-deficient and mesenchymal tumors, the most lethal subtype of GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049087437ba8fac6737126641a3cf7ff0b99a6f7" target='_blank'>
              Combined inactivation of MEK and mTOR can lead to synergistic cell death in glioblastoma models and associates with NF1-deficiency and a mesenchymal subtype.
              </a>
            </td>
          <td>
            F. Cornelissen, Yoran Broersma, R. Narayan, Rogier Dik, S. Piersma, Richard de Goeij-de Haas, T. Pham, D. Noske, W. Vandertop, Connie R. Jimenez, Bart A Westerman
          </td>
          <td>2025-08-01</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Microsatellite instability (MSI) serves as an important therapeutic and prognostic marker. Immunohistochemistry (IHC) and polymerase chain reaction (PCR) have long been regarded as the “gold standard” for MSI detection. In recent years, next-generation sequencing (NGS)-based methods, offering expanded target coverage of microsatellite (MS) loci and improved analytical performance, have gained widespread acceptance. However, discrepancies between NGS and traditional methods have been occasionally reported. In this study, we conducted a large-scale retrospective analysis of MSI results of 35,563 Chinese pan-cancer cases which underwent a NGS test in a central clinical laboratory. Our work introduced a novel algorithm for NGS-based MSI detection, examined MSI-H prevalence and value distribution across cancer types, identifies MSI-associated genes and variants, evaluates the MSI detection discordance between PCR and NGS in a pan-cancer context, and distilled 7 MS loci suitable for pan-cancer MSI detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c293d8863f4abc1b0f16238e217e4bad714bd24" target='_blank'>
              Applying next-generation sequencing to detect microsatellite instability in pan-cancer patients: a retrospective study of 35,563 Chinese cases
              </a>
            </td>
          <td>
            Yun Shang, Longfeng Yang, Pengtao Hu, Sihui Zhao, Jianbiao Xu, Yanwei Zhao, Xing Ren, Ding Zhang, Qingzhong He, Xuefei Liu
          </td>
          <td>2025-08-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related morbidity and mortality worldwide. Its marked heterogeneity - encompassing molecular subtypes, histological characteristics, and variable therapeutic responses - continues to pose persistent clinical challenges Although advances in surgery, hormone therapy, chemotherapy, and targeted therapies have significantly improved patient outcomes, issues such as therapeutic resistance and disease relapse are still common, underscoring the need for novel molecular targets. Within this context, non-coding RNAs (ncRNAs) have emerged as pivotal regulators of breast cancer biology and hold promise as diagnostics and therapeutic agents. These non-protein-coding RNA molecules include diverse subclasses, such as long non-coding RNAs (lncRNAs), circular RNAs (circRNAs), and small non-coding RNAs (sncRNAs), each characterized by distinct structural features and biological functions. Mounting evidence implicates ncRNAs in key oncogenic processes - such as tumor initiation, progression, metastasis, immune evasion, and treatment resistance - often in a subtype-specific manner. Importantly, ncRNA expression profiles differ significantly across BC subtypes, and their stability in body fluids underscores their potential utility in liquid biopsy-based diagnostics. This review provides an integrated overview of the multifaceted roles of ncRNAs in BC, emphasizing their mechanisms of action, contributions to tumor heterogeneity, and translational potential as both biomarkers and therapeutic targets. Understanding ncRNAs complexity and context-specific functions may pave the way toward more precise, personalized interventions for BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5de9a0dd3e6ab3c3ee75baeac6b79a2d1c7dff" target='_blank'>
              Non-coding RNAs as novel biomarkers and therapeutic targets in breast cancer
              </a>
            </td>
          <td>
            V. Barbi, Sara De Martino, A. Aiello, M. Gottardi Zamperla, Sara Negri, Luca Cis, V. Pecci, Simona Nanni, A. Farsetti, Fabio Martelli, C. Gaetano, S. Atlante
          </td>
          <td>2025-08-29</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens.
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Advances in genetic and transcriptomic technologies have identified large numbers of genes and variants of potential importance to human disease. Determining the function of these genes and variants is a critical bottleneck in understanding disease etiology. Variants of uncertain significance (VUS) are highly prevalent in our genomes, but our ability to identify them significantly outpaces our ability to determine their molecular and clinical consequences. We developed a genetically tractable induced pluripotent stem cell (iPSC) based platform to investigate gene variant pathogenicity in lung disease, using primary ciliary dyskinesia (PCD) as a model. We identified an individual with a clinical diagnosis of PCD and a VUS in the gene Multiciliate differentiation and DNA synthesis associated cell cycle protein (MCIDAS). Through gene-editing of iPSC-derived airway basal stem cells (iBCs), we precisely defined the molecular and cellular pathogenicity of the variant providing a successful application of the iPSC system to diagnose a lung disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1192b86ac009b9b71e1d325755d48cbbdf2ac789" target='_blank'>
              A stem cell-based platform for functional analysis of genetic variants in lung disease
              </a>
            </td>
          <td>
            D. Wallman, Sachiko T Homma, Madeleine Finton, MaryLou Beermann, Anjali Jacob, S. Choksi, S. Pletcher, Huihui Xu, P. S. Bawa, Jeremy F. Reiter, A. Horani, Steven L. Brody, A. Berical, F.J. Hawkins
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Resistance to CDK4/6 inhibitors (CDK4/6i) is a major challenge in treating hormone receptor-positive, HER2-negative metastatic breast cancer. This study aimed to identify a biomarker predictive of resistance that could also serve as a therapeutic target. Circulating IL-6 levels in 166 patients significantly increased at progression, making IL-6 a non-invasive biomarker to predict CDK4/6i resistance. Knockdown of IL-6 re-sensitized resistant cells to palbociclib and endocrine therapy, underscoring IL-6’s critical role in maintaining resistance. Patient-derived xenograft models from patients who progressed within 3 months versus ≥6 months of palbociclib therapy revealed distinct transcriptomic profiles, with later progressors exhibiting IL-6/STAT3 activation, epithelial-to-mesenchymal transition, and immune evasion signatures. Treatment with TTI-101, a STAT3 inhibitor, significantly reduced tumor growth and improved survival in these models, providing preclinical validation for targeting the IL-6/STAT3 axis. These findings support using IL-6 to guide personalized treatment and combining STAT3 inhibitors with CDK4/6i as a transformative strategy to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08735c14db476a16d97e533ff51d162f3e014327" target='_blank'>
              IL-6 predicts CDK4/6 inhibitor resistance, identifying STAT3 as a target in HR + /HER2-negative metastatic breast cancer
              </a>
            </td>
          <td>
            N. Kettner, Tuyen N Bui, Juliana Navarro-Yepes, Sanaz Ghotbaldini, Bethanie Quintela, Catherine K. Luo, Nghi Lam, X. Rao, A. Raghavendra, Yan Wang, Nancy Azizian, T. Kris Eckols, Moses M. Kasembeli, K. Evans, Min Yi, Hannah F Wingate, Jing Wang, Aysegul A. Sahin, Funda Meric-Bernstam, Kelly K Hunt, S. Damodaran, David J Tweardy, Debu Tripathy, K. Keyomarsi
          </td>
          <td>2025-07-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Abstract Background Osteosarcoma is a highly aggressive cancer, and the efficacy of existing therapies has plateaued. Multiomics integration analysis can identify novel therapeutic targets for various cancers and therefore shows potential toward osteosarcoma treatment. This study aimed to leverage multiomics integration to develop a new risk model, characterizing the immune features of osteosarcoma to uncover novel therapeutic targets. Methods Metabolomics profiling was conducted to identify key metabolites in osteosarcoma. Transcriptomic sequencing datasets were analyzed to identify prognostic genes related to key metabolic pathways and develop a prognostic risk model. Patients were then divided into high-risk and low-risk groups with distinct clinical outcomes based on the risk model. The single-sample gene set enrichment analysis, Estimation of Stromal and Immune cells in Malignant Tumor tissues using Expression data (ESTIMATE) algorithm, and xCell algorithms were used to evaluate the immune cell infiltration and activity. Single-cell RNA sequencing was used to explore cell-to-cell interactions within the tumor microenvironment. In vitro coculture functional assays were performed to validate the role of macrophage migration inhibitory factor (MIF) in macrophage polarization and chemotaxis. In vivo studies were used to evaluate the effectiveness of MIF inhibition in combination with immune checkpoint blockade in murine models. Results Elevated lactate levels in osteosarcoma patients correlated with poorer overall survival. We identified SLC7A7 and CYP27A1 as prognostic lactate metabolism genes and developed a risk model to stratify patients into high-risk and low-risk groups with distinct outcomes. Bioinformatics analyses highlighted the differences in immune infiltration patterns and activity between the groups. Notably, the infiltration and phenotype of macrophages varied significantly between the groups, and MIF was identified as a critical mediator in this process. In osteosarcoma cells, lactate regulated MIF expression through histone H3K9 lactylation. Combining the MIF inhibitor 4-IPP with a programmed cell death 1 (PD-1) monoclonal antibody treatment demonstrated a significant antitumor effect. Conclusion MIF acts as a novel therapeutic target by regulating macrophage polarization and chemotaxis. Lactate regulated MIF expression through histone lactylation. Targeting MIF holds promise for enhancing the efficacy of anti-PD-1 treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e40fea73eb193df251f667b830b628b26326e1d6" target='_blank'>
              Multiomics integration analysis identifies tumor cell-derived MIF as a therapeutic target and potentiates anti-PD-1 therapy in osteosarcoma
              </a>
            </td>
          <td>
            Weidong Chen, Yan Liao, Hao Yao, Yutong Zou, Ji Fang, Jiongfeng Zhang, Zehao Guo, Jian Tu, Junkai Chen, Zijun Huo, Lili Wen, Xianbiao Xie
          </td>
          <td>2025-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) evolves as an aggressive phenotype during prolonged androgen deprivation therapy, lacking effective clinical management. Here, we elucidated a reciprocal metabolic-epigenetic mechanism involving a positive feedback loop between glycolysis and the transcription factor LHX2 that promotes PCa progression. Mechanistically, enzalutamide-mediated inhibition of the androgen receptor (AR) led to upregulation of key glycolytic enzymes. Elevated glycolysis resulted in lactate accumulation and subsequent histone lactylation, which in turn enhanced LHX2 expression. Reciprocally, LHX2 transcriptionally activated the lactate dehydrogenase LDHA, which further amplified lactate production. Furthermore, LHX2 augmented the expression of the lineage-determining enzyme DNMT1, potentiating neuroendocrine gene expression and tumor growth. Significantly, the antiviral agent paritaprevir could directly bind to LHX2, effectively suppressing neuroendocrine marker expression and tumor progression. These findings uncover a potential role for LHX2 in orchestrating NEPC development, offering putative therapeutic targets for treating this aggressive cancer phenotype and overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef2fefc98e5fd3352706deab4254aa3b33bda2c" target='_blank'>
              LHX2 Rewires the Metabolic and Epigenetic Landscape to Drive Tumor Progression in Prostate Cancer.
              </a>
            </td>
          <td>
            Jun Jiang, Shaojie Liu, Chao Xu, Limin He, Hongji Li, Yike Zhou, Zhengxuan Li, Yu Li, Fa Yang, Yuchen Wei, Tong Lu, Keying Zhang, Jingliang Zhang, Pang Wang, Li Guo, Changhong Shi, Weihong Wen, Rui Zhang, Donghui Han, Weijun Qin
          </td>
          <td>2025-09-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Prostate cancer (PCa) is an extremely heterogeneous cancer and is highly prevalent in the older male population. Since intra‐tumour heterogeneity (ITH) commonly results in PCa chemotherapy resistance and recurrence, it is critical to explore its effects on tumour behaviour. Prognostic genes related to ITH were identified, and a signature was constructed using Cox regression analyses and multiple machine learning algorithms. Single‐cell RNA sequencing data extracted from PCa and CRPC samples were analysed via sub‐clustering, pseudotime, cell communication and drug sensitivity approaches to elucidate their function. The oncogenic potential of hub genes was confirmed by immunohistochemistry and cell proliferation assays. An 11‐gene signature underlying a prostate cancer meta‐program (PCMP) was generated by selecting an optimal combination of machine learning methods. Survival assays and multivariate Cox regression analyses conducted in multiple cohorts revealed the superior prognostic value of the PCMP signature. Functional enrichment analyses indicated that it dysregulates the cell cycle. Using trajectory and cell–cell communication analyses, we illustrated that PCMP genes exert oncogenic effects by enhancing the proliferation and oxidative phosphorylation of epithelial cells. Intra‐cellular assays also demonstrated that CENPA and CKS1B had promising malignant potential. In summary, our research not only establishes the association between the PCMP signature and reveals its malignant characteristics, but also deepens our understanding of the mechanisms underlying PCa progression and ITH. It holds promise for the development of targeted therapeutic interventions, thereby offering clinical benefits to patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/045ffd0cd32bcb5869978af9b2509598525fb2d8" target='_blank'>
              Single‐Cell and Bulk RNA Sequencing Highlights Intra‐Tumoral Heterogeneity and Malignant Progression Mechanisms in Prostate Cancer
              </a>
            </td>
          <td>
            Junchao Wu, Ziqi Chen, Wentian Wu, Jiaxuan Qin, Rongfang Zhong, Jialin Meng, Yu Yin, Peng Guo, Song Fan
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0e77a0dc71f8f861306d03d6b2410b93243bdc" target='_blank'>
              Next-generation T cell immunotherapies engineered with CRISPR base and prime editing: challenges and opportunities.
              </a>
            </td>
          <td>
            Karl Petri, E. D’Ippolito, A. Künkele, Ulrike Köhl, Dirk H. Busch, H. Einsele, M. Hudecek
          </td>
          <td>2025-09-19</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Colorectal cancer (CRC) treatment remains challenging due to genetic heterogeneity and resistance mechanisms. To address this, we developed a drug discovery pipeline using patient-derived primary CRC cultures with diverse genomic profiles. These cultures closely resemble certain molecular characteristics of primary and metastatic CRC, highlighting their promise as a translational platform for therapeutic evaluation. Importantly, our engineered model and patient-derived cells reflect the complexity and heterogeneity of primary tumors, not observed with standard immortalized cell lines, offering a more clinically relevant system, although further validation is needed. High-throughput screening (HTS) of 4255 compounds identified 33 with selective efficacy against CRC cells, sparing normal, healthy epithelial cells. Among the tested combinations, everolimus (mTOR inhibitor) and uprosertib (AKT inhibitor) demonstrated promising synergy at clinically relevant concentrations, with favorable therapeutic windows confirmed across tested patient-derived cultures. Notably, this synergy, revealed through advanced models, might have been overlooked in traditional immortalized cell lines, highlighting the translational advantage of patient-derived systems. Furthermore, the integration of machine learning into the HTS pipeline significantly improved scalability, cost-efficiency, and predictive accuracy. Our findings underscore the potential of patient-derived materials combined with machine learning-enhanced drug discovery to advance personalized therapies. Specifically, mTOR-AKT inhibition emerges as a promising strategy for CRC treatment, paving the way for more effective and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b44bc31581c51d6eaf637f51510217abbfb946c" target='_blank'>
              Integrated transcriptomic and functional modeling reveals AKT and mTOR synergy in colorectal cancer
              </a>
            </td>
          <td>
            M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Szklarczyk, Olga Dracz, A. Kurzejamski, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Karolina Pyziak, Joanna Drozdowska, Krzysztof Baczyński, Konrad Wojtowicz, Maurycy Chronowski, Krzysztof Rataj, Magdalena Otrocka, M. Mikula, Paula Feliksiak, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka, Andrzej Mazan
          </td>
          <td>2025-07-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common type of leukemia in Western populations and remains incurable despite significant therapeutic advancements. Over the past decade, the treatment landscape has evolved from traditional chemoimmunotherapy to targeted oral agents, including Bruton’s tyrosine kinase inhibitors (BTKis) and BCL2 inhibitors (BCL2is), which have demonstrated superior efficacy and tolerability, especially in elderly patients. Venetoclax, a BCL2i, induces apoptosis in CLL cells through selective inhibition of the anti-apoptotic BCL2 protein, while BTKis, such as ibrutinib and its next-generation analogs, disrupt B-cell receptor signaling critical to CLL cell survival. However, resistance to both drug classes has emerged, including mutations in BTK and BCL2, prompting the exploration of novel therapeutic strategies. This review outlines the molecular basis and clinical implications of these resistance mechanisms, as well as emerging therapeutic solutions, including non-covalent BTKis like pirtobrutinib and BTK-targeting PROTAC degraders such as BGB-16673 and NX-2127. Additionally, we discuss promising combination therapies incorporating BTKis, BCL2is, and anti-CD20 monoclonal antibodies. Finally, we highlight the growing role of measurable residual disease (MRD) as a biomarker to guide treatment duration and evaluate therapeutic success. As resistance mechanisms continue to emerge, tailoring therapy based on underlying biology will be critical to sustaining disease control and enhancing outcomes in patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0956883756565c425ce66e36bee2fa5445ca9e94" target='_blank'>
              Resistance Mutations in CLL: Genetic Mechanisms Shaping the Future of Targeted Therapy
              </a>
            </td>
          <td>
            Samantha Sekeres, Erica N. Lamkin, Eduardo Bravo, Allison Cool, Justin Taylor
          </td>
          <td>2025-09-10</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Tumor-associated macrophages have been implicated in the progression and treatment resistance of glioblastoma (GBM). This study aimed to identify macrophage-related genes associated with prognosis and therapeutic response in GBM. Materials and methods Bulk RNA-seq data from 533 patients with GBM were downloaded from the Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) databases. Bioinformatic tools were used to detect the co-expression gene modules associated with the infiltration of immune cells, identify a prognostic macrophage-related gene signature, and explore their association with sensitivity to chemotherapeutic drugs and immune checkpoint blockade. Single-cell RNA-seq data and multiplexed immunofluorescence were used to validate ISG20 expression (a member of the identified gene signature) in macrophages. Results We detected gene modules associated with macrophages and identified a signature consisting of three macrophage-related genes (ISG20, PARP12 and IFIT5) in the discovery set (TCGA–GBM, n = 159), and validated its prognostic value in the validation set (CGGA–GBM, n = 374). This gene signature demonstrated favorable accuracy in predicting prognosis and resistance of immuno- and chemo-therapy. The co-expression of ISG20 and PD-1 in macrophages was verified by single-cell RNA-seq data and multiplex immunofluorescence. Conclusions This study presents a macrophage-related gene signature to predict prognosis and therapeutic response in GBM. ISG20, PARP12 and IFIT5 are interferon-stimulated genes, and further investigations may provide new insights into the interplay between macrophages and interferon signaling in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f715f9dcfc9de2bd4b3f11c175f24e467cbcae2" target='_blank'>
              Identification of a prognostic signature consisting of three macrophage-related genes for glioblastoma based on bulk and single-cell transcriptomes analyses
              </a>
            </td>
          <td>
            Qiyan Wu, Chuanbiao Chen, Lina Li, Hongyu Liu, Lingling Cui, Yang Bai, Liping Zhang, Lingxiong Wang, Yaojun Peng
          </td>
          <td>2025-08-14</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Cancers involving RAS gene mutations are among the most difficult to treat, particularly in advanced stages. These genes for RAS—KRAS, HRAS, and NRAS—control normal cellular processes, but when mutated, they help tumors grow, spread, and resist treatment. Decades of research have targeted RAS genes and Ras proteins, yet many therapies still fail due to oversimplified signaling interpretations and poor translational reliability. This review compares how Ras functions in healthy and cancerous cells and evaluates model systems across human, mice, and fly studies. We highlight the limitations of current model systems and the need for more physiologically relevant research tools. The goal is to push research toward more effective tools, refined models, and improved outcomes for patients with late-stage RAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796354d7dffc674d220d0fc486fafe630423dfaf" target='_blank'>
              RESEARCH CHALLENGES IN STAGE III AND IV RAS-ASSOCIATED CANCERS: A Narrative Review of the Complexities and Functions of the Family of RAS Genes and Ras Proteins in Housekeeping and Tumorigenesis
              </a>
            </td>
          <td>
            Richard A. McDonald, A. Varela-Ramírez, Amanda K Ashley
          </td>
          <td>2025-07-25</td>
          <td>Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Myeloid cells - including monocytes, macrophages, dendritic cells, and granulocytes - are critical architects of the tumor microenvironment, where they exert diverse functions ranging from immunosuppressive to immunostimulatory. Advances in single-cell omics and high-dimensional immune profiling have unveiled the remarkable heterogeneity and plasticity of these cells, revealing lineage-specialized functions that shape cancer immunity. These discoveries have sparked growing interest in therapeutically targeting myeloid cells as a next generation strategy in cancer immunotherapy. As a complementary or alternative approach to T cell-centered immunotherapies, myeloid-directed therapies offer unique opportunities to reprogram the immune landscape, enhance antitumor responses, and overcome resistance mechanisms. In this review, we highlight recent discoveries in myeloid cell biology in cancer and discuss emerging therapeutic targets, with an emphasis on antibody-based therapies that have reached clinical development. We further provide perspective on translational challenges to implement these approaches into the clinic, and discuss how Fc-engineering and rational antibody design can optimize myeloid cell engagement and amplify their immune effector functions. Together, these advances position myeloid-directed immunotherapies as a promising approach to enhance the efficacy and durability of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0766685af3f8f2b4c14f90c4b851b211a5e328" target='_blank'>
              Targeting Myeloid Cells for Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Lucas Blanchard, Andrew Mijacika, Juan C. Osorio
          </td>
          <td>2025-08-28</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The protein p53, often referred to as the “guardian of the genome,” is essential for preserving cellular balance and preventing cancerous transformations. As one of the most commonly altered genes in human cancers, its impaired function is associated with tumor initiation, development, and resistance to treatment. Exploring the diverse roles of p53, which include regulating the cell cycle, repairing DNA, inducing apoptosis, reprogramming metabolism, and modulating immunity, provides valuable insights into cancer mechanisms and potential treatments. This review integrates recent findings on p53′s dual nature, functioning as both a tumor suppressor and an oncogenic promoter, depending on the context. Wild-type p53 suppresses tumors by inducing cell cycle arrest or apoptosis in response to genotoxic stress, while mutated variants often lose these functions or gain novel pro-oncogenic activities. Emerging evidence highlights p53′s involvement in non-canonical pathways, such as regulating tumor microenvironment interactions, metabolic flexibility, and immune evasion mechanisms. For instance, p53 modulates immune checkpoint expression and influences the efficacy of immunotherapies, including PD-1/PD-L1 blockade. Furthermore, advancements in precision diagnostics, such as liquid biopsy-based detection of p53 mutations and AI-driven bioinformatics tools, enable early cancer identification and stratification of patients likely to benefit from targeted therapies. Therapeutic strategies targeting p53 pathways are rapidly evolving. Small molecules restoring wild-type p53 activity or disrupting mutant p53 interactions, such as APR-246 and MDM2 inhibitors, show promise in clinical trials. Combination approaches integrating gene editing with synthetic lethal strategies aim to exploit p53-dependent vulnerabilities. Additionally, leveraging p53′s immunomodulatory effects through vaccine development or adjuvants may enhance immunotherapy responses. In conclusion, deciphering p53′s complex biology underscores its unparalleled potential as a biomarker and therapeutic target. Integrating multi-omics analyses, functional genomic screens, and real-world clinical data will accelerate the translation of p53-focused research into precision oncology breakthroughs, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6708657f0ff7cd8075abb4af281defbd7a1577" target='_blank'>
              The Multifaceted Role of p53 in Cancer Molecular Biology: Insights for Precision Diagnosis and Therapeutic Breakthroughs
              </a>
            </td>
          <td>
            Bolong Xu, Ayitila Maimaitijiang, Dawuti Nuerbiyamu, Zhengding Su, Wenfang Li
          </td>
          <td>2025-07-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While antiretroviral therapy (ART) has significantly improved disease prognosis in people with HIV (PWH), understanding the biological mechanisms underlying plasma HIV-1 RNA viral load (VL) can inform additional strategies to slow HIV/AIDS disease progression. Here, we integrated multi-omic datasets and used two machine learning network biology tools (GRIN and MENTOR) to identify biological mechanisms associated with VL across 10 cohorts from multiple omics data sets. We integrated the following gene sets: 3 genes from HIV set point VL GWAS, 258 genes whose expression was associated with set point VL in CD4+ T-cells, 143 genes based on DNA methylation associations with VL, and 8 genes previously known to affect the pharmacokinetics of ART. Using GRIN, we retained 194 VL genes based on their high network interconnectivity. We then used MENTOR to collaboratively interpret subsets of these genes and identified the following biological processes: cell cycle checkpoint pathways associated with non-AIDS defining cancers, oxidative stress, viral replication, and interferon signaling. Using these network tools for multi-omic integration, we present a conceptual model of mechanisms underlying HIV VL, and identify drug repurposing candidates to complement existing ART to enhance treatment response and reduce HIV-related comorbidities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba231c5a3dc9e179c84b4cbd72ec6cc335455d1" target='_blank'>
              Omics Integration Uncovers Mechanisms Associated with HIV Viral Load and Potential Therapeutic Insights
              </a>
            </td>
          <td>
            Kyle A. Sullivan, M. S. Minto, Xinyu Zhang, William Carr, B. Quach, Caryn Willis, Alice Townsend, Peter Kruse, Matthew Lane, Richard Morgan, Ke Xu, Bradley E. Aouizerat, Dana B. Hancock, Daniel A. Jacobson, E. O. Johnson
          </td>
          <td>2025-07-30</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Breast cancer is one of the most common malignancies among women globally, with an annually increasing incidence rate. Its complex pathogenesis and high heterogeneity pose significant challenges to clinical diagnosis and treatment. Traditional diagnostic methods and therapeutic approaches have limitations in improving patient survival rates and prognosis, thus urgently necessitating the identification of novel biomarkers and therapeutic targets. MicroRNA (miRNA), a class of endogenous non-coding small RNA molecules with a length of approximately 20–24 nucleotides, finely regulates gene expression by binding to the 3’ untranslated region of target mRNAs, inhibiting gene translation, or promoting mRNA degradation. In 2024, the Nobel Prize in Physiology or Medicine was awarded for research related to miRNA. Numerous studies have demonstrated that miRNAs play pivotal roles in the initiation, progression, invasion, metastasis, and drug resistance of breast cancer. Aberrant expression of specific miRNAs is closely associated with the molecular subtypes, prognosis, and treatment response of breast cancer, suggesting their potential as diagnostic and prognostic biomarkers. To explore the potential value of miRNA in the diagnosis and treatment of breast cancer, this article systematically reviews the latest research progress on the role of miRNAs in the diagnosis and treatment of breast cancer, with a focus on their application as tumor markers in early diagnosis, molecular subtyping, and therapeutic response monitoring. It elucidates the possibilities of miRNAs as therapeutic targets and tools in targeted therapy, including the current research status of miRNA mimics and inhibitors in breast cancer treatment. Furthermore, it analyzes the role of miRNAs in prognosis assessment, exploring their correlation with patient survival rates, recurrence risks, and treatment responsiveness. Additionally, this article discusses the challenges faced by miRNA research in precision medicine for breast cancer and future directions, providing new insights and strategies for early diagnosis, individualized treatment, and prognosis assessment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302a8130c8ecf1046d266c188e11099501cd566d" target='_blank'>
              MicroRNAs in breast cancer—new frontiers in diagnosis, targeted therapy, and prognosis assessment
              </a>
            </td>
          <td>
            Jinsuo Xiao, Lanhui Zhang, Ruifan Su, Bo Zhao, Yuanyuan Dang, Chuanlin Zhao, Sujin Wang, Teng Qi, Fuqing Ji
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer has recently overtaken other conditions as the foremost cause of death worldwide. While numerous
conventional treatment modalities and cytotoxic immunotherapies are available, the intricate nature of tumor biology—
driven by a multitude of genetic and cellular factors underlying tumorigenesis and metastasis—necessitates the
development of advanced treatment approaches capable of acting on cellular as well as genetic fronts. Amid these
advancements, Chimeric Antigen Receptor (CAR) T-cell therapy has risen as a pioneering frontier in the domain of T-cell
engineering. Here, T cells obtained from the patient are engineered ex vivo to carry synthetic receptors designed to
recognize tumor-associated antigens. A key feature is that these modified receptors can identify tumor antigens without
requiring MHC presentation. Recently, CAR-T cell therapy has achieved significant clinical milestones, inducing
remission in up to 80% of patients with hematologic malignancies, particularly those with acute lymphoblastic leukemia
(ALL) and certain subtypes of non-Hodgkin lymphoma, such as large B-cell lymphoma. The anti-CD19 CAR design,
UCART19, has demonstrated remarkable therapeutic potency in managing relapsed or refractory hematologic
malignancies. Furthermore, other surface antigens, including CD20 and CD22, prevalent in various leukemias and
lymphomas, are being actively explored as therapeutic targets, with multiple clinical trials underway. Although its
application is presently concentrated on blood cancers, the integration of advanced modalities—such as bispecific CARs,
tandem CARs, inhibitory CARs, multi-antigen targeting, CRISPR-based gene editing, and nanoparticle-mediated
delivery—holds the promise of significantly enhancing its efficacy. These advancements may extend its utility to both
hematologic and solid tumors, offering a treatment paradigm that is not only rapid and precise but also safer compared to
traditional modalities. The purpose of this review is to offer an in-depth analysis of the benefits and emerging
developments in CAR T-cell immunotherapy, while underscoring its rising advantage over conventional treatments like
chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5c74588893737227ed28d56c33635129d1547e" target='_blank'>
              Targeting Cancer at its Core: Cellular and Genetic Approaches Via CAR-T Cell Therapy CAR-T Cell Therapy of Cancer
              </a>
            </td>
          <td>
            Bandaru Aashritha, Ambati Akshara, Nandini Kongara, P. V. Babu
          </td>
          <td>2025-09-04</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a unique subtype of breast cancer characterized by high invasiveness, high metastasis rates, and poor prognosis, making it an important focus within global malignancies. Due to the absence of estrogen receptor, progesterone receptor, and HER2 expression, TNBC presents significant challenges in treatment. Metastatic progression markedly increases treatment complexity, drastically reducing patient survival rates. The metastatic and drug resistance processes of TNBC involve complex, multi-step biological mechanisms regulated through various molecular mechanisms and signaling pathways within and outside tumor cells. In recent years, immunotherapy has brought new hope for TNBC. Compared to other breast cancer subtypes, TNBC demonstrates higher immunogenicity, often accumulating a higher mutational burden that generates more neoantigens, thus typically resulting in a tumor microenvironment (TME) enriched with tumor-infiltrating lymphocytes (TILs). Additionally, PD-L1 expression is significantly higher in TNBC compared to other subtypes, closely correlating with TIL abundance. These characteristics position TNBC as a strong candidate for immune checkpoint inhibitor (ICI) therapy. Clinical trials have demonstrated promising efficacy of ICIs in TNBC, overturning previous beliefs that breast cancer is generally insensitive to immunotherapy. This review summarizes recent advances regarding resistance types, molecular mechanisms, associated genes and pathways, the role of the tumor microenvironment, and clinical strategies related to immunotherapy resistance in the neoadjuvant setting of TNBC, aiming to provide insights and guidance for future research exploration and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03cacb55534d35ef52652d44531e93baccfb5003" target='_blank'>
              Immunotherapy resistance in triple-negative breast cancer: Molecular mechanisms, tumor microenvironment, and therapeutic implications
              </a>
            </td>
          <td>
            Zijian Zhou, Qin Zhou
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcaf09592f5e3c1f92fc05933cdb4d9765107436" target='_blank'>
              Evolutionary Overview and Future Perspectives: ESR1 Mutations, Liquid Biopsy, and Artificial Intelligence for a New Era of Personalized Medicine in ER+ Breast Cancer.
              </a>
            </td>
          <td>
            Serafina Martella, Giacomo Cusumano, T. H. Senevirathne, Dimitrios Stylianakis, Enrico Palmas, N. Denaro, Chiara Tommasi, M. Scartozzi, Lorenzo Gerratana, C. Solinas
          </td>
          <td>2025-08-30</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Breast cancer, a major health concern worldwide, involves diverse molecular subtypes and complex gene expression patterns. This study conducted a comprehensive bioinformatics analysis of breast cancer, analyzing 10 gene expression datasets from the Gene Expression Omnibus archive to find common genes that exhibit differential expression (DEGs). Then, we conducted pan-cancer and functional enrichment analyses, including single-cell level investigations of DEGs. To identify potential hub genes, we built protein–protein interaction networks, which were further analyzed for methylation patterns and expression in various cancer stages. The study also explored interactions between these hub genes with microRNAs and transcription factors, along with their correlation with cytokine expression and infiltration of immune cells in the tumor microenvironment. The TCGA BRCA dataset was used for hub gene survival analysis. Key findings include the identification of 36 shared DEGs, with distinct expression patterns in principal component analysis, suggesting their potential as molecular signatures for diagnosis and prognosis. Investigation at the single-cell level revealed upregulated DEGs mainly expressed by tumor-associated fibroblast cells. Gene ontology unveiled upregulated genes mainly involved in the extracellular matrix, but most of the downregulated genes are tumor suppressors. The protein–protein interaction network analysis highlighted 7 hub genes: CAV1, FN1, BGN, CXCL12, SPTBN1, COL11A1, and INHBA. Methylation analysis revealed intricate epigenetic regulation of these genes. MicroRNA and transcription factor interaction analysis underscored the complex regulatory networks influencing gene expression. Cytokine profile analysis in tumor microenvironment and its correlation with hub gene expression provided insights into the immunological landscape of breast cancer. Survival analysis indicated that both upregulated and downregulated hub genes are linked with patient overall survival outcomes, although it is not statistically significant. This study provides a detailed view of the underlying molecular mechanisms in breast cancer, suggesting potential biomarkers and therapeutic targets. Its findings can contribute significantly to understanding the complexity of the breast cancer microenvironment, leading to the development of more sophisticated and targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf4528f9aeae9acc9b474eba2a50c1c17c8b995" target='_blank'>
              Identification of potential hub genes and molecular mechanisms in breast cancer microenvironment: A comprehensive transcriptomics approach
              </a>
            </td>
          <td>
            MS Abdullah Al Noman, MS a Abdullah Al Saba, MS a Mohammad Sayem, PhD Tahirah Yasmin, PhD A. H. M. Nurun Nabi, A. N. Nabi
          </td>
          <td>2025-08-29</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome integrity is a critical driver of cellular stability, and defects in the processes that maintain genome health are potent sources of cancer progression. Homologous recombination deficiency (HRD), which damages cells through absent or erroneous repair of double-stranded DNA breaks, is a prime example of such cellular dysfunction. Poly(ADP-ribose) polymerase (PARP) inhibitors exploit these aberrancies in the cellular repair process by arresting the ability of the PARP enzyme to repair cellular and genetic damage, inducing the accumulation of DNA damage and cancer cell death. While the utility of PARP inhibitors has been established in many HRD-associated tumors — particularly breast, pancreatic, ovarian, and prostate cancer — less robust evidence exists for the efficacy of PARP inhibitors in melanoma. Increasing efforts are underway to investigate PARP inhibitors as a viable treatment option for advanced and metastatic melanoma, both as monotherapy and in combination with other agents such as immune checkpoint inhibitors and BRAF/MEK inhibitors. Though several gaps in our knowledge of the use of PARP inhibitors in melanoma still exist, promising headway is being made in our understanding of its efficacy and safety. Here, we present a review of the utility of PARP inhibitors in melanoma, current clinical trials, and future avenues for further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e9847a79c6d01c47c9645097708f725557ad2a" target='_blank'>
              PARP inhibitors in melanoma treatment: potential, challenges, and future directions
              </a>
            </td>
          <td>
            Angela Anaeme, Karen Moussa, Abdallah N. Mansour, George Nassief, George Ansstas
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo– and targeted therapy resistant state even after long term culture ex vivo . These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo– and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P Ji, Nancy R. Zhang
          </td>
          <td>2025-09-15</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Over the past decade, increasing evidence has linked the dysregulation of human protein O-fucosyltransferase 1 (POFUT1), overwhelmingly through gene overexpression, to tumor progression in multiple cancers, including colorectal, breast, gastric, lung, hepatocellular carcinoma, and glioblastoma. This review provides a comprehensive analysis of the molecular and cellular consequences of POFUT1 dysfunction in cancer. POFUT1 overexpression driven by copy number variations (CNVs), epigenetic alterations, and/or post-transcriptional modifications enhances tumorigenesis by activating key oncogenic pathways such as Notch, Wingless-type MMTV integration site family (Wnt)/β-catenin, and phosphoinositide 3-kinase/protein kinase B/mammalian target of rapamycin (PI3K/AKT/mTOR). These pathways promote cell proliferation, migration, and epithelial-mesenchymal transition (EMT) while simultaneously suppressing apoptosis. Additionally, POFUT1 promotes an immunosuppressive tumor microenvironment that contributes to treatment resistance by immune checkpoint inhibitors. Mainly, its overexpression is detectable at early stages of tumor development and in some cancer patient sera, highlighting its potential as a non-invasive biomarker for early cancer detection and disease monitoring. Given its role in immune evasion and therapy resistance, POFUT1 represents a promising therapeutic target, warranting further investigation into its clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc32b216cd93b5ffb3fc4906eb7ae996906a5999" target='_blank'>
              POFUT1 as a cancer biomarker: insights into its oncogenic mechanisms and clinical relevance
              </a>
            </td>
          <td>
            Oumaima Mazour, Sébastien Legardinier, B. Badaoui, A. Germot
          </td>
          <td>2025-08-07</td>
          <td>Journal of Molecular Medicine (Berlin, Germany)</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Therapeutic strategies based on gene editing provide the ability to modify faulty genes contributing to the development of diseases such as cancer by directly altering the cellular machinery. The clustered regularly interspaced short palindromic repeats associated nuclease 9 (CRISPR/Cas9) system is currently the primary tool used for gene editing. Several effective Cas9 variants have already been established to address the complex genetic modifications that arise during diseases. Although gene-editing systems have made significant advancements, a primary obstacle that requires attention is the transportation of CRISPR/Cas to diverse target cells, both in vivo and in vitro, to render them suitable for clinical implementation. Various strategies can be utilized to facilitate the transportation of the CRISPR/Cas systems into mammalian cells. Herein, we reviewed contemporary research about delivery systems for gene-editing systems that interact effectively in biological systems. This review explores the benefits and drawbacks of using extracellular vesicles and viral vectors as vehicles for delivering the CRISPR/Cas system in the context of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfd50f0c66284d92243c7f008886cbbd521c72ac" target='_blank'>
              Overcoming the Delivery Challenges in CRISPR/Cas9 Gene Editing for Effective Cancer Treatment: A Review of Delivery Systems
              </a>
            </td>
          <td>
            Shuting Tang, Xiaoyi Chen, Xiangmin Tong, Lifen Zhu
          </td>
          <td>2025-07-28</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer is the most common in the world with multifaceted and diverse molecular etiology. Lung cancer initiation, progression and metastasis are orchestrated by a network of enzymes and proteins. They include TP53, KRAS, EGFR and ALK, which are expressed and mutated at abnormally high levels. These molecules control signaling in various cellular pathways including growth, survival, differentiation and invasion.

This review focuses on the malignantly regulated TP53, which is commonly and mutated most frequently in non-small cell lung cancer Overall, we also highlighted the role of TP53 in lung cancer development, the existing therapeutic approaches against TP53 and its relation about the efficacy and adverse events such as resistance and side effects and the next steps to achieve better outcomes in TP53 -positive lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24095e3a50d4594724b1890874f6611abcf39d57" target='_blank'>
              The Role of TP53 Mutations in Lung Cancer: Molecular Mechanisms, Therapeutic Implications and Future Directions
              </a>
            </td>
          <td>
            Thirunavukkarasu Muthu Kumar
          </td>
          <td>2025-08-31</td>
          <td>Research Journal of Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract DNA double-strand breaks (DSBs) represent critical events in genome integrity, arising from both endogenous cellular processes and exogenous factors. These breaks are implicated in various genomic aberrations and chromosomal rearrangements, leading to cancers and genetic disorders. Common and rare fragile sites, containing repetitive elements and non-B DNA structures, are particularly prone to breakage under replication stress, which play a pivotal role in cancer development and genetic diseases. Accurate quantification of DNA breaks in the context of repetitive sequences such as microsatellites or non-B DNA structures is technically challenging. We have been comparing four different methods to reliably quantify DSBs in repetitive DNA, namely Southern blot, DSB-PCR, real-time DSB-qPCR, and digital PCR (dPCR). We show here that dPCR offers enhanced sensitivity and specificity compared to other methods. This provides significant applications for future disease diagnosis, understanding molecular mechanisms generating chromosomal breakage and for the development of gene therapies for microsatellite expansion disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccab30f2a498f797c04595f2b9078ba3ca2f3603" target='_blank'>
              Fast and accurate quantification of double-strand breaks in microsatellites by digital PCR
              </a>
            </td>
          <td>
            Cécile Palao, Adèle Kovacs, M. Teixeira, G. Richard
          </td>
          <td>2025-08-09</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527c49eaf6701e153dab5ae1e99ef449da850bc8" target='_blank'>
              In vivo CRISPR screening links NFKB1 to endocrine resistance in ER⁺ breast cancer.
              </a>
            </td>
          <td>
            Cancan Lyu, Sam Hall, Mark A. Stamnes, Songhai Chen
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Systems biology approaches have been applied to prostate cancer to model how individual cellular and molecular components interact to influence cancer development, progression, and treatment responses. The integration of multi-omic experimental data with computational models has provided insights into the molecular characteristics of prostate cancer and emerging treatment strategies that have the potential to improve patient outcomes. Here, we highlight recent advancements that have emerged from systems modeling in prostate cancer. These include descriptions of the molecular landscape of prostate cancer and how genomic alterations inform computational models of disease progression, how evolutionary processes give rise to mechanisms of therapeutic resistance, and the development of innovative treatment strategies such as adaptive therapy. We also highlight current challenges in prostate cancer that can be addressed through systems biology approaches. These include tumor heterogeneity, poor immunotherapy response, a paucity of experimental model systems, and the ongoing translation of computational models for clinical decision making. Leveraging systems biology approaches has the potential to lead to a better understanding of the disease and better patient outcomes in the treatment of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aeffc4990ae5d3cf0820d80475621089e3b441" target='_blank'>
              Systems biology successes and areas for opportunity in prostate cancer
              </a>
            </td>
          <td>
            Michael V Orman, Laura S Graham, Scott D. Cramer, James C. Costello
          </td>
          <td>2025-08-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Resistance is inevitable and a major challenge in treating Lung adenocarcinoma (LUAD) patients with EGFR mutations. This study aimed to investigate the mechanism of EGFR-TKI resistance in lung adenocarcinoma using longitudinal single-cell RNA sequencing (scRNA-seq) data. We collected tumor samples of LUAD patients before and after EGFR inhibitor treatment and performed single-cell RNA sequencing. We used machine learning models for cell annotation and classified cells into subgroups. The inferCNV algorithm was used for CNV score calculation and tumor cell identification, and metabolic analysis was done using a gene-scoring approach. EGFR resistance score (ERscore), a gene signature derived from resistant tumor cells, was established to evaluate the predictiveness to EGFR-TKI inhibitors. The investigation classified subgroups of cells and identified three tumor cell types as critical cells mediating EGFR-TKI resistance. Our data also analyzed the metabolic aspects of EGFR-TKI resistance using a single-cell approach. It showed that some tumor cell subtypes had a consistent metabolic profile, significantly upregulating purine metabolism, oxidative phosphorylation, glycogen, and lipid metabolism. An assessment system called ERscore was established to evaluate the association between EGFR-TKI resistance and tumor ecosystem. The analysis showed a significant correlation between the ERscore and EGFR-TKI resistance, lung cancer phenotype, and prognosis. The findings suggest that the molecular mechanisms driving EGFR-TKI resistance in lung cancer may also contribute to poorer prognosis, particularly in lung adenocarcinomas with high EGFR mutation rates. Overall, the study provides important insights into the mechanisms of EGFR-TKI resistance in lung cancer at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f04790ae4365e12380b845f56d6a28c5640baa" target='_blank'>
              Longitudinal single-cell RNA model aids prediction of EGFR-TKI resistance.
              </a>
            </td>
          <td>
            Guoxin Hou, Zhimin Lu, D. Zeng, Qican Chen, Subing Cheng, B. Song
          </td>
          <td>2025-07-31</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b348bc9a3e4abf3cb6086b89e0d2182bceec0bd" target='_blank'>
              NEK9-mediated Wnt signalling repressor TLE3 rewires Docetaxel resistance in cancer cells by inducing pyroptosis.
              </a>
            </td>
          <td>
            S. Ansari, Sibasish Mohanty, P. Mohapatra, Sudeshna Datta, Mamuni Swain, Rachna Rath, D. Muduly, S. Majumdar, R. Swain, Sunil K Raghav, Rupesh Dash
          </td>
          <td>2025-09-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Given the propensity of aggressive epithelial tumors to form hepatic metastases, we performed an in vivo cDNA screen using the mouse liver and KRASG12D/TP53R273H pancreatic cells to identify the RNA binding protein GCN1 as integral component of hepatic outgrowth. RNAi experiments reveal that GCN1 triggers the ISR to activate serine, folate, and methionine biosynthetic pathways together with amino acid transporters, which act in concert to facilitate acquisition of metabolites and to restore redox homeostasis. Alongside activation of the ISR, we found that GCN1 also functions in the nucleus where it interacts with HNRNPK to suppress the expression of MHC-I molecules, and natural killer (NK) ligands. Intriguingly, we identified IMPACT as an endogenous competitive inhibitor of GCN1 that blocks both ISR-dependent metabolic control and disrupts HNRNPK interaction. In doing so, IMPACT enhances tumor immunogenicity to unleash NK cell killing, in addition to sensitizing metastatic tumor cells to immune checkpoint blockade (ICB).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940084da31cd31fac078b9eabda5419618abe96d" target='_blank'>
              Expression of IMPACT curtails metabolic plasticity and augments NK cell killing to abrogate metastatic growth.
              </a>
            </td>
          <td>
            Surajit Sinha, A. Panda, Rodrigo Xavier das Neves, Z. Nwosu, Ke Xu, Elke van Beek, Priyanka P Desai, Sivasish Sindiri, Sruthi Chempati, K. Remmert, Billel Gasmi, Linda Bojmar, Constantinos Zambirinis, Alexander J Rossi, Reed I Ayabe, Michael M. Wach, James D McDonald, Samantha M Ruff, Emily A Verbus, A. Saif, Alyssa V. Eade, C. Larrain, Lindsay R Friedman, Shreya Gupta, Alok Ranjan, Martha E Teke, Tahsin M. Khan, Tracey Pu, A. Sarvestani, Carrie E Ryan, Jacob T Lambdin, K. Luberice, Stephanie N. Gregory, Stephanie C Lux, Hanna Hong, Allen J Luna, Imani A Alexander, Sarfraz R. Akmal, Shahyan Rehman, Ashley Rainey, Todd D Prickett, Vishal N. Kopardé, Samantha Sevilla, Skyler A. Kuhn, King Chan, Zhonghe Sun, Nina Bubunenko, Eileen Li, C. Hannah, Geneti Gaga, T. Andresson, Margaret C. Cam, Xiaolin Wu, Lisa M. Jenkins, A. Blakely, Jeremy L. Davis, Giorgio Trinchieri, Pankaj K. Singh, James C. Yang, M. Pasca di Magliano, C. Lyssiotis, Micheal B Yaffe, Ethan M Shevach, Jonathan M Hernandez
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="This review provides a comprehensive synthesis of current knowledge on immunotherapy resistance in non-small cell lung cancer (NSCLC), a disease that accounts for approximately 85% of all lung cancer cases and remains the leading cause of cancer-related death worldwide. Although immune checkpoint inhibitors (ICIs) have significantly improved survival for a subset of patients with advanced NSCLC, over 70% of cases ultimately exhibit primary or acquired resistance, underscoring the urgent need to understand the underlying mechanisms. The review categorizes resistance into tumor-intrinsic and tumor-extrinsic processes and provides an in-depth mechanistic analysis of how factors such as tumor antigen loss, impaired antigen presentation, cGAS-STING pathway dysregulation, metabolic reprogramming in tumor microenvironment (TME), immune cell exhaustion, and microbiomes collectively contribute to immune escape. In parallel, the influence of the lung and gut microbiome on shaping immunotherapy responses is discussed, with emphasis on microbial dysbiosis, immunosuppressive metabolite production, and TME remodeling. Therapeutic strategies to overcome resistance are also discussed, including combination approaches involving chemotherapy, radiotherapy, and antiangiogenic agents, as well as epigenetic modulators (HDAC and BET inhibitors). Moreover, the review explores bispecific antibodies, antibody-drug conjugates, and small-molecule agents that enhance T cell function or disrupt immunosuppressive signaling networks. By integrating insights from preclinical models and clinical trials, the review underscores the necessity of biomarker-guided patient stratification, combination immunotherapy approaches, and interventions that restore tumor immunogenicity. It concludes that a multipronged therapeutic strategy, one that addresses both immune evasion and TME-induced suppression, holds the greatest promise for improving response durability and advancing personalized immunotherapy for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df76f6889b4c3ea7f132198d5c6301465fae771" target='_blank'>
              Immunotherapy resistance in non-small cell lung cancer: from mechanisms to therapeutic opportunities
              </a>
            </td>
          <td>
            Huiyu Wang, Xiaomin Niu, Zhenning Jin, Shaoxing Zhang, Rong Fan, Hua Xiao, Shen S. Hu
          </td>
          <td>2025-08-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The purpose of the study was a comprehensive analysis of molecular genetic changes in tumor and normal tissues of patients with colorectal cancer using whole-genome sequencing to identify de novo mutations, microsatellite instability patterns, and mutational signatures associated with tumor development.Material and Methods. A single-time cross-sectional study was conducted in the Kaliningrad region. Colorectal cancer samples were analyzed using whole-genome next-generation sequencing (WGS). The human reference genome GRCh38.p14 was used for read mapping. Bioinformatic analysis of signatures was conducted separately for each patient. 71 pairs of samples were sequenced: complete genomes of tumor biopsies and DNA isolated from blood samples for 71 patients. The average age among the analyzed patients was 69 years, with a median of 71 years.Results. Genetic alterations associated with tumor development and progression, as well as potential therapeutic targets were identified. Using the Oncovar database, 151 driver mutations were detected in 62 patients. Using the CIVIC database, 70 marker mutations were found in 45 patients. It was found that the most common driver mutations in the study cohort were mutations in the KRAS gene (KRAS:G35T, KRAS:G35A, KRAS:G38A), associated with tumor resistance to Panitumumab and Cetuximab. The frequency of the most common driver mutations in the study cohort was low, indicating high heterogeneity of colorectal cancer. Analysis of the tumor mutational burden (TMB) revealed a high average value of 5.5, suggesting a significant potential of immunotherapy in colorectal cancer treatment. In the studied population, the mutational signatures SBS1, SBS5, and SBS39 were the most common.Conclusion. The identified diversity of genomic changes may be associated with DNA replication errors, exposure to external factors, and genetic predisposition. The pattern of mutational signatures was very similar among all patients in the studied samples, suggesting a single mutagenic factor is involved in the development of colorectal cancer in all patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410fb5988f2bc832f5b1b887492d03eded83e97b" target='_blank'>
              Genetic landscape of colorectal cancer. A pilot study in the Russian Federation
              </a>
            </td>
          <td>
            E. V. Semina, K. A. Kubasova, V. Kakotkin, S. V. Rodimov, M. Agapov
          </td>
          <td>2025-08-04</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mature T-cell leukemias and lymphomas (mTCL) comprise a clinically and genetically heterogeneous group of lymphoid malignancies. Most subtypes of peripheral T-cell lymphomas and leukemic T-cell malignancies show an aggressive clinical course and poor prognosis. Thus, these diseases urgently require novel therapeutic strategies. Taking advantage of recent progress deciphering the genetic basis of mTCL, we generated a comprehensive database of genetic alterations from 1 825 patients with mTCL and utilized bioinformatic methodology developed to support treatment decisions in molecular tumorboards to identify novel potential therapeutics. Based on the results of this approach, we performed a systematic in vitro drug screen in molecularly characterized cell lines of mTCL to assess the activity of potential therapeutics. Of these, the cell cycle regulator WEE1 stood out as a novel therapeutic target in a genetically defined subgroup of mTCL. Indeed, WEE1 kinase inhibitors potently induced replication stress, premature mitotic entry, accumulation of DNA damage and induction of apoptosis in mTCL cell lines. Building on the single-agent activity of clinical grade WEE1 inhibitors, we next explored potential drug combination strategies through mechanistic studies. With this, we identified strong synergistic effects of combined WEE1 and JAK inhibition in genetically defined subtypes of mTCL, including primary patient samples of T-cell prolymphocytic leukemia. In summary, our study provides a comprehensive overview of the genetic landscape in mTCL and potential therapeutics in genetically defined subtypes. As such, we identified synergistic effects of dual WEE1 and JAK inhibition in cases with altered JAK/STAT signaling as a blueprint for clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5a1d19178c29c05a81757105f01cb9db0fb931d" target='_blank'>
              Discovery of molecularly-informed therapeutic strategies for mature T-cell leukemias and lymphomas.
              </a>
            </td>
          <td>
            N. Schmidt, Kevin Kornrumpf, L. Ries, K. Markus, N. Ksionsko, P. Mazzeo, M. Lohrberg, Maren Sitte, Gabriela Salinas-Riester, T. Braun, Annika Dechow, Marco Herling, A. Ianevski, T. Aittokallio, B. Chapuy, Holger Bastians, Gerald Wulf, Tim Beissbarth, Jürgen Dönitz, R. Koch
          </td>
          <td>2025-08-04</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Purpose Drug resistance severely compromises therapeutic efficacy in hepatocellular carcinoma (HCC); however, the selection of precise treatment strategies for patients remains a critical unmet clinical need. This study investigated PANoptosis-related mechanisms underlying HCC progression to identify actionable therapeutic targets and optimize patient-specific treatment outcomes. Patients and Methods Multi-omics analysis (single-cell/bulk RNA sequencing) combined with machine learning was used to identify the PANoptosis-related prognostic features. The association of PANoptosis-related expression with the tumor immune microenvironment and drugs was explored using bioinformatic analysis and experimental studies. Results High PANoptosis risk exhibited immunosuppressive microenvironments and therapeutic resistance. The PANoptosis-related gene YIF1B has emerged as a dual prognostic biomarker and tumor driver that promotes proliferation, and is linked to immune dysfunction and drug resistance. Conclusion YIF1B may be a promising therapeutic target. This PANoptosis framework bridges molecular mechanisms to clinical management, offering strategies for personalized HCC therapy and overcoming treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24c213b38fc84a86ff385bba02fbc05b8cafc8c" target='_blank'>
              Single-Cell RNA Sequencing Integrated with Bulk-RNA Sequencing Analysis Reveals Prognostic Signatures Based on PANoptosis in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Jiyin Wang, Xue Yin, Ziyi Li, Pu Liang, Yipeng Wang, Xinglin Li, Wenying Qiao, Chaoyang Xiong, Minghang Yu, Xiaoyan Ding, Xi Wang
          </td>
          <td>2025-07-29</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Globally, breast cancer remains one of the most prevalent malignancies and a leading cause of cancer‐related death in women, with over 2.3 million new cases reported annually. Despite treatment advances, one breast cancer type in particular, HER2‐negative breast cancer, lacks precise therapeutic targets. Given the role of N7‐methylguanosine (m7G) in gene regulation and its links to cancer progression, we investigated m7G regulatory gene expression and prognostic potential in HER2‐negative breast cancer. We analysed publicly available breast cancer datasets (The Cancer Genome Atlas and the Gene Expression Omnibus (GEO)) to analyse the differential expression of 14 m7G‐regulatory genes. Clustering analysis, based on m7G patterns, categorised HER2‐negative patients into two subgroups. A prognostic model was established through LASSO and Cox regression; subsequently validated by survival analysis, and further supported by functional assays confirming gene function. Our model identified CCNB1 and EIF4E as high‐risk genes, with EIF4E overexpression enhancing cell proliferation, migration and invasion. RNA‐sequencing (RNA‐seq) and pathway analyses showed that upregulated EIF4E activated Wnt signalling and extracellular matrix (ECM) components, processes required for tumour progression. High‐risk patients showed reduced immune cell infiltration and poorer survival outcomes. We highlight m7G regulatory gene potential, particularly EIF4E, as prognostic markers and therapeutic targets for HER2‐negative breast cancer. Targeting EIF4E‐related pathways could provide new therapeutic strategies to improve breast cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/897240b04d6470a1f2b1ebf7ff2e27af08659e70" target='_blank'>
              N7‐Methylguanine‐Related Gene Signature Highlights EIF4E as a Novel Therapeutic Target in HER2‐Negative Breast Cancer
              </a>
            </td>
          <td>
            Yangyang Cui, Yuhan Dai, Yiqin Xia, Wenxin Yu, Jiangdong Jin, Shui Wang, Hui Xie
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background DDX27, a member of the DEAD-box RNA helicase family, plays a pivotal role in RNA metabolism and is essential for diverse cellular processes, including transcription, pre-mRNA splicing, translation, and ribosome biogenesis. Recent findings have implicated DDX27 as a substantial contributor to tumorigenesis and cancer progression across various malignancies, establishing its significance as a molecular hub that interacts with key oncogenic partners such as major vault protein (MVP) and nucleophosmin 1 (NPM1). Methods We conducted systematic search in the following comprehensive academic databases: PubMed, MEDLINE or Web of Science. The keywords such as DDX27, DEAD-box protein 27, RNA helicase DDX27 and cancer, tumor or carcinoma were used for searching. This review consolidates the existing literature on DDX27, examining its structural features and biological functions within the context of tumorigenesis. We systematically explore the molecular mechanisms by which DDX27 influences tumor development and progression, focusing particularly on its roles across different cancer types, including colorectal cancer (CRC), gastric cancer (GC), breast cancer (BC), hepatocellular carcinoma (HCC), and oral squamous cell carcinoma (OSCC). Key molecular mechanisms such as NF-κB activation and ERK1/2 phosphorylation involved in DDX27-related pathways are discussed. Results Our comprehensive summary elucidates the context-dependent roles of DDX27 across various cancers, highlighting its associations with advanced disease stages, metastasis, and therapeutic resistance. We also assess the potential of DDX27 as a diagnostic and prognostic biomarker, correlating its expression levels with negative clinical outcomes. Conclusion Novel therapeutic strategies targeting DDX27 are proposed, including RNA interference techniques (siRNA and shRNA), miRNA-based therapies (miR-617 mimics), pathway modulation, and synthetic lethality approaches. Furthermore, we identify notable limitations in current research surrounding DDX27 and offer potential avenues for future investigation. These innovative strategies present significant promise for the development of precision cancer therapies aimed at improving treatment outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67002d435ab4b67914e4176d8c14913d0cf8ce8b" target='_blank'>
              DDX27 in cancer: molecular mechanisms, clinical implications, and therapeutic potential
              </a>
            </td>
          <td>
            Le Yang, Simon Wing Fai Mok, Hua Li, Io Nam Wong, Li Jun Yang
          </td>
          <td>2025-08-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy in developed countries. Risk factors for EC include metabolic alterations (obesity, metabolic syndrome, insulin resistance), hormonal imbalance, age at menopause, reproductive factors, and inherited conditions, such as Lynch syndrome. For the inherited forms, several genes had been implicated in EC occurrence and development, such as POLE, MLH1, TP53, PTEN, PIK3CA, PIK3R1, CTNNB1, ARID1A, PPP2R1A, and FBXW7, all mutated at high frequency in EC patients. However, gene function impairment is not necessarily caused by mutations in the coding sequence of these and other genes. Gene function alteration may also occur through post-transcriptional control of messenger RNA translation, frequently caused by microRNA action, but transcriptional impairment also has a profound impact. Here, we review how chromatin modifications change the expression of genes whose impaired function is directly related to EC etiopathogenesis. Chromatin modification plays a central role in EC. The modification of chromatin structure alters the accessibility of genes to transcription factors and other regulatory proteins, thus altering the intracellular protein amount. Thus, DNA structural alterations may impair gene function as profoundly as mutations in the coding sequences. Hence, its central importance is in the diagnostic and prognostic evaluation of EC patients, with the caveat that chromatin alteration is often difficult to identify and needs investigations that are specific and not broadly used in common clinical practice. The different phases of the healthy endometrium menstrual cycle are characterized by differential gene expression, which, in turn, is also regulated through epigenetic mechanisms involving DNA methylation, histone post-translational modifications, and non-coding RNA action. From a medicolegal and policy-making perspective, the implications of using epigenetics in cancer care are briefly explored as well. Epigenetics in endometrial cancer is not only a topic of biomedical interest but also a crossroads between science, ethics, law, and public health, requiring integrated approaches and careful regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12c1dec29d6d5a8d8725991b296e46655f27376" target='_blank'>
              Epigenetics of Endometrial Cancer: The Role of Chromatin Modifications and Medicolegal Implications
              </a>
            </td>
          <td>
            R. Piergentili, E. Marinelli, Lina De Paola, G. Cucinella, V. Billone, Simona Zaami, G. Gullo
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Induced pluripotent stem cells (iPSCs) are widely used to model human genetic diseases. The most common strategy involves collecting cells from relevant individuals and then reprogramming them into iPSCs. This strategy is very powerful, but finding enough individuals with a specific genetic disease can be challenging, especially since most are rare. In addition, making numerous iPSC lines is time-consuming and expensive. As a result, most studies have included relatively small numbers of iPSC lines, sometimes from the same individual. Considering the experimental variability obtained using different iPSC lines, there has been great interest in delineating the most efficient number of lines needed to achieve a robust and reproducible result. Several recommendations have been published, although most conclusions have been based on methods where experimental variance from individual cases is difficult to separate from technical issues related to the preparation of iPSCs. The current study used gene expression profiles determined by RNA sequencing (RNAseq) to empirically evaluate the impact of the number of unique individuals and the number of replicate iPSC lines from each individual for modeling Lesch-Nyhan disease (LND). This disease is caused by mutations in the HPRT1 gene, which encodes the enzyme hypoxanthine-guanine phosphoribosyltransferase. Results for detecting disease-relevant changes in gene expression depended on the analytical method employed, and whether or not statistical procedures were used to address multiple iPSC lines from the same individual. In keeping with prior studies, the best results were obtained with iPSC lines from 3-4 unique individuals per group. In contrast to prior studies, results were improved with 2 lines per individual, without statistical corrections for duplicate lines from the same individual. In the current study where all lines were produced in parallel using the same methods, most variance in gene expression came from technical factors unrelated to the individual from whom the iPSC lines were prepared.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e088e96623fcc62d8d7dc77831018934ca79e1d" target='_blank'>
              Modeling rare genetic disease with patient-derived induced pluripotent stem cells: reassessment of the minimum numbers of lines needed
              </a>
            </td>
          <td>
            Ashok R. Dinasarapu, Diane J. Sutcliffe, Lauren Grychowski, Erkin Ozel, Anika Thite, Jasper E Visser, Ellen J. Hess, Sharon M Kolk, H. Jinnah
          </td>
          <td>2025-07-24</td>
          <td>Stem Cells Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James O. J. Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly lethal, aggressive malignancy. Little is known about the evolutionary trajectories of HCC and how clinical decision-making could be informed based on biopsies of the initial tumour. Here, we report the whole-exome sequencing of a unique series of resected HCC tumours and matched recurrences. This cohort included patients who received a liver transplant and who were immunosuppressed at time of recurrence, in comparison to patients who underwent liver resection for HCC and immunocompetent at time of recurrence, therefore facilitating analyses of immune selection in driving evolutionary divergence. We find extensive evolutionary divergence between baseline and recurrent tumours, with the majority of mutations in our cohort being private, in the process informing sampling guidelines for precision oncology in this disease. We also find no evidence that immunosuppression relaxes immune selection pressures, given the absence of a genomic footprint reflecting the presentation of neoantigens or altered dynamics of genomic evolution. We attribute this to the presence of genetic lesions that confer the capabilities of immune evasion in these tumours prior to transplantation, and then validate the link between immune selection pressures and the emergence of these lesions in publicly available HCC datasets. Our findings point to HCC as a cancer with extensive evolutionary divergence over time, partly defined by an irreversible, genetically determined trajectory towards immune escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9c9a24625027fee25b759620a813b537ff8ac7" target='_blank'>
              Evolutionary dynamics of recurrent hepatocellular carcinoma under divergent immune selection pressures
              </a>
            </td>
          <td>
            A. Chakravarthy, Elisa Pasini, Xun Zhao, Jeffrey To, Shu Yi (Roxana) Shen, Sandra Fischer, A. Ghanekar, Arndt Vogel, Robert C. Grant, Jennifer J Knox, Gonzalo Sapisochin, Gregory J. Gores, Daniel De Carvalho, Mamatha Bhat
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="INTRODUCTION
Tumor cell heterogeneity poses a challenge to monotherapy, as a single drug cannot kill all heterogeneous cancer cells of a tumor. The surviving cells may develop resistance, potentially leading to tumor recurrence. The combination therapy targets the disease through multiple mechanisms. Combinations of histone deacetylase (HDAC) inhibitor(s) with other inhibitor(s) have represented promising cancer chemotherapeutics by altering the epigenetic landscape of cancer cells by restoring acetylation and reactivating tumor suppressor genes, leading to cell cycle arrest, promoting apoptosis, and thus inhibiting cancer cell proliferation.


AREAS COVERED
The patent literature (2020-present) on rational combinations of HDAC inhibitor(s) for cancer chemotherapy has been searched and reviewed from Google Patents, Patentscope, Espacenet, WIPO, and USPTO to provide an expert opinion on rational combinations for improved, optimized, and precise cancer therapy. This first part of a two-part review, highlights the patent granted for the combination.


EXPERT OPINION
The combination of HDAC inhibitors with other inhibitors, including Janus kinase (JAK), aurora kinase, tubulin, Sirtuin 2, and/or topoisomerase inhibitors, demonstrated a synergistic anti-cancer effect. Dual and multi-target inhibitors showed enhanced therapeutic efficacy in relapsed and refractory cases characterized by epigenetic dysregulation via histone modifications and alterations that contribute to disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab733a8835d2537eb7a1dc0f915fffde2f98ed35" target='_blank'>
              An updated patent review on rational combinations of HDAC inhibitors for cancer chemotherapy (2020 - present): part 1-patent granted.
              </a>
            </td>
          <td>
            Yugal Kishor Shukla, Vandana, Vivekananda Mandal, V. Asati, Raj K. Keservani, Sanjay Kumar Bharti
          </td>
          <td>2025-08-12</td>
          <td>Expert opinion on therapeutic patents</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The aberrant expression of RNA binding proteins (RBPs) is linked to various diseases, including cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC). However, the single-cell landscape of RBPs in CESC remains unclear. We analyzed single-cell data from 2 HPV + and 2 HPV- CESC samples to assess cell subtype composition and differential gene expression. Meanwhile, based on 2,141 reported RBPs, the expression patterns of different cell type-specific RBPs were further investigated. SUVA software was used to perform differential alternative splicing analysis on the TCGA CESC data. Finally, we did a lot of follow-up work and immunohistochemical studies to understand the true expression of RBP and its correlation with CESC prognosis. We observed a significant increase in the proportion of epithelial cells in the HPV + sample. The expression of RBP exhibited heterogeneity between HPV + and HPV- samples, with epithelial cells demonstrating the most diverse composition of RBP-expressing cell clusters. Results indicated that RNA splicing events were strongly associated with CESC progression. Covariation analysis identified 4 RBPs (CSTB, TIPARP, NDRG1, and NDRG2) correlated with 25 RNA alternative splicing events. Immunohistochemical experiments revealed that TIPARP expression was down-regulated in HPV + cervical cancer(p = 0.033). And the prognostic model that includes all RBPs, stage, and treatments may has have certain practical guiding significance for patients with CESC. Our study indicates that RBPs display specific expression patterns across HPV + and HPV- CESC. The regulatory checkpoints of these RBPs may serve as potential markers and therapeutic targets for the detection of HPV + CESC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14823-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4339b9f80b067b996e103fd9ccc8c10be3c72e47" target='_blank'>
              Single-cell and bulk transcriptome profiling reveals RNA-binding protein regulatory programs in cervical cancer
              </a>
            </td>
          <td>
            Shasha Yang, Jingjing Zeng, Yong Wang, Yudi Tan, Li Yu, Jinkong Wei, Yuying Wei, Yuxuan Jin, Junying Chen
          </td>
          <td>2025-08-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Spermatogenesis is a pivotal biological process for the precise transmission of paternal genetic information, governed by a highly complex and dynamically regulated testicular microenvironment. Although mammalian research has characterized germ cell development at the single-cell level, differences in reproductive strategies limit the relevance of these findings to avian species. Results We employed single-cell RNA sequencing (scRNA-seq) to analyze the testes of the “Yufen 1” H line roosters at five distinct developmental stages: birth, rapid testicular development, sexual maturity, physical maturity, and senescence. By constructing a single-cell transcriptomic atlas, we identified ten somatic cell subtypes and four germ cell subtypes, thereby elucidating the dynamic changes in gene expression during spermatogenesis. Notably, our findings indicate that meiosis initiates relatively early in chickens, with the formation of the blood-testis barrier being closely associated with pachytene spermatocytes. Additionally, the testicular microenvironment undergoes age-related adaptive changes. Furthermore, we observed that support cells at 20 and 80 weeks of age exhibit similar transcriptional profiles, while macrophages and T cells play a pivotal role in the formation of the testicular cords and vascular networks during the early developmental stages. Conclusion This study offers a comprehensive atlas of testicular development in chickens, elucidating the sequential cell fate transitions from spermatogonial stem cells to mature sperm, alongside the dynamic and intricate developmental trajectories of somatic cells within the testicular microenvironment. These findings present novel insights into avian testicular development and establish a theoretical foundation for future research in reproductive biology and breeding strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11944-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318aed180cb59e08dde10af23502e4ddc7594b71" target='_blank'>
              Single-cell RNA sequencing uncovers dynamic roadmap during chicken spermatogenesis
              </a>
            </td>
          <td>
            Haishan Guo, Jianing Liu, Mingzhen Xu, Tingqi Zhu, Shuangxing Liu, Pingquan Liu, Huayuan Liu, Shi Tang, Zhunan Li, Weiwei Jin, Donghua Li, Yadong Tian, Xiangtao Kang, Guirong Sun
          </td>
          <td>2025-08-13</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="As the use of molecular profiling of tumors expands, cancer diagnosis, prognosis, and treatment planning increasingly rely on the information it provides. Although primarily designed to detect somatic variants, next-generation sequencing (NGS) tumor-based profiling also identifies germline DNA alterations, necessitating careful clinical interpretation of the data. Traditionally, germline risk testing has depended on prioritizing individuals based on physical exam findings consistent with known hereditary cancer syndromes, tumor-specific features, age at diagnosis, personal history, and family history. As NGS-based molecular profiling is used increasingly to diagnose, prognosticate, and follow cancer progression, DNA variants that are likely to be of germline origin are identified with increased frequency. Because pathogenic/likely pathogenic germline variants are critical biomarkers for risk stratification and treatment planning, consensus guidelines are expanding to recommend comprehensive germline testing for more cancer patients. This Review highlights the nuances of identifying DNA variants of potential germline origin incidentally at the time of NGS-based molecular profiling and emphasizes key differences between comprehensive germline versus tumor-based platforms, sample types, and analytical methodologies. In the growing era of precision oncology, clinicians should be adept at navigating these distinctions to optimize testing strategies and leverage insights regarding germline cancer risk surveillance and management for all people with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0232380705d61c18060a7a21764109d7d97d9def" target='_blank'>
              Detecting likely germline variants during tumor-based molecular profiling
              </a>
            </td>
          <td>
            Diana Jaber, Jessica Zhang, Lucy A. Godley
          </td>
          <td>2025-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id=" 
Background: Accurate and early identification of lung cancer biomarkers is crucial for effective management yet remains a clinical challenge. CRISPR-based technologies provide highly sensitive and specific tools that can be adapted for both diagnostic and therapeutic use in oncology.
Methods: A prospective study was conducted including 120 patients with confirmed lung cancer and 60 healthy individuals as controls at a tertiary care center. A CRISPR-Cas12a diagnostic platform was developed to detect EGFR exon 19 deletions, KRAS G12C mutations, and ALK rearrangements in tumor tissue and plasma samples. In parallel, CRISPR-Cas9 editing was applied to patient-derived lung cancer cell cultures to selectively disrupt EGFR mutant alleles. The diagnostic performance of the CRISPR assay was compared against conventional PCR and next-generation sequencing (NGS).
Results: The CRISPR-Cas12a assay successfully detected as few as 10 DNA copies/µL within one hour. In tissue samples, the assay achieved 96.7% sensitivity and 98.3% specificity relative to NGS. In circulating tumor DNA extracted from plasma, sensitivity reached 92.1%. Functional assays demonstrated that CRISPR-Cas9-mediated disruption of EGFR mutations reduced cell viability by approximately 65% and suppressed downstream oncogenic signaling pathways.
Conclusion: CRISPR-based strategies demonstrate considerable promise for both precise detection and targeted intervention in lung cancer. The integration of CRISPR diagnostics with gene-editing applications could significantly advance precision oncology by enabling earlier diagnosis and patient-tailored therapeutic options.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b66e666da19106418ca11e5bfbe956b66a8dd301" target='_blank'>
              CRISPR-BASED APPROACHES FOR THE DETECTION AND TARGETING OF LUNG CANCER BIOMARKERS: A NEW ERA IN PRECISION ONCOLOGY
              </a>
            </td>
          <td>
            Ibtasam Mazhar, Maimona Sadia, Muhammad Hassam Saleem, Ateeqah Siddique, Areeba Musferah, Areej Safdar, Anood Malik
          </td>
          <td>2025-08-31</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 p53 is a transcription factor that regulates the expression of target genes involved in anti-proliferative processes such as cell cycle arrest, apoptosis, and senescence, thereby establishing its canonical role as a tumor suppressor. Due to its substantial network of target genes, p53 is also involved in non-canonical tumor suppression pathways that are yet to be fully explored. During the search of these pathways, we identified one of the methyltransferases as a novel p53 target gene. It’s been reported that this methyltransferase is downregulated in various cancers, suggesting potential tumor suppressive function. However, the specific target of this methyltransferase and its role in the mechanism of p53-mediated tumor suppression remains elusive. In this study, we show that p53 directly binds to intron 1 of this methyltransferase gene and activates its transcription and protein expression. Furthermore, we identify dihydrolipoic acid (DHLA), an important redox cofactor, as a novel target of this methyltransferase. We show that this methyltransferase enhances cellular sensitivity to oxidative stress-induced cell death by potentially methylating DHLA. Our studies reveal a novel role for p53 in regulating the well-known antioxidant DHLA to control cell death and suggest potential therapeutic strategies in promoting cancer cell death through p53 activation drugs.



 Fatima Alaydaroos, Yoko Itahana, Koji Itahana. p53-induced Methyltransferase Increases Cancer Cell Vulnerability to Cell Death [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P18.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1c8311abf9a4ec56ed8b5af3129410222f89d5" target='_blank'>
              Abstract P18: p53-induced Methyltransferase Increases Cancer Cell Vulnerability to Cell Death
              </a>
            </td>
          <td>
            Fatima Alaydaroos, Yoko Itahana, Koji Itahana
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The prevalence and death rates of early-onset colorectal cancer (EOCRC) have been increasing at an alarming rate since 1994. Compared with late-onset colorectal cancer (LOCRC), EOCRC is more aggressive and resistant to treatment. Despite
 KRAS
 mutations and the mitogen-activated protein kinase/extracellular signal-related kinase (MAPK/ERK) pathway's significant role in colorectal cancer tumorigenesis and progression, these mutations are less frequent in patients with EOCRC.




 In this study, we conducted proteotranscriptomic profiling of two independent cohorts (MD Anderson for testing and The Cancer Genome Atlas for validation), comparing EOCRC and LOCRC tumors. Our focus was on sporadic, microsatellite-stable, treatment-naïve patients with early-onset (<50 years) and late-onset (>60 years) colorectal adenocarcinomas.




 Our analysis revealed consistent molecular differences between EOCRC and LOCRC, particularly in pathway enrichment profiles. The two pathways with the highest levels of significant gene set enrichment analysis in EOCRC tumors were the downregulation of epithelial-mesenchymal transition and upregulation of Wnt/β-catenin signaling pathways. Both testing and validation cohorts showed a consistent enrichment in hallmark KRAS signaling. However, using the novel KRAS-ERK gene signatures to better capture the pathway signaling functionality, we identified a consistent upregulation of MAPK signaling specifically in EOCRC tumors. Further analysis of this enrichment in wild-type RAS GTPase (
 KRAS
 ,
 NRAS
 , and
 HRAS
 ) tumors confirmed significant MAPK upregulation in EOCRC wild-type RAS tumors (
 P
 < .001). Proteomic analysis further supported these transcriptomic findings, reinforcing the molecular distinctions in EOCRC.




 This study not only elucidates the unique molecular attributes of EOCRC but also lays the groundwork for exploring MAPK-targeted therapies specifically for this age group. Given the increasing prevalence of EOCRC, our findings highlight the need for larger investigations into MAPK upregulation in EOCRC and its potential as a target for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5365b8787a6860734caebe70b8eefa54d1dc39aa" target='_blank'>
              Proteotranscriptomic Analyses of Microsatellite-Stable Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            J. Alshenaifi, Giulia Maddalena, Jessica C. Lal, Oluwadara Coker, Hey Min Lee, O. Villarreal, G. Manyam, Alaa M. Mohamed, G. Vetere, Micheal White, N. Ajami, Preeti Kanikarla Marie, Alexey Sorokin, David Menter, Y. N. You, Andy Futreal, S. Kopetz
          </td>
          <td>2025-08-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the second leading cause of cancer deaths worldwide, with roughly 41% of CRC cases harboring a KRAS mutation. Acquired resistance to KRAS-targeted treatments has occurred with mechanisms including increased HER family expression among other receptor tyrosine kinases. HER3, a member of the HER family that is kinase impaired, has been shown to be a resistance mechanism upon inhibition of the HER family and downstream targets including RAS/MEK/ERK and PI3K/AKT. We find that KRAS mutations tend to co-occur with HER3 alterations in a large panel of cancers and in CRCs. Our results show that both total and activated HER3 levels increase in CRC patient derived organoids and cell lines after treatment with KRASG12D targeted agents, indicating that HER3 could be a potential adaptive response mechanism to KRAS-targeted therapy. Further, we found that genetic knock-down of KRAS and HER3 resulted in a reduction in growth of CRC cells compared to single knockdown of either KRAS or HER3. We observed that kinase impaired HER3 binding partners, as assessed by immunoprecipitation, is cell dependent with EGFR binding HER3 in one cell line. After co-treating CRC cells with pan-HER inhibitors in combination with MRTX1133, a KRASG12D inhibitor, synergistic and additive effects in reduction in cell growth were observed. Finally, we found that co-targeting KRASG12D mutant cells with a KRASG12D inhibitor and a HER3 antibody-drug conjugate further reduced cell viability. We posit that co-targeting both KRASG12D and HER3, whether directly or indirectly, is a potential therapeutic strategy in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dddc145036e05434447d0cbfa960ac516707627a" target='_blank'>
              Co-Targeting KRASG12D and the HER family is efficacious in colorectal cancer.
              </a>
            </td>
          <td>
            Mary Kate Kilroy-Gehret, Cecilia Wischmeier, SoYoung Park, Daniel Choi, Wasim Feroz, Rosalin Mishra, J. Garrett
          </td>
          <td>2025-07-31</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The activation of cellular death programs does not necessarily predetermine an inevitable outcome. Identifying the precise moment when a cell irreversibly transitions from life to death presents a significant challenge in its assessment and measurement. In this review, we explore the critical alterations in cellular structures that have been proposed as the point-of-no-return. Using glioblastoma as a model—one of the most aggressive and lethal tumor types with a remarkable ability to evade cell death—we highlight the challenge of reaching the point-of-no-return. Glioblastoma cells often exhibit impaired function of the apoptotic endonuclease, DFF40/CAD/CPAN, leading to incomplete apoptosis and genomic instability. The sublethal activation of DFF40/CAD/CPAN not only allows tumor cells to survive but can also drive more aggressive phenotypes and enhance therapeutic resistance. We underscore the need to reassess glioblastoma treatment strategies from broad cytotoxic approaches to more targeted therapies that exploit specific vulnerabilities within regulated cell death (RCD) pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1636e6d9600419681b175aaf823d3301df5b0b8e" target='_blank'>
              Reaching the point-of-no-return: The cornerstone of glioblastoma treatment?
              </a>
            </td>
          <td>
            M. Alemany, Jordi Bruna, Victor J Yuste
          </td>
          <td>2025-08-06</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background the STS is a rare type of tumor, and although its treatment has improved greatly in recent decades, the treatment of STS and the development of new drugs remains a major challenge. A new identification of prognostic biomarkers that reflect the biological heterogeneity of STS could therefore lead to better interventions for STS patients. In recent years, there has been a growing interest in the investigation of the impact of immune-related genes on cancer prognosis. Methods based on RNA-seq data obtained from TCGA-STS and GTEx patients, differential expression analysis, consensus clustering, enrichment analysis, tumor microenvironment assessment, risk model construction and other data analysis were performed. Last but not least, CALR, a central regulator inSTS, demonstrated oncogenic properties through overexpression/knockdown assays, supported by qRT-PCR and immunofluorescence data. Results we constructed a prognostic model containing 8 IRGs for predicting STS prognosis by using the LASSO regression. Furthermore, the samples were categorized as either high-risk or low-risk based on the risk score computed by the model. Additionally, we compared the tumor microenvironment of STS samples using the ESTIMATE and CIBERSORT algorithms. Last, our experimental results proved that CALR was up-regulated in sarcoma cells compared to in normal cell. Conclusions conclusively, IRGPM is a promising immune-related prognostic biomarker. As a prognostic indicator of immunotherapy, IRGPM might also help differentiate molecular and immune characteristics in STS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c100dde99406f8a76a8fbd31710984fd85b6beba" target='_blank'>
              Identification and comprehensive analysis of an immune-related gene prognostic model for indicating tumor immune microenvironment features in soft tissue sarcoma
              </a>
            </td>
          <td>
            Shizhao Xu, Yu-qiang Liu, Qian Cai, Jianzhi Sun, Xuefeng Guan
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, Hisato Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539ae3d242133cec21106a2f8dc4ec07c55b0985" target='_blank'>
              Data-Driven Discovery of a Key Regulator Cyclin A2 as a Promising Therapeutic Target in Hormone-Sensitive Cancers.
              </a>
            </td>
          <td>
            Suvitha Anbarasu, Anand Anbarasu
          </td>
          <td>2025-09-11</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE Tumor comprehensive genomic profiling (CGP) may detect potential germline pathogenic/likely pathogenic (P/LP) alterations as secondary findings. We analyzed the frequency of potentially germline variants and large rearrangements (LRs) in the RATIONAL study, an Italian multicenter, observational clinical trial that collects next-generation sequencing–based tumor profiling data, and evaluated how these findings were managed by the enrolling centers. PATIENTS AND METHODS Patients prospectively enrolled in the pathway-B of the RATIONAL study and undergoing CGP with the FoundationOne CDx assays were included in the analysis. Potentially germline variants detected in 40 cancer susceptibility genes (CSGs) were classified in three classes with different actionability, most (MA), high (HA), and standard (SA), on the basis of penetrance, mutational spectrum, and intervention for prevention/early detection. RESULTS On the basis of the European Society of Medical Oncology recommendations, we identified 225 potentially germline P/LP variants in 193/1,339 (14.4%) enrolled patients. In particular, 62/225 (27.5%) variants were detected in genes classified as MA-CSG class, 53/225 (23.6%) in genes belonging to the HA-CSG class, and 110/225 (48.9%) in the SA-CSG class. In addition, we detected 58 LRs in the 16/40 CSGs in 53/1,339 (3.95%) patients. Information about germline-focused analysis and follow-up was available for 99 patients with potentially germline variants. Surprisingly, 95/99 (96%) patients were not referred to oncogenetic consultation and follow-up, including 30/32 (93.75%) patients with variants in the MA-CSG class. CONCLUSION Our data confirm the utility of CGP for the identification of potentially germline variants in CSGs, highlighting the importance of reporting LRs in addition to single-nucleotide variants and insertions/deletions. However, our findings also demonstrate a relative lack of knowledge of the implications of germline findings detected on tumor-only sequencing among oncologists and underline the need for specific training in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3205dd2d3d3b99613c9aba83f72f519f1c6e231" target='_blank'>
              Germline Findings From Tumor-Only Comprehensive Genomic Profiling in the RATIONAL Study: A Missed Opportunity?
              </a>
            </td>
          <td>
            R. Esposito Abate, Alessandro Morabito, Michele Milella, F. Tabbò, Valentina Guarneri, G. Pelizzari, I. Rapposelli, R. Berardi, Lucio Buffoni, Elisa Bennicelli, F. Zanelli, C. Tondini, Laura Attademo, T. Latiano, S. Corallo, Giancarlo Pruneri, F. Marmorino, O. Caffo, L. Antonuzzo, Simona Tessitore, S. Novello, G. Curigliano, Carmine Pinto, Nicola Normanno, Antonella De Luca
          </td>
          <td>2025-09-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Purpose The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Methods Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type-tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Results Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. Conclusion: These findings may provide guidance for development of clinical trial combination regimens including cirtuvivint, CC-671 or iadademstat. Full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd17fd73c746ccc133073188d85cd1b611794f0" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H. Doroshow
          </td>
          <td>2025-09-12</td>
          <td>Cancer Chemotherapy and Pharmacology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer.
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Abstract Recombination plays a pivotal role in generating within-host diversity and enabling HIV’s evolutionary success, particularly in evading the host immune response. Despite this, the variability in recombination rates across different settings and the underlying factors that drive these differences remain poorly understood. In this study, we analysed a large dataset encompassing hundreds of untreated, longitudinally sampled infections using both whole-genome long-read and short-read sequencing datasets. By quantifying recombination rates, we uncover substantial variation across subtypes, viral loads, and stages of infection. We also map recombination hot and cold spots across the genome using a sliding window approach, finding that previously reported inter-subtype regions of high or low recombination are replicated at the within-host level. Importantly, our findings reveal the significant influence of selection on recombination, showing that the presence and success of recombinant genomes is strongly interconnected with the fitness landscape. These results offer valuable insights into the contribution of recombination to evolutionary dynamics and demonstrate the enhanced resolution that long-read sequencing offers for studying viral evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33ee9e95b99508db7397f8d4a5fee56772922cf3" target='_blank'>
              Estimates of HIV-1 within-host recombination rates across the whole genome
              </a>
            </td>
          <td>
            Harriet Longley, D. Bonsall, Joshua T. Herbeck, G. Macintyre-Cockett, Sandra E Chaudron, L. Thomson, Nicholas E. Grayson, Andrew Mujugira, Christophe Fraser, J. Lingappa, K. Lythgoe
          </td>
          <td>2025-07-25</td>
          <td>Virus Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The SHCBP1 (SHC SH2-domain-binding protein 1) is identified as an important regulator of cancer biology, participating in the modulation of multiple cancer hallmarks. Initially discovered as a component of the mitotic midbody essential for cytokinesis, SHCBP1 is now recognized for orchestrating a broad spectrum of oncogenic processes such as persistent proliferation, apoptosis resistance, epithelial–mesenchymal transition, and immune system evasion. This review comprehensively explores the molecular features of SHCBP1, its regulatory networks, and its multifaceted roles in cancer progression. SHCBP1 is commonly overexpressed in diverse cancers, with elevated expression levels strongly associated with more aggressive tumors and unfavorable patient prognosis. Mechanistically, SHCBP1 serves as a potential mediator of oncogenic signaling pathways, thereby regulating mitotic processes, transcriptional alterations, and cytoskeletal reorganization. In addition to its biological functions, SHCBP1 offers translational promise as a prognostic marker and a prospective therapeutic target. Preclinical models indicate that genetic depletion or pharmacologic disruption of SHCBP1 limits tumor growth, increases sensitivity to chemotherapy, and reduces metastatic capacity. Despite significant progress, the development of selective SHCBP1 inhibitors remain challenging areas. This review summarizes SHCBP1’s diverse roles in tumor pathogenesis and outlines future research directions to develop SHCBP1-targeted strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf0d05ca742d0377d12c88b1478764c0210782e5" target='_blank'>
              The Roles of SHCBP1 in Cancer Hallmarks: Molecular Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Hye-Youn Kim, Ye-Jin Park, Soyeon Ryu, Suntaek Hong
          </td>
          <td>2025-09-09</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, Divij Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z. Lin, Dylan Schaff, Sydney M. Shaffer, Drew M Pardoll, Christina Jackson, Nancy R. Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Cytosine DNA methylation patterns vary widely across eukaryotes, with its ancestral roles being understood to have included both transposable element (TE) silencing and host gene regulation. To further explore these claims, in this study, we reevaluate the evolutionary origins of DNA methyltransferases and characterize the roles of cytosine methylation on underexplored lineages, including the amoebozoan Acanthamoeba castellanii, the glaucophyte Cyanophora paradoxa, and the heterolobosean Naegleria gruberi. Our analysis of DNA methyltransferase evolution reveals a rich ancestral eukaryotic repertoire, with several eukaryotic lineages likely subsequently acquiring enzymes through lateral gene transfer (LGT). In the three species examined, DNA methylation is enriched on young TEs and silenced genes, suggesting an ancestral repressive function, without the transcription-linked gene body methylation of plants and animals. Consistent with this link with silencing, methylated genomic regions co-localize with heterochromatin marks, including H3K9me3 and H3K27me3. Notably, the closest homologs of many of the silenced, methylated genes in diverse eukaryotes belong to viruses, including giant viruses. Given the widespread occurrence of this pattern across diverse eukaryotic groups, we propose that cytosine methylation was a silencing mechanism originally acquired from bacterial donors, which was used to mitigate the expression of both transposable and viral elements, and that this function may persist in creating a permissive atmosphere for LGT in diverse eukaryotic lineages. These findings further highlight the importance of epigenetic information to annotate eukaryotic genomes, as it helps delimit potentially adaptive LGTs from silenced parasitic elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48579c32d97033a0fd54ac2bedb8cb3bebb700ed" target='_blank'>
              Repressive Cytosine Methylation is a Marker of Viral Gene Transfer Across Divergent Eukaryotes
              </a>
            </td>
          <td>
            Luke A. Sarre, Giselle Azucena Gastellou Peralta, Pedro Romero Charria, Vladimir Ovchinnikov, Alex de Mendoza
          </td>
          <td>2025-07-25</td>
          <td>Molecular Biology and Evolution</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188c988056f790ffe3551531682c3604b9aecad" target='_blank'>
              Integrative Gene-Centric Analysis of Breast Cancer: High-Confidence Germline Predisposition Genes from Population-Scale Cohorts
              </a>
            </td>
          <td>
            R. Zucker, S. Schreiber, A. Stern, M. Linial
          </td>
          <td>2025-09-12</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. M. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Androgen receptor (AR) splice variants (AR-Vs) have emerged as potential resistance mechanisms to AR-targeted agents (ARTAs) in prostate cancer (PC), particularly in the context of castration-resistant disease. Among them, AR-V3 has been frequently detected, yet its biological function remains unclear due to conflicting results from initial studies. This study aimed to comprehensively investigate the molecular structure, activity, and clinical relevance of AR-V3. Methods We constructed plasmids encoding two AR-V3 isoforms—one newly identified isoform matching the human reference genome (AR-V3ref) and the other based on the 22Rv1 cell line sequence (AR-V322Rv1)—and transfected them into AR-negative PC-3 cells alone or co-expressed with AR full-length (AR-FL). Localization, transcriptional activity (via luciferase assays), and RNA sequencing were performed. Protein structure modeling was conducted using AlphaFold2. Nonsense-mediated decay (NMD) was assessed through pharmacological inhibition. Clinically, AR-V3 expression in circulating tumor cells (CTCs) from 65 patients starting ARTA treatment was analyzed in relation to progression-free survival (PFS) and overall survival (OS). Results RNA-seq of AR-FL + AR-V3 vs. AR-FL alone showed no AR-V3-specific gene expression. Structure modeling revealed poor overall prediction confidence, particularly in the N-terminal domain, with no consistent structural features differentiating AR-V3ref and AR-V322Rv1. Both isoforms localized mainly to the cytoplasm, regardless of hormonal stimulation or AR-FL co-expression. Neither isoform showed androgen receptor element (ARE) binding activity unless co-expressed with AR-FL. NMD analysis indicated neither isoform was degraded by this pathway. Clinically, AR-V3 + patients had significantly shorter OS (median 13 vs. 23 months; p = 0.02) among CTC + patients but showed no difference in PSA response or PFS under ARTA treatment. Conclusions Our data demonstrate that both AR-V3 isoforms are functionally inactive, lacking autonomous nuclear translocation or transcriptional activity. AR-V3 is not a substrate for NMD, and its protein structure remains poorly defined. While associated with worse overall survival, AR-V3 lacks predictive value for ARTA response, underscoring its limited utility as a biomarker. These findings emphasize the need for functional validation before integrating putative biomarkers into clinical practice. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03948-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb7c5021ac8d6b1f031194f8d6bc410625f1343" target='_blank'>
              Molecular analysis of androgen receptor splice variant AR-V3 reveals eminent ambiguity regarding activity and clinical utility
              </a>
            </td>
          <td>
            Neele Wüstmann, Verena Humberg, Julia Vieler, K. Seifert, Martin Bögemann, K. Schlack, A. J. Schrader, Christof Bernemann
          </td>
          <td>2025-08-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play complex roles in the tumor microenvironment (TME) of melanoma. However, their impact on prognosis and treatment response in melanoma remains unclear. In this study, ScRNA-seq data (GSE115978) were utilized to characterize CAF heterogeneity and identify marker genes in melanoma. Prognostic CAF genes were identified from the TCGA dataset and employed to construct a risk signature, which was subsequently validated in an independent cohort (GSE65904). Mutation, copy number variation (CNV), pathway enrichment, immune infiltration, and drug sensitivity were analyzed to determine the signature’s clinical relevance. Immunohistochemistry (IHC), immunofluorescence (IF), and qPCR were performed to validate the expression of CAF signatures on clinical melanoma samples. We identified CAFs in patients with melanoma through single-cell RNA sequencing data. A 28-gene CAF signature was constructed using the Least Absolute Shrinkage and Selection Operator (LASSO) regression based on 271 prognostic CAF genes. This signature demonstrated excellent prediction accuracy for survival, with area under the curve (AUC) values of 0.737, 0.737, and 0.779 for 1-year, 3-year, and 5-year survival, respectively. The signature was an independent prognostic factor and was correlated with CNVs, and immunosuppressive TME features (reduced CD8+ T cells, M1 macrophages). Additionally, our CAF signature could predict the efficacy of multiple chemotherapy drugs and serve as a potential prognostic marker for immunotherapy. Experimental validation confirmed the expression of CAF signature genes in melanoma tissue. Our model may help predict the prognosis and response to chemotherapy and immunotherapy in patients diagnosed with melanoma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14979-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2590463eef329c26ec20b4fd4f71fd42d9d7d1d3" target='_blank'>
              Combining single-cell and bulk RNA sequencing to identify CAF-related signature for prognostic prediction and treatment response in patients with melanoma
              </a>
            </td>
          <td>
            Jing Wang, Ying Song, Zifu Li, Tianxiang Gao, Wei Shen, Zijian Kang, Chong Xu
          </td>
          <td>2025-08-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite extensive transcriptomic alterations observed in tumors, the global landscape of isoform-level alternative splicing in cancer remains largely unexplored. Leveraging long-read sequencing, we successfully identified and characterized full-length isoforms, along with tumor-specific splicing events in non-small cell lung cancer (NSCLC). Our analysis identified 38,058 previously unannotated isoforms, which were subsequently validated using orthogonal multi-omics datasets to confirm their transcriptional and translational activities. Notably, 269 splicing events were characterized as tumor-specific, with 17 showing significant associations with NSCLC subtypes, while 13 were enriched across all NSCLC cases. Among these events, skipped exons in IFI27, PUF60 and ANAPC11, as well as an alternative first exon in YBEY, were absent from GENCODE annotations. These findings underscore the intricate complexity of isoforms and their clinical significance, particularly in terms of NSCLC subtype specificity and their potential as therapeutic targets. In conclusion, this study provides a valuable resource for the discovery of tumor-specific splicing targets in NSCLC, leveraging the unique advantages of long-read sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee33e3741376f135abd3727ea0d3306f3ece04e5" target='_blank'>
              Long-Read Sequencing Reveals Tumor-Specific Splicing Isoforms as Therapeutic Targets In NSCLC.
              </a>
            </td>
          <td>
            Yifei Li, Liying Zhou, Hexin Li, Gaoyuan Sun, Siyuan Xu, Xiaokun Tang, Li Wan, Lili Zhang, Min Tang
          </td>
          <td>2025-08-25</td>
          <td>American journal of respiratory cell and molecular biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Alterations to N6-methyladenosine modifications can promote malignant progression by modulating gene expression through regulation of transcript metabolism. Quantifying the causal impact of m6A dysregulation at the population level could help to guide personalized therapeutic interventions. Here, we developed a causal framework that enables precise estimation of the driving effects (DE) of m6A dysregulation on tumor survival, establishing DE-based enhanced prioritization rules for anti-m6A therapies. The global average DE of m6A dysregulation across 9,647 tumors resulted in a decrease in overall survival of 180.1 days considering a 5-year survival endpoint, suggesting potential overall benefits of treatment. Profiling of tumors most susceptible to m6A dysregulation and application of a modifier-mining tool revealed an effect modification in the PI3K/AKT/mTOR pathway. Specifically, the benefits of m6A targeted treatment varied depending on baseline mTOR levels, which was validated in vitro and in vivo. Overall, focusing on m6A, this study established a paradigm leveraging DE over conventional risk to optimize personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162d993b80f3bf7514bb1cd84bb57a415070ddc9" target='_blank'>
              Heterogeneous Driving Effects Guide Personalized Tumor Treatments Targeting N6-methyladenosine.
              </a>
            </td>
          <td>
            Xudong Mao, Zhehao Xu, Xingbo Yang, Shihan Chen, Liyang Li, Zeyi Lu, Haohua Lu, Yudong Lin, Ruyue Wang, Yang Li, Fan Li, Lifeng Ding, Wenqin Luo, Xianjiong Chen, Yi Lu, Ziwei Zhu, Xinyu Zhao, Zhifan Ding, Liqun Xia, Qi Liu, Gonghui Li
          </td>
          <td>2025-07-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor immunity has emerged as a focal point for cancer research. Although tumor immunotherapy represents a promising approach to cancer treatment, its effectiveness is often hindered by the heterogeneity of the tumor microenvironment (TME) and immune escape mechanisms. In recent years, small non-coding RNAs (sncRNAs) have attracted increasing attention in tumor immunology due to their essential role in gene regulation. This review systematically examines the multifaceted regulatory functions of sncRNAs in tumor immunity, with a focus on six major subtypes: microRNA, siRNA, piRNA, snoRNA, tsRNA, and snRNA. The molecular mechanisms by which these sncRNAs reshape the TME are discussed, including their roles in modulating immune cell differentiation (e.g., T cell polarization, macrophage phenotype transition), regulating immune checkpoint expression (PD-1/PD-L1, CTLA-4, Tim-3, LAG-3), and influencing tumor antigen presentation. This review also explores the dynamic network through which sncRNAs contribute to tumor immune escape. Furthermore, this study highlights the clinical potential of sncRNAs as liquid biopsy biomarkers and their application prospects in therapeutic strategies, such as targeted silencing of immunosuppressive molecules via nano-delivery systems, combination treatments with radiotherapy and chemotherapy, and Chimeric Antigen Receptor T-cell (CAR-T) therapy. Despite current challenges, including limited delivery efficiency and off-target effects, emerging technologies like AI-assisted sequence design and organ-on-a-chip models present new opportunities for clinical translation. This comprehensive review provides a theoretical foundation and translational insights for elucidating the functional network of sncRNAs in tumor immunology and advancing precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dcee9c888595b1a7bd3be0ebd8ff37c9ea4926" target='_blank'>
              Small non-coding RNAs: key regulatory factors and potential therapeutic targets in tumor immunity
              </a>
            </td>
          <td>
            Zihan Liu, Haotian Dong, Chengyuan Ye, Jianing Yan, Min Miao, Y. Shao
          </td>
          <td>2025-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47887f5c85f735acc88fc608cab90528d242a67" target='_blank'>
              Baseline expression of c-Myc defines the tissue specificity of oncogenic K-Ras.
              </a>
            </td>
          <td>
            Olesja Popow, Qing Yu, Clarance Yapp, Shannon Hull, Joao A. Paulo, Benjamin Hanna, Shidong Xu, Y. Kuang, Carlie Sigel, C. Paweletz, Steve P Gygi, Kevin M. Haigis
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) has emerged as a revolutionary weapon in oncology, particularly in breast cancer, enabling precise mutant profiles and the evolution of individualized treatment systems. NGS-based mutant profiling in breast cancer, contributing to the development of a better understanding of familial variations and their results in clinical practice. The NGS makes it possible to call multiple genetic variations, including the well-known BRCA1/2 gene, as well as a fresh variation that may influence the curative response. Despite its constancy, there are still several impediments to NGS integration into routine clinical practice, including data interpretation, cost, ease of use, and insufficient standard protocols. It is necessary to validate NGS results and translate them into capable, personalized treatment, hands-on statistics, and clinical trials. Reverence must also be accorded to the fair results of family testing, in particular about incidental consequences. To ensure that all patients benefit from the personalized therapy, the future of NGS in breast cancer lies in exultant these problems and improving productivity. NGS is capable of redefining breast cancer medicines, providing a powerful, target therapy based on human characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9434adbda6b0accb6c46fb07821f72ebd73a1d93" target='_blank'>
              NGS-DRIVEN MUTATION PROFILING IN BREAST CANCER: BRIDGING THE GAP BETWEEN REAL-WORLD DATA AND PERSONALIZED THERAPY
              </a>
            </td>
          <td>
            S. Malik, A. Malik, J. Islam, A. Zahid, J. Iqbal, M. Marvi, Q. Ali, A. Fatima
          </td>
          <td>2025-07-30</td>
          <td>Bulletin of Biological and Allied Sciences Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lineage switching (LS) is the conversion of cancer cell lineage during the course of a disease. LS in leukemia cell lineage facilitates cancer cells escaping targeting strategy like CD19 targeted immunotherapy. However, the genetic and biological mechanisms underlying immune evasion by LS leukemia cells are not well understood. Here, we conduct a multi-omics analysis of patient samples and find that lineage-switched acute myeloid leukemia (LS AML) cells with KMT2A rearrangement (KMT2A-r) possess monocytic myeloid derived suppressor cell (M-MDSC)-like characteristics. Single-cell mass cytometry analysis reveals an increase in the M-MDSC like LS AML as compared to those of lineage-consistent KMT2A-r AML, and single-cell transcriptomics identify distinct expression patterns of immunoregulatory genes within this population. Furthermore, in vitro assays confirm the immunosuppressive capacity of LS AML cells against T cells, which is analogous to that of MDSCs. These data provide insight into the immunological aspects of the complex pathogenesis of LS AML, as well as development of future treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3db080fb3d59476095f43db762e37a90778795" target='_blank'>
              Multi-omics analysis identifies an M-MDSC-like immunosuppressive phenotype in lineage-switched AML with KMT2A rearrangement
              </a>
            </td>
          <td>
            Takashi Mikami, I. Kato, Akira Nishimura, M. Eguchi-Ishimae, T. Kamitori, Keiji Tasaka, Hirohito Kubota, Tomoya Isobe, Yoshinori Uchihara, Yui Namikawa, Satoru Hamada, Shinichi Tsujimoto, Shotaro Inoue, Takayuki Hamabata, K. Izawa, T. Miyamura, Daisuke Tomizawa, Toshihiko Imamura, Hidemi Toyoda, Mariko Eguchi, Hiroaki Goto, Seishi Ogawa, Masatoshi Takagi, James B. Wing, Junko Takita
          </td>
          <td>2025-08-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68e6c79788b842ce0d6d08c58f642ad054ed8c2" target='_blank'>
              Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue
              </a>
            </td>
          <td>
            Z. Chen, Sankha Subhra Das, Asha Kar, S. H. T. Lee, Kevin D. Abuhanna, Marcus Alvarez, Mihir G. Sukhatme, Zitian Wang, Kyla Z. Gelev, Matthew G. Heffel, Yi Zhang, Oren Avram, Elior Rahmani, S. Sankararaman, Markku Laakso, S. Heinonen, H. Peltoniemi, Eran Halperin, Kirsi H Pietiläinen, Chongyuan Luo, P. Pajukanta
          </td>
          <td>2025-08-20</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Objective The development of acquired endocrine resistance and reduced chemosensitivity in oestrogen receptor-positive (ER+) breast cancer presents significant challenges. Microtubule-based process-related genes (MBPRGs) play essential biological roles in the cell cycle and the development of migration. This study aimed to establish a novel prognostic signature based on MBPRGs to improve patient outcomes and offer additional treatment options for those with ER+ breast cancer. Methods Clinical data along with relevant RNA information with ER+ breast cancer were sourced from The Cancer Genome Atlas and the Molecular Taxonomy of Breast Cancer International Consortium. Consensus clustering was subsequently utilised to identify new molecular subgroups. Evaluations of the tumour immune microenvironment and immune status of these subgroups were performed via ESTIMATE, CIBERSORT, MCP, and ssGSEA. Additionally, functional analyses were conducted to investigate the underlying mechanisms involved. Prognostic risk models were developed via random forest, support vector machines and the least absolute shrinkage and selection operator algorithm. Single-cell analysis revealed differences in the expression levels of key genes among various cell types. Western blotting was used to measure protein levels in breast cancer cell lines. Immunohistochemical staining was used to assess protein expression in paraffin-embedded tissues, and Kaplan–Meier survival curves were generated to evaluate survival differences between the high- and low-expression groups of key genes. Transwell and cell viability assays were used to examine the biological functions of CHORDC1. Results Two molecular subgroups with significantly different survival outcomes were identified. Longer survival was linked to a high immune score, low tumour purity, a greater presence of immune infiltrating cells, and an overall positive immune status. Risk models derived from MBPRGs exhibited strong potential for predicting survival in patients with ER+ breast cancer. Key genes had elevated protein levels in differentiated breast cancer cell lines, and elevated CHORDC1 expression was linked to a tendency towards a worse outcome in patients with ER+ breast cancer. Silencing CHORDC1 inhibited cell viability and invasion, reducing sensitivity to tamoxifen and paclitaxel in vitro. Conclusion MBPRG expression is linked to the immune microenvironment and drug resistance in ER+ breast cancer patients, providing a reliable prognostic indicator for this group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ae1cb68449d2685b386413614d1719f45655c6" target='_blank'>
              Association of microtubule-based processes gene expression with immune microenvironment and its predictive value for drug response in oestrogen receptor-positive breast cancer
              </a>
            </td>
          <td>
            Zhenfeng Huang, Minghui Zhang, Nana Zhang, Mengyao Zeng, Yao Qian, Meng Zhu, Xiangyan Meng, Ming Shan, Guoqiang Zhang, Feng Liu
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal diagnostics have evolved significantly over the past several decades, from classical karyotyping to high-resolution genomic technologies. This review traces the development and integration of cytogenetic toolskaryotyping, fluorescence in situ hybridization (FISH), chromosomal microarray analysis (CMA), and whole genome sequencing (WGS)in modern clinical diagnostics. We examine each technologys diagnostic utility, limitations, and role in clinical workflows. Importantly, insights from a practicing clinical cytogeneticist highlight the increasing automation of laboratory processes and the continued relevance of karyotyping in routine diagnostics due to its cost-effectiveness and structural resolution. As genome-wide testing becomes more accessible, we explore future directions including the rise of artificial intelligence (AI), non-invasive screening technologies, and emerging ethical challenges surrounding incidental findings, data privacy, and equitable access. Together, these perspectives underscore the importance of combining technological innovation with clinical expertise and ethical foresight to guide the future of precision diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d4b14a05606752d08d57c2ff217d9ddb756ec" target='_blank'>
              From Chromosome Karyotyping to Genomic Precision: The Evolution of Diagnostic Technologies in Medicine
              </a>
            </td>
          <td>
            Run Liu
          </td>
          <td>2025-08-13</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-12</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aim. To identify clinical, molecular, genetic, and immunological predictors of response to PARP inhibitor olaparib therapy in patients with metastatic castration-resistant prostate cancer (mCRPC) harboring germline and somatic mutations in homologous recombination repair (HRR) genes.Materials and methods. A prospective analysis was conducted on data from 39 patients with mCRPC and HRR gene mutations (BRCA1/2, ATM, CHEK2, CDK12, among others) receiving оlaparib therapy after prior treatment with androgen receptor signaling inhibitors. Diagnoses were histologically verified according to the current WHO classification (5th edition). Genetic status was determined by next-generation sequencing (NGS). The tumor immune microenvironment was evaluated by immunohistochemical analysis assessing expression of CD4, CD8, CD68, CD163 markers and tumor-infiltrating lymphocyte (TILs) levels. Statistical analysis involved Cox proportional-hazards regression and Kaplan–Meier survival analysis.Results. The BRCA2 mutation was the most frequently detected alteration (61 %), correlating with increased expression of CD163 and decreased CD68 expression. The most significant clinical predictors associated with improved progressionfree survival during оlaparib therapy included BRCA2 mutation positivity, patient age under 64 years, initially metastatic disease presentation, and ≥90 % reduction in prostate-specific antigen levels at 3 months post-treatment initiation. Immunological markers indicative of favorable therapeutic response included elevated levels of M2-polarized macrophages (CD163 ≥30 %) and a high CD163/CD68 ratio (≥2.83). Conversely, robust prior response to еnzalutamide therapy and administration of docetaxel during hormone-sensitive stages significantly worsened the prognosis for olaparib efficacy.Conclusion. Olaparib therapy is effective in patients with HRRm mCRPC, with particularly effectiveness in BRCA2m tumors.  Clinical (age, disease dissemination pattern, prostate-specific antigen dynamics), immunological (CD163 expression and CD163/CD68 ratio), and molecular and genetic tumor characteristics should be considered when selecting patients for olaparib treatment, enhancing personalized therapeutic strategies in mCRPC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bedca89d6cf003aa836f25f27df6fe20a943841" target='_blank'>
              HRR gene mutations and characteristics of the tumor microenvironment as predictors of response to olaparib therapy in metastatic castration-resistant prostate cancer
              </a>
            </td>
          <td>
            A. I. Stukan, S. Vtorushin, A. P. Bogdan, Т. Y. Semiglazova, Kh. R. Tovbulatova, V. Bodnya, V. Porkhanov, A. A. Dovlatbekyan, M. A. Chagiev, D. V. Khoreva
          </td>
          <td>2025-08-06</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gliomas are the most common primary malignant tumors of the central nervous system (CNS), and despite progress in molecular diagnostics and targeted therapies, their prognosis remains poor. In recent years, immunotherapy has emerged as a promising treatment modality in cancer therapy. However, the inevitable immune evasion by tumor cells is a key barrier affecting therapeutic efficacy. Epigenetic regulation, such as DNA methylation, histone modification, and non-coding RNA expression, plays a crucial role in the occurrence, development, and immune evasion of gliomas. These modifications can dynamically regulate gene expression, leading to the silencing of tumor-associated antigens, dysregulation of pro-inflammatory cytokines, and dynamic modulation of immune checkpoints (such as PD-L1). This review systematically elucidates the key mechanisms by which epigenetic regulation promotes immune evasion in gliomas and details three interconnected mechanisms: 1) epigenetic silencing of tumor-associated antigens and antigen-presenting machinery; 2) dysregulation of pro-inflammatory cytokine secretion; and 3) dynamic modulation of PD-L1 expression through chromatin remodeling. We emphasize the potential of combining epigenetic therapies with immunotherapies to enhance anti-tumor immune responses and overcome treatment resistance in gliomas. Future research should focus on developing biomarker-driven epigenetic immunotherapies and exploring the complex interplay between epigenetic modifications, glioma cells, and the tumor immune microenvironment to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd05063be53a8d164f1a96c934eb33dcd3607ec2" target='_blank'>
              Unraveling epigenetic drivers of immune evasion in gliomas: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Dan Wu, Dongen Ju, Yujia Zhao, Wenna Liu, Qingqing Liu, Ying Liang
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The BRAF protein regulates cell growth and division through key signaling pathways. Mutations in BRAF, particularly the V600E variant, are frequently observed in colorectal cancer (CRC) and are associated with poor prognosis and therapeutic challenges. Tumors harboring certain BRAF mutations often exhibit primary resistance to BRAF inhibitor monotherapies. Over time, these tumors can also develop acquired resistance, further complicating treatment. In this study, we employed replica exchange molecular dynamics simulations combined with machine learning techniques to investigate the structural alterations induced by BRAF mutations and their contribution to drug resistance. Our analyses revealed that conformational changes in mutant BRAF proteins associated with dabrafenib residues psi494, phi600, phi644, phi663, psi675, and phi677 were sufficient for classifying drug-resistant vs. drug-sensitive variants. Similarly, for vemurafenib, residues psi450, phi484, phi495, phi518, psi622, and phi622 were the key residues that influence drug binding and resistance mechanisms. These residues are located in the N-lobe of CR3, which is responsible for ATP binding and the regulation of BRAF kinase activity. These findings offer deeper insights into the molecular basis of BRAF-driven resistance and provide predictive models for phenotypic outcomes of various BRAF mutations. The study underscores the importance of targeting specific BRAF variants for more effective, personalized therapeutic strategies in drug-resistant CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcf6858b6f2686bd3d3197f896e957a981c35dc" target='_blank'>
              Combining Molecular Dynamics and Machine Learning to Predict Drug Resistance Causing Variants of BRAF in Colorectal Cancer
              </a>
            </td>
          <td>
            Longsheng Xie, Christopher Lockhart, Dmitri K Klimov, M. S. Jafri
          </td>
          <td>2025-08-30</td>
          <td>Molecules</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Purpose Next-generation sequencing (NGS) has revolutionized cancer treatment by enabling comprehensive cancer genomic profiling (CGP) to guide genotype-directed therapies. While several prospective trials have demonstrated varying outcomes with CGP in patients with advanced solid tumors, its clinical utility in colorectal cancer (CRC) remains to be evaluated. Methods We conducted a prospective observational study of CGP in our hospital between September 2019 and March 2024. Overall survival (OS) of the patients who received CGP-based therapy and those did not was compared, and genomic variables associated with OS were evaluated. Results A total of 100 patients with CRC underwent CGP using four platforms. The median patient age was 67 years, and most had a good performance status. The most frequent genomic alterations were TP53 (82%), APC (82%), and KRAS (55%). Actionable mutations such as ERBB2 amplification and BRAF V600E were identified in some patients, and 9% received CGP-based therapy, including immune checkpoint inhibitors for tumor mutational burden-high or microsatellite instability-high tumors. Patients receiving CGP-based therapy had longer OS from expert panel discussion (16.0 vs. 10.8 months) compared to those who did not. Alterations in TP53, SMAD4, and NF1 were associated with worse OS. Interestingly, PTEN mutations were linked to improved survival. TP53 alterations were more common in left-sided CRC. Conclusion Although some patients with CRC received CGP-guided therapy, a statistically significant survival benefit was not observed. However, TP53 and SMAD4 mutations were identified as negative prognostic markers, indicating their potential as targets for future drug development. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06295-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f03483e8e9f29b9ac2b6a966e10869cf85e5a3" target='_blank'>
              Clinical utility of comprehensive genomic profiling test for colorectal cancer: a single institution prospective observational study
              </a>
            </td>
          <td>
            Hiroki Tanabe, Katsuyoshi Ando, Keitaro Takahashi, Tomomi Kamanaka, S. Yuzawa, Junko Kikuchi, Y. Ohhara, Shin Ariga, Tatsuya Shonaka, C. Tani, Shin Otake, Takaaki Sasaki, Kenji Takahashi, Nobuhiro Ueno, K. Moriichi, M. Tanino, Ichiro Kinoshita, Yusuke Mizukami, M. Fujiya
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="CRISPR/Cas9 gene-editing technology has revolutionized biomedical research by enabling precise, efficient, and cost-effective genome manipulation. This paper focuses on the application of CRISPR/Cas9 in cancer research, with an emphasis on triple-negative breast cancer (TNBC), a particularly aggressive and treatment-resistant subtype of breast cancer that lacks expression of estrogen, progesterone, and HER2 receptors. Breast cancer is the most commonly diagnosed cancer among women worldwide, and TNBC represents one of its most challenging forms due to the absence of effective targeted therapies. While hormone receptor-positive and HER2-positive subtypes benefit from endocrine and targeted treatments, TNBC patients are often limited to chemotherapy, which is associated with high toxicity and limited efficacy. This review synthesizes recent CRISPR-based studies that identify oncogenes and tumor suppressors relevant to TNBC, explores advanced delivery systems such as non-viral nanoparticles, and evaluates the therapeutic promise of this technology. Despite current limitations—including off-target effects, delivery challenges, and ethical concerns—CRISPR/Cas9 offers a promising path toward developing personalized, gene-targeted treatments for TNBC and other hard-to-treat cancers.

Key words: CRISPR/Cas9, gene editing, breast cancer, triple negative breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c17f19abfb2d0fe3b181a39613fc90c46bf2f21" target='_blank'>
              Review of Current Applications of CRISPR Gene Editing Techniques in Breast Cancer Therapeutics Development
              </a>
            </td>
          <td>
            Yichen Zhang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Health Sciences and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Comprehensive genomic profiling (CGP) is a molecular diagnostic tool with increasing use in cancer research and treatment. There are several commercialized CGP assays with variable targeted genes, while the differences between medium- and large-sized panels in refining genomic profiling strategies, optimizing resource allocation and enhancing clinical applications require evaluation. In the present study, patients with triple-negative breast cancer (TNBC) from the Veterans General Hospital TAipei-Yung-Ling foundation sinO-canceR study were initially assayed using a medium-sized CGP panel (Oncomine comprehensive panel), and the remaining nucleic acid specimens were re-sequenced with a large-sized panel (TruSight Oncology 500). The molecular profiling between the two sequencing panels was compared. A total of 108 breast cancer samples were successfully assayed using both platforms and 272 variants were reported at least once by either type of CGP. The reported variants were among actionable genes (AKT1, BRCA1/2, PALB2, ERBB2, PIK3CA and PTEN), which can be acted upon or have clinical relevance for therapeutic intervention and TP53. The concordance rate between the medium- and large-sized panels was 34.6%, which was enhanced to 58.9% after excluding polymorphisms, out-of-targeted region variants and those with low variant allele frequency (#x003C;10%), with variants of TP53, ERBB2 and PTEN being mostly enhanced. A majority of discordance came from TruSight Oncology 500-detected only variants, especially BRCA1, BRCA2 and PALB2. In conclusion, the results indicated that only one-third of actionable mutations could be detected consistently between the medium- and large-sized CGP panels using the default analytical pipelines, while extensive bioinformatics analyses improved concordance substantially. The large-sized panel detected more variants, thereby enhancing clinical actionability. With more therapeutic targets revealed in the future, CGP may be particularly impactful in refining strategies for TNBC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8f8c51dffb87429763b41f009f6f95cce17938" target='_blank'>
              Comprehensive genomic profiling of Taiwanese triple-negative breast cancer samples with medium- and large-sized sequencing panels: A comparative study implicating treatment allocations
              </a>
            </td>
          <td>
            Chi-Cheng Huang, Yi-Chen Yeh, Y. Tsai, Yen-Shu Lin, Ta-Chung Chao, Chun-Yu Liu, Hsiang-Ling Ho, Ling-Ming Tseng
          </td>
          <td>2025-08-05</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a3a7fc693a064b9e09834f53317ec734d6a1c07" target='_blank'>
              Single-cell transcriptome profiling reveals blast cell heterogeneity and identifies novel therapeutic target IKZF2 in t(8;21) acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Yu Liu, Wenbing Liu, Yihan Mei, Qianqian Huang, Xiaoyu Liu, Chengcai Guo, Manling Chen, Junli Mou, Shangshang Wang, Wanqing Xie, Zheng Tian, Kejing Tang, H. Xing, Ying Wang, Hui Wei, R. Gu, Qing Rao, Min Wang, Shaowei Qiu, Jianxiang Wang
          </td>
          <td>2025-08-08</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="INTRODUCTION
Pancreatic Cancer (PC) is recognized as a highly aggressive malignancy and is anticipated to become the second leading cause of cancer-associated deaths across the United States by 2030. Owing to its late-stage diagnosis and the substantial risk of metastasis, current therapeutic strategies exhibit limited efficacy, resulting in a five-year survival rate below 10%. Consequently, identifying reliable biomarkers and therapeutic approaches remains imperative for enhancing treatment effectiveness.


METHODS
In this study, using the data of Cancer Genome Atlas (TCGA) and Genome-Wide Association Study (GWAS), we identified macrophage subsets and key gene arrb2 closely related to the progression of Pancreatic Adenocarcinoma (PAAD) via the scpagwas method combined with single- cell and bulk transcriptome analysis.


RESULTS
The results showed that in macrophage subpopulations closely related to disease progression, the ARRB2 gene was significantly associated with the prognosis of patients, and low expression suggested poor survival outcomes. The high-expression subgroup of ARRB2 exhibited higher sensitivity to a variety of drugs, and i-bet-762 showed strong molecular binding ability with ARRB2. The experimental detection further confirmed the low expression of ARRB2 in PAAD tissue, which provided the basis for its use as a prognostic marker and potential therapeutic target.


DISCUSSION
The results of this study suggest that ARRB2 is not only a prognostic biomarker of PAAD but may also be involved in the regulation of metabolic and immune pathways, thereby affecting drug responsiveness. There was a significant difference in drug sensitivity between the high and low-expression subgroups of ARRB2, suggesting that there may be a potential mechanism between the signal pathway and the therapeutic effect, which warrants further functional research. Considering the heterogeneity of the macrophage population and its dual role in tumor promotion and inhibition, in-depth analysis of the context-dependent function of ARRB2 in the immune-rich microenvironment is expected to provide new insights for the development of combination therapy, especially the potential of combining it with BET inhibitors (such as i-bet-762).


CONCLUSION
Macrophages, along with ARRB2, serve an essential function in the progression of PAAD and immune regulation. The identification of ARRB2 as a prognostic biomarker and its involvement in critical oncogenic pathways furnish a theoretical foundation for targeted therapeutic interventions. These discoveries contribute to the ongoing exploration of diagnostic, prognostic, and treatment strategies for PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/420334015532528f89d5ba5aeb1438dfd993bcbb" target='_blank'>
              Identifying ARRB2 as a Prognostic Biomarker and Key Player in the Tumor Microenvironment of Pancreatic Cancer through scPagwas Methodology.
              </a>
            </td>
          <td>
            Gaonan Tian, Chengcheng Liu, Chang Che, Shiqi Ren, Kun Zhang, Pengcheng Zhou, Kaidong Wang, Guanyi Lu, Yuchen Xia, YIfan Wang, Kailai Li, Li Yang, Xiangjun Fan, Lei Wang
          </td>
          <td>2025-09-08</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND/AIM
Comprehensive genomic profiling (CGP) with tissue- and blood-based next-generation sequencing (NGS) is integral to the delivery of personalized medicine for targeted cancer therapy. This study aimed to evaluate the variant concordance for somatic variants using two clinical NGS systems for conducting both tissue- and blood-based analyses: Genexus-OCA v3 (OCA) vs. FoundationOne CDx (F1) for tissues and Genexus OPA (OPA) vs. FoundationOne CDx Liquid (F1L) for blood.


PATIENTS AND METHODS
The concordance of genomic alterations between the two NGS analyses was compared in six patients with breast, head, and neck cancers using tissue and circulating tumor DNA biopsies.


RESULTS
A total of 130 genes were common between F1 and OCA, and 41 between F1L and OPA. When comparing FoundationOne to Genexus for common genes, the sensitivity and specificity of OCA and OPA were 55% and 99%, respectively. Nine single-nucleotide variants (SNVs), one copy number alteration (CNA), and one fusion were detected by both Genexus and FoundationOne. However, one SNV (MAP2K1 F53V), two CNAs (AKT3 and MYC), and one fusion (ESR-CCDC170) were detected only in Genexus, whereas two SNVs (TP53 Q331* and KRAS G12V) were detected only in FoundationOne.


CONCLUSION
The two cancer genome panels were equivalent but not perfect in terms of the detection of variants using tissue and blood, indicating that different assays and analytical methods may have influenced the results. When performing CGPs, it is important to consider the characteristics of each NGS-based CGP test and the genetic variants associated with each disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe1f8fa1288486304d422d7baf60accd4be21e6" target='_blank'>
              Comparison of Comprehensive Genomic Profiling Testing "Ion Torrent Genexus Sequencer" With FoundationOne.
              </a>
            </td>
          <td>
            Kenji Fujiyoshi, Rie Sugihara, Naoki Miyamoto, Yoriko Watanabe, Tomoya Sudo, Sanae Numata, Jun Akiba, H. Abe, Yuka Ichinose, Kenji Inoue, Shuich Ozono, T. Ono, Kentaro Orioka, Masaki Kashihara, Ryousuke Kajiwara, Hiroyuki Kawano, Akihiko Kawahara, Ryuta Takase, Uhi Toh, Kazuaki Hashimoto, T. Hisaka, Shingo Hirai, M. Mitsuoka, Daiki Miyazaki, Fumi Yoshitomi, Ken Yamamoto, H. Umeno, Masahisa Nomura, Yoshiki Naito
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258c585d858690cd52a8f16499c3f52ebcd32f02" target='_blank'>
              Genetic regulation of cell type-specific chromatin accessibility shapes immune function and disease risk
              </a>
            </td>
          <td>
            A. Xue, J. Fan, O. A. Dong, H. L. Huang, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, B. Bowen, A. Cuomo, A. Henry, H. Tanudisastro, Z. Qiao, L. Chen, E. Ben-David, K. Farh, D. Neavin, A. S. Lee, A. Senabouth, C. Bartie, R. A. McCloy, V. Chin, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Insufficient infiltration of cytotoxic T cells into solid tumors remains a critical obstacle in cancer immunotherapy. Despite extensive efforts to comprehend the mechanisms governing this limited infiltration, few studies have focused on the evolution of T cell motility behavior after co-culture. In this study, we combined quantitative cell trajectory analysis, computational modeling, and bulk/single-cell RNA sequencing to systematically characterize the impact of cell interactions. We reveal that in a 2.5D co-culture system with multiple cancer-cell clusters, cancer-specific T cells exhibit increased directional persistence, which facilitates their efficient searching of cancer-cell clusters. Additionally, these T cells form prolonged interactions with cancer cells, which is the most crucial factor for their accumulation on cancer-cell clusters. Furthermore, post-interaction, a cancer-cell subpopulation displays immunosuppressive traits, reducing T cell attractant expression, and undergoing epithelial-to-mesenchymal transition. These findings offer valuable insights into improving immunotherapy efficacy and tackling T cell infiltration challenges in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0a380330210bd455d4b1fb482cbeaabdb64b1d" target='_blank'>
              Deciphering antigen-specific T cell navigation tactics and cancer immune evasion in co-cultures
              </a>
            </td>
          <td>
            Xinyue Li, Taoli Jin, Lisha Wang, Senbin Wu, Yerong Liu, Ming Li, Weijing Han, Xuefei Li
          </td>
          <td>2025-07-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af84640a910b5c7e72c72fa38163e38d00a66c4" target='_blank'>
              Morphological map of under- and overexpression of genes in human cells.
              </a>
            </td>
          <td>
            S. Chandrasekaran, Eric Alix, John Arevalo, Adriana Borowa, Patrick J. Byrne, William G. Charles, Zitong S. Chen, B. Cimini, Boxiong Deng, John G Doench, Jessica D. Ewald, Briana Fritchman, Colin J. Fuller, Jedidiah Gaetz, Amy Goodale, Marzieh Haghighi, Yu Han, Zahra Hanifehlou, Holger Hennig, Desiree Hernandez, Christina B. Jacob, Tim James, Tomasz Jetka, Alexandr A. Kalinin, B. Komalo, Maria Kost-Alimova, Tomasz Krawiec, Brittany A. Marion, Glynn Martin, Nicola Jane McCarthy, Lisa Miller, Arne Monsees, Nikita Moshkov, Al'an F. Munoz, Arnaud Ogier, Magdalena Otrocka, Krzysztof Rataj, D. Root, Francesco Rubbo, Simon Scrace, Douglas W Selinger, Rebecca Senft, Peter Sommer, A. Thibaudeau, Sarah Trisorus, Rahul Valiya Veettil, William J. Van Trump, Sui Wang, Michał Warchoł, Erin Weisbart, A. Weiss, Michael Wiest, Agata Zaremba, Andrei Zinovyev, Shantanu Singh, A. Carpenter
          </td>
          <td>2025-08-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8f50562c127ac79c0cdedb311535fc46568766" target='_blank'>
              Longitudinal single-cell analysis reveals treatment-resistant stem and mast cells with potential treatments for pediatric AML.
              </a>
            </td>
          <td>
            Denis Ohlstrom, Mojtaba Bakhtia, Hope L. Mumme, Marina E. Michaud, Alejandro De Janon Gutierrez, D. DeRyckere, Katherine E. Ferguson, Franklin Chien, William C. Pilcher, Sarthak Satpathy, Sean Jordan, Douglas K Graham, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The advent of CRISPR-Cas9 has revolutionized gene editing, offering vast possibilities for disease treatment. This review dissects its mechanisms, structural aspects, and classification, underlining its superiority to preceding technologies such as ZFNs and TALENs. The evolution of CRISPR’s applications is traced from bacterial immunity to the forefront of genome editing. Delivery methods—both viral and non-viral—are assessed, with detailed exploration of emerging solutions like lipid nanoparticles and DNA nanocarriers. CRISPR/Cas9 demonstrates significant therapeutic value in oncology by inactivating oncogenes, restoring tumor suppressors, and enhancing immunotherapeutic strategies. Furthermore, promising results have emerged in treating inherited disorders such as beta-thalassemia and sickle cell anaemia, as well as in clinical studies focused on T-cell therapies and viral diseases. Although these developments are encouraging, obstacles persist, including off-target genetic modifications and the intricacies of delivery. Innovations, notably base editing and prime editing, show promise to resolve these barriers. This review highlights the immense therapeutic prospects of CRISPR/Cas9 and stresses the importance of sustained research to fully unlock its clinical benefits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937dc42e57cb7d44775b157991f2406ddf45a511" target='_blank'>
              CRISPR-Cas9 PROTEIN: PHARMACEUTICAL FRONTIERS IN GENETIC RESTORATION
              </a>
            </td>
          <td>
            Senthil Prabhu R, Gunaseelan M, J. R.
          </td>
          <td>2025-08-30</td>
          <td>International Journal of Pharmaceutical Sciences and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gene signatures predictive of chemotherapeutic response have the potential to extend the reach of precision medicine by allowing oncologists to optimize treatment for individuals. Most published predictive signatures are only capable of predicting response for individual drugs, but most chemotherapy regimens utilize combinations of different agents. We propose a unified framework, called the chemogram, that uses predictive signatures to rank the relative predicted sensitivity of different drugs for individual tumors. Using this approach, providers could efficiently screen against many therapeutics to optimize chemotherapy at any time, whether it be for a treatment-naive tumor or a chemo-resistant tumor requiring a new treatment strategy. To demonstrate the utility of the chemogram, we used predictive signatures (extracted from a previously established method) in our framework to rank predicted sensitivity among drugs within cell lines. We then compared the rank order of predicted and observed response against each drug. Across most cancer types, chemogram-generated predictions were more accurate than predictions made by randomly generated gene signatures, signatures extracted from differential expression alone, and was comparable to another established method of drug response prediction. Our framework demonstrates the ability of transcriptomic signatures to not only predict chemotherapeutic response, but also correctly assign rankings of drug sensitivity on an individual basis. Additionally, scaling the chemogram to include more drugs does not compromise accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2faa968db922d9bf628e93088cb5fa3dc44c7a8" target='_blank'>
              Personalizing chemotherapy drug selection using a novel transcriptomic chemogram
              </a>
            </td>
          <td>
            Kristi Lin-Rahardja, J. Scarborough, Jacob G. Scott
          </td>
          <td>2025-09-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Recently, cancer has become a leading cause of death worldwide, prompting increased research to understand the pathways involved in cancer development and to identify solutions for its treatment. The PI3K/AKT pathway has garnered significant attention because of its involvement in promoting cell proliferation and inhibiting programmed cell death. The protein phosphatase and tensin homolog on chromosome 10 (PTEN) plays a crucial role in inhibiting this pathway, thereby limiting uncontrolled cell proliferation. PTEN gene expression is strictly regulated at the transcriptional, posttranscriptional, and posttranslational levels, and recent research has focused on PTEN due to its reduced levels in cancer cells. This review aims to provide a deep understanding of the PTEN protein from structural and regulatory perspectives, its mutated forms, and its interactions with the occurrence of various malignant tumors to summarize the recent work performed to combat cancers via molecular strategies to enhance PTEN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006440d6b28c83ed5863d6ee8a11b4692d78170c" target='_blank'>
              PTEN defects in cancer, from gene to protein molecular causes and therapeutic targets
              </a>
            </td>
          <td>
            Ghaith Khasarah, Darina Al Nabilsi, Odai Madani, Farah Khazem, Ranwa Alsayed
          </td>
          <td>2025-08-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d735b44b6ee6328d790d08642f35e3daac2ffe" target='_blank'>
              Integrated in vivo combinatorial functional genomics and spatial transcriptomics of tumours to decode genotype-to-phenotype relationships.
              </a>
            </td>
          <td>
            Marco Breinig, Artem Lomakin, Elyas Heidari, M. Ritter, G. Rukhovich, Lio Böse, Luise Butthof, Lena Wendler-Link, Hendrik Wiethoff, Tanja Poth, Felix Sahm, Peter Schirmacher, Oliver Stegle, M. Gerstung, Darjus F. Tschaharganeh
          </td>
          <td>2025-07-28</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Multiple myeloma (MM) continues to be an incurable malignancy, even with recent therapeutic advancements. While epigenetic dysregulation at cis-regulatory elements is known to drive disease progression, the complete molecular mechanisms underlying these alterations are poorly understood. Using ATAC-seq analysis combined with computational footprinting of CD138+ cells from 55 MM patients, we depicted the dynamic changes in chromatin accessibility during disease progression and identified Nuclear Respiratory Factor 1 (NRF1) as a master regulator of vital MM survival pathways. We demonstrated that NRF1 maintains proteasome homeostasis by orchestrating the ubiquitination pathway, which is essential for MM cell survival. We discovered a novel enhancer element that physically interacts with the NRF1 promoter, sustaining its expression. Targeting this enhancer RNA reduced NRF1 levels and increased tumor cell sensitivity to bortezomib (BTZ), suggesting therapeutic potential. In xenograft models, we showed that antisense oligonucleotides (ASOs) targeting the NRF1 enhancer, either alone or combined with BTZ, significantly decreased tumor burden and improved survival. Our findings reveal a previously unknown NRF1-dependent mechanism regulating MM cell survival and present a promising therapeutic approach through manipulation of its regulatory network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cc8489d79911ba7bfacf778b92835e6e2c18c2c" target='_blank'>
              Nuclear respiratory factor 1 promotes cell survival in multiple myeloma under proteasome inhibition therapy.
              </a>
            </td>
          <td>
            T. Bruno, Maria Chiara Cappelletto, C. Cortile, S. Di Giovenale, Bruno Amadio, F. de Nicola, I. Falcone, Stefano Giuliani, Belinda Palermo, V. Catena, L. Ciuffreda, F. Cerruti, P. Cascio, Roberta Merola, Serena Masi, Valentina De Pascale, O. Annibali, Silvia Ferraro, S. Gumenyuk, Francesco Pisani, Francesco Marchesi, A. Mengarelli, Maurizio Fanciulli, G. Corleone
          </td>
          <td>2025-09-08</td>
          <td>Blood</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: The COVID-19 pandemic accelerated the paradigm of drug repurposing, leading to the rapid identification of new uses for existing therapeutics under urgent clinical need. This success story has ignited a broader movement towards leveraging repurposing strategies for relapsed, refractory, and traditionally difficult-to-treat diseases, specifically cancers. Chronic myeloid leukemia (CML), although treatable in the chronic phase (CP), is usually fatal in the blast crisis phase (BC-CML), exemplifying a pressing challenge to oncology, where standard tyrosine kinase inhibitor (TKI) therapies often fail, resulting in poor survival outcomes.
Methods and Results: In a multi-institutional cohort of 141 CML patients, we performed WES across disease phases (123 CP-CML, 6 AP-CML, 12 BC-CML). Mutational landscapes were interrogated using the AI-driven PanDrugs2 platform to identify druggable targets and repurposable FDA-approved anti-leukemic therapies. A 54% surge in mutational burden was observed transitioning from AP-CML to BC-CM, revealing 67 recurrent pan-leukemic gene mutations. Notably, actionable alterations were found in NPM1, DNMT3A, PML, AKT1, CBL, JAK2, TET2, IDH1, and BCL2, with therapeutic opportunities using existing agents such as venetoclax, ivosidenib, decitabine, and azacitidine. Emerging vulnerabilities, including RPTOR and BCR mutations, suggest further avenues for mTOR and BTK inhibitor applications beyond traditional TKI paradigms.
Conclusions: Our integrated genomic and AI-guided approach demonstrates the transformative potential of drug repurposing for BC-CML, highlighting immediate actionable options where conventional therapies fail. This strategy not only offers hope for patients with BC-CML but also paves a visionary path toward precision medicine frameworks for relapsed, refractory, and otherwise intractable cancers in the post-pandemic clinical era. Prospective multi-omics studies and tailored clinical trials are urgently warranted to expand these opportunities across the oncology landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d30d0400773a9b8f9c8bd737970fc55fe286c56b" target='_blank'>
              AI-Guided Repurposing of FDA-Approved Anti-Leukemic Molecularly Targeted Therapies for Fatal Blast Crisis Chronic Myeloid Leukemia: Integrative Genomics for Precision Medicine of Relapsed/Refractory Cancers in the Post-Pandemic Era
              </a>
            </td>
          <td>

          </td>
          <td>2025-09-05</td>
          <td>International Journal of Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract ORMDL proteins (ORMDL1, ORMDL2, ORMDL3) are transmembrane proteins in the endoplasmic reticulum (ER) that regulate sphingolipid metabolism, maintain ER homeostasis, and modulate cellular stress responses. They influence cell proliferation, apoptosis, and metabolic balance. Recent studies have highlighted the altered expression and function of ORMDL proteins in various tumors, including breast cancer, DLBCL, colorectal cancer, and lung cancer. ORMDLs negatively regulate serine palmitoyltransferase (SPT), affecting ceramide and sphingolipid metabolism, which plays a key role in tumor cell proliferation, invasiveness, and resistance to therapy. The dysregulation of ORMDL expression may disrupt sphingolipid metabolism, trigger ER stress, and impair autophagy. Investigating ORMDL functions in cancer could lead to novel insights into tumor development and progression. ORMDL expression may serve as a potential biomarker for cancer diagnosis, prognosis, and therapeutic response prediction. Targeting ORMDL or its metabolic networks offers promising strategies for cancer therapy. Although research on ORMDLs is still in its early stages, further studies are needed to explore their roles in the tumor microenvironment, interactions with the immune system, and applications in personalized medicine. A deeper understanding of ORMDL proteins will enhance tumor diagnosis, treatment, and the development of new therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0954ad738e9f0a73a8f9ee7eb939ae776195251" target='_blank'>
              Advances in ORMDL Research in Malignant Tumors: A Review
              </a>
            </td>
          <td>
            Hao Wang, Zhongquan Yi, Song Yan, Yihao Wang, Weisong Zhang, Rongqi Guo, Yangyang Li, Rui Wang, Heng Li, Xia Li, Jianxiang Song
          </td>
          <td>2025-07-28</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transcription factors (TFs) play a crucial role in tumorigenesis by driving oncogene expression in key signaling pathways. However, their small size and flat surfaces make them challenging targets for small-molecule inhibitors, while macromolecular therapies struggle to cross the cell membrane. Modulating TF activity at the genetic level offers a promising alternative. Antisense oligonucleotides (ASOs), which regulate protein expression by targeting mRNA, have emerged as effective therapeutics for previously undruggable proteins, including TFs. Over the past two decades, ASO therapeutics have advanced significantly, demonstrating long-lasting efficacy by promoting mRNA degradation. c-Myc, a key regulator of oncogene expression, drives cancer cell growth and proliferation but remains undruggable due to its nuclear localization and dynamic structure. In this study, we utilized our ASO development platform to design ASOs targeting c-Myc. Our sequence optimization algorithm achieved high accuracy, with one of three designed ASOs successfully silencing c-Myc. Ex vivo validation showed that ASO3 inhibited A549 cell growth with an IC50 of 152.5 nM. At the molecular level, ASO3 significantly reduced both c-Myc mRNA and protein expression. Functional assays, including trypan blue exclusion assay and CCK-8, confirmed that ASO3 decreased cell viability, suppressed proliferation, and induced apoptosis. These findings highlight ASO3's therapeutic potential and support further investigation as an anti-cancer agent targeting c-Myc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b33b3079149d5ba644ab8a2a4958404ec130feef" target='_blank'>
              Targeting c-Myc with antisense oligonucleotides to induce apoptosis in tumor cells.
              </a>
            </td>
          <td>
            Yuemei Ye, Yanhui Wang, Zhaoyun Zong, Shiyu Chen
          </td>
          <td>2025-08-12</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Initially, pseudogenes were considered to be “junk DNA”, and their biological role was unclear. However, some of the pseudogenes are engaged in the process of cancerogenesis and perform essential functions in competing for endogenous ribonucleic acid (ceRNA) networks and competing for RNA binding proteins (RBPs). They either positively or negatively regulate gene expression and act as suppressive and oncogenic transcripts. In this review, we look at some of the pseudogenes that play a role in the epithelial-to-mesenchymal transition (EMT) process and the maintenance of cancer-initiating cells (CIC), which are essential in understanding cancer development and progression metastasis and resistance to commonly used therapies. The group of discussed pseudogenes consists of CHIAP2, PTENP1, SUMO1P3, NANOGP8, OCT4-PG1/4, or HMGA1-P6, which are connected with different molecular pathways. Moreover, we discussed pseudogenes as potential diagnostic molecules that can be used as a new class of biomarkers. This potential usage may be valuable for oncology and personalized medicine in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8fe0d4350cd2131303d74c06087d56d8dfd818f" target='_blank'>
              Pseudogenes in the carcinogenesis: epithelial-to-mesenchymal transition process and cancer initiating cells
              </a>
            </td>
          <td>
            Tomasz Kolenda, Piotr Chałaj, Aleksandra Cichowicz, Aleksandra Trojańska, Andrzej Bałoniak, Malwina Kwaśniewska, Magda Odrobińska, Kacper Guglas, Joanna Kozłowska-Masłoń, Paulina Gieremek, Paulina Poter, Marlena Janiczek-Polewska, Anna Florczak-Substyk, Anna Przybyła, Patrycja Mantaj, Katarzyna Regulska, Beata J Stanisz, Zefiryn Cybulski, U. Kazimierczak
          </td>
          <td>2025-08-08</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Leukemic stem cells (LSCs) contribute to relapse and resistance in patients with t(8;21) acute myeloid leukemia (AML). Chromatin accessibility remodeled by epigenetic alterations represents a defining hallmark of LSCs that endows them with enhanced survival and self-renewal capacities, which may offer potential therapeutic opportunities for intervention. Here, we showed that SETD8, a lysine methyltransferase that monomethylates lysine 20 of histone H4 (H4K20me1), is essential for the maintenance of stemness in t(8;21) AML LSCs. Genetic deletion or pharmacological inhibition of SETD8 impaired the survival and self-renewal of LSCs in retroviral AML1-ETO9a-driven t(8;21) AML mice and primary t(8;21) AML CD34+ cells. Mechanistically, SETD8 promoted the expression of the mitochondrial outer membrane protein RHOT1 by increasing chromatin accessibility at the enhancer region, thereby reprogramming mitochondrial homeostasis. These findings improve our understanding of gene regulation through chromatin accessibility remodeling and establish a link between histone lysine methylation and mitochondrial homeostasis, suggesting a potential strategy for eliminating LSCs in t(8;21) AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c6048a50159c8e4f2386ae05cf8a7057f3e84a5" target='_blank'>
              Loss of SETD8 Impairs Mitochondrial Homeostasis to Suppress Leukemia Stem Cell Function in t(8;21) Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Yongwei Su, Shuyi Li, Lei Wang, Yiran Wang, Shujiao He, Xiaotong Li, J. Qiao, Jingfeng Zhou, Li Yu
          </td>
          <td>2025-08-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Error-corrected next-generation sequencing (ecNGS) enables the sensitive detection of chemically induced mutations. Matsumura et al. reported Hawk-Seq™, an ecNGS method, demonstrating its utility in clarifying mutagenicity both qualitatively and quantitatively. To further promote the adoption of ecNGS-based assays, it is important to evaluate their inter-laboratory transferability and reproducibility. Therefore, we evaluated the inter-laboratory reproducibility of Hawk-Seq™ and its concordance with the transgenic rodent mutation (TGR) assay. Results The Hawk-Seq™ protocol was successfully transferred from the developer’s laboratory (lab A) to two additional laboratories (labs B, C). Whole genomic mutations were analyzed independently using the same genomic DNA samples from the livers of gpt delta mice exposed to benzo[a]pyrene (BP), N-ethyl-N-nitrosourea (ENU), and N-methyl-N-nitrosourea (MNU). In all laboratories, clear dose-dependent increases in base substitution (BS) frequencies were observed, specific to each mutagen (e.g. G:C to T:A for BP). Statistically significant increases in overall mutation frequencies (OMFs) were identified at the same doses across all laboratories, suggesting high reproducibility in mutagenicity assessment. The correlation coefficient (r2) of the six types of BS frequencies exceeded 0.97 among the three laboratories for BP- or ENU-exposed samples. Thus, Hawk-Seq™ provides qualitatively and quantitatively reproducible results across laboratories. The OMFs in the Hawk-Seq™ analysis positively correlated (r2 = 0.64) with gpt mutant frequencies (MFs). The fold induction of OMFs in the Hawk-Seq™ analysis of ENU- and MNU-exposed samples was at least 14.2 and 4.5, respectively, compared to 6.1 and 2.5 for gpt MFs. Meanwhile, the fold induction of OMFs in BP-exposed samples was ≤ 4.6, compared to 8.2 for gpt MFs. These observations suggest that Hawk-Seq™ demonstrates good concordance with the transgenic rodent (TGR) gene mutation assay, whereas the induction of mutation frequency by each mutagen might not directly correspond. Conclusions Hawk-Seq™-based whole-genome mutagenicity evaluation demonstrated high inter-laboratory reproducibility and concordance with gpt assay results. Our results contribute to the growing evidence that ecNGS assays provide a suitable, or improved, alternative to the TGR assay. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00336-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b65b989c318db09cda5a2bf3706ed7bbf8e0b" target='_blank'>
              Whole genome mutagenicity evaluation using Hawk-Seq™ demonstrates high inter-laboratory reproducibility and concordance with the transgenic rodent gene mutation assay
              </a>
            </td>
          <td>
            S. Matsumura, Sayaka Hosoi, Takako Hirose, Yuki Otsubo, Kazutoshi Saito, Masaaki Miyazawa, Akihiro Kawade, Atsushi Hakura, Dai Kakiuchi, Shoji Asakura, Naoki Koyama, Yuki Okada, Satsuki Chikura, T. Kimoto, Kenichi Masumura, Takayoshi Suzuki, Kei-ichi Sugiyama
          </td>
          <td>2025-07-29</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Metabolic changes in cancer cells are crucial for maintaining their high growth and proliferation rate. As a result, many tumors are characterized by high glucose consumption and intensified aerobic glycolysis, a phenomenon known as the Warburg effect. Through the Warburg effect, cancer cells can rapidly acquire energy, obtain intermediates for biosynthesis, and ensure a source of NAD+ for oxidized biomass synthesis. Altered metabolism and the Warburg effect are characteristic features not only of most transformed proliferating cells but also of normal, rapidly dividing cells, thus posing a challenge for potential anticancer strategies disrupting cellular metabolism. Therefore, targeting the Warburg effect requires a carefully considered strategy so as not to affect the basal metabolism of normal cells and prevent the various side effects in the patient commonly observed with classical chemotherapies targeting DNA replication. On the other hand, strategies/agents that slow metabolic rate are likely to be less toxic to normal cells than to highly metabolically deregulated cancer cells. The aim of this work is to discuss the most optimal approach for inhibiting these favorable metabolic changes in cancer cells while ensuring specificity. The work discusses proteins, enzymes and pathways that, according to the current state of knowledge, can be optimal candidates for cancer specific targeting such as: HK2, PKM2, PFKFB3, PFKFB4, NAD+ de novo metabolism, NADH oxidation, MCT4, MCT1, LDHA and LDHB. In the era of rapid progress in diagnostic tools providing more and more data on molecular changes, the therapeutic strategy should take into account not only the specificity of the cancer, but also a personalized, optimal approach for each individual patient. This article presents an overview, including available databases, showing the heterogeneity of expression of genes involved in metabolic reprogramming among various cancer patients, which clearly suggests the need to develop a specific theranostic approach for targeting the Warburg effect in a personalized manner. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff178a6fbcc213466980c2abda4cba7cd208c2ca" target='_blank'>
              Deciphering enemy tactics – the narrow path to an optimal anti-cancer strategy targeting the Warburg effect
              </a>
            </td>
          <td>
            K. Kocemba-Pilarczyk, Barbara Ostrowska, S. Trojan, P. Dudzik
          </td>
          <td>2025-08-01</td>
          <td>Pharmacological Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer is an extremely complex disease characterized by abnormal cell growth due to genetic and environmental factors. With the rise of the field of epigenetic transcriptomics, 5-methylcytidine (m5C) modification has been identified as one of the most common chemical modifications occurring in various RNA types. The writers, erasers, and readers of m5C modification regulate cancer initiation, progression, and therapeutic responses, such as the proliferation, metastasis, angiogenesis, metabolic reprogramming, immune escape, and therapeutic resistance of tumour cells, by regulating RNA stability, translation, nuclear export, and splicing processes. In this review, we elucidate the biological process of m5C modification, summarize the abnormal expression of RNA-modifying proteins (RMPs) in common malignant tumours, explore their functional effects on malignant hallmarks of cancer and molecular mechanisms, and prospect the potential clinical application value of m5C.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6bbf35e8695fe42d19c1e3cacddf4b9f360aa51" target='_blank'>
              Epitranscriptomic mechanisms and implications of RNA m5C modification in cancer
              </a>
            </td>
          <td>
            Zhonghao Mao, Yan Tian, Lisha Wu, Yu Zhang
          </td>
          <td>2025-07-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Programmable epigenome editors modify gene expression in mammalian cells by altering the local chromatin environment at target loci without inducing DNA breaks. However, the large size of CRISPR-based epigenome editors poses a challenge to their broad use in biomedical research and as future therapies. Here, we present Robust ENveloped Delivery of Epigenome-editor Ribonucleoproteins (RENDER) for transiently delivering programmable epigenetic repressors (CRISPRi, DNMT3A-3L-dCas9, CRISPRoff) and activator (TET1-dCas9) as ribonucleoprotein complexes into human cells to modulate gene expression. After rational engineering, we show that RENDER induces durable epigenetic silencing of endogenous genes across various human cell types, including primary T cells. Additionally, we apply RENDER to epigenetically repress endogenous genes in human stem cell-derived neurons, including the reduction of the neurodegenerative disease associated V337M-mutated Tau protein. Together, our RENDER platform advances the delivery of CRISPR-based epigenome editors into human cells, broadening the use of epigenome editing in fundamental research and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c94b69b6620f871c67be9429011819288d6b9c0" target='_blank'>
              Programmable epigenome editing by transient delivery of CRISPR epigenome editor ribonucleoproteins
              </a>
            </td>
          <td>
            Da Xu, Swen Besselink, Gokul N. Ramadoss, Philip H. Dierks, Justin P. Lubin, Rithu K. Pattali, Jinna I. Brim, Anna E. Christenson, Peter J. Colias, Izaiah J. Ornelas, Carolyn D. Nguyen, Sarah E. Chasins, Bruce R. Conklin, James K. Nuñez
          </td>
          <td>2025-08-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The interaction of plasma with biological systems has emerged as a transformative approach in modern biomedicine, with applications spanning cancer therapy,wound healing,tissue regeneration,and microbial decontamination.Among various plasma types,Cold Atmospheric Plasma(CAP)has gained prominence due to its ability to generate reactive oxygen and nitrogen species (RONS) at near-room temperatures, making it suitable for direct biological applications. However, CAP can also cause DNA damage, which raises concerns about its potential genotoxic effects.This paper provides a comprehensive overview of how CAP induces various types of DNA lesions, such as single- and double-strand breaks, base modifications, and crosslinks, through the action of RONS. It further delves into the complex cellular responses activated upon such damage, including key DNA repair pathways like base excision repair (BER), nucleotide excision repair (NER), and homologous recombination (HR). Understanding these mechanisms is crucial for harnessing plasmas therapeutic potential while ensuring genomic safety in clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3b520693ce72b969352fd39c7da9bbb53cea2a" target='_blank'>
              PLASMA-INDUCED DNA DAMAGE AND REPAIR MECHANISMS: INVESTIGATING GENETIC IMPACT AND CELLULAR RESPONSE PATHWAYS
              </a>
            </td>
          <td>
            Punit Kumar
          </td>
          <td>2025-07-31</td>
          <td>International Journal of Advanced Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92e018af588041a85427514fa53a7493c1155f6d" target='_blank'>
              XBP1s Mediates Cross-resistance to Combination Treatment of CDK4/6 Inhibitors plus Endocrine Therapy in Breast Cancer.
              </a>
            </td>
          <td>
            Y. Sang, Shiyang Liu, Xujie Zhou, W. Chi, Min Xiong, Ming Chen, Hengyu Ren, Douwaner Liu, Li-Ying Zhang, Jingyan Xue, Y. Chi, Jiong Wu
          </td>
          <td>2025-09-12</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype, characterized by a high propensity for metastasis, poor prognosis, and limited treatment options. Research has demonstrated a substantial correlation between the expression of protein arginine N-methyltransferase 1 (PRMT1) and enhanced proliferation, metastasis, and poor outcomes in TNBC. However, the specific role of PRMT1 in lung metastasis and chemoresistance remains unclear. Single-cell RNA sequencing coupled with bioinformatics analysis was employed to identify pertinent genes within metastatic TNBC samples. Functional assays, including cell cycle, apoptosis, wound healing, Transwell migration, colony formation, and Cell Counting Kit-8 Assay (CCK-8), were conducted to evaluate the role of PRMT1. The interaction between PRMT1 and PARP1 was validated by mass spectrometry (MS) and immunoprecipitation. Downstream signaling pathways were explored, with a focus on P65 activation. Enzyme-linked immunosorbent assay was used to quantify the effect of PRMT1 on interleukin-1β secretion. Our study identified a significant association between elevated PRMT1 expression and both lung metastasis and chemoresistance in TNBC. PRMT1 boosts TNBC cell growth, invasion, and lung metastasis. Additionally, high PRMT1 expression contributed to increased resistance to docetaxel in TNBC. Mechanistically, PRMT1 methylates PARP1. On the one hand, this methylation promotes the DNA damage repair ability of PAPA1. On the other hand, it in turn modulates the NF-κB signaling pathway. This modulation enhances the stemness of tumor cells and induces immune suppression within the tumor microenvironment, thereby exacerbating chemoresistance in TNBC. PRMT1 drives lung metastasis and chemoresistance in TNBC through PARP1 methylation and P65 activation. These findings position PRMT1 as a promising biomarker and therapeutic target to overcome resistance and limit metastatic progression in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1d4f60c6703e626c9728e3e8779f0e40de8a3f7" target='_blank'>
              PRMT1-Mediated PARP1 Methylation Drives Lung Metastasis and Chemoresistance via P65 Activation in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Jinhui Zhang, Zi-Yao Huang, Cailu Song, Song Wu, Jindong Xie, Yutian Zou, Xiaoming Xie, Tao Wu, Han-lin Yang, Hailin Tang
          </td>
          <td>2025-08-04</td>
          <td>Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Angelman syndrome is a rare neurodevelopmental disorder caused by the loss of function of the maternally inherited UBE3A gene within the chr15q11-q13 region. This gene is subjected to a tissue-specific form of genomic imprinting leading to the silencing of the paternal allele in neurons. Angelman syndrome can result from various (epi)genetic mechanisms, with paternal uniparental disomy of chromosome 15 (patUPD15) being one of the rarest and least studied due to the absence of suitable models. To address this gap, we generated three independent induced pluripotent stem cell (iPSC) lines from individuals with Angelman syndrome caused by patUPD15, alongside genetically matched unaffected familial controls. Peripheral blood mononuclear cells (PBMCs) were reprogrammed into iPSCs using a non-integrative Sendai virus-based approach expressing the Yamanaka factors. All iPSC lines underwent rigorous quality control, confirming stem cell identity, trilineage differentiation potential, and genetic and epigenetic integrity. This newly established iPSC toolkit provides a powerful platform to investigate the molecular underpinnings of Angelman syndrome caused by patUPD15, paving the way for future translational research and therapeutic development tailored for this understudied form of the disorder. Supplementary Information The online version contains supplementary material available at 10.1007/s13577-025-01287-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae78ebe134ab798fd26cfb00b4b9c50bc04c8fa" target='_blank'>
              A stem cell-based toolkit to model Angelman syndrome caused by paternal uniparental disomy of chromosome 15
              </a>
            </td>
          <td>
            Francisca Cazaux Mateus, João Camões Dos Santos, M. Arez, E. Bekman, S. T. da Rocha
          </td>
          <td>2025-09-16</td>
          <td>Human Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Cell type-specific gene delivery, which restricts gene transfer to therapy-relevant cells and reduces unwanted side effects from ectopic transgene expression, remains a major challenge. To mitigate off-target effects, we developed a programmable cell type-specific gene therapy using an RNA trans-splicing-based approach for gene replacement or suicide gene therapy. This method selectively expresses therapeutic proteins or suicide signals based on the cell’s endogenous pre-mRNA expression profile. Targeting the liver, we developed an HCC-specific trans-splicing based HSVtk/GCV suicide gene therapy approach. We designed suicide RNAs targeting multiple HCC-related pre-mRNA biomarkers, increasing the concentration of targetable pre-mRNA. Non-viral, redosable dumbbell-shaped DNA vectors were designed, manufactured, and tested in tissue culture cells, 3D patient-derived xenograft spheroids, mouse and porcine models. Liver and lung-specific lipid nanoparticles (LNPs) enable precise delivery of therapeutic agents directly to these organs, enhancing treatment efficacy while minimizing systemic side effects. Meanwhile, conjugation of dumbbell vectors with GalNAc3 residues enabled targeted delivery into hepatoblastoma-derived human tissue culture cells. To deliver nucleic acids into mitochondria, we studied the long non-coding β2.7 RNA of the human cytomegalovirus, known to colocalize with mitochondrial complex I. Thermodynamic profiling identified four conserved structural subdomains within β2.7 RNA responsible for mitochondrial targeting. A vector with eight mitochondrial targeting domains showed three times higher activity than the parental β2.7 RNA. Combining mitochondrial targeting RNA and trans-splicing allows cell type-specific transcription of mRNA for mitochondrial gene therapy. Trans-splicing based gene therapy offers promising perspectives for suicide gene therapy of various cancers, incurable viral infections, and cell type-specific mitochondrial gene therapy.



 Pei She Loh (Rachel), Sushmita Poddar, Kaustav Chatterjee, Chiu Jing Wen, Derrick Wee Zan Liang, Fritz Lai, Chen Qingfeng and Volker Patzel. Advancing Targeted Gene Therapy: Leveraging Dumbbell-Shaped DNA for Cell-Specific Precision in Cancer, Virus Infection and Mitochondrial Disease [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT08.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dd36ced25e22f4fa3dd8c1aa47c64b63ba6d014" target='_blank'>
              Abstract LT08: Advancing Targeted Gene Therapy: Leveraging Dumbbell-Shaped DNA for Cell-Specific Precision in Cancer, Virus Infection and Mitochondrial Disease
              </a>
            </td>
          <td>
            Pei She Loh (Rachel), Sushmita Poddar, Kaustav Chatterjee, Chiu Jing Wen, Derrick Wee Zan Liang, Fritz Lai, Qingfeng Chen, Volker Patzel
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Thalassemia, a prevalent single-gene inherited disorder, relies on hematopoietic stem cell or bone marrow transplantation as its definitive treatment. However, the scarcity of suitable donors and the severe complications from anemia and iron overload pose significant challenges. An immediate need exists for a therapeutic method that addresses both the illness and its associated complications. Advancements in stem cell technology and gene-editing methods, such as clustered regularly interspaced short palindromic repeats along with its associated protein (CRISPR/Cas), offer encouraging prospects for a therapy that could liberate patients from the need for ongoing blood transfusions and iron chelation treatments. The potential of genetic reprogramming using induced pluripotent stem cells (iPSCs) to address thalassemia is highly promising. Furthermore, mesenchymal stem cells (MSCs), recognized for their capacity to self-renew and differentiate into multiple lineages that include bone, cartilage, adipose tissue, and liver, demonstrate potential in alleviating several complications faced by thalassemia patients, including osteoporosis, cirrhosis, heart conditions, respiratory issues, and immune-related disorders. In this review, we synthesize and summarize relevant studies to assess the therapeutic potential and predict the curative effects of these cellular approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983b6b15c2e2e9d04817e5e4d461fe2aeabe9943" target='_blank'>
              CRISPR/Cas-edited iPSCs and mesenchymal stem cells: a concise review of their potential in thalassemia therapy
              </a>
            </td>
          <td>
            Jiaojiao Shu, Xin Xie, Sixi Wang, Zuochen Du, Pei Huang, Yan Chen, Zhixu He
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The genetic landscape of pancreatic ductal adenocarcinoma (PDAC) is well-established and dominated by four key genetic driver mutations. Mutational activation of the KRAS oncogene is the initiating genetic event, followed by genetic loss of function of the CDKN2A, TP53, and SMAD4 tumor suppressor genes. Disappointingly, this information has not been leveraged to develop clinically effective targeted therapies for PDAC treatment, where current standards of care remain cocktails of conventional cytotoxic drugs. Nearly all (~95%) PDAC harbors KRAS mutations, and experimental studies have validated the essential role of KRAS mutation in PDAC tumorigenic and metastatic growth. Identified in 1982 as the first gene shown to be aberrantly activated in human cancer, KRAS has been the focus of intensive drug discovery efforts. Widely considered “undruggable,” KRAS has been the elephant in the room for PDAC treatment. This perception was shattered recently with the approval of two KRAS inhibitors for the treatment of KRASG12C-mutant lung and colorectal cancer, fueling hope that KRAS inhibitors will lead to a breakthrough in PDAC therapy. In this Review, we summarize the key role of aberrant KRAS signaling in the biology of pancreatic cancer; provide an overview of past, current, and emerging anti-KRAS treatment strategies; and discuss current challenges that limit the clinical efficacy of directly targeting KRAS for pancreatic cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0b6bef6cbec7fb9929f6c28793ef027fb9b72f" target='_blank'>
              KRAS: the Achilles’ heel of pancreas cancer biology
              </a>
            </td>
          <td>
            Kristina Drizyte-Miller, Taiwo Talabi, Ashwin Somasundaram, Adrienne D. Cox, C. Der
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>123</td>
        </tr>

        <tr id="Chronic active Epstein-Barr virus (EBV) infection (CAEBV) is an orphan disease characterized by the proliferation and infiltration of EBV-infected T/natural killer (NK) cells into multiple organs. Although CAEBV is a heterogeneous disease with diverse clinical courses, its pathogenesis remains poorly understood. In this study, we explored the molecular mechanisms underlying CAEBV by performing a comprehensive multi-omics analysis, including genome, transcriptome, epigenome, and single-cell transcriptome and surface proteome analyses, of 65 CAEBV patients. Methylation analysis identified two distinct subtypes of NK cell-type CAEBV based on the CpG island methylator phenotype (CIMP). In CIMP-positive CAEBV, regions associated with enhancer of zeste homolog 2 binding sites and histone H3 lysine 27 trimethylation exhibited increased DNA hypermethylation, resulting in downregulation of tumor suppressor and anti-herpes virus genes. CIMP-positive CAEBV had a particularly poor prognosis and displayed a "neoplastic" phenotype with a DNA methylation pattern similar to that of extranodal NK/T-cell lymphoma, a higher tumor mutation burden, and frequent copy number alterations. In addition, both in vitro and in vivo functional assays demonstrated that 5-Azacytidine, a hypomethylating agent, was a potentially effective agent for high-risk CIMP-positive CAEBV. Finally, we established a method to effectively detect EBV-infected cells in single-cell analysis, suggesting that EBV-infected NK cells have tissue-resident properties and that innate and adaptive immunity to EBV is compromised in patients with CAEBV. The present findings provide insight into the complex molecular features of CAEBV and suggest potential molecular therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22ef5eba4ec0b720d045ea61d202dc765a7abf44" target='_blank'>
              Multiomics analysis reveals the genetic and epigenetic features of high-risk NK cell-type chronic active EBV infection.
              </a>
            </td>
          <td>
            Ryo Akazawa, Takashi Mikami, Masaki Yamada, I. Kato, Hirohito Kubota, S. Saida, Yoshinori Uchihara, Yuriko Ishikawa, T. Kamitori, Keiji Tasaka, Kiyotaka Isobe, Tomoya Isobe, K. Izawa, K. Umeda, H. Hiramatsu, Keita Jinnouchi, M. Hirata, Masakazu Fujimoto, T. Daifu, Hiroo Ueno, S. Nodomi, Machiko Sawada, Hisanori Fujino, K. Koh, M. Hiwatari, Motohiro Kato, Hiroaki Goto, I. Katano, Ryoji Ito, Mamoru Ito, N. Kakiuchi, M. Nakagawa, Y. Shiraishi, Y. Honda, Hiroyuki Yoshitomi, Hideki Ueno, Maho Sato, Satoru Miyano, Hironori Haga, Akihisa Sawada, K. Imadome, Seishi Ogawa, Junko Takita
          </td>
          <td>2025-07-30</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Adenomyosis is a complex gynecological disease characterized by endometrial tissue invasion into the myometrium. Current interventions, such as hormonal therapy or hysterectomy, are associated with significant side effects and compromise fertility, underscoring the urgent need for safe and effective treatments. Using single-cell RNA sequencing (scRNA-seq) of uterine samples from patients, we identified prolactin (PRL) signaling as a key pathological driver of adenomyosis. Specifically, scRNA-seq revealed a distinct epithelial subcluster with enriched PRL receptor (PRLR) expression. PRL signaling is overactivated in this epithelial subcluster, promoting cellular survival and proliferation, which contributes to lesion formation and expansion in adenomyosis. Concurrently, PRLR is also highly expressed in a fibroblast subcluster characterized by strong expression of inflammation-related genes. Pathological PRL hyperactivation was further validated in preclinical animal models, where transgenic overexpression of PRL or pituitary transplantation induced an adenomyosis phenotype. Importantly, we demonstrated that dysregulation of local PRL signaling led to the development and progression of adenomyosis, whereas inhibition of PRLR with the monoclonal antibody HMI-115 markedly ameliorated pathological manifestations. These findings establish PRL signaling as a critical driver of adenomyosis pathogenesis, highlighting PRLR inhibition as a promising therapeutic strategy and demonstrating the translational potential of HMI-115 for treating adenomyosis, a gynecological condition that has long been neglected in drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2480ddb54ba2a0e95fd154971afab6611f2be380" target='_blank'>
              Single-cell RNA sequencing identifies the prolactin receptor as a therapeutic target in adenomyosis
              </a>
            </td>
          <td>
            Runze Wang, Shuai Xu, Qionghua Cui, Xin Chen, Xuelian Wang, Jinghao Liu, Jie Liu, Yuxuan Xin, Hao Shen, Fengxiang Lv, Lan Zhu, Xinli Hu, Ruiping Xiao
          </td>
          <td>2025-08-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Brain metastasis occurs in up to 40% of patients with non-small cell lung cancer (NSCLC). Considerable genomic heterogeneity exists between the primary lung tumor and respective brain metastasis; however, the identity of the genes capable of driving brain metastasis is incompletely understood. Here, we carried out an in vivo genome-wide CRISPR activation (CRISPRa) screen to identify molecular drivers of brain metastasis from an orthotopic NSCLC patient-derived xenograft model. We discovered activating expression of the Alzheimer’s disease associated β-site amyloid precursor protein cleaving enzyme 1 (BACE1) led to a significant increase in brain metastasis. Furthermore, genetic and pharmacological inhibition of BACE1 blocked NSCLC brain metastasis. Mechanistically, we identified BACE1 acts through its novel substrate EGFR to drive this metastatic phenotype. Together, our data highlights the power of in vivo CRISPR screening to unveil novel molecular drivers and potential therapeutic targets of NSCLC brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f345106700d366f8dea4f4646de89615abfeab1" target='_blank'>
              PRCA-06 GENOME-WIDE IN VIVO CRISPR ACTIVATION SCREEN IDENTIFIES BACE1 AS A THERAPEUTIC VULNERABILITY OF LUNG CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Shawn C Chafe, Kui Zhai, Nikoo Aghaei, P. Miletic, Zhi Huang, Kevin R. Brown, Daniel Mobilio, Daniel Young, Yujin Suk, Shan Grewal, D. Mckenna, Zahra Alizada, Agata M Kieliszek, Fred Lam, Laura Escudero, Qian Huang, A. Huebner, Jack Lu, Patrick Ang, Alisha Anand, Stefan Custers, Erika Apel, Sarah Slassi, Benjamin A Brakel, Jongmyung Kim, James K C Liu, B. Bassey-Archibong, Rober Abdo, Y. Shargall, Jianwei Lu, J. Cutz, Qi Zhang, Shawn Li, C. Venugopal, Rob Hynds, A. Dufour, Jason Moffat, Charles Swanton, Shideng Bao, Sheila K Singh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7a100e23150d73084b13fa87d7c5ad45a0fbaf" target='_blank'>
              The multifaceted role of KIF15 in cancer progression and therapy.
              </a>
            </td>
          <td>
            Sarah Qutayba Badraldin, Karar H. Alfarttoosi, Hayder Naji Sameer, A. K. Bishoyi, Subbulakshmi Ganesan, Aman Shankhyan, Subhashree Ray, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-08-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Repair of double-strand DNA breaks generated by site-directed endonucleases, like Cas9, is the hallmark of gene editing based on homology-directed repair (HDR). HDR uses an exogenous DNA template to restore the cleaved DNA sequence and can facilitate specific gene corrections as well as insertion of genes or partial complementary DNA (cDNA) sequences. For CRISPR/Cas-directed gene editing, co-administration of the Cas9/single guide RNA (sgRNA) ribonucleoprotein (RNP) complex and a DNA template typically involves two different delivery strategies or different types of vehicles. This requires exquisite timing of delivery and may potentially challenge safety and therapeutic applicability. There is a need therefore for technologies that can ferry complete editing tool kits into cells. Here, we demonstrate the use of lentivirus-derived nanoparticles (LVNPs) to transport both RNP complexes and vector RNA, which upon reverse transcription serves as a repair template for HDR-directed gene editing. Such ‘all-in-one’ LVNPs support targeted gene insertion with reduced off-target effects relative to nucleofection procedures. We show potent editing in the HBB gene in human erythroid progenitor cells as well as HDR-directed editing in hematopoietic stem and progenitor cells. Our findings mark a first step toward using a single virus-derived vehicle for delivering a full HDR gene editing kit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdfb1d5c66a1c255398b06979f49213b97baa713" target='_blank'>
              Gene editing in hematopoietic stem cells by co-delivery of Cas9/sgRNA ribonucleoprotein and templates for homology-directed repair in ‘all-in-one’ lentivirus-derived nanoparticles
              </a>
            </td>
          <td>
            Sofie Andersen, J. H. Wolff, Thomas Wisbech Skov, Jacob Hørlück Janns, Luther John Davis, J. Haldrup, Didde Haslund, A. L. Revenfeld, Dinko Relkovic, B. Møller, Jacob Lund, R. Bak, E. A. Thomsen, Jacob G. Mikkelsen
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a3af1d478fb62c55f744123c113ba99279a9a91" target='_blank'>
              Advanced nanotheranostic approaches for targeted glioblastoma treatment: a synergistic fusion of CRISPR-Cas gene editing, AI-driven tumor profiling, and BBB-modulation.
              </a>
            </td>
          <td>
            Chethan Patil, R. Priyanka, B. M. Harshitha, S. Oshik, S. Yashwanth, B. R. Darshan, Shradha Patil, K. A. Prajwal, Prasiddhi Naik, Prakash Goudanavar, T. Mallamma
          </td>
          <td>2025-08-07</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Copy number variation (CNV) represents a significant contributor to intraspecific genomic diversity, yet remains undercharacterized in rhesus macaque (Macaca mulatta) populations. In this study, we performed whole-genome sequencing of eight M. m. lasiotis individuals and integrated these with six subspecies-representative genomes, yielding a comparative dataset of 60 rhesus macaque genomes. Utilizing a read-depth-based approach for CNV detection, our analysis identified approximately 59.6 Mb (2.2% of the reference genome) exhibiting copy number variation. We established a comprehensive CNV map consisting of 1272 non-overlapping CNV regions (CNVRs) across the M. mulatta genome. Hierarchical clustering analysis revealed discordance between CNVR-derived population structure and SNV-based phylogenetic relationships among Chinese subspecies. Comparative genomic analysis identified highly differentiated CNVRs between Indian and Chinese populations enriched for genes associated with immune response and xenobiotic metabolism. Subspecies-specific CNVRs (su_CNVRs) showed functional enrichment in pathways related to environmental adaptation and phenotypic variation. Notably, M. m. tcheliensis exhibited higher gene copies in METRNL and FBP1/FBP2 genes than other subspecies, potentially helping it adapt to cold environments and enhance gluconeogenesis capacity under food scarcity in the cold season. Furthermore, we identified six body size-associated genes with considerable copy number variation among different subspecies, highlighting CNV-mediated regulation of body size in M. mulatta subspecies. This investigation not only elucidates the evolutionary significance of CNVs in shaping genomic architecture and adaptive traits in M. mulatta, but also establishes a foundational resource for future population genetics studies utilizing structural variation markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3289015afda7bf5ca7bb20a4e8d98b923315c12c" target='_blank'>
              Genome-wide copy number variation drives adaptive evolution in Macaca mulatta Populations
              </a>
            </td>
          <td>
            Yun Liu, Lin Zhang, Xianlin Jin, Zhijin Liu, Jiwei Qi, Zhenxin Fan, Ming Li, Jing Li
          </td>
          <td>2025-08-07</td>
          <td>Current Zoology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="With rapid advancements in single-cell DNA sequencing (scDNA-seq), various computational methods have been developed to study evolution and call variants on single-cell level. However, modeling deletions remains challenging because they affect total coverage in ways that are difficult to distinguish from technical artifacts. We present DelSIEVE, a statistical method that infers cell phylogeny and single-nucleotide variants, accounting for deletions, from scDNA-seq data. DelSIEVE distinguishes deletions from mutations and artifacts, detecting more evolutionary events than previous methods. Simulations show high performance, and application to cancer samples reveals varying amounts of deletions and double mutants in different tumors. Supplementary information The online version contains supplementary material available at 10.1186/s13059-025-03738-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2dc8c16f201338fcfd20a88555bd6fa0eb8acd" target='_blank'>
              DelSIEVE: cell phylogeny modeling of single nucleotide variants and deletions from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            Senbai Kang, Nico Borgsmüller, Monica V. Valecha, Magda Markowska, Jack Kuipers, N. Beerenwinkel, D. Posada, E. Szczurek
          </td>
          <td>2025-08-25</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Wolf–Hirschhorn syndrome (WHS) is a devastating congenital disease caused by deletions on the short arm of chromosome 4 (4p), for which no curative treatments currently exist. To facilitate the development of therapeutic strategies, the development of experimental models of WHS is crucial for investigating its etiology and pathogenesis, which remain elusive. In this study, we successfully generated human induced pluripotent stem cells (hiPSCs) from three fibroblast lines from WHS patients. We then characterized these hiPSCs, along with one hiPSC line previously generated from peripheral blood mononuclear cells, as part of a Japanese nationwide project. All four hiPSC lines exhibited characteristics of self-renewal, pluripotency, and karyotypes with expected 4p deletions. Copy number variation microarray analysis revealed that these WHS-specific hiPSCs carried hemizygous deletions in p15.1–p16.3 regions, commonly encompassing 100 genes. Transcriptome analysis showed that the expression of these genes faithfully reflected hemizygous deletion in these WHS-specific hiPSCs and that these down-regulated genes were associated with the development of neural crest cells. These results indicate that WHS-specific hiPSCs can recapitulate the abnormal genomic structure genes related to and the gene expression profile observed in WHS patients. Given the limited understanding of the molecular pathogenesis of WHS, these cellular resources will be instrumental in modeling disease phenotypes and in advancing novel therapies for this syndrome. Supplementary Information The online version contains supplementary material available at 10.1007/s13577-025-01292-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4197267e72692a90f6c48b04c1de23e164e16450" target='_blank'>
              Generation of human induced pluripotent stem cell lines derived from Wolf–Hirschhorn syndrome patients with chromosomal 4p deletion
              </a>
            </td>
          <td>
            Tomoya Shimizu, Miho Takami, M. Matsuo‐Takasaki, Michiya Noguchi, Yukio Nakamura, Tadayoshi Hayata, Yohei Hayashi
          </td>
          <td>2025-09-19</td>
          <td>Human Cell</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Functional gene programs play a wide range of roles in health and disease by orchestrating transcriptional coregulation to govern cell identity. Understanding these intricate gene programs is essential for unraveling the complexities of biological systems; however, deciphering them remains a significant challenge. Recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) technologies have empowered the comprehensive characterization of gene programs at both single-cell and spatial resolutions. Here, we present DeCEP, a computational framework designed to characterize context-specific gene programs using scRNA-seq and ST data. DeCEP leverages functional gene lists and directed graphs to construct functional networks underlying distinct cellular or spatial contexts. It then identifies context-dependent hub genes associated with specific gene programs based on network topology and assigns gene program activity to individual cells or spatial locations. Through evaluation on both simulated and real biological datasets, DeCEP demonstrates complementary strengths over existing methods by enabling more fine-grained characterization of gene programs within specific contexts, particularly those characterized by pronounced transcriptional heterogeneity. Furthermore, we showcase the ability of DeCEP in elucidating biological insights through case studies on normal liver tissue, Alzheimer' disease, and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96597ffaca75dedeb7b0d163cbb7f4a20ab05156" target='_blank'>
              Deciphering context-specific gene programs from single-cell and spatial transcriptomics data with DeCEP.
              </a>
            </td>
          <td>
            Lin Li, Xianbin Su, Ze-Guang Han
          </td>
          <td>2025-08-21</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Uveal melanoma, the most common eye cancer in adults, remains limited to surgical intervention and chemotherapy, with a dismal survival rate that has not improved in over 50 years. To address this therapeutic impasse, we systematically analyzed public gene expression, RNAi, and CRISPR knockout datasets and identified RASGRP3 as an essential gene specifically for uveal melanoma. RasGRP3 is uniquely overexpressed and essential for survival in uveal melanoma cells, but dispensable in healthy cells. RasGRP3 remains “undruggable” due to its intracellular localization and lack of targetable binding pockets. To overcome this, we developed a CRISPR-Cas13d RNA-targeting therapeutic that specifically knocks down RasGRP3 mRNA. This Cas13d-based therapeutic mediates selective uveal melanoma killing through two synergistic mechanisms: (i) direct silencing of the essential RasGRP3 transcript, and (ii) collateral RNA degradation triggered by the cleavage of overexpressed RasGRP3. When delivered via optimized lipid nanoparticles encoding Cas13d mRNA and guide RNA, this strategy eliminated >97% of uveal melanoma cells while sparing healthy cells, including retinal pigment epithelial cells. This approach outperformed conventional Cas9 and siRNA methods in potency without inducing permanent genomic alterations. Our findings establish a RNA-targeting therapeutic for uveal melanoma and a framework for Cas13d-based interventions against broad “undruggable” cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bb413d9ad08e88d74f39a8ba7b106b39795e58" target='_blank'>
              CRISPR-Cas13d-Mediated Targeting of a Context-Specific Essential Gene Enables Selective Elimination of Uveal Melanoma
              </a>
            </td>
          <td>
            Daniel Stauber, Lucas Sosnick, Yitong Ma, Sopida Pimcharoen, Atip Lawanprasert, Niren Murthy, David Myung, L. S. Qi
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>