<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-17 09:28:26 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>27</td>
          <td>42</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid.
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) has transformed high-throughput DNA and RNA analysis, facilitating the rapid identification of clinically relevant genetic variants. Opioid Use Disorder (OUD), a chronic condition characterized by relapse and remission cycles, poses significant challenges for genomic investigations. While whole-exome sequencing (WES) and whole genome sequencing (WGS) serve as robust approaches for variant detection, their high cost restricts widespread use. RNA sequencing (RNA-Seq) presents a viable alternative; however, the complexity of the transcriptome complicates reliable variant identification. Despite these challenges, RNA-Seq has emerged as a valuable tool for detecting single nucleotide polymorphisms (SNPs) in conditions with limited WES data, such as OUD. In this study, RNA-Seq data from postmortem ventral midbrain specimens of chronic opioid users (PRJNA492904) were analyzed to identify variants associated with OUD. Given its established involvement in opioid addiction and impulsivity, we hypothesized that the NRXN3 gene would harbor a significant number of variants. Variant analysis was conducted across eight genes: BDNF, DRD2, DRD3, NRXN3, OPRD1, OPRM1, and NGFB with a primary focus on NRXN3. Our results revealed that NRXN3 exhibited the highest variant burden among the analyzed genes, highlighting its potential role in OUD pathogenesis and reinforcing its association with opioid addiction">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c5022e821728aae9de62cff2d23519c700c306" target='_blank'>
              Genomic Insights into Opioid Addiction: Identification of genomics variants from Gene Expression data
              </a>
            </td>
          <td>
            Swati Ajmeriya, Biswadip Chatterjee, Subhradip Karmakar, ID Orcid
          </td>
          <td>2025-02-23</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is marked by the proliferation of abnormal myeloid progenitor cells in the bone marrow and blood, leading to low cure rates despite new drug approvals from 2017 to 2018. Current therapies often fail due to the emergence of drug resistance mechanisms, such as those involving anti-apoptotic pathways and immune evasion, highlighting an urgent need for novel approaches to overcome these limitations. Programmed cell death (PCD) is crucial for tissue homeostasis, with PANoptosis-a form of PCD integrating pyroptosis, apoptosis, and necroptosis-recently identified. This process, regulated by the PANoptosome complex, could be key to overcoming AML drug resistance. Targeting multiple PCD pathways simultaneously may prove more effective than single-target therapies. Research suggests that disrupting anti-apoptotic mechanisms, such as those involving Bcl-2, can enhance drug sensitivity in AML. This study hypothesizes that PANoptosis-associated resistance genes (PARGs) play a critical role in AML drug resistance by modulating immune responses and offers a multi-faceted approach to tackle this challenge. Using RNA sequencing data from the Cancer Genome Atlas and Gene Expression Omnibus databases, we performed differential expression analysis to identify significantly dysregulated PARGs in AML. Regression analysis identified prognostic PARGs, bridging a key gap in understanding how these genes contribute to treatment resistance. We then verified their expression in AML cell lines and cell samples treated with cytarabine using RT-qPCR. Hierarchical clustering revealed distinct PARG expression patterns, and functional enrichment analysis highlighted their involvement in immune-related pathways. The combination of bioinformatics and experimental validation underscores how these genes may mediate immune modulation in drug resistance, providing a robust framework for further study. Our findings suggest that PARGs contribute to AML resistance by modulating immune responses and provide potential targets for therapeutic intervention. This study highlights the potential of targeting PARGs to improve treatment outcomes in AML. By analyzing the expression changes of these genes in response to standard clinical treatments, we provide a framework for developing multi-target therapeutic strategies that simultaneously disrupt multiple programmed cell death pathways. Such an approach directly addresses the limitations of current treatments by offering a method to enhance drug sensitivity and mitigate resistance, potentially improving survival rates. Our findings underscore the importance of a comprehensive understanding of PCD mechanisms and pave the way for innovative treatments that could significantly impact AML management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064037ccaeb8393d522fb1895074e0d8f9cf70c8" target='_blank'>
              Integrated Genomics Reveal Potential Resistance Mechanisms of PANoptosis-Associated Genes in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Cong Liang, Zhiqing Long, Mengjie Lei, Ran Ding, Mengke Chen
          </td>
          <td>2025-01-27</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a major cause of cancer-related mortality worldwide, with a significant impact on public health. Current treatment options include surgery, chemotherapy, radiotherapy, molecular-targeted therapy, and immunotherapy. Despite advancements in these therapeutic modalities, resistance remains a significant challenge, often leading to treatment failure, poor progression-free survival, and cancer recurrence. Mechanisms of resistance in CRC are multifaceted, involving genetic mutations, epigenetic alterations, tumor heterogeneity, and the tumor microenvironment. Understanding these mechanisms at the molecular level is crucial for identifying novel therapeutic targets and developing strategies to overcome resistance. This review provides an overview of the diverse mechanisms driving drug resistance in sporadic CRC and discusses strategies currently under investigation to counteract this resistance. Several promising strategies are being explored, including targeting drug transport, key signaling pathways, DNA damage response, cell death pathways, epigenetic modifications, cancer stem cells, and the tumor microenvironment. The integration of emerging therapeutic approaches that target resistance mechanisms aims to enhance the efficacy of current CRC treatments and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f0190dfc671312c21a6e2fa1ac0dcb0cf54adc6" target='_blank'>
              Mechanisms and Strategies to Overcome Drug Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Jennifer Haynes, Prasath Manogaran
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All cancers are diseases of the genome, since the cancer cell genome typically consists of 10,000s of passenger alterations, 5-10 biologically relevant alterations, and 1-2 "actionable" alterations. Therefore, somatic mutations in cancer cells can have diagnostic, prognostic, and predictive value. Traditional methods are widely used for testing, such as immunohistochemistry, Sanger sequencing, and allele-specific PCR. However, due to the low throughput, these methods are focused exclusively on testing the most common mutations in target genes. The modern next generation sequencing (NGS) is a technology that enables precision oncology in its current form. ESCAT and ESMO Guidelines defined NGS for routine use in patients with advanced cancers such as non-squamous non-small cell lung cancer, prostate cancer, ovarian cancer, and cholangiocarcinoma. The high sensitivity of the NGS method allows it to be used to search for specific mutations in circulating tumor DNA in blood plasma and other body fluids. NGS testing has evolved from hotspot panels, actionable gene panels, and disease-specific panels to more comprehensive panels. The exome and whole genome sequencing approaches are just beginning to emerge, that is why panel-based testing remains most optimal in oncology practice. NGS is also widely used to identify new and rare mutations in cancer genes and detect inherited cancer mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207a8f97841f6a0acdc06c228c5f1967ea8ee827" target='_blank'>
              Application of Next-Generation Sequencing to Realize Principles of Precision Therapy in Management of Cancer Patients.
              </a>
            </td>
          <td>
            N. Khranovska, O. Gorbach, O. Skachkova, G. Klimnyuk
          </td>
          <td>2025-02-20</td>
          <td>Experimental oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8773538db860e37609b781e5433e56f0729eca07" target='_blank'>
              Large-scale CRISPRi screens link metabolic stress to glioblastoma chemoresistance
              </a>
            </td>
          <td>
            Xing Li, Wansong Zhang, Yitong Fang, Tianhu Sun, Jian Chen, Ruilin Tian
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Telomere elongation is essential for the proliferation of cancer cells. Telomere length control is achieved by either the activation of the telomerase enzyme or the recombination-based Alternative Lengthening of Telomeres (ALT) pathway. ALT is active in about 10-15% of human cancers, but its molecular underpinnings remain poorly understood, preventing the discovery of potential novel therapeutic targets. Pooled CRISPR-based functional genomic screens enable the unbiased discovery of molecular factors involved in cancer biology. Recently, Optical Pooled Screens (OPS) have significantly extended the capabilities of pooled functional genomics screens to enable sensitive imaging-based readouts at the single cell level and large scale. To gain a better understanding of the ALT pathway, we developed a novel OPS assay that employs telomeric native DNA FISH (nFISH) as an optical quantitative readout to measure ALT activity. The assay uses standard OPS protocols for library preparation and sequencing. As a critical element, an optimized nFISH protocol is performed before in situ sequencing to maximize the assay performance. We show that the modified nFISH protocol faithfully detects changes in ALT activity upon CRISPR knock-out (KO) of the FANCM and BLM genes which were previously implicated in ALT. Overall, the OPS-nFISH assay is a reliable method that can provide deep insights into the ALT pathway in a high-throughput format.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In the evolving field of precision oncology, the synthesis of gene circuits that specifically target cancer cells while preserving normal tissue marks a significant breakthrough. However, traditional approaches typically concentrate on single-gene targets, lacking the directed recognition and control among the intricate networks of signaling pathways. Our study presents a synthetic gene circuit, the Logic "AND" Gate Dual-Target Genetic Circuit (LAG-DTGC), which integrates multiple signals to achieve comprehensive reprogramming of various signaling pathways in bladder cancer (BC) cells. This circuit's development hinged on detailed bioinformatics analysis, pinpointing more unique biomarkers with similar expression pattern in BC. LAG-DTGC is engineered to selectively activate in cells where these biomarkers are abnormally expressed. Its precision and the remodeling cell behavior capability are further enhanced by incorporating a logic "AND" gate, triggering the circuit only in the presence of these aberrant cancer-specific biomarkers. LAG-DTGC exhibits an extraordinary ability to reprogram cancer cell signaling pathways, turning the cells' own mechanisms against them for therapeutic effect. This work highlights the potential of synthetic biology in developing precise, less toxic treatments for BC. The LAG-DTGC represents a promising new paradigm in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd49383fc8e846f3a7bdc91c9cc99940e4850c1" target='_blank'>
              Innovative logic "AND" gate gene circuits for bladder cancer treatment: preclinical study.
              </a>
            </td>
          <td>
            Chaojie Xu, Ying Dong, Dongchen Pei, Xintao Zhang, Xiaohong Han, Congcong Cao, Baorui Wu, Changning Lv, Zhengjun Kang, Liqun Zhou, Yuchen Liu, Lin Yao
          </td>
          <td>2025-02-04</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd64f2509541e7332b94c3a04278ee5d7e930fb" target='_blank'>
              Molecular principles underlying aggressive cancers
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-02-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>2</td>
          <td>47</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (AML) exhibits distinct genetic characteristics, including unique driver alterations and mutations with prognostic and therapeutic significance. Emerging rare, recurrent genetic abnormalities and their associations with outcomes emphasize the need for high-throughput molecular diagnostic tools. Whole genome sequencing (WGS) reliably detects key AML biomarkers such as structural variants, mutations, and copy number alterations. Whole transcriptome sequencing (WTS) complements WGS by uncovering oncogene expression patterns, allele-specific expression, and gene expression signatures. In this study, we describe an integrated WGS and WTS clinical workflow for routine pediatric AML diagnosis and present a systematic evaluation of its application compared to conventional cytogenetics and standard molecular diagnostic methods. Our findings demonstrate that the integrated WGS and WTS (iWGS-WTS) approach improves the identification of clinically relevant genetic alterations, enhancing precise disease classification and risk assessment. Moreover, with advancements in workflow and bioinformatics pipelines, the testing turnaround time can be optimized to meet the demands of clinical decision-making, positioning iWGS-WTS as a practical and superior alternative to traditional diagnostic methods in pediatric AML management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de0f3555bd4b9f7c58df5cd98e038c6f7a42e14" target='_blank'>
              Integrated Whole Genome and Transcriptome Sequencing as a Framework for Pediatric and Adolescent AML Diagnosis and Risk Assessment
              </a>
            </td>
          <td>
            Lu Wang, Rebecca Voss, Victor Pastor, Maria Cardenas, Priyadarshini Kumar, J. Maciaszek, Maria Namwanje, Jingqun Ma, Jennifer L. Neary, Meiling Jin, Masayuki Umeda, M. Wilkinson, D. Payne-Turner, Mohammad Eldomery, Jingqun Ma, J. Gu, James Dalton, Samantha Melton, Yen-Chun Liu, S. Foy, Michael C. Rusch, David A. Wheeler, Jinghui Zhang, Kim E. Nichols, Seth E. Karol, Hiroto Inaba, Raul C Ribeiro, J. Rubnitz, J. Klco
          </td>
          <td>2025-01-22</td>
          <td>Research Square</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="
 256


 Background:
 Resistance to Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by complex genomic alterations. The evolution of lineage state transitions from adenocarcinomas to neuroendocrine prostate cancer (NEPC) has also been shown to drive treatment resistance and poor survival. Identifying the timing and association of genomic mutations with lineage state transitions has been limited by the need for serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data and mPC lineage states with a novel circulating tumor cell (CTC) RNA sequencing methodology.
 Methods:
 We collected 273 samples from 117 unique patients with mPC in a prospective biomarker trial. CTCs were purified via immunomagnetic capture on an automated microfluidic technology and analyzed via RNA-seq. A subset of patients had a clinical grade genetic test performed on a contemporary metastatic tissue biopsy. We compared CTC gene expression for lineage states (luminal A, luminal B, NEPC) with somatic gene mutation subtypes known to confer resistance to ARPIs (AR, p53, RB, PTEN, HRR genes) and overall survival (OS).
 Results:
 Single sample pathway analysis of high CTC purity samples identified four transcriptional phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (median OS not reached), patients with LumA and LP phenotypes had similar survival, patients with LumB and NE CTC phenotypes had shorter survival (LumB: median OS 6m, HR 9.1 [3.8-21.8], p<0.0001, NE: median OS 3.7m, HR 11.8 [2.4-57.2], p=0.0019). AR alterations were found in samples with prior ARPI exposure and at similar frequencies across all CTC phenotypes but NE where they were absent. RB mutations were enriched in the unfavorable CTC phenotypes (LumB, NE; p=0.0288). Integrating CTC phenotype with presence or absence of at least one high risk genomic alteration (AR, RB, TP53, PTEN), patients with favorable CTC phenotype (Low burden, LP and LumA) and no high risk alterations had the longest median survival (median OS NR), followed by patients with favorable phenotype but a high risk alteration (median OS 12.7m), patients with an unfavorable phenotype (LumB, NE) but no high risk alteration (median OS 8m), and unfavorable phenotype with a high risk alteration (median OS 4m).
 Conclusions:
 Somatic mutations in mCRPC influence lineage state acquisitions and treatment resistance. Lineage state transitions themselves are associated with poor outcomes and decreased survival which is worsened in the presence of high-risk genetic mutations including RB. This data presents potentially targetable patient populations that would benefit from treatment intensification and early disease monitoring for more aggressive mCRPC subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0203e20c6bc4f26349dbeefa5213e0b374637b7e" target='_blank'>
              Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer
              </a>
            </td>
          <td>
            Slim Mzoughi, Megan Schwarz, Xuedi Wang, D. Demircioglu, Gulay Ulukaya, Kevin Mohammed, Habiba Zorgati, Denis Torre, Lewis E. Tomalin, Federico Di Tullio, Carlos Company, Yuliia Dramaretska, M. Leushacke, Bruno Giotti, T. Lannagan, D. Lozano-Ojalvo, P. Karras, Peter B Vermeulen, Dan Hasson, Robert Sebra, A. Tsankov, Owen J. Sansom, J. Marine, Nick Barker, G. Gargiulo, Ernesto Guccione
          </td>
          <td>2025-02-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer, yet the specific genes driving recurrent chromosome-arm losses remain largely unknown. Here, we present a systematic framework integrating gene expression, mutation, and copy number data to identify candidate driver genes of cancer type-specific recurrent chromosome-arm losses across 20 cancer types, using ∼7,500 tumors from The Cancer Genome Atlas. By analyzing focal deletions and point mutations that co-occur with, or are mutually exclusive with, chromosome-arm losses, we pinpoint 311 candidate drivers associated with 160 cancer type-specific recurrent events. Our approach identifies known aneuploidy drivers such as TP53 and PTEN, while revealing multiple novel candidates, including established tumor suppressor genes not previously linked to aneuploidy. Furthermore, we leverage gene expression changes associated with these chromosome-arm losses to propose pathway-level alterations that may drive cancer progression. Integrating these findings highlights key candidate drivers underlying the observed gene expression alterations, thereby reinforcing their biological relevance. This work provides the first comprehensive catalogue of candidate driver genes for recurrently lost chromosome-arms in human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance.Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation.We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation.This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In recent years, significant advancements in biochemistry, materials science, engineering, and computer-aided testing have driven the development of high-throughput tools for profiling genetic information. Single-cell RNA sequencing (scRNA-seq) technologies have established themselves as key tools for dissecting genetic sequences at the level of single cells. These technologies reveal cellular diversity and allow for the exploration of cell states and transformations with exceptional resolution. Unlike bulk sequencing, which provides population-averaged data, scRNA-seq can detect cell subtypes or gene expression variations that would otherwise be overlooked. However, a key limitation of scRNA-seq is its inability to preserve spatial information about the RNA transcriptome, as the process requires tissue dissociation and cell isolation. Spatial transcriptomics is a pivotal advancement in medical biotechnology, facilitating the identification of molecules such as RNA in their original spatial context within tissue sections at the single-cell level. This capability offers a substantial advantage over traditional single-cell sequencing techniques. Spatial transcriptomics offers valuable insights into a wide range of biomedical fields, including neurology, embryology, cancer research, immunology, and histology. This review highlights single-cell sequencing approaches, recent technological developments, associated challenges, various techniques for expression data analysis, and their applications in disciplines such as cancer research, microbiology, neuroscience, reproductive biology, and immunology. It highlights the critical role of single-cell sequencing tools in characterizing the dynamic nature of individual cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8242b8eb0f45c6f5b5cf16b8ec7b8ce82d36e19b" target='_blank'>
              Advancements in single-cell RNA sequencing and spatial transcriptomics: transforming biomedical research
              </a>
            </td>
          <td>
            Getnet Molla Desta, A. Birhanu
          </td>
          <td>2025-02-05</td>
          <td>Acta Biochimica Polonica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here we show that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells display increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence is rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimics the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slows growth of multiple PAX3-FOXO1+ tumor xenograft models, but not fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50231dc8772bfbbe71ed944e599efde5187dc0b" target='_blank'>
              PAX3-FOXO1 drives targetable cell state-dependent metabolic vulnerabilities in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Jacob A. Boyer, Angela M. Montero, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, Andrew L Kung, Daoqi You, Filemon Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The classical paradigm of drug screening often faces significant limitations due to the challenges associated with identifying molecular or cellular read-outs that are relevant to specific diseases. This issue is particularly pronounced for the thousands of diseases of genetic origin, given the abundance of databases listing disease-associated changes in gene expression and various transcripts, indicating that potential read-outs may be concealed within these resources. To remedy this, an alternative approach was tested: compounds were evaluated for their effects on gene expression and alternative splicing in a healthy cell model, and the resulting data were matched to molecular signatures of diseases. A subset of 50 FDA-approved drugs was tested on mesenchymal stem cells derived from a human pluripotent stem cell line. Over half of the compounds altered gene expression, many affecting pathways linked to monogenic diseases. One hit, increased SQSTM1 expression induced by prazosin, was further validated in ALS models caused by SQSTM1 haploinsufficiency, including patient-derived fibroblasts, SQSTM1-depleted hiPSC-derived motor neurons, and a zebrafish model. Extending this paradigm could involve testing diverse cell types and larger drug libraries. One Sentence Summary A novel drug screening approach for monogenic diseases integrating human pluripotent stem cell derivatives and RNA sequencing to profile gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2756ee22739ee10642a0c818c71c07402f6f6f2e" target='_blank'>
              Integrative Analysis of Drug-Gene Expression Signatures in Human Pluripotent Stem Cells Identifies Novel Drug Candidates for ALS and Monogenic Diseases
              </a>
            </td>
          <td>
            Florine Roussange, Jacqueline Gide, Johana Tournois, Michel Cailleret, A. Boland, Christophe Battail, J. Deleuze, Hélène Polvèche, Didier Auboeuf, Knut Brockmann, E. Kabashi, A. Marian, Lina El Kassar, Sophie Blondel, François Salachas, G. Bruneteau, Marc Peschanski, C. Martinat, Sandrine Baghdoyan
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Adoptive immune cell therapies, such as CAR T, have revolutionized how we can approach cancer therapy, however challenges remain to increase their efficacy, response rates, and resistance to dysfunction. The systematic investigation of gene programs governing human T cell function would be advantageous for reprogramming cell products for clinical benefit. Recent studies have demonstrated the power of CRISPR-based forward genetic screens in primary human T cells to nominate candidate perturbations for enhanced T cell function, however combinations of perturbations will likely be necessary to overcome the most challenging tumors. To enable the systematic and high-throughput interrogation of combinations of gain and loss-of-function perturbations, we have developed a bidirectional CRISPR activation and interference (CRISPRai) system, where pairs of genes can be simultaneously activated and repressed in primary human T cells. Leveraging hits from previous genome-wide CRISPRa and CRISPRi screens, we constructed a library of >66,000 sgRNA combinations and screened for regulators of acute T cell stimulation responses, including cytokine production and proliferation. Reassuringly, these screens identified expected gene epistasis interactions (e.g. rescue of PLCG1 repression by PLCG2 activation), while also discovering novel genetic interactions, providing mechanistic insights into the regulatory networks governing stimulation response. To further identify gene “building blocks” for evaluation as combinations, we completed genome-wide CRISPRa and CRISPRi screens for regulators of human CD8+ CAR T cell exhaustion. These screens identified known potential enhancements (e.g. gain of BATF, loss of MED12), and also uncovered novel regulators of T cell function, with CRISPRa discovering genes not normally expressed in T cells but whose overexpression could cause resistance to exhaustion by synthetic mechanisms. Gene expression programs controlled by perturbation of key regulators were mapped by pairing CRISPR with single-cell RNA-seq (Perturb-seq) and arrayed functional assays. Finally, a top gain-of-function candidate was evaluated for in vivo tumor control, using an antigen-inducible Synthetic Notch system. Together, this work demonstrates how advanced forward genetics approaches can discover and elucidate both natural and synthetic programs that regulate T cell states, and the nomination of perturbations to encode in logic-gated synthetic biology designs for cell therapies.
 Citation Format: Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole Roybal, Brian Shy, Alexander Marson. Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Chromosomal instability in gastric cancer cells is associated with the amplification of oncogenes that encode receptor tyrosine kinases (RTKs), such as HER2 and FGFR2; such gene amplification varies from cell to cell and manifests as genetic heterogeneity within tumours. The intratumoural genetic heterogeneity of RTK gene amplification causes heterogeneity in RTK protein expression, which has been suggested to be associated with therapeutic resistance to RTK inhibitors; however, the underlying mechanism is not fully understood. Here, we show that extrachromosomal DNA (ecDNA) causes intratumoural genetic heterogeneity in RTKs and drug resistance due to diverse dynamic changes. We analysed the dynamics of FGFR2 and MYC ecDNA in a gastric cancer cell line after single-cell cloning. Similar to those in parental cells, the copy numbers of FGFR2 and MYC in subclones differed significantly between cells, indicating intraclonal genetic heterogeneity. Furthermore, the ecDNA composition differed between subclones, which affected FGFR2 protein expression and drug sensitivity. Interestingly, clone cells that were resistant to the FGFR2 inhibitor AZD4547 presented diverse changes in ecDNA, including chimeric ecDNA, large ecDNA and increased ecDNA numbers; these changes were associated with high expression and rephosphorylation of FGFR2. Conversely, when resistant clone cells were cultured under conditions that excluded AZD4547, the ecDNA status became similar to that of the original clone cells, and the inhibitory effect on cell growth was restored. Implications: Our results show that dynamic quantitative and qualitative changes in ecDNA can drive the intratumoural genetic heterogeneity of RTKs and resistance to RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/611724b4911f171cfc8e55580dc1cddab6f13349" target='_blank'>
              Early tolerance and late persistence as alternative drug responses in cancer
              </a>
            </td>
          <td>
            S. Punzi, D. Cittaro, Guido Gatti, Gemma Crupi, O. Botrugno, Antonino Alex Cartalemi, Alon Gutfreund, C. Oneto, Valentina Giansanti, Chiara Battistini, Giovanni Santacatterina, Lucrezia Patruno, Ilaria Villanti, Martina Palumbo, D. Laverty, Francesca Giannese, Alex Graudenzi, G. Caravagna, M. Antoniotti, Zachary D. Nagel, Ugo Cavallaro, L. Lanfrancone, T.A. Yap, Giulio Draetta, Nathalie Q Balaban, Giovanni Tonon
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f873b54c9840f07bddb284c4e229ec7800362c" target='_blank'>
              Genetic advancements in breast cancer treatment: a review
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Sadaf Sedaghatshoar, Homaira Arian, Milad Mokarami, Fatemeh Habibi, Fatemeh Bamarinejad
          </td>
          <td>2025-02-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ecf8ff6118aceaa2b844571144d55332e29e96" target='_blank'>
              Tumor microenvironment and drug resistance in lung adenocarcinoma: molecular mechanisms, prognostic implications, and therapeutic strategies
              </a>
            </td>
          <td>
            Qian Cao, Chenxuan Li, Ying Li, Xiangjing Kong, Shoushi Wang, Jun Ma
          </td>
          <td>2025-02-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55fb3b529ad45aae174b0876baadaa438800c1a7" target='_blank'>
              Comprehensive molecular analysis of 26 newly established human pancreatic ductal adenocarcinoma cell lines reveals two clusters with variating drug sensitivities
              </a>
            </td>
          <td>
            Ju Eun Maeng, Jae-Hyeon Kim, Soon-Chan Kim, W. Yun, W. Kwon, Youngmin Han, Do-Youn Oh, Sang Hyub Lee, Jin-Young Jang, Ja-Lok Ku
          </td>
          <td>2025-02-19</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Systematic screening of gain- or loss-of-function genetic perturbations can be used to characterize the genetic dependencies and mechanisms of regulation for essentially any cellular process of interest. These experiments typically involve profiling from a pool of single gene perturbations and how each genetic perturbation affects the relative cell fitness. When applied in the context of drug efficacy studies, often called chemo-genetic profiling, these methods should be effective at identifying drug mechanisms of action. Unfortunately, fitness-based chemo-genetic profiling studies are ineffective at identifying all components of a drug response. For instance, these studies generally fail to identify which genes regulate drug-induced cell death. Several issues contribute to obscuring death regulation in fitness-based screens, including the confounding effects of proliferation rate variation, variation in the drug-induced coordination between growth and death, and, in some cases, the inability to separate DNA from live and dead cells. MEDUSA is an analytical method for identifying death-regulatory genes in conventional chemo-genetic profiling data. It works by using computational simulations to estimate the growth and death rates that created an observed fitness profile rather than scoring fitness itself. Effective use of the method depends on optimal tittering of experimental conditions, including the drug dose, starting population size, and length of the assay. This manuscript will describe how to set up a chemo-genetic profiling study for MEDUSA-based analysis, and we will demonstrate how to use the method to quantify death rates in chemo-genetic profiling data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ce7ec0649fbc5d3e5e12bff74054318a2f6bce" target='_blank'>
              MEDUSA for Identifying Death Regulatory Genes in Chemo-genetic Profiling Data.
              </a>
            </td>
          <td>
            Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-02-07</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Comparative genomics has emerged as a pivotal methodology for elucidating genetic variations in microbial studies. However, conventional analytical approaches for diploid and polyploid microorganisms have demonstrated limited efficacy in discriminating new mutations from background heterozygosity. This study presents an innovative microbial comparative genomics framework adapted from tumor-normal sequencing methodology. Our approach establishes the original strain as the “normal sample” and the derived strain as the “tumor sample”, enabling precise identification of new mutations (“somatic variants”) while filtering pre-existing heterozygous sites (“germline variations”). The analytical pipeline also includes assessment of loss-of-heterozygosity (LOH) events and genome-wide detection of copy number variations (CNVs) with resolution to identify both regional CNV and whole-chromosome aneuploidy through integrated CNV and variant allele frequency analysis. We validated this framework using diploid Saccharomyces cerevisiae strains before successfully extending its application to Kluyveromyces marxianus, Candida spp., and Hortaea werneckii, encompassing haploid, diploid, polyploid, and aneuploid states. The methodology revealed previously undetected variations across experimental evolution studies, demonstrating superior resolution compared to conventional approaches. This adaptable platform establishes a new paradigm for microbial genome studies, particularly for organisms with diploid or polyploid states where traditional comparative genomics methods prove inadequate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21cbb2f5aa4a29eac0f638906ce12441220483a" target='_blank'>
              Deciphering single-cell genomic architecture: insights into cellular heterogeneity and regulatory dynamics
              </a>
            </td>
          <td>
            Byunghee Kang, Hyeonji Lee, Tae-Young Roh
          </td>
          <td>2025-02-11</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose
The epidermal growth factor receptor (EGFR) is a therapeutic target with confirmed clinical efficacy for several cancer types. We aimed to identify EGFR aberrations and their associations with other genomic alterations in patients with metastatic diseases of various cancers.


Materials and Methods
We used real-world data from the next-generation sequencing (NGS) of 3,286 patients with metastatic cancer at the Samsung Medical Center. We analyzed the distribution of EGFR amplification, mutation, and fusion, as well as their correlations with microsatellite instability (MSI), tumor mutation burden (TMB), and other gene aberrations.


Results
A total of 3,286 patients were tested using NGS of a panel covering 523 cancer-related genes (TSO500, Illumina) as part of clinical practice between October 2019 and October 2022. Patients with lung cancer and gliomas were not included in the analysis. Of the 3,286 patients, 175 (5.3%) had EGFR amplification, 38 (1.2%) had EGFR mutations, and 8 (0.2%) had EGFR fusion. All 175 patients with EGFR amplifications had microsatellite-stable (MSS) tumors, but 102 had co-amplifications in other cancer-related genes, and 78 had mutations with clinical significance (tier I/II). Among the 38 patients with EGFR mutations, three (8%) showed MSI-high status, and eleven (29%) demonstrated high TMB (≥ 10 mutations/mb). Among eight patients with EGFR fusion, three exhibited possible functionalities of the EGFR gene.


Conclusion
EGFR aberrations, mainly amplification, followed by mutation and fusion, were present in 6.4% of patients with metastatic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae0df005b2d8a0f47444591e98d3dc4bb550830" target='_blank'>
              Epidermal Growth Factor Receptor Aberrations Identified by Next-generation Sequencing in Patients with Metastatic Cancers.
              </a>
            </td>
          <td>
            Minkyue Shin, Dae-Ho Choi, Jaeyun Jung, Deokgeun Kim, Sung Hee Lim, Seung-Tae Kim, Jung Yong Hong, Se Hoon Park, Joon-Oh Park, Kyoung-Mee Kim, Jeeyun Lee
          </td>
          <td>2025-02-21</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by expanding blood cell clones carrying somatic mutations in healthy aged individuals and is associated with various age-related diseases and all-cause mortality. While CH mutations affect diverse genes associated with myeloid malignancies, their mechanisms of expansion and disease associations remain poorly understood. We investigate the relationship between clonal fitness and clinical outcomes by integrating data from three longitudinal aging cohorts (n=713, observations=2,341). We demonstrate pathway-specific fitness advantage and clonal composition influence clonal dynamics. Further, the timing of mutation acquisition is necessary to determine the extent of clonal expansion reached during the host individual’s lifetime. We introduce MACS120, a metric combining mutation context, timing, and variant fitness to predict future clonal growth, outperforming traditional variant allele frequency measurements in predicting clinical outcomes. Our unified analytical framework enables standardized clonal dynamics inference across cohorts, advancing our ability to predict and potentially intervene in CH-related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df776ea20c82ef08bfb6dc73e82dc94bd703a7e7" target='_blank'>
              Clinical progression of clonal hematopoiesis is determined by a combination of mutation timing, fitness, and clonal structure
              </a>
            </td>
          <td>
            Eric Latorre-Crespo, Neil A. Robertson, E. Kosebent, Louise Macgillivray, Lee Murphy, M. Uddin, Eric Whitsel, Michael C. Honigberg, Alex Bick, A. Reiner, Valeria Orru, Michele Marongiu, Francesco Cucca, E. Fiorillo, I. Deary, Sarah Harris, Simon Cox, Riccardo Marioni, Linus Schumacher, Tamir Chandra, Kristina Kirschner
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The rise of single-cell sequencing technologies has revolutionized the exploration of drug resistance, revealing the crucial role of cellular heterogeneity in advancing precision medicine. By building computational models from existing single-cell drug response data, we can rapidly annotate cellular responses to drugs in subsequent trials. To this end, we developed scGSDR, a model that integrates two computational pipelines grounded in the knowledge of cellular states and gene signaling pathways, both essential for understanding biological gene semantics. scGSDR enhances predictive performance by incorporating gene semantics and employs an interpretability module to identify key pathways contributing to drug resistance phenotypes. Our extensive validation, which included 16 experiments covering 11 drugs, demonstrates scGSDR's superior predictive accuracy, when trained with either bulk-seq or scRNA-seq data, achieving high AUROC, AUPR, and F1 Scores. The model's application has extended from single-drug predictions to scenarios involving drug combinations. Leveraging pathways of known drug target genes, we found that scGSDR's cell-pathway attention scores are biologically interpretable, which helped us identify other potential drug-related genes. Literature review of top-ranking genes in our predictions such as BCL2, CCND1, the AKT family, and PIK3CA for PLX4720; and ICAM1, VCAM1, NFKB1, NFKBIA, and RAC1 for Paclitaxel confirmed their relevance. In conclusion, scGSDR, by incorporating gene semantics, enhances predictive modeling of cellular responses to diverse drugs, proving invaluable for scenarios involving both single drug and combination therapies and effectively identifying key resistance-related pathways, thus advancing precision medicine and targeted therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e7847baa8f0e94557a12876cbaf996f1f4b3e13" target='_blank'>
              scGSDR: Harnessing Gene Semantics for Single-Cell Pharmacological Profiling
              </a>
            </td>
          <td>
            Yu-an Huang, Xiyue Cao, Zhu-Hong You, Yue C. Li, Xue-Qun Shang, Zhi-an Huang
          </td>
          <td>2025-02-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 130


 Background:
 Chemotherapy regimens are selected based on information about the primary organ or its histological type and are administered based on the results of previous clinical trials. Even now, as cancer genomic research has progressed and precision medicine, which optimizes therapeutic interventions based on tumor genome profiling using next-generation sequencing(NGS), has become standard treatment, cytotoxic anticancer drugs still remain key drugs. We examined the real-world database of genomic and clinical information established by the national central datacenter, the Center for Cancer Genomics and Advanced Therapy (C-CAT) to systematically analyze the effects of mutations across multiple cancers and therapies.
 Methods:
 This study delineated the association between genetic mutations and the therapeutic efficacy of cytotoxic chemotherapy in advanced solid tumors in a real-world Japanese clinical context. We incorporated data from 15,474 patients enrolled in the C-CAT database between June 2019 and June 2022. We analyzed the overall response and time to next treatment (TNT) for genetic alterations and five categories of cytotoxic anticancer drugs in gastrointestinal cancers.
 Results:
 In the C-CAT data, gastrointestinal cancers were frequently registered, with colorectal cancer being the most common, followed by pancreatic cancer, esophageal/gastric cancer, and biliary tract cancer. For platinum-based drugs, in colorectal cancer, the presence of gene X was associated with not only a higher response rate but also a longer TNT (hazard ratio 0.82, p<0.001). In the case of colorectal cancer and pancreatic cancer with KRAS gene mutations, the ORR was low, and the TNT was also short in the case of pancreatic cancer (hazard ratio 1.33, p<0.001). Interestingly, when BRCA2 genes are mutant, TNT tended to be longer in many organs, suggesting that platinum-based drugs are effective across organs in carcinomas with BRCA2 mutations. Additionally, gene X showed differences in efficacy not only for platinum but also for antimetabolites and topoisomerase inhibitors. Moreover, KRAS mutation was observed in almost all pancreatic cancers and was a treatment-resistant mechanism for all platinum drugs and antimetabolites.
 Conclusions:
 This study demonstrated that the effects of cytotoxic anticancer drugs differ depending on genetic mutations. When determining chemotherapy regimens, considering not only the organ but also genetic mutations may enable more efficient drug selection.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d0f28bc90a1314a9a663e72dc465d8d8de56df" target='_blank'>
              Genetic mutations and cytotoxic chemotherapy response for advanced solid tumors: A detailed analysis using the C-CAT database in Japan.
              </a>
            </td>
          <td>
            N. Ishibashi, T. Kamatani, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb087ff58a5d637f585f09e70365a21011c3f17" target='_blank'>
              The RNA-binding protein LARP6 regulates the alternative splicing of related genes in MDA-MB-231 cells
              </a>
            </td>
          <td>
            Li Guo, Yaobang Liu, Shuxun Yan, Hong Li, Kai Zhang, Jinping Li
          </td>
          <td>2025-03-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The advent of gene editing has significantly advanced the field of medicine, opening new frontiers in the treatment of genetic disorders, cancer, and infectious diseases. Gene editing technology remains a dynamic and promising area of research and development. Recent advancements in protein and RNA engineering within this field have addressed critical issues such as imprecise edits, poor editing efficiency, and off-target effects. Advancements in delivery methods have allowed the achievement of therapeutic or even selection-free gene editing efficiency with reduced toxicity in primary cells, thereby enhancing the safety and efficacy of gene manipulation. This progress paves the way for transformative changes in molecular biology, medicine, and other fields. This review provides a comprehensive overview of the advancements in gene editing techniques, focusing on prime editor proteins and their engineered variants. It also explores alternative systems that expand the toolkit for precise genomic modifications and highlights the potential of these innovations in treating hematological disorders, while also discussing the limitations and challenges that remain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad85009d033296d6539b20b7bcf7b1d494981e9" target='_blank'>
              Advancing Precision Medicine: Recent Innovations in Gene Editing Technologies.
              </a>
            </td>
          <td>
            Abhijith Koodamvetty, Saravanbhavan Thangavel
          </td>
          <td>2025-03-02</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The current review delves into the transformative role of precision medicine in addressing Colorectal Cancer [CRC], a pressing global health challenge. It examines closely signalling pathways, genetic and epigenetic modifications, and microsatellite in-stability. The primary focus is on elucidating biomarkers revolutionizing CRC diagnosis and treatment. Genetic biomarkers encompass non-coding RNA, epigenetic markers, TP53 mutations, and KRAS, NRAS, and BRAF gene alterations. Targeted therapies, including anti-EGFR, anti-VEGF, immune checkpoint inhibitors, and HER2-targeted treatments, are explored along with their mechanisms and clinical applications. Additionally, we highlight the importance of utilizing personalized treatment strategies by employing molecular pro-filing and genetic testing. These approaches facilitate the identification of appropriate pa-tients for targeted therapies. Clinical trials supporting these treatments are presented, em-phasizing response rates and survival outcomes. Detailed exploration of resistance mecha-nisms to targeted therapies and strategies to overcome resistance is also provided, paving the way for more effective regimens. Directions for future research in precision medicine, such as biomarkers, combinations, and liquid biopsy in the oncology field, are described. However, the precise application of precision medicine in CRC comes with questions such as costs, considerations of ethical factors, and the need to sensitize patients. Nonetheless, based on a meticulous analysis of several aspects, precision medicine finds itself equipped with the ability to enhance the patients' prognosis ,thereby, lessening the global oncologic burden of CRC and provide important information to the clinician-scientist and policy-maker that might benefit from the ongoing development of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9df55b47e156335e90e003be1a18ab4d05e8ee9" target='_blank'>
              Precision Medicine in Colorectal Cancer: Targeted Therapies and Biomarker Insights.
              </a>
            </td>
          <td>
            Shriyansh Srivastava, Amit Anand, Anil Goud Kandhula, Pooja Jaiswal, Sachin Kumar, Monica Gulati, Monika Sachdeva, Tapan Behl
          </td>
          <td>2025-01-28</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung cancer exhibits high morbidity and mortality rates, characterized by a heterogeneous genetic landscape and an immunosuppressive tumor microenvironment, despite recent therapeutic advances. High-resolution single-cell sequencing has facilitated a comprehensive analysis of malignant and host cell types, enhancing the understanding of lung cancer’s heterogeneity and adaptability under basal conditions and in response to therapeutic intervention. In this review, we provide an overview of the utilization of single-cell sequencing technology in characterizing the microenvironment of lung cancer, facilitating early diagnosis, monitoring tumor progression, and elucidating mechanisms of drug resistance, thereby offering new insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Lin28, a conserved RNA-binding protein, promotes cancer stem cell features, epithelial-to-mesenchymal transition, and treatment resistance. Lin28 regulates mRNA translation, RNA stability, and transcription, which improve tumor plasticity and treatment resistance, in addition to suppressing let-7 microRNA production. Lin28 is an interesting cancer target due to its many functions; however, its structural complexity makes treatment development difficult. Lin28's proven and new non-canonical activities in cancer biology are examined in this work, focusing on cancer stem cell maintenance, metastasis, and treatment resistance. We highlight computational drug discovery advances targeting Lin28 utilizing virtual screening and machine learning. Generative artificial intelligence has made it easier to develop inhibitors for challenging targets like Lin28. This work integrates current knowledge and technology to demonstrate Lin28's therapeutic potential and outline future techniques to overcome RNA-binding protein targeting challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f268ca999e90fb84abf8448f8bf69292bbf18883" target='_blank'>
              Targeting Lin28: Insights into Biology and Advances with AI-Driven Drug Development
              </a>
            </td>
          <td>
            Victor M Matias-Barrios, Ekaterina Manskaia, Artem Cherkasov, Xuesen Dong
          </td>
          <td>2025-01-18</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic medicine, which integrates genomics and bioinformatics into clinical care and diagnostics, is transforming healthcare by enabling personalized treatment approaches. Advances in technologies such as DNA sequencing, proteomics, and computational power have laid the foundation for individualized therapies that account for genetic variations influencing disease risk, progression, and treatment response. This review explores the historical milestones leading to current applications of genomic medicine, such as targeted therapies, gene therapies, and precision medicine, in fields including cardiovascular diseases, oncology, and rare genetic disorders. It highlights the use of next-generation sequencing (NGS) and third-generation sequencing (TGS) to improve diagnostic accuracy and treatment outcomes, emphasizing the role of genomic data in advancing personalized treatments. Furthermore, emerging therapies such as CRISPR/Cas-based genome editing and adeno-associated viral vectors showcase the potential of gene therapy in addressing complex diseases, including rare genetic disorders. Despite promising advancements, challenges remain in fully integrating genomic medicine into routine clinical practice, including cost barriers, data interpretation complexities, and the need for widespread genomic literacy among healthcare professionals. The future of genomic medicine holds transformative potential for revolutionizing the diagnosis, treatment, and management of both common and rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48c7f88008ee3f6f5e730cd7155fc1a412d588b" target='_blank'>
              Genomic medicine and personalized treatment: a narrative review
              </a>
            </td>
          <td>
            Adil Khan, Anchal Ramesh Barapatre, Nadir Babar, Joy Doshi, Mohamd Ghaly, Kirtan Ghanshyam Patel, Shayan Nawaz, Uswa Hasana, Swara Punit Khatri, Shilpa Pathange, Abhinya Reddy Pesaru, C. Puvvada, Marium Billoo, Usama Jamil
          </td>
          <td>2025-02-13</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ability to identify gene functions and interactions in specific cellular contexts has been greatly enabled by functional genomics technologies. CRISPR-based genetic screens have proven invaluable in elucidating gene function in mammalian cells. Single-cell functional genomics methods, such as Perturb-seq and Spear-ATAC, have made it possible to achieve high-throughput mapping of the functional effects of gene perturbations by profiling transcriptomes and DNA accessibility, respectively. Combining single-cell chromatin accessibility and transcriptomic data via multiomic approaches has facilitated the discovery of novel cis and gene regulatory interactions. However, pseudobulk readouts from cell populations can often cloud the interpretation of results due to a heterogeneous response from cells receiving the same genetic perturbation, which could be mitigated by using transcriptional profiles of single cells to subset the ATAC-seq data. Existing methods to capture CRISPR guide RNAs to simultaneously assess the impact of genetic perturbations on RNA and ATAC profiles require either cloning of gRNA libraries in specialized vectors or implementing complex protocols with multiple rounds of barcoding. Here, we introduce CAT-ATAC, a technique that adds CRISPR gRNA capture to the existing 10X Genomics Multiome assay, generating paired transcriptome, chromatin accessibility and perturbation identity data from the same individual cells. We demonstrate up to 77% guide capture efficiency for both arrayed and pooled delivery of lentiviral gRNAs in induced pluripotent stem cells (iPSCs) and cancer cell lines. This capability allows us to construct gene regulatory networks (GRNs) in cells under drug and genetic perturbations. By applying CAT-ATAC, we were able to identify a GRN associated with dasatinib resistance, indirectly activated by the HIC2 gene. Using loss of function experiments, we further validated that the gene, ZFPM2, a component of the predicted GRN, also contributes to dasatinib resistance. CAT-ATAC can thus be used to generate high-content multidimensional genotype-phenotype maps to reveal novel gene and cellular interactions and functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5994854c014d82e61919b765b3c023100d1d1160" target='_blank'>
              Simultaneous single-cell CRISPR, RNA, and ATAC-seq enables multiomic CRISPR screens to identify gene regulatory relationships
              </a>
            </td>
          <td>
            Kaivalya Shevade, Yeqing Angela Yang, Kevin Feng, Karl Mader, V. Sevim, Jacob Parsons, Gunisha Arora, Hasnaa Elfawy, Rachel Mace, S. Federman, Rustam Esanov, Shawn Shafer, Eric D Chow, Laralynne Przybyla
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fbe09900693cf8126b4ed7a1f742723962eeea9" target='_blank'>
              A comprehensive analysis of scRNA-Seq and RNA-Seq unveils B cell immune suppression in the AAV-loaded brain
              </a>
            </td>
          <td>
            Shunyu Wu, Lu Xue, Xiang Li, Yaoxuan Wang, Yuting Zhu, Yuanbo Luo, Jiayu Sun, Tingting Jin, Wenying Shu, Zhaoyan Wang
          </td>
          <td>2025-03-05</td>
          <td>Immunologic Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the deadliest neoplasia. Intrinsic or acquired resistance is the main cause of failure of therapy regimens that leads to relapse and death in CRC patients. The widely used chemotherapeutic agent 5-fluorouracil (5-FU) remains the mainstay for therapeutic combinations. Unfortunately, chemotherapeutic resistance and side effects are frequent events that compromise the success of these therapies; the dysregulation of enzymes that regulate 5-FU metabolism increases the expression and activity of efflux pumps. Additional tumor cell adaptations such as epithelial–mesenchymal transition (EMT), autophagy shaping of the tumor microenvironment, and inflammation contribute to chemoresistance. Finding new strategies and alternatives to enhance conventional chemotherapies has become necessary. Recently, the study of natural compounds has been gaining strength as an alternative to chemotherapeutics in different cancers. Curcumin, trimethylglycine, resveratrol, artemisinin, and some helminth-derived molecules, among others, are some natural compounds studied in the context of CRC. This review discusses the main benefits, mechanisms, advances, and dark side of conventional chemotherapeutics currently evaluated in CRC treatment. We also analyzed the landscape of alternative non-conventional compounds and their underlying mechanisms of action, which could, in the short term, provide fundamental knowledge to harness their anti-tumor effects and allow them to be used as alternative adjuvant therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a52e6869256b3ec61847f7e51859687b36490d2c" target='_blank'>
              Natural Alternatives in the Treatment of Colorectal Cancer: A Mechanisms Perspective
              </a>
            </td>
          <td>
            K. V. Fernandez-Muñoz, C. A. Sánchez-Barrera, M. Meraz-Ríos, J. L. Reyes, E. Pérez-Yépez, M. T. Ortiz-Melo, Luis I. Terrazas, Mónica G Mendoza-Rodríguez
          </td>
          <td>2025-02-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Biological and dynamic mechanisms by which Drug-tolerant persister (DTP) cells contribute to the development of acquired drug resistance have not been fully elucidated. Here, by integrating multidimensional data from drug-treated PC9 cells, we developed a novel multiscale mathematical model from an evolutionary perspective that encompasses epigenetic and cellular population dynamics. By coupling stochastic simulation with quantitative analysis, we identified epigenetic instability as the most prominent kinetic feature related to the emergence of DTP cell subpopulations and the effectiveness of intermittent treatment. Moreover, we revealed the optimal schedule for intermittent treatment, including the optimal area for therapeutic time and drug holidays. By leveraging single-cell RNA-seq data characterizing the drug tolerance of lung cancer, we validated the predictions made by our model and further revealed previously unrecognized biological features of DTP cells, such as cell autophagy and migration, as well as new biomarker genes of therapeutic tolerance. Our work not only provides a paradigm for the integration of multiscale mathematical models with newly emerging genomics data but also improves our understanding of the crucial roles of DTP cells and offers guidance for developing new intermittent treatment strategies against acquired drug resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b2c7b90ea2e9a9d298ad9d7420ea2b6b5b8ec4" target='_blank'>
              Integrating multiscale mathematical modeling and multidimensional data reveals the effects of epigenetic instability on acquired drug resistance in cancer
              </a>
            </td>
          <td>
            Shun-guan Wang, Jinzhi Lei, Xiufen Zou, Suoqin Jin
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3833ef2c0218061364fcad4c0598f346a7f4c6cb" target='_blank'>
              Synthetic lethality of mRNA quality control complexes in cancer
              </a>
            </td>
          <td>
            Vivian Prindle, Adam E Richardson, Kimberly R Sher, Sarah Kongpachith, Kaitlin Kentala, Sakina M. Petiwala, Dong Cheng, Deborah Widomski, Phuong Le, Maricel Torrent, Anlu Chen, Stephen Walker, Marianne B. Palczewski, Diya Mitra, V. Manaves, Xu Shi, Charles Lu, Stephanie Sandoval, Zoltan Dezso, F. Buchanan, Daniel Verduzco, Brian Bierie, Jonathan A Meulbroek, W. Pappano, Joshua P Plotnik
          </td>
          <td>2025-02-05</td>
          <td>Nature</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Tumors with low expression of Interferon-Stimulated Genes (ISG) and Antigen Presentation (AP) genes respond relatively poorly to current immunotherapies. One of the early hallmarks of cancer is DNA hypomethylation in genomic repeat regions, resulting in the expression of normally silenced endogenous “viral” elements. Such epigenetic changes have the potential to augment anti-tumor immune responses as well as reduce tumor cell fitness through the generation of aberrant nucleic acid species (NAS) and consequent activation of NAS-sensing pathways. Therefore, tumor evolution should favor additional selective events that suppress NAS generation, possibly yielding specific therapeutic vulnerabilities. Here, we show that the Lysine Demethylase 5 (KDM5) family of epigenetic regulatory enzymes suppress R-loop formation in genomic repeat regions in cancer cells. We find that KDM5 inhibition in luminal breast cancer cells results in R-loop-mediated DNA damage, reduced cell fitness and an increase in ISG and AP signatures as well as cell surface Major Histocompatibility Complex (MHC) class I, mediated by RNA:DNA hybrid activation of the CGAS/STING pathway. KDM5 inhibition does not result in DNA damage or activation of the CGAS/STING pathway in normal breast epithelial cells, suggesting that KDM5 inhibitors may enable a wide therapeutic window in this setting, as compared to STING agonists or Type I Interferons. These findings provide new insights into the interplay between epigenetic regulation of genomic repeats, R-loop formation, innate immunity, and cell fitness in the context of cancer evolution and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction: Prostate cancer, notably prostate adenocarcinoma (PARD), has high incidence and mortality rates. Although typically resistant to immunotherapy, recent studies have found immune targets for prostate cancer. Stratifying patients by molecular subtypes may identify those who could benefit from immunotherapy. Methods: We used single-cell and bulk RNA sequencing data from GEO and TCGA databases. We characterized the tumor microenvironment at the single-cell level, analyzing cell interactions and identifying fibroblasts linked to mitophagy. Target genes were narrowed down at the bulk transcriptome level to construct a PARD prognosis prediction nomogram. Unsupervised consensus clustering classified PARD into subtypes, analyzing differences in clinical features, immune infiltration, and immunotherapy. Furthermore, the cellular functions of the genes of interest were verified in vitro. Results: We identified ten cell types and 160 mitophagy-related single-cell differentially expressed genes (MR-scDEGs). Strong interactions were observed between fibroblasts, endothelial cells, CD8+ T cells, and NK cells. Fibroblasts linked to mitophagy were divided into six subtypes. Intersection of DEGs from three bulk datasets with MR-scDEGs identified 26 key genes clustered into two subgroups. COX regression analysis identified seven prognostic key genes, enabling a prognostic nomogram model. High and low-risk groups showed significant differences in clinical features, immune infiltration, immunotherapy, and drug sensitivity. In prostate cancer cell lines, CAV1, PALLD, and ITGB8 are upregulated, while CLDN7 is downregulated. Knockdown of PALLD significantly inhibits the proliferation and colony-forming ability of PC3 and DU145 cells, suggesting the important roles of this gene in prostate cancer progression. Conclusions: This study analyzed mitophagy-related genes in PARD, predicting prognosis and aiding in subtype identification and immunotherapy response analysis. This approach offers new strategies for treating prostate cancer with specific molecular subtypes and helps develop potential biomarkers for personalized medicine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab8cd0df51baf0c065fc9d46908e143b732254" target='_blank'>
              An Integrated Approach Utilizing Single-Cell and Bulk RNA-Sequencing for the Identification of a Mitophagy-Associated Genes Signature: Implications for Prognostication and Therapeutic Stratification in Prostate Cancer
              </a>
            </td>
          <td>
            Yuke Zhang, Li Ding, Zhijin Zhang, Liliang Shen, Yadong Guo, Wentao Zhang, Yang Yu, Zhuoran Gu, Ji Liu, Aimaitiaji Kadier, Jiang Geng, Shiyu Mao, Xudong Yao
          </td>
          <td>2025-01-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Introduction: Mutations in KEAP1 and NFE2L2 (NFR2) occur in approximately 20% of non-small cell lung cancer (NSCLC) patients and are associated with resistance to radiotherapy and systemic treatments. In previous research, we found that not all KEAP1 mutations found in NSCLC patients are pathogenic. These benign mutations exhibit a wild-type phenotype that does not confer therapy resistance. With only roughly 100 of 13,000 (<1%) potential single amino acid substitution mutations in KEAP1 characterized thus far, here we comprehensively evaluated all possible point mutations in KEAP1 using deep mutational scanning (DMS). Methods: The High-throughput Mutagenesis by Integrated TilEs (MITE) method was used to generate the KEAP1 DMS library. In short, the KEAP1 sequence was segmented into 21 variable regions (tiles), each comprising a 90-base pair variable region flanked by a 30-base pair constant region. Each codon position included the 20 possible natural amino acids plus a stop codon, for a total of 13,076 KEAP1 variants. The tiles were then assembled to ensure that each full-length KEAP1 construct contained only a single amino acid substitution. The KEAP1 DMS lentivirus library was transduced into H1299-KEAP1NULLcells, followed by selective pressure with H2O2, radiation, or vehicle control. The KEAP1 ORF was then amplified from surviving cell genomic DNA, and mutations were quantified using next-generation sequencing. Functional annotations were derived through mutation enrichment analysis. Results: Both H2O2 and radiation selective pressure of the KEAP1 DMS library induced a bimodal distribution of KEAP1 mutations, with cells harboring KEAP1 nonsense mutations being enriched and cells with silent mutations being depleted. By setting a threshold of 95% specificity for silent mutations to differentiate between benign and pathogenic mutations, we identified 94% of nonsense mutations and 88% of previously characterized pathogenic mutations as pathogenic. While most KEAP1 mutations were categorized as benign (64%), analysis of mutations in TCGA NSCLC patients showed pathogenic mutations were enriched (72%). Mapping mutation phenotypes onto a KEAP1 structural model revealed an enrichment of pathogenic mutations at critical functional sites, including the NRF2 binding pocket of the Kelch domain, the BTB homodimerization interface, and the hydrophobic core of the oxidative stress-sensing IVR domain. Conclusions: Through deep mutational scanning experiments, we generated a comprehensive list of pathogenic KEAP1 single amino acid substitutions. These results align with previously known biological functions and prior single amino acid substitutions characterized in the literature. These findings will facilitate the identification of benign and pathogenic KEAP1 mutations in clinical settings, enabling personalized radiotherapeutic and systemic therapy strategies for patients harboring KEAP1 mutations.
 Citation Format: Noah Kastelowitz, Shashank Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn. Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr BO23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, A. M. Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive malignant brain tumor with almost inevitable recurrence despite multimodal management with surgical resection and radio-chemotherapy. While several genetic, proteomic, cellular, and anatomic factors interplay to drive recurrence and promote treatment resistance, the epigenetic component remains among the most versatile and heterogeneous of these factors. Herein, the epigenetic landscape of GBM refers to a myriad of modifications and processes that can alter gene expression without altering the genetic code of cancer cells. These processes encompass DNA methylation, histone modification, chromatin remodeling, and non-coding RNA molecules, all of which have been found to be implicated in augmenting the tumor’s aggressive behavior and driving its resistance to therapeutics. This review aims to delve into the underlying interactions that mediate this role for each of these epigenetic components. Further, it discusses the two-way relationship between epigenetic modifications and tumor heterogeneity and plasticity, which are crucial to effectively treat GBM. Finally, we build on the previous characterization of epigenetic modifications and interactions to explore specific targets that have been investigated for the development of promising therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc0fa03a9f64a66bc6beac8c2a3a49bfa95cc71" target='_blank'>
              The epigenetic mechanisms involved in the treatment resistance of glioblastoma
              </a>
            </td>
          <td>
            Aanya Shahani, Hasan Slika, Ahmad Elbeltagy, Alexandra Lee, Christopher Peters, Toriyn Dotson, Divyaansh Raj, Betty Tyler
          </td>
          <td>2025-02-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Genomic and transcriptomic sequencing technologies have revolutionized our ability to characterize prostate cancer at the molecular level. The underlying premise of next-generation sequencing technologies and their current and evolving applications in prostate cancer management are provided in the review.


RECENT FINDINGS
Improved methodologies are allowing timely sequencing of the coding regions or both the coding and noncoding regions of the genome to help identify potential mutations and structural variations in the prostate cancer genome, some of which are currently also targetable therapeutically. DNA microarray- based differential gene expression has been supplanted by RNA sequencing (RNA-seq), which not only allows for more accurate quantitation but also nucleotide-level resolution to investigate the entire transcriptome, including alternative gene spliced transcripts and noncoding RNA transcripts, whose full clinical implications have yet to be fully understood and realized. Gene classifier platforms that predict risk of recurrence or metastasis are being incorporated into prostate cancer management algorithms. In the appropriate clinical context, not only somatic but also germline mutation testing is being recommended.


SUMMARY
Continued clinical integration of sequencing technologies and ongoing research will lead to improved understanding of prostate cancer biology and prostate cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e945cd17bb6878accfa941b57523d14ed0d229c" target='_blank'>
              Genomic and transcriptomic sequencing in prostate cancer.
              </a>
            </td>
          <td>
            Safiullah Rifai, Azimullah Rifai, Xiaolei Shi, M. A. Khan, Wei Guang, Linbo Wang, Luke Tallon, Arif Hussain
          </td>
          <td>2025-02-28</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c673a6c02d35f0c2838ab5179fa8927b41b41fc" target='_blank'>
              Genomic characterization of chronic lymphocytic leukemia in patients of African ancestry
              </a>
            </td>
          <td>
            Cecilia Bonolo de Campos, Chantal E McCabe, L. Bruins, Daniel R. O’Brien, Sochilt Brown, R. Tschumper, C. Allmer, Y. Zhu, K. Rabe, S. Parikh, Neil E. Kay, Huihuang Yan, J. Cerhan, John N Allan, Richard R Furman, J. B. Weinberg, Danielle M Brander, D. Jelinek, M. Chesi, S. Slager, Esteban Braggio
          </td>
          <td>2025-02-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Simple Summary Prostate cancer (PCa) is a heterogenous malignancy in men, representing the second most diagnosed cancer and the fifth leading cause of cancer-related death globally in 2022. PCa incidence and progression is dependent on several predisposing factors, most significantly genetics and germline/somatic mutations. With the development of whole-genome sequencing (WGS) technology, it is now possible to determine the mutational statuses of PCa-related genes for improved clinical decision making and individualized patient care. The National Comprehensive Cancer Network (NCCN) Guidelines now recommend germline multigene testing followed by post-test genetic counseling. Depending on the germline mutations in homologous recombination (HR) repair genes or in mismatch repair (MMR) genes, specific treatment options, including poly-ADP-ribose polymerase (PARP) inhibition or PD-1 blockade, may be of more clinical benefit. This review highlights specific PCa-related germline mutations in racially diverse populations and therapeutic approaches to treat PCa patients with DNA damage repair gene alterations in advanced disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f685df01c211531229e0572b44cf7405f9d7cc" target='_blank'>
              Ancestry-Specific DNA Damage Repair Gene Mutations and Prostate Cancer
              </a>
            </td>
          <td>
            Talaibek Borbiev, Kevin Babcock, K. Sinopole, Gregory T. Chesnut, G. Petrovics
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Stemness-associated cell states are linked to chemotherapy resistance in AML. We uncovered a direct mechanistic link between expression of the stem cell transcription factor GATA2 and drug resistance. The GATA-binding protein 2 (GATA2) plays a central role in blood stem cell generation and maintenance. We find substantial intra- and inter-patient variability in GATA2 expression across AML patient samples. GATA2 expression varies by molecular subtype and has been linked to outcome. In a murine model, KMT2A-MLL3 driven AML originating from a stem cell or immature progenitor cell population have higher Gata2 expression and are more resistant to the standard AML chemotherapy agent doxorubicin. Deletion of Gata2 resulted in more robust induction of p53 following exposure to doxorubicin. ChIP-Seq, RNA-Seq and functional studies revealed that GATA2 regulates the expression of RASSF4, a modulator of the p53 inhibitor MDM2. GATA2 and RASSF4 are anti-correlated in human cell lines and AML patient cell bulk and single cell expression datasets. Knockdown of Rassf4 in Gata2 low cells resulted in doxorubicin or nutlin-3 resistance. Conversely, overexpression of Rassf4 results in sensitization of cells expressing high levels of Gata2. Finally, doxorubicin and nutlin-3 are synergistic in Gata2-high murine AML, as well as AML patient samples. We discovered a previously unappreciated role for GATA2 in dampening p53-mediated apoptosis via transcriptional regulation of RASSF4, a modulator of MDM2. This role for GATA2 directly links the expression of a stemness associated transcription factor to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e4d0de77d706d929171b654f103344701ea6a" target='_blank'>
              GATA2 links stemness to chemotherapy resistance in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Fatemeh Alikarami, H. Xie, Simone S. Riedel, Haley Goodrow, D. Barrett, Leila Mahdavi, Alexandra Lenard, Changya Chen, T. Yamauchi, Etienne Danis, Zhendong Cao, Vu L. Tran, M. Jung, Yapeng Li, Hua Huang, Junwei Shi, Kai Tan, D. Teachey, E. Bresnick, Tobias Neff, Kathrin M Bernt
          </td>
          <td>2025-01-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Background: Persisters are a sub-population of tumor cells that survive anti-cancer therapy and are thought to be a major cause of tumor recurrence. Persisters have been identified following both drug- and immune-therapy but are generally considered to be very distinct entities. The focus of this study is to examine any similarities/differences between immunotherapy persister cells (IPCs) and drug tolerant persister cells (DTPs). Both pharmacological agents and immune cells often kill via apoptosis. Therefore, we investigated if reduced apoptotic sensitivity is a survival mechanism common to IPCs and DTPs and whether the use of BH3 mimetics to enhance apoptotic sensitivity could reduce the survival of IPCs and DTPs. Multi-therapy resistance is often observed in the clinic, therefore, we investigated if IPCs and DTPs exhibit a reciprocal multi-therapy resistant phenotype which could be conferred through a reduction in apoptotic sensitivity. Methods: IPCs were generated by co-culturing HeLa and U251 tumor cell lines with EGFR-CAR T cells for 3 weeks, replenishing CAR T cells every 3 days. Using three tumor cell line models (Hela, MC38 and B16), nine DTP cell lines were generated by chronically treating tumor cells for 7 days with an IC90 dose of drug (5-FU, oxaliplatin and etoposide), replenishing drug dose every 3 days. A CellTiter-glo or CellTox assay was used to determine the sensitivity of IPCs and DTPs to drugs and radiation. Sensitivity to T cell attack was assessed by flow cytometry (cell counting beads and zombie viability dye. Microscopy-based BH3 profiling was employed to measure the mitochondrial apoptotic sensitivity of persisters and to probe for therapeutically exploitable anti-apoptotic dependencies. Results: IPCs acquired a reduced sensitivity to multiple drug classes and radiotherapy. Likewise, DTPs also developed a reduced sensitivity to multiple drug classes and radiotherapy but also acquired a reduced sensitivity to CAR T cell killing. Depending on the cell line and therapy, some persisters developed either an increased or decreased sensitivity to mitochondrial apoptosis. Inhibition of specific anti-apoptotic dependencies using BH3 mimetics increased apoptotic sensitivity of persisters. Conclusions: Both immunotherapy and drug-tolerant persisters develop a reduced sensitivity to multiple therapeutic modalities, suggesting non-immune mechanisms are important in the survival of cancer cells after immunotherapy. These results suggest there are common mechanisms involved in producing persisters from dramatically different therapies. Our study highlights the potential of BH3 profiling to guide the selection of therapies to target persisters.
 Citation Format: Maria Davern, Cole J Turner, Lara Bencsics, Brenda Chan, Jasmine Yun-Tong Kung, Michael Olson, Cheyanne Walker-Williams, David Barbie, Shaili Soni, Patrick Bhola, Marcela V Maus, Anthony Letai. Survival mechanisms common to immunotherapy and drug tolerant persister tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d109d6041879491cc459a88c72e3556ce8634101" target='_blank'>
              Abstract A040: Survival mechanisms common to immunotherapy and drug tolerant persister tumor cells
              </a>
            </td>
          <td>
            Maria Davern, Cole J Turner, Lara Bencsics, Brenda Chan, Jasmine Yun-Tong Kung, Michael L. Olson, Cheyanne Walker-Williams, David A. Barbie, Shaili Soni, P. Bhola, Marcela V. Maus, Anthony Letai
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070c22030bfb86f8cf3bc0c2063075e7b4258116" target='_blank'>
              Identification of the clinical and genetic characteristics of gliomas with gene fusions by integrated genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            Guo-zhong Yi, Hua-yang Zhang, Tian-shi Que, Shan-qiang Qu, Zhi-yong Li, Song-tao Qi, Wen-yan Feng, Guang-long Huang
          </td>
          <td>2025-01-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Precision medicine in oncology is rapidly emerging as the next frontier for cancer therapy, and accordingly there exists a need to identify new therapeutically actionable targets. Gene fusions have long been established as biomarkers and therapeutic targets for cancer treatment, with targetable fusions such as BCR-ABL1, EML4-ALK, and TPM3-NTRK1 already having shown sustained success in the clinic. While historically studies have focused on DNA rearrangements as the source of fusion transcripts and proteins, recent advances in RNA sequencing have revealed a new landscape of fusions originating at the RNA level through intergenic cis and trans splicing. Combined, this class of molecule, termed “chimeric RNA”, presents new opportunities for molecular profiling of the fusion transcriptome. The purpose of this study is to systematically characterize chimeric RNAs across cancer to identify molecular drivers of cell growth. To build an atlas of chimeric transcripts, we applied multiple fusion-calling programs followed by stringent filtering to predict fusions in 9509 primary tumors from The Cancer Genome Atlas (TCGA) and 1019 cell lines from The Cancer Cell Line Encyclopedia (CCLE). From this we identified 7047 recurrent chimeric transcripts, 2085 of which were not found in normal tissue. To further characterize these fusions, we integrated matched structural variant predictions from whole-genome sequencing to establish the origin of formation at the DNA or RNA level and mapped transcript breakpoint coordinates to protein annotations to establish the potential for the formation of a fusion protein. To identify fusions driving growth and proliferation we developed a functional screen, integrating shRNA dependency screens from DepMap with our cell line chimeric RNA predictions. By mapping the shRNA probes to the sequences of the fusion parental genes, we are able to compare the difference in cell growth between the chimera-mapping and parent specific probes. Using this method, we are able to identify fusions functioning as cancer dependencies independently of the parental genes. Our method accurately detected well-established fusions such as EML4-ALK, FGFR3-TACC3, and TPM3-NTRK1 and showed a significant enrichment of oncogenic fusions from the COSMIC database, fusions containing established oncogenes, and fusions predicted to code for proteins. Out of the 2225 total fusion transcripts screened, we identified 47 cancer-specific transcripts predicted to function as promoters of growth in cancer. Based on degree of essentiality, expression across multiple cancer indications, and potential of forming a targetable fusion protein, we selected candidate fusions for further characterization through cell-based gain and loss of function studies to confirm oncogenicity. Our results reveal a new landscape of chimeric transcripts as drivers of cancer and hold potential for the development of new precision medicine-based treatments.
 Citation Format: Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li. Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer chemoresistance presents a challenge in oncology, often leading to treatment failure and disease progression. CD44, a multifunctional cell surface glycoprotein, has garnered attention for its involvement in various aspects of cancer biology. Through alternative splicing, CD44 can form isoforms with the inclusion of only standard exons, typical for normal tissue, or with the addition of variant exons, frequently expressed in cancer tissue and associated with chemoresistance. The functions of CD44 involved in regulation of cancer signaling pathways are being actively studied, and the significance of specific variant exons in modulating cell death pathways, central to the response of cancer cells to chemotherapy, begins to become apparent. This review provides a comprehensive analysis of the association of CD44 variant exons/total CD44 with clinical outcomes of patients undergoing chemotherapy. The role of CD44 variant exons v6, v9 and others with a significant effect on patient chemotherapy outcomes by means of key cellular death pathways such as apoptosis, ferroptosis and autophagy modulation is further identified, and their impact on drug resistance is highlighted. An overview of clinical trials aimed at targeting variant exon-containing isoforms is provided, and possible directions for further development of CD44-targeted therapeutic strategies are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c896fa2073872384fe9590cc27feab2b38f88e9" target='_blank'>
              CD44 variant exons induce chemoresistance by modulating cell death pathways
              </a>
            </td>
          <td>
            Maria Yanova, Evgeniya Stepanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene therapy and genome editing have emerged as transformative approaches in the management of a diverse range of genetic and acquired diseases. This evaluation offers a thorough examination of the present state and prospects of these innovative technologies. Gene therapy is a prospective approach to the treatment and prevention of a variety of conditions, including complex cancers and inherited genetic disorders, which entail the introduction, removal, or modification of genetic material within a patient's cells. Genome editing, particularly through techniques such as CRISPR-Cas9, enables targeted corrections of genetic defects and opens new possibilities for personalized medicine by allowing for precise modifications at the DNA level. The review addresses the ethical implications, clinical applications, and significant advancements of these technologies. This article endeavors to underscore the substantial influence of gene therapy and genome editing on contemporary medicine by assessing the most recent research and clinical trials, thereby emphasizing their potential to revolutionize disease treatment and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fcf14d7421a285bda69ccae261660da8cc8fcc" target='_blank'>
              Gene Therapy and Gene Editing: Current Trends and Future Prospects of Molecular Medicine.
              </a>
            </td>
          <td>
            Aniruddha Sen, Vijay Singh, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar, Sudhir Shyam Kushwaha
          </td>
          <td>2025-02-18</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transposable elements (TEs) are reactivated in tumors and serve as significant contributors to tumor transcriptomic complexity and heterogeneity. However, their repetitive nature and diverse transcriptional forms have hindered efforts to characterize TE-derived transcripts independent of host gene contexts. Here, using our self-developed splicing-junction analysis tool ASJA, we systematically interrogated TE transcription across 32 cancer types, identifying transcripts autonomously initiated from and exclusively spliced among TEs with high TE content, termed TEtrans. Our analysis revealed 5,361 TEtrans exhibiting pan-cancer prevalence, tumor-specific expression and features of mature transcripts, including canonical splicing, 5’ caps and polyA tails. Mostly unannotated and enriched in intergenic regions, TEtrans are predominantly derived from primate-specific classes with conserved splice pattern. TEtrans burden demonstrates heterogeneous associations with prognosis and immune activity across cancers, while their expression can be epigenetically dual-regulated by stemness- and inflammation-associated transcription factors. Functional studies uncovered that TEtrans could act as oncogenes, exemplified by tsTE1, a HERVH-derived transcript that promotes colorectal cancer proliferation by enhancing TOP1-mediated DNA supercoil relaxation. Remarkably, ∼7.2% of TEtrans are identified to encode tumor neoantigens, including viral proteins and unannotated peptides, which are shared among patients and validated by proteogenomic analysis. These TEtrans-derived neoantigens are immunogenic both in vitro and in vivo, and exceed neoepitopes from genomic alterations in abundance per tumor, particularly in cancers with low mutational burden. Collectively, as a gene-independent form of TE-derived transcripts, TEtrans represents a unique source of oncogenes and tumor neoantigens, expanding the functional repertoire of TEs in cancer biology and offering new avenues for therapeutic targeting. Highlights TEtrans are tumor-specific transcripts with conserved splicing patterns, predominantly derived from primate TEs in intergenic regions. TEtrans expression is epigenetically activated and dually regulated by stemness/inflammation transcription factors. TEtrans function as oncogenes, exemplified by tsTE1 enhancing TOP1-mediated DNA relaxation. TEtrans encode tumor neoantigens that elicit CD8+ T cell responses, providing targets for immunotherapy, especially in low-mutation tumors. Abstract figure">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90153db4362d7e931500fbc74364483bb05472a" target='_blank'>
              Transcriptomic splicing analysis reveals a gene-independent form of TE reactivation in tumors representing unrecognized source of oncogenes and neoantigens
              </a>
            </td>
          <td>
            Ziteng Li, Yichao Bao, Qin Li, Hongwu Yu, He-Yun Zhang, Yifan Wen, Yu Yang, Jingjing Zhao, Peng Lin, Yan Li, Zhixiang Hu, Xichun Hu, Xiaodong Zhu, Shenglin Huang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer.
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway.
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            Mohamed J Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Background: Bladder cancer poses a significant threat to human health. In recent years, genetic circuit therapy has emerged as a novel alternative for precision tumor treatment, demonstrating promising potential for clinical application. Compared to traditional drugs, genetic circuits – typically carried by plasmids – offer advantages such as modularity, druggability, and shorter drug development cycles. These circuits can identify multiple molecular signals from tumors and integrate them through logic gates to specifically target tumor cells. Furthermore, by assembling effector modules, they can induce specific forms of cell death in tumor cells or alter malignant phenotypes, thereby reshaping the immune microenvironment to produce efficient, durable, and controllable antitumor effects. The urinary system serves as an ideal model for this new therapy due to its accessibility from the outside. Several genetic circuits have already been validated for the treatment of bladder cancer. This review outlined the effectiveness and potential value of genetic circuits in bladder cancer therapy. Objective: This review focused on the design principles of genetic circuits, their ability to recognize and convert signals, their therapeutic signal output, and the associated delivery vehicles. We also discussed the challenges and future prospects of genetic circuits as a novel form of “intelligent biotherapy.” Conclusion: The gene circuit can identify multiple signals, processing complex information, and generating multiple effects, thus providing a new approach for personalized treatment of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Rationale: Anaplastic thyroid carcinoma (ATC) is an extraordinarily aggressive form of thyroid cancer, frequently presenting with locally advanced infiltration or distant metastases at the time of initial diagnosis, thus missing the optimal window for surgical intervention. Consequently, systemic chemotherapy and targeted therapies are vital for improving the prognosis of ATC. However, ATC exhibits significant resistance to conventional treatments, highlighting the need to elucidate the biological mechanisms underlying this drug resistance and identify novel therapeutic targets to overcome it. Methods: We conducted a comprehensive analysis of both bulk and single-cell RNA sequencing (scRNA-seq) data from ATC samples to screen for m5C modification-related genes associated with multidrug resistance (MDR). We then performed IC50 assays, flow cytometry, and employed a spontaneous tumorigenic ATC mouse model with Nsun2 knockout to demonstrate that NSUN2 promotes MDR in ATC. To investigate the mechanisms of NSUN2-mediated drug resistance, we generated NSUN2-knockout ATC cell lines and performed transcriptomic, proteomic, and MeRIP-seq analyses. Additionally, RNA sequencing and alternative splicing analyses were conducted to determine global changes upon NSUN2 knockout. We further explored the underlying mechanisms of the NSUN2/SRSF6/UAP1 axis through glycoprotein staining, denaturing IP ubiquitination, nuclear-cytoplasmic fractionation, and PCR. Lastly, we evaluated the synergistic effects of a small-molecule NSUN2 inhibitor with anticancer agents both in vitro and in vivo. Results: Our findings reveal that NSUN2 expression correlates significantly with MDR in ATC. NSUN2 operates as a "writer" and ALYREF as a "reader" of m5C on SRSF6 mRNA, inducing alternative splicing reprogramming and redirecting the splice form of the UAP1 gene from AGX1 to AGX2. As a result, AGX2 enhances the N-linked glycosylation of ABC transporters, stabilizing them by preventing ubiquitination-mediated degradation. Furthermore, an NSUN2 inhibitor reduces NSUN2 enzymatic activity and diminishes downstream target expression, presenting a novel, promising therapeutic approach to overcome MDR in ATC. Conclusions: These findings suggest that the NSUN2/SRSF6/UAP1 signaling axis plays a vital role in MDR of ATC and identify NSUN2 as a synergistic target for chemotherapy and targeted therapy in ATC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0103a2915d22e848d6f65e7673d036ac4bf8979b" target='_blank'>
              NSUN2-mediated m5C modification drives alternative splicing reprogramming and promotes multidrug resistance in anaplastic thyroid cancer through the NSUN2/SRSF6/UAP1 signaling axis
              </a>
            </td>
          <td>
            Xiukun Hou, Qiman Dong, Jie Hao, Min Liu, Junya Ning, Mei Tao, Zhongyu Wang, Fengli Guo, Dongmei Huang, Xianle Shi, Ming Gao, Dapeng Li, Xiangqian Zheng
          </td>
          <td>2025-01-27</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a3dbca1c9498d07a50c2a165aadccc038c723f" target='_blank'>
              Genomic profiling at a single center cracks the code in inborn errors of immunity.
              </a>
            </td>
          <td>
            Alessandro Andriano, V. Desantis, C. Marasco, A. Marzollo, Silvia Bresolin, N. Resta, Lucia Di Marzo, F. Pappagallo, Antonella Mascolo, Ingrid Catalina Caradonna, Simona D'Amore, Angelo Vacca, A. Solimando
          </td>
          <td>2025-01-28</td>
          <td>Internal and emergency medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The human peripheral blood displays diverse molecular characteristics across populations, understanding the drivers and underlying mechanisms of which remains challenging. Here, we introduce the Chinese Immune Multi-Omics Atlas (CIMA), elucidating sex-, age-, and genetic-related molecular variations by analyzing multi-omics data from 428 adults with over 10 million immune cells. CIMA generated an enhancer-driven gene regulatory network, identifying 237 high-quality regulons and revealing cell type-specific regulatory mechanisms. Additionally, 11,521 lead cis-expression quantitative trait loci (eQTLs) and 46,339 chromatin accessibility QTLs (caQTLs) were identified at cell type level. CIMA also uncovered pleiotropic associations among immune-related disease risk loci, eQTLs, and caQTLs in a cell type-specific manner. Lastly, a novel cell language model, CIMA-CLM, was developed to predict chromatin accessibility and noncoding variant effects using chromatin sequences and gene expressions. This work represents a population-scale multi-omics resource of human immune cells, providing a valuable reference for future investigation of immune-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f8b66ae23a1d254045e2d47bbe65620395c404" target='_blank'>
              Single-Cell Genomics Elucidates Molecular Variations and Regulatory Mechanisms in Circulating Immune Cells
              </a>
            </td>
          <td>
            Jianhua Yin, Yuhui Zheng, Zhuoli Huang, Wenwen Zhou, Yue Yuan, Pengfei Cai, Yong Bai, Shichen Yang, Yue Gao, Shanshan Duan, Yang Wang, Wenxi Zhang, Xinyu Zhang, Yilin Wei, Zekai Xu, Yaling Huang, Ying Liu, Weikai Wang, Tao Yang, Jingzhi Lv, Zhongjin Zhang, Xiaoya Chen, Xiru Zhang, Fupeng Li, Yan Zhang, Guodan Zeng, Xue Wang, Wen Ma, Guixue Hou, Shijie Hao, Chang Liu, Yiwei Lai, Bo Wang, Yuxiang Li, Wenwei Zhang, Peng Gao, Jun Xie, Miguel A. Esteban, Ying Gu, Jiansong Ji, Ting Qi, Boxiang Liu, Jian Wang, Jian Yang, Xun Xu, Longqi Liu, Xin Jin, Chuanyu Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ad7065cbf4c15026ee68bc1c1e9443de9a4af3" target='_blank'>
              Identification of T cell dysfunction molecular subtypes and exploration of potential immunotherapy targets in BRAF V600E-mutant colorectal cancer
              </a>
            </td>
          <td>
            Tiefeng Gu, Haonan Qi, Jiaqi Wang, Liangwei Sun, Yongqi Su, Hanqing Hu
          </td>
          <td>2025-02-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd82847b06d814bccf91fe62dfe773b93ac1d7" target='_blank'>
              Exploring the principles behind antibiotics with limited resistance
              </a>
            </td>
          <td>
            Elvin Maharramov, Márton Simon Czikkely, P. Szili, Z. Farkas, Gábor Grézal, Lejla Daruka, Eszter Kurkó, Léna Mészáros, Andreea Daraba, Terézia Kovács, Bence Bognár, Szilvia Juhász, Balázs Papp, Viktória Lázár, Csaba Pál
          </td>
          <td>2025-02-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="
 My research program employs a pipeline of functional approaches for analysis of primary cells from patients with acute myeloid leukemia (AML) and other hematologic malignancies. This platform aims to identify the druggable landscape across hematologic malignancies and prioritize novel combination therapies. The pipeline is comprised of gene knockdown (e.g. siRNA, CRISPR/Cas) and small-molecule libraries to interrogate the genes, pathways, cellular processes, and microenvironmental crosstalk that govern cancer cell growth and drug response, and these functional screens are complemented with deep sequencing. Simultaneous application of these functional approaches with omic data has accelerated our understanding of druggable pathways that contribute to neoplasia. One example of this platform has been our Beat AML 1.0 platform in which we have assessed ex vivo drug sensitivity, whole exome sequencing, and RNA-sequencing for a cohort of nearly 1,000 AML patient samples, which provides a functional genomic landscape for this disease. We have also utilized genome-wide CRISPR screens with overlay of key therapeutic agents in cell line models, and coupling of these genome-wide screen data with the large patient sample dataset has illuminated a diversity of determinants of therapeutic response. In this lecture, I will provide an overview of these functional screening platforms, and I will illustrate strategies of data integration that have enabled rapid discovery and translation of new targeted therapy strategies. This will include discussion of novel biomarkers of drug response in AML, including mutational drivers of drug response and strong correlation of AML tumor cell maturation state with response to a diversity of drugs and drug families. Finally, the lecture will provide examples of the way in which these data can be leveraged to identify novel drug combinations with broad activity and to examine the mechanistic underpinnings of improved combination activity relative to single agents. Collectively, this dataset and integrative approach provide a functional genomic landscape of AML with new biomarkers of drug response.
 Citation Format: Jeffrey W Tyner. Functional omics for novel therapeutic combinations in myeloid malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr IA002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W. Drong, Bryan Thornlow, Jimmy Z. Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M. Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy affecting women, manifesting as a heterogeneous disease with diverse molecular characteristics and clinical presentations. Recent studies have elucidated the role of epigenetic modifications in the pathogenesis of breast cancer, including drug resistance and efflux characteristics, offering potential new diagnostic and prognostic markers, treatment efficacy predictors, and therapeutic agents. Key modifications include DNA cytosine methylation and the covalent modification of histone proteins. Unlike genetic mutations, reprogramming the epigenetic landscape of the cancer epigenome is a promising targeted therapy for the treatment and reversal of drug resistance. Epidrugs, which target DNA methylation and histone modifications, can provide novel options for the treatment of breast cancer by reversing the acquired resistance to treatment. Currently, the most promising approach involves combination therapies consisting of epidrugs with immune checkpoint inhibitors. This review examines the aberrant epigenetic regulation of breast cancer initiation and progression, focusing on modifications related to estrogen signaling, drug resistance, cancer progression, and the epithelial–mesenchymal transition (EMT). It examines existing epigenetic drugs for treating breast cancer, including agents that modify DNA, inhibitors of histone acetyltransferases, histone deacetylases, histone methyltransferases, and histone demethyltransferases. It also delves into ongoing studies on combining epidrugs with other therapies and addresses the upcoming obstacles in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1d287b99832f39586fe0fd89b9b4c10e300e3c" target='_blank'>
              Cellular Epigenetic Targets and Epidrugs in Breast Cancer Therapy: Mechanisms, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Ibrahim S Alalhareth, Saleh M. Alyami, Ali H. Alshareef, Ahmed O. Ajeibi, M. A. Al Munjem, Ahmad A. Elfifi, Meshal M. Alsharif, S. A. Alzahrani, Mohammed A. Alqaad, Marwa B. Bakir, Basel A. Abdel-Wahab
          </td>
          <td>2025-02-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Precision medicine has revolutionized the treatment of colorectal cancer by enabling a personalized approach tailored to each patient’s unique genetic characteristics. Genomic profiling allows for the identification of specific mutations in genes such as KRAS, BRAF, and PIK3CA, which play a crucial role in cell signaling pathways that regulate cell proliferation, apoptosis, and differentiation. This information enables doctors to select targeted therapies that inhibit specific molecular pathways, maximizing treatment effectiveness and minimizing side effects. Precision medicine also facilitates adaptive monitoring of tumor progression, allowing for adjustments in therapy to maintain treatment effectiveness. While challenges such as high costs, limited access to genomic technology, and the need for more representative genomic data for diverse populations remain, collaboration between researchers, medical practitioners, policymakers, and the pharmaceutical industry is crucial to ensure that precision medicine becomes a standard of care accessible to all. With continued advances and support, precision medicine has the potential to improve treatment outcomes, reduce morbidity and mortality rates, and enhance the quality of life for colorectal cancer patients worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ce8f60143b7348cbfaa508b5bbdc79c01b3a" target='_blank'>
              Precision medicine in colorectal cancer: genomics profiling and targeted treatment
              </a>
            </td>
          <td>
            Adi Muradi Muhar, A. J. Velaro, Arya Tjipta Prananda, S. E. Nugraha, Princella Halim, R. A. Syahputra
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="We examine the complex relationship between genomic copy number variation (CNV) and gene expression, highlighting the relevance to cancer biology and other biological contexts. By tracing the history of genometranscriptome correlations, we emphasize the complexity and challenges in understanding these interactions, particularly within the heterogeneous landscape of human cancers. Recent advances in computational algorithms and high-throughput single-cell multi-omic sequencing technologies are discussed, demonstrating their potential to refine our understanding of cancer biology and their limitations. The integration of genomic and transcriptomic analyses, which offers novel insights into tumor evolution and heterogeneity as well as therapeutic strategies, is presented as a crucial approach for advancing cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor prognosis; however, advancements in cancer genome profiling using next-generation sequencing have provided new perspectives. KRAS mutations are the most frequently observed genomic alterations in patients with PDAC. However, until recently, it was not considered a viable therapeutic target. Although KRAS G12C mutations for which targeted therapies are already available are infrequent in PDAC, treatments targeting KRAS G12D and pan-KRAS are still under development. Similarly, new treatment methods for KRAS, such as chimeric antigen receptor T-cell therapy, have been developed. Several other potential therapeutic targets have been identified for KRAS wild-type PDAC. For instance, immune checkpoint inhibitors have demonstrated efficacy in PDAC treatment with microsatellite instability-high/deficient mismatch repair and tumor mutation burden-high profiles. However, for other PDAC cases with low immunogenicity, combination therapies that enhance the effectiveness of immune checkpoint inhibitors are being considered. Additionally, homologous recombination repair deficiencies, including BRCA1/2 mutations, are prevalent in PDAC and serve as important biomarkers for therapies involving poly (adenosine diphosphate-ribose) polymerase inhibitors and platinum-based therapies. Currently, olaparib is available for maintenance therapy of BRCA1/2 mutation-positive PDAC. Further therapeutic developments are ongoing for genetic abnormalities involving BRAF V600E and the fusion genes RET, NTRK, NRG, ALK, FGFR2, and ROS1. Overcoming advanced PDAC remains a formidable challenge; however, this review outlines the latest therapeutic strategies that are expected to lead to significant advancements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb24cef2792625badc2e58e84464e22d77097b0" target='_blank'>
              Current status of cancer genome medicine for pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Toshifumi Doi, Takeshi Ishikawa, M. Moriguchi, Yoshito Itoh
          </td>
          <td>2025-02-02</td>
          <td>Japanese journal of clinical oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Advancing precision medicine research relies heavily on laboratory models that accurately reflect the rich intra- and inter-patient diversity of human tumors. While large-scale resources built on historical cell lines have provided valuable insights, these models often lack the molecular diversity needed for comprehensive translational research and may undergo genetic drift in culture, reducing their fidelity. Recent advances in technologies, such as organoid culture, have improved model derivation; however, systematic pan-cancer comparisons of genomic, transcriptomic, and epigenomic integrity following long-term ex vivo expansion remain limited. Here, we report on the Human Cancer Models Initiative (HCMI), an international program that has generated and systematically characterized 665 organoid, spheroid, and cell line models derived from over 2,500 consented donors. These models have undergone comprehensive whole-genome, exome, methylome, and transcriptome analyses. The HCMI collection includes 47 unique models representing 16 rare and histologically distinct cancer subtypes, including one of a kind disease models (e.g. desmoid tumor) and greatly broadening the diversity of available cancer models. Through integrative concordance analysis, we demonstrate that 96% of the 417 analyzed models closely mirror the molecular profiles of their parental tumors. Interesting rare exceptions to this concordance include subsets of models, where cell culture media conditions, tumor stroma, or population selection appear to influence cellular characteristics. We validated observed cellular state shifts using single-cell RNA sequencing, providing insights into the impact of ex vivo culture on model integrity. We further explored the translational utility of this resource by investigating treatment exposures, post-treatment mutational signatures, and extrachromosomal DNA (ecDNA) amplifications and their potential implications on treatment resistance within these models. This community resource offers a comprehensive roadmap for capturing a broader spectrum of cancer diversity in preclinical models, ultimately supporting and advancing therapeutic discovery.
 Citation Format: Dina ElHarouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, Toshinori Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Perez-Mayoral, Megan J Stine, Eva Tonsing-Carter, James M Clinton, The HCMI Network, Peter W Laird, Calvin J Kuo, Olivier Elemento, David L Spector, Andrew D Cherniack, Kyle Ellrott, Martin L Ferguson, Rameen Beroukhim, Katherine A Hoadley, Nicolas Robine, Mathew Garnett, Andrea Califano, Paul T Spellman, David A Tuveson, Keith L Ligon, Daniela S Gerhard, Louis M Staudt, Jesse Boehm. Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d11642e20fef657158a832ae45b3c22aa7b26f" target='_blank'>
              Abstract B025: Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Pérez-Mayoral, Megan J Stine, Eva Y. Tonsing-Carter, James M Clinton, The Hcmi Network, Peter W. Laird, Calvin J Kuo, O. Elemento, David L. Spector, A. Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Mathew Garnett, Andrea Califano, Paul T Spellman, D. Tuveson, Keith L. Ligon, Daniela S Gerhard, Louis M Staudt, Jesse S Boehm
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 The emerging use of immunotherapy for cancer treatment has led to a greater need for characterizing markers of response. Incorporating immune profiling tools into whole genome and transcriptome sequencing and analysis (WGTA) pipelines facilitates the identification of biomarkers of response that may be used to inform clinical trial design. The Personalized OncoGenomics (POG) program has enrolled >1,200 patients with advanced/metastatic cancers of diverse histologies and underwent WGTA, which allowed for a retrospective investigation of biomarkers for immune checkpoint inhibitor (ICI) therapy from 194 adult patients. We demonstrated that a combination of biomarkers, including tumour mutation burden (TMB), mutations in SWI/SNF genes, presence of viruses, and inferred RNA-derived immune cell expression scores could predict response to therapy more effectively than individual biomarkers alone. This analysis led to the design of a combined biomarkers score termed the immune burden variant (IBV) score, informing the Canadian Atezolizumab Precision Targeting for Immunotherapy Intervention (CAPTIV-8) trial (NCT04273061). Despite of its utility in adult cancers, the IBV score has limited application to pediatric cancers as biomarkers such as high TMB or the presence of virus are rarely present. Based on previous work, we constructed a scoring system using three biomarkers from transcriptome: (a) CD8+ T-cell score: RNA-based deconvolution to estimate CD8+ T cell abundance using CIBERSORT; (b) M1/M2 score: a ten-gene signature to assess expression patterns of M1:M2 macrophages, and (c) IPASS score: a 15-gene signature that predicts T-cell infiltration. For each biomarker, we calculated percentiles with reference to a pediatric pan-cancer cohort with 222 samples from the Canadian PROFYLE (PRecision Oncology For Young peopLE) program and validated using 600 samples from the Australian ZERO (Zero Childhood Cancer precision medicine) program. Outlier high scores were defined as two scores >80th percentile, excluding samples of lymph node origin and hematologic malignancies. Within the PROFYLE + ZERO combined cohorts, we observed outlier high scores across many pediatric cancer types such as sarcoma, rhabdoid tumor, glioma, neuroblastoma, chordoma, melanoma, mesothelioma, and carcinoma. We constructed an immune profiling bioinformatic workflow container called RICO (Rna-seq Immune COntainer), which takes RNA sequencing data as input, and calculates scores and percentiles of these three biomarkers. RICO will be used to assess patient eligibility for the ICI combination therapy arm in the international pediatric precision oncology basket trial OPTIMISE “Optimal Precision Therapies to CustoMISE Care in Childhood and Adolescent Cancer” (NCT06208657). The development of the CAPTIV-8 and OPTIMISE trials demonstrates the utility of WGTA analysis to aid in clinical trial design and investigate key factors of cancer treatment response. In addition, a cohort of over 2000 samples with WGTA and immune profiling has been generated as a resource for ongoing investigation.
 Citation Format: Yaoqing Shen, Chelsea Mayoh, Kathleen Wee, Erin Pleasance, Emma Titmuss, Richard Corbett, Laura Williamson, Zakhar Krekhno, Arash Nabbi, Raoul Santiago, Stephanie Bianco, Scott Davidson, Adam Shlien, Kyoko E Yuki, Denise Connolly, Daniel Morgenstern, Melika Bonakdar, Greg Taylor, Veronika Csizmok, Cameron J Grisdale, Melissa McConechy, Jing Xu, John H Dupuis, Karen Mungall, Eric Chuah, Andrew Mungall, Jessica Nelson, Stephen Yip, Sophie Sun, Howard Lim, Daniel Renouf, Janessa Laskin, Sarah Cohen-Gogo, Shahrad R Rassekh, Rebecca J Deyell, Marco A Marra, Paul G Ekert, Steven JM Jones. Immune profiling tools for adult and pediatric cancer: From whole genome and transcriptome sequencing to immunotherapy clinical trial design [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6ca066f0de020212e2771b80c63997af778abe" target='_blank'>
              Abstract A044: Immune profiling tools for adult and pediatric cancer: From whole genome and transcriptome sequencing to immunotherapy clinical trial design
              </a>
            </td>
          <td>
            Yaoqing Shen, Chelsea Mayoh, Kathleen Wee, E. Pleasance, E. Titmuss, R. Corbett, L. Williamson, Zakhar Krekhno, A. Nabbi, Raoul Santiago, Stéphanie Bianco, Scott Davidson, A. Shlien, K. Yuki, Denise Connolly, Daniel A. Morgenstern, M. Bonakdar, Greg Taylor, V. Csizmok, C. Grisdale, Melissa McConechy, Jing Xu, John H. Dupuis, K. Mungall, E. Chuah, A. Mungall, J. Nelson, Stephen Yip, Sophie Sun, Howard Lim, D. Renouf, J. Laskin, S. Cohen-Gogo, S. Rassekh, R. Deyell, M. Marra, P. Ekert, Steven JM Jones
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Natural Killer (NK) cell therapy has emerged as a transformative approach in cancer immunotherapy, driven by decades of research since their discovery in the 1970s. NK cells possess a unique ability to target tumor cells through antigen-independent recognition, which has been expanded with the discovery of antigen-specific memory and licensing mechanisms. This review explores the evolution of NK cell biology, highlighting key findings such as the enhanced response of NK cells upon re-exposure to cytomegalovirus (CMV), defying conventional immune paradigms. Recent advancements in gene editing, particularly with CRISPR technology, have led to the development of "armored" NK cells. For example, knock-in strategies have enabled the insertion of chimeric antigen receptors (CAR), enhancing tumor specificity, while knock-out approaches have improved NK cell activation by removing inhibitory molecules like PD-1 and NKG2A. Early clinical trials demonstrate improved tumor-targeting efficacy and reduced resistance. However, challenges remain, including high R&D costs, limited clinical data, and technical uncertainties such as off-target effects. In China, NK cell therapy is in its nascent stages, with increasing interest but limited clinical applications due to regulatory and funding barriers. This review provides a comprehensive analysis of NK cell evolution, CRISPR-based advancements, and future directions for overcoming current limitations and optimizing therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719dc00daf58fbc383533083eb81aecec4828ea4" target='_blank'>
              Epigenetic marvels: exploring the landscape of colorectal cancer treatment through cutting-edge epigenetic-based drug strategies
              </a>
            </td>
          <td>
            A. Tahghighi, Effat Seyedhashemi, Javad Mohammadi, Arash Moradi, Aria Esmaeili, Majid Pornour, Kimia Jafarifar, S. M. Ganji
          </td>
          <td>2025-02-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f993cc37705c650add4d10eacea4b3e32cdfd64" target='_blank'>
              BCL-2 dependence is a favorable predictive marker of response to therapy for chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            S. Chong, Junyan Lu, Rebecca Valentin, Timothy Lehmberg, J. Eu, Jing Wang, Fengfang Zhu, L. Kong, Stacey M. Fernandes, Jeremy Zhang, Charles Herbaux, Boon Cher Goh, Jennifer R. Brown, Carsten U Niemann, W. Huber, Thorsten Zenz, M. Davids
          </td>
          <td>2025-03-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Cancer cells are defined by rapid, unchecked proliferation, a hallmark that distinguishes them from normal cells. This aggressive growth often leads to a nutrient- and oxygen-deprived tumor core, where limited blood supply cannot sufficiently reach. While most cells typically undergo cell death under such glutamine-deficient conditions, a subset of cancer cells survives through specific epigenetic modifications, facilitating metastatic progression. Using spatial transcriptomics analysis, we reveal that colorectal cancer cells exploit the downregulation of GLS2 expression to avoid ferroptosis and support metastasis in the tumor's glutamine-starved core. We further identified an increase in H3K27Me3 modification as a key epigenetic driver enabling these adaptations in glutamine-deficient regions. This hypothesis has been validated in in vivo mouse colon cancer models and in patient samples compared with healthy controls, reinforcing its translational relevance. Importantly, by employing amino acid profiling through 19F NMR spectroscopy, we identified key biomarkers in the tumor microenvironment that may enhance response to immunotherapy and overcome current therapeutic resistance. These findings underscore the potential of precision medicine strategies that target ferroptosis, epigenetic regulation, and tumor-specific biomarkers, offering new avenues for therapeutic intervention in colorectal cancer patients, particularly those with core-originating tumor regions. Our work exemplifies the broader goal of functional and genomic precision medicine in cancer, aligning distinct mechanistic insights with shared therapeutic objectives.
 Citation Format: Ji Min Lee. Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) carries a poor prognosis, and new therapeutic strategies are necessary to improve outcomes for patients with this disease. Alkylating chemotherapies including temozolomide (TMZ) and lomustine (CCNU) are critical for treating GBM, but resistance mechanisms, including hypomethylation of O6-methylguanine-DNA methyltransferase (MGMT) promoter, undermine treatment. CRISPRoff is a programmable epigenetic memory editor that can induce stable and heritable gene silencing after transient delivery, and we hypothesize that CRISPRoff could potentiate the activity of TMZ and CCNU through long term suppression of target genes.


METHODS
We transiently delivered CRISPRoff mRNA along with sgRNAs against target genes using both electroporation and lipid nanoparticles (LNPs) into established GBM cell lines, patient-derived primary GBM cultures, and orthotopic GBM xenografts. Gene repression, specificity, and stability was measured by RT-qPCR, Western blot, bisulfite sequencing, and RNA-sequencing. Sensitivity to chemotherapies was measured by cell viability dose response, microscopy, and bioluminescence imaging. Genome-wide mapping of CCNU sensitizers was performed using CRISPRi screens.


RESULTS
CRISPRoff induced complete suppression of MGMT and sensitization to TMZ that were stable for over 8 months of continuous cell propagation. GBM orthotopic tumors treated with CRISPRoff against MGMT demonstrated sensitivity to TMZ in vivo, and CRISPRoff delivery resulted in chemosensitivity in patient-derived primary GBM. Genome-wide CRISPRi screens identified combinatorial genetic vulnerabilities (BRIP1, FANCE) that were targetable by multiplexed CRISPRoff to achieve sensitization to CCNU.


CONCLUSION
Transient delivery of a site-specific epigenetic memory can induce stable, complete, and multiplexed suppression of target genes for therapeutic application in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Purpose: Small cell lung cancer (SCLC) is a highly aggressive and deadly malignancy. Two major factors contributing to the high mortality of SCLC are early metastasis and rapid development of therapy resistance. Recent research suggests upregulation of the epithelial-mesenchymal transition (EMT) program and the EMT transcription factor Twist1 correlated with accelerated tumor progression and chemoradiation (CRT) resistance in SCLC. However, a causal relationship between Twist1 and these aspects of SCLC biology has not been rigorously studied. Here, we investigated whether Twist1 upregulation could promote SCLC tumorigenesis, metastasis, and resistance to CRT. Materials and Methods: We analyzed transcriptomic data from the IMpower133 phase 3 trial of extensive stage SCLC stratified by median TWIST1 levels using Kaplan-Meir statistic. To investigate the roles of Twist1 directly in SCLC biology, we have generated a novel genetically engineered mouse model (GEMM) termed RPGT (Rb1Flox; Trp53Flox; ROSA26LSL-rtTA-IRES-EGFP; Twist1-TetO7-Luc). The RPGT GEMM enables the generation of autochthonous SCLC tumors after induction with Cre recombinase adenovirus. By withdrawing or providing doxycycline to mice, we can control Twist1 expression to activate or inactivate EMT in tumors. By characterizing mouse tumor samples with histological, immunostaining, and transcriptomic profiling analyses, we evaluated the impact of Twist1 and EMT upregulation on SCLC tumor biology and plasticity as well as metastatic spread and outgrowth. To evaluate the impact of EMT activation on SCLC sensitivity to CRT, we performed in vitro viability assays on primary tumor cell lines established from RPGT tumors. We also treated RPGT mice with vs. without Twist1 overexpression with CRT to validate our in vitro data. Results: In the chemotherapy alone control arm of IMpower133 there was a trend towards a difference in overall survival (OS) between TWIST1-high or -low patients (p=025 by log-rank) and in the NMF1/NEUROD1 subtype, patients with TWIST1-high had statistically significantly inferior OS (p=0.01). We then observed that both control (no Twist1 overexpression) and Twist1-overexpressing RPGT mice developed neuroendocrine SCLC tumors. While SCLC-ASCL1 was the predominant subtype in both cohorts, RPGT tumors exhibited more plasticity, with features associated with SCLC-NEUROD1 subtype. Furthermore, Twist1 overexpression dramatically increased the metastatic incidence in RPGT compared to control animals, indicating an important role of EMT in SCLC dissemination. Transcriptomic profiling of primary tumors and matching metastases in RPGT mice also revealed downregulation of Twist1 and EMT in metastases, suggesting that EMT suppression was necessary for metastatic outgrowth. Furthermore, we found that repressing Twist1 expression enhanced SCLC susceptibility to CRT both in vitro and in vivo. Conclusions: Overall, our data suggest that TWIST1/Twist1 plays an important role in promoting SCLC plasticity, metastasis, and treatment resistance.
 Citation Format: Triet Nguyen, Jinhee Chang, Kathleen Gabrielson, Amol Shetty, Yang Song, Apaala Chatterjee, Audrey Lafargue, Yoo Sun Kim, Danielle Council, Aaron Chan, Dipanwita Dutta Chowdhury, Muhammad Ajmal Khan, Nick Connis, Daniel Sforza, Eric Gardner, Christopher McFarland, Mohammad Rezaee, Nitin Roper, Christine Hann, Phuoc T. Tran. TWIST1 Associates with resistance to treatment and Twist1 drives tumor progression in vivo for small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a1d1d7c8c25abea1123d9456e9598c5d89538a" target='_blank'>
              Abstract B029: TWIST1 Associates with resistance to treatment and Twist1 drives tumor progression in vivo for small cell lung cancer
              </a>
            </td>
          <td>
            Triet Nguyen, J. Chang, Kathleen L Gabrielson, A. Shetty, Yang Song, Apaala Chatterjee, A. Lafargue, Yoo Sun Kim, Danielle N. Council, Aaron Chan, D. Chowdhury, Muhammad Ajmal Khan, N. Connis, Daniel Sforza, Eric Gardner, Christopher McFarland, Mohammad Rezaee, Nitin Roper, C. Hann, Phuoc T Tran
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11552fa246e107609d4eac99ffa639f2882fddc5" target='_blank'>
              How to combine multiple tools for the genetic diagnosis work-up of pediatric B-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Gloria Hidalgo-Gómez, Bárbara Tazón-Vega, Carlos Palacio, Silvia Saumell, Noemí Martínez-Morgado, Víctor Navarro, Laura Murillo, Pablo Velasco, Thais Murciano, C. Díaz de Heredia, Francesc Bosch, Gemma Armengol, Margarita Ortega
          </td>
          <td>2025-01-23</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Immunotherapy is now standard of care for multiple myeloma, where the most common targets are B cell maturation antigen, CD38, and G protein-coupled receptor class C group 5 member D but strategies to identify additional targets are needed. We have utilized two large datasets of genomic data and integrated them with existing databases to identify expressed cell surface targets in myeloma patients. Importantly, we also identify targets specific to genomic defined subgroups of patients including primary translocations and high-risk subgroups. Examples of subgroup targets include ROBO3 in t(4;14), CD109 in t(14;16), CD20 in t(11;14), GPRC5D in 1q+, and ADAM28 in biallelic TP53 samples. Expression was validated by flow cytometry and CRISPR-Cas9 knock out models. Sub-clonal differences in expression were noted, as was alternative splicing of existing immunotherapy targets such as FCRL5. These results highlight the use of genomic stratification to identify novel therapeutic targets. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42ceba6c4e482c0574ea90fed2e86ea972444c" target='_blank'>
              Utilizing Genomics to Identify Novel Immunotherapeutic Targets in Multiple Myeloma High-Risk Subgroups
              </a>
            </td>
          <td>
            Enze Liu, Oumaima Jaouadi, Riya Sharma, Nathan Becker, Travis S Johnson, Parvathi Sudha, Vivek S. Chopra, Faiza Zafar, Habib Hamidi, Charlotte Pawlyn, A. Suvannasankha, R. Abonour, Brian A. Walker
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a61455bde5450ac8c919b6a07222c913fc3cde" target='_blank'>
              Aberrant DNA methylation as a key modulator of cell death pathways: insights into cancer progression and other diseases.
              </a>
            </td>
          <td>
            Ambreen Zahoor, Rafia Khazer, Insha Mehraj, Ubaid Gani, Falah Fayaz, F. Khanday, S. S. Bhat
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Prostate cancer (PCa) is the most commonly diagnosed cancer and the second-leading cause of cancer death among men in the United States, representing 24.3% of all new cancer cases in the US Metastatic castration-resistant prostate cancer (mCRPC) is a lethal variant of prostate cancer that is associated with increased aggressiveness, cancer stemness, morbidity, and the risk of developing into taxane drug-resistant PCa. As taxanes are currently the first-line chemotherapeutic agents for mCRPC, there is a critical need to develop novel agents for the treatment of mCRPC. Clofazimine (CLF) is a potential immunomodulator drug that is FDA-approved for the treatment of leprosy. Recently, using a phenotype-based high-throughput drug screening, we demonstrated the in vitro (cell lines), in vivo (mouse xenograft models), and ex vivo (patient-derived primary tumor cells) efficacy of CLF in drug-resistant forms of chronic myeloid leukemia and multiple myeloma. In this study, we demonstrate that CLF is effective as a single agent and in combination with docetaxel (DTX) in a panel of PCa cell lines representing the diversity of CRPC patients. The response to CLF in the PCa cell lines was quantified using in vitro cytotoxicity assays, caspase 3/7 activity assay for apoptosis, cell cycle analysis, and aldehyde dehydrogenase activity assay for cancer stemness. We show that following CLF treatment, cellular pathways associated with apoptosis and ER stress were up-regulated, providing additional understanding of the mechanism of action of CLF in PCa cells. Further, we also found that CLF reduces ALDH activity, which is a marker for cancer cell stemness, a subtype of cancer cells that are associated with the aggressiveness of the cancer. Finally, drug-induced changes were investigated using whole transcriptome (next-generation RNA sequencing) analysis, where CLF treatment was found to modulate the expression of several non-coding RNAs that are associated with tumor cell proliferation, cell migration, and PCa drug resistance. Western Blotting was performed to validate the dysregulation of proteins associated with cell death and apoptosis, as well as the top dysregulated pathways in mCRPC following treatment with CLF. Our results support the preclinical development of CLF as a combination therapy candidate against aggressive forms of Prostate Cancer and provide additional insights into its mechanism of action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15bb80576bbd8d5340c20144f6b3a46d4763d6f" target='_blank'>
              Clofazimine treatment modulates key non-coding RNAs associated with tumor progression and drug resistance in lethal Prostate Cancer
              </a>
            </td>
          <td>
            Sarah Batten, Harish Kumar, Suman Mazumder, T. Ghosh, Amit Kumar
          </td>
          <td>2025-02-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Background: Gastroesophageal adenocarcinomas (GEA) are characterized by chromosomal instability (CIN) predominantly consisting of high amplitude focal amplifications of oncogenes, including Cyclin E1. An initial pan-cancer analysis revealed that CCNE1 is one of the top genes whose expression correlates with focal CIN. Objective: We proposed a causal role for excess CCNE1 as a driver of CIN and gastroesophageal tumorigenesis, and investigated the therapeutic vulnerabilities of CCNE1 amplification/overexpression. Design: We engineered a novel CCNE1-overexpressing mouse gastric organoid model with Trp53 deletion and studied the functional effects on tumor initiation and progression. Whole genome sequencing of tumors was performed to characterize the effects of CCNE1 overexpression on genomic instability. To identify dependencies of CCNE1-amplified GEA, we executed a CRISPRi functional genomic screen. Top genetic dependencies were translated into potential pharmacologic vulnerabilities and tested in monotherapy and combination. Results: CCNE1 overexpression accelerated tumor initiation and progression of mouse gastric organoids with Trp53 deletion. These tumors demonstrated increased chromosomal complexity, mainly focal amplifications. Our functional genomic screen identified specific genetic dependencies of CCNE1-amplified GEA, including CCNE1/CDK2 as the strongest hits, and SKP2 as a novel dependency likely through its function as a p27 degrader. Pharmacologic inhibition of CDK2 was insufficient and antagonistic with concurrent inhibition of the DNA damage response via targeting the ATR/CHEK1 axis. However, staggered inhibition of these distinct vulnerabilities demonstrated efficacy. Conclusions: Our study identifies CCNE1 as a driver of CIN in GEA and builds a foundation for further exploration of therapies for CCNE1-amplified GEA characterized by genomic instability.
 Citation Format: ZHONG WU, Ankur Nagaraja, Chunyang Bao, Jin Zhou, Adam Bass. Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Functional precision medicine (FPM) is a new approach in oncology that uses advanced technologies to personalize anticancer treatments based on the dynamic, functional characteristics of a patient's cancer at the molecular and cellular levels. Unlike traditional precision medicine, which often focuses on genomic information alone, FPM emphasizes the use of functional assays, such as cellular response profiling, to assess how a patient’s cancer reacts to various therapeutic agents in real time. This approach aims to bridge the gap between genetic susceptibilities and treatment outcomes, providing more accurate predictions of treatment efficacy and improving clinical decision-making. The utilization of functional diagnostic assays, such as the ChemoID assay, further enhances the potential of FPM. By directly assessing drug efficacy in patient-derived cancer stem cells (CSCs), the ChemoID assay provides crucial insights into tumor behavior and treatment resistance. The clinical validation of this assay in phase-3 randomized trials for glioblastoma and platinum-resistant ovarian cancer underscores its potential to significantly improve patient outcomes, particularly in cases where standard treatments have failed. The integration of functional assays like ChemoID into clinical trial designs represents a significant step forward in personalized cancer treatment. By bridging the gap between genomic information and treatment outcomes, this approach has the potential to transform the clinical management of cancer and improve the lives of patients. Clinical efficacy data from ChemoID randomized trials will be presented.
 Citation Format: Pier Paolo Claudio, Candace M Howard. Using cancer stem cells to advance functional medicine in the era of precision oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr IA036.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3478dec1fdaf2f3843250ec5b2474daa9e3a4077" target='_blank'>
              Abstract IA036: Using cancer stem cells to advance functional medicine in the era of precision oncology
              </a>
            </td>
          <td>
            Pier Paolo Claudio, C. Howard
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The metabolic enzyme aldehyde dehydrogenase 1A1 (ALDH1A1), a cancer stem cell marker associated with poor outcomes in breast cancer, has emerged as a promising therapeutic target in TNBC. The aim of this study was to investigate the role of ALDH1A1 in radiation resistance and redox stress in triple negative breast cancer (TNBC). Functional knockouts of ALDH1A1 were generated by the CRISPR/Cas9-mediated deletion of ALDH1A1 in the SUM159 cell line, and three distinct clonal populations were isolated. Genetic targeting was confirmed by Sanger sequencing, and the loss of ALDH1A1 protein expression was validated by Western blotting. Functional assays assessed ALDEFLUOR activity, cell viability, self-renewal capacity, and reactive oxygen species (ROS) levels with or without radiation in both the bulk population and clonal lines. Interestingly, ALDEFLUOR activity was uniformly lost across all clonal lines; however, functional effects of ALDH1A1 loss on redox stress, survival, and radiation sensitivity were observed in only one clonal population. These findings highlight significant variability in the role of ALDH1A1 among clonal populations, reflecting the complexity of tumor heterogeneity. This underscores the importance of accounting for tumor heterogeneity when targeting ALDH1A1, as certain TNBC subpopulations may rely more heavily on ALDH1A1 function. These insights are critical for developing effective ALDH1A1-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39f8fe1789f12e83b81bf503275e5a974f1d83" target='_blank'>
              CRISPR/Cas9 Targeting of Aldehyde Dehydrogenase 1A1 Reveals Heterogeneous Roles in Radiation Response and Redox Stress Across Clonal Lines in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Grace O. Ajayi, Aihui Ma, Shirin R. Modarai, Lynn M. Opdenaker, Jennifer Sims-Mourtada
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a hereditary blood disorder resulting from the production of distorted hemoglobin molecules that cause red blood cells to adopt a sickle or crescent-like shape. This disease affects millions of people, particularly those of African, Mediterranean, Middle Eastern, or South Asian descent. In recent years, however, advancements in the CRISPR-Cas9 gene-editing systems have surged. CRISPR stands for clustered regularly interspaced short palindromic repeats, referring to a specific organization of short, partially repeated DNA sequences in prokaryotic genomes. The CRISPR-Cas9 technique is based on the type II CRISPR system of bacteria and involves the Cas9 nuclease, which is targeted to a particular genome section with the help of single-guide RNA. Initially used for random mutations and small sequence alterations, genome editing methods have advanced to achieve large-scale DNA segment manipulation. The BE and PE-- type CRISPR-Cas9 genome editing variants provide new therapeutic options for genetic disorders, improving patients' prognosis. Curative gene editing using CRISPR-Cas9 technology to correct HBB gene mutations that cause SCD represents a revolutionary therapeutic development. These advances bring new hope to patients with previously untreatable diseases, potentially offering a future where genetic disorders can be addressed at their roots. A major objective of CRISPR technology is to enhance its precision and speed, both critical for effective gene editing. This review focuses on molecular mechanisms of CRISPR-Cas9 technology, CRISPR-- Cas9-based approaches for HBB gene modification, clinical trials, patients with sickle cell disease, and advances in CRISPR technology for sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4625a1647e0225fc85616b697ca8ae26c76a3c30" target='_blank'>
              Transformative CRISPR-Cas9 Technologies: A Review of Molecular Mechanisms, Precision Editing Techniques, and Clinical Progress in Sickle Cell Disease.
              </a>
            </td>
          <td>
            Komal, Prabhjot Kaur, Nidhi Arora, Jyotiram A. Sawale, Amandeep Singh
          </td>
          <td>2025-03-05</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer poses a major global health issue, profoundly affecting both mortality and morbidity rates across the world. A key obstacle in understanding the pathogenesis of colorectal cancer lies in its high inter-patient and spatial heterogeneity, making standard treatments ineffective. Commonly, the study on colorectal cancer relies on bulk RNA sequencing, offering an average gene expression profile for a heterogenous cell population. However, this approach obscures the heterogeneous characteristics of the cancer cells and hinders a comprehensive understanding of the complex interplay among different cell populations. Recently, the advent of single-cell RNA sequencing has been revolutionary, enabling researchers to analyze individual immune cells and overcome the limitations of bulk RNA sequencing. Through single-cell RNA sequencing, researchers have gained insights into the dynamic nature of the immune response to cancer and potential targets for immunotherapy. In this review, we discuss the technical aspects of single-cell RNA sequencing, the application of single-cell RNA sequencing in cancer immunology, and the potential of single-cell RNA in clinical settings. We believe that harnessing singlecell RNA sequencing in cancer research holds immense potential to drive the development of personalized immunotherapies, aiming to improve patient outcomes in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CART) cell therapy is an innovative form of targeted immunotherapy that has revolutionized the treatment of cancer. However, the durable response remains limited. Recent studies have shown that the epigenetic landscape of preinfusion CART cell products can influence response to therapy, and gene editing has been proposed as a solution. However, more work needs to be done to determine the optimal gene editing strategy. Genome-wide CRISPR screens have become popular tools to both investigate mechanisms of resistance and optimize gene editing strategies. Yet their application to primary cells presents many challenges. Here we describe a method to complete a genome-wide CRISPR knockout screen in CART cells from healthy donors. As a proof-of-concept model, we designed this method to investigate the development of exhaustion in CART cells targeting the CD19 antigen. However, we believe that this approach can be used to address a variety of mechanisms of resistance to therapy in different CAR constructs and tumor models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/995da50c8649d8341a6a265386cac02363ea1e3b" target='_blank'>
              Performing an In Vitro Genome-Wide CRISPR Knockout Screen in Chimeric Antigen Receptor T Cells.
              </a>
            </td>
          <td>
            Carli M. Stewart, Claudia Manriquez Roman, S. Kenderian
          </td>
          <td>2025-01-31</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immune checkpoint blockade (ICB) has revolutionized the therapeutic landscape across various cancer types. However, the emergence of resistance to ICB therapy limits its clinical application. Therefore, it is necessary to better understand immune-resistance mechanisms that could be targeted by actionable drugs, and important to identify predictive markers for selecting patients. Here, by analyzing transcriptomic data from patients treated with PD-1 blockade and tumor models refractory to anti-PD-1 therapy, we identified WEE1 as a resistance factor conferring cancer stem cell (CSC)-like properties as well as immune-refractory phenotypes to tumor cells. WEE1 is transcriptionally upregulated by stemness factor NANOG and predominantly localized in the cytoplasm, not the nucleus, following AKT-dependent S642 phosphorylation in immune-refractory tumor cells. Mechanistically, cytoplasmic WEE1 drove AKT hyperactivation via the HSP90A/TCL1A/AKT auto-amplification loop andupregulated the expression of refractory factors such as CYCLIN A for hyperproliferation and MCL-1 for resistance to T cell killing. Of note, CXCL10 was downregulated, resulting in insufficient T cell infiltration. The NANOG/WEE1/AKT axis was also conserved in various human cancers. Importantly, targeting WEE1 with a clinically relevant inhibitor sensitized NANOG+ immune-refractory tumors to ICB, reinvigorating antitumor immunity by disrupting the HSP90A/TCL1A/AKT loop. Thus, our findings demonstrate the oncogenic role of cytoplasmic WEE1 in immune-refractoriness and CSC-like properties of tumor cells through AKT hyperactivation and provide a rationale for combining a WEE1 inhibitor to control anti-PD-1 therapy-refractory tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d171557e9628c9a32e1f797d2ae45ffca94dcd" target='_blank'>
              Cytoplasmic WEE1 promotes resistance to PD-1 blockade through hyperactivation of the HSP90A/TCL1/AKT signaling axis in NANOGhigh tumors.
              </a>
            </td>
          <td>
            Suyeon Kim, Hyo-Jung Lee, Seungho Lee, Jo Eun Chung, Se Jin Oh, , Eunho Cho, Min Kyu Son, Heeju Kwon, Seung-Jong Kim, Chaeleen Lee, Suhwan Chang, Tae Woo Kim
          </td>
          <td>2025-03-11</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c98636c9111085040b447b74deddfb7a8fbd77" target='_blank'>
              Multi-gene panel sequencing reveals the relationship between driver gene mutation and clinical characteristics in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jinmao Li, Xianyun Li, Hao Guo, Shuni Zhou, Hua Ding, Bo Li, Wei Li, Bo Zhang, Yi Zou
          </td>
          <td>2025-03-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The mechanisms of epigenetic regulation emerge as a fundamental determinant in the complex landscape of cancer initiation and advancement. However, the specific impact of epigenetic regulation on cancer progression remains unclear. To explore the relationship between epigenetic regulation and cancer progression, we utilized transcriptomic data from The Cancer Genome Atlas (TCGA) datasets to investigate the association. Methods We obtained transcriptomic data of epigenetic gene dataset from the TCGA database and calculated an epigenetic score using the Least Absolute Shrinkage and Selection Operator (LASSO) Cox model. Additionally, we created a nomogram that integrates the epigenetic score and clinical features, providing a more comprehensive tool for tumor patients prognosis assessment. Results We calculated the epigenetic score based on the expression levels of epigenetic-related genes. The nomogram we developed incorporates the epigenetic score and clinical characteristics. The epigenetic score was positively correlated with the expression of genes related to hallmarkers of cancer, including glycolysis, epithelial-mesenchymal transition (EMT), cell cycle, DNA repair, angiogenesis, and inflammatory response. Furthermore, we performed gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG) analysis to explore the signaling pathways and biological processes in high epigenetic score group. Conclusion The epigenetic scoring system developed in this investigation represents an innovative approach that demonstrates remarkable potential in forecasting survival trajectories across diverse cancer types. These groundbreaking insights not only illuminate the intricate interactions between epigenetic mechanisms and gene expression regulation in oncological contexts, but also indicate that the derived epigenetic metric could potentially emerge as a significant prognostic biomarker for cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b405c467d91505b22b6bd6b604ad22bfbaf352f" target='_blank'>
              Exploring the role of epigenetic regulation in cancer prognosis with epigenetic score
              </a>
            </td>
          <td>
            Ruiguang Zhang, Shimin Jin, Qi Xu, Rongxiao Dai
          </td>
          <td>2025-02-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="

Globally, lung cancer continues to be the primary cause of cancer-related fatalities. Despite
significant advancements in chemotherapy, targeted therapy, and immunotherapy that have improved
treatment effectiveness and increased survival rates, the five-year relative survival rate for
lung cancer patients remains very low. Most cases receive a diagnosis at an advanced stage, which
primarily contributes to this. Drug resistance is still a significant challenge, which is responsible for
90% of cancer-related deaths. Epigenetic alterations, including DNA methylation, RNA modification,
and histone modification, play crucial roles in cancer treatment resistance. Among these, the methyltransferase-
like 3 (METTL3)-induced N6-methyladenosine (m6A) RNA modification has received a
lot of attention because it plays a part in lung cancer tumor growth, carcinogenesis, proliferation, and
resistance to treatment. Researchers have identified or developed several METTL3 inhibitors that
have shown promising therapeutic benefits in both in vivo and in vitro models. This review focuses
on the progress that has been made in the research investigating the role of METTL3 in lung cancer
treatment resistance. It also elucidates the processes that are at play and investigates prospective therapeutic
options that target METTL3 in order to enhance treatment results and overcome drug resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b3a2a7fdf1a317ed911bfc6a066f5d8abc14563" target='_blank'>
              Exploring the Function of METTL3 in Lung Cancer: Understanding Drug Resistance and Exploring Innovative Therapeutic Approaches
              </a>
            </td>
          <td>
            Shiqing Huang, Ye Li, F. Liew
          </td>
          <td>2025-03-12</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that continued expression is required for maintaining their leukemic phenotype and they therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20366f8a8e956d20a8164d2739ca733da77de50e" target='_blank'>
              RUNX1::RUNX1T1 Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. Daniel, Mata Casimiro, Luca van den Brink, Anita T. van Oort, Minoo Ashtiani, C. Zwaan, Anja Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, H. J. Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4f2fa5b1b737b1780de159ebc71c5f0fc213ff" target='_blank'>
              Molecular characterization of colorectal cancer (CRC) using next generation sequencing (NGS) in bridging the gap between research and clinical practice: from biomarker discovery to clinical implementation
              </a>
            </td>
          <td>
            H. Afolabi, S. Salleh, Zaidi Zakaria, Ch'ng Ewe Seng, S. N. M. Nafi, Ahmad Aizat Bin Abdul Aziz, Wan Mohd Nazri Wan Zainon, Ahmad Adebayo Irekola, Yusuf Wada, S. B. Al-Mhanna, R. Elesho
          </td>
          <td>2025-03-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast Cancer (BrCa) exhibits a high phenotypic heterogeneity, leading to the emergence of aggressive clones and the development of drug resistance. Considering the BrCa heterogeneity and that metabolic reprogramming is a cancer hallmark, we selected seven BrCa cell lines with diverse subtypes to provide their comprehensive metabolome characterization: five lines commonly used (SK-Br-3, T-47D, MCF-7, MDA-MB-436, and MDA-MB-231), and two patient-derived xenografts (Hbcx39 and Hbcx9). We characterized their endometabolomes using 1H-NMR spectroscopy. We found distinct metabolite profiles, with certain metabolites being common but differentially accumulated across the selected BrCa cell lines. High levels of glycine, lactate, glutamate, and formate, metabolites known to promote invasion and metastasis, were detected in all BrCa cells. In our experiment setting were identified unique metabolites to specific cell lines: xanthine and 2-oxoglutarate in SK-Br-3, 2-oxobutyrate in T-47D, cystathionine and glucose-1-phosphate in MCF-7, NAD+ in MDA-MB-436, isocitrate in MDA-MB-231, and NADP+ in Hbcx9. The unique and enriched metabolites enabled us to identify the metabolic pathways modulated in a cell-line-specific manner, which may represent potential candidate targets for therapeutic intervention. We believe this study may contribute to the functional characterization of BrCa cells and assist in selecting appropriate cell lines for drug-response studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4c21e05117b322ca21c171873276565e0f755d" target='_blank'>
              Metabolic Profiling of Breast Cancer Cell Lines: Unique and Shared Metabolites
              </a>
            </td>
          <td>
            Mariana Gallo, Elena Ferrari, F. Brugnoli, Anna Terrazzan, Pietro Ancona, Stefano Volinia, V. Bertagnolo, Carlo M. Bergamini, A. Spisni, T. Pertinhez, Nicoletta Bianchi
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="CRISPR/Cas is an adaptive defense mechanism protecting prokaryotes from viruses and other potentially harmful genetic elements. Through an adaptation process, short “spacer” sequences, captured from these elements and incorporated into a CRISPR array, provide target specificity for the immune response. CRISPR arrays and array expansion are also central to many emerging biotechnologies. The rates at which spacers integrate into native arrays within bacterial populations have not been quantified. Here, we measure naïve spacer acquisition rates in Escherichia coli Type I-E CRISPR, identify factors that affect these rates, and model this process fundamental to CRISPR/Cas defense. Prolonged Cas1–Cas2 expression produced fewer new spacers per cell on average than predicted by the model. Subsequent experiments revealed that this was due to a mean fitness reduction linked to array-expanded populations. In addition, the expression of heterologous non-homologous end-joining DNA-repair genes was found to augment spacer acquisition rates, translating to enhanced phage infection defense. Together, these results demonstrate the impact of intracellular factors that modulate spacer acquisition and identify an intrinsic fitness effect associated with array-expanded populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c43c73a0772d31975ee08adb600c71962fad799" target='_blank'>
              Assessing spacer acquisition rates in E. coli type I-E CRISPR arrays
              </a>
            </td>
          <td>
            Luke J. Peach, Haoyun Zhang, Brian P. Weaver, J. Q. Boedicker
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="EGFR tyrosine kinase inhibitors (EGFR-TKIs) have garnered substantial clinical success in treating nonsmall cell lung cancer (NSCLC) harboring epidermal growth factor receptor (EGFR) mutations. However, the inevitable emergence of drug resistance, frequently attributed to activation, mutation, or deletion of multiple signaling pathways, poses a significant challenge. Notably, the loss of PTEN protein expression has emerged as a pivotal mechanism fostering resistance in EGFR mutant lung cancers. Consequently, strategies aimed at upregulating PTEN expression hold great promise for restoring drug sensitivity. Leveraging the versatility, precision, and efficacy of nuclease-deactivated Cas9 (dCas9) as a transcriptional activator, we designed a CRISPR-dCas9 system to stimulate PTEN expression. To further enhance target specificity and drug delivery efficiency, we innovatively harnessed the tumor cell membrane (CCM) as a homologous targeting surface coating for our vector, thereby creating a targeted activation nanoplatform. Comprehensive in vitro and in vivo evaluations demonstrated that the synergistic interplay between gefitinib and the CRISPR-dCas9 system significantly enhanced drug sensitivity. The finding underscores the potential of our approach in addressing the issue of lung cancer resistance, offering a promising avenue for personalized and effective cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f86d44b9e09159a099ee094747765812cf22c69" target='_blank'>
              CRISPR-dCas9-Mediated PTEN Activation via Tumor Cell Membrane-Coated Nanoplatform Enhances Sensitivity to Tyrosine Kinase Inhibitors in Nonsmall Cell Lung Cancer.
              </a>
            </td>
          <td>
            Kaiyong Yang, Chunli Zhang, Zeyu Wang, Qiqing Huang, Jing Qian, Gaoyu Shi, Wenwen Sun, Jinqiu Wang, Yu Ji, Zhaorui Sun, Yanni Song, Xin Han
          </td>
          <td>2025-02-20</td>
          <td>ACS applied materials & interfaces</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Spatial organization of the genome is fundamental for ensuring accurate gene expression. This process depends on the communication between gene promoters and distal cis-regulatory elements (CREs), which together make up 8% of the human genome and are supported by the chromatin structure. It is estimated that over 90% of disease-associated variants are located in the non-coding region of the genome and may affect CRE. For the cystic fibrosis transmembrane conductance regulator (CFTR) gene, a complete understanding of tissue-specific CFTR expression and regulation is missing, in particular in the pancreas. Mechanistic insights into tissue-specific expression may provide clarity on the clinical heterogeneity observed in Cystic Fibrosis and CFTR-related disorders. Methods To understand the role of 3D chromatin architecture in establishing tissue-specific expression of the CFTR gene, we mapped chromatin interactions via circular chromosome conformation capture (4C) and epigenomic regulation through H3K27ac and DNase Hypersensitive site I (DHS) in Capan-1 pancreatic cells. Candidate regulatory regions are validated by luciferase reporter assay and CRISPR-knock out. Results We identified active regulatory regions not only around the CFTR gene but also outside the topologically associating domain (TAD). By performing functional assays, we validated our targets and revealed a cooperative effect of the −44 kb, −35 kb, +15.6 kb and 37.7 kb regions, which share common predicted transcription factor (TF) motifs. Comparative 3D genomic analysis and functional assays using the Caco-2 intestinal cell line revealed the presence of tissue-specific CREs. Conclusion By studying the chromatin architecture of the CFTR locus in Capan-1 cells, we demonstrated the involvement of multiple CREs upstream and downstream of the CFTR gene. We also extend our analysis to compare intestinal and pancreatic cells and provide information on the tissue-specificity of CRE. These findings highlight the importance of expanding the search for causative variants beyond the gene coding sequence but also by considering the tissue-specific 3D genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9279897acf411b562291c89812e3d397609427" target='_blank'>
              New insights into the cis-regulation of the CFTR gene in pancreatic cells
              </a>
            </td>
          <td>
            Clara Blotas, A. Le Nabec, M. Collobert, Mattijs Bulcaen, Marianne S. Carlon, Claude Férec, S. Moisan
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Chromosome instability is highly prevalent in cancer and drives large scale chromosomal imbalances, known as aneuploidies. However, how aneuploidy contributes to tumorigenesis remains difficult to study due to the vast numbers of genes affected. To address these limitations, we have developed a CRISPR-Knock Out and Activation Linked Assay (CRISPR-KOALA), which enables systematic high-throughput bidirectional genetic screens in immune-competent mouse models of cancer. Using CRISPR-KOALA we screened the mouse orthologs of all 3,752 genes residing on the ten most frequently altered human chromosome arms in basal-like breast cancer (BLBC), which to date is the largest bidirectional screen performed in vivo. Our screen identified 90 cancer driver genes, the vast majority of which have hitherto unknown functions in cancer. These genes drive distinct signalling pathways, reflecting the high degree of BLBC heterogeneity. Individual manipulation of the identified cancer driver genes completely overcomes the need for copy number alterations (CNAs) in p53-mutant BLBC mouse models. Mechanistically, we uncover PLGRKT as a potent oncogene and show that its tumor-promoting activity is associated with the creation of highly stress-resistant mitochondria that promote tumor cell survival, rather than through its canonical role of regulating the extracellular matrix. Overall, our work reveals that arm-level CNAs can function to select specific driver genes to promote heterogenous biological processes.
 Citation Format: Khalid N Al-Zahrani, Ellen R Langille, Andreea Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia Chiu, Alexander Bahcheli2, Ricky Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. Idil Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean Egan, Jüri Reimand, Jeffrey Wrana, Daniel Schramek. Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd956491f5700a578d8ac40b6d89d08cd7f2d58" target='_blank'>
              CRISPR/Cas12a-mediated gene silencing across diverse functional genes demonstrates single gene-specific spacer efficacy in Mycobacterium smegmatis
              </a>
            </td>
          <td>
            Megha Sodani, C. S. Misra, Savita Kulkarni, D. Rath
          </td>
          <td>2025-02-28</td>
          <td>Journal of Biological Engineering</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background/Objectives: Aberrant hypermethylation in the promoter regions of tumor suppressor genes facilitates the pathogenesis and progression of cancer. Therefore, inhibitors targeting DNA methyltransferase (DNMT) have been tested in clinical studies. However, the current monotherapy of DNMT inhibitors shows limited efficacy. Furthermore, the mechanism of action of DNMT inhibitors is DNA replication-dependent. To address these limitations, we developed a novel core–shell-type “epigenetics control (EpC) nanocarrier” that encapsulated decitabine (5-aza-dC) in the PLGA core nanoparticle and hybridized TET1 gene-encoding pDNA on the lipid shell surface. This study aimed to evaluate whether the dual delivery of DNMT inhibitors and pDNA of TET1 could synergistically enhance tumor suppressor gene expression and induce cell cycle arrest and/or apoptosis in cancer cells. Herein, we demonstrate the potential of the EpC carrier in HCT116 human colon cancer cells to upregulate tumor suppressor gene expression and rapidly achieve cell cycle arrest. Methods: PLGA core nanoparticles were prepared by the W/O/W double emulsion method. The formation of core–shell nanoparticles and complexation with pDNA were investigated and optimized by dynamic light scattering, zeta potential measurement, and agarose gel electrophoresis. The cellular uptake and transfection efficiency were measured by confocal laser scanning microscopy and a luciferase assay, respectively. The expression of p53 protein was detected by Western blotting. The anti-tumor effects of the EpC nanocarrier were evaluated by cell cycle analysis and an apoptosis assay. Results: The EpC nanocarrier delivered the DNMT inhibitor and TET gene-encoding pDNA into HCT116 cells. It promoted the expression of the tumor suppressor protein p53 and induced rapid cell cycle arrest in the G2/M phase in HCT116 cells. Conclusions: Our findings suggest that the dual-targeting of DNMT and TET enzymes effectively repairs aberrant DNA methylation and induces growth arrest in cancer cells, and the dual-targeting strategy may contribute to the advancement of epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310794284715bd67edbc662582ab25897ba7a464" target='_blank'>
              Novel Epigenetics Control (EpC) Nanocarrier for Cancer Therapy Through Dual-Targeting Approach to DNA Methyltransferase and Ten-Eleven Translocation Enzymes
              </a>
            </td>
          <td>
            Risa Mitsuhashi, Kiyoshi Sato, Hiroyoshi Kawakami
          </td>
          <td>2025-02-11</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. We aim to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumours in various organs. Lineage tracing and clonal heterogeneity were evaluated using the state-of-the-art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. To identify melanoma clones across different metastatic sites, specifically subcutaneous, lungs and brain, C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7PV1 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor the tumour growth of mice injected intravenously and intracranially. Subcutaneous tumours were measured by calliper. Mice were euthanized before ethical endpoints. Tumors were harvested and single cell analysis as well as DNA sequencing was performed to identify barcodes expressed by the tumor cells across different tumor-sites. Barcode analysis was conducted both before cell implantation and at tumor harvest. Clonality exhibited significant variability between sites. Single-cell RNA sequencing will unveil states associated with dominant clones present at different metastatic sites. This information will provide a new understanding of melanoma transcriptional phenotype behaviours and biological characteristics at different metastatic sites and in response to therapy.
 Citation Format: Veronica Aedo-Lopez. Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction As the lifespan of companion animals extends, the incidence of tumor also increases. Among these tumors, lymphoma is reported as the most prevalent hematopoietic tumor with a 80-90% prevalence rate. Ongoing research spans multiple domains, aiming to uncover novel therapeutic targets, including small molecular weight inhibitors, antibody treatments, and subtype-specific selective agents. Methods Transcriptional profiling was performed on canine lymphoma samples to identify genes and functional pathways associated with pathogenesis, treatment response, and prognosis. Additionally, genes with potential relevance to the clinical characteristics of T-cell lymphoma (TCL), which is characterized by a low treatment response and poor prognosis, were identified through a comparative analysis of different lymphoma subtypes. Results Within the canine lymphoma group, HERC5 showed consistent upregulation, a gene similarly implicated in human acute myeloid leukemia but previously no reports exist. Additionally, noteworthy genes, including IKZF2, CCL4, SAA1, and CD40, exhibited differential expression in the TCL group compared to the B-cell lymphoma (BCL) group. Discussion The upregulation of HERC5 may impact on canine lymphoma pathogenicity. Furthermore, the upregulation of IKZF2, CCL4, and SAA1, along with the downregulation of CD40, may contribute to adverse clinical characteristics of TCL in dogs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9e40523f791db37ac12e2ba1d1230c46fad0b79" target='_blank'>
              Sequential transcriptome profiling: comparative analysis of normal and canine lymphoma preceding detailed T-cell and B-cell subtype comparison
              </a>
            </td>
          <td>
            Yeji Kim, Jihyun Kim, Yunji Song, Keunhwan Jang, Se Eun Kim, Ha-Jung Kim
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Veterinary Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="As a significant cause of global mortality, the cancer has also economic impacts. In the era of cancer therapy, mitigating side effects and costs and overcoming drug resistance is crucial. Microbial species can grow inside the tumor microenvironment and inhibit cancer growth through direct killing of tumor cells and immunoregulatory effects. Although microbiota or their products have demonstrated anticancer effects, the possibility of acting as pathogens and exerting side effects in certain individuals is a risk. Hence, several genetically modified/engineered bacteria (GEB) have been developed to this aim with ability of diagnosing and selective targeting and destruction of cancers. Additionally, GEB are expected to be considerably more efficient, safer, more permeable, less costly, and less invasive theranostic approaches compared to wild types. Potential GEB strains such as Escherichia coli (Nissle 1917, and MG1655), Salmonella typhimurium YB1 SL7207 (aroA gene deletion), VNP20009 (∆msbB/∆purI) and ΔppGpp (PTet and PBAD), and Listeria monocytogenes Lmat-LLO have been developed to combat cancer cells. When used in tandem with conventional treatments, GEB substantially improve the efficacy of anticancer therapy outcomes. In addition, public acceptance, optimal timing (s), duration (s), dose (s), and strains identification, interactions with other strains and the host cells, efficacy, safety and quality, and potential risks and ethical dilemmas include major challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba6f5c96035da6ef412a12d773af66d8566a484" target='_blank'>
              Genetically Engineered Bacteria as a Promising Therapeutic Strategy Against Cancer: A Comprehensive Review.
              </a>
            </td>
          <td>
            Z. Zahedifard, Shirin Mahmoodi, A. Ghasemian
          </td>
          <td>2025-02-21</td>
          <td>Biotechnology and applied biochemistry</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Copy number variations of the human CNTN6 gene, resulting from megabase-scale microdeletions or microduplications in the 3p26.3 region, are frequently implicated in neurodevelopmental disorders such as intellectual disability and developmental delay. However, duplication of the full-length human CNTN6 gene presents with variable penetrance, resulting in phenotypes that range from neurodevelopmental disorders to no visible pathologies, even within the same family. Previously, we obtained a set of induced pluripotent stem cell lines derived from a patient with a CNTN6 gene duplication and from two healthy donors. Our findings demonstrated that CNTN6 expression in neurons carrying the duplication was significantly reduced. Additionally, the expression from the CNTN6 duplicated allele was markedly lower compared to the wild-type allele. Here, we first introduce a system for correcting megabase-scale duplications in induced pluripotent stem cells and secondly analyze the impact of this correction on CNTN6 gene expression. We showed that the deletion of one copy of the CNTN6 duplication did not affect the expression levels of the remaining allele in the neuronal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c9c747915f6b48c8d45b21bda1f171cd233c0f" target='_blank'>
              Targeted correction of megabase-scale CNTN6 duplication in induced pluripotent stem cells and impacts on gene expression
              </a>
            </td>
          <td>
            Maria Gridina, Polina Orlova, Oleg Serov
          </td>
          <td>2025-01-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Variants of unknown significance represent one of the biggest challenges for genomics-based precision oncology. High throughput functional genomics is thus essential to characterize the vast number of cancer mutations and tailor the most promising treatment accordingly. Aberrantly activated fibroblast growth factor receptors (FGFRs) frequently drive tumorigenesis with a variety of genetic aberrations across a wide range of tumor entities. Approved selective inhibitors (FGFRis) are available. However, it remains largely unknown which of the many different point mutations affecting FGFR1, FGFR2, FGFR3 or FGFR4 are druggable, i.e. activating FGFR signaling while not mediating FGFRi resistance thereby substantially limiting the therapeutic potential of approved FGFRis. Here, we implemented a saturation mutational scanning platform to screen all 11520 possible point mutations covering the kinase domains of FGFR1-4. In pooled positive selection screens, we identified 474 activating mutations and 738 resistance-mediating mutations to the FGFR inhibitors pemigatinib and/or futibatinib - with many clustered in particular structural elements. Combining these comprehensive datasets revealed 301 druggable FGFR point mutations analogous to a strong PS3/BS3 evidence level. Notably, mutations in the same codon or the corresponding mutations in different FGFRs could strongly differ in their impact, thus underlining the necessity for a saturation approach to study each mutation individually. The screens also identified loss-of-function mutations and FGFR2-specific activating nonsense mutations. Importantly, the functional screens identified 97% of acquired resistance mutations in a clinical trial. In summary, we provide a comprehensive and clinically highly relevant catalog of every single druggable point mutations in the FGFR kinase domains which is readily available for clinical decision support to select the optimal cancer therapy and to push the boundaries of approved inhibitor applications.
 Citation Format: Carla Tangermann, Avantika Ghosh, Martin Ziegler, Francesco Facchinetti, Jannis Stappenbeck, Marisa Riester, Luise Carmina Viardot, Tobias Zundel, Luc Friboulet, Antoine Hollebecque, José Naveja Romero, Angela Wanninger, Maria Elena Hess, Tilman Brummer, Melanie Börries, Sonja Loges, Yohann Loriot, Lena Illert, Sven Diedrichs. Saturation mutational scanning uncovers druggability of all FGFR kinase point mutations [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1767b38084076cdfaf10fe2b85fb296d8eecabbf" target='_blank'>
              Abstract B021: Saturation mutational scanning uncovers druggability of all FGFR kinase point mutations
              </a>
            </td>
          <td>
            Carla Tangermann, Avantika Ghosh, Martin Ziegler, Francesco Facchinetti, Jannis Stappenbeck, M. Riester, Luise Carmina Viardot, Tobias Zundel, L. Friboulet, A. Hollebecque, José Naveja Romero, Angela Wanninger, Maria Elena Hess, Tilman Brummer, Melanie Börries, Sonja Loges, Yohann Loriot, Lena Illert, Sven Diedrichs
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) accumulate somatic mutations over time, some conferring a fitness advantage that can lead to clonal hematopoiesis (CH). Mutations in DNMT3A, particularly at hotspot R882, are the most prevalent in CH and carry an increased risk of acute myeloid leukemia (AML). Although DNMT3A R882 mutations are linked to global DNA hypomethylation, the mechanisms underlying their selective advantage remain unclear. Here, we show that Dnmt3a-R882H mutant HSCs exhibit resilience under inflammatory and genotoxic stress. During IL-1β-induced emergency granulopoiesis, Dnmt3a R882H/+ HSCs uncouple increased proliferation from stem cell exhaustion. In contrast, wild-type HSCs rapidly progress to terminal differentiation. We link this phenotype to a delayed activation of the p53-p21-DREAM axis, that allows mutant HSCs to avoid attrition, despite increased replication stress. Similarly, mutant HSCs exhibit delayed Trp53 activation following irradiation, but eventually recover a physiological Trp53 response. Analysis of patient data reveals shared phenotypic features between DNMT3A and monoallelic TP53 mutations in CH and myeloid neoplasms, highlighting potential functional similarities. Collectively, these findings suggest that the expansion of DNMT3A-mutant clones is affected by impaired TP53 signaling, which confers resilience against stressors. Therapeutic strategies targeting inflammatory pathways or the p53-p21-DREAM axis may reduce DNMT3A-CH expansion and/or progression and its associated risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e56fa3aad088d2f09d1437903fb4b00ec728238c" target='_blank'>
              Delayed Trp53 activation protects Dnmt3a-mutant hematopoietic stem cells from inflammatory attrition
              </a>
            </td>
          <td>
            Ludovica Marando, G. Giotopoulos, Pedro Madrigal, Dhoyazan M A Azazi, Ryan Asby, M. E. Calleja-Cervantes, M. Gu, David Lara-Astiaso, Nicola K Wilson, R. Hannah, Jaana Bagri, Emily F. Calderbank, E. Laurenti, E. Bernard, Sarah J Horton, Berthold Göttgens, E. Papaemmanuil, Shuchi Agrawal-Singh, M. Gozdecka, G. Vassiliou, B. Huntly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions.
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b690f44fb31641c4b057abb18dc7da4f6e88c306" target='_blank'>
              The widely used Ucp1-Cre transgene elicits complex developmental and metabolic phenotypes
              </a>
            </td>
          <td>
            M. Halurkar, Oto Inoue, Archana Singh, Rajib Mukherjee, Meghana Ginugu, Christopher Ahn, C. L. Bonatto Paese, Molly Duszynski, Samantha A Brugmann, Hee-Woong Lim, Joan Sanchez-Gurmaches
          </td>
          <td>2025-01-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Precision medicine approaches to cancer treatment aim to exploit genomic alterations that are specific to individual patients to tailor therapeutic strategies. Yet, some targetable genes and pathways are essential for tumor cell viability even in the absence of direct genomic alterations. In underrepresented populations, the mutational landscape and determinants of response to existing therapies are poorly characterized because of limited inclusion in clinical trials and studies. One way to reveal tumor essential genes is with genetic screens. Most screens are conducted on cell lines that bear little resemblance to patient tumors, after years of culture under nonphysiologic conditions. To address this problem, we aimed to develop a CRISPR screening pipeline in three-dimensionally grown patient-derived tumor organoid (PDTO) models. A breast cancer PDTO biobank that focused on underrepresented populations, including West African patients, was established and used to conduct a negative-selection kinome-focused CRISPR screen to identify kinases essential for organoid growth and potential targets for combination therapy with EGFR or MEK inhibitors. The screen identified several previously unidentified kinase targets, and the combination of FGFR1 and EGFR inhibitors synergized to block organoid proliferation. Together, these data demonstrate the feasibility of CRISPR-based genetic screens in patient-derived tumor models, including PDTOs from underrepresented patients with cancer, and identify targets for cancer therapy. Significance: Generation of a breast cancer patient-derived tumor organoid biobank focused on underrepresented populations enabled kinome-focused CRISPR screening that identified essential kinases and potential targets for combination therapy with EGFR or MEK inhibitors. See related commentary by Trembath and Spanheimer, p. 407.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde16e03b04b6ab8e7612d1796bad37a03db0723" target='_blank'>
              Kinome-Focused CRISPR-Cas9 Screens in African Ancestry Patient-Derived Breast Cancer Organoids Identify Essential Kinases and Synergy of EGFR and FGFR1 Inhibition.
              </a>
            </td>
          <td>
            Florencia P. Madorsky Rowdo, Rachel Martini, Sarah Ackermann, Colin P Tang, Marvel Tranquille, Adriana Irizarry, Ilkay Us, Omar Alawa, Jenna E Moyer, M. Sigouros, John Nguyen, Majd Al Assaad, Esther Cheng, Paula S Ginter, J. Manohar, Brian Stonaker, Richard Boateng, J. Oppong, E. Adjei, B. Awuah, I. Kyei, F. Aitpillah, M. Adinku, K. Ankomah, E. Osei-Bonsu, K. Gyan, Syed A Hoda, Lisa A Newman, Juan Miguel Mosquera, A. Sboner, O. Elemento, Lukas E. Dow, Melissa B. Davis, M. L. Martin
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="DNA damage is a central driver of the aging process. We previously found that KIF2C, known to play a role in DNA repair, is repressed in aged cells. Here, we investigated if increased KIF2C activity counteracts DNA damage and its effects on aging phenotypes. We show that a small-molecule agonist of KIF2C enhances DNA repair in two distinct genetic disorders exhibiting DNA damage and accelerated aging, the Hutchinson-Gilford progeria (HGPS) and Down (DS) syndromes. Mechanistically, the KIF2C agonist improves the repair of DNA double-strand breaks by inducing nuclear envelope invaginations poked by cytoplasmic microtubules, which translated into amended epigenetic and transcriptional signatures of HGPS and DS. Moreover, subcutaneous administration of the KIF2C agonist in progeria mice mitigated aging phenotypes, extending their healthspan. Our study discloses a unique geroprotective pharmacological approach targeting DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bae82e2a8f4a414d538a8210faf20fea3eff9d" target='_blank'>
              Molecular and pharmacological heterogeneity of ETV6::RUNX1 acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenhua Li, Huanbin Zhao, Wenjian Yang, Maud Maillard, S. Yoshimura, Yu-Chih Hsiao, Xingyu Huang, Yoshihiro Gocho, L. Rowland, Anthony Brown, Landon Choi, K R Crews, C. Mullighan, Samuel Brady, Cheng Cheng, Ti-Cheng Chang, Gang Wu, M. Loh, A. E. J. Yeoh, Federico Antillon-Klussmann, S. Jeha, H. Inaba, Jiyang Yu, Ching-Hon Pui, Seth E. Karol, William E Evans, Jun J Yang
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancer is the second leading cause of death among humans worldwide. Despite remarkable improvements in cancer therapies, drug resistance remains a significant challenge. The tumor microenvironment (TME) is intimately associated with therapeutic resistance. Tumor-associated neutrophils (TANs) are a crucial component of the TME, which, along with other immune cells, play a role in tumorigenesis, development and metastasis. In the current review, the roles of TANs in the TME, as well as the mechanisms of neutrophil-mediated resistance to cancer therapy, including immunotherapy, chemotherapy, radiotherapy and targeted therapy, were summarized. Furthermore, strategies for neutrophil therapy were discussed and TANs were explored as potential targets for cancer treatment. In conclusion, the need to explore the precise roles, recruitment pathways and mechanisms of action of TANs was highlighted for the purpose of developing therapies that precisely target TANs and reverse drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f741b0d8e7223ab3e910617355fcfe705132fe" target='_blank'>
              Tumor-associated neutrophils: Critical regulators in cancer progression and therapeutic resistance (Review)
              </a>
            </td>
          <td>
            Rui Hou, Xi Wu, Cenzhu Wang, Hanfang Fan, Yuhan Zhang, Hanchi Wu, Huiyu Wang, Junli Ding, Huning Jiang, Junying Xu
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Currently, mankind is fiercely struggling with cancer. Recently, we have been winning the battle against cancer through precision medicine and accompanying diagnostic methods, and we are raising many hopes with blockbuster drugs. It would be even better if we could read the cancer nucleotide sequence, identify them in advance, and suggest treatments simultaneously. However, this may be an impossible dream because it takes a lot of time and effort to diagnose and ensure all the long gene sequences of cancer at once. Thus, victory will be even closer if a rapid and accurate diagnosis of the cancer-specific gene biomarkers that will soon be imprinted can be made. With the advent of nanotechnology, a new short cancer diagnostic toolkit has been proposed to achieve the goal. This review presents a small diagnostic device that detects genetic fragments (simply 'Gizmo') of certain cancers. The development of numerous diagnostic methods has focused on (1) directly detecting pre-selectively targeted genes using novel diagnostic systems, and (2) indirectly detecting substantial improvements in diagnostic sensitivity only through detection signal amplification without existing gene amplification steps. Our fight against cancer is not a dream, but the result of success, and it is assumed that victory will accelerate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eccae0a3a741529fc24b2b02ce5646b06a22df12" target='_blank'>
              Engineering Gizmos for Short Cancer Genetic Fragments Discrimination.
              </a>
            </td>
          <td>
            Jisoo Yuk, Jeonghun Kim, Sunghwan Jung, Soong Ho Um
          </td>
          <td>2025-02-05</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 CRISPR/Cas9-mediated homology-directed repair (HDR) allows precise gene editing, but its efficiency remains low for certain cell types, such as human induced pluripotent stem cells (hiPSCs). In this study, we aimed to introduce the CTNNA1: c.2023C>T (p.Q675*) genetic alteration, which is associated with Hereditary Diffuse Gastric Cancer, into hiPSCs using CRISPR/Cas9. We designed a single-guide RNA (sgRNA) targeting the alteration site and a single-stranded oligonucleotide donor DNA template for HDR-based repair. Herein, we report the successful introduction of the CTNNA1: c.2023C>T homozygous alteration in one hiPSC line, which resulted in severe phenotypic changes, including impaired colony formation and cell proliferation. Additionally, we established a straightforward protocol to assess hiPSCs karyotype integrity, ensuring the chromosomal stability required for the gene-editing process. This protocol involves routine G-banding analysis that is required for regular quality controls during handling of hiPSCs. This study demonstrates an efficient approach to precisely edit hiPSCs by CRISPR/Cas9 and highlights the essential role of CTNNA1 expression in maintaining hiPSC viability. Our methodology provides a valuable framework for modeling disease-associated alterations in human-derived cellular models that can be reproduced for other genes and other types of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b6de2d94791247728d12901bea9b29daa88c32" target='_blank'>
              A protocol for karyotyping and genetic editing of induced pluripotent stem cells with homology-directed repair mediated CRISPR/Cas9
              </a>
            </td>
          <td>
            Silvana Lobo, Rita Barbosa-Matos, Sofia Dória, Ana Maria Pedro, Ana Brito, Daniel Ferreira, Carla Oliveira
          </td>
          <td>2025-02-28</td>
          <td>Biology Methods and Protocols</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Monosomy 7 and deletion 7q are common chromosomal abnormalities in myeloid malignancies, and they are associated with a poor prognosis. The mechanism underlying their acquisition remains elusive. We identified a cohort of 24 patients exhibiting clones with different chromosome 7 abnormalities, such as deletion 7q, unstable derivatives (ring chromosomes or ‘naked’ centromeres), and monosomy 7. We designated this group as having cytogenetic clonal evolution of chromosome 7 abnormalities (CCE7). In some cases, CCE7 correlated with disease progression, suggesting that deletions or other derivatives involving the q-arm of chromosome 7 may arise early in the disease course. These abnormalities may be transient but can potentially evolve into monosomy 7. Within the CCE7 group, telomere loss or shortening may contribute to chromosomal instability and the emergence of unstable derivatives, as the chromosome 7 derivatives displayed loss or rearrangement of subtelomeric regions. Moreover, we identified variants in genes implicated in telomere biology disorders and observed specific genetic mutation profiles associated with different chromosome 7 abnormalities. These findings shed light on a potential mechanism leading to monosomy 7 through the evolution of chromosome 7q abnormalities. Identifying patients at risk of developing monosomy 7, based on the presence of unstable derivatives with telomere loss or a specific mutation profile, could potentially enhance patient management and guide the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd97cfdef751d5eb8d87f19388c6ddb34eef62a2" target='_blank'>
              Emergence and Cytogenetic Clonal Evolution of Chromosome 7 Abnormalities in Myeloid Malignancies: Investigating the Role of Telomere Dysfunction
              </a>
            </td>
          <td>
            Carmen Baldazzi, L. Bandini, V. Robustelli, A. Patuelli, C. Venturi, Alessandra Grassi, G. Marzocchi, Angela Ielpo, V. Solli, M. Bochicchio, Stefania Paolini, Chiara Sartor, F. Zingarelli, Antonio Curti, E. Ottaviani, N. Testoni
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Next-generation sequencing has revealed the disruptive reality that advanced/metastatic cancers have complex and individually distinct genomic landscapes, necessitating a rethinking of treatment strategies and clinical trial designs. Indeed, the molecular reclassification of cancer suggests that it is the molecular underpinnings of the disease, rather than the tissue of origin, that mostly drives outcomes. Consequently, oncology clinical trials have evolved from standard phase 1, 2, and 3 tissue-specific studies; to tissue-specific, biomarker-driven trials; to tissue-agnostic trials untethered from histology (all drug-centered designs); and, ultimately, to patient-centered, N-of-1 precision medicine studies in which each patient receives a personalized, biomarker-matched therapy/combination of drugs. Innovative technologies beyond genomics, including those that address transcriptomics, immunomics, proteomics, functional impact, epigenetic changes, and metabolomics, are enabling further refinement and customization of therapy. Decentralized studies have the potential to improve access to trials and precision medicine approaches for underserved minorities. Evaluation of real-world data, assessment of patient-reported outcomes, use of registry protocols, interrogation of exceptional responders, and exploitation of synthetic arms have all contributed to personalized therapeutic approaches. With greater than 1 × 1012 potential patterns of genomic alterations and greater than 4.5 million possible three-drug combinations, the deployment of artificial intelligence/machine learning may be necessary for the optimization of individual therapy and, in the near future, also may permit the discovery of new treatments in real time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed31a3c6403c1b0469b4430cade90e86c2a9bb22" target='_blank'>
              Novel clinical trial designs emerging from the molecular reclassification of cancer.
              </a>
            </td>
          <td>
            M. Nikanjam, S. Kato, Teresa Allen, J. Sicklick, R. Kurzrock
          </td>
          <td>2025-01-22</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>2</td>
          <td>48</td>
        </tr>

        <tr id="The PPM1D gene and its protein product (serine-threonine protein phosphatase, PPM1D or Wip1) are involved in regulation of cell's DNA damage response, cell cycle control, and repair. Amplification, overexpression, or mutations of the PPM1D gene have a significant impact on cell responses to stress factors and genetic instability as well as impairments of processes of double-strand break repair, nucleotide excision repair, base excision repair, cell cycle, and apoptosis. PPM1D dephosphorylates and thus inactivates p53, proteins that respond to DNA strand integrity damage, cell cycle checkpoint proteins, and apoptotic proteins. This contributes to tumor development, growth, and maintenance of the tumor phenotype. In this review we consider data on the role of the PPM1D gene in the formation and maintenance of various oncological processes, including tumors of the mammary glands, ovaries, prostate gland, esophagus, stomach, intestines, liver and pancreas, hemoblastoses, and others.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54d1904727e65e93a89095c3025ebc1f384c120" target='_blank'>
              A genome-wide atlas of human cell morphology
              </a>
            </td>
          <td>
            Meraj Ramezani, Erin Weisbart, J. Bauman, Avtar Singh, John Yong, Maria Lozada, Gregory P. Way, Sanam L. Kavari, Celeste Diaz, Eddy Leardini, Gunjan Jetley, Jenlu Pagnotta, Marzieh Haghighi, Thiago M. Batista, Joaquín Pérez-Schindler, M. Claussnitzer, Shantanu Singh, B. Cimini, P. Blainey, A. Carpenter, Calvin H Jan, James T Neal
          </td>
          <td>2025-01-27</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Significance Estrogen receptor (ER)-positive breast cancer often develops resistance to endocrine therapies like tamoxifen. Understanding the molecular mechanisms underlying both carcinogenesis and drug resistance is critical for improving therapeutic strategies. This study identifies circFOXK2 as a key regulator of CCND1 in ER-positive breast cancer. By stabilizing CCND1 mRNA through RNA–RNA pairing and recruitment of ELAVL1/HuR, circFOXK2 enhances the CCND1–CDK4/6–p-RB–E2F axis, promoting cell proliferation and therapy resistance. Genetic silencing and pharmacological inhibition of circFOXK2 suppress tumor growth and sensitize or restore tamoxifen responsiveness in ER-positive breast cancer. These findings highlight circFOXK2 as a potential therapeutic target for overcoming resistance and improving outcomes in ER-positive breast cancer patients, offering avenues for clinical intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a572f11006892aa8b4e142fead4939f8a023665b" target='_blank'>
              A circRNA–mRNA pairing mechanism regulates tumor growth and endocrine therapy resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Jia Yi, Jiao Du, Xue Chen, Rui-chao Nie, Guo-Sheng Hu, Lei Wang, Yue-ying Zhang, Shang Chen, Xiao-sha Wen, Di-xian Luo, Hua He, Wen Liu
          </td>
          <td>2025-02-18</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 genome-editing technology has revolutionized human stem cell (HSC) research, offering novel therapeutic and diagnostic applications. HSCs play a crucial role in regenerative medicine and genetic therapies due to their ability to self-renew and differentiate into various blood cell lineages. The precise genome-editing capability of CRISPR/Cas9 allows for targeted gene modifications, enabling the correction of inherited disorders, disease modeling and the discovery of novel biomarkers. Despite significant advancements, challenges such as off-target effects, delivery efficiency, and ethical concerns persist, requiring further research and optimization.
Objective: This systematic review evaluates the therapeutic and diagnostic potential of CRISPR/Cas9 genome editing in HSCs, focusing on its efficacy in gene correction for hematologic disorders, disease modeling and biomarker discovery.
Methods: A systematic review was conducted following PRISMA guidelines, analyzing studies published between 2015 and 2024. Literature searches were performed in PubMed, Web of Science, and Scopus using MeSH-aligned keywords. The inclusion criteria encompassed peer-reviewed studies utilizing CRISPR/Cas9 for gene modification in HSCs for therapeutic and diagnostic applications. Exclusion criteria included studies that lacked experimental validation, did not focus on HSCs, or were non-English publications. Out of 85 initially retrieved studies, 40 met the inclusion criteria, and 15 were selected for final synthesis.
Results: CRISPR/Cas9 gene-editing strategies in HSCs were categorized as gene knockout (53%), gene activation (40%), and dual knockout/activation (7%). Hematological disorders, including sickle cell anemia and beta-thalassemia, accounted for 35% of studies, demonstrating up to 90% correction in β-globin mutations. Neurodegenerative diseases constituted 20% of studies, where knockout of amyloid precursor protein (APP) in Alzheimer’s models resulted in a 60% reduction in plaque accumulation. Muscular dystrophy studies (10%) showed 75% improvement in dystrophin expression through gene activation. High-throughput CRISPR screening was employed in 15% of studies for biomarker identification. Despite promising outcomes, off-target mutations were observed in 28% of studies, and viral vector-based delivery methods were used in 65%, raising safety concerns.
Conclusion: CRISPR/Cas9 genome editing in HSCs presents a ground-breaking approach for treating genetic disorders and enhancing precision medicine. Its potential to correct disease-causing mutations, model complex disorders, and identify novel therapeutic targets is substantial. However, challenges in delivery methods, long-term safety, and ethical considerations remain barriers to clinical translation. Future research should focus on optimizing high-fidelity Cas9 variants, improving non-viral delivery methods, and addressing ethical concerns to ensure the safe and effective application of CRISPR/Cas9 in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20400a815c119b74c31ff8ba0f49e856aea421b" target='_blank'>
              EVALUATION OF CRISPR/CAS9 GENOME-EDITING SYSTEM IN HUMAN STEM CELLS HSCS: THERAPEUTICS AND DIAGNOSTICS PROSPECTS
              </a>
            </td>
          <td>
            Ehsan ul Haq, Haseeb Khaliq, Ayesha Muddasser, Misha Aslam, Farwa Zafar, Naveera Mazhar, Amna Naheed Khan
          </td>
          <td>2025-01-20</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) frequently harbors genomic alterations in DNA Damage Response (DDR) genes, with up to 25% of metastatic cases exhibiting DDR mutations, primarily in the Homologous Recombination Repair (HRR) and Mismatch Repair (MMR) pathways. Additionally, Programmed Death-Ligand 1 (PDL-1), an immune checkpoint protein, plays a pivotal role in immune evasion, particularly in Metastatic Castration-Resistant Prostate Cancer (mCRPC). Despite their critical impact, the connection between DDR mutations and PDL-1 expression in mPCa remains underexplored. This study aims to uncover this vital relationship by assessing DDR protein expression and PDL-1 in a cohort of mPCa cases. We analyzed 50 mPCa patients using FFPE tissue blocks retrieved from archives to evaluate DDR proteins (BRCA1, BRCA2, ATM, CDK12, MLH1, and MSH2) and PDL-1 through a standardized immunohistochemistry (IHC) protocol. Protein expression was quantified based on staining intensity and the percentage of positive cells, while PDL-1 expression was assessed using the CPS score (≥1%). Statistical analysis included the Student’s t-test to evaluate associations and Pearson’s chi-square test to explore relationships between protein expression and clinicopathological parameters. Statistical significance was set at P < 0.05 with a 95% confidence interval. BRCA2 expression was significantly elevated compared to BRCA1 (P < 0.01) across all Gleason grades. ATM expression showed a moderate correlation with Gleason scores >4. Furthermore, loss of MLH1 and MSH2 was more frequently observed in cases with Gleason scores >4. No significant differences in CDK12 expression were found across the Gleason grades. PDL-1 expression was detected in 42% of cases (21/50), primarily in patients with metastatic castration-resistant prostate cancer (mCRPC). Notably, BRCA1 positivity exhibited a strong correlation with higher Gleason scores (R2 = 0.75, P < 0.001). In conclusion, BRCA1 expression is strongly associated with higher Gleason scores, while loss of MMR proteins is commonly observed in cases with higher Gleason scores. CDK12 expression shows no significant variation across Gleason grades. These findings highlight the potential of targeting DDR pathways and immune checkpoints as a promising strategy for developing more effective therapies for prostate cancer.
 Citation Format: Himanshu Himanshu, Seema Kaushal, Amlesh Seth, Atul Batra, Ranjit Kumar Sahoo, Brusabhanu Nayak, Riyaz Ahmad Mir, Asit Ranjan Mridha. Profiling DNA damage response and immune checkpoint proteins in metastatic prostate cancer: Unraveling mechanisms of progression and therapeutic potential [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A041.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f048a6a5da85aa61efe0ff4e6965b2674d694af5" target='_blank'>
              Abstract A041: Profiling DNA damage response and immune checkpoint proteins in metastatic prostate cancer: Unraveling mechanisms of progression and therapeutic potential
              </a>
            </td>
          <td>
            Himanshu Himanshu, S. Kaushal, Amlesh Seth, A. Batra, R. Sahoo, B. Nayak, R.A. Mir, A.R. Mridha
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="As research into tumour immunotherapy continues to accelerate, new frontiers are being revealed in the field of cancer treatment. A significant focus has been drawn to neoantigen-based personalised tumour vaccines, a pioneering immunotherapy. This approach involves the use of genetic mutations that are unique to tumor cells to custom-design personalized tumor vaccines. These vaccines elicit an immune response that is specifically directed at targeting and eliminating cancer cells. The incorporation of neoantigens, arising from mutations within tumor cells, confers a distinct advantage to personalized tumor vaccines in terms of precision and the mitigation of adverse effects. However, the intricate pathways from antigen presentation to the activation of tumor immunogenicity remain to be elucidated. This paper primarily delves into the origins and characteristics of neoantigens, and also neoantigen prediction, highlights existing screening methods, and addresses the limitations of current approaches. It is hoped that this review will act as a catalyst, accelerating the understanding of relevant knowledge and illuminating research hotspots for scientists poised to venture into neoantigen research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea61347bed9e76289661e280b6f539cf9500a162" target='_blank'>
              Neoantigens: new hope for cancer therapy
              </a>
            </td>
          <td>
            Yitong Hu, Tengda Zhou, Ping Cai, Zihao He
          </td>
          <td>2025-03-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c55ad387d82ce60d43ef750c8b35debeea4164" target='_blank'>
              Programmable mRNA therapeutics for controlled epigenomic modulation of single and multiplexed gene expression in diverse diseases.
              </a>
            </td>
          <td>
            Charles W O'Donnell, J. Farelli, Houda Belaghzal, Justin Chen, Lauren Beech, James Sullivan, Chevaun D. Morrison-Smith, Stephen Siecinski, Adam Katz, Samuel Mildrum, Mayur Gurnani, Prachi Dhanania, Caitlyn R Webb, Giuliana Castello Coatti, Pranjali Rumale, Daniel F G Costa, Marcus I Gibson, Yaoyu E Wang, J. Newman, T. McCauley
          </td>
          <td>2025-03-13</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma.
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc5699f8d775d7c0b21dfb5b4e13161b3a6de2f" target='_blank'>
              Appraisal of CRISPR Technology as an Innovative Screening to Therapeutic Toolkit for Genetic Disorders.
              </a>
            </td>
          <td>
            A. Shahid, Ambreen Zahra, S. Aslam, Amen Shamim, Waqas Rafique Ali, Bilal Aslam, Sultan Habibullah Khan, Muhammad Imran Arshad
          </td>
          <td>2025-02-02</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is the most prevalent type of malignant epithelial neoplasm that affects the oral cavity. It has been a significant health concern in many countries for a long time since it was usually treated with surgery, radiation, and/or chemotherapy. Drug resistance is the primary issue in patient populations and scientific research, which promotes OSCC tumour cell invasion and migration. Thus, identifying highly specific therapeutic targets could be the potential approach for more successful treatment of OSCC. It is still challenging to understand the genetic causes of oral carcinogenesis due to its highly varied clinic-pathological parameters. It is important to remember that signaling channels and complexes that affect chromatin accessibility control gene expression, which in turn affects cell development and differentiation. Histones undergo post-translational alteration to give this platform. Understanding the processes of gene regulation through histone methylation and its modifications could enhance the early detection, prognostic prediction, and therapy of OSCC. To be properly used as a therapeutic target, histone methylation in OSCC requires more investigation. This review details the dysregulated histone methylation and the modifying enzymes linked to the development and aetiology of OSCC. Furthermore, the part that lysine methylation plays in cell migration, chemo-resistance, and OSCC invasion is also investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ac8fe27c84ad3ea1843eef61f7ad80802b8dbb" target='_blank'>
              Molecular insight into histone methylation as a novel target for oral squamous cell carcinoma: future hope in personalised medicine
              </a>
            </td>
          <td>
            P. Rajendran, Monisha Prasad, Enas M. Ali, Ramya Sekar, Abdullah M AlZahrani, M. Karobari, M. A. M. Genena, B. Abdallah
          </td>
          <td>2025-01-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Genetic materials, including DNA and RNA molecules in the nucleus and cytosol, contain vital information related to protein production, metabolic regulation, and signal transduction. Understanding these molecules and their expression levels is important for research in different fields of biology, especially in the study of human disease. Owing to the development of next-generation sequencing technology, single-cell sequencing technology measures gene expression levels from mRNAs through the process of reverse transcription, allowing researchers to gain insights into the overexpression/low expression of specific proteins in various cells in particular disease models, which could help researchers explore the signaling pathways associated with the cellular and molecular mechanisms of these diseases. For example, single-cell sequencing has revealed the selective vulnerability of dopamine neurons and nuclear factor κB signaling pathway pathways associated with the progression of Parkinson’s disease. In Alzheimer’s disease research, single-cell sequencing has helped identify subpopulations of cells associated with disease pathology and has provided insights into the inflammatory response associated with the disease. Herein, we review the application of single-cell sequencing technology and its clinical implications in Parkinson’s disease and Alzheimer’s disease. As life expectancy increases annually, the incidence of neurodegenerative diseases is on the rise. This trend underscores the urgency and relevance of the use of single-cell sequencing to explore the anomalous expression levels of pathogenic genes and related signaling pathways in different brain regions. This paper introduces three subtypes of total RNA sequencing, namely, single-nucleus sequencing, single-cell sequencing, and spatial sequencing. It also introduces several advanced applications of these technologies in the research of Parkinson’s disease and Alzheimer’s disease. At the same time, this paper also explores the challenges contained in these sequencing methods and discusses their possible developments in the future. Overall, single-cell sequencing technology provides new tools and methods for Parkinson’s disease and Alzheimer’s disease research. Despite these challenges, there is great potential to improve our understanding of these complex diseases and to develop new therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6061f6b22f4eeaa5754cf5de2d91103b0a9be5f" target='_blank'>
              Application of single-cell sequencing technology and its clinical implications in Parkinson’s disease and Alzheimer’s disease: a narrative review
              </a>
            </td>
          <td>
            Zhonghao Chen, Jack Shi, Longfei Li
          </td>
          <td>2025-02-06</td>
          <td>Advanced Technology in Neuroscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/196e4f5c9c9b8a27230e96850b6632f862c0dcae" target='_blank'>
              SKI complex loss renders 9p21.3-deleted or MSI-H cancers dependent on PELO
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony C Lau, Lucy Parker-Burns, Daniel A Lubicki, Tianxia Li, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Tessla Chan, Riya V Kishen, Anisah Adeagbo, Srivatsan Raghavan, Elisa Aquilanti, John R. Prensner, J. Krill-Burger, Todd R. Golub, Catarina D. Campbell, Joshua M Dempster, Edmond M Chan, Francisca Vazquez
          </td>
          <td>2025-02-05</td>
          <td>Nature</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="BACKGROUND/AIM
As an increasing number of drugs are approved for targeted cancer therapy, comprehensive genomic profiling of cancer patients is frequently conducted to identify potentially relevant genetic variants. Different sequencing technologies are used for this purpose; targeted gene panels cover a selected set of biomarker genes and hotspot regions, while whole genome sequencing (WGS) delivers genome-wide data. This comparison study aimed at evaluating whether one method performs superiorly to the other regarding the detection of targetable variants.


PATIENTS AND METHODS
To evaluate the performance of the two sequencing technologies, we compared the results of targeted sequencing using the Ion Torrent Oncomine Comprehensive Assay Plus (OCA-Plus) panel to Illumina WGS reports in 11 patients diagnosed with pancreatic cancer (PC). All pathogenic and likely pathogenic variants, including those relevant for targeted therapy, reported by WGS and OCA-Plus, were included in the final comparison.


RESULTS
Both techniques identified common driver mutations implicated in PC with high concordance (81%) across all variants. For variants relevant to targeted therapy, a 100% concordance between the technologies was observed.


CONCLUSION
A comparable number of variants were reported by WGS and OCA-Plus, and all genetic variants relevant for targeted therapy were identified by both technologies. Thus, WGS does not provide substantial additional information in this patient group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b34f2239a29bc4dc0f37c50d80fd5e063284e99d" target='_blank'>
              Paired Comparison of Whole Genome Sequencing and Comprehensive Targeted Sequencing of Pancreatic Cancer Tissue.
              </a>
            </td>
          <td>
            Thea Amalie Hvidtfeldt, Tim S Poulsen, I. Chen, Louise Laurberg Klarskov, Estrid Høgdall
          </td>
          <td>2025-01-31</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4233622be9dcbd1fb28cce4bd3a5ada680860" target='_blank'>
              Towards understanding cancer dormancy over strategic hitching up mechanisms to technologies
              </a>
            </td>
          <td>
            Sumin Yang, Jieun Seo, Jeonghyeon Choi, Sung-Hyun Kim, Yunmin Kuk, Kyung Chan Park, Mingon Kang, Sangwon Byun, Jae-Yeol Joo
          </td>
          <td>2025-02-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Kirsten Rat Sarcoma Virus mutation and its complications have been a major concern. Limitations to drawing its accurate configuration, its complications; mutates and thermodynamic properties, have limited their effectiveness as a molecular diagnostic tool. To counteract these challenges, over the years a different set of sophisticated Immunotherapeutic approaches have been adopted such as Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Caspase Protein System (CAS), and Chimeric Antigen Receptor-T cell (CAR-T). Some of them have already managed approval of the Food and Drug Administration. One of the primary goals of CRISPR/CAS technology is to correct mutations within the genetic sequence, thereby potentially repairing defective genes like KRAS. Meanwhile, CAR-T immunotherapies capitalise on genetically modified T cells. In comparison to traditional therapies, immunotherapies can offer greater safety and precision. However, their efficacy is still questionable while treating solid tumours. Here authors have covered the various key targets such as p53, p21, WNT, and RAS, their success stories over the years and have proposed the importance of targets like KRAS oncogene to manage a better output. KRAS is a RAS family oncoprotein responsible for the development of 85% of different cancers. The missense mutations at the various KRAS hotspots can lead to the development of lung and Pancreatic Cancers. Some of the most recently conducted revolutionary studies synthesize CAR-NK, CAR-M, and CRISPR-Cas9/3/10/Novel CAS systems, as well as their prospective molecular combination to target the KRAS. The purpose of this study is to provide a present status of these therapies, limitations, clinical trials, and a proposal to sketch a combinatorial impact via the PRISMA model for 62 different manuscripts (2001-2024). This format aligns and organizes everything in Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, providing a clear and systematic infographic that represents the process, including stages of selection, screening, and documentation.
 Citation Format: SDanish Kadir. Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer is a common and fatal disease that affects many people globally. CRC is classified as the third most prevalent cancer among males and the second most frequent cancer among females worldwide. The purpose of this article is to examine how personalized medicine might be used to treat colorectal cancer. The classification of colorectal cancer based on molecular profiling, including the detection of significant gene mutations, genomic instability, and gene dysregulation, is the main topic of this discussion. Advanced technologies and biomarkers are among the detection methods that are explored, demonstrating their potential for early diagnosis and precise prognosis. In addition, the essay explores the world of treatment possibilities by providing light on FDA-approved personalized medicine solutions that provide individualized and precise interventions based on patient characteristics. This article assesses targeted treatments like cetuximab and nivolumab, looks at the therapeutic usefulness of biomarkers like microsatellite instability (MSI) and circulating tumor DNA (ctDNA), and investigates new approaches to combat resistance. Through this, our review provides a thorough overview of personalized medicine in the context of colorectal cancer, ultimately highlighting its potential to revolutionize the field and improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b80ad33c609a3090e37d7111c49bfa155aa4a9e" target='_blank'>
              Personalized medicine in colorectal cancer: a comprehensive study of precision diagnosis and treatment.
              </a>
            </td>
          <td>
            Fatemeh Gila, Somayeh Khoddam, Zahra Jamali, Mohmmad Ghasemian, Shayan Shakeri, Zeinab Dehghan, Jafar Fallahi
          </td>
          <td>2025-02-09</td>
          <td>Personalized medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The recombinant human p53 adenovirus (Ad-p53) offers a promising approach for cancer therapy, yet its chromatin structure and effects on host chromatin organization and gene expression are not fully understood.In this study, we employed in situ ChIA-PET to investigate the colorectal cancer cell line HCT116 with p53 knockout, comparing them to cells infected with the adenovirus-vector expressing p53. We examined alterations in chromatin interactions and gene expression following treatment with the anti-cancer drug 5-fluorouracil (5-FU).Our results indicate that Ad-p53 forms a specific chromatin architecture within the vector and mainly interacts with repressive or inactive regions of host chromatin, without significantly affecting the expression of associated genes. Additionally, Ad-p53 does not affect topologically associating domains (TADs) or A/B compartments in the host genome.These findings suggest that while Ad-p53 boosts p53 expression, enhancing drug sensitivity without substantially altering host HCT116 chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2495bcc5f798b26cfeb646dc7337f34c34784" target='_blank'>
              Chromatin structure and gene transcription of recombinant p53 adenovirus vector within host
              </a>
            </td>
          <td>
            Duo Ning, Yuqing Deng, S. Tian
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma is the deadliest form of skin cancer, and its treatment poses significant challenges due to its aggressive nature and resistance to conventional therapies. Long non-coding RNAs (lncRNAs) represent a new frontier in the search for suitable targets to control melanoma progression and invasiveness. Indeed, lncRNAs exploit a wide range of regulatory functions along chromatin remodeling, gene transcription, post-transcription, transduction, and post-transduction to ultimately tune multiple cellular processes. The understanding of this intricate and flexible regulatory network orchestrated by lncRNAs in pathological conditions can strategically support the rational identification of promising targets, ultimately speeding up the setup of new therapeutics to integrate the currently available approaches. Here, the most recent findings on lncRNAs involved in melanoma will be analyzed. In particular, the functional links between their mechanisms of action and some frequently underestimated features, like their different subcellular localizations, will be highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08119f9fccb5c4f5a79dd1ca1781581cce224f1" target='_blank'>
              Unravelling the Regulatory Roles of lncRNAs in Melanoma: From Mechanistic Insights to Target Selection
              </a>
            </td>
          <td>
            Beatrice Moras, C. Sissi
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Chromatin remodeling proteins contribute to DNA replication, transcription, repair, and recombination. The chromodomain helicase DNA-binding (CHD) family of remodelers plays crucial roles in embryonic development, neurogenesis, and carcinogenesis. As the founding member, CHD1 is capable of assembling nucleosomes, remodeling chromatin structure, and regulating gene transcription. Dysregulation of CHD1 at genetic, epigenetic, and post-translational levels is common in prostate cancer and other malignancies. Through interacting with different genetic alterations, CHD1 possesses the capabilities to exert oncogenic or tumor-suppressive functions in context-dependent manners. In the past decade, my study uncovered the novel cell-intrinsic and -extrinsic roles of CHD1 during prostate cancer development, progression, and therapy resistance (Nature 2017, Cancer Discover 2020, Cancer Research 2022, Nature Cancer 2024). Cancer genome studies have uncovered a unique subtype of prostate cancer characterized by speckle-type BTB/POZ protein (SPOP) mutations, which stabilize AR and its co-activators and have been associated with prolonged responses to androgen deprivation therapies and favorable prognoses. Loss of CHD1 frequently co-occurs with SPOP mutations; however, how concurrent CHD1 loss impacts SPOP-mutated prostate cancer remains unclear. Most recently, by generating the first-in-the-field genetically engineered mouse models, we provided genetic evidence supporting that CHD1 loss accelerates SPOP-mutated prostate cancer progression and confers resistance to castration in this context. We uncover a non-canonical epigenetic function of CHD1 and demonstrate that CHD1 reprograms lipid metabolism by governing SREBPs. CHD1 loss–induced lipid reprogramming supplies intratumoral androgen biosynthesis and retains AR transcriptional activity in SPOP-mutated prostate tumors upon castration. Furthermore, the mechanistic studies facilitate the development of a novel biomarker-driven combinatorial therapy for castration-resistant prostate cancer harboring CHD1 loss and SPOP mutations. These findings advance our understanding of an emerging prostate cancer subtype and mechanism of action and provide novel therapeutic strategies for men with advanced prostate cancer. *This research article has been accepted by Nature Cancer for publication.
 Citation Format: Feiyu Chen, Di Zhao. Decipher actionable tumor suppressor defects for personalized cancer medicine [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lysosomal-associated protein transmembrane-4 beta (LAPTM4B) protein expression was increased in solid tumors, whereas few studies were performed in hematologic malignancies. We aimed to study the effect of the LAPTM4B gene in pan-cancer and Philadelphia chromosome-positive acute B cell lymphoblastic leukemia (Ph+ B-ALL).The differential expression, diagnosis, prognosis, genetic and epigenetic alterations, tumor microenvironment, stemness, immune infiltration cells, function enrichment, single-cell analysis, and drug response across cancers were conducted based on multiple computational tools. Additionally, Ph+ B-ALL transgenic mouse model with Laptm4b knockout was used to analyze the function of LAPTM4B in vivo. BrdU incorporation method, flow cytometry, and Witte-lock Witte culture were used to evaluate the roles of LAPTM4B in vitro.We identified that LAPTM4B expression was increased in various cancers, with significant associations with clinical outcomes. LAPTM4B expression correlated with DNA and RNA methylation patterns and was associated with drug resistance. It also influenced the tumor immune microenvironment, with implications for immunotherapy response. In leukemia, LAPTM4B was expressed in stem cells and associated with specific subtypes. Knockout of LAPTM4B impeded B-ALL progression in mice and reduced cell proliferation and caused G0/G1 arrest in vitro.Our study elucidated the role LAPTM4B that promoted the development and progression in Ph+ B-ALL. Furthermore, LAPTM4B played a diagnostic, prognostic, and immunological factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec638e3775596d8397e609fbd999f736d99d3a7b" target='_blank'>
              A comprehensive prognostic and immune analysis of LAPTM4B in pan-cancer and Philadelphia chromosome-positive acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Hui Zhou, Yuyao Yi, Wei He, Li Zheng, Yiguo Hu, Ting Niu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a366c7d16dafc61bb9a48ebda7fc255bd8a1ee2d" target='_blank'>
              HER2-targeted therapy in colorectal cancer: a comprehensive review.
              </a>
            </td>
          <td>
            Yeliz Benli, Helin Arıkan, Özge Akbulut-Çalışkan
          </td>
          <td>2025-03-14</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-cell expression quantitative trait loci (sceQTL) mapping offers a powerful approach for understanding gene regulation and its heterogeneity across cell types and states. It has profound applications in genetics and genomics, particularly causal gene regulatory network (cGRN) inference to unravel the molecular circuits governing cell identity and function. However, computational scalability remains a critical bottleneck for sceQTL mapping, prohibiting thorough evaluation and optimization of statistical accuracy. We present airqtl, a novel method to overcome these challenges through algorithmic advances and efficient implementations of linear mixed models. Airqtl achieves superior time complexity and over eight orders of magnitude of acceleration, enabling objective method evaluation and optimization. Airqtl offers de novo inference of robust, experimentally validated cell state-specific cGRNs that reflect perturbation outcomes. Our results dissect the drivers of cGRN heterogeneity and underscore the value of natural genetic variations in primary human cell types for biologically relevant single-cell cGRN inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f2cf8821d213360434445483db3b6f179c6897c" target='_blank'>
              Airqtl dissects cell state-specific causal gene regulatory networks with efficient single-cell eQTL mapping
              </a>
            </td>
          <td>
            Lingfei Wang
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c56d0e33616064b81832853efdda25a0062176f" target='_blank'>
              Exploring Epigenetic Complexity in Regulation of Hematopoietic Stem Cells Niche: A Mechanistic Journey from Normal to Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            N. A. Yusoff, Zariyantey Abd Hamid, Izatus Shima Taib, Siti Razila Abdul Razak, S. B. Budin
          </td>
          <td>2025-01-23</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy.The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity.We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered.This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Summary Recently formed allopolyploid species offer unprecedented insights into the early stages of polyploid evolution. This review examines seven well‐studied neopolyploids (we use ‘neopolyploid’ to refer to very recently formed polyploids, i.e. during the past 300 years), spanning different angiosperm families, exploring commonalities and differences in their evolutionary trajectories. Each neopolyploid provides a unique case study, demonstrating both shared patterns, such as rapid genomic and phenotypic changes, and unique responses to hybridization and genome doubling. While previous studies of these neopolyploids have improved our understanding of polyploidy, significant knowledge gaps remain, highlighting the need for further research into the varied impacts of whole‐genome duplication on gene expression, epigenetic modifications, and ecological interactions. Notably, all of these neopolyploids have spontaneously arisen due to human activity in natural environments, underscoring the profound consequences of polyploidization in a rapidly changing world. Understanding the immediate effects of polyploidy is crucial not only for evolutionary biology but also for applied practices, as polyploidy can lead to novel traits, as well as stress tolerance and increased crop yields. Future research directions include investigating the genetic and epigenetic mechanisms underlying polyploid evolution, as well as exploring the potential of neopolyploids for crop improvement and environmental adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5a204f000a3e9de1ebd9bce03b0ead5230c52b" target='_blank'>
              Natural neopolyploids: a stimulus for novel research
              </a>
            </td>
          <td>
            P. Edger, D. Soltis, Shunsuke Yoshioka, Mario Vallejo-Marín, Rie Shimizu‐Inatsugi, Kentaro K. Shimizu, A. Salmon, Simon Hiscock, Malika Ainouche, Pamela S Soltis
          </td>
          <td>2025-02-14</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumors employ a range of strategies to evade detection and eradication by the host’s immune system. These include downregulating antigen expression, altering antigen presentation processes, and inhibiting immune checkpoint pathways. etc. Adoptive Cell Therapy (ACT) represents a strategy that boosts anti-tumor immunity. This is achieved by amplifying or genetically engineering immune cells, which are either sourced from the patient or a donor, in a laboratory setting. Subsequently, these cells are reintroduced into the patient to bolster their immune response against cancer. ACT has successfully restored anti-tumor immune responses by amplifying the activity of T cells from patients or donors. This review focuses on the mechanisms underlying tumor escape, including alterations in tumor cell antigens, the immunosuppressive tumor microenvironment (TME), and modulation of immune checkpoint pathways. It further explores how ACT can avddress these factors to enhance therapeutic efficacy. Additionally, the review discusses the application of gene-editing technologies (such as CRISPR) in ACT, highlighting their potential to strengthen the anti-tumor capabilities of T cells. Looking forward, the personalized design of ACT, combined with immune checkpoint inhibitors and targeted therapies, is expected to significantly improve treatment outcomes, positioning this approach as a key strategy in the field of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3fa264efca4970b6216caa0126f6890f69da527" target='_blank'>
              Adoptive cell therapy against tumor immune evasion: mechanisms, innovations, and future directions
              </a>
            </td>
          <td>
            Liqin Ruan, Lu Wang
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 The benefit of adoptive cell transfer (ACT) therapies with T cells engineered to express chimeric antigen receptors (CAR) or T cell receptors (TCR) is limited by T cell dysfunction upon repetitive antigen exposure. We integrated a pooled genome-wide CRISPR knockout screen with in vitro repetitive antigen exposure and phenotypic selection to identify gene knockouts that improved the ability of NY-ESO-1 TCR transgenic T cells to repeatedly kill the antigen- and HLA-matched melanoma cell line M407. In two replicate screens, knocking out of the genes CUL5, RNF7 or ARIH2, which are required for proper cullin 5-RING ligase complex (CRL5) assembly, significantly improved long-term tumor cell killing ability of T cells past five rechallenges (one-way ANOVA p<0.0001). Mechanistically, we discovered that CRL5 interacts with SOCS-box containing proteins to regulate TCR stimulation and cytokine signaling through ubiquitination and proteasomal degradation. Ablation of CRL5 in T cells leads to an increase in polyfunctionality with augmented production and release of cytokines and cytolytic molecules across multiple rounds of killing when compared to control T cells (IFNg/TNFa producing T cells; Challenge 1: 18% versus 10%, Stimulation 3: 6% versus 3%, unpaired t-test p<0.0001). We demonstrated through both CRISPR knockout, and pharmacological inhibition of CRL5 using the neddylation inhibitor MLN4924, the negative regulatory mechanism of CRL5 in CD8+ T cells. These results highlight the previously unrecognized regulation of T cell function through E3 ligase-mediated proteasomal degradation, and that CRL5 may be a target to endow lasting effector function to antigen-specific T cells for cancer immunotherapies.
 Citation Format: Justin Saco, Daniel Karin, Nataly Naser Al Deen, Katie Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas. Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain- and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML), a refractory leukemia stem cell (LSC) disease, is characterized by the Philadelphia chromosome (Ph) t(9;22)(q34;q11.2) that generates the gene fusion between BCR and ABL1 genes. The BCR::ABL1 fusion gene products have a constitutively activated tyrosine kinase that causes leukemogenesis of CML. Tyrosine kinase inhibitors (TKIs) are effective in chronic phase of CML (CML-CP) but ineffective in blast phase of CML (CML-BC). The mechanisms of blast crisis (BC) in CML are highly complicated. This review aims to depict several mechanisms of BC induction in CML-BP. Blast transformation is caused by various mechanisms such as genetic alterations (IKZF1, TP53, INK4-ARF and Rb), epigenetic dysfunctions (aberrant methylation of CEBPα and PU.1, and underexpression of histone demethylase RBP2), suppression of differentiation-mediated transcription factors (CEBPα, CEBPβ, PU.1 and UBE2a), abnormal splicing (IKZF1 and GSK3β) and aberrantly regulated HIF-1α in bone marrow microenvironment. To develop novel therapies against blast cells in CML-BP based on the mechanisms of blast transformation, further studies are required.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee87ee3aa4ca681ef934614f3f0d367c700553ae" target='_blank'>
              Blast Transformation of Chronic Myeloid Leukemia Caused by Genetic Alterations, Epigenetic Dysfunctions and Aberrantly Regulated Factors in Bone Marrow Microenvironment
              </a>
            </td>
          <td>
            MD Mitsuru Sakitani
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Research in  Medical and Clinical Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Epithelial Ovarian Cancer (EOC) is the most lethal gynecologic cancer and those affected are in urgent need of new therapeutic strategies. Standard treatment is surgery followed by taxane- and platinum-based chemotherapy, but the rate of relapse is high and the 5-year survival is only 45%. Oncolytic viruses (OV) are a promising approach to EOC therapy through remodeling the immune composition of the tumor microenvironment (TME). Treatment response in EOC tumors can differ based on the presence of key tumorigenic mutations. This study evaluated the impact of specific tumor mutations on the response to the current standard of care carboplatin, two promising OV candidates VSV∆M51 and MG1, an infected cell vaccine (ICV-MG1) regimen, and the anti-angiogenic drug Fc3TSR. Mice with tumors harboring constitutive K-Ras activation showed an enhanced response to carboplatin and VSV∆M51 treatment. Additionally, VSV∆M51 treatment prolonged survival of syngeneic mice bearing tumors with mutations in Pten and Kras, Pten and Trp53, or Trp53 and Brca2 with increased activation of CD4+ and CD8+ T lymphocytes in the peritoneal TME. To enhance OV potency, an MG1-based infected cell vaccine inducing expression of IL-21 or IL15+21 was developed and found to enable strong and long-lasting antitumoral immunity in two carboplatin refractory syngeneic models, ID8 Trp53-/- and STOSE. VSV∆M51 combined with the anti-angiogenic Fc3TSR enhanced efficacy in the ID8 model. In summary, OV-based immunotherapy has shown promise in diverse murine models of EOC-bearing clinically relevant mutations, thus laying the foundation for developing new OV-based strategies to target a large spectrum of EOC genotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0be8c226447376238aa375a0e7bc3031f54f154" target='_blank'>
              Specific Genetic Mutations Impact Chemotherapy Resistance and Therapeutic Efficacy of Oncolytic Viruses in Ovarian Cancer.
              </a>
            </td>
          <td>
            Alison O Cudmore, G. Rodriguez, Vincent Maranda, Salar Farokhi Boroujeni, Humaira Murshed, Elizabeth A Macdonald, Melanie Grondin, K. Garson, K. Matuszewska, J. Diallo, James J Petrik, B. Vanderhyden
          </td>
          <td>2025-03-10</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) is the most prevalent malignancy among women, characterized by extensive heterogeneity stemming from molecular and genetic alterations. This review explores the intricate epigenetic landscape of BC, highlighting the significant role of epigenetic modifications—particularly DNA methylation, histone modifications, and the influence of non-coding RNAs—in the initiation, progression, and prognosis of the disease. Epigenetic alterations drive crucial processes, including gene expression regulation, cell differentiation, and tumor microenvironment interactions, contributing to tumorigenesis and metastatic potential. Notably, aberrations in DNA methylation patterns, including global hypomethylation and hypermethylation of CpG islands, have been associated with distinct BC subtypes, with implications for early detection and risk assessment. Furthermore, histone modifications, such as acetylation and methylation, affect cancer cell plasticity and aggressiveness by profoundly influencing chromatin dynamics and gene transcription. Finally, non-coding RNAs contribute by modulating epigenetic machinery and gene expression. Despite advances in our knowledge, clinical application of epigenetic therapies in BC is still challenging, often yielding limited efficacy when used alone. However, combining epi-drugs with established treatments shows promise for enhancing therapeutic outcomes. This review underscores the importance of integrating epigenetic insights into personalized BC treatment strategies, emphasizing the potential of epigenetic biomarkers for improving diagnosis, prognosis, and therapeutic response in affected patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a596476c40bed6ddb55424b08c446ae8d78faf" target='_blank'>
              Decoding the Epigenome of Breast Cancer
              </a>
            </td>
          <td>
            Elisa Cortellesi, Isabella Savini, Matteo Veneziano, Alessandra Gambacurta, M. Catani, V. Gasperi
          </td>
          <td>2025-03-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f9719dd384f2308e83ceab545b5fdb3fcb41a6" target='_blank'>
              Functional apoptosis profiling reveals vulnerabilities in T-cell large granular lymphocytic leukemia
              </a>
            </td>
          <td>
            Evgenii Shumilov, P. Mazzeo, M. Trautmann, Lena Levien, Kerstin Menck, Katharina Richter, Katharina Markus, Lena Ries, Detlef Haase, Elena Oberle, Philipp Berning, Wolfgang Hartmann, Philipp Ströbel, Andrea Kerkhoff, Georg Lenz, Gerald Wulf, Raphael Koch
          </td>
          <td>2025-02-06</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Second-generation anti-CD19 chimeric antigen receptor (CAR) T-cells incorporating CD28 (1928z) are now standard of care for treatment of refractory/relapsed large B-cell and mantle cell lymphomas. However, clinical use is limited by high toxicity rates, especially cytokine release syndrome (CRS) and immune effector cell-associated neurotoxicity syndrome (ICANS). Modifying intracellular signalling domains can influence CAR T-cell toxicity risk. Although rare, reports of CAR T-cell-derived malignancies underscore the risks of gene-modified cell therapies and the need to assess transgene insertion sites in new products. Toll-like receptor 2 (TLR2) a pattern recognition receptor, is highly expressed by activated CD8+ human T-cells, and can serve as a T-cell costimulatory signal, enhancing, memory formation, and modifying T-cell cytokine profile. In support of a phase 1 clinical trial (ENABLE-1, NCT04049513) evaluating a new third-generation anti-CD19 CAR T-cell product incorporating CD28 and TLR2 co-stimulatory domains (1928T2z), we explored the functional impact of the TLR2 domain in vitro, and characterised gene insertion sites within the 1928T2z product. Incorporating TLR2-derived Toll/IL-1 receptor resistance domain (TIR) between CD28 and CD3ζ domains in the 1928T2z CAR provided comparable CAR T-cell efficacy while producing lower levels of IFN-γ and GM-CSF, cytokines linked to CRS and ICANS pathogenesis, than 1928z. In contrast, positioning the TLR2 domain at the cytoplasmic tail (1928zT2) did not lower IFN-γ and GM-CSF production. Inhibiting TLR2 TIR function using ortho-vanillin, or through targeted mutations abrogating MYD88 recruitment and/or dimerization, impaired CAR T-cell activity, indicating the TLR2 domain contributes to 1928T2z CAR function. Gene expression identified downregulation of IL-6 signalling and upregulation of an anti-inflammatory cytokine pathway in 1928T2z cells compared to comparator CAR T-cells. A phosphoprotein array revealed that a functional TLR2 domain was associated with in higher levels of phosphorylated MEK1/2 (involved in TLR signalling), GATA3 (Th2-linked transcription factor) and CREB (a transcription factor that limits inflammatory responses). Using long-read sequencing and bioinformatics, we characterised insertion sites in a clonal line and polyclonal human CAR T-cells. Results showed mostly intronic or intergenic sites, with less than 10% mapping to exons, aligning with known lentiviral transgene integration profiles. In summary, incorporating a TLR2 co-stimulatory domain into CD28-costimulated anti-CD19 CAR T-cells modulates cytokine profile in a position-dependent manner, with TLR2 placement between CD28 and CD3ζ reducing CRS-and ICANS-linked cytokines while preserving in vitro anti-tumour activity. These findings align with our phase 1 trial data suggesting that 1928T2z CAR T-cells are associated with low incidence of CRS and ICANS, while exhibiting similar clinical efficacy to commercial comparators.
 Citation Format: Nathaniel Dasyam, Yasmin Nouri, Joshua Halpin, Alexander McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, Markus Templin, Qin Le, Peng Li, Ian Hermans, Rachel Perret, Robert Weinkove. Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739d0a74d9a81f832abc4486254c5ebe8688f593" target='_blank'>
              Abstract A018: Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain
              </a>
            </td>
          <td>
            N. Dasyam, Yasmin Nouri, J. Halpin, A. McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, M. Templin, Qin Le, Peng Li, I. Hermans, Rachel Perret, R. Weinkove
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Totipotent cells (TSCs), the origin of mammalian life and foundation to early mammalian embryogenesis, are characterized by the highest differentiation capacity and extensive developmental potential. However, none of the existing embryonic databases have provided epigenetic and transcriptomic resources on totipotency, greatly limiting our understanding of the mechanisms governing the establishment and exit of totipotency. Here, we present Toti, a pioneering multi-omics database exclusively developed for totipotency, covering in vivo, in vitro and genome-edited human and mouse embryonic TSCs, TSC-like cells, pluripotent cells (PSCs), and embryos spanning preimplantation stages, with a total of 8,265 samples. Toti facilitates an in-depth exploration of the molecular mechanisms underlying totipotency by offering Search, Browse and Analysis modules available at http://toti.zju.edu.cn/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0143151fabb07316572f667e24a48e297b397f77" target='_blank'>
              Toti: an integrated multi-omics database to decipher the epigenetic regulation of gene expression in totipotent stem cells
              </a>
            </td>
          <td>
            Yi Chai, Ruiying Zhang, Shunze Jia, Danfei Zhu, Siyi Chen, Xudong Fu, Xin Sheng
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Induced pluripotent stem cell (iPSC) models are powerful tools for neurodegenerative disease modelling, as they allow mechanistic studies in a human genetic environment and they can be differentiated into a range of neuronal and non-neuronal cells. However, these models come with inherent challenges due to line-to-line and clonal variability. To combat this issue, the iPSC Neurodegenerative Disease Initiative (iNDI) has generated an iPSC repository using a single clonal reference line, KOLF2.1J, into which disease-causing mutations and revertants are introduced via gene editing. Here we describe the generation and validation of lines carrying the most common causative mutation for amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), a repeat expansion in the C9orf72 gene, for the iNDI collection of neurodegenerative iPSC models. We demonstrate that these C9orf72 knock-in lines differentiate efficiently into neurons and display characteristic C9orf72-associated pathologies, including reduced C9orf72 levels and the presence of dipeptide repeat proteins (DPRs) and RNA foci, which increase in abundance over time in culture. These pathologies are not present in revertant cells lacking the repeat expansion. These repeat expansion and revertant cell lines are now available to academic and for-profit institutions through the JAX iPS cell repository and will help to facilitate and standardise iPSC-based ALS/FTD research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/692f124101507029a0d3e982b484f5b3a016b0d5" target='_blank'>
              Generation of C9orf72 repeat knock-in iPSC lines for modelling ALS and FTD
              </a>
            </td>
          <td>
            R. Coneys, Alexander J. Cammack, R. Nair, David Thompson, Jonas Mechtersheimer, M. Carcolé, Yashica Gupta, Gabriel E Rech, M. Flower, Niamh O’Brien, M. Ruepp, S. Mizielinska, Fiona Ducotterd, Sarah J Tabrizi, Elizabeth M. C. Fisher, Thomas J. Cunningham, Michael E. Ward, William Skarnes, Adrian M Isaacs
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Aging is considered the contributory accumulation of abruptions occurring through cell signaling cascades, which ultimately cause changes in physical functions, cell fate, and damage across all organ systems. DNA damage response (DDR) also occurs through telomere shortening, tumor formation, mitochondrial dysfunction, and so forth. Cellular aging occurs through cell cycle arrest, which is the result of extended DDR cascade signaling networks via MDC1, 53BP1, H2AX, ATM, ARF, P53, P13-Akt, BRAF, Sirtuins, NAD + , and so forth. These persistent cell cycle arrests initiated by DDR and other associated stress-induced signals promote a permanent state of cell cycle arrest called senescence-associated secretory phenotype (SASP). However, cellular aging gets accelerated with faulty DNA repair systems, and the produced senescent cells further generate various promoting contributors to age-related dysfunctional diseases including SASP. Any changes to these factors contribute to age-related disease development. Therefore, this review explores anti-aging factors targeting DDR and SASP regulation and their detailed signaling networks. In addition, it allows researchers to identify anti-aging targets and anti-aging therapeutic strategies based on identified and nonidentified targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3959b933b4c238cc534f4bd3bbf4f10f6618c5" target='_blank'>
              Recent Advances in Anti-Aging Therapeutic Strategies Targeting DNA Damage Response and Senescence-Associated Secretory Phenotype-Linked Signaling Cascade.
              </a>
            </td>
          <td>
            Jawad Nadeem, Razia Sultana, Amna Parveen, Sun Yeou Kim
          </td>
          <td>2025-02-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185bf82df0621d29d2214d3aa74691b2937269d4" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="The canonical DNA damage response (DDR) maintains genome stability, involving DNA synthesis/cell cycle arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously discovered and characterized a noncanonical cellular response that is specific to nonblocking replication stress, i.e., low-level stress (LoL-DDR), in primary cells. Although this response generates replication stress-induced reactive oxygen species (RIR), it triggers a program that prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a precise threshold triggers the canonical DDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: 1-RIR are not detoxified under high-level stress conditions, resulting in a continuous increase in the dose-response curve of RIR production; 2-RIR are not produced by NADPH oxidases; and 3-replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” involving replication-stress and ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfa2fab8a78a43f5b506788fc0cb6d9a375f31e" target='_blank'>
              Dysregulation of replication stress-induced ROS in transformed cell lines: a vicious circle at cancer initiation
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract Background The efficacy of immune checkpoint blockade (ICB) depends on restoring immune recognition of cancer cells that have evaded immune surveillance. Transforming growth factor-beta (TGFβ) is associated with immune-poor, so-called cold tumors whereas loss of its signaling promotes DNA misrepair that could stimulate immune response. Methods We analyzed transcriptomic data from IMvigor210, The Cancer Genome Atlas, and Tumor Immune Syngeneic MOuse data sets to evaluate the predictive value of high βAlt, a score representing low expression of a signature consisting of TGFβ targets and high expression of genes involved in error-prone DNA repair. The immune context of βAlt was assessed by evaluating tumor-educated immune signatures. An ICB-resistant, high βAlt preclinical tumor model was treated with a TGFβ inhibitor, radiation, and/or ICB and assessed for immune composition and tumor control. Results We found that a high βAlt score predicts ICB response yet is paradoxically associated with an immune-poor tumor microenvironmentcancer in both human and mouse tumors. We postulated that high βAlt cancers consist of cancer cells in which loss of TGFβ signaling generates a TGFβ rich, immunosuppressive tumor microenvironment. Accordingly, preclinical modeling showed that TGFβ inhibition followed by radiotherapy could convert an immune-poor, high βAlt tumor to an immune-rich, ICB-responsive tumor. Mechanistically, TGFβ inhibition increased activated natural killer (NK) cells, which were required to recruit lymphocytes to respond to ICB in irradiated tumors. NK cell activation signatures were also increased in high βAlt, cold mouse and human tumors that responded to ICB. Conclusions These studies indicate that loss of TGFβ signaling competency and gain of error-prone DNA repair identifies a subset of cold tumors that are responsive to ICB. Our mechanistic studies show that inhibiting TGFβ activity can convert a high βAlt, cold tumor into ICB-responsive tumors via NK cells. A biomarker consisting of combined TGFβ, DNA repair, and immune context signatures is a means to prospectively identify patients whose cancers may be converted from cold to hot with appropriate therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16f08083204e934f5372293d784704ac2a30ac3a" target='_blank'>
              Identification of a conserved subset of cold tumors responsive to immune checkpoint blockade
              </a>
            </td>
          <td>
            Jade Moore, Jim Gkantalis, Inés Guix, William Chou, Kobe Yuen, Ann A. Lazar, Matthew Spitzer, Alexis J. Combes, M. Barcellos-Hoff
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Comprehensive genetic analysis of tumors with exome or whole genome sequencing has enabled the identification of the genes that are recurrently mutated in cancer. This has stimulated a series of exciting advances over the past 15 years, guiding us to new molecular biomarkers and therapeutic targets among the common mature B-cell neoplasms. In particular, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL) and Burkitt lymphoma (BL) have each been the subject of considerable attention in this field. Currently, more than 850 genes have been reported as targets of protein-coding mutations in at least one of these entities. To reduce this to a manageable size, we describe a systematic approach to prioritize and categorize these genes, based on the quality and type of supporting data. For each entity, we provide a list of candidate driver genes categorized into Tier 1 (high-confidence genes), Tier 2 (candidate driver genes), or Tier 3 (lowest-confidence genes). Collectively, this reduces the number of high-confidence genes for these three lymphomas to a mere 144. This further affirms the substantial overlap between the genes relevant in DLBCL and each of FL and BL. These highly curated and annotated gene lists will continue to be maintained as a resource to the community. These results emphasize the extent of the knowledge gap regarding the role of each of these genes in lymphomagenesis. We offer our perspective on how to accelerate the experimental confirmation of drivers using a variety of model systems, using these lists as a guide for prioritizing genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf53884a55922c51a797521fda638d01af815c" target='_blank'>
              Audit of B-cell Cancer Genes.
              </a>
            </td>
          <td>
            K. Coyle, K. Dreval, Daniel J Hodson, Ryan D Morin
          </td>
          <td>2025-01-24</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 The development of tumoroid (or cancer organoid) technology has enabled the in vitro culture of patient-derived cancer cells that retain key morphological, mutational, and transcriptomic characteristics of patient tumors. In a number of studies, the response of tumoroids to therapies has mirrored that seen in the patients from which the tumoroids were derived. As a result, there is increasing interest in the use of tumoroids in functional precision oncology, in which an individual patient’s cancer cells are used as ex vivo tumor avatars and tested for response to a range of various potential treatments to aid in therapy selection. Here, we explored a workflow to integrate genomic sequencing with functional drug sensitivity testing for precision medicine applications. Excess FFPE tissue from 6 colorectal cancer donors was processed to generate H&E slides and unstained sections. Slides (2 x 10 µm sections per donor) were digested, and RNA and DNA were isolated using an automated nucleic acid purification system. The output samples were analyzed on the Genexus™ Integrated Sequencer for targeted detection of SNVs, CNVs, gene fusions, and indels for each sample using the Oncomine™ Comprehensive Assay v3. Purified RNA was also utilized for bulk RNA sequencing to generate transcriptomic profiles for each donor. In parallel, viably cryopreserved dissociated tumor cells (DTCs) were thawed and plated in Gibco™ OncoPro™ Tumoroid Culture Medium, a commercially available, serum-free, and conditioned medium-free system. OncoPro medium was developed and optimized for the culture of patient-derived tumor cells from solid cancers to provide an easy-to-use, reproducible, cGMP manufactured culture system for these samples. Where sample size allowed, RNA was isolated from DTC samples and sequenced to compare transcriptomic results generated from FFPE vs. viably cryopreserved material. Additionally, DTCs were sampled for untargeted proteomic and metabolomic profiling. DTCs were cultured for 3-7 days in OncoPro medium prior to exposure to chemotherapy drugs used in standard-of-care treatment for colorectal cancer, and cell response was quantified. Dissociated tumoroid cells formed tumoroids at high rates in culture, in line with previous studies showing that patient-derived cancer cells across colorectal, endometrial, lung, breast, and head and neck cancers form tumoroids in OncoPro medium in >85% of samples within 7 days of tissue receipt. We observed heterogeneous genomic, proteomic, transcriptomic, and drug response profiles across donors. Current efforts are focused on development of computational methods to tie multiomic characterization with functional response. Overall, this data presents an integrated workflow to combine genomic approaches with functional testing to inform patient care.
 Citation Format: Colin D Paul, Amber Bullock, Anthony Chatman, Km Shams Ud Doha, Harrison Leong, Sylvia Beam, Chris Yankaskas, Pradip Shahi Thakuri, Matt Dallas, David Kuninger. An integrated workflow for multiomic characterization and functional precision medicine testing of solid tumor samples [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/564696ec86cb40931955901184003710aa166dc0" target='_blank'>
              Abstract B017: An integrated workflow for multiomic characterization and functional precision medicine testing of solid tumor samples
              </a>
            </td>
          <td>
            Colin Paul, Amber Bullock, Anthony Chatman, Km Shams Ud Doha, Harrison Leong, Sylvia Beam, Chris Yankaskas, Pradip S. Thakuri, Matt Dallas, David Kuninger
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Perturb-seq is a powerful approach to systematically assess how genes and enhancers impact the molecular and cellular pathways of development and disease. However, technical challenges have limited its application in stem cell-based systems. Here, we benchmarked Perturb-seq across multiple CRISPRi modalities, on diverse genomic targets, in multiple human pluripotent stem cells, during directed differentiation to multiple lineages, and across multiple sgRNA delivery systems. To ensure cost-effective production of large-scale Perturb-seq datasets as part of the Impact of Genomic Variants on Function (IGVF) consortium, our optimized protocol dynamically assesses experiment quality across the weeks-long procedure. Our analysis of 1,996,260 sequenced cells across benchmarking datasets reveals shared regulatory networks linking disease-associated enhancers and genes with downstream targets during cardiomyocyte differentiation. This study establishes open tools and resources for interrogating genome function during stem cell differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a019d0592b40aaea0bb2bdf58ea166a338fcf9" target='_blank'>
              Benchmarking and optimizing Perturb-seq in differentiating human pluripotent stem cells
              </a>
            </td>
          <td>
            Sushama Sivakumar, Yihan Wang, Sean C Goetsch, Vrushali Pandit, Lei Wang, Huan Zhao, Anjana Sundarrajan, Daniel A Armendariz, Chikara Takeuchi, Mpathi Z. Nzima, Wei-Chen Chen, Ashley E. Dederich, Lauretta El Hayek, Taosha Gao, Renad Ghazawi, Ashlesha Gogate, Kiran J. Kaur, Hyung Bum Kim, Melissa K. McCoy, H. Niederstrasser, Seiya Oura, Carolos A Pinzon-Arteaga, Menaka Sanghvi, Daniel A. Schmitz, Leqian Yu, Yanfeng Zhang, Qinbo Zhou, W. Kraus, Lin Xu, Jun Wu, Bruce A. Posner, Maria H. Chahrour, Gary C. Hon, N. Munshi
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Chemoresistance represents a major challenge for osteosarcoma treatment. Despite the improved knowledge of cancer biology, the core determinants of cisplatin (DDP) resistance in osteosarcoma remain unclear and deserve further exploration. Here, RFWD3 is identified as a key regulator of DDP sensitivity in osteosarcoma using a genome-wide CRISPR screen. It is demonstrated that RFWD3 is overexpressed in post-chemotherapy osteosarcoma tissues compared to pre-chemotherapy tissues. Knocking out RFWD3 increased the sensitivity of osteosarcoma cells to DDP treatment. Mechanistically, RFWD3 bound to and ubiquitinated PHGDH at the Lys137 residue, promoting its degradation and conserving cellular oxidized nicotinamide adenine dinucleotide (NAD+). The resulting surplus of NAD+ enhanced the TCA cycle, leading to increased production of aspartic acid and glutamic acid for de novo nucleotide biosynthesis. In addition, virtual screening techniques are employed to identify Lomitapide as a specific inhibitor of the RFWD3-PHGDH interaction, capable of disrupting the binding between RFWD3 and PHGDH. It is found that Lomitapide exhibits a significant synergistic anti-osteosarcoma effect when combined with DDP. In conclusion, a specific role of RFWD3 in regulating nucleotide metabolism is revealed and comprised of targetable candidates for overcoming chemoresistance in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe4a6c0c662b6150e0f25a7a03fb8e70f8ebcc2" target='_blank'>
              RFWD3 Reprograms Nucleotide Metabolism Through PHGDH to Induce Chemoresistance In Osteosarcoma.
              </a>
            </td>
          <td>
            Wenchao Zhang, Chi Yin, Lin Qi, Zhongyue Liu, Ruiling Xu, Chao Tu, Zhihong Li
          </td>
          <td>2025-02-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cb9fafc9e4ae8646c09de8af8d088ea1242ab7" target='_blank'>
              Mammalian Species-Specific Resistance to Mammary Cancer
              </a>
            </td>
          <td>
            I. Barash
          </td>
          <td>2025-03-06</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0687d954e3a7ce3f1c8da610c5b175b62bd0c491" target='_blank'>
              Single-cell multiomics reveals a gene regulatory circuit driving leukemia cell differentiation.
              </a>
            </td>
          <td>
            Xin Tian, Liuqingqing Zhang, Guiqiyang Xiang, Yijia Tang, Ping Zhu, Shuting Yu, Fangying Jiang, Shuai Wang, Jinzeng Wang, Yao Dai, Desheng Zheng, Jianbiao Wang, Xiangqin Weng, Shengyue Wang, Yun Tan, Feng Liu
          </td>
          <td>2025-02-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background and Objective: The discovery of novel molecular biomarkers via next-generation sequencing technologies has revolutionized how glioblastomas (GBMs) are classified nowadays. This has resulted in more precise diagnostic, prognostic, and therapeutic approaches to address this malignancy. The present work examines the applications of single-cell RNA sequencing (scRNA-seq) in GBM, focusing on its potential to address tumor complexity and therapeutic resistance and improve patient outcomes. Methods: A scoping review of original studies published between 2009 and 2024 was conducted using the PUBMED and EMBASE databases. Studies in English or Spanish related to single-cell analysis and GBM were included. Key Findings: The database search yielded 453 publications. Themes related to scRNA-seq applied for the diagnosis, prognosis, treatment, and understanding of the cancer biology of GBM were used as criteria for article selection. Of the 24 studies that were included in the review, 11 focused on the tumor microenvironment and cell subpopulations in GBM samples, 5 investigated the use of sequencing to elucidate the GBM cancer biology, 3 examined disease prognosis using sequencing models, 3 applied translational research through scRNA-seq, and 2 addressed treatment-related problems in GBM elucidated by scRNA-seq. Conclusions: This scoping review explored the various clinical applications of scRNA-seq technologies in approaching GBM. The findings highlight the utility of this technology in unraveling the complex cellular and immune landscapes of GBM, paving the way for improved diagnosis and personalized treatments. This cutting-edge approach might strengthen treatment strategies against tumor progression and recurrence, setting the stage for multi-targeted interventions that could significantly improve outcomes for patients with aggressive, treatment-resistant GBMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f255f1e2df6321b7a03cd03e70b813bcc99665" target='_blank'>
              Current Applications of Single-Cell RNA Sequencing in Glioblastoma: A Scoping Review
              </a>
            </td>
          <td>
            Edgar G. Ordóñez-Rubiano, Nicolás Rincón-Arias, William J. Shelton, A. F. Salazar, María Alejandra Sierra, Raphael Bertani, D. Gómez-Amarillo, Fernando Hakim, Matías Baldoncini, César Payán-Gómez, A. Cómbita, Sandra C. Ordonez-Rubiano, Rafael Parra-Medina
          </td>
          <td>2025-03-14</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) remains a serious threat to global health, with rising incidence and mortality rates. Therapeutic options for advanced HCC are quite limited, and the overall prognosis remains poor. Recent advancements in immunotherapy, particularly immune-checkpoint blockade (ICB) targeting anti-PD1/PD-L1 and anti-CTLA4, have facilitated a paradigm shift in cancer treatment, demonstrating substantial survival benefits across various cancer types, including HCC. However, only a subset of HCC patients exhibit a favorable response to ICB therapy, and its efficacy is often hindered by the development of resistance. There are many studies to explore the underlying mechanisms of ICB response. In this review, we compiled the latest progression in immunotherapies for HCC and systematically summarized the sophisticated mechanisms by which components of the tumor microenvironment (TME) regulate resistance to ICB therapy. Additionally, we also outlined some scientific rationale strategies to boost antitumor immunity and enhance the efficacy of ICB in HCC. These insights may serve as a roadmap for future research and help improve outcomes for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7cb79517a5aec504caaabf1fd85cb4e7d72745b" target='_blank'>
              Sophisticated roles of tumor microenvironment in resistance to immune checkpoint blockade therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yi-Zhe Zhang, Yunshu Ma, Ensi Ma, Xizhi Chen, Yue Zhang, Baobing Yin, Jing Zhao
          </td>
          <td>2025-02-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Large-scale international and regional human genomic and pangenomic resources derived from population-scale biobanks and ancient DNA sequences have provided significant insights into human evolution and the genetic determinants of complex diseases and traits. Despite these advances, challenges persist in optimizing the integration of phasing tools, merging haplotype reference panels (HRPs), developing imputation algorithms, and fully exploiting the diverse applications of post-imputation data. This review comprehensively summarizes the advancements, applications, limitations, and future directions of HRPs in human genomics research. Recent progress in the reconstruction of HRPs, based on over 830,000 human whole-genome sequences, has been synthesized, highlighting the broad spectrum of human genetic diversity captured. Additionally, we recapitulate advancements in fifty-six HRPs for global and regional populations. The evaluation of imputation accuracy indicated that Beagle and GLIMPSE are the most effective tools for phasing and imputing data from genotyping arrays and low-coverage sequencing, respectively. A critical strategy for selecting an appropriate HRP involves matching the population background of target groups with HRP reference populations and considering multi-ancestry or homogeneous genetic structures. The necessity of a single, integrative, high-quality HRP that captures haplotype structures and genetic diversity across various genetic variation types from globally representative populations is emphasized to support both modern and ancient genomic research and advance human precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c6106c6635da051acdf061d2d28fcc5819c939" target='_blank'>
              High-quality Population-specific Haplotype-resolved Reference Panel in the Genomic and Pangenomic Eras.
              </a>
            </td>
          <td>
            Qingxin Yang, Yuntao Sun, Shuhan Duan, S. Nie, Chao Liu, Hong Deng, Mengge Wang, Guanglin He
          </td>
          <td>2025-03-10</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A large fraction of the human genome is transcribed in RNA molecules that do not encode for proteins but that do have a crucial role in regulating almost every level of gene expression and, thus, define the specific phenotype of each cell. These non-coding RNAs include well-characterized microRNAs and thousands of less-defined longer transcripts, named long non-coding RNAs. Both types markedly affect the onset and the progression of numerous pathologies, ranging from cancer to vascular and neuro-degenerative diseases. In recent years, a substantial effort has been made to design drugs targeting ncRNAs, and promising advancements have been produced from micro-RNA mimics and inhibitors. Each ncRNA controls several targets, and the overall effect of its inhibition or overexpression depends on the function of the set of genes it regulates. Therefore, in selecting the most appropriate target, and predicting the final outcome of ncRNA-based therapies, it is crucial to have and utilize detailed and accurate knowledge of their functional interactions. In this review, I recapitulate the principal resources which collect information on microRNA and lncRNA networks, focusing on the non-homogeneity of the data that result from disparate approaches. I highlight the role of RNA identifiers and interaction evidence standardization in helping the user to filter and integrate data derived from different databases in a reliable functional web of regulative relations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45039e8fa4423e21f993b2688bba0179d8a2c57" target='_blank'>
              The Relevance of the Accurate Annotation of Micro and Long Non-Coding RNA Interactions for the Development of Therapies
              </a>
            </td>
          <td>
            Simona Panni
          </td>
          <td>2025-02-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9281e0d64a5756a86b842209be41d38103ffa691" target='_blank'>
              Colorectal carcinogenesis in the Lynch syndromes and familial adenomatous polyposis: trigger events and downstream consequences
              </a>
            </td>
          <td>
            Pål Møller, A. Ahadova, Matthias Kloor, Toni T. Seppälä, John Burn, Saskia Haupt, F. Macrae, M. Dominguez-Valentin, Gabriela Möslein, Annika Lindblom, Lone Sunde, Ingrid Winship, Gabriel Capellá, Kevin Monahan, D. Buchanan, D. G. Evans, E. Hovig, Julian R. Sampson
          </td>
          <td>2025-01-23</td>
          <td>Hereditary Cancer in Clinical Practice</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Primary human cells cultured in organoid format have great promise as potential regenerative cellular therapies. However, their immunogenicity and mutational profile remain unresolved, impeding effective long-term translation to the clinic. In this study we report, for the first time, the generation of human leukocyte antigen (HLA)-I and HLA-II knock-out expandable human primary cholangiocyte organoids (PCOs) using CRISPR-Cas9 as a potential ‘universal’ low-immunogenic therapy for bile duct disorders. HLA-edited PCOs (ePCOs) displayed the same phenotypical and functional characteristics as parental un-edited PCOs. Despite minimal off-target edits, single-molecule DNA-sequencing demonstrated that ePCOs and PCOs acquire substantial mutations in culture at similar rates but without evident selection for cancer-driver mutations. ePCOs induced reduced T cell-mediated immunity and a donor-dependent NK cell cytotoxicity in vitro and evaded cytotoxic responses with increased graft survival in humanized mice in vivo. Our findings have important implications for assessment of safety and immunogenicity of organoid cellular therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45d4053045048435e735430b496d5825010d393" target='_blank'>
              Immune and Mutational Profile of Gene-Edited Low-Immunogenic Human Primary Cholangiocyte Organoids
              </a>
            </td>
          <td>
            Sandra Petrus-Reurer, A. Baez-Ortega, Winnie Lei, Girishkumar Kumaran, James Williamson, Julia Jones, Daniel Trajkovski, A. Lawson, Krishnaa T. Mahbubani, Cara Brodie, Paul Lehner, Matthew J. Bottomley, I. Martincorena, K. Saeb‐Parsy
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Chronic inflammation is a well-established risk factor for cancer, but the underlying molecular mechanisms remain unclear. Using a mouse model of colitis, we demonstrate that colonic stem cells retain an epigenetic memory of inflammation following disease resolution, characterized by a cumulative gain of activator protein 1 (AP-1) transcription factor activity. Further, we develop SHARE-TRACE, a method that enables simultaneous profiling of gene expression, chromatin accessibility and clonal history in single cells, enabling high resolution tracking of epigenomic memory. This reveals that inflammatory memory is propagated cell-intrinsically and inherited through stem cell lineages, with certain clones demonstrating dramatically stronger memory than others. Finally, we show that colitis primes stem cells for amplified expression of regenerative gene programs following oncogenic mutation that accelerate tumor growth. This includes a subpopulation of tumors that have exceptionally high AP-1 activity and the additional upregulation of pro-oncogenic programs. Together, our findings provide a mechanistic link between chronic inflammation and malignancy, revealing how long-lived epigenetic alterations in regenerative tissues may contribute to disease susceptibility and suggesting potential therapeutic strategies to mitigate cancer risk in patients with chronic inflammatory conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor-specific antigens, also known as neoantigens, have potential utility in anti-cancer immunotherapy, including immune checkpoint blockade (ICB), neoantigen-specific T cell receptor-engineered T (TCR-T), chimeric antigen receptor T (CAR-T), and therapeutic cancer vaccines (TCVs). After recognizing presented neoantigens, the immune system becomes activated and triggers the death of tumor cells. Neoantigens may be derived from multiple origins, including somatic mutations (single nucleotide variants, insertion/deletions, and gene fusions), circular RNAs, alternative splicing, RNA editing, and polymorphic microbiome. An increasing amount of bioinformatics tools and algorithms are being developed to predict tumor neoantigens derived from different sources, which may require inputs from different multi-omics data. In addition, calculating the peptide-major histocompatibility complex (MHC) affinity can aid in selecting putative neoantigens, as high binding affinities facilitate antigen presentation. Based on these approaches and previous experiments, many resources were developed to reveal the landscape of tumor neoantigens across multiple cancer types. Herein, we summarized these tools, algorithms, and resources to provide an overview of computational analysis for neoantigen discovery and prioritization, as well as the future development of potential clinical utilities in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b6cef04b3606a814455c2e9548b49b282ba61" target='_blank'>
              Characterization of Tumor Antigens from Multi-omics Data: Computational Approaches and Resources.
              </a>
            </td>
          <td>
            Yunzhe Wang, James Wengler, Yuzhu Fang, Joseph Zhou, Hang Ruan, Zhao Zhang, Leng Han
          </td>
          <td>2025-01-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412c192baedece3963b4137ef5579bc1b8117afb" target='_blank'>
              Genetic variants in NHEJ1 and related DNA repair disorders: insights into phenotypic heterogeneity and links to hypoplastic myelodysplastic syndromes and familial hematological malignancies susceptibility.
              </a>
            </td>
          <td>
            Mahmoud I Elbadry, Elsayed Abdelkreem, Ahmed Tawfeek, Go Hun Seo, S. Aziz
          </td>
          <td>2025-03-06</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="SKA3, an important factor in cell cycle regulation, is involved in spindle assembly and kinetochore function, playing a critical role in maintaining cancer cell proliferation and division. However, its specific roles and regulatory mechanisms in cancer remain not fully understood. Large-scale datasets from multiple public databases, including The Cancer Genome Atlas and Genotype-Tissue Expression, covering various cancer types, were integrated. Systematic analysis revealed that SKA3 exhibits aberrant expression patterns in multiple cancers and is significantly associated with tumor progression and poor patient prognosis in certain cancers. We explored the status of SKA3 gene mutation, gene amplification and promoter region methylation in various tumors. In the context of immunotherapy, we assessed the value of SKA3 in cancer. Analyzing the correlation between SKA3 expression levels and immune checkpoints and immune cell infiltration, we discovered that SKA3 could serve as a novel immunotherapy biomarker across multiple cancers, guiding clinical immunotherapy decisions. Finally, SKA3 knockdown inhibited lung adenocarcinoma cell proliferation and metastasis. In conclusion, this study provides new insights into the role of SKA3 in cancer and offers significant theoretical and experimental evidence for its development as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cytokine release syndrome (CRS) is a common complication associated with chimeric antigen receptor T cell (CAR-T) therapy, primarily driven by excessive interleukin-6 (IL-6) production. Although the use of IL-6 inhibitors and corticosteroids can partially alleviate CRS, these approaches often yield limited success and may compromise CAR-T cell functionality. Recent advancements in gene-editing technologies, particularly CRISPR-Cas9, offer significant opportunities by enabling site-specific knockout of the IL-6 receptor (IL-6R) in CAR-T cells. In vitro studies have demonstrated that IL-6 gene deletion substantially reduces CRS-related inflammatory responses while preserving the anti-tumor capabilities of CAR-T cells. This strategy not only addresses inflammation at its source but also mitigates harmful systemic inflammatory responses, thereby improving therapeutic outcomes. This review highlights the pivotal role of IL-6 in CRS pathophysiology, the shortcomings of conventional treatment strategies, and the potential of genome-editing approaches such as IL-6R knockout. By precisely targeting the root causes of IL-6-mediated inflammation, this approach aims to optimize both the efficacy and safety of CAR-T cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f7401161a6069ba8245d4505d68dfc98e30045" target='_blank'>
              The Feasibility of Using CRISPR-Cas9 to Knock Out IL-6 Receptor in CAR-T Cells to Mitigate Cytokine Release Syndrome in Cancer Therapy
              </a>
            </td>
          <td>
            Zhiyuan Wang
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in genes encoding components of the SWI/SNF chromatin remodeling complex, including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, models with SMARCA4 loss showed decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells and CD4+ T cells into the tumor microenvironment. SMARCA4 loss in tumor cells led to profound downregulation of STING1, IL1B, and other components of the innate immune system, as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. The deregulation of gene expression was caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response, such as STING1, IL1B, type I interferon, and inflammatory cytokines. Interestingly, the transcription factor NF-κB binding motif was enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Furthermore, SMARCA4 and NF-κB co-occupied the same genomic loci on enhancers associated with STING1 and IFNB1, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, these findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to immunotherapy and establish a functional link between SMARCA4 and NF-κB in innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f316b25c9fd66656561befcef17f13bf1d2230" target='_blank'>
              Mcm5 mutation leads to silencing of Stat1-bcl2 which accelerating apoptosis of immature T lymphocytes with DNA damage
              </a>
            </td>
          <td>
            Min Liu, Yuanyuan Li, Zhilin Deng, Ke Zhang, Shuying Huang, Jiamin Xia, Yi Feng, Yundan Liang, Chengfu Sun, Xindong Liu, Shurong Li, B. Su, Yong Dong, Sizhou Huang
          </td>
          <td>2025-02-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Leukemia is characterized by multiple rearrangements of signal transduction genes and overexpression of nonmutated genes, such as transcription factors (TFs) genes. Super-enhancers (SEs) are prevalent in human cancers and are associated with the accumulation of numerous core TFs. SEs drive the expression of core TF genes by delivering robust transcriptional activation signals. Additionally, core TFs sustain the stability and activity of SEs through mutual auto-regulation loops, creating a positive feedback loop known as the Core Transcriptional Regulation Circuit (CRC). Using ChIP-seq data, we identified the involvement of the SE-related gene ZNF217 in acute myeloid leukemia (AML), in which its functional role and underlying mechanism remain unclear. We demonstrated that ZNF217, ELF1, MEF2D, RUNX2, and FOXP1 are likely integral components of the AML CRC through various experimental techniques, including CUT&Tag, short hairpin RNA (shRNA) transduction, and Luciferase reporter assays. Notably, ZNF217 was determined to be indispensable for the proliferation and viability of AML cells both in vitro and in vivo. Subsequent analysis of RNA-seq and CUT&Tag results identified MYB as a key direct target of ZNF217. Overall, our research highlights ZNF217 as a critical oncogene in AML and offers new insights into the transcriptional regulatory mechanisms at play in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/388657aaeaf8efe74b215605c5baaa2e335b1840" target='_blank'>
              Core transcriptional regulatory circuitry molecule ZNF217 promotes AML cell proliferation by up-regulating MYB
              </a>
            </td>
          <td>
            Bi Zhou, Fang Fang, Yongping Zhang, Zhiheng Li, Yixin Hu, Yan Li, Wanyan Jiao, Yumeng Wu, X. Wan, Ying Yang, Fenli Zhang, Ling Xu, Tongting Ji, Jian Pan, Shaoyan Hu
          </td>
          <td>2025-02-18</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Introduction Acute myeloid leukemia (AML) is a leading cause of mortality among Indian children and adults, driven by diverse genetic and epigenetic abnormalities. Limited access to genomic sequencing in India due to resource constraints has hindered a comprehensive understanding of prognostic factors specific to this population.
 Objectives This study aims to analyze publicly available genomic data using statistical and bioinformatics tools to identify key prognostic markers relevant to Indian AML patients.
 Materials and Methods The study utilized tumor/normal pair data from 200 adult de novo AML patients, obtained from The Cancer Genome Atlas (TCGA) and the Genotype-Tissue Expression database, analyzed using cBioPortal. Statistical and bioinformatics tools were employed to assess the impact of existing prognostic targets on disease response and to identify variables with clinical relevance and practical testing feasibility.
 Results Analysis of the TCGA-AML data set identified high-frequency gene mutations (≥ 10%) and well-defined cytogenetic subtypes, including t(8;21)(q22;q22), NPM1 mutations, and CEBPA mutations as key factors for future prognostic evaluation. These findings will contribute to the development of a prognostic scoring system using R programming in future.
 Conclusion This study offers insights into the cytogenetic and mutational landscape of AML in the Indian population, identifying critical genetic and cytogenetic markers with the potential to enhance prognostication, guide treatment strategies, and inform transplant decisions. Using R tools like limma and edgeR, differential expression analysis identified five key genes—NPM1, FLT3, IDH2, RUNX1, and STAG2—as significantly upregulated in AML. Notably, STAG2 emerges as a novel marker with potential prognostic significance, warranting validation in larger Indian cohorts. These findings may help uncover novel therapeutic targets not currently recognized in Western populations, paving the way for a more tailored and personalized approach to treatment in India. By leveraging global genomic databases, this research addresses regional gaps in knowledge. Future work should focus on validating these findings through large-scale studies in Indian cohorts to ensure their broader applicability and impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a9d42cdf5ba552b6ee663d661dbe857122a4db" target='_blank'>
              Analysis of TCGA (The Cancer Genome Atlas) Data for Prognosis, Risk Categorization, and Survival of AML Patients Using Bioinformatics
              </a>
            </td>
          <td>
            Bhavika Rishi, Shamsuz Zaman, Fouzia Siraj, Sree Vishmaya, Himanshu Dhanda, Manpreet Kaur, Sandeep Swain, Baseer Noor, S. Chaudhry, Ankur Jain, Prashant Prabhakar, Amitabh Singh, Aroonima Misra
          </td>
          <td>2025-02-24</td>
          <td>Indian Journal of Medical and Paediatric Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, M. Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>