<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-10 09:22:52 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>26</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) have demonstrated remarkable efficacy in treating non-small cell lung cancer (NSCLC), but acquired resistance greatly reduces efficacy and poses a significant challenge to patients. While numerous studies have investigated the mechanisms underlying EGFR-TKI resistance, its complexity and diversity make the existing understanding still incomplete. Traditional approaches frequently struggle to adequately reveal the process of drug resistance development through mean value analysis at the overall cellular level. In recent years, the rapid development of single-cell RNA sequencing technology has introduced a transformative method for analyzing gene expression changes within tumor cells at a single-cell resolution. It not only deepens our understanding of the tumor microenvironment and cellular heterogeneity associated with EGFR-TKI resistance but also identifies potential biomarkers of resistance. In this review, we highlight the critical role of single-cell RNA sequencing in lung cancer research, with a particular focus on its application to exploring the mechanisms of EGFR-TKI-acquired resistance in NSCLC. We emphasize its potential for elucidating the complexity of drug resistance mechanism and its promise in informing more precise and personalized treatment strategies. Ultimately, this approach aims to advance NSCLC treatment toward a new era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57a8423fcdeff9dbe3c23c8eb5bf687e8a733ce" target='_blank'>
              Single-Cell RNA Sequencing in Unraveling Acquired Resistance to EGFR-TKIs in Non-Small Cell Lung Cancer: New Perspectives
              </a>
            </td>
          <td>
            Lin Peng, Siyou Deng, Jinjie Li, Yujie Zhang, Li Zhang
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Synthetic lethality represents a novel paradigm in molecular targeted cancer therapy. In synthetic lethality, perturbation of one gene alone does not hinder cell viability, yet simultaneous perturbation of both genes results in a loss of cellular viability. The presence of gene mutations in cancer cells, as opposed to normal cells, provides an opportunity for targeted therapies that mimic the effects of the second genetic mutation, enabling selective eradication of cancer cells. Recent advances in high-throughput screening technologies, such as CRISPR-Cas9 and RNA interference, have significantly enhanced the identification of synthetic lethal interactions, expanding the potential targets for therapeutic intervention. Challenges in exploiting synthetic lethality for cancer treatment include the complexities of tumor biology, limited comprehension of synthetic lethal interactions, drug resistance, and impediments in screening and clinical translation. Emerging strategies, such as combination therapies and novel drug designs, are being developed to overcome these obstacles. By virtue of its selective lethality towards cancer cells bearing specific genetic alterations, targeting synthetic lethal genes holds the promise to provide wider therapeutic windows compared to traditional cytotoxic chemotherapy. This review describes the current state of synthetic lethality applications in cancer treatment, encompassing both biological and methodological perspectives. It highlights the latest advancements in synthetic lethality with emerging interventional strategies. Furthermore, it explores future directions for research and clinical implementation, aiming to refine and expand the therapeutic potential of synthetic lethality in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c9ad6ae81a9b46e74e1e6708428804d26e2ad" target='_blank'>
              Advances in Synthetic Lethality in Potential Oncology Therapeutic Approaches.
              </a>
            </td>
          <td>
            Feifei Yang, Huiyu Wang, Shule Fan, Huiran Qiu, Xiangzhi Li, Guangyao Shi, Zihao Li, Xiaotian Luan, Haigang Wu
          </td>
          <td>2025-01-30</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3eb15125d0d2e4a0c6a634c10b2bffbe96d5f8" target='_blank'>
              Epigenetic modifications and metabolic gene mutations drive resistance evolution in response to stimulatory antibiotics
              </a>
            </td>
          <td>
            Hui Lin, Donglin Wang, Qiaojuan Wang, Jie Mao, Lutong Yang, Yaohui Bai, Jiuhui Qu
          </td>
          <td>2025-01-16</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is marked by the proliferation of abnormal myeloid progenitor cells in the bone marrow and blood, leading to low cure rates despite new drug approvals from 2017 to 2018. Current therapies often fail due to the emergence of drug resistance mechanisms, such as those involving anti-apoptotic pathways and immune evasion, highlighting an urgent need for novel approaches to overcome these limitations. Programmed cell death (PCD) is crucial for tissue homeostasis, with PANoptosis-a form of PCD integrating pyroptosis, apoptosis, and necroptosis-recently identified. This process, regulated by the PANoptosome complex, could be key to overcoming AML drug resistance. Targeting multiple PCD pathways simultaneously may prove more effective than single-target therapies. Research suggests that disrupting anti-apoptotic mechanisms, such as those involving Bcl-2, can enhance drug sensitivity in AML. This study hypothesizes that PANoptosis-associated resistance genes (PARGs) play a critical role in AML drug resistance by modulating immune responses and offers a multi-faceted approach to tackle this challenge. Using RNA sequencing data from the Cancer Genome Atlas and Gene Expression Omnibus databases, we performed differential expression analysis to identify significantly dysregulated PARGs in AML. Regression analysis identified prognostic PARGs, bridging a key gap in understanding how these genes contribute to treatment resistance. We then verified their expression in AML cell lines and cell samples treated with cytarabine using RT-qPCR. Hierarchical clustering revealed distinct PARG expression patterns, and functional enrichment analysis highlighted their involvement in immune-related pathways. The combination of bioinformatics and experimental validation underscores how these genes may mediate immune modulation in drug resistance, providing a robust framework for further study. Our findings suggest that PARGs contribute to AML resistance by modulating immune responses and provide potential targets for therapeutic intervention. This study highlights the potential of targeting PARGs to improve treatment outcomes in AML. By analyzing the expression changes of these genes in response to standard clinical treatments, we provide a framework for developing multi-target therapeutic strategies that simultaneously disrupt multiple programmed cell death pathways. Such an approach directly addresses the limitations of current treatments by offering a method to enhance drug sensitivity and mitigate resistance, potentially improving survival rates. Our findings underscore the importance of a comprehensive understanding of PCD mechanisms and pave the way for innovative treatments that could significantly impact AML management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064037ccaeb8393d522fb1895074e0d8f9cf70c8" target='_blank'>
              Integrated Genomics Reveal Potential Resistance Mechanisms of PANoptosis-Associated Genes in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Cong Liang, Zhiqing Long, Mengjie Lei, Ran Ding, Mengke Chen
          </td>
          <td>2025-01-27</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All cancers are diseases of the genome, since the cancer cell genome typically consists of 10,000s of passenger alterations, 5-10 biologically relevant alterations, and 1-2 "actionable" alterations. Therefore, somatic mutations in cancer cells can have diagnostic, prognostic, and predictive value. Traditional methods are widely used for testing, such as immunohistochemistry, Sanger sequencing, and allele-specific PCR. However, due to the low throughput, these methods are focused exclusively on testing the most common mutations in target genes. The modern next generation sequencing (NGS) is a technology that enables precision oncology in its current form. ESCAT and ESMO Guidelines defined NGS for routine use in patients with advanced cancers such as non-squamous non-small cell lung cancer, prostate cancer, ovarian cancer, and cholangiocarcinoma. The high sensitivity of the NGS method allows it to be used to search for specific mutations in circulating tumor DNA in blood plasma and other body fluids. NGS testing has evolved from hotspot panels, actionable gene panels, and disease-specific panels to more comprehensive panels. The exome and whole genome sequencing approaches are just beginning to emerge, that is why panel-based testing remains most optimal in oncology practice. NGS is also widely used to identify new and rare mutations in cancer genes and detect inherited cancer mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207a8f97841f6a0acdc06c228c5f1967ea8ee827" target='_blank'>
              Application of Next-Generation Sequencing to Realize Principles of Precision Therapy in Management of Cancer Patients.
              </a>
            </td>
          <td>
            N. Khranovska, O. Gorbach, O. Skachkova, G. Klimnyuk
          </td>
          <td>2025-02-20</td>
          <td>Experimental oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The second and third most frequently diagnosed cancers worldwide are breast (2.3 million new cases) and colorectal (1.9 million new cases), respectively. Although advances in cancer therapies and early detection have improved the overall survival of patients, patients still develop resistance or cancer recurrence. Thus, the development of novel therapies that can affect multiple mechanisms of drug resistance and cell survival is ideal for the treatment of advanced and metastatic cancers. CHD1L is a novel oncogenic protein involved in regulating chromatin remodeling, DNA damage repair, epithelial-mesenchymal transition (EMT), and programmed cell death via PARthanatos. Herein, we assess in real-time how the CHD1L inhibitor (CHD1Li) OTI-611 modulates cell cycle progression in Colo678, SUM149PT, and SW620 cell lines. By utilizing a cell cycle reporter, we tracked the real-time cell cycle progression of cancer cells treated with OTI-611 alone and in combination with standard-of-care (SOC) therapies. Our results indicate that OTI-611 causes G1 phase cell cycle arrest through a CHD1L-mediated mechanism that regulates Cyclin D1 expression and localization. As a result of this mechanism, OTI-611 can reprogram the cell cycle effects of other antitumor agents to modulate and arrest cells in G1 when used in combination, including agents commonly known to arrest cells in the G2/M phase. Therefore, we conclude that OTI-611-induced G1 arrest represents a critical component of its unique mechanism of action, contributing significantly to its anticancer activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775d92956dd4575de627e0b27e219c4a04b6d817" target='_blank'>
              Targeted Inhibition of CHD1L by OTI-611 Reprograms Chemotherapy and Targeted Therapy-Induced Cell Cycle Arrest and Suppresses Proliferation to Produce Synergistic Antitumor Effects in Breast and Colorectal Cancer
              </a>
            </td>
          <td>
            Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-02-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, Manoj Kumar Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="High grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy, killing more than 9,000 women each year in the United States alone. Nearly 80% of patients with HGSOC tumors will experience recurrence within 5 years, but little is known about the mechanisms that drive this process. Intratumor heterogeneity is believed to be a key feature of recurrence and resistance in HGSOC tumors, with early studies reporting diverse and complex mechanisms of the seeding of metastatic sites, including metastasis reseeding the primary tumor. Few studies have investigated temporal changes to clonality and structural variants through disease recurrence and the development of chemoresistance as surgical debulking is not frequently performed at this later stage. We performed multi-omic profiling in paired chemo-naive and chemoresistant tumors from 32 HGSOC patients to investigate the mutational and genomic landscape of disease progression. Somatic mutation profiles were largely conserved through disease progression. Mutational burdens did not significantly differ across recurrence but were driven by homologous recombination repair deficiency status. Clonal composition and dynamics were measured through variant allele frequency alterations as tumors progressed from primary chemo-naïve to recurrent chemo-resistant tumors. A novel candidate driver gene, MDC1, from the homologous recombination repair pathway, was significantly mutated and over-represented in patients with homologous recombination proficient tumors, with somatic mutations clustered in a single exon. Tumor evolution and phylogeny revealed that few changes in clonal abundance and complexity occur across the disease course in these patients. Taking structural variants into account, homologous recombination repair proficient (HRP) tumors tend to be polyclonal while homologous recombination repair deficient (HRD) tumors tend to be monoclonal, accompanied by a longer progression-free survival than the HRP patients. Three distinct classes of tumors were identified by structural variant signature analysis: tumors defined by DNA losses, tumors defined by DNA gains, and tumors defined by copy number neutral changes, which were largely defined by HRD status. Each class displayed distinct regions of the chromosome that were frequently affected by large scale SV events (>5Mb). Although no regions were frequently altered in recurrent tumors, GO analysis revealed that recurrent tumors have a significantly reduced immune response, which was not seen in the primary tumors. Ultra long read sequencing validated a majority of the SVs identified in short read sequencing and identified additional SVs undetected by short reads. These analyses identify that the phenotype of high grade serous ovarian tumors as defined by mutation and clonality profiles is established early in disease development and remain largely unchanged through chemotherapy and recurrence. This, when considered with the significant inter-patient heterogeneity identified in HGSOC, demonstrates the need for personalized therapies based on tumor profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd64f2509541e7332b94c3a04278ee5d7e930fb" target='_blank'>
              Molecular principles underlying aggressive cancers
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-02-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (AML) exhibits distinct genetic characteristics, including unique driver alterations and mutations with prognostic and therapeutic significance. Emerging rare, recurrent genetic abnormalities and their associations with outcomes emphasize the need for high-throughput molecular diagnostic tools. Whole genome sequencing (WGS) reliably detects key AML biomarkers such as structural variants, mutations, and copy number alterations. Whole transcriptome sequencing (WTS) complements WGS by uncovering oncogene expression patterns, allele-specific expression, and gene expression signatures. In this study, we describe an integrated WGS and WTS clinical workflow for routine pediatric AML diagnosis and present a systematic evaluation of its application compared to conventional cytogenetics and standard molecular diagnostic methods. Our findings demonstrate that the integrated WGS and WTS (iWGS-WTS) approach improves the identification of clinically relevant genetic alterations, enhancing precise disease classification and risk assessment. Moreover, with advancements in workflow and bioinformatics pipelines, the testing turnaround time can be optimized to meet the demands of clinical decision-making, positioning iWGS-WTS as a practical and superior alternative to traditional diagnostic methods in pediatric AML management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de0f3555bd4b9f7c58df5cd98e038c6f7a42e14" target='_blank'>
              Integrated Whole Genome and Transcriptome Sequencing as a Framework for Pediatric and Adolescent AML Diagnosis and Risk Assessment
              </a>
            </td>
          <td>
            Lu Wang, Rebecca Voss, Victor Pastor, Maria Cardenas, Priyadarshini Kumar, J. Maciaszek, Maria Namwanje, Jingqun Ma, Jennifer L. Neary, Meiling Jin, Masayuki Umeda, M. Wilkinson, D. Payne-Turner, Mohammad Eldomery, Jingqun Ma, J. Gu, James Dalton, Samantha Melton, Yen-Chun Liu, S. Foy, Michael C. Rusch, David A. Wheeler, Jinghui Zhang, Kim E. Nichols, Seth E. Karol, Hiroto Inaba, Raul C Ribeiro, J. Rubnitz, J. Klco
          </td>
          <td>2025-01-22</td>
          <td>Research Square</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) are a type of RNA with a length of more than 200 nucleotides. They do not encode proteins but are crucial in regulating gene expression and affecting the malignant biological behavior of cancer. Small nucleolar RNA host gene 10 (SNHG10) is a novel lncRNA that plays a regulatory role in many malignant tumors. Several recent studies have shown that SNHG10 is aberrantly expressed in various forms of cancer. This instability is closely related to important tumorigenic processes, such as cell proliferation, migration, invasion, epithelial-mesenchymal transition (EMT), and chemotherapy resistance. SNHG10 has been reported to play a role through a variety of molecular mechanisms, including serving as a competing endogenous RNA (ceRNA), regulating epigenetic processes, and affecting immune responses and tumor microenvironment. Furthermore, SNHG10 is involved in metabolic reprogramming, immune evasion, and chromatin remodeling, highlighting its diverse roles in tumor biology. Due to the specificity and selectivity of its expression level, the potential of SNHG10 as a diagnostic biomarker and therapeutic target has attracted significant attention, and its correlation with the prognosis and treatment of various tumor types is of great significance. This review focuses on the biological function and molecular mechanism of SNHG10 and its relationship with various malignant tumors. In addition, this review highlights the potential of SNHG10 to improve precision oncology and develop novel cancer therapies by investigating its upstream regulators, downstream targets, and interactions with nuclear and cytoplasmic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d77495b125c5aaab4dcf465d06c71c35fe30e9b" target='_blank'>
              SNHG10: A Novel Long Non-coding RNA with Multifaceted Roles in Human Cancers.
              </a>
            </td>
          <td>
            Haodong He, Jingjie Yang, Yan Zhou, Lihan Chen, Chuyuan Liao, Ting Gong, Tongtong Li, Haoran Liu, Yunfei Pan, Pengbo Zhang, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2025-03-05</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 256


 Background:
 Resistance to Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by complex genomic alterations. The evolution of lineage state transitions from adenocarcinomas to neuroendocrine prostate cancer (NEPC) has also been shown to drive treatment resistance and poor survival. Identifying the timing and association of genomic mutations with lineage state transitions has been limited by the need for serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data and mPC lineage states with a novel circulating tumor cell (CTC) RNA sequencing methodology.
 Methods:
 We collected 273 samples from 117 unique patients with mPC in a prospective biomarker trial. CTCs were purified via immunomagnetic capture on an automated microfluidic technology and analyzed via RNA-seq. A subset of patients had a clinical grade genetic test performed on a contemporary metastatic tissue biopsy. We compared CTC gene expression for lineage states (luminal A, luminal B, NEPC) with somatic gene mutation subtypes known to confer resistance to ARPIs (AR, p53, RB, PTEN, HRR genes) and overall survival (OS).
 Results:
 Single sample pathway analysis of high CTC purity samples identified four transcriptional phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (median OS not reached), patients with LumA and LP phenotypes had similar survival, patients with LumB and NE CTC phenotypes had shorter survival (LumB: median OS 6m, HR 9.1 [3.8-21.8], p<0.0001, NE: median OS 3.7m, HR 11.8 [2.4-57.2], p=0.0019). AR alterations were found in samples with prior ARPI exposure and at similar frequencies across all CTC phenotypes but NE where they were absent. RB mutations were enriched in the unfavorable CTC phenotypes (LumB, NE; p=0.0288). Integrating CTC phenotype with presence or absence of at least one high risk genomic alteration (AR, RB, TP53, PTEN), patients with favorable CTC phenotype (Low burden, LP and LumA) and no high risk alterations had the longest median survival (median OS NR), followed by patients with favorable phenotype but a high risk alteration (median OS 12.7m), patients with an unfavorable phenotype (LumB, NE) but no high risk alteration (median OS 8m), and unfavorable phenotype with a high risk alteration (median OS 4m).
 Conclusions:
 Somatic mutations in mCRPC influence lineage state acquisitions and treatment resistance. Lineage state transitions themselves are associated with poor outcomes and decreased survival which is worsened in the presence of high-risk genetic mutations including RB. This data presents potentially targetable patient populations that would benefit from treatment intensification and early disease monitoring for more aggressive mCRPC subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0203e20c6bc4f26349dbeefa5213e0b374637b7e" target='_blank'>
              Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer
              </a>
            </td>
          <td>
            Slim Mzoughi, Megan Schwarz, Xuedi Wang, D. Demircioglu, Gulay Ulukaya, Kevin Mohammed, Habiba Zorgati, Denis Torre, Lewis E. Tomalin, Federico Di Tullio, Carlos Company, Yuliia Dramaretska, M. Leushacke, Bruno Giotti, T. Lannagan, D. Lozano-Ojalvo, P. Karras, Peter B Vermeulen, Dan Hasson, Robert Sebra, A. Tsankov, Owen J. Sansom, J. Marine, Nick Barker, G. Gargiulo, Ernesto Guccione
          </td>
          <td>2025-02-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance.Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation.We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation.This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In recent years, significant advancements in biochemistry, materials science, engineering, and computer-aided testing have driven the development of high-throughput tools for profiling genetic information. Single-cell RNA sequencing (scRNA-seq) technologies have established themselves as key tools for dissecting genetic sequences at the level of single cells. These technologies reveal cellular diversity and allow for the exploration of cell states and transformations with exceptional resolution. Unlike bulk sequencing, which provides population-averaged data, scRNA-seq can detect cell subtypes or gene expression variations that would otherwise be overlooked. However, a key limitation of scRNA-seq is its inability to preserve spatial information about the RNA transcriptome, as the process requires tissue dissociation and cell isolation. Spatial transcriptomics is a pivotal advancement in medical biotechnology, facilitating the identification of molecules such as RNA in their original spatial context within tissue sections at the single-cell level. This capability offers a substantial advantage over traditional single-cell sequencing techniques. Spatial transcriptomics offers valuable insights into a wide range of biomedical fields, including neurology, embryology, cancer research, immunology, and histology. This review highlights single-cell sequencing approaches, recent technological developments, associated challenges, various techniques for expression data analysis, and their applications in disciplines such as cancer research, microbiology, neuroscience, reproductive biology, and immunology. It highlights the critical role of single-cell sequencing tools in characterizing the dynamic nature of individual cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8242b8eb0f45c6f5b5cf16b8ec7b8ce82d36e19b" target='_blank'>
              Advancements in single-cell RNA sequencing and spatial transcriptomics: transforming biomedical research
              </a>
            </td>
          <td>
            Getnet Molla Desta, A. Birhanu
          </td>
          <td>2025-02-05</td>
          <td>Acta Biochimica Polonica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here we show that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells display increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence is rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimics the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slows growth of multiple PAX3-FOXO1+ tumor xenograft models, but not fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50231dc8772bfbbe71ed944e599efde5187dc0b" target='_blank'>
              PAX3-FOXO1 drives targetable cell state-dependent metabolic vulnerabilities in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Jacob A. Boyer, Angela M. Montero, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, Andrew L Kung, Daoqi You, Filemon Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chimeric Antigen Receptor T (CAR-T) cells offer a promising strategy for cancer treatment. These CAR-T cells are either autologous or allogeneic T cells that are genetically modified to express a chimeric antigen receptor targeting a specific tumor antigen. Ongoing research aims to optimize the CAR-T cell efficacy, including strategies to modulate their metabolism. One such approach involves inducing transgene expression by activating the GCN2 kinase signaling pathway through dietary deprivation of an essential amino acid. In this study, we investigated the general impact of a 6-hour leucine deprivation on primary activated human T cells using RNA-seq technology. Our analysis identified 3,431 differentially expressed genes between T cells cultured in regular medium and those cultured in leucine-deprived medium. Gene Set Enrichment Analysis revealed that “TNFα signaling via NFκB”, “interferon-γ response”, and “unfolded protein response” gene sets were positively enriched, while “mTORC1 signaling”, “Myc targets”, and “oxidative phosphorylation” gene sets were negatively enriched. To further evaluate the involvement of GCN2 kinase in regulating the differential gene expression during the 6-hour leucine deprivation, T cells were cultured with or without a GCN2 inhibitor. We found that 59% of the differentially expressed genes in our dataset were dependent on the kinase GCN2 (n = 2028), with 1,140 up-regulated and 888 down-regulated genes. These findings suggest a promising strategy to enhance CAR-T cell efficacy by combining short amino acid starvation with transient overexpression of a target gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7518ff0dd3a5481cf6365679f81c8157f814beb" target='_blank'>
              Transcriptomic analysis of human primary T cells after short-term leucine-deprivation and evaluation of kinase GCN2’s role in regulating differential gene expression
              </a>
            </td>
          <td>
            Aurore Dougé, Gwendal Cueff, Céline Keime, V. Carraro, C. Jousse, Paul Rouzaire, A. Bruhat
          </td>
          <td>2025-02-18</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Adoptive immune cell therapies, such as CAR T, have revolutionized how we can approach cancer therapy, however challenges remain to increase their efficacy, response rates, and resistance to dysfunction. The systematic investigation of gene programs governing human T cell function would be advantageous for reprogramming cell products for clinical benefit. Recent studies have demonstrated the power of CRISPR-based forward genetic screens in primary human T cells to nominate candidate perturbations for enhanced T cell function, however combinations of perturbations will likely be necessary to overcome the most challenging tumors. To enable the systematic and high-throughput interrogation of combinations of gain and loss-of-function perturbations, we have developed a bidirectional CRISPR activation and interference (CRISPRai) system, where pairs of genes can be simultaneously activated and repressed in primary human T cells. Leveraging hits from previous genome-wide CRISPRa and CRISPRi screens, we constructed a library of >66,000 sgRNA combinations and screened for regulators of acute T cell stimulation responses, including cytokine production and proliferation. Reassuringly, these screens identified expected gene epistasis interactions (e.g. rescue of PLCG1 repression by PLCG2 activation), while also discovering novel genetic interactions, providing mechanistic insights into the regulatory networks governing stimulation response. To further identify gene “building blocks” for evaluation as combinations, we completed genome-wide CRISPRa and CRISPRi screens for regulators of human CD8+ CAR T cell exhaustion. These screens identified known potential enhancements (e.g. gain of BATF, loss of MED12), and also uncovered novel regulators of T cell function, with CRISPRa discovering genes not normally expressed in T cells but whose overexpression could cause resistance to exhaustion by synthetic mechanisms. Gene expression programs controlled by perturbation of key regulators were mapped by pairing CRISPR with single-cell RNA-seq (Perturb-seq) and arrayed functional assays. Finally, a top gain-of-function candidate was evaluated for in vivo tumor control, using an antigen-inducible Synthetic Notch system. Together, this work demonstrates how advanced forward genetics approaches can discover and elucidate both natural and synthetic programs that regulate T cell states, and the nomination of perturbations to encode in logic-gated synthetic biology designs for cell therapies.
 Citation Format: Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole Roybal, Brian Shy, Alexander Marson. Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/611724b4911f171cfc8e55580dc1cddab6f13349" target='_blank'>
              Early tolerance and late persistence as alternative drug responses in cancer
              </a>
            </td>
          <td>
            S. Punzi, D. Cittaro, Guido Gatti, Gemma Crupi, O. Botrugno, Antonino Alex Cartalemi, Alon Gutfreund, C. Oneto, Valentina Giansanti, Chiara Battistini, Giovanni Santacatterina, Lucrezia Patruno, Ilaria Villanti, Martina Palumbo, D. Laverty, Francesca Giannese, Alex Graudenzi, G. Caravagna, M. Antoniotti, Zachary D. Nagel, Ugo Cavallaro, L. Lanfrancone, T.A. Yap, Giulio Draetta, Nathalie Q Balaban, Giovanni Tonon
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Lung cancer exhibits substantial inter- and intra-tumor heterogeneity, with features that present significant challenges in advancing biomarker discovery and the development of targeted therapeutics. To fill this gap, we employed single-cell RNA sequencing (scRNA-seq) and advanced bioinformatics tools to evaluate the transcriptomic heterogeneity of immortalized, non-transformed (BEAS2B) and transformed (H460) lung epithelial cell lines and their responses to carcinogen challenge. Gene expression profiles resolved four primary clusters further discretized into unique subclusters based on genetic signatures and phenotypic profiles. Profiles of long non-coding RNAs (lncRNAs) identified microRNA host genes, antisense RNA genes, divergent transcript, and long intergenic non-coding RNAs as contributors to cellular heterogeneity. These findings indicate that distinct patterns of gene expression, remarkably in lncRNAs, define cellular heterogeneity in non-transformed versus transformed cells. These features can be exploited for the development of therapies directed at specific cell subpopulations in precancerous lesions and within lung tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/233c01bff81dc65b1be1e7c0ebbee44d4a90ef36" target='_blank'>
              Single-Cell RNA Sequencing Reveals Extensive Heterogeneity and Unique Gene Trajectories in Non-Transformed and Transformed Human Lung Epithelial Cells: Insights into the Role of LncRNAs in Tumor Heterogeneity
              </a>
            </td>
          <td>
            Sokviseth Moeng, Andres D. Chamorro-Parejo, Minsun S. Jeon, James J. Cai, Kenneth S Ramos
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Alternative end-joining (alt-EJ) is an error-prone DNA repair pathway that cancer cells deficient in homologous recombination rely on, making them vulnerable to synthetic lethality via inhibition of poly(ADP-ribose) polymerase (PARP). Targeting alt-EJ effector DNA polymerase theta (POLθ), which synergizes with PARP inhibitors and can overcome resistance, is of significant preclinical and clinical interest. However, the transcriptional regulation of alt-EJ and its interactions with processes driving cancer progression remain poorly understood. Here, we show that alt-EJ is suppressed by hypoxia while positively associated with MYC (myelocytomatosis oncogene) transcriptional activity. Hypoxia reduces PARP1 and POLQ expression, decreases MYC binding at their promoters, and lowers PARylation and alt-EJ-mediated DNA repair in cancer cells. Tumors with HIF1A mutations overexpress the alt-EJ gene signature. Inhibition of hypoxia-inducible factor 1α or HIF1A expression depletion, combined with PARP or POLθ inhibition, synergistically reduces the colony-forming capacity of cancer cells. Deep learning reveals the anticorrelation between alt-EJ and hypoxia across regions in tumor images, and the predictions for these and MYC activity achieve area under the curve values between 0.70 and 0.86. These findings further highlight the critical role of hypoxia in modulating DNA repair and present a strategy for predicting and improving outcomes centered on targeting alt-EJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2d05c118cddd3cc2623e904498c569be8cdf72" target='_blank'>
              Harnessing transcriptional regulation of alternative end-joining to predict cancer treatment
              </a>
            </td>
          <td>
            Roderic Espín, Ferran Medina-Jover, Javier Sigüenza-Andrade, Sònia Farran-Matas, Francesca Mateo, Agnès Figueras, Rosario T Sanz, Guillermo P Vicent, Arzoo Shabbir, Lara Ruiz-Auladell, Emilio Racionero-Andrés, Irene García, A. Baiges, Lídia Franco-Luzón, Adrián Martínez-Tébar, Miguel Angel Pardo-Cea, M. Martínez-Iniesta, Xieng C. Wang, E. Cuyás, Javier A. Menendez, Marta Lopez-Cerda, Purificacion Muñoz, Ivonne Richaud, Angel Raya, Isabel Fabregat, Alberto Villanueva, Xènia Serrat, Julián Cerón, M. Alemany, Inés Guix, A. Herencia-Ropero, V. Serra, Rehna Krishnan, Karim Mekhail, Razqallah Hakem, Jordi Bruna, M. Barcellos-Hoff, Francesc Viñals, Á. Aytés, M. Pujana
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f873b54c9840f07bddb284c4e229ec7800362c" target='_blank'>
              Genetic advancements in breast cancer treatment: a review
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Sadaf Sedaghatshoar, Homaira Arian, Milad Mokarami, Fatemeh Habibi, Fatemeh Bamarinejad
          </td>
          <td>2025-02-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a major cause of cancer-related mortality worldwide, with a significant impact on public health. Current treatment options include surgery, chemotherapy, radiotherapy, molecular-targeted therapy, and immunotherapy. Despite advancements in these therapeutic modalities, resistance remains a significant challenge, often leading to treatment failure, poor progression-free survival, and cancer recurrence. Mechanisms of resistance in CRC are multifaceted, involving genetic mutations, epigenetic alterations, tumor heterogeneity, and the tumor microenvironment. Understanding these mechanisms at the molecular level is crucial for identifying novel therapeutic targets and developing strategies to overcome resistance. This review provides an overview of the diverse mechanisms driving drug resistance in sporadic CRC and discusses strategies currently under investigation to counteract this resistance. Several promising strategies are being explored, including targeting drug transport, key signaling pathways, DNA damage response, cell death pathways, epigenetic modifications, cancer stem cells, and the tumor microenvironment. The integration of emerging therapeutic approaches that target resistance mechanisms aims to enhance the efficacy of current CRC treatments and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f0190dfc671312c21a6e2fa1ac0dcb0cf54adc6" target='_blank'>
              Mechanisms and Strategies to Overcome Drug Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Jennifer Haynes, Prasath Manogaran
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Systematic screening of gain- or loss-of-function genetic perturbations can be used to characterize the genetic dependencies and mechanisms of regulation for essentially any cellular process of interest. These experiments typically involve profiling from a pool of single gene perturbations and how each genetic perturbation affects the relative cell fitness. When applied in the context of drug efficacy studies, often called chemo-genetic profiling, these methods should be effective at identifying drug mechanisms of action. Unfortunately, fitness-based chemo-genetic profiling studies are ineffective at identifying all components of a drug response. For instance, these studies generally fail to identify which genes regulate drug-induced cell death. Several issues contribute to obscuring death regulation in fitness-based screens, including the confounding effects of proliferation rate variation, variation in the drug-induced coordination between growth and death, and, in some cases, the inability to separate DNA from live and dead cells. MEDUSA is an analytical method for identifying death-regulatory genes in conventional chemo-genetic profiling data. It works by using computational simulations to estimate the growth and death rates that created an observed fitness profile rather than scoring fitness itself. Effective use of the method depends on optimal tittering of experimental conditions, including the drug dose, starting population size, and length of the assay. This manuscript will describe how to set up a chemo-genetic profiling study for MEDUSA-based analysis, and we will demonstrate how to use the method to quantify death rates in chemo-genetic profiling data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ce7ec0649fbc5d3e5e12bff74054318a2f6bce" target='_blank'>
              MEDUSA for Identifying Death Regulatory Genes in Chemo-genetic Profiling Data.
              </a>
            </td>
          <td>
            Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-02-07</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Comparative genomics has emerged as a pivotal methodology for elucidating genetic variations in microbial studies. However, conventional analytical approaches for diploid and polyploid microorganisms have demonstrated limited efficacy in discriminating new mutations from background heterozygosity. This study presents an innovative microbial comparative genomics framework adapted from tumor-normal sequencing methodology. Our approach establishes the original strain as the “normal sample” and the derived strain as the “tumor sample”, enabling precise identification of new mutations (“somatic variants”) while filtering pre-existing heterozygous sites (“germline variations”). The analytical pipeline also includes assessment of loss-of-heterozygosity (LOH) events and genome-wide detection of copy number variations (CNVs) with resolution to identify both regional CNV and whole-chromosome aneuploidy through integrated CNV and variant allele frequency analysis. We validated this framework using diploid Saccharomyces cerevisiae strains before successfully extending its application to Kluyveromyces marxianus, Candida spp., and Hortaea werneckii, encompassing haploid, diploid, polyploid, and aneuploid states. The methodology revealed previously undetected variations across experimental evolution studies, demonstrating superior resolution compared to conventional approaches. This adaptable platform establishes a new paradigm for microbial genome studies, particularly for organisms with diploid or polyploid states where traditional comparative genomics methods prove inadequate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The integration of whole-genome sequencing (WGS) and whole-transcriptome sequencing (WTS) has revolutionized cancer diagnostics, enabling comprehensive molecular profiling of tumours. While WGS uncovers genomic alterations such as single nucleotide variants (SNVs), structural variants (SVs), and copy number changes, WTS provides complementary insights into transcriptomic changes, including gene expression levels, alternative splicing, and fusion events. Together, these technologies offer unparalleled potential to guide precision oncology by identifying actionable biomarkers and stratifying patients for targeted therapies or clinical trials. However, the integration of WGS and WTS data remains a significant bioinformatics challenge due to biological variability, tumour-specific complexities, and the lack of harmonized reference datasets. To address this, we developed RNAsum, an open-source tool designed to combine and interpret WGS and WTS data from individual cancer patient samples. RNAsum compares patient data against curated cohorts from The Cancer Genome Atlas (TCGA) and integrates quantitative expression data with genomic insights to enhance diagnostic accuracy, validate genomic findings, and prioritize clinically relevant alterations. We demonstrate the utility of RNAsum through case studies, showcasing its ability to confirm clinically reportable variants, refine diagnoses, and identify novel therapeutic targets. By bridging the gap between genome and transcriptome analyses, RNAsum represents a step forward in the routine clinical application of multi-omics for personalized cancer care. The tool is freely available as an R package on GitHub, supporting accessibility and scalability for diverse research and clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82fa64d9bf4f88f6a630cd910ec88f706feaf7b1" target='_blank'>
              RNAsum: A tool for personalised genome and transcriptome interpretation for improved cancer diagnostics
              </a>
            </td>
          <td>
            S. Kanwal, J. Marzec, J. Vissers, P. Diakumis, L. Varghese, L. Tork, O. Hofmann, S. Grimmond, S. J. Luen
          </td>
          <td>2025-01-12</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bdfee01c97ef1e7e57c28ea058914128289212e" target='_blank'>
              Establishment and transcriptomic characteristics of radio-resistant meningioma cell lines.
              </a>
            </td>
          <td>
            Jinxiu Yu, Leihao Ren, Tianqi Wu, Lingyang Hua, Daijun Wang, Yang Wang, Qing Xie, Jiaojiao Deng, Ye Gong
          </td>
          <td>2025-02-28</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21cbb2f5aa4a29eac0f638906ce12441220483a" target='_blank'>
              Deciphering single-cell genomic architecture: insights into cellular heterogeneity and regulatory dynamics
              </a>
            </td>
          <td>
            Byunghee Kang, Hyeonji Lee, Tae-Young Roh
          </td>
          <td>2025-02-11</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis.We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment.Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis.This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Denggang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by expanding blood cell clones carrying somatic mutations in healthy aged individuals and is associated with various age-related diseases and all-cause mortality. While CH mutations affect diverse genes associated with myeloid malignancies, their mechanisms of expansion and disease associations remain poorly understood. We investigate the relationship between clonal fitness and clinical outcomes by integrating data from three longitudinal aging cohorts (n=713, observations=2,341). We demonstrate pathway-specific fitness advantage and clonal composition influence clonal dynamics. Further, the timing of mutation acquisition is necessary to determine the extent of clonal expansion reached during the host individual’s lifetime. We introduce MACS120, a metric combining mutation context, timing, and variant fitness to predict future clonal growth, outperforming traditional variant allele frequency measurements in predicting clinical outcomes. Our unified analytical framework enables standardized clonal dynamics inference across cohorts, advancing our ability to predict and potentially intervene in CH-related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df776ea20c82ef08bfb6dc73e82dc94bd703a7e7" target='_blank'>
              Clinical progression of clonal hematopoiesis is determined by a combination of mutation timing, fitness, and clonal structure
              </a>
            </td>
          <td>
            Eric Latorre-Crespo, Neil A. Robertson, E. Kosebent, Louise Macgillivray, Lee Murphy, M. Uddin, Eric Whitsel, Michael C. Honigberg, Alex Bick, A. Reiner, Valeria Orru, Michele Marongiu, Francesco Cucca, E. Fiorillo, I. Deary, Sarah Harris, Simon Cox, Riccardo Marioni, Linus Schumacher, Tamir Chandra, Kristina Kirschner
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) arises primarily through neuroendocrine transdifferentiation (NEtD) as an adaptive mechanism of therapeutic resistance. Models to define the functional effects of putative drivers of this process on androgen receptor (AR) signaling and NE cancer lineage programs are lacking. We adapted a genetically defined strategy from the field of cellular reprogramming to directly convert AR-active prostate cancer (ARPC) to AR-independent NEPC using candidate factors. We delineated critical roles of the pioneer factors ASCL1 and NeuroD1 in NEtD and uncovered their abilities to silence AR expression and signaling by remodeling chromatin at the somatically acquired AR enhancer and global AR binding sites with enhancer activity. We also elucidated the dynamic temporal changes in the transcriptomic and epigenomic landscapes of cells undergoing acute lineage conversion from ARPC to NEPC which should inform future therapeutic development. Further, we distinguished the activities of ASCL1 and NeuroD1 from the inactivation of RE-1 silencing transcription factor (REST), a master suppressor of a major neuronal gene program, in establishing a NEPC lineage state and in modulating the expression of genes associated with major histocompatibility complex class I (MHC I) antigen processing and presentation. These findings provide important, clinically relevant insights into the biological processes driving NEtD of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b450ad259d614ef1f0c3aeece63e1bbfcda82938" target='_blank'>
              Defined cellular reprogramming of androgen receptor-active prostate cancer to neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Shan Li, Kai Song, Huiyun Sun, Yong Tao, Arthur Huang, Vipul Bhatia, B. Hanratty, Radhika A Patel, Henry W. Long, C. Morrissey, Michael C Haffner, Peter S. Nelson, Thomas G. Graeber, John K. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 130


 Background:
 Chemotherapy regimens are selected based on information about the primary organ or its histological type and are administered based on the results of previous clinical trials. Even now, as cancer genomic research has progressed and precision medicine, which optimizes therapeutic interventions based on tumor genome profiling using next-generation sequencing(NGS), has become standard treatment, cytotoxic anticancer drugs still remain key drugs. We examined the real-world database of genomic and clinical information established by the national central datacenter, the Center for Cancer Genomics and Advanced Therapy (C-CAT) to systematically analyze the effects of mutations across multiple cancers and therapies.
 Methods:
 This study delineated the association between genetic mutations and the therapeutic efficacy of cytotoxic chemotherapy in advanced solid tumors in a real-world Japanese clinical context. We incorporated data from 15,474 patients enrolled in the C-CAT database between June 2019 and June 2022. We analyzed the overall response and time to next treatment (TNT) for genetic alterations and five categories of cytotoxic anticancer drugs in gastrointestinal cancers.
 Results:
 In the C-CAT data, gastrointestinal cancers were frequently registered, with colorectal cancer being the most common, followed by pancreatic cancer, esophageal/gastric cancer, and biliary tract cancer. For platinum-based drugs, in colorectal cancer, the presence of gene X was associated with not only a higher response rate but also a longer TNT (hazard ratio 0.82, p<0.001). In the case of colorectal cancer and pancreatic cancer with KRAS gene mutations, the ORR was low, and the TNT was also short in the case of pancreatic cancer (hazard ratio 1.33, p<0.001). Interestingly, when BRCA2 genes are mutant, TNT tended to be longer in many organs, suggesting that platinum-based drugs are effective across organs in carcinomas with BRCA2 mutations. Additionally, gene X showed differences in efficacy not only for platinum but also for antimetabolites and topoisomerase inhibitors. Moreover, KRAS mutation was observed in almost all pancreatic cancers and was a treatment-resistant mechanism for all platinum drugs and antimetabolites.
 Conclusions:
 This study demonstrated that the effects of cytotoxic anticancer drugs differ depending on genetic mutations. When determining chemotherapy regimens, considering not only the organ but also genetic mutations may enable more efficient drug selection.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d0f28bc90a1314a9a663e72dc465d8d8de56df" target='_blank'>
              Genetic mutations and cytotoxic chemotherapy response for advanced solid tumors: A detailed analysis using the C-CAT database in Japan.
              </a>
            </td>
          <td>
            N. Ishibashi, T. Kamatani, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb087ff58a5d637f585f09e70365a21011c3f17" target='_blank'>
              The RNA-binding protein LARP6 regulates the alternative splicing of related genes in MDA-MB-231 cells
              </a>
            </td>
          <td>
            Li Guo, Yaobang Liu, Shuxun Yan, Hong Li, Kai Zhang, Jinping Li
          </td>
          <td>2025-03-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Altered levels of reactive oxygen species (ROS) are recognized as one of the key factors in mediating tumor cell survival in the tissue microenvironment, where they play a role in the initiation, progression and recurrence/relapse of colorectal cancer (CRC). Tumor cells can adapt to oxidative stress (OS) using genetic or metabolic reprogramming in the long or short term. In addition, tumor cells defend themselves through positive regulation of antioxidant molecules, enhancing ROS-driven proliferation. Balanced oxidative eustress levels can influence chemotherapy resistance, allowing tumor cells to survive treatment. Secondary effects of chemotherapy include increased ROS production and redox stress, which can kill cancer cells and eliminate drug resistance. Anticancer treatments based on manipulating ROS levels could represent the gold standard in CRC therapy. Therefore, exploring the modulation of the response to OS in deregulated signaling pathways may lead to the development of new personalized CRC treatments to overcome therapy resistance. In this review, we explore the role of ROS in the initiation and progression of CRC and their diagnostic implications as biomarkers of disease. Furthermore, we focused on the involvement of ROS in different CRC therapeutic options, such as surgery, radiotherapy, theranostic imaging, chemotherapy and immunotherapy and other precision medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a640d71941f73c58db088f0e2e9e726b55b6048" target='_blank'>
              The Role of Reactive Oxygen Species in Colorectal Cancer Initiation and Progression: Perspectives on Theranostic Approaches
              </a>
            </td>
          <td>
            Teresa Catalano, F. Selvaggi, Roberto Cotellese, G. Aceto
          </td>
          <td>2025-02-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Bruton's tyrosine kinase (BTK) inhibitors are effective for the treatment of chronic lymphocytic leukemia (CLL) due to BTK's role in B cell survival and proliferation. Treatment resistance is most commonly caused by the emergence of the hallmark BTK C481S mutation that inhibits drug binding. In this study, we aimed to investigate cancer subclones harboring a BTK C481S mutation and identify cells with co-occurring CLL driver mutations. In addition, we sought to determine whether BTK-mutated subclones exhibit distinct transcriptomic behavior when compared to other cancer subclones. To achieve these goals, we use scBayes, which integrates bulk DNA sequencing and single-cell RNA sequencing (scRNA-seq) data to genotype individual cells for subclone-defining mutations. While the most common approach for scRNA-seq includes short-read sequencing, transcript coverage is limited due to the vast majority of the reads being concentrated at the priming end of the transcript. Here, we utilized MAS-seq, a long-read scRNA-seq technology, to substantially increase transcript coverage and expand the set of informative mutations to link cells to cancer subclones in six CLL patients who acquired BTK C481S mutations during BTK inhibitor treatment. In two patients who developed two independent BTK-mutated subclones, we found that most BTK-mutated cells have an additional CLL-driver gene mutation. When examining subclone-specific gene expression, we found that in one patient, BTK-mutated subclones are transcriptionally distinct from the rest of the malignant B cell population with an overexpression of CLL-relevant genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f68c71be9a5f4d5ac85f15c79176d7904112ca" target='_blank'>
              Long-read single-cell RNA sequencing enables the study of cancer subclone-specific genotypes and phenotypes in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            Gage S. Black, Xiaomeng Huang, Yi Qiao, Philip Moos, Deepa Sampath, Deborah M. Stephens, J. Woyach, Gabor T. Marth
          </td>
          <td>2025-02-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The current review delves into the transformative role of precision medicine in addressing Colorectal Cancer [CRC], a pressing global health challenge. It examines closely signalling pathways, genetic and epigenetic modifications, and microsatellite in-stability. The primary focus is on elucidating biomarkers revolutionizing CRC diagnosis and treatment. Genetic biomarkers encompass non-coding RNA, epigenetic markers, TP53 mutations, and KRAS, NRAS, and BRAF gene alterations. Targeted therapies, including anti-EGFR, anti-VEGF, immune checkpoint inhibitors, and HER2-targeted treatments, are explored along with their mechanisms and clinical applications. Additionally, we highlight the importance of utilizing personalized treatment strategies by employing molecular pro-filing and genetic testing. These approaches facilitate the identification of appropriate pa-tients for targeted therapies. Clinical trials supporting these treatments are presented, em-phasizing response rates and survival outcomes. Detailed exploration of resistance mecha-nisms to targeted therapies and strategies to overcome resistance is also provided, paving the way for more effective regimens. Directions for future research in precision medicine, such as biomarkers, combinations, and liquid biopsy in the oncology field, are described. However, the precise application of precision medicine in CRC comes with questions such as costs, considerations of ethical factors, and the need to sensitize patients. Nonetheless, based on a meticulous analysis of several aspects, precision medicine finds itself equipped with the ability to enhance the patients' prognosis ,thereby, lessening the global oncologic burden of CRC and provide important information to the clinician-scientist and policy-maker that might benefit from the ongoing development of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9df55b47e156335e90e003be1a18ab4d05e8ee9" target='_blank'>
              Precision Medicine in Colorectal Cancer: Targeted Therapies and Biomarker Insights.
              </a>
            </td>
          <td>
            Shriyansh Srivastava, Amit Anand, Anil Goud Kandhula, Pooja Jaiswal, Sachin Kumar, Monica Gulati, Monika Sachdeva, Tapan Behl
          </td>
          <td>2025-01-28</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic medicine, which integrates genomics and bioinformatics into clinical care and diagnostics, is transforming healthcare by enabling personalized treatment approaches. Advances in technologies such as DNA sequencing, proteomics, and computational power have laid the foundation for individualized therapies that account for genetic variations influencing disease risk, progression, and treatment response. This review explores the historical milestones leading to current applications of genomic medicine, such as targeted therapies, gene therapies, and precision medicine, in fields including cardiovascular diseases, oncology, and rare genetic disorders. It highlights the use of next-generation sequencing (NGS) and third-generation sequencing (TGS) to improve diagnostic accuracy and treatment outcomes, emphasizing the role of genomic data in advancing personalized treatments. Furthermore, emerging therapies such as CRISPR/Cas-based genome editing and adeno-associated viral vectors showcase the potential of gene therapy in addressing complex diseases, including rare genetic disorders. Despite promising advancements, challenges remain in fully integrating genomic medicine into routine clinical practice, including cost barriers, data interpretation complexities, and the need for widespread genomic literacy among healthcare professionals. The future of genomic medicine holds transformative potential for revolutionizing the diagnosis, treatment, and management of both common and rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48c7f88008ee3f6f5e730cd7155fc1a412d588b" target='_blank'>
              Genomic medicine and personalized treatment: a narrative review
              </a>
            </td>
          <td>
            Adil Khan, Anchal Ramesh Barapatre, Nadir Babar, Joy Doshi, Mohamd Ghaly, Kirtan Ghanshyam Patel, Shayan Nawaz, Uswa Hasana, Swara Punit Khatri, Shilpa Pathange, Abhinya Reddy Pesaru, C. Puvvada, Marium Billoo, Usama Jamil
          </td>
          <td>2025-02-13</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a24af38423fb1cff9abd834b78aff5969e9b847" target='_blank'>
              Establishment and characterization of three gemcitabine-resistant human intrahepatic cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Jiandong Li, Yanxin Hu, Jiayao Zhang, Weiguang Zhang, Jianhua Yu, Baochun Lu
          </td>
          <td>2025-02-09</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is driven by a complex network of resistance mechanisms against standard-of-care therapies, resulting in poor long-term outcomes. This review offers a uniquely comprehensive and integrative perspective on these resistance pathways, systematically examining both androgen receptor (AR)-dependent factors (including AR overexpression, point mutations, glucocorticoid receptor signaling, splice variants, post-translational modifications, altered coregulators, and intratumoral hormone biosynthesis) and AR-independent pathways (such as neuroendocrine differentiation, lineage plasticity, and alternative growth factor signaling). We also highlight resistance mechanisms influencing immunotherapy, chemotherapy, radiopharmaceutical therapy and targeted therapy. By synthesizing emerging insights across these domains, this review not only clarifies the underlying biology of mCRPC resistance but also identifies key leverage points for more effective interventions. Building on this foundation, we propose a forward-looking framework for overcoming mCRPC drug resistance, emphasizing the importance of biomarker-guided patient selection, combination strategies that simultaneously target multiple resistance mechanisms, and novel therapies under investigation. These recommendations are intended to guide future clinical trial designs and research priorities that move beyond incremental improvements. Ultimately, this comprehensive synthesis aims to serve as a resource for clinicians and researchers to accelerate the development of durable, precision-based treatment strategies in mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b50b02beb4327daa61573fe429543e569947b38" target='_blank'>
              Overcoming drug resistance in castrate-resistant prostate cancer: current mechanisms and emerging therapeutic approaches
              </a>
            </td>
          <td>
            Adam Khorasanchi, Feng Hong, Yuanquan Yang, Eric A. Singer, Peng Wang, Mingjia Li, Linghua Zheng, P. Monk, Amir Mortazavi, Lingbin Meng
          </td>
          <td>2025-02-19</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Purpose Endometriosis is a common chronic neuroinflammatory disease with a poorly understood pathogenesis. Molecular changes and specific immune cell infiltration in the eutopic endometrium are critical to disease progression. This study aims to explore immune mechanisms and molecular differences in the proliferative eutopic endometrium of endometriosis by integrating bulk RNA-seq and single-cell RNA sequencing (scRNA-seq) data, and to develop diagnostic and predictive models for the disease. Methods Gene expression profiles from the proliferative endometrium of endometriosis patients and healthy controls were obtained from the Gene Expression Omnibus. Single-cell RNA-seq data were processed using R packages, and cell clusters’ contributions to endometriosis were calculated. Differentially expressed genes (DEGs) from bulk RNA-seq were intersected with significant mesenchymal cell genes from scRNA-seq, and a predictive model was constructed using LASSO analysis. Key gene mechanisms were explored through Gene Set Enrichment and Variation Analyses. miRNA networks and transcriptional regulation analyses were conducted, and potential drugs were predicted using the Connectivity Map database. RT-qPCR validated key gene expression. Results Mesenchymal cells in the proliferative eutopic endometrium were identified as major contributors to endometriosis pathogenesis. LASSO regression identified eight key genes: SYNE2, TXN, NUPR1, CTSK, GSN, MGP, IER2, and CXCL12. The predictive model based on these genes achieved AUC values of 1.00 and 0.8125 in training and validation cohorts. Immune infiltration analysis showed increased CD8+ T cells and monocytes in the eutopic endometrium of endometriosis patients. Drug target prediction indicated that drugs like Retinol, Orantinib, Piperacillin, and NECA were negatively correlated with the expression profiles of endometriosis. RT-qPCR validated gene expression in patients aligned with bioinformatics analysis. Conclusion Significant transcriptomic changes and altered immune cell infiltration in the proliferative eutopic endometrium potentially contribute to endometriosis pathogenesis. Our predictive model based on the key genes demonstrates high diagnostic accuracy, offering insights for diagnosis and potential treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a24e34bf48eebfe993286a3f22d60c5eec8cac" target='_blank'>
              Integration of Single Cell and Bulk RNA-Sequencing Reveals Key Genes and Immune Cell Infiltration to Construct a Predictive Model and Identify Drug Targets in Endometriosis
              </a>
            </td>
          <td>
            Hanke Zhang, Yu-Qing Fang, Dan Luo, Yan-Hui Li
          </td>
          <td>2025-02-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fbe09900693cf8126b4ed7a1f742723962eeea9" target='_blank'>
              A comprehensive analysis of scRNA-Seq and RNA-Seq unveils B cell immune suppression in the AAV-loaded brain
              </a>
            </td>
          <td>
            Shunyu Wu, Lu Xue, Xiang Li, Yaoxuan Wang, Yuting Zhu, Yuanbo Luo, Jiayu Sun, Tingting Jin, Wenying Shu, Zhaoyan Wang
          </td>
          <td>2025-03-05</td>
          <td>Immunologic Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) presents a formidable global health challenge, marked by its aggressive behavior and significant treatment resistance. This subtype, devoid of estrogen, progesterone, and HER2 receptors, largely relies on breast cancer stem cells (BCSCs) for its progression, metastasis, and recurrence. BCSCs, characterized by their self-renewal capacity and resistance to conventional therapies, exploit key surface markers and critical signaling pathways like Wnt, Hedgehog, Notch, TGF-β, PI3K/AKT/mTOR and Hippo-YAP/TAZ to thrive. Their adaptability is underscored by mechanisms including drug efflux and enhanced DNA repair, contributing to poor prognosis and high recurrence rates. The tumor microenvironment (TME) further facilitates BCSC survival through complex interactions with stromal and immune cells. Emerging therapeutic strategies targeting BCSCs - ranging from immunotherapy and nanoparticle-based drug delivery systems to gene-editing technologies - aim to disrupt these resistant cells. Additionally, innovative approaches focusing on exosome-mediated signaling and metabolic reprogramming show promise in overcoming chemoresistance. By elucidating the distinct characteristics of BCSCs and their role in TNBC, researchers are paving the way for novel treatments that may effectively eradicate these resilient cells, mitigate metastasis, and ultimately improve patient outcomes. This review highlights the urgent need for targeted strategies that address the unique biology of BCSCs in the pursuit of more effective therapeutic interventions for TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66a990eb955ac9d487f1abe946d8a6fae0a4afcb" target='_blank'>
              An update on cancer stem cell survival pathways involved in chemoresistance in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Asma Jan, S. Sofi, Nusrat Jan, M. A. Mir
          </td>
          <td>2025-02-12</td>
          <td>Future oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Treatment decisions in metastatic castration-resistant prostate cancer (mCRPC) are mostly guided by clinical variables, but efforts to molecularly monitor the disease remain hampered by challenges in acquiring tumor tissue repeatedly. Here, we simultaneously profiled the genome copy number and exome in longitudinal plasma circulating tumor DNA (ctDNA) acquired before, during, and upon progression to serial treatments with androgen signaling inhibitors (ASI) and taxane chemotherapy from 60 mCRPC patients (2-10 samples per patient). The genomic data was used to delineate the clonal substructure and evolutionary dynamics of each patient, and an evolutionary dynamic index (EDI) was developed to measure the longitudinal changes of the tumor subclones. Treatment with ASI resulted in greater subclonal selection and population structure changes than taxane treatment. The subclones that emerged in association with serial therapy resistance harbored recurrent aberrations in previously identified and new candidate genes, with particular enrichment in genes related to PI3K-AKT signaling. These findings indicate that the integration of detailed clinical and genomics data can provide a framework for future unbiased genomic applications for ctDNA in the clinic to enable precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460492d7d53dee1267ea7e9d4f8344913e2a4226" target='_blank'>
              Dead-End protein expression, function, and mutation in cancer: a systematic review.
              </a>
            </td>
          <td>
            Homa Faraji, F. Banakar, Arash Sadri, A. Ebrahim-Habibi
          </td>
          <td>2025-03-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor resistance to chemotherapy is a common cause of cancer recurrence in patients with head and neck squamous cell carcinoma. The goal of this study is to establish and characterize a chemoresistant laryngeal cancer cell model and test its potential utility for chemosensitizing therapy. At the genotypic level, RNA sequencing confirmed that the cells acquired putative resistance with upregulated docetaxel‐resistant (DR) genes (e.g., TUBB3, CYP24A1) and signaling pathways (e.g., PI3K/mTOR, autophagy). For phenotypic analysis, DR cells were co‐cultured with laryngeal fibroblasts in a 2‐channel microfluidic chip that mimics a hypoxic tumor core in vivo. A drug sensitivity test with a chemosensitizer, metformin (MTF), was performed on the laryngeal tumor‐on‐a‐chip. Compared to non‐treated controls, MTF‐primed cancer cells exhibit higher sensitivity to docetaxel (DTX), that is, cell death. Collectively, this resistance‐acquired cell model displayed presumed genotypic and phenotypic profiles of chemoresistance providing a viable option for testing new therapeutic strategies for restoring tumor sensitivity to DTX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6777a39362ca5671b1a3fd98435a3160aca77da" target='_blank'>
              Establishment of a chemoresistant laryngeal cancer cell model to study chemoresistance and chemosensitization responses via transcriptomic analysis and a tumor‐on‐a‐chip platform
              </a>
            </td>
          <td>
            Christian R. Moya‐Garcia, Meghana Munipalle, Alain Pacis, Nader Sadeghi, Maryam Tabrizian, N. Y. Li-Jessen
          </td>
          <td>2025-01-22</td>
          <td>Bioengineering &amp; Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eefe391651ce22dbb5bada06a6b81eaabe2e0d4" target='_blank'>
              Cancer-associated fibroblasts promote EGFR-TKI resistance via the CTHRC1/glycolysis/H3K18la positive feedback loop.
              </a>
            </td>
          <td>
            Chen Zhang, Wenxin Zhou, Hai Xu, Jiali Xu, Jun Li, Xinyin Liu, Xiyi Lu, Jiali Dai, Yuqin Jiang, Wei Wang, Erbao Zhang, Renhua Guo
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the deadliest neoplasia. Intrinsic or acquired resistance is the main cause of failure of therapy regimens that leads to relapse and death in CRC patients. The widely used chemotherapeutic agent 5-fluorouracil (5-FU) remains the mainstay for therapeutic combinations. Unfortunately, chemotherapeutic resistance and side effects are frequent events that compromise the success of these therapies; the dysregulation of enzymes that regulate 5-FU metabolism increases the expression and activity of efflux pumps. Additional tumor cell adaptations such as epithelial–mesenchymal transition (EMT), autophagy shaping of the tumor microenvironment, and inflammation contribute to chemoresistance. Finding new strategies and alternatives to enhance conventional chemotherapies has become necessary. Recently, the study of natural compounds has been gaining strength as an alternative to chemotherapeutics in different cancers. Curcumin, trimethylglycine, resveratrol, artemisinin, and some helminth-derived molecules, among others, are some natural compounds studied in the context of CRC. This review discusses the main benefits, mechanisms, advances, and dark side of conventional chemotherapeutics currently evaluated in CRC treatment. We also analyzed the landscape of alternative non-conventional compounds and their underlying mechanisms of action, which could, in the short term, provide fundamental knowledge to harness their anti-tumor effects and allow them to be used as alternative adjuvant therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a52e6869256b3ec61847f7e51859687b36490d2c" target='_blank'>
              Natural Alternatives in the Treatment of Colorectal Cancer: A Mechanisms Perspective
              </a>
            </td>
          <td>
            K. V. Fernandez-Muñoz, C. A. Sánchez-Barrera, M. Meraz-Ríos, J. L. Reyes, E. Pérez-Yépez, M. T. Ortiz-Melo, Luis I. Terrazas, Mónica G Mendoza-Rodríguez
          </td>
          <td>2025-02-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Small noncoding RNAs (smRNAs) play critical roles in regulating various cellular processes, including development, stress response, and disease pathogenesis. However, functional characterization of smRNAs remains limited by the scale and simplicity of phenotypic readouts. Recently, single-cell perturbation screening methods, which link CRISPR-mediated genetic perturbations to rich transcriptomic profiling, have emerged as foundational and scalable approaches for understanding gene functions, mapping regulatory networks, and revealing genetic interactions. However, a comparable approach for probing the regulatory consequences of smRNA perturbations is lacking. Here, we present Decoy-seq as an extension of this approach for high-content, single-cell perturbation screening of smRNAs. This method leverages U6-driven tough decoys (TuD), which form stable duplexes with their target smRNAs, for inhibition in the cell. Lentiviral-encoded TuDs are compatible with conventional single-cell RNA-sequencing (scRNA-seq) technologies, allowing joint identification of the smRNA perturbation in each cell and its associated transcriptomic profile. We applied Decoy-seq to 336 microRNAs (miRNAs) and 196 tRNA-derived fragments (tRFs) in a human breast cancer cell line, demonstrating its ability to uncover complex regulatory pathways and novel functions of these smRNAs. Notably, we show that tRFs influence mRNA polyadenylation and regulate key cancer-associated processes, such as cell cycle progression and proliferation. Therefore, Decoy-seq provides a powerful framework for exploring the functional roles of smRNAs in normal physiology and disease, and holds promise for accelerating future discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98aed8337c8669c29add6c06f708bd3a96665623" target='_blank'>
              Decoy-seq unlocks scalable genetic screening for regulatory small noncoding RNAs
              </a>
            </td>
          <td>
            Benedict Choi, Sushil Sobti, Larisa M. Soto, Trey Charbonneau, Aiden M. Sababi, A. Navickas, H. Najafabadi, Hani Goodarzi
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Biological and dynamic mechanisms by which Drug-tolerant persister (DTP) cells contribute to the development of acquired drug resistance have not been fully elucidated. Here, by integrating multidimensional data from drug-treated PC9 cells, we developed a novel multiscale mathematical model from an evolutionary perspective that encompasses epigenetic and cellular population dynamics. By coupling stochastic simulation with quantitative analysis, we identified epigenetic instability as the most prominent kinetic feature related to the emergence of DTP cell subpopulations and the effectiveness of intermittent treatment. Moreover, we revealed the optimal schedule for intermittent treatment, including the optimal area for therapeutic time and drug holidays. By leveraging single-cell RNA-seq data characterizing the drug tolerance of lung cancer, we validated the predictions made by our model and further revealed previously unrecognized biological features of DTP cells, such as cell autophagy and migration, as well as new biomarker genes of therapeutic tolerance. Our work not only provides a paradigm for the integration of multiscale mathematical models with newly emerging genomics data but also improves our understanding of the crucial roles of DTP cells and offers guidance for developing new intermittent treatment strategies against acquired drug resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b2c7b90ea2e9a9d298ad9d7420ea2b6b5b8ec4" target='_blank'>
              Integrating multiscale mathematical modeling and multidimensional data reveals the effects of epigenetic instability on acquired drug resistance in cancer
              </a>
            </td>
          <td>
            Shun-guan Wang, Jinzhi Lei, Xiufen Zou, Suoqin Jin
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumors with low expression of Interferon-Stimulated Genes (ISG) and Antigen Presentation (AP) genes respond relatively poorly to current immunotherapies. One of the early hallmarks of cancer is DNA hypomethylation in genomic repeat regions, resulting in the expression of normally silenced endogenous “viral” elements. Such epigenetic changes have the potential to augment anti-tumor immune responses as well as reduce tumor cell fitness through the generation of aberrant nucleic acid species (NAS) and consequent activation of NAS-sensing pathways. Therefore, tumor evolution should favor additional selective events that suppress NAS generation, possibly yielding specific therapeutic vulnerabilities. Here, we show that the Lysine Demethylase 5 (KDM5) family of epigenetic regulatory enzymes suppress R-loop formation in genomic repeat regions in cancer cells. We find that KDM5 inhibition in luminal breast cancer cells results in R-loop-mediated DNA damage, reduced cell fitness and an increase in ISG and AP signatures as well as cell surface Major Histocompatibility Complex (MHC) class I, mediated by RNA:DNA hybrid activation of the CGAS/STING pathway. KDM5 inhibition does not result in DNA damage or activation of the CGAS/STING pathway in normal breast epithelial cells, suggesting that KDM5 inhibitors may enable a wide therapeutic window in this setting, as compared to STING agonists or Type I Interferons. These findings provide new insights into the interplay between epigenetic regulation of genomic repeats, R-loop formation, innate immunity, and cell fitness in the context of cancer evolution and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction: Prostate cancer, notably prostate adenocarcinoma (PARD), has high incidence and mortality rates. Although typically resistant to immunotherapy, recent studies have found immune targets for prostate cancer. Stratifying patients by molecular subtypes may identify those who could benefit from immunotherapy. Methods: We used single-cell and bulk RNA sequencing data from GEO and TCGA databases. We characterized the tumor microenvironment at the single-cell level, analyzing cell interactions and identifying fibroblasts linked to mitophagy. Target genes were narrowed down at the bulk transcriptome level to construct a PARD prognosis prediction nomogram. Unsupervised consensus clustering classified PARD into subtypes, analyzing differences in clinical features, immune infiltration, and immunotherapy. Furthermore, the cellular functions of the genes of interest were verified in vitro. Results: We identified ten cell types and 160 mitophagy-related single-cell differentially expressed genes (MR-scDEGs). Strong interactions were observed between fibroblasts, endothelial cells, CD8+ T cells, and NK cells. Fibroblasts linked to mitophagy were divided into six subtypes. Intersection of DEGs from three bulk datasets with MR-scDEGs identified 26 key genes clustered into two subgroups. COX regression analysis identified seven prognostic key genes, enabling a prognostic nomogram model. High and low-risk groups showed significant differences in clinical features, immune infiltration, immunotherapy, and drug sensitivity. In prostate cancer cell lines, CAV1, PALLD, and ITGB8 are upregulated, while CLDN7 is downregulated. Knockdown of PALLD significantly inhibits the proliferation and colony-forming ability of PC3 and DU145 cells, suggesting the important roles of this gene in prostate cancer progression. Conclusions: This study analyzed mitophagy-related genes in PARD, predicting prognosis and aiding in subtype identification and immunotherapy response analysis. This approach offers new strategies for treating prostate cancer with specific molecular subtypes and helps develop potential biomarkers for personalized medicine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab8cd0df51baf0c065fc9d46908e143b732254" target='_blank'>
              An Integrated Approach Utilizing Single-Cell and Bulk RNA-Sequencing for the Identification of a Mitophagy-Associated Genes Signature: Implications for Prognostication and Therapeutic Stratification in Prostate Cancer
              </a>
            </td>
          <td>
            Yuke Zhang, Li Ding, Zhijin Zhang, Liliang Shen, Yadong Guo, Wentao Zhang, Yang Yu, Zhuoran Gu, Ji Liu, Aimaitiaji Kadier, Jiang Geng, Shiyu Mao, Xudong Yao
          </td>
          <td>2025-01-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Adaptation to challenging environmental conditions is crucial for the survival/fitness of all organisms. Alongside genetic mutations that provide adaptive potential during environmental challenges, epigenetic modifications offer dynamic, reversible, and rapid mechanisms for regulating gene expression in response to environmental changes in both evolution and daily life, without altering DNA sequences or relying on accidental favorable mutations. The widespread conservation of diverse epigenetic mechanisms - like DNA methylation, histone modifications, and RNA interference across diverse species, including plants - underscores their significance in evolutionary biology. Remarkably, environmentally induced epigenetic alterations are passed to daughter cells and inherited transgenerationally through germline cells, shaping offspring phenotypes while preserving adaptive epigenetic memory. Throughout anthropoid evolution, epigenetic modifications have played crucial roles in: i) suppressing transposable elements and viral genomes intruding into the host genome; ii) inactivating one of the X chromosomes in female cells to balance gene dosage; iii) genetic imprinting to ensure expression from one parental allele; iv) regulating functional alleles to compensate for dysfunctional ones; and v) modulating the epigenome and transcriptome in response to influence from the gut microbiome among other functions. Understanding the interplay between environmental factors and epigenetic processes may provide valuable insights into developmental plasticity, evolutionary dynamics, and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7f5656f1a791fd7b047008e717c5695b95280c" target='_blank'>
              Epigenetics in evolution and adaptation to environmental challenges: pathways for disease prevention and treatment.
              </a>
            </td>
          <td>
            H. Abdolmaleky, S. Nohesara, Jin-Rong Zhou, S. Thiagalingam
          </td>
          <td>2025-02-13</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Introduction: Mutations in KEAP1 and NFE2L2 (NFR2) occur in approximately 20% of non-small cell lung cancer (NSCLC) patients and are associated with resistance to radiotherapy and systemic treatments. In previous research, we found that not all KEAP1 mutations found in NSCLC patients are pathogenic. These benign mutations exhibit a wild-type phenotype that does not confer therapy resistance. With only roughly 100 of 13,000 (<1%) potential single amino acid substitution mutations in KEAP1 characterized thus far, here we comprehensively evaluated all possible point mutations in KEAP1 using deep mutational scanning (DMS). Methods: The High-throughput Mutagenesis by Integrated TilEs (MITE) method was used to generate the KEAP1 DMS library. In short, the KEAP1 sequence was segmented into 21 variable regions (tiles), each comprising a 90-base pair variable region flanked by a 30-base pair constant region. Each codon position included the 20 possible natural amino acids plus a stop codon, for a total of 13,076 KEAP1 variants. The tiles were then assembled to ensure that each full-length KEAP1 construct contained only a single amino acid substitution. The KEAP1 DMS lentivirus library was transduced into H1299-KEAP1NULLcells, followed by selective pressure with H2O2, radiation, or vehicle control. The KEAP1 ORF was then amplified from surviving cell genomic DNA, and mutations were quantified using next-generation sequencing. Functional annotations were derived through mutation enrichment analysis. Results: Both H2O2 and radiation selective pressure of the KEAP1 DMS library induced a bimodal distribution of KEAP1 mutations, with cells harboring KEAP1 nonsense mutations being enriched and cells with silent mutations being depleted. By setting a threshold of 95% specificity for silent mutations to differentiate between benign and pathogenic mutations, we identified 94% of nonsense mutations and 88% of previously characterized pathogenic mutations as pathogenic. While most KEAP1 mutations were categorized as benign (64%), analysis of mutations in TCGA NSCLC patients showed pathogenic mutations were enriched (72%). Mapping mutation phenotypes onto a KEAP1 structural model revealed an enrichment of pathogenic mutations at critical functional sites, including the NRF2 binding pocket of the Kelch domain, the BTB homodimerization interface, and the hydrophobic core of the oxidative stress-sensing IVR domain. Conclusions: Through deep mutational scanning experiments, we generated a comprehensive list of pathogenic KEAP1 single amino acid substitutions. These results align with previously known biological functions and prior single amino acid substitutions characterized in the literature. These findings will facilitate the identification of benign and pathogenic KEAP1 mutations in clinical settings, enabling personalized radiotherapeutic and systemic therapy strategies for patients harboring KEAP1 mutations.
 Citation Format: Noah Kastelowitz, Shashank Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn. Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr BO23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, A. M. Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive malignant brain tumor with almost inevitable recurrence despite multimodal management with surgical resection and radio-chemotherapy. While several genetic, proteomic, cellular, and anatomic factors interplay to drive recurrence and promote treatment resistance, the epigenetic component remains among the most versatile and heterogeneous of these factors. Herein, the epigenetic landscape of GBM refers to a myriad of modifications and processes that can alter gene expression without altering the genetic code of cancer cells. These processes encompass DNA methylation, histone modification, chromatin remodeling, and non-coding RNA molecules, all of which have been found to be implicated in augmenting the tumor’s aggressive behavior and driving its resistance to therapeutics. This review aims to delve into the underlying interactions that mediate this role for each of these epigenetic components. Further, it discusses the two-way relationship between epigenetic modifications and tumor heterogeneity and plasticity, which are crucial to effectively treat GBM. Finally, we build on the previous characterization of epigenetic modifications and interactions to explore specific targets that have been investigated for the development of promising therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc0fa03a9f64a66bc6beac8c2a3a49bfa95cc71" target='_blank'>
              The epigenetic mechanisms involved in the treatment resistance of glioblastoma
              </a>
            </td>
          <td>
            Aanya Shahani, Hasan Slika, Ahmad Elbeltagy, Alexandra Lee, Christopher Peters, Toriyn Dotson, Divyaansh Raj, Betty Tyler
          </td>
          <td>2025-02-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary Prostate cancer (PCa) is a heterogenous malignancy in men, representing the second most diagnosed cancer and the fifth leading cause of cancer-related death globally in 2022. PCa incidence and progression is dependent on several predisposing factors, most significantly genetics and germline/somatic mutations. With the development of whole-genome sequencing (WGS) technology, it is now possible to determine the mutational statuses of PCa-related genes for improved clinical decision making and individualized patient care. The National Comprehensive Cancer Network (NCCN) Guidelines now recommend germline multigene testing followed by post-test genetic counseling. Depending on the germline mutations in homologous recombination (HR) repair genes or in mismatch repair (MMR) genes, specific treatment options, including poly-ADP-ribose polymerase (PARP) inhibition or PD-1 blockade, may be of more clinical benefit. This review highlights specific PCa-related germline mutations in racially diverse populations and therapeutic approaches to treat PCa patients with DNA damage repair gene alterations in advanced disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f685df01c211531229e0572b44cf7405f9d7cc" target='_blank'>
              Ancestry-Specific DNA Damage Repair Gene Mutations and Prostate Cancer
              </a>
            </td>
          <td>
            Talaibek Borbiev, Kevin Babcock, K. Sinopole, Gregory T. Chesnut, G. Petrovics
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Stemness-associated cell states are linked to chemotherapy resistance in AML. We uncovered a direct mechanistic link between expression of the stem cell transcription factor GATA2 and drug resistance. The GATA-binding protein 2 (GATA2) plays a central role in blood stem cell generation and maintenance. We find substantial intra- and inter-patient variability in GATA2 expression across AML patient samples. GATA2 expression varies by molecular subtype and has been linked to outcome. In a murine model, KMT2A-MLL3 driven AML originating from a stem cell or immature progenitor cell population have higher Gata2 expression and are more resistant to the standard AML chemotherapy agent doxorubicin. Deletion of Gata2 resulted in more robust induction of p53 following exposure to doxorubicin. ChIP-Seq, RNA-Seq and functional studies revealed that GATA2 regulates the expression of RASSF4, a modulator of the p53 inhibitor MDM2. GATA2 and RASSF4 are anti-correlated in human cell lines and AML patient cell bulk and single cell expression datasets. Knockdown of Rassf4 in Gata2 low cells resulted in doxorubicin or nutlin-3 resistance. Conversely, overexpression of Rassf4 results in sensitization of cells expressing high levels of Gata2. Finally, doxorubicin and nutlin-3 are synergistic in Gata2-high murine AML, as well as AML patient samples. We discovered a previously unappreciated role for GATA2 in dampening p53-mediated apoptosis via transcriptional regulation of RASSF4, a modulator of MDM2. This role for GATA2 directly links the expression of a stemness associated transcription factor to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e4d0de77d706d929171b654f103344701ea6a" target='_blank'>
              GATA2 links stemness to chemotherapy resistance in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Fatemeh Alikarami, H. Xie, Simone S. Riedel, Haley Goodrow, D. Barrett, Leila Mahdavi, Alexandra Lenard, Changya Chen, T. Yamauchi, Etienne Danis, Zhendong Cao, Vu L. Tran, M. Jung, Yapeng Li, Hua Huang, Junwei Shi, Kai Tan, D. Teachey, E. Bresnick, Tobias Neff, Kathrin M Bernt
          </td>
          <td>2025-01-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genomic rearrangements play an important role in shaping genetic diversity, as they enable the generation of novel structural variations through specific genome manipulation tools. These variations contribute to phenotypic differences among individuals within a population, thereby serving as the foundation for natural selection and driving evolutionary processes. In recent years, Synthetic Chromosome Rearrangement and Modification by LoxP-mediated Evolution (SCRaMbLE) has emerged as a promising tool for studying genomic rearrangements. SCRaMbLE utilizes site-specific recombination mediated by loxPsym sites to induce targeted chromosomal rearrangements in yeast cells. In this review, we provide a comprehensive overview of recent advancements in optimization strategies of the SCRaMbLE system and discuss influential factors that affect its performance based on recent research findings. We demonstrate how the SCRaMbLE system can be employed for pathway engineering, phenotype improvement, genome minimization, and dissection of genotype-to-phenotype relationships. We highlight both the advantages and challenges associated with SCRaMbLE and envision its potential applications beyond yeast genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0314619e822b8ab25ecabe0ad0985b3e4d4d9a40" target='_blank'>
              SCRaMbLE: a versatile tool for genome manipulation.
              </a>
            </td>
          <td>
            Li Cheng, Jielin Li, Yaojun Chen, Junbiao Dai
          </td>
          <td>2025-02-12</td>
          <td>Physiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer chemoresistance presents a challenge in oncology, often leading to treatment failure and disease progression. CD44, a multifunctional cell surface glycoprotein, has garnered attention for its involvement in various aspects of cancer biology. Through alternative splicing, CD44 can form isoforms with the inclusion of only standard exons, typical for normal tissue, or with the addition of variant exons, frequently expressed in cancer tissue and associated with chemoresistance. The functions of CD44 involved in regulation of cancer signaling pathways are being actively studied, and the significance of specific variant exons in modulating cell death pathways, central to the response of cancer cells to chemotherapy, begins to become apparent. This review provides a comprehensive analysis of the association of CD44 variant exons/total CD44 with clinical outcomes of patients undergoing chemotherapy. The role of CD44 variant exons v6, v9 and others with a significant effect on patient chemotherapy outcomes by means of key cellular death pathways such as apoptosis, ferroptosis and autophagy modulation is further identified, and their impact on drug resistance is highlighted. An overview of clinical trials aimed at targeting variant exon-containing isoforms is provided, and possible directions for further development of CD44-targeted therapeutic strategies are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c896fa2073872384fe9590cc27feab2b38f88e9" target='_blank'>
              CD44 variant exons induce chemoresistance by modulating cell death pathways
              </a>
            </td>
          <td>
            Maria Yanova, Evgeniya Stepanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene therapy and genome editing have emerged as transformative approaches in the management of a diverse range of genetic and acquired diseases. This evaluation offers a thorough examination of the present state and prospects of these innovative technologies. Gene therapy is a prospective approach to the treatment and prevention of a variety of conditions, including complex cancers and inherited genetic disorders, which entail the introduction, removal, or modification of genetic material within a patient's cells. Genome editing, particularly through techniques such as CRISPR-Cas9, enables targeted corrections of genetic defects and opens new possibilities for personalized medicine by allowing for precise modifications at the DNA level. The review addresses the ethical implications, clinical applications, and significant advancements of these technologies. This article endeavors to underscore the substantial influence of gene therapy and genome editing on contemporary medicine by assessing the most recent research and clinical trials, thereby emphasizing their potential to revolutionize disease treatment and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fcf14d7421a285bda69ccae261660da8cc8fcc" target='_blank'>
              Gene Therapy and Gene Editing: Current Trends and Future Prospects of Molecular Medicine.
              </a>
            </td>
          <td>
            Aniruddha Sen, Vijay Singh, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar, Sudhir Shyam Kushwaha
          </td>
          <td>2025-02-18</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Precision medicine has revolutionized the treatment of colorectal cancer by enabling a personalized approach tailored to each patient’s unique genetic characteristics. Genomic profiling allows for the identification of specific mutations in genes such as KRAS, BRAF, and PIK3CA, which play a crucial role in cell signaling pathways that regulate cell proliferation, apoptosis, and differentiation. This information enables doctors to select targeted therapies that inhibit specific molecular pathways, maximizing treatment effectiveness and minimizing side effects. Precision medicine also facilitates adaptive monitoring of tumor progression, allowing for adjustments in therapy to maintain treatment effectiveness. While challenges such as high costs, limited access to genomic technology, and the need for more representative genomic data for diverse populations remain, collaboration between researchers, medical practitioners, policymakers, and the pharmaceutical industry is crucial to ensure that precision medicine becomes a standard of care accessible to all. With continued advances and support, precision medicine has the potential to improve treatment outcomes, reduce morbidity and mortality rates, and enhance the quality of life for colorectal cancer patients worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ce8f60143b7348cbfaa508b5bbdc79c01b3a" target='_blank'>
              Precision medicine in colorectal cancer: genomics profiling and targeted treatment
              </a>
            </td>
          <td>
            Adi Muradi Muhar, A. J. Velaro, Arya Tjipta Prananda, S. E. Nugraha, Princella Halim, R. A. Syahputra
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69cfefb50db5bb84559d86b5f736278ac09bb363" target='_blank'>
              LINC00626 drives tamoxifen resistance in breast cancer cells by interaction with UPF1
              </a>
            </td>
          <td>
            Hui Yuan, Lianbang Zhou, Wei Hu, Min Yang
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer.
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            Mohamed J Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Background: Bladder cancer poses a significant threat to human health. In recent years, genetic circuit therapy has emerged as a novel alternative for precision tumor treatment, demonstrating promising potential for clinical application. Compared to traditional drugs, genetic circuits – typically carried by plasmids – offer advantages such as modularity, druggability, and shorter drug development cycles. These circuits can identify multiple molecular signals from tumors and integrate them through logic gates to specifically target tumor cells. Furthermore, by assembling effector modules, they can induce specific forms of cell death in tumor cells or alter malignant phenotypes, thereby reshaping the immune microenvironment to produce efficient, durable, and controllable antitumor effects. The urinary system serves as an ideal model for this new therapy due to its accessibility from the outside. Several genetic circuits have already been validated for the treatment of bladder cancer. This review outlined the effectiveness and potential value of genetic circuits in bladder cancer therapy. Objective: This review focused on the design principles of genetic circuits, their ability to recognize and convert signals, their therapeutic signal output, and the associated delivery vehicles. We also discussed the challenges and future prospects of genetic circuits as a novel form of “intelligent biotherapy.” Conclusion: The gene circuit can identify multiple signals, processing complex information, and generating multiple effects, thus providing a new approach for personalized treatment of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Rationale: Anaplastic thyroid carcinoma (ATC) is an extraordinarily aggressive form of thyroid cancer, frequently presenting with locally advanced infiltration or distant metastases at the time of initial diagnosis, thus missing the optimal window for surgical intervention. Consequently, systemic chemotherapy and targeted therapies are vital for improving the prognosis of ATC. However, ATC exhibits significant resistance to conventional treatments, highlighting the need to elucidate the biological mechanisms underlying this drug resistance and identify novel therapeutic targets to overcome it. Methods: We conducted a comprehensive analysis of both bulk and single-cell RNA sequencing (scRNA-seq) data from ATC samples to screen for m 5 C modification–related genes associated with multidrug resistance (MDR). We then performed IC 50 assays, flow cytometry, and employed a spontaneous tumorigenic ATC mouse model with Nsun2 knockout to demonstrate that NSUN2 promotes MDR in ATC. To investigate the mechanisms of NSUN2-mediated drug resistance, we generated NSUN2-knockout ATC cell lines and performed transcriptomic, proteomic, and MeRIP-seq analyses. Additionally, RNA sequencing and alternative splicing analyses were conducted to determine global changes upon NSUN2 knockout. We further explored the underlying mechanisms of the NSUN2/SRSF6/UAP1 axis through glycoprotein staining, denaturing IP ubiquitination, nuclear–cytoplasmic fractionation, and PCR. Lastly, we evaluated the synergistic effects of a small-molecule NSUN2 inhibitor with anticancer agents both in vitro and in vivo . Results: Our findings reveal that NSUN2 expression correlates significantly with MDR in ATC. NSUN2 operates as a "writer" and ALYREF as a "reader" of m 5 C on SRSF6 mRNA, inducing alternative splicing reprogramming and redirecting the splice form of the UAP1 gene from AGX1 to AGX2. As a result, AGX2 enhances the N-linked glycosylation of ABC transporters, stabilizing them by preventing ubiquitination-mediated degradation. Furthermore, an NSUN2 inhibitor reduces NSUN2 enzymatic activity and diminishes downstream target expression, presenting a novel, promising therapeutic approach to overcome MDR in ATC. Conclusions: These findings suggest that the NSUN2/SRSF6/UAP1 signaling axis plays a vital role in MDR of ATC and identify NSUN2 as a synergistic target for chemotherapy and targeted therapy in ATC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de612cc82a82113a9f1d339ec0d2251cf86a8e92" target='_blank'>
              NSUN2-mediated m5C modification drives alternative splicing reprogramming and promotes multidrug resistance in anaplastic thyroid cancer through the NSUN2/SRSF6/UAP1 signaling axis
              </a>
            </td>
          <td>
            Xiukun Hou, Qiman Dong, Jie Hao, Min Liu, Junya Ning, Mei Tao, Zhongyu Wang, Fengli Guo, Dongmei Huang, Xianle Shi, Ming Gao, Dapeng Li, Xiangqian Zheng
          </td>
          <td>2025-01-27</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The human peripheral blood displays diverse molecular characteristics across populations, understanding the drivers and underlying mechanisms of which remains challenging. Here, we introduce the Chinese Immune Multi-Omics Atlas (CIMA), elucidating sex-, age-, and genetic-related molecular variations by analyzing multi-omics data from 428 adults with over 10 million immune cells. CIMA generated an enhancer-driven gene regulatory network, identifying 237 high-quality regulons and revealing cell type-specific regulatory mechanisms. Additionally, 11,521 lead cis-expression quantitative trait loci (eQTLs) and 46,339 chromatin accessibility QTLs (caQTLs) were identified at cell type level. CIMA also uncovered pleiotropic associations among immune-related disease risk loci, eQTLs, and caQTLs in a cell type-specific manner. Lastly, a novel cell language model, CIMA-CLM, was developed to predict chromatin accessibility and noncoding variant effects using chromatin sequences and gene expressions. This work represents a population-scale multi-omics resource of human immune cells, providing a valuable reference for future investigation of immune-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f8b66ae23a1d254045e2d47bbe65620395c404" target='_blank'>
              Single-Cell Genomics Elucidates Molecular Variations and Regulatory Mechanisms in Circulating Immune Cells
              </a>
            </td>
          <td>
            Jianhua Yin, Yuhui Zheng, Zhuoli Huang, Wenwen Zhou, Yue Yuan, Pengfei Cai, Yong Bai, Shichen Yang, Yue Gao, Shanshan Duan, Yang Wang, Wenxi Zhang, Xinyu Zhang, Yilin Wei, Zekai Xu, Yaling Huang, Ying Liu, Weikai Wang, Tao Yang, Jingzhi Lv, Zhongjin Zhang, Xiaoya Chen, Xiru Zhang, Fupeng Li, Yan Zhang, Guodan Zeng, Xue Wang, Wen Ma, Guixue Hou, Shijie Hao, Chang Liu, Yiwei Lai, Bo Wang, Yuxiang Li, Wenwei Zhang, Peng Gao, Jun Xie, Miguel A. Esteban, Ying Gu, Jiansong Ji, Ting Qi, Boxiang Liu, Jian Wang, Jian Yang, Xun Xu, Longqi Liu, Xin Jin, Chuanyu Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b664fbb9ca2f0d479f54b9221d29d13ae0d1a6" target='_blank'>
              Transcriptional regulatory network analysis identifies GRN as a key regulator bridging chemotherapy and immunotherapy response in small cell lung cancer
              </a>
            </td>
          <td>
            S. Yoo, Ayushi S. Patel, S. Karam, Yi Zhong, Li Wang, Feng Jiang, Ranran Kong, Sharon Bikvan, Wenhui Wang, A. Sinha, Charles A. Powell, Jun Zhu, Hideo Watanabe
          </td>
          <td>2025-02-05</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ad7065cbf4c15026ee68bc1c1e9443de9a4af3" target='_blank'>
              Identification of T cell dysfunction molecular subtypes and exploration of potential immunotherapy targets in BRAF V600E-mutant colorectal cancer
              </a>
            </td>
          <td>
            Tiefeng Gu, Haonan Qi, Jiaqi Wang, Liangwei Sun, Yongqi Su, Hanqing Hu
          </td>
          <td>2025-02-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: The cell cycle, a critical and intricate biological process, comprises various phases, and its dysregulation plays a pivotal role in tumorigenesis and metastasis. The exploration of cell cycle-based molecular subtypes across pan-cancers, along with the application of synthetic lethality concepts, holds promise for advancing cancer therapies. Methods: A pan-cancer analysis was conducted to assess the cell cycle serves as a reliable signature for classifying molecular subtypes and to understand the potential clinical application of genes as potential drug targets based on synthetic lethality. Results: Molecular subtypes derived from cell cycle features in certain cancers, particularly kidney-related malignancies, exhibited distinct immune characteristics. Synthetic lethal interactions within the cell cycle pathway were common, with significant genetic interactions further identifying potential drug targets through the exploitation of genetic relationships with key driver genes. Additionally, miRNAs and lncRNAs may influence the cell cycle through miRNA:mRNA interactions and ceRNA networks, thereby enriching the genetic interaction landscape. Conclusions: These findings suggest that the cell cycle pathway could serve as a promising molecular subtype signature to enhance cancer prognostication and offer potential targets for anticancer drug development through synthetic lethality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f0509708b74787ef228103fd17aa6cf4b5bae5" target='_blank'>
              Cell Cycle-Based Molecular Features via Synthetic Lethality and Non-Coding RNA Interactions in Cancer
              </a>
            </td>
          <td>
            Shizheng Xiong, Jiaming Jin, Xinmiao Zhao, Yang Zhao, Zhiheng He, Haochuan Guo, Chengjun Gong, Jiafeng Yu, Li Guo, Tingming Liang
          </td>
          <td>2025-03-05</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd82847b06d814bccf91fe62dfe773b93ac1d7" target='_blank'>
              Exploring the principles behind antibiotics with limited resistance
              </a>
            </td>
          <td>
            Elvin Maharramov, Márton Simon Czikkely, P. Szili, Z. Farkas, Gábor Grézal, Lejla Daruka, Eszter Kurkó, Léna Mészáros, Andreea Daraba, Terézia Kovács, Bence Bognár, Szilvia Juhász, Balázs Papp, Viktória Lázár, Csaba Pál
          </td>
          <td>2025-02-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Regulation of protein production in response to physiological signals is achieved through precise control of Eukaryotic Elongation Factor 2 (eEF2), whose distinct translocase function is crucial for cell survival. Phosphorylation of eEF2 at its Thr56 (T56) residue inactivates this function in translation. Using genetically modified paralogue of a colon cancer cell line, HCT116 which carries a point mutation at Ser595-to-Alanine in the eEF2 gene we were able to create a constitutively active form of eEF2. Then, we compared these cells with their wild-type counterparts  under complete growth and nutrient deprivation conditions. The mutant forms were created by using CRISPR/Cas9-mediated gene cassette knock-in (KI) strategy. The samples were analyzed by a microarray based approach. Carrying out a WikiPath analysis, we identified distinct signaling pathways and gene expression patterns that are differentially regulated in the mutant cells (HCT116KI), but not in wild-type HCT116 (HCT116WT) cells, particularly under nutrient deprived conditions. Our results suggest that the constitutive activation of eEF2 enables increase in the malignant gene expression in colon cancer cells compared to the wild type counterparts under nutrient deprived conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7a023295ed87a1f8110a5f37ed0be7b05014ddf" target='_blank'>
              Improved Malignancy of Colon Cancer Cells at Gene Expression Level with Constitutive Activation of the Eukaryotic Elongation Factor 2 (eEF2) Under Nutrition Deficient Conditions.
              </a>
            </td>
          <td>
            A. H. Deniz, E. Aydemir, Nur Ekimci Gürcan, Abdulahi Dabo Jakir, Ö. Bayrak, F. Şahin, Furkan Ayaz
          </td>
          <td>2025-01-13</td>
          <td>Chemistry & biodiversity</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy affecting women, manifesting as a heterogeneous disease with diverse molecular characteristics and clinical presentations. Recent studies have elucidated the role of epigenetic modifications in the pathogenesis of breast cancer, including drug resistance and efflux characteristics, offering potential new diagnostic and prognostic markers, treatment efficacy predictors, and therapeutic agents. Key modifications include DNA cytosine methylation and the covalent modification of histone proteins. Unlike genetic mutations, reprogramming the epigenetic landscape of the cancer epigenome is a promising targeted therapy for the treatment and reversal of drug resistance. Epidrugs, which target DNA methylation and histone modifications, can provide novel options for the treatment of breast cancer by reversing the acquired resistance to treatment. Currently, the most promising approach involves combination therapies consisting of epidrugs with immune checkpoint inhibitors. This review examines the aberrant epigenetic regulation of breast cancer initiation and progression, focusing on modifications related to estrogen signaling, drug resistance, cancer progression, and the epithelial–mesenchymal transition (EMT). It examines existing epigenetic drugs for treating breast cancer, including agents that modify DNA, inhibitors of histone acetyltransferases, histone deacetylases, histone methyltransferases, and histone demethyltransferases. It also delves into ongoing studies on combining epidrugs with other therapies and addresses the upcoming obstacles in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1d287b99832f39586fe0fd89b9b4c10e300e3c" target='_blank'>
              Cellular Epigenetic Targets and Epidrugs in Breast Cancer Therapy: Mechanisms, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Ibrahim S. Alalhareth, Saleh M. Alyami, Ali H. Alshareef, Ahmed O. Ajeibi, M. A. Al Munjem, Ahmad A. Elfifi, Meshal M. Alsharif, S. A. Alzahrani, Mohammed A. Alqaad, Marwa B. Bakir, Basel A. Abdel-Wahab
          </td>
          <td>2025-02-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="We examine the complex relationship between genomic copy number variation (CNV) and gene expression, highlighting the relevance to cancer biology and other biological contexts. By tracing the history of genometranscriptome correlations, we emphasize the complexity and challenges in understanding these interactions, particularly within the heterogeneous landscape of human cancers. Recent advances in computational algorithms and high-throughput single-cell multi-omic sequencing technologies are discussed, demonstrating their potential to refine our understanding of cancer biology and their limitations. The integration of genomic and transcriptomic analyses, which offers novel insights into tumor evolution and heterogeneity as well as therapeutic strategies, is presented as a crucial approach for advancing cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cyclin-dependent kinase 4 and 6 (CDK4/6) inhibitors have transformed the therapeutic landscape for hormone receptor-positive, human epidermal growth factor receptor 2-negative breast cancer, demonstrating significant efficacy in both early and advanced stages of the disease. Combined with endocrine therapy, these inhibitors have dramatically improved survival outcomes. However, resistance to CDK4/6 inhibitors inevitably develops, posing a significant challenge in clinical management. Resistance to CDK4/6 inhibitors can develop through inherent and acquired fashions, and their mechanisms are explored in this comprehensive review. Inherent resistance arises from pre-existing genetic or signaling pathway alterations that diminish cancer cell sensitivity to CDK4/6 inhibitors. Acquired resistance, on the other hand, develops over time through mechanisms, such as the activation of alternative signaling pathways or changes in the tumor microenvironment. The review also examines potential biomarkers for predicting resistance, especially circulating tumor DNA markers, and discusses strategies to overcome resistance. These include combination therapies targeting multiple pathways simultaneously, sequential approaches to delay the onset of resistance, and the development of next-generation CDK4/6 inhibitors with improved efficacy and reduced resistance potential. Understanding resistance mechanisms and developing effective countermeasures are crucial for optimizing patient outcomes and extending the clinical benefits of CDK4/6 inhibitors in cancer therapy. Leveraging these insights will enable clinicians to personalize treatment strategies, ultimately enhancing the long-term effectiveness of CDK4/6 inhibitors in managing breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbd50afe53f28765713814f13749bfc2d847b6c" target='_blank'>
              Cyclin-dependent kinase 4/6 inhibitor resistance mechanisms and strategies for subsequent treatment in breast cancer: A comprehensive review
              </a>
            </td>
          <td>
            Yuling Zhang, Bingfeng Chen, Rendong Zhang, Jundong Wu, Chunfa Chen
          </td>
          <td>2025-02-05</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Characterization of tumor epigenetic aberrations is integral to understanding the mechanisms of tumorigenesis and provide diagnostic, prognostic, and predictive information of high clinical relevance. Among the different tumor-associated epigenetic signatures, 5 methyl-cytosine (5mC) and 5-hydroxymethylcytosine (5hmC) are the two most well-characterized DNA methylation alterations linked to cancer pathogenesis. 5hmC has a tissue-specific distribution and its abundance is subjected to changes in tumor DNA, making it a promising biomarker. Detecting tumor-related DNA methylation alterations in tissues is highly invasive, while the analysis of the cell-free DNA (cfDNA) is poised to supplement, if not replace, surgical biopsies. Despite many studies attempted to identify new epigenetic targets for liquid biopsy assays, little is known about the regulatory roles of 5hmC, its impacts on the molecular phenotypes in tumors. Most importantly, whether the oncogenic-associated 5hmC signatures found in tumor tissues can be recapitulated in patient cfDNA. In this study, we performed the unbiased and simultaneous detection of 5mC and 5hmC whole-genome DNA modifications at base-resolution from two distinct cancer cohorts, from patients with bladder cancer or B-Cell lymphoma, their corresponding normal tissues, and cfDNAs from plasma. We analyzed tissue-specific methylation patters and searched for signatures in gene coding and regulatory regions linked to cancerous states. We then looked for methylation signatures in patients cfDNA to determine if they were consistent with the tumor-specific patterns. We determined the functional significance of 5hmC in tissue specific transcription and uncovered hundreds of tumor-associated 5hmC signatures. These tumor-associated 5hmC changes, particularly in genes and enhancers, were functionally significant in tumorigenesis pathways and correlated with tumor specific gene expression. To investigate if cfDNA is a faithful surrogate for tumor-associated 5hmC, we devised a targeted capture strategy to examine the alterations of 5hmC in cfDNA from patients with bladder cancer and lymphoma with sufficient sensitivity and specificity and confirmed that they recapitulated the patterns we observed in tumor tissues. Our results provide analytic validation of 5hmC as a cancer-specific biomarker. The methods described here for systematic characterization of 5hmC at functional elements open new avenues to discover epigenetic markers for non-invasive diagnosis, monitoring, and stratifying cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233a1b3a2e680f52000d36e68abfc546f2d4b33" target='_blank'>
              Global DNA methylomes reveal oncogenic-associated 5-hydroxylmethylated cytosine (5hmC) signatures in the cell-free DNA of cancer patients
              </a>
            </td>
          <td>
            Gabriel E Rech, Alyssa C Lau, Rachel L Goldfeder, Rahul Maurya, Alexey V Danilov, Chia-Lin Wei
          </td>
          <td>2025-01-15</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Natural Killer (NK) cell therapy has emerged as a transformative approach in cancer immunotherapy, driven by decades of research since their discovery in the 1970s. NK cells possess a unique ability to target tumor cells through antigen-independent recognition, which has been expanded with the discovery of antigen-specific memory and licensing mechanisms. This review explores the evolution of NK cell biology, highlighting key findings such as the enhanced response of NK cells upon re-exposure to cytomegalovirus (CMV), defying conventional immune paradigms. Recent advancements in gene editing, particularly with CRISPR technology, have led to the development of "armored" NK cells. For example, knock-in strategies have enabled the insertion of chimeric antigen receptors (CAR), enhancing tumor specificity, while knock-out approaches have improved NK cell activation by removing inhibitory molecules like PD-1 and NKG2A. Early clinical trials demonstrate improved tumor-targeting efficacy and reduced resistance. However, challenges remain, including high R&D costs, limited clinical data, and technical uncertainties such as off-target effects. In China, NK cell therapy is in its nascent stages, with increasing interest but limited clinical applications due to regulatory and funding barriers. This review provides a comprehensive analysis of NK cell evolution, CRISPR-based advancements, and future directions for overcoming current limitations and optimizing therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Based on the notion that hypomorphic germline genetic variants are linked to autoimmune diseases, we reasoned that novel targets for cancer immunotherapy might be identified through germline variants associated with greater T-cell infiltration into tumors. Here, we report that while investigating germline polymorphisms associated with a tumor immune gene signature, we identified PKCδ as a candidate. Genetic deletion of PKCδ in mice resulted in improved endogenous antitumor immunity and increased efficacy of anti-PD-L1. Single-cell RNA sequencing revealed myeloid cell expression of Prkcd, and PKCδ deletion caused a shift in macrophage gene expression from an M2-like to an M1-like phenotype. Conditional deletion of PKCδ in myeloid cells recapitulated improved tumor control that was augmented further with anti-PD-L1. Analysis of clinical samples confirmed an association between PRKCD variants and M1/M2 phenotype, as well as between a PKCδ KO-like gene signature and clinical benefit from anti-PD-1. Our results identify PKCδ as a candidate therapeutic target that modulates myeloid cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31ff12050b59b63f4c02e53380b59da37e0eb557" target='_blank'>
              PKCδ germline variants and genetic deletion in mice augment antitumor immunity through regulation of myeloid cells.
              </a>
            </td>
          <td>
            Kyle R Cron, A. Sivan, K. Aquino-Michaels, Andrea Ziblat, Emily F. Higgs, R.F. Sweis, Ruxandra Tonea, Seoho Lee, Thomas F Gajewski
          </td>
          <td>2025-01-14</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 226


 Background:
 Metastatic prostate cancer (PC) accumulates significant genomic alterations as the disease progresses, exhibiting the highest levels of chromosomal instability (CIN) among all metastatic tumors (Bakhoum, Nature 2018). Furthermore, tumor suppressor genes (TSG;
 RB1
 ,
 PTEN
 and
 TP53
 ) are frequently altered in metastatic PC and are associated with adverse outcomes. Here, we aim to investigate CIN in metastatic hormone-sensitive PC (mHSPC) harbouring alterations in TSG.
 Methods:
 TSG and whole transcriptome gene expression was assessed by nCounter platform (N=354) and RNA-seq (N=60) in mHPSC samples, respectively. TSG
 low
 was considered when ≥2 out of 3 TSG presented low expression of a previously stablished cut-off, and TSG
 wt
 in the remaining cases. Differential gene expression analysis between TSG
 low
 vs TSG
 wt
 tumors of our RNA-seq cohort (N=60) and TCGA cohort (N=333) was performed. Differential expressed genes (DEGs) were considered if FDR <0.05 and log2FoldChange (LFC)>0.5. DEGs were correlated with castration-resistance PC-free survival (CRPC-FS) and overall survival (OS) by Cox analysis. CIN gene signatures were obtained from literature and the rest of Hallmark signatures from MSigDB.
 Results:
 In our cohort of mHSPC patients treated with androgen deprivation therapy + docetaxel, TSG
 low
 tumors (16.7%) displayed higher levels of CIN compared with TSG
 wt
 (CIN25 normalized enrichment score [NES] 2.13, p<0.001 and CIN70 NES 2.54, p<0001). Cell cycle and DNA-repair signatures were also overexpressed in TSG
 low
 tumors, contrasting with a low expression of androgen receptor pathway. Results were validated in the TCGA cohort, where TSG
 low
 presented also overexpression of CIN signatures. In our RNA-seq cohort, we found 22 DEGs associated with CIN (29.3% of the total DEGs) in TSG
 low
 tumors compared with TSG
 wt
 . 51 genes were significantly correlated with CRPC-FS in a multivariate analysis and 16 of them (31,3%) were CIN-associated genes. Among these genes, 5 encode proteins that are targetable for metastatic PC, with specific inhibitors already developed:
 BUB1
 (LFC 1.06, FDR 0.034),
 CDC7
 (LFC 12.4 FDR 0.003),
 KIFC1
 , (LFC 11,7, FDR 0.01),
 PLK1
 (LFC 1.08, FDR <0.001) and
 WEE1
 (LFC 0.79, FDR 0.011).
 Conclusions:
 TSG-altered mHSPC tumors harbor high levels of CIN, and differentially expressed CIN-associated genes could potentially be targets in mHSPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719dc00daf58fbc383533083eb81aecec4828ea4" target='_blank'>
              Epigenetic marvels: exploring the landscape of colorectal cancer treatment through cutting-edge epigenetic-based drug strategies
              </a>
            </td>
          <td>
            A. Tahghighi, Effat Seyedhashemi, Javad Mohammadi, Arash Moradi, Aria Esmaeili, Majid Pornour, Kimia Jafarifar, S. M. Ganji
          </td>
          <td>2025-02-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) carries a poor prognosis, and new therapeutic strategies are necessary to improve outcomes for patients with this disease. Alkylating chemotherapies including temozolomide (TMZ) and lomustine (CCNU) are critical for treating GBM, but resistance mechanisms, including hypomethylation of O6-methylguanine-DNA methyltransferase (MGMT) promoter, undermine treatment. CRISPRoff is a programmable epigenetic memory editor that can induce stable and heritable gene silencing after transient delivery, and we hypothesize that CRISPRoff could potentiate the activity of TMZ and CCNU through long term suppression of target genes.


METHODS
We transiently delivered CRISPRoff mRNA along with sgRNAs against target genes using both electroporation and lipid nanoparticles (LNPs) into established GBM cell lines, patient-derived primary GBM cultures, and orthotopic GBM xenografts. Gene repression, specificity, and stability was measured by RT-qPCR, Western blot, bisulfite sequencing, and RNA-sequencing. Sensitivity to chemotherapies was measured by cell viability dose response, microscopy, and bioluminescence imaging. Genome-wide mapping of CCNU sensitizers was performed using CRISPRi screens.


RESULTS
CRISPRoff induced complete suppression of MGMT and sensitization to TMZ that were stable for over 8 months of continuous cell propagation. GBM orthotopic tumors treated with CRISPRoff against MGMT demonstrated sensitivity to TMZ in vivo, and CRISPRoff delivery resulted in chemosensitivity in patient-derived primary GBM. Genome-wide CRISPRi screens identified combinatorial genetic vulnerabilities (BRIP1, FANCE) that were targetable by multiplexed CRISPRoff to achieve sensitization to CCNU.


CONCLUSION
Transient delivery of a site-specific epigenetic memory can induce stable, complete, and multiplexed suppression of target genes for therapeutic application in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Background Neoadjuvant, endocrine, and targeted therapies have significantly improved the prognosis of breast cancer (BC). However, due to the high heterogeneity of cancer, some patients cannot benefit from existing treatments. Increasing evidence suggests that amino acids and their metabolites can alter the tumor malignant behavior through reshaping tumor microenvironment and regulation of immune cell function. Breast cancer cell lines have been identified as methionine-dependent, and methionine restriction has been proposed as a potential cancer treatment strategy. Methods We integrated transcriptomic and single-cell RNA sequencing (ScRNA-seq) analyses based on The Cancer Genome Atlas (TCGA) database and Gene Expression Omnibus (GEO) datasets. Then we applied weighted gene co-expression network analysis (WGCNA) and Cox regression to evaluate methionine metabolism-related genes (MRGs) in BC, constructing and validating a prognostic model for BC patients. Immune landscapes and immunotherapy were further explored. Finally, in vitro experiments were conducted to assess the expression and function of key genes APOC1. Results In this study, we established and validated a prognostic signature based on eight methionine-related genes to predict overall survival (OS) in BC patients. Patients were further stratified into high-risk and low-risk groups according to prognostic risk score. Further analysis revealed significant differences between two groups in terms of pathway alterations, immune microenvironment characteristics, and immune checkpoint expression. Our study shed light on the relationship between methionine metabolism and immune infiltration in BC. APOC1, a key gene in the prognostic signature, was found to be upregulated in BC and closely associated with immune cell infiltration. Notably, APOC1 was primarily expressed in macrophages. Subsequent in vitro experiments demonstrated that silencing APOC1 reduced the generation of tumor-associated macrophages (TAMs) with an M2 phenotype while significantly decreasing the proliferation, invasion, and migration of MDA-MB-231 and MDA-MB-468 breast cancer cell lines. Conclusion We established a prognostic risk score consisting of genes associated with methionine metabolism, which helps predict prognosis and response to treatment in BC. The function of APOC1 in regulating macrophage polarization was explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d865c02ec071f3b620933e096d65cab7e82d29" target='_blank'>
              Integrating single-cell RNA-seq and bulk RNA-seq to explore prognostic value and immune landscapes of methionine metabolism-related signature in breast cancer
              </a>
            </td>
          <td>
            Yanxian Gao, Ziyu Feng, Hailong Zhao, Xinghai Liu, Muyu Zhu, Xiafei Yu, Xiaoan Liu, Xian Wu, Jing Tao
          </td>
          <td>2025-01-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homology-directed repair (HDR)-based genome editing is an approach that could permanently correct a broad range of genetic diseases. However, its utility is limited by inefficient and imprecise DNA repair mechanisms in terminally differentiated tissues. Here, we tested Repair Drive, a platform technology for selectively expanding HDR-corrected hepatocytes in adult mice in vivo. Repair Drive involves transient conditioning of the liver by knocking down an essential gene, fumarylacetoacetate hydrolase (Fah), and delivering an untargetable version of the essential gene in cis with a therapeutic transgene. We show that Repair Drive increased the percentage of correctly targeted hepatocytes in healthy wild-type mice up to 25%, which resulted in a fivefold increased expression of a therapeutic transgene, human factor IX (FIX). Repair Drive was well tolerated and did not induce toxicity or tumorigenesis during a 1-year follow-up. This approach may broaden the range of liver diseases that can be treated with somatic genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5883dfe71ffdf910e9c20145e8db93ca834ac43" target='_blank'>
              In vivo expansion of gene-targeted hepatocytes through transient inhibition of an essential gene.
              </a>
            </td>
          <td>
            M. De Giorgi, So Hyun Park, Adam B. Castoreno, Mingming Cao, Ayrea Hurley, Lavanya Saxena, Marcel A Chuecos, Christopher J Walkey, Alexandria M. Doerfler, Mia N Furgurson, M. C. Ljungberg, Kalyani R Patel, Sarah Hyde, Tyler Chickering, Stephanie Lefebvre, Kelly Wassarman, Patrick Miller, June Qin, M. Schlegel, Ivan Zlatev, Jun Han, C. Beeton, R. Li, J. Kim, James F. Martin, K. Bissig, Vasant Jadhav, Gang Bao, William R. Lagor
          </td>
          <td>2025-02-12</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f20bcfbeb6577568a7ff02bdc23116269869d367" target='_blank'>
              TFAP2A-regulated CRNDE enhances colon cancer progression and chemoresistance via RIPK3 interaction
              </a>
            </td>
          <td>
            Xin Gao, Yanming Huang, Tonghui Wei, Jingmin Xue, Filippov Iurii, Laishou Yang, Liying Wang, Hao Li, Genshen Mo, Yuze Huang, Haonan Xie, Hang Wang, Shenghan Lou, Peng Han
          </td>
          <td>2025-02-15</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Purpose: Small cell lung cancer (SCLC) is a highly aggressive and deadly malignancy. Two major factors contributing to the high mortality of SCLC are early metastasis and rapid development of therapy resistance. Recent research suggests upregulation of the epithelial-mesenchymal transition (EMT) program and the EMT transcription factor Twist1 correlated with accelerated tumor progression and chemoradiation (CRT) resistance in SCLC. However, a causal relationship between Twist1 and these aspects of SCLC biology has not been rigorously studied. Here, we investigated whether Twist1 upregulation could promote SCLC tumorigenesis, metastasis, and resistance to CRT. Materials and Methods: We analyzed transcriptomic data from the IMpower133 phase 3 trial of extensive stage SCLC stratified by median TWIST1 levels using Kaplan-Meir statistic. To investigate the roles of Twist1 directly in SCLC biology, we have generated a novel genetically engineered mouse model (GEMM) termed RPGT (Rb1Flox; Trp53Flox; ROSA26LSL-rtTA-IRES-EGFP; Twist1-TetO7-Luc). The RPGT GEMM enables the generation of autochthonous SCLC tumors after induction with Cre recombinase adenovirus. By withdrawing or providing doxycycline to mice, we can control Twist1 expression to activate or inactivate EMT in tumors. By characterizing mouse tumor samples with histological, immunostaining, and transcriptomic profiling analyses, we evaluated the impact of Twist1 and EMT upregulation on SCLC tumor biology and plasticity as well as metastatic spread and outgrowth. To evaluate the impact of EMT activation on SCLC sensitivity to CRT, we performed in vitro viability assays on primary tumor cell lines established from RPGT tumors. We also treated RPGT mice with vs. without Twist1 overexpression with CRT to validate our in vitro data. Results: In the chemotherapy alone control arm of IMpower133 there was a trend towards a difference in overall survival (OS) between TWIST1-high or -low patients (p=025 by log-rank) and in the NMF1/NEUROD1 subtype, patients with TWIST1-high had statistically significantly inferior OS (p=0.01). We then observed that both control (no Twist1 overexpression) and Twist1-overexpressing RPGT mice developed neuroendocrine SCLC tumors. While SCLC-ASCL1 was the predominant subtype in both cohorts, RPGT tumors exhibited more plasticity, with features associated with SCLC-NEUROD1 subtype. Furthermore, Twist1 overexpression dramatically increased the metastatic incidence in RPGT compared to control animals, indicating an important role of EMT in SCLC dissemination. Transcriptomic profiling of primary tumors and matching metastases in RPGT mice also revealed downregulation of Twist1 and EMT in metastases, suggesting that EMT suppression was necessary for metastatic outgrowth. Furthermore, we found that repressing Twist1 expression enhanced SCLC susceptibility to CRT both in vitro and in vivo. Conclusions: Overall, our data suggest that TWIST1/Twist1 plays an important role in promoting SCLC plasticity, metastasis, and treatment resistance.
 Citation Format: Triet Nguyen, Jinhee Chang, Kathleen Gabrielson, Amol Shetty, Yang Song, Apaala Chatterjee, Audrey Lafargue, Yoo Sun Kim, Danielle Council, Aaron Chan, Dipanwita Dutta Chowdhury, Muhammad Ajmal Khan, Nick Connis, Daniel Sforza, Eric Gardner, Christopher McFarland, Mohammad Rezaee, Nitin Roper, Christine Hann, Phuoc T. Tran. TWIST1 Associates with resistance to treatment and Twist1 drives tumor progression in vivo for small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a1d1d7c8c25abea1123d9456e9598c5d89538a" target='_blank'>
              Abstract B029: TWIST1 Associates with resistance to treatment and Twist1 drives tumor progression in vivo for small cell lung cancer
              </a>
            </td>
          <td>
            Triet Nguyen, J. Chang, Kathleen L Gabrielson, A. Shetty, Yang Song, Apaala Chatterjee, A. Lafargue, Yoo Sun Kim, Danielle N. Council, Aaron Chan, D. Chowdhury, Muhammad Ajmal Khan, N. Connis, Daniel Sforza, Eric Gardner, Christopher McFarland, Mohammad Rezaee, Nitin Roper, C. Hann, Phuoc T Tran
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/051f9e93e924c3e30e3096e0cd89e4a17082501b" target='_blank'>
              Integrative analysis of genetic variability and functional traits in lung adenocarcinoma epithelial cells via single-cell RNA sequencing, GWAS, bayesian deconvolution, and machine learning.
              </a>
            </td>
          <td>
            Chenggen Gao, Jintao Wu, Fangyan Zhong, Xianxin Yang, Hanwen Liu, Junming Lai, Jing Cai, Weimin Mao, Huijuan Xu
          </td>
          <td>2025-02-24</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This review explores the progression of CRISPR-Cas9 genome editing technology in gene therapy for acute myeloid leukemia. Acute Myeloid Leukemia, a serious blood cancer, presents formidable treatment obstacles and a dismal prognosis. Effective treatments for acute myeloid leukemia rely on a clear understanding of its intricate genetic mechanisms, and CRISPR-Cas9 technology offers precision and adaptability for targeting the disease's oncogenic drivers. The dual intron-targeting method of CRISPR-Cas9 is showcased in this review for its potential to improve acute myeloid leukemia treatment efficacy and decrease relapse risks. This approach emphasizes targeting oncogenic driver genes in acute myeloid leukemia through the dual intron-targeting CRISPR-Cas9 methodology, which identifies gene-specific targets, designs constructs to hit intronic regions, and explores viral and non-viral techniques for introducing CRISPR-Cas9 into leukemia cells. Preclinical model and primary acute myeloid leukemia cell evaluations indicate precise targeting and editing of oncogen-ic driver genes, significant reduction in cell viability and proliferation, minimal off-target effects, and im-proved treatment outcomes for acute myeloid leukemia patients. Finally, gene therapy, particularly CRISPR-Cas9, holds transformative potential in reshaping acute myeloid leukemia treatment. Despite challenges in optimizing delivery methods, enhancing target specificity, and minimizing off-target effects, CRISPR-Cas9 shows great promise in improving treatment efficacy and reducing relapse risk. This innovative approach of-fers hope for better survival rates and outcomes for acute myeloid leukemia patients, marking a significant advancement in the fight against this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f5d2a19ddfe84eb40ce3d240e162476c8b6ecf" target='_blank'>
              CRISPR-Cas9 Is Revolutionizing AML Treatment As A Novel Therapy
              </a>
            </td>
          <td>
            Fakhra Alafeefi, Afsheen Raza
          </td>
          <td>2025-01-24</td>
          <td>Cancer Immunology Connect</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The epithelial–mesenchymal transition (EMT) is a dynamic transdifferentiation of epithelial cells into mesenchymal cells. EMT programs exhibit great diversity, based primarily on the distinct impact of molecular activities of the EMT transcription factors. Using a panel of cancer cell lines and a series of 71 triple-negative primary breast tumors, we report that the EMT transcription factor ZEB1 modulates site-specific chemical modifications of ribosomal RNA (rRNA). Overexpression of ZEB1 and ZEB2, but not TWIST1, decreased the level of 2′-O-ribose methylation (2′Ome) of 28S rRNA at position Um2402. ZEB1 overexpression specifically reduced the expression of the corresponding C/D box small nucleolar RNAs (snoRNAs) SNORD143/144, which guide the rRNA 2′Ome complex at the 28S_Um2402 site. During ZEB1-induced EMT induction/reversion, the levels of both 2′Ome at 28S_Um2402 and SNORD143/144 were dynamically comodulated. Taken together, these data demonstrate that 2′Ome rRNA epitranscriptomics is a novel marker of ZEB1-induced EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3736219d26e530108dcb3de1bf5b51cdbebaad" target='_blank'>
              Specific modulation of 28S_Um2402 rRNA 2′-O-ribose methylation as a novel epitranscriptomic marker of ZEB1-induced epithelial–mesenchymal transition in different mammary cell contexts
              </a>
            </td>
          <td>
            Chloé Morin, Hermes Paraqindes, Flora Nguyen Van Long, Caroline Isaac, Emilie Thomas, D. Pedri, Carlos Ariel Pulido-Vicuña, A. Morel, V. Marchand, Y. Motorin, Marjorie Carrere, Jessie Auclair, V. Attignon, Roxane M. Pommier, Emmanuelle Ruiz, Fleur Bourdelais, Frédéric Catez, Sébastien Durand, Anthony Ferrari, Alain Viari, J. Marine, Alain Puisieux, Jean-Jacques Diaz, C. Moyret-Lalle, V. Marcel
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Phenotypic outcomes can be heavily affected by environmental factors. In this study, we exploited the previously observed nutrient-dependency of cell biological phenotypic features, captured by a cross-condition image-based profiling assay of Escherichia coli deletion strains, to examine this in more detail. We identified several general principles, including the existence of a spectrum of deviating phenotypes across nutrient conditions (i.e., from nutrient- or feature-specific to pleiotropic phenotypic deviations), limited conservation of phenotypic deviations across nutrient conditions (i.e., limited phenotypic robustness), and a subset of nutrient-independent phenotypic deviations (indicative of consistent genetic determinants of specific phenotypic features). In a subsequent step, we used this cross-condition dataset to identify five genes of unknown function of which the deletion displayed either nutrient-independent phenotypic deviations or phenotypic similarities to genes of known function: yibN, yaaY, yfaQ, ybiJ, and yijD. These genes showed different levels of phylogenetic conservation, ranging from conserved across the tree of life (yibN) to only present in some genera of the Enterobacterales (yaaY). Analysis of the structural properties of the proteins encoded by these y-genes, identification of structural similarities to other proteins, and the examination of their subcellular localization yielded new insights into their contribution to E. coli cell morphogenesis, cell cycle progression and cell growth. Together, our approach showcases how bacterial image-based profiling assays and datasets can serve as a gateway to reveal the function of uncharacterized proteins. Importance Despite unprecedented access to genomic information, predicting phenotypes based on genotypes remains notoriously difficult. One major confounding factor is the environment and its ability to modulate phenotypic outcomes. Another is the fact that a large fraction of protein-coding genes in bacterial genomes remains uncharacterized and have no known function. In this work, we use a large-scale cross-condition image-based profiling dataset to characterize nutrient-dependent phenotypic variability of E. coli deletion strains and exploit it to provide insight into the cellular role of genes of unknown function. Through our analysis, we identified five genes of unknown function that we subsequently further characterized by examining their phylogenetic conservation, predicted structural properties and similarities, and their intracellular localization. Combined, this approach highlights the potential of cross-condition image-based profiling, which extracts many cell biological phenotypic readouts across multiple conditions, to better understand nutrient-dependent phenotypic variability and uncover protein function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e64fdb79587408307645dda0d8efcc1410cda8f" target='_blank'>
              Complex interplay between gene deletions and the environment uncovers cellular roles for genes of unknown function in Escherichia coli
              </a>
            </td>
          <td>
            Kaat Sondervorst, Kristina Nesporova, Matthew Herdman, Bart Steemans, Joëlle Rosseels, Sander K. Govers
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79d34836731e3bc98f26657fe4be3cd3ea9eb09" target='_blank'>
              The systemic evolutionary theory of the origin of cancer (SETOC): an update
              </a>
            </td>
          <td>
            Antonio Mazzocca, Giovanni Ferraro, Giovanni Misciagna
          </td>
          <td>2025-01-14</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a61455bde5450ac8c919b6a07222c913fc3cde" target='_blank'>
              Aberrant DNA methylation as a key modulator of cell death pathways: insights into cancer progression and other diseases.
              </a>
            </td>
          <td>
            Ambreen Zahoor, Rafia Khazer, Insha Mehraj, Ubaid Gani, Falah Fayaz, F. Khanday, S. S. Bhat
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Prostate cancer (PCa) is the most commonly diagnosed cancer and the second-leading cause of cancer death among men in the United States, representing 24.3% of all new cancer cases in the US Metastatic castration-resistant prostate cancer (mCRPC) is a lethal variant of prostate cancer that is associated with increased aggressiveness, cancer stemness, morbidity, and the risk of developing into taxane drug-resistant PCa. As taxanes are currently the first-line chemotherapeutic agents for mCRPC, there is a critical need to develop novel agents for the treatment of mCRPC. Clofazimine (CLF) is a potential immunomodulator drug that is FDA-approved for the treatment of leprosy. Recently, using a phenotype-based high-throughput drug screening, we demonstrated the in vitro (cell lines), in vivo (mouse xenograft models), and ex vivo (patient-derived primary tumor cells) efficacy of CLF in drug-resistant forms of chronic myeloid leukemia and multiple myeloma. In this study, we demonstrate that CLF is effective as a single agent and in combination with docetaxel (DTX) in a panel of PCa cell lines representing the diversity of CRPC patients. The response to CLF in the PCa cell lines was quantified using in vitro cytotoxicity assays, caspase 3/7 activity assay for apoptosis, cell cycle analysis, and aldehyde dehydrogenase activity assay for cancer stemness. We show that following CLF treatment, cellular pathways associated with apoptosis and ER stress were up-regulated, providing additional understanding of the mechanism of action of CLF in PCa cells. Further, we also found that CLF reduces ALDH activity, which is a marker for cancer cell stemness, a subtype of cancer cells that are associated with the aggressiveness of the cancer. Finally, drug-induced changes were investigated using whole transcriptome (next-generation RNA sequencing) analysis, where CLF treatment was found to modulate the expression of several non-coding RNAs that are associated with tumor cell proliferation, cell migration, and PCa drug resistance. Western Blotting was performed to validate the dysregulation of proteins associated with cell death and apoptosis, as well as the top dysregulated pathways in mCRPC following treatment with CLF. Our results support the preclinical development of CLF as a combination therapy candidate against aggressive forms of Prostate Cancer and provide additional insights into its mechanism of action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15bb80576bbd8d5340c20144f6b3a46d4763d6f" target='_blank'>
              Clofazimine treatment modulates key non-coding RNAs associated with tumor progression and drug resistance in lethal Prostate Cancer
              </a>
            </td>
          <td>
            Sarah Batten, Harish Kumar, Suman Mazumder, T. Ghosh, Amit Kumar
          </td>
          <td>2025-02-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A notable number of locally advanced cervical carcinoma (LACC) patients experience local or distant disease relapse following radiotherapy. The contribution of tumor microenvironment (TME) to tumor recurrence at different sites remains unclear. Here, single-nucleus RNA sequencing data from 28 pre- and on-treatment LACC samples from patients with different disease relapse patterns is analyzed. The findings revealed opposing alterations in the expression levels of the cellular senescence pathway after radiotherapy in patients with local and distant relapses. In contrast, an increase in the expression of the epithelial-mesenchymal transition module after radiotherapy in both relapse groups is observed. Cell-cell interactions, drug-target expression analyses in malignant cells after radiation, and multiplex immunofluorescence of tumor tissue identified interleukin-1 receptor type I (IL1R1) as a potential therapeutic target. It is demonstrated that combining the IL1R1 inhibitor anakinra with radiation can mitigate the effects of radiation on tumor cells. This study highlights the distinct roles of cellular senescence and EMT in tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbbbce00028dc96d46abb92302a5b678fc78bfd" target='_blank'>
              Radiotherapy-Associated Cellular Senescence and EMT Alterations Contribute to Distinct Disease Relapse Patterns in Locally Advanced Cervical Cancer.
              </a>
            </td>
          <td>
            Lei Zhang, Jun Ma, Jun Zhang, Minjie Hu, Jinlin Cheng, Bin Hu, Junjun Zhou, Di Zhou, Yongrui Bai, Xiumei Ma, Jianming Tang, Haiyan Chen, Ying Jing
          </td>
          <td>2025-02-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Purpose The complexity and heterogeneity of the tumor immune microenvironment (TIME) are linked to the development and poor prognosis of hepatocellular carcinoma (HCC). However, the cell type within the TIME that is most closely associated with HCC development remains unclear. Herein, we aimed to identify cell clusters that significantly contribute to HCC development and their underlying mechanisms. Method and Results Using single-cell RNA sequencing (scRNA-seq), we analyzed changes in the TIME of normal and tumor tissues, identifying plasma cells as the key cluster in HCC development. Based on plasma cell-related genes (PCRGs), we constructed and validated an eight-gene prognostic model (ST6GALNAC4, SEC61A1, SSR3, RPN2, PRDX4, TRAM1, SPCS2, CD79A) using internal and external datasets and a nomogram. Functional enrichment, miRNA network construction, and transcriptional regulation analyses were performed to explore underlying mechanisms. TIDE scores and the GDSC database were used to predict immunotherapy and chemotherapy sensitivity in different risk groups. Finally, SSR3’s biological function was validated in vitro in HCC cell lines. Conclusion Plasma cells are key clusters in HCC development. A prognostic model based on the PCRGs can accurately predict the prognosis of patients with HCC and guide clinical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/789d040bed058d7536fc99da4ae75b9692f60293" target='_blank'>
              Integrative Analysis of scRNA-Seq and Bulk RNA-Seq Identifies Plasma Cell Related Genes and Constructs a Prognostic Model for Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Mingyang Tang, Yuyan Xu, Mingxin Pan
          </td>
          <td>2025-02-01</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Constitutively active mutations of KRAS are prevalent in non-small cell lung cancer (NSCLC). However, the relationship between these mutations and resistance to platinum-based chemotherapy and the underlying mechanisms remain elusive. In this study, we demonstrated that KRAS mutants confer resistance to platinum in NSCLC. Mechanistically, KRAS mutants mediate platinum resistance in NSCLC cells by activating ERK/JNK signaling, which inhibits ALKBH5 m6A demethylase activity by regulating post-translational modifications (PTMs) of ALKBH5. Consequently, the KRAS mutant leads to a global increase in m6A methylation of mRNAs, particularly DDB2 and XPC, which are essential for nucleotide excision repair. This methylation stabilized the mRNA of these two genes, thus enhancing NSCLC cells' ability to repair platinum-induced DNA damage and avoid apoptosis, thereby contributing to drug resistance. Furthermore, blocking KRAS-mutant-induced m6A methylation, either by overexpressing a SUMOylation-deficient mutant of ALKBH5, or by inhibiting METTL3 pharmacologically, significantly sensitizes KRAS-mutant NSCLC cells to platinum drugs in vitro and in vivo. Collectively, our study uncovers a previously unrecognized mechanism that mediates KRAS mutant-induced chemoresistance in NSCLC cells by activating DNA repair through the modulation of the ERK/JNK/ALKBH5 PTMs-induced m6A modification in DNA damage repair-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b30cd92ed6de2a868acc5ae23a99dadaae25a64" target='_blank'>
              KRAS Mutants Confer Platinum Resistance by Regulating ALKBH5 Post-translational Modifications in Lung Cancer.
              </a>
            </td>
          <td>
            Fang Yu, Shikan Zheng, Chunjie Yu, Sanhui Gao, Zuqi Shen, Rukiye Nar, Zhexin Liu, Shuang Huang, Lizi Wu, Tongjun Gu, Zhijian Qian
          </td>
          <td>2025-02-04</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Precision medicine represents a transformative change in healthcare that offers the ability to discover information to design, validate, or alter medical decisions based on an individual's unique characteristics instead of on evidence for the average patient. This has brought a paradigm shift from the classic “one-size-fits-all” approach, where treatments were designed ignoring individual differences, to personalized care. Advancements in genomics have been pivotal for this change, enabling healthcare prviders to tailor therapies to a patient's unique genetic makeup. Efforts in personalizing treatment through genomics have been very successful in the treatment of highly heterogeneous diseases such as lung cancer, as it has been found that patients with specific mutations in certain genes often respond well to targeted therapies, while other mutations tend to be associated with drug sensitivity. This approach not only aims to maximize benefits but also to minimize unnecessary side effects, in order to deliver precise and effective care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32f2e6420632960bb2c7051162939d61a92183c" target='_blank'>
              Precision Medicine and Epigenetics: Personal Care for a Healthier Future
              </a>
            </td>
          <td>
            Andrea Aparicio
          </td>
          <td>2025-01-27</td>
          <td>2025 Pan Pacific Strategic Electronics Symposium (Pan Pacific)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer poses a major global health issue, profoundly affecting both mortality and morbidity rates across the world. A key obstacle in understanding the pathogenesis of colorectal cancer lies in its high inter-patient and spatial heterogeneity, making standard treatments ineffective. Commonly, the study on colorectal cancer relies on bulk RNA sequencing, offering an average gene expression profile for a heterogenous cell population. However, this approach obscures the heterogeneous characteristics of the cancer cells and hinders a comprehensive understanding of the complex interplay among different cell populations. Recently, the advent of single-cell RNA sequencing has been revolutionary, enabling researchers to analyze individual immune cells and overcome the limitations of bulk RNA sequencing. Through single-cell RNA sequencing, researchers have gained insights into the dynamic nature of the immune response to cancer and potential targets for immunotherapy. In this review, we discuss the technical aspects of single-cell RNA sequencing, the application of single-cell RNA sequencing in cancer immunology, and the potential of single-cell RNA in clinical settings. We believe that harnessing singlecell RNA sequencing in cancer research holds immense potential to drive the development of personalized immunotherapies, aiming to improve patient outcomes in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CART) cell therapy is an innovative form of targeted immunotherapy that has revolutionized the treatment of cancer. However, the durable response remains limited. Recent studies have shown that the epigenetic landscape of preinfusion CART cell products can influence response to therapy, and gene editing has been proposed as a solution. However, more work needs to be done to determine the optimal gene editing strategy. Genome-wide CRISPR screens have become popular tools to both investigate mechanisms of resistance and optimize gene editing strategies. Yet their application to primary cells presents many challenges. Here we describe a method to complete a genome-wide CRISPR knockout screen in CART cells from healthy donors. As a proof-of-concept model, we designed this method to investigate the development of exhaustion in CART cells targeting the CD19 antigen. However, we believe that this approach can be used to address a variety of mechanisms of resistance to therapy in different CAR constructs and tumor models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/995da50c8649d8341a6a265386cac02363ea1e3b" target='_blank'>
              Performing an In Vitro Genome-Wide CRISPR Knockout Screen in Chimeric Antigen Receptor T Cells.
              </a>
            </td>
          <td>
            Carli M. Stewart, Claudia Manriquez Roman, S. Kenderian
          </td>
          <td>2025-01-31</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c98636c9111085040b447b74deddfb7a8fbd77" target='_blank'>
              Multi-gene panel sequencing reveals the relationship between driver gene mutation and clinical characteristics in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jinmao Li, Xianyun Li, Hao Guo, Shuni Zhou, Hua Ding, Bo Li, Wei Li, Bo Zhang, Yi Zou
          </td>
          <td>2025-03-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, with prostate cancer, lung cancer, colorectal cancer, and breast cancer accounting for nearly half of all diagnoses. Despite advancements in cancer treatment, metastasis to distant organs continues to be the leading cause of cancer-related mortality. The progression of cancer involves the alteration of numerous genes, with dynamic changes in chromatin organization and histone modifications playing a critical role in regulating cancer-associated genes. Special AT-rich sequence-binding protein 1 (SATB1), a critical chromatin organizer, plays a pivotal role in cancer progression by regulating gene expression, chromatin remodeling, and cell signaling pathways. SATB1 binds to AT-rich DNA sequences, acting as a scaffold for chromatin-modifying enzymes and transcription factors, thus coordinating the regulation of extensive gene networks. Its overexpression has been implicated in a wide range of cancers and is associated with poor prognosis, aggressive tumor phenotypes, and enhanced epithelial–mesenchymal transition (EMT). Moreover, SATB1’s activity is modulated by microRNAs (miRNAs) and post-translational modifications, further contributing to its complex regulatory functions. Given its crucial involvement in cancer progression and metastasis, SATB1 has emerged as a promising target for novel therapeutic strategies. This review delves into the molecular mechanisms of SATB1 in cancer and explores potential therapeutic approaches for targeting this key regulator in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b48f08c648e41770b775be56070039b081ae8" target='_blank'>
              SATB1 in cancer progression and metastasis: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Jinping Bai, Gege Yang, Qi Yu, Qianya Chi, Xianlu Zeng, Wenjing Qi
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Recent investigations have indicated that circRNAs (circular RNAs), a novel member of the Noncoding RNA family, are a crucial genetic factor in Colorectal Cancer (CRC), development. These circRNAs have a range of biological functions, including regu-lating the expression of target genes, altering protein activity, and producing proteins. Addi-tionally, circRNAs significantly affect tumor cell proliferation, aggression, migration, and cell death. CircRNAs have an essential role in carcinogenesis, metastasis, and drug toler-ance. Various signaling pathways are linked by circRNAs, providing opportunities for a more tailored chemotherapy regimen as well as early diagnosis and prognosis determination. By identifying new and potent circRNAs in signaling pathways, we can aim for them with ASO or siRNAs and provide a powerful combination therapy. In this review, we strive to summarize the biological and carcinogenic functions of circRNAs and their impact on spe-cific signaling pathways like PI3K/AKT, MAPK, Notch, JAK/STAT, Hippo/YAP, and WNT/β-catenin which lead to Epithelial-mesenchymal-transition(EMT) in CRC. Further-more, we explore the clinical significance of circRNAs and important signaling proteins in treating CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b02a5edf5833a9413d878e11e7ef959bc982d0" target='_blank'>
              Embracing the Circular Journey of RNA in Colorectal Cancer.
              </a>
            </td>
          <td>
            A. Barjasteh, Ayat Heidar Abdolamir, Abdulridha Mohammed Al-Asady, Hanieh Latifi, Amir Avan, Majid Khazaei, Mikhail Ryzhikov, S. M. Hassanian
          </td>
          <td>2025-03-03</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) analysis offers a powerful, non-invasive approach to cancer diagnostics and monitoring by revealing tumor-specific genomic and epigenetic alterations. Here, we demonstrate the versatility of MeD-seq, a methylation-dependent sequencing assay, for comprehensive cfDNA analysis, including methylation profiling, chromosomal copy number (CN) alterations, and tumor fraction (TF) estimation. MeD-seq-derived CN profiles and TF estimates from 38 colorectal cancer with liver metastases (CRLM) and 5 ovarian cancer patients were highly comparable to shallow whole-genome sequencing (sWGS) validating our approach. These findings establish MeD-seq as a robust and cost-effective platform for detecting cancer-specific signals directly from plasma without prior tissue-based information. Future work should expand its application to other cancer types, solidifying MeD-seq as a versatile tool for minimally-invasive cfDNA profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8a8bea4181cf5b880c2869962e79354192f059" target='_blank'>
              From Background to Signal: Simultaneously derived Copy Numbers from Methylation-Dependent Sequencing cell-free DNA
              </a>
            </td>
          <td>
            Daan M. Hazelaar, R. Boers, J. Boers, Vanja de Weerd, M. Jansen, Henk M. Verheul, Cornelis Verhoef, Joost Gribnau, John W M Martens, Stavros Makrodimtris, S. Wilting
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background The mechanisms of epigenetic regulation emerge as a fundamental determinant in the complex landscape of cancer initiation and advancement. However, the specific impact of epigenetic regulation on cancer progression remains unclear. To explore the relationship between epigenetic regulation and cancer progression, we utilized transcriptomic data from The Cancer Genome Atlas (TCGA) datasets to investigate the association. Methods We obtained transcriptomic data of epigenetic gene dataset from the TCGA database and calculated an epigenetic score using the Least Absolute Shrinkage and Selection Operator (LASSO) Cox model. Additionally, we created a nomogram that integrates the epigenetic score and clinical features, providing a more comprehensive tool for tumor patients prognosis assessment. Results We calculated the epigenetic score based on the expression levels of epigenetic-related genes. The nomogram we developed incorporates the epigenetic score and clinical characteristics. The epigenetic score was positively correlated with the expression of genes related to hallmarkers of cancer, including glycolysis, epithelial-mesenchymal transition (EMT), cell cycle, DNA repair, angiogenesis, and inflammatory response. Furthermore, we performed gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG) analysis to explore the signaling pathways and biological processes in high epigenetic score group. Conclusion The epigenetic scoring system developed in this investigation represents an innovative approach that demonstrates remarkable potential in forecasting survival trajectories across diverse cancer types. These groundbreaking insights not only illuminate the intricate interactions between epigenetic mechanisms and gene expression regulation in oncological contexts, but also indicate that the derived epigenetic metric could potentially emerge as a significant prognostic biomarker for cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b405c467d91505b22b6bd6b604ad22bfbaf352f" target='_blank'>
              Exploring the role of epigenetic regulation in cancer prognosis with epigenetic score
              </a>
            </td>
          <td>
            Ruiguang Zhang, Shimin Jin, Qi Xu, Rongxiao Dai
          </td>
          <td>2025-02-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) comprise a group of genetically heterogeneous hematological malignancies that result in the abnormal growth of leukemic cells and halt the maturation process of normal hematopoietic stem cells. Despite using molecular and cytogenetic risk classification to guide treatment decisions, most AML patients survive for less than five years. A deeper comprehension of the disease’s biology and the use of new, targeted therapy approaches could potentially increase cure rates. RAS oncogene mutations are common in AML patients, being observed in about 15–20% of AML cases. Despite extensive efforts to find targeted therapy for RAS-mutated AMLs, no effective and tolerable RAS inhibitor has received approval for use against AMLs. The frequency of RAS mutations increases in the context of AMLs’ chemoresistance; thus, novel anti-RAS strategies to overcome drug resistance and improve patients’ therapy responses and overall survival are the need of the hour. In this article, we aim to update the current knowledge on the role of RAS mutations and anti-RAS strategies in AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607005d4dc391e9d5c0e31b65105281e28950846" target='_blank'>
              Rat Sarcoma Virus Family Genes in Acute Myeloid Leukemia: Pathogenetic and Clinical Implications
              </a>
            </td>
          <td>
            Shaimaa Khattab, Adriatik Berisha, Natalia Baran, P. Piccaluga
          </td>
          <td>2025-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05abf56ac7febcd59ec911620962ed978222a49b" target='_blank'>
              Copy number amplification of FLAD1 promotes the progression of triple-negative breast cancer through lipid metabolism
              </a>
            </td>
          <td>
            Xiao-Qing Song, Tian-Jian Yu, Ou-Yang Yang, Jianlong Ding, Yizhou Jiang, Zhi-Ming Shao, Yi Xiao
          </td>
          <td>2025-02-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that continued expression is required for maintaining their leukemic phenotype and they therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20366f8a8e956d20a8164d2739ca733da77de50e" target='_blank'>
              RUNX1::RUNX1T1 Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. Daniel, Mata Casimiro, Luca van den Brink, Anita T. van Oort, Minoo Ashtiani, C. Zwaan, Anja Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, H. J. Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast Cancer (BrCa) exhibits a high phenotypic heterogeneity, leading to the emergence of aggressive clones and the development of drug resistance. Considering the BrCa heterogeneity and that metabolic reprogramming is a cancer hallmark, we selected seven BrCa cell lines with diverse subtypes to provide their comprehensive metabolome characterization: five lines commonly used (SK-Br-3, T-47D, MCF-7, MDA-MB-436, and MDA-MB-231), and two patient-derived xenografts (Hbcx39 and Hbcx9). We characterized their endometabolomes using 1H-NMR spectroscopy. We found distinct metabolite profiles, with certain metabolites being common but differentially accumulated across the selected BrCa cell lines. High levels of glycine, lactate, glutamate, and formate, metabolites known to promote invasion and metastasis, were detected in all BrCa cells. In our experiment setting were identified unique metabolites to specific cell lines: xanthine and 2-oxoglutarate in SK-Br-3, 2-oxobutyrate in T-47D, cystathionine and glucose-1-phosphate in MCF-7, NAD+ in MDA-MB-436, isocitrate in MDA-MB-231, and NADP+ in Hbcx9. The unique and enriched metabolites enabled us to identify the metabolic pathways modulated in a cell-line-specific manner, which may represent potential candidate targets for therapeutic intervention. We believe this study may contribute to the functional characterization of BrCa cells and assist in selecting appropriate cell lines for drug-response studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4c21e05117b322ca21c171873276565e0f755d" target='_blank'>
              Metabolic Profiling of Breast Cancer Cell Lines: Unique and Shared Metabolites
              </a>
            </td>
          <td>
            Mariana Gallo, Elena Ferrari, F. Brugnoli, Anna Terrazzan, Pietro Ancona, Stefano Volinia, V. Bertagnolo, Carlo M. Bergamini, A. Spisni, T. Pertinhez, Nicoletta Bianchi
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Background: Patients with metastatic castration-resistant prostate cancer (mCRPC) are generally resistant to immune checkpoint inhibitors (ICIs), partly due to the presence of immunosuppressive myeloid cells within tumors. However, broad targeting strategies, such as CSF1R antagonism, have faced clinical limitations given the heterogeneity of these cells. We hypothesize that a more precise understanding of specific immunosuppressive myeloid subsets and their molecular mechanisms at the single-cell level will be essential in the development of effective immunotherapies. Methods: We performed single-cell profiling on tumor biopsies across patients with localized (n=13), metastatic hormone-sensitive (n=24), and metastatic castration-resistant (n=6) prostate cancer. Results from these analyses were reverse-translated into multi syngeneic mouse models of prostate cancer, where we conducted multi-omic single-cell assessments and mechanistic studies. Findings were subsequently correlated with clinical outcomes. Results: Single-cell analysis of patient biopsies identified a distinct subset of tumor-associated macrophages with high expression of SPP1 (SPP1 hi-TAMs), encoding the osteogenic protein osteopontin (OPN), which were more prevalent in mCRPC than in earlier disease stages. These macrophages showed elevated immunosuppressive molecular programs and significantly lower CSF1R expression than other macrophages, potentially explaining the limited efficacy of CSF1R blockade in mCRPC patients. Multi-omic profiling of syngeneic mouse models of prostate cancer (MyC-CaP and TRAMP-C2) identified an analogous macrophage subset resistant to CSF1R blockade, which strongly suppressed CD8+ T cell activity in co-culture. Adoptive transfer of Spp1 hi-TAMs into CRPC increased the frequency of exhausted CD8+ T cells, promoted ICI resistance and worsened survival outcomes. Pathway analysis implicated adenosine signaling as a possible mechanism. Blocking adenosine pathways, either with the A2AR inhibitor ciforadenant or a CD73-blocking antibody, significantly reversed CD8+ T cell suppression by Spp1 hi-TAMs in vitro and enhanced the sensitivity of CRPC cells to PD-1 blockade in vivo. Moreover, we observed a reduction in Spp1 hi-TAMs following A2AR blockade, consistent with the high expression of Adora2a transcripts (encoding A2AR) and activation of downstream pathways, suggesting a feed-forward loop of adenosine pathways contribute to ICI resistance mediated by SPP1 hi-TAMs. Clinical correlation showed that a subset of mCRPC patients treated with combined adenosine receptor and PD-L1 blockade had improved anti-tumor efficacy compared to those on monotherapy, with SPP1 hi-TAMs potentially serving as predictive biomarkers for treatment response. Conclusions: Our studies reveal a significant increase in SPP1 hi-TAMs abundance with disease progression, contributing to ICI resistance through adenosine pathways. These findings support the therapeutic targeting of SPP1 hi-TAMs and their associated pathways as promising strategies to overcome resistance.
 Citation Format: Aram Lyu, Zenghua Fan, Matthew Clark, Averey Lea, Diamond Luong, Ali Setayesh, Alec Starzinski, Rachel Wolters, Marcel Arias-Badia, Kate Allaire, Kai Wu, Vibha Gurunathan, Laura Valderrábano, Xiao Wei, Richard Miller, Eliezer Van Allen, Lawrence Fong. A feed-forward loop in the adenosine signaling pathway drives myeloid-mediated resistance to immune checkpoint inhibition in prostate cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d917567d521f771de6596d03f0f8c85e1a9bdcbb" target='_blank'>
              Abstract PR003: A feed-forward loop in the adenosine signaling pathway drives myeloid-mediated resistance to immune checkpoint inhibition in prostate cancer
              </a>
            </td>
          <td>
            Aram Lyu, Zenghua Fan, Matt Clark, Averey Lea, Diamond Luong, Ali Setayesh, Alec Starzinski, Rachel Wolters, Marcel Arias-Badia, Kathryn M Allaire, Kai Wu, Vibha Gurunathan, Laura Valderrábano, Xiao X Wei, Richard A. Miller, E. Van Allen, Lawrence Fong
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Spatial organization of the genome is fundamental for ensuring accurate gene expression. This process depends on the communication between gene promoters and distal cis-regulatory elements (CREs), which together make up 8% of the human genome and are supported by the chromatin structure. It is estimated that over 90% of disease-associated variants are located in the non-coding region of the genome and may affect CRE. For the cystic fibrosis transmembrane conductance regulator (CFTR) gene, a complete understanding of tissue-specific CFTR expression and regulation is missing, in particular in the pancreas. Mechanistic insights into tissue-specific expression may provide clarity on the clinical heterogeneity observed in Cystic Fibrosis and CFTR-related disorders. Methods To understand the role of 3D chromatin architecture in establishing tissue-specific expression of the CFTR gene, we mapped chromatin interactions via circular chromosome conformation capture (4C) and epigenomic regulation through H3K27ac and DNase Hypersensitive site I (DHS) in Capan-1 pancreatic cells. Candidate regulatory regions are validated by luciferase reporter assay and CRISPR-knock out. Results We identified active regulatory regions not only around the CFTR gene but also outside the topologically associating domain (TAD). By performing functional assays, we validated our targets and revealed a cooperative effect of the −44 kb, −35 kb, +15.6 kb and 37.7 kb regions, which share common predicted transcription factor (TF) motifs. Comparative 3D genomic analysis and functional assays using the Caco-2 intestinal cell line revealed the presence of tissue-specific CREs. Conclusion By studying the chromatin architecture of the CFTR locus in Capan-1 cells, we demonstrated the involvement of multiple CREs upstream and downstream of the CFTR gene. We also extend our analysis to compare intestinal and pancreatic cells and provide information on the tissue-specificity of CRE. These findings highlight the importance of expanding the search for causative variants beyond the gene coding sequence but also by considering the tissue-specific 3D genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9279897acf411b562291c89812e3d397609427" target='_blank'>
              New insights into the cis-regulation of the CFTR gene in pancreatic cells
              </a>
            </td>
          <td>
            Clara Blotas, A. Le Nabec, M. Collobert, Mattijs Bulcaen, Marianne S. Carlon, Claude Férec, S. Moisan
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. We aim to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumours in various organs. Lineage tracing and clonal heterogeneity were evaluated using the state-of-the-art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. To identify melanoma clones across different metastatic sites, specifically subcutaneous, lungs and brain, C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7PV1 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor the tumour growth of mice injected intravenously and intracranially. Subcutaneous tumours were measured by calliper. Mice were euthanized before ethical endpoints. Tumors were harvested and single cell analysis as well as DNA sequencing was performed to identify barcodes expressed by the tumor cells across different tumor-sites. Barcode analysis was conducted both before cell implantation and at tumor harvest. Clonality exhibited significant variability between sites. Single-cell RNA sequencing will unveil states associated with dominant clones present at different metastatic sites. This information will provide a new understanding of melanoma transcriptional phenotype behaviours and biological characteristics at different metastatic sites and in response to therapy.
 Citation Format: Veronica Aedo-Lopez. Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Basal-like breast cancer (BLBC) is characterized by an aggressive clinical course, high genomic instability, and limited therapeutic options. The Inhibitor of Differentiation 4 (ID4) protein has been identified as a critical regulator of BLBC, where its overexpression correlates with poor prognosis. However, the mechanistic contributions of ID4 to BLBC tumorigenesis remain incompletely understood. In this study, we employed an integrative approach combining CRISPR-Cas9-mediated ID4 knockout, small-molecule inhibition, in vivo tumor modeling, and in silico transcriptional analyses to investigate the functional role of ID4 in BLBC. CRISPR-Cas9-mediated knockout of ID4 in MDA-MB-231 cells resulted in significant reductions in proliferation, colony formation, and Ki67 expression, indicating a loss of aggressive phenotypic traits. In vivo xenograft studies further revealed that ID4-silenced cells exhibited markedly delayed tumor formation and a significant reduction in metastatic potential compared to controls. Kaplan-Meier survival analysis of basal-like tumors from The Cancer Genome Atlas (TCGA) dataset demonstrated that patients with low ID4 expression had improved relapse-free survival. Gene set enrichment analysis (GSEA) of BLBC tumors stratified by ID4 expression revealed a shift toward luminal-like transcriptional programs in the ID4-low subgroup, including increased estrogen response and inflammatory signaling pathways. Furthermore, transcription factor activity analysis identified the activation of MYC, JUN, and STAT in ID4-low tumors, suggesting a transition toward a more differentiated phenotype. Finally, pharmacological inhibition of ID4 using the small-molecule degrader AGX51 effectively reduced proliferation in TNBC cells, highlighting ID4 as a potential therapeutic target. Together, these findings establish ID4 as a key driver of BLBC aggressiveness and suggest that its inhibition may represent a viable therapeutic strategy. This study provides compelling evidence supporting the development of ID4-targeted therapies for TNBC patients, with the potential to improve clinical outcomes in this challenging disease subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147bfd57152033ce1c79fd9d00f386b5bffcdc1b" target='_blank'>
              Exploring ID4 as a Driver of Aggression and a Therapeutic Target in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            C. Toro, S. Real, S. Laurito, MT Branham
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="As a significant cause of global mortality, the cancer has also economic impacts. In the era of cancer therapy, mitigating side effects and costs and overcoming drug resistance is crucial. Microbial species can grow inside the tumor microenvironment and inhibit cancer growth through direct killing of tumor cells and immunoregulatory effects. Although microbiota or their products have demonstrated anticancer effects, the possibility of acting as pathogens and exerting side effects in certain individuals is a risk. Hence, several genetically modified/engineered bacteria (GEB) have been developed to this aim with ability of diagnosing and selective targeting and destruction of cancers. Additionally, GEB are expected to be considerably more efficient, safer, more permeable, less costly, and less invasive theranostic approaches compared to wild types. Potential GEB strains such as Escherichia coli (Nissle 1917, and MG1655), Salmonella typhimurium YB1 SL7207 (aroA gene deletion), VNP20009 (∆msbB/∆purI) and ΔppGpp (PTet and PBAD), and Listeria monocytogenes Lmat-LLO have been developed to combat cancer cells. When used in tandem with conventional treatments, GEB substantially improve the efficacy of anticancer therapy outcomes. In addition, public acceptance, optimal timing (s), duration (s), dose (s), and strains identification, interactions with other strains and the host cells, efficacy, safety and quality, and potential risks and ethical dilemmas include major challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba6f5c96035da6ef412a12d773af66d8566a484" target='_blank'>
              Genetically Engineered Bacteria as a Promising Therapeutic Strategy Against Cancer: A Comprehensive Review.
              </a>
            </td>
          <td>
            Z. Zahedifard, Shirin Mahmoodi, A. Ghasemian
          </td>
          <td>2025-02-21</td>
          <td>Biotechnology and applied biochemistry</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5668e09864551d73b8ef36f697d610eddc6e8f2b" target='_blank'>
              MircoRNAs predict and modulate responses to chemotherapy in leukemic patients.
              </a>
            </td>
          <td>
            D. Ellakwa, Mohamed Ahmed Abdelmalek, Mostafa M Mostafa, Takwa E. Ellakwa, Al-Hassan Soliman Wadan
          </td>
          <td>2025-01-14</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>3</td>
          <td>12</td>
        </tr>

        <tr id="Copy number variations of the human CNTN6 gene, resulting from megabase-scale microdeletions or microduplications in the 3p26.3 region, are frequently implicated in neurodevelopmental disorders such as intellectual disability and developmental delay. However, duplication of the full-length human CNTN6 gene presents with variable penetrance, resulting in phenotypes that range from neurodevelopmental disorders to no visible pathologies, even within the same family. Previously, we obtained a set of induced pluripotent stem cell lines derived from a patient with a CNTN6 gene duplication and from two healthy donors. Our findings demonstrated that CNTN6 expression in neurons carrying the duplication was significantly reduced. Additionally, the expression from the CNTN6 duplicated allele was markedly lower compared to the wild-type allele. Here, we first introduce a system for correcting megabase-scale duplications in induced pluripotent stem cells and secondly analyze the impact of this correction on CNTN6 gene expression. We showed that the deletion of one copy of the CNTN6 duplication did not affect the expression levels of the remaining allele in the neuronal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c9c747915f6b48c8d45b21bda1f171cd233c0f" target='_blank'>
              Targeted correction of megabase-scale CNTN6 duplication in induced pluripotent stem cells and impacts on gene expression
              </a>
            </td>
          <td>
            Maria Gridina, Polina Orlova, Oleg Serov
          </td>
          <td>2025-01-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non-coding RNA (ncRNA) has been recognized to be an essential regulator of cellular processes and gene expression in cancer. The present study covers the various roles of ncRNAs, including circular RNAs (circRNAs), long non-coding RNAs (lncRNAs), and microRNAs (miR-NAs), that affect cancer properties. Oncogenesis, metastasis, and treatment resistance are all pro-cesses involving ncRNAs, which have tremendous potential as new therapeutic agents and tar-gets. The review covers the broad spectrum of ncRNAs in cancer biology, including their types and activities, epigenetic control, function in metastasis and angiogenesis, detection and profiling ap-proaches, potential as biomarkers, and therapeutic possibilities. Recent advancements in next-gener-ation sequencing and other molecular methods have helped us better understand how ncRNAs work and their potential therapeutic uses. However, there are still challenges to standardizing detection technologies and producing effective RNA-based therapeutics. Therefore, further studies are needed to solve important issues in this sector. Standardization efforts are also essential to developing iden-tical methods for ncRNA collection, quantification, and analysis throughout multiple laboratories and ensuring the findings are reliable and comparable. Large-scale, multi-recentre studies are required to verify the diagnostic usefulness of ncRNA biomarkers across a wide range of patient groups. Also, more detailed mechanistic knowledge is necessary for understanding the particular molecular mech-anisms by which ncRNAs affect cancer growth, metastasis, and treatment response. This review high-lights the complex relationships between ncRNAs and cancer biology and also focuses on their po-tential effect on cancer diagnosis and treatment. It also highlights the necessity for more studies to fully understand the therapeutic potential of ncRNAs in cancer. As studies advance, using ncRNA results in clinical practice might change cancer treatment by novel opportunities for specific therapy and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d029995f049d5872068ca5f1e2ab3843bccfc74" target='_blank'>
              An Overview of Unlocking the Power of Non-Coding RNAs in Revolutionizing Cancer Therapy.
              </a>
            </td>
          <td>
            Sagarika Kabra, Neetu Sharma, Ankita Kumari, Saba Khan, Rashmi Pathak, Himanshu Sharma, Abdullah Al Noman
          </td>
          <td>2025-02-26</td>
          <td>Current drug discovery technologies</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The domains rearranged methyltransferases (DRMs) play a critical role in the RNA-directed DNA methylation (RdDM) pathway in plants. However, the effects of inactivating the RdDM pathway on gene expression, transposable element (TE) activity, and phenotype in soybean remain unexplored. Here, we employed clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 gene editing to generate a quintuple mutant line in soybean (Gmdrm2a-/-2b-/-2c-/-3a-/-3b-/-, designated Gmdrm). Gmdrm exhibited severe developmental abnormalities, including dwarfism and delayed growth, albeit remaining viable and fertile; however, the fully homozygous mutant could be maintained for a limited number of generations (T0-T3). Whole genome bisulfite sequencing revealed a significant reduction in DNA methylation across all cytosine sequence contexts, with an average loss of 10%. The loss of mC was biased toward euchromatic regions, which is in contrast to the chromomethylase mutant. Transcriptome profiling identified 1,685 up-regulated genes, including photosynthesis-related genes, accompanied with altered chloroplast ultrastructure. Additionally, a cluster of resistance (R) genes on chromosome 16 was significantly up-regulated, coinciding with their reduced non-CG methylation. We also observed 3,164 differentially expressed TEs (DETs), of which, 2,655 were up-regulated and hypomethylated along their entire length. A substantial reduction in the abundance of 24-nt small interfering RNAs (siRNAs) in the Gmdrm mutant was detected by small RNA sequencing. Of note, the DRM-targeted TEs typically display higher levels of 24-nt siRNA abundance, shorter lengths, and are more AT-rich compared to chromomethylase-targeted TEs, highlighting 24-nt siRNAs as key determinants of DRM-dependent TE regulation. Together, this study documents a critical role of DRM-mediated DNA methylation in regulating gene expression, TE silencing, and normal development in soybean.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47287c3d47ac736b24fe418c263277c4788b373b" target='_blank'>
              Domains rearranged methyltransferases (DRMs)-mediated DNA methylation plays key roles in modulating gene expression and maintaining transposable element silencing in soybean.
              </a>
            </td>
          <td>
            Hongwei Xun, Lijie Lian, Jing Yuan, Jianhui Hong, Shanmeng Hao, Haonan Zhao, Shuhan Liu, Wanjie Feng, Huanran Yin, Bao Liu, Xutong Wang
          </td>
          <td>2025-03-07</td>
          <td>Journal of integrative plant biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), the sixth most prevalent cancer globally, is characterized by high recurrence rates and poor prognosis. Investigating the heterogeneity of relapsed HCC and identifying key therapeutic targets may facilitate the design of effective anticancer therapies. In this study, integrative analysis of single-cell RNA sequencing data of primary and early-relapsed HCC revealed increased proportions of infiltrating CD8+ T cells along with malignant cells and a decrease in CD4+ T cells in relapsed HCC. Cellular interaction and immunohistochemical analysis proposed MIF-(CD74 + CXCR4) signaling pathway as a key mechanism by which malignant cells influence immune cells within the tumor microenvironment. Notably, primary malignant cells showed greater differentiation and proliferation potential, whereas relapsed cells exhibited enhanced epithelial-mesenchymal transition and inflammation, along with upregulated glycogen synthesis and metabolism-related gene expression. Using machine learning techniques on bulk RNA-seq data, we developed a relapsed tumor cell-related risk score (RTRS) that independently predicts overall and recurrence-free survival time with higher accuracy compared with conventional clinical variables. Prognostic biomarkers and potential therapeutic targets were validated via RT-qPCR using mouse implantation models. This comprehensive investigation elucidates the heterogeneity of relapsed HCC and constructs a novel postoperative recurrence prognostic model, paving the way for targeted therapies and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1cb3b766dd5c2064ddcc362b2b05b2d16bf869" target='_blank'>
              Exploration of heterogeneity and recurrence signatures in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Wen-Jing Wu, Jianchao Wang, Fuqing Chen, Xuefeng Wang, Bin Lan, Ruyi Fu, Hong Wen, Fangfang Chen, Wei Hong, , Ying He, Gang Chen, Jianyin Zhou, Hai-Long Piao, Di Chen, Shuyong Lin
          </td>
          <td>2025-02-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3848f39931c448a59bb24bbf06c225042530e10f" target='_blank'>
              TNBC response to paclitaxel phenocopies interferon response which reveals cell cycle-associated resistance mechanisms
              </a>
            </td>
          <td>
            Nicholas L. Calistri, T. Liby, Zhi Hu, Hongmei Zhang, Mark A. Dane, Sean M. Gross, Laura M. Heiser
          </td>
          <td>2025-02-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Human gene expression is controlled from distance via enhancers, which can form longer ‘super-enhancer’-regions of intense regulatory activity. Whether super-enhancers constitute a separate regulatory paradigm remains unclear, largely due to the difficulty of dissecting the contributions and interactions of individual elements within their natural chromosomal context. To address this challenge, we developed enhancer scrambling, a high-throughput strategy to generate stochastic inversions and deletions of targeted enhancer regions by combining CRISPR prime editing insertion of symmetrical loxP sites with Cre recombinase-induced rearrangements. We applied our approach to dissect a distal super-enhancer of the OTX2 gene, generating up to 134 alternative regulatory configurations in a single experiment, and establishing how they drive gene expression and chromatin accessibility, as well as the individual contributions of its elements to this activity. Surprisingly, the presence of the sequence containing a single DNase I hypersensitive site predominantly controls OTX2 expression. Our findings highlight that enhancer-driven regulation of some highly expressed, cell-type-specific genes can rely on an individual element within a cluster of non-interacting, dispensable components, and suggest a simple functional core to a subset of super-enhancers. The targeted randomisation method to scramble enhancers can scale to resolve many super-enhancers and human gene regulatory landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cecb86ee79d0fa4af20cbd670e19b80d640e006e" target='_blank'>
              Resolution of a human super-enhancer by targeted genome randomisation
              </a>
            </td>
          <td>
            Jonas Koeppel, Pierre Murat, Gareth Girling, Elin Madli Peets, Mélanie Gouley, Valentin Rebernig, Anant Maheshwari, Jacob Hepkema, Juliane Weller, Jenie Hannah Johnkingsly Jebaraj, Ronnie Crawford, Fabio Giuseppe Liberante, Leopold Parts
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) accumulate somatic mutations over time, some conferring a fitness advantage that can lead to clonal hematopoiesis (CH). Mutations in DNMT3A, particularly at hotspot R882, are the most prevalent in CH and carry an increased risk of acute myeloid leukemia (AML). Although DNMT3A R882 mutations are linked to global DNA hypomethylation, the mechanisms underlying their selective advantage remain unclear. Here, we show that Dnmt3a-R882H mutant HSCs exhibit resilience under inflammatory and genotoxic stress. During IL-1β-induced emergency granulopoiesis, Dnmt3a R882H/+ HSCs uncouple increased proliferation from stem cell exhaustion. In contrast, wild-type HSCs rapidly progress to terminal differentiation. We link this phenotype to a delayed activation of the p53-p21-DREAM axis, that allows mutant HSCs to avoid attrition, despite increased replication stress. Similarly, mutant HSCs exhibit delayed Trp53 activation following irradiation, but eventually recover a physiological Trp53 response. Analysis of patient data reveals shared phenotypic features between DNMT3A and monoallelic TP53 mutations in CH and myeloid neoplasms, highlighting potential functional similarities. Collectively, these findings suggest that the expansion of DNMT3A-mutant clones is affected by impaired TP53 signaling, which confers resilience against stressors. Therapeutic strategies targeting inflammatory pathways or the p53-p21-DREAM axis may reduce DNMT3A-CH expansion and/or progression and its associated risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e56fa3aad088d2f09d1437903fb4b00ec728238c" target='_blank'>
              Delayed Trp53 activation protects Dnmt3a-mutant hematopoietic stem cells from inflammatory attrition
              </a>
            </td>
          <td>
            Ludovica Marando, G. Giotopoulos, Pedro Madrigal, Dhoyazan M A Azazi, Ryan Asby, M. E. Calleja-Cervantes, M. Gu, David Lara-Astiaso, Nicola K Wilson, R. Hannah, Jaana Bagri, Emily F. Calderbank, E. Laurenti, E. Bernard, Sarah J Horton, Berthold Göttgens, E. Papaemmanuil, Shuchi Agrawal-Singh, M. Gozdecka, G. Vassiliou, B. Huntly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d28c424d36dd039343d97e3e9dacf850657da3e" target='_blank'>
              Crosstalk between non-coding RNAs and programmed cell death in colorectal cancer: implications for targeted therapy
              </a>
            </td>
          <td>
            S. R. Taha, Mehdi Karimi, Bahar Mahdavi, Milad Yousefi Tehrani, Ali Bemani, Shahriar Kabirian, Javad Mohammadi, Sina Jabbari, Meysam Hushmand, Alireza Mokhtar, M. H. Pourhanifeh
          </td>
          <td>2025-01-15</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ca816cc15851962002a66601f774cc9384fd1d" target='_blank'>
              Comprehensive genome-scale CRISPR knockout screening of CHO cells
              </a>
            </td>
          <td>
            Sungwook Shin, Su Hyun Kim, Aghiles Gasselin, Gyun Min Lee, Jae Seong Lee
          </td>
          <td>2025-01-15</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions.
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b690f44fb31641c4b057abb18dc7da4f6e88c306" target='_blank'>
              The widely used Ucp1-Cre transgene elicits complex developmental and metabolic phenotypes
              </a>
            </td>
          <td>
            M. Halurkar, Oto Inoue, Archana Singh, Rajib Mukherjee, Meghana Ginugu, Christopher Ahn, C. L. Bonatto Paese, Molly Duszynski, Samantha A Brugmann, Hee-Woong Lim, Joan Sanchez-Gurmaches
          </td>
          <td>2025-01-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Precision medicine approaches to cancer treatment aim to exploit genomic alterations that are specific to individual patients to tailor therapeutic strategies. Yet, some targetable genes and pathways are essential for tumor cell viability even in the absence of direct genomic alterations. In underrepresented populations, the mutational landscape and determinants of response to existing therapies are poorly characterized because of limited inclusion in clinical trials and studies. One way to reveal tumor essential genes is with genetic screens. Most screens are conducted on cell lines that bear little resemblance to patient tumors, after years of culture under nonphysiologic conditions. To address this problem, we aimed to develop a CRISPR screening pipeline in three-dimensionally grown patient-derived tumor organoid (PDTO) models. A breast cancer PDTO biobank that focused on underrepresented populations, including West African patients, was established and used to conduct a negative-selection kinome-focused CRISPR screen to identify kinases essential for organoid growth and potential targets for combination therapy with EGFR or MEK inhibitors. The screen identified several previously unidentified kinase targets, and the combination of FGFR1 and EGFR inhibitors synergized to block organoid proliferation. Together, these data demonstrate the feasibility of CRISPR-based genetic screens in patient-derived tumor models, including PDTOs from underrepresented patients with cancer, and identify targets for cancer therapy. Significance: Generation of a breast cancer patient-derived tumor organoid biobank focused on underrepresented populations enabled kinome-focused CRISPR screening that identified essential kinases and potential targets for combination therapy with EGFR or MEK inhibitors. See related commentary by Trembath and Spanheimer, p. 407.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde16e03b04b6ab8e7612d1796bad37a03db0723" target='_blank'>
              Kinome-Focused CRISPR-Cas9 Screens in African Ancestry Patient-Derived Breast Cancer Organoids Identify Essential Kinases and Synergy of EGFR and FGFR1 Inhibition.
              </a>
            </td>
          <td>
            Florencia P. Madorsky Rowdo, Rachel Martini, Sarah Ackermann, Colin P Tang, Marvel Tranquille, Adriana Irizarry, Ilkay Us, Omar Alawa, Jenna E Moyer, M. Sigouros, John Nguyen, Majd Al Assaad, Esther Cheng, Paula S Ginter, J. Manohar, Brian Stonaker, Richard Boateng, J. Oppong, E. Adjei, B. Awuah, I. Kyei, F. Aitpillah, M. Adinku, K. Ankomah, E. Osei-Bonsu, K. Gyan, Syed A Hoda, Lisa A Newman, Juan Miguel Mosquera, A. Sboner, O. Elemento, Lukas E. Dow, Melissa B. Davis, M. L. Martin
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="DNA damage is a central driver of the aging process. We previously found that KIF2C, known to play a role in DNA repair, is repressed in aged cells. Here, we investigated if increased KIF2C activity counteracts DNA damage and its effects on aging phenotypes. We show that a small-molecule agonist of KIF2C enhances DNA repair in two distinct genetic disorders exhibiting DNA damage and accelerated aging, the Hutchinson-Gilford progeria (HGPS) and Down (DS) syndromes. Mechanistically, the KIF2C agonist improves the repair of DNA double-strand breaks by inducing nuclear envelope invaginations poked by cytoplasmic microtubules, which translated into amended epigenetic and transcriptional signatures of HGPS and DS. Moreover, subcutaneous administration of the KIF2C agonist in progeria mice mitigated aging phenotypes, extending their healthspan. Our study discloses a unique geroprotective pharmacological approach targeting DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bae82e2a8f4a414d538a8210faf20fea3eff9d" target='_blank'>
              Molecular and pharmacological heterogeneity of ETV6::RUNX1 acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenhua Li, Huanbin Zhao, Wenjian Yang, Maud Maillard, S. Yoshimura, Yu-Chih Hsiao, Xingyu Huang, Yoshihiro Gocho, L. Rowland, Anthony Brown, Landon Choi, K R Crews, C. Mullighan, Samuel Brady, Cheng Cheng, Ti-Cheng Chang, Gang Wu, M. Loh, A. E. J. Yeoh, Federico Antillon-Klussmann, S. Jeha, H. Inaba, Jiyang Yu, Ching-Hon Pui, Seth E. Karol, William E Evans, Jun J Yang
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="Cancer is the second leading cause of death among humans worldwide. Despite remarkable improvements in cancer therapies, drug resistance remains a significant challenge. The tumor microenvironment (TME) is intimately associated with therapeutic resistance. Tumor‑associated neutrophils (TANs) are a crucial component of the TME, which, along with other immune cells, play a role in tumorigenesis, development and metastasis. In the current review, the roles of TANs in the TME, as well as the mechanisms of neutrophil‑mediated resistance to cancer therapy, including immunotherapy, chemotherapy, radiotherapy and targeted therapy, were summarized. Furthermore, strategies for neutrophil therapy were discussed and TANs were explored as potential targets for cancer treatment. In conclusion, the need to explore the precise roles, recruitment pathways and mechanisms of action of TANs was highlighted for the purpose of developing therapies that precisely target TANs and reverse drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f741b0d8e7223ab3e910617355fcfe705132fe" target='_blank'>
              Tumor‑associated neutrophils: Critical regulators in cancer progression and therapeutic resistance (Review).
              </a>
            </td>
          <td>
            Rui Hou, Xi Wu, Cenzhu Wang, Hanfang Fan, Yuhan Zhang, Hanchi Wu, Huiyu Wang, Junli Ding, Huning Jiang, Junying Xu
          </td>
          <td>2025-02-25</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6201c33f4b59a0efc18c440991fab3c666877247" target='_blank'>
              Integration of CRISPR/dCas9-Based methylation editing with guide positioning sequencing identifies dynamic changes of mrDEGs in breast cancer progression
              </a>
            </td>
          <td>
            Baolong Zhang, Jin Li, Wenqiang Yu
          </td>
          <td>2025-01-21</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Evidence has been mounting over the last decade demonstrating the crucial functions of ncRNAs, including miRNAs and long non-coding RNAs (lncRNAs), in cellular signalling networks and gene regulation. These ncRNAs are highly expressed and play a crucial role in regulating processes including metastasis and endothelial cell invasiveness. They facilitate crosstalk between interconnected signaling pathways, including PTEN/PI3K/AKT/mTOR, by acting as linking molecules within signal circuits. In addition to gene expression and conventional apical measures, studying miRNAs provides a better understanding of both beneficial and detrimental cellular reactions to external exposures. Additionally, miRNAs—which may be found in extracellular and intracellular environments, such as biofluids—have become potential early and precise molecular indicators of tissue and cellular conditions. This review highlights the regulatory functions of ncRNAs, their involvement in cellular homeostasis, and their potential as biomarkers, shedding light on their significance in gene regulation, toxicological studies, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b11830d2e556b399c1b5c04bd83303250065b32" target='_blank'>
              The Role of Non-Coding RNAs in Gene Regulation and Cellular Signaling Pathways
              </a>
            </td>
          <td>
            K. A. kumari
          </td>
          <td>2025-02-22</td>
          <td>International Journal of Innovations in Science Engineering And Management</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Monosomy 7 and deletion 7q are common chromosomal abnormalities in myeloid malignancies, and they are associated with a poor prognosis. The mechanism underlying their acquisition remains elusive. We identified a cohort of 24 patients exhibiting clones with different chromosome 7 abnormalities, such as deletion 7q, unstable derivatives (ring chromosomes or ‘naked’ centromeres), and monosomy 7. We designated this group as having cytogenetic clonal evolution of chromosome 7 abnormalities (CCE7). In some cases, CCE7 correlated with disease progression, suggesting that deletions or other derivatives involving the q-arm of chromosome 7 may arise early in the disease course. These abnormalities may be transient but can potentially evolve into monosomy 7. Within the CCE7 group, telomere loss or shortening may contribute to chromosomal instability and the emergence of unstable derivatives, as the chromosome 7 derivatives displayed loss or rearrangement of subtelomeric regions. Moreover, we identified variants in genes implicated in telomere biology disorders and observed specific genetic mutation profiles associated with different chromosome 7 abnormalities. These findings shed light on a potential mechanism leading to monosomy 7 through the evolution of chromosome 7q abnormalities. Identifying patients at risk of developing monosomy 7, based on the presence of unstable derivatives with telomere loss or a specific mutation profile, could potentially enhance patient management and guide the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd97cfdef751d5eb8d87f19388c6ddb34eef62a2" target='_blank'>
              Emergence and Cytogenetic Clonal Evolution of Chromosome 7 Abnormalities in Myeloid Malignancies: Investigating the Role of Telomere Dysfunction
              </a>
            </td>
          <td>
            Carmen Baldazzi, L. Bandini, V. Robustelli, A. Patuelli, C. Venturi, Alessandra Grassi, G. Marzocchi, Angela Ielpo, V. Solli, M. Bochicchio, Stefania Paolini, Chiara Sartor, F. Zingarelli, Antonio Curti, E. Ottaviani, N. Testoni
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Next-generation sequencing has revealed the disruptive reality that advanced/metastatic cancers have complex and individually distinct genomic landscapes, necessitating a rethinking of treatment strategies and clinical trial designs. Indeed, the molecular reclassification of cancer suggests that it is the molecular underpinnings of the disease, rather than the tissue of origin, that mostly drives outcomes. Consequently, oncology clinical trials have evolved from standard phase 1, 2, and 3 tissue-specific studies; to tissue-specific, biomarker-driven trials; to tissue-agnostic trials untethered from histology (all drug-centered designs); and, ultimately, to patient-centered, N-of-1 precision medicine studies in which each patient receives a personalized, biomarker-matched therapy/combination of drugs. Innovative technologies beyond genomics, including those that address transcriptomics, immunomics, proteomics, functional impact, epigenetic changes, and metabolomics, are enabling further refinement and customization of therapy. Decentralized studies have the potential to improve access to trials and precision medicine approaches for underserved minorities. Evaluation of real-world data, assessment of patient-reported outcomes, use of registry protocols, interrogation of exceptional responders, and exploitation of synthetic arms have all contributed to personalized therapeutic approaches. With greater than 1 × 1012 potential patterns of genomic alterations and greater than 4.5 million possible three-drug combinations, the deployment of artificial intelligence/machine learning may be necessary for the optimization of individual therapy and, in the near future, also may permit the discovery of new treatments in real time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed31a3c6403c1b0469b4430cade90e86c2a9bb22" target='_blank'>
              Novel clinical trial designs emerging from the molecular reclassification of cancer.
              </a>
            </td>
          <td>
            M. Nikanjam, S. Kato, Teresa Allen, J. Sicklick, R. Kurzrock
          </td>
          <td>2025-01-22</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>2</td>
          <td>48</td>
        </tr>

        <tr id="The PPM1D gene and its protein product (serine-threonine protein phosphatase, PPM1D or Wip1) are involved in regulation of cell's DNA damage response, cell cycle control, and repair. Amplification, overexpression, or mutations of the PPM1D gene have a significant impact on cell responses to stress factors and genetic instability as well as impairments of processes of double-strand break repair, nucleotide excision repair, base excision repair, cell cycle, and apoptosis. PPM1D dephosphorylates and thus inactivates p53, proteins that respond to DNA strand integrity damage, cell cycle checkpoint proteins, and apoptotic proteins. This contributes to tumor development, growth, and maintenance of the tumor phenotype. In this review we consider data on the role of the PPM1D gene in the formation and maintenance of various oncological processes, including tumors of the mammary glands, ovaries, prostate gland, esophagus, stomach, intestines, liver and pancreas, hemoblastoses, and others.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. S. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Significance Estrogen receptor (ER)-positive breast cancer often develops resistance to endocrine therapies like tamoxifen. Understanding the molecular mechanisms underlying both carcinogenesis and drug resistance is critical for improving therapeutic strategies. This study identifies circFOXK2 as a key regulator of CCND1 in ER-positive breast cancer. By stabilizing CCND1 mRNA through RNA–RNA pairing and recruitment of ELAVL1/HuR, circFOXK2 enhances the CCND1–CDK4/6–p-RB–E2F axis, promoting cell proliferation and therapy resistance. Genetic silencing and pharmacological inhibition of circFOXK2 suppress tumor growth and sensitize or restore tamoxifen responsiveness in ER-positive breast cancer. These findings highlight circFOXK2 as a potential therapeutic target for overcoming resistance and improving outcomes in ER-positive breast cancer patients, offering avenues for clinical intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a572f11006892aa8b4e142fead4939f8a023665b" target='_blank'>
              A circRNA–mRNA pairing mechanism regulates tumor growth and endocrine therapy resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Jia Yi, Jiao Du, Xue Chen, Rui-chao Nie, Guo-Sheng Hu, Lei Wang, Yue-ying Zhang, Shang Chen, Xiao-sha Wen, Di-xian Luo, Hua He, Wen Liu
          </td>
          <td>2025-02-18</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de6693e1cc9fcaaa6e44ac852bdcdb47fc66775d" target='_blank'>
              Double-strand break repair pathways differentially affect processing and transduction by dual AAV vectors
              </a>
            </td>
          <td>
            Anna C Maurer, Brian Benyamini, O. Whitney, Vinson B. Fan, Claudia Cattoglio, Djem U. Kissiov, G. Dailey, X. Darzacq, Matthew D Weitzman, Robert Tjian
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 genome-editing technology has revolutionized human stem cell (HSC) research, offering novel therapeutic and diagnostic applications. HSCs play a crucial role in regenerative medicine and genetic therapies due to their ability to self-renew and differentiate into various blood cell lineages. The precise genome-editing capability of CRISPR/Cas9 allows for targeted gene modifications, enabling the correction of inherited disorders, disease modeling and the discovery of novel biomarkers. Despite significant advancements, challenges such as off-target effects, delivery efficiency, and ethical concerns persist, requiring further research and optimization.
Objective: This systematic review evaluates the therapeutic and diagnostic potential of CRISPR/Cas9 genome editing in HSCs, focusing on its efficacy in gene correction for hematologic disorders, disease modeling and biomarker discovery.
Methods: A systematic review was conducted following PRISMA guidelines, analyzing studies published between 2015 and 2024. Literature searches were performed in PubMed, Web of Science, and Scopus using MeSH-aligned keywords. The inclusion criteria encompassed peer-reviewed studies utilizing CRISPR/Cas9 for gene modification in HSCs for therapeutic and diagnostic applications. Exclusion criteria included studies that lacked experimental validation, did not focus on HSCs, or were non-English publications. Out of 85 initially retrieved studies, 40 met the inclusion criteria, and 15 were selected for final synthesis.
Results: CRISPR/Cas9 gene-editing strategies in HSCs were categorized as gene knockout (53%), gene activation (40%), and dual knockout/activation (7%). Hematological disorders, including sickle cell anemia and beta-thalassemia, accounted for 35% of studies, demonstrating up to 90% correction in β-globin mutations. Neurodegenerative diseases constituted 20% of studies, where knockout of amyloid precursor protein (APP) in Alzheimer’s models resulted in a 60% reduction in plaque accumulation. Muscular dystrophy studies (10%) showed 75% improvement in dystrophin expression through gene activation. High-throughput CRISPR screening was employed in 15% of studies for biomarker identification. Despite promising outcomes, off-target mutations were observed in 28% of studies, and viral vector-based delivery methods were used in 65%, raising safety concerns.
Conclusion: CRISPR/Cas9 genome editing in HSCs presents a ground-breaking approach for treating genetic disorders and enhancing precision medicine. Its potential to correct disease-causing mutations, model complex disorders, and identify novel therapeutic targets is substantial. However, challenges in delivery methods, long-term safety, and ethical considerations remain barriers to clinical translation. Future research should focus on optimizing high-fidelity Cas9 variants, improving non-viral delivery methods, and addressing ethical concerns to ensure the safe and effective application of CRISPR/Cas9 in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20400a815c119b74c31ff8ba0f49e856aea421b" target='_blank'>
              EVALUATION OF CRISPR/CAS9 GENOME-EDITING SYSTEM IN HUMAN STEM CELLS HSCS: THERAPEUTICS AND DIAGNOSTICS PROSPECTS
              </a>
            </td>
          <td>
            Ehsan ul Haq, Haseeb Khaliq, Ayesha Muddasser, Misha Aslam, Farwa Zafar, Naveera Mazhar, Amna Naheed Khan
          </td>
          <td>2025-01-20</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 194


 Background:
 The heterogeneous efficacy of PARPi in patients with mCRPC and homologous recombination deficiency (HRd)-associated genetic alterations has emphasized the need for a more accurate selection of responding patients. sWGS offers a cost-effective method to identify HRd through GI analysis. However, the correlation between GI from sWGS and specific genotypes, along with its predictive value for PARPi response, remains unreported.
 Methods:
 A total of 289 samples from 267 patients were analyzed for GI by sWGS. Of these, 121 were newly sequenced with a mean depth of 2.05x and 168 were
 in silico
 down-sampled to 2x. Eighty-four percent of the samples were collected in the CRPC setting. Copy number analysis was performed using iChorCNA with a 50 Kb window size, and multiple genome-wide copy number plots were generated per sample with varying tumor content and ploidy. Two researchers independently selected the optimal plot, with discordant cases reviewed by a third, to quantify large-scale transitions and telomeric allelic imbalances per sample, which together constituted an adjusted GI score (aGIS).
 Results:
 The median aGIS of all samples was 21 (interquartile range [IQR] 16 – 30). Samples with HRd-associated genetic alterations (43%) had a higher median aGIS (26) than to those without (19, P < 0.0001). Deleterious aberrations in
 PALB2
 resulted in the highest aGIS, followed by
 BRCA2
 and
 ATM
 . The aGIS for genotypes with >3 samples are detailed (Table). Out of the 267 patients, 71 received PARPi. Their median aGIS was 27 (IQR: 19–37). The 33 patients with ≥50% PSA decline had a higher aGIS than the 37 without (aGIS 35 vs 22, P = 0.0001). In 60 patients with radiologic assessment, 22 had a partial response, 27 stable disease, and 11 progressive disease. The median aGIS for these groups were 32 (IQR: 25.5–49), 26 (IQR: 17.5–35), and 21 (IQR: 16.5–26.5), respectively, with a significant difference between partial and progressive disease (P = 0.0053). The Cox proportional hazards model did not show a significant effect of aGIS on progression-free survival (P = 0.144). Still, patients with the highest 25% aGIS had a median progression-free survival of 11 months, compared to 4 months for the lowest 25%, and 8 months for the remaining.
 Conclusions:
 The quantity of large-scale transitions and telomeric allelic imbalances, from sWGS, correlates with HRd-associated genetic alterations. In PARPi-treated patients, this aGIS is significantly associated with biochemical and radiologic response, even after selection by genotyping. These findings support the development of an sWGS-adjusted GI score for predicting PARPi response in mCRPC.




 Aberration in:
 N
 Median aGIS (IQR)





 PALB2

 4
 30.5 (15 – 44.75)



 BRCA2

 50
 29 (22 – 38.75)



 ATM

 37
 28 (20 – 34)



 BARD1

 4
 26.5 (15.5 – 34.5)



 RAD51B

 5
 26 (25 – 26)



 CDK12

 20
 18.5 (10.75 – 20.25)



 CHEK2

 5
 16 (12 – 17)




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95f3acfe80f93b65e6acf665da89a1f8178290f" target='_blank'>
              Assessing genomic instability (GI) by shallow whole-genome sequencing (sWGS) to predict PARP inhibitor (PARPi) response in metastatic castration-resistant prostate cancer (mCRPC).
              </a>
            </td>
          <td>
            P. Slootbeek, Yarah Quint, Julian J.R. Kokke, Samhita Pamidimarri Naga, I. Kloots, M. V. Luna-Velez, M. Ligtenberg, R. D. de Voer, N. Mehra
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc5699f8d775d7c0b21dfb5b4e13161b3a6de2f" target='_blank'>
              Appraisal of CRISPR Technology as an Innovative Screening to Therapeutic Toolkit for Genetic Disorders.
              </a>
            </td>
          <td>
            A. Shahid, Ambreen Zahra, S. Aslam, Amen Shamim, Waqas Rafique Ali, Bilal Aslam, Sultan Habibullah Khan, Muhammad Imran Arshad
          </td>
          <td>2025-02-02</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prognostic transcriptomic signatures for prostate cancer (PCa) often overlook the cellular origin of expression changes, an important consideration given the heterogeneity of the disorder. Current clinicopathological factors inadequately predict biochemical recurrence, a critical indicator guiding post-treatment strategies following radical prostatectomy. To address this, we conducted a meta-analysis of four large-scale PCa datasets and found 33 previously reported PCa-associated genes to be consistently up-regulated in prostate tumours. By analysing single-cell RNA-sequencing data, we found these genes predominantly as markers in epithelial cells. Subsequently, we applied 97 advanced machine-learning algorithms across five PCa cohorts and developed an 11-gene epithelial expression signature. This signature robustly predicted biochemical recurrence-free survival (BCRFS) and stratified patients into distinct risk categories, with high-risk patients showing worse survival and altered immune cell populations. The signature outperformed traditional clinical parameters in larger cohorts and was overall superior to published PCa signatures for BCRFS. By analysing peripheral blood data, four of our signature genes showed potential as biomarkers for radiation response in patients with localised cancer and effectively stratified castration-resistant patients for overall survival. In conclusion, this study developed a novel epithelial gene-expression signature that enhanced BCRFS prediction and enabled effective risk stratification compared to existing clinical- and gene-expression-derived prognostic tools. Furthermore, a set of genes from the signature demonstrated potential utility in peripheral blood, a tissue amenable to minimally invasive sampling in a primary care setting, offering significant prognostic value for PCa patients without requiring a tumour biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0bf6f0e9c87d76cfdec8216eb4d7e07bdd3f760" target='_blank'>
              Integration of single-cell and bulk RNA-sequencing data reveals the prognostic potential of epithelial gene markers for prostate cancer.
              </a>
            </td>
          <td>
            Z. Mou, Lorna W Harries
          </td>
          <td>2025-02-19</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f63f6a23a3983ad60b3216e3e5876ec300a9938" target='_blank'>
              Deciphering the generation of heterogeneity in esophageal squamous cell carcinoma metastasis via single-cell multiomics analysis
              </a>
            </td>
          <td>
            Kaiwen Sheng, Jun Chen, Ruitang Xu, Haoqiang Sun, Ran Liu, Yongjie Wang, Wenwen Xu, Jiao Guo, Miao Zhang, Shuai Liu, Juan Lei, Yawen Sun, Yang Jia, Dianhao Guo
          </td>
          <td>2025-02-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is the most prevalent type of malignant epithelial neoplasm that affects the oral cavity. It has been a significant health concern in many countries for a long time since it was usually treated with surgery, radiation, and/or chemotherapy. Drug resistance is the primary issue in patient populations and scientific research, which promotes OSCC tumour cell invasion and migration. Thus, identifying highly specific therapeutic targets could be the potential approach for more successful treatment of OSCC. It is still challenging to understand the genetic causes of oral carcinogenesis due to its highly varied clinic-pathological parameters. It is important to remember that signaling channels and complexes that affect chromatin accessibility control gene expression, which in turn affects cell development and differentiation. Histones undergo post-translational alteration to give this platform. Understanding the processes of gene regulation through histone methylation and its modifications could enhance the early detection, prognostic prediction, and therapy of OSCC. To be properly used as a therapeutic target, histone methylation in OSCC requires more investigation. This review details the dysregulated histone methylation and the modifying enzymes linked to the development and aetiology of OSCC. Furthermore, the part that lysine methylation plays in cell migration, chemo-resistance, and OSCC invasion is also investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ac8fe27c84ad3ea1843eef61f7ad80802b8dbb" target='_blank'>
              Molecular insight into histone methylation as a novel target for oral squamous cell carcinoma: future hope in personalised medicine
              </a>
            </td>
          <td>
            P. Rajendran, Monisha Prasad, Enas M. Ali, Ramya Sekar, Abdullah M AlZahrani, M. Karobari, M. A. M. Genena, B. Abdallah
          </td>
          <td>2025-01-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA methylation and chromatin accessibility are fundamental epigenetic mechanisms that orchestrate gene expression programs, define cellular states, and drive developmental trajectories. scCOOL-seq has enabled simultaneously measuring the two modalities in the same single cells, but in quite a low throughput manner. We present single-cell split-pool ligation-based multi-omics sequencing technology (SpliCOOL-seq), which improves the throughput to thousands of cells by combining split-pool ligation based single-cell indexing after in situ tagmentation with universal Tn5 transposase and scCOOL-seq. SpliCOOL-seq achieved higher sensitivity than previous high throughput single-cell DNA methylation sequencing methods and can clearly distinguish different lung cancer cells based on both genetic and multiple epigenetic modalities. We show that the two DNMT inhibitors, 5-Azacitidine and Decitabine, both cause large scale demethylation but in distinct patterns. Applied to the primary lung tumor, SpliCOOL-seq clearly captured subclones within the tumor lesion and revealed candidate genes related to tumorigenesis. Furthermore, we presented the first report on the heterogeneity of scDNAm age acceleration among tumor subclones as predicted from a single-cell perspective. In conclusion, SpliCOOL-seq achieves parallel profiling of whole genome DNA methylation and chromatin accessibility in the same individual cells in a high-throughput manner and is hopefully used to illustrate regulatory interactions under different cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9113b4cede2cf7486243e4f5408b0e991ee33d2d" target='_blank'>
              High-throughput single-cell DNA methylation and chromatin accessibility co-profiling with SpliCOOL-seq
              </a>
            </td>
          <td>
            Qingmei Shen, Enze Deng, Jingna Zhang, Qifeng Yang, Dan Su, Xiaoying Fan
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9342acbba0e99dc63b9eedf38a334853e0c4e4e7" target='_blank'>
              The role of mitophagy-related genes in prognosis and immunotherapy of cutaneous melanoma: a comprehensive analysis based on single-cell RNA sequencing and machine learning.
              </a>
            </td>
          <td>
            Jun Tian, Lei Zhang, Kexin Shi, Li Yang
          </td>
          <td>2025-01-11</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963a56be4a953e4f275f13aa1a14a7847b02c333" target='_blank'>
              Finding a needle in a haystack: functional screening for novel targets in cancer immunology and immunotherapies
              </a>
            </td>
          <td>
            Yi Du, Yang Yang, Bohao Zheng, Qian Zhang, Shengtao Zhou, Linjie Zhao
          </td>
          <td>2025-01-25</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND/AIM
As an increasing number of drugs are approved for targeted cancer therapy, comprehensive genomic profiling of cancer patients is frequently conducted to identify potentially relevant genetic variants. Different sequencing technologies are used for this purpose; targeted gene panels cover a selected set of biomarker genes and hotspot regions, while whole genome sequencing (WGS) delivers genome-wide data. This comparison study aimed at evaluating whether one method performs superiorly to the other regarding the detection of targetable variants.


PATIENTS AND METHODS
To evaluate the performance of the two sequencing technologies, we compared the results of targeted sequencing using the Ion Torrent Oncomine Comprehensive Assay Plus (OCA-Plus) panel to Illumina WGS reports in 11 patients diagnosed with pancreatic cancer (PC). All pathogenic and likely pathogenic variants, including those relevant for targeted therapy, reported by WGS and OCA-Plus, were included in the final comparison.


RESULTS
Both techniques identified common driver mutations implicated in PC with high concordance (81%) across all variants. For variants relevant to targeted therapy, a 100% concordance between the technologies was observed.


CONCLUSION
A comparable number of variants were reported by WGS and OCA-Plus, and all genetic variants relevant for targeted therapy were identified by both technologies. Thus, WGS does not provide substantial additional information in this patient group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b34f2239a29bc4dc0f37c50d80fd5e063284e99d" target='_blank'>
              Paired Comparison of Whole Genome Sequencing and Comprehensive Targeted Sequencing of Pancreatic Cancer Tissue.
              </a>
            </td>
          <td>
            Thea Amalie Hvidtfeldt, Tim S Poulsen, I. Chen, Louise Laurberg Klarskov, Estrid Høgdall
          </td>
          <td>2025-01-31</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3095fb742b0c6045977fafeab8ed1d06964edc6e" target='_blank'>
              FXYD5 regulates gastric cancer cell metastasis and drug resistance by EMT modulation.
              </a>
            </td>
          <td>
            Yuning Mao, Yaohua Hu, Han Meng, Jing Qin, Qingling An, Caiqin Zhang, Chenbo Guo, Yong Zhao, Dengxu Tan, Xu Ge, Changhong Shi
          </td>
          <td>2025-02-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4233622be9dcbd1fb28cce4bd3a5ada680860" target='_blank'>
              Towards understanding cancer dormancy over strategic hitching up mechanisms to technologies
              </a>
            </td>
          <td>
            Sumin Yang, Jieun Seo, Jeonghyeon Choi, Sung-Hyun Kim, Yunmin Kuk, Kyung Chan Park, Mingon Kang, Sangwon Byun, Jae-Yeol Joo
          </td>
          <td>2025-02-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d0967e94330094078ab60dc71040acbf79f1f7c" target='_blank'>
              Targeting LMO2-induced autocrine FLT3 signaling to overcome chemoresistance in early T-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            C. Tremblay, J. Saw, Feng Yan, Jacqueline A Boyle, Ovini Amarasinghe, Shokoufeh Abdollahi, Anh Vo, Benjamin J Shields, Chelsea Mayoh, H. McCalmont, K. Evans, Anna Steiner, Kevin Parsons, M. McCormack, David R Powell, Nick Wong, Stephen M. Jane, Richard B Lock, David J Curtis
          </td>
          <td>2025-01-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Sequencing of human patient tumors has identified recurrent missense mutations in genes encoding core histones. We report that mutations that convert histone H3 amino acid 50 from a glutamate to a lysine (H3E50K) support an oncogenic phenotype. Expression of H3E50K is sufficient to transform human cells as evidenced by an increase in cell migration and invasion, and an increase in proliferation and clonogenicity. H3E50K also increases the invasive phenotype in the context of co-occurring BRAF mutations, which are present in patient tumors characterized by H3E50K. H3E50 lies on the globular domain surface in a region that contacts H4 within the nucleosome. We find that H3E50K selectively increases chromatin accessibility and perturbs proximal H3 post-translational modifications including H3K27me3; together these changes to chromatin dynamics dysregulate gene expression to support the epithelial-to-mesenchymal transition. Functional studies using Saccharomyces cerevisiae reveal that, while yeast cells that express H3E50K as the sole copy of histone H3 show sensitivity to cellular stressors, including caffeine, H3E50K cells display some genetic interactions that are distinct from the characterized H3K36M oncohistone yeast model. Taken together, these data suggest that additional H3 mutations have the potential to support oncogenic activity and function through distinct mechanisms that dysregulate gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab101c8206ab8a7a20f301f0b33f3fd7807cac2" target='_blank'>
              Histone H3E50K remodels chromatin to confer oncogenic activity and support an EMT phenotype
              </a>
            </td>
          <td>
            K. Sad, Dorelle V. Fawwal, Celina Y Jones, Emily J Hill, Katie T. Skinner, Miranda Adams, Severin Lustenberger, Richard S Lee, Sandhya V Lohano, Satvik R Elayavalli, Jonathan Farhi, Christina C Mehta, Laramie D. Lemon, Milo B Fasken, Andrew L Hong, Steven A. Sloan, Anita H Corbett, Jennifer Spangle
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Kirsten Rat Sarcoma Virus mutation and its complications have been a major concern. Limitations to drawing its accurate configuration, its complications; mutates and thermodynamic properties, have limited their effectiveness as a molecular diagnostic tool. To counteract these challenges, over the years a different set of sophisticated Immunotherapeutic approaches have been adopted such as Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Caspase Protein System (CAS), and Chimeric Antigen Receptor-T cell (CAR-T). Some of them have already managed approval of the Food and Drug Administration. One of the primary goals of CRISPR/CAS technology is to correct mutations within the genetic sequence, thereby potentially repairing defective genes like KRAS. Meanwhile, CAR-T immunotherapies capitalise on genetically modified T cells. In comparison to traditional therapies, immunotherapies can offer greater safety and precision. However, their efficacy is still questionable while treating solid tumours. Here authors have covered the various key targets such as p53, p21, WNT, and RAS, their success stories over the years and have proposed the importance of targets like KRAS oncogene to manage a better output. KRAS is a RAS family oncoprotein responsible for the development of 85% of different cancers. The missense mutations at the various KRAS hotspots can lead to the development of lung and Pancreatic Cancers. Some of the most recently conducted revolutionary studies synthesize CAR-NK, CAR-M, and CRISPR-Cas9/3/10/Novel CAS systems, as well as their prospective molecular combination to target the KRAS. The purpose of this study is to provide a present status of these therapies, limitations, clinical trials, and a proposal to sketch a combinatorial impact via the PRISMA model for 62 different manuscripts (2001-2024). This format aligns and organizes everything in Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, providing a clear and systematic infographic that represents the process, including stages of selection, screening, and documentation.
 Citation Format: SDanish Kadir. Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer is a common and fatal disease that affects many people globally. CRC is classified as the third most prevalent cancer among males and the second most frequent cancer among females worldwide. The purpose of this article is to examine how personalized medicine might be used to treat colorectal cancer. The classification of colorectal cancer based on molecular profiling, including the detection of significant gene mutations, genomic instability, and gene dysregulation, is the main topic of this discussion. Advanced technologies and biomarkers are among the detection methods that are explored, demonstrating their potential for early diagnosis and precise prognosis. In addition, the essay explores the world of treatment possibilities by providing light on FDA-approved personalized medicine solutions that provide individualized and precise interventions based on patient characteristics. This article assesses targeted treatments like cetuximab and nivolumab, looks at the therapeutic usefulness of biomarkers like microsatellite instability (MSI) and circulating tumor DNA (ctDNA), and investigates new approaches to combat resistance. Through this, our review provides a thorough overview of personalized medicine in the context of colorectal cancer, ultimately highlighting its potential to revolutionize the field and improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b80ad33c609a3090e37d7111c49bfa155aa4a9e" target='_blank'>
              Personalized medicine in colorectal cancer: a comprehensive study of precision diagnosis and treatment.
              </a>
            </td>
          <td>
            Fatemeh Gila, Somayeh Khoddam, Zahra Jamali, Mohmmad Ghasemian, Shayan Shakeri, Zeinab Dehghan, Jafar Fallahi
          </td>
          <td>2025-02-09</td>
          <td>Personalized medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The recombinant human p53 adenovirus (Ad-p53) offers a promising approach for cancer therapy, yet its chromatin structure and effects on host chromatin organization and gene expression are not fully understood.In this study, we employed in situ ChIA-PET to investigate the colorectal cancer cell line HCT116 with p53 knockout, comparing them to cells infected with the adenovirus-vector expressing p53. We examined alterations in chromatin interactions and gene expression following treatment with the anti-cancer drug 5-fluorouracil (5-FU).Our results indicate that Ad-p53 forms a specific chromatin architecture within the vector and mainly interacts with repressive or inactive regions of host chromatin, without significantly affecting the expression of associated genes. Additionally, Ad-p53 does not affect topologically associating domains (TADs) or A/B compartments in the host genome.These findings suggest that while Ad-p53 boosts p53 expression, enhancing drug sensitivity without substantially altering host HCT116 chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2495bcc5f798b26cfeb646dc7337f34c34784" target='_blank'>
              Chromatin structure and gene transcription of recombinant p53 adenovirus vector within host
              </a>
            </td>
          <td>
            Duo Ning, Yuqing Deng, Simon Zhongyuan Tian
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Ovarian cancer is a deadly gynecologic disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for homologous recombination–deficient high-grade serous ovarian cancers (HGSOC). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both heat shock factor 1 (HSF1) and MYC, wherein these two genes are co-amplified in more than 30% of patients with HGSOC. We further found that HSF1 and MYC transcriptional activities were highly correlated with human HGSOC tumors and cell lines, suggesting that they may cooperate in the disease. CUT&RUN sequencing for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, the binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein–protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, the growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1–MYC co-amplified HGSOC cells compared with those with wild-type HSF1 and MYC copy numbers. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 using a precision medicine approach based on HSF1–MYC co-amplification as a biomarker in HGSOC would improve therapy response and patient outcomes. Significance: We show that HSF1 and MYC genes are co-amplified in more than 30% of HGSOC and demonstrate that HSF1 and MYC functionally cooperate to drive the growth of HGSOC cells. This work provides the foundation for HSF1 and MYC co-amplification as a biomarker for treatment efficacy of the polo-like kinase 1 inhibitor volasertib in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High-grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Pseudogene‐derived lncRNAs are widely dysregulated in cancer. Technological advancements have facilitated the functional characterization of increasing pseudogenes in cancer progression. However, the association between pseudogenes and RNA N6‐methyladenosine (m6A) modification in cancer, as well as the underlying mechanisms, remains largely unexplored. Methods We analyzed the expression of 12 146 pseudogenes and comprehensively examined the m6A modification of RNAs derived from them and their paralogs. Through integrative analysis of multi‐omics data, we explored the associations between pseudogene dysregulation and m6A, identifying critical pseudogenes involved in HGSOC progression. Tumour promotion role of RPS15AP12 and its cognate parent gene was characterized by cell proliferation, transwell assays, and scratch assays in ovarian cells and xenograft nude mice. RNA decay assays were used to reveal the participation of m6A in decreasement of RPS15AP12 lncRNA stability. Luciferase reporter assays were performed to verify that RPS15AP12 enhances RPS15A expression by competitively binding to miR‐96‐3p. Western blot and phosphorylation assays were performed to investigate the impairment of RPS15AP12 towards the sensors of MAVS (RIG‐I and MDA5), and downstream p‐TBK1 and p‐IRF3. Finally, ELISA assays were performed to explore the regulatory role of RPS15AP12 in IFN‐β expression. Results M6A is distributed across over a thousand pseudogenes, and hypomethylation leads to their upregulation in HGSOC. We identified a processed pseudogene, RPS15AP12, upregulated by FTO‐mediated m6A demethylation. RPS15AP12 enhances the growth ability and metastatic capabilities of ovarian cancer (OC) cells via functioning as a competitive endogenous RNA (ceRNA) for its host gene, RPS15A, through the sequestration of miR‐96‐3p. Importantly, the deletion of RPS15AP12 diminishes the expression of RPS15A, leading to the upregulation of anti‐tumour immune responses by activating RIG‐I and MDA5 and downstream p‐TBK1 and p‐IRF3 as well as IFN‐β levels. Conclusion Our findings expand the understanding of m6A‐modulated pseudogenes in tumour growth and anti‐tumour innate immunity in OC. Key Points Genome‐wide profiling reveals the redistribution of m6A modification on pseudogene‐derived lncRNAs and m6A redistribution‐relevant dysregulation of pseudogenes in HGSOC. RPS15AP12, as a representative processed pseudogene, is up‐regulated by FTO‐mediated demethylation and acts as a miRNA sponge to promote RPS15A expression via competitively binding to miR‐96‐3p. RPS15AP12/RPS15A axis inhibits MAVS sensors (RIG‐I and MDA5) and downstream IFN‐β levels in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f32a6bc1ac1ae8503c9d4c32bcdde0ae2bb380" target='_blank'>
              Genome‐wide profiling of N6‐methyladenosine‐modified pseudogene‐derived long noncoding RNAs reveals the tumour‐promoting and innate immune‐restraining function of RPS15AP12 in ovarian cancer
              </a>
            </td>
          <td>
            Jie Xu, Yifei Ren, Jiayi Lu, Fengjiang Qin, Dan Yang, Chunyan Tang, Yu Yang, Jing Xu, Tao Liu, Ping Yi
          </td>
          <td>2025-02-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c56d0e33616064b81832853efdda25a0062176f" target='_blank'>
              Exploring Epigenetic Complexity in Regulation of Hematopoietic Stem Cells Niche: A Mechanistic Journey from Normal to Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            N. A. Yusoff, Zariyantey Abd Hamid, Izatus Shima Taib, Siti Razila Abdul Razak, S. B. Budin
          </td>
          <td>2025-01-23</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly prevalent and deadly type of cancer characterized by intricate molecular pathways that drive tumor development, metastasis, and resistance to conventional treatments. Recently, ferroptosis, a controlled mechanism of cell death instigated by iron-dependent lipid peroxidation, has gained attention for its role in LC progression and treatment. Noncoding RNAs (ncRNAs), such as microRNAs (miRNAs) and long noncoding RNAs (lncRNAs), are emerging as key modulators of ferroptosis, significantly influencing LC biology. This review explores how ncRNAs control ferroptotic pathways and affect tumor growth, metastasis, and therapy resistance in LC. By understanding the dual functions of ncRNAs in both activating and inhibiting ferroptosis, we aim to uncover new therapeutic targets and strategies for LC. These insights provide a promising direction for the development of ncRNA-based treatments designed to induce ferroptosis, potentially improving therapeutic outcomes for patients with LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2604d726c6b3c3745d518fb5d2c37fc2627c33b3" target='_blank'>
              Ferroptosis and noncoding RNAs: exploring mechanisms in lung cancer treatment
              </a>
            </td>
          <td>
            Nadi Rostami Ravari, Farzad Sadri, Mohammad Ali Mahdiabadi, Yaser Mohammadi, Zahra Ourang, Z. Rezaei
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA methylation plays a critical role in gene regulation, affecting cellular differentiation and disease progression, particularly in non-coding regions. However, predicting the epigenetic consequences of non-coding mutations at single-cell resolution remains a challenge. Existing tools have limited prediction capacity and struggle to capture dynamic, cell-type-specific regulatory changes that are crucial for understanding disease mechanisms. Here, Methven, a deep learning framework designed is presented to predict the effects of non-coding mutations on DNA methylation at single-cell resolution. Methven integrates DNA sequence with single-cell ATAC-seq data and models SNP-CpG interactions over 100 kbp genomic distances. By using a divide-and-conquer approach, Methven accurately predicts both short- and long-range regulatory interactions and leverages the pre-trained DNA language model for enhanced precision in classification and regression tasks. Methven outperforms existing methods and demonstrates robust generalizability to monocyte datasets. Importantly, it identifies CpG sites associated with rheumatoid arthritis, revealing key pathways involved in immune regulation and disease progression. Methven's ability to detect progressive epigenetic changes provides crucial insights into gene regulation in complex diseases. These findings demonstrate Methven's potential as a powerful tool for basic research and clinical applications, advancing this understanding of non-coding mutations and their role in disease, while offering new opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd41b7e43e4befc2efcfbe9286141d3e14f4002" target='_blank'>
              Epigenetic Impacts of Non-Coding Mutations Deciphered Through Pre-Trained DNA Language Model at Single-Cell Resolution.
              </a>
            </td>
          <td>
            Zhe Liu, An Gu, Yihang Bao, Guan Ning Lin
          </td>
          <td>2025-01-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22203f23dc18f7e477336e6b95f3919f3954fe27" target='_blank'>
              Mechanisms and therapeutic implications of gene expression regulation by circRNA-protein interactions in cancer
              </a>
            </td>
          <td>
            Nan Zhang, Xinjia Wang, Yu Li, Yiwei Lu, Chengcheng Sheng, Yumeng Sun, Ningye Ma, Yisheng Jiao
          </td>
          <td>2025-01-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ubiquitin-proteasome system is a fundamental regulatory mechanism that governs protein stability and intracellular signaling in eukaryotic cells. This system relies on a coordinated cascade of enzymatic activities involving activating enzymes, conjugating enzymes, and ligases to assemble distinct ubiquitin signals. These signals are subsequently edited, removed, or interpreted by deubiquitinases and ubiquitin-binding proteins. While E3 ligases have traditionally been recognized as the primary determinants of substrate specificity in the ubiquitination process, recent studies have revealed that the dysregulation of E2 enzymes can also lead to significant pathological outcomes, including chromatin instability, immune dysregulation, metabolic dysfunction, and an elevated risk of cancer. Consequently, E2 enzymes have emerged as promising therapeutic targets for the treatment of various dis-eases. This review provides a comprehensive examination of the roles and mechanisms of the ubiquitin-conjugating enzyme E2T (UBE2T) in cancer initiation, progression, and therapy resistance, highlighting its potential as a compelling target for cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676ca1002e380b93dec30d356ec90ce333f0b086" target='_blank'>
              Biological Functions and Therapeutic Potential of UBE2T in Human Cancer.
              </a>
            </td>
          <td>
            Keshen Wang, Qichen He, Xiangyan Jiang, Yong Ma, Tao Wang, Huinian Zhou, Zeyuan Yu, Zuoyi Jiao
          </td>
          <td>2025-03-06</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumors employ a range of strategies to evade detection and eradication by the host’s immune system. These include downregulating antigen expression, altering antigen presentation processes, and inhibiting immune checkpoint pathways. etc. Adoptive Cell Therapy (ACT) represents a strategy that boosts anti-tumor immunity. This is achieved by amplifying or genetically engineering immune cells, which are either sourced from the patient or a donor, in a laboratory setting. Subsequently, these cells are reintroduced into the patient to bolster their immune response against cancer. ACT has successfully restored anti-tumor immune responses by amplifying the activity of T cells from patients or donors. This review focuses on the mechanisms underlying tumor escape, including alterations in tumor cell antigens, the immunosuppressive tumor microenvironment (TME), and modulation of immune checkpoint pathways. It further explores how ACT can avddress these factors to enhance therapeutic efficacy. Additionally, the review discusses the application of gene-editing technologies (such as CRISPR) in ACT, highlighting their potential to strengthen the anti-tumor capabilities of T cells. Looking forward, the personalized design of ACT, combined with immune checkpoint inhibitors and targeted therapies, is expected to significantly improve treatment outcomes, positioning this approach as a key strategy in the field of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3fa264efca4970b6216caa0126f6890f69da527" target='_blank'>
              Adoptive cell therapy against tumor immune evasion: mechanisms, innovations, and future directions
              </a>
            </td>
          <td>
            Liqin Ruan, Lu Wang
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 The benefit of adoptive cell transfer (ACT) therapies with T cells engineered to express chimeric antigen receptors (CAR) or T cell receptors (TCR) is limited by T cell dysfunction upon repetitive antigen exposure. We integrated a pooled genome-wide CRISPR knockout screen with in vitro repetitive antigen exposure and phenotypic selection to identify gene knockouts that improved the ability of NY-ESO-1 TCR transgenic T cells to repeatedly kill the antigen- and HLA-matched melanoma cell line M407. In two replicate screens, knocking out of the genes CUL5, RNF7 or ARIH2, which are required for proper cullin 5-RING ligase complex (CRL5) assembly, significantly improved long-term tumor cell killing ability of T cells past five rechallenges (one-way ANOVA p<0.0001). Mechanistically, we discovered that CRL5 interacts with SOCS-box containing proteins to regulate TCR stimulation and cytokine signaling through ubiquitination and proteasomal degradation. Ablation of CRL5 in T cells leads to an increase in polyfunctionality with augmented production and release of cytokines and cytolytic molecules across multiple rounds of killing when compared to control T cells (IFNg/TNFa producing T cells; Challenge 1: 18% versus 10%, Stimulation 3: 6% versus 3%, unpaired t-test p<0.0001). We demonstrated through both CRISPR knockout, and pharmacological inhibition of CRL5 using the neddylation inhibitor MLN4924, the negative regulatory mechanism of CRL5 in CD8+ T cells. These results highlight the previously unrecognized regulation of T cell function through E3 ligase-mediated proteasomal degradation, and that CRL5 may be a target to endow lasting effector function to antigen-specific T cells for cancer immunotherapies.
 Citation Format: Justin Saco, Daniel Karin, Nataly Naser Al Deen, Katie Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas. Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain-and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ea44886e24e6da919cb1a70e35d9430057e6c35" target='_blank'>
              Prognostic, biological, and structural implications of FLT3-JMD point mutations in acute myeloid leukemia: an analysis of Alliance studies
              </a>
            </td>
          <td>
            N. Anabtawi, D. Nicolet, Najla Alotaibi, D. Buelow, S. Orwick, Thomas Gregory, Ruchika Raj, Kennedy Coleman, J. Kolitz, B. Powell, William G Blum, Maria R. Baer, John C Byrd, W. Stock, G. Uy, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Xiaolin Cheng, Sharyn D Baker, J. Blachly
          </td>
          <td>2025-01-13</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML), a refractory leukemia stem cell (LSC) disease, is characterized by the Philadelphia chromosome (Ph) t(9;22)(q34;q11.2) that generates the gene fusion between BCR and ABL1 genes. The BCR::ABL1 fusion gene products have a constitutively activated tyrosine kinase that causes leukemogenesis of CML. Tyrosine kinase inhibitors (TKIs) are effective in chronic phase of CML (CML-CP) but ineffective in blast phase of CML (CML-BC). The mechanisms of blast crisis (BC) in CML are highly complicated. This review aims to depict several mechanisms of BC induction in CML-BP. Blast transformation is caused by various mechanisms such as genetic alterations (IKZF1, TP53, INK4-ARF and Rb), epigenetic dysfunctions (aberrant methylation of CEBPα and PU.1, and underexpression of histone demethylase RBP2), suppression of differentiation-mediated transcription factors (CEBPα, CEBPβ, PU.1 and UBE2a), abnormal splicing (IKZF1 and GSK3β) and aberrantly regulated HIF-1α in bone marrow microenvironment. To develop novel therapies against blast cells in CML-BP based on the mechanisms of blast transformation, further studies are required.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee87ee3aa4ca681ef934614f3f0d367c700553ae" target='_blank'>
              Blast Transformation of Chronic Myeloid Leukemia Caused by Genetic Alterations, Epigenetic Dysfunctions and Aberrantly Regulated Factors in Bone Marrow Microenvironment
              </a>
            </td>
          <td>
            MD Mitsuru Sakitani
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Research in  Medical and Clinical Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1d734aed102ed089c1bedbb1c15be484c90edb1" target='_blank'>
              tRNA-derived small RNAs: their role in the mechanisms, biomarkers, and therapeutic strategies of colorectal cancer
              </a>
            </td>
          <td>
            Bo Zhang, Yanru Pan, Zhe Li, Kefeng Hu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The metabolic enzyme aldehyde dehydrogenase 1A1 (ALDH1A1), a cancer stem cell marker associated with poor outcomes in breast cancer, has emerged as a promising therapeutic target in TNBC. The aim of this study was to investigate the role of ALDH1A1 in radiation resistance and redox stress in triple negative breast cancer (TNBC). Functional knockouts of ALDH1A1 were generated by the CRISPR/Cas9-mediated deletion of ALDH1A1 in the SUM159 cell line, and three distinct clonal populations were isolated. Genetic targeting was confirmed by Sanger sequencing, and the loss of ALDH1A1 protein expression was validated by Western blotting. Functional assays assessed ALDEFLUOR activity, cell viability, self-renewal capacity, and reactive oxygen species (ROS) levels with or without radiation in both the bulk population and clonal lines. Interestingly, ALDEFLUOR activity was uniformly lost across all clonal lines; however, functional effects of ALDH1A1 loss on redox stress, survival, and radiation sensitivity were observed in only one clonal population. These findings highlight significant variability in the role of ALDH1A1 among clonal populations, reflecting the complexity of tumor heterogeneity. This underscores the importance of accounting for tumor heterogeneity when targeting ALDH1A1, as certain TNBC subpopulations may rely more heavily on ALDH1A1 function. These insights are critical for developing effective ALDH1A1-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39f8fe1789f12e83b81bf503275e5a974f1d83" target='_blank'>
              CRISPR/Cas9 Targeting of Aldehyde Dehydrogenase 1A1 Reveals Heterogeneous Roles in Radiation Response and Redox Stress Across Clonal Lines in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Grace O. Ajayi, Aihui Ma, Shirin R. Modarai, Lynn M. Opdenaker, Jennifer Sims-Mourtada
          </td>
          <td>2025-03-05</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Second-generation anti-CD19 chimeric antigen receptor (CAR) T-cells incorporating CD28 (1928z) are now standard of care for treatment of refractory/relapsed large B-cell and mantle cell lymphomas. However, clinical use is limited by high toxicity rates, especially cytokine release syndrome (CRS) and immune effector cell-associated neurotoxicity syndrome (ICANS). Modifying intracellular signalling domains can influence CAR T-cell toxicity risk. Although rare, reports of CAR T-cell-derived malignancies underscore the risks of gene-modified cell therapies and the need to assess transgene insertion sites in new products. Toll-like receptor 2 (TLR2) a pattern recognition receptor, is highly expressed by activated CD8+ human T-cells, and can serve as a T-cell costimulatory signal, enhancing, memory formation, and modifying T-cell cytokine profile. In support of a phase 1 clinical trial (ENABLE-1, NCT04049513) evaluating a new third-generation anti-CD19 CAR T-cell product incorporating CD28 and TLR2 co-stimulatory domains (1928T2z), we explored the functional impact of the TLR2 domain in vitro, and characterised gene insertion sites within the 1928T2z product. Incorporating TLR2-derived Toll/IL-1 receptor resistance domain (TIR) between CD28 and CD3ζ domains in the 1928T2z CAR provided comparable CAR T-cell efficacy while producing lower levels of IFN-γ and GM-CSF, cytokines linked to CRS and ICANS pathogenesis, than 1928z. In contrast, positioning the TLR2 domain at the cytoplasmic tail (1928zT2) did not lower IFN-γ and GM-CSF production. Inhibiting TLR2 TIR function using ortho-vanillin, or through targeted mutations abrogating MYD88 recruitment and/or dimerization, impaired CAR T-cell activity, indicating the TLR2 domain contributes to 1928T2z CAR function. Gene expression identified downregulation of IL-6 signalling and upregulation of an anti-inflammatory cytokine pathway in 1928T2z cells compared to comparator CAR T-cells. A phosphoprotein array revealed that a functional TLR2 domain was associated with in higher levels of phosphorylated MEK1/2 (involved in TLR signalling), GATA3 (Th2-linked transcription factor) and CREB (a transcription factor that limits inflammatory responses). Using long-read sequencing and bioinformatics, we characterised insertion sites in a clonal line and polyclonal human CAR T-cells. Results showed mostly intronic or intergenic sites, with less than 10% mapping to exons, aligning with known lentiviral transgene integration profiles. In summary, incorporating a TLR2 co-stimulatory domain into CD28-costimulated anti-CD19 CAR T-cells modulates cytokine profile in a position-dependent manner, with TLR2 placement between CD28 and CD3ζ reducing CRS-and ICANS-linked cytokines while preserving in vitro anti-tumour activity. These findings align with our phase 1 trial data suggesting that 1928T2z CAR T-cells are associated with low incidence of CRS and ICANS, while exhibiting similar clinical efficacy to commercial comparators.
 Citation Format: Nathaniel Dasyam, Yasmin Nouri, Joshua Halpin, Alexander McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, Markus Templin, Qin Le, Peng Li, Ian Hermans, Rachel Perret, Robert Weinkove. Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739d0a74d9a81f832abc4486254c5ebe8688f593" target='_blank'>
              Abstract A018: Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain
              </a>
            </td>
          <td>
            N. Dasyam, Yasmin Nouri, J. Halpin, A. McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, M. Templin, Qin Le, Peng Li, I. Hermans, Rachel Perret, R. Weinkove
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Totipotent cells (TSCs), the origin of mammalian life and foundation to early mammalian embryogenesis, are characterized by the highest differentiation capacity and extensive developmental potential. However, none of the existing embryonic databases have provided epigenetic and transcriptomic resources on totipotency, greatly limiting our understanding of the mechanisms governing the establishment and exit of totipotency. Here, we present Toti, a pioneering multi-omics database exclusively developed for totipotency, covering in vivo, in vitro and genome-edited human and mouse embryonic TSCs, TSC-like cells, pluripotent cells (PSCs), and embryos spanning preimplantation stages, with a total of 8,265 samples. Toti facilitates an in-depth exploration of the molecular mechanisms underlying totipotency by offering Search, Browse and Analysis modules available at http://toti.zju.edu.cn/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0143151fabb07316572f667e24a48e297b397f77" target='_blank'>
              Toti: an integrated multi-omics database to decipher the epigenetic regulation of gene expression in totipotent stem cells
              </a>
            </td>
          <td>
            Yi Chai, Ruiying Zhang, Shunze Jia, Danfei Zhu, Siyi Chen, Xudong Fu, Xin Sheng
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Stem cells are undifferentiated cells that exhibit a bivalent chromatin state that determines their fate. These cells have potential applications in human and animal health and livestock production. Somatic cell nuclear transfer or cloning is currently being used to produce genetically edited animals. A highly differentiated genome is the main obstacle to correcting epigenetic reprogramming by enucleated oocytes during cloning. Activation of pluripotency genes in the somatic genome is a promising strategy to contribute to more efficient epigenetic reprogramming, improving this technique. Recently, epigenome editing has emerged as a new generation of clustered regularly interspaced short palindromic repeats-clustered regularly interspaced short palindromic repeats-associated protein 9 technology with the aim of modifying the cellular epigenome to turn genes on or off without modifying DNA. Here, we characterize the DNA methylation profile of the CpG island spanning the 5' untranslated region to intron 1 of the bovine octamer-binding transcription factor (Oct4) gene in gametes, embryos, and fibroblasts. DNA methylation patterns were categorized into three levels: low (0%-20%), moderate (21%-50%), and high (51%-100%). Sperm and embryos showed a hypomethylation pattern, whereas oocytes exhibited a hypo- to moderate methylation pattern. Fetal and adult skin fibroblasts were hypomethylated and moderately methylated, respectively. These results are essential for future studies aimed at manipulating the expression of Oct4. Thus, epigenome editing can be used to turn on the Oct4 in somatic cells to generate induced pluripotent stem cells. This strategy could potentially convert a fully differentiated cell into a cell with certain degree of pluripotency, facilitating nuclear reprogramming by the enucleated oocyte and improving cloning success rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b18df3c001a51eac902f935e05595c844edc904" target='_blank'>
              Characterization of DNA methylation profile of the entire CpG island spanning the 5' untranslated region to intron 1 of the Oct4/POU5F1 gene in bovine gametes, embryos, and somatic cells.
              </a>
            </td>
          <td>
            Amanda Oliveira Moura, Thainara Christie Ferreira Silva, Alexandre Rodrigues Caetano, N. R. Kussano, M. Dode, Maurício Machaim Franco
          </td>
          <td>2025-02-01</td>
          <td>Animal genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de55f56ba523aef371ff5e151fde820a36dfcf57" target='_blank'>
              Translational and clinical comparison of whole genome and transcriptome to panel sequencing in precision oncology
              </a>
            </td>
          <td>
            I. A. Kerle, Thomas Gross, Anja Kögler, Jonas S. Arnold, Maximilian Werner, Jan-Niklas Eckardt, Elena E. Möhrmann, Marie Arlt, B. Hutter, J. Hüllein, Daniela Richter, Martin M K Schneider, Mario Hlevnjak, L. Möhrmann, D. Hanf, C. Heilig, S. Kreutzfeldt, Maria-Veronica Teleanu, Evelin Schröck, Daniel Hübschmann, Peter Horak, Christoph Heining, Stefan Fröhling, H. Glimm
          </td>
          <td>2025-01-10</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Long interspersed nuclear element 1 (L1) retrotransposons represent a vast source of divergent genetic information. However, mechanistic analysis of whether and how L1s contribute to human developmental programs is lacking, in part due to the challenges associated with specific profiling and manipulation of human L1 expression. Here we show that thousands of hominoid-specific L1 integrants are expressed in human induced pluripotent stem cells and cerebral organoids. The activity of individual L1 promoters is surprisingly divergent and correlates with an active epigenetic state. Efficient on-target CRISPRi silencing of L1s revealed nearly a hundred co-opted L1-derived chimeric transcripts and L1 silencing resulted in changes in neural differentiation programs and reduced cerebral organoid size. Together, these data implicate L1s and L1-derived transcripts in hominoid-specific CNS developmental processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76dd8be460b1842f859ff053be4194a72fd59c71" target='_blank'>
              LINE-1 retrotransposons regulate the exit of human pluripotency and early brain development
              </a>
            </td>
          <td>
            Anita Adami, R. Garza, Patricia Gerdes, Pia A. Johansson, F. Dorazehi, Symela Koutounidou, Laura Castilla-Vallmanya, Diahann A. M. Atacho, Yogita Sharma, Jenny G. Johansson, Oliver Tam, A. Kirkeby, Roger A. Barker, Molly Gale-Hammell, C. Douse, Johan Jakobsson
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Aging is considered the contributory accumulation of abruptions occurring through cell signaling cascades, which ultimately cause changes in physical functions, cell fate, and damage across all organ systems. DNA damage response (DDR) also occurs through telomere shortening, tumor formation, mitochondrial dysfunction, and so forth. Cellular aging occurs through cell cycle arrest, which is the result of extended DDR cascade signaling networks via MDC1, 53BP1, H2AX, ATM, ARF, P53, P13-Akt, BRAF, Sirtuins, NAD + , and so forth. These persistent cell cycle arrests initiated by DDR and other associated stress-induced signals promote a permanent state of cell cycle arrest called senescence-associated secretory phenotype (SASP). However, cellular aging gets accelerated with faulty DNA repair systems, and the produced senescent cells further generate various promoting contributors to age-related dysfunctional diseases including SASP. Any changes to these factors contribute to age-related disease development. Therefore, this review explores anti-aging factors targeting DDR and SASP regulation and their detailed signaling networks. In addition, it allows researchers to identify anti-aging targets and anti-aging therapeutic strategies based on identified and nonidentified targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3959b933b4c238cc534f4bd3bbf4f10f6618c5" target='_blank'>
              Recent Advances in Anti-Aging Therapeutic Strategies Targeting DNA Damage Response and Senescence-Associated Secretory Phenotype-Linked Signaling Cascade.
              </a>
            </td>
          <td>
            Jawad Nadeem, Razia Sultana, Amna Parveen, Sun Yeou Kim
          </td>
          <td>2025-03-01</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The canonical DNA damage response (DDR) maintains genome stability, involving DNA synthesis/cell cycle arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously discovered and characterized a noncanonical cellular response that is specific to nonblocking replication stress, i.e., low-level stress (LoL-DDR), in primary cells. Although this response generates replication stress-induced reactive oxygen species (RIR), it triggers a program that prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a precise threshold triggers the canonical DDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: 1-RIR are not detoxified under high-level stress conditions, resulting in a continuous increase in the dose-response curve of RIR production; 2-RIR are not produced by NADPH oxidases; and 3-replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” involving replication-stress and ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfa2fab8a78a43f5b506788fc0cb6d9a375f31e" target='_blank'>
              Dysregulation of replication stress-induced ROS in transformed cell lines: a vicious circle at cancer initiation
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract Background The efficacy of immune checkpoint blockade (ICB) depends on restoring immune recognition of cancer cells that have evaded immune surveillance. Transforming growth factor-beta (TGFβ) is associated with immune-poor, so-called cold tumors whereas loss of its signaling promotes DNA misrepair that could stimulate immune response. Methods We analyzed transcriptomic data from IMvigor210, The Cancer Genome Atlas, and Tumor Immune Syngeneic MOuse data sets to evaluate the predictive value of high βAlt, a score representing low expression of a signature consisting of TGFβ targets and high expression of genes involved in error-prone DNA repair. The immune context of βAlt was assessed by evaluating tumor-educated immune signatures. An ICB-resistant, high βAlt preclinical tumor model was treated with a TGFβ inhibitor, radiation, and/or ICB and assessed for immune composition and tumor control. Results We found that a high βAlt score predicts ICB response yet is paradoxically associated with an immune-poor tumor microenvironmentcancer in both human and mouse tumors. We postulated that high βAlt cancers consist of cancer cells in which loss of TGFβ signaling generates a TGFβ rich, immunosuppressive tumor microenvironment. Accordingly, preclinical modeling showed that TGFβ inhibition followed by radiotherapy could convert an immune-poor, high βAlt tumor to an immune-rich, ICB-responsive tumor. Mechanistically, TGFβ inhibition increased activated natural killer (NK) cells, which were required to recruit lymphocytes to respond to ICB in irradiated tumors. NK cell activation signatures were also increased in high βAlt, cold mouse and human tumors that responded to ICB. Conclusions These studies indicate that loss of TGFβ signaling competency and gain of error-prone DNA repair identifies a subset of cold tumors that are responsive to ICB. Our mechanistic studies show that inhibiting TGFβ activity can convert a high βAlt, cold tumor into ICB-responsive tumors via NK cells. A biomarker consisting of combined TGFβ, DNA repair, and immune context signatures is a means to prospectively identify patients whose cancers may be converted from cold to hot with appropriate therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16f08083204e934f5372293d784704ac2a30ac3a" target='_blank'>
              Identification of a conserved subset of cold tumors responsive to immune checkpoint blockade
              </a>
            </td>
          <td>
            Jade Moore, Jim Gkantalis, Inés Guix, William Chou, Kobe Yuen, Ann A. Lazar, Matthew Spitzer, Alexis J. Combes, M. Barcellos-Hoff
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Comprehensive genetic analysis of tumors with exome or whole genome sequencing has enabled the identification of the genes that are recurrently mutated in cancer. This has stimulated a series of exciting advances over the past 15 years, guiding us to new molecular biomarkers and therapeutic targets among the common mature B-cell neoplasms. In particular, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL) and Burkitt lymphoma (BL) have each been the subject of considerable attention in this field. Currently, more than 850 genes have been reported as targets of protein-coding mutations in at least one of these entities. To reduce this to a manageable size, we describe a systematic approach to prioritize and categorize these genes, based on the quality and type of supporting data. For each entity, we provide a list of candidate driver genes categorized into Tier 1 (high-confidence genes), Tier 2 (candidate driver genes), or Tier 3 (lowest-confidence genes). Collectively, this reduces the number of high-confidence genes for these three lymphomas to a mere 144. This further affirms the substantial overlap between the genes relevant in DLBCL and each of FL and BL. These highly curated and annotated gene lists will continue to be maintained as a resource to the community. These results emphasize the extent of the knowledge gap regarding the role of each of these genes in lymphomagenesis. We offer our perspective on how to accelerate the experimental confirmation of drivers using a variety of model systems, using these lists as a guide for prioritizing genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf53884a55922c51a797521fda638d01af815c" target='_blank'>
              Audit of B-cell Cancer Genes.
              </a>
            </td>
          <td>
            K. Coyle, K. Dreval, Daniel J Hodson, Ryan D Morin
          </td>
          <td>2025-01-24</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Perturb-seq is a powerful approach to systematically assess how genes and enhancers impact the molecular and cellular pathways of development and disease. However, technical challenges have limited its application in stem cell-based systems. Here, we benchmarked Perturb-seq across multiple CRISPRi modalities, on diverse genomic targets, in multiple human pluripotent stem cells, during directed differentiation to multiple lineages, and across multiple sgRNA delivery systems. To ensure cost-effective production of large-scale Perturb-seq datasets as part of the Impact of Genomic Variants on Function (IGVF) consortium, our optimized protocol dynamically assesses experiment quality across the weeks-long procedure. Our analysis of 1,996,260 sequenced cells across benchmarking datasets reveals shared regulatory networks linking disease-associated enhancers and genes with downstream targets during cardiomyocyte differentiation. This study establishes open tools and resources for interrogating genome function during stem cell differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a019d0592b40aaea0bb2bdf58ea166a338fcf9" target='_blank'>
              Benchmarking and optimizing Perturb-seq in differentiating human pluripotent stem cells
              </a>
            </td>
          <td>
            Sushama Sivakumar, Yihan Wang, Sean C Goetsch, Vrushali Pandit, Lei Wang, Huan Zhao, Anjana Sundarrajan, Daniel A Armendariz, Chikara Takeuchi, Mpathi Z. Nzima, Wei-Chen Chen, Ashley E. Dederich, Lauretta El Hayek, Taosha Gao, Renad Ghazawi, Ashlesha Gogate, Kiran J. Kaur, Hyung Bum Kim, Melissa K. McCoy, H. Niederstrasser, Seiya Oura, Carolos A Pinzon-Arteaga, Menaka Sanghvi, Daniel A. Schmitz, Leqian Yu, Yanfeng Zhang, Qinbo Zhou, W. Kraus, Lin Xu, Jun Wu, Bruce A. Posner, Maria H. Chahrour, Gary C. Hon, N. Munshi
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cb9fafc9e4ae8646c09de8af8d088ea1242ab7" target='_blank'>
              Mammalian Species-Specific Resistance to Mammary Cancer
              </a>
            </td>
          <td>
            I. Barash
          </td>
          <td>2025-03-06</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0687d954e3a7ce3f1c8da610c5b175b62bd0c491" target='_blank'>
              Single-cell multiomics reveals a gene regulatory circuit driving leukemia cell differentiation.
              </a>
            </td>
          <td>
            Xin Tian, Liuqingqing Zhang, Guiqiyang Xiang, Yijia Tang, Ping Zhu, Shuting Yu, Fangying Jiang, Shuai Wang, Jinzeng Wang, Yao Dai, Desheng Zheng, Jianbiao Wang, Xiangqin Weng, Shengyue Wang, Yun Tan, Feng Liu
          </td>
          <td>2025-02-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Significance: Growing evidence indicates the importance of redox reactions homeostasis, mediated predominantly by reactive oxygen species (ROS) in influencing the development, differentiation, progression, metastasis, programmed cell death, tumor microenvironment, and therapeutic resistance of cancer. Therefore, reviewing the ROS-linked epigenetic changes in cancer is fundamental to understanding the progression and prevention of cancer. Recent Advances: We review in depth the molecular mechanisms involved in ROS-mediated epigenetic changes that lead to alteration of gene expression by altering DNA, modifying histones, and remodeling chromatin and noncoding RNA. Critical Issues: In cancerous cells, alterations of the gene-expression regulatory elements could be generated by the virtue of imbalance in tumor microenvironment. Various oxidizing agents and mitochondrial electron transport chain are the major pathways that generate ROS. ROS plays a key role in carcinogenesis by activating pro-inflammatory signaling pathways and DNA damage. This loss of ROS-mediated epigenetic regulation of the signaling pathways may promote tumorigenesis. We address all such aspects in this review. Future Directions: Developments in this growing field of epigenetics are expected to contribute to further our understanding of human health and diseases such as cancer and to test the clinical applications of redox-based therapy. Recent studies of the cancer-epigenetic landscape have revealed pervasive deregulation of the epigenetic factors in cancer. Thus, the study of interaction between ROS and epigenetic factors in cancer holds a great promise in the development of effective and targeted treatment modalities. Antioxid. Redox Signal. 00, 000-000.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90fb27d72b5f41dadb3feb89a750c00e2da4835a" target='_blank'>
              Epigenetic Control of Redox Pathways in Cancer Progression.
              </a>
            </td>
          <td>
            Vandit Shah, Hiu Yan Lam, Charlene Hoi-Mun Leong, Reo Sakaizawa, Jigna S Shah, Alan Prem Kumar
          </td>
          <td>2025-01-16</td>
          <td>Antioxidants & redox signaling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) remains a serious threat to global health, with rising incidence and mortality rates. Therapeutic options for advanced HCC are quite limited, and the overall prognosis remains poor. Recent advancements in immunotherapy, particularly immune-checkpoint blockade (ICB) targeting anti-PD1/PD-L1 and anti-CTLA4, have facilitated a paradigm shift in cancer treatment, demonstrating substantial survival benefits across various cancer types, including HCC. However, only a subset of HCC patients exhibit a favorable response to ICB therapy, and its efficacy is often hindered by the development of resistance. There are many studies to explore the underlying mechanisms of ICB response. In this review, we compiled the latest progression in immunotherapies for HCC and systematically summarized the sophisticated mechanisms by which components of the tumor microenvironment (TME) regulate resistance to ICB therapy. Additionally, we also outlined some scientific rationale strategies to boost antitumor immunity and enhance the efficacy of ICB in HCC. These insights may serve as a roadmap for future research and help improve outcomes for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7cb79517a5aec504caaabf1fd85cb4e7d72745b" target='_blank'>
              Sophisticated roles of tumor microenvironment in resistance to immune checkpoint blockade therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yi-Zhe Zhang, Yunshu Ma, Ensi Ma, Xizhi Chen, Yue Zhang, Baobing Yin, Jing Zhao
          </td>
          <td>2025-02-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cutaneous melanoma, one of the most fatal and aggressive types of skin cancer, is a malignant tumor created from pigment-producing cells called melanocytes. The mor-bidity and incidence rate of melanoma are increasing around the world. Considering all developments in diagnostic and therapeutic approaches, early diagnosis and more effective targeted therapies are still urgent for melanoma. Non-coding RNAs (ncRNAs), including microRNAs and long non-coding RNAs, are key role players in fundamental cellular pro-cesses and have various biological properties. It has been demonstrated that the dysregula-tion of these biomarkers contributes to melanoma pathogenesis. Furthermore, programmed cell death (PCD), as an important cellular event, has been shown to play a role in the initi-ation and progression of cutaneous melanoma. Recent investigations have reported that the interplay between ncRNAs and PCD can be considered a novel choice for targeted therapy in melanoma. Thus, this review discusses PCD-related ncRNAs in melanoma, focusing on underlying pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c1c5de5601f2f61c348bb80645910cf0834e556" target='_blank'>
              Regulatory Effect of Non-Coding RNAs on Programmed Cell Death in Melanoma: Novel Therapeutic Crosstalk for Targeted Therapy.
              </a>
            </td>
          <td>
            Andarz Fazlollahpour-Naghibi, S. R. Taha, Mahdieh Shariatzadeh, Danyal Daneshdoust, Kimia Bagheri, T. Farkhondeh, M. Samini, M. H. Pourhanifeh, Saeed Samarghandian
          </td>
          <td>2025-02-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3200dafb5f7a30deaf8fb5764586a7d405d1e8" target='_blank'>
              Novel histone modifications and liver cancer: emerging frontiers in epigenetic regulation
              </a>
            </td>
          <td>
            Zhonghua Wang, Ziwen Liu, Mengxin Lv, Zhou Luan, Tao Li, Jinhua Hu
          </td>
          <td>2025-02-20</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A large fraction of the human genome is transcribed in RNA molecules that do not encode for proteins but that do have a crucial role in regulating almost every level of gene expression and, thus, define the specific phenotype of each cell. These non-coding RNAs include well-characterized microRNAs and thousands of less-defined longer transcripts, named long non-coding RNAs. Both types markedly affect the onset and the progression of numerous pathologies, ranging from cancer to vascular and neuro-degenerative diseases. In recent years, a substantial effort has been made to design drugs targeting ncRNAs, and promising advancements have been produced from micro-RNA mimics and inhibitors. Each ncRNA controls several targets, and the overall effect of its inhibition or overexpression depends on the function of the set of genes it regulates. Therefore, in selecting the most appropriate target, and predicting the final outcome of ncRNA-based therapies, it is crucial to have and utilize detailed and accurate knowledge of their functional interactions. In this review, I recapitulate the principal resources which collect information on microRNA and lncRNA networks, focusing on the non-homogeneity of the data that result from disparate approaches. I highlight the role of RNA identifiers and interaction evidence standardization in helping the user to filter and integrate data derived from different databases in a reliable functional web of regulative relations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45039e8fa4423e21f993b2688bba0179d8a2c57" target='_blank'>
              The Relevance of the Accurate Annotation of Micro and Long Non-Coding RNA Interactions for the Development of Therapies
              </a>
            </td>
          <td>
            Simona Panni
          </td>
          <td>2025-02-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9281e0d64a5756a86b842209be41d38103ffa691" target='_blank'>
              Colorectal carcinogenesis in the Lynch syndromes and familial adenomatous polyposis: trigger events and downstream consequences
              </a>
            </td>
          <td>
            Pål Møller, A. Ahadova, Matthias Kloor, Toni T. Seppälä, John Burn, Saskia Haupt, F. Macrae, M. Dominguez-Valentin, Gabriela Möslein, Annika Lindblom, Lone Sunde, Ingrid Winship, Gabriel Capellá, Kevin Monahan, D. Buchanan, D. G. Evans, E. Hovig, Julian R. Sampson
          </td>
          <td>2025-01-23</td>
          <td>Hereditary Cancer in Clinical Practice</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f17ec0f574b24441e84e586e6063343737a2a9" target='_blank'>
              Massively parallel in vivo Perturb-seq screening.
              </a>
            </td>
          <td>
            Xinhe Zheng, Patrick C Thompson, Cassandra M. White, Xin Jin
          </td>
          <td>2025-02-12</td>
          <td>Nature protocols</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Clinical risk scores in chronic myeloid leukemia (CML) are inadequate for identifying chronic phase (CP) patients at high-risk of blast crisis (BC) progression. The lack of accurate predictive tests hamper timely interventions, including stem cell transplants, that are more effective in early disease. By interrogating a single cell atlas of primary imatinib resistance for a BC-like gene expression signature, we identify aberrant CD42A+ megakaryocytic and CD10+CD19+ lymphoid progenitor expansion, as well as STAT1- and IFNγ-related inflammatory programs, as consistent features in the bone marrow and peripheral blood of CP patients at high-risk of BC transformation. We develop multi-color flow cytometry-based tests (MFC) to detect these features in CP patients at the time of diagnosis. Validating our MFC panels on a combined Australia-Singapore cohort comprising 28 CP patients, including 14 who underwent transformation, we demonstrate their ability to detect 100% of CP patients who transform with no false positives. Our findings highlight small populations of inflamed hematopoietic stem and progenitor cells, present at CP diagnosis, as powerful harbingers of future BC transformation. The ability of MFC panels to detect these cells at diagnosis support their inclusion as accurate risk-assessment tools to improve management of high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695b8c7eb368f693a97922122a5ffdc4d6cd3154" target='_blank'>
              Single-cell transcriptomics-guided development of flow cytometric tests predicting chronic myeloid leukemia blast crisis transformation at chronic phase diagnosis
              </a>
            </td>
          <td>
            Mengge Yu, Vaidehi Krishnan, Pavithra Shyamsundar, K. Lee, Florian Schmidt, Xi Ren, Charles T.H. Chuah, Shyam Prabhakar, V. Saunders, D. Yeung, N. Shanmuganathan, Agnes S. M. Yong, Timothy P. Hughes, S. T. Ong
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="LMNA-associated congenital muscular dystrophy is a currently incurable rare genetic disorder characterized by early-onset muscle weakness, dilated cardiomyopathy and respiratory failure, resulting from mutations in the LMNA gene. In this study, we assessed the potential of a CRISPR-mediated strategy to eliminate the mutant allele Lmna c.745C>T, p.R249W using a mutation specific guide (sg745T). Results from R249W-mutation-carrying cellular models showed specific activity of the Cas9/sg745T complex towards the mutant allele. This property varied depending on the concentration of CRISPR components, with a loss of specificity observed with increased dosage. We tested this strategy in vivo using adeno-associated virus delivery in LmnaR249W mice. Despite being associated with a modest CRISPR activity, this therapeutic approach resulted in a 10% increase in the survival of R249W homozygous mice. Interestingly, a similar CRISPR activity improved the cardiac pathology developed by Lmna+/R249W animals, significantly extending their median survival. These results represent the first therapeutic validation of a CRISPR/Cas9-mediated gene editing strategy for the treatment of LMNA-associated congenital muscular dystrophy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eff96c5b33b1fd18b969ab18137992e335a5be4" target='_blank'>
              CRISPR/Cas9-mediated elimination of the LMNA c.745C>G pathogenic mutation enhances survival and cardiac function in LMNA-associated congenital muscular dystrophy
              </a>
            </td>
          <td>
            Déborah Gómez-Domínguez, Carolina Epifano, Iván Hernández, Borja Vilaplana-Martí, Sergi César, Antonio de Molina-Iracheta, Miguel Sena-Esteves, Georgia Sarquella-Brugada, Ignacio Pérez de Castro
          </td>
          <td>2025-02-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chronic inflammation is a well-established risk factor for cancer, but the underlying molecular mechanisms remain unclear. Using a mouse model of colitis, we demonstrate that colonic stem cells retain an epigenetic memory of inflammation following disease resolution, characterized by a cumulative gain of activator protein 1 (AP-1) transcription factor activity. Further, we develop SHARE-TRACE, a method that enables simultaneous profiling of gene expression, chromatin accessibility and clonal history in single cells, enabling high resolution tracking of epigenomic memory. This reveals that inflammatory memory is propagated cell-intrinsically and inherited through stem cell lineages, with certain clones demonstrating dramatically stronger memory than others. Finally, we show that colitis primes stem cells for amplified expression of regenerative gene programs following oncogenic mutation that accelerate tumor growth. This includes a subpopulation of tumors that have exceptionally high AP-1 activity and the additional upregulation of pro-oncogenic programs. Together, our findings provide a mechanistic link between chronic inflammation and malignancy, revealing how long-lived epigenetic alterations in regenerative tissues may contribute to disease susceptibility and suggesting potential therapeutic strategies to mitigate cancer risk in patients with chronic inflammatory conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor-specific antigens, also known as neoantigens, have potential utility in anti-cancer immunotherapy, including immune checkpoint blockade (ICB), neoantigen-specific T cell receptor-engineered T (TCR-T), chimeric antigen receptor T (CAR-T), and therapeutic cancer vaccines (TCVs). After recognizing presented neoantigens, the immune system becomes activated and triggers the death of tumor cells. Neoantigens may be derived from multiple origins, including somatic mutations (single nucleotide variants, insertion/deletions, and gene fusions), circular RNAs, alternative splicing, RNA editing, and polymorphic microbiome. An increasing amount of bioinformatics tools and algorithms are being developed to predict tumor neoantigens derived from different sources, which may require inputs from different multi-omics data. In addition, calculating the peptide-major histocompatibility complex (MHC) affinity can aid in selecting putative neoantigens, as high binding affinities facilitate antigen presentation. Based on these approaches and previous experiments, many resources were developed to reveal the landscape of tumor neoantigens across multiple cancer types. Herein, we summarized these tools, algorithms, and resources to provide an overview of computational analysis for neoantigen discovery and prioritization, as well as the future development of potential clinical utilities in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b6cef04b3606a814455c2e9548b49b282ba61" target='_blank'>
              Characterization of Tumor Antigens from Multi-omics Data: Computational Approaches and Resources.
              </a>
            </td>
          <td>
            Yunzhe Wang, James Wengler, Yuzhu Fang, Joseph Zhou, Hang Ruan, Zhao Zhang, Leng Han
          </td>
          <td>2025-01-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>