<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-14 06:32:30 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>8</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, P. Maini, Tyler Cassidy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Tumors consist of cancer cells with different genetic, epigenetic, and phenotypic properties. Cancer stem cells are an important subpopulation of heterogeneous cancer cells and are capable of initiating and propagating tumors. The term cancer stem cells has become broader in efforts to understand their phenotypic plasticity to switch fates between self-renewal and differentiation. Cancer stem cell plasticity is significantly associated with the initiation of metastasis, resistance to therapy, and tumor recurrence. With our broadened knowledge of epigenetic regulation and metabolic reprogramming as key elements enabling such capabilities, an expansive body of literature has demonstrated the functional importance of each element in contributing to cancer stem cell characteristics. Recently, the direct interplay between epigenetic regulation and metabolic reprogramming has begun to be appreciated in the context of cancer stem cells with growing interest. In this review, we discuss the mechanisms by which cancer stem cells orchestrate the reciprocal regulation of cellular metabolism and epigenetic alterations. In the discussion, compelling, unanswered questions on this topic have been elaborated for the interest of the research community and how recent technological developments help tackle such research ideas. A comprehensive understanding of cancer stem cell attributes that are largely governed by epigenetic and metabolic reprogramming would enable the advancement of precise therapeutic options and the prediction of better responses to drugs, holding great promise in cancer treatment and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a352f514fc793046331539c8fc8b91b79542075" target='_blank'>
              Interplay between epigenetics and metabolism controls cancer stem cell plasticity
              </a>
            </td>
          <td>
            Jee-Eun Choi, Inwha Baek
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Neuroblastoma (NB) is the most common pediatric solid tumor, responsible for about 15% of childhood cancer-related mortality in the United States. Multiple therapies have been established, which include chemotherapy, radiotherapy, immunotherapy and molecularly-targeted therapy, however resistance to these therapies is the major hinderance in curing NB. RAS pathway mutations are prevalent in relapsed NB and are associated with a poorer overall survival. Therefore, comprehending the mechanisms leading to therapy resistance has become an urgent need. In this study, we attempted to identify the resistance mechanism of NB cell lines towards RAS inhibitors. RAS-altered NB cell lines are treated with trametinib, a MAP kinase pathway inhibitor to perform cell viability assays, immunoblotting and RT PCR. Copy number alteration and translocation studies are performed using spectral karyotyping (SKY) and multiplex interphase fluorescent in-situ hybridization (miFISH). We observed that unlike other RAS-altered NB cell lines (SK-N-AS, CHP-212, LA-N-6, SK-N-BE(2)-C), NB-Eb-C1, which harbors a KRAS G12D mutation shows resistance to both MRTX1133 (KRAS G12D direct inhibitor) and trametinib. Immunoblot against crucial genes regulating NB showed high expression of MYCN, the most relevant biomarker of high-risk neuroblastoma. MYCN is unfortunately undruggable hence, standard treatment for MYCN-amplified neuroblastoma patients does not include targeting MYCN itself. We therefore pursued a hypothesis-driven approach towards identifying possible mechanisms responsible for MYCN driven MEKi-resistance in RAS pathway-altered NB cells. MYCN is known to cross talk with the RAS pathway, however this mechanism is not well understood in NB. Therefore, we performed the clonal assessment of NB cell lines using miFISH and to the best of our knowledge are the first to report that, NB-Eb-C1 has a hypertriploid to hypotetraploid baseline with a low copy-number gain of MYCN. To have a profound understanding of the functionality of MYCN in NB, we have taken 3 different NB cell lines with varied MYCN status. CHP-212 (MYCN amplified-extrachromosomal), SK-N-BE(2)-C (MYCN amplified-homogenously staining region), SK-N-AS (MYCN neutral). SKY and FISH revealed that the MYCN copy-number gain in NBEB cells is the result of a translocation between chromosome 2 and 5 involving MYCN. Of note, another frequently abnormally expressed gene in NB, ALK, is co-translocated with MYCN suggesting potential involvement of both genes in imparting chemoresistance. To understand the mechanism of MYCN overexpression in NBEB we will be performing HiC to check for promoter/enhancer hijacking. Also, to better define this translocation and studying other translocations involved we are performing optical genome mapping using BioNano. Other functional experiments will be performed to look at the impact of the RAS/MAPK alteration on MYCN protein stability. We are confident that this study will contribute towards developing new combinations of targeted therapies to improve outcomes in RAS-altered NB.
 Citation Format: Subhra Dash, Kerstin Heselmeyer-Haddad, Stacy Stauffer, Lucas Stauffer, Danny Wangsa, Marielli E. Yohe. Investigating mechanisms responsible for MAP kinase pathway resistance in RAS-altered neuroblastoma cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b3e4035267dd00bede1ab23d0c552e939dc4a96" target='_blank'>
              Abstract A053 Investigating mechanisms responsible for MAP kinase pathway resistance in RAS-altered neuroblastoma cells
              </a>
            </td>
          <td>
            Subhra Dash, Kerstin Heselmeyer-Haddad, Stacy Stauffer, Lucas Stauffer, Danny Wangsa, Marielli E. Yohe
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, Takayuki Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e459f679b760cbc6c5f4d15f5d578e006d0e8b9" target='_blank'>
              Establishment and characterization of 18 Sarcoma Cell Lines: Unraveling the Molecular Mechanisms of Doxorubicin Resistance in Sarcoma Cell Lines
              </a>
            </td>
          <td>
            Young-Eun Cho, Soon-Chan Kim, Ha Jeong Kim, Ilkyu Han, Ja-Lok Ku
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401dfb6b1e61a2b7931608ee7b632746b618204c" target='_blank'>
              Potential promising of synthetic lethality in cancer research and treatment.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Behnam Najafiyan, Reza Morovatshoar, Mahdieh Khorsandi, Adib Dashtizadeh, Arash Kiani, Farnoosh Farzam, Kimia Sadat Kazemi, Mohsen Nabi Afjadi
          </td>
          <td>2024-09-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cervical cancer (CC), one of the most prevalent and detrimental gynaecologic cancers, evolves through genetic and epigenetic alterations resulting in the promotion of oncogenic activity and dysfunction of tumour-suppressing mechanisms. Despite medical advancement, the prognosis for advanced-stage patients remains extremely low due to high recurrence rates and resistance to existing treatments. Thereby, the search for potential prognostic biomarkers is heightened to unravel new modalities of CC pathogenesis and to develop novel anti-cancer therapies. Epitranscriptomic modifications, reversible epigenetic RNA modifications, regulate various biological processes by deciding RNA fate to mediating RNA interactions. This narrative review provides insight into the cellular and molecular roles of endogenous RNA-editing proteins and their associated epitranscriptomic modifications, especially N6-methyladenosine (m6A), 5-methylcytosine (m5C) and N1-methyladenosine (m1A), in governing the development, progression and metastasis of CC. We discussed the in-depth epitranscriptomic mechanisms underlying the regulation of over 50 RNAs responsible for tumorigenesis, proliferation, migration, invasion, survival, autophagy, stemness, epithelial-mesenchymal transition, metabolism (glucose, lipid, glutamate and glutamine), resistance (drug and radiation), angiogenesis and recurrence of CC. Additionally, we provided a concise overview of the therapeutic potential of targeting the altered expression of endogenous RNA-editing proteins and aberrant deposition of RNA modifications on both coding and non-coding RNAs in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d247af5a7e732d73b316957acc73d51f868974f4" target='_blank'>
              Epitranscriptomics and cervical cancer: the emerging role of m6A, m5C and m1A RNA modifications.
              </a>
            </td>
          <td>
            Akshat D. Modi, Hira Zahid, Ashlyn Chase Southerland, Dharmeshkumar M Modi
          </td>
          <td>2024-10-08</td>
          <td>Expert reviews in molecular medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Laryngeal cancer poses a substantial challenge in head and neck oncology, and there is a growing focus on customized medicine techniques. The present state of gene expression indicators in laryngeal cancer and their potential to inform tailored therapy choices are thoroughly examined in this review. We examine significant molecular changes, such as TP53, CDKN2A, PIK3CA, and NOTCH1 mutations, which have been identified as important participants in the development of laryngeal cancer. The study investigates the predictive and prognostic significance of these genetic markers in addition to the function of epigenetic changes such as the methylation of the MGMT promoter. We also go over the importance of cancer stem cell-related gene expression patterns, specifically CD44 and ALDH1A1 expression, in therapy resistance and disease progression. The review focuses on indicators, including PD-L1, CTLA-4, and tumor mutational burden (TMB) in predicting immunotherapy responses, highlighting recent developments in our understanding of the intricate interactions between tumor genetics and the immune milieu. We also investigate the potential for improving prognosis accuracy and treatment selection by the integration of multi-gene expression panels with clinicopathological variables. The necessity for uniform testing and interpretation techniques is one of the difficulties, in implementing these molecular insights into clinical practice, that are discussed. This review seeks to provide a comprehensive framework for promoting personalized cancer therapy by combining the most recent data on gene expression profiling in laryngeal cancer. Molecularly guided treatment options may enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f13a062350e0524f07e4277eec8cead4cabd370" target='_blank'>
              Personalized Treatment Strategies via Integration of Gene Expression Biomarkers in Molecular Profiling of Laryngeal Cancer
              </a>
            </td>
          <td>
            Antonio Maniaci, Giovanni Giurdanella, Carlos Chiesa Estomba, Simone Mauramati, Andy Bertolin, Marco Lionello, M. Mayo-Yáñez, Paolo Boscolo Rizzo, J. Lechien, M. Lentini
          </td>
          <td>2024-10-10</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this review, we revisit the pivotal role of fibroblast growth factor receptor 3 (FGFR3) in bladder cancer (BLCA), underscoring its prevalence in both non-muscle-invasive and muscle-invasive forms of the disease. FGFR3 mutations in up to half of BLCAs play a well-established role in tumorigenesis, shaping distinct tumor initiation patterns and impacting the tumor microenvironment (TME). Emphasizing the importance of considering epithelial-mesenchymal transition profile and TME status, we revisit their relevance in predicting responses to immune checkpoint inhibitors in FGFR3-mutated BLCAs. This writing highlights the initially promising yet transient efficacy of the FGFR inhibitor Erdafitinib on FGFR3-mutated BLCA, stressing the pressing need to unravel resistance mechanisms and identify co-targets for future combinatorial studies. A thorough analysis of recent preclinical and clinical evidence reveals resistance mechanisms, including secondary mutations, epigenetic alterations in pathway effectors, phenotypic heterogeneity, and population-specific variations within FGFR3 mutational status. Lastly, we discuss the potential of combinatorial treatments and concepts like synthetic lethality for discovering more effective targeted therapies against FGFR3-mutated BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d32b54c415deb1782db39b2832fb5448db0b31" target='_blank'>
              FGFR3 alterations in bladder cancer: Sensitivity and resistance to targeted therapies.
              </a>
            </td>
          <td>
            Maxim Noeraparast, Katarina Krajina, Renate Pichler, Dora Niedersüß-Beke, S. Shariat, Viktor Grünwald, S. Ahyai, Martin Pichler
          </td>
          <td>2024-08-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer, a heterogeneous and intricate disease, ranks among the leading causes of mortality in women. Restricted therapeutic choices, drug resistance, recurrence, and metastasis are the predominant conditions that lead to mortality. Accumulating evidence has shown breast cancer initiation and progression happen through a multifaceted and intricate process that involves numerous genetic and epigenetic alterations. The modulation of gene expression through epigenetic modifications, encompassing DNA methylation, histone alterations, and non-coding RNA regulation, has emerged as a fascinating field that represents a new avenue for breast cancer therapy. This review emphasizes various aberrant epigenetic regulations implicated in the onset and advancement of breast cancer. The critical epigenetic modifications closely associated with estrogen signaling, epithelial-to-mesenchymal transition (EMT), cancer stemness, and drug resistance have been discussed extensively. Moreover, it highlights current epi-drugs, including DNA modifying agents, histone acetyltransferase inhibitors, histone deacetylase inhibitors, histone methyltransferase inhibitors, and histone demethyltransferase inhibitors used for breast cancer treatment. Nonetheless, we described current investigations pertaining to combination therapy employing epi-drugs and future challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f756e86bcaa3afefdda7ee5d7e501dcc5e41b11" target='_blank'>
              Epigenetic Modulations in Breast Cancer: An Emerging Paradigm in Therapeutic Implications.
              </a>
            </td>
          <td>
            Shilpi Sarkar, Dheepika Venkatesh, Thirukumaran Kandasamy, S. S. Ghosh
          </td>
          <td>2024-08-19</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole-genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2,726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirically testing all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated two compounds as potent PGCC inhibitors. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f39f65e370bc9aafe05599fe5e71c2036a3f471" target='_blank'>
              High-Throughput Empirical and Virtual Screening to Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract RNA sequencing technology combining short read and long read analysis can be used to detect chimeric RNAs in malignant cells. Here, we propose an integrated approach that uses k-mers to analyze indexed datasets. This approach is used to identify chimeric RNA in chronic myelomonocytic leukemia (CMML) cells, a myeloid malignancy that associates features of myelodysplastic and myeloproliferative neoplasms. In virtually every CMML patient, new generation sequencing identifies one or several somatic driver mutations, typically affecting epigenetic, splicing and signaling genes. In contrast, cytogenetic aberrations are currently detected in only one third of the cases. Nevertheless, chromosomal abnormalities contribute to patient stratification, some of them being associated with higher risk of poor outcome, e.g. through transformation into acute myeloid leukemia (AML). Our approach selects four chimeric RNAs that have been detected and validated in CMML cells. We further focus on NRIP1-MIR99AHG, as this fusion has also recently been detected in AML cells. We show that this fusion encodes three isoforms, including a novel one. Further studies will decipher the biological significance of such a fusion and its potential to improve disease stratification. Taken together, this report demonstrates the ability of a large-scale approach to detect chimeric RNAs in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20cd2e1e29866db7633afbaeb3e311e423e92cc0" target='_blank'>
              Effective requesting method to detect fusion transcripts in chronic myelomonocytic leukemia RNA-seq
              </a>
            </td>
          <td>
            Florence Rufflé, Jérôme Reboul, A. Boureux, Benoit Guibert, Chloé Bessière, Raíssa Silva, Eric Jourdan, Jean-Baptiste Gaillard, A. Boland, J. Deleuze, Catherine Senamaud-Beaufort, D. Sélimoglu-Buet, Eric Solary, Nicolas Gilbert, Thérèse Commes
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Despite development of several effective therapies for multiple myeloma (MM), the prognosis of patients with partial deletion of chromosome 17 (del(17p)) and TP53 aberrations remains poor. By applying comprehensive multi-omics profiling analyses (whole exome and transcriptome sequencing plus proteomics) and functional ex vivo drug screening to samples from 167 patients with MM, we uncovered novel therapeutic vulnerabilities specific to TP53 mutated MM. Our findings revealed a distinct sensitivity profile to a range of inhibitors (mitotic, topoisomerase, HDAC, HSP90, IGF1R and PI3K/AKT/mTOR inhibitors) irrespective of 17p deletion status. Conversely, no increase in sensitivity was observed for monoallelic TP53 (del(17p) with WT TP53) when compared to other samples, highlighting the remaining unmet clinical need. Notably, plicamycin, an RNA synthesis inhibitor linked to modulation of chromatin structure and increased transcription, emerged as particularly efficacious for TP53 mutated MM. The increased sensitivity correlated with higher protein expression of the drug targets: HDAC2, HSP90AA1 and multiple ribosomal subunits. Additionally, we observed increased RNA expression of G2M checkpoint, E2F targets and mTORC1 signaling in our cohort and the MMRF-CoMMpass (NCT01454297) study in TP53 mutated MM. Harmonization of multi-omics data with ex vivo drug screening results revealed that TP53 mutated MM is functionally distinct from MM with monoallelic TP53, and demonstrate that MM with mutated TP53, with and without del(17p), may be targetable by approved drugs, and further indicates the need for regular monitoring by sequencing to identify these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a52d0d33d267850e10246d4013cb5921df1165" target='_blank'>
              Multi-omic and functional screening reveal targetable vulnerabilities in TP53 mutated multiple myeloma
              </a>
            </td>
          <td>
            Dimitrios Tsallos, Nemo Ikonen, Juho J. Mienen, M. M. Majumder, S. Eldfors, I. Västrik, A. Parsons, M. Suvela, Katie Dunphy, Paul Dowling, D. Bazou, Peter O’Gorman, J. Lievonen, R. Silvennoinen, Pekka Anla, C. Heckman
          </td>
          <td>2024-08-23</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) is the second-leading cause of cancer mortality in U.S. men, with African American (AA) men showing higher incidence and mortality compared to European American men. In addition, AA men are more likely to be diagnosed with more aggressive PCa resulting in reduction of treatment options, especially as tumors develop therapy resistance. Thus, understanding the mechanisms underlying PCa therapy resistance is crucial for developing new therapeutic options. The upregulation of the Lens Epithelium Derived Growth Factor p75 (LEDGF/p75) by glucocorticoid receptor (GR) is a novel mechanism of PCa therapy resistance recently identified by our group. GR upregulation in PCa bypasses the blockade of androgen receptor by inhibitors such as enzalutamide and abiraterone. LEDGF/p75 is an oncogenic transcription co-activator that promotes DNA damage repair and cancer cell survival against environmental stressors including chemotherapeutic drugs such as docetaxel (DTX). An emerging concern in PCa treatment is therapy cross-resistance, in which the molecular mechanisms of resistance to a particular drug confer resistance to another drug. Our recent studies showed that GR upregulates and interacts with LEDGF/p75 forming a transcriptional complex that includes other oncoproteins such as Menin, JPO2, c-Myc and HRP2, and promotes DTX resistance (DTX-R) in PCa cells. In addition, we reported that both GR and LEDGF/p75 are upregulated in enzalutamide resistant (ENZ-R) PCa cells. We hypothesized that the GR- LEDGF/p75 transcriptional network plays a role in PCa therapy cross-resistance to both ENZ and DTX and is a promising therapeutic target. Cell viability assays revealed that ENZ-R in PCa cells confers cross-resistance to DTX. Western blot analysis revealed upregulation of members of the GR-LEDGF/p75 network such as GR, LEDGF/p75, Menin and c-Myc in ENZ-R PCa cells, consistent with similar observations in DTX-R PCa cells. Upon knockdown of GR and LEDGF/p75 in DTX-R PCa cells, we identified by RNA-seq analysis overlapping DEGs associated with both ENZ-R and DTX-R, suggesting that the GR-LEDGF/p75 transcriptional network may contribute to therapy cross-resistance. Consistent with this, mining of publicly available RNA-seq databases identified 46 overlapping differentially expressed genes (DEGs) between ENZ-R and DTX-R PCa cells. Pathway analysis of DEGs regulated by both GR and LEDGF/p75 revealed significant enrichment of Apoptosis, p53, and DNA repair pathways, which are associated with therapy resistance. Ongoing studies are evaluating the anti-cancer properties and synergistic/additive potential of various inhibitors of the GR-LEDGF/p75 transcriptional network in cellular models of ENZ-R and DTX-R. These studies provide novel mechanistic insights into PCa therapy cross-resistance and may lead to the development of novel therapeutic strategies to reduce PCa mortality disparities.
 Citation Format: Pedro T. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, Carlos A. Casiano. The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C096.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0c6225006bf01cdfdfbdc7e8efd4e751393fc1" target='_blank'>
              Abstract C096: The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer
              </a>
            </td>
          <td>
            P. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, C. Casiano
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is an aggressive malignancy. The inherent resistance of GBM to radiotherapy poses great challenges for clinical treatment.


OBJECTIVES
The primary objective of this study is to explore the molecular mechanisms of radiotherapy resistance in GBM and identify the key influencing factors that contribute to this phenomenon.


METHODS
The single-cell RNA sequencing (scRNA-seq) data of GBM were downloaded from the Gene Expression Omnibus (GEO) database. Cells were clustered using the Seurat R package, and the clusters were annotated using the CellMarker database. Pseudotime analysis was conducted using Monocle2. Marker scores were calculated based on the RNA-seq data of GBM from the UCSC database, and the enrichment of Hallmark gene sets was measured with the AUCell package. Furthermore, the most frequently mutated genes were identified using the simple nucleotide variation data from The Cancer Genome Atlas (TCGA) applying the maftools package.


RESULTS
This study identified two oligodendrocyte subsets (ODC3 and ODC4) as radiotherapy-resistant groups in GBM. Enrichment and Pseudotime analysis revealed that the inflammatory response and immune activation pathways were enriched in ODC3, while the cell division and interferon response pathways were enriched in ODC4. The enrichment scores of hallmark gene sets further confirmed that ODC3 and ODC4 subpopulations developed radiotherapy resistance via distinct molecular mechanisms. Analysis of gene mutation frequencies showed that TP53 exhibited the most significant change in mutation frequency, indicating that it was an important risk factor involved in radiotherapy resistance in GBM.


CONCLUSION
We identified two ODC subpopulations that exhibited resistance to radiotherapy, providing a new perspective and potential targets for personalized treatment strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593a813498357e1fd5d3f302a2e0d7d52d1643a" target='_blank'>
              Analysis of the Mechanism Underlying Radiotherapy Resistance Caused by Oligodendroglia Cells in Glioblastoma by Applying the Single-cell RNA Sequencing Technology.
              </a>
            </td>
          <td>
            Qinghua Yuan, Weida Gao, Mian Guo, Bo Liu
          </td>
          <td>2024-09-12</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Relapse remains a determinant of treatment failure and contributes significantly to mortality in acute myeloid leukemia (AML) patients. Despite efforts to understand AML progression and relapse mechanisms, findings on acquired gene mutations in relapse vary, suggesting inherent genetic heterogeneity and emphasizing the role of epigenetic modifications. We conducted a multi-omic analysis using Omni-C, ATAC-seq, and RNA-seq on longitudinal samples from two adult AML patients at diagnosis and relapse. Herein, we characterized genetic and epigenetic changes in AML progression to elucidate the underlying mechanisms of relapse. Differential interaction analysis showed significant 3D chromatin landscape reorganization between relapse and diagnosis samples. Comparing global open chromatin profiles revealed that relapse samples had significantly fewer accessible chromatin regions than diagnosis samples. In addition, we discovered that relapse-related upregulation was achieved either by forming new active enhancer contacts or by losing interactions with poised enhancers/potential silencers. Altogether, our study highlights the impact of genetic and epigenetic changes on AML progression, underlining the importance of multi-omic approaches in understanding disease relapse mechanisms and guiding potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce349ab8147752454b7d482ecbf7663d5857bab7" target='_blank'>
              Multi-omic analysis of longitudinal acute myeloid leukemia patient samples reveals potential prognostic markers linked to disease progression
              </a>
            </td>
          <td>
            Nisar Ahmed, Irene Cavattoni, William Villiers, Chiara Cugno, S. Deola, Borbala Mifsud
          </td>
          <td>2024-09-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This review article explores the fundamental role of receptor targeting in overcoming drug resistance in cancer therapy, an area of critical concern given the persistently high rates of cancer morbidity and mortality globally. We highlight how receptor biology intersects with the development of therapeutic resistance with a specific focus on anti-angiogenic agents, immune checkpoint inhibitors, and monoclonal antibodies, which directly or indirectly influence receptor pathways. We also explore how other receptor tyrosine kinases can initially suppress tumor growth, yet often lead to resistance, underscoring the need for novel combinatorial approaches that incorporate advanced receptor modulation techniques. Further, the review delves into the mechanisms by which modulation of the tumor microenvironment and immune system via receptor pathways can overcome resistance to traditional immunotherapies. Additionally, emerging technologies in receptor-targeted nanomedicine are also highlighted, showcasing their potential to revolutionize drug delivery and improve therapeutic outcomes by targeting specific receptor interactions. Ultimately, this review calls for a deeper understanding of receptor dynamics to develop more precise interventions, including insights from various healthcare settings that can prevent or circumvent drug resistance, thus enhancing patient outcomes in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c1e2feed067b3c87cc3d2349034c65489de2b9" target='_blank'>
              Receptor-Based Strategies for Overcoming Resistance in Cancer Therapy
              </a>
            </td>
          <td>
            Naresh Sah, Abdul Althaf Shaik, Ganesh Acharya, Manikantha Dunna, Ashok Silwal, Sejal Sharma, Sabiha Khan, Sounak Bagchi
          </td>
          <td>2024-09-24</td>
          <td>Receptors</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Myelodysplastic syndromes (MDS) are myeloid malignancies with heterogeneous genotypes and phenotypes, characterized by ineffective haematopoiesis and a high risk of progression towards acute myeloid leukaemia (AML). Prognosis for patients treated with hypomethylating agents (HMAs), as is azacytidine, the main drug used as frontline therapy for MDS is mostly based on cytogenetics and next generation sequencing (NGS) of the initial myeloid clone. Although the critical influence of the epigenetic landscape upon cancer cells survival and development as well on tumour environment establishment is currently recognized and approached within current clinical practice in MDS, the heterogenous response of the patients to epigenetic therapy is suggesting a more complex mechanism of action, as is the case of RNA methylation. In this sense, the newly emerging field of epitranscriptomics could provide a more comprehensive perspective upon the modulation of gene expression in malignancies, as is the proof‐of‐concept of MDS. We initially did RNA methylation sequencing on MDS patients (n = 6) treated with azacytidine and compared responders with non‐responders. Afterwards, the genes identified were assessed in vitro and afterwards validated on a larger cohort of MDS patients treated with azacytidine (n = 58). Our data show that a more accurate prognosis could be based on analysing the methylome and thus we used methylation sequencing to differentially split high‐grade MDS patients with identical demographical and cytogenetic features, between azacytidine responders and non‐responders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302c378a460869cbc4ab47244a1bece112c425c0" target='_blank'>
              RNA methylation sequencing shows different gene expression signatures for response to azacytidine therapy in high‐grade myelodysplastic syndromes
              </a>
            </td>
          <td>
            Diana Gulei, Vlad Moisoiu, David Kegyes, Rareș Drula, Sabina Iluta, A. Țigu, Mădălina Nistor, Ciprian Jitaru, Anamaria Bancos, Petra Rotariu, Corina Popovici, D. Dima, R. Tomai, Ioana Rus, Cătălin Constantinescu, R. Munteanu, Diana Cenariu, U. Sezerman, M. Zdrenghea, Jaroslav Čermák, Hermann Einsele, Gabriel Ghiaur, C. Tomuleasa
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24869ef91b9d9f698a0aa1c1c25527b7385dd280" target='_blank'>
              Deciphering resistance mechanisms in cancer: final report of MATCH-R study with a focus on molecular drivers and PDX development
              </a>
            </td>
          <td>
            Damien Vasseur, L. Bigot, Kristi Beshiri, Juan-David Florez-Arango, F. Facchinetti, A. Hollebecque, L. Tselikas, M. Aldea, F. Blanc-Durand, A. Gazzah, D. Planchard, Ludovic Lacroix, N. Pata-Merci, Catline Nobre, Alice Da Silva, C. Nicotra, M. Ngo-Camus, F. Braye, Sergey Nikolaev, S. Michiels, Gerome Jules-Clement, K. Olaussen, Fabrice André, J. Scoazec, F. Barlesi, Santiago Ponce, Jean-Charles Soria, Benjamin Besse, Yohann Loriot, L. Friboulet
          </td>
          <td>2024-10-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immunoglobulin (IG) gene analysis provides fundamental insight into B-cell receptor structure and function. In B-cell tumors, it can inform the cell of origin and clinical outcomes. Its clinical value has been established in the two types of chronic lymphocytic leukemia with unmutated or mutated IGHV genes and is emerging in other B-cell tumors. The traditional PCR-based techniques, which are labor-intensive, rely on the attainment of either a dominant sequence or a small number of subclonal sequences and do not allow automated matching with the clonal phenotypic features. Extraction of the expressed tumor IG transcripts using high-throughput RNA sequencing (RNA-seq) can be faster and allow the collection of multiple sequences matched with the transcriptome profile. Analytical tools are regularly sought to increase the accuracy, depth, and speed of acquisition of the full IGV-(IGD)-IGJ-IGC sequences and combine the IG characteristics with other RNA-seq data. We provide here a user-friendly protocol for the rapid extraction, identification, and accurate determination of the full (leader to constant region) tumor IG templated and non-templated transcript sequence from RNA-seq. The derived amino acid sequences can be interrogated for their physico-chemical characteristics and, in certain lymphomas, predict tumor glycan types occupying acquired N-glycosylation sites. These features will then be available for association studies with the tumor transcriptome. The resulting information can also help refine diagnosis, prognosis, and potential therapeutic targeting in the most common lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea7468231c76081901be5a9f5bad586bdb3d961" target='_blank'>
              IgSeqR: a protocol for the identification, assembly, and characterization of full-length tumor Immunoglobulin transcripts from unselected RNA sequencing data
              </a>
            </td>
          <td>
            Dean Bryant, B. Sale, G. Chiodin, D. Tatterton, Ben Stevens, Alyssa Adlaon, Erin Snook, James Batchelor, Alberto Orfao, Francesco Forconi
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Genomic instability is a core characteristic of cancer, often stemming from defects in DNA damage response (DDR) or increased replication stress. DDR defects can lead to significant genetic alterations, including changes in gene copy numbers, gene rearrangements, and mutations, which accumulate over time and drive the clonal evolution of cancer cells. However, these vulnerabilities also present opportunities for targeted therapies that exploit DDR deficiencies, potentially improving treatment efficacy and patient outcomes. The development of PARP inhibitors like Olaparib has significantly improved the treatment of cancers with DDR defects (e.g., BRCA1 or BRCA2 mutations) based on synthetic lethality. This achievement has spurred further research into identifying additional therapeutic targets within the DDR pathway. Recent progress includes the development of inhibitors targeting other key DDR components such as DNA-PK, ATM, ATR, Chk1, Chk2, and Wee1 kinases. Current research is focused on optimizing these therapies by developing predictive biomarkers for treatment response, analyzing mechanisms of resistance (both intrinsic and acquired), and exploring the potential for combining DDR-targeted therapies with chemotherapy, radiotherapy, and immunotherapy. This article provides an overview of the latest advancements in targeted anti-tumor therapies based on DDR and their implications for future cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdf72d2a9fa59533e1f8a82e5fc32e8e54f1de8" target='_blank'>
              Advancing cancer therapy: new frontiers in targeting DNA damage response
              </a>
            </td>
          <td>
            Jiekun Qian, Guoliang Liao, Maohui Chen, Ren-Wang Peng, Xin Yan, Jianting Du, Renjie Huang, Maojie Pan, Yuxing Lin, Xian Gong, Guo-bing Xu, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most prevalent malignant brain tumor in children that demonstrates clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), and particularly the 3gsubtype, exhibit high levels of MYC, high rates of metastasis, and have the worst prognosis. Despite treatment with surgery, radiation and chemotherapy, Group 3 medulloblastoma patients are more likely to develop recurrent tumors which are far more aggressive resulting in a 5-year survival rate after recurrence of 12%. Thus, there is an imperative need to examine the underlying mechanisms mediating treatment resistance. Recent evidence has identified cellular plasticity, as a major driver of therapy resistance. To investigate whether epigenetic mediated cell plasticity contributes to Myc-MB therapy response we performed Single Nuclei Multiome ATAC + Gene Expression analysis of paired primary and relapsed tumors. We identified the emergence of novel progenitor populations in resistant tumors with stem-like gene expression networks and altered chromatin accessibility. These emergent popuoations demostrated new enhancer-promoer interactions and exhibited increased metabolic reprogramming with potential for theraputic targeting. Because radiation is one of the key treatment modalities for Myc-MB, we chose to model radiation resistance in further studies. Single-cell RNA sequencing of outgrown in vivo irradiated orthotopic xenograft MB tumors showed an overall upregulation of metabolic pathways with the enhancement of genes that mediate oxidative phosphorylation and glutamate metabolism centering on wild-type IDH1 (wtIDH1), consistent with our patient data. IR-resistant cells showed altered chromatin occupancy of H3K3me3 and H3K27Ac. Disruption of IDH1 activity in IR resistant cells in vitro altered the epigenomic landscape, facilitating reversion to the pre-irradiated state. Interestingly, we observed that pluripotent genes with bivalent regions (H3K4me3 and H3K27me3) were enhanced in radiation-resistant cells and suppressed by IDH1 inhibition. Metabolic mass spectrometry reveled that metabolites from the citric acid cycle and synthesis of unsaturated fatty acids were among the main metabolically elevated molecules in radiation-resistant cells consistnat with the observed epigenomic reprograming. IDH1 inhibition reversed the metabolic reprograming and resensitized MB cells to radiation. In total our data identify a novel mechanism of radiation resistance in medulloblastoma with potential theraputic implications.
 Citation Format: Bethany Veo, Dong Wang, John Desisto, Rajeev Vibhakar. Mechanisms of radiation resistance in medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaed4654cb7c14fb325db4bb243c211a30ba0383" target='_blank'>
              Abstract A071 Mechanisms of radiation resistance in medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. A. Guertin, J. B. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, P. Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract B‐cell acute lymphoblastic leukaemia (B‐ALL) is the most prevalent hematologic malignancy in children and a leading cause of mortality. Managing B‐ALL remains challenging due to its heterogeneity and relapse risk. This study aimed to delineate the molecular features of paediatric B‐ALL and explore the clinical utility of circulating tumour DNA (ctDNA). We analysed 146 patients with paediatric B‐ALL who received systemic chemotherapy. The mutational landscape was profiled in bone marrow (BM) and plasma samples using next‐generation sequencing. Minimal residual disease (MRD) testing on day 19 of induction therapy evaluated treatment efficacy. RNA sequencing identified gene fusions in 61% of patients, including 37 novel fusions. Specifically, the KMT2A‐TRIM29 novel fusion was validated in a boy who responded well to initial therapy but relapsed after 1 year. Elevated mutation counts and maximum variant allele frequency in baseline BM were associated with significantly poorer chemotherapy response (p = 0.0012 and 0.028, respectively). MRD‐negative patients exhibited upregulation of immune‐related pathways (p < 0.01) and increased CD8+ T cell infiltration (p = 0.047). Baseline plasma ctDNA exhibited high mutational concordance with the paired BM samples and was significantly associated with chemotherapy efficacy. These findings suggest that ctDNA and BM profiling offer promising prognostic insights for paediatric B‐ALL management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8abd916dce8e7ef63366da12ae5e6c40a1a52ff" target='_blank'>
              Molecular characterization and biomarker identification in paediatric B‐cell acute lymphoblastic leukaemia
              </a>
            </td>
          <td>
            Yu Du, Xiankai Zhang, M. Sun, Li Yang, F. Long, Shanshan Qi, Linlin Luo, Xiaoyan Lv, Chenxuan Wang, Xiaoying Wu, Liuqing Zhu, Qiuxiang Ou, Hao Xiong
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary This review provides a comprehensive overview of the current knowledge on mutational signatures in colorectal cancer (CRC), highlighting their potential clinical applications. It discusses the challenges and limitations in translating these analyses into clinical practice and proposes strategies to overcome these obstacles. Additionally, it provides insights into future directions and emerging proof-of-concept studies that highlight the translational potential of mutational signature analysis in improving patient care and outcomes in CRC. Abstract A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), a leading cause of cancer mortality in the United States, presents significant treatment challenges due to its late diagnosis and poor prognosis. Despite advances, the five-year survival rates remain dismally low, with only a fraction of patients eligible for potentially curative surgical interventions. This review aims to comprehensively examine the current landscape of targeted therapies in PDAC, focusing on recent developments in precision medicine approaches. We explore various molecular targets, including KRAS mutations, DNA damage repair deficiencies, mismatch repair pathway alterations, and rare genetic fusions. The review discusses emerging therapies, such as PARP inhibitors, immune checkpoint inhibitors, and novel targeted agents, like RET and NTRK inhibitors. We analyze the results of key clinical trials and highlight the potential of these targeted approaches in specific patient subgroups. Recent developments in PDAC research have emphasized precision oncology, facilitated by next-generation sequencing and the identification of genetic and epigenetic alterations. This approach tailors treatments to individual genetic profiles, improving outcomes and reducing side effects. Significant strides have been made in classifying PDAC into various subtypes, enhancing therapeutic precision. The identification of specific mutations in genes like KRAS, along with advancements in targeted therapies, including small molecule inhibitors, offers new hope. Furthermore, emerging therapies targeting DNA repair pathways and immunotherapeutic strategies also show promising results. As research evolves, integrating these targeted therapies with conventional treatments might improve survival rates and quality of life for PDAC patients, underscoring the shift towards a more personalized treatment paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0623bca1b0876fdbb0324db3ab4221bfe60d5eb8" target='_blank'>
              Targeted Therapies in Pancreatic Cancer: A New Era of Precision Medicine
              </a>
            </td>
          <td>
            Bingyu Li, Qiong Zhang, Claire Castaneda, Shelly Cook
          </td>
          <td>2024-09-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance remains a major clinical challenge in the treatment of colorectal cancer (CRC) with conventional chemotherapy. Analyzing changes within tumor cells and tumor microenvironment (TME) after treatment and in metastases is essential to understanding how resistance develops. In this study, we analyzed scRNA-seq data from 56 CRCs including treatment-naïve tumors and tumors treated with standard chemotherapy with the known response status (18 responders and 6 progressors). In our cohort, primary left-sided CRCs were associated with metastatic potential mesenchymal phenotype and with depleted B cells. In the post-treatment CRC, there was a high prevalence of dendritic cells (DC) in the TME in the response group. The DC-derived signature was associated with better survival in a large CRC cohort from the TCGA. In progressors there was an enrichment of pericyte-like fibroblasts, which appeared to be associated with poor survival in a CRC-TCGA cohort. Progressors also showed elevated fractions of exhausted CD8+ T memory cells suggesting a pro-inflammatory TME. In tumor cells of progressors group, we identified specific expression of chemo-protective markers MTRNR2L1 and CDX1; and their co-expression with stemness-related immune-checkpoint CD24. In summary, scRNA-seq provides a valuable information for the discovery of prognostic markers, and reveals distinct features potentially underlying response to chemotherapy or disease progression in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d907795ada63036c087467c94e2007531b414d" target='_blank'>
              Single-cell Transcriptome Profiling of Post-treatment and Treatment-naïve Colorectal Cancer: Insights into Putative Mechanisms of Chemoresistance
              </a>
            </td>
          <td>
            Grigory A. Puzanov, Clémence Astier, Andrey A. Yurchenko, Gerome Jules-Clement, Fabrice André, Aurélien Marabelle, A. Hollebecque, Sergey Nikolaev
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, weisen fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d61b6a296f0f4f8539e1a19d49c6d27ddf36337d" target='_blank'>
              Deciphering the role of sphingolipid metabolism in the immune microenvironment and prognosis of esophageal cancer via single-cell sequencing and bulk data analysis
              </a>
            </td>
          <td>
            Rongzhang He, Jing Tang, Haotian Lai, Tianchi Zhang, Linjuan Du, Siqi Wei, Ping Zhao, Guobin Tang, Jie Liu, Xiufang Luo
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (AML) exhibits distinct characteristics between infants and children, manifested by variations in clinical features, cytogenetic abnormalities, and molecular aberrations. While most studies have focused on the associations between genomic abnormalities, age groups, and disease status, the transcriptional profiles and gene regulatory mechanisms underlying these differences remain underexplored. In this study, through differential gene expression analysis and Weighted Gene Co-expression Network Analysis, I have identified gene groups associated with AML and further categorized them by infant and childhood subgroups. The findings reveal three distinct gene groups with age- and tissue-specific expression patterns. Additionally, I propose a gene regulatory circuitry that elucidates the differences between infant and childhood AML. Several novel markers demonstrating significant expression changes in tumors were identified. Moreover, a comprehensive gene regulatory network for pediatric AML was constructed using differentially expressed transcription factors and protein-gene interaction data. This study highlights new gene regulation mechanisms in pediatric AML and offers potential avenues for developing novel biomarkers and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdcb7517a7ea15fd2b0b9215c835acfed1519a12" target='_blank'>
              Dissecting the transcriptional regulation of infant and childhood acute myeloid leukemia
              </a>
            </td>
          <td>
            Rasoul Godini
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="T cells recognize an immense spectrum of pathogens to initiate immune responses by means of a large repertoire of T cell receptors (TCRs) that arise from somatic rearrangements of variable, diversity and joining gene segments at the TCR loci. These gene segments have emerged from a limited number of ancestral genes through a series of gene duplication events, resulting in a greatly variable number of such genes across different species. Apart from the complete V(D)J gene annotations in the human and mouse reference assemblies, little is known about the structure of TCR loci in other species. Here, we performed a comprehensive comparison of the TCRα and TCRβ gene segment clusters in mice and three of its closely related sister species. We show that the TCRα variable gene cluster is frequently rearranged, leading to deletions and sequence inversions in this region. The resulting complexity of TCR loci severely complicates the assembly of these loci and the annotation of gene segments. By jointly utilizing genomic and transcriptomic data, we show that in Mus musculus castaneus the variable gene cluster at the α locus has undergone a recent major locus contraction, leading to the loss of 74 variable gene segments. Additionally, we validated the expression of functional variable genes, including atypical ones with inverted orientation relative to other such segments. Disentangling the fine-scale structure of TCR loci in different species can provide valuable insights in the evolution and diversity of TCR repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/050dd2ec4e870352e83af91f6ed80d0e6588c1e2" target='_blank'>
              Distinct evolution at TCRα and TCRβ loci in the genus Mus
              </a>
            </td>
          <td>
            Moritz Peters, Volker Soltys, Dingwen Su, Y. F. Chan
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf274ccb3cfdd7406ad686ecfcdad4f5ec5eb3b" target='_blank'>
              Evolutionary dependency of cancer mutations in gene pairs inferred by nonsynonymous-synonymous mutation ratios
              </a>
            </td>
          <td>
            Dong-Jin Han, Sunmin Kim, Seo-Young Lee, Youngbeen Moon, Su Jung Kang, Jinseon Yoo, Hye Young Jeong, Hae Jin Cho, Jeong Yang Jeon, Byeong Chang Sim, Jaehoon Kim, Seungho Lee, Ruibin Xi, Tae-Min Kim
          </td>
          <td>2024-08-19</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Approximately 50% of patients diagnosed with ovarian cancer harbor tumors with mutations in BRCA1, BRCA2, or other genes involved in homologous recombination repair (HR). The presence of homologous recombination deficiency (HRD) is an approved biomarker for poly-ADP-ribose polymerase inhibitors (PARPis) as a maintenance treatment following a positive response to initial platinum-based chemotherapy. Despite this treatment option, the development of resistance to PARPis is common among recurrent disease patients, leading to a poor prognosis. In this study, we conducted a comprehensive analysis using publicly available datasets to elucidate the molecular mechanisms driving PARPi resistance in BRCA1-deficient ovarian cancer. Our findings reveal a central role for the interferon (IFN) pathway in mediating resistance in the context of BRCA1 deficiency. Through integrative bioinformatics approaches, we identified LY6E, an interferon-stimulated gene, as a key mediator of PARPi resistance, with its expression linked to an immunosuppressive tumor microenvironment (TME) encouraging tumor progression and invasion. LY6E amplification correlates with poor prognosis and increased expression of immune-related gene signatures, which is predictive of immunotherapy response. Interestingly, LY6E expression upon PARPi treatment resistance was found to be dependent on BRCA1 status. Gene expression analysis in the Orien/cBioPortal database revealed an association between LY6E and genes involved in DNA repair, such as Rad21 and PUF60, emphasizing the interplay between DNA repair pathways and immune modulation. Moreover, PUF60, Rad21, and LY6E are located on chromosome 8q24, a locus often amplified and associated with the progression of ovarian cancer. Overall, our study provides novel insights into the molecular determinants of PARPi resistance and highlights LY6E as a promising prognostic biomarker in the management of HRD ovarian cancer. Future studies are needed to fully elucidate the molecular mechanisms underlying the role of LY6E in PARPi resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4052305f8854485a3b85613abf68315aff09bfc" target='_blank'>
              Exploring Molecular Drivers of PARPi Resistance in BRCA1-Deficient Ovarian Cancer: The Role of LY6E and Immunomodulation
              </a>
            </td>
          <td>
            T. Petta, Joseph Carlson
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Osteosarcoma is a common type of bone cancer characterized by a poor prognosis due to its metastatic nature. The tumor microenvironment (TME) plays a critical role in tumor metastasis and therapy response. Therefore, our study aims to explore the metastatic mechanism of osteosarcoma, potentially opening new avenues for cancer treatment. Methods In this study, we collected data from the GSE152048, GSE14359, and GSE49003 datasets. Differentially expressed genes (DEGs) were identified in osteosarcoma cases with primary and metastatic features using R software and the limma package. Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses were performed to investigate metastasis-related genes. A protein–protein interaction (PPI) network was established using the STRING database to further analyze these metastasis-associated genes. The abundances of different cell types with a mixed cell population were estimated using the CIBERSORT approach. The scRNA-seq data were analyzed by the Seurat package in R software, and intercellular communications were elucidated using the CellChat R package. Results In this study, 92 DEGs related to metastasis were identified, including 41 upregulated and 51 downregulated genes in both the GSE14359 and GSE49003 datasets. Metastasis-associated pathways were identified, including those involving the cyclin-dependent protein kinase holoenzyme complex, transferase complex, transferring phosphorus-containing groups, SCF ubiquitin ligase complex, and the serine/threonine protein kinase complex. KEGG and PPI network analyses revealed 15 hub genes, including Skp2, KIF20A, CCNF, TROAP, PHB, CKS1B, MCM3, CCNA2, TRIP13, CENPM, Hsp90AB1, JUN, CKS2, TK1, and KIF4A. Skp2 has been known as an E3 ubiquitin ligase involved in osteosarcoma progression. The proportion of CD8+ T cells was found to be higher in metastatic osteosarcoma tissues, and high expression of PHB was associated with a favorable prognosis in osteosarcoma patients. Additionally, 23 cell clusters were classified into eight cell types, including chondrocytes, MSC, T cells, monocytes, tissue stem cells, neurons, endothelial cells, and macrophages. The 15 hub genes were expressed across various cell types, and interactions between different cell types were observed. Conclusion Our study reveals the intricate communication between tumor microenvironment components and tumor metastasis in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff33ba2ff2e5acd8821d77a975c86bee16bbf2ba" target='_blank'>
              Single-cell RNA sequencing reveals the communications between tumor microenvironment components and tumor metastasis in osteosarcoma
              </a>
            </td>
          <td>
            Jiatong Li, Yang Bai, He Zhang, Ting Chen, Guanning Shang
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Neuroblastoma (NBL) is the most common extra-cranial solid tumor with <60% long-term survival in high-risk patients. Approximately 15% of high-risk NBL patients develop progressive disease (PD) during induction chemotherapy. As tumor biopsies are often not obtained at time of PD, identifying underlying genetic/genomic features that contribute to disease progression is challenging. Patient-derived xenografts (PDX) established from NBL cells in bone marrow aspirates or blood samples obtained pre-therapy and at time of PD can be phenotypically and molecularly characterized to potentially identify biomarkers predictive of treatment outcome in high-risk NBL patients. Thirty-five high-risk patient tumor, marrow, or blood samples, 16 at diagnosis (DX) and 19 at PD, were received via Children’s Oncology Group (COG) protocol ANBL00B1 and established as PDXs. PDXs were classified by response to chemotherapy (cyclophosphamide, 30mg/kg + topotecan, 0.6 mg/kg daily for 5 days every 21 days, total of 3 cycles) as non-responders (NR) and responders (R), the latter then divided into partial responders (PR) and complete responders (CR). The gene expression in the PDXs was analyzed by RNA-sequencing, and the expression profiles were analyzed by differential analysis (DEGs) via edgeR, binomial regression model, and student’s t-test to identify the marker genes for NR group. The markers were validated using real-time RT-PCR. PDX cyclo/topo response was: non-response (NR, n=13), partial response (PR, n=9), and complete response (CR, n=13). The event-free survival in all treatment groups was significantly extended relative to their control (P < 0.001). Nine of 13 (69%) of the NR PDXs were established from post-mortem PD (PD-PM) samples while nine of thirteen (69%) of the CR group were from diagnosis (DX) samples, demonstrating the clinical relevance of the PDX models. The RNA-seq data analysis identified 639 differentially expressed genes across the NR versus response groups. A regression model and t-test significance further filtered these genes to define a signature comprised of 21 genes for the NR group (P < 1.8e-12). When assessing only DX PDX models, a set of 26 genes was identified as a signature for the NR group (P < 4.1e-05). WNT signaling and Hedgehog pathways were the two most highly activated in the NR group relative to the R group. Of the 639 differentially expressed genes, we investigated nine genes that showed highly significant differential expression by RT-PCR, and seven of the nine genes tested (P < 0.05). Response of PDXs to cyclophosphamide/topotecan was greater for pretherapy (DX) PDXs than PD-PM PDXs. The gene expression signature of NR PDXs can potentially provide a biomarker to identify patients destined to have suboptimal responses to induction chemotherapy. Genes overexpressed in NR PDXs provide potential molecular targets to reverse drug resistance. In addition, the well-characterized and clinically relevant panel of PDXs will be valuable in preclinical studies of novel therapeutic approaches for high-risk neuroblastoma patients.
 Citation Format: Nighat Noureen, Harry May, In-Hyoung Yang, Kristyn McCoy, Jonas Nance, Diana Ixlamati-Nava, Meredith S Irwin, Michael D. Hogarty, Charles P. Reynolds, Min H. Kang. Response of neuroblastoma patient-derived xenografts to cyclophosphamide + topotecan enables identification of gene expression patterns associated with drug resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/080da9bbd6cc73e1417f72d197d94604906e67f5" target='_blank'>
              Abstract A058 Response of neuroblastoma patient-derived xenografts to cyclophosphamide + topotecan enables identification of gene expression patterns associated with drug resistance
              </a>
            </td>
          <td>
            Nighat Noureen, Harry A. May, In-Hyoung Yang, Kristyn McCoy, Jonas Nance, Diana Ixlamati-Nava, Meredith S Irwin, M. Hogarty, C. P. Reynolds, Min H. Kang
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="DNA rearrangements are thought to arise from two classes of processes. The first class involves DNA breakage and fusion (“cut-and-paste”) without net DNA gain or loss. The second class involves aberrant DNA replication (“copy-and-paste”) and can produce either net DNA gain or loss. We previously demonstrated that the partitioning of chromosomes into aberrant structures of the nucleus, micronuclei or chromosome bridges, can generate cut-and-paste rearrangements by chromosome fragmentation and ligation. Surprisingly, in the progeny clones of single cells that have undergone chromosome bridge breakage, we identified large segmental duplications and short sequence insertions that are commonly attributed to copy-and-paste processes. Here, we demonstrate that both large duplications and short insertions are inherent outcomes of the replication and fusion of unligated DNA ends, a process we term breakage-replication/fusion (B-R/F). We propose that B-R/F provides a unifying explanation for complex rearrangement patterns including chromothripsis and chromoanasynthesis and enables rapid DNA amplification after chromosome fragmentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c002d451b4a62962b702db7317ff655026459317" target='_blank'>
              Chromosome breakage-replication/fusion enables rapid DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, David Pellman
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Background: Immune checkpoint blockades (ICBs) lack clinical efficacy in all-coming pediatric solid tumors. Gene expression-based classification of the tumor immune environment (TiME) can identify a subset of pediatric tumors with a rich immune environment, potentially actionable by ICBs. Methylation reprogramming is a major player in TiME modeling and ICB resistance, its role in pediatric tumors has been little explored. Hypothesis: dual-omic classification, combining DNA-methylation and RNA-sequencing, can identify tumors with epigenetically altered TiME, potentially responsible for ICB resistance. Objective: identify and characterize pediatric solid tumors with epigenetically altered immune phenotypes. Methods: We studied RNA-sequencing and DNA-methylation quantitative data from pediatric extra-cranial solid tumors. We used similarity network fusion (SNF) to individualize immune phenotypes. A preselected customized panel of 1226 immune genes, and their corresponding probes, were used for dual-omics clustering. Differential expression (DE) and methylation (DM) were performed between clusters using volcano3D. Scatter plots visualized the cis-regulation of probe-gene pairs with significant DE-DM. Gene set enrichment analyses (GSEA) were performed with clusterProfiler and gometh packages for DE and DM, respectively. Adjusted p values <0.01 were retained. Results: 184 samples were included. SNF clustering identified 3 phenotypes regrouping 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 was characterized by a low expression of immune genes (cold phenotype), cl2 by overexpression in immune genes (hot phenotype), and cl3 by global hypermethylation (epigenetically altered phenotype). Both cl2 and cl3 overexpressed genes of immune checkpoints (CD274, PDCD1) and T-cell activator chemokines (CXCL9, CXCL10), central for ICB sensitivity. However, only cl2 overexpressed major histocompatibility complex (MHC) class I-II genes and their regulators (CIITA, TAP2), essential for antigen (Ag)-presenting machinery and immune recognition. Cl2 and 3 were also enriched for T-, B-cell and pro-inflammatory interferon gamma signaling pathways, known as ICB sensitivity biomarkers. Only cl2 was enriched in Ag processing and presentation pathway, confirming prior observation. DE-DM correlation showed that DNA-methylation programming induced overexpression of most immune genes (78%) in cl2, including CIITA, CXCR3 and MHC genes, when most immune genes (63%) were repressed by methylation in cl3. Methylation-based GSEA confirmed the epigenetic regulation of T- and B-cell signaling in cl2 and cl3, and for Ag presentation in cl2 only. Conclusion We demonstrated that DNA-methylation can reshape the TiME of pediatric solid tumors. We identified a subset of tumors with epigenetically altered immune phenotype characterized by inflamed immune environment but altered for Ag presentation by methylation reprogramming. Future studies should investigate methylation modulators to reverse these mechanisms and enable ICB sensitivity.
 Citation Format: Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-Cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, Thai Hoa Tran, Sonia Cellot, Daniel Sinnett, Arnaud Droit, Raoul Santiago. Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a9f318a4bf2de77816f3008a669473ec30397b" target='_blank'>
              Abstract B057: Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, T. Tran, Sonia Cellot, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef86e2f41cfc310fa982c0b28be1cd0a1d6909de" target='_blank'>
              A protracted war against cancer drug resistance
              </a>
            </td>
          <td>
            Yuan Tian, Xiaowei Wang, Cong Wu, Jiaming Qiao, Hai Jin, Huafei Li
          </td>
          <td>2024-09-28</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a593c69280acd8e1ddd89d1994ea815f977cea" target='_blank'>
              Chromosomal instability: a key driver in glioma pathogenesis and progression
              </a>
            </td>
          <td>
            Adele Mazzoleni, W. A. Awuah, Vivek Sanker, H. Bharadwaj, Nicholas Aderinto, J. K. Tan, H. Huang, Jeisun Poornaselvan, Muhammad Hamza Shah, O. Atallah, Aya Tawfik, Mohamed Elmanzalawi, Sama Hesham Ghozlan, T. Abdul-Rahman, Jeremiah Adepoju Moyondafoluwa, Athanasios Alexiou, Marios Papadakis
          </td>
          <td>2024-09-04</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Background: Hispanic American (HA) men with prostate cancer (PC) are more likely to have more aggressive and advanced diseases and greater mortality rates compared to non-Hispanic white (NHW) men. Resistance to the current anticancer therapy is the major hurdle that leads to poor clinical outcomes. Hedgehog, Notch, and Wnt signaling pathways play a crucial role in development, progression, and resistance to the various anticancer therapies. Here, we characterized the enrichment of Hedgehog, Notch, and Wnt signaling pathways and molecular tumors differences between HA and NHW patients. Methods: Our data consist of 88 PC tumors samples (HA=34, NHW=54) obtained from treatment-naive metastatic hormone-sensitive prostate cancer (mHSPC) that were analyzed by next-generation sequencing (592, NextSeq; Whole Exome Sequencing, NovaSeq) and Whole Transcriptome Sequencing (WTS; NovaSeq) (Caris Life Sciences, Phoenix, AZ). Pathway enrichment was determined by Gene Set Enrichment Analysis (GSEA, Broad Institute) by measurable TPM (transcripts per kilobase million). The medical records were reviewed in a deidentified fashion for clinical features. Statistical significance was determined using chi-square and Mann-Whitney U (p<0.05). Results: HA PC had enrichment of Hedgehog signaling (NES: 1.45, FDR=0.04), Notch signaling (NES: 1.41, FDR=0.07), and Wnt/β-catenin signaling (NES: 1.38, FDR=0.11) pathways compared to NHW tumors. HA tumors had higher expression of genes (HES1, Notch1, WNT1, WNT2; FC:1.1-2.5) associated with crosstalk between Hedgehog, Notch, and Wnt/β- catenin signaling pathways (all p<0.05). Moreover, HA tumors had higher expression of stemness (ALDH1A2, ALDH1A1, PROM1; FC: 1.7-3.0), drug efflux (ABCC1, ABCB1; FC: 1.1- 1.5), epithelial to mesenchymal transition (EMT) (TWIST1, MMP2; FC: 1.1-1.5), cell cycle (CCND2; FC: 1.2), inhibition of apoptosis (BIRC2, XIAP; FC: 1.1) and epigenetic (EP300, TET1, TET2; FC: 1.3-1.4) related genes compared to NHW tumors (all p<0.05). HA PC had higher frequency of Hedgehog and Wnt/β-catenin associated SPOP (16% vs 6.6%, p=0.2) and CTNNB1 (12% vs 2.2%, p=0.1) mutation compared to NHW PC. Conclusion: These results demonstrate that tumors from HA patients had enrichment of oncogenic Hedgehog, Notch, and Wnt/β-catenin signaling pathways, higher expression of stemness, drug efflux, EMT, cell cycle, inhibition of apoptosis and epigenetic-related genes, and mutations directly involved in activating the Hedgehog and Wnt/β-catenin signaling pathways. Together, these findings suggest that tumors from HA patients are enriched for gene signatures that promote cancer progression and therapy resistance. Additional research on collaboration and crosstalk between Hedgehog, Notch, and Wnt/β-catenin signaling pathways may help in designing better therapeutic strategies for HA men with mHSPC.
 Citation Format: Alejandro Recio-Boiles, Sachin Kumar Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, Milan Radovich, Elisabeth I. Heath, Rana McKay, Juan Chipollini, Cynthia K. Miranti. Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C100.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084e712d5ece7637137ae910e70858f7d21562a6" target='_blank'>
              Abstract C100: Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men
              </a>
            </td>
          <td>
            Alejandro Recio-Boiles, S. Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, M. Radovich, Elisabeth I. Heath, Rana Mckay, Juan Chipollini, Cynthia K. Miranti
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation is a relatively stable epigenetic mark with important roles in development and disease.1 Since cell-to-cell variation in epigenetic programming can reflect important differences in cell state and fate, it is clear that single-cell methods are essential to understanding this key epigenetic mark in heterogeneous tissues. Existing single-cell whole-genome bisulfite sequencing (scWGBS) methods2 have significant shortcomings, including very low CpG coverage3–7 or inefficient library generation requiring extremely deep sequencing.8 These methods offer limited insight into focal regulatory regions and generally preclude direct cell-to-cell comparisons. To address these shortcomings, we have developed an improved method, scDEEP-mC (single-cell Deep and Efficient Epigenomic Profiling of methyl-C). We show that high-coverage promoter methylation profiling by scDEEP-mC can identify multiple cell types, while allele-resolved methylation calls allow assessment of X-inactivation state in single cells and identification of transcription factor binding sites (TFBS) enriched for hemi-methylation. We also use scDEEP-mC to profile single-cell copy number alterations, identify actively replicating cells, and track DNA methylation dynamics during and after DNA replication. The high coverage of scDEEP-mC creates an exceptional opportunity to explore DNA methylation biology in individual cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce6e8d494aa190daf1503d002a071a38ed1668a3" target='_blank'>
              High-coverage allele-resolved single-cell DNA methylation profiling by scDEEP-mC reveals cell lineage, X-inactivation state, and replication dynamics
              </a>
            </td>
          <td>
            Nathan J. Spix, Walid Abi Habib, Zhouwei Zhang, Emily Eugster, Hsiao-yun Milliron, David Sokol, KwangHo Lee, P. Nolte, Jamie L. Endicott, Kelly F. Krzyzanowski, T. Hinoue, Jacob Morrison, Benjamin K. Johnson, Wanding Zhou, Hui Shen, Peter W. Laird
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Intragenomic mutation rates can vary dramatically due to transcription-associated mutagenesis or transcription-coupled repair, which vary based on local epigenomic modifications that are non-uniformly distributed across genomes. One feature associated with decreased mutation is higher expression level, which depends on environmental cues. To understand the magnitude of expression-dependent mutation rate variation, we perturbed expression through a heat treatment in Arabidopsis thaliana. We quantified gene expression to identify differentially expressed genes, which we then targeted for mutation detection using Duplex Sequencing. This approach provided a highly accurate measurement of the frequency of rare somatic mutations in vegetative plant tissues, which has been a recent source of uncertainty. Somatic mutations in plants may be useful for understanding drivers of DNA damage and repair in the germline since plants experience late germline segregation and both somatic and germline cells share common repair machinery. We included mutant lines lacking mismatch repair (MMR) and base excision repair (BER) capabilities to understand how repair mechanisms may drive biased mutation accumulation. We found wild type (WT) and BER mutant mutation frequencies to be very low (mean variant frequency 1.8´10-8 and 2.6´10-8, respectively), while MMR mutant frequencies were significantly elevated (1.13´10-6). Interestingly, in the MMR mutant lines there was no difference in the somatic mutation frequencies between temperature treatments or between highly vs. lowly expressed genes. The extremely low somatic variant frequencies in WT plants indicate that larger datasets will be needed to address fundamental evolutionary questions about whether environmental change leads to gene-specific changes in mutation rate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c6a9ca240c4acb07b722d7f1e2c985281e1839" target='_blank'>
              Exploring the relationship between gene expression and low frequency somatic mutations in Arabidopsis with Duplex Sequencing.
              </a>
            </td>
          <td>
            Gus Waneka, Braden Pate, J. G. Monroe, Daniel B. Sloan
          </td>
          <td>2024-10-04</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db9094952a2d4596bce488ed054cdce260574fd" target='_blank'>
              Integrative genomic analyses identify neuroblastoma risk genes involved in neuronal differentiation.
              </a>
            </td>
          <td>
            Matilde Tirelli, Ferdinando Bonfiglio, Sueva Cantalupo, Annalaura Montella, M. Avitabile, Teresa Maiorino, S. Diskin, Achille Iolascon, Mario Capasso
          </td>
          <td>2024-08-27</td>
          <td>Human genetics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5edea2d9a83527bbebc5d48f3ba44b53a9b424d3" target='_blank'>
              Single-cell profiling uncovers proliferative cells as key determinants of survival outcomes in lower-grade glioma patients
              </a>
            </td>
          <td>
            Jian-Bing Peng, Qing Zhang, Xiaofeng Zhu, Zhu Yan, Meng Zhu
          </td>
          <td>2024-09-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T-cells and bispecific antibodies (BsAb) targeting B-cell maturation antigen (BCMA) have significantly advanced the treatment of relapsed and refractory multiple myeloma (MM). Resistance to BCMA-targeting therapies, nonetheless, remains a significant challenge. BCMA shedding by gamma-secretase is a known resistance mechanism, and preclinical studies suggest that inhibition may improve anti-BCMA therapy. Leveraging a phase I clinical trial of the gamma-secretase inhibitor (GSI), crenigacestat, with anti-BCMA CAR T-cells (FCARH143), we utilized single-nuclei RNA sequencing and Assay for Transposase-Accessible Chromatin (ATAC) sequencing to characterize the effects of GSI on the tumor microenvironment. The most significant impacts of GSI involved effects on monocytes, which are known to promote tumor growth. In addition to observing a reduction in the frequency of non-classical monocytes, we also detected significant changes in gene expression, chromatin accessibility, and inferred cell-cell interactions following exposure to GSI. Although many genes with altered expression are associated with gamma-secretase-dependent signaling, such as Notch, other pathways were affected, indicating GSI has far-reaching effects. Finally, we detected monoallelic deletion of the BCMA locus in some patients with prior exposure to anti-BCMA therapy, which significantly correlated with reduced progression-free survival (median PFS 57 days versus 861 days). GSIs are being explored in combination with the full spectrum of BCMA targeting agents, and our results reveal widespread effects of GSI on both tumor and immune cell populations, providing insight into mechanisms for enhancing BCMA-directed therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0270c45ce4ccff6ee8256dcb44e77e0a31404f58" target='_blank'>
              Single-cell analysis of the multiple myeloma microenvironment after gamma-secretase inhibition and CAR T-cell therapy.
              </a>
            </td>
          <td>
            D. Coffey, Pinar Ataca Atilla, Erden Atilla, O. Landgren, Andrew J Cowan, Sylvain Simon, Margot Pont, Melissa L. Comstock, Geoffrey R. Hill, Stanley R. Riddell, Damian J Green
          </td>
          <td>2024-10-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Patients with triple-negative breast cancer (TNBC) have limited targeted therapeutic options. PARP inhibitors (PARPi) have demonstrated an important role for BRCA-mutant patients with early TNBC. Combination approaches with PARPi can broaden the use of PARPi to a larger cohort of TNBC patients. We selected six genes from our previously identified 63-gene signature that was associated with PARPi response. siFEN1 increased cells in G2/M arrest, DNA damage and particularly apoptosis. Targeting FEN1 with a chemical inhibitor enhanced the efficacy of PARPi in 7/10 cell lines, and synergy was demonstrated mainly in PARPi-resistant TNBC cell lines. A BRCA2-mutant cell line with acquired resistance to olaparib (HCC1395-OlaR) was strongly synergistic, with a combination index value of 0.20. The combination of PARPi and FEN1 inhibition also showed synergy in a PARPi-resistant xenograft-derived organoid model. Two mechanisms which explain the underlying efficacy are rapid progression in DNA replication fork speed and enhancement of DNA damage. The combination induced the highest fork speed (47% difference in comparison to control, P<0.0001) when FEN1 inhibition and PARPi equally increased fork speed individually in a cell line with a pre-existing increase in replication stress. The combination also increased DNA damage at lower drug concentrations, driving response in most of the synergistic cell lines. Gene expression analysis suggested that the sensitizing role of FEN1 inhibition in PARPi-resistant cell lines may be due to downregulation of pathways including mismatch repair. Therefore, targeting FEN1 shows great therapeutic potential as a targeted combination approach, particularly in the context of PARPi-resistant TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51da609111db5e5375ca86bde9a72eef75e23b2b" target='_blank'>
              Targeting FEN1 to enhance efficacy of PARP inhibition in triple-negative breast cancer
              </a>
            </td>
          <td>
            Mallory I. Frederick, Elicia Fyle, Anna Clouvel, Djihane Abdesselam, Saima Hassan
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 METTL1 is an RNA methyltransferase which catalyzes N7-methylguanosine (m7G) methylation on various RNAs, particularly on tRNAs. METTL1 has been reported to be amplified and/or overexpressed and high expression correlates with poor patient prognosis in different types of adult cancers. Our ZERO Childhood Cancer Program focuses on identifying and validating potential new drug targets and developing novel therapies for clinical use. ZERO has sequenced the whole genome and transcriptome in over 500 high-risk pediatric tumors and found that METTL1 gene copy number and gene expression are significantly higher in a subset of high-grade glioma, neuroblastoma, diffuse midline glioma and fusion positive rhabdomyosarcoma. Analysis of the publicly available SEQC microarray dataset showed that high METTL1 expression correlates with poor outcomes in MYCN amplified neuroblastoma patients. We validated the functional roles of METTL1 gene amplification and overexpression in a few pediatric cancer cell line models using the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) Cas13d knockdown system, in dual lentiviral constructs. We found that knocking down METTL1 significantly decreased METTL1 amplified neuroblastoma and rhabdomyosarcoma cell viability and colony formation but not in METTL1 non-amplified neuroblastoma and rhabdomyosarcoma cells. Gene Set Enrichment Analysis from RNA sequencing data revealed that knocking down METTL1 resulted in a decrease in Myc target gene expression, and an increased in apoptosis, interferon alpha response and p53 pathway hallmarks. Western blot analysis confirmed that knocking down METTL1 led to a decrease in the MCL-1 antiapoptotic protein expression and an increase in the proapoptotic protein expression such as cleaved PARP and caspase 3. Taken together, our data demonstrate that METTL1 is a potential therapeutic target in METTL1 amplified pediatric cancers.
 Citation Format: Pei Y. Liu, Steven He, Chelsea Mayoh, Alice Salib, Antoine De Weck, Paul G. Ekert. Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64085373809830d692bd2bd083b18380125ea0bf" target='_blank'>
              Abstract A012 Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d
              </a>
            </td>
          <td>
            Pei Y. Liu, Steven He, C. Mayoh, A. Salib, Antoine de Weck, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The role of Erythroid cells in immune regulation and immunosuppression is one of the emerging topics in modern immunology that still requires further clarification as Erythroid cells from different tissues and different species express different immunoregulatory molecules. In this study, we performed a thorough investigation of human bone marrow Erythroid cells from adult healthy donors and adult acute lymphoblastic leukemia patients using the state-of-the-art single-cell targeted proteomics and transcriptomics via BD Rhapsody and cancer-related gene copy number variation analysis via NanoString Sprint Profiler. We found that human bone marrow Erythroid cells express the ARG1, LGALS1, LGALS3, LGALS9, and C10orf54 (VISTA) immunosuppressive genes, CXCL5, CXCL8, and VEGFA cytokine genes, as well as the genes involved in antimicrobial immunity and MHC Class II antigen presentation. We also found that ARG1 gene expression was restricted to the single erythroid cell cluster that we termed ARG1-positive Orthochromatic erythroblasts and that late Erythroid cells lose S100A9 and gain MZB1 gene expression in case of acute lymphoblastic leukemia. These findings show that steady-state erythropoiesis bone marrow Erythroid cells express myeloid signature genes even without any transdifferentiating stimulus like cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b037304bfa3fec87e38c7d6b632f2e503c9b8fe" target='_blank'>
              Single-cell multi-omics reveal stage of differentiation and trajectory-dependent immunity-related gene expression patterns in human erythroid cells
              </a>
            </td>
          <td>
            R. Perik-Zavodskii, Olga Perik-Zavodskaia, S. Alrhmoun, M. Volynets, J. Shevchenko, K. Nazarov, Vera Denisova, S. Sennikov
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A R Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ec02f56417ae90246c83ab25ae98a1c2c7e30" target='_blank'>
              Targeting treatment resistance: unveiling the potential of RNA methylation regulators and TG-101,209 in pan-cancer neoadjuvant therapy
              </a>
            </td>
          <td>
            Yaoyao Zhou, Ziyun Liu, Cheng Gong, Jie Zhang, Jing Zhao, Xia Zhang, Xiangyu Liu, Bin Li, Rui Li, Zhenyu Shi, Yongjie Xie, Li Bao
          </td>
          <td>2024-08-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Richter's transformation (RT) is defined as the evolution of chronic lymphocytic leukemia (CLL) or small lymphocytic lymphoma (SLL) into an aggressive lymphoma, most commonly diffuse large B-cell lymphoma. This complication is rare and aggressive, with poor prognosis and dismal survival. Clonal relationship with the underlying CLL/SLL, observed in ∼80% of cases, represents one of the main factors affecting prognosis. Treatment has been historically based on chemoimmunotherapy, but frequent mutations in genes involved in cell survival and proliferation-such as TP53, NOTCH1, MYC, CDKN2A-confer resistance to standard treatments. During the last years, advances in the knowledge of the biological mechanisms underlying RT allowed to identify genetic and molecular lesions that can potentially be targeted by novel selective agents. Pathway and checkpoint inhibitors, bispecific antibodies and CAR T-cell therapy are currently under investigation and represent promising treatment options. This review summarizes current biological evidence and available data on novel therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a1bac15f3b796b2ef346e32ef1ee6b7161f4b12" target='_blank'>
              Advances in the understanding of molecular genetics and therapy of Richter transformation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            M. Deodato, A. Frustaci, Arianna Zappaterra, Alberto Rapella, C. Gambacorti-Passerini, R. Cairoli, Marco Montillo, Alessandra Tedeschi
          </td>
          <td>2024-09-02</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d35e0bab7dec79077bc3332fba4617308d5113" target='_blank'>
              Comprehensive evaluation of immunological attributes and immunotherapy responses of positive T cell function regulators in colorectal cancer
              </a>
            </td>
          <td>
            Ke Pu, Jingyuan Gao, Yang Feng, Jian Hu, Shunli Tang, Guodong Yang, Chuan Xu
          </td>
          <td>2024-10-01</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The fusion expression of DNA replication-related proteins with nucleotide deaminase enzymes promotes random mutations in bacterial genomes, thereby increasing genetic diversity among population. Most previous studies have focused on cytosine deaminase, which produces only C→T mutations, significantly limiting the variety of mutation types. In this study, we developed a fusion expression system by combining DnaG (RNA primase) with adenine deaminase TadA-8e (DnaG-TadA) in Escherichia coli, which is capable of rapidly introducing A→G mutations into the E. coli genome, resulting in a 664-fold increase in terms of mutation rate. Additionally, we engineered a dual-functional TadA variant, TadAD, and then fused it with DnaG. This construct introduced both C→T and A→G mutations into the E. coli genome, with the mutation rate further increased by 370-fold upon co-expression with an uracil glycosylase inhibitor (DnaG-TadAD-UGI). We applied DnaG-TadA and DnaG-TadAD-UGI systems to the adaptive laboratory evolution for Cd2+ and kanamycin resistance, achieving an 8.0 mM Cd2+ and 200 μg/mL kanamycin tolerance within just 17 days and 132 hours, respectively. Compared to conventional evolution methods, the final tolerance levels were increased by 320% and 266%, respectively. Our work offers a novel strategy for random mutagenesis in E. coli and potentially other prokaryotic species. For TOC only">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dc3e641872269a76809a2a9f58812a1c2451c2" target='_blank'>
              Genome-wide A→G and C→T Mutations Induced by Functional TadA Variants in Escherichia coli
              </a>
            </td>
          <td>
            Hao Wang, Zhengxin Dong, Jingyi Shi, Lei Chen, Tao Sun, Weiwen Zhang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The genus Ilex belongs to the sole family and is the single genus within the order Aquifoliales, exhibiting significant phenotypic diversity. However, the genetic differences underlying these phenotypic variations have rarely been studied. In this study, collinearity analyses of three Ilex genomes, Ilex latifolia Thunb., Ilex polyneura (Hand.-Mazz.) S. Y. Hu, and Ilex asprella Champ. ex Benth., indicated a recent fusion event contributing to the reduction of chromosomes in I. asprella. Comparative genome analyses showed slight differences in gene annotation among the three species, implying a minimal disruption of genes following chromosomal fusion in I. asprella. Comprehensive annotation of transposable elements (TEs) revealed that TEs constitute a significant portion of the Ilex genomes, with LTR transposons being predominant. TEs exhibited an inverse relationship with gene density, potentially influencing gene regulation and chromosomal architecture. TE insertions were shown to affect the conformation and binding sites of key genes such as 7-deoxyloganetin glucosyltransferase and transmembrane kinase (TMK) genes, highlighting potential functional impacts. The structural variations caused by TE insertions suggest significant roles in the evolutionary dynamics, leading to either loss or gain of gene function. This study underscores the importance of TEs in shaping the genomic landscape and evolutionary trajectories of Ilex species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/603a3cf5c3cd4429400b92b277299162fe5c83b5" target='_blank'>
              Impact of Chromosomal Fusion and Transposable Elements on the Genomic Evolution and Genetic Diversity of Ilex Species
              </a>
            </td>
          <td>
            Zhenxiu Xu, Haikun Wei, Mingyue Li, Yingjie Qiu, Lei Li, Kewang Xu, Zhonglong Guo
          </td>
          <td>2024-09-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39abde50129bd51799285e63a0661a0178d6716" target='_blank'>
              DNA damage response in breast cancer and its significant role in guiding novel precise therapies
              </a>
            </td>
          <td>
            Jiayi Li, Ziqi Jia, Lin Dong, Heng Cao, Yansong Huang, Hengyi Xu, Zhixuan Xie, Yiwen Jiang, Xiang Wang, Jiaqi Liu
          </td>
          <td>2024-09-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Small cell lung cancer (SCLC) is a lethal form of lung cancer with few treatment options and a high rate of relapse. While SCLC is initially sensitive to first-line DNA-damaging chemo- and radiotherapy, relapse disease is almost universally therapy-resistant. As a result, there has been interest in understanding the mechanisms of therapeutic resistance in this disease. Conclusions: Progress has been made in elucidating these mechanisms, particularly as they relate to the DNA damage response and SCLC differentiation and transformation, leading to many clinical trials investigating new therapies and combinations. Yet there remain many gaps in our understanding, such as the effect of epigenetics or the tumor microenvironment on treatment response, and no single mechanism has been found to be ubiquitous, suggesting a significant heterogeneity in the mechanisms of acquired resistance. Nevertheless, the advancement of techniques in the laboratory and the clinic will improve our ability to study this disease, especially in patient populations, and identify methods to surmount therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8e3e9c33346440e6cb3e34523b67202203f080" target='_blank'>
              Strategies to Target Chemoradiotherapy Resistance in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Tony Yu, Benjamin H. Lok
          </td>
          <td>2024-10-10</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Acute lymphoblastic leukemia (ALL) is an aggressive hematological malignancy that affects thousands of pediatric and adult patients each year. A major hurdle to developing more effective treatments is a limited understanding of the genes and pathways that drive ALL malignancy, and which are required for continued tumor growth. ALL is often diagnosed with the onset of advanced symptoms, making it difficult to study the early drivers of oncogenic initiation. Identifying novel genetic interactions and pathways capable of driving ALL will help to develop new diagnostic criteria and identify actionable drug targets for clinical treatments. In addition, genetic interactions between co-expressed oncogenes and their impact on initiation, progression, and therapy resistance is difficult to study and requires the costly use of genetically engineered animal models, usually mice. Here, we report a novel screening method using transgenic zebrafish to unbiasedly identify collaborating oncogenic drivers that when co-expressed lead to T- and B-ALL. We screened a transgene pool of 68 putative oncogenes identified from human ALL for synergies in driving the initiation of leukemia using a large-scale F0 transgenic gain-of-function screen in zebrafish. We recovered zebrafish with tumors that were predominantly T- and B-ALL, but also identified several fish with thymic T cell lymphomas, muscle sarcomas, and liposarcomas, demonstrating the robust capability of our approach to discover novel oncogenic collaborations. Comparative transcriptome analysis using bulk RNA sequencing between zebrafish tumors (>150 tumors) revealed candidate transgene combinations that likely act as drivers of tumor formation in T- and B-ALL. When microinjected for validation studies, two transgene combinations: 1) rag2:SET + rag2:intracellular notch1a and 2) rag2:SET + rag2:mutationally-activated IL7R lead to robust ALL formation in zebrafish, validating the oncogenic role of these factors in leukemia initiation. SET is a multifunctional protein involved in transcriptional regulation, histone binding, and cell death. While SET has been well characterized in other cancers, we found that SET is capable of collaborating with known drivers to initiate T-ALL in vivo and highly expressed in a vast majority of human T-ALL. Our SET models also revealed an enrichment of key pathways that drive proliferation in both zebrafish and human disease. We are currently performing molecular characterization of SET in human ALL cell lines to investigate SET driven ALL growth and maintenance. In addition, our panel of unique zebrafish tumors will allow help establish new models of hematologic malignancies, launch future investigations into leukemia pathology, and identify novel targets of therapy resistance.
 Citation Format: James R. Allen, Luis Antonio Corchete Sanchez, Mohamed N. Bakr, Miriam Fernandez-Lajarin, Alexandra Hazelwood, Nathan Ford, Alexandra Veloso, Victoriano Mulero, Maria L. Cayuela, Esther Rheinbay, David M. Langenau. A transgenic zebrafish screen identifies new collaborating oncogenic drivers in acute lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091a253fa76c63ad6deaa84d3f6f708b6e59ab16" target='_blank'>
              Abstract A002 A transgenic zebrafish screen identifies new collaborating oncogenic drivers in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            J. R. Allen, Luis Antonio Corchete Sanchez, Mohamed N. Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Alexandra Veloso, Victoriano Mulero, M. Cayuela, Esther Rheinbay, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="A fundamental challenge in biomedicine is understanding the mechanisms predisposing individuals to disease. While previous research has suggested that switch-like gene expression is crucial in driving biological variation and disease susceptibility, a systematic analysis across multiple tissues is still lacking. By analyzing transcriptomes from 943 individuals across 27 tissues, we identified 1,013 switch-like genes. We found that only 31 (3.1%) of these genes exhibit switch-like behavior across all tissues. These universally switch-like genes appear to be genetically driven, with large exonic genomic structural variants explaining five (∼18%) of them. The remaining switch-like genes exhibit tissue-specific expression patterns. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, likely under the influence of tissue-specific master regulators, including hormonal signals. Among our most significant findings, we identified hundreds of concordantly switched-off genes in the stomach and vagina that are linked to gastric cancer (41-fold, p<10-4) and vaginal atrophy (44-fold, p<10-4), respectively. Experimental analysis of vaginal tissues revealed that low systemic levels of estrogen lead to a significant reduction in both the epithelial thickness and the expression of the switch-like gene ALOX12. We propose a model wherein the switching off of driver genes in basal and parabasal epithelium suppresses cell proliferation therein, leading to epithelial thinning and, therefore, vaginal atrophy. Our findings underscore the significant biomedical implications of switch-like gene expression and lay the groundwork for potential diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8ed2dab6e33efaca4d5da4963155c33d9525bc" target='_blank'>
              Switch-like Gene Expression Modulates Disease Susceptibility
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kunovac Kallak, Marie Saitou, Omer Gokcumen, N. Masuda
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c443aed28fd474af23dca9ce71d466278127b676" target='_blank'>
              Multi-omics and clustering analyses reveal the mechanisms underlying unmet needs for patients with lung adenocarcinoma and identify potential therapeutic targets
              </a>
            </td>
          <td>
            Ken Asada, S. Kaneko, Ken Takasawa, K. Shiraishi, Norio Shinkai, Yoko Shimada, Satoshi Takahashi, Hidenori Machino, Kazuma Kobayashi, Amina Bolatkan, M. Komatsu, Masayoshi Yamada, M. Miyake, Hirokazu Watanabe, A. Tateishi, T. Mizuno, Y. Okubo, Masami Mukai, Tatsuya Yoshida, Yukihiro Yoshida, H. Horinouchi, Shun-ichi Watanabe, Y. Ohe, Yasushi Yatabe, Takashi Kohno, R. Hamamoto
          </td>
          <td>2024-09-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484b45daaba1111e0c4d69a75c275aee523f917f" target='_blank'>
              Multi-omics profiling of longitudinal samples reveals early genomic changes in follicular lymphoma
              </a>
            </td>
          <td>
            Baoyan Bai, J. F. Wise, Daniel Vodák, S. Nakken, Ankush Sharma, Y. Blaker, M. Brodtkorb, V. Hilden, G. Trøen, W. Ren, S. Lorenz, Michael S Lawrence, O. Myklebost, E. Kimby, Q. Pan-Hammarström, Chloé B Steen, L. Meza-Zepeda, Klaus Beiske, E. Smeland, E. Hovig, O. Lingjærde, H. Holte, J. Myklebust
          </td>
          <td>2024-08-27</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Gastrointestinal tumors, the second leading cause of human mortality, are characterized by their association with inflammation. Currently, progress in the early diagnosis and effective treatment of gastrointestinal tumors is limited. Recent whole‐genome analyses have underscored their profound heterogeneity and extensive genetic and epigenetic reprogramming. Epigenetic reprogramming pertains to dynamic and hereditable alterations in epigenetic patterns, devoid of concurrent modifications in the underlying DNA sequence. Common epigenetic modifications encompass DNA methylation, histone modifications, noncoding RNA, RNA modifications, and chromatin remodeling. These modifications possess the potential to invoke or suppress a multitude of genes associated with cancer, thereby governing the establishment of chromatin configurations characterized by diverse levels of accessibility. This intricate interplay assumes a pivotal and indispensable role in governing the commencement and advancement of gastrointestinal cancer. This article focuses on the impact of epigenetic reprogramming in the initiation and progression of gastric cancer, esophageal cancer, and colorectal cancer, as well as other uncommon gastrointestinal tumors. We elucidate the epigenetic landscape of gastrointestinal tumors, encompassing DNA methylation, histone modifications, chromatin remodeling, and their interrelationships. Besides, this review summarizes the potential diagnostic, therapeutic, and prognostic targets in epigenetic reprogramming, with the aim of assisting clinical treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa46852e2f3c4f502e85931e028a5b5bde3a9141" target='_blank'>
              Epigenetic reprogramming in gastrointestinal cancer: biology and translational perspectives
              </a>
            </td>
          <td>
            Yingjie Wang, Hongyu Liu, Mengsha Zhang, Jing Xu, Liuxian Zheng, Pengpeng Liu, Jingyao Chen, Hongyu Liu, Chong Chen
          </td>
          <td>2024-08-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell genomic analysis enables researchers to gain novel insights across diverse research areas, including developmental biology, tumor heterogeneity, and disease pathogenesis. Conducting single-cell genomic analysis using next-generation sequencing (NGS) methods has traditionally been challenging as the amount of genomic DNA present in a single cell is limited. Advancements in multiple displacement amplification (MDA) technologies allow the unbiased amplification of limited quantities of DNA under conditions that maintain its integrity. This method of amplification results in high yield and facilitates the generation of high-complexity NGS libraries that ensure the highest coverage to effectively allow variant calling. With the introduction of new sequencing platforms and chemistry, whole genome sequencing became a more cost-effective application, but enrichment of specific regions of interest further reduces the amount of required sequencing output and associated costs. There are two enrichment methods, polymerase chain reaction (PCR)-based and hybrid-capture-based methods. PCR-based methods are very flexible and highly effective but focus on specific loci, typically known to be associated with disease. Inherited diseases of unknown genetic origin require a more comprehensive approach to capture the genetic variation that is not yet associated with a specific disease. Hybrid capture enrichment methods require considerable amounts of DNA such that exome enrichment from single cells is only possible after preamplification of this limited material. This article describes the complete workflow from single cells and small quantities of DNA to exome-NGS libraries for Illumina sequencing instruments and includes the following protocols: © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Whole genome amplification from single cells or small amounts of gDNA Basic Protocol 2: NGS library generation of MDA-amplified material Basic Protocol 3: Exome enrichment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891307a5d438a6375e34496a127912e0af6f10c1" target='_blank'>
              Exome Sequencing Starting from Single Cells.
              </a>
            </td>
          <td>
            Ioanna Andreou, Markus Storbeck, Peter Hahn, Samuel J. Rulli, Eric Lader
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precise transcriptional regulation is critical for cellular function and development, yet the mechanism of this process remains poorly understood for many genes. To gain a deeper understanding of the regulation of neuropsychiatric disease risk genes, we identified a total of 39 functional enhancers for four dosage-sensitive genes, APP, FMR1, MECP2, and SIN3A, using CRISPR tiling deletion screening in human induced pluripotent stem cell (iPSC)-induced excitatory neurons. We found that enhancer annotation provides potential pathological insights into disease-associated copy number variants. More importantly, we discovered that allelic enhancer deletions at SIN3A could be compensated by increased transcriptional activities from the other intact allele. Such allelic compensation effects (ACE) on transcription is stably maintained during differentiation and, once established, cannot be reversed by ectopic SIN3A expression. Further, ACE at SIN3A occurs through dosage sensing by the promoter. Together, our findings unravel a regulatory compensation mechanism that ensures stable and precise transcriptional output for SIN3A, and potentially other dosage-sensitive genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c9cf4cae8ed81690d40ccc0776865b484f3141" target='_blank'>
              CRISPR tiling deletion screens reveal functional enhancers of neuropsychiatric risk genes and allelic compensation effects (ACE) on transcription
              </a>
            </td>
          <td>
            Xingjie Ren, Lina Zheng, Lenka Maliskova, Tsz Wai Tam, Yifan Sun, Hongjiang Liu, Jerry Lee, M. Takagi, Bin Li, Bing Ren, Wei Wang, Yin Shen
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction and Purpose: Melanoma, an aggressive malignancy from melanocytes, has a poor prognosis. Despite advances in targeted therapies and immunotherapies, drug resistance remains a challenge. This review examines the molecular pathways involved in melanoma and therapeutic strategies targeting them, aiming to improve patient outcomes and overcome treatment resistance by understanding genetic alterations and signaling cascades driving melanoma progression. 
State of Knowledge: Melanoma arises from genetic predisposition and UV radiation exposure, involving mutations in pathways like RAS/RAF/MAPK and PI3K/AKT. Targeted therapies, such as BRAF and MEK inhibitors, are effective, especially in BRAF-mutant patients. However, resistance mechanisms, including alternative pathway activation and immune evasion, limit their long-term success. Combination therapies with immunotherapies show promise in overcoming resistance. Emerging targets like NRAS, c-KIT, and c-MET offer new treatment possibilities. Ongoing challenges include identifying biomarkers for patient stratification and managing treatment-related toxicity. 
Conclusions: Understanding the molecular mechanisms of melanoma is crucial for advancing therapies. Targeted therapies and combination approaches have improved outcomes, especially in advanced cases. However, challenges like treatment resistance and toxicity require ongoing research. By exploring signaling pathways and new targets, treatment efficacy and durability can be enhanced. Continued research and clinical trials are essential to translate these advancements into practice, improving survival rates and quality of life for melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8820446b375d0816aa0b1e3bab05e5e88814d21" target='_blank'>
              Advances in Targeted Therapies and Combination Approaches for Melanoma: A Comprehensive Review
              </a>
            </td>
          <td>
            Magda Przestrzelska, Iga Ślesicka, Natalia Zozula, Zuzanna Tomczewska, Aleksandra Rykucka, Marcin Wąs, Aleksandra Latała, Justyna Kiełbasa, Agata Kowalczyk, Katarzyna Bil
          </td>
          <td>2024-10-03</td>
          <td>Quality in Sport</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0914eac9368db3a26273ef88f18ab51249d37f" target='_blank'>
              Advances in targeting cancer epigenetics using CRISPR-dCas9 technology: A comprehensive review and future prospects.
              </a>
            </td>
          <td>
            Jeevitha Rajanathadurai, Elumalai Perumal, Jospin Sindya
          </td>
          <td>2024-09-18</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Purine and pyrimidine nucleotides are crucial building blocks for the survival of cells, and there are layers of pathways to make sure a stable supply of them including de novo nucleotide biosynthesis. Fast-growing cells including cancer cells have high demand for nucleotide, and they highly utilize the nucleotide biosynthesis pathways. Due to the nature of the fast-growing cells, they tend to make more errors in replication compared with the normal cells. Naturally, DNA repair and DNA lesion bypass are heavily employed in cancer cells to ensure fidelity and completion of the replication without stalling. There have been a lot of drugs targeting cancer that mimic the chemical structures of the nucleobase, nucleoside, and nucleotides, and the resistance toward those drugs is a serious problem. Herein, we have reviewed some of the representative nucleotide analog anticancer agents such as 5-fluorouracil, specifically their mechanism of action and resistance is discussed. Also, we have chosen several enzymes in nucleotide biosynthesis, DNA repair, and DNA lesion bypass, and we have discussed the known and potential roles of these enzymes in maintaining genomic fidelity and cancer chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c452f57e681001ec1bbb61831e85fc6edf3296" target='_blank'>
              Connecting dots between nucleotide biosynthesis and DNA lesion repair/bypass in cancer
              </a>
            </td>
          <td>
            Jackson C Lin, Ayobami Oludare, Hunmin Jung
          </td>
          <td>2024-08-27</td>
          <td>Bioscience Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Biological processes such as aging, carcinogenesis, and immune response rely on the ability to maintain or rapidly expand cell populations. The fitness of the involved cells is constrained by their replicative potential, which is reflected in the cellular telomere content. Method We apply TelomereHunter to scATAC-seq data to determine telomere content on single-cell level, in a hematopoietic dataset consisting of 35,139 cells from samples of basal cell carcinoma patients receiving programmed cell death protein 1 (PD1) blockade treatment. Integrating information from open-chromatin-based signatures to assess cell identity, we characterize the heterogeneity of telomere length for individual cell populations pre- and post-immunotherapy. Results The extracted telomeric reads reflect the expected telomereome-to-genome fraction. Telomere content distributions differ significantly between cell populations, and the median telomere content in intermediate and terminal exhausted CD8+ T-cells pre-treatment is significantly correlated to response to PD-1 checkpoint blockade. Likewise, telomere content correlates with post-treatment cell proliferation in terminally exhausted and T follicular helper cells from responding patients. Conclusion Telomere content measurement from scATAC-seq data has a sufficiently high signal-to-noise ratio to detect significant differences between cell types. Furthermore, the telomere content of CD8+ exhausted T-cells pre-treatment is a putative biomarker for successful PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c0b43ae3c8ae5883f7e563f1131356d52bd5e2" target='_blank'>
              Quantifying immune cell telomere content at single-cell resolution in context of PD-1 checkpoint immunotherapy
              </a>
            </td>
          <td>
            Niklas L. Engel, Lea Herzel, Julie Surmely, Hanna Frieß, Malte Simon, B. Brors, Charles D. Imbusch, L. Feuerbach
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Building on our prior work that RNA alternative splicing modulates the druggability of kinase fusions, this study probes the clinical significance of sole reciprocal fusions. These rare genomic arrangements, despite lacking kinase domains at the DNA level, demonstrated potential RNA‐level druggability in sporadic cases from our prior research. Methods Utilizing the large‐scale multicenter approach, we performed RNA sequencing and clinical follow‐up to evaluate a broad spectrum of kinase fusions, including ALK, ROS1, RET, BRAF, NTRK, MET, NRG1, and EGFR, in 1943 patients. Results Our findings revealed 51 instances (2.57%) of sole reciprocal fusions, predominantly in lung (57%), colorectal (14%), and glioma (10%) cancers. Comparative analysis with an MSKCC cohort confirmed the prevalence in diverse cancer types and identified unique fusion partners and chromosomal locales. Cross‐validation through RNA‐NGS and FISH authenticated the existence of functional kinase domains in subsets including ALK, ROS1, RET, and BRAF, which correlated with positive clinical responses to targeted kinase inhibitors (KIs). Conversely, fusions involving EGFR, NRG1, and NTRK1/2/3 generated nonfunctional transcripts, suggesting the need for alternative therapeutic interventions. Conclusion This inaugural multicenter study introduces a novel algorithm for detecting and treating sole reciprocal fusions in advanced cancers, expanding the patient population potentially amenable to KIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc03ed5b4cf3438881405d76d4a3c612f2c0230" target='_blank'>
              Clinical relevance and druggability of sole reciprocal kinase fusions: A large‐scale study
              </a>
            </td>
          <td>
            Jiao Feng, Tonghui Ma, Chunyang Wang, Baoming Wang, Qian Liu, Zhengchuang Liu, H. Tao, Zaiyuan Ye
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Epigenetics research in evolutionary biology encompasses a variety of research areas, from regulation of gene expression to inheritance of environmentally mediated phenotypes. Such divergent research foci can occasionally render the umbrella term “epigenetics” ambiguous. Here I discuss several areas of contemporary epigenetics research in the context of evolutionary biology, aiming to provide balanced views across timescales and molecular mechanisms. The importance of epigenetics in development is now being assessed in many nonmodel species. These studies not only confirm the importance of epigenetic marks in developmental processes, but also highlight the significant diversity in epigenetic regulatory mechanisms across taxa. Further, these comparative epigenomic studies have begun to show promise toward enhancing our understanding of how regulatory programs evolve. A key property of epigenetic marks is that they can be inherited along mitotic cell lineages, and epigenetic differences that occur during early development can have lasting consequences on the organismal phenotypes. Thus, epigenetic marks may play roles in short-term (within an organism's lifetime or to the next generation) adaptation and phenotypic plasticity. However, the extent to which observed epigenetic variation occurs independently of genetic influences remains uncertain, due to the widespread impact of genetics on epigenetic variation and the limited availability of comprehensive (epi)genomic resources from most species. While epigenetic marks can be inherited independently of genetic sequences in some species, there is little evidence that such “transgenerational inheritance” is a general phenomenon. Rather, molecular mechanisms of epigenetic inheritance are highly variable between species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35f12f63feab9bcf31e03453111260d3f9bbfb1" target='_blank'>
              Epigenetics Research in Evolutionary Biology: Perspectives on Timescales and Mechanisms
              </a>
            </td>
          <td>
            Soojin V Yi
          </td>
          <td>2024-09-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Telomeres protect chromosome ends from DNA damage responses, and their dysfunction triggers genomic alterations like chromosome fusion and rearrangement, which can lead to cellular death. Certain cells, including specific cancer cells, adopt alternative lengthening of telomere (ALT) to counteract dysfunctional telomeres and proliferate indefinitely. While telomere instability and ALT activity are likely major sources of genomic alteration, the patterns and consequences of such changes at the nucleotide level in ALT cells remain unexplored. Here we generated haplotype-resolved genome assemblies for type I ALT mouse embryonic stem cells, facilitated by highly accurate or ultra-long reads and Hi-C reads. High-quality genome revealed ALT-specific complex chromosome end structures and various genomic alterations including over 1000 structural variants (SVs). The unique sequence (mTALT) used as a template for type I ALT telomeres showed traces of being recruited into the genome, with mTALT being replicated with remarkably high accuracy. Subtelomeric regions exhibited distinct characteristics: resistance to the accumulation of SVs and small variants. We genotyped SVs at allele resolution, identifying genes (Rgs6, Dpf3 and Tacc2) crucial for maintaining ALT telomere stability. Our genome assembly-based approach elucidated the unique characteristics of ALT genome, offering insights into the genome evolution of cells surviving telomere-derived crisis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d9657866100fc15991f3ce13ef281bd3238fb24" target='_blank'>
              Haplotype-resolved de novo assembly revealed unique characteristics of alternative lengthening of telomeres in mouse embryonic stem cells.
              </a>
            </td>
          <td>
            Hyunji Lee, Hiroyuki Niida, Sanghyun Sung, Junho Lee
          </td>
          <td>2024-10-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Treatment with the hypomethylating agent 5-azacytidine (AZA) increases survival in high-risk (HR) myelodysplastic syndrome (MDS) patients, but predicting patient response and overall survival remains challenging. To address these issues, we analyzed mutational and transcriptional profiles in CD34+ hematopoietic stem/progenitor cells (HSPCs) before and following AZA therapy in MDS patients. AZA treatment led to a greater reduction in the mutational burden in both blast and hematological responders than non-responders. Blast and hematological responders showed transcriptional evidence of pre-treatment enrichment for pathways such as oxidative phosphorylation, MYC targets, and mTORC1 signaling. While blast non-response was associated with TNFa signaling and leukemia stem cell signature, hematological non-response was associated with cell-cycle related pathways. AZA induced similar transcriptional responses in MDS patients regardless of response type. Comparison of blast responders and non-responders to normal controls, allowed us to generate a transcriptional classifier that could predict AZA response and survival. This classifier outperformed a previously developed gene signature in a second MDS patient cohort, but signatures of hematological responses were unable to predict survival. Overall, these studies characterize the molecular consequences of AZA treatment in MDS HSPCs and identify a potential tool for predicting AZA therapy responses and overall survival prior to initiation of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f64e8b321545ad08389931dee01148ac0d9c897" target='_blank'>
              Molecular profiling of pre- and post- 5-azacytidine myelodysplastic syndrome samples identifies predictors of response
              </a>
            </td>
          <td>
            Mónica del Rey González, Sohini Chakraborty, J. Hernández-Sánchez, M. Díez Campelo, Christopher Y. Park, J. H. Hernández Rivas
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="While immune checkpoint inhibitors have revolutionized cancer therapy, many patients exhibit poor outcomes. Here, we show immunotherapy responses in bladder and non–small cell lung cancers are effectively predicted by factoring tumor mutation burden (TMB) into burdens on specific protein assemblies. This approach identifies 13 protein assemblies for which the assembly-level mutation burden (AMB) predicts treatment outcomes, which can be combined to powerfully separate responders from nonresponders in multiple cohorts (e.g., 76% versus 37% bladder cancer 1-year survival). These results are corroborated by (i) engineered disruptions in the predictive assemblies, which modulate immunotherapy response in mice, and (ii) histochemistry showing that predicted responders have elevated inflammation. The 13 assemblies have diverse roles in DNA damage checkpoints, oxidative stress, or Janus kinase/signal transducers and activators of transcription signaling and include unexpected genes (e.g., PIK3CG and FOXP1) for which mutation affects treatment response. This study provides a roadmap for using tumor cell biology to factor mutational effects on immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5f818fb592baa5f66559ad69838daf202c25a6" target='_blank'>
              Prediction of immunotherapy response using mutations to cancer protein assemblies
              </a>
            </td>
          <td>
            JungHo Kong, Xiaoyu Zhao, Akshat Singhal, Sungjoon Park, Robin Bachelder, Jeanne Shen, Haiyu Zhang, Jimin Moon, Changho Ahn, C. Ock, Hannah Carter, T. Ideker
          </td>
          <td>2024-09-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Heart failure (HF) is a prevalent cardiovascular disease with significant morbidity and mortality rates worldwide. Due to the intricate structure of the heart, diverse cell types, and the complex pathogenesis of HF, further in-depth investigation into the underlying mechanisms  is required. The elucidation of the heterogeneity of cardiomyocytes and the intercellular communication network is particularly important. Traditional high-throughput sequencing methods provide an average measure of gene expression, failing to capture the "heterogeneity" between cells and impacting the accuracy of gene function knowledge. In contrast, single-cell sequencing techniques allow for the amplification of the entire genome or transcriptome at the individual cell level, facilitating the examination of gene structure and expression with unparalleled precision. This approach offers valuable insights into disease mechanisms, enabling the identification of changes in cellular components and gene expressions during hypertrophy associated with HF. Moreover, it reveals distinct cell populations and their unique roles in the HF microenvironment, providing a comprehensive understanding of the cellular landscape that underpins HF pathogenesis. This review focuses on the insights provided by single-cell sequencing techniques into the mechanisms underlying HF and discusses the challenges encountered in current cardiovascular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1186f68f600fee1e7602ca4a8c686869da13bf" target='_blank'>
              From Cell to Gene: Deciphering the Mechanism of Heart Failure With Single-Cell Sequencing.
              </a>
            </td>
          <td>
            Dan Zhang, Qiang Wen, Rui Zhang, K. Kou, Miao Lin, Shiyu Zhang, Jun Yang, Hangchuan Shi, Yan Yang, Xiaoqiu Tan, Shigang Yin, X. Ou
          </td>
          <td>2024-08-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genetically engineered CD8+ T cells are being explored for the treatment of various cancers. Analytical characterization represents a major challenge in the development of genetically engineered cell therapies, especially assessing the potential off-target editing and product heterogeneity. As conventional sequencing techniques only provide information at the bulk level, they are unable to detect off-target CRISPR translocation or editing events occurring in minor cell subpopulations. In this study, we report the analytical development of a single-cell multi-omics DNA and protein assay to characterize genetically engineered cell products for safety and genotoxicity assessment. We were able to quantify on-target edits, off-target events, and potential translocations at the targeting loci with per-cell granularity, providing important characterization data of the final cell product. Conclusion: A single-cell multi-omics approach provides the resolution required to understand the composition of cellular products and identify critical quality attributes (CQAs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a403ecc7f675cd4ff2179f421e45809aa747f" target='_blank'>
              Assessing a single-cell multi-omic analytic platform to characterize ex vivo-engineered T-cell therapy products
              </a>
            </td>
          <td>
            Maryam Moshref, Jerry Hung-Hao Lo, Andrew McKay, Julien Camperi, Joseph Schroer, Norikiyo Ueno, Shu Wang, Saurabh Gulati, Somayeh Tarighat, Steffen Durinck, Ho Young Lee, Dayue Chen
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Precision oncology promises individually tailored drugs and clinical care for patients with cancer: That is, "the right drug, for the right patient, at the right dose, and at the right time." Although stratification of the risk for treatment resistance and toxicity is key to precision oncology, there are multiple ways in which such stratification can be achieved, for example, genetic, functional pathway based, among others. Moving toward precision oncology is sorely needed in the case of acute lymphoblastic leukemia (ALL) wherein adult patients display survival rates ranging from 30% to 70%. The present study reports on the pathway activity signature of adult B-ALL, with an eye to precision oncology. Transcriptome profiles from three different expression datasets, comprising 346 patients who were adolescents or adults with B-ALL, were harnessed to determine the activity of signaling pathways commonly disrupted in B-ALL. Pathway activity analyses revealed that Ph-like ALL closely resembles Ph-positive ALL. Although this was the case at the average pathway activity level, the pathway activity patterns in B-ALL differ from genetic subtypes. Importantly, clustering analysis revealed that five distinct clusters exist in B-ALL patients based on pathway activity, with each cluster displaying a unique pattern of pathway activation. Identifying pathway-based subtypes thus appears to be crucial, considering the inherent heterogeneity among patients with the same genetic subtype. In conclusion, a pathway-based stratification of the B-ALL could potentially allow for simultaneously targeting highly active pathways within each ALL subtype, and thus might open up new avenues of innovation for personalized/precision medicine in this cancer that continues to have poor prognosis in adult patients compared with the children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72b528a52f9a1c25407f3a5bc1baf86dc4c7b9e9" target='_blank'>
              Expanding Beyond Genetic Subtypes in B-Cell Acute Lymphoblastic Leukemia: A Pathway-Based Stratification of Patients for Precision Oncology.
              </a>
            </td>
          <td>
            Ozlem Ulucan
          </td>
          <td>2024-08-19</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Children with Down syndrome (DS) and B-cell acute lymphoblastic leukemia (B-ALL) are at increased risk for treatment-related mortality and relapse, highlighting the need for new therapies. Leukemia cell lines (CLs) have been fundamental to understanding therapeutic responses to pharmacological agents. We generated three DS B-ALL CLs characterized with diverse genomic alterations, including IGH::CRLF2 rearrangement (BCR::ABL1 like), mutations in FLT3 and TP53, and a novel ERG::CEBPD rearrangement. DS CLs had diminished proliferation, metabolism, and mitochondrial function when compared to non-DS (NDS) CLs and interestingly, these findings were similar to NDS Philadelphia chromosome-like (Ph-like) B-ALL CLs. Based on similar mitochondrial defects and prior preclinical data using Venetoclax for Ph-like B-ALL, we hypothesized that Venetoclax would be effective in DS. Intriguingly, Venetoclax was more effective in DS when compared to both NDS and Ph-like CLs. Efficacy was observed in DS patient derived xenografts (PDXs) and diagnostic/relapsed patient samples treated with Venetoclax, which synergized with Trametinib and Vincristine. Mass spectrometry-based multiomics analyses in DS and NDS B-ALL patient samples revealed an enriched metabolite profile in DS, particularly in the hubs of glucose metabolism and polyunsaturated phosphatidylcholines and phosphatidylinositols. Transcriptome analyses in DS B-ALL patients (n=249) supported enhanced glucose and fatty acid metabolism; along with a reliance on BCL2 family members for apoptosis resistance. Venetoclax synergized with inhibition of glucose metabolism in DS B-ALL CLs. In summary, we have generated novel tools for studying DS B-ALL and identify altered metabolism in DS that responds to Venetoclax.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f404242cc6274ce2ef0f772a4cb26039fa167f" target='_blank'>
              Metabolic vulnerabilities in Down syndrome B-cell acute lymphoblastic leukemia can be targeted using Venetoclax
              </a>
            </td>
          <td>
            Dallas Jones, Elena Woods, Zhenhua Li, Kristin L. Schaller, Eric Hoffmeyer, Ben Kooiman, Joseph Fernandez, Tegan Wharton, Dejene M. Tufa, Spencer C. Hall, George Trahan, Kelly W. Maloney, Holly L. Pacenta, Kelly Sullivan, Joaquin Espinosa, James R. Roede, John O. Marentette, Brett M Stevens, Anagha Inguva, D. Stephenson, J. Reisz, Angelo D’Alessandro, Craig T. Jordan, Jun J Yang, Michael R Verneris
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background: Bone sarcomas, a rare group of malignant neoplasms arising from mesenchymal cells, present significant diagnostic and therapeutic challenges due to their heterogeneity and overlapping characteristics with other tumours. Conventional diagnostic methods are often inadequate, necessitating advanced techniques to improve diagnostic accuracy and treatment outcomes. Objectives: This review aims to evaluate the role of genetic and molecular profiling in bone sarcomas, focusing on its potential to develop personalized treatment approaches and overcome therapeutic resistance. Methods: A systematic search was conducted across PubMed and Google Scholar to identify relevant studies published in the last ten years. Inclusion criteria encompassed randomized controlled trials, cohort studies, and systematic reviews investigating genetic and molecular profiling techniques such as RT-PCR, FISH, and NGS in bone sarcomas. Exclusion criteria included studies on soft tissue sarcomas, animal models, and non-peer-reviewed articles. Discussion: Genetic and molecular profiling has revolutionized the diagnosis and management of bone sarcomas by identifying key genetic alterations and enabling targeted therapies. While early clinical data are promising, larger randomized trials are needed to validate these findings. Resistance mechanisms, such as miRNA dysregulation, present ongoing challenges, highlighting the need for combination therapies to optimize treatment outcomes. Integrating genetic and molecular profiling into clinical practice offers the potential for personalized treatment regimens, improving survival rates and quality of life for patients with bone sarcomas. Further research is essential to fully realize the benefits of these advanced techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81eb80f85c15cdbb59566542cc86af0d9cf4d8" target='_blank'>
              Genetic and molecular profiling in bone sarcomas: advancing personalized treatment approaches
              </a>
            </td>
          <td>
            Gandhi Dhruv, Aastha N. Raj, Saksham Sharma
          </td>
          <td>2024-08-19</td>
          <td>InterConf</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The complexity of cancer requires a comprehensive approach to understand its diverse manifestations and underlying mechanisms. Initially outlined by Hanahan and Weinberg in 2000 and updated in 2010, the hallmarks of cancer provide a conceptual basis for understanding inherent variability in cancer biology. Recent expansions have further elucidated additional hallmarks, including phenotypic plasticity and senescent cells. The International Agency for Research on Cancer (IARC) has identified the key characteristics of carcinogens (KCCs) to evaluate their carcinogenic potential. We analyzed chemicals of concern for environmental exposure that interact with specific receptors to induce genomic instability, epigenetic alterations, immune suppression, and receptor-mediated effects, thereby contributing to chronic inflammation. Despite their varying degrees of carcinogenicity, these chemicals have similar KCC profiles. Our analysis highlights the pivotal role of receptor binding in activating most other KCCs, underscoring their significance in cancer initiation. Although KCCs are associated with early molecular or cellular events, they do not encompass processes directly linked to full cellular malignancy. Thus, there is a need to integrate clear endpoints that anchor KCCs to the acquisition of a complete malignant phenotype into chemical testing. From the perspective of toxicology and cancer research, an all-encompassing strategy that incorporates both existing and novel KCCs and cancer hallmarks is essential to enable the targeted identification of prevalent carcinogens and facilitate zone-specific prevention strategies. To achieve this goal, collaboration between the KCC and cancer hallmarks communities becomes essential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a462cc73a8d245ed18a298dc440b46e953accd9" target='_blank'>
              Key characteristics of carcinogens meet hallmarks for prevention-cutting the Gordian knot
              </a>
            </td>
          <td>
            S. Senga, William H. Bisson, Annamaria Colacci
          </td>
          <td>2024-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Barriers in access to clinical care among under-served populations in combination with a vast underrepresentation at a scientific research level perpetuate inequalities in cancer treatment. These disparities are further disseminated due to the lack of understanding underlying genomic differences contributing to cancer outcome among these patients. The contributions of germline and somatic variations in cancer often shape treatment regimens, and these variations are not well investigated in non-European populations. We have described a class of potential therapeutic targets (GEMINI genes) that reflect a combination of germline alterations with somatic loss of heterozygosity (LOH). GEMINI genes are essential genes with frequent germline polymorphisms, whose frequency varies by ancestry. Indeed, because genomes of African ancestry tend to be the most diverse, many GEMINI genes are the most polymorphic in this patient population. When normal cells are heterozygous, LOH leaves cancer cells reliant on the gene products encoded by a single allele, where as healthy cells retain both copies. LOH is a common genetic event, often affecting over 20% of the genome in certain cancers, making this approach widely useful. Targeting GEMINI vulnerabilities opens up possibilities for allele-specific inhibitors or degrader therapeutics that can be catered in an ancestry-dependent manner, to ameliorate inequities in cancer genomics research. One approach to readily target individual alleles is to utilize nucleic acid-based methods, as we’ve previously demonstrated using PRIM1. This therapeutic is expected to be the most specific when targeting polymorphisms that create the greatest differences between alleles. Here, we have prioritized a set of GEMINI targets based upon the extent of differences between alleles, and rates of heterozygosity and LOH across populations. Our initial analysis of TCGA data identified 5664 variants in 1278 essential genes that undergo LOH in cancer. We filtered GEMINI vulnerabilities by means of insertion-deletion (indel) difference of the alternate allele compared to the reference allele, followed by sorting based off maximum allele frequency by genetic ancestry. Here, we identify 31 variants across 23 genes enriched in African ancestry, 7 variants across 7 genes enriched in American ancestry, 12 variants across 12 genes enriched in East Asian ancestry, 4 variants enriched in 4 genes in South Asian ancestry, and 3 variants enriched in 3 genes in European ancestry. In addition, each of these ancestry-specific GEMINI genes exhibit a minimum 10% pan-cancer LOH rate, with the highest LOH frequencies observed in ovarian and kidney chromophobe cancers. As both these cancers exhibit disparities in mortality at a population level, targeting GEMINI genes as a therapeutic strategy may aid in lessening the disproportionate rate of outcome resultant of these cancers. Overall, we identify novel GEMINI genes to assess ancestry-specific genomic variations to serve as targets for precision treatment specific to underserved patient populations.
 Citation Format: Nicole Peiris, Catherine Hazard, Charlotte E. Farquhar, Andrei Loas, Bradley L. Pentelute, Rameen Beroukhim. Genetic ancestry specific loss of heterozygosity of essential genes as therapeutic vulnerabilities in cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C099.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e337d55b2fcfc1ea91f26641b13ccce525e55ce5" target='_blank'>
              Abstract C099: Genetic ancestry specific loss of heterozygosity of essential genes as therapeutic vulnerabilities in cancer
              </a>
            </td>
          <td>
            Nicole Peiris, Catherine Hazard, C. Farquhar, A. Loas, Bradley L. Pentelute, R. Beroukhim
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive and highly malignant primary brain tumor characterized by rapid growth and a poor prognosis for patients. Despite advancements in treatment, the median survival time for GBM patients remains low. One of the crucial challenges in understanding and treating GBMs involves its remarkable cellular heterogeneity and adaptability. Central to the survival and proliferation of GBM cells is their ability to undergo metabolic reprogramming. Metabolic reprogramming is a process that allows cancer cells to alter their metabolism to meet the increased demands of rapid growth and to survive in the often oxygen- and nutrient-deficient tumor microenvironment. These changes in metabolism include the Warburg effect, alterations in several key metabolic pathways including glutamine metabolism, fatty acid synthesis, and the tricarboxylic acid (TCA) cycle, increased uptake and utilization of glutamine, and more. Despite the complexity and adaptability of GBM metabolism, a deeper understanding of its metabolic reprogramming offers hope for developing more effective therapeutic interventions against GBMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78eb9193ec031d5621dbde1edebe814a7acad9" target='_blank'>
              Metabolic Reprogramming in Glioblastoma Multiforme: A Review of Pathways and Therapeutic Targets
              </a>
            </td>
          <td>
            Ashley Irin Cortes Ballen, Maryam Amosu, Surya Ravinder, Joey Chan, Emre Derin, Hasan Slika, Betty Tyler
          </td>
          <td>2024-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Fragile X Syndrome (FXS) is a genetic neurodevelopmental disorder closely associated with intellectual disability and autism spectrum disorders. The core of the disease lies in the abnormal expansion of the CGG trinucleotide repeat sequence at the 5′end of the FMR1 gene. When the repetition exceeds 200 times, it causes the silencing of the FMR1 gene, leading to the absence of the encoded Fragile X mental retardation protein 1 (FMRP). Although the detailed mechanism by which the CGG repeat expansion triggers gene silencing is yet to be fully elucidated, it is known that this process does not alter the promoter region or the coding sequence of the FMR1 gene. This discovery provides a scientific basis for the potential reversal of FMR1 gene silencing through interventional approaches, thereby improving the symptoms of FXS. Epigenetics, a mechanism of genetic regulation that does not depend on changes in the DNA sequence, has become a new focus in FXS research by modulating gene expression in a reversible manner. The latest progress in molecular genetics has revealed that epigenetics plays a key role in the pathogenesis and pathophysiological processes of FXS. This article compiles the existing research findings on the role of epigenetics in Fragile X Syndrome (FXS) with the aim of deepening the understanding of the pathogenesis of FXS to identify potential targets for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f2a0a0d20e60f40e1927dda3be0710c0759d2d" target='_blank'>
              Epigenetic insights into Fragile X Syndrome
              </a>
            </td>
          <td>
            Liangqun Xie, Huiying Li, MengLiang Xiao, Ningjing Chen, Xiaoxiao Zang, Yingying Liu, Hong Ye, Chaogang Tang
          </td>
          <td>2024-08-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background and aims: Colorectal carcinomas (CRCs) in patients with Lynch syndrome (LS) exhibit heightened immunogenicity due to mismatch repair deficiency (MMR-d), often resulting in favorable responses to T cell immune checkpoint therapies. Recent studies indicate that the phenotype and genotype of LS-associated CRCs vary depending on the specific MMR gene mutated. Here, we investigated whether the immune profiles of LS-associated CRCs differ based on the MMR gene defects. Methods: Tissue material from 18 MLH1-, 16 MSH2-, 40 MSH6-, and 23 PMS2-mutated CRCs and 35 sporadic MMR-d CRCs were included in the study. Imaging mass cytometry (IMC) analysis, along with targeted multiplex immunofluorescence imaging (mIF) and immunohistochemistry, were applied to examine the tumor immune microenvironment, including Human Leukocyte Antigen (HLA) class I and programmed death-ligand 1 (PD-L1) expression. Results: Unsupervised hierarchical clustering of cell phenotypes identified by IMC, followed by mIF validation, revealed comparable lymphoid and myeloid cell infiltration levels across CRCs from all MMR groups. Infiltrating T cell levels negatively correlated with the number of mutations at coding microsatellite sequences, particularly in MLH1-mutated CRCs. HLA class I defects were observed in 76% of all CRCs. These defects were more frequently accompanied by {beta}2M defects in hereditary MMR-d CRCs (67%) compared to sporadic MMR-d CRCs (37%), and did not associate with the number of {gamma}{delta} T cells, which were present in CRCs from all MMR groups. PD-L1 expression in tumor cells was only detected in 8% of all CRCs. Conclusion: Our findings illustrate that, from an immunological perspective, there is no evidence of differing immunogenic features across MMR defects. This is important to consider when developing preventive vaccine strategies and evaluating immunotherapy for LS patients and those with MMR-d CRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0cdd08de3e8c7b38b9b131001cb730c9f4dade" target='_blank'>
              Immunological profiles in Lynch syndrome colorectal cancers are not specific to mismatch repair gene defects
              </a>
            </td>
          <td>
            Noah C. Helderman, M. Ijsselsteijn, Madalina Cabuta, Manon van der Ploeg, T. Wezel, A. Ahadova, Matthias Kloor, H. Morreau, Maartje Nielsen, N. F. D. Miranda
          </td>
          <td>2024-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="In the fields of medicine and bioscience, gene editing is increasingly recognized as a promising therapeutic approach for treating pathogenic variants in humans and other living organisms. With advancements in technology and knowledge, it is now understood that most genetic defects are caused by single-base pair variants. The ability to substitute genes using genome editing tools enables scientists and doctors to cure genetic diseases and disorders. Starting with CRISPR (clustered regularly interspaced short palindromic repeats)/Cas, the technology has evolved to become more efficient and safer, leading to the development of base and prime editors. Furthermore, various approaches are used to treat genetic disorders such as hemophilia, cystic fibrosis, and Duchenne muscular dystrophy. As previously mentioned, most genetic defects leading to specific diseases are caused by single-base pair variants, which can occur at many locations in corresponding gene, potentially causing the same disease. This means that, even when using the same genome editing tool, results in terms of editing efficiency or treatment effectiveness may differ. Therefore, different approaches may need to be applied to different types of diseases. Prevalently, due to the safety of adeno-associated virus (AAV) vectors in gene therapy, most clinical trials of gene therapy are based on AAV delivery methods. However, despite their safety and nonintegration into the host genome, their limitations, such as confined capacity, dosage-dependent viral toxicity, and immunogenicity, necessitate the development of new approaches to enhance treatment effects. This review provides the structure and function of each CRISPR-based gene editing tool and focuses on introducing new approaches in gene therapy associated with improving treatment efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175c7ca99490d5250dd5cadc163e90b91db26af6" target='_blank'>
              Mechanism of Genome Editing Tools and Their Application on Genetic Inheritance Disorders
              </a>
            </td>
          <td>
            Dae Hwan Oh
          </td>
          <td>2024-09-16</td>
          <td>Global Medical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Global healthcare systems have a great challenge in the form of inherited hematological diseases, which necessitates the development of new remedial strategies. By precisely targeting inherited abnormalities, Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-mediated ex vivo gene editing has surfaced as a promising approach to treat these diseases. This review offers a comprehensive examination of the advancements, challenges, and clinical eventuality of CRISPR-Cas9-intermediated ex vivo gene editing for inherited hematological diseases. With advancements in CRISPR-Cas9 technology, the eventuality to correct inheritable mutations responsible for inherited hematological diseases is within reach. However, challenges such as off-target effects, immune responses, and ethical considerations need to be addressed for the safe and effective perpetration of this technology. A promising understanding of how CRISPR-Cas9-intermediated gene editing functions in practice is handed by ongoing clinical studies, giving rise to the possibility of advanced remedial approaches and bettered patient issues. By addressing these complications in a human-readable format, this review attempts to provide greater understanding and appreciation for the eventuality of CRISPR-Cas9 technology in revolutionizing the treatment landscape for these challenging disorders and contribute to the ongoing discussion in the field and facilitate further exploration towards effective treatments for these challenging disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d496cce2c3ddc8c2b97b6be801d229f168c15" target='_blank'>
              CRISPR-CAS9-MEDIATED EX VIVO GENE EDITING FOR INHERITED HEMATOLOGICAL DISORDERS: ADVANCEMENTS, CHALLENGES, AND CLINICAL POTENTIAL
              </a>
            </td>
          <td>
            Atasi Ranjan Panda, Shreeya Das
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Pharmacy and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Fröse, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel R. Goulet, Michael T. Hemann. Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a130506e26e2da6b9c6338dc0032b746f1f4ca46" target='_blank'>
              Abstract B060: Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Fröse, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel R. Goulet, M. Hemann
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd905b03493f5b109589f0899bdd4480c79c0580" target='_blank'>
              Targeting mutant p53: a key player in breast cancer pathogenesis and beyond
              </a>
            </td>
          <td>
            Hina Qayoom, Burhan Ul Haq, Shazia Sofi, Nusrat Jan, Asma Jan, M. A. Mir
          </td>
          <td>2024-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Understanding the molecular mechanisms underlying tumour progression and resistance to treatment is crucial to the development of new treatment approaches. In high risk pediatric cancers, somatic DNA alterations such as genomic amplifications, copy number alterations, translocations or mutations play an important role as molecular diagnostic, prognostic and predictive biomarkers. However it is now known that in many high risk pediatric cancers, clonal evolution most likely plays an important role in tumor progression and treatment resistance. Circulating tumour DNA, a fraction of cell free DNA can be readily isolated from plasma and now provides an important tool and surrogate for tumor molecular analyses at diagnosis, during treatment and follow-up. At diagnosis, the prospective clinical trials MICCHADO (NCT03496402) enrolled 599 patients, including high-risk pediatric cancer (Neuroblastoma, Rhabdomyosarcoma, Ewing sarcoma, other high-risk cancers). Whole -Exome-Sequencing (WES) was performed on tumor, germline DNA and cfDNA extracted from plasma at diagnosis, during treatment and follow up. Whereas all cfDNA samples obtained at follow-up in patients without evidence of disease revealed no or few tumor cell-specific SNVs, cfDNA samples obtained at relapse harbored additional, new relapse-specific SNVs in all cases, targeting genes of interest. Deep sequencing capture techniques enable to develop models of clonal evolution. In pediatric embryonal brain tumours, ctDNA isolated from CSF enables detection of tumor cell specific genetic alterations with a high sensitivity. At relapse, ctDNA studies can provide complementary information to molecular analyses of tumour samples performed within programs such as MAPPYACTS (NCT02613962), with 76% of actionable alterations detected in tumor also identified in ctDNA, while also highlighting the importance of spatial heterogeneity. MONALISA, a SIOPEN pragmatic clinical trial to MOnitor NeuroblastomA relapse with LIquid biopsy Sensitive Analysis aims to establish liquid biopsies as standard-of-care to monitor relapsed/refractory neuroblastoma. Reliable, early assessment of molecular progression or relapse determined by mRNA and/or ctDNA analysis is the main aim of this randomized clinical trial. ctDNA also enables to infer expression profiles. Gene expression levels are reflected by nucleosome positioning, and differences in nucleosome organization at transcription start sites (TSS) lead to differential clipping of fragments upon ctDNA release and distinct nucleosome footprints depending on the expression of a given gene in the originating cells. Altogether the presence of tumor genetic and epigenetic abnormalities in ctDNA can be documented in most patients with high risk pediatric cancer and frequently suggest spatial and temporal heterogeneity. Sequential studies will further elucidate mechanisms of clonal evolution, tumor progression and therapy resistance. Thus, sequential studies based on liquid biopsies are now integrated into the development and optimization of targeted treatment strategies.
 Citation Format: Gudrun Schleiermacher. Opportunities and challenges for liquid biopsies in pediatric oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d5d9b1f8524120f05332a3107841ee2c1bb8ff8" target='_blank'>
              Abstract IA010: Opportunities and challenges for liquid biopsies in pediatric oncology
              </a>
            </td>
          <td>
            G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d7559338158829d62b6b4147b9bfe3e99b724d9" target='_blank'>
              HLA-G neo-expression modifies genetic programs governing tumor cell lines
              </a>
            </td>
          <td>
            D. Tronik-Le Roux, M. Daouya, I. Poras, François Desgrandchamps, E. Carosella
          </td>
          <td>2024-10-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Recent studies demonstrate growing roles for genetic mosaicism in neurodevelopmental and neuropsychiatric disorders, with the paradigm being drug-resistant pediatric focal epilepsy related to activating somatic variants in the PI3K-mTOR pathway. While identifying the genotype-associated changes at the single-cell level is fundamental to understanding disease pathophysiology, this remains technically challenging in human tissue samples with existing methods. Here, we performed single-nucleus RNA-sequencing (snRNA-seq) of 20 focal cortical dysplasia (FCD) samples removed surgically for treatment of drug-resistant epilepsy, and 10 non-FCD controls, and we developed a new approach, Genotyping Of Transcriptomes Enhanced with Nanopore sequencing (GO-TEN), that combines targeted complementary (c)DNA sequencing with snRNA-seq to perform concurrent single-nucleus genotyping and transcriptional analysis. We find that mosaic pathogenic variants in FCD do not produce a detectable novel cell identity, but instead we observe conserved cell types present both in FCD cases and non-FCD control specimens. Similarly, GO-TEN analysis shows that most pathogenic variant-carrying cells have well-differentiated neuronal or glial identities and are enriched for layer II-III excitatory neurons. We identify cell-intrinsic disruption of glutamate and GABA-A signaling pathways in variant-carrying neurons and altered intercellular signaling, making potential mechanisms for epileptogenesis in FCD. In summary, by addressing genotype-specific changes in mosaic epilepsy-associated lesions, our study highlights new potential disease mechanisms and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc181f7065f3cb1a379b3d4368b2d029b87b1b3b" target='_blank'>
              Cell-type-informed genotyping of mosaic focal epilepsies reveals cell-autonomous and non-cell-autonomous disease-associated transcriptional programs
              </a>
            </td>
          <td>
            S. Bizzotto, E. Stronge, Maya Talukdar, Qiwen Hu, Zinan Zhou, A. Huang, B. Chhouk, Alissa M. D’Gama, Edward Yang, T. E. Green, David C. Reutens, Saul A. Mullen, Michael S. Hildebrand, Russell J Buono, A. Poduri, S. Khoshkhoo, Christopher A. Walsh
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3305dbdb10abe125333dc69b5540a59698a94ae3" target='_blank'>
              Signaling pathways involved in colorectal cancer: pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Qing Li, Shan Geng, Hao Luo, Wei Wang, Ya-Qi Mo, Qing Luo, Lu Wang, Guan-Bin Song, Jian-Peng Sheng, Bo Xu
          </td>
          <td>2024-10-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most phenotype-associated genetic variants map to non-coding regulatory regions of the human genome. Moreover, variants associated with blood cell phenotypes are enriched in regulatory regions active during hematopoiesis. To systematically explore the nature of these regions, we developed a highly efficient strategy, Perturb-multiome, that makes it possible to simultaneously profile both chromatin accessibility and gene expression in single cells with CRISPR-mediated perturbation of a range of master transcription factors (TFs). This approach allowed us to examine the connection between TFs, accessible regions, and gene expression across the genome throughout hematopoietic differentiation. We discovered that variants within the TF-sensitive accessible chromatin regions, while representing less than 0.3% of the genome, show a ∼100-fold enrichment in heritability across certain blood cell phenotypes; this enrichment is strikingly higher than for other accessible chromatin regions. Our approach facilitates large-scale mechanistic understanding of phenotype-associated genetic variants by connecting key cis-regulatory elements and their target genes within gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd75703a3e0196006c63fecef421b6d1dc25cee5" target='_blank'>
              Transcription factor networks disproportionately enrich for heritability of blood cell phenotypes
              </a>
            </td>
          <td>
            Jorge D. Martin-Rufino, Alexis Caulier, Seayoung Lee, Nicole Castano, Emily King, Samantha Joubran, Marcus Jones, Seth R Goldman, Uma P. Arora, Lara Wahlster, Eric S. Lander, V. Sankaran
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background While immune checkpoint inhibitors (ICIs) are adopted as standard therapy in non-small cell lung cancer (NSCLC) patients, factors that influence variable prognosis still remain elusive. Therefore, a deeper understanding is needed of how germline variants regulate the transcriptomes of circulating immune cells in metastasis, and ultimately influence immunotherapy outcomes. Methods We collected peripheral blood mononuclear cells (PBMCs) from 73 ICI-treated NSCLC patients, conducted single-cell RNA sequencing, and called germline variants via SNP microarray. Determination of expression quantitative trait loci (eQTL) allows elucidating genetic interactions between germline variants and gene expression. Utilizing aggregation-based eQTL mapping and network analysis across eight blood cell types, we sought cell-type-specific and ICI-prognosis-dependent gene regulatory signatures. Results Our sc-eQTL analysis identified 3,616 blood- and 702 lung-cancer-specific eGenes across eight major clusters and treatment conditions, highlighting involvement of immune-related pathways. Network analysis revealed TBX21-EOMES regulons activity in CD8+ T cells and the enrichment of eQTLs in higher-centrality genes as predictive factors of ICI response. Conclusions Our findings suggest that in the circulating immune cells of NSCLC patients, transcriptomic regulation differs in a cell type- and treatment-specific manner. They further highlight the role of eQTL loci as broad controllers of ICI-prognosis-predicting gene networks. The predictive networks and identification of eQTL contributions can lead to deeper understanding and personalized ICI therapy response prediction based on germline variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70aa2fedf9166d7283b9f0e6d961d358ca399349" target='_blank'>
              Systemic CD8+ T cell effector signature predicts prognosis of lung cancer immunotherapy
              </a>
            </td>
          <td>
            Hyungtai Sim, Geun-Ho Park, Woong-Yang Park, Se-Hoon Lee, Murim Choi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Since their discovery, CRISPR/Cas9 systems have been repurposed for programmable targeted genomic editing. This has led to unprecedented advancement of gene editing for therapeutic benefit. Initial uses of CRISPR/Cas9 were focused on gene disruption via DNA cleavage, but significant engineering led to systems for single base editing as well as insertion, deletion and manipulation of short stretches of genomic sequences using nicking Cas9 and RT-based methods. These technologies allowed safer and more precise editing but were limited to small corrections and showed significantly reduced efficiencies in nondividing cells, presenting difficulty for translation to in vivo therapies. To find an alternate editing strategy that could address these shortcomings, we revisited the mechanism of DNA nicking by nCas9. nCas9 nicking creates a free 5’ phosphate group and a 3’ hydroxyl group on the complementary strand of the target sequence. Under ordinary conditions in the cell these ends are re-joined by endogenously expressed ligases to repair DNA back to wild-type. If, however, a DNA fragment containing the desired edit were present, ligation of the nicked genomic DNA with the delivered fragment could result in gene editing. We demonstrate that optimization of each component and introduction of a chemically modified high affinity splinting DNA allows a variety of ligase-based edits, including longer edits not efficient with RT-based systems, at high efficiencies and fidelities that minimize genomic byproducts in both dividing and nondividing cells as well as in vivo in adult mice. Here we present the first therapeutically relevant ligation-based programmable gene editing technology, L-PGI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05db8d5146707d667516d894067c8635a13e5573" target='_blank'>
              Ligase-mediated programmable genomic integration (L-PGI): an efficient site-specific gene editing system that overcomes the limitations of reverse transcriptase-based editing systems
              </a>
            </td>
          <td>
            Angela Xinyi Nan, Michael Chickering, Christopher Bartolome, Neeta Shadija, Dan Li, Brett Estes, Jessica Von Stetina, Wei Li, Jason Andresen, Jesse C Cochrane, Chen Bai, Jason Gatlin, Jie Wang, Davood Norouzi, Sandeep Kumar, Maike Thamsen Dunyak, Leonard Chavez, Anmol Seth, Shakked Halperin, Jonathan D. Finn, Jenny Xie
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma A. Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="If one must prioritize among the vast array of contributing factors to cancer evolution, environmental-stress-mediated chromosome instability (CIN) should easily surpass individual gene mutations. CIN leads to the emergence of genomically unstable life forms, enabling them to grow dominantly within the stable life form of the host. In contrast, stochastic gene mutations play a role in aiding the growth of the cancer population, with their importance depending on the initial emergence of the new system. Furthermore, many specific gene mutations among the many available can perform this function, decreasing the clinical value of any specific gene mutation. Since these unstable life forms can respond to treatment differently than stable ones, cancer often escapes from drug treatment by forming new systems, which leads to problems during the treatment for patients. To understand how diverse factors impact CIN-mediated macroevolution and genome integrity–ensured microevolution, the concept of two-phased cancer evolution is used to reconcile some major characteristics of cancer, such as bioenergetic, unicellular, and multicellular evolution. Specifically, the spiral of life function model is proposed, which integrates major historical evolutionary innovations and conservation with information management. Unlike normal organismal evolution in the microevolutionary phase, where a given species occupies a specific location within the spiral, cancer populations are highly heterogenous at multiple levels, including epigenetic levels. Individual cells occupy different levels and positions within the spiral, leading to supersystems of mixed cellular populations that exhibit both macro and microevolution. This analysis, utilizing karyotype to define the genetic networks of the cellular system and CIN to determine the instability of the system, as well as considering gene mutation and epigenetics as modifiers of the system for information amplification and usage, explores the high evolutionary potential of cancer. It provides a new, unified understanding of cancer as a supersystem, encouraging efforts to leverage the dynamics of CIN to develop improved treatment options. Moreover, it offers a historically contingent model for organismal evolution that reconciles the roles of both evolutionary innovation and conservation through macroevolution and microevolution, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4dc885b10926269c63b9c0c31f322d9ccdff48" target='_blank'>
              The Spiral Model of Evolution: Stable Life Forms of Organisms and Unstable Life Forms of Cancers
              </a>
            </td>
          <td>
            Andrzej Kasperski, Henry H. Heng
          </td>
          <td>2024-08-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Gastric cancer (GC) is the fifth most common human cancer worldwide, but the genetic etiology is largely unknown. We performed a Bayesian genome-wide association study and selection analyses in a naturally-occurring canine model of GC, the Belgian Tervuren and Sheepdog breeds, to elucidate underlying genetic risk factors. We identified 15 loci with over 90% predictive accuracy for the GC phenotype. Variant filtering revealed germline putative regulatory variants for the EPAS1 (HIF2A) and PTEN genes and a coding variant in CD101. Although closely related to Tervuren and Sheepdogs, Belgian Malinois rarely develop GC. Across-breed analyses uncovered protective haplotypes under selection in Malinois at SOX2-OT and IGF2BP2. Among Tervuren and Sheepdogs, HDAC2 putative regulatory variants were present at comparatively high frequency and were associated with GC. Here, we describe a complex genetic architecture governing GC in a dog model, including genes such as PDZRN3, that have not been associated with human GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1797dc7272399029f05047a73a22158a3c2ed9d7" target='_blank'>
              Genomic analyses identify 15 susceptibility loci and reveal HDAC2 , SOX2-OT , and IGF2BP2 in a naturally-occurring canine model of gastric cancer
              </a>
            </td>
          <td>
            S. Cook, S. Hugen, Jessica J. Hayward, T. Famula, Janelle M. Belanger, Elizabeth McNiel, H. Fieten, Anita M. Oberbauer, P. A. Leegwater, Elaine A. Ostrander, P. Mandigers, Jacquelyn M. Evans
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7a9a7a504580b14d647edcfb85c539f3ac48" target='_blank'>
              Enhanced depletion of MLL-fusion proteins in acute leukemia: potential for improved therapeutic outcomes
              </a>
            </td>
          <td>
            Noelia Che, S. Cantilena, R. Looi-Somoye, Danesh Sundar, Kent Fung, J. de Boer, Owen Williams
          </td>
          <td>2024-08-16</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d59a35bbbd452d19f5eb530222e3178b220b08" target='_blank'>
              Tumour mutational burden: clinical utility, challenges and emerging improvements.
              </a>
            </td>
          <td>
            J. Budczies, D. Kazdal, Michael Menzel, Susanne Beck, K. Kluck, Christian Altbürger, C. Schwab, M. Allgäuer, A. Ahadova, Matthias Kloor, P. Schirmacher, Solange Peters, Alwin Krämer, P. Christopoulos, A. Stenzinger
          </td>
          <td>2024-08-27</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Oncogenic-addicted non-small cell lung cancer (NSCLC) has emerged as the most prevalent form of lung cancer, presenting a dynamic landscape in treatment modalities. Among these, epidermal growth factor receptor (EGFR)-mutant NSCLC remains the predominant oncogenic mutation, particularly prevalent in regions such as Asia and Latin America.


CASE PRESENTATION
This case study highlights the experience of a woman diagnosed with EGFR-sensitive (del exon 19) mutant NSCLC who demonstrated an extended duration of response (DOR) to third-generation EGFR-TKI therapy. Upon disease progression, detection of MET gene amplification prompted the addition of a selective MET inhibitor to the existing EGFR-TKI regimen, resulting in a complete response for the patient.


DISCUSSION/CONCLUSION
The molecular heterogeneity of this condition has significantly increased in complexity over recent years, marked by the identification of baseline co-alterations and development of a broad spectrum of resistance mechanisms post-EGFR tyrosine kinase inhibitor (TKI) therapy. This complexity poses a substantial challenge to clinicians. Despite the rapid advancement of targeted therapies and the implementation of treatment escalation through combination strategies, there remains an ongoing debate regarding which patients would benefit most from combination therapies, both in the initial treatment phase and in the setting of disease progression, particularly when off-target resistance mechanisms or co-alterations are identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aee7b0dfd69142858e9dd93edb7ec143688a294" target='_blank'>
              EGFR plus MET targeted therapies for overcoming treatment resistance in EGFR mutant NSCLC. A Case Report.
              </a>
            </td>
          <td>
            Maria F Martínez-Hernandez, L. Lara-Mejía, C. Izquierdo-Tolosa, L. Cabrera-Miranda, Oscar Arrieta
          </td>
          <td>2024-09-18</td>
          <td>Oncology research and treatment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, S. Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 High-grade gliomas (HGGs) are the most aggressive type of gliomas and have the poorest outcomes. Chromatin remodeling (CR) genes have been implicated in multiple oncogenic pathways in numerous cancer types. In gliomagenesis, CR genes have been implicated in regulating stemness of glioma cells, the tumor microenvironment (TME), and resistance to therapies.



 We performed molecular profiling of 4244 HGGs and evaluated associations of CR mutations with other cancer related biomarkers, infiltration by immune cells, and immune gene expression. We also evaluated the association between CR mutations and survival in IDH WT HGG patients.



 Nearly 10% of HGGs carry mutations in CR genes, with higher prevalence (15%) in HGGs with IDH mutations. Analysis of co-occurrence with other biomarkers revealed that CR-mutated HGGs possess favorable genetic alterations which may have prognostic value. CR-mutated HGGs with wild type IDH demonstrated colder TME and worse OS overall compared to the CR-wild type HGGs.



 Our study reveals the prognostic effects of CR mutations in HGG and points to several biomarker candidates that could suggest sensitivity to emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a6ec214f04da421e41c3579984b7e7f5d8d94d" target='_blank'>
              Biological and prognostic relevance of epigenetic regulatory genes in high-grade gliomas (HGGs)
              </a>
            </td>
          <td>
            Sonikpreet Aulakh, Joanne Xiu, Andrew Hinton, S. Darabi, M. Demeure, Soma Sengupta, Santosh Kesari, David M Ashley, Ashley Love Sumrall, M. Glantz, D. Spetzler
          </td>
          <td>2024-10-04</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Synovial Sarcoma (SySa) is an aggressive soft tissue sarcoma that accounts for 5 – 10% of all soft tissue sarcomas. Current treatment involves radiation and radical surgery including limb amputation, highlighting the urgent need to develop targeted therapies. We reasoned that transcriptional rewiring by the fusion protein SS18-SSX, the sole oncogenic driver in SySa, creates specific vulnerabilities that can be exploited for treatment. To uncover genes that are selectively essential for SySa, we mined The Cancer Dependency Map (DepMap) data to identify genes that specifically impact the fitness of SySa compared to other tumor cell lines. Targeted CRISPR library screening of SySa-selective candidates revealed that the small ubiquitin-like modifier 2 (SUMO2) was one of the strongest dependencies both in vitro as well as in vivo. TAK-981, a clinical-stage small molecule SUMO2 inhibitor potently inhibited growth and colony-forming ability. Strikingly, transcriptomic studies showed that pharmacological SUMO2 inhibition with TAK-981 treatment elicited a profound reversal of a gene expression program orchestrated by SS18-SSX fusions. Of note, genetic or pharmacological SUMO2 inhibition reduced global and chromatin levels of the SS18-SSX fusion protein with a concomitant reduction in histone 2A lysine 119 ubiquitination (H2AK119ub), an epigenetic mark that plays an important role in SySa pathogenesis. Taken together, our studies identify SUMO2 as a novel, selective vulnerability in SySa. Since SUMO2 inhibitors are currently in Phase 1/2 clinical trials for other cancers, our findings present a novel avenue for targeted treatment of synovial sarcoma. SIGNIFICANCE Our study identifies SUMO2 as a selective dependency in synovial sarcoma. We demonstrate that the SUMO2/3 inhibitor TAK-981 impairs sarcomagenesis and reverses the SS18-SSX fusion-driven oncotranscriptome. Our study indicates that SUMO2 inhibition may be an attractive therapeutic option in synovial sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1863d8b2a5acaf7a4dc2a730005a178b4452ec" target='_blank'>
              SUMO2 Inhibition Reverses Aberrant Epigenetic Rewiring Driven by Synovial Sarcoma Fusion Oncoproteins and Impairs Sarcomagenesis
              </a>
            </td>
          <td>
            Rema Iyer, Anagha Deshpande, Aditi Pedgaonkar, Pramod Akula Bala, Taehee Kim, Gerard L. Brien, D. Finlay, K. Vuori, A. Soragni, Rabi Murad, Aniruddha J. Deshpande
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background: Genetic studies of ovarian cancer (OC) have historically focused on BRCA1/2 mutations, lacking other studies of homologous recombination repair (HRR). Poly (ADP-ribose) polymerase inhibitors (PARPi) exploit synthetic lethality to significantly improve OC treatment outcomes, especially in BRCA1/2 deficiency patients. Objectives: Our study aims to construct a mutation map of HRR genes in OC and identify factors influencing the efficacy of PARPi. Design: A retrospective observational analysis of HRR gene variation data from 695 OC patients from March 2019 to February 2022 was performed. Methods: The HRR gene variation data of 695 OC patients who underwent next-generation sequencing (NGS) in the First Affiliated Hospital of Zhengzhou University were retrospectively collected. Clinical data on the use of PARPi in these patients were also gathered to identify factors that may interfere with the efficacy of PARPi. Results: Out of 127 pathogenic variants in the BRCA1/2 genes, 104 (81.9%) were BRCA1 mutations, and 23 (18.1%) were BRCA2 mutations. Among the 59 variants of uncertain significance (VUS), 20 (33.9%) were BRCA1, while 39 (66.1%) were BRCA2 mutations. In addition to BRCA1/2, HRR gene results showed that 9 (69%) of 13 were HRR pathway pathogenic variants; and 16 (1.7%) of 116 VUS were Food and Drug Administration (FDA)-approved mutated HRR genes. Notably, the treatment regimen significantly influenced the effectiveness of PARPi, especially when using first-line maintenance therapy, leading to enhanced progression-free survival (PFS) compared to alternative protocols. Conclusion: Focusing on HRR gene mutations and supporting clinical research about PARPi in OC patients is crucial for developing precision treatment strategies and enhancing prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d41e2645926ac18cf48bd055b15a6838f0eb8b2d" target='_blank'>
              Genetic landscape of homologous recombination repair and practical outcomes of PARPi therapy in ovarian cancer management
              </a>
            </td>
          <td>
            Mengyu Chen, Ningjing Lei, Ruixia Guo, Liping Han, Qinghe Zhao, Yang Zhao, Luojie Qiu, Fengling Wu, Shan Jiang, Ningyao Tong, Kunmei Wang, Siyu Li, Lei Chang
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the frequent tumors that seriously endanger the physical and mental well-being in women with strong heterogeneity, and its pathogenesis involves multiple risk factors. Depending on the type of BC, hormonal therapy, targeted therapy, and immunotherapy are the current systemic treatment options along with conventional chemotherapy. Despite significant progress in understanding BC pathogenesis and therapeutic options, there is still a need to identify new therapeutic targets and develop more effective treatments. According to recent sequencing and profiling studies, non-coding (nc) RNAs genes are deregulated in human cancers via deletion, amplification, abnormal epigenetic, or transcriptional regulation, and similarly, the expression of many ncRNAs is altered in breast cancer cell lines and tissues. The ability of single ncRNAs to regulate the expression of multiple downstream gene targets and related pathways provides a theoretical basis for studying them for cancer therapeutic drug development and targeted delivery. Therefore, it is far-reaching to explore the role of ncRNAs in tumor development and their potential as therapeutic targets. Here, our review outlines the potential of two major ncRNAs, long non-coding RNAs (lncRNAs) and microRNAs (miRNAs) as diagnostic and prognostic biomarkers as well as targets for new therapeutic strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c75f06853600c6ed1247db1e2526cd7f187129" target='_blank'>
              Potential therapies for non-coding RNAs in breast cancer
              </a>
            </td>
          <td>
            Ruonan Li, Yuxin Ji, Ruyin Ye, Guohui Tang, Wenrui Wang, Changjie Chen, Qingling Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94914bd4d265668ea63fab9b4a1aa10d1231c91c" target='_blank'>
              Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling
              </a>
            </td>
          <td>
            Chen Weng, Jonathan S. Weissman, V. Sankaran
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Ganciclovir (GCV) is a clinically important drug as it is used to treat viral infections. GCV is incorporated into the DNA during replication, where it interferes with subsequent replication on GCV-incorporated templates. However, the effects of GCV on the host genome and the mechanisms underlying cellular tolerance to GCV remain unclear. In this study, we explored these mechanisms using a collection of mutant DT40 cells. We identified RAD17/-, BRCA1-/-, and RAD18-/- cells as highly GCV-sensitive. RAD17, a component of the alternative checkpoint-clamp loader RAD17-RFC, was required for the activation of the intra-S checkpoint following GCV treatment. BRCA1, a critical factor for promoting homologous recombination (HR), was required for suppressing DNA double-strand breaks (DSBs). Moreover, RAD18, an E3-ligase involved in DNA repair, was critical in suppressing the aberrant ligation of broken chromosomes caused by GCV. We found that BRCA1 suppresses DSBs through HR-mediated repair and template switching (TS)-mediated damage bypass. Moreover, the strong GCV sensitivity of BRCA1-/- cells was rescued by the loss of 53BP1, despite the only partial restoration in the sister chromatid exchange events which are hallmarks of HR. These results indicate that BRCA1 promotes cellular tolerance to GCV through two mechanisms, TS and HR-mediated repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb90975f7c5cd2ebcb118dd3d319f550cae4f8b1" target='_blank'>
              RAD18- and BRCA1-dependent pathways promote cellular tolerance to the nucleoside analog ganciclovir.
              </a>
            </td>
          <td>
            Tasnim Ahmad, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2024-08-22</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The transcriptional state of a cell provides a multi-parameter representation of gene expression programs that reflect its identity and phenotype. However, current transcriptomic profiling technologies result in destruction of the biological sample, preventing direct analysis of transcriptional dynamics in the same living cells over time. Here, we developed a synthetic RNA export system called cellular ‘self-reporting’ to address this fundamental technological limitation. We repurposed the murine leukemia virus retroviral protein Gag to enable diverse types of immortalized and primary mammalian cells to package cellular RNA molecules in virus-like particles (VLPs) for export into the extracellular environment. We applied self-reporting to interrogate the transcriptome-wide dynamics that occur during neuronal differentiation from induced pluripotent stem cells and detected gene expression profiles from individual live cells. Leveraging this genetically encodable approach, we expanded the capabilities of self-reporting through molecular engineering of VLP components. Pseudotyping VLPs with epitope-tagged envelope proteins enabled multiplexed selective live-cell readout of transcriptional states from heterogeneous co-cultures. Furthermore, structure-guided protein engineering of Gag fusions with human RNA binding domains improved the mRNA representation in self-reporting readouts and enabled the directed export of libraries of synthetic barcode transcripts. Taken together, this work establishes self-reporting as a facile and broadly enabling technology for live-cell, transcriptome-scale profiling of dynamic processes across diverse cell types and biological applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd9363ab1dcbb456eb0b3e0944d84bfdb375d0c" target='_blank'>
              Live-cell transcriptomics with engineered virus-like particles
              </a>
            </td>
          <td>
            M. Najia, Jacob Borrajo, Anna Le, FuNien Tsai, Jeremy Y. Huang, Linda G. Griffith, George Q Daley, P. Blainey
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Alterations in Fibroblast growth factor receptor (FGFR)-family proteins frequently occur as oncogenes in many cancers, including a subset of pediatric gliomas. Here, we performed a genomic analysis of 11,635 gliomas across ages and found that 4.5% of all gliomas harbor FGFR alterations including structural variants (SV) and single nucleotide variants (SNV), with an incidence of almost 10% in pediatric gliomas. FGFR family members are differentially enriched by age, tumor grade, and histological subtype, with FGFR1-alterations associated with glioneuronal histologies and pediatric low-grade gliomas. Across development, we find FGFR1 expression in both neuronal and glial precursors, while FGFR3 expression is largely restricted to astrocytic lineages. Leveraging novel isogenic model systems, we confirm FGFR1 alterations to be sufficient to activate MAPK and mTOR signaling, drive gliomagenesis, activate neuronal transcriptional programs and exhibit sensitivity to MAPK pathway inhibitors, including pan-FGFR inhibitors. Models driven by FGFR1 SVs exhibited different patterns of sensitivity compared to those driven by SNVs. Finally, we performed a retrospective analysis of clinical responses in children diagnosed with FGFR-driven gliomas and found that targeted MAPK or FGFR-inhibition with currently available inhibitors is largely associated with stability of disease. This study provides key insights into the biology of FGFR1-altered gliomas, therapeutic strategies to target them and associated challenges that still need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc242f4fb57f142e32352b142311d05135cc1f" target='_blank'>
              A diverse landscape of FGFR alterations and co-mutations defines novel therapeutic strategies in pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Eric Morin, April Apfelbaum, D. Sturm, Georges Ayoub, Jeromy J. Digiacomo, Sher Bahadur, Bhavyaa Chandarana, Phoebe C Power, Margaret Cusick, Dana Novikov, Robert Jones, J. Vogelzang, Connor Bossi, Seth W. Malinowski, John Jeang, Jared Collins, Sehee Oh, Hyesung Jeon, A. Cameron, Patrick Rechter, Angela Deleon, K. Murugesan, M. Montesion, L. Albacker, S. Ramkissoon, C. M. Tilburg, Emily C Hardin, P. Sievers, Felix Sahm, K. Yeo, Tom Rosenberg, Susan N Chi, Karen D Wright, S. Hébert, Sydney Peck, A. Picca, Valérie Larouche, Samuele Renzi, T. Bale, Amy A. Smith, M. Touat, Nada Jabado, Eric S Fischer, Michael Eck, Lissa Baird, Olaf Witt, Claudia L. Kleinman, Quang-De Nguyen, S. Alexandrescu, D. Jones, Keith L. Ligon, P. Bandopadhayay
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a therapeutically challenging malignancy notorious for its resilience to therapeutic strategies and ability to thrive under extreme conditions. The cancer's inherent ability to adaptively resist treatments underscores the need for novel and combinatorial therapeutic approaches. Our research focuses on elucidating the role of Apurinic-apyrimidinic endonuclease/Redox effector factor 1 (APE1/Ref-1), a critical multifunctional protein in PDAC pathobiology. APE1 serves two essential functions: as the principal endonuclease in the base excision repair (BER) pathway for DNA repair and as a redox regulator of transcription factors essential for cancer cell proliferation. While the therapeutic targeting of APE1’s redox function has been effective and continues to yield significant advances, the development of treatments aimed at its DNA repair function has proven more challenging with comparatively limited progress. Since standard PDAC treatment regimens involve DNA damaging agents potentially repaired via BER and APE1, we engineered three mutant cell lines with knock-in mutations resulting in a reduction of APE1’s BER activity while maintaining full redox activity. Biochemical evaluations confirmed the effects of the mutation, revealing a 30-fold reduction in DNA repair capacity. Surprisingly, with respect to growth and proliferation, the mutant and wild-type cell lines behaved similarly. Analyses of cytotoxic vulnerabilities revealed similar findings with the mutant lines exhibiting minimal phenotypic differences in sensitivity when challenged with cytotoxic and DNA damaging agents in proliferation-based assays. These findings led us to posit that the mutant cell lines were relying on compensatory DNA repair pathways which were providing sufficient compensation for the decrease in APE1 activity. We further hypothesized that the cell lines may still have hidden, long-term deficiencies in DNA damage processing which are not apparent during the initial stress response. Subsequent colony formation assays revealed that the mutant cell lines do exhibit poor long-term survival even after only short-term exposure to DNA damage. In vivo orthotopic mouse model experiments further corroborated these findings, showing a significant reduction in tumor size and decreased metastasis to the lungs and liver in mutant cell lines compared to wild-type controls. While APE1 is the primary endonuclease in the BER pathway, studies have demonstrated APE1-independent BER in rare, specific contexts. We are probing the hypothesis that these APE1-independent BER pathways function as the compensatory mechanism safeguarding these cells in short-term stress situations. Identifying and characterizing these backup pathways will not only offer a better understanding of APE1’s DNA repair capabilities within PDAC but will also provide a better understanding of how DNA repair mechanisms can be therapeutically targeted for novel therapeutic options or combinations. This opens the path to enhance standard of care treatments such as FOLFIRINOX.
 Citation Format: Eyram K Kpenu, Randall Wireman, Mahmut Mijiti, Silpa Gampala, Melissa L Fishel, Mark R Kelley. Challenging PDAC adaptability: Uncovering therapeutic vulnerabilities in APE1 DNA repair mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45264b7cf12c7d27e58987210445d6c2b0d4261" target='_blank'>
              Abstract C028: Challenging PDAC adaptability: Uncovering therapeutic vulnerabilities in APE1 DNA repair mechanisms
              </a>
            </td>
          <td>
            E. Kpenu, R. Wireman, Mahmut Mijiti, S. Gampala, M. Fishel, Mark R. Kelley
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) presents a formidable clinical challenge owing to its aggressive progression and resistance to conventional therapies. A key driver of NEPC is the overexpression of MYCN, a well-established oncogene associated with neuroendocrine tumors. However, efforts to directly inhibit the N-Myc protein encoded by this gene have resulted in limited success, thereby hindering therapeutic advancements. To overcome this obstacle, we conducted unbiased genome-wide screening using isogenic prostate cancer cell lines to identify the synthetic vulnerabilities of MYCN. Among the identified candidates, NEUROG2 emerged as a significant candidate. Neurog2 is a proneural transcription factor (PTF) known for its role in developmental processes and trans-differentiation of adult cells. Our findings demonstrate that Neurog2 depletion does not affect non-malignant cells, but significantly suppresses the growth of MYCN-overexpressing cells and tumors in orthotopic NEPC models. Furthermore, our observations indicate that the Neurog2-mediated regulation of PTFs can facilitate NEPC development. Thus, targeting Neurog2 holds promise as an effective therapeutic strategy for MYCN-overexpressing NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6178b90a3b078d38756c518ec592d0bd4f1191a1" target='_blank'>
              A novel role for Neurog2 in MYCN driven neuroendocrine plasticity of prostate cancer
              </a>
            </td>
          <td>
            Prachi Walke, Jared D.W. Price, Frederick S. Vizeacoumar, Nickson Joseph, Vincent Maranda, Bari Chowdhury, Jay Patel, Yue Zhang, He Dong, Lara New, Ashtalakshmi Ganapathysamy, Lihui Gong, Hussain Elhasasna, Kalpana K Bhanumathy, Yuliang Wu, A. Freywald, Anand Krishnan, Franco J Vizeacoumar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) poses significant challenges in veterinary medicine, with limited treatment options and poor survival rates. While substantial progress has been made in characterizing human AML, translating these advancements to veterinary practice has been hindered by limited molecular understanding and diagnostic tools. The case study presented illustrates the application of whole genome sequencing in diagnosing AML in a dog, showcasing its potential in veterinary oncology. Our approach facilitated comprehensive genomic analysis, identifying mutations in genes that may be associated with AML pathogenesis in dogs, such as KRAS, IKZF1, and RUNX1. However, without supportive evidence of its clinical utility (eg, association with response to treatment or prognosis), the information is limited to exploration. This article reviews the comparative features of canine AML with human AML and discusses strategies to shrink the knowledge gap between human and veterinary medicine with cost-effective next-generation sequencing (NGS) techniques. By utilizing these approaches, the unique and shared molecular features with human AML can be identified, aiding in molecular classification and therapeutic development for both species. Despite the promise of NGS, challenges exist in implementing it into routine veterinary diagnostics. Cost considerations, turnaround times, and the need for robust bioinformatics pipelines and quality control measures must be addressed. Most importantly, analytical and clinical validation processes are essential to ensure the reliability and clinical utility of NGS-based assays. Overall, integrating NGS technologies into veterinary oncology holds great potential for advancing our understanding of AML and improving disease stratification, in hopes of improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8bd6fdd9b3631dd5e67d1617f989917515a9291" target='_blank'>
              Advancements in genetic analysis: Insights from a case study and review of next-generation sequencing techniques for veterinary oncology applications.
              </a>
            </td>
          <td>
            R. Harris, Jillian Nolan, Dylan Ammons, Samantha Beeson, Douglas Thamm, Anne Avery
          </td>
          <td>2024-10-04</td>
          <td>Veterinary clinical pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA repair genomic aberrations in the Homologous Recombination pathway are identifiable in up to 25% of patients with advanced prostate cancer, making them more likely to benefit from treatment with poly (ADP-ribose) polymerase inhibitors (PARPi) alone or in combination with other therapies, particularly when BRCA driver genomic aberrations are documented. Although several clinical trials have demonstrated the efficacy of this approach, the validation of reliable biomarkers predictive of response still needs further improvement to refine patient selection. In this setting, the characterization of resistance mechanisms and the validation of novel biomarkers are critical to maximize clinical benefit and to develop novel treatment combinations to improve outcomes. In this review, we summarize the development of PARPi in prostate cancer as single agent as well as the efficacy of their combination with other drugs, and the future directions for their implementation in the management of advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332cf66eb0d0e0669852e95223ac366cb113738f" target='_blank'>
              PARP inhibitors alone or in combination for prostate cancer
              </a>
            </td>
          <td>
            M. D. Fenor de la Maza, J. L. Pérez Gracia, B. Miñana, Elena Castro
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Urology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, J. Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f2554beac8f2504f8636ae14089de94d4df8bc" target='_blank'>
              Deciphering the impact of aggregated autophagy-related genes TUBA1B and HSP90AA1 on colorectal cancer evolution: a single-cell sequencing study of the tumor microenvironment
              </a>
            </td>
          <td>
            Qianping Xu, Chao Liu, Hailin Wang, Shujuan Li, Hanshen Yan, Ziyang Liu, Kexin Chen, Yaoqin Xu, Runqin Yang, Jingfang Zhou, Xiaolin Yang, Jie Liu, Lexin Wang
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The use and publication of research utilising CRISPR/cas9 in gene editing and knock outs (KOs) within cell lines is now widespread and has proved extremely powerful in the interrogation of gene function. However, the potential of experimental artefacts resulting from the need to generate single cell clones post gene-manipulation has been somewhat overlooked. In this study, we show that the commonly used pulmonary cell line A549 displays significant heterogeneity in terms of their gene expression, and that individual cells from a population exhibit dramatically different susceptibility to viral infection from a range of viruses including respiratory syncytial virus (RSV), influenza A virus (IAV), Hazara virus (HAZV), lymphocytic choriomeningitis virus (LCMV) and Bunyamwera virus (BUNV). Moreover, we demonstrate the rapid re-emergence of phenotypic heterogeneity even after cloning. These results demonstrate the need for caution in interpreting results from CRISPR screens and CRISPR-KO validation studies, especially in the study of viral infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0e26c9049ed60692f4658ae9936a4fcc5067296" target='_blank'>
              Single cell variation and rapid emergence of phenotypic heterogeneity in cell lines- a cautionary tale for devotees of CRISPR-Cas9
              </a>
            </td>
          <td>
            Hayley Bradley, John N Barr, Martin Stacey
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic reinforcement of T cell exhaustion is known to be a major barrier limiting T cell responses during immunotherapy. However, the core epigenetic regulators restricting antitumor immunity during prolonged antigen exposure are not clear. We investigated three commonly mutated epigenetic regulators that promote clonal hematopoiesis to determine whether they affect T cell stemness and response to checkpoint blockade immunotherapy. CD8 T cells lacking Dnmt3a, Tet2, or Asxl1 preserved a progenitor-exhausted (Tpex) population for more than 1 year during chronic antigen exposure without undergoing malignant transformation. Asxl1 controlled the self-renewal capacity of T cells and reduced CD8 T cell differentiation through H2AK119 ubiquitination and epigenetic modification of the polycomb group–repressive deubiquitinase pathway. Asxl1-deficient T cells synergized with anti–PD-L1 immunotherapy to improve tumor control in experimental models and conferred a survival advantage to mutated T cells from treated patients. Editor’s summary T cell immunotherapies have been successful in treating some intractable cancers, but there are limitations caused by the inability of T cells to persist after prolonged stimulation. Kang et al. undertook a reverse-translation approach to investigate the molecular mechanisms regulating the durability of T cell responsiveness to checkpoint blockade immunotherapy (see the Perspective by Tsui and Kallies). Inspired by clinical observations from individuals with myelodysplastic syndrome, the epigenetic regulators (Dnmt3a, Tet2, and Asxl1) associated with clonal hematopoiesis were identified using experimental models of T cell exhaustion. Epigenetic disruption of the polycomb group repressive deubiquitinase complex resulted in preservation of a stem cell–like T cell population that was responsive to immunotherapy. Extending this mechanism to cancer adoptive cell therapy, Asxl1 disruption was found to endow T cells with superior therapeutic efficacy and an ability to synergize with checkpoint blockade immunotherapy —Priscilla N. Kelly INTRODUCTION Sustained stimulation of CD8 T cells promotes the development of functional exhaustion, a terminal cell fate that has been broadly recognized to limit the durability of T cell–based immunotherapies. This state of exhaustion arises though a progressive transition of multipotent progenitor T cells (Tpex) into a terminally differentiated population (Tex) that has been defined as being nonresponsive to immune checkpoint blockade (ICB). The molecular determinants dictating the developmental transition are now being investigated as engineering targets to block the progression of T cell exhaustion and preserve the durability of immunotherapy approaches. RATIONALE We recently observed that the long-term survival of a small cohort of myelodysplastic syndrome (MDS) patients treated with anti–PD-L1 therapy was coupled to mutation of the ASXL1 gene among their T cells. Mutations in ASXL1, in addition to DNMT3A and TET2, are commonly associated with conferring a survival advantage among hematopoietic stem cells, resulting in subclonal outgrowth referred to as clonal hematopoiesis. Because of their general link to cellular stemness, we investigated the role of these regulators in the developmental transition of Tpex into Tex and the impact of deletion of these regulators on therapeutic durability during anti–PD-L1 treatment. RESULTS To investigate the role of Dnmt3a, Tet2, and Asxl1 in the development and maintenance of an ICB-responsive T cell population, we engineered T cells to contain mutations within each of these genes and then exposed them to a chronic source of antigen by adoptively transferring them into mice infected with the clone 13 strain of lymphocytic choriomeningitis virus (LCMV). Disruption of these regulators in a canonical model of T cell exhaustion allowed them to maintain their quantity and ICB-responsive state for >1 year during chronic antigen exposure. Despite being stimulated and undergoing antigen-driven proliferation for 1 year, these T cells did not exhibit any signs of malignant outgrowth. Moreover, the total pool of T cells containing each of these mutations was enriched for a large quantity of stem-like Tcf1+ T cells. Fate-tracking adoptive transfer experiments revealed that this Tcf1+ subset of ASXL1 KO T cells could both self-renew and give rise to potent effectors. Transcriptional and epigenetic profiling of the knockout (KO) T cells identified Asxl1 as a regulator of the polycomb group–repressive deubiquitinase (PR-DUB) complex, controlling the deubiquitination of histone 2A lysine 119, which serves as a molecular checkpoint for the developmental transition of Tpex into Tex. Extension of this mechanistic insight into tumor models revealed that adoptively transferred T cells with Asxl1 disruption resisted tumor microenvironment–induced exhaustion. Moreover, the adoptively transferred Asxl1 KO T cells synergized with anti–PD-L1 blockade, resulting in heightened tumor control. Finally, we show that Asxl1 disruption of tumor-specific T cells correlates with increased expansion of T cells in MDS patients after treatment with anti–PD-L1. CONCLUSION Results from these studies show that DNMT3A, TET2, and ASXL1 control the developmental checkpoint between Tpex and terminally exhausted T cells. Specific investigation into Asxl1 revealed its role in T cell differentiation through epigenetic modification of the PR-DUB pathway. These data articulate a reverse-translation approach for defining molecular mechanisms that restrict maintenance of the ICB-responsive stem-like T cells within a suppressive tumor microenvironment. Through our mechanistic investigation of genes associated with patient survival, we have established a scientific rationale for specific genetic modification of discrete epigenetic regulators for future engineering to enhance the durability of cancer immunotherapy. Core epigenetic regulators in maintaining therapeutic durable multipotent Tpex populations. Using the canonical murine model system of exhaustion, disruption of Dnmt3a, Tet2, and Asxl1 in CD8 T cells preserved an ICB-responsive Tpex population for >1 year during chronic antigen exposure. Specific investigation into the new epigenetic regulator Asxl1 revealed that this unprecedented maintenance of Tpex is mediated by presence of monoubiqutinated histone 2A lysine 119 (H2AK119Ub) through disruption of the PR-DUB complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e5351a39641c751005cc32a06480cddb05203" target='_blank'>
              Epigenetic regulators of clonal hematopoiesis control CD8 T cell stemness during immunotherapy
              </a>
            </td>
          <td>
            Tae Gun Kang, Xin Lan, Tian Mi, Hongfeng Chen, Shanta Alli, Song-Eun Lim, Sheetal Bhatara, A. Vasandan, Grace Ward, Sofia Bentivegna, Josh Jang, Marianne L. Spatz, Jin-Hwan Han, B. Schlotmann, J. S. Jespersen, Christopher Derenzo, Peter Vogel, Jiyang Yu, Stephen B Baylin, Peter Jones, Casey O’Connell, Kirsten Grønbæk, Ben Youngblood, Caitlin C. Zebley
          </td>
          <td>2024-10-11</td>
          <td>Science</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09486473d5ae00655afa329978bc6a48b65d161f" target='_blank'>
              The response to influenza vaccination is associated with DNA methylation-driven regulation of T cell innate antiviral pathways
              </a>
            </td>
          <td>
            Hongxiang Fu, H. Pickering, L. Rubbi, Ted M. Ross, Wanding Zhou, Elaine F. Reed, Matteo Pellegrini
          </td>
          <td>2024-08-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Epigenetics mechanisms play a significant role in human diseases by altering DNA methylation status, chromatin structure, and/or modifying histone proteins. By modulating the epigenetic status, the expression of genes can be regulated without any change in the DNA sequence itself. Epigenetic drugs exhibit promising therapeutic efficacy against several epigenetically originated diseases including several cancers, neurodegenerative diseases, metabolic disorders, cardiovascular disorders, and so forth. Currently, a considerable amount of research is focused on discovering new drug molecules to combat the existing research gap in epigenetic drug therapy. A novel and efficient delivery system can be established as a promising approach to overcome the drawbacks associated with the current epigenetic modulators. Therefore, formulating the existing epigenetic drugs with distinct encapsulation strategies in nanocarriers, including solid lipid nanoparticles, nanogels, bio‐engineered nanocarriers, liposomes, surface modified nanoparticles, and polymer–drug conjugates have been examined for therapeutic efficacy. Nonetheless, several epigenetic modulators are untouched for their therapeutic potential through different delivery strategies. This review provides a comprehensive up to date discussion on the research findings of various epigenetics mechanism, epigenetic modulators, and delivery strategies utilized to improve their therapeutic outcome. Furthermore, this review also highlights the recently emerged CRISPR tool for epigenome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a8af65ee273c5b2755ef2a82edb72c808ec6577" target='_blank'>
              Targeting the epigenome with advanced delivery strategies for epigenetic modulators
              </a>
            </td>
          <td>
            Sonia Guha, Yogeswaran Jagadeesan, M. M. Pandey, A. Mittal, Deepak Chitkara
          </td>
          <td>2024-08-17</td>
          <td>Bioengineering &amp; Translational Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E. Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small population of less differentiated cells with robust self-renewal ability. CSCs have been recognized as the root cause of tumor initiation, progression, relapse, and drug resistance. Recent studies from us and others have highlighted that N6-methyladenosine (m6A), the most prevalent modification in mRNA, plays a crucial role in carcinogenesis and CSC homeostasis. Dysregulation of the m6A modification machinery has been implicated in CSC survival and self-renewal, thereby regulating cancer progression and therapeutic resistance. In this review, we provide an overview of the roles and molecular mechanisms of the RNA m6A modification machinery in CSC survival and self-renewal. Additionally, we summarize the currently known small-molecule inhibitors targeting the dysregulated m6A modification machinery and discuss proof-of-concept studies focusing on the efficacy of these compounds in eliminating CSCs and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a2c463331686ce335a111580d49921b4543bf" target='_blank'>
              Combating cancer stem cells: RNA m6A methylation and small-molecule drug discovery
              </a>
            </td>
          <td>
            Honghai Zhang, Xueer Wang, Jianjun Chen, R. Su
          </td>
          <td>2024-09-30</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Background: Dynamic regulation of gene expression is fundamental for cellular adaptation to exogenous stressors. PTEFb-mediated promoter proximal pause-release of Pol II is a conserved regulatory mechanism for synchronous transcriptional induction best described in response to heat shock, but this pro-survival role has not been examined in the applied context of cancer therapy. Design/Method: In order to examine the dynamics of chromatin reorganization following radiotherapy, we performed a combination of ChIP-, ATAC-, and RNA-seq in model systems of diffuse intrinsic pontine glioma (DIPG) and other pediatric high-grade gliomas (pHGG) following IR exposure. We interrogated IR-induced gene expression in the presence or absence of PTEFb blockade, including both mechanistic and functional consequences of concurrent inhibition or genetic depletion. We utilized culture models with live cell imaging to assess the therapeutic synergy of PTEFb inhibition with IR, as well as the therapeutic index of this intervention relative to normal controls. Finally, we employed orthotopic models of pHGG treated with conformal radiotherapy and CNS-penetrant PTEFb inhibitors in order to assess tolerability and anti-tumor effect in vivo. Results: Rapid genome-wide redistribution of active chromatin features and PTEFb facilitates Pol II pause-release to drive nascent transcriptional induction within hours of exposure to therapeutic ionizing radiation. Concurrent inhibition of PTEFb imparts a transcription elongation defect, abrogating canonical adaptive programs such as DNA damage repair and cell cycle regulation. This combination demonstrates a potent, synergistic therapeutic potential agnostic of glioma subtype, leading to a marked induction of tumor cell apoptosis and prolongation of xenograft survival.Conclusion: These studies reveal a central role for PTEFb underpinning the early adaptive response to radiotherapy, opening new avenues for combinatorial treatment in these lethal malignancies.
 Citation Format: Faye M. Walker, Lays Martin Sobral, Etienne Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela Pierce, Sana Karam, Natalie J. Serkova, Nicholas K. Foreman, Sujatha Venkataraman, Robin Dowell, Rajeev Vibhakar, Nathan A. Dahl. Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cbc389c63c005639d0ccef6f75bb33d60a2a03" target='_blank'>
              Abstract B078: Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy
              </a>
            </td>
          <td>
            Faye Walker, L. M. Sobral, Etienne P. Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Sana D Karam, Natalie J Serkova, Nicholas K. Foreman, S. Venkataraman, Robin Dowell, Rajeev Vibhakar, N. Dahl
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Despite the success of CRISPR/Cas9 in inducing DNA double-strand breaks (DSBs) for genome editing, achieving targeted recombination in somatic cells remains challenging, particularly at recombination cold spots like the Tomato Mosaic Virus (ToMV) resistance locus in Solanum lycopersicum. We investigated the potential of CRISPR/Cas9-induced targeted recombination in somatic cells to overcome linkage drag surrounding the ToMV locus. We employed two strategies: first, inducing DSBs in both alleles of F1 tomato seedlings to promote non-homologous end joining (NHEJ) and homology-directed repair (HDR); second, targeting a single allele in a heterozygous background to induce HDR in seedlings. CRISPR/Cas9 activity was confirmed in F₁ seedlings by detecting NHEJ-mediated mutations at the target sites in ToMV. We developed a bioinformatics pipeline to identify targeted recombinants by analyzing single nucleotide polymorphisms (SNPs) between parental haplotypes, allowing precise tracking of SNP variations. A two-dimensional pooling strategy was employed to distinguish genuine recombination events from PCR artifacts. Despite these advances and the active CRISPR/Cas9 system in F1 progeny, no increase in recombination frequency was observed compared to wild-type plants. We extended our research to protoplasts to assess whether CRISPR/Cas9 could induce targeted recombination under different cellular conditions at the same locus. Consistent with our findings in F1 plants, we observed no increase in recombinant patterns compared to wild-type controls in protoplasts. Our findings suggest that CRISPR/Cas9-induced DSBs are insufficient to break the genetic linkage at the ToMV locus on chromosome 9 in recombination cold spots within somatic cells. Article Summary This research targets plant biologists and geneticists interested in enhancing plant breeding techniques. The study used CRISPR/Cas9 technology to induce DNA breaks in tomato plants. It specifically targeted the Tomato Mosaic Virus (ToMV) resistance gene, which resists natural recombination. The aim was to induce genetic recombination via CRISPR/Cas9. The highly active CRISPR/Cas9 system did not increase the expected genetic changes, indicating challenges in achieving targeted recombination. These findings highlight the challenges in breaking genetic linkages in specific genome regions using current CRISPR methods. These findings are relevant for developing techniques for targeted recombination in plant breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0b58aaaa05e1e2ef0cf4dbd56fab2aa20c1e65" target='_blank'>
              CRISPR/Cas9-induced breaks are insufficient to break linkage drag surrounding the ToMV locus of Solanum lycopersicum
              </a>
            </td>
          <td>
            Jillis Grubben, Gerard Bijsterbosch, Burak Aktürk, Richard G.F. Visser, Henk J. Schouten
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic dysfunction-associated Fatty Liver Disease (MAFLD) has emerged as one of the leading cardiometabolic diseases. Friend of GATA2 (FOG2) is a transcriptional co-regulator that has been shown to regulate hepatic lipid metabolism and accumulation. Using meta-analysis from several different biobank datasets, we identified a coding variant of FOG2 (rs28374544, A1969G, S657G) predominantly found in individuals of African ancestry (minor allele frequency~20%), which is associated with liver failure/cirrhosis phenotype and liver injury. To gain insight into potential pathways associated with this variant, we interrogated a previously published genomics dataset of 38 human induced pluripotent stem cell (iPSCs) lines differentiated into hepatocytes (iHeps). Using Differential Gene Expression Analysis and Gene Set Enrichment Analysis, we identified the mTORC1 pathway as differentially regulated between iHeps from individuals with and without the variant. Transient lipid-based transfections were performed on the human hepatoma cell line (Huh7) using wild-type FOG2 and FOG2S657G and demonstrated that FOG2S657G increased mTORC1 signaling, de novo lipogenesis, and cellular triglyceride synthesis and mass. In addition, we observed a significant downregulation of oxidative phosphorylation in FOG2S657G cells in fatty acid-loaded cells but not untreated cells, suggesting that FOG2S657G may also reduce fatty acid to promote lipid accumulation. Taken together, our multi-pronged approach suggests a model whereby the FOG2S657G may promote MAFLD through mTORC1 activation, increased de novo lipogenesis, and lipid accumulation. Our results provide insights into the molecular mechanisms by which FOG2S657G may affect the complex molecular landscape underlying MAFLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2945851076c2f51a928c3f7a7537ad4ec5a83e4e" target='_blank'>
              Genotype-First Approach Identifies an Association between rs28374544/FOG2S657G and Liver Disease through Alterations in mTORC1 Signaling
              </a>
            </td>
          <td>
            Donna Conlon, Siri Kanakala, Tess A. Cherlin, Yi-An Ko, Cecilia Vitali, Sharavana Gurunathan, Rasika Venkatesh, Jakob Woerner, Lindsay Guare, Penn Medicine Biobank, Anurag Verma, S. Verma, Marie A. Guerraty
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e692b2096b801074500379b186d120038a0326ab" target='_blank'>
              Chromosome 9p trisomy increases stem cells clonogenic potential and fosters T-cell exhaustion in JAK2-mutant myeloproliferative neoplasms
              </a>
            </td>
          <td>
            C. Carretta, S. Parenti, Matteo Bertesi, S. Rontauroli, Filippo Badii, L. Tavernari, E. Genovese, Marica Malerba, Elisa Papa, S. Sperduti, E. Enzo, M. Mirabile, Francesca Pedrazzi, Anita Neroni, Camilla Tombari, Barbara Mora, M. Maffioli, Marco Mondini, M. Brociner, M. Maccaferri, E. Tenedini, Silvia Martinelli, N. Bartalucci, E. Bianchi, L. Casarini, L. Potenza, Mario Luppi, Enrico Tagliafico, P. Guglielmelli, Manuela Simoni, Francesco Passamonti, Ruggiero Norfo, A. Vannucchi, R. Manfredini
          </td>
          <td>2024-08-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Direct lineage reprogramming into dopaminergic (DA) neurons holds great promise for the more effective production of DA neurons, offering potential therapeutic benefits for conditions such as Parkinson's disease. However, the reprogramming pathway for fully reprogrammed DA neurons remains largely unclear, resulting in immature and dead-end states with low efficiency. In this study, using single-cell RNA sequencing, the trajectory of reprogramming DA neurons at multiple time points, identifying a continuous pathway for their reprogramming is analyzed. It is identified that intermediate cell populations are crucial for resetting host cell fate during early DA neuronal reprogramming. Further, longitudinal dissection uncovered two distinct trajectories: one leading to successful reprogramming and the other to a dead end. Notably, Arid4b, a histone modifier, as a crucial regulator at this branch point, essential for the successful trajectory and acquisition of mature dopaminergic neuronal identity is identified. Consistently, overexpressing Arid4b in the DA neuronal reprogramming process increases the yield of iDA neurons and effectively reverses the disease phenotypes observed in the PD mouse brain. Thus, gaining insights into the cellular trajectory holds significant importance for devising regenerative medicine strategies, particularly in the context of addressing neurodegenerative disorders like Parkinson's disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bfa61a76443945c179bc5c94c72de94ea5c5df" target='_blank'>
              Epigenetic Dynamics in Reprogramming to Dopaminergic Neurons for Parkinson's Disease.
              </a>
            </td>
          <td>
            Byounggook Cho, Junyeop Kim, Sumin Kim, Saemin An, Yerim Hwang, Yunkyung Kim, Daeyeol Kwon, Jongpil Kim
          </td>
          <td>2024-09-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genomic structural variants (SVs) play a pivotal role in driving the evolution of hematologic malignancies, particularly in leukemia, in which genetic abnormalities are crucial features. Detecting SVs is essential for achieving precise diagnosis and prognosis in these cases. Karyotyping, often complemented by fluorescence in situ hybridization and/or chromosomal microarray analysis, provides standard diagnostic outcomes for various types of SVs in front-line testing for leukemia. Recently, optical genome mapping (OGM) has emerged as a promising technique due to its ability to detect all SVs identified by other cytogenetic methods within one single assay. Furthermore, OGM has revealed additional clinically significant SVs in various clinical laboratories, underscoring its considerable potential for enhancing front-line testing in cases of leukemia. This review aims to elucidate the principles of conventional cytogenetic techniques and OGM, with a focus on the technical performance of OGM and its applications in diagnosing and prognosticating myelodysplastic syndromes, acute myeloid leukemia, acute lymphoblastic leukemia, and chronic lymphocytic leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7daafa70669857dc79dda86074368e05463db4df" target='_blank'>
              Genomic structural variants analysis in leukemia by a novel cytogenetic technique: Optical genome mapping.
              </a>
            </td>
          <td>
            Song Lu, Kefu Liu, Di Wang, Yuan Ye, Zhiping Jiang, Yunhua Gao
          </td>
          <td>2024-08-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background/Objectives: Irinotecan, a camptothecin (CPT) derivative, is commonly used as a first-line therapy for colorectal cancer (CRC), but resistance remains a significant challenge. This study aims to explore the therapeutic potential of FL118, another CPT derivative, with a focus on overcoming resistance to irinotecan. Methods: The effects of FL118 on CRC cells were evaluated, and bioinformatics analysis was performed on RNA-seq data. Transfection was conducted to observe the knockdown effect of survivin, and the in vivo efficacy of FL118 was assessed using a xenograft model. Results: FL118 induces apoptosis, G2/M arrest, and DNA damage. A notable mechanism of action of FL118 is a reduction in survivin levels, which downregulates the expression of RAD51, a key marker of homologous recombination, and attenuates DNA repair processes. Given that SN38 is the active metabolite of irinotecan, FL118 reduces cell viability and RAD51 in SN38-resistant LOVO cells. Conclusions: Our findings provide effective insights into the antitumor activity of FL118 and its potential as a therapeutic agent for overcoming irinotecan resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d19093f191d9d1383ff5d9089dfc1860caa3a82" target='_blank'>
              FL118 Enhances Therapeutic Efficacy in Colorectal Cancer by Inhibiting the Homologous Recombination Repair Pathway through Survivin–RAD51 Downregulation
              </a>
            </td>
          <td>
            Jungyoun Kim, Yeyeong Jeong, Youkeun Shin, Sung Eun Kim, Sang Joon Shin
          </td>
          <td>2024-10-03</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION
Kawasaki disease [KD] is a systemic disorder characterized by acute febrile illness due to widespread medium-vessel vasculitis, mainly affecting children. Despite the ongoing advanced research into the disease pathophysiology and molecular mechanisms, the exact etiopathogenesis of KD is still an enigma. Recently, single-cell RNA sequencing [scRNA-seq], has been utilized to elucidate the pathophysiology of KD at a resolution higher than that of previous methods.


AREA COVERED
In the present article, we re-emphasize the pivotal role of this high-resolution technique, scRNA-seq, in the characterization of immune cell transcriptomic profile and signaling/response pathways in KD and explore the diagnostic, prognostic, and therapeutic potential of this new technique in KD. Using combinations of the search phrases 'KD, scRNA-seq, CAA, childhood vasculitis' a literature search was carried out on Scopus, Google Scholar, PubMed until the beginning of 2024.


EXPERT OPINION
scRNA-seq presents a transformative tool for dissecting KD at the cellular level. By revealing rare cell populations, gene expression alterations, and disease-specific pathways, scRNA-seq aids in understanding the intricacies of KD pathogenesis. This review will provide new insights into pathogenesis of KD and the field of applications of single-cell RNA sequencing in personalized therapeutics for KD in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c5f1bc26b6bf664c7b6e2c9eba580048b46b219" target='_blank'>
              Single-cell RNA sequencing: an emerging tool revealing dysregulated innate and adaptive immune response at single cell level in kawasaki disease.
              </a>
            </td>
          <td>
            Saniya Sharma, Sumit Goel, Taru Goyal, R. Pilania, Ridhima Aggarwal, Taranpreet Kaur, M. Dhaliwal, Amit Rawat, Surjit Singh
          </td>
          <td>2024-09-04</td>
          <td>Expert review of clinical immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In the ever-evolving landscape of oncology, the battle against prostate cancer (PCa) stands at a transformative juncture, propelled by the integration of molecular diagnostics into traditional cytopathological frameworks. This synthesis not only heralds a new epoch of precision medicine but also significantly enhances our understanding of the disease’s genetic intricacies. Our comprehensive review navigates through the latest advancements in molecular biomarkers and their detection technologies, illuminating the potential these innovations hold for the clinical realm. With PCa persisting as one of the most common malignancies among men globally, the quest for early and precise diagnostic methods has never been more critical. The spotlight in this endeavor shines on the molecular diagnostics that reveal the genetic underpinnings of PCa, offering insights into its onset, progression, and resistance to conventional therapies. Among the genetic aberrations, the TMPRSS2-ERG fusion and mutations in genes such as phosphatase and tensin homolog (PTEN) and myelocytomatosis viral oncogene homolog (MYC) are identified as significant players in the disease’s pathology, providing not only diagnostic markers but also potential therapeutic targets. This review underscores a multimodal diagnostic approach, merging molecular diagnostics with cytopathology, as a cornerstone in managing PCa effectively. This strategy promises a future where treatment is not only tailored to the individual’s genetic makeup but also anticipates the disease’s trajectory, offering hope for improved prognosis and quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040ce4884f4e6bdcb8041b33d1c1868319f07888" target='_blank'>
              From microscopes to molecules: The evolution of prostate cancer diagnostics
              </a>
            </td>
          <td>
            Junyue Tao, Xiaokang Bian, Jun Zhou, Meng Zhang
          </td>
          <td>2024-08-29</td>
          <td>CytoJournal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains the most lethal cancer and will soon be the second most common cause of cancer related death. While regimens containing DNA damaging agents such as FOLFIRINOX and PARP inhibitors have derived clinical benefits for some patients, their efficacy invariably fails over time. This presents a significant clinical challenge, and thus there is an urgent need for novel therapeutic strategies which are able to overcome the acquisition of resistance in PDAC. Clinically relevant models of treatment resistance were generated from patient-derived cell lines by extended exposure to chemotherapy agents. Synergy scoring, clonogenicity assays, flow cytometry, immunofluorescence and transcriptomic analysis were used to investigate the efficacy of combined ATR and PARP inhibition in re-sensitising resistant PDAC to treatment. Acquisition of resistance was associated with transcriptomic shifts in cell cycle checkpoint regulation, metabolic control, DNA damage response (DDR), programmed cell death and the replication stress response. Additionally, combined treatment with the ATR inhibitor (ceralasertib), and the PARP inhibitor (olaparib) was synergistic in all models of acquired resistance. Sequential treatment using ceralasertib prior to olaparib was highly effective at low dose for DDR proficient cell lines, whereas DDR deficient models responded better when treated with olaparib first. We provide in vitro evidence of a novel therapeutic strategy to overcome acquired PARP inhibitor and platinum resistance in PDAC by using sequential exposure to ceralasertib and Olaparib. A sequential regimen may be more tolerable and should be investigated clinically to circumvent dose limiting toxicity in concurrent combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3162301dd3095937530db76522192901b74cb218" target='_blank'>
              Sequential ATR and PARP Inhibition Overcomes Acquired DNA Damaging Agent Resistance in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Katharine J. Herbert, Rosie Upstill-Goddard, Stephan B. Dreyer, Selma Rebus, Christian Pilarsky, Debabrata Mukhopadhyay, Christopher J. Lord, Andrew Biankin, F. Froeling, David K. Chang
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract SF3B1 is the most recurrently mutated RNA splicing gene in cancer. However, research of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, our study compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV homology-directed repair editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. Our approach was further refined by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes, including altered splicing, copy number variations, and cell-growth defect. Moreover, we discovered that the SF3B1 mutation can cause the loss of Y chromosome in chronic lymphocytic leukemia. Our results showcase that PRECIS is an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context. Significance: This study developed an approach that can reliably and efficiently engineer SF3B1 mutation into different cellular contexts, thereby revealing novel roles of SF3B1 mutation in driving aberrant splicing, clonal evolution, and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering Oncogenic Hotspot Mutations on SF3B1 via CRISPR-Directed PRECIS Mutagenesis
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human inborn errors of immunity (IEI) represent a diverse group of genetic disorders affecting the innate and/or adaptive immune system. Some IEI entities comprise defects in DNA repair factors, resulting in (severe) combined immunodeficiencies, bone marrow failure, predisposition to malignancies, and potentially result in radiosensitivity (RS). While other IEI subcategories such as common variable immunodeficiency (CVID) and immune dysregulation disorders also associate with lymphoproliferative and malignant complications, the occurrence of RS phenotypes in the broader IEI population is not well characterized. Nonetheless, identifying RS in IEI patients through functional testing is crucial to reconsider radiation-related therapeutic protocols and to improve overall patient management. This study aimed to investigate chromosomal RS in a diverse cohort of 107 IEI patients using the G0 cytokinesis-block micronucleus (MN) assay. Our findings indicate significant variability in RS across specific genetic and phenotypical subgroups. Severe RS was detected in all ataxia-telangiectasia (AT) patients, a FANCI deficient and ERCC6L2 deficient patient, but not in any other IEI patient included in this cohort. Age emerged as the single influencing factor for both spontaneous and radiation-induced MN yields, while the manifestation of additional clinical features, including infection susceptibility, immune dysregulation, or malignancies did not associate with increased MN levels. Our extensive analysis of RS in the IEI population underscores the clinical importance of RS assessment in AT patients and supports RS testing in all IEI patients suspected of having a DNA repair disorder associated with radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4f50595c5c29676228d24464dde5631290eb13" target='_blank'>
              Investigating chromosomal radiosensitivity in inborn errors of immunity: insights from DNA repair disorders and beyond.
              </a>
            </td>
          <td>
            E. Beyls, Evi Duthoo, Lynn Backers, Karlien Claes, Marieke De Bruyne RAPID clinicians, L. Pottie, Victoria Bordon, C. Bonroy, S. Tavernier, Kathleen BM Claes, A. Vral, A. Baeyens, F. Haerynck, T. Kerre, B. Lambrecht, L. Hoste, B. Meertens, T. V. Genechten, A. Ferster, K. V. Driessche, C. D. Vriendt, L. Dedeken, I. Moors, C. Dhooge, L. Kornreich, C. Heijmans, M. L. Roux, M. Colard, A. V. Damme, K. V. Schil, B. D. Wilde
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Overexpression of the MYC oncogene, encoding c-MYC protein, contributes to the pathogenesis and drug resistance of acute myeloid leukemia (AML) and many other hematopoietic malignancies. Although standard chemotherapy has predominated in AML therapy over the past five decades, the clinical outcomes and patient response to treatment remain suboptimal. Deeper insight into the molecular basis of this disease should facilitate the development of novel therapeutics targeting specific molecules and pathways that are dysregulated in AML, including fms-like tyrosine kinase 3 (FLT3) gene mutation and cluster of differentiation 33 (CD33) protein expression. Elevated expression of c-MYC is one of the molecular features of AML that determines the clinical prognosis in patients. Increased expression of c-MYC is also one of the cytogenetic characteristics of drug resistance in AML. However, direct targeting of c-MYC has been challenging due to its lack of binding sites for small molecules. In this review, we focused on the mechanisms involving the bromodomain and extra-terminal (BET) and cyclin-dependent kinase 9 (CDK9) proteins, phosphoinositide-Akt-mammalian target of rapamycin (PI3K/AKT/mTOR) and Janus kinase-signal transduction and activation of transcription (JAK/STAT) pathways, as well as various inflammatory cytokines, as an indirect means of regulating MYC overexpression in AML. Furthermore, we highlight Food and Drug Administration (FDA)-approved drugs for AML, and the results of preclinical and clinical studies on novel agents that have been or are currently being tested for efficacy and tolerability in AML therapy. Overall, this review summarizes our current knowledge of the molecular processes that promote leukemogenesis, as well as the various agents that intervene in specific pathways and directly or indirectly modulate c-MYC to disrupt AML pathogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1135bf881c820d3eca90352e7790a726703df16e" target='_blank'>
              Targeting Molecular Signaling Pathways and Cytokine Responses to Modulate c-MYC in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Kyle Gu, Harry A. May, Min H. Kang
          </td>
          <td>2024-09-14</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/431b8d9fdd349b2296cea4bbf8173c562f2afaeb" target='_blank'>
              Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes
              </a>
            </td>
          <td>
            Sheri Skerget, Daniel A. Peñaherrera, A. Chari, Sundar Jagannath, David Siegel, Ravi Vij, G. Orloff, A. Jakubowiak, R. Niesvizky, Darla Liles, J. Berdeja, Moshe Levy, J. Wolf, S. Usmani, Austin Christofferson, Sara Nasser, J. Aldrich, Christophe Legendre, Brooks A. Benard, Chase Miller, Bryce Turner, Ahmet A. Kurdoglu, M. Washington, Venkata Yellapantula, J. Adkins, Lori Cuyugan, Martin Boateng, A. Helland, Shari Kyman, Jackie McDonald, Rebecca Reiman, Kristi Stephenson, Erica E Tassone, A. Blanski, Brianne Livermore, Meghan Kirchhoff, D. Rohrer, M. D'Agostino, Manuela Gamella, K. Collison, Jennifer Stumph, Pam Kidd, A. Donnelly, B. Zaugg, Maureen Toone, K. McBride, M. Derome, Jennifer Rogers, David Craig, Winnie S Liang, N. Gutiérrez, Scott D. Jewell, J. Carpten, Kenneth C Anderson, H. Cho, D. Auclair, S. Lonial, J. Keats
          </td>
          <td>2024-08-19</td>
          <td>Nature Genetics</td>
          <td>2</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2352f41e08e0411c3c2eeaf443ff948521637f4" target='_blank'>
              Deletion of exons 45 to 55 in the DMD gene: from the therapeutic perspective to the in vitro model
              </a>
            </td>
          <td>
            Javier Poyatos-García, Patricia Soblechero-Martín, A. Liquori, A. López-Martínez, Pilar Maestre, Elisa González-Romero, Rafael P. Vázquez-Manrique, Nuria Muelas, Gema García-García, Jessica Ohana, Virginia Arechavala-Gomeza, J. J. Vilchez
          </td>
          <td>2024-10-01</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Disruptions in normal development and the emergence of health conditions often result from the malfunction of vital genes in the human body. Decades of scientific research have focused on techniques to modify or substitute defective genes with healthy alternatives, marking a new era in disease treatment, prevention and cure. Recent strides in science and technology have reshaped our understanding of disorders, medication development and treatment recommendations, with human gene and cell therapy at the forefront of this transformative shift. Its primary objective is the modification of genes or adjustment of cell behaviour for therapeutic purposes. In this review, we focus on the latest advances in gene and cell therapy for treating human genetic diseases, with a particular emphasis on FDA and EMA‐approved therapies and the evolving landscape of genome editing. We examine the current state of innovative gene editing technologies, particularly the CRISPR‐Cas systems. As we explore the progress, ethical considerations and prospects of these innovations, we gain insight into their potential to revolutionize the treatment of genetic diseases, along with a discussion of the challenges associated with their regulatory pathways. This review traces the origins and evolution of these therapies, from conceptual ideas to practical clinical applications, marking a significant milestone in the field of medical science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605617113766e025c90c18b7b7452f93c13a0842" target='_blank'>
              Gene and cell therapy of human genetic diseases: Recent advances and future directions
              </a>
            </td>
          <td>
            Busra Cetin, Fulya Erendor, Y. E. Eksi, A. Şanlioğlu, S. Sanlioglu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="BACKGROUND
The clinical management of chronic myeloid leukemia (CML) patients requires the identification of the type of BCR::ABL1 transcript at diagnosis and the monitoring of its expression and potential tyrosine kinase inhibitor (TKI) resistance mutations during treatment. Detection of resistant mutation requires transcript type-specific amplification of BCR::ABL1 from RNA.


METHODS
In this study, a custom RNA-based next-generation sequencing (NGS) assay (Dup-Seq BCR::ABL1) that enables (a) the identification of BCR::ABL1 transcript type and (b) the detection of resistance mutations from common and atypical BCR::ABL1 transcript types was developed and validated. The assay design covers BCR exon 1 to ABL1 exon 10 and employs duplicate PCR amplification for error correction. The custom data analysis pipeline enables breakpoint determination and overlapped mutation calling from duplicates, which minimizes the low-level mutation artifacts.


RESULTS
This study demonstrates that this novel assay achieves high accuracy (positive percent agreement (PPA) for fusion: 98.5%; PPA and negative percent agreement (NPA) for mutation at 97.8% and 100.0%, respectively) and sensitivity (limit of detection (LOD) for mutation detection at 3% from 10 000 copies of BCR::ABL1 input).


CONCLUSIONS
The Dup-Seq BCR::ABL1 assay not only allows for the identification of BCR::ABL1 typical and atypical transcript types and accurate and sensitive detection of TKI-resistant mutations but also simplifies molecular testing work flow for the clinical management of CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c413e19fd7d542721e914a796a2298f73b3d650" target='_blank'>
              A Novel Next-Generation Sequencing Assay for the Identification of BCR::ABL1 Transcript Type and Accurate and Sensitive Detection of TKI-Resistant Mutations.
              </a>
            </td>
          <td>
            Zhenyu Yan, Lin Shi, Wei Li, Weihua Liu, C. Galderisi, Cynthia Spittle, Jin Li
          </td>
          <td>2024-09-03</td>
          <td>The journal of applied laboratory medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2b8937040c82aaaf4474d35899e8a2ae8efb870" target='_blank'>
              Genomic, epigenomic and transcriptomic landscape of glioblastoma.
              </a>
            </td>
          <td>
            T. C. Dakal, Ganesh S. Kakde, Pawan Kumar Maurya
          </td>
          <td>2024-08-24</td>
          <td>Metabolic brain disease</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9323b395426f00928696c5f63d0f88e7dffceb5" target='_blank'>
              The evolving genetic landscape of telomere biology disorder dyskeratosis congenita.
              </a>
            </td>
          <td>
            H. Tummala, A. Walne, M. Badat, Manthan Patel, Abigail M Walne, Jenna Alnajar, Chi Ching Chow, Ibtehal Albursan, J. Frost, David Ballard, Sally Killick, Peter Szitányi, Anne M Kelly, Manoj Raghavan, Corrina Powell, Reinier A P Raymakers, Tony Todd, Elpis Mantadakis, Sophia Polychronopoulou, Nikolas Pontikos, Tianyi Liao, Pradeepa M. Madapura, Upal Hossain, T. Vulliamy, I. Dokal
          </td>
          <td>2024-08-28</td>
          <td>EMBO molecular medicine</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b273ccaf2ba1c9f2d8cb9b79c03b12ee6eed17c1" target='_blank'>
              Modulatory effects of miRNAs in doxorubicin resistance: A mechanistic view.
              </a>
            </td>
          <td>
            Fatemeh Ebadi Meinag, Mina Fatahi, Vahid Vahedian, N. F. Maroufi, Bashir Mosayyebi, Elham Ahmadi, Mohammad Rahmati
          </td>
          <td>2024-09-02</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution.
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer M Shelton, Minita Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="There is increasing recognition that the sex chromosomes, X and Y, play an important role in health and disease that goes beyond the determination of biological sex. Loss of the Y chromosome (LOY) in blood, which occurs naturally in aging men, has been found to be a driver of cardiac fibrosis and heart failure mortality. LOY also occurs in most solid tumors in males and is often associated with worse survival, suggesting that LOY may give tumor cells a growth or survival advantage. We analyzed LOY in lung adenocarcinoma (LUAD) using both bulk and single-cell expression data and found evidence suggesting that LOY affects the tumor immune environment by altering cancer/testis antigen expression and consequently facilitating tumor immune evasion. Analyzing immunotherapy data, we show that LOY and changes in expression of particular cancer/testis antigens are associated with response to pembrolizumab treatment and outcome, providing a new and powerful biomarker for predicting immunotherapy response in LUAD tumors in males.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03627d4b1b03380930579b6c3a6ee53571e9ff88" target='_blank'>
              Selective loss of Y chromosomes in lung adenocarcinoma modulates the tumor immune environment through cancer/testis antigens
              </a>
            </td>
          <td>
            Jonas Fischer, K. Shutta, Chen Chen, Viola Fanfani, Enakshi Saha, Panagiotis Mandros, Marouen Ben Guebila, Joanne Xiu, Jorge Nieva, Stephen V. Liu, D. Uprety, D. Spetzler, C. Lopes-Ramos, Dawn L. DeMeo, John Quackenbush
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="CRISPR-based genome engineering holds enormous promise for basic science and therapeutic applications. Integrating and editing DNA sequences is still challenging in many cellular contexts, largely due to insufficient control of the repair process. We find that repair at the genome-cargo interface is predictable by deep-learning models and adheres to sequence context specific rules. Based on in silico predictions, we devised a strategy of triplet base-pair repeat repair arms that correspond to microhomologies at double-strand breaks (trimologies), which facilitated integration of large cargo (>2 kb) and protected the targeted locus and transgene from excessive damage. Successful integrations occurred in >30 loci in human cells and in in vivo models. Germline transmissible transgene integration in Xenopus, and endogenous tagging of tubulin in adult mice brains demonstrated integration during early embryonic cleavage and in non-dividing differentiated cells. Further, optimal repair arms for single- or double nucleotide edits were predictable, and facilitated small edits in vitro and in vivo using oligonucleotide templates. We provide a design-tool (Pythia, pythia-editing.org) to optimize custom integration, tagging or editing strategies. Pythia will facilitate genomic integration and editing for experimental and therapeutic purposes for a wider range of target cell types and applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ad57fb5d6450173260d95ec721d1be7ca9efcc" target='_blank'>
              Pythia: Non-random DNA repair allows predictable CRISPR/Cas9 integration and gene editing
              </a>
            </td>
          <td>
            Thomas Naert, Taiyo Yamamoto, Shuting Han, Melanie Horn, Phillip Bethge, Nikita Vladimirov, F. Voigt, Joana Figueiro-Silva, Ruxandra Bachmann-Gagescu, F. Helmchen, S. Lienkamp
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f0f651c9fac9c8c40b22fc5b21b4578646df27" target='_blank'>
              Multi-omics features of immunogenic cell death in gastric cancer identified by combining single-cell sequencing analysis and machine learning
              </a>
            </td>
          <td>
            Shu-Long Dai, Jian-Qiang Pan, Zhen-Rong Su
          </td>
          <td>2024-09-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>50</td>
        </tr>

        <tr id="BACKGROUND/AIM
The emergence of novel DNA damage repair (DDR) pathways in molecular-target therapy drugs (MTTD) has shown promising outcomes in treating patients with metastatic castration-resistant prostate cancer (mCRPC). About 25% of mCRPC patients have actionable deleterious aberrations in DDR genes, primarily in the homologous recombination (HR) pathway. However, the response rate in patients with BRCA1/2 or mutations in HRR-related genes is only 45%-55%, when exposed to poly ADP ribose polymerase (PARP) inhibitor-based therapy (PARPi). A frequent characteristic feature of prostate cancer (PC) is the occurrence of genomic rearrangement that affects the transmembrane protease serine 2 (TMPRSS2) and E26 transformation-specific (ETS)- transcription factor-related gene (ERG).


MATERIALS AND METHODS
In this study, a total of 114 patients with mCRPC had their RNA and DNA sequenced using next-generation sequencing.


RESULTS
Based on their genetic profile of deleterious gene alterations of BRCA1/2 or ATM, six patients were selected for PARPi. Patients with TMPRSS2:ERG gene fusion and homozygous alteration in ATM or BRCA2 (n=2) or heterozygous alterations (BRCA1 or BRCA2) and lack of TMPRSS2:ERG gene fusion (n=2) did not show clinical benefit from PARPi (treatment duration <16 weeks). In contrast, patients (n=2) without TMPRSS2:ERG gene fusion and homozygous deleterious alterations in ATM or BRCA2 all had clinical benefit from PARPi (treatment duration ≥16 weeks).


CONCLUSION
The TMPRSS2:ERG transcript product might be used as a PARPi resistance biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb09c69e7cd8aed30d5976f567f7199cd33c56e" target='_blank'>
              TMPRSS2:ERG Gene Fusion Might Predict Resistance to PARP Inhibitors in Metastatic Castration-resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Tim S Poulsen, Anders Nygaard Lørup, Per Kongsted, R. Eefsen, M. Højgaard, Estrid Høgdall
          </td>
          <td>2024-09-30</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Radiation is a mainstay of lung cancer treatment; however, resistance frequently develops. Identifying novel therapeutic targets to increase radiation sensitivity is crucial. S6K1 is a serine/threonine kinase known to regulate protein translation which is associated with radioresistance, but the mechanisms involved are unknown. We proposed to determine whether S6K1 promotes radioresistance by regulating DNA repair in lung cancer. Colony formation, protein expression and proliferation were assessed. S6K1 was modulated pharmacologically by either PF-4708671 or genetically by Crispr-Cas9. Higher radioresistance levels in lung cancer cells were associated with lower phosphoactivation of MRN complex members, a key activator of radiation-induced DNA repair signaling. We also found lower levels of p-ATM, a target of the MRN complex, in more radioresistant cells, which was associated with a lower expression of γ-H2AX cafter radiation. Further, genetic and pharmacological S6K1 targeting sensitized lung cancer cells to low doses of radiation (p ≤ 0.01). Additionally, S6K1−/− deletion increased the phosphoactivation of MRN complex members, indicating that S6K1 itself can shut down DNA damage regulated by MRN signaling. This is the first report showing that S6K1 inhibition radiosensitizes lung cancer cells by decreasing MRN complex-regulated DNA repair signaling. Future studies should evaluate the role of S6K1 as a target to overcome radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623356ed9108e3db105e7395f2b3d04852458dc3" target='_blank'>
              S6K1 Controls DNA Damage Signaling Modulated by the MRN Complex to Induce Radioresistance in Lung Cancer
              </a>
            </td>
          <td>
            Ali Calderon-Aparicio, Jun He, N. Simone
          </td>
          <td>2024-09-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction: The role of genetic variants in response to chemotherapy has been investigated in several studies. This study aimed to investigate genetic variants associated with response to chemotherapy in breast cancer (BC) patients. Methods: Significant variants (p < 5 × 10-8) associated with response to chemotherapy were obtained from GWA studies. Candidate variants were identified by haplotype analysis (r2 ≥ 0.9, D'≥0.9) using 1000Genome LD data. To determine the effects of the variants on gene expression, expression quantitative trait loci (eQTL) were evaluated. To compare the expression of the identified genes in tumor samples, expression levels were compared between TCGA tumor types and adjacent normal tissues. Results: Six rs3820706, rs147451859, rs4784750, rs17587029, rs16830728, and rs16972207 variants were significantly associated with response to chemotherapy in BC patients (p < 5 × 10-8). Seven novel haplotypic structures were identified to be associated with adverse response to chemotherapy in BC patients. These haplotypes formed two genetic structures associated with neutropenia, leukopenia, chemotherapy-induced cytotoxicity (GAG-TTAT), and chemotherapy-induced alopecia (CC-CAACTCCCGTTGCGG). These variants are located on PPCDC, NLRC5, STAM2, and TNFSF13B genes, and the expression of these genes significantly changed in BC tissues than normal tissues (P ≤ 0.05), also showing gene-gene correlation (P ≤ 0.05). Conclusions: These genetic variants and their associated novel haplotypic structures can predict adverse response to chemotherapy in BC patients and could potentially form BC-associated genetic panel for adverse response to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6f813884867edf6f2b79ca7de2ac3b63826948" target='_blank'>
              Novel genetic structures associated with adverse response to chemotherapy in breast cancer.
              </a>
            </td>
          <td>
            Morteza Gholami, Mohsen Asouri, A. Ahmadi, Mehrab Nasirikenari
          </td>
          <td>2024-08-28</td>
          <td>Journal of oncology pharmacy practice : official publication of the International Society of Oncology Pharmacy Practitioners</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a hereditary blood disorder characterized by the presence of abnormal hemoglobin molecules and thus distortion (sickling) of the red blood cells. SCD causes chronic pain and organ damage and shortens life expectancy. Gene therapy emerges as a potentially curative approach for people with SCD who lack a matched sibling donor for hematopoietic stem cell transplantation. Here, we review recent progress in gene therapy for SCD and focus on innovative technologies that target the genetic roots of the disease. We also review the challenges associated with gene therapy, including oncogenic risks, and the need for refined delivery methods. Despite these hurdles, the rapidly evolving landscape of gene therapy for SCD raises hope for a paradigm shift in the treatment of this debilitating disease. As research progresses, a deeper understanding of the molecular mechanisms involved and continuous improvements in gene-editing technologies promise to bring gene therapy for SCD closer to mainstream clinical application, offering a transformative, curative option for patients with this genetic disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e785d7cbf41248336e372c5a5fb5eb1bf4e0c" target='_blank'>
              Treating Sickle Cell Disease: Gene Therapy Approaches.
              </a>
            </td>
          <td>
            M. Cavazzana, Alice Corsia, Mégane Brusson, A. Miccio, Michaela Semeraro
          </td>
          <td>2024-09-11</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Oncogenic KRAS, the genetic driver of 90% of PDAC, induces a metabolic rewiring characterized, in part, by dependency on de novo pyrimidine biosynthesis. Dihydroorotate dehydrogenase (DHODH) is the enzymatic ‘chokepoint’ of the de novo pyrimidine biosynthesis pathway and an in vivo metabolic liability in pancreatic ductal adenocarcinoma (PDAC). Targeting enzymes in the de novo pyrimidine synthesis pathway such as dihydroorotate dehydrogenase (DHODH) with clinically available inhibitor brequinar (BQ) starved the tumor of nucleotide synthesis and restrained growth of PDAC cells in vitro. However, therapeutic resistance limited BQ long-term effects in PDAC xenografts, which suggests compensatory pathways and that combinatorial strategies are required for enhanced efficacy. Here, we integrated a mass spectrometry (MS)-based quantitative temporal proteomics workflow, LC/MS- based metabolomics and in vitro and in vivo drug-anchored CRISPR/Cas9 genetic screens to identify compensatory pathways to DHODH inhibition (DHODHi) and targets for combination strategies. We demonstrate that DHODHi alters the apoptotic regulatory proteome thereby enhancing sensitivity to inhibitors of the anti-apoptotic BCL2L1 (BCL-XL) protein. Combinatorial regimens with DHODH and BCL-XL inhibition synergistically induce apoptosis in PDAC cell lines and patient-derived PDAC organoids. In vivo DHODH inhibition with Brequinar and BCL-XL degradation with DT2216, a proteolysis targeting chimera (PROTAC), significantly inhibits the growth of PDAC tumors. Our data defines mechanisms of adaptation to DHODH inhibition and identifies a combination therapy strategy in PDAC.
 Citation Format: huan zhang, Qijia yu, Naiara Santana-Codina, Clara Poupault, Claudia Campos, Xingping Qin, Aparna Padhye, Nicole Sindoni, Miljan Kuljanin, Junning Wang, Matthew J. Dorman, Andrew J. Aguirre, Stephanie K. Dougan, Kristopher A. Sarosiek, Joseph D. Mancias. De novo pyrimidine biosynthesis inhibition synergizes with BCL-XL targeting in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e6764c284dc6a039dffbb1acd0079baf1b7f23" target='_blank'>
              Abstract C032: De novo pyrimidine biosynthesis inhibition synergizes with BCL-XL targeting in pancreatic cancer
              </a>
            </td>
          <td>
            huan zhang, Qijia yu, Naiara Santana-Codina, Clara Poupault, Claudia Campos, Xingping Qin, A. Padhye, Nicole Sindoni, Miljan Kuljanin, Junning Wang, Matthew J. Dorman, A. Aguirre, Stephanie K Dougan, K. Sarosiek, J. Mancias
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background Alterations in DNA damage repair genes in advanced prostate cancer (PC) may impact responses to therapy and clinical outcomes. This study described homologous recombination repair (HRR) testing patterns and clinical outcomes among patients with metastatic castration-resistant prostate cancer (mCRPC) by HRR alteration status and race in the United States (US). Methods Clinical data in the nationwide (US-based) Flatiron Health-Foundation Medicine, Inc. (FMI) Metastatic PC Clinico-Genomic Database were evaluated (01/01/2011–12/31/2022). Patients initiating first-line (1L) mCRPC therapy on or after mCRPC diagnosis were included. Testing patterns, time-to-next treatment, overall survival (OS), and time-to-prostate specific antigen response were described. Results Of the 1367 patients with mCRPC and at least one HRR panel test prior to or on the date of 1L mCRPC therapy initiation, 332 (24.3%) were HRR positive (White patients: n = 219 [66.0%]; Black patients: n = 37 [11.1%]) and 1035 (75.7%) were HRR negative (White patients: n = 702 [67.8%]; Black patients: n = 84 [8.1%]). The mean time between first positive test and 1L mCRPC therapy initiation date was 588 days (White patients: 589 days; Black patients: 639 days). Among HRR positive relative to negative patients, trends for faster progression (respective 12-month rate overall: 71.1% and 63.7%; White patients: 72.5% and 64.0%; Black patients: 65.4% and 56.4%), shorter OS (respective 24-month rate overall: 46.8% and 51.9%; White patients: 48.6% and 46.2%; Black patients: 52.8% and 54.1%), and decreased treatment response (respective 12-month rate overall: 24.3% and 37.9%; White patients: 24.5% and 35.2%; Black patients: 17.0% and 43.9%) were observed. Conclusion Patients with mCRPC positive for HRR alterations tended to exhibit poorer treatment responses and clinical outcomes than those with a negative status. These findings highlight the importance of timely genetic testing in mCRPC, particularly among Black patients, and the need for improved 1L targeted therapies to address the unmet need in HRR positive mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b5b000d1e3f13e0b5a22c9691a361f7dc5edd2" target='_blank'>
              Homologous Recombination Repair Testing Patterns and Outcomes in mCRPC by Alteration Status and Race
              </a>
            </td>
          <td>
            M. Bilen, I. Khilfeh, C. Rossi, Erik Muser, L. Morrison, A. Hilts, Lilian Diaz, Patrick Lefebvre, D. Pilon, Daniel George
          </td>
          <td>2024-09-01</td>
          <td>ClinicoEconomics and Outcomes Research: CEOR</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Pediatric classic Hodgkin lymphoma (cHL) patients have a high survival rate but suffer from severe long‐term side effects induced by chemo‐ and radiotherapy. cHL tumors are characterized by the low fraction (0.1%–10%) of malignant Hodgkin and Reed–Sternberg (HRS) cells in the tumor. The HRS cells depend on the surrounding immune cells for survival and growth. This dependence is leveraged by current treatments that target the PD‐1/PD‐L1 axis in cHL tumors. The development of more targeted therapies that are specific for the tumor and are therefore less toxic for healthy tissue compared with conventional chemotherapy could improve the quality of life of pediatric cHL survivors. Here, we applied single‐cell RNA sequencing (scRNA‐seq) on isolated HRS cells and the immune cells from the same cHL tumors. Besides TNFRSF8 (CD30), we identified other genes of cell surface proteins that are consistently overexpressed in HRS cells, such as NRXN3 and LRP8, which can potentially be used as alternative targets for antibody–drug conjugates or CAR T cells. Finally, we identified potential interactions by which HRS cells inhibit T cells, among which are the galectin‐1/CD69 and HLA‐II/LAG3 interactions. RNAscope was used to validate the enrichment of CD69 and LAG3 expression on T cells near HRS cells and indicated large variability of the interaction strength with the corresponding ligands between patients and between tumor tissue regions. In conclusion, this study identifies new potential therapeutic targets for cHL and highlights the importance of studying heterogeneity when identifying therapy targets, specifically those that target tumor‐immune cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963141cd2c957924d6cb5b66200fd817e1bf6a44" target='_blank'>
              Single‐cell RNA sequencing of pediatric Hodgkin lymphoma to study the inhibition of T cell subtypes
              </a>
            </td>
          <td>
            Jurrian K de Kanter, A. Steemers, D. M. Gonzalez, Ravian L. van Ineveld, Catharina Blijleven, N. Groenen, L. Trabut, M. Scheijde-Vermeulen, Liset Westera, Auke Beishuizen, Anne C. Rios, F. C. Holstege, A. Brandsma, T. Margaritis, R. van Boxtel, F. Meyer-Wentrup
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a6c27093146f06f4b8893e39507755c64cdc82" target='_blank'>
              BAP1 regulates HSF1 activity and cancer immunity in pancreatic cancer
              </a>
            </td>
          <td>
            Weiwei Yuan, Qiyue Zhang, Yuhan Zhao, Wentao Xia, Shilin Yin, Xueyi Liang, Taoyu Chen, Gaofeng Li, Yanshen Liu, Zhiqiang Liu, Jinxi Huang
          </td>
          <td>2024-09-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Purpose of review FLT3 mutations are among the most common myeloid drivers identified in adult acute myeloid leukemia (AML). Their identification is crucial for the precise risk assessment because of the strong prognostic significance of the most recurrent type of FLT3 alterations, namely internal tandem duplications (ITDs). Recent advances in the pathogenesis and biology of FLT3-mutated AML have opened an opportunity for development and application of selective inhibition of FLT3 pathway. Recent findings In the last decade, at least three targeted treatments have been approved by regulatory agencies and several others are currently under investigations. Here, we review the latest advance in the role of FLT3 mutations in AML, providing an outline of the available therapeutic strategies, their mechanisms of actions and of resistance, as well as routes for potential improvement. Summary The availability of FLT3 inhibitors has improved outcomes in AML harboring such mutations, currently also reflected in disease stratification and recommendations. Newer inhibitors are under investigations, and combinations with chemotherapy or other targeted treatments are being explored to further improve disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f19a10a5bb85b8ee615c65a99e7f2184b3ca7" target='_blank'>
              Advances in the pathogenesis of FLT3-mutated acute myeloid leukemia and targeted treatments
              </a>
            </td>
          <td>
            S. Travaglini, C. Gurnari, T. Ottone, M. Voso
          </td>
          <td>2024-08-26</td>
          <td>Current Opinion in Oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Mutations in the male germline are a driving force behind rare genetic diseases. Driver mutations enjoying a selective advantage expand to mutant clusters within the aged testis and are thus overrepresented in sperm with age. Other kinds of driver mutations, occurring prepubescently, have been the focus of recent attention given their high occurrence rate independent of age. Here, we investigated the ErbB2 gene via error-corrected sequencing and detected a high percentage of missense mutations, including recurrent mutations, which were observed mainly in the tyrosine kinase domain with likely functional consequences, as we verified for a subset via biophysical methods. While these mutations increased with age, we found no evidence that they originated from mutational clusters in the aged testis, and young donors also showed an accumulation of driver mutations, suggesting that mutational enrichment is not exclusive to the sexually mature germline but can occur earlier during germline development, and are likely evenly distributed in the testis and stable in size.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9f975fbf6b3e62de966f4ae5746c898526f4b3" target='_blank'>
              Mutations in ErbB2 accumulating in the male germline measured by error-corrected sequencing
              </a>
            </td>
          <td>
            Atena Yasari, Monika Heinzl, Theresa Mair, Tina Karimian, Shehab Moukbel Ali Aldawla, Ingrid Hartl, Andrea J. Betancourt, Peter Lanzerstorfer, Irene Tiemann-Boege
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background/Aim Despite therapeutic advancements, metastatic colorectal cancer is usually fatal, necessitating novel approaches based on the molecular pathogenesis to improve outcomes. Some colorectal cancers have no mutations in the extended RAS panel (KRAS, NRAS, BRAF) genes and represent a special subset, which deserves particular therapeutic considerations. Materials and Methods The genomic landscape of colorectal cancers from publicly available genomic series was interrogated, using the cBioportal platform. Colorectal cancer cohorts with cancers devoid of KRAS/NRAS or BRAF mutations were evaluated for the presence of mutations in the catalytic sub-unit alpha of kinase PI3K, encoded by the gene PIK3CA. Results PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations were observed in 3.7% to 7.6% of colorectal cancers in the different series examined. Patients with all four genes in wildtype configuration (quadruple wild type) represented 32.2% to 39.9% of cases in the different series examined. Compared with quadruple wild type cancers, triple (KRAS/NRAS/BRAF) wild type/PIK3CA mutated cancers had a higher prevalence of high TMB cases and additional mutations in colorectal cancer associated genes except for mutations in TP53. Mutations in genes encoding for epigenetic modifiers and the DNA damage response (DDR) were also more frequent in triple wild type/PIK3CA mutated cancers. The prognosis of the two groups was comparable. Conclusion Colorectal cancers with PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations have frequently mutations in epigenetic modifiers and DDR response genes, which may provide opportunities for targeting. These mutations are present in a smaller subset of quadruple wild type cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fec366ce24e54405084d6854f8aadb7604b8213" target='_blank'>
              PIK3CA Mutated Colorectal Cancers Without KRAS, NRAS and BRAF Mutations Possess Common and Potentially Targetable Mutations in Epigenetic Modifiers and DNA Damage Response Genes
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-08-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Radiation‐induced tissue injury (RITI) is the most common complication in clinical tumor radiotherapy. Due to the heterogeneity in the response of different tissues to radiation (IR), radiotherapy will cause different types and degrees of RITI, which greatly limits the clinical application of radiotherapy. Efforts are continuously ongoing to elucidate the molecular mechanism of RITI and develop corresponding prevention and treatment drugs for RITI. Single‐cell sequencing (Sc‐seq) has emerged as a powerful tool in uncovering the molecular mechanisms of RITI and for identifying potential prevention targets by enhancing our understanding of the complex intercellular relationships, facilitating the identification of novel cell phenotypes, and allowing for the assessment of cell heterogeneity and spatiotemporal developmental trajectories. Based on a comprehensive review of the molecular mechanisms of RITI, we analyzed the molecular mechanisms and regulatory networks of different types of RITI in combination with Sc‐seq and summarized the targeted intervention pathways and therapeutic drugs for RITI. Deciphering the diverse mechanisms underlying RITI can shed light on its pathogenesis and unveil new therapeutic avenues to potentially facilitate the repair or regeneration of currently irreversible RITI. Furthermore, we discuss how personalized therapeutic strategies based on Sc‐seq offer clinical promise in mitigating RITI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cb0e2375143a24b5e75c3b8bc90e1f65f24386" target='_blank'>
              Mechanisms of radiation‐induced tissue damage and response
              </a>
            </td>
          <td>
            Lin Zhou, Jiaojiao Zhu, Yuhao Liu, Ping Zhou, Yongqing Gu
          </td>
          <td>2024-09-20</td>
          <td>MedComm</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Activating FLT3 mutations plays a crucial role in leukemogenesis, but identifying the optimal candidates for FLT3 inhibitor therapy remains controversial. This study aims to explore the impacts of FLT3 mutations in pediatric acute lymphoblastic leukemia (ALL) and acute myeloid leukemia (AML) and to compare the mutation profiles between the two types to inspire the targeted application of FLT3 inhibitors. We retrospectively analyzed 243 ALL and 62 AML cases, grouping them into FLT3-mutant and wild-type categories, respectively. We then assessed the associations between FLT3 mutations and the clinical manifestations, genetic characteristics, and prognosis in ALL and AML. Additionally, we compared the distinct features of FLT3 mutations between ALL and AML. In ALL patients, those with FLT3 mutations predominantly exhibited hyperdiploidy (48.6% vs. 14.9%, p < 0.001) and higher FLT3 expression (108.02 [85.11, 142.06] FPKM vs. 23.11 [9.16, 59.14] FPKM, p < 0.001), but lower expression of signaling pathway-related genes such as HRAS, PIK3R3, BAD, MAP2K2, MAPK3, and STAT5A compared to FLT3 wild-type patients. There was no significant difference in prognosis between the two groups. In contrast, AML patients with FLT3 mutations were primarily associated with leucocytosis (82.90 [47.05, 189.76] G/L vs. 20.36 [8.90, 55.39] G/L, p = 0.001), NUP98 rearrangements (30% vs. 4.8%, p = 0.018), elevated FLT3 expression (74.77 [54.31, 109.46] FPKM vs. 34.56 [20.98, 48.28] FPKM, p < 0.001), and upregulated signaling pathway genes including PIK3CB, AKT1, MTOR, BRAF, and MAPK1 relative to FLT3 wild-type, correlating with poor prognosis. Notably, internal tandem duplications were the predominant type of FLT3 mutation in AML (66.7%) with higher inserted base counts, whereas they were almost absent in ALL (6.3%, p < 0.001). In summary, our study demonstrated that the forms and impacts of FLT3 mutations in ALL differed significantly from those in AML. The gene expression profiles of FLT3-related pathways may provide a rationale for using FLT3 inhibitors in AML rather than ALL when FLT3 mutations are present.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192ec2014a877a84ea9223fc11825ef49ab80d7e" target='_blank'>
              Distinct FLT3 Pathways Gene Expression Profiles in Pediatric De Novo Acute Lymphoblastic and Myeloid Leukemia with FLT3 Mutations: Implications for Targeted Therapy
              </a>
            </td>
          <td>
            Lizhen Zhao, Hongbo Chen, Fengli Lan, Jinjin Hao, Wenzhi Zhang, Ying Li, Yuhong Yin, Minchun Huang, Xiaoyan Wu
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id=": The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)/CRISPR-associated sequence (CRISPR/Cas) system is a cutting-edge genome-editing tool employed to explore the functions of normal and disease-related genes. The CRISPR/Cas system has a remarkable diversity in the composition and architecture of genomic loci and Cas protein sequences. Owing to its excellent efficiency and specificity, this system adds an outstanding dimension to biomedical research on genetic manipulation of eukaryotic cells. However, safe, efficient, and specific delivery of this system to target cells and tissues and their off-target effects are considered critical bottlenecks for the therapeutic applications. Recently discovered anti-CRISPR proteins (Acr) play a significant role in limiting the effects of this system. Acrs are relatively small proteins that are highly specific to CRISPR variants and exhibit remarkable structural diversity. The in silico approaches, crystallography, and cryo-electron microscopy play significant roles in elucidating the mechanisms of action of Acrs. Acrs block the CRISPR/Cas system mainly by employing four mechanisms: CRISPR/ Cas complex assembly interruption, target-binding interference, target cleavage prevention, and degradation of cyclic oligonucleotide signaling molecules. Engineered CRISPR/Cas systems are frequently used in gene therapy, diagnostics, and functional genomics. Understanding the molecular mechanisms underlying Acr action may help in the safe and effective use of CRISPR/Cas tools for genetic modification, particularly in the context of medicine. Thus, attempts to regulate prokaryotic CRISPR/Cas surveillance complexes will advance the development of antimicrobial drugs and treatment of human diseases. In this review, recent updates on CRISPR/Cas systems, especially CRISPR/Cas9 and Acrs, and their novel mechanistic insights are elaborated. In addition, the role of Acrs in the novel applications of CRISPP/Cas biotechnology for precise genome editing and other applications is discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7efd6cdfb344d21c10d023356037b1be90525f16" target='_blank'>
              Current Updates of CRISPR/Cas System and Anti-CRISPR Proteins: Innovative Applications to Improve the Genome Editing Strategies
              </a>
            </td>
          <td>
            Khaled S. Allemailem, A. Almatroudi, Faris A Alrumaihi, A. Alradhi, Abdulrahman Theyab, Mohammad Algahtani, Mohmmed Alhawas, G. Dobie, Amira A Moawad, A. Rahmani, Amjad Khan
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Nanomedicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103c83e1829743ff7261d3403d3aa51e48f6908e" target='_blank'>
              Histone marks identify novel transcription factors that parse CAR-T subset-of-origin, clinical potential and expansion
              </a>
            </td>
          <td>
            S. Fiorenza, Y. Zheng, J. Purushe, T. J. Bock, J. Sarthy, D. Janssens, A. S. Sheih, E. L. Kimble, D. Kirchmeier, T. Phi, J. Gauthier, A. Hirayama, S. R. Riddell, Q. Wu, R. Gottardo, D. G. Maloney, J. Y. H. Yang, S. Henikoff, C. J. Turtle
          </td>
          <td>2024-09-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background: To date, targeted therapies have largely failed in eliciting sustained responses in advanced pancreatic ductal adenocarcinoma (PDAC). KRAS-targeted therapies have the potential to drastically transform clinical management. Emerging data indicate that only a subset of patients respond, and that acquired resistance is common. Here, we used a combination of in vivo CRISPR activation (CRISPRa) screening and in vitro/preclinical models to identify synergistic combinations with KRAS inhibition (KRASi) in pancreatic and lung cancer. Material and Methods: We used our CRISPRa-competent PPKS (P53(F/F), Kras(LSL-G12D/+), R26(LSL-SAM)) model of adenocarcinoma to conduct targeted screening for drivers of resistance to MRTX133. Our library consisted of ∼450 frequently mutated and amplified putative oncogenes across human adenocarcinoma based on TCGA data. Mice were treated with 1e8 TU of lentiviral library through nasal instillation. After 6 weeks, treatment was initiated with 3 mg/kg MRTX1133 daily, 5 days/week (i.p.). Mice were sacrificed after 4 weeks of treatment and the guide distribution determined through bioinformatic analysis. Genes statistically significantly enriched (FDR<0.05) in the treatment group were determined. Cell lines with acquired resistance to MRTX1133 derived from the KPC PDAC model and the KPP lung adenocarcinoma (LUAD) model were generated using long-term, low-dose treatment. Resistance was confirmed with MTT drug response assays. In vitro combinatorial drug response was determined using a 72h, 96-well MTT assay, and clonogenic response using a 2 week, 384-well FACS assay. Mice were treated with 20 mg/kg MRTX1133 and/or 20mg/kg Temuterkib once daily, 7 days/week (i.p.) for 2 weeks. Results: We performed targeted in vivo CRISPRa screening for drivers of resistance to KRASi in an autochthons model of lung adenocarcinoma (LUAD) using a library targeting 452 genes commonly amplified or mutated in cancer. We identified a set of genes that were specifically enriched in at least 50% of tumors treated with MRTX1133, including MAPK1/ERK2. We also generated 16 cell lines with acquired resistance to MRTX1133, derived from KPC PDAC or KPP LUAD tumors. Transcriptomic analysis identified a large set of genes differentially expressed in cell with acquired resistance relative to controls. GSEA indicated that reactivation of the MAPK cascade as a common trait in resistant cells. To test if ERK inhibition synergizes with KRASi, we quantified the combinatorial response to MRTX1133 and Temuterkib, a specific inhibitor of ERK1 and ERK2, in a set of KPC and KPP derived cell lines, revealing significant synergy and reduced clonogenic potential in a majority of cell lines. A majority of cell lines with acquired resistance to MRTX1133 remained sensitive to Temuterkib. Finally, we tested combinatorial treatment of MRTX1133 and Temuterkib in a syngenetic subcutaneous model of PDAC, which showed at minimum additive effect. In conclusion we have identified ERK inhibition as a promising combination with KRAS inhibition in KRAS-mutant PDAC and LUAD.
 Citation Format: Fredrik I Thege, Amber Hoskins, Sonja M Woermann, Anirban Maitra. Identifying synergistic combinations with KRAS inhibition in PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/172a9d516cee228f00e1cc1c95f94a25e631e8bb" target='_blank'>
              Abstract PR-15: Identifying synergistic combinations with KRAS inhibition in PDAC
              </a>
            </td>
          <td>
            Fredrik I. Thege, Amber Hoskins, Sonja M. Woermann, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a prevalent and heterogeneous tumor with limited treatment options and unfavorable prognosis. The crucial role of a disintegrin and metalloprotease (ADAM) gene family in the tumor microenvironment of HCC remains unclear. Methods This study employed a novel multi-omics integration strategy to investigate the potential roles of ADAM family signals in HCC. A series of single-cell and spatial omics algorithms were utilized to uncover the molecular characteristics of ADAM family genes within HCC. The GSVA package was utilized to compute the scores for ADAM family signals, subsequently stratified into three categories: high, medium, and low ADAM signal levels through unsupervised clustering. Furthermore, we developed and rigorously validated an innovative and robust clinical prognosis assessment model by employing 99 mainstream machine learning algorithms in conjunction with co-expression feature spectra of ADAM family genes. To validate our findings, we conducted PCR and IHC experiments to confirm differential expression patterns within the ADAM family genes. Results Gene signals from the ADAM family were notably abundant in endothelial cells, liver cells, and monocyte macrophages. Single-cell sequencing and spatial transcriptomics analyses have both revealed the molecular heterogeneity of the ADAM gene family, further emphasizing its significant impact on the development and progression of HCC. In HCC tissues, the expression levels of ADAM9, ADAM10, ADAM15, and ADAM17 were markedly elevated. Elevated ADAM family signal scores were linked to adverse clinical outcomes and disruptions in the immune microenvironment and metabolic reprogramming. An ADAM prognosis signal, developed through the utilization of 99 machine learning algorithms, could accurately forecast the survival duration of HCC, achieving an AUC value of approximately 0.9. Conclusions This study represented the inaugural report on the deleterious impact and prognostic significance of ADAM family signals within the tumor microenvironment of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000a5d98243250c138efd38742555d600655a9e5" target='_blank'>
              Unraveling the role of ADAMs in clinical heterogeneity and the immune microenvironment of hepatocellular carcinoma: insights from single-cell, spatial transcriptomics, and bulk RNA sequencing
              </a>
            </td>
          <td>
            Junhong Chen, Qihang Yuan, Hewen Guan, Yuying Cui, Chang Fu, Tianfu Wei, Kai Liu
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pediatric cancers are estimated to be related to a genetic predisposition in about 8 to 10% of cases. This estimation mostly relies on already discovered cancer predisposing genes. However, the numerous new cancer predisposing genes that have been discovered in the past decade thanks to high throughput sequencing in vast cohort of pediatric patients illustrate that new genes may still be unravelled. Moreover, the frequency of early post-zygotic mosaicism is being readdressed thanks to more dedicated sequencing and research. The actual clinical impact of all those discoveries is the main challenge for the community: defining the penetrance, delineating the cancer spectrum and therefore the appropriate surveillance, and setting up innovative follow-up techniques are necessary. Here, we’ll describe recent findings on brain tumor predisposing genes (ELP1 and SMARCB1, among others) to illustrate the increasing recognition of genetic predisposition in pediatric cancers and how the clinical translation of these laboratory findings remain challenging at clinical and ethical levels.
 Citation Format: Franck Bourdeaut. Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6b854ba212ba8f4846fc2365983567ae6c2486" target='_blank'>
              Abstract IA003: Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines
              </a>
            </td>
          <td>
            F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Over the last decade with the advent of new genetic technologies such as CRISPR and more recently described, seekRNA, we now have the ability to transfer genetic materials through retroviruses or other tools to directly edit the aberrant genes and repair them. We describe and categorize the genodermatoses in tabular forms devoting each genodermatosis to its own table which describes the clinical characteristics, genetic inheritance pattern, pathophysiology with molecular and genetic pathways, current treatments and future anticipated treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44072062d6572b1a7db4c4c78619b3ba1799f97" target='_blank'>
              Genodermatoses and Therapeutics on the Horizon: A Review and Table Summary
              </a>
            </td>
          <td>
            Edward M Klepper
          </td>
          <td>2024-08-31</td>
          <td>Journal of Clinical Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-11</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) maintain homeostasis in the hematopoietic ecosystem, which is tightly regulated at multiple layers. Acute myeloid leukemia (AML) is a severe hematologic malignancy driven by genetic and epigenetic changes that lead to the transformation of leukemia stem cells (LSCs). Since somatic mutations in DNA methylation-related genes frequently occur in AML, DNA methylation is widely altered and functions as a starting engine for initiating AML. Additionally, RNA modifications, especially N6-methyladenosine (m6A), also play an important role in the generation and maintenance of the hematopoietic ecosystem, and AML development requires reprogramming of m6A modifications to facilitate cells with hallmarks of cancer. Given the complex pathogenesis and poor prognosis of AML, it is important to fully understand its pathogenesis. Here, we mainly focus on DNA methylation and RNA m6A modification in hematopoiesis and AML and summarize recent advances in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f8b80ecba345e57d84b39e3572cdbd8a3dd106" target='_blank'>
              Epigenetic modifications in hematopoietic ecosystem: a key tuner from homeostasis to acute myeloid leukemia
              </a>
            </td>
          <td>
            Shu-Qi Yao, Rongxia Guo, Wen Tian, Yanbing Zheng, Jin Hu, G-X Han, Rong Yin, Fuling Zhou, Haojian Zhang
          </td>
          <td>2024-09-12</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVE
The identification of mutation hot spots in the isocitrate dehydrogenase (IDH) genes is one of the most important cancer genome-wide sequencing discoveries with relevant impact in the treatment of some orphan tumors. These genes were mostly found mutated in lower-grade gliomas (LGGs), acute myeloid leukaemia (AML), myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPNs) and in cholangiocarcinoma. This aberrant genomic condition represents a therapeutic target of great interest in cancer research, especially in AML, given the limitations of currently approved therapies in this field. In this review, we investigate the role of IDH mutation and the mutant IDH (mIDH)- targeted therapies for cholangiocarcinoma and glioma.


METHODS
Here, we provide an overview of the IDH mutation role and discuss its role in tumorigenesis and progression of some solid cancers, in which the therapeutic strategy can be completely changed thanks to these brand-new therapeutic options.


KEY CONTENT AND FINDINGS
The encouraging early clinical data demonstrated to be a proof of concept for investigational mIDH1/2 inhibitors in tumors with a paucity of therapeutic possibilities.


CONCLUSIONS
Moreover, we list the most important randomised clinical trials still active with their preliminary results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a20edde7b8a46ad9d9cedbb5e90b751d02d2ad2f" target='_blank'>
              Therapeutic inhibition of isocitrate dehydrogenase mutations in glioma and cholangiocarcinoma: new insights and promises-a narrative review.
              </a>
            </td>
          <td>
            V. Internò, A. Melaccio, Pasquale Vitale, Roberta Spedaliere, Massimo Buonfantino, R. Messina, Anna Laura Lippolis, Francesco Signorelli, Raffaele Addeo, Francesco Giuliani
          </td>
          <td>2023-01-01</td>
          <td>Chinese clinical oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
The identification of the most appropriate targeted therapies for advanced cancers is challenging. We performed a molecular profiling of metastatic solid tumors utilizing a comprehensive next-generation sequencing (NGS) assay to determine genomic alterations' type, frequency, actionability, and potential correlations with PD-L1 expression.


METHODS
A total of 304 adult patients with heavily pretreated metastatic cancers treated between January 2019 and March 2021 were recruited. The CLIA-/UKAS-accredit Oncofocus assay targeting 505 genes was used on newly obtained or archived biopsies. Chi-square, Kruskal-Wallis, and Wilcoxon rank-sum tests were used where appropriate. Results were significant for P < .05.


RESULTS
A total of 237 tumors (78%) harbored potentially actionable genomic alterations. Tumors were positive for PD-L1 in 68.9% of cases. The median number of mutant genes/tumor was 2.0 (IQR: 1.0-3.0). Only 34.5% were actionable ESCAT Tier I-II with different prevalence according to cancer type. The DNA damage repair (14%), the PI3K/AKT/mTOR (14%), and the RAS/RAF/MAPK (12%) pathways were the most frequently altered. No association was found among PD-L1, ESCAT, age, sex, and tumor mutational status. Overall, 62 patients underwent targeted treatment, with 37.1% obtaining objective responses. The same molecular-driven treatment for different cancer types could be associated with opposite clinical outcomes.


CONCLUSIONS
We highlight the clinical value of molecular profiling in metastatic solid tumors using comprehensive NGS-based panels to improve treatment algorithms in situations of uncertainty and facilitate clinical trial recruitment. However, interpreting genomic alterations in a tumor type-specific manner is critical.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13a3de4d8a4a44c9264bbeddda708851f093875" target='_blank'>
              Next-generation sequencing-based evaluation of the actionable landscape of genomic alterations in solid tumors: the "MOZART" prospective observational study.
              </a>
            </td>
          <td>
            F. Schettini, Marianna Sirico, M. Loddo, Gareth H Williams, Keeda-Marie Hardisty, Paul Scorer, R. Thatcher, Pablo Rivera, Manuela Milani, C. Strina, G. Ferrero, Marco Ungari, C. Bottin, Fabrizio Zanconati, Nicolò de Manzini, S. Aguggini, Richard Tancredi, E. Fiorio, Antonio Fioravanti, Maurizio Scaltriti, Daniele Generali
          </td>
          <td>2024-08-23</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2b01750179731a3467c6191264b3a3625ebdeb" target='_blank'>
              Altered methylation of imprinted genes in neuroblastoma: implications for prognostic refinement
              </a>
            </td>
          <td>
            Medha Suman, Maja Löfgren, S. Fransson, Jewahri Idris Yousuf, Johanna Svensson, A. Djos, Tommy Martinsson, P. Kogner, T. Kling, Helena Carén
          </td>
          <td>2024-08-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="TRIM44, a tripartite motif (TRIM) family member, is pivotal in linking the ubiquitin-proteasome system (UPS) to autophagy in multiple myeloma (MM). However, its prognostic impact and therapeutic potential remain underexplored. Here, we report that TRIM44 overexpression is associated with poor prognosis in a Multiple Myeloma Research Foundation (MMRF) cohort of 858 patients, persisting across primary and recurrent MM cases. TRIM44 expression notably increases in advanced MM stages, indicating its potential role in disease progression. Single-cell RNA sequencing across MM stages showed significant TRIM44 upregulation in smoldering MM (SMM) and MM compared to normal bone marrow, especially in patients with t(4;14) cytogenetic abnormalities. This analysis further identified high TRIM44 expression as predictive of lower responsiveness to proteasome inhibitor (PI) treatments, underscoring its critical function in the unfolded protein response (UPR) in TRIM44-high MM cells. Our findings also demonstrate that TRIM44 facilitates SQSTM1 oligomerization under oxidative stress, essential for its phosphorylation and subsequent autophagic degradation. This process supports the survival of PI-resistant MM cells by activating the NRF2 pathway, which is crucial for oxidative stress response and, potentially, other chemotherapy-induced stressors. Additionally, TRIM44 counters the TRIM21-mediated suppression of the antioxidant response, enhancing MM cell survival under oxidative stress. Collectively, our discoveries highlight TRIM44’s significant role in MM progression and resistance to therapy, suggesting its potential value as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016505694bfc0a30037baa711cb807808634a1de" target='_blank'>
              TRIM44, a Novel Prognostic Marker, Supports the Survival of Proteasome-Resistant Multiple Myeloma Cells
              </a>
            </td>
          <td>
            Trung Vu, Yuqin Wang, Annaliese Fowler, Anton Simieou, Nami McCarty
          </td>
          <td>2024-08-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Telomeres—special DNA–protein structures at the ends of linear eukaryotic chromosomes—define the proliferation potential of cells. Extremely short telomeres promote a DNA damage response and cell death to eliminate cells that may have accumulated mutations after multiple divisions. However, telomere elongation is associated with the increased proliferative potential of specific cell types, such as stem and germ cells. This elongation can be permanent in these cells and is activated temporally during immune response activation and regeneration processes. The activation of telomere lengthening mechanisms is coupled with increased proliferation and the cells’ need for energy and building resources. To obtain the necessary nutrients, cells are capable of finely regulating energy production and consumption, switching between catabolic and anabolic processes. In this review, we focused on the interconnection between metabolism programs and telomere lengthening mechanisms during programmed activation of proliferation, such as in germ cell maturation, early embryonic development, neoplastic lesion growth, and immune response activation. It is generally accepted that telomere disturbance influences biological processes and promotes dysfunctionality. Here, we propose that metabolic conditions within proliferating cells should be involved in regulating telomere lengthening mechanisms, and telomere length may serve as a marker of defects in cellular functionality. We propose that it is possible to reprogram metabolism in order to regulate the telomere length and proliferative activity of cells, which may be important for the development of approaches to regeneration, immune response modulation, and cancer therapy. However, further investigations in this area are necessary to improve the understanding and manipulation of the molecular mechanisms involved in the regulation of proliferation, metabolism, and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ab06d09eb5b7b71fe7be1c13b67665bab6c417" target='_blank'>
              Telomere Reprogramming and Cellular Metabolism: Is There a Link?
              </a>
            </td>
          <td>
            M. Rubtsova, Denis A. Nikishin, M. Y. Vyssokikh, Maria S. Koriagina, Andrey V. Vasiliev, O. A. Dontsova
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Sarcomas constitute a large group of rare, heterogeneous cancers of the bone and soft tissue affecting children and young adults. There are over 80 clinically distinct subtypes of sarcoma, and their heterogeneity is a contributing factor to the moderate and poor survival outcomes observed in primary and metastatic disease, respectively. While they are heterogeneous, many subtypes harbor a balanced chromosomal translocation fusing the Ewing Sarcoma RNA binding protein 1 (EWSR1) gene with a gene encoding a transcription factor (TF). This translocation generates two distinct fusions: a canonical fusion consisting of the N-terminal transactivation domain of EWSR1 and the C-terminal DNA-binding domain of a TF, and a reciprocal fusion fusing the N-terminus of the TF and C-terminal RNA-binding domain of EWSR1. Canonical fusion protein products exhibit aberrant TF and pioneer factor activity, leading to alterations in transcriptional programs and the epigenetic landscape within a cell that contributes to oncogenesis. The functions of the reciprocal fusion proteins, however, have remained elusive. Ewing sarcoma (ES) and clear cell sarcoma (CCS), a bone and soft tissue sarcoma, respectively, are two subtypes that have been shown to express reciprocal fusions. The ES fusion implicates EWSR1 and Friend Leukemia Integration Factor 1 (FLI1), while the CCS fusion involves Activating Transcription Factor 1 (ATF1). A challenge in studying the roles of these and other fusion proteins in sarcoma development is the lack of effective model systems, primarily due to their toxicity upon exogenous introduction into fusion-naïve cells. Here, we outline the development of a cumate-inducible lentiviral vector to study the canonical and reciprocal EWSR1 fusion proteins present in ES and CCS. To track cellular localization of the fusion proteins and distinguish fusions from their respective wildtype proteins, we designed fusion sequences with N-terminal fluorescent tags. Tightly controlled expression of canonical and reciprocal fusion proteins reduces their toxicity when expressed in cells. We have successfully established 293T and HT-1080 cells that stably express these vectors. We have also validated cumate-inducible expression of fluorescently tagged canonical and reciprocal FLI1 and ATF1 fusions at the RNA and protein levels in both cell lines. The canonical fusion proteins produced by our model have the expected effects on expression levels of known downstream canonical fusion target genes. Initial cellular localization studies using our model and fluorescence microscopy indicate that reciprocal fusion proteins may localize in the nucleus, which was previously unknown. Overall, our cumate-inducible model represents a viable system for fusion protein expression. Using this model, we aim to gain insight into the roles of reciprocal fusions in ES and CCS development with hopes of uncovering potentially targetable vulnerabilities, opening the door for the development of new treatments in a patient population in need of novel and efficacious therapies.
 Citation Format: Sarah Gawlak, Joyce Ohm. Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B079.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6104d5296aa07ecd63e21655a418e97199e7a8ba" target='_blank'>
              Abstract B079: Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis
              </a>
            </td>
          <td>
            Sarah Gawlak, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The acquisition of subsequent genetic lesions (clonal evolution, CE) and/or the expansion of existing clones (CEXP) contributes to clonal dynamics (CD) in myelodysplastic syndromes (MDS). Although CD plays an important role in high‐risk patients in disease progression and transformation into acute myeloid leukemia (AML), knowledge about CD in lower‐risk MDS (LR‐MDS) patients is limited due to lack of robust longitudinal data considering the long clinically stable courses of the disease. In this retrospective analysis, we delineate the frequency and the prognostic impact of CD in an unselected real‐world cohort of LR‐MDS patients. We screened 68 patients with a median follow‐up of 40.5 months and a median of 7.5 (range: 2–22) timepoints for CE and CEXP detected by chromosomal banding analysis, fluorescence in situ hybridization, sequencing, and molecular karyotyping. In 30/68 patients, 47 CE events and a CD rate of 1 event per 4 years were documented. Of note, patients with at least 1 CE event had an increased probability for subsequent treatment. Unexpectedly, CE did not correlate with inferior outcomes, which could be reasonably explained by CD detection triggering the subsequent start of a disease‐modifying therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea97769d53cd530a3310f70504efa64dd62b64b" target='_blank'>
              Comprehensive sequential genetic analysis delineating frequency, patterns, and prognostic impact of genomic dynamics in a real‐world cohort of patients with lower‐risk MDS
              </a>
            </td>
          <td>
            P. Mazzeo, C. Ganster, John Wiedenhöft, K. Shirneshan, K. Rittscher, Elzbieta B. Brzuszkiewicz, D. Steinemann, Maximilian Schieck, C. Müller-Thomas, Hannes Treiber, F. Braulke, U. Germing, K. Sockel, E. Balaian, J. Schanz, U. Platzbecker, Katharina S. Götze, D. Haase
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Combined tracking of clonal evolution and chimeric cell phenotypes could enable detection of the key cellular populations associated with response following therapy, including after allogeneic hematopoietic stem cell transplantation (HSCT). We demonstrate that mitochondrial DNA (mtDNA) mutations co-evolve with somatic nuclear DNA mutations at relapse post-HSCT and provide a sensitive means to monitor these cellular populations. Further, detection of mtDNA mutations via single-cell ATAC with select antigen profiling by sequencing (ASAP-seq) simultaneously determines not only donor and recipient cells, but also their phenotype, at frequencies of 0.1-1%. Finally, integration of mtDNA mutations, surface markers, and chromatin accessibility profiles enables the phenotypic resolution of leukemic populations from normal immune cells, thereby providing fresh insights into residual donor-derived engraftment and short-term clonal evolution following therapy for post-transplant leukemia relapse. As throughput evolves, we envision future development of single-cell sequencing-based post-transplant monitoring as a powerful approach for guiding clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454ae3e4512380b5fd1a0c7892e2f418e6fca333" target='_blank'>
              Tracking rare single donor and recipient immune and leukemia cells after allogeneic hematopoietic cell transplantation using mitochondrial DNA mutations.
              </a>
            </td>
          <td>
            Livius Penter, Nicoletta Cieri, Katie Maurer, M. Kwok, Haoxiang Lyu, Wesley S Lu, Giacomo Oliveira, Satyen H Gohil, I. Leshchiner, Caleb A. Lareau, Leif S. Ludwig, Donna S. Neuberg, Haesook T Kim, Shuqiang Li, L. Bullinger, Jerome Ritz, Gaddy Getz, Jacqueline S Garcia, Robert J. Soiffer, K. Livak, Catherine J. Wu
          </td>
          <td>2024-09-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Endometrial carcinoma (EC) is a prevalent gynecological malignancy that poses a significant threat to women’s health worldwide. However, its pathogenesis and underlying mechanisms remains unclear. In this study, expression quantitative trait loci data, Mendelian randomization analysis, and differential expression analysis were performed to identify potential targets. A prognostic risk signature was subsequently constructed for EC patients based on the expression of these genes. Four bioinformatics algorithms, including generalized linear model, extreme gradient boosting, support vector machine, and random forest, were used to identify hub genes in EC. The expression of ring finger protein 144A (RNF144A) was validated using quantitative real-time polymerase chain reaction. Cellular proliferation and migration ability were evaluated using CCK-8 and Transwell assays, respectively. The genes RNF144A, ketohexokinase, and Rab interacting lysosomal protein like 2 were identified as potential targets for EC. Their differential expression was observed in EC patients, and Mendelian randomization analysis revealed a negative correlation between these genes and the development of EC. Mechanistic analyses suggested a strong association between these genes and the tumor immune microenvironment. The constructed risk signature was significantly associated with the prognosis, age, cancer stage, and grade of EC patients. Furthermore, based on interacted model algorithms, RNF144A was identified as a hub gene in EC. It was found to be significantly downregulated in EC samples, and its expression was positively correlated with the stage and grade of EC patients. In vitro experiments showed that overexpression of RNF144A significantly promoted cell growth and migration in EC cells. In conclusion, this study provides insights into the molecular mechanisms underlying EC progression and identifies preliminary candidate biomarkers for the development of EC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30ccab2586145d3cccf21eccbbc93cc5b295b360" target='_blank'>
              RNF144A as a potential risk gene for endometrial carcinoma: Insights from Mendelian randomization, bulk RNA sequencing, single-cell RNA, and experimental analysis
              </a>
            </td>
          <td>
            Xiqin Ruan, Ni Wang, Qingwen Xie, Yi Du
          </td>
          <td>2024-10-04</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Prostate cancer is frequently treated with radiotherapy. Unfortunately, aggressive radioresistant relapses can arise, and the molecular underpinnings of radioresistance are unknown. Modern clinical radiotherapy is evolving to deliver higher doses of radiation in fewer fractions (hypofractionation). We therefore analyzed genomic, transcriptomic, and proteomic data to characterize prostate cancer radioresistance in cells treated with both conventionally fractionated and hypofractionated radiotherapy. Independent of fractionation schedule, resistance to radiotherapy involved massive genomic instability and abrogation of DNA mismatch repair. Specific prostate cancer driver genes were modulated at the RNA and protein levels, with distinct protein subcellular responses to radiotherapy. Conventional fractionation led to a far more aggressive biomolecular response than hypofractionation. Testing preclinical candidates identified in cell lines, we revealed POLQ (DNA Polymerase Theta) as a radiosensitizer. POLQ-modulated radioresistance in model systems and was predictive of it in large patient cohorts. The molecular response to radiation is highly multimodal and sheds light on prostate cancer lethality. Significance: Radiation is standard of care in prostate cancer. Yet, we have little understanding of its failure. We demonstrate a new paradigm that radioresistance is fractionation specific and identified POLQ as a radioresistance modulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9fa9b6b4bc258239503f1fab9f687905a6c0b2b" target='_blank'>
              The Proteogenomics of Prostate Cancer Radioresistance
              </a>
            </td>
          <td>
            Roni Haas, Gavin Frame, Shahbaz Khan, B. Neilsen, Boon Hao Hong, Celestia P X Yeo, Takafumi N. Yamaguchi, E.H.W. Ong, Wenyan Zhao, Benjamin Carlin, E. Yeo, Kah M Tan, Yuan Zhe Bugh, Chenghao Zhu, Rupert Hugh-White, J. Livingstone, Dennis J J Poon, Pek L Chu, Yash Patel, Shu Tao, V. Ignatchenko, Natalie J Kurganovs, G. Higgins, Michelle R Downes, A. Loblaw, Danny Vesprini, A. Kishan, Melvin L K Chua, T. Kislinger, P. Boutros, Stanley K Liu
          </td>
          <td>2024-08-21</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, Jae-Ll Park, Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>