<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-11-04 06:33:35 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>8</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41df9adbcb86e982509871382b3eba588480f4fb" target='_blank'>
              Base editing screens define the genetic landscape of cancer drug resistance mechanisms.
              </a>
            </td>
          <td>
            Matthew A Coelho, Magdalena E Strauss, Alex Watterson, Sarah Cooper, S. Bhosle, G. Illuzzi, Emre Karakoc, Cansu Dincer, Sara F Vieira, Mamta Sharma, Marie Moullet, Daniela Conticelli, Jonas Koeppel, Katrina McCarten, C. M. Cattaneo, Vivien Veninga, Gabriele Picco, Leopold Parts, J. Forment, E. Voest, J. Marioni, Andrew Bassett, Mathew J. Garnett
          </td>
          <td>2024-10-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Tumors consist of cancer cells with different genetic, epigenetic, and phenotypic properties. Cancer stem cells are an important subpopulation of heterogeneous cancer cells and are capable of initiating and propagating tumors. The term cancer stem cells has become broader in efforts to understand their phenotypic plasticity to switch fates between self-renewal and differentiation. Cancer stem cell plasticity is significantly associated with the initiation of metastasis, resistance to therapy, and tumor recurrence. With our broadened knowledge of epigenetic regulation and metabolic reprogramming as key elements enabling such capabilities, an expansive body of literature has demonstrated the functional importance of each element in contributing to cancer stem cell characteristics. Recently, the direct interplay between epigenetic regulation and metabolic reprogramming has begun to be appreciated in the context of cancer stem cells with growing interest. In this review, we discuss the mechanisms by which cancer stem cells orchestrate the reciprocal regulation of cellular metabolism and epigenetic alterations. In the discussion, compelling, unanswered questions on this topic have been elaborated for the interest of the research community and how recent technological developments help tackle such research ideas. A comprehensive understanding of cancer stem cell attributes that are largely governed by epigenetic and metabolic reprogramming would enable the advancement of precise therapeutic options and the prediction of better responses to drugs, holding great promise in cancer treatment and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a352f514fc793046331539c8fc8b91b79542075" target='_blank'>
              Interplay between epigenetics and metabolism controls cancer stem cell plasticity
              </a>
            </td>
          <td>
            Jee-Eun Choi, Inwha Baek
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most commonly diagnosed type of cancer worldwide and is responsible for numerous deaths. 5-fluorouracil (5-FU) is an effective chemotherapy drug commonly used in the treatment of CRC, either as monotherapy or in combination with other drugs. However, half of CRC cases are resistant to 5-FU-based therapies. To contribute to the understanding of the mechanisms underlying CRC resistance or recurrence after 5-FU-based therapies, we performed a comprehensive study integrating in silico, in vitro, and in vivo approaches. We identified differentially expressed genes and enrichment of pathways associated with recurrence after 5-FU-based therapies. Using these bioinformatics data as a starting point, we selected a group of drugs that restored 5-FU sensitivity to 5-FU resistant cells. Interestingly, treatment with the novel Rac1 inhibitor, 1A-116, reversed morphological changes associated with 5-FU resistance.. Moreover, our in vivo studies have shown that 1A-116 affected tumor growth and the development of metastasis. All our data allowed us to postulate that targeting Rac1 represents a promising avenue for the development of new treatments for patients with CRC resistant to 5-FU-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d58edbfaa7adfef34124f9175133ca3a305e44a" target='_blank'>
              Overcoming Therapy Resistance in Colorectal Cancer: Targeting the Rac1 Signaling Pathway as a Potential Therapeutic Approach
              </a>
            </td>
          <td>
            L. Anselmino, F. Malizia, Aylén Avila, Nahuel Cesatti Laluce, Macarena Mamberto, Lucía C. Zanotti, Cecilia Farré, Vincent Sauzeau, M. Menacho Márquez
          </td>
          <td>2024-10-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8682cef5538b95d1f2e68873fc143c3785f2658e" target='_blank'>
              Reciprocal inhibition of NOTCH and SOX2 shapes tumor cell plasticity and therapeutic escape in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Morgane Fournier, Joaquim Javary, Vincent Roh, Nadine Fournier, Freddy Radtke
          </td>
          <td>2024-10-30</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Neuroblastoma is the most prevalent solid tumor in early childhood, with a 5-year overall survival rate of 40–60% in high-risk cases. Therefore, the identification of novel biomarkers for the diagnosis, prognosis, and therapy of neuroblastoma is crucial for improving the clinical outcomes of these patients. In this study, we conducted the whole-exome sequencing of 48 freshly frozen tumor samples obtained from the Biobank. Somatic variants were identified and selected using a bioinformatics analysis pipeline. The mutational signatures were determined using the Mutalisk online tool. Cancer driver genes and druggable mutations were predicted using the Cancer Genome Interpreter. The most common mutational signature was single base substitution 5. MUC4, MUC16, and FLG were identified as the most frequently mutated genes. Using the Cancer Genome Interpreter, we identified five recurrent cancer driver mutations spanning MUC16, MUC4, ALK, and CTNND1, with the latter being novel and containing a missense mutation, R439C. We also identified 11 putative actionable mutations including NF1 Q1798*, Q2616*, and S636X, ALK F1174L and R1275Q, SETD2 P10L and Q1829E, BRCA1 R612S, NOTCH1 D1670V, ATR S1372L, and FGFR1 N577K. Our findings provide a comprehensive overview of the novel information relevant to the underlying molecular pathogenesis and therapeutic targets of neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b01557419f7eaa825838a38b11cbcc7b2c62d9" target='_blank'>
              Whole-Exome Sequencing Reveals Novel Candidate Driver Mutations and Potential Druggable Mutations in Patients with High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Giorgio D’Alessandris, C. Ronchi, Natakorn Nokchan, Praewa Suthapot, Pongsakorn Choochuen, Natthapon Khongcharoen, S. Hongeng, U. Anurathapan, K. Surachat, S. Sangkhathat, Thai Pediatric Cancer Atlas Tpca, Consortium
          </td>
          <td>2024-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401dfb6b1e61a2b7931608ee7b632746b618204c" target='_blank'>
              Potential promising of synthetic lethality in cancer research and treatment.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Behnam Najafiyan, Reza Morovatshoar, Mahdieh Khorsandi, Adib Dashtizadeh, Arash Kiani, Farnoosh Farzam, Kimia Sadat Kazemi, Mohsen Nabi Afjadi
          </td>
          <td>2024-09-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study delves into the proteomic intricacies of drug-resistant cells (DRCs) within prostate cancer, which are known for their pivotal roles in therapeutic resistance, relapse, and metastasis. Utilizing single-cell proteomics (SCP) with an optimized high-throughput Data Independent Acquisition (DIA) approach with the throughput of 60 sample per day, we characterized the proteomic landscape of DRCs in comparison to parental PC3 cells. This optimized DIA method allowed for robust and reproducible protein quantification at the single-cell level, enabling the identification and quantification of over 1,300 proteins per cell on average. Distinct proteomic sub-clusters within the DRC population were identified, closely linked to variations in cell size. The study uncovered novel protein signatures, including the regulation of proteins critical for cell adhesion and metabolic processes, as well as the upregulation of surface proteins and transcription factors pivotal for cancer progression. Furthermore, by integrating SCP and single-cell RNA-seq (scRNA-seq) data, we identified six upregulated and ten downregulated genes consistently altered in drug-treated cells across both SCP and scRNA-seq platforms. These findings underscore the heterogeneity of DRCs and their unique molecular signatures, providing valuable insights into their biological behavior and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3153689159ed447ea718eb77046a432729554a1" target='_blank'>
              Single-Cell Proteomic and Transcriptomic Characterization of Drug-Resistant Prostate Cancer Cells Reveals Molecular Signatures Associated with Morphological Changes
              </a>
            </td>
          <td>
            Jong-Su Woo, Michael A. Loycano, Md Amanullah, Jiang Qian, Sarah Amend, Kenneth J Pienta, Hui Zhang
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ability of cancer cells to change and adapt poses a critical challenge to identifying curative solutions. Tumor evolution has been extensively studied from a genetic perspective, to guide clinicians in selecting the most appropriate therapeutic option based on a patient's mutational profile. However, several studies reported that tumors can evolve toward more aggressive stages or become resistant to therapies without changing their genetic makeup. Indeed, several cell-intrinsic and cell-extrinsic mechanisms contribute to tumor evolution. In this viewpoint, I focus on how chromatin, epigenetic, and transcriptional changes contribute to tumor evolution, allowing cancer cells to transition to different cell states and bypass response to therapies. Although tumor nongenetic evolution is harder to trace and predict, understanding its principles might open new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0869302e828b2c81eac62f89984d51de011a3b74" target='_blank'>
              Nongenetic evolution of the tumor: from challenges to new therapeutic opportunities.
              </a>
            </td>
          <td>
            E. Oricchio
          </td>
          <td>2024-10-18</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole-genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2,726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirically testing all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated two compounds as potent PGCC inhibitors. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f39f65e370bc9aafe05599fe5e71c2036a3f471" target='_blank'>
              High-Throughput Empirical and Virtual Screening to Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a hematological malignancy associated with poor prognosis and relapses. MM has a strong inherited component, and while the mutational and transcriptomic landscapes of MM have been characterized extensively, the molecular consequences of germline genetic variation in MM have not been previously explored. Here, we have leveraged matched genotype and expression data from 607 patients to map the germline and somatic regulatory variation in MM. We find that non-coding variation shapes the transcriptional landscape of MM, contributing to overall survival. These regulatory variants alter the binding of transcription factors with known importance in MM, such as IRF4, providing insights into the underlying mechanisms. These data contribute to the increasing understanding of the genetic, molecular, and cellular forces shaping MM risk and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5c5bba32932ad19d6d4a4784baebd131dde155a" target='_blank'>
              Germline and somatic genetic effects on gene expression and outcome in patients with Multiple Myeloma
              </a>
            </td>
          <td>
            H. Natri, Linh Bui, L. Peter, Bianca Argente, A. Federico, A. Gutierrez, Mei-i Chung, Jonathan Keats, Nicholas E Banovich
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>
              Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.
              </a>
            </td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to KRAS therapy in pancreatic ductal adenocarcinoma (PDAC) involves cellular plasticity, particularly the epithelial-to-mesenchymal transition (EMT), which poses challenges for effective targeting. Chronic pancreatitis, a known risk factor for PDAC, elevates TGFβ levels in the tumor microenvironment (TME), promoting resistance to KRAS therapy. Mechanistically, TGFβ induces the formation of a novel protein complex composed of SMAD3, SMAD4, and the nuclear factor NFAT5, triggering EMT and resistance by activating key mediators such as S100A4. Inhibiting NFAT5 attenuates pancreatitis-induced resistance to KRAS inhibition and extends mouse survival. Additionally, TGFβ stimulates PDAC cells to secrete CCL2, recruiting macrophages that contribute to KRAS bypass through paracrine S100A4. Our findings elucidate the role of TGFβ signaling in EMT-associated KRAS therapy resistance and identify NFAT5 as a druggable target. Targeting NFAT5 could disrupt this regulatory network, offering a potential avenue for preventing resistance in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9e8d46b9839aa24c7f0f53fe6a51ff006d43ff" target='_blank'>
              NFAT5 governs cellular plasticity-driven resistance to KRAS-targeted therapy in pancreatic cancer.
              </a>
            </td>
          <td>
            Daiyong Deng, Habeebunnisa Begum, Tong Liu, Jiangyan Zhang, Qiang Zhang, Ting-Yu Chu, Hong Li, A. Lemenze, M. Hoque, Patricia Soteropoulos, Pingping Hou
          </td>
          <td>2024-10-21</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN amplification is classified as high risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in patients with neuroblastoma, and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody nivolumab. Analysis of single-cell RNA-Seq datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multiomics approach, we uncovered H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Gemcitabine-based monotherapy or combination therapy has become the standard treatment for locally advanced and metastatic pancreatic cancer. However, the emergence of resistance within weeks of treatment severely compromises therapeutic efficacy. The intricate biological process of gemcitabine resistance in pancreatic cancer presents a complex challenge, as the underlying mechanisms remain unclear. Identifying the target protein of gemcitabine is crucial for studying its drug-resistance mechanism. An activity-based probe is a powerful tool for studying drug target proteins, but the current lack of activity-based gemcitabine probes with robust biological activity hinders research on gemcitabine. In this study, we developed three active probes based on gemcitabine, among which Gem-3 demonstrated excellent stability and labeling efficacy. We utilized Gem-3 in conjunction with chemical proteomics to identify intracellular target proteins. We identified 79 proteins that interact with gemcitabine, most of which were previously unknown and represented various functional classes. Additionally, we validated the increased expression of IFIT3 and MARCKS in drug-resistant cells, along with the activation of the NF-κB signaling pathway. These findings substantially contribute to our comprehension of gemcitabine's target proteins and further our understanding of the mechanisms driving gemcitabine resistance in pancreatic cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ac59d987afe9d30b0e83196487e44b2e63262f1" target='_blank'>
              Identification of Aberrant Expression of Gemcitabine-Targeting Proteins in Drug-Resistant Cells Using an Activity-Based Gemcitabine Probe.
              </a>
            </td>
          <td>
            Xiaomei Zhu, Yuqing Yuan, Kai Wang, Wei Shen, Qing Zhu
          </td>
          <td>2024-10-28</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precision gene editing enables massively parallel identification of cancer-promoting genes Since its discovery in 2012 as a gene-editing tool, CRISPR-Cas9 has been touted as a turning point for drug discovery. Indeed, CRISPR-based technologies have an unprecedented power to interrogate the mechanistic underpinnings of disease by systematically testing the cellular effects of targeted genetic alterations (1). Disease hallmarks specific to cancer, such as unchecked cell proliferation and drug resistance, readily lend themselves to massively parallel CRISPR screens. In these screens, individual cells harboring a guide RNA that also acts as a barcode for a DNA edit of interest may acquire a competitive or diminished proliferative advantage relative to other cells in the same test pool. This allows the perturbation of thousands of genes in a single experiment. The resulting genetic maps generated in cancer cell lines (CCLs) and in cancer-derived tissues now support exceptional speed and economy in the rationalization of new drug development and may shorten bench-to-bedside timelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2013426359d6875b2659e5a9c78a646561186a" target='_blank'>
              Impact of CRISPR in cancer drug discovery
              </a>
            </td>
          <td>
            Jason Moffat, Alexis C. Komor, Lawrence Lum
          </td>
          <td>2024-10-25</td>
          <td>Science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is an aggressive malignancy. The inherent resistance of GBM to radiotherapy poses great challenges for clinical treatment.


OBJECTIVES
The primary objective of this study is to explore the molecular mechanisms of radiotherapy resistance in GBM and identify the key influencing factors that contribute to this phenomenon.


METHODS
The single-cell RNA sequencing (scRNA-seq) data of GBM were downloaded from the Gene Expression Omnibus (GEO) database. Cells were clustered using the Seurat R package, and the clusters were annotated using the CellMarker database. Pseudotime analysis was conducted using Monocle2. Marker scores were calculated based on the RNA-seq data of GBM from the UCSC database, and the enrichment of Hallmark gene sets was measured with the AUCell package. Furthermore, the most frequently mutated genes were identified using the simple nucleotide variation data from The Cancer Genome Atlas (TCGA) applying the maftools package.


RESULTS
This study identified two oligodendrocyte subsets (ODC3 and ODC4) as radiotherapy-resistant groups in GBM. Enrichment and Pseudotime analysis revealed that the inflammatory response and immune activation pathways were enriched in ODC3, while the cell division and interferon response pathways were enriched in ODC4. The enrichment scores of hallmark gene sets further confirmed that ODC3 and ODC4 subpopulations developed radiotherapy resistance via distinct molecular mechanisms. Analysis of gene mutation frequencies showed that TP53 exhibited the most significant change in mutation frequency, indicating that it was an important risk factor involved in radiotherapy resistance in GBM.


CONCLUSION
We identified two ODC subpopulations that exhibited resistance to radiotherapy, providing a new perspective and potential targets for personalized treatment strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593a813498357e1fd5d3f302a2e0d7d52d1643a" target='_blank'>
              Analysis of the Mechanism Underlying Radiotherapy Resistance Caused by Oligodendroglia Cells in Glioblastoma by Applying the Single-cell RNA Sequencing Technology.
              </a>
            </td>
          <td>
            Qinghua Yuan, Weida Gao, Mian Guo, Bo Liu
          </td>
          <td>2024-09-12</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), a leading cause of cancer mortality in the United States, presents significant treatment challenges due to its late diagnosis and poor prognosis. Despite advances, the five-year survival rates remain dismally low, with only a fraction of patients eligible for potentially curative surgical interventions. This review aims to comprehensively examine the current landscape of targeted therapies in PDAC, focusing on recent developments in precision medicine approaches. We explore various molecular targets, including KRAS mutations, DNA damage repair deficiencies, mismatch repair pathway alterations, and rare genetic fusions. The review discusses emerging therapies, such as PARP inhibitors, immune checkpoint inhibitors, and novel targeted agents, like RET and NTRK inhibitors. We analyze the results of key clinical trials and highlight the potential of these targeted approaches in specific patient subgroups. Recent developments in PDAC research have emphasized precision oncology, facilitated by next-generation sequencing and the identification of genetic and epigenetic alterations. This approach tailors treatments to individual genetic profiles, improving outcomes and reducing side effects. Significant strides have been made in classifying PDAC into various subtypes, enhancing therapeutic precision. The identification of specific mutations in genes like KRAS, along with advancements in targeted therapies, including small molecule inhibitors, offers new hope. Furthermore, emerging therapies targeting DNA repair pathways and immunotherapeutic strategies also show promising results. As research evolves, integrating these targeted therapies with conventional treatments might improve survival rates and quality of life for PDAC patients, underscoring the shift towards a more personalized treatment paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0623bca1b0876fdbb0324db3ab4221bfe60d5eb8" target='_blank'>
              Targeted Therapies in Pancreatic Cancer: A New Era of Precision Medicine
              </a>
            </td>
          <td>
            Bingyu Li, Qiong Zhang, Claire Castaneda, Shelly Cook
          </td>
          <td>2024-09-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="While FLT3 inhibitors have significantly improved the treatment of aggressive FLT3-mutated acute myeloid leukemia (AML), the emergence of resistance remains as a major challenge. Here, we applied our recently developed single-cell lineage-tracing method ReSisTrace to identify cells that are pre-resistant or pre-sensitive to FLT3 inhibitors midostaurin and quizartinib in FLT3-ITD-positive AML. By comparing the gene expression profiles of these cells, we unraveled the transcriptional pre-resistance signatures, including G1 to S phase transition 1 (GSPT1) gene. Targeting GSPT1 with the small molecule CC-90009 exhibited strong synergistic effect when combined with FLT3 inhibitors in the FLT3-ITD-mutated MOLM-13 and MV4-11 cell lines and primary AML patient samples. Further, we identified novel compounds that induced transcriptomic changes opposite to the pre-resistance signatures, thereby driving cells to FLT3 inhibitor-sensitive states. Vistusertib (mTOR inhibitor), linsitinib (IGF1R and insulin receptor inhibitor), and meisoindigo (IGF1R and Src family kinase inhibitor), all inhibiting pathways parallel to or downstream of oncogenic FLT3 signaling, were predicted and validated to pre-sensitize the FLT3-ITD-mutated cell lines and primary cells to FLT3 inhibitors. Collectively, these findings demonstrate the validity of our lineage-tracing method in unveiling pre-existing transcriptional features of treatment vulnerability in hematological cancers, and elucidate novel strategies for enhancing FLT3 inhibitor treatment efficacy in FLT3-ITD-positive AML by preventing the emergence of treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0428ff40d99003abc28b0d9e495047e8ef959354" target='_blank'>
              Uncovering pre-sensitizing agents to FLT3 inhibitors in acute myeloid leukemia with ReSisTrace lineage tracing
              </a>
            </td>
          <td>
            Johanna Eriksson, Shuyu Zheng, Jie Bao, Jun Dai, Wenyu Wang, Anna Vähärautio, Jing Tang
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04d632e2b6ac550f2911954ea295c050c33ca2" target='_blank'>
              Development of a polygenic score predicting drug resistance and patient outcome in breast cancer
              </a>
            </td>
          <td>
            Divya Sahu, Jeffrey Shi, Isaac Andres Segura Rueda, Ajay Chatrath, A. Dutta
          </td>
          <td>2024-10-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8719f1e85495bf521f29d9969611ef5b8435eeb3" target='_blank'>
              The evolutionary theory of cancer: challenges and potential solutions.
              </a>
            </td>
          <td>
            L. Laplane, Carlo C. Maley
          </td>
          <td>2024-09-10</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: The rat osteosarcoma cell line UMR-106 is widely used for the study of bone cancer biology but it has not been well characterized with modern genomic methods. Methods: To better understand the biology of UMR-106 cells we used a combination of optical genome mapping (OGM), long-read sequencing nanopore sequencing and RNA sequencing.The UMR-106 genome was compared to a strain-matched Sprague-Dawley rat for variants associated with human osteosarcoma while expression data were contrasted with a public osteoblast dataset. Results: Using the COSMIC database to identify the most affected genes in human osteosarcomas we found somatic mutations in Tp53 and H3f3a. OGM identified a relatively small number of differences between the cell line and a strain-matched control animal but did detect a ~45 Mb block of amplification that included Myc on chromosome 7 which was confirmed by long-read sequencing. The amplified region showed several blocks of non-contiguous rearranged sequence implying complex rearrangements during their formation and included 14 genes reported as biomarkers in human osteosarcoma, many of which also showed increased transcription. A comparison of 5mC methylation from the nanopore reads of tumor and control samples identified genes with distinct differences including the OS marker Cdkn2a. Conclusions: This dataset illustrates the value of long DNA methods for the characterization of cell lines and how inter-species analysis can inform us about the genetic nature underlying mutations that underpin specific tumor types. The data should be a valuable resource for investigators studying osteosarcoma, in general, and specifically the UMR-106 model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0957ae8779454bb6e3009d9a818af23c55e01e" target='_blank'>
              Characterization of the Rat Osteosarcoma Cell Line UMR-106 by Long-Read Technologies Identifies a Large Block of Amplified Genes Associated with Human Disease
              </a>
            </td>
          <td>
            Alan F. Scott, David W. Mohr, William A. Littrell, Reshma Babu, Michelle Kokosinski, Victoria Stinnett, Janvi Madhiwala, John Anderson, Ying S. Zou, Kathleen L. Gabrielson
          </td>
          <td>2024-09-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cell fate determination during early embryonic development is a complex process modulated by gene expression. The intricate interplay of transcriptional and post-transcriptional regulation is integral to the developmental trajectory of embryogenesis, yet how RNA processing may contribute to early development programming is largely elusive. Leveraging recent technological advances in single-molecule nanopore sequencing, we developed a single-cell long-read transcriptome sequencing technology, allowing a clear view of transcript diversity during zebrafish embryogenesis during pre- and post-zygotic genome activation (ZGA). A closer examination of the dynamic transcript usage and potential alternative splicing revealed that abundant stage-specific transcripts with differential coding potentials are involved in distinct biological functions. Specifically, we identified two cell populations at the onset of ZGA based on isoform diversity instead of gene profiling, which followed divergent developmental trajectories toward the ectoderm and the presumptive ectoderm. These two populations of cells were characterized by divergent splicing regulations linked to differential RNA-binding proteins, including SNRPA and SFPQ. Altogether, using the single-cell long-read transcriptome sequencing strategy, we work has revealed the cell-specific transcriptome dynamics contributing to the cell fate determination during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff8ff37e792e364247b854277ed75fe81491e8a" target='_blank'>
              Deciphering the Cell-Specific Transcript Heterogeneity and Alternative Splicing during the Early Embryonic Development of Zebrafish
              </a>
            </td>
          <td>
            Xiumei Lin, Xue Wang, Chang Liu, Chuanyu Liu, Tao Zeng, Ziqi Yuan, Meidi Hu, Rong Xiang, Kaichen Zhao, Jie Zhou, Shichen Yang, Yang Wang, Kaifeng Meng, Hui Wang, G. He, Rui Zhao, Jiaheng Liu, Yun Huang, Jingfang Pan, Jialu Wang, Junyi Chen, Fei Guo, Yuliang Dong, Xun Xu, Daji Luo, Ying Gu, Longqi Liu, Zhiqiang Dong, Liang Chen
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Breast cancer's complex transcriptional landscape requires an improved understanding of cellular diversity to identify effective treatments. The study of genetic variations among breast cancer subtypes at single‐cell resolution has potential to deepen our insights into cancer progression. Methods In this study, we amalgamate single‐cell RNA sequencing data from patient tumours and matched lymph metastasis, reduction mammoplasties, breast cancer patient‐derived xenografts (PDXs), PDX‐derived organoids (PDXOs), and cell lines resulting in a diverse dataset of 117 samples with 506 719 total cells. These samples encompass hormone receptor positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple‐negative breast cancer (TNBC) subtypes, including isogenic model pairs. Herein, we delineated similarities and distinctions across models and patient samples and explore therapeutic drug efficacy based on subtype proportions. Results PDX models more closely resemble patient samples in terms of tumour heterogeneity and cell cycle characteristics when compared with TNBC cell lines. Acquired drug resistance was associated with an increase in basal‐like cell proportions within TNBC PDX tumours as defined with SCSubtype and TNBCtype cell typing predictors. All patient samples contained a mixture of subtypes; compared to primary tumours HR+ lymph node metastases had lower proportions of HER2‐Enriched cells. PDXOs exhibited differences in metabolic‐related transcripts compared to PDX tumours. Correlative analyses of cytotoxic drugs on PDX cells identified therapeutic efficacy was based on subtype proportion. Conclusions We present a substantial multimodel dataset, a dynamic approach to cell‐wise sample annotation, and a comprehensive interrogation of models within systems of human breast cancer. This analysis and reference will facilitate informed decision‐making in preclinical research and therapeutic development through its elucidation of model limitations, subtype‐specific insights and novel targetable pathways. Key points Patient‐derived xenografts models more closely resemble patient samples in tumour heterogeneity and cell cycle characteristics when compared with cell lines. 3D organoid models exhibit differences in metabolic profiles compared to their in vivo counterparts. A valuable multimodel reference dataset that can be useful in elucidating model differences and novel targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1b9e5e204010f0ff424f3f3d4b1b7c96f99e7a" target='_blank'>
              Single‐cell transcriptional atlas of human breast cancers and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, Brian D Lehmann, Jane E. Visvader, J. C. Harrell
          </td>
          <td>2024-10-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>161</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb72e1941a1af14f302d1d6e0b549c2a4a1aa635" target='_blank'>
              Defining the transcriptome of PIK3CA-altered cells in a human capillary malformation using single cell long-read sequencing
              </a>
            </td>
          <td>
            Michelle A. Wedemeyer, Tianli Ding, Elizabeth A. R. Garfinkle, Jesse J. Westfall, Jaye B. Navarro, Maria Elena Hernandez Gonzalez, Elizabeth A. Varga, Patricia Witman, Elaine R. Mardis, Catherine E. Cottrell, Anthony R. Miller, Katherine E. Miller
          </td>
          <td>2024-10-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract B‐cell acute lymphoblastic leukaemia (B‐ALL) is the most prevalent hematologic malignancy in children and a leading cause of mortality. Managing B‐ALL remains challenging due to its heterogeneity and relapse risk. This study aimed to delineate the molecular features of paediatric B‐ALL and explore the clinical utility of circulating tumour DNA (ctDNA). We analysed 146 patients with paediatric B‐ALL who received systemic chemotherapy. The mutational landscape was profiled in bone marrow (BM) and plasma samples using next‐generation sequencing. Minimal residual disease (MRD) testing on day 19 of induction therapy evaluated treatment efficacy. RNA sequencing identified gene fusions in 61% of patients, including 37 novel fusions. Specifically, the KMT2A‐TRIM29 novel fusion was validated in a boy who responded well to initial therapy but relapsed after 1 year. Elevated mutation counts and maximum variant allele frequency in baseline BM were associated with significantly poorer chemotherapy response (p = 0.0012 and 0.028, respectively). MRD‐negative patients exhibited upregulation of immune‐related pathways (p < 0.01) and increased CD8+ T cell infiltration (p = 0.047). Baseline plasma ctDNA exhibited high mutational concordance with the paired BM samples and was significantly associated with chemotherapy efficacy. These findings suggest that ctDNA and BM profiling offer promising prognostic insights for paediatric B‐ALL management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8abd916dce8e7ef63366da12ae5e6c40a1a52ff" target='_blank'>
              Molecular characterization and biomarker identification in paediatric B‐cell acute lymphoblastic leukaemia
              </a>
            </td>
          <td>
            Yu Du, Xiankai Zhang, M. Sun, Li Yang, F. Long, Shanshan Qi, Lin Luo, Xiaoyan Lv, Chenxuan Wang, Xiaoying Wu, Liuqing Zhu, Qiuxiang Ou, Hao Xiong
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="TP53 functions as a central regulator in response to DNA damage and other cell stressors by inducing the expression of many protective pathways such as cell cycle arrest and apoptosis. Consequently, this gene is often found disrupted in human cancers. Elephants are a particularly interesting species for the study of cancer, by virtue of their large number of cell divisions and long lives yet low incidence of cancer. Elephants also possess multiple retrogene copies of TP53, which have previously been shown to induce strong cellular responses to DNA damage. However, most previous studies have largely focused only on African elephant TP53 retrogenes and often in non-native backgrounds. Here we generated CRISPR-Cas9 knockouts of TP53, all 29 TP53 retrogenes, or both in combination in Asian elephant fibroblasts. We find that while there is considerable overlap in the DNA damage responses of the TP53 and retrogene knockouts, there are also many unique pathways enriched in both. In particular, the retrogene knockouts exhibit strong enrichment of many extracellular pathways suggesting they may play a large role in the tumor microenvironment and mitigating metastatic growth. We also find that only a small fraction of these 29 retrogenes appear to be expressed across a variety of tissues and identify three loci that are likely driving this response. This work shows for the first time the transcriptomic effect of these retrogenes within their native background and establishes a foundation for future research into the relative contributions of these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01cd22e9caf6290532e5ca297543c409aee6bcfc" target='_blank'>
              Engineering Asian elephant TP53: TP53 retrogene knockouts activate common and unique cancer-relevant pathways
              </a>
            </td>
          <td>
            Emil Karpinski, Nikil Badey, Esther Mintzer, Asaf Ashkenazy-Titelman, George M. Church
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Expression quantitative trait loci (eQTL) represent genetic variants that regulate gene expression levels. eQTL analysis has become a crucial method for identifying the functional roles of disease-associated genetic variants in the post-genome-wide association study (GWAS) era, yielding numerous significant discoveries. Traditional eQTL analysis relies on whole-genome sequencing combined with bulk RNA-seq, which obscures gene expression differences between cells and thus fails to identify cell type- or state-dependent eQTL. This limitation makes it challenging to elucidate the roles of disease-associated genetic variants under specific conditions. In recent years, with the development and widespread application of single-cell RNA sequencing (scRNA-seq) technology, scRNA-seq-based eQTL (sc-eQTL) research has emerged as a focal point. The advantage of this approach lies in its ability to leverage the resolution and granularity of single-cell sequencing to uncover eQTL that are dependent on cell type, cell state, and cellular dynamics. This significantly enhances our ability to analyze genetic variants associated with gene expression. Consequently, it holds substantial significance for advancing our understanding of the formation of complex organs and the mechanisms underlying disease onset, progression, intervention, and treatment. This review comprehensively examines the recent advancements in sc-eQTL studies, focusing on their development, experimental design strategies, modeling approaches, and current challenges. The aim is to offer researchers novel perspectives for identifying disease-associated loci and elucidating gene regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288571bd0963aa078ce1cb68c068b644a9880e64" target='_blank'>
              Research progress on single-cell expression quantitative trait loci.
              </a>
            </td>
          <td>
            Xiao-Peng Xu, Xiao-Ying Fan
          </td>
          <td>2024-10-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d11267e3052403e1a0f2c358857a352b19807b" target='_blank'>
              RAS-mutant leukaemia stem cells drive clinical resistance to venetoclax.
              </a>
            </td>
          <td>
            Junya Sango, Saul Carcamo, M. Sirenko, Abhishek Maiti, Hager Mansour, Gulay Ulukaya, Lewis E Tomalin, Nataly Cruz-Rodriguez, Tiansu Wang, Malgorzata Olszewska, Emmanuel Olivier, Manon Jaud, B. Nadorp, B. Kroger, Feng Hu, Lewis Silverman, Stephen S Chung, Elvin Wagenblast, R. Chaligné, Ann-Kathrin Eisfeld, Deniz Demircioglu, D. Landau, P. Lito, E. Papaemmanuil, Courtney D. DiNardo, Dan Hasson, Marina Y Konopleva, E. Papapetrou
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf0847e7afd3b17c26839196bb1a05ffa786834" target='_blank'>
              Aneuploidy as a driver of human cancer.
              </a>
            </td>
          <td>
            Eran Sdeor, Hajime Okada, Ron Saad, Tal Ben-Yishay, Uri Ben-David
          </td>
          <td>2024-10-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, particularly in advanced non-small cell lung cancer (NSCLC) and muscle-invasive bladder cancer (MIBC). However, identifying reliable predictive biomarkers for ICI response remains a significant challenge. In this study, we analyzed real-world cohorts of advanced NSCLC and MIBC patients treated with ICIs as first-line therapy. Tumor samples underwent Whole Genome Sequencing (WGS) to identify specific somatic variants and assess tumor mutational burden (TMB). Additionally, mutational signature extraction and pathway enrichment analyses were performed to uncover the underlying mechanisms of ICI response. We also characterized HLA-I haplotypes and investigated LINE-1 retrotransposition. Distinct mutation patterns were identified in patients who responded to treatment, suggesting potential biomarkers for predicting ICI effectiveness. In NSCLC, tumor mutational burden (TMB) did not differ significantly between responders and non-responders, while in MIBC, higher TMB was linked to better responses. Specific mutational signatures and HLA haplotypes were associated with ICI response in both cancers. Pathway analysis showed that NSCLC responders had active inflammatory and immune pathways, while non-responders had pathways such related to FGFR3 and neural crest differentiation associated to resistance mechanisms. In MIBC, responders had alterations in DNA repair, leading to more neoantigens and a stronger ICI response. Importantly, for the first time, we found that LINE-1 activation was positively linked to ICI response, especially in MIBC. These findings reveal promising biomarkers and mechanistic insights, offering a new perspective on predicting ICI response and opening up exciting possibilities for more personalized immunotherapy strategies in NSCLC and MIBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b70492d02c2b0d60bcbf3b1913b9cfd98468b" target='_blank'>
              Genomic Insights Guiding Personalized First-Line Immunotherapy Response in Lung and Bladder Tumors
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, María Gallardo-Gómez, Ana Oitaben, Martin E. Lázaro-Quintela, Luis León, , L. Juaneda-Magdalena, Carme García-Benito, I. Abdulkader, L. Muinelo, J. Paramio, Mónica Martínez-Fernández
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1167f70aefb28c2bd36dcb2ed6f87d34742f535" target='_blank'>
              Single-cell landscape of innate and acquired drug resistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            R. Wegmann, Ximena Bonilla, Ruben Casanova, Stéphane Chevrier, Ricardo Coelho, Cinzia Esposito, Joanna Ficek-Pascual, Sandra Goetze, Gabriele Gut, Francis Jacob, Andrea Jacobs, Jack Kuipers, Ulrike Lischetti, J. Mena, Emanuela S Milani, M. Prummer, Jacobo Sarabia del Castillo, Franziska Singer, Sujana Sivapatham, Nora C Toussaint, Oliver Vilinovszki, M. H. Wildschut, T. Thavayogarajah, D. Malani, R. Aebersold, M. Bacac, N. Beerenwinkel, C. Beisel, B. Bodenmiller, V. Heinzelmann-Schwarz, V. Koelzer, Mitchell P Levesque, H. Moch, L. Pelkmans, Gunnar Rätsch, Markus Tolnay, Andreas Wicki, Bernd Wollscheid, M. Manz, B. Snijder, A. Theocharides
          </td>
          <td>2024-10-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Abstract Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of CTC-IGC was significantly associated with poorer progression-free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13c77b6b25150646e97a913d6046b3f971cd07d" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            James B. Hicks, Michael Schmidt, Amin Nahgdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, Isla Garraway, Michael Lewis, Ana Aparicio, Amado Zurita, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah Amend
          </td>
          <td>2024-10-14</td>
          <td>Research Square</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="MicroRNAs are a conserved class of small, tissue-specific, non-coding RNAs that regulate gene expression to preserve cellular homeostasis. Proper miRNA expression is crucial for physiological balance because it affects numerous genetic pathways, including cell cycle control, proliferation, and apoptosis, through gene expression targeting. Deregulated miRNA expression has been implicated in several cancer types, including prostate cancer (PC), acting as tumor suppressors or oncogenes. Despite the availability of promising therapies to control tumor growth and progression, effective diagnostic and therapeutic strategies for different types of cancer are still lacking. PC continues to be a significant health challenge, particularly its castration-resistant (CRPC) form, which presents major therapeutic obstacles because of its resistance to conventional androgen deprivation treatments. This review explores miRNAs’ critical roles in gene regulation and cancer biology, as well as various miRNA delivery systems, highlighting their potential and the challenges in effectively targeting cancer cells. It aims to provide a comprehensive overview of the status of miRNA research in the fight against CRPC, summarizing miRNA-based therapies’ successes and limitations. It also highlights the promise of miRNAs as therapeutic agents for CRPC, underlining the need for further research to overcome existing challenges and move these therapies toward clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90859ee13ab8e70d2bc0534d1943825261dccf29" target='_blank'>
              MicroRNAs as Promising Therapeutic Agents Against Prostate Cancer Resistant to Castration—Where Are We Now?
              </a>
            </td>
          <td>
            Mariana Ferreira, Mariana Morais, Rui Medeiros, A. Teixeira
          </td>
          <td>2024-10-22</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="

This study was to explore role of immune landscape and the immune cells in acute lymphoblastic leukemia (ALL) progression.



The most prevalent genetic alteration in childhood ALL is the ETV6-RUNX1 fusion. The increased proliferation of B progenitor cells could expedite the disease’s progression due to irregularities in the cell cycle. Nevertheless, the mechanisms by which particular cell clusters influence the cell cycle and promote the advancement of ALL are still not well understood.



This study was to explore role of immune landscape and the immune cells in ALL progression.



Single-cell RNA sequencing (scRNA-seq) data of ETV6-RUNX1 and healthy pediatric samples obtained from GSE132509 were clustered and annotated using the Seurat package, and differentially highly expressed genes identified in each cluster were analyzed using DAVID for pathway annotation. Chromosome amplification and deletion were analyzed using the inferCNV package. SCENIC evaluated the regulation of transcription factors and target gene formation in cells. cellphoneDB and CellChat were served to infer ligand-receptor pairs that mediate interactions between subpopulations. The role of the target gene in regulating ALL progression was assessed using RT-qPCR, Transwell and scratch healing assays.



The bone marrow mononuclear cells (BMMCs) from ETV6-RUNX1 and healthy pediatric samples in GSE132509 were divided into 11 clusters, and B cell cluster 1 was identified as B progenitor cell, which was amplified on chromosome 6p. B progenitor cells were divided into seven clusters. Expression levels of amplified genes in chromosome 6p of B progenitor cell cluster 5 were the highest, and its specific highly expressed genes were annotated to pathways promoting cell cycle progression. Regulons formed in B progenitor cell cluster 5 were all involved in promoting cell cycle progression, so it was regarded as the B progenitor cell cluster that drives cell cycle progression. The key regulator of the B progenitor cell is E2F1, which promotes the migration and invasion ability of the cell line HAP1. The major ligand-receptor pairs that mediate the communication of B progenitor cell cluster 5 with cytotoxic NK/T cells or naive T cells included FAM3C−CLEC2D, CD47−SIRPG, HLAE−KLRC2, and CD47−KLRC2. HLAE−KLRC1 and TGFB1−(TGFBR1+TGFBR2).



This study outlined the immune cell landscape of ETV6-RUNX1 ALL and identified chromosome 6p amplification in B progenitor cells, described the major B progenitor cell cluster driving cell cycle progression and its potential regulatory mechanisms on NK cells and T cells, providing cellular and molecular insights into ETV6-RUNX1 ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de3198365fdda74c6d3b8026f58bc539920ce2b9" target='_blank'>
              Potential molecular mechanisms of ETV6-RUNX1-positive B progenitor cell cluster in acute lymphoblastic leukemia revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Ning Qu, Yue Wan, Xin Sui, Tianyi Sui, Yang Yang
          </td>
          <td>2024-11-01</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric malignancy. Based on gene expression profiling, B-ALL can be classified into distinct transcriptional subtypes with differing disease outcomes. Many of these transcriptional subtypes are defined by mutations in transcription factors and chromatin-modifying enzymes, but how such diverse mutations lead to distinct transcriptional subtypes remains unclear. To illuminate the chromatin regulatory landscape in B-ALL, we analyzed 3D genome organization, open chromatin, and gene expression in 53 primary patient samples. At the level of 3D genome organization, we identified chromatin interactions that vary across transcriptional subtypes. These sites of variable 3D chromatin interactions correlate with local gene expression changes and are enriched for core drivers of B-ALL observed in genome-wide CRISPR knock-out screens. Sites of variable 3D genome interactions are frequently shared across multiple transcriptional subtypes and are enriched for open chromatin sites found in normal B-cell development but repressed in mature B-cells. Within an individual patient sample, the chromatin landscape can resemble progenitor chromatin states at some loci and mature B-cell chromatin at others, suggesting that the chromatin in B-ALL patient tumor cells is in a partially arrested immature state. By analyzing transcriptomic data from large cohorts of B-ALL patients, we identify gene expression programs that are shared across transcriptional subtypes, associated with B-cell developmental stages, and predictive of patient survival. In combination, these results show that the 3D genome organization of B-ALL reflects B-cell developmental stages and helps illustrate how B-cell developmental arrest interacts with transcriptional subtypes to drive B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1e7461bf425fea64b8e5097538780b0bffaf" target='_blank'>
              Heterogeneity in chromatin structure drives core regulatory pathways in B-cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Arko Sen, Zhichao Xu, Sélène T. Tyndale, Jean Yasis, Chae Yun Cho, Rosalind Bump, Sahaana Chandran, Linda Luo, Yi Fu, Lillian Kay Petersen, Max Shokhirev, D. J. Kuo, G. McVicker, Jesse R. Dixon
          </td>
          <td>2024-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Identification of somatic driver mutations in the noncoding genome remains challenging. To comprehensively characterize noncoding driver mutations for pancreatic ductal adenocarcinoma (PDAC), we first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3,614 noncoding somatic mutations (NCSMs). Functional assessment using massively parallel reporter assays (MPRA) identified 178 NCSMs impacting reporter activity (19.45% of those tested). Focused luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near CDKN2A and ZFP36L2. For the latter, CRISPR interference (CRISPRi) further identified ZFP36L2 as a target gene (16.0 - 24.0% reduced expression, P = 0.023 - 0.0047) with disrupted KLF9 binding likely mediating the effect. Our integrative approach provides a catalog of potentially functional noncoding driver mutations and nominates ZFP36L2 as a PDAC driver gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db992992dda460b3aa1dd358fc40078a768ec148" target='_blank'>
              Large-scale multi-omic analysis identifies noncoding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jun Zhong, Aidan O'Brien, Minal Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Maura J. O’Neill, Courtney D. Dill, A. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E. Connelly, Jianxin Shi, H. E. Arda, Jason W. Hoskins, L. Amundadottir
          </td>
          <td>2024-09-24</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) has previously been considered as a single disease. However, it is currently characterized as a heterogeneous group of tumors that differ in histological features, genetic abnormalities, and variable clinical course. In normal cells, energy is produced by the cleavage of chemical bonds in nutrients through the oxidation of fats, proteins, or carbohydrates. Mutational alterations in genes associated with RCC, including VHL, FLCN, PTEN and SDH, lead to abnormal cellular adaptation to changes in oxygen status, iron metabolism and nutrients. The present paper reviews the known genetic abnormalities observed in RCC and their impact on metabolic alterations. Understanding the genetic and metabolic mechanisms underlying RCC is crucial for the development of effective therapies. Targeting specific genetic abnormalities or metabolic pathways represents a promising approach to the RCC treatment. In addition, studies into the metabolic basis of RCC contribute to the development of new biomarkers for early diagnosis and monitoring of the disease. Moreover, investigating the role of VHL, FLCN, PTEN, and SDH genes in the development of RCC provides valuable information on the molecular mechanisms behind the disease. As a result, it may lead to the development of new treatment strategies aimed at restoring the normal function of these genes or compensating for their abnormalities. Overall, an integrated approach to the study of RCC that considers genetic, metabolic, and clinical aspects will ensure that more effective treatments are developed and prognosis for patients with this disease are improved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1601eb66db07531257237b1f84d2ed09ae8e4e" target='_blank'>
              Genetic Variation and Metabolic Basis of Kidney Cancer: New Opportunities for Targeted Therapy
              </a>
            </td>
          <td>
            M. Urmantsev, R. I. Tavabilov, M. R. Bakeev
          </td>
          <td>2024-10-14</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827407d1ee135266c2bdec8a7b4b96ce85a8df0f" target='_blank'>
              A genome-wide CRISPR/Cas9 knockout screen identifies SEMA3F gene for resistance to cyclin-dependent kinase 4 and 6 inhibitors in breast cancer.
              </a>
            </td>
          <td>
            Yuko Kawai, Aiko Nagayama, Kazuhiro Miyao, Makoto Takeuchi, Takamichi Yokoe, Tomoe Kameyama, Xinyue Wang, T. Seki, Maiko Takahashi, T. Hayashida, Yuko Kitagawa
          </td>
          <td>2024-10-01</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Precision cancer medicine is widely established, and numerous molecularly targeted drugs for various tumor entities are approved or in development. Personalized pharmacotherapy in oncology has so far been based primarily on tumor characteristics, e.g., somatic mutations. However, the response to drug treatment also depends on pharmacological processes summarized under the term ADME (absorption, distribution, metabolism, and excretion). Variations in ADME genes have been the subject of intensive research for more than five decades, considering individual patients' genetic makeup, referred to as pharmacogenomics (PGx). The combined impact of a patient's tumor and germline genome is only partially understood and often not adequately considered in cancer therapy. This may be attributed, in part, to the lack of methods for combined analysis of both data layers. Optimized personalized cancer therapies should, therefore, aim to integrate molecular information about the tumor and the germline, taking into account existing PGx guidelines for drug therapy. Moreover, such strategies should provide the opportunity to consider genetic variants of previously unknown functional significance. Bioinformatic analysis methods and corresponding algorithms for data interpretation need to be developed to consider PGx data in interdisciplinary molecular tumor boards, where cancer patients are discussed to provide evidence-based recommendations for clinical management based on individual tumor profiles. Significance Statement The era of personalized oncology has seen the emergence of drugs tailored to genetic variants associated with cancer biology. However, full potential of targeted therapy remains untapped due to the predominant focus on acquired tumor-specific alterations. Optimized cancer care must integrate tumor and patient genomes, guided by pharmacogenomic principles. An essential prerequisite for realizing truly personalized drug treatment of cancer patients is the development of bioinformatic tools for comprehensive analysis of all data layers generated in modern precision oncology programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/722bafa6b9eef8db0fc1994a1490f6eb4c5b03d1" target='_blank'>
              Innovation in cancer pharmacotherapy through integrative consideration of germline and tumor genomes.
              </a>
            </td>
          <td>
            Roman Tremmel, Daniel Hübschmann, E. Schaeffeler, Sebastian Pirmann, Stefan Fröhling, Matthias Schwab
          </td>
          <td>2024-10-15</td>
          <td>Pharmacological reviews</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the deadliest and most common form of brain tumour. The average length of survival for GBM patients remains low, in part, due to a lack of targeted therapeutics. Identiﬁcation and validation of novel therapeutic targets is therefore crucial. Chomobox 2 (CBX2; a component of polycomb repressive complex 1 (PRC1)), represses gene expression and has a pro-oncogenic role in several aggressive cancer subtypes. CBX2 expression is elevated in GBM, however, very little is known about its role in this deadly disease. We aim to assess the phenotypic and gene expression effects of preventing CBX2-chromatin interaction in patient-derived GBM cell models, and to determine the functional mechanisms by which CBX2 promotes GBM progression; thereby determining its potential as a novel therapeutic target.



 Crucial to impactful target validation is the use of translationally relevant models, this study utilises a 3D spheroid model, using patient-derived cells, which more closely recapitulates the in vivo tumour microenvironment. GBM cell monolayers and spheroids were treated with SW2_152F, a selective CBX2 inhibitor. The phenotypic effects of CBX2 inhibition were assessed by cell count, CellTiter-Glo viability assays, ﬂuorescence microscopy, and ﬂow cytometry. Genes and biological pathways differentially regulated following CBX2 inhibition were assessed by RNA-sequencing and Gene Set Enrichment Analysis (GSEA).



 Inhibition of CBX2 induced cell death and reduced cell viability in both 2D and 3D GBM cultures. RNA-seq and GSEA of SW2_152F-treated cells identiﬁed dysregulation of gene signatures involved in pro-oncogenic MYC and E2F signalling pathways, and dysregulation of genes associated with G2/M cell cycle progression.



 Preventing CBX2-chromatin interaction induces GBM cell death. Further research will identify CBX2- chromatin binding sites to characterise the activity and direct regulatory role of CBX2 in GBM. Furthermore, we will assess the phenotypic and gene expression effects of CBX2 inhibition and depletion on ex vivo maintained GBM tissue biopsies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41607adec8f62477f0d798d3212dbec12fcc049c" target='_blank'>
              THE EPIGENETIC REGULATORY PROTEIN CBX2 AS A NOVEL THERAPEUTIC TARGET FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            Ms Danielle Marsh, Prof John Greenman, Dr Mark Wade
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND/AIM
Biomarkers indicating sensitivity to poly ADP-ribose polymerase (PARP) inhibitors have not yet been identified in gastric cancer. PARP inhibitors target homologous recombination deficiency (HRD); however, homologous recombination (HR) induces complex changes in gene expression, which makes it difficult to identify reliable biomarkers. In this study, we identified a multi-gene expression signature as a marker of PARP inhibitor sensitivity in gastric cancer.


MATERIALS AND METHODS
Seven gastric cancer cell lines were evaluated for susceptibility to PARP inhibitors using a growth inhibition assay. Gene expression profiling (GEP) was used to identify differentially expressed genes between PARP inhibitor-sensitive and -resistant cell lines. The resulting gene set was subjected to cluster analysis using tumor samples from 250 patients who underwent gastrectomy for primary gastroesophageal junction and gastric adenocarcinoma. HRD was defined as a truncating mutation in one or more of 22 HR-related genes and HRD scores were calculated using whole-exome sequencing data.


RESULTS
In the growth inhibition assays, the HGC27 and HSC39 cell lines were sensitive to the PARP inhibitors, olaparib, and rucaparib, and were significantly correlated with the GEP results. Seven (2.8%) patients harbored truncating mutations in HR-related genes. A gene expression signature based on the top 100 high and low differentially expressed genes between sensitive and resistant cell lines revealed a patient cluster with a high prevalence of HR-related gene mutations and high HRD scores.


CONCLUSION
The 100-gene expression signature identified in this study may serve as a valuable predictive biomarker for PARP inhibitor sensitivity in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28788afa14a601dcbc3b636a50a525919ea5413e" target='_blank'>
              A Gene Expression Signature that Predicts Gastric Cancer Sensitivity to PARP Inhibitor Therapy.
              </a>
            </td>
          <td>
            Keiichi Fujiya, M. Serizawa, K. Ohshima, R. Umehara, Yuko Watanabe, T. Nagashima, E. Bando, K. Urakami, Y. Akiyama, Yasuhiro Tsubosa, Takashi Sugino, Ken Yamaguchi, Masanori Terashima
          </td>
          <td>2024-11-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea66538fc66995f8a97d32c51497b62496c4728f" target='_blank'>
              Single-cell dissection reveals promotive role of ENO1 in leukemia stem cell self-renewal and chemoresistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yun Tian, Jiafan Guo, Lipeng Mao, Zhixi Chen, Xingwei Zhang, Yangqiu Li, Yikai Zhang, Xianfeng Zha, Oscar Junhong Luo
          </td>
          <td>2024-10-08</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chemical screens across hundreds of cell lines have shown that the drug sensitivities of human cancers can vary by genotype or lineage. However, most drug discovery studies have relied on culture media that poorly reflect metabolite levels in human blood. Here, we perform drug screens in traditional and Human Plasma–Like Medium (HPLM). Sets of compounds that show conditional anticancer activity span different phases of global development and include non-oncology drugs. Comparisons of the synthetic and serum-derived components that comprise typical media trace sets of conditional phenotypes to nucleotide synthesis substrates. We also characterize a unique dual mechanism for brivudine, a compound approved for antiviral use. Brivudine selectively impairs cell growth in low folate conditions by targeting two enzymes involved in one-carbon metabolism. Cataloged gene essentiality data further suggest that conditional phenotypes for other compounds are linked to off-target effects. Our findings establish general strategies for identifying drug-nutrient interactions and mechanisms of action by exploiting conditional lethality in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52075b84c4effc26f41a203f90514fe9f3c95c9f" target='_blank'>
              Conditional lethality profiling reveals anticancer mechanisms of action and drug-nutrient interactions
              </a>
            </td>
          <td>
            K. Flickinger, Kelli M. Wilson, Nicholas J. Rossiter, Andrea L. Hunger, Paresh V Vishwasrao, Tobie D. Lee, Carlos A. Mellado Fritz, Rebecca M Richards, Matthew D. Hall, Jason R. Cantor
          </td>
          <td>2024-10-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Background: Hispanic American (HA) men with prostate cancer (PC) are more likely to have more aggressive and advanced diseases and greater mortality rates compared to non-Hispanic white (NHW) men. Resistance to the current anticancer therapy is the major hurdle that leads to poor clinical outcomes. Hedgehog, Notch, and Wnt signaling pathways play a crucial role in development, progression, and resistance to the various anticancer therapies. Here, we characterized the enrichment of Hedgehog, Notch, and Wnt signaling pathways and molecular tumors differences between HA and NHW patients. Methods: Our data consist of 88 PC tumors samples (HA=34, NHW=54) obtained from treatment-naive metastatic hormone-sensitive prostate cancer (mHSPC) that were analyzed by next-generation sequencing (592, NextSeq; Whole Exome Sequencing, NovaSeq) and Whole Transcriptome Sequencing (WTS; NovaSeq) (Caris Life Sciences, Phoenix, AZ). Pathway enrichment was determined by Gene Set Enrichment Analysis (GSEA, Broad Institute) by measurable TPM (transcripts per kilobase million). The medical records were reviewed in a deidentified fashion for clinical features. Statistical significance was determined using chi-square and Mann-Whitney U (p<0.05). Results: HA PC had enrichment of Hedgehog signaling (NES: 1.45, FDR=0.04), Notch signaling (NES: 1.41, FDR=0.07), and Wnt/β-catenin signaling (NES: 1.38, FDR=0.11) pathways compared to NHW tumors. HA tumors had higher expression of genes (HES1, Notch1, WNT1, WNT2; FC:1.1-2.5) associated with crosstalk between Hedgehog, Notch, and Wnt/β- catenin signaling pathways (all p<0.05). Moreover, HA tumors had higher expression of stemness (ALDH1A2, ALDH1A1, PROM1; FC: 1.7-3.0), drug efflux (ABCC1, ABCB1; FC: 1.1- 1.5), epithelial to mesenchymal transition (EMT) (TWIST1, MMP2; FC: 1.1-1.5), cell cycle (CCND2; FC: 1.2), inhibition of apoptosis (BIRC2, XIAP; FC: 1.1) and epigenetic (EP300, TET1, TET2; FC: 1.3-1.4) related genes compared to NHW tumors (all p<0.05). HA PC had higher frequency of Hedgehog and Wnt/β-catenin associated SPOP (16% vs 6.6%, p=0.2) and CTNNB1 (12% vs 2.2%, p=0.1) mutation compared to NHW PC. Conclusion: These results demonstrate that tumors from HA patients had enrichment of oncogenic Hedgehog, Notch, and Wnt/β-catenin signaling pathways, higher expression of stemness, drug efflux, EMT, cell cycle, inhibition of apoptosis and epigenetic-related genes, and mutations directly involved in activating the Hedgehog and Wnt/β-catenin signaling pathways. Together, these findings suggest that tumors from HA patients are enriched for gene signatures that promote cancer progression and therapy resistance. Additional research on collaboration and crosstalk between Hedgehog, Notch, and Wnt/β-catenin signaling pathways may help in designing better therapeutic strategies for HA men with mHSPC.
 Citation Format: Alejandro Recio-Boiles, Sachin Kumar Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, Milan Radovich, Elisabeth I. Heath, Rana McKay, Juan Chipollini, Cynthia K. Miranti. Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C100.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084e712d5ece7637137ae910e70858f7d21562a6" target='_blank'>
              Abstract C100: Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men
              </a>
            </td>
          <td>
            Alejandro Recio-Boiles, S. Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, M. Radovich, Elisabeth I. Heath, Rana Mckay, Juan Chipollini, Cynthia K. Miranti
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. J. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T. Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Mutations in genes involved in DNA damage repair (DDR) often lead to premature aging syndromes. While recent evidence suggests that inflammation, alongside mutation accumulation and cell death, may drive disease phenotypes, its precise contribution to in vivo pathophysiology remains unclear. Here, by modeling Ataxia Telangiectasia (A-T) and Bloom Syndrome in the African turquoise killifish (N. furzeri), we replicate key phenotypes of DDR syndromes, including infertility, cytoplasmic DNA fragments, and reduced lifespan. The link between DDR defects and inflammation is attributed to the activation of the cGAS-STING pathway and interferon signaling by cytoplasmic DNA. Accordingly, mutating cGAS partially rescues germline defects and senescence in A-T fish. Double mutants also display reversal of telomere abnormalities and suppression of transposable elements, underscoring cGAS’s non-canonical role as a DDR inhibitor. Our findings emphasize the role of interferon signaling in A-T pathology and identify the cGAS-STING pathway as a potential therapeutic target for genomic instability syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0762a0a7a477e0941f31815c1aa8f1b5b79a238" target='_blank'>
              The cGAS-STING pathway is an in vivo modifier of genomic instability syndromes
              </a>
            </td>
          <td>
            Marva Bergman, Uri Goshtchevsky, Tehila Atlan, Gwendoline Astre, Ryan Halabi, Hosniyah El, Eitan Moses, A. J. Lemus, BA Benayoun, Y. Tzfati, Ido Ben-Ami, I. Harel
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5edea2d9a83527bbebc5d48f3ba44b53a9b424d3" target='_blank'>
              Single-cell profiling uncovers proliferative cells as key determinants of survival outcomes in lower-grade glioma patients
              </a>
            </td>
          <td>
            Jian-Bing Peng, Qing Zhang, Xiaofeng Zhu, Zhu Yan, Meng Zhu
          </td>
          <td>2024-09-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios, Lei Peng, Luojia Yang
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea678de24f2edf0463772717c15047d50861f390" target='_blank'>
              Characterization of mitochondrial DNA mutations in colorectal cancer progression by in silico approach and use as potential biomarkers for diagnosis and prognosis
              </a>
            </td>
          <td>
            Ramya Gadicherla, Niraj Rai, Rajath Othayoth, Srinivasulu Kamma
          </td>
          <td>2024-10-28</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T-cells and bispecific antibodies (BsAb) targeting B-cell maturation antigen (BCMA) have significantly advanced the treatment of relapsed and refractory multiple myeloma (MM). Resistance to BCMA-targeting therapies, nonetheless, remains a significant challenge. BCMA shedding by gamma-secretase is a known resistance mechanism, and preclinical studies suggest that inhibition may improve anti-BCMA therapy. Leveraging a phase I clinical trial of the gamma-secretase inhibitor (GSI), crenigacestat, with anti-BCMA CAR T-cells (FCARH143), we utilized single-nuclei RNA sequencing and Assay for Transposase-Accessible Chromatin (ATAC) sequencing to characterize the effects of GSI on the tumor microenvironment. The most significant impacts of GSI involved effects on monocytes, which are known to promote tumor growth. In addition to observing a reduction in the frequency of non-classical monocytes, we also detected significant changes in gene expression, chromatin accessibility, and inferred cell-cell interactions following exposure to GSI. Although many genes with altered expression are associated with gamma-secretase-dependent signaling, such as Notch, other pathways were affected, indicating GSI has far-reaching effects. Finally, we detected monoallelic deletion of the BCMA locus in some patients with prior exposure to anti-BCMA therapy, which significantly correlated with reduced progression-free survival (median PFS 57 days versus 861 days). GSIs are being explored in combination with the full spectrum of BCMA targeting agents, and our results reveal widespread effects of GSI on both tumor and immune cell populations, providing insight into mechanisms for enhancing BCMA-directed therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0270c45ce4ccff6ee8256dcb44e77e0a31404f58" target='_blank'>
              Single-cell analysis of the multiple myeloma microenvironment after gamma-secretase inhibition and CAR T-cell therapy.
              </a>
            </td>
          <td>
            D. Coffey, Pinar Ataca Atilla, Erden Atilla, O. Landgren, Andrew J Cowan, Sylvain Simon, Margot Pont, Melissa L. Comstock, Geoffrey R. Hill, Stanley R. Riddell, Damian J Green
          </td>
          <td>2024-10-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Comprehensive genomic profiling (CGP) identifies genetic alterations and patterns that are crucial for therapy selection and precise treatment development. In Colombia, limited access to CGP tests underscores the necessity of documenting the prevalence of treatable genetic alterations. This study aimed to describe the somatic genetic profile of specific cancer types in Colombian patients and assess its impact on treatment selection. Methods: A retrospective cohort study was conducted at Clínica Colsanitas S.A. from March 2023 to June 2024. Sequencing was performed on the NextSeq2000 platform with the TruSight Oncology 500 (TSO500) assay, which simultaneously evaluates 523 genes for DNA analysis and 55 for RNA; additionally, analyses were performed with the SOPHiA DDM software. The tumor mutational burden (TMB), microsatellite instability (MSI), and programmed cell death ligand 1 (PDL1) were assessed. Results: Among 111 patients, 103 were evaluated, with gastrointestinal (27.93%), respiratory (13.51%), and central nervous system cancers (10.81%) being the most prevalent. TP53 (37%), KMT2C (28%), and KRAS (21%) were frequent mutations. Actionable findings were detected in 76.7% of cases, notably in digestive (20 patients) and lung cancers (8 patients). MSI was stable at 82.52% and high at 2.91%, whilst TMB was predominantly low (91.26%). Conclusions: The test has facilitated access to targeted therapies, improving clinical outcomes in Colombian patients. This profiling test is expected to increase opportunities for personalized medicine in Colombia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93aa6bc1a8a2f18dd281151deeb80b4be0a23d78" target='_blank'>
              Advancing Cancer Care in Colombia: Results of the First In Situ Implementation of Comprehensive Genomic Profiling
              </a>
            </td>
          <td>
            Juan Javier López Rivera, Paula Rueda-Gaitán, Laura Camila Rios Pinto, Diego Alejandro Rodríguez Gutiérrez, Natalia Gomez-Lopera, J. Lamilla, Fabio Andrés Rojas Aguirre, Laura Bernal Vaca, M. Isaza-Ruget
          </td>
          <td>2024-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The emergence of novel cell types fuels evolutionary innovations and contributes to the diversity of life forms and their morphological and functional traits. Cell types are fundamental functional units of multicellular organisms defined by their specific gene expression programs. The evolution of these transcriptional programs is driven by genetic changes, such as gene co-option and cis- regulatory evolution, known to facilitate the assembly or rewiring of molecular networks and give rise to new cell types with specialized functions. However, the role of novel genes in this complex evolutionary process is underexplored. Here, we examine the trends in skeletal cell type evolution with a focus on lineage-specific genes. We find that immature chondrocytes express the oldest transcriptome and resemble ancestral skeletogenic cell type, supporting the existence of a conserved genetic program for cartilage development in bilaterians. The subsequent acquisition of lineage-restricted genes led to the individuation of the ancient gene expression program and powered the emergence of osteoblasts and hypertrophic chondrocytes. We found a significant enrichment of Vertebrate-specific genes in osteoblasts and Gnathostome-specific genes in hypertrophic chondrocytes. By identifying the functional properties of the recruited genes, coupled with the recently discovered fossil evidence, our findings challenge the long-standing view on the evolution of vertebrate skeletal structures and suggest that endochondral ossification and chondrocyte hypertrophy evolved already in the last common ancestors of gnathostomes. Finally, our findings highlight the critical role of novel genes in shaping cellular diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677ea0e1eca83b3b7673677deb5e3a705075d5bc" target='_blank'>
              Evolutionary trends in the emergence of skeletal cell types
              </a>
            </td>
          <td>
            Amor M. Damatac, Sara Koska, Kristian K Ullrich, Tomislav Domazet-Lošo, Alexander Klimovich, M. Kaucká
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Therapeutic targeting of dysregulated transcription has emerged as a promising strategy for the treatment of cancers, such as leukaemias. The therapeutic response to small molecule inhibitors of Bromodomain-Containing Proteins (BRD), such as BRD2 and BRD4, P300/cAMP-response element binding protein (CBP) and Cyclin Dependent Kinases (CDKs), is generally attributed to the selective disruption of oncogenic gene expression driven by enhancers, super-enhancers (SEs) and lineage-specific transcription factors (TFs), including the c-MYC oncogene. The selectivity of compounds targeting the transcriptional machinery may be further shaped by post-transcriptional processes. To quantitatively assess the contribution of post-transcriptional regulation in responses to transcription inhibition, we performed multi-omics analyses to accurately measure mRNA production and decay kinetics. We demonstrate that it is not only the selective disruption of mRNA production, but rather mRNA decay rates that largely influence the selectivity associated with transcriptional inhibition. Accordingly, genes down-regulated with transcriptional inhibitors are largely characterized by extremely rapid mRNA production and turnover. In line with this notion, stabilization of the c-MYC transcript through swapping of its 3′ untranslated region (UTR) rendered c-MYC insensitive to transcriptional targeting. This failed to negate the impact on c-MYC downstream targets and did not abrogate therapeutic responses. Finally, we provide evidence that modulating post-transcriptional pathways, such as through ELAVL1 targeting, can sensitize long-lived mRNAs to transcriptional inhibition and be considered as a combination therapy approach in leukaemia. Taken together, these data demonstrate that mRNA kinetics influence the therapeutic response to transcriptional perturbation and can be modulated for novel therapeutic outcomes using transcriptional agents in leukaemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec2d129b2fe7b06dd3e5f4a0e3e5eecfda7adb45" target='_blank'>
              RNA kinetics influence the response to transcriptional perturbation in leukaemia cell lines
              </a>
            </td>
          <td>
            Izabela Todorovski, Mary-Jane Tsang, B. Feran, Zheng Fan, Sreeja Gadipally, David Yoannidis, Isabella Y. Kong, Stefan Bjelosevic, S. Rivera, Olivia Voulgaris, Magnus Zethoven, Edwin D. Hawkins, K. Simpson, Gisela Mir Arnau, A. Papenfuss, Ricky W Johnstone, S. Vervoort
          </td>
          <td>2024-10-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Fibroblast Growth Factor Receptor 2 inhibition presents a promising therapeutic approach for restraining the growth and survival of cancer cells, particularly in breast tumours. However, the emergence of resistance to FGFR2 inhibitors like PD173074 highlights the importance of understanding the molecular mechanisms driving resistance and identifying effective therapeutic strategies. In this study, we employed temporal quantitative proteomics and phosphoproteomics, complemented by computational clustering, PTM-SEA analysis, and kinase activity prediction, to monitor the response of MFM223 triple negative breast cancer cells to FGFR2 inhibition. Strikingly, we observed a marked enrichment of ribosome biogenesis function modules immediately following treatment, a phenomenon not observed with other inhibitors. Additionally, we discovered that CX-5461, an RNA polymerase I inhibitor, synergistically enhanced the growth inhibition induced by PD173074, mechanistically attributed to its significant suppression of rDNA transcription stimulated by PD173074. Moreover, our phosphoproteomics dynamic profiling identified the clustering of kinases within MAPK and ErbB signalling pathways, indicative of their reactivation in response to FGFR2 inhibition. Experimentally validating this finding, we observed a notable rebound in phosphorylation levels of key kinases such as ERK1/2 and ErbB3, and demonstrated a substantial synergistic effect of PD173074 in combination with Trametinib, a MEK inhibitor, in suppressing cancer cell growth. Collectively, our findings provide critical insights into the network rewiring triggered by FGFR2 inhibition and offer a foundation for the rational design of combinatorial therapeutic strategies to overcome resistance mechanisms associated with FGFR2 inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc089ba7e1dffd9ff2361b087cdde2575bba907" target='_blank'>
              Ribosomal Biogenesis Hyperactivation and ErbB signalling Mediated Network Rewiring Causes Adaptive Resistance to FGFR2 Inhibition
              </a>
            </td>
          <td>
            Tao Zang, Sung-young Shin, Callan McCrimmon, Mandy Theocharous, Ralf B. Schittenhelm, Thierry Jarde, Roger J. Daly, Lan K. Nguyen
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Phenotypic diversity in tumors makes treatment difficult and leads to poor prognosis. A better understanding of what causes intratumoral heterogeneity and differences in cells that are within the same tumor will advance therapeutic options and improve patient outcomes. This review will summarize recent studies that have examined the various cancer cell phenotypes along the epithelial–mesenchymal spectrum present in tumors and how epigenetic factors and the mechanical properties of tumor tissue mediate these phenotypes. This review will also discuss the therapeutic implications of intratumoral heterogeneity, such as drug resistance and tumor recurrence, and highlight opportunities to address current limitations in cancer treatments. Abstract The tumor microenvironment comprises various cell types and experiences dynamic alterations in physical and mechanical properties as cancer progresses. Intratumoral heterogeneity is associated with poor prognosis and poses therapeutic challenges, and recent studies have begun to identify the cellular mechanisms that contribute to phenotypic diversity within tumors. This review will describe epithelial–mesenchymal (E/M) plasticity and its contribution to phenotypic heterogeneity in tumors as well as how epigenetic factors, such as histone modifications, histone modifying enzymes, DNA methylation, and chromatin remodeling, regulate and maintain E/M phenotypes. This review will also report how mechanical properties vary across tumors and regulate epigenetic modifications and E/M plasticity. Finally, it highlights how intratumoral heterogeneity impacts therapeutic efficacy and provides potential therapeutic targets to improve cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b710fff2d564553c94ffb93440614a32c0f2e22e" target='_blank'>
              Epithelial–Mesenchymal Plasticity and Epigenetic Heterogeneity in Cancer
              </a>
            </td>
          <td>
            Jessica L Sacco, Esther W. Gomez
          </td>
          <td>2024-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5aeebfc014f88f36ffbdee87dcef5097f55ec3d" target='_blank'>
              Development of a novel treatment based on PKMYT1 inhibition for cisplatin-resistant bladder cancer with miR-424-5p-dependent cyclin E1 amplification
              </a>
            </td>
          <td>
            Wataru Fukumoto, S. Okamura, M. Tamai, Junya Arima, Ichiro Kawahara, Ikumi Fukuda, Akihiko Mitsuke, Takashi Sakaguchi, S. Sugita, R. Matsushita, S. Tatarano, Yasutoshi Yamada, M. Nakagawa, H. Enokida, H. Yoshino
          </td>
          <td>2024-10-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to drive divergent regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under “cellular constraint”. We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and non-coding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share paradoxical evolutionary signals of positive and negative selection, suggesting a dynamic role in driving human adaptation. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0bc98afde8c329e9647a592102463c5d074b6c9" target='_blank'>
              Epigenetic constraint of cellular genomes evolutionarily links genetic variation to function
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, Fei-Fei Cheng, Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Biological processes are fundamentally dynamic, yet existing methods for capturing these temporal changes are limited. We present scDynaBar, a novel approach that integrates CRISPR-Cas9 dynamic barcoding with single-cell sequencing to enable the recording of temporal cellular events. In this system, genetic barcodes accumulate mutations over a 4-week timeframe and then are sequenced together with the transcriptome of each single cell. We propose that this gradual accumulation of genetic diversity can be exploited to create a time-ordered record of cellular events. To demonstrate this, we apply the system to track the transition from a pluripotent state to a two-cell (2C)–like state in mouse embryonic stem cells (mESCs). The results provide compelling evidence for the transient nature of the 2C-like state. Additionally, our system shows consistent mutation rates across diverse cell types in a mouse gastruloid model, underscoring its robustness and versatility across various biological contexts. This technique not only improves our ability to study cellular dynamics but also creates exciting new opportunities for future applications based on recording temporal signals at the single-cell level—in other words, using dynamic barcoding as a molecular clock.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0a4bc0e689a5dd87c70b2f28e78dbbf5756625" target='_blank'>
              Using CRISPR barcoding as a molecular clock to capture dynamic processes at single-cell resolution
              </a>
            </td>
          <td>
            Yolanda Andres-Lopez, Alice Santambrogio, Ioannis Kafetzopoulos, Christopher D Todd, Celia Alda-Catalinas, Stephen J. Clark, Wolf Reik, I. Hernando-Herraez
          </td>
          <td>2024-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8e33c5b973a4103516f253546861d556660e98" target='_blank'>
              Integrated immunogenomic analyses of single-cell and bulk profiling construct a T cell-related signature for predicting prognosis and treatment response in osteosarcoma
              </a>
            </td>
          <td>
            Chicheng Niu, Weiwei Wang, Qingyuan Xu, Zhao Tian, Hao Li, Qiang Ding, Liang Guo, Ping Zeng
          </td>
          <td>2024-10-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung cancer, categorized into non-small cell lung cancer (NSCLC) and small cell lung cancer (SCLC), remains a significant global health challenge. The development of drug resistance and the heterogeneity of the disease necessitate the identification of novel therapeutic targets to improve patient outcomes.We conducted a genome-wide Mendelian randomization (MR) and colocalization analysis using a comprehensive dataset of 4,302 druggable genes and cis-expressed quantitative trait loci (cis-eQTLs) from 31,884 blood samples. The study integrated genomic analysis with eQTL data to identify key genes associated with lung cancer risk.The analysis revealed five actionable therapeutic targets for NSCLC, including LTB4R, LTBP4, MPI, PSMA4, and TCN2. Notably, PSMA4 demonstrated a strong association with both NSCLC and SCLC risks, with odds ratios of 3.168 and 3.183, respectively. Colocalization analysis indicated a shared genetic etiology between these gene expressions and lung cancer risk.Our findings contribute to precision medicine by identifying druggable targets that may be exploited for subtype-specific lung cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b3c04313914f9908a3cc17acdf2a462d0a4f43" target='_blank'>
              Therapeutic targets for lung cancer: genome-wide Mendelian randomization and colocalization analyses
              </a>
            </td>
          <td>
            Yi Luan, Desheng Xian, Changwen Zhao, Xin Qing, Hanlin He, Kaixuan Zheng, Wenjun Song, Taijiao Jiang, Wenjian Wang, Chaohui Duan
          </td>
          <td>2024-10-28</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, A. Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Lynch Syndrome, a hereditary disorder characterized by germline mutations in mismatch repair (MMR) genes, is a major contributor to colorectal cancers. It has also been identified in endometrial cancer. Despite the established role of MMR deficiency in tumorigenesis, the specific genomic alterations driving Lynch syndrome-associated endometrial cancer, and their overlap with colorectal cancer, remain incompletely understood. This study aims to fill this gap by performing a detailed comparative analysis of germline and somatic mutations in endometrial cancer within the context of Lynch syndrome.


METHODS
We conducted whole exome sequencing on matched germline and somatic DNA from 13 patients diagnosed with Lynch syndrome-associated endometrial cancer. High-depth sequencing was performed, followed by rigorous bioinformatics analysis to identify and annotate variants, focusing on their potential pathogenicity and relevance to both endometrial and colorectal cancer.


RESULTS
Our analysis revealed 1,118 germline and 14,051 somatic variants, with 493 variants common to both. Recurrent pathogenic mutations in MLH1, MSH2, and MSH6 were confirmed, highlighting their critical role in Lynch syndrome. Notably, frequent somatic mutations in the PIK3CA and PTEN genes were identified, implicating the PI3K/AKT/mTOR pathway as a key oncogenic driver in these cancers. Additionally, novel somatic mutations in genes related to the extracellular matrix such as FBN1 and SPARC were uncovered, suggesting a possible unique role in endometrial tumor progression.


CONCLUSIONS
This study provides new insights into the molecular basis of Lynch syndrome-associated endometrial cancer, emphasizing the overlap in oncogenic pathways with colorectal cancer. The discovery of shared and unique genetic mutations highlights the importance of developing combined treatment strategies and suggests that targeting these specific mutations could improve therapy for patients with Lynch syndrome-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b204016ea6a27e3ec168c78e8d560614b4b705" target='_blank'>
              Lynch Syndrome-associated Genomic Variants.
              </a>
            </td>
          <td>
            Robert Botea, Madalina Piron-Dumitrascu, Tiberiu-Augustin Georgescu, C. Bohiltea, S. Voinea, Valentin Nicolae Varlas, Simona Raluca Iacoban, Nicolae Suciu
          </td>
          <td>2024-09-09</td>
          <td>Journal of gastrointestinal and liver diseases : JGLD</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd7b87d57c150bf5b9368e6def7dc5609b418052" target='_blank'>
              siRNA-based strategies to combat drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Abdulrahman Qais Khaleel, Mohammad Y Alshahrani, J. Rizaev, H. Malathi, Seema Devi, Atreyi Pramanik, Yasser Fakri Mustafa, Ahmed Hjazi, Ismoilova Muazzamxon, Beneen Husseen
          </td>
          <td>2024-10-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed8af38ff728f1fd91a0f56c0da64fa1f39b833" target='_blank'>
              Clinical relevance of protein-truncating variants of germline DNA repair genes in prostate cancer
              </a>
            </td>
          <td>
            , Cai-Sian Liao, Yu-Ching Hsu, Yu-Chiao Chiu, Tsai-Jung Lu, Yen-Chuan Ou, Tzu-Hung Hsiao
          </td>
          <td>2024-10-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ed2057a00d451d2750eb4c6690ca3020a9e3a6" target='_blank'>
              The prominent pervasive oncogenic role and tissue specific permissiveness of RAS gene mutations
              </a>
            </td>
          <td>
            Ming Yi, Daniel Soppet, Frank McCormick, D. Nissley
          </td>
          <td>2024-10-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Vitamin C plays a significant role in various physiological functions. Humans depend on external sources of vitamin C due to the loss of the L-gulono-γ-lactone oxidase (GULO) gene that contributes to the synthesis of vitamin C. During the evolutionary loss of the GULO gene, physical, chemical, and biological factors were different from the present environmental settings. Besides the evolutionary genetic loss of the GULO gene, there is a gap in the insightful discussion on the potential implications of the non-functional GULO gene towards the predisposition of humans to cancer that faces hostile and carcinogenic environments. Various methods by which vitamin C modulates cellular processes related to cancer, including DNA repair, epigenetic changes, and redox balance, are discussed. Furthermore, we present experimental and clinical evidence indicating that vitamin C deficiency promotes tumor growth, metastasis, and therapy resistance, emphasizing its potential as a cancer phenotypic modulator. Therapeutic implications of restoring vitamin C levels in cancer treatment range from improving the efficacy of conventional medicines to exploiting metabolic vulnerabilities in tumors. The relevance of assessing vitamin C status in cancer patients and the basis for additional research into vitamin C supplementation as an adjuvant therapy is emphasized. This paper presents a comprehensive overview of the implications associated with the functional deficiency of the GULO gene in human subjects exhibiting diverse tumor hallmarks, encompassing ECM remodeling, hypoxia, epigenetic reprogramming, oxidative stress, and drug responsiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b5b27fd6a0a394480679fcc72328139bc69dd8" target='_blank'>
              Potentiation of Tumor Hallmarks by the Loss of GULO, a Vitamin C Biosynthesis Gene in Humans.
              </a>
            </td>
          <td>
            Gopinath Sekar, Anjali Bahot, Mahima Bansode, Anushka Phadnis, S. Sarode, Nilesh Kumar Sharma
          </td>
          <td>2024-10-14</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c962e49dc98cf761dec124db24055472f555a9" target='_blank'>
              Mapping multimodal phenotypes to perturbations in cells and tissue with CRISPRmap.
              </a>
            </td>
          <td>
            Jiacheng Gu, Abhishek Iyer, Ben Wesley, A. Taglialatela, G. Leuzzi, Sho Hangai, Aubrianna Decker, Ruoyu Gu, Naomi Klickstein, Yuanlong Shuai, Kristina Jankovic, Lucy Parker-Burns, Yinuo Jin, Jia Yi Zhang, Justin Hong, Xiang Niu, Jonathon A. Costa, Mikael Pezet, Jacqueline Chou, H. Snoeck, Dan A Landau, Elham Azizi, Edmond M. Chan, Alberto Ciccia, Jellert T. Gaublomme
          </td>
          <td>2024-10-07</td>
          <td>Nature biotechnology</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="The discovery of growth factors and their involvement in cancer represents the foundation of precision oncology. The preclinical and clinical development of agents targeting epidermal growth factor receptor (EGFR) in colorectal cancer (CRC) were accompanied by big hype and hopes, though the clinical testing of such agents clashed with intrinsic and acquired resistance, greatly limiting their therapeutic value. However, a better understanding of the biology of the EGFR signaling pathway in CRC, coupled with the development of liquid biopsy methodologies to study cancer evolution in real time, fostered the clinical refinement of anti-EGFR treatment in CRC. Such a workflow, based on the co-evolution of biology knowledge and clinical development, allowed to couple the discovery of relevant therapy resistance mechanisms to the development of strategies to bypass this resistance. A broader application of this paradigm could prove successful and create an effective shortcut between the bench and the bedside for treatment strategies other than targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a608cc6a9618f352b70a4c24f44bb5a5875ce0fc" target='_blank'>
              When molecular biology transforms clinical oncology: the EGFR journey in colorectal cancer.
              </a>
            </td>
          <td>
            P. P. Vitiello, N. Saoudi González, A. Bardelli
          </td>
          <td>2024-10-29</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4420591a28c5d2d5cd263ca1cea25300248881f" target='_blank'>
              Five latent factors underlie response to immunotherapy
              </a>
            </td>
          <td>
            Joseph Usset, Axel Rosendahl Huber, Maria Andrianova, Eduard Batlle, Joan Carles, Edwin Cuppen, E. Élez, E. Felip, Marina Gómez-Rey, Deborah Lo Giacco, Francisco Martínez-Jiménez, E. Muñoz-Couselo, L. Siu, J. Tabernero, A. Vivancos, Ferran Muiños, A. González-Pérez, Núria López-Bigas
          </td>
          <td>2024-09-12</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd91146f26c8e3d190bc5c9a508247726a0ba6a" target='_blank'>
              Identifying genetic variants associated with chromatin looping and genome function
              </a>
            </td>
          <td>
            Sourya Bhattacharyya, F. Ay
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma (DLBCL) is an aggressive non-Hodgkin lymphoma and the most frequently diagnosed hematologic malignancy in the United States. DLBCL exhibits significant molecular and clinical heterogeneity, and at least a third of patients are left uncured with standard frontline chemoimmunotherapy. As such, there is a critical need to identify novel targeted therapies to improve outcomes. We conducted a phenotypic screen of kinase inhibitors against DLBCL cell lines and non-malignant controls. We identified the cyclin G-associated kinase (GAK) as a tumor-selective, readily druggable target whose inhibition killed DLBCL cell lines, while sparing non-malignant blood cells. Upon investigation of GAK’s cellular function, we discovered that inhibition results in G2/M-phase cell cycle arrest. Immunofluorescent confocal microscopy revealed significant chromosome misalignment and spindle distortion in DLBCL cells following GAK-inhibition, disrupting progression through mitosis. Analysis of RNA-seq data from clinical samples showed increased GAK expression associates strongly with RB1 deficiency in DLBCL cases, suggesting dependency on GAK for proper mitotic progression linked to retinoblastoma associated protein (RB) loss of function, a common DLBCL driver. In cell-cycle analyses and under microscopy, RB-deficient DLBCL cells treated with an exquisitely selective GAK inhibitor showed complete arrest at G2/M, pronounced distortion of mitotic spindles, and widespread chromosomal damage. Finally, in vivo studies of DLBCL xenograft-bearing NSG mice achieved a dramatic tumor-burden reduction in response to targeted GAK inhibition. These results reveal a novel cell cycle kinase suitable for therapeutic exploitation in DLBCL patients and linked to the common, undruggable biomarker of RB loss of function. KEY POINTS Phenotypic screening and target deconvolution revealed GAK as a novel therapeutic target in diffuse large B-cell lymphoma GAK kinase activity is a key requirement for successful mitosis in DLBCL linked to deficiency of the retinoblastoma (RB) tumor supressor">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dbd2dded9e4b2613776188deface431c8db78e0" target='_blank'>
              The cyclin-G associated kinase (GAK) is a novel mitotic kinase and therapeutic target in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            O. Lightfuss, Preet Kumar, Lingxiao Li, Toral Shastri, Austin D. Newsam, Paola Manara, A. Y. Alaoui, Diego E. Hernandez Parets, Pierre A. Younes, Dhanvantri Chahar, Artavazd Arumov, C. Coughlin, Indigo P. Williams, Nikolai Fattakhov, Alicia Bilbao Martinez, O. Landgren, Stephan C. Schürer, Vance Lemmon, Francesco Maura, Daniel Bilbao, Yangbo Feng, Hassan Al-Ali, Jonathan H. Schatz
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The repair of DNA double-strand breaks (DSBs) through homologous recombination (HR) is vital for maintaining the stability and integrity of the genome. RNA binding proteins (RBPs) intricately regulate the DNA damage repair process, yet the precise molecular mechanisms underlying their function remain incompletely understood. In this study, we highlight the pivotal role of RPS15, a representative RBP, in homologous recombination repair. Specifically, we demonstrate that RPS15 promotes DNA end resection, a crucial step in homologous recombination. Notably, we identify an interaction between RPS15 and CtIP, a key factor in homologous recombination repair. This interaction is essential for CtIP recruitment to DSB sites, subsequent RPA coating, and RAD51 replacement, all critical steps in efficient homologous recombination repair and conferring resistance to genotoxic treatments. Functionally, suppressing RPS15 expression sensitizes cancer cells to X-ray radiation and enhances the therapeutic synergistic effect of PARP1 inhibitors in breast cancer cells. In summary, our findings reveal that RPS15 promotes DNA end resection to ensure effective homologous recombination repair, suggesting its potential as a therapeutic target in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2aedc66ec12ddcad5319b720ce461d7c9233af5" target='_blank'>
              RPS15 Coordinates with CtIP to Facilitate Homologous Recombination and Confer Therapeutic Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Baohang Lin, Guan Huang, Zishan Yuan, Xun Peng, Chunliang Yu, Jialu Zheng, Zequn Li, Juanyun Li, Jinan Liang, Bo Xu
          </td>
          <td>2024-10-03</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells are heterogeneous, each harboring distinct molecular aberrations and are dependent on different genes for their survival and proliferation. While successful targeted therapies have been developed based on driver DNA mutations, many patient tumors lack druggable mutations and have limited treatment options. Here, we hypothesize that new precision oncology targets may be identified through “expression-driven dependency”, whereby cancer cells with high expression of a targeted gene are more vulnerable to the knockout of that gene. We introduce a Bayesian approach, BEACON, to identify such targets by jointly analyzing global transcriptomic and proteomic profiles with genetic dependency data of cancer cell lines across 17 tissue lineages. BEACON identifies known druggable genes, e.g., BCL2, ERBB2, EGFR, ESR1, MYC, while revealing new targets confirmed by both mRNA- and protein-expression driven dependency. Notably, the identified genes show an overall 3.8-fold enrichment for approved drug targets and enrich for druggable oncology targets by 7 to 10-fold. We experimentally validate that the depletion of GRHL2, TP63, and PAX5 effectively reduce tumor cell growth and survival in their dependent cells. Overall, we present the catalog of express-driven dependency targets as a resource for identifying novel therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328c4c5a157ff3ba7e44391b07b799fdfa4b7395" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Medicine
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D. Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0914eac9368db3a26273ef88f18ab51249d37f" target='_blank'>
              Advances in targeting cancer epigenetics using CRISPR-dCas9 technology: A comprehensive review and future prospects.
              </a>
            </td>
          <td>
            Jeevitha Rajanathadurai, Elumalai Perumal, Jospin Sindya
          </td>
          <td>2024-09-18</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nearly a quarter century ago, Hanahan and Weinberg conceived six unifying principles explaining how normal cells transform into malignant tumors. Their provisional set of biological capabilities acquired during tumor development  cancer hallmarks  would evolve to fourteen tenets as knowledge of cancer genomes, molecular mechanisms, and the tumor microenvironment expanded, most recently adding four emerging enabling characteristics: phenotypic plasticity, epigenetic reprogramming, polymorphic microbiomes, and senescent cells. AKT kinases are critical signaling molecules that regulate cellular physiology upon receptor tyrosine kinases and phosphatidylinositol 3-kinase activation. The complex branching of the AKT signaling network involves several critical downstream nodes that significantly magnify its functional impact, such that nearly every organ system and cell in the body may be affected by AKT activity. Conversely, tumor intrinsic dysregulation of AKT can have numerous adverse cellular and pathological ramifications, particularly in oncogenesis, as multiple tumor suppressors and oncogenic proteins regulate AKT signaling. Herein, we review the mounting evidence implicating the AKT pathway in the aggregate of currently recognized hallmarks of cancer underlying the complexities of human malignant diseases. The challenges, recent successes, and likely areas for exciting future advances in targeting this complex pathway are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc996efb642ec81ce6e4555cd7fa9142e0c17d3" target='_blank'>
              AKT and the Hallmarks of Cancer.
              </a>
            </td>
          <td>
            Eleonora Sementino, Dalal Hassan, Alfonso Bellacosa, Joseph R Testa
          </td>
          <td>2024-10-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c899c38678aa8e86ec7ec4d5de561bdad4078c7d" target='_blank'>
              CDK4/6 inhibition initiates cell cycle arrest by nuclear translocation of RB and induces a multistep molecular response
              </a>
            </td>
          <td>
            Ting Hong, Anna C Hogger, Dongbiao Wang, Qi Pan, Julie Gansel, T. Engleitner, R. Öllinger, Jürgen E Gschwend, R. Rad, R. Nawroth
          </td>
          <td>2024-10-26</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8944ad87f93a33edd3ce6c09cbae53782b87ab0f" target='_blank'>
              Single-cell transcriptomics of pediatric Burkitt lymphoma reveals intra-tumor heterogeneity and markers of therapy resistance.
              </a>
            </td>
          <td>
            Clarissa Corinaldesi, A. Holmes, Gaia Martire, A. Tosato, Domenico Rizzato, F. Lovisa, I. Gallingani, Qiong Shen, Lavinia Ferrone, Marian Harris, Kimberly Davies, Luca Molinaro, Umberto Mortara, Angelo Paolo dei Tos, Kenneth Ofori, E. d'Amore, Roberto Chiarle, Bo Ngan, E. Carraro, Marta Pillon, Shafinaz Hussein, Govind Bhagat, Marco Pizzi, L. Mussolin, Katia Basso
          </td>
          <td>2024-10-18</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Introduction: The worldwide challenge presented by human immunodeficiency virus type 1, affecting 39 million individuals, endures despite the highly active antiretroviral therapy. Limitations of it, including insufficiently addressing viral reservoirs leading to prolonged drug toxicity, resistance concerns, and economic burdens, have fueled exploration into alternative approaches. Gene editing and gene repression techniques have emerged as potential solutions. Objective: Conduct a comprehensive analysis of recent research following Preferred Reporting Items for Systematic Reviews and Meta-Analyses guidelines, focusing on genetic modification and gene repression in HIV-1-infected samples. Methods: A meticulous search identified 413 articles, of which 76 met strict criteria. Inclusion criteria involved studies on gene editing and HIV-1 expression (2019–2023), emphasizing CCR5 repression. Exclusions targeted weak evidence, lacking practical efficacy, and non-relevance to HIV-1 or genetic tools. Duplicates and non-open-access articles were also excluded. Results: The identified strategies show promise in their ability to mitigate the chronicity of the infection. However, challenges such as off-target effects, ethical considerations surrounding in vivo, ex vivo, and germline editing, and variable efficacy across studies are evident. The imperative for continued evaluation, refinement of techniques, and the development of effective administration methods is underscored. Conclusions: This topic review provides a nuanced exploration of recent advancements in gene editing and gene repression techniques for managing HIV-1. While the findings suggest promising avenues, the identified challenges necessitate ongoing research and innovation in order to harness the full potential of these strategies in the quest for an effective HIV-1 solution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fae1f8a23cc0b10ee128f34eb4680a5249d869" target='_blank'>
              Novel Approaches in HIV-1 Cure: Genetic Editing and Gene Repression Techniques. Topic Review
              </a>
            </td>
          <td>
            Nicolas Alejandro Castilla Salcedo
          </td>
          <td>2024-10-23</td>
          <td>Salud &amp; Sociedad Uptc</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precise transcriptional regulation is critical for cellular function and development, yet the mechanism of this process remains poorly understood for many genes. To gain a deeper understanding of the regulation of neuropsychiatric disease risk genes, we identified a total of 39 functional enhancers for four dosage-sensitive genes, APP, FMR1, MECP2, and SIN3A, using CRISPR tiling deletion screening in human induced pluripotent stem cell (iPSC)-induced excitatory neurons. We found that enhancer annotation provides potential pathological insights into disease-associated copy number variants. More importantly, we discovered that allelic enhancer deletions at SIN3A could be compensated by increased transcriptional activities from the other intact allele. Such allelic compensation effects (ACE) on transcription is stably maintained during differentiation and, once established, cannot be reversed by ectopic SIN3A expression. Further, ACE at SIN3A occurs through dosage sensing by the promoter. Together, our findings unravel a regulatory compensation mechanism that ensures stable and precise transcriptional output for SIN3A, and potentially other dosage-sensitive genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c9cf4cae8ed81690d40ccc0776865b484f3141" target='_blank'>
              CRISPR tiling deletion screens reveal functional enhancers of neuropsychiatric risk genes and allelic compensation effects (ACE) on transcription
              </a>
            </td>
          <td>
            Xingjie Ren, Lina Zheng, Lenka Maliskova, Tsz Wai Tam, Yifan Sun, Hongjiang Liu, Jerry Lee, M. Takagi, Bin Li, Bing Ren, Wei Wang, Yin Shen
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Metabolic enzymes perform life-sustaining functions in various cellular compartments. Anecdotally, metabolic activity is observed to vary between genetically identical cells, which impacts drug resistance, differentiation, and immune cell activation. However, no large-scale resource systematically reporting metabolic cellular heterogeneity exists. Here, we leverage imaging-based single-cell spatial proteomics to reveal the extent of non-genetic variability of the human enzymatic proteome, as a proxy for metabolic states. Nearly two fifths of enzymes exhibit cell-to-cell variable expression, and half localize to multiple cellular compartments. Metabolic heterogeneity arises largely autonomously of cell cycling, and individual cells reestablish these myriad metabolic phenotypes over several cell divisions. These results establish that cell-to-cell enzymatic heterogeneity is an organizing principle of cell biology and one that may rewire our understanding of drug resistance, treatment design, and other aspects of medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc03cc47c0d94e752c4532dd652b1361ad12454" target='_blank'>
              Dissecting autonomous enzyme variability in single cells
              </a>
            </td>
          <td>
            Christian Gnann, Anthony J. Cesnik, A. Sigaeva, Trang Le, Sanem Sariyar, Diana Mahdessian, Rutger Schutten, Preethi Raghavan, Manuel D Leonetti, Cecilia Lindskog, Mathias Uhlén, Ulrika Axelsson, Emma Lundberg
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99b084e2208e5b0e67e8e029bea90456f03863d" target='_blank'>
              A multiplex single-cell RNA-Seq pharmacotranscriptomics pipeline for drug discovery.
              </a>
            </td>
          <td>
            Alice Dini, Harlan Barker, Emilia Piki, Subodh Sharma, Juuli Raivola, A. Murumägi, Daniela Ungureanu
          </td>
          <td>2024-10-31</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In recent years, large-scale sequencing efforts have identified targetable driver mutations in haematopoietic stem cells. These efforts have led to the development and approval of nine novel agents for relapsed or refractory acute myelogenous leukaemia (R/R AML). However, despite an expansion in targeted therapies, achieving a durable remission in AML and high-risk myelodysplastic syndrome (HR-MDS) remains a significant challenge, and there is an urgent need for new effective treatments. Modulation of aberrant RNA splicing has emerged as a novel therapeutic approach in myeloid diseases. Aberrant splicing drives dysregulated gene expression that promotes tumourigenesis through increased proliferation and metastatic potential, immune evasion, decreased apoptosis, and chemotherapy resistance. Mutations in spliceosomal components have been identified in numerous cancer subtypes, with mutations in RNA binding proteins SF3B1, SRSF2, U2AF1, and ZRSR2 occurring frequently in AML and in up to 60% of patients with MDS, as well as in chronic myelomonocytic leukaemia and in 10% of patients with chronic lymphocytic leukaemia. In this review, we explore therapeutic strategies targeting aberrant splicing and the potential of these approaches to drive clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b179523c7f6d0d667d6bb250cabd47b56f2a64" target='_blank'>
              Therapeutic strategies targeting aberrant RNA splicing in myeloid malignancies.
              </a>
            </td>
          <td>
            Leora S Boussi, Jeetayu Biswas, Omar Abdel-Wahab, E. Stein
          </td>
          <td>2024-10-15</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Cell type-specific transcriptional heterogeneity in embryonic mouse skin is well-documented, but few studies have investigated the regulatory mechanisms. Results Here, we present high throughput single-cell chromatin accessibility and transcriptome sequencing (HT-scCAT-seq), a method that simultaneously profiles transcriptome and chromatin accessibility. We utilized HT-scCAT-seq to dissect the gene regulatory mechanism governing epidermal stratification, periderm terminal differentiation, and fibroblast specification. Conclusions By linking chromatin accessibility to gene expression, we identified candidate cis-regulatory elements (cCREs) and target genes crucial for dermal and epidermal development. We described cells with similar gene expression profiles that exhibit distinct chromatin accessibility statuses during periderm terminal differentiation. Finally, we characterized the underlying lineage-determining transcription factors (TFs), and demonstrated that ALX4 and RUNX2 were candidate TF regulators of the dermal papilla lineage development through in silico perturbation analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dbfdb1c62a89b951b0864a8a764e5479e4ed902" target='_blank'>
              Deciphering gene regulatory programs in mouse embryonic skin through single-cell multiomics analysis
              </a>
            </td>
          <td>
            Q. Deng, Pengfei Cai, Yingjie Luo, Zhongjin Zhang, Wen Ma, Zijie Huang, Xiaoya Chen, Shijie Hao, Weiguang Ma, Jiangshan Xu, Mengnan Cheng, Xiumei Lin, Ru Zhou, Shanshan Duan, Junjie Chen, Ronghai Li, Xuyang Shi, Chang Liu, Peng Gao, Jianting Li, Jun Xie, Longqi Liu, Yue Yuan, Chuanyu Liu
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutations are not always phenotypically active or show different effects in different individuals. While the mechanisms underlying this variable relationship between mutations and phenotypes are largely elusive, some specific genes may influence the phenotypic effects of cryptic variation. We employ the toolbox of Saccharomyces cerevisiae to perform a genome-wide screen aimed at identifying these so-called genetic buffer genes. Measuring the fitness of 1.8 million mutated strains identified a small set of evolutionary conserved buffer genes involved in protein folding and chromatin organization, including GIM3, SSA2, HOG1 and FKH2. Deletion of these genes increases the fitness effect of de novo mutations as well as standing genetic variation, with some mutations becoming adaptive. Moreover, losing a buffer gene results in a decline of standing genetic variation. Together, these results reveal a set of conserved genes that influence the phenotypic outcome of mutations and play a role in complex genetics and incomplete penetrance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23f77a1888d72233722ce01119fc36d042aeaae" target='_blank'>
              Genes involved in protein folding and chromatin organization buffer genetic variation
              </a>
            </td>
          <td>
            Jens Frickel, Mohammed T Tawfeeq, Emily Baker, Sara Baco, Jonas Rombout, K. Voordeckers, Daniel F. Jarosz, Sibylle C. Vonesch, K. Verstrepen
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Allele-specific expression (ASE) is the differential abundance in levels of mRNAs that originated from the paternal and maternal copies of a gene. Such allelic imbalances can contribute to phenotypic variation and influence disease traits, including cancer. There is common ASE in tumors that results from somatic copy-number alterations (CNAs) at the DNA level, but there also exist other causes of ASE: cis-acting genetic or epigenetic variation that can lead to differential expression between the two alleles. However, the latter, non-CNA mechanisms of ASE remain understudied in cancer, as well as their role in tumor evolution and impact on clinical outcomes. By integrating a wide variety of genomic and transcriptomic pan-cancer data from the TCGA project, we show that ASE favoring the preferential expression of the mutant allele in some driver genes is subject to positive selection, and that these events are associated with worse overall survival across all cancer types. We found that the impact of ASE triggered by non-CNA causes is substantial, and we propose that some instances of cis-ASE are explained by the epigenetic changes affecting alleles differently. Furthermore, as a second mechanism, we find that splicing-altering mutations are selected in various cancer genes and result in ASE. We anticipate that the study and understanding of the role of mutant allele imbalances at the mRNA level can help understand epigenetic changes during cancer evolution, as well as identify new prognostic markers and therapeutic approaches that target altered allelic expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a918fe3f2472f99c1403cdc4bbe583961852532" target='_blank'>
              Allele-specific expression is selected in tumorigenesis, results from epigenetic changes and has prognostic relevance
              </a>
            </td>
          <td>
            Guillermo Palou-Márquez, Pere Pericot-Masdevall, F. Supek
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Optimizing prevention and early detection of cancer requires understanding the number, types and timing of driver mutations. To quantify this, we exploited the elevated cancer incidence and mutation rates in germline BRCA1 and BRCA2 (gBRCA1/2) carriers. Using novel statistical models, we identify genomic deletions as the likely rate-limiting mutational processes, with 1-3 deletions required to initiate breast and ovarian tumors. gBRCA1/2-driven hereditary and sporadic tumors undergo convergent evolution to develop a similar set of driver deletions, and deletions explain the elevated cancer risk of gBRCA1/2-carriers. Orthogonal mutation timing analysis identifies deletions of chromosome 17 and 13q as early, recurrent events. Single-cell analyses confirmed deletion rate differences in gBRCA1/2 vs. non-carrier tumors as well as cells engineered to harbor gBRCA1/2. The centrality of deletion-associated chromosomal instability to tumorigenesis shapes interpretation of the somatic evolution of non-malignant tissue and guides strategies for precision prevention and early detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/142f79a57f8e5624ca9c9770400c8298f772b241" target='_blank'>
              Deletions Rate-Limit Breast and Ovarian Cancer Initiation
              </a>
            </td>
          <td>
            Kathleen E. Houlahan, Mahad Bihie, Julián Grandvallet Contreras, Daniel J. Fulop, Gonzalo Lopez, Hsin-Hsiung Huang, P. Van Loo, Christina Curtis, P. Boutros, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a heterogeneous disease where multiple mutations in genes encoding transcriptional and growth regulators lead to an extensive rewiring of the gene regulatory network (GRN), thereby changing the identity of hematopoietic stem and progenitor cells in a way that blocks myeloid differentiation [1]. One hallmark of AML is the acquisition of mutations in growth factor receptor and growth signalling genes such as FMS-like tyrosine kinase 3 ( FLT3 ), the receptor for stem cell factor ( KIT ) and RAS . FLT3 is one of the most frequently mutated genes in AML. 25% of AMLs display an internal tandem duplication (ITD) which renders this receptor constitutively active. The prognosis for FLT3-ITD AML patients is dire as even after complete remission after chemotherapy, many patients relapse. Hence, inhibitors that also target the aberrant FLT3 receptor have been introduced but even then most patients relapse and become resistant, by either acquiring additional FLT3 mutations or mutations in downstream signalling molecules. However, many patients do not show additional mutations and rewire their signalling pathways to evade FLT3 inhibition by becoming dependent on other cytokines [2]. Significantly, AML cells can become also become resistant in a non-genetic way by changing their gene expression programs [3] but the role of aberrant signalling in this process remained unclear.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7212e2ffaa1e48f58fa237164fd146932496cb53" target='_blank'>
              Gene regulatory network and signalling pathway rewiring: How blood cancer cells shift their shapes to evade drug treatment
              </a>
            </td>
          <td>
            Constanze Bonifer, P. Cockerill
          </td>
          <td>2024-01-06</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is a dynamic nexus where cancer cell metabolism and the immune system intricately converge, with nucleotide metabolism (NM) playing a pivotal role. This review explores the critical function of NM in cancer cell proliferation and its profound influence on the TME and immune landscape. NM is essential for DNA and RNA synthesis and is markedly upregulated in cancer cells to meet the demands of rapid growth. This metabolic rewiring fuels cancer progression, but also shapes the TME, impacting the function and viability of immune cells. The altered nucleotide milieu in the TME can suppress immune response, aiding cancer cell evasion from immune surveillance. Drug discoveries in the field of NM have revealed different therapeutic strategies, including inhibitors of nucleotide synthesis and drugs targeting salvage pathways, which are discussed thoroughly in this review. Furthermore, the emerging strategy of combining NM-targeted therapies with immunotherapies is emphasised, particularly their effect on sensitising tumours to immune checkpoint inhibitors and enhancing overall treatment efficacy. The Human Genome Project paved the way for personalised medicine, countering the established 'one size fits all' approach to cancer treatment. Advances in understanding the TME and NM have spurred interest in personalised therapeutic strategies. This review highlights the potential of leveraging individual tumour metabolic profiles to guide treatment selection, aiming to optimise efficacy and minimise adverse effects. The strategic importance of targeting NM in cancer therapy and its synergistic potential with immunotherapies offers a path towards more effective and personalised cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bc17614b60cc3e87bf59862d1663d401029e4eb" target='_blank'>
              Harnessing nucleotide metabolism and immunity in cancer: a tumour microenvironment perspective.
              </a>
            </td>
          <td>
            Hadil Sulieman, Alexandra J Emerson, Peter M. Wilson, Karl A Mulligan, R. Ladner, Melissa J LaBonte
          </td>
          <td>2024-09-22</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Treatment with the hypomethylating agent 5-azacytidine (AZA) increases survival in high-risk (HR) myelodysplastic syndrome (MDS) patients, but predicting patient response and overall survival remains challenging. To address these issues, we analyzed mutational and transcriptional profiles in CD34+ hematopoietic stem/progenitor cells (HSPCs) before and following AZA therapy in MDS patients. AZA treatment led to a greater reduction in the mutational burden in both blast and hematological responders than non-responders. Blast and hematological responders showed transcriptional evidence of pre-treatment enrichment for pathways such as oxidative phosphorylation, MYC targets, and mTORC1 signaling. While blast non-response was associated with TNFa signaling and leukemia stem cell signature, hematological non-response was associated with cell-cycle related pathways. AZA induced similar transcriptional responses in MDS patients regardless of response type. Comparison of blast responders and non-responders to normal controls, allowed us to generate a transcriptional classifier that could predict AZA response and survival. This classifier outperformed a previously developed gene signature in a second MDS patient cohort, but signatures of hematological responses were unable to predict survival. Overall, these studies characterize the molecular consequences of AZA treatment in MDS HSPCs and identify a potential tool for predicting AZA therapy responses and overall survival prior to initiation of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f64e8b321545ad08389931dee01148ac0d9c897" target='_blank'>
              Molecular profiling of pre- and post- 5-azacytidine myelodysplastic syndrome samples identifies predictors of response
              </a>
            </td>
          <td>
            Mónica del Rey González, Sohini Chakraborty, J. Hernández-Sánchez, M. Díez Campelo, Christopher Y. Park, J. H. Hernández Rivas
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Caspase activated DNase (CAD) induced DNA breaks promote cell differentiation and therapy-induced cancer cell resistance. CAD targeting activity is assumed to be unique to each condition, as differentiation and cancer genesis are divergent cell fates. Here, we made the surprising discovery that a subset of CAD-bound targets in differentiating muscle cells are the same genes involved in the genesis of cancer-causing translocations. In muscle cells, a prominent CAD-bound gene pair is Pax7 and Foxo1a, the mismatched reciprocal loci that give rise to alveolar rhabdomyosarcoma. We show that CAD-targeted breaks in the Pax7 gene are physiologic to reduce Pax7 expression, a prerequisite for muscle cell differentiation. A cohort of these CAD gene targets are also conserved in early differentiating T cells and include genes that spur leukemia/lymphoma translocations. Our results suggest the CAD targeting of translocation prone oncogenic genes is non-pathologic biology and aligns with initiation of cell fate transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95535f8b5c0fda938ec705d36c58cef6067d4ac1" target='_blank'>
              Caspase-Activated DNase localizes to cancer causing translocation breakpoints during cell differentiation
              </a>
            </td>
          <td>
            Dalal Alsowaida, Brian D. Larsen, Sarah Hachmer, M. Azimi, Eric Arezza, Steve Brunette, Steven Tur, C. Palii, Bassam Albraidy, Claus S. Sorensen, Marjorie Brand, F. J. Dilworth, L. Megeney, Senior Scientist
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract DNA damage response (DDR) pathway is the coordinated cellular network dealing with the identification, signaling, and repair of DNA damage. It tightly regulates cell cycle progression and promotes DNA repair to minimize DNA damage to daughter cells. Key proteins involved in DDR are frequently mutated/inactivated in human cancers and promote genomic instability, a recognized hallmark of cancer. Besides being an intrinsic property of tumors, DDR also represents a unique therapeutic opportunity. Indeed, inhibition of DDR is expected to delay repair, causing persistent unrepaired breaks, to interfere with cell cycle progression, and to sensitize cancer cells to several DNA‐damaging agents, such as radiotherapy and chemotherapy. In addition, DDR defects in cancer cells have been shown to render these cells more dependent on the remaining pathways, which could be targeted very specifically (synthetic lethal approach). Research over the past two decades has led to the synthesis and testing of hundreds of small inhibitors against key DDR proteins, some of which have shown antitumor activity in human cancers. In parallel, the search for synthetic lethality interaction is broadening the use of DDR inhibitors. In this review, we discuss the state‐of‐art of ataxia‐telangiectasia mutated, ataxia‐telangiectasia‐and‐Rad3‐related protein, checkpoint kinase 1, Wee1 and Polθ inhibitors, highlighting the results obtained in the ongoing clinical trials both in monotherapy and in combination with chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3effb948dd60b6e28e5a1e98f095e523b94a901" target='_blank'>
              Targeting the DNA damage response in cancer
              </a>
            </td>
          <td>
            Guffanti Federica, Chiappa Michela, Damia Giovanna
          </td>
          <td>2024-10-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract In multiple myeloma, as in B‐cell malignancies, mono‐ and especially bi‐allelic TP53 gene inactivation is a high‐risk factor for treatment resistance, and there are currently no therapies specifically targeting p53 deficiency. In this study, we evaluated if the loss of cell cycle control in p53‐deficient myeloma cells would confer a metabolically actionable vulnerability. We show that CTP synthase 1 (CTPS1), which encodes a CTP synthesis rate‐limiting enzyme essential for DNA and RNA synthesis in lymphoid cells, is overexpressed in samples from myeloma patients displaying a high proliferation rate (high MKI67 expression) or a low p53 score (synonymous with TP53 deletion and/or mutation). This overexpression of CTPS1 was associated with reduced survival in two cohorts. Using scRNA‐seq analysis in 24 patient samples, we further demonstrate that myeloma cells in the S or G2/M phase display high CTPS1 expression. Pharmacological inhibition of CTPS1 by STP‐B induced cell cycle arrest in early S phase in isogenic NCI‐H929 or XG7 TP53 +/+, TP53 −/−, and TP53 R175H/R175H cells and in a TP53 −/R123STOP patient sample. The functional annotation of transcriptional changes in 10 STP‐B‐treated myeloma cell lines revealed a decrease in protein translation and confirmed the blockade of cells into the S phase. The pharmacological inhibition of ATR, which governs the intrinsic S/G2 checkpoint, in STP‐B‐induced S‐phase arrested cells synergistically induced cell death in TP53 +/+, TP53 −/−, and TP53 R175H/R175H isogenic cell lines (Bliss score >15). This combination induced replicative stress and caspase‐mediated cell death and was highly effective in resistant/refractory patient samples with TP53 deletion and/or mutation and in TP53 −/− NCI‐H929 xenografted NOD‐scid IL2Rgamma mice. Our in vitro, ex vivo, and in vivo data provide the rationale for combined CTPS1 and ATR inhibition for the treatment of p53‐deficient patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb4c0e19c2b9b0bfd1a07ec3cde49d50f22f3de" target='_blank'>
              Combined inhibition of CTPS1 and ATR is a metabolic vulnerability in p53‐deficient myeloma cells
              </a>
            </td>
          <td>
            Romane Durand, C. Bellanger, G. Descamps, Christelle Dousset, S. Maïga, J. Derrien, Laura Thirouard, Louise Bouard, H. Asnagli, Philip A Beer, Andrew Parker, P. Gomez‐Bougie, Marie‐Claire Devilder, P. Moreau, C. Touzeau, A. Moreau-Aubry, D. Chiron, C. Pellat-Deceunynck
          </td>
          <td>2024-10-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Leukemia represents the most prevalent malignancy in children, constituting 30% of childhood cancer cases, with acute lymphoblastic leukemia (ALL) being particularly heterogeneous. This paper explores the role of alternative splicing in leukemia, highlighting its significance in cancer development and progression. Aberrant splicing is often driven by mutations in splicing-factor genes, which can lead to the production of variant proteins that contribute to oncogenesis. The spliceosome, a complex of small nuclear RNAs and proteins, facilitates RNA splicing, a process critical for generating diverse mRNA and protein products from single genes. Mutations in splicing factors, such as U2AF1, SF3B1, SRSF2, ZRSR2, and HNRNPH1, are frequently observed across various hematological malignancies and are associated with poor prognosis and treatment resistance. This research underscores the necessity of understanding the mechanisms of RNA splicing dysregulation in order to develop targeted therapies to correct these aberrant processes, thereby improving outcomes for patients with leukemia and related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f8fbeead7c45cf409ca53e4eca94f42389d1bc" target='_blank'>
              Alternative Splicing: A Potential Therapeutic Target in Hematological Malignancies
              </a>
            </td>
          <td>
            G. Temaj, S. Chichiarelli, Sarmistha Saha, Pelin Telkoparan-Akillilar, Nexhibe Nuhii, R. Hadziselimovic, Luciano Saso
          </td>
          <td>2024-10-29</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Chronic Myeloid Leukemia (CML) is a myeloproliferative stem cell malignancy. Chronic Phase CML (CP) is treatable with overall survival equivalent to general public. Nevertheless, a proportion of CP-CML progresses to the accelerated phase (AP-) and ultimately blast crisis (BC), with the later having an overall survival of 3-23 months, making it almost a fatal manifestation. Therefore, treatment of BC-CML is of the biggest challenges in modern cancer medicine. FDA-approved drugs are available against a large number of mutated genes reported in AML and ALL. As BC-CML resembles AML (myeloid BC) or ALL (lymphoid BC), this study was designed to find out AML-/ALL lineage gene mutations in BC-CML, find their druggability and feasibility of their utilization in patient-tailored treatment of BC-CML. Patients & Methods: The study included 141 CML patients (123 CP-CML as control groups; 6 AP-CML and 12 BC-CML as experimental groups). Most of the patients received imatinib mesylate (IM) as first-line treatment. All response criteria were per European LeukemiaNet (ELN) guidelines 2020. Whole exome sequencing (WES) was carried out to find out druggable gene mutations and the druggability of the mutated genes was determined using online tool www.pandrugs.com. SAS/STAT software version 9.4 was used for data analysis (SAS Institute Inc., Cary, NC, USA). For statistical computing, the R package was employed (Vienna, Austria). The study was approved by ethical committee of KAIMRC and carried out per guidelines o of the Helsinki Declaration Results: Overall male-to-female ratio was 1.6:1 and the mean age was 36.4 (range: 9 -67) years. Eighteen (12.8%) patients progressed to AP-CML while 12 (8.5%) to BC-CML finally. BC-ML patients had overall poorer response to TKIs and higher mortality rate (75%) that prompted to look for druggable gene mutations in advanced phase CML. WES showed overall 64 AML-/ALL- associated gene mutated in advanced phase CML patients. AP-CML had 1644 variants, whereas BC-CML had 2531 variants, with a 54% gain in mutations from AP-CML to BC-CML (P< 0.000001). Among AML-/ALL- related mutated genes were NPM1 (%1.98), DNMT3A (%1.86), PML (%1.82), AKT1 (%1.62), CBL (%1.30), JAK2 (%0.71), TET2 (%0.59), IDH1 (%0.32), and BCL2, which have FDA-approved drug to target them. Conclusions: NGS found druggable mutations in many AML-/ALL-lineage genes. Many of the corresponding drugs are either already approved for BC-CML treatment or in clinical trial phase. We conclude that our approach can help in finding druggable gene mutations related to AML-/ALL-lineage genes in almost every BC-CML patients and provide a practical guidance for drug repurposing to individualize BC-CML patient treatment. Keywords: Blast crisis Chronic Myeloid Leukemia; druggable mutations; pa-tient-tailored treatment; AML lineage genes; ALL lineage genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca4648f1ee7dfecd9d611776df7b63c74e5fcb8" target='_blank'>
              Investigations on druggable gene mutations related to AML/ALL lineage genes in Advanced Phases of CML: Implications in patient-tailored therapy of blast crisis CML in TKI era
              </a>
            </td>
          <td>
            N. Alanazi, A. AlGarni, S. Almukhaylid, M. AlMajed, S. Alanazi, M. Khan, M. Sabar, M. Iqbal, A. Jameel, A. Hussian, D. Almaghlouth, A. Alsuwaidani, G. Alsalem, N. AlMutairi, H. Almasoudi, B. AlShehab, S. Alfayez, M. Butwyibah, B. Alnajad, F. Alali, A. Al-rasasi, K. Adeel, S. Al Hakeem, T. Karar, F. Alsaab, Y. Taleb, S. Shahbaz, S. Malik, A. Mahmood, S. Basit, M. Anharullah, A. Aleem, I. F. Zafar, R. Naeem, M. Shamas, Z. Iqbal
          </td>
          <td>2024-09-09</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4c265b900d210c2847c325efc074e2d026f85f2" target='_blank'>
              The BIM deletion polymorphism potentiates the survival of leukemia stem and progenitor cells and impairs response to targeted therapies.
              </a>
            </td>
          <td>
            Mengge Yu, G. Nah, Vaidehi Krishnan, Fatin Nasha Bte Sulaimi, K. Ng, Chuqi Wang, Shruti Bhatt, Charles T.H. Chuah, David E Bergstrom, S. Ong
          </td>
          <td>2024-10-22</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39f338bea38e6dfbcc4292c9fad6a48a280f8ab6" target='_blank'>
              A Bruton tyrosine kinase inhibitor-resistance gene signature predicts prognosis and identifies TRIP13 as a potential therapeutic target in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Yangyang Ding, Keke Huang, Cheng Sun, Zelin Liu, Jinli Zhu, Xunyi Jiao, Ya Liao, Xiangjiang Feng, Jingjing Guo, Chunhua Zhu, Zhimin Zhai, Shudao Xiong
          </td>
          <td>2024-09-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background: Melanoma remains one of the most challenging cancers to treat effectively with drug resistant remaining a constant concern, primarily with activating BRAF mutations. Mutations in the BRAF gene appear in approximately 50% of patients, 90% of which are V600E. Two frontline BRAF inhibitors (BRAFi), vemurafenib and dabrafenib, are frequently used to treat unresectable or metastatic BRAF V600E melanoma. Initial response rates are high, but soon thereafter, 70-80% of patients develop resistance to treatment within a year. A major mechanism of resistance is the generation of a secondary Q61K mutation in the NRAS gene. Methods: We have developed an approach in which a CRISPR-Cas complex can be designed to distinguish between mutant genes enabling resistance to standard care in tumor cells and normal genomes of healthy cells. For the first time, we demonstrated the utility of two CRISPR-directed mutation-specific editing approaches to restore BRAFi sensitivity in BRAFV600E/NRASQ61K resistant A375 cells. Results: We utilize an AsCas12a protospacer adjacent motif site created by the NRAS Q61K mutation and the Q61K mutation in the critical seed region of an SaCas9 sgRNA for Q61K-selective targeting. We show here that both approaches allow for effective NRAS targeting of only mutated-Q61K and after CRISPR-directed Q61K-targeting, previously resistant A375 cells are re-sensitized to BRAFi treatment. Conclusion: Our data support the feasibility of the development of CRISPR-Cas therapeutic approaches to the treatment of melanoma. Successful therapeutic CRISPR-directed gene editing would enable both specific and efficient editing of a mutation-specific targeting approach eliminate concern for on- and off-target damage to the genomes of healthy cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0aea0b595d066144e478150c4bdebe3df9d61c5" target='_blank'>
              Mutation-Specific CRISPR Targeting with SaCas9 and AsCas12a Restores Therapeutic Sensitivity in Treatment-Resistant Melanoma.
              </a>
            </td>
          <td>
            Brett M. Sansbury, Sophia B. Masciarelli, Salma Kaouser, Olivia M. Tharp, Kelly Banas, Eric B Kmiec
          </td>
          <td>2024-10-10</td>
          <td>The CRISPR journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="While immune checkpoint inhibitors have revolutionized cancer therapy, many patients exhibit poor outcomes. Here, we show immunotherapy responses in bladder and non–small cell lung cancers are effectively predicted by factoring tumor mutation burden (TMB) into burdens on specific protein assemblies. This approach identifies 13 protein assemblies for which the assembly-level mutation burden (AMB) predicts treatment outcomes, which can be combined to powerfully separate responders from nonresponders in multiple cohorts (e.g., 76% versus 37% bladder cancer 1-year survival). These results are corroborated by (i) engineered disruptions in the predictive assemblies, which modulate immunotherapy response in mice, and (ii) histochemistry showing that predicted responders have elevated inflammation. The 13 assemblies have diverse roles in DNA damage checkpoints, oxidative stress, or Janus kinase/signal transducers and activators of transcription signaling and include unexpected genes (e.g., PIK3CG and FOXP1) for which mutation affects treatment response. This study provides a roadmap for using tumor cell biology to factor mutational effects on immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5f818fb592baa5f66559ad69838daf202c25a6" target='_blank'>
              Prediction of immunotherapy response using mutations to cancer protein assemblies
              </a>
            </td>
          <td>
            JungHo Kong, Xiaoyu Zhao, Akshat Singhal, Sungjoon Park, Robin Bachelder, Jeanne Shen, Haiyu Zhang, Jimin Moon, Changho Ahn, C. Ock, Hannah Carter, T. Ideker
          </td>
          <td>2024-09-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Eukaryotic protein translation has slowly gained the scientific community’s attention for its advanced and powerful therapeutic potential. However, recent technical developments in studying ribosomes and global translation have revolutionized our understanding of this complex multistep process. These developments have improved and deepened the current knowledge of mRNA translation, sparking excitement and new possibilities in this field. Translation factors are crucial for maintaining protein synthesis homeostasis. Since actively proliferating cancer cells depend on protein synthesis, dysregulated protein translation is central to tumorigenesis. Translation factors and their abnormal expressions directly affect multiple oncogenes and tumor suppressors. Recently, small molecules have been used to target translation factors, resulting in translation inhibition in a gene-specific manner, opening the door for developing translation inhibitors that can lead to novel chemotherapeutic drugs for treating multiple cancer types caused by dysregulated translation machinery. This review comprehensively summarizes the involvement of translation factors in tumor progression and oncogenesis. Also, it sheds light on the evolution of translation factors as novel drug targets for developing future therapeutic drugs for treating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a240b5f9df564644b46e31b401802c9115dd173" target='_blank'>
              Exploiting Translation Machinery for Cancer Therapy: Translation Factors as Promising Targets
              </a>
            </td>
          <td>
            Urmila Sehrawat
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). Using a functional genome-wide CRISPRi screen in PDAC cells, we identified activators of selective MYC translation through its 5’ untranslated region (5’UTR) and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. Among these RBPs, our top hit was RBM42, which is highly expressed in PDAC and predicts poor survival. Combining polysome sequencing and CLIP-seq analyses, we find that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing IP-mass spectrometry analysis, we find that RMB42 is a novel ribosome-associated protein (RAP). Using DMS-Seq and mutagenesis analysis, we show that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex. Importantly, RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/168afbf485caea55a52d0fa0de95551cbcbbc5ea" target='_blank'>
              Functional screen for mediators of onco-mRNA translation specificity
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetic intra-tumour heterogeneity (gITH) is a universal property of all cancers. It emerges from the interplay of cell division, mutation accumulation and selection with important implications for the evolution of treatment resistance. Theoretical and data-driven approaches extensively studied gITH in ageing somatic tissues or cancers at detection. Yet, the expected patterns of gITH during and after treatment are less well understood. Here, we use stochastic birth-death processes to investigate the expected patterns of gITH across different treatment scenarios. We consider homogeneous treatment response with shrinking, growing and stable disease, and follow up investigating heterogeneous treatment response with sensitive and resistant cell types. We derive analytic expressions for the site frequency spectrum, the total mutational burden and the single-cell mutational burden distribution that we validate with computer simulations. We find that the SFS after homogeneous treatment response retains its characteristic power-law tail, while emergent resistant clones cause peaks corresponding to their sizes. The frequency of the largest resistant clone is subdominant and independent of the population size at detection, whereas the relative total number of resistant cells increases with detection size. Furthermore, the growth dynamics under treatment determine whether the total mutational burden is dominated by preexisting or newly acquired mutations, suggesting different possible treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b068763b28793a776bfaf3fe0647aae0b724cad" target='_blank'>
              On the patterns of genetic intra-tumour heterogeneity before and after treatment
              </a>
            </td>
          <td>
            Alexander Stein, Benjamin Werner
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This review offers an in-depth analysis of mitochondrial DNA (mtDNA) mutations in colorectal cancer stem cells (CSCs), emphasizing their significant impact on tumor dynamics and potential therapeutic strategies. CSCs are a special subpopulation due to their unique capabilities for self-renewal, differentiation, and resistance to conventional therapies. Given that CSCs significantly differ from other tumor cell subpopulations, particularly in their metabolic properties, and considering that colorectal cancer is a malignancy characterized by mitochondrial dysfunction, this review aims to put together existing data on the differences in the mitochondrial genome of CSCs compared to other colorectal tumor cell subpopulations. Additionally, the review seeks to explore the potential roles of these differences and to identify new ideas for therapeutic strategies. Key topics include the identification and properties of CSCs in colorectal cancer, the distinctive features of the mitochondrial genome, and the functional consequences of mtDNA mutations. The review hypothesizes that CSCs rely on well-functioning mitochondria for crucial aspects like energy production; yet, mtDNA mutations can lead to mitochondrial dysfunction, altering CSC characteristics and influencing cancer progression. The article discusses emerging therapeutic approaches targeting mitochondrial function in colorectal CSCs and highlights the need for advanced research, including the development of preclinical models and exploration of targeted therapies, to improve the understanding and treatment of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebae925d5c7100b4c278e4586e85603f7ee40d86" target='_blank'>
              Mitochondrial DNA Mutations in Colorectal Cancer Stem Cells: Implications for Tumor Dynamics and Therapeutic Strategies.
              </a>
            </td>
          <td>
            N. Shakhpazyan, L. Mikhaleva, Arcady L Bedzhanyan, Z. Gioeva, A. Mikhalev, K. Midiber, A. Konyukova, D. Atiakshin, Igor Buchwalow, Markus Tiemann, A. Orekhov
          </td>
          <td>2024-09-11</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 At the Gene Editing Institute, we are advancing a novel strategy for the treatment of pancreatic ductile adenocarcinoma by employing CRISPR-directed gene editing to disable the functions of key genes that block effective drug application including chemotherapy, radiation and immunotherapy. No matter how careful one is at designing a specific drug against a specific protein, ultimately, the tumor cell develops a workaround. A paradigm shift is required to truly overcome this formidable barrier. We’ve accomplished this by leveraging the effectiveness of standard care and augmenting it, thereby reducing patient suffering, increasing treatment tolerance and, most importantly, improving outcomes. With an enthusiastic response for this concept from the FDA- INTERFACT meeting, we have demonstrated that a carefully selected CRISPR/Cas complex disables the NRF2 gene in the tumor cell and significantly reduces the dose of chemotherapy required to halt pancreatic tumor cell growth, and consequently promoting cell death. The efficiency of gene editing approaches 70 % routinely, which destroys the function of NRF2 and enables additional killing of residual tumor cells by chemotherapeutic agents. As a consequence of CRISPR/Cas activity, we effectively reduce the amount of chemotherapy needed to kill tumor cells by over 20-fold. This level of reduction should enable patients to complete their treatment regimens more successfully, at lower dosages and with far fewer adverse side effects. In addition to reduced suffering, this will invariably prolong survival as we are augmenting already proven treatments. This approach is agnostic to the haplotype of the tumor cell and does not rely on specific biomarker composition; radically expanding the patient population that would be suitable for this type of treatment. We will discuss the impact of CRISPR/Cas-gene editing to disable specific genes, including NRF2 and mutant KRAS, to enable current therapies to work at greater efficacy and at lower dosages.
 Citation Format: London P McGill, Kelly H Banas, Gregory Tiesi, Eric B Kmiec. A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/159a6092b196d29302afbdfaca37a6f4ece01dff" target='_blank'>
              Abstract A010: A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma
              </a>
            </td>
          <td>
            L. McGill, Kelly Banas, Gregory Tiesi, Eric B Kmiec
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Treatment of advanced prostate cancer is challenging due to a lack of effective therapies. Therefore, it is important to understand the molecular mechanisms underlying therapeutic resistance in prostate cancer and to identify promising drug targets offering significant clinical advantages. Given the pivotal role of dysregulated transcriptional programs in the therapeutic response, it is essential to prioritize translational efforts targeting cancer-associated transcription factors (TFs). The present study investigated whether chromatin accessibility was associated with therapeutic resistance in prostate cancer using Assay for Transposase-Accessible Chromatin with sequencing (ATAC-seq) data. The bioinformatics analysis identified differences in chromatin accessibility between the drug response (Remission) and drug resistance (Disease) groups. Additionally, a significant association was observed between chromatin accessibility, transcriptional output and TF activity. Among TFs, forkhead box protein M1 (FOXM1) was identified as a TF with high activity and expression in the Disease group. Notably, the results of the computational analysis were validated by FOXM1 knockdown experiments, which resulted in suppressed cell proliferation and enhanced therapeutic sensitivity in prostate cancer cells. The present findings demonstrated that chromatin accessibility and TF activity may be associated with therapeutic resistance in prostate cancer. Additionally, these results provide the basis for future investigations aimed at understanding the molecular mechanisms of drug resistance and developing novel therapeutic approaches for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b100f863b1689ff6606a9905e012f9724b28e5ed" target='_blank'>
              Chromatin accessibility is associated with therapeutic response in prostate cancer
              </a>
            </td>
          <td>
            Sanghoon Lee, Da Young Lee, I. So, J. Chun, Ju-Hong Jeon
          </td>
          <td>2024-10-14</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Gliomas, originating from the most common non-neuronal cells in the brain (glial cells), are the most common brain tumors and are associated with high mortality and poor prognosis. Glioma cells exhibit a tendency to disrupt normal cell-cycle regulation, leading to abnormal proliferation and malignant growth. This study investigated the predictive potential of GJC1 in gliomas and explored its relationship with the cell cycle. Methods Retrospective analysis of RNA-seq and single-cell sequencing data was conducted using the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. The differential expression of GJC1 in gliomas with various pathological features and in different non-neuronal cell groups was analyzed. Functional data were examined using gene set variation analysis (GSVA). Furthermore, CellMiner was used to evaluate the relationship between GJC1 expression and predicted treatment response across these databases. Results GJC1 expression was enriched in high-grade gliomas and 1p/19q non-codeletion gliomas. GJC1 enrichment was observed in classical and mesenchymal subtypes within the TCGA glioma subtype group. In single-cell subgroup analysis, GJC1 expression was higher in glioma tissues compared to other non-neuronal cells. Additionally, the TCGA classical subtype of glioma cells exhibited more GJC1 expression than the other subgroups. GJC1 emerged as an independent prognostic factor for overall survival in glioma. GSVA unveiled potential mechanisms by which GJC1 may impact cell-cycle regulation in glioma. Finally, a significant correlation was observed between GJC1 expression and the sensitivity of multiple anti-cancer drugs. Conclusion These findings confirmed GJC1 as a novel biomarker and provided insights into the differential gene expression in non-neuronal cells and the impact of the cell cycle on gliomas. Consequently, GJC1 may be used to predict glioma prognosis and has potential therapeutic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a5a8d74969f90d7980595b424382f99d6cd110" target='_blank'>
              Discovery of GJC1 as a prognostic biomarker in glioma cells: insights into its cell-cycle relationship and differential expression in non-neuronal cells
              </a>
            </td>
          <td>
            Xiangtian Ji, Xin Chen, Guozhong Lin, Kai-Li Ma, Jun Yang, Xiaofang Zhao, Su-hua Chen, Jun Yang
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Prostate cancer (PC) is a leading cause of death in men because of the high incidence and long-term inefficacy of the existing treatment options. Furthermore, it exhibits significant health disparities that affect African-American (AA) men more adversely than others do. Previously, we established CYP3A5, a highly expressed protein in AAs PC, as a positive regulator of androgen receptor (AR) signaling. We examined the impact of CYP3A5 depletion on genome-wide transcriptional output using RNA sequencing to gain deeper mechanistic insights. The data revealed that 561 genes were downregulated and 263 were upregulated upon silencing of CYP3A5 in PC cells. Furthermore, in silico pathway analyses of differentially expressed genes suggested that the cell cycle regulation pathway was most significantly affected by CYP3A5 inhibition. Cell cycle analysis of CYP3A5-silenced cells and those treated with clobetasol, a specific CYP3A5 pharmacological inhibitor, showed G1/S phase blockade. Both CYP3A5-depletion and pharmacological inhibition resulted in the downregulation of cyclin D, cyclin B, and CDK2, along with the upregulation of p27kip1 but had minimal effects on CDK4/6 levels. Combination treatment with clobetasol and the CDK4/6 inhibitor palbociclib exhibited synergy with combination index (CI) values ranging from 0.28-0.78. Our findings support the utility of CYP3A5 as a druggable therapeutic target that works more effectively in combination with CDK4/6 inhibition to limit the progression of PC, especially for AA patients with AA. This combination addresses CDK4/6 inhibitor resistance, which is often linked to CDK2 overexpression, and can potentially be useful in reducing disparities in the clinical outcomes of PC. Significance Our study highlights CYP3A5 as a key regulator of the cell cycle in prostate cancer (PC). Its overexpression in African American (AA) patients may be a key molecular driver of disparities in outcomes. The combination of CYP3A5 and CDK4/6 inhibitors shows a synergistic effect on therapeutic outcomes and addresses CDK2-mediated resistance. Thus, targeting both CYP3A5 and CDK4/6 could improve treatment outcomes, especially in AA PC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04bad07ffad0a8a7cbaae50a2fcf79091b02b95e" target='_blank'>
              CYP3A5 inhibition causes G1/S blockade and synergizes with CDK4/6 inhibitor to suppress prostate cancer cell growth: Implications in reducing health disparity
              </a>
            </td>
          <td>
            Jeetesh Sharma, Imran K. Mohammed, R. Tillett, Jake McLean, Shirley Shen, Ajay P. Singh, Oscar B. Goodman, Edwin C. Oh, R. Mitra
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Tumorigenesis is commonly attributed to Darwinian processes involving natural selection among cells and groups of cells. However, progressing tumors are those that also achieve an appropriate group phenotypic composition (GPC). Yet, the selective processes acting on tumor GPCs are distinct from that associated with classical Darwinian evolution (i.e. natural selection based on differential reproductive success) as tumors are not genuine evolutionary individuals and do not exhibit heritable variation in fitness. This complex evolutionary scenario is analogous to the recently proposed concept of ‘selection for function’ invoked for the evolution of both living and non-living systems. Therefore, we argue that it is inaccurate to assert that Darwinian processes alone account for all the aspects characterizing tumorigenesis and cancer progression; rather, by producing the genetic and phenotypic diversity required for creating novel GPCs, these processes fuel the evolutionary success of tumors that is dependent on selection for function at the tumor level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed9ff2b803aab7c7484cdca4ce05128b6e07445" target='_blank'>
              A new perspective on tumor progression
              </a>
            </td>
          <td>
            Frédéric Thomas, James DeGregori, A. Marusyk, Antoine M. Dujon, Beata Ujvari, Jean-Pascal Capp, Robert Gatenby, A. Nedelcu
          </td>
          <td>2024-09-19</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a839b7ca2ef04f3b61acc9f2cdd2aeccc65b7e8c" target='_blank'>
              Cells and signals of the leukemic microenvironment that support progression of T-cell acute lymphoblastic leukemia (T-ALL).
              </a>
            </td>
          <td>
            Aram Lyu, Seo Hee Nam, Ryan S. Humphrey, T. Horton, Lauren I R Ehrlich
          </td>
          <td>2024-11-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd905b03493f5b109589f0899bdd4480c79c0580" target='_blank'>
              Targeting mutant p53: a key player in breast cancer pathogenesis and beyond
              </a>
            </td>
          <td>
            Hina Qayoom, Burhan Ul Haq, Shazia Sofi, Nusrat Jan, Asma Jan, M. A. Mir
          </td>
          <td>2024-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Parallel single-cell multimodal sequencing is the most intuitive and precise tool for cellular status research. In this study, we propose AMAR-seq to automate methylation, chromatin accessibility, and RNA expression coanalysis with single-cell precision. We validated the accuracy and robustness of AMAR-seq in comparison with standard single-omics methods. The high gene detection rate and genome coverage of AMAR-seq enabled us to establish a genome-wide gene expression regulatory atlas and triple-omics landscape with single base resolution and implement single-cell copy number variation analysis. Applying AMAR-seq to investigate the process of mouse embryonic stem cell differentiation, we revealed the dynamic coupling of the epigenome and transcriptome, which may contribute to unraveling the molecular mechanisms of early embryonic development. Collectively, we propose AMAR-seq for the in-depth and accurate establishment of single-cell multiomics regulatory patterns in a cost-effective, efficient, and automated manner, paving the way for insightful dissection of complex life processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa2f0b9dd274ee3a6047b970e14792c843b3fd0" target='_blank'>
              AMAR-seq: Automated Multimodal Sequencing of DNA Methylation, Chromatin Accessibility, and RNA Expression with Single-Cell Resolution.
              </a>
            </td>
          <td>
            Xi Zeng, Xiaoping Yang, Zhixing Zhong, Xinrui Lin, Qiuyue Chen, Shaowei Jiang, Mengwu Mo, Shichao Lin, Huimin Zhang, Zhi Zhu, Jin Li, Jia Song, Chao Yang
          </td>
          <td>2024-09-09</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Therapy-related myeloid neoplasm (t-MN), characterized by its association with prior exposure to cytotoxic therapy, remains poorly understood and is a major impediment to long-term survival even in the era of novel targeted therapies due to its aggressive nature and treatment resistance. Previously, cytotoxic therapy–induced genomic changes in hematopoietic stem cells were considered sine qua non in pathogenesis; however, recent research demonstrates a complex interaction between acquired and hereditary genetic predispositions, along with a profoundly senescent bone marrow (BM) microenvironment. We review emerging data on t-MN risk factors and explore the intricate interplay among clonal hematopoiesis, genetic predisposition, and the abnormal BM microenvironment. Significance: t-MN represents a poorly understood blood cancer with extremely poor survival and no effective therapies. We provide a comprehensive review of recent preclinical research highlighting complex interaction among emerging therapies, hereditary and acquired genetic factors, and BM microenvironment. Understanding the risk factors associated with t-MN is crucial for clinicians, molecular pathologists, and cancer biologists to anticipate and potentially reduce its incidence in the future. Moreover, better understanding of the molecular pathogenesis of t-MN may enable preemptive screening and even intervention in high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a381c438c0051da067518ff9a38cd40bb1928975" target='_blank'>
              Therapy-Related Myeloid Neoplasms: Complex Interactions among Cytotoxic Therapies, Genetic Factors, and Aberrant Microenvironment
              </a>
            </td>
          <td>
            D. Singhal, M. Kutyna, Christopher C Hahn, M. Shah, D. Hiwase
          </td>
          <td>2024-10-18</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d7559338158829d62b6b4147b9bfe3e99b724d9" target='_blank'>
              HLA-G neo-expression modifies genetic programs governing tumor cell lines
              </a>
            </td>
          <td>
            D. Tronik-Le Roux, M. Daouya, I. Poras, François Desgrandchamps, E. Carosella
          </td>
          <td>2024-10-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Backgrounds: Assessing homologous recombination deficiency (HRD) has been recommended by clinical guidelines for patients with ovarian cancer (OC) as it predicts sensitivity to poly (ADP-ribose) polymerase inhibitors (PARPi). However, HRD testing is complex and either inaccessible or unaffordable for majority of OC patients in developing countries. Consequently, the prevalence of HRD in OC remains unknown. Methods: We examined HRD status of 77 Vietnamese patients with OC using a new laboratory-developed test (HRD Insight, Gene Solutions). Tumor DNA was extracted from FFPE samples, followed by next-generation sequencing to detect deleterious or suspected deleterious variants in BRCA1/2 genes. Shallow whole genome sequencing was performed to determine the whole Genomic Instability (wGI) score by assessing the presence of large-scale intra-chromosomal copy number alterations. Results: The assay was first benchmarked against commercial HRD kits including TruSight Oncology 500 HRD (Illumina), SOPHiA DDM HRD Solutions (Sophia Genetics) and HRD Focus Panel (AmoyDx), and showed an overall percent agreement of 90.0%, 96.3%, and 96.4% respectively. The successful rate of sequencing was 94.8% (73/77) and the prevalence of HRD in OC patients was 54.8% (40/73). BRCA mutations and positive wGI scores were found in 16.4% (12/73) and 47.9% (35/73) of the patients respectively. Among those with wild-type BRCA1/2, 40.5% of them had positive wGI scores and hence positive HRD. Age at diagnosis was not affected by both BRCA and wGI status. Conclusions: HRD Insight assay could accurately and robustly determine the HRD status of ovarian tissue samples, including those with low quality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a155429dcb3b51c5b55a82e27e1ae34e4f488fb" target='_blank'>
              Combination of BRCA deep targeted sequencing and shallow whole genome sequencing to detect homologous recombination deficiency in ovarian cancer
              </a>
            </td>
          <td>
            Thien-Phuc Hoang Nguyen, Nam H. B. Tran, T. Nguyen, My TT Ngo, Anh Duong Doan, D. Nguyen, Hung-Sang Tang, Duy Nguyen, Cam Tu Nguyen, T. T. Do, Hoai-Nghia Nguyen, H. Giang, Lan N Tu
          </td>
          <td>2024-09-10</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d175f8dd810d13a4f1256bdc9f679caa427edf2" target='_blank'>
              Systematic screening of protein-coding gene expression identified VWF as a potential key regulator in anthracycline-based chemotherapy-exacerbated metastasis of breast cancer
              </a>
            </td>
          <td>
            Yawei Zhao, Meihui He, Lianzhi Cui, Min Zhang, Tianyu Zhao, Xuehan Yang, Yang Xu, Jianhua Dong, Kan He, Hansi Zhang, Li Chen
          </td>
          <td>2024-10-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Stem cells have the potential to replace defective cells in several human diseases by depending on their self-renewal and differentiation capacities that are controlled by genes. Currently, exploring the regulation mechanism for stem cell capacities from the perspective of methyltransferase-like 3 (METTL3)-mediated N6-methyladenosine modification has obtained great advance, which functions by regulating target genes post-transcriptionally. However, reviews that interpret the regulatory network of METTL3 in stem cells are still lacking. In this review, we systematically analyze the available publications that report the role and mechanisms of METTL3 in stem cells, including embryonic stem cells, pluripotent stem cells, mesenchymal stem cells, and cancer stem cells. The analysis of such publications suggests that METTL3 controls stem cell fates and is indispensable for maintaining its normal capacities. However, its dysfunction induces various pathologies, particularly cancers. To sum up, this review suggests METTL3 as a key regulator for stem cell capacities, with further exploration potential in translational and clinical fields. In conclusion, this review promotes the understanding of how METTL3 functions in stem cells, which provides a valuable reference for further fundamental studies and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ed73aafb376e2732c137150f513994ca26c14b" target='_blank'>
              Regulatory Network of Methyltransferase-Like 3 in Stem Cells: Mechanisms and Medical Implications
              </a>
            </td>
          <td>
            Yan Zeng, Fengyang Wang, Silu Li, Bin Song
          </td>
          <td>2024-01-01</td>
          <td>Cell Transplantation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235fec0818688f0c80daa0063ad7c555bc1856b" target='_blank'>
              Hypoxia-dependent recruitment of error-prone DNA polymerases to genome replication.
              </a>
            </td>
          <td>
            Ran Yehuda, Ido Dromi, Y. Levin, Thomas Carell, Nicholas Geacintov, Z. Livneh
          </td>
          <td>2024-10-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) and Blast Crisis Chronic Myeloid Leukemia (BC CML) share several clinical characteristics, including the accumulation of immature myeloid cells and disrupted hematopoiesis. Despite the similarities, the two diseases exhibit different genetic profiles, contributing to distinct disease progressions and therapeutic outcomes. In this study, we perform a comparative analysis using bioinformatics and systems biology approaches to explore shared and unique mutational signatures, transcriptional networks, and molecular regulators in AML and BC CML. Our analysis identifies 29 genes common to both diseases, which are involved in key biological processes such as cell proliferation, cell death regulation, and transcriptional control. A subsequent pathway enrichment highlights common pathways, including the PI3K-AKT signaling and cytokine-mediated signaling, that may contribute to disease progression and resistance to therapy. Additionally, we construct a protein-protein interaction network and a transcription factor-miRNA coregulatory network, revealing key hub genes such as TP53, RUNX1, and BCL2. These genes could be potential shared therapeutic targets for treating AML and BC CML. Therefore, our analysis offers insights into the molecular underpinnings of these aggressive hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1aff1a0450bf81f4feefcd2ff55f60cc322c2c" target='_blank'>
              Shared Genetic and Regulatory Mechanisms in Acute Myeloid Leukaemia and Blast Crisis Chronic Myeloid Leukaemia
              </a>
            </td>
          <td>
            Sulagna Basu, Rakesh Pandey
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3305dbdb10abe125333dc69b5540a59698a94ae3" target='_blank'>
              Signaling pathways involved in colorectal cancer: pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Qing Li, Shan Geng, Hao Luo, Wei Wang, Ya-Qi Mo, Qing Luo, Lu Wang, Guan-Bin Song, Jian-Peng Sheng, Bo Xu
          </td>
          <td>2024-10-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most phenotype-associated genetic variants map to non-coding regulatory regions of the human genome. Moreover, variants associated with blood cell phenotypes are enriched in regulatory regions active during hematopoiesis. To systematically explore the nature of these regions, we developed a highly efficient strategy, Perturb-multiome, that makes it possible to simultaneously profile both chromatin accessibility and gene expression in single cells with CRISPR-mediated perturbation of a range of master transcription factors (TFs). This approach allowed us to examine the connection between TFs, accessible regions, and gene expression across the genome throughout hematopoietic differentiation. We discovered that variants within the TF-sensitive accessible chromatin regions, while representing less than 0.3% of the genome, show a ∼100-fold enrichment in heritability across certain blood cell phenotypes; this enrichment is strikingly higher than for other accessible chromatin regions. Our approach facilitates large-scale mechanistic understanding of phenotype-associated genetic variants by connecting key cis-regulatory elements and their target genes within gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd75703a3e0196006c63fecef421b6d1dc25cee5" target='_blank'>
              Transcription factor networks disproportionately enrich for heritability of blood cell phenotypes
              </a>
            </td>
          <td>
            Jorge D. Martin-Rufino, Alexis Caulier, Seayoung Lee, Nicole Castano, Emily King, Samantha Joubran, Marcus Jones, Seth R Goldman, U. Arora, Lara Wahlster, Eric S. Lander, V. Sankaran
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Transposable elements (TEs) are key contributors to genetic novelty. Despite increasing evidence of their importance, their roles in shaping the regulatory landscape of diverse immune cell populations remain largely unclear. Using single-cell multiome data from human peripheral blood mononuclear cells, we annotated the cell-specific cis-regulatory elements for major immune cell populations and identified a highly cell-specific signature of the overrepresented TE families. Focusing on monocytes that bear fast-evolving transcriptomes, we found that high proportions of their enhancers are TE-derived and bound by multiple pioneer transcription factors. Among them, we confirmed that the core myeloid regulator SPI1 can bind and regulate hundreds of TE-derived enhancers, which further affect the expression of adjacent immune genes. Additionally, interspecies comparison reveals that non-conserved monocyte enhancers are frequently generated by lineage-specific TE insertions, and correlate with the evolved gene expression between human and mouse. Overall, our study supports the importance of TEs in shaping the regulatory landscape of diverse immune cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec49246bea325aa9a378a750ef1fb9175032369" target='_blank'>
              Unraveling transposable element-mediated regulatory landscape in diverse human immune cells
              </a>
            </td>
          <td>
            Cui-Qiong Du, Hairui Fan, Jiayao Jiang, Juan Yang, Shuai Chen, Nadezhda E. Vorobyeva, Congrui Zhu, Liming Mao, Chenxi Li, Yanhua Li, Wen-hua Bao, Ming-an Sun
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background While immune checkpoint inhibitors (ICIs) are adopted as standard therapy in non-small cell lung cancer (NSCLC) patients, factors that influence variable prognosis still remain elusive. Therefore, a deeper understanding is needed of how germline variants regulate the transcriptomes of circulating immune cells in metastasis, and ultimately influence immunotherapy outcomes. Methods We collected peripheral blood mononuclear cells (PBMCs) from 73 ICI-treated NSCLC patients, conducted single-cell RNA sequencing, and called germline variants via SNP microarray. Determination of expression quantitative trait loci (eQTL) allows elucidating genetic interactions between germline variants and gene expression. Utilizing aggregation-based eQTL mapping and network analysis across eight blood cell types, we sought cell-type-specific and ICI-prognosis-dependent gene regulatory signatures. Results Our sc-eQTL analysis identified 3,616 blood- and 702 lung-cancer-specific eGenes across eight major clusters and treatment conditions, highlighting involvement of immune-related pathways. Network analysis revealed TBX21-EOMES regulons activity in CD8+ T cells and the enrichment of eQTLs in higher-centrality genes as predictive factors of ICI response. Conclusions Our findings suggest that in the circulating immune cells of NSCLC patients, transcriptomic regulation differs in a cell type- and treatment-specific manner. They further highlight the role of eQTL loci as broad controllers of ICI-prognosis-predicting gene networks. The predictive networks and identification of eQTL contributions can lead to deeper understanding and personalized ICI therapy response prediction based on germline variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70aa2fedf9166d7283b9f0e6d961d358ca399349" target='_blank'>
              Systemic CD8+ T cell effector signature predicts prognosis of lung cancer immunotherapy
              </a>
            </td>
          <td>
            Hyungtai Sim, Geun-Ho Park, Woong-Yang Park, Se-Hoon Lee, Murim Choi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction. The TP53 gene encodes the p53 protein, crucial for DNA damage response, apoptosis, and cell cycle regulation. In chronic lymphocytic leukemia (CLL), TP53 loss due to 17p deletion or mutation leads to poor chemoimmunotherapy response and shorter survival. 
Patients with unmutated immunoglobulin heavy chain variable (IGHV) status exhibit a more aggressive disease course, poorer outcomes, and reduced response to standard chemoimmunotherapy. These genetic variations significantly affect disease progression, treatment choice, and response to therapy. 
Case Report. In November 2023, a 61-year-old man presented to our clinic with complaints of generalized weakness and rapid fatigue since October 2023. CLL was diagnosed in 2018 through pathohistological and immunohistochemical analysis of the bone marrow. The patient received no therapy until 2023 due to the absence of active disease signs. From March 28, 2023, to July 25, 2023, the patient received chemoimmunotherapy with the bendamustine and rituximab (BR) regimen. Five courses of therapy were administered; however, further treatment was discontinued due to side effects that occurred during the fifth course of bendamustine administration. A subsequent diagnostic evaluation aimed at assessing treatment efficacy, prognosis, and genetic profiling included a cytomorphological analysis, which revealed that lymphocytes comprised 80% of the cellular composition. Immunocytological analysis confirmed the presence of monoclonal atypical lymphocytes, accounting for 70% of the cell population. Next-generation sequencing identified a pathogenic TP53 mutation, and fragment analysis confirmed an unmutated IGHV status. The presence of the TP53 mutation and unmutated IGHV status categorizes the patient as being in the very high-risk group, for which the use of Bruton’s tyrosine kinase inhibitors (BTKIs) and/or B-cell lymphoma-2 (BCL-2) inhibitors is recommended. 
Conclusions. This case highlights the significance of comprehensive genetic profiling in CLL patients using techniques such as fluorescence in situ hybridization, polymerase chain reaction, and next-generation sequencing. Such profiling detects molecular genetic alterations, facilitating personalized and effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9a770a1a98466f2dfc1c687a0f49ae58b9afa5" target='_blank'>
              Genetic Profiling in Chronic Lymphocytic Leukemia: The Role of TP53 Mutations and IGHV Status in Treatment Decision-Making – A Case Report
              </a>
            </td>
          <td>
            Kateryna Khurdepa, Oksana Karnabeda
          </td>
          <td>2024-10-16</td>
          <td>Galician Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The complexity of cancer requires a comprehensive approach to understand its diverse manifestations and underlying mechanisms. Initially outlined by Hanahan and Weinberg in 2000 and updated in 2010, the hallmarks of cancer provide a conceptual basis for understanding inherent variability in cancer biology. Recent expansions have further elucidated additional hallmarks, including phenotypic plasticity and senescent cells. The International Agency for Research on Cancer (IARC) has identified the key characteristics of carcinogens (KCCs) to evaluate their carcinogenic potential. We analyzed chemicals of concern for environmental exposure that interact with specific receptors to induce genomic instability, epigenetic alterations, immune suppression, and receptor-mediated effects, thereby contributing to chronic inflammation. Despite their varying degrees of carcinogenicity, these chemicals have similar KCC profiles. Our analysis highlights the pivotal role of receptor binding in activating most other KCCs, underscoring their significance in cancer initiation. Although KCCs are associated with early molecular or cellular events, they do not encompass processes directly linked to full cellular malignancy. Thus, there is a need to integrate clear endpoints that anchor KCCs to the acquisition of a complete malignant phenotype into chemical testing. From the perspective of toxicology and cancer research, an all-encompassing strategy that incorporates both existing and novel KCCs and cancer hallmarks is essential to enable the targeted identification of prevalent carcinogens and facilitate zone-specific prevention strategies. To achieve this goal, collaboration between the KCC and cancer hallmarks communities becomes essential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a462cc73a8d245ed18a298dc440b46e953accd9" target='_blank'>
              Key characteristics of carcinogens meet hallmarks for prevention-cutting the Gordian knot
              </a>
            </td>
          <td>
            S. Senga, William H. Bisson, Annamaria Colacci
          </td>
          <td>2024-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e419ef20d77c35c04caa3dddc4e91b5a8579b30f" target='_blank'>
              Recurrent somatic mutations of FAT family cadherins induce an aggressive phenotype and poor prognosis in anaplastic large cell lymphoma.
              </a>
            </td>
          <td>
            M. Villa, Geeta G Sharma, Federica Malighetti, M. Mauri, Giulia Arosio, N. Cordani, C. Lobello, Hugo Larose, A. Pirola, D. D’Aliberti, L. Massimino, Lucrezia Criscuolo, Lisa Pagani, C. Chinello, Cristina Mastini, Diletta Fontana, S. Bombelli, Raffaella Meneveri, F. Lovisa, L. Mussolin, A. Janíková, Š. Pospíšilová, S. Turner, G. Inghirami, Fulvio Magni, Mario Urso, Fabio Pagni, D. Ramazzotti, R. Piazza, Roberto Chiarle, C. Gambacorti-Passerini, Luca Mologni
          </td>
          <td>2024-10-30</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Base editing shows promise for the correction of human mutations at a higher efficiency than other repair methods and is especially attractive for mutations in large genes that are not amenable to gene augmentation therapy. Here, we demonstrate a comprehensive workflow for in vitro screening of potential therapeutic base editing targets for the USH2A gene and empirically validate the efficiency of adenine and cytosine base editor/guide combinations for correcting 35 USH2A mutations. Editing efficiency and bystander edits are compared between different target templates (plasmids versus transgenes) and assays (Next generation sequencing versus Sanger), as well as comparisons between unbiased empirical results and computational predictions. Based on these observations, practical assay recommendations are discussed. Finally, a humanized knock-in mouse model was created with the best-performing target, the nonsense mutation c.11864G>A p.(Trp3955*). Split-intein AAV9 delivery of editing reagents resulted in the restoration of USH2A protein and a correction rate of 65 ± 3% at the mutant base pair and of 52 ± 3% excluding bystander amino acid changes. This efficiency compares favorably to a prior genome editing strategy tested in the retina that completed a clinical trial and demonstrates the effectiveness of this overall strategy to identify and test base editing reagents with the potential for human therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b11e55d59e66e72cd37b7653ddc5c10089e197" target='_blank'>
              Systematic empirical evaluation of individual base editing targets: validating therapeutic targets in USH2A and comparison of methods
              </a>
            </td>
          <td>
            Yuki Tachida, K. Manian, Rossano Butcher, Jonathan M. Levy, Nachiket C. Pendse, E. Hennessey, David R. Liu, Eric A. Pierce, Qin Liu, J. Comander
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epigenetic dysregulation has long been recognized as a significant contributor to tumorigenesis and tumor maintenance, impacting all recognized cancer hallmarks. Although some epigenetic drugs have received regulatory approval for certain hematological malignancies, their efficacy in treating solid tumors has so far been largely disappointing. However, recent advancements in developing new compounds and a deeper understanding of cancer biology have led to success in specific solid tumor subtypes through precision medicine approaches. Moreover, epigenetic drugs may play a crucial role in synergizing with other anticancer treatments, enhancing the sensitivity of cancer cells to various anticancer therapies, including chemotherapy, radiation therapy, hormone therapy, targeted therapy, and immunotherapy. In this review, we critically evaluate the evolution of epigenetic drugs, tracing their development from initial use as monotherapies to their current application in combination therapies. We explore the preclinical rationale, completed clinical studies, and ongoing clinical trials. Finally, we discuss trial design strategies and drug scheduling to optimize the development of possible combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/357590ca2c8c3ea804892d4f86c25aabfdeb5726" target='_blank'>
              Progresses and Pitfalls of Epigenetics in Solid Tumors Clinical Trials
              </a>
            </td>
          <td>
            Alice Rossi, F. Zacchi, Anna Reni, Michele Rota, Silvia Palmerio, Jessica Menis, Andrea Zivi, Stefano Milleri, Michele Milella
          </td>
          <td>2024-10-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumour suppressor factor p53 plays an essential role in regulating numerous cellular processes, including the cell cycle, DNA repair, apoptosis, autophagy, cell metabolism and immune response. TP53 is the most commonly mutated gene in human cancers. These mutations are primarily non‑synonymous changes that produce mutant p53 proteins characterized by loss of function, a dominant negative effect on p53 tetramerisation and gain of function (GOF). GOF mutations not only disrupt the tumour‑suppressive activities of p53 but also endow the mutant proteins with new oncogenic properties. Recent studies analysing different pathogenic features of mutant p53 in cancer‑derived cell lines have demonstrated that restoring wild‑type p53, rather than removing GOF mutations, reduces cancer cell growth. These findings suggest that therapeutic strategies for reactivating wild‑type p53 function in cancer cells may bring a greater benefit than approaches halting mutant p53. This approach could involve the use of small molecules, gene therapy and other methods to re‑establish wild‑type p53 activity. This review describes the complexity of the biological activities of different p53 mutants and summarizes the current therapeutic approaches to restore p53 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48833d89250389a497918ef135eacb5f7e0504d" target='_blank'>
              TP53 mutations in cancer: Molecular features and therapeutic opportunities (Review).
              </a>
            </td>
          <td>
            M. Tornesello
          </td>
          <td>2024-10-24</td>
          <td>International journal of molecular medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a258e30c79cfe98cd4c2597aa4e57162f07486" target='_blank'>
              Emerging CRISPR Therapies for Precision Gene Editing and Modulation in the Cardiovascular Clinic.
              </a>
            </td>
          <td>
            Nicholas J Legere, J. T. Hinson
          </td>
          <td>2024-09-17</td>
          <td>Current cardiology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome 1q copy number variations, collectively termed +1q, are 1 of the most common cytogenetic abnormalities in multiple myeloma. 1q abnormalities are associated with overexpression of a high-risk gene signature promoting cell proliferation, apoptosis resistance, genomic instability, and treatment resistance, and acquisition or expansion of +1q subclones mediate disease development and relapse. While there remains significant controversy as to whether the presence of +1q is itself an independent driver of poor prognosis or is simply a marker of other high-risk features, +1q has recently been incorporated into multiple prognostic scoring models as a new high-risk cytogenetic abnormality. In this review, we present possible underlying genetic mechanisms of high-risk disease in +1q myeloma, implications for subclonal development, its role in modifying the tumor microenvironment, current evidence for clinical significance in newly-diagnosed and relapsed patients, and current controversies in +1q classification and prognostication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d04b16ebbddbed7bed25e1883e91238060b83" target='_blank'>
              The role of 1q abnormalities in multiple myeloma: Genomic insights, clinical implications, and therapeutic challenges.
              </a>
            </td>
          <td>
            Zachary M. Avigan, C. Mitsiades, A. Laganà
          </td>
          <td>2024-10-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 High-grade gliomas (HGGs) are the most aggressive type of gliomas and have the poorest outcomes. Chromatin remodeling (CR) genes have been implicated in multiple oncogenic pathways in numerous cancer types. In gliomagenesis, CR genes have been implicated in regulating stemness of glioma cells, the tumor microenvironment (TME), and resistance to therapies.



 We performed molecular profiling of 4244 HGGs and evaluated associations of CR mutations with other cancer related biomarkers, infiltration by immune cells, and immune gene expression. We also evaluated the association between CR mutations and survival in IDH WT HGG patients.



 Nearly 10% of HGGs carry mutations in CR genes, with higher prevalence (15%) in HGGs with IDH mutations. Analysis of co-occurrence with other biomarkers revealed that CR-mutated HGGs possess favorable genetic alterations which may have prognostic value. CR-mutated HGGs with wild type IDH demonstrated colder TME and worse OS overall compared to the CR-wild type HGGs.



 Our study reveals the prognostic effects of CR mutations in HGG and points to several biomarker candidates that could suggest sensitivity to emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a6ec214f04da421e41c3579984b7e7f5d8d94d" target='_blank'>
              Biological and prognostic relevance of epigenetic regulatory genes in high-grade gliomas (HGGs)
              </a>
            </td>
          <td>
            Sonikpreet Aulakh, Joanne Xiu, Andrew Hinton, S. Darabi, M. Demeure, S. Sengupta, Santosh Kesari, David M Ashley, Ashley Love Sumrall, M. Glantz, D. Spetzler
          </td>
          <td>2024-10-04</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background: Genetic studies of ovarian cancer (OC) have historically focused on BRCA1/2 mutations, lacking other studies of homologous recombination repair (HRR). Poly (ADP-ribose) polymerase inhibitors (PARPi) exploit synthetic lethality to significantly improve OC treatment outcomes, especially in BRCA1/2 deficiency patients. Objectives: Our study aims to construct a mutation map of HRR genes in OC and identify factors influencing the efficacy of PARPi. Design: A retrospective observational analysis of HRR gene variation data from 695 OC patients from March 2019 to February 2022 was performed. Methods: The HRR gene variation data of 695 OC patients who underwent next-generation sequencing (NGS) in the First Affiliated Hospital of Zhengzhou University were retrospectively collected. Clinical data on the use of PARPi in these patients were also gathered to identify factors that may interfere with the efficacy of PARPi. Results: Out of 127 pathogenic variants in the BRCA1/2 genes, 104 (81.9%) were BRCA1 mutations, and 23 (18.1%) were BRCA2 mutations. Among the 59 variants of uncertain significance (VUS), 20 (33.9%) were BRCA1, while 39 (66.1%) were BRCA2 mutations. In addition to BRCA1/2, HRR gene results showed that 9 (69%) of 13 were HRR pathway pathogenic variants; and 16 (1.7%) of 116 VUS were Food and Drug Administration (FDA)-approved mutated HRR genes. Notably, the treatment regimen significantly influenced the effectiveness of PARPi, especially when using first-line maintenance therapy, leading to enhanced progression-free survival (PFS) compared to alternative protocols. Conclusion: Focusing on HRR gene mutations and supporting clinical research about PARPi in OC patients is crucial for developing precision treatment strategies and enhancing prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d41e2645926ac18cf48bd055b15a6838f0eb8b2d" target='_blank'>
              Genetic landscape of homologous recombination repair and practical outcomes of PARPi therapy in ovarian cancer management
              </a>
            </td>
          <td>
            Mengyu Chen, Ningjing Lei, Ruixia Guo, Liping Han, Qinghe Zhao, Yang Zhao, Luojie Qiu, Fengling Wu, Shan Jiang, Ningyao Tong, Kunmei Wang, Siyu Li, Lei Chang
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Hepatocellular adenomas (HAs) are tumors that can develop under different conditions, including in patients harboring a germline mutation in HNF1A. However, little is known about the pathogenesis of such disease. This work aims to better define what mechanisms lie under the development of this condition. Six HAs were sampled from the liver of a 17-year-old male affected by diabetes and multiple hepatic adenomatosis harboring the heterozygous pathogenic germline variant c.815G>A, p.(Arg272His) in HNF1A, which has a dominant negative effect. All HAs were molecularly characterized. Four of them were shown to harbor a second somatic HNF1A variant and one had a mutation in the ARID1A gene, while no additional somatic changes were found in the remaining HA and normal parenchyma. A transcriptomic profile of the same HA samples was also performed. HNF1A biallelic mutations were associated with the up-regulation of several pathways including the tricarboxylic acid cycle, the metabolism of fatty acids, and mTOR signaling while angiogenesis, endothelial and vascular proliferation, cell migration/adhesion, and immune response were down-regulated. Contrariwise, in the tumor harboring the ARID1A variant, angiogenesis was up-modulated while fatty acid metabolism was down-modulated. Histological analyses confirmed the molecular data. Independently of the second mutation, energetic processes and cholesterol metabolism were up-modulated, while the immune response was down-modulated. This work provides a complete molecular signature of HNF1A-associated HAs, analyzing the association between specific HNF1A variants and the development of HA while identifying potential new therapeutic targets for non-surgical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26474672d542f89d0fa38cb2c1556ad93882621f" target='_blank'>
              Genomic and Transcriptomic Profile of HNF1A-Mutated Liver Adenomas Highlights Molecular Signature and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            A. Faini, F. Arruga, M. Pinon, Valeria Bracciamà, Francesco Edoardo Vallone, Fiorenza Mioli, M. Sorbini, Martina Migliorero, A. Gambella, Damiano Carota, Isaac Giraudo, P. Cassoni, Silvia Catalano, Renato Romagnoli, Antonio Amoroso, P. L. Calvo, T. Vaisitti, Silvia Deaglio
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c590d34aa4895f063579c9cf74d2f705da09c384" target='_blank'>
              Developmental interplay between transcriptional alterations and a targetable cytokine signaling dependency in pediatric ETO2::GLIS2 leukemia
              </a>
            </td>
          <td>
            Verónica Alonso-Pérez, Klaudia Galant, Fabien Boudia, E. Robert, Zakia Aid, Laurent Renou, V. Barroca, Saryiami Devanand, Loélia Babin, Virginie Rouiller-Fabre, Delphine Moison, Didier Busso, Guillaume Piton, Christophe Metereau, N. Abermil, Paola Ballerini, Pierre Hirsch, Rima Haddad, Jelena Martinovic, Arnaud Petit, H. Lapillonne, Erika Brunet, T. Mercher, Françoise Pflumio
          </td>
          <td>2024-09-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Genome editing technology is an emerging toolbox that can specifically target genes. At present, the existing gene editing technologies include ZFN, TALEN, and CRISPR system editing technology with clusters of regularly spaced short palindromic repeats. However, ZFN and TALEN are no longer the preferred tools for gene editing due to some of their problems, and their complex protein design, high cost, and high difficulty are the main reasons that limit their use. The CRISPR system, on the other hand, is able to achieve specific binding by base complementary pairing of sgRNA to the target sequence. Traditional cancer treatment methods have limited efficacy and high recurrence rates, and the disease can be fundamentally treated by inactivating oncogenes or activating tumor suppressor genes through gene editing technology and then changing downstream signaling pathways for cancer treatment. With CRISPR-Cas9 technology, it is possible to have gene knockouts, insertions, and mutations. Here, we will investigate and explore the feasibility of the CRISPR system for tumor treatment, including the following three topics: oncogene knockout, augmentation of tumor suppressor gene expression, and enhancement of engineered T cell viability for effective tumor Through these methods, the CRISPR system is of great help and promising tumor therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa5e9e8d18667744c4c9905968635b74973ccacc" target='_blank'>
              Advances in CRISPR-Cas9 Technology for Tumor Therapy
              </a>
            </td>
          <td>
            Haochuan Xu
          </td>
          <td>2024-10-29</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to therapy remains a significant challenge in cancer treatment, often due to the presence of a stem-like cell population that drives tumor recurrence post-treatment. Moreover, many anticancer drugs inadvertently induce plasticity, reverting differentiated cancer cells to drug-resistant stem-like states. Addressing this phenomenon is crucial for advancing cancer therapeutics. In this study, we introduce a robust statistical framework based on multi-type branching processes that can dissect tumor dynamics and drug effects from high throughput drug screening data. Through comprehensive in silico experiments, we show the efficacy of our framework in estimating parameters governing population dynamics and drug responses in a heterogeneous tumor population where cell state transitions are influenced by the drug. Finally, using recent in vitro data involving AGS-SORE6+/- cells treated with ciclopirox olamine, we show how our framework can be used to confirm the presence of drug-induced cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85013788a6ef1ec3c633fabb86b8c0544b0624bd" target='_blank'>
              Inferring Drug-induced Plasticity via Drug Screen Data
              </a>
            </td>
          <td>
            Chenyu Wu, E. B. Gunnarsson, J. Foo, K. Leder
          </td>
          <td>2024-09-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Anti-cancer therapies can induce cellular senescence, which is highly stable, or drug-tolerant persistence, which is efficiently reversed upon therapy termination. While approaches to target senescent cells have been extensively studied, further understanding of the processes regulating persistence is needed to develop treatment strategies to suppress persister cell survival. Here, we used mTOR/PI3K inhibition to develop and characterize a model of persistence-associated arrest in human cancer cells of various origins. Persister and senescent cancer cells shared an expanded lysosomal compartment and hypersensitivity to BCL-XL inhibition. However, persister cells lacked other features of senescence, such as loss of lamin B1, senescence-associated β-galactosidase activity, upregulation of MHC-I, and an inflammatory and secretory phenotype (SASP). Genome-wide CRISPR/Cas9 screening for genes required for the survival of persister cells revealed that they are hypersensitive to the inhibition of one-carbon (1C) metabolism, which was validated by the pharmacological inhibition of SHMT, a key enzyme that feeds methyl groups from serine into 1C metabolism. Connecting 1C metabolism with the epigenetic regulation of transcription, the repressive heterochromatic mark H4K20me3 was enriched at the promoters of SASP and interferon response genes in persister cells, while it was absent in proliferative or senescent cells. Moreover, persister cells overexpressed the H4K20 methyltransferases KMT5B/C, and their downregulation unleashed inflammatory programs and compromised the survival of persister cells. In summary, this study defined distinctive features of persister cancer cells, identified actionable vulnerabilities, and provided mechanistic insight into their low inflammatory activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abdfb65f3c9de7ecccd1a62a48f1352298560afb" target='_blank'>
              H4K20me3-Mediated Repression of Inflammatory Genes is a Characteristic and Targetable Vulnerability of Persister Cancer Cells.
              </a>
            </td>
          <td>
            Valentina Ramponi, Laia Richart, M. Kovatcheva, Camille Stephan-Otto Attolini, Jordi Capellades, Alice E Lord, O. Yanes, Gabriella Ficz, M. Serrano
          </td>
          <td>2024-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a therapeutically challenging malignancy notorious for its resilience to therapeutic strategies and ability to thrive under extreme conditions. The cancer's inherent ability to adaptively resist treatments underscores the need for novel and combinatorial therapeutic approaches. Our research focuses on elucidating the role of Apurinic-apyrimidinic endonuclease/Redox effector factor 1 (APE1/Ref-1), a critical multifunctional protein in PDAC pathobiology. APE1 serves two essential functions: as the principal endonuclease in the base excision repair (BER) pathway for DNA repair and as a redox regulator of transcription factors essential for cancer cell proliferation. While the therapeutic targeting of APE1’s redox function has been effective and continues to yield significant advances, the development of treatments aimed at its DNA repair function has proven more challenging with comparatively limited progress. Since standard PDAC treatment regimens involve DNA damaging agents potentially repaired via BER and APE1, we engineered three mutant cell lines with knock-in mutations resulting in a reduction of APE1’s BER activity while maintaining full redox activity. Biochemical evaluations confirmed the effects of the mutation, revealing a 30-fold reduction in DNA repair capacity. Surprisingly, with respect to growth and proliferation, the mutant and wild-type cell lines behaved similarly. Analyses of cytotoxic vulnerabilities revealed similar findings with the mutant lines exhibiting minimal phenotypic differences in sensitivity when challenged with cytotoxic and DNA damaging agents in proliferation-based assays. These findings led us to posit that the mutant cell lines were relying on compensatory DNA repair pathways which were providing sufficient compensation for the decrease in APE1 activity. We further hypothesized that the cell lines may still have hidden, long-term deficiencies in DNA damage processing which are not apparent during the initial stress response. Subsequent colony formation assays revealed that the mutant cell lines do exhibit poor long-term survival even after only short-term exposure to DNA damage. In vivo orthotopic mouse model experiments further corroborated these findings, showing a significant reduction in tumor size and decreased metastasis to the lungs and liver in mutant cell lines compared to wild-type controls. While APE1 is the primary endonuclease in the BER pathway, studies have demonstrated APE1-independent BER in rare, specific contexts. We are probing the hypothesis that these APE1-independent BER pathways function as the compensatory mechanism safeguarding these cells in short-term stress situations. Identifying and characterizing these backup pathways will not only offer a better understanding of APE1’s DNA repair capabilities within PDAC but will also provide a better understanding of how DNA repair mechanisms can be therapeutically targeted for novel therapeutic options or combinations. This opens the path to enhance standard of care treatments such as FOLFIRINOX.
 Citation Format: Eyram K Kpenu, Randall Wireman, Mahmut Mijiti, Silpa Gampala, Melissa L Fishel, Mark R Kelley. Challenging PDAC adaptability: Uncovering therapeutic vulnerabilities in APE1 DNA repair mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45264b7cf12c7d27e58987210445d6c2b0d4261" target='_blank'>
              Abstract C028: Challenging PDAC adaptability: Uncovering therapeutic vulnerabilities in APE1 DNA repair mechanisms
              </a>
            </td>
          <td>
            E. Kpenu, R. Wireman, Mahmut Mijiti, S. Gampala, M. Fishel, Mark R. Kelley
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) poses significant challenges in veterinary medicine, with limited treatment options and poor survival rates. While substantial progress has been made in characterizing human AML, translating these advancements to veterinary practice has been hindered by limited molecular understanding and diagnostic tools. The case study presented illustrates the application of whole genome sequencing in diagnosing AML in a dog, showcasing its potential in veterinary oncology. Our approach facilitated comprehensive genomic analysis, identifying mutations in genes that may be associated with AML pathogenesis in dogs, such as KRAS, IKZF1, and RUNX1. However, without supportive evidence of its clinical utility (eg, association with response to treatment or prognosis), the information is limited to exploration. This article reviews the comparative features of canine AML with human AML and discusses strategies to shrink the knowledge gap between human and veterinary medicine with cost-effective next-generation sequencing (NGS) techniques. By utilizing these approaches, the unique and shared molecular features with human AML can be identified, aiding in molecular classification and therapeutic development for both species. Despite the promise of NGS, challenges exist in implementing it into routine veterinary diagnostics. Cost considerations, turnaround times, and the need for robust bioinformatics pipelines and quality control measures must be addressed. Most importantly, analytical and clinical validation processes are essential to ensure the reliability and clinical utility of NGS-based assays. Overall, integrating NGS technologies into veterinary oncology holds great potential for advancing our understanding of AML and improving disease stratification, in hopes of improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8bd6fdd9b3631dd5e67d1617f989917515a9291" target='_blank'>
              Advancements in genetic analysis: Insights from a case study and review of next-generation sequencing techniques for veterinary oncology applications.
              </a>
            </td>
          <td>
            R. Harris, Jillian Nolan, Dylan Ammons, Samantha Beeson, Douglas Thamm, Anne Avery
          </td>
          <td>2024-10-04</td>
          <td>Veterinary clinical pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BTK inhibitor therapy induces peripheral blood lymphocytosis in chronic lymphocytic leukemia (CLL) lasting for several months. It remains unclear whether non-genetic adaptation mechanisms exist, allowing CLL cells' survival during BTK inhibitor-induced lymphocytosis and/or playing a role in therapy resistance. We show that in approximately 70 % of CLL cases, ibrutinib treatment in vivo increases Akt activity above pre-therapy levels within several weeks, leading to compensatory CLL cell survival and a more prominent lymphocytosis on therapy. Ibrutinib-induced Akt phosphorylation (pAktS473) is caused by the upregulation of FoxO1 transcription factor, which induces expression of Rictor, an assembly protein for mTORC2 protein complex that directly phosphorylates Akt at serine 473 (S473). Knock-out or inhibition of FoxO1 or Rictor led to a dramatic decrease in Akt phosphorylation and growth disadvantage for malignant B cells in the presence of ibrutinib (or PI3K inhibitor idelalisib) in vitro and in vivo. FoxO1/Rictor/pAktS473 axis represents an early non-genetic adaptation to BCR inhibitor therapy not requiring PI3Kδ or BTK kinase activity. We further demonstrate that FoxO1 can be targeted therapeutically, and its inhibition induces CLL cells' apoptosis alone or in combination with BTK inhibitors (ibrutinib, acalabrutinib, pirtobrutinib) and blocks their proliferation triggered by T-cell factors (CD40L, IL-4, and IL-21).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9196e36f3efd04dcada6ed6f664575c51279c08" target='_blank'>
              FoxO1/Rictor axis induces a non-genetic adaptation to Ibrutinib via Akt activation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            L. Ondrisova, V. Seda, K. Hlavac, Petra Pavelkova, E. Hoferkova, G. Chiodin, L. Košťálová, Gabriela Mladonicka Pavlasova, D. Filip, Josef Vecera, P. F. Zeni, J. Oppelt, Z. Kahounová, Ráchel Víchová, Karel Souček, A. Panovska, K. Plevová, Š. Pospíšilová, M. Šimkovič, F. Vrbacký, Daniel Lysák, Stacey M. Fernandes, M. Davids, A. Maiques-Diaz, S. Charalampopoulou, J. Martín-Subero, Jennifer R. Brown, M. Doubek, Francesco Forconi, J. Mayer, M. Mraz
          </td>
          <td>2024-10-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b66450c0c679cd1bceb0db2087411b8321c6d3" target='_blank'>
              Evolution of translational control and the emergence of genes and open reading frames in human and non-human primate hearts
              </a>
            </td>
          <td>
            Jorge Ruiz-Orera, Duncan Miller, J. Greiner, Carolin Genehr, Aliki Grammatikaki, S. Blachut, Jeanne Mbebi, Giannino Patone, Anna Myronova, E. Adami, Nikita Dewani, Ning Liang, Oliver Hummel, Michael B Muecke, Thomas B Hildebrandt, Guido Fritsch, Lisa Schrade, Wolfram H Zimmermann, I. Kondova, Sebastian Diecke, Sebastiaan van Heesch, Norbert Hübner
          </td>
          <td>2024-09-24</td>
          <td>Nature Cardiovascular Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Froese, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel Goulet, Michael T. Hemann. Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81348949cfc84943625452ad98cf593484f76b30" target='_blank'>
              Abstract B039: Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Froese, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, Michael Hemann
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Several therapeutic agents have been approved for treating multiple myeloma (MM), a cancer of bone marrow resident plasma cells. Predictive biomarkers for drug response could help guide clinical strategies to optimize outcomes. Here, we present an integrated functional genomic analysis of tumor samples from MM patients that were assessed for their ex vivo drug sensitivity to 37 drugs, clinical variables, cytogenetics, mutational profiles, and transcriptomes. This analysis revealed a MM transcriptomic topology that generates "footprints" in association with ex vivo drug sensitivity that have both predictive and mechanistic applications. Validation of the transcriptomic footprints for the anti-CD38 monoclonal antibody daratumumab and the nuclear export inhibitor selinexor demonstrated that these footprints can accurately classify clinical responses. The analysis further revealed that daratumumab and selinexor have anti-correlated mechanisms of resistance, and treatment with a selinexor-based regimen immediately after a daratumumab-containing regimen was associated with improved survival in three independent clinical trials, supporting an evolutionary-based strategy involving sequential therapy. These findings suggest that this unique repository and computational framework can be leveraged to inform underlying biology and to identify therapeutic strategies to improve treatment of MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b324ac9181f2db70d574620d1a3e7ec5783d8859" target='_blank'>
              The Functional Transcriptomic Landscape Informs Therapeutic Strategies in Multiple Myeloma.
              </a>
            </td>
          <td>
            P. Sudalagunta, R. Canevarolo, M. Meads, Maria Silva, Xiaohong Zhao, Christopher L Cubitt, S. Sansil, Gabriel Deavila, R. Alugubelli, Ryan T Bishop, Alexandre Tungesvik, Qi Zhang, Oliver Hampton, J. Teer, Eric A Welsh, Sean J Yoder, Bijal D Shah, Lori Hazlehurst, Robert A Gatenby, Dane R. Van Domelen, Yi Chai, Feng Wang, Andrew DeCastro, Amanda M Bloomer, E. Siegel, Conor C Lynch, Daniel M Sullivan, Melissa Alsina, T. Nishihori, J. Brayer, John L Cleveland, William S. Dalton, C. Walker, Y. Landesman, Rachid Baz, A. Silva, K. Shain
          </td>
          <td>2024-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, J. Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While anticancer drug discovery has seen dramatic innovations and successes, sequential single therapies are time-limited by resistance, and combinatorial strategies have been lagging. The number of possible drug combinations is vast. To select drug combinations the oncologist requires knowledge of the optimal combination of proteins to co-target. Currently, combinations that the oncologist considers are primarily from empirical observations and clinical praxis. Our aim is to develop a signaling-based method to discover optimal proteins for the oncologist to co-target with drug combinations, and test it on available, patient-derived data. To temper the expected resistance to single drug regimen, we offer a concept-based stratified pipeline aimed at selecting co-targets for drug combinations. Our strategy is unique in its co-target selection being based on signaling pathways. This is significant since in cancer, drug resistance commonly bypasses blocked proteins by wielding alternative, or complementary, routes to execute cell proliferation. Our network-informed signaling-based approach harnesses advanced network concepts and metrics, and our compiled, tissue-specific co-existing mutations. Co-existing driver mutations are common in resistance. Thus, to mimic cancer and counter drug resistance scenarios, our pipeline seeks co-targets that when targeted by drug combinations, can shut off cancer’s modus operandi. That is, its parallel or complementary signaling pathways would be blocked. Rotating through combinations could further lessen emerging resistance. We applied it to patient-derived breast and colorectal ESR1|PIK3CA and BRAF|PIK3CA subnetworks. Consistently, in breast cancer, our results suggest co-targeting proteins from the ESR1|PIK3CA subnetwork with an alpelisib-LJM716 combination. In colorectal cancer, they co-target BRAF|PIK3CA with alpelisib, cetuximab, and encorafenib combination. Collectively, our pipeline’s results are promising, and validated by patient-based xenografts. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22ff3c0679606594ddbe475542399357426fef69" target='_blank'>
              Anticancer Target Combinations: Network-Informed Signaling-Based Approach to Discovery
              </a>
            </td>
          <td>
            B. R. Yavuz, Hyunbum Jang, Ruth Nussinov
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Multiple myeloma is a plasma cell malignancy that accounts for 1-2% of all cancer diagnoses and 10-15% of hematopoietic neoplasms. Clonal rearrangement of the IG locus suggests a monoclonal model for pathogenesis. The presence of cytogenetic abnormalities in >90% of cases suggests transformation by chromosomal instability and rearrangement. Classification and consensus guidelines by The International Myeloma Working Group (IMWG) and others reflect this by categorizing myeloma based on ploidy, translocations, and chromosomal abnormalities.
 Interestingly, a portion of Emory’s test volume for cancer gene mutation profiling by second generation short-read sequencing is for multiple myeloma cases. This suggests our 75-gene myeloid neoplasm panel (MMP75) is being utilized for medical purposes that are not met by cytogenetic and microarray characterizations performed during the diagnostic workup of multiple myeloma. Inspired by this observation, this study investigates providers motivations for ordering MMP75 and assesses the adequacy of our current implementation for their stated purposes.
 Anecdotes from ordering physicians indicate interest in MMP75 results when informing prognosis, selecting treatments, and evaluating for treatment associated secondary neoplasms. A more formal survey is underway that will statistically capture how the results of MMP75 testing were ultimately used: to inform diagnosis, determine prognosis, guide treatment, explain progression, identify/rule-out secondary neoplasia, or no effect on management.
 We also explore the variants called by MMP75 when performed using bone marrow from patients who received a cytogenetic workup for multiple myeloma at some point. A retrospective analysis of the 5-year interval between Jan 2019 and Jan 2024 identified 156 cases meeting this criterion. The number of pathogenic variants per case ranged between 1 and 8, with mutations detected across 42 genes. The genes most frequently identified with pathogenic mutations were TP53 (22.4%), DNMT3A (19.2%), TET2 (14.7%), and ASXL1 (12.2%). All four have been associated with adverse outcomes in patients with multiple myeloma. Activating events in therapeutic targets included mutations in the RAS/RAF pathway (16.7%), JAK/STAT signaling (2.6%), and the kinase KIT (0.6%).
 While these findings support the use of MMP75 for detection of relevant mutations, the genetic variants identified are not unique to multiple myeloma. Mutations detected in DNMT3A, TET2, and ASXL1 are common in clonal hematopoiesis of indeterminant potential (CHIP) and myeloid neoplasms. Hotspot mutations in TP53 and activating events in the RAS/RAF pathway also occur in various hematologic malignancies. This test therefore detects events that occur in both the plasma cell compartment and other cells of origin. This encumbers distinguishing variants that inform on a patient’s multiple myeloma from events that suggest a secondary neoplasm. Future directions include determining whether the effect of plasma cell enrichment on a mutations variant allele frequency (VAF) in repeat testing can help distinguish true multiple myeloma variants from events in other cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987b88a74ed80e2065d8c8a3c7edc9ab6f63f8af" target='_blank'>
              Improving the Utility of a Cancer Gene Mutation Panel for Multiple Myeloma Patients
              </a>
            </td>
          <td>
            Lalit Patel, Linsheng Zhang, Geoffrey Smith, David L. Jaye, Thomas Schneider
          </td>
          <td>2024-10-01</td>
          <td>American Journal of Clinical Pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a93850c21d9d18806483d79ec8a05dda4a648d" target='_blank'>
              Treatment resistance to melanoma therapeutics on a single cell level
              </a>
            </td>
          <td>
            Lijun Yao, B. Krasnick, Ye Bi, Sunantha Sethuraman, S. Goedegebuure, A. Weerasinghe, Christopher Wetzel, Qingsong Gao, A. Oyedeji, Jacqueline L Mudd, Matthew A. Wyczalkowski, M. Wendl, Li Ding, Ryan C. Fields
          </td>
          <td>2024-09-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Dysfunction of epigenetic modulators, such as the SWI/SNF complex, is a wide-spread but relatively ill-defined feature of a broad spectrum of cancer entities. Among SWI/SNF-mutant entities, SMARCB1-deficient cancers, such as the highly aggressive Epithelioid Sarcoma (EpS), are characterized by this genetic event in an otherwise rather silent mutational landscape. This renders EpS an ideal model to study how epigenetic reprogramming by a single mutation can contribute to tumorigenesis. Hence, to characterize and compare the function of the SMARCB1-deficient, residual and the physiological SWI/SNF complex in cancer, we generated a panel of SMARCB1 re-expressing EpS cell lines and employed a functional multi-omics approach. Here, we show that SWI/SNF holds canonical characteristics of both tumor-suppressors and proto-oncogenes due to its multi-faceted role in the regulation of the epigenome. Our data indicates that the loss of SMARCB1 causes an overall loss of SWI/SNF chromatin affinity at cis-regulatory enhancer elements, inducing a preference for uncontrolled proliferation and cell cycle progression as opposed to development and differentiation. We further demonstrate that EpS cell lines depend on residual SWI/SNF action to maintain clonogenicity and proliferation. Consequently, EpS cell lines exhibit markedly increased sensitivity to pharmacological inhibition of the residual SWI/SNF when compared with SWI/SNF-proficient cancer entities. Collectively, our results from the EpS model shed new light on how a single mutation can rewire the pleiotropic effects of an epigenetic master regulator and provide inroads for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba81f1fa7d5881ab59ab9ddb07d9a1d9290af17" target='_blank'>
              Loss of SMARCB1 evokes targetable epigenetic vulnerabilities in Epithelioid Sarcoma
              </a>
            </td>
          <td>
            Jia Xiang Jin, Fabia Fuchslocher, Martha J. Carreño-Gonzalez, Felina Zahnow, A. K. Ceranski, Rainer Will, Dominic Helm, Felix Bestvater, A. Banito, R. Imle, S. Ohmura, F. Cidre-Aranaz, T. G. Grünewald
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In this review article we will highlight the evidences that how oncogenes are formed due to the physical genetic variations in proto-oncogenes and tumor suppressor genes and various planned immunotherapies which will include- The immune checkpoint inhibitor-opposing antibodies, adoptive cell treatments, and biologic modifiers (cytokines and vaccines). We will make an effort to provide guidance and potential fixes for these issues, along with pertinent sources for foundational research. For suitable studies, a literature search was undertaken from various database sources such as PubMed, EMBASE, and Google Scholar. One type of gene known as an oncogene-a cellular gene that becomes dysfunctional owing to mutation and overexpression-is the cause of cancer. Certain oncogenes seem to inhibit the homeostatic mechanism by limiting the single cell lineage of leukemia stem cells. According to the clonal theory of oncogenes, tumors are thought to begin in a single cell, Moreover, the growth of tumors is closely linked to the prevention of apoptosis, or programmed cell death. These activities of oncogene can be minimized by some immunological therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77360e601c608aabfa1825689be8b174a0154240" target='_blank'>
              Oncogenetics: Unraveling the Genetic Underpinnings of Cancer for Improved Immunotherapeutic Outcomes.
              </a>
            </td>
          <td>
            Stuti Dwivedi, Praveencumar R, T. Sivakumar, M. K. Posa, R. Dhakar, Ruchi Tiwari
          </td>
          <td>2024-10-29</td>
          <td>Zhongguo ying yong sheng li xue za zhi = Zhongguo yingyong shenglixue zazhi = Chinese journal of applied physiology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic reinforcement of T cell exhaustion is known to be a major barrier limiting T cell responses during immunotherapy. However, the core epigenetic regulators restricting antitumor immunity during prolonged antigen exposure are not clear. We investigated three commonly mutated epigenetic regulators that promote clonal hematopoiesis to determine whether they affect T cell stemness and response to checkpoint blockade immunotherapy. CD8 T cells lacking Dnmt3a, Tet2, or Asxl1 preserved a progenitor-exhausted (Tpex) population for more than 1 year during chronic antigen exposure without undergoing malignant transformation. Asxl1 controlled the self-renewal capacity of T cells and reduced CD8 T cell differentiation through H2AK119 ubiquitination and epigenetic modification of the polycomb group–repressive deubiquitinase pathway. Asxl1-deficient T cells synergized with anti–PD-L1 immunotherapy to improve tumor control in experimental models and conferred a survival advantage to mutated T cells from treated patients. Editor’s summary T cell immunotherapies have been successful in treating some intractable cancers, but there are limitations caused by the inability of T cells to persist after prolonged stimulation. Kang et al. undertook a reverse-translation approach to investigate the molecular mechanisms regulating the durability of T cell responsiveness to checkpoint blockade immunotherapy (see the Perspective by Tsui and Kallies). Inspired by clinical observations from individuals with myelodysplastic syndrome, the epigenetic regulators (Dnmt3a, Tet2, and Asxl1) associated with clonal hematopoiesis were identified using experimental models of T cell exhaustion. Epigenetic disruption of the polycomb group repressive deubiquitinase complex resulted in preservation of a stem cell–like T cell population that was responsive to immunotherapy. Extending this mechanism to cancer adoptive cell therapy, Asxl1 disruption was found to endow T cells with superior therapeutic efficacy and an ability to synergize with checkpoint blockade immunotherapy —Priscilla N. Kelly INTRODUCTION Sustained stimulation of CD8 T cells promotes the development of functional exhaustion, a terminal cell fate that has been broadly recognized to limit the durability of T cell–based immunotherapies. This state of exhaustion arises though a progressive transition of multipotent progenitor T cells (Tpex) into a terminally differentiated population (Tex) that has been defined as being nonresponsive to immune checkpoint blockade (ICB). The molecular determinants dictating the developmental transition are now being investigated as engineering targets to block the progression of T cell exhaustion and preserve the durability of immunotherapy approaches. RATIONALE We recently observed that the long-term survival of a small cohort of myelodysplastic syndrome (MDS) patients treated with anti–PD-L1 therapy was coupled to mutation of the ASXL1 gene among their T cells. Mutations in ASXL1, in addition to DNMT3A and TET2, are commonly associated with conferring a survival advantage among hematopoietic stem cells, resulting in subclonal outgrowth referred to as clonal hematopoiesis. Because of their general link to cellular stemness, we investigated the role of these regulators in the developmental transition of Tpex into Tex and the impact of deletion of these regulators on therapeutic durability during anti–PD-L1 treatment. RESULTS To investigate the role of Dnmt3a, Tet2, and Asxl1 in the development and maintenance of an ICB-responsive T cell population, we engineered T cells to contain mutations within each of these genes and then exposed them to a chronic source of antigen by adoptively transferring them into mice infected with the clone 13 strain of lymphocytic choriomeningitis virus (LCMV). Disruption of these regulators in a canonical model of T cell exhaustion allowed them to maintain their quantity and ICB-responsive state for >1 year during chronic antigen exposure. Despite being stimulated and undergoing antigen-driven proliferation for 1 year, these T cells did not exhibit any signs of malignant outgrowth. Moreover, the total pool of T cells containing each of these mutations was enriched for a large quantity of stem-like Tcf1+ T cells. Fate-tracking adoptive transfer experiments revealed that this Tcf1+ subset of ASXL1 KO T cells could both self-renew and give rise to potent effectors. Transcriptional and epigenetic profiling of the knockout (KO) T cells identified Asxl1 as a regulator of the polycomb group–repressive deubiquitinase (PR-DUB) complex, controlling the deubiquitination of histone 2A lysine 119, which serves as a molecular checkpoint for the developmental transition of Tpex into Tex. Extension of this mechanistic insight into tumor models revealed that adoptively transferred T cells with Asxl1 disruption resisted tumor microenvironment–induced exhaustion. Moreover, the adoptively transferred Asxl1 KO T cells synergized with anti–PD-L1 blockade, resulting in heightened tumor control. Finally, we show that Asxl1 disruption of tumor-specific T cells correlates with increased expansion of T cells in MDS patients after treatment with anti–PD-L1. CONCLUSION Results from these studies show that DNMT3A, TET2, and ASXL1 control the developmental checkpoint between Tpex and terminally exhausted T cells. Specific investigation into Asxl1 revealed its role in T cell differentiation through epigenetic modification of the PR-DUB pathway. These data articulate a reverse-translation approach for defining molecular mechanisms that restrict maintenance of the ICB-responsive stem-like T cells within a suppressive tumor microenvironment. Through our mechanistic investigation of genes associated with patient survival, we have established a scientific rationale for specific genetic modification of discrete epigenetic regulators for future engineering to enhance the durability of cancer immunotherapy. Core epigenetic regulators in maintaining therapeutic durable multipotent Tpex populations. Using the canonical murine model system of exhaustion, disruption of Dnmt3a, Tet2, and Asxl1 in CD8 T cells preserved an ICB-responsive Tpex population for >1 year during chronic antigen exposure. Specific investigation into the new epigenetic regulator Asxl1 revealed that this unprecedented maintenance of Tpex is mediated by presence of monoubiqutinated histone 2A lysine 119 (H2AK119Ub) through disruption of the PR-DUB complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e5351a39641c751005cc32a06480cddb05203" target='_blank'>
              Epigenetic regulators of clonal hematopoiesis control CD8 T cell stemness during immunotherapy
              </a>
            </td>
          <td>
            Tae Gun Kang, Xin Lan, Tian Mi, Hongfeng Chen, Shanta Alli, Song-Eun Lim, Sheetal Bhatara, A. Vasandan, Grace Ward, Sofia Bentivegna, Josh Jang, Marianne L. Spatz, Jin-Hwan Han, B. Schlotmann, J. S. Jespersen, Christopher Derenzo, Peter Vogel, Jiyang Yu, Stephen B Baylin, P. Jones, Casey O’Connell, Kirsten Grønbæk, Ben Youngblood, Caitlin C. Zebley
          </td>
          <td>2024-10-11</td>
          <td>Science</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="High expression of the long non-coding RNA (lncRNA) FAM30A has been previously associated with leukemic stem cell (LSC) activity and poor prognosis in both adult and paediatric acute myeloid leukaemia (AML) patients, yet it has not been functionally studied. This study provides the first cellular characterization of FAM30A focussing on an internal tandemly organised region, referred to as FAM30A repeats. FAM30A levels correlated with canonical AML LSC signatures and FAM30A depletion decreased cell viability as well as increased sensitivity to chemotherapeutics. It also inhibited colony formation, promoted granulocytic differentiation and abrogated leukemic engraftment in murine bone marrow in vivo. Overexpression of FAM30A repeats in this setting enhanced stemness, proliferation, chemoresistance, and engraftment thus highlighting the biological relevance of this region for LSC biology. On the molecular level, FAM30A repeats interact with the pro-LSC regulator Musashi-2 (MSI2), positively influencing expression of its targets including RUNX1 isoforms. We herein uncover that this FAM30A-MSI2-RUNX1 regulatory loop is of potential relevance for LSC maintenance in AML. These findings provide valuable insights into FAM30A’s cellular role and highlight its targeting potential for eliminating LSCs and improving treatment outcomes in AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e944155596f6ba606bda7e6372bd8e29a902812" target='_blank'>
              The long non-coding RNA FAM30A regulates the Musashi2-RUNX1 axis and is required for LSC function in AML cells
              </a>
            </td>
          <td>
            Jaime Calvo Sánchez, Mary T Scott, Athina Varnava, Tala Alakhras, Alice Wedler, Danny Misiak, N. Bley, C. Ihling, A. Sinz, Karen Keeshan, S. Hüttelmaier, David Vetrie, Marcel Köhn
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Adding aPD1 to 5-FU/platinum in advanced gastroesophageal adenocarcinomas (GEA) has yielded modest and heterogeneous results. Understanding cooperativity between these two treatment modalities will inform novel combination treatments for GEA. Towards this end, we conducted a trial (n = 47) sequentially adding pembrolizumab to 5-FU/platinum in previously untreated advanced GEA. Using serial biopsy of the primary tumor at baseline, after one cycle of 5-FU/platinum, and after the addition of pembrolizumab we profiled 66,816 tumor-resident T cells with matched single-cell RNA-seq and single-cell TCR-seq. Stratifying patients into slow- (> 6 months) and fast-progressing (< 6 months) groups based on progression-free survival allowed us to quantify clonal and transcriptional differences in T cell subpopulations across three time points. We annotated CD4 and CD8 T cell phenotypes according to canonical marker genes and found evidence of 17 different subpopulations along various stages of memory and exhaustion trajectories. We found differences in the abundance of T cell subtypes between slow- and fast-progressors at various treatment time points. T cells with exhausted phenotypes were more abundant in slow progressors in the post-chemo and post-immunotherapy time points. Additionally, we found changes in phenotypic abundances between clones shared between time points. For example, clonotypes that were predominantly of the progenitor exhausted phenotype pre-immunotherapy had a notable proportion of shared post-ICB clones exhibiting an activated phenotype. Previous studies have highlighted exhausted T cells as having important roles in maintaining tumor immunity as these tend to be subject to chronic antigen stimulation due to their specificity to tumor antigen. For this reason, we identified tumor-reactive T cell clonotypes among our data using literature-defined proxies such as CXCL13 expression, exhaustion status and degree of post-therapeutic clonal expansion, in addition to transcription signature-based predictions using machine learning tools such as predicTCR and TRTpred. We found that tumor-reactive T cell clonotypes tended to be present in slow progressors across time points. CXCL13 had variable patterns of upregulation and downregulation depending on whether a particular tumor-reactive clonotype was expanding or contracting at a specific treatment time point. Moreover, we quantified whether these tumor reactive clonotypes shared clones with non-exhausted T cell subsets prior to treatment or whether they were newly present in the tumor microenvironment (TME) post-treatment to determine the relative contributions of phenotypic transition and peripheral replenishment to tumor reactive T cells in the gastric cancer tumor microenvironment. Our data provide an invaluable resource to understand T cell behavior in the context of chemoimmunotherapy and propose strategies to drive anti-tumor immunity to expand the portion of patients benefiting from immunotherapy approaches.
 Citation Format: Samuel J. Wright, Minae An, Byung Hoon Min, You Jeong Heo, Milan Parikh, Lynn Bi, Hyuk Lee, Taejun Kim, Song-Yi Lee, Jeonghyeon Moon, Matthew R. Strickland, Woong Yang Park, Won Ki Kang, Kyoung-Mee Kim, Seung Tae Kim, Ryan J. Park, Arnav Mehta, Samuel Klempner, Jeeyun Lee. Tumor-reactive T cell clonal dynamics across treatment time points drive resistance and progression to frontline chemoimmunotherapy in advanced gastric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41a98abd5d238514b71c6450ac3a4ce0efd3962" target='_blank'>
              Abstract A010: Tumor-reactive T cell clonal dynamics across treatment time points drive resistance and progression to frontline chemoimmunotherapy in advanced gastric cancer
              </a>
            </td>
          <td>
            Samuel J Wright, M. An, B. Min, Y. Heo, Milan Parikh, Lynn Bi, Hyuk Lee, Taejun Kim, Song-Yi Lee, Jeonghyeon Moon, Matthew R Strickland, Woong Yang Park, Wonseok Kang, Kyoung-Mee Kim, S. Kim, Ryan J. Park, Arnav Mehta, S. Klempner, Jeeyun Lee
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small population of less differentiated cells with robust self-renewal ability. CSCs have been recognized as the root cause of tumor initiation, progression, relapse, and drug resistance. Recent studies from us and others have highlighted that N6-methyladenosine (m6A), the most prevalent modification in mRNA, plays a crucial role in carcinogenesis and CSC homeostasis. Dysregulation of the m6A modification machinery has been implicated in CSC survival and self-renewal, thereby regulating cancer progression and therapeutic resistance. In this review, we provide an overview of the roles and molecular mechanisms of the RNA m6A modification machinery in CSC survival and self-renewal. Additionally, we summarize the currently known small-molecule inhibitors targeting the dysregulated m6A modification machinery and discuss proof-of-concept studies focusing on the efficacy of these compounds in eliminating CSCs and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a2c463331686ce335a111580d49921b4543bf" target='_blank'>
              Combating cancer stem cells: RNA m6A methylation and small-molecule drug discovery
              </a>
            </td>
          <td>
            Honghai Zhang, Xueer Wang, Jianjun Chen, R. Su
          </td>
          <td>2024-09-30</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The oncogenic mechanisms by which TFE3 fusion proteins drive translocation renal cell carcinoma (tRCC) are poorly characterised. Here, we integrated loss and gain of function experiments with multi-omics analyses in tRCC cell lines and patient tumors. High nuclear accumulation of NONO-TFE3 or PRCC-TFE3 fusion proteins promotes their broad binding across the genome at H3K27ac-marked active chromatin, engaging a core set of M/E-box-containing regulatory elements to activate specific gene expression programs as well as promiscuous binding to active promoters to stimulate mRNA synthesis. Within the core program, TFE3 fusions directly regulate genes involved in ferroptosis resistance and oxidative phosphorylation metabolism (OxPhos) increasing functional OxPhos levels. Consequently, human tRCC tumors display high OxPhos scores that persist during their epithelial to mesenchymal transition (EMT). We further show that tRCC tumour aggressiveness is related to their EMT which although enriched in tumours harbouring ASPSCR1-TFE3 fusions, is also a frequent feature of tumours harbouring other TFE3 fusions. In tRCC tumours, the presence of mesenchymal tRCC cancer cells associates with that of myofibroblast cancer-associated fibroblasts (myCAFs) that are both hallmarks of poor prognostic outcomes. We define tRCC as a novel metabolic subtype of renal cancer and provide unique insights into how broad genomic binding of TFE3 fusion proteins regulates OxPhos and ferroptosis resistance and more generally stimulates RNA synthesis. Significance. This study advances understanding of the molecular mechanisms underlying oncogenic transformation by TFE3 fusion proteins in translocation renal cell carcinoma and defines features of tumour cells and their microenvironment that negatively impact patient outcome. Integrative multi-omics analyses reveal how extensive genomic binding of TFE3 fusion proteins drives oncogenic programs and identify novel potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/918502a23f46233e22d63334acfc338923bba14a" target='_blank'>
              TFE3 fusion proteins drive oxidative metabolism, ferroptosis resistance and general RNA synthesis in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Alexandra Helleux, G. Davidson, A. Lallement, Alexandre Haller, I. Michel, A. Fadloun, C. Thibault-Carpentier, Xiaoping Su, Véronique Lindner, Thibaut Tricard, Hervé Lang, N. Tannir, Irwin Davidson, G. Malouf
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Saccharomyces cerevisiae is an excellent model to study the effect of external cues on cell division and stress response. 5-Fluorocuracil (5-FU) has been used to treat solid tumors since several decades. The drug was initially designed to interfere with DNA replication but was later found to exert its antiproliferative effect also via RNA-dependent processes. Since 5-FU inhibits the activity of the 3'-5'-exoribonuclease Rrp6 in yeast and mammals, earlier work has compared the effect of 5-FU treatment and RRP6 deletion at the transcriptome level in diploid synchronized yeast cells. To facilitate interpreting the expression data we have developed an improved 5-Fluorouracil RNA (5-FUR) expression viewer. Users can access information via genome coordinates and systematic or standard names for mRNAs and Xrn1-dependent-, stable-, cryptic-, and meiotic unannotated transcripts (XUTs, SUTs, CUTs, and MUTs). Normalized log2-transformed or linear data can be displayed as filled diagrams, line graphs or color-coded heatmaps. The expression data are useful for researchers interested in processes such as cell cycle regulation, mitotic repression of meiotic genes, the effect of 5-FU treatment and Rrp6 deficiency on the transcriptome and expression profiles of sense/antisense loci that encode overlapping transcripts. The viewer is accessible at http://5fur.genouest.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad442f4d43c5032d0d97fedb33e587d2cd06478" target='_blank'>
              The 5-Fluorouracil RNA Expression Viewer (5-FUR) Facilitates Interpreting the Effects of Drug Treatment and RRP6 Deletion on the Transcriptional Landscape in Yeast.
              </a>
            </td>
          <td>
            Ugo Szachnowski, Oliver Sallou, Matéo Boudet, Anthony Bretaudeau, M. Wéry, Antonin Morillon, Michael Primig
          </td>
          <td>2024-09-30</td>
          <td>Yeast</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Introduction Reprogramming of pancreas cell fate drives development of pancreatic ductal adenocarcinoma (PDAC). Acinar cells, the most probable origin of pancreatic cancer, undergo a rapid cell identity switch towards a duct-like phenotype upon KrasG12D expression and when combined with pancreatitis or the loss of acinar differentiation factors. Metaplastic and neoplastic duct-like cells are heterogeneous with a proportion acquiring features reminiscent of gastric lineages. While some gastric signatures are maintained in the classical PDAC subtype, they are eroded in the more aggressive, basal-like PDAC. Since subtype identity has a major impact on prognosis and therapeutic targetability, druggable targets that regulate cellular reprogramming in pancreatic cancer can be exploited to increase sensitivity to therapy. Methods To elucidate mechanisms responsible for early reprogramming, pancreatic tissue of mice with conditional KrasG12D expression and loss of Pdx1 was analyzed by single-nucleus ATAC-seq. Expression of identified target genes was studied in precancerous lesions and PDAC by using multiplex RNAscope and IHC staining. Computational analyses of publicly available sequencing data were used to establish correlation to PDAC subtype identity. Genes were functionally studied in PDAC cell lines. Results By performing snATAC-seq and RNA-seq of early transformed pancreatic tissue, we discovered that acinar cells with the combined expression of KrasG12D and loss of Pdx1 activate expression of a gastric metaplastic gene signature accompanied by elevated levels of the receptor kinase Ror2. Ror2 is also highly expressed in distinct subpopulations of metaplastic and neoplastic cells, associated with a gastric neck cell phenotype and enhanced proliferative capacity. In contrast, Ror2Low lesions are characterized by a gastric pit cell-like and a senescent phenotype. Genetic ablation of Ror2 resulted in a shift in lesion identity, enriching those with a pit cell and senescent identity. In PDAC, we found that Ror2 anti-correlates with a similar gastric pit cell phenotype that is maintained in the classical subtype PDAC, but is strongly associated with the more aggressive basal-like subtype. Overexpression of ROR2 in human PDAC cell lines with a classical differentiation induced loss of the classical gene signature as well as epithelial-to-mesenchymal transition. Moreover, ROR2 enforces a strong dependency on AKT signaling, causing increased vulnerability of ROR2-expressing cells to AKT inhibition, but increased resistance to the KRAS inhibitor MRTX1133. Conclusions We discovered Ror2 as a critical determinant of precancerous lesion as well as PDAC subtype identity. Its role in driving an aggressive PDAC phenotype that is inherently resistant to Kras inhibition suggests that inhibiting this receptor tyrosine kinase will enhance sensitivity to the new generation of targeted therapies.
 Citation Format: Simone Benitz, Alec Steep, Malak Nasser, Jonathan Preall, Ujjwal Mukund Mahajan, Holly McQuithey, Ian Loveless, Erick T. Davis, Hui-Ju Wen, Daniel W. Long, Thomas Metzler, Samuel Zwernik, Michaela Louw, Donald Rempinski, Daniel Salas-Escabillas, Sydney Brender, Linghao Song, Ling Huang, Brian K. Theisen, Zhenyu Zhang, Nina G. Steele, Ivonne Regel, Filip Bednar, Howard C. Crawford. Ror2, a Novel Key Regulator Driving Cell Fate Decisions throughout Pancreatic Carcinogenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada52d632c14a2e67876713dc628440690c3b38e" target='_blank'>
              Abstract C075: Ror2, a Novel Key Regulator Driving Cell Fate Decisions throughout Pancreatic Carcinogenesis
              </a>
            </td>
          <td>
            S. Benitz, Alexander Steep, Malak M Nasser, Jonathan Preall, U. Mahajan, Holly McQuithey, Ian Loveless, Erick T Davis, Hui-Ju Wen, Daniel W Long, Thomas Metzler, Samuel Zwernik, Michaela Louw, D. Rempinski, Daniel J. Salas-Escabillas, Sydney M Brender, Linghao Song, Ling Huang, Brian Theisen, Zhenyu Zhang, Nina G. Steele, I. Regel, Filip Bednar, Howard C. Crawford
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rationale: Both bulk RNA-sequencing and GEO database upon chemotherapy to non-small cell lung cancer (NSCLC) cells reveal that ZNF131 (Zinc Finger Protein 131) maybe a crucial transcriptional factor involved. However, it is a recently discovered protein with largely unexplored expression patterns and biological functions. Methods: Bioinformatics analyses and immunohistochemistry staining were assessed to detect both mRNA and protein levels of ZNF131 in NSCLC specimens and cell lines. Next, colony formation assay, MTT assay, EdU assay, transwell assay, flow cytometric analysis, sphere formation assay, western blotting analysis, mouse xenograft model analysis, immunofluorescence assay, and reverse transcriptase ‐ polymerase chain reaction were performed to investigate the effect of ZNF131 interaction on proliferation, invasion, stemness, chemotherapy sensitivity. RNA-sequencing assay, RNA-microarray, and ChIP-sequencing assay were used to identify candidate downstream target genes. Further, liquid chromatography-tandem mass spectrometry analysis, GST pull-down assay, and immunoprecipitation assays were performed to evaluate the interactions between ZNF131, BACH1, and CUL3. Results: ZNF131 was elevated in NSCLC specimens and cell lines, which significantly correlates with advanced TNM stage and poor prognosis in NSCLC patients. ZNF131 overexpression promotes NSCLC cell proliferation, invasion, and stemness both in vitro and in vivo . ZNF131 appears to target the RAD51 gene within a well-defined region (-668bp to -403bp) of the RAD51 promoter. ZNF131 contributes to RAD51-dependent homologous recombination (HR), primarily through its Zinc Finger and BTB domains. ZNF131-BACH1 interaction, mediated by their respective BTB domains, enhances the stability of both proteins, effectively preventing their ubiquitin-mediated degradation by CUL3. The ZNF131-BACH1 partnership significantly amplifies RAD51-dependent HR, resulting in expedited resistance to both radiotherapy and chemotherapy in NSCLC patients. Desoxyrhaponticin was shown to halt NSCLC progression and orchestrate a synergistic effect together with chemotherapy at least partially by targeting ZNF131.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b84d4dd3e29e2628ded0f7bfec71ddec9627fb" target='_blank'>
              ZNF131-BACH1 transcriptionally accelerates RAD51-dependent homologous recombination repair and therapy-resistance of non-small-lung cancer cells by preventing their degradation from CUL3
              </a>
            </td>
          <td>
            Mingwei Fan, Quanbo Liu, Xiaowen Ma, Yufeng Jiang, Yilong Wang, Shuting Jia, Yingtong Nie, Ruoyi Deng, Pengchong Zhou, Shuyu Zhang, Siyu Jiang, Mengyao Guan, Yuekang Hou, Yuan Miao, Yong Zhang, Xiupeng Zhang
          </td>
          <td>2024-10-28</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Direct lineage reprogramming into dopaminergic (DA) neurons holds great promise for the more effective production of DA neurons, offering potential therapeutic benefits for conditions such as Parkinson's disease. However, the reprogramming pathway for fully reprogrammed DA neurons remains largely unclear, resulting in immature and dead-end states with low efficiency. In this study, using single-cell RNA sequencing, the trajectory of reprogramming DA neurons at multiple time points, identifying a continuous pathway for their reprogramming is analyzed. It is identified that intermediate cell populations are crucial for resetting host cell fate during early DA neuronal reprogramming. Further, longitudinal dissection uncovered two distinct trajectories: one leading to successful reprogramming and the other to a dead end. Notably, Arid4b, a histone modifier, as a crucial regulator at this branch point, essential for the successful trajectory and acquisition of mature dopaminergic neuronal identity is identified. Consistently, overexpressing Arid4b in the DA neuronal reprogramming process increases the yield of iDA neurons and effectively reverses the disease phenotypes observed in the PD mouse brain. Thus, gaining insights into the cellular trajectory holds significant importance for devising regenerative medicine strategies, particularly in the context of addressing neurodegenerative disorders like Parkinson's disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bfa61a76443945c179bc5c94c72de94ea5c5df" target='_blank'>
              Epigenetic Dynamics in Reprogramming to Dopaminergic Neurons for Parkinson's Disease.
              </a>
            </td>
          <td>
            Byounggook Cho, Junyeop Kim, Sumin Kim, Saemin An, Yerim Hwang, Yunkyung Kim, Daeyeol Kwon, Jongpil Kim
          </td>
          <td>2024-09-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genetic mutations are being thoroughly mapped in human cancers, yet a fundamental question in cancer biology is whether such mutations are functionally required for cancer initiation, maintenance of established cancer, or both. Here, we study this question in the context of human acute myeloid leukemia (AML), where DNMT3AR882 missense mutations often arise early, in pre-leukemic clonal hematopoiesis, and corrupt the DNA methylation landscape to initiate leukemia. We developed CRISPR-based methods to directly correct DNMT3AR882 mutations in leukemic cells obtained from patients. Surprisingly, DNMT3AR882 mutations were largely dispensable for disease maintenance. Replacing DNMT3AR882 mutants with wild-type DNMT3A did not impair the ability of AML cells to engraft in vivo, and minimally altered DNA methylation. Taken together, DNMT3AR882 mutations are initially necessary for AML initiation, but are largely dispensable for disease maintenance. The notion that initiating oncogenes differ from those that maintain cancer has important implications for cancer evolution and therapy. STATEMENT OF SIGNIFICANCE Understanding which driver mutations are required for cancer initiation, maintenance, or both phases remains poorly understood. Here, we uncover that highly prevalent pre-leukemic DNMT3A mutations are only required during disease initiation, but become dispensable after leukemic transformation, uncovering the context-specific role of this driver mutation with important therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94506bb3eb524303455ffe91b550e1d6fd5d7a8d" target='_blank'>
              DNMT3AR882H Is Not Required for Disease Maintenance in Primary Human AML, but Is Associated With Increased Leukemia Stem Cell Frequency
              </a>
            </td>
          <td>
            Thomas Köhnke, Daiki Karigane, Eleanor Hilgart, Amy C. Fan, Kensuke Kayamori, Masashi Miyauchi, Cailin T Collins, Fabian P. Suchy, Athreya S Rangavajhula, Yang Feng, Yusuke Nakauchi, Eduardo Martinez-Montes, J. Fowler, Kyle M. Loh, Hiromitsu Nakauchi, Michael A Koldobskiy, Andy Feinberg, Ravi Majeti
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ceac36006253317f77204fdfb7e8de57c83f85" target='_blank'>
              Population variability in X-chromosome inactivation across 10 mammalian species
              </a>
            </td>
          <td>
            Jonathan M. Werner, John Hover, Jesse Gillis
          </td>
          <td>2024-10-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aging is an important risk factor for tumorigenesis. Metabolic reprogramming is a hallmark of both aging and tumor initiation. However, the manner in which the crosstalk between aging and metabolic reprogramming affects the tumor microenvironment (TME) to promote tumorigenesis was poorly explored. We utilized a computational approach proposed by our previous work, MMP3C (Modeling Metabolic Plasticity by Pathway Pairwise Comparison), to characterize aging-related metabolic plasticity events using pan-cancer bulk RNA-seq data. Our analysis revealed a high degree of metabolically organized heterogeneity across 17 aging-related cancer types. In particular, a higher degree of several energy generation pathways, i.e., glycolysis and impaired oxidative phosphorylation, was observed in older patients. Similar phenomena were also found via single-cell RNA-seq analysis. Furthermore, those energy generation pathways were found to be weakened in activated T cells and macrophages, whereas they increased in exhausted T cells, immunosuppressive macrophages, and Tregs in older patients. It was suggested that aging-induced metabolic switches alter glucose utilization, thereby influencing immune function and resulting in the remodeling of the TME. This work offers new insights into the associations between tumor metabolism and the TME mediated by aging, linking with novel strategies for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92f52b4b6e5f7172ec007a7f5c79b9ac1800330" target='_blank'>
              Metabolic Reprogramming Induced by Aging Modifies the Tumor Microenvironment
              </a>
            </td>
          <td>
            Xingyu Chen, Zihan Wang, Bo Zhu, Min Deng, Jiayue Qiu, Yunwen Feng, Ning Ding, Chen Huang
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Phosphoinositides (PIs) are minor but essential phospholipids that play crucial roles in cellular signaling pathways, membrane dynamics, and the regulation of various cellular processes. We developed and utilized a novel PI-focused CRISPR gRNA library to perform negative-selection and positive-selection screens in PANC-1 and OCI-AML2 cells, models of pancreatic ductal adenocarcinoma (PDAC) and acute myeloid leukemia (AML), respectively. Through these screens, we identified 28 essential genes in PANC-1, 84 essential genes in OCI-AML2, and 28 regulators of colony formation in OCI-AML2. Our results using this small and focused library uncovered false negatives and subtle effects that may be missed in genome-wide approaches, while enabling adaptation to different screening conditions. Overall, our results uncovered previously uncharacterized essential genes in PDAC and AML that can be leveraged as therapeutic targets and biomarkers. We also demonstrate that focused libraries offer a more efficient and targeted approach to uncovering critical genetic determinants of cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468ec30b5ec6c72c51b0f5fb5f674977328a427d" target='_blank'>
              Phosphoinositide-focused CRISPR screen identifies novel genetic vulnerabilities in PDAC and AML cells
              </a>
            </td>
          <td>
            Daniel K.C. Lee, Ryan Loke, Jonathan TS Chow, M. Gabra, L. Salmena
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Simple Summary The therapeutic landscape of Chronic Lymphocytic Leukemia (CLL) has dramatically changed in recent years, with a shift from chemoimmunotherapy towards many new targeted agents, such as Bruton Tyrosine Kinase inhibitors (BTKi) and anti-BCL-2. We now need an accurate re-definition of prognostic/predictive parameters to support clinicians in choosing the more appropriate option. This review explores recent advances in the molecular biology of chronic lymphocytic leukemia, with the aim of identifying the most important biomarkers to define prognosis and to guide first-line and second-line treatment. Abstract Chronic Lymphocytic Leukemia (CLL) is the most frequent type of leukemia in Western countries. In recent years, there have been important advances in the knowledge of molecular alterations that underlie the disease’s pathogenesis. Very heterogeneous prognostic subgroups have been identified by the mutational status of immunoglobulin heavy variable genes (IGVH), FISH analysis and molecular evaluation of TP53 mutations. Next-generation sequencing (NGS) technologies have provided a deeper characterization of the genomic and epigenomic landscape of CLL. New therapeutic targets have led to a progressive reduction of traditional chemoimmunotherapy in favor of specific biological agents. Furthermore, in the latest clinical trials, the minimal residual disease (MRD) has emerged as a potent marker of outcome and a guide to treatment duration. This review focuses on recent insights into the understanding of CLL biology. We also consider the translation of these findings into the development of risk-adapted and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3c13c68ecb506ccc3f3b3f0d7d44f962adc11fb" target='_blank'>
              Recent Advances in the Molecular Biology of Chronic Lymphocytic Leukemia: How to Define Prognosis and Guide Treatment
              </a>
            </td>
          <td>
            A. Arcari, Lucia Morello, Elena Borotti, Elena Ronda, Angela Rossi, Daniele Vallisa
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Overexpression of the MYC oncogene, encoding c-MYC protein, contributes to the pathogenesis and drug resistance of acute myeloid leukemia (AML) and many other hematopoietic malignancies. Although standard chemotherapy has predominated in AML therapy over the past five decades, the clinical outcomes and patient response to treatment remain suboptimal. Deeper insight into the molecular basis of this disease should facilitate the development of novel therapeutics targeting specific molecules and pathways that are dysregulated in AML, including fms-like tyrosine kinase 3 (FLT3) gene mutation and cluster of differentiation 33 (CD33) protein expression. Elevated expression of c-MYC is one of the molecular features of AML that determines the clinical prognosis in patients. Increased expression of c-MYC is also one of the cytogenetic characteristics of drug resistance in AML. However, direct targeting of c-MYC has been challenging due to its lack of binding sites for small molecules. In this review, we focused on the mechanisms involving the bromodomain and extra-terminal (BET) and cyclin-dependent kinase 9 (CDK9) proteins, phosphoinositide-Akt-mammalian target of rapamycin (PI3K/AKT/mTOR) and Janus kinase-signal transduction and activation of transcription (JAK/STAT) pathways, as well as various inflammatory cytokines, as an indirect means of regulating MYC overexpression in AML. Furthermore, we highlight Food and Drug Administration (FDA)-approved drugs for AML, and the results of preclinical and clinical studies on novel agents that have been or are currently being tested for efficacy and tolerability in AML therapy. Overall, this review summarizes our current knowledge of the molecular processes that promote leukemogenesis, as well as the various agents that intervene in specific pathways and directly or indirectly modulate c-MYC to disrupt AML pathogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1135bf881c820d3eca90352e7790a726703df16e" target='_blank'>
              Targeting Molecular Signaling Pathways and Cytokine Responses to Modulate c-MYC in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Kyle Gu, Harry A. May, Min H. Kang
          </td>
          <td>2024-09-14</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Rearrangement during Transfection (RET) proto-oncogene is involved in the pathogenesis of various thyroid cancer subtypes and drugs that block the RET tyrosine kinase pathways are used in the management of locally advanced and metastatic Medullary Thyroid Cancer (MTC). Acquired resistance to RET inhibitors likely occurs via two different mechanisms: (1) On-target mutations that prevent drug binding and (2) Activated alternative mechanisms which bypass the targeted kinase. We present a unique mechanism of selpercatinib resistance via secondary NTRK fusion mutation in a case of RET altered medullary thyroid cancer. Case Presentation: The patient is a 72-year-old female with stage III metastatic MTC who developed recurrence with newly found RET mutation 7 years after complete surgical resection. Despite initial stable response with selpercatinib for 2 years she rapidly progressed with widespread disease. Cell-free DNA (cfDNA) testing done at the time of progression revealed an NTRK1 fusion. Unfortunately, the patient succumbed to the disease despite starting targeted therapy with larotrectinib. Conclusion: This case highlights the importance of further investigation into mechanisms of resistance to RET inhibitors and drug studies directed towards overcoming them">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e3045475bb371d9a0f6fe4e0d9d1e76f2be3bf" target='_blank'>
              NTRK Fusion as an Acquired Mechanism of Resistance to Selpercatinib in RET Positive Thyroid Cancer
              </a>
            </td>
          <td>
            Nikita Chintapally, Kenneth D Burman, Irina Veytsman
          </td>
          <td>2024-09-10</td>
          <td>Advances in Hematology and Oncology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 While neoantigen-directed therapies have primarily focused on tumor-specific somatic mutations, emerging evidence suggests that some malignancies aberrantly translate regions of the genome outside of annotated open reading frames (ORFs), leading to HLA-I presentation of cryptic peptides. Non-canonical HLA-I bound peptides (ncHLAp) can arise from aberrant translation of genomic elements like 5’ and 3’ untranslated regions (UTRs), long noncoding RNAs (lncRNAs), retained introns, and translation in alternative reading frames. Pancreatic cancer has a low-to-intermediate mutational burden; hence, efforts to broaden the landscape targetable antigens in this disease are greatly needed. Here, we leveraged twelve pancreatic cancer (PDAC) patient-derived organoids (PDOs) to purify and enrich the malignant compartment from low tumor cellularity tumor specimens. PDOs were subjected to extensive genomic (whole genome sequencing) and transcriptomic (RNA-sequencing) profiling to enable mutation calling and HLA typing. We then employed a personalized proteogenomic platform coupled with high-depth immunopeptidomics and empirically identified >90,000 unique PDAC HLA-I-bound peptides (HLAp). We detected HLAp arising from somatic mutations (both missense and frameshift) in a subset of PDAC patients; however, we did not detect any shared mutation-derived neoepitopes in our patient cohort. Moreover, we empirically identified >1,700 ncHLAp, primarily arising from translation of novel unannotated open reading frames (nuORFs), and a substantial proportion of ncHLAp were shared amongst PDAC patients with the appropriate HLA haplotype. We developed a highly stringent pipeline using immunopeptidomics and ribosome-sequencing to investigate translation of nuORFs across a range of healthy tissues, including healthy thymus, and found that >500 ncHLAp exhibit highly-specific cancer-restricted translation. We next investigated the immunogenicity of PDAC-restricted ncHLAp and mutation-derived HLAp using an ex vivo T cell priming and expansion platform. Here we demonstrate that a subset of both mutation-derived and PDAC-restricted ncHLAp harbor robust immunogenicity. ncHLAp-specific T cell receptors (TCRs) were identified from reactive cytotoxic T lymphocytes (CTLs) via single-cell TCR-sequencing, and antigen specificity was confirmed after TCR reconstruction and cloning. Finally, using a newly developed organoid: T cell co-culture platform, we demonstrate that ncHLAp-reactive TCR-redirected T cells exhibit robust cytotoxicity and tumoricidal activity in PDAC, underscoring the translational potential for this novel class of antigens. Collectively, we have shown that cryptic epitopes, arising from aberrant translation in PDAC, represent a particularly promising class of antigens for next generation immune-based therapies.
 Citation Format: Zackery A Ely, Zachary J Kulstad, Gurcan Gunaydin, Sudarsana Addepalli, Eva K Verzani, Jennifer G Abelin, Marta Casarrubios, Karl R Clauser, Xilin Wang, Isabelle Lippincott, Cedric Louvet, Tom Schmitt, Miles Agus, Kevin S Kapner, Connor J Hennessey, James Cleary, Sine R Hadrup, Susan Klaeger, Jennifer Su, Alex M Jaeger, Brian M Wolpin, Srivatsan Raghavan, Eric Smith, Philip D Greenberg, Andrew J Aguirre, Steven A Carr, Tyler Jacks, William A Freed-Pastor. Discovery and therapeutic potential of novel cryptic peptides in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-07.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04f2440d3a56000134da94c90e982d27b5f5ca1" target='_blank'>
              Abstract PR-07: Discovery and therapeutic potential of novel cryptic peptides in pancreatic cancer
              </a>
            </td>
          <td>
            Zackery A. Ely, Zachary J. Kulstad, Gurcan Gunaydin, Sudarsana Addepalli, Eva K. Verzani, Jennifer G. Abelin, Marta Casarrubios, Karl R Clauser, Xilin Wang, Isabelle Lippincott, Cedric Louvet, Tom Schmitt, Miles Agus, Kevin S. Kapner, Connor J Hennessey, James M. Cleary, S. Hadrup, Susan Klaeger, Jennifer Su, Alex M. Jaeger, B. Wolpin, Srivatsan Raghavan, Eric Smith, Philip D. Greenberg, A. Aguirre, Steven A. Carr, T. Jacks, William A. Freed-Pastor
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background: The heterogeneity of breast cancer (BC) subtypes poses a significant challenge, with carcinogenesis involving multiple stages and genes, including proto-oncogenes, tumor suppressor genes, and DNA repair genes. Next-generation sequencing has expanded access to multigene panels, such as RAD51 paralogs, which increase the risk of ovarian cancer and possibly triple-negative (TN) BC.
Case presentation: We present a rare case of a 45-year-old woman with TNBC and a RAD51D gene mutation. Mammography and breast ultrasonography revealed an irregular, 30 mm hypoechoic area and dystrophic calcifications in the right breast. Immunohistochemistry showed a lack of expression of ER, RP, HER-2, and P53, with 50% of neoplastic cell nuclei positive for Ki-67. Next-generation sequencing revealed a mutation in RAD51D and MUYTH genes. The patient underwent partial mastectomy, chemotherapy, and prophylactic mastectomy.
Conclusion: Genetic analysis is crucial for identifying specific mutations contributing to TNBC development. Current preventive interventions primarily address BRCA1 and BRCA2 mutations, following established guidelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad46462fe1e999efcd6af2efedf0788ba216b934" target='_blank'>
              A Rare Case of Triple-Negative Breast Cancer with RAD51D Gene Mutation
              </a>
            </td>
          <td>
            Annie Monica Ch, Rithwik Goud Burri, Sriharsha Lankala, Jun Yi Lim, Altynai Zhumabekova
          </td>
          <td>2024-10-25</td>
          <td>Archives of Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49942c2e98eb891183931291f94ca9c9c0dc3993" target='_blank'>
              A compendium of genetic variations associated with promoter usage across 49 human tissues
              </a>
            </td>
          <td>
            Jiapei Yuan, Yang Tong, Le Wang, Xiaoxiao Yang, Xiaochuan Liu, Meng Shu, Zekun Li, Wen Jin, Chenchen Guan, Yuting Wang, Qiang Zhang, Yang Yang
          </td>
          <td>2024-10-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Although single cell RNA-sequencing (scRNA-seq) provides unprecedented insights into the biology of complex tissues, analyzing such data on a gene-by-gene basis is challenging due to the large number of tested hypotheses and consequent low statistical power and difficult interpretation. These issues are magnified by the increased noise, significant sparsity and multi-modal distributions characteristic of single cell data. One promising approach for addressing these challenges is gene set testing, or pathway analysis. Unfortunately, statistical and biological differences between single cell and bulk transcriptomic data make it challenging to use existing gene set collections, which were developed for bulk tissue analysis, on scRNA-seq data. In this paper, we describe a procedure for customizing gene set collections originally created for bulk tissue analysis to reflect the structure of gene activity within specific cell types. Our approach leverages information about mean gene expression in the 81 human cell types profiled via scRNA-seq by the Human Protein Atlas (HPA) Single Cell Type Atlas. This HPA information is used to compute cell type-specific gene and gene set weights that can be used to filter or weight gene set collections. As demonstrated through the analysis of immune cell scRNA-seq data using gene sets from the Molecular Signatures Database (MSigDB), accounting for cell type-specificity can significantly improve gene set testing power and interpretability. An example vignette along with gene and gene set weights for the 81 HPA SCTA cell types and the MSigDB collections are available at https://hrfrost.host.dartmouth.edu/SCGeneSetOpt/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09881903d0a088b514af64709adaa8454ee2f8e2" target='_blank'>
              Leveraging cell type-specificity for gene set analysis of single cell transcriptomics
              </a>
            </td>
          <td>
            H. R. Frost
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Transcriptional enhancers orchestrate cell type- and time point-specific gene expression programs. Genetic variation within enhancer sequences is an important contributor to phenotypic variation including evolutionary adaptations and human disease. Certain genes and pathways may be more prone to regulatory evolution than others, with different patterns across diverse organisms, but whether such patterns exist has not been investigated at a sufficient scale. To address this question, we identified signatures of accelerated sequence evolution in conserved enhancer elements throughout the mammalian phylogeny at an unprecedented scale. While different genes and pathways were enriched for regulatory evolution in different parts of the tree, we found a striking overall pattern of pleiotropic genes involved in gene regulatory and developmental processes being enriched for accelerated enhancer evolution. These genes were connected to more enhancers than other genes, which was the basis for having an increased amount of sequence acceleration over all their enhancers combined. We provide evidence that sequence acceleration is associated with turnover of regulatory function. Detailed study of one acceleration event in an enhancer of HES1 revealed that sequence evolution led to a new activity domain in the developing limb that emerged concurrently with the evolution of digit reduction in hoofed mammals. Our results provide evidence that enhancer evolution has been a frequent contributor to regulatory innovation at conserved developmental signaling genes in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f787749d36ce9eb176cc01c130701746e5627c" target='_blank'>
              Evolutionary Innovations in Conserved Regulatory Elements Associate With Developmental Genes in Mammals
              </a>
            </td>
          <td>
            Severin Uebbing, Acadia A. Kocher, Marybeth Baumgartner, Yu Ji, Suxia Bai, Xiaojun Xing, Timothy Nottoli, James P Noonan
          </td>
          <td>2024-09-20</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is a global challenge, and genetics play a major role in onsets and in designing the next generation of cancer therapies. One of the key approaches is the downregulation of genes through RNA interference (RNAi) which has been proposed as a promising tool for controlling cancer-causing genes by employing a complementary base-pairing mechanism. Crucial tools in RNAi; small interfering RNA (siRNA), and microRNA (miRNA) have been extensively utilized in cancer therapy. Despite their discovery nearly two decades ago, only a handful of RNAi-based products have recently gained approval from regulatory authorities like the FDA, principally due to inherent limitations. The efficacy of RNAi therapies is significantly hindered by barriers related to absorption, distribution, metabolism, and excretion, leading to rapid elimination from the systemic circulation before reaching the cytosol of targeted cells. Nevertheless, RNAi therapeutics hold immense promise in various diseases, especially in various types of cancer. Overcoming these challenges demands the design of suitable drug delivery systems and the implementation of strategies aimed at enhancing pharmacokinetic parameters associated with RNAi therapies. Nanotechnology-based vehicles such as polymer-based nanoparticles, lipid nanoparticles, liposomes, dendrimers, and inorganic nanoparticles may serve as efficient carriers for targeting RNAi therapies to their desired sites. This review discusses the recent advances in nanotechnology strategies for RNAi delivery, with the overarching objective of facilitating effective targeting and gene silencing for the advancement of RNAi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d3ac4d02e434d3ba852ff719ff6ff9da7bfae5" target='_blank'>
              Recent Advancements in Cancer RNA-interference Therapy with Nanotechnology Strategies
              </a>
            </td>
          <td>
            A. Abosalha, P. Islam, J. Boyajian, A. Kassab, S. Makhlouf, Madison Santos, Satya Prakash
          </td>
          <td>2024-10-09</td>
          <td>Journal of Cancer Biomoleculars and Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a29762d25b9a77c8fd5e2a5d7d19911938e8b1" target='_blank'>
              Integrate and conquer: pan-cancer proteogenomics uncovers cancer vulnerabilities and therapeutic opportunities
              </a>
            </td>
          <td>
            Debomita Chakraborty, Rossana Romero, K. Rajalingam
          </td>
          <td>2024-10-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Epigenetic silencing is essential for regulating gene expression and cellular diversity in eukaryotes. While transposable elements (TEs) are mostly silenced with DNA and H3K9 methylation, gene silencing is mediated by H3K27me3, an epigenetic mark deposited by the Polycomb repressive complex 2 (PRC2). Despite the major role epigenetic silencing plays in the development of multicellular eukaryotes, little is known about how epigenetically-controlled regulatory networks were shaped over evolutionary time. Here, we analyse epigenomes from a diverse group of species across the green lineage and infer the chronological epigenetic recruitment of genes that occurred during land plant evolution. We first reveal the nature of plant heterochromatin in the unicellular green microalga Chlorella sorokiniana and identify a substantial number of genes marked with H3K27me3, highlighting the deep origin of PRC2-regulated genes in the green lineage. By incorporating genomic phylostratigraphy, we show how genes of differing evolutionary age are partitioned into distinct epigenetic states in plants, with evolutionarily young genes incorporated into developmental programs controlled by H3K9 methylation in Arabidopsis. We further reveal a major wave of PRC2 recruitment to genes that emerged during land terrestrialisation and flowering plant evolution, and identify an ancestral PRC2 network with a shared functional topology in green algae through to land plants, providing a glimpse of the earliest types of genes regulated by PRC2 during the course of plant evolution. Finally, we analyse the potential regulation of these ancestral PRC2 target genes and find a strong enrichment of motifs bound by ancient AP2/ERF transcription factors (TFs) known to interact with PRC2, which we hypothesise were key determinants in shaping some of the first gene regulatory networks controlled by PRC2 in plants. Our data thus reveal pivotal epigenetic adaptations that occurred during a significant period in the evolutionary history of plants, which likely contributed to key regulatory innovations that influenced major morphological and developmental change into the modern-day. More broadly, our findings offer insight into the evolutionary dynamics and molecular triggers that drive the adaptation and elaboration of epigenetic regulation, laying the groundwork for its future consideration in other major eukaryotic lineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/578bce57cf922c3e4cbf1c7dcaf1f5ced20825df" target='_blank'>
              The expansion and diversification of epigenetic regulatory networks underpins major transitions in the evolution of land plants
              </a>
            </td>
          <td>
            Romy Petroll, Ranjith K. Papareddy, Rafal Krela, Alice Laigle, Quentin Rivière, Kateřina Bišová, I. Mozgová, Michael Borg
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dosage-sensitive genes have characteristic patterns of evolution that include being refractory to small-scale duplication, depleted on human benign copy number variants (CNVs) and enriched on pathogenic CNVs. This intolerance to copy number change is likely due to an expression constraint that exists in one or more tissues. While genomic copy number changes alter the encompassed genes’ expression across all tissues, expression quantitative trait loci (eQTLs) –genomic regions harbouring sequence variants that influence the expression level of one or more genes– can act in a tissue-specific manner. In this work we examine expression variation of presumed dosage-sensitive and non-dosage-sensitive genes to discover how the locus duplicability constraints translate into gene expression constraints. Here we test the hypothesis that expression changes due to the presence of eQTLs acting in unconstrained tissues will not be deleterious and thus allow dosage-sensitive genes to vary expression while obeying constraints in other tissues. Using eQTLs across 48 human tissues from The Genotype-Tissue Expression (GTEx) project, we find that dosage-sensitive genes are enriched for being affected by eQTLs and that the eQTLs affecting dosage-sensitive genes are biased towards having narrow tissue-specificity with these genes having fewer eQTL-affected tissues than non-dosage-sensitive genes. Additionally, we find that dosage-sensitive genes are depleted for being affected by broad tissue breadth eQTLs, likely due to the increased chance of these eQTLs conflicting with expression constraints and being removed by purifying selection. These patterns suggest that dosage-sensitivity shapes the evolution of these genes by precluding copy number evolution and restricting their evolutionary trajectories to changes in expression regulation compatible with their functional constraints. Thus deeper interpretation of the patterns of constraints can be informative of the temporal or spatial location of the gene dosage sensitivity and contribute to our understanding of functional genomics. Author summary Gene duplication is an important and powerful evolutionary force that is responsible for the expansion of the coding capacity of genomes ultimately resulting in great genetic novelty. However, the opportunity for this evolutionary change can be limited by dosage constraints on some genes, meaning they are not normally duplicable, except in a balanced, whole genome event. This results in important, biologically relevant, differences between genes that are retained from whole genome duplication events versus those retained from small scale duplications, especially in terms of dosage sensitivity. We explored how the different dosage sensitivity in these sets of genes relates to quantitative expression variation present in populations. We found that while dosage-sensitive genes are more likely to have their expression levels influenced by genetic variation, these changes are often specific a small number of tissues. In contrast, genes that are less sensitive to dosage changes show greater variation in expression levels across multiple tissues. Our findings suggest that dosage-sensitive genes evolve through fine-tuned adjustments in their expression levels in specific tissues, thus bypassing constraints operating on other tissues. This understanding sheds light on how dosage-sensitive genes evolve and could have implications for understanding human diseases caused by these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05cd1cac23ae98b3aac115ebfd59b3d377a2304" target='_blank'>
              Evolution of dosage-sensitive genes by tissue-restricted expression changes
              </a>
            </td>
          <td>
            Alan M. Rice, Yuanshuo Li, Pauric Donnelly, A. McLysaght
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Novel ZNF genes, such as ZNF671, that are located on chromosome 19q13 are known to be hypermethylated at a high frequency in HNSCC as well as in other epithelial solid tumors. Their function is largely unknown. Results: Here, we show that ZNF671 is epigenetically silenced in HNSCC primary tumors compared to matched adjacent normal tissue. Moreover, low expression of ZNF671 is significantly associated with decreased survival in HNSCC patients. Over-expression of ZNF671 in UM-SCC-1 oral cancer cells resulted in a significant reduction in tumor cell mobility and invasion compared to the empty-vector control cells. Transcriptomic analysis showed that ZNF671 re-expression resulted in a significant decrease in the expression of a major oncogenic long non-coding RNA LINC00665. Conclusions: Together, these results suggest that epigenetic silencing of ZNF671 may activate multiple oncogenic signaling pathways via the resulting up-regulation of LINC00665.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ca89a64cea88b2e9933932b6aee162b10603a7" target='_blank'>
              ZNF671 Silencing Affects Signaling Pathways in Head and Neck Cancer via Activation of Oncogenic Non-Coding RNAs
              </a>
            </td>
          <td>
            Kendra Smith, Rufa'i Umar Zubair, Richard V Smith, Stelby Augustine, Nicholas F. Schlecht, Thomas J. Ow, M. Prystowsky, T. Belbin
          </td>
          <td>2024-10-29</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA repair activity diminishes with age and genetic mutations, leading to a significantly increased risk of cancer and other diseases. Upregulating the DNA repair system has emerged as a potential strategy to mitigate disease susceptibility while minimizing cytotoxic side effects. However, enhancing DNA repair activity presents significant challenges due to the inherent inefficiency in activator screening processes. Additionally, pinpointing a critical target that can effectively upregulate overall repair processes is complicated as the available information is somewhat sporadic. In this review, we discuss potential therapeutic targets for upregulating DNA repair pathways, along with the chemical structures and properties of reported small-molecule activators. We also elaborate on the diverse mechanisms by which these targets modulate repair activity, highlighting the critical need for a comprehensive understanding to guide the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf64add08ecc3d0be655106d39c828b55e519b54" target='_blank'>
              Therapeutic upregulation of DNA repair pathways: strategies and small molecule activators.
              </a>
            </td>
          <td>
            Juhyung Song, Cheoljun Park, Francis Cabanting, Y. Jun
          </td>
          <td>2024-10-11</td>
          <td>RSC medicinal chemistry</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature. Methods Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature. Results A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo. Conclusion This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Synovial Sarcomas (SS) are characterized by the presence of the SS18::SSX fusion gene, which protein product induce chromatin changes through remodeling of the BAF complex. To elucidate the genomic events that drive phenotypic diversity in SS, we performed RNA and targeted DNA sequencing on 91 tumors from 55 patients. Our results were verified by proteomic analysis, public gene expression cohorts and single-cell RNA sequencing. Transcriptome profiling identified three distinct SS subtypes resembling the known histological subtypes: SS subtype I and was characterized by hyperproliferation, evasion of immune detection and a poor prognosis. SS subtype II and was dominated by a vascular-stromal component and had a significantly better outcome. SS Subtype III was characterized by biphasic differentiation, increased genomic complexity and immune suppression mediated by checkpoint inhibition, and poor prognosis despite good responses to neoadjuvant therapy. Chromosomal abnormalities were an independent significant risk factor for metastasis. KRT8 was identified as a key component for epithelial differentiation in biphasic tumors, potentially controlled by OVOL1 regulation. Our findings explain the histological grounds for SS classification and indicate that a significantly larger proportion of patients have high risk tumors (corresponding to SS subtype I) than previously believed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5e414a7a3b033cc20bf7f0e410a6830f2ad049" target='_blank'>
              Molecular Profiling Defines Three Subtypes of Synovial Sarcoma.
              </a>
            </td>
          <td>
            Yi Chen, Yanhong Su, Xiaofang Cao, I. Siavelis, I. Leo, Jianming Zeng, P. Tsagkozis, A. Hesla, A. Papakonstantinou, Xiao Liu, Wen-Kuan Huang, Binbin Zhao, Cecilia Haglund, M. Ehnman, Henrik Johansson, Yingbo Lin, Janne Lehtiö, Yifan Zhang, O. Larsson, Xuexin Li, Felix Haglund de Flon
          </td>
          <td>2024-09-10</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) remains a disease with a poor prognosis despite the advances in therapies. NSCLC with actionable oncogenic alterations represent a subgroup of diseases for which tyrosine kinase inhibitors (TKIs) have shown relevant and robust impact on prognosis, both in early and advanced stages. While the introduction of powerful TKIs increases the ratio of potentially curable patients, the disease does develop resistance over time through either secondary mutations or bypass activating tracks. Therefore, new treatment strategies are being developed to either overcome this inevitable resistance or to prevent it, and proteolysis targeting chimera agents (PROTACs) are among them. They consist of two linked molecules that bind to a target protein and an E3 ubiquitin ligase that causes ubiquitination and degradation of proteins of interest. In this paper, we review the rationale for PROTAC therapy and the current development of PROTACs for oncogene-addicted lung cancer. Moreover, we critically analyze the strengths and limitations of this promising technique that may help pave the way for future perspectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/866e44b9a44363bae9549987792f66d0f87f7e0e" target='_blank'>
              Proteolysis Targeting Chimera Agents (PROTACs): New Hope for Overcoming the Resistance Mechanisms in Oncogene-Addicted Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            N. Cordani, Daniele Nova, Luca Sala, M. Abbate, F. Colonese, D. Cortinovis, S. Canova
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Gliomas are neoplasms of the central nervous system that originate in glial cells. The genetic characteristics of this type of neoplasm are the loss of function of tumor suppressor genes such as TP53 and somatic mutations in genes such as IDH1/2. Additionally, in clinical cases, de novo single nucleotide polymorphisms (SNP) are reported, of which their pathogenicity and their effects on the function and stability of the protein are known. Methodology: Non-synonymous SNPs were analyzed for their structural and functional effect on proteins using a set of bioinformatics tools such as SIFT, PolyPhen-2, PhD-SNP, I-Mutant 3.0, MUpro, and mutation3D. A structural comparison between normal and mutated residues for disease-associated coding SNPs was performed using TM-aling and the SWISS MODEL. Results: A total of 13 SNPs were obtained for the TP53 gene, 1 SNP for IDH1, and 1 for IDH2, which would be functionally detrimental and associated with disease. Additionally, these changes compromise the structure and function of the protein; the A161S SNP for TP53 that has not been reported in any databases was classified as detrimental. Conclusions: All non-synonymous SNPs reported for TP53 were in the region of the deoxyribonucleic acid (DNA) binding domain and had a great impact on the function and stability of the protein. In addition, the two polymorphisms detected in IDH1 and IDH2 genes compromise the structure and activity of the protein. Both genes are related to the development of high-grade gliomas. All the data obtained in this study must be validated through experimental approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6597e0eb6fecbfff6b6800003b6d0c3a0127247" target='_blank'>
              Bioinformatic Characterization of the Functional and Structural Effect of Single Nucleotide Mutations in Patients with High-Grade Glioma
              </a>
            </td>
          <td>
            Sara Vélez Gómez, Juliana María Martínez Garro, León Darío Ortiz Gómez, Jorge Emilio Salazar Flórez, Fernando P Monroy, R. G. Peláez Sánchez
          </td>
          <td>2024-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome-wide CRISPR screens have emerged as powerful tools for uncovering the genetic underpinnings of diverse biological processes. Incisive screens often depend on directly measuring molecular phenotypes, such as regulated gene expression changes, provoked by CRISPR-mediated genetic perturbations. Here, we provide quantitative measurements of transcriptional responses in human cells across genome-scale perturbation libraries by coupling CRISPR interference (CRISPRi) with barcoded expression reporter sequencing (CiBER-seq). To enable CiBER-seq in mammalian cells, we optimize the integration of highly complex, barcoded sgRNA libraries into a defined genomic context. CiBER-seq profiling of a nuclear factor kappa B (NF-κB) reporter delineates the canonical signaling cascade linking the transmembrane TNF-alpha receptor to inflammatory gene activation and highlights cell-type-specific factors in this response. Importantly, CiBER-seq relies solely on bulk RNA sequencing to capture the regulatory circuit driving this rapid transcriptional response. Our work demonstrates the accuracy of CiBER-seq and its potential for dissecting genetic networks in mammalian cells with superior time resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f33a5d37dba6ea92909caf57b17709293f40346" target='_blank'>
              CRISPRi with barcoded expression reporters dissects regulatory networks in human cells
              </a>
            </td>
          <td>
            Jinyoung Kim, Ryan Y Muller, Eliana R. Bondra, Nicholas T. Ingolia
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has revolutionized cell biology by enabling the profiling of transcriptomes at a single-cell resolution, leading to important discoveries that have advanced our understanding of cellular and tissue heterogeneity, developmental trajectories, and disease progression. Despite these important advances, scRNA-seq is limited to measuring the transcriptome providing a partial view of cellular function. To address this limitation, multimodal scRNA-seq assays have emerged, allowing for the simultaneous measurement of RNA expression and protein. Intracellular Transcriptomic and Protein Sequencing (InTraSeq), a novel multimodal scRNA-seq technology described here, enables the concurrent measurement of mRNA, surface markers, cytoplasmic proteins, and nuclear proteins within individual cells through oligo-barcoded antibodies. This method offers a comprehensive approach to studying cellular function by combining RNA and protein profiling from the same sample and utilizing a relatively simple protocol. The InTraSeq method enables researchers to expand their view of critical intracellular protein expression including post-translational modifications (PTMs) and transcription factors, allowing for the identification of novel cellular subtypes and states that may be obscured by RNA-based analyses alone. This is particularly valuable in understanding the heterogeneity of cell populations and identifying distinct functional states. In this report, we used InTraSeq to characterize the complex cellular states and regulatory mechanisms during Th17 cell differentiation. We simultaneously profiled RNA and protein expression in over 85,000 cells, capturing transcriptional changes, changes in protein expression and the dynamics of signaling pathways at a high resolution. Our results revealed novel insights into Th17 cell differentiation, including the identification of key regulatory factors and their target genes. By simultaneously measuring mRNA, extra and intra-cellular proteins, signaling proteins, and PTMs, InTraSeq offers a comprehensive understanding of cellular processes and enables the identification of novel regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281131467bb5dc8461ee132b2238aeb8b1cafd01" target='_blank'>
              InTraSeq: A Multimodal Assay that Uncovers New Single-Cell Biology and Regulatory Mechanisms
              </a>
            </td>
          <td>
            Majd M. Ariss, Linglin Huang, Xiaokai Ding, Shivani Sheth, Tyler Levy, Jeremy Fisher, Jean Loebelenz, Keith Arlotta, Karen Dixon, Roberto Polakiewicz, V. K. Kuchroo, Sean A. Beausoleil
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The relationship between BRAF-V600E mutations, mitochondrial DNA amplification and microsatellite instability-high (MSI-H) in colorectal cancer (CRC) has yet to be fully elucidated. The aim of the present study was to assess the association between the MSI status and BRAF-V600E gene mutations/clinicopathological features/mitochondrial DNA amplification in CRC. A non-interventional study analysis was performed using the clinicopathological features of 455 patients with CRC. Immunohistochemistry was used to evaluate four mismatch repair proteins (MutS homolog 2, MutS homolog 6, MutL homolog 1 and postmeiotic segregation increased 2), Ki-67 index, and programmed cell death protein 1 (PD-1) and programmed cell death-ligand 1 (PD-L1) expression. Additionally, PCR coupled with capillary electrophoresis were used to ascertain the MSI status. Moreover, amplification refractory mutation system-PCR was used to detect BRAF-V600E gene mutation and fluorescence in situ hybridization analysis was used to assess mitochondrial DNA. A total of 455 patients were divided into the MSI high (MSI-H) group (n=52) and microsatellite stability (MSS) group (n=403) based on their MSI status. Compared with the results of immunohistochemistry of four mismatch repair proteins, the consistency rate between mismatch repair protein deficiency and MSI was 94.23%. There were significant differences in PD-L1, primary tumor site, clinical stage, degree of differentiation, tumor size, lymph node metastasis and the occurrence of multiple primary tumors between the MSI-H group and MSS group (P<0.05 or P<0.001). However, there were no significant differences for sex, age, PD-1, Ki-67 expression and BRAF-V600E. The 24–60-month survival rate of the patients in the MSI-H group was significantly higher than that of those in the MSS group (P<0.05). Furthermore, the number of mitochondrial DNA was significantly amplified in the MSI-H group. In conclusion, the present study demonstrated that the combined detection of PD-L1 and MSI in patients with CRC can provide more accurate and effective guidance for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5feaa7383704c34d700372d523136046568f74f8" target='_blank'>
              Association between microsatellite instability status, clinicopathological features and mitochondrial DNA amplification in patients with colorectal cancer
              </a>
            </td>
          <td>
            Junmi Lu, Hong Tan, Tao Guo, Xi Chen, Zhongyi Tong
          </td>
          <td>2024-09-26</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a6c27093146f06f4b8893e39507755c64cdc82" target='_blank'>
              BAP1 regulates HSF1 activity and cancer immunity in pancreatic cancer
              </a>
            </td>
          <td>
            Weiwei Yuan, Qiyue Zhang, Yuhan Zhao, Wentao Xia, Shilin Yin, Xueyi Liang, Taoyu Chen, Gaofeng Li, Yanshen Liu, Zhiqiang Liu, Jinxi Huang
          </td>
          <td>2024-09-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

MicroRNAs (miRNAs) are small, non-coding RNAs that play a crucial role in post-transcriptional
gene regulation by modulating mRNA stability and translation. These evolutionarily
conserved molecules undergo processing by ribonucleases Drosha and Dicer, ultimately joining
the RNA-induced silencing complex to silence gene expression. Among them, miR-23a, located
on chromosome 19, is significant for its roles in cell growth, differentiation, and various diseases,
including cancer. Depending on the cancer type, miR-23a can function as both an oncogene and a
tumour suppressor. While its overexpression often correlates with aggressive tumours, miR-23a
holds promise as a biomarker for early cancer detection and a therapeutic target. Its diverse functions
in cancer include promoting tumour growth and hindering immune responses, highlighting
its potential for personalized medicine. Beyond cancer, miR-23a is involved in blood sugar regulation,
insulin resistance, muscle atrophy, and other diseases. It modulates pathways in type 2 diabetes
mellitus, muscle atrophy, leukemia, epilepsy, and osteoarthritis. This paper aims to comprehensively
analyze the roles of miR-23a in cancer and other diseases, focusing on its regulatory
mechanisms, target genes, and pathways. It also evaluates the potential of miR-23a as a biomarker
and therapeutic target. Despite its significance, research gaps remain, particularly in understanding
the interactions of miR-23a with other miRNAs and the detailed mechanisms underlying its
role in various diseases. More research into miR-23a clustering and how it works with other miRNAs
could help us learn more about it and find better ways to use it to diagnose and treat diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5ae944098ccfea82803a28707ded6a2a685f935" target='_blank'>
              The Multifaceted Role of miR-23a in Cancer and Disease Progression
              </a>
            </td>
          <td>
            Ruping Deng, Dongmei Nong, F. Liew
          </td>
          <td>2024-10-15</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genomic copy-number variations (CNVs) that can cause neurodevelopmental disorders often encompass many genes, which complicates our understanding of how individual genes within a CNV contribute to pathology. MECP2 duplication syndrome (MDS or MRXSL in OMIM; OMIM#300260) is one such CNV disorder caused by duplications spanning methyl CpG-binding protein 2 (MECP2) and other genes on Xq28. Using an antisense oligonucleotide (ASO) to normalize MECP2 dosage is sufficient to rescue abnormal neurological phenotypes in mouse models overexpressing MECP2 alone, implicating the importance of increased MECP2 dosage within CNVs of Xq28. However, because MDS CNVs span MECP2 and additional genes, we generated human neurons from multiple MDS patient-derived induced pluripotent cells (iPSCs) to evaluate the benefit of using an ASO against MECP2 in a MDS human neuronal context. Importantly, we identified a signature of genes that is partially and qualitatively modulated upon ASO treatment, pinpointed genes sensitive to MeCP2 function, and altered in a model of Rett syndrome, a neurological disorder caused by loss of MeCP2 function. Furthermore, the signature contained genes that are aberrantly altered in unaffected control human neurons upon MeCP2 depletion, revealing gene expression programs qualitatively sensitive to MeCP2 levels in human neurons. Lastly, ASO treatment led to a partial rescue of abnormal neuronal morphology in MDS neurons. All together, these data demonstrate that ASOs targeting MECP2 benefit human MDS neurons. Moreover, our study establishes a paradigm by which to evaluate the contribution of individual genes within a CNV to pathogenesis and to assess their potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/204ab0081ef3960da9757eb616c690c4273ee78b" target='_blank'>
              Modeling antisense oligonucleotide therapy in MECP2 duplication syndrome human iPSC-derived neurons reveals gene expression programs responsive to MeCP2 levels.
              </a>
            </td>
          <td>
            Sameer S. Bajikar, Yehezkel Sztainberg, Alexander J. Trostle, Harini P Tirumala, Ying-Wooi Wan, Caroline L Harrop, J. Bengtsson, Claudia M B Carvalho, D. Pehlivan, Bernhard Suter, J. Neul, Zhandong Liu, P. Jafar‐nejad, Frank Rigo, H. Zoghbi
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="To dissect variant-function relationships in the KRAS oncoprotein, we performed deep mutational scanning (DMS) screens for both wild-type and KRASG12D mutant alleles. We defined the spectrum of oncogenic potential for nearly all possible KRAS variants, identifying several novel transforming alleles and elucidating a model to describe the frequency of KRAS mutations in human cancer as a function of transforming potential, mutational probability, and tissue-specific mutational signatures. Biochemical and structural analyses of variants identified in a KRASG12D second-site suppressor DMS screen revealed that attenuation of oncogenic KRAS can be mediated by protein instability and conformational rigidity, resulting in reduced binding affinity to effector proteins, such as RAF and PI3-kinases, or reduced SOS-mediated nucleotide exchange activity. These studies define the landscape of single amino acid alterations that modulate the function of KRAS, providing a resource for the clinical interpretation of KRAS variants and elucidating mechanisms of oncogenic KRAS inactivation for therapeutic exploitation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a23afebcd7cdf77fb0cc51e7cfdf5ed4086508" target='_blank'>
              Comprehensive structure-function analysis reveals gain- and loss-of-function mechanisms impacting oncogenic KRAS activity
              </a>
            </td>
          <td>
            Jason J. Kwon, Julien Dilly, Shengwu Liu, Eejung Kim, Yuemin Bian, S. Dharmaiah, Timothy H Tran, Kevin S. Kapner, S. H. Ly, Xiaoping Yang, Dana Rabara, Timothy Waybright, Andrew O. Giacomelli, Andrew L. Hong, Sean Misek, Belinda Wang, Arvind Ravi, John G Doench, R. Beroukhim, Christopher T. Lemke, Kevin M. Haigis, Dominic Esposito, David E. Root, D. Nissley, Andrew G Stephen, Frank McCormick, D. Simanshu, William C. Hahn, Andrew J. Aguirre
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>146</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/145eeb292c624be8f86f64a77055ade9f8ec3328" target='_blank'>
              Tumor microenvironment as a complex milieu driving cancer progression: a mini review.
              </a>
            </td>
          <td>
            Zheng Li, Jing Li, Xiaolei Bai, Xufeng Huang, Qi Wang
          </td>
          <td>2024-09-28</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="
 An often-overlooked component in our cancer genomics workflows is the reference genome. While most genomics analytical tools and findings are based on reference builds GRCh37 and GRCh38, these references are largely Eurocentric, and their representativeness among diverse populations has not been studied comprehensively. A previously published study conducted deep whole genome sequencing of over 900 individuals of African ancestry, where they revealed ∼296 million unique base pairs, assembled into 125,715 African Pan Genome (APG) contig sequences that are not represented in the reference genome. To further characterize these APG contig sequences, we sought to determine the epigenetic and expression potential of these sequences. We have previously reported preliminary efforts to characterize the unexplored epigenetic potential across the 125,715 APG contig sequences, where we have utilized Emboss CpG plot for CpG island prediction. Across all contig sequences, we identified 8,353 potential CpG islands across 6,352 contig sequences, where contigs had 1 to 9 predicted CpG islands. Taken together ∼5% of all APG contigs contained at least 1 predicted CpG island. To determine the expression potential of these contigs, we mapped RNAseq reads from an African ancestry-enriched triple negative breast cancer cohort to APG contigs that previously failed to map to the reference genome. Approximately 9% of APG contigs had transcriptomic reads mapping, and 1106 APG contigs had both reads mapping and predicted CpG islands. Given our RNAseq mapping from our African ancestry enriched cohort, we have begun to explore differential expression measures among contigs by African ancestry population groups, highlighting the potential for differential regulation at these understudied loci. Since the publication of the APG contig sequences, the recent telomere-to-telomere (T2T) reference has filled the gaps of GRCh37 and GRCh38, however, is even less diverse. With T2T containing novel sequences that were not present at the time the APG contigs were assembled, we performed BLASTn queries of APG contigs to the T2T genome, where preliminary analysis shows ∼17% fail to map to T2T. For those with BLASTn mapping results, we are currently assessing the coverage and identity of the mapping results to evaluate the alignment. In conclusion, these APG contig sequences harbor potential epigenetic regulatory function and expressed sequences not represented in our standard reference genomes. While some APG contig sequences are represented in the newer T2T genome, preliminary investigations show that these APG contig sequences map to the T2T genome in regions novel from GRCh37 and GRCh38. As we continue to explore these APG contig sequences, we may find ancestry-specific regulatory mechanisms not yet described.
 Citation Format: Rachel Martini, Kyriaki Founta, Sebastian Maurice, Jason White, Lisa Newman, Onyinye Balogun, Nyasha Chambwe, Melissa Davis. Epigenetic insights and gene expression in African pan genome contig sequences [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C093.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204ffa4a287b7dff49e752eaa867aa52631ce73" target='_blank'>
              Abstract C093: Epigenetic insights and gene expression in African pan genome contig sequences
              </a>
            </td>
          <td>
            R. Martini, Kyriaki Founta, Sebastian Maurice, Jason White, Lisa Newman, Onyinye D. Balogun, N. Chambwe, Melissa B. Davis
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The underlying gene regulatory networks (GRN) that govern leukemia stem cells (LSC) in acute myeloid leukemia (AML) and hematopoietic stem cells (HSC) are not well understood. Here, we identified GRNs by integrating gene expression (GE) and chromatin accessibility data derived from functionally defined cell populations enriched for HSC and LSC. We analyzed n=32 LSC+ and n=32 LSC-cell fractions from n=22 AML patients, along with n=7 stem and n=10 progenitor enriched cell populations sorted from human umbilical cord blood (hUCB), producing a database of n≈17,000 transcription factor (TF) regulatory interactions for hUCB-HSPC and AML. We developed an iterative algorithm that associates the degree of chromatin openness with TF binding preferences, and the GE of candidate TF and target genes within 100kb upstream of transcription start sites. A putative regulatory structure was found to be enriched in HSC-enriched cell populations, comprising TF-target gene interactions between ETS1, EGR1, RUNX2, and ZNF683 oriented in a self-reinforcing configuration. A regulatory loop comprising FOXK1 and MEIS1, rather than the 4-factor HSC subnetwork, was detected in the LSC-specific GRN. The core HSC and LSC TF networks were extended using protein-protein interaction (PPI) data to determine connectivity with interacting genes whose expression strongly associated with LSC/HSC frequency estimates, producing a database of n=103,516 PPI target pathways. The effect of perturbing genes along the identified pathways on functional HSC and LSC frequency was predicted based on statistical regression analyses. To validate GRN predictions, we used pharmacologic and CRISPR targeting, in addition to re-examining published functional data associated with several network nodes that were predicted to impact stemness. Notably, we found that inhibition of CDK6 in AML samples markedly reduced LSC numbers as assessed in de novo serial xenotransplantation studies (fold change ≈ 10), as predicted by the LSC GRN model. Additionally, in-house CRISPR-based knockdown of ETS1 resulted in a significant decrease in HSC quiescence-associated microRNA-126 expression, and increased HSC frequency. Taken together, our models provide a comprehensive view of the underlying regulatory structures governing functional human HSC and LSC. This approach has translational potential as it can be used as a high-throughput in-silico screening tool for the systematic identification of gene targets for LSC elimination and HSC expansion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41e7b578bd41a7f8073095de491190663b9cecb5" target='_blank'>
              Distinct gene regulatory networks govern hematopoietic and leukemia stem cells
              </a>
            </td>
          <td>
            Boyang Zhang, Alexander Murison, Angelica Varesi, Naoya Takayama, Liqing Jin, Nathan Mbong, Erwin M. Schoof, Stephanie Z. Xie, Amanda Mitchell, J. Etchin, A. T. Look, Mathieu Lupien, Mark D. Minden, J. C. Wang, Peter W. Zandstra, Elvin Wagenblast, John E. Dick, Stanley W.K. Ng
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Telomeres—special DNA–protein structures at the ends of linear eukaryotic chromosomes—define the proliferation potential of cells. Extremely short telomeres promote a DNA damage response and cell death to eliminate cells that may have accumulated mutations after multiple divisions. However, telomere elongation is associated with the increased proliferative potential of specific cell types, such as stem and germ cells. This elongation can be permanent in these cells and is activated temporally during immune response activation and regeneration processes. The activation of telomere lengthening mechanisms is coupled with increased proliferation and the cells’ need for energy and building resources. To obtain the necessary nutrients, cells are capable of finely regulating energy production and consumption, switching between catabolic and anabolic processes. In this review, we focused on the interconnection between metabolism programs and telomere lengthening mechanisms during programmed activation of proliferation, such as in germ cell maturation, early embryonic development, neoplastic lesion growth, and immune response activation. It is generally accepted that telomere disturbance influences biological processes and promotes dysfunctionality. Here, we propose that metabolic conditions within proliferating cells should be involved in regulating telomere lengthening mechanisms, and telomere length may serve as a marker of defects in cellular functionality. We propose that it is possible to reprogram metabolism in order to regulate the telomere length and proliferative activity of cells, which may be important for the development of approaches to regeneration, immune response modulation, and cancer therapy. However, further investigations in this area are necessary to improve the understanding and manipulation of the molecular mechanisms involved in the regulation of proliferation, metabolism, and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ab06d09eb5b7b71fe7be1c13b67665bab6c417" target='_blank'>
              Telomere Reprogramming and Cellular Metabolism: Is There a Link?
              </a>
            </td>
          <td>
            M. Rubtsova, Denis A. Nikishin, M. Y. Vyssokikh, Maria S. Koriagina, Andrey V. Vasiliev, O. A. Dontsova
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is widely prevalent and represents a significant contributor to global cancer‑related mortality. There remains a pressing demand for advancements in CRC treatment modalities. The E3 ubiquitin ligase is a critical enzyme involved in modulating protein expression levels via posttranslational ubiquitin‑mediated proteolysis, and it is reportedly involved in the progression of various cancers, making it a target of recent interest in anticancer therapy. In the present study, using comprehensive expression analysis involving spatial transcriptomic analysis with single‑cell RNA sequencing in clinical CRC datasets, the ubiquitin‑associated protein Shank‑associated RH domain interactor (SHARPIN) was identified, located on amplified chromosome 8q, which could promote CRC progression. SHARPIN was found to be upregulated in tumor cells, with elevated expression observed in tumor tissues. This heightened expression of SHARPIN was positively associated with lymphatic invasion and served as an independent predictor of a poor prognosis in patients with CRC. In vitro and in vivo analyses using SHARPIN‑overexpressing or ‑knockout CRC cells revealed that SHARPIN overexpression upregulated MDM2, resulting in the downregulation of p53, while SHARPIN silencing or knockout downregulated MDM2, leading to p53 upregulation, which affects cell cycle progression, tumor cell apoptosis and tumor growth in CRC. Furthermore, SHARPIN was found to be overexpressed in several cancer types, exerting significant effects on survival outcomes. In conclusion, SHARPIN represents a newly identified novel gene with the potential to promote tumor growth following apoptosis inhibition and cell cycle progression in part by inhibiting p53 expression via MDM2 upregulation; therefore, SHARPIN represents a potential therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fea138b30e525fb1b159c88f092baedda580cc" target='_blank'>
              SHARPIN is a novel gene of colorectal cancer that promotes tumor growth potentially via inhibition of p53 expression.
              </a>
            </td>
          <td>
            Yusuke Nakano, Takaaki Masuda, Takeharu Sakamoto, Noritaka Tanaka, Taro Tobo, M. Hashimoto, Takanari Tatsumi, Hideyuki Saito, Junichi Takahashi, Kensuke Koike, Tadashi Abe, Yuki Ando, Yuki Ozato, Kiyotaka Hosoda, Kosuke Hirose, Satoshi Higuchi, Tomohiko Ikehara, Y. Hisamatsu, Takeo Toshima, Yusuke Yonemura, T. Ogino, Mamoru Uemura, H. Eguchi, Y. Doki, K. Mimori
          </td>
          <td>2024-10-21</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Simple Summary When oncogenes, which are genes that can cause cancer, become active, they disrupt normal cell processes, especially DNA replication. This disruption is known as replication stress (RS), in which the DNA copying process is stalled or damaged. To prevent cancer, cells usually have a defense system called the DNA damage response (DDR), which can prevent damaged cells from turning into cancer. However, some cells manage to survive this stress by developing replication stress tolerance (RST). These cells can continue to grow, leading to genomic instability (GIN), which is a key feature of cancer. GIN causes various genetic changes that make cells more likely to become cancerous and more difficult to treat. This review explains how oncogenes cause RS and how cells cope with it, leading to the development of cancer. Understanding these processes can help in developing new cancer treatments. Abstract Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b40c6e97d270349f6f8663c159874a551edaef3" target='_blank'>
              Transcriptional regulation by MYC: an emerging new model.
              </a>
            </td>
          <td>
            S. T. Jakobsen, Rasmus Siersbæk
          </td>
          <td>2024-10-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="T cell receptor (TCR) engagement causes a global cellular response that entrains signaling pathways, cell cycle regulation, and cell death. The molecular regulation of mRNA translation in these processes is poorly understood. Using a whole-genome CRISPR screen for regulators of CD95 (FAS/APO-1)-mediated T cell death, we identified AMBRA1, a protein previously studied for its roles in autophagy, E3 ubiquitin ligase activity, and cyclin regulation. T cells lacking AMBRA1 resisted FAS-mediated cell death by down-regulating FAS expression at the translational level. We show that AMBRA1 is a vital regulator of ribosome protein biosynthesis and ribosome loading on select mRNAs, whereby it plays a key role in balancing TCR signaling with cell cycle regulation pathways. We also found that AMBRA1 itself is translationally controlled by TCR stimulation via the CD28-PI3K-mTORC1-EIF4F pathway. Together, these findings shed light on the molecular control of translation after T cell activation and implicate AMBRA1 as a translational regulator governing TCR signaling, cell cycle progression, and T cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4853edfbfe0d545464ba66a1aa91bc944e6cbbc" target='_blank'>
              AMBRA1 controls the translation of immune-specific genes in T lymphocytes.
              </a>
            </td>
          <td>
            Simone Gottlieb, Wanjing Shang, Deji Ye, S. Kubo, P. Jiang, Samantha Shafer, Leilei Xu, Lixin Zheng, Ann Y Park, Jian Song, Waipan Chan, Zhiqin Zeng, Tingyan He, Benjamin Schwarz, Björn Häupl, T. Oellerich, Michael J. Lenardo, Yikun Yao
          </td>
          <td>2024-10-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises a diverse array of cells, both cancerous and non-cancerous, including stromal cells and immune cells. Complex interactions among these cells play a central role in driving cancer progression, impacting critical aspects such as tumor initiation, growth, invasion, response to therapy, and the development of drug resistance. While targeting the TME has emerged as a promising therapeutic strategy, there is a critical need for innovative approaches that accurately replicate its complex cellular and non-cellular interactions; the goal being to develop targeted, personalized therapies that can effectively elicit anti-cancer responses in patients. Microfluidic systems present notable advantages over conventional in vitro 2D co-culture models and in vivo animal models, as they more accurately mimic crucial features of the TME and enable precise, controlled examination of the dynamic interactions among multiple human cell types at any time point. Combining these models with next-generation technologies, such as bioprinting, single cell sequencing and real-time biosensing, is a crucial next step in the advancement of microfluidic models. This review aims to emphasize the importance of this integrated approach to further our understanding of the TME by showcasing current microfluidic model systems that integrate next-generation technologies to dissect cellular intra-tumoral interactions across different tumor types. Carefully unraveling the complexity of the TME by leveraging next generation technologies will be pivotal for developing targeted therapies that can effectively enhance robust anti-tumoral responses in patients and address the limitations of current treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee78b792b563d1d9bcfce83a027e1da609d44dc5" target='_blank'>
              Improving tumor microenvironment assessment in chip systems through next-generation technology integration
              </a>
            </td>
          <td>
            Daniela Gaebler, Stephanie J. Hachey, Christopher C. W. Hughes
          </td>
          <td>2024-09-25</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103c83e1829743ff7261d3403d3aa51e48f6908e" target='_blank'>
              Histone marks identify novel transcription factors that parse CAR-T subset-of-origin, clinical potential and expansion
              </a>
            </td>
          <td>
            S. Fiorenza, Y. Zheng, J. Purushe, T. J. Bock, J. Sarthy, D. Janssens, A. S. Sheih, E. L. Kimble, D. Kirchmeier, T. Phi, J. Gauthier, A. Hirayama, S. R. Riddell, Q. Wu, R. Gottardo, D. G. Maloney, J. Y. H. Yang, S. Henikoff, C. J. Turtle
          </td>
          <td>2024-09-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Directed evolution seeks to evolve target genes at a rate far exceeding the natural mutation rate, thereby endowing cellular and enzymatic properties with desired traits. In vivo continuous directed evolution achieves these purposes by generating libraries within living cells, enabling a continuous cycle of mutant generation and selection, enhancing the exploration of gene variants. Continuous evolution has become powerful tools for unraveling evolution mechanism and improving cellular and enzymatic properties. This review categorizes current continuous evolution into three distinct classes: non-targeted chromosomal, targeted chromosomal, and extra-chromosomal hypermutation approaches. It also compares various continuous evolution strategies based on different principles, providing a reference for selecting suitable methods for specific evolutionary goals. Furthermore, this review discusses the two primary limitations for further widespread application of in vivo continuous evolution, which are lack of general applicability and insufficient mutagenic capability. We envision that developing generally applicable mutagenic components and methods to enhance mutation rates for in vivo continuous evolution are promising future directions for wide range applications of continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76efdd9593a3f334b04aa2d1d64916396aef98c2" target='_blank'>
              Enhancing cellular and enzymatic properties through in vivo continuous evolution.
              </a>
            </td>
          <td>
            Weiran Chu, Yaxin Guo, Yaokang Wu, Xueqin Lv, Jianghua Li, Long Liu, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2024-09-09</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e5355a245efc577e7cf979feda1b1adefcc1f5" target='_blank'>
              Genome-wide CRISPR screens identify CLC-2 as a drug target for anti-herpesvirus therapy: tackling herpesvirus drug resistance.
              </a>
            </td>
          <td>
            Fayu Yang, Nan Wei, Shuo Cai, Jing Liu, Qingping Lan, Hao Zhang, Lu Shang, Bo Zheng, Mi Wang, Yingchun Liu, Lifang Zhang, Chenzhong Fei, Wu Tong, Changlong Liu, Ersheng Kuang, Guangzhi Tong, Feng Gu
          </td>
          <td>2024-10-12</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Characterizing DNA methylation patterns is important for addressing key questions in evolutionary biology, geroscience, and medical genomics. While costs are decreasing, whole-genome DNA methylation profiling remains prohibitively expensive for most population-scale studies, creating a need for cost-effective, reduced representation approaches (i.e., assays that rely on microarrays, enzyme digests, or sequence capture to target a subset of the genome). Most common whole genome and reduced representation techniques rely on bisulfite conversion, which can damage DNA resulting in DNA loss and sequencing biases. Enzymatic methyl sequencing (EM-seq) was recently proposed to overcome these issues, but thorough benchmarking of EM-seq combined with cost-effective, reduced representation strategies has not yet been performed. To do so, we optimized Targeted Methylation Sequencing protocol (TMS)—which profiles ∼4 million CpG sites—for miniaturization, flexibility, and multispecies use at a cost of ∼$80. First, we tested modifications to increase throughput and reduce cost, including increasing multiplexing, decreasing DNA input, and using enzymatic rather than mechanical fragmentation to prepare DNA. Second, we compared our optimized TMS protocol to commonly used techniques, specifically the Infinium MethylationEPIC BeadChip (n=55 paired samples) and whole genome bisulfite sequencing (n=6 paired samples). In both cases, we found strong agreement between technologies (R² = 0.97 and 0.99, respectively). Third, we tested the optimized TMS protocol in three non-human primate species (rhesus macaques, geladas, and capuchins). We captured a high percentage (mean=77.1%) of targeted CpG sites and produced methylation level estimates that agreed with those generated from reduced representation bisulfite sequencing (R² = 0.98). Finally, we applied our protocol to profile age-associated DNA methylation variation in two subsistence-level populations—the Tsimane of lowland Bolivia and the Orang Asli of Peninsular Malaysia—and found age-methylation patterns that were strikingly similar to those reported in high income cohorts, despite known differences in age-health relationships between lifestyle contexts. Altogether, our optimized TMS protocol will enable cost-effective, population-scale studies of genome-wide DNA methylation levels across human and non-human primate species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ea49a3598e0f07df9c655927729fa7dc601194" target='_blank'>
              Cost-effective solutions for high-throughput enzymatic DNA methylation sequencing
              </a>
            </td>
          <td>
            Amy Longtin, Marina M. Watowich, Baptiste Sadoughi, Rachel M. Petersen, Sarah F. Brosnan, Kenneth Buetow, Q. Cai, M. Gurven, Heather M. Highland, Yi-Ting Huang, Hillard Kaplan, Thomas S Kraft, Yvonne A L Lim, J. Long, Amanda D. Melin, Jamie Roberson, K. Ng, J. Stieglitz, B. Trumble, Vivek V Venkataraman, Ian J Wallace, Jie Wu, N. Snyder‐Mackler, Angela Jones, Alexander G. Bick, Amanda J. Lea
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b42acb0d0eb60fb675a97026237e9429fa851a1b" target='_blank'>
              Early-life tobacco exposure is causally implicated in aberrant RAG-mediated recombination in childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tanxin Liu, Keren Xu, Anmol Pardeshi, S. Myint, Alice Y Kang, L. Morimoto, Michael R Lieber, Joseph L Wiemels, Scott Kogan, C. Metayer, A. D. de Smith
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Currently, Cancer is the number one killer of human health. The death rate from cancer is increasing year over year as the population ages and grows, particularly since 2019, when there were almost 10 million cases globally recorded by the World Health Organization every year. In light of this trend, it is imperative to continue developing medications and to update and iterate on anti-cancer therapeutic technologies. CAR-T cell therapy, microphage-based therapy, oncolytic virotherapy, and checkpoint blockade therapy are the most popular cancer treatments, but a few patients still cannot tolerate these therapies. A post-optimized strategy is to use gene editing to alter the characteristics of cancer treatment. New therapeutic options for the treatment of complex disorders have become available with the development of CRISPR/Cas9 gene editing technology in recent years. This method is widely used in market research for gene knockdown, endogenous gene expression, and chromosome locus markers because of its accuracy and efficiency in processing harmful gene fragments. Also, it is extremely valuable from a medical standpoint when researching immunomodulator resistance. Despite its great potential, there remain concerns over the technology’s usefulness and efficacy and negative prognostic reactions. This review primarily focuses on the current use of CRISPR/Cas9 in cancer immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cb7ad71a9b9241a33354f318dab5664b26cb85a" target='_blank'>
              The Application of CRISPR/Cas9 in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Jiale Yu
          </td>
          <td>2024-10-29</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The evolution of operons has puzzled evolutionary biologists since their discovery, and many theories exist to explain their emergence, spreading, and evolutionary conservation. In this work, we suggest that DNA replication introduces a selective force for the clustering of functionally related genes on chromosomes, which we interpret as a preliminary and necessary step in operon formation. Our reasoning starts from the observation that DNA replication produces copy number variations of genomic regions, and we propose that such changes perturb metabolism. The formalization of this effect by exploiting concepts from metabolic control analysis suggests that the minimization of such perturbations during evolution could be achieved through the formation of gene clusters and operons. We support our theoretical derivations with simulations based on a realistic metabolic network, and we confirm that present-day genomes have a degree of compaction of functionally related genes, which is significantly correlated to the proposed perturbations introduced by replication. The formation of clusters of functionally related genes in microbial genomes has puzzled microbiologists since their first discovery. Here, we suggest that replication, and the copy number variations due to the replisome passage, might play a role in the process through a perturbation in metabolite homeostasis. We provide theoretical support to this hypothesis, and we found that both simulations and genomic analysis support our hypothesis.


IMPORTANCE
The formation of clusters of functionally related genes in microbial genomes has puzzled microbiologists since their discovery. Here, we suggest that replication, and the copy number variations due to the replisome passage, might play a role in the process through a perturbation in metabolite homeostasis. We provide theoretical support to this hypothesis, and we found that both simulations and genomic analysis support our hypothesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cd6d52f1dc8897095d217b532c18ad627ad2bb" target='_blank'>
              A new selective force driving metabolic gene clustering.
              </a>
            </td>
          <td>
            M. Fondi, Francesco Pini, Christopher Riccardi, Pietro Gemo, Matteo Brilli
          </td>
          <td>2024-10-28</td>
          <td>mSystems</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Myeloid neoplasms originate from the clonal proliferation of hematopoietic stem cells, which is driven by the acquisition of somatic genetic mutations. Within these disorders, myelodysplastic syndromes (MDS) are specifically characterized by morphologic abnormalities (dysplasia) and impaired maturation of myeloid precursors (ineffective hematopoiesis), resulting in peripheral blood cytopenia. Several studies have advanced the field of MDS, with a few landmark papers leading to a paradigm shift, opening new avenues of research and enabling a molecular revolution. These seminal papers include the first description of the 5q- syndrome, the identification of somatic mutations of TET2 in myeloid neoplasms, the detection of common pathway mutations in the splicing machinery, and the discovery of clonal hematopoiesis. The somatic genomic landscape of MDS is now well-defined. Genes that are recurrently mutated include epigenetic regulators, as well as genes of RNA splicing machinery, transcription regulation, DNA repair control, cohesin complex, and signal transduction. Furthermore, several disorders with a germline genetic predisposition to MDS have been identified, collectively accounting for up to 15% of all MDS cases. Genomic profiling can significantly improve the diagnostic approach to MDS, allowing the identification of distinct nosologic entities such as SF3B1-mutant or TP53-mutant MDS. The Molecular International Prognostic Scoring System for MDS (IPSS-M) has already proven to be a valuable tool for individualized risk assessment and treatment decisions. In addition, the recently developed molecular taxonomy of MDS will likely facilitate the implementation of precision medicine approaches for these disorders. This will necessitate the establishment of specialized infrastructures within public health systems, involving close collaboration between healthcare institutions, academia, and the life sciences industry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6811c5986693d3b027b3f548b29079c86dbe146d" target='_blank'>
              Genome sequencing in the management of myelodysplastic syndromes and related disorders.
              </a>
            </td>
          <td>
            Mario Cazzola, L. Malcovati
          </td>
          <td>2024-10-24</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f8a43614dcaaf12716b1614411bf7e1f17fff04" target='_blank'>
              AXL as immune regulator and therapeutic target in Acute Myeloid Leukemia: from current progress to novel strategies
              </a>
            </td>
          <td>
            N. Vandewalle, Nathan De Beule, A. De Becker, E. De Bruyne, E. Menu, K. Vanderkerken, Karine Breckpot, N. Devoogdt, K. De Veirman
          </td>
          <td>2024-10-04</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 The alteration of the glioblastoma genome by epigenetic mechanisms that share functions with normal developmental processes, such as self-renewal and fate specification of neural stem cells (NSC), is a key piece of evidence that links brain cancer pathogenesis with dysregulated stem cell functions. A patient-specific comparison of glioblastoma stem cells (GSC) with NSC, a putative cell of origin of at least a proportion of these tumours, is not feasible as patient-matched endogenous NSC are not surgically accessible and all epigenetic studies in glioblastoma have so far compared epigenetic changes of different tumours with each other or to comparators obtained from foetal brains or an unrelated donor. We reasoned that availability of syngeneic GSC and NSC pairs would allow to identify crucial epigenetic differences on a patient-specific basis and would provide essential therapeutic contrast to define disease-and patient-intrinsic biomarkers of drug response that are less confounded by germline variation.



 We have derived GSC from IDH-wildtype glioblastoma and harnessed state-of-the-art stem cell technologies to generate patient-matched fibroblast-derived EPSC, which were induced to NSC (iNSC). We generated transcriptome, DNA methylome, ChIP-Seq (H3K4me3, H3K27ac, H3K3me3, H3K27me3) and ATACSeq datasets for 10 GIC/iNSC pairs.



 Comparative integrated analysis has shown that redistribution of selected histone marks as well as shifts in chromatin states across the genome identifies differentially expressed genes, including novel druggable regulatory mechanisms in GSC, which are specific of the neoplastic context. In particular, we identified METTL7B as an essential regulator of lineage specification and plasticity in human glioblastoma, with impact on both tumour size and invasiveness in an in vivo xenograft model. Single cell transcriptomic analysis of these tumours and of cerebral organoids derived from expanded potential stem cells (EPSCs) overexpressing METTL7B reveal a regulatory role for the gene in the neural stem cells-to-astrocyte differentiation trajectory. Mechanistically, METTL7B downregulates the expression of key neuronal differentiation players, including SALL2, via post-translational modifications of histone marks.



 SYNGN is an epigenetic platform which can identify novel disease mechanisms and druggable targets in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be96bd21ac249c6fcb9083c430d82766e4fcbf5a" target='_blank'>
              P01.14.B SYNGN: A NOVEL EPIGENETIC PLATFORM TO DISCOVER PATIENT-SPECIFIC DISEASE MECHANISMS AND DRUGGABLE TARGETS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Marino, C. Vinel, M. Constantinou
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8352ce06dc7c139e4b3281df64d69ea0476999e3" target='_blank'>
              Molecular genetics, therapeutics and RET inhibitor resistance for medullary thyroid carcinoma and future perspectives
              </a>
            </td>
          <td>
            Ying Zhang, Wei-Hui Zheng, Shi-Hong Zhou, Jia-Lei Gu, Qing Yu, Yi-Zhou Zhu, Yu-Jie Yan, Zhi Zhu, Jin-Biao Shang
          </td>
          <td>2024-09-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting the effects of non-coding mutations on DNA methylation is crucial for advancing our understanding of gene expression, epigenetic inheritance, and its role in disease mechanisms. Current methods lack the capability to predict the impact of non-coding mutations on DNA methylation at single-cell resolution and long range, while remain challenges in tracking SNP influences throughout disease progression. Here, we introduce Methven, a deep learning-based framework designed to predict the effects of non-coding mutations on DNA methylation at single-cell resolution, to overcome the challenges. Methven integrates DNA sequences and ATAC-seq data, employing a divide-and-conquer approach to handle varying scales of SNP-CpG interactions. By leveraging a pretrained DNA language model, Methven accurately predicts both the direction and magnitude of methylation changes across a 100kbp range with a lightweight architecture. The evaluation results demonstrate the superior performance of Methven in prioritizing functional non-coding mutation, model interpretability, and its potential for revealing personalized mutation-disease associations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c29a5330f80b31c8ec7716b2f8598345b09c61" target='_blank'>
              Predicting the effect of non-coding mutations on single-cell DNA methylation using deep learning
              </a>
            </td>
          <td>
            Zhe Liu, An Gu, Yihang Bao, Guan Ning Lin
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This article describes how the transcriptional alterations of the innate immune system divide dysplasias into aggressive forms that, despite the treatment, relapse quickly and more easily, and others where the progression is slow and more treatable. It elaborates on how the immune system can change the extracellular matrix, favoring neoplastic progression, and how infections can enhance disease progression by increasing epithelial damage due to the loss of surface immunoglobulin and amplifying the inflammatory response. We investigated whether these dysregulated genes were linked to disease progression, delay, or recovery. These transcriptional alterations were observed using the RNA-based next-generation sequencing (NGS) panel Oncomine Immune Response Research Assay (OIRRA) to measure the expression of genes associated with lymphocyte regulation, cytokine signaling, lymphocyte markers, and checkpoint pathways. During the analysis, it became apparent that certain alterations divide dysplasia into two categories: progressive or not. In the future, these biological alterations are the first step to provide new treatment modalities with different classes of drugs currently in use in a systemic or local approach, including classical chemotherapy drugs such as cisplatin and fluorouracile, older drugs like fenretinide, and new checkpoint inhibitor drugs such as nivolumab and pembrolizumab, as well as newer options like T cell therapy (CAR-T). Following these observed alterations, it is possible to differentiate which dysplasias progress or not or relapse quickly. This information could, in the future, be the basis for determining a close follow-up, minimizing surgical interventions, planning a correct and personalized treatment protocol for each patient and, after specific clinical trials, tailoring new drug treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7733a13a6989d51b26a85ab5fd3970e4ce88c18" target='_blank'>
              A Transcriptomic Analysis of Laryngeal Dysplasia
              </a>
            </td>
          <td>
            F. Maffini, D. Lepanto, F. Chu, M. Tagliabue, D. Vacirca, R. De Berardinis, S. Gandini, Silvano Vignati, Alberto Ranghiero, S. Taormina, A. Rappa, M. Cossu Rocca, D. Alterio, Susanna Chiocca, Massimo Barberis, Lorenzo Preda, F. Pagni, Nicola Fusco, M. Ansarin
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9875ac28ea781567d1b798ee476f9d9fa7efb581" target='_blank'>
              Miniaturized protein profiling permits targeted signaling pathway analysis in individual circulating tumor cells to improve personalized treatment
              </a>
            </td>
          <td>
            Mahdi Rivandi, A. Franken, Liwen Yang, A. Abramova, Nadia Stamm, J. Eberhardt, B. Gierke, Meike Beer, Tanja Fehm, D. Niederacher, Michael Pawlak, Hans Neubauer
          </td>
          <td>2024-09-20</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1376592151e93e6ac67511ba542270ca30fd676f" target='_blank'>
              Analysis of Cancer-Resisting Evolutionary Adaptations in Wild Animals and Applications for Human Oncology.
              </a>
            </td>
          <td>
            Bokai K Zhang, Leonard Gines
          </td>
          <td>2024-09-11</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) maintain homeostasis in the hematopoietic ecosystem, which is tightly regulated at multiple layers. Acute myeloid leukemia (AML) is a severe hematologic malignancy driven by genetic and epigenetic changes that lead to the transformation of leukemia stem cells (LSCs). Since somatic mutations in DNA methylation-related genes frequently occur in AML, DNA methylation is widely altered and functions as a starting engine for initiating AML. Additionally, RNA modifications, especially N6-methyladenosine (m6A), also play an important role in the generation and maintenance of the hematopoietic ecosystem, and AML development requires reprogramming of m6A modifications to facilitate cells with hallmarks of cancer. Given the complex pathogenesis and poor prognosis of AML, it is important to fully understand its pathogenesis. Here, we mainly focus on DNA methylation and RNA m6A modification in hematopoiesis and AML and summarize recent advances in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f8b80ecba345e57d84b39e3572cdbd8a3dd106" target='_blank'>
              Epigenetic modifications in hematopoietic ecosystem: a key tuner from homeostasis to acute myeloid leukemia
              </a>
            </td>
          <td>
            Shu-Qi Yao, Rongxia Guo, Wen Tian, Yanbing Zheng, Jin Hu, G-X Han, Rong Yin, Fuling Zhou, Haojian Zhang
          </td>
          <td>2024-09-12</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Since the turn of the 21st century, genome-wide association study (GWAS) have successfully identified genetic signals associated with a myriad of common complex traits and diseases. As we transition from establishing robust genetic associations with diverse phenotypes, the central challenge is now focused on characterizing the underlying functional mechanisms driving these signals. Previous GWAS efforts have revealed multiple variants, each conferring relatively subtle susceptibility, collectively contributing to the pathogenesis of various common diseases. Such variants can further exhibit associations with multiple other traits and differ across ancestries, plus disentangling causal variants from non-causal due to linkage disequilibrium complexities can lead to challenges in drawing direct biological conclusions. Combined with cellular context considerations, such challenges can reduce the capacity to definitively elucidate the biological significance of GWAS signals, limiting the potential to define mechanistic insights. This review will detail current and anticipated approaches for functional interpretation of GWAS signals, both in terms of characterizing the underlying causal variants and the corresponding effector genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2c1abf9b3908ef75af33080a59276f046b73ed0" target='_blank'>
              Translation of genome-wide association study: from genomic signals to biological insights
              </a>
            </td>
          <td>
            Winter S. Bruner, Struan F. A. Grant
          </td>
          <td>2024-10-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c904b2f681f70c0b549880d31ccc37b010f98cbc" target='_blank'>
              Characterisation of APOBEC3B-Mediated RNA editing in breast cancer cells reveals regulatory roles of NEAT1 and MALAT1 lncRNAs.
              </a>
            </td>
          <td>
            Chi Zhang, Yu-Jing Lu, Mei Wang, Bingjie Chen, Feifei Xiong, C. Mitsopoulos, Olivia Rossanese, Xiuling Li, Paul A Clarke
          </td>
          <td>2024-09-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bba8ffa37402689d1b771c33d5b55e131826ac5" target='_blank'>
              A narrative review: exploring viral-induced malignancies through the lens of dysregulated cellular metabolism and glucose transporters
              </a>
            </td>
          <td>
            Amirhossein Shahpar, Vahideh Hamidi Sofiani, Nazanin Zeinali Nezhad, Marzieh Charostad, Reza Ghaderi, Niloofar Farsiu, Amin Karimzadeh Kiskani, Sara Pezeshki, Mohsen Nakhaie
          </td>
          <td>2024-10-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa55e8229406f95ad183633e37da53bba0d171d3" target='_blank'>
              An automated network-based tool to search for metabolic vulnerabilities in cancer
              </a>
            </td>
          <td>
            L. Valcárcel, Edurne San José-Enériz, Raquel Ordoñez, Iñigo Apaolaza, Danel Olaverri-Mendizabal, Naroa Barrena, Ana Valcárcel, L. Gárate, J. San Miguel, Antonio Pineda-Lucena, X. Agirre, F. Prósper, F. J. Planes
          </td>
          <td>2024-10-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Since the approval of the first checkpoint inhibitor, ipilimumab (α-CTLA4), there continues to be enthusiasm around leveraging the immune system to eradicate solid tumors. However, unknowns about the cellular mechanism(s) that cause some patients to durably respond while others relapse, continue to stymie the field. To support ongoing efforts elucidating cell subsets responsible for differences in patient responses, we aim to evaluate the functional heterogeneity of T-cell subsets responding to checkpoint inhibitors in mouse models of solid tumors. Single-cell RNA-seq has enabled discoveries of transcriptionally defined cell subsets, but understanding their function require inference from transcription, flow-based validation biased by pre-selection of subsets with specific markers, or reliance on surrogate markers for cell-extrinsic functions (e.g., cytotoxicity). Thus, functional heterogeneity could be obscured. To address this, we leveraged a novel technology that converges live, cellular functions with transcriptomics at the resolution of single cell-cell interactions. This technology captures tens of thousands of individual cells or defined combinations of groups of cells in bio-compatible hydrogel compartments, enabling longitudinal imaging over days to weeks. Throughout this period, functions are characterized via bright-field or immunofluorescence readouts, including antibodies, cytokine capture beads, and dyes. Cells are then lysed within compartments to generate barcoded libraries for mRNA-sequencing and gene signatures from individual cells computationally deconvoluted. With this technology, we developed a novel assay that enables longitudinal, unbiased functional characterization of individual T cells mapped to single-cell transcriptomics. Single mouse primary T cells were encapsulated with one or more MC38 target cells and imaged over time. We validated combinations of different readouts can be measured from the same T-cell, including cell-counts for proliferation, CD8/3 expression to differentiate T cells from target cells, activation markers, and / or target cell death via Sytox. Thus, single T cells can be multi-functionally evaluated over time, enabling the study of attributes inaccessible at high scale with prior technologies, including kinetics of killing or the number of target cells killed by a single T-cell. For assays combining cytotoxicity and surface receptors, we generated barcoded libraries from functionally characterized T cells for sequencing and developed an analytical pipeline that deconvolutes mRNA from single T cells mapped to their multi-functional profiles, to identify novel pathways that regulate anti-tumor functions. We will leverage this workflow to functionally characterize single T cells in tumor-bearing mice treated with checkpoint inhibitors and map the data to underlying transcriptomics. In doing so, we aim to discover functional subsets, potentially with identical transcriptional profiles, to support hypothesis generation around mechanisms governing anti-tumor efficacy of checkpoint inhibitors.
 Citation Format: Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer. Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b89db5bb53ca20d5ac128fa18040b839f5041ca9" target='_blank'>
              Abstract A025: Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors
              </a>
            </td>
          <td>
            Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Infant ALL (iALL) is initiated in utero, most often by rearrangement of the KMT2A gene (KMT2Ar). It carries a very poor prognosis despite a lack of additional oncogenic driver mutations common in childhood ALL. Here, we aimed to identify specific properties of human fetal hematopoietic stem/progenitor cells (HSPC) that promote leukemic transformation in KMT2Ar iALL using molecular, functional and in vivo assays. First, by comparing transcriptomes of human fetal HSPC to adult HSPC we derived a fetal-specific gene signature and identified the fetal oncogene LIN28B and its downstream effectors among the top hits. These genes were also expressed in iALL. Functional assays revealed that LIN28B was essential in human fetal liver (FL) CD34+ cells to maintain proliferation and stemness, and support B- and NK-lymphopoiesis. To interrogate the role of LIN28B in iALL, we utilised a human FL-derived CRISPR-Cas9 KMT2A::AFF1 model. In this model, LIN28B-expressing leukemias were more proliferative in vitro and in vivo, with this advantage being lost upon LIN28B knockdown. Mechanistic studies showed that LIN28B acts by stabilizing key early B-lymphoid genes, epigenetic regulators, and cell cycle and anti-apoptotic genes. Finally, In the absence of LIN28B, human FL CD34+ cells fail to transform upon induction of KMT2A::AFF1 translocation. Thus, LIN28B has an essential role in normal human fetal B-lymphopoiesis, and is necessary for the initiation of KMT2A::AFF1 iALL in fetal cells in the absence of co-operating mutations. It has a role in making leukemias more aggressive, suggesting it is a potential target in LIN28B-expressing leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cc286f51631e81c564d7b507e1a0ef71703dbb" target='_blank'>
              The fetal specific gene LIN28B is essential for human fetal B-lymphopoiesis and initiation of KMT2A::AFF1 infant leukemia
              </a>
            </td>
          <td>
            Rebecca E. Ling, T. Jackson, N. Elliott, Joe Cross, Lucy Hamer, Arundhati Wuppalapati, Alastair L. Smith, Catherine Chahrour, Okan Sevim, Deena Iskander, Guanlin Wang, S. Rice, Sorcha O’Byrne, Joe R Harman, Bethan Psaila, R. Morgan, Irene Roberts, Thomas A. Milne, A. Roy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa97a0ec1c0279a03e3ac7f307fe7b7f9d3560ca" target='_blank'>
              The SRC-family serves as a therapeutic target in triple negative breast cancer with acquired resistance to chemotherapy.
              </a>
            </td>
          <td>
            E. V. Egeland, Kotryna Seip, Eleni Skourti, G. F. Øy, S. Pettersen, Abhilash D Pandya, M. A. Dahle, M. H. Haugen, Alexander Kristian, S. Nakken, O. Engebraaten, G. Mælandsmo, L. Prasmickaite
          </td>
          <td>2024-10-10</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Objective
Successful gene editing technology is crucial in molecular biology and related fields. An essential part of an efficient knock-in system is increasing homologous recombination (HR) efficiency in the double-strand break (DSB) repair pathways. Interestingly, HR is closely related to the DNA mismatch repair (MMR) pathway, whereby MMR-related gene Msh2 recognizes a mismatch of nucleotides in recombinant intermediates or gene conversion formed during HR. This study aimed to investigate how the knockdown of Msh2 affects HR-mediated knock-in efficiency at the mouse β-casein locus. Therefore, we investigated the effect of inhibiting Msh2 expression on the expression of the HR-related gene Rad51 and the key enzyme DNA ligase IV involved in non-homologous end joining (NHEJ).


Methods
The knock-in vector targeting the mouse β-casein gene locus, programmed guide RNA, and Msh2 siRNA expression vector were co-transfected in HC11 cells, or only the Msh2 siRNA expression vector was transfected. Knock-in efficiency was confirmed by polymerase chain reaction (PCR). The mRNA and protein expression of Msh2, HR-related gene Rad51, and NHEJ-related gene DNA ligase IV were evaluated by quantitative reverse transcription PCR (RT-qPCR) and Western blot analysis.


Results
The knock-in vector efficiency at the mouse β-casein gene locus significantly decreased upon Msh2 knockdown in HC11 mouse mammary epithelial cells (HC11 cell). Additionally, the knockdown of the DNA MMR-related gene Msh2 protein significantly downregulated the nuclear protein expression of the HR-related Rad51 and NHEJ-related DNA ligase IV genes.


Conclusion
The decreased Msh2 protein expression in the nucleus downregulated the Rad51 and ligase IV protein expressions. Consequently, reduced Rad51 expression results in decreased knock-in efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6fb442a421eb6d0724723f4d91e317f706705e" target='_blank'>
              The decrease in Rad51 and DNA ligase IV nuclear protein expression in Msh2 knockdown HC11 cells induced the low CRISPR/Cas9-mediated knock-in efficiency at the β-casein gene locus.
              </a>
            </td>
          <td>
            Ga-Yeon Kim, Man Kang
          </td>
          <td>2024-10-24</td>
          <td>Animal bioscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe05bbee4b4093b8e7b08a50c4cd4811883bc49" target='_blank'>
              MYC upstream region orchestrates resistance to PI3K inhibitors in cancer cells through FOXO3a-mediated autophagic adaptation.
              </a>
            </td>
          <td>
            Rosa Bordone, D. Ivy, Rodrigo D'Amico, Martina Barba, Miriam Gaggianesi, Fiorella Di Pastena, Bianca Cesaro, Francesca Bufalieri, Alessio Balzerano, E. De Smaele, Giuseppe Giannini, L. Di Marcotullio, A. Fatica, Giorgio Stassi, Laura Di Magno, S. Coni, G. Canettieri
          </td>
          <td>2024-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Oncogenic growth places great strain and dependence on the proteostasis network. This has made proteostasis pathways attractive therapeutic targets in cancer, but efforts to drug these pathways have yielded disappointing clinical outcomes. One exception is proteasome inhibitors, which are approved for frontline treatment of multiple myeloma. However, proteasome inhibitors are largely ineffective for treatment of other cancers, including acute myeloid leukemia (AML), although reasons for these differences are unknown. Here, we determined that proteasome inhibitors are ineffective in AML due to inability to disrupt proteostasis. In response to proteasome inhibition, AML cells activated HSF1 and autophagy, two key stem cell proteostasis pathways, to prevent unfolded protein accumulation. Inactivation of HSF1 sensitized human AML cells to proteasome inhibition, marked by unfolded protein accumulation, activation of the PERK-mediated integrated stress response, severe reductions in protein synthesis, proliferation and cell survival, and significant slowing of disease progression and extension of survival in vivo. Similarly, combined autophagy and proteasome inhibition suppressed proliferation, synergistically killed AML cells, and significantly reduced AML burden and extended survival in vivo. Furthermore, autophagy and proteasome inhibition preferentially suppressed protein synthesis and induced apoptosis in primary patient AML cells, including AML stem/progenitor cells, without severely affecting normal hematopoietic stem/progenitor cells. Combined autophagy and proteasome inhibition also activated the integrated stress response, but surprisingly this occurred in a PKR-dependent manner. These studies unravel how proteostasis pathways are co-opted to promote AML growth, progression and drug resistance, and reveal that disabling the proteostasis network is a promising strategy to therapeutically target AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028b5051007a2ad0752d56809a3e864ed86f20ae" target='_blank'>
              The Proteostasis Network is a Therapeutic Target in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Kentson Lam, Yoon Joon Kim, Carlo M. Ong, Andrea Z. Liu, Fanny J. Zhou, M. Sunshine, Bernadette A. Chua, Silvia Vicenzi, Pierce W. Ford, Jie-Hua Zhou, Yuning Hong, Eric J. Bennett, Leslie A. Crews, Edward D. Ball, Robert Signer
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9a6a91b5f3ac052c57405c773236aad0d8d4b4" target='_blank'>
              Recent insights into the causes and consequences of chromosome mis-segregation.
              </a>
            </td>
          <td>
            Romain Devillers, Alexsandro dos Santos, Quentin Destombes, Mathieu Laplante, S. Elowe
          </td>
          <td>2024-09-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Each cancer is the result of an individually unique evolutionary process. Yet, as demonstrated by extensive genetic analyses of human specimens, the genetic endpoints of cancers across different tissues are remarkably specific, with individual cancer driver genes being typically associated with only a limited set of tumor types. Tissue and cellular contexts thus impose strong and genetically predictable evolutionary constraints on carcinogenesis. Lineage-specific transcription factors, central regulators of organismal development, tissue homeostasis, and regeneration, often also support cancer cell fitness in a lineage-specific manner. In this review, we discuss recent results on the interactions between transcriptional lineage factor programs and oncogenic pathways and how such interactions may determine the oncogenic competence of cancer-associated genetic alterations. These developments are starting to shed light on the molecular basis of tissue specificity in carcinogenesis, with relevance for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f236b52bc63786a8f5d7d9e1f3c0d9457cd40be6" target='_blank'>
              Lineage-Specific Transcription Factors in Carcinogenesis
              </a>
            </td>
          <td>
            Charles J. David, Sakari Vanharanta
          </td>
          <td>2024-10-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2076e25ef24711c61a952d9cef52490a04939f92" target='_blank'>
              Comparative small molecule screening of primary human acute leukemias, engineered human leukemia and leukemia cell lines.
              </a>
            </td>
          <td>
            Safia Safa-Tahar-Henni, Karla Páez Martinez, Verena Gress, Nayeli Esparza, É. Roques, Florence Bonnet-Magnaval, Mélanie Bilodeau, Valérie Gagné, Eva Bresson, Sophie Cardin, Nehme El-Hachem, Isabella Iasenza, Gabriel Alzial, I. Boivin, N. Nakamichi, Anne-Cécile Soufflet, Cristina Mirela Pascariu, Jean Duchaine, Simon Mathien, É. Bonneil, Kolja Eppert, A. Marinier, Guy Sauvageau, Geneviève Deblois, Pierre Thibault, J. Hébert, Connie J. Eaves, S. Cellot, F. Barabé, Brian T Wilhelm
          </td>
          <td>2024-10-29</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Enhancers are critical genetic elements controlling transcription from promoters, but the mechanisms by which they convey regulatory information across large genomic distances remain elusive. Here, we engineered pluripotent stem cells in which cohesin loop extrusion can be inducibly disrupted without causing confounding cell cycle defects. While evident, transcriptional dysregulation was cell-type specific, and not all loci with distal enhancers depend equally on cohesin extrusion. Using comparative genome editing, we demonstrate that enhancer-promoter communication across as little as 20 kilobases can rely on cohesin. However, promoter-proximal regulatory elements can support long-range, cohesin-independent enhancer action – either upon disabling extrusion or across strong CTCF insulators. Finally, transcriptional dynamics and the emergence of new embryonic cell types in response to differentiation cues remained largely robust to disrupting cohesin extrusion. Beyond establishing novel experimental strategies to study cohesin functions in enhancer biology, our work provides mechanistic insight accounting for both cell type- and genomic context-specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24ae69cf573c71b5501cf7f33d094562bf63c019" target='_blank'>
              Synergy between cis-regulatory elements can render cohesin dispensable for distal enhancer function
              </a>
            </td>
          <td>
            Karissa Hansen, Annie S. Adachi, L. Braccioli, Smit Kadvani, Ryan M. Boileau, Bozhena Pokorny, Rini Shah, Erika C. Anderson, Moreno Martinovic, Kaite Zhang, Irié Carel, Kenya Bonitto, R. Blelloch, G. Fudenberg, Elzo de Wit, Elphѐge P. Nora
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>