<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-11 09:34:57 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>48</td>
          <td>44</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="DNA damage from routine cellular processes or exogenous insults can have a lasting impact on gene regulation beyond genetic mutations. The prevailing paradigm for the consequences of DNA damage repair revolves around restoration of the original genetic sequence, but long-term changes in chromatin configuration, gene expression and DNA modifications have not been analyzed. We introduced numerous, simultaneous Cas9-mediated DNA double strand breaks (DSBs) at defined locations in human glioblastoma cells and tracked both non-genetic and genetic alterations over time. Megabase-scale genomic alterations that endured two weeks after the initial damage were detected, involving a shift from transiently increased intra-TAD interactions to persistent long range cis and trans contacts, alterations in gene-expression and associated large structural variations. These findings reveal that widespread DNA damage, such as chemotherapy or radiotherapy, can trigger long-term genetic and non-genetic modifications which alter cellular function and may impact tumor outcome and the emergence of resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Multidrug resistance (MDR) significantly contributes to colon cancer recurrence, making it essential to understand its molecular basis for improved therapies. This study aimed to identify genes and pathways involved in resistance to standard chemotherapeutics by comparing transcriptome profiles of sensitive and paclitaxel-induced MDR colonospheres. Cell viability and growth were assessed following treatment with 5-fluorouracil, oxaliplatin, irinotecan, bevacizumab, and cetuximab. Drug concentrations in culture media posttreatment were measured using high-performance liquid chromatography (HPLC). RNA sequencing (RNA-seq) of untreated sensitive and resistant colonospheres identified differentially expressed genes linked to baseline resistance. Our results confirmed cross-resistance in the resistant model, showing highest oxaliplatin tolerance may involve mechanisms beyond efflux. Transcriptome analysis highlighted upregulation of PIGR and activation of the ribosomal signaling pathway as potential resistance mediators. Notably, AKR1B10, a gene linked to chemotherapeutic detoxification, was overexpressed, whereas genes related to adhesion and membrane transport were downregulated. The overexpression of ribosomal protein genes suggests ribosome biogenesis plays a key role in acquired resistance. These findings suggest that targeting ribosome biogenesis and specific deregulated genes such as PIGR and AKR1B10 may offer promising strategies to overcome MDR in colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6aa3128769906dcf129a6a3196212e61e40551f" target='_blank'>
              Transcriptomic Landscape of Paclitaxel-Induced Multidrug Resistance in 3D Cultures of Colon Cancer Cell Line DLD1
              </a>
            </td>
          <td>
            S. Dragičević, J. Dinić, Milena Ugrin, Marija Vidovic, T. Babic, A. Nikolić
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Docetaxel is a chemotherapeutic agent widely used for breast cancer treatment; however, its efficacy is often limited by drug resistance and associated toxicity. This review examines the molecular mechanisms of docetaxel resistance in breast cancer and discusses research advances and future directions for overcoming this challenge. Key resistance mechanisms include alterations in drug targets (microtubules), increased drug efflux, suppression of apoptosis, activation of survival signalling pathways, epithelial-to-mesenchymal transition (EMT), and cancer stem cell enrichment. An evolutionary perspective distinguishes between intrinsic and acquired resistance, emphasising the need for adaptive therapeutic strategies. Recent advances in genomic profiling, non-coding RNA research, novel drug combinations, and biomarker-guided therapies have also been reviewed. Emerging approaches, such as targeting the tumour microenvironment, harnessing immunotherapy, and implementing adaptive dosing schedules, have been discussed. This review emphasises the understanding of resistance as a multifactorial phenomenon that requires multipronged interventions. Research has aimed to identify predictive biomarkers, develop targeted agents to reverse resistance, and design rational combination strategies to improve patient outcomes. Progress in deciphering and targeting docetaxel resistance mechanisms holds promise for enhancing treatment responses and extending survival in patients with breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb368d7af0b16140c6442111947c4c43ba088b35" target='_blank'>
              Docetaxel Resistance in Breast Cancer: Current Insights and Future Directions
              </a>
            </td>
          <td>
            F. Postigo-Corrales, Asunción Beltrán-Videla, A. D. Lázaro-Sánchez, Ana María Hurtado, P. Conesa-Zamora, Ana Belén Arroyo, G. Luengo-Gil
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Calcium signaling is a key controller of numerous cellular events and is intricately linked to many processes that are critical pathways in cancer progression. This review revisits the calcium signaling toolkit in cancer, with a focus on calcium regulation of processes that go beyond the originally defined "classic" hallmarks of cancer such as those associated with proliferation, metastasis, and resistance to cell death pathways. We will consider calcium signaling in the context of the more recently proposed hallmarks of cancer, emerging hallmarks, and cancer-enabling characteristics. This broader examination of calcium signaling and its toolkit members will encompass processes such as metabolic reprogramming, evasion of immune destruction, cellular phenotypic plasticity, senescence, genome instability, and nonmutational epigenetic reprogramming. These cancer features and their interactions with calcium signaling will frequently be analyzed through the lenses of therapy resistance and the complexities of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8339a4a948b497d0310db8cdaa1d952eeb9805ad" target='_blank'>
              Cancer Progression and the Calcium Signaling Toolkit: Expanding Dimensions and Perspectives.
              </a>
            </td>
          <td>
            M. Robitaille, G. Monteith
          </td>
          <td>2025-06-16</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Bruton's tyrosine kinase inhibitors (BTKi) and cell therapy have successfully been used to treat mantle cell lymphoma (MCL); however, therapy resistance inevitably emerges. Cancer cells can progressively develop stable resistance by traversing through a transient drug-tolerant persister (DTP) state. The mechanisms enabling DTP cells to reversibly adapt to therapies and evolve to acquire heterogeneity remain poorly understood, and characterizing DTP cells in MCL continues to pose a challenge for clinic translation. Here using pirtobrutinib, a recently FDA-approved non-covalent BTKi, we identified pirtobrutinib-tolerant persister cells exhibiting morphological variability by presenting a unique population of enlarged cells (Giant cells) with reversible fate transitions. During treatment, Giant cells enter a non-proliferative, dedifferentiated state, addicted to an activated cytosolic tricarboxylic acid (TCA) cycle coupled with the malate-aspartate shuttle to engage in biosynthesis. Upon drug removal, the TCA cycle shifts to oxidative catabolism, promoting Giant cells to differentiate into regular-sized cells. Throughout the transition, acetyl-CoA modulates cell fate by fine-tuning stemness. Our biphasic model demonstrates that the metabolic switch governs the phenotypic plasticity of DTP cells in MCL, resulting a dynamic presence of DTP cells across various developmental states in response to systemic therapies. Targeting Giant cells prior to their differentiation offers a promising strategy to overcoming therapy resistance in MCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a70e87ba9cf5ac8f079a2c41ffecedfcf35616a9" target='_blank'>
              TCA cycle mode switch determines the fate of pirtobrutinib-tolerant persister cells in mantle cell lymphoma.
              </a>
            </td>
          <td>
            Wei Wang, Qingsong Cai, Yang Liu, L. Nie, H. Lee, F. Yan, Yue Fei, Yixin Yao, Yijing Li, Lin Tan, Philip L. Lorenzi, Ying-Nai Wang, Jun Yao, Zhihong Chen, J. McIntosh, Chengzhang Yu, P. Jain, V. Jiang, J. Vargas, Xiaolin Li, Tianci Zhang, Shaoying Li, D. Santos, S. Thirumurthi, Erin H. Seeley, Lukas M. Simon, C. Flowers, C. Ok, Michael L Wang
          </td>
          <td>2025-07-23</td>
          <td>Blood</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4a2e884a937abc2345c5e03e509d49bf56bc09" target='_blank'>
              Transcriptome-wide decoding the roles of aberrant splicing in melanoma MAPK-targeted resistance evolution.
              </a>
            </td>
          <td>
            Jing Yu, Xiujing He, Xueyan Wang, Chune Yu, Xian Jiang, Yanna Li, Xinyu Liu, Ya Luo, Xuemei Chen, Sisi Wu, Lu Si, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-07-18</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Anticancer chemotherapy is an essential part of cancer treatment, but the emergence of resistance remains a major hurdle. Metabolic reprogramming is a notable phenotype associated with the acquisition of drug resistance. Here, we develop a computational framework that predicts metabolic gene targets capable of reverting the metabolic state of drug-resistant cells to that of drug-sensitive parental cells, thereby sensitizing the resistant cells. The computational framework performs single-gene knockout simulation of genome-scale metabolic models that predicts genome-wide metabolic flux distribution in drug-resistant cells, and clusters the resulting knockout flux data using uniform manifold approximation and projection, followed by k-means clustering. From the clustering analysis, knockout genes that lead to the flux data near that of drug-sensitive cells are considered drug sensitization targets. This computational approach is demonstrated using doxorubicin- and paclitaxel-resistant MCF7 breast cancer cells. Drug sensitization targets are further refined based on proteome and metabolome data, which generate GOT1 for doxorubicin-resistant MCF7, GPI for paclitaxel-resistant MCF7, and SLC1A5 as a common target. These targets are experimentally validated where treating drug-resistant cancer cells with small-molecule inhibitors results in increased sensitivity of drug-resistant cells to doxorubicin or paclitaxel. The applicability of the developed framework is further demonstrated using drug-resistant triple-negative breast cancer cells. Taken together, the computational framework predicts drug sensitization targets in an intuitive and cost-efficient manner and can be applied to overcome drug-resistant cells associated with various cancers and other metabolic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7052bad828f26940bd1911c2893c8896c16d13b8" target='_blank'>
              Genome-scale knockout simulation and clustering analysis of drug-resistant breast cancer cells reveal drug sensitization targets.
              </a>
            </td>
          <td>
            JinA Lim, Hae Deok Jung, S. Park, Moonhyeon Jeon, Da Sol Kim, Ryeongeun Cho, Dohyun Han, Han Suk Ryu, Yoo-Ree Kim, Hyun Uk Kim
          </td>
          <td>2025-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c251c2e80393955c5ed269cb0f5004e71c3f6" target='_blank'>
              Expanding the utility of transcriptome analysis for mutation detection in high-risk childhood precision oncology
              </a>
            </td>
          <td>
            Chelsea Mayoh, P. Barahona, Angela Lin, Lujing Cui, P. Ajuyah, A. Altekoester, L. Lau, Dong-Anh Khuong-Quang, Patricia Sullivan, A. Senapati, S. Nagabushan, Ashleigh Sullivan, Natacha Omer, Andrew S Moore, Wayne Nicholls, R. Endersby, N. Gottardo, G. McCowage, L. dalla Pozza, Jordan R Hansford, S. Khaw, Paul J Wood, T. Trahair, G. Marshall, D. S. Ziegler, Vanessa Tyrrell, Michelle Haber, Marie Wong, P. Ekert, Mark J. Cowley
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Metastatic uveal melanoma is an aggressive disease with limited effective therapeutic options. To comprehensively map monogenic and digenic dependencies, we performed CRISPR–Cas9 screening in ten extensively profiled human uveal melanoma cell line models. Analysis involved genome-wide single-gene and combinatorial paired-gene CRISPR libraries. Among our 76 uveal melanoma-specific essential genes and 105 synthetic lethal gene pairs, we identified and validated the CDP-diacylglycerol synthase 2 gene (CDS2) as a genetic dependency in the context of low CDP-diacylglycerol synthase 1 gene (CDS1) expression. We further demonstrate that CDS1/CDS2 forms a synthetic lethal interaction in vivo and reveal that CDS2 knockout results in the disruption of phosphoinositide synthesis and increased cellular apoptosis and that re-expression of CDS1 rescues this cell fitness defect. We extend our analysis using pan-cancer data, confirming increased CDS2 essentiality in diverse tumor types with low CDS1 expression. Thus, the CDS1/CDS2 axis is a therapeutic target across a range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cea3171a10bd393fb0f6c1167b72877c7aa37fbb" target='_blank'>
              The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability in uveal melanoma and across multiple tumor types
              </a>
            </td>
          <td>
            Pui Ying Chan, Diana Alexander, Ishan Mehta, Larissa Satiko Alcantara Sekimoto Matsuyama, V. Harle, R. Olvera-León, Jun Sung Park, Fernanda G Arriaga-González, L. van der Weyden, S. Cheema, Vivek Iyer, Victoria Offord, David Barneda, Phillip Hawkins, Len Stephens, Z. Kozik, Michael Woods, K. Wong, G. Balmus, A. Vinceti, N. A. Thompson, M. Del Castillo Velasco-Herrera, L. Wessels, J. van de Haar, Emanuel Gonçalves, Sanju Sinha, M. E. Vázquez‐Cruz, L. Bisceglia, F. Raimondi, J. Choudhary, Sumeet Patiyal, Anjan Venkatesh, F. Iorio, Colm J. Ryan, D. J. Adams
          </td>
          <td>2025-07-04</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Epigenetic modifications provide a mechanism through which environmental factors can induce heritable changes in gene expression without altering the underlying DNA sequence. While the Modern Synthesis emphasises genetic mutation and natural selection as the primary drivers of evolution, growing evidence highlights the role of DNA methylation, histone modifications, and non-coding RNAs in enabling phenotypic plasticity and transgenerational inheritance. These mechanisms allow organisms to respond to environmental conditions more rapidly than genetic changes alone would permit, and in some cases, these responses are transmitted across generations via the germline. Empirical studies from both plant and animal systems demonstrate how exposures to stress, nutritional changes, and chemical disruptors can leave stable epigenetic marks. Additionally, large-scale genomic rearrangements such as chromoanagenesis point to the potential for abrupt structural variation to interact with epigenetic processes. Incorporating such findings within the framework of the Extended Evolutionary Synthesis expands traditional evolutionary models, offering a more integrated understanding of how environmental stimuli shape heritable variation and influence evolutionary trajectories over both short and long timescales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa60e2219c1da1273bd416f9b243ed47b34a3d3" target='_blank'>
              The Role of Epigenetic Modifications in Evolution: Understanding How Environmental Factors Shape Heritable Gene Expression Across Generations
              </a>
            </td>
          <td>
            Damla Naz Haymanalı
          </td>
          <td>2025-07-10</td>
          <td>Next Generation Journal for The Young Researchers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance and distant metastases are leading causes of mortality in colorectal cancer (CRC), yet the molecular mechanisms linking these processes remain elusive. In this study, we demonstrate that acquired resistance to oxaliplatin, a first-line chemotherapeutic in CRC, enhances metastatic potential through transcriptional reprogramming. Using a clinically relevant dosing regimen, we generated oxaliplatin-resistant CRC cells that displayed increased metastatic potential. Integrated transcriptomic and phenotypic analyses revealed that dysregulated cholesterol biogenesis amplifies TGF-β signaling, which in turn drives expression of SERPINE1, which serves as a key effector of both oxaliplatin resistance and metastasis. Furthermore, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that robustly predicts overall and relapse-free survival across distinct patient cohorts. Notably, RESIST-M stratifies a high-risk subtype of CMS4/iCMS3-fibrotic patients that display the poorest prognosis, underscoring its clinical relevance. Targeting of SERPINE1 or cholesterol biosynthesis re-sensitized resistant, pro-metastatic cells to oxaliplatin in mouse xenograft models. Altogether, this study uncovers a mechanistic link between metabolic rewiring and transcriptional plasticity underlying therapy-induced metastasis in primary CRC. Additionally, it also reveals actionable vulnerabilities that offer both prognostic value and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461233a4140dd31d355e46865c3174f542e2818f" target='_blank'>
              Modeling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Kenzom Tenzin, Niranjan Shirgaonkar, Huiwen Chua, Lin Xuan Chee, Ahpa Sae Yeoh, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, Ethan Subel, M. Thangavelu, Jane Vin Chan, Choon-Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj Dasgupta
          </td>
          <td>2025-07-16</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Balanced chromosomal abnormalities (BCAs) are structural variations that can underlie a wide spectrum of neurodevelopmental disorders, often remaining undetected by conventional diagnostic approaches. Whole-genome sequencing (WGS) allows for base-pair resolution of structural variants across the entire genome, making it a powerful tool to detect cryptic chromosomal rearrangements and refine breakpoint mapping. RNA sequencing (RNA-Seq), by enabling the detection of gene expression changes and fusion transcripts, provides complementary functional insights into the consequences of genomic alterations. This study integrated WGS and RNA-Seq to precisely characterize the breakpoints and assess the functional impact of de novo BCAs in two unsolved cases of Neurodevelopmental Disorders. Materials and methods Short read WGS was used to identify the chromosomal breakpoints and gene disruptions caused by BCAs. RNA-Seq on blood RNA was employed to detect differential gene expression and potential fusion transcripts of disrupted genes. Results In the first case, the inversion inv(8) (p11.2q13) disrupted two genes at the breakpoints, namely, CHD7 and SLC20A2. These genes are in opposite orientations, and the inversion realigned them in the same direction, generating two novel fusion genes. Disruption of CHD7 confirmed the suspected diagnosis of CHARGE syndrome. The interruption of SLC20A2, commonly associated with neurological symptoms, prompted further clinical evaluation. RNA-Seq identified in-frame fusion transcripts from the chimeric genes in the blood, suggesting a potential deleterious phenotypic effect. In the second case, WGS revealed a balanced translocation t(17; 22) (q25; q13) that disrupted EP300 at 22q25, confirming Rubinstein-Taybi syndrome. The concurrent disruption of RBFOX3 at 17q13 suggested additional neurological implications, particularly related to epilepsy. Transcriptomic analysis demonstrated the monoallelic and significantly reduced expression of EP300. Conclusion These findings highlight the crucial role of WGS in identifying disease-associated BCAs and underscore the complementary value of RNA-Seq in assessing their functional consequences. This integrated approach enhanced diagnostic accuracy and clinical management, paving the way for more comprehensive and personalized care in these two patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8f54f85ebf927d3f8b7d5f4a6236e7f717fa51" target='_blank'>
              Integrating genome and transcriptome analysis to decipher balanced structural variants in unsolved cases of neurodevelopmental disorders
              </a>
            </td>
          <td>
            S. Mellone, Alice Spano, Denise Vurchio, Giulia Borgonovi, Alessandro Ugonotti, Giulia Paglino, Alba Bianco, Sara Ronzani, Maurizio Sciancalepore, Flavia Prodam, Amanda Papa, Maurizio Viri, Umberto Dianzani, Mara Giordano
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In ~70% of patients with pancreatic ductal adenocarcinoma, the TP53 gene acquires gain-of-function (GOF) mutations leading to rapid disease progression. Specifically, missense p53 (misp53) GOF mutations associate with therapy resistance and worse clinical outcomes. However, the molecular functions of distinct misp53 mutants in plasticity and therapy response remain unclear. Integrating multicenter patient data and multi-omics, we report that the misp53R273H/C mutant is associated with cell cycle progression and a basal-like state compared to the misp53R248W/Q mutant. Loss of misp53R273H/C decreased tumor growth and liver metastasis while prolonging survival in preclinical models. We found that misp53R273H/C specifically regulated the Rb/DREAM axis involved in cell cycle regulation. Notably, a clinical CDK4/6 inhibitor reduced misp53R273H/C mutant expression. However, it triggered MAPK/ERK-mediated resistance mechanisms, enhancing cell survival and resistance to CDK4/6 inhibitors. Combining MAPK/ERK and CDK4/6 inhibitors reduced misp53R273H/C-associated oncogenic functions. Thus, distinct misp53 mutants show unique cell-intrinsic plasticity, therapeutic vulnerabilities, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8366ad2cf60feed0070f18be69dfac6ab275f732" target='_blank'>
              TP53 missense–specific transcriptional plasticity drives resistance against cell cycle inhibitors in pancreatic cancer
              </a>
            </td>
          <td>
            Laura Urbach, Lena Wieland, Frederike Penz, Rebecca Diya Samuel, Stefan Küffer, Lukas Klein, Christof Lenz, Ulrich Sax, Michael Ghadimi, Ramona Schulz-Heddergott, E. Hessmann, V. Ellenrieder, Nelson Dusetti, Shiv K. Singh
          </td>
          <td>2025-07-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Comprehensive analysis of single-cell transcriptome and chromatin accessibility will contribute to interpret the heterogeneity of acute myeloid leukemia (AML). We hypothesize that integrating single-cell transcriptomic and chromatin accessibility landscapes underlying t(8;21) AML will provide valuable insights into its heterogeneous cellular properties and gene regulatory programs. Methods Here, we conducted paired single-cell RNA-sequencing (scRNA-seq) and single-cell ATAC sequencing on bone marrow samples from newly diagnosed t(8;21) AML patients and healthy controls. Genetic signatures extracted from scRNA-seq were built and validated across three independent cohorts (German AMLCG1999, GSE106291 and TCGA LAML). Results We identified TCF12, a core component of AML1-ETO-containing transcription factor complex (AETFC), as the most active transcription factor in blast cells, driving a universally repressed chromatin state. Furthermore, we delineated two functionally distinct T cell subsets, revealing that EOMES-mediated transcriptional regulation promotes the expansion of a cytotoxic T cell population (T cells_2; high GNLY, NKG7 and GZMB expression), with an increased clonality and a tendency for drug resistance. In addition, we discovered a novel leukemic CMP-like cluster characterized by high TPSAB1, HPGD and FCER1A expression. Leveraging machine learning-based integration of multi-omic profiles, we identified a robust 9-gene prognostic signature, which demonstrated significant predictive value for AML outcomes across three independent cohorts. Conclusions This multi-omics study provides unprecedented insights into the transcriptional and epigenetic heterogeneity of t(8;21) AML, providing a potential actionable tool for clinical risk stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06659-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d672d2579a9875bb647a603a272b4cc733a93" target='_blank'>
              A multi-omic single-cell landscape reveals transcription and epigenetic regulatory features of t(8;21) AML
              </a>
            </td>
          <td>
            Xue-Ping Li, Yan Gao, Bai-Tian Zhao, Yujun Dai, J. Mao, Yang Liang, Lu Jiang
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Here we present a new iPSC resource of apparently healthy subject biospecimens available to the research community through the National Institute of General Medical Sciences Human Genetic Cell Repository (NIGMS Repository). This resource includes five iPSCs and matched parental cell lines with accompanying publicly available, HiFi whole-genome sequencing data. Structural variant (SV) and single nucleotide variant (SNV) concordance between iPSC and parental lines was generally high; however, we found a notable reduction in concordance between the iPSC reprogrammed with retroviral reprogramming and its parental line consistent with previous work showing newer Sendai approaches to be more robust in preserving genomic integrity. This iPSC resource additionally includes pharmacogenomic and human leukocyte antigen (HLA) gene annotations as well as a set of user-friendly, web-based search tools to visualize and explore SVs and SNVs. This new resource is designed to offer a highly characterized set of in vitro models for research into cell-type specific functional characterization of genetic, genomic and pharmacogenomic variation. More generally, these renewable biospecimens and genomic data search tools are available to the scientific community to support high-quality and reproducible biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/664bbdaaf704bf6f26d8a71b37d0bd99ff2c3655" target='_blank'>
              New iPSC resource with long-read whole genome sequencing characterizations for enhanced in vitro modeling
              </a>
            </td>
          <td>
            Laura Scheinfeldt, Anthony Pompetti, G. Calendo, Tatyana Pozner, Christine Grandizio, Gretchen Smith, Kelly Hodges, Neda Gharani, D. Kusic, Matthew W Mitchell, N. Turan
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The dynamic nature of longitudinal tumor evolution across patients presents challenges in designing effective drugs. Here we aimed to determine tumor evolution resistance mechanisms and explore candidate drugs for specific tumor evolution types. We conducted longitudinal pharmacogenomic analysis of datasets of 73 samples in 34 patients among a National Cancer Center refractory lung cancer cohort (n = 199). Genomic profiles were determined to identify evolutionary trees in each patient, which were classified into tumor evolution groups according to the predominant truncal mutations, TP53 and epidermal growth factor receptor. These groups were categorized into persistence, extinction and expansion groups according to the status of these two clones. Pharmacogenomic profile analysis identified that XAV-939 was effective for the epidermal growth factor receptor-extinction group exhibiting epithelial-to-mesenchymal transition-activated resistance. In addition, MYC+ subclones were maintained similarly to drug-tolerant residual cells throughout the evolution period. Moreover, MYC+ lung adenocarcinoma showed a poor outcome and had higher risk of transformation to small-cell lung cancer. Furthermore, the epithelial-to-mesenchymal transition-activated and MYC+ subclones were implicated in concurrent epidermal growth factor receptor–tyrosine kinase inhibitor resistance. Finally, our drug screening identified barasertib, an aurora kinase inhibitor, as a triple-combination candidate with epidermal growth factor receptor–tyrosine kinase inhibitors and XAV-939 for MYC+ cells. This study demonstrates the utility of longitudinal pharmacogenomic analysis to develop treatment strategies according to individual tumor evolution type. The study underscores the importance of integrating genomic and pharmacogenomic profiling in clinical practice to tailor treatments according to tumor evolution type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95eff8844cf7d953e515c95691ab37e19818828b" target='_blank'>
              Longitudinal pharmacogenomic analysis of refractory lung cancer to identify therapeutic candidates for epidermal growth factor receptor–tyrosine kinase inhibitor resistance subclones
              </a>
            </td>
          <td>
            Namhee Yu, Mihwa Hwang, Beung chul Ahn, Youngjoo Lee, Sehwa Hong, Hanna Sim, Bo Ram Song, Sunshin Kim, Charny Park, Ji-Youn Han
          </td>
          <td>2025-07-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Lung cancer remains a leading cause of cancer-related mortality due to its capacity for silent metastasis and the significant challenges in achieving effective treatment. Currently, targeted therapies and chemotherapies are the primary options for advanced or inoperable lung cancer; however, their efficacy is often undermined by the cancer's ability to develop resistance through both genetic and non-genetic mechanisms. This review explores recent advances in understanding metabolic reprogramming in non-small cell lung cancer (NSCLC), focusing on its critical role in cancer progression. NSCLC cells exhibit heterogeneous activation of metabolic pathways influenced by their oncogenic mutations. Notably, their metabolic phenotypes evolve in response to environmental stressors and therapeutic pressures. Moreover, NSCLC cells engage in metabolic crosstalk with their microenvironment to enhance survival, leveraging distinct metabolic adaptations at both primary and metastatic sites. Despite extensive preclinical studies evaluating novel therapeutic strategies targeting these metabolic pathways, many have failed in clinical trials due to severe adverse effects. This is because the targeted pathways are crucial not only for cancer cells but also for normal cellular functions. Future research must prioritize approaches that selectively disrupt cancer-specific metabolic regulation to improve therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a854e0477bbe5fbd82c8f4b77d0195460cff4ca2" target='_blank'>
              The Emerging Roles of Metabolic Reprogramming in Non-Small Cell Lung Cancer Progression.
              </a>
            </td>
          <td>
            Beatriz P Peixoto, Rebecca A Clague, Joshua P Reddy, Hiromi I Wettersten
          </td>
          <td>2025-07-08</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) is a heterogeneous disease that can be molecularly classified based on the expression of the ERBB2 receptor (also known as HER2) and hormone receptors. Targeted therapies for HER2-positive BC, such as trastuzumab, antibody-drug conjugates (ADCs) and tyrosine kinase inhibitors, have improved patient outcomes but primary/acquired resistance still pose challenges that can limit treatments' long-term efficacy. Addressing these obstacles is vital for enhancing therapeutic strategies and patient care. Alternative splicing, a post-transcriptional mechanism that enhances transcript diversity (isoforms), can produce proteins with varied functions, cellular localizations, or binding properties. Here, we comprehensively characterized the HER2 alternative splicing isoforms, assessed their expression in primary BC patients and cell lines, and explored their role in resistance to anti-HER2 therapies. We expanded the catalog of known HER2 protein-coding isoforms from 13 to 90, revealing distinct patterns of protein domains, cellular localizations, and protein structures, along with their antibody-binding sites. By profiling expression in 561 primary BC samples and mass spectrometry data, we discovered a complex landscape of HER2 isoform, revealing novel transcripts that were previously unrecognized and are not assessed in routine clinical practice. Finally, the assessment of HER2 isoform expression in BC cell cultures sensitive or resistant to trastuzumab and ADCs revealed that drug-resistant cells shifted their expression toward isoforms lacking antibody-binding domains. Our results broaden the understanding of HER2 isoforms, revealing distinct mechanisms of potential resistance to anti-HER2 therapies, particularly ADCs. This expanded landscape of HER2 isoforms emphasizes the crucial role of alternative splicing investigations in advancing precision-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde1cb673888229ad9d05dcf1add5eeba27a1400" target='_blank'>
              Alternative splicing generates HER2 isoform diversity underlying antibody-drug conjugate resistance in breast cancer.
              </a>
            </td>
          <td>
            G. Guardia, C. D. Dos Anjos, Aline Rangel-Pozzo, F. F. dos Santos, A. Birbrair, P. Asprino, A. A. Camargo, P. Galante
          </td>
          <td>2025-07-15</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Breast cancer remains a prevalent and diverse disease, significantly contributing to cancer- related deaths among women worldwide. Recent advancements in molecular biology have paved the way for targeted therapies and pharmacogenomics, which are crucial for developing personalized treatment strategies. This literature review synthesizes findings from recent studies on these approaches, emphasizing clinical trials, genomic profiling, and personalized medicine. It aims to focus on studies examining targeted treatments, such as human epidermal growth factor receptor- 2 (HER2) inhibitors and CDK4/6 inhibitors, alongside pharmacogenomic data that influence drug metabolism, efficacy, and toxicity. Additionally, it examines the role of gene SNPs (Single Nucleotide Polymorphisms) correlated with treatment resistance, which have emerged as key biomarkers affecting therapeutic outcomes in breast cancer. These SNPs, found in genes involved in drug metabolism and tumor progression, contribute to variability in treatment responses and resistance in specific subtypes. They encompass various breast cancer subtypes, including hormone receptorpositive (HR+), HER2-positive, and triple-negative breast cancer (TNBC). The targeted therapies, particularly HER2 inhibitors, have markedly improved outcomes for specific subtypes. Furthermore, pharmacogenomics personalizes treatment by identifying genetic variations that affect drug response, optimizing therapy selection, and minimizing adverse effects. Despite these advancements, drug resistance remains a significant challenge, highlighting the necessity for ongoing research in molecular diagnostics and innovative therapeutic combinations. The literature suggests that precision medicine, driven by genomic profiling, pharmacogenomic data, and single nucleotide polymorphisms (SNPs) analysis, is enhancing treatment efficacy for breast cancer patients. HER2- positive and HR+ patients have especially benefitted from these targeted therapies while emerging treatments are addressing the complexities of TNBC. Additionally, genetic testing, such as BRCA1/2 mutation screening, is vital for guiding treatment decisions. Targeted therapies and pharmacogenomics have revolutionized breast cancer treatment, providing more personalized and effective care. Nevertheless, overcoming drug resistance and expanding access to genomic testing are essential for future advancements in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd705c1549c5bd8681f0a403abe74109eee0036" target='_blank'>
              Advancements in Targeted Therapies and Pharmacogenomics for Personalized Breast Cancer Treatment: The Role of Gene SNPs in Treatment Resistance.
              </a>
            </td>
          <td>
            D. Tripathi, Neal M Davies, P. S. Rajinikanth, Prashant Pandey
          </td>
          <td>2025-06-27</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eaa0126e7f9afd60d57ab46895cddf6ea4cbe3" target='_blank'>
              Pangenome-based identification of cryptic pathogenic variants in undiagnosed rare disease patients
              </a>
            </td>
          <td>
            S. S. Jang, S. Kim, S. Lee, S. Kim, J. Moon, J. Kim, J. Chae
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Dysregulated RNA modifications contribute to cancer progression and therapy resistance, yet the underlying mechanism often remains unknown. Here, we perform CRISPR-based synthetic lethality screens to systematically explore the role of RNA modifications in mediating resistance to anti-leukaemic drugs. We identify the TRMT5-mediated formation of N1-methylguanosine (m1G) in the tRNA anticodon loop as essential for mediating drug tolerance to cytarabine and venetoclax in acute myeloid leukemia (AML). TRMT5 methylates nearly all mitochondrial and nuclear tRNAs with a guanosine at position 37, but its role in promoting drug tolerance specifically depends on its mitochondrial function. TRMT5 is essential for the dynamic upregulation of mitochondrial mRNA translation and oxidative phosphorylation (OXPHOS), which are critical for sustaining drug tolerance in leukemia cells. This mitochondrial dependency correlates with therapy outcomes in leukemia patients: lower expression of electron transport chain genes is linked to poorer outcomes in a cohort of nearly 100 AML patients undergoing first induction therapy. Finally, we demonstrate that targeted depletion of TRMT5 protein using a conditional degron, in conjunction with cytarabine and venetoclax treatment, synergistically induces cell death in drug-tolerant AML cells. Thus, our study reveals TRMT5 as promising drug target for therapy-resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f65b910416a4ff6ee4895803cf1d0c098e3d11" target='_blank'>
              Disrupting tRNA modifications to target mitochondrial vulnerabilities in drug-resistant leukemia cells.
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, Sarah Mn Zimmermann, Laura Werner, D. Heid, Fu Xu, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Laia Llovera, Ollivier Alexane, Anke Heit-Mondrzyk, U. Platzbecker, C. Baldus, H. Serve, M. Bornhäuser, C. Vågbø, S. Benitah, Jeroen Krijgsveld, Eva Maria Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2025-08-01</td>
          <td>Blood</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="This study investigated whether therapeutic X-ray doses can affect the expression of mismatch repair (MMR) genes and proteins using Lynch syndrome-associated human colorectal cancer cell lines. MMR-deficient cell lines (HCT116, SW48, LoVo) and an MMR-proficient control cell line (HT29) were exposed to X rays [a 2 Gy dose or 2 Gy daily for five consecutive days (10 Gy)]. Reverse transcription quantitative real-time PCR (RT-qPCR) and Western blotting were used to detect the radiation-induced changes in the expression of RNAs and proteins, respectively. RT-qPCR revealed that MLH1 and MSH6 genes were stably expressed regardless of the MMR status of the cell line and the radiation dose. In contrast, the MSH2 gene was either up-regulated or down-regulated after 2 Gy or 10 Gy or both. The expression of PMS2 increased after 10 Gy irradiation in all MMR-deficient cell lines, even though the data were not statistically significant compared to other doses, except for the LoVo cell line. Protein expression analysed using Western blotting demonstrated that MLH1 protein expression was stable, whereas the expression of MSH2 was significantly affected by radiation exposure in both MLH1-deficient cell lines. No correlation between the expression of RNA and protein could be identified. In conclusion, radiation may have significantly differential effects on MMR RNA and protein expression when different cell lines, doses, and specific genes are considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b8835db8d3ae7872bbc2b8269596b668e92380" target='_blank'>
              The Effect of X rays on the Expression of Mismatch Repair Genes and Proteins in Lynch Syndrome Associated Human Colorectal Cancer Cell Lines.
              </a>
            </td>
          <td>
            Mingzhu Sun, L. Cruz-Garcia, Danny Freestone, Kevin Monahan, C. Badie, Yannick Comoglio, Hannah Mancey, J. Moquet, Stephen Barnard
          </td>
          <td>2025-06-20</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer therapy resistance remains a major challenge, with limited resources available for systematically studying its underlying mechanisms at the patient level. The existing databases are either restricted to bulk RNA-seq data, lack single-cell resolution, or provide limited clinical annotations, making them insufficient for in-depth exploration of the tumor microenvironment (TME) dynamics in therapy resistance. To bridge this gap, we present CellResDB, a patient-derived platform comprising nearly 4.7 million cells from 1391 patient samples across 24 cancer types. CellResDB provides comprehensive annotations of TME features linked to therapy resistance. To enhance accessibility, we include an intelligent robot, CellResDB-Robot, which facilitates intuitive data retrieval and analysis. In summary, CellResDB represents a valuable resource for cancer therapy and provides an experimental protocol for applying large language models (LLMs) within the biomedical database. CellResDB is freely available at https://cellknowledge.com.cn/cellresponse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e259dc3f38a852bd1ee92860ed4d64506f3ee24b" target='_blank'>
              Deciphering cancer therapy resistance via patient-level single-cell transcriptomics with CellResDB
              </a>
            </td>
          <td>
            Tianyuan Liu, Huiyuan Qiao, Liping Ren, Xiucai Ye, Quan Zou, Yang Zhang
          </td>
          <td>2025-07-15</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Activation of oncogenes, such as through mutations in Kirsten rat sarcoma viral oncogene homolog (KRAS), triggers profound disruptions in cellular homeostasis that set off a cascade of stress responses. These responses enable cells to cope with the array of challenges encountered during tumorigenesis by activating defense mechanisms that promote adaptation and survival. Key components of this oncogenic stress response include heat shock proteins, the ubiquitin-proteasome system, autophagy, nuclear factor erythroid 2-related factor 2–antioxidant response element signaling, DNA damage response proteins, p53, redox-regulating proteins, and stress granules. This review concentrates on KRAS-driven oncogenic transformation, as KRAS mutations are among the most common in human cancers, accounting for over 90% of pancreatic ductal adenocarcinoma cases, around 30% of lung cancers, and approximately 50% of colorectal cancers. We examine the intricate molecular interplay between oncogenic stress and the associated cellular defense mechanisms, emphasizing the key molecular events that follow KRAS activation. Importantly, the very pathways that allow cancer cells to adapt to oncogenic stress also offer novel therapeutic opportunities. By selectively targeting pivotal regulators within these stress response pathways, we can potentially disrupt the survival mechanisms of cancer cells. This strategy not only promises to enhance the effectiveness of existing treatments but also paves the way for the development of innovative therapies designed to combat tumor progression. In essence, exploiting oncogenic stress responses represents an original and promising therapeutic approach in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a19b0d4dfe04d02b6a9051c867a7e87fe82159" target='_blank'>
              Oncogenic stress response mechanisms as new therapeutic targets in cancer treatment: A review
              </a>
            </td>
          <td>
            J. Iovanna, Matías Estaras, Daniel Grasso, Martin E. Fernández Zapico, J. L. Neira, P. Santofimia-Castaño
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Breast cancer pathogenesis is closely linked to the mechanisms of cellular reprogramming, the process of converting differentiated cells into undifferentiated or different cell types. This reprogramming is achieved through the orchestrated expression of transcription factors which alter the cell’s existing chromatin landscape and establish a transcriptome corresponding to the reprogrammed cell state. Pioneer factors, a type of transcription factor, initiate cell development and cell fate transition by binding to specific DNA motifs within nucleosomes. This binding leads to the recruitment of co-factors, such as nucleosome remodelers, leading to chromatin opening and subsequent activation of specific genes essential for reprogrammed cell function. Improper engagement of these factors can lead to unanticipated or harmful gene expression, potentially contributing to various human diseases, including cancer. GATA3, a pioneer factor, is essential for normal mammary gland development and differentiation. In breast cancer cells, GATA3 has been shown to suppress tumor growth by inducing mesenchymal-to-epithelial transition (MET), the reverse process of epithelial-to-mesenchymal transition (EMT). Interestingly, although mutations in GATA3 are frequent and considered driver mutations, breast cancer patients carrying GATA3 mutations are generally associated with better patient survival. The roles of these mutations remain largely unexplored. M294R and M294K are hot spot missense mutations found in luminal breast cancer patients. These mutations occur in the DNA-binding domain of GATA3, which is crucial for its function as a transcription factor. For functional characterization of these mutants, we utilized MDA-MB-231 mesenchymal cells, a highly aggressive and invasive breast cancer cell line to model mesenchymal-to-epithelial transition (MET). Since ectopic expression of GATA3 induces MET in this cell line, these cells provide an excellent system to observe phenotypic and molecular alterations caused by its missense mutations M294R/K. To quantitatively measure changes in GATA3 mobility and its DNA binding dynamics in living cells, we utilized Fluorescence Recovery After Photobleaching (FRAP) assay. Additionally, to further elucidate the crucial role of GATA3, genomics analyses and phenotypic assays were conducted. Functional characterization of M294R/K mutants in MDA-MB-231 breast cancer cells demonstrated distinct phenotypic and molecular alterations compared to wild-type GATA3. While wild-type GATA3 induced MET, mutant-expressing cells exhibited differential responses. Remarkably, mutants derived from breast cancer patients displayed increased chromatin binding, leading to a tighter chromatin structure and differential gene expression of mesenchymal or epithelial markers. Altered cellular phenotypes such as cell migration are also observed in M294R/K expressed cells. Our finding underscores the importance of precise regulation of pioneer factor-chromatin interactions in cellular reprogramming processes. Understanding the molecular mechanisms underlying GATA3 mutations and their effects on cellular reprogramming in breast cancer provides valuable insights into the roles of GATA3 in breast cancer progression.
 Citation Format: Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku. Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ca95880c3c403f49e5578c8d1dc155a7b3a608" target='_blank'>
              Abstract P5-06-09: Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model
              </a>
            </td>
          <td>
            Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer is one of the most common malignant tumors occurring in the male reproductive system. Androgen deprivation therapy (ADT) is the primary treatment for prostate cancer. However, approximately 10–20% of advanced prostate cancer patients progress to castration-resistant prostate cancer (CRPC) within five years, posing a significant challenge to treatment. There is an urgent need to explore the causes and mechanisms of prostate cancer progression, and to identify new therapeutic targets and predictive markers. Utilize RNA-seq to screen for differentially expressed genes and alternative splicing events between androgen-sensitive prostate cancer cells and castration-resistant prostate cancer cells. Perform bioinformatics analysis on the differentially expressed genes. The expression levels of these genes were confirmed by real-time quantitative PCR (RT-qPCR); alternative splicing events related to castration resistance were verified through RT-PCR. The impact of the S100A6 gene on the proliferation and growth capabilities of PC3 and DU145 cells was analyzed using CCK8, colony formation, and EdU assays. Cell migration was assessed using a wound healing assay. The RNA-seq analysis results showed that there were 4,830 differentially expressed genes in non-hormone-sensitive prostate cancer. GO enrichment analysis of these differentially expressed genes revealed significant enrichment in tumor-related signaling pathways such as regulation of cell cycle process, cell division, and regulation of cell projection organization. Further RT-qPCR detection results indicated that the mRNA expression of four genes, including S100A6, was upregulated, consistent with the transcriptome sequencing results. Downregulation of S100A6 expression significantly inhibited the proliferation and migration of non-hormone-dependent prostate cancer. In this study, we also identified many splicing alterations in cancer-related genes, with castration resistance-induced alternative splicing events closely related to the interaction networks of regulation of cell cycle process, cell division, and regulation of cell projection organization. Our research findings suggest that S100A6 may serve as a molecular marker or therapeutic target associated with castration resistance in prostate cancer, providing new insights into the molecular basis of this disease. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-11397-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8350b9dbff378bc43ca0f122bb3ffa5b66be6348" target='_blank'>
              Screening and validation of key genes involved in castration-resistant prostate cancer based on transcriptomics sequencing
              </a>
            </td>
          <td>
            Guangquan Tong, Xiaomeng Jing, Yuning Yang, Xiaowei Wang, Junlin Lu, Jingyi Hu, Yang Wang, F. A. Jumuddin, Wenjing Zhang, Yuesheng Lv
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer remains one of the leading health threats globally, with therapeutic resistance being a long-standing challenge across chemotherapy, radiotherapy, targeted therapy, and immunotherapy. In recent years, the association between epigenetic modification abnormalities and therapeutic resistance in tumors has garnered widespread attention, spurring interest in the development of approaches to target epigenetic factors. In this review, we explore the widespread dysregulation and crosstalk of various types of epigenetic modifications, including DNA methylation, histone modifications, and non-coding RNA changes, which interact through complex regulatory networks in tumors. Clinically, single-targeted therapy based on epigenetic modification usually has its limited effect against cancer. However, the combination of epigenetic drugs with other treatment modalities, such as chemotherapy, targeted therapy, or immunotherapy, shows potential for synergistically enhancing efficacy and reducing drug resistance. Therefore, we evaluate the possibility and potential mechanisms of targeting epigenetic modifications to overcome resistance in cancer therapy, and discuss the challenges and opportunities in moving epigenetic therapy into clinical practice. Moreover, the application of multi-omics technologies will aid in identifying core epigenetic factors from complex epigenetic networks, enabling precision treatment and overcoming therapeutic resistance in tumors. Furthermore, the development of spatial multi-omics technologies, by providing spatial coordinates of cellular and molecular heterogeneity, revolutionizes our understanding of the tumor microenvironment, offering new perspectives for precision therapy. In summary, the combined application of epigenetic therapies and the integration of multi-omics technologies herald a new direction for cancer treatment, holding the potential to achieve more effective personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8def3b2fb87b962d999f696cd613916b46b863b4" target='_blank'>
              Targeting epigenetic regulators as a promising avenue to overcome cancer therapy resistance
              </a>
            </td>
          <td>
            Jiawei Song, Ping Yang, Canting Chen, Weiqun Ding, O. Tillement, Hao Bai, Shuyu Zhang
          </td>
          <td>2025-07-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cancer treatments that use drugs resembling DNA building blocks, known as nucleotide analogs, that insert themselves into DNA and prevent replication often stop working over time because cancer cells become resistant. This review explains how reversible epigenetic changes help cancer cells avoid being killed by these drugs. These changes do not alter the DNA sequence but can turn important genes on or off. We describe how cancer cells use these mechanisms to repair damaged DNA, pump drugs out of the cell, or break them down. We highlight the role of DNA packaging proteins (histones), small RNA molecules, and chemical tags on DNA that control how genes behave. Understanding these changes is important because it may allow clinicians to resensitize tumors and make them sensitive to treatment again and prevent relapse. We also explore new therapies, including drugs and gene editing tools, that target these epigenetic changes to improve the success of chemotherapy. These findings could lead to better outcomes for patients with difficult-to-treat cancers, especially those with pancreatic, colorectal, or breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeef0502ae8a946f9634749621e6cdd28f4f464" target='_blank'>
              Epigenetic Drivers of Chemoresistance in Nucleobase and Nucleoside Analog Therapies
              </a>
            </td>
          <td>
            John Kaszycki, Minji Kim
          </td>
          <td>2025-07-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c33305648009e1896ce514a1bce98da6447d96" target='_blank'>
              Epigenetic dysregulation in cancer: mechanisms, diagnostic biomarkers and therapeutic strategies.
              </a>
            </td>
          <td>
            Komal Imran, Muhammad Javed Iqbal, Muhammad Mushtaq Ahmed, Ammara Khalid, H. Cortés, O. Reyes-Hernández, Gabriela Figueroa-González, G. Leyva-Gómez, L. Falzone, Massimo Libra, Federica Longo, Javad Sharifi‐Rad, D. Calina
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Disease-associated variants can lead to variable phenotypic outcomes, but the biological mechanisms underlying this variability remain poorly understood. We developed a framework to investigate this phenomenon using the 16p12.1 deletion as a paradigm of variable expressivity. Using induced pluripotent stem cell models from affected families and CRISPR-edited lines with the 16p12.1 deletion, we found that the deletion and secondary variants in the genetic background jointly influenced chromatin accessibility and expression of neurodevelopmental genes. Cellular analyses identified family-specific phenotypes, including altered inhibitory neuron production and neural progenitor cell proliferation, which correlated with head-size variation. CRISPR activation of individual 16p12.1 genes variably rescued these defects by modulating key signaling pathways such as TGF-{beta} and PI3K-AKT. Integrative analyses further identified regulatory hubs, including transcription factors FOXG1 and JUN, as mediators of these effects. Our study provides a functional framework for investigating how individual genetic architectures contribute to phenotypic variability in genetic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eba5322d12d293981db603db996dc82168694a5" target='_blank'>
              An integrated framework for functional dissection of variable expressivity in genetic disorders
              </a>
            </td>
          <td>
            Jiawan Sun, Serena Noss, Deepro Banerjee, V. H. Bhavana, Corrine Smolen, Maitreya Das, Belinda Giardine, Anisha Prabhu, David J. Amor, K. Pope, Paul J Lockhart, S. Girirajan
          </td>
          <td>2025-07-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Therapeutic resistance to radiotherapy and chemotherapy remains a major challenge in non-small cell lung cancer treatment, limiting the long-term efficacy of conventional therapies. This review provides a comprehensive overview of DNA repair regulation beyond canonical pathways, elucidating novel upstream modulators, downstream effectors, and the intricate network of their interactions. We further highlight potential therapeutic targets, emerging treatment strategies, and current obstacles to clinical translation. These insights offer innovative directions to refine precision medicine approaches and contribute to the development of more effective, individualized therapies for patients facing resistant disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8b2d50d7a892a1da7f8f591a57318c14a3b2f5" target='_blank'>
              Targeting DNA Damage Response-Mediated Resistance in Non-Small Cell Lung Cancer: From Mechanistic Insights to Drug Development
              </a>
            </td>
          <td>
            Xue Gong, Yongzhao Zhou, Yi Deng
          </td>
          <td>2025-06-23</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcaffd86557cddf1faae3fdc7b2f95bd830f2b9" target='_blank'>
              Pan-cancer analysis in the real-world setting uncovers immunogenomic drivers of acquired resistance post-immunotherapy
              </a>
            </td>
          <td>
            Mohamed Reda Keddar, Sebastian Carrasco Pro, Roy Rabbie, Z. Atak, A. Stewart, S. Hammond, Douglas C. Palmer, Ross Stewart, Kathleen Burke, Benjamin Sidders, Jessica Davies, Jonathan R Dry, I. Martincorena, Sajan Khosla, A. Schoenfeld, Martin L. Miller
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background Osteosarcoma (OS) shows a multitude of genetic and chromosomal abnormalities together with large biological heterogeneity. These features limited the identification of novel drugs to treat patients with metastases and/or chemo-resistant tumors. The purpose of this study was to create additional resources for drug screening by generating patient-derived xenograft (PDXs) and PDX-derived cell lines that reflect the spectrum of OS heterogeneity. Methods PDXs were derived from OS collected at diagnosis, surgical resections, or metastases. PDX-derived cell lines were also established. Targeted DNA sequencing and digital PCR were applied to identify major genetic alterations. High-throughput drug screening by using a library of 2880-FDA approved compounds and conventional MTT assays were performed to identify the most effective drugs against in vitro and in vivo growth of chemo-resistant OS. Results Targeted DNA sequencing demonstrated alterations in the most commonly amplified oncogenes, such as MYC, CCNE1, DDR2, CDK4, MDM2, and AURKA. Recurrent deletions and SNVs were found in TP53, CDKN2A, RB1, PTEN, and VHL. Copy number variant (CNV) alterations in PDXs, PDX-derived cell lines and xenografts developed from cell lines (CDX) correlated very well with those observed in the matched original human tumors. Drug screenings identified and repurposed five compounds with efficacy against chemoresistant OS. In this context, we prioritized ixabepilone as a drug capable of inducing tumor regression in mice. Conclusions We enriched the scientific community with additional, molecularly characterized OS models to be used for testing novel therapies and supported the inclusion of ixabepilone into treatment plans for chemoresistant OS. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03440-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23016fdd463bf1bcdb3a8bfe081d0f07e246c765" target='_blank'>
              Genomic profiling of a collection of patient-derived xenografts and cell lines identified ixabepilone as an active drug against chemo-resistant osteosarcoma
              </a>
            </td>
          <td>
            M. Manara, F. Bruzzese, Laura Formentini, L. Landuzzi, L. Pazzaglia, M. Laginestra, Marianna Carrabotta, M. Roca, Federica Iannelli, Laura Grumetti, L. Addi, A. Parra, C. Cristalli, M. Pasello, Alberto Bavelloni, Francesca Carreras, F. Ruzzi, G. Bianchi, M. Gambarotti, A. Righi, Alfredo Budillon, Pier-Luigi Lollini, Katia Scotlandi
          </td>
          <td>2025-07-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Growth hormone (GH) signaling has been implicated in tumor progression and therapy resistance across multiple cancer types, yet its role in bladder cancer remains largely unexplored. In this study, we investigated the impact of GH and its receptor (GHR) on therapy resistance and disease progression in urothelial carcinoma (UC) through integrated transcriptomic and in vitro analyses. Transcriptomic profiling of The Cancer Genome Atlas bladder cancer cohort revealed that high tumoral GHR expression was associated with differential upregulation of genes involved in drug efflux, epithelial-to-mesenchymal transition (EMT), and extracellular matrix (ECM) remodeling. Notably, elevated GHR levels correlated with significantly reduced overall survival in patients with UC. In parallel, in vitro experiments demonstrated that GH promotes chemoresistance in UC cell lines via upregulation of ATP-binding cassette-containing (ABC) transporters and activation of EMT. GH also modulated ECM-remodeling-associated genes in a chemotherapy-dependent manner, including matrix metalloproteinases and tissue inhibitors of metalloproteinases. Importantly, these effects were abrogated by Pegvisomant, a GHR antagonist, indicating the functional relevance of GH/GHR signaling in the mediation of these phenotypes. Collectively, our findings support a mechanistic role for GH signaling in driving therapy resistance and tumor aggressiveness in bladder cancer and suggest GHR antagonism as a potential therapeutic strategy to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f8b97219a1f5f0a9e16e7142057f197a9d88cb" target='_blank'>
              Growth Hormone Signaling in Bladder Cancer: Transcriptomic Profiling of Patient Samples and In Vitro Evidence of Therapy Resistance via ABC Transporters and EMT Activation
              </a>
            </td>
          <td>
            Emily Davis, Lydia J. Caggiano, Hannah Munholland, R. Basu, D. Berryman, J. Kopchick
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2aba01b3716bf8fef6d8f3f3f6e3783dfc01cf" target='_blank'>
              Recent advances in novel targeting mechanisms for colorectal cancer.
              </a>
            </td>
          <td>
            Zhiping Li, Hailing Wang, Yan Liu, Ahequeli Gemingnuer, Yinan Wang, Xin Meng
          </td>
          <td>2025-07-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6adfd64895599aa6bae6a1f8896e93b0d56bc5ac" target='_blank'>
              ADAR1-mediated RNA editing in breast cancer: molecular mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Si-Yu Chen, Si-Ying Chen, Shu Yang, Yi Li, 
          </td>
          <td>2025-08-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423474067572f39ac510a609277d4e54dd172eb6" target='_blank'>
              YAP signaling promotes resistance to MEK and AKT inhibition in NF1-related MPNSTs
              </a>
            </td>
          <td>
            L. E. McGee, J. Grit, Curt J. Essenburg, Sophia Agrusa, Elizabeth A. Tovar, Lisa Turner, K. Becker, J. Zbieg, Anwesha Dey, Jennifer E. Klomp, Jeffrey A. Klomp, Marie Adams, Ian Beddows, Emily Wolfrom, Zhen Fu, A. Hirbe, Carrie R. Graveel, Matthew R. Steensma
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common types of malignancy diagnosed globally. Metastasis plays a major role in most of the cancer-related mortality among affected patients. Despite the advances in the areas of early detection and localized treatment modalities, there prevail several challenges which the therapeutic strategies encounter, like drug resistance, tumor heterogeneity, and drug delivery. This review presents a comprehensive and detailed overview of organ-specific metastasis that occur in BC, specifically emphasizing key sites such as the bone, liver, lung, and brain. It also outlines the significance of various therapies like chemotherapies, endocrine therapies, targeted therapies and immunotherapies that have been clinically approved to date. The review specifically emphasizes the molecular mechanisms by which non-coding RNAs (ncRNAs) act to exert their effects in regulating drug resistance. It also addresses the new advances in nanotechnology-based drug delivery systems (DDS) that function to enhance the specificity of treatments while simultaneously reducing systemic toxicity. Beyond ncRNAs, this review also explores other critical mechanisms of drug resistance in metastatic BC, including efflux transporter activity, target gene mutations, and micro-environmental factors, to mention a few. Moreover, the review also discusses the clinical significance of combination therapies and new therapeutic strategies, including the use of repurposed drugs and the concepts of personalized medicine. A greater understanding of the ncRNA-mediated signaling pathways, in combination with the latest advances in drug delivery systems, has the potential to greatly improve therapeutic efficacy and could result in more favorable clinical outcomes in the treatment of metastatic BC (MBC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2fe23da14c71b938c6019fd6995e4bfcec570b" target='_blank'>
              Bridging molecular insights and clinical application: non-coding RNAs, targeted drug delivery, and metastatic breast cancer therapy
              </a>
            </td>
          <td>
            Sohini Chakraborty, Satarupa Banerjee
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3b7231b6624e72e968c03b2d88845df441192" target='_blank'>
              ABCC10-mediated cGAMP efflux drives cancer cell radiotherapy resistance.
              </a>
            </td>
          <td>
            Zhengyang Zhang, Jie Gao, Xiang Liao, Zining Zhang, Xiongfeng Cao, Yi Gong, Wenlong Chen, Lirong Zhang, Hsiang-I Tsai, Dongqing Wang, Haitao Zhu
          </td>
          <td>2025-08-06</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 The combination of CDK4/6 inhibitors (CDK4/6i) with endocrine therapy prolongs survival of patients with ER+/HER2- metastatic breast cancer. Nearly all patients with advanced disease treated with this combination eventually progress, highlighting a need for therapeutic interventions that prevent the emergence of drug resistance. Drug tolerant persister (DTP) cells represent a reservoir of surviving cells from which drug-resistant clones eventually emerge. We hypothesize that targeting the DTP state which emerges under selective pressure from CDK4/6i could prevent the emergence of drug resistance. In vitro treatment of MCF7 and T47D ER+/HER2- breast cancer cells with the CDK4/6i Palbociclib and estrogen deprivation for 4 weeks suppressed cell viability with 2-5% of cells surviving treatment and achieving a DTP state. Cell cycle analysis by PI staining revealed that 95% of the DTP cells were arrested in G0/G1 after 4 weeks of continuous therapy. Upon drug washout, the DTP population resumed proliferation within ∼1 week and regained sensitivity to retreatment with CDK4/6i, suggesting epigenetic reprogramming is responsible for the DTP state. Whole exome sequencing did not show enrichment of known genomic alterations associated with CDK4/6i resistance (e.g., RB1, FAT1, PTEN). Bulk RNA sequencing of MCF7 DTP cells revealed transcriptome reprogramming compared to untreated controls, which reverted upon drug washout to that of the treatment naïve state. While cell cycle-related gene signatures such as E2F targets and G2M checkpoint were downregulated, interferon response signatures were upregulated in MCF7 DTP cells relative to untreated controls. MCF7 persisters were characterized by enrichment in senescence and chemotherapy-induced stressed gene signatures, while diapause and MYC target signatures were downregulated. These same signatures were similarly enriched in ER+/HER2- tumors from patients in the NeoPalAna study treated with neoadjuvant palbociclib and the aromatase inhibitor anastrozole compared to baseline biopsies. To study evolution of drug resistance, we transduced MCF7 and T47D cells with the LARRY (Lineage and RNA recovery) barcode library to uniquely label and track each persister cells long-term, and identify programs that are causal to the emergence of drug resistant clones. We performed single cell RNA-seq on barcoded cells treated with palbociclib plus estrogen deprivation for 0 h, 72 h, and 4 weeks. Several clones that arrested in G0/G1 initially eventually expanded at 4 weeks, while other clones were eliminated. We are currently investigating which genes/pathways are upregulated in the barcoded clones enriched at the 4-week time point. We will then validate results from these single cell approach in cells surviving neoadjuvant treatment in the NeoPalAna trial. In summary, a small subset of ER+/HER2- breast cancer cells survive long-term treatment with CDK4/6i plus estrogen deprivation and exhibits reversible transcriptomic features of DTPs. Our ongoing single cell-seq analyses may reveal transcriptional programs that enable the DTP state which, in turn, could be targeted to reduce survival of drug-tolerant cells and prevent the emergence of drug resistance.
 Citation Format: Yuki Matsunaga, Hima Milan Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, Dan Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, Ariella B. Hanker. Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-30.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a0b4864d21ce42a1a87671ea1a257909c574fe" target='_blank'>
              Abstract P3-04-30: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, H. M. Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6c0eb75e7f5a0febc3a26c4b5b85dcca6192f2" target='_blank'>
              Validation of the MAGIC genomic signature on RNAseq in soft-tissue sarcomas
              </a>
            </td>
          <td>
            A. Benhaddou, G. Pérot, P. Rochaix, T. Valentin, G. Ferron, F. Chibon
          </td>
          <td>2025-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells utilize codon-biased translation to fuel tumorigenesis and drug resistance, but underlying mechanisms remain poorly understood. Here, we show ALKBH1 is overexpressed in acute myeloid leukemia (AML) and essential for leukemia stem/initiating cell (LSC/LIC) self-renewal and AML development/maintenance, whereas dispensable for normal hematopoiesis. ALKBH1 enhances mitochondrial assembly/function and oxidative phosphorylation (OXPHOS), crucial for AML survival/proliferation and resistance to venetoclax, a potent BCL2 inhibitor and widely-used first-line targeted therapy for AML in clinic. Mechanistically, ALKBH1 catalyzes 5-formylcytosine (f5C) at tRNA wobble positions, reprograming decoding and facilitating codon-biased translation, a mechanism we term "Epitranslatomic Midas touch", which in turn drives leukemogenesis and drug resistance by promoting synthesis of key oncogenic proteins like WDR43. Targeting ALKBH1, particularly together with venetoclax, exhibited potent anti-leukemia efficacy in preclinical models with favorable safety profiles. Collectively, our findings elucidate ALKBH1's pivotal role in codon-biased translation and tumorigenesis, and propose a novel therapeutic strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3e11cf9a76aba0c7266c1034a7a9bfa25e9f8a" target='_blank'>
              ALKBH1 drives tumorigenesis and drug resistance via tRNA decoding reprogramming and codon-biased translation.
              </a>
            </td>
          <td>
            Chao Shen, Yuan Che, Keren Zhou, Kitty Wang, Wei Li, Diyuan Xue, Tong Wu, Lu Yang, Meiling Chen, Yue Sheng, Chengwan Zhang, S. Robinson, Huiying Chen, Lillian Sau, Zhenhua Chen, M. Wunderlich, Li Han, Tingting Tang, Y. Qing, Dong Wu, Miao Sun, Keith Leung, Katarzyna Dąbrowska, P. Pirrotte, Yueh-Chiang Hu, Lucy Y Ghoda, Bin Zhang, J. Mulloy, Minjie Wei, Guido Marcucci, Chuan He, Xiaolan Deng, Jianjun Chen
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="The role of G-quadruplexes (G4s) in gene regulation has been widely documented, especially in gene promoters. However, the transcriptional mechanisms involving G4s in other regulatory regions remain largely unexplored. In this study, we integrated the G4-DNA data derived from 22 breast cancer patient-derived tumor xenograft (PDTX) models and MCF7 cell line as potential breast cancer-associated G4s (BC-G4s). Genome-wide analysis showed that BC-G4s are more prevalent in gene promoters and the first introns. The genes accommodating promoter or intronic BC-G4s show significantly higher transcriptional output than their non-G4 counterparts. The biased distribution of BC-G4s in close proximity to the transcription start site (TSS) is associated with an enrichment of transcription factor (TF) interactions. A significant negative correlation was detected between the G4–TF interactions within the first introns and their cognate promoters. These different interactions are complementary rather than redundant. Furthermore, the differentially expressed genes (DEGs) harboring promoter and first intron BC-G4s are significantly enriched in the cell cycle pathway. Notably, promoter BC-G4s of DEGs could be a central hub for TF–TF co-occurrence. Our analysis also revealed that G4-related single nucleotide variants (SNVs) affect the stability of G4 structures and the transcription of disease-related genes. Collectively, our results shed light on how BC-G4s within promoters and first introns regulate gene expression and reinforce the critical role of G4s and G4-related genes in breast cancer-associated processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6951122fcae4ae20d0e429a03632112d1f8ad46" target='_blank'>
              The Relevance of G-Quadruplexes in Gene Promoters and the First Introns Associated with Transcriptional Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Huiling Shu, Ke Xiao, Wenyong Zhu, Rong Zhang, Tiantong Tao, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The treatment of choice for prostate cancer is androgen deprivation (ADT) and novel hormonal agents such as Abiraterone, Enzalutamide, or Apalutamide. Initially, this therapy is highly effective, but a significant challenge arises as most patients eventually develop resistance, resulting in castration-resistant prostate cancer (CRPC). Furthermore, the sequential use of these drugs can lead to cross-resistance, diminishing their efficacy. Tumor heterogeneity plays a pivotal role in the development of resistance to different treatments. This study utilized cellular models of CRPC to assess the response to Apalutamide when it was administered as a second- or third-line treatment. Functional and genetic analyses were conducted in various CRPC cell models exposed to Apalutamide. These analyses included real-time cell monitoring assays, flow cytometry, clonogenicity assays, and RT-qPCR. CRPC cell models were capable of continued proliferation, maintained cell cycle profiles similar to those of untreated cells, and retained their clonogenic potential. Cross-resistance to Apalutamide in models of ADT, ADT plus Enzalutamide, or Abiraterone resistance did not correlate with the expression levels of AR-V7 and AR-V9 variants. Gene expression analysis of resistant prostate cancer cell lines revealed that treatment with Apalutamide induced the emergence of more aggressive phenotypes, including cancer stem cells or neuroendocrine differentiation profiles. Most CRPC cell models developed cross-resistance to Apalutamide and were able to proliferate and retain their clonogenic capability. Apalutamide resistance was not linked to the expression of AR-V7 or AR-V9 variants but was instead associated to bypass of AR signaling pathway and the emergence of more aggressive expression profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e7f49946c95ade49f84698155e95968426b19" target='_blank'>
              Phenotypic Plasticity and Androgen Receptor Bypass Drive Cross-Resistance to Apalutamide in Castration-Resistant Prostate Cancer Cell Models
              </a>
            </td>
          <td>
            Iris Simon, Jose Manuel Sanchez-Manas, Sonia Perales, G. Martínez-Navajas, J. Ceron-Hernandez, Pedro J. Real
          </td>
          <td>2025-06-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Introduction: Survival rates for patients with breast cancer have improved significantly with widespread screening and effective systemic therapies. However, patients who present with advanced nodal stage at diagnosis have higher rates of distant relapse and require more aggressive therapies than patients with pathologic N1 disease (pN1). Understanding the molecular alterations that occur in primary breast cancer (BC) that drive progression to a higher nodal stage (>pN1) could uncover potential therapeutic targets for patients with regionally metastatic BC. Given that epigenetic mechanisms represent that crucial layer of dynamic change that regulates genome-wide gene expression, we assessed the DNA methylation profiles of primary BC at various nodal stages. Methods: We conducted an exploratory study using a well-annotated, retrospective cohort of ER+/HER2- treatment-naive primary BC from patients with clinically positive nodes and complete pathologic nodal data treated at Duke University (n=49, 30 cases pN1 vs. 19 cases >pN1). DNA was extracted from paraffin-embedded samples, and DNA methylation profiling was performed using the Illumina EPIC BeadChip v1 arrays. The ChAMP package was utilized to correct batch effect, and the Wilcoxon test was used to identify Differentially Methylated Sites (DMS). Tumor content purity was assessed using the LUMP algorithm and Gene Ontology enrichment of the DMSs was examined with the GOmeth tool. Data analysis and visualization were performed using the R packages ggplot2 (v. 3.4.4), circlize (v. 0.4.16), and gplots (v. 3.1.3.1), with data management facilitated by the tidyverse (v. 2.0.0) collection. Results: Tumor purity was similar between the two groups (p>0.05). The comparison revealed distinct DNA methylation patterns in 962 genomic regions between pN1 and >pN1 tumors (differential methylation > 15%, p<0.01); 683 regions were hypermethylated and 279 were hypomethylated in the >pN1 group. There was a significant enrichment of biological processes associated with morphogenesis among the hypermethylated sites, with 22 of the 43 affected molecular pathways linked to cellular morphogenesis. Additionally, epigenetic alterations were predominantly found in pathways related to the response to bone morphogenic protein (BMP). Expression-methylation Quantitative Trait Loci (emQTLs) were linked to epithelial to mesenchymal transition (EMT) genes in >pN1 tumors. Finally, higher methylation of the protocadherin cluster genes (PCDHGA) and regions encompassing DUSP6 and KRT7 genes were identified in tumors of higher nodal stage. These findings suggest inherent or acquired epigenetic alterations in primary BC associated with higher nodal stage. Conclusions: Understanding the mechanisms underlying lymph node invasion in breast cancer progression is crucial for improving the clinical management of patients with BC. This study indicates that ER+/HER2- primary tumors from patients with higher nodal involvement have an epigenetic predisposition to activate morphogenic differentiation and EMT more efficiently than patients with lower nodal involvement. Deeper insight into these alterations could contribute to more accurate prognostic tools, the development of novel therapeutic approaches, and potentially help avoid more aggressive axillary procedures.
 Citation Format: Sookyung Ahn, Andrés F. Bedoya-López, Miquel Ensenyat-Mendez, Pere Llinàs-Arias, Diego M. Marzese, Maggie L. DiNome. Epigenetic Alterations Affecting Cell Morphogenesis and Tumor Plasticity Pathways in Primary Tumors of Breast Cancer Patients with High Nodal Involvement [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-11-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de4a53a7cbc29ade67d8866b12ed7218a58382b" target='_blank'>
              Abstract P4-11-09: Epigenetic Alterations Affecting Cell Morphogenesis and Tumor Plasticity Pathways in Primary Tumors of Breast Cancer Patients with High Nodal Involvement
              </a>
            </td>
          <td>
            Sookyung Ahn, Andrés F. Bedoya-López, Miquel Ensenyat-Mendez, P. Llinàs-Arias, D. Marzese, Maggie L. DiNome
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cellular senescence is a stable form of cell cycle arrest that contributes to aging and age-associated diseases through the secretion of inflammatory factors collectively known as the senescence-associated secretory phenotype (SASP). While senescence is driven by transcriptional and epigenetic changes, the contribution of higher-order genome organization remains poorly defined. Here, we present the highest-resolution Hi-C maps (∼3 kb) to date of proliferating, quiescent, and replicative senescent (RS) human fibroblasts, enabling a comprehensive analysis of 3D genome architecture during senescence. Our analyses reveal widespread senescence-associated remodeling of chromatin architecture, including extensive compartment and subcompartment switching toward transcriptionally active states, and a dramatic increase in unique chromatin loops. These structural features correlate with local DNA hypomethylation and are largely independent of canonical CTCF binding. The altered 3D genome landscape supports expression of SASP genes, inflammation-related pathways, and neuronal gene signatures consistent with age-associated epigenetic drift. We further demonstrate that architectural changes at multiple levels, including compartments, subcompartments, and loops, facilitate the derepression of LINE-1 retrotransposons, linking 3D chromatin structure to activation of proinflammatory transposable elements. Interestingly, quiescent cells, commonly used as senescence controls, exhibited substantial overlap in inflammatory gene expression with senescent cells, raising important considerations for experimental design. Structural analysis of cell cycle genes showed distinct chromatin configurations in senescence versus quiescence, despite similar transcriptional repression. Together, our results establish a high-resolution framework for understanding how genome architecture contributes to the senescent state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hypoxia, characterized by a reduction in tissue oxygen levels, is a hallmark of many solid tumors and affects a range of cellular processes, including DNA repair. In low-oxygen conditions, cancer cells often suppress key DNA repair pathways such as homologous recombination (HR), leading to the accumulation of DNA damage and increased genomic instability. These changes not only drive tumor progression but also contribute to resistance against conventional therapies. Hypoxia significantly reduces the effectiveness of oxygen-dependent treatments, including radiotherapy and many chemotherapeutic agents. To address this limitation, bioreductive drugs have been developed that become selectively activated in hypoxic environments, providing targeted cytotoxic effects within oxygen-deprived tumor regions. Additionally, the rapid growth of tumors often results in disorganized and inefficient vasculature, further impairing the delivery of oxygen and therapeutic agents. This review explores the molecular mechanisms by which hypoxia disrupts DNA repair and contributes to treatment resistance. It also presents emerging therapeutic strategies aimed at targeting the hypoxic tumor microenvironment to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34398b4aa06a09890c501084c80149cba79331e4" target='_blank'>
              Influence of Hypoxia on Tumor Heterogeneity, DNA Repair, and Cancer Therapy: From Molecular Insights to Therapeutic Strategies
              </a>
            </td>
          <td>
            Dominika Kunachowicz, Paulina Tomecka, Mikołaj Sędzik, Jarosław Kalinin, Jacek Kuźnicki, Nina Rembiałkowska
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a09fe0116b674341564f20d065726804460dc81" target='_blank'>
              Identification of hub genes and potential molecular mechanisms of tumor immune microenvironment-related radiotherapy sensitivity in locally advanced cervical cancer.
              </a>
            </td>
          <td>
            Yichen Zan, Qing Liu, Yue Yin, Chongxin Guo, Jing Xu, Junhan Gao, Guanglu Dong
          </td>
          <td>2025-06-30</td>
          <td>Reproductive sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Aurora kinase A (AURKA) is aberrantly expressed in a large number of tumors and promotes tumor progression by regulating the cell cycle, chromosomal instability, and drug resistance. However, its heterogeneous expression and combination therapy benefit in prostate adenocarcinoma (PRAD) is unclear. Methods In this study, we integrated TCGA pan-cancer multi-omics data and GEO data to analyze the RNA, methylation, protein expression, and genomic alteration characteristics of AURKA. We then used single-cell RNA sequencing to resolve the functional heterogeneity of AURKA in the PRAD epithelial cell subpopulation and verified its impact on the malignant phenotype of desmoplasia-resistant prostate cancer cells in in vitro experiments. This research also analyzed the prognostic risk stratification of AURKA subpopulations in combination with various indicators and the potential benefit of AURKA inhibitors in combination with various treatments. Results The pan-cancer analysis demonstrated that AURKA expression heterogeneity was present among urological tumors at different molecular levels, and the positive correlation of AURKA alteration with MYC and E2F pathways was conserved in pan-cancer. Epithelial cell subpopulations with high expression of AURKA (epi3/4/6) promoted proliferation by regulating cell cycle and DNA repair, while low expression subsets (epi1/2/7) activated TNF-α and androgen receptor (AR) pathways to mediate drug resistance. In particular, AURKA may serve as a compensatory pathway to support tumor activity after AR inhibition in prostate cancer, a complex mechanism not seen in other tumors. AURKA-overexpressing patients with low Gleason scores or high PSA have a poor prognosis in clinical analysis. Furthermore, a comprehensive drug sensitivity co-analysis found that AURKA inhibitors may benefit from targeted therapy, ADC therapy, and immunotherapy. TMB and CD274 expression were the biomarkers of AURKA high-expression patients with PRAD for clinical outcome. Conclusion AURKA expression heterogeneity has been identified as a critical factor in the progression of PRAD and the development of drug resistance. The molecular subtyping of AURKA can serve as a precise strategy for combination therapy and provide a theory for the combination of AURKA inhibitors and targeted/immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396eba19ede00209d3509820e66ea1756b82e606" target='_blank'>
              Aurora kinase-a expression heterogeneity and potential benefit of combination therapy in prostate adenocarcinoma
              </a>
            </td>
          <td>
            Ru Chen, Qianyi Qiu, Weiting Xie, Jun Lin, Rong Liu, Jianhui Chen, Shaoxing Zhu, Yiming Su
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Copy number gene amplification and associated overexpression of driver oncogenes are genetic events that contribute to cancer progression and drug resistance. MDS1 and EVI1 Complex locus (MECOM) gene is copy number amplified and overexpressed in aggressive epithelial ovarian cancers. The biological function and precise molecular mechanism of MECOM in the progression and drug resistance of ovarian cancer remain unclear. Here, we unravel MECOM as a regulator of KRAS and its downstream MAP Kinase signalling pathway, and also identify epigenetic inhibitor JIB-04 as a pharmacological agent targeting MECOM/KRAS axis. RNAi-mediated attenuation of MECOM in ovarian cancer cells harboring MECOM amplification reduced their proliferation, impaired colony formation, and impeded cellular migration. ChIP-qPCR analysis confirmed binding of MECOM to the KRAS promoter, suggesting direct regulation of the KRAS gene at the transcriptional level. Further, MECOM promoted cellular proliferation by regulating KRAS-mediated ERK/ZEB1 signalling cascade. The anti-tumorigenic effects due to MECOM loss were phenocopied by the treatment of ovarian cancer cells harboring MECOM amplification with JIB-04 epigenetic inhibitor targeting Jumonji domain histone demethylase enzymes. By ChIP-qPCR, we show that JIB-04 induced transcriptional changes of MECOM by altering H3K27me3 demethylation at its promoter region. We further report that ovarian cancer cells expressing high-MECOM levels exhibit cisplatin resistance, which could be effectively reversed upon pre-treatment with JIB-04. The therapeutic efficacy of JIB-04 was further demonstrated in mice bearing ovarian cancer cell xenografts, where JIB-04 slowed down the tumor growth in corroboration with diminishing MECOM expression. RNA-sequencing analysis identified potential cisplatin resistance gene, SUB1, being regulated by JIB-04-mediated modulation of MECOM expression. Altogether, these data suggest that epigenetic silencing of MECOM by JIB-04 mediated H3K27me3 modulation is an important mechanism in ovarian cancer and provide a new therapeutic target for the treatment of ovarian cancers harboring MECOM amplification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a776f5b72c3bc81fd8c696f0bdd3032a4ca281b9" target='_blank'>
              Epigenetic targeting of MECOM/KRAS axis by JIB-04 impairs tumorigenesis and cisplatin resistance in MECOM-amplified ovarian cancer
              </a>
            </td>
          <td>
            Ibha Singh, Amarnath Karna, Anita Prajapati, Ujjawal Solanki, Archana Mukherjee, Sheetal Uppal, Pawan Malhotra, Manoj Kumar, P. Agarwal
          </td>
          <td>2025-07-15</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Proliferating cell nuclear antigen (PCNA) is a central regulator of numerous cellular processes impacting DNA metabolism and genetic integrity, including DNA damage, cell cycle progression, and transcriptional regulation. PCNA aberrations manifest in different disease phenotypes, including neoplastic proliferation, chromatin disassembly, genomic instability, and impaired DNA repair. Although the role of PCNA in control of DNA homeostasis has been extensively studied in some cancers, its pan-cancer relevance in tumorigenesis, immune interactions, and therapeutic resistance remains underexplored. Here, we undertook a comprehensive analysis of publicly available databases to assess the relationship between PCNA expression and the immunological, survival, genetic, functional, and drug sensitivity profiles across multiple cancer types. PCNA mRNA levels were altered in across cancers and associated with altered cancer signaling networks constituting Wnt, Hippo, and mTOR pathways. Increased transcript levels were associated with poor overall survival in some cancers, including ACC, CESC, LGG, and THYM. Amplification was the predominant genetic alteration of PCNA in multiple tumors. In several tumors, upregulation of PCNA was linked with differences in tumor-infiltrating lymphocytes, and specific immune-inhibitors, and chemokines. Moreover, elevated PCNA expression was linked with increased sensitivity to several drugs, particularly to Navitoclax, NPK76-II-72-1, and Ciclopirox across cancers. Our study orients the first comprehensive pan-cancer analysis of PCNA, uncovering its prognostic significance and altered expression across various cancers through multiomics data. Unlike previous studies, tumor-specific genetic alterations, such as amplification and hypomethylation, and the paradoxical immune microenvironment linked to PCNA were explored, suggesting potential immune evasion mechanisms. Additionally, new therapeutic avenues reveal PCNA’s relationship with drug sensitivity to agents like Navitoclax and Ciclopirox, providing invaluable insights for pharmacological interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3571f2ad7f72f9a390320cd46deb0b77021902" target='_blank'>
              PCNA in Pan-Cancer: A Prognostic Biomarker Unveiled Through a Data-Driven, Multidimensional Analysis of Transcriptomics, Immunity, and Functional Profiling
              </a>
            </td>
          <td>
            Depanshi Pandit, Ravindranath Bilachi Sanganabasappa, Amardeep Dhillon, Sanjiban Chakrabarty
          </td>
          <td>2025-06-26</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Prostate cancer continues to be the most common cause of cancer-related disease and mortality among men worldwide, especially in the advanced stages, notably metastatic castration-resistant prostate cancer (mCRPC), which poses significant treatment challenges. Docetaxel, a widely used chemotherapeutic agent, has long served as the standard treatment, offering survival benefits and mitigation. However, its clinical impact is frequently undermined by the development of chemoresistance, which is a formidable challenge that leads to treatment failure and disease progression. The mechanisms driving docetaxel resistance are diverse and complex, encompassing modifications in androgen receptor signaling, drug efflux transporters, epithelial-mesenchymal transition (EMT), microtubule alterations, apoptotic pathway deregulation, and tumor microenvironmental influences. Recent evidence suggests that extracellular RNAs influence drug responses, further complicating the resistance landscape. This review offers a broad discussion on the mechanisms of resistance and explores novel therapeutic approaches to address them. These include next-generation taxanes, targeted molecular inhibitors, immunotherapies, and combination regimens that can be designed to counteract specific resistance pathways. By broadening our understanding of docetaxel resistance, this review highlights potential strategies to improve therapeutic efficacy and the potential to enhance outcomes in patients with advanced treatment-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/538206b7fcdd2ed8debda33969b113a8ee54b834" target='_blank'>
              Beyond Docetaxel: Targeting Resistance Pathways in Prostate Cancer Treatment
              </a>
            </td>
          <td>
            Tayo Alex Adekiya
          </td>
          <td>2025-08-01</td>
          <td>BioChem</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c29c85da72e04b5001274eb1b2f06a493d988d5" target='_blank'>
              Reprogramming of Cellular Plasticity via ETS and MYC Core-regulatory Circuits During Response to MAPK Inhibition in BRAF-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            H. Lee, Zhao Zheng, Alexey Sorokin, Chi Wut Wong, Stefania Napolitano, S. Chowdhury, Preeti Kanikarla, A. Singh, Veena Kochat, Christopher Bristow, Sanjana Srinivasan, Mike Peoples, Emre Arslan, J. Alshenaifi, O. Villarreal, V. Morris, J. P. Shen, F. Meric-Bernstam, Abhinav K. Jain, Natalie W Fowlkes, Amanda Anderson, D. Menter, A. Saw, Kunal Rai, S. Kopetz
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1683b65b8227a142ec0662e619dfd830c76c770" target='_blank'>
              Major effect mutations drive DNA methylation variation after colonization of a novel habitat
              </a>
            </td>
          <td>
            Johan Zicola, Emmanuel Tergemina, Mehmet Göktay, Célia Neto, Robert J. Schmitz, Angela M. Hancock
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Cancer stem cells (CSCs) play a key role in tumor initiation, progression, and resistance to conventional therapies. These cells possess the ability to self-renew and differentiate, contributing to tumor heterogeneity and recurrence. Preclinical data indicate that targeting cancer stem cell-specific pathways could lead to more effective treatments and prevent relapse, thereby improving outcomes. However, there is limited clinical data to support this, in part due to the challenges of measuring CSCs in the clinical setting. Single-cell RNA sequencing (scRNAseq) data in combination with stemness gene expression signatures were investigated as a novel approach to detect and assess changes in CSC numbers in response to treatment. Methods: Single-cell RNAseq datasets from two clinical trials were interrogated – OPPORTUNE, a window-of-opportunity trial evaluating anastrozole vs anastrozole plus pictilisib (PI3K inhibitor) in 75 patients with ER+ HER2- early breast cancer, and FELINE, a neoadjuvant trial which compared letrozole plus placebo with letrozole plus ribociclib (CDK4/6 inhibitor) in 120 patients with ER+ HER2- early breast cancer. The primary endpoint for OPPORTUNE was the inhibition of tumor cell proliferation as measured by Ki67. The primary endpoint for FELINE was the rate of preoperative endocrine prognostic index (PEPI) score 0 after neoadjuvant endocrine therapy. Thirty-four patients from the FELINE study had available scRNAseq data for analysis; this number was 62 patients for the OPPORTUNE study. Stemness gene expression signatures with published evidence of selectivity for breast CSCs were identified from the literature and their utility in detecting CSCs compared in the datasets. Subsequently, changes in CSC fraction with treatment was assessed. Results: Eight different stemness gene signatures were identified from the published literature. Among these, four (Kim_myc, benporath_es2, Bhattacharya_hESC, and Shats_consensus) exhibited selective expression in a minority of tumor cells, with the Shats consensus signature demonstrating the highest specificity to tumor cells over non-malignant cells. Utilizing a Gaussian mixture model, we estimated that approximately 4.5% of cells within a tumor are likely stem cells. In the OPPORTUNE study, higher stemness scores were observed in luminal B compared to luminal A tumors, but there was no association with PIK3CA mutation status. Changes to Ki67 were inversely associated with the CSC fraction – tumors with higher stemness scores were less likely to achieve a complete cell cycle arrest versus endocrine-sensitive tumors. Similarly, in the FELINE trial, non-responders in the letrozole arm showed a trend towards elevated stemness scores. Conclusion: The use of stemness gene expression signatures in scRNAseq data is a feasible method to assess changes in putative CSC fraction with treatment in breast cancer. Increased stemness scores were associated with more aggressive subtypes and resistance to treatment.
 Citation Format: Peter Hall,Alejandro Chibly, Peter Schmid, Sarah E Pinder, Arnie Purushotham, Alastair Thompson, Steven Gendreau. Evaluation of breast cancer stem cell gene expression signatures in single-cell RNA sequencing (scRNAseq) data from the OPPORTUNE and FELINE trials, and the association with treatment resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-01.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287e61c3ae41000382799829a61a7b50a728f81b" target='_blank'>
              Abstract PS13-01: Evaluation of breast cancer stem cell gene expression signatures in single-cell RNA sequencing (scRNAseq) data from the OPPORTUNE and FELINE trials, and the association with treatment resistance
              </a>
            </td>
          <td>
            Peter Hall, A. Chibly, Peter Schmid, Sarah E. Pinder, A. Purushotham, Alastair Thompson, S. Gendreau
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3ab807859eab4d5f327f3a532083765a9ef4e89" target='_blank'>
              High-throughput single-cell CRISPRi screens stratify neurodevelopmental functions of schizophrenia-associated genes
              </a>
            </td>
          <td>
            Umut Yildiz, A. Claringbould, Mikael Marttinen, Víctor Campos-Fornés, Mantha Lamprousi, Manu Saraswat, Mathias Saver, Daria Bunina, Michael W. Dorrity, Judith B. Zaugg, K. Noh
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Gastric cancer remains a leading cause of cancer-related morbidity and mortality worldwide. The genetic factors contributing to gastric cancer progression and resistance to therapies, particularly radiotherapy, are not fully understood. TP53, ATM, RAD51, and BAX are genes involved in DNA repair, apoptosis, and response to stress. The aim of this study was to investigate the expression patterns of these genes in gastric cancer, their potential role in radiotherapy resistance, and their diagnostic value. Methodology Gene expression levels of TP53, ATM, RAD51, and BAX were assessed using RT-qPCR across 9 gastric cancer cell lines and 6 normal control cell lines. Additionally, protein expression was confirmed via IHC and TCGA dataset analysis. Methylation levels of these genes were evaluated in gastric cancer tissues using the GSCA database. Mutational analysis was conducted using cBioPortal, and survival analysis was performed using Kaplan-Meier and meta-analysis. The radiotherapy resistance study was carried out by knocking down TP53, RAD51, and BAX in AGS and MKN-45 gastric cancer cell lines, followed by expression analysis, colony formation, and wound healing assays. Results The expression of TP53, RAD51, and BAX was significantly upregulated, while ATM was downregulated in gastric cancer cell lines compared to normal controls. All four genes demonstrated good discriminatory power (AUC = 1) in distinguishing gastric cancer from normal samples. Methylation analysis revealed significant hypomethylation of TP53, RAD51, and BAX, and hypermethylation of ATM in gastric cancer tissues. Mutational analysis showed that TP53 was altered in 88% of gastric cancer samples, while ATM, RAD51, and BAX exhibited lower mutation rates. Survival analysis suggested that elevated expression of TP53, RAD51, and BAX may be linked to poorer survival outcomes, while reduced ATM expression appeared to associate with decreased overall survival. However, these associations require further validation through additional studies. Knockdown of TP53, RAD51, and BAX in AGS and MKN-45 cells resulted in significantly reduced cell proliferation and slower wound healing, highlighting their role in radiotherapy resistance. Conclusion The TP53, RAD51, and BAX genes are significantly involved in gastric cancer progression and resistance to radiotherapy. Their expression and mutation status provide valuable diagnostic and prognostic information. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00496-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8363e00ef3c38472bf63112f24092628b9ff0683" target='_blank'>
              In silico and in vitro assessment of TP53, ATM, RAD51, and BAX genes in gastric cancer and their contribution to radiotherapy resistance
              </a>
            </td>
          <td>
            Junwei Zhang, Pengtao He
          </td>
          <td>2025-07-12</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78884b796044723309cb646cb50e973d65185744" target='_blank'>
              A proteogenomic approach to identifying a gene signature associated with HOXB13 G84E carrier status in prostate cancer tumours
              </a>
            </td>
          <td>
            K. Raspin, Z. Noor, A. T. Aref, J. Marthick, C.-h. Jung, S. Donovan, S. Williams, P. Hains, P. J. Robinson, R. L. Balleine, Q. Zhong, R. Reddel, L. M. FitzGerald, J. L. Dickinson
          </td>
          <td>2025-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21690ca2e89ea2f16a09c71f18f6490b7208397c" target='_blank'>
              High Throughput Screening Identifies Small Molecules that Synergize with MRTX1133 Against Acquired Resistant KRAS G12D Mutated CRC
              </a>
            </td>
          <td>
            Natalie Thielen, Ning Wei, Emiko Nagai, Edward Chu, S. Kitamura, Chaoyuan Kuang
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Osteosarcoma is the most common primary malignant bone tumor, with high invasiveness and metastatic potential and a poor prognosis in patients with metastatic cancer. Despite the rapid advancements in genomics in recent years that provided new perspectives for studying the molecular mechanisms of osteosarcoma, the understanding of its tumor heterogeneity and evolutionary mutation process remains limited. Methods In this study, whole-exome evolutionary profiling was performed on data from the TARGET database representing 61 osteosarcoma cases. Subclonal architectures were reconstructed to characterize mutational trajectories. Differential mutation analysis was used to identify candidate metastasis-associated mutations. These features were used to build a metastasis-prediction classifier, which was cross-validated and tested on an independent external cohort. Finally, Suppes’ probabilistic theory of causality was integrated with cohort data to infer high-frequency evolutionary paths linked to metastasis. Results A linear evolutionary trajectory was observed in 62% of patients, indicating sequential clonal expansion. Eight key mutations were closely associated with metastatic progression. The classifier achieved 83% accuracy in cross-validation and maintained robust performance on the external validation set. Through causal inference, distinct evolutionary routes underpinning metastasis were uncovered, with ATRX mutations frequently occurring as early events that reshaped clonal dynamics and facilitated tumor spread. Conclusions In this study, the dynamic evolutionary landscape of osteosarcoma metastasis was delineated, an early metastasis classification model was constructed, and the impact of early clonal ATRX mutations on metastasis initiation were highlighted. These findings offer potential avenues for the early diagnosis and risk assessment of osteosarcoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06796-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68812275cd636690cb798f47d80f2494eb2d39fb" target='_blank'>
              Whole‑exome evolutionary profiling of osteosarcoma uncovers metastasis‑related driver mutations and generates an independently validated predictive classifier
              </a>
            </td>
          <td>
            Zhen Wang, Zhe Wang, Ruoyu Wang, Zumin Wang, Xiangyang Cao, Rui Chen, Zebing Ma, Shanshan Liang, Shuai Tao
          </td>
          <td>2025-07-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in shaping the response to immunotherapy in non-small cell lung cancer (NSCLC). While immune checkpoint inhibitors (ICIs) have revolutionized NSCLC treatment, a significant proportion of patients exhibit primary or acquired resistance. Emerging evidence highlights the role of specific genetic alterations, Kelch-like ECH-associated protein 1 (KEAP1), SWI/SNF-related, matrix-associated, actin-dependent regulator of chromatin subfamily A member 4 (SMARCA4), and phosphatase and tensin homolog (PTEN) mutations, in driving immune evasion and limiting the effectiveness of ICIs. KEAP1 mutations, through constitutive nuclear factor erythroid 2-related factor 2 activation, promote oxidative stress adaptation and metabolic reprogramming, creating an immune-excluded TME that reduces T-cell infiltration. SMARCA4 loss disrupts chromatin remodeling and epigenetic regulation, impairing antigen presentation and fostering a poorly immunogenic tumor phenotype. PTEN inactivation, by dysregulating the PI3K/AKT pathway, enhances tumor proliferation while contributing to an immunosuppressive cytokine milieu. The presence of these mutations is associated with poor outcomes, independent of tumor mutational burden or programmed death-ligand 1 expression, and their co-occurrence, particularly with KRAS and STK11, further compounds resistance. In this review, we explore the biological impact of KEAP1, SMARCA4, and PTEN mutations on the immune landscape of NSCLC, their implications for immunotherapy resistance, and potential strategies to overcome these barriers. As precision oncology advances, identifying therapeutic vulnerabilities in these genetically defined subgroups will be critical to improving patient outcomes and expanding the efficacy of immunotherapy in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e12ca155ee4398319cfc9ce8288f727fdb399ee" target='_blank'>
              Genetic drivers of tumor microenvironment and immunotherapy resistance in non-small cell lung cancer: the role of KEAP1, SMARCA4, and PTEN mutations.
              </a>
            </td>
          <td>
            Rodrigo Paredes, Roberto Borea, Francesco Drago, Alessandro Russo, Giovanni Nigita, Christian Rolfo
          </td>
          <td>2025-08-05</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction and Objective: Genetic variation and its interaction with environmental cues are crucial in the etiology of metabolic diseases, which are strongly linked to adipocyte dysfunction and are highly cell state- and context-specific.
 Methods: Here, we leverage a population-scale biobank (CellGenBank) to conduct pooled natural genetic variation screens in primary human adipocyte villages for single-nucleus transcriptomic and chromatin accessibility profiling under various disease-relevant stimuli.
 Results: Processing 338k nuclei from 118 donors allowed us to identified key cell states characterized by canonical marker genes, including quiescent (PDGFRA) and proliferative (PDGFRA, CDK1, AURKB) adipose tissue-derived mesenchymal stem cells (AMSCs), structural Wnt-regulated adipose tissue-resident (SWAT; DCN, PLAC9, APOD) cells and adipogenic (ADIPOQ, PLIN1) cells, with distinct transcriptional responses to stimuli. By leveraging individual polygenic risk scores, we identified correlations between disease risk and shifts in cell state proportions. These states have been also mapped to metabolic disease relevant traits using single-cell heritability analysis, with body mass index highly enriched in AMSCs, whereas traits informative of metabolic health show strong enrichment in adipogenic cells and a subpopulation of SWAT cells. Furthermore, genome-wide eQTL mapping revealed hundreds of context- and state-specific eQTLs that were enriched in predicted gene-enhancer/promoter regions, linking these variants to their functional transcripts and chromatin accessibility.
 Conclusion: Collectively, the human adipocyte village approach coupled with single-nucleus functional genomics enabled the discovery of genetic mechanisms underlying metabolic diseases, highlighting its high potential as a tool in the development of effective therapeutic targets. (*Yi Huang and Joaquin Perez-Schindler contribute to this work equally)


 Y. Huang: None. J. Perez-Schindler: None. S. Datta: None. B. Min: None. N. Nambrath: None. M. Murali: None. H. Dashti: None. B. Sharma: None. S. SinghPoma: None. P. Kubitz: None. W. Qiu: None. R. Andersson: None. T. Jones: None. M. Claussnitzer: None.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1a482f8b4cba32761087321e7d04e821730314" target='_blank'>
              2126-LB: Mapping Functional Genetic Variants and Disease Heritability on Gene Regulatory Programs in Human Adipocyte Villages under Metabolic Disease–Relevant Stimuli
              </a>
            </td>
          <td>
            YI Huang, Joaquín Pérez-Schindler, Sanchari Datta, Bellis Min, Nivedita Nambrath, Mayank Murali, Hesam Dashti, Bandana Sharma, Sunita Singhpoma, Phil Kubitz, Wei-Lin Qiu, Robin Andersson, Thouis Jones, Melina Claussnitzer
          </td>
          <td>2025-06-13</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As a predominant oncogenic driver in non-small cell lung cancer (NSCLC), EGFR frequently undergoes amplification or mutation, with EGFR-tyrosine kinase inhibitors (EGFR-TKIs) like gefitinib and erlotinib constituting frontline therapy for advanced EGFR-mutant cases. However, both primary and acquired resistance significantly limit clinical efficacy. Here, we revealed that glutathione metabolic pathway controlled by glutathione peroxidase GPX2 was abnormally activated in gefitinib-resistant A549 and HCC827-GR cell lines. Mechanistically, GPX2 triggers Hedgehog signaling activation through releasing GLI transcriptional regulator, promoting cancer stem cell (CSC) characteristics and TKI resistance. Notably, N6-methyladenosine (m6A) modification on GPX2 mRNA mediated by METTL14 diminished its stability. In vivo, GPX2 deletion constrained glutathione metabolism and boosted the effectiveness of TKI in cell line-derived xenograft models. Collectively, these findings demonstrate that GPX2 serves as a positive regulator of both primary and acquired EGFR-TKI resistance and could be a promising therapeutic target for precise treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50746db76f6a5070ba253127ed90d322b08f1eb2" target='_blank'>
              N6-methyladenosine-modified GPX2 impacts cancer cell stemness and TKI resistance through regulating of redox metabolism
              </a>
            </td>
          <td>
            Xu Yang, Long Yu, Miaomiao Shao, Huiling Yang, Kangwei Qi, Gaofei He, Lanxin Wang, Di Kong, Jianxin Gu, Xiaolin Xu, Lanxin Wang
          </td>
          <td>2025-06-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a poor prognosis that is driven primarily by oncogenic KRAS mutations present in > 90% of cases. KRAS mutations, particularly the G12D mutation which dominates in PDAC, fuel tumor initiation, progression, and immune evasion, thereby contributing to therapy resistance. Nevertheless, KRAS has long been considered “undruggable” due to its structure. Recent advances have spurred transformative progress in direct KRAS inhibition. While FDA-approved mutation-specific and pan-KRAS inhibitors show limited efficacy in PDAC, emerging agents (MRTX1133 and RMC-9805) have demonstrated preclinical promise. However, resistance remains a critical hurdle and is driven by pathway reactivation, secondary mutations, and metabolic adaptations. Alternative strategies targeting upstream regulators (SHP2 and SOS1) aim to block KRAS activation and associated resistance mechanisms. Preclinical studies have also highlighted synergistic benefits of combining KRAS inhibitors with MEK, PI3K, or CDK4/6 inhibitors, which are now undergoing clinical evaluation. Immunotherapies, including KRAS-targeted vaccines and adoptive T-cell therapies, have further expanded the therapeutic landscape of enhancing KRAS-targeted therapies in PDAC. The molecular basis of KRAS-driven PDAC, current inhibitors, resistance mechanisms, and innovative strategies are discussed herein to address treatment barriers. Opportunities to improve clinical outcomes are underscored in this challenging malignancy by integrating insights from preclinical and clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8cfbe6080878693ce57a6e261fa2c4f9ddf499" target='_blank'>
              Drugging the ‘undruggable’ KRAS: breakthroughs, challenges, and opportunities in pancreatic cancer
              </a>
            </td>
          <td>
            Nawaz Khan, Umar Raza, Syed Aqib Ali Zaidi, Muhadaisi Nuer, Kayisaier Abudurousuli, Yipaerguli Paerhati, Alifeiye Aikebaier, Wenting Zhou
          </td>
          <td>2025-07-07</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recent large-scale multi-omics studies have characterized the heterogeneity of esophageal squamous cell carcinoma (ESCC), but inconsistent clinical management has hindered the identification of prognostic markers and patient stratification. Here, we conducted genomic and transcriptomic profiling of 203 patients from the ECTOP-2002 study with full clinical information. Mutation in the mucin family, as well as APOBEC signature, were associated with poor prognosis. In contrast, activation of the epithelial-keratinization (EpK) pathway was strongly linked to favorable prognosis and lower post-chemotherapy recurrence rates. Independent validation supported S100A8 + S100A9 complex as a key marker of EpK pathway. Furthermore, we established a prognostic stratification system, FU-ESCC subtyping, which defines three subtypes with distinct molecular and clinical features. The EpK-activated subtype retained characteristics of healthy squamous epithelial cells, showed high expression of the S100A8 + S100A9 complex, and was associated with favorable prognosis. The cancer-associated fibroblast (CAF)-enriched subtype showed elevated FAP and Vimentin expression, abundant CAFs, high proliferative activity, and poor prognosis. The immune-desert subtype was characterized by low immune infiltration, suppressed immune signaling, and similarly poor prognosis. Our study provides a valuable resource and insights to better understand ESCC in the era of precision medicine and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76484e9cc049fb0a04587ff7d1198a866a229ed2" target='_blank'>
              Comprehensive genomic and transcriptomic analyses reveal prognostic stratification for esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jian Gao, Qiming Wang, F. Fu, Yue Zhao, Teng Yang, Xiangze Li, Yihua Sun, Hong Hu, Long-fei Ma, L. Miao, Xiao-yang Luo, Ting Ye, Yiliang Zhang, Yang Zhang, Ziling Huang, Hang Li, L. Shao, Midie Xu, Kuaile Zhao, Shiyue Zhang, Mou Zhang, Jun Wang, Chong Dai, Xiaoxiao Shang, Tingyi An, Yawei Zhang, Jiaqing Xiang, Zhiwei Cao, Bin Li, Haiquan Chen
          </td>
          <td>2025-07-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Germline pathogenic variants (GPVs, 'mutations') causing inherited susceptibility to certain cancers (cancer susceptibility genes, CSGs) broadly belong to one of two main classes-loss of function variants in tumour suppressor genes (TSGs) or gain of function variants in proto-oncogenes (an over-simplification). Genomic analyses of tumours identify 'driver mutations' promoting tumour growth and somatic variants which contribute to 'mutation signatures' which, with histopathology, can be used to subclassify cancers with implications for causality and treatment. The identification of susceptible individuals is important, as they and their relatives may be at elevated risk of tumours, and this can influence optimal cancer treatment. Classically, cancer risk assessment utilises family history, lifestyle/environment factors, and any non-neoplastic clinical findings, followed by genetic testing of high/moderate penetrance CSGs. In cancer cases not caused by highly penetrant CSGs, multiple variants conferring relatively small risks play a major role. These were discovered by genome-wide association (GWAS) studies. The utility of polygenic risk scores (PRS) derived from multiple such variants for clinical risk profiling is being assessed. Access to genetic tests is improved by widening eligibility criteria for testing and empowering non-genetic clinicians to identify CSG GPVs and manage carriers. This will contribute to expanding programmes of screening, prevention and early detection (SPED), with personalised surveillance and prophylactic interventions, and exploit knowledge of the molecular mechanisms of cancer susceptibility to develop novel cancer therapies. In some jurisdictions, population testing is being considered, but GPV penetrance in this setting can be unclear, and the public health implications are complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee21b45930d11565d54a3022b9abc97a84ec0804" target='_blank'>
              Inherited Susceptibility to Cancer: Past, Present and Future.
              </a>
            </td>
          <td>
            Shirley V Hodgson, W. Foulkes, E. Maher, C. Turnbull
          </td>
          <td>2025-07-21</td>
          <td>Annals of human genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The circadian clock governs daily rhythms in numerous physiological processes through precise regulation of gene expression and biochemical functions. Dysregulation of the circadian rhythm has been implicated in carcinogenesis and cancer progression. However, the mechanisms by which the circadian clock influences cancer phenotype and chemotherapy resistance, particularly in cholangiocarcinoma (CCA), remain poorly understood. Using cell lines established from primary CCA and metastatic ascites of two male patients, we manipulated core clock genes (BMAL1, PER2, and NR1D1) to evaluate their effects on circadian rhythms. We analyzed alterations in circadian phenotypes at dynamic and single time points and assessed their impact on cancer-related phenotypic changes, including proliferation, apoptosis, cell cycle regulation, migration, invasion, and the expression of epithelial-to-mesenchymal transition (EMT) and cancer stem cell markers. Additionally, we examined the impact of circadian disruption on gemcitabine sensitivity. Genetic deletion of BMAL1, PER2, and NR1D1 disrupted circadian rhythm and significantly altered cancer phenotypes. Notably, BMAL1 and NR1D1 impairment exacerbated cell migration, invasion, and EMT activation in CCA cells. BMAL1 loss also induced gemcitabine resistance. In contrast, PER2 repression enhanced chemosensitivity and inhibited metastasis. The modulation of the circadian gene triggered phenotypic changes in CCA cells, indicating a crucial involvement of core-clock components in the pathological mechanisms hastening bile duct cancer malignancy. Our findings advance the understanding of regulating CCA malignancy and may offer a novel target for its treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63df1b948313a08b9d2b74bdfc742ccab84a19a9" target='_blank'>
              Dysfunction of core clock genes regulates malignant phenotype and gemcitabine sensitivity of cholangiocarcinoma cells.
              </a>
            </td>
          <td>
            Yin Li, Aimin Zheng, Yangang Cui, Tianbao Liu
          </td>
          <td>2025-06-17</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading cause of cancer-related mortality among men. Radiotherapy is the cornerstone of PCa treatment. However, a major limitation of radiotherapy is the development of resistance, which compromises treatment efficacy. Reactive oxygen species (ROS), which are generated by radiation, have a dual role in PCa by inducing DNA damage and apoptosis, while also promoting tumor progression and radioresistance. Elevated ROS levels enhance metabolic reprogramming, activate oncogenic pathways, and influence the tumor microenvironment by modulating immune responses and promoting the epithelial-mesenchymal transition (EMT). Key molecular mechanisms, including the Nrf2/Keap1 signaling axis, Bcl-2 mutations, and Speckle-type POZ protein alterations, contribute to radioresistance by enhancing antioxidant defenses and DNA repair capacity. Additionally, the interplay between hypoxia, androgen receptor variants (AR-Vs), and ferroptosis regulators further influence radiotherapy outcomes. Understanding these resistance mechanisms is crucial for developing targeted strategies to enhance radiosensitivity and improve therapeutic outcomes in PCa patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f9f60cf373e93dea28bab21b0e3ffe7de720751" target='_blank'>
              Mechanisms underlying prostate cancer sensitivity to reactive oxygen species: overcoming radiotherapy resistance and recent clinical advances
              </a>
            </td>
          <td>
            Meidan Wang, Rui Xing, Liqun Wang, Mingyue Pan, Ruoyun Zhang, Ting Li, Weiqiang Sun, Jing Zhou
          </td>
          <td>2025-07-10</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The PI3K/AKT/mTOR signaling pathway interferes with cell proliferation, survival, metabolism, and angiogenesis.
Overactivation of this signal transduction is one of the hallmarks of several human cancers, usually traceable to
genetic alterations like PIK3CA mutation, PTEN loss, or AKT amplification. Inhibition of the pathway therefore
opens attractive therapeutic windows currently applied or being tested in clinical trials with drugs ranging from PI3K
isoform-specific inhibitors to dual PI3K/mTOR inhibitors. Drug resistance mechanisms, feedback activation of
parallel pathways, and toxicity are some roadblocks limiting their long-term efficacy. However, these can
increasingly be addressed by the advent of combination therapies, biomarker-based patient selection, and new drug
delivery systems. This review dwells on a comprehensive assessment of the biological significance of the
PI3K/AKT/mTOR pathway in cancer, therapeutics in development, mechanisms mediating resistance, and emerging
concepts that may help improve and broaden the clinical interventions in target therapy for oncology.
Keywords: PI3K/AKT/mTOR pathway, cancer, targeted therapy, PI3K inhibitors, AKT inhibitors, mTOR inhibitors,
resistance mechanisms, combination therapy, biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c6ab340e148ad718baae46a411103aa0d5d9cfe" target='_blank'>
              Molecular & Targeted therapy: THE PI3K/AKT/MTOR Pathway in cancer: New avenues for targeted therapy
              </a>
            </td>
          <td>
            Matsa Ganga Bhavani, Karri Naga Pravallika, Nukala Mounika Sivani, Bollarapu Syam Deven Babu, Pemmada Sowmya, Mekala Poojitha, Agarapu Lahari, M.S.R Bapi Raju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Current Research in Medical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) displays remarkable cell state plasticity, a major contributor to therapeutic resistance and tumor progression. While epigenetic mechanisms play a central role in driving this plasticity, the key regulators remain poorly understood, and developing effective therapeutic strategies targeting them has been challenging.


METHODS
We investigated the role of BRD2, a key regulator of NF-κB mediated mesenchymal (MES) transition, using GBM patient-derived xenograft (PDX) cell lines, CRISPR-mediated knock-in/knockout approaches, RNA-seq, and in vitro and in vivo modeling. BET inhibitors were employed to target MES gene expression and sensitize GBM to radiation therapy.


RESULTS
We found that PTEN loss induces RelA chromatin localization and acetylation-mediated recruitment of BRD2 to the MES gene promoters. BRD2 binding is essential for maintaining MES gene expression and phenotype. Genetic ablation or loss-of-function mutation of BRD2 bromodomains reverses MES transition, enhances radiation sensitivity, and improves survival in orthotopic xenograft models. Additionally, treatment with a brain-penetrant BD2-selective inhibitor suppresses the MES phenotype and increases radiation sensitivity of GBM stem cells in vitro.


CONCLUSION
Our study identifies BRD2 as a key mediator of MES transition in GBM, with its bromodomains playing a crucial role in driving cell state plasticity. Targeting BRD2 with BD2-selective inhibitors offers a promising therapeutic strategy to overcome radiation resistance and improve outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa778fbdf5d6872b2c63bfc97f4589c5f7dfb10" target='_blank'>
              BRD2 Bromodomain-Mediated Regulation of Cell State Plasticity Modulates Therapy Response in Glioblastoma.
              </a>
            </td>
          <td>
            Raghavendra Vadla, Brett Taylor, Yohei Miyake, Benjamin Lin, Daisuke Kawauchi, Shunichiro Miki, Nidhi Nathwani, Brandon M Jones, Yashpreet Kaur, Abhinaba Banerjee, Philip Pham, J. Tsang, Albert Baldwin, David A Nathanson, Donald P. Pizzo, C. Miller, Frank B Furnari
          </td>
          <td>2025-07-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous disease, with castration-resistant prostate cancer (CRPC) and neuroendocrine prostate cancer (NEPC) representing its most aggressive and therapy-resistant forms. Emerging evidence indicates that lineage plasticity—driven by key transcription factors such as Octamer Binding Factor 4 (OCT4)—plays a crucial role in therapeutic resistance and disease progression. OCT4, in coordination with SOX2 and NANOG, acts as a master regulator of stemness and is frequently upregulated in prostate cancer stem cells (PCSCs). This upregulation contributes to tumor initiation, metastasis, and resistance to both androgen deprivation therapy (ADT) and chemotherapy. In this review, we explore the role of OCT4 in mediating lineage plasticity in prostate cancer, with particular emphasis on its involvement in treatment resistance and neuroendocrine differentiation. We also examine therapeutic strategies aimed at targeting OCT4 directly, such as microRNA-mediated suppression, small-molecule inhibitors, and suicide gene therapy, as well as indirect approaches that modulate OCT4 expression via FGFR and NF-κB signaling pathways. While these strategies offer promising avenues, challenges such as adaptive resistance and the intricate signaling networks within PCSCs remain significant hurdles. A deeper understanding of the molecular mechanisms underlying OCT4-driven plasticity may pave the way for novel therapeutic approaches and improved outcomes in advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8ce283789b3c96575337b7647df8d11de9b2ba" target='_blank'>
              Unlocking the Role of OCT4 in Cancer Lineage Plasticity: A Cross-Cancer Perspective with an Emphasis on Prostate Cancer
              </a>
            </td>
          <td>
            Mohammad Esfini Farahani, Yanquan Zhang, A. O. Akinyemi, Fatemeh Seilani, Md Rakibul Alam, Xiaoqi Liu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Objective DDX17, an ATP-dependent RNA/DNA helicase, is implicated in the regulation of RNA metabolism and has been linked to tumorigenesis and metastasis in various cancers. While studies have explored the role of DDX17 in specific cancers, further research is needed to understand its mechanisms across different cancer types. Methods We leveraged several public databases, including TIMER, The Cancer Genome Atlas (TCGA), and the Genotype-Tissue Expression (GTEx), to investigate DDX17 mRNA expression across 33 types of tumors. The GEPIA2 database was utilized to assess the impact of DDX17 on overall survival (OS) and disease-free survival (DFS) in patients with these tumors. Additionally, we employed cBioPortal to examine DDX17 gene alterations in various tumor tissues. Further analysis was conducted using the R language to explore the correlation between DDX17 and a range of clinical features, including tumor microenvironment (TME), immune regulatory genes, immune checkpoints, tumor mutational burden (TMB), microsatellite instability (MSI), DNA methylation, RNA methylation, and drug sensitivity. Gene Set Enrichment Analysis (GSEA) was also applied to elucidate the molecular mechanisms mediated by DDX17. Results DDX17 showed significant differences in expression between cancer and normal tissues. Expression of DDX17 was associated with patient prognosis, TMB, MSI, and drug sensitivity in certain cancers. DDX17 is additionally involved in modulating immune functions and influencing the tumor microenvironment. Further analysis of DDX17 mutation sites and types showed that the mutation frequency was highest in endometrial cancer and the major mutations of DDX17 were missense mutations. Conclusion These findings indicated that DDX17 may be considered a potential prognostic biomarker and a promising target for novel immunotherapeutic approaches in cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02955-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c500003015dd54739438713ee9cd7419ce63da8" target='_blank'>
              Prognostic and immunological landscape of DDX17 in pan-cancer analysis: a comprehensive study
              </a>
            </td>
          <td>
            Yu Lei, Xin He, Peng Chen, Yiping Qin, Bin Ge, Yan Liu, Pu Li, Xing Wei
          </td>
          <td>2025-06-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: Triple-negative breast cancer (TNBC) is clinically significant for its high invasiveness (metastasis) and poor prognosis. The molecular classification of breast cancers and characterization of TNBC have been aided by the analysis of both bulk and single-cell RNA-sequencing data. Specifically, they have facilitated the study of the roles of various regulator and effector genes associated with pro-metastatic behaviors such as invasion, and drug resistance-related epithelial-to-mesenchymal transition (EMTI). Yet, it remains unknown how informative and consistent bulk and scRNA-seq TNBC data are in regards to co-expressions of EMTI genes.Methods: Bulk TNBC RNA-seq data was retrieved from the TCGA-BRCA cohort of the TCGA-TARGET-GTEx Recompute Project, cleaned, and normalized. Single-nucleus TNBC tumor cell data curated by Pal et al. (2021) was separated into six datasets based on patient(s) of origin/cell cycling status and preprocessed. A list of 363 EMTI genes from the EMTome was defined for all gene-gene correlation analyses. Significantly correlated gene pairs with co-expressions concordant among TNBC dataset pairs and larger groupings were found. To identify possibly TNBC-specific correlations, co-expression analysis was repeated for both bulk normal breast from the GTEx cohort of the Recompute project and snRNA-seq normal breast epithelial data from Pal et al. (2021). EMTI regulatory networks were reconstructed using co-expression data between 26 key EMT genes for both TNBC and normal. Co-expression comparisons and differential co-expression analysis were performed between the TNBC and normal samples. Differential expression analysis was also performed between the bulk TNBC and normal to investigate the relationship between differential EMTI gene expression and co-expression. Results: EMTI gene co-expression patterns exhibited greater similarity between TNBC single-cell datasets than between any single-cell and bulk TNBC. Moreover, the datasets most concordant with each other were single-cell datasets originating from the same patient(s), indicating that EMTI gene co-expressions do not significantly differ with cell cycling status. The SPINT1 and LSR genes were significantly positively correlated in all TNBC samples but not in normal, revealing a previously undocumented link. Additionally, we found 11 TNBC single-cell-specific gene-gene correlations (7 positive, 4 negative), relationships that would not have been identified in heterogeneous bulk TNBC. We observed known interactions between EMTI genes, including an EpCAM-keratin-claudin link well preserved across both TNBC and normal breast. Out of the three breast epithelial lineages (luminal progenitor (LP), mature luminal (ML), basal), EMTI gene co-expressions in TNBC single-cell were most similar to those in LP; however, those in the bulk TNBC were most similar to basal. With the exception of 2 gene pairs common to the bulk datasets, there were no EMTI gene pairs with consistent co-expression patterns across the different TNBC/normal groupings. Interestingly, differential EMTI gene co-expression was weakly associated with differential gene expression; this applied for genes non-differentially expressed (KRT5), upregulated in TNBC (EPCAM), or downregulated (TGFB1I1) (Spearman’s ρ: 0.139, 0.145, 0.0839; all p < 10e-07). Conclusions: Our integrated analysis reveals that EMTI gene pairs are moderately concordant among TNBC or normal datasets, but not between the two. Moreover, we show both known and new interactions in reconstructed networks. We identify a TNBC single-cell-specific 11 gene pair signature that may be promising for further study. Finally, although contrary to previous work we find that differential EMTI gene co-expression can be partially explained by differential gene expression, the association is weak enough where the two remain complementary approaches.
 Citation Format: Melody Hong, Gábor Balázsi. An Integrated Analysis of Metastatic Gene Co-expression in Triple-Negative Breast Cancer via Bulk and Single-Cell Transcriptome Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c45f5648505bec985b6f5272b9daf7404e8c62" target='_blank'>
              Abstract P2-06-17: An Integrated Analysis of Metastatic Gene Co-expression in Triple-Negative Breast Cancer via Bulk and Single-Cell Transcriptome Data
              </a>
            </td>
          <td>
            Melody Hong, Gábor Balázsi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) remains one of the most aggressive and therapeutically challenging subtypes of BC due to its lack of targeted treatment options and high plasticity. Increasing evidence suggests that epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNA regulation, play a pivotal role in facilitating TNBC phenotype by influencing the expression of genes involved in subtype reprogramming. Modulating these epigenetic marks offers a promising strategy to drive TNBC subtype conversion toward less aggressive forms that are more responsive to treatment, thereby enhancing therapeutic efficacy. By targeting enzymes such as DNA methyltransferases and histone modifiers using epigenetic drugs (epidrugs), TNBC cells can be re-sensitized to hormone therapy, chemotherapy, and targeted treatments. This review delves into the role of epigenetic modulation in harnessing the plasticity of TNBC to drive its conversion into subtypes with better prognoses and discusses problems and limitations associated with the use of epidrugs in this context. By promoting subtype conversion, epidrugs offer a promising strategy for providing more personalized and effective therapies for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ada35a90892ec0be34375adfcf309b46149ff70" target='_blank'>
              Epigenetic Therapeutics: Reprogramming Triple-Negative Breast Cancer into Responsive Subtypes.
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, A. Al-Azawi, L. Lozon, Rawan R. Kawaf, Ragheb Alsheikh Zein, Yahia El-Gharib, Raafat El-Awady
          </td>
          <td>2025-07-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) continues to be a major contributor to global cancer-related mortality, with KRAS and NRAS oncogenic mutations playing a pivotal role in tumorigenesis and therapeutic resistance. These genetic alterations drive uncontrolled proliferation and modulate responses to targeted treatments, significantly influencing clinical outcomes. This review examines the prevalence and prognostic implications of KRAS and NRAS mutations, with particular attention paid to the Venezuelan population. Reported frequencies in Venezuela (13–23% for KRAS and 1.6–2.6% for NRAS) are markedly lower than global averages (36–45% for KRAS), a discrepancy potentially attributable to population-specific genetic variations or methodological constraints of conventional sequencing techniques, which may lack sensitivity for variant detection. The presence of RAS mutations is a well-established biomarker of resistance to anti-EGFR monoclonal antibodies, necessitating alternative treatment regimens for affected patients. Although novel agents such as KRAS G12C inhibitors exhibit therapeutic potential, their efficacy remains under investigation in clinical trials, highlighting the need for regionally tailored drug development and improved access to innovative therapies in Venezuela. Beyond proliferative signaling, KRAS-mediated oncogenesis involves metabolic adaptation, angiogenic activation, and immune system evasion, underscoring its multifaceted role in CRC progression. Advanced molecular diagnostic techniques, such as next-generation sequencing (NGS) and circulating tumor DNA (ctDNA) analysis, have significantly improved the detection of oncogenic mutations, offering higher sensitivity and specificity compared to traditional methods. However, their widespread adoption in resource-limited settings remains hindered by infrastructural, economic, and technical barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191bb8159cd664cc4ac3633314f93ce9a0d86dba" target='_blank'>
              RAS oncogenes in Venezuelan colorectal cancer: detection methods and mutational profile review
              </a>
            </td>
          <td>
            Carlos Darío Ramírez, Daniela Rodríguez-Carrascal, Rafael Puche
          </td>
          <td>2025-06-25</td>
          <td>Academia Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="In precision medicine, DNA-based assays are currently necessary but not always sufficient for predicting therapeutic efficacy of cancer drugs based on the mutational findings in a patient’s tumor specimen. Most drugs target proteins, but it is challenging and not yet cost-effective to perform high-throughput proteomics profiling, including mutational analysis, on cancer specimens. RNA may be an effective mediator for bridging the “DNA to protein divide” and provide more clarity and therapeutic predictability for precision oncology. While RNA sequencing (RNA-seq) has been increasingly used alongside DNA cancer mutation screening panels to assess the impact of variants on gene transcript expression and splicing, comprehensive evaluations of RNA panels and the integration of expressed mutation data analytics to supplement DNA panels are still limited. In this study, we conducted targeted RNA-seq on a reference sample set for expressed variant detection to explore its potential capability to complement DNA variant results or detect variants independently. The results indicated that, with a carefully controlled false positive rate ensuring high accuracy, RNA-seq uniquely identified variants with significant pathological relevance that were missed by DNA-seq, demonstrating its potential to uncover clinically actionable mutations. On the other hand, while some variants were detected by both approaches, others were missed by one or the other, reflecting either the nature of these variants or limitations of the bioinformatics tools used. Variants missed by RNA-seq are often not expressed or expressed at very low levels, suggesting they may be of lower clinical relevance. Incorporating RNA-seq into clinical biomarker panels will ultimately advance precision medicine and improve patient outcomes by improving the strength and reliability of somatic mutation findings for clinical diagnosis, prognosis and prediction of therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f90149304c1d7c32369d59f225270973f7ce13" target='_blank'>
              Augmenting precision medicine via targeted RNA-Seq detection of expressed mutations
              </a>
            </td>
          <td>
            Dan Li, Jianying Li, Donald J. Johann, Daniel Butler, Guangchun Chen, J. Foox, Binsheng Gong, W. Jones, David P. Kreil, Rebecca Kusko, Paweł P. Łabaj, A. Lucas, Christopher E. Mason, C. Mozsary, N. Novoradovskaya, C. Pabón-Peña, Bohu Pan, Todd A. Richmond, Roberta Maestro, Sayed Mohammad Ebrahim Sahraeian, Andreas Scherer, Hagen U. Tilgner, James C. Willey, P. Bushel, Joshua Xu
          </td>
          <td>2025-06-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Oncogenic retroviruses are known for their pathogenesis via insertional mutagenesis, in which the presence of a provirus and its transcriptional control elements alter the expression of a nearby or surrounding host gene. There are reports of proviral integration driving oncogenesis in people with HIV and the use of HIV-derived vectors for gene therapy has raised concern about oncogenic side effects. To study this issue, we used an in vitro primary human CD4 + T cell infection model developed in our laboratory to identify HIV-1 integration sites that might influence cell proliferation or survival. Combining integration site analysis and bulk RNA sequencing, we found that an upregulated STAT3 signature due to proviral insertional mutagenesis was associated with persistent HIV-infected CD4 + T cells. HIV+ persistent cells also expressed a STAT3-related anti-apoptotic and cytotoxic phenotype that resembles that of HIV-associated T cell lymphomas. HIV insertional mutagenesis of STAT3 and expression of its downstream targets provides a model of HIV-associated T cell lymphomas that can be used to further determine the oncogenic drivers of HIV-associated lymphomas, both AIDS- and gene therapy-associated, and, potentially, to evaluate therapeutics against these HIV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309400b908a0604bb16da7990788bd142c72eac7" target='_blank'>
              In vitro HIV DNA integration in STAT3 drives T cell persistence—A model of HIV-associated T cell lymphoma
              </a>
            </td>
          <td>
            M. Rist, M. Kaku, John M. Coffin
          </td>
          <td>2025-07-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic changes, such as DNA methylation, chromatin remodeling, and histone modifications, regulate gene expression without altering the DNA sequence. This review systematically analyzed over 500 studies including human cell line experiments (n>200), animal models (n>50), clinical cohort studies (n>100), and bioinformatics analyses retrieved from PubMed, Web of Science, and TCGA (The Cancer Genome Atlas). Studies increasingly show that genes involved in glucose and lipid metabolism, energy production, and modulation of metabolic hormones are regulated through epigenetic mechanisms. On the other hand, various metabolites participate in epigenetic modifications as coenzymes or substrates. Therefore, a greater understanding of the crosstalk between metabolism and epigenetics in cancer-related pathways could lead to the identification of key signaling molecules for targeted therapies, and raise the possibility of using dietary interventions to modulate epigenetic markers for individualized treatment. In this review, we have summarized the metabolic and epigenetic regulatory networks in cancer development, including glycolipid metabolic reprograming, the role of metabolites produced by the glut flora and tumor microenvironment, and key epigenetic drivers such as non-coding RNAs (ncRNAs). Data were curated from peer-reviewed articles, grounded in mechanistic studies using cell lines (SW480, MCF7 (Michigan cancer foundation-7)) and animal models (APC-mutant mice), with a focus on mechanistic studies, omics analyses, and translational research. Furthermore, we have discussed the potential of therapeutically targeting these pathways, along with the current challenges and future research directions, and a new strategy for reversing therapeutic drug resistance based on metabolism and epigenetic interaction was systematically explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447a75110b1468dd9ba2d78ee2627a7a828fee6" target='_blank'>
              Metabolism and epigenetics in cancer: toward personalized treatment
              </a>
            </td>
          <td>
            Xiaoman Zhang, Dequan Liu, Sulan Yin, Yaru Gao, Xiaorui Li, Guangzhen Wu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes.
              </a>
            </td>
          <td>
            Steven A Frank
          </td>
          <td>2025-08-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose: Although the combination of androgen deprivation therapy with docetaxel or abiraterone acetate and prednisone has become a standard treatment for metastatic hormone-sensitive prostate cancer (mHSPC) and has shown improved overall survival, a subset of patients still progress to castration-resistant disease. However, the underlying molecular features in these patients remain poorly understood.Materials and Methods: We performed single-cell RNA sequencing (scRNA-seq) on 12 tissue samples, including 2 mHSPC samples, 7 primary prostate cancer (PCa) samples, and 3 matched normal samples from 7 patients. Raw sequencing data was processed by Cell Ranger software (10X Genomics) and aligned to the human reference genome (GRCh38). A comprehensive analysis of samples from patients with mHSPC was also conducted and validated using a cohort of 52 patients with mHSPC.Results: Our results identified distinct subpopulations within luminal and mononuclear phagocyte (MNP) clusters characterized by proliferative activation associated with unfavorable clinical outcomes. Furthermore, we observed that the MNP cluster exhibited significant proliferation activity. To understand the underlying mechanisms associated with aggressiveness, we conducted cell-cell interaction, copy number variation and pseudotime analysis. Utilizing 87 network genes from scRNA-seq, we classified 52 mHSPC patients into 2 molecular subtypes and demonstrated their correlation with distinct transcriptomic profiles and survival outcomes. Importantly, a 14-gene signature derived from 3 distinct gene sets exhibited a strong association with patient survival and drug response. The prognostic value of our findings was further validated in large-scale cohorts comprising localized PCa, metastatic PCa, mHSPC, and metastatic castration-resistant PCa patients.Conclusion: This study provides valuable insights into the identification of high-risk patients, novel biomarkers, and potential therapeutic targets for individuals with mHSPC. Furthermore, the results in this study can serve as a basis for future investigations aimed at refining prognostic strategies and developing targeted therapies for patients with mHSPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450db38b74f006113af6f667e3cedb5cf553cec7" target='_blank'>
              Determinant of Aggressive Phenotype in Metastatic Hormone-Sensitive Prostate Cancer Depends on an Intrinsic, Highly Aggressive Cell Cluster: Integrated Single-Cell RNA and Whole Transcriptomic Sequencing Analyses
              </a>
            </td>
          <td>
            Minyong Kang, B. Jee, Jiwoong Yu, Soohyun Hwang, Kyunghee Park, Kyung Yeon Han, Wan Song, H. Sung, H. Jeon, Byong Change Jeong, S. Seo, Se Hoon Park, Woong-Yang Park, S. Jeon
          </td>
          <td>2025-07-31</td>
          <td>Journal of Urologic Oncology</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Lenvatinib, a multiple-receptor tyrosine kinase inhibitor, has gained recent approval for its use as a first-line treatment of hepatocellular carcinoma (HCC). While lenvatinib demonstrates notable therapeutic efficacy, the drug resistance undermines its sustained tumor control potential. The restricted clinical utility of lenvatinib underscores the imperative necessity to elucidate the mechanisms underpinning drug resistance. We established lenvatinib-resistant cell lines and investigated the changes in their biological characteristics. Next-generation sequencing was performed to identify genes associated with lenvatinib resistance. Western blots were utilized to confirm the involvement of these genes. Using lentiviral technology, we generated cell lines with lowered nuclear receptor coactivator 5 (NCOA5), a pivotal drug resistance-related gene, to explore the underlying resistance mechanism. Moreover, we developed a subcutaneous HCC xenograft tumor model to explore strategies for reversing drug resistance. Our study showed that HCC cells acquire resistance to lenvatinib through the activation of NCOA5, thereby stimulating the NCOA5-Protein Kinase B-mammalian target of rapamycin (AKT-mTOR) axis. Furthermore, the clinical evaluation of HCC specimens established a correlation between the activation of the NCOA5 pathway and the response to lenvatinib treatment. Everolimus, an mTOR inhibitor, in combination with lenvatinib and everolimus, exerted significant synergistic effects against HCC in vivo and in vitro. HCC cells develop resistance to lenvatinib by activating the NCOA5-AKT-mTOR pathway. The combination therapy of lenvatinib with everolimus is a promising strategy to overcome acquired resistance, thereby enhancing the clinical efficacy of lenvatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4645ca240d89c81caab05ab94ee824d557e8c782" target='_blank'>
              Inhibition of nuclear receptor coactivator 5 overcomes acquired lenvatinib resistance driven by protein kinase B-mammalian target of rapamycin signaling in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Hongyuan Zhou, Qin Zhang, Lu Yang, Zhao-Long Pan, Haijing Zheng, Zewu Zhang, Dongyang Li, Guangtao Li, Xiaomeng Liu, Xu Bao, Chen Liu, Wei Zhang
          </td>
          <td>2025-07-31</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Small Ubiquitin-like Modifier 1 Pseudogene 3 (SUMO1P3) is a novel long non-coding RNA (lncRNA) located at the 1q23.2 locus of the human chromosome. Recent evidence indicates that SUMO1P3 is aberrantly upregulated in nine types of human cancer and functions as an oncogene. Elevated SUMO1P3 expression is strongly associated with unfavorable clinicopathological features and poor prognosis in eight cancer types. Mechanistically, SUMO1P3 functions as a miRNA sponge, an epigenetic regulator, and directly interacting with proteins. It activates key signaling pathways, such as the Wnt/β-catenin and AKT pathways, and regulates Epithelial- Mesenchymal Transition (EMT), which facilitates cancer progression and therapy resistance. Due to its diverse functional roles, SUMO1P3 emerges as a promising diagnostic and prognostic biomarker, as well as a potential therapeutic target in precision oncology. This review provides a comprehensive summary of current research on SUMO1P3, highlighting its regulatory mechanisms, biological functions, and clinical significance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4f3e3fbd6a4fd472988ea5e4c6612a35b0592f" target='_blank'>
              The Vital Role of Long Non-Coding RNA SUMO1P3 in the Regulation of Human Cancer: Current Perspectives and Future Challenges.
              </a>
            </td>
          <td>
            Jingjie Yang, Yuzhang Wei, Chengran Gao, Zihang Wang, Yulong Liu, Haodong He, Hao Zhou, Guihua Liao, Gang Zhou, Chengfu Yuan
          </td>
          <td>2025-07-29</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Cancer development and progression are driven by the accumulation of somatic genetic alterations, which occur in a specific temporal order. However, how the order of mutations impacts cancer phenotypes of solid tumors remains poorly understood. To address this, we developed a novel computational framework, IMOP-Cancer (Identifying Mutation Order Pairs in Cancer), to identify mutation gene pairs whose order influences cancer phenotypes. We applied IMOP-Cancer to The Cancer Genome Atlas-Lung Adenocarcinoma (TCGA-LUAD) cohort and identified 446 key mutation order pairs, with 34 pairs significantly associated with prognosis. Mutation order impacts cancer phenotypes, as demonstrated by CSMD3 and PTPRD (tumor proliferation) and TP53 and NAV3 (immune modulation), with effects validated in four independent datasets. We further presented the impact of mutation pairs on cancer phenotypes through case studies in the TCGA cohorts of bladder urothelial carcinoma (BLCA), and colon adenocarcinoma (COAD). We extended this analysis to 33 cancer cohorts from TCGA portal, identifying 106 034 critical mutation pairs across 17 cancers, with 3036 pairs co-occurring in multiple cancers. Shared mutation pairs across cancers also showed distinct effects on cancer phenotype. Our study highlights the importance of mutation order in cancer progression and diversity, offering new insights into the temporal dynamics of co-occurring mutations and paving the way for personalized treatment strategies and improved diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348ae5004fc1076373867ea177acfe998124027" target='_blank'>
              IMOP-Cancer: identifying mutation order pairs impacting cancer phenotypes
              </a>
            </td>
          <td>
            Yijing Zhang, Shaobo Kang, Renjie Dou, Wanmei Zhang, Yuanyuan Liu, Yang Wu, Dongxue Li, Fangfang Fan, Yanyan Ping
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Chemoresistance, the primary cause of mortality among ovarian cancer (OC) patients, is a multifaceted process encompassing numerous biological phenomena. As sequencing technology continues to advance, single-cell sequencing has surfaced as a potent strategy to elucidate the pathogenesis of OC. Methods We examined single-cell sequencing data derived from five OC samples (three resistant and two sensitive) and identified an epithelial subcluster associated with chemotherapy resistance and poor prognosis. Using GSVA and cell communication analysis, we explored the unique biological functions and communication characteristics of this resistant subcluster. We performed high dimensional weighted gene co-expression network analysis and differential expression analysis to identify the hub genes of c3. Lastly, we investigated the correlation between the hub gene, CLIC3, and chemotherapy drug sensitivity. We also validated their involvement in specific pathways using TCGA data. The effects and primary mechanism to chemoresistance of CLIC3 was explored. Results We identified a cell subcluster, denoted as c3, strongly linked to chemoresistance and poor prognosis in OC. This subcluster demonstrated a correlation with both extracellular matrix (ECM) formation and angiogenesis signature, with CLIC3 identified as its key marker. The expression levels of CLIC3 exhibit a significant association with the sensitivity to various chemotherapeutic drugs in OC. Mechanistically, CLIC3 increases OC resistance to cisplatin by promoting integrin β1 redistribution and PI3K-AKT pathway. Conclusions This study offers a novel insight into the progression and chemoresistance of OC. Additionally, we identified a specific cell cluster highly associated with chemoresistance. The marker for this cluster, CLIC3, increases OC resistance to cisplatin by promoting integrin β1 redistribution and PI3K-AKT pathway and holds significant potential as a new therapeutic target for OC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02882-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f56e8cc05a82444a4de8832553570aed3f3323" target='_blank'>
              Single-cell transcriptomics identify a chemotherapy-resistance related cluster overexpressed CLIC3 in ovarian cancer
              </a>
            </td>
          <td>
            Zhefeng Li, Jie Li, Yue Li, Junjie Yi, Xiaoting Zhao, Wentao Yue
          </td>
          <td>2025-06-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Integrating genome-wide association studies (GWAS) and transcriptomic datasets can identify potential mediators for germline genetic risk of breast cancer. However, traditional methods have been largely unsuccessful at most loci because of an overreliance on total gene expression. These approaches overlook alternative splicing, which can produce multiple isoforms from the same gene, each with potentially different effects on cancers. Here, we integrate genetic and multi-tissue isoform-level expression from the Genotype Tissue-Expression Project (GTEx, N = 838) with publicly available European-ancestry GWAS summary statistics to identify both isoform- and gene-level risk associations with overall breast cancer and estrogen receptor-positive (ER+) and -negative (ER-) subtypes. We employ our recently developed isoform-level Transcriptome-Wide Association Study (isoTWAS) framework that jointly models germline genetic variation, all isoforms of a gene, and phenotypic associations to identify transcriptomic mediators of genetic risk loci. Directly modeling isoform expression substantially increases discovery of transcriptomic mechanisms underlying genetic associations, compared to traditional methods leveraging total gene expression. Across the 3 indications, isoTWAS identified 2,304 more unique genes than TWAS, reflecting an 34.3% increase (jackknife SE = 3.8%) in effective sample size. In total, we find that isoTWAS-prioritized genes are enriched for pathways involved in DNA repair and damage responses, apoptosis, DNA and transcription factor binding, and cell cycle regulation. In addition, these genes are enriched for transcription binding sites of oncogenic transcription factors, like RUNX2, ESR1, and MYC. Critically, isoTWAS is able to pinpoint an isoform-specific mediation of the ESR1 and FOXA1 loci using isoform expression of non-cancerous breast tumor. Next, isoTWAS associations are enriched for genes that are evolutionary constrained and showing intolerance to protein-truncating variation (pLI score 0.90), prioritizing 587 constrained genes compared to 153 by TWAS (P = 2.8 x 10-9). This result suggests that isoform associations capture more likely disease signal. Of the 309 independent GWAS loci across the 3 indications, isoTWAS tagged 42% additional GWAS loci with transcriptomic associations (221 loci with isoTWAS, 156 with TWAS, 124 in common). Relatedly, the cis-genetic component of isoform expression (20.8 ± 7.3%) mediates an estimated 92.4% more of cancer risk SNP heritability compared to that of gene expression (10.8 ± 2.6%), using Mediated Expression Score Regression. Lastly, we highlight several salient and novel isoTWAS associations that exhibit isoform but not gene expression colocalization, including BABAM1 and FDPS, associated with overall breast cancer risk, and L3MBTL3 and CASP8, associated with ER- breast cancer risk. Our results strongly demonstrate that modeling isoform expression is crucial to maximize discovery of genetic risk mechanisms for breast cancer that are testable and actionable.
 Citation Format: Taylor Head, Yung-Han Chang, Tabitha Harrison, Yao Yu, Chad D. Huff, Bogdan Pasaniuc, Sara Lindström, Arjun Bhattacharya. Isoform-level analyses of breast cancer and its subtypes uncover extensive genetic risk mechanisms undetected at the gene-level [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS16-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2fd47608517f7db55237fbed695d206620b97ed" target='_blank'>
              Abstract PS16-03: Isoform-level analyses of breast cancer and its subtypes uncover extensive genetic risk mechanisms undetected at the gene-level
              </a>
            </td>
          <td>
            Taylor Head, Yung-Han Chang, Tabitha Harrison, Yao Yu, Chad D. Huff, Bogdan Pasaniuc, Sara Lindström, Arjun Bhattacharya
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc32b216cd93b5ffb3fc4906eb7ae996906a5999" target='_blank'>
              POFUT1 as a cancer biomarker: insights into its oncogenic mechanisms and clinical relevance.
              </a>
            </td>
          <td>
            Oumaima Mazour, Sébastien Legardinier, B. Badaoui, A. Germot
          </td>
          <td>2025-08-07</td>
          <td>Journal of molecular medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dec18318e074ddb92fb8c2657c66f417e7bed4dd" target='_blank'>
              Prime editor-based high-throughput screening reveals functional synonymous mutations in human cells.
              </a>
            </td>
          <td>
            Xuran Niu, Wei Tang, Yongshuo Liu, Binrui Mo, Ying Yu, Ying Liu, Wensheng Wei
          </td>
          <td>2025-06-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The epigenetic deregulation of CpG islands (CGIs) plays a crucial role in cancer initiation and progression. CGIs comprise 1-2% of the human genome and are rich in differentially methylated regions (DMRs) that can serve as cancer biomarkers in clinical samples and liquid biopsies. Focusing epigenetic sequencing on CpG-rich sequences, including CGIs and avoiding non-informative regions, offers an efficient and sensitive approach for cancer identification and tracking, especially within samples containing excess of unaltered, normal DNA. To this end, we have developed Adaptor-anchored Methylation amplification via Proximity Primers (aMAPP), a versatile PCR-based enrichment method. aMAPP employs specially designed primers to selectively enrich either methylated or unmethylated CpGs, depending on the upstream methylation conversion method employed. aMAPP achieves high coverage of genome-wide CGIs and detects hundreds of DMRs in tumor samples compared to adjacent normal tissue using ultra-low depth sequencing (∼300,000 reads). It enables tracing of aberrant methylation down to allelic frequency 0.01% in dilutions of tumor DNA and in cell-free DNA samples, can be applied using picogram amounts of DNA, and can be adapted to enrich either small panels of cancer-specific DMRs, or the majority (>90%) of genomic CGIs and CpGs. aMAPP offers a simple, cost-effective, and highly sensitive approach for capturing the epigenetic footprint of genome-wide CpGs and identifying aberrantly methylated or un-methylated genomic regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Prostate cancer (PCa) therapy faces challenges due to tumor heterogeneity, plasticity, and progression. Metabolic reprogramming, a dynamic process, has emerged as a key focus in PCa treatment. However, conventional therapies targeting cancer-specific metabolic pathways or employing chemosensitizers are often limited by compensatory mechanisms and metabolic complexity. This review highlights the roles of transcription factors, including AR, p53, c-Myc, HIF-1, Nrf2, and PPARγ, in regulating PCa metabolism by influencing signaling pathways, enzymes, and gene expression. Multi-target compounds, particularly natural products, show potential for disrupting multiple metabolic enzymes, opening up new research possibilities. Notable examples include β-elemene, juglone, tannic acid, and withaferin A, which target critical metabolic processes through enzyme inhibition, transcription factor modulation, epigenetic changes, and protein interaction disruption. Naturally derived metabolites can elicit transversal responses in diverse metabolic pathways, particularly in p53 and MYC transcription factors. Additionally, compounds such as pentacyclic terpenoids (ursolic acid with ursane skeleton), sulforaphane, and isothiocyanate-related moieties may induce metabolic and epigenetic changes through S-adenosyl methionine (SAM) and acetyl-CoA modulation, potentially affecting new areas of research through metabolic processes. We propose a cooperative crosstalk between metabolic reprogramming and transcription factors/epigenetic modulation in PCa. This approach holds potential for expanding PCa therapeutics and opening new avenues for research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d862b38e59640986b307371db97d2019ba1b7b42" target='_blank'>
              Targeting Prostate Cancer Metabolism Through Transcriptional and Epigenetic Modulation: A Multi-Target Approach to Therapeutic Innovation
              </a>
            </td>
          <td>
            P. Espitia-Pérez, Lyda Espitia-Pérez, Mario Negrette-Guzmán
          </td>
          <td>2025-06-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Nucleosome assembly protein 1-like 5 (NAP1L5), a critical regulator of gene transcription and nucleosome assembly, has been implicated in the progression and poor prognosis of various cancers. However, its specific role and molecular mechanisms in acute myeloid leukemia (AML) remain largely unexplored. Methods To identify key genes associated with AML, we analyzed gene expression profiles from AML patients and healthy controls using microarray datasets obtained from the GEO database. Differential expression analysis was performed to identify differentially expressed genes (DEGs), among which NAP1L5 emerged as a critical candidate based on its expression patterns and prognostic relevance, and we validated NAP1L5 expression in clinical AML samples. To elucidate the functional role of NAP1L5, we conducted Gene Set Enrichment Analysis (GSEA) and Gene Ontology (GO) analysis, which revealed its involvement in specific signaling pathways and biological processes. Furthermore, we constructed an interaction network and predictive model for NAP1L5, complemented by an assessment of its role in immune infiltration and drug sensitivity. Finally, we conducted in vitro experiments to explore its biological functions and underlying molecular mechanisms. Results In AML, elevated expression of NAP1L5 was significantly associated with reduced overall survival, underscoring its prognostic relevance. GSEA revealed that NAP1L5 was prominently enriched in pathways related to apoptosis and DNA replication. GO analysis further indicated that its co-expressed genes were closely linked to autophagy and stress response mechanisms. Interaction network analysis revealed that NAP1L5 engages in complex regulatory interactions with multiple genes, miRNAs, transcription factors (TFs), and RNA-binding proteins (RBPs). Notably, high NAP1L5 expression correlated with increased infiltration of resting CD4+ memory T cells, implicating its potential influence on the tumor immune microenvironment. A predictive model integrating NAP1L5 expression and clinical AML features exhibited robust prognostic utility. Drug sensitivity analysis identified NAP1L5 overexpression as a marker of resistance to Zibotentan, along with associations with 49 additional therapeutic agents. In vitro functional assays demonstrated that NAP1L5 overexpression promoted cellular proliferation, migration, and colony formation while concurrently inhibiting apoptosis, highlighting its oncogenic potential in AML pathogenesis. Conclusions NAP1L5 emerges as a promising prognostic biomarker and therapeutic target in AML, offering potential for improved patient outcomes and precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be458e5cdd1834f6b055dae4c08b4d9e77e28c87" target='_blank'>
              NAP1L5 in acute myeloid leukemia: a prognostic biomarker and potential therapeutic target
              </a>
            </td>
          <td>
            Meng Wang, Zhibin Xie, Yuanyuan Tan, Yan Zhou, Tingting Zhang, Yuqing Du, Huan Wu, Lili Zhou, Jian Ge
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Histone post-translational modifications (PTMs) have long been recognized as critical regulators of chromatin dynamics and gene expression, with aberrations in these processes driving tumorigenesis, immune escape, metastasis, and therapy resistance. While multi-omics technologies are generating ever more detailed maps of the histone landscape, translating these insights into clinical practice remains challenging. The ongoing convergence of high-throughput omics technologies and Artificial Intelligence (AI) is revolutionizing drug repositioning strategies, offering new precision tools to identify histone-targeted therapies for solid tumors. In this review, we explore how AI-driven multi-omics integration is currently reshaping therapeutic opportunities by uncovering novel drug–target–patient associations with unprecedented accuracy. Special focus is given to gynecologic and breast cancers, where chromatin remodeling dysregulation is particularly widespread, conventional therapeutic approaches have demonstrated substantial limitations and drug resistance represents a major clinical obstacle. These aggressive and lethal cancers exemplify areas where AI-powered repurposing of epi-drugs is making tangible clinical advances, enhancing tumor sensitivity to treatments like immunotherapy, but also offering new avenues to overcome challenging phenomena such as drug resistance and cancer relapse. We critically discuss these challenges and the effectiveness of a combination strategy approaches based on AI-driven patient stratification and biomarker-guided therapy optimization to maximize clinical benefits. In an era where precision oncology demands both specific drugs and the application of smarter strategies, the integration of AI, multi-omics, and targeting of chromatin remodelers may herald a transformative shift in the management of solid tumors, bridging the gap between biological insights and therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93d5e311cf51a6ad8df9c768a84ae34a6dc7d61" target='_blank'>
              Multi-omics based and AI-driven drug repositioning for epigenetic therapy in female malignancies
              </a>
            </td>
          <td>
            A. Salvati, Viola Melone, A. Giordano, J. Lamberti, Domenico Palumbo, L. Palo, Dilia Rea, Domenico Memoli, Vittoria Simonis, E. Alexandrova, Francesco Silvestro, Francesca Rizzo, Alessandro Weisz, Roberta Tarallo, G. Nassa
          </td>
          <td>2025-07-25</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Therapeutic resistance remains a primary obstacle to curing cancer. Healthy cells exposed to genotoxic insult rapidly activate both p53-dependent and -independent non-genetic programs that pause the cell cycle and direct either DNA repair or apoptosis. Cancer cells exploit these same pathways as they respond to stresses induced by cancer therapies. In this study, we investigated a potential role for upstream stimulatory factor 1 (USF1) and USF2 in the p53-independent response of lymphoma cells to genotoxic therapy. We previously found that lymphocytes utilize the responsiveness of USF1 to double-stranded DNA breaks to coordinate T cell receptor beta (Tcrb) gene expression during V(D)J recombination. Here, microarray gene expression analysis of derivatives of the p53-deficient mouse B lymphoma cell line, M12, revealed that simultaneously depleting cells of both USF1 and USF2 altered the expression of 940 gene transcripts (>1.50-fold change, < 0.05 FDR), relative to cells expressing a scrambled control shRNA. Seven days after exposure to a single sublethal (5 Gy) dose of ionizing radiation, USF-depleted (USFKD) cells exhibited widespread and distinct transcriptional responses from those of irradiated controls (5035 and 5054 differentially expressed gene transcripts, respectively, with roughly half shared between both cell types). Gene ontology analyses revealed that USF knockdown induced numerous changes in the expression of genes critical for immune development and function while diminishing the responsiveness of genes linked to DNA damage pathways. Microarray findings were confirmed by RT-qPCR for a panel of genes responsive to USF knockdown and/or irradiation. These findings shed further light on transcriptional responses to ionizing radiation that manifest over time in transformed cells, identifying a novel p53-independent role in lymphocytic DNA damage stress responses for USF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6bb6187205d63162ff7fe5e73d3f94fd90940ba" target='_blank'>
              Knockdown of USF1 and USF2 drives prolonged changes in the gene expression response of M12-5B3 cells to DNA damage
              </a>
            </td>
          <td>
            Kimberly S Bellingham-Johnstun, Lisa A Metzger, Jennifer L Stone, Kevin Gerrish, Pierre R Bushel, Michael L Sikes
          </td>
          <td>2025-07-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Breast cancer (BC) is the leading cause of cancer-related deaths in women. It is molecularly and clinically heterogeneous, driven by distinct genetic and epigenetic alterations, making prognosis prediction and treatment selection challenging. Identifying novel markers for BC progression and therapeutic targets remains an urgent need. Although growing research has highlighted the role of epigenetic mechanisms, particularly cancer-associated histone mutations (onco-histone), in regulating BC pathogenesis and its hallmarks, including stemness and drug resistance, the role of histone variants—the naturally derived sequence variants—remains largely unknown, with the exception of H3.3. The newly identified, mammalian-specific histone variant H2A.J has been shown to be highly enriched in human luminal BC and implicated in a stem cell signature correlated with metastatic risk in primary BC. However, it is unclear what role H2A.J plays in BC and how it may function in other BC subtypes. Using cell lines of multiple BC subtypes and patient-derived xenograft (PDX) tumor microarrays, our studies have discovered that H2A.J is present in human triple-negative BC (TNBC), with significantly elevated levels in metastatic TNBC compared to the primary counterpart. Our unpublished results show that H2A.J is the only histone variant that strongly and specifically interacts with the tumor suppressor, retinoblastoma 1 (RB1) in metastatic TNBC cells and luminal BC. Our data also demonstrate that RB1 protein level is significantly correlated with H2A.J in estrogen receptor-positive (ER+) luminal BC PDX samples. Our preliminary results suggest that H2A.J chromatin occupancy affects RB1 recruitment to chromatin, RB1-bound chromatin targets, and ultimately RB1-dependent phenotypes. We will discuss how H2A.J regulates RB1-bound chromatin loci, and hence, the canonical and non-canonical functions of RB1 in tumorigenesis. Our bulk RNA-sequencing and western blotting data show that knocking down (KD) H2A.J interferes with G1/S cell cycle progression and results in the upregulation of estrogen response, apoptosis, and mTOR signaling. Notably, the H2A.J KD-transcriptome signature strongly correlates with that of CDK4/6 inhibitor treatment. Our data also suggest H2A.J's role in maintaining cell lineage fidelity. Overall, our results suggest that H2A.J regulates cellular heterogeneity, stemness, and metastasis in BC. The outcome of our research will not only advance our understanding of the underlying mechanisms leading to BC progression and metastasis but also offer new opportunities for utilizing H2A.J as a target for diagnostic and therapeutic interventions in BC.
 Citation Format: Rana Gbyli, Tyler Jensen, Joseph Balowski, Yao Li, Xiang Zhang, Qin Yan, Andrew Xiao. Harnessing the Power of the Variant: Breast Cancer's Strategic Incorporation of the Histone H2A.J [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6e23951bd3e66f36e177a58521abd15ef2dc11" target='_blank'>
              Abstract P5-06-22: Harnessing the Power of the Variant: Breast Cancer's Strategic Incorporation of the Histone H2A.J
              </a>
            </td>
          <td>
            Rana Gbyli, Tyler Jensen, Joseph Balowski, Yao Li, Xiang Zhang, Qin Yan, Andrew Xiao
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Polymerase theta (POLθ) inhibitors were developed to overcome resistance to PARP inhibitor treatment in homologous recombination (HR) deficient cancer. Biomarkers identifying PARP inhibitor-resistant cancer cases that specifically rely on POLθ activity for cancer cell survival will help to identify the sensitive patient population. From the TCGA RNAseq data, we determined POLθ expression levels, and from whole-exome and whole-genome sequencing data, we determined the number of POLθ-associated mutational signatures in solid tumors with various HR deficiency status. We found that POLθ expression levels did not differ significantly between HR-proficient and HR-deficient cancers. However, POLθ expression correlated strongly with proliferation-associated gene expression signatures and was predominantly observed in the S and G2/M phases of the cell cycle. POLθ-associated mutational signatures are correlated with POLθ expression levels only in BRCA2-deficient cancers. POLθ expression level and POLθ-associated mutational signatures may be indicative of POLθ inhibitor sensitivity in BRCA2-deficient tumors, but are unlikely to be informative in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498ae751ba9f700cb39c5f192432c94acf2d269" target='_blank'>
              Polymerase theta expression is correlated with proliferative capacity but not with DNA repair deficiency status in solid tumors
              </a>
            </td>
          <td>
            Z. Sztupinszki, Regina Fiam, O. Pipek, M. Dióssy, J. Börcsök, Aurél Prósz, I. Csabai, Zoltan Szallasi
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Breast cancer remains one of the most prevalent malignancies worldwide, and radiation therapy is a central component of its management. However, intrinsic or acquired resistance to radiation significantly compromises therapeutic efficacy. This systematic review aimed to identify and evaluate molecular mechanisms and interventions that influence radiation sensitivity in breast cancer models. A comprehensive PubMed search was conducted using the terms “breast cancer” and “radiation resistance” for studies published between 2002 and 2024. Seventy-nine eligible studies were included. The most frequently investigated mechanisms included the dysregulation of the PI3K/AKT/mTOR and MAPK signaling pathways, enhanced DNA damage repair via non-homologous end joining (NHEJ), and the overexpression of cancer stem cell markers such as CD44+/CD24−/low and ALDH1. Several studies highlighted the role of non-coding RNAs, particularly the lncRNA DUXAP8 and microRNAs such as miR-21, miR-144, miR-33a, and miR-634, in modulating radiation response. Components of the tumor microenvironment, including cancer-associated fibroblasts and immune regulators, also contributed to radiation resistance. By synthesizing current evidence, this review provides a consolidated resource to guide future mechanistic studies and therapeutic development. This review highlights promising molecular targets and emerging strategies to enhance radiosensitivity and offers a foundation for translational research aimed at improving outcomes in radiation-refractory breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a4cbc0cd4ca4c1e106a823d69b14920681deda3" target='_blank'>
              Molecular Mechanisms of Radiation Resistance in Breast Cancer: A Systematic Review of Radiosensitization Strategies
              </a>
            </td>
          <td>
            Emma Mageau, Ronan Derbowka, Noah Dickinson, Natalie Lefort, A. T. Kovala, D. Boreham, T. C. Tai, Christopher Thome, S. Tharmalingam
          </td>
          <td>2025-07-24</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is crucial for balancing X-linked gene dosage in female cells by randomly silencing one X chromosome during early embryogenesis. However, accurately classifying cells based on the parental origin of the inactivated X chromosome in single-cell samples remains challenging. Here we present FemXpress, a computational tool leveraging X-linked single nucleotide polymorphisms (SNPs) to group cells based on the origin of the inactivated X chromosome in female single-cell RNA sequencing (scRNA-Seq) data. FemXpress performs robustly on both simulated and real datasets, without requiring parental genomic information, and can also identify genes that escape XCI. Applying FemXpress to single-cell RNA-Seq data from multiple tissues of a cynomolgus monkey, we reveal heterogeneity in XCI origin across organs and cell types. In each organ, we identify candidate XCI-escaping genes, and within each cell type, we observe gene expression differences associated with XCI origin, potentially contributing to phenotypic variability. Furthermore, FemXpress demonstrated strong performance in phasing XCI in scRNA-Seq datasets from embryos and colon tumors. In summary, FemXpress provides a powerful approach for XCI status analysis, offering new insights into XCI dynamics at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478a669d81b3cf05029932ee293479ee1f9bcebc" target='_blank'>
              FemXpress: Systematic Analysis of X Chromosome Inactivation Heterogeneity in Female Single-Cell RNA-Seq Samples.
              </a>
            </td>
          <td>
            Xin Wang, Yingke Ma, Fan Li, Wentao Cui, Tianshi Pan, Siqi Wang, Sinan Ma, Qingtong Shan, Chao Liu, Yukai Wang, Ying Zhang, Yuanchun Zhou, Wei Li, Pengfei Wang, Qi Zhou, Guihai Feng
          </td>
          <td>2025-06-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The ASXL1 gene is one of the most frequently mutated genes in acute myeloid leukemia (AML). It is associated with signs of aggressiveness and adverse clinical outcomes. The aim of the current study was to analyze the genetic profile of ASXL1 gene mutations and its impact on the overall survival in AML patients from Pakistan. Thirty-eight well characterized AML patients were enrolled, and DNA sequencing of the ASXL1 was performed using the Illumina NextSeq500 next generation sequencing (NGS) system. Standard pipeline of bioinformatics tools was used to determine the mutational profile. The mutational profile of the enrolled AML patients was compared with that of 1000 Genomes project, and TCGA AML datasets. The analysis revealed 43 genetic variants in ASXL1 across the 38 AML patients (1.13 variant/patient). Eight rare variants were observed in exons 12, 13 of the ASXL1 gene. Notably, a recurrent rare nonsynonymous deleterious variant p.G1336S in exon 13 (NM_015338 transcript) was found in two patients (5.26%). The overall survival of the ASXL1+ (but TP53, FLT3, NPM1, EZH2, and WT1 negative) AML was shorter compared with the ASXL1- (p < 0.05). Further, the overall survival of current study ASXL1 + AML was found comparable with that of the TCGA AML. In conclusion, the non-silent mutations in ASXL1 were associated with lower survival in AML. Further studies with larger cohort are suggested for subsequent clinical implementation. Supplementary Information The online version contains supplementary material available at 10.1186/s12863-025-01324-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03f43436989187938c1f8f140ca42098c22feb4" target='_blank'>
              Genetic profile of ASXL1 gene in risk assessment in acute myeloid leukemia
              </a>
            </td>
          <td>
            Salem Ahmed Bamusa, Wardah Qureshi, Atia Gohar, Muhammad Irfan, I. Khan, Muhammad Shakeel
          </td>
          <td>2025-07-12</td>
          <td>BMC Genomic Data</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background/Aim: Recent advancements in omics analysis have significantly enhanced our understanding of the molecular pathology of malignant melanoma, leading to the development of novel therapeutic strategies that target specific vulnerabilities within the disease. Despite these improvements, the factors contributing to the poor prognosis of patients with malignant melanoma remain incompletely understood. The aim of this study was to investigate the role of C1orf50 (Chromosome 1 open reading frame 50), a gene previously of unknown function, as a prognostic biomarker in melanoma. Materials and Methods: We performed comprehensive transcriptome data analysis and subsequent functional validation of the human Skin Cutaneous Melanoma project from The Cancer Genome Atlas (TCGA). Results: Elevated expression levels of C1orf50 correlated with worse survival outcomes. Mechanistically, we revealed that C1orf50 plays a significant role in the regulation of cell cycle processes and cancer cell stemness, providing a potential avenue for novel therapeutic interventions in melanoma. Conclusion: This study is the first to identify C1orf50 as a prognostic biomarker in melanoma. The clinical relevance of our results sheds light on the importance of further investigation into the biological mechanisms underpinning C1orf50’s impact on melanoma progression and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97ee80e750fee0c874b5ff3d0f859ce68b0b3f4" target='_blank'>
              C1orf50 Drives Malignant Melanoma Progression Through the Regulation of Stemness
              </a>
            </td>
          <td>
            Yusuke Otani, Masaki Maekawa, Atsushi Tanaka, Tirso Peña, Vanessa D. Chin, Anna Rogachevskaya, Shinichi Toyooka, M. H. Roehrl, Atsushi Fujimura
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers, with an incidence that has increased significantly over the last 20 years. The World Health Organization (WHO), under its cancer program, identifies CRC as the third most prevalent cancer worldwide with a high mortality rate, especially in patients under the age of 50. Despite advances in surgery, chemotherapy, radiotherapy, and molecular targeted therapy, CRC shares a low survival rate and poor prognosis due to late diagnosis. To address these challenges, research into alternative diagnostic and therapeutic strategies has increasingly focused on molecular mechanisms, including noncoding RNAs (ncRNAs). Circular RNAs (circRNAs), a subclass of endogenous ncRNAs characterized by their covalently enclosed loop structure, demonstrate greater stability than linear RNAs, making them potential candidates for clinical applications. The circRNAs possess differential expressions in cancers and function as tumor suppressive or oncogenic activities. This review discusses the recent findings on circRNAs and their potential for biomarkers and therapeutic targets in CRC. As circRNAs can serve as miRNA sponges, increase or decrease mRNA expression, and function to regulate an oncogenic or tumor suppressor pathway, there may be the possibility for an early diagnosis, prognosis, or therapeutic role of circRNAs in CRC. Highlighting the clinical implications of circRNAs, this review discusses their potential to transform current CRC management strategies and address critical gaps in timely diagnosis and effective treatment. It also emphasized the need for further clinical research to validate their utility and translate these findings into practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db80f20f575f8c248f315abb0f46ec8b7f78d57b" target='_blank'>
              A review of circular RNAs in colorectal cancer: insights into biomarker discovery and therapeutic targeting
              </a>
            </td>
          <td>
            Sibin Nambidi, Karunakaran Bhuvaneswari Shruthi, Sujatha Padmanaban Kanimozhe, Antara Banerjee, S. Kondaveeti, A. Duttaroy, S. Pathak
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Neuroblastoma (NB) is the most common extracranial solid tumor in early childhood and frequently presents with bone marrow (BM) metastasis, particularly in high-risk cases. Metastatic NB cells residing in the BM exhibit distinct biological behaviors and are closely associated with treatment resistance and poor prognosis. Emerging evidence suggests that metabolic reprogramming is a hallmark of NB progression; however, its regulatory landscape within the bone marrow microenvironment remains poorly understood. This study aimed to systematically elucidate the molecular mechanisms underlying metabolic reprogramming in bone marrow–infiltrating metastatic NB cells by integrating single-cell transcriptomic data, bulk RNA-sequencing profiles, and in vitro functional validation. Single-cell RNA-sequencing data from 17 bone marrow aspirates of NB patients—with and without marrow infiltration—were retrieved from the GEO database, while bulk RNA-seq data from 155 NB tumor samples were obtained from the TARGET database. A suite of bioinformatics tools, including Seurat, Harmony, CellChat, and SCENIC, was employed to perform multi-dimensional analyses such as cell subtype annotation, intercellular communication mapping, and transcription factor regulon construction. AUCell scoring, differential gene expression analysis, random survival forest modeling, and Kaplan–Meier survival analysis were conducted to identify key genes involved in metabolic reprogramming and to characterize their immune infiltration features. Gene Set Enrichment Analysis (GSEA), Gene Set Variation Analysis (GSVA), and single-cell pathway scoring were applied to predict the metabolic and immune pathways associated with these candidate genes. Furthermore, co-expression network analysis was used to evaluate the relationships between candidate genes and known NB regulatory factors. Finally, in vitro experiments were carried out to validate the roles of selected genes in regulating NB cell proliferation, migration, invasion, and mitochondrial function. Single-cell transcriptomic analysis of bone marrow–derived NB samples identified nine distinct cell subpopulations with diverse intercellular communication networks. Among these, the ligand–receptor pair MDK–NCL emerged as a key mediator of cell–cell signaling. Regulatory network analysis revealed five critical regulons—JUND, JUNB, FOS, E2F1, and KLF16—closely associated with metabolic reprogramming in NB. The “Neuroblastoma cell” cluster displayed markedly elevated metabolic activity. Through integrated analyses, five core metabolic reprogramming genes (MRPL21, NHP2, RPL13, RPL18A, and RPL27A) were identified and shown to be significantly associated with poor prognosis. High expression of these genes correlated with an immunosuppressive tumor microenvironment, characterized by reduced infiltration of monocytes, M1 macrophages, and T cells. Functional enrichment analysis revealed that these genes were primarily enriched in oxidative phosphorylation, MYC targets, PI3K-Akt, and p53 signaling pathways. Co-expression network analysis further demonstrated that MRPL21 and NHP2 positively correlated with known NB regulatory genes TP53, NRAS, and NKIF1B. In vitro assays confirmed that MRPL21 knockdown significantly impaired NB cell proliferation, migration, invasion, and mitochondrial oxidative phosphorylation. This study identified five key genes involved in metabolic reprogramming of bone marrow–infiltrating NB cells, which are closely associated with immunosuppressive microenvironment formation and enrichment in tumor-associated metabolic pathways. Among these, MRPL21 plays a pivotal role in regulating NB cell proliferation and mitochondrial function, underscoring its potential as a promising therapeutic target. These findings suggest that integrated strategies targeting both tumor metabolism and the immune microenvironment may provide new avenues for the treatment of high-risk, metastatic NB. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13626-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b395447d21c8867ac393546695c70f8056f2bca" target='_blank'>
              Integrated single-cell and transcriptomic analysis of bone marrow-derived metastatic neuroblastoma reveals molecular mechanisms of metabolic reprogramming
              </a>
            </td>
          <td>
            Jing Chu, Rong Qin, Shu-Jing Wang, Qiang Wang, Qiang Wu
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290d92451b26f978216d79d0c80c8e4f6746115f" target='_blank'>
              LaGrACE: estimating gene program dysregulation with latent regulatory network.
              </a>
            </td>
          <td>
            Minxue Jia, Haiyi Mao, Mengli Zhou, Yu-Chih Chen, P.V. Benos
          </td>
          <td>2025-06-30</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Breast cancer is subdivided into four distinct subtypes based on the status of hormone receptors (HR) and human epidermal growth factor receptor 2 (HER2) as HER2−/HR+, HER2+/HR+, HER2+/HR− and HER2−/HR−. Among this, ERα positive breast cancer, even though they respond to endocrine treatment, half of the patients acquire resistance and progress with metastasis despite ERα status. Spatio-temporal changes in ERα and their loss under treatment pressure have been reported in a subset of patients, which is a serious problem. Results We have demonstrated that in vitro-generated resistance is correlated with the down regulation of ERα. To study the ERα status transition in live cells, triple-negative breast cancer cells were engineered to express EGFP-ERα, which further supported the existence of complex intracellular signaling that regulates ERα plasticity even in unperturbed conditions. Single-cell clones generate heterogeneity and loss of expression depending on proliferative cues. However, the initial response of cells to 4 μM of 4-hydroxytamoxifen and 1 μM of endoxifen involves up-regulation of ERα, likely due to its early effect on the proteasome or autophagy pathway. Supporting this, inhibition of autophagy and the proteasome further enhanced the expression of ERα. Systematic analysis of RNA sequencing of ERα stable cells further confirmed that ERα regulates diverse intracellular signaling networks such as ubiquitin, proteasome pathways, cell proliferation and Unfolded Protein Responses (UPR), implicating its direct role in post-translational protein modifications. Cell cycle indicator probe expressing receptor-positive breast cancer cells confirmed the ERα expression heterogeneity both in 2D and 3D culture in a cell cycle phase-independent manner. Conclusions Overall, the study confirms the cell’s intrinsic post-transcriptional mechanisms of ERα plasticity that could play a role in receptor heterogeneity and tumor progression under endocrine treatment, which warrants further investigation. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00653-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eba37c41f7988b889cf7020b223e234c792d04" target='_blank'>
              Estrogen receptor alpha dynamics and plasticity during endocrine resistance
              </a>
            </td>
          <td>
            Aswathy Sivasailam, Kiran S. Kumar, Aparna Geetha Jayaprasad, Shine Varghese Jancy, Ashwathi Harikumar, P. S. Unnikrishnan, K. C. Sivakumar, P. J. J. Tiffee, A. M. Halikar, S. S. Nithin, P. R. Pillai, Shivanshu Kumar Tiwari, Vishnu S. Sanjeev, A. V. Surabhi, T. Santhoshkumar
          </td>
          <td>2025-06-13</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The treatment of prostate cancer (PCa) remains a major challenge due to the development of resistance to androgen receptor (AR)-targeted therapies and paclitaxel-based chemotherapeutic agents, highlighting the need for novel therapeutic approaches. Unlike the majority of tumor cells that depend on aerobic glycolysis (the Warburg effect) for energy production, early stage PCa primarily utilizes oxidative phosphorylation (OXPHOS), while advanced-stage PCa shifts to aerobic glycolysis. Consequently, targeting metabolic alterations specific to PCa may represent a promising therapeutic strategy. This review initially delineates the glucose metabolism phenotype characteristic of healthy prostate cells as well as primary and metastatic PCa cells, which underpin the development and progression of the disease. Subsequently, it elaborates on the roles and mechanisms of inhibitors targeting key enzymes involved in glycolysis and the synthesis of OXPHOS in the context of PCa treatment. Finally, the review discusses inhibitors that concurrently target both glycolysis and OXPHOS, potentially mitigating the compensatory pathways that support cancer cell survival. Collectively, this review contributes to a deeper understanding of metabolic alterations in PCa and informs the ongoing development of therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b63cc1f7a4165fbfee5541d9551a7ece3070ea7" target='_blank'>
              Targeted Metabolic Alterations in Prostate Cancer: A Promising Therapeutic Strategy.
              </a>
            </td>
          <td>
            Zhanning Qu, Feng Hao, Zhaojun Li, Liu Yang, Hongxia Gao, Junying Li, Shuang Chen, Liguo Wang, Zhiming Xiu, Cheng Hu
          </td>
          <td>2025-08-04</td>
          <td>Molecular pharmaceutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Mutations in tumor protein p53 (TP53) and ataxia telangiectasia mutated (ATM) genes are frequently observed across various solid and hematologic malignancies and are associated with genomic instability, treatment resistance, and poor clinical outcomes. These alterations compromise the G1/S cell cycle checkpoint and increase cellular dependence on compensatory DNA damage response (DDR) pathways, including the ataxia telangiectasia and Rad3-related protein kinase (ATR)–checkpoint kinase 1 (CHK1)–WEE1 G2 checkpoint kinase (WEE1) axis. This has led to the development of DDR-targeted therapies that exploit synthetic lethality in tumors with TP53 or ATM dysfunction. Inhibitors targeting ATM, ATR, CHK1, and WEE1 have all shown encouraging activity in early-phase clinical trials, particularly in biomarker-enriched subgroups. Poly(ADP-ribose) polymerase (PARP) inhibitors—originally approved for BRCA1/2-mutated breast cancers—are now being evaluated in TP53- or ATM-deficient tumors, often in combination with other DDR-targeting agents to enhance efficacy. Clinical trials increasingly support the efficacy of DDR inhibitors in biomarker-defined DDR-deficient tumors, specifically beyond BRCA mutations. This review summarizes current understanding of DDR-targeted strategies in TP53- and ATM-mutant cancers, with an emphasis on relevant clinical data and ongoing trials. Expanding the clinical use of DDR inhibitors based on molecular profiles may provide new therapeutic options for genomically unstable tumors across adult and pediatric populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c9cb81a9e0dfbd38939e93feb0489096524fb3" target='_blank'>
              Therapeutic Targeting of DNA Damage Response Pathways in TP53- and ATM-Mutated Tumors
              </a>
            </td>
          <td>
            Y. Shim
          </td>
          <td>2025-07-01</td>
          <td>Brain Tumor Research and Treatment</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith N Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Breast cancer is driven by complex oncogenic mechanisms, including aberrant signaling pathways and genetic mutations. Current therapies often face limitations, such as drug resistance and adverse effects, necessitating the identification of novel therapeutic agents. Protopine, known for its anticancer properties in other malignancies, remains unexplored for breast cancer. This study investigates the therapeutic potential of protopine in breast cancer by examining its effects on critical molecular targets and signaling pathways involved in tumor progression and metastasis through computational pharmacological approaches. Breast cancer‐associated targets were identified, and protopine's binding affinity for PIK3CA was evaluated using molecular docking and molecular dynamics simulations. Binding stability and dynamics were assessed using RMSD, hydrogen bonding analysis, and MM‐PBSA energy calculations. Protopine exhibited multitargeted activity, interacting with 17 breast cancer‐associated targets, including PIK3CA, CHEK1, and CDK2. It modulated critical pathways such as PI3K‐Akt, JAK‐STAT, and VEGF, disrupting processes like cell proliferation and angiogenesis. Molecular docking showed strong binding affinity for PIK3CA (−9.2 kcal/mol), while molecular dynamics confirmed stable binding with favorable free energy (−6.71 ± 0.64 kcal/mol). Protopine demonstrated significant therapeutic promise as a multitargeted agent, modulating key oncogenic pathways and providing the potential to overcome resistance and improve breast cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfa9c411160ba1ed4ea959d12995e9684354661" target='_blank'>
              Protopine as a Potential Breast Cancer Therapeutic: A Computational Study Targeting PIK3CA and Cancer Networks
              </a>
            </td>
          <td>
            Jagdish Chand, Hannah Lalengzuali Fanai, Sheikh F. Ahmad, Sabry M. Attia, T. Emran
          </td>
          <td>2025-07-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Indroduction Ara‐C (cytarabine) resistance remains a significant contributor to the poor clinical outcomes in adult acute myeloid leukemia (AML). However, predicting Ara‐C resistance and developing effective targeted therapies remain challenging. Methods In this study, we integrated transcriptional data from Ara‐C‐resistant cell lines in the GEO database and the TCGA‐LAML cohort to establish an Ara‐C resistancerelated gene risk score (ARRGRS). Kaplan‐Meier survival analysis revealed that AML patients with high ARRGRS had significantly worse prognosis compared to those with low ARRGRS in both cohorts. Additionally, ARRGRS effectively predicted chemotherapy response in AML patients across both cohorts. To further elucidate the mechanisms underlying Ara‐C resistance, we constructed Ara‐C‐resistant AML cell lines and validated our findings using qPCR, Western blotting, flow cytometry (FCM), and in vivo experiments. Results We discovered that high expression of S100A4 promotes Ara‐C resistance in AML. Mechanistically, we identified that the transcription factor NR6A1 directly binds to the S100A4 promoter, enhancing its transcriptional activity. Subsequently, S100A4 upregulates p53 expression, thereby promoting AML cell proliferation and resistance to Ara‐C. Discussion In summary, our comprehensive investigation of the ARRGRS not only deepens the understanding of Ara‐C resistance mechanisms but also provides promising insights for targeting S100A4 to inhibit tumor growth and overcome chemotherapy resistance in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53bd4c13b15638258f10cd0ef72e9392cef867e" target='_blank'>
              Identification of an Ara-C resistance-related gene risk score and the role of S100A4 in AML via NR6A1-dependent activation and p53 regulation
              </a>
            </td>
          <td>
            Li Wang, Aoshuang Huang, Bingqing Cheng, Xiuying Hu, Jishi Wang
          </td>
          <td>2025-06-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is among the most common oncological diseases in the world, associated with a high mortality rate. Recently, immunotherapeutic approaches to the treatment of CRC have been developed, which have enabled achievement of long-term and stable remission in a certain group of patients. The success of immunotherapy depends on the immune processes associated with oncogenesis in the colon. The key role in modulating immune environment of the tumor is played by the activity of genes regulating differentiation and functions of immune cells, as well as by a variety of microRNAs that perform posttranscriptional expression control of target genes. In this regard, the aim of our work was to study the expression profiles of immune-associated genes and microRNAs to determine the immune signaling mechanisms of colon cancer and search for potential therapeutic targets. The study included 18 persons (12 women, 6 men, median age – 66 years) diagnosed with colon cancer, having been treated at the National Medical Research Center of Oncology in 2018-2019. All tumors had microsatellite instability status. The transcription profiles of microRNAs and immune response genes were determined by next-generation sequencing. The study has revealed 28 differentially expressed microRNAs in the tumor tissues, including 15 microRNAs with increased expression and 13 microRNAs with decreased expression. Of the 395 genes examined, 156 were differentially expressed, including 62 genes with increased expression and 94 genes with decreased activity. Identification of cell populations based on the transcriptional pattern of tumors has discerned four types of immune cells: neutrophils, B lymphocytes, CD8+T lymphocytes, and M1 macrophages. Neutrophils were the most common cell type (16 of 18 samples). A stepwise analysis of possible relationships between transcriptional patterns allowed us to select 713 miRNA-mRNA pairs. After filtering by validated interactions, 24 such pairs were found. On this basis, a miRNA-mRNA interaction network was constructed, which could be involved into the regulation of multiple immune signaling mechanisms and cell cycle control. Most of the revealed signaling pathways contained CDK1, for which therapeutic inhibitors are known, being at different stages of clinical trials. The possibilities of influencing CDK1 described in the present work may be used for further clinical studies and development of therapeutic strategies employing selective molecular suppression of immunemediated carcinogenesis in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ac153b8c05d98ac275bf9faf7493386df7f7150" target='_blank'>
              MicroRNAs and target genes as regulators of colon cancer immune signaling
              </a>
            </td>
          <td>
            O. I. Kit, D. Gvaldin, N. A. Petrusenko, N. Timoshkina, D. A. Savchenko, I. A. Novikova
          </td>
          <td>2025-07-07</td>
          <td>Medical Immunology (Russia)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background: Combinations of endocrine therapy and CDK4/6 inhibitors (CDK4/6i) used in metastatic breast cancer (MBC) result in heterogenous clinical benefit with subsets of patients having disease control ranging anywhere from a few months to more than 5 years. Understanding the mechanisms that account for these differences can enable new pharmacologic approaches to convert more of these cancers into the long-term control subset. Methods: To elucidate the causal factors that underlie the observed clinical variability, we have utilized multi-omic analyses of patient samples as well as experimental modeling using a diverse cohort of ER+ breast cancers. For clinicogenomic analyses, we analyzed 467 patients with ER+/ HER2- MBC treated with first-line CDK4/6i at our institution (MSK) with detailed clinical follow annotation and follow up. For genomic/transcriptomic studies, we analyzed single cell RNA sequencing data, whole genome sequencing data, and Ki67 score from the neoadjuvant CDK4/6i containing arms in the FELINE trial (NCT02712723). Patient derived xenografts, patient derived organoids, isogenic cell lines were used to define the underlying mechanisms. Live cell imaging assays were performed by FUCCI cell cycle sensor to assess cell cycle transitions. Results: Of 467 patients from MSK cohort, 129 (27.6%) had pre-CDK4/6i loss-of-function variants in TP53, corresponding to short PFS (median PFS= 8.7 months, 95% CI: 6.6–10.3; HR = 2.12, 95% CI: 1.68 to 2.68; p = 2.64e-10). This finding was validated in two large independent clinical cohorts outside MSK. In patient derived and isogenic cell lines, p53 loss had no impact on initial drug response but promoted long-term cell outgrowth and prevented emergence of senescence phenotypes. Following cells in detailed time courses revealed the propensity of p53 mutant but not wild type cells to reenter the cell cycle after initial arrest by CDK4/6 inhibition. Gene expression analysis of RNA seq data revealed a significant positive association between TP53 mutation status and DREAM complex genes, assessed by GSEA (NES= 1.51, p = 1e-4) In models, deletion of DREAM complex component, p130-Rbl2 was found to fully recapitulate the effects of p53 loss: promoting long-term outgrowth from CDK4/6i, blocking therapy induced senescence, and promoting cell cycle re-entry. To validate these results, we analyzed the multi-omic and histologic characteristics of serial samples from the FELINE trial. Compared with WT cases, baseline TP53 LOF had a higher amount of proliferating cells at the end of treatment, measured by Ki67 > 10% (4.5% of WT cases vs. 60% of mutant cases, respectively, OR = 26.8, p = 0.014), the per cell proliferation score, measured by ssGSEA (log2FC: 0.2 vs. 2.8, respectively) as well as a higher pseudobulk expression of DREAM complex related genes (GSEA: NES = 2.0, q = 3.7e-4). Finally, to identify strategies that might facilitate stable cell cycle control of p53KO tumors, we investigated which targets of p53 regulate DREAM and identified p21. Given role of p21 in regulating both CDK4/6 and CDK2 we investigated whether CDK2 inhibitors would provide additional benefit to CDK4/6i. Addition of CDK2i was able to fully restore efficacy in p53KO lines as well as a patient derived model with TP53 mutation enabling irreversible cell cycle arrest and long-term growth suppression. Conclusion: Wild type p53 facilitates DREAM complex assembly in response to CDK4/6i in ER+ breast cancer, underlying long-term disease control. p53 loss enables cells to reenter the cell cycle and facilitates the development of CDK4/6i resistance. Combined inhibition of CDK2 with CDK4/6 represents a promising therapeutic strategy for overcoming loss of p53 and enabling long-term antitumor efficacy.
 Citation Format: Rei Kudo, Anton Safonov, Enrico Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, Edaise M. da Silva, Qing Li, Marie Will, Atsushi Fushimi, Harikrishna Nakshatri, Andrew Koff, Britta Weigelt, Qamar J. Khan, Pedram Razavi, Sarat Chandarlapaty. DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c291e911332efcfe859d324b671af4bba597c2" target='_blank'>
              Abstract PS13-04: DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition
              </a>
            </td>
          <td>
            Rei Kudo, A. Safonov, E. Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, E. D. da Silva, Qing Li, Marie Will, A. Fushimi, H. Nakshatri, Andrew Koff, B. Weigelt, Qamar J Khan, P. Razavi, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="The rarity of recurrent somatic mutations poses a challenge for the targeted treatment of neuroblastoma (NB). Differentiation therapy is an encouraging prospect, with cyclin-dependent kinase inhibitors (CDKis) representing a promising avenue for promoting NB differentiation. This study investigated three CDKis (abemaciclib, fadraciclib, and dinaciclib) alone or combined with retinoic acid (RA) to assess the effects on morphology, growth, gene expression, and the induction of immunogenic cell death in NB cell lines with (LAN-1 and CHLA-90) and without (CHLA-172) MYCN amplification. All cell lines demonstrated sensitivity to CDK inhibition. Notably, low-dose abemaciclib promoted cellular differentiation, as evidenced by the emergence of stromal-like morphological features and upregulation of the differentiation markers STMN4 and ROBO2. Treatment with abemaciclib or fadraciclib led to the upregulation of calnexin and holocytochrome C, which are part of the global stress response, along with the protein p27, which arrests the cell cycle. Molecularly, CDKis sensitivity correlated with an increased CDK4-specific copy number, along with a partial deletion of CDKN2a in two cases (LAN-1, CHLA-172). The addition of RA augmented the effects of the monotherapy, particularly in LAN-1 cells, in both 2D and 3D culture, and both treatments triggered immunogenic cell death, evidenced by calreticulin translocation. Transcriptomic analysis of LAN-1 and CHLA-90 cells revealed that genes deregulated by monotherapy (fadraciclib or RA) were re-regulated in the presence of the second drug. Combination therapy significantly downregulated CRABP2 and CYP26B1, both of which are involved in RA metabolism and its degradation. Furthermore, CCNE2, MYBL2, and MCM4 were strongly suppressed in the fadraciclib/RA combination, confirming the induction of cell cycle arrest. CDKi treatments promote NB differentiation via ER stress, with cytotoxicity enhanced by RA co-treatment. This may increase NB immunogenicity and support immunotherapy eligibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85ca193304bf6f6433a4c52b2f5a75f164abeec5" target='_blank'>
              CDK inhibitors promote neuroblastoma cell differentiation and increase sensitivity to retinoic acid—a promising combination strategy for therapeutic intervention
              </a>
            </td>
          <td>
            Fatemeh Shokraie, Larissa Lechermeier, Pia Bordihn, Philipp Kaps, Steffen Möller, Anna Sophie Schulz, Björn Schneider, Dirk Koczan, Samira Khanipour Roshan, Holger N Lode, C. Classen, Olga Hahn, S. Troschke-Meurer, Claudia Maletzki
          </td>
          <td>2025-08-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8030ad41247de8846ce913550c0870c5e34232e" target='_blank'>
              Colon cancer cells evade drug action by enhancing drug metabolism.
              </a>
            </td>
          <td>
            Bojie Cong, Teena Thakur, Alejandro Huerta Uribe, Evangelia Stamou, Sindhura Gopinath, Owen Sansom, Oliver D. K. Maddocks, Ross L Cagan
          </td>
          <td>2025-07-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Fanconi anemia (FA) is a DNA repair deficiency disorder associated with genomic and chromosomal instability and a high cancer risk. In a small percentage of cases, FA is caused by biallelic pathogenic variants (PVs) in the BRCA2/FANCD1 gene, defining the FA-D1 subtype. Experimental and epidemiologic data indicate that the complete absence of BRCA2 is incompatible with viability. Therefore, cells from individuals affected with FA caused by biallelic BRCA2 PVs must have a residual BRCA2 function. This activity may be maintained through hypomorphic missense mutations, translation termination–reinitiation associated with a translational stop mutation, or other non-canonical or uncommon translation initiation and elongation events. In some cases, however, residual BRCA2 function is provided by alternatively or aberrantly spliced BRCA2 transcripts. Here, we review and debate aspects of the contribution of splicing in the 5′ segment to BRCA2 functions in the context of PVs affecting this largely intrinsically disordered protein region, with a focus on recent findings in individuals with FA-D1. In this Perspective, we also discuss some of the broader biological implications and open questions that arise from considering 5′-terminal BRCA2 splicing in light of old and new findings from FA-D1 patients and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea89691296a2efa5c1ed91fcd358d20e993d695b" target='_blank'>
              BRCA2 Pre-mRNA Differential 5′ Splicing: A Rescue of Functional Protein Properties from Pathogenic Gene Variants and a Lifeline for Fanconi Anemia D1 Patients
              </a>
            </td>
          <td>
            Roberto Paredes, Kiran Batta, Daniel H. Wiseman, Reham Gothbi, Vineet Dalal, Christine K. Schmidt, Reinhard Kalb, Stefan Meyer, Detlev Schindler
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, Natalia Y Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eee44887a88c522cc74478c5f67d3f1813be9b0" target='_blank'>
              DPYSL3B is a regulator of chemoresistance via DNA repair and metabolic reprogramming in prostate cancer
              </a>
            </td>
          <td>
            R. Kaarijärvi, H. Kaljunen, Nella Itkonen, Kseniia Bureiko, K. Ketola
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: SARS-CoV-2, which drove the 2019 COVID-19 pandemic, continues to engender inquiries into the role of host genetic factors in disease susceptibility. Despite the identification of over 1,000 genes potentially associated with SARS-CoV-2 and COVID-19, the mechanisms connecting genetic variants to phenotype remain elusive. To shed light on these mechanisms, we undertook an integrated analysis, merging data from whole-genome association analyses of COVID-19 with methylome and transcriptomic. Methods: Study includes African American adults from the GENE-FORECAST study, encompassing 371 individuals with whole genome sequencing (WGS), 203 with DNA methylation, and 321 with RNA sequencing (RNA-Seq) of blood. 53.3% of participants reported COVID-19. Significant loci associated with COVID-19 were examined within the framework of methylation quantitative trait loci (mQTL) which located near the gene-of-original (cis-mQTL) and expression quantitative trait loci (eQTL) which located near the gene-of-origin (cis-eQTL), enabling analysis to assess mediators between genetic variants and COVID-19 status. Results: Our analysis identified four intronic variants and confirmed a missense variant, rs1052067, in PMF1 associated with COVID-19. Causal mediation analysis revealed that the combination of genetic variants within PMF1, epigenomics, and transcriptomics mapped four pathways influencing COVID-19 status. These pathways include: rs9659072 -> DNAm at chr1:156285845 (annotated to TMEM79) -> ENSG00000198715:13 (annotated to GLMP); rs12083543 -> DNAm at chr1:155951748 (ARHGEF2) -> ENSG00000198715:13 (GLMP); rs1052067 -> DNAm at chr1:155951748 (ARHGEF2)-> ENSG00000198715:13 (GLMP); rs1543294 -> ENSG00000198715:13 (GLMP) -> DNAm at chr1:156077518 (MEX3A). Conclusions: Through integrated multi-omics analyses, we identified genetic variants whose effects on COVID-19 susceptibility are mediated by changes in DNA methylation and mRNA expression. These findings offer insights into potential mechanistic pathways that merit further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00c7bc8e6839adbc276042c4324afad16196de83" target='_blank'>
              Multi-omics analyses of the complex interplay between genetic variants, DNA methylation, and gene expression in COVID-19.
              </a>
            </td>
          <td>
            Guanjie Chen, Lisa DeRoo, Gabriel Goodney, A. Doumatey, Jie Zhou, A. Adeyemo, C. Rotimi, Amadou Gaye
          </td>
          <td>2025-07-01</td>
          <td>American journal of physiology. Heart and circulatory physiology</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Existing approaches to identifying cancer genes rely overwhelmingly on DNA sequencing data. Here, we introduce RVdriver, a computational tool that leverages paired bulk genomic and transcriptomic data to classify RNA variant allele frequencies (VAFs) of non-synonymous mutations relative to a synonymous mutation background. We analyze 7882 paired exomes and transcriptomes from 31 cancer types and identify novel, as well as known, cancer genes, complementing other DNA-based approaches. Furthermore, RNA VAFs of individual mutations are able to distinguish “driver” from “passenger” mutations within established cancer genes. This approach highlights the value of multi-omic approaches for cancer gene discovery. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03557-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fff6bf27033e3de68705c3b36b17d1f26265844a" target='_blank'>
              Cancer gene identification from RNA variant allelic frequencies using RVdriver
              </a>
            </td>
          <td>
            James R. M. Black, Thomas P. Jones, Carlos Martínez-Ruiz, Maria Litovchenko, C. Puttick, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-06-13</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background The APOBEC (apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like) family plays a vital mutagenic role in diverse human malignancies. Nevertheless, the biological characteristics of APOBEC family members and their clinical significance in cancer have not been comprehensively explored. Our primary objective was to characterize the distribution and clinical relevance of APOBEC family members and their mutations across multiple cancer types, with a particular focus on thyroid carcinoma (THCA). Methods In this study, we employed an integrated, multi-faceted approach combining diverse statistical and bioinformatics methods. Data sources included whole-exome sequencing (WES), targeted next-generation sequencing (NGS), bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), Western blot assays, drug sensitivity analyses, and in vivo animal models. Results APOBEC mutations occupy a significant position in the mutation landscape of THCA. High APOBEC mutation enrichment scores were strongly associated with poor prognosis, immune evasion, and increased risk of malignant progression in THCA patients. These mutations contribute to tumor advancement and influence cellular differentiation within the tumor microenvironment. Additionally, we developed a prognostic APOBEC mutagenesis model using machine learning, which was validated across multiple THCA cohorts. In vitro and in vivo experiments demonstrated that APOBEC2 inhibition effectively suppressed tumor proliferation, metastasis, and glycolytic activity, while simultaneously enhancing immune activation and boosting the efficacy of immune checkpoint inhibitors. Conclusion Our study systematically characterizes APOBEC mutations across various cancer types and underscores their promise as biomarkers for aggressive tumor phenotypes, prognostic assessment, and immunotherapy response to THCA. These findings were accomplished through multi-omics sequencing and validated through comprehensive in vitro and in vivo experiments. Supplementary Information The online version contains supplementary material available at 10.1186/s12575-025-00288-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95761133ff7f07fa29822ab7f50151c4ca2027f9" target='_blank'>
              Comprehensive Insights into APOBEC Mutations in Thyroid Cancer: Prognostic and Therapeutic Discoveries
              </a>
            </td>
          <td>
            Wei Luo, Feng Liu, Mengyu Li, Jialong Yu, Ziyun Liu, Xuan Cheng, Yue Huang, Yu Liu, Mei Tao, Yuqi Wang, Yiping Zou, Xiaobin Shang, Chao Yang, X. Ruan, Yanchao Qin, X. Zheng
          </td>
          <td>2025-07-26</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, D. Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da4d10721badd9a65f0ec25fedb84c3d3183aaf" target='_blank'>
              Direct functional HOXA9/DNA-binding competitors versus epigenetic inhibitors of HOXA9 expression on cell proliferation, death and differentiation processes in the model of MLL-rearranged acute myeloid leukemia.
              </a>
            </td>
          <td>
            Julie Vrevin, Mélanie Lambert, Marine Andrique, Nathalie Jouy, M. David-Cordonnier
          </td>
          <td>2025-08-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract To date, most studies explored changes in 3D-genome organization between different tissues or during differentiation, which involve massive reprogramming of transcriptional programs. Much fewer studies examined alterations in genome organization in response to cellular stress, which involves less pervasive transcriptional modulation. Here, we examined associations between spatial chromatin organization and gene expression in two different biological contexts: transcriptional programs determining cell identity and transcriptional responses to stress, using p53 activation as a model. We selected 10 cell lines of diverse tissues, and in each performed micro-C, RNA-seq, and p53 ChIP-seq, before and after p53 induction. In the comparison between cell types, we delineated marked correlations between gene expression and spatial genome organization and identified hundreds of active enhancer–promoter loops associated with the expression of cell-type marker genes. In contrast, within each cell type, no such links were observed for expression changes induced by p53 activation, even for enhancers and promoters activated by p53 binding. Our analysis points to a fundamental difference between chromatin interactions that define cell identity and those that are established in response to cellular stress. Our results on p53-induced transcriptional responses support the recently proposed TF activity gradient model, which speculated a contact-independent mechanism for enhancer–promoter communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a93dd5bc7bcee25d57244f60e3d9ea722c2f779" target='_blank'>
              Transcriptional programs of cell identity and p53-induced stress responses are associated with distinctive features of spatial genome organization
              </a>
            </td>
          <td>
            Gony Shanel, Tsung-Han S Hsieh, Claudia Cattoglio, Hadar Amira Haham, Hsin-Jung Chou, Jack Z Li, R. Shamir, Xavier Darzacq, Ran Elkon
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Several mechanisms are involved in the resistance to endocrine therapy (ET) in estrogen receptor (ERα)-positive breast cancer (BC), including acquired mutations of ERα gene (ESR1). For example, the frequent mutation, Y537S, was shown to trigger a constitutively active receptor leading to reduced affinity for both agonist and antagonist ligands. The development of more comprehensive therapies remains a challenge in BC patients exhibiting activating mutations in ERα. Here, we show that Poly (ADP-ribose) polymerase-1 (PARP-1) may be considered as a novel therapeutic target in ERα-positive BC. Methods ERα wild type or Y537S mutated MCF7 and T47D BC cell lines were used as model systems. Immunoblotting, immunofluorescence, gene silencing, real-time PCR, promoter assays, chromatin immunoprecipitation sequencing (ChIP-seq) as well as cell viability, colony and cell cycle assays served to investigate the involvement of PARP-1 in BC progression. The growth of MCF7 ERα Y537S cells injected into the mammary ducts of NSG mice and treated with the ERα antagonist lasofoxifene or the PARP-1 inhibitor niraparib was monitored by luminescence imaging, weight measurement, and histological analysis. RNA sequencing studies were performed on the above-described xenograft tumors. METABRIC dataset was used to evaluate the clinical significance of PARP-1 and the biological role of the PARP-1-associated genes in ERα-positive BC patients. Results We first demonstrated that the up-regulation of PARP-1 expression induced by estrogens is abrogated either by inhibiting or silencing ERα in MCF7 and T47D BC cells expressing ERα wild type or Y537S mutation. We then showed that PARP-1 is involved in the binding of ERα and its co-activator FoxA1 to the promoters of several target genes, as determined by ChIP-sequencing studies. Of note, the inhibition of PARP-1 prevented the proliferative effects mediated by ERα in BC cells expressing either wild type or Y537S ERα. In accordance with these findings, the growth of xenograft tumors derived from MCF7 ERα Y537S BC cells was significantly reduced using niraparib and lasofoxifene. Finally, RNA-sequencing analyses showed that ERα signaling is downregulated by niraparib compared to vehicle-treated tumors. Conclusions Overall, our results suggest that PARP-1 should be explored as a potential target in comprehensive therapeutic approaches in ET-resistant BC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03441-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ed7e206cb1e21fd30b72a077e4d0a8d37cb618" target='_blank'>
              PARP-1 as a novel target in endocrine-resistant breast cancer
              </a>
            </td>
          <td>
            Azzurra Zicarelli, M. Talia, Muriel Lainé, R. Lappano, M. Maggiolini, G. Greene
          </td>
          <td>2025-06-16</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The anaplastic lymphoma kinase (alk) gene on chromosome 2 encodes a receptor tyrosine kinase protein essential for key signaling pathways regulating cell proliferation and differentiation. Mutations in alk have been implicated in multiple cancers, including non-small cell lung cancer (NSCLC) and anaplastic large cell lymphoma. While ALK inhibitors have demonstrated efficacy in targeted therapies, resistance due to specific amino acid substitutions requires the development of novel therapeutic strategies. This study aims to identify ALK tyrosine kinase domain mutations using data from the Cancer Genome Atlas and to evaluate the potential of lorlatinib, a third-generation ALK inhibitor, in overcoming these mutations. Using the SIFT and Polyphen-2 algorithms, we identified 53 deleterious ALK mutations associated with different newly recognized cancer types. These mutations were subjected to in silico molecular docking with lorlatinib. Our results indicate strong binding affinities (ranging from −9.4 to −10.8 kcal/mol) across all identified mutations, suggesting a significant interaction between lorlatinib and mutated ALK variants. Furthermore, protein-ligand interaction analysis revealed critical hydrophobic interactions, hydrogen bonds, and essential halogen bonds reinforcing lorlatinib as a potential utility in treating a broader spectrum of ALK-positive tumors beyond NSCLC. This research underscores the importance of repurposing in silico drugs and highlights the need for continued exploration of ALK mutations in cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c457f57fc03ab55c914622583804d14a03ad5d9e" target='_blank'>
              An in silico evaluation of lorlatinib as a potential therapy for novel amino acid substitutions in the tyrosine kinase domain of the ALK protein associated with cancer
              </a>
            </td>
          <td>
            Richard Junior Zapata Dongo, Julio A Poterico, D. Fontana, L. Mologni, Carla Alvarez-Chacon, Juan Rojas-Armas, Jaeson Calla
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Infants diagnosed with B cell acute lymphoblastic leukemia (B-ALL) and t(4;11) chromosomal rearrangement display poor therapeutic response, associated to the low expression of B lymphocyte factor HDAC7. This study was conceived to identify a therapeutic strategy for t(4;11) B-ALL that restores optimal HDAC7 expression. Methods A multiomics approach in a large infant pro-B-ALL cohort was employed to identify HDAC7’s repression mechanism. These data, combined with cell culture assays in a variety of pro-B-ALL cell lines with differential HDAC7 levels, led us to define a novel combination therapy. Murine leukemia models and ex vivo assays using patient-derived xenografts (PDX) were employed to assess the benefits of this therapy when incorporated to glucocorticoid-based chemotherapy. Results Our data demonstrates that HDAC7 is epigenetically silenced by EZH2 and KMT2A::AFF1 fusion protein. Remarkably, the Menin-1 inhibitor MI-538 restores HDAC7 expression, and the effect is enhanced by class I HDAC inhibitor chidamide. This treatment drives leukemic pro-B cells towards a more differentiated state and impairs aberrant proliferation in an HDAC7-dependent manner. This newly identified therapy increases glucocorticoid sensitivity of PDX cells ex vivo, by repressing RUNX2 transcription factor. Finally, combining MI-538 and chidamide with standard chemotherapy reduces PDX cells engraftment in vivo and delays relapse. Conclusions The combined therapy proposed, based on Menin-1 inhibition, improves t(4;11) B-ALL cells’ response to standard therapy, an effect partially mediated by HDAC7 induction. Therefore, this novel therapy opens a new field for personalized treatments in high-risk leukemia, especially for infants presenting low expression of HDAC7 B cell factor. Supplementary Information The online version contains supplementary material available at 10.1186/s40364-025-00810-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cbcee9d6f996ab0d44da486acb7d87805ed9357" target='_blank'>
              HDAC7 induction combined with standard-of-care chemotherapy provides a therapeutic advantage in t(4;11) infant B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            O. de Barrios, Ingrid Ocón-Gabarró, M. Gusi-Vives, O. Collazo, A. Meler, P. A. Romecín, A. Martínez-Moreno, J. R. Tejedor, Mario F Fraga, P. Schneider, M. Bardini, G. Cazzaniga, Rolf Marschalek, Ronald W W Stam, Clara Bueno, Pablo Menéndez, Maribel Parra
          </td>
          <td>2025-07-28</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a leading example of a malignancy where a molecular targeted therapy revolutionized treatment but has rarely led to cures. Overcoming tyrosine kinase inhibitor (TKI) drug resistance remains a challenge in the treatment of CML. We have recently identified miR-185 as a predictive biomarker where reduced expression in CD34+ treatment-naïve CML cells was associated with TKI resistance. We have also identified PAK6 as a target gene of miR-185 that was upregulated in CD34+ TKI-nonresponder cells. However, its role in regulating TKI resistance remains largely unknown. In this study, we specifically targeted PAK6 in imatinib (IM)-resistant cells and CD34+ stem/progenitor cells from IM-nonresponders using a lentiviral-mediated PAK6 knockdown strategy. Interestingly, the genetic and pharmacological suppression of PAK6 significantly reduced proliferation and increased apoptosis in TKI-resistant cells. Cell survivability was further diminished when IM was combined with PAK6 knockdown. Importantly, PAK6 inhibition in TKI-resistant cells induced cell cycle arrest in the G2-M phase and cellular senescence, accompanied by increased levels of DNA damage-associated senescence markers. Mechanically, we identified a PAK6-mediated MDM2-p21 axis that regulates cell cycle arrest and senescence. Thus, PAK6 plays a critical role in determining alternative cell fates in leukemic cells, and targeting PAK6 may offer a therapeutic strategy to selectively eradicate TKI-resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d13c94ff00c650649c31950307bc9a23bc8c504" target='_blank'>
              Identification of a PAK6-Mediated MDM2/p21 Axis That Modulates Survival and Cell Cycle Control of Drug-Resistant Stem/Progenitor Cells in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            Andrew Wu, Min Chen, Athena Phoa, Zesong Yang, Donna L. Forrest, Xiaoyan Jiang
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer is a growing concern around the world with more than 20 million new cases confirmed and counting. This is partly caused by the overuse of Chemotherapeutic drugs against Cancer, leading to its potency. Cancer starts with an accumulation of genetic mutations in the genome, and the ignorance to cell checks by CD8 and NK cells. The Cancer cell utilizes its hallmark capabilities and uses one of its cells to metastasizes in other areas of the body either through the perineural, circulatory, or lymphatic system. p53 is a tumor suppressor gene which repairs the mutations of the cell by conducting cell cycle arrest and inducing apoptosis. However, cancer cells can increase the expression of SB100 which can inhibit p53. One solution is to combine PRIMA-1 and CDDP which restores p53 and it’s wild type functions to induce apoptosis. This effect is increased with the use of adenoviral DN-Akt since it could increase p53 phosphorylation at Ser15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928e7ed5dc7a58f8574294e13f4f74e2d872a648" target='_blank'>
              Evolving Resistance: The Interplay of p53, Chemoresistance, and Metastasis in Cancer's Journey
              </a>
            </td>
          <td>
            Zenith Kwong
          </td>
          <td>2025-06-17</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cancer remains a significant global health threat, characterized by high morbidity and mortality rates. Despite advancements, numerous challenges persist in cancer treatment, underscoring the urgent need for novel therapeutic strategies. As a burgeoning area of cancer research, increasing evidence highlights the critical role of 5-methylcytosine (m5C) methylation in tumorigenesis, progression, and clinical application. In this review, we systematically review the biological mechanism and clinical translational potential of m5C RNA methylation in cancer and analyze in detail the dynamic interactions between m5C regulatory proteins (writer proteins, eraser proteins, and reader proteins). We focus on the role of m5C regulatory proteins in regulating oncogenic signaling pathways, metabolic reprogramming, and immune microenvironment remodeling in different tumor types, while also exploring its clinical potential as a diagnostic marker, prognostic indicator, and target for therapy. This review not only offers a comprehensive analysis of the molecular mechanisms and tumor regulatory networks associated with m5C modification but also focuses on the clinical translational potential of m5C methylation in cancer, aiming to provide novel insights into precision cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07aa242baf6069f9d71bd233815a9263406948fe" target='_blank'>
              m5C RNA methylation in cancer: from biological mechanism to clinical perspectives
              </a>
            </td>
          <td>
            Fang Yuan, Yuxuan Xia, Zhen Huang, Huihui Gao, Yuqi Bian, Xiangqin Zhou
          </td>
          <td>2025-06-21</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The development of multiple myeloma is typically associated with various cytogenetic abnormalities; however, these genetic changes alone do not fully account for the observed heterogeneity in patient prognosis and treatment response. Recent studies leveraging next-generation sequencing and genomic approaches have shown that epigenetic alterations are crucial in myeloma development and therapeutic resistance. These changes contribute to high levels of transcriptomic instability and enables cellular adaptation to targeted therapies and immunotherapies through diverse evolutionary trajectories. In this regard, aberrations of histone modifications and chromatin remodeling affect various cellular processes such as DNA repair, DNA damage response, cellular survival, and apoptosis signaling, which provides a strong rationale for developing epigenetic-targeted therapies for myeloma treatment. In this review, we focus on recent advances and research gaps in understanding the deregulation of histone acetylation, a widespread and versatile process of histone modification occurring at lysine residues at the N-terminus of histone tails, and its intimate interplay with chromatin remodeling complexes in orchestrating dynamic chromatin functional states and transcriptional outputs. We also provide an updated review of epigenetic modulatory drugs targeting HDAC, CBP/p300, and bromodomain and extraterminal proteins, along with a discussion of their limitations and future perspectives in myeloma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d100926ef56ad097039cb56ac307d5417be61de8" target='_blank'>
              Deciphering the dynamics of histone acetylation and chromatin remodeling in multiple myeloma: a tale beyond the tails.
              </a>
            </td>
          <td>
            Sinan Xiong, Jianbiao Zhou, WeeJoo Chng
          </td>
          <td>2025-07-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we develop a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals from 10 localities in Europe and Africa. We identify 6957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter chromatin annotations, as expected. The expression of 58% of these regulatory elements is modulated by methylation, which is generally associated with decreased transcription. Within our set of regulatory elements, we use allele-specific expression analyses to identify 8020 sites with genetic effects on gene regulation; further, we find that 42.3% of these genetic effects vary in direction or magnitude between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects are enriched for GWAS and EWAS annotations, implicating them in human disease. Compared with data sets that assay DNA from a single European ancestry individual, our multiplexed assay is able to uncover more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse genomes in assays that aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8c0aba589430fda6268dbe62323f10c40bdeec" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2025-07-14</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8601188410654744947f6690297e3eb471fcec4a" target='_blank'>
              Selective silencing of antibiotic-tethered ribosomes as a resistance mechanism against aminoglycosides
              </a>
            </td>
          <td>
            Nilanjan Ghosh Dastidar, Nicola S. Freyer, V. Petrychenko, A. C. Schwarzer, B. Peng, E. Samatova, C. Kothe, Marlen Schmidt, F. Peske, Antonio Z. Politi, H. Urlaub, Niels Fischer, Marina V. Rodnina, Ingo Wohlgemuth
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="During carcinogenesis, cells must acquire a telomere maintenance mechanism in order to avoid telomere shortening-induced replicative senescence. While most tumors activate telomerase, a minority of them employ a recombinational mechanism called Alternative Lengthening of Telomeres (ALT). One of the most investigated features is the association between ALT and ATRX mutations, since this has been shown to be the gene with the highest rate of mutations among ALT tumors. However, most of these studies, and in particular, mechanistic studies in vitro, have been carried out on mesenchymal tumors (sarcomas). In the present study, using genomic and expression data from the DepMap portal, we identified several non-mesenchymal ALT cell lines, and we compared the incidence of ATRX and other gene mutations between ALT cell lines of different origins (mesenchymal, neural, epithelial, hematopoietic). We confirmed that ATRX is frequently mutated in mesenchymal and neural ALT cell lines but not in epithelial ones. Our results showed that mutations of ATRX or other proteins involved in the maintenance of telomere integrity are needed for ALT activation in all cell types, and ATRX is preferentially mutated in mesenchymal ALT cells. Besides a more precise interpretation of the role of ATRX loss in ALT establishment, we proposed a model in which mutation of this gene impairs differentiation in mesenchymal and neural cells (but not in epithelial ones). Therefore, we explained the high incidence of ATRX mutations in mesenchymal and neural tumors with the fact that they both trigger ALT and impair differentiation, thus promoting two steps at once in the process of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f454c9a5bcf29e99c79d4426a0a12fbbf069c0ed" target='_blank'>
              Alternative Lengthening of Telomeres: The Need for ATRX Mutations Is Lineage-Dependent
              </a>
            </td>
          <td>
            I. Udroiu, Antonella Sgura
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background T-cell suppression in patients with Acute myeloid leukemia (AML) limits tumor cell clearance. This study aimed to explore the role of T-cell senescence-related genes in AML progression using single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing (RNA-seq), and survival data of patients with AML in the TCGA database. Methods The Uniform Manifold Approximation and Projection (UMAP) algorithm was used to identify different cell clusters in the GSE116256, and differentially expressed genes (DEGs) in T-cells were identified using the FindAllMarkers analysis. GSE114868 was used to identify DEGs in AML and control samples. Both were crossed with the CellAge database to identify aging-related genes. Univariate and multivariate regression analyses were performed to screen prognostic genes using the AML Cohort in The Cancer Genome Atlas (TCGA) Database (TCGA-LAML), and risk models were constructed to identify high-risk and low-risk patients. Line graphs showing the survival of patients with AML were created based on the independent prognostic factors, and Receiver Operating Characteristic Curve (ROC) curves were used to calculate the predictive accuracy of the line graph. GSE71014 was used to validate the prognostic ability of the risk score model. Tumor immune infiltration analysis was used to compare differences in tumor immune microenvironments between high- and low-risk AML groups. Finally, the expression levels of prognostic genes were verified using polymerase chain reaction (RT-qPCR). Results 31 AMLDEGs associated with aging identified 4 prognostic genes (CALR, CDK6, HOXA9, and PARP1) by univariate, multivariate, and stepwise regression analyses with risk modeling The ROC curves suggested that the line graph based on the independent prognostic factors accurately predicted the 1-, 3-, and 5-year survival of patients with AML. Tumor immune infiltration analyses suggested significant differences in the tumor immune microenvironment between low- and high-risk groups. Prognostic genes showed strong binding activity to target drugs (IGF1R and ABT737). RT-qPCR verified that prognostic gene expression was consistent with the data prediction results. Conclusion CALR, CDK6, HOXA9, and PARP1 predicted disease progression and prognosis in patients with AML. Based on these, we developed and validated a new AML risk model with great potential for predicting patients’ prognosis and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84874cd54294efd5d73e09436e1f2629d12de384" target='_blank'>
              Integrated single-cell and bulk RNA dequencing to identify and validate prognostic genes related to T Cell senescence in acute myeloid leukemia
              </a>
            </td>
          <td>
            Mengyao Sha, Jun Chen, Haifeng Hou, Huaihui Dou, Yan Zhang
          </td>
          <td>2025-06-25</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 International efforts focused on discovery of biological targets in inflammatory breast cancer (IBC) have suggested that aberrant inter-cellular relationship instead of genomic aberrations is the key driver of IBC. To identify such IBC-specific aberrations, we generated single nucleus chromatin accessibility and transcriptome atlas of IBCs (9,628 nuclei from three donors) and compared IBC atlas with a similar atlas of the healthy breast (81,735 nuclei from 92 donors). We recently reported single nucleus atlas of breast tissues of healthy women of diverse genetic ancestry (Nature Medicine in press). In that report, we described markers that identify three major epithelial cell subtypes [Basal-myoepithelial (BM), luminal adaptive secretory precursor (LASP), luminal hormone sensing (LHS)], two endothelial cell subtypes, two adipocyte subtypes, fibroblasts, macrophages, and T cells of the healthy breast. We also showed that LHS cells are the likely cells-of-origin of Luminal A, Luminal B and HER2+ breast cancers. Unlike these breast cancer subtypes, IBC does not appear to have a cell-of-origin, as epithelial cells in IBCs shared gene expression pattern across BM, LASP and LHS cells. Similar results were obtained when comparison was restricted to ancestry-specific healthy breast atlas. However, individual gene level expression differences affecting specific signaling pathways were observed in epithelial cells of IBCs. LASP, and LHS cells of IBCs overexpressed GPR137C, a positive regulator of mTORC1 signaling, and HS6ST3, an enzyme required for heparan sulfate synthesis. LASP and LHS cells of IBCs displayed activation of mitotic and matrix metalloproteinase signaling, respectively. Endothelial cells of IBCs, which showed significant gene expression differences compared to endothelial cells of the healthy breast, displayed enhanced integrin cell surface interaction but loss of cell junction organization. Most critically, BM, LASP, LHS, and endothelial cells of IBCs compared to their counterparts in the healthy breast showed downregulation of TEX14, an inter-cellular bridge forming factor in germ cells and a regulator of mitosis. Downregulation of TEX14 expression in these cells was accompanied with changes in the chromatin accessibility patterns of this gene. These results reinforce the notion that defective inter-cellular communication is a hallmark of IBC and TEX14 expression levels may serve as a biomarker of this defect. We suggest that greater attention to vascular biology and vascular-epithelial cell communication has to be given for better understanding of IBC biology and therapeutic targeting.
 Citation Format: Harikrishna Nakshatri, Poornima Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu. Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-05-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4562fd16c140b8e77bf1f62e51f30b753052c4ad" target='_blank'>
              Abstract P3-05-23: Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer
              </a>
            </td>
          <td>
            H. Nakshatri, P. Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 Background: Breast cancer is the most frequently diagnosed cancer and the leading cause of cancer-related death among women worldwide. Triple negative breast cancer (TNBC) represents 15-20% of all incident breast cancers and is a highly aggressive subtype. Neo(adjuvant) chemotherapy (NAC) is standard for early stage TNBC; however, for those with residual breast cancer following NAC, recurrence is common and associated with few effective therapeutic approaches. Thus, identification of novel resistance mechanisms and alternative treatment strategies remain an unmet clinical need. Circular RNAs (circRNAs) are a newly identified class of largely noncoding RNA molecules with covalently closed circular structures. Recent studies suggest that circRNAs play crucial roles in regulating tumor development, progression and chemoresistance. However, the role of circRNAs in mediating chemotherapy resistance remains unknown. Methods: As a first step towards identifying circRNAs that participate in the development of chemoresistance, and to determine if targeting such circRNAs may be an efficacious strategy, doxorubicin-resistant (Doxo-R), paclitaxel-resistant (PTX-R), and double-resistant (DP-R) cell lines were generated from MDA-MB-231 cell lines. Human circRNA microarrays were utilized to profile the expression of approximately 14,000 known circRNAs in normal breast tissue, matched TNBC patient-derived xenografts (PDX) generated prior to and following NAC, and TNBC chemosensitive and chemoresistant cell lines. Top hits were validated using RT-PCR. Bioinformatic approaches were employed to identify circRNA-miRNA-mRNA networks regulated by differentially expressed circRNA transcripts. A reverse genetics approach was also employed using CRISPR to interrogate the contributions of over 4000 specific circRNA in driving the development of resistance to doxorubicin and paclitaxel. Results: circRNA microarray profiling identified 429 and 310 transcripts differentially expressed in doxorubicin and paclitaxel resistant cells, respectively, compared to parental chemosensitive cell lines (|FC| ≥ 1.5; p value < 0.05). Given that circRNAs can act as microRNA (miRNA) sponges, thereby inhibiting miRNA activity, we computationally identified circRNA-miRNA pairs, and assessed the abundance of mRNA transcripts predicted to be targets of paired miRNAs via RNA-seq. A differentially expressed circRNA-miRNA-mRNA network was further constructed. Dysregulated canonical pathways and gene ontology (GO) enrichment analysis were determined from these networks and revealed multiple GO term included negative regulation of miRNA transcription, miRNA and RNA metabolic process, and endoplasmic reticulum-associated degradation (ERAD) pathway. The CRISPR-Cas13 screen is ongoing at this time but results will be available prior to the meeting. Conclusions: We have identified cancer specific circRNAs in TNBC cell lines, PDX models and patient tumors in the context of chemotherapy sensitive and chemotherapy resistant disease. Specific biological processes, pathways and signaling cascades have been implicated in the down-stream consequences of dysregulated circRNAs and in the potential development of resistance. Our ongoing studies seek to identify specific circRNAs sufficient to drive resistance to chemotherapy. Outcomes of these studies highlight the potential of specific circRNAs to serve as predictive/prognostic biomarkers and novel therapeutic targets for advanced forms of TNBC.
 Citation Format: Xiyin Wang, Michael J. Emch, Xiaojia Tang, Esther P.B. Rodman, Liewei Wang, Judy C. Boughey, Matthew P. Goetz, Krishna R. Kalari, John R. Hawse. The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb284df6818a5ca75827ea02126a84e48b2886e9" target='_blank'>
              Abstract P3-02-22: The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance
              </a>
            </td>
          <td>
            Xiyin Wang, M. Emch, Xiaojia Tang, E. Rodman, Liewei Wang, J. Boughey, Matthew P. Goetz, K. Kalari, John R. Hawse
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Endocrine resistance is frequently encountered in the clinic through a variety of mechanisms such as NF1 loss that induce alternative survival pathways and suppress estrogen responsiveness. While targeting the induced pathways such as the RAS-MAPK pathway can have antitumor effects, it can also incur toxicities. To identify novel and potentially more specific therapeutic vulnerabilities in this context, we performed CRISPR/Cas9 screens in wildtype and NF1 knockout isogenic ER+ models and identified NR2F2, an orphan nuclear receptor, to be essential specifically in NF1 loss cells. Our in vitro and in vivo results revealed that NR2F2 is a critical regulator of the estrogen signaling pathway, largely playing a repressive role. We found that NF1 loss could upregulate NR2F2 expression level in ER+ breast cancer cells through the activation of the MAPK pathway. NR2F2 overexpression by itself conferred endocrine resistance, while genetic knockout and pharmacologic inhibition of NR2F2 sensitized ER+ breast cancer cells to endocrine therapies and attenuated resistance across multiple endocrine refractory tumor models including those driven by NF1 loss, ARID1A loss, and PTEN loss. These results reveal upregulation of NR2F2 to be a recurrent mechanism whereby hormone responsiveness is suppressed. At a mechanistic level, investigation into chromatin accessibility and transcription revealed that altered NR2F2 expression modulates genome-wide chromatin accessibility including ER-regulated loci and modulates ER transcriptional activity through its direct interactions with ER and its transcriptional coregulators. Collectively, our data unveil a critical mechanism whereby endocrine resistance occurs through the repression of estrogen responsiveness and highlight a new therapeutic approach to restore endocrine response through pharmacologic inhibition of NR2F2.
 Citation Format: Yanyan Cai, Peihua Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O’Connell, Emma Fink, Matthew M Miele, Zhuoning Li, Elisa De Stanchina, Emiliano Cocco, Pedram Razavi, Eneda Toska, Sean W Fanning, Guotai Xu, Anna A Sablina, Sarat Chandarlapaty. Targeting NR2F2 overcomes multiple forms of endocrine resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-03-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97507505295680ec94e45659684f5586171f75f5" target='_blank'>
              Abstract P2-03-17: Targeting NR2F2 overcomes multiple forms of endocrine resistance
              </a>
            </td>
          <td>
            Yanyan Cai, P. Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O'Connell, Emma C. Fink, Matthew M. Miele, Zhuoning Li, Elisa de Stanchina, Emiliano Cocco, P. Razavi, Eneda Toska, S. Fanning, Guotai Xu, A. Sablina, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5226b4a71c28c02d3cc3172715553144cf7200ae" target='_blank'>
              Inherited STAT3 mutation mediates familial autoimmune disease, leukemogenesis, and responses to immunomodulator
              </a>
            </td>
          <td>
            ∗. DaijingNie, Qian Zhan
          </td>
          <td>2025-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Warburg effect, a hallmark of cancer, entails a metabolic shift from oxidative phosphorylation to aerobic glycolysis and is tightly regulated by complex signaling pathways. Decoding the dynamic signaling pathways underlying oncogenic metabolic reprogramming remains a major analytical challenge. Here, we report a cascaded DNA logic gate strategy capable of profiling the miR-210/CYGB/p53 axis that regulates the Warburg effect in triple-negative breast cancer (TNBC) cells. By logically integrating an H2O2-responsive DNAzyme (Dz) module with an entropy-driven reaction (EDR), the INHIBIT-AND logic device simultaneously interrogates upstream (miR-210) and downstream (CYGB mRNA, H2O2) effectors, producing dual amplified fluorescent outputs in response to specific pathway states. This programmable system enables live-cell, logic-based decoding of the Warburg effect, distinguishing TNBC from other breast cancer subtypes and dynamically reflecting pathway modulation. Our approach establishes a generalizable framework for using molecular computation to analyze complex biological circuits, providing a powerful tool for precision pathway profiling and therapeutic strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1e118a5499a6460056978aa8415b46448de10aa" target='_blank'>
              Cascade DNA Logic Strategy for Profiling the MiRNA-210-Targeted Signaling Pathway in the Warburg Effect of Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Jingyuan Yu, Xiaojun Zhang, Hao Sun, Yan Du
          </td>
          <td>2025-07-01</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic modifications are heritable, reversible alterations that causally reshape chromatin architecture and thereby influence DNA repair without changing nucleotide sequence. DNA methylation, histone modifications and non-coding RNAs profoundly influence DNA repair mechanisms and genomic stability. Aberrant epigenetic patterns in cancer compromise DNA damage recognition and repair, therefore impairing homologous recombination (HR), non-homologous end joining (NHEJ), and base excision repair (BER) by suppressing key repair genes and lowering access to repair sites. Then it is dissected how loss-of-function mutations in Switch/Sucrose non-fermentable, imitation switch and CHD (Chromodomain helicase DNA-binding) chromatin-remodeling complexes impair nucleosome repositioning, preventing effective damage sensing and assembly of repair machinery. Non-coding RNAs contribute to epigenetic silencing at DNA break sites, exacerbating repair deficiencies. This review evaluates recent advances concerning epigenetic dysfunction and DNA repair impairment. It is also highlighted that nanoparticle-mediated delivery strategies are designed to overcome pharmacologic resistance. It is presented how epigenetic dysregulation of DNA repair can guide more effective and drug-resistant cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d922eca60c4fcba055a3683ed299a04149ee715f" target='_blank'>
              Epigenetic Dysregulation in Cancer: Implications for Gene Expression and DNA Repair-Associated Pathways
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Katarzyna Rekiel, Piotr Urbanowicz, Mateusz Mamala, Karolina Marczuk, Maria Wojtaszek, Marta Żywica, Eivina Radzevičiūtė-Valčiukė, V. Novickij, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality globally, posing significant treatment challenges, particularly in its metastatic form (mCRC). This review comprehensively examines the pivotal role of RAS mutations, specifically KRAS and NRAS, which are detected in approximately 40–45% of mCRC cases, and their impact on treatment decisions and patient outcomes. We assess the effectiveness of standard treatments within the RAS mutant population, highlighting the challenges and limitations these therapies face. Recent advancements in targeted therapies, particularly the focus on novel agents such as KRAS G12C inhibitors, including sotorasib and adagrasib, have shown promising efficacy in overcoming resistance to conventional treatments. Furthermore, this review discusses future directions, emphasizing the need for research into non-RAS targets to address the complexities of resistance mechanisms and improve therapeutic outcomes. This review aims to provide a detailed overview of the current treatments and innovative approaches, supporting the development of personalized management strategies for patients with mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e15cd6a754c3946344410b05543b29153cfa7" target='_blank'>
              RAS Mutations in Advanced Colorectal Cancer: Mechanisms, Clinical Implications, and Novel Therapeutic Approaches
              </a>
            </td>
          <td>
            O. Sütçüoğlu, H. Yıldırım, Elvina Almuradova, Damla Günenç, Suayib Yalcin
          </td>
          <td>2025-06-30</td>
          <td>Medicina</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Mitochondria are critical for cellular energy, and while large deletions in their genome (mtDNA) are linked to primary mitochondrial diseases, their significance in cancer is less understood. Given cancer’s metabolic nature, investigating mtDNA deletions in tumors at various stages could provide insights into disease origins and treatment responses. In this study, we analyzed 148 bone marrow samples from 129 pediatric patients with B-cell (B-ALL) and T-cell (T-ALL) acute lymphoblastic leukemia at diagnosis, remission, and relapse using long-range PCR, next-generation sequencing, and the Splice-Break2 pipeline. Both T-ALL and B-ALL exhibited significantly more mtDNA deletions than did the controls, with T-ALL showing a ~100-fold increase and B-ALL a ~15-fold increase. The T-ALL samples also exhibited larger deletions (median size > 2000 bp) and greater heterogeneity, suggesting increased mitochondrial instability. Clustering analysis revealed distinct deletion profiles between ALL subtypes and across disease stages. Notably, large clonal deletions were detected in some B-ALL remission samples, including one affecting up to 88% of mtDNA molecules, which points toward treatment-driven selection or toxicity. A multivariate analysis confirmed that disease type, timepoint, and WHO subtype significantly influenced mtDNA deletion metrics, while age and gender did not. These findings suggest that mtDNA deletion profiling could serve as a biomarker for pediatric ALL and may indicate mitochondrial toxicity contributing to late effects in survivors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4c62c1b9c4085a3dab116b06e29653419eb877" target='_blank'>
              Distinct Mitochondrial DNA Deletion Profiles in Pediatric B- and T-ALL During Diagnosis, Remission, and Relapse
              </a>
            </td>
          <td>
            Hesamedin Hakimjavadi, Elizabeth Eom, Eirini Christodoulou, Brooke E. Hjelm, A. A. Omidsalar, D. Ostrow, Jaclyn A. Biegel, Xiaowu Gai
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Continuous passaging of cancer cell lines can drive phenotypic and genotypic divergence, potentially compromising the reliability of such models. In this study, we show that two late passage strains (S1 and S2) of ovarian cancer cell line SKOV3, though authenticated via short tandem repeat (STR) profiling as identical, exhibit substantial differences in morphology, transcriptomic signatures, ability to form 3D cultures and chemotherapeutic responses. Notably, S1 formed compact 3D spheroids and exhibited enhanced epithelial-mesenchymal transition (EMT) pathway activity, whereas S2 displayed a more proliferative, MYC-driven phenotype with larger spheroid structures requiring higher seeding densities. Transcriptomic analysis revealed pathways associated with hypoxia, EMT, and angiogenesis in 3D culture, highlighting the complexity introduced by dimensionality in tumour modelling. Critically, S1 showed higher sensitivity to doxorubicin than S2 (IC50 of 0.12 µM vs. 1.28 µM, p= 0.0001), indicating how clonal evolution can confound drug-response assays. Ultimately, our findings suggest that while STR profiling remains essential for cell line authentication, functionally distinct subpopulations can arise and coexist within the same culture, and their isolation may reveal divergent phenotypes that compromise reproducibility in preclinical cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b19ad530ecaca77ba40f9ce705c8944938d900" target='_blank'>
              Extended passaging of SKOV3 cell line leads to two phenotypically different strains.
              </a>
            </td>
          <td>
            Eglė Žymantaitė, M. Gabrielaitė, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-08-04</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Although long interspersed element-1 (LINE-1) retrotransposons are poor prognostic indicators of non-small cell lung cancer, the genetic consequences of aberrant LINE-1 expression remain poorly understood. In this study, human bronchial epithelial cells were exposed chronically to 25 µM NiCl₂ to induce malignant transformation and compared to PBS-treated control cells or lung cancer cell lines. A 4 kb intronic LINE-1 insertion into the NACC2 (nucleus accumbens-associated protein 2) locus in nickel-transformed cells was associated with significant reductions in NACC2 mRNA and protein, along with constitutive increases in HDM2 and TP53α mRNAs, and aberrant expression of TP53β and TP53γ mRNAs. Steady-state levels of p53 and RB decreased, while EGFR and LINE-1 ORF1p increased. Silencing of NACC2 in TP53 wildtype NCI-H460 or TP53-null NCI-1299 cell lines replicated many of these changes, with profiles varying as a function of p53 status. Stable overexpression of HDM2 increased LINE-1 ORF1p in cancer cell lines. Human lung adenocarcinomas with wild-type TP53 showed sexually dimorphic profiles, with higher HDM2 levels in females than males, and shifts in LINE-1 ORF1p, p53, and RB that mirrored cancer cell lines. This study identifies the NACC2 locus as a target of LINE-1 retrotransposition and highlights critical interactions with the HDM2/TP53/RB regulatory axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adfffe6caad02c3b19f108090ea32e2ac145796" target='_blank'>
              LINE1 insertion into the NACC2 locus disrupts the HDM2 axis and activates lung oncogenic signaling
              </a>
            </td>
          <td>
            P. Bojang, Yingshan Wang, Kenneth S Ramos
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC), an aggressive and highly malignant subtype of castration-resistant prostate cancer (CRPC), arises through drug resistance mechanisms involving genomic alterations, epigenetic remodeling, tumor microenvironment (TME) reprogramming, and lineage plasticity. A hallmark of NEPC is its independence from the androgen receptor (AR) pathway, evidenced by diminished or absent AR expression—a key barrier to effective clinical management. Therefore, a comprehensive understanding of NEPC pathogenesis and progression is essential, as it facilitates the identification of potential therapeutic targets and the development of more effective therapies. In this review, we summarize the typical characteristics of NEPC and describe its clinical diagnosis, relevant imaging modalities, current treatment strategies, and associated therapeutic difficulties. We also highlight the core events driving NEPC formation. Furthermore, we discuss potential therapeutic targets for the disease and review pharmacological agents that have demonstrated efficacy against NEPC, aiming to offer innovative perspectives and potential research directions for future NEPC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58ac5d561d4b3dff4622ddbdfd3ef0e9f2868489" target='_blank'>
              New Insights into Potential Therapeutic Targets for Neuroendocrine Prostate Cancer: From Bench to Clinic
              </a>
            </td>
          <td>
            Maoping Cai, Fengtao Zheng, Yang-Zi Ren, Chuqian Zhen, Dujiang Fu, Xian-lu Song, Qing Li, Yuanyuan Qu, Zhe-Sheng Chen, Shan-Chao Zhao
          </td>
          <td>2025-06-29</td>
          <td>Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncolytic virotherapy is a rapidly evolving approach to cancer treatment. Our group previously designed VV-GMCSF-Lact, a recombinant oncolytic vaccinia virus targeting solid tumors including gliomas. In this study, we used single-cell RNA sequencing to compare transcriptional responses in human glioma cells, non-malignant brain cells, and immortalized glioblastoma U87 MG cells following infection with this oncolytic virus. We found that proneural glioblastoma cells and microglia-like cells from patient-derived glioma cultures were the most susceptible to VV-GMCSF-Lact. Increased expressions of histones, translational regulators, and ribosomal proteins positively correlated with viral load at the transcript level. Furthermore, higher viral loads were accompanied by a large-scale downregulation of genes involved in mitochondrial translation, metabolism, and oxidative phosphorylation. Levels of early vaccinia virus transcripts are also positively correlated with infection intensity, suggesting that the fate of cells is determined at the early stage of infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd0ce99f16e9e13e522700a614d5b988909f5218" target='_blank'>
              Single-Cell Transcriptomic Changes in Patient-Derived Glioma and U87 Glioblastoma Cell Cultures Infected with the Oncolytic Virus VV-GMCSF-Lact
              </a>
            </td>
          <td>
            D. Semenov, N. Vasileva, M. Menyailo, S. Mishinov, Yulya I Savinovskaya, A. Ageenko, Anna S. Chesnokova, M. Dymova, G. Stepanov, G. Kochneva, Vladimir A. Richter, E. Kuligina
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background The three-prime repair exonuclease 1, TREX1, degrades cytosolic DNA to prevent aberrant immune activation. Its inactivation results in DNA accumulation in the cytosol and induction of the cGAS-STING DNA sensing pathway, interferon signaling, and inflammation. Germline pathogenic TREX1 mutations are known to lead to hereditary autoimmune and autoinflammatory disorders, whereas the consequences of TREX1 mutations in cancer remain poorly understood. Results To assess the importance of human TREX1 amino acid variants, we analyzed protein sequences of the functional TREX1b isoform from 168 mammalian species and integrated available data on TREX1 sequence and copy number alterations in hereditary autoimmune and autoinflammatory disorders, cancer, and in human populations. While the entire TREX1b protein was conserved in placental mammals, egg-laying mammals and marsupials had their own unique C-terminal regions, with each predicted to contain a transmembrane domain. We modeled human TREX1 variants occurring in autoimmune disease and cancer samples at 12 protein positions to evaluate their predicted impact on protein stability and function. Conclusions Our findings provide novel insight into the role of TREX1 molecular variation in cancer, where genetic or epigenetic loss of TREX1 activity may improve susceptibility to treatment. However, TREX1 gene deletion in tumors was associated with unfavorable patient outcomes, most likely due its frequent co-occurrence with the loss of the entire 3p chromosomal arm, which contains known cancer-related genes. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00785-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53a6c0f4b15cdd23d8488729c5a843f6b9a322d6" target='_blank'>
              Prevalence and impact of molecular variation in the three-prime repair exonuclease 1 TREX1 and its implications for oncology
              </a>
            </td>
          <td>
            Marwa Shekfeh, Mariam M. Konaté, Julia Krushkal
          </td>
          <td>2025-06-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The Mismatch Repair (MMR) pathway, a crucial tumor suppressor mechanism, is dysregulated across cancer types. We previously identified the loss of the MutL complex of the MMR pathway, as a key factor inducing resistance to endocrine therapy in Estrogen Receptor positive (ER+) breast cancer patients. Preliminary data previously generated in the lab supports a role for MutL loss in promoting metastasis of ER+ breast cancer. A mammary-specific knockout mouse model for MLH1, a principal component of MutL, created in the Haricharan lab develops aggressive mammary tumors that metastasize to multiple organs. To unveil the mechanism by which MLH1 loss triggers metastasis, an unbiased RPPA and RNA-Seq was conducted which revealed cytokine secretion in the immune system as a distinctive signature of MLH1- ER+ breast cancer cells. The tumor secretome, particularly cytokines, can manipulate macrophage plasticity to skew polarization states towards either an anti- or pro- metastatic phenotype. Secreted cytokine levels altered by MLH1 loss were identified in multiple ER+ breast cancer cells lines from concentrated conditioned media. Additionally, in vivo experiments with xenograft MLH1- ER+ tumors in nude mice, revealed an elevated number of CD206 expressing tumor-promoting M2 macrophages in the tumor microenvironment. Furthermore, MLH1- ER+ breast cancer cells grown in the presence of macrophages exhibit an increase in their migratory capacity. Our preliminary data strongly indicate that loss of MLH1 alters the ER+ breast cancer secretome and alters macrophage maturation, suggesting a role for MLH1 in immunomodulation supporting metastatic progression. Investigation of this new role will uncover key tumor-intrinsic drivers of “cold” tumor immune microenvironment profiles and present unique opportunities for therapeutic interventions to convert them into “hot” profiles, thereby improving ER+ breast cancer patient outcomes.
 Citation Format: Megha Raghunathan, Aloran Mazumder, Sabrina Carrell, Daniel Lozano, Nindo Punturi, Svasti Haricharan. MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-14.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f9ff414c291cb2d359a6c911b717a2e6733c88" target='_blank'>
              Abstract P1-07-14: MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity
              </a>
            </td>
          <td>
            M. Raghunathan, A. Mazumder, Sabrina Carrell, D. Lozano, Nindo B Punturi, S. Haricharan
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Purpose Bladder cancer is one of the ten most common cancers in the world, with a high incidence rate and mortality, and therefore a major burden on the global health care system. PID1 (Phosphotyrosine Interaction Domain 1) functions as an intracellular receptor protein for LRP1. The purpose of this study was to explore the role of PID1 in bladder cancer. Methods RNA-seq data analysis was conducted on 404 BLCA specimens and 28 normal specimens to identify differentially expressed genes. The findings indicated a strong correlation between PID1 expression levels and bladder cancer. We constructed a bladder cancer cell line stably overexpressing PID1 and assessed its impact on cell proliferation and migration. Additionally, We used RT-112 cells to induce tumor formation in nude mice to study the function of the PID1 gene in vivo. Results PID1 expression was notably low in bladder cancer tissues. Compared to SV-HUC-1, RT-112, and SCaBER bladder cells exhibited significantly reduced PID1 expression. Overexpressing PID1 in cells led to the promotion of apoptosis in bladder cancer cells and suppressed cell proliferation and metastasis. In vivo, the overexpression of PID1 demonstrated a significant inhibitory effect on bladder cancer. Furthermore, it was capable of activating the AMPK-mTOR signaling pathway, thereby inhibiting tumor progression. Conclusion PID1 exhibits a potent inhibitory effect on bladder cancer and activates the AMPK-mTOR signaling pathway to hinder tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2abd7d0b87a1062be291a6fc848727bf43e3bfb7" target='_blank'>
              As a New Tumor Suppressor Gene, PID1 Activates the AMPK-mTOR Signal to Inhibit the Progression of Bladder Cancer
              </a>
            </td>
          <td>
            Lingfeng Sun, Chengyi Liu, Yu Cao, Jianghao Li, Lin Yuan
          </td>
          <td>2025-07-01</td>
          <td>Research and Reports in Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Background: The molecular subtype largely governs therapeutic strategy in breast cancer patients, but the subtype is not static but frequently changes during treatment courses. The selective pressure on anti-tumor therapy is believed to drive lineage plasticity in luminal breast cancers, promoting a transition to basal-like phenotype and enhancing resistance against endocrine and targeted therapy. This study aims to elucidate the molecular mechanisms underlying luminal-to-basal (LB) subtype conversion through comprehensive genomic and molecular analyses. Methods: We utilized the prospective metastatic breast cancer database to identify patients with LB subtype conversion and luminal-to-luminal (LL) subtype maintenance in matched primary and secondary tumor tissues. Whole-exome sequencing (WES) and RNA-seq were performed on available tumors of the patients (exploratory cohort). We also validated genomic alterations in a separate cohort of patients who received next-generation sequencing (NGS) for clinical practice (validation cohort). Molecular factors associated with LB conversion were manipulated in MCF7 and T47D cell lines to further explore changes driving lineage plasticitiy, including chromatin accessibility studies using ATAC-seq. Results: Our analysis of metastatic breast cancer cohort with paired biopsy (n= 359) revealed overall 29.2% conversion rate of subtypes. LB conversion (from HR+HER2− to TNBC, 8.6%) was the most common subtype conversion and was associated with poorer survival compared to LL subtype maintenance. In the exploratory cohort, TP53 mutations were significantly more prevalent in patients undergoing LB conversion (64.3% vs. 15.4%), whereas GATA3 mutations appeared to be protective (0% vs. 23.1%). The validation NGS cohort (n=70) further confirmed the enrichment of TP53 mutation (85.7% vs. 23.2%) and the protective role of GATA3 (7.1% vs. 24.6%) in patients with LB conversion. The RNA-seq analysis indicated upregulation of KRAS-dependent signatures and decreased GATA3 expression in secondary tumors after LB conversion in TP53-mutant patients. TP53 depletion in MCF7 cells led to reduced ERα level and tamoxifen resistance, accompanied by increased accessibility in genes related to development regulation and AP-1 binding. Further combined TP53 and GATA3 depletion synergistically induced basal phenotype by activating the KRAS-MAPK pathway. MAPK inhibition restored luminal phenotype and tamoxifen sensitivity in luminal breast cancer against LB conversion induced by TP53 and GATA3 knockdown. MAPK inhibition was also effective in overcoming endocrine therapy resistance in tamoxifen- and palbociclib-resistant breast cancer cells exhibiting the LB conversion phenotype. Conclusion: Our findings establish TP53 mutations as a crucial predictor of LB subtype conversion in luminal breast cancers, with GATA3 mutations offering a protective effect. Combined TP53 and GATA3 loss synergistically promotes induction of basal phenotype through KRAS activation, and MAPK inhibition is a potential therapeutic strategy for luminal breast cancer patients with LB conversion.
 Citation Format: Min Hwan Kim, Won-Ji Ryu, Tae Yeong Kim, Yumi Hwang, Hyun-Yi Kim, Hyun Ju Han, Seul-Gi Kim, Gun Min Kim, Ja Seung Koo, Hyung Seok Park, Seung Il Kim, Joohyuk Sohn. Luminal-to-basal subtype conversion is governed by TP53-GATA3 mutational status and is targetable by MAPK inhibition in patients with breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-07.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0adb85d71d667a6b401b605d77a2d8d33145413e" target='_blank'>
              Abstract P5-06-07: Luminal-to-basal subtype conversion is governed by TP53-GATA3 mutational status and is targetable by MAPK inhibition in patients with breast cancer
              </a>
            </td>
          <td>
            Min Hwan Kim, W. Ryu, Tae Yeong Kim, Yumi Hwang, Hyun-Yi Kim, H. Han, Seul-Gi Kim, G. Kim, Ja Seung Koo, Hyung Seok Park, S. Kim, J. Sohn
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/805d87d60453c7f2f9127c01c1cd531181467286" target='_blank'>
              Potential therapeutic targets in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            Manvi Bansal, Malkhey Verma
          </td>
          <td>2025-07-17</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In recent years, the development of targeted therapies for tumors with KRAS mutations has progressed rapidly, rendering the notion of KRAS as “undruggable” outdated. However, targeted therapies for KRAS mutations still face numerous challenges, including resistance, efficacy concerns, toxicity issues, and hurdles in drug development. Exploring alternative treatment modalities is thus essential. Extensive research has demonstrated that KRAS mutations significantly influence the immune microenvironment, presenting both challenges and opportunities for immunotherapy. Interestingly, it has been observed that different KRAS mutations and co-mutation subtypes exhibit significant variations in their immunological microenvironments, which undoubtedly impact immunotherapy choices. Here, we review the history of KRAS-targeted therapy, highlighting existing challenges, and summarize changes in the immune microenvironment of KRAS-mutated cancers and their potential therapeutic targets. We compare differences in the immune microenvironment across various mutation types and co-mutation subtypes, and offer perspectives on future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993d360fa5818a3754af4b74cbddfac722d927a4" target='_blank'>
              Precision immune regulation in KRAS-mutated cancers: the final piece of the puzzle?
              </a>
            </td>
          <td>
            Shenao Fu, Jiayao Ma, Changjing Cai, Jun Tan, Xiangying Deng, Hong Shen, Shan Zeng, Yihong Chen, Ying Han
          </td>
          <td>2025-07-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background EpCAM (epithelial cell adhesion molecule) is a key regulator of epithelial cell–cell adhesion, signal transduction, tissue regeneration, and serves as a stem cell marker. It is frequently overexpressed in epithelial cancers and is linked to tumor progression, survival, and metastasis. However, the functional impact of EpCAM mutations in cancer remains poorly understood. Methods To investigate the role of EpCAM mutations, we performed a comprehensive analysis of cancer cohorts from multiple genomic datasets, identifying novel somatic EpCAM mutations across diverse epithelial cancers. Using bioinformatics tools (SIFT, PolyPhen-2, Mutation Assessor) and molecular modeling, we assessed the potential impact of these mutations. Further, homology modeling and all-atom molecular dynamics (MD) simulations were conducted to evaluate structural changes. From an analysis of 300 studies comprising 300,300 cancer samples, we identified 160 recurrent somatic mutations across epithelial cancers. Of these, seven mutations most frequently associated with lung cancer were further validated through molecular dynamics simulations, evaluation of ERK signaling activity, and assessment of sensitivity to the MEK inhibitor Trametinib. Results Our findings revealed that cancer-associated mutations, particularly in the TY-1 and RCD regions, induce structural instability in EpCAM, leading to altered functional properties. Patient cohort analyses indicated that EpCAM mutations correlate with reduced survival rates in colon and hepatocellular carcinoma and contribute to early tumor progression in lung cancer. Moreover, introducing these mutations into lung cancer cells enhanced their sensitivity to MEK inhibitors, suggesting a potential therapeutic vulnerability. Conclusion This study provides novel insights into the structural and functional consequences of EpCAM mutations in cancer, demonstrating their association with reduced survival, tumor progression, and drug sensitivity. These findings highlight EpCAM as a promising therapeutic target in epithelial cancers. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14455-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593d8c16237b699357d475f55f406f3effbf3755" target='_blank'>
              Landscape of cancer associated EpCAM mutations: molecular modeling, predictive insights and impact on patient survival
              </a>
            </td>
          <td>
            P. S. Dhotare, Audrey C. Bochi-Layec, Timothy P. Fleming, William E Gillanders, Ross M. Bremner, K. Sonawane, N. Sankpal
          </td>
          <td>2025-07-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="CRISPR activation and interference (CRISPRa/i) are highly effective tools to regulate transcription by fusing dead Cas9 (dCas9) with transcriptional regulatory factors guided by small guide RNA (sgRNA) in mammalian cells and mice. Still, a controllable gene regulation system is desired to investigate and manipulate dynamic biological processes. Here, we reported flexible drug-responsive CRISPRa/i systems by fusing mutated human estrogen receptor (ERT2) domains, which responded to estrogen analogue tamoxifen or its active metabolite 4-hydroxy-tamoxifen (4OHT), to CRISPRa/i components for transcriptional regulation. Upon 4OHT treatment, the optimal variants, ERT2-ERT2-CRISPRa/i-ERT2 (iCRISPRa/i), showed rapid protein translocation of iCRISPRa/i from cytoplasm to nucleus and subsequent transcriptional response. The inducible transcriptional manipulation could be restored to its original level when 4OHT was withdrawn. Moreover, the efficiencies of gene expression regulation of iCRISPRa/i were comparable to those of non-inducible and doxycycline-inducible counterparts, with a lower leakage and a faster drug response activity. The iCRISPRa/i systems successfully induced phenotypic changes in various cell lines. These results highlight that iCRISPRa/i systems could achieve fast and flexible drug-responsive transcriptional modulation and phenotypic changes, and thus provide better options for gain- and loss-of-function model construction and gene therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05786-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1338656a4c84c744345ea4f2ecdb230967ac2ab" target='_blank'>
              Novel drug-inducible CRISPRa/i systems for rapid and reversible manipulation of gene transcription
              </a>
            </td>
          <td>
            Ming Sui, Meiling Zhou, Mengge Cui, Huan Liu, Xiaolin Zhang, Na Hu, Yang Li, Beibei Wang, Guojun Yang, Pengling Gui, Lingqiang Zhu, Feng Wan, Bin Zhang
          </td>
          <td>2025-06-23</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Purpose RNF213 encodes one of the largest E3 ubiquitin ligases in the human proteome, playing essential roles in ubiquitination, DNA damage repair, and immune system regulation. Although RNF213 has been extensively studied in Moyamoya disease, its involvement in cancer biology remains insufficiently understood. This study seeks to conduct a comprehensive pan-cancer analysis to explore RNF213 expression, mutation characteristics, its correlation with clinical outcomes, and its potential as a prognostic biomarker and a therapeutic target, particularly in the context of immunotherapy. Methods Leveraging data from The Cancer Genome Atlas (TCGA) and additional publicly available datasets, we performed a systematic analysis of RNF213 expression patterns, mutation frequencies, and their relationships with clinical outcomes. We also conducted survival analyses to assess the impact of RNF213 expression on the efficacy of immune checkpoint blockade therapies. Results RNF213 demonstrated variable expression levels across different cancer types, with notable overexpression observed in glioblastoma and endometrial carcinoma, while underexpression was detected in lung adenocarcinoma and prostate adenocarcinoma. Elevated RNF213 expression was linked to unfavorable survival outcomes in acute myeloid leukemia and low-grade glioma, whereas it was associated with better survival in sarcoma and skin cutaneous melanoma. RNF213 expression showed a positive association with immune-related genes and immune cell infiltration, particularly in glioblastoma and breast cancer. Furthermore, patients with high RNF213 expression experienced significant survival advantages when receiving combined anti-PD-1 and anti-CTLA-4 immunotherapies. Conclusion This study highlights RNF213’s dual functionality in tumor progression and immune modulation, underscoring its potential as both a prognostic biomarker and a therapeutic target. The findings suggest that RNF213 may play a pivotal role in determining immunotherapy outcomes, warranting further exploration into its underlying mechanisms and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03237-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd17eb7e2de6f21c63c696e22b05947b1a82bc9b" target='_blank'>
              RNF213 governs divergent tumor-immune dynamics across human cancers: a prognostic biomarker for immunotherapy stratification
              </a>
            </td>
          <td>
            Jinpeng Wen, Zhekuan Lv, Mojiao Lu, Chuanming Zheng, Changtian Yin
          </td>
          <td>2025-07-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background MED12 is a key regulator of transcription and chromatin architecture, essential for normal hematopoiesis. While its dysregulation has been implicated in hematological malignancies, the mechanisms driving its upregulation in acute myeloid leukemia (AML) remain poorly understood. We investigated MED12 expression across AML subgroups by integrating chromatin accessibility profiling, histone modification landscapes, and DNA methylation (DNAm) patterns. Functional assays using DNMT inhibition were performed to dissect the underlying regulatory mechanisms. Results MED12 shows subtype-specific upregulation in AML compared to hematopoietic stem and progenitor cells, independent of somatic mutations. Chromatin accessibility profiling reveals that the MED12 locus is epigenetically primed in AML blasts, with increased DNase hypersensitivity at regulatory elements. Histone modification analysis demonstrates strong H3K4me3 and H3K27ac enrichment around the transcription start site (TSS), consistent with promoter activation, while upstream and intragenic regions exhibit enhancer-associated marks (H3K4me1, H3K27ac). Notably, hypermethylation within TSS-proximal regulatory regions (TPRRs)—including promoter-overlapping and adjacent CpG islands—correlates with ectopic MED12 overexpression, challenging the canonical view of DNAm as strictly repressive. Functional studies show that DNMT inhibition via 5-azacytidine reduces MED12 expression despite promoter demethylation in cells with hypermethylated TPRRs, suggesting a noncanonical role for DNA methylation in maintaining active transcription. Furthermore, MED12 expression positively correlates with DNMT3A and DNMT3B expression, implicating these methyltransferases in sustaining its epigenetic activation. Conclusion This study identifies a novel regulatory axis in which aberrant DNA methylation, rather than genetic mutation, drives MED12 upregulation in AML. Our findings suggest that TPRR hypermethylation may function noncanonically to support transcriptional activation, likely in cooperation with enhancer elements. These results underscore the importance of epigenetic mechanisms in AML and highlight enhancer-linked methylation as a potential contributor to oncogene dysregulation. Future studies should further explore the role of noncanonical methylation-mediated gene activation in AML pathogenesis and therapeutic targeting. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00610-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd4516155d5e48bceb4d470aa0e3620ac17928a" target='_blank'>
              Epigenetic regulation of MED12: a key contributor to the leukemic chromatin landscape and transcriptional dysregulation
              </a>
            </td>
          <td>
            Arundhati Chavan, Cassidy Jones, Whit Lawrence, S. R. Choudhury
          </td>
          <td>2025-07-14</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Glioblastoma multiforme (GBM) is a lethal brain tumor. With the current gold standard chemotherapy treatment, temozolomide (TMZ), many patients do not survive beyond one year. While the urgency of researching novel treatments is understandable, the prohibitively high costs and the prolonged duration of research and clinical trials significantly delay the availability of medical advancements to the general public. This highlights the urgent need for adjuvant therapies to enhance treatment effectiveness. Main body: Recent research has suggested the potential of repurposing FDA-approved drugs such as temozolomide (TMZ), disulfiram (DSF), and aspirin for the treatment of glioblastoma, with encouraging evidence particularly for DSF and aspirin. Additionally, compounds like histone deacetylase inhibitors (e.g., vorinostat) are being investigated for their impact on non-coding RNA (ncRNA) modulation, including microRNAs (miRNAs) and long non-coding RNAs (lncRNAs). Combining traditional therapies with ncRNA modulation has shown potential in enhancing therapeutic efficacy and targeting specificity. NcRNAs play a crucial role in regulating gene expression and have been implicated in tumor growth, invasion, and treatment resistance. Recent discoveries, such as cuproptosis, offer new insights into tumor cell death mechanisms. Conclusion This review focuses on how these molecular insights can serve as novel therapeutic targets and how drug adjuvant therapy may improve GBM treatment strategies. It focuses on how the integration of ncRNA modulation with conventional therapies and the combination strategy of enhancing efficacy of drugs can enhance treatment efficacy and pave the way for innovative approaches in managing GBM. In short, we will explore how non-coding RNAs (ncRNAs) might serve as promising targets and how repurposing TMZ, DSF, and aspirin could help enhance the efficacy of GBM treatment. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f928ba5f4bc7b8cba2127288e9078cdfc0a40a8" target='_blank'>
              A novel approach to enhance glioblastoma multiforme treatment efficacy: non-coding RNA targeted therapy and adjuvant approaches
              </a>
            </td>
          <td>
            Meijun Liu, Yuying Wang, Xiaoli Chen, Yu Zeng, Wenqiong Huang, Jiawen Yang, Hong Dai, Lixin Cheng, Claudio Mauro, Kenneth Cheung
          </td>
          <td>2025-06-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous loci linked to late-onset Alzheimer's disease (LOAD), but the pan-brain regional effects of these loci remain largely uncharacterized. To address this, we systematically analyzed all LOAD-associated regions reported by Bellenguez et al. using the FILER functional genomics catalog across 174 datasets, including enhancers, transcription factors, and quantitative trait loci. We identified 42 candidate causal variant-effector gene pairs and assessed their impact using enhancer-promoter interaction data, variant annotations, and brain cell-type-specific gene expression. Notably, the LOAD risk allele of rs74504435 at the SEC61G locus was computationally predicted to increase EGFR expression in LOAD related cell types: microglia, astrocytes, and neurons. Functional validation using promoter-focused Capture C, ATAC-seq, and CRISPR interference in the HMC3 human microglia cell line confirmed this regulatory relationship. Our findings reveal a microglial enhancer regulating EGFR in LOAD, suggesting EGFR inhibitors as a potential therapeutic avenue for the disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff28c54445a4cd1f4b5d1395aeed199bc5292bb6" target='_blank'>
              Integrated genomic analysis and CRISPRi implicates EGFR in Alzheimer’s disease risk
              </a>
            </td>
          <td>
            Y. Leung, Pavel P. Kuksa, Luke Carter, Jeffrey Cifello, Emily Greenfest-Allen, O. Valladares, Louisa Boateng, Shannon Laub, Natalia Tulina, Sofia Moura, Aura Maria Ramirez, K. Celis, Fulai Jin, Ru Feng, Gao Wang, P. D. De Jager, Jeff Vance, Liyong Wang, Struan F. A. Grant, G. Schellenberg, A. Chesi, Li-San Wang
          </td>
          <td>2025-06-26</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Proto-oncogenes in the RAS superfamily play dual roles in maintaining cellular homeostasis, such as regulating growth signals and contributing to cancer development through proliferation and deregulation. Activating proto-oncogenes in vitro transforms cells, underscoring their centrality in gene regulation and cellular networks. Despite decades of research, poor outcomes in advanced cancers reveal gaps in understanding Ras-driven mechanisms or therapeutic strategies. This narrative review examines RAS genes and Ras proteins in both housekeeping functions, such as cell growth, apoptosis, and protein trafficking, as well as in tumorigenesis, integrating insights from human (HRAS, KRAS, NRAS), mouse (Hras, Kras, Nras), and Drosophila melanogaster (ras) models. While RAS mutations are tightly linked to human tumors, the interplay between their standard and oncogenic functions remains complex. Even within the same tissue, distinct cancer pathways—such as the mitogen-activated protein kinase (MAPK) and phosphoinositide 3-kinase (PI3K) pathways—can drive varied disease courses, complicating treatment. Advanced-stage cancers add further challenges, including heterogeneity, protective microenvironments, drug resistance, and adaptive progression. This synthesis organizes current knowledge of RAS gene regulation and Ras protein function from genomic alterations and intracellular signaling to membrane dynamics and extracellular interactions, offering a layered perspective on the Ras pathway’s role in both housekeeping and tumorigenic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796354d7dffc674d220d0fc486fafe630423dfaf" target='_blank'>
              RESEARCH CHALLENGES IN STAGE III AND IV RAS-ASSOCIATED CANCERS: A Narrative Review of the Complexities and Functions of the Family of RAS Genes and Ras Proteins in Housekeeping and Tumorigenesis
              </a>
            </td>
          <td>
            Richard A. McDonald, A. Varela-Ramírez, Amanda K. Ashley
          </td>
          <td>2025-07-25</td>
          <td>Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND/AIM
While normal cells are highly regulated, cancer cells take a dysregulated path which bolsters their survival. Currently, a limited number of uniform treatments are available for cancer cure. Our goal was to deprive cancer cells of the key nutrient methionine and determine what effect it would have on cell death and alterations in DNA methylation of prostate cancer cells.


MATERIALS AND METHODS
PC3 and other cell lines were transfected with plasmid gene constructs for methionine gamma lyase deaminase (MEGL), a methionine-degrading enzyme, targeted for expression in the cytoplasm (cMEGL) or the nucleus (nMEGL). For assessing cell death due to MEGL-mediated methionine deprivation, a standard 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay was used. PC3, and DU145 prostate cancer cells were selected for whole-methylome sequencing to determine the effects of MEGL expression. Key gene products comprising the Prolaris Molecular Score, specifically 31 cell-cycle progression genes, were chosen for assessing putative differences in methylome.


RESULTS
Treatment with MEGL gene targeted for expression in either the cytoplasm or nucleus caused significant cell death, similar to that due to the anticancer drug methotrexate. Azacytidine showed no effect on PC3 cell death. Propargylglycine, an inhibitor of MEGL, prevented cell death. Methylome analysis showed increased methylation of two genes: Spindle and kinetochore-associated complex subunit 1 (SKA1), origin recognition complex subunit 6 (ORC6L), and reduced methylation of six promoters: BUB1 mitotic checkpoint serine/threonine kinase B (BUB1B), PDZ binding kinase (PBK), baculoviral IAP repeat-containing 5 (BIRC5), centromere protein M (CENPM), DNA topoisomerase II alpha (TOP2A), minichromosome maintenance 10 replication initiation factor (MCM10), upon forced expression of MEGL.


CONCLUSION
Methionine deprivation through MEGL-targeted gene therapy may be a viable option for inducing cancer cell death compared to unrestricted levels of methionine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4960faa090cc3a8364fc997a8a0c4944dac73e5" target='_blank'>
              Methionine Deprivation-induced Cancer Cell Death and Methylation Changes in Key Genes and Gene Promoters of Prostate Cancer Cell Lines.
              </a>
            </td>
          <td>
            Yatin Srinivash Ramesh Babu, K. Venkatachalam
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mutations in TP53 are mutually exclusive with other known drivers of myeloid transformation and define a distinct molecular subtype within de novo Acute Myeloid Leukemia (AML) that is associated with a complex karyotype, resistance to chemotherapy, and poor prognosis. Although TP53 defects are rare in de novo AML, biallelic mutations are a defining molecular feature of erythroleukemia. The genetic alterations that cooperate with defective TP53 to transform erythroid progenitors remain unknown. We found that loss of BAP1 (BRCA1 Associated Protein-1) co-occurs in one-third of patients with TP53-mutated AML, is associated with an erythroid-primed gene expression signature, and confers an additional adverse effect on overall survival. BAP1 is a tumor suppressor involved in the DNA damage response as well as epigenetic regulation through histone H2AK119 de-ubiquitination. While Bap1KO mice develop myelodysplasia with prominent dyserythropoiesis, combined deletion of Bap1 and Trp53 caused transplantable erythroleukemia, and occasionally mixed AML, mirroring the heterogeneity of human disease. Bulk and single-cell RNA-seq coupled to ChIP-seq in hematopoietic progenitors revealed that Bap1 loss triggers a proinflammatory response and cooperates with Trp53 deficiency to transform erythroid-primed multipotent progenitors. Mechanistically, genomic instability led to the development of erythroleukemia, while epigenetic deregulation caused myelomonocytic skewing suggesting a dichotomous and context dependent role for BAP1. We also demonstrate that BAP1 deficient erythroleukemia is dependent on BCL2L1 expression and is sensitive to BCL-xL inhibitors in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/544862deccf569b5878b747c297d81ba58bf024e" target='_blank'>
              Loss of BAP1 defines a unique subtype of TP53-mutated de novo AML and confers sensitivity to BCL-xL inhibitors.
              </a>
            </td>
          <td>
            Jaclyn Andricovich, Coen J. Lap, A. Tzatsos
          </td>
          <td>2025-06-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14aba9861491c1dd938896b3ff280930aaa89511" target='_blank'>
              HiFIseek: gene-specific enrichment of high-impact mutations in associated genomic regions
              </a>
            </td>
          <td>
            Rafael Riudavets Puig, I. S. Brorson, Denise G. O’Mahony, Vessela N. Kristensen, Anthony Mathelier
          </td>
          <td>2025-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oral squamous cell cancer (OSCC) is a major global health issue, ranking sixth in prevalence, particularly in Asia. The diagnosis often occurs late due to inadequate early screening, resulting in a dismal five-year survival rate of around 50%. This document provides a comprehensive analysis of drug-based treatments for oral cavity carcinoma, focusing on chemotherapy, immune modulation, and novel approaches like nanoparticle therapies. Despite advancements in these methods, drug resistance remains a significant obstacle that adversely affects patient outcomes. The research highlights the critical role of long ncRNAs in the progression and treatment of OSCC. These long ncRNAs, which are over 200 nucleotides long, play essential roles in gene regulation and tumor growth, including mechanisms of drug resistance. Some long ncRNAs may promote or inhibit tumor development and influence the effectiveness of anti-cancer drugs like cisplatin. Additionally, the review explores how the tumor microenvironment and immune responses interact, suggesting that inflammation may accelerate the progression of oral cancer. By synthesizing insights from extensive literature, this review clarifies the complex relationship between long ncRNAs and OSCC treatment. The study aimed to improve treatment efficacy and increase survival rates for patients with oral squamous cell carcinoma by identifying potential therapeutic targets. The findings underscore the importance of integrating molecular insights into treatment strategies to combat drug resistance and enhance patient outcomes in oral cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680f8ab3d2bbad8990d7480a9760079550d9fbe0" target='_blank'>
              Oral squamous cell carcinoma pharmacological treatment; A long non-coding RNAs (long ncRNAs) story
              </a>
            </td>
          <td>
            Azita Sadeghzade, Ali Jafarian, Amir Hossein Davodpour, Mohsen Pouresmaeliyan Roumani, Meysam Mohammadikhah, Sarah Qutaiba Badraldeen, Sajad Ataei Azimi
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular and Cellular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutations that activate the small GTPase KRAS are a frequent genetic alteration in cancer, and drug discovery efforts have led to inhibitors that block KRAS activity. We sought to better understand oncogenic KRAS signaling and the cytostatic effects of drugs that target this system. We performed proteomic analyses to investigate changes in protein abundance and posttranslational modifications in inhibitor-treated human KRAS-mutant pancreatic (KRAS G12C and G12D) and lung cancer (KRAS G12C) cells. The inhibitors used target these mutant forms of KRAS, the downstream effectors MEK and ERK, and the upstream regulators SHP2 and SOS1. Comparisons of phosphoproteomes between cell lines revealed a core KRAS signaling signature and cell line-specific signaling networks. In all cell lines, phosphoproteomes were dominated by different degrees of autonomous, oncogenic KRAS activity. Comparison of phosphoproteomes after short and long drug exposures revealed the temporal dynamics of KRAS-MEK-ERK axis inhibition that resulted in cell cycle exit. This transition to a quiescent state occurred in the absence of substantial proteome remodeling but included broad changes in protein phosphorylation and ubiquitylation. The collective data reveal insights into oncogenic KRAS signaling, place many additional proteins into this functional context, and implicate cell cycle exit as a mechanism by which cells evade death upon KRAS signaling inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2163ec601f10210697958516b4e39127c69ccee" target='_blank'>
              Proteomic analyses identify targets, pathways, and cellular consequences of oncogenic KRAS signaling.
              </a>
            </td>
          <td>
            Nicole Kabella, Florian P. Bayer, Konstantinos Stamatiou, Miriam Abele, Amirhossein Sakhteman, Yun-Chien Chang, Vinona Wagner, Antje Gabriel, Johannes Krumm, Maria Reinecke, Melanie Holzner, Michael Aigner, Matthew The, Hannes Hahne, Florian Bassermann, Christina Ludwig, Paola Vagnarelli, Bernhard Kuster
          </td>
          <td>2025-07-29</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary Gynecological cancers, such as endometrial, cervical, and ovarian cancer, are among the leading causes of death in women worldwide. Scientists are searching for new ways to detect these diseases earlier and to tailor treatment options better. To achieve this, changes in genes responsible for DNA repair are being analyzed. When these repair processes are weakened, DNA damage can accumulate, increasing the risk of cancer development. In our study, we reviewed the latest scientific publications to identify genetic variations that may be linked to the development of gynecological cancers. These findings may contribute in the future to the early detection of these diseases and to the development of therapies tailored to the individual genetic profiles of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e542d9860b43270a69e0d944b9a7034ae1638d99" target='_blank'>
              Genetic Polymorphisms in Base Excision Repair (BER) and Nucleotide Excision Repair (NER) Pathways as Potential Biomarkers for Gynecological Cancers: A Comprehensive Literature Review
              </a>
            </td>
          <td>
            Magdalena Szatkowska, Julita Zdrada-Nowak
          </td>
          <td>2025-06-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The initiation and progression of breast cancer generally involve complex immune regulatory mechanisms, with increased expression of programmed cell death ligand 1 (PD-L1) as an essential factor for immune evasion and the formation of a tumor-promoting immune microenvironment. Emerging evidence underscores the regulatory role of non-coding RNAs (ncRNAs) in modulating PD-L1 expression, influencing immune evasion, tumorigenesis, and therapy resistance in breast cancer. Therefore, it is crucial further to clarify alternative regula-tory mechanisms that control PD-L1 expression. The variations in PD-L1 expression among different breast cancer subtypes and the mechanisms by which ncRNAs regulate the expres-sion of PD-L1 are delineated. This study explores the potential and challenges of combining ncRNA-based therapy with PD-L1 inhibitors, offering insights into PD-L1 regulation and per-sonalized treatment strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e488b9798e3dd6ded6d2b5f4bc4c5b44375c7e67" target='_blank'>
              Role of Non-Coding RNAs in Regulating PD-L1 Expression in Breast Cancer: Emerging Insights and Implications.
              </a>
            </td>
          <td>
            Jianqin Zhu, Weijin Zhu, Xulin Zhou, Jingwen Hua, Xiaochun Sun
          </td>
          <td>2025-06-18</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transfer RNA-derived fragments (tRFs) are a class of small non-coding RNAs that have exhibited several functions in cancer. Recent studies have shown that mutations in tRNA genes can lead to global changes in tRF expression levels and may affect tRF function, highlighting the need to further elucidate the regulation and functions of tRFs in cancer. Here, we conducted a pan-cancer analysis of tRF quantitative trait loci (tRFQTLs), encompassing 16,703 genetic variants associated with tRF expression across 31 cancer types. A joint analysis of GWAS data revealed that tRFQTLs were preferentially enriched in cancer risk loci and colocalized with 106 GWAS variants, explaining a substantial portion of cancer heritability. Moreover, tRFs regulated by tRFQTLs were enriched in cancer-related pathways and correlated with drug response and immune infiltration. Notably, polygenic risk score models incorporating tRFQTLs improved high-risk population identification. Investigation of large-scale population cohorts revealed a tRFQTL, rs9461276, associated with colorectal cancer (CRC) risk. In biological assays, the rs9461276-C allele increased tRF-18-HSQS52D2 expression, which suppressed CRC malignant phenotypes. Mechanistically, tRF-18-HSQS52D2 bound to the 3'UTR of POU2F1, destabilizing the oncogenic transcript. Integrated RNA sequencing and ChIP-seq assays indicated that POU2F1 enhanced CRC cell proliferation by activating various pathological pathways associated with oxidative and glycolytic metabolism, mitotic stability, and cell cycle regulation. Finally, a database (Cancer-tRFQTL) was generated as a resource to support investigation into tRF-mediated mechanisms and genetic basis of tRF expression in human cancers. Overall, this study helps advance the understanding of tRFs in cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/888d9f82216c90d7528814f40ba0327b18c81f36" target='_blank'>
              Genetic Control of tRNA-Derived Fragments Contributes to Cancer Risk.
              </a>
            </td>
          <td>
            Bin Li, Hanting Li, Yan Li, Zhirui Chen, Danyi Zou, Chunyi He, Songqiang Li, Kexin Hu, Caibo Ning, Yanmin Li, Hui Geng, Shuhui Yang, Chaoqun Huang, Xiaojun Yang, Qiuhong Wang, Xu Zhu, Xiangpan Li, Ying Zhu, Meng Jin, Xiaoping Miao, Jianbo Tian
          </td>
          <td>2025-08-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 The substantial improvement in childhood cancer survival rates has been primarily achieved through therapy intensification, often at the cost of life-long adverse effects. Therapies can be associated with characteristic patterns of mutations found in the tumor genome, known as mutational signatures. These signatures, therefore, can act as markers of resistance and late effects of therapy. We employed a large-scale analysis of 611 whole-genome sequenced pediatric tumors with harmonized therapy data, integrating de novo mutational signature extraction with ensemble machine learning (ML) models trained on comprehensive genomic features (single- and double-base substitutions, indels, structural variants, and copy number alterations) to identify therapy-associated patterns. De novo extraction across single-nucleotide variants, indels, structural variants, and copy number alterations revealed 95 distinct mutational signatures, including 31 novel ones. Platinum therapy was associated with the highest mutagenic burden, with canonical signatures (e.g., SBS31, SBS35, DBS5) detected in 41% of exposed tumors. By aligning with treatment timelines, we defined a median latency of 91 days and a burden threshold of ∼1500 mutations for platinum signatures to emerge—appearing in all osteosarcomas (most common primary bone cancer) and two-thirds of neuroblastomas (most common extracranial solid tumor in children). Despite these insights, the reasons why some tumors do not exhibit these signatures, and how this relates to their response to therapy, remain poorly understood. To uncover subtler therapy-associated signals beyond known COSMIC signatures, we trained ensemble ML models on various types of genomic alterations. Our ML models achieved high predictive performance (F1-score = 0.89) in classifying platinum-exposed tumors. Importantly, predictive accuracy persisted even after masking features corresponding to canonical platinum signatures, indicating the presence of non-canonical mutational patterns. Feature analysis identified biologically plausible non-canonical genomic features enriched in platinum-exposed tumors across pediatric cancers, suggesting previously unannotated mutational footprints of therapy. Notably, ML analysis revealed distinct non-canonical genomic features associated with anthracyclines, and select antimetabolites and alkylating agents, shedding light on therapies previously lacking characterized mutational signatures. These novel genomic signals were also validated across independent pediatric and adult cancer cohorts. This work highlights the power of ML to detect therapy-induced mutational patterns beyond known mutational signatures. As cancer remains the leading cause of disease-related death among children in many countries, this integrated ML-driven approach offers a powerful strategy for improved therapy monitoring, early resistance prediction, and ultimately, the development of more personalized and effective treatment strategies (e.g., stratification and de-escalation) for children with cancer.



 Mehdi Layeghifard, Marcos Díaz-Gay, Pedro L. Ballester, Elli Papaemmanuil, Mark Cowley, Anita Villani, Ludmil B. Alexandrov, Adam Shlien. Unmasking hidden mutational footprints of therapy in pediatric tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A038.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6789b44a7ab2f8906895c50474f02a3bfc36f14" target='_blank'>
              Abstract A038: Unmasking hidden mutational footprints of therapy in pediatric tumors
              </a>
            </td>
          <td>
            M. Layeghifard, M. Díaz-Gay, Pedro L. Ballester, E. Papaemmanuil, Mark J. Cowley, A. Villani, Ludmil B. Alexandrov, A. Shlien
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="KDM4-A/B/C, preferentially demethylating di- and tri-methylated lysine 9 on histone H3, are overexpressed in cancers and considered interesting therapeutic targets. Consequently, KDM4 inhibitors have been developed to block their enzymatic activity. However, the potential lack of specificity of such small molecules (epi-drugs) may contribute to dose-limiting toxicities. In the pursuit of more specific interventions, epigenetic editing (epi-editing) has emerged as a powerful tool to modulate gene expression by modifying the epigenetic profile of specific genomic locations. The recently developed CRISPRoff (dCas9 fused to DNMT3A/3L and KRAB), guided by sgRNAs, is successfully used for gene repression by introducing methylation of DNA and (indirectly) of histones at the targeted genomic region. We propose that combining epi-editing (here to prevent the expression of KDM4) with epi-drugs (to inhibit the KDM4 protein activity) may represent a novel path for synergistic anticancer effects through simultaneous inhibition of gene expression and protein activity. Upon validating the downregulation of KDM4A in HEK293T cells through epi-editing, we demonstrated its repression in colon, breast and hepatocellular carcinomas which was effective in preventing (breast, MCF7) or inhibiting (colon, HCT116) cancer cell growth. Anticancer effect was also confirmed for these cell lines using the KDM4 inhibitor QC6352. In parallel, our studies demonstrate a previously unnoticed increase in the expression of KDM4-A/B/C genes following the inhibition of protein activity using the pan-KDM4 inhibitors QC6352 and JIB-04. Importantly, this induction of gene expression was fully prevented or even further inhibited by epi-editing. Then, we assessed the efficacy of our dual-targeted silencing approach in cancer cells and demonstrated that the inhibition in cancer cell growth by epi-drug or epigenetic editing could be further improved by combining the treatments. Building upon these findings, we introduce a novel, potentially synergistic, therapeutic strategy that combines epi-drug administration with epi-editing. This innovative approach aims to reduce drug toxicity and the potential development of resistance by preventing drug-induced upregulation of target enzyme expression, thereby further increasing anticancer effects. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01913-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6fdb1ade40cd946b854092ca3ed1de01f3bbd62" target='_blank'>
              Epigenetic editing and epi-drugs: a combination strategy to simultaneously target KDM4 as a novel anticancer approach
              </a>
            </td>
          <td>
            Federica Sarno, Jim J. Jacob, Roos E. Eilers, A. Nebbioso, Lucia Altucci, Marianne G. Rots
          </td>
          <td>2025-06-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James O. J. Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdefbda29fca27b21f2cc0c3e7a4827b0d0e9113" target='_blank'>
              The Role of Epigenetic Modifications in Enhancing Bone Marrow Transplantation Efficacy in Cancer.
              </a>
            </td>
          <td>
            B. Arjmand, Fatemeh Shahrahmani, Alieh Mirzaei, Arian Emamifar, Sepideh Alavi-Moghadam
          </td>
          <td>2025-06-17</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Recurrent somatic mutations in the key spliceosome component, SF3B1, have been identified at various frequencies across several cancer types. The most common hotspot mutation is the K700E missense mutation, and while its effects on splicing have been well characterised at the molecular level, the mis-spliced genes that contribute to cancer progression and/or dictate responses to therapy are still unclear. Here, we used we use cell line modelling to assess the impact of the SF3B1K700E mutation on the cellular response to various apoptosis-inducing agents. Our data suggest that the SF3B1K700E mutation leads to reduced cFLIP levels, along with defects in the splicing and translation of BCL2, causing a shift in the balance of pro- and anti-apoptotic genes and proteins, which confers greater sensitivity to the bivalent SMAC mimetic, BV-6. As such, BV-6 may represent a therapeutic opportunity for patients with SF3B1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78c6127f794d73c9e1a0f7e0fc6e3deedb616b30" target='_blank'>
              The cancer-associated SF3B1K700E spliceosome mutation confers enhanced sensitivity to BV-6-induced cytotoxicity
              </a>
            </td>
          <td>
            Lydia E Roets, J. Blayney, Hayley McMillan, Patrick J Preston, Alexander Mutch, Ken I Mills, K. Savage, K. Lappin
          </td>
          <td>2025-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer patients who express estrogen receptor α (ER) typically receive endocrine therapy as a first-line treatment. Most ER-positive breast cancer patients initially respond to endocrine therapy, but up to 40% of patients develop resistance over time, and the main mechanism is aberrant activation of ER signaling pathways. Recent research has shown that cancer's uncontrolled proliferative capacity and a hostile microenvironment can induce endoplasmic reticulum (EnR) stress and trigger unfolded protein response (UPR), which plays a crucial role in determining cell fate. Furthermore, increasing evidence suggests that noncoding ribonucleic acids (RNAs), which are involved during the crosstalk between the UPR and ER signaling pathways, play an essential role in endocrine resistance. In this review, we provide an overview of long noncoding RNA (lncRNA) that is involved both in endocrine resistance and UPR. We provide new insights into potential treatment strategies to overcome endocrine resistance in ER-positive breast cancer patients by targeting the lncRNA that plays key roles in endocrine resistance and UPR signaling. Finally, we discuss the current state and future directions for targeting lncRNAs to improve clinical outcomes in endocrine-resistant breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4692ed07176028a188031985ca7bc44cfc350d2e" target='_blank'>
              The role of UPR-related long noncoding RNAs in development of endocrine resistance in breast cancer.
              </a>
            </td>
          <td>
            Wen Liu, Sahil Gupta, Sanjeev Gupta
          </td>
          <td>2025-08-01</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Transcriptional regulation plays a pivotal role in coordinating the complex morphogenetic and molecular events involved in heart development and function. In the early stages of cardiovascular diseases (CVDs), the Mediator complex (MED) performs a variety of essential functions. While initial studies focused on correlating MED components with specific CVDs, more recent research has shifted toward a deeper exploration of the MED's role in the early pathogenesis of these diseases. This review highlights the latest findings published between January 2018 and February 2025, with a particular focus on the protein subunits MED1, MED12, MED13, MED13L, MED15, and MED23, and their implications in various cardiovascular pathologies. The MED complex is a crucial regulator of gene transcription, bridging transcription factors and RNA polymerase II. In “-omic” sciences, studying MED functions is essential to understanding molecular interactions that regulate gene expression. Within precision medicine, the MED complex is a key node in gene expression networks. Its study within the -omic framework can provide valuable insights into how molecular interactions shape cellular processes in both health and disease, ultimately enhancing the diagnosis, understanding, and treatment of CVDs through personalized medicine. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2962b5985f4ef86033a3760d78a66a5b8788022" target='_blank'>
              Mediator complex: update of key insights into transcriptional regulation of ancestral framework and its role in cardiovascular diseases
              </a>
            </td>
          <td>
            C. Schiano, Claudio Napoli
          </td>
          <td>2025-06-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 p53 is a transcription factor that regulates the expression of target genes involved in anti-proliferative processes such as cell cycle arrest, apoptosis, and senescence, thereby establishing its canonical role as a tumor suppressor. Due to its substantial network of target genes, p53 is also involved in non-canonical tumor suppression pathways that are yet to be fully explored. During the search of these pathways, we identified one of the methyltransferases as a novel p53 target gene. It’s been reported that this methyltransferase is downregulated in various cancers, suggesting potential tumor suppressive function. However, the specific target of this methyltransferase and its role in the mechanism of p53-mediated tumor suppression remains elusive. In this study, we show that p53 directly binds to intron 1 of this methyltransferase gene and activates its transcription and protein expression. Furthermore, we identify dihydrolipoic acid (DHLA), an important redox cofactor, as a novel target of this methyltransferase. We show that this methyltransferase enhances cellular sensitivity to oxidative stress-induced cell death by potentially methylating DHLA. Our studies reveal a novel role for p53 in regulating the well-known antioxidant DHLA to control cell death and suggest potential therapeutic strategies in promoting cancer cell death through p53 activation drugs.



 Fatima Alaydaroos, Yoko Itahana, Koji Itahana. p53-induced Methyltransferase Increases Cancer Cell Vulnerability to Cell Death [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P18.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1c8311abf9a4ec56ed8b5af3129410222f89d5" target='_blank'>
              Abstract P18: p53-induced Methyltransferase Increases Cancer Cell Vulnerability to Cell Death
              </a>
            </td>
          <td>
            Fatima Alaydaroos, Yoko Itahana, Koji Itahana
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Cells adapt to nutrient fluctuations through both signaling and epigenetic mechanisms. While amino acid (AA) deprivation is known to suppress protein synthesis via mTORC1 inactivation, the epigenetic pathways that support cellular adaptation and recovery remain poorly understood. We investigated how chromatin and transcriptional changes contribute to maintaining translational capacity during AA restriction and priming cells for growth upon AA repletion. Methods Human cells were cultured under amino acid-replete or -depleted conditions, and global histone methylation levels were assessed by Western blotting and ChIP-seq. RNA-seq and chromatin-associated RNA-seq (chromRNA-seq) were used to evaluate gene expression and transcriptional output. Ribosome profiling and [35S]-methionine/cysteine or O-propargyl-puromycin (OPP) incorporation assays measured protein synthesis. Functional contributions of SETD8 and MYC were tested through knockdown and overexpression experiments. Results AA deprivation induced a selective, genome-wide loss of H4K20me1, particularly from gene bodies, and led to increased MYC expression and binding at promoter regions. These changes were most pronounced at genes encoding ribosomal proteins and translation initiation factors. Although overall protein synthesis declined during AA restriction, these cells showed increased translational capacity evidenced by accumulation of monomeric ribosomes and enhanced translation upon AA repletion. Loss of H4K20me1 was independent of mTORC1 signaling and partly driven by SETD8 protein downregulation. While MYC overexpression alone was insufficient to upregulate translation-related genes, its combination with SETD8 knockdown in nutrient-rich conditions was both necessary and sufficient to induce expression of these genes and enhance protein synthesis. Conclusions Our findings reveal a chromatin-based mechanism by which cells integrate metabolic status with transcriptional regulation to adapt to amino acid limitation. Loss of H4K20me1 and increased MYC activity act in parallel to prime the translational machinery during AA deprivation, enabling rapid recovery of protein synthesis upon nutrient restoration. This mechanism may help explain how cells maintain competitive growth potential under fluctuating nutrient conditions and has implications for understanding MYC-driven cancer progression. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00399-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef73061499b95075a889645925006f15494b0f75" target='_blank'>
              Coordinated histone methylation loss and MYC activation promote translational capacity under amino acid restriction
              </a>
            </td>
          <td>
            Chen Cheng, Trent Su, Marco Morselli, S. Kurdistani
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, with acute myeloid leukemia (AML) being no exception. Mitochondrial function, particularly its role in protecting tumor cells against chemotherapy, is of significant interest in AML chemoresistance. In this study, we identified mitochondrial DNA content (mtDNAc), measured by quantitative PCR, as a simple and precise marker to stratify the metabolic states of AML patients. We show that patients with high mtDNAc are associated with increased mitochondrial metabolism and a higher dependency on oxidative phosphorylation (OXPHOS), often correlating with chemoresistance. Clinically, patients receiving cytarabine and an anthracycline-based regimen (7 + 3 regimen) experienced inferior relapse-free survival and a higher overall rate of leukemia recurrence. Ex vivo experiments using primary AML samples confirmed cytarabine resistance in high mtDNAc patients, which could be overcome by inhibiting mitochondrial complex I. The FDA-approved drug metformin, which targets mitochondrial metabolism, significantly enhanced apoptosis in response to chemotherapy or targeted agents, such as venetoclax, in AML models. However, metformin-treated cells adapted by increasing glycolysis and NAD+ production, a resistance mechanism that could be bypassed by targeting the nicotinamide phosphoribosyltransferase (NAMPT) enzyme. In summary, we demonstrated that mtDNAc is an effective tool for assessing the metabolic state of AML cells. This method can be easily implemented in clinical practice to identify chemoresistant patients and guide personalized treatment strategies, including novel combination therapies for those with a high reliance on mitochondrial metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db6620c6600d583bbbb5e7f68f851355099008a3" target='_blank'>
              High mtDNA content identifies oxidative phosphorylation-driven acute myeloid leukemias and represents a therapeutic vulnerability
              </a>
            </td>
          <td>
            D. Pereira-Martins, I. Weinhäuser, Emmanuel F Griessinger, J. L. Coelho-Silva, D. Silveira, Dominique Sternadt, Ayşegül Erdem, B. K. Duarte, P. Chatzikyriakou, L. Quek, Antônio Bruno Alves-Silva, Fabíola Traina, S. T. Olalla Saad, J. Hilberink, Amanda Moreira-Aguiar, Maria L Salustiano-Bandeira, Marinus M Lima, Pedro L. Franca-Neto, Marcos A Bezerra, Nisha K van der Meer, E. Ammatuna, E. Rego, G. Huls, J. Schuringa, A. Lucena-Araújo
          </td>
          <td>2025-07-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 In the United States, breast cancer is one of the leading causes of cancer death in women. Triple-negative breast cancer (TNBC) is the most aggressive of all molecular subtypes of breast cancer. It is highly heterogeneous and difficult to target and treat due to its lack of targetable receptors. Our aim is to identify new molecular targets that could aid the treatment of TNBC cases and reduce cytotoxicity of the current therapeutic regimen. The TP53 gene, a known tumor suppressor is the most frequently mutated gene in triple-negative breast cancer occurring in >80 % of TNBC cases. Statistical pathway analysis from RNA-Seq and ChIP-Seq done on the MCF10A mammary epithelial cell lines expressing the 10 most clinically prevalent p53 mutants identified the Hippo pathway to be dysregulated when associated with aggressive cellular phenotypes. Upon suppression of the upstream Hippo tumor suppressor pathway, the activation of a series of genes promoting cancer hallmark phenotypes ensues. A cell-based TEAD reporter assay showed transcriptional activity of TEAD to be higher in MCF10A cells expressing more aggressive p53 mutants compared to the cell lines with less aggressive mutants. Knockdown of the TAZ protein decreased the invasiveness of the invasive p53 mutant expressing cell lines. We hypothesized that, in addition to neo-morphic activities of different p53 mutants, the heterogeneity in clinical phenotypes observed in TNBC is due to functional interactions of the driver mutation in p53 with other functionally important co-existing mutations and dysregulated pathways, called ‘co-drivers’. To identify potential functionally important ‘co-driver’ mutations that cause cancer progression and heterogeneity in clinical phenotypes, a genome-wide screen was performed using lentiviral CRISPR screen library on the invasive MCF10A-p53-R273C-MycOE cell line. In vivo experiments showed that injection of the library-transduced cells yielded tumor in mice. PCR and whole-exome sequencing of the tumor showed frameshift mutations in the enriched EVC2 (EvC Ciliary Complex Subunit 2), NR1D1 (Nuclear Receptor 1D1), and ARAF (a Ras-activating RAF family kinase) genes. Confirming the hits from screening, knockout of these genes in R273C-MycOE cells also showed increased cell invasion. In parallel, in vitro screens carried out using CRISR knock-out library on the least aggressive MCF10A-p53-Y234C cells identified potential co-drivers. Knockout of GCNT1 and FGD6 genes in MCF10A-p53-Y234C cells increased cell invasion and induced dephosphorylation and activation of TAZ. TEAD activity was also found to be higher in these knock-out cells compared to the Y234C cells. Knockdown of the TAZ protein showed decreased TEAD activity in the FGD6 and GCNT1 knock-out cells and substantially decreased cell invasion. We have carried out functional characterization of the TP53 mutations in TNBC and confirmed the cellular phenotypes for specific p53 mutants obtained from the in vitro and in vivo screens. We are validating these identified ‘co-driver’ genes and targets to discover the molecular mechanisms affecting phenotypic heterogeneity of the p53 mutants and investigating the mechanisms through which Hippo pathway drives the progression of TNBC, as well as assessing the pathway as a target for potential therapy for TNBC with specific TP53 mutations.
 Citation Format: Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura Gonzalez-Malerva, Duatin Grief, Lydia Sakala, Jin Park, Joshua LaBaer. Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f9063c7f21ea86d31d2953e319cbe0d8301969" target='_blank'>
              Abstract P2-11-20: Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches
              </a>
            </td>
          <td>
            Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura González‐Malerva, Duatin Grief, Lydia Sakala, Jin G Park, Joshua LaBaer
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Given the prevalence of RAS mutations in various cancers, personalized therapeutic approaches, guided by molecular markers, are essential. Farnesyltransferase inhibitors (FTIs) have emerged as potential therapeutic options; however, they also face obstacles such as toxicity and limited efficacy. Alternative strategies, such as direct inhibitors combined with pathway modulators, RNA interference, and gene-editing technologies, are under clinical investigation. The targeting of RAS, complicated by its structural nuances, particularly in the G domain, has advanced with the identification of druggable pockets such as the SW-II pocket. This breakthrough has led to the development of targeted therapeutics, such as sotorasib and adagrasib, for KRAS G12C-mutated non-small cell lung cancer (NSCLC). However, these advancements face challenges, including adaptive resistance and the necessity for isoform selectivity. New inhibitors, such as LY3537982 or GDC-6036, are promising, but achieving effective and selective RAS inhibition remains a significant challenge. Additionally, clinical trials have highlighted variability in patient responses, attributing limited treatment efficacy to resistance mechanisms, including on-target mutations and off-target pathway activations. Finally, the RAS oncogene, traditionally viewed as predominantly pro-cancerous, plays a complex role in oncogenesis, with recent evidence suggesting context-dependent effects, such as inducing senescence in certain cells. This shift in understanding underscores the therapeutic potential of manipulating the interplay between RAS and TP53 mutations in cancer. In conclusion, the complexity of effectively targeting the RAS-RAF-ERK pathway is exacerbated by the diverse resistance mechanisms. Challenges such as off-target effects and delivery issues remain significant barriers in the introduction of effective therapies based on RAS inhibitors. This overview highlights the evolving nature of targeting RAS in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dda4fec514efa73d1390e9273305a191b377020" target='_blank'>
              The complex journey of targeting RAS in oncology
              </a>
            </td>
          <td>
            Katarzyna Wasiak, Damian Ciunowicz, Amelia Kierasińska-Kałka, Marta Węgierska, Marcin Pacholczyk, Piotr Rieske, Ewelina Stoczyńska-Fidelus
          </td>
          <td>2025-07-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Breast cancer is the second leading cause of cancer deaths worldwide, affecting 1 in 8 women. Disparities in breast cancer incidence and mortality have been observed across different racial and ethnic groups, with Hispanic and African American women experiencing higher incidence and mortality rates. Differences in the genomic landscape of breast cancer patients have been linked to early age at diagnosis, worse disease progression, drug resistance, and relapse. Using the All of Us research hub, one of the largest biomedical databases, we are investigating genetic variants and the genetic ancestry related to PEAK1, PALB2, SLC2A4RG, and XIRP2 across racial and ethnic groups to identify potential variations that can contribute to differences observed among these groups. These genes are involved in DNA repair, epithelial-to-mesenchymal transition (EMT), and cell cycle regulation. PEAK1, for instance, regulates cell proliferation, promotes invasion and metastasis in various cancers, including breast cancer, and is associated with drug resistance. PALB2 functions as a tumor suppressor involved in homologous recombination repair, with pathogenic variants linked to hereditary breast cancer in Hispanics. SLC2A4RG acts as a tumor suppressor by regulating the cell cycle and has been studied in glioblastoma and breast cancer. XIRP2, known for its roles in cardiovascular diseases, is implicated in breast cancer metastasis and colon cancer promotion. Despite identifying many genes involved in breast cancer development and metastasis, there remains a need to understand genetic variability across different populations to develop better diagnostic tools and improve treatment for breast cancer patients. Our cohort includes 6,056 cases of breast cancer, from which our preliminary results showed that Hispanics and African Americans are diagnosed with breast cancer at an earlier age (median = 64 and 67 years, respectively) compared to Non-Hispanics (median = 71 years). Using a genetic ancestry approach and in accordance with other studies, we identified that the majority of our Hispanic population are related to the Admixed American ancestry, with smaller portions of European and African ancestry. By accessing the All of Us research hub's control tier for genetic data, we identified the frequency of genetic variants of PEAK1, PALB2, SLC2A4RG, and XIRP2. Using a Minor Allele Frequency (MAF) of 0.01 as a parameter, we identified 27 variants for PEAK1, 12 for PALB2, 2 for SLC2A4RG, and 99 for XIRP2. Our current and upcoming analyses include associating the identified genetic variants with ancestry and race/ethnicity among cases already identified in our cohort and dataset, and comparing them with our control groups to identify possible genetic variants involved in breast cancer development across the different populations under study.
 Citation Format: Grace Vélez Crespo, Julie Dutil, Vivian Colón López. Variants of PEAK1, SLC2A4RG, PALB2, and XIRP2 and their relationship with Genetic Ancestry across Breast Cancer Patients from different Racial/Ethnic groups [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-04-14.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aef26a704a93186650e4e80a8a3c56f19425b5e" target='_blank'>
              Abstract P5-04-14: Variants of PEAK1, SLC2A4RG, PALB2, and XIRP2 and their relationship with Genetic Ancestry across Breast Cancer Patients from different Racial/Ethnic groups
              </a>
            </td>
          <td>
            G. V. Crespo, Julie Dutil, Vivian Colón López
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="There is a lack of genetic biomarkers for predicting response to immune checkpoint blockade (ICB) therapy in cancer. The recent discovery that loss-of-function mutations in the gene encoding the protein phosphatase 2A (PP2A) scaffold protein PPP2R1A confer sensitivity to immune checkpoint blockade in ovarian clear cell carcinoma, therefore represents a breakthrough. Mechanistically, mutations in the PPP2R1A gene induce a strong interferon gamma response in tumor cells, which enhances infiltration of activated CD8+ T cells into the tumor. The activity of these T cells is then fortified by ICB. Furthermore, preclinical studies have shown that PP2A inhibition leads to the generation of neoantigens by disrupting RNA splicing, and PP2A inhibition can remodel the immune microenvironment of tumors to enhance responses to ICB. The finding that loss-of-function PPP2R1A mutations predict benefit from immunotherapy also suggests that pharmacological inhibition of PP2A may act synergistically with ICB therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8541b81fbcfabd62dc3e05a2a27e83fb1d85025" target='_blank'>
              A single gene mutation predicts response to immune checkpoint blockade in ovarian clear cell carcinoma.
              </a>
            </td>
          <td>
            M. H. Dias, R. Bernards
          </td>
          <td>2025-07-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e75d3ecf904d9e7aa51e15582bd637748462a3" target='_blank'>
              Novel mutations of SRPX facilitate the stemness and malignant progression of glioma.
              </a>
            </td>
          <td>
            Siyuan Tang, Chunhui Qu, Mingyu Zhang, Peijun Zhou, Xingzhi Peng, Zhuan Zhou, Liangfang Shen, Lifang Yang
          </td>
          <td>2025-07-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Currently, there are no targeted therapies for triple-negative breast cancer (TNBC). 1 While radiation therapy plays a crucial role in breast cancer treatment, its effectiveness against TNBC is hindered by acquired resistance to radiation over time. This highlights the need for more sensitive and specific therapeutic approaches. 2 ALDH1A1 is associated with stemness and poor outcomes in TNBC and is a potential new target for treatment. ALDH1A1 contributes to chemoresistance in TNBC3, and we propose that it also promotes radiation resistance through both its antioxidant properties and regulation of cancer stemness. Previous studies have indicated that the radiation resistance observed in TNBC is linked to increased levels of ALDH1A14, suggesting it plays a protective role against radiation-induced cell death. In this study, we did a CRISPR/Cas9 knock out (KO) of ALDH1A1 in SUM 159 cells and derived single cell clones to screen for the effects of ALDH1A1 KO. Confirmation of ALDH1A1 KO was done at the genomic level by Sanger sequencing and at the protein level by western blotting analysis. The response of ALDH1A1 knockout clonal derived cells to radiation was analyzed using colony forming units (CFU), proliferation assays (Alamar blue), and Annexin V/PI staining. All the ALDH1A1 knockout clones analyzed showed enhanced radiation response with reduced proliferation, and colony formation units post-radiation, as compared to the parental cells. Furthermore, tumor initiation was measured in the parental cells and ALDH1A1 KO clones using the organoid culture system platform. Overall, ALDH1A1 KO clonal derived cells formed fewer tumoroids post-radiation, as measured by cell proliferation assays in a 3D culture system. Together these findings highlight the critical relationship between ALDH1A1 expression and radiation sensitivity, emphasizing the potential of ALDH1A1 as a target for improved treatment of TNBC.
 Citation Format: Grace Aijayi, Shirin R. Modarai, Aihui Ma, Lynn M. Opdenaker, Jennifer Sims Mourtada. CRISPR/Cas9 knockout of ALDH1A1 increases radiation response in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/addcc6f0bdb6fd3863e3c5a752c6092ba78d179c" target='_blank'>
              Abstract P3-02-25: CRISPR/Cas9 knockout of ALDH1A1 increases radiation response in triple-negative breast cancer
              </a>
            </td>
          <td>
            Grace Aijayi, Shirin R. Modarai, Aihui Ma, Lynn M. Opdenaker, J. Mourtada
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: The HOX genes encode a family of homeodomain-containing transcription factors that have important roles in defining cell and tissue identity in embryonic development, but which also show deregulated expression in many cancers and have been shown to have pro-oncogenic roles. Due to their functionally redundant nature, strategies to target HOX protein function in cancer have focused on their interaction with their PBX cofactor using competitive peptides such as HXR9. HOX/PBX inhibition triggers apoptosis through a sudden increase in target gene expression, including Fos, DUSP1, and ATF3, which are otherwise repressed by HOX/PBX binding. Methods: We analyzed publicly available transcriptomic data in the R2 platform. Results: We show that a specific subgroup of HOX genes is negatively correlated with Fos, DUSP1, and ATF3 expression in prostate cancer, and that this subgroup also shows a strong positive corelation with pathways that support tumour growth, most notably DNA repair and aminoacyl tRNA biosynthesis, and a negative correlation with genes that promote cell adhesion and prevent motility. In addition, this set of HOX genes strongly correlates with patient age, reflecting a previously identified progressive loss of regulation of HOX expression in normal peripheral blood cells. Conclusions: Our findings indicate these HOX genes may have pro-oncogenic functions in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac048edd372281c4cd71437128a80c986a12e4bb" target='_blank'>
              A Subset of HOX Genes Negatively Correlates with HOX/PBX Inhibitor Target Gene Expression and Is Associated with Apoptosis, DNA Repair, and Metabolism in Prostate Cancer
              </a>
            </td>
          <td>
            R. Morgan, Christopher Smith, H. Pandha
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract Melanoma being one of the most common and deadliest skin cancers has been increasing since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays, the standard of care of advanced melanoma is resection, followed by immune checkpoint inhibition–based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistance. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long noncoding RNAs (lncRNA) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistance; however, systematic screens to uncover novel functional lncRNAs are scarce. In this study, we profile differentially expressed lncRNAs in patient-derived short-term metastatic cultures and BRAF–MEK inhibition–resistant cells. We conduct a focused growth-related CRISPR inhibition screen of overexpressed lncRNAs, validate, and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities, and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance in melanoma. Significance: LncRNAs have emerged as novel players in regulating many cellular aspects also in melanoma. The number of functional significances of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR inhibition growths screens. Our results broaden the characterized lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5174cc80205717609c47290b3fdd1147a088cc5c" target='_blank'>
              Uncovering Novel lncRNAs Linked to Melanoma Growth and Migration with CRISPR Inhibition Screening
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, P. Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Jeffrey L. Schloßhauer, A. Tsirigos, Iman Osman, Ioannis Aifantis, Jochen Imig
          </td>
          <td>2025-06-24</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Exacerbating the DNA replication problems of cancer cells serves as a viable therapeutic approach. Nevertheless, the cytotoxicity of cancer drugs is often hampered by therapy-induced senescence, leading to unfavorable patient outcomes. Here, we employ acute replisome dysfunction in combination with Ataxia telangiectasia and Rad3-related (ATR) inhibition as a strategy to divert senescent cells toward death by triggering DNA replication catastrophe, a form of irreversible replication fork collapse caused by excessive single-stranded DNA (ssDNA) accumulation. RNA-sequencing revealed a distinct set of p53-responsive genes responsible for death. We identify c-Jun N-terminal kinase (JNK) to be essential for augmenting p53-dependent apoptotic programs and inducing pan-nuclear distribution of γH2AX, together constituting a feed-forward loop to drive cell death. Activation of DNA-PKcs initiates the signaling cascade of replication catastrophe, including CHK1-dependent JNK activation, which relies on MRE11 and PARP1 to expand and recognize ssDNA gaps, defining replication-associated gaps as an underlying basis for replication catastrophe. Our study elucidates the dynamic regulation of proximal and distal effectors along the DNA-PKcs/JNK/p53 axis that govern the cell fate decision between senescence and death. We propose that key determinants of replication catastrophe signaling are targetable vulnerabilities that can be exploited to limit senescent cell populations and increase the efficacy of anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c9be1cc99c822e2708983101c9fab63e3ac941" target='_blank'>
              The DNA-PKcs/JNK/p53 pathway underlies changes in cell fate decision toward death during DNA replication catastrophe
              </a>
            </td>
          <td>
            Jinal A. Patel, Julie Rageul, Natalie Lo, Auntara Nandi, Camryn Zezelic, Cynthia T Lee, Arafat Khan, Hyungjin Kim
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) exhibits exceptionally high levels of DNA methylation, with silencing of the DNA demethylating enzyme TET2 implicated in T-ALL’s hypermethylation phenotype. We propose that DNA hypomethylating agents (HMAs) could be particularly potent in T-ALL cells with this phenotype. Here, we used a reversible DNMT1-specific inhibitor and the conventional HMAs, 5-azacytidine and decitabine, to assess the effects of global DNA methylation loss in T-ALL cell lines and the potential of using HMAs as targeted therapeutic agents in T-ALL. We demonstrate that removal of DNA methylation, even in the absence of DNA damage, results in cell death and that toxicity is negatively correlated with methylation levels. Notably, whereas DNA demethylation caused limited transcriptional changes, key tumour suppressor genes, including TET2, were upregulated in a methylation-dependent manner. Few endogenous retroviruses or immune-related genes were reactivated after demethylation, challenging the contribution of ‘viral mimicry’ to HMA toxicity. Together, these findings provide fundamental insights into the role of DNA methylation in T-ALL, demonstrating that the removal of DNA methylation alone is sufficient to (i) induce cell death in T-ALL cell lines and (ii) reactivate silenced tumour suppressor genes. Our findings support the development of therapies targeting the unique methylation phenotype of T-ALL. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01915-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815502bed5a016e503e5f469d0f05d1f341e3aa5" target='_blank'>
              Exposing the DNA methylation-responsive compartment of the leukaemic genome in T-ALL cell lines support its potential as a novel therapeutic target in T-ALL
              </a>
            </td>
          <td>
            Maike Bensberg, Aida Selimović-Pašić, Lisa Haglund, Júlia Goldmann, Sandra Hellberg, Colm E. Nestor
          </td>
          <td>2025-07-03</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. For the first, mapping the proteins onto pathways assist combinatorial drug selections and help detect metastases. The second provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients' genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns.
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-04</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a hematologic malignancy characterized by abnormal plasma cell proliferation, with lenalidomide emerging as a primary treatment. However, prolonged use often leads to drug resistance, underscoring the need to understand the resistance mechanisms. Protein post-translational modifications (PTMs) play crucial roles in disease development, including chemoresistance. Here, we investigate the involvement of new types of PTMs, focusing on lysine lactylation (Kla), in lenalidomide-resistance. Glycolysis-driven elevation of Kla levels was observed in lenalidomide-resistant MM cells, and the subsequent inhibition of glycolytic activity significantly reversed the lenalidomide-resistance phenotype. Through quantitative proteome, lactylome, and acetylome analyses, we identified 7493 proteins, 1241 Kla sites, and 9313 lysine acetylation (Kac) sites, thereby revealing differential protein expression and PTM profiles in lenalidomide-resistant cells. Proteomic analysis revealed that a series of chemoresistance-related proteins were upregulated, and a number of Cullin-RING Ligase 4-Cereblon (CRL4CRBN) regulatory factors were downregulated. Lactylome analysis revealed that numerous chemoresistance-related proteins exhibited increased Kla levels in lenalidomide-resistant MM cells, suggesting that Kla played an important role in the development of lenalidomide-resistance in LenR MM cells. Notably, histone H4K8la was associated with upregulation of chemoresistance-related genes cyclin-dependent kinase 6 (CDK6) and enoyl-CoA hydratase (ECHS1). Our findings shed light on the epigenetic mechanisms underlying lenalidomide-resistance in MM, offering insights for overcoming chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d724e5318db67692c1b88de82a846d5c939a37" target='_blank'>
              Quantitative Proteomics Reveals the Role of Lysine Lactylation in Lenalidomide-Resistance in Multiple Myeloma Cells.
              </a>
            </td>
          <td>
            Xinlong Guo, Xuelian Ren, Cong Yan, He Huang
          </td>
          <td>2025-07-01</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Immunotherapy has emerged as a key modality in cancer treatment, yet its effectiveness varies significantly among patients, often due to the metabolic stress imposed by the tumor microenvironment. Hypoxia, a major factor in the tumor microenvironment, results from the high metabolic rate of tumor cells and inadequate vascularization, impairing immune cells’ function and potentially influencing gene expression profiles. Despite the widespread use of quantitative real-time PCR in immunological studies, to the best of our knowledge, data on reference gene stability in human peripheral blood mononuclear cells under hypoxic conditions is limited. In our study, we assessed the expression stability of commonly used reference genes (S18, HPRT, IPO8, RPL13A, SDHA, PPIA, and UBE2D2) in both non-stimulated and CD3/CD28-activated peripheral blood mononuclear cells cultured under normoxic, hypoxic (1% O2), and chemically induced hypoxic conditions for 24 h. Analysis using four different algorithms—delta Ct, geNorm, NormFinder, and BestKeeper—identified RPL13A, S18, and SDHA as the most suitable reference genes for human peripheral blood mononuclear cells under hypoxic conditions. In contrast, IPO8 and PPIA were found to be the least suitable housekeeping genes. The study provides essential insights into the stability of reference genes in peripheral blood mononuclear cells under hypoxic conditions, a critical but understudied aspect of immunological research. Given the significant impact of hypoxia on T cell metabolism and function in the tumor microenvironment, selecting reliable reference genes is crucial for accurate gene expression analysis. Our findings will be valuable for future studies investigating hypoxia-driven metabolic reprogramming in immune cells, ultimately contributing to a better understanding of T cell responses in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b494539091eb61f0af4b8a426f9120314cfa058f" target='_blank'>
              Selection of Stable Reference Genes for Gene Expression Studies in Activated and Non-Activated PBMCs Under Normoxic and Hypoxic Conditions
              </a>
            </td>
          <td>
            Artur Wardaszka, A. Smolarska, Piotr Bednarczyk, Joanna Katarzyna Bujak
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chromosome-derived circular DNAs have different characters from linear DNAs and RNAs, which exert distinctive functions, and participate in a widespread range of physiological and pathological processes. Although, the composition, structure and genome-wide frequency of circular DNAs have not yet been illuminated systemically, circular DNAs are identified to act as unanticipated major sources of somatic rearrangements, which is an important genomic feature in tumorigenesis and represents a multihit and ongoing mutagenic process. And, this process remarkably contributes to oncogenic remodeling and oncogenes amplification through chimeric circularization and reintegration of circular DNAs into the linear genome, which is closely associated with high cancer mortality and morbidity. It has been found that circular DNAs-stimulated oncogenes amplification and overexpression implicate in diverse cytopathological processes during cancer progression, including cell proliferation, apoptosis, autophagy, epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) stiffness and angiogenesis. Herein, we comprehensively introduced circular DNAs’ formation and characteristics, and emphatically elaborated circular DNAs-induced oncogenes amplification and overexpression and their functional mechanisms regulating cancer behaviors. Lastly, some mysteries in oncogenic circular DNAs’ study were introduced, and the perspectives were also discussed in cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825b11b400d7213811736a403636f24a5a529f78" target='_blank'>
              Circular DNAs Participate in Tumorigenesis and Its Mechanisms
              </a>
            </td>
          <td>
            Yuan Tan, Faqing Tang
          </td>
          <td>2025-07-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rare genetic DNA repair deficiency syndromes can lead to immunodeficiency, neurological disorders, and cancer. In the general population, inter-individual variation in DNA repair capacity (DRC) influences susceptibility to cancer and several age-related diseases. Genome wide association studies and functional analyses show that defects in multiple DNA repair pathways jointly increase disease risk, but previous technologies did not permit comprehensive analyses of DNA repair in populations. To overcome these limitations, we used fluorescence multiplex host cell reactivation (FM-HCR) assays that directly quantify DRC across six major DNA repair pathways. We assessed DRC in phytohemagglutinin-stimulated primary lymphocytes from 56 healthy individuals and validated assay reproducibility in 10 individuals with up to five independent blood draws. We furthermore developed generalized analytical pipelines for systematically adjusting for batch effects and both experimental and biological confounders. Our results reveal significant inter-individual variation in DRC for each of 10 reporter assays that measure the efficiency of distinct repair processes. Our data also demonstrate that correlations between the activities of different DNA repair pathways are relatively weak. This finding suggests that each pathway may independently influence susceptibility to the health effects of DNA damage. We furthermore developed a pipeline for analyzing comet repair kinetics and related our new functional data to previously reported comet assay data for the same individuals. Our pioneering analysis underscores the sensitivity of FM-HCR assays for detecting subtle biological differences between individuals and establishes standardized methodologies for population studies. Our findings and open source analytical tools advance precision medicine by enabling comprehensive exploration of genetic, demographic, clinical, and lifestyle factors and supporting targeted interventions to enhance DNA repair and maintain genomic integrity, thereby promoting personalized healthcare and disease prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db3f634648b2c3c7a12699cca388efeca2e26cd" target='_blank'>
              Comprehensive Measurement of Inter-Individual Variation in DNA Repair Capacity in Healthy Individuals
              </a>
            </td>
          <td>
            Ting Zhai, Patrizia Mazzucato, Catherine Ricciardi, David C. Christiani, Liming Liang, Leona D Samson, I. A. Chaim, Z. Nagel
          </td>
          <td>2025-06-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Breast cancer is the most common cancer among women, with approximately 70% of breast cancer cases being estrogen receptor-positive (ER+). These cases are commonly treated with endocrine therapy. However, resistance mechanisms can arise during treatment such as the development of mutations in the ESR1 gene, occurring in 20-40% of patients. ESR1 mutations, which can significantly impact treatment efficacy, can be detected in circulating tumor DNA (ctDNA) before, and at the time of disease progression. However, these mutations are often present at low allele frequencies, requiring highly sensitive detection methods. To address this challenge, various methods have been developed to enrich low-frequency allelic variants, thereby enhancing detection sensitivity. In this study, we assessed three enrichment methods, to enable ultra-sensitive detection of ESR1 mutations. The methods evaluated included ICE-COLD PCR, whole genome amplification using REPLI-g, and multiplex pre-amplification. Contrived samples of known variant allele frequencies were used to quantify the extent of enrichment achieved by each method. Mutation detection was performed using the APIS ESR1 Mutations Kit, capable of detecting eleven mutations within exon 5 (E380Q), 7 (S463P), and 8 (P535H, L536R/Q/H/P, Y537C/S/N, D538G). The limit of detection of the qPCR assay with each enrichment method was compared to determine their effectiveness. Our findings suggest that enrichment methods notably enhance the sensitivity of ESR1 mutation detection in ctDNA. For instance, ICE-COLD PCR demonstrated potential for improving detection sensitivity by selectively amplifying mutant alleles, while REPLI-g and multiplex pre-amplification can increase the number of mutant copies in the reaction, without impacting amplification specificity. These advancements could enable more accurate systemic assessment and dynamic monitoring of ESR1 mutations in breast cancer patients, potentially before disease progression is evident. This capability allows for more tailored therapeutic approaches, improving patient outcomes by facilitating early intervention and more personalized treatment strategies. In conclusion, employing enrichment techniques for low-allele frequency variants can improve the sensitivity of ESR1 mutation detection in breast cancer. This approach has the potential to enhance our ability to monitor and respond to disease progression dynamically, ultimately supporting more effective, individualized patient care.
 Citation Format: Anna Gasior, Joanna Gorniak, Andreas Voss, Ryan Nana, Ava Read, Luke Matthews, Kimberly Howard, Leanne Gough, Christine Hoy, Colette Whitfield. Improving ESR1 Mutations Detection in Breast Cancer Circulating Tumor DNA: Comparative Analysis of Enrichment Techniques [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-01-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a9532691f514c216b9f5a3d0769b2f224b4034" target='_blank'>
              Abstract P5-01-09: Improving ESR1 Mutations Detection in Breast Cancer Circulating Tumor DNA: Comparative Analysis of Enrichment Techniques
              </a>
            </td>
          <td>
            A. Gasior, Joanna Gorniak, Andreas Voss, Ryan Nana, Ava Read, Luke Matthews, K. Howard, Leanne Gough, Christine Hoy, Colette Whitfield
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Breast cancer is the second leading cause of cancer death in women. The primary cause of mortality is associated with the metastasis process of cancer cells to other organs. It has become obvious that epigenetic alteration plays a crucial role in breast cancer progression through the modulation of various signaling pathways. Additionally, the plastic nature of cancer cells is essential to their adaption in different metastatic sites. However, implementing epigenetic therapies in the clinic has been challenging due to the ubiquitous expression of many epigenetic factors. Here, we evaluate a library of epigenetic drugs to determine their impact on receptors commonly targeted in breast cancer. These include estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2). We coupled an immuno-fluorescence imaging approach to bioluminescence imaging to capture receptor heterogeneity and the proliferation index of each epigenetic drug. So far, we have assessed 460 drugs that target various factors, including Epigenetics, JAK/STAT, DNA Damage, Angiogenesis, Cell Cycle, PI3K/Akt/mTOR and Cytoskeletal Signaling targeting and histone deacetylases (HDACs), DNA methyltransferases (DNMTs), histone methyltransferases (HMTs), aurora kinase PARP, Sirtiin, EGFR, JAK, Plm pathways. We also analyzed the expression of key epigenetic markers (e.g., EZH2) and stemness-associated factors (e.g., SOX9). As a result, we found BIO, an inhibitor of GSK-3 and JAK, increases ER expression. Similarly, Curcumin, an inhibitor of p300 histone acetyl-transferase and Histone deacetylase (HDAC) promotes a broader range of factors, including Her2, Sox9, and ER. Interestingly, (+)-JQ1(BET bromodomain inhibitor), I-BRD9 (BRD9 inhibitor) and CUDC-101 (HDAC inhibitor) drastically reduced Ki67 level. Overall, these findings suggest a potential for combination therapy. While further investigation of these promising candidates is needed (3D, ex vivo, and in vivo testing), this investigation reveals new possibilities to overcome therapeutic resistance in breast cancer.
 Citation Format: Rahat Alam, Igor Bado. Epigenetic Drug Profiling for Breast Cancer Therapy: A Focus on Receptor Modulation and Heterogeneity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d889ed4ca5a9cd73bc3f7725a940c34fe41655cb" target='_blank'>
              Abstract P4-10-27: Epigenetic Drug Profiling for Breast Cancer Therapy: A Focus on Receptor Modulation and Heterogeneity
              </a>
            </td>
          <td>
            Rahat Alam, Igor L Bado
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer genomics has advanced a lot; a main problem is still turning molecular findings into treatments due to the variety of cells in each tumor and gaps in knowing certain pathway functions. Predicting whether a treatment will be effective is difficult right now which means that patient outcomes are often less than ideal. This research tackled the issue by studying whether looking at many genes and oncogenic pathways at the same time helps identify which patients may show cancer progression or treatment resistance. For this study, we looked at the genes, proteins and other molecules from a total of 1,500 patients in six major types of cancer (breast, colorectal, lung, melanoma, ovarian and prostate). We used genomic variant calling, analysis of gene pathways and detailed study of proteins with advanced bioinformatics and machine learning tools. The study found that TP53 mutations (OR: 2.14, 95% CI: 1.72-2.66, p < 0.001) and the PI3K/AKT/mTOR pathway being activated (OR: 1.89, 95% CI: 1.51-2.37, p = 0.003) were strongly linked to cancer not responding to treatment and negative outcomes. Tumors with a high number of mutations (≥ 10/Mb) reacted much better to immunotherapy (OR: 3.02, p < 0.001). Three subgroups of tumors were found through unsupervised analysis and these showed different chances of success: tumors with many mutations and high immune activity performed best (39.1% complete response), but those with mutations in PI3K and TP53 had the worst prognosis (28.5% progressive disease). Patients whose tumors were active in the PI3K/AKT pathway had progression-free survival of 8.2 months, significantly less (log-rank p < 0.001) than those whose tumors were not active. Evidence shows that integrative molecular profiling is better at predicting outcomes than the traditional method of classifying tumors by looking at slides. Study results show which molecular subgroups are most important to oncology and help link their treatment response to which therapy is best,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e766178e1eebb59cae34bf9ab40a3c9e125bd68f" target='_blank'>
              Integrative Molecular Profiling of Oncogenic Pathways and Genetic Mutations in Cancer Progression and Therapeutic Response
              </a>
            </td>
          <td>
            Javeria Taj, Muhammad Noman Ajmal, Tayyaba Arshad, Afifa Dawood, Ali Abbas, Misbah Hafeez, Hamza Rafeeq, Abdul Malik, Aiman Nishat, Mujahid Hussain
          </td>
          <td>2025-06-20</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0e7e32e3545d727024253e68ada05454a1fe358" target='_blank'>
              The role of ASXL1, SRSF2, and EZH2 mutations in chromatin dysregulation of myelodysplastic neoplasia and acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hosang Yu, Junshik Hong, Dong-Yeop Shin, Chul-Hwan Lee
          </td>
          <td>2025-06-25</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor-derived biomarkers can precisely reflect cell types, offering major potential for early detection and therapy. An engineered platform capable of distinguishing various biomarkers for cancer type identification and autonomous responsive therapeutics is urgently needed. Herein, a "one stop" production line with logic gated composite for tumor cell identification, related biomarker imaging, and subsequent gene therapy is proposed. The designed composite can specifically target cancer cells through incorporated aptamers, which endows it with identification preference for cancer cells with overexpressed membrane protein receptors. Internalized materials collapse in the acidic lysosome milieu, releasing the functioning DNA strands. Endogenous miR-210 then can initiate cascade reactions with released strands, resulting in the liberation of therapeutic antisense oligonucleotides (ASO). The strategy is to construct a platform for cancer therapy based on second-order AND Boolean logic operation for multiple cancer biomarkers (Mucin 1 (MUC1), Protein tyrosine kinase 7 (PTK7), and miR-210) identification and succeeding therapy toward identified cells. Through the logic operation, produced fluorescent signals can disclose the types of biomarkers being recognized, while nano-material endocytosis, subsequent miRNA imaging, and endogenous miRNA-triggered gene therapy can all be performed concurrently. This "one stop" multifunctional production line demonstrates excellent sensitivity and accuracy, and presents promising prospects in the early diagnosis of tumors and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2e7e35eedc8f5095ca22e7d62f436db09d918b" target='_blank'>
              Logic-Gated DNA Production Line for Cancer Cell Recognition and Intracellular Cascaded Gene Therapy.
              </a>
            </td>
          <td>
            Hui Xin, Yan Zhong, Bo-lun Li, Chun-yang Wang
          </td>
          <td>2025-06-25</td>
          <td>Small</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Mismatch repair (MMR) deficiency is a hallmark of microsatellite instability (MSI) in hereditary non-polyposis colorectal cancer, Lynch syndrome, contributing to resistance against conventional chemotherapy and posing a significant therapeutic challenge. In this study, we introduce UNI110, a novel small molecule derived from Baicalein, engineered for enhanced selectivity against MMR-deficient cancer cells. UNI110 exhibits a remarkable sevenfold increase in potency over Baicalein, demonstrating significantly lower IC50 values and heightened cytotoxic effects in MMR-deficient cell lines. Mechanistically, UNI110 selectively induces DNA damage in MMR-deficient cancer cells, ultimately resulting in cell death. Furthermore, UNI110 disrupts homologous recombination (HR) repair by inhibiting the MSH2-MSH3 complex, specifically blocking the interaction between MSH2 and EXO1, thereby impairing long-range end resection during double-strand break (DSB) repair. These findings establish UNI110 as a promising lead compound for the targeted treatment of MMR-deficient colorectal cancers, offering a potential breakthrough in overcoming chemotherapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5c30430d4a1e738e7be9c0b1bd6508b0c419a8" target='_blank'>
              Targeting MMR-deficient colorectal cancer with a potent small molecule UNI110
              </a>
            </td>
          <td>
            Enkhzul Amarsanaa, Jung-Min Oh, Seon Young Lee, Saikat Maiti, Sung You Hong, Kyungjae Myung
          </td>
          <td>2025-08-04</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Non-small cell lung cancer (NSCLC) often harbors oncogenic driver alterations such as epidermal growth factor receptor (EGFR), anaplastic lymphoma kinase (ALK), or Kirsten rat sarcoma viral oncogene homolog (KRAS), which can be effectively targeted with tyrosine kinase inhibitors (TKIs). However, tumors frequently contain additional co-occurring mutations in genes like tumor protein p53 (TP53), serine/threonine kinase 11 (STK11), and Kelch-like ECH-associated protein 1 (KEAP1), which may affect tumor growth, immune evasion, and treatment response. These co-mutations contribute to molecular heterogeneity and are associated with worse prognosis or resistance to therapy. This review summarizes the spectrum and frequency of co-mutations in NSCLC, their prognostic and predictive implications, and their relevance to targeted therapy and immune checkpoint inhibitors (ICIs). Understanding co-mutational profiles is critical to improving precision oncology strategies and tailoring treatment in advanced NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e66415b6ab52cee6417f7da434d635431ca9b4e" target='_blank'>
              Co-Occurring Genomic Alterations in NSCLC: Making Order into a Crowded List
              </a>
            </td>
          <td>
            I. Attili, Federico Pio Fabrizio, F. Marinis
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b369f79b329c117da716bed78cecbcb615823b8f" target='_blank'>
              JAK2 signalling to HNRNPA1 represses retrotransposon activity in haematopoietic stem cells
              </a>
            </td>
          <td>
            Jeremy W. Deuel, Andrejs Makarovs, Sheidasadat Goosheh, George Winder, Hisham Mohammed, Jason S. Carroll, Michael Spencer Chapman, C. Kapadia, Peter J. Campbell, J. Nangalia, E. Laurenti, C. Schneider, T. Oellerich, Anthony R Green, S. Loughran
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer remains one of the most formidable global health challenges due to its complex pathogenesis, late-stage diagnosis, and limited therapeutic options for many subtypes. Despite advancements in reducing cancer-related mortality through lifestyle modifications and early interventions, it continues to rank among the top ten causes of death worldwide. A growing body of evidence highlights the critical role of non-coding RNAs, particularly microRNAs (miRNAs), in cancer biology. MiRNAs are short, non-coding RNA molecules approximately 19–25 nucleotides in length that regulate gene expression at the post-transcriptional level. Among these, miRNA-1303 has garnered increasing attention due to its aberrant expression across a variety of malignancies and its involvement in diverse cellular processes, including cell proliferation, apoptosis, migration, invasion, and chemoresistance. In this review, we provide a comprehensive overview of miRNA-1303, emphasizing its dysregulation in different cancer types such as colorectal, breast, prostate, and lung cancers. We explore the molecular mechanisms and signaling pathways modulated by miRNA-1303, including its interaction with oncogenes, tumor suppressors, and key cellular networks like PI3K/AKT, Wnt/β-catenin, and MAPK pathways. We also discuss its functional role in tumor development and progression, supported by both in vitro and in vivo studies. Furthermore, we evaluate the potential of miRNA-1303 as a clinically relevant biomarker for early cancer detection, prognosis prediction, and treatment monitoring. Its expression patterns have been correlated with tumor stage, metastasis, and patient survival, suggesting promising utility in clinical decision-making. In addition, we explore the emerging therapeutic strategies targeting miRNA-1303, including miRNA mimics, inhibitors, and delivery systems designed to modulate its expression in cancer cells. By synthesizing current findings, this review underscores the clinical significance of miRNA-1303 in oncology and proposes future directions for its application in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc4113503577ac3e30314feefc6d7bf89e1993d" target='_blank'>
              MicroRNA-1303 in cancer pathogenesis and therapy: clinical implications for biomarker development and targeted treatment strategies
              </a>
            </td>
          <td>
            Lingzi Zheng, Ling Hu, Bita Badehnoosh
          </td>
          <td>2025-07-11</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="INTRODUCTION
Chromatin regulators (CRs) are critical in cancer development, yet their prognostic value in lung squamous cell carcinoma (LUSC) remains unclear. This study aimed to develop a CR-based prognostic model and explore the role of APOBEC1 in LUSC progression.


METHODS
Transcriptomic and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were analyzed. Gene Ontology and KEGG pathway enrichment were performed. Prognostic CRs were identified using univariate Cox and Lasso analyses. Drug sensitivity was assessed via the Drug Signatures Database. Key CRs were validated by PCR. APOBEC1 expression and prognosis were evaluated through pan-cancer analysis and experimentally validated in LUSC cell lines using colony formation, CCK-8, wound healing, and transwell assays.


RESULTS
An eight-gene CR-based signature was established, achieving 5-year AUCs of 0.87, 0.92, and 0.73 in the TCGA training, validation, and GEO datasets, respectively. High-risk patients showed enrichment in cancer-related pathways, greater immune infiltration, and elevated immune checkpoint expression. They were also more sensitive to Dasatinib, Bexarotene, and Bicalutamide. APOBEC1 was overexpressed across multiple cancer types and promoted proliferation and migration in LUSC cell lines.


CONCLUSIONS
This study presents a robust CR-based survival model and highlights APOBEC1 as a potential therapeutic target in LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1d0130aeae62ff4d007467ce0d311ddfbd09e2" target='_blank'>
              Prognostic significance of APOBEC1 and its role in lung squamous cell carcinoma: insights from chromatin regulator-based modeling and experimental validation.
              </a>
            </td>
          <td>
            Min Liang, Chaohao Zhang, Shengming Liu
          </td>
          <td>2025-08-05</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved by specialized helicases. Previous in vitro studies showed that DEAH-box helicase 36 (DHX36/G4R1/RHAU) has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in the accumulation of cytoplasmic DNA fragments, an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression, and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 messenger RNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd74a217dd008b29b89824a440aee3aba255159" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, Shangming Tang, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Yuh-Hwa Wang
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Resveratrol (RSV) is one of the most studied and used biomolecules, for which many pharmacological effects targeting multiple tissues have been described. However, a common underlying mechanism driving its full pharmacological activity has not been described in detail. G-quadruplexes (G4s) are non-canonical nucleic acid structures found in regulatory genomic locations and involved in controlling gene transcription, telomere maintenance, or genome stability, among others. This study provides a genome-wide characterization of RSV G4-binding properties, explaining its multi-target traits. Methods Immunofluorescence assays using a nucleolar and a G4-specific antibody were used to characterize RSV cellular effects on the nucleolus and G4 stabilization. DNA damage and cell cycle analyses were performed via western blot and flow cytometry. Breaks lLbeling In Situ and Sequencing (BLISS) was used to map double strand breaks (DSB) in response to treatment, and identify G4s targeted by RSV. mRNA sequencing was used to identify changes at the transcriptional level upon treatment and relate them to a direct targeting of G4s. Biophysical assays (circular dichroism, ultraviolet–visible [UV–Vis] titration, differential scanning calorimetry, and nuclear magnetic resonance) were used to characterize RSV–G4 interactions. Lastly, luciferase-based transcription assays were performed to confirm RSV–G4 interaction in vitro and its direct influence on gene expression. Results In a cellular context, RSV treatment showed classic G4-ligand effects, such as nucleolar disassembly, inhibition of RNA polymerase I, DNA damage, and cell cycle arrest. RSV was shown to stabilize cellular G4s, which accumulated around double strand breaks in the promoters of differentially expressed genes. Upon treatment, G4 stabilization triggered DNA damage and controlled gene expression. The interaction between RSV and target G4s was confirmed in vitro by biophysical assays and through luciferase-based transcription assays. Conclusions A G4-dependent mode of action was demonstrated as the main mechanism underlying RSV pleiotropic effects, along with the identification of target genes and G4s. This in-depth analysis of the mode of action of RSV will be helpful to improve its therapeutic potential in a wide variety of health scenarios. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00747-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b9a5bcda358645827e337bc4418daea949d2f" target='_blank'>
              In-depth analysis of the mode of action of resveratrol: genome-wide characterization of G-quadruplex binding properties
              </a>
            </td>
          <td>
            Ana Soriano-Lerma, Victoria Sánchez-Martín, Javier Murciano-Calles, Matilde Ortiz-González, Maria J. Tello-Lopez, Virginia Pérez-Carrasco, Ángel Linde-Rodríguez, Inmaculada Ramírez-Macías, Irene Gómez-Pìnto, Inmaculada López-Aliaga, M. Soriano, Jose A. Garcia Salcedo
          </td>
          <td>2025-06-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Anthracyclines have been clinically well established in cancer chemotherapy for decades. The main limitations of this drug class are the development of resistance and severe side effects. In the present investigation, we analyzed 30 anthracyclines in a panel of 59 cell lines of the National Cancer Institute, USA. The log10IC50 values varied from −10.49 M (3′-deamino-3′-(4″-(3″-cyano)morpholinyl)-doxorubicin, 1) to −4.93 M (N,N-dibenzyldaunorubicin hydrochloride, 30). Multidrug-resistant NCI-ADR-Res ovarian cancer cells revealed a high degree of resistance to established anthracyclines (between 18-fold to idarubicin (4) and 166-fold to doxorubicin (13) compared to parental, drug-sensitive OVCAR8 cells). The resistant cells displayed only low degrees of resistance (1- to 5-fold) to four other anthracyclines (7, 18, 28, 30) and were even hypersensitive (collaterally sensitive) to two compounds (1, 26). Live cell time-lapse microscopy proved the cross-resistance of the three chosen anthracyclines (4, 7, 9) on sensitive CCRF/CEM and multidrug-resistant CEM/ADR5000 cells. Structure–activity relationships showed that the presence of tertiary amino functions is helpful in avoiding resistance, while primary amines rather increased resistance development. An α-aminonitrile function as in compound 1 was favorable. Investigating the mRNA expression of 49 ATP-binding cassette (ABC) transporter genes showed that ABCB1/MDR1 encoding P-glycoprotein was the most important one for acquired and inherent resistance to anthracyclines. Molecular docking demonstrated that all anthracyclines bound to the same binding domain at the inner efflux channel side of P-glycoprotein with high binding affinities. Kaplan–Meier statistics of RNA sequencing data of more than 8000 tumor biopsies of TCGA database revealed that out of 23 tumor entities high ABCB1 expression was significantly correlated with worse survival times for acute myeloid leukemia, multiple myeloma, and hepatocellular carcinoma patients. This indicates that ABCB1 may serve as a prognostic marker in anthracycline-based chemotherapy regimens in these tumor types and a target for the development of novel anthracycline derivatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0509a300730a6303969f2ff8af79884f58309d0" target='_blank'>
              Predictive and Prognostic Relevance of ABC Transporters for Resistance to Anthracycline Derivatives
              </a>
            </td>
          <td>
            Rümeysa Yücer, Rossana Piccinno, Ednah Ooko, M. Dawood, Gerhard Bringmann, T. Efferth
          </td>
          <td>2025-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bromodomain-and-extraterminal-domain (BET) proteins are promising therapeutic targets for refractory solid tumors, including triple-negative breast cancer (TNBC). However, acquired resistance to BET inhibitors (BETi) remains a significant clinical challenge. Elucidation of the underlying mechanisms of BETi resistance is therefore of critical importance. In this study, we identified the RNA-binding protein IGF2BP2 as a key driver of acquired BETi resistance in TNBC, primarily through its role in enhancing the translation of c-MYC mRNA. Given that IGF2BP2 is not an ideal target for small-molecular drugs, we performed RNA immunoprecipitation sequencing (RIP-Seq) and found circRNA-BISC as a potent IGF2BP2 repressor. BISC effectively inhibited both c-MYC translation and BETi resistance. Notably, BISC contains a "CAC-linker-XGGX" motif that specifically binds IGF2BP2 rather than to IGF2BP1 and IGF2BP3. The efficacy and selectivity of BISC in targeting IGF2BP2 prompted further exploration of BISC-based RNA therapeutics for TNBC. In vitro transcribed and circularized BISC, when combined with the BETi OTX-015, demonstrated impressive tumor regression in BETi-resistant TNBC models without detectable toxicity. These findings establish BISC as a potent IGF2BP2 repressor and highlight the feasibility of circRNA-based therapeutic strategies to overcome BETi resistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033e837d89a8b29610b2b0ee6157dde8b14222f2" target='_blank'>
              A circular RNA overcomes acquired resistance to BET inhibitors by antagonizing IGF2BP2-mediated c-MYC translation in TNBC.
              </a>
            </td>
          <td>
            Jiawei Guo, Ke Li, Yue Ming, Yitong Pan, Shuangyan Tan, Hulin Ma, Shuang Chen, Yingying Duan, Yong Peng
          </td>
          <td>2025-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87e2a56734bd4735ea966fc9adf5ca89a77f1711" target='_blank'>
              Increased antiandrogen enzalutamide sensitivity via altering prostate cancer stem cell traits through modulating the androgen receptor-mediated CDR1/circCDR1-AS/miR-1290/BMP4 signaling.
              </a>
            </td>
          <td>
            Diwei Huo, Minggui Si, Kexin Yu, Xiaoxue Fang, Hongbo Liu, Donglong Li, Zhengxing Chen, Jinguo Li, Ruicong Xu, Xinwang Su, Yongfeng Du, Xuebin Ma, Xunwei Wang, Pengbo Li, Huike Yang, Xiujie Chen, Keliang Wang
          </td>
          <td>2025-07-04</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncogenic KRAS induces metabolic rewiring in pancreatic ductal adenocarcinoma (PDAC) characterized, in part, by dependency on de novo pyrimidine biosynthesis. Pharmacologic inhibition of dihydroorotate dehydrogenase (DHODH), an enzyme in the de novo pyrimidine synthesis pathway, delays pancreatic tumor growth; however, limited monotherapy efficacy suggests that compensatory pathways may drive resistance. Here, we use an integrated metabolomic, proteomic and in vitro and in vivo DHODH inhibitor-anchored genetic screening approach to identify compensatory pathways to DHODH inhibition (DHODHi) and targets for combination therapy strategies. We demonstrate that DHODHi alters the apoptotic regulatory proteome thereby enhancing sensitivity to inhibitors of the anti-apoptotic BCL2L1 (BCL-XL) protein. Co-targeting DHODH and BCL-XL synergistically induces apoptosis in PDAC cells and patient-derived organoids. The combination of DHODH inhibition with Brequinar and BCL-XL degradation by DT2216, a proteolysis targeting chimera (PROTAC), significantly inhibits PDAC tumor growth. These data define mechanisms of adaptation to DHODHi and support combination therapy targeting BCL-XL in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9183ca2d7210b8ec4357f3959c0684f349b63b" target='_blank'>
              De novo pyrimidine biosynthesis inhibition synergizes with BCL-XL targeting in pancreatic cancer
              </a>
            </td>
          <td>
            Huan Zhang, Naiara Santana-Codina, Qijia Yu, Clara Poupault, Claudia Campos, Xing-Lei Qin, Nicole Sindoni, Marina Ciscar, A. Padhye, Miljan Kuljanin, Junning Wang, Matthew J. Dorman, Peter Bross, Andrew J. Aguirre, Stephanie K Dougan, K. Sarosiek, J. Mancias
          </td>
          <td>2025-07-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Prostate cancer is the most prevalent cancer among men worldwide and exhibits significant genetic heritability. In this study, we performed an integrative analysis combining chromatin accessibility profiling, transcriptomics, and two-stage case-control studies, alongside an unbiased phenome-wide exploration in the FinnGen cohort. This comprehensive approach identified the enhancer-associated single-nucleotide polymorphism rs7077830 at chromosome 10q11 as a critical modulator of prostate cancer susceptibility. Mechanistic investigations revealed that rs7077830 exerted allele-specific enhancer activity, driving NCOA4 expression via a ZNF384-mediated enhancer-promoter interaction. CRISPR/Cas9-mediated single-nucleotide editing confirmed the direct regulatory role of rs7077830 on NCOA4 and its contribution to prostate cancer progression. Functional studies demonstrated that NCOA4 acted as a tumor suppressor by promoting ferroptosis. Furthermore, NCOA4 expression modulated sensitivity to ferroptosis-inducing agents. Strikingly, rs7077830 genotypes were directly linked to ferroptosis status in prostate cancer. These findings highlight the role of noncoding genetic variants in prostate cancer pathogenesis, provide insights into enhancer-driven ferroptosis regulation, and suggest a promising therapeutic avenue for prostate cancer patients carrying specific rs7077830 genotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78490ad8e9f9afb3d3853a6b0cad7e6e00f061a7" target='_blank'>
              Integrated Omics and Multi-Cohort Analyses Identify an Enhancer Variant Linking Ferroptosis to Precision Therapy in Prostate Cancer.
              </a>
            </td>
          <td>
            Sheng Ma, Yue Ge, Zequn Lu, Junbiao Zhang, Qin Zhang, Yuzheng Peng, Zirui Xi, Hao Peng, Meng-Yao Xu, Zezhong Xiong, Yuan Gao, Yanan Wang, Le Li, Chunyu Zhang, Zheng Chao, Tengfei Li, Ruiyun Zhang, Qiang Zhang, David A Horne, Baojun Wang, Xu Gao, Xu Zhang, Jianbo Tian, Jing Li, Meng Jin, Gonghong Wei, Zhihua Wang
          </td>
          <td>2025-08-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignant tumor and a leading cause of cancer-related mortality among women worldwide. The disease comprises distinct molecular subtypes that influence tumor behavior, metastatic potential, and therapeutic response. This review presents recent advances in breast cancer research, with a particular emphasis on molecular and epigenetic mechanisms that contribute to tumor development, progression, and treatment resistance. Key signaling pathways—including estrogen receptor (ER), human epidermal growth factor receptor 2 (HER2), Notch, fibroblast growth factor receptor (FGFR), Wnt, and Hedgehog—play essential roles in the regulation of mammary stem cells and oncogenesis. Increasing evidence highlights the significance of epigenetic alterations such as DNA methylation, histone modifications, and microRNA expression in modulating gene activity relevant to tumor initiation and therapy resistance. Epigenetic molecular targets, including DNA methyltransferases (DNMTs), histone deacetylases (HDACs), EZH2, and non-coding RNAs, are gaining attention for their potential use in diagnosis, prognosis, and targeted therapy. Integration of multigene panel testing with epigenetic biomarkers has facilitated improved risk assessment and the development of individualized treatment strategies. Moreover, novel therapeutic approaches—such as CAR-T cell therapy, nanoparticle-mediated drug delivery systems, and the involvement of circular RNAs (circRNAs) in immune modulation—offer promising directions for precision medicine. This review consolidates current insights into the molecular and epigenetic landscape of breast cancer to provide a comprehensive understanding of disease complexity and to inform the development of more effective, personalized treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8400e20214b339b7e7810249446b370e6720099d" target='_blank'>
              Recent Insights Into Breast Cancer: Molecular Pathways, Epigenetic Regulation, and Emerging Targeted Therapies
              </a>
            </td>
          <td>
            Quratulain Maqsood, Muhammad Umer Khan, Tehreem Fatima, Sania Khalid, Zaryab Ikram Malik
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer : Basic and Clinical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The t(8;21)(q22;q22) translocation is one of the most recurrent cytogenetic aberrations in acute myeloid leukaemia (AML). While most patients achieve complete remission, approximately 40% of them still relapse. Early identification and elimination of leukaemia clones with relapse potential could improve prognosis for t(8;21) AML patients. Here, through single-cell RNA sequencing, we characterized the intra-tumoral heterogeneity of t(8;21) AML and identified haematopoietic stem cell (HSC)-like subset as the most quiescent and primitive subgroup among all leukaemia cell populations. Further investigations revealed IKZF2 as the master regulator for HSC-like subset. Notably, IKZF2 was upregulated in t(8;21) AML compared with other AML subtypes and was specifically targeted by AML1-ETO. Using primary samples and mouse models, we verified the high enrichment of IKZF2 in primitive and quiescent leukaemic cells. Moreover, IKZF2 knockout hindered the accumulation of aberrant stem cells driven by AML1-ETO and promoted cellular differentiation both in vitro and in vivo. These facilitate a better understanding of the leukaemia cell heterogeneity in t(8;21) AML and unveil IKZF2 as a potential target for improving current treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a3a7fc693a064b9e09834f53317ec734d6a1c07" target='_blank'>
              Single-cell transcriptome profiling reveals blast cell heterogeneity and identifies novel therapeutic target IKZF2 in t(8;21) acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Yu Liu, Wenbing Liu, Yihan Mei, Qianqian Huang, Xiaoyu Liu, Chengcai Guo, Manling Chen, Junli Mou, Shangshang Wang, Wanqing Xie, Zheng Tian, Kejing Tang, H. Xing, Ying Wang, Hui Wei, R. Gu, Qing Rao, Min Wang, Shaowei Qiu, Jianxiang Wang
          </td>
          <td>2025-08-08</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The occurrence and development of malignant tumors involve abnormalities in complex molecular regulatory networks, among which the abnormal activation of the transcriptional regulator hairy and enhancer of split 1 (Hes1) has attracted significant attention in recent years and is closely associated with prognosis in various malignancies. Hes1 exhibits high expression in various solid tumors and hematological malignancies, where it participates in alterations involving diverse immune cells, inflammatory factors, and the immune microenvironment, thereby promoting tumor cell proliferation, invasion, metastasis, and resistance to treatment. Recent studies have widely investigated the potential of targeting Hes1 and inhibiting its expression as a cancer therapeutic strategy, although its precise mechanisms of action are not yet fully elucidated. Hes1 interacts with critical pathways including Notch, JAK/STAT, PI3K/AKT/mTOR, and Wnt/β-catenin. These interactions form complex crosstalk networks that drive malignant transformation and progression. Furthermore, Hes1 plays a central role in the formation of an immunosuppressive tumor microenvironment (TME) and immune escape by regulating the expression of immune checkpoint-associated proteins, extracellular matrix (ECM) remodeling, and other processes, making it a highly promising therapeutic target. Notably, the expression level of Hes1 is significantly correlated with tumor clinical stage, prognosis, and drug resistance. This review comprehensively introduces the mechanisms of Hes1 in the progression of malignant tumors, with a particular focus on discussing its application and underlying mechanisms in tumor immunotherapy. It integrates the latest clinical evidence and preclinical research perspectives. The goal is to highlight the translational potential of Hes1 as a novel biomarker and molecular target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca4a5e14b5411c1cf587b921ebef2a69839952a" target='_blank'>
              Hes1 in malignant tumors: from molecular mechanism to therapeutic potential
              </a>
            </td>
          <td>
            Liping Zhang, Qian Zhang, Cheng Guo, Zixin Ru, Zetian Yang, Yi Geng, Junjie Yang, Daigui Zhang, Zhenhuai Yang, Shuicai Huang
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Response gene to complement 32 (RGC32), a complement activation-inducible factor broadly expressed in normal human tissues, has been implicated in tumorigenesis through its dysregulated expression in various malignancies and its involvement in critical oncogenic processes. Despite its established roles in cancer biology, RGC32 remains uncharacterized in diffuse large B-cell lymphoma (DLBCL). This study provides the first comprehensive investigation of RGC32 expression patterns and functional contributions to DLBCL pathogenesis, elucidating its potential as a novel therapeutic target or prognostic biomarker in this disease. Methods Immunohistochemical (IHC) staining of RGC32 was performed on specimens from 32 Reactive hyperplasia lymphoid (RHL) patients and 80 DLBCL patients. To evaluate the role of RGC32 in DLBCL, lentivirus vectors either encoding shRGC32 or shControl were transfected into DLBCL cell lines. RNA-sequencing (RNA-seq) analysis was performed between shRGC32 and shControl stably transfected OCI-LY1 cells and functional enrichment analyses used gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG). In order to explored its functions in vivo, xenograft models were established by subcutaneously injecting shRGC32 and shControl transfected DLBCL cells into SCID beige mice. Results Immunohistochemical analysis revealed RGC32 overexpression in DLBCL tissues contrast with RHL, and was associated advanced Ann Arbor stage (p = 0.043), B symptoms (p = 0.020), and poor progression-free survival (p = 0.015) and overall survival (p = 0.035). Functional studies demonstrated that RGC32 knockdown via shRNA significantly suppressed DLBCL cell proliferation in vitro and in vivo, with xenograft models showing reduced tumor growth and Ki-67 expression. RNA-seq analysis linked RGC32 depletion to downregulation of cell proliferation and impaired DNA damage repair (DDR) mechanisms. Western blot showed RGC32 knockdown could suppress ATM/ATR/CHK1 pathway and increase the tumor mutational burden (TMB). Furthermore, after inhibition of RGC32, infiltration of CD8+ T cells was increased in DLBCL tumor microenvironment (TME). Conclusions This study highlights that RGC32 is a novel molecule in DLBCL progression and might be a potential therapeutic target for DLBCL therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78dc8ef431c9b4ad9b48222a8eef16b0fca96480" target='_blank'>
              Response gene to complement 32 promotes tumorigenesis by mediating DNA damage repair and inhibits CD8+ T cells infiltration in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Xiyuan Zhang, T. Lu, Chunlei Shi, M. Ding, Ling Wang, Xinting Hu, Xin Wang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While immunotherapeutic targeting of cell surface proteins is an increasingly effective cancer therapy, identification of new surface proteins, particularly those with biological importance, is critical. Here, we uncover delta-like non-canonical Notch ligand 1 (DLK1) as a cell surface protein with limited normal tissue expression and high expression in multiple refractory adult metastatic cancers including small cell lung cancer (SCLC) and adrenocortical carcinoma (ACC), a rare cancer with few effective therapies. In ACC, ADCT-701, a DLK1 targeting antibody-drug conjugate (ADC), shows in vitro and in vivo activity but is overall limited due to high expression and activity of the drug efflux protein ABCB1 (MDR1, P-glycoprotein). In contrast, ADCT-701 induces complete responses in DLK1+ ACC and SCLC in vivo models with low or no ABCB1 expression. Genetic deletion of DLK1 in ACC dramatically downregulates ABCB1 and increases ADC payload and chemotherapy sensitivity through NOTCH1-mediated transdifferentiation. This work identifies DLK1 as an immunotherapeutic target that regulates tumor cell plasticity and chemoresistance in ACC and supports an active phase I clinical trial targeting DLK1 with an ADC in ACC and neuroendocrine neoplasms (NCT06041516).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254284cf5e9a69169bbda3e651fa63c7ad31b13e" target='_blank'>
              Identification of the Notch ligand DLK1 as an immunotherapeutic target and regulator of tumor cell plasticity and chemoresistance in adrenocortical carcinoma
              </a>
            </td>
          <td>
            Nai-Yun Sun, Suresh Kumar, Yoo Sun Kim, D. Varghese, Arnulfo Mendoza, Rosa Nguyen, Reona Okada, Karlyne Reilly, Brigitte C Widemann, Y. Pommier, Fathi Elloumi, A. Dhall, Daiki Taniyama, Mayank Patel, Etan Aber, Cristina F Contreras, Rosandra N Kaplan, K. Kiseljak-Vassiliades, M. Wierman, Dan Martinez, J. Pogoriler, Amber K. Hamilton, S. Diskin, John M. Maris, RW Robey, Michael M. Gottesman, Jaydira Del Rivero, Nitin Roper
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive subtype, typically requiring neoadjuvant chemotherapy (NAC) as an obligatory component of the treatment regimen. Achieving a pathological complete response to NAC is associated with improved long-term outcomes for patients with TNBC. The functional status of the immune system plays a critical role in NAC efficacy. Herein, we presented the investigation of systemic and local immune landscape during the initial course of NAC treatment and identify factors that contribute to chemotherapy resistance of TNBC. Using single-cell RNA sequencing, we demonstrated that the transcriptional profile remained stable in a patient who responded to NAC, while a non-responder exhibited significant dysregulation in the expression of genes involved in stress response, apoptosis, immune cell proliferation, and differentiation within lymphocyte and monocyte populations. During the first course of NAC, circulating cytotoxic CD8 T cells in the non-responder patient overexpressed granzymes B and H, granulysin, and perforin. In contrast, expression of these factors decreased in CD8 T cells within the tumor. Finally, we identified for a first time a signature of myeloid-derived suppressor cells (MDSC) within the S100АhighMHClow monocyte population and calculated an MDSC score for both the responder and the non-responder TNBC patients. An elevated MDSC score in the non-responder was validated using data from an independent cohort of patients with poor NAC response. Our data underscores the importance of immune system functionality in determining chemotherapy efficacy in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3928ea132c8b634271958e7b89fc6aedf77c3fc" target='_blank'>
              Single-cell transcriptomic profiling of immune landscape in triple-negative breast cancer during neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            M. Patysheva, P. Iamshchikov, A. A. Fedorenko, O. Bragina, M. A. Vostrikova, E. Garbukov, N. Cherdyntseva, E. V. Denisov, T. Gerashchenko
          </td>
          <td>2025-07-07</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite considerable advances in cancer treatment, gastric cancer (GC) remains a formidable challenge for oncologists worldwide, especially due to the poor survival rates associated with advanced-stage cases. Circular RNAs (circRNAs) stand out as potential targets for more effective therapeutic strategies. The present review synthesizes insights into the roles of circRNAs in GC, highlighting their multifaceted influence on cancer progression and behaviors. circRNAs can regulate gene expression at multiple levels through modulating transcription, affecting alternative splicing, acting as molecular sponges for microRNAs, serving as RNA-protein complexes and even encoding functional proteins. The marked stability of circRNAs in bodily fluids has also positioned them as promising diagnostic biomarkers, with some circRNA-based tests demonstrating high accuracy. Furthermore, emerging evidence indicates that circRNAs carry out a key role in therapy resistance, affecting the therapeutic responses of patients to chemotherapy, targeted therapy and immunotherapy. Collectively, circRNA-based therapeutic strategies, even with existing challenges in delivery methods, hold considerable promise, particularly when integrated with conventional treatment modalities, offering new avenues for improving GC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684411f0ed837898b4015fd4ed14022804ffc8a5" target='_blank'>
              Emerging role of circular RNAs in gastric cancer: From basic biology to clinical applications (Review)
              </a>
            </td>
          <td>
            Linkun Wang, Peng Zhou, H. Qu, Zhi-hui Yang, Yuyang Li, Nan Wang, Jiazeng Xia
          </td>
          <td>2025-07-08</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Mutational signatures are increasingly used to understand the mechanisms causing cancer. However, their important applications in predicting prognosis and stratifying patients for therapy are hampered by inaccurate inference of the various featureless, dense trinucleotide mutational spectra, which are often confounded with one another. One of them is the homologous recombination deficiency (HRd)-associated signature SBS3, relevant because of its association with prognosis in ovarian and breast cancer and because of its potential as a biomarker for synthetic lethality therapies. Methods Here, we highlight strong benefits of a multimodal approach for mutational signature extraction, applied on top of standard bioinformatic pipelines. By jointly operating on single-base substitution (SBS) and indel (ID) spectra, this method enables accurate identification of various DNA repair deficiency signatures and patient survival prediction. Results Across four different cohorts of whole-genome sequenced high-grade serous ovarian cancers (HGSOC), the multimodal SBS + ID approach correctly distinguished the commonly confused signatures SBS3, SBS5, SBS8, SBS39, and SBS40. Importantly, we robustly identified two different multimodal SBS3 signatures, m-SBS3a and m-SBS3b, with distinct patterns in the indel spectrum. Multimodal SBS3b signature was strongly predictive of longer survival in ovarian cancer patients, replicating across four cohorts, with effect sizes greatly exceeding other genetic markers. Our m-SBS3 also predicted survival in platinum-treated patients with various cancer types, and moreover, the SBS + ID joint inference was successfully applied to mismatch repair-deficient colorectal cancer and immunotherapy response, supporting a general utility of the multimodal mutational signatures approach. Conclusions Overall, combining SBS and ID mutations improves detection of HR deficiency-associated signatures and reveals distinct SBS3 subtypes with prognostic value. This multimodal approach outperforms existing markers and is readily applicable to therapy stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01497-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01c11401f870f4aa8e6266e2fe8e3ab75a0da51" target='_blank'>
              Joint inference of mutational signatures from indels and single-nucleotide substitutions reveals prognostic impact of DNA repair deficiencies
              </a>
            </td>
          <td>
            Patricia Ferrer-Torres, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-07-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Background: Patients exhibiting exceptional response to targeted treatments represent a unique opportunity towards identifying predictive biomarkers of response. The introduction of edits into the genome using Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) creates cancer models that can unravel the mechanisms driving tumorigenesis and response to drug agents. The study aimed to identify candidate molecular alterations associated with exceptional response to CDK inhibitors (CDKi) and validate their effects using genome editing. Methods: Patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy at Departments of Oncology-affiliated with the Hellenic Cooperative Oncology Group (HeCOG), were included. Exceptional response was defined as either complete response (CR) to the treatment regimen or partial response (PR) for 3 years, per physician’s assessment. Formalin-fixed paraffin-embedded (FFPE) tumor tissue samples, obtained prior to CDKi administration from these patients were retrospectively collected. Whole-exome sequencing (WES) was performed to analyze tumor mutational landscape of exceptional responders to CDKi. Using CRISPR-mediated genome editing, a breast cancer cell model harboring a patient-specific SNP and the negative control were prepared, and subsequently evaluated for their responsiveness to palbociclib. Transcriptomic and proteomic analyses were conducted to uncover alterations in network interactions, between the two cell lines, elucidating the exceptional response to CDKi. Results: Overall, 12 women identified as exceptional responders, initiated CDKi treatment from March 2017 to February 2020; median age at diagnosis was 46 (range, 37 to 78). All patients had received the CDKi as first-line treatment; most commonly palbociclib (10 patients, 83.3%). Based on physician assessment, 50% of patients achieved CR with the combination therapy. After a median follow up of 65 months, 10 patients are still alive, and 7 are still receiving treatment with CDKi in combination with endocrine treatment. WES results with adequate mapped reads and acceptable technical characteristics in both tumor and control blood samples were obtained from 6 of 12 patients. A set of germline and somatic single nucleotide polymorphisms (SNPs) commonly shared in this cohort was identified. The focus was on MINDY1, a member of the MINDY family of deubiquitinases (DUBs), known as a deubiquitylase of ERα in breast cancer, the common SNP among all 6 patients. Utilizing CRISPR-based genome editing, a breast cancer cell model harboring a specific MINDY1 SNP was engineered, and their response to palbociclib was subsequently evaluated. Depletion of MINDY1 transcripts through RNA interference with short-hairpin RNA, increased sensitivity to palbociclib in both cell lines, confirming that MINDY1 protein is involved in molecular pathways related to palbociclib response. The functional role of MINDY1 rs771205 was also investigated by combining transcriptomic and proteomic data. Our data demonstrated that the MINDY1 rs771205 SNP, which causes an amino acid substitution, while it does not impact the respective transcript and protein levels, alters the MINDY1 protein’s interaction network. Conclusions: The Rs771205 single mutation alters the MINDY1 protein’s interaction network, sensitizing cells to palbociclib. Our study sheds new light on the underlying mechanisms driving exceptional response in patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy , demonstrating that it is an opportune moment to invest attention and resources towards probing the underlying mechanisms of exceptional response in cancer patients.
 Citation Format: Eleni Balla, Yi Li, Kyriaki Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, Eleni Galani, Rania Romanidou, George Fountzilas, Evangelia Razis, Angelos Koutras, Leonidas Bleris, Elena Fountzilas. MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-03-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea823fc49d305c67d0c88bebfe1f1411182d2dc" target='_blank'>
              Abstract P4-03-26: MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer
              </a>
            </td>
          <td>
            Eleni Balla, Yi Li, K. Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, E. Galani, Rania Romanidou, G. Fountzilas, E. Razis, A. Koutras, L. Bleris, E. Fountzilas
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Bladder cancer (BC) displays a huge phenotypic variation and widespread clinical outcomes, attributed to the high mutational heterogeneity of the disease. Mutational landscaping became integral to cancer patient management as it unveils driver genes and yields genotype–phenotype associations. This study aims to identify somatic mutations, their frequencies and their associations with clinical and prognostic outcomes in BC. Eighty-one BC patients were analyzed by next-generation sequencing using the Ion AmpliSeq Cancer Hotspots Panel v2. Bioinformatics analysis, correlation studies and Kaplan–Meier curve were used to evaluate the relationship between genes’ mutational status and patients’ clinical parameters and outcomes. Our results indicated that the BC cohort exhibited a higher mutation burden than the TCGA data. Mutations were identified in 46 out of 50 genes, including 21 novel mutations not previously reported in BC. The TP53 gene was mutated in 82.5% of the analyzed cohort, followed by PIK3 CA (45%), FGFR3 (43.75%) and APC (35%). TP53 mutations were associated with poor survival (p = 0.003) while the FGFR3 mutation group exhibited signs of good prognosis (p = 0.018). Bioinformatics highlighted significant gene interactions associated with poor prognosis. These findings underline the importance of identifying novel genetic mutations that could significantly improve prognostic stratification and expand therapeutic options for managing BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def8cff968adcdbff8ae094040404982997ef901" target='_blank'>
              Clinical relevance of the somatic mutational landscaping in predicting outcome of bladder cancer patients
              </a>
            </td>
          <td>
            T. Nedjadi, H. Ansari, Muhammad A Khan, Naif S Sannan, Mubarak M. Al-Mansour, J. Al-Maghrabi, Ashraf Dallol
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c594286b8ee68137b34bcacb0dd61d37f1113d7" target='_blank'>
              Noncoding RNAs as mechanistic regulators and therapeutic modulators of YAP/TAZ signaling in colorectal cancer.
              </a>
            </td>
          <td>
            Mehrnaz Mostafavi, Elham Dehbashi, Faraz Rahmani Khajeh, Masoumeh Nouri, Mahdis Kadkhodaei Eliaderani, Nahal Aghajamal Avval, Moein Ghasemi, Dina Seyedi, Malihe Sharafi, Reza Morovatshoar, Sepide Javankiani, Alireza Azani, Farzad Sadri, Qumars Behfar
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b94cf63206b221ede960c719c7b5d998ebadf2b" target='_blank'>
              Specific TP53 mutations impair the recruitment of 53BP1 to DNA double-strand breaks underlying the mechanism of radioresistance.
              </a>
            </td>
          <td>
            Paolo Fagherazzi, L. Stixová, E. Bártová
          </td>
          <td>2025-07-14</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2968d582c4d09600f6a6aab6083cc272395bfdb6" target='_blank'>
              MicroRNA-driven organ-specific metastasis in triple-negative breast cancer: biogenesis, mechanisms, and therapeutic approaches.
              </a>
            </td>
          <td>
            Deepshikha Rathore, Sweny Jain, Shanaya Patel, Nandani Dharwal, N. Shukla, Chirag Desai, Jigna Shah, Heena V. Dave
          </td>
          <td>2025-06-28</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3997846e172a52e0436c7851168e81909eb612" target='_blank'>
              The histone-methyltransferase DOT1L cooperates with LSD1 to control cell division in blast-phase MPN.
              </a>
            </td>
          <td>
            Karl Kapahnke, Thomas Plenge, Tabea Klaus, Manoj K Gupta, Disha Anand, Tamer T. Önder, Birgit Perner, T. Schnöder, Felicitas R Thol, Frederik Damm, F. Heidel, Florian Perner
          </td>
          <td>2025-08-08</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary The drug romidepsin, which is known to regulate gene expression, has been shown to be an effective treatment in a subset of patients with cutaneous T-cell lymphoma (CTCL). We examined gene expression changes that occurred through repeated romidepsin treatments using carefully collected blood cell samples from treated CTCL patients. Focusing on data from a highly responsive CTCL patient through 12 months of treatment, including a period free from disease, we identified new markers that are characteristic of the malignant cells and difficult-to-detect residual disease. We identified effected cell functions and potential mechanisms of dysregulation through changes in miRNA expression and DNA methylation patterns. We also found a significant impact of transcriptional regulators HLF (activator) and NFIL3 (suppressor) that control the expression of malignant-specific genes. We report new surface markers for the malignant cells, including markers identifiable in residual disease, and new potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04dc922afbb3e523da20705b085778383003e9ce" target='_blank'>
              Cutaneous T-Cell Lymphoma: Yin-Yang Effects of Transcription Factors HLF and NFIL3 in Regulation of Malignant T-Cell Markers in the Context of HDAC Inhibitor Romidepsin Treatment
              </a>
            </td>
          <td>
            Andrew V. Kossenkov, Noor Dawany, Sonali Majumdar, Celia Chang, C. Nichols, M. Wysocka, R. Piekarz, M. Showe, Susan E. Bates, A. Rook, Ellen J. Kim, Louise C. Showe
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31620c8deab30d15f98e2b8cf7eeeade98239959" target='_blank'>
              LINC01410 in cancer: a comprehensive review of its oncogenic role, regulatory mechanisms, and clinical implications.
              </a>
            </td>
          <td>
            Saman Morovat, Pejman Morovat, Ashkan Kalantary-Charvadeh, Marzieh Mojbafan, S. Teimourian
          </td>
          <td>2025-07-12</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc739b29aa041a82d43e91798c3e41caf831aa6b" target='_blank'>
              Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants
              </a>
            </td>
          <td>
            Ethan W. Hollingsworth, Zhuoxin Chen, Cindy X. Chen, Sandra H. Jacinto, Taryn A Liu, Evgeny Z. Kvon
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f43f79f58977259121120764f54b519011ee5e" target='_blank'>
              RBM5 recruiting MGC32805 in a sandwich mode and inducing ΔFAS neoantigen and triggering FAS properties switch: implication in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhe Wu, Xiaoyun Hu, Yilin Wang, Xianglong Zhu, Qing Zhao, Ying Zhao, Wanlin Cui, Mingrong Zhang, Haishan Zhao, Kai Li, Minjie Wei
          </td>
          <td>2025-06-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Chemicals reactivating epigenetically silenced genes may cause strong transcriptome changes inducing both beneficial and hazardous effects, in particular, promoting carcinogenesis. Among chemicals used in medicine and commerce, only a small fraction has been studied with respect to their influence on epigenetic regulation. Screening of non-genotoxic chemicals reactivating silent genes requires adequate test-systems mimicking whole-genome processes. In our study we used a HeLa TSA-Inducible cell population (HeLa TI) harboring epigenetically silenced chromosomally dispersed GFP-reporter genes. This system is based on the principle that epigenetic gene silencing is reversible, and that a silent reporter gene can be reactivated by inhibition of key epigenetic silencing factors. 15 chromatin-based epigenetic silencing factors were shown to participate in GFP silencing in these cells. In our study we demonstrated that different epigenetically active compounds, including six HDAC inhibitors, three DNMT inhibitors; three HMT inhibitors, two BET inhibitors and chromatin remodeler–curaxin CBL0137 are able to activate GFP expression. We also showed that combinations of epigenetic modulators caused a significant increase in the number of cells with reactivated GFP compared to the individual effects of each agent. HeLa TI cells are competent to metabolize xenobiotics and possess constitutively expressed and inducible cytochrome P450 monooxygenases metabolizing xenobiotics. Thus, HeLa TI cells may be used as an adequate test system for the extensive screening of epigenetically active chemicals, including those that must be metabolically activated. Further approaches to improve the test-system for the screening of epigenetically active carcinogens including the use of new fluorophores, different cell lines and cell signaling analysis will be also discussed.



 Kirsanov K.I., Maksimova V.P., Vlasova O.A., Zhidkova E.M., Zenkov R.G., Lylova E.S., Karpechenko N.Y., Osipova A.V., Grigorieva D.D., Popova V.G., Fetisov T.I., Borunova A.A., Zabotina T.N., Belitsky G.A., Yakubovskaya M.G.. Sorted cell populations with epigenetically silenced chromosomally dispersed reporter fluorophore gene as a test-system for the screening of non-genotoxic carcinogens [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P84.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b8055dda1f803f37b6f92aca70aff4f30e51902" target='_blank'>
              Abstract P84: Sorted cell populations with epigenetically silenced chromosomally dispersed reporter fluorophore gene as a test-system for the screening of non-genotoxic carcinogens
              </a>
            </td>
          <td>
            K. Kirsanov, V. Maksimova, O. Vlasova, E. Zhidkova, R. Zenkov, E. Lylova, N.Y. Karpechenko, A.V. Osipova, D. D. Grigorieva, V. Popova, T. I. Fetisov, A. Borunova, T. Zabotina, G. A. Belitsky, M. Yakubovskaya
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aging is a complex biological process characterized by a gradual decline in cellular and physiological function, increasing vulnerability to chronic diseases and mortality. It involves a set of interconnected mechanisms known as the hallmarks of aging, including genomic instability, telomere attrition, epigenetic alterations, loss of proteostasis, mitochondrial dysfunction, cellular senescence, stem cell exhaustion, altered intercellular communication, and dysregulated nutrient sensing. These processes act at molecular, cellular, and systemic levels, contributing to age-related disorders such as neurodegeneration, cardiovascular disease, and metabolic syndromes. Emerging therapeutic strategies aim to delay or reverse aging by targeting specific hallmarks. These include senolytics to eliminate senescent cells, NAD+ boosters and mitophagy inducers to improve mitochondrial health, epigenetic reprogramming, and caloric restriction mimetics such as metformin and rapamycin to modulate nutrient-sensing pathways. Advances in regenerative medicine, gene editing, and organ cross-talk modulation are also contributing to the development of personalized, multi-targeted anti-aging therapies. Integration of omics technologies and biomarker research is expected to enhance our ability to monitor biological aging and optimize interventions for healthy longevity. This review highlights the current understanding of the hallmarks of aging and explores potential treatment strategies in light of our recent findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e00b3dd3b432a8da2e03290cf422ea07fd98916" target='_blank'>
              Targeting the hallmarks of aging: mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Fumihiro Sanada, Shinichiro Hayashi, R. Morishita
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Cardiovascular Medicine</td>
          <td>0</td>
          <td>92</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [2],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>