<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-24 09:28:10 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>29</td>
          <td>42</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) have demonstrated remarkable efficacy in treating non-small cell lung cancer (NSCLC), but acquired resistance greatly reduces efficacy and poses a significant challenge to patients. While numerous studies have investigated the mechanisms underlying EGFR-TKI resistance, its complexity and diversity make the existing understanding still incomplete. Traditional approaches frequently struggle to adequately reveal the process of drug resistance development through mean value analysis at the overall cellular level. In recent years, the rapid development of single-cell RNA sequencing technology has introduced a transformative method for analyzing gene expression changes within tumor cells at a single-cell resolution. It not only deepens our understanding of the tumor microenvironment and cellular heterogeneity associated with EGFR-TKI resistance but also identifies potential biomarkers of resistance. In this review, we highlight the critical role of single-cell RNA sequencing in lung cancer research, with a particular focus on its application to exploring the mechanisms of EGFR-TKI-acquired resistance in NSCLC. We emphasize its potential for elucidating the complexity of drug resistance mechanism and its promise in informing more precise and personalized treatment strategies. Ultimately, this approach aims to advance NSCLC treatment toward a new era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57a8423fcdeff9dbe3c23c8eb5bf687e8a733ce" target='_blank'>
              Single-Cell RNA Sequencing in Unraveling Acquired Resistance to EGFR-TKIs in Non-Small Cell Lung Cancer: New Perspectives
              </a>
            </td>
          <td>
            Lin Peng, Siyou Deng, Jinjie Li, Yujie Zhang, Li Zhang
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Objective Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance. Methods Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation. Results We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation. Conclusion This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af44a1fa5b9e22f43f29a690d71566f240ee00b4" target='_blank'>
              Exploring the potential mechanisms of sorafenib resistance in hepatocellular carcinoma cell lines based on RNA sequencing
              </a>
            </td>
          <td>
            Minghui Sun, Zhi Zhang, Chunyan Chen, Juan Zhong, Zhongrong Long, Ling Shen, Hai Huang, Jianxun Lu
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, Krishnan K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064037ccaeb8393d522fb1895074e0d8f9cf70c8" target='_blank'>
              Integrated Genomics Reveal Potential Resistance Mechanisms of PANoptosis-Associated Genes in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Cong Liang, Zhiqing Long, Mengjie Lei, Ran Ding, Mengke Chen
          </td>
          <td>2025-01-27</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a major cause of cancer-related mortality worldwide, with a significant impact on public health. Current treatment options include surgery, chemotherapy, radiotherapy, molecular-targeted therapy, and immunotherapy. Despite advancements in these therapeutic modalities, resistance remains a significant challenge, often leading to treatment failure, poor progression-free survival, and cancer recurrence. Mechanisms of resistance in CRC are multifaceted, involving genetic mutations, epigenetic alterations, tumor heterogeneity, and the tumor microenvironment. Understanding these mechanisms at the molecular level is crucial for identifying novel therapeutic targets and developing strategies to overcome resistance. This review provides an overview of the diverse mechanisms driving drug resistance in sporadic CRC and discusses strategies currently under investigation to counteract this resistance. Several promising strategies are being explored, including targeting drug transport, key signaling pathways, DNA damage response, cell death pathways, epigenetic modifications, cancer stem cells, and the tumor microenvironment. The integration of emerging therapeutic approaches that target resistance mechanisms aims to enhance the efficacy of current CRC treatments and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f0190dfc671312c21a6e2fa1ac0dcb0cf54adc6" target='_blank'>
              Mechanisms and Strategies to Overcome Drug Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Jennifer Haynes, Prasath Manogaran
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8773538db860e37609b781e5433e56f0729eca07" target='_blank'>
              Large-scale CRISPRi screens link metabolic stress to glioblastoma chemoresistance
              </a>
            </td>
          <td>
            Xing Li, Wansong Zhang, Yitong Fang, Tianhu Sun, Jian Chen, Ruilin Tian
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd49383fc8e846f3a7bdc91c9cc99940e4850c1" target='_blank'>
              Innovative logic "AND" gate gene circuits for bladder cancer treatment: preclinical study.
              </a>
            </td>
          <td>
            Chaojie Xu, Ying Dong, Dongchen Pei, Xintao Zhang, Xiaohong Han, Congcong Cao, Baorui Wu, Changning Lv, Zhengjun Kang, Liqun Zhou, Yuchen Liu, Lin Yao
          </td>
          <td>2025-02-04</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd64f2509541e7332b94c3a04278ee5d7e930fb" target='_blank'>
              Molecular principles underlying aggressive cancers
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-02-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>3</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0203e20c6bc4f26349dbeefa5213e0b374637b7e" target='_blank'>
              Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer
              </a>
            </td>
          <td>
            Slim Mzoughi, Megan Schwarz, Xuedi Wang, D. Demircioglu, Gulay Ulukaya, Kevin Mohammed, Habiba Zorgati, Denis Torre, Lewis E. Tomalin, Federico Di Tullio, Carlos Company, Yuliia Dramaretska, M. Leushacke, Bruno Giotti, T. Lannagan, D. Lozano-Ojalvo, P. Karras, Peter B Vermeulen, Dan Hasson, Robert P Sebra, A. Tsankov, Owen J. Sansom, J. Marine, Nick Barker, G. Gargiulo, Ernesto Guccione
          </td>
          <td>2025-02-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149c84e6fa83110753a75063447fffdfde31626" target='_blank'>
              Differential DNA methylation in response to host environment changes in Drosophila gouveai.
              </a>
            </td>
          <td>
            Adriano S Santos, E. S. Ramos, V. L. S. Valente, M. Manfrin
          </td>
          <td>2025-03-08</td>
          <td>Genetica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In recent years, significant advancements in biochemistry, materials science, engineering, and computer-aided testing have driven the development of high-throughput tools for profiling genetic information. Single-cell RNA sequencing (scRNA-seq) technologies have established themselves as key tools for dissecting genetic sequences at the level of single cells. These technologies reveal cellular diversity and allow for the exploration of cell states and transformations with exceptional resolution. Unlike bulk sequencing, which provides population-averaged data, scRNA-seq can detect cell subtypes or gene expression variations that would otherwise be overlooked. However, a key limitation of scRNA-seq is its inability to preserve spatial information about the RNA transcriptome, as the process requires tissue dissociation and cell isolation. Spatial transcriptomics is a pivotal advancement in medical biotechnology, facilitating the identification of molecules such as RNA in their original spatial context within tissue sections at the single-cell level. This capability offers a substantial advantage over traditional single-cell sequencing techniques. Spatial transcriptomics offers valuable insights into a wide range of biomedical fields, including neurology, embryology, cancer research, immunology, and histology. This review highlights single-cell sequencing approaches, recent technological developments, associated challenges, various techniques for expression data analysis, and their applications in disciplines such as cancer research, microbiology, neuroscience, reproductive biology, and immunology. It highlights the critical role of single-cell sequencing tools in characterizing the dynamic nature of individual cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8242b8eb0f45c6f5b5cf16b8ec7b8ce82d36e19b" target='_blank'>
              Advancements in single-cell RNA sequencing and spatial transcriptomics: transforming biomedical research
              </a>
            </td>
          <td>
            Getnet Molla Desta, A. Birhanu
          </td>
          <td>2025-02-05</td>
          <td>Acta Biochimica Polonica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="The classical paradigm of drug screening often faces significant limitations due to the challenges associated with identifying molecular or cellular read-outs that are relevant to specific diseases. This issue is particularly pronounced for the thousands of diseases of genetic origin, given the abundance of databases listing disease-associated changes in gene expression and various transcripts, indicating that potential read-outs may be concealed within these resources. To remedy this, an alternative approach was tested: compounds were evaluated for their effects on gene expression and alternative splicing in a healthy cell model, and the resulting data were matched to molecular signatures of diseases. A subset of 50 FDA-approved drugs was tested on mesenchymal stem cells derived from a human pluripotent stem cell line. Over half of the compounds altered gene expression, many affecting pathways linked to monogenic diseases. One hit, increased SQSTM1 expression induced by prazosin, was further validated in ALS models caused by SQSTM1 haploinsufficiency, including patient-derived fibroblasts, SQSTM1-depleted hiPSC-derived motor neurons, and a zebrafish model. Extending this paradigm could involve testing diverse cell types and larger drug libraries. One Sentence Summary A novel drug screening approach for monogenic diseases integrating human pluripotent stem cell derivatives and RNA sequencing to profile gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2756ee22739ee10642a0c818c71c07402f6f6f2e" target='_blank'>
              Integrative Analysis of Drug-Gene Expression Signatures in Human Pluripotent Stem Cells Identifies Novel Drug Candidates for ALS and Monogenic Diseases
              </a>
            </td>
          <td>
            Florine Roussange, Jacqueline Gide, Johana Tournois, Michel Cailleret, A. Boland, Christophe Battail, J. Deleuze, Hélène Polvèche, Didier Auboeuf, Knut Brockmann, E. Kabashi, A. Marian, Lina El Kassar, Sophie Blondel, François Salachas, G. Bruneteau, Marc Peschanski, C. Martinat, Sandrine Baghdoyan
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/611724b4911f171cfc8e55580dc1cddab6f13349" target='_blank'>
              Early tolerance and late persistence as alternative drug responses in cancer
              </a>
            </td>
          <td>
            S. Punzi, D. Cittaro, Guido Gatti, Gemma Crupi, O. Botrugno, Antonino Alex Cartalemi, Alon Gutfreund, C. Oneto, Valentina Giansanti, Chiara Battistini, Giovanni Santacatterina, Lucrezia Patruno, Ilaria Villanti, Martina Palumbo, D. Laverty, Francesca Giannese, Alex Graudenzi, G. Caravagna, M. Antoniotti, Zachary D. Nagel, Ugo Cavallaro, L. Lanfrancone, T.A. Yap, Giulio Draetta, Nathalie Q Balaban, Giovanni Tonon
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df97a5ece885215c844356881d36483043d3e173" target='_blank'>
              RNA sequencing identifies MAP1A and PTTG1 as predictive genes of aging CD264+ human mesenchymal stem cells at an early passage
              </a>
            </td>
          <td>
            Margaret K. Giler, H. A. Tucker, Amanda K. Foote, Avery G. Francis, Sean D. Madsen, Yao-Zhong Liu, Kim C. O’Connor
          </td>
          <td>2025-02-19</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="InfoScan is a novel bioinformatics tool designed for the comprehensive analysis of full-length single-cell RNA sequencing (scRNA-seq) data. It enables the identification of unannotated transcripts and rare cell populations, providing a powerful platform for transcriptome characterization. In this study, InfoScan was applied to glioblastoma multiforme (GBM), identifying a rare “neoplastic-stemness” subpopulation exhibiting cancer stem cell-like features. Functional analyses suggested that tumor-associated macrophages (TAMs) secrete SPP1, which binds to CD44 on neoplastic-stemness cells, activating the PI3K/AKT pathway and driving lncRNA transcription to promote metastasis. Integration of TCGA and CGGA datasets further supported these findings, highlighting key mutations associated with the neoplastic-stemness subpopulation. Drug sensitivity assays indicated that neoplastic-stemness cells might be sensitive to omipalisib, a PI3K inhibitor, pointing to a potential therapeutic target. InfoScan offers a robust framework for exploring complex transcriptomic landscapes and characterizing rare cell populations, providing valuable insights into GBM biology and advancing precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d19fc728a932329a1c6b81702c24f21513cb2f5" target='_blank'>
              InfoScan: A New Transcript Identification Tool Based on scRNA-Seq and Its Application in Glioblastoma
              </a>
            </td>
          <td>
            Shiqiang Mei, Jinjin Huang, Zhen Zhang, Haotian Lei, Qiaojuan Huang, Lianghu Qu, Lingling Zheng
          </td>
          <td>2025-02-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Methylation processes in different molecular contexts (DNA, RNA, and histones) are controlled by different regulatory factors and serve as critical determinants in cancer development. However, the mechanistic links between these epigenetic modifications during malignant transformation, metastasis, disease relapse, and therapeutic resistance remain incompletely understood. In this research, we investigated the transcriptional and genetic alterations of regulators associated with 3 major types of methylation modifications in clear cell renal cell carcinoma. Utilizing ChIP/MeRIP-seq and 450K methylation array data, we identified genes regulated by multiple methylation modifications and constructed a scoring model to quantify the methylation patterns for each patient. Our findings indicate that patients with a low score may be more likely to respond to immunotherapy, whereas patients with a high score may be more sensitive to targeted therapy, such as RITA, Pazopanib, Irlotinib, SU-11274, BRD-K16762525, and FCCP. In conclusion, the score model can serve as a valuable biomarker to guide clinical selection of immunotherapy and targeted drugs and help to improve personalized clear cell renal cell carcinoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4691220e407b79a42a6b06dcf7ad89ec71062f" target='_blank'>
              Investigating the efficacy of immune checkpoint inhibitors in clear cell renal cell carcinoma based on methylation cross talk scoring
              </a>
            </td>
          <td>
            Qinglong Du, Qiyuan Wang, Chen Yang, Yiping Wang, Huiyang Yuan, Bing Zhang, Hong Ji, Shuai Fu, Chunlei Xue
          </td>
          <td>2025-03-14</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4672987b5633b49ff2536422175c3660e10c3a" target='_blank'>
              Identification of a shared persistence program in triple-negative breast cancer across treatments and patients
              </a>
            </td>
          <td>
            Léa Baudre, Grégoire Jouault, Pacôme Prompsy, Mélissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Aneuploidy is prevalent in cancer and associates with fitness advantage and poor patient prognosis. Yet, experimentally induced aneuploidy initially leads to adverse effects and impaired proliferation, suggesting that cancer cells must adapt to aneuploidy. We performed in vitro evolution of cells with extra chromosomes and obtained cell lines with improved proliferation and gene expression changes congruent with changes in aneuploid cancers. Integrated analysis of cancer multi-omics data and model cells revealed increased expression of DNA replicative and repair factors, reduced genomic instability, and reduced lysosomal degradation. We identified E2F4 and FOXM1 as transcription factors required for adaptation to aneuploidy in vitro and in cancers and validated this finding. The adaptation to aneuploidy also coincided with specific copy number aberrations that correlate with poor patient prognosis. Chromosomal engineering mimicking these aberrations improved aneuploid cell proliferation, while loss of previously present extra chromosome impaired it. The identified common adaptation strategies suggest replication stress, genomic instability, and lysosomal stress as common liabilities of aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2024-03-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ce7ec0649fbc5d3e5e12bff74054318a2f6bce" target='_blank'>
              MEDUSA for Identifying Death Regulatory Genes in Chemo-genetic Profiling Data.
              </a>
            </td>
          <td>
            Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-02-07</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21cbb2f5aa4a29eac0f638906ce12441220483a" target='_blank'>
              Deciphering single-cell genomic architecture: insights into cellular heterogeneity and regulatory dynamics
              </a>
            </td>
          <td>
            Byunghee Kang, Hyeonji Lee, Tae-Young Roh
          </td>
          <td>2025-02-11</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by expanding blood cell clones carrying somatic mutations in healthy aged individuals and is associated with various age-related diseases and all-cause mortality. While CH mutations affect diverse genes associated with myeloid malignancies, their mechanisms of expansion and disease associations remain poorly understood. We investigate the relationship between clonal fitness and clinical outcomes by integrating data from three longitudinal aging cohorts (n=713, observations=2,341). We demonstrate pathway-specific fitness advantage and clonal composition influence clonal dynamics. Further, the timing of mutation acquisition is necessary to determine the extent of clonal expansion reached during the host individual’s lifetime. We introduce MACS120, a metric combining mutation context, timing, and variant fitness to predict future clonal growth, outperforming traditional variant allele frequency measurements in predicting clinical outcomes. Our unified analytical framework enables standardized clonal dynamics inference across cohorts, advancing our ability to predict and potentially intervene in CH-related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df776ea20c82ef08bfb6dc73e82dc94bd703a7e7" target='_blank'>
              Clinical progression of clonal hematopoiesis is determined by a combination of mutation timing, fitness, and clonal structure
              </a>
            </td>
          <td>
            Eric Latorre-Crespo, Neil A. Robertson, E. Kosebent, Louise Macgillivray, Lee Murphy, M. Uddin, Eric Whitsel, Michael C. Honigberg, Alex Bick, A. Reiner, Valeria Orru, Michele Marongiu, Francesco Cucca, E. Fiorillo, I. Deary, Sarah Harris, Simon Cox, Riccardo Marioni, Linus Schumacher, Tamir Chandra, Kristina Kirschner
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e7847baa8f0e94557a12876cbaf996f1f4b3e13" target='_blank'>
              scGSDR: Harnessing Gene Semantics for Single-Cell Pharmacological Profiling
              </a>
            </td>
          <td>
            Yu-an Huang, Xiyue Cao, Zhu-Hong You, Yue C. Li, Xue-Qun Shang, Zhi-an Huang
          </td>
          <td>2025-02-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d71f4d600cde047c309be7d9f03d4993b2ab30" target='_blank'>
              EXO1's pan-cancer roles: diagnostic, prognostic, and immunological analyses through bioinformatics
              </a>
            </td>
          <td>
            Zheng Liu, Shu Huang, Rui Luo, Xiaomin Shi, Mingzhu Xiu, Yizhou Wang, Ruiyu Wang, Wei Zhang, Muhan Lv, Xiaowei Tang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Hepatocellular carcinoma is the most prevalent primary liver cancer and arises from hepatocytes, which are very metabolically active and contain abundant mitochondria. Despite extensive characterisation of nuclear driver genes and pathways, the role of crosstalk between the mitochondrial and nuclear genomes in mitochondrial mutagenesis remains poorly understood. To address this, we leveraged a chemical carcinogenesis model of liver cancer in four mouse strains. We analysed hundreds of tumours with paired whole-genome sequencing and total RNA sequencing to comprehensively characterise mitochondrial mutations and expression. Following de novo assembly of three new mitochondrial genomes, we devised an accurate and efficient heteroplasmy detection approach and analysed mitochondrial DNA (mtDNA) content and transcription. Across strains, there was high concordance of the number of heteroplasmies, mutation signatures, and variant allele frequencies. In contrast, the heteroplasmy rate and genic locations differed between strains, suggesting strain-specificity in DNA repair mechanisms in early tumour development. Independent of age or environmental factors, tumours had lower mtDNA content than adjacent normal liver, suggesting mitochondrial loss during tumourigenesis. Additionally, within individual strains, mtDNA content differences were associated with nuclear driver gene choice. Our comprehensive characterisation of mitochondrial genomes and expression patterns demonstrates that both germline and somatic genetic variation influence tumour mtDNA content, mutation patterns, and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7507e40e7369ffa2dd7aab3b39dffb6074ecd240" target='_blank'>
              Germline and somatic variation influence nuclear-mitochondrial crosstalk in tumourigenesis
              </a>
            </td>
          <td>
            Maëlle Daunesse, Vasavi Sundaram, Elissavet Kentepozidou, John Connelly, Craig J. Anderson, Núria López-Bigas, Colin A. Semple, D. T. Odom, Martin S. Taylor, Maša Roller, Sarah J. Aitken, Paul Flicek
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Abstract Epigenetic mechanisms are crucial for developmental programming and can be disrupted by environmental stressors, increasing susceptibility to disease. This has sparked interest in therapies for restoring epigenetic balance, but it remains uncertain whether disordered epigenetic mechanisms can be fully corrected. Disruption of DNA methyltransferase 1 (DNMT1), responsible for DNA methylation maintenance, has particularly devastating biological consequences. Therefore, here we explored if rescuing DNMT1 activity is sufficient to reverse the effects of its loss utilizing mouse embryonic stem cells. However, only partial reversal could be achieved. Extensive changes in DNA methylation, histone modifications, and gene expression were detected, along with transposable element derepression and genomic instability. Reduction of cellular size, complexity, and proliferation rate were observed, as well as lasting effects in germ layer lineages and embryoid bodies. Interestingly, by analyzing the impact on imprinted regions, we uncovered 20 regions exhibiting imprinted-like signatures. Notably, while many permanent effects persisted throughout Dnmt1 inactivation and rescue, others arose from the rescue intervention. Lastly, rescuing DNMT1 after differentiation initiation worsened outcomes, reinforcing the need for early intervention. Our findings highlight the far-reaching functions of DNMT1 and provide valuable perspectives on the repercussions of epigenetic perturbations during early development and the challenges of rescue interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c888744288cbb5c0e8c3ec88a53008ee7b90e2a" target='_blank'>
              Rescuing DNMT1 fails to fully reverse the molecular and functional repercussions of its loss in mouse embryonic stem cells
              </a>
            </td>
          <td>
            Elizabeth Elder, Anthony Lemieux, Lisa-Marie Legault, Maxime Caron, Virginie Bertrand-Lehouillier, Thomas Dupas, Noël J-M Raynal, Guillaume Bourque, Daniel Sinnett, Nicolas Gévry, Serge McGraw
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f68c71be9a5f4d5ac85f15c79176d7904112ca" target='_blank'>
              Long-read single-cell RNA sequencing enables the study of cancer subclone-specific genotypes and phenotypes in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            Gage S. Black, Xiaomeng Huang, Yi Qiao, Philip Moos, Deepa Sampath, Deborah M. Stephens, J. Woyach, Gabor T. Marth
          </td>
          <td>2025-02-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac8334d0db490037d00a83f0b8a5cb44474c730" target='_blank'>
              Single-cell analysis reveals cellular heterogeneity, gene expression profiles, and pathway dynamics in acne vulgaris.
              </a>
            </td>
          <td>
            Xiaoyi Yang, Xiaoyan Wang, Jiankang Yang
          </td>
          <td>2025-02-07</td>
          <td>Archives of dermatological research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48c7f88008ee3f6f5e730cd7155fc1a412d588b" target='_blank'>
              Genomic medicine and personalized treatment: a narrative review
              </a>
            </td>
          <td>
            Adil Khan, Anchal Ramesh Barapatre, Nadir Babar, Joy Doshi, Mohamd Ghaly, Kirtan Ghanshyam Patel, Shayan Nawaz, Uswa Hasana, Swara Punit Khatri, Shilpa Pathange, Abhinya Reddy Pesaru, C. Puvvada, Marium Billoo, Usama Jamil
          </td>
          <td>2025-02-13</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ability to identify gene functions and interactions in specific cellular contexts has been greatly enabled by functional genomics technologies. CRISPR-based genetic screens have proven invaluable in elucidating gene function in mammalian cells. Single-cell functional genomics methods, such as Perturb-seq and Spear-ATAC, have made it possible to achieve high-throughput mapping of the functional effects of gene perturbations by profiling transcriptomes and DNA accessibility, respectively. Combining single-cell chromatin accessibility and transcriptomic data via multiomic approaches has facilitated the discovery of novel cis and gene regulatory interactions. However, pseudobulk readouts from cell populations can often cloud the interpretation of results due to a heterogeneous response from cells receiving the same genetic perturbation, which could be mitigated by using transcriptional profiles of single cells to subset the ATAC-seq data. Existing methods to capture CRISPR guide RNAs to simultaneously assess the impact of genetic perturbations on RNA and ATAC profiles require either cloning of gRNA libraries in specialized vectors or implementing complex protocols with multiple rounds of barcoding. Here, we introduce CAT-ATAC, a technique that adds CRISPR gRNA capture to the existing 10X Genomics Multiome assay, generating paired transcriptome, chromatin accessibility and perturbation identity data from the same individual cells. We demonstrate up to 77% guide capture efficiency for both arrayed and pooled delivery of lentiviral gRNAs in induced pluripotent stem cells (iPSCs) and cancer cell lines. This capability allows us to construct gene regulatory networks (GRNs) in cells under drug and genetic perturbations. By applying CAT-ATAC, we were able to identify a GRN associated with dasatinib resistance, indirectly activated by the HIC2 gene. Using loss of function experiments, we further validated that the gene, ZFPM2, a component of the predicted GRN, also contributes to dasatinib resistance. CAT-ATAC can thus be used to generate high-content multidimensional genotype-phenotype maps to reveal novel gene and cellular interactions and functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5994854c014d82e61919b765b3c023100d1d1160" target='_blank'>
              Simultaneous single-cell CRISPR, RNA, and ATAC-seq enables multiomic CRISPR screens to identify gene regulatory relationships
              </a>
            </td>
          <td>
            Kaivalya Shevade, Yeqing Angela Yang, Kevin Feng, Karl Mader, V. Sevim, Jacob Parsons, Gunisha Arora, Hasnaa Elfawy, Rachel Mace, S. Federman, Rustam Esanov, Shawn Shafer, Eric D Chow, Laralynne Przybyla
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a24af38423fb1cff9abd834b78aff5969e9b847" target='_blank'>
              Establishment and characterization of three gemcitabine-resistant human intrahepatic cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Jiandong Li, Yanxin Hu, Jiayao Zhang, Weiguang Zhang, Jianhua Yu, Baochun Lu
          </td>
          <td>2025-02-09</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Each stage of the Central Dogma contributes to proteome diversity through mechanisms such as heterozygosity, somatic mutations, transcriptional errors, and translational errors. As a result, a diverse array of protein variants can coexist within a single proteome, such as that of humans. However, until now, methods to detect, quantify, and evaluate the functional consequences of these variants have been lacking. Here we examined a large-scale proteogenomic dataset from 29 healthy human tissues and uncovered over 46,000 of unique single amino acid variants co-existing alongside their corresponding reference proteoforms. Our analysis reveals that substitutions preferentially occur in proteins with low expression levels and often affect less frequently encoded amino acids such as tryptophan, cysteine, histidine, or methionine. We found that the abundance of both genetic (SNP’s, somatic mutations) and mistranslated protein variants mirror their allele frequencies in the human population. Moreover, we show that non-genetic substitutions provide a distinct route for exploring protein sequence space, circumventing the mutational constraints imposed by the genetic code. Further, we demonstrate that substitution prevalence correlates negatively with their predicted pathogenicity, particularly in proteins expressed at low levels. We identified hundreds of substituted non-genetic proteoforms that recur consistently in multiple individuals and map to annotated protein functional sites. We propose that these substitutions constitute a novel class of functional protein phenotypic variants. Finally, we illustrate the impact of active-site substitutions in genes such as GAPDH and hemoglobin and highlight potential non-genetic routes of immunoglobulin diversification. Collectively, our findings indicate that non-genetic amino acid substitutions in human proteins provide a recurring and specific route to expanding the functional proteome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31fb45136e53e1b41199e270ecb2a14301ab02a1" target='_blank'>
              Encoded and non-genetic alternative protein variants expand human functional proteome
              </a>
            </td>
          <td>
            Vyacheslav Tretyachenko, T. Leiman, Omer Asraf, Orna Dahan, Dvir Dahary, Y. Pilpel
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fbe09900693cf8126b4ed7a1f742723962eeea9" target='_blank'>
              A comprehensive analysis of scRNA-Seq and RNA-Seq unveils B cell immune suppression in the AAV-loaded brain
              </a>
            </td>
          <td>
            Shunyu Wu, Lu Xue, Xiang Li, Yaoxuan Wang, Yuting Zhu, Yuanbo Luo, Jiayu Sun, Tingting Jin, Wenying Shu, Zhaoyan Wang
          </td>
          <td>2025-03-05</td>
          <td>Immunologic Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a52e6869256b3ec61847f7e51859687b36490d2c" target='_blank'>
              Natural Alternatives in the Treatment of Colorectal Cancer: A Mechanisms Perspective
              </a>
            </td>
          <td>
            K. V. Fernandez-Muñoz, C. A. Sánchez-Barrera, M. Meraz-Ríos, J. L. Reyes, E. Pérez-Yépez, M. T. Ortiz-Melo, Luis I. Terrazas, Mónica G Mendoza-Rodríguez
          </td>
          <td>2025-02-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd57d0a97983513ebb6d083a8255399d3304d8d" target='_blank'>
              Multiplex generation and single-cell analysis of structural variants in mammalian genomes.
              </a>
            </td>
          <td>
            Sudarshan Pinglay, Jean-Benoît Lalanne, R. Daza, Sanjay Kottapalli, Faaiz Quaisar, Jonas Koeppel, Riddhiman K. Garge, Xiaoyi Li, David S Lee, Jay Shendure
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Genome sequencing of cancer and normal tissues, alongside single-cell transcriptomics, continues to produce findings that challenge the idea that cancer is a 'genetic disease', as posited by the somatic mutation theory (SMT). In this prevailing paradigm, tumorigenesis is caused by cancer-driving somatic mutations and clonal expansion. However, results from tumor sequencing, motivated by the genetic paradigm itself, create apparent 'paradoxes' that are not conducive to a pure SMT. But beyond genetic causation, the new results lend credence to old ideas from organismal biology. To resolve inconsistencies between the genetic paradigm of cancer and biological reality, we must complement deep sequencing with deep thinking: embrace formal theory and historicity of biological entities, and (re)consider non-genetic plasticity of cells and tissues. In this Essay, we discuss the concepts of cell state dynamics and tissue fields that emerge from the collective action of genes and of cells in their morphogenetic context, respectively, and how they help explain inconsistencies in the data in the context of SMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b3a10a05d5290d45b66f194c7384d1f7517bb" target='_blank'>
              The end of the genetic paradigm of cancer.
              </a>
            </td>
          <td>
            Sui Huang, Ana M Soto, C. Sonnenschein
          </td>
          <td>2025-03-01</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Biological and dynamic mechanisms by which Drug-tolerant persister (DTP) cells contribute to the development of acquired drug resistance have not been fully elucidated. Here, by integrating multidimensional data from drug-treated PC9 cells, we developed a novel multiscale mathematical model from an evolutionary perspective that encompasses epigenetic and cellular population dynamics. By coupling stochastic simulation with quantitative analysis, we identified epigenetic instability as the most prominent kinetic feature related to the emergence of DTP cell subpopulations and the effectiveness of intermittent treatment. Moreover, we revealed the optimal schedule for intermittent treatment, including the optimal area for therapeutic time and drug holidays. By leveraging single-cell RNA-seq data characterizing the drug tolerance of lung cancer, we validated the predictions made by our model and further revealed previously unrecognized biological features of DTP cells, such as cell autophagy and migration, as well as new biomarker genes of therapeutic tolerance. Our work not only provides a paradigm for the integration of multiscale mathematical models with newly emerging genomics data but also improves our understanding of the crucial roles of DTP cells and offers guidance for developing new intermittent treatment strategies against acquired drug resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b2c7b90ea2e9a9d298ad9d7420ea2b6b5b8ec4" target='_blank'>
              Integrating multiscale mathematical modeling and multidimensional data reveals the effects of epigenetic instability on acquired drug resistance in cancer
              </a>
            </td>
          <td>
            Shun-guan Wang, Jinzhi Lei, Xiufen Zou, Suoqin Jin
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7f5656f1a791fd7b047008e717c5695b95280c" target='_blank'>
              Epigenetics in evolution and adaptation to environmental challenges: pathways for disease prevention and treatment.
              </a>
            </td>
          <td>
            H. Abdolmaleky, S. Nohesara, Jin-Rong Zhou, S. Thiagalingam
          </td>
          <td>2025-02-13</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, A. M. Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive malignant brain tumor with almost inevitable recurrence despite multimodal management with surgical resection and radio-chemotherapy. While several genetic, proteomic, cellular, and anatomic factors interplay to drive recurrence and promote treatment resistance, the epigenetic component remains among the most versatile and heterogeneous of these factors. Herein, the epigenetic landscape of GBM refers to a myriad of modifications and processes that can alter gene expression without altering the genetic code of cancer cells. These processes encompass DNA methylation, histone modification, chromatin remodeling, and non-coding RNA molecules, all of which have been found to be implicated in augmenting the tumor’s aggressive behavior and driving its resistance to therapeutics. This review aims to delve into the underlying interactions that mediate this role for each of these epigenetic components. Further, it discusses the two-way relationship between epigenetic modifications and tumor heterogeneity and plasticity, which are crucial to effectively treat GBM. Finally, we build on the previous characterization of epigenetic modifications and interactions to explore specific targets that have been investigated for the development of promising therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc0fa03a9f64a66bc6beac8c2a3a49bfa95cc71" target='_blank'>
              The epigenetic mechanisms involved in the treatment resistance of glioblastoma
              </a>
            </td>
          <td>
            Aanya Shahani, Hasan Slika, Ahmad Elbeltagy, Alexandra Lee, Christopher Peters, Toriyn Dotson, Divyaansh Raj, Betty Tyler
          </td>
          <td>2025-02-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e945cd17bb6878accfa941b57523d14ed0d229c" target='_blank'>
              Genomic and transcriptomic sequencing in prostate cancer.
              </a>
            </td>
          <td>
            Safiullah Rifai, Azimullah Rifai, Xiaolei Shi, M. A. Khan, Wei Guang, Linbo Wang, Luke Tallon, Arif Hussain
          </td>
          <td>2025-02-28</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006d2b04fc1884e4bfd784cf684ee4d760f680ad" target='_blank'>
              Exploring the tumor suppressor role of RIN1 in familial thyroid carcinoma.
              </a>
            </td>
          <td>
            Luna Picello, Mattia Dalle Nogare, Daniele Puggina, Cecilia Salvoro, G. Pennelli, G. Gaudenzi, S. Carra, Monica Oldani, Davide Gentilini, Laura Fugazzola, A. Faggiano, Giovanni Vitale, Gianluca Occhi, Giovanni Vazza
          </td>
          <td>2025-03-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d109d6041879491cc459a88c72e3556ce8634101" target='_blank'>
              Abstract A040: Survival mechanisms common to immunotherapy and drug tolerant persister tumor cells
              </a>
            </td>
          <td>
            Maria Davern, Cole J Turner, Lara Bencsics, Brenda Chan, Jasmine Yun-Tong Kung, Michael L. Olson, Cheyanne Walker-Williams, David A. Barbie, Shaili Soni, P. Bhola, Marcela V. Maus, Anthony Letai
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a common malignancy with high mortality. Despite advancements in understanding its molecular causes and improved drug therapies, patient survival rates remain low. The main reasons for the high mortality rate are cancer metastasis and the emergence of drug-resistant cancer cell populations. While genetic changes are recognized as the main driver of CRC occurrence and progression, recent studies suggest that epigenetic regulation is a crucial marker in cancer, influencing the interplay between genetics and the environment. Research has shown the significant regulatory roles of non-coding RNAs (ncRNAs) in CRC development. This review explores epigenetically regulated ncRNAs and their functions, aiming to understand key regulatory mechanisms that impact CRC development. Additionally, it discusses the potential use of these ncRNAs in CRC diagnosis, prognosis, and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c88c2a819f1e9676a6d273b84bfd94da9274e4c" target='_blank'>
              Epigenetically Regulating Non-coding RNAs in Colorectal Cancer: Promises and Potentials
              </a>
            </td>
          <td>
            Zahra Taheri, Majid Zaki-Dizaji
          </td>
          <td>2025-01-31</td>
          <td>Middle East Journal of Digestive Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fcf14d7421a285bda69ccae261660da8cc8fcc" target='_blank'>
              Gene Therapy and Gene Editing: Current Trends and Future Prospects of Molecular Medicine.
              </a>
            </td>
          <td>
            Aniruddha Sen, Vijay Singh, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar, Sudhir Shyam Kushwaha
          </td>
          <td>2025-02-18</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Pediatric leukemia is one of the most prevalent childhood malignancies, and its pathogenesis involves complex molecular and genetic mechanisms. Among the critical players in leukemogenesis, the Janus kinase 2 (JAK2) gene has garnered significant attention due to its role in aberrant signal transduction and hematopoietic dysregulation. Mutations such as JAK2 V617F, while more commonly associated with myeloproliferative neoplasms, have also been implicated in pediatric leukemia subtypes, including acute lymphoblastic leukemia (ALL) and acute myeloid leukemia (AML). These mutations result in constitutive activation of the JAK-STAT pathway, promoting unchecked cell proliferation, survival, and resistance to apoptosis. The dysregulation of the JAK2 signaling pathway not only drives malignant transformation but also facilitates interactions with the bone marrow microenvironment, creating a niche that supports leukemia cell survival and immune evasion. Therapeutic advancements have focused on JAK2 inhibitors such as ruxolitinib and fedratinib, which show promise in preclinical and early clinical settings. However, challenges such as drug resistance and off-target effects limit their efficacy, necessitating the exploration of combination therapies and novel drug formulations. Current strategies include combining JAK2 inhibitors with chemotherapy, immune checkpoint inhibitors, or epigenetic modulators to achieve synergistic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8c5bd5328b2c81b8552a0d00878e9d6a8e64d6" target='_blank'>
              JAK2 in pediatric leukemia: mechanisms of pathogenesis and drug development- a narrative review
              </a>
            </td>
          <td>
            E. Obeagu
          </td>
          <td>2025-03-18</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ab64f4c6c0ffff56fec877ba1f8433465976" target='_blank'>
              Single-cell RNA sequencing revealed changes in the tumor microenvironment induced by radiotherapy for cervical cancer and the molecular mechanism of mast cells in immunosuppression.
              </a>
            </td>
          <td>
            L. Xue, Linzhi Gao, Shimin Zhou, Chaofan Yan, Xian Zhang, Wei Lin, Hu Li, Yuan Shen, Xiaoyu Wang
          </td>
          <td>2025-03-14</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposable elements (TEs) are reactivated in tumors and serve as significant contributors to tumor transcriptomic complexity and heterogeneity. However, their repetitive nature and diverse transcriptional forms have hindered efforts to characterize TE-derived transcripts independent of host gene contexts. Here, using our self-developed splicing-junction analysis tool ASJA, we systematically interrogated TE transcription across 32 cancer types, identifying transcripts autonomously initiated from and exclusively spliced among TEs with high TE content, termed TEtrans. Our analysis revealed 5,361 TEtrans exhibiting pan-cancer prevalence, tumor-specific expression and features of mature transcripts, including canonical splicing, 5’ caps and polyA tails. Mostly unannotated and enriched in intergenic regions, TEtrans are predominantly derived from primate-specific classes with conserved splice pattern. TEtrans burden demonstrates heterogeneous associations with prognosis and immune activity across cancers, while their expression can be epigenetically dual-regulated by stemness- and inflammation-associated transcription factors. Functional studies uncovered that TEtrans could act as oncogenes, exemplified by tsTE1, a HERVH-derived transcript that promotes colorectal cancer proliferation by enhancing TOP1-mediated DNA supercoil relaxation. Remarkably, ∼7.2% of TEtrans are identified to encode tumor neoantigens, including viral proteins and unannotated peptides, which are shared among patients and validated by proteogenomic analysis. These TEtrans-derived neoantigens are immunogenic both in vitro and in vivo, and exceed neoepitopes from genomic alterations in abundance per tumor, particularly in cancers with low mutational burden. Collectively, as a gene-independent form of TE-derived transcripts, TEtrans represents a unique source of oncogenes and tumor neoantigens, expanding the functional repertoire of TEs in cancer biology and offering new avenues for therapeutic targeting. Highlights TEtrans are tumor-specific transcripts with conserved splicing patterns, predominantly derived from primate TEs in intergenic regions. TEtrans expression is epigenetically activated and dually regulated by stemness/inflammation transcription factors. TEtrans function as oncogenes, exemplified by tsTE1 enhancing TOP1-mediated DNA relaxation. TEtrans encode tumor neoantigens that elicit CD8+ T cell responses, providing targets for immunotherapy, especially in low-mutation tumors. Abstract figure">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90153db4362d7e931500fbc74364483bb05472a" target='_blank'>
              Transcriptomic splicing analysis reveals a gene-independent form of TE reactivation in tumors representing unrecognized source of oncogenes and neoantigens
              </a>
            </td>
          <td>
            Ziteng Li, Yichao Bao, Qin Li, Hongwu Yu, He-Yun Zhang, Yifan Wen, Yu Yang, Jingjing Zhao, Peng Lin, Yan Li, Zhixiang Hu, Xichun Hu, Xiaodong Zhu, Shenglin Huang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80dc7e24dfe7584b23d2cf124c8124643c955a6e" target='_blank'>
              Role of PGC-1α in the proliferation and metastasis of malignant tumors.
              </a>
            </td>
          <td>
            Tianyi Zhang, Shilei Zhao, C. Gu
          </td>
          <td>2025-01-30</td>
          <td>Journal of molecular histology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            Mohamed J. Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae408f6dbcd8637b77675c5c1327b34b01ef6d58" target='_blank'>
              Genotype-Epigenome-Phenotype Integration Reveals the Contributions of Peripheral Immune Cells to Bipolar Disorder Pathogenesis, Phenotypic Heterogeneity, and Therapy
              </a>
            </td>
          <td>
            , Y. Li, X. Xiong, Y. Tanigawa, Y. Park, S. W. Lenz, A. Grayson, J.-H. Lee, E. Ryu, J. E. Olson, J. M. Biernacka, M. A. Frye, T. Ordog, M. Kellis
          </td>
          <td>2025-03-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ee3ba4be7a7e3b5e63b304d989c55900905b7" target='_blank'>
              Integrated analysis of single-cell and bulk RNA-sequencing to predict prognosis and therapeutic response for colorectal cancer
              </a>
            </td>
          <td>
            Liyang Cai, Xin Guo, Yucheng Zhang, Huajie Xie, Yongfeng Liu, Jianlong Zhou, Huolun Feng, Jiabin Zheng, Yong Li
          </td>
          <td>2025-03-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f8b66ae23a1d254045e2d47bbe65620395c404" target='_blank'>
              Single-Cell Genomics Elucidates Molecular Variations and Regulatory Mechanisms in Circulating Immune Cells
              </a>
            </td>
          <td>
            Jianhua Yin, Yuhui Zheng, Zhuoli Huang, Wenwen Zhou, Yue Yuan, Pengfei Cai, Yong Bai, Shichen Yang, Yue Gao, Shanshan Duan, Yang Wang, Wenxi Zhang, Xinyu Zhang, Yilin Wei, Zekai Xu, Yaling Huang, Ying Liu, Weikai Wang, Tao Yang, Jingzhi Lv, Zhongjin Zhang, Xiaoya Chen, Xiru Zhang, Fupeng Li, Yan Zhang, Guodan Zeng, Xue Wang, Wen Ma, Guixue Hou, Shijie Hao, Chang Liu, Yiwei Lai, Bo Wang, Yuxiang Li, Wenwei Zhang, Peng Gao, Jun Xie, Miguel A. Esteban, Ying Gu, Jiansong Ji, Ting Qi, Boxiang Liu, Jian Wang, Jian Yang, Xun Xu, Longqi Liu, Xin Jin, Chuanyu Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ad7065cbf4c15026ee68bc1c1e9443de9a4af3" target='_blank'>
              Identification of T cell dysfunction molecular subtypes and exploration of potential immunotherapy targets in BRAF V600E-mutant colorectal cancer
              </a>
            </td>
          <td>
            Tiefeng Gu, Haonan Qi, Jiaqi Wang, Liangwei Sun, Yongqi Su, Hanqing Hu
          </td>
          <td>2025-02-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e17321e8898e722005a2009e4fe82616f4dc5" target='_blank'>
              Role of genetic frustrations in cell reprogramming
              </a>
            </td>
          <td>
            Yuxiang Yao, Jieying Zhu, Wenfei Li, Duanqing Pei
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy affecting women, manifesting as a heterogeneous disease with diverse molecular characteristics and clinical presentations. Recent studies have elucidated the role of epigenetic modifications in the pathogenesis of breast cancer, including drug resistance and efflux characteristics, offering potential new diagnostic and prognostic markers, treatment efficacy predictors, and therapeutic agents. Key modifications include DNA cytosine methylation and the covalent modification of histone proteins. Unlike genetic mutations, reprogramming the epigenetic landscape of the cancer epigenome is a promising targeted therapy for the treatment and reversal of drug resistance. Epidrugs, which target DNA methylation and histone modifications, can provide novel options for the treatment of breast cancer by reversing the acquired resistance to treatment. Currently, the most promising approach involves combination therapies consisting of epidrugs with immune checkpoint inhibitors. This review examines the aberrant epigenetic regulation of breast cancer initiation and progression, focusing on modifications related to estrogen signaling, drug resistance, cancer progression, and the epithelial–mesenchymal transition (EMT). It examines existing epigenetic drugs for treating breast cancer, including agents that modify DNA, inhibitors of histone acetyltransferases, histone deacetylases, histone methyltransferases, and histone demethyltransferases. It also delves into ongoing studies on combining epidrugs with other therapies and addresses the upcoming obstacles in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1d287b99832f39586fe0fd89b9b4c10e300e3c" target='_blank'>
              Cellular Epigenetic Targets and Epidrugs in Breast Cancer Therapy: Mechanisms, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Ibrahim S Alalhareth, Saleh M. Alyami, Ali H. Alshareef, Ahmed O. Ajeibi, M. A. Al Munjem, Ahmad A. Elfifi, Meshal M. Alsharif, S. A. Alzahrani, Mohammed A. Alqaad, Marwa B Bakir, Basel A. Abdel-Wahab
          </td>
          <td>2025-02-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d11642e20fef657158a832ae45b3c22aa7b26f" target='_blank'>
              Abstract B025: Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Pérez-Mayoral, Megan J Stine, Eva Y. Tonsing-Carter, James M Clinton, The Hcmi Network, Peter W. Laird, Calvin J Kuo, O. Elemento, David L. Spector, A. Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Mathew Garnett, Andrea Califano, P. Spellman, D. Tuveson, Keith L. Ligon, Daniela S Gerhard, Louis M Staudt, Jesse S Boehm
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Although GBM’s immunosuppressive environment is well known, the tumor’s resistance to CD8+ T cell killing is not fully understood. Our previous study identified Checkpoint Kinase 2 (Chek2) as the key driver of CD8+ T cell resistance in mouse glioma through an in vivo CRISPR screen and demonstrated that Chk2 inhibition, combined with PD-1/PD-L1 blockade, significantly enhanced CD8+ T cell-mediated tumor killing and improved survival in preclinical model. Here, we aimed to elucidate the immunosuppressive function of Chek2. Immunoprecipitation (IP) followed by mass spectrometry (MS) and phosphoproteomics identified an association between Chek2 with the DNA/RNA-binding proteins YBX1 and YBX3 that are implicated in transcriptional repression of pro-inflammatory genes. Single-gene knock-out and overexpression studies of CHEK2, YBX1, and YBX3 in multiple glioma cell lines revealed that these proteins positively regulate each other’s expression. RNA sequencing coupled with chromatin immunoprecipitation-sequencing (ChIP-seq) analysis demonstrated common inflammatory genes repressed by CHK2-YBX1&YBX3 hub. Targeting one of the hub proteins, YBX1, with the YBX1 inhibitor SU056 led to degradation of CHK2-YBX1&YBX3 hub. Targeting of this hub by SU056 led to enhanced antigen presentation and antigen specific CD8+ T cell proliferation. Further, combination of SU056 with ICB significantly improved survival in multiple glioma models. Collectively, these findings reveal an immunosuppressive mechanism mediated by the CHK2-YBX1&YBX3 hub proteins. Therefore, CHK2-YBX1&YBX3 hub targeting in combination with immune checkpoint blockade therapies in gliomas is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5eee0adfec60774c8a1b6ada43414f89d5d5752" target='_blank'>
              Targeting CHEK2-YBX1&YBX3 regulatory hub to potentiate immune checkpoint blockade response in gliomas
              </a>
            </td>
          <td>
            Heba Ali, Ningjia Zhou, Li Chen, Levi van Hijfte, Vivekanudeep Karri, Yalu Zhou, K. Habashy, Victor A. Arrieta, Kwang-Soo Kim, J. Duffy, Ragini Yeeravalli, D. Tiek, Xiao Song, Snehasis Mishra, Catalina Lee-Chang, Atique U. Ahmed, Dieter H. Heiland, A. Sonabend, C. Dmello
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer, characterized by the uncontrolled proliferation of cells, is one of the leading causes of death globally, with approximately one in five people developing the disease in their lifetime. While many driver genes were identified decades ago, and most cancers can be classified based on morphology and progression, there is still a significant gap in knowledge about genetic aberrations and nuclear DNA damage. The study of two critical groups of genes—tumor suppressors, which inhibit proliferation and promote apoptosis, and oncogenes, which regulate proliferation and survival—can help to understand the genomic causes behind tumorigenesis, leading to more personalized approaches to diagnosis and treatment. Aberration of tumor suppressors, which undergo two-hit and loss-of-function mutations, and oncogenes, activated forms of proto-oncogenes that experience one-hit and gain-of-function mutations, are responsible for the dysregulation of key signaling pathways that regulate cell division, such as p53, Rb, Ras/Raf/ERK/MAPK, PI3K/AKT, and Wnt/β-catenin. Modern breakthroughs in genomics research, like next-generation sequencing, have provided efficient strategies for mapping unique genomic changes that contribute to tumor heterogeneity. Novel therapeutic approaches have enabled personalized medicine, helping address genetic variability in tumor suppressors and oncogenes. This comprehensive review examines the molecular mechanisms behind tumor-suppressor genes and oncogenes, the key signaling pathways they regulate, epigenetic modifications, tumor heterogeneity, and the drug resistance mechanisms that drive carcinogenesis. Moreover, the review explores the clinical application of sequencing techniques, multiomics, diagnostic procedures, pharmacogenomics, and personalized treatment and prevention options, discussing future directions for emerging technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4acb0abcdb89e4becb0be9ae1ca6990d3d6cdf49" target='_blank'>
              Translational Advances in Oncogene and Tumor-Suppressor Gene Research
              </a>
            </td>
          <td>
            R. Stojchevski, Edward Agus Sutanto, Rinni Sutanto, N. Hadzi-Petrushev, M. Mladenov, Sajal Raj Singh, J. K. Sinha, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, Rakesh Bhaskar, Dimiter B Avtanski
          </td>
          <td>2025-03-17</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719dc00daf58fbc383533083eb81aecec4828ea4" target='_blank'>
              Epigenetic marvels: exploring the landscape of colorectal cancer treatment through cutting-edge epigenetic-based drug strategies
              </a>
            </td>
          <td>
            A. Tahghighi, Effat Seyedhashemi, Javad Mohammadi, Arash Moradi, Aria Esmaeili, Majid Pornour, Kimia Jafarifar, S. M. Ganji
          </td>
          <td>2025-02-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f993cc37705c650add4d10eacea4b3e32cdfd64" target='_blank'>
              BCL-2 dependence is a favorable predictive marker of response to therapy for chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            S. Chong, Junyan Lu, Rebecca Valentin, Timothy Lehmberg, J. Eu, Jing Wang, Fengfang Zhu, L. Kong, Stacey M. Fernandes, Jeremy Zhang, Charles Herbaux, Boon Cher Goh, Jennifer R. Brown, Carsten U Niemann, W. Huber, Thorsten Zenz, M. Davids
          </td>
          <td>2025-03-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This study aimed to elucidate the transcriptomic signatures and dysregulated pathways associated with the autoimmune response in Cd38-/- mice compared to wild-type (WT) mice within the bm12 chronic graft-versus-host disease (cGVHD) lupus model. We conducted bulk RNA sequencing on peritoneal exudate cells (PECs) and spleen cells (SPC) at two and four weeks following adoptive cell transfer. We also analyzed cells from healthy, untreated mice. These analyses revealed a sustained upregulation of a transcriptional profile of purinergic receptors and ectonucleotidases in cGVHD WT PECs, which displayed a coordinated expression with several type I interferon-stimulated genes (ISGs) and with key molecules involved in the cyclic GMP-AMP synthase-stimulator of interferon genes (cGAS-STING) signaling pathway, two hallmarks in the lupus pathology. A second purinergic receptor transcriptomic profile, which included P2rx7 and P2rx4, showed a coordinated gene expression of the components of the NLRP3 inflammasome with its potential activators. These processes were transcriptionally less active in cGVHD Cd38-/- PECs than in WT PECs. We have also shown evidence of a distinct enrichment in pathways signatures that define processes such as Ca2+ ion homeostasis, cell division, phagosome, autophagy, senescence, cytokine/cytokine receptor interactions, Th17 and Th1/Th2 cell differentiation in Cd38-/- versus WT samples, which reflected the milder inflammatory and autoimmune response elicited in Cd38-/- mice relative to WT counterparts in response to the allogeneic challenge. Last, we have shown an intense metabolic reprogramming toward oxidative phosphorylation in PECs and SPC from cGVHD WT mice, which may reflect an increased cellular demand for oxygen consumption, in contrast to PECs and SPC from cGVHD Cd38-/- mice, which showed a short-lived metabolic effect at the transcriptomic level. Overall, these findings support the pro-inflammatory and immunomodulatory role of CD38 during the development of the cGVHD-lupus disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47a5fade16c476abb5f2145f8e83f353870ec91" target='_blank'>
              CD38 deficiency leads to a defective short-lived transcriptomic response to chronic graft-versus-host disease induction, involving purinergic signaling-related genes and distinct transcriptomic signatures associated with lupus
              </a>
            </td>
          <td>
            M. Zubiaur, L. Terrón-Camero, Fernando Gordillo-González, Eduardo Andrés-León, Alicia Barroso-del Jesús, Luz María Canet-Antequera, María M. Pérez Sánchez-Cañete, África Martínez-Blanco, M. Domínguez-Pantoja, María Botía-Sánchez, Sonia Pérez-Cabrera, Nerea Bello-Iglesias, A. Alcina, A. Abadía-Molina, F. Matesanz, Esther Zumaquero, Ramón Merino, Jaime Sancho
          </td>
          <td>2025-02-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7519cd5eee8915a40b0574f88da21fc8b5e2d5" target='_blank'>
              Therapeutic horizon of acute myeloid leukemia: Success, optimism, and challenges.
              </a>
            </td>
          <td>
            J. Senapati, T. Kadia, Naval G Daver, Courtney D DiNardo, G. Borthakur, F. Ravandi, H. Kantarjian
          </td>
          <td>2025-03-19</td>
          <td>Cancer</td>
          <td>0</td>
          <td>226</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Background: Lung cancer has the highest mortality rate among all cancers, for which immunotherapy can frequently lead to drug resistance. To understand the molecular mechanisms behind immune escape in patients with lung cancer and develop predictive and therapeutic targets, we carried out analytical experiments using single-cell sequencing. Methods: We collected eight tumor tissue samples from eight patients with lung adenocarcinoma and categorized them based on the positive reactions for programmed cell death ligand 1 (PD-L1) expression levels. Single-cell sequencing analysis was employed to create a comprehensive cellular landscape. Uniform Manifold Approximation and Projection was used to show the proportion of immune and endothelial cells, along with a map depicting the distribution of different cell types. Cells were subdivided according to molecular markers; the subpopulations were grouped based on PD-L1 levels and tumor marker-positive reactions. The correlation between the occurrence of the PD-L1 reaction and the response time of immune cells was explored; differential gene expression between the groups was elucidated. Finally, quantitative polymerase chain reaction (qPCR) was used to examine the relationship between key differentially expressed genes and PD-L1 immune escape checkpoint response. Results: A total of 58,810 single cells were analyzed, identifying seven distinct cell types. In the PD-L1-positive sample group, B cells, astrocytes, endothelial cells, outer skin cells, and tissue stem cells were present in higher proportions, whereas T and dendritic cells were the main cells in the PD-L1-negative sample group. According to the molecular markers, the seven cell types were divided into 17 cell clusters, with one cluster classified as tumor cells, showing PD-L1 positivity. Eleven molecular markers with different expression levels were simultaneously screened (NAPSA, MUC1, WFDC2, MYO6, LYZ, IGHG4, IGLL5, IGHM, IGKC, AQP3, and IGFBP7), and their association with the PD-L1/PD-1 immune escape axis response was confirmed by qPCR. Conclusion: Our study suggests that PD-L1-mediated immune escape may occur at a later stage of tumor progression, involving both PD-L1-positive and negative immune cells. Additionally, we identified 11 differentially expressed genes that could provide insights into the potential mechanisms of immune escape in patients with lung cancer. These findings offer promising molecular targets for the detection and treatment of immune escape in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b724ed5847ed5cdec385126142a8a403030c10" target='_blank'>
              Single-Cell Sequencing Reveals PD-L1-Mediated Immune Escape Signaling in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Anbing Zhang, Jianping Liang, Xiaoli Lao, Xiuqiong Xia, Siqi Li, Shengming Liu
          </td>
          <td>2025-01-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f5d2a19ddfe84eb40ce3d240e162476c8b6ecf" target='_blank'>
              CRISPR-Cas9 Is Revolutionizing AML Treatment As A Novel Therapy
              </a>
            </td>
          <td>
            Fakhra Alafeefi, Afsheen Raza
          </td>
          <td>2025-01-24</td>
          <td>Cancer Immunology Connect</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8599e8314fc9026e2db49b39f76e5203a6d7080" target='_blank'>
              Allele-specific genomics decodes gene targets and mechanisms of the non-coding genome
              </a>
            </td>
          <td>
            Tim P. Hasenbein, Sarah Hoelzl, S. Engelhardt, Daniel Andergassen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2902b0310b2c07b6ab1dea3f9572b47252e74d6f" target='_blank'>
              Drug tolerant persisters and immunotherapy persister cells exhibit cross-resistance and share common survival mechanisms
              </a>
            </td>
          <td>
            Maria Davern, Cole J Turner, Lara Bencsics, Brenda Chan, Jasmine Yun-Tong Kung, Michael L. Olson, Cheyanne Walker Williams, Shaili Soni, Leah Krotee, Michael Yorsz, Gabriella Antonellis, Patrick H. Lizotte, C. Paweletz, Jeremy Ryan, Filippo Birocchi, Antonio J Almazan, David A. Barbie, P. Bhola, Marcela V Maus, Anthony Letai
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Colibactin is a genotoxic secondary metabolite produced by certain Enterobacteriaceae strains that populate the intestine and produces a specific mutational signature in human colonocytes. However, the host pathways involved in colibactin response remain unclear. To address this gap, we performed genome-wide CRISPR/Cas9 knockout screens and RNA sequencing utilizing live pks+ bacteria and a synthetic colibactin analog. We identified 20 enriched genes with a MAGeCK score of >2.0 in both screens, including proteasomal subunits (e.g., PSMG4 and PSMD4), RNA processing factors (e.g., SF1 and PRPF8), and RNA polymerase III (e.g., CRCP), and validated the role of PSMD4 in colibactin sensitization. PSMD4 knockout in HEK293T and HT-29 cells promoted cell viability and ameliorated G2-M cell cycle arrest but did not affect the amount of phosphorylated H2AX foci after exposure to synthetic colibactin 742. Consistent with these observations, PSMD4-/- cells had a significantly higher colony formation rate and bigger colony size than control cells after 742 exposure. These findings suggest that PSMD4 regulates cell cycle arrest following colibactin-induced DNA damage and that cells with PSMD4 deficiency may continue to replicate despite DNA damage, potentially increasing the risk of malignant transformation.


IMPORTANCE
Colibactin has been implicated as a causative agent of colorectal cancer. However, colibactin-producing bacteria are also present in many healthy individuals, leading to the hypothesis that some aspects of colibactin regulation or host response dictate the molecule's carcinogenic potential. Elucidating the host-response pathways involved in dictating cell fate after colibactin intoxication has been difficult, partially due to an inability to isolate the molecule. This study provides the first high-throughput CRISPR/Cas9 screening to identify genes conferring colibactin sensitivity. Here, we utilize both bacterial infection and a synthetic colibactin analog to identify genes directly involved in colibactin response. These findings provide insight into how differences in gene expression may render certain individuals more vulnerable to colibactin-initiated tumor formation after DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c10ad4bc58eb16a2a76999a8e88cc374bfe8d0" target='_blank'>
              Genome-scale CRISPR/Cas9 screening reveals the role of PSMD4 in colibactin-mediated cell cycle arrest.
              </a>
            </td>
          <td>
            Michael W. Dougherty, Ryan M. Hoffmann, Maria C. Hernandez, Yougant Airan, R. Gharaibeh, Seth B. Herzon, Ye Yang, Christian Jobin
          </td>
          <td>2025-02-07</td>
          <td>mSphere</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95b9504bba9cb27a525f77f6f19beab513280064" target='_blank'>
              Ribosome Profiling Reveals Translational Reprogramming via mTOR Activation in Omacetaxine Resistant Multiple Myeloma.
              </a>
            </td>
          <td>
            Z. Walker, Katherine F. Vaeth, Amber Baldwin, Denis J Ohlstrom, Lauren T. Reiman, Kady A Dennis, Kate Matlin, Beau M. Idler, Brett M Stevens, Neelanjan Mukherjee, Daniel W. Sherbenou
          </td>
          <td>2025-03-06</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a61455bde5450ac8c919b6a07222c913fc3cde" target='_blank'>
              Aberrant DNA methylation as a key modulator of cell death pathways: insights into cancer progression and other diseases.
              </a>
            </td>
          <td>
            Ambreen Zahoor, Rafia Khazer, Insha Mehraj, Ubaid Gani, Falah Fayaz, F. Khanday, S. S. Bhat
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83f8e329366e14ce122de3c8f5608b479cd4df9" target='_blank'>
              Mapping glioma progression: single-cell RNA sequencing illuminates cell–cell interactions and immune response variability
              </a>
            </td>
          <td>
            Xia Li, Shenbo Chen, Ming Ding, Hui Ding, Kun Yang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15bb80576bbd8d5340c20144f6b3a46d4763d6f" target='_blank'>
              Clofazimine treatment modulates key non-coding RNAs associated with tumor progression and drug resistance in lethal Prostate Cancer
              </a>
            </td>
          <td>
            Sarah Batten, Harish Kumar, Suman Mazumder, T. Ghosh, Amit Kumar
          </td>
          <td>2025-02-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6f1a197237e49a8b6045fada2a0bb94a7b46f0" target='_blank'>
              Gene Expression Profiling in Acute Myeloid Leukemia Patient Subgroups With High and Low Sensitivity Toward SYK Inhibitors.
              </a>
            </td>
          <td>
            Marte Karen Brattås, Franziska Görtler, Silje Johansen, K. P. Rye, K. Hatfield, H. Reikvam
          </td>
          <td>2025-03-01</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4625a1647e0225fc85616b697ca8ae26c76a3c30" target='_blank'>
              Transformative CRISPR-Cas9 Technologies: A Review of Molecular Mechanisms, Precision Editing Techniques, and Clinical Progress in Sickle Cell Disease.
              </a>
            </td>
          <td>
            Komal, Prabhjot Kaur, Nidhi Arora, Jyotiram A. Sawale, Amandeep Singh
          </td>
          <td>2025-03-05</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/995da50c8649d8341a6a265386cac02363ea1e3b" target='_blank'>
              Performing an In Vitro Genome-Wide CRISPR Knockout Screen in Chimeric Antigen Receptor T Cells.
              </a>
            </td>
          <td>
            Carli M. Stewart, Claudia Manriquez Roman, S. Kenderian
          </td>
          <td>2025-01-31</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d171557e9628c9a32e1f797d2ae45ffca94dcd" target='_blank'>
              Cytoplasmic WEE1 promotes resistance to PD-1 blockade through hyperactivation of the HSP90A/TCL1/AKT signaling axis in NANOGhigh tumors.
              </a>
            </td>
          <td>
            Suyeon Kim, Hyo-Jung Lee, Seungho Lee, Jo Eun Chung, Se Jin Oh, , Eunho Cho, Min Kyu Son, Heeju Kwon, Seung-Jong Kim, Chaeleen Lee, Suhwan Chang, Tae Woo Kim
          </td>
          <td>2025-03-11</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immortalized cell lines are commonly used as proxies for primary cells in human biology research. For example, Jurkat leukemic T cells fundamentally contributed to uncovering T cell signaling, activation, and immune responses. However, the immortalization process can alter key cellular properties, and researchers widely believe this process could significantly change RNA modification machinery and modification sites. In this study, we focus on pseudouridine (Ψ), one of the most abundant mRNA modifications, and compare Ψ profiles in mRNA from primary and immortalized T cells using direct RNA sequencing (DRS). Surprisingly, 87% of Ψ-sites were shared between the two cell types, primarily in transcripts encoding proteins involved in essential cellular processes, including RNA-modification regulation. Furthermore, the analysis of the 13% of sites unique to each cell type reveals that Jurkat cells contained transcripts linked to immune activation and oncogenesis, while primary T cells contained transcripts associated with calcium signaling and intracellular trafficking. We provide a list of these genes, which should be considered when using immortalized cells to study RNA modifications in immunology contexts. Most differences were driven by whether the mRNA was present or absent in the immortalized or primary cell type. Interestingly, RNA-modification enzyme expression levels were highly conserved in both cell types. This suggests that site-specific differences in Ψ levels arise from regulatory processes acting in trans rather than differences in modification enzyme levels. Significance Statement It is widely believed that RNA modification machinery and modification sites could be significantly altered in immortalized cells, yet this has never been tested. Focusing on pseudouridine (Ψ), we map Ψ in the transcriptomes of primary and immortalized human T cells. Surprisingly, most sites are conserved in the primary and immortalized T cells, with several important examples of cases with cell type specificity and should be considered on a case-by-case basis. Furthermore, we evaluated RNA-modification machinery levels in primary and immortalized T cells, finding high conservation across the cell lines. Our findings demonstrate that RNA modifications are largely conserved between primary and immortalized cells, and the edge cases can be considered individually.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b46d555b004bfc6033b490f3c93648df1820f4" target='_blank'>
              Direct RNA sequencing of primary human T cells reveals the impact of immortalization on mRNA pseudouridine modifications
              </a>
            </td>
          <td>
            Oleksandra Fanari, Dylan Bloch, Yuchen Qiu, Michele Meseonznik, Amr Makhamreh, M. Wanunu, S. Rouhanifard
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a99a90f864b231f78785ccac2d5080c6a00e33" target='_blank'>
              Explainable AI-driven prediction of APE1 inhibitors: enhancing cancer therapy with machine learning models and feature importance analysis.
              </a>
            </td>
          <td>
            Aga Basit Iqbal, Tariq Ahmad Masoodi, A. Bhat, M. Macha, Assif Assad, Syed Zubair Ahmad Shah
          </td>
          <td>2025-02-21</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b02a5edf5833a9413d878e11e7ef959bc982d0" target='_blank'>
              Embracing the Circular Journey of RNA in Colorectal Cancer.
              </a>
            </td>
          <td>
            A. Barjasteh, Ayat Heidar Abdolamir, Abdulridha Mohammed Al-Asady, Hanieh Latifi, Amir Avan, Majid Khazaei, Mikhail Ryzhikov, S. M. Hassanian
          </td>
          <td>2025-03-03</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The mechanisms of epigenetic regulation emerge as a fundamental determinant in the complex landscape of cancer initiation and advancement. However, the specific impact of epigenetic regulation on cancer progression remains unclear. To explore the relationship between epigenetic regulation and cancer progression, we utilized transcriptomic data from The Cancer Genome Atlas (TCGA) datasets to investigate the association. Methods We obtained transcriptomic data of epigenetic gene dataset from the TCGA database and calculated an epigenetic score using the Least Absolute Shrinkage and Selection Operator (LASSO) Cox model. Additionally, we created a nomogram that integrates the epigenetic score and clinical features, providing a more comprehensive tool for tumor patients prognosis assessment. Results We calculated the epigenetic score based on the expression levels of epigenetic-related genes. The nomogram we developed incorporates the epigenetic score and clinical characteristics. The epigenetic score was positively correlated with the expression of genes related to hallmarkers of cancer, including glycolysis, epithelial-mesenchymal transition (EMT), cell cycle, DNA repair, angiogenesis, and inflammatory response. Furthermore, we performed gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG) analysis to explore the signaling pathways and biological processes in high epigenetic score group. Conclusion The epigenetic scoring system developed in this investigation represents an innovative approach that demonstrates remarkable potential in forecasting survival trajectories across diverse cancer types. These groundbreaking insights not only illuminate the intricate interactions between epigenetic mechanisms and gene expression regulation in oncological contexts, but also indicate that the derived epigenetic metric could potentially emerge as a significant prognostic biomarker for cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b405c467d91505b22b6bd6b604ad22bfbaf352f" target='_blank'>
              Exploring the role of epigenetic regulation in cancer prognosis with epigenetic score
              </a>
            </td>
          <td>
            Ruiguang Zhang, Shimin Jin, Qi Xu, Rongxiao Dai
          </td>
          <td>2025-02-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b3a2a7fdf1a317ed911bfc6a066f5d8abc14563" target='_blank'>
              Exploring the Function of METTL3 in Lung Cancer: Understanding Drug Resistance and Exploring Innovative Therapeutic Approaches
              </a>
            </td>
          <td>
            Shiqing Huang, Ye Li, F. Liew
          </td>
          <td>2025-03-12</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20366f8a8e956d20a8164d2739ca733da77de50e" target='_blank'>
              RUNX1::RUNX1T1 Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. Daniel, Mata Casimiro, Luca van den Brink, Anita T. van Oort, Minoo Ashtiani, C. Zwaan, Anja Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, H. J. Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f86d44b9e09159a099ee094747765812cf22c69" target='_blank'>
              CRISPR-dCas9-Mediated PTEN Activation via Tumor Cell Membrane-Coated Nanoplatform Enhances Sensitivity to Tyrosine Kinase Inhibitors in Nonsmall Cell Lung Cancer.
              </a>
            </td>
          <td>
            Kaiyong Yang, Chunli Zhang, Zeyu Wang, Qiqing Huang, Jing Qian, Gaoyu Shi, Wenwen Sun, Jinqiu Wang, Yu Ji, Zhaorui Sun, Yanni Song, Xin Han
          </td>
          <td>2025-02-20</td>
          <td>ACS applied materials & interfaces</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tissues and organs develop from single founder cells, which give rise to distinct cell lineages that contribute to regeneration and maintenance of homeostasis in the adult. Tracing the genealogical relationships between individual cells and their gene expression signatures is an important step towards understanding how these processes are regulated in human health and disease. Here, we present mt-SCITE, a computational method for inferring the evolutionary history of dividing cells based on mitochondrial mutations detected in single cells. We show that mt-SCITE outperforms existing methods in accurately estimating the correct mitochondrial mutation tree from simulated data. To validate our method, we applied mt-SCITE on in vitro expanded T-cells profiled with bulk ATAC sequencing, where the clonal relationships were determined independently from T cell receptor sequences. We used our method to reconstruct the division histories and transcriptional heterogeneity of clonally related CD8+ T cells from single-cell RNA sequencing data obtained from a healthy human donor. This allowed us to track the in vivo development of CD8+ T cell clones post vaccination and resolve subclonal lineage relationships without the need for genetic barcoding. Our analysis revealed that T cell clones can adopt predominantly memory-like or effector-like states, while some exhibit mixed identities. Taken together, our method represents a novel approach to study cell lineage development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b46463e933e01c11da7ed98772b7d2ff4e8ab033" target='_blank'>
              Cell lineage inference from mitochondrial mutations
              </a>
            </td>
          <td>
            Joanna Hård, S. Seidel, Pedro F. Ferreira, Pietro Berkes, Katharina Jahn, Carl-Johan Eriksson, Jeff E. Mold, Jakob Michaelsson, Jack Kuipers, N. Beerenwinkel
          </td>
          <td>2025-03-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401aff8c0f114a1e1c91ffcb779a01c90497fc5c" target='_blank'>
              Molecular insights and treatment innovations: Advancing outcomes in acute myeloid leukemia with myelodysplasia‑related changes (Review).
              </a>
            </td>
          <td>
            Hong Qiu, Chaowei Zhang, Xiaochen Ma, Ying Li
          </td>
          <td>2025-03-19</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Aberrant hypermethylation in the promoter regions of tumor suppressor genes facilitates the pathogenesis and progression of cancer. Therefore, inhibitors targeting DNA methyltransferase (DNMT) have been tested in clinical studies. However, the current monotherapy of DNMT inhibitors shows limited efficacy. Furthermore, the mechanism of action of DNMT inhibitors is DNA replication-dependent. To address these limitations, we developed a novel core–shell-type “epigenetics control (EpC) nanocarrier” that encapsulated decitabine (5-aza-dC) in the PLGA core nanoparticle and hybridized TET1 gene-encoding pDNA on the lipid shell surface. This study aimed to evaluate whether the dual delivery of DNMT inhibitors and pDNA of TET1 could synergistically enhance tumor suppressor gene expression and induce cell cycle arrest and/or apoptosis in cancer cells. Herein, we demonstrate the potential of the EpC carrier in HCT116 human colon cancer cells to upregulate tumor suppressor gene expression and rapidly achieve cell cycle arrest. Methods: PLGA core nanoparticles were prepared by the W/O/W double emulsion method. The formation of core–shell nanoparticles and complexation with pDNA were investigated and optimized by dynamic light scattering, zeta potential measurement, and agarose gel electrophoresis. The cellular uptake and transfection efficiency were measured by confocal laser scanning microscopy and a luciferase assay, respectively. The expression of p53 protein was detected by Western blotting. The anti-tumor effects of the EpC nanocarrier were evaluated by cell cycle analysis and an apoptosis assay. Results: The EpC nanocarrier delivered the DNMT inhibitor and TET gene-encoding pDNA into HCT116 cells. It promoted the expression of the tumor suppressor protein p53 and induced rapid cell cycle arrest in the G2/M phase in HCT116 cells. Conclusions: Our findings suggest that the dual-targeting of DNMT and TET enzymes effectively repairs aberrant DNA methylation and induces growth arrest in cancer cells, and the dual-targeting strategy may contribute to the advancement of epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310794284715bd67edbc662582ab25897ba7a464" target='_blank'>
              Novel Epigenetics Control (EpC) Nanocarrier for Cancer Therapy Through Dual-Targeting Approach to DNA Methyltransferase and Ten-Eleven Translocation Enzymes
              </a>
            </td>
          <td>
            Risa Mitsuhashi, Kiyoshi Sato, Hiroyoshi Kawakami
          </td>
          <td>2025-02-11</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb21c2801e782dbb110eb435c1cbbe7b47f6623" target='_blank'>
              Identifying gene expression and cellular pathways involved in glomerular AL-amyloidosis and correlation with experimental data: seeking novel therapeutic interventions.
              </a>
            </td>
          <td>
            Guillermo A. Herrera, J. Teng, Chun Zeng, L. Del Pozo-Yauner, Bing Liu, E. Turbat-Herrera
          </td>
          <td>2025-02-21</td>
          <td>Ultrastructural pathology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1767b38084076cdfaf10fe2b85fb296d8eecabbf" target='_blank'>
              Abstract B021: Saturation mutational scanning uncovers druggability of all FGFR kinase point mutations
              </a>
            </td>
          <td>
            Carla Tangermann, Avantika Ghosh, Martin Ziegler, Francesco Facchinetti, Jannis Stappenbeck, M. Riester, Luise Carmina Viardot, Tobias Zundel, L. Friboulet, A. Hollebecque, José Naveja Romero, Angela Wanninger, Maria Elena Hess, Tilman Brummer, Melanie Börries, Sonja Loges, Yohann Loriot, Lena Illert, Sven Diedrichs
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, with its incidence and mortality rates projected to rise due to population aging. In this study, we utilized PCa transcriptome data from public databases and applied bioinformatics methods to identify three prognostic genes (CDC20, RAD51, and TTK) related to centrosome duplication in PCa. CDC20 is involved in cell cycle regulation, RAD51 in deoxyribonucleic acid double-strand break repair, and TTK in spindle assembly checkpoint function and cell proliferation. We constructed a risk model and a nomogram model, both demonstrating moderate to good predictive performance with area under the curve values ranging from 0.611 to 0.765 at different time points. Gene set enrichment analysis revealed that these genes were enriched in 64 pathways, including the cell cycle pathway, which is dysregulated in cancer. Furthermore, we analyzed the immune microenvironment and identified 13 differential immune cells and 13 differential immune checkpoints between high- and low-risk groups, providing insights into potential immunotherapy targets for PCa. In conclusion, this study contributes to a deeper understanding of PCa pathogenesis and lays important theoretical and experimental foundations for developing new diagnostic markers and treatment strategies. Future research requires more clinical samples and continued monitoring of the mechanism of these genes in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8581bf4b2f1cfe894fadfd64d88a1382b8cc282" target='_blank'>
              Prognostic Value of Centrosome Replication-Related Genes in Prostate Cancer Based on Transcriptomic and Mendelian Randomization
              </a>
            </td>
          <td>
            Qizhong Lu, Yufan Wu, Qiwei Yu, Jun Ouyang
          </td>
          <td>2025-03-01</td>
          <td>American Journal of Men's Health</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e56fa3aad088d2f09d1437903fb4b00ec728238c" target='_blank'>
              Delayed Trp53 activation protects Dnmt3a-mutant hematopoietic stem cells from inflammatory attrition
              </a>
            </td>
          <td>
            Ludovica Marando, G. Giotopoulos, Pedro Madrigal, Dhoyazan M A Azazi, Ryan Asby, M. E. Calleja-Cervantes, M. Gu, David Lara-Astiaso, Nicola K Wilson, R. Hannah, Jaana Bagri, Emily F. Calderbank, E. Laurenti, E. Bernard, Sarah J Horton, Berthold Göttgens, E. Papaemmanuil, Shuchi Agrawal-Singh, M. Gozdecka, G. Vassiliou, B. Huntly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Precision medicine approaches to cancer treatment aim to exploit genomic alterations that are specific to individual patients to tailor therapeutic strategies. Yet, some targetable genes and pathways are essential for tumor cell viability even in the absence of direct genomic alterations. In underrepresented populations, the mutational landscape and determinants of response to existing therapies are poorly characterized because of limited inclusion in clinical trials and studies. One way to reveal tumor essential genes is with genetic screens. Most screens are conducted on cell lines that bear little resemblance to patient tumors, after years of culture under nonphysiologic conditions. To address this problem, we aimed to develop a CRISPR screening pipeline in three-dimensionally grown patient-derived tumor organoid (PDTO) models. A breast cancer PDTO biobank that focused on underrepresented populations, including West African patients, was established and used to conduct a negative-selection kinome-focused CRISPR screen to identify kinases essential for organoid growth and potential targets for combination therapy with EGFR or MEK inhibitors. The screen identified several previously unidentified kinase targets, and the combination of FGFR1 and EGFR inhibitors synergized to block organoid proliferation. Together, these data demonstrate the feasibility of CRISPR-based genetic screens in patient-derived tumor models, including PDTOs from underrepresented patients with cancer, and identify targets for cancer therapy. Significance: Generation of a breast cancer patient-derived tumor organoid biobank focused on underrepresented populations enabled kinome-focused CRISPR screening that identified essential kinases and potential targets for combination therapy with EGFR or MEK inhibitors. See related commentary by Trembath and Spanheimer, p. 407.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde16e03b04b6ab8e7612d1796bad37a03db0723" target='_blank'>
              Kinome-Focused CRISPR-Cas9 Screens in African Ancestry Patient-Derived Breast Cancer Organoids Identify Essential Kinases and Synergy of EGFR and FGFR1 Inhibition.
              </a>
            </td>
          <td>
            Florencia P. Madorsky Rowdo, Rachel Martini, Sarah Ackermann, Colin P Tang, Marvel Tranquille, Adriana Irizarry, Ilkay Us, Omar Alawa, Jenna E Moyer, M. Sigouros, John Nguyen, Majd Al Assaad, Esther Cheng, Paula S Ginter, J. Manohar, Brian Stonaker, Richard Boateng, J. Oppong, E. Adjei, B. Awuah, I. Kyei, F. Aitpillah, M. Adinku, K. Ankomah, E. Osei-Bonsu, K. Gyan, Syed A Hoda, Lisa A Newman, Juan Miguel Mosquera, A. Sboner, O. Elemento, Lukas E. Dow, Melissa B. Davis, M. L. Martin
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86677aefd5550e942dc49063256f0df11e1c06ea" target='_blank'>
              Multi-lineage transcriptional and cell communication signatures define pathways in individuals at-risk for developing rheumatoid arthritis that initiate and perpetuate disease
              </a>
            </td>
          <td>
            Cong Liu, E. B. Prideaux, Peiyao Wu, David L. Boyle, Amy Westermann, K. Nguyen, Vlad Tsaltskan, Leander Lazaro, Andrea Ochoa, Kevin D. Deane, M. Feser, M. K. Demoruelle, Kristine A. Kuhn, V. M. Holers, Fan Zhang, L. Moss, Megan A. L. Criley, Brian C. Hattel, Marguerite D. Siedschlag, L. Okada, Mark A. Gillespie, P. Genge, M. Weiss, Veronica Hernandez, Julian Reading, Lynne A. Becker, Jane H. Buckner, C. Speake, Thomas F. Bumol, Peter J. Skene, G. Firestein, Wei Wang
          </td>
          <td>2025-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8319ec7e3e2dd161ac113462d4f85e344ef0e334" target='_blank'>
              NCBP2 predicts the prognosis and the immunotherapy response of cancers: a pan-cancer analysis
              </a>
            </td>
          <td>
            Shichao Li, Yulan Wang, Xi Yang, Miao Li, Guoxiang Li, Qiangqiang Song, Junyu Liu
          </td>
          <td>2025-03-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancer is the second leading cause of death among humans worldwide. Despite remarkable improvements in cancer therapies, drug resistance remains a significant challenge. The tumor microenvironment (TME) is intimately associated with therapeutic resistance. Tumor-associated neutrophils (TANs) are a crucial component of the TME, which, along with other immune cells, play a role in tumorigenesis, development and metastasis. In the current review, the roles of TANs in the TME, as well as the mechanisms of neutrophil-mediated resistance to cancer therapy, including immunotherapy, chemotherapy, radiotherapy and targeted therapy, were summarized. Furthermore, strategies for neutrophil therapy were discussed and TANs were explored as potential targets for cancer treatment. In conclusion, the need to explore the precise roles, recruitment pathways and mechanisms of action of TANs was highlighted for the purpose of developing therapies that precisely target TANs and reverse drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f741b0d8e7223ab3e910617355fcfe705132fe" target='_blank'>
              Tumor-associated neutrophils: Critical regulators in cancer progression and therapeutic resistance (Review)
              </a>
            </td>
          <td>
            Rui Hou, Xi Wu, Cenzhu Wang, Hanfang Fan, Yuhan Zhang, Hanchi Wu, Huiyu Wang, Junli Ding, Huning Jiang, Junying Xu
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf4afc79561ecf01107d9cb1fb29509ee0c751e0" target='_blank'>
              Harnessing ALDH1A2 vulnerability in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Andrieu, L. Courtois, Mickael Bonnet, Camille Gillet, Mélodie Queri, M. Féroul, L. Lhermitte, P. Villarese, Aurélie Cabanes-Hamy, Phillippe Rousselot, Mileydis Perez, I. Ceylan, Salvatore Spicuglia, Vahid Asnafi
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b6de2d94791247728d12901bea9b29daa88c32" target='_blank'>
              A protocol for karyotyping and genetic editing of induced pluripotent stem cells with homology-directed repair mediated CRISPR/Cas9
              </a>
            </td>
          <td>
            Silvana Lobo, Rita Barbosa-Matos, Sofia Dória, Ana Maria Pedro, Ana Brito, Daniel Ferreira, Carla Oliveira
          </td>
          <td>2025-02-28</td>
          <td>Biology Methods and Protocols</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The escalating global cancer burden necessitates the development of biomarkers with enhanced specificity and sensitivity for early diagnosis and therapeutic efficacy monitoring. The CNIH4 gene, an emerging biomarker, is increasingly recognized for its role in the malignant progression across various cancers. Methods We conducted a comprehensive multi-omics analysis of CNIH4, including pan-cancer expression profiles, epigenetic alterations, immune microenvironment characteristics, and therapeutic response patterns. Our focus was on clinical features, molecular underpinnings, and drug sensitivity in breast cancer (BRCA) associated with CNIH4. In vitro studies were also performed to assess the effects of CNIH4 knockdown on cell proliferation and cell cycle in the MDA-MB-231 cell line. Results CNIH4 upregulation was observed in multiple cancers, significantly correlating with genomic instability. High CNIH4 expression levels were linked to poor prognosis across cancers and associated with key cancer-related pathways, particularly those in cell cycle regulation and DNA repair. Correlation analyses suggest a role for CNIH4 in the tumor immune microenvironment, as evidenced by its association with immune subtypes, immune-related genes, and immune cell infiltration. Single-cell and spatial transcriptome analyses confirmed that CNIH4 expression in BRCA predicts tumor malignancy. Drug sensitivity analysis revealed a significant correlation between CNIH4 and responsiveness to various kinase inhibitors and chemotherapeutic agents. In vitro experiments demonstrated that CNIH4 knockdown significantly impacts the proliferation and cell cycle of MDA-MB-231 cells. Conclusion Our study highlights CNIH4 as a promising pan-cancer biomarker with significant implications for tumor progression and a critical role in cell cycle regulation in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7a2eade185851de2705c33f65b5768b5aafc69" target='_blank'>
              Deciphering the role of CNIH4 in pan-cancer landscapes and its significance in breast cancer progression
              </a>
            </td>
          <td>
            Yao Xu, Zengzhen Lai, Chaolin Li
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Monosomy 7 and deletion 7q are common chromosomal abnormalities in myeloid malignancies, and they are associated with a poor prognosis. The mechanism underlying their acquisition remains elusive. We identified a cohort of 24 patients exhibiting clones with different chromosome 7 abnormalities, such as deletion 7q, unstable derivatives (ring chromosomes or ‘naked’ centromeres), and monosomy 7. We designated this group as having cytogenetic clonal evolution of chromosome 7 abnormalities (CCE7). In some cases, CCE7 correlated with disease progression, suggesting that deletions or other derivatives involving the q-arm of chromosome 7 may arise early in the disease course. These abnormalities may be transient but can potentially evolve into monosomy 7. Within the CCE7 group, telomere loss or shortening may contribute to chromosomal instability and the emergence of unstable derivatives, as the chromosome 7 derivatives displayed loss or rearrangement of subtelomeric regions. Moreover, we identified variants in genes implicated in telomere biology disorders and observed specific genetic mutation profiles associated with different chromosome 7 abnormalities. These findings shed light on a potential mechanism leading to monosomy 7 through the evolution of chromosome 7q abnormalities. Identifying patients at risk of developing monosomy 7, based on the presence of unstable derivatives with telomere loss or a specific mutation profile, could potentially enhance patient management and guide the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd97cfdef751d5eb8d87f19388c6ddb34eef62a2" target='_blank'>
              Emergence and Cytogenetic Clonal Evolution of Chromosome 7 Abnormalities in Myeloid Malignancies: Investigating the Role of Telomere Dysfunction
              </a>
            </td>
          <td>
            Carmen Baldazzi, L. Bandini, V. Robustelli, A. Patuelli, C. Venturi, Alessandra Grassi, G. Marzocchi, Angela Ielpo, V. Solli, M. Bochicchio, Stefania Paolini, Chiara Sartor, F. Zingarelli, Antonio Curti, E. Ottaviani, N. Testoni
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09704648420ca313fc3a94dc7cc40706df2fbbe8" target='_blank'>
              Blood-Based Epigenetic Instability Linked to Human Aging and Disease
              </a>
            </td>
          <td>
            Salman Basrai, Ido Nofech-Mozes, Rajesh Detroja, Fernando L. Scolari, Mehran Bakhtiari, Andrea Arruda, Tracy Murphy, Scott V. Bratman, Steven M. Chan, , , D. D. H. Kim, R. Kridel, Filio Billia, Sagi Abelson
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54d1904727e65e93a89095c3025ebc1f384c120" target='_blank'>
              A genome-wide atlas of human cell morphology
              </a>
            </td>
          <td>
            Meraj Ramezani, Erin Weisbart, J. Bauman, Avtar Singh, John Yong, Maria Lozada, Gregory P. Way, Sanam L. Kavari, Celeste Diaz, Eddy Leardini, Gunjan Jetley, Jenlu Pagnotta, Marzieh Haghighi, Thiago M. Batista, Joaquín Pérez-Schindler, M. Claussnitzer, Shantanu Singh, B. Cimini, P. Blainey, A. Carpenter, Calvin H Jan, James T Neal
          </td>
          <td>2025-01-27</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7375f0846811e9250a784d2b67a30ac0f6b0da37" target='_blank'>
              Clonal hematopoiesis: elements associated with clonal expansion and diseases
              </a>
            </td>
          <td>
            Gangpyo Ryu, Youngil Koh, S. Jaiswal, S. Yoon
          </td>
          <td>2025-03-13</td>
          <td>Blood Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Objective. Unlike genetic changes, epigenetic aberrations in prostate cancer can be reversed under the influence of a chemical agent. This fact makes the study of epigenetic changes an important object as potential therapeutic targets. Material and methods. PubMed, Medline, eLibrary.ru databases were analyzed for the keywords: epigenetic prostate cancer, lineage plasticity, neuroendocrine differentiation. For this literature review, 84 relevant publications were selected. The review included studies from 1982 to 2024. Results. The most widely studied epigenetic mutations are DNA hypo- and hypermethylation, histone variability (methylation and acetylation), and neuroendocrine differentiation. Conclusion. The study of the genomic landscape can reveal new opportunities for improving the diagnosis and therapy of castration-resistant prostate cancer (CRPC), which is a potentially lethal form of the disease. It is important not only to search for new biomarkers to identify genetic disorders, but also to study the optimal therapy for advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6160d8f350a3cb980348d4d0719c54ff7d5b6123" target='_blank'>
              Epigenetic abnormalities and neuroendocrine differentiation in prostate cancer
              </a>
            </td>
          <td>
            G. Kovchenko, A. Sivkov, L. N. Lyubchenko, A. Kaprin
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea61347bed9e76289661e280b6f539cf9500a162" target='_blank'>
              Neoantigens: new hope for cancer therapy
              </a>
            </td>
          <td>
            Yitong Hu, Tengda Zhou, Ping Cai, Zihao He
          </td>
          <td>2025-03-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc5699f8d775d7c0b21dfb5b4e13161b3a6de2f" target='_blank'>
              Appraisal of CRISPR Technology as an Innovative Screening to Therapeutic Toolkit for Genetic Disorders.
              </a>
            </td>
          <td>
            A. Shahid, Ambreen Zahra, S. Aslam, Amen Shamim, Waqas Rafique Ali, Bilal Aslam, Sultan Habibullah Khan, Muhammad Imran Arshad
          </td>
          <td>2025-02-02</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055214dbdb29cbe5585cc333e89737a58f2ba504" target='_blank'>
              The Epigenetic Landscape: From Molecular Mechanisms to Biological Aging.
              </a>
            </td>
          <td>
            Rachel Evangelina, Subhashree Ganesan, Melvin George
          </td>
          <td>2025-03-17</td>
          <td>Rejuvenation research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is the most prevalent type of malignant epithelial neoplasm that affects the oral cavity. It has been a significant health concern in many countries for a long time since it was usually treated with surgery, radiation, and/or chemotherapy. Drug resistance is the primary issue in patient populations and scientific research, which promotes OSCC tumour cell invasion and migration. Thus, identifying highly specific therapeutic targets could be the potential approach for more successful treatment of OSCC. It is still challenging to understand the genetic causes of oral carcinogenesis due to its highly varied clinic-pathological parameters. It is important to remember that signaling channels and complexes that affect chromatin accessibility control gene expression, which in turn affects cell development and differentiation. Histones undergo post-translational alteration to give this platform. Understanding the processes of gene regulation through histone methylation and its modifications could enhance the early detection, prognostic prediction, and therapy of OSCC. To be properly used as a therapeutic target, histone methylation in OSCC requires more investigation. This review details the dysregulated histone methylation and the modifying enzymes linked to the development and aetiology of OSCC. Furthermore, the part that lysine methylation plays in cell migration, chemo-resistance, and OSCC invasion is also investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ac8fe27c84ad3ea1843eef61f7ad80802b8dbb" target='_blank'>
              Molecular insight into histone methylation as a novel target for oral squamous cell carcinoma: future hope in personalised medicine
              </a>
            </td>
          <td>
            P. Rajendran, Monisha Prasad, Enas M. Ali, Ramya Sekar, Abdullah M AlZahrani, M. Karobari, M. A. M. Genena, B. Abdallah
          </td>
          <td>2025-01-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fe7537a61f4a7001d830003b960ec3b41322ce" target='_blank'>
              Increased interferon I signaling, DNA damage response and evidence of T-cell exhaustion in a patient with combined interferonopathy (Aicardi-Goutières Syndrome, AGS) and cohesinopathy (Cornelia de Lange Syndrome, CdLS)
              </a>
            </td>
          <td>
            Sorina Boiu, Nikolaos Paschalidis, G. Sentis, T. Manolakou, A. Nezos, Manolis Gialitakis, M. Grigoriou, E. Atsali, Melpomeni Giorgi, A. Ntinopoulos, Clio Mavragani, Periklis Makrythanasis, Dimitrios T Boumpas, A. Banos
          </td>
          <td>2025-01-27</td>
          <td>Pediatric Rheumatology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Kinetoplastid parasites cause diseases that threaten human and animal health. To survive transitions between vertebrate hosts and insect vectors, these parasites rely on precise regulation of gene expression to adapt to environmental changes. Since gene regulation in kinetoplastids is primarily post-transcriptional, developing efficient genetic tools for modifying genes at their endogenous loci while preserving regulatory mRNA elements is crucial for studying their complex biology. We present a CRISPR/Cas9-based tagging system that preserves untranslated regulatory elements and uses a viral 2A peptide from Thosea asigna to generate two separate proteins from a single transcript: a drug-selectable marker and a tagged protein of interest. This dual-function design maintains native control elements, allowing discrimination between regulation of transcript abundance, translational efficiency, and post-translational events. We validate the system by tagging six Trypanosoma brucei proteins and demonstrate (i) high-efficiency positive selection and separation of drug-selectable marker and target protein, (ii) preservation of regulatory responses to environmental cues like heat shock and iron availability, and (iii) maintenance of stage-specific regulation during developmental transitions. This versatile toolkit is applicable to all kinetoplastids amenable to CRISPR/Cas9 editing, providing a powerful reverse genetic tool for studying post-transcriptional regulation and protein function in organisms where post-transcriptional control is dominant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382fcafb0f0fec787fffc5d8fb4bf2648ec96055" target='_blank'>
              A high-throughput protein tagging toolkit that retains endogenous untranslated regions for studying gene regulation in kinetoplastids
              </a>
            </td>
          <td>
            Carla Gilabert Carbajo, Xiaoyang Han, Bhairavi Savur, Arushi Upadhyaya, Fatima Taha, Michele Tinti, Richard J. Wheeler, Phillip A Yates, Calvin Tiengwe
          </td>
          <td>2025-02-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963a56be4a953e4f275f13aa1a14a7847b02c333" target='_blank'>
              Finding a needle in a haystack: functional screening for novel targets in cancer immunology and immunotherapies
              </a>
            </td>
          <td>
            Yi Du, Yang Yang, Bohao Zheng, Qian Zhang, Shengtao Zhou, Linjie Zhao
          </td>
          <td>2025-01-25</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6061f6b22f4eeaa5754cf5de2d91103b0a9be5f" target='_blank'>
              Application of single-cell sequencing technology and its clinical implications in Parkinson’s disease and Alzheimer’s disease: a narrative review
              </a>
            </td>
          <td>
            Zhonghao Chen, Jack Shi, Longfei Li
          </td>
          <td>2025-02-06</td>
          <td>Advanced Technology in Neuroscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/196e4f5c9c9b8a27230e96850b6632f862c0dcae" target='_blank'>
              SKI complex loss renders 9p21.3-deleted or MSI-H cancers dependent on PELO
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony C Lau, Lucy Parker-Burns, Daniel A Lubicki, Tianxia Li, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Tessla Chan, Riya V Kishen, Anisah Adeagbo, Srivatsan Raghavan, Elisa Aquilanti, John R. Prensner, J. Krill-Burger, Todd R. Golub, Catarina D. Campbell, Joshua M Dempster, Edmond M Chan, Francisca Vazquez
          </td>
          <td>2025-02-05</td>
          <td>Nature</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4233622be9dcbd1fb28cce4bd3a5ada680860" target='_blank'>
              Towards understanding cancer dormancy over strategic hitching up mechanisms to technologies
              </a>
            </td>
          <td>
            Sumin Yang, Jieun Seo, Jeonghyeon Choi, Sung-Hyun Kim, Yunmin Kuk, Kyung Chan Park, Mingon Kang, Sangwon Byun, Jae-Yeol Joo
          </td>
          <td>2025-02-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary NRAS proto-oncogene GTPase (NRAS) G12 mutations are generally considered to be activating mutations, but little is known about how they may differ. DEAPR, a novel approach for evaluating RNA-sequencing results, was used to identify the most prominent and reproducible changes to an AML cell line (THP-1) when the NRASG12D mutation was replaced by an NRASG12V mutation (B11 cells). The results demonstrate that the most common changes involved cell cycle activity. However, there was a dramatic difference in genes involved in the innate immune response, thus suggesting opportunities for NRAS G12 mutant-specific treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab2ab0d668d8eb9655fff3905a8185ad21747d0" target='_blank'>
              DEAPR: Differential Expression and Pathway Ranking Tool Demonstrates NRASG12V and NRASG12D Mutations Have Differing Effects in THP-1 Cells
              </a>
            </td>
          <td>
            S. Rathe, Jeremy P. White, Z. Sachs, D. A. Largaespada
          </td>
          <td>2025-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Melanoma is the deadliest form of skin cancer, and its treatment poses significant challenges due to its aggressive nature and resistance to conventional therapies. Long non-coding RNAs (lncRNAs) represent a new frontier in the search for suitable targets to control melanoma progression and invasiveness. Indeed, lncRNAs exploit a wide range of regulatory functions along chromatin remodeling, gene transcription, post-transcription, transduction, and post-transduction to ultimately tune multiple cellular processes. The understanding of this intricate and flexible regulatory network orchestrated by lncRNAs in pathological conditions can strategically support the rational identification of promising targets, ultimately speeding up the setup of new therapeutics to integrate the currently available approaches. Here, the most recent findings on lncRNAs involved in melanoma will be analyzed. In particular, the functional links between their mechanisms of action and some frequently underestimated features, like their different subcellular localizations, will be highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08119f9fccb5c4f5a79dd1ca1781581cce224f1" target='_blank'>
              Unravelling the Regulatory Roles of lncRNAs in Melanoma: From Mechanistic Insights to Target Selection
              </a>
            </td>
          <td>
            Beatrice Moras, C. Sissi
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69e4dbaba941d1cd07858705c4708f9bc19fb8f" target='_blank'>
              Targeting PI3K inhibitor resistance in breast cancer with metabolic drugs
              </a>
            </td>
          <td>
            N. Gremke, I. Besong, Alina Stroh, Luise von Wichert, Marie Witt, S. Elmshäuser, M. Wanzel, Martin F Fromm, R. V. Taudte, S. Schmatloch, T. Karn, M. Reinisch, N. Hirmas, S. Loibl, Thomas Wündisch, Anne-Sophie Litmeyer, Paul Jank, Carsten Denkert, Sebastian Griewing, U. Wagner, Thorsten Stiewe
          </td>
          <td>2025-03-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Summary Recently formed allopolyploid species offer unprecedented insights into the early stages of polyploid evolution. This review examines seven well‐studied neopolyploids (we use ‘neopolyploid’ to refer to very recently formed polyploids, i.e. during the past 300 years), spanning different angiosperm families, exploring commonalities and differences in their evolutionary trajectories. Each neopolyploid provides a unique case study, demonstrating both shared patterns, such as rapid genomic and phenotypic changes, and unique responses to hybridization and genome doubling. While previous studies of these neopolyploids have improved our understanding of polyploidy, significant knowledge gaps remain, highlighting the need for further research into the varied impacts of whole‐genome duplication on gene expression, epigenetic modifications, and ecological interactions. Notably, all of these neopolyploids have spontaneously arisen due to human activity in natural environments, underscoring the profound consequences of polyploidization in a rapidly changing world. Understanding the immediate effects of polyploidy is crucial not only for evolutionary biology but also for applied practices, as polyploidy can lead to novel traits, as well as stress tolerance and increased crop yields. Future research directions include investigating the genetic and epigenetic mechanisms underlying polyploid evolution, as well as exploring the potential of neopolyploids for crop improvement and environmental adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5a204f000a3e9de1ebd9bce03b0ead5230c52b" target='_blank'>
              Natural neopolyploids: a stimulus for novel research
              </a>
            </td>
          <td>
            P. Edger, D. Soltis, Shunsuke Yoshioka, Mario Vallejo-Marín, Rie Shimizu‐Inatsugi, Kentaro K. Shimizu, A. Salmon, Simon Hiscock, Malika Ainouche, Pamela S Soltis
          </td>
          <td>2025-02-14</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing.
              </a>
            </td>
          <td>
            Qiuhui Li, Ayse G Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb803b3e058115ac9f37e42f7b7a8e8748c0c02" target='_blank'>
              Next-Generation Sequencing-Based In Silico Transgene Integration Profiling Tool Accelerates Cell Line Genetic Characterizations.
              </a>
            </td>
          <td>
            Zhenqiu Huang, Hyo-Young Jeong, Luke Nelson, Caitlyn Devine, Guanghua Benson Li, Lin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Biotechnology journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ERBB2 (HER2) is a well-studied oncogene with several driver mutations apart from the well-known amplification defect in some breast cancers. We used the GigaAssay to test the functional effect of HER2 missense mutations on its receptor tyrosine kinase function. The GigaAssay is a modular high-throughput one-pot assay system for simultaneously measuring molecular function of thousands of genetic variants at very high accuracy. The activities of 5,886 mutations were classified, significantly more than mutants previously reported. These variants include 112 new in vitro, 10 known, and 9 new in vivo gain-of-function (GOF) mutations. Many of the GOFs spatially cluster in sequence and structure, supporting the activation mechanisms of heterodimerization with EGFR and release of kinase inhibition by the juxtamembrane domain. Retrospective analysis of patient outcomes from the Genomic Data Commons predicts increased survival with the newly identified HER2 GOF variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b136d3cab282e49a197992ab3575cc9958742822" target='_blank'>
              New Gain-of-Function Mutations Prioritize Mechanisms of HER2 Activation
              </a>
            </td>
          <td>
            C. Giacoletto, Liz J. Valente, Lancer Brown, Sara Patterson, Rewatee H Gokhale, S. Mockus, Wayne W. Grody, Hong-Wen Deng, Jerome I Rotter, Martin R. Schiller
          </td>
          <td>2025-03-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain- and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Comprehensive and systematic proteome-wide experiments require financial and technical resources unavailable to most researchers. Here we describe a scalable CRISPR/Cas9-non-homologous end joining (NHEJ) based method for Pooled Recombinant Integration of Seamless Markers (PRISM) into protein coding genes. We created two gRNA libraries for 5’- and 3’-tagging of 18,804 human protein-coding genes. Selection for in-frame integration of the donor cassette can be guaranteed by fusing it to an antibiotic resistance enzyme (ARE) and P2A self-cleaving peptide, resulting in tagged proteins and free ARE to select for clones with inserted donor cassettes. We achieved a library integration rate of 19.75% and tagging of ∼80% for genes expressed in Hek293T cells and notably, 89.7% of essential genes, with donor DNA. Our strategy is scalable, specific, and selective, paving the way for genome-scale construction of human cell lines tagged with different types of reporter genes for protein functional characterization. Significance statement Here we report the first practical method to achieve near complete 5’- or 3’-end integration of reporter protein-coding sequences to express seamless fusions of human protein and reporter proteins that we call Pooled Recombinant Integration of Seamless Markers (PRISM). PRISM is independent of homology templates, yet it works with high efficiency and precision. Using gRNA-directed CRISPR/Cas9 genome editing and mini-plasmid donor cassettes we were able to tag about 80% of the protein-coding sequences of human genes with a green florescent protein at an integration efficiency of about 20% and most notably, of 90% of essential genes in the human cell line HEK 293T. This is important because the characterization of gene functions in a particular cell type usually begins with essential genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834136786fe808caa704ff04cd6e336c885cd2d" target='_blank'>
              A strategy for genome-wide seamless tagging of human protein-coding genes
              </a>
            </td>
          <td>
            Minoo Karimi, Masih M. Saber, Nicolas Stifani, Louis Gauthier, Adrian Serohijos, S. Michnick
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee87ee3aa4ca681ef934614f3f0d367c700553ae" target='_blank'>
              Blast Transformation of Chronic Myeloid Leukemia Caused by Genetic Alterations, Epigenetic Dysfunctions and Aberrantly Regulated Factors in Bone Marrow Microenvironment
              </a>
            </td>
          <td>
            MD Mitsuru Sakitani
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Research in  Medical and Clinical Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="Epithelial Ovarian Cancer (EOC) is the most lethal gynecologic cancer and those affected are in urgent need of new therapeutic strategies. Standard treatment is surgery followed by taxane- and platinum-based chemotherapy, but the rate of relapse is high and the 5-year survival is only 45%. Oncolytic viruses (OV) are a promising approach to EOC therapy through remodeling the immune composition of the tumor microenvironment (TME). Treatment response in EOC tumors can differ based on the presence of key tumorigenic mutations. This study evaluated the impact of specific tumor mutations on the response to the current standard of care carboplatin, two promising OV candidates VSV∆M51 and MG1, an infected cell vaccine (ICV-MG1) regimen, and the anti-angiogenic drug Fc3TSR. Mice with tumors harboring constitutive K-Ras activation showed an enhanced response to carboplatin and VSV∆M51 treatment. Additionally, VSV∆M51 treatment prolonged survival of syngeneic mice bearing tumors with mutations in Pten and Kras, Pten and Trp53, or Trp53 and Brca2 with increased activation of CD4+ and CD8+ T lymphocytes in the peritoneal TME. To enhance OV potency, an MG1-based infected cell vaccine inducing expression of IL-21 or IL15+21 was developed and found to enable strong and long-lasting antitumoral immunity in two carboplatin refractory syngeneic models, ID8 Trp53-/- and STOSE. VSV∆M51 combined with the anti-angiogenic Fc3TSR enhanced efficacy in the ID8 model. In summary, OV-based immunotherapy has shown promise in diverse murine models of EOC-bearing clinically relevant mutations, thus laying the foundation for developing new OV-based strategies to target a large spectrum of EOC genotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0be8c226447376238aa375a0e7bc3031f54f154" target='_blank'>
              Specific Genetic Mutations Impact Chemotherapy Resistance and Therapeutic Efficacy of Oncolytic Viruses in Ovarian Cancer.
              </a>
            </td>
          <td>
            Alison O Cudmore, G. Rodriguez, Vincent Maranda, Salar Farokhi Boroujeni, Humaira Murshed, Elizabeth A Macdonald, Melanie Grondin, K. Garson, K. Matuszewska, J. Diallo, James J Petrik, B. Vanderhyden
          </td>
          <td>2025-03-10</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a596476c40bed6ddb55424b08c446ae8d78faf" target='_blank'>
              Decoding the Epigenome of Breast Cancer
              </a>
            </td>
          <td>
            Elisa Cortellesi, Isabella Savini, Matteo Veneziano, Alessandra Gambacurta, M. Catani, V. Gasperi
          </td>
          <td>2025-03-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Aging is a significant risk factor for male fertility and can lead to severe developmental disorders in offspring. It disrupts testicular function and spermatogenesis, resulting in sperm abnormalities and DNA fragmentation. Male aging alters the genome and epigenome of germ cells due to persistent oxidative stress caused by the cumulative effects of environmental factors over a lifetime. At the molecular level, DNA damage occurs and is poorly repaired due to impaired DNA repair pathways, leading to unrepaired lesions and de novo mutations. Aging also creates distinct epigenetic landscapes that modify gene expression in germ cells, affect the DNA damage response, and generate de novo DNA and epigenetic mutations that are transmitted to the sperm and inherited by the offspring. This review discusses current knowledge on the age-associated effects on male germ cells and the genomic and epigenomic mechanisms contributing to altered male reproductive health and outcomes in progeny. We propose a male reproductive aging threshold, where cumulative exposure to risk factors leads to oxidative stress, impaired spermatogenesis, and altered reproductive outcomes. Finally, we discuss novel interventions to prevent premature testicular aging and emphasize the need for public health policies and counseling guidelines for men seeking paternity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d844e8efcdd820138081e295a9dc2bad6b75fd5" target='_blank'>
              Male aging in germ cells: What are we inheriting?
              </a>
            </td>
          <td>
            Arturo Elías-Llumbet, Sebastián Lira, Marcia Manterola
          </td>
          <td>2025-02-18</td>
          <td>Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739d0a74d9a81f832abc4486254c5ebe8688f593" target='_blank'>
              Abstract A018: Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain
              </a>
            </td>
          <td>
            N. Dasyam, Yasmin Nouri, J. Halpin, A. McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, M. Templin, Qin Le, Peng Li, I. Hermans, Rachel Perret, R. Weinkove
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0143151fabb07316572f667e24a48e297b397f77" target='_blank'>
              Toti: an integrated multi-omics database to decipher the epigenetic regulation of gene expression in totipotent stem cells
              </a>
            </td>
          <td>
            Yi Chai, Ruiying Zhang, Shunze Jia, Danfei Zhu, Siyi Chen, Xudong Fu, Xin Sheng
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract The rising trend in global cancer incidence has caused widespread concern, one of the main reasons being the aging of the global population. Statistical data show that cancer incidence and mortality rates show a clear upward trend with age. Although there is a commonality between dysregulated nutrient sensing, which is one of the main features of aging, and metabolic reprogramming of tumor cells, the specific regulatory relationship is not clear. This manuscript intends to comprehensively analyze the relationship between senescence and tumor metabolic reprogramming; as well as reveal the impact of key factors leading to cellular senescence on tumorigenesis. In addition, this review summarizes the current intervention strategies targeting nutrient sensing pathways, as well as the clinical cases of treating tumors targeting the characteristics of senescence with the existing nanodelivery research strategies. Finally, it also suggests sensible dietary habits for those who wish to combat aging. In conclusion, this review attempts to sort out the link between aging and metabolism and provide new ideas for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0661e30e7288383f69d3a83270f071d914ccb70" target='_blank'>
              Metabolic reprogramming in cancer and senescence
              </a>
            </td>
          <td>
            Yuzhu Zhang, Jiaxi Tang, Can Jiang, Hanxi Yi, Shu Guang, Gang Yin, Maonan Wang
          </td>
          <td>2025-03-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cells accumulate mutations throughout development, contributing to cancer, aging, and evolution. Quantitative data on the abundance of de novo mutations within plants or animals are limited, as new mutations are often rare within a tissue and fall below the limits of current sequencing depths and error rates. Here, we show that mutations induced by the maize Mutator (Mu) transposon can be reliably quantified down to a detection limit of 1 part in 12,000. We measured the abundance of millions of de novo Mu insertions across four tissue types. Within a tissue, the distribution of de novo Mu allele frequencies was highly reproducible between plants, showing that, despite the stochastic nature of mutation, repeated statistical patterns of mutation abundance emerge. In contrast, there were significant differences in the allele frequency distribution between tissues. At the extremes, root was dominated by a small number of highly abundant de novo insertions, while endosperm was characterized by thousands of insertions at low allele frequencies. Finally, we used the measured pollen allele frequencies to reinterpret a classic genetic experiment, showing that evidence for late Mu activity in pollen are better explained by cell division statistics. These results provide insight into the complexity of mutation accumulation in multicellular organisms and a system to interrogate the factors that shape mutation abundance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3df0d169f514a87d0e0fd7316c58b59b7680232" target='_blank'>
              Ultra-deep sequencing of somatic mutations induced by a maize transposon
              </a>
            </td>
          <td>
            Justin Scherer, Michael Hinczewski, B. Nelms
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3959b933b4c238cc534f4bd3bbf4f10f6618c5" target='_blank'>
              Recent Advances in Anti-Aging Therapeutic Strategies Targeting DNA Damage Response and Senescence-Associated Secretory Phenotype-Linked Signaling Cascade.
              </a>
            </td>
          <td>
            Jawad Nadeem, Razia Sultana, Amna Parveen, Sun Yeou Kim
          </td>
          <td>2025-02-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185bf82df0621d29d2214d3aa74691b2937269d4" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The canonical DNA damage response (DDR) maintains genome stability, involving DNA synthesis/cell cycle arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously discovered and characterized a noncanonical cellular response that is specific to nonblocking replication stress, i.e., low-level stress (LoL-DDR), in primary cells. Although this response generates replication stress-induced reactive oxygen species (RIR), it triggers a program that prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a precise threshold triggers the canonical DDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: 1-RIR are not detoxified under high-level stress conditions, resulting in a continuous increase in the dose-response curve of RIR production; 2-RIR are not produced by NADPH oxidases; and 3-replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” involving replication-stress and ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfa2fab8a78a43f5b506788fc0cb6d9a375f31e" target='_blank'>
              Dysregulation of replication stress-induced ROS in transformed cell lines: a vicious circle at cancer initiation
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Comprehensive genetic analysis of tumors with exome or whole genome sequencing has enabled the identification of the genes that are recurrently mutated in cancer. This has stimulated a series of exciting advances over the past 15 years, guiding us to new molecular biomarkers and therapeutic targets among the common mature B-cell neoplasms. In particular, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL) and Burkitt lymphoma (BL) have each been the subject of considerable attention in this field. Currently, more than 850 genes have been reported as targets of protein-coding mutations in at least one of these entities. To reduce this to a manageable size, we describe a systematic approach to prioritize and categorize these genes, based on the quality and type of supporting data. For each entity, we provide a list of candidate driver genes categorized into Tier 1 (high-confidence genes), Tier 2 (candidate driver genes), or Tier 3 (lowest-confidence genes). Collectively, this reduces the number of high-confidence genes for these three lymphomas to a mere 144. This further affirms the substantial overlap between the genes relevant in DLBCL and each of FL and BL. These highly curated and annotated gene lists will continue to be maintained as a resource to the community. These results emphasize the extent of the knowledge gap regarding the role of each of these genes in lymphomagenesis. We offer our perspective on how to accelerate the experimental confirmation of drivers using a variety of model systems, using these lists as a guide for prioritizing genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf53884a55922c51a797521fda638d01af815c" target='_blank'>
              Audit of B-cell Cancer Genes.
              </a>
            </td>
          <td>
            K. Coyle, K. Dreval, Daniel J Hodson, R. D. Morin
          </td>
          <td>2025-01-24</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9124f513a579068358157bc6335b2150cb3dde2" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma
              </a>
            </td>
          <td>
            Lays Martin Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cb9fafc9e4ae8646c09de8af8d088ea1242ab7" target='_blank'>
              Mammalian Species-Specific Resistance to Mammary Cancer
              </a>
            </td>
          <td>
            I. Barash
          </td>
          <td>2025-03-06</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0687d954e3a7ce3f1c8da610c5b175b62bd0c491" target='_blank'>
              Single-cell multiomics reveals a gene regulatory circuit driving leukemia cell differentiation.
              </a>
            </td>
          <td>
            Xin Tian, Liuqingqing Zhang, Guiqiyang Xiang, Yijia Tang, Ping Zhu, Shuting Yu, Fangying Jiang, Shuai Wang, Jinzeng Wang, Yao Dai, Desheng Zheng, Jianbiao Wang, Xiangqin Weng, Shengyue Wang, Yun Tan, Feng Liu
          </td>
          <td>2025-02-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f255f1e2df6321b7a03cd03e70b813bcc99665" target='_blank'>
              Current Applications of Single-Cell RNA Sequencing in Glioblastoma: A Scoping Review
              </a>
            </td>
          <td>
            Edgar G. Ordóñez-Rubiano, Nicolás Rincón-Arias, William J. Shelton, A. F. Salazar, María Alejandra Sierra, Raphael Bertani, D. Gómez-Amarillo, Fernando Hakim, Matías Baldoncini, César Payán-Gómez, A. Cómbita, Sandra C. Ordonez-Rubiano, Rafael Parra-Medina
          </td>
          <td>2025-03-14</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09992e3740e0c66b2f55eb33131f8350c3e4f847" target='_blank'>
              Cancer-independent somatic mutation of the wild-type NF1 allele in normal tissues in neurofibromatosis type 1
              </a>
            </td>
          <td>
            Thomas R. W. Oliver, A. Lawson, H. Lee-Six, Anna Tollit, Hyunchul Jung, Y. Hooks, R. Sanghvi, Matthew D. Young, T. Butler, Pantelis Nicola, T. Treger, S. Lensing, G. A. Burke, Kristian Aquilina, Ulrike Löbel, I. Cortés-Ciriano, Darren Hargrave, Mette Jorgensen, Flora A. Jessop, Tim H. H. Coorens, Adrienne M. Flanagan, K. Allinson, I. Martincorena, Thomas S Jacques, S. Behjati
          </td>
          <td>2025-02-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c9245366839c365f2d0640fb23a67a476978edf" target='_blank'>
              VDAC2 loss elicits tumour destruction and inflammation for cancer therapy.
              </a>
            </td>
          <td>
            Sujing Yuan, Renqiang Sun, Hao Shi, Nicole M. Chapman, Haoran Hu, C. Guy, Sherri Rankin, Anil Kc, Gustavo Palacios, Xiaoxi Meng, Xiang Sun, Peipei Zhou, Xiaoyang Yang, Stephen Gottschalk, Hongbo Chi
          </td>
          <td>2025-03-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) remains a serious threat to global health, with rising incidence and mortality rates. Therapeutic options for advanced HCC are quite limited, and the overall prognosis remains poor. Recent advancements in immunotherapy, particularly immune-checkpoint blockade (ICB) targeting anti-PD1/PD-L1 and anti-CTLA4, have facilitated a paradigm shift in cancer treatment, demonstrating substantial survival benefits across various cancer types, including HCC. However, only a subset of HCC patients exhibit a favorable response to ICB therapy, and its efficacy is often hindered by the development of resistance. There are many studies to explore the underlying mechanisms of ICB response. In this review, we compiled the latest progression in immunotherapies for HCC and systematically summarized the sophisticated mechanisms by which components of the tumor microenvironment (TME) regulate resistance to ICB therapy. Additionally, we also outlined some scientific rationale strategies to boost antitumor immunity and enhance the efficacy of ICB in HCC. These insights may serve as a roadmap for future research and help improve outcomes for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7cb79517a5aec504caaabf1fd85cb4e7d72745b" target='_blank'>
              Sophisticated roles of tumor microenvironment in resistance to immune checkpoint blockade therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yi-Zhe Zhang, Yunshu Ma, Ensi Ma, Xizhi Chen, Yue Zhang, Baobing Yin, Jing Zhao
          </td>
          <td>2025-02-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The acquisition of drug resistance poses a significant challenge for successful cancer treatment. Understanding the mechanisms by which cancer evades drug action is crucial for developing effective therapies. PARP inhibitors (PARPi) induce synthetic lethality in BRCA-deficient cancer cells; however, these cells eventually develop PARPi resistance. Here, we demonstrate that prolonged exposure of BRCA-deficient cancer cells to PARPi or Cisplatin leads to alterations in YAP1 pre-mRNA splicing patterns, resulting in selective upregulation of a minor YAP1 isoform, YAP1-2α. Elevated YAP1-2α specifically heterodimerizes with TAZ, triggering liquid-liquid phase separation (LLPS). This process generates nuclear condensates that activate super-enhancers containing the YAP1-2α/TAZ heterodimer, TEAD, and BRD4. These super-enhancers (here referred to as YAP1-2α/TAZ/TEAD super-enhancers) are also activated in CD44(variant)high cancer stem-like populations in parental cells. In both PARPi-resistant and CD44(variant) high cancer cell populations, treatments that disrupt YAP1-2α/TAZ/TEAD super-enhancers abolish cancer stem-like properties. Thus, the activation of YAP1-2α/TAZ/TEAD super-enhancers reinforces cancer cell stemness, which confers resistance to adverse conditions, such as those caused by anticancer drug treatment, including PARPi and Cisplatin. Our investigation reveals that YAP1-2α/TAZ/TEAD super-enhancers transcriptionally facilitate the malignant grade of cancer cells. Disrupting these super-enhancers represents a promising strategy to hinder neoplastic progression and prevent multidrug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a34b15f56fa0ab0ce3e33d624b269f9e4ee39eca" target='_blank'>
              YAP1-2α/TAZ heterodimerization drives phase separation, activating TEAD-positive super-enhancers that enhance cancer stemness and multidrug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1fa79a96dc2bc5104b7e66300caf2368b25848" target='_blank'>
              Ribosome specialization by cancer-associated ribosomal protein mutations: progress made and open questions.
              </a>
            </td>
          <td>
            Marino Caruso, K. De Keersmaecker
          </td>
          <td>2025-03-06</td>
          <td>Philosophical transactions of the Royal Society of London. Series B, Biological sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f154627a6e9fbb35a9c2baccde3a2a62d9723cc8" target='_blank'>
              HIF-1α-induced long noncoding RNA LINC02776 promotes drug resistance of ovarian cancer by increasing polyADP-ribosylation.
              </a>
            </td>
          <td>
            Yangjun Wu, Yu Zeng, Yong Wu, Xinyu Ha, Zheng Feng, Chaohua Liu, Ziqi Liu, Jiajia Wang, X. Ju, Shenglin Huang, Linhui Liang, Bin Zheng, Lulu Yang, Jun Wang, Xiaohua Wu, Shengli Li, Hao Wen
          </td>
          <td>2025-03-01</td>
          <td>Clinical and translational medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ded24f4b8584549c62a69b47b4fbedd7863bbb9" target='_blank'>
              Epigenetic regulators combined with tumour immunotherapy: current status and perspectives.
              </a>
            </td>
          <td>
            Huan Zhang, Yutong Pang, Ling Yi, Xiao-jue Wang, Panjian Wei, Haichao Wang, Shuye Lin
          </td>
          <td>2025-03-21</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412c192baedece3963b4137ef5579bc1b8117afb" target='_blank'>
              Genetic variants in NHEJ1 and related DNA repair disorders: insights into phenotypic heterogeneity and links to hypoplastic myelodysplastic syndromes and familial hematological malignancies susceptibility.
              </a>
            </td>
          <td>
            Mahmoud I Elbadry, Elsayed Abdelkreem, Ahmed Tawfeek, Go Hun Seo, S. Aziz
          </td>
          <td>2025-03-06</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f52bf7d7dd9aa39c810354f1880eb8e042f5dd78" target='_blank'>
              Abstract IA003: Integrated consideration of constitutional and somatic genetic features to improve knowledge and care of AML patients
              </a>
            </td>
          <td>
            Ann-Kathrin Eisfeld
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f316b25c9fd66656561befcef17f13bf1d2230" target='_blank'>
              Mcm5 mutation leads to silencing of Stat1-bcl2 which accelerating apoptosis of immature T lymphocytes with DNA damage
              </a>
            </td>
          <td>
            Min Liu, Yuanyuan Li, Zhilin Deng, Ke Zhang, Shuying Huang, Jiamin Xia, Yi Feng, Yundan Liang, Chengfu Sun, Xindong Liu, Shurong Li, B. Su, Yong Dong, Sizhou Huang
          </td>
          <td>2025-02-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c55ad387d82ce60d43ef750c8b35debeea4164" target='_blank'>
              Programmable mRNA therapeutics for controlled epigenomic modulation of single and multiplexed gene expression in diverse diseases
              </a>
            </td>
          <td>
            Charles W O'Donnell, J. Farelli, Houda Belaghzal, Justin Chen, Lauren Beech, James Sullivan, Chevaun D. Morrison-Smith, Stephen Siecinski, Adam Katz, Samuel Mildrum, Mayur Gurnani, Prachi Dhanania, Caitlyn R Webb, Giuliana Castello Coatti, Pranjali Rumale, Daniel F G Costa, Marcus I Gibson, Yaoyu E. Wang, J. Newman, T. McCauley
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da248ee7a941cb4fe66c4c17c284b6ea6d864ede" target='_blank'>
              Different In Vitro Models of Chronic Myeloid Leukemia Show Different Characteristics: Biological Replicates Are Not Biologically Equivalent.
              </a>
            </td>
          <td>
            Alessia Cavalleri, Besjana Xhahysa, Silvia Mutti, R. Ferraro, E. L. Mazzoldi, M. Farina, A. Leoni, Luca Garuffo, Federica Trenta, Federica Re, V. Radici, Eugenia Accorsi Buttini, E. Morello, Gabriele Magliano, V. Cancelli, S. Giliani, Michele Malagola, Domenico Russo, S. Bernardi
          </td>
          <td>2025-03-01</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Leukemia is characterized by multiple rearrangements of signal transduction genes and overexpression of nonmutated genes, such as transcription factors (TFs) genes. Super-enhancers (SEs) are prevalent in human cancers and are associated with the accumulation of numerous core TFs. SEs drive the expression of core TF genes by delivering robust transcriptional activation signals. Additionally, core TFs sustain the stability and activity of SEs through mutual auto-regulation loops, creating a positive feedback loop known as the Core Transcriptional Regulation Circuit (CRC). Using ChIP-seq data, we identified the involvement of the SE-related gene ZNF217 in acute myeloid leukemia (AML), in which its functional role and underlying mechanism remain unclear. We demonstrated that ZNF217, ELF1, MEF2D, RUNX2, and FOXP1 are likely integral components of the AML CRC through various experimental techniques, including CUT&Tag, short hairpin RNA (shRNA) transduction, and Luciferase reporter assays. Notably, ZNF217 was determined to be indispensable for the proliferation and viability of AML cells both in vitro and in vivo. Subsequent analysis of RNA-seq and CUT&Tag results identified MYB as a key direct target of ZNF217. Overall, our research highlights ZNF217 as a critical oncogene in AML and offers new insights into the transcriptional regulatory mechanisms at play in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/388657aaeaf8efe74b215605c5baaa2e335b1840" target='_blank'>
              Core transcriptional regulatory circuitry molecule ZNF217 promotes AML cell proliferation by up-regulating MYB
              </a>
            </td>
          <td>
            Bi Zhou, Fang Fang, Yongping Zhang, Zhiheng Li, Yixin Hu, Yan Li, Wanyan Jiao, Yumeng Wu, X. Wan, Ying Yang, Fenli Zhang, Ling Xu, Tongting Ji, Jian Pan, Shaoyan Hu
          </td>
          <td>2025-02-18</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eed7b2fdc5b17bdbf958e634178449e843736cc" target='_blank'>
              Genome-Wide CRISPR Screening Identifies Cellular Factors Controlling Nonviral Genome Editing Efficiency
              </a>
            </td>
          <td>
            Shivani Saxena, Meha Kabra, Amr Abdeen, Divya Sinha, Min Zhu, Ruosen Xie, Giovanni Hanstad, Maria A. Fernandez Zepeda, David M. Gamm, B. Pattnaik, Shaoqin Gong, Krishanu Saha
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/001a8921a4a19ca700e79dd988864ea673f09f50" target='_blank'>
              Gain of 20q11.21 in human pluripotent stem cells enhances differentiation to retinal pigment epithelium
              </a>
            </td>
          <td>
            L. Vitillo, Fabiha Anjum, Zoe Hewitt, Owen Laing, N. Ababneh, Duncan Baker, Ivana Barbaric, Peter J Coffey
          </td>
          <td>2025-02-21</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb9672d2fcbfed129bd061551ff24b1fe5d4981" target='_blank'>
              FOSL1 drives the malignant progression of pancreatic cancer cells by regulating cell stemness, metastasis and multidrug efflux system
              </a>
            </td>
          <td>
            Xiaolong Liu, Xueyan Zhang, Tingyu Zeng, Yali Chen, Liu Ye, Shuping Wang, Yulan Li
          </td>
          <td>2025-03-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            M. U. Ahmad, Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/effa3cdf47f124f2fc25cc824cb0bcd26047dfd8" target='_blank'>
              Comprehensive dissection of cis-regulatory elements in a 2.8 Mb topologically associated domain in six human cancers
              </a>
            </td>
          <td>
            Christina M. Caragine, Victoria T Le, Meer Mustafa, Bianca Jay Diaz, John A Morris, Simon Müller, A. Méndez-Mancilla, Evan Geller, Noa Liscovitch-Brauer, Neville E. Sanjana
          </td>
          <td>2025-02-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10a2ab1ac270b68e0530d0e78e9d779c180238b" target='_blank'>
              Histone Modifications and DNA Methylation in Psoriasis: A Cellular Perspective.
              </a>
            </td>
          <td>
            Jing Pan, Siji Chen, Xianzhen Chen, Yinjing Song, Hao Cheng
          </td>
          <td>2025-01-27</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K. Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a517176c60f9a80290d19966f93c06f80af4cac7" target='_blank'>
              Emerging technologies of single-cell multi-omics.
              </a>
            </td>
          <td>
            Yi June Kim, Koichi Takahashi
          </td>
          <td>2025-02-06</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f07daeb74e80fef9e4142d5ed3323341effa067" target='_blank'>
              Abstract B090: ATRA reprograms neutrophils in the tumor microenvironment and overcomes resistance to immune checkpoint inhibition in murine models of LKB1-deficient non-small cell lung cancer
              </a>
            </td>
          <td>
            William P Crosson, Rui Li, R. Salehi-Rad, Bitta Kahangi, Samantha Chin, R. Lim, J. Abascal, E. P. Reyes, M. Oh, C. Dumitras, Nico Edgar, Ryan Chew, Rashel Jacobo, Adam Tfayli, K. Krysan, L. Tran, Steven M Dubinett, Bin Liu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>