<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-27 09:32:36 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>62</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5ee3b9a577273caee9d178991970953321a673" target='_blank'>
              Integrated Genomic Profiling Reveals Mechanisms of Broad Drug Resistance and Opportunities for Phenotypic Reprogramming in Cancer Cells
              </a>
            </td>
          <td>
            Ian Mersich, Brian S. J. Blagg, Aktar Ali
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Platinum-based chemotherapy resistance remains a critical barrier in colorectal cancer (CRC) treatment. In this study, cytogenetic karyotyping was combined with transcriptomic profiling (RNA-seq) to elucidate resistance mechanisms by comparing two CRC cell lines: oxaliplatin-sensitive HCT116 and its resistant derivative HCT116 oxpl-R. Karyotyping unveiled tetraploidization and extensive genomic rearrangements in resistant cells, accompanied by prominent transcriptomic changes: 1807 differentially expressed genes (1216 upregulated and 519 downregulated). Pathway enrichment highlighted altered redox homeostasis and metabolic adaptation. Specifically, HCT116 oxpl-R cells exhibited elevated reactive oxygen species (ROS) production and enhanced energy metabolism. These findings establish a direct association between structural genomic alterations, transcriptional rewiring, and functional phenotypes in platinum resistance, providing a framework for targeting metabolic vulnerabilities in refractory CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1c5397ea3bf691ad1c8a6e4e4cf327eb00a1d" target='_blank'>
              Transcriptomic and Cytogenetic Analysis of Oxaliplatin-Resistant Colorectal Adenocarcinoma HCT116 Cells to Identify Markers Associated with Platinum Resistance
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, Ksenia Fedotova, Natalija Yartseva, Nikolay Aksenov, M. Igotti
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide vari- ants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alter- ations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational frame- work for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by indepen- dent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            Saeid Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01966-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, , Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, Rosa Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G Swanson, Jeffrey M. Weiss, SMaHT Assembly, Claudia Carvalho, Tim H. H. Coorens, Kelley Harris, , E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Chemotherapy is a foundational element in cancer therapy; however, the development of resistance to antineoplastic agents presents a significant challenge to achieving enduring therapeutic triumphs. The acquisition of drug resistance by malignant cells is the result of a complex interplay between inherent and acquired mechanisms. This phenomenon leads to a reduction in pharmacological effectiveness and contributes to treatment failure. This review investigates the primary mechanisms contributing to chemoresistance in specific malignancies, namely lung, pancreatic, and thyroid cancers, along with non-Hodgkin lymphoma. The primary focus of this research is on genetic modifications, interactions within the tumor microenvironment, and cellular adaptations at the molecular level. The present study places particular emphasis on the contributions of drug efflux pumps, DNA repair pathways, epithelial-mesenchymal transition (EMT), microRNAs, and the suppression of apoptosis in mediating this resistance. In addition, we examine promising approaches to counteract drug resistance, encompassing multi-agent regimens, inhibitors targeting specific molecular pathways, and innovative therapeutic modalities. A comprehensive understanding of these fundamental mechanisms is imperative for the development of sophisticated therapeutic interventions that not only improve patient outcomes but also effectively address resistance. The resistance pathways and prospective translational approaches discussed are drawn from both preclinical and clinical investigations, providing a thorough perspective. Keywords: The following terms are relevant to the study: chemoresistance, tumor microenvironment, molecular pathways, targeted therapy, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec567f4501baf205432eef42dce8eb9f17d9a8c" target='_blank'>
              Mechanisms of Chemoresistance in Solid and Hematologic Malignancies: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            Badiaa Batlamous, Boutaina Elgharbaoui, Imane Bensalim, Mohamed Khalis
          </td>
          <td>2025-09-24</td>
          <td>Middle Eastern Cancer and Oncology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The recurrence of clinically advanced cancers is an evolutionary consequence of standard-of-care chemotherapies generally administered at maximum tolerated doses to kill as many cancer cells as possible. The inevitable appearance of resistance raises the possibility of shifting treatment goals from complete tumor eradication to long-term disease control. The latter approach is employed by adaptive therapy, which aims to inhibit the evolutionary dynamics governing the spread of resistant tumor phenotypes. Adaptive therapy changes focus from the cancer cells that are responsive to therapy to those that are resistant and ultimately govern outcome. This therapeutic approach retains a pool of sensitive cancer cells to compete with the therapy-resistant ones through dynamic dose modulation and/or timing. Thus, fluctuations of treatment-sensitive cells are used to control the resistant population and prolong tumor control with existing therapy agents. Here, we explore non-genetic mechanisms of resistance, including the protective role of the tumor stroma, the epithelial-to-mesenchymal transition, the overexpression of drug efflux pumps, and the extracellular vesicle-mediated transfer of them. These mechanisms can increase the size of the resistant population at the expense of the sensitive one, reducing the ability of adaptive therapy to force tumor evolution into controllable cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b73fae020e6752c501ebd1de56b61a2010abe7" target='_blank'>
              Cancer metabolism: bridging tumorigenesis mechanisms to treatment susceptibility.
              </a>
            </td>
          <td>
            Jian Liu, Dan Shan, Zhaokai Zhou, Xutao Wen, Rao Fu, Bo Xu, Peng Luo, Zhengrui Li, Ling Zhang
          </td>
          <td>2025-09-28</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ∼19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorec tal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, , Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, H. Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, K. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, 
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa96a0707b7c6c78fe5bd4b2a45727093019bee" target='_blank'>
              Targeting the crosstalk of metabolism reprogramming and replication stress: novel strategy to combat cancer.
              </a>
            </td>
          <td>
            Wenbo Liu, Xiangyan Jiang, Yong Ma, Huiguo Qing, Yingcun Bao, Zuoyi Jiao
          </td>
          <td>2025-09-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="This study explores the molecular impact of cisplatin, Actemra (tocilizumab), remdesivir, and SARS-CoV-2 infection on colorectal cancer (CRC) SW-480 cells using RNA sequencing (RNA-Seq). Differential gene expression analysis revealed treatment-specific transcriptional changes. Cisplatin led to the downregulation of genes involved in lipid metabolism and focal adhesion, while remdesivir upregulated chromatin remodeling pathways. SARS-CoV-2 infection altered cytokine signaling, particularly IL-17 and TNF-α. Principal component and hierarchical clustering analyses confirmed distinct gene expression profiles across treatments. ELISA assays validated the reduction of ACE2 and CD147 protein levels following drug treatment. MTT assays demonstrated remdesivir’s cytotoxicity at high doses. These findings highlight potential drug-gene interactions and suggest that combining transcriptomic profiling with functional assays may guide personalized therapeutic strategies for CRC patients, especially those co-infected with SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eae066da47093f217085c9697d40cc942624f2a" target='_blank'>
              RNA sequencing reveals transcriptional signatures of drug response and SARS-CoV-2 interaction in colorectal cancer cells
              </a>
            </td>
          <td>
            Alaa K. Hameed, Ifad K. Abd Al-Shibly, Rana A. Ghaleb
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hematologic malignancies, including leukemia, lymphoma, and multiple myeloma, represent a diverse group of blood cancers driven by complex oncogenic pathways and genetic aberrations. Despite advances in treatment, resistance and relapse remain significant clinical challenges. This review presents an integrative approach to blood cancer therapy, highlighting the convergence of precision medicine, CRISPR-based gene editing, immunotherapy (including CAR-T cells and immune checkpoint inhibitors), and targeted modulation of amino acid metabolism. We investigate the molecular pathways driving tumour development and resistance, highlighting how multi-omic profiling facilitates personalised therapy approaches. Recent breakthroughs in reprogramming immune responses and editing driver mutations have revolutionized therapeutic paradigms. Additionally, the metabolic dependencies of cancer cells, particularly in amino acid biosynthesis and catabolism, offer novel vulnerabilities. This meticulous synthesis outlines a framework for translational techniques that integrate molecular targeting with immunological and metabolic regulation to improve treatment effectiveness and surmount therapeutic resistance in haematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602acd2f85cdfeb967bf671ca25b0e041a71b90f" target='_blank'>
              Deciphering and targeting oncogenic pathways through integrated approaches and amino acid metabolism in hematologic malignancies
              </a>
            </td>
          <td>
            Farhan Ikhtiar, Adil Jamal, Syed M Safeer Mehdi Bokhari
          </td>
          <td>2025-09-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Introduction Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations. Methods To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation. Results Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies. Discussion This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chemotherapy is the most widely used and effective treatment method in today's cancer treatment. However, the rise of cancer drug resistance has limited chemotherapy effectiveness. The molecular mechanisms and related genes by which many cancer cells acquire resistance to chemotherapy drugs remain to be identified. Recent advancements of CRISPR-Cas9 genome-editing techniques have drawn rigorous exploration of precision oncology applications. CRISPR-Cas9 technology can knock out, inhibit or activate the expression of specific genes. By observing the phenotype of gene-edited cells, drug-resistant genes in carcinoma cells can be identified. This paper will decipher the molecular basis of CRISPR-Cas9 technology in targeted gene modification and give examples to illustrate its application in the screening of drug resistance genes in three types of human cancers. Additionally, the application limitations and emerging horizons of CRISPR-Cas9 gene modification technique in strategies for overcoming anticancer drug resistance are discussed. The continuous development and optimization of this technology are expected to provide strong support for breaking through the bottlenecks in tumor treatment and pushing precision cancer medicine to new heights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0492def30740e765a8ca58409631a4cc538a87f" target='_blank'>
              CRISPR-Cas9 Gene-Editing Technology for the Screening of Drug Resistance Genes in Tumor Cells
              </a>
            </td>
          <td>
            Zisheng Zeng
          </td>
          <td>2025-09-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance limits the therapeutic efficacy of osimertinib in lung adenocarcinoma (LUAD). Redox homeostasis is crucial for LUAD progression. However, how redox imbalance interacts with the tumor microenvironment (TME) to drive osimertinib resistance (OR) remains unclear.


METHODS
The single-cell RNA sequencing (scRNA-seq) data GSE243562 were combined with the Cancer Genome Atlas (TCGA)-LUAD transcriptomes to map the TME cell population heterogeneity in osimertinib-resistant LUAD. Through univariate Cox regression and least absolute shrinkage and selection operator (LASSO) regularization, a prognostic signature founded on redox-related genes (RRGs) was built. Therapeutic compounds targeting these signature genes were prioritized by molecular docking. Their expression patterns were subsequently validated in vitro.


RESULTS
Cancer-associated fibroblasts (CAFs) were central hubs in the TME of osimertinib-resistant LUAD, exhibiting enhanced intercellular communication. Computational profiling identified 10 differentially expressed RRGs, predominantly enriched in CAFs. Using a six-gene signature comprising AGER, CYP2J2, FMO2, HSPA1B, SOD3, and VASN, we categorized LUAD patients into separate risk categories. High-risk patients showed significantly reduced survival, an immunosuppressive status, and a higher tumor mutation burden (p < 0.05). The overexpression of these six genes was confirmed in OR cells. Critically, inhibiting SOD3 restored osimertinib sensitivity in vitro (p < 0.05). Clinically, SOD3 expression was lower in patients sensitive to third-generation epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs) than in those with resistant disease.


CONCLUSIONS
Targeting CAFs represents a promising strategy to overcome osimertinib resistance. Our six-gene redox signature offers a clinical framework for patient risk stratification and novel therapeutic strategy design. Future work will explore these targets to develop new treatments for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1eb1b2e535167099275ea725fe5cce81f5d8d82" target='_blank'>
              Integrated Analysis of Single-cell and Bulk-RNA Sequencing Data to Identify Redox-related Genes as Prognostic Biomarkers and Therapeutic Targets of Lung Adenocarcinoma With Osimertinib Resistance.
              </a>
            </td>
          <td>
            Chen Fang, Chaoxing Liu, Rong Qi, Jiabin Ding, Ting Luo, Feng Yu, Guohua Zhang, Chao Shi, Daya Luo, Feng Qiu
          </td>
          <td>2025-09-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma remains one of the most treatment-refractory solid tumors due to rapid adaptive resistance across multiple convergent molecular pathways. In particular, persistent activation of the PI3K-AKT axis combined with mesenchymal transition (EMT) reprogramming contributes to rapid failure of standard therapeutic regimens, including temozolomide, radiation, and immunotherapeutics. To address this clinical gap, we developed a synthetic RNA-guided dual-targeting system designed to simultaneously suppress both canonical PI3K-AKT activation and mesenchymal escape signatures, using strand-engineered molecules optimized for combinatorial silencing. Using patient-derived glioblastoma stem cell lines and isogenic resistance models, we performed transcriptomic profiling to identify co-expressed resistance-driving nodes. Computational integration of CRISPR perturbation datasets, RNA interference screens, and synthetic lethality maps enabled selection of targetable PI3K-AKT intermediates and EMT regulators including SOX2, TWIST1, and ZEB1. Synthetic RNA molecules were designed to co-suppress both vertical (PI3K-AKT) and horizontal (EMT plasticity) resistance axes, with in vitro validation demonstrating >70% knockdown efficiency and significant synergistic reduction in proliferation (p<0.001) across diverse resistance phenotypes. Mechanistic assays revealed reversal of mesenchymal signatures with restored epithelial markers (E-cadherin upregulation; vimentin downregulation) alongside AKT dephosphorylation. Combination treatment with low-dose temozolomide further amplified cell death, suggesting sensitization to standard chemotherapy. In vivo murine xenograft models exhibited >60% reduction in tumor volume compared to controls (p<0.01), without evidence of off-target cytotoxicity or systemic toxicity. This study demonstrates the feasibility of multi-axis synthetic RNA-guided combinatorial targeting to suppress glioblastoma therapeutic resistance. These findings support future clinical translation of modular RNA-based therapeutics capable of overcoming the heterogeneity and plasticity that characterize high-grade gliomas.



 Shivi Kumar. Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746db514f096cab5d3002088013773d14b8c8b84" target='_blank'>
              Abstract A036: Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways
              </a>
            </td>
          <td>
            Shivi Kumar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ovarian cancer (OC) is lethal due to near universal development of resistance to platinum-based chemotherapy. Metabolic adaptations can play a pivotal role in therapy resistance. Here, we aimed to identify key metabolic pathways that regulate platinum response and represent potential therapeutic targets. Transcriptomic and metabolomic analyses in cisplatin sensitive and resistant ovarian cancer cells identified enrichment of pyrimidine metabolism related to upregulated de novo pyrimidine synthesis. 15N-glutamine flux analysis confirmed increased de novo pyrimidine synthesis in cisplatin resistant cells. Targeting this pathway using brequinar (BRQ), an inhibitor of the key enzyme dihydroorotate dehydrogenase (DHODH), decreased cell viability, delayed G2/M cell cycle progression, and altered expression of genes related to mitochondrial electron transport in resistant cells. Under basal conditions, cisplatin resistant cells had a lower oxygen consumption rate (OCR) and spare respiratory capacity (SRC) than sensitive cells. BRQ suppressed OCR in both sensitive and resistant but only inhibited SRC in resistant cells. In cell line-derived and patient-derived xenograft models, BRQ attenuated the growth of cisplatin resistant ovarian tumors and enhanced the inhibitory effects of carboplatin. Together, these results identify metabolic reprogramming in cisplatin resistant ovarian cancer that induces an acquired dependency on de novo pyrimidine synthesis, which can be targeted to sensitize tumors to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d708b6c5cad44de1ded3e0fd0aa39875db711a4c" target='_blank'>
              Metabolic Dependency on De Novo Pyrimidine Synthesis is a Targetable Vulnerability in Platinum Resistant Ovarian Cancer.
              </a>
            </td>
          <td>
            H. Cardenas, Yinu Wang, Guangyuan Zhao, Delan Xingyue Hao, A. Isac, Vanessa Hernandez, Ujin Kim, Wenan Qiang, Hao F. Zhang, Daniela Matei
          </td>
          <td>2025-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3565fa0fad40d6b64564b524def98cd2cdcfa949" target='_blank'>
              Metabolic plasticity drives mechanisms of PARP inhibitor persistence in ovarian cancer
              </a>
            </td>
          <td>
            Adriana Del Pino Herrera, Joon-Hyun Song, Karla Torres-Arciga, Susoma Halder, Kadin El Bakkouri, Mehdi Damaghi, Meghan C. Ferrall-Fairbanks
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a200a976abe18d2ada20b9002895dd27ac2f5" target='_blank'>
              Prognostic and Therapeutic Implications of BRAF Mutations in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Darren Lee, Yazan Abu-Shihab, Kate Plas, D. Nicolet, K. Mrózek, M. Routbort, , C. Walker, Jill Buss, Andrew Stiff, Andrea Laganson, C. Dinardo, N. Daver, T. Kadia, F. Ravandi, Andrew J Carroll, J. Kolitz, B. Powell, William Blum, Maria R. Baer, G. Marcucci, Geoffrey L. Uy, Wendy Stock, Richard M. Stone, L. J. Medeiros, John C Byrd, J. Blachly, Robert L. Bowman, J. Tyner, S. Loghavi, Ann-Kathrin Eisfeld, Linde A. Miles
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Antimycin A, an antifungal agent that inhibits mitochondrial respiration, provides a useful model for studying resistance mechanisms. Antifungal resistance is an escalating clinical concern with limited treatment options available. To understand the molecular mechanisms of antimycin A-resistance, a genetically stable, antimycin A-resistant Saccharomyces cerevisiae strain was successfully developed for the first time through an evolutionary engineering strategy, based on long-term systematic application of gradually increasing antimycin A-stress in repetitive batch cultures without prior chemical mutagenesis. Comparative whole genome re-sequencing analysis of the evolved strain ant905-9 revealed two missense mutations in PDR1 and PRP8 genes involved in pleiotropic drug resistance and RNA splicing, respectively. Using CRISPR/Cas9 genome editing tools, the identified mutations were introduced individually and together into the reference strain, and it was confirmed that the Pdr1p.M732R mutation alone confers antimycin A-resistance in S. cerevisiae. Comparative transcriptomic analysis of the reverse engineered Pdr1p.M732R strain showed alterations in PDR (pleiotropic drug response), transmembrane transport, vesicular trafficking and autophagy pathways. Our results highlight the potential key role of PDR1 in antifungal drug resistance. This study provides new insights into mitochondrial drug resistance and the adaptive potential of yeast under respiratory stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b896127a479ec689ce1d1e9d511ec8bcaa5efae" target='_blank'>
              Evolutionary Engineering and Molecular Characterization of an Antimycin A-Resistant Saccharomyces cerevisiae Strain: the Key Role of Pleiotropic Drug Resistance (PDR1).
              </a>
            </td>
          <td>
            A. Topaloğlu, C. Holyavkin, Ö. Esen, O. Morkoç, K. Persson, C. Geijer, Z. P. Çakar
          </td>
          <td>2025-10-17</td>
          <td>FEMS yeast research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Disclosure: S. Gadad: None. M. Sedano: None. Triple-negative breast cancer (TNBC) accounts for approximately 12-20% of all new breast cancer cases diagnosed annually and is known to be associated with poorer clinical outcomes compared to other molecular subtypes of breast cancer. Patients with TNBC typically present with larger tumors and higher tumor grades, resulting in decreased overall and recurrence-free survival rates compared to individuals with non-TNBC tumors. The strategies involved in treating TNBC include chemotherapy agents such as doxorubicin, an anthracycline chemotherapeutic agent; however, its efficacy may vary due to the development of resistance. The specific mechanisms behind the constantly evolving resistance of TNBC to chemotherapy remain unclear, emphasizing the need for further investigation. In our study, we utilized two TNBC cell lines engineered with gradually increasing concentrations of doxorubicin to develop resistance. We conducted integrative genomic analyses to uncover the cellular mechanisms involved, identified altered molecular pathways relevant to cancer, and ascertained the factors contributing to the phenotypic characteristics of doxorubicin-resistant TNBC. Our findings indicate that many evolutionarily young transcripts with coding or limited coding potential are differentially expressed compared to non-resistant TNBC cells. Furthermore, analyses of adjacent genomic regions containing aberrantly expressed genes revealed that these genes are located in areas implicated in cellular plasticity, allowing cancer cells to adapt to the presence of doxorubicin, potentially through transcriptional reprogramming. We are also investigating the genetic and epigenetic alterations that may have facilitated the evolution of TNBC cells into doxorubicin-resistant variants. Additionally, we are assessing the clinical implications of the identified factors by studying them in vivo using mouse models with genetically engineered chemo-sensitive and -resistant cells, linking the outcomes to changes observed in patient samples associated with clinical prognosis. By elucidating the specific mechanisms contributing to the development of doxorubicin resistance in TNBC, we aim to uncover the underlying processes driving resistance and recurrence, potentially paving the way for novel therapeutic strategies. S.S.G. is a CPRIT Scholar in Cancer Research. S.S.G. was supported by a First-time faculty recruitment award from the Cancer Prevention and Research Institute of Texas (CPRIT; RR170020). S.S.G. is supported by the The American Cancer Society (RSG-22-170-01-RMC), NIH 1R16GM149497 grants and CPRIT-TREC (RP230420). Presentation: Sunday, July 13, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b4373af8394b05d1e7e23058cecf5a8e12738f" target='_blank'>
              SUN-210 The role of transcriptional reprogramming and genomic alterations in the transition from chemo-sensitive to chemoresistant triple-negative breast cancer
              </a>
            </td>
          <td>
            Shrikanth S. Gadad, Melina J. Sedano
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane E. Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L. Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>82</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In Mycobacterium tuberculosis, a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            Anna G Green, Mahbuba Tasmin, R. Vargas, M. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The mismatch repair system is critical for correcting base–base mismatches and insertion-deletion loops during DNA replication. Deficiencies in MMR (due to mutations in MLH1, MSH2, MSH6, or PMS2) lead to microsatellite instability and contribute to the development of various cancers, such as Lynch syndrome-related colorectal cancer and sporadic tumors. This review will delve into the molecular basis of MMR deficiency. Additionally, the review will cover diagnostic approaches for detecting MSI and MMR deficiency, including next-generation sequencing and PCR-based methods. The implications for treatment will be discussed, emphasizing immune checkpoint inhibitors (e.g., pembrolizumab) that target tumors with high mutational burdens due to MMR deficiency, as well as novel therapeutic approaches like synthetic lethality exploiting DNA repair vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c3fe776b0060c3cc36839d638f2791e2a3b03a" target='_blank'>
              Mismatch Repair Deficiency and the Role of Non-Canonical Functions in Cancer: Diagnosis and Therapeutic Implications
              </a>
            </td>
          <td>
            Alicja Dąbrowska, Jakub Mastalerz, Zofia Łapińska, I. Deszcz, Agnieszka Chwiłkowska, Nina Rembiałkowska
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Hypoxia is a well-recognized clinical feature of solid tumors, including lung cancer, and is associated with poor prognosis due to its role in promoting resistance to chemotherapy and radiotherapy. To investigate the cellular consequences of hypoxia, we cultured A549 lung adenocarcinoma cells under 1% O2 and examined their growth, cell cycle distribution, and redox status. Hypoxia significantly reduces cell proliferation and induced G1 phase cell cycle arrest, suggesting a cytostatic effect. Biochemical analysis showed a 2.64- and 2.04-fold increase in total and mitochondrial reactive oxygen species (ROS) levels, respectively, along with an elevated total thiol levels under hypoxic conditions compared to normoxia. To assess the reversibility of the hypoxic response, we performed a reciprocal oxygen exposure experiment where cells initially grown under hypoxia were re-exposed to normoxia, and vice versa. To explore the underlying molecular mechanism, we analyzed transcriptomic datasets (GEO accession: GSE48134 and GSE42416) which revealed that hypoxia downregulated key genes involved in energy metabolism (e.g., PDK4, G6PD), cell cycle progression (e.g., CCND1, CDK2), and redox regulation (e.g., GCLM, TXNRD1, NQO1, GCLC). Further, few of redox-related genes were validated by RT-PCR in A549 cells cultured under hypoxia and normoxia for 24 h. Importantly, cyclic hypoxia (intermittent hypoxia-reoxygenation) conditions showed partial restoration of some of these transcripts, supporting the transient nature of hypoxic stress, consistent with our in vitro observations. Furthermore, transcriptome profiles from adenocarcinoma patients (GEO accession: GSE30979) also match our cell line observations. Thus, our results clearly show that hypoxia causes a temporary cell cycle arrest in lung cancer cells, which is reversible when oxygen is restored. Supplementary Information The online version contains supplementary material available at 10.1007/s12032-025-03058-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77bbcdb23f2ddac965c8e06176945c4fad76ded" target='_blank'>
              Hypoxia-drives reversible cell cycle arrest in lung cancer cells via modulation of cellular redox and gene expression
              </a>
            </td>
          <td>
            D. Maurya, Varshita Mehta, Babita Singh
          </td>
          <td>2025-09-29</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840b437f838fbeb4893166e383606b0c1b14d251" target='_blank'>
              Integrated Metabolic Complex Genetic Interaction Network of Chromosome 4p Loss in Basal Breast Cancer
              </a>
            </td>
          <td>
            Lynn Karam, Alain Pacis, Rohan Dandage, Michael Schwartz, Gabriel Alzial, Anastasia Gherghi, H. Djambazian, Gerardo Zapata, Marina A Schapfl, Paria Asadi, Mayerly Castrillón, Alysh Orr, Belen Hernandez, Pegah Kargar, Michelle Vandeloo, Sylvia Santosa, A. Villunger, Traver Hart, Jiannis Ragoussis, Geneviève Deblois, Andreas Bergdahl, Morag Park, Guillaume Bourque, Elena Kuzmin
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Microsatellite-stable metastatic colorectal cancer (MSS mCRC) is currently treated with chemotherapy and targeted agents based on RAS and BRAF mutational status. Although these therapies offer initial benefit, most patients rapidly develop resistance, with fewer than 20% remaining progression-free at two years. This review aims to synthesize emerging evidence on the metabolic mechanisms driving treatment resistance in MSS mCRC, with a particular focus on the immune-metabolic signature (IMMETCOLS) classification. We conducted a comprehensive review of preclinical models, transcriptomic datasets, and clinical trial results addressing metabolic adaptations to chemotherapy and targeted therapies in MSS mCRC. The IMMETCOLS framework defines three metabolic subtypes—IMC1, IMC2, and IMC3—each associated with distinct resistance mechanisms. IMC1 exhibits glycolysis and transforming growth factor-β (TGF-β)-dependent signaling enriched in inflammatory fibroblasts, conferring resistance to chemotherapy. IMC2 relies on oxidative phosphorylation and glutamine metabolism, supporting antioxidant defenses and resistance to both cytotoxic agents and anti-EGFR therapies. IMC3 demonstrates lactate-fueled respiration and pentose phosphate pathway activation, contributing to redox balance, DNA repair, and resistance to targeted therapies such as anti-BRAF or KRAS inhibitors. All subtypes display metabolic plasticity under therapeutic pressure. Emerging clinical data support tailoring targeted therapy combinations based on IMMETCOLS subtype, particularly in BRAF- and HER2-positive populations. Understanding subtype-specific metabolic rewiring in MSS mCRC offers novel opportunities to overcome drug resistance. Targeting the metabolic vulnerabilities defined by the IMMETCOLS signature may improve response durability and inform precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662ed873ba87c81960ffe96e54fa075a4697d04f" target='_blank'>
              Metabolic plasticity drives specific mechanisms of chemotherapy and targeted therapy resistance in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Mariam Rojas, Malena Manzi, S. Madurga, Fernando Enrique García Velásquez, Maira Alejandra Romero, Silvia Marín, Marta Cascante, Joan Maurel
          </td>
          <td>2025-09-23</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Aim: Urinary bladder cancer (UBC) often develops chemoresistance, reducing treatment effectiveness. This study aimed to investigate diverse molecular mechanisms underlying acquired resistance by establishing and characterizing a comprehensive panel of UBC cell lines resistant to common chemotherapeutics. Methods: Fifteen UBC cell lines were examined: three parental lines (RT-112, TCC-SUP, UMUC-3) and twelve derived sublines adapted to cisplatin, vinblastine, or gemcitabine. Drug sensitivity was assessed using the SRB assay. Resistance mechanisms were explored via quantitative real-time PCR (targeting genes including ABCB1, dCK, hENT1, ECHDC1, TUBB3), Western blotting (assessing proteins such as p21, Cyclin B, and Mcl-1), and biochemical assessment of glutathione levels and redox state. Results: The adapted sublines exhibited distinct resistance profiles and cross-resistance patterns. Gene expression and protein analyses revealed drug- and lineage-specific alterations, involving factors such as p21, Cyclin B, and Mcl-1. Changes in glutathione metabolism were also associated with resistance. Notably, no single, universal mechanism accounted for resistance across the entire panel. Conclusion: UBC cells develop diverse, context-dependent adaptive strategies to resist cisplatin, vinblastine, and gemcitabine. These findings highlight the complexity of chemoresistance mechanisms. The characterized cell line panel represents a valuable resource for future studies aimed at understanding and overcoming drug resistance in bladder cancer, suggesting that personalized therapeutic approaches may be necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fbd01421411fd7eb4380d44e0e070090cf7648" target='_blank'>
              Distinct adaptive strategies to cisplatin, vinblastine and gemcitabine in a panel of chemoresistant bladder cancer cell lines
              </a>
            </td>
          <td>
            Monika Cuprych-Belter, Agnieszka Łupicka-Słowik, Artur Anisiewicz, Martin Michaelis, J. Cinatl jr., Mateusz Psurski
          </td>
          <td>2025-09-12</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 CDK4/6 inhibitors exhibit therapeutic potential in several tumor entities. Their clinical efficacy in glioblastoma might be limited by intrinsic and acquired resistance mechanisms. Understanding underlying genetic and molecular factors of this resistance might inform the design of effective therapeutic strategies. Genome-wide CRISPR-Cas9 screens offer a powerful approach to systematically identify resistance drivers and synthetic lethal interactions, paving the way for rational combination therapies.



 We conducted genome-wide CRISPR-Cas9 knockout and activation screens in human glioma cell lines LN229 and GS9 under CDK4/6 inhibition using Brunello and Calabrese libraries. Top hits were validated using functional assays. Furthermore, we conducted bulk RNA sequencing of treated and untreated cells for transcriptomic insights into drug response modifiers and molecular pathways involved.



 Functional genomics identified Ambra1 and CCNE1 as key drivers of resistance. Both Ambra1 knockout and CCNE1 overexpression, conferred resistance across both long-term and stem-like glioma models. Additionally, knockout of CHEK1 and FAM122A uncovered synthetic lethal interactions with CDK4/6 inhibition. Functional validation confirmed that glioma cells rely on CHEK1 or FAM122A for cell cycle progression, with cell line specific sensitivity. Transcriptomic analyses consistently highlighted alterations in pathways associated with cell cycle regulation, DNA replication, and the Fanconi anemia pathway. Of note, several differentially expressed genes overlapped with screen hits, supporting the robustness of our findings. Notably, pharmacological inhibition of CHEK1, both in vitro and ex vivo, enhanced sensitivity to CDK4/6 inhibition in a sequence-dependent manner, indicating a critical role of the sequence of drug administration.



 Our functional genomics approach uncovers mediators of acquired resistance and synthetic lethality in the context to CDK4/6 inhibitors in glioma. Co-targeting of CHEK1 or FAM122A with CDK4/6 inhibitors represents a promising therapeutic strategy to overcome resistance and improve the therapeutic efficacy. Ongoing in vivo studies using xenograft and syngeneic glioma models aim to validate the survival benefit of these combinations and to elucidate their impact on the tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaeb6b136b57493744dd9b571df497dca29c55b" target='_blank'>
              OS05.8.A DISSECTING RESISTANCE TO CDK4/6 INHIBITION IN EXPERIMENTAL GLIOMA VIA GENOME-WIDE CRISPR-CAS9 SCREENING
              </a>
            </td>
          <td>
            S. Surender, L. Haeusser, L. Kuhlburger, F. Tsiami, L. Maise, S. Nahnsen, S. Beck, D. Merk, G. Tabatabai
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-16</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive malignancy of the bile ducts, and resistance to gemcitabine, a first-line chemotherapy, significantly complicates treatment. Despite extensive research, the molecular mechanisms underlying gemcitabine resistance in iCCA are not fully understood. This study aims to identify key genes associated with gemcitabine resistance in iCCA, investigate the role of lactylation, and propose potential therapeutic targets. Methods A comprehensive bioinformatics analysis was conducted using publicly available transcriptomic data from gemcitabine-resistant iCCA cell lines and patient samples. Differential expression analysis was performed to identify upregulated and downregulated genes. GSEA were used to explore relevant molecular pathways. Immune landscape analysis was carried out using CIBERSORT to assess immune cell infiltration in the tumor microenvironment. Key resistance-related genes were identified through Lasso, RF, and SVM-REF analyses. ITGB4 function was further validated in vitro by siRNA knockdown in HUCCT1 and RBE cells, followed by cell viability and apoptosis assays with or without gemcitabine treatment. Results Pathway analysis revealed the involvement of cell cycle regulation, DNA replication, and p53 signaling in gemcitabine resistance. The high group associated with resistance showed significantly worse survival outcomes, with a positive correlation between resistance and lactylation levels. Immune landscape analysis indicated altered immune cell infiltration, including increased M2 macrophages and decreased CD8+ T cells in the high group. Key resistance-related genes, including CDC20, H2AX, HK2, and ITGB4, were identified as critical in drug resistance. Experimentally, ITGB4 knockdown markedly enhanced gemcitabine’s antiproliferative and pro-apoptotic effects on cholangiocarcinoma cells, supporting its role in mediating resistance. Molecular docking revealed Dioscin and Deacetyllanatoside C as potential ITGB4-interacting compounds. Conclusion This study sheds light on the molecular mechanisms of gemcitabine resistance in iCCA, emphasizing lactylation’s role and the significance of immune modulation. ITGB4 is identified as a promising therapeutic target, and the findings suggest that targeting these genes could help overcome resistance in iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea10bda53b7cbe7c506a45067aed62ce5a1a0fe" target='_blank'>
              Integrated transcriptomic analysis identifies lactylation-linked gemcitabine resistance and therapeutic targets in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Wenwei Xie, Jialiang Hu, Hanmei Xu
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genetic mechanisms regulating gene expression encompass complex processes such as transcription, translation, epigenetic modifications, and interactions of regulatory elements. These mechanisms play a crucial role in shaping phenotypic diversity in humans. High-throughput technologies, such as expression microarrays and next-generation sequencing (NGS), have enabled precise analysis of transcripts for thousands of genes genome-wide. These methods have enabled researchers to measure gene expression levels in various tissues and cells and to gain deeper insights into previously inaccessible biological processes. Numerous studies show that gene expression varies significantly among individuals. However, there are also notable differences between populations from different continental groups, driven by genetic, epigenetic, environmental factors, and natural selection. Furthermore, disease states represent an important factor influencing gene activity, as they can significantly alter the transcriptomic profiles of individual cells. In this context, comparative population genetic studies help uncover the molecular mechanisms underlying complex phenotypic traits and identify population-specific features of transcriptomic profiles in both health and disease. However, despite significant progress in this field, many aspects remain underexplored. Specifically, the distribution of gene expression variability among populations, the degree of research coverage for specific ethnic groups, the spectrum of biological materials used, and the contribution of population affiliation to observed differences in gene expression during pathological conditions require further investigation. This review presents an overview of contemporary research focused on analyzing variability in expression profiles across different human populations. It summarizes findings from individual studies, outlines the advantages and limitations of the methods employed, highlights key research directions in population transcriptomics, and discusses potential practical applications of the data obtained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e020705578f481ead7a3caa26fc484de5385fb09" target='_blank'>
              Population transcriptomics: a novel tool for studying genetic diversity in human populations under normal and pathological conditions
              </a>
            </td>
          <td>
            A. A. Babovskaya, E. A. Trifonova, V. A. Stepanov
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. Methods We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. Results ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Conclusion Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07216-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, 
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Psoriasis is a chronic inflammatory skin disease involving dysregulated immune responses and complex genetic factors. This study combines single-cell RNA sequencing (scRNA-seq), gene expression profiling, and genetic analysis to explore cellular and molecular contributors to psoriasis. Methods Single-cell RNA-seq data (n = 3 psoriasis, n = 2 control; GSE228421) were used for cell-type annotation and functional characterization. T cell subsets were analyzed for differentiation trajectories and cell-cell communication. Differentially expressed genes in mucosal-associated invariant T (MAIT) cells were evaluated by enrichment analysis. Candidate gene causality was tested via eQTL-based Mendelian randomization (MR) and supported by bulk RNA-seq validation. Results MAIT cells were enriched in psoriatic lesions and exhibited strong intercellular interactions. Functional analyses revealed activation of IL6-JAK-STAT3 signaling, TNF-NFκB pathway, and glycolysis in MAIT cells. MR identified RPS20 as a protective factor (OR = 0.5994, p = 0.011) and PFN1 as a potential risk gene (OR = 1.7229, p = 0.037), with PFN1 highly expressed in MAIT cells. Colocalization analysis showed no significant genetic overlap between PFN1 expression and psoriasis risk. Metabolic profiling revealed differential pathway involvement in PFN1+ and PFN1− MAIT cells. Conclusion Our integrative analysis highlights MAIT cells and PFN1 as likely contributors to psoriasis pathogenesis. These findings offer insights into immune and metabolic alterations, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48055881b868991adfc41d78b968062c6c8aae6e" target='_blank'>
              MAIT Cells with High PFN1 Expression Mediate Immune Activation and Metabolic Reprogramming in Psoriasis
              </a>
            </td>
          <td>
            Man-Ning Wu, Yue-Min Zou, Xiang-Nan Zhou, Sangwon Hong, Lei Wang, Yan-Ping Bai
          </td>
          <td>2025-09-09</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematologic cancer marked by clonal expansion of plasma cells in the bone marrow. Although its genomic landscape has been extensively characterized, the transcriptional mechanisms that govern malignant progression and long-term tumor cell survival remain incompletely understood. Methods We integrated single-cell RNA sequencing (scRNA-seq) data from healthy donors (HD), monoclonal gammopathy of undetermined significance (MGUS), smoldering multiple myeloma (SMM), and newly diagnosed multiple myeloma (NDMM) patients, retrieved from GEO datasets GSE124310 and GSE271107, to construct a comprehensive transcriptional landscape of plasma cell differentiation. Pseudotime and enrichment analyses identified PDIA4 as a terminal-state–associated gene. The prognostic significance of PDIA4 was validated using the multiple myeloma research foundation (MMRF) CoMMpass cohort. Functional analyses were performed in vitro and in vivo to validate the role of PDIA4 in MM cell survival and therapeutic response. Results Pseudotime trajectory analysis revealed progressive upregulation of genes involved in protein processing in the endoplasmic reticulum (ER), with PDIA4 identified as a top candidate in terminal-stage plasma cells. Survival analysis in the MMRF CoMMpass cohort further demonstrated that high PDIA4 expression correlated with poor overall survival. In RPMI-8226 cells, PDIA4 knockout activated the IRE1α/XBP1s branch of the unfolded protein response (UPR), impaired proliferation, and induced G1-phase arrest. PDIA4 depletion also sensitized cells to bortezomib. In vivo, sg-PDIA4 suppressed tumor growth in RPMI-8226 xenografts. Conclusions PDIA4 is a key regulator of the unfolded protein response and MM cell survival. Targeting PDIA4 may enhance the efficacy of proteasome inhibitors and offers a potential strategy to overcome therapeutic resistance in multiple myeloma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07098-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881fab1a35c430a7450d4d3fe7ea84a60cfe6f7c" target='_blank'>
              Single-cell transcriptomics identifies PDIA4 as a marker of progression and therapeutic vulnerability in multiple myeloma
              </a>
            </td>
          <td>
            Wenjie Yu, Jiamin Zhan, Yanxia Wang, Xu Cao, Aoyang Yu, Yuexuan Rao, Can Huang, Xuan Zhang, Shunv Lu, Yanhua Liu, Chong Chen, Zhenyu Li
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells frequently reprogramme their metabolism to support growth and survival, making metabolic pathways attractive targets for therapy. In this study, we investigated the metabolic effects of three kinase inhibitors and their synergistic combinations in the gastric cancer cell line AGS using genome-scale metabolic models and transcriptomic profiling. We applied the tasks inferred from the differential expression (TIDE) algorithm to infer pathway activity changes in the different conditions. We also explored a variant of TIDE that uses task-essential genes to infer metabolic task changes, providing a complementary perspective to the original algorithm. Our results revealed widespread down-regulation of biosynthetic pathways, particularly in amino acid and nucleotide metabolism. Combinatorial treatments induced condition-specific metabolic alterations, including strong synergistic effects in the PI3Ki–MEKi condition affecting ornithine and polyamine biosynthesis. These metabolic shifts provide insight into drug synergy mechanisms and highlight potential therapeutic vulnerabilities. To support reproducibility, we developed an open-source Python package, MTEApy, implementing both TIDE frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3017a5eb28e4bc7906f6c3abd507b3d2f862ad" target='_blank'>
              Constraint based modeling of drug induced metabolic changes in a cancer cell line
              </a>
            </td>
          <td>
            Xavier Benedicto, Åsmund Flobak, M. Ponce-de-León, Alfonso Valencia
          </td>
          <td>2025-10-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, O. Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, , Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f305a3d3aa4c5343ff379224a4358725aa81e74e" target='_blank'>
              ACSS2-Mediated Metabolic-Epigenetic Crosstalk Drives Fulvestrant Resistance and Represents a Novel Therapeutic Target
              </a>
            </td>
          <td>
            A. Mogol, Jin Young Yoo, A. Eve, Mahima Goel, David J Dutton, Claire P. Schane, Audrey Lam, Debapriya Dutta, Betsy Barnick, Eda D Erdogan, Erik R. Nelson, Maria Grosse-Perdekamp, Zeynep Madak Erdogan
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Telomere maintenance enables unlimited cell proliferation by counteracting telomere erosion. While the majority of tumors activate telomerase, a significant subset—approximately 10–15%—utilizes alternative lengthening of telomeres (ALT), a recombination-based mechanism. ALT-positive cancers are classically associated with genomic instability, anaphase bridges, chromosomal rearrangements, and resistance to DNA-damaging therapies. This process is closely associated with genetic instability, which contributes to chromosomal rearrangements and tumor evolution. Consequently, ALT has traditionally been considered an adverse prognostic marker in aggressive malignancies such as osteosarcoma, pancreatic neuroendocrine tumors, and high-grade sarcomas. Paradoxically, recent evidence demonstrates that ALT positivity correlates with improved survival in glioblastoma (GBM) and chondrosarcoma, two tumor types that have historically been regarded as immune-cold and therapeutically intractable. This favorable outcome likely reflects a convergence of factors, including replication stress and DNA damage that impose a fitness cost in slow-growing or metabolically constrained tumors. Loss of ATRX/DAXX, while enabling ALT, further amplifies chromatin fragility, and ALT-mediated instability may paradoxically enhance immunogenicity within immune-quiescent microenvironments. Moreover, ALT-positive cells exhibit unique therapeutic vulnerabilities, particularly to ATR and PARP inhibitors. Together, these observations support a context-dependent model in which ALT functions as a double-edged sword, acting as a driver of malignant aggressiveness in rapidly proliferating cancers while serving as a relative liability in slower-growing, immune-cold tumors. Understanding this duality not only refines prognostic stratification but also opens opportunities for precision oncology. By integrating ALT-specific biomarkers into clinical workflows and exploiting ALT-related DNA repair dependencies, clinicians may transform a once uniformly negative prognostic factor into an actionable therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6ed85bfa1a7b22ce5afc4a049e72d82de6bf4" target='_blank'>
              Alternative Lengthening of Telomeres: A Prognostic Paradox in Cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung
          </td>
          <td>2025-10-17</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Paclitaxel is a widely used chemotherapeutic agent that plays a vital role in the treatment of various cancers, including breast, ovarian, and lung cancers. Despite its success, the development of resistance significantly limits its long-term efficacy. Paclitaxel resistance involves a range of molecular mechanisms, including alterations in drug transport, mutations in β-tubulin, and activation of pro-survival signaling pathways. Recent studies have highlighted the crucial role of circular RNAs (circRNAs) in mediating paclitaxel resistance. CircRNAs are characterized by their stable, covalently closed structures, which protect them from exonuclease degradation. They regulate drug resistance through various mechanisms, including modulating transcription factors and influencing cellular processes such as apoptosis, immune response, and cancer stem cell dynamics by sponging microRNAs. This review focuses on the mechanisms by which circRNAs contribute to paclitaxel resistance and discusses their dual roles as both oncogenes and tumor suppressors. Furthermore, we explore the potential of circRNAs as novel therapeutic targets to overcome paclitaxel resistance. By targeting specific circRNAs or restoring tumor-suppressive circRNAs, it may be possible to enhance paclitaxel sensitivity and improve treatment outcomes. Future research is essential to further understand the role of circRNAs in paclitaxel resistance and to develop effective circRNA-targeted therapies for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b2a64750b1e25583cb541b2ce40bf7626415eaa" target='_blank'>
              The dual role of circular RNAs in paclitaxel resistance: insights into molecular mechanisms and clinical implications.
              </a>
            </td>
          <td>
            Ehsan Maghrebi-Ghojogh, O. Q. B. Allela, Karar H. Alfarttoosi, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq, Mitra Zareanshahraki, Hossein Gandomkar
          </td>
          <td>2025-10-17</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells.
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous and aggressive hematologic malignancy, with the activated B-cell-like (ABC) subtype displaying particularly poor prognosis due to inherent treatment resistance and elevated recurrence rates. Despite advances in targeted therapies and immunotherapies, a significant proportion of patients experience relapse or refractory disease, highlighting the urgent need for novel biomarkers and innovative therapeutic strategies to improve clinical outcomes. Methods A multi-dimensional analysis of SH3BP5 expression was performed across DLBCL subtypes, integrating transcriptomic, proteomic, and clinical datasets to assess its correlation with immune infiltration, tumor metabolism, and patient prognosis. Single-cell RNA sequencing data were employed to examine the tumor microenvironment (TME) with higher resolution. Further analysis of the association between SH3BP5 and immune checkpoint gene expression was conducted to explore its potential role in immunotherapy response. Functional in vitro assays were carried out to assess the impact of SH3BP5 knockdown on DLBCL cell proliferation and apoptosis. Results The analysis revealed that SH3BP5 is preferentially overexpressed in the ABC subtype of DLBCL across multiple datasets and validated cohorts, and its high expression is significantly associated with poor overall survival. Single-cell transcriptomic profiling demonstrated that SH3BP5 is mainly expressed in malignant B cells and inversely correlated with immune cell infiltration, particularly CD8 + T cells. Mechanistically, pathway enrichment and metabolic assays indicated that SH3BP5 is linked to mitochondrial metabolic reprogramming, promoting oxidative phosphorylation (OXPHOS) and potentially contributing to reduced responsiveness to immune checkpoint inhibitors (ICIs). Functional studies showed that SH3BP5 knockdown significantly suppressed DLBCL cell proliferation, induced apoptosis, and reduced tumor cell viability in vitro. Conclusion This study suggests that SH3BP5 may serve as a prognostic biomarker and a potential therapeutic target in DLBCL, particularly within the ABC subtype. By delineating its associations with immune evasion and metabolic reprogramming, these findings provide a mechanistic basis for further exploration of SH3BP5-targeted interventions to help overcome therapy resistance. Future studies in larger clinical cohorts and functional models are warranted to validate these results and assess the potential of integrating SH3BP5 expression profiling into precision medicine strategies for DLBCL. Trial registration The study was registered in the Chinese Clinical Trial Registry (ChiCTR2200060430; http://www.chictr.org.cn/) on June 1, 2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06951-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27222190fd1301e27cfd9869b08b07fc58aa185" target='_blank'>
              SH3BP5-driven metabolic-immune crosstalk in DLBCL: a prognostic biomarker and therapeutic target for reshaping immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Tong Wu, Yi Yang, Yuan Zong, Jiawen Zhao, Xiaoyu Zhao, Lei Li, Yiming Gao, Ning Li, Liting Jiang, Yinyin Xie
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract BRCA2 mutations contribute to the pathogenesis and treatment sensitivity of a subset of ovarian, breast, prostate, and pancreatic cancers. When these cancers become therapy resistant, secondary mutations that restore the BRCA2 open reading frame are found in half the cases, but other causes of resistance remain incompletely understood. Here, we identified translational readthrough of a premature termination codon (PTC) as a cause of resistance to poly(ADP-ribose) polymerase inhibitors (PARPis) and cisplatin in cells derived from the BRCA2-mutated ovarian cancer line PEO1 by PARPi selection. Despite persistence of the signature 4965C > G (p.Y1655X) BRCA2 mutation, low-level expression of full-length BRCA2 protein was detectable in these cells by immunoblotting and tandem mass spectrometry. Either BRCA2 knockdown or gene interruption 5′ or 3′ to the PTC restored treatment sensitivity, implicating BRCA2 in the resistance. Reporter assays demonstrated UAG-selective readthrough in the resistant clones but not parental cells. Moreover, custom searching of global proteomic data indicated readthrough of stop codons, particularly UAGs, in additional proteins in the resistant clones. Finally, multi-omic analysis identified multiple changes in the nonsense-mediated decay and termination machineries that favor readthrough. Accordingly, the present results identify PTC readthrough as a potential mechanism of drug resistance in cells with BRCA2 nonsense mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5f812b097b8ef89a4b208e38861c4246f71092" target='_blank'>
              Codon specific readthrough as a mechanism of BRCA2 restoration in acquired PARP inhibitor and chemotherapy resistance
              </a>
            </td>
          <td>
            Cordelia McGehee, X Wei Meng, Xinyan Wu, Cristina Correia, A. Venkatachalam, K. Flatten, Kevin L. Peterson, , Olivia K. Rossman, Clara Wong, R. Hurley, Jill M. Wagner, Paula A. Schneider, Haiming Dai, Guy G Poirier, Andrea E. Wahner Hendrickson, Elizabeth M. Swisher, Akhilesh Pandey, Hu Li, Scott H. Kaufmann
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b535784b00dc4f23ce87d1f10a7f009abae40d" target='_blank'>
              Extensive and differential platinum chemotherapy mutagenesis in children
              </a>
            </td>
          <td>
            Anna Wenger, H. Lee-Six, Manas Dave, M. Layeghifard, A. Lawson, Federico Abascal, Pantelis A. Nicola, T. D. Treger, Toochi Ogbonnah, Conor Parks, Thomas R. W. Oliver, Jonathan Kennedy, A. Hodder, Nathaniel D Anderson, Felipe Luz Torres Silva, Mi K Trinh, Thomas Dowe, Marwo Habarwaa, James J. Sun, Sergio Assia-Zamora, M. Cortes-Cerisuelo, W. Jassem, Charlotte Town, Anil Dhawan, Vandana Jain, Karin Straathof, M. Deheragoda, I. Martincorena, Liina Palm, J. C. Hutchinson, Tim H H Coorens, Claire Trayers, Nigel Heaton, A. Shlien, Yoh Zen, Foad J. Rouhani, Sam Behjati
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background Human papillomavirus (HPV)-associated oropharyngeal cancer (OPC) is increasing in prevalence, but the drivers of metastasis remain poorly understood, which impacts the ability to personalise management decisions. Much of the genomic research to date focuses on the HPV-negative population. Here, we utilise single-cell and spatial single-cell techniques to understand the drivers of metastasis. Methods Patients with HPV-positive OPC and cervical lymph node metastases treated with curative surgery had matched samples from the primary and lymph nodes collected for research. Single-cell RNA sequencing, single-cell spatial sequencing (Visium) and in-situ spatial platforms were performed. Cancer clones were delineated using inferred copy number variation. Expression phenotypes and interactions with the tumour microenvironment were compared between the metastasising and non-metastasising cancer clones. Results Individual cancer clones have varied abilities to metastasise and undergo clonal expansion in the lymph node, with only a subset of clones present in the primary expanding in the lymph node. Four mechanisms were identified as defining the metastatic phenotype, including protein translation adaptation, immunoproteasome dysfunction and immune evasion, suppression of the IFN immune response and cap-independent protein translation. Conclusions This research elucidates multiple mechanisms driving the expansion of cancer clones in HPV-positive oropharyngeal cancer. By detailing the roles of translational adaptation, immunoproteasome dysfunction, suppression of the interferon immune response and cap-independent protein translation, we provide insights into how these processes contribute to immune evasion and tumour survival. Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04392-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fda352f2dd3f74c2b6cb392920d418565101f0" target='_blank'>
              Protein translation dysregulation and immune cell evasion mediated by IFN and immunoproteasome downregulation define metastatic clones in HPV-related cancer of the oropharynx
              </a>
            </td>
          <td>
            Venessa T. Chin, W. Muskovic, Rachael A. McCloy, Drew R Neavin, José Alquicira-Hernández, Himanshi Arora, A. Senabouth, Kavitha Krishna Sudhakar, Patricia Keith, Ellie Spenceley, Dominik Kaczorowski, Peter Floros, Brett Leavers, Julia Crawford, Richard Gallagher, Joseph E. Powell
          </td>
          <td>2025-10-14</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Late diagnosis and chemotherapy resistance, particularly to 5-fluorouracil (5-FU), contribute to the low survival rate in cholangiocarcinoma (CCA) patients. Identifying relevant genes and pathways, as well as novel targeted molecules, is crucial to overcoming 5-FU resistance and improving treatment outcomes for CCA patients.


OBJECTIVES
This study aimed to determine the potential molecules associated with 5-FU resistance in CCA cells.


MATERIAL AND METHODS
Transcriptomic datasets from 4 stable 5-FU-resistant cell lines and their corresponding parental lines were retrieved from the Gene Expression Omnibus. A series of bioinformatics analyses were conducted to identify key genes upregulated in 5-FU-resistant cells compared to their parental counterparts. The expression levels of candidate genes identified through bioinformatics analysis were validated in CCA tissues and cell lines.


RESULTS
Differential gene expression, protein-protein interaction, and Hub genes analysis revealed 8 genes that were significantly upregulated in 5-FU resistance cells compared to their parental cells. Six of the 8 genes, including TCP1, RPS6, RPS29, HSPA5, RPS15A, and NOTCH1, were upregulated in patient CCA tissues. Using real-time PCR, only the expression levels of NOTCH1 and TCP1 were significantly higher in the 5-FU insensitive CCA cell lines, KKU-213A and KKU-213B, than that of the 5-FU sensitive CCA cell line, KKU-055. A similar result was observed in stable 5-FU-resistant cell lines (KKU-213A-FR and KKU-213B-FR) compared to their parental cells.


CONCLUSIONS
The bioinformatic analysis and PCR results revealed that NOTCH1 and TCP1 might be associated with 5-FU resistance and serve as potential molecular targets to enhance 5-FU sensitivity in CCA cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8104f35b6140638792fb57197dadae1949d4723" target='_blank'>
              Bioinformatics analysis identified TCP1 and NOTCH1 as potential target molecules to overcome 5-fluorouracil resistance in cholangiocarcinoma.
              </a>
            </td>
          <td>
            Sonexai Kidoikhammouan, Nopkamol Kanchanangkul, Worachart Lert-itthiporn, Raksawan Deenonpoe, Charupong Saengboonmee, S. Obchoei, S. Wongkham, Wunchana Seubwai
          </td>
          <td>2025-09-04</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b1867c4003902c154bf44ef7de285c3bf97bccb" target='_blank'>
              Selective mRNA translation determines adaptative mutability of melanoma cells to anti-BRAF/MEK combination therapy
              </a>
            </td>
          <td>
            Lucilla Fabbri, Lucie Lagadec, Eva Guérin, Hélène Lecourt, Dorothée Baille, Laetitia Besse, Cédric Messaoudi, Laurent Désaubry, Hussein Abou-Hamdan, Sévérine Roy, Bérangère Lombard, D. Loew, Jean Yves Scoazec, Caroline Robert, S. Vagner
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi A. Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acute leukemias are highly aggressive hematologic malignancies that demand intensive chemotherapy regimens. However, drug toxicity remains a major barrier to treatment success and patient survival. In this context, pharmacogenomics offers a promising strategy by identifying single-nucleotide variants (SNVs) that influence drug metabolism, efficacy, and toxicity, ultimately impacting treatment outcomes. This study analyzed data from the ClinPGx/PharmGKB database to identify clinically annotated variants related to chemotherapy response in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL). A total of 24 variants were curated for AML and 57 for ALL. Among these, nonsynonymous variants were most frequent in ALL (31.6%), while synonymous variants predominated in AML (33.3%). Although traditionally considered neutral, synonymous and intronic variants may influence gene expression through regulatory or splicing mechanisms. The analysis revealed clinically significant variants associated with chemotherapy response, particularly in the ABCB1 gene, observed in 12.5% of AML and 10.5% of ALL cases. Several variants, particularly TPMT, NUDT15, ABCC1, SLC28A3, and RARG, were associated with severe adverse effects such as myelotoxicity, mucositis, cardiotoxicity, and hepatotoxicity. This study reinforces the importance of genetic variants in modulating the therapeutic response and toxicity to chemotherapy drugs in acute leukemias. Analysis of ClinPGx/PharmGKB data emphasizes ABCB1 as a potential resistance marker and supports pre-treatment genotyping of genes like TPMT and NUDT15 to prevent severe toxicities. Future advances should include the expansion of pharmacogenetic studies in underrepresented populations and the clinical validation of new markers in prospective trials, aiming to consolidate precision medicine as a routine part of the therapeutic management of acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d398f2f78483d0f180028688c26b519950cb78" target='_blank'>
              Pharmacogenomics Applied to Acute Leukemias: Identifying Clinically Relevant Genetic Variants
              </a>
            </td>
          <td>
            Flávia Melo Cunha de Pinho Pessoa, Isabelle Magalhães Farias, B. Nogueira, C. B. Machado, Igor Valentim Barreto, Anna Karolyna da Costa Machado, Guilherme Passos de Morais, Leidivan Sousa da Cunha, Deivide de Sousa Oliveira, André Pontes Thé, Rodrigo Monteiro Ribeiro, Patrícia Maria Pontes Thé, Manoel Odorico de Moraes Filho, Maria Elisabete Amaral de Moraes, Caroline Aquino Moreira-Nunes
          </td>
          <td>2025-10-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe8f189447c321586a776e941cd069962ea01d63" target='_blank'>
              Identification and analysis of metabolic reprogramming-related genes in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Husheng Kong, Mengyuan Pan, Li Sun
          </td>
          <td>2025-10-25</td>
          <td>Clinical and experimental medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Synthetic lethality offers a powerful strategy for precision cancer therapy by exploiting vulnerabilities in gene-gene interactions. Unlike single-gene perturbations, many therapeutics – such as CDK4/6 inhibitors – target paralogous proteins, suggesting that synthetic lethal nodes might be composed of gene-pairs that share redundant or compensatory functions. Historically, efforts to identify synthetic lethal relationships have relied on correlating single-gene dependencies with gene expression profiles across panels of cancer models. However, this correlative approach is non-experimental in nature and may miss important synthetic lethal pairs, especially when the number of models harboring loss of one gene in the pair is limited. Systematic and experimental interrogation is thus essential to uncover these vulnerabilities. Identifying such interactions, along with their associated biomarkers, provides a foundation for the development of novel combinatorial therapies targeting oncogenic drivers, tumor-suppressor and passenger gene deletions.



 To systematically map paralog dependencies, we performed large-scale digenic CRISPR screens across 240 diverse cancer models spanning 17 lineages. Our library targets 4,546 single genes and 7,644 gene pairs encompassing druggable targets and their extended networks. Each gene is targeted with 6 sgRNAs, and each gene-pair with 18 sgRNA combinations – constituting the most comprehensive digenic CRISPR library to date. Each Cas9-expressing cancer model was transduced at low multiplicity of infection with ∼500x sgRNA coverage, and cellular fitness assessed at screen day 21.



 Leveraging an adapted version of the Chronos pipeline optimized for inferring gene-pair interactions, we identified 65 pan-essential paralog pairs, as well as numerous context-specific paralog dependencies associated with genetic alterations, lineage features and expression profiles. Among these, the INTS6/INTS6L pair emerged as a novel paralog synthetic lethal interaction, with INTS6L representing a targetable vulnerability in tumors including prostate cancer that are co-deficient for INTS6 and RB1 – a finding we validated in vitro and in vivo. We further uncovered strong dependency on KRAS/NRAS activity in FGFR-fusion cancers, which exhibit marked sensitivity to the pan-RAS inhibitor RMC-6236 and to the combination of RMC-6236 and the FGFR2 inhibitor RLY-4008 which synergistically enhanced antiproliferative effects in these models.



 This study presents a systematic and high-resolution map of paralog dependencies across cancer, uncovering novel synthetic lethal interactions and biomarker-associated vulnerabilities. By expanding functional genomic screens to paralog pairs, we unveil a new dimension of cancer dependencies. These insights provide a valuable foundation for the rational development of next-generation therapeutic strategies that harness genetic redundancy and synthetic lethality – offering new precision oncology opportunities for tumors currently lacking effective targeted therapies.



 Jayu Jen, Yao He, Dennie T. Frederick, Alex Marzoratti, Sean Chen, Tanay Thakar, Alan Zhang, Arianny Acosta, Guangyan Li, Baomou Feng, Mira Chokshi, Alison Liu, Alina Guo, Devishi Kesar, Diego Rodriguez, Nancy Jung, Zaid Bayyat, William R. Sellers. A pan-cancer map of paralog dependencies reveals novel synthetic lethal targets and biomarker-defined vulnerabilities [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f75061555a30ae7d53b61ac9047336e69a100ee" target='_blank'>
              Abstract C039: A pan-cancer map of paralog dependencies reveals novel synthetic lethal targets and biomarker-defined vulnerabilities
              </a>
            </td>
          <td>
            Jayu Jen, Yao He, Dennie T. Frederick, Alex Marzoratti, Sean Chen, Tanay Thakar, Alan Zhang, Arianny Acosta, Guangyan Li, Baomou Feng, Mira K Chokshi, Alison Liu, Alina Guo, Devishi Kesar, Diego Rodriguez, Nancy Y. Jung, Zaid Bayyat, William R Sellers
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-12</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis through their capacity for self-renewal and multilineage differentiation. However, the isolation and functional characterization of HSCs remain challenging due to their cellular heterogeneity and dynamically regulated transcriptional and epigenetic landscapes. Advances in experimental and computational biology, including single-cell RNA sequencing (scRNA-seq), chromatin immunoprecipitation sequencing (ChIP-seq), network inference algorithms, and machine learning, have improved our ability to resolve transcriptional states, trace lineage trajectories, and reconstruct gene regulatory networks (GRN) at single-cell resolution. These approaches enable the discovery of novel HSC subtypes and regulatory factors, and facilitate the integration of multi-omics data to uncover epigenetic and transcriptional mechanisms that drive stem cell fate decisions. Additionally, machine learning models trained on high-throughput datasets provide predictive power for identifying novel enhancers, transcription factors, and therapeutic targets. This review underscores the synergistic role of computational tools in deciphering HSC biology and highlights their potential to improve stem cell therapies and precision treatments for hematologic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3c26c697c565c33ced832e7a44009a59ac4768" target='_blank'>
              Unlocking hematopoietic stem cell potential: integrative computational approaches for genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            P. Raghav, Basudha Banerjee, Rajni Chadha
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Anticancer drug resistance remains a significant challenge to the efficacy of cancer treatment, with DNA repair enzymes contributing to this resistance. We hypothesized that thymine DNA glycosylases (TDGs) may be involved in anticancer drug resistance given their dual function of DNA repair and demethylation as well as investigated their possible involvement in the induction of β-catenin in SNUC5 cells resistant to 5-fluorouracil (SNUC5/5-FUR) and oxaliplatin (SNUC5/OXTR). The expression of TDG and phospho-β-catenin increased in both resistant cell types when compared to that in SNUC5 cells. Moreover, knockdown of TDG significantly suppressed phospho-β-catenin expression in both resistant cell types, resulting in enhanced sensitivity to anticancer drugs. TDG binding to the β-catenin promoter was stronger in both resistant cell types than in SNUC5 cells, showing a decreased methylation pattern in the CpG islands of the β-catenin promoter. Furthermore, another DNA demethylase, ten-eleven translocation 1 (TET1), showed the same pattern as TDG in both resistant cell types. Additionally, TDG significantly interacted more with TET1 in both resistant cell types than in SNUC5 cells, enhancing binding to the same locus in the β-catenin promoter. These findings suggest that TDG may be a promising target molecule for overcoming drug resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def655ba0cc554974966a66e65dd66380f445b21" target='_blank'>
              Involvement of the DNA Demethylase Thymine DNA Glycosylase in Anticancer Drug Resistance.
              </a>
            </td>
          <td>
            K. Kang, M. Piao, Herath Mudiyanselage Maheshika Madhuwanthi Senavirathna, Hye-Jin Boo, Sang-Pil Yoon, Joo Mi Yi, Jinny Park, J. Hyun
          </td>
          <td>2025-10-15</td>
          <td>Biomolecules & therapeutics</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary Breast cancer (BC) remains a significant global health challenge, partly due to its ability to develop resistance to treatment. BC resistance arises from various biological mechanisms, including cancer cells actively expelling drugs, altering their internal metabolic and signaling pathways, evading the immune system, or persisting as drug-tolerant cancer stem cells. These changes occur in the context of a tumor microenvironment that further shields the cancer, limiting treatment access. As the disease advances, particularly when it spreads to other body parts, effective treatment becomes increasingly difficult. Recent advancements in artificial intelligence (AI) offer promising avenues for improving early detection, refining diagnoses, enhancing clinical decision-making and thereby personalizing treatment strategies. This review examines the biological mechanisms that cause BC resistance and its treatment. It further explores AI’s potential to address critical challenges, transforming BC therapy to provide improved survival rates and enhanced quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb5357bf49f1fab994fbcce36bcc1738760cec3" target='_blank'>
              The Underlying Mechanisms and Emerging Strategies to Overcome Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Krishnaswamy Kannan, Alagarsamy Srinivasan, Aarthi Kannan, Nawab Ali
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="CDK7 has emerged as a cancer target because of its pivotal roles in cell cycle progression and transcription. Several CDK7 inhibitors (CDK7i) are now in clinical evaluation. Identifying patients most likely to respond to treatment and early detection of tumour evolution towards resistance are necessary for optimal implementation of cancer therapies. Continuous culturing of prostate cancer cells with Samuraciclib, a non-covalent ATP-competitive CDK7i, led to outgrowth of resistant cells. These were characterised by the acquisition of a single base change in the CDK7 gene, Asp97 to Asn (D97N). Mutant cells were resistant to other non-covalent CDK7i but remained sensitive to covalent CDK7i. Cryo-EM structure and kinase ligand affinity determinations revealed reduced affinity of the CDK7-D97N mutant for non-covalent CDK7i. Remarkably, Asp97 is absolutely conserved in human CDKs, inferring its importance for the activities of all CDKs. Consistent with this, mutation of the homologous residue in CDK12 (D819N) or CDK4 (D99N) promoted resistance to drugs that inhibit these CDKs. Our findings reveal a general mechanism for acquired resistance with obvious implications for patients treated with CDK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4e4fc33eb2e8814d38d33c359427c463ea1f89" target='_blank'>
              Resistance to CDK7 inhibitors directed by acquired mutation of a conserved residue in cancer cells
              </a>
            </td>
          <td>
            Chun-Fui Lai, Victoria I. Cushing, Ellen Olden, C. Bevan, R. C. Coombes, Basil J. Greber, L. Buluwela, Simak Ali
          </td>
          <td>2025-09-08</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematologic malignancy characterized by the accumulation of myeloid blasts in the bone marrow. Despite the availability of potential curative treatments, patients frequently experience unfavorable outcomes. One crucial aspect contributing to relapse is the plasticity of leukemic clones, which enables them to switch between active proliferation and dormancy. The adaptability of AML underscores the need for novel therapies targeting AML-specific proteins. To address this, genome-wide CRISPR screens can be utilized to identify cancer entity-specific vulnerabilities. Leveraging publicly available functional genomics datasets and comparing AML with non-AML cancer cell lines, we identified a significant dependency on the cell cycle-regulating gene MYBL2 in AML. We describe MYBL2 as a key driver of AML cell growth and proliferation, highlighting its established role as a cell cycle regulator. Also, our findings uncover its previously unrecognized function as an inhibitor of cellular senescence. A knockdown of MYBL2 induces cell cycle arrest in the G2/M phase with subsequent induction of apoptosis in vitro, and reduces leukemic burden in a patient-derived xenograft (PDX) model in vivo. Interestingly, some AML cells evade apoptosis and enter a senescent-like phenotype upon MYBL2-knockdown, which is reversible upon re-expression of MYBL2. Finally, analyses of clinical data from two publicly available patient cohorts demonstrate a lower probability of survival in patients with higher MYBL2 expression, further hinting at the potential relevance of MYBL2 in AML. In conclusion, our findings demonstrate the essential role of MYBL2 in AML, governing the balance between cell proliferation, cell survival and senescence, ultimately influencing the fate of AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bab0ba7401319632cc10fcda395ad12a7cf902e" target='_blank'>
              Cell cycle regulator MYBL2 is a distinct vulnerability in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sandra Küchler, Silke Brilloff, Silvia Schäfer, Elahe Rahimian, Vida Kufrin, Shraddha S Peri, J. Musa, T. Grünewald, Denis M Schewe, C. Ball, M. Bornhäuser, H. Glimm, Marius Bill, A. A. Wurm
          </td>
          <td>2025-10-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beea8538ea0fee6e0373855f4448e35dd9857e52" target='_blank'>
              From Genes to Personalized Cancer Care: A Systematic Review of Toxicity-Associated Genetic Variants in Solid Tumor Treatments
              </a>
            </td>
          <td>
            A. Gonzalez-Hernandez, A. Escamilla-Sanchez, E. Perez-Ruiz, A. Rios, C. A. Frecha, F. Vaca-Paniagua, I. Barragan, S. Perdomo, A. Rueda-Domínguez, J. Oliver
          </td>
          <td>2025-10-24</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, Andrew C White
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
To analyze LUAD cases in The Cancer Genome Atlas (TCGA), the mRNA expression-based stemness index (mRNAsi) was used. Models of cancer immunity and LUAD prognosis were developed on the basis of correlations between immune and stem cell genes.


METHODS
We investigated the differential expression of mRNA dryness index (mRNAsi) in LUAD, survival prognosis, and correlation with clinical parameters. Iden-tify key mRNAsi-related modules and genes by weighted gene co-expression network analysis (WGCNA). Gene set enrichment analysis (GSEA/GSVA) was used to identify stem cell markers and immune-related differentially expressed genes (SC IRGs), and 10 key genes were enriched. Subgroup enrichment, gene mutations, genetic correlated-ness, gene expression, immunity, tumor mutational burden (TMB), and drug sensitivity were further performed in the comprehensive analysis of pivot genes and subgroups.


RESULTS
Compared with normal cells, LUAD cells presented significantly greater mRNAsi values through differential expression analysis. The mRNAsi was highly cor-related with clinical parameters (age, sex, and T stage). On the basis of WGCNA, blue-green and brown modules were identified as the most significant modules (including positive and negative correlations) associated with mRNAsi expression. The functions and pathways of the two mRNAsi-related modules were enriched mainly in tumor oc-currence, development, and metastasis. Cox regression analysis was used to identify 30 SCIRGs associated with prognosis by combining the stem cell indices of the DEGs and the immune-related DEGs. A LASSO regression model was constructed after 10 DEGs related to the prognosis of patients with LUAD were detected. There were significant differences between the high-risk and low-risk groups in terms of GSEA/GSVA, im-mune cell correlation, clinical correlation, etc., following model validation (P<0.05).


DISCUSSION
There are a total of 10 genes in our study model, including four key pre-dictors: DGRIK2, PTTG1, LGR4, and PDGFB. The other 6 genes need to be further delineated and verified. To date, our research has some limitations and has not been validated in cell or animal experiments. These findings provide a relevant theoretical basis for subsequent experimental research on lung adenocarcinoma stem cells. Further research into these cancer stem cell genes will increase the likelihood that they play a role in cancer. There is an opportunity to use it as a therapeutic target for targeted ther-apy for lung adenocarcinoma in the future.


CONCLUSION
mRNAsi is associated with immunity, which was previously overlooked in the gene analysis of LUAD stem cells. These key genes have a strong overall corre-lation, which can be achieved by inhibiting the stemness characteristics of cancer cells, which may lay the foundation for future research on LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c303cbba9ae27f76f9e2be614e68543e2587be" target='_blank'>
              Identification of the Immune Microenvironment, Mutation Burden, Immunotherapy, and Drug Sensitivity Related to Lung Adenocarcinoma Tumor Stem Cells via WGCNA.
              </a>
            </td>
          <td>
            Qi Liu, Liusheng Wu, Meiling Lu, H. Jia, Xiaoqiang Li
          </td>
          <td>2025-09-26</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96cb29c7c4ee18f843780f85e8c5d424d7abe77" target='_blank'>
              Epigenetic modulation in cancer drug discovery: promising targets and clinical applications.
              </a>
            </td>
          <td>
            S. Vickram, Shofia Saghya Infant, M. Sivasubramanian, Saravanan Anbalagan, Mathan Muthu Chinnakannu Marimuthu, Hitesh Chopra
          </td>
          <td>2025-09-02</td>
          <td>Pharmacological reports : PR</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a93b5a70ed710c9aacd9cdfba3788cae5ad699" target='_blank'>
              Large-scale single-cell phylogenetic mapping of clonal evolution in the human aging esophagus
              </a>
            </td>
          <td>
            Tamara Prieto, Dennis J. Yuan, John Zinno, Clayton Hughes, Nicholas Midler, Sheng Kao, Jani Huuhtanen, Ramya Raviram, Foteini Fotopoulou, Neil Ruthen, Srinivas Rajagopalan, Joshua S. Schiffman, Andrew R. D’Avino, Sang-Ho Yoon, Jesus Sotelo, Nathaniel D. Omans, Noelle Wheeler, Alejandro Garces, Barun Pradhan, A. P. Cheng, N. Robine, Catherine Potenski, Katharine Godfrey, N. Kakiuchi, Akira Yokoyama, Seishi Ogawa, Julian Abrams, Ivan Raimondi, D. Landau
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040ef40dcc6831cbd55de051d8493936b568f2e" target='_blank'>
              A Rapid Gene Expression Profiler Classifies AML Tumor Responsiveness to Standard Therapies
              </a>
            </td>
          <td>
            S. Kurtz, C. Eide, Andy Kaempf, Nicola Long, Alexandria Miller, D. Bottomly, Shannon K. McWeeney, Curtis A Lachowiez, Sudharshan Anand, , J. C. Wang, J. Dick, J. Tyner
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aadf318bf49329614cbd65f8b3f0ff419cae30" target='_blank'>
              Pan-cancer analysis reveals context-dependent roles of LINE-1 ORF1p in immune regulation and copy number alterations
              </a>
            </td>
          <td>
            E. Kaparos, Wenjing Zhang, Ryusei Miyanaga, Wilson H. McKerrow, J. Boeke, Teresa Davoli, P. Mita, Kelly Ruggles, David Fenyö
          </td>
          <td>2025-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome in bladder cancer drives metabolic reprogramming
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC.Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene.Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes.This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ea6290df028815fb911512fc1b477390936835" target='_blank'>
              Pan-Cancer Single-Cell Profiling Uncovers the Biological Characteristics of Cancer-Testis Genes
              </a>
            </td>
          <td>
            Chunyang Fu, Ke Liu
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d27cc9b475222eaa7f4e83e26c9db176ee78f1" target='_blank'>
              Novel mutations in transducin-like enhancer of split 6 cause infertility in women with embryonic arrest.
              </a>
            </td>
          <td>
            Jifan Tan, Yue Jiang, Ching Wan, Kehan Zhu, Yan Xu, Yanwen Xu, Jia Huang
          </td>
          <td>2025-09-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third leading cause of cancer-related deaths. Integrating cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present a comparative mRNA-seq analysis of tissue samples from 32 CRC patients, pairing tumors with adjacent healthy tissues. Differential expression gene (DEG) analysis revealed dysregulated metabolic programs. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2, which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potential of the Xc- system, we analyzed gene perturbations from CRISPR screens across various cell types and used functional assays in five primary patient-derived organoid models. We identified a previously uncharacterized cell surface protein signature predicting chemotherapy resistance and highlighted the causality and potential of pharmacological blockage of ferroptosis as a promising avenue for cancer therapy. Redox homeostasis, ion/amino acid transporters, and regulators of neuronal survival and differentiation were pathways associated with these co-dependent genes in patient specimens. This study highlights several potential clinical targets for CRC therapy and promotes the use of patient-derived organoids oids to functionally validate in silico predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af3aa92ab85625c6d369fc2c084fc1ee49ca6118" target='_blank'>
              Patient-specific pharmacogenomics demonstrates xCT as predictive therapeutic target in colon cancer with possible implications in tumor connectivity.
              </a>
            </td>
          <td>
            M. Strecker, Keren Zohar, Martin Böttcher, T. Wartmann, H. Freudenstein, Maximilian Doelling, M. Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, D. Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. Kahlert
          </td>
          <td>2025-09-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly lethal malignancy characterized by a complex pathological mechanism involving multiple genes and progressive stages. The efficacy of targeted and immunotherapy remains limited, highlighting the urgent need for a reliable model to predict prognosis and response to immune checkpoint inhibitors (ICIs). Methods We developed an integrated model based on genes related to autophagy, senescence, dormancy, mitochondrial function, and tumor stemness. The predictive capability of this model for HCC prognosis and ICI response was evaluated. Single-cell transcriptomic analysis and immunocompetent mouse models were further utilized to elucidate the role of model-associated genes in regulating the tumor immune microenvironment. Results A 16-gene integrated model was constructed using genes associated with mitochondrial function, autophagy, dormancy, stemness, and senescence. This model demonstrated robust predictive power for HCC prognosis and ICI responsiveness. Single-cell trajectory analysis revealed that EZH2 plays a crucial role in immune cell infiltration, activation, and HCC progression. Additionally, in vivo mouse models further indicated that EZH2 may regulate CD8+ T cell activation and exhaustion through the MIF-CD74 signaling pathway. Conclusion The integrated model holds potential as a prognostic and predictive tool for HCC immunotherapy. EZH2 may influence CD8+ T cell activation and exhaustion via the MIF-CD74 axis, providing insights for patient stratification and potential therapeutic strategies to enhance immunotherapy efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07071-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7937b56d5b80810f28cf602b4f87ac567637d0a" target='_blank'>
              Integrated single-cell RNA-seq analysis reveals that EZH2 regulates the MIF-CD74 axis to modulate T cell activation and exhaustion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yangyang Zhou, Yi Xu, Mengfan Ye, Jiayi Xing, Di Chen, Al-Ameer Wail Hussein Ahmed, Al-Rahabi Rehab Abdullah Mohammed Ali, Chenwei Pan, Xiangchou Yang, Zan Shen
          </td>
          <td>2025-10-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Precision oncology, defined as molecular profiling of an individual’s tumor to guide therapy, has rapidly evolved in the past few decades with the advancement of NGS technologies. Since not all patients’ tumors show targetable DNA alterations, a comprehensive genomic and transcriptomic profiling (CGTP) approach is warranted to provide therapy options for patients beyond those with actionable DNA alterations. Here, we conducted a prospective study to determine the added value of CGTP using BostonGene Tumor Portrait™ test in identifying actionable DNA and RNA alterations and therapy selection for patients who have had targeted panel sequencing (TPS) on an institutional panel.



 117 patients with advanced solid tumors who had prior TPS, and no College of American Pathologists (CAP) Tier-1 actionable targets underwent CGTP between August 2022 and October 2024 at MD Anderson Cancer Center. A CGTP biomarker result was considered actionable if it was therapeutically targetable or predictive of response or resistance to a clinically available therapy. Patients matched to CGTP-informed therapies were evaluated for clinical benefit.



 99 patients (84.6%) were included in the final analysis, while 18 patients (15.4%) were screen failed due to tissue insufficiency or incomplete data. Median turn-around time from consent to sample acquisition (SA) and from SA to CGTP report were 14 and 15 days, respectively. Median age of the patients was 50 years (13-74). Fifty-four (54.5%) patients were female and 78 (78.8%) were white. Head and neck (26.3%), sarcoma (19.2%), ovarian (8.1%) and mesothelioma (7.1%) were the most common histologies. Median prior lines of therapy in the metastatic setting was 3 (0-13). CGTP identified at least one actionable DNA alteration not captured by prior TPS for 59 (59.6%) cases. Of those variants, 43.8% were deletions, 26% amplifications and 15.6% SNVs. CGTP found previously undocumented pathogenic germline variants for 3 (3.0%) cases. Actionable RNA expressions (AREs) were identified for all 99 patients with a median ARE of 22 (interquartile range (IQR), 17.0-26.0). The majority of AREs were RNA overexpressions (86.0%). Of 33 patients with actionable copy number alterations (CNAs), 16 (48.5%) had concomitant AREs in the corresponding genes. Of 11 patients with AREs who also had IHC, 7 (63.6%) had corresponding actionable protein expressions. Forty patients had a treatment change after CGTP, 19 of whom (47.5%) received CGTP-matched therapies (12 DNA matches and 7 RNA matches) at the time of data cut-off, for whom the overall response rate and clinical benefit rate were 15.8% and 31.6%, respectively. The median progression-free survival was 4.0 months (IQR, 2.0-8.0).



 Our data shows the feasibility and utility of CGTP to match patients to molecularly informed treatments within a clinically relevant time frame. Transcriptome analysis not only confirms actionable CNAs but also helps identify changes in the expression of novel therapeutic targets, enhancing opportunities for precision oncology.



 Burak Uzunparmak, Fei Su, Amber M. Johnson, Kenna R. Shaw, Elisabeth Kathleen. Kong, Anil Korkut, Camila Braganca-Xavier, Ying Yuan, Nathan Fowler, Francesca Paradiso, Lile Kontselidze, Anna Butusova, Aleksander Bagaev, Ecaterina E. Dumbrava, Jordi Rodon-Ahnert, David S. Hong, Siqing Fu, Timothy Anthony. Yap, Aung Naing, Sarina Anne. Piha-Paul, Renata Ferrarotto, Bryan Iorgulescu, Shashikant Kulkarni, Keyur P. Patel, Funda Meric-Bernstam. Clinical Utility of Comprehensive Genomic and Transcriptomic Analysis to Guide Therapy for Patients with Metastatic Solid Tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C054.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2126ae20346cd383bf746372ed354cdaca6a44e4" target='_blank'>
              Abstract C054: Clinical Utility of Comprehensive Genomic and Transcriptomic Analysis to Guide Therapy for Patients with Metastatic Solid Tumors
              </a>
            </td>
          <td>
            B. Uzunparmak, Fei Su, Amber M. Johnson, K. Shaw, Elisabeth Kong, Anil Korkut, Camila Braganca-Xavier, Ying Yuan, N. Fowler, Francesca Paradiso, Lile Kontselidze, A. Butusova, Aleksander Bagaev, E. Dumbrava, Jordi Rodon-Ahnert, David S. Hong, Siqing Fu, Timothy A. Yap, A. Naing, S. Piha-Paul, R. Ferrarotto, B. Iorgulescu, Shashikant Kulkarni, , Funda Meric-Bernstam
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573448152ca34657743158301dc6d5b323113cd1" target='_blank'>
              TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition
              </a>
            </td>
          <td>
            Jean-François Spinella, J. Chagraoui, I. Boivin, Guillaume Richard Carpentier, Céline Moison, N. Mayotte, François Béliveau, Vincent-Philippe Lavallée, J. Hébert, Guy Sauvageau
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            , , Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Hypoxia is a recognized driver of transcriptional heterogeneity in high-grade glioblastoma (GBM), yet its functional significance in low-grade gliomas (LGGs) remains poorly understood. We hypothesized that hypoxia orchestrates a reversible quiescence-activation axis in LGGs via EGFR-dependent transcriptional reprogramming, thereby influencing tumor progression.



 We conducted integrative analyses of single-cell RNA sequencing (n=7 patients) and spatial transcriptomics (n=8 samples) across molecularly defined IDH-mutant LGGs to delineate hypoxia-associated transcriptional states. EGFR mutation, and clinical correlations with overall survival, were assessed in The Cancer Genome Atlas (TCGA) bulk RNA-seq cohorts. To functionally validate microenvironmental contributions, we established innovative co-culture models of LGG cells with hypoxia-conditioned primary microglia and astrocytes.



 We identified a hypoxia-associated, LGG-specific ribosomal-enriched (RE) transcriptional program characterizing a non-cycling subpopulation enriched in quiescence and activation stages. Quiescent RE cells exhibited glioma stem cell (GSC)-like features, with EGFR signaling dependency for transition to an activated phenotype. Spatial transcriptomics revealed distinct co-localization of hypoxia, RE program expression, and quiescent states, whereas EGFR-activated cells formed discrete micro-niches. Hypoxia-conditioned microglia significantly enhanced RE program activation and EGFR signaling, establishing a functional microenvironmental driver of phenotypic plasticity.



 Our findings delineate a novel hypoxia-EGFR signaling axis sustaining a quiescent yet activation-capable subpopulation in IDH-mutant LGGs. These insights nominate hypoxia-modulated EGFR pathways as potential therapeutic targets to disrupt tumor cell plasticity. Future studies will interrogate vulnerabilities of RE-programmed LGG cells under hypoxic conditions to inform next-generation therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748507ebfe3412d3babe3bb777f9dae226192bc1" target='_blank'>
              P02.22.A HYPOXIA-DRIVEN EGFR SIGNALING PROMOTES A QUIESCENT-ACTIVATION AXIS IN IDH-MUTANT LOW-GRADE GLIOMAS
              </a>
            </td>
          <td>
            D. Das, V. Jariwala, A. Pandele, A. Mohammed, T. Das, N. Ghariwala, C. Krolla, S. Arosh, N. Hossain, I. Hosen, S. Smith, J. Beck, J. Grauvogel, I. Vasilikos, R. Roelz, R. Rahman, M. Börries, K. Joseph, V. Ravi, G. Andrieux, S. Chakraborty
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite extensive prior research on prostate cancer (PCa) transcriptomics, the molecular mechanisms underlying the disease’s progression, particularly in the castration-resistant or metastatic stages, remain incompletely understood. The majority of recent research has concentrated on bulk RNA sequencing, which could mask the variation found in tumor microenvironments. This study aims to address this gap by integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing with weighted gene co-expression network analysis (WGCNA) to investigate the molecular mechanisms of PCa at a higher resolution. In order to further individualized treatment plans for PCa, we aim to discover important genes and signaling pathways that could be used as therapeutic targets. We first preprocessed expression profile data from prostate cancer tissue samples, selecting 9,809 high-quality cells from a dataset. Following batch correction with Harmony and dimensionality reduction with principal component analysis (PCA), we used the Louvain clustering algorithm to divide the cells into discrete subtypes. The clusters were then visualized using t-SNE. This resulted in 16 cellular subtypes categorized into five major cell types: epithelial cells, monocytes, endothelial cells, CD8 + T-cells, and fibroblasts. Analysis of receptor-ligand pairs uncovered significant interactions between monocytes and both tumor cells and endothelial cells. Applying the high-dimensional WGCNA (hdWGCNA) method to construct a gene co-expression network, we detected seven gene modules, four of which were highly expressed in tumor cell subtypes and contained 380 key genes. Combining pathway analysis, we ultimately screened six key genes: CNPY2, CPE, DPP4, IDH1, NIPSNAP3A, and WNK4. We used Cox univariate regression and least absolute shrinkage and selection operator (lasso) regression techniques to build a prognostic prediction model that included these six important genes based on clinical data gathered from PCa patients. The prognostic prediction model constructed in this study demonstrated excellent predictive performance in both the training set and an external validation set, with the high-risk group showing significantly lower overall survival (OS) than the low-risk group. Furthermore, there was a substantial correlation found between risk scores and several immune-related gene sets, chemotherapeutic drug sensitivity, and tumor immune infiltration. High- and low-risk groups exhibited significant differences in immune cell content, immune factor levels, and immune dysfunction. Further analysis revealed significant correlations between the expression levels of model genes and multiple disease-related genes. Through Gene Set Variation Analysis (GSVA) and Gene Set Enrichment Analysis (GSEA), we uncovered perturbations in multiple signaling pathways in high- and low-risk groups, potentially impacting the prognosis of PCa patients. This study uncovers key genes and signaling pathways in the prostate cancer tumor microenvironment, particularly genes such as CNPY2, CPE, DPP4, IDH1, NIPSNAP3A and WNK4, which have potential as therapeutic targets. Our findings provide new insights into personalized treatment strategies for PCa and warrant further clinical validation in the future. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15682-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779fa642ce8a3880c524211b95657271b91e6425" target='_blank'>
              Integrative analysis of molecular mechanisms in prostate cancer via single-cell RNA sequencing and weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Jing Zhai, Yizhou Wang, Yu Zhang, Wenhui Zhu, Xinyu Xu, Yu Peng, Guanxiong Ding
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c72346f8c39e207ff855c0ca9bc888a2b42dad" target='_blank'>
              The Ubiquitin–Proteasome System is an Important Driver of EBV-Associated Nasopharyngeal Carcinoma Progression: A Meta-Analysis of Transcriptomics Data
              </a>
            </td>
          <td>
            Hana Ratnawati, Ardo Sanjaya, Aldrich Christiandy, Lawrence S Young, Sascha Ott
          </td>
          <td>2025-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematological malignancy driven by diverse genetic aberrations, with the Philadelphia chromosome and its resultant BCR-ABL1 fusion gene constituting key pathogenic drivers. Atypical BCR-ABL1 fusion transcripts have distinctive structural and functional properties. Structural divergence in these variants leads to functional alterations of encoded oncoproteins, potentially influencing disease progression and therapeutic responsiveness. Conventional diagnostic modalities, including reverse transcription-PCR and fluorescence in situ hybridization, may fail to detect rare variants, necessitating complementary high-sensitivity techniques such as next-generation sequencing). Tyrosine kinase inhibitors (TKIs), including imatinib and dasatinib, remain cornerstone treatments; however, marked inter-variant heterogeneity in TKI responsiveness is observed: Patients harboring e13a3/e14a3 transcripts generally show favorable prognoses, while those with e1a3/e6a2 variants demonstrate an increased risk of relapse and/or TKI resistance, often requiring multimodal strategies combining chemotherapy or allogeneic hematopoietic stem cell transplantation. Although Chimeric Antigen Receptor)-T cell therapy has shown promise in treating (Philadelphia chromosome-positive B-cell Acute Lymphoblastic Leukemia, its application in CML, particularly in variants such as e1a3 or e6a2, is not currently recommended as a first-line treatment. Despite advances in elucidating the clinical implications of fusion gene heterogeneity in leukemogenesis, the prognostic value of atypical BCR-ABL1 isoforms requires further validation through multicenter studies with extended cohorts. This review aimed to summarize cases of atypical fusion genes in CML, with analysis of clinical characteristics, therapeutic interventions, and prognostic outcomes, to provide clinicians with enhanced reference material for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec3c00c917da742543a693d4c05a294ce191255" target='_blank'>
              The silent players: Atypical BCR-ABL isoforms as biomarkers and therapeutic hurdles in CML pathogenesis (Review)
              </a>
            </td>
          <td>
            Xin Zhou, Ai Li, Dexiao Kong, Yuqi Shi, Peipei Zhang, Ningning Shan
          </td>
          <td>2025-09-24</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming.
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To systematically identify causal genetic mechanisms that confer risk for coronary artery disease (CAD) in GWAS loci, we mapped genome-wide variant-to-enhancer-to-gene (V2E2G) links in vascular smooth muscle cells (SMC). Enhancers identified by active chromatin features, and further prioritized by base-resolution deep learning models of chromatin accessibility in 108 CAD loci, were studied with CRISPRi targeting and Direct-Capture Targeted Perturb-seq (DCTAP- seq) evaluation of 470 genes. Seventy-six V2E2G links were identified for 59 candidate CAD genes representing gene programs including epithelial-mesenchymal transformation, ubiquitination, and protein folding as well as BMP and TGFB signaling. Similar methods employed with an independent focused screen targeting one candidate locus at 9p21.3 identified 10 enhancers regulating expression of multiple genes at this location. Detailed molecular studies revealed that two enhancers mediating transcription factor binding and transcriptional regulation contribute to ancestry-specific and sex-specific risk for CAD and the surrogate biomarker vascular calcification. Together, these studies advance our identification of GWAS CAD V2E2G links across the genome, and specific mechanisms of risk at the complex 9p21.3 locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89dcaa475d1380644b65c37776876b145ab92042" target='_blank'>
              Enhancer-targeting CRISPR screens at coronary artery disease loci suggest shared mechanisms of disease risk
              </a>
            </td>
          <td>
            Markus O Ramste, Chad S. Weldy, Soumya Kundu, Quanyi Zhao, D. Li, Kayla Brand, Disha Sharma, Amanda Ramste, Evelyn Jagoda, Judhajeet Ray, Roxanne Diaz Caceres, James Galante, A. Gschwind, Nuutti Lahtinen, Trieu Nguyen, Junedh M. Amrute, Chong Park, J. Kim, M. U. Kaikkonen, N. O. Stitziel, Lars M. Steinmetz, Anshul B Kundaje, Jesse Engreitz, Thomas Quertermous
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcb21ab8df09d5daf66bd07fdf4abe868e528e74" target='_blank'>
              RIPPLET: Mutation-Only Gene and Pathway Profiling for Precision Oncology
              </a>
            </td>
          <td>
            Fabius Wiesmann, Cassandra Litchfield, M. Mayoux, Tumor Profiler Consortium, A. Wicki, Burkhard Becher, V. Koelzer, H. Moch, Marija Buljan, Sonia Tugues, Bettina Sobottka
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks that causes DNA damage and tumor cell death. While these drugs are front-line treatments for multiple human cancers – including high-grade serous ovarian cancer (HGSOC) – platinum resistance and tumor recurrence remains a major clinical challenge. Here, we report findings from both a genome-wide CRISPRi screen and an RNA-binding protein (RBP)-focused CRISPRi screen aimed at identifying modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, as well as to an microtubule-targeting taxane. Using an in vitro treatment regimen designed to mimics clinical treatment protocols, we found that loss of genes in the Fanconi anemia (FA) and homologous recombination (HR) DNA repair pathways markedly increased the killing of p53-deficient HGSOC cells following treatment with cisplatin or oxaliplatin. In contrast, knockdown of genes in the nucleotide excision repair (NER) pathway—despite its known role in repairing platinum-induced intrastrand crosslinks, particularly in G1—had a significantly lesser effect. These results suggest that platinum-induced cytotoxicity in HGSOC is primarily driven by replication stress, which is exacerbated by disruption of error-free DNA repair pathways active during the S- and G2 phases of the cell cycle. Our screens also identified several known and novel RBPs as potent sensitizers to both platinum agents and taxanes. Depletion of these RBPs resulted in substantial activation of interferon signaling, a hallmark of innate immune activation and cellular stress responses. Among them, the RNA-editing enzyme ADAR1 emerged as a key modulator of drug resistance. ADAR1 catalyzes adenosine-to-inosine RNA editing, thereby altering mRNA coding potential, secondary structure, and subcellular localization. Additionally, ADAR1 has been implicated in regulating the integrated stress response (ISR), a signaling network that enables cellular adaptation to environmental stress. As part of this stress adaptation, we observed that platinum and taxol treatment caused a translational shift from canonical cap-dependent translation mediated by eIF4G1 to a cap-independent mechanism reliant on eIF4G2, supporting selective translation of stress-responsive mRNAs. Together, our findings suggest that ADAR1-dependent suppression of interferon signaling and activation of the ISR contributes to resistance to both platinum-based chemotherapy and microtubule-targeting agents, which together comprise the current standard of care for HGSOC.



 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912428b4f14100051b63d5819642408cb6578a72" target='_blank'>
              Abstract PR009: CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This study aimed to identify novel prognostic biomarkers and therapeutic strategies by focusing on drug resistance-related phenotypes. We integrated multi-omics data from various databases encompassing sequencing, clinical information, resistance-related gene sets, and genomic alteration data. Single-sample gene set enrichment analysis (ssGSEA) was employed to calculate resistance scores for individual samples, which were subsequently applied in survival analysis. Furthermore, we utilized machine learning algorithms to develop a robust prognostic model validated across multiple independent datasets. Our findings revealed 12 genes consistently linked to PAAD prognosis in diverse datasets. Pathway enrichment analysis indicated that the high-risk group was enriched in pathways associated with systemic lupus erythematosus and the cell cycle, while the low-risk group showed significant enrichment in neuroactive ligand-receptor interaction pathways. Additionally, immune cell infiltration analysis exhibited substantial differences between risk groups, with the high-risk cohort presenting lower levels of activated CD8 + T cells but higher levels of regulatory T cells. The random survival forest (RSF) model demonstrated superior predictive performance, achieving a concordance index of 0.634 and time-dependent ROC AUCs of 0.973, 0.978, and 0.996 at 1, 2, and 3 years, respectively. In conclusion, this study identifies 12 critical drug resistance genes in PAAD and highlights the associated immune differences in patient risk, paving the way for targeted immunotherapy research to improve therapeutic strategies against this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9036a9dfcaada4e83527a56e65b2bda1ef503c41" target='_blank'>
              To Develop Prognostic Markers Related to Drug Resistance in Pancreatic Cancer Patients Based on Multiple Machine Learning Methods
              </a>
            </td>
          <td>
            Song Xu, Chuanmin Deng, Z. Man, Song Yang, Ming Xu
          </td>
          <td>2025-10-10</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a devastating hematological malignancy and one of the most prevalent forms of leukemia in adults. Despite recent advancements and approval of novel targeted therapies, drug resistance remains a formidable clinical challenge. In this study, we conducted an unbiased CRISPR‐Cas9 knockout screen in AML cells to uncover novel mediators of resistance to the clinically approved FLT3 inhibitor gilteritinib. This screen identified chromodomain helicase DNA binding protein 7 (CHD7) as a new regulator of drug resistance. Strikingly, CHD7 loss not only conferred resistance to FLT3 inhibitors but also extended resistance to a broad range of therapeutics, including venetoclax and daunorubicin (DNR). Mechanistic investigations integrating transcriptomic and proteomic data revealed that CHD7 deletion upregulates angiopoietin‐1 (ANGPT1), which drives resistance by activating the PI3K/AKT and MAPK/ERK signaling pathways. Significantly, genetic knockdown of ANGPT1 or pharmacological inhibition of its receptor TIE2 partially restored drug sensitivity in CHD7‐deficient cells. Together, these findings identify the CHD7‐ANGPT1 axis as a novel mechanism of multi‐drug resistance in AML. Preclinical studies further suggest that combining targeted therapies with TIE2 inhibitors offers a promising strategy to overcome drug resistance in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e0200e4b4ccdb724bf00ba727aaf4d318dc17ab" target='_blank'>
              Integration of CRISPR‐Cas9 screens and multi‐omics profiling reveals chromodomain helicase DNA binding protein 7‐angiopoietin‐1 as a novel multidrug resistance axis in acute myeloid leukemia
              </a>
            </td>
          <td>
            Dandan Yang, Tingting Zhang, Yang Yang, Dong Zhang, Zhiwei Zhou, Ning Wang, Jinting Fan, Chenghua Xu, Youyang Fang, Lichun Kang, Mingming Niu, Chunliang Li, Long Shen, Hong Wang
          </td>
          <td>2025-09-01</td>
          <td>Interdisciplinary Medicine</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="


 Alterations in chromatin-associated proteins contribute to the rewiring of transcriptional networks, promoting lymphomagenesis, disease progression, and resistance to therapies. Thus, epigenetic modulators offer a promising therapeutic avenue. Marginal zone lymphoma (MZL) is characterized by methylation changes, recurrent mutations in genes encoding chromatin remodelers and transcription regulators. Epigenetic reprogramming can sustain resistance to targeted agents. To identify novel therapeutic targets in MZL, we took advantage of the Structural Genomics Consortium (SGC) collection of chemical probes that inhibit or antagonize proteins involved in epigenetic signaling (www.thesgc.org). We exposed MZL cell lines, including models of secondary resistance to BTK, BCL2 and PI3K inhibitors, to 46 epigenetic compounds, spanning molecules targeting bromodomain (BRD), methyltransferases (MT), lysine demethylases (KDM) and histone acetyltransferases (HAT).



 Cells were exposed to the SGC library compounds. Cell viability was assessed after 72h. Compounds giving >40% reduction in cell proliferation versus control were considered active. Selected hits were validated in full dose-response assays.



 Two MZL cell lines (VL51, Karpas1718) and four derivatives with secondary resistance to BTK, PI3K, and BCL2 inhibitors (VL51-Ide, VL51-Copa, VL51-Ibru, Karpas1718-Ibru) were exposed to the SGC library compounds at 100 nM and 5 μM. Resistant derivatives were generally more sensitive to various epigenetic compounds than their parental cells. Only the BET inhibitor (i) JQ1 and the DOT1L-i SGC 0946 showed activity in the parental cells (both 100 nM and 5 μM, both models). Conversely, six compounds were active in resistant models at 100 nM: JQ1 and SGC 0946 (4/4), the PRMT5-i LLY 283 (2/4), and the BRPF2/TAF1-i BAY 299 (1/4). At 5 μM, 13 compounds were active: JQ1 and SGC 0946 in 4/4, 8 compounds in at least 2 models (targeting PRMT5, BRPF2/TAF1, G9a/GLP, JMJD3/UTX, BRD9, p300/CBP, EZH2) and 3 compounds in at least 1 model (PRMT7, p300/CBP, PRMT1/3/4/6/8). We then performed validation MTT experiments for 8 compounds (BAY 299, BI 9564, GSK 2801, I-BRD9, LLY 238, SGC 0946, SGC 3027, UNC 0642) using 6 concentrations of each molecule in all models. Higher activity in resistant derivatives was largely confirmed for the Karpas1718 model, and not in VL51, in which similar activity was seen in parental and resistant cells. The most active compounds were BAY 299 (BRPF2/TAF1; median IC50 6 nM), SGC 3027 (PRMT7; 229 nM) and SGC 0946 (DOT1L, 380 nM), exhibiting dose-dependent cytotoxicity across parental and resistant MZL models, followed by the others, all with median IC50s lower than or equal to 2 μM.



 Our findings reveal novel potential epigenetic vulnerabilities (PRMT7, DOT1L, and BRPF2/TAF1) in lymphoma models, also with acquired resistance to targeted therapies, besides confirming PRMT5 and BET bromodomains as valid targets. Compounds targeting these proteins should be further explored as single agents and in combinations.



 Alberto Jesus. Arribas, Luca Licenziato, Eleonora Cannas, Luca Aresu, Afua Adjeiwaa. Mensah, Francesco Bertoni. Epigenetic vulnerabilities in marginal zone lymphoma [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbf4328ab2c7b5c9c31fe8bd744d96696423fe" target='_blank'>
              Abstract C027: Epigenetic vulnerabilities in marginal zone lymphoma
              </a>
            </td>
          <td>
            A. Arribas, L. Licenziato, Eleonora Cannas, Luca Aresu, A. Mensah, Francesco Bertoni
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Located on chromosome 10, the RET gene encodes a transmembrane receptor tyrosine kinase that, when activated by a ligand, is essential for cell differentiation, proliferation, and survival. RET specifically supports the development of neuronal tissues and neural crest-derived cell types. However, RET is a proto-oncogene and mutations can lead to aggressive cancers like thyroid and lung tumours, or endocrine syndromes. Treatments for RET-driven cancers remain limited, relying on therapies that use RET-specific inhibitors. However, they show limited success in clinical settings due to adverse side effects and acquired resistance over time. To find alternative treatments, integrating the principles of synthetic dosage lethality (a genetic interaction causing a lethal effect in cancer cells) with machine learning may identify novel gene targets to complement existing therapies. In previous studies, we sought candidate genes that induce cell death when depleted in cells overexpressing oncogenic RET, but not in normal cells. A previously identified candidate, NCAM1, which is involved in neuronal cell migration, has been found to contribute to the differentiation of neural cell types. Using the SH-SY5Y neuroblastoma cell line as a model, multiple techniques and assays were performed to validate the synthetic lethal relationship between NCAM1 and RET. NCAM1 knockdown cell lines were generated using lentiviral transduction, and a live/dead cell assay was used to assess potential cytotoxic effects of gene knockdown. Additionally, a cycloheximide assay was performed to examine RET half-life by tracking its degradation over twelve hours. Over the summer, the project enabled us to reassess gene priority rankings using synthetic dosage lethal logic, characterize phenotypes of NCAM1 knockdown SH-SY5Y cells, optimize various assays under in-vitro conditions, and consolidate our understanding of NCAM1’s relationship to RET. The project thus demonstrates the value of combining bioinformatic and in-lab techniques as a productive approach to finding precise therapeutic targets against RET-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08538d4a42ece74adec0afb573a8e0619996891d" target='_blank'>
              Identifying Novel Gene Targets in RET-Driven Cancers Using Synthetic Dosage Lethality Models
              </a>
            </td>
          <td>
            Zahra Simpson-Stairs
          </td>
          <td>2025-08-29</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Fleur Chapus, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, Ming-Ming Xiao, Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            Jianyou Liao, Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematologic malignancy originating from hematopoietic stem cells and driven by the BCR-ABL fusion oncogene. Imatinib (IM), a tyrosine kinase inhibitor, is commonly used as a frontline therapy for CML. However, some patients exhibit primary resistance or show persistent molecular evidence of disease despite treatment. Emerging studies indicate that exosome-derived microRNAs (miRNAs) play a role in mediating drug resistance and may serve as promising biomarkers for cancer diagnosis and predicting therapeutic response. This study aimed to investigate the plasma exosomal miRNA expression profiles in CML patients to identify potential biomarkers associated with IM resistance. Exosomes were isolated from plasma samples of both IM-sensitive and IM-resistant CML patients. The exosomal miRNA content was analyzed using RNA sequencing, followed by differential expression analysis, which revealed 13 upregulated and 21 downregulated miRNAs in IM-resistant patients. Subsequent bioinformatics analysis indicated significant enrichment in pathways related to autophagy and PI3K-Akt signaling. Notably, miR-451a and miR-16–2-3p were among the most significantly upregulated miRNAs in exosomes from IM-resistant individuals. Interestingly, miR-16–2-3p expression showed a strong inverse correlation with clinical laboratory results, specifically blood urea nitrogen and creatinine levels. This pilot study identified plasma exosomal miRNAs, particularly miR-451a and miR-16–2-3p, as potential biomarkers for imatinib resistance in chronic myeloid leukemia. Target gene prediction was performed to explore their regulatory roles. Despite the limited sample size, these findings enhance our understanding of drug resistance mechanisms and warrant further validation in larger cohorts to assess their clinical relevance and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa66d21071c796372f54b926f9da678ac08d63" target='_blank'>
              Exosomal miRNA expression profiling in patients with imatinib resistant Chronic myeloid leukemia: A pilot study
              </a>
            </td>
          <td>
            R. Navakanitworakul, Pirun Saelue, Tipparat Penglong, P. Molika, Natakorn Nokchan, Natta Tansila, H. Buncherd, Supinya Thanapongpichat, Kanitta Srinoun
          </td>
          <td>2025-08-29</td>
          <td>PLOS One</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd59fd638b49487acab5f65a27e1f99a668332d" target='_blank'>
              Mechanistic insights into MET exon 14 skipping mutations and their role in tumor progression.
              </a>
            </td>
          <td>
            Promita Ghosh, Isabella Pecora, Morag Park
          </td>
          <td>2025-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease characterized by a broad spectrum of molecular alterations that influence clinical outcomes. TP53 mutations define one of the most lethal subtypes of acute myeloid leukemia (AML), driving resistance to nearly all available treatment modalities, including venetoclax plus azacitidine (VenAza). Yet, the molecular basis of this resistance, beyond affecting transactivation of BCL-2 family genes, has remained elusive. Here, we demonstrate that VenAza treatment leads to reduced transcriptional upregulation of the p53 signaling pathway in TP53 mutant/deficient AML compared to wild-type AML. Functionally, TP53 mutant/deficient AML exhibits selective failure in apoptosis induction rather than impaired G1 arrest or senescence. Despite inhibition of pro-apoptotic BAX and selective enrichment for MCL-1 in TP53 mutant isogenic AML cells, compensatory upregulation of BIM preserved functional mitochondrial outer membrane permeabilization (MOMP). TP53 mutant primary AML tumors at baseline also had retained capacity for MOMP. Instead, TP53 mutant AML exhibited disruption in caspase-3/7 activation to evade apoptosis after VenAza therapy – decoupling the mitochondrial and executioner phases of apoptosis. Importantly, this “post-MOMP brake” is not a bystander effect but itself a driver of VenAza and chemotherapy resistance in TP53 mutant/deficient AML. This previously unrecognized mechanistic insight shifts the focus from mitochondrial priming to terminal caspase blockade in TP53 mutant AML and opens the door for urgently needed therapeutic strategies that reignite apoptosis at its execution point.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56075976a6297045972be1b8a77a5e2796abdc68" target='_blank'>
              TP53 mutations drive therapy resistance via post-mitochondrial caspase blockade
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, E. Olesinski, Fang Qi Lim, S. Jasdanwala, Yang Mi, Nicole Si, En Lin, Daniel Tan, En Liang, Nishtha Chitkara, L. Hogdal, R. C. Lindsley, A. Letai, Koji Itahana, Shruti Bhatt
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c454f3d7e351cc169def09b1bb6a18830063a225" target='_blank'>
              PAN-CANCER ANALYSES IDENTIFY ONCOGENIC DRIVERS, EXPRESSION SIGNATURES, AND THERAPEUTIC VULNERABILITIES IN RHO GTPase PATHWAY GENES
              </a>
            </td>
          <td>
            Rubén Fernández, L. F. Lorenzo-Martín, Víctor Quesada, Xosé R. Bustelo
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis. Significance In this study we uncovered a novel regulatory link between ADAR1 and the METTL-family of RNA methyltransferases in hematopoietic stem cells. Overexpression of ADAR1 uniquely upregulated METTL13 while suppressing other METTL genes. Loss of orphan gene METTL13 affected proliferation, apoptosis and p53 signaling in hematopoietic stem cells. Furthermore, loss of METTL13 suppressed cell proliferation and survival in pediatric T-cell acute lymphoblastic leukemia. Our findings suggest a potential role for METTL13 in pre-leukemia transformation and oncogenic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transcriptional regulatory elements (TREs) orchestrate gene expression programs fundamental to cellular identity and transitions between physiological and pathological states. Decoding the regulatory logic of human biology requires resolving where, when, and how these elements are transcriptionally engaged. Here, we profiled the active transcriptional regulatory landscape across all major organ systems and a broad spectrum of developmental and disease states using PRO-cap, a high-resolution method that captures nascent transcription start sites with unprecedented sensitivity and specificity. This atlas of active TREs highlights elements shaped by their cellular contexts and evolutionary constraints, sheds light on the genetic architecture of human traits and diseases, and reveals how patterns of transcription initiation and pausing encode regulatory logic. In cancer, nascent transcription enables the delineation of lineage-specific regulatory states, metastatic adaptations, and the co-option of pre-existing programs. Together, these findings establish nascent transcription as a core dimension of gene regulation, illuminating principles that govern development, physiology, and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa390b89a37848a8f83c759acd4c4c93e92f26e" target='_blank'>
              The regulatory landscape of nascent transcription in human health and disease
              </a>
            </td>
          <td>
            Sagar R. Shah, You Chen, A. K. Leung, Priscila V. Castilho Navarro, Mauricio I. Paramo, Juhi Gupta, Aishwarya Gurumurthy, Rebuma Firdessa Fite, Annika K. Weimer, Qian Du, Ahmed M. Mohyeldin, Dieter Egli, Remi J. Creusot, Russell J H Ryan, Michael P. Snyder, Andrew G. Clark, John T. Lis, Haiyuan Yu
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cisplatin resistance presents a significant challenge in cancer therapy, emphasizing the necessity for identifying new regulatory elements that influence drug response. Recent research has revealed the importance of long noncoding RNAs (lncRNAs) in chemotherapy resistance, with LINC02381 identified as a potential regulatory factor. 
Through an in-depth bioinformatics analysis, we investigated the impact of LINC02381 on cisplatin resistance in ovarian cancer across various datasets. By conducting differential expression analysis, survival analysis, gene set enrichment analysis (GSEA), and constructing protein-protein interaction (PPI) networks, we identified key pathways associated with LINC02381 expression. 
The results indicated that the altered expression of LINC02381 in patients treated with cisplatin was associated with reduced survival. Functional studies and correlation analyses further demonstrated that this LncRNA influences critical pathways and genes related to apoptosis, efflux, DNA repair, and EMT. Lastly, through an examination of its interactions with microRNA and protein networks, we identified LINC02381 as a ceRNA implicated in cisplatin resistance. 
Our findings suggest that LINC02381 may influence cisplatin sensitivity in ovarian cancer and establish a basis for further experimental validation, including molecular assays or in vivo analyses, and suggest the potential therapeutic targeting of LINC02381 to combat chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a8ae366b9c19996905f6b9e1ed9f603948832f" target='_blank'>
              The Role of LINC02381 in Modulating Cisplatin Resistance in Ovarian Cancer: A Bioinformatics Approach
              </a>
            </td>
          <td>
            Zeinab Karbalaei Pazoki, B. M. Soltani, Mostafa Hosseini, Shiva Irani
          </td>
          <td>2025-09-26</td>
          <td>Iranian Journal of Allergy, Asthma and Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, Alexa J. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, Mukulika Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is characterized by the abnormal proliferation of B-lineage lymphocytes in the bone marrow (BM). The roles of immune cells within the BM microenvironment remain incompletely understood. Single-cell RNA sequencing (scRNA-seq) provides the potential for groundbreaking insights into the pathogenesis of B-ALL. In this study, scRNA-seq was conducted on BM samples from 17 B-ALL patients (B-ALL cohorts) and 13 healthy controls (HCs). Bioinformatics analyses, including clustering, differential expression, pathway analysis, and gene set variation analysis, systematically identified immune cell types and assessed T-cell prognostic and metabolic heterogeneity. A metabolic-feature-based machine learning model was developed for B-ALL subtyping. Furthermore, T-cell–monocyte interactions, transcription factor (TF) activity, and drug enrichment analyses were performed to identify therapeutic targets. The results indicated significant increases in Pro-B cells, alongside decreases in B cells, NK cells, monocytes, and plasmacytoid dendritic cells (pDCs) among B-ALL patients, suggesting immune dysfunction. Clinical prognosis correlated significantly with the distribution of T-cell subsets. Metabolic heterogeneity categorized patients into four distinct groups (A–D), all exhibiting enhanced major histocompatibility class I (MHC-I)-mediated intercellular communication. The metabolic-based machine learning model achieved precise classification of B-ALL groups. Analysis of TF activity underscored the critical roles of MYC, STAT3, and TCF7 within the B-ALL immunometabolic network. Drug targeting studies revealed that dorlimomab aritox and palbociclib specifically target dysregulation in ribosomal and CDK4/6 pathways, offering novel therapeutic avenues. This study elucidates immunometabolic dysregulation in B-ALL, characterized by altered cellular composition, metabolic disturbances, and abnormal cellular interactions. Key TFs were identified, and targeted drug profiles were established, demonstrating the significant clinical potential of integrating immunological mechanisms with metabolic regulation for the treatment of B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fd3e9884a415a1e1d17726c2c024bcc416a45d" target='_blank'>
              Immunometabolic Dysregulation in B-Cell Acute Lymphoblastic Leukemia Revealed by Single-Cell RNA Sequencing: Perspectives on Subtypes and Potential Therapeutic Targets
              </a>
            </td>
          <td>
            Dingya Sun, Dun Hu, Jialu Wang, Jun Peng, Shan Wang
          </td>
          <td>2025-10-14</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1980148077174708f4b58d5435d5f4b71871118c" target='_blank'>
              Determination of Copy Number Variations and Affected Gene Networks in Breast Cancer from Mexican Patients
              </a>
            </td>
          <td>
            Voleta Larios-Serrato, Hilda A. Valdez-Salazar, Javier Torres, M. Camorlinga-Ponce, P. Piña-Sánchez, Héctor Mayani, M. Ruiz-Tachiquín
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glutamine-dependence of cancer cells reduces local glutamine availability, which hinders anti-tumor T-cell functionality and facilitates immune evasion. We thus speculated that glutamine deprivation might be limiting efficacy of CAR T-cell therapies in cancer patients. We have seen that antigen-specific T cells are unable to proliferate or produce IFN-γ in response to antigen stimulation when glutamine concentration is limited. Using multiple myeloma (MM) as a glutamine-dependent disease model, we found that murine CAR-T cells selectively targeting BCMA in MM cells were sensitive to glutamine deprivation. However, CAR-T cells engineered to increase glutamine uptake by expression of the glutamine transporter Asct2 exhibited enhanced proliferation and responsiveness to antigen stimulation, increased production of IFN-γ, and heightened cytotoxic activity, even under conditions of low glutamine concentration. Mechanistically, Asct2 overexpression reprogrammed CART cell metabolic fitness of CART cells by upregulating the mTORC1 gene signature, modifying the Solute Carrier transporter (SLC) repertoire, and improving both basal oxygen consumption rate and glycolytic function thereby enhancing CART cell persistence in vivo. Accordingly, expression of Asct2 increased the efficacy of BCMA-CART cells in syngeneic and genetically-engineered mouse models of MM, which prolonged mouse survival. In patients, reduced expression of Asct2 by MM cells predicted poor outcome to combined immunotherapy and BCMA-CAR T-cell therapy. Our results indicate that reprogramming glutamine metabolism may enhance anti-tumor CAR T-cell functionality in MM. This approach may also be effective for other cancers that depend on glutamine as a key energy source and metabolic hallmark.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/538f82c530b0189b52d7cd8d3af17c6f114cf936" target='_blank'>
              Reprogramming glutamine metabolism enhances BCMA-CART cell fitness and therapeutic efficacy in multiple myeloma.
              </a>
            </td>
          <td>
            Flor Navarro, Teresa Lozano, Andrea Fuentes-García, Inés Sánchez-Moreno, M. Larrayoz, Pedro Justicia, Beatriz Perucha, Maialen Martinez-Tabar, Rebeca Martinez-Turrillas, N. Casares, Celia Martín-Otal, M. Gorraiz, Erin W. Meermeier, M. Chesi, Douglas Lake, P. L. Bergsagel, Eva Santamaría, M. E. Calleja-Cervantes, Patxi San Martín-Úriz, Lorea Jordana-Urriza, X. Agirre, S. Hervás-Stubbs, J. R. Rodríguez-Madoz, Jose-Al Martinez-Climent, F. Prósper, J.J. Lasarte
          </td>
          <td>2025-09-19</td>
          <td>Blood</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55be761462c41a8fdd2d1a07cb3c86165b3561a4" target='_blank'>
              Systematic identification of genomic non-response biomarkers to cancer therapies
              </a>
            </td>
          <td>
            Joseph Usset, J. de Ligt, Sophie Roerink, P. Roepman, E. Cuppen, Francisco Martínez-Jiménez
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="ABSTRACT: Cancer represents a multifaceted global health burden shaped by diverse genetic, epigenetic, and environmental factors. The GLOBOCAN 2022 data emphasize the growing incidence and regional variation of cancers such as breast, lung, colorectal, and oral malignancies. Underlying this burden are complex oncogenic signaling pathways—including Wnt/β-catenin, Notch, Hedgehog, PI3K/AKT/mTOR, and JAK/STAT—that regulate cell proliferation, apoptosis, and metastasis. While therapeutic interventions targeting these pathways have shown promise, durable clinical outcomes remain limited due to pathway redundancy, crosstalk, and tumor heterogeneity. This review systematically explores these signaling networks, detailing current and investigational therapeutic targets and the clinical progress of pathway-specific inhibitors. We identify critical gaps in translating molecular insights into personalized therapies and emphasize the need for integrated, multi-targeted approaches. By synthesizing recent advances in molecular oncology and emerging therapeutic strategies, this work offers a forward-looking perspective on overcoming current challenges and optimizing cancer treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dacf4df973514ee12b917472cd4d3ec9febb3957" target='_blank'>
              Targeting Oncogenic Pathways in Cancer: Therapeutic Advances, Challenges, and Future Directions
              </a>
            </td>
          <td>
            Ritika Gera, Rajesh Kumar, Kamaljit Panchal, Vikas Sharma, Ramika Garg
          </td>
          <td>2025-09-30</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a501c277b7efc44209952f58d1318feb42e211" target='_blank'>
              YAP1 drives aggressive and therapy resistant state in melanoma through reprogramming the chromatin and regulating immune evasive programs
              </a>
            </td>
          <td>
            Nana Chen, Gracia M. Bonilla, Abdullah AI Emran, Brian Lin, Haymanti Bhanot, Yang Sun, Avinash Waghray, Mateo Useche, David Liu, Genevieve M Boland, Ivana L de la Serna, V. Sancisi, Martin Sattler, Jayaraj Rajagopal, Xu Wu, Ruslan Sadreyev, David E Fisher, S. Saladi
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Multidrug resistance is a significant barrier in cancer therapy largely due to poorly understood regulatory mechanisms. Here we reveal that certain anticancer drugs can bind to newly synthesized proteins prior to reaching their canonical targets, resulting in various forms of protein damage. This binding disrupts protein functions, particularly those of mitochondrial proteins, resulting in substantial cytotoxicity. The protein damage is further exacerbated by mitochondrial reactive oxygen species generated as a consequence of the initial damage, creating a positive feedback loop. In response, cancer cells rapidly initiate a chain of events, which we term the Protein Damage Response (PDR). This includes damage recognition primarily mediated by protein ubiquitination and subsequent damage clearance via the proteasome system. Notably, patients with advanced, drug-resistant metastatic breast or colon cancers exhibit elevated proteasome activity. In an effort to predict drug resistance, we developed a sensitive kit for detecting proteasome levels, enabling the identification and subtyping of patients with high proteasome activity to support tailored therapeutic strategies. Using a three-dimensional tumor slice culture-based drug sensitivity assay and an investigator-initiated clinical trial, we demonstrate that three clinically approved proteasome inhibitors effectively overcome multidrug resistance in colon and breast cancer patients with elevated proteasome activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1785188ad7f7984f485f66a5779a54c95bdf57ed" target='_blank'>
              Suppressing protein damage response to overcome multidrug resistance in cancer therapy
              </a>
            </td>
          <td>
            Fangyuan Shao, Zongjie Li, Hao Xiao, Yujun Chen, Yuheng Zhang, Ling Li, Yuzhong Peng, Xinyi Li, Yuxing Hou, Bo Li, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Xi Han, Jiaxin Yao, Cuiting Zhang, Lijian Wang, Haifeng Li, N. Shao, Kai Miao, Xiaoling Xu, Yanxia Shi, Changhua Zhang, Jun Yan, Ying Lin, Chu-Xia Deng
          </td>
          <td>2025-09-30</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is among the deadliest cancers, in part because we do not sufficiently understand its genetic basis to design effective therapies. Although KRAS mutations are present in 96% of PDACs, only 60% have concurrent TP53 mutations, leaving 40% with wild-type TP53 and undefined drivers. Identifying additional mutations is challenging due to the long tail of low-frequency events in PDAC (<10%), yet these rare mutations could reveal key drivers of tumorigenesis and therapeutic vulnerabilities. To identify novel driver mutations beyond TP53, we performed a genetic interaction analysis based on mutual exclusivity, which suggests that mutations exclusive to one another function within the same pathway. Using unbiased genetic interaction analyses, we queried >37,000 PDAC cases to prioritize rare mutations mutually exclusive with TP53 mutations. Mutations in the splicing factors SF3B1 and RBM10 (∼15% combined) emerged as the most significant and mutually exclusive with mutant TP53. Thus, we hypothesized that mutations K700E in SF3B1 and truncating mutations in RBM10 drives tumorigenesis and therapy resistance. We established the first autochthonous mouse models co-expressing mutant Kras with either Sf3b1K700E or Rbm10 loss, revealing these splicing alterations can cause PDAC. To uncover how splicing-factor mutations drive PDAC, we performed deep RNA sequencing and eCLIP in lineage-traced tumor cells from mouse models with wild-type or mutant Sf3b1/Rbm10. While tumors with these mutations lack chromosomal deletions, they exhibit widespread splicing defects, particularly in exon selection. Thousands of mRNAs transcribed from chromosomes 2, 7, 11, and 17 are mis-spliced and degraded by nonsense-mediated decay (NMD), leading to post-transcriptional gene loss. These patterns are conserved in human PDAC models. Notably, TP53-mutant PDACs require chromosomal deletions of chr11 and chr7 for tumor development, suggesting that SF3B1 and RBM10 mutations mimic this effect through splicing-dependent mechanisms. To define precision therapies for splicing-mutant tumors, we found that PDACs with these mutations are up to 5000X more sensitive to Gemcitabine versus 5FU. We combined Gemcitabine with the splicing modulator H3B-8800, which synergistically decreased tumor growth and extended survival of xenograft mice. Together, our findings reveal that diverse genetic events—TP53 mutations, SF3B1 mutations, or RBM10 loss—collaborate with mutant KRAS to by converging on a shared tumorigenic mechanism: loss of gene expression from specific chromosomes. In the case of inactivating mutations in TP53, this decrease in gene expression is caused by chromosome deletions, a genetic mechanism. In contrast, mutations in SF3B1 and RBM10 result in decreased gene expression through alternative splicing and mRNA decay, a post-transcriptional mechanism. This work uncovers a novel genetic convergence in PDAC and suggests that targeting splicing-mediated vulnerabilities may offer therapeutic opportunities across molecular subtypes.



 Natasha Pinto Medici, Diana Martinez-Saucedo, Tianyi Chu, Saúl Rojas-Sánchez, Daniel Lee, Robert Tseng, Vincent Cannataro, Julia C.F. Quintanilha, Ryon P. Graf, Gilbert Giri, Sheila Oliveira, Amelia Lower, Isabela Fuentes, Mariana do Carmo, Lyanne Delgado-Coka, Lucia Roa-Pena, Sumedha Chowdhury, Muaz Furaque, Emma Delannoy, Nicolas Lecomte, Christine Iacobuzio-Donahue, John P. Morris IV, Katerina Politi, Pablo Perez-Piñera, Marie E. Robert, Kenneth Shroyer, Scott Lowe, Michael Cecchini, Omar Abdel-Wahab, Steven D. Leach, Jeffrey P. Townsend, Daniel Dominguez, Mathieu Quesnel-Vallières, Luisa Escobar-Hoyos. Altered RNA splicing drives pancreatic cancer by mimicking chromosome deletions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0561d01de151e0fbb07ad1104d32fbef86079b" target='_blank'>
              Abstract A004: Altered RNA splicing drives pancreatic cancer by mimicking chromosome deletions
              </a>
            </td>
          <td>
            Natasha Pinto Medici, Diana Martinez-Saucedo, Tianyi Chu, Saúl Rojas-Sánchez, Danny Lee, Robert Tseng, Vincent L. Cannataro, J. Quintanilha, Ryon P. Graf, Gilbert Giri, Sheila Oliveira, Amelia Lower, Isabela Fuentes, Mariana do Carmo, Lyanne A Delgado-Coka, Lucia Roa-Peña, Sumedha Chowdhury, Muaz Furaque, Emma Delannoy, Nicolas Lecomte, Christine A. Iacobuzio-Donahue, John P Morris, Katerina Politi, Pablo Perez-Piñera, Marie E. Robert, Kenneth R. Shroyer, Scott W. Lowe, M. Cecchini, Omar Abdel-Wahab, Steven D. Leach, Jeffrey P. Townsend, Daniel Dominguez, Mathieu Quesnel-Vallières, Luisa F. Escobar-Hoyos
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND AND PURPOSE
In order to significantly improve the therapeutic treatment of breast cancer, the exploration of underlying genetic and molecular differences is absolutely necessary. Here, our goal was to integrate mutational status of entire pathways to reveal molecular pathway interactions determining survival.


EXPERIMENTAL APPROACH
A comprehensive analysis of breast cancer mutations was conducted by integrating data from three distinct databases with a total of 4586 samples encompassing over 25,000 genes. For each gene, we filtered mutations that disruptively affect the protein structure. Cox proportional hazard regression was employed to link altered pathways to outcome. We also identified the co-occurring and mutually exclusive disruptive mutations.


KEY RESULTS
We identified 17 genes, the mutation status of which alone seriously affects relapse-free survival. The three most significant genes were TP53 (HR: 2.04, p: 4.65 × 10-33), CARD11 (HR: 2.59, p: 1.54 × 10-5) and PIK3R1 (HR: 2.27, p: 3.66 × 10-5). The five most significant biological processes and KEGG pathways affecting relapse-free survival include negative regulation of cell population proliferation, positive regulation of DNA-templated transcription, protein stabilisation, and MicroRNAs in cancer, hepatocellular carcinoma, and breast cancer. Co-mutation and mutual exclusivity analysis identified significant enrichment in 241 gene pairs. Finally, we also established an online platform to enable future analysis of the established cohort for any selected pathway.


CONCLUSIONS AND IMPLICATIONS
We assembled a comprehensive database of breast cancer samples and used this cohort to identify cancer-specific disruptive mutation signatures linked to altered survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919db15729dadcc0b973aea2e0ec780f64f60ace" target='_blank'>
              Pathway-level mutational signatures predict breast cancer outcomes and reveal therapeutic targets.
              </a>
            </td>
          <td>
            Máté Posta, Balázs Győrffy
          </td>
          <td>2025-10-07</td>
          <td>British journal of pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Anthracycline-induced cardiomyopathy is a leading cause of morbidity and mortality in survivors of childhood cancer. The mitochondrion is a key mediator of the cytotoxic effects of anthracycline treatment and mitochondrial dysfunction is a hallmark of cardiomyopathy and heart failure. We sought to evaluate whether mitochondrial processes differ between anthracycline-exposed childhood cancer survivors who developed cardiomyopathy versus those who did not. Methods Peripheral blood was collected from 40 childhood cancer survivors who developed cardiomyopathy (cases) and 64 matched survivors who did not (controls). From these samples, gene expression was determined by RNA-Sequencing. Following bioinformatic processing, differential gene expression at the mRNA-level between cases and controls was determined. Human MitoCarta3.0, was utilized to determine if genes involved in mitochondrial processes were enriched for differential expression, and to identify differentially regulated mitochondrial pathways at the mRNA-level. Results 900 genes were identified as differentially expressed at the mRNA-level. The odds of a gene being differentially expressed were 2.43 times greater if it encodes for a protein that localizes to the mitochondria. Mitochondrial processes that were enriched for differentially expressed genes at the mRNA-level included electron transport chain complexes; reactive oxygen species metabolism; apoptosis, mitophagy, and autophagy; mitochondrial ribosome; mitochondrial transport and chaperones; and heme synthesis and processing. Additionally, we observed that a measure of pro-apoptotic balance (BAX to BCL-2 gene expression at the mRNA-level) was highest in severe cardiomyopathy, intermediate in mild cardiomyopathy, and lowest in survivors without cardiomyopathy. Conclusions We observed substantial evidence that the expression of genes involved in mitochondrial processes differs in childhood cancer survivors who develop cardiomyopathy versus those who do not. Supplementary Information The online version contains supplementary material available at 10.1186/s40959-025-00391-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/158b58f0807a42f8480a4748ec952f658ac5aa51" target='_blank'>
              Anthracycline-induced cardiomyopathy in childhood cancer survivors is associated with gene signatures of mitochondrial dysfunction—a COG ALTE03N1 report
              </a>
            </td>
          <td>
            P. Trainor, Purnima Singh, Xuexia Wang, Noha Sharafeldin, Liting Zhou, L. Hageman, S. Armenian, JillP Ginsberg, Douglas S. Hawkins, Frank G. Keller, M. Hudson, Joseph P. Neglia, W. Landier, Smita Bhatia
          </td>
          <td>2025-10-06</td>
          <td>Cardio-oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d108c58970b821d1c2170e04d8c4196841969e5f" target='_blank'>
              Phage therapy and its role in cancer treatment and control.
              </a>
            </td>
          <td>
            H. A. Abd El-Moaty, A. Doghish, Hebatallah Ahmed Mohamed Moustafa, Wagiha S. Elkalla, Ghadir A Sayed, Nourhan H Elshami, S. S. A. Mageed, Osama A Mohammed, Mahmoud A. Elrebehy, Kareem AlFarsi, Amr M. Abdelfatah, Aly M Waseem, Ahmed O Abdulkader, Rabab S. Hamad, Akram N Salah
          </td>
          <td>2025-09-17</td>
          <td>Folia microbiologica</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233ca1d62c11343dd5ca5fc5bc8ce7a97d29e8c" target='_blank'>
              Phylogenetic tree inference from single-cell RNA sequencing data
              </a>
            </td>
          <td>
            Norio Zimmermann, Xiaoyu Sun, Joanna Hård, Jack Kuipers, N. Beerenwinkel
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Modeling cancer recurrence and associated conditions such as cancer cell dormancy remains a significant challenge in cancer research. We developed a novel in vitro model based on platinum-resistant tumor cells treated with a platinum drug, in which cells progress through all stages of recurrence within 30–40 days post-exposure. Our results demonstrate that treatment of platinum-resistant colorectal cancer cells with oxaliplatin enriches the population with quiescent, dormant cells arrested in the G0/G1 phase. These cells exhibit increased autophagy, elevated expression of stem cell markers, reduced reactive oxygen species (ROS) levels, and heightened resistance to chemotherapy. A key advantage of our model is the high survival rate of residual cells, enabling the maintenance of a large cell population. This facilitates the application of conventional research techniques that are often limited in other recurrence models due to the small size of residual populations. Overall, this model provides opportunities for a wide range of further research for studying specific recurrence- and dormancy-associated signaling pathways and for identifying novel therapeutic targets for preventing tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4b53e10213b4d15495530a1d5bcc0ce84af24bb" target='_blank'>
              An in vitro tumor recurrence model based on platinum-resistant colon cancer cells as a research tool for studying cancer cell dormancy
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, M. Igotti
          </td>
          <td>2025-10-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite the initial responses to the tyrosine kinase inhibitor (TKI) for cancer therapy, many patients often relapse with no curative regimens available. Further, the ability to target therapeutic agents to cancer cells with appropriate doses remains challenging in the clinic, especially for leukemia. Here, we show that naïve CML cells are dynamically heterogeneous in colony formation. Larger clones expand while smaller ones diminish and eventually disappear. Compared to resistant cells, parental populations, including CD44+ stem cells, form a greater number of larger, solid spheroids. Upregulation of fat mass and obesity associated protein (FTO), an RNA N6-methyladenosine demethylase, and stem cell markers (e.g., CD44, CD133, CD25) is more obvious in resistant cells compared to parental cells. FTO inhibitors (e.g., CS1, FB23-2) appreciably impair the growth of resistant cells either alone or in combination with nilotinib. FTO protein expression is unexpectedly upregulated by CS1 or FB23-2 treatment in multiple leukemia cell lines. We then constructed RNA nanoparticles encapsulating FTO siRNAs and conjugated with anti-CD133 RNA aptamers. We showed that, compared to negative control, these nanoparticles were taken up much more efficiently by resistant cells that highly express CD133. Treatment with the CD133-guided FTO siRNA nanoparticles efficiently silenced FTO expression in resistant cells, which leads to a significant reduction in their colony and spheroid formation. These findings offer new insights into cancer drug resistance and advance the application of RNA nanotechnology for treating leukemia. The research provides a foundation for developing novel, targeted therapies for resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610e20fd12fcf52179a62018bf638a0c7e862c87" target='_blank'>
              CD133-Guided RNA Nanoparticle Delivery of FTO siRNA Impairs Leukemia Resistance to Tyrosine Kinase Inhibitor Therapy
              </a>
            </td>
          <td>
            Huiqin Bian, Changli Zhou, Hiroaki Koyama, Wen Gao, Sicheng Bian, Tao Cheng, Xiaonan Han, William Tse, Shujun Liu, Alexander Miron
          </td>
          <td>2025-10-15</td>
          <td>RNA NanoMed</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Cell identity, tightly regulated by transcription factors (TFs), is frequently disrupted in cancer, driving tumor progression and metastasis. In pancreatic ductal adenocarcinoma (PDAC), alterations in cell identity contribute to disease heterogeneity and therapy resistance. Integrated genomic, transcriptomic, and proteomic analyses have defined two major PDAC subtypes: classical and basal. Notably, recent evidence suggests that changes in subtype identity can enable resistance to targeted therapies. These subtypes are distinguished by relative levels of endodermal lineage-specifying TFs such as FOXA1/2 and HNF4α, which are maintained in the classical subtype but downregulated in the basal subtype. Understanding how these TFs shape cellular identity is critical for identifying therapeutic vulnerabilities and mechanisms of drug resistance. We have shown that HNF4⍺ promotes the classical subtype program of PDAC. Although FOXA1/2 and HNF4α regulate gene expression in normal tissues, it is still unclear whether such cooperation is necessary to drive the classical transcriptional program in PDAC. We hypothesize that FOXA1/2 and HNF4α cooperatively regulates the gene expression program that regulates the classical subtype program in PDAC. Using human cell lines and murine PDAC organoid models, we reconstituted and modulated TF expression to evaluate downstream gene expression and TF genomic binding via ChIP-seq. Conditional alleles of FoxA1/2 in our murine organoid lines reveal that loss of FOXA1/2 lead to a decrease in the classical program. When we overexpress HNF4α in FOXA1/2-null organoids, it leads to a decrease in organoid growth; therefore, illustrating the ability of HNF4α to regulate growth independent of FOXA1/2. In a human cell line, we find that co-expression of FOXA1/2 and HNF4a activates the classical B signature while FoxA2 alone activates the Classical A signature, indicating that FOXA2 and HNF4a activate subtype-specific transcriptional programs. Additionally, we demonstrate that the overexpression of HNF4a in a subcutaneous xenograft model confers resistance to KRASG12C inhibition, leading to an increase in tumor size and weight. Unexpectedly, we find that these TFs do not cooperatively regulate gene expression; rather, in the human cell line, FOXA2 interferes with HNF4α genomic binding, suggesting a competitive or inhibitory interaction. Overall, our work highlights how lineage-defining TFs govern PDAC cell identity and therapeutic response.



 Walter A. Orellana, Sydney N. Larsen, Eric L. Snyder. FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B107.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10be5668747dfa95fe8d23992fe6617771de029e" target='_blank'>
              Abstract B107: FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Walter A Orellana, Sydney N. Larsen, Eric L. Snyder
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb9b509dc6cd6165a216f7f2331a61922945369" target='_blank'>
              Pooled CRISPR screens identify genes and non-coding genomic regions that regulate red blood cell density
              </a>
            </td>
          <td>
            Nicolas Brosseau, Thomas Pincez, Ken Sin Lo, M. Beaudoin, Guillaume Lettre
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer remains the most frequently diagnosed malignancy among women worldwide, with significant morbidity and mortality rates despite advances in early detection and treatment. This review provides a comprehensive overview of recent developments in breast cancer research, spanning genetic and molecular insights, novel diagnostic techniques, and emerging therapeutic strategies. The advent of next-generation sequencing and multi-omics approaches has deepened our understanding of tumor heterogeneity, revealing key genetic drivers, epigenetic regulators, and the role of cancer stem cells in disease progression. Early detection strategies have also evolved with digital breast tomosynthesis and molecular breast imaging, offering improved sensitivity and specificity. On the therapeutic front, breakthroughs in targeted treatments—including cyclin-dependent kinase 4/6 and phosphoinositide 3-kinase inhibitors, antibody–drug conjugates, and immune checkpoint inhibitors—have transformed patient outcomes. The integration of chimeric antigen receptor T-cell therapy and mRNA-based therapeutics holds great promise in overcoming treatment resistance and improving long-term survival. However, challenges such as treatment accessibility, drug resistance, and disparities in healthcare persist, particularly in low- and middle-income regions. Emerging technologies, including artificial intelligence-driven diagnostics and risk-adapted screening, are paving the way for more precise and personalized interventions. This review highlights the latest innovations and ongoing challenges in breast cancer research, emphasizing the need for continued efforts to translate scientific advancements into clinical practice to improve patient outcomes globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c3c761954bdae167d685c04a34eda5ce49f0c7" target='_blank'>
              Clinical advancements in breast cancer research: A comprehensive review
              </a>
            </td>
          <td>
            P. Puvvula, Ria Sahasra Puvvula
          </td>
          <td>2025-09-23</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Aberrant pre-mRNA splicing is a hallmark of many cancers, yet the underlying genetic mechanisms driving these alterations remain incompletely understood. While splice-altering variants can explain a subset of events, many tumor-specific splicing changes arise in the absence of nearby sequence variants. This study aims to systematically dissect the contribution of proximal splice-altering variants to tumor-specific splicing in pediatric brain tumors and to define the residual cohort likely driven by regulatory mechanisms such as mutations affecting splicing factors or the spliceosome machinery. We hypothesized that some fraction of tumor-specific splice events in pediatric brain tumors are not associated with nearby genetic variants, and that these may highlight broader disruptions to splicing regulation.



 We applied replicate Multivariate Analysis of Transcript Splicing (rMATS-turbo) to stranded total RNA-seq data (N=635) from the Pediatric Brain Tumor Atlas (PBTA). Tumor-specific single exon (SE) splicing events were defined as a change in percent spliced in (| ΔPSI |) > 0.3 between tumors and healthy brain samples sourced from GTEx (<40 years, n=2,642) and pediatric normals (n=7). Tumor-normal matched whole genome sequencing data were used to identify somatic and rare (<0.1% Allele Frequency in gnomad) germline SNVs/Indels proximal (<250nt) to rMATS-defined splice junctions.



 Across the cohort (N=635), we identified 107,827 tumor-specific SE alternative splicing events. Only 44 tumors harbored a variant proximal to the affected splice site (11 germline, 39 somatic), leaving most events without a clear cause. We are now investigating this variant-negative subgroup as a preliminary cohort. Current efforts focus on excluding structural drivers before turning to potential regulatory mechanisms, including the disruption of spliceosome components. Preliminary findings suggest some tumors show altered expression or mutations in spliceosome components, though further validation is needed.



 This dual-ended analysis of tumor-specific splicing events reveals that while a subset can be attributed to local genetic variation, the majority likely arise from broader disruptions in splicing regulation. These findings underscore the importance of integrating RNA- and DNA-based analyses to fully capture the molecular basis of splicing dysregulation in cancer. Moreover, the variant-negative cohort may serve as a rich resource to uncover novel regulatory mechanisms and potential therapeutic targets in splicing-driven pediatric brain tumors.



 Patricia J Sullivan, Ryan J Corbett, Ammar S Naqvi, Alexander Sickler, Rebecca Kaufman, Bo Zhang, Chuwei Zhong, Sharon J Diskin, Jo Lynne Rokita. Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0c6326494c3f8ecc5b90e1fc51e9fc24433d6c" target='_blank'>
              Abstract B031: Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors
              </a>
            </td>
          <td>
            Patricia J. Sullivan, Ryan J. Corbett, Ammar S. Naqvi, A. Sickler, Rebecca S. Kaufman, Bo Zhang, Chuwei Zhong, S. Diskin, J. L. Rokita
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed2431373f3afb26d382f9fbc1e05e4679be852" target='_blank'>
              Activating NRF2E79Q mutation alters the differentiation of human non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Hamad, Hansa Joshi, T. Hess, S. Jefferys, Zena Saleh, N. Bendjilali, Rani S Sellers, G. Zhu, Travis Shrank, R. Moore, David Corcoran, Jeremy M. Simon, Francis R. Spitz, David Shersher, M. Major, Bernard Weissman
          </td>
          <td>2025-10-11</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the second leading cause of cancer deaths, with a five-year survival rate of 65%. Oxaliplatin was the first platinum drug shown to improve CRC patient outcomes and is now a common adjuvant therapy for advanced disease, yet 90% of patients develop resistance. Oxaliplatin was developed as a third-generation derivative of cisplatin, but recent evidence points to divergent modes of action. Genome-wide CRISPR activation and knockout screens were conducted to identify genetic changes that confer resistance to oxaliplatin in two CRC cell lines with distinct molecular backgrounds (SW620 and RKO). Guide RNAs corresponding to the neutral amino acid transporter SLC43A1 (LAT3) were the most significantly enriched in knockout screens and depleted in activation screens, suggesting a potential role for LAT3 in modulating oxaliplatin resistance. In vitro CRISPR knockout and overexpression of LAT3 in SW620 and RKO cell lines confirm increased resistance or sensitivity to oxaliplatin, respectively, and this correlated with levels of intracellular accumulation of oxaliplatin and DNA damage, demonstrating that enhanced LAT3-mediated uptake of oxaliplatin can exert its expected mechanism of action and induce cytotoxicity. Further studies to investigate the specificity of this mechanism of resistance to oxaliplatin in CRC and other cancers are ongoing to clarify the molecular basis of oxaliplatin resistance. These findings suggest a novel use for LAT3 as a potential biomarker and predictor of oxaliplatin sensitivity and can provide insights into the interplay between essential nutrient uptake and drug transport.



 Nisha R. Pawar, Heidi M. Wade, Allison V. Mitchell, Connor P. Jewell, Lisa M. Jenkins, Robert W. Robey, Michael M. Gottesman. A whole genomic CRISPR-Cas9 screen identifies the LAT3 (SLC43A1) neutral amino acid transporter as a major determinant of oxaliplatin resistance in colorectal cancer cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb9e1d72d03ca8da071d5c00ee84e7d75c353cf" target='_blank'>
              Abstract A039: A whole genomic CRISPR-Cas9 screen identifies the LAT3 (SLC43A1) neutral amino acid transporter as a major determinant of oxaliplatin resistance in colorectal cancer cells
              </a>
            </td>
          <td>
            Nisha R Pawar, Heidi M Wade, Allison V. Mitchell, Connor P. Jewell, Lisa M. Jenkins, Robert W. Robey, Michael M. Gottesman
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80e02b4b634cdeb2be956bc4074a559e6c262ab4" target='_blank'>
              Integrating large-scale in vitro functional genomic screen and multi-omics data to identify novel breast cancer targets.
              </a>
            </td>
          <td>
            Hao-Kuen Lin, J. Dai, Lajos Pusztai
          </td>
          <td>2025-09-01</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Disclosure: M. Jabeen: None. X. Dai: None. J. Zhu: None. P. Hergenrother: SAB Systems Oncology. D.J. Shapiro: None. Classically, the development of resistance to therapies for estrogen receptor (ER) positivebreast cancer, is thought to often involve a single mutational change in ER. In contrast,detailed time course analysis of the process by which ER positive breast cancer cellsbecome resistant to the promising small molecule anticancer drug, ErSO, shows that theacquisition of resistance is a dynamic multi-step process in which cells initially acquireresistance through relatively high frequency events that result in poor growth and cellfitness. These cells are rapidly and progressively replaced by resistant cells with other,apparently lower frequency, changes until resistant cells acquire near wild-type growthrates. We identified the ErSO compounds that work by inducing lethal hyperactivation ofthe normally tumor protective anticipatory unfolded protein response (a-UPR). In ERpositive breast cancer cells key steps in ErSO action are initial activation through anER:Src:PLC complex, resulting in calcium release from endoplasmic reticulum stores,activation of the TRPM4 sodium channel, leading to cell swelling, plasma membranerupture and necrotic cell death. In orthotopic breast and ovarian cancer mousexenografts, ErSO often induced complete regression without recurrence of large primarytumors and most metastases and was effective in PDOs and fresh patient tumor samples.In cell culture and PDOs, we generated more than 10 clonal ErSO-resistant lines.Unexpectedly the resistant clones did not arise from a large pool of non-dividing persistercells and instead derived from a few single survivor cells with initial very slow proliferation,leading to clonal outgrowth. Surprisingly, the initial resistance event in most of the cloneswas massive overexpression of progesterone receptor (PR). PR was known to bind Srctightly, enabling it to sequester Src away from the ErSO activated ER:Src:PLC complex.But overexpression of PR is toxic to cells, and resulted in very slow growing resistantcells. We found that within weeks, these slow-growing clones lost overexpression of PRand engaged other mechanisms that enabled both ErSO-resistance and more robustproliferation. From multiple cycles of RNA-seq done on more than 10 clonal lines asresistance progressed, we obtained snapshots of the evolution of resistance. We did notobserve down-regulator of the intensively studied regulator of plasma membrane rupture,NINJ1. Instead, down-regulation of the TRPM4 sodium channel, other transporters andsystems that might enhance cell survival progressively evolved. These data suggest thatthe evolution of resistance to anticancer agents may often be a dynamic multi-stepprocess rather than a single rare event and that there may be different, potentiallytargetable, vulnerabilities in cancer cells as they progressively evolve resistance toanticancer therapies. Presentation: Saturday, July 12, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1cc113504dfa0f1063d26be01aa07b7174beef" target='_blank'>
              SAT-302 Acquired Resistance to Breast Cancer Therapy is a Dynamic Multi-step Evolutionary Process
              </a>
            </td>
          <td>
            M. Jabeen, Xinyi Dai, Junyao Zhu, Paul Hergenrother, D. Shapiro
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Kyinno provides TMD8 cell lines with endogenous BTK resistance mutations (e.g., C481S, L528W, V416L, T474I, A428D) under native regulation, enabling detailed studies of covalent and non-covalent inhibitor responses. These models faithfully recapitulate clinical escape alleles, supporting both mechanistic investigations and drug-resistance profiling.



 High-fidelity CRISPR/Cas9 RNP electroporation with 200-bp ssODN donors and proprietary HDR enhancers routinely yields >80 % BTK knock-in efficiency. Following optimized limiting-dilution cloning, each clone undergoes genomic PCR, Sanger sequencing, and RT-PCR to confirm on-target integration, allelic zygosity.



 Mutant TMD8 lines display broad IC50 shifts: A428D and L528W confer >100-fold resistance to both ibrutinib and pirtobrutinib, C481S remains nanomolar to pirtobrutinib, whereas T474I/V416L stay responsive to zanubrutinib; all variants expect A428D retain low-nanomolar BGB-16673 and NX-5948 susceptibility. In vivo, C481F tumors show modest ibrutinib delay, whereas L528W xenografts resist ibrutinib yet regress under daily NX-5948 without overt weight loss. Proliferation assays mirror these drug-response patterns across the panel.



 Kyinno’s BTK-mutant lines accelerate high-throughput screening of small-molecule inhibitors and targeted degraders, delineate resistance circuitry via isogenic comparisons, and improve translational relevance by faithfully modeling patient-derived escape variants. This platform supports rapid go/no-go decision-making in early drug discovery and biomarker identification for next-generation BTK therapies.



 Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong. BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B099.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723b954bdd0f1f0d06162f08a8fe3aac141048aa" target='_blank'>
              Abstract B099: BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery
              </a>
            </td>
          <td>
            Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Complete or partial deletions of chromosome 7 (-7/del7q) represent the most frequent chromosomal abnormalities observed in myeloid neoplasms (MNs) and are associated with a poor prognosis. -7/del7q is observed in 10–15% of adult patients with myelodysplasia (MDS) or with acute myeloid leukemia (AML). The occurrence of -7/del7q is particularly frequent in pediatric MDS, often associated with germline mutations of GATA2 or SAMD9/SAMD9L genes. The disease biology of -7/del7q and the genes driving leukemic development have not been completely elucidated, but the haploinsufficiency of tumor suppressor genes located in chromosome 7 deleted regions seems to play a relevant role. The response to standard treatments based either on chemotherapy or hypomethylating agents plus Venetoclax is limited. No approved targeted therapies exist for patients with -7/del7q; however, some recent studies have discovered some vulnerabilities of these myeloid neoplasms than can be efficiently targeted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a97a077b3eb4226178acf43f03df4eb2019c8ffd" target='_blank'>
              Molecular Complexity of MDS and AML with Aberrations of Chromosome 7
              </a>
            </td>
          <td>
            Ugo Testa, E. Pelosi, G. Castelli
          </td>
          <td>2025-09-18</td>
          <td>Hemato</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are an insignificant, however enormous population of tumor cells that display capacities of self-renewal, differentiation, and tumor initiation, consequently being the core feature in cancer progression, recurrence, and drug resistance. The phosphoinositide 3-kinase/protein kinase B (PI3K/AKT) is one of the most critical signalling cascades regulating CSCs and controlling their stemness, survival, evasion of the immune system under stressful conditions, as well as metabolic reprogramming. This review provides an overview of the morphological features and functional aspects of the PI3K/AKT immune cascade and pathway, and how it essentially connects with both upstream and downstream effectors in CSC biology. The cross-communication of PI3K/AKT with other pathways, e.g., Wnt, Notch, and Hedgehog, is elaborated to emphasize the redundancy of the networks facilitating CSC maintenance and drug resistance. Additionally, we provide an in-depth scrutiny of the processes through which PI3K/AKT signalling leads to CSC resistance to chemotherapy, radiotherapy, and targeted therapy, as well as their plasticity, metastasis, and immune escape mechanisms. Current and future therapeutic approaches targeting the PI3K/AKT axis, such as small molecule inhibitors, combination therapy, and drug delivery nanotechnology, are also discussed. Finally, we present clinical issues and prospects for improving CSC-based therapy by using PI3K/AKT blockade to eliminate resistance and induce protracted, long-lasting cancer remission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f966dace17da846800f374af44f626bb75044a1c" target='_blank'>
              The Role of PI3K/AKT Signalling Pathway in Cancer Stem Cells: Emerging Therapeutic Targets and Resistance Mechanisms
              </a>
            </td>
          <td>
            Komal Arooj, Hassan Imam, Zarlish Attique, Zoha Naeem, Ali Ahmad, Hafiz Muhammad Faraz Azhar, Fariha Javaid, Zeenat Nawaz
          </td>
          <td>2025-09-30</td>
          <td>Futuristic Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The biological mechanisms driving the long survival in glioblastoma (GBM). Five‐year long‐term survival (LTS) and 10‐year survival very long‐term survival (VLTS) remain significantly understudied. Here we molecularly detailed two cases. AR10‐046 (VLTS) was affected by a giant cell‐GBM, classified as the pedHGG_RTK1a subtype according to the v12.5 Heidelberg brain tumor methylation classifier. Somatic and germline MSH6 mutations, typically in Lynch syndrome, and high tumour mutational burden were detected. The copy number variation plots showed chromosome 1q gain and chromosome 13 loss with no other typical GBM alterations. AR10‐037 (LTS) suffered from a classical GBM, identified as pedHGG_MYCN subclass. Apart from the canonical chromosome 7 gain and chromosome 10q loss, we observed MDM2 gene amplification and possible rearrangements on chromosome 12 and 18 with the typical aspect of chromothripsis, harbouring two putative new gene fusions: CPSF6::CPM and PTPRR::RAB3IP. We described two patients with peculiar tumour molecular profile, widening the scenario of clinical and molecular variability in such patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8551ad58743a80e7a359a8ec1bcd9af76ef1c7" target='_blank'>
              Unique Genetic and Epigenetic Alterations in Glioblastoma Long‐Term Survivors: Insights From Two Clinical Cases
              </a>
            </td>
          <td>
            E. Anghileri, E. Miele, S. Patrizi, S. Barresi, E. Lazzarini, L. Maddaloni, M. Patanè, L. Pedace, R. Paterra, A. Silvani, Franco Locatelli, S. Indraccolo, B. Pollo
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Ageing is a general, intrinsic and progressively deleterious process that affects all cells, tissues and organs albeit at different extent and rate in each individual. The complexity and universality of its phenotypic manifestations suggest a multifactorial origin. The autosomal recessive disorder Werner syndrome likely represents a segmental progeroid disorder since patients show several, but not all phenotypes of premature ageing.


SUMMARY
Proliferative senescence of diploid cells in culture provided a model system in which ageing can be studied experimentally. Cultures of cells from patients with Werner syndrome experienced an extreme form of proliferative senescence and a clonal succession of translocations, known as variegated translocation mosaicism. In addition, Werner syndrome cells showed spontaneous deletion formation and a prolongation of and arrest in the S phase of the cell cycle. The WRN protein harbors a helicase, an exonuclease and a RecQ interaction domain. With the latter the WRN protein may interact with NBS1, RPA, MRE11, TREX1, MUTYH, POT1, TRF1, FEN-1, PAPRP-1, p97/VCP, TRF2, DNA polymerase(beta), Ku76/80, EXO-1, NEIL1, and p53, which are key to DNA damage response pathways including canonical NHEJ, homologous recombination, base excision repair and telomere maintenance. The WRN exonuclease domain is a target of WRNIP1 binding, which links WRN to resolution of stalled replication due to collision with transcription and the ATM-mediated cell cycle checkpoint. . Patients with an incomplete complement of Werner syndrome phenotypes, called atypical Werner syndrome patients, were found to carry variants in LMNA, POLD1, SPRTN, MDM2, CTC1, SAMHD1.


KEY MESSAGES
These findings broaden the genotypic landscape and the phenotypic spectrum of Werner syndrome. In this review potential molecular mechanisms underlying genomic instability in Werner syndrome, including chromothripsis due to asynchronous S phase traverse and telomere crises followed by bridge fusion breakage cycles are discussed. The participation of WRN in multiple gene networks is consistent with the multifactorial nature of ageing in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c57a230c575d7a7f034afedf23d1a5bec7bbac" target='_blank'>
              Molecular mechanisms of proliferative senescence and genomic instability in Werner syndrome and the WRN gene network.
              </a>
            </td>
          <td>
            Martin Poot
          </td>
          <td>2025-09-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The persistent residual tumor cells that survive after chemotherapy are a major cause of treatment failure, but their survival mechanisms remain largely elusive. These cancer cells are typically characterized by a quiescent state with suppressed activity of MYC and MTOR. We observed that the MYC-suppressed persistent triple-negative breast cancer (TNBC) cells are metabolically flexible and can upregulate mitochondrial oxidative phosphorylation (OXPHOS) genes and respiratory function (“OXPHOS-high” cell state) in response to DNA-damaging anthracyclines such as doxorubicin, but not to taxanes. The elevated biomass and respiratory function of mitochondria in OXPHOS-high persistent cancer cells were associated with mitochondrial elongation and remodeling, suggestive of increased mitochondrial fusion. A genome-wide CRISPR editing screen in doxorubicin-persistent OXPHOS-high TNBC cells revealed the BCL-XL gene as the top survival dependency in these quiescent tumor cells, but not in their untreated proliferating counterparts. Quiescent OXPHOS-high TNBC cells were highly sensitive to BCL-XL inhibitors, but not to inhibitors of BCL2 and MCL1. Interestingly, inhibition of BCL-XL in doxorubicin-persistent OXPHOS-high TNBC cells rapidly abrogated mitochondrial elongation and respiratory function, followed by caspase 3/7 activation and cell death. The platelet-sparing proteolysis-targeted chimera (PROTAC) BCL-XL degrader DT2216 enhanced the efficacy of doxorubicin against TNBC xenografts in vivo without induction of thrombocytopenia that is often observed with the first-generation BCL-XL inhibitors, supporting the development of this combinatorial treatment strategy for eliminating dormant tumor cells that persist after treatment with anthracycline-based chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb7ad9e0cb28b546f19e574e42e36713a6ee110" target='_blank'>
              Quiescent OXPHOS-High Triple-Negative Breast Cancer Cells That Persist After Chemotherapy Depend on BCL-XL for Survival
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Marie Winter, Leandro Encarnacao Garcia, O. Akinrinmade, Francisco Madeira Marques, Emmanouil Zacharioudakis, Anna Skwarska, J. Aguirre-Ghiso, Marina Konopleva, Guangrong Zheng, Susan A. Fineberg, Daohong Zhou, E. Gavathiotis, Tao Wang, Eugen Dhimolea
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865df3fb5c09b06e06a771bfbf99e3462bf63974" target='_blank'>
              Cell-type-agnostic differential gene expression uncovers conserved principles of cellular regulation
              </a>
            </td>
          <td>
            Alexander R Hummels, C. Camacho
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The regulatory mechanisms governing transcriptional programs in the cancer genome remain elusive, particularly those concerning cell-type specificity. We carefully curated single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) and single-cell RNA sequencing (scRNA-seq) data from eight distinct carcinoma tissues, including breast, skin, colon, endometrium, lung, ovary, liver, and kidney. Using single-cell multi-omics analysis, we identified extensive open chromatin regions and constructed peak-gene link networks, which can reveal distinct cancer gene regulation and genetic risks. We further explored conserved epigenetic regulation across cell types within cancer and elucidated their functional implications. Moreover, we identified cell-type-associated transcription factors (TFs) that regulate key cellular functions, such as the TEAD family of TFs, which widely control cancer-related signaling pathways in tumor cells. In colon cancer, we further identified tumor-specific TFs that are more highly activated in tumor cells than in normal epithelial cells, including CEBPG, LEF1, SOX4, TCF7, and TEAD4, which are pivotal in driving malignant transcriptional programs and represent potential therapeutic targets, as corroborated by single-cell sequencing data from multiple sources and in vitro experiments. Our findings provide a comprehensive understanding of the regulatory dynamics underlying carcinomas and offer valuable insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdeeefa6fa9c95e73d3a569b4614cf1e9a4ee17" target='_blank'>
              Single-cell multi-omics analysis reveals cancer regulatory elements of transcriptional programs and clinical implications
              </a>
            </td>
          <td>
            Xiaowei Tang, Qiaoling Zhang, Zichu Shen, Jian Xiao, Minghao Li, Xiangyan Meng, Chenyi Wang, Guangze Zhang, Anhang Liu, Yuxin Yin
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is composed of a diverse and dynamic spectrum of cell types, cellular activities, and cell–cell interactions (CCI). Understanding the complex CCI within the TME is critical for advancing cancer treatment strategies, including modulating or predicting drug responses. Recent advances in omics technologies, including spatial transcriptomics and proteomics, have allowed improved mapping of CCI within the TME. The integration of omics insights from different platforms may facilitate the identification of novel biomarkers and therapeutic targets. This review discusses the latest computational methods for inferring CCIs from different omics data and various CCI and drug databases, emphasizing their applications in predicting drug responses. We also comprehensively summarize recent patents, clinical trials, and publications that leverage these cellular interactions to refine cancer treatment approaches. We believe that the integration of these CCI-focused technologies can improve personalized therapy for cancer patients, thereby optimizing treatment outcomes and paving the way for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb307e16bbb2fc4780518da53d8301520562b427" target='_blank'>
              Cell–cell interactions as predictive and prognostic markers for drug responses in cancer
              </a>
            </td>
          <td>
            Xuehan Lu, Xiao Tan, Eun Ju Kim, Xinnan Jin, Meg L. Donovan, Jazmina L. Gonzalez Cruz, Zherui Xiong, Maria Reyes Becerra de los Reyes Becerra Perez, Jialei Gong, J. Monkman, Divya Agrawal, A. Kulasinghe, Quan Nguyen, Z. Tuong
          </td>
          <td>2025-10-08</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="This review analyzed approximately 115 peer-reviewed studies published between 2010 and 2025, focusing on molecular subtyping and circulating tumor DNA (ctDNA)-guided approaches in Diffuse Large B-Cell Lymphoma (DLBCL). Evidence was synthesized from retrospective cohorts, prospective clinical trials, and translational studies, highlighting how molecular heterogeneity, clonal evolution, and the tumor microenvironment complicate classification and treatment. While molecular subtypes such as MCD, BN2, EZB, A53, and ST2 have improved prognostication, their routine use in clinical practice remains limited due to cost, complexity, and restricted access to sequencing platforms. Tumor-informed ctDNA assays show promise for minimal residual disease (MRD) monitoring and adaptive therapy, yet their predictive power for CAR-T therapy, bispecific antibodies, and checkpoint inhibitors is still incompletely understood. Overall, the literature converges on the need for integrated strategies combining ctDNA, molecular subtyping, and immune microenvironment analysis to personalize frontline therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5768d4ecc136da89bacb2b8af1b5cdad6bfb931" target='_blank'>
              Personalized Frontline Therapy in Diffuse Large B-Cell Lymphoma: Integrating Circulating Tumor DNA for Real-Time Adaptive Treatment Stratification
              </a>
            </td>
          <td>
            Vasisht Karri, Samir Dalia
          </td>
          <td>2025-10-17</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-14</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="M2-type pyruvate kinase (PKM2) serves as the key rate-limiting enzyme in aerobic glycolysis within tumor cells, where its aberrantly high expression in numerous human malignancies facilitates tumor progression by enhancing glycolytic flux through diverse signaling pathways. Beyond its metabolic function, extensive studies have established PKM2 as a critical non-metabolic signaling regulator implicated in multiple oncogenic processes, including tumor proliferation, invasion, migration, immune evasion, and resistance to chemotherapy. The elucidation of PKM2-mediated oncogenic pathways has spurred the development of targeted therapeutic strategies, positioning PKM2 as a promising target in cancer therapy. However, comprehensive reviews addressing the relationship between PKM2 and tumorigenesis remain limited. This review systematically examines the biological functions of PKM2, the signaling mechanisms through which it exerts its effects in malignant tumors, and the latest advances in the development of PKM2-targeted therapeutics, offering insights into potential directions for future drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00425408db2494b0cf2536b5348cfea19b9b7329" target='_blank'>
              Targeting PKM2 in cancer therapeutics: mechanistic advances and translational opportunities
              </a>
            </td>
          <td>
            Lin Liu, Junyi Wang, Libo Liang, Chunhua Chen, Meng Xie, Wencheng Tang
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A rare disease is generally defined as a medical condition that affects a small proportion of the population, though specific thresholds vary across countries. Despite regional differences, these definitions consistently reflect the low prevalence of such conditions, the limited availability of effective treatments, and the pressing need for targeted research and regulatory support. As a result of their rarity and low commercial potential, rare diseases have historically represented an area of market failure, where investment and research have been limited and often neglected. However, since the 1990s, each country has guaranteed continuous support to research and development projects to promote the advancements of rare disease treatments, achieving a growth rate greater than that of the entire pharmaceutical industry. In this review, we examine the status of orphan drug development using an advanced therapy medicinal product (ATMP) approach in the growing rare disease market, with a particular focus on cell therapies and gene therapies, which constitute the most actively developed and clinically applied categories within ATMPs. We also explore strategic approaches through which the orphan drug industry can utilize ATMPs, especially these two modalities, to enhance its competitiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb2ccc6113bccae4bf1abe5e01112d77e7abd03" target='_blank'>
              Advances in Cell and Gene Therapy for Rare Disease Treatment.
              </a>
            </td>
          <td>
            Hyun Jin Baek, Yoojun Nam, Yeri Alice Rim, Ji Hyeon Ju
          </td>
          <td>2025-09-01</td>
          <td>International journal of stem cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768ee203fcd258706f2498c5fcf4ab4120a2eda" target='_blank'>
              Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages
              </a>
            </td>
          <td>
            Moein Rajaei, Alexander Yang, Kira A. Glasmacher, Christopher N. Cross, Nic Fisk, Elizabeth B. Perry, Jeff Mandell, S. Gaffney, Takafumi N. Yamaguchi, J. Livingstone, Jose Costa, Peter Humphrey, Vincent L. Cannataro, P. Boutros, Jeffrey P. Townsend
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Adoptive T cell therapies (ACT) have revolutionized the management of hematologic malignancies; however, their efficacy in solid tumors remains limited. Accumulating evidence implicates the tumor microenvironment (TME) - a highly complex and immunosuppressive niche as a major barrier to their effectiveness. In this review, we propose that the next generation of ACT will require a fundamental shift from a reductionist focus on T cell engineering alone to an integrated approach that considers the interactions between immune cells and the TME. A comprehensive literature review identified several emerging strategies to enhance the efficacy of ACT, including reprogramming tumor vasculature, repolarizing immunosuppressive myeloid and stromal cells, leveraging oncolytic viruses to remodel antigen presentation, inducing acute sterile inflammation, and targeting the physical properties of the extracellular matrix. While many of these approaches remain in early-stage development, some have already progressed to clinical trials, indicating their potential for clinical translation. Additionally, we found that conventional therapies, such as surgery, chemotherapy, and radiotherapy, can be strategically integrated with ACT to improve therapeutic outcomes. These findings highlight a shift in the field toward more integrative approaches. Future advances will likely depend on reprogramming the TME to support T cell persistence and functions. Addressing these interconnected challenges will require closer collaboration between immunology, oncology, and bioengineering disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5845940292899bbb8a96147c6b79326179c978e6" target='_blank'>
              Reprogramming the tumor microenvironment to boost adoptive T cell therapy
              </a>
            </td>
          <td>
            Maria Fernanda Meza Pacheco, Lee-Hwa Tai
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Osteosarcoma is the most common primary malignant tumor of bone and occurs predominantly in children and adolescents during periods of accelerated skeletal growth. Within the tumor microenvironment (TME), vascular smooth muscle cells (VSMCs) may acquire osteogenic characteristics. Nuclear factor of activated T cells c1 (NFATc1) is central to skeletal biology, yet its behavior in VSMCs in osteosarcoma remains insufficiently defined. Methods We profiled fresh osteosarcoma tissues by single-cell RNA sequencing on the 10x Genomics Chromium platform. Mechanical and enzymatic dissociation yielded single-cell suspensions; viable cells (> 85%) were enriched; libraries were prepared with Single Cell 3′ v3 reagents and sequenced on NovaSeq 6000. Reads were processed with Cell Ranger (GRCh38). Following log-normalization, 2,000 highly variable genes were retained. Harmony integration reduced batch structure. We computed 50 principal components, generated UMAP/t-SNE embeddings, and applied k-nearest neighbor/Louvain clustering (k = 20; resolution = 0.5). Cell types were assigned from differentially expressed markers cross-checked with CellMarker and PanglaoDB. NFATc1 distribution and co-expression patterns were assessed by Pearson correlation. Results Nine major cell classes were identified, including VSMCs, fibroblasts, macrophages, T/NK subsets, B cells, neutrophils, mast cells, and inflammatory myeloid cells. NFATc1 expression was most prominent in VSMCs, with a bimodal distribution within this lineage. In VSMCs, NFATc1 co-varied with SERGEF, APOE, and CCS, pointing to coordinated cytoskeletal remodeling, lipid handling, and copper homeostasis during osteogenic reprogramming. Conclusions At single-cell resolution, NFATc1 marks a VSMC fraction with osteogenic features in osteosarcoma. The associated network suggests testable links to metabolism and matrix remodeling; functional perturbation will be required to establish causality. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03606-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3638bf4253781f5bd9cf49aea68da6335438a6" target='_blank'>
              Single-cell RNA sequencing reveals NFATc1-mediated regulatory mechanisms of vascular smooth muscle cell osteogenic transformation in the osteosarcoma microenvironment
              </a>
            </td>
          <td>
            Xuejiao Yi, Shilin Fu, Chunliang Zhang, Tianle Zhao, Huan Feng, Dehui Qian
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8540fbcce548c20c4ac2831a0c9929750289f6c4" target='_blank'>
              Decoding ferroptosis for cancer therapy.
              </a>
            </td>
          <td>
            A. Wahida, Marcus Conrad
          </td>
          <td>2025-10-10</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Despite occurring in an essential molecule, mutations in RNA polymerase readily emerge and elicit complex pleiotropic effects across different levels of biological organization, which are all modulated by environment. We investigated the impact of temperature on the effects of six mutations on sequence, structure, transcriptome, and organismal traits. We found temperature altered the transcriptomic response and key organismal traits such as growth rate and biofilm formation in a genotype-specific manner. Critically, mechanistic insights into the possible drivers of mutational effects emerged only when examining the relationships between different levels of organization: location of mutations in the tertiary structure and distance to key interacting molecules partly explained the observed transcriptomic differences, which in turn drove the impact of mutations on organismal traits. While falling short of capturing the full complexity of the system, our findings underscore the benefits of integrating insights across multiple biological levels to understand the relationship between environment and mutational effects in molecules with extensive pleiotropic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3646be5bb3c5019f620c6e2499c30f9214f99d4e" target='_blank'>
              Deciphering the Impact of Temperature on Pleiotropic Consequences of RNA Polymerase Mutations
              </a>
            </td>
          <td>
            D. Ozbilek, Jake K. Soley, Danna R. Gifford, Christopher G Knight, Simon C Lovell, M. Lagator
          </td>
          <td>2025-09-16</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The mutagenic translesion synthesis (TLS) pathway, which is critically dependent on REV1's ability to recruit inserter TLS polymerases and the POLζ extender polymerase, enables cancer cells to bypass DNA lesions while introducing mutations that likely contribute to the development of chemotherapy resistance and secondary malignancies. Targeting this pathway represents a promising therapeutic strategy. Here, we demonstrate that the expression of the C-terminal domain (CTD) of human REV1, a ca. 100 amino acid scaffold essential for TLS polymerase interactions, disrupts REV1/POLζ-dependent TLS in mammalian cells. Inducible expression of REV1-CTD in multiple human and murine cancer cell lines sensitizes cells to DNA-damaging agents such as cisplatin, benzo[a]pyrene diol epoxide, and methyl methanesulfonate, without intrinsic cytotoxicity. REV1-CTD expression increases genomic instability, decreases mutagenesis, and enhances G2 arrest following genotoxic stress. Mutational disruption of the CTD's interaction interfaces abrogates these effects, confirming a dominant-negative mechanism via sequestration of TLS components. In a xenograft mouse model, REV1-CTD expression markedly enhances cisplatin efficacy, significantly reducing tumor burden. These findings establish the REV1-CTD as an effective dominant-negative inhibitor of TLS and support its development as a therapeutic agent delivered to cancer cells to enhance the efficacy of genotoxic chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/119ebde0313a7f12ca9995ad4532ab0aaa8b95c3" target='_blank'>
              Sensitization of cancer cells to DNA-damaging agents by expression of the REV1 C-terminal domain: Implications for chemotherapy.
              </a>
            </td>
          <td>
            Ke Bian, Charley Gruber, Emine Byers, Paul Leclerc, Michael T. Hemann, Graham C. Walker
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-21082-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Advanced sarcomas have a poor prognosis and limited therapeutic options. Disease recurrence is caused by persistent cells that survive drug treatments. The alkylating agent trabectedin, when combined with the poly (ADP-ribose) polymerase 1 (PARP1) inhibitor olaparib, exhibits variable antitumor effects in advanced sarcomas. In this study, we demonstrate that the expression of the transcription factor OCT4 is upregulated in persistent cells that survive treatment with trabectedin and olaparib, through the cGAS-STING-IRF3-IFNβ pathway. This route also leads to the upregulation of natural killer (NK) and cytokine-induced killer (CIK) lymphocyte activating ligands. These molecular events enhance the antitumor efficacy of immunotherapy with NK and CIK cells, targeting both the bulk population and residual drug-tolerant cells. In conclusion, the activation of the cGAS-STING pathway has a double-edged effect, enriching the OCT4+ persistent cell population while increasing the expression of NK/CIK ligands. The addition of olaparib to trabectedin potentiates the cGAS-STING pathway activation and the upregulation of NKG2DLs, while simultaneously counteracting the OCT4 overexpression. Therefore, sequential treatment with trabectedin and olaparib followed by NK/CIK immunotherapy represents a promising strategy against advanced sarcomas and warrants further investigation. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04141-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506de56eb87cad4e65432310520f88b9e3f761bd" target='_blank'>
              The cGAS-STING pathway is a master regulator of OCT4 expression in persistent sarcoma cells and enhances cellular immunotherapy with NK and CIK lymphocytes
              </a>
            </td>
          <td>
            Giorgia Giordano, A. Merlini, F. Capozzi, Giulio Ferrero, Cristina Tucciarello, Shahab Majidi, Simona Comparato, G. Mesiano, Elisabetta Liuzzi, F. Galvagno, Annamaria Massa, Silvia Brusco, V. Leuci, Chiara Riganti, S. Cutrupi, Michele De Bortoli, Lorenzo D’Ambrosio, Dario Sangiolo, G. Grignani, Y. Pignochino
          </td>
          <td>2025-09-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated.The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development.Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation.Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Transcriptional regulatory proteins are frequent drivers of oncogenesis and common targets for drug discovery. The transcriptional co-activator, ENL, which is localized to chromatin through its acetyllysine-binding YEATS domain, is preferentially required for the survival and pathogenesis of acute leukemia. Small molecules that inhibit the ENL YEATS domain show anti-leukemia effects in preclinical models, which is thought to be caused by the downregulation of pro-leukemic ENL target genes. However, the transcriptional effects of ENL YEATS domain inhibitors have not been studied in models of intrinsic or acquired resistance and, therefore, the connection between proximal transcriptional effects and downstream anti-proliferative response is poorly understood. To address this, we identified models of intrinsic and acquired resistance and used them to study the effects of ENL YEATS domain inhibitors. We first discovered that ENL YEATS domain inhibition produces similar transcriptional responses in naive models of sensitive and resistant leukemia. We then performed a CRISPR/Cas9-based genetic modifier screen and identified in-frame deletions of the essential transcriptional regulator, PAF1, that confer resistance to ENL YEATS domain inhibitors. Using these drug-resistance alleles of PAF1 to construct isogenic models, we again found that the downregulation of ENL target genes is shared in both sensitive and resistant leukemia. Altogether, these data support the conclusion that the suppression of ENL target genes is not sufficient to explain the anti-leukemia effects of ENL antagonists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Traditional pooled genetic dependency screens, such as DepMap, have proven powerful for identifying essential genes in cancer cell lines. However, these pooled formats are prone to the bystander effect, where secreted factors from wild-type cells support the survival of knockout (KO) cells within the same culture, potentially masking dependencies on specific secreted proteins. To address this limitation, we performed a well-by-well CRISPR screen targeting genes encoding secreted factors in the pancreatic ductal adenocarcinoma (PDAC) cell line HPAFII. This arrayed format eliminates cross-feeding and enables precise assessment of cell-intrinsic dependencies on targeted proteins and could thus expose hidden vulnerabilities. CellTiter-Glo (CTG) viability assays were used to assess gene lethality following target gene KOs. Furthermore, RNA sequencing verified effective disruption of target genes, establishing the specificity and reliability of the screening workflow. We observed strong growth inhibition in EFNA1 (Ephrin-A1) KO samples compared to lipofectamine-treated controls. However, this inhibitory effect was missed and underrepresented in the conventional pooled format. EFNA1 is a ligand for EPH receptors, involved in cell-cell communication, migration, and proliferation signaling. Differential gene expression analysis comparing EFNA1 KO to lipofectamine-treated controls revealed enriched pathways of oxidative phosphorylation, E2F targets, MYC targets, and DNA repair in EFNA1 KO samples. This result supports our observation of lethality in EFNA1 KO samples. It suggests that EFNA1 contributes to cell-cell communication and is critical for HPAFII cell survival. Its KO triggers stress pathways, including E2F, MYC, OXPHOS, and DNA repair, and acts as a suppressor of excessive proliferative programs. In conclusion, we identified EFNA1 as a critical non-cell-autonomous essential gene through an arrayed CRISPR screen, with its loss showing lethality to HPAFII cells exclusively in isolated settings, likely as a result of disrupted cell-cell interactions. Defining such dependencies may facilitate the discovery of directly druggable and more tumor-selective therapeutic strategies.



 Xingyi Shi, David Ruddy, Millicent Gabriel, Michelle Piquet, Kris Muskiewicz, Joel Wagner, Joshua Korn, Cory Johannessen. Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd6ce939e1194e9cb9b823d2bb2e7b39fb1e35f" target='_blank'>
              Abstract C035: Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells
              </a>
            </td>
          <td>
            Xingyi Shi, David A Ruddy, Millicent Gabriel, M. Piquet, Kris Muskiewicz, Joel P Wagner, Joshua Korn, Cory Johannessen
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the second leading cause of cancer death worldwide, largely due to the limited efficacy of current therapies in advanced stages of the disease. Most cases of HCC develop in the setting of chronic liver disease, particularly cirrhosis, where ongoing cycles of inflammation, hepatocyte death and regeneration foster the gradual accumulation of genetic and epigenetic alterations that promote malignant transformation. These molecular changes contribute to the high degree of tumour heterogeneity observed in HCC, a major factor underlying resistance to current treatments. As a result, sustained clinical responses to existing therapies, such as tyrosine kinase inhibitors, anti-angiogenic agents and immune checkpoint inhibitors, remain uncommon. In this context, a growing body of evidence has identified epigenetic dysregulation as a key driver of tumour progression and therapeutic resistance, highlighting a new frontier for intervention. This review provides clinicians and researchers with a comprehensive overview of the emerging field of epigenetic therapies in HCC, summarising results from both completed and ongoing clinical trials involving the so-called 'epidrugs'. Importantly, we discuss how targeting epigenetic mechanisms may not only suppress tumour growth but also enhance the effectiveness of current therapies by reversing resistance pathways. By translating complex molecular insights into tangible therapeutic strategies, epigenetics is poised to reshape the future of HCC management, offering renewed hope for more durable and personalised treatment responses in a disease where progress is urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca914bc3e6f4b196e21231ee5479dd3a9a6ded8" target='_blank'>
              Epigenetic therapies in hepatocellular carcinoma: emerging clinical tools and applications.
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite G Fernandez-Barrena, Esteban J. Fiore, Matías A. Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-09-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Aim: Cisplatin resistance in head and neck squamous cell carcinoma (HNSCC) is thought to involve both reduced drug uptake and altered molecular responses. However, the relative contribution of these mechanisms remains unclear. Methods: Two HNSCC cell lines with differing sensitivity (HNO97 and HNO41) were analyzed using cytotoxicity assays, atomic absorption spectroscopy-based quantification of intracellular cisplatin, caspase 3/7 assays, Western blotting, polymerase chain reaction (PCR)-based transcriptomic analysis of DNA damage response and cell cycle arrest pathways, and RNA-seq data from The Cancer Genome Atlas (TCGA) to characterize the resistance phenotype. Results: HNO97 (IC50 = 440 µM) was 7.6-fold more resistant to cisplatin than HNO41 (IC50 = 57.8 µM; P = 0.0286). After quantifying intracellular uptake (pg Pt/µg protein) and normalizing cytotoxicity to intracellular drug levels, HNO97 (IC50 = 778.9 pg Pt/µg protein) remained 5-fold more resistant than HNO41 (IC50 = 153.5 pg Pt/µg protein), indicating only a partial reduction in resistance (33% decrease, from 7.6-fold to 5-fold; P = 0.0286). At cisplatin concentrations yielding comparable intracellular exposure (HNO97: 440 µM; HNO41: 196 µM; both ≈ 725 pg Pt/µg protein), caspase 3/7 activation and induction of CDKN1A, GADD45A, GADD45G, and PPP1R15A were weaker in HNO97 than in HNO41. Notably, baseline expression of these genes was significantly higher in HNO97. In the TCGA cohort, multivariate analysis showed that high FANCD2 expression was associated with unfavorable recurrence-free survival in platinum-treated patients (hazard ratio = 4.0; P = 0.011), but not in those who did not receive platinum chemotherapy. Conclusion: Cisplatin resistance in HNSCC appears to be driven primarily by molecular mechanisms involving DNA damage response and cell cycle arrest pathways, rather than poor drug uptake.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba27336a14cf094e2bc0e409273ca8370a971da1" target='_blank'>
              Cisplatin resistance in head and neck squamous cell carcinoma is linked to DNA damage response and cell cycle arrest transcriptomics rather than poor drug uptake
              </a>
            </td>
          <td>
            Ketaki Sandu, R. Warta, Uddipta Biswas, Wang Zhang, Patrick Michl, Christel Herold-Mende, Johanna Weiss, Dirk Theile
          </td>
          <td>2025-09-19</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcd8098bae7c85358689bf5f35afc60da63caead" target='_blank'>
              Integrated in silico and in vitro approaches identify SNX.2112 as a drug vulnerability in t(7;12) AML stem-like cells
              </a>
            </td>
          <td>
            Ylenia Cicirò, Denise Ragusa, , Ayona Johns, Sina Kanannejad, Remisha Gurung, Macarena Oporto Espuelas, Chun-Wai Suen, G. Torregrosa-Cortés, Alice Giustacchini, Sabrina Tosi, Cristina Pina
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Drug-resistant tuberculosis (TB) is a major global public health concern. Although isoniazid is currently considered one of the most effective first-line drugs for TB treatment, its efficacy is limited by the emergence of resistance. Therefore, it is imperative to develop new methods for detecting drug-resistant TB. In this study, we developed a nucleic acid detection system based on the clustered regularly interspaced short palindromic repeat (CRISPR) Cas12a_RR protein. The system combines recombinase polymerase amplification with an engineered CRISPR/Cas12a_RR protein to enable rapid and specific detection of the katG G944C mutation in isoniazid-resistant Mycobacterium tuberculosis (Mtb). It could detect the target DNA at concentrations as low as 1% in a mixed sample. Compared with TaqMan quantitative polymerase chain reaction and DNA sequencing, the CRISPR/Cas12a_RR system demonstrated superior detection performance in terms of sensitivity, specificity, and cost-effectiveness. Furthermore, it effectively differentiated between drug-resistant Mtb strains from wild-type Mtb strains in clinically isolated samples, with the entire detection process completed in 60 min. In conclusion, the CRISPR/Cas12a_RR detection system offers a novel, rapid, simple, sensitive, and specific approach for identifying isoniazid-resistant Mtb, with significant potential for clinical application, particularly in resource-limited settings. IMPORTANCE This study presents a novel method for detecting isoniazid-resistant Mycobacterium tuberculosis (Mtb) using clustered regularly interspaced short palindromic repeat (CRISPR)/Cas12a mutants, offering rapid detection, cost-effectiveness, and high specificity, and thereby providing a promising new avenue for detecting isoniazid-resistant Mtb.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e356a9b39fa569e7328badc40596da35e80b2078" target='_blank'>
              Application of engineered CRISPR/Cas12a variants with altered protospacer adjacent motif specificities for the detection of isoniazid resistance mutations in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Pengfei Liu, Jinping Zhang, Yaqi Gong, Wenqi Liu, Guohui Xiao, Juan Liang, Xuan Wang, Jing Bi, Guoliang Zhang
          </td>
          <td>2025-09-03</td>
          <td>Microbiology Spectrum</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear.We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells.High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells.Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer risk increases with age, and cellular senescence may be a major contributor to cellular carcinogenesis. Enormous efforts have been made to investigate the interrelation between aging and tumors, but little is known about the comparative features of normal aging, cellular senescence, and cancer at single-cell resolution. By integrating analyses of genomics, epigenomics, and bulk and single-cell transcriptomics, we revealed a directionally opposite transcriptional profile between cellular senescence and tumorigenesis at the single-cell level, which may be affected by epigenomic regulations. A total of 648 aging-dependent senescence-associated coregulated modules (SACMs), disproportionately affecting the reproductive systems of both females and males, were initially defined across 17 tissues. Single-cell analysis revealed that aging primarily affects endothelial cells, followed by T cells, epithelial cells, macrophages, and fibroblasts. Opposite directions of change in gene expression between aging and cancer can commonly be observed in endothelial, fibroblast, and epithelial cells, which may prompt the opposing patterns of gene expression between tissue aging and epithelial carcinoma at the bulk level. A similar pattern of expression can be observed in immune cells, which are characterized by decreased self-renewal with aging, but this pattern is reversed in epithelial carcinoma. Our study highlighted the role of senescence as a natural barrier against tumor formation and supported the idea that aging-related systemic environment changes create a protumorigenic milieu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4073352af61f280e6d1b0c8dc5f8cd7b466fe5" target='_blank'>
              Integrative Transcriptomic Analysis Decodes the Interplay Between Aging, Senescence, and Cancer.
              </a>
            </td>
          <td>
            Jiale Cai, Deng Wu, Dahua Xu, Fujin Liu, Peihu Li, Dehua Zheng, Zhizhou Xu, Meng Cao, Yutong Shen, Nihui Zhang, Guanghui Tian, Chengyi Fan, Bo Wang, Kongning Li, Xiaoman Bi
          </td>
          <td>2025-09-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Cutaneous T-cell Lymphomas (CTCLs) refer to a group of rare cancers that affect immune cells in the skin and often the blood. Their exact cause is not entirely understood, and they are difficult to diagnose because they can appear similar to common, non-malignant skin conditions. Recent advances in single-cell RNA sequencing have made it possible for researchers to explore these tumors at an unprecedented level of detail, revealing important differences between CTCL subtypes in how they interact with the immune system. By analyzing these patterns, researchers can begin to explain differences in disease behavior and identify targets for new therapies. This summary highlights how single-cell technologies are transforming our understanding of CTCL and may pave the way for more precise diagnosis and treatment strategies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504bc3b3f8515fbfaaeba0942215a380977f71da" target='_blank'>
              An Update on Single-Cell RNA Sequencing in Illuminating Disease Mechanisms of Cutaneous T-Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Suhl, Alexander Kaminsky, Caroline Chen, B. Lapolla, Maggie H. Zhou, Joshua Kent, Abigail Marx, I. Nebo, G. Ramush, Sophia Luyten, Yoni Sacknovitz, Julie Sung, Christina M. Bear, Celine M. Schreidah, Alejandro A. Gru, L. J. Geskin
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly aggressive and heterogeneous malignancy, in which natural killer (NK) cells play a crucial role in tumor progression and immune surveillance. This study aimed to characterize the transcriptomic landscape of NK cell–associated genes (NAGs) and explore their associations with clinical outcomes and therapeutic responses in HCC. Using transcriptomic data from The Cancer Genome Atlas (TCGA), we identified key NAGs through comprehensive statistical analyses. Patients were stratified into distinct risk groups based on NAG expression profiles. Low-risk patients demonstrated better survival, higher immune infiltration, and greater predicted sensitivity to immunotherapy, whereas high-risk patients were associated with reduced chemotherapy responsiveness. These findings contribute to a deeper understanding of the immunogenomic features of HCC and provide a basis for developing personalized therapeutic approaches centered on NK cell–related mechanisms. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03692-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f099d43ef2898930decd4284d14d3171c835aebb" target='_blank'>
              Transcriptomic landscape of NK cell–related genes in hepatocellular carcinoma: associations with prognosis and therapeutic response
              </a>
            </td>
          <td>
            Yuxiang Wang, Wenqiang Liao, Jianbo Lin, Jun Li, Jianping Gong
          </td>
          <td>2025-10-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Histone deacetylase inhibitors (HDACi’s) have multiple effects on cancer cell lines, including inhibition of cell growth in vitro, and of human tumor xenografts in vivo, and have had modest therapeutic efficacy in a subset of human malignancies. Previously, we have shown that chronic treatment of colon cancer cells with HDACI’s can result in adaptation, associated with marked decreases in cell growth, colony formation, and tumorigenicity, and extensive changes in gene expression, suggesting an attenuation of malignant phenotype and a possible novel approach to using these agents. Ewing Sarcoma (ES), likely derived from a very primitive mesenchymal-related cell, is an aggressive malignancy of children often resistant to therapy. ES cells classically contain the characteristic EWS-FLI1 translocation and fusion gene, known to dramatically alter histone modifications and epigenetic regulation. We hypothesized that chronic treatment of ES cell lines with HDACi’s will major changes in the program of ES cells gene expression that may alter their malignant phenotype. We treated the ES cell line A673, which contains the EWS-FLI fusion gene, with either stepwise increasing amounts of Entinostat (selective class I HDACi, up to 1.5µM) or single dose chronic treatment (1.5µM). We observed treatment-associated dramatic reductions in growth rates, colony formation, anchorage-independent growth, and major changes in gene expression (RNA sequencing, RT-qPCR) and chromatin organization (ATAC-seq). Gene expression analysis revealed hundreds of differentially expressed genes including changes associated with differentiation along both neural and mesenchymal pathways, consistent with observed changes in lineage-determining transcription factors of the Nkx, Sox, and Pax families. Other genes with changed expression levels included potential novel therapeutic targets such as the tyrosine kinase, RET, whose expression was reduced at both the RNA and protein levels, in association with reduced chromatin accessibility. We hypothesized that Ret may play a role in the malignant phenotype of ES and were interested in whether HDACi-induced reduction in Ret expression might sensitize A673 cells to Ret inhibitors. Co-treatment of A673 cells with Entinostat resulted in over two-fold increase in sensitivity to Cabozantinib, a tyrosine kinase inhibitor that inhibits a select group of tyrosine kinases including Reg. We are continuing to characterize the role of Ret as a potential therapeutic target, and as a mediator of the ES cell malignant phenotype through CRISPR-mediated gene.Acknowledgement: This work was supported by funding from the NJ Pediatric Hematology and Oncology Center of Excellence of the Rutgers Cancer Institute.



 Hsin-Ching Lin, Erica Raj, Luis Sanchez-Gonzalez, Aisling Byrne, Shin-Heng Chiou, Arnold B Rabson. Effects of chronic histone deacetylase inhibitor treatment on Ewing sarcoma cells highlight Ret as a potential therapeutic target [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43276da437d6f15c2c1a18ce683e6dea744bab69" target='_blank'>
              Abstract B014: Effects of chronic histone deacetylase inhibitor treatment on Ewing sarcoma cells highlight Ret as a potential therapeutic target
              </a>
            </td>
          <td>
            Hsin-Ching Lin, Erica Raj, L. Sánchez-González, Aisling Byrne, Shin-Heng Chiou, Arnold B. Rabson
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Synthetic lethal interactions are attractive therapeutic candidates as they enable selective targeting of cancer cells in which somatic alterations have disrupted one member of a synthetic lethal gene pair while leaving normal tissues untouched, thus minimising off-target toxicity. Despite this potential, the number of well-established and validated synthetic lethal gene pairs is modest. Results We generate a dual-guide CRISPR/Cas9 Library and analyse 472 predicted synthetic lethal pairs in 27 cancer cell Lines from melanoma, pancreatic and lung cancer Lineages. We report a robust collection of 117 genetic interactions within and across cancer types and explore their candidacy as therapeutic targets. We show that SLC25A28 is an attractive target since its synthetic lethal paralog partner SLC25A37 is homozygously deleted pan-cancer. We generate knockout mice for Slc25a28 revealing that, except for cataracts in some mice, these animals are normal; suggesting inhibition of SLC25A28 is unlikely to be associated with profound toxicity. Conclusions We provide and validate an extensive collection of synthetic lethal interactions across cancer types. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03737-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d43269824feb99e7e4f4f573a2dcaf7ac03c77" target='_blank'>
              A compendium of synthetic lethal gene pairs defined by extensive combinatorial pan-cancer CRISPR screening
              </a>
            </td>
          <td>
            V. Harle, Victoria Offord, Birkan Gökbağ, Lazaros Fotopoulos, Thomas Williams, Diana Alexander, Ishan Mehta, N. Thompson, R. Olvera-León, Stefan Peidli, Vivek Iyer, Emanuel Gonçalves, Narod Kebabci, Barbara De Kegel, J. van de Haar, Lang Li, Colm J. Ryan, David J. Adams
          </td>
          <td>2025-09-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b348bc9a3e4abf3cb6086b89e0d2182bceec0bd" target='_blank'>
              NEK9-mediated Wnt signalling repressor TLE3 rewires Docetaxel resistance in cancer cells by inducing pyroptosis.
              </a>
            </td>
          <td>
            S. Ansari, Sibasish Mohanty, P. Mohapatra, Sudeshna Datta, Mamuni Swain, Rachna Rath, D. Muduly, Saroj K D Majumdar, R. Swain, Sunil K Raghav, Rupesh Dash
          </td>
          <td>2025-09-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cells can acquire a reversible, dormant drug-tolerant persister state mimicking embryonic diapause to evade therapy pressure. Deciphering the precise mechanisms driving cancer cells into or out of a diapause-like persister cell-state could provide strategies to overcome resistance. Here, we showed that following chemotherapy, diverse therapeutic agents converge on WNT pathway activation to induce a de novo diapause-like cell state across various triple-negative breast cancer (TNBC) cell line, xenograft, and patient-derived organoid models. Among early persister cells, only transcriptionally WNT-active persisters exhibited the transcriptional and functional characteristics typical of diapause-like cells, including a negative correlation with MYC transcriptional activity and reversible restricted proliferation. The WNT signaling pathway functioned as both an inducer and biomarker of the diapause-like early persister cell state in both parental (chemo-naïve) and chemotherapy-treated cells. Entry into and exit from the diapause-like early persister cell state was triggered by the transcriptional upregulation of components essential for canonical WNT ligand secretion. A combinatorial treatment strategy inhibiting WNT ligand secretion alongside chemotherapy effectively targeted the early mechanisms underlying the acquisition and enrichment of a diapause-like cell phenotype. These findings reveal WNT pathway activation as an early event that leads to a reversible diapause-like persister state and highlight the potential of targeting this axis to prevent the development of drug-resistant populations before they are firmly established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9bf626a72b9d5ae7b70a19b1f05585d0584500" target='_blank'>
              Chemotherapy-Treated Breast Cancer Cells Activate the WNT Signaling Pathway to Enter a Diapause-Like Early Persister State.
              </a>
            </td>
          <td>
            Youssef El Laithy, Willy Antoni Abreu de Oliveira, A. Pabba, Alessandra Qualizza, Gwenny Cosemans, Paula Garcia-Diaz, François Richard, Paraskevi Athanasouli, Carla Rios Luci, Wout De Wispelaere, Larissa Mourao, Siân Hamer, Stijn Moens, A. de Jaime-Soguero, M. Baietti, S. Hutten, Jos Jonkers, Stephen-John Sammut, S. Soenen, C. Scheele, Alejandra Bruna, C. Desmedt, D. Annibali, F. Lluis
          </td>
          <td>2025-10-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="


 Despite aggressive treatment, glioblastoma patients suffer from an invariably poor prognosis. Whilst substan- tial progress has been made in understanding the role of DNA methylation in brain tumour pathology, methy- lation of RNA has received comparatively little attention, despite being a key driver of cellular differentiation and neurodevelopment. N6-methyladenosine (m6A) is the most abundant RNA modiﬁcation and is, crucially, reversible. The immense intra-tumoral heterogeneity and phenotypic plasticity of glioblastoma are major barri- ers to successful treatment, and dynamic m6A methylation may mediate such behaviour, by ﬂexibly inﬂuencing transcriptional networks via the regulation of mRNA metabolism.



 Nanopore sequencing technology uniquely enables direct sequencing of RNA, and therefore detection of RNA modiﬁcations within their sequence context. We present nanopore direct RNA sequencing data from four in- house glioma-derived cell lines. We validate this approach using a novel drug inhibitor of the major m6A methyl- transferase, METTL3.



 We deﬁne transcriptome-wide RNA methylome proﬁles for four cell lines and identify common candidate path- ways which exhibit methylation. We conﬁrm robust reduction in m6A marks following treatment with a novel METTL3 inhibitor. We further compare the transcriptional landscape of wild-type tumours against these treated samples, demonstrating differential expression of biological processes linked to RNA metabolism.



 We demonstrate that nanopore direct RNA sequencing can resolve m6A modiﬁcations at single-base resolu- tion, preserving m6A stoichiometry. We validate this approach with a known inhibitor of METTL3, and further identify differentially expressed pathways in response to such inhibition. We show that glioma cell lines demon- strate extensive methylation throughout the transcriptome, and exhibit a degree of inter-tumoral heterogeneity of these m6A signatures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad119dfd67a68d80f7976dca916d99c54e9f801" target='_blank'>
              EXPLORING THE GLIOMA M6A RNA EPITRANSCRIPTOME WITH DIRECT RNA SEQUENCING
              </a>
            </td>
          <td>
            Simon Deacon, M. Radhi, Lauryn Walker, J. Rowlinson, S. Paine, M. Loose, Stuart Smith
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acdfefa7d1e9916c129010f57a43699739821df" target='_blank'>
              Cancer gene therapy: historical perspectives, current applications, and future directions.
              </a>
            </td>
          <td>
            Jingyi Zeng, Jingwen Luo, Yingchun Zeng
          </td>
          <td>2025-09-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9e3d7e8367b6848517e2b91b0fabe3f4c124a0" target='_blank'>
              Identification of a novel PML exon 6 splice variant in atypical PML::RARα transcripts in acute promyelocytic leukemia.
              </a>
            </td>
          <td>
            Haimin Chen, Meihong Chen, Zhongjie Yang, Shuzhen Liao, Yun Lin, Linlin Yan
          </td>
          <td>2025-09-26</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Of the thousands of genes and substrates identified in KRAS-mutant signaling networks in pancreatic ductal adenocarcinoma (PDAC), more than 200 are transcription factors, implying extensive and complex transcriptional regulation. However, we observed that genetic suppression of the transcription factor MYC alone was sufficient to phenocopy the effect of KRAS suppression in signaling, growth, and metabolic processes in PDAC cells. We determined the gene transcription changes caused by acute suppression of MYC function in KRAS-mutant PDAC cell lines and performed dependency map and pathway analyses on the affected gene sets. The expression of 1685 genes was increased upon suppression of MYC, and this gene set may comprise the bulk of the MYC-regulated genes essential for PDAC growth. In contrast, the 1325 genes whose expression was inhibited may comprise a compensatory response to oncogenic stress, mediated in part by the GTPase RHO. MYC-dependent transcriptional activity was largely ERK dependent, and almost one-third of ERK-regulated genes were also regulated by MYC in PDAC cells. Furthermore, chemical proteomic profiling revealed MYC-regulated protein kinases that can be targeted therapeutically. Together, these data provide a molecular portrait of MYC-dependent signaling that encompasses potentially exploitable mechanisms for treating PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2dfd12aa532b6fe57212e6285b3679bdee8785" target='_blank'>
              Defining the MYC-regulated transcriptome and kinome that support KRAS- and ERK-dependent growth of pancreatic cancer.
              </a>
            </td>
          <td>
            Priya S. Hibshman, Clint A. Stalnecker, Jeffrey A. Klomp, Kristina Drizyte-Miller, Jennifer E. Klomp, A. C. Edwards, Lily M Pita, Richard G Hodge, J. Diehl, Ryan D. Mouery, Brandon L. Mouery, Kayla R Snare, Andrew M. Waters, Sen Peng, Natalie K Barker, M. Pierobon, N. Rashid, Nhan L Tran, Laura A Herring, Lee M. Graves, Emanuel F. Petricoin, Kirsten L. Bryant, Adrienne D. Cox, C. Der
          </td>
          <td>2025-09-30</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="The study by Li et al. , “Single-cell transcriptome analysis reveals the association between histone lactylation and cisplatin resistance in bladder cancer”, investigated how histone lactylation contributes to cisplatin resistance in bladder cancer (BCa). Using high-resolution single-cell lineage tracing, the authors identified a distinct subpopulation of BCa cells with enhanced glycolytic metabolism that exhibited significant cisplatin resistance. Further analyses showed significant enrichment of histone H3 lysine 18 lactylation (H3K18la) at the promoter regions of the transcription factors YY1 and YBX1, which promoted their expression and facilitated the development of a drug-resistant phenotype. This study was the first to combine single-cell omics with lactylation modification mechanisms, providing new insights into strategies for overcoming cisplatin resistance in BCa. Nonetheless, potential limitations of the study should be carefully considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d10bc76434e826705d215ff116fbc298bccf9ac" target='_blank'>
              Histone H3K18 lactylation and cisplatin resistance in bladder cancer: insights from single-cell transcriptomic analysis
              </a>
            </td>
          <td>
            Zongchi Yang, Shaohua Chen
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40935d544b5f896144e035981022bfa9b8389a70" target='_blank'>
              Investigation of Double-Stranded DNA Donors and CRISPR-Cas9 RNP for Universal Correction of Mutations Causing Cystic Fibrosis in Human Airway Cells
              </a>
            </td>
          <td>
            Vrishti Sinha, P. Ayoub, C. Juett, Lindsay E. Lathrop, Ruth A. Foley, Ruby A. Sims, Joseph D. Long, Emily C. Duggan, Neil Fernandes, Beate Illek, B. Gomperts, S. Jonas, Donald B. Kohn
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Among the various subtypes of acute myeloid leukemia (AML), TP53-mutated (TP53-Mut) AML is one of the most aggressive in both adults and children due to its resistance to available therapies. Our goal is to identify vulnerabilities that these cancer cells exhibit in order to exploit them as potential therapeutic targets. Using the Cancer Dependency Map (DepMap), we stratified the 26 AML cell lines into TP53-Mut and TP53-wildtype (TP53-WT) categories and identified adenosine deaminase (ADAR1) as the top selective dependency in TP53-Mut AML. ADAR1 is an enzyme that catalyzes adenosine-to-inosine (A-to-I) editing of double stranded RNA (dsRNA), thereby marking it for degradation. Loss of ADAR1 therefore leads to an accumulation of dsRNAs which activates cytoplasmic dsRNA sensors that trigger increased interferon (IFN) signaling and activation of PKR, leading to translation inhibition and ultimately cell death. To validate this dependency, we generated ADAR1 knockout (KO) cells in TP53-Mut and TP53-WT cell lines. Our results showed that TP53-Mut AML cells are sensitive to ADAR1 loss in vitro and in vivo. Due to the role of ADAR1 in dsRNA editing, we hypothesized that TP53 negatively regulates the expression of dsRNA, which renders TP53-Mut AML cells more reliant on ADAR1 editing. Notably, we found through immunofluorescent and flow staining of dsRNA that TP53-Mut AML cells have higher basal levels of dsRNA compared to TP53-WT cells. After loss of ADAR1, the amount of dsRNA increases above the activation threshold of PKR and activates it in a TP53-Mut dependent manner.To identify the molecular mechanism behind the selective sensitivity of TP53-Mut AML cells to ADAR1, we performed a genome-wide CRISPR screen in TP53-KO MV4-11 AML cells. The screen revealed that KO of IFN pathway activators (e.g., IRF9, STAT2, and IFNAR1) conferred resistance to ADAR1 loss, while KO of genes that inhibit inflammation (e.g., ISG15, USP18), enhanced sensitivity to ADAR1 loss. Notably, KO of IRF9 rescued the cells from ADAR1 loss and decreased the levels of PKR activation. In contrast, loss of USP18 further enhanced ADAR sensitivity and increased the levels of PKR activation. This suggests that PKR activation is a biomarker for sensitivity to ADAR1 loss. Overall, a high inflammatory environment present in TP53-AML cells leads to increased PKR expression, which primes them to sensitivity to the loss of ADAR1. Thus, we hypothesized that activating PKR in combination with ADAR1 loss would exacerbate cell death in TP53-Mut AML cells. Two FDA-approved small molecules that activate PKR were successful in producing anti-proliferative effects in TP53-Mut AML cells when combined with ADAR1 loss compared to TP53-WT AML cells in vitro. Our findings show that TP53-Mut AML cells rely on the ADAR-1/PKR pathway for survival and provide a new therapeutic target that can be combined with FDA-approved drugs as a promising basis for treating an aggressive subtype of AML.



 Audrey Taillon, Yara Rodriguez, Gabriela Alexe, Allen Basanthakumar, Lucy Merickel, Silvi Salhotra, Olasunkanmi Olaoye, Constanze Schneider, Oivin Guicherit, Paige Stocker, Haley Curtis, Kimberly Stegmaier. ADAR1 loss in combination with PKR activation leads to anti-tumor effects in TP53-Mut AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2aad1c7dba5d80f761f04288372688e01f21ae" target='_blank'>
              Abstract A027: ADAR1 loss in combination with PKR activation leads to anti-tumor effects in TP53-Mut AML
              </a>
            </td>
          <td>
            A. Taillon, Yara Rodriguez, G. Alexe, A. Basanthakumar, L. Merickel, S. Salhotra, Olasunkanmi Olaoye, C. Schneider, O. Guicherit, Paige Stocker, Haley Curtis, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background The genetic characteristics of surgically removed cancerous tissues are examined using cancer gene panel testing in cancer genome medicine to detect the pathogenic variants involved in the proliferation and progression of cancer cells. An antitumor drug is prescribed if it directly acts on the detected pathogenic variant; however, some aspects require careful consideration by medical professionals in such cases. The genetic mutations involved in the progression or onset of malignant tumors differ with race. Furthermore, genetic mutations that are variants of unknown significance (VUS) may be involved in the progression or onset of malignant tumors in some races according to the ClinVar results from the National Center for Biotechnology Information. Single nucleotide variations can result in silent mutations or splice sites. Methods We therefore reexamined the CGP results (VUS) of patients suspected of developing hereditary tumors based on their family background using IGV and RT-PCR. Results KRAS Q61K, which is found in many gastrointestinal cancers, was identified as a VUS by ClinVar, but this gene mutation was found to cause splicing. The cancer gene panel test of a 41-year-old male patient with paraganglioma identified succinate dehydrogenase complex, iron-sulfur subunit B (SDHB) G642T as a VUS. However, this mutation was later discovered to cause the splicing site to shift, preventing SDHB from translating from the correct mRNA. In addition, a cancer gene panel test of a 47-year-old patient with right breast cancer determined that breast cancer susceptibility gene 2 (BRCA2) 631 3A>T was a VUS. However, this mutation may create a splicing site, which means that the correct BRCA2 mRNA for BRCA2 is not produced. Conclusions The diagnosis of gene mutations based on the results of cancer gene panel testing may not always be correct, and a detailed examination of gene mutations is necessary. Our medical staff has performed cancer gene panel testing on approximately 5,500 cases of intractable malignant tumors to date and are investigating new treatments for these tumors. In this article, we discuss our experience with cancer gene panel testing as well as the problems encountered and new findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a638c15c6a8b23fbe602cb20d96ff59a3c3e13d" target='_blank'>
              Problems in Cancer Genome Medicine: Base Mutations Cause Intron Start Signals, Resulting in Unexpected Splicing
              </a>
            </td>
          <td>
            Takuma Hayashi, I. Konishi
          </td>
          <td>2025-09-17</td>
          <td>World Journal of Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20084-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lysosome-dependent cell death (LDCD) is an emerging form of regulated cell death with critical implications in tumor development, immune modulation, and therapy responsiveness. However, the role of LDCD-related genes in colon adenocarcinoma (COAD) remains poorly understood. We comprehensively analyzed LDCD-related gene expression profiles using transcriptomic data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus databases (GEO). Unsupervised clustering was performed to identify molecular subtypes. A prognostic signature was developed using LASSO and Cox regression analyses. Immune infiltration characteristics and immunotherapy responses were assessed via multiple algorithms. Single-cell RNA sequencing (scRNA-seq) analysis was conducted to explore the cellular distribution of LDCD genes. Functional assays, including colony formation, Transwell migration, and western blotting, were performed to validate the role of key LDCD regulators in COAD cell line. LDCD-related genes were differentially expressed in COAD tissues and stratified patients into two distinct subtypes with significant differences in survival, immune infiltration, and biological pathways. A five-gene prognostic signature demonstrated robust predictive ability for overall survival and was associated with tumor immune microenvironment characteristics. High-risk patients showed distinct immune checkpoint profiles and predicted sensitivity to immune checkpoint blockade therapy. scRNA-seq analysis revealed cell type-specific expression patterns of LDCD genes. In vitro experiments confirmed that modulation of a key LDCD regulator, SLC11A1, affected the proliferation and invasiveness of COAD cells. Moreover, the impact of SLC11A1 on apoptosis and intracellular ROS level confirmed the potential association between SLC11A1 and COAD cell apoptosis. Our study reveals the prognostic value and immunological relevance of LDCD-related genes in COAD. These findings provide new insights into molecular classification and may aid in the development of personalized therapeutic strategies targeting LDCD pathways in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18267-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9901f6ba7f419afbbf7d8a4c7ff078253e88b294" target='_blank'>
              Comprehensive characterization of lysosome-dependent cell death reveals prognostic significance and immune landscape in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhenhua Liu, Lianming Cai, B. Liao, Baowen Wang, T. Xia, Xiangxin Wu
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable, as most patients relapse or become refractory (RRMM), and the molecular nexus among clonal evolution, immune escape and therapy response remains unknown. Bulk RNA-seq from 24 MM to 6 matched marrows (subset of 132) plus single-cell datasets were analysed by differential expression, pseudotime, WGCNA, enrichment, immune deconvolution and GDSC drug screens. PRKD2 was knocked down (KD) or over-expressed (OE) in H929, 8226 and U266 cells. Viability (CCK-8) and apoptosis (Annexin V/7-AAD) were assessed after 48 h of axitinib (1.25–10 µM). PRKD2 was the sole gene uniformly up-regulated in newly diagnosed MM, RRMM and poor-survival cohorts, rising along the malignant plasma-cell trajectory. High PRKD2 aligned with ISS-III stage, elevated mRNAsi/EREG_mRNAsi and worse 5-year survival (38% vs. 73%, P = 0.012). PRKD2-KD reduced proliferation (~ 30%) and restored HLA-E, whereas PRKD2-OE suppressed antigen-presentation genes and polarised macrophages toward an IL-6/MERTK tumour-promoting phenotype. PRKD2-high marrow contained more naïve/memory B cells but fewer mature plasma cells. PRKD2 expression correlated positively with axitinib sensitivity (– log10 IC50, r = 0.68; P < 0.001). Experimentally, PRKD2-OE cells showed a 1.4-fold increase in axitinib-induced apoptosis, while PRKD2-KD cells were intrinsically pro-apoptotic and further sensitised; axitinib abolished the growth advantage conferred by PRKD2-OE. PRKD2 integrates secretory stress, stem-like programmes and immune evasion, driving aggressive MM yet exposing a vulnerability to VEGFR/PDGFR blockade. Direct PRKD2 targeting or axitinib-based combinations—including proteasome inhibition—deserve clinical evaluation for high-risk MM. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20615-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9cd27c9216096c4a23f1fd03d51048eaa29b1af" target='_blank'>
              Single-cell and bulk transcriptomics uncovers PRKD2-driven tumor stemness and progression in multiple myeloma
              </a>
            </td>
          <td>
            Guihua Zhang, Shengya Cao, Chong Geng, Yueyue Sun, Jinge Xu, Chong Chen, Faan Miao, Huanxin Zhang, Yunxin Kong, Rongke Jiang, Kaige Liu, Jiaqian Qi, Zhenyu Li, Hong Liu
          </td>
          <td>2025-10-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07528eee7ff3d627e8787aab243a7521b7075a4" target='_blank'>
              Targeting p53-MDM2 pathway with novel triazole-oxazole hybrids: a fragment-based drug discovery approach for next-generation cancer therapies.
              </a>
            </td>
          <td>
            Apurva Prajapati, Hitesh Patel
          </td>
          <td>2025-09-23</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. Methods We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. Results EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. Conclusion This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03631-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Waleed K. Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 DHX9, a multi-functional RNA helicase, plays important roles in maintaining genome stability by unwinding various nucleic acid secondary structures that may lead to replication stress and DNA damage if left unresolved. DHX9 is overexpressed in multiple cancers, with selective vulnerability to DHX9 inhibition noted in tumors characterized by genomic instability and high replicative stress. ATX-559 is an orally bioavailable, small-molecule inhibitor of DHX9 helicase activity currently in Phase 1 clinical development. ATX-559 induces robust anti-tumor activity in a variety of preclinical models with genomic instability, including those with microsatellite instability-high (MSI-H) and/or deficient mismatch repair (dMMR), BRCA alterations, and homologous recombination repair deficiency (HRD).



 ATX-559 anti-proliferative activity was tested in a multi-indication cell panel screen. Triple negative breast cancer (TNBC) and colorectal cancer (CRC) cells were used to evaluate ATX-559 mechanistic activity using immunofluorescence imaging, dot blot, and western blot assays. ATX-559 was dosed orally BID to evaluate its anti-tumor activity in CRC and TNBC cell line-derived xenograft (CDX) models, as well as a panel of 64 patient-derived xenograft (PDX) models including TNBC, Ovarian, CRC, endometrial, and gastric cancers. qPCR was used to evaluate the target engagement PD biomarker, circBRIP1.



 ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress, subsequent DNA damage, and cell death selectively in cancer models with genomic instability and replicative stress. ATX-559 sensitivity was enriched in dMMR/MSI-H cell lines including CRC. Sensitivity was also enriched in BRCA altered TNBC and HRD ovarian cancer cell lines. ATX-559 is well tolerated in vivo, resulting in robust dose-dependent tumor growth inhibition and regression selectively observed in CRC dMMR/MSI-H and BRCA altered TNBC CDX models, along with a well correlated PK/PD/efficacy relationship. Additionally, ATX-559 demonstrated enriched sensitivity in a large panel of BRCA altered TNBC (75%), dMMR/MSI-H (68%), and HRD ovarian (55%) PDX models.



 Together, these findings represent preclinical validation of DHX9 as a selective therapeutic in cancers with genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1/2, open-label, dose-escalation and expansion study to evaluate the safety profile of ATX-559 and to determine the recommended phase 2 dose (RP2D) and preliminary antitumor activity in patients with locally advanced or metastatic solid tumors, and molecularly defined cancers (NCT06625515).



 Sunaina Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver. ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2a8e9dfeed1b9e094182590e0697b62580f888" target='_blank'>
              Abstract B045: ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            S. Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen M. Lynes, Stuart Ince, Jason A. Sager, Serena J Silver
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Aim Typically expressed on T-cells and NK cells, FASLG induces apoptosis in target cells upon binding Fas. However, assessing potential FASLG expression in tumor cells with convenient genomics approaches has been challenging. Materials and Methods This study applied a novel assessment of FASLG copy numbers (CNs) and gene expression levels, applicable to bulk exome and RNAseq files. Results Analyses indicated high FASLG CN associated with worse survival outcomes. Interestingly, higher FASLG gene expression was found to be associated with better survival outcomes, which led to a determination of whether this result was due to FASLG expression from tumor-infiltrating lymphocytes (TILs) instead of cancer cells demonstrating the higher CNs. In fact, T-cell markers CD4 and CD8A highly correlated with FASLG expression, consistent with the hypothesis that the high FASLG expression was associated with the TILs. Subsequent analyses confirmed that CN increases led to increased gene expression in the genomic region of the FASLG gene, particularly with an assessment of the expression of the neighboring PRRC2C gene. In sum, FASLG CN assessments, even independently of a corresponding gene expression correlation, may provide important characterizations of tumor cells. Conclusion This study indicates that FASLG CN increases could represent a mechanism of tumor escape from TILs and a prognostic indicator; and tumor FASLG may be a suitable drug target for reducing tumor evasion of T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0153f78ee8bea9bd0c80db1c2166e2902441e45" target='_blank'>
              Programmatically Efficient Separation of Immune Infiltrate and Tumor Gene Expression Overlap Potentials in a Big Data Setting: FASLG Gene Expression-related Survival Distinctions
              </a>
            </td>
          <td>
            Elizabeth A. Fletcher, Toriana R. Dabkowski, Mallika Varkhedi, George Blanck
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889c401af57de3889b95cf28e228d76bc1fc7c6d" target='_blank'>
              Repeated losses of self-fertility shaped heterozygosity and polyploidy in yeast evolution
              </a>
            </td>
          <td>
            Nina Vittorelli, C. Gómez-Muñoz, Irina Andriushchenko, Louis Ollivier, Nicolas Agier, Stéphane Delmas, Yann Corbeau, Guillaume Achaz, M. Cosentino Lagomarsino, Gianni Liti, Bertrand Llorente, G. Fischer
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Chromosome instability (CIN) and Genetic instability are hallmark features of malignant tumor cells, contributing to tumour progression, drug resistance and metastasis. KIF18A, a member of the kinesin superfamily of motor proteins, is essential for preserving the structural integrity of the bipolar spindle apparatus during cell division. Recent studies have demonstrated that genetic depletion of KIF18A impacts the chromosome alignment, elongation of mitotic spindles, and activates spindle assembly checkpoint in chromosomal instable tumour cells ultimately leading to cell death, while having negligible impact towards the survival of normal cells. These findings suggest that targeting KIF18A represents a promising therapeutic approach for selectively inhibiting the growth of chromosomally instable tumours, while sparing the normal division of euploid cells. We have designed novel macrocyclic KIF18A inhibitors based on our structural understanding from Cryo-EM structures of initial acyclic binders. Structure guided SAR led to the identification of highly potent and selective compounds with optimized properties. Lead compounds exhibited significant inhibition of KIF18A ATPase activity and demonstrated robust cellular anti-proliferative potency in chromosomally instable (CIN-high) high-grade serous ovarian carcinoma cells (HGSOC, OVCAR-3) while retaining a good selectivity over the CIN-low MCF7 cells. This in vitro potency was supported by a significant and dose-dependent upregulation of phospho-Histone H3 and gamma-H2AX, consistent with the induction of mitotic arrest and DNA damage. Further, the lead compounds exhibited encouraging drug-like properties, including acceptable in vitro ADMET profile and excellent in vivo exposure with low clearance resulting in good oral bioavailability in rodents. Additionally, these compounds showed significant anti-tumor efficacy that was associated with robust induction of phospho-histone H3 in the OVCAR3 CDX model. The macrocyclic lead compound is being extensively evaluated in advanced in vitro and in vivo assays to support further development.



 Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh Vishwanath. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, Samiulla D S, Amith A, Aravind A B, Girish C. Daginakatte, Kavitha Nellore, Kishore Narayanan, Chandrasekhar Abbineni, Murali Ramachandra, Susanta Samajdar. Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab5f82e03ba5520dcf239a8a55ec749b43cc168" target='_blank'>
              Abstract A030: Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors
              </a>
            </td>
          <td>
            Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh V. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, Samiulla D S, A. A., A. B, G. Daginakatte, K. Nellore, Kishore Narayanan, C. Abbineni, Murali Ramachandra, S. Samajdar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Checkpoint inhibitors have emerged as a standard of care therapy for non-small cell lung cancer (NSCLC). Mutations in STK11 occur in 20% of NSCLC and are associated with poor responsiveness to anti-PD-1 agents, creating an increased need to identify novel therapeutic targets for this tumor subtype.



 Functional genomic screening utilizing CRISPR-Cas9 is a powerful tool to identify new therapeutic targets in cancer. We applied this technology specifically to STK11-mutant syngeneic tumor models in mice to identify targets which restore sensitivity to anti-PD-1 therapy.



 Knockout of MGAT1, an enzyme critical for maturation of high-mannose N-glycans into hybrid and complex glycan types, was able to reverse STK11-mutant-driven resistance to anti-PD-1 treatment. Parallel in vitro screens using co-culture systems showed that disruption of N-glycosylation was also a powerful mechanism in sensitizing tumor cells to direct killing by antigen-matched CD8 T cells. Our investigation suggests that global disruption of N-glycans likely impacts the ability of T cells to recognize tumor cells. The role of MGAT1 in immune evasion was dependent on its enzymatic activity, making it a promising target for therapeutic small molecule inhibitor development.



 Our work implicates N-glycosylation as a key mediator of immune evasion in STK11 mutant NSCLC and nominates MGAT1 a novel target for therapeutic development to overcome this mechanism.



 Kiera M. Vassallo, Yingnan Chen, Katarzyna B. Handing, Alvin Lu, Samuel Meier, Shangtao Liu, Binzhang Shen, Teng Teng, Yi Yu, Alan Huang, Chengyin Min, Brian Haines, William Mallender, Sining Sun, Jannik Andersen, Wenhai Zhang, Serge Gueroussov. Inhibition of MGAT1 overcomes STK11-driven immune evasion in non-small cell lung cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40d005e425f0a02d6e0f4c38828ede7a7ec1245d" target='_blank'>
              Abstract C103: Inhibition of MGAT1 overcomes STK11-driven immune evasion in non-small cell lung cancer
              </a>
            </td>
          <td>
            Kiera M. Vassallo, Yingnan Chen, Katarzyna B. Handing, Alvin Lu, Samuel R. Meier, Shangtao Liu, B. Shen, T. Teng, Yi Yu, Alan Huang, Chengyin Min, Brian B Haines, W. Mallender, Sining Sun, Jannik N Andersen, Wenhai Zhang, Serge Gueroussov
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Nancy R Zhang, Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M. Shaffer, Drew M Pardoll, Christina Jackson
          </td>
          <td>2025-09-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f58efafad79666afbd80c8e6304e3d7db77f3c" target='_blank'>
              Whole genome CRISPR knockout screen reveals ID3 as a key regulator of myeloma cell survival via TCF3 and c-MYC
              </a>
            </td>
          <td>
            Clara Andersson-Rusch, Md Abu Hanif, Ingrid Quist-Løkken, P. Sætrom, Miłosz Roliński, Johanna Fredrikke Nordstrand Møen, Naiara Campos Roman, K. Misund, V. Beisvåg, P. A. Aas, Barbara van Loon, T. Holien
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>