<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-04 09:40:36 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>48</td>
          <td>44</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="DNA damage from routine cellular processes or exogenous insults can have a lasting impact on gene regulation beyond genetic mutations. The prevailing paradigm for the consequences of DNA damage repair revolves around restoration of the original genetic sequence, but long-term changes in chromatin configuration, gene expression and DNA modifications have not been analyzed. We introduced numerous, simultaneous Cas9-mediated DNA double strand breaks (DSBs) at defined locations in human glioblastoma cells and tracked both non-genetic and genetic alterations over time. Megabase-scale genomic alterations that endured two weeks after the initial damage were detected, involving a shift from transiently increased intra-TAD interactions to persistent long range cis and trans contacts, alterations in gene-expression and associated large structural variations. These findings reveal that widespread DNA damage, such as chemotherapy or radiotherapy, can trigger long-term genetic and non-genetic modifications which alter cellular function and may impact tumor outcome and the emergence of resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Multidrug resistance (MDR) significantly contributes to colon cancer recurrence, making it essential to understand its molecular basis for improved therapies. This study aimed to identify genes and pathways involved in resistance to standard chemotherapeutics by comparing transcriptome profiles of sensitive and paclitaxel-induced MDR colonospheres. Cell viability and growth were assessed following treatment with 5-fluorouracil, oxaliplatin, irinotecan, bevacizumab, and cetuximab. Drug concentrations in culture media posttreatment were measured using high-performance liquid chromatography (HPLC). RNA sequencing (RNA-seq) of untreated sensitive and resistant colonospheres identified differentially expressed genes linked to baseline resistance. Our results confirmed cross-resistance in the resistant model, showing highest oxaliplatin tolerance may involve mechanisms beyond efflux. Transcriptome analysis highlighted upregulation of PIGR and activation of the ribosomal signaling pathway as potential resistance mediators. Notably, AKR1B10, a gene linked to chemotherapeutic detoxification, was overexpressed, whereas genes related to adhesion and membrane transport were downregulated. The overexpression of ribosomal protein genes suggests ribosome biogenesis plays a key role in acquired resistance. These findings suggest that targeting ribosome biogenesis and specific deregulated genes such as PIGR and AKR1B10 may offer promising strategies to overcome MDR in colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6aa3128769906dcf129a6a3196212e61e40551f" target='_blank'>
              Transcriptomic Landscape of Paclitaxel-Induced Multidrug Resistance in 3D Cultures of Colon Cancer Cell Line DLD1
              </a>
            </td>
          <td>
            S. Dragičević, J. Dinić, Milena Ugrin, Marija Vidovic, T. Babic, A. Nikolić
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Calcium signaling is a key controller of numerous cellular events and is intricately linked to many processes that are critical pathways in cancer progression. This review revisits the calcium signaling toolkit in cancer, with a focus on calcium regulation of processes that go beyond the originally defined "classic" hallmarks of cancer such as those associated with proliferation, metastasis, and resistance to cell death pathways. We will consider calcium signaling in the context of the more recently proposed hallmarks of cancer, emerging hallmarks, and cancer-enabling characteristics. This broader examination of calcium signaling and its toolkit members will encompass processes such as metabolic reprogramming, evasion of immune destruction, cellular phenotypic plasticity, senescence, genome instability, and nonmutational epigenetic reprogramming. These cancer features and their interactions with calcium signaling will frequently be analyzed through the lenses of therapy resistance and the complexities of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8339a4a948b497d0310db8cdaa1d952eeb9805ad" target='_blank'>
              Cancer Progression and the Calcium Signaling Toolkit: Expanding Dimensions and Perspectives.
              </a>
            </td>
          <td>
            M. Robitaille, G. Monteith
          </td>
          <td>2025-06-16</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) is the most aggressive form of thyroid cancer. Despite recent advances in treating BRAFV600E-driven ATC, therapy resistance remains a significant challenge, often resulting in disease progression and death. Leveraging a focused CRISPR/KO screen in parallel with a CRISPR/activation screen, both tailored on response to BRAFV600E inhibitor treatment, we identified TAZ (encoded by the WWTR1 gene) deficiency as synthetically lethal with BRAF inhibitor in ATC. TAZ is overexpressed in ATC compared to well-differentiated thyroid tumors. We demonstrate that TAZ-deficient ATC cells display heightened sensitivity to BRAF inhibitors both in vitro and in vivo. Using gene essentiality score across a large panel of cancer cell lines, we found that BRAFV600E-driven cancers are highly sensitive to TAZ loss, unlike their counterparts with wild-type BRAF and non-BRAFV600E. Mechanistically, we demonstrate that dabrafenib triggers the Unfolded Protein Response (UPR) under ER stress and suppresses protein synthesis. TAZ loss represses the UPR, reverses the inhibition of protein synthesis, and triggers increased cell death by ferroptosis in dabrafenib-treated ATC. Collectively, our findings unveil TAZ as a new target to overcome resistance to BRAF inhibitors in undifferentiated thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2537688c884004a06a6382cdb89bfdc63b85ffff" target='_blank'>
              CRISPR-Based Gene Dependency Screens reveal Mechanism of BRAF Inhibitor Resistance in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            Shawn Noronha, Yue Liu, Gaga Geneti, Haojian Li, Xiaolin Wu, David Sun, Vaibhavi Gujar, Takashi Furusawa, Alexei Lobanov, Maggie Cam, Lipika R. Pal, N. Nair, Chi-Ping Day, E. Ruppin, Chandrayee Gosh, Jiangnan Hu, Suresh Kumar, T. Andresson, King Chan, Maura O’Neill, Raj Chari, Y. Pommier, Jaydira Del Rivero, Urbain Weyemi, E. Kebebew, Myriem Boufraqech
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Anticancer chemotherapy is an essential part of cancer treatment, but the emergence of resistance remains a major hurdle. Metabolic reprogramming is a notable phenotype associated with the acquisition of drug resistance. Here, we develop a computational framework that predicts metabolic gene targets capable of reverting the metabolic state of drug-resistant cells to that of drug-sensitive parental cells, thereby sensitizing the resistant cells. The computational framework performs single-gene knockout simulation of genome-scale metabolic models that predicts genome-wide metabolic flux distribution in drug-resistant cells, and clusters the resulting knockout flux data using uniform manifold approximation and projection, followed by k-means clustering. From the clustering analysis, knockout genes that lead to the flux data near that of drug-sensitive cells are considered drug sensitization targets. This computational approach is demonstrated using doxorubicin- and paclitaxel-resistant MCF7 breast cancer cells. Drug sensitization targets are further refined based on proteome and metabolome data, which generate GOT1 for doxorubicin-resistant MCF7, GPI for paclitaxel-resistant MCF7, and SLC1A5 as a common target. These targets are experimentally validated where treating drug-resistant cancer cells with small-molecule inhibitors results in increased sensitivity of drug-resistant cells to doxorubicin or paclitaxel. The applicability of the developed framework is further demonstrated using drug-resistant triple-negative breast cancer cells. Taken together, the computational framework predicts drug sensitization targets in an intuitive and cost-efficient manner and can be applied to overcome drug-resistant cells associated with various cancers and other metabolic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7052bad828f26940bd1911c2893c8896c16d13b8" target='_blank'>
              Genome-scale knockout simulation and clustering analysis of drug-resistant breast cancer cells reveal drug sensitization targets.
              </a>
            </td>
          <td>
            JinA Lim, Hae Deok Jung, S. Park, Moonhyeon Jeon, Da Sol Kim, Ryeongeun Cho, Dohyun Han, Han Suk Ryu, Yoo-Ree Kim, Hyun Uk Kim
          </td>
          <td>2025-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c251c2e80393955c5ed269cb0f5004e71c3f6" target='_blank'>
              Expanding the utility of transcriptome analysis for mutation detection in high-risk childhood precision oncology
              </a>
            </td>
          <td>
            Chelsea Mayoh, P. Barahona, Angela Lin, Lujing Cui, P. Ajuyah, A. Altekoester, L. Lau, Dong-Anh Khuong-Quang, Patricia Sullivan, A. Senapati, S. Nagabushan, Ashleigh Sullivan, Natacha Omer, Andrew S Moore, Wayne Nicholls, R. Endersby, N. Gottardo, G. McCowage, L. dalla Pozza, Jordan R Hansford, S. Khaw, Paul J Wood, T. Trahair, G. Marshall, D. S. Ziegler, Vanessa Tyrrell, Michelle Haber, Marie Wong, P. Ekert, Mark J. Cowley
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Epigenetic modifications provide a mechanism through which environmental factors can induce heritable changes in gene expression without altering the underlying DNA sequence. While the Modern Synthesis emphasises genetic mutation and natural selection as the primary drivers of evolution, growing evidence highlights the role of DNA methylation, histone modifications, and non-coding RNAs in enabling phenotypic plasticity and transgenerational inheritance. These mechanisms allow organisms to respond to environmental conditions more rapidly than genetic changes alone would permit, and in some cases, these responses are transmitted across generations via the germline. Empirical studies from both plant and animal systems demonstrate how exposures to stress, nutritional changes, and chemical disruptors can leave stable epigenetic marks. Additionally, large-scale genomic rearrangements such as chromoanagenesis point to the potential for abrupt structural variation to interact with epigenetic processes. Incorporating such findings within the framework of the Extended Evolutionary Synthesis expands traditional evolutionary models, offering a more integrated understanding of how environmental stimuli shape heritable variation and influence evolutionary trajectories over both short and long timescales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa60e2219c1da1273bd416f9b243ed47b34a3d3" target='_blank'>
              The Role of Epigenetic Modifications in Evolution: Understanding How Environmental Factors Shape Heritable Gene Expression Across Generations
              </a>
            </td>
          <td>
            Damla Naz Haymanalı
          </td>
          <td>2025-07-10</td>
          <td>Next Generation Journal for The Young Researchers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance and distant metastases are leading causes of mortality in colorectal cancer (CRC), yet the molecular mechanisms linking these processes remain elusive. In this study, we demonstrate that acquired resistance to oxaliplatin, a first-line chemotherapeutic in CRC, enhances metastatic potential through transcriptional reprogramming. Using a clinically relevant dosing regimen, we generated oxaliplatin-resistant CRC cells that displayed increased metastatic potential. Integrated transcriptomic and phenotypic analyses revealed that dysregulated cholesterol biogenesis amplifies TGF-β signaling, which in turn drives expression of SERPINE1, which serves as a key effector of both oxaliplatin resistance and metastasis. Furthermore, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that robustly predicts overall and relapse-free survival across distinct patient cohorts. Notably, RESIST-M stratifies a high-risk subtype of CMS4/iCMS3-fibrotic patients that display the poorest prognosis, underscoring its clinical relevance. Targeting of SERPINE1 or cholesterol biosynthesis re-sensitized resistant, pro-metastatic cells to oxaliplatin in mouse xenograft models. Altogether, this study uncovers a mechanistic link between metabolic rewiring and transcriptional plasticity underlying therapy-induced metastasis in primary CRC. Additionally, it also reveals actionable vulnerabilities that offer both prognostic value and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461233a4140dd31d355e46865c3174f542e2818f" target='_blank'>
              Modeling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Kenzom Tenzin, Niranjan Shirgaonkar, Huiwen Chua, Lin Xuan Chee, Ahpa Sae Yeoh, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, Ethan Subel, M. Thangavelu, Jane Vin Chan, Choon-Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj Dasgupta
          </td>
          <td>2025-07-16</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cellular senescence, a state of irreversible cell cycle arrest accompanied by a senescence‐associated secretory phenotype (SASP), plays dual roles in cancer biology. Initially recognized as a tumor‐suppressive mechanism by halting the proliferation of damaged cells, senescence paradoxically fosters tumor progression through SASP‐mediated immunosuppression and chronic inflammation. Thus, the role of senescent cells in tumors still needs to be further elucidated. Our review comprehensively examines the triggers and molecular pathways of senescence. We also summarized the characteristics and functions of senescent tumor and nontumor cells, delineating the heterogeneous tumor senescence microenvironment. Here, we highlight the functional reprogramming of senescent cells, including enhanced stemness, secretome and metabolome reprogramming, which can sustain tumorigenesis and therapeutic resistance. Furthermore, we discuss emerging therapeutic strategies, notably the “one‐two punch” approach to overcome therapy resistance. By integrating recent advances in senescence‐targeted therapies, our review underscores the necessity of context‐specific strategies to harness senescence's tumor‐suppressive effects while mitigating its protumorigenic consequences. These insights provide a roadmap for developing precision therapies and refining biomarker‐driven approaches to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06af336e646768d23ecd9b6a5c9859e5e3492bc9" target='_blank'>
              Cellular Senescence in Cancer: Mechanisms, Roles in Tumor Progression, and Therapeutic Implications
              </a>
            </td>
          <td>
            Jingrui Yan, Yu Zhang, Guohua Mao, Jun Yu, Tianxing Zhou, Jihui Hao
          </td>
          <td>2025-07-07</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Balanced chromosomal abnormalities (BCAs) are structural variations that can underlie a wide spectrum of neurodevelopmental disorders, often remaining undetected by conventional diagnostic approaches. Whole-genome sequencing (WGS) allows for base-pair resolution of structural variants across the entire genome, making it a powerful tool to detect cryptic chromosomal rearrangements and refine breakpoint mapping. RNA sequencing (RNA-Seq), by enabling the detection of gene expression changes and fusion transcripts, provides complementary functional insights into the consequences of genomic alterations. This study integrated WGS and RNA-Seq to precisely characterize the breakpoints and assess the functional impact of de novo BCAs in two unsolved cases of Neurodevelopmental Disorders. Materials and methods Short read WGS was used to identify the chromosomal breakpoints and gene disruptions caused by BCAs. RNA-Seq on blood RNA was employed to detect differential gene expression and potential fusion transcripts of disrupted genes. Results In the first case, the inversion inv(8) (p11.2q13) disrupted two genes at the breakpoints, namely, CHD7 and SLC20A2. These genes are in opposite orientations, and the inversion realigned them in the same direction, generating two novel fusion genes. Disruption of CHD7 confirmed the suspected diagnosis of CHARGE syndrome. The interruption of SLC20A2, commonly associated with neurological symptoms, prompted further clinical evaluation. RNA-Seq identified in-frame fusion transcripts from the chimeric genes in the blood, suggesting a potential deleterious phenotypic effect. In the second case, WGS revealed a balanced translocation t(17; 22) (q25; q13) that disrupted EP300 at 22q25, confirming Rubinstein-Taybi syndrome. The concurrent disruption of RBFOX3 at 17q13 suggested additional neurological implications, particularly related to epilepsy. Transcriptomic analysis demonstrated the monoallelic and significantly reduced expression of EP300. Conclusion These findings highlight the crucial role of WGS in identifying disease-associated BCAs and underscore the complementary value of RNA-Seq in assessing their functional consequences. This integrated approach enhanced diagnostic accuracy and clinical management, paving the way for more comprehensive and personalized care in these two patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8f54f85ebf927d3f8b7d5f4a6236e7f717fa51" target='_blank'>
              Integrating genome and transcriptome analysis to decipher balanced structural variants in unsolved cases of neurodevelopmental disorders
              </a>
            </td>
          <td>
            S. Mellone, Alice Spano, Denise Vurchio, Giulia Borgonovi, Alessandro Ugonotti, Giulia Paglino, Alba Bianco, Sara Ronzani, Maurizio Sciancalepore, Flavia Prodam, Amanda Papa, Maurizio Viri, Umberto Dianzani, Mara Giordano
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In ~70% of patients with pancreatic ductal adenocarcinoma, the TP53 gene acquires gain-of-function (GOF) mutations leading to rapid disease progression. Specifically, missense p53 (misp53) GOF mutations associate with therapy resistance and worse clinical outcomes. However, the molecular functions of distinct misp53 mutants in plasticity and therapy response remain unclear. Integrating multicenter patient data and multi-omics, we report that the misp53R273H/C mutant is associated with cell cycle progression and a basal-like state compared to the misp53R248W/Q mutant. Loss of misp53R273H/C decreased tumor growth and liver metastasis while prolonging survival in preclinical models. We found that misp53R273H/C specifically regulated the Rb/DREAM axis involved in cell cycle regulation. Notably, a clinical CDK4/6 inhibitor reduced misp53R273H/C mutant expression. However, it triggered MAPK/ERK-mediated resistance mechanisms, enhancing cell survival and resistance to CDK4/6 inhibitors. Combining MAPK/ERK and CDK4/6 inhibitors reduced misp53R273H/C-associated oncogenic functions. Thus, distinct misp53 mutants show unique cell-intrinsic plasticity, therapeutic vulnerabilities, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8366ad2cf60feed0070f18be69dfac6ab275f732" target='_blank'>
              TP53 missense–specific transcriptional plasticity drives resistance against cell cycle inhibitors in pancreatic cancer
              </a>
            </td>
          <td>
            Laura Urbach, Lena Wieland, Frederike Penz, Rebecca Diya Samuel, Stefan Küffer, Lukas Klein, Christof Lenz, Ulrich Sax, Michael Ghadimi, Ramona Schulz-Heddergott, E. Hessmann, V. Ellenrieder, Nelson Dusetti, Shiv K. Singh
          </td>
          <td>2025-07-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95eff8844cf7d953e515c95691ab37e19818828b" target='_blank'>
              Longitudinal pharmacogenomic analysis of refractory lung cancer to identify therapeutic candidates for epidermal growth factor receptor-tyrosine kinase inhibitor resistance subclones.
              </a>
            </td>
          <td>
            Namhee Yu, Mihwa Hwang, Beung chul Ahn, Youngjoo Lee, Sehwa Hong, Hanna Sim, Bo Ram Song, Sunshin Kim, Charny Park, Ji-Youn Han
          </td>
          <td>2025-07-04</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Here we present a new iPSC resource of apparently healthy subject biospecimens available to the research community through the National Institute of General Medical Sciences Human Genetic Cell Repository (NIGMS Repository). This resource includes five iPSCs and matched parental cell lines with accompanying publicly available, HiFi whole-genome sequencing data. Structural variant (SV) and single nucleotide variant (SNV) concordance between iPSC and parental lines was generally high; however, we found a notable reduction in concordance between the iPSC reprogrammed with retroviral reprogramming and its parental line consistent with previous work showing newer Sendai approaches to be more robust in preserving genomic integrity. This iPSC resource additionally includes pharmacogenomic and human leukocyte antigen (HLA) gene annotations as well as a set of user-friendly, web-based search tools to visualize and explore SVs and SNVs. This new resource is designed to offer a highly characterized set of in vitro models for research into cell-type specific functional characterization of genetic, genomic and pharmacogenomic variation. More generally, these renewable biospecimens and genomic data search tools are available to the scientific community to support high-quality and reproducible biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/664bbdaaf704bf6f26d8a71b37d0bd99ff2c3655" target='_blank'>
              New iPSC resource with long-read whole genome sequencing characterizations for enhanced in vitro modeling
              </a>
            </td>
          <td>
            Laura Scheinfeldt, Anthony Pompetti, G. Calendo, Tatyana Pozner, Christine Grandizio, Gretchen Smith, Kelly Hodges, Neda Gharani, D. Kusic, Matthew W Mitchell, N. Turan
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer (BC) is a heterogeneous disease that can be molecularly classified based on the expression of the ERBB2 receptor (also known as HER2) and hormone receptors. Targeted therapies for HER2-positive BC, such as trastuzumab, antibody-drug conjugates (ADCs) and tyrosine kinase inhibitors, have improved patient outcomes but primary/acquired resistance still pose challenges that can limit treatments' long-term efficacy. Addressing these obstacles is vital for enhancing therapeutic strategies and patient care. Alternative splicing, a post-transcriptional mechanism that enhances transcript diversity (isoforms), can produce proteins with varied functions, cellular localizations, or binding properties. Here, we comprehensively characterized the HER2 alternative splicing isoforms, assessed their expression in primary BC patients and cell lines, and explored their role in resistance to anti-HER2 therapies. We expanded the catalog of known HER2 protein-coding isoforms from 13 to 90, revealing distinct patterns of protein domains, cellular localizations, and protein structures, along with their antibody-binding sites. By profiling expression in 561 primary BC samples and mass spectrometry data, we discovered a complex landscape of HER2 isoform, revealing novel transcripts that were previously unrecognized and are not assessed in routine clinical practice. Finally, the assessment of HER2 isoform expression in BC cell cultures sensitive or resistant to trastuzumab and ADCs revealed that drug-resistant cells shifted their expression toward isoforms lacking antibody-binding domains. Our results broaden the understanding of HER2 isoforms, revealing distinct mechanisms of potential resistance to anti-HER2 therapies, particularly ADCs. This expanded landscape of HER2 isoforms emphasizes the crucial role of alternative splicing investigations in advancing precision-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde1cb673888229ad9d05dcf1add5eeba27a1400" target='_blank'>
              Alternative splicing generates HER2 isoform diversity underlying antibody-drug conjugate resistance in breast cancer.
              </a>
            </td>
          <td>
            G. Guardia, C. D. Dos Anjos, Aline Rangel-Pozzo, F. F. dos Santos, A. Birbrair, P. Asprino, A. A. Camargo, P. Galante
          </td>
          <td>2025-07-15</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54dd2ec057c7a25df4bed262e9ae15c8f28acaf6" target='_blank'>
              Mapping the Transcriptional Landscape of Drug Responses in Primary Human Cells Using High-Throughput DRUG-seq
              </a>
            </td>
          <td>
            Lauren Baugh, Sébastien Vigneau, Srijani Sridhar, Sarah Boswell, George Pilitsis, John Bradley, Olga Allen, Laura M. Rand, Amanda Riccardi, Kelsey Roach, Kevin Lema, Ayla Ergun
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer remains a prevalent and diverse disease, significantly contributing to cancer- related deaths among women worldwide. Recent advancements in molecular biology have paved the way for targeted therapies and pharmacogenomics, which are crucial for developing personalized treatment strategies. This literature review synthesizes findings from recent studies on these approaches, emphasizing clinical trials, genomic profiling, and personalized medicine. It aims to focus on studies examining targeted treatments, such as human epidermal growth factor receptor- 2 (HER2) inhibitors and CDK4/6 inhibitors, alongside pharmacogenomic data that influence drug metabolism, efficacy, and toxicity. Additionally, it examines the role of gene SNPs (Single Nucleotide Polymorphisms) correlated with treatment resistance, which have emerged as key biomarkers affecting therapeutic outcomes in breast cancer. These SNPs, found in genes involved in drug metabolism and tumor progression, contribute to variability in treatment responses and resistance in specific subtypes. They encompass various breast cancer subtypes, including hormone receptorpositive (HR+), HER2-positive, and triple-negative breast cancer (TNBC). The targeted therapies, particularly HER2 inhibitors, have markedly improved outcomes for specific subtypes. Furthermore, pharmacogenomics personalizes treatment by identifying genetic variations that affect drug response, optimizing therapy selection, and minimizing adverse effects. Despite these advancements, drug resistance remains a significant challenge, highlighting the necessity for ongoing research in molecular diagnostics and innovative therapeutic combinations. The literature suggests that precision medicine, driven by genomic profiling, pharmacogenomic data, and single nucleotide polymorphisms (SNPs) analysis, is enhancing treatment efficacy for breast cancer patients. HER2- positive and HR+ patients have especially benefitted from these targeted therapies while emerging treatments are addressing the complexities of TNBC. Additionally, genetic testing, such as BRCA1/2 mutation screening, is vital for guiding treatment decisions. Targeted therapies and pharmacogenomics have revolutionized breast cancer treatment, providing more personalized and effective care. Nevertheless, overcoming drug resistance and expanding access to genomic testing are essential for future advancements in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd705c1549c5bd8681f0a403abe74109eee0036" target='_blank'>
              Advancements in Targeted Therapies and Pharmacogenomics for Personalized Breast Cancer Treatment: The Role of Gene SNPs in Treatment Resistance.
              </a>
            </td>
          <td>
            D. Tripathi, Neal M Davies, P. S. Rajinikanth, Prashant Pandey
          </td>
          <td>2025-06-27</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Molecular genetic diagnoses are critical to prevention and treatment of inherited polyposis and colorectal cancer. 19 genes responsible for these conditions are known, but many severely affected patients and families remain unsolved. Cryptic intronic variants that alter splicing of these genes and incomplete characterisation of recessive predisposition contribute to these diagnostic gaps. Methods Adaptive sampling long-read DNA sequencing targeted to 19 colon cancer genes, paired with direct long-read RNA whole-transcriptome sequencing, was undertaken for four patients referred for deficiency of mismatch repair proteins and/or familial polyposis, for whom multigene panel testing yielded negative or uncertain germline results. Results Genetic diagnoses were obtained for all four patients. Each patient carried a cryptic intronic germline variant in a different colon cancer gene. The variants abrogated splicing by various mechanisms, all leading to loss of gene function. Patient 1 was heterozygous for intronic insertion into MSH2 of an Alu element, leading to extension of transcription into the affected intron and a stop. Patient 2 was heterozygous for deep intronic insertion into APC of a Long Interspersed Nuclear Element (LINE), creating a pseudoexon and a stop. Patient 3 was compound heterozygous at MLH3, including a cryptic intronic substitution leading to exon skipping and a stop. Patient 4 was compound heterozygous at MUTYH, including a deep intronic deletion yielding an extremely short intron and transcriptional loss of an exon encoding a critical protein domain. Conclusion Paired long-read DNA and RNA sequencing can enhance diagnostic yield through detection of cryptic intronic variants that impact cancer predisposition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5066c2b7365798ed65ee7d8872b1b51cf8054789" target='_blank'>
              Long-read DNA and RNA sequencing for inherited polyposis and colorectal cancer: cryptic intronic variants and multiple mutational mechanisms
              </a>
            </td>
          <td>
            A. Jacobson, Amal AbuRayyan, S. Gulsuner, Haley Slater, Yagiz Anasiz, Sirajummuneer M Ahmad, Ming K. Lee, Jessica B Mandell, Emily J Rettner, E. Konnick, Colin C Pritchard, M. King, T. Walsh, Brian H Shirts
          </td>
          <td>2025-06-26</td>
          <td>Journal of Medical Genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eaa0126e7f9afd60d57ab46895cddf6ea4cbe3" target='_blank'>
              Pangenome-based identification of cryptic pathogenic variants in undiagnosed rare disease patients
              </a>
            </td>
          <td>
            S. S. Jang, S. Kim, S. Lee, S. Kim, J. Moon, J. Kim, J. Chae
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Dysregulated RNA modifications contribute to cancer progression and therapy resistance, yet the underlying mechanism often remains unknown. Here, we perform CRISPR-based synthetic lethality screens to systematically explore the role of RNA modifications in mediating resistance to anti-leukaemic drugs. We identify the TRMT5-mediated formation of N1-methylguanosine (m1G) in the tRNA anticodon loop as essential for mediating drug tolerance to cytarabine and venetoclax in acute myeloid leukemia (AML). TRMT5 methylates nearly all mitochondrial and nuclear tRNAs with a guanosine at position 37, but its role in promoting drug tolerance specifically depends on its mitochondrial function. TRMT5 is essential for the dynamic upregulation of mitochondrial mRNA translation and oxidative phosphorylation (OXPHOS), which are critical for sustaining drug tolerance in leukemia cells. This mitochondrial dependency correlates with therapy outcomes in leukemia patients: lower expression of electron transport chain genes is linked to poorer outcomes in a cohort of nearly 100 AML patients undergoing first induction therapy. Finally, we demonstrate that targeted depletion of TRMT5 protein using a conditional degron, in conjunction with cytarabine and venetoclax treatment, synergistically induces cell death in drug-tolerant AML cells. Thus, our study reveals TRMT5 as promising drug target for therapy-resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f65b910416a4ff6ee4895803cf1d0c098e3d11" target='_blank'>
              Disrupting tRNA modifications to target mitochondrial vulnerabilities in drug-resistant leukemia cells.
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, Sarah Mn Zimmermann, Laura Werner, D. Heid, Fu Xu, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Laia Llovera, Ollivier Alexane, Anke Heit-Mondrzyk, U. Platzbecker, C. Baldus, H. Serve, M. Bornhäuser, C. Vågbø, S. Benitah, Jeroen Krijgsveld, Eva Maria Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2025-08-01</td>
          <td>Blood</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="This study investigated whether therapeutic X-ray doses can affect the expression of mismatch repair (MMR) genes and proteins using Lynch syndrome-associated human colorectal cancer cell lines. MMR-deficient cell lines (HCT116, SW48, LoVo) and an MMR-proficient control cell line (HT29) were exposed to X rays [a 2 Gy dose or 2 Gy daily for five consecutive days (10 Gy)]. Reverse transcription quantitative real-time PCR (RT-qPCR) and Western blotting were used to detect the radiation-induced changes in the expression of RNAs and proteins, respectively. RT-qPCR revealed that MLH1 and MSH6 genes were stably expressed regardless of the MMR status of the cell line and the radiation dose. In contrast, the MSH2 gene was either up-regulated or down-regulated after 2 Gy or 10 Gy or both. The expression of PMS2 increased after 10 Gy irradiation in all MMR-deficient cell lines, even though the data were not statistically significant compared to other doses, except for the LoVo cell line. Protein expression analysed using Western blotting demonstrated that MLH1 protein expression was stable, whereas the expression of MSH2 was significantly affected by radiation exposure in both MLH1-deficient cell lines. No correlation between the expression of RNA and protein could be identified. In conclusion, radiation may have significantly differential effects on MMR RNA and protein expression when different cell lines, doses, and specific genes are considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b8835db8d3ae7872bbc2b8269596b668e92380" target='_blank'>
              The Effect of X rays on the Expression of Mismatch Repair Genes and Proteins in Lynch Syndrome Associated Human Colorectal Cancer Cell Lines.
              </a>
            </td>
          <td>
            Mingzhu Sun, L. Cruz-Garcia, Danny Freestone, Kevin Monahan, C. Badie, Yannick Comoglio, Hannah Mancey, J. Moquet, Stephen Barnard
          </td>
          <td>2025-06-20</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer therapy resistance remains a major challenge, with limited resources available for systematically studying its underlying mechanisms at the patient level. The existing databases are either restricted to bulk RNA-seq data, lack single-cell resolution, or provide limited clinical annotations, making them insufficient for in-depth exploration of the tumor microenvironment (TME) dynamics in therapy resistance. To bridge this gap, we present CellResDB, a patient-derived platform comprising nearly 4.7 million cells from 1391 patient samples across 24 cancer types. CellResDB provides comprehensive annotations of TME features linked to therapy resistance. To enhance accessibility, we include an intelligent robot, CellResDB-Robot, which facilitates intuitive data retrieval and analysis. In summary, CellResDB represents a valuable resource for cancer therapy and provides an experimental protocol for applying large language models (LLMs) within the biomedical database. CellResDB is freely available at https://cellknowledge.com.cn/cellresponse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e259dc3f38a852bd1ee92860ed4d64506f3ee24b" target='_blank'>
              Deciphering cancer therapy resistance via patient-level single-cell transcriptomics with CellResDB
              </a>
            </td>
          <td>
            Tianyuan Liu, Huiyuan Qiao, Liping Ren, Xiucai Ye, Quan Zou, Yang Zhang
          </td>
          <td>2025-07-15</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Activation of oncogenes, such as through mutations in Kirsten rat sarcoma viral oncogene homolog (KRAS), triggers profound disruptions in cellular homeostasis that set off a cascade of stress responses. These responses enable cells to cope with the array of challenges encountered during tumorigenesis by activating defense mechanisms that promote adaptation and survival. Key components of this oncogenic stress response include heat shock proteins, the ubiquitin-proteasome system, autophagy, nuclear factor erythroid 2-related factor 2–antioxidant response element signaling, DNA damage response proteins, p53, redox-regulating proteins, and stress granules. This review concentrates on KRAS-driven oncogenic transformation, as KRAS mutations are among the most common in human cancers, accounting for over 90% of pancreatic ductal adenocarcinoma cases, around 30% of lung cancers, and approximately 50% of colorectal cancers. We examine the intricate molecular interplay between oncogenic stress and the associated cellular defense mechanisms, emphasizing the key molecular events that follow KRAS activation. Importantly, the very pathways that allow cancer cells to adapt to oncogenic stress also offer novel therapeutic opportunities. By selectively targeting pivotal regulators within these stress response pathways, we can potentially disrupt the survival mechanisms of cancer cells. This strategy not only promises to enhance the effectiveness of existing treatments but also paves the way for the development of innovative therapies designed to combat tumor progression. In essence, exploiting oncogenic stress responses represents an original and promising therapeutic approach in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a19b0d4dfe04d02b6a9051c867a7e87fe82159" target='_blank'>
              Oncogenic stress response mechanisms as new therapeutic targets in cancer treatment: A review
              </a>
            </td>
          <td>
            J. Iovanna, Matías Estaras, Daniel Grasso, Martin E. Fernández Zapico, J. L. Neira, P. Santofimia-Castaño
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Breast cancer pathogenesis is closely linked to the mechanisms of cellular reprogramming, the process of converting differentiated cells into undifferentiated or different cell types. This reprogramming is achieved through the orchestrated expression of transcription factors which alter the cell’s existing chromatin landscape and establish a transcriptome corresponding to the reprogrammed cell state. Pioneer factors, a type of transcription factor, initiate cell development and cell fate transition by binding to specific DNA motifs within nucleosomes. This binding leads to the recruitment of co-factors, such as nucleosome remodelers, leading to chromatin opening and subsequent activation of specific genes essential for reprogrammed cell function. Improper engagement of these factors can lead to unanticipated or harmful gene expression, potentially contributing to various human diseases, including cancer. GATA3, a pioneer factor, is essential for normal mammary gland development and differentiation. In breast cancer cells, GATA3 has been shown to suppress tumor growth by inducing mesenchymal-to-epithelial transition (MET), the reverse process of epithelial-to-mesenchymal transition (EMT). Interestingly, although mutations in GATA3 are frequent and considered driver mutations, breast cancer patients carrying GATA3 mutations are generally associated with better patient survival. The roles of these mutations remain largely unexplored. M294R and M294K are hot spot missense mutations found in luminal breast cancer patients. These mutations occur in the DNA-binding domain of GATA3, which is crucial for its function as a transcription factor. For functional characterization of these mutants, we utilized MDA-MB-231 mesenchymal cells, a highly aggressive and invasive breast cancer cell line to model mesenchymal-to-epithelial transition (MET). Since ectopic expression of GATA3 induces MET in this cell line, these cells provide an excellent system to observe phenotypic and molecular alterations caused by its missense mutations M294R/K. To quantitatively measure changes in GATA3 mobility and its DNA binding dynamics in living cells, we utilized Fluorescence Recovery After Photobleaching (FRAP) assay. Additionally, to further elucidate the crucial role of GATA3, genomics analyses and phenotypic assays were conducted. Functional characterization of M294R/K mutants in MDA-MB-231 breast cancer cells demonstrated distinct phenotypic and molecular alterations compared to wild-type GATA3. While wild-type GATA3 induced MET, mutant-expressing cells exhibited differential responses. Remarkably, mutants derived from breast cancer patients displayed increased chromatin binding, leading to a tighter chromatin structure and differential gene expression of mesenchymal or epithelial markers. Altered cellular phenotypes such as cell migration are also observed in M294R/K expressed cells. Our finding underscores the importance of precise regulation of pioneer factor-chromatin interactions in cellular reprogramming processes. Understanding the molecular mechanisms underlying GATA3 mutations and their effects on cellular reprogramming in breast cancer provides valuable insights into the roles of GATA3 in breast cancer progression.
 Citation Format: Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku. Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ca95880c3c403f49e5578c8d1dc155a7b3a608" target='_blank'>
              Abstract P5-06-09: Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model
              </a>
            </td>
          <td>
            Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The evolution of techniques used to identify structural variants (SVs) and copy number variants (CNVs) in genomes have seen significant development in the last decade. With the growing use of more technologies including chromosomal microarray, genome sequencing and genome mapping in clinical cytogenetics laboratories, reporting the frequency of SVs and CNVs has increased the complexity of genomic results. In conventional testing (e.g. karyotype or FISH) individual cells are analyzed and abnormalities are reported at the single cell level directly as a proportion of the analyzed cells. Whereas for bulk genome assays structural and sequence changes are often reported as variant allele frequencies and fractional copy number states. The International System of Cytogenomic Nomenclature (ISCN) recommends converting these values into a “proportion of the sample”, which requires different calculations and underlying assumptions based on the data type. This review illustrates how the different methods of interpreting and reporting data are performed and identifies challenges in the conversion of these values to a proportion of the sample. We stress the need for careful interpretation of data with consideration for factors that may alter how proportions are reported including overlapping SVs and CNVs or regions with acquired homozygosity. We also demonstrate, using validation data of SVs and CNVs tested by multiple techniques how results are largely consistent across methodologies, but can show dramatic differences in rare circumstances. This review focuses on illustrating many of the challenges with aligning reporting using different techniques and their underlying assumptions. As hematologic disease classifications start to incorporate numeric limits (e.g. VAF defining thresholds), it is important for laboratory geneticists, pathologists and clinicians to appreciate the differences in methodologies, potential pitfalls and the nuances when comparing bulk genome analyses to the more conventional single cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883b8047db9fe9c01ef1b4982d92e6a1387b4be0" target='_blank'>
              What the VAF? A guide to the interpretation of variant allele fraction, percent mosaicism, and copy number in cancer
              </a>
            </td>
          <td>
            Adam C Smith, Hubert Tsui, Sila Usta, Jose-Mario Capo-Chichi
          </td>
          <td>2025-07-08</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer is one of the most common malignant tumors occurring in the male reproductive system. Androgen deprivation therapy (ADT) is the primary treatment for prostate cancer. However, approximately 10–20% of advanced prostate cancer patients progress to castration-resistant prostate cancer (CRPC) within five years, posing a significant challenge to treatment. There is an urgent need to explore the causes and mechanisms of prostate cancer progression, and to identify new therapeutic targets and predictive markers. Utilize RNA-seq to screen for differentially expressed genes and alternative splicing events between androgen-sensitive prostate cancer cells and castration-resistant prostate cancer cells. Perform bioinformatics analysis on the differentially expressed genes. The expression levels of these genes were confirmed by real-time quantitative PCR (RT-qPCR); alternative splicing events related to castration resistance were verified through RT-PCR. The impact of the S100A6 gene on the proliferation and growth capabilities of PC3 and DU145 cells was analyzed using CCK8, colony formation, and EdU assays. Cell migration was assessed using a wound healing assay. The RNA-seq analysis results showed that there were 4,830 differentially expressed genes in non-hormone-sensitive prostate cancer. GO enrichment analysis of these differentially expressed genes revealed significant enrichment in tumor-related signaling pathways such as regulation of cell cycle process, cell division, and regulation of cell projection organization. Further RT-qPCR detection results indicated that the mRNA expression of four genes, including S100A6, was upregulated, consistent with the transcriptome sequencing results. Downregulation of S100A6 expression significantly inhibited the proliferation and migration of non-hormone-dependent prostate cancer. In this study, we also identified many splicing alterations in cancer-related genes, with castration resistance-induced alternative splicing events closely related to the interaction networks of regulation of cell cycle process, cell division, and regulation of cell projection organization. Our research findings suggest that S100A6 may serve as a molecular marker or therapeutic target associated with castration resistance in prostate cancer, providing new insights into the molecular basis of this disease. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-11397-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8350b9dbff378bc43ca0f122bb3ffa5b66be6348" target='_blank'>
              Screening and validation of key genes involved in castration-resistant prostate cancer based on transcriptomics sequencing
              </a>
            </td>
          <td>
            Guangquan Tong, Xiaomeng Jing, Yuning Yang, Xiaowei Wang, Junlin Lu, Jingyi Hu, Yang Wang, F. A. Jumuddin, Wenjing Zhang, Yuesheng Lv
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer remains one of the leading health threats globally, with therapeutic resistance being a long-standing challenge across chemotherapy, radiotherapy, targeted therapy, and immunotherapy. In recent years, the association between epigenetic modification abnormalities and therapeutic resistance in tumors has garnered widespread attention, spurring interest in the development of approaches to target epigenetic factors. In this review, we explore the widespread dysregulation and crosstalk of various types of epigenetic modifications, including DNA methylation, histone modifications, and non-coding RNA changes, which interact through complex regulatory networks in tumors. Clinically, single-targeted therapy based on epigenetic modification usually has its limited effect against cancer. However, the combination of epigenetic drugs with other treatment modalities, such as chemotherapy, targeted therapy, or immunotherapy, shows potential for synergistically enhancing efficacy and reducing drug resistance. Therefore, we evaluate the possibility and potential mechanisms of targeting epigenetic modifications to overcome resistance in cancer therapy, and discuss the challenges and opportunities in moving epigenetic therapy into clinical practice. Moreover, the application of multi-omics technologies will aid in identifying core epigenetic factors from complex epigenetic networks, enabling precision treatment and overcoming therapeutic resistance in tumors. Furthermore, the development of spatial multi-omics technologies, by providing spatial coordinates of cellular and molecular heterogeneity, revolutionizes our understanding of the tumor microenvironment, offering new perspectives for precision therapy. In summary, the combined application of epigenetic therapies and the integration of multi-omics technologies herald a new direction for cancer treatment, holding the potential to achieve more effective personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8def3b2fb87b962d999f696cd613916b46b863b4" target='_blank'>
              Targeting epigenetic regulators as a promising avenue to overcome cancer therapy resistance
              </a>
            </td>
          <td>
            Jiawei Song, Ping Yang, Canting Chen, Weiqun Ding, O. Tillement, Hao Bai, Shuyu Zhang
          </td>
          <td>2025-07-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cancer treatments that use drugs resembling DNA building blocks, known as nucleotide analogs, that insert themselves into DNA and prevent replication often stop working over time because cancer cells become resistant. This review explains how reversible epigenetic changes help cancer cells avoid being killed by these drugs. These changes do not alter the DNA sequence but can turn important genes on or off. We describe how cancer cells use these mechanisms to repair damaged DNA, pump drugs out of the cell, or break them down. We highlight the role of DNA packaging proteins (histones), small RNA molecules, and chemical tags on DNA that control how genes behave. Understanding these changes is important because it may allow clinicians to resensitize tumors and make them sensitive to treatment again and prevent relapse. We also explore new therapies, including drugs and gene editing tools, that target these epigenetic changes to improve the success of chemotherapy. These findings could lead to better outcomes for patients with difficult-to-treat cancers, especially those with pancreatic, colorectal, or breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeef0502ae8a946f9634749621e6cdd28f4f464" target='_blank'>
              Epigenetic Drivers of Chemoresistance in Nucleobase and Nucleoside Analog Therapies
              </a>
            </td>
          <td>
            John Kaszycki, Minji Kim
          </td>
          <td>2025-07-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c33305648009e1896ce514a1bce98da6447d96" target='_blank'>
              Epigenetic dysregulation in cancer: mechanisms, diagnostic biomarkers and therapeutic strategies.
              </a>
            </td>
          <td>
            Komal Imran, Muhammad Javed Iqbal, Muhammad Mushtaq Ahmed, Ammara Khalid, H. Cortés, O. Reyes-Hernández, Gabriela Figueroa-González, G. Leyva-Gómez, L. Falzone, Massimo Libra, Federica Longo, Javad Sharifi‐Rad, D. Calina
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with metastatic potential. Single-cell RNA sequencing revealed malignant cells segregated into Melanocytic and Neural Crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the Melanocytic toward the Neural Crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive Class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies. SIGNIFICANCE STATEMENT We developed a mouse model that resembles the genetic progression and phenotypic plasticity of human UM. This spatially controlled model confirms the critical role of driver mutations in GNAQ and BAP1, proposes MYC as a promoter of malignant transformation in coordination with other chromosome 8q genes, and reveals UM progression through distinct cellular states. This model offers an urgently needed preclinical platform for understanding the immunogenomics of UM and for testing immune and targeted treatments for this lethal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35df183db757e5d1e691fe22c0b60fdf40bee0e5" target='_blank'>
              A multi-step immune-competent genetic mouse model reveals phenotypic plasticity in uveal melanoma
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, J. Dollar, Xiao Liu, Neel Jasani, Benjamin Posorske, S. Sriramareddy, Vinesh Jarajapu, J. Kuznetsoff, John Sinard, Richard L. Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla T. O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcaffd86557cddf1faae3fdc7b2f95bd830f2b9" target='_blank'>
              Pan-cancer analysis in the real-world setting uncovers immunogenomic drivers of acquired resistance post-immunotherapy
              </a>
            </td>
          <td>
            Mohamed Reda Keddar, Sebastian Carrasco Pro, Roy Rabbie, Z. Atak, A. Stewart, Scott A. Hammond, Douglas C. Palmer, Ross Stewart, Kathleen Burke, Benjamin Sidders, Jessica Davies, Jonathan R Dry, I. Martincorena, Sajan Khosla, A. Schoenfeld, Martin L. Miller
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background Osteosarcoma (OS) shows a multitude of genetic and chromosomal abnormalities together with large biological heterogeneity. These features limited the identification of novel drugs to treat patients with metastases and/or chemo-resistant tumors. The purpose of this study was to create additional resources for drug screening by generating patient-derived xenograft (PDXs) and PDX-derived cell lines that reflect the spectrum of OS heterogeneity. Methods PDXs were derived from OS collected at diagnosis, surgical resections, or metastases. PDX-derived cell lines were also established. Targeted DNA sequencing and digital PCR were applied to identify major genetic alterations. High-throughput drug screening by using a library of 2880-FDA approved compounds and conventional MTT assays were performed to identify the most effective drugs against in vitro and in vivo growth of chemo-resistant OS. Results Targeted DNA sequencing demonstrated alterations in the most commonly amplified oncogenes, such as MYC, CCNE1, DDR2, CDK4, MDM2, and AURKA. Recurrent deletions and SNVs were found in TP53, CDKN2A, RB1, PTEN, and VHL. Copy number variant (CNV) alterations in PDXs, PDX-derived cell lines and xenografts developed from cell lines (CDX) correlated very well with those observed in the matched original human tumors. Drug screenings identified and repurposed five compounds with efficacy against chemoresistant OS. In this context, we prioritized ixabepilone as a drug capable of inducing tumor regression in mice. Conclusions We enriched the scientific community with additional, molecularly characterized OS models to be used for testing novel therapies and supported the inclusion of ixabepilone into treatment plans for chemoresistant OS. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03440-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23016fdd463bf1bcdb3a8bfe081d0f07e246c765" target='_blank'>
              Genomic profiling of a collection of patient-derived xenografts and cell lines identified ixabepilone as an active drug against chemo-resistant osteosarcoma
              </a>
            </td>
          <td>
            M. Manara, F. Bruzzese, Laura Formentini, L. Landuzzi, L. Pazzaglia, M. Laginestra, Marianna Carrabotta, M. Roca, Federica Iannelli, Laura Grumetti, L. Addi, A. Parra, C. Cristalli, M. Pasello, Alberto Bavelloni, Francesca Carreras, F. Ruzzi, G. Bianchi, M. Gambarotti, A. Righi, Alfredo Budillon, Pier-Luigi Lollini, Katia Scotlandi
          </td>
          <td>2025-07-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Growth hormone (GH) signaling has been implicated in tumor progression and therapy resistance across multiple cancer types, yet its role in bladder cancer remains largely unexplored. In this study, we investigated the impact of GH and its receptor (GHR) on therapy resistance and disease progression in urothelial carcinoma (UC) through integrated transcriptomic and in vitro analyses. Transcriptomic profiling of The Cancer Genome Atlas bladder cancer cohort revealed that high tumoral GHR expression was associated with differential upregulation of genes involved in drug efflux, epithelial-to-mesenchymal transition (EMT), and extracellular matrix (ECM) remodeling. Notably, elevated GHR levels correlated with significantly reduced overall survival in patients with UC. In parallel, in vitro experiments demonstrated that GH promotes chemoresistance in UC cell lines via upregulation of ATP-binding cassette-containing (ABC) transporters and activation of EMT. GH also modulated ECM-remodeling-associated genes in a chemotherapy-dependent manner, including matrix metalloproteinases and tissue inhibitors of metalloproteinases. Importantly, these effects were abrogated by Pegvisomant, a GHR antagonist, indicating the functional relevance of GH/GHR signaling in the mediation of these phenotypes. Collectively, our findings support a mechanistic role for GH signaling in driving therapy resistance and tumor aggressiveness in bladder cancer and suggest GHR antagonism as a potential therapeutic strategy to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f8b97219a1f5f0a9e16e7142057f197a9d88cb" target='_blank'>
              Growth Hormone Signaling in Bladder Cancer: Transcriptomic Profiling of Patient Samples and In Vitro Evidence of Therapy Resistance via ABC Transporters and EMT Activation
              </a>
            </td>
          <td>
            Emily Davis, Lydia J. Caggiano, Hannah Munholland, R. Basu, D. Berryman, J. Kopchick
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2aba01b3716bf8fef6d8f3f3f6e3783dfc01cf" target='_blank'>
              Recent advances in novel targeting mechanisms for colorectal cancer.
              </a>
            </td>
          <td>
            Zhiping Li, Hailing Wang, Yan Liu, Ahequeli Gemingnuer, Yinan Wang, Xin Meng
          </td>
          <td>2025-07-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common types of malignancy diagnosed globally. Metastasis plays a major role in most of the cancer-related mortality among affected patients. Despite the advances in the areas of early detection and localized treatment modalities, there prevail several challenges which the therapeutic strategies encounter, like drug resistance, tumor heterogeneity, and drug delivery. This review presents a comprehensive and detailed overview of organ-specific metastasis that occur in BC, specifically emphasizing key sites such as the bone, liver, lung, and brain. It also outlines the significance of various therapies like chemotherapies, endocrine therapies, targeted therapies and immunotherapies that have been clinically approved to date. The review specifically emphasizes the molecular mechanisms by which non-coding RNAs (ncRNAs) act to exert their effects in regulating drug resistance. It also addresses the new advances in nanotechnology-based drug delivery systems (DDS) that function to enhance the specificity of treatments while simultaneously reducing systemic toxicity. Beyond ncRNAs, this review also explores other critical mechanisms of drug resistance in metastatic BC, including efflux transporter activity, target gene mutations, and micro-environmental factors, to mention a few. Moreover, the review also discusses the clinical significance of combination therapies and new therapeutic strategies, including the use of repurposed drugs and the concepts of personalized medicine. A greater understanding of the ncRNA-mediated signaling pathways, in combination with the latest advances in drug delivery systems, has the potential to greatly improve therapeutic efficacy and could result in more favorable clinical outcomes in the treatment of metastatic BC (MBC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2fe23da14c71b938c6019fd6995e4bfcec570b" target='_blank'>
              Bridging molecular insights and clinical application: non-coding RNAs, targeted drug delivery, and metastatic breast cancer therapy
              </a>
            </td>
          <td>
            Sohini Chakraborty, Satarupa Banerjee
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer, a heterogeneous malignancy with high metastatic potential, remains a global health burden. The Y-box binding protein 1 (YBX1), a
multifunctional nucleic acid-binding protein, has emerged as a pivotal regulator in breast cancer progression, mediating Epithelial-Mesenchymal Transition
(EMT), Cancer Stem Cell (CSC) maintenance, and therapeutic resistance. This review synthesizes preclinical and clinical evidence from 2023–2025, detailing
YBX1’s structural features, molecular mechanisms in tumorigenesis, prognostic/diagnostic significance, and emerging therapeutic strategies. Data from 12+
cohorts (n > 2,500) demonstrate that high YBX1 expression correlates with poor survival (HR = 2.15, P < 0.001) and treatment resistance. Targeting YBX1
via small molecules, RNA therapies, and immune combinations shows promise in preclinical models, highlighting its potential as a precision oncology target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fcdd64a1da81e4ca8608af2299114df40b183e" target='_blank'>
              YBX1 in Breast Cancer: From Molecular Mechanisms to Therapeutic Opportunities
              </a>
            </td>
          <td>
            Houhong Wang
          </td>
          <td>2025-06-30</td>
          <td>Journal of Surgery &amp; Anesthesia Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are repetitive DNA sequences that contribute to genetic diversity and play a significant role in disease susceptibility. The human genome contains approximately 1.5 million STR loci, collectively covering around 3% of the total sequence. Certain repeat expansions can significantly impact cellular function by altering protein synthesis, impairing DNA repair, and leading to neurodegenerative and neuromuscular diseases. Traditional short-read sequencing struggles to accurately characterize STRs due to its limited read length, which limits the ability to resolve repeat expansions, increases mapping errors, and reduces sensitivity for detecting large insertions or interruptions. This review examines how long-read sequencing technologies, particularly Oxford Nanopore and PacBio, overcome these limitations by enabling direct sequencing of full STR regions with improved accuracy. We discuss challenges in sequencing, bioinformatics workflows, and the latest computational tools for STR detection. Additionally, we highlight the strengths and limitations of different methods, providing deeper insight into the future of STR genotyping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bca970ab6c190594afe0092773766d77d84cc3b" target='_blank'>
              Get ready for short tandem repeats analysis using long reads-the challenges and the state of the art
              </a>
            </td>
          <td>
            Marija Chaushevska, Karmele Alapont-Celaya, Anne Kristine Schack, Lukasz Krych, M. C. Garrido Navas, Anastasia Krithara, Gjorgji Madjarov
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6c0eb75e7f5a0febc3a26c4b5b85dcca6192f2" target='_blank'>
              Validation of the MAGIC genomic signature on RNAseq in soft-tissue sarcomas
              </a>
            </td>
          <td>
            A. Benhaddou, G. Pérot, P. Rochaix, T. Valentin, G. Ferron, F. Chibon
          </td>
          <td>2025-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Jumping translocations (JTs) are rare chromosomal abnormalities that play a crucial role in the pathogenesis of various cancer types. These rearrangements, especially those involving chromosome 1q, are frequently associated with tumor progression, therapeutic resistance, and poor prognosis. One gene of particular interest, human Jumping Translocation Breakpoint (JTB), has been identified at the site of translocation breakpoints and exhibits complex, context-dependent roles in cancer biology. JTB protein functions as a pivotal regulator in mitosis, chromosomal segregation, apoptosis, and cellular metabolism. It is functionally linked with the chromosomal passenger complex (CPC) and is implicated in processes such as epithelial–mesenchymal transition (EMT), immune evasion, and therapy resistance, especially in breast and prostate cancers. Advances in genomic, transcriptomic, and proteomic research have highlighted the significant potential of JTB as a diagnostic biomarker and a target for therapeutic interventions. This review underscores the dual role of JTB as both a tumor suppressor and oncogene, depending on the cellular context, and advocates for its continued investigation at the genomic, transcriptomic, and proteomic levels. Understanding JTB’s multifaceted contributions to tumor biology may pave the way for novel biomarkers and targeted treatments in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48960a3d042990da8ae30d21a39837a5c48dded4" target='_blank'>
              From Jumping Gene to Cancer: Revisiting the Role of JTB Protein
              </a>
            </td>
          <td>
            Taniya M Jayaweera, Madhuri Jayathirtha, Krishan Weraduwage, Petra Kraus, C. Darie, A. Neagu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The role of G-quadruplexes (G4s) in gene regulation has been widely documented, especially in gene promoters. However, the transcriptional mechanisms involving G4s in other regulatory regions remain largely unexplored. In this study, we integrated the G4-DNA data derived from 22 breast cancer patient-derived tumor xenograft (PDTX) models and MCF7 cell line as potential breast cancer-associated G4s (BC-G4s). Genome-wide analysis showed that BC-G4s are more prevalent in gene promoters and the first introns. The genes accommodating promoter or intronic BC-G4s show significantly higher transcriptional output than their non-G4 counterparts. The biased distribution of BC-G4s in close proximity to the transcription start site (TSS) is associated with an enrichment of transcription factor (TF) interactions. A significant negative correlation was detected between the G4–TF interactions within the first introns and their cognate promoters. These different interactions are complementary rather than redundant. Furthermore, the differentially expressed genes (DEGs) harboring promoter and first intron BC-G4s are significantly enriched in the cell cycle pathway. Notably, promoter BC-G4s of DEGs could be a central hub for TF–TF co-occurrence. Our analysis also revealed that G4-related single nucleotide variants (SNVs) affect the stability of G4 structures and the transcription of disease-related genes. Collectively, our results shed light on how BC-G4s within promoters and first introns regulate gene expression and reinforce the critical role of G4s and G4-related genes in breast cancer-associated processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6951122fcae4ae20d0e429a03632112d1f8ad46" target='_blank'>
              The Relevance of G-Quadruplexes in Gene Promoters and the First Introns Associated with Transcriptional Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Huiling Shu, Ke Xiao, Wenyong Zhu, Rong Zhang, Tiantong Tao, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hypoxia, characterized by a reduction in tissue oxygen levels, is a hallmark of many solid tumors and affects a range of cellular processes, including DNA repair. In low-oxygen conditions, cancer cells often suppress key DNA repair pathways such as homologous recombination (HR), leading to the accumulation of DNA damage and increased genomic instability. These changes not only drive tumor progression but also contribute to resistance against conventional therapies. Hypoxia significantly reduces the effectiveness of oxygen-dependent treatments, including radiotherapy and many chemotherapeutic agents. To address this limitation, bioreductive drugs have been developed that become selectively activated in hypoxic environments, providing targeted cytotoxic effects within oxygen-deprived tumor regions. Additionally, the rapid growth of tumors often results in disorganized and inefficient vasculature, further impairing the delivery of oxygen and therapeutic agents. This review explores the molecular mechanisms by which hypoxia disrupts DNA repair and contributes to treatment resistance. It also presents emerging therapeutic strategies aimed at targeting the hypoxic tumor microenvironment to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34398b4aa06a09890c501084c80149cba79331e4" target='_blank'>
              Influence of Hypoxia on Tumor Heterogeneity, DNA Repair, and Cancer Therapy: From Molecular Insights to Therapeutic Strategies
              </a>
            </td>
          <td>
            Dominika Kunachowicz, Paulina Tomecka, Mikołaj Sędzik, Jarosław Kalinin, Jacek Kuźnicki, Nina Rembiałkowska
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a09fe0116b674341564f20d065726804460dc81" target='_blank'>
              Identification of hub genes and potential molecular mechanisms of tumor immune microenvironment-related radiotherapy sensitivity in locally advanced cervical cancer.
              </a>
            </td>
          <td>
            Yichen Zan, Qing Liu, Yue Yin, Chongxin Guo, Jing Xu, Junhan Gao, Guanglu Dong
          </td>
          <td>2025-06-30</td>
          <td>Reproductive sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Aurora kinase A (AURKA) is aberrantly expressed in a large number of tumors and promotes tumor progression by regulating the cell cycle, chromosomal instability, and drug resistance. However, its heterogeneous expression and combination therapy benefit in prostate adenocarcinoma (PRAD) is unclear. Methods In this study, we integrated TCGA pan-cancer multi-omics data and GEO data to analyze the RNA, methylation, protein expression, and genomic alteration characteristics of AURKA. We then used single-cell RNA sequencing to resolve the functional heterogeneity of AURKA in the PRAD epithelial cell subpopulation and verified its impact on the malignant phenotype of desmoplasia-resistant prostate cancer cells in in vitro experiments. This research also analyzed the prognostic risk stratification of AURKA subpopulations in combination with various indicators and the potential benefit of AURKA inhibitors in combination with various treatments. Results The pan-cancer analysis demonstrated that AURKA expression heterogeneity was present among urological tumors at different molecular levels, and the positive correlation of AURKA alteration with MYC and E2F pathways was conserved in pan-cancer. Epithelial cell subpopulations with high expression of AURKA (epi3/4/6) promoted proliferation by regulating cell cycle and DNA repair, while low expression subsets (epi1/2/7) activated TNF-α and androgen receptor (AR) pathways to mediate drug resistance. In particular, AURKA may serve as a compensatory pathway to support tumor activity after AR inhibition in prostate cancer, a complex mechanism not seen in other tumors. AURKA-overexpressing patients with low Gleason scores or high PSA have a poor prognosis in clinical analysis. Furthermore, a comprehensive drug sensitivity co-analysis found that AURKA inhibitors may benefit from targeted therapy, ADC therapy, and immunotherapy. TMB and CD274 expression were the biomarkers of AURKA high-expression patients with PRAD for clinical outcome. Conclusion AURKA expression heterogeneity has been identified as a critical factor in the progression of PRAD and the development of drug resistance. The molecular subtyping of AURKA can serve as a precise strategy for combination therapy and provide a theory for the combination of AURKA inhibitors and targeted/immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396eba19ede00209d3509820e66ea1756b82e606" target='_blank'>
              Aurora kinase-a expression heterogeneity and potential benefit of combination therapy in prostate adenocarcinoma
              </a>
            </td>
          <td>
            Ru Chen, Qianyi Qiu, Weiting Xie, Jun Lin, Rong Liu, Jianhui Chen, Shaoxing Zhu, Yiming Su
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Copy number gene amplification and associated overexpression of driver oncogenes are genetic events that contribute to cancer progression and drug resistance. MDS1 and EVI1 Complex locus (MECOM) gene is copy number amplified and overexpressed in aggressive epithelial ovarian cancers. The biological function and precise molecular mechanism of MECOM in the progression and drug resistance of ovarian cancer remain unclear. Here, we unravel MECOM as a regulator of KRAS and its downstream MAP Kinase signalling pathway, and also identify epigenetic inhibitor JIB-04 as a pharmacological agent targeting MECOM/KRAS axis. RNAi-mediated attenuation of MECOM in ovarian cancer cells harboring MECOM amplification reduced their proliferation, impaired colony formation, and impeded cellular migration. ChIP-qPCR analysis confirmed binding of MECOM to the KRAS promoter, suggesting direct regulation of the KRAS gene at the transcriptional level. Further, MECOM promoted cellular proliferation by regulating KRAS-mediated ERK/ZEB1 signalling cascade. The anti-tumorigenic effects due to MECOM loss were phenocopied by the treatment of ovarian cancer cells harboring MECOM amplification with JIB-04 epigenetic inhibitor targeting Jumonji domain histone demethylase enzymes. By ChIP-qPCR, we show that JIB-04 induced transcriptional changes of MECOM by altering H3K27me3 demethylation at its promoter region. We further report that ovarian cancer cells expressing high-MECOM levels exhibit cisplatin resistance, which could be effectively reversed upon pre-treatment with JIB-04. The therapeutic efficacy of JIB-04 was further demonstrated in mice bearing ovarian cancer cell xenografts, where JIB-04 slowed down the tumor growth in corroboration with diminishing MECOM expression. RNA-sequencing analysis identified potential cisplatin resistance gene, SUB1, being regulated by JIB-04-mediated modulation of MECOM expression. Altogether, these data suggest that epigenetic silencing of MECOM by JIB-04 mediated H3K27me3 modulation is an important mechanism in ovarian cancer and provide a new therapeutic target for the treatment of ovarian cancers harboring MECOM amplification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a776f5b72c3bc81fd8c696f0bdd3032a4ca281b9" target='_blank'>
              Epigenetic targeting of MECOM/KRAS axis by JIB-04 impairs tumorigenesis and cisplatin resistance in MECOM-amplified ovarian cancer
              </a>
            </td>
          <td>
            Ibha Singh, Amarnath Karna, Anita Prajapati, Ujjawal Solanki, Archana Mukherjee, Sheetal Uppal, Pawan Malhotra, Manoj Kumar, P. Agarwal
          </td>
          <td>2025-07-15</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is among the most frequently diagnosed cancer types, and affected patients frequently experience poor prognostic outcomes and high mortality rates. Many genomic studies of ESCC have been performed in recent years, yet the mutational mechanisms driving ESCC and their clinical implications remain incompletely understood. In this study, paired tumor and normal tissue samples from 22 patients with ESCC were used for whole genome sequencing-based analyses of genome-wide mutational events. These comprehensive analyses enabled the detection and characterization of various mutation subtypes in ESCC including somatic single-nucleotide variants, small insertions and deletions, copy number variations, structural variations, and circular extrachromosomal DNA. Of identified genes harboring non-silent mutations, TP53, NOTCH1, CSMD3, EP300, and FAM135B were the most frequently mutated genes in this study and they were annotated in the COSMIC Cancer Gene Census. With the exception of aging-related signatures, an APOBEC-associated mutational signature was the dominant mutational feature detected in ESCC samples, suggesting that APOBEC-mediated cytidine deamination is likely a major driver of mutations in this cancer type. Notably, our study also detected circular extrachromosomal DNA (ecDNA) events in these ESCC patient samples. The oncogenes COX6C, PVT1, and MMP12 as well as the oncogenic long non-coding RNA AZIN1-AS1 which were detected in ecDNA regions in these analyses may be associated with worse disease-free survival in ESCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55745d3238cc107d72d837356b7050823d1fc40" target='_blank'>
              Whole genome sequencing revealed esophageal squamous cell carcinoma related biomarkers
              </a>
            </td>
          <td>
            Mingjun Li, Lei Li, Xizi Wang, Yanwei Zhao, Peina Du, Wei Wang, , Yadong Wang, Yanxing Sheng, Mingliang Gu, Xiaodong Jia
          </td>
          <td>2025-06-26</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Proliferating cell nuclear antigen (PCNA) is a central regulator of numerous cellular processes impacting DNA metabolism and genetic integrity, including DNA damage, cell cycle progression, and transcriptional regulation. PCNA aberrations manifest in different disease phenotypes, including neoplastic proliferation, chromatin disassembly, genomic instability, and impaired DNA repair. Although the role of PCNA in control of DNA homeostasis has been extensively studied in some cancers, its pan-cancer relevance in tumorigenesis, immune interactions, and therapeutic resistance remains underexplored. Here, we undertook a comprehensive analysis of publicly available databases to assess the relationship between PCNA expression and the immunological, survival, genetic, functional, and drug sensitivity profiles across multiple cancer types. PCNA mRNA levels were altered in across cancers and associated with altered cancer signaling networks constituting Wnt, Hippo, and mTOR pathways. Increased transcript levels were associated with poor overall survival in some cancers, including ACC, CESC, LGG, and THYM. Amplification was the predominant genetic alteration of PCNA in multiple tumors. In several tumors, upregulation of PCNA was linked with differences in tumor-infiltrating lymphocytes, and specific immune-inhibitors, and chemokines. Moreover, elevated PCNA expression was linked with increased sensitivity to several drugs, particularly to Navitoclax, NPK76-II-72-1, and Ciclopirox across cancers. Our study orients the first comprehensive pan-cancer analysis of PCNA, uncovering its prognostic significance and altered expression across various cancers through multiomics data. Unlike previous studies, tumor-specific genetic alterations, such as amplification and hypomethylation, and the paradoxical immune microenvironment linked to PCNA were explored, suggesting potential immune evasion mechanisms. Additionally, new therapeutic avenues reveal PCNA’s relationship with drug sensitivity to agents like Navitoclax and Ciclopirox, providing invaluable insights for pharmacological interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3571f2ad7f72f9a390320cd46deb0b77021902" target='_blank'>
              PCNA in Pan-Cancer: A Prognostic Biomarker Unveiled Through a Data-Driven, Multidimensional Analysis of Transcriptomics, Immunity, and Functional Profiling
              </a>
            </td>
          <td>
            Depanshi Pandit, Ravindranath Bilachi Sanganabasappa, Amardeep Dhillon, Sanjiban Chakrabarty
          </td>
          <td>2025-06-26</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c59d7bda81acc4db797cc49795cf4efa86255bd" target='_blank'>
              Long Read Genome Sequencing Elucidates Diverse Functional Consequences of Structural and Repeat Variation in Autism
              </a>
            </td>
          <td>
            M. Mortazavi, James Guevara, Joshua Diaz, Stephen Tran, Helyaneh Ziaei Jam, S. Batalov, Matthew Bainbridge, Aaron D. Besterman, Melissa Gymrek, Abraham A Palmer, Jonathan Sebat
          </td>
          <td>2025-07-23</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ea5280910649de8b63e584d7975f22a13b00b4" target='_blank'>
              Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code
              </a>
            </td>
          <td>
            Marina Murillo-Recio, Marina Salvadores, Adrián Gabriel Torres, Lina Tsapanou, Aina Vaquer Picó, F. Supek, L. R. de Pouplana
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific KRAS inhibitors in colorectal cancer (CRC). Here, we investigated whether broad-spectrum, active-state RAS inhibitors meet similar limitations. We found that KRAS-mutant CRC cell lines were sensitive to the RAS(ON) multiselective RAS inhibitor RMC-7977, given that treatment resulted in RAS-RAF-MEK-ERK pathway inhibition; halted proliferation; and, in some cases, induced apoptosis. RMC-7977 initially reduced the activity of a compartment-specific, dual-color reporter of ERK activity, with reporter reactivation emerging after long-term dose escalation. These drug-resistant cell populations exhibited distinct patterns of phospho-protein abundance, transcriptional activities, and genomic mutations, including a Y71H mutation in KRAS and an S257L mutation in RAF1. Transgenic expression of KRASG13D, Y71H or RAF1S257L in drug-sensitive CRC cells induced resistance to RMC-7977. CRC cells that were resistant to RMC-7977 and harboring RAF1S257L exhibited synergistic sensitivity to concurrent inhibition of RAS and RAF. Our findings demonstrate the power of reporter-assisted screening together with single-cell analyses for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract the emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062fb5066a8ceecc8901f69c1d9a2835ca85a048" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibitors in colorectal cancer.
              </a>
            </td>
          <td>
            Oleksandra Aust, Moritz R T Thiel, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Francisca Böhning, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, C. Sers, C. Der, M. Morkel
          </td>
          <td>2025-07-22</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c29c85da72e04b5001274eb1b2f06a493d988d5" target='_blank'>
              Reprogramming of Cellular Plasticity via ETS and MYC Core-regulatory Circuits During Response to MAPK Inhibition in BRAF-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            H. Lee, Zhao Zheng, Alexey Sorokin, Chi Wut Wong, Stefania Napolitano, S. Chowdhury, Preeti Kanikarla, A. Singh, Veena Kochat, Christopher Bristow, Sanjana Srinivasan, Mike Peoples, Emre Arslan, J. Alshenaifi, O. Villarreal, V. Morris, J. P. Shen, F. Meric-Bernstam, Abhinav K. Jain, Natalie W Fowlkes, Amanda Anderson, D. Menter, A. Saw, Kunal Rai, S. Kopetz
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Up to 40% of diffuse large B-cell lymphoma (DLBCL) patients do not experience a durable response to frontline immunochemotherapy, and prospective identification of high-risk cases that may benefit from personalized therapeutic management remains an unmet need. Molecular phenotyping techniques have established a landscape of genomic variants in diagnostic DLBCL; however, these have not yet been applied in large-scale studies of relapsed/refractory DLBCL, resulting in incomplete characterization of mechanisms driving tumor progression and treatment resistance. Here, we performed an integrated multiomic analysis on 228 relapsed/refractory DLBCL samples, including 24 with serial biopsies. Refined cell-of-origin subtyping identified patients harboring GCB and DZsig+ relapsed/refractory tumors in cases with primary refractory disease with remarkably poor outcomes, and comparative analysis of genomic features between relapsed and diagnostic samples identified subtype-specific mechanisms of therapeutic resistance driven by frequent alteration to MYC, BCL2, BCL6, and TP53 among additional strong lymphoma driver genes. Tumor evolution dynamics suggest innate mechanisms of chemoresistance are present in many DLBCL tumors at diagnosis, and that relapsed/refractory tumors are primarily comprised of a homogenous clonal expansion with reduced tumor microenvironment activity. Adaptation of personalized therapeutic strategies targeting DLBCL subtype-specific resistance mechanisms should be considered to benefit these high-risk populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf3e7eefa27b6aa5bb2bc1d3568bfab1d01f42f" target='_blank'>
              Integrated genomics with refined cell-of-origin subtyping distinguishes subtype-specific mechanisms of treatment resistance and relapse in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            J. Walker, K. Wenzl, J. Novak, M. Stokes, M. Hopper, A. Dropik, Miranda S Siminski, Allison M. Bock, V. Sarangi, María Ortiz, Nicholas Stong, C. Huang, Matt J. Maurer, B. Link, Stephen M Ansell, T. Habermann, T. Witzig, Rebecca L King, Grzegorz S. Nowakowski, J. Cerhan, A. K. Gandhi, A. Novak
          </td>
          <td>2025-07-12</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1683b65b8227a142ec0662e619dfd830c76c770" target='_blank'>
              Major effect mutations drive DNA methylation variation after colonization of a novel habitat
              </a>
            </td>
          <td>
            Johan Zicola, Emmanuel Tergemina, Mehmet Göktay, Célia Neto, Robert J. Schmitz, Angela M. Hancock
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Genetic instability is a major hazard threatening the fate of cells and ultimately of organisms. DNA double-strand break (DSB) is a highly toxic lesion, jeopardizing genome stability. Using cytogenetic and differential exome sequencing, we show here that upregulation of the kinase PKB/AKT1 leads to genomic rearrangements and chromosome fusions. By combining various approaches, at the genome scale and at precise loci, we show that PKB/AKT1 stimulates DSB end-joining, leading to inter- and intrachromosomal genomic rearrangements. The MRE11–RAD50–NBS1 (MRN) complex plays an essential role in the early steps of DSB signaling/repair. We show here that PKB/AKT1 favors the assembly of MRN, leading to the stimulation of DSB signaling via the MRE11/ATM axis. We identify MRE11 as a phosphorylation effector of PKB/AKT1 and reveal several sites whose phosphorylation is required for PKB-mediated stimulation of DSB end-joining and chromosome fusions. These data reveal that PKB/AKT1 actively promotes genetic instability by increasing the efficiency of DSB end-joining through MRE11 phosphorylation on these sites. These results highlight that not only a defect of DSB signaling/repair but also its stimulation, can lead to genome rearrangements and underline the importance of a precise regulation of the DNA damage response to maintain genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d7a378d4ec28789fd47d885beb264a6236c373" target='_blank'>
              Genome rearrangements induced by the stimulation of end-joining of DNA double strand breaks through multiple phosphorylation of MRE11 by the kinase PKB/AKT1
              </a>
            </td>
          <td>
            J. Guirouilh-Barbat, Iman Litchy Boueya, C. Gelot, Gaëlle Pennarun, Christine Granotier-Beckers, E. Dardillac, Wei Yu, C. Lescale, E. Rass, Olivier Ariste, Nicolas Siaud, Benjamin Renouf, Armêl Millet, Nadine Puget, Pascale Bertrand, Pierre de la Grange, Erika Brunet, L. Deriano, Bernard S. Lopez
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Gastric cancer remains a leading cause of cancer-related morbidity and mortality worldwide. The genetic factors contributing to gastric cancer progression and resistance to therapies, particularly radiotherapy, are not fully understood. TP53, ATM, RAD51, and BAX are genes involved in DNA repair, apoptosis, and response to stress. The aim of this study was to investigate the expression patterns of these genes in gastric cancer, their potential role in radiotherapy resistance, and their diagnostic value. Methodology Gene expression levels of TP53, ATM, RAD51, and BAX were assessed using RT-qPCR across 9 gastric cancer cell lines and 6 normal control cell lines. Additionally, protein expression was confirmed via IHC and TCGA dataset analysis. Methylation levels of these genes were evaluated in gastric cancer tissues using the GSCA database. Mutational analysis was conducted using cBioPortal, and survival analysis was performed using Kaplan-Meier and meta-analysis. The radiotherapy resistance study was carried out by knocking down TP53, RAD51, and BAX in AGS and MKN-45 gastric cancer cell lines, followed by expression analysis, colony formation, and wound healing assays. Results The expression of TP53, RAD51, and BAX was significantly upregulated, while ATM was downregulated in gastric cancer cell lines compared to normal controls. All four genes demonstrated good discriminatory power (AUC = 1) in distinguishing gastric cancer from normal samples. Methylation analysis revealed significant hypomethylation of TP53, RAD51, and BAX, and hypermethylation of ATM in gastric cancer tissues. Mutational analysis showed that TP53 was altered in 88% of gastric cancer samples, while ATM, RAD51, and BAX exhibited lower mutation rates. Survival analysis suggested that elevated expression of TP53, RAD51, and BAX may be linked to poorer survival outcomes, while reduced ATM expression appeared to associate with decreased overall survival. However, these associations require further validation through additional studies. Knockdown of TP53, RAD51, and BAX in AGS and MKN-45 cells resulted in significantly reduced cell proliferation and slower wound healing, highlighting their role in radiotherapy resistance. Conclusion The TP53, RAD51, and BAX genes are significantly involved in gastric cancer progression and resistance to radiotherapy. Their expression and mutation status provide valuable diagnostic and prognostic information. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00496-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8363e00ef3c38472bf63112f24092628b9ff0683" target='_blank'>
              In silico and in vitro assessment of TP53, ATM, RAD51, and BAX genes in gastric cancer and their contribution to radiotherapy resistance
              </a>
            </td>
          <td>
            Junwei Zhang, Pengtao He
          </td>
          <td>2025-07-12</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78884b796044723309cb646cb50e973d65185744" target='_blank'>
              A proteogenomic approach to identifying a gene signature associated with HOXB13 G84E carrier status in prostate cancer tumours
              </a>
            </td>
          <td>
            K. Raspin, Z. Noor, A. T. Aref, J. Marthick, C.-h. Jung, S. Donovan, S. Williams, P. Hains, P. J. Robinson, R. L. Balleine, Q. Zhong, R. Reddel, L. M. FitzGerald, J. L. Dickinson
          </td>
          <td>2025-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21690ca2e89ea2f16a09c71f18f6490b7208397c" target='_blank'>
              High Throughput Screening Identifies Small Molecules that Synergize with MRTX1133 Against Acquired Resistant KRAS G12D Mutated CRC
              </a>
            </td>
          <td>
            Natalie Thielen, Ning Wei, Emiko Nagai, Edward Chu, S. Kitamura, Chaoyuan Kuang
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Osteosarcoma is the most common primary malignant bone tumor, with high invasiveness and metastatic potential and a poor prognosis in patients with metastatic cancer. Despite the rapid advancements in genomics in recent years that provided new perspectives for studying the molecular mechanisms of osteosarcoma, the understanding of its tumor heterogeneity and evolutionary mutation process remains limited. Methods In this study, whole-exome evolutionary profiling was performed on data from the TARGET database representing 61 osteosarcoma cases. Subclonal architectures were reconstructed to characterize mutational trajectories. Differential mutation analysis was used to identify candidate metastasis-associated mutations. These features were used to build a metastasis-prediction classifier, which was cross-validated and tested on an independent external cohort. Finally, Suppes’ probabilistic theory of causality was integrated with cohort data to infer high-frequency evolutionary paths linked to metastasis. Results A linear evolutionary trajectory was observed in 62% of patients, indicating sequential clonal expansion. Eight key mutations were closely associated with metastatic progression. The classifier achieved 83% accuracy in cross-validation and maintained robust performance on the external validation set. Through causal inference, distinct evolutionary routes underpinning metastasis were uncovered, with ATRX mutations frequently occurring as early events that reshaped clonal dynamics and facilitated tumor spread. Conclusions In this study, the dynamic evolutionary landscape of osteosarcoma metastasis was delineated, an early metastasis classification model was constructed, and the impact of early clonal ATRX mutations on metastasis initiation were highlighted. These findings offer potential avenues for the early diagnosis and risk assessment of osteosarcoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06796-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68812275cd636690cb798f47d80f2494eb2d39fb" target='_blank'>
              Whole‑exome evolutionary profiling of osteosarcoma uncovers metastasis‑related driver mutations and generates an independently validated predictive classifier
              </a>
            </td>
          <td>
            Zhen Wang, Zhe Wang, Ruoyu Wang, Zumin Wang, Xiangyang Cao, Rui Chen, Zebing Ma, Shanshan Liang, Shuai Tao
          </td>
          <td>2025-07-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As a predominant oncogenic driver in non-small cell lung cancer (NSCLC), EGFR frequently undergoes amplification or mutation, with EGFR-tyrosine kinase inhibitors (EGFR-TKIs) like gefitinib and erlotinib constituting frontline therapy for advanced EGFR-mutant cases. However, both primary and acquired resistance significantly limit clinical efficacy. Here, we revealed that glutathione metabolic pathway controlled by glutathione peroxidase GPX2 was abnormally activated in gefitinib-resistant A549 and HCC827-GR cell lines. Mechanistically, GPX2 triggers Hedgehog signaling activation through releasing GLI transcriptional regulator, promoting cancer stem cell (CSC) characteristics and TKI resistance. Notably, N6-methyladenosine (m6A) modification on GPX2 mRNA mediated by METTL14 diminished its stability. In vivo, GPX2 deletion constrained glutathione metabolism and boosted the effectiveness of TKI in cell line-derived xenograft models. Collectively, these findings demonstrate that GPX2 serves as a positive regulator of both primary and acquired EGFR-TKI resistance and could be a promising therapeutic target for precise treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50746db76f6a5070ba253127ed90d322b08f1eb2" target='_blank'>
              N6-methyladenosine-modified GPX2 impacts cancer cell stemness and TKI resistance through regulating of redox metabolism
              </a>
            </td>
          <td>
            Xu Yang, Long Yu, Miaomiao Shao, Huiling Yang, Kangwei Qi, Gaofei He, Lanxin Wang, Di Kong, Jianxin Gu, Xiaolin Xu, Lanxin Wang
          </td>
          <td>2025-06-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a poor prognosis that is driven primarily by oncogenic KRAS mutations present in > 90% of cases. KRAS mutations, particularly the G12D mutation which dominates in PDAC, fuel tumor initiation, progression, and immune evasion, thereby contributing to therapy resistance. Nevertheless, KRAS has long been considered “undruggable” due to its structure. Recent advances have spurred transformative progress in direct KRAS inhibition. While FDA-approved mutation-specific and pan-KRAS inhibitors show limited efficacy in PDAC, emerging agents (MRTX1133 and RMC-9805) have demonstrated preclinical promise. However, resistance remains a critical hurdle and is driven by pathway reactivation, secondary mutations, and metabolic adaptations. Alternative strategies targeting upstream regulators (SHP2 and SOS1) aim to block KRAS activation and associated resistance mechanisms. Preclinical studies have also highlighted synergistic benefits of combining KRAS inhibitors with MEK, PI3K, or CDK4/6 inhibitors, which are now undergoing clinical evaluation. Immunotherapies, including KRAS-targeted vaccines and adoptive T-cell therapies, have further expanded the therapeutic landscape of enhancing KRAS-targeted therapies in PDAC. The molecular basis of KRAS-driven PDAC, current inhibitors, resistance mechanisms, and innovative strategies are discussed herein to address treatment barriers. Opportunities to improve clinical outcomes are underscored in this challenging malignancy by integrating insights from preclinical and clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8cfbe6080878693ce57a6e261fa2c4f9ddf499" target='_blank'>
              Drugging the ‘undruggable’ KRAS: breakthroughs, challenges, and opportunities in pancreatic cancer
              </a>
            </td>
          <td>
            Nawaz Khan, Umar Raza, Syed Aqib Ali Zaidi, Muhadaisi Nuer, Kayisaier Abudurousuli, Yipaerguli Paerhati, Alifeiye Aikebaier, Wenting Zhou
          </td>
          <td>2025-07-07</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/196ee85ccabe285a00ef4209f93a8dd533568146" target='_blank'>
              Identifying rare spontaneous mutations through wildtype E. coli population sequencing
              </a>
            </td>
          <td>
            R. Green, M. Bawn, A. Angus-Whiteoak, M. Jago, FJ Whelan, M. Lagator, N. Hall, C. Knight, R. Krašovec
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glucose metabolism reprogramming as a defining hallmark of cancer has become a pivotal frontier in oncology research. Recent technological advances in single-cell sequencing, spatial omics, and metabolic imaging have transformed the field from static bulk analyses to dynamic investigations of spatiotemporal heterogeneity at a single-cell resolution. This review systematically summarizes the current knowledge on tumor glucose metabolism dynamics, discussing spatial heterogeneity and temporal evolution patterns, metabolic subpopulation interactions revealed by single-cell metabolomics, the glucose metabolism–epigenetics–immunology regulatory axis, and therapeutic strategies targeting metabolic vulnerabilities. Recent technological advances in single-cell sequencing and spatial omics have transformed our understanding of tumor glucose metabolism by providing high-resolution insights into metabolic heterogeneity and regulatory mechanisms, contrasting with classical bulk analyses. Spatiotemporal heterogeneity critically influences therapeutic outcomes by enabling tumor cells to adapt metabolically under selective pressures (e.g., hypoxia, nutrient deprivation), fostering treatment resistance and relapse. Deciphering these dynamics is essential for developing spatiotemporally targeted strategies that address intratumoral diversity and microenvironmental fluctuations. By integrating cutting-edge advances, this review deepens our understanding of tumor metabolic complexity and provides a conceptual framework for developing spatiotemporally precise interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eed1c2a93a86e6df9bbaf577827b208495bf5a0" target='_blank'>
              Spatiotemporal Heterogeneity of Tumor Glucose Metabolism Reprogramming: From Single-Cell Mechanisms to Precision Interventions
              </a>
            </td>
          <td>
            Xiaoxue Chai, Qian Tao, Lili Li
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription‐replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription‐induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription‐Replication Conflicts: Unlocking New Frontiers in Cancer
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Germline pathogenic variants (GPVs, 'mutations') causing inherited susceptibility to certain cancers (cancer susceptibility genes, CSGs) broadly belong to one of two main classes-loss of function variants in tumour suppressor genes (TSGs) or gain of function variants in proto-oncogenes (an over-simplification). Genomic analyses of tumours identify 'driver mutations' promoting tumour growth and somatic variants which contribute to 'mutation signatures' which, with histopathology, can be used to subclassify cancers with implications for causality and treatment. The identification of susceptible individuals is important, as they and their relatives may be at elevated risk of tumours, and this can influence optimal cancer treatment. Classically, cancer risk assessment utilises family history, lifestyle/environment factors, and any non-neoplastic clinical findings, followed by genetic testing of high/moderate penetrance CSGs. In cancer cases not caused by highly penetrant CSGs, multiple variants conferring relatively small risks play a major role. These were discovered by genome-wide association (GWAS) studies. The utility of polygenic risk scores (PRS) derived from multiple such variants for clinical risk profiling is being assessed. Access to genetic tests is improved by widening eligibility criteria for testing and empowering non-genetic clinicians to identify CSG GPVs and manage carriers. This will contribute to expanding programmes of screening, prevention and early detection (SPED), with personalised surveillance and prophylactic interventions, and exploit knowledge of the molecular mechanisms of cancer susceptibility to develop novel cancer therapies. In some jurisdictions, population testing is being considered, but GPV penetrance in this setting can be unclear, and the public health implications are complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee21b45930d11565d54a3022b9abc97a84ec0804" target='_blank'>
              Inherited Susceptibility to Cancer: Past, Present and Future.
              </a>
            </td>
          <td>
            Shirley V Hodgson, W. Foulkes, E. Maher, C. Turnbull
          </td>
          <td>2025-07-21</td>
          <td>Annals of human genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The circadian clock governs daily rhythms in numerous physiological processes through precise regulation of gene expression and biochemical functions. Dysregulation of the circadian rhythm has been implicated in carcinogenesis and cancer progression. However, the mechanisms by which the circadian clock influences cancer phenotype and chemotherapy resistance, particularly in cholangiocarcinoma (CCA), remain poorly understood. Using cell lines established from primary CCA and metastatic ascites of two male patients, we manipulated core clock genes (BMAL1, PER2, and NR1D1) to evaluate their effects on circadian rhythms. We analyzed alterations in circadian phenotypes at dynamic and single time points and assessed their impact on cancer-related phenotypic changes, including proliferation, apoptosis, cell cycle regulation, migration, invasion, and the expression of epithelial-to-mesenchymal transition (EMT) and cancer stem cell markers. Additionally, we examined the impact of circadian disruption on gemcitabine sensitivity. Genetic deletion of BMAL1, PER2, and NR1D1 disrupted circadian rhythm and significantly altered cancer phenotypes. Notably, BMAL1 and NR1D1 impairment exacerbated cell migration, invasion, and EMT activation in CCA cells. BMAL1 loss also induced gemcitabine resistance. In contrast, PER2 repression enhanced chemosensitivity and inhibited metastasis. The modulation of the circadian gene triggered phenotypic changes in CCA cells, indicating a crucial involvement of core-clock components in the pathological mechanisms hastening bile duct cancer malignancy. Our findings advance the understanding of regulating CCA malignancy and may offer a novel target for its treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63df1b948313a08b9d2b74bdfc742ccab84a19a9" target='_blank'>
              Dysfunction of core clock genes regulates malignant phenotype and gemcitabine sensitivity of cholangiocarcinoma cells.
              </a>
            </td>
          <td>
            Yin Li, Aimin Zheng, Yangang Cui, Tianbao Liu
          </td>
          <td>2025-06-17</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Wnt/β-catenin signaling pathway is a highly conserved signaling pathway closely linked to cancer development through various biological processes, including oncogenic transformation, genomic instability, cancer cell proliferation, stemness, metabolism, cell death, immune regulation, and metastasis. Notably, its activation plays a crucial role in drug resistance to chemotherapy, targeted therapy and immunotherapy. Recent advances in drug development have identified several targeted inhibitors acting at key nodal points of this pathway, with some demonstrating synergistic efficacy when combined with immunotherapeutic agents. This review provides a comprehensive analysis of current understanding regarding the Wnt/β-catenin pathway in malignancy, emphasizing its multifaceted roles in tumor initiation, therapeutic resistance, and immune regulation. Additionally, we summarized the clinical performance of combination therapies targeting the Wnt/β-catenin pathway in conjunction with chemotherapy, targeted therapy, and immunotherapy. Although clinical development remains at a relatively early stage, pharmacological modulation of Wnt/β-catenin signaling offers considerable potential as a novel therapeutic paradigm in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a16ef3c883188a8043ab3dce283646cebd1d89" target='_blank'>
              Wnt/β-catenin mediated signaling pathways in cancer: recent advances, and applications in cancer therapy
              </a>
            </td>
          <td>
            Xiuzhu Wu, H. Que, Qingfang Li, Xiawei Wei
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>3</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed7953efc8b662a0f9dd292cd306d987c8eb0ba8" target='_blank'>
              Breast Cancer Clustering Integrating Complete Gene Expression Profiles and Genetic Ancestry
              </a>
            </td>
          <td>
            Johanna Stepanian, Alejandro Mejía-Garcia, Carlos A. Orozco, Jorge Duitama
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) displays remarkable cell state plasticity, a major contributor to therapeutic resistance and tumor progression. While epigenetic mechanisms play a central role in driving this plasticity, the key regulators remain poorly understood, and developing effective therapeutic strategies targeting them has been challenging.


METHODS
We investigated the role of BRD2, a key regulator of NF-κB mediated mesenchymal (MES) transition, using GBM patient-derived xenograft (PDX) cell lines, CRISPR-mediated knock-in/knockout approaches, RNA-seq, and in vitro and in vivo modeling. BET inhibitors were employed to target MES gene expression and sensitize GBM to radiation therapy.


RESULTS
We found that PTEN loss induces RelA chromatin localization and acetylation-mediated recruitment of BRD2 to the MES gene promoters. BRD2 binding is essential for maintaining MES gene expression and phenotype. Genetic ablation or loss-of-function mutation of BRD2 bromodomains reverses MES transition, enhances radiation sensitivity, and improves survival in orthotopic xenograft models. Additionally, treatment with a brain-penetrant BD2-selective inhibitor suppresses the MES phenotype and increases radiation sensitivity of GBM stem cells in vitro.


CONCLUSION
Our study identifies BRD2 as a key mediator of MES transition in GBM, with its bromodomains playing a crucial role in driving cell state plasticity. Targeting BRD2 with BD2-selective inhibitors offers a promising therapeutic strategy to overcome radiation resistance and improve outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa778fbdf5d6872b2c63bfc97f4589c5f7dfb10" target='_blank'>
              BRD2 Bromodomain-Mediated Regulation of Cell State Plasticity Modulates Therapy Response in Glioblastoma.
              </a>
            </td>
          <td>
            Raghavendra Vadla, Brett Taylor, Yohei Miyake, Benjamin Lin, Daisuke Kawauchi, Shunichiro Miki, Nidhi Nathwani, Brandon M Jones, Yashpreet Kaur, Abhinaba Banerjee, Philip Pham, J. Tsang, Albert Baldwin, David A Nathanson, Donald P. Pizzo, C. Miller, Frank B Furnari
          </td>
          <td>2025-07-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Objective DDX17, an ATP-dependent RNA/DNA helicase, is implicated in the regulation of RNA metabolism and has been linked to tumorigenesis and metastasis in various cancers. While studies have explored the role of DDX17 in specific cancers, further research is needed to understand its mechanisms across different cancer types. Methods We leveraged several public databases, including TIMER, The Cancer Genome Atlas (TCGA), and the Genotype-Tissue Expression (GTEx), to investigate DDX17 mRNA expression across 33 types of tumors. The GEPIA2 database was utilized to assess the impact of DDX17 on overall survival (OS) and disease-free survival (DFS) in patients with these tumors. Additionally, we employed cBioPortal to examine DDX17 gene alterations in various tumor tissues. Further analysis was conducted using the R language to explore the correlation between DDX17 and a range of clinical features, including tumor microenvironment (TME), immune regulatory genes, immune checkpoints, tumor mutational burden (TMB), microsatellite instability (MSI), DNA methylation, RNA methylation, and drug sensitivity. Gene Set Enrichment Analysis (GSEA) was also applied to elucidate the molecular mechanisms mediated by DDX17. Results DDX17 showed significant differences in expression between cancer and normal tissues. Expression of DDX17 was associated with patient prognosis, TMB, MSI, and drug sensitivity in certain cancers. DDX17 is additionally involved in modulating immune functions and influencing the tumor microenvironment. Further analysis of DDX17 mutation sites and types showed that the mutation frequency was highest in endometrial cancer and the major mutations of DDX17 were missense mutations. Conclusion These findings indicated that DDX17 may be considered a potential prognostic biomarker and a promising target for novel immunotherapeutic approaches in cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02955-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c500003015dd54739438713ee9cd7419ce63da8" target='_blank'>
              Prognostic and immunological landscape of DDX17 in pan-cancer analysis: a comprehensive study
              </a>
            </td>
          <td>
            Yu Lei, Xin He, Peng Chen, Yiping Qin, Bin Ge, Yan Liu, Pu Li, Xing Wei
          </td>
          <td>2025-06-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Neoantigen-specific T cells play a major role in cancer immunotherapy. Mutated proteins derived from non-synonymous mutations in tumor genomic DNA are the major source of neoantigens. It is conceivable that non-synonymous mutations found in tumor mitochondrial DNA (mtDNA) can also generate neoantigens that can be recognized by T cell receptors (TCRs). However, no tumor mitochondrial mutation-specific TCRs have been reported. As a proof-of-concept study, we obtained 3,798 non-synonymous single-nucleotide mutations across 38 tumor types through The Cancer Mitochondria Atlas project. These mutations were subjected to an epitope prediction algorithm to predict binding affinities for HLA-A*0201. The top 300 candidate peptides were synthesized and screened against 10 healthy donors with homozygous HLA-A*0201 alleles. Mutation-reactive T cells were subjected to single-cell sequencing to identify TCR sequences, followed by validations. Here, four MT-ND2 (A103T) mutation-specific TCRs were isolated from a donor. These TCRs interacted with HLA-A*0201-restricted IMMAMTMKL peptide. Additionally, through a single-cell sequencing approach, we demonstrated that a non-synonymous mutation, MT-CO1 (V274I), could be detected in nearly all tumor cells in a colorectal tumor specimen, whereas other mutations in 2 out of 4 tumors were detected in subclonal populations. This study suggests that isolating tumor mitochondrial mutation-specific TCRs is possible, but some biological barriers need to be considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1e1b3cf669d6e64ff59e594e8300039fb1835d" target='_blank'>
              Isolation of mitochondrial mutation-specific T cell receptors.
              </a>
            </td>
          <td>
            Catherine Kirkpatrick, Shanshan Hao, Charles M Quick, Steven R Post, Yuet-Kin Leung, L. Burdine, Yong-Chen William Lu
          </td>
          <td>2025-06-30</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) remains one of the most aggressive and therapeutically challenging subtypes of BC due to its lack of targeted treatment options and high plasticity. Increasing evidence suggests that epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNA regulation, play a pivotal role in facilitating TNBC phenotype by influencing the expression of genes involved in subtype reprogramming. Modulating these epigenetic marks offers a promising strategy to drive TNBC subtype conversion toward less aggressive forms that are more responsive to treatment, thereby enhancing therapeutic efficacy. By targeting enzymes such as DNA methyltransferases and histone modifiers using epigenetic drugs (epidrugs), TNBC cells can be re-sensitized to hormone therapy, chemotherapy, and targeted treatments. This review delves into the role of epigenetic modulation in harnessing the plasticity of TNBC to drive its conversion into subtypes with better prognoses and discusses problems and limitations associated with the use of epidrugs in this context. By promoting subtype conversion, epidrugs offer a promising strategy for providing more personalized and effective therapies for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ada35a90892ec0be34375adfcf309b46149ff70" target='_blank'>
              Epigenetic Therapeutics: Reprogramming Triple-Negative Breast Cancer into Responsive Subtypes.
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, A. Al-Azawi, L. Lozon, Rawan R. Kawaf, Ragheb Alsheikh Zein, Yahia El-Gharib, Raafat El-Awady
          </td>
          <td>2025-07-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) continues to be a major contributor to global cancer-related mortality, with KRAS and NRAS oncogenic mutations playing a pivotal role in tumorigenesis and therapeutic resistance. These genetic alterations drive uncontrolled proliferation and modulate responses to targeted treatments, significantly influencing clinical outcomes. This review examines the prevalence and prognostic implications of KRAS and NRAS mutations, with particular attention paid to the Venezuelan population. Reported frequencies in Venezuela (13–23% for KRAS and 1.6–2.6% for NRAS) are markedly lower than global averages (36–45% for KRAS), a discrepancy potentially attributable to population-specific genetic variations or methodological constraints of conventional sequencing techniques, which may lack sensitivity for variant detection. The presence of RAS mutations is a well-established biomarker of resistance to anti-EGFR monoclonal antibodies, necessitating alternative treatment regimens for affected patients. Although novel agents such as KRAS G12C inhibitors exhibit therapeutic potential, their efficacy remains under investigation in clinical trials, highlighting the need for regionally tailored drug development and improved access to innovative therapies in Venezuela. Beyond proliferative signaling, KRAS-mediated oncogenesis involves metabolic adaptation, angiogenic activation, and immune system evasion, underscoring its multifaceted role in CRC progression. Advanced molecular diagnostic techniques, such as next-generation sequencing (NGS) and circulating tumor DNA (ctDNA) analysis, have significantly improved the detection of oncogenic mutations, offering higher sensitivity and specificity compared to traditional methods. However, their widespread adoption in resource-limited settings remains hindered by infrastructural, economic, and technical barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191bb8159cd664cc4ac3633314f93ce9a0d86dba" target='_blank'>
              RAS oncogenes in Venezuelan colorectal cancer: detection methods and mutational profile review
              </a>
            </td>
          <td>
            Carlos Darío Ramírez, Daniela Rodríguez-Carrascal, Rafael Puche
          </td>
          <td>2025-06-25</td>
          <td>Academia Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutational signature analysis has greatly enhanced our understanding of the mutagenic processes found in cancer and normal tissues. As part of a recent study, we analyzed 802 treatment-naïve, microsatellite-stable colorectal cancers (CRC) and identified a de novo signature, SBS_D, which was conservatively decomposed into SBS18, a signature associated with reactive oxygen species. Here, we re-evaluate this decomposition and provide evidence that SBS_D represents a distinct mutational process from that of SBS18. Through an independent analysis of 2,616 whole-genome sequenced microsatellite-stable CRCs across three distinct cohorts, we demonstrate that SBS_D is consistently present at a similar prevalence, suggesting that this signature may have been previously overlooked. Using a naïve decomposition approach, we demonstrate that the pattern of SBS_D better aligns with signatures previously associated with deficiencies in DNA polymerase delta (POLD1) proofreading and mismatch repair. However, multiple lines of evidence, including the absence of pathogenic mutations in the exonuclease domain of POLD1 or in mismatch repair-associated genes, indicate that SBS_D is not driven by canonical defects in these DNA repair pathways. Overall, this study identifies a previously unrecognized mutational signature in microsatellite-stable CRC and proposes that its etiology may be linked to DNA repair infidelity emerging late in tumor development in samples without canonical defects in DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d7a0ecc45c91a460045023891d09c9a49fcad6" target='_blank'>
              Identification and Validation of a Previously Missed Mutational Signature in Colorectal Cancer
              </a>
            </td>
          <td>
            Mariya Kazachkova, Burçak Otlu, M. Díaz-Gay, Ammal Abbasi, Sarah Moody, Sandra Perdomo, David C Wedge, Paul Brennan, Michael R Stratton, Ludmil B. Alexandrov
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In precision medicine, DNA-based assays are currently necessary but not always sufficient for predicting therapeutic efficacy of cancer drugs based on the mutational findings in a patient’s tumor specimen. Most drugs target proteins, but it is challenging and not yet cost-effective to perform high-throughput proteomics profiling, including mutational analysis, on cancer specimens. RNA may be an effective mediator for bridging the “DNA to protein divide” and provide more clarity and therapeutic predictability for precision oncology. While RNA sequencing (RNA-seq) has been increasingly used alongside DNA cancer mutation screening panels to assess the impact of variants on gene transcript expression and splicing, comprehensive evaluations of RNA panels and the integration of expressed mutation data analytics to supplement DNA panels are still limited. In this study, we conducted targeted RNA-seq on a reference sample set for expressed variant detection to explore its potential capability to complement DNA variant results or detect variants independently. The results indicated that, with a carefully controlled false positive rate ensuring high accuracy, RNA-seq uniquely identified variants with significant pathological relevance that were missed by DNA-seq, demonstrating its potential to uncover clinically actionable mutations. On the other hand, while some variants were detected by both approaches, others were missed by one or the other, reflecting either the nature of these variants or limitations of the bioinformatics tools used. Variants missed by RNA-seq are often not expressed or expressed at very low levels, suggesting they may be of lower clinical relevance. Incorporating RNA-seq into clinical biomarker panels will ultimately advance precision medicine and improve patient outcomes by improving the strength and reliability of somatic mutation findings for clinical diagnosis, prognosis and prediction of therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f90149304c1d7c32369d59f225270973f7ce13" target='_blank'>
              Augmenting precision medicine via targeted RNA-Seq detection of expressed mutations
              </a>
            </td>
          <td>
            Dan Li, Jianying Li, Donald J. Johann, Daniel Butler, Guangchun Chen, J. Foox, Binsheng Gong, W. Jones, David P. Kreil, Rebecca Kusko, Paweł P. Łabaj, A. Lucas, Christopher E. Mason, C. Mozsary, N. Novoradovskaya, C. Pabón-Peña, Bohu Pan, Todd A. Richmond, Roberta Maestro, Sayed Mohammad Ebrahim Sahraeian, Andreas Scherer, Hagen U. Tilgner, James C. Willey, P. Bushel, Joshua Xu
          </td>
          <td>2025-06-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Oncogenic retroviruses are known for their pathogenesis via insertional mutagenesis, in which the presence of a provirus and its transcriptional control elements alter the expression of a nearby or surrounding host gene. There are reports of proviral integration driving oncogenesis in people with HIV and the use of HIV-derived vectors for gene therapy has raised concern about oncogenic side effects. To study this issue, we used an in vitro primary human CD4 + T cell infection model developed in our laboratory to identify HIV-1 integration sites that might influence cell proliferation or survival. Combining integration site analysis and bulk RNA sequencing, we found that an upregulated STAT3 signature due to proviral insertional mutagenesis was associated with persistent HIV-infected CD4 + T cells. HIV+ persistent cells also expressed a STAT3-related anti-apoptotic and cytotoxic phenotype that resembles that of HIV-associated T cell lymphomas. HIV insertional mutagenesis of STAT3 and expression of its downstream targets provides a model of HIV-associated T cell lymphomas that can be used to further determine the oncogenic drivers of HIV-associated lymphomas, both AIDS- and gene therapy-associated, and, potentially, to evaluate therapeutics against these HIV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309400b908a0604bb16da7990788bd142c72eac7" target='_blank'>
              In vitro HIV DNA integration in STAT3 drives T cell persistence—A model of HIV-associated T cell lymphoma
              </a>
            </td>
          <td>
            M. Rist, M. Kaku, John M. Coffin
          </td>
          <td>2025-07-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes.
              </a>
            </td>
          <td>
            Steven A Frank
          </td>
          <td>2025-08-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ee072bd4b29ca666b639b84ed04de31adafdbe" target='_blank'>
              Discovery and validation of non-canonical antigens for Hepatocellular Carcinoma immunotherapy
              </a>
            </td>
          <td>
            Stephen Li, Yujuan Dong, J. Liu, Shanglin Li, Dandan Pu, Yue Guo, C. Wong, Nathalie Wong, Jason Wing, Hon Wong
          </td>
          <td>2025-06-09</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lenvatinib, a multiple-receptor tyrosine kinase inhibitor, has gained recent approval for its use as a first-line treatment of hepatocellular carcinoma (HCC). While lenvatinib demonstrates notable therapeutic efficacy, the drug resistance undermines its sustained tumor control potential. The restricted clinical utility of lenvatinib underscores the imperative necessity to elucidate the mechanisms underpinning drug resistance. We established lenvatinib-resistant cell lines and investigated the changes in their biological characteristics. Next-generation sequencing was performed to identify genes associated with lenvatinib resistance. Western blots were utilized to confirm the involvement of these genes. Using lentiviral technology, we generated cell lines with lowered nuclear receptor coactivator 5 (NCOA5), a pivotal drug resistance-related gene, to explore the underlying resistance mechanism. Moreover, we developed a subcutaneous HCC xenograft tumor model to explore strategies for reversing drug resistance. Our study showed that HCC cells acquire resistance to lenvatinib through the activation of NCOA5, thereby stimulating the NCOA5-Protein Kinase B-mammalian target of rapamycin (AKT-mTOR) axis. Furthermore, the clinical evaluation of HCC specimens established a correlation between the activation of the NCOA5 pathway and the response to lenvatinib treatment. Everolimus, an mTOR inhibitor, in combination with lenvatinib and everolimus, exerted significant synergistic effects against HCC in vivo and in vitro. HCC cells develop resistance to lenvatinib by activating the NCOA5-AKT-mTOR pathway. The combination therapy of lenvatinib with everolimus is a promising strategy to overcome acquired resistance, thereby enhancing the clinical efficacy of lenvatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4645ca240d89c81caab05ab94ee824d557e8c782" target='_blank'>
              Inhibition of nuclear receptor coactivator 5 overcomes acquired lenvatinib resistance driven by protein kinase B-mammalian target of rapamycin signaling in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Hongyuan Zhou, Qin Zhang, Lu Yang, Zhao-Long Pan, Haijing Zheng, Zewu Zhang, Dongyang Li, Guangtao Li, Xiaomeng Liu, Xu Bao, Chen Liu, Wei Zhang
          </td>
          <td>2025-07-31</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aberrant activation of the RAS/MAPK signaling limits the clinical efficacy of several targeted therapies in acute myeloid leukemia (AML). In FLT3-mutant AML, the selection of clones harboring heterogeneous RAS mutations drives resistance to FLT3 inhibitors (FLT3i). RAS activation is also associated with resistance to other AML targeted therapies, including the BCL2 inhibitor venetoclax. Despite the critical need to inhibit RAS/MAPK signaling in AML, no targeted therapies have demonstrated clinical benefit in RAS-driven AML. To address this unmet need, we investigated the preclinical activity of RMC-7977, a multi-selective inhibitor of GTP-bound active [RAS(ON)] isoforms of mutant and wild-type RAS in AML models. RMC-7977 exhibited potent antiproliferative and pro-apoptotic activity across AML cell lines with MAPK-activating signaling mutations. In cell line models with acquired FLT3i resistance due to secondary RAS mutations, treatment with RMC-7977 restored sensitivity to FLT3i. Similarly, RMC-7977 effectively reversed resistance to venetoclax in RAS-addicted cell line models with both RAS wild-type and mutant genetic backgrounds. In murine patient-derived xenograft models of RAS-mutant AML, RMC-7977 was well tolerated and significantly suppressed leukemic burden in combination with gilteritinib or venetoclax. Our findings strongly support clinical investigation of broad-spectrum RAS(ON) inhibition in AML to treat and potentially prevent drug resistance due to activated RAS signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad770e71268541d6923a6a441040fe4dc007ffe" target='_blank'>
              Multi-selective RAS(ON) Inhibition Targets Oncogenic RAS Mutations and Overcomes RAS/MAPK-Mediated Resistance to FLT3 and BCL2 Inhibitors in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bogdan Popescu, Matthew F. Jones, Madison Piao, Elaine Tran, Andrew Koh, Isabelle Lomeli, C. Peretz, Natalia Murad, Sydney Abelson, Carolina E. Morales, Jose M. Rivera, Y. Pikman, Michael L. Cheng, Aaron C. Logan, Elliot Stieglitz, Catherine C. Smith
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Cancer development and progression are driven by the accumulation of somatic genetic alterations, which occur in a specific temporal order. However, how the order of mutations impacts cancer phenotypes of solid tumors remains poorly understood. To address this, we developed a novel computational framework, IMOP-Cancer (Identifying Mutation Order Pairs in Cancer), to identify mutation gene pairs whose order influences cancer phenotypes. We applied IMOP-Cancer to The Cancer Genome Atlas-Lung Adenocarcinoma (TCGA-LUAD) cohort and identified 446 key mutation order pairs, with 34 pairs significantly associated with prognosis. Mutation order impacts cancer phenotypes, as demonstrated by CSMD3 and PTPRD (tumor proliferation) and TP53 and NAV3 (immune modulation), with effects validated in four independent datasets. We further presented the impact of mutation pairs on cancer phenotypes through case studies in the TCGA cohorts of bladder urothelial carcinoma (BLCA), and colon adenocarcinoma (COAD). We extended this analysis to 33 cancer cohorts from TCGA portal, identifying 106 034 critical mutation pairs across 17 cancers, with 3036 pairs co-occurring in multiple cancers. Shared mutation pairs across cancers also showed distinct effects on cancer phenotype. Our study highlights the importance of mutation order in cancer progression and diversity, offering new insights into the temporal dynamics of co-occurring mutations and paving the way for personalized treatment strategies and improved diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348ae5004fc1076373867ea177acfe998124027" target='_blank'>
              IMOP-Cancer: identifying mutation order pairs impacting cancer phenotypes
              </a>
            </td>
          <td>
            Yijing Zhang, Shaobo Kang, Renjie Dou, Wanmei Zhang, Yuanyuan Liu, Yang Wu, Dongxue Li, Fangfang Fan, Yanyan Ping
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Chemoresistance, the primary cause of mortality among ovarian cancer (OC) patients, is a multifaceted process encompassing numerous biological phenomena. As sequencing technology continues to advance, single-cell sequencing has surfaced as a potent strategy to elucidate the pathogenesis of OC. Methods We examined single-cell sequencing data derived from five OC samples (three resistant and two sensitive) and identified an epithelial subcluster associated with chemotherapy resistance and poor prognosis. Using GSVA and cell communication analysis, we explored the unique biological functions and communication characteristics of this resistant subcluster. We performed high dimensional weighted gene co-expression network analysis and differential expression analysis to identify the hub genes of c3. Lastly, we investigated the correlation between the hub gene, CLIC3, and chemotherapy drug sensitivity. We also validated their involvement in specific pathways using TCGA data. The effects and primary mechanism to chemoresistance of CLIC3 was explored. Results We identified a cell subcluster, denoted as c3, strongly linked to chemoresistance and poor prognosis in OC. This subcluster demonstrated a correlation with both extracellular matrix (ECM) formation and angiogenesis signature, with CLIC3 identified as its key marker. The expression levels of CLIC3 exhibit a significant association with the sensitivity to various chemotherapeutic drugs in OC. Mechanistically, CLIC3 increases OC resistance to cisplatin by promoting integrin β1 redistribution and PI3K-AKT pathway. Conclusions This study offers a novel insight into the progression and chemoresistance of OC. Additionally, we identified a specific cell cluster highly associated with chemoresistance. The marker for this cluster, CLIC3, increases OC resistance to cisplatin by promoting integrin β1 redistribution and PI3K-AKT pathway and holds significant potential as a new therapeutic target for OC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02882-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f56e8cc05a82444a4de8832553570aed3f3323" target='_blank'>
              Single-cell transcriptomics identify a chemotherapy-resistance related cluster overexpressed CLIC3 in ovarian cancer
              </a>
            </td>
          <td>
            Zhefeng Li, Jie Li, Yue Li, Junjie Yi, Xiaoting Zhao, Wentao Yue
          </td>
          <td>2025-06-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Integrating genome-wide association studies (GWAS) and transcriptomic datasets can identify potential mediators for germline genetic risk of breast cancer. However, traditional methods have been largely unsuccessful at most loci because of an overreliance on total gene expression. These approaches overlook alternative splicing, which can produce multiple isoforms from the same gene, each with potentially different effects on cancers. Here, we integrate genetic and multi-tissue isoform-level expression from the Genotype Tissue-Expression Project (GTEx, N = 838) with publicly available European-ancestry GWAS summary statistics to identify both isoform- and gene-level risk associations with overall breast cancer and estrogen receptor-positive (ER+) and -negative (ER-) subtypes. We employ our recently developed isoform-level Transcriptome-Wide Association Study (isoTWAS) framework that jointly models germline genetic variation, all isoforms of a gene, and phenotypic associations to identify transcriptomic mediators of genetic risk loci. Directly modeling isoform expression substantially increases discovery of transcriptomic mechanisms underlying genetic associations, compared to traditional methods leveraging total gene expression. Across the 3 indications, isoTWAS identified 2,304 more unique genes than TWAS, reflecting an 34.3% increase (jackknife SE = 3.8%) in effective sample size. In total, we find that isoTWAS-prioritized genes are enriched for pathways involved in DNA repair and damage responses, apoptosis, DNA and transcription factor binding, and cell cycle regulation. In addition, these genes are enriched for transcription binding sites of oncogenic transcription factors, like RUNX2, ESR1, and MYC. Critically, isoTWAS is able to pinpoint an isoform-specific mediation of the ESR1 and FOXA1 loci using isoform expression of non-cancerous breast tumor. Next, isoTWAS associations are enriched for genes that are evolutionary constrained and showing intolerance to protein-truncating variation (pLI score 0.90), prioritizing 587 constrained genes compared to 153 by TWAS (P = 2.8 x 10-9). This result suggests that isoform associations capture more likely disease signal. Of the 309 independent GWAS loci across the 3 indications, isoTWAS tagged 42% additional GWAS loci with transcriptomic associations (221 loci with isoTWAS, 156 with TWAS, 124 in common). Relatedly, the cis-genetic component of isoform expression (20.8 ± 7.3%) mediates an estimated 92.4% more of cancer risk SNP heritability compared to that of gene expression (10.8 ± 2.6%), using Mediated Expression Score Regression. Lastly, we highlight several salient and novel isoTWAS associations that exhibit isoform but not gene expression colocalization, including BABAM1 and FDPS, associated with overall breast cancer risk, and L3MBTL3 and CASP8, associated with ER- breast cancer risk. Our results strongly demonstrate that modeling isoform expression is crucial to maximize discovery of genetic risk mechanisms for breast cancer that are testable and actionable.
 Citation Format: Taylor Head, Yung-Han Chang, Tabitha Harrison, Yao Yu, Chad D. Huff, Bogdan Pasaniuc, Sara Lindström, Arjun Bhattacharya. Isoform-level analyses of breast cancer and its subtypes uncover extensive genetic risk mechanisms undetected at the gene-level [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS16-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2fd47608517f7db55237fbed695d206620b97ed" target='_blank'>
              Abstract PS16-03: Isoform-level analyses of breast cancer and its subtypes uncover extensive genetic risk mechanisms undetected at the gene-level
              </a>
            </td>
          <td>
            Taylor Head, Yung-Han Chang, Tabitha Harrison, Yao Yu, Chad D. Huff, Bogdan Pasaniuc, Sara Lindström, Arjun Bhattacharya
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="RET gene rearrangements yield oncogenic fusion proteins that drive a subset of lung adenocarcinomas (LUAD). The tyrosine kinase inhibitors (TKIs) selpercatinib and pralsetinib are approved therapies for RET+ lung cancers and have markedly improved clinical outcomes in these patients, but acquired resistance remains a hurdle to their durable management. Using a recently developed murine model of RET+ lung cancer driven by a Trim24-Ret fusion protein, two Trim24-Ret cell lines (TR.1 and TR.2) were established. Orthotopic lung tumors generated by transplantation of these cell lines initially respond to selpercatinib followed by prompt progression within ∼3 weeks of initiating TKI treatment. Cell lines derived from the selpercatinib-resistant TR.1 and TR.2 tumors exhibited in vitro sensitivity to MET and ERBB-targeted TKIs, indicating acquired bypass signaling through these receptor tyrosine kinases. Moreover, the selpercatinib-resistant TR.1 and TR.2 cell lines exhibited increased sensitivity to MEK and PTPN11 inhibitors relative to the parental cell lines, indicating a greater dependence on MAPK pathway signaling. The TKI-resistant cell lines showed no evidence for MET gene amplification, but exhibited varied induction of multiple genes that function within MET and ERBB2:ERBB4 interaction networks including ligands (HGF, NRG1), adaptors (GAB1) and co-receptors (NRP1). Consistent with an important role for MET signaling in driving acquired selpercatinib resistance, mice bearing orthotopic lung tumors derived from TR.1 or TR.2 cells that had progressed on selpercatinib treatment underwent significant re-shrinkage upon co-treatment with the MET inhibitor, crizotinib, although progression re-occurred. By contrast, upfront treatment with selpercatinib and crizotinib in orthotopic tumors yielded complete elimination of 78% of TR.1 tumors and a prolonged duration of response in TR.2 tumors. The findings highlight the failings inherent in treating acquired resistance mechanisms at progression and the potential therapeutic impact of predicting and targeting dominant mechanisms of resistance prior to or early after initiating oncogene-targeting TKI treatment in RTK-driven LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c46a9a33fd1358451f6b77308b227944809fc93" target='_blank'>
              Modeling acquired TKI resistance and effective combination therapeutic strategies in murine RET+ lung adenocarcinoma
              </a>
            </td>
          <td>
            T. Hinz, A. Le, Tristan Doan, Abby Ast, Sophia Jaramillo, Samantha Haines, Andre Navarro, Tejas Patil, Raphael A. Nemenoff, Lynn E. Heasley
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The epigenetic deregulation of CpG islands (CGIs) plays a crucial role in cancer initiation and progression. CGIs comprise 1-2% of the human genome and are rich in differentially methylated regions (DMRs) that can serve as cancer biomarkers in clinical samples and liquid biopsies. Focusing epigenetic sequencing on CpG-rich sequences, including CGIs and avoiding non-informative regions, offers an efficient and sensitive approach for cancer identification and tracking, especially within samples containing excess of unaltered, normal DNA. To this end, we have developed Adaptor-anchored Methylation amplification via Proximity Primers (aMAPP), a versatile PCR-based enrichment method. aMAPP employs specially designed primers to selectively enrich either methylated or unmethylated CpGs, depending on the upstream methylation conversion method employed. aMAPP achieves high coverage of genome-wide CGIs and detects hundreds of DMRs in tumor samples compared to adjacent normal tissue using ultra-low depth sequencing (∼300,000 reads). It enables tracing of aberrant methylation down to allelic frequency 0.01% in dilutions of tumor DNA and in cell-free DNA samples, can be applied using picogram amounts of DNA, and can be adapted to enrich either small panels of cancer-specific DMRs, or the majority (>90%) of genomic CGIs and CpGs. aMAPP offers a simple, cost-effective, and highly sensitive approach for capturing the epigenetic footprint of genome-wide CpGs and identifying aberrantly methylated or un-methylated genomic regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Therapeutic resistance remains a primary obstacle to curing cancer. Healthy cells exposed to genotoxic insult rapidly activate both p53-dependent and -independent non-genetic programs that pause the cell cycle and direct either DNA repair or apoptosis. Cancer cells exploit these same pathways as they respond to stresses induced by cancer therapies. In this study, we investigated a potential role for upstream stimulatory factor 1 (USF1) and USF2 in the p53-independent response of lymphoma cells to genotoxic therapy. We previously found that lymphocytes utilize the responsiveness of USF1 to double-stranded DNA breaks to coordinate T cell receptor beta (Tcrb) gene expression during V(D)J recombination. Here, microarray gene expression analysis of derivatives of the p53-deficient mouse B lymphoma cell line, M12, revealed that simultaneously depleting cells of both USF1 and USF2 altered the expression of 940 gene transcripts (>1.50-fold change, < 0.05 FDR), relative to cells expressing a scrambled control shRNA. Seven days after exposure to a single sublethal (5 Gy) dose of ionizing radiation, USF-depleted (USFKD) cells exhibited widespread and distinct transcriptional responses from those of irradiated controls (5035 and 5054 differentially expressed gene transcripts, respectively, with roughly half shared between both cell types). Gene ontology analyses revealed that USF knockdown induced numerous changes in the expression of genes critical for immune development and function while diminishing the responsiveness of genes linked to DNA damage pathways. Microarray findings were confirmed by RT-qPCR for a panel of genes responsive to USF knockdown and/or irradiation. These findings shed further light on transcriptional responses to ionizing radiation that manifest over time in transformed cells, identifying a novel p53-independent role in lymphocytic DNA damage stress responses for USF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6bb6187205d63162ff7fe5e73d3f94fd90940ba" target='_blank'>
              Knockdown of USF1 and USF2 drives prolonged changes in the gene expression response of M12-5B3 cells to DNA damage
              </a>
            </td>
          <td>
            Kimberly S Bellingham-Johnstun, Lisa A Metzger, Jennifer L Stone, Kevin Gerrish, Pierre R Bushel, Michael L Sikes
          </td>
          <td>2025-07-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Breast cancer (BC) is the leading cause of cancer-related deaths in women. It is molecularly and clinically heterogeneous, driven by distinct genetic and epigenetic alterations, making prognosis prediction and treatment selection challenging. Identifying novel markers for BC progression and therapeutic targets remains an urgent need. Although growing research has highlighted the role of epigenetic mechanisms, particularly cancer-associated histone mutations (onco-histone), in regulating BC pathogenesis and its hallmarks, including stemness and drug resistance, the role of histone variants—the naturally derived sequence variants—remains largely unknown, with the exception of H3.3. The newly identified, mammalian-specific histone variant H2A.J has been shown to be highly enriched in human luminal BC and implicated in a stem cell signature correlated with metastatic risk in primary BC. However, it is unclear what role H2A.J plays in BC and how it may function in other BC subtypes. Using cell lines of multiple BC subtypes and patient-derived xenograft (PDX) tumor microarrays, our studies have discovered that H2A.J is present in human triple-negative BC (TNBC), with significantly elevated levels in metastatic TNBC compared to the primary counterpart. Our unpublished results show that H2A.J is the only histone variant that strongly and specifically interacts with the tumor suppressor, retinoblastoma 1 (RB1) in metastatic TNBC cells and luminal BC. Our data also demonstrate that RB1 protein level is significantly correlated with H2A.J in estrogen receptor-positive (ER+) luminal BC PDX samples. Our preliminary results suggest that H2A.J chromatin occupancy affects RB1 recruitment to chromatin, RB1-bound chromatin targets, and ultimately RB1-dependent phenotypes. We will discuss how H2A.J regulates RB1-bound chromatin loci, and hence, the canonical and non-canonical functions of RB1 in tumorigenesis. Our bulk RNA-sequencing and western blotting data show that knocking down (KD) H2A.J interferes with G1/S cell cycle progression and results in the upregulation of estrogen response, apoptosis, and mTOR signaling. Notably, the H2A.J KD-transcriptome signature strongly correlates with that of CDK4/6 inhibitor treatment. Our data also suggest H2A.J's role in maintaining cell lineage fidelity. Overall, our results suggest that H2A.J regulates cellular heterogeneity, stemness, and metastasis in BC. The outcome of our research will not only advance our understanding of the underlying mechanisms leading to BC progression and metastasis but also offer new opportunities for utilizing H2A.J as a target for diagnostic and therapeutic interventions in BC.
 Citation Format: Rana Gbyli, Tyler Jensen, Joseph Balowski, Yao Li, Xiang Zhang, Qin Yan, Andrew Xiao. Harnessing the Power of the Variant: Breast Cancer's Strategic Incorporation of the Histone H2A.J [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6e23951bd3e66f36e177a58521abd15ef2dc11" target='_blank'>
              Abstract P5-06-22: Harnessing the Power of the Variant: Breast Cancer's Strategic Incorporation of the Histone H2A.J
              </a>
            </td>
          <td>
            Rana Gbyli, Tyler Jensen, Joseph Balowski, Yao Li, Xiang Zhang, Qin Yan, Andrew Xiao
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Polymerase theta (POLθ) inhibitors were developed to overcome resistance to PARP inhibitor treatment in homologous recombination (HR) deficient cancer. Biomarkers identifying PARP inhibitor-resistant cancer cases that specifically rely on POLθ activity for cancer cell survival will help to identify the sensitive patient population. From the TCGA RNAseq data, we determined POLθ expression levels, and from whole-exome and whole-genome sequencing data, we determined the number of POLθ-associated mutational signatures in solid tumors with various HR deficiency status. We found that POLθ expression levels did not differ significantly between HR-proficient and HR-deficient cancers. However, POLθ expression correlated strongly with proliferation-associated gene expression signatures and was predominantly observed in the S and G2/M phases of the cell cycle. POLθ-associated mutational signatures are correlated with POLθ expression levels only in BRCA2-deficient cancers. POLθ expression level and POLθ-associated mutational signatures may be indicative of POLθ inhibitor sensitivity in BRCA2-deficient tumors, but are unlikely to be informative in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498ae751ba9f700cb39c5f192432c94acf2d269" target='_blank'>
              Polymerase theta expression is correlated with proliferative capacity but not with DNA repair deficiency status in solid tumors
              </a>
            </td>
          <td>
            Z. Sztupinszki, Regina Fiam, O. Pipek, M. Dióssy, J. Börcsök, Aurél Prósz, I. Csabai, Zoltan Szallasi
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1edd21201b5143958af0631e20736d1ea970c7b" target='_blank'>
              Heterogeneity of Genetic Sequence within Quasi-species of Influenza Virus Revealed by Single-Molecule Sequencing
              </a>
            </td>
          <td>
            Kenji Tamao, Hiroyuki Noji, K. Tabata
          </td>
          <td>2025-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background/Aim: Recent advancements in omics analysis have significantly enhanced our understanding of the molecular pathology of malignant melanoma, leading to the development of novel therapeutic strategies that target specific vulnerabilities within the disease. Despite these improvements, the factors contributing to the poor prognosis of patients with malignant melanoma remain incompletely understood. The aim of this study was to investigate the role of C1orf50 (Chromosome 1 open reading frame 50), a gene previously of unknown function, as a prognostic biomarker in melanoma. Materials and Methods: We performed comprehensive transcriptome data analysis and subsequent functional validation of the human Skin Cutaneous Melanoma project from The Cancer Genome Atlas (TCGA). Results: Elevated expression levels of C1orf50 correlated with worse survival outcomes. Mechanistically, we revealed that C1orf50 plays a significant role in the regulation of cell cycle processes and cancer cell stemness, providing a potential avenue for novel therapeutic interventions in melanoma. Conclusion: This study is the first to identify C1orf50 as a prognostic biomarker in melanoma. The clinical relevance of our results sheds light on the importance of further investigation into the biological mechanisms underpinning C1orf50’s impact on melanoma progression and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97ee80e750fee0c874b5ff3d0f859ce68b0b3f4" target='_blank'>
              C1orf50 Drives Malignant Melanoma Progression Through the Regulation of Stemness
              </a>
            </td>
          <td>
            Yusuke Otani, Masaki Maekawa, Atsushi Tanaka, Tirso Peña, Vanessa D. Chin, Anna Rogachevskaya, Shinichi Toyooka, M. H. Roehrl, Atsushi Fujimura
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers, with an incidence that has increased significantly over the last 20 years. The World Health Organization (WHO), under its cancer program, identifies CRC as the third most prevalent cancer worldwide with a high mortality rate, especially in patients under the age of 50. Despite advances in surgery, chemotherapy, radiotherapy, and molecular targeted therapy, CRC shares a low survival rate and poor prognosis due to late diagnosis. To address these challenges, research into alternative diagnostic and therapeutic strategies has increasingly focused on molecular mechanisms, including noncoding RNAs (ncRNAs). Circular RNAs (circRNAs), a subclass of endogenous ncRNAs characterized by their covalently enclosed loop structure, demonstrate greater stability than linear RNAs, making them potential candidates for clinical applications. The circRNAs possess differential expressions in cancers and function as tumor suppressive or oncogenic activities. This review discusses the recent findings on circRNAs and their potential for biomarkers and therapeutic targets in CRC. As circRNAs can serve as miRNA sponges, increase or decrease mRNA expression, and function to regulate an oncogenic or tumor suppressor pathway, there may be the possibility for an early diagnosis, prognosis, or therapeutic role of circRNAs in CRC. Highlighting the clinical implications of circRNAs, this review discusses their potential to transform current CRC management strategies and address critical gaps in timely diagnosis and effective treatment. It also emphasized the need for further clinical research to validate their utility and translate these findings into practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db80f20f575f8c248f315abb0f46ec8b7f78d57b" target='_blank'>
              A review of circular RNAs in colorectal cancer: insights into biomarker discovery and therapeutic targeting
              </a>
            </td>
          <td>
            Sibin Nambidi, Karunakaran Bhuvaneswari Shruthi, Sujatha Padmanaban Kanimozhe, Antara Banerjee, S. Kondaveeti, A. Duttaroy, S. Pathak
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Simple Summary Lynch syndrome (LS) is an autosomal dominant disorder resulting from germline pathogenic variants in the DNA mismatch repair (MMR) or the EPCAM genes. The DNA MMR gene PMS2 has been defined as a low-penetrance gene for cancers associated with LS. Here, we describe two unrelated patients with a personal and family history of cancer who harbor a novel PMS2 missense variant (c.184G>A; p.Gly62Arg) with uncertain clinical significance. To evaluate the effect of this novel variant, we used an integrated approach combining the genetic and clinical findings of the affected families, the tumor phenotype, and a bioinformatics analysis to predict its impact on protein structure and stability. Tumor molecular characterization and bioinformatics analyses indicate that this variant negatively affects PMS2 structural stability and functionality within the MMR system. Integrating germline genetic data, clinical phenotypes, tumor characteristics, and bioinformatics analyses, we provide relevant evidence suggesting that this novel PMS2 variant is linked to LS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23fe7fde6a2cc774f9a2f617efab99b5705cafb8" target='_blank'>
              An Integrated Clinical, Germline, Somatic, and In Silico Approach to Assess a Novel PMS2 Gene Variant Identified in Two Unrelated Lynch Syndrome Families
              </a>
            </td>
          <td>
            C. Fasano, A. L. Buonadonna, G. Forte, Martina Lepore Signorile, V. Grossi, Katia De Marco, P. Sanese, Andrea Manghisi, Nicoletta Maria Tutino, R. Armentano, A. Valentini, Vittoria Disciglio, Cristiano Simone
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c635cae84c7bdc8795ab539e7b406a70e03c5a4" target='_blank'>
              A transient mutational burst occurs during yeast colony development.
              </a>
            </td>
          <td>
            Nicolas Agier, Nina Vittorelli, Louis Ollivier, Frédéric Chaux, Alexandre Gillet-Markowska, Samuel O'Donnell, Fanny Pouyet, G. Fischer, Stéphane Delmas
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is driven by complex oncogenic mechanisms, including aberrant signaling pathways and genetic mutations. Current therapies often face limitations, such as drug resistance and adverse effects, necessitating the identification of novel therapeutic agents. Protopine, known for its anticancer properties in other malignancies, remains unexplored for breast cancer. This study investigates the therapeutic potential of protopine in breast cancer by examining its effects on critical molecular targets and signaling pathways involved in tumor progression and metastasis through computational pharmacological approaches. Breast cancer‐associated targets were identified, and protopine's binding affinity for PIK3CA was evaluated using molecular docking and molecular dynamics simulations. Binding stability and dynamics were assessed using RMSD, hydrogen bonding analysis, and MM‐PBSA energy calculations. Protopine exhibited multitargeted activity, interacting with 17 breast cancer‐associated targets, including PIK3CA, CHEK1, and CDK2. It modulated critical pathways such as PI3K‐Akt, JAK‐STAT, and VEGF, disrupting processes like cell proliferation and angiogenesis. Molecular docking showed strong binding affinity for PIK3CA (−9.2 kcal/mol), while molecular dynamics confirmed stable binding with favorable free energy (−6.71 ± 0.64 kcal/mol). Protopine demonstrated significant therapeutic promise as a multitargeted agent, modulating key oncogenic pathways and providing the potential to overcome resistance and improve breast cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfa9c411160ba1ed4ea959d12995e9684354661" target='_blank'>
              Protopine as a Potential Breast Cancer Therapeutic: A Computational Study Targeting PIK3CA and Cancer Networks
              </a>
            </td>
          <td>
            Jagdish Chand, Hannah Lalengzuali Fanai, Sheikh F. Ahmad, Sabry M. Attia, T. Emran
          </td>
          <td>2025-07-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is among the most common oncological diseases in the world, associated with a high mortality rate. Recently, immunotherapeutic approaches to the treatment of CRC have been developed, which have enabled achievement of long-term and stable remission in a certain group of patients. The success of immunotherapy depends on the immune processes associated with oncogenesis in the colon. The key role in modulating immune environment of the tumor is played by the activity of genes regulating differentiation and functions of immune cells, as well as by a variety of microRNAs that perform posttranscriptional expression control of target genes. In this regard, the aim of our work was to study the expression profiles of immune-associated genes and microRNAs to determine the immune signaling mechanisms of colon cancer and search for potential therapeutic targets. The study included 18 persons (12 women, 6 men, median age – 66 years) diagnosed with colon cancer, having been treated at the National Medical Research Center of Oncology in 2018-2019. All tumors had microsatellite instability status. The transcription profiles of microRNAs and immune response genes were determined by next-generation sequencing. The study has revealed 28 differentially expressed microRNAs in the tumor tissues, including 15 microRNAs with increased expression and 13 microRNAs with decreased expression. Of the 395 genes examined, 156 were differentially expressed, including 62 genes with increased expression and 94 genes with decreased activity. Identification of cell populations based on the transcriptional pattern of tumors has discerned four types of immune cells: neutrophils, B lymphocytes, CD8+T lymphocytes, and M1 macrophages. Neutrophils were the most common cell type (16 of 18 samples). A stepwise analysis of possible relationships between transcriptional patterns allowed us to select 713 miRNA-mRNA pairs. After filtering by validated interactions, 24 such pairs were found. On this basis, a miRNA-mRNA interaction network was constructed, which could be involved into the regulation of multiple immune signaling mechanisms and cell cycle control. Most of the revealed signaling pathways contained CDK1, for which therapeutic inhibitors are known, being at different stages of clinical trials. The possibilities of influencing CDK1 described in the present work may be used for further clinical studies and development of therapeutic strategies employing selective molecular suppression of immunemediated carcinogenesis in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ac153b8c05d98ac275bf9faf7493386df7f7150" target='_blank'>
              MicroRNAs and target genes as regulators of colon cancer immune signaling
              </a>
            </td>
          <td>
            O. I. Kit, D. Gvaldin, N. A. Petrusenko, N. Timoshkina, D. A. Savchenko, I. A. Novikova
          </td>
          <td>2025-07-07</td>
          <td>Medical Immunology (Russia)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance to CDK4/6 inhibitors (CDK4/6i) is a major challenge in treating hormone receptor-positive, HER2-negative metastatic breast cancer. This study aimed to identify a biomarker predictive of resistance that could also serve as a therapeutic target. Circulating IL-6 levels in 166 patients significantly increased at progression, making IL-6 a non-invasive biomarker to predict CDK4/6i resistance. Knockdown of IL-6 re-sensitized resistant cells to palbociclib and endocrine therapy, underscoring IL-6’s critical role in maintaining resistance. Patient-derived xenograft models from patients who progressed within 3 months versus ≥6 months of palbociclib therapy revealed distinct transcriptomic profiles, with later progressors exhibiting IL-6/STAT3 activation, epithelial-to-mesenchymal transition, and immune evasion signatures. Treatment with TTI-101, a STAT3 inhibitor, significantly reduced tumor growth and improved survival in these models, providing preclinical validation for targeting the IL-6/STAT3 axis. These findings support using IL-6 to guide personalized treatment and combining STAT3 inhibitors with CDK4/6i as a transformative strategy to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08735c14db476a16d97e533ff51d162f3e014327" target='_blank'>
              IL-6 predicts CDK4/6 inhibitor resistance, identifying STAT3 as a target in HR + /HER2-negative metastatic breast cancer
              </a>
            </td>
          <td>
            N. Kettner, Tuyen N Bui, Juliana Navarro-Yepes, Sanaz Ghotbaldini, Bethanie Quintela, Catherine K. Luo, Nghi Lam, X. Rao, A. Raghavendra, Yan Wang, Nancy Azizian, T. Kris Eckols, Moses M. Kasembeli, K. Evans, Min Yi, Hannah F Wingate, Jing Wang, Aysegul A. Sahin, Funda Meric-Bernstam, Kelly K Hunt, S. Damodaran, David J Tweardy, Debu Tripathy, K. Keyomarsi
          </td>
          <td>2025-07-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e86dbb7bd01d6e90b441b62dcc070c2bc321518" target='_blank'>
              High-fidelity long-read sequencing reveals a complex RCCX locus at the single-nucleotide level in Korean patients with congenital adrenal hyperplasia
              </a>
            </td>
          <td>
            S. Kim, J.-H. Yoon, D. Kim, S. Park, G.-H. Kim,, H.-W. Yoo, J.-H. Choi,, J. Kim, J. Kim
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8030ad41247de8846ce913550c0870c5e34232e" target='_blank'>
              Colon cancer cells evade drug action by enhancing drug metabolism.
              </a>
            </td>
          <td>
            Bojie Cong, Teena Thakur, Alejandro Huerta Uribe, Evangelia Stamou, Sindhura Gopinath, Owen Sansom, Oliver D. K. Maddocks, Ross L Cagan
          </td>
          <td>2025-07-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Fanconi anemia (FA) is a DNA repair deficiency disorder associated with genomic and chromosomal instability and a high cancer risk. In a small percentage of cases, FA is caused by biallelic pathogenic variants (PVs) in the BRCA2/FANCD1 gene, defining the FA-D1 subtype. Experimental and epidemiologic data indicate that the complete absence of BRCA2 is incompatible with viability. Therefore, cells from individuals affected with FA caused by biallelic BRCA2 PVs must have a residual BRCA2 function. This activity may be maintained through hypomorphic missense mutations, translation termination–reinitiation associated with a translational stop mutation, or other non-canonical or uncommon translation initiation and elongation events. In some cases, however, residual BRCA2 function is provided by alternatively or aberrantly spliced BRCA2 transcripts. Here, we review and debate aspects of the contribution of splicing in the 5′ segment to BRCA2 functions in the context of PVs affecting this largely intrinsically disordered protein region, with a focus on recent findings in individuals with FA-D1. In this Perspective, we also discuss some of the broader biological implications and open questions that arise from considering 5′-terminal BRCA2 splicing in light of old and new findings from FA-D1 patients and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea89691296a2efa5c1ed91fcd358d20e993d695b" target='_blank'>
              BRCA2 Pre-mRNA Differential 5′ Splicing: A Rescue of Functional Protein Properties from Pathogenic Gene Variants and a Lifeline for Fanconi Anemia D1 Patients
              </a>
            </td>
          <td>
            Roberto Paredes, Kiran Batta, Daniel H. Wiseman, Reham Gothbi, Vineet Dalal, Christine K. Schmidt, Reinhard Kalb, Stefan Meyer, Detlev Schindler
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, Natalia Y Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification is a pivotal mechanism in RNA epigenetics, with profound implications for lung cancer (LC) biology. This review synthesizes current knowledge on m6A’s multifaceted regulatory networks in non-small cell lung cancer (NSCLC), elucidating its roles in tumor proliferation, apoptosis, invasion, and metastasis. We further explore how m6A governs metabolic reprogramming—including glycolysis and ferroptosis—angiogenesis, and tumor microenvironment (TME) remodeling. Additionally, m6A-mediated modification of non-coding RNAs contributes to LC malignancy, underscoring its potential as a diagnostic and prognostic biomarker. These findings also offer novel strategies to overcome therapeutic resistance, a critical challenge in NSCLC treatment. Despite its promise, clinical translation of m6A-targeted interventions faces hurdles, such as the lack of standardized detection methods, the complexity of m6A-associated regulatory networks, and unresolved crosstalk with other RNA modifications. Future research should prioritize multi-omics approaches to resolve these challenges and advance m6A from mechanistic discovery toward clinical application. By addressing these gaps, m6A modulation may emerge as a transformative avenue in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd3ea37bc218eecdd95b3e5b51ea4cf4cc54e93" target='_blank'>
              M6A RNA modification: focusing on non-small cell lung cancer progression, therapeutic strategies and challenges
              </a>
            </td>
          <td>
            Yuyang Yan, Jiarui Yin, Quan Ding, Yan Lu, Shuhua Gou, Xi Xu, Yulin Li
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background Melanoma, the deadliest form of skin cancer, exhibits resistance to conventional therapies, particularly in advanced and metastatic stages. Mitochondrial pathways, including oxidative phosphorylation and mitochondrial translation, have emerged as critical drivers of melanoma progression and therapy resistance. This study investigates the mitochondrial proteome in melanoma to uncover novel therapeutic vulnerabilities. Methods Quantitative proteomics was performed on 151 melanoma‐related samples from a prospective cohort and postmortem tissues. Differential expression analysis identified mitochondrial proteins linked to disease aggression and treatment resistance. Functional enrichment analyses and in vitro validation using mitochondrial inhibitors were conducted to evaluate therapeutic potential. Results Mitochondrial translation and oxidative phosphorylation (OXPHOS) were significantly upregulated in aggressive melanomas, particularly in BRAF‐mutant and metastatic tumors. Inhibition of mitochondrial pathways using antibiotics (doxycycline, tigecycline, and azithromycin) and OXPHOS inhibitors (VLX600, IACS‐010759, and BAY 87‐2243) demonstrated dose‐dependent antiproliferative effects in melanoma cell lines, sparing noncancerous melanocytes. These treatments disrupted mitochondrial function, suppressed key metabolic pathways, and induced apoptosis, highlighting the clinical relevance of targeting these pathways. Conclusions This study reveals mitochondrial pathways as critical drivers of melanoma progression and resistance, providing a rationale for targeting mitochondrial translation and OXPHOS in advanced melanoma. Combining mitochondrial inhibitors with existing therapies could overcome treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361245499d6c69af98713d5bd55c9f9ceda286b1" target='_blank'>
              Mitochondrial proteome landscape unveils key insights into melanoma severity and treatment strategies
              </a>
            </td>
          <td>
            Yonghyo Kim, V. Doma, Uğur Çakır, M. Kuras, L. Betancourt, I. Pla, Aniel Sanchez, Y. Sugihara, R. Appelqvist, Henriett Oskolás, Boram Lee, Jéssica Guedes, G. Monnerat, G. Carneiro, Fábio Nogueira, G. Domont, Johan Malm, Bo Baldetorp, E. Wieslander, I. Németh, A. M. Szasz, R. Hong, Krzysztof Pawłowski, M. Rezeli, H. Kwon, József Timár, David Fenyö, S. Kárpáti, Gyorgy Marko-Varga, Jeovanis Gil
          </td>
          <td>2025-06-23</td>
          <td>Cancer</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: SARS-CoV-2, which drove the 2019 COVID-19 pandemic, continues to engender inquiries into the role of host genetic factors in disease susceptibility. Despite the identification of over 1,000 genes potentially associated with SARS-CoV-2 and COVID-19, the mechanisms connecting genetic variants to phenotype remain elusive. To shed light on these mechanisms, we undertook an integrated analysis, merging data from whole-genome association analyses of COVID-19 with methylome and transcriptomic. Methods: Study includes African American adults from the GENE-FORECAST study, encompassing 371 individuals with whole genome sequencing (WGS), 203 with DNA methylation, and 321 with RNA sequencing (RNA-Seq) of blood. 53.3% of participants reported COVID-19. Significant loci associated with COVID-19 were examined within the framework of methylation quantitative trait loci (mQTL) which located near the gene-of-original (cis-mQTL) and expression quantitative trait loci (eQTL) which located near the gene-of-origin (cis-eQTL), enabling analysis to assess mediators between genetic variants and COVID-19 status. Results: Our analysis identified four intronic variants and confirmed a missense variant, rs1052067, in PMF1 associated with COVID-19. Causal mediation analysis revealed that the combination of genetic variants within PMF1, epigenomics, and transcriptomics mapped four pathways influencing COVID-19 status. These pathways include: rs9659072 -> DNAm at chr1:156285845 (annotated to TMEM79) -> ENSG00000198715:13 (annotated to GLMP); rs12083543 -> DNAm at chr1:155951748 (ARHGEF2) -> ENSG00000198715:13 (GLMP); rs1052067 -> DNAm at chr1:155951748 (ARHGEF2)-> ENSG00000198715:13 (GLMP); rs1543294 -> ENSG00000198715:13 (GLMP) -> DNAm at chr1:156077518 (MEX3A). Conclusions: Through integrated multi-omics analyses, we identified genetic variants whose effects on COVID-19 susceptibility are mediated by changes in DNA methylation and mRNA expression. These findings offer insights into potential mechanistic pathways that merit further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00c7bc8e6839adbc276042c4324afad16196de83" target='_blank'>
              Multi-omics analyses of the complex interplay between genetic variants, DNA methylation, and gene expression in COVID-19.
              </a>
            </td>
          <td>
            Guanjie Chen, Lisa DeRoo, Gabriel Goodney, A. Doumatey, Jie Zhou, A. Adeyemo, C. Rotimi, Amadou Gaye
          </td>
          <td>2025-07-01</td>
          <td>American journal of physiology. Heart and circulatory physiology</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6786fefc42e4d77082eb7a0b7de5ec1cad7e96" target='_blank'>
              MITF maintains genome stability in non-melanocytic cell lineages and suppresses Hippo pathway signaling
              </a>
            </td>
          <td>
            Drifa H. Gudmundsdottir, Adrián López García de Lomana, Thejus B. Venkatesh, Kritika Kirty, S. Sigurdsson, Linda Vidarsdottir, R. Dilshat, Erla Sveinbjornsdottir, Snædís Ragnarsdóttir, Daniel H Magnusson, Maria R. Bustos, E. Steingrímsson, Thorkell Gudjónsson, Stefan Sigurdsson
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Existing approaches to identifying cancer genes rely overwhelmingly on DNA sequencing data. Here, we introduce RVdriver, a computational tool that leverages paired bulk genomic and transcriptomic data to classify RNA variant allele frequencies (VAFs) of non-synonymous mutations relative to a synonymous mutation background. We analyze 7882 paired exomes and transcriptomes from 31 cancer types and identify novel, as well as known, cancer genes, complementing other DNA-based approaches. Furthermore, RNA VAFs of individual mutations are able to distinguish “driver” from “passenger” mutations within established cancer genes. This approach highlights the value of multi-omic approaches for cancer gene discovery. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03557-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fff6bf27033e3de68705c3b36b17d1f26265844a" target='_blank'>
              Cancer gene identification from RNA variant allelic frequencies using RVdriver
              </a>
            </td>
          <td>
            James R. M. Black, Thomas P. Jones, Carlos Martínez-Ruiz, Maria Litovchenko, C. Puttick, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-06-13</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background The APOBEC (apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like) family plays a vital mutagenic role in diverse human malignancies. Nevertheless, the biological characteristics of APOBEC family members and their clinical significance in cancer have not been comprehensively explored. Our primary objective was to characterize the distribution and clinical relevance of APOBEC family members and their mutations across multiple cancer types, with a particular focus on thyroid carcinoma (THCA). Methods In this study, we employed an integrated, multi-faceted approach combining diverse statistical and bioinformatics methods. Data sources included whole-exome sequencing (WES), targeted next-generation sequencing (NGS), bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), Western blot assays, drug sensitivity analyses, and in vivo animal models. Results APOBEC mutations occupy a significant position in the mutation landscape of THCA. High APOBEC mutation enrichment scores were strongly associated with poor prognosis, immune evasion, and increased risk of malignant progression in THCA patients. These mutations contribute to tumor advancement and influence cellular differentiation within the tumor microenvironment. Additionally, we developed a prognostic APOBEC mutagenesis model using machine learning, which was validated across multiple THCA cohorts. In vitro and in vivo experiments demonstrated that APOBEC2 inhibition effectively suppressed tumor proliferation, metastasis, and glycolytic activity, while simultaneously enhancing immune activation and boosting the efficacy of immune checkpoint inhibitors. Conclusion Our study systematically characterizes APOBEC mutations across various cancer types and underscores their promise as biomarkers for aggressive tumor phenotypes, prognostic assessment, and immunotherapy response to THCA. These findings were accomplished through multi-omics sequencing and validated through comprehensive in vitro and in vivo experiments. Supplementary Information The online version contains supplementary material available at 10.1186/s12575-025-00288-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95761133ff7f07fa29822ab7f50151c4ca2027f9" target='_blank'>
              Comprehensive Insights into APOBEC Mutations in Thyroid Cancer: Prognostic and Therapeutic Discoveries
              </a>
            </td>
          <td>
            Wei Luo, Feng Liu, Mengyu Li, Jialong Yu, Ziyun Liu, Xuan Cheng, Yue Huang, Yu Liu, Mei Tao, Yuqi Wang, Yiping Zou, Xiaobin Shang, Chao Yang, X. Ruan, Yanchao Qin, X. Zheng
          </td>
          <td>2025-07-26</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Post-transcriptional modifications on mRNA are crucial for mRNA fate and function. The current lack of a comprehensive method for high-coverage and sensitive quantitative analysis of mRNA modifications significantly limits the discovery of new mRNA modifications and understanding mRNA modifications’ occurrence, dynamics and function. Here, a highly sensitive, high-throughput and robust LC-MS/MS-based technique, mRQuant, was developed to simultaneously detect and quantify 84 modified ribonucleosides in cellular mRNA. Using mRQuant, we quantified 32–34 modified ribonucleosides across several human cancer and non-cancer cell lines and uncovered cancer- and cancer type-specific signatures. Analyses of cisplatin- and paclitaxel-treated HeLa cells and drug-resistant variants revealed several drug resistance-associated modifications. Among them, m1A exhibited significant differences across multiple cell types and between cancerous and non-cancerous cells. Knocking down mRNA m1A writer or eraser protein resulted in altered cell viability, cell cycle and apoptosis in HeLa cells, suggesting a role of mRNA m1A in cancer. Transcriptomic and proteomic analyses further revealed the molecular mechanisms underlying the phenotypic variation. Supplementary information The online version contains supplementary material available at 10.1007/s00018-025-05805-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b614524ca44941ff0254c44b4d860cba08475a2" target='_blank'>
              Quantifying the mRNA epitranscriptome reveals epitranscriptome signatures and roles in cancer
              </a>
            </td>
          <td>
            Ying Feng, Xiaoli He, Mingxin Guo, Ying Tang, Guantong Qi, Qian Huang, Wenran Ma, Hong Chen, Yifan Qin, Ruiqi Li, Jin Wang, Yu Liu
          </td>
          <td>2025-07-28</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, D. Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract To date, most studies explored changes in 3D-genome organization between different tissues or during differentiation, which involve massive reprogramming of transcriptional programs. Much fewer studies examined alterations in genome organization in response to cellular stress, which involves less pervasive transcriptional modulation. Here, we examined associations between spatial chromatin organization and gene expression in two different biological contexts: transcriptional programs determining cell identity and transcriptional responses to stress, using p53 activation as a model. We selected 10 cell lines of diverse tissues, and in each performed micro-C, RNA-seq, and p53 ChIP-seq, before and after p53 induction. In the comparison between cell types, we delineated marked correlations between gene expression and spatial genome organization and identified hundreds of active enhancer–promoter loops associated with the expression of cell-type marker genes. In contrast, within each cell type, no such links were observed for expression changes induced by p53 activation, even for enhancers and promoters activated by p53 binding. Our analysis points to a fundamental difference between chromatin interactions that define cell identity and those that are established in response to cellular stress. Our results on p53-induced transcriptional responses support the recently proposed TF activity gradient model, which speculated a contact-independent mechanism for enhancer–promoter communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a93dd5bc7bcee25d57244f60e3d9ea722c2f779" target='_blank'>
              Transcriptional programs of cell identity and p53-induced stress responses are associated with distinctive features of spatial genome organization
              </a>
            </td>
          <td>
            Gony Shanel, Tsung-Han S Hsieh, Claudia Cattoglio, Hadar Amira Haham, Hsin-Jung Chou, Jack Z Li, R. Shamir, Xavier Darzacq, Ran Elkon
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc486e9a397957b4d882241501f31a8794a03818" target='_blank'>
              Precisely defining disease variant effects in CRISPR-edited single cells.
              </a>
            </td>
          <td>
            Y. Baglaenko, Zepeng Mu, Michelle Curtis, Hafsa M Mire, Vidyashree Jayanthi, Majd Al Suqri, Cassidy Liu, Ryan Agnew, A. Nathan, A. Mah-Som, David R. Liu, Gregory A. Newby, Soumya Raychaudhuri
          </td>
          <td>2025-07-23</td>
          <td>Nature</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancer is a growing concern around the world with more than 20 million new cases confirmed and counting. This is partly caused by the overuse of Chemotherapeutic drugs against Cancer, leading to its potency. Cancer starts with an accumulation of genetic mutations in the genome, and the ignorance to cell checks by CD8 and NK cells. The Cancer cell utilizes its hallmark capabilities and uses one of its cells to metastasizes in other areas of the body either through the perineural, circulatory, or lymphatic system. p53 is a tumor suppressor gene which repairs the mutations of the cell by conducting cell cycle arrest and inducing apoptosis. However, cancer cells can increase the expression of SB100 which can inhibit p53. One solution is to combine PRIMA-1 and CDDP which restores p53 and it’s wild type functions to induce apoptosis. This effect is increased with the use of adenoviral DN-Akt since it could increase p53 phosphorylation at Ser15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928e7ed5dc7a58f8574294e13f4f74e2d872a648" target='_blank'>
              Evolving Resistance: The Interplay of p53, Chemoresistance, and Metastasis in Cancer's Journey
              </a>
            </td>
          <td>
            Zenith Kwong
          </td>
          <td>2025-06-17</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d271ccc22e71dc47432dac9f4b41d18e3a1dd713" target='_blank'>
              Linking regulatory variants to target genes by integrating single-cell multiome methods and genomic distance.
              </a>
            </td>
          <td>
            E. Dorans, K. Jagadeesh, K. Dey, A. L. Price
          </td>
          <td>2025-06-12</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer remains a significant global health threat, characterized by high morbidity and mortality rates. Despite advancements, numerous challenges persist in cancer treatment, underscoring the urgent need for novel therapeutic strategies. As a burgeoning area of cancer research, increasing evidence highlights the critical role of 5-methylcytosine (m5C) methylation in tumorigenesis, progression, and clinical application. In this review, we systematically review the biological mechanism and clinical translational potential of m5C RNA methylation in cancer and analyze in detail the dynamic interactions between m5C regulatory proteins (writer proteins, eraser proteins, and reader proteins). We focus on the role of m5C regulatory proteins in regulating oncogenic signaling pathways, metabolic reprogramming, and immune microenvironment remodeling in different tumor types, while also exploring its clinical potential as a diagnostic marker, prognostic indicator, and target for therapy. This review not only offers a comprehensive analysis of the molecular mechanisms and tumor regulatory networks associated with m5C modification but also focuses on the clinical translational potential of m5C methylation in cancer, aiming to provide novel insights into precision cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07aa242baf6069f9d71bd233815a9263406948fe" target='_blank'>
              m5C RNA methylation in cancer: from biological mechanism to clinical perspectives
              </a>
            </td>
          <td>
            Fang Yuan, Yuxuan Xia, Zhen Huang, Huihui Gao, Yuqi Bian, Xiangqin Zhou
          </td>
          <td>2025-06-21</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background RNA sequencing (RNA-seq) is emerging as a valuable tool for identifying disease-causing RNA transcript aberrations that cannot be identified by DNA-based testing alone. Previous studies demonstrated some success in utilizing RNA-seq as a first-line test for rare inborn genetic conditions. However, DNA-based testing (increasingly, whole genome sequencing) remains the standard initial testing approach in clinical practice. The indications for RNA-seq after a patient has undergone DNA-based sequencing remain poorly defined, which hinders broad implementation and funding/reimbursement. Methods In this study, we identified four specific and familiar clinical scenarios, and investigated in each the diagnostic utility of RNA-seq on clinically accessible tissues: (i) clarifying the impact of putative intronic or exonic splice variants (outside of the canonical splice sites), (ii) evaluating canonical splice site variants in patients with atypical phenotypes, (iii) defining the impact of an intragenic copy number variation on gene expression, and (iv) assessing variants within regulatory elements and genic untranslated regions. Results These hypothesis-driven RNA-seq analyses confirmed a molecular diagnosis and pathomechanism for 45% of participants with a candidate variant, provided supportive evidence for a DNA finding for another 21%, and allowed us to exclude a candidate DNA variant for an additional 24%. We generated evidence that supports two novel Mendelian gene-disease associations (caused by variants in PPP1R2 and MED14) and several new disease mechanisms, including the following: (1) a splice isoform switch due to a non-coding variant in NFU1, (2) complete allele skew from a transcriptional start site variant in IDUA, and (3) evidence of a germline gene fusion of MAMLD1-BEND2. In contrast, RNA-seq in individuals with suspected rare inborn genetic conditions and negative whole genome sequencing yielded only a single new potential diagnostic finding. Conclusions In summary, RNA-seq had high diagnostic utility as an ancillary test across specific real-world clinical scenarios. The findings also underscore the ability of RNA-seq to reveal novel disease mechanisms relevant to diagnostics and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01494-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1a4792ea15366ba9539ebb29120d269348c3bb" target='_blank'>
              Clinical applications of and molecular insights from RNA sequencing in a rare disease cohort
              </a>
            </td>
          <td>
            Jamie C. Stark, Neta Pipko, Yijing Liang, Anna Szuto, Chung Ting Tsoi, Megan A. Dickson, Kyoko E. Yuki, Huayun Hou, Sydney Scholten, Kenzie Pulsifer, Meryl Acker, Meredith Laver, Harsha Murthy, Olivia M. Moran, Emily Bonnell, Nicole S. Y. Liang, Jashanpreet Sidhu, Lucie Dupuis, Mohammad M. Ghahramani Seno, M. Chard, R. Jobling, Jessie Cameron, Rose Chami, M. Inbar‐Feigenberg, Michael D. Wilson, David Chitayat, K. Boycott, L. Kyriakopoulou, Roberto Mendoza-Londono, Christian R. Marshall, J. J. Dowling, Gregory Costain, A. Deshwar
          </td>
          <td>2025-07-01</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The development of multiple myeloma is typically associated with various cytogenetic abnormalities; however, these genetic changes alone do not fully account for the observed heterogeneity in patient prognosis and treatment response. Recent studies leveraging next-generation sequencing and genomic approaches have shown that epigenetic alterations are crucial in myeloma development and therapeutic resistance. These changes contribute to high levels of transcriptomic instability and enables cellular adaptation to targeted therapies and immunotherapies through diverse evolutionary trajectories. In this regard, aberrations of histone modifications and chromatin remodeling affect various cellular processes such as DNA repair, DNA damage response, cellular survival, and apoptosis signaling, which provides a strong rationale for developing epigenetic-targeted therapies for myeloma treatment. In this review, we focus on recent advances and research gaps in understanding the deregulation of histone acetylation, a widespread and versatile process of histone modification occurring at lysine residues at the N-terminus of histone tails, and its intimate interplay with chromatin remodeling complexes in orchestrating dynamic chromatin functional states and transcriptional outputs. We also provide an updated review of epigenetic modulatory drugs targeting HDAC, CBP/p300, and bromodomain and extraterminal proteins, along with a discussion of their limitations and future perspectives in myeloma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d100926ef56ad097039cb56ac307d5417be61de8" target='_blank'>
              Deciphering the dynamics of histone acetylation and chromatin remodeling in multiple myeloma: a tale beyond the tails.
              </a>
            </td>
          <td>
            Sinan Xiong, Jianbiao Zhou, WeeJoo Chng
          </td>
          <td>2025-07-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we develop a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals from 10 localities in Europe and Africa. We identify 6957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter chromatin annotations, as expected. The expression of 58% of these regulatory elements is modulated by methylation, which is generally associated with decreased transcription. Within our set of regulatory elements, we use allele-specific expression analyses to identify 8020 sites with genetic effects on gene regulation; further, we find that 42.3% of these genetic effects vary in direction or magnitude between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects are enriched for GWAS and EWAS annotations, implicating them in human disease. Compared with data sets that assay DNA from a single European ancestry individual, our multiplexed assay is able to uncover more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse genomes in assays that aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8c0aba589430fda6268dbe62323f10c40bdeec" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2025-07-14</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="During carcinogenesis, cells must acquire a telomere maintenance mechanism in order to avoid telomere shortening-induced replicative senescence. While most tumors activate telomerase, a minority of them employ a recombinational mechanism called Alternative Lengthening of Telomeres (ALT). One of the most investigated features is the association between ALT and ATRX mutations, since this has been shown to be the gene with the highest rate of mutations among ALT tumors. However, most of these studies, and in particular, mechanistic studies in vitro, have been carried out on mesenchymal tumors (sarcomas). In the present study, using genomic and expression data from the DepMap portal, we identified several non-mesenchymal ALT cell lines, and we compared the incidence of ATRX and other gene mutations between ALT cell lines of different origins (mesenchymal, neural, epithelial, hematopoietic). We confirmed that ATRX is frequently mutated in mesenchymal and neural ALT cell lines but not in epithelial ones. Our results showed that mutations of ATRX or other proteins involved in the maintenance of telomere integrity are needed for ALT activation in all cell types, and ATRX is preferentially mutated in mesenchymal ALT cells. Besides a more precise interpretation of the role of ATRX loss in ALT establishment, we proposed a model in which mutation of this gene impairs differentiation in mesenchymal and neural cells (but not in epithelial ones). Therefore, we explained the high incidence of ATRX mutations in mesenchymal and neural tumors with the fact that they both trigger ALT and impair differentiation, thus promoting two steps at once in the process of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f454c9a5bcf29e99c79d4426a0a12fbbf069c0ed" target='_blank'>
              Alternative Lengthening of Telomeres: The Need for ATRX Mutations Is Lineage-Dependent
              </a>
            </td>
          <td>
            I. Udroiu, Antonella Sgura
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 International efforts focused on discovery of biological targets in inflammatory breast cancer (IBC) have suggested that aberrant inter-cellular relationship instead of genomic aberrations is the key driver of IBC. To identify such IBC-specific aberrations, we generated single nucleus chromatin accessibility and transcriptome atlas of IBCs (9,628 nuclei from three donors) and compared IBC atlas with a similar atlas of the healthy breast (81,735 nuclei from 92 donors). We recently reported single nucleus atlas of breast tissues of healthy women of diverse genetic ancestry (Nature Medicine in press). In that report, we described markers that identify three major epithelial cell subtypes [Basal-myoepithelial (BM), luminal adaptive secretory precursor (LASP), luminal hormone sensing (LHS)], two endothelial cell subtypes, two adipocyte subtypes, fibroblasts, macrophages, and T cells of the healthy breast. We also showed that LHS cells are the likely cells-of-origin of Luminal A, Luminal B and HER2+ breast cancers. Unlike these breast cancer subtypes, IBC does not appear to have a cell-of-origin, as epithelial cells in IBCs shared gene expression pattern across BM, LASP and LHS cells. Similar results were obtained when comparison was restricted to ancestry-specific healthy breast atlas. However, individual gene level expression differences affecting specific signaling pathways were observed in epithelial cells of IBCs. LASP, and LHS cells of IBCs overexpressed GPR137C, a positive regulator of mTORC1 signaling, and HS6ST3, an enzyme required for heparan sulfate synthesis. LASP and LHS cells of IBCs displayed activation of mitotic and matrix metalloproteinase signaling, respectively. Endothelial cells of IBCs, which showed significant gene expression differences compared to endothelial cells of the healthy breast, displayed enhanced integrin cell surface interaction but loss of cell junction organization. Most critically, BM, LASP, LHS, and endothelial cells of IBCs compared to their counterparts in the healthy breast showed downregulation of TEX14, an inter-cellular bridge forming factor in germ cells and a regulator of mitosis. Downregulation of TEX14 expression in these cells was accompanied with changes in the chromatin accessibility patterns of this gene. These results reinforce the notion that defective inter-cellular communication is a hallmark of IBC and TEX14 expression levels may serve as a biomarker of this defect. We suggest that greater attention to vascular biology and vascular-epithelial cell communication has to be given for better understanding of IBC biology and therapeutic targeting.
 Citation Format: Harikrishna Nakshatri, Poornima Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu. Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-05-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4562fd16c140b8e77bf1f62e51f30b753052c4ad" target='_blank'>
              Abstract P3-05-23: Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer
              </a>
            </td>
          <td>
            H. Nakshatri, P. Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 Background: Breast cancer is the most frequently diagnosed cancer and the leading cause of cancer-related death among women worldwide. Triple negative breast cancer (TNBC) represents 15-20% of all incident breast cancers and is a highly aggressive subtype. Neo(adjuvant) chemotherapy (NAC) is standard for early stage TNBC; however, for those with residual breast cancer following NAC, recurrence is common and associated with few effective therapeutic approaches. Thus, identification of novel resistance mechanisms and alternative treatment strategies remain an unmet clinical need. Circular RNAs (circRNAs) are a newly identified class of largely noncoding RNA molecules with covalently closed circular structures. Recent studies suggest that circRNAs play crucial roles in regulating tumor development, progression and chemoresistance. However, the role of circRNAs in mediating chemotherapy resistance remains unknown. Methods: As a first step towards identifying circRNAs that participate in the development of chemoresistance, and to determine if targeting such circRNAs may be an efficacious strategy, doxorubicin-resistant (Doxo-R), paclitaxel-resistant (PTX-R), and double-resistant (DP-R) cell lines were generated from MDA-MB-231 cell lines. Human circRNA microarrays were utilized to profile the expression of approximately 14,000 known circRNAs in normal breast tissue, matched TNBC patient-derived xenografts (PDX) generated prior to and following NAC, and TNBC chemosensitive and chemoresistant cell lines. Top hits were validated using RT-PCR. Bioinformatic approaches were employed to identify circRNA-miRNA-mRNA networks regulated by differentially expressed circRNA transcripts. A reverse genetics approach was also employed using CRISPR to interrogate the contributions of over 4000 specific circRNA in driving the development of resistance to doxorubicin and paclitaxel. Results: circRNA microarray profiling identified 429 and 310 transcripts differentially expressed in doxorubicin and paclitaxel resistant cells, respectively, compared to parental chemosensitive cell lines (|FC| ≥ 1.5; p value < 0.05). Given that circRNAs can act as microRNA (miRNA) sponges, thereby inhibiting miRNA activity, we computationally identified circRNA-miRNA pairs, and assessed the abundance of mRNA transcripts predicted to be targets of paired miRNAs via RNA-seq. A differentially expressed circRNA-miRNA-mRNA network was further constructed. Dysregulated canonical pathways and gene ontology (GO) enrichment analysis were determined from these networks and revealed multiple GO term included negative regulation of miRNA transcription, miRNA and RNA metabolic process, and endoplasmic reticulum-associated degradation (ERAD) pathway. The CRISPR-Cas13 screen is ongoing at this time but results will be available prior to the meeting. Conclusions: We have identified cancer specific circRNAs in TNBC cell lines, PDX models and patient tumors in the context of chemotherapy sensitive and chemotherapy resistant disease. Specific biological processes, pathways and signaling cascades have been implicated in the down-stream consequences of dysregulated circRNAs and in the potential development of resistance. Our ongoing studies seek to identify specific circRNAs sufficient to drive resistance to chemotherapy. Outcomes of these studies highlight the potential of specific circRNAs to serve as predictive/prognostic biomarkers and novel therapeutic targets for advanced forms of TNBC.
 Citation Format: Xiyin Wang, Michael J. Emch, Xiaojia Tang, Esther P.B. Rodman, Liewei Wang, Judy C. Boughey, Matthew P. Goetz, Krishna R. Kalari, John R. Hawse. The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb284df6818a5ca75827ea02126a84e48b2886e9" target='_blank'>
              Abstract P3-02-22: The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance
              </a>
            </td>
          <td>
            Xiyin Wang, M. Emch, Xiaojia Tang, E. Rodman, Liewei Wang, J. Boughey, Matthew P. Goetz, K. Kalari, John R. Hawse
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Lesch–Nyhan disease (LND) is associated with a complete deficiency of hypoxanthine-guanine phosphoribosyltransferase (HPRT) activity due to mutations in the HPRT1 gene. Although the physiopathology of LND-related neurological manifestations remains unknown, a defective neuronal developmental process is the most widely accepted hypothesis. We generated an HPRT-deficient line from the pluripotent human embryonic cell line NT2/D1 by CRISPR-Cas9 and induced its differentiation along neuroectodermal lineages by retinoic acid treatment. As levels of folic acid in the culture media may affect results in LND models, we employed physiological levels of folate. The effect of HPRT deficiency on neural development-related gene expression was evaluated using two methodological approaches: a directed qPCR array of genes related to neuronal differentiation, and global gene expression by RNAseq. HPRT-deficient pluripotent cells presented altered expression of genes related to pluripotency in human embryonic stem cells, such as DPPA3 and CFAP95, along with genes of the homeobox gene family. HPRT-deficient pluripotent cells were able to differentiate along neuro-ectodermal lineages but presented consistent dysregulation of several genes from the homeobox gene family, including EN1 and LMX1A. GO enrichment analysis of up- and downregulated genes in HPRT-deficient cells showed that the most significant biological processes affected are related to development and nervous system development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9428cd1b573671431fe9d600913d73a1c68291" target='_blank'>
              Gene Expression Analysis of HPRT-Deficient Cells Maintained with Physiological Levels of Folic Acid
              </a>
            </td>
          <td>
            Rosa J Torres, Gerard Valentines-Casas, Claudia Cano-Estrada, Neus Ontiveros, José M López
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The human genome contains sequences of DNA enriched in cytosine-guanine dinucleotides known as CpG islands (CGIs). CGIs play a crucial role in gene regulation and expression, making them an important target for genetic therapies. In this article, hidden Markov models (HMMs) and adaptive window techniques (AWTs) were used to identify CGIs in MUC5B and DSP genes. Both genes are associated with idiopathic pulmonary fibrosis, a progressive pulmonary disease that leads to a lung transplant. The University of California, Santa Cruz Genome Browser was used to obtain the MUC5B and DSP gene sequences and predefined CGI locations. The HMM and AWT algorithms were developed using Python version 3.11.5, and the outcomes analyzed were sensitivity, specificity, computational memory, and runtime. Both HMM and AWT exhibited high specificity; however, HMM was more accurate than AWT for both genes, 99% versus 96%, respectively. The HMM sensitivity was higher for both MUC5B and DSP genes (87% and 88%) compared with only 58% for MUC5B and 57% for DSP with AWT. Regarding computational efficiency, AWT was faster and required less memory than HMM for both genes. By accurately detecting CpG-rich regions, HMM offers a powerful approach to understanding gene regulation mechanisms. This could pave the way for more precise therapeutic interventions, enabling targeted treatment strategies for a range of genetic disorders, including idiopathic pulmonary fibrosis, improving patient outcomes, and advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0af497de5236a5da0b2fa9b64a0e620669cb88" target='_blank'>
              Hidden Markov Models Offer a Powerful Approach for Understanding Gene Regulation Mechanisms Relevant for Organ Transplantation.
              </a>
            </td>
          <td>
            Carlos Goncalves, Marissa Di Napoli, David Schwartz, Bruce Kaplan
          </td>
          <td>2025-06-24</td>
          <td>Transplantation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated member of the RAS family of small GTPases (RAS). It affects about one-fifth of cancer cases. The tumor microenvironment (TME) is a multifaceted network of immune cells, metabolites, microbiota, stromal components, and extracellular matrix. It creates a dynamic ecosystem that supports malignant initiation, progression, and therapy resistance through bidirectional crosstalk with tumor cells. Emerging evidence reveals distinct TME landscapes shaped by wild-type versus oncogenic KRAS variants. Additionally, TME rewiring occurs during KRAS-targeted therapies. Deciphering these KRAS-dependent TME architectures and their therapeutic vulnerabilities represents a critical frontier for precision oncology. This review synthesizes key milestones and persistent challenges in KRAS inhibitor development. And it systematically evaluates how KRAS mutations orchestrated immunosuppressive niches, metabolic symbiosis, stromal remodeling, and microbiome dysbiosis, supported by mechanistic insights from preclinical and clinical studies. It further explores therapeutic opportunities arising from targeting TME interactions, including rational combinations of KRAS inhibitors with immune checkpoint blockade, metabolic agents, or microbiota-modulating strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2718254ccab31c0317c655b38e3998d0c1a62d" target='_blank'>
              Mechanistic Foundations of KRAS-Driven Tumor Ecosystems: Integrating Crosstalk among Immune, Metabolic, Microbial, and Stromal Microenvironment.
              </a>
            </td>
          <td>
            Jiayao Ma, Shenao Fu, Jun Tan, Ying Han, Yihong Chen, Xiangying Deng, Hong Shen, Shan Zeng, Yinghui Peng, Changjing Cai
          </td>
          <td>2025-06-09</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5870c7c37807f788d6b6da82817760500b862c7f" target='_blank'>
              Genetic modifiers of somatic expansion and clinical phenotypes in Huntington's disease highlight shared and tissue-specific effects.
              </a>
            </td>
          <td>
            Jong-Min Lee, Zachariah L. McLean, Kevin Correia, Jun Wan Shin, Sujin Lee, Jae-Hyun Jang, Yukyeong Lee, Kyung-Hee Kim, Doo Eun Choi, Jeffrey D. Long, D. Lucente, Ihn Sik Seong, Ricardo Mouro Pinto, James V. Giordano, J. Mysore, Jacqueline Siciliano, Emanuela Elezi, Jayla Ruliera, T. Gillis, V. Wheeler, M.E Macdonald, J. Gusella, Anna Gatseva, Marc Ciosi, Vilija Lomeikaite, Hossameldin Loay, Darren G. Monckton, Christopher Wills, Tom Massey, Lesley Jones, P. Holmans, Seung Kwak, Cristina Sampaio, Michael Orth, G. B. Landwehrmeyer, J. Paulsen, E. R. Dorsey, Richard H. Myers
          </td>
          <td>2025-06-09</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>127</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Endocrine resistance is frequently encountered in the clinic through a variety of mechanisms such as NF1 loss that induce alternative survival pathways and suppress estrogen responsiveness. While targeting the induced pathways such as the RAS-MAPK pathway can have antitumor effects, it can also incur toxicities. To identify novel and potentially more specific therapeutic vulnerabilities in this context, we performed CRISPR/Cas9 screens in wildtype and NF1 knockout isogenic ER+ models and identified NR2F2, an orphan nuclear receptor, to be essential specifically in NF1 loss cells. Our in vitro and in vivo results revealed that NR2F2 is a critical regulator of the estrogen signaling pathway, largely playing a repressive role. We found that NF1 loss could upregulate NR2F2 expression level in ER+ breast cancer cells through the activation of the MAPK pathway. NR2F2 overexpression by itself conferred endocrine resistance, while genetic knockout and pharmacologic inhibition of NR2F2 sensitized ER+ breast cancer cells to endocrine therapies and attenuated resistance across multiple endocrine refractory tumor models including those driven by NF1 loss, ARID1A loss, and PTEN loss. These results reveal upregulation of NR2F2 to be a recurrent mechanism whereby hormone responsiveness is suppressed. At a mechanistic level, investigation into chromatin accessibility and transcription revealed that altered NR2F2 expression modulates genome-wide chromatin accessibility including ER-regulated loci and modulates ER transcriptional activity through its direct interactions with ER and its transcriptional coregulators. Collectively, our data unveil a critical mechanism whereby endocrine resistance occurs through the repression of estrogen responsiveness and highlight a new therapeutic approach to restore endocrine response through pharmacologic inhibition of NR2F2.
 Citation Format: Yanyan Cai, Peihua Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O’Connell, Emma Fink, Matthew M Miele, Zhuoning Li, Elisa De Stanchina, Emiliano Cocco, Pedram Razavi, Eneda Toska, Sean W Fanning, Guotai Xu, Anna A Sablina, Sarat Chandarlapaty. Targeting NR2F2 overcomes multiple forms of endocrine resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-03-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97507505295680ec94e45659684f5586171f75f5" target='_blank'>
              Abstract P2-03-17: Targeting NR2F2 overcomes multiple forms of endocrine resistance
              </a>
            </td>
          <td>
            Yanyan Cai, P. Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O'Connell, Emma C. Fink, Matthew M. Miele, Zhuoning Li, Elisa de Stanchina, Emiliano Cocco, P. Razavi, Eneda Toska, S. Fanning, Guotai Xu, A. Sablina, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), ranks among the most lethal malignancies, with a 5-year survival rate of under 10%. The most prevalent KRAS mutations occur in three hotspot residues: glycine-12 (G12), glycine-13 (G13), and glutamine-61 (Q61), leading to the constant activation of the Ras pathway, making them the primary focus in oncologic drug development. Selective KRAS G12C inhibitors (e.g., sotorasib, adagrasib) have demonstrated moderate efficacy in clinical trials; however, this mutation is infrequent in PDAC. Emerging therapies targeting KRAS G12D and G12V mutations, such as MRTX1133, PROTACs, and active-state inhibitors, show promise in preclinical studies. Pan-RAS inhibitors like ADT-007, RMC-9805, and RMC-6236 compounds provide broader coverage of mutations. Their efficacy and safety are currently being investigated in several clinical trials. A major challenge is the development of resistance mechanisms, including secondary mutations and pathway reactivation. Combination therapies targeting the RAS/MAPK axis, SHP2, mTOR, or SOS1 are under clinical investigation. Immunotherapy alone has demonstrated limited effectiveness, attributed to an immunosuppressive tumor microenvironment, although synergistic effects are noted when paired with KRAS-targeted agents. Furthermore, KRAS mutations reprogram cancer metabolism, enhancing glycolysis, macropinocytosis, and autophagy, which are being explored therapeutically. RNA interference technologies have also shown potential in silencing mutant KRAS and reducing tumorigenicity. Future strategies should emphasize the combination of targeted therapies with metabolic or immunomodulatory agents to overcome resistance and enhance survival in KRAS-mutated PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43732db4b90e179a61acd390753f5b4ab59d2952" target='_blank'>
              Treatment of KRAS-Mutated Pancreatic Cancer: New Hope for the Patients?
              </a>
            </td>
          <td>
            Kamila Krupa, Marta Fudalej, Emilia Włoszek, Hanna Miski, A. Badowska-Kozakiewicz, D. Mękal, M. Budzik, A. Czerw, Andrzej Deptała
          </td>
          <td>2025-07-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Background: Approximately 10% of breast cancer patients are metastatic at the time of diagnosis with a 27% survival rate. Currently, endocrine therapy, including tamoxifen, aromatase inhibitors (letrozole, anastrozole, and exemestane), Selective Estrogen Receptor Degraders (fulvestrant and elacestrant), is the standard of care for patients with estrogen receptor-positive (ER+) breast cancer. The major treatment challenge is endocrine therapy resistance and metastasis. Epithelial Mesenchymal Transition (EMT) is the process in which cells lose epithelial properties and acquire motility and mesenchymal phenotype, which leads to migration and invasion. EMT is characterized by the destabilization of adherens junctions by loss of E-cadherin at the plasma membrane. We previously reported a novel signaling pathway that is present in tamoxifen-resistant (TR) breast cancer cells. This pathway involves transcriptional repression of p120catenin by upstream regulators PKCa and FOXC2, resulting in the loss of E-cadherin at the plasma membrane (Pham, 2017). EMT features were characterized by decreased expression of epithelial markers and increased migration and invasion. Initially, estrogen stimulates breast cancer growth but becomes inhibitory after long-term estrogen deprivation. A novel Selective Human Estrogen Receptor Partial Agonist (ShERPA) TTC-352 is an estrogen mimic that induces unfolded protein response and shrinks the TR tumors (Molloy, 2014; Abderrahman, 2021). In this study, we investigated the impact of various ER-targeting drugs including 17b-estradiol (E2), tamoxifen, fulvestrant, and TTC-352 on the novel PKCa-FOXC2-p120catenin signaling axis. We hypothesize that ER-targeting drugs modulate the novel EMT pathway in TR breast cancer cells. Materials and Methods: TR cell lines used in this study include PKCa overexpressing cells MCF-7/PKCa, and MCF-7:5C and represent migratory ER+ breast cancer. Migration assays were performed in response to ER ligands using Falcon Permeable inserts. Migrated cells in response to chemoattractant were stained and counted after 24 hours. Luciferase reporter assays were performed to evaluate the effect of ER ligands on p120catenin transcription. Cells were transiently co-transfected with a p120catenin promoter-reporter plasmid using renilla vector for normalization. The reporter system readout is based on FOXC2 binding to the p120catenin promoter resulting in repression of luciferase transcription. Luciferase luminescence was measured on a plate reader corresponding to p120catenin transcription. Chromatin immunoprecipitation was performed using a FOXC2 antibody (Abcam-308055) to assess FOXC2 binding to the p120catenin promoter after ER ligand treatment. Results: E2 and TTC-352 treatment lowered FOXC2 binding on the p120catenin promoter, increased its transcription, and decreased the migration of MCF-7/PKCa and MCF-7:5C breast cancer cells compared to vehicle. E2 and TTC-352 treatment does not activate the PKCa-FOXC2-p120catenin EMT signaling axis and reduces the migratory potential of TR cells. Conclusion and Discussion: Our findings suggest that patients with endocrine-resistant ER+ breast cancer may benefit from E2 and TTC-352 therapy by inhibiting the activation of the PKCa-FOXC2-p120catenin signaling and cell migration identified in TR cell lines. Results of the Phase I clinical trial of TTC-352 reveal a reduced side effects profile and improved progression-free survival (Dudek, 2020). Strategizing less toxic treatment options may potentially reduce metastasis and prolong patient survival with a better quality of life.
 Citation Format: Nivida Shete, Debra A. Tonetti. Impact of Estrogen Receptor Targeting Drugs on a Novel Epithelial-Mesenchymal Transition Pathway in Endocrine Resistant Breast Cancer: Implications for Metastasis [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-05-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6f868b8504cb501f8c0ff82c7811f21d550430" target='_blank'>
              Abstract P4-05-25: Impact of Estrogen Receptor Targeting Drugs on a Novel Epithelial-Mesenchymal Transition Pathway in Endocrine Resistant Breast Cancer: Implications for Metastasis
              </a>
            </td>
          <td>
            Nivida Shete, D. Tonetti
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic modifications are heritable, reversible alterations that causally reshape chromatin architecture and thereby influence DNA repair without changing nucleotide sequence. DNA methylation, histone modifications and non-coding RNAs profoundly influence DNA repair mechanisms and genomic stability. Aberrant epigenetic patterns in cancer compromise DNA damage recognition and repair, therefore impairing homologous recombination (HR), non-homologous end joining (NHEJ), and base excision repair (BER) by suppressing key repair genes and lowering access to repair sites. Then it is dissected how loss-of-function mutations in Switch/Sucrose non-fermentable, imitation switch and CHD (Chromodomain helicase DNA-binding) chromatin-remodeling complexes impair nucleosome repositioning, preventing effective damage sensing and assembly of repair machinery. Non-coding RNAs contribute to epigenetic silencing at DNA break sites, exacerbating repair deficiencies. This review evaluates recent advances concerning epigenetic dysfunction and DNA repair impairment. It is also highlighted that nanoparticle-mediated delivery strategies are designed to overcome pharmacologic resistance. It is presented how epigenetic dysregulation of DNA repair can guide more effective and drug-resistant cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d922eca60c4fcba055a3683ed299a04149ee715f" target='_blank'>
              Epigenetic Dysregulation in Cancer: Implications for Gene Expression and DNA Repair-Associated Pathways
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Katarzyna Rekiel, Piotr Urbanowicz, Mateusz Mamala, Karolina Marczuk, Maria Wojtaszek, Marta Żywica, Eivina Radzevičiūtė-Valčiukė, V. Novickij, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Aging is a multifaceted process influenced by many elements. During cell division, the repetitive DNA sequences at the ends of chromosomes called telomeres protect them from degradation. Telomeres shorten alongside each cell division, eventually contributing to cellular senescence and aging. Telomerase as an enzyme has a role in the maintenance of telomere length. Reduced function of telomerase is linked to acceleration of aging and age-related diseases. By affecting cellular function, mutations in particular genes can cause aging. Genes involved in DNA repair, cellular metabolism, and inflammation play the key roles in this process. Accumulated mutations result in cellular dysfunction and age-related diseases over time. Epigenetic changes are the modifications that impact gene expression without altering the DNA sequence. Lifestyle factors (diet, exercise, stress) and environmental influences (toxins, trauma) can cause epigenetic alterations. DNA methylation as well as histone modifications are examples of epigenetic alterations. They influence how cells work and are essential to the aging process. Understanding these molecular mechanisms is essential for developing interventions to promote healthy aging and prevent age-related diseases. This paper explores the potential of CRISPR/Cas9 as a gene-editing tool to target these mechanisms and mitigate age-related conditions, ultimately enhancing longevity and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f9eeb8cd6bb47cacfd1747bd987a2823682b4c" target='_blank'>
              Applications of CRISPR-Cas9 in mitigating cellular senescence and age-related disease progression
              </a>
            </td>
          <td>
            Alireza Azani, Malihe Sharafi, Reyhaneh Doachi, Sama Akbarzadeh, Parsa Lorestani, Arash Haji Kamanaj Olia, Zahra Zahed, Hossein Gharedaghi, Haniyeh Ghasrsaz, Hassan Foroozand, Hossein Rahimi, Safa Tahmasebi, Qumars Behfar
          </td>
          <td>2025-07-08</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Targeted sequencing is critical in cancer diagnosis, treatment selection, and monitoring. However, the effectiveness of these methods for reflecting whole-exome sequencing (WES)-level mutational signatures remains unclear. Therefore, we addressed this issue through simulation-based analysis to clarify how well targeted sequencing can reproduce WES-level mutational signatures. Methods We compared the correlation and similarity of mutational signatures between whole-exome sequencing-level mutation data and downsampled data for gene sets targeted by each sequencing method in 13 cancer types. Additionally, a similarity analysis of the mutational signatures was conducted using randomly downsampled data. Results The comparison between whole-exome sequencing and targeted sequencing showed a low correlation based on Pearson’s correlation coefficient but a high similarity based on the Dice similarity index. As a result of the downsampling of data with cancer-related genes and whole genes to evaluate similarity, the cancer-related gene random set showed high similarity when 200–400 genes were selected. However, the whole-genome random set required 2–3 times as many genes as the cancer-related gene random set to show high similarity. Among cancer types, colorectal and lung cancers demonstrated high similarity with fewer downsampled genes, whereas breast and prostate cancers required more downsampled genes to achieve high similarity. Conclusion This study demonstrated that current clinically used targeted sequencing methods can reflect whole-exome sequencing-level mutational signatures. This suggests that considering the cancer type and average number of gene mutations in each patient when selecting targeted sequencing methods can lead to more effective treatment choices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89cf22b23dffb414f1a7e7cc4a7de2334cfe0a1e" target='_blank'>
              Evaluation of potential of targeted sequencing through mutational signature simulation
              </a>
            </td>
          <td>
            Keisuke Kodama, Yiwei Ling, Hiroshi Ichikawa, Toshifumi Wakai, Shujiro Okuda
          </td>
          <td>2025-06-25</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Multiple myeloma is a hematologic malignancy characterized by complex interactions within the tumor microenvironment, where mesenchymal stem cells (MSCs) contribute significantly to disease progression, immune suppression, and drug resistance. Methods This study investigated the heterogeneity of MSCs in multiple myeloma using single-cell RNA sequencing (10X) and bulk transcriptomic data. Further analysis was performed by Seurat, SCENIC, CellChat. GSE4581 and GSE136337 were used as training set and validation set to construct a newly described prognostic model through COX and LASSO. Results By analyzing bone marrow samples from healthy donors and multiple myeloma patients at different Revised International Staging System (R-ISS) stages, this study identified distinct MSC subpopulations, including osteogenic, angiogenic, immune regulatory, and multipotent clusters, each of which plays unique roles in the tumor microenvironment. Interestingly, we found a unique subclone with upregulated expression of high mobility group proteins, these MSC exert a strong regulatory effect, which was defined as “HMGhMSC”. Conclusions Our findings reveal that the proportion of osteogenic MSCs, which are crucial for bone health, decreases as the disease progresses, which is correlated with the bone lysis commonly observed in advanced multiple myeloma. Additionally, immune regulatory MSCs contribute to the formation of an immunosuppressive microenvironment, promoting tumor immune evasion. A prognostic model based on HMGhMSC subpopulations was developed, which demonstrated that these cells have significant potential as therapeutic targets for improving the prognosis and developing treatments for bone disease in multiple myeloma patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06637-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d201f41537186cac07b0c328ff5a2a24776617ee" target='_blank'>
              Integrative analysis of bulk and single-cell gene expression profiles to identify bone marrow mesenchymal cell heterogeneity and prognostic significance in multiple myeloma
              </a>
            </td>
          <td>
            Fei-Er Ju, Bei-Hui Huang, Hao Wu, Bo Zou, Shu-Na Chen, Xiao-Yan Sang, Wei-Yao Liang, Zixuan Liu, Zixuan Zhang, Ziyi Yang, Yan-Ting Liang, Yuelan Liang, Huan Liu, Zhao-Xia Dong, Xueqi Liu, Li-Yuan Zheng, Jin-Cheng Zeng, Jin-Heng Wang, Lin Qi, Xing-Ding Zhang, Yongjiang Zheng, Juan Li
          </td>
          <td>2025-06-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality globally, posing significant treatment challenges, particularly in its metastatic form (mCRC). This review comprehensively examines the pivotal role of RAS mutations, specifically KRAS and NRAS, which are detected in approximately 40–45% of mCRC cases, and their impact on treatment decisions and patient outcomes. We assess the effectiveness of standard treatments within the RAS mutant population, highlighting the challenges and limitations these therapies face. Recent advancements in targeted therapies, particularly the focus on novel agents such as KRAS G12C inhibitors, including sotorasib and adagrasib, have shown promising efficacy in overcoming resistance to conventional treatments. Furthermore, this review discusses future directions, emphasizing the need for research into non-RAS targets to address the complexities of resistance mechanisms and improve therapeutic outcomes. This review aims to provide a detailed overview of the current treatments and innovative approaches, supporting the development of personalized management strategies for patients with mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e15cd6a754c3946344410b05543b29153cfa7" target='_blank'>
              RAS Mutations in Advanced Colorectal Cancer: Mechanisms, Clinical Implications, and Novel Therapeutic Approaches
              </a>
            </td>
          <td>
            O. Sütçüoğlu, H. Yıldırım, Elvina Almuradova, Damla Günenç, Suayib Yalcin
          </td>
          <td>2025-06-30</td>
          <td>Medicina</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cisplatin is a widely used chemotherapy agent in the treatment of non-small cell lung cancer (NSCLC). However, its clinical efficacy is limited by the development of drug resistance in patients with NSCLC. Recently, we demonstrated that HMGB1-BoxA gene therapy (BoxA) is an ideal cancer treatment that revitalizes normal cells while promoting cancer cells’ DNA break cascade. Increasing cytoplasmic HMGB1 is a reason for cisplatin resistance (Cis-R); in this study, we investigated the potential of BoxA to reverse acquired cisplatin resistance in two NSCLC cell lines, A549/Cis-R, and H460/Cis-R. The growth capacity of these cancer cells was significantly impaired upon BoxA treatment, resulting in a notable reduction in cell viability, colony formation, cancer stemness, and self-renewal capacity. Significantly, BoxA enhanced cisplatin sensitivity and promoted apoptosis in cisplatin-treated Cis-R cells. Furthermore, BoxA altered the subcellular localization of the HMGB1 protein, decreasing its cytoplasmic localization. BoxA can potentially reverse cisplatin resistance by altering the translocation of HMGB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d2a6c39bb4e559378d713a6bcbc5484070b471" target='_blank'>
              HMGB1-BoxA gene therapy in reversing cisplatin resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            Nattapong Puthdee, Van-Hieu Mai, Sirapat Settayanon, P. Chanvorachote, C. Aporntewan, Apiwat Mutirangura, C. Vinayanuwattikun
          </td>
          <td>2025-06-25</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Although long interspersed element-1 (LINE-1) retrotransposons are poor prognostic indicators of non-small cell lung cancer, the genetic consequences of aberrant LINE-1 expression remain poorly understood. In this study, human bronchial epithelial cells were exposed chronically to 25 µM NiCl₂ to induce malignant transformation and compared to PBS-treated control cells or lung cancer cell lines. A 4 kb intronic LINE-1 insertion into the NACC2 (nucleus accumbens-associated protein 2) locus in nickel-transformed cells was associated with significant reductions in NACC2 mRNA and protein, along with constitutive increases in HDM2 and TP53α mRNAs, and aberrant expression of TP53β and TP53γ mRNAs. Steady-state levels of p53 and RB decreased, while EGFR and LINE-1 ORF1p increased. Silencing of NACC2 in TP53 wildtype NCI-H460 or TP53-null NCI-1299 cell lines replicated many of these changes, with profiles varying as a function of p53 status. Stable overexpression of HDM2 increased LINE-1 ORF1p in cancer cell lines. Human lung adenocarcinomas with wild-type TP53 showed sexually dimorphic profiles, with higher HDM2 levels in females than males, and shifts in LINE-1 ORF1p, p53, and RB that mirrored cancer cell lines. This study identifies the NACC2 locus as a target of LINE-1 retrotransposition and highlights critical interactions with the HDM2/TP53/RB regulatory axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adfffe6caad02c3b19f108090ea32e2ac145796" target='_blank'>
              LINE1 insertion into the NACC2 locus disrupts the HDM2 axis and activates lung oncogenic signaling
              </a>
            </td>
          <td>
            P. Bojang, Yingshan Wang, Kenneth S Ramos
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC), an aggressive and highly malignant subtype of castration-resistant prostate cancer (CRPC), arises through drug resistance mechanisms involving genomic alterations, epigenetic remodeling, tumor microenvironment (TME) reprogramming, and lineage plasticity. A hallmark of NEPC is its independence from the androgen receptor (AR) pathway, evidenced by diminished or absent AR expression—a key barrier to effective clinical management. Therefore, a comprehensive understanding of NEPC pathogenesis and progression is essential, as it facilitates the identification of potential therapeutic targets and the development of more effective therapies. In this review, we summarize the typical characteristics of NEPC and describe its clinical diagnosis, relevant imaging modalities, current treatment strategies, and associated therapeutic difficulties. We also highlight the core events driving NEPC formation. Furthermore, we discuss potential therapeutic targets for the disease and review pharmacological agents that have demonstrated efficacy against NEPC, aiming to offer innovative perspectives and potential research directions for future NEPC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58ac5d561d4b3dff4622ddbdfd3ef0e9f2868489" target='_blank'>
              New Insights into Potential Therapeutic Targets for Neuroendocrine Prostate Cancer: From Bench to Clinic
              </a>
            </td>
          <td>
            Maoping Cai, Fengtao Zheng, Yang-Zi Ren, Chuqian Zhen, Dujiang Fu, Xian-lu Song, Qing Li, Yuanyuan Qu, Zhe-Sheng Chen, Shan-Chao Zhao
          </td>
          <td>2025-06-29</td>
          <td>Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background The three-prime repair exonuclease 1, TREX1, degrades cytosolic DNA to prevent aberrant immune activation. Its inactivation results in DNA accumulation in the cytosol and induction of the cGAS-STING DNA sensing pathway, interferon signaling, and inflammation. Germline pathogenic TREX1 mutations are known to lead to hereditary autoimmune and autoinflammatory disorders, whereas the consequences of TREX1 mutations in cancer remain poorly understood. Results To assess the importance of human TREX1 amino acid variants, we analyzed protein sequences of the functional TREX1b isoform from 168 mammalian species and integrated available data on TREX1 sequence and copy number alterations in hereditary autoimmune and autoinflammatory disorders, cancer, and in human populations. While the entire TREX1b protein was conserved in placental mammals, egg-laying mammals and marsupials had their own unique C-terminal regions, with each predicted to contain a transmembrane domain. We modeled human TREX1 variants occurring in autoimmune disease and cancer samples at 12 protein positions to evaluate their predicted impact on protein stability and function. Conclusions Our findings provide novel insight into the role of TREX1 molecular variation in cancer, where genetic or epigenetic loss of TREX1 activity may improve susceptibility to treatment. However, TREX1 gene deletion in tumors was associated with unfavorable patient outcomes, most likely due its frequent co-occurrence with the loss of the entire 3p chromosomal arm, which contains known cancer-related genes. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00785-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53a6c0f4b15cdd23d8488729c5a843f6b9a322d6" target='_blank'>
              Prevalence and impact of molecular variation in the three-prime repair exonuclease 1 TREX1 and its implications for oncology
              </a>
            </td>
          <td>
            Marwa Shekfeh, Mariam M. Konaté, Julia Krushkal
          </td>
          <td>2025-06-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The Mismatch Repair (MMR) pathway, a crucial tumor suppressor mechanism, is dysregulated across cancer types. We previously identified the loss of the MutL complex of the MMR pathway, as a key factor inducing resistance to endocrine therapy in Estrogen Receptor positive (ER+) breast cancer patients. Preliminary data previously generated in the lab supports a role for MutL loss in promoting metastasis of ER+ breast cancer. A mammary-specific knockout mouse model for MLH1, a principal component of MutL, created in the Haricharan lab develops aggressive mammary tumors that metastasize to multiple organs. To unveil the mechanism by which MLH1 loss triggers metastasis, an unbiased RPPA and RNA-Seq was conducted which revealed cytokine secretion in the immune system as a distinctive signature of MLH1- ER+ breast cancer cells. The tumor secretome, particularly cytokines, can manipulate macrophage plasticity to skew polarization states towards either an anti- or pro- metastatic phenotype. Secreted cytokine levels altered by MLH1 loss were identified in multiple ER+ breast cancer cells lines from concentrated conditioned media. Additionally, in vivo experiments with xenograft MLH1- ER+ tumors in nude mice, revealed an elevated number of CD206 expressing tumor-promoting M2 macrophages in the tumor microenvironment. Furthermore, MLH1- ER+ breast cancer cells grown in the presence of macrophages exhibit an increase in their migratory capacity. Our preliminary data strongly indicate that loss of MLH1 alters the ER+ breast cancer secretome and alters macrophage maturation, suggesting a role for MLH1 in immunomodulation supporting metastatic progression. Investigation of this new role will uncover key tumor-intrinsic drivers of “cold” tumor immune microenvironment profiles and present unique opportunities for therapeutic interventions to convert them into “hot” profiles, thereby improving ER+ breast cancer patient outcomes.
 Citation Format: Megha Raghunathan, Aloran Mazumder, Sabrina Carrell, Daniel Lozano, Nindo Punturi, Svasti Haricharan. MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-14.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f9ff414c291cb2d359a6c911b717a2e6733c88" target='_blank'>
              Abstract P1-07-14: MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity
              </a>
            </td>
          <td>
            M. Raghunathan, A. Mazumder, Sabrina Carrell, D. Lozano, Nindo B Punturi, S. Haricharan
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Purpose Bladder cancer is one of the ten most common cancers in the world, with a high incidence rate and mortality, and therefore a major burden on the global health care system. PID1 (Phosphotyrosine Interaction Domain 1) functions as an intracellular receptor protein for LRP1. The purpose of this study was to explore the role of PID1 in bladder cancer. Methods RNA-seq data analysis was conducted on 404 BLCA specimens and 28 normal specimens to identify differentially expressed genes. The findings indicated a strong correlation between PID1 expression levels and bladder cancer. We constructed a bladder cancer cell line stably overexpressing PID1 and assessed its impact on cell proliferation and migration. Additionally, We used RT-112 cells to induce tumor formation in nude mice to study the function of the PID1 gene in vivo. Results PID1 expression was notably low in bladder cancer tissues. Compared to SV-HUC-1, RT-112, and SCaBER bladder cells exhibited significantly reduced PID1 expression. Overexpressing PID1 in cells led to the promotion of apoptosis in bladder cancer cells and suppressed cell proliferation and metastasis. In vivo, the overexpression of PID1 demonstrated a significant inhibitory effect on bladder cancer. Furthermore, it was capable of activating the AMPK-mTOR signaling pathway, thereby inhibiting tumor progression. Conclusion PID1 exhibits a potent inhibitory effect on bladder cancer and activates the AMPK-mTOR signaling pathway to hinder tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2abd7d0b87a1062be291a6fc848727bf43e3bfb7" target='_blank'>
              As a New Tumor Suppressor Gene, PID1 Activates the AMPK-mTOR Signal to Inhibit the Progression of Bladder Cancer
              </a>
            </td>
          <td>
            Lingfeng Sun, Chengyi Liu, Yu Cao, Jianghao Li, Lin Yuan
          </td>
          <td>2025-07-01</td>
          <td>Research and Reports in Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Despite many new drugs, multiple myeloma (MM) remains an incurable plasma cell malignancy, and drug resistance is a long-standing topic in this field. Characterized by efficient transcription without being limited by the double helix structure and promoter, extrachromosomal circular DNA (EccDNA) has been proven to be widely involved in cancer development and drug resistance. Methods We performed circle-seq and mRNA-seq on samples from three MM patients at the time of complete response and relapse to screen EccDNA candidate molecules. Outward PCR and Sanger sequencing were used to identify EccDNA molecules. RT‒qPCR and WB were performed to detect gene expression levels. Fluorescence in situ hybridization (FISH) was carried out to detect the deletion of chromosome 17p (del (17p)). Transmission electron microscopy (TEM) was conducted to observe autophagosomes. Luciferase reporter assays were performed to validate the binding of microRNAs to target genes. Cell viability assays and apoptosis assays were employed to assess drug resistance. Xenograft tumor mouse models were established for in vivo experiments. Immunohistochemistry (IHC) was used to detect protein expression levels. Results We successfully identified an EccDNA molecule (EccDNAchr17:38719676–38719812) in one relapsed MM patient with del(17p) and named it MIR4726EccDNA. We demonstrated that the overexpression of MIR4726EccDNA in MM cells can increase bortezomib resistance. We further confirmed that the precursor miRNA carried by MIR4726EccDNA can be efficiently transcribed in MM cells and that MIR4726EccDNA drives bortezomib resistance via the MIR4726-5p/NXF1/NKIRAS2 axis. We further revealed that downregulation of NFKB inhibitor interacting Ras like 2 (NKIRAS2) activated the NF-κB pathway and increased autophagy. Moreover, we established a xenograft model of human MM via subcutaneous inoculation. We administered intra-tumoral injection of AgoMIR4726-5p and intraperitoneal injection of bortezomib and found that AgoMIR4726-5p promoted tumor progression and partially drove bortezomib resistance. Conclusions In summary, our findings indicate that artificially synthesized MIR4726EccDNA is functional in cells and that MIR4726EccDNA enhances tumor progression and partially mediates drug resistance by enhancing MIR4726-5p/NXF1/NKIRAS2 axis dependent autophagy. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02340-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799fad2e2c9bef9366577ed2d5320bd4f2c64f37" target='_blank'>
              MIR4726EccDNA drives bortezomib resistance in multiple myeloma by enhancing MIR4726-5p/NXF1/NKIRAS2 axis dependent autophagy
              </a>
            </td>
          <td>
            Fangfang Li, Xinyi Long, Sishi Tang, Jinhua Yan, Jing Liu, Yunfeng Fu
          </td>
          <td>2025-07-18</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: The molecular subtype largely governs therapeutic strategy in breast cancer patients, but the subtype is not static but frequently changes during treatment courses. The selective pressure on anti-tumor therapy is believed to drive lineage plasticity in luminal breast cancers, promoting a transition to basal-like phenotype and enhancing resistance against endocrine and targeted therapy. This study aims to elucidate the molecular mechanisms underlying luminal-to-basal (LB) subtype conversion through comprehensive genomic and molecular analyses. Methods: We utilized the prospective metastatic breast cancer database to identify patients with LB subtype conversion and luminal-to-luminal (LL) subtype maintenance in matched primary and secondary tumor tissues. Whole-exome sequencing (WES) and RNA-seq were performed on available tumors of the patients (exploratory cohort). We also validated genomic alterations in a separate cohort of patients who received next-generation sequencing (NGS) for clinical practice (validation cohort). Molecular factors associated with LB conversion were manipulated in MCF7 and T47D cell lines to further explore changes driving lineage plasticitiy, including chromatin accessibility studies using ATAC-seq. Results: Our analysis of metastatic breast cancer cohort with paired biopsy (n= 359) revealed overall 29.2% conversion rate of subtypes. LB conversion (from HR+HER2− to TNBC, 8.6%) was the most common subtype conversion and was associated with poorer survival compared to LL subtype maintenance. In the exploratory cohort, TP53 mutations were significantly more prevalent in patients undergoing LB conversion (64.3% vs. 15.4%), whereas GATA3 mutations appeared to be protective (0% vs. 23.1%). The validation NGS cohort (n=70) further confirmed the enrichment of TP53 mutation (85.7% vs. 23.2%) and the protective role of GATA3 (7.1% vs. 24.6%) in patients with LB conversion. The RNA-seq analysis indicated upregulation of KRAS-dependent signatures and decreased GATA3 expression in secondary tumors after LB conversion in TP53-mutant patients. TP53 depletion in MCF7 cells led to reduced ERα level and tamoxifen resistance, accompanied by increased accessibility in genes related to development regulation and AP-1 binding. Further combined TP53 and GATA3 depletion synergistically induced basal phenotype by activating the KRAS-MAPK pathway. MAPK inhibition restored luminal phenotype and tamoxifen sensitivity in luminal breast cancer against LB conversion induced by TP53 and GATA3 knockdown. MAPK inhibition was also effective in overcoming endocrine therapy resistance in tamoxifen- and palbociclib-resistant breast cancer cells exhibiting the LB conversion phenotype. Conclusion: Our findings establish TP53 mutations as a crucial predictor of LB subtype conversion in luminal breast cancers, with GATA3 mutations offering a protective effect. Combined TP53 and GATA3 loss synergistically promotes induction of basal phenotype through KRAS activation, and MAPK inhibition is a potential therapeutic strategy for luminal breast cancer patients with LB conversion.
 Citation Format: Min Hwan Kim, Won-Ji Ryu, Tae Yeong Kim, Yumi Hwang, Hyun-Yi Kim, Hyun Ju Han, Seul-Gi Kim, Gun Min Kim, Ja Seung Koo, Hyung Seok Park, Seung Il Kim, Joohyuk Sohn. Luminal-to-basal subtype conversion is governed by TP53-GATA3 mutational status and is targetable by MAPK inhibition in patients with breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-07.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0adb85d71d667a6b401b605d77a2d8d33145413e" target='_blank'>
              Abstract P5-06-07: Luminal-to-basal subtype conversion is governed by TP53-GATA3 mutational status and is targetable by MAPK inhibition in patients with breast cancer
              </a>
            </td>
          <td>
            Min Hwan Kim, W. Ryu, Tae Yeong Kim, Yumi Hwang, Hyun-Yi Kim, H. Han, Seul-Gi Kim, G. Kim, Ja Seung Koo, Hyung Seok Park, S. Kim, J. Sohn
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="RNA modifications are crucial for post-transcriptional gene regulation. Research on RNA modifications has become a novel frontier of epitranscriptomics. Up to now, over 170 kinds of modifications have been identified on mRNA and diverse non-coding RNA. Three classes of proteins (writers, erasers, and readers) regulate the addition, removal, and identification of epigenetic marks, thus affecting RNA biological functions. Increasing evidence identifies the dysregulation of RNA modifications in different cancer types and the therapeutic potential of targeting RNA-modifying enzymes. The ability of RNA modifications to improve mRNA stability and translation efficacy and decrease immunogenicity has been exploited for the clinical use of mRNA cancer vaccines. This review aims to shed light on several vital cap, tail, and internal modifications of RNA with a focus on the connection between RNA epigenetic pathways and cancer pathogenesis. We further explore the clinical potential of RNA modifications as dynamic biomarkers for cancer diagnosis, prognosis, and therapeutic response prediction, addressing both technological challenges and translational opportunities. Finally, we analyze the limitations of current studies and discuss the research focus in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1031a202549cfe25c76606fc3cff87195a4354" target='_blank'>
              RNA epigenetic modifications as dynamic biomarkers in cancer: from mechanisms to clinical translation
              </a>
            </td>
          <td>
            Yingchao Zhao, Xiang Chen, Xingli Zhang, Hong Liu
          </td>
          <td>2025-06-07</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In recent years, the development of targeted therapies for tumors with KRAS mutations has progressed rapidly, rendering the notion of KRAS as “undruggable” outdated. However, targeted therapies for KRAS mutations still face numerous challenges, including resistance, efficacy concerns, toxicity issues, and hurdles in drug development. Exploring alternative treatment modalities is thus essential. Extensive research has demonstrated that KRAS mutations significantly influence the immune microenvironment, presenting both challenges and opportunities for immunotherapy. Interestingly, it has been observed that different KRAS mutations and co-mutation subtypes exhibit significant variations in their immunological microenvironments, which undoubtedly impact immunotherapy choices. Here, we review the history of KRAS-targeted therapy, highlighting existing challenges, and summarize changes in the immune microenvironment of KRAS-mutated cancers and their potential therapeutic targets. We compare differences in the immune microenvironment across various mutation types and co-mutation subtypes, and offer perspectives on future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993d360fa5818a3754af4b74cbddfac722d927a4" target='_blank'>
              Precision immune regulation in KRAS-mutated cancers: the final piece of the puzzle?
              </a>
            </td>
          <td>
            Shenao Fu, Jiayao Ma, Changjing Cai, Jun Tan, Xiangying Deng, Hong Shen, Shan Zeng, Yihong Chen, Ying Han
          </td>
          <td>2025-07-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Given the propensity of aggressive epithelial tumors to form hepatic metastases, we performed an in vivo cDNA screen using the mouse liver and KRASG12D/TP53R273H pancreatic cells to identify the RNA binding protein GCN1 as integral component of hepatic outgrowth. RNAi experiments reveal that GCN1 triggers the ISR to activate serine, folate, and methionine biosynthetic pathways together with amino acid transporters, which act in concert to facilitate acquisition of metabolites and to restore redox homeostasis. Alongside activation of the ISR, we found that GCN1 also functions in the nucleus where it interacts with HNRNPK to suppress the expression of MHC-I molecules, and natural killer (NK) ligands. Intriguingly, we identified IMPACT as an endogenous competitive inhibitor of GCN1 that blocks both ISR-dependent metabolic control and disrupts HNRNPK interaction. In doing so, IMPACT enhances tumor immunogenicity to unleash NK cell killing, in addition to sensitizing metastatic tumor cells to immune checkpoint blockade (ICB).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940084da31cd31fac078b9eabda5419618abe96d" target='_blank'>
              Expression of IMPACT curtails metabolic plasticity and augments NK cell killing to abrogate metastatic growth.
              </a>
            </td>
          <td>
            Surajit Sinha, A. Panda, Rodrigo Xavier das Neves, Z. Nwosu, Ke Xu, Elke van Beek, Priyanka P Desai, Sivasish Sindiri, Sruthi Chempati, K. Remmert, Billel Gasmi, Linda Bojmar, Constantinos Zambirinis, Alexander J Rossi, Reed I Ayabe, Michael M. Wach, James D McDonald, Samantha M Ruff, Emily A Verbus, A. Saif, Alyssa V Eade, C. Larrain, Lindsay R Friedman, Shreya Gupta, Alok Ranjan, Martha E Teke, Tahsin M. Khan, Tracey Pu, A. Sarvestani, Carrie E Ryan, Jacob T Lambdin, K. Luberice, Stephanie N. Gregory, Stephanie C Lux, Hanna Hong, Allen J Luna, Imani A Alexander, Sarfraz R. Akmal, Shahyan Rehman, Ashley Rainey, Todd D Prickett, Vishal N. Kopardé, Samantha Sevilla, Skyler A. Kuhn, King Chan, Zhonghe Sun, Nina Bubunenko, Eileen Li, C. Hannah, Geneti Gaga, T. Andresson, Margaret C. Cam, Xiaolin Wu, Lisa M. Jenkins, A. Blakely, Jeremy L. Davis, Giorgio Trinchieri, Pankaj K. Singh, James C. Yang, M. Pasca di Magliano, C. Lyssiotis, Micheal B Yaffe, Ethan M Shevach, Jonathan M Hernandez
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="As a diverse family of diseases, cancer is unified by a set of common dysfunctions, such as limitless growth potential and an insensitivity to antigrowth signals. These shared overarching biological processes have been termed the hallmarks of cancer. To better understand the root cause of cellular dysregulation, intense molecular characterization of tumors has utilized DNA, RNA, and protein measurement techniques to produce proteogenomic data. In large cancer cohort studies, genomic and proteogenomic data have frequently identified many cancer hallmarks including cell cycle and cell signaling. However, altered metabolism, a known cancer hallmark, is not as clearly identified in mutation screens or differential expression analyses. Here, we introduce a new computational method to identify changes in cellular regulation by focusing on the mRNA/protein relationship. We create a metric, Δ_corr, to capture when the mRNA/protein correlation changes significantly between tumor and normal tissues and show that it is distinct from differential expression and also not associated with DNA mutation profiles. Our method clearly highlights altered metabolic pathways across multiple tumor types. Δ_corr gives researchers a new perspective on the dysfunction of tumor cells and introduces a novel method for proteogenomic data integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0188673355274102e7547e046958401284e61db8" target='_blank'>
              Activating Cancer Hallmarks through Changes in mRNA/Protein Regulation.
              </a>
            </td>
          <td>
            Jose Humberto Giraldez Chavez, Nathaniel Barton, Caleb M Lindgren, Bryn Mendenhall, Benjamin Kimball, Samuel H. Payne
          </td>
          <td>2025-07-17</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CRISPR activation and interference (CRISPRa/i) are highly effective tools to regulate transcription by fusing dead Cas9 (dCas9) with transcriptional regulatory factors guided by small guide RNA (sgRNA) in mammalian cells and mice. Still, a controllable gene regulation system is desired to investigate and manipulate dynamic biological processes. Here, we reported flexible drug-responsive CRISPRa/i systems by fusing mutated human estrogen receptor (ERT2) domains, which responded to estrogen analogue tamoxifen or its active metabolite 4-hydroxy-tamoxifen (4OHT), to CRISPRa/i components for transcriptional regulation. Upon 4OHT treatment, the optimal variants, ERT2-ERT2-CRISPRa/i-ERT2 (iCRISPRa/i), showed rapid protein translocation of iCRISPRa/i from cytoplasm to nucleus and subsequent transcriptional response. The inducible transcriptional manipulation could be restored to its original level when 4OHT was withdrawn. Moreover, the efficiencies of gene expression regulation of iCRISPRa/i were comparable to those of non-inducible and doxycycline-inducible counterparts, with a lower leakage and a faster drug response activity. The iCRISPRa/i systems successfully induced phenotypic changes in various cell lines. These results highlight that iCRISPRa/i systems could achieve fast and flexible drug-responsive transcriptional modulation and phenotypic changes, and thus provide better options for gain- and loss-of-function model construction and gene therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05786-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1338656a4c84c744345ea4f2ecdb230967ac2ab" target='_blank'>
              Novel drug-inducible CRISPRa/i systems for rapid and reversible manipulation of gene transcription
              </a>
            </td>
          <td>
            Ming Sui, Meiling Zhou, Mengge Cui, Huan Liu, Xiaolin Zhang, Na Hu, Yang Li, Beibei Wang, Guojun Yang, Pengling Gui, Lingqiang Zhu, Feng Wan, Bin Zhang
          </td>
          <td>2025-06-23</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Purpose RNF213 encodes one of the largest E3 ubiquitin ligases in the human proteome, playing essential roles in ubiquitination, DNA damage repair, and immune system regulation. Although RNF213 has been extensively studied in Moyamoya disease, its involvement in cancer biology remains insufficiently understood. This study seeks to conduct a comprehensive pan-cancer analysis to explore RNF213 expression, mutation characteristics, its correlation with clinical outcomes, and its potential as a prognostic biomarker and a therapeutic target, particularly in the context of immunotherapy. Methods Leveraging data from The Cancer Genome Atlas (TCGA) and additional publicly available datasets, we performed a systematic analysis of RNF213 expression patterns, mutation frequencies, and their relationships with clinical outcomes. We also conducted survival analyses to assess the impact of RNF213 expression on the efficacy of immune checkpoint blockade therapies. Results RNF213 demonstrated variable expression levels across different cancer types, with notable overexpression observed in glioblastoma and endometrial carcinoma, while underexpression was detected in lung adenocarcinoma and prostate adenocarcinoma. Elevated RNF213 expression was linked to unfavorable survival outcomes in acute myeloid leukemia and low-grade glioma, whereas it was associated with better survival in sarcoma and skin cutaneous melanoma. RNF213 expression showed a positive association with immune-related genes and immune cell infiltration, particularly in glioblastoma and breast cancer. Furthermore, patients with high RNF213 expression experienced significant survival advantages when receiving combined anti-PD-1 and anti-CTLA-4 immunotherapies. Conclusion This study highlights RNF213’s dual functionality in tumor progression and immune modulation, underscoring its potential as both a prognostic biomarker and a therapeutic target. The findings suggest that RNF213 may play a pivotal role in determining immunotherapy outcomes, warranting further exploration into its underlying mechanisms and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03237-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd17eb7e2de6f21c63c696e22b05947b1a82bc9b" target='_blank'>
              RNF213 governs divergent tumor-immune dynamics across human cancers: a prognostic biomarker for immunotherapy stratification
              </a>
            </td>
          <td>
            Jinpeng Wen, Zhekuan Lv, Mojiao Lu, Chuanming Zheng, Changtian Yin
          </td>
          <td>2025-07-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long interspersed nucleotide element 1 (LINE1), the most abundant repetitive element in the human genome, plays a crucial role in genomic instability. Aberrant LINE1 activation, primarily regulated by DNA methylation, is a hallmark of cancer. Non-small cell lung cancer (NSCLC), the most prevalent form of lung cancer worldwide, continues to pose significant challenges due to the invasiveness, high cost, and susceptibility to false positives of current diagnostic methods, as well as the emergence of treatment resistance. This review highlights the potential of LINE1 methylation as a biomarker for NSCLC, offering novel insights into its role in diagnosis, prognosis, and therapeutic strategies. Recent studies uncovered that LINE1 hypomethylation was strongly associated with poor overall survival, suggesting its utility as both a prognostic marker and a therapeutic target. However, further research is required to elucidate its precise regulatory mechanisms in LINE1 retrotransposition and to evaluate its potential as a non-invasive biomarker for improving NSCLC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4219f213fb15eaa64e362c434180801a85ab734e" target='_blank'>
              Long interspersed nuclear element 1 methylation in non-small cell lung cancer: implications for diagnosis, prognosis, and therapeutic targeting
              </a>
            </td>
          <td>
            Dileesha Prabani Wanasundara Arachchillage, Wanvisa Udomsinprasert
          </td>
          <td>2025-07-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="An R-loop is a complex nucleic acid structure consisting of an RNA–DNA hybrid and an associated single-stranded DNA. This structure plays an important role in many biological processes, including gene regulation, DNA replication, and maintenance of genome stability. Under physiological conditions, the R-loop is in a dynamic balance of generation and decomposition, which is a complex process involving multiple mechanisms of its formation, clearance, and regulation. Emerging evidence indicates that R-loops can drive inflammation and participate in immune processes. This review summarizes immune dysregulation resulting from impaired R-loop clearance and analyzes recent advances in computational methods for R-loop regulation, aiming to identify new avenues for developing R-loop-targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bce6b2f2435a1bf918ec22e6cdf3b596fdd7f0" target='_blank'>
              Unraveling R-loops: The hidden drivers of inflammation and immune dysregulation
              </a>
            </td>
          <td>
            Fan Zhang, Dingyun Wang, Guodong Zhao, Dingmao Wang
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Glioblastoma multiforme (GBM) is a lethal brain tumor. With the current gold standard chemotherapy treatment, temozolomide (TMZ), many patients do not survive beyond one year. While the urgency of researching novel treatments is understandable, the prohibitively high costs and the prolonged duration of research and clinical trials significantly delay the availability of medical advancements to the general public. This highlights the urgent need for adjuvant therapies to enhance treatment effectiveness. Main body: Recent research has suggested the potential of repurposing FDA-approved drugs such as temozolomide (TMZ), disulfiram (DSF), and aspirin for the treatment of glioblastoma, with encouraging evidence particularly for DSF and aspirin. Additionally, compounds like histone deacetylase inhibitors (e.g., vorinostat) are being investigated for their impact on non-coding RNA (ncRNA) modulation, including microRNAs (miRNAs) and long non-coding RNAs (lncRNAs). Combining traditional therapies with ncRNA modulation has shown potential in enhancing therapeutic efficacy and targeting specificity. NcRNAs play a crucial role in regulating gene expression and have been implicated in tumor growth, invasion, and treatment resistance. Recent discoveries, such as cuproptosis, offer new insights into tumor cell death mechanisms. Conclusion This review focuses on how these molecular insights can serve as novel therapeutic targets and how drug adjuvant therapy may improve GBM treatment strategies. It focuses on how the integration of ncRNA modulation with conventional therapies and the combination strategy of enhancing efficacy of drugs can enhance treatment efficacy and pave the way for innovative approaches in managing GBM. In short, we will explore how non-coding RNAs (ncRNAs) might serve as promising targets and how repurposing TMZ, DSF, and aspirin could help enhance the efficacy of GBM treatment. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f928ba5f4bc7b8cba2127288e9078cdfc0a40a8" target='_blank'>
              A novel approach to enhance glioblastoma multiforme treatment efficacy: non-coding RNA targeted therapy and adjuvant approaches
              </a>
            </td>
          <td>
            Meijun Liu, Yuying Wang, Xiaoli Chen, Yu Zeng, Wenqiong Huang, Jiawen Yang, Hong Dai, Lixin Cheng, Claudio Mauro, Kenneth Cheung
          </td>
          <td>2025-06-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Proto-oncogenes in the RAS superfamily play dual roles in maintaining cellular homeostasis, such as regulating growth signals and contributing to cancer development through proliferation and deregulation. Activating proto-oncogenes in vitro transforms cells, underscoring their centrality in gene regulation and cellular networks. Despite decades of research, poor outcomes in advanced cancers reveal gaps in understanding Ras-driven mechanisms or therapeutic strategies. This narrative review examines RAS genes and Ras proteins in both housekeeping functions, such as cell growth, apoptosis, and protein trafficking, as well as in tumorigenesis, integrating insights from human (HRAS, KRAS, NRAS), mouse (Hras, Kras, Nras), and Drosophila melanogaster (ras) models. While RAS mutations are tightly linked to human tumors, the interplay between their standard and oncogenic functions remains complex. Even within the same tissue, distinct cancer pathways—such as the mitogen-activated protein kinase (MAPK) and phosphoinositide 3-kinase (PI3K) pathways—can drive varied disease courses, complicating treatment. Advanced-stage cancers add further challenges, including heterogeneity, protective microenvironments, drug resistance, and adaptive progression. This synthesis organizes current knowledge of RAS gene regulation and Ras protein function from genomic alterations and intracellular signaling to membrane dynamics and extracellular interactions, offering a layered perspective on the Ras pathway’s role in both housekeeping and tumorigenic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796354d7dffc674d220d0fc486fafe630423dfaf" target='_blank'>
              RESEARCH CHALLENGES IN STAGE III AND IV RAS-ASSOCIATED CANCERS: A Narrative Review of the Complexities and Functions of the Family of RAS Genes and Ras Proteins in Housekeeping and Tumorigenesis
              </a>
            </td>
          <td>
            Richard A. McDonald, A. Varela-Ramírez, Amanda K. Ashley
          </td>
          <td>2025-07-25</td>
          <td>Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND/AIM
While normal cells are highly regulated, cancer cells take a dysregulated path which bolsters their survival. Currently, a limited number of uniform treatments are available for cancer cure. Our goal was to deprive cancer cells of the key nutrient methionine and determine what effect it would have on cell death and alterations in DNA methylation of prostate cancer cells.


MATERIALS AND METHODS
PC3 and other cell lines were transfected with plasmid gene constructs for methionine gamma lyase deaminase (MEGL), a methionine-degrading enzyme, targeted for expression in the cytoplasm (cMEGL) or the nucleus (nMEGL). For assessing cell death due to MEGL-mediated methionine deprivation, a standard 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay was used. PC3, and DU145 prostate cancer cells were selected for whole-methylome sequencing to determine the effects of MEGL expression. Key gene products comprising the Prolaris Molecular Score, specifically 31 cell-cycle progression genes, were chosen for assessing putative differences in methylome.


RESULTS
Treatment with MEGL gene targeted for expression in either the cytoplasm or nucleus caused significant cell death, similar to that due to the anticancer drug methotrexate. Azacytidine showed no effect on PC3 cell death. Propargylglycine, an inhibitor of MEGL, prevented cell death. Methylome analysis showed increased methylation of two genes: Spindle and kinetochore-associated complex subunit 1 (SKA1), origin recognition complex subunit 6 (ORC6L), and reduced methylation of six promoters: BUB1 mitotic checkpoint serine/threonine kinase B (BUB1B), PDZ binding kinase (PBK), baculoviral IAP repeat-containing 5 (BIRC5), centromere protein M (CENPM), DNA topoisomerase II alpha (TOP2A), minichromosome maintenance 10 replication initiation factor (MCM10), upon forced expression of MEGL.


CONCLUSION
Methionine deprivation through MEGL-targeted gene therapy may be a viable option for inducing cancer cell death compared to unrestricted levels of methionine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4960faa090cc3a8364fc997a8a0c4944dac73e5" target='_blank'>
              Methionine Deprivation-induced Cancer Cell Death and Methylation Changes in Key Genes and Gene Promoters of Prostate Cancer Cell Lines.
              </a>
            </td>
          <td>
            Yatin Srinivash Ramesh Babu, K. Venkatachalam
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mutations that activate the small GTPase KRAS are a frequent genetic alteration in cancer, and drug discovery efforts have led to inhibitors that block KRAS activity. We sought to better understand oncogenic KRAS signaling and the cytostatic effects of drugs that target this system. We performed proteomic analyses to investigate changes in protein abundance and posttranslational modifications in inhibitor-treated human KRAS-mutant pancreatic (KRAS G12C and G12D) and lung cancer (KRAS G12C) cells. The inhibitors used target these mutant forms of KRAS, the downstream effectors MEK and ERK, and the upstream regulators SHP2 and SOS1. Comparisons of phosphoproteomes between cell lines revealed a core KRAS signaling signature and cell line-specific signaling networks. In all cell lines, phosphoproteomes were dominated by different degrees of autonomous, oncogenic KRAS activity. Comparison of phosphoproteomes after short and long drug exposures revealed the temporal dynamics of KRAS-MEK-ERK axis inhibition that resulted in cell cycle exit. This transition to a quiescent state occurred in the absence of substantial proteome remodeling but included broad changes in protein phosphorylation and ubiquitylation. The collective data reveal insights into oncogenic KRAS signaling, place many additional proteins into this functional context, and implicate cell cycle exit as a mechanism by which cells evade death upon KRAS signaling inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2163ec601f10210697958516b4e39127c69ccee" target='_blank'>
              Proteomic analyses identify targets, pathways, and cellular consequences of oncogenic KRAS signaling.
              </a>
            </td>
          <td>
            Nicole Kabella, Florian P. Bayer, Konstantinos Stamatiou, Miriam Abele, Amirhossein Sakhteman, Yun-Chien Chang, Vinona Wagner, Antje Gabriel, Johannes Krumm, Maria Reinecke, Melanie Holzner, Michael Aigner, Matthew The, Hannes Hahne, Florian Bassermann, Christina Ludwig, Paola Vagnarelli, Bernhard Kuster
          </td>
          <td>2025-07-29</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The initiation and progression of breast cancer generally involve complex immune regulatory mechanisms, with increased expression of programmed cell death ligand 1 (PD-L1) as an essential factor for immune evasion and the formation of a tumor-promoting immune microenvironment. Emerging evidence underscores the regulatory role of non-coding RNAs (ncRNAs) in modulating PD-L1 expression, influencing immune evasion, tumorigenesis, and therapy resistance in breast cancer. Therefore, it is crucial further to clarify alternative regula-tory mechanisms that control PD-L1 expression. The variations in PD-L1 expression among different breast cancer subtypes and the mechanisms by which ncRNAs regulate the expres-sion of PD-L1 are delineated. This study explores the potential and challenges of combining ncRNA-based therapy with PD-L1 inhibitors, offering insights into PD-L1 regulation and per-sonalized treatment strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e488b9798e3dd6ded6d2b5f4bc4c5b44375c7e67" target='_blank'>
              Role of Non-Coding RNAs in Regulating PD-L1 Expression in Breast Cancer: Emerging Insights and Implications.
              </a>
            </td>
          <td>
            Jianqin Zhu, Weijin Zhu, Xulin Zhou, Jingwen Hua, Xiaochun Sun
          </td>
          <td>2025-06-18</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to cisplatin (CDDP) in oral cancer presents a significant challenge, driven by mechanisms such as enhanced DNA repair, increased drug efflux, and altered cell cycle regulation. BRCA1 and BRCA2 are tumour suppressor genes. They play a crucial role in DNA repair and cell cycle control, helping to prevent cancer development. Mutations in these genes increase the risk of developing several cancers. These key genes, BRCA1 and BRCA2, are crucial in repairing cisplatin-induced DNA damage. Resistance involves off-target mechanisms like ERBB2, DYRK1B, TMEM205, and RAB8A overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6df8cb8291587e881ade3e7977c52dbc0b0856" target='_blank'>
              Spherical k-means clustering in the cisplatin resistome of oral cancer
              </a>
            </td>
          <td>
            S. Ayyachamy
          </td>
          <td>2025-06-17</td>
          <td>Gaceta Médica de Caracas</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/117910c4d3fcbf935a679f9492965ebebd549451" target='_blank'>
              Molecular Interplay of Gene Network Dynamics, Epigenetic Regulation, and Therapeutic Mapping in Cardiovascular Disease.
              </a>
            </td>
          <td>
            Md Rashedunnabi Akanda, Md Shiblee Sadik Sabuj, S. M. A. Salam, Eshrat Jahan
          </td>
          <td>2025-06-20</td>
          <td>Cardiovascular drugs and therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 The substantial improvement in childhood cancer survival rates has been primarily achieved through therapy intensification, often at the cost of life-long adverse effects. Therapies can be associated with characteristic patterns of mutations found in the tumor genome, known as mutational signatures. These signatures, therefore, can act as markers of resistance and late effects of therapy. We employed a large-scale analysis of 611 whole-genome sequenced pediatric tumors with harmonized therapy data, integrating de novo mutational signature extraction with ensemble machine learning (ML) models trained on comprehensive genomic features (single- and double-base substitutions, indels, structural variants, and copy number alterations) to identify therapy-associated patterns. De novo extraction across single-nucleotide variants, indels, structural variants, and copy number alterations revealed 95 distinct mutational signatures, including 31 novel ones. Platinum therapy was associated with the highest mutagenic burden, with canonical signatures (e.g., SBS31, SBS35, DBS5) detected in 41% of exposed tumors. By aligning with treatment timelines, we defined a median latency of 91 days and a burden threshold of ∼1500 mutations for platinum signatures to emerge—appearing in all osteosarcomas (most common primary bone cancer) and two-thirds of neuroblastomas (most common extracranial solid tumor in children). Despite these insights, the reasons why some tumors do not exhibit these signatures, and how this relates to their response to therapy, remain poorly understood. To uncover subtler therapy-associated signals beyond known COSMIC signatures, we trained ensemble ML models on various types of genomic alterations. Our ML models achieved high predictive performance (F1-score = 0.89) in classifying platinum-exposed tumors. Importantly, predictive accuracy persisted even after masking features corresponding to canonical platinum signatures, indicating the presence of non-canonical mutational patterns. Feature analysis identified biologically plausible non-canonical genomic features enriched in platinum-exposed tumors across pediatric cancers, suggesting previously unannotated mutational footprints of therapy. Notably, ML analysis revealed distinct non-canonical genomic features associated with anthracyclines, and select antimetabolites and alkylating agents, shedding light on therapies previously lacking characterized mutational signatures. These novel genomic signals were also validated across independent pediatric and adult cancer cohorts. This work highlights the power of ML to detect therapy-induced mutational patterns beyond known mutational signatures. As cancer remains the leading cause of disease-related death among children in many countries, this integrated ML-driven approach offers a powerful strategy for improved therapy monitoring, early resistance prediction, and ultimately, the development of more personalized and effective treatment strategies (e.g., stratification and de-escalation) for children with cancer.



 Mehdi Layeghifard, Marcos Díaz-Gay, Pedro L. Ballester, Elli Papaemmanuil, Mark Cowley, Anita Villani, Ludmil B. Alexandrov, Adam Shlien. Unmasking hidden mutational footprints of therapy in pediatric tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A038.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6789b44a7ab2f8906895c50474f02a3bfc36f14" target='_blank'>
              Abstract A038: Unmasking hidden mutational footprints of therapy in pediatric tumors
              </a>
            </td>
          <td>
            M. Layeghifard, M. Díaz-Gay, Pedro L. Ballester, E. Papaemmanuil, Mark J. Cowley, A. Villani, Ludmil B. Alexandrov, A. Shlien
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdefbda29fca27b21f2cc0c3e7a4827b0d0e9113" target='_blank'>
              The Role of Epigenetic Modifications in Enhancing Bone Marrow Transplantation Efficacy in Cancer.
              </a>
            </td>
          <td>
            B. Arjmand, Fatemeh Shahrahmani, Alieh Mirzaei, Arian Emamifar, Sepideh Alavi-Moghadam
          </td>
          <td>2025-06-17</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Recurrent somatic mutations in the key spliceosome component, SF3B1, have been identified at various frequencies across several cancer types. The most common hotspot mutation is the K700E missense mutation, and while its effects on splicing have been well characterised at the molecular level, the mis-spliced genes that contribute to cancer progression and/or dictate responses to therapy are still unclear. Here, we used we use cell line modelling to assess the impact of the SF3B1K700E mutation on the cellular response to various apoptosis-inducing agents. Our data suggest that the SF3B1K700E mutation leads to reduced cFLIP levels, along with defects in the splicing and translation of BCL2, causing a shift in the balance of pro- and anti-apoptotic genes and proteins, which confers greater sensitivity to the bivalent SMAC mimetic, BV-6. As such, BV-6 may represent a therapeutic opportunity for patients with SF3B1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78c6127f794d73c9e1a0f7e0fc6e3deedb616b30" target='_blank'>
              The cancer-associated SF3B1K700E spliceosome mutation confers enhanced sensitivity to BV-6-induced cytotoxicity
              </a>
            </td>
          <td>
            Lydia E Roets, J. Blayney, Hayley McMillan, Patrick J Preston, Alexander Mutch, Ken I Mills, K. Savage, K. Lappin
          </td>
          <td>2025-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer patients who express estrogen receptor α (ER) typically receive endocrine therapy as a first-line treatment. Most ER-positive breast cancer patients initially respond to endocrine therapy, but up to 40% of patients develop resistance over time, and the main mechanism is aberrant activation of ER signaling pathways. Recent research has shown that cancer's uncontrolled proliferative capacity and a hostile microenvironment can induce endoplasmic reticulum (EnR) stress and trigger unfolded protein response (UPR), which plays a crucial role in determining cell fate. Furthermore, increasing evidence suggests that noncoding ribonucleic acids (RNAs), which are involved during the crosstalk between the UPR and ER signaling pathways, play an essential role in endocrine resistance. In this review, we provide an overview of long noncoding RNA (lncRNA) that is involved both in endocrine resistance and UPR. We provide new insights into potential treatment strategies to overcome endocrine resistance in ER-positive breast cancer patients by targeting the lncRNA that plays key roles in endocrine resistance and UPR signaling. Finally, we discuss the current state and future directions for targeting lncRNAs to improve clinical outcomes in endocrine-resistant breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4692ed07176028a188031985ca7bc44cfc350d2e" target='_blank'>
              The role of UPR-related long noncoding RNAs in development of endocrine resistance in breast cancer.
              </a>
            </td>
          <td>
            Wen Liu, Sahil Gupta, Sanjeev Gupta
          </td>
          <td>2025-08-01</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Transcriptional regulation plays a pivotal role in coordinating the complex morphogenetic and molecular events involved in heart development and function. In the early stages of cardiovascular diseases (CVDs), the Mediator complex (MED) performs a variety of essential functions. While initial studies focused on correlating MED components with specific CVDs, more recent research has shifted toward a deeper exploration of the MED's role in the early pathogenesis of these diseases. This review highlights the latest findings published between January 2018 and February 2025, with a particular focus on the protein subunits MED1, MED12, MED13, MED13L, MED15, and MED23, and their implications in various cardiovascular pathologies. The MED complex is a crucial regulator of gene transcription, bridging transcription factors and RNA polymerase II. In “-omic” sciences, studying MED functions is essential to understanding molecular interactions that regulate gene expression. Within precision medicine, the MED complex is a key node in gene expression networks. Its study within the -omic framework can provide valuable insights into how molecular interactions shape cellular processes in both health and disease, ultimately enhancing the diagnosis, understanding, and treatment of CVDs through personalized medicine. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2962b5985f4ef86033a3760d78a66a5b8788022" target='_blank'>
              Mediator complex: update of key insights into transcriptional regulation of ancestral framework and its role in cardiovascular diseases
              </a>
            </td>
          <td>
            C. Schiano, Claudio Napoli
          </td>
          <td>2025-06-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="
 In the United States, breast cancer is one of the leading causes of cancer death in women. Triple-negative breast cancer (TNBC) is the most aggressive of all molecular subtypes of breast cancer. It is highly heterogeneous and difficult to target and treat due to its lack of targetable receptors. Our aim is to identify new molecular targets that could aid the treatment of TNBC cases and reduce cytotoxicity of the current therapeutic regimen. The TP53 gene, a known tumor suppressor is the most frequently mutated gene in triple-negative breast cancer occurring in >80 % of TNBC cases. Statistical pathway analysis from RNA-Seq and ChIP-Seq done on the MCF10A mammary epithelial cell lines expressing the 10 most clinically prevalent p53 mutants identified the Hippo pathway to be dysregulated when associated with aggressive cellular phenotypes. Upon suppression of the upstream Hippo tumor suppressor pathway, the activation of a series of genes promoting cancer hallmark phenotypes ensues. A cell-based TEAD reporter assay showed transcriptional activity of TEAD to be higher in MCF10A cells expressing more aggressive p53 mutants compared to the cell lines with less aggressive mutants. Knockdown of the TAZ protein decreased the invasiveness of the invasive p53 mutant expressing cell lines. We hypothesized that, in addition to neo-morphic activities of different p53 mutants, the heterogeneity in clinical phenotypes observed in TNBC is due to functional interactions of the driver mutation in p53 with other functionally important co-existing mutations and dysregulated pathways, called ‘co-drivers’. To identify potential functionally important ‘co-driver’ mutations that cause cancer progression and heterogeneity in clinical phenotypes, a genome-wide screen was performed using lentiviral CRISPR screen library on the invasive MCF10A-p53-R273C-MycOE cell line. In vivo experiments showed that injection of the library-transduced cells yielded tumor in mice. PCR and whole-exome sequencing of the tumor showed frameshift mutations in the enriched EVC2 (EvC Ciliary Complex Subunit 2), NR1D1 (Nuclear Receptor 1D1), and ARAF (a Ras-activating RAF family kinase) genes. Confirming the hits from screening, knockout of these genes in R273C-MycOE cells also showed increased cell invasion. In parallel, in vitro screens carried out using CRISR knock-out library on the least aggressive MCF10A-p53-Y234C cells identified potential co-drivers. Knockout of GCNT1 and FGD6 genes in MCF10A-p53-Y234C cells increased cell invasion and induced dephosphorylation and activation of TAZ. TEAD activity was also found to be higher in these knock-out cells compared to the Y234C cells. Knockdown of the TAZ protein showed decreased TEAD activity in the FGD6 and GCNT1 knock-out cells and substantially decreased cell invasion. We have carried out functional characterization of the TP53 mutations in TNBC and confirmed the cellular phenotypes for specific p53 mutants obtained from the in vitro and in vivo screens. We are validating these identified ‘co-driver’ genes and targets to discover the molecular mechanisms affecting phenotypic heterogeneity of the p53 mutants and investigating the mechanisms through which Hippo pathway drives the progression of TNBC, as well as assessing the pathway as a target for potential therapy for TNBC with specific TP53 mutations.
 Citation Format: Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura Gonzalez-Malerva, Duatin Grief, Lydia Sakala, Jin Park, Joshua LaBaer. Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f9063c7f21ea86d31d2953e319cbe0d8301969" target='_blank'>
              Abstract P2-11-20: Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches
              </a>
            </td>
          <td>
            Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura González‐Malerva, Duatin Grief, Lydia Sakala, Jin G Park, Joshua LaBaer
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Given the prevalence of RAS mutations in various cancers, personalized therapeutic approaches, guided by molecular markers, are essential. Farnesyltransferase inhibitors (FTIs) have emerged as potential therapeutic options; however, they also face obstacles such as toxicity and limited efficacy. Alternative strategies, such as direct inhibitors combined with pathway modulators, RNA interference, and gene-editing technologies, are under clinical investigation. The targeting of RAS, complicated by its structural nuances, particularly in the G domain, has advanced with the identification of druggable pockets such as the SW-II pocket. This breakthrough has led to the development of targeted therapeutics, such as sotorasib and adagrasib, for KRAS G12C-mutated non-small cell lung cancer (NSCLC). However, these advancements face challenges, including adaptive resistance and the necessity for isoform selectivity. New inhibitors, such as LY3537982 or GDC-6036, are promising, but achieving effective and selective RAS inhibition remains a significant challenge. Additionally, clinical trials have highlighted variability in patient responses, attributing limited treatment efficacy to resistance mechanisms, including on-target mutations and off-target pathway activations. Finally, the RAS oncogene, traditionally viewed as predominantly pro-cancerous, plays a complex role in oncogenesis, with recent evidence suggesting context-dependent effects, such as inducing senescence in certain cells. This shift in understanding underscores the therapeutic potential of manipulating the interplay between RAS and TP53 mutations in cancer. In conclusion, the complexity of effectively targeting the RAS-RAF-ERK pathway is exacerbated by the diverse resistance mechanisms. Challenges such as off-target effects and delivery issues remain significant barriers in the introduction of effective therapies based on RAS inhibitors. This overview highlights the evolving nature of targeting RAS in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dda4fec514efa73d1390e9273305a191b377020" target='_blank'>
              The complex journey of targeting RAS in oncology
              </a>
            </td>
          <td>
            Katarzyna Wasiak, Damian Ciunowicz, Amelia Kierasińska-Kałka, Marta Węgierska, Marcin Pacholczyk, Piotr Rieske, Ewelina Stoczyńska-Fidelus
          </td>
          <td>2025-07-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="There is a lack of genetic biomarkers for predicting response to immune checkpoint blockade (ICB) therapy in cancer. The recent discovery that loss-of-function mutations in the gene encoding the protein phosphatase 2A (PP2A) scaffold protein PPP2R1A confer sensitivity to immune checkpoint blockade in ovarian clear cell carcinoma, therefore represents a breakthrough. Mechanistically, mutations in the PPP2R1A gene induce a strong interferon gamma response in tumor cells, which enhances infiltration of activated CD8+ T cells into the tumor. The activity of these T cells is then fortified by ICB. Furthermore, preclinical studies have shown that PP2A inhibition leads to the generation of neoantigens by disrupting RNA splicing, and PP2A inhibition can remodel the immune microenvironment of tumors to enhance responses to ICB. The finding that loss-of-function PPP2R1A mutations predict benefit from immunotherapy also suggests that pharmacological inhibition of PP2A may act synergistically with ICB therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8541b81fbcfabd62dc3e05a2a27e83fb1d85025" target='_blank'>
              A single gene mutation predicts response to immune checkpoint blockade in ovarian clear cell carcinoma.
              </a>
            </td>
          <td>
            M. H. Dias, R. Bernards
          </td>
          <td>2025-07-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are alternative DNA structures with diverse biological roles, but their examination in highly repetitive parts of the human genome has been hindered by the lack of reliable sequencing technologies. Recent long-read based genome assemblies have enabled their characterization in previously inaccessible parts of the human genome. Here, we examine the topography and genomic instability of potential G4-forming sequences in the gap-less, reference human genome assembly and in 88 haplotypes of diverse ancestry. We report that G4s are highly enriched in specific repetitive regions, including in certain centromeric and pericentromeric repeat types, and in ribosomal DNA arrays, and experimentally validate the most prevalent G4s detected. G4s tend to have lower methylation than expected throughout the human genome and are genomically unstable, showing an excess of all mutation types, including substitutions, insertions and deletions and most prominently structural variants. Finally, we show that G4s are consistently enriched at PRDM9 binding sites, a protein involved in meiotic recombination. Together, our findings establish G4s as dynamic and functionally significant elements of the human genome and highlight new avenues for investigating their contributions to human disease and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8869d55f4afe1d8deeba75e4afa7c23281aaeaa0" target='_blank'>
              Landscape and mutational dynamics of G-quadruplexes in the complete human genome and in haplotypes of diverse ancestry
              </a>
            </td>
          <td>
            Nikol Chantzi, Shi Wei. Liew, Aurell Wijaya, Candace S Y Chan, Ioannis Mouratidis, Emilyane de Oliveira Santana Amaral, Yasin Uzun, M. Hemberg, Karen M. Vasquez, C. Kwok, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bedd60872e779fcb4da0741c90e0eefb4d2f29f" target='_blank'>
              Unique molecular identifier-based single strand mitochondrial DNA sequencing for rare substitution mutations
              </a>
            </td>
          <td>
            Z. Li, Rachel Dunn, Loïc C. Caloren, Adam S. Ziada, Hailey Chapman, Izabelle Gadawska, H. Côté
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e75d3ecf904d9e7aa51e15582bd637748462a3" target='_blank'>
              Novel mutations of SRPX facilitate the stemness and malignant progression of glioma.
              </a>
            </td>
          <td>
            Siyuan Tang, Chunhui Qu, Mingyu Zhang, Peijun Zhou, Xingzhi Peng, Zhuan Zhou, Liangfang Shen, Lifang Yang
          </td>
          <td>2025-07-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Melanoma being one of the most common and deadliest skin cancers has been increasing since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays, the standard of care of advanced melanoma is resection, followed by immune checkpoint inhibition–based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistance. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long noncoding RNAs (lncRNA) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistance; however, systematic screens to uncover novel functional lncRNAs are scarce. In this study, we profile differentially expressed lncRNAs in patient-derived short-term metastatic cultures and BRAF–MEK inhibition–resistant cells. We conduct a focused growth-related CRISPR inhibition screen of overexpressed lncRNAs, validate, and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities, and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance in melanoma. Significance: LncRNAs have emerged as novel players in regulating many cellular aspects also in melanoma. The number of functional significances of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR inhibition growths screens. Our results broaden the characterized lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5174cc80205717609c47290b3fdd1147a088cc5c" target='_blank'>
              Uncovering Novel lncRNAs Linked to Melanoma Growth and Migration with CRISPR Inhibition Screening
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, P. Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Jeffrey L. Schloßhauer, A. Tsirigos, Iman Osman, Ioannis Aifantis, Jochen Imig
          </td>
          <td>2025-06-24</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The PIK3R1 gene encodes the class IA PI3K regulatory subunit p85α, which is frequently altered in cancer. PIK3R1 functions as a tumor suppressor by stabilizing and inhibiting the catalytic activity of p110, and it directly binds to and enhances the activity of the PTEN lipid phosphatase. Aberrations in the PIK3R1 gene are associated with poor prognosis in cancer; available data underscore the significant role of PIK3R1 mutations in mediating tumorigenesis by promoting the signaling of the PI3K/AKT/mTOR pathway. Moreover, copy number variations, driver mutations, and epigenetic alterations in PIK3R1 contribute to tumorigenesis and progression through distinct mechanisms. This article reviews the cancer-promoting effects of PIK3R1 gene aberrations across major cancer types and elucidates their underlying mechanisms. It also discusses the targeted therapies for related aberrations, aiming to provide a comprehensive understanding of the dynamic interplay of PIK3R1 in cancer, thereby advancing precision medicine and the development of targeted interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1f74c89d744eac6c65e6a4ebb2236a33792d6e" target='_blank'>
              Cancer-Associated PIK3R1 Genetic Aberrations and Precision Medicine
              </a>
            </td>
          <td>
            Huan Xie, Yirong Li, Xinran Li
          </td>
          <td>2025-06-12</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cancer cells manage to live longer and grow uncontrollably by protecting the ends of their chromosomes, known as telomeres. In healthy cells, telomeres naturally shorten over time, leading to aging and death of the cell. But in cancer, special mechanisms keep these telomeres from shrinking, allowing the cells to avoid death and continue dividing. This review explains how cancer cells achieve this and the relationship between telomere protection and the systems that repair damaged DNA. The study also explores new treatment approaches that aim to block these processes in cancer cells. These include drugs, immune therapies, and even natural substances that target the proteins or structures involved in telomere maintenance. While some treatments are still being tested in early trials, others have already shown promise in specific cancers. The authors also highlight the challenges of targeting telomeres without harming normal cells, and how telomere length and behavior can help doctors predict treatment success or detect cancer earlier. This work is important because it may lead to better, more personalized treatments and help fight cancer more effectively in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d33d40917a29f13f31b9de10ea636994f17fba8" target='_blank'>
              Telomere Maintenance and DNA Repair: A Bidirectional Relationship in Cancer Biology and Therapy
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Mikołaj Sędzik, Monika Kisielewska, Wiktoria Łuniewska, Kamil Sebastianka, Klaudia Molik, Katarzyna Skinderowicz, Jacek Kuźnicki, Joanna Tunikowska, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Exacerbating the DNA replication problems of cancer cells serves as a viable therapeutic approach. Nevertheless, the cytotoxicity of cancer drugs is often hampered by therapy-induced senescence, leading to unfavorable patient outcomes. Here, we employ acute replisome dysfunction in combination with Ataxia telangiectasia and Rad3-related (ATR) inhibition as a strategy to divert senescent cells toward death by triggering DNA replication catastrophe, a form of irreversible replication fork collapse caused by excessive single-stranded DNA (ssDNA) accumulation. RNA-sequencing revealed a distinct set of p53-responsive genes responsible for death. We identify c-Jun N-terminal kinase (JNK) to be essential for augmenting p53-dependent apoptotic programs and inducing pan-nuclear distribution of γH2AX, together constituting a feed-forward loop to drive cell death. Activation of DNA-PKcs initiates the signaling cascade of replication catastrophe, including CHK1-dependent JNK activation, which relies on MRE11 and PARP1 to expand and recognize ssDNA gaps, defining replication-associated gaps as an underlying basis for replication catastrophe. Our study elucidates the dynamic regulation of proximal and distal effectors along the DNA-PKcs/JNK/p53 axis that govern the cell fate decision between senescence and death. We propose that key determinants of replication catastrophe signaling are targetable vulnerabilities that can be exploited to limit senescent cell populations and increase the efficacy of anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c9be1cc99c822e2708983101c9fab63e3ac941" target='_blank'>
              The DNA-PKcs/JNK/p53 pathway underlies changes in cell fate decision toward death during DNA replication catastrophe
              </a>
            </td>
          <td>
            Jinal A. Patel, Julie Rageul, Natalie Lo, Auntara Nandi, Camryn Zezelic, Cynthia T Lee, Arafat Khan, Hyungjin Kim
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) exhibits exceptionally high levels of DNA methylation, with silencing of the DNA demethylating enzyme TET2 implicated in T-ALL’s hypermethylation phenotype. We propose that DNA hypomethylating agents (HMAs) could be particularly potent in T-ALL cells with this phenotype. Here, we used a reversible DNMT1-specific inhibitor and the conventional HMAs, 5-azacytidine and decitabine, to assess the effects of global DNA methylation loss in T-ALL cell lines and the potential of using HMAs as targeted therapeutic agents in T-ALL. We demonstrate that removal of DNA methylation, even in the absence of DNA damage, results in cell death and that toxicity is negatively correlated with methylation levels. Notably, whereas DNA demethylation caused limited transcriptional changes, key tumour suppressor genes, including TET2, were upregulated in a methylation-dependent manner. Few endogenous retroviruses or immune-related genes were reactivated after demethylation, challenging the contribution of ‘viral mimicry’ to HMA toxicity. Together, these findings provide fundamental insights into the role of DNA methylation in T-ALL, demonstrating that the removal of DNA methylation alone is sufficient to (i) induce cell death in T-ALL cell lines and (ii) reactivate silenced tumour suppressor genes. Our findings support the development of therapies targeting the unique methylation phenotype of T-ALL. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01915-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815502bed5a016e503e5f469d0f05d1f341e3aa5" target='_blank'>
              Exposing the DNA methylation-responsive compartment of the leukaemic genome in T-ALL cell lines support its potential as a novel therapeutic target in T-ALL
              </a>
            </td>
          <td>
            Maike Bensberg, Aida Selimović-Pašić, Lisa Haglund, Júlia Goldmann, Sandra Hellberg, Colm E. Nestor
          </td>
          <td>2025-07-03</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposons are mobile genetic elements capable of moving within the genome. Leveraging this property—particularly the cut-and-paste mechanism of DNA transposons—has enabled the development of technologies for inserting exogenous DNA fragments into host genomes. While targeted integration is a key goal for therapeutic applications, this review highlights the value of their intrinsic randomness. By combining the ability to freely design the DNA cargo with the stochastic nature of transposon integration, it becomes possible to generate highly sensitive reporter cells. These can be used to efficiently identify functional markers, uncover novel signaling pathways, and establish innovative platforms for drug screening. As more subfamilies of transposons become available for research use, their complementary biases may enhance the coverage and diversity of genome-wide screening approaches. Although inherently unpredictable, this strategy embraces randomness as a strength, and we propose that it holds great promise for driving new advances in biology, cellular engineering, and medical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d806cc8a8f4473c66fef98ec788d4bd5b59391" target='_blank'>
              Random Insertion Reporter Gimmicks Powered by Cut-and-Paste DNA Transposons
              </a>
            </td>
          <td>
            Yamato Kasahara, Kentaro Semba, Shinya Watanabe, K. Ishikawa
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a hematologic malignancy characterized by abnormal plasma cell proliferation, with lenalidomide emerging as a primary treatment. However, prolonged use often leads to drug resistance, underscoring the need to understand the resistance mechanisms. Protein post-translational modifications (PTMs) play crucial roles in disease development, including chemoresistance. Here, we investigate the involvement of new types of PTMs, focusing on lysine lactylation (Kla), in lenalidomide-resistance. Glycolysis-driven elevation of Kla levels was observed in lenalidomide-resistant MM cells, and the subsequent inhibition of glycolytic activity significantly reversed the lenalidomide-resistance phenotype. Through quantitative proteome, lactylome, and acetylome analyses, we identified 7493 proteins, 1241 Kla sites, and 9313 lysine acetylation (Kac) sites, thereby revealing differential protein expression and PTM profiles in lenalidomide-resistant cells. Proteomic analysis revealed that a series of chemoresistance-related proteins were upregulated, and a number of Cullin-RING Ligase 4-Cereblon (CRL4CRBN) regulatory factors were downregulated. Lactylome analysis revealed that numerous chemoresistance-related proteins exhibited increased Kla levels in lenalidomide-resistant MM cells, suggesting that Kla played an important role in the development of lenalidomide-resistance in LenR MM cells. Notably, histone H4K8la was associated with upregulation of chemoresistance-related genes cyclin-dependent kinase 6 (CDK6) and enoyl-CoA hydratase (ECHS1). Our findings shed light on the epigenetic mechanisms underlying lenalidomide-resistance in MM, offering insights for overcoming chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d724e5318db67692c1b88de82a846d5c939a37" target='_blank'>
              Quantitative Proteomics Reveals the Role of Lysine Lactylation in Lenalidomide-Resistance in Multiple Myeloma Cells.
              </a>
            </td>
          <td>
            Xinlong Guo, Xuelian Ren, Cong Yan, He Huang
          </td>
          <td>2025-07-01</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06f979ef3bcf83b0645cd826aee48f3828f2a1b5" target='_blank'>
              Integrating gene mutation spectra from tumors and the general population with gene expression topological networks to identify novel cancer driver genes.
              </a>
            </td>
          <td>
            Shuangyu Yang, Dan He, Ling Li, Zhiya Lu, Shaoying Li, Tianjun Lan, Feiyi Liu, Huasong Zhang, David N Cooper, Huiying Zhao
          </td>
          <td>2025-06-14</td>
          <td>Human genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Breast cancer (BC) is the most common cancer diagnosis in women globally, accounting for more than 1 in 10 new cancer diagnoses. While BC has historically been a poor candidate for immune checkpoint blockade (ICB), recent studies in triple-negative BC (TNBC) have demonstrated anti-PD-1 treatment efficacy when employed in conjunction with chemotherapy1. APOBEC3 (A3) enzymes, such as A3A and A3B, catalyze cytosine-to-uracil deamination in single-stranded DNA, leading to signature single base substitution mutations (SBS2 and SBS13). A3 expression and mutagenesis have been associated with poor clinical outcomes, including drug resistance2,3. Murine models for studying ICB and mutagenesis by human A3A and A3B are underdeveloped. Here, we address this gap by generating two independent mammary tumor cell lines and studying responsiveness to ICB. Two novel mammary cancer cell lines, MM001 and MM008, were isolated from spontaneously arising tumors in C57BL/6 MMTV-Cre+;p53fl/+ mice. Each tumor line was passaged serially in the mammary fat pads of C57BL/6 (WT) mice to obtain conditioned cell lines MM001i and MM008i, which formed consistent and reproducible tumors. Tumor immune infiltrates were investigated by immunohistochemistry for CD45 (pan-immune marker), CD3 (pan-T cell), CD8 (cytotoxic T cells), B220 (B cells), Mac-1 or CD11b (macrophages), and PD-L1. Cells were also assessed via flow cytometry for surface expression and interferon inducibility of immune markers PD-L1, MHC-I component H2-Kb, and CD155. Reproducibly growing tumors for each of MM001i and MM008i were achieved, with 50% of the tumors reaching the endpoint of 1000 mm3 around day 18 and day 21, respectively. Interestingly, despite a shared ancestral genetic background, ICB treatment of WT mice injected with MM001i and MM008i revealed differential responses. MM001i was poorly responsive, whereas MM008i showed complete tumor regression following treatment with anti-PD-1. In contrast, both MM001i and MM008i responded rapidly to anti-CTLA-4. These differential responses were not explained by surface expression of PD-L1 and MHC-1 component H2-Kb, which appeared to be basally and inducibly (IFN-β or IFN-γ) expressed at similar levels. Experiments are ongoing to deduce the underlying molecular mechanism, test additional ICB therapies, and study the impact of A3 mutagenesis. Thus, two novel murine mammary breast cancer cell lines have been established to evaluate immunotherapies in fully immune-competent C57BL/6 mice.
 Selected References:
 1) Schmid, P. et. al. Pembrolizumab for Early Triple-Negative Breast Cancer. N Engl J Med, 382, 810-821 (2020).
 2) Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv, 2, e1601737 (2016).
 3) Gupta, A. et. al. APOBEC3 mutagenesis drives therapy resistance in breast cancer. bioRxiv, https://doi.org/10.1101/2024.04.29.591453 (2024).
 Citation Format: Thomas Kalantzakos, Anusha Soni, Cameron Durfee, Chris Mullally, Benjamin Troness, Reuben S. Harris, Harshita B. Gupta. New Murine Mammary Tumor Cell Lines for Syngeneic Studies Including Immunotherapy [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-02-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627f15e34d39099a44c9ec865f3f46e6de37a2af" target='_blank'>
              Abstract P1-02-25: New Murine Mammary Tumor Cell Lines for Syngeneic Studies Including Immunotherapy
              </a>
            </td>
          <td>
            Thomas J Kalantzakos, Anusha Soni, Cameron Durfee, Chris Mullally, Benjamin Troness, R. Harris, Harshita B. Gupta
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="In recent years, due to increased exposure to ultraviolet rays, the incidence of melanoma has increased significantly. Despite the progress in early diagnosis and treatment methods, managing metastatic melanoma remains challenging. Resistance to programmed cell death (PCD) is a hallmark of various cancers, including melanoma. This resistance can affect cell growth, survival, metastasis, and resistance to therapies. In recent years, non-coding RNAs (ncRNAs) have been acknowledged as master regulators of various biological processes, including PCD. This regulatory network acts as a double-edged sword in melanoma progression, either promoting tumor survival by preventing PCD or acting as an anti-tumor agent by inducing it. Such diverse functions make ncRNAs invaluable as diagnostic and prognostic biomarkers and potential therapeutic targets. This review explores the multifaceted roles of ncRNAs in regulating different forms of PCD, comprising apoptosis, autophagy, necroptosis, pyroptosis, ferroptosis, cuproptosis, and anoikis, highlighting their potential applications in melanoma research and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053863720c0dc487c8d0088a7f483d115262c0aa" target='_blank'>
              The role of non-coding RNAs in the regulation of cell death pathways in melanoma
              </a>
            </td>
          <td>
            Yekta Metanat, Maria Sviridova, Bareq N Al-Nuaimi, Fateme Janbazi, Mahtab Jalali, Nogol Ghalamkarpour, Elnaz Khodabandehloo, Ehsan Ahmadi
          </td>
          <td>2025-06-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rare genetic DNA repair deficiency syndromes can lead to immunodeficiency, neurological disorders, and cancer. In the general population, inter-individual variation in DNA repair capacity (DRC) influences susceptibility to cancer and several age-related diseases. Genome wide association studies and functional analyses show that defects in multiple DNA repair pathways jointly increase disease risk, but previous technologies did not permit comprehensive analyses of DNA repair in populations. To overcome these limitations, we used fluorescence multiplex host cell reactivation (FM-HCR) assays that directly quantify DRC across six major DNA repair pathways. We assessed DRC in phytohemagglutinin-stimulated primary lymphocytes from 56 healthy individuals and validated assay reproducibility in 10 individuals with up to five independent blood draws. We furthermore developed generalized analytical pipelines for systematically adjusting for batch effects and both experimental and biological confounders. Our results reveal significant inter-individual variation in DRC for each of 10 reporter assays that measure the efficiency of distinct repair processes. Our data also demonstrate that correlations between the activities of different DNA repair pathways are relatively weak. This finding suggests that each pathway may independently influence susceptibility to the health effects of DNA damage. We furthermore developed a pipeline for analyzing comet repair kinetics and related our new functional data to previously reported comet assay data for the same individuals. Our pioneering analysis underscores the sensitivity of FM-HCR assays for detecting subtle biological differences between individuals and establishes standardized methodologies for population studies. Our findings and open source analytical tools advance precision medicine by enabling comprehensive exploration of genetic, demographic, clinical, and lifestyle factors and supporting targeted interventions to enhance DNA repair and maintain genomic integrity, thereby promoting personalized healthcare and disease prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db3f634648b2c3c7a12699cca388efeca2e26cd" target='_blank'>
              Comprehensive Measurement of Inter-Individual Variation in DNA Repair Capacity in Healthy Individuals
              </a>
            </td>
          <td>
            Ting Zhai, Patrizia Mazzucato, Catherine Ricciardi, David C. Christiani, Liming Liang, Leona D Samson, I. A. Chaim, Z. Nagel
          </td>
          <td>2025-06-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) affects genomic stability and has potential as a biomarker for the effectiveness of poly (ADP-ribose) polymerase (PARP) inhibitors and immune checkpoint inhibitors. However, the clinical and molecular profile of HRD in non-small cell lung cancer (NSCLC), particularly in the Chinese population, remains poorly characterized. Based on the next-generation sequencing data of 158 Chinese NSCLC patients, we analyzed the HRD scores of mutations in homologous recombination repair (HRR) genes and dissected the correlation between HRD state and programmed death-ligand 1 (PD-L1) expression. Alterations in HRR genes were observed in 8.9% of the patients, with ATM and BRCA2 being the most commonly affected genes. HRD-high (HRD-H) status was significantly associated with advanced disease stage (≥III) and lung squamous cell carcinoma (LUSC). Transcriptomic analysis revealed distinct gene expression profiles between HRD-H and HRD-low (HRD-L) subgroups, with HRD-H tumors exhibiting predominantly downregulated genes. While EGFR mutations occurred at similar frequencies across HRD status, TP53 mutations were significantly enriched in HRD-H cases. HRD-H status correlated with higher PD-L1 positivity in NSCLC overall, but not within the lung adenocarcinoma (LUAD) subgroup in our cohort. The Cancer genome atlas analysis showed higher PD-L1 protein expression in HRD-H LUAD, but not in LUSC. Kyoto Encyclopedia of Genes and Genomes analysis identified enrichment of complement and coagulation cascades, ABC transporters, and bile secretion pathways in HRD-H tumors, suggesting links to immune evasion and drug resistance. This study elucidates the genomic landscape of HRD in Chinese NSCLC patients and provides insights into its potential clinical utility for therapeutic targeting. Our findings suggest that integrated HRD scoring may guide the application of PARP inhibitors and immunotherapy in specific NSCLC patient subgroups. Further prospective clinical studies are needed to validate the predictive value of HRD scoring in NSCLC treatment and to optimize patient selection strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c403957972723c45a13a1636514a41ce8c5b8d3" target='_blank'>
              Genomic alterations of homologous recombination deficiency in Chinese NSCLC patients
              </a>
            </td>
          <td>
            Shuang Xiang, Changqiong Shen, Chun Huang, Ya-ting Yang, Jing Guo, Yi Liu, Mingzhu Yin, Song Duan
          </td>
          <td>2025-06-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Breast cancer is the second leading cause of cancer death in women. The primary cause of mortality is associated with the metastasis process of cancer cells to other organs. It has become obvious that epigenetic alteration plays a crucial role in breast cancer progression through the modulation of various signaling pathways. Additionally, the plastic nature of cancer cells is essential to their adaption in different metastatic sites. However, implementing epigenetic therapies in the clinic has been challenging due to the ubiquitous expression of many epigenetic factors. Here, we evaluate a library of epigenetic drugs to determine their impact on receptors commonly targeted in breast cancer. These include estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2). We coupled an immuno-fluorescence imaging approach to bioluminescence imaging to capture receptor heterogeneity and the proliferation index of each epigenetic drug. So far, we have assessed 460 drugs that target various factors, including Epigenetics, JAK/STAT, DNA Damage, Angiogenesis, Cell Cycle, PI3K/Akt/mTOR and Cytoskeletal Signaling targeting and histone deacetylases (HDACs), DNA methyltransferases (DNMTs), histone methyltransferases (HMTs), aurora kinase PARP, Sirtiin, EGFR, JAK, Plm pathways. We also analyzed the expression of key epigenetic markers (e.g., EZH2) and stemness-associated factors (e.g., SOX9). As a result, we found BIO, an inhibitor of GSK-3 and JAK, increases ER expression. Similarly, Curcumin, an inhibitor of p300 histone acetyl-transferase and Histone deacetylase (HDAC) promotes a broader range of factors, including Her2, Sox9, and ER. Interestingly, (+)-JQ1(BET bromodomain inhibitor), I-BRD9 (BRD9 inhibitor) and CUDC-101 (HDAC inhibitor) drastically reduced Ki67 level. Overall, these findings suggest a potential for combination therapy. While further investigation of these promising candidates is needed (3D, ex vivo, and in vivo testing), this investigation reveals new possibilities to overcome therapeutic resistance in breast cancer.
 Citation Format: Rahat Alam, Igor Bado. Epigenetic Drug Profiling for Breast Cancer Therapy: A Focus on Receptor Modulation and Heterogeneity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d889ed4ca5a9cd73bc3f7725a940c34fe41655cb" target='_blank'>
              Abstract P4-10-27: Epigenetic Drug Profiling for Breast Cancer Therapy: A Focus on Receptor Modulation and Heterogeneity
              </a>
            </td>
          <td>
            Rahat Alam, Igor L Bado
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer genomics has advanced a lot; a main problem is still turning molecular findings into treatments due to the variety of cells in each tumor and gaps in knowing certain pathway functions. Predicting whether a treatment will be effective is difficult right now which means that patient outcomes are often less than ideal. This research tackled the issue by studying whether looking at many genes and oncogenic pathways at the same time helps identify which patients may show cancer progression or treatment resistance. For this study, we looked at the genes, proteins and other molecules from a total of 1,500 patients in six major types of cancer (breast, colorectal, lung, melanoma, ovarian and prostate). We used genomic variant calling, analysis of gene pathways and detailed study of proteins with advanced bioinformatics and machine learning tools. The study found that TP53 mutations (OR: 2.14, 95% CI: 1.72-2.66, p < 0.001) and the PI3K/AKT/mTOR pathway being activated (OR: 1.89, 95% CI: 1.51-2.37, p = 0.003) were strongly linked to cancer not responding to treatment and negative outcomes. Tumors with a high number of mutations (≥ 10/Mb) reacted much better to immunotherapy (OR: 3.02, p < 0.001). Three subgroups of tumors were found through unsupervised analysis and these showed different chances of success: tumors with many mutations and high immune activity performed best (39.1% complete response), but those with mutations in PI3K and TP53 had the worst prognosis (28.5% progressive disease). Patients whose tumors were active in the PI3K/AKT pathway had progression-free survival of 8.2 months, significantly less (log-rank p < 0.001) than those whose tumors were not active. Evidence shows that integrative molecular profiling is better at predicting outcomes than the traditional method of classifying tumors by looking at slides. Study results show which molecular subgroups are most important to oncology and help link their treatment response to which therapy is best,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e766178e1eebb59cae34bf9ab40a3c9e125bd68f" target='_blank'>
              Integrative Molecular Profiling of Oncogenic Pathways and Genetic Mutations in Cancer Progression and Therapeutic Response
              </a>
            </td>
          <td>
            Javeria Taj, Muhammad Noman Ajmal, Tayyaba Arshad, Afifa Dawood, Ali Abbas, Misbah Hafeez, Hamza Rafeeq, Abdul Malik, Aiman Nishat, Mujahid Hussain
          </td>
          <td>2025-06-20</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0e7e32e3545d727024253e68ada05454a1fe358" target='_blank'>
              The role of ASXL1, SRSF2, and EZH2 mutations in chromatin dysregulation of myelodysplastic neoplasia and acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hosang Yu, Junshik Hong, Dong-Yeop Shin, Chul-Hwan Lee
          </td>
          <td>2025-06-25</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor-derived biomarkers can precisely reflect cell types, offering major potential for early detection and therapy. An engineered platform capable of distinguishing various biomarkers for cancer type identification and autonomous responsive therapeutics is urgently needed. Herein, a "one stop" production line with logic gated composite for tumor cell identification, related biomarker imaging, and subsequent gene therapy is proposed. The designed composite can specifically target cancer cells through incorporated aptamers, which endows it with identification preference for cancer cells with overexpressed membrane protein receptors. Internalized materials collapse in the acidic lysosome milieu, releasing the functioning DNA strands. Endogenous miR-210 then can initiate cascade reactions with released strands, resulting in the liberation of therapeutic antisense oligonucleotides (ASO). The strategy is to construct a platform for cancer therapy based on second-order AND Boolean logic operation for multiple cancer biomarkers (Mucin 1 (MUC1), Protein tyrosine kinase 7 (PTK7), and miR-210) identification and succeeding therapy toward identified cells. Through the logic operation, produced fluorescent signals can disclose the types of biomarkers being recognized, while nano-material endocytosis, subsequent miRNA imaging, and endogenous miRNA-triggered gene therapy can all be performed concurrently. This "one stop" multifunctional production line demonstrates excellent sensitivity and accuracy, and presents promising prospects in the early diagnosis of tumors and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2e7e35eedc8f5095ca22e7d62f436db09d918b" target='_blank'>
              Logic-Gated DNA Production Line for Cancer Cell Recognition and Intracellular Cascaded Gene Therapy.
              </a>
            </td>
          <td>
            Hui Xin, Yan Zhong, Bo-lun Li, Chun-yang Wang
          </td>
          <td>2025-06-25</td>
          <td>Small</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To systematically investigate the molecular and pathological mechanisms of enhancer RNA (eRNA)–mediated transcriptional regulation in glioma recurrence and progression, transcriptomic, regulatome, and genomic data were integrated to analyze eRNA behavior in lower-grade gliomas (stages II/III) and glioblastomas (stage IV). Most eRNAs exhibited dynamic expression during glioma progression, regulated by master transcription factors (TFs) and affected by genomic mutations. The constructed perturbed TF-mediated eRNA-promoter regulatory landscape revealed that rewiring eRNA-promoter networks contributed to glioma malignancy. Drug response–related eRNAs associated with poor prognosis were identified, highlighting their clinical potential. Overall, integrative analysis highlights the critical role of eRNA-mediated regulatory rewiring in glioma progression, providing valuable insights into transcriptional mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b9a681aad2033d6a1c3522f91909b48b06e614" target='_blank'>
              Enhancer RNA–mediated transcriptional regulatory programs reveal the malignant progression of glioma
              </a>
            </td>
          <td>
            Yingying Ma, Shuyuan Wang, Yijie Jia, Shijie Wang, Wendian Liu, Jiaxin Yu, Jun Wang, Jingkai Geng, Jiayi Zhao, Zheng Zhao, Juan Xu, Hui Liu
          </td>
          <td>2025-06-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Therapeutic gene editing strategies utilize endogenous DNA repair pathways—nonhomologous end joining (NHEJ) or homology-directed repair (HDR)—to introduce targeted genomic modifications. Because HDR is restricted to dividing cells, whereas NHEJ functions in both dividing and non-dividing cells, NHEJ-based approaches are better suited for in vivo gene editing in the largely post-mitotic airway epithelium. Homology-independent targeted insertion (HITI), an NHEJ-based method, offers a promising strategy for cystic fibrosis (CF) gene therapy. Here, we applied HITI to drive the expression of a promoterless reporter through an exon trap strategy in both proliferating airway basal cells and well-differentiated primary airway epithelial cultures derived from transgenic ROSAmTmG ferrets. We also established a versatile human gene editing reporter (GER) airway basal cell line capable of multipotent differentiation, enabling real-time visualization of editing outcomes and the quantitative assessment of HDR- and NHEJ-based editing efficiencies. Together, these platforms provide easily accessible tools for optimizing genome editing strategies in the respiratory epithelium and advancing clinically relevant delivery strategies for CF gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45657c253d56071b296fc91d8d35ac630ce7585c" target='_blank'>
              A Versatile Reporter Platform for Evaluating HDR- and NHEJ-Based Genome Editing in Airway Epithelial Cell Cultures Using an rAAV Vector
              </a>
            </td>
          <td>
            S. Y. Park, Zehua Feng, Xiujuan Zhang, Yinghua Tang, Donovan Richart, Kai E. Vorhies, Jianming Qiu, John F. Engelhardt, Ziying Yan
          </td>
          <td>2025-06-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction and Objective: The global rise in overweight and obesity is linked to the initiation of cancer and poorer survival outcomes in triple-negative breast cancer (TNBC), making it a modifiable risk factor for both cancer prevention and treatment. The tumor microenvironment (TME) plays a critical role in tumor development, progression, and response to chemotherapy. The mammary gland consists of lobules and ducts embedded in a connective stroma containing fibro-adipose pockets. In obese patients, excessive fat deposition alters adipose tissue quantity, function, and plasticity. Adipocyte precursor cells (APCs) regulate adipose tissue plasticity and can differentiate into either adipogenic or fibrotic cell types.Based on the above, We hypothesize that obesity promotes APC differentiation into fibrotic cells, accelerating cancer progression.
 Methods: Single-cell multiomics (ATAC+RNA sequencing) has emerged as a powerful tool for studying the TME, enabling the simultaneous analysis of epigenetic and genetic profiles from the same tumor sample. We performed single-cell multiomics sequencing on primary TNBC samples from 7 lean and 7 obese patients, stratified by body mass index (BMI).
 Results: Our results showed that fibroblast populations were more abundant in obese patients, with increased interactions with cancer cells. The dominant fibroblast subtype in the obese group was identified as myofibroblasts (mCAFs), which were enriched in the TGFβ signaling pathway and correlated with epithelial-mesenchymal transition (EMT) signatures in cancer cells. Furthermore, comparison with a published human obese white adipocyte tissue atlas revealed that mCAFs shared significant transcriptomic similarity with APCs.
 Conclusion: These findings suggest that obesity promotes APC differentiation into mCAFs via the TGFβ pathway, contributing to cancer progression. This offers a novel direction for targeted therapies in obese patients.


 X. Xu: None. L. Yang: None. S. Bahl: None. M. Lupien: None.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f950d77bced847cec63715c87b6f7d94acef36ea" target='_blank'>
              1781-P: Obesity Promotes Triple-Negative Breast Cancer Progression through Regulating the Plasticity of Adipose Tissue
              </a>
            </td>
          <td>
            Xi Xu, Lin Yang, S. Bahl, Mathieu Lupien
          </td>
          <td>2025-06-13</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Hormone Receptor Positive (HR+) breast cancer is predominantly managed with endocrine therapies that target the Estrogen Receptor Alpha (ERα), a key mediator of estrogen's role in promoting tumor growth. However, the emergence of endocrine resistance poses a significant clinical challenge, limiting the efficacy of these treatments and impeding optimal patient outcomes. Understanding the mechanisms behind ERα's importance and the development of resistance is crucial for advancing therapeutic strategies against HR+ breast cancer. Methods: In our study, we utilized the GEO database to identify ADP-Ribosylation Factor-Like GTPase 3 (ARL3) as a gene associated with tamoxifen resistance in HR+ breast cancer. We constructed cell lines with ARL3 knockout and overexpression to assess the impact of ARL3 on cell proliferation, migration, and tamoxifen sensitivity. RNA sequencing (RNA-seq) and liquid chromatography-mass spectrometry (LC/MS) were employed to demonstrate the influence of ARL3 knockout on downstream ERα pathways. Furthermore, co-immunoprecipitation (CO-IP), immunofluorescence and Western blot (WB) assays confirmed the role of ARL3 in the ubiquitination and degradation pathway of ERα. Results: In our analysis of TCGA breast cancer subtypes, we also observed that ARL3 is highly expressed in luminal breast cancer. Additionally, our collection of breast cancer tissue mRNA and immunohistochemistry data revealed that ARL3 expression is elevated in HR+ breast cancer, indicating a correlation with estrogen receptors. We found that ARL3 promotes the proliferation and migration of HR+ breast cancer. Furthermore, overexpression of ARL3 was associated with reduced sensitivity to tamoxifen, and in vivo tumorigenesis in nude mice showed that tumor volume and weight were significantly decreased compared to the control group. To investigate the underlying mechanisms, RNA-seq analysis of ARL3 knockout and control groups revealed a pronounced inhibition of the MYC pathway. Western blot (WB) validation confirmed that ARL3 knockout suppresses the expression of downstream pathways of ERα. Our data suggest that the association between ARL3 and ERα primarily occurs in the cytoplasm, mainly binding to the ligand-binding domain (LBD) of ERα, which is consistent with ARL3 not being recruited to the promoters occupied by ERα. We also discovered that the ARL3/ERα cascade promotes mitochondrial autophagy to enhance mitochondrial oxidative phosphorylation. In summary, our study establishes a non-genomic mechanism whereby ARL3 controls the transcription of estrogen-dependent genes associated with breast cancer cell proliferation by stabilizing ERα levels. Conclusion: In our study, we posit that ADP-ribosylation factor-like GTPase 3 (ARL3) exerts a pivotal influence on the proliferative capacity, metastatic potential, and responsiveness to endocrine therapies in hormone receptor-positive (HR+) breast cancer. ARL3 modulates the stability of estrogen receptor alpha (ERα), thereby regulating the activation of downstream signaling cascades and the mitochondrial functionality within neoplastic cells. Our findings suggest that ARL3 represents a promising therapeutic target for HR+ breast cancer. We envision that in-depth investigation into the molecular underpinnings of ARL3's role could pave the way for the development of innovative pharmacological agents capable of overcoming endocrine resistance in patients, ultimately enhancing their clinical outcomes and survival rates.
 Citation Format: Han Li, Yang Liu, Zehao Cai, Dan Shu, Yang Peng, Kang Li, Shengchun Liu. ARL3 Promotes Hormone Receptor Positive Breast Cancer Progression and Tamoxifen Resistance through ERα Stablization [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-04-10.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aec8ed2ad5da8e6d551e26dd1027baa5c75f898" target='_blank'>
              Abstract P2-04-10: ARL3 Promotes Hormone Receptor Positive Breast Cancer Progression and Tamoxifen Resistance through ERα Stablization
              </a>
            </td>
          <td>
            Han Li, Yang Liu, Zehao Cai, Dan-hua Shu, Yang Peng, Kang Li, Shengchun Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="INTRODUCTION
Rearranged during transfection (RET) is a receptor tyrosine kinase essential for cell signaling processes such as proliferation, differentiation, and migration. RET alterations, including point mutations and gene fusions, contribute to oncogenesis. Recent advances in precision oncology have led to the development of selective RET inhibitors with improved safety and efficacy profiles.


AREAS COVERED
This review discusses recent innovations in RET inhibitor development, with a focus on small-molecule patents published from 2022 to 2024. A comprehensive literature search was conducted using SciFinder and PatentScope with the keywords 'RET' and 'RET inhibitor.' The review summarizes the structural classes, target profiles, and potential advantages of newly patented compounds, particularly those designed to overcome known resistance mechanisms.


EXPERT OPINION
While selective RET inhibitors such as selpercatinib have shown clinical success and expanded indications, their use is limited by adverse events and emerging resistance mechanisms. Next-generation RET inhibitors that address these limitations represent a critical frontier in drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5802a6edcf57500f4589eee6d01ab1a1541cf4a1" target='_blank'>
              An updated patent review of rearranged during transfection (RET) kinase inhibitors (2022-present).
              </a>
            </td>
          <td>
            Daniel Armstrong, Maha Hanafi, Brendan Frett
          </td>
          <td>2025-06-12</td>
          <td>Expert opinion on therapeutic patents</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While somatic variants are well-characterized drivers of tumor evolution, their influence on cellular fitness in nonmalignant contexts remains understudied. We identified a mosaic synonymous variant (m.7076A > G) in the mitochondrial DNA (mtDNA)-encoded cytochrome c-oxidase subunit 1 (MT-CO1, p.Gly391=), present at homoplasmy in 47% of immune cells from a healthy donor. Single-cell multiomics revealed strong, lineage-specific selection against the m.7076G allele in CD8+ effector memory T cells, but not other T cell subsets, mirroring patterns of purifying selection of pathogenic mtDNA alleles. The limited anticodon diversity of mitochondrial tRNAs forces m.7076G translation to rely on wobble pairing, unlike the Watson-Crick-Franklin pairing used for m.7076A. Mitochondrial ribosome profiling confirmed stalled translation of the m.7076G allele. Functional analyses demonstrated that the elevated translational and metabolic demands of short-lived effector T cells (SLECs) amplify dependence on MT-CO1, driving this selective pressure. These findings suggest that synonymous variants can alter codon syntax, impacting mitochondrial physiology in a cell type-specific manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77e5c9949a4e7b360acc964029cfc77cf08593d" target='_blank'>
              Cell type-specific purifying selection of synonymous mitochondrial DNA variation.
              </a>
            </td>
          <td>
            Caleb A. Lareau, Patrick Maschmeyer, Yajie Yin, Jacob C. Gutierrez, R. Dhindsa, Anne-Sophie Gribling-Burrer, Sebastian Zielinski, Yu-Hsin Hsieh, Lena Nitsch, Veronika Dimitrova, Benan N. Nalbant, Frank A Buquicchio, Tsion Abay, Robert R. Stickels, Jacob C. Ulirsch, Patrick K. Yan, Fangyi Wang, Zhuang Miao, Katalin D. Sandor, Bence Daniel, Vincent Liu, Paul-Lennard Mendez, Petra Knaus, Manpreet Meyer, William J. Greenleaf, Anshul Kundaje, R. Smyth, Mathias Munschauer, Leif S. Ludwig, Ansuman T. Satpathy
          </td>
          <td>2025-07-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved by specialized helicases. Previous in vitro studies showed that DEAH-box helicase 36 (DHX36/G4R1/RHAU) has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in the accumulation of cytoplasmic DNA fragments, an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression, and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 messenger RNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd74a217dd008b29b89824a440aee3aba255159" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, Shangming Tang, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Yuh-Hwa Wang
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Resveratrol (RSV) is one of the most studied and used biomolecules, for which many pharmacological effects targeting multiple tissues have been described. However, a common underlying mechanism driving its full pharmacological activity has not been described in detail. G-quadruplexes (G4s) are non-canonical nucleic acid structures found in regulatory genomic locations and involved in controlling gene transcription, telomere maintenance, or genome stability, among others. This study provides a genome-wide characterization of RSV G4-binding properties, explaining its multi-target traits. Methods Immunofluorescence assays using a nucleolar and a G4-specific antibody were used to characterize RSV cellular effects on the nucleolus and G4 stabilization. DNA damage and cell cycle analyses were performed via western blot and flow cytometry. Breaks lLbeling In Situ and Sequencing (BLISS) was used to map double strand breaks (DSB) in response to treatment, and identify G4s targeted by RSV. mRNA sequencing was used to identify changes at the transcriptional level upon treatment and relate them to a direct targeting of G4s. Biophysical assays (circular dichroism, ultraviolet–visible [UV–Vis] titration, differential scanning calorimetry, and nuclear magnetic resonance) were used to characterize RSV–G4 interactions. Lastly, luciferase-based transcription assays were performed to confirm RSV–G4 interaction in vitro and its direct influence on gene expression. Results In a cellular context, RSV treatment showed classic G4-ligand effects, such as nucleolar disassembly, inhibition of RNA polymerase I, DNA damage, and cell cycle arrest. RSV was shown to stabilize cellular G4s, which accumulated around double strand breaks in the promoters of differentially expressed genes. Upon treatment, G4 stabilization triggered DNA damage and controlled gene expression. The interaction between RSV and target G4s was confirmed in vitro by biophysical assays and through luciferase-based transcription assays. Conclusions A G4-dependent mode of action was demonstrated as the main mechanism underlying RSV pleiotropic effects, along with the identification of target genes and G4s. This in-depth analysis of the mode of action of RSV will be helpful to improve its therapeutic potential in a wide variety of health scenarios. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00747-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b9a5bcda358645827e337bc4418daea949d2f" target='_blank'>
              In-depth analysis of the mode of action of resveratrol: genome-wide characterization of G-quadruplex binding properties
              </a>
            </td>
          <td>
            Ana Soriano-Lerma, Victoria Sánchez-Martín, Javier Murciano-Calles, Matilde Ortiz-González, Maria J. Tello-Lopez, Virginia Pérez-Carrasco, Ángel Linde-Rodríguez, Inmaculada Ramírez-Macías, Irene Gómez-Pìnto, Inmaculada López-Aliaga, M. Soriano, Jose A. Garcia Salcedo
          </td>
          <td>2025-06-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Gleason score (GS) is one of the best predictors of prostate cancer aggressiveness. GS classifies cancer cells based on the histological patterns of prostate tissue sections and does not evaluate the nuclear grade or proliferation of the cancer cells. A small number of studies focused on the association between gene expression signatures and GS using an in silico translational approach. GS is associated with cell cycle-related genes, changes in DNA repair genes, stromal-related genes, immune-related genes, cuprotosis-related genes, and several specific genes. To our knowledge, this is the first report showing that GS was positively correlated not only with homologous recombination deficiency mutations but also with intratumor heterogeneity, fractional mutations, single nucleotide variant neoantigen, silent mutation rate, and non-silent mutation rate. Our findings emphasize that GS reflects not only morphological abnormalities but also differences in cancer cell proliferation, immune cell infiltration, and high mutation rates, which may affect prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd14eb5db59e8928cb0b1b4ccebf042c1ae25bde" target='_blank'>
              Unique Biological Characteristics of Patients with High Gleason Score and Localized/Locally Advanced Prostate Cancer Using an In Silico Translational Approach
              </a>
            </td>
          <td>
            Shiori Miyachi, M. Oshi, Takeshi Sasaki, I. Endo, K. Makiyama, Takahiro Inoue
          </td>
          <td>2025-07-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="RNA-binding proteins (RBPs) play a fundamental role in cellular metabolism, with their disturbance leading to large-scale transcriptomic dysregulation. RBP dysregulation is highly prevalent in human cancers; however, its role in lung adenocarcinoma (LUAD) has not been systematically investigated. To establish a more effective and robust risk model, a machine learning integration program was used to screen hub prognostic RBPs. Our risk model C-index performed extremely well among 103 published signatures. The high-risk group had a lower immune score and worse immunotherapy effects. As one of the members of the RNA helicase family, DDX56 can interact with certain transcription factors, thereby regulating the expression of its downstream targets. DDX56 exerts an anti-apoptotic effect and reduces the sensitivity to carboplatin treatment by promoting Bcl-2 transcription in LUAD cells. Additionally, DDX56 activates NF-kB signaling pathways, which may be related to DDX56-mediated promotion of Bcl-2 transcription, proliferation, migration, and invasion in LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fde85cff32f22eff04840eb926ae6c38bea8809" target='_blank'>
              Identifying the crucial oncogenic mechanisms of DDX56 based on a machine learning-based integration model of RNA-binding proteins
              </a>
            </td>
          <td>
            Hui Jiang, hao zheng, Xinjie Zhao, Yunzhi Xiang, Jiahao Li, Kai Wang, Guanghui Wang, Jiajun Du
          </td>
          <td>2025-07-17</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Anthracyclines have been clinically well established in cancer chemotherapy for decades. The main limitations of this drug class are the development of resistance and severe side effects. In the present investigation, we analyzed 30 anthracyclines in a panel of 59 cell lines of the National Cancer Institute, USA. The log10IC50 values varied from −10.49 M (3′-deamino-3′-(4″-(3″-cyano)morpholinyl)-doxorubicin, 1) to −4.93 M (N,N-dibenzyldaunorubicin hydrochloride, 30). Multidrug-resistant NCI-ADR-Res ovarian cancer cells revealed a high degree of resistance to established anthracyclines (between 18-fold to idarubicin (4) and 166-fold to doxorubicin (13) compared to parental, drug-sensitive OVCAR8 cells). The resistant cells displayed only low degrees of resistance (1- to 5-fold) to four other anthracyclines (7, 18, 28, 30) and were even hypersensitive (collaterally sensitive) to two compounds (1, 26). Live cell time-lapse microscopy proved the cross-resistance of the three chosen anthracyclines (4, 7, 9) on sensitive CCRF/CEM and multidrug-resistant CEM/ADR5000 cells. Structure–activity relationships showed that the presence of tertiary amino functions is helpful in avoiding resistance, while primary amines rather increased resistance development. An α-aminonitrile function as in compound 1 was favorable. Investigating the mRNA expression of 49 ATP-binding cassette (ABC) transporter genes showed that ABCB1/MDR1 encoding P-glycoprotein was the most important one for acquired and inherent resistance to anthracyclines. Molecular docking demonstrated that all anthracyclines bound to the same binding domain at the inner efflux channel side of P-glycoprotein with high binding affinities. Kaplan–Meier statistics of RNA sequencing data of more than 8000 tumor biopsies of TCGA database revealed that out of 23 tumor entities high ABCB1 expression was significantly correlated with worse survival times for acute myeloid leukemia, multiple myeloma, and hepatocellular carcinoma patients. This indicates that ABCB1 may serve as a prognostic marker in anthracycline-based chemotherapy regimens in these tumor types and a target for the development of novel anthracycline derivatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0509a300730a6303969f2ff8af79884f58309d0" target='_blank'>
              Predictive and Prognostic Relevance of ABC Transporters for Resistance to Anthracycline Derivatives
              </a>
            </td>
          <td>
            Rümeysa Yücer, Rossana Piccinno, Ednah Ooko, M. Dawood, Gerhard Bringmann, T. Efferth
          </td>
          <td>2025-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transposable elements (TEs) have an important role in genome evolution but are challenging for bioinformatics detection due to their repetitive nature and ability to move and replicate within genomes. New sequencing technologies now enable the characterization of nucleotide and structural variations within species. Among them, TE polymorphism is critical to identify as it may influence species adaptation or trigger diseases. Despite the development of numerous bioinformatic programs, identifying the most effective tool is challenging due to non-overlapping results and varying efficiency across studies. Benchmarking efforts have highlighted some of the limitations of these tools, often evaluated on either real or simulated data. However, real data may be incomplete or contain unannotated TEs, while simulated data may not accurately reflect real genomes. This study introduces a simulation method generating data based on real genomes to control all genomic parameters. Evaluating several TE polymorphic detection tools using data from Drosophila melanogaster and Arabidopsis thaliana, our study investigates factors like copy size, sequence divergence, and GC content that influence detection efficiency. Our results indicate that only a few programs perform satisfactorily and that all are sensitive to TE and genomic characteristics that may differ according to the species considered. Using Bos taurus population data as a case study to identify polymorphic LTR-retrotransposon insertions, we found low-frequency insertions particularly challenging to detect due to a high number of false positives. Increased sequencing coverage improved sensitivity but reduced precision. Our work underscores the importance of selecting appropriate tools and thresholds according to the specific research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/921439c88209ade5b5fb5afef1e030676c1b9102" target='_blank'>
              None
              </a>
            </td>
          <td>
            Marie Verneret, Van Anthony Le, Thomas Faraut, Jocelyn Turpin, Emmanuelle Lerat
          </td>
          <td>2025-06-11</td>
          <td>Peer Community Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA is inherently unstable and is susceptible to damage from both endogenous sources (such as reactive oxygen species) and exogenous factors (including UV, ionizing radiation, and chemicals). The accumulation of DNA damage manifests as genetic mutations, chromosomal instability, and the stalling of DNA replication and transcription processes. Accumulated DNA damage influences apoptosis and cell cycle checkpoints, serving as one of the key triggers for the manifestation of the senescent phenotype. Both aging and cancer are associated with the accumulation of mutations in somatic cells. Disruption of cell cycle control and uncontrolled proliferation are fundamental characteristics of any cancer cell, with the majority of anticancer drugs acting as inhibitors of cyclin-dependent kinases, thereby inducing a transition of cells into a senescent state. Consequently, disturbances in the dynamics and regulation of inflammatory responses, oxidative stress, cell proliferation, DNA damage repair, and epigenetic anomalies, along with the influence of retroviruses and transposons, lead to the accumulation of senescent cells within the human body, characterized by blocked replication and cell cycle, as well as a distinct secretory phenotype. The age-related or disease-associated accumulation of these senescent cells significantly alters the physiology of tissues and the organism as a whole. Many secondary metabolites of higher plants exhibit senolytic and senomorphic activities, although most of them are not fully characterized. In this review, we will explore the principal signaling pathways in mammalian cells that govern the cell cycle and cellular senescence, with a particular emphasis on how their dynamics, expression, and regulation have been modified through the application of senotherapeutic compounds. The second section of the review will identify key target genes for the metabolic engineering, primarily aimed at enhancing the accumulation of plant secondary metabolites with potential therapeutic benefits. Lastly, we will discuss the rationale for utilizing liver cells as a model system to investigate the effects of senolytic compounds on human physiology and health, as well as how senotherapeutic substances can be leveraged to improve gene therapy approaches based on CRISPR/Cas9 and prime-editing technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9598e54ea3c1cd68d00ed2151290086010037de" target='_blank'>
              Hepatocytes as Model for Investigating Natural Senotherapeutic Compounds and Their Effects on Cell Cycle Dynamics and Genome Stability
              </a>
            </td>
          <td>
            A. Fizikova, Anna Prokhorova, Daria Churikova, Zahar Konstantinov, Roman Ivanov, A. Karabelsky, Stanislav Rybtsov
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignant tumor and a leading cause of cancer-related mortality among women worldwide. The disease comprises distinct molecular subtypes that influence tumor behavior, metastatic potential, and therapeutic response. This review presents recent advances in breast cancer research, with a particular emphasis on molecular and epigenetic mechanisms that contribute to tumor development, progression, and treatment resistance. Key signaling pathways—including estrogen receptor (ER), human epidermal growth factor receptor 2 (HER2), Notch, fibroblast growth factor receptor (FGFR), Wnt, and Hedgehog—play essential roles in the regulation of mammary stem cells and oncogenesis. Increasing evidence highlights the significance of epigenetic alterations such as DNA methylation, histone modifications, and microRNA expression in modulating gene activity relevant to tumor initiation and therapy resistance. Epigenetic molecular targets, including DNA methyltransferases (DNMTs), histone deacetylases (HDACs), EZH2, and non-coding RNAs, are gaining attention for their potential use in diagnosis, prognosis, and targeted therapy. Integration of multigene panel testing with epigenetic biomarkers has facilitated improved risk assessment and the development of individualized treatment strategies. Moreover, novel therapeutic approaches—such as CAR-T cell therapy, nanoparticle-mediated drug delivery systems, and the involvement of circular RNAs (circRNAs) in immune modulation—offer promising directions for precision medicine. This review consolidates current insights into the molecular and epigenetic landscape of breast cancer to provide a comprehensive understanding of disease complexity and to inform the development of more effective, personalized treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8400e20214b339b7e7810249446b370e6720099d" target='_blank'>
              Recent Insights Into Breast Cancer: Molecular Pathways, Epigenetic Regulation, and Emerging Targeted Therapies
              </a>
            </td>
          <td>
            Quratulain Maqsood, Muhammad Umer Khan, Tehreem Fatima, Sania Khalid, Zaryab Ikram Malik
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer : Basic and Clinical Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7030c5dd461e9cf38738c39264263c35a95a9a9" target='_blank'>
              Structural variation in 1,019 diverse humans based on long-read sequencing.
              </a>
            </td>
          <td>
            Siegfried Schloissnig, Samarendra Pani, Jana Ebler, C. Hain, Vasiliki Tsapalou, Arda Söylev, Patrick Hüther, Hufsah Ashraf, Timofey Prodanov, Mila Asparuhova, Hugo Magalhães, Wolfram Höps, Jesus Emiliano Sotelo-Fonseca, Tomas Fitzgerald, Walter Santana-Garcia, Ricardo Moreira-Pinhal, Sarah E. Hunt, Francy J. Pérez-Llanos, T. E. Wollenweber, S. Sivalingam, Dagmar Wieczorek, Mario Cáceres, Christian Gilissen, E. Birney, Zhihao Ding, Jan-Nygaard Jensen, Nikhil Podduturi, J. Stutzki, Bernardo Rodriguez-Martin, Tobias Rausch, T. Marschall, Jan O. Korbel
          </td>
          <td>2025-07-23</td>
          <td>Nature</td>
          <td>1</td>
          <td>130</td>
        </tr>

        <tr id="The kinesin family member 18 A (KIF18A) is an essential regulator of microtubule dynamics and chromosome alignment during mitosis. Functional dependency on KIF18A varies by cell type and genetic context but the heritable factors that influence this dependency remain unknown. To address this, we took advantage of the variable penetrance observed in different mouse strain backgrounds to screen for loci that modulate germ cell depletion in the absence of KIF18A. We found a significant association at a Chr5 locus where anaphase promoting complex subunits 5 (Anapc5) and 7 (Anapc7) were the top candidate genes. We found that both genes were differentially expressed in a sensitive strain background when compared to resistant strain background at key timepoints in gonadal development. We also identified a novel retroviral insertion in Anapc7 that may in part explain the observed expression differences. In cell line models, we found that depletion of KIF18A induced mitotic arrest, which was partially rescued by co-depletion of ANAPC7 (APC7) and exacerbated by co-depletion of ANAPC5 (APC5). These findings suggest that differential expression and activity of Anapc5 and Anapc7 may influence sensitivity to KIF18A depletion in germ cells and CIN cells, with potential implications for optimizing antineoplastic therapies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08766-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/965c0dbbb6128d27bea8370bf402a688e4926421" target='_blank'>
              Anapc5 and Anapc7 as genetic modifiers of KIF18A function in fertility and mitotic progression
              </a>
            </td>
          <td>
            Carleigh Nesbit, Whitney Martin, A. Czechanski, Candice Byers, Narayanan Raghupathy, Ardian Ferraj, Jason K. Stumpff, L. Reinholdt
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The occurrence and development of malignant tumors involve abnormalities in complex molecular regulatory networks, among which the abnormal activation of the transcriptional regulator hairy and enhancer of split 1 (Hes1) has attracted significant attention in recent years and is closely associated with prognosis in various malignancies. Hes1 exhibits high expression in various solid tumors and hematological malignancies, where it participates in alterations involving diverse immune cells, inflammatory factors, and the immune microenvironment, thereby promoting tumor cell proliferation, invasion, metastasis, and resistance to treatment. Recent studies have widely investigated the potential of targeting Hes1 and inhibiting its expression as a cancer therapeutic strategy, although its precise mechanisms of action are not yet fully elucidated. Hes1 interacts with critical pathways including Notch, JAK/STAT, PI3K/AKT/mTOR, and Wnt/β-catenin. These interactions form complex crosstalk networks that drive malignant transformation and progression. Furthermore, Hes1 plays a central role in the formation of an immunosuppressive tumor microenvironment (TME) and immune escape by regulating the expression of immune checkpoint-associated proteins, extracellular matrix (ECM) remodeling, and other processes, making it a highly promising therapeutic target. Notably, the expression level of Hes1 is significantly correlated with tumor clinical stage, prognosis, and drug resistance. This review comprehensively introduces the mechanisms of Hes1 in the progression of malignant tumors, with a particular focus on discussing its application and underlying mechanisms in tumor immunotherapy. It integrates the latest clinical evidence and preclinical research perspectives. The goal is to highlight the translational potential of Hes1 as a novel biomarker and molecular target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca4a5e14b5411c1cf587b921ebef2a69839952a" target='_blank'>
              Hes1 in malignant tumors: from molecular mechanism to therapeutic potential
              </a>
            </td>
          <td>
            Liping Zhang, Qian Zhang, Cheng Guo, Zixin Ru, Zetian Yang, Yi Geng, Junjie Yang, Daigui Zhang, Zhenhuai Yang, Shuicai Huang
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Androgen receptor–positive prostate cancer (PC), castration-resistant prostate cancer (CRPC), and neuroendocrine prostate cancer (NEPC) invariably become resistant to treatment with targeted and cytotoxic agents. Multiple pathways have been identified as being responsible for these pleiotropic mechanisms of resistance. The mucin 1 (MUC1) gene is aberrantly expressed in CRPC/NEPC in association with poor clinical outcomes; however, it is not known if the oncogenic MUC1-C/M1C protein drives treatment resistance. We demonstrated that MUC1-C is necessary for resistance of (i) PC cells to enzalutamide (ENZ) and (ii) CRPC and NEPC cells to docetaxel (DTX). Our results showed that MUC1-C–mediated resistance is conferred by upregulation of aerobic glycolysis and suppression of reactive oxygen species necessary for self-renewal. Dependence of these resistant phenotypes on MUC1-C for the cancer stem cell (CSC) state identified a potential target for treatment. In this regard, we further demonstrated that targeting MUC1-C with an M1C antibody-drug conjugate (ADC) is highly effective in suppressing (i) self-renewal of drug-resistant CRPC/NEPC CSCs and (ii) growth of treatment-emergent NEPC tumor xenografts derived from drug-resistant cells and a patient with refractory disease. These findings uncovered a common MUC1-C–dependent pathway in treatment-resistant CRPC/NEPC progression and identified MUC1-C as a target for their therapy with an M1C ADC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369f7925ccd9dc6cd2f8026334a0398635028420" target='_blank'>
              MUC1-C dependence in treatment-resistant prostate cancer uncovers a target for antibody-drug conjugate therapy
              </a>
            </td>
          <td>
            Keisuke Shigeta, Tatsuaki Daimon, Hiroshi Hongo, Sheng-Yu Ku, Hiroki Ozawa, Naoki Haratake, A. Fushimi, Ayako Nakashoji, Atrayee Bhattacharya, Shinkichi Takamori, Michihisa Kono, Masahiro Rokugo, Y. Baba, Takeo Kosaka, M. Oya, Justine J Jacobi, Mark D Long, H. Beltran, Donald W Kufe
          </td>
          <td>2025-06-24</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Background Response gene to complement 32 (RGC32), a complement activation-inducible factor broadly expressed in normal human tissues, has been implicated in tumorigenesis through its dysregulated expression in various malignancies and its involvement in critical oncogenic processes. Despite its established roles in cancer biology, RGC32 remains uncharacterized in diffuse large B-cell lymphoma (DLBCL). This study provides the first comprehensive investigation of RGC32 expression patterns and functional contributions to DLBCL pathogenesis, elucidating its potential as a novel therapeutic target or prognostic biomarker in this disease. Methods Immunohistochemical (IHC) staining of RGC32 was performed on specimens from 32 Reactive hyperplasia lymphoid (RHL) patients and 80 DLBCL patients. To evaluate the role of RGC32 in DLBCL, lentivirus vectors either encoding shRGC32 or shControl were transfected into DLBCL cell lines. RNA-sequencing (RNA-seq) analysis was performed between shRGC32 and shControl stably transfected OCI-LY1 cells and functional enrichment analyses used gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG). In order to explored its functions in vivo, xenograft models were established by subcutaneously injecting shRGC32 and shControl transfected DLBCL cells into SCID beige mice. Results Immunohistochemical analysis revealed RGC32 overexpression in DLBCL tissues contrast with RHL, and was associated advanced Ann Arbor stage (p = 0.043), B symptoms (p = 0.020), and poor progression-free survival (p = 0.015) and overall survival (p = 0.035). Functional studies demonstrated that RGC32 knockdown via shRNA significantly suppressed DLBCL cell proliferation in vitro and in vivo, with xenograft models showing reduced tumor growth and Ki-67 expression. RNA-seq analysis linked RGC32 depletion to downregulation of cell proliferation and impaired DNA damage repair (DDR) mechanisms. Western blot showed RGC32 knockdown could suppress ATM/ATR/CHK1 pathway and increase the tumor mutational burden (TMB). Furthermore, after inhibition of RGC32, infiltration of CD8+ T cells was increased in DLBCL tumor microenvironment (TME). Conclusions This study highlights that RGC32 is a novel molecule in DLBCL progression and might be a potential therapeutic target for DLBCL therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78dc8ef431c9b4ad9b48222a8eef16b0fca96480" target='_blank'>
              Response gene to complement 32 promotes tumorigenesis by mediating DNA damage repair and inhibits CD8+ T cells infiltration in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Xiyuan Zhang, T. Lu, Chunlei Shi, M. Ding, Ling Wang, Xinting Hu, Xin Wang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A Genome-wide association study (GWAS) on a European-American cohort identified chr11p11.2 as a neuroblastoma predisposition locus. Combining in-house and public genomic data from neuroblastoma cell lines, this work implicates rs2863002 as the candidate causal variant at the 11p11.2 locus, confirming its cis-regulatory activity through a luciferase reporter assay. The genetic association of rs2863002 with neuroblastoma risk is validated in an Italian case-control cohort. Using ChIP-qPCR, Hi-C, and CRISPR genome editing, this work deciphers the regulatory mechanisms at the risk locus, demonstrating that the rs2863002-C risk allele regulates HSD17B12 expression and reduces GATA3 binding affinity. In vitro functional assays and targeted lipidomic analyses reveal the involvement of the rs2863002-C risk allele in tumorigenicity and modulation of lipid metabolism in neuroblastoma cells through HSD17B12 regulation. This study provides new insights into the genetic basis of neuroblastoma and underscores the importance of post-GWAS functional characterization of risk loci in uncovering relevant biological findings for understanding complex diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07b7a19d2bc29c61b3a57e2ffd3f546770c0c34b" target='_blank'>
              The Non-Coding Regulatory Variant rs2863002 at chr11p11.2 Increases Neuroblastoma Risk by Affecting HSD17B12 Expression and Lipid Metabolism.
              </a>
            </td>
          <td>
            Teresa Maiorino, M. Avitabile, Vincenzo Aievola, Annalaura Montella, V. A. Lasorsa, Ferdinando Bonfiglio, Mariagrazia Cantalupo, Sueva Cantalupo, Gilda Estinto, Matilde Tirelli, Martina Morini, Martina Ardito, Alessandra Eva, Vincenza Cerbone, Lucia Mauriello, M. Caterino, M. Ruoppolo, John M. Maris, S. Diskin, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-06-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Metastable epialleles (MEs) are genomic loci at which epigenetic marks are established stochastically during early embryonic development and maintained during subsequent differentiation and throughout life, leading to stable epigenetic and phenotypic variation among genetically identical individuals. Although MEs were first described in mice over 20 years ago, the extent of epigenetic metastability in the mouse genome remains unknown. We present the first unbiased genome-wide screen for MEs in mice. Using deep whole-genome bisulfite sequencing across tissues derived from the three embryonic germ layers in isogenic C57BL/6J mice, we identified only 29 MEs, precisely localizing them and documenting their rarity. Consistent with recent findings, we found no effects of maternal dietary methyl donor supplementation on ME methylation in the offspring, challenging previous assertions that MEs generally exhibit developmental plasticity. Most but not all MEs are associated with intracisternal A-particle (IAP) elements, tending to localize to the 5′ end of the IAP. Additionally, we discovered autosomal regions at which systemic interindividual variation in DNA methylation is associated with sex, providing insights into sex-associated epigenetic development that apparently precedes sexual differentiation. Our findings indicate that expression of transcription factors, including CCCTC-binding factor (CTCF) and specific KRAB zinc finger proteins during early embryonic development, plays a key role in orchestrating stochastic establishment and/or maintenance of DNA methylation at metastable transposable elements. Overall, these findings advance our understanding of the genomic determinants of epigenetic metastability and suggest that interindividual epigenetic variation at MEs is unlikely to be a major determinant of phenotypic variation among isogenic mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708a43f6a2e4e91d381b16e3a28eb5d04dfb8092" target='_blank'>
              Mouse metastable epialleles are extremely rare
              </a>
            </td>
          <td>
            Chathura J Gunasekara, U. Maduranga, Taylor Zhang, Jonathan N Wells, Maria S. Baker, Eleonora Laritsky, Yumei Li, C. Coarfa, Yi Zhu, R. Waterland
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are abundant in the human genome with approximately 300,000 embedded in gene introns, exons, and untranslated regions. High penetrance STR variants cause human diseases such as Myotonic dystrophy, Baratela-Scott syndrome, and various ataxias. The possibility that STRs contribute to polygenic disease is supported by recent high-powered datasets that link STRs to more subtle effects on gene expression. Indeed, STR variants can induce Z-DNA and H-DNA folding; alter nucleosome positioning; and change the spacing of DNA binding sites. On the other hand, little is known about how STR variants affect RNA secondary structure and accessibility. These factors could affect rates of splicing, nuclear export, and translation. We hypothesize that effects on RNA structure can be predicted using computational tools and associated with gene expression using DNA and RNA sequencing data. We test this hypothesis using data from the 1000 Genomes Project and ViennaRNA. We identify 17,255 transcribed STRs that affect RNA folding (fSTRs); 356 are possibly associated with gene expression. We characterize fSTRs by repeat motif, length, and gene level annotation. Transcribed fSTR variants tend to affect RNA multiloops and external loops. Effects on RNA accessibility depends on the repeat motif: a surprising result that is checked against simulation. These results shed light on how transcribed STRs affect RNA structure and pave the way for experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ac5e9e6ba805badefe346bba5943b6b1c9b8c40" target='_blank'>
              Short tandem repeat variants are possibly associated with RNA secondary structure and gene expression
              </a>
            </td>
          <td>
            Nick Kinney, Dikshya Pathak, Emma Evans, Paola Arias
          </td>
          <td>2025-06-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Foot-and-mouth disease (FMD) is an infectious disease in animals caused by the foot-and-mouth disease virus (FMDV). However, the mechanism of FMDV infection in host cells remains unclear. In this study, we utilized 12C6 heavy ion irradiation technology to process BHK-21 cells and systematically screened and evaluated cell lines with distinct virus replication characteristics. We subsequently employed proteomics to detect the differences between these cell lines and the control BHK-21 cell line following 12C6 heavy ion irradiation. Both cell lines exhibited common downregulation of cell adhesion molecules but also exhibited distinct upregulation pathways. In terms of immune and metabolic responses, BHK-5 infection triggered an immune response, including the upregulation of cytokine–cytokine receptor signaling pathways and lysosome-related pathways, while the upregulation of drug metabolism pathways enhanced the ability to metabolize exogenous substances. Conversely, BHK-7 infection tended to promote metabolic pathway changes that favor virus replication, such as the upregulation of folate biosynthesis, polysaccharide degradation, and linolenic acid metabolism pathways. Additionally, we observed significant downregulation of Cbr3 in cell lines that promoted virus replication and significant upregulation in those that inhibited virus replication. Upon validating the results in Cbr3 knockout cells, we found that knocking out Cbr3 could increase FMDV replication by increasing the cellular content of prostaglandin E2 (PGE2), suggesting a close relationship between FMDV replication and PGE2 levels. This method can increase the production efficiency of FMDV vaccines while reducing manufacturing costs. This study innovatively employed 12C6 heavy ion irradiation technology to induce cell transformation and explored its impact on FMDV, offering a new perspective for understanding virus replication mechanisms and potentially providing a target and idea for developing novel antiviral strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05628-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ceb7df4add47002d29186ee424c9a7ff044a17f" target='_blank'>
              The impact of 12C6 heavy ion irradiation-induced cellular mutations on the replication of the foot-and-mouth disease virus and the role of Cbr3
              </a>
            </td>
          <td>
            Xiangdong Song, Shiyu Tao, Fanglan An, Xiaoming Li, Yanyan Chang, Xuerong Liu, Yan Cui
          </td>
          <td>2025-06-28</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Unchecked growth in three-dimensions (3D) in culture is a key feature of immortalized cells on the path to malignant transformation and hence a potential target phenotype for prevention. Also, expression of genes driving this process, but not that of 2D growth, would likely be more specific to cancer development and their inhibition would be less toxic to normal cells, many of which can grow in 2D but rarely in 3D culture. To define such genes, we compared CRISPR depletion screens performed in HBLAK, a spontaneously immortalized, non-tumorigenic human urothelial cell line, grown in 2D to those in 3D. Using the CRISPR Bassik DTKP (drug target kinase phosphatase) deletion library targeting 2,333 genes, we identified 85 genes which were specifically lost in 3D cultures. Prioritizing hits to those associated with bladder cancer in patients provided us with a set of 11 genes. Only one gene, MAPK1 remained relevant if a human pan-cancer criteria was applied. Single gene in vitro validation confirmed that MAPK1 inhibition was specific to 3D growth. We also found that MAPK1 depletion led to significant growth reductions in human tumor xenografts in vivo. Inhibition of MAPK1 by Ulixertinib, an orally active MAPK1 inhibitor, led to human bladder cancer growth inhibition in both 3D in vitro and in vivo models. In summary, screening for genes specifically driving 3D growth in immortalized cells may provide targets for both prevention and early therapy in bladder and other cancers while potentially limiting therapeutic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c39ec53d27180d1be55595d4ca83d125348b0422" target='_blank'>
              Genes driving three-dimensional growth of immortalized cells and cancer
              </a>
            </td>
          <td>
            Mukta Basu, Jin-Fen Xiao, Saravana Kumar Kailasam Mani, Fangyuan Qu, Yongqi Lin, Jason Duex, Huihui Ye, Vanessa Neang, Dan Theodorescu
          </td>
          <td>2025-06-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The canonical DNA damage response (cDDR) maintains genome stability, involving DNA synthesis arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously characterized a noncanonical response specific to nonblocking replication stress, i.e. low-level stress (LoL-DDR), in primary cells. Although LoL-DDR generates replication stress-induced ROS (RIR), it prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a threshold triggers the cDDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: (1) RIR are not detoxified under high-level stress, resulting in a continuous increase in the dose‒response curve of RIR production; (2) RIR are not produced by NADPH oxidases; (3) replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” replication-stress/ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05172-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1826ee141a3bb5d6a8079b9d816c3e423819f87e" target='_blank'>
              Dysregulation of the low-level replication stress response in transformed cell lines
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-06-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Mutational signatures are increasingly used to understand the mechanisms causing cancer. However, their important applications in predicting prognosis and stratifying patients for therapy are hampered by inaccurate inference of the various featureless, dense trinucleotide mutational spectra, which are often confounded with one another. One of them is the homologous recombination deficiency (HRd)-associated signature SBS3, relevant because of its association with prognosis in ovarian and breast cancer and because of its potential as a biomarker for synthetic lethality therapies. Methods Here, we highlight strong benefits of a multimodal approach for mutational signature extraction, applied on top of standard bioinformatic pipelines. By jointly operating on single-base substitution (SBS) and indel (ID) spectra, this method enables accurate identification of various DNA repair deficiency signatures and patient survival prediction. Results Across four different cohorts of whole-genome sequenced high-grade serous ovarian cancers (HGSOC), the multimodal SBS + ID approach correctly distinguished the commonly confused signatures SBS3, SBS5, SBS8, SBS39, and SBS40. Importantly, we robustly identified two different multimodal SBS3 signatures, m-SBS3a and m-SBS3b, with distinct patterns in the indel spectrum. Multimodal SBS3b signature was strongly predictive of longer survival in ovarian cancer patients, replicating across four cohorts, with effect sizes greatly exceeding other genetic markers. Our m-SBS3 also predicted survival in platinum-treated patients with various cancer types, and moreover, the SBS + ID joint inference was successfully applied to mismatch repair-deficient colorectal cancer and immunotherapy response, supporting a general utility of the multimodal mutational signatures approach. Conclusions Overall, combining SBS and ID mutations improves detection of HR deficiency-associated signatures and reveals distinct SBS3 subtypes with prognostic value. This multimodal approach outperforms existing markers and is readily applicable to therapy stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01497-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01c11401f870f4aa8e6266e2fe8e3ab75a0da51" target='_blank'>
              Joint inference of mutational signatures from indels and single-nucleotide substitutions reveals prognostic impact of DNA repair deficiencies
              </a>
            </td>
          <td>
            Patricia Ferrer-Torres, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-07-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c9d768db6c320ae3141a12d174f6fdda8a0e39f" target='_blank'>
              Modified hTERT promoters-driven purine nucleoside phosphorylase-gene therapy in association with chemo- and targeted therapy in the context of ovarian cancer.
              </a>
            </td>
          <td>
            Quoc Manh Nguyen, Pierre-François Dupré, Mathieu Berchel, Marylène Lévêque, Sylvie Choblet-Thery, F. d’Arbonneau, T. Montier
          </td>
          <td>2025-06-27</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Background: Patients exhibiting exceptional response to targeted treatments represent a unique opportunity towards identifying predictive biomarkers of response. The introduction of edits into the genome using Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) creates cancer models that can unravel the mechanisms driving tumorigenesis and response to drug agents. The study aimed to identify candidate molecular alterations associated with exceptional response to CDK inhibitors (CDKi) and validate their effects using genome editing. Methods: Patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy at Departments of Oncology-affiliated with the Hellenic Cooperative Oncology Group (HeCOG), were included. Exceptional response was defined as either complete response (CR) to the treatment regimen or partial response (PR) for 3 years, per physician’s assessment. Formalin-fixed paraffin-embedded (FFPE) tumor tissue samples, obtained prior to CDKi administration from these patients were retrospectively collected. Whole-exome sequencing (WES) was performed to analyze tumor mutational landscape of exceptional responders to CDKi. Using CRISPR-mediated genome editing, a breast cancer cell model harboring a patient-specific SNP and the negative control were prepared, and subsequently evaluated for their responsiveness to palbociclib. Transcriptomic and proteomic analyses were conducted to uncover alterations in network interactions, between the two cell lines, elucidating the exceptional response to CDKi. Results: Overall, 12 women identified as exceptional responders, initiated CDKi treatment from March 2017 to February 2020; median age at diagnosis was 46 (range, 37 to 78). All patients had received the CDKi as first-line treatment; most commonly palbociclib (10 patients, 83.3%). Based on physician assessment, 50% of patients achieved CR with the combination therapy. After a median follow up of 65 months, 10 patients are still alive, and 7 are still receiving treatment with CDKi in combination with endocrine treatment. WES results with adequate mapped reads and acceptable technical characteristics in both tumor and control blood samples were obtained from 6 of 12 patients. A set of germline and somatic single nucleotide polymorphisms (SNPs) commonly shared in this cohort was identified. The focus was on MINDY1, a member of the MINDY family of deubiquitinases (DUBs), known as a deubiquitylase of ERα in breast cancer, the common SNP among all 6 patients. Utilizing CRISPR-based genome editing, a breast cancer cell model harboring a specific MINDY1 SNP was engineered, and their response to palbociclib was subsequently evaluated. Depletion of MINDY1 transcripts through RNA interference with short-hairpin RNA, increased sensitivity to palbociclib in both cell lines, confirming that MINDY1 protein is involved in molecular pathways related to palbociclib response. The functional role of MINDY1 rs771205 was also investigated by combining transcriptomic and proteomic data. Our data demonstrated that the MINDY1 rs771205 SNP, which causes an amino acid substitution, while it does not impact the respective transcript and protein levels, alters the MINDY1 protein’s interaction network. Conclusions: The Rs771205 single mutation alters the MINDY1 protein’s interaction network, sensitizing cells to palbociclib. Our study sheds new light on the underlying mechanisms driving exceptional response in patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy , demonstrating that it is an opportune moment to invest attention and resources towards probing the underlying mechanisms of exceptional response in cancer patients.
 Citation Format: Eleni Balla, Yi Li, Kyriaki Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, Eleni Galani, Rania Romanidou, George Fountzilas, Evangelia Razis, Angelos Koutras, Leonidas Bleris, Elena Fountzilas. MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-03-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea823fc49d305c67d0c88bebfe1f1411182d2dc" target='_blank'>
              Abstract P4-03-26: MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer
              </a>
            </td>
          <td>
            Eleni Balla, Yi Li, K. Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, E. Galani, Rania Romanidou, G. Fountzilas, E. Razis, A. Koutras, L. Bleris, E. Fountzilas
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Bladder cancer (BC) displays a huge phenotypic variation and widespread clinical outcomes, attributed to the high mutational heterogeneity of the disease. Mutational landscaping became integral to cancer patient management as it unveils driver genes and yields genotype–phenotype associations. This study aims to identify somatic mutations, their frequencies and their associations with clinical and prognostic outcomes in BC. Eighty-one BC patients were analyzed by next-generation sequencing using the Ion AmpliSeq Cancer Hotspots Panel v2. Bioinformatics analysis, correlation studies and Kaplan–Meier curve were used to evaluate the relationship between genes’ mutational status and patients’ clinical parameters and outcomes. Our results indicated that the BC cohort exhibited a higher mutation burden than the TCGA data. Mutations were identified in 46 out of 50 genes, including 21 novel mutations not previously reported in BC. The TP53 gene was mutated in 82.5% of the analyzed cohort, followed by PIK3 CA (45%), FGFR3 (43.75%) and APC (35%). TP53 mutations were associated with poor survival (p = 0.003) while the FGFR3 mutation group exhibited signs of good prognosis (p = 0.018). Bioinformatics highlighted significant gene interactions associated with poor prognosis. These findings underline the importance of identifying novel genetic mutations that could significantly improve prognostic stratification and expand therapeutic options for managing BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def8cff968adcdbff8ae094040404982997ef901" target='_blank'>
              Clinical relevance of the somatic mutational landscaping in predicting outcome of bladder cancer patients
              </a>
            </td>
          <td>
            T. Nedjadi, H. Ansari, Muhammad A Khan, Naif S Sannan, Mubarak M. Al-Mansour, J. Al-Maghrabi, Ashraf Dallol
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b94cf63206b221ede960c719c7b5d998ebadf2b" target='_blank'>
              Specific TP53 mutations impair the recruitment of 53BP1 to DNA double-strand breaks underlying the mechanism of radioresistance.
              </a>
            </td>
          <td>
            Paolo Fagherazzi, L. Stixová, E. Bártová
          </td>
          <td>2025-07-14</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2968d582c4d09600f6a6aab6083cc272395bfdb6" target='_blank'>
              MicroRNA-driven organ-specific metastasis in triple-negative breast cancer: biogenesis, mechanisms, and therapeutic approaches.
              </a>
            </td>
          <td>
            Deepshikha Rathore, Sweny Jain, Shanaya Patel, Nandani Dharwal, N. Shukla, Chirag Desai, Jigna Shah, Heena V. Dave
          </td>
          <td>2025-06-28</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="For chromosome abnormalities (CA), such as Down syndrome (DS), the influence of genomic variations to chromosome conformation and gene transcription remains elusive. Based on the complete genomic sequence from the parents of the DS trisomy patient, we systematically delineated an atlas of parental-specific haploid single nucleotide polymorphism (SNP), copy number variation (CNV), three-dimensional (3D) genome, and RNA expression profiles of the diencephalon in the DS patient. The integrated haploid multi-omics analysis demonstrated that one-dimensional genomic variations including SNPs and CNVs in the DS patient are highly correlated with the alterations of the 3D genome and the subsequent gene transcription. The correlation relationship remains valid in haploid-levels. Moreover, we revealed the 3D genome alteration associated mis-regulation of DS-related genes, which facilitates to understanding the pathogenesis of CA. Together, our study contributes to decipher the coding from one-dimensional genomic variations to 3D genomic architecture and the subsequent gene transcription in healthy and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9de44c4aa76967640e3414bc7acd3b2037c35" target='_blank'>
              Deciphering Haploid Chromosome Conformation Alternation in Down Syndrome by Multiple Haploid Omics Analysis.
              </a>
            </td>
          <td>
            Chengchao Wu, Tianshu Zhou, Wenfu Ke, Wei Xiong, Zhihui Zhang, Siheng Zhang, Jinyue Wang, Lulu Deng, Keji Yan, Man Wang, Shenglong He, Qi Gong, Chao Ma, Xiaping Chen, Yan Li, He Long, Chong Guo, Gang Cao, Zhijun Zhang
          </td>
          <td>2025-06-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc739b29aa041a82d43e91798c3e41caf831aa6b" target='_blank'>
              Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants
              </a>
            </td>
          <td>
            Ethan W. Hollingsworth, Zhuoxin Chen, Cindy X. Chen, Sandra H. Jacinto, Taryn A Liu, Evgeny Z. Kvon
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Epigenetic modifications prove critical to cancer development, so the disease stands as a major cause of death in people, according to scientific evidence. Environment and cellular alterations affect modifications that differ from genetic mutations because DNA methylation, histone modification, chromatin remodeling, and non-coding RNA activity show reversible potential. This paper thoroughly examines epigenetic methods while demonstrating how their regulatory functions help cancer development without altering DNA sequences. The research reviews how irregular DNA methylation patterns combined with modifiable histone codes and chromatin structural modifications affect tumorous heterogeneity and create drug resistance while preventing immune responses. The document evaluates epigenetic biomarker accuracy for early cancer detection and their forecasting abilities alongside modern developments in anti-epigenetic therapeutic strategies. This article integrates present multi-omics research with translational studies to develop full knowledge about how epigenetic factors define cancer biology for precise medical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9bf265cc39518378d4e53c6395df1bc723ba17" target='_blank'>
              The Role of Epigenetics in Cancer Development: Investigating the Link between Epigenetic Modifications and Cancer
              </a>
            </td>
          <td>
            Sinigdha Islam, Bibi Ayesa, Dr. Mahmudul Hasan, Md. Jobaer Rahman Rashed, Banasree Roy Urmi, R. Malakar, Shekh Mohammad Mostafa, Aditta Das, Zahidul Mostafa, M. Nesa
          </td>
          <td>2025-06-12</td>
          <td>Journal of Artificial Intelligence General science (JAIGS) ISSN:3006-4023</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f43f79f58977259121120764f54b519011ee5e" target='_blank'>
              RBM5 recruiting MGC32805 in a sandwich mode and inducing ΔFAS neoantigen and triggering FAS properties switch: implication in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhe Wu, Xiaoyun Hu, Yilin Wang, Xianglong Zhu, Qing Zhao, Ying Zhao, Wanlin Cui, Mingrong Zhang, Haishan Zhao, Kai Li, Minjie Wei
          </td>
          <td>2025-06-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Cellular reprogramming is a fundamental process in specific developmental stages and is also invaluable for disease modeling. It involves the conversion of differentiated cells into undifferentiated or specialized cell types. This transformation relies on coordinated action of transcription factors or chemical stimuli, which reshapes the cell's chromatin structure and establishes a different gene expression profile. Among these transcription factors, pioneer factors play a critical role, initiating cellular reprogramming and guiding cell fate transitions by selectively binding to specific DNA motifs within nucleosomes at inactive chromatin. The GATA3 protein serves as a pioneer transcription factor essential in mammary gland development and is involved in breast tumor development. GATA3 facilitates the cellular reprogramming of mesenchymal breast cancer cells into epithelial breast cancer cells. Despite the established role of pioneer factors in chromatin remodeling and cell fate regulation, the precise molecular mechanisms that govern their selective binding to chromatin are still not fully understood. Using the mesenchymal-to-epithelial transition (MET) model, we aimed to understand how GATA3 selectively binds to genes that are essential for successful cellular reprogramming. Utilizing a combination of machine learning and experimental approaches, we investigated GATA3's binding affinity to DNA and nucleosome substrates. Machine learning analysis identified two distinct zinc-finger domains which are required for effective GATA3 nucleosome binding. The results of this deep learning analysis were used to design a series of GATA3 mutant cell lines and DNA substrates. The experimental approach involved two strategies: first, analyzing the binding affinity of wild-type GATA3 to DNA substrates designed based on machine learning predictions, and second, examining the nucleosome binding affinity of GATA3 mutants, including those predicted by computational modeling and those derived from breast cancer patients. Electrophoretic mobility shift assays confirmed the differential binding affinities of GATA3 mutants, indicating the importance of specific motifs and zinc finger domains. Our findings highlight the significance of DNA sequence specificity in modulating pioneer transcription factor activity and chromatin accessibility. Mutational analysis revealed distinct roles for each zinc finger domain, with mutations affecting DNA binding affinity and chromatin accessibility. This study provides insights into the molecular mechanisms governing GATA3-mediated chromatin remodeling during MET. By characterizing the interaction between DNA/nucleosome binding affinity and chromatin accessibility, we expand our understanding of pioneer transcription factor function in breast cancer progression.
 Citation Format: Jill Goodman, Mika Saotome, Motoki Takaku. Nucleosome binding affinity of the pioneer factor GATA3 influences chromatin reprogramming in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4cf68bc1ac895d343e311d0ea750e083006c2b" target='_blank'>
              Abstract P5-06-15: Nucleosome binding affinity of the pioneer factor GATA3 influences chromatin reprogramming in breast cancer
              </a>
            </td>
          <td>
            Jill Goodman, Mika Saotome, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Chemicals reactivating epigenetically silenced genes may cause strong transcriptome changes inducing both beneficial and hazardous effects, in particular, promoting carcinogenesis. Among chemicals used in medicine and commerce, only a small fraction has been studied with respect to their influence on epigenetic regulation. Screening of non-genotoxic chemicals reactivating silent genes requires adequate test-systems mimicking whole-genome processes. In our study we used a HeLa TSA-Inducible cell population (HeLa TI) harboring epigenetically silenced chromosomally dispersed GFP-reporter genes. This system is based on the principle that epigenetic gene silencing is reversible, and that a silent reporter gene can be reactivated by inhibition of key epigenetic silencing factors. 15 chromatin-based epigenetic silencing factors were shown to participate in GFP silencing in these cells. In our study we demonstrated that different epigenetically active compounds, including six HDAC inhibitors, three DNMT inhibitors; three HMT inhibitors, two BET inhibitors and chromatin remodeler–curaxin CBL0137 are able to activate GFP expression. We also showed that combinations of epigenetic modulators caused a significant increase in the number of cells with reactivated GFP compared to the individual effects of each agent. HeLa TI cells are competent to metabolize xenobiotics and possess constitutively expressed and inducible cytochrome P450 monooxygenases metabolizing xenobiotics. Thus, HeLa TI cells may be used as an adequate test system for the extensive screening of epigenetically active chemicals, including those that must be metabolically activated. Further approaches to improve the test-system for the screening of epigenetically active carcinogens including the use of new fluorophores, different cell lines and cell signaling analysis will be also discussed.



 Kirsanov K.I., Maksimova V.P., Vlasova O.A., Zhidkova E.M., Zenkov R.G., Lylova E.S., Karpechenko N.Y., Osipova A.V., Grigorieva D.D., Popova V.G., Fetisov T.I., Borunova A.A., Zabotina T.N., Belitsky G.A., Yakubovskaya M.G.. Sorted cell populations with epigenetically silenced chromosomally dispersed reporter fluorophore gene as a test-system for the screening of non-genotoxic carcinogens [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P84.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b8055dda1f803f37b6f92aca70aff4f30e51902" target='_blank'>
              Abstract P84: Sorted cell populations with epigenetically silenced chromosomally dispersed reporter fluorophore gene as a test-system for the screening of non-genotoxic carcinogens
              </a>
            </td>
          <td>
            K. Kirsanov, V. Maksimova, O. Vlasova, E. Zhidkova, R. Zenkov, E. Lylova, N.Y. Karpechenko, A.V. Osipova, D. D. Grigorieva, V. Popova, T. I. Fetisov, A. Borunova, T. Zabotina, G. A. Belitsky, M. Yakubovskaya
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Women who harbor germline heterozygous mutations of BRCA1 or BRCA2 have a high risk of breast cancer. Our previous study showed that patient-derived, ostensibly normal BRCA2mut/+ luminal progenitor (LP) cells are more prone to exhibit sub-chromosomal copy number variations and associated DNA damage relative to non-carriers, potentially reflecting early breast tumorigenesis. Clinically assessable biomarkers for early pathological changes of BRCA1/2 mutation in LP cells remain unknown. Single-cell RNA sequencing (scRNAseq) of LP cells of BRCA1/2mut/+ carriers revealed enrichment of KIT expression (KIT+) and subsequent transcriptional factor activations were observed in LP cells of BRCA1/2mut/+ carriers relative to non-carriers. Moreover, pathway analysis uncovered that KIT+ BRCA-mutated LP cells were enriched in pathways involving DNA binding transcription activator activity and oxidative phosphorylation. These gene signature profiles were recapitulated in bulk RNA-seq of BRCA2 mut/+ carrier LP cells. Collectively, our preliminary data suggest such analyses may identify potential biomarkers for near-term risk prediction that reflect early pathogenesis of BRCA1/2-associated carcinogenesis. Following further validation via tissue microassay analysis and clinical studies, our findings may eventually assist patients and clinicians in decision making regarding the timing and necessity of preventive surgeries for BRCA1/2 mutation carriers.
 Citation Format: Zuen Ren, Siang Boon Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, Veerle Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W. Ellisen. Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a865906cd027bf501016cc53a811825bba9923" target='_blank'>
              Abstract P3-04-06: Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer
              </a>
            </td>
          <td>
            Zuen Ren, S. Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, V. Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W Ellisen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignancies in men, posing a significant public health challenge due to its high incidence and long-term treatment-related toxicities. Long-lived patients often experience prolonged side effects that can severely diminish their quality of life. Despite advancements in radiotherapy techniques like IMRT and VMAT, some patients still experience acute and late side effects. Current treatment protocols do not account for individual variability in normal-tissue radiosensitivity, highlighting the need for predictive tools and a personalised treatment approach. Genetic factors and molecular regulators like microRNAs (miRNAs) contribute to these variations by influencing DNA repair, inflammation, and apoptosis. This review explores potential biomarkers of radiotoxicity, focusing on immune-related factors such as IL-6 and TGF-β1, SNPs influencing radiosensitivity, miRNAs involved in radiation responses, and functional assays including the radiation-induced lymphocyte apoptosis (RILA) test. These approaches offer promising tools for identifying radiosensitive patients and enabling risk-adapted radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d835fafe61bcfabd703da02feed9ff1eb1a8495" target='_blank'>
              The Role of Predictive Biomarkers in Modern Prostate Cancer Radiotherapy: A Literature Review on Personalised Treatment Strategies and the Prediction of Adverse Effects
              </a>
            </td>
          <td>
            Jelena Stanić, Ivana Šović, Luka Jovanovic, Ivana Z Matić, P. Nikić, M. Nikitović
          </td>
          <td>2025-07-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982238bfa802534a4f1e695ccfc316aaa317bc9c" target='_blank'>
              Phased epigenetic fragility and stability accelerate the genetic evolution of B-cells
              </a>
            </td>
          <td>
            Mark Y Xiang, Haripriya Vaidehi Narayanan, Vaibhava Kesarwani, Tiffany Wang, Alexander Hoffmann
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0cb77984e400e74b61266bf02611dd881e27a75" target='_blank'>
              A cancer-associated TP53 synonymous mutation induces synthesis of the p53 isoform p53/47.
              </a>
            </td>
          <td>
            Rhythm Sajwan, Lixiao Wang, O. Casar-Borota, Konstantinos Karakostis, Sa Chen, Robin Fåhraeus, Xiaolian Gu, Sivakumar Vadivel Gnanasundram
          </td>
          <td>2025-07-26</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transposable elements (TEs) are abundant in the human genome, and they provide the source for genetic and functional diversity. Previous studies have suggested that TEs are repressed by DNA methylation and chromatin modifications. Here through integrating transcriptome and 3D genome architecture studies, we showed that haploinsufficient loss of NIPBL selectively activates alternative promoters (altPs) at the long terminal repeats (LTRs) of the TE subclasses. This activation occurs through the reorganization of topologically associating domain (TAD) hierarchical structures and the recruitment of proximal enhancers. These observations indicate that TAD hierarchy restricts transcriptional activation of LTRs that already possess open chromatin features. Perturbation of hierarchical chromatin topology can lead to co-option of LTRs as functional altPs, driving aberrant transcriptional activation of oncogenes. These data uncovered a new layer of regulatory mechanisms of TE expression and posit TAD hierarchy dysregulation as a new mechanism for altP-mediated oncogene activation and transcriptional diversity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, P. Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="PIK3CA and KRAS are among the most frequently mutated oncogenes and often co-mutated in colorectal cancers. Understanding how KRAS codon-specific mutations affect cross-talks between the PI3K and MAPK pathways and response to targeted therapies, such as the p110α-specific inhibitor inavolisib (GDC-0077), is critical for advancing precision oncology. Focusing on colorectal PIK3CA+KRAS co-mutated models, we found that KRAS G12D-mutated cells were more sensitive to inavolisib than models with KRAS G13D, or other MAPK pathway mutations, even though the PI3K and MAPK pathways were active in both genotypes. In most co-mutated models, regardless of the type of KRAS alteration, combination of inavolisib with MAPK pathway inhibitors showed synergy in vitro and in vivo. Our work highlights how specific codon substitutions in KRAS differentially toggle pathway activity and alter sensitivity to inavolisib, which could inform whether patients would benefit more from single-agent inavolisib or combination with MAPK pathway inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70c098958b33ecc57b400045bf2df06bb8439fa" target='_blank'>
              KRAS codon-specific mutations differentially toggle PI3K pathway signaling and alter sensitivity to inavolisib (GDC-0077).
              </a>
            </td>
          <td>
            Kyung W Song, Christy C Ong, Eva Lin, Jeff Lau, Nicole M Sodir, Dexter Jin, Katherine E. Hutchinson, Shiqi Xie, Jenille Tan, Yuxin Liang, Z. Modrusan, Scott E Martin, Danilo Maddalo, M. Hafner, Anwesha Dey
          </td>
          <td>2025-07-08</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Approximately half of all cancers bear mutations in the tumor suppressor p53. Despite decades of research studying p53 function, treatment of p53-mutant cancers remains challenging owing to the effects of p53 mutations on many complex and interrelated signaling networks that promote tumor metastasis and chemoresistance. Mutations in p53 promote tumor survival by dysregulating cellular homeostasis and preventing activation of regulated cell death (RCD) pathways, which normally promote organismal health by eliminating dysregulated cells. Activation of RCD is a hallmark of effective cancer therapies, and p53-mutant cancers may be particularly susceptible to activation of certain RCD pathways. In this review, we discuss four RCD pathways that are the targets of emerging cancer therapeutics to treat p53-mutant cancers. These RCD pathways include E2F1-dependent apoptosis, necroptosis, mitochondrial permeability transition-driven necrosis, and ferroptosis. We discuss mechanisms of RCD activation, effects of p53 mutation on RCD activation, and current pharmaceutical strategies for RCD activation in p53-mutant cancers. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22a5ed3e88436170b46362ce1b75142d9eeb20c" target='_blank'>
              Bypassing the guardian: regulated cell death pathways in p53-mutant cancers
              </a>
            </td>
          <td>
            Jonathan Y Chung, Bruce A Knutson
          </td>
          <td>2025-06-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Senescence has been shown to contribute to the progression of aging related diseases including degenerative disc disease (DDD). However, the mechanisms regulating senescence in the intervertebral disc (IVD) and other tissues/diseases remain poorly understood. Recently, in a CRISPRa genome-wide screen, our lab identified a previously uncharacterized zinc finger protein, ZNF865 (BLST), that regulates a wide array of genes related to protein processing, cell senescence and DNA damage repair. Here, we demonstrate that ZNF865 expression is correlated with age and disease state in human patient IVD samples and mouse IVD. Utilizing CRISPR-guided gene modulation, we show that ZNF865 is necessary for healthy cell function and is a critical protein in regulating senescence and DNA damage in intervertebral disc cells, with implications for a wide range of tissues and organs. We also demonstrate that downregulation of ZNF865 induces senescence and upregulation mitigates senescence and DNA damage in human nucleus pulposus (NP) cells. Importantly, upregulation of ZNF865 shifts the chromatin landscape and gene expression profile of human degenerative NP cells towards a healthy cell phenotype. Collectively, our findings establish ZNF865 as a novel modulator of genome stability and senescence and as a potential therapeutic target for mediating senescence/DNA damage in senescence related diseases and disorders. Summary Degenerative disc disease (DDD) is a major contributor to chronic low back pain, a leading cause of disability globally1–3. Cellular senescence has emerged as a key driver of disc degeneration4,5, characterized by cell-cycle arrest and the secretion of pro-inflammatory and matrix-degrading factors collectively termed the senescence-associated secretory phenotype (SASP). While the pathological role of senescent cells in musculoskeletal aging is increasingly recognized6–8, the upstream molecular regulators remain poorly understood. Here we identify a previously uncharacterized zinc finger protein, ZNF865, as a novel regulator of senescence and genomic stability in human nucleus pulposus (NP) cells. CRISPRi-mediated downregulation of ZNF865 in healthy NP cells induced senescence, increased expression of p16 and p21, and led to increases in DNA damage. Conversely, upregulation of ZNF865 in degenerative NP cells restored proliferation, suppressed senescence markers, reduced DNA damage, significantly diminished SASP factor secretion and restored transcriptomic and epigenetic profiles to a healthy phenotype. This study represents the first functional characterization of ZNF865 and establishes it as an important regulator for senescence in disc cells. These findings highlight ZNF865 as a promising therapeutic target for mitigating senescence-driven pathologies in DDD and potentially other age-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D. Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="This Editorial article intends to unravel the relationships among cellular ageing, epigenetic changes, and tumorigenesis, thereby offering perspectives that could improve therapeutic approaches in cancer management and promote future research on these topics. Furthermore, selected fundamental principles concerning cellular ageing will be presented to elucidate how this process contributes to the comprehension of tumorigenesis. As humans age, there is a progressive decline in physiological functions, which significantly increases the risk of cancer. Epigenetic alterations-heritable yet reversible modifications of the genome without changes in DNA sequence-play a pivotal role in both ageing and tumorigenesis. Age-associated epigenetic drift, involving widespread DNA methylation changes, histone modification shifts, and chromatin remodelling, disrupts normal gene regulatory networks, leading to genomic instability and impaired cellular homeostasis. Additionally, the accumulation of senescent cells, driven by epigenetic dysregulation, fosters a pro-inflammatory environment that can promote tumorigenesis. Moreover, the epigenetic landscape of aged tissues resembles that of cancerous tissues, suggesting that ageing establishes a permissive environment for malignant transformation. Understanding the interplay between ageing, epigenetic regulation, and cancer is critical for the development of preventive strategies and novel therapeutics. Epigenetic reprogramming technologies, aiming to restore youthful epigenetic states, hold promise for delaying ageing and reducing cancer incidence. However, challenges remain in selectively targeting pathogenic epigenetic changes without disrupting essential cellular functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e7c46509e5629fb53472a6dff6527ea523bab0" target='_blank'>
              Unveiling the Relation between Cellular Ageing, Epigenetics and Cancer.
              </a>
            </td>
          <td>
            Paweł Kordowitzki, Arkadiusz Grzeczka
          </td>
          <td>2025-06-27</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies have revealed that polyclonality—where multiple distinct subclones cooperate during early tumor development—is a critical feature of tumor evolution, as demonstrated by Sadien et al. and Lu et al. in Nature (October 2024). These findings show that early polyclonal interactions can overcome fitness barriers, ultimately transitioning to monoclonality as dominant clones emerge. Understanding and targeting these interclonal dynamics offers new therapeutic opportunities. In this perspective, we outline how computational modeling and artificial intelligence (AI) tools can provide deeper insights into tumor polyclonality and identify actionable therapeutic strategies. By applying ligand–receptor interaction analysis, clonal trajectory reconstruction, network and pathway modeling, and spatial analysis, researchers can prioritize communication hubs, evolutionary bottlenecks, and microenvironmental niches that sustain tumor progression. These approaches, when integrated with experimental validation, offer a translational pathway from foundational discoveries to personalized cancer treatments aimed at disrupting cooperative subclonal ecosystems and preventing malignant progression. We commend the recent Nature publications, “Polyclonality overcomes fitness barriers in Apc-driven tumorigenesis” by Sadien et al. [1] and “Polyclonal-to-monoclonal transition in colorectal precancerous evolution” by Lu et al. [2], both featured on 2024 October 30. These groundbreaking studies employed distinct lineage tracing methods to investigate the origins and evolutionary dynamics of colorectal and intestinal tumorigenesis. Despite their different approaches, both studies reached convergent conclusions: Polyclonality plays a pivotal role in the early stages of tumor development, providing critical insights into how diverse cellular populations collaborate to overcome fitness barriers and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6580f8163812203e6792c0b0ae89c7c42827a3" target='_blank'>
              Artificial Intelligence-Powered Insights into Polyclonality and Tumor Evolution
              </a>
            </td>
          <td>
            Hong Zhao, T. Ideker, Stephen T C Wong
          </td>
          <td>2025-06-13</td>
          <td>Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Abstract Despite being a consolidated practice in modern cancer genomics, mutational signature analysis poses various challenges. First, determining the number of signatures is a complex task and depends on heuristics. Second, several signatures lack a clear etiology, raising concerns about whether they result from computational artifacts or actual mutagenic processes. Last, approaches for signature assignment are highly influenced by the catalogue of signatures used for the analysis. To overcome these limitations, we introduce RESOLVE (Robust EStimation Of mutationaL signatures Via rEgularization), a framework designed for the efficient extraction, assignment, and confidence estimation of mutational signatures. RESOLVE enables the stratification of cancer genomes according to the active mutational processes, providing insights into those that are consistently shared among various cancer types. We applied RESOLVE to 20 000 samples from adult and pediatric cancers, providing a comprehensive characterization of subtypes associated with specific mutational processes and alterations in driver genes. Our analysis demonstrates that RESOLVE accurately fits observed mutations with a smaller set of signatures compared to existing catalogues, suggesting the existence of a limited number of dominant mutational processes in cancer genomes. Clustering analysis revealed distinct patient groups characterized by specific signatures, with significant associations between certain signatures and patient prognosis. Additionally, we identified strong associations between signatures and driver gene mutations, offering insights into cancer subtype mechanisms and evolution. Our findings highlight the efficiency of RESOLVE and its potential impact on personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f33c3f703237609301199e20410a7b8d03b3cefb" target='_blank'>
              Comprehensive analysis of mutational processes across 20 000 adult and pediatric tumors
              </a>
            </td>
          <td>
            M. Villa, Federica Malighetti, Luca De Sano, Alberto Maria Villa, N. Cordani, A. Aroldi, M. Antoniotti, G. Caravagna, Alex Graudenzi, R. Piazza, L. Mologni, Daniele Ramazzotti
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Oncolytic reovirus is a potential therapeutic for colorectal cancer patients with a mutant KRAS gene. Reovirus hijacks the autophagic machinery and preferentially induces apoptosis in patients with a KRAS mutation. However, reovirus on its own is not currently a viable treatment and requires enhancement with combination therapies. Carbamazepine, an FDA-approved drug in use for epilepsy, is an autophagy inducer and is used in this study in conjunction with reovirus. The dual treatment was able to reduce cancer viability in mutated KRAS cell lines and was more effective than with reovirus alone. Carbamazepine and reovirus increased autophagy-related proteins and mRNA in mutant KRAS compared to wildtype KRAS which is crucial for autophagy-induced apoptosis. Transmission electron microscopy results show increased autophagosome formation in the combination therapy, as well as a decrease in condensed chromatin. The combination therapy effectively increased the apoptosis induced by reovirus alone and is a viable treatment for patients with a mutant KRAS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401501608c8aeb4d407aac8234f18f1e96bb8ca1" target='_blank'>
              Modulating autophagy in KRAS mutant colorectal cancer using combination of oncolytic reovirus and carbamazepine
              </a>
            </td>
          <td>
            Aaron Shaykevich, Danbee Chae, Isaac Silverman, Sanjay Goel, R. Maitra
          </td>
          <td>2025-06-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oncogenic mutations of Ras are among the most common genetic alterations in human cancer, with an estimated disease burden of >3 million new patients per year worldwide. Despite widespread appreciation of the importance of Ras in cancer, direct binding ligands, which block downstream signaling, were not reported until 2013 due to the lack of obvious drug binding pockets in the protein. The clinically approved K-Ras inhibitors are mutant-selective as they rely on covalent recognition of the highly nucleophilic somatic cysteine residue of K-Ras(G12C). Recent preclinical reports of noncovalent K-Ras binding inhibitors have emerged, which lack mutant specificity and exhibit varying degrees of biochemical preference for mutant K-Ras over the wild-type. An adjacent glycine-13 mutation, p. G13C, particularly abundant in lung, colorectal, and pancreatic cancer, has not been targeted with an approved therapeutic molecule. Here, we report a series of targeted electrophiles designed to covalently modify Cys13 in K-Ras(G13C), overcoming the structural challenge posed by its shifted position relative to Cys12 in K-Ras(G12C). These inhibitors effectively alkylate K-Ras(G13C) in both GDP- and GTP-bound states, block effector interactions, and suppress the growth of K-Ras(G13C)-mutation cancer cell lines. Our findings expand the landscape of covalent K-Ras inhibitors beyond G12 mutations, providing a new therapeutic strategy for K-Ras(G13C)-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7630138c5c420d8de4571d182b52233bbb51cc61" target='_blank'>
              Distal Covalent Targeting Suppresses Signaling of Oncogenic K-Ras(G13C) in Cancer Cells.
              </a>
            </td>
          <td>
            Qinheng Zheng, Tianfang Shen, Julius Pampel, K. Shokat
          </td>
          <td>2025-07-09</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Burkitt Lymphoma (BL), an aggressive B-cell lymphoma driven by MYC translocations, requires intensive chemotherapy treatments which deliver high effectiveness yet increase future risks of developing secondary malignancies. We have previously shown that DDX3X, an RNA helicase frequently mutated in BL, is essential for B cell lymphomagenesis in mice.To assess if DDX3X could therefore represent a promising therapeutic target for BL, we tested two DDX3X inhibitors, the well characterized RK-33 and the more potent newly developed C1, in three BL cell lines (CA46, Raji, Daudi). We found that the 3 cell lines exhibited differential sensitivities to the drugs in vitro, with Daudi being the most susceptible and Raji the most resistant. In vivo, RK-33 treatment in a xenograft BL model reduced tumor progression in all cell lines, albeit with variable efficacy compared to the clinical drug Pevonedistat, and again with the Daudi cells being the most responsive to the treatment. Transcriptomic and proteomic analyses indicated that RK-33-mediated inhibition of DDX3X, and DDX3X ablation through siRNA affects oxidative phosphorylation among other pathways and leads to an increase of intracellular reactive oxygen species (ROS). A CRISPR chemogenomic screen to identify synthetic lethalities linked to RK-33 implicated enzymes of the glutathione synthesis pathway and the Keap1-Nrf2-ARE pathway. We therefore tested the inhibition of the glutathione pathway with buthionine sulfoximine and showed that it reduced the CC50 of RK-33 in BL cells lines.Our findings not only support DDX3X as a therapeutic target in BL but also provide evidence for a combinatorial treatment strategy to improve the efficacy of current treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af8e47b46f9074e8482f490db8d4008cd6713d8" target='_blank'>
              Inhibiting the RNA helicase DDX3X in Burkitt lymphoma induces oxydative stress and impedes tumor progression in xenografts
              </a>
            </td>
          <td>
            H. Beauchemin, Zeinab Dalloul, Eva-Maria Piskor, Virginie Calderon, Andrew Chatr-Aryamontri, Thierry Bertomeu, T. Möröy
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="VIRMA (also known as KIAA1429), as a core regulatory subunit of the m6A methyltransferase complex, plays a key role in tumorigenesis and progression by dynamically regulating RNA methylation modifications. Studies have shown that VIRMA is aberrantly overexpressed in more than 20 types of malignant tumors, including liver cancer, breast cancer, and lung cancer, and is significantly associated with chromosome 8q amplification and poor prognosis. Its mechanism of action involves regulating the expression of tumor-associated genes through both m6A-dependent and m6A-independent pathways, thereby promoting tumor proliferation, metastasis, and drug resistance. These findings suggest that VIRMA has the potential to serve as a pan-cancer diagnostic and prognostic biomarker. This review summarizes the role of VIRMA in malignant tumors from multiple perspectives and explores its potential applications in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1f0fb758f358894db60bb1aa6efc139d365cc4" target='_blank'>
              The Multifaceted Role of VIRMA, a Core Component of the Methyltransferase Complex, in Cancer and Cancer Therapy
              </a>
            </td>
          <td>
            Jinmeng Lu, Chengyu Zhang, Mengshuang Yin, Huili You, Chao Xiong, Jing Wu, Ying Gong, Zhangang Xiao, Jing Shen
          </td>
          <td>2025-06-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="While switch-like gene expression (“on” in some individuals and “off” in others) has been linked to biological variation and disease susceptibility, a systematic analysis across tissues is lacking. Here, we analyze genomes, transcriptomes, and methylomes from 943 individuals across 27 tissues, identifying 473 switch-like genes. The identified genes are enriched for associations with cancers and immune, metabolic, and skin diseases. Only 40 (8.5%) switch-like genes show genetically controlled switch-like expression in all tissues, i.e., universally switch-like expression. The rest show switch-like expression in specific tissues. Methylation analysis suggests that genetically driven epigenetic silencing explains the universally switch-like pattern, whereas hormone-driven epigenetic modification likely underlies the tissue-specific pattern. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, driven by tissue-specific master regulators. In the vagina, we identified seven concordantly switched-off genes linked to vaginal atrophy in females. Experimental analysis of vaginal tissues shows that low estrogen levels lead to decreased epithelial thickness and ALOX12 expression. We propose that switched-off driver genes in basal and parabasal epithelia suppress cell proliferation, leading to epithelial thinning and vaginal atrophy. Our findings underscore the implications of switch-like genes for diagnostic and personalized therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a8bc142a1fa908c8d231b3fb99478b1a7a0c89" target='_blank'>
              Switch-like gene expression modulates disease risk
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kallak, M. Saitou, Omer Gokcumen, Naoki Masuda
          </td>
          <td>2025-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Alternative splicing (AS) dysregulation is increasingly recognized as a critical factor in cancer progression and drug response. However, precisely detecting and characterizing complex splicing events, particularly those involving microexons, remains technically challenging. Ribosomal Protein 24 (RPS24), which contains three microexons (3, 18, and 22 bp), serves as an ideal model for studying complex AS regulation in cancer. We developed a high-resolution detection method for RPS24 microexon variations and investigate their relationship with KRAS proto-oncogene, GTPase (KRAS) inhibition in lung adenocarcinoma (LUAD) to identify potential biomarkers for KRAS-targeted inhibitors. We established an integrated methodological approach combining RNA-seq analysis with fragment analysis to detect RPS24 AS patterns. Using this method, we analyzed RPS24 AS across a panel of lung cancer cell lines and examined AS changes in KRAS-mutant cell lines following treatment with KRAS inhibitors. Our method successfully characterized distinct RPS24 AS isoform compositions across lung cancer cell lines, demonstrating high accuracy in detecting 3 bp variations. In KRAS-mutant cell lines, we observed a consistent upregulation of the 3 bp-containing isoform following KRAS inhibition, indicating a specific correlation with treatment response. This study provides a robust methodology for analyzing complex AS events and supports the RPS24 3 bp-containing isoform as a potential biomarker for KRAS inhibitor response in LUAD. These findings offer new insights into the molecular mechanisms of KRAS inhibitor therapy and strategies for monitoring treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21ce2d06805a31042a8c5858c7a0eac61b833f6" target='_blank'>
              High-resolution detection of RPS24 microexon variations reveals novel splicing patterns in response to KRAS-targeted therapy in lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Nam, Bin Tian, Jiyeon Park, Yeun-Jun Chung
          </td>
          <td>2025-06-10</td>
          <td>BMB Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>