<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-20 09:30:45 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>62</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells exhibit reprogrammed metabolic pathways to sustain aggressive phenotypes, including continuous cell division, stemness, invasion, and metastasis. Emerging evidence suggests that these metabolic adaptations profoundly impact DNA repair pathways, which contribute to the responses to therapy and influence overall outcomes. Metabolic processes such as the Warburg effect, nicotinamide adenine dinucleotide (NAD) metabolism, glutamine metabolism, and one-carbon metabolism support DNA repair by expanding the metabolite pool and facilitating post-translational modifications. Conversely, oncometabolites impair DNA repair pathways through epigenetic reprogramming, thereby promoting genomic instability. This review highlights recent discoveries that elucidate the intricate connections between metabolic hallmarks in cancer cells and DNA repair mechanisms, offering insights into potential therapeutic targets for future cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0683240e558e0cfd022d73899ecd3cc85f65ee47" target='_blank'>
              Cancer Metabolism Meets DNA Repair: The Hidden Link to Therapy Resistance.
              </a>
            </td>
          <td>
            Chunzhang Yang
          </td>
          <td>2025-09-28</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species. Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures. Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign. Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            Saeid Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01966-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chemotherapy is a foundational element in cancer therapy; however, the development of resistance to antineoplastic agents presents a significant challenge to achieving enduring therapeutic triumphs. The acquisition of drug resistance by malignant cells is the result of a complex interplay between inherent and acquired mechanisms. This phenomenon leads to a reduction in pharmacological effectiveness and contributes to treatment failure. This review investigates the primary mechanisms contributing to chemoresistance in specific malignancies, namely lung, pancreatic, and thyroid cancers, along with non-Hodgkin lymphoma. The primary focus of this research is on genetic modifications, interactions within the tumor microenvironment, and cellular adaptations at the molecular level. The present study places particular emphasis on the contributions of drug efflux pumps, DNA repair pathways, epithelial-mesenchymal transition (EMT), microRNAs, and the suppression of apoptosis in mediating this resistance. In addition, we examine promising approaches to counteract drug resistance, encompassing multi-agent regimens, inhibitors targeting specific molecular pathways, and innovative therapeutic modalities. A comprehensive understanding of these fundamental mechanisms is imperative for the development of sophisticated therapeutic interventions that not only improve patient outcomes but also effectively address resistance. The resistance pathways and prospective translational approaches discussed are drawn from both preclinical and clinical investigations, providing a thorough perspective. Keywords: The following terms are relevant to the study: chemoresistance, tumor microenvironment, molecular pathways, targeted therapy, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec567f4501baf205432eef42dce8eb9f17d9a8c" target='_blank'>
              Mechanisms of Chemoresistance in Solid and Hematologic Malignancies: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            Badiaa Batlamous, Boutaina Elgharbaoui, Imane Bensalim, Mohamed Khalis
          </td>
          <td>2025-09-24</td>
          <td>Middle Eastern Cancer and Oncology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>82</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The recurrence of clinically advanced cancers is an evolutionary consequence of standard-of-care chemotherapies generally administered at maximum tolerated doses to kill as many cancer cells as possible. The inevitable appearance of resistance raises the possibility of shifting treatment goals from complete tumor eradication to long-term disease control. The latter approach is employed by adaptive therapy, which aims to inhibit the evolutionary dynamics governing the spread of resistant tumor phenotypes. Adaptive therapy changes focus from the cancer cells that are responsive to therapy to those that are resistant and ultimately govern outcome. This therapeutic approach retains a pool of sensitive cancer cells to compete with the therapy-resistant ones through dynamic dose modulation and/or timing. Thus, fluctuations of treatment-sensitive cells are used to control the resistant population and prolong tumor control with existing therapy agents. Here, we explore non-genetic mechanisms of resistance, including the protective role of the tumor stroma, the epithelial-to-mesenchymal transition, the overexpression of drug efflux pumps, and the extracellular vesicle-mediated transfer of them. These mechanisms can increase the size of the resistant population at the expense of the sensitive one, reducing the ability of adaptive therapy to force tumor evolution into controllable cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The human genome is partitioned into functional compartments that replicate at specific times during the S-phase. This temporal program, referred to as replication timing (RT), is co-regulated with the 3D genome organization, is cell type-specific, and changes during development in coordination with gene expression. Moreover, RT alterations are linked to abnormal gene expression, genome instability, and structural variation in multiple diseases, including cancer. However, mechanistic links between RT, large-scale 3D genome architecture, and transcriptional regulation remain poorly understood. A major limitation is that current approaches require the separate profiling of RT and transcriptomes from independent batches of samples, obscuring the complex co-regulation between the epigenome and transcriptome. Here, we developed PARTAGE, a multiomics approach that enables joint profiling of copy number variation (CNV), RT, and gene expression from the same sample, providing a more accurate integrative view of the complex relationships between RT and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957eda64f84a18622679de092756bc343443ffa1" target='_blank'>
              PARTAGE: Parallel analysis of replication timing and gene expression
              </a>
            </td>
          <td>
            Lakshana Sruthi Sadu Murari, Quinn Dickinson, Silvia Meyer-Nava, J. Rivera-Mulia
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa96a0707b7c6c78fe5bd4b2a45727093019bee" target='_blank'>
              Targeting the crosstalk of metabolism reprogramming and replication stress: novel strategy to combat cancer.
              </a>
            </td>
          <td>
            Wenbo Liu, Xiangyan Jiang, Yong Ma, Huiguo Qing, Yingcun Bao, Zuoyi Jiao
          </td>
          <td>2025-09-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The discovery of genomics has redefined oncology into precision rather than standardized treatment. But
therapeutic resistance of either kind, intrinsic or acquired, is a serious impediment to sustained success. The present review
outlines the domains on the crossroad between genomic intelligence and resistance evolution, where multi-omics profiling,
high-throughput sequencing, and AI-based analytics are explaining the challenging complexity of the tumor and predicting
resistance pathways. We review the contribution of clonal evolution, tumor plasticity and adaptive signaling in resistance to
therapy, and we promote moving towards active molecular monitoring rather than snapshot ting genome. To preempt and
target therapeutic escape we suggest a framework of adaptive precision oncology that comprehends real-time biomarkers,
liquid biopsy follow-up and resistance-predictive algorithms. New approaches like combination therapy, recalibration of
treatment with the help of AI, and longitudinal monitoring of genomes are mentioned. We also deal with moral and logistical
obstacles of adopting these strategies, and especially in low-resource environments. The new era of precision oncology We
now find ourselves in the next era of precision oncology that aims to personalization extend to real-time responsiveness, so
that cancer treatment can increasingly be more adaptable, more resilient and more durable, achieved through personalizing
the rapidly changing tumor biology with real-time dynamically changing genomic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7508321bc4abb3899cb6fc37344230d9a86147e" target='_blank'>
              Genomic Intelligence and Resistance Evolution: Redefining Oncotherapeutic Strategies in Precision Oncology
              </a>
            </td>
          <td>
            Muchukota Sushma, Bharathi Bhogenahalli Venkatappa, Gowthami V., Nigel Viju Thomas, Souman Samanta
          </td>
          <td>2025-08-28</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study explores the molecular impact of cisplatin, Actemra (tocilizumab), remdesivir, and SARS-CoV-2 infection on colorectal cancer (CRC) SW-480 cells using RNA sequencing (RNA-Seq). Differential gene expression analysis revealed treatment-specific transcriptional changes. Cisplatin led to the downregulation of genes involved in lipid metabolism and focal adhesion, while remdesivir upregulated chromatin remodeling pathways. SARS-CoV-2 infection altered cytokine signaling, particularly IL-17 and TNF-α. Principal component and hierarchical clustering analyses confirmed distinct gene expression profiles across treatments. ELISA assays validated the reduction of ACE2 and CD147 protein levels following drug treatment. MTT assays demonstrated remdesivir’s cytotoxicity at high doses. These findings highlight potential drug-gene interactions and suggest that combining transcriptomic profiling with functional assays may guide personalized therapeutic strategies for CRC patients, especially those co-infected with SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eae066da47093f217085c9697d40cc942624f2a" target='_blank'>
              RNA sequencing reveals transcriptional signatures of drug response and SARS-CoV-2 interaction in colorectal cancer cells
              </a>
            </td>
          <td>
            Alaa K. Hameed, Ifad K. Abd Al-Shibly, Rana A. Ghaleb
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) is a key biological process driving tumor invasion and metastasis, and its activation is widely regarded as one of the core mechanisms leading to tumor resistance to multiple therapeutic approaches. Current research indicates that EMT mediates drug resistance through multiple pathways, such as conferring stem cell-like properties to cancer cells, miRNA delivery via exosomes, and promoting drug efflux. The underlying mechanisms of related signaling pathways and key transcription factors are being intensively investigated. However, effectively translating the basic research findings targeting EMT to overcome drug resistance into clinical applications still faces significant challenges. This article systematically reviews the core molecular mechanisms by which EMT mediates tumor drug resistance, with a focus on how EMT confers drug-resistant phenotypes to cancer cells by regulating tumor stem cell characteristics and drug pharmacokinetics. It also reviews experimental strategies targeting key EMT molecules to reverse drug resistance and their preclinical/clinical research evidence. This review provides a theoretical framework for a deeper understanding of the role of EMT in tumor drug resistance and lays the foundation for developing novel combination treatment regimens based on EMT intervention. Future research urgently needs to explore more selective EMT-targeting strategies, identify reliable predictive biomarkers, and optimize their synergistic effects with traditional radiotherapy, chemotherapy, and immunotherapy to ultimately improve the prognosis of patients with refractory tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8627c99bba513b22e7affef08f860cf5a9870601" target='_blank'>
              Research on Tumor Drug Resistance Mediated by Epithelial-mesenchymal Transition
              </a>
            </td>
          <td>
            You Wu
          </td>
          <td>2025-08-26</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A complex and gradual process, the epithelial–mesenchymal transition (EMT) occurs both during embryonic development and tumor progression. Cells undergo a transition from an epithelial to a mesenchymal state throughout this process. More and more evidence points to EMT as a cause of increased metastatic spread of prostate cancer (PCa), along with stemness enhancement and therapy resistance. Here, we used bioinformatic methods to analyze gene expression microarray data, single-cell RNA sequencing, oncogenes, and tumor suppressor genes (TSGs) in order to reconstruct the network of differentially expressed genes (DEGs) involved in the epithelial–mesenchymal transition with PCa. No prior study has documented this sort of analysis. We next validated our results using data from the Cancer Genome Atlas (TCGA), which included microarray and single-cell RNA sequencing. Potentially useful in PCa diagnosis and treatment are extracellular matrix in epithelial–mesenchymal transition genes, including ITGBL1, DSC3, COL4A6, ANGPT1, ARMCX1, MICAL2, and EPHA5. In this study, we aimed to shed light on the molecular characteristics and pathways of DEGs in PCa, as well as to identify possible biomarkers that are important in the development and advancement of this cancer. These insights have important implications for understanding prostate cancer progression and for the development of therapeutic strategies targeting ECM-mediated pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15775ad90d8cab3fed2c521129b0f7f8c9fd8d19" target='_blank'>
              Analysis of Microarray and Single-Cell RNA-Seq Finds Gene Co-Expression and Tumor Environment Associated with Extracellular Matrix in Epithelial–Mesenchymal Transition in Prostate Cancer
              </a>
            </td>
          <td>
            Ali Shakeri Abroudi, Mahtab Mashhouri Moghaddam, Danial Hashemi Karoii, Melika Djamali, Hossein Azizi, Thomas Skutella
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Genetic variants (both coding and noncoding) can impact gene function and expression, driving disease mechanisms such as cancer progression. The systematic study of endogenous genetic variants is hindered by inefficient precision editing tools, combined with technical limitations in confidently linking genotypes to gene expression at single-cell resolution. We developed single-cell DNA–RNA sequencing (SDR-seq) to simultaneously profile up to 480 genomic DNA loci and genes in thousands of single cells, enabling accurate determination of coding and noncoding variant zygosity alongside associated gene expression changes. Using SDR-seq, we associate coding and noncoding variants with distinct gene expression in human induced pluripotent stem cells. Furthermore, we demonstrate that in primary B cell lymphoma samples, cells with a higher mutational burden exhibit elevated B cell receptor signaling and tumorigenic gene expression. SDR-seq provides a powerful platform to dissect regulatory mechanisms encoded by genetic variants, advancing our understanding of gene expression regulation and its implications for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a9aba01e0cb2e8c3213b471e7d8f34414b59cc" target='_blank'>
              Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing
              </a>
            </td>
          <td>
            Dominik Lindenhofer, Julia R. Bauman, John A. Hawkins, Donnacha Fitzgerald, Umut Yildiz, Haeyeon Jung, Anastasiia Korosteleva, Mikael Marttinen, M. Kueblbeck, Judith B. Zaugg, K. Noh, Sascha Dietrich, W. Huber, O. Stegle, Lars M. Steinmetz
          </td>
          <td>2025-09-01</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Introduction Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations. Methods To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation. Results Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies. Discussion This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chemotherapy is the most widely used and effective treatment method in today's cancer treatment. However, the rise of cancer drug resistance has limited chemotherapy effectiveness. The molecular mechanisms and related genes by which many cancer cells acquire resistance to chemotherapy drugs remain to be identified. Recent advancements of CRISPR-Cas9 genome-editing techniques have drawn rigorous exploration of precision oncology applications. CRISPR-Cas9 technology can knock out, inhibit or activate the expression of specific genes. By observing the phenotype of gene-edited cells, drug-resistant genes in carcinoma cells can be identified. This paper will decipher the molecular basis of CRISPR-Cas9 technology in targeted gene modification and give examples to illustrate its application in the screening of drug resistance genes in three types of human cancers. Additionally, the application limitations and emerging horizons of CRISPR-Cas9 gene modification technique in strategies for overcoming anticancer drug resistance are discussed. The continuous development and optimization of this technology are expected to provide strong support for breaking through the bottlenecks in tumor treatment and pushing precision cancer medicine to new heights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0492def30740e765a8ca58409631a4cc538a87f" target='_blank'>
              CRISPR-Cas9 Gene-Editing Technology for the Screening of Drug Resistance Genes in Tumor Cells
              </a>
            </td>
          <td>
            Zisheng Zeng
          </td>
          <td>2025-09-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance limits the therapeutic efficacy of osimertinib in lung adenocarcinoma (LUAD). Redox homeostasis is crucial for LUAD progression. However, how redox imbalance interacts with the tumor microenvironment (TME) to drive osimertinib resistance (OR) remains unclear.


METHODS
The single-cell RNA sequencing (scRNA-seq) data GSE243562 were combined with the Cancer Genome Atlas (TCGA)-LUAD transcriptomes to map the TME cell population heterogeneity in osimertinib-resistant LUAD. Through univariate Cox regression and least absolute shrinkage and selection operator (LASSO) regularization, a prognostic signature founded on redox-related genes (RRGs) was built. Therapeutic compounds targeting these signature genes were prioritized by molecular docking. Their expression patterns were subsequently validated in vitro.


RESULTS
Cancer-associated fibroblasts (CAFs) were central hubs in the TME of osimertinib-resistant LUAD, exhibiting enhanced intercellular communication. Computational profiling identified 10 differentially expressed RRGs, predominantly enriched in CAFs. Using a six-gene signature comprising AGER, CYP2J2, FMO2, HSPA1B, SOD3, and VASN, we categorized LUAD patients into separate risk categories. High-risk patients showed significantly reduced survival, an immunosuppressive status, and a higher tumor mutation burden (p < 0.05). The overexpression of these six genes was confirmed in OR cells. Critically, inhibiting SOD3 restored osimertinib sensitivity in vitro (p < 0.05). Clinically, SOD3 expression was lower in patients sensitive to third-generation epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs) than in those with resistant disease.


CONCLUSIONS
Targeting CAFs represents a promising strategy to overcome osimertinib resistance. Our six-gene redox signature offers a clinical framework for patient risk stratification and novel therapeutic strategy design. Future work will explore these targets to develop new treatments for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1eb1b2e535167099275ea725fe5cce81f5d8d82" target='_blank'>
              Integrated Analysis of Single-cell and Bulk-RNA Sequencing Data to Identify Redox-related Genes as Prognostic Biomarkers and Therapeutic Targets of Lung Adenocarcinoma With Osimertinib Resistance.
              </a>
            </td>
          <td>
            Chen Fang, Chaoxing Liu, Rong Qi, Jiabin Ding, Ting Luo, Feng Yu, Guohua Zhang, Chao Shi, Daya Luo, Feng Qiu
          </td>
          <td>2025-09-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian cancer (OC) is lethal due to near universal development of resistance to platinum-based chemotherapy. Metabolic adaptations can play a pivotal role in therapy resistance. Here, we aimed to identify key metabolic pathways that regulate platinum response and represent potential therapeutic targets. Transcriptomic and metabolomic analyses in cisplatin sensitive and resistant ovarian cancer cells identified enrichment of pyrimidine metabolism related to upregulated de novo pyrimidine synthesis. 15N-glutamine flux analysis confirmed increased de novo pyrimidine synthesis in cisplatin resistant cells. Targeting this pathway using brequinar (BRQ), an inhibitor of the key enzyme dihydroorotate dehydrogenase (DHODH), decreased cell viability, delayed G2/M cell cycle progression, and altered expression of genes related to mitochondrial electron transport in resistant cells. Under basal conditions, cisplatin resistant cells had a lower oxygen consumption rate (OCR) and spare respiratory capacity (SRC) than sensitive cells. BRQ suppressed OCR in both sensitive and resistant but only inhibited SRC in resistant cells. In cell line-derived and patient-derived xenograft models, BRQ attenuated the growth of cisplatin resistant ovarian tumors and enhanced the inhibitory effects of carboplatin. Together, these results identify metabolic reprogramming in cisplatin resistant ovarian cancer that induces an acquired dependency on de novo pyrimidine synthesis, which can be targeted to sensitize tumors to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d708b6c5cad44de1ded3e0fd0aa39875db711a4c" target='_blank'>
              Metabolic Dependency on De Novo Pyrimidine Synthesis is a Targetable Vulnerability in Platinum Resistant Ovarian Cancer.
              </a>
            </td>
          <td>
            H. Cardenas, Yinu Wang, Guangyuan Zhao, Delan Xingyue Hao, A. Isac, Vanessa Hernandez, Ujin Kim, Wenan Qiang, Hao F Zhang, Daniela Matei
          </td>
          <td>2025-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND: Currently, identification of breast cancer stem cells (BCSCs) commonly relies on CD24-/CD44+ expression profiles. However, few studies have integrated genomic mutation data with experimental gene expression validation in CSC and non-CSC populations. Genotyping results of CD24-/CD44+ MDAMB-231 cells revealed 36 mutations in BCSCs compared to non-BCSCs, with upregulated FOXO1 and FYN that might represent promising candidate biomarkers for this subpopulation. Therefore, in this study, single nucleotide polymorphism (SNP) and quantitative polymerase chain reaction (qPCR) analysis were performed to assess the association between mutations and expression trends of FOXO1 and FYN in MDAMB-231 cell, as breast cancer cell model with stem-like traits and well-characterized profile.METHODS: Genomic DNA was isolated from BCSC and non-BCSC DNA from the MDAMB-231 cell line. Mutation analysis was conducted using PLINK, while gene expressions of FOXO1 and FYN were quantified by one-step SYBR Green-based qPCR, using 18s rRNA as a reference. Data was then analyzed with the Livak (2−∆∆Ct) method.RESULTS: Among 36 mutations found in BCSCs of the MDAMB-231 cell line, PTEN (rs786204914) and CHEK2 (rs587782401) were identified as pathogenic. While FOXO1 (2.989±2.817 vs. 1.072±0.388) and FYN (1.405±0.072 vs. 0.855±0.140) mRNA levels were found to be higher in CSCs compared to non-CSCs, though these differences was not statistically significant.CONCLUSION: Pathogenic mutations in CHEK2 and PTEN were detected within BCSC population, implying a potential influence on the expression of FOXO1 and FYN, though not statistically significant. These findings suggest a possible, but as yet unverified, association between gene mutations and expression patterns, emphasizing the importance of further functional studies to validate FOXO1 and FYN as biomarkers for BCSCs.KEYWORDS: breast cancer stem cells, FOXO1, FYN, PTEN, CHEK2, mutation, biomarker">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5612edfc37bd3adbc0b6810a6197250ec6336f30" target='_blank'>
              FOXO1 and FYN Expression Trends in Breast Cancer Stem Cells: An Integrative Study of Single Nucleotide Polymorphism (SNP) Array and Quantitative PCR (qPCR) Analysis
              </a>
            </td>
          <td>
            Ayorinde Margaret, S. Wanandi, Fadilah Fadilah, R. I. Paramita
          </td>
          <td>2025-08-25</td>
          <td>The Indonesian Biomedical Journal</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy.
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318599f6810efcf5bf3014e92955379d25d16eda" target='_blank'>
              Synthetic lethality in cancer drug discovery: challenges and opportunities.
              </a>
            </td>
          <td>
            Emanuel Gonçalves, Colm J. Ryan, David J. Adams
          </td>
          <td>2025-09-11</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding genetic dependencies in cancer is key to identifying novel actionable drug targets to advance precision medicine. Whole-genome CRISPR-knockout library screening methods have facilitated this goal. Pooled libraries of single guide RNAs (sgRNAs) targeting over 90% of the annotated protein coding genome are used to induce gene knockouts in pre-clinical cancer models. Novel genes of interest are identified by evaluating sgRNA dropout or enrichment following selection pressure application. This method is particularly beneficial for researching cancers where effective treatment strategies are limited. One example of a commonly chemoresistant cancer, particularly at relapse, is the low survival malignancy epithelial ovarian cancer (EOC), made up of multiple histotypes with distinct molecular profiles. CRISPR-knockout library screens in pre-clinical EOC models have demonstrated the ability to predict biomarkers of treatment response, identify targets synergistic with standard-of-care chemotherapy, and determine novel actionable targets which are synthetic lethal with cancer-associated mutations. Robust experimental design of CRISPR-knockout library screens, including the selection of strong pre-clinical cell line models, allows for meaningful conclusions to be made. We discuss essential design criteria for the use of CRISPR-knockout library screens to discover genetic dependencies in cancer and draw attention to discoveries with translational potential for EOC. Supplementary Information The online version contains supplementary material available at 10.1007/s13402-025-01102-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ~19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorectal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, Da-Eun Kim, Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, H. Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, Kyle K.-H. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, Dae-Kyum Kim
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In Mycobacterium tuberculosis, a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            Anna G Green, Mahbuba Tasmin, R. Vargas, M. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The mismatch repair system is critical for correcting base–base mismatches and insertion-deletion loops during DNA replication. Deficiencies in MMR (due to mutations in MLH1, MSH2, MSH6, or PMS2) lead to microsatellite instability and contribute to the development of various cancers, such as Lynch syndrome-related colorectal cancer and sporadic tumors. This review will delve into the molecular basis of MMR deficiency. Additionally, the review will cover diagnostic approaches for detecting MSI and MMR deficiency, including next-generation sequencing and PCR-based methods. The implications for treatment will be discussed, emphasizing immune checkpoint inhibitors (e.g., pembrolizumab) that target tumors with high mutational burdens due to MMR deficiency, as well as novel therapeutic approaches like synthetic lethality exploiting DNA repair vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c3fe776b0060c3cc36839d638f2791e2a3b03a" target='_blank'>
              Mismatch Repair Deficiency and the Role of Non-Canonical Functions in Cancer: Diagnosis and Therapeutic Implications
              </a>
            </td>
          <td>
            Alicja Dąbrowska, Jakub Mastalerz, Zofia Łapińska, I. Deszcz, Agnieszka Chwiłkowska, Nina Rembiałkowska
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Hypoxia is a well-recognized clinical feature of solid tumors, including lung cancer, and is associated with poor prognosis due to its role in promoting resistance to chemotherapy and radiotherapy. To investigate the cellular consequences of hypoxia, we cultured A549 lung adenocarcinoma cells under 1% O2 and examined their growth, cell cycle distribution, and redox status. Hypoxia significantly reduces cell proliferation and induced G1 phase cell cycle arrest, suggesting a cytostatic effect. Biochemical analysis showed a 2.64- and 2.04-fold increase in total and mitochondrial reactive oxygen species (ROS) levels, respectively, along with an elevated total thiol levels under hypoxic conditions compared to normoxia. To assess the reversibility of the hypoxic response, we performed a reciprocal oxygen exposure experiment where cells initially grown under hypoxia were re-exposed to normoxia, and vice versa. To explore the underlying molecular mechanism, we analyzed transcriptomic datasets (GEO accession: GSE48134 and GSE42416) which revealed that hypoxia downregulated key genes involved in energy metabolism (e.g., PDK4, G6PD), cell cycle progression (e.g., CCND1, CDK2), and redox regulation (e.g., GCLM, TXNRD1, NQO1, GCLC). Further, few of redox-related genes were validated by RT-PCR in A549 cells cultured under hypoxia and normoxia for 24 h. Importantly, cyclic hypoxia (intermittent hypoxia-reoxygenation) conditions showed partial restoration of some of these transcripts, supporting the transient nature of hypoxic stress, consistent with our in vitro observations. Furthermore, transcriptome profiles from adenocarcinoma patients (GEO accession: GSE30979) also match our cell line observations. Thus, our results clearly show that hypoxia causes a temporary cell cycle arrest in lung cancer cells, which is reversible when oxygen is restored. Supplementary Information The online version contains supplementary material available at 10.1007/s12032-025-03058-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77bbcdb23f2ddac965c8e06176945c4fad76ded" target='_blank'>
              Hypoxia-drives reversible cell cycle arrest in lung cancer cells via modulation of cellular redox and gene expression
              </a>
            </td>
          <td>
            D. Maurya, Varshita Mehta, Babita Singh
          </td>
          <td>2025-09-29</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα+ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα+ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Chapus Fleur, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Microsatellite-stable metastatic colorectal cancer (MSS mCRC) is currently treated with chemotherapy and targeted agents based on RAS and BRAF mutational status. Although these therapies offer initial benefit, most patients rapidly develop resistance, with fewer than 20% remaining progression-free at two years. This review aims to synthesize emerging evidence on the metabolic mechanisms driving treatment resistance in MSS mCRC, with a particular focus on the immune-metabolic signature (IMMETCOLS) classification. We conducted a comprehensive review of preclinical models, transcriptomic datasets, and clinical trial results addressing metabolic adaptations to chemotherapy and targeted therapies in MSS mCRC. The IMMETCOLS framework defines three metabolic subtypes—IMC1, IMC2, and IMC3—each associated with distinct resistance mechanisms. IMC1 exhibits glycolysis and transforming growth factor-β (TGF-β)-dependent signaling enriched in inflammatory fibroblasts, conferring resistance to chemotherapy. IMC2 relies on oxidative phosphorylation and glutamine metabolism, supporting antioxidant defenses and resistance to both cytotoxic agents and anti-EGFR therapies. IMC3 demonstrates lactate-fueled respiration and pentose phosphate pathway activation, contributing to redox balance, DNA repair, and resistance to targeted therapies such as anti-BRAF or KRAS inhibitors. All subtypes display metabolic plasticity under therapeutic pressure. Emerging clinical data support tailoring targeted therapy combinations based on IMMETCOLS subtype, particularly in BRAF- and HER2-positive populations. Understanding subtype-specific metabolic rewiring in MSS mCRC offers novel opportunities to overcome drug resistance. Targeting the metabolic vulnerabilities defined by the IMMETCOLS signature may improve response durability and inform precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662ed873ba87c81960ffe96e54fa075a4697d04f" target='_blank'>
              Metabolic plasticity drives specific mechanisms of chemotherapy and targeted therapy resistance in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Mariam Rojas, Malena Manzi, S. Madurga, Fernando Enrique García Velásquez, Maira Alejandra Romero, Silvia Marín, Marta Cascante, Joan Maurel
          </td>
          <td>2025-09-23</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is driven by a complex spectrum of somatic mutations and structural variants that contribute to tumor heterogeneity and therapy resistance. In this study, we performed a comparative analysis of short-read Illumina and long-read Nanopore sequencing technologies across multiple CRC sample groups, encompassing diverse tissue morphologies. Our evaluation included general base-level metrics—such as nucleotide ratios, sequence match rates, and coverage—as well as variant calling performance, including variant allele frequency (VAF) distributions and pathogenic mutation detection rates. Focusing on clinically relevant genes (KRAS, BRAF, TP53, APC, PIK3CA, and others), we characterized platform-specific detection profiles and completed the ground truth validation of somatic KRAS and BRAF mutations. Structural variant (SV) analysis revealed Nanopore’s enhanced ability to resolve large and complex rearrangements, with consistently high precision across SV types, though recall varied by variant class and size. To enable direct comparison with the Illumina exome panel, we applied an exonic position reference file. To assess the impact of depth and PCR amplification, we completed an additional high-coverage Nanopore sequencing run. This analysis confirmed that PCR-free protocols preserve methylation signals more accurately, reinforcing Nanopore’s utility for integrated genomic and epigenomic profiling. Together, these findings underscore the complementary strengths of short- and long-read sequencing platforms in high-resolution cancer genomics, and we highlight the importance of coverage normalization, epigenetic fidelity, and rigorous benchmarking in variant discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4037869e56706b3aeec547e05839c0679ce3d95b" target='_blank'>
              Methodological Comparison of Short-Read and Long-Read Sequencing Methods on Colorectal Cancer Samples
              </a>
            </td>
          <td>
            Nikolett Szakállas, Alexandra Kalmár, K. Rada, M. Kucarov, Tamás Richárd Linkner, B. Barták, István Takács, Béla Molnár
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non‑homologous end joining and homologous recombination are well‑characterized pathways for repairing DNA double‑strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end‑joining (alt‑EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt‑EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt‑EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt‑EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt‑EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end‑joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review).
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-16</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract The transcription factor PAX8 is expressed in most ovarian tumors, being associated with increased tumorigenesis. Although recent studies have addressed the gene regulatory functions of PAX8 in ovarian cancer, an integrative analysis of multi-omic and patient data is required to identify the core regulatory network of PAX8 and its prognostic and therapeutic value. Here, we integrate PAX8 chromatin binding and accessibility data in ovarian cancer cells with transcriptomic and patients’ data to gain insight into the core gene regulatory network orchestrated by PAX8 in ovarian tumors. Integration of differential chromatin accessibility, transcription factor binding, and gene expression upon PAX8 knockout provides a core regulatory network that explains most of the genes regulated by PAX8. We combine these target genes with patient expression data and find a PAX8 gene signature associated with tumor stemness, a property related to therapy resistance. Indeed, we show that the PAX8 gene signature predicts disease outcome and response to therapy in ovarian cancer patients. Finally, we validated experimentally our results from bioinformatic analyses, thus reassuring their robustness. Our findings uncover a PAX8 core network that represents a promising strategy for targeted antitumor therapies and open new pathways to fight against ovarian cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcca83817abeffa86ae1659150f6c28d6af09a3f" target='_blank'>
              Integrative multi-omic analysis reveals a PAX8-driven gene network linking tumor stemness to therapy response in ovarian cancer
              </a>
            </td>
          <td>
            J. Santos-Pereira, Amancio Carnero, S. Muñoz-Galván
          </td>
          <td>2025-08-27</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive malignancy of the bile ducts, and resistance to gemcitabine, a first-line chemotherapy, significantly complicates treatment. Despite extensive research, the molecular mechanisms underlying gemcitabine resistance in iCCA are not fully understood. This study aims to identify key genes associated with gemcitabine resistance in iCCA, investigate the role of lactylation, and propose potential therapeutic targets. Methods A comprehensive bioinformatics analysis was conducted using publicly available transcriptomic data from gemcitabine-resistant iCCA cell lines and patient samples. Differential expression analysis was performed to identify upregulated and downregulated genes. GSEA were used to explore relevant molecular pathways. Immune landscape analysis was carried out using CIBERSORT to assess immune cell infiltration in the tumor microenvironment. Key resistance-related genes were identified through Lasso, RF, and SVM-REF analyses. ITGB4 function was further validated in vitro by siRNA knockdown in HUCCT1 and RBE cells, followed by cell viability and apoptosis assays with or without gemcitabine treatment. Results Pathway analysis revealed the involvement of cell cycle regulation, DNA replication, and p53 signaling in gemcitabine resistance. The high group associated with resistance showed significantly worse survival outcomes, with a positive correlation between resistance and lactylation levels. Immune landscape analysis indicated altered immune cell infiltration, including increased M2 macrophages and decreased CD8+ T cells in the high group. Key resistance-related genes, including CDC20, H2AX, HK2, and ITGB4, were identified as critical in drug resistance. Experimentally, ITGB4 knockdown markedly enhanced gemcitabine’s antiproliferative and pro-apoptotic effects on cholangiocarcinoma cells, supporting its role in mediating resistance. Molecular docking revealed Dioscin and Deacetyllanatoside C as potential ITGB4-interacting compounds. Conclusion This study sheds light on the molecular mechanisms of gemcitabine resistance in iCCA, emphasizing lactylation’s role and the significance of immune modulation. ITGB4 is identified as a promising therapeutic target, and the findings suggest that targeting these genes could help overcome resistance in iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea10bda53b7cbe7c506a45067aed62ce5a1a0fe" target='_blank'>
              Integrated transcriptomic analysis identifies lactylation-linked gemcitabine resistance and therapeutic targets in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Wenwei Xie, Jialiang Hu, Hanmei Xu
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Psoriasis is a chronic inflammatory skin disease involving dysregulated immune responses and complex genetic factors. This study combines single-cell RNA sequencing (scRNA-seq), gene expression profiling, and genetic analysis to explore cellular and molecular contributors to psoriasis. Methods Single-cell RNA-seq data (n = 3 psoriasis, n = 2 control; GSE228421) were used for cell-type annotation and functional characterization. T cell subsets were analyzed for differentiation trajectories and cell-cell communication. Differentially expressed genes in mucosal-associated invariant T (MAIT) cells were evaluated by enrichment analysis. Candidate gene causality was tested via eQTL-based Mendelian randomization (MR) and supported by bulk RNA-seq validation. Results MAIT cells were enriched in psoriatic lesions and exhibited strong intercellular interactions. Functional analyses revealed activation of IL6-JAK-STAT3 signaling, TNF-NFκB pathway, and glycolysis in MAIT cells. MR identified RPS20 as a protective factor (OR = 0.5994, p = 0.011) and PFN1 as a potential risk gene (OR = 1.7229, p = 0.037), with PFN1 highly expressed in MAIT cells. Colocalization analysis showed no significant genetic overlap between PFN1 expression and psoriasis risk. Metabolic profiling revealed differential pathway involvement in PFN1+ and PFN1− MAIT cells. Conclusion Our integrative analysis highlights MAIT cells and PFN1 as likely contributors to psoriasis pathogenesis. These findings offer insights into immune and metabolic alterations, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48055881b868991adfc41d78b968062c6c8aae6e" target='_blank'>
              MAIT Cells with High PFN1 Expression Mediate Immune Activation and Metabolic Reprogramming in Psoriasis
              </a>
            </td>
          <td>
            Man-Ning Wu, Yue-Min Zou, Xiang-Nan Zhou, Sangwon Hong, Lei Wang, Yan-Ping Bai
          </td>
          <td>2025-09-09</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer research has traditionally focused on identifying driver genes, those with mutations that initiate tumorigenesis. The Cancer Driver Gene (CDG) paradigm, further supported by the observation of oncogene addiction in tumors, has successfully guided the development of targeted therapies. However, the limitations of this driver-centric view, highlighted by the broad emergence of frequent therapeutic resistance, the presence of driver mutations in healthy tissues or individuals, and the lack of identifiable drivers in many tumors, call for a shift in perspective and clinical practice. The latest network controllability perspective on cancer cells introduced the concept of Cancer Keeper Genes (CKGs) and a CKG-based paradigm for cancer therapeutics. The new concept encompasses the concept of non-oncogene addiction, emphasizing reliance on non-mutated pathways crucial for maintaining oncogenic cellular states. Here, we explore the transition towards a system-level understanding of cancer based on the CKG paradigm, emphasizing the essential role of genes required for tumor maintenance, irrespective of their initiating function or mutational capacity. We discuss clinical implications of this paradigm shift, highlighting the progress made so far and potential of targeting non-driver CKGs, genes involved in processes like DNA damage response, proteostasis, and metabolism, as a promising strategy to overcome therapeutic challenges and achieve more durable cancer control. Targeting these maintenance vulnerabilities represents a critical evolution in precision oncology, moving towards therapies designed to dismantle the networks sustaining malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e80123ffc1524bf35e125c1b2645e60b9bc7703" target='_blank'>
              From Cancer Drivers to Cancer Keepers: Paradigm Shift and Clinical Implications
              </a>
            </td>
          <td>
            Xizhe Zhang, Weixiong Zhang
          </td>
          <td>2025-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT Cancer treatment is an ongoing challenge, as directly targeting oncogenic drivers is often unfeasible in many patients due to the lack of druggable targets. This has led to the exploration of alternative strategies, such as exploiting synthetic lethality (SL) relationships between genes. SL facilitates the indirect targeting of oncogenic drivers, as exemplified by the clinical success of PARP inhibitors against BRCA-mutated tumors. Advances in high-throughput perturbation screens and multi-omics technologies have deepened our understanding of SL relationships, while computational models enhance SL predictions to better reflect biological complexity. However, while numerous experimental and computational methods have been developed to identify SL interactions, difficulties remain in translating these findings into clinical applications. This review combines recent progress on SL relationships in cancer with emerging insights into epigenetic regulation, highlighting how epigenetic drugs (epidrugs) can provide new opportunities for targeted interventions, offering a way to minimize off-target effects and enhance therapeutic precision. To advance SL-based therapies, efforts must focus not only on identifying new SL interactions but also on consolidating existing knowledge and integrating experimental and computational approaches to characterize the vulnerabilities of cancer cells. Strengthening this foundation will be critical for the effective development of SL-based cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a9988cdf76b342d040b8e183dfc1c5b3f9d7baa" target='_blank'>
              Epigenetic synthetic lethality as a cancer therapeutic strategy: synergy of experimental and computational approaches
              </a>
            </td>
          <td>
            Maria Farina-Morillas, Laia Ollé-Monràs, Silvana Ce Maas, Isabel de Rojas-P, Miguel F. Segura, Jose A. Seoane
          </td>
          <td>2025-08-25</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="INTRODUCTION
Breast cancer accounts for one of the leading causes of cancer deaths in women globally. Breast cancer is characterized by molecular heterogeneity, and different patients show various responses to therapy. In addition to hormone therapies, there are targeted agents and immunotherapies, but resistance to therapy and recurrence remain a critical clinical problem for patients.


METHODS
This review is aimed at presenting innovations in multi-targeted therapies against breast cancer. It also includes an understanding of the biological mechanisms in pathways of this disease in relation to these agents to address treatment failure. The molecular pathways examined are ER, HER2, EGFR, VEGFR, PI3K/AKT/mTOR, MAPK, PARP, and CDK4/6. Additionally, KAT6A is identified as a molecular target along with new clinical agents that are being reviewed, as well as combinations of strategies using molecular profiling and pathology reports.


RESULTS
The major signaling pathways that control breast cancer progression and resistance, and/or sensitivity to therapy. KAT6A, a histone acetylase, was amplified in all of the ER+ breast cancers, suggesting that it can be used as a biomarker for assessing the effectiveness of CDK4/6 inhibitors and is an epigenetic therapeutic target in mammary cell lines. KAT6A is being developed as a selective KAT6 inhibitor; however, it seems most benefits may come from additional inhibitors of Menin, which could help overcome endocrine therapeutic resistance. We also reviewed CDK4/6 Mutations and resistance invariably associated with palbociclib, as well as immunotherapy with antibody- drug conjugate, including trastuzumab deruxtecan (DS-8201).


DISCUSSION
Multi-pathway targeting holds promise to overcome shortcomings of current monotherapies. Due to the complexity of the breast cancer molecular landscape, we must plan for and potentially target resistance mechanisms. The use of KAT6A as a biomarker, along with the use of novel inhibitors, may help inform treatment decisions and improve outcomes. Additionally, this review has identified the need to position the surveillance and addressing of our resistance mechanisms for current therapies, like CDK4/6 inhibitors and antibody-drug conjugates, via rationalized combinations.


CONCLUSION
The future of breast cancer therapy lies in combination strategies that are developed via molecular profiling and guided by resistance biomarkers. Integrative, biomarker-driven treatment approaches will provide a rationalized and likely more effective means of treating advanced and refractory breast cancer in our effort to improve patient outcomes in a targeted and personalized approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed313d6f6c8ca51ca54204284a5a09feeeb6ef21" target='_blank'>
              Breaking Barriers in Breast Cancer: Multi-Targeted Therapeutic Insights.
              </a>
            </td>
          <td>
            Apsara Unni, K. Rajagopal, Krishna Shevate, Gowramma Byran
          </td>
          <td>2025-10-08</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells.
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous and aggressive hematologic malignancy, with the activated B-cell-like (ABC) subtype displaying particularly poor prognosis due to inherent treatment resistance and elevated recurrence rates. Despite advances in targeted therapies and immunotherapies, a significant proportion of patients experience relapse or refractory disease, highlighting the urgent need for novel biomarkers and innovative therapeutic strategies to improve clinical outcomes. Methods A multi-dimensional analysis of SH3BP5 expression was performed across DLBCL subtypes, integrating transcriptomic, proteomic, and clinical datasets to assess its correlation with immune infiltration, tumor metabolism, and patient prognosis. Single-cell RNA sequencing data were employed to examine the tumor microenvironment (TME) with higher resolution. Further analysis of the association between SH3BP5 and immune checkpoint gene expression was conducted to explore its potential role in immunotherapy response. Functional in vitro assays were carried out to assess the impact of SH3BP5 knockdown on DLBCL cell proliferation and apoptosis. Results The analysis revealed that SH3BP5 is preferentially overexpressed in the ABC subtype of DLBCL across multiple datasets and validated cohorts, and its high expression is significantly associated with poor overall survival. Single-cell transcriptomic profiling demonstrated that SH3BP5 is mainly expressed in malignant B cells and inversely correlated with immune cell infiltration, particularly CD8 + T cells. Mechanistically, pathway enrichment and metabolic assays indicated that SH3BP5 is linked to mitochondrial metabolic reprogramming, promoting oxidative phosphorylation (OXPHOS) and potentially contributing to reduced responsiveness to immune checkpoint inhibitors (ICIs). Functional studies showed that SH3BP5 knockdown significantly suppressed DLBCL cell proliferation, induced apoptosis, and reduced tumor cell viability in vitro. Conclusion This study suggests that SH3BP5 may serve as a prognostic biomarker and a potential therapeutic target in DLBCL, particularly within the ABC subtype. By delineating its associations with immune evasion and metabolic reprogramming, these findings provide a mechanistic basis for further exploration of SH3BP5-targeted interventions to help overcome therapy resistance. Future studies in larger clinical cohorts and functional models are warranted to validate these results and assess the potential of integrating SH3BP5 expression profiling into precision medicine strategies for DLBCL. Trial registration The study was registered in the Chinese Clinical Trial Registry (ChiCTR2200060430; http://www.chictr.org.cn/) on June 1, 2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06951-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27222190fd1301e27cfd9869b08b07fc58aa185" target='_blank'>
              SH3BP5-driven metabolic-immune crosstalk in DLBCL: a prognostic biomarker and therapeutic target for reshaping immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Tong Wu, Yi Yang, Yuan Zong, Jiawen Zhao, Xiaoyu Zhao, Lei Li, Yiming Gao, Ning Li, Liting Jiang, Yinyin Xie
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is a highly heterogeneous tumor that lacks reliable biological markers for diagnosis and prognostic monitoring. Currently, the differentially expressed genes between paired adjacent normal tissues and ccRCC tumor tissues at single-cell resolution remained to be further discovered. To address this challenge, we performed an integrative analysis of multiple single-cell databases containing paired ccRCC samples. Using the “CopyKAT” algorithm, we accurately identified ccRCC tumor cells. Subsequently, various pseudotime algorithms were employed to identify malignant cells with tumor stem cell-like properties and high plasticity. This cell subgroup exhibited high expression of malignant features, including hypoxia, epithelial-mesenchymal transition (EMT), and proliferation/invasion phenotypes. We then performed differential analysis to identify genes highly expressed in this subgroup and constructed a reliable clinical diagnostic model for ccRCC using multiple machine learning algorithms. Furthermore, we identified AXL as a key gene with significant oncogenic activity, where high expression of AXL correlated with poor patient prognosis. Immune infiltration and spatial transcriptomics analyses further revealed that AXL promotes tumor progression interaction with M2 macrophages. Taken together, our analysis establishes a reliable 13-gene panel diagnostic model and AXL gene as reliable biological markers for ccRCC, providing valuable targets and a theoretical foundation for the development of precision-targeted therapies for ccRCC. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00563-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c5c1d87281d2808c2e2e599af16c985ed97a58" target='_blank'>
              Integrated single-cell and bulk transcriptome analysis revealed high plasticity subpopulation and promising diagnosis model for clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zhongwen Lu, Fanyi Kong, Jiahuan Sun, Jing Ge, Jiajin Wu, Kunpeng Wang
          </td>
          <td>2025-09-30</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Human papillomavirus (HPV)-associated oropharyngeal cancer (OPC) is increasing in prevalence, but the drivers of metastasis remain poorly understood, which impacts the ability to personalise management decisions. Much of the genomic research to date focuses on the HPV-negative population. Here, we utilise single-cell and spatial single-cell techniques to understand the drivers of metastasis. Methods Patients with HPV-positive OPC and cervical lymph node metastases treated with curative surgery had matched samples from the primary and lymph nodes collected for research. Single-cell RNA sequencing, single-cell spatial sequencing (Visium) and in-situ spatial platforms were performed. Cancer clones were delineated using inferred copy number variation. Expression phenotypes and interactions with the tumour microenvironment were compared between the metastasising and non-metastasising cancer clones. Results Individual cancer clones have varied abilities to metastasise and undergo clonal expansion in the lymph node, with only a subset of clones present in the primary expanding in the lymph node. Four mechanisms were identified as defining the metastatic phenotype, including protein translation adaptation, immunoproteasome dysfunction and immune evasion, suppression of the IFN immune response and cap-independent protein translation. Conclusions This research elucidates multiple mechanisms driving the expansion of cancer clones in HPV-positive oropharyngeal cancer. By detailing the roles of translational adaptation, immunoproteasome dysfunction, suppression of the interferon immune response and cap-independent protein translation, we provide insights into how these processes contribute to immune evasion and tumour survival. Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04392-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fda352f2dd3f74c2b6cb392920d418565101f0" target='_blank'>
              Protein translation dysregulation and immune cell evasion mediated by IFN and immunoproteasome downregulation define metastatic clones in HPV-related cancer of the oropharynx
              </a>
            </td>
          <td>
            Venessa T. Chin, W. Muskovic, Rachael A. McCloy, Drew R Neavin, José Alquicira-Hernández, Himanshi Arora, A. Senabouth, Kavitha Krishna Sudhakar, Patricia Keith, Ellie Spenceley, Dominik Kaczorowski, Peter Floros, Brett Leavers, Julia Crawford, Richard Gallagher, Joseph E. Powell
          </td>
          <td>2025-10-14</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common malignancies worldwide. Its complex molecular mechanisms make traditional treatments ineffective in some patients. In recent years, advances in genetic engineering technologies have opened up new possibilities for precision medicine in BC. Researchers have attempted to intervene in cancer cell proliferation and induce apoptosis through various genetic strategies. Preclinical studies are increasing, and therapeutic targets and delivery technologies are expanding. However, current research has largely focused on specific cases or single technologies, lacking a systematic integration and analysis of various genetic strategies from a mechanistic perspective. This paper reviews three major genetic engineering strategies for BC based on their therapeutic mechanisms: RNA interference, gene compensatory expression, and gene repair and deletion. Representative studies, key technologies, challenges, and development trends are analyzed for each. Furthermore, this paper summarizes common challenges across these different mechanisms regarding delivery efficiency, stability, and targeting, and proposes potential directions for future synergistic strategies. This paper helps clarify current research approaches in BC gene therapy from a mechanistic perspective, providing a reference for the design of subsequent cross-mechanism integration and precision intervention strategies. However, issues such as mechanism overlap and diversification of delivery platforms still need further in-depth exploration. Future research can focus more on multi-mechanism combination strategies, individualized treatment pathways, and feasibility assessment of clinical transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38bb75ae7dd66792d49d2225441735e8c43eeb18" target='_blank'>
              Applications of Three Gene Engineering Mechanisms in Breast Cancer Therapy
              </a>
            </td>
          <td>
            Yifan Tian
          </td>
          <td>2025-09-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for elucidating tumor heterogeneity and clonal evolution. However, comprehensive scDNA-seq studies and tailored mutation panels for CSCC are lacking. Methods We analyzed the genomic landscape of Chinese CSCC patients via a Multi-Patient-Targeted (MPT) scDNA-seq approach. This method combined bulk exome sequencing with Tapestri scDNA-seq. Mutations identified through bulk sequencing were used to design a targeted panel for scDNA-seq. Comparative analysis was conducted to explore the associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis was performed to understand the evolutionary trajectories of the tumors. Results Bulk sequencing revealed a diverse spectrum of somatic mutations in CSCC tumors, with missense mutations being predominant. The top tumor mutations, such as those in NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A, presented similar frequencies to those reported in studies in Korean and Caucasian populations. However, the mutation frequencies of HRAS, TTN, MUC16 and MUC4 were significantly different from the Korean and Caucasian populations. Comparative analysis revealed associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis via scDNA-seq revealed distinct evolutionary trajectories and their potential correlation with tumor development and patient prognosis. Furthermore, scDNA-seq identified two low-frequency mutation clones, NLRP5 and HMMR, which play important roles in the clonal evolution of CSCC. Conclusions Our study introduced a novel MPT sequencing approach for CSCC, providing insights into tumor heterogeneity and clonal evolution. We identified novel mutations and their potential associations with patient survival and tumor characteristics. Overall, our study layed the groundwork for personalized treatment strategies and provides a reference for future applications of the MPT panel in CSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14585-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667c56c3222da6ac371627cb03ea1bf3e1f96087" target='_blank'>
              Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing
              </a>
            </td>
          <td>
            Wen Chen, Jiawei Xu, Chengdong Yu, Meng Zhou, Yong Ai, Wenbing Rao, Qingchuan Wang, Siyi Xu, Lei Tang, Si Gong, Jun Rao
          </td>
          <td>2025-08-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: The elucidation of the human genome has fundamentally transformed our understanding of disease etiology, positioning gene mutations as central players in susceptibility, pathogenesis, and progression across a vast spectrum of human disorders. The translation of this genetic knowledge into targeted therapeutic strategies represents the cornerstone of precision medicine, heralding a new era in clinical management.
Objective: This narrative review aims to synthesize the current landscape of how specific gene mutations influence disease development and progression, and to explore how these discoveries are shaping the development and application of novel, targeted therapeutic approaches.
Main Discussion Points: The review thematically explores the paradigm of oncogenic mutations in driving targeted cancer therapies, such as tyrosine kinase inhibitors and PARP inhibitors, while also examining the role of germline mutations in hereditary cancer syndromes. It further expands into non-oncological domains, including cardiology and neurology, highlighting the development of treatments like PCSK9 inhibitors and antisense oligonucleotides. The discussion also covers groundbreaking advanced therapies, such as gene replacement and gene editing, using examples from spinal muscular atrophy and sickle cell disease. Critical analysis is provided on the challenges of therapeutic resistance, variants of uncertain significance (VUS), and issues of health equity.
Conclusion: The collective evidence firmly establishes that targeting specific gene mutations is a powerful and transformative therapeutic strategy. However, realizing the full potential of precision medicine requires overcoming significant hurdles, including resistance mechanisms, the high cost of therapies, and a lack of diversity in genetic research. Future efforts must focus on innovative trial designs, long-term safety monitoring, and equitable implementation to ensure these advances benefit all patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382be60380435d45208a6a43c1baabce83169b3e" target='_blank'>
              NARRATIVE REVIEW ON THE IMPACT OF GENE MUTATIONS IN DISEASE SUSCEPTIBILITY, PROGRESSION, AND TARGETED THERAPEUTIC APPROACHES: A NARRATIVE REVIEW.NARRATIVE REVIEW ON THE IMPACT OF GENE MUTATIONS IN DISEASE SUSCEPTIBILITY, PROGRESSION, AND TARGETED THERAPEUTIC APPROACHES
              </a>
            </td>
          <td>
            Noor Ul, Ain Khaliq¹, Muhammad Yasir Yasin², Talha Waheed³, Atif Maqsood⁴, Ayesha Maalik⁵, Musa Khan⁶
          </td>
          <td>2025-10-04</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3017cd20ec1bd27c4516abd6034d6436513f0d42" target='_blank'>
              Balanced chromosomal insertions as the mechanism of recurrent familial microstructural abnormalities: detailed analyses using long-read whole-genome sequencing.
              </a>
            </td>
          <td>
            Hironao Shirai, Keiko Shimojima Yamamoto, Hirokazu Arai, Yukio Sawaishi, Saori Fujita, Yoko Kuriyama, Masaki Miura, Jun Tohyama, Toshiyuki Yamamoto
          </td>
          <td>2025-09-24</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi A. Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018e4ec4dd8ef81f5b174f65fbe205bcc992d85a" target='_blank'>
              The Impact of Polyploid Giant Cancer Cells: The Root of Stress Resilience.
              </a>
            </td>
          <td>
            Yuta Ogawa, Lydia Fisher, Tomonori Matsumoto
          </td>
          <td>2025-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="DNA mismatch repair (MMR) preserves genomic integrity by correcting replication errors. Deficiency in MMR (MMRd) results in microsatellite instability, increased tumor mutational burden, neoantigen generation, and activation of immune response. In this review, we first outlined how MMR loss promotes immune activation and responsiveness to immune checkpoint blockade (ICB), establishing MMRd status as the first tumor-agnostic biomarker for ICB therapy. Subsequently, we summarized the compelling evidence that defines MMR status as a dynamic, context-dependent process influenced by environmental and therapeutic pressures, rather than a fixed, binary trait. Accordingly, we discussed the implications of spatial and temporal heterogeneity of MMR status for both the diagnosis and treatment of cancer, the differential response of MMRd tumors to ICB, as well as the occasional benefits observed in MMR-proficient (MMRp) immune-cold cancers. We then explored strategies to exploit MMR dynamics and mimic MMRd-like phenotypes through alkylating agents, pharmacological MMR inhibition, and stress-mediated modulation, with the aim of sensitizing refractory tumors to immunotherapy. Finally, we reported emerging therapeutic opportunities in MMRd tumors, including Werner helicase inhibition, nonsense-mediated decay blockade, and neoantigen-targeted vaccines. Altogether, reframing MMR as a dynamic and targetable axis may broaden immunotherapy applicability and advance precision immune oncology across different tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6face00bfd040e4e182ff419f940662d44199ca5" target='_blank'>
              Mismatch Repair as a Dynamic and Clinically Actionable Vulnerability in Cancer.
              </a>
            </td>
          <td>
            Eleonora Piumatti, P. P. Vitiello, V. Amodio, A. Bardelli, Giovanni Germano
          </td>
          <td>2025-09-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis through their capacity for self-renewal and multilineage differentiation. However, the isolation and functional characterization of HSCs remain challenging due to their cellular heterogeneity and dynamically regulated transcriptional and epigenetic landscapes. Advances in experimental and computational biology, including single-cell RNA sequencing (scRNA-seq), chromatin immunoprecipitation sequencing (ChIP-seq), network inference algorithms, and machine learning, have improved our ability to resolve transcriptional states, trace lineage trajectories, and reconstruct gene regulatory networks (GRN) at single-cell resolution. These approaches enable the discovery of novel HSC subtypes and regulatory factors, and facilitate the integration of multi-omics data to uncover epigenetic and transcriptional mechanisms that drive stem cell fate decisions. Additionally, machine learning models trained on high-throughput datasets provide predictive power for identifying novel enhancers, transcription factors, and therapeutic targets. This review underscores the synergistic role of computational tools in deciphering HSC biology and highlights their potential to improve stem cell therapies and precision treatments for hematologic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3c26c697c565c33ced832e7a44009a59ac4768" target='_blank'>
              Unlocking hematopoietic stem cell potential: integrative computational approaches for genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            P. Raghav, Basudha Banerjee, Rajni Chadha
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Tumor heterogeneity impacts invasive behaviors, treatment response, and clinical outcomes in triple-negative breast cancer (TNBC). However, this heterogeneity remains incompletely characterized. This study aims to utilize multi-scale data to investigate inter-tumoral heterogeneity and identify potential TNBC biomarkers. Methods Single-cell RNA expression profiles were analyzed using R packages. Specifically, the infercnv, Pyscenic, GeneNMF, SCP, Vector, CellChat, and hdWGCNA packages were employed to identify malignant cells and characterize heterogeneity in transcription factors, metaprograms, lineage evolution, developmental trajectories, cell–cell interactions, and co-expression networks. Bulk RNA datasets were incorporated to assess the prognostic value of cell clusters and candidate genes. G Protein Subunit Alpha 15 (GNA15) expression was determined via reverse transcription–quantitative PCR (RT–qPCR) and immunohistochemistry. Cell functional assays were performed to evaluate proliferation, migration, and invasion capabilities. Results A total of 14,335 malignant cells were isolated from epithelial cells across 15 single-cell RNA samples. Six tumor cell clusters were identified, which exhibited distinct prognoses, biological functions, driver transcription factors, and co-expression networks. Notably, the S2 cluster demonstrated association with multiple malignancy-related pathways and inferior survival outcomes. GNA15 emerged as the S2 cluster hub gene. In vitro experiments confirmed that GNA15 knockdown significantly attenuated proliferation, migration, and invasion in TNBC cell lines. Conclusions Our study comprehensively delineated TNBC tumor cell heterogeneity and established the critical role of GNA15 in TNBC progression. These findings enhance the understanding of TNBC heterogeneity and provide a theoretical foundation for TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e149422e4d37b7c64ca2e5cbee9d7c8957b702e6" target='_blank'>
              Multiomics profiling identifies the poor prognostic role of a tumor cluster with GNA15 overexpression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Guixin Wang, Jun Cao, Chenglu Lu, Yu-Peng Cao, Shuo Wang, Ziyi Chen, Zhaohui Chen, Yingxi Li, Yue Yu, Yao Tian, Xin Wang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1949495332a12dd110f8a642abdf8beee61d934d" target='_blank'>
              Advanced strategies to overcome multidrug resistance in cancer therapy: progress in P-glycoprotein inhibitors, drug delivery, and personalized medicine.
              </a>
            </td>
          <td>
            Ankitha Harish, N. Deepika, Vedamurthy Joshi, Prakash S Goudanavar
          </td>
          <td>2025-09-02</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Breast cancer (BC) remains a significant global health challenge, partly due to its ability to develop resistance to treatment. BC resistance arises from various biological mechanisms, including cancer cells actively expelling drugs, altering their internal metabolic and signaling pathways, evading the immune system, or persisting as drug-tolerant cancer stem cells. These changes occur in the context of a tumor microenvironment that further shields the cancer, limiting treatment access. As the disease advances, particularly when it spreads to other body parts, effective treatment becomes increasingly difficult. Recent advancements in artificial intelligence (AI) offer promising avenues for improving early detection, refining diagnoses, enhancing clinical decision-making and thereby personalizing treatment strategies. This review examines the biological mechanisms that cause BC resistance and its treatment. It further explores AI’s potential to address critical challenges, transforming BC therapy to provide improved survival rates and enhanced quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb5357bf49f1fab994fbcce36bcc1738760cec3" target='_blank'>
              The Underlying Mechanisms and Emerging Strategies to Overcome Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Krishnaswamy Kannan, Alagarsamy Srinivasan, Aarthi Kannan, Nawab Ali
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="CDK7 has emerged as a cancer target because of its pivotal roles in cell cycle progression and transcription. Several CDK7 inhibitors (CDK7i) are now in clinical evaluation. Identifying patients most likely to respond to treatment and early detection of tumour evolution towards resistance are necessary for optimal implementation of cancer therapies. Continuous culturing of prostate cancer cells with Samuraciclib, a non-covalent ATP-competitive CDK7i, led to outgrowth of resistant cells. These were characterised by the acquisition of a single base change in the CDK7 gene, Asp97 to Asn (D97N). Mutant cells were resistant to other non-covalent CDK7i but remained sensitive to covalent CDK7i. Cryo-EM structure and kinase ligand affinity determinations revealed reduced affinity of the CDK7-D97N mutant for non-covalent CDK7i. Remarkably, Asp97 is absolutely conserved in human CDKs, inferring its importance for the activities of all CDKs. Consistent with this, mutation of the homologous residue in CDK12 (D819N) or CDK4 (D99N) promoted resistance to drugs that inhibit these CDKs. Our findings reveal a general mechanism for acquired resistance with obvious implications for patients treated with CDK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4e4fc33eb2e8814d38d33c359427c463ea1f89" target='_blank'>
              Resistance to CDK7 inhibitors directed by acquired mutation of a conserved residue in cancer cells
              </a>
            </td>
          <td>
            Chun-Fui Lai, Victoria I. Cushing, Ellen Olden, C. Bevan, R. C. Coombes, Basil J. Greber, L. Buluwela, Simak Ali
          </td>
          <td>2025-09-08</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Advancements in chemotherapy have improved outcomes for rhabdomyosarcoma (RMS), the most common soft tissue sarcoma in children. However, relapse-free survival rates remain below 20% in metastatic alveolar rhabdomyosarcoma (ARMS). Tumor-initiating cells (TICs) drive recurrence. Targeting this pool of cells has been shown to reduce relapse rates and metastasis in various cancers. Emerging evidence implicates lipid metabolism in supporting stem-like properties in cancer. However, TICs remain poorly characterized in ARMS. This study investigates the transcriptomic and metabolic profile of TICs in ARMS. Methods Transcriptomic and lipidomic profiling were performed on tumorsphere-derived and adherent cells from ARMS cells. Differential expression of metabolic genes and lipid species was assessed via RNA-Sequencing and mass spectrometry. Functional dependence on Phospholipase A2 (PLA2) signaling was tested on TICs in vitro and in vivo. Seahorse assays were used to measure glycolytic and mitochondrial flux. Statistical significance was determined using Student’s t-test or one-way ANOVA. Results Transcriptomic analysis of ARMS tumorspheres showed upregulation of lipid metabolism especially different PLA2 enzyme subtypes, and suppression of glycolytic gene expression. Lipidomic analyses revealed enrichment of linoleic acid-derived triglycerides and lysophospholipids, paralleled by increased PLA2 activity. Tumorspheres also showed elevated expression of lipid storage regulators PPARG and CD36 and correspondingly, significant lipid droplet accumulation. Inhibition of PLA2 with Darapladib reduced tumorsphere formation and cellular motility in vitro and tumor volume in vivo. These defects were rescued by exogenous linoleic acid (LA) supplementation, implicating PLA2-driven lipid signalling in TIC maintenance. Interestingly, TICs display high metabolic flexibility in energy consumption, as neither glycolysis nor fatty acid oxidation (FAO) inhibition significantly impaired tumorsphere formation. Conclusions PLA2-mediated lipid remodelling supports TIC by promoting linoleic acid–linked lipid signalling. PLA2 inhibition impairs self-renewal, motility, and tumorigenic potential, highlighting it as a promising therapeutic target in ARMS. These findings uncover lipid remodelling as a key metabolic adaptation in ARMS TICs, independent of canonical energy pathways, with implications for targeting the stem-like compartment in pediatric sarcomas. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02409-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe186d79f283fafcded05829d4a4d4c46ef02bf" target='_blank'>
              PLA2 driven lipid signaling drives ARMS tumorigenic cell properties
              </a>
            </td>
          <td>
            Amogh Gupta, Bharathi Ramanathan, Dipanwita Das, Aiswariya Vadivellu, A. Loh, R. Taneja
          </td>
          <td>2025-10-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, Andrew C White
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
To analyze LUAD cases in The Cancer Genome Atlas (TCGA), the mRNA expression-based stemness index (mRNAsi) was used. Models of cancer immunity and LUAD prognosis were developed on the basis of correlations between immune and stem cell genes.


METHODS
We investigated the differential expression of mRNA dryness index (mRNAsi) in LUAD, survival prognosis, and correlation with clinical parameters. Iden-tify key mRNAsi-related modules and genes by weighted gene co-expression network analysis (WGCNA). Gene set enrichment analysis (GSEA/GSVA) was used to identify stem cell markers and immune-related differentially expressed genes (SC IRGs), and 10 key genes were enriched. Subgroup enrichment, gene mutations, genetic correlated-ness, gene expression, immunity, tumor mutational burden (TMB), and drug sensitivity were further performed in the comprehensive analysis of pivot genes and subgroups.


RESULTS
Compared with normal cells, LUAD cells presented significantly greater mRNAsi values through differential expression analysis. The mRNAsi was highly cor-related with clinical parameters (age, sex, and T stage). On the basis of WGCNA, blue-green and brown modules were identified as the most significant modules (including positive and negative correlations) associated with mRNAsi expression. The functions and pathways of the two mRNAsi-related modules were enriched mainly in tumor oc-currence, development, and metastasis. Cox regression analysis was used to identify 30 SCIRGs associated with prognosis by combining the stem cell indices of the DEGs and the immune-related DEGs. A LASSO regression model was constructed after 10 DEGs related to the prognosis of patients with LUAD were detected. There were significant differences between the high-risk and low-risk groups in terms of GSEA/GSVA, im-mune cell correlation, clinical correlation, etc., following model validation (P<0.05).


DISCUSSION
There are a total of 10 genes in our study model, including four key pre-dictors: DGRIK2, PTTG1, LGR4, and PDGFB. The other 6 genes need to be further delineated and verified. To date, our research has some limitations and has not been validated in cell or animal experiments. These findings provide a relevant theoretical basis for subsequent experimental research on lung adenocarcinoma stem cells. Further research into these cancer stem cell genes will increase the likelihood that they play a role in cancer. There is an opportunity to use it as a therapeutic target for targeted ther-apy for lung adenocarcinoma in the future.


CONCLUSION
mRNAsi is associated with immunity, which was previously overlooked in the gene analysis of LUAD stem cells. These key genes have a strong overall corre-lation, which can be achieved by inhibiting the stemness characteristics of cancer cells, which may lay the foundation for future research on LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c303cbba9ae27f76f9e2be614e68543e2587be" target='_blank'>
              Identification of the Immune Microenvironment, Mutation Burden, Immunotherapy, and Drug Sensitivity Related to Lung Adenocarcinoma Tumor Stem Cells via WGCNA.
              </a>
            </td>
          <td>
            Qi Liu, Liusheng Wu, Meiling Lu, H. Jia, Xiaoqiang Li
          </td>
          <td>2025-09-26</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome.
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Schillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA replication, as the foundation of genetic information transmission and the continuity of life, relies on high fidelity and dynamic regulatory mechanisms that are central to maintaining genome stability. Through literature analysis, this review systematically summarizes the core molecular mechanisms of DNA replication initiation, elongation, proofreading, and repair, while comparing the commonalities and differences between prokaryotic and eukaryotic systems. Comparative analysis demonstrates that while prokaryotic and eukaryotic DNA replication share conserved high-fidelity principles, they exhibit fundamental mechanistic divergences. Advances in technologies such as cryo-electron microscopy have uncovered the structure and dynamics of the replication machinery and its coupling with epigenetic regulation. Future research should employ integrative multi-omics approaches to decipher the spatiotemporal regulation of replication, facilitating therapeutic interventions for replication-associated pathologies including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951febf5def0fd8aa3519a39b3eb248cfca4ae46" target='_blank'>
              DNA Replication and the Transmission of Genetic Information: A Molecular Mechanism Perspective
              </a>
            </td>
          <td>
            Ziying Wang
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite the success of endocrine therapy (ET) in treating hormone receptor-positive breast cancer, a significant proportion of patients relapse during or after treatment, making ET resistance a major clinical challenge. Previously we have shown that ET-resistant breast cancer cells exhibit reduced ceramide levels and an increased sensitivity to ceramide-induced cell death. Here, we demonstrate that ceramides induce a distinct transcriptional reprogramming in ET-resistant cells, characterized by upregulation of endoplasmic reticulum stress (EnRS) pathways. Ceramide-induced EnRS is PERK-dependent and functionally linked to cell death in multiple models of ET resistance. Using a photoactivatable ceramide probe, we identify TRAM1 as a functionally important ceramide-interacting protein (CIP) in ET-resistant cells that correlates with worse relapse-free survival and a more aggressive breast cancer phenotype in luminal breast cancer patients. Additionally, knockdown of TRAM1 phenocopies ceramide action in ET resistance, thereby suggesting its role in mediating ceramide-induced lethal actions in ET resistance. Together, our findings reveal that ET-resistant breast cancer cells are more sensitive to PERK-mediated EnRS as compared to their ET-sensitive counterparts. Ceramides can exploit this dependence by interacting with CIPs such as TRAM1, leading to PERK activation and consequential cell death preferentially in the ET-resistant breast cancer models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b745d88155de10769304cbcee438fd06edfb58c0" target='_blank'>
              Ceramide-induced Endoplasmic Reticulum Stress as a Targetable Vulnerability in Endocrine Therapy-Resistant Breast Cancer
              </a>
            </td>
          <td>
            Purab Pal, S. Chitkara, Godwin K. Sarpey, Fatimah Alani, Huiping Zhao, Malik Ata, Jun Qu, Rachel Schiff, Debra A. Tonetti, Geoffrey L. Greene, Jonna Frasor, G. E. Atilla‐Gokcumen, Jonathan L. Coloff
          </td>
          <td>2025-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Hypoxia is a hallmark of the colorectal cancer (CRC) tumor microenvironment (TME) that drives malignant progression, chemoresistance, and immune evasion. However, the cellular heterogeneity underpinning hypoxic responses in CRC and its impact on prognosis remain incompletely understood. Methods We integrated single-cell RNA sequencing data from 15 CRC samples (GSE166555 and GSE221575) to delineate hypoxic and normoxic cell populations and identify hypoxia-related genes (HRGs). Weighted gene co-expression network analysis (WGCNA) and enrichment profiling elucidated key biological processes associated with hypoxia. Cell–cell communication networks were inferred using CellChat, and transcription factor regulatory modules were reconstructed via SCENIC and GRNBoost2. A hypoxia-based prognostic signature was developed from unique H3 cluster genes using univariate Cox and Lasso regression on The Cancer Genome Atlas (TCGA; n = 606) and validated in GSE39582 (n = 579). Drug sensitivity correlations were derived from the GDSCv2 database. Finally, in vitro assays assessed the functional role of GIPC2, a model gene, in CRC cell lines. Results Single-cell profiling uncovered eight distinct hypoxic clusters, with H3 exhibiting the highest hypoxia scores and enrichment in glycoprotein metabolic and angiogenesis pathways. The eight-gene prognostic model stratified patients into high- and low-risk groups with significantly different overall survival in both TCGA (P = 0.0026) and validation cohorts (P = 0.011). Drug analysis highlighted associations of model genes with PI3K/MTOR and apoptosis pathways. GIPC2 knockdown in LS180 and HT-29 cells markedly inhibited proliferation, migration, and invasion, while inducing apoptosis and reversing EMT phenotypes. Conclusions We present a robust hypoxia-related gene signature that accurately predicts CRC patient prognosis and nominate GIPC2 as a potential biomarker and therapeutic target, offering new insights into hypoxia-driven CRC biology and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8575947e15fbc35fb6bf68728fce7870efdbff1" target='_blank'>
              Decoding the hypoxic tumor microenvironment in colorectal cancer for prognostic modeling and therapeutic target discovery
              </a>
            </td>
          <td>
            Xiao-Cui Duan, Yan Zhou, Fan Feng, Hai-Bo Jiang, Mei-Lin Wang, Zhe Han, Hong-Fei Pang, Yu-Hang Liu, Heng-Zhe Jia, Meng- He, Hongpan Xu, Yuan-Yuan Wang
          </td>
          <td>2025-08-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemotherapy resistance is a critical barrier to effective cancer treatment, contributing to the high number of cancers related death in patients. This review comprehensively examines the molecular mechanisms driving resistance, including innate and acquired resistance, tumor heterogeneity, and tumor microenvironment (TME) interactions. Key mechanisms include enhanced drug efflux via ABC transporters, increased DNA repair, altered drug targets, drug detoxification, and evasion of apoptosis. Tumor heterogeneity fosters resistant subclones, while TME factors like hypoxia and cancer-associated fibroblasts (CAFs) exacerbate resistance. Crosstalk between signaling pathways, such as PI3K/AKT and MAPK, fuels resistance but offers hope for combination therapies. Insights into these mechanisms are vital for developing targeted therapies to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215edb205618059ddf2fe5de4347e899f8539201" target='_blank'>
              Mechanism of Chemotherapy Resistance: a Comprehensive Review
              </a>
            </td>
          <td>
            Subash Paudel, Laxmi Parajuli, Santosh Shrestha, Indira Rai
          </td>
          <td>2025-09-30</td>
          <td>Nepalese Journal of Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d27cc9b475222eaa7f4e83e26c9db176ee78f1" target='_blank'>
              Novel mutations in transducin-like enhancer of split 6 cause infertility in women with embryonic arrest.
              </a>
            </td>
          <td>
            Jifan Tan, Yue Jiang, Ching Wan, Kehan Zhu, Yan Xu, Yanwen Xu, Jia Huang
          </td>
          <td>2025-09-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third leading cause of cancer-related deaths. Integrating cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present a comparative mRNA-seq analysis of tissue samples from 32 CRC patients, pairing tumors with adjacent healthy tissues. Differential expression gene (DEG) analysis revealed dysregulated metabolic programs. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2, which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potential of the Xc- system, we analyzed gene perturbations from CRISPR screens across various cell types and used functional assays in five primary patient-derived organoid models. We identified a previously uncharacterized cell surface protein signature predicting chemotherapy resistance and highlighted the causality and potential of pharmacological blockage of ferroptosis as a promising avenue for cancer therapy. Redox homeostasis, ion/amino acid transporters, and regulators of neuronal survival and differentiation were pathways associated with these co-dependent genes in patient specimens. This study highlights several potential clinical targets for CRC therapy and promotes the use of patient-derived organoids oids to functionally validate in silico predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af3aa92ab85625c6d369fc2c084fc1ee49ca6118" target='_blank'>
              Patient-specific pharmacogenomics demonstrates xCT as predictive therapeutic target in colon cancer with possible implications in tumor connectivity.
              </a>
            </td>
          <td>
            M. Strecker, Keren Zohar, Martin Böttcher, T. Wartmann, H. Freudenstein, Maximilian Doelling, M. Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, D. Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. Kahlert
          </td>
          <td>2025-09-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is one of the leading cause of cancer-associated mortality globally. Its tumor microenvironment consists of various types of cells, which have a great impact on patient response and prognosis. The present study used scRNA-seq technology to investigate gene expression pattern of various cell types in NSCLC and their correlation with immune microenvironment. Methods We acquired NSCLC single-cell RNA sequencing data from GSE117570 and employed UMAP clustering analysis to explore the major cell subpopulations. Additionally, we explored the differential expression of important genes among various cell types and their associations with immune cell infiltration, scores of tumor microenvironment. Results Our results exhibited the heterogenicity of cell types in NSCLC and also indicated more than 60 genes whose expression displayed significant differences between cell groups, such as AP1S1, BTK, FUCA1, NDEL14, TMEM106B and UNC13D. Expression of these genes correlated with immunocyte infiltration, tumor microenvironment scores, indicating their potential roles in tumor progression and therapeutics. Conclusion Our single-cell transcriptomic dissection of NSCLC highlights cellular heterogeneity as well as potential biomarkers associated with immune microenvironment. The findings provide a foundation for the rational development of biomarker-driven personalized treatment and therapy by extending the cell-type specificity of current designs of NSCLC therapeutics. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03494-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfe159d40376ac64bc4970c70d60376979b77ad" target='_blank'>
              Single-cell RNA-seq reveals gene expression heterogeneity in NSCLC and its link to the immune microenvironment
              </a>
            </td>
          <td>
            Jinzhuo Ren, Hua Zhang, Haiyong Wu
          </td>
          <td>2025-09-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly lethal malignancy characterized by a complex pathological mechanism involving multiple genes and progressive stages. The efficacy of targeted and immunotherapy remains limited, highlighting the urgent need for a reliable model to predict prognosis and response to immune checkpoint inhibitors (ICIs). Methods We developed an integrated model based on genes related to autophagy, senescence, dormancy, mitochondrial function, and tumor stemness. The predictive capability of this model for HCC prognosis and ICI response was evaluated. Single-cell transcriptomic analysis and immunocompetent mouse models were further utilized to elucidate the role of model-associated genes in regulating the tumor immune microenvironment. Results A 16-gene integrated model was constructed using genes associated with mitochondrial function, autophagy, dormancy, stemness, and senescence. This model demonstrated robust predictive power for HCC prognosis and ICI responsiveness. Single-cell trajectory analysis revealed that EZH2 plays a crucial role in immune cell infiltration, activation, and HCC progression. Additionally, in vivo mouse models further indicated that EZH2 may regulate CD8+ T cell activation and exhaustion through the MIF-CD74 signaling pathway. Conclusion The integrated model holds potential as a prognostic and predictive tool for HCC immunotherapy. EZH2 may influence CD8+ T cell activation and exhaustion via the MIF-CD74 axis, providing insights for patient stratification and potential therapeutic strategies to enhance immunotherapy efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07071-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7937b56d5b80810f28cf602b4f87ac567637d0a" target='_blank'>
              Integrated single-cell RNA-seq analysis reveals that EZH2 regulates the MIF-CD74 axis to modulate T cell activation and exhaustion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yangyang Zhou, Yi Xu, Mengfan Ye, Jiayi Xing, Di Chen, Al-Ameer Wail Hussein Ahmed, Al-Rahabi Rehab Abdullah Mohammed Ali, Chenwei Pan, Xiangchou Yang, Zan Shen
          </td>
          <td>2025-10-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor dormancy represents a clinically significant but poorly understood state in which disseminated cancer cells persist in a quiescent, non-proliferative state, evading conventional therapies and driving late relapse. This review summarizes recent advancements in experimental models—both in vitro and in vivo—that recapitulate the full spectrum of dormancy, including its induction, maintenance, and reactivation. Crucial intrinsic pathways such as ERK/p38 signaling shifts, epigenetic remodeling, and metabolic adaptations and microenvironmental and immune-mediated cues that regulate dormant cell fate are discussed. Therapeutic strategies aimed at maintaining dormancy, reactivating dormant cells for elimination, or directly targeting their survival pathways have been highlighted. By integrating insights from model systems, molecular regulation, and therapy, this review aims to provide a comprehensive framework that informs future efforts to target dormant cancer cells and ultimately reduce recurrence and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/780c88da1c462505181b638ab331143519a6fc32" target='_blank'>
              Understanding Tumor Dormancy: from Experimental Models to Mechanisms and Therapeutic Strategies
              </a>
            </td>
          <td>
            Jaebeom Cho
          </td>
          <td>2025-08-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The mechanism underlying the pathogenic impact of mutations within intrinsically disordered regions of proteins remains enigmatic, and the mechanisms behind compensatory responses to these perturbations lie within an even deeper veil of obscurity. This study focuses on the compensatory mechanisms of single nucleotide variants within the disordered C-terminal tail of the BTG2 protein, a crucial regulator in cell cycle control. Here, we develop a novel approach by combining molecular dynamics simulations with time-dependent linear response theory to accurately compute the long-distance coupling dynamics between the tail and the structured domain. Using this approach, we reveal how specific mutations can counteract the functional disruptions caused by a known disease-associated mutation, V141M. Our findings demonstrate that the disordered tail regulates critical binding sites allosterically, and a weakening of this modulation may contribute to disease manifestation. However, compensatory mutations restore lost interactions between the disordered region and binding sites, exerting long-distance dynamic control over both critical binding sites and the mutation site 141M. This secondhand allosteric control could be a general mechanism for compensatory mutations to rescue function. These insights not only illuminate the pathogenic mechanisms at play but also offer a framework for identifying potential therapeutic targets in diseases associated with disordered protein regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44953c05dc0147543f5bd00f5cc928d283c8664e" target='_blank'>
              Second-order allosteric control as a mechanism for compensatory mutations in B-cell translocation gene 2.
              </a>
            </td>
          <td>
            Nicholas J. Ose, P. Campitelli, Tushar Modi, S. Ozkan
          </td>
          <td>2025-08-25</td>
          <td>Protein science : a publication of the Protein Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite extensive prior research on prostate cancer (PCa) transcriptomics, the molecular mechanisms underlying the disease’s progression, particularly in the castration-resistant or metastatic stages, remain incompletely understood. The majority of recent research has concentrated on bulk RNA sequencing, which could mask the variation found in tumor microenvironments. This study aims to address this gap by integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing with weighted gene co-expression network analysis (WGCNA) to investigate the molecular mechanisms of PCa at a higher resolution. In order to further individualized treatment plans for PCa, we aim to discover important genes and signaling pathways that could be used as therapeutic targets. We first preprocessed expression profile data from prostate cancer tissue samples, selecting 9,809 high-quality cells from a dataset. Following batch correction with Harmony and dimensionality reduction with principal component analysis (PCA), we used the Louvain clustering algorithm to divide the cells into discrete subtypes. The clusters were then visualized using t-SNE. This resulted in 16 cellular subtypes categorized into five major cell types: epithelial cells, monocytes, endothelial cells, CD8 + T-cells, and fibroblasts. Analysis of receptor-ligand pairs uncovered significant interactions between monocytes and both tumor cells and endothelial cells. Applying the high-dimensional WGCNA (hdWGCNA) method to construct a gene co-expression network, we detected seven gene modules, four of which were highly expressed in tumor cell subtypes and contained 380 key genes. Combining pathway analysis, we ultimately screened six key genes: CNPY2, CPE, DPP4, IDH1, NIPSNAP3A, and WNK4. We used Cox univariate regression and least absolute shrinkage and selection operator (lasso) regression techniques to build a prognostic prediction model that included these six important genes based on clinical data gathered from PCa patients. The prognostic prediction model constructed in this study demonstrated excellent predictive performance in both the training set and an external validation set, with the high-risk group showing significantly lower overall survival (OS) than the low-risk group. Furthermore, there was a substantial correlation found between risk scores and several immune-related gene sets, chemotherapeutic drug sensitivity, and tumor immune infiltration. High- and low-risk groups exhibited significant differences in immune cell content, immune factor levels, and immune dysfunction. Further analysis revealed significant correlations between the expression levels of model genes and multiple disease-related genes. Through Gene Set Variation Analysis (GSVA) and Gene Set Enrichment Analysis (GSEA), we uncovered perturbations in multiple signaling pathways in high- and low-risk groups, potentially impacting the prognosis of PCa patients. This study uncovers key genes and signaling pathways in the prostate cancer tumor microenvironment, particularly genes such as CNPY2, CPE, DPP4, IDH1, NIPSNAP3A and WNK4, which have potential as therapeutic targets. Our findings provide new insights into personalized treatment strategies for PCa and warrant further clinical validation in the future. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15682-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779fa642ce8a3880c524211b95657271b91e6425" target='_blank'>
              Integrative analysis of molecular mechanisms in prostate cancer via single-cell RNA sequencing and weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Jing Zhai, Yizhou Wang, Yu Zhang, Wenhui Zhu, Xinyu Xu, Yu Peng, Guanxiong Ding
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Eukaryotic ribosomal RNA (rRNA) genes exhibit hyper-variability at non-conserved regions known as Expansion Segments (ESs). Due to the numerous rRNA copies in the genome, editing ESs is challenging, and their significance remains unclear. In this study, we analyze rRNA variant frequencies in the UK Biobank population, revealing that highly abundant ES variations are causally linked to human health and physiology. We developed a Ribosome Variation Analysis (RiboVAn) method, identifying both heritable germline variants and a larger proportion of low-heritability, likely somatic variants. The most heritable variants cluster within four ESs of the 28S rRNA, with specific variants in es15l associated with adiposity, es39l linked to body dimensions, and es27l associated with blood-related traits and diseases. Variant-chromosome specificity is observed where functional variants are linked to certain rDNA chromosomes. These findings causally link rRNA sequence variation to human traits and establish that ESs have distinct and important functions in human physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7471647088548ae368436bbc488b4110ad53685" target='_blank'>
              rRNA paralogs with variations, rRNA-subtypes, affect diverse human phenotypes
              </a>
            </td>
          <td>
            Daphna Rothschild, Jonathan K. Pritchard, Maria Barna
          </td>
          <td>2025-09-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d6f72ebd8a3733c60e8ee75cfbcd9b5da5d9c6" target='_blank'>
              Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes.
              </a>
            </td>
          <td>
            Zhe Liu, Yihang Bao, An Gu, Weichen Song, Guan Ning Lin
          </td>
          <td>2025-09-26</td>
          <td>Nature computational science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Hypoxia-inducible factor (HIF) is a master regulator of cancer cell adaptation to tumor hypoxia and is involved in cancer progression. Single-cell (sc) differences in the HIF response allow for tumor evolution and cause therapy resistance. These sc-differences are usually ascribed to tumor microenvironmental differences and/or clonal (epi)genetic variability. However, the sc-heterogeneity of the HIF response in otherwise identical cells cultured under defined in vitro conditions has not yet been addressed. Therefore, we analyzed the sc-response to hypoxia in nonclonal cell lines and multiple clonal derivatives, including HIF-1α or HIF-2α knockouts. While HIF-1α and HIF-1 target mRNA sc-heterogeneity was slightly higher than global transcription or specific housekeeping messenger RNAs (mRNAs), HIF-2α and especially HIF-2 target mRNA sc-heterogeneity was extraordinary, and remained in independent clones following HIFα knockouts. Unexpectedly, neither HIF-2α mRNA nor nuclear protein levels correlated with target mRNA levels. Unsupervised but not supervised HIF target gene dimensionality reduction revealed the initial sample composition after scRNA-seq, demonstrating that, owing to sc-heterogeneity, individual HIF target genes are not sufficient to unequivocally identify hypoxic cancer cells. In conclusion, the pronounced intrinsic sc-heterogeneity of the HIF response represents a hitherto unrecognized feature of cancer cells that impairs clinical HIF pathway-dependent cancer cell identification and targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b54dd117e53122de89768b0fc8a01ce303d40e2" target='_blank'>
              Inherent single-cell heterogeneity of the transcriptional response to hypoxia in cancer cells
              </a>
            </td>
          <td>
            Małgorzata Wilk, Thomas Knöpfel, Stana M Burger, S. N. Khodo, Roland H. Wenger
          </td>
          <td>2025-08-28</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematological malignancy driven by diverse genetic aberrations, with the Philadelphia chromosome and its resultant BCR-ABL1 fusion gene constituting key pathogenic drivers. Atypical BCR-ABL1 fusion transcripts have distinctive structural and functional properties. Structural divergence in these variants leads to functional alterations of encoded oncoproteins, potentially influencing disease progression and therapeutic responsiveness. Conventional diagnostic modalities, including reverse transcription-PCR and fluorescence in situ hybridization, may fail to detect rare variants, necessitating complementary high-sensitivity techniques such as next-generation sequencing). Tyrosine kinase inhibitors (TKIs), including imatinib and dasatinib, remain cornerstone treatments; however, marked inter-variant heterogeneity in TKI responsiveness is observed: Patients harboring e13a3/e14a3 transcripts generally show favorable prognoses, while those with e1a3/e6a2 variants demonstrate an increased risk of relapse and/or TKI resistance, often requiring multimodal strategies combining chemotherapy or allogeneic hematopoietic stem cell transplantation. Although Chimeric Antigen Receptor)-T cell therapy has shown promise in treating (Philadelphia chromosome-positive B-cell Acute Lymphoblastic Leukemia, its application in CML, particularly in variants such as e1a3 or e6a2, is not currently recommended as a first-line treatment. Despite advances in elucidating the clinical implications of fusion gene heterogeneity in leukemogenesis, the prognostic value of atypical BCR-ABL1 isoforms requires further validation through multicenter studies with extended cohorts. This review aimed to summarize cases of atypical fusion genes in CML, with analysis of clinical characteristics, therapeutic interventions, and prognostic outcomes, to provide clinicians with enhanced reference material for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec3c00c917da742543a693d4c05a294ce191255" target='_blank'>
              The silent players: Atypical BCR-ABL isoforms as biomarkers and therapeutic hurdles in CML pathogenesis (Review)
              </a>
            </td>
          <td>
            Xin Zhou, Ai Li, Dexiao Kong, Yuqi Shi, Peipei Zhang, Ningning Shan
          </td>
          <td>2025-09-24</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming.
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background ADTKD-MUC1 is caused by frameshift mutations in MUC1 gene that produce a frameshifted protein (MUC1fs) toxic to kidney cells. The gene’s variable number of tandem repeats (VNTR), with high GC content, makes it largely inaccessible to standard sequencing. As a result, both the reference sequence and natural variation in this region remain poorly defined, complicating mutation detection and data interpretation. Standard methods also fail to pinpoint the exact VNTR unit affected, limiting insight into mutation mechanisms and genotype–phenotype correlations. Methods We employed Single Molecule, Real-Time (SMRT) sequencing and characterized the genomic sequence of MUC1 in 300 individuals including 279 individuals from 143 families suspected of having ADTKD-MUC1. We compared these results to those obtained using the CLIA-approved mass spectrometry-based probe extension (PE) assay, which specifically detect the most prevalent 59dupC mutation. We correlated the structural features of the MUC1 VNTR with the rate of kidney function decline in affected individuals. Results We identified MUC1 consensus sequences for 205 unique VNTR alleles, with 9 distinct types of frameshift mutations present on 52 distinct mutated VNTR alleles. MUC1 frameshift mutations were identified in 71 of 143 families (50%) with suspected ADTKD, comprising 135 genetically affected individuals (48%). The SMRT assay exhibited complete concordance and revealed that the PE assay is capable of detecting frameshift mutations in approximately 85% of affected families. The constellation of VNTR structures supports a genotype–progression model, in which fast progressors exhibit a significantly lower number of repeat units on the wild-type allele and a higher number of repeats on the mutation-bearing allele, including an increased number of frameshifted repeat units. Conclusions SMRT sequencing outperforms current diagnostic methods for ADTKD-MUC1 and reveals the prognostic value of VNTR structures. Although their contribution to disease progression is modest (~6% variance explained), it remains biologically and clinically meaningful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcff66f92581ab116e48866b395087d3b037c6c9" target='_blank'>
              Long-Read Sequencing of the MUC1 VNTR: Genomic Variation, Mutational Landscape, and Its Impact on ADTKD Diagnosis and Progression
              </a>
            </td>
          <td>
            Alena Vrbacká, A. Přistoupilová, K. Kidd, Václav Janoušek, M. Radina, Petr Vyleťal, Ibrahim Bitar, V. Stránecký, Lenka Steiner-Mrázová, Helena Trešlová, J. Sovová, K. Hodaňová, H. Hartmannová, D. Mušálková, Klára Svojšová, Tereza Kmochová, V. Barešová, Abby Taylor, Lauren Martin, Antonio Sanchez, Romana Ryšavá, Innet Lajtmanová, Silvie Rajnochova-Bloudickova, O. Viklický, Gregorius Papagregorius, Constantinos Deltas, C. Stavrou, Sofia Jorge, José António Lopes, Márcia Rodrigues, E. Elhassan, Michelle Clince, Colm Rowan, Peter J. Conlon, Omri Teltsh, Gianpiero L Cavalleri, B. Blumenstiel, Diana Toledo, Marina DiStefano, M. DeFelice, Martina Živná, Anthony J. Bleyer, Stanislav Kmoch
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Single base substitution (SBS) mutations, particularly C > T and T > C, are increased owing to unrepaired DNA replication errors in mismatch repair-deficient (MMRd) cancers. Excess CpG > TpG mutations have been reported in MMRd cancers defective in mismatch detection (dMutSα), but not in mismatch correction (dMutLα). Somatic CpG > TpG mutations conventionally result from unrepaired spontaneous deamination of 5’-methylcytosine throughout the cell cycle, causing T:G mismatches and signature SBS1. It has been proposed that MutSα detects those mismatches, prior to error correction by base excision repair (BER). However, other evidence appears inconsistent with that hypothesis: for example, MutSα is specifically expressed in S/G2 phases of the cell cycle, and defects in replicative DNA polymerase proofreading specifically cause excess CpG > TpG mutations in signature SBS10b. Methods We analysed mutation spectra and COSMIC mutation signatures in whole-genome sequencing data from 1803 colorectal cancers (164 dMutLα, 20 dMutSα) and 596 endometrial cancers (103 dMutLα, 9 dMutSα) from the UK 100,000 Genomes Project. We mapped each C > T mutation to its genomic features, including normal DNA methylation state, replication timing, transcription strand, and replication strand, to investigate the mechanism(s) by which these mutations arise. Results We confirmed that dMutSα tumours specifically had higher CpG > TpG burdens than dMutLα tumours. We could fully reconstitute the observed dMutSα CpG > TpG mutation spectrum by adding CpG > TpG mutations in proportion to their SBS1 activity to the dMutLα spectrum. However, other evidence indicated that the SBS1 excess in dMutSα cancers did not come from 5’-methylcytosine deamination alone: non-CpG C > T mutations were also increased in dMutSα cancers; and, in contrast to tumours deficient in BER, CpG > TpG mutations were biased to the leading DNA replication strand, at similar levels in dMutSα and dMutLα cancers, suggesting an origin in DNA replication. Other substitution mutations usually corrected by BER were not increased in dMutSα tumours. Conclusions There is a CpG > TpG and SBS1 excess specific to dMutSα MMRd tumours, consistent with previous reports, and we find a general increase in somatic C > T mutations. Contrary to some other studies, the similar leading replication strand bias in both dMutSα and dMutLα tumours indicates that at least some of the excess CpG > TpG mutations arise via DNA replication errors, and not primarily via the replication-independent deamination of 5’-methylcytosine. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01525-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0be2946a1331e8205341cd8d8b7fbe7da7996" target='_blank'>
              Replication-associated mechanisms contribute to an increased CpG > TpG mutation burden in mismatch repair-deficient cancers
              </a>
            </td>
          <td>
            Joseph C Ward, Ignacio Soriano, Steve Thorn, Juan Fernández-Tajes, Kitty Sherwood, Güler Gül, Joost Scheffers, A. Frangou, Ben Kinnersley, Ioannis Kafetzopoulos, Duncan Sproul, Sara Galavotti, C. Palles, Andreas J. Gruber, David N. Church, Ian Tomlinson
          </td>
          <td>2025-08-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Solitary fibrous tumors (SFTs) of the central nervous system (CNS) are rare, aggressive mesenchymal neoplasms with high recurrence and metastasis rates. Here, we perform a comprehensive multi-omics analysis of 189 cases of CNS SFTs integrating 94 whole genome sequencing, 88 transcriptomics, 7 single-nucleus RNA sequencing and 3 spatial transcriptome sequencing. We find that receptor tyrosine kinase mutations are significantly more prevalent besides the widespread NAB2-STAT6 fusion and correlate with tumor grade. Transcriptomic analysis reveals four molecular subtypes—classical, neural-like, inflamed and migratory—each associated with distinct clinical and biological characteristics. Single-nucleus RNA sequencing identifies five tumor cell states, with the SFT_classical state serving as a precursor to other states influenced by hypoxia and inflammation. Notably, we identify FER kinase as a key therapeutic target, with FER inhibition significantly reducing tumor cell proliferation, migration and invasion. These findings provide important insights into CNS SFT biology and suggest potential therapeutic strategies for this challenging tumor type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b5476111a14e85f839f2a38bdf41cc7317dd08" target='_blank'>
              Molecular landscape, subtypes, and therapeutic vulnerabilities of central nervous system solitary fibrous tumors
              </a>
            </td>
          <td>
            Chenhui Zhao, Xueyan Hu, Xiudong Guan, Xiaojun Fu, Tingting Wang, Mengyuan Li, Xinze Liu, Jiarui Zhao, Di Wu, Fan Zhang, Jiaying Fu, Jiang Li, Tieqiang Zhang, Xiaochun Jiang, Changxiang Yan, Wang Jia, Ence Yang, Jian Chen
          </td>
          <td>2025-08-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Breast cancer impacts millions globally, with a significant annual rise in the incidence of new cases. Although considerable progress has been made in cancer therapies and numerous options exist to address this disease, the overall survival rates for individuals with breast cancer remain distinctly low. This situation is attributed to the emergence of resistance against chemotherapeutic agents used for breast cancer, along with the inadequate characteristics of the existing generation of oncological medications. Therefore, there is an imperative necessity for the creation of innovative treatments that exhibit minimal adverse effects, enhanced properties, and the absence of drug resistance. A noteworthy development in this field is the exploration of antimicrobial peptides (AMPs), which are natural constituents of the innate immune system, as a novel class of anticancer therapeutics. AMPs display multiple advantageous traits that make them attractive candidates for cancer therapy. In this review, we will review recent research focused on the application of AMPs in breast cancer therapy and determine their mechanisms of action. We will also investigate the potential of combining AMP-based treatments with existing breast cancer management approaches. This study highlights the promising potential of AMPs as a novel approach to treating breast cancer, providing promise for less harmful and more efficient therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad93dd213cc6dd2a6017685559170e58bf3aeb63" target='_blank'>
              Anticancer Potential of Antimicrobial Peptides: Focus on Breast Cancer.
              </a>
            </td>
          <td>
            Wen Yang, Hongyan Hou, Ran Chai, Jiaxiang Guo
          </td>
          <td>2025-09-24</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Malignant tumors remain one of the leading causes of mortality worldwide. Conventional therapies, though effective at killing cancer cells, are accompanied by systemic cytotoxicity. It highlights the priority for precision oncology. Tumor neoantigens represent promising targets for selective immunotherapy, arising from cancer-specific mutations. At the same time, CRISPRCas9 genome editing provides remarkable accuracy in genetic manipulation, opening new avenues for cancer research. This paper reviews the emerging convergence of CRISPRCas9 technology and neoantigen-based interventions. It showcases recent progress in CRISPR-based neoantigen validation and the engineering of immune effector cells, as well as their application in personalized therapies. Additionally, this paper focuses on several important translational challenges that need to be overcome before broad clinical application, including tumor heterogeneity, off-target effects, and clinical safety. By assembling the current evidence and future scenarios in this field, this review aims to highlight that the combination of CRISPR genome editing with neoantigen targets would be a new direction in the field of precision oncology that may shape the future of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a4a7b9e7b67fef6945b9291a7e77c60ecc9358" target='_blank'>
              Precision Oncology at the Intersection of CRISPRCas9 and Tumor Neoantigens
              </a>
            </td>
          <td>
            Qingya Wang
          </td>
          <td>2025-10-02</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Oestrogen receptor alpha (ERα)-positive (ER+) breast cancers are driven by the binding of 17β-oestradiol (E2) to ERα, which transcriptionally regulates target genes. Although microarrays and conventional RNA sequencing have identified E2 target genes, pre-designed probes and short read lengths are limited in their ability to accurately capture complex transcript structures. Long-read sequencing offers a solution by spanning entire transcripts, providing a more complete view of the transcriptome. Here, we employed nanopore long-read direct RNA sequencing (DRS) complemented with 3’-end sequencing, in vitro experiments, and deep learning-based protein modelling to explore the landscape of the E2-responsive transcriptome and protein level implications. Our analysis revealed a range of E2-responsive non-coding and coding isoforms, including intronically polyadenylated (IPA) mRNAs. One of these IPA isoforms was detected for TLE1, which assists ERα-chromatin interactions for a subset of E2 targets. The IPA isoform produces a C-terminus truncated protein that lacks the WDR interaction domain but retains dimerization/tetramerization capacity through its intact N-terminal Q-domain. Structural modelling and protein-based assays confirmed the dimerization potential and nuclear localization of the truncated protein. Functional assays showed that the overexpression of truncated TLE1 reduced the E2-induced upregulation of TFF1 and GREB1, E2-responsive genes, thereby disrupting transcriptional regulation. Importantly, a lower IPA isoform ratio is associated with worse survival in ER+ breast cancers, highlighting clinical relevance. Our study revealed new layers of complexity in the E2-regulated transcriptome, providing insights into truncated proteins. These findings contribute to a deeper understanding of gene regulation and may help the development of new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0bac48baf3ce0314e9eaa5b424b9e568bd79fa" target='_blank'>
              E2-regulated transcriptome complexity revealed by long-read direct RNA sequencing: from isoform discovery to truncated proteins
              </a>
            </td>
          <td>
            Didem Naz Dioken, Ibrahim Ozgul, Irem Yilmazbilek, Elanur Almeric, I. C. Eroglu, Mustafa Cicek, Utku Cem Yilmaz, Deniz Karagozoglu, Esra Çiçek, E. Karaca, Tolga Can, A. E. Erson-Bensan
          </td>
          <td>2025-09-18</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="This study aimed to identify novel prognostic biomarkers and therapeutic strategies by focusing on drug resistance-related phenotypes. We integrated multi-omics data from various databases encompassing sequencing, clinical information, resistance-related gene sets, and genomic alteration data. Single-sample gene set enrichment analysis (ssGSEA) was employed to calculate resistance scores for individual samples, which were subsequently applied in survival analysis. Furthermore, we utilized machine learning algorithms to develop a robust prognostic model validated across multiple independent datasets. Our findings revealed 12 genes consistently linked to PAAD prognosis in diverse datasets. Pathway enrichment analysis indicated that the high-risk group was enriched in pathways associated with systemic lupus erythematosus and the cell cycle, while the low-risk group showed significant enrichment in neuroactive ligand-receptor interaction pathways. Additionally, immune cell infiltration analysis exhibited substantial differences between risk groups, with the high-risk cohort presenting lower levels of activated CD8 + T cells but higher levels of regulatory T cells. The random survival forest (RSF) model demonstrated superior predictive performance, achieving a concordance index of 0.634 and time-dependent ROC AUCs of 0.973, 0.978, and 0.996 at 1, 2, and 3 years, respectively. In conclusion, this study identifies 12 critical drug resistance genes in PAAD and highlights the associated immune differences in patient risk, paving the way for targeted immunotherapy research to improve therapeutic strategies against this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9036a9dfcaada4e83527a56e65b2bda1ef503c41" target='_blank'>
              To Develop Prognostic Markers Related to Drug Resistance in Pancreatic Cancer Patients Based on Multiple Machine Learning Methods
              </a>
            </td>
          <td>
            Song Xu, Chuanmin Deng, Z. Man, Song Yang, Ming Xu
          </td>
          <td>2025-10-10</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bb8c6ba0429e042acd5b6b051c6318cb9fb300" target='_blank'>
              Identification, functional insights and therapeutic targeting of EMT tumour states.
              </a>
            </td>
          <td>
            Anqi Dong, Cédric Blanpain
          </td>
          <td>2025-09-25</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Located on chromosome 10, the RET gene encodes a transmembrane receptor tyrosine kinase that, when activated by a ligand, is essential for cell differentiation, proliferation, and survival. RET specifically supports the development of neuronal tissues and neural crest-derived cell types. However, RET is a proto-oncogene and mutations can lead to aggressive cancers like thyroid and lung tumours, or endocrine syndromes. Treatments for RET-driven cancers remain limited, relying on therapies that use RET-specific inhibitors. However, they show limited success in clinical settings due to adverse side effects and acquired resistance over time. To find alternative treatments, integrating the principles of synthetic dosage lethality (a genetic interaction causing a lethal effect in cancer cells) with machine learning may identify novel gene targets to complement existing therapies. In previous studies, we sought candidate genes that induce cell death when depleted in cells overexpressing oncogenic RET, but not in normal cells. A previously identified candidate, NCAM1, which is involved in neuronal cell migration, has been found to contribute to the differentiation of neural cell types. Using the SH-SY5Y neuroblastoma cell line as a model, multiple techniques and assays were performed to validate the synthetic lethal relationship between NCAM1 and RET. NCAM1 knockdown cell lines were generated using lentiviral transduction, and a live/dead cell assay was used to assess potential cytotoxic effects of gene knockdown. Additionally, a cycloheximide assay was performed to examine RET half-life by tracking its degradation over twelve hours. Over the summer, the project enabled us to reassess gene priority rankings using synthetic dosage lethal logic, characterize phenotypes of NCAM1 knockdown SH-SY5Y cells, optimize various assays under in-vitro conditions, and consolidate our understanding of NCAM1’s relationship to RET. The project thus demonstrates the value of combining bioinformatic and in-lab techniques as a productive approach to finding precise therapeutic targets against RET-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08538d4a42ece74adec0afb573a8e0619996891d" target='_blank'>
              Identifying Novel Gene Targets in RET-Driven Cancers Using Synthetic Dosage Lethality Models
              </a>
            </td>
          <td>
            Zahra Simpson-Stairs
          </td>
          <td>2025-08-29</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDA) remains one of the most lethal malignancies, largely due to limited therapeutic options and the rapid development of treatment resistance. Recent progress in the development of RAS targeting agents, including mutant-specific and pan-RAS inhibitors, has expanded potential treatment strategies for tumors harboring oncogenic KRAS mutations. However, clinical and preclinical evidence indicates that resistance to these agents emerges rapidly, limiting their long-term efficacy. This study aims to investigate the mechanisms underlying resistance to RAS inhibitors and to identify rational combination strategies that enhance therapeutic durability in PDA. Using advanced organoid and in vivo models, we are characterizing resistance to clinically relevant RAS inhibitors and evaluating the therapeutic potential of combining RAS inhibition with other targeted agents. We will also examine the influence of treatment timing, drug synergy, and tumor microenvironmental factors, including immune interactions, on treatment response. Ultimately, this research seeks to inform the development of more effective combination strategies for PDA by elucidating resistance mechanisms and identifying therapeutic vulnerabilities. Insights from this work may support future translational efforts in a disease with pressing unmet clinical needs.



 Vasiliki Pantazopoulou, Casie Kubota, Satoshi Ogawa, Kassidy Curtis, Araceli Herrera Morales, Dannielle Engle. Investigating combination therapies to overcome RAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B092.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb5ac3708ad5c62f9b5c3d584781b0becdc5e90" target='_blank'>
              Abstract B092: Investigating combination therapies to overcome RAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Vasiliki I. Pantazopoulou, Casie S. Kubota, Satoshi Ogawa, Kassidy Curtis, Araceli Herrera Morales, Dannielle D. Engle
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, , Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            , Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematologic malignancy originating from hematopoietic stem cells and driven by the BCR-ABL fusion oncogene. Imatinib (IM), a tyrosine kinase inhibitor, is commonly used as a frontline therapy for CML. However, some patients exhibit primary resistance or show persistent molecular evidence of disease despite treatment. Emerging studies indicate that exosome-derived microRNAs (miRNAs) play a role in mediating drug resistance and may serve as promising biomarkers for cancer diagnosis and predicting therapeutic response. This study aimed to investigate the plasma exosomal miRNA expression profiles in CML patients to identify potential biomarkers associated with IM resistance. Exosomes were isolated from plasma samples of both IM-sensitive and IM-resistant CML patients. The exosomal miRNA content was analyzed using RNA sequencing, followed by differential expression analysis, which revealed 13 upregulated and 21 downregulated miRNAs in IM-resistant patients. Subsequent bioinformatics analysis indicated significant enrichment in pathways related to autophagy and PI3K-Akt signaling. Notably, miR-451a and miR-16–2-3p were among the most significantly upregulated miRNAs in exosomes from IM-resistant individuals. Interestingly, miR-16–2-3p expression showed a strong inverse correlation with clinical laboratory results, specifically blood urea nitrogen and creatinine levels. This pilot study identified plasma exosomal miRNAs, particularly miR-451a and miR-16–2-3p, as potential biomarkers for imatinib resistance in chronic myeloid leukemia. Target gene prediction was performed to explore their regulatory roles. Despite the limited sample size, these findings enhance our understanding of drug resistance mechanisms and warrant further validation in larger cohorts to assess their clinical relevance and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa66d21071c796372f54b926f9da678ac08d63" target='_blank'>
              Exosomal miRNA expression profiling in patients with imatinib resistant Chronic myeloid leukemia: A pilot study
              </a>
            </td>
          <td>
            R. Navakanitworakul, Pirun Saelue, Tipparat Penglong, P. Molika, Natakorn Nokchan, Natta Tansila, H. Buncherd, Supinya Thanapongpichat, Kanitta Srinoun
          </td>
          <td>2025-08-29</td>
          <td>PLOS One</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd59fd638b49487acab5f65a27e1f99a668332d" target='_blank'>
              Mechanistic insights into MET exon 14 skipping mutations and their role in tumor progression.
              </a>
            </td>
          <td>
            Promita Ghosh, Isabella Pecora, Morag Park
          </td>
          <td>2025-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) has emerged as a highly versatile and robust isothermal amplification technology, offering exceptional sensitivity, specificity, and scalability for next-generation molecular diagnostics and multi-omics research. Its ability to generate long, repetitive DNA sequences with high fidelity has made it a pivotal tool in disease diagnostics, genomic analysis, and spatial transcriptome profiling. Recent advancements have expanded RCA into various formats, including solution-phase, solid-phase, hydrogel-based, and digital RCA, enhancing its analytical performance and adaptability across diverse biological applications. The integration of RCA with CRISPR-based detection, biosensors, and nanoparticle-assisted signal amplification has significantly improved its diagnostic potential, enabling ultrasensitive detection of nucleic acids, proteins, extracellular vesicles, and single cells. In addition to diagnostics, RCA has profoundly impacted genomic research, particularly in extrachromosomal circular DNA sequencing, offering insights into genome instability and cancer evolution. In spatial transcriptomics, RCA-based in situ RNA sequencing and hybridization have enabled high-resolution gene expression mapping in tissue architecture, providing invaluable insights into cellular heterogeneity and disease microenvironments. This review comprehensively explores the fundamental principles, design modifications, and advanced signal amplification strategies of RCA, with a focus on its applications in molecular diagnostics, genome analysis, and spatial transcriptome profiling. Additionally, it addresses key challenges, including reaction specificity, automation, and cost-effectiveness, and discusses future directions for integrating RCA into high-throughput molecular diagnostics, biomarker discovery, and single-cell and spatial multi-omics platforms. Through continuous innovation, RCA is poised to play a transformative role in precision medicine, next-generation sequencing, and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/576501d76c4b29ed935e06956faf76592e2b944c" target='_blank'>
              Rolling circle amplification for next-generation molecular diagnostics, genome analysis, and spatial transcriptome profiling.
              </a>
            </td>
          <td>
            Jueun Han, Yejin Song, Jihee Lee, Khulan Nyamzaya, Eunjung Kim
          </td>
          <td>2025-09-08</td>
          <td>Nanoscale</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cisplatin resistance presents a significant challenge in cancer therapy, emphasizing the necessity for identifying new regulatory elements that influence drug response. Recent research has revealed the importance of long noncoding RNAs (lncRNAs) in chemotherapy resistance, with LINC02381 identified as a potential regulatory factor. 
Through an in-depth bioinformatics analysis, we investigated the impact of LINC02381 on cisplatin resistance in ovarian cancer across various datasets. By conducting differential expression analysis, survival analysis, gene set enrichment analysis (GSEA), and constructing protein-protein interaction (PPI) networks, we identified key pathways associated with LINC02381 expression. 
The results indicated that the altered expression of LINC02381 in patients treated with cisplatin was associated with reduced survival. Functional studies and correlation analyses further demonstrated that this LncRNA influences critical pathways and genes related to apoptosis, efflux, DNA repair, and EMT. Lastly, through an examination of its interactions with microRNA and protein networks, we identified LINC02381 as a ceRNA implicated in cisplatin resistance. 
Our findings suggest that LINC02381 may influence cisplatin sensitivity in ovarian cancer and establish a basis for further experimental validation, including molecular assays or in vivo analyses, and suggest the potential therapeutic targeting of LINC02381 to combat chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a8ae366b9c19996905f6b9e1ed9f603948832f" target='_blank'>
              The Role of LINC02381 in Modulating Cisplatin Resistance in Ovarian Cancer: A Bioinformatics Approach
              </a>
            </td>
          <td>
            Zeinab Karbalaei Pazoki, B. M. Soltani, Mostafa Hosseini, Shiva Irani
          </td>
          <td>2025-09-26</td>
          <td>Iranian Journal of Allergy, Asthma and Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Breast cancer, particularly the luminal subtype, often responds to endocrine therapies. However, 20–30% of patients develop resistance, resulting in more aggressive disease. Long non-coding RNAs (lncRNAs) are implicated in cancer progression and treatment resistance. Objective: This study aimed to evaluate the role of the lncRNA insulin-like growth factor 2 antisense (IGF2-AS) in tamoxifen-resistant breast cancer and assess its potential as a therapeutic target. Methods: Two tamoxifen-resistant breast cancer cell lines (TAMR-V and TAMR-H) were used to assess IGF2-AS expression via qPCR. Knockdown experiments with siRNA evaluated the role of IGF2-AS in cell proliferation, invasion, and migration. Next-generation sequencing (NGS) analyzed gene expression differences between the cell lines. Kaplan–Meier survival analysis determined the clinical significance of IGF2-AS expression in breast cancer patients. Results: IGF2-AS expression was significantly upregulated in TAMR-V and TAMR-H cell lines compared to control MCF-7 cells. Knockdown of IGF2-AS reduced cell proliferation and invasion in TAMR-V cells but did not significantly affect TAMR-H cells, indicating a cell line-specific role in tamoxifen resistance. NGS revealed differential gene expression profiles between TAMR-V and TAMR-H cells, suggesting variability in resistance mechanisms. Survival analysis demonstrated that higher IGF2-AS expression was associated with poorer prognosis in breast cancer patients, including those with hormone-positive and triple-negative subtypes. Conclusions: IGF2-AS is upregulated in tamoxifen-resistant breast cancer and promotes cell proliferation and invasion in a cell line-specific manner. Its differential expression in TAMR-V and TAMR-H cells highlights the complexity of resistance mechanisms, suggesting IGF2-AS as a potential therapeutic target for overcoming tamoxifen resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1badd18e936bca9206a1c8e4ee429aebea737b56" target='_blank'>
              Differential Expression of Long Non-Coding RNA IGF2-AS in Tamoxifen-Resistant Breast Cancer Cells
              </a>
            </td>
          <td>
            J. Lee, B. Kang, E. A. Kim, Jieun Kang, Yeesoo Chae, Ho Yong Park, Soo Jung Lee, I. Lee, Ji-Young Park, Nora Jee-Young Park, J. Jung
          </td>
          <td>2025-08-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Emerging RAS inhibitors show promise in treating KRAS-mutated malignancies, but resistance mechanisms limit their clinical efficacy. Given recent clinical findings associating KRAS mutations with reduced response to neoadjuvant therapy in rectal cancer (RC), we aimed to investigate their impact on treatment outcomes and explore potential therapeutic strategies.


METHODS
We conducted a retrospective analysis of 390 rectal cancer patients to evaluate the association of KRAS mutations with disease-free survival (DFS) and response to therapy. We assessed the efficacy of KRAS inhibitors in rectal cancer cell lines, patient-derived organoids (PDOs), and patient-derived cell lines (PDCLs), and explored adaptive resistance mechanisms through transcriptomic profiling and unbiased drug screening experiments.


RESULTS
Mutant KRAS was associated with a reduced DFS and RCs harboring G12C and G12V mutations had less complete pathological responses to neo-adjuvant therapies. KRAS-mutated RC cells demonstrated adaptive resistance to KRAS inhibitors, characterized by transcriptomic restoration of oncogenic pathways, including MYC and E2F, and upregulation of ERBB2/3 expression. Consistently, drug screening identified EGFR family inhibitors as potent combinatorial partners, effectively overcoming KRAS inhibitor tolerance by inducing apoptosis. In patient-derived models, the pan-RAS inhibitor RMC-6236 combined with EGFR inhibitors demonstrated significant synergistic effects and prevented long-term tumor cell outgrowth.


CONCLUSION
Our findings point to the negative impact of KRAS mutations, particularly G12C and G12V, on RC treatment outcomes. Adaptive resistance by upregulation of ERBB genes limits the efficacy of KRAS inhibitors. Combining these with pan-ERBB inhibitors enhances anti-tumor effects in patient-derived cellular RC models, showing its potential as an alternative to the combination with anti-EGFR antibodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9be2135f6b5cf4f9e01a3b8ac31ac51a76a3401" target='_blank'>
              Pan-ERBB Inhibitors Synergize With KRAS Inhibitors in Rectal Cancer.
              </a>
            </td>
          <td>
            Jonas Buchloh, M. Spitzner, Hauke Zimmermann, Xin Fang, Constanza Tapia Contreras, Carolin Schneider, Tiago de Oliveria, Stefan Küffer, Michael Linnebacher, Felix Rühlmann, Lena-Christin Conradi, Matthias Wirth, Michael Ghadimi, Marian Grade, Jochen Gaedcke, Günter Schneider
          </td>
          <td>2025-09-16</td>
          <td>United European gastroenterology journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Epidermal growth factor receptor (EGFR) inhibitors are used to treat some patients with metastatic colorectal cancer, but many tumors do not respond or develop resistance during therapy. This review explores how understanding the molecular mechanisms of resistance can help guide treatment decisions. We highlight the concept of negative hyperselection, which uses extended genetic profiling to exclude tumors unlikely to benefit from EGFR inhibitors. In addition, we discuss the role of circulating tumor DNA, a blood-based test that monitors tumor evolution in real time. Together, these strategies form a precision oncology platform that supports more personalized and effective care. While these advances offer promise, challenges remain regarding cost, access, and demonstrating meaningful survival benefit in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716984fb2012cf7cb0f4128ef11a21a1a21d5293" target='_blank'>
              Anti-EGFR Therapy in Metastatic Colorectal Cancer: Identifying, Tracking, and Overcoming Resistance
              </a>
            </td>
          <td>
            Luís Felipe Leite, M. Noronha, Júnior Samuel Alonso de Menezes, Lucas Diniz da Conceição, Luiz F. Costa Almeida, A. P. Cappellaro, M. Belotto, Tiago Biachi de Castria, R. D. Peixoto, T. B. C. Megid
          </td>
          <td>2025-08-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Aim Cytarabine is the main chemotherapy agent used to treat acute myeloid leukemia (AML), but drug resistance remains a major challenge. Imbalances in cytokine secretion are known to play a role in the survival and proliferation of AML blast cells. While numerous studies have investigated cytokine secretion in AML, the precise role of cytokines in the pathogenesis of AML remains unclear. This study aimed to compare cytokine-related gene expression in parental and resistant HL60 cells and assess their prognostic relevance in AML. Materials and Methods This study compared gene expression profiles of parental HL60 and R-HL60 cells by analyzing gene expression arrays. Further, the correlation between the differential expression genes (DEGs) and overall survival in patients with AML was obtained from the Cancer Genome Atlas (TCGA) database, and the expression of genes was validated by quantitative PCR (QPCR). Results Our results showed that twenty-six genes involved in cytokine regulation were significantly different between HL60 and R-HL60 cells. The DEGs associated with cytokine production between parental HL60 and R-HL60 was subjected to a functional enrichment analysis by ShinyGO. The expression of BCL3, OAS3, RELB, AIM2, NFKB2, CLEC9A, and OAS2 involved in the regulation of cytokine production was associated with survival probability of AML patient in the TCGA database. Among them, the expression of BCL3, OAS3, RELB, AIM2, and NFKB2 genes in R-HL60 cells was higher than that in HL-60 cells. Conclusion This study identified key genes involved in cytokine dysregulation in cytarabine-resistant HL60 cells, providing potential targets for overcoming drug resistance in AML. These findings offer new avenues for the development of more effective therapies for relapsed or refractory AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2300df04048d3985552216d8d426868f79631fd" target='_blank'>
              Gene Expression Insights into Cytarabine Resistance in Acute Myeloid Leukemia: The Role of Cytokines
              </a>
            </td>
          <td>
            CHIEH-MING Lee, Yin-Hwa Shih, I-CHEN Chen, Tzong-Ming Shieh, Chung-Yu Yeh, Yu-Hsin Tseng
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Multidrug resistance is a significant barrier in cancer therapy largely due to poorly understood regulatory mechanisms. Here we reveal that certain anticancer drugs can bind to newly synthesized proteins prior to reaching their canonical targets, resulting in various forms of protein damage. This binding disrupts protein functions, particularly those of mitochondrial proteins, resulting in substantial cytotoxicity. The protein damage is further exacerbated by mitochondrial reactive oxygen species generated as a consequence of the initial damage, creating a positive feedback loop. In response, cancer cells rapidly initiate a chain of events, which we term the Protein Damage Response (PDR). This includes damage recognition primarily mediated by protein ubiquitination and subsequent damage clearance via the proteasome system. Notably, patients with advanced, drug-resistant metastatic breast or colon cancers exhibit elevated proteasome activity. In an effort to predict drug resistance, we developed a sensitive kit for detecting proteasome levels, enabling the identification and subtyping of patients with high proteasome activity to support tailored therapeutic strategies. Using a three-dimensional tumor slice culture-based drug sensitivity assay and an investigator-initiated clinical trial, we demonstrate that three clinically approved proteasome inhibitors effectively overcome multidrug resistance in colon and breast cancer patients with elevated proteasome activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1785188ad7f7984f485f66a5779a54c95bdf57ed" target='_blank'>
              Suppressing protein damage response to overcome multidrug resistance in cancer therapy
              </a>
            </td>
          <td>
            Fangyuan Shao, Zongjie Li, Hao Xiao, Yujun Chen, Yuheng Zhang, Ling Li, Yuzhong Peng, Xinyi Li, Yuxing Hou, Bo Li, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Xi Han, Jiaxin Yao, Cuiting Zhang, Lijian Wang, Haifeng Li, N. Shao, Kai Miao, Xiaoling Xu, Yanxia Shi, Changhua Zhang, Jun Yan, Ying Lin, Chu-Xia Deng
          </td>
          <td>2025-09-30</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7ab6bec4cbb03839af3fabebb27d4a6903a92d9" target='_blank'>
              Genome-wide characterization of clonal hematopoiesis reveals extensive non-coding putative driver mutations
              </a>
            </td>
          <td>
            J. Weinstock, K. Conneely, J. Woo, M. Arvanitis, M. J. Machiela, C. Russell
          </td>
          <td>2025-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Carcinogenesis, while traditionally attributed to the accumulation of driver mutations in genes regulating cell proliferation and apoptosis, may also be explored as a consequence of fundamental metabolic reprogramming, an idea catalyzed by the Warburg effect, where cancer cells exhibit a paradoxical preference for glycolysis over the far more efficient oxidative phosphorylation. This implies that metabolic dysregulation may be a primary instigator of neoplastic transformation. Our hypothesis proposes that the abrupt loss of cellular energy may stimulate an atavistic response, wherein rapid proliferation and migration are triggered to enhance survival in fluctuating environments. These responses lead to pathological angiogenesis and unchecked cell growth, thereby bridging the gap between genetic and metabolic pathways of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba64bccb0f058f3f967d3066ee9360fe210d037d" target='_blank'>
              Carcinogenesis: An Alternative Hypothesis Comparing Mutagenic Versus Metabolic Models
              </a>
            </td>
          <td>
            A. Alhatem, C. Gagna, Muriel Lambert, Emily Keenan, W. Lambert
          </td>
          <td>2025-09-23</td>
          <td>Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Alternative splicing in HIV-1 has been a central focus of decades of research, uncovering key mechanisms of viral gene regulation, immune evasion, and therapeutic response – yet, no reference resource has existed to support transcriptome-wide analysis, limiting adoption of modern computational methods. We present HIV Atlas (https://ccb.jhu.edu/HIV_Atlas), the first reference-quality annotation of HIV-1 and SIV transcriptional diversity. We manually curated transcriptomes for HIV-1HXB2 and SIVmac239 and developed Vira, an automated annotation-transfer method specifically designed to address unique challenges of viral genome biology, to generate high-quality annotations for 2,077 complete HIV-1 genomes. Using the resources presented in our work, we evaluated conservation of splice sites, revealing near-perfect preservation of major donors and acceptors. Furthermore, using several public datasets, we demonstrate how HIV Atlas enhances methodology, improves the quality and novelty of results, and opens novel avenues for research, supporting more accurate and comprehensive analyses of bulk, single-cell, and spatial RNA-seq in HIV-1 studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1270dba893a67448bd87a98a989e0077c4e63841" target='_blank'>
              Comprehensive Transcriptome Annotation of Thousands of HIV-1 Genomes
              </a>
            </td>
          <td>
            Ales Varabyou, Mykhaylo Artamonov, Sophia Cheng, Diane L. Bolton, Steven L Salzberg, M. Perțea
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cardiac myxoma (CM), a rare primary cardiac tumor, poses significant life-threatening risks. Current CM research has remained largely limited to clinical case observations and pathological analyses, thus restricting its clinical therapeutic impact. Fundamental research should be urgently strengthened to better support future CM treatment strategies. In this work, single-cell sequencing is used to elucidated the intricate cellular composition of the CM microenvironments. The mechanisms of heart myxoma cell growth are investigated via proteomics and organoid models, while our western blot analysis reveals cardiac myxoma’s immune evasion strategies. This study successfully characterizes diverse cell types within the CM microenvironment. Notably, ap-CAF cells are found to effectively recruit immune cells via chemokine secretion, fostering immune microenvironment formation. The work’s pseudotime trajectory analysis also demonstrates that CM tumor cells derive from mesenchymal stem cells. Additionally, this work demonstrated that the glycolysis pathway is significantly activated and fuels CM cell growth. Tumor cells exploit the SIRPα-CD47 immune checkpoint to evade the immune system by inhibiting antigen-presenting cell phagocytosis. Tumor-associated macrophages (TAMs) concurrently assume M2 polarization and suppress autoimmune activity through IL-10. This research comprehensively examines CM’s microenvironmental cellular architecture, metabolic features, and immune escape mechanisms. These work’s findings not only deepen the current understanding of CM’s biological nature but also offer vital theoretical foundations for developing safer, more effective CM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaa9c5f44a7fada1677fd86632e043c456f245b" target='_blank'>
              Single-cell RNA sequencing and proteomics uncover SIRPα-CD47 immune checkpoint and glycolysis-driven immune evasion in cardiac myxoma
              </a>
            </td>
          <td>
            Xinyuan Zhu, Qingle Gao, Xintong Dai, Baofa Sun, Yanping Li, Hongyan Zhai, Changliang Shan
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4671f71130c02c103abea19511db04e587b6744b" target='_blank'>
              Evolutionary drivers of divergent collateral sensitivity responses during antibiotic therapy.
              </a>
            </td>
          <td>
            Sander K. Casier, Bram Lories, Hans P. Steenackers
          </td>
          <td>2025-08-22</td>
          <td>Nature ecology & evolution</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54525feaa0ad7b16a4eefd8a46c2d6fd3ccc01e5" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of Single-Nucleotide Variant and Copy Number Aberration Evolution from Single-Cell DNA Sequencing of Tumors.
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2025-09-15</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT The DNA damage response (DDR), an evolutionarily conserved surveillance mechanism enabling cells to detect genomic lesions and cell cycle arrest for repair, coordinates genomic stability through key effector kinases. In Candida albicans, our prior work characterized Rad53-mediated transcriptional reprogramming based on the RAD53 deletion strain, establishing its canonical DDR functions. However, Rad53 activation dynamically orchestrates cell cycle synchronization and repair coordination during genotoxic challenge. Investigating cellular modifications resulting from increased Rad53 functionality is essential for a deeper understanding of its regulatory response. To this end, we overexpressed Rad53 revealing its dual regulatory capacity in stress adaptation and developmental plasticity. Global transcriptomic profiling demonstrated that RAD53 overexpression enhances methyl methanesulfonate (MMS) resistance while paradoxically stimulating filamentous growth, accompanied by upregulation of DNA repair/replication genes (RFA3, POL30) and non-canonical DDR targets including cell wall remodeling factors. Comparative analysis of RAD53 deletion and overexpression transcriptomes identified enriched transcription factors, notably Sfl1, Yox1, and Rfx1. In particular, Rfx1 mediated the resistance to genotoxic stress resulting from RAD53 overexpression or deletion. Surprisingly, Rad53 overexpression substantially hindered hyphal formation and virulence, whereas its kinase-dead mutation restored these deficiencies. Notably, Rad53’s role in mediating morphogenesis was independent of Rfx1. This multi-modal functionality positions Rad53 as a critical node interfacing genomic stability, developmental plasticity, and pathogenicity in C. albicans. Our study enhances the understanding of the DDR pathway in C. albicans, providing a potential target for treating this fungal pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618ff870ecbc422dc2701275aa24d5074e975660" target='_blank'>
              Rad53 orchestrates divergent pathways governing genotoxic resistance, morphogenesis, and virulence in Candida albicans: Mechanistic decoupling through overexpression analysis
              </a>
            </td>
          <td>
            Jia Wang, Shaling Yang, Yi Zhou, Huaxin Cai, Jinrong Feng
          </td>
          <td>2025-10-06</td>
          <td>Virulence</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d108c58970b821d1c2170e04d8c4196841969e5f" target='_blank'>
              Phage therapy and its role in cancer treatment and control.
              </a>
            </td>
          <td>
            H. A. Abd El-Moaty, A. Doghish, Hebatallah Ahmed Mohamed Moustafa, Wagiha S. Elkalla, Ghadir A Sayed, Nourhan H Elshami, S. S. A. Mageed, Osama A Mohammed, Mahmoud A. Elrebehy, Kareem AlFarsi, Amr M. Abdelfatah, Aly M Waseem, Ahmed O Abdulkader, Rabab S. Hamad, Akram N Salah
          </td>
          <td>2025-09-17</td>
          <td>Folia microbiologica</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Modeling cancer recurrence and associated conditions such as cancer cell dormancy remains a significant challenge in cancer research. We developed a novel in vitro model based on platinum-resistant tumor cells treated with a platinum drug, in which cells progress through all stages of recurrence within 30–40 days post-exposure. Our results demonstrate that treatment of platinum-resistant colorectal cancer cells with oxaliplatin enriches the population with quiescent, dormant cells arrested in the G0/G1 phase. These cells exhibit increased autophagy, elevated expression of stem cell markers, reduced reactive oxygen species (ROS) levels, and heightened resistance to chemotherapy. A key advantage of our model is the high survival rate of residual cells, enabling the maintenance of a large cell population. This facilitates the application of conventional research techniques that are often limited in other recurrence models due to the small size of residual populations. Overall, this model provides opportunities for a wide range of further research for studying specific recurrence- and dormancy-associated signaling pathways and for identifying novel therapeutic targets for preventing tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4b53e10213b4d15495530a1d5bcc0ce84af24bb" target='_blank'>
              An in vitro tumor recurrence model based on platinum-resistant colon cancer cells as a research tool for studying cancer cell dormancy
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, M. Igotti
          </td>
          <td>2025-10-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="INTRODUCTION
This systematic review assesses the role of the tumor microenvironment (TME) in cancer progression and therapy resistance by defining drug-microenvironment interactions and determining the molecular determinants in the TME that could help improve the efficacy of administered treatments and alleviate existing adverse effects.


METHODS
This systematic review follows the PRISMA protocol and the PICOS selection framework to retrieve studies from PubMed/MEDLINE, Web of Science, Scopus, and the Cochrane Library. Only original human-related research published in English between 2008 and 2023 was used to explore the reciprocal relation between tumor cells and TME components. The ROBINS-I tool assessed the risk of bias.


RESULTS
Out of 258 articles initially identified, 15 met the inclusion criteria for this review. The results showed that TMEs significantly influence treatment outcomes in cancer progression, metastasis, and drug resistance. Focusing on TMEs like CAFs, immune cells, and ECM enhances drug efficacy. The study highlighted potential strategies to improve drug delivery, suppress metastatic processes, and restore immune function, ultimately leading to better outcomes for cancer patients.


DISCUSSION
Original evidence suggests that Cancer-Associated Fibroblasts (CAFs), immune cells, and Extracellular Matrix (ECM) contribute to therapeutic resistance and metastasis within the TME. They also promote metastasis by inducing Epithelial- Mesenchymal Transition (EMT) and affecting Cancer Stem Cell (CSC) populations. Moreover, the immunosuppressive TME consists of regulatory T cells and myeloidderived suppressor cells that allow tumors to evade the immune system, a concern for immunotherapy.


CONCLUSION
The TME plays a vital role in cancer development, metastasis formation, and therapy failure. The perspectives for innovative ECM-modulating treatments and interventions targeting the direct interactions between TME and cancer cells can be revolutionary and suggest better outcomes for treatment-naïve and refractory cancers. Future research should use these results as inputs to apply clinical and therapy studies to enhance cancer management outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef86b2266fb89e2b550a09af051f3aa0d3d175c" target='_blank'>
              The Tumor Microenvironment: Impact on Tumor Growth, Metastasis, and Therapeutic Resistance: A Systematic Review.
              </a>
            </td>
          <td>
            Jiazhi Yan, Shuo Tu
          </td>
          <td>2025-09-26</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Cell identity, tightly regulated by transcription factors (TFs), is frequently disrupted in cancer, driving tumor progression and metastasis. In pancreatic ductal adenocarcinoma (PDAC), alterations in cell identity contribute to disease heterogeneity and therapy resistance. Integrated genomic, transcriptomic, and proteomic analyses have defined two major PDAC subtypes: classical and basal. Notably, recent evidence suggests that changes in subtype identity can enable resistance to targeted therapies. These subtypes are distinguished by relative levels of endodermal lineage-specifying TFs such as FOXA1/2 and HNF4α, which are maintained in the classical subtype but downregulated in the basal subtype. Understanding how these TFs shape cellular identity is critical for identifying therapeutic vulnerabilities and mechanisms of drug resistance. We have shown that HNF4⍺ promotes the classical subtype program of PDAC. Although FOXA1/2 and HNF4α regulate gene expression in normal tissues, it is still unclear whether such cooperation is necessary to drive the classical transcriptional program in PDAC. We hypothesize that FOXA1/2 and HNF4α cooperatively regulates the gene expression program that regulates the classical subtype program in PDAC. Using human cell lines and murine PDAC organoid models, we reconstituted and modulated TF expression to evaluate downstream gene expression and TF genomic binding via ChIP-seq. Conditional alleles of FoxA1/2 in our murine organoid lines reveal that loss of FOXA1/2 lead to a decrease in the classical program. When we overexpress HNF4α in FOXA1/2-null organoids, it leads to a decrease in organoid growth; therefore, illustrating the ability of HNF4α to regulate growth independent of FOXA1/2. In a human cell line, we find that co-expression of FOXA1/2 and HNF4a activates the classical B signature while FoxA2 alone activates the Classical A signature, indicating that FOXA2 and HNF4a activate subtype-specific transcriptional programs. Additionally, we demonstrate that the overexpression of HNF4a in a subcutaneous xenograft model confers resistance to KRASG12C inhibition, leading to an increase in tumor size and weight. Unexpectedly, we find that these TFs do not cooperatively regulate gene expression; rather, in the human cell line, FOXA2 interferes with HNF4α genomic binding, suggesting a competitive or inhibitory interaction. Overall, our work highlights how lineage-defining TFs govern PDAC cell identity and therapeutic response.



 Walter A. Orellana, Sydney N. Larsen, Eric L. Snyder. FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B107.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10be5668747dfa95fe8d23992fe6617771de029e" target='_blank'>
              Abstract B107: FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Walter A Orellana, Sydney N. Larsen, Eric L. Snyder
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. V. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer remains the most frequently diagnosed malignancy among women worldwide, with significant morbidity and mortality rates despite advances in early detection and treatment. This review provides a comprehensive overview of recent developments in breast cancer research, spanning genetic and molecular insights, novel diagnostic techniques, and emerging therapeutic strategies. The advent of next-generation sequencing and multi-omics approaches has deepened our understanding of tumor heterogeneity, revealing key genetic drivers, epigenetic regulators, and the role of cancer stem cells in disease progression. Early detection strategies have also evolved with digital breast tomosynthesis and molecular breast imaging, offering improved sensitivity and specificity. On the therapeutic front, breakthroughs in targeted treatments—including cyclin-dependent kinase 4/6 and phosphoinositide 3-kinase inhibitors, antibody–drug conjugates, and immune checkpoint inhibitors—have transformed patient outcomes. The integration of chimeric antigen receptor T-cell therapy and mRNA-based therapeutics holds great promise in overcoming treatment resistance and improving long-term survival. However, challenges such as treatment accessibility, drug resistance, and disparities in healthcare persist, particularly in low- and middle-income regions. Emerging technologies, including artificial intelligence-driven diagnostics and risk-adapted screening, are paving the way for more precise and personalized interventions. This review highlights the latest innovations and ongoing challenges in breast cancer research, emphasizing the need for continued efforts to translate scientific advancements into clinical practice to improve patient outcomes globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c3c761954bdae167d685c04a34eda5ce49f0c7" target='_blank'>
              Clinical advancements in breast cancer research: A comprehensive review
              </a>
            </td>
          <td>
            P. Puvvula, Ria Sahasra Puvvula
          </td>
          <td>2025-09-23</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer (BC) recurrence remains a major clinical challenge, leaving patients in perpetual uncertainty about disease relapse after primary treatment. BC dormancy, an adaptive survival state of disseminated tumour cells, is a key driver of both early and late recurrence. However, the mechanisms regulating BC dormancy remain poorly understood. Emerging evidence suggests that tumour hypoxia, extracellular matrix (ECM) remodelling, and therapy-induced stress drive dormancy by altering cellular metabolism, gene expression, and immune interactions, enabling long-term survival of dormant BC cells. With no dormancy-specific therapies currently approved, a deeper understanding of dormancy-associated survival mechanisms is crucial for identifying therapeutic targets and developing strategies to eradicate dormant BC cells, thereby preventing recurrence and improving patient outcomes. This review comprehensively examines major dormancy-inducing factors and the adaptive survival mechanisms of dormant BC cells. We also highlight critical gaps in preclinical models that hinder the translation of preclinical cancer dormancy insights into clinical applications and propose potential therapeutic strategies to prevent BC recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce26cf9af7969ba38df185997f5bd7aa995c5f0" target='_blank'>
              Decoding the adaptive survival mechanisms of breast cancer dormancy
              </a>
            </td>
          <td>
            Francis M Barnieh, Jamie Morton, Olaitan Olanrewaju, Sherif F. El-khamisy
          </td>
          <td>2025-08-27</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="


 Aberrant pre-mRNA splicing is a hallmark of many cancers, yet the underlying genetic mechanisms driving these alterations remain incompletely understood. While splice-altering variants can explain a subset of events, many tumor-specific splicing changes arise in the absence of nearby sequence variants. This study aims to systematically dissect the contribution of proximal splice-altering variants to tumor-specific splicing in pediatric brain tumors and to define the residual cohort likely driven by regulatory mechanisms such as mutations affecting splicing factors or the spliceosome machinery. We hypothesized that some fraction of tumor-specific splice events in pediatric brain tumors are not associated with nearby genetic variants, and that these may highlight broader disruptions to splicing regulation.



 We applied replicate Multivariate Analysis of Transcript Splicing (rMATS-turbo) to stranded total RNA-seq data (N=635) from the Pediatric Brain Tumor Atlas (PBTA). Tumor-specific single exon (SE) splicing events were defined as a change in percent spliced in (| ΔPSI |) > 0.3 between tumors and healthy brain samples sourced from GTEx (<40 years, n=2,642) and pediatric normals (n=7). Tumor-normal matched whole genome sequencing data were used to identify somatic and rare (<0.1% Allele Frequency in gnomad) germline SNVs/Indels proximal (<250nt) to rMATS-defined splice junctions.



 Across the cohort (N=635), we identified 107,827 tumor-specific SE alternative splicing events. Only 44 tumors harbored a variant proximal to the affected splice site (11 germline, 39 somatic), leaving most events without a clear cause. We are now investigating this variant-negative subgroup as a preliminary cohort. Current efforts focus on excluding structural drivers before turning to potential regulatory mechanisms, including the disruption of spliceosome components. Preliminary findings suggest some tumors show altered expression or mutations in spliceosome components, though further validation is needed.



 This dual-ended analysis of tumor-specific splicing events reveals that while a subset can be attributed to local genetic variation, the majority likely arise from broader disruptions in splicing regulation. These findings underscore the importance of integrating RNA- and DNA-based analyses to fully capture the molecular basis of splicing dysregulation in cancer. Moreover, the variant-negative cohort may serve as a rich resource to uncover novel regulatory mechanisms and potential therapeutic targets in splicing-driven pediatric brain tumors.



 Patricia J Sullivan, Ryan J Corbett, Ammar S Naqvi, Alexander Sickler, Rebecca Kaufman, Bo Zhang, Chuwei Zhong, Sharon J Diskin, Jo Lynne Rokita. Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0c6326494c3f8ecc5b90e1fc51e9fc24433d6c" target='_blank'>
              Abstract B031: Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors
              </a>
            </td>
          <td>
            Patricia J Sullivan, Ryan J. Corbett, Ammar S. Naqvi, Alexander Sickler, Rebecca S. Kaufman, Bo Zhang, Chuwei Zhong, S. Diskin, J. L. Rokita
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The tumor microenvironment (TME) is a complex network driving endometrioid endometrial cancer (EC) progression. Previous analyses of the EC TME were often limited by a lack of detailed cellular annotations. Integrating single-cell RNA sequencing (scRNA-seq) data offers a comprehensive approach to reconstruct the TME, aiming to uncover novel mechanisms and therapeutic targets. Methods We integrated public scRNA-seq data from 15 EC and 5 normal endometrium samples. Through detailed cell annotation, we performed comprehensive bioinformatic analyses, including pseudotime trajectories, copy number variation, and cell communication, to investigate mechanisms of EC development. Results We identified nine cell types and characterized subpopulations of epithelial, macrophage, lymphocyte, and stromal fibroblast cells. The SOX9+LGR5- epithelial subtype showed elevated malignancy and NFKB pathway enrichment. The M2_like2 macrophage subtype played a critical role, engaging in robust MIF-(CD74+CD44) mediated communication with SOX9+LGR5- cells. Experimental validation confirmed MIF co-expression with E-cadherin in EC tissues. Furthermore, the transcription factor NFKB2 was found to mediate MIF’s effect on the CD44 receptor in malignant epithelial cells. A pericyte-to-fibroblast transition in stromal cells may also support tumor growth, while an increase in CD8 exhausted/Treg cells and a decrease in cytotoxic CD8 cells suggest potential immune evasion. Conclusion Our single-cell analysis details the EC TME landscape, revealing robust communication between M2_like2 macrophages and SOX9+LGR5- epithelial cells. We highlight a key mechanism where NFKB2 mediates MIF’s pro-tumorigenic effects via the CD44 receptor, offering new insights into EC progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84d976acc17e07f02157c06df3f07e8d3258236f" target='_blank'>
              Single-Cell Transcriptomic Analysis Highlights the Impact of NFKB2-Mediated MIF-CD44 Signaling Axis in Endometrioid Endometrial Cancer
              </a>
            </td>
          <td>
            Lu Zhang, Mengjie Yang, Quan Zhang, Yiqian Zhang, Qiyuan Li, Qionghua Chen
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Women's Health</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Recent bulk analysis of Chinese hamster ovary (CHO) cell mitochondrial DNA revealed widespread heteroplasmy across cell lines and even within clones of the same parental host. To address this, we applied our previously developed single‐cell mtDNA sequencing (scmtDNAseq) method to 84 single CHO cells. We identified widespread intercellular heteroplasmy across the CHO cell population and predicted possible phenotypic impacts. 3/11 (27%) of the most variable mutations were only identified by scmtDNAseq, indicating greater resolution when compared to bulk cell analysis. Single‐cell RNAseq (scRNAseq) was also performed at the same time point and, compared to scmtDNAseq, significant differences in intercellular heteroplasmy were observed. Using an inducible mAb expression system demonstrated that short‐term additional biosynthetic burden of exogenous protein production had little impact on intercellular heteroplasmy. We additionally monitored bulk heteroplasmy over 38 days, reflecting the typical timespan from vial thaw to production vessel in a Biopharmaceutical upstream cell culture process. We observed minimal change in heteroplasmy, finding no evidence that a mAb‐producing CHO cell line develops impactful changes in heteroplasmy over that timeframe. This would suggest that for our cell line, the heteroplasmy profile established on Day 1 should be maintained throughout a full fed‐batch bioprocess run.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d8c2d19d7057b8a8f3648254122a541737446d" target='_blank'>
              Single‐Cell Mitochondrial DNA Analysis of Recombinant Chinese Hamster Ovary Cells Reveals Widespread Heteroplasmy
              </a>
            </td>
          <td>
            Alan Foley, N. Lao, Ciara McGuirk, Colin Clarke, Niall Barron
          </td>
          <td>2025-09-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Leiomyosarcoma (LMS) is a rare and aggressive soft tissue sarcoma with limited therapeutic options and poor prognosis. Identifying reliable prognostic markers and therapeutic targets is critical for improving personalized treatment strategies. Methods We integrated single-cell RNA sequencing (scRNA-seq) data of LMS from the Gene Expression Omnibus (GEO) with bulk RNA-seq and clinical data from The Cancer Genome Atlas (TCGA). Malignant cells were identified using machine learning approaches, and their stemness index was calculated. Cells were stratified into high and low stemness index groups, and differential gene expression analysis was performed. Prognostic markers were identified through a sequential pipeline: univariate Cox regression to screen candidate genes, Lasso regression for feature selection, and multivariate Cox regression for model construction and survival analysis. Results Cells with a high stemness index exhibited a more complex tumor immune microenvironment (TIME) and enhanced intercellular interactions compared to those with a low stemness index. Differential expression analysis identified genes distinguishing high versus low stemness cells. Through the regression pipeline, six prognostic markers were identified: BOP1, CTBP1, DSE, PMSD10, SRPK1, and HACD4. These markers were significantly associated with tumor cell proliferation and patient survival outcomes. Discussion Our findings suggest that stemness-related heterogeneity in LMS shapes the tumor immune microenvironment and contributes to disease progression. The six identified prognostic markers not only provide insights into the molecular mechanisms of LMS but also represent potential therapeutic targets for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8914a51d2e0d775a0ec016871e9455ef5edac1b" target='_blank'>
              Integrated single-cell and bulk RNA-seq analysis reveals prognostic stemness genes in leiomyosarcoma
              </a>
            </td>
          <td>
            Aiju Lou, Yu Cai, Tuquan Zheng, Long Zhang, Yupeng Hu, Huachi Li, Lang Ying
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03559-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Meiotic crossovers promote correct chromosome segregation and the shuffling of genetic diversity. However, the measurement of crossovers remains challenging, impeding our ability to decipher the molecular mechanisms that are necessary for their formation and regulation. Here we demonstrate a novel repurposing of the single-nucleus Assay for Transposase Accessible Chromatin with sequencing (snATAC-seq) as a simple and high-throughput method to identify and characterize meiotic crossovers from haploid testis nuclei. We first validate the feasibility of obtaining genome-wide coverage from snATAC-seq by using ATAC-seq on bulk haploid mouse testis nuclei, ensuring adequate variant detection for haplotyping. Subsequently, we adapt droplet-based snATAC-seq for crossover detection, revealing >25 000 crossovers in F1 hybrid mice. Comparison between the wild type and a hyper-recombinogenic Fancm-deficient mutant mouse model confirmed an increase in crossover rates in this genotype, however with a distribution which was unchanged. We also find that regions with the highest rate of crossover formation are enriched for PRDM9. Our findings demonstrate the utility of snATAC-seq as a robust and scalable tool for high-throughput crossover detection, offering insights into meiotic crossover dynamics and elucidating the underlying molecular mechanisms. It is possible that the research presented here with snATAC-seq of haploid post-meiotic nuclei could be extended into fertility-related diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b651b832a12539ba036217c6dc855d9399ad822" target='_blank'>
              A high-resolution meiotic crossover map from single-nucleus ATAC-seq reveals insights into the recombination landscape in mammals
              </a>
            </td>
          <td>
            S. Novakovic, Caitlin Harris, Ruijie Liu, Davis J. McCarthy, W. Crismani
          </td>
          <td>2025-09-01</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80e02b4b634cdeb2be956bc4074a559e6c262ab4" target='_blank'>
              Integrating large-scale in vitro functional genomic screen and multi-omics data to identify novel breast cancer targets.
              </a>
            </td>
          <td>
            Hao-Kuen Lin, J. Dai, Lajos Pusztai
          </td>
          <td>2025-09-01</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 Structural rearrangements are increasingly understood to signiﬁcantly alter the prognosis of a wide range of CNS tumours. For example, co-deletion of CDKN2A/B is a hallmark of high-grade astrocytoma, oligoden- droglioma and meningioma. Despite their clinical importance, detection of intra-chromosomal rearrangements is challenging using short-read technologies and copy number variations (CNV) are commonly inferred from methylation array. Such testing has limited resolution to detect small, intra-chromosomal CNV calls with high conﬁdence. Nanopore technology is an emerging technique in CNS diagnostics, enabling targeted long-read sequencing alongside methylome classiﬁcation. However, current implementations of adaptive targeting are restricted to an a priori deﬁned set of key driver oncogenes. In contrast, genomic breakpoints underpinning SVs are unique to individual patients and are thus challenging to sequence at high coverages.



 Here, we present BATMAN, a development of the ROBIN diagnostic tool that enables real-time enrichment of patient-speciﬁc genomic breakpoints. BATMAN infers genomic breakpoint loci during sequencing and dynam- ically adapts the target panel in real-time.



 We present the performance of BATMAN on CNS tumours harbouring a variety of CNVs. We demonstrate the potential to generate long-reads spanning deletion of CDKN2A/B in high-grade astrocytoma, oligodendroglioma, and glioblastoma. Furthermore, we demonstrate the ability of BATMAN to resolve complex rearrangements such as chromothripsis events and accurately infer genome-wide copy number variation load, an emerging prognostic biomarker.



 BATMAN enables the sequencing of long reads spanning breakpoints at high coverages, enabling the robust detection of clinically relevant SVs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a421839bacfdf0fe835b8745c6eaa1153ef57" target='_blank'>
              BATMAN: BREAKPOINT ADAPTIVE TARGETING ALONGSIDE METHYLATION ANALYSIS ON NANOPORE
              </a>
            </td>
          <td>
            Simon Deacon, T. Murray, J. Trotman, P. Tarpey, Stuart Smith, S. Paine, M. Loose
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The immune system has been linked to periodontitis risk in oral inflammation and systemic consequences. Specifically, this study investigated whether hub genes were associated with immune cells via integrating single-cell RNA sequencing (scRNA-seq) and Mendelian randomization (MR). We analyzed scRNA-seq data to identify differentially expressed genes (DEGs) and immune cell subtypes in periodontitis. First, MR analysis was conducted using eQTL data to determine causal relationships between immune cell gene expression and periodontitis risk. Then, Gene ontology (GO) and pathway enrichment analyses were performed to understand functional implications. In addition, CellChat trajectory analysis explored intercellular communication. Finally, reverse transcription quantitative polymerase chain reaction (RT-qPCR) and immunohistochemistry (IHC) were used to validate hub genes. Comprehensive bioinformatics analysis delineated 23 immunologically distinct cell populations and identified 7 pivotal regulatory genes (ANXA1, ARL4C, CD79B, LRRC25, NKG7, SLC11A1, and VIM) demonstrating significant causal associations with periodontitis pathogenesis. Functional pathway analysis demonstrated these molecular markers participate in critical immunomodulatory pathways. RT-qPCR validation confirmed concordance between mRNA expression patterns of these DEGs and MR findings. Immunohistochemical analysis substantiated the causal relationships predicted by MR analysis, showing positive correlations for ANXA1 and SLC11A1 expression, while CD79B exhibited an inverse association. Advanced computational modeling of immune cell receptor-ligand interactions and cellular communication networks uncovered distinct functional roles during periodontitis development, providing novel insights into disease progression mechanisms. This study highlights the crucial role of immune cells and hub genes in periodontitis. These findings provide valuable insights into immune-related mechanisms and potential interventions for periodontitis. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-19438-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16044c01e3a056e09099a6f0a90c0aa778f58282" target='_blank'>
              Single-cell RNA sequencing and Mendelian randomization, revealing molecular mechanisms and causal correlation of immune-related genes in periodontitis
              </a>
            </td>
          <td>
            Xuedi Qiu, Fan Yang, Chenxi Li, Jian Wang, Yawen Yuan, Chao Guo
          </td>
          <td>2025-10-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69f1ebb5999d2aa0b3b1b9147382f2b93bbbf41" target='_blank'>
              Influence of Cultivation Conditions on Expression of Transcription Factor Genes in Glioblastoma Cells
              </a>
            </td>
          <td>
            A. I. Rezekina, D. V. Mazur, M. I. Shakhparonov, N. V. Antipova
          </td>
          <td>2025-09-28</td>
          <td>Russian Journal of Bioorganic Chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Schizophrenia is a highly polygenic and clinically heterogeneous disorder. In this paper, we first review layer-specific evidence across genetics, epigenetics, transcriptomics, proteomics, and patient-derived induced pluripotent stem cell (iPSC) models, then integrate cross-layer findings. Genetics research identifies widespread risk architecture. Hundreds of loci from common, rare, and CNV analyses. Epigenetics reveals disease-associated DNA methylation and histone-mark changes. These occur at neuronally active enhancers and promoters, together with chromatin contacts that link non-coding risk to target genes. Transcriptomics show broad differential expression, isoform-level dysregulation, and disrupted co-expression modules. These alterations span synaptic signaling, mitochondrial bioenergetics, and immune programs. Proteomics demonstrates coordinated decreases in postsynaptic scaffold and mitochondrial respiratory-chain proteins in cortex, with complementary inflammatory signatures in serum/plasma. iPSC models recapitulate disease-relevant phenotypes: including fewer synaptic puncta and excitatory postsynaptic currents, electrophysiological immaturity, oxidative stress, and progenitor vulnerability. These same models show partial rescue under targeted perturbations. Integration across layers highlights convergent pathways repeatedly supported by ≥3 independent data types: synaptic signaling, immune/complement regulation, mitochondrial/energetic function, neurodevelopmental programs and cell-adhesion complexes. Within these axes, several cross-layer convergence genes/proteins (e.g., DLG4/PSD-95, C4A, RELN, NRXN1/NLGN1, OXPHOS subunits, POU3F2/BRN2, PTN) recur across cohorts and modalities. Framing results through cross-layer and shared-pathway convergence organizes heterogeneous evidence and prioritizes targets for mechanistic dissection, biomarker development, and translational follow-up.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01124c16012c6b3c2bf42146bf8eecc8ada716b9" target='_blank'>
              Molecular Signatures of Schizophrenia and Insights into Potential Biological Convergence
              </a>
            </td>
          <td>
            Malak Saada, Shani Stern
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Human Papillomavirus (HPV), particularly high-risk strains such as HPV16 and HPV18, is a leading cause of cervical cancer and a significant risk factor for several other epithelial malignancies. While the oncogenic mechanisms of viral proteins E6 and E7 are well characterized, the broader effects of HPV infection on host transcriptional regulation remain less clearly defined. This study explores the hypothesis that conserved genomic motifs within the HPV genome may act as molecular decoys, sequestering human transcription factors (TFs) and thereby disrupting normal gene regulation in host cells. Such interactions could contribute to oncogenesis by altering the transcriptional landscape and promoting malignant transformation.We conducted a computational analysis of the genomes of high-risk HPV types using MEME-ChIP for de novo motif discovery, followed by Tomtom for identifying matching human TFs. Protein–protein interactions among the predicted TFs were examined using STRING, and biological pathway enrichment was performed with Enrichr. The analysis identified conserved viral motifs with the potential to interact with host transcription factors (TFs), notably those from the FOX, HOX, and NFAT families, as well as various zinc finger proteins. Among these, SMARCA1, DUX4, and CDX1 were not previously associated with HPV-driven cell transformation. Pathway enrichment analysis revealed involvement in several key biological processes, including modulation of Wnt signaling pathways, transcriptional misregulation associated with cancer, and chromatin remodeling. These findings highlight the multifaceted strategies by which HPV may influence host cellular functions and contribute to pathogenesis. In this context, the study underscores the power of in silico approaches for elucidating viral–host interactions and reveals promising therapeutic targets in computationally predicted regulatory network changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019a01c1572bf807f787812d1b2ed2b11781bb4f" target='_blank'>
              HPV as a Molecular Hacker: Computational Exploration of HPV-Driven Changes in Host Regulatory Networks
              </a>
            </td>
          <td>
            M. Chetta, Alessandra Rosati, N. Bukvic
          </td>
          <td>2025-08-27</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are an insignificant, however enormous population of tumor cells that display capacities of self-renewal, differentiation, and tumor initiation, consequently being the core feature in cancer progression, recurrence, and drug resistance. The phosphoinositide 3-kinase/protein kinase B (PI3K/AKT) is one of the most critical signalling cascades regulating CSCs and controlling their stemness, survival, evasion of the immune system under stressful conditions, as well as metabolic reprogramming. This review provides an overview of the morphological features and functional aspects of the PI3K/AKT immune cascade and pathway, and how it essentially connects with both upstream and downstream effectors in CSC biology. The cross-communication of PI3K/AKT with other pathways, e.g., Wnt, Notch, and Hedgehog, is elaborated to emphasize the redundancy of the networks facilitating CSC maintenance and drug resistance. Additionally, we provide an in-depth scrutiny of the processes through which PI3K/AKT signalling leads to CSC resistance to chemotherapy, radiotherapy, and targeted therapy, as well as their plasticity, metastasis, and immune escape mechanisms. Current and future therapeutic approaches targeting the PI3K/AKT axis, such as small molecule inhibitors, combination therapy, and drug delivery nanotechnology, are also discussed. Finally, we present clinical issues and prospects for improving CSC-based therapy by using PI3K/AKT blockade to eliminate resistance and induce protracted, long-lasting cancer remission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f966dace17da846800f374af44f626bb75044a1c" target='_blank'>
              The Role of PI3K/AKT Signalling Pathway in Cancer Stem Cells: Emerging Therapeutic Targets and Resistance Mechanisms
              </a>
            </td>
          <td>
            Komal Arooj, Hassan Imam, Zarlish Attique, Zoha Naeem, Ali Ahmad, Hafiz Muhammad Faraz Azhar, Fariha Javaid, Zeenat Nawaz
          </td>
          <td>2025-09-30</td>
          <td>Futuristic Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common type of leukemia in Western populations and remains incurable despite significant therapeutic advancements. Over the past decade, the treatment landscape has evolved from traditional chemoimmunotherapy to targeted oral agents, including Bruton’s tyrosine kinase inhibitors (BTKis) and BCL2 inhibitors (BCL2is), which have demonstrated superior efficacy and tolerability, especially in elderly patients. Venetoclax, a BCL2i, induces apoptosis in CLL cells through selective inhibition of the anti-apoptotic BCL2 protein, while BTKis, such as ibrutinib and its next-generation analogs, disrupt B-cell receptor signaling critical to CLL cell survival. However, resistance to both drug classes has emerged, including mutations in BTK and BCL2, prompting the exploration of novel therapeutic strategies. This review outlines the molecular basis and clinical implications of these resistance mechanisms, as well as emerging therapeutic solutions, including non-covalent BTKis like pirtobrutinib and BTK-targeting PROTAC degraders such as BGB-16673 and NX-2127. Additionally, we discuss promising combination therapies incorporating BTKis, BCL2is, and anti-CD20 monoclonal antibodies. Finally, we highlight the growing role of measurable residual disease (MRD) as a biomarker to guide treatment duration and evaluate therapeutic success. As resistance mechanisms continue to emerge, tailoring therapy based on underlying biology will be critical to sustaining disease control and enhancing outcomes in patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0956883756565c425ce66e36bee2fa5445ca9e94" target='_blank'>
              Resistance Mutations in CLL: Genetic Mechanisms Shaping the Future of Targeted Therapy
              </a>
            </td>
          <td>
            Samantha Sekeres, Erica N. Lamkin, Eduardo Bravo, Allison Cool, Justin Taylor
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078e803bd0f54c4fecbac4d86660275e0c2ba3b8" target='_blank'>
              Epigenetic alterations facilitate transcriptional and translational programs in hypoxia.
              </a>
            </td>
          <td>
            Kathleen Watt, Bianca Dauber, Krzysztof J. Szkop, Laura Lee, Predrag Jovanovic, Shan Chen, Ranveer Palia, Julia A. Vassalakis, Tyler T. Cooper, David Papadopoli, Laìa Masvidal, Michael Jewer, Kristofferson V. Tandoc, Hannah Plummer, Gilles A. Lajoie, I. Topisirovic, Ola Larsson, Lynne-Marie Postovit
          </td>
          <td>2025-10-16</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Environmental exposures during early life are increasingly recognized as key determinants of health and disease in adulthood (1) but how they durably shape disease risk across the lifecourse remains poorly understood. Here, we show early-life toxicants reprogram the epigenome and redirect age-associated transcriptional trajectories—polarizing cell-specific gene expression and predisposing to liver disease. The TaRGET II Consortium (2) exposed mice to diverse toxicants from pre-conception through weaning and followed individual animals though adulthood with multi-omic profiling. Analysis of >800 liver epigenomic and transcriptomic profiles from male and female mice revealed that despite differing chemical classes and mechanisms of action, multiple toxicants — BPA, TBT, TCDD, and PM2.5 — produced exposure signatures that converged on genes normally differentially expressed in the liver as animals aged. Histone modifications at enhancers emerged as key targets for epigenomic reprogramming of these liver aging-associated plasticity genes (LAAsP genes). Reprogrammed LAAsP genes exhibited a striking, bidirectional signature. In hepatocytes LAAsP genes that typically increase with age, such as those involved in metabolism, were repressed. Conversely, in non-parenchymal cells, LAAsP genes that normally decline with age, including those for extracellular matrix production, remained elevated. An attenuated LAAsP gene signature and polarized transcriptional states were mirrored in human liver disease and hepatocellular carcinoma and could effectively distinguished healthy from diseased human liver transcriptomes. Together, these findings demonstrate that early-life environmental exposures can hijack the plasticity of epigenomic aging, durably reprogram expression trajectories, and lock in polarized states that foreshadow chronic liver disease and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e50202ac8b33f8ab48ea3ab4ef30e279c45916f2" target='_blank'>
              Plasticity of Epigenomic and Transcriptomic Aging Reveals Common Targets for Reprogramming by Environmental Exposures
              </a>
            </td>
          <td>
            Cheryl Walker, Sandra L. Grimm, Rahul K. Jangid, Marisa S. Bartolomei, Dana C. Dolinoy, D. Aylor, Gokhan Mutlu, Shyam Biswal, Bo A Zhang, Robert Hamanaka, Justin A. Colacino, Maureen Sartor, Laurie Svoboda, Ting Wang, Cristian Coarfa
          </td>
          <td>2025-09-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Ageing is a general, intrinsic and progressively deleterious process that affects all cells, tissues and organs albeit at different extent and rate in each individual. The complexity and universality of its phenotypic manifestations suggest a multifactorial origin. The autosomal recessive disorder Werner syndrome likely represents a segmental progeroid disorder since patients show several, but not all phenotypes of premature ageing.


SUMMARY
Proliferative senescence of diploid cells in culture provided a model system in which ageing can be studied experimentally. Cultures of cells from patients with Werner syndrome experienced an extreme form of proliferative senescence and a clonal succession of translocations, known as variegated translocation mosaicism. In addition, Werner syndrome cells showed spontaneous deletion formation and a prolongation of and arrest in the S phase of the cell cycle. The WRN protein harbors a helicase, an exonuclease and a RecQ interaction domain. With the latter the WRN protein may interact with NBS1, RPA, MRE11, TREX1, MUTYH, POT1, TRF1, FEN-1, PAPRP-1, p97/VCP, TRF2, DNA polymerase(beta), Ku76/80, EXO-1, NEIL1, and p53, which are key to DNA damage response pathways including canonical NHEJ, homologous recombination, base excision repair and telomere maintenance. The WRN exonuclease domain is a target of WRNIP1 binding, which links WRN to resolution of stalled replication due to collision with transcription and the ATM-mediated cell cycle checkpoint. . Patients with an incomplete complement of Werner syndrome phenotypes, called atypical Werner syndrome patients, were found to carry variants in LMNA, POLD1, SPRTN, MDM2, CTC1, SAMHD1.


KEY MESSAGES
These findings broaden the genotypic landscape and the phenotypic spectrum of Werner syndrome. In this review potential molecular mechanisms underlying genomic instability in Werner syndrome, including chromothripsis due to asynchronous S phase traverse and telomere crises followed by bridge fusion breakage cycles are discussed. The participation of WRN in multiple gene networks is consistent with the multifactorial nature of ageing in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c57a230c575d7a7f034afedf23d1a5bec7bbac" target='_blank'>
              Molecular mechanisms of proliferative senescence and genomic instability in Werner syndrome and the WRN gene network.
              </a>
            </td>
          <td>
            Martin Poot
          </td>
          <td>2025-09-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell signaling plays a critical role in regulating cellular state, yet uncovering regulators of signaling pathways and understanding their molecular consequences remains challenging. Here, we present an iterative experimental and computational framework to identify and characterize regulators of signaling proteins, using the mTOR marker phosphorylated RPS6 (pRPS6) as a case study. We present a customized workflow that uses the 10x Flex assay to jointly profile intracellular protein levels, transcriptomes, and CRISPR perturbations in single cells. We use this to generate a “glossary” dataset of paired protein–RNA measurements across targeted perturbations, which we leverage to train a predictive model of pRPS6 levels based solely on transcriptomic data. Applying this model to a genome-wide Perturb-seq dataset enables in silico screening for pRPS6 and nominates novel regulators of mTOR signaling. Experimental validation confirms these predictions and reveals mechanistic diversity among hits, including changes in signaling output driven by anabolic activity, cellular proliferation and multiple stress pathways. Our work demonstrates how integrated experimental and computational approaches provide a scalable framework for multimodal phenotyping and discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a589887aa331a2fbdb712cfa97ab0770a36bd6c" target='_blank'>
              Iterative, multimodal, and scalable single-cell profiling for discovery and characterization of signaling regulators
              </a>
            </td>
          <td>
            John D Blair, Alexandra Bradu, Carol Dalgarno, Isabella N. Grabski, R. Satija
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Prime editing (PE) represents a significant advancement in genome editing, offering high precision for diverse genetic modifications without inducing double-strand breaks or requiring exogenous donor DNA templates. This "search-and-replace" technology employs a Cas9 nickase-reverse transcriptase fusion protein, guided by a PE guide RNA (pegRNA), to directly install specified edits including all 12 base-to-base conversions and targeted insertions/deletions with high fidelity. Since its introduction, PE systems have undergone rapid evolution (e.g., PE2-PE6, PEmax), markedly improving editing efficiency, product purity, and targeting scope. Although PE efficacy is context dependent, influenced by pegRNA design, cellular milieu, and DNA repair pathway engagement, ongoing research focuses on comprehensive system optimization. These efforts include engineering the Cas9 nickase and reverse transcriptase components for enhanced performance and processivity, alongside developing improved pegRNA architectures and chemical modifications to increase their stability and editing efficiency. Furthermore, strategies to modulate the cellular environment, such as transiently altering DNA repair pathway activities, particularly mismatch repair, are being explored to boost the accuracy and yield of precise edits. PE holds substantial promise for basic research, including precise disease modeling, and has demonstrated successful correction of pathogenic mutations in preclinical models of various genetic disorders like sickle cell disease, cystic fibrosis, and inherited retinal diseases. A significant milestone was the US Food and Drug Administration's granting of Investigational New Drug (IND) clearance for the first clinical trial of PM359, a therapeutic based on PE. This agent employs an ex vivo strategy, correcting the NCF1 gene in patient-derived hematopoietic stem cells for the treatment of chronic granulomatous disease. Despite considerable progress, unlocking the complete therapeutic promise of PE requires overcoming significant hurdles, particularly in developing effective in vivo delivery systems for its sizable components, with ongoing research actively investigating diverse viral and nonviral approaches. The translation of this versatile platform into transformative precision gene therapies is critically dependent upon its continued responsible advancement under robust ethical and regulatory oversight.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d7bd8913eb8c664247b6dfb6b80a653fc1fbc3" target='_blank'>
              Prime Editing: The Next Frontier in Precision Gene Therapy.
              </a>
            </td>
          <td>
            Melike Aliciaslan, Ezgi Erbasan, Fulya Erendor, S. Sanlioglu
          </td>
          <td>2025-10-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Thyroid cancer, a prevalent endocrine malignancy with rising global incidence, encompasses four primary subtypes: Papillary (PTC), follicular (FTC), medullary (MTC) and anaplastic thyroid carcinoma (ATC). PTC, accounting for 85–90% of cases, is primarily driven by BRAF V600E mutations alongside dysregulated non-coding RNAs, such as long non-coding RNA metastasis-associated lung adenocarcinoma transcript 1 and microRNA (miR)-1270. These alterations collectively activate MAPK signaling, promoting tumorigenesis. Furthermore, PTC exhibits metabolic reprogramming characterized by dysregulated glucose and lipid metabolism, where tumor suppressors, including family with sequence similarity 111 member B and fat mass and obesity-associated genes, constrain glycolytic flux. FTC, characterized by Ras mutations, exhibits enhanced lipid metabolism and PI3K/AKT pathway activation. Methyltransferase-like protein 16 and sclerostin domain-containing protein 1 have been highlighted as regulators of FTC progression. MTC, associated with rearranged during transfection (RET) proto-oncogene mutations, demonstrates programmed cell death protein-1/programmed death ligand-1 pathway involvement, which offers potential immunotherapy targets. ATC, the most aggressive subtype, is characterized by recurrent genetic alterations such as telomerase reverse transcriptase promoter and tumor protein p53 mutations, cAMP-responsive element-binding protein 3-like 1-driven activation of cancer-associated fibroblasts and hematological and neurological expressed 1-stathmin 1 signaling-mediated invasiveness. Recent diagnostic innovations encompass serum biomarkers, such as stanniocalcin-1, microRNA signatures (including miR-26b-5p) for PTC and MTC detection, radiomics-based differentiation of ATC from other subtypes and optical imaging techniques for precision diagnosis. Molecularly targeted therapies constitute the cornerstone of current strategies, with vemurafenib inhibiting BRAF/MEK in PTC, sorafenib acting as a multikinase suppressor in FTC, vandetanib blocking RET in MTC and berberine-doxorubicin combinations overcoming chemoresistance in ATC. Metabolic interventions, including metformin for glucose modulation in PTC and novel delivery systems such as micelle-encapsulated AB3 for MTC, demonstrate translational potential. The present review summarizes molecular mechanisms, diagnostic tools and emerging therapies while emphasizing the necessity of subtype-specific approaches to improve clinical outcomes in thyroid oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/802e70c9b1f839edc5befa96c6ac30494456d7fe" target='_blank'>
              Thyroid cancer: From molecular insights to therapy (Review)
              </a>
            </td>
          <td>
            Zhuozheng Li, Nuofan Wang, Xiao Li, Yongfang Xie, Zemin Dou, Hongbing Xin, Yuzhuo Lin, Yan Si, Tingting Feng, Guohui Wang
          </td>
          <td>2025-09-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Hematopoietic stem cell (HSC) therapy remains essential in treating blood disorders, autoimmune diseases, neurodegenerative conditions, and cancers. Despite its potential, challenges arise from the inherent heterogeneity of HSCs and the complexity of their regulatory niche. Recent advancements in single-cell RNA sequencing and chromatin accessibility sequencing have provided deeper insights into HSC markers and chromatin dynamics, highlighting the intricate balance between intrinsic and extrinsic regulatory mechanisms. Zebrafish models have emerged as valuable tools in HSC research, particularly through live imaging and cellular barcoding techniques. These models have allowed us to describe critical interactions between HSCs and embryonic macrophages, involving reactive oxygen species and calreticulin signaling. These are essential for ensuring HSC quality and proper differentiation, with implications for improving HSC transplant outcomes. Furthermore, the review examines clonal hematopoiesis, with a focus on mutations in epigenetic regulators such as DNMT3A, TET2, and ASXL1, which elevate the risk of myelodysplastic syndromes and acute myeloid leukemia. Emerging technologies, including in vivo cellular barcoding and CRISPR-Cas9 gene editing, are being investigated to enhance clonal diversity and target specific mutations, offering potential strategies to mitigate these risks. Additionally, macrophages play a pivotal role in maintaining HSC clonality and ensuring niche localization. Interactions mediated by factors such as VCAM-1 and CXCL12/CXCR4 signaling are crucial for HSC homing and the stress response, opening new therapeutic avenues for enhancing HSC transplantation success and addressing clonal hematopoiesis. This review synthesizes findings from zebrafish models, cutting-edge sequencing technologies, and novel therapeutic strategies, offering a comprehensive framework for advancing HSC biology and improving clinical outcomes in stem cell therapy and the treatment of hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6985cdf565a94b5e4ce65dadd2a9ba4efa0f62f7" target='_blank'>
              Advancements in Hematopoietic Stem Cell Therapy: From Biological Pathways to Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Viviana Cortiana, H. Chorya, R. H. Abbas, Jade Gambill, Adhith Theyver, Chandler Park, Yan Leyfman
          </td>
          <td>2025-10-15</td>
          <td>Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cab8ffb00b0f66c1500dfb2ff83a6f9807f5f0" target='_blank'>
              A Mammalian Genomic Signature Shaped by Single Nucleotide Variants Controlling Transcriptome Integrity and Diversity.
              </a>
            </td>
          <td>
            Jian Yang, Samuel Ogunsola, Jason Wong, Aydan Wang, R. Joehanes, Daniel Levy, Shalini Sharma, Chunyu Liu, Jiuyong Xie
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The persistent residual tumor cells that survive after chemotherapy are a major cause of treatment failure, but their survival mechanisms remain largely elusive. These cancer cells are typically characterized by a quiescent state with suppressed activity of MYC and MTOR. We observed that the MYC-suppressed persistent triple-negative breast cancer (TNBC) cells are metabolically flexible and can upregulate mitochondrial oxidative phosphorylation (OXPHOS) genes and respiratory function (“OXPHOS-high” cell state) in response to DNA-damaging anthracyclines such as doxorubicin, but not to taxanes. The elevated biomass and respiratory function of mitochondria in OXPHOS-high persistent cancer cells were associated with mitochondrial elongation and remodeling, suggestive of increased mitochondrial fusion. A genome-wide CRISPR editing screen in doxorubicin-persistent OXPHOS-high TNBC cells revealed the BCL-XL gene as the top survival dependency in these quiescent tumor cells, but not in their untreated proliferating counterparts. Quiescent OXPHOS-high TNBC cells were highly sensitive to BCL-XL inhibitors, but not to inhibitors of BCL2 and MCL1. Interestingly, inhibition of BCL-XL in doxorubicin-persistent OXPHOS-high TNBC cells rapidly abrogated mitochondrial elongation and respiratory function, followed by caspase 3/7 activation and cell death. The platelet-sparing proteolysis-targeted chimera (PROTAC) BCL-XL degrader DT2216 enhanced the efficacy of doxorubicin against TNBC xenografts in vivo without induction of thrombocytopenia that is often observed with the first-generation BCL-XL inhibitors, supporting the development of this combinatorial treatment strategy for eliminating dormant tumor cells that persist after treatment with anthracycline-based chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb7ad9e0cb28b546f19e574e42e36713a6ee110" target='_blank'>
              Quiescent OXPHOS-High Triple-Negative Breast Cancer Cells That Persist After Chemotherapy Depend on BCL-XL for Survival
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Marie Winter, Leandro Encarnacao Garcia, O. Akinrinmade, Francisco Madeira Marques, Emmanouil Zacharioudakis, Anna Skwarska, J. Aguirre-Ghiso, Marina Konopleva, Guangrong Zheng, Susan A. Fineberg, Daohong Zhou, E. Gavathiotis, Tao Wang, Eugen Dhimolea
          </td>
          <td>2025-08-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey I. Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="The SS18::SSX oncogene is the driver of synovial sarcoma, an aggressive cancer presenting in young adults that has poor long-term outcomes. Over the past five years, significant progress has been made in understanding the molecular, genomic, and epigenetic mechanisms underlying synovial sarcoma. This review synthesizes recent advancements in synovial sarcoma, including diagnostic pathology, genomic profiling, SS18::SSX biology, epigenetic dysregulation, proteomics, targetable pathways and immunotherapy. Key findings include the identification of rare but instructive alternative gene fusions, the roles of PRC1 and of liquid-liquid phase separation in SS18::SSX-mediated oncogenesis, and the development of epigenetic and engineered T-cell therapies. These advances offer new hope for improved treatments and outcomes in synovial sarcoma patients, though challenges remain in overcoming resistance and ensuring equitable access to emerging therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9194070a3830c48d1eb33aa8623f50b92cbe3b84" target='_blank'>
              Molecular and epigenetic oncogenesis in synovial sarcoma: implications for cancer biology, diagnosis and treatment
              </a>
            </td>
          <td>
            Amy Xueqi Wang, Kevin B. Jones, Torsten O Nielsen
          </td>
          <td>2025-08-23</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-14</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ~2.5×10−9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="A rare disease is generally defined as a medical condition that affects a small proportion of the population, though specific thresholds vary across countries. Despite regional differences, these definitions consistently reflect the low prevalence of such conditions, the limited availability of effective treatments, and the pressing need for targeted research and regulatory support. As a result of their rarity and low commercial potential, rare diseases have historically represented an area of market failure, where investment and research have been limited and often neglected. However, since the 1990s, each country has guaranteed continuous support to research and development projects to promote the advancements of rare disease treatments, achieving a growth rate greater than that of the entire pharmaceutical industry. In this review, we examine the status of orphan drug development using an advanced therapy medicinal product (ATMP) approach in the growing rare disease market, with a particular focus on cell therapies and gene therapies, which constitute the most actively developed and clinically applied categories within ATMPs. We also explore strategic approaches through which the orphan drug industry can utilize ATMPs, especially these two modalities, to enhance its competitiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb2ccc6113bccae4bf1abe5e01112d77e7abd03" target='_blank'>
              Advances in Cell and Gene Therapy for Rare Disease Treatment.
              </a>
            </td>
          <td>
            Hyun Jin Baek, Yoojun Nam, Yeri Alice Rim, Ji Hyeon Ju
          </td>
          <td>2025-09-01</td>
          <td>International journal of stem cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Accounting for the highest cancer-related death burden worldwide, NSCLC, presents a significant public health challenge due to its high incidence and persistently low survival rates. Despite progress in low-dose spiral CT screening and smoking cessation campaigns, the five-year survival rate for patients diagnosed at advanced stages remains below 20%, underscoring the urgent need for optimized treatment strategies. This article synthesizes recent research advances concerning pathogenic mechanisms, diagnostic classification, and therapeutic innovations. It begins by dissecting the interaction networks between driver gene mutations and the tumor immune microenvironment, elucidating the critical role of epigenetic dysregulation in tumor progression. It subsequently reviews how the integration of liquid biopsy with artificial intelligence (AI)-enhanced imaging techniques is refining molecular subtyping. The article then provides a systematic evaluation of breakthrough clinical developments, including the iterative evolution of targeted agents (such as fourth-generation EGFR inhibitors), the expansion of immunotherapy across all treatment lines (from neoadjuvant settings to advanced disease), and the emergence of antibody-drug conjugates (ADCs) targeting HER3 and TROP2. These transformative approaches have propelled the median survival of advanced patients beyond 30 months and increased the pathological complete response rate in early-stage patients by tenfold. Studies demonstrate that biomarker-directed precision therapy is profoundly reshaping clinical practice. Future efforts must prioritize unraveling complex resistance mechanisms to further extend therapeutic benefits to all patient populations through multi-omics-guided personalized combination strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67b8c6e9f6adaa36c36105fd4cd3a5e640e941ac" target='_blank'>
              Comprehensive Advances in Non-Small Cell Lung Cancer: From Molecular Mechanisms to Precision Therapy
              </a>
            </td>
          <td>
            Jiayi Sun
          </td>
          <td>2025-08-26</td>
          <td>MedScien</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo- and targeted therapy resistant state even after long term culture ex vivo. These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo- and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648ee47786050552f00a0c31f0d21f6a61f09642" target='_blank'>
              LEF1 confers resistance to DNA-damaging chemotherapies through upregulation of PARP1 and NUMA1 in ovarian cancer.
              </a>
            </td>
          <td>
            Lin Huang, Mengna Zhu, Mengqing Chen, F. Ying, Qiulei Wu, Pan Liu, Wenhan Li, Guoqing Li, Yuewen Gao, Shuyan Yi, Wenwen Wang, Yiping Wen, Si Sun, Jing Cai, Man Xiao
          </td>
          <td>2025-09-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8540fbcce548c20c4ac2831a0c9929750289f6c4" target='_blank'>
              Decoding ferroptosis for cancer therapy.
              </a>
            </td>
          <td>
            A. Wahida, Marcus Conrad
          </td>
          <td>2025-10-10</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Despite occurring in an essential molecule, mutations in RNA polymerase readily emerge and elicit complex pleiotropic effects across different levels of biological organization, which are all modulated by environment. We investigated the impact of temperature on the effects of six mutations on sequence, structure, transcriptome, and organismal traits. We found temperature altered the transcriptomic response and key organismal traits such as growth rate and biofilm formation in a genotype-specific manner. Critically, mechanistic insights into the possible drivers of mutational effects emerged only when examining the relationships between different levels of organization: location of mutations in the tertiary structure and distance to key interacting molecules partly explained the observed transcriptomic differences, which in turn drove the impact of mutations on organismal traits. While falling short of capturing the full complexity of the system, our findings underscore the benefits of integrating insights across multiple biological levels to understand the relationship between environment and mutational effects in molecules with extensive pleiotropic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3646be5bb3c5019f620c6e2499c30f9214f99d4e" target='_blank'>
              Deciphering the Impact of Temperature on Pleiotropic Consequences of RNA Polymerase Mutations
              </a>
            </td>
          <td>
            D. Ozbilek, Jake K. Soley, Danna R. Gifford, Christopher G Knight, Simon C Lovell, M. Lagator
          </td>
          <td>2025-09-16</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The mutagenic translesion synthesis (TLS) pathway, which is critically dependent on REV1's ability to recruit inserter TLS polymerases and the POLζ extender polymerase, enables cancer cells to bypass DNA lesions while introducing mutations that likely contribute to the development of chemotherapy resistance and secondary malignancies. Targeting this pathway represents a promising therapeutic strategy. Here, we demonstrate that the expression of the C-terminal domain (CTD) of human REV1, a ca. 100 amino acid scaffold essential for TLS polymerase interactions, disrupts REV1/POLζ-dependent TLS in mammalian cells. Inducible expression of REV1-CTD in multiple human and murine cancer cell lines sensitizes cells to DNA-damaging agents such as cisplatin, benzo[a]pyrene diol epoxide, and methyl methanesulfonate, without intrinsic cytotoxicity. REV1-CTD expression increases genomic instability, decreases mutagenesis, and enhances G2 arrest following genotoxic stress. Mutational disruption of the CTD's interaction interfaces abrogates these effects, confirming a dominant-negative mechanism via sequestration of TLS components. In a xenograft mouse model, REV1-CTD expression markedly enhances cisplatin efficacy, significantly reducing tumor burden. These findings establish the REV1-CTD as an effective dominant-negative inhibitor of TLS and support its development as a therapeutic agent delivered to cancer cells to enhance the efficacy of genotoxic chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/119ebde0313a7f12ca9995ad4532ab0aaa8b95c3" target='_blank'>
              Sensitization of cancer cells to DNA-damaging agents by expression of the REV1 C-terminal domain: Implications for chemotherapy.
              </a>
            </td>
          <td>
            Ke Bian, Charley Gruber, Emine Byers, Paul Leclerc, Michael T. Hemann, Graham C. Walker
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-21082-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetically identical cells in blood. Genetic lesions that confer a fitness advantage, such as leukemogenic point mutations or mosaic chromosomal alterations (mCAs), are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of known driver genetic lesions. To characterize CH without known driver genetic lesions, we use 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantify CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we develop a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We perform a genome-wide association study of GEM to discover the germline determinants of CH-LPMneg. We identify seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1).Functional analyses of SMC4 and NRIP1 implicated altered hematopoietic stem cell self-renewal and proliferation as the primary mediator of mutation burden in blood. We then perform comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we perform phenotypic association meta-analyses across four cohorts, finding that GEM is associated with increased white blood cell count, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a90296a3758d552c4d460a056b5b1554e1d3253" target='_blank'>
              Genetic determinants and genomic consequences of non-leukemogenic somatic point mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, K. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, Nathalie Chami, Eimear Kenny, Ruth J. F. Loos, B. Psaty, R. Tracy, J. Brody, Jeong H Yun, Michael H Cho, Ramachandran S. Vasan, Sharon L. R. Kardia, Jennifer A Smith, L. Raffield, A. Bidulescu, Emily C O'Brien, Mariza de Andrade, Jerome I. Rotter, S. Rich, Yii-DerIda Chen, C Charles Gu, C. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika A Mathias, Alexander P. Reiner, Vijay G. Sankaran, Charlie Lowenstein, T. Blackwell, G. Abecasis, Albert V. Smith, Hyun M Kang, P. Natarajan, S. Jaiswal, Alexander G. Bick, Wendy S. Post, P. Scheet, Paul L. Auer, Theodoros Karantanos, Alexis Battle, M. Arvanitis
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>187</td>
        </tr>

        <tr id="Cancer remains a leading global health challenge, with conventional diagnostic and treatment methods often lacking precision and adaptability. This review explores transformative advancements that are reshaping oncology by addressing these limitations. It begins with an overview of cancer’s complexity, emphasizing the shortcomings of conventional tools such as imaging and chemotherapy, which frequently fail to deliver targeted care. The discussion then shifts to biomarkers, which represent a groundbreaking frontier in early detection, enabling the identification of unique biological signatures that signal the presence of cancer with heightened sensitivity. Building on this foundation, the review examines personalized molecular therapies, which target the specific genetic and molecular vulnerabilities of tumors. These therapies not only enhance treatment efficacy but also minimize adverse effects, offering patients improved outcomes and quality of life. By integrating biomarker-driven diagnostics with tailored therapeutic strategies, a new paradigm of precision oncology emerges, bridging the gap between early detection and effective intervention. Real-world case studies highlight both successes, such as significantly improved survival rates, and persistent challenges, including accessibility and cost barriers. Looking ahead, the review outlines pathways by which to scale these innovations, emphasizing the critical need for robust infrastructure, sustained research investment, and equitable healthcare policies. It concludes by envisioning a future where biomarkers and personalized therapies converge to redefine cancer care, offering earlier detection, precise interventions, and better patient experiences. This work underscores the urgency of adopting cutting-edge approaches to overcome cancer’s persistent threats, paving the way for a more effective and humane era in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab27e64ac341fefee806bee332bab63039656c" target='_blank'>
              The Future of Cancer Diagnosis and Treatment: Unlocking the Power of Biomarkers and Personalized Molecular-Targeted Therapies
              </a>
            </td>
          <td>
            Getnet Molla, M. Bitew
          </td>
          <td>2025-08-28</td>
          <td>Journal of Molecular Pathology</td>
          <td>3</td>
          <td>14</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the second leading cause of cancer death worldwide, largely due to the limited efficacy of current therapies in advanced stages of the disease. Most cases of HCC develop in the setting of chronic liver disease, particularly cirrhosis, where ongoing cycles of inflammation, hepatocyte death and regeneration foster the gradual accumulation of genetic and epigenetic alterations that promote malignant transformation. These molecular changes contribute to the high degree of tumour heterogeneity observed in HCC, a major factor underlying resistance to current treatments. As a result, sustained clinical responses to existing therapies, such as tyrosine kinase inhibitors, anti-angiogenic agents and immune checkpoint inhibitors, remain uncommon. In this context, a growing body of evidence has identified epigenetic dysregulation as a key driver of tumour progression and therapeutic resistance, highlighting a new frontier for intervention. This review provides clinicians and researchers with a comprehensive overview of the emerging field of epigenetic therapies in HCC, summarising results from both completed and ongoing clinical trials involving the so-called 'epidrugs'. Importantly, we discuss how targeting epigenetic mechanisms may not only suppress tumour growth but also enhance the effectiveness of current therapies by reversing resistance pathways. By translating complex molecular insights into tangible therapeutic strategies, epigenetics is poised to reshape the future of HCC management, offering renewed hope for more durable and personalised treatment responses in a disease where progress is urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca914bc3e6f4b196e21231ee5479dd3a9a6ded8" target='_blank'>
              Epigenetic therapies in hepatocellular carcinoma: emerging clinical tools and applications.
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite G Fernandez-Barrena, Esteban J. Fiore, Matías A. Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-09-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P. Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="The PLOD gene family, involved in extracellular matrix (ECM) remodeling, plays a role in tumor progression, but its comprehensive role and clinical significance in clear cell renal cell carcinoma (ccRCC) remains unclear.We integrated multi-omics bioinformatics analyses from public databases (TCGA, GEO) with experimental validation using RT-qPCR, western blotting, and functional assays to systematically evaluate the expression patterns, prognostic value, immune microenvironment associations and drug resistance of PLOD genes in ccRCC. Computational approaches, including the comparative toxicogenomics database and molecular docking, were further employed to identify potential chemical modulators.PLOD1, PLOD2, and PLOD3 were consistently overexpressed at both mRNA and protein levels in ccRCC tissues and cell lines. High PLOD expression was significantly correlated with reduced overall survival, and poor disease-free survival. Functional enrichment analysis revealed the involvement of PLOD gene family in collagen biosynthesis, ECM-receptor interaction, and lysine degradation pathways. PLOD expression was also linked to an immunosuppressive microenvironment and resistance to conventional therapeutics. Through toxicogenomics screening and molecular docking, acetaminophen was identified as a potential regulator of all three PLOD proteins.This study underscores the pivotal role of the PLOD family in ccRCC pathogenesis through ECM remodeling, immune modulation, and therapy resistance. Our results support their utility as diagnostic and prognostic biomarkers, and acetaminophen may serve as a candidate for targeting PLOD-mediated pathways, providing a foundation for future preclinical and therapeutic investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4232679d22860e65ac43c0ad98ce3da9db904a" target='_blank'>
              Prognostic biomarker and clinical significance of PLOD gene family in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xuan Shang, Liu Liu, Zhenwei Yang, Ming Yan, Ruimin Ren, Kexin Guo, Jie Wang, Wei Zhang, Jiasong Chang, Jialei Li, Ji-Min Cao, Guang Li, Li Gao
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear.We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells.High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells.Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) is the most common hematological malignancy. More than half of DLBCL patients achieve long‐term remission after treatment, but a third relapse after conventional Rituximab (R)‐based chemotherapy regimens, such as CHOP (cyclophosphamide, doxorubicin, vincristine and prednisone). Cancer cells are exposed to chronic replication stress, which impedes the duplication of their genome. Functional DNA repair pathways are therefore important for the survival of cancer cells. This dependence can be exploited therapeutically to hamper repair of the intrinsic DNA damage occurring during replication or to exacerbate DNA damage induced by chemotherapy. Using CRISPR‐Cas9 screening, we identified CHEK1, WEE1, ATR and RAD51 DNA repair factors as essential genes in DLBCL cells. According to these results, we investigated the effect of small molecules targeting DNA replication stress and DNA repair mechanisms, alone or in combination with the R‐CHOP genotoxic agents, cyclophosphamide and doxorubicin. Applying a standard threshold of 2 SDs below the IC50 of the genotoxic agent alone, a total of 3 synthetic lethal combinations have been identified including cyclophosphamide with CHK1/2 inhibitor, cyclophosphamide and ATR inhibitor and doxorubicin with DNAPK inhibitor. Co‐treatment with these molecules led to cell death, DNA damage induction and cell cycle arrest in DLBCL cells more efficiently than genotoxic agents alone. These data have been validated using primary DLBCL cells from patients. Our results open new perspectives for therapeutic approaches exploiting the synthetic lethality of genotoxic agents with DNA repair inhibitors to improve the therapeutic outcome of patients with DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ea8280a7e78b4aa4b06c11d4f0a4eaa409f0ab" target='_blank'>
              Synthetic Lethal Combinations of DNA Repair Inhibitors and Genotoxic Agents to Target High‐Risk Diffuse Large B Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Ovejero, Julie Devin, Laura Alibert, Camille Soun, Yea-Lih Lin, Laure Dutrieux, M. Abouladze, Elvira Garcia de Paco, Ouissem Karmous Gadacha, A. Constantinou, Guillaume Cartron, C. Herbaux, O. Elemento, Philippe Pasero, Sandrine Roulland, J. Moreaux, C. Bret
          </td>
          <td>2025-08-23</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are pervasively present in human cancers and have a fundamental role in treatment failure and disease recurrence. Identifying critical elements that sustain the CSC phenotype may lead to novel strategies for cancer treatment. Here, we provide evidence of an essential link between the σ1 receptor (σ1R), a ligand-regulated chaperone protein residing preferentially at the endoplasmic reticulum-mitochondria contact sites, and CSCs in castration-resistant prostate cancers (CRPCs). Integrating functional assays in multiple preclinical models with transcriptomic and proteomic data, we found that σ1R controls CSC self-renewal capacity and tumorigenic proficiency by coordinating mitochondrial dynamics and mitochondrial-nuclear signaling. Inhibiting σ1R with synthetic antagonists and RNA interference led to the progressive exhaustion and loss of tumorigenicity of the CSC progeny. Mechanistically, interfering with σ1R function disrupted mitochondria homeostasis and triggered β-catenin degradation. Examining clinical CRPC samples, we found a tight correlation between σ1R and mitochondrial gene expression. Furthermore, σ1R and β-catenin protein levels were highly correlated in prostate tumors with significant upregulation in metastatic CRPCs, sustaining a role of the σ1R-mitochondria-β-catenin axis in disease progression. This σ1R-centered axis is essential for preserving the self-renewal and tumorigenic capability of CSCs and represents a critical vulnerability exploitable for discovering novel CSC-directed therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cancer risk increases with age, and cellular senescence may be a major contributor to cellular carcinogenesis. Enormous efforts have been made to investigate the interrelation between aging and tumors, but little is known about the comparative features of normal aging, cellular senescence, and cancer at single-cell resolution. By integrating analyses of genomics, epigenomics, and bulk and single-cell transcriptomics, we revealed a directionally opposite transcriptional profile between cellular senescence and tumorigenesis at the single-cell level, which may be affected by epigenomic regulations. A total of 648 aging-dependent senescence-associated coregulated modules (SACMs), disproportionately affecting the reproductive systems of both females and males, were initially defined across 17 tissues. Single-cell analysis revealed that aging primarily affects endothelial cells, followed by T cells, epithelial cells, macrophages, and fibroblasts. Opposite directions of change in gene expression between aging and cancer can commonly be observed in endothelial, fibroblast, and epithelial cells, which may prompt the opposing patterns of gene expression between tissue aging and epithelial carcinoma at the bulk level. A similar pattern of expression can be observed in immune cells, which are characterized by decreased self-renewal with aging, but this pattern is reversed in epithelial carcinoma. Our study highlighted the role of senescence as a natural barrier against tumor formation and supported the idea that aging-related systemic environment changes create a protumorigenic milieu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4073352af61f280e6d1b0c8dc5f8cd7b466fe5" target='_blank'>
              Integrative Transcriptomic Analysis Decodes the Interplay Between Aging, Senescence, and Cancer.
              </a>
            </td>
          <td>
            Jiale Cai, Deng Wu, Dahua Xu, Fujin Liu, Peihu Li, Dehua Zheng, Zhizhou Xu, Meng Cao, Yutong Shen, Nihui Zhang, Guanghui Tian, Chengyi Fan, Bo Wang, Kongning Li, Xiaoman Bi
          </td>
          <td>2025-09-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly aggressive and heterogeneous malignancy, in which natural killer (NK) cells play a crucial role in tumor progression and immune surveillance. This study aimed to characterize the transcriptomic landscape of NK cell–associated genes (NAGs) and explore their associations with clinical outcomes and therapeutic responses in HCC. Using transcriptomic data from The Cancer Genome Atlas (TCGA), we identified key NAGs through comprehensive statistical analyses. Patients were stratified into distinct risk groups based on NAG expression profiles. Low-risk patients demonstrated better survival, higher immune infiltration, and greater predicted sensitivity to immunotherapy, whereas high-risk patients were associated with reduced chemotherapy responsiveness. These findings contribute to a deeper understanding of the immunogenomic features of HCC and provide a basis for developing personalized therapeutic approaches centered on NK cell–related mechanisms. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03692-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f099d43ef2898930decd4284d14d3171c835aebb" target='_blank'>
              Transcriptomic landscape of NK cell–related genes in hepatocellular carcinoma: associations with prognosis and therapeutic response
              </a>
            </td>
          <td>
            Yuxiang Wang, Wenqiang Liao, Jianbo Lin, Jun Li, Jianping Gong
          </td>
          <td>2025-10-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BCL-2 is a central regulator of apoptosis and inhibits cell death by sequestering pro-apoptotic BH3 alpha-helices within a hydrophobic surface groove. While venetoclax, a BH3-mimetic drug, has transformed the treatment of BCL-2–driven malignancies, its efficacy is increasingly limited by acquired resistance mutations that disrupt small-molecule binding yet preserve anti-apoptotic function—reflecting a remarkable structural adaptation. Here, we employ hydrocarbon-stapled alpha-helices derived from the BAD BH3 motif as conformation-sensitive molecular probes to investigate this therapeutic challenge. The stapled peptides not only retain high-affinity binding to all BCL-2 variants but also show enhanced potency to select venetoclax-resistant mutants. Structural analyses, including X-ray crystallography and hydrogen-deuterium exchange mass spectrometry (HDX MS), demonstrate that these stapled helices restore native BH3 engagement by reversing the conformational consequences of resistance mutations. Notably, we identify a serendipitous interaction between the α3–α4 region of BCL-2 and hydrocarbon staple, which further compensates for altered groove conformation and contributes to mutant binding affinity. Together, these findings offer mechanistic insights into BCL-2 drug resistance and reveal a blueprint for designing next-generation inhibitors that overcome this clinically significant barrier to durable treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c989e0417fad14adefa4b6808df4b60e2e5666a7" target='_blank'>
              Structural insights into chemoresistance mutants of BCL-2 and their targeting by stapled BAD BH3 helices
              </a>
            </td>
          <td>
            Thomas M. DeAngelo, Utsarga Adhikary, Kyle J. Korshavn, H. Seo, Clara R. Brotzen-Smith, Christina M Camara, S. Dhe-Paganon, Gregory H. Bird, T. Wales, L. Walensky
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Drug tolerant persisters (DTPs) drive cancer therapy resistance by temporarily evading drug action, allowing multiple routes to eventual permanent resistance. Despite clear evidence for DTPs, the timing of their emergence, proliferative nature, and how their population dynamics arise from measured single-cell kinetics remain poorly understood. Here we use time-lapse microscopy data from two cancer cell lines, integrating single-cell and population measurements, to develop a quantitative description of drug persistence. Contrary to the expectation that increasing levels of genotoxic stress should lead to slower times to division and faster times to death, we observe minor changes in the single-cell intermitotic and death time distributions upon increasing cisplatin concentration. Yet, population decay rates increase 3-fold, suggesting a surprising independence of the overall dynamics from the measured birth and death rates. To explain this phenomenon, we argue that the observed lineage correlations and concentration-dependent decay rates imply cell-state dependent fate choices made both pre and post-cisplatin as opposed to just post-drug birth/death rate-based competitive fate choices. We demonstrate that these cell-states, present in the ancestors of DTP and sensitive cells, exhibit no difference in cycling speed and are inherited across 2-3 cellular generations. A stochastic model implementing these rules simultaneously recapitulates the observed decay rates and cell-fate correlations, also explaining how pre-drug fate decisions are consistent with barcoding experiments where barcode diversity remains unchanged after drug administration. Our results provide a powerful perspective on drug tolerance based on general arguments, without requiring knowledge of the underlying molecular architecture of the heterogeneous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b499c056c673d79d24c4d2c5acdf399c0d8012aa" target='_blank'>
              Inheritable cell-states shape drug-persister correlations and population dynamics in cancer cells
              </a>
            </td>
          <td>
            Anton Iyer, Adrián E. Granada, Shaon Chakrabarti
          </td>
          <td>2024-11-03</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b348bc9a3e4abf3cb6086b89e0d2182bceec0bd" target='_blank'>
              NEK9-mediated Wnt signalling repressor TLE3 rewires Docetaxel resistance in cancer cells by inducing pyroptosis.
              </a>
            </td>
          <td>
            S. Ansari, Sibasish Mohanty, P. Mohapatra, Sudeshna Datta, Mamuni Swain, Rachna Rath, D. Muduly, Saroj K D Majumdar, R. Swain, Sunil K Raghav, Rupesh Dash
          </td>
          <td>2025-09-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acdfefa7d1e9916c129010f57a43699739821df" target='_blank'>
              Cancer gene therapy: historical perspectives, current applications, and future directions.
              </a>
            </td>
          <td>
            Jingyi Zeng, Jingwen Luo, Yingchun Zeng
          </td>
          <td>2025-09-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The study by Li et al. , “Single-cell transcriptome analysis reveals the association between histone lactylation and cisplatin resistance in bladder cancer”, investigated how histone lactylation contributes to cisplatin resistance in bladder cancer (BCa). Using high-resolution single-cell lineage tracing, the authors identified a distinct subpopulation of BCa cells with enhanced glycolytic metabolism that exhibited significant cisplatin resistance. Further analyses showed significant enrichment of histone H3 lysine 18 lactylation (H3K18la) at the promoter regions of the transcription factors YY1 and YBX1, which promoted their expression and facilitated the development of a drug-resistant phenotype. This study was the first to combine single-cell omics with lactylation modification mechanisms, providing new insights into strategies for overcoming cisplatin resistance in BCa. Nonetheless, potential limitations of the study should be carefully considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d10bc76434e826705d215ff116fbc298bccf9ac" target='_blank'>
              Histone H3K18 lactylation and cisplatin resistance in bladder cancer: insights from single-cell transcriptomic analysis
              </a>
            </td>
          <td>
            Zongchi Yang, Shaohua Chen
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer treatment is often complicated by the intricate dynamics of gene mutations, which
vary across different tumor sizes. Understanding these variations is essential for the development of targeted
therapies and the improvement of patient outcomes. In this study, we conducted a comprehensive analysis of gene
variant counts and mutation rates in breast cancer tumors, categorized by size into three groups: 0-49 mm, 50-
99 mm, and 100-149 mm. Genomic data were systematically extracted and analyzed to identify key genes with the
highest variant counts and to calculate mutation rates normalized per 1,000 observations. Data visualization
techniques were employed to elucidate trends in mutation frequencies across the different tumor sizes. Our
findings identified PARP1P1 as a gene with consistently high variant counts and mutation rates across all tumor
sizes, indicating its potential as a robust biomarker for cancer progression. Conversely, CCDC7, DLGAP1, and
SYN3 exhibited higher mutation rates predominantly in smaller tumors, suggesting their involvement in the early
stages of tumorigenesis. These results underscore the critical importance of gene-specific mutation patterns in
breast cancer, with PARP1P1 emerging as a gene of significant clinical relevance. Additionally, the sizedependent mutation patterns of CCDC7, DLGAP1, and SYN3 emphasize the need for early detection and the
development of targeted interventions. Future research should aim to validate these findings in larger patient
cohorts and investigate the functional implications of these mutations to further advance precision medicine in
breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07cd2288a49ab4f4df24f64b69bb548c18f6cb3" target='_blank'>
              Single Nucleotide Polymorphisms Reveal Tumor SizeDependent Mutation Patterns in Breast Cancer
              </a>
            </td>
          <td>
            Hongyu Gao Hongyu Gao
          </td>
          <td>2025-09-01</td>
          <td>Journal of Medical and Dental Science Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aim: ERG overexpression driven by gene fusions is well-characterized event in prostate cancer, yet its impact on gene expression profiles and regulatory networks remains unclear. This study aims to identify most frequently mutated 20 genes in prostate adenocarcinoma, compare their expression levels in ERG+ and ERG- tumors, and construct gene correlation networks to uncover ERG-dependent molecular interactions.
Materials and Methods: RNA-seq and somatic mutation data from the TCGA-PRAD cohort were used. Most frequently mutated 20 genes were identified. Patients were grouped by ERG expression status. Differential expression analysis and gene co-expression network construction were performed for each subgroup.
Results: TP53, SPOP, FOXA1, KMT2D, KDM6A, and MUC17 showed higher expression in ERG+ tumors, while TTN, LRP1B, PTEN, and RYR2 were more expressed in ERG- tumors. Network analysis revealed distinct regulatory patterns between subgroups.
Conclusion: ERG status significantly shapes transcriptional and regulatory landscape of prostate cancer, offering potential targets for ERG-specific therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715787fdd68a02367b7887119163c38e134df38d" target='_blank'>
              Integrative analysis of somatic mutations, gene expression, and co-expression networks in ERG-stratified prostate adenocarcinoma
              </a>
            </td>
          <td>
            Perçin Pazarcı
          </td>
          <td>2025-08-30</td>
          <td>Adıyaman Üniversitesi Sağlık Bilimleri Dergisi</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mouse embryonic stem cells (ESCs) occasionally transit into a rare two-cell-like (2C) state characterized by transient activation of endogenous retroviruses such as MERVL and expression of 2C-specific genes including the Zscan4 cluster. These 2C-like cells (2CLCs) resemble early blastomeres and display expanded developmental potential, but their unstable and sporadic nature has hindered mechanistic studies. Here, we establish stable 2CLCs (s2CLCs) that maintain persistent MERVL expression and homogeneous 2C gene activation. Live-cell imaging revealed uniform and sustained MERVL activity in s2CLCs, contrasting with the heterogeneous and transient expression observed in conventional ESCs. Transcriptome profiling demonstrated robust induction of 2C-specific regulatory networks, and embryoid body differentiation combined with machine learning uncovered increased lineage variability and expanded developmental trajectories. Epigenomic profiling further revealed unique chromatin states, distinctive super enhancer landscapes, and active enhancer marking at MERVL loci. Together, these findings demonstrate that stable maintenance of the 2C-like state is achievable in vitro, providing a powerful model to dissect ERV-driven transcriptional regulation, epigenomic remodeling, and totipotent-like developmental potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0d205ec741e491b63d47b95a2bdcf988b0e740" target='_blank'>
              Stable Maintenance of Two-Cell-Like Cells from Embryonic Stem Cells Reveals Chromatin and Super Enhancer Regulation of MERVL Elements
              </a>
            </td>
          <td>
            Rui Geng, Benjamin L. Kidder
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Lysosome-dependent cell death (LDCD) is an emerging form of regulated cell death with critical implications in tumor development, immune modulation, and therapy responsiveness. However, the role of LDCD-related genes in colon adenocarcinoma (COAD) remains poorly understood. We comprehensively analyzed LDCD-related gene expression profiles using transcriptomic data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus databases (GEO). Unsupervised clustering was performed to identify molecular subtypes. A prognostic signature was developed using LASSO and Cox regression analyses. Immune infiltration characteristics and immunotherapy responses were assessed via multiple algorithms. Single-cell RNA sequencing (scRNA-seq) analysis was conducted to explore the cellular distribution of LDCD genes. Functional assays, including colony formation, Transwell migration, and western blotting, were performed to validate the role of key LDCD regulators in COAD cell line. LDCD-related genes were differentially expressed in COAD tissues and stratified patients into two distinct subtypes with significant differences in survival, immune infiltration, and biological pathways. A five-gene prognostic signature demonstrated robust predictive ability for overall survival and was associated with tumor immune microenvironment characteristics. High-risk patients showed distinct immune checkpoint profiles and predicted sensitivity to immune checkpoint blockade therapy. scRNA-seq analysis revealed cell type-specific expression patterns of LDCD genes. In vitro experiments confirmed that modulation of a key LDCD regulator, SLC11A1, affected the proliferation and invasiveness of COAD cells. Moreover, the impact of SLC11A1 on apoptosis and intracellular ROS level confirmed the potential association between SLC11A1 and COAD cell apoptosis. Our study reveals the prognostic value and immunological relevance of LDCD-related genes in COAD. These findings provide new insights into molecular classification and may aid in the development of personalized therapeutic strategies targeting LDCD pathways in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18267-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9901f6ba7f419afbbf7d8a4c7ff078253e88b294" target='_blank'>
              Comprehensive characterization of lysosome-dependent cell death reveals prognostic significance and immune landscape in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhenhua Liu, Lianming Cai, B. Liao, Baowen Wang, T. Xia, Xiangxin Wu
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07528eee7ff3d627e8787aab243a7521b7075a4" target='_blank'>
              Targeting p53-MDM2 pathway with novel triazole-oxazole hybrids: a fragment-based drug discovery approach for next-generation cancer therapies.
              </a>
            </td>
          <td>
            Apurva Prajapati, Hitesh Patel
          </td>
          <td>2025-09-23</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. Methods We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. Results EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. Conclusion This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03631-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Walid K Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            Atul Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic instability is a key feature of many neurological disorders, with transposon activation and nucleotide triplet repeat instability playing critical roles. Transposons, which are also referred to as mobile genetic elements, have the potential to destabilize the genome and interfere with gene expression. Conversely, changes in nucleotide triplet sequences, such as expansions or contractions, can lead to the production of abnormal proteins or nonfunctional RNAs. In this perspective, we discussed the intricate relationship between these two forms of genomic instability and their influence on brain disorders. We analyzed the molecular mechanisms that contribute to these phenomena, the shared regulatory systems that govern them, and their role in neurological conditions. Additionally, we provided some insights into the development of potential therapies for brain disorders linked to these genomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26a53ff31acf2b91b1ca79c1a339add14ba8dd2" target='_blank'>
              Transposon activity and nucleotide triplet instability: new perspectives on their potential interplay in brain disorders
              </a>
            </td>
          <td>
            Giuseppe Pepe, Marianna Storto, A. di Pardo, V. Maglione
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Neuroscience</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer remains a major global health challenge. This review focuses on DNA-PKcs, a protein that enables cancer cells to repair DNA damage caused by radiotherapy and chemotherapy, promoting tumor survival. We synthesize recent evidence on how DNA-PKcs drives tumor progression and therapy resistance, highlighting its potential as a therapeutic target. Current advances in DNA-PKcs inhibitors are examined, including their promise in combination therapies and hurdles in clinical development. Uniquely, this review connects DNA-PKcs to emerging roles in immunomodulation and cancer metabolism, evaluates next-generation inhibitors, and proposes solutions to overcome translational challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f0939416a1ca4dadfa5d4cc33f441bbed022050" target='_blank'>
              Unlocking the Therapeutic Potential of DNA-PKcs in Cancer: Comprehensive Insights into Mechanisms and Clinical Applications
              </a>
            </td>
          <td>
            Tong Zheng, Chao Sun, Cijun Yun, Hui Wang, Xiongxiong Liu
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background/Aim Typically expressed on T-cells and NK cells, FASLG induces apoptosis in target cells upon binding Fas. However, assessing potential FASLG expression in tumor cells with convenient genomics approaches has been challenging. Materials and Methods This study applied a novel assessment of FASLG copy numbers (CNs) and gene expression levels, applicable to bulk exome and RNAseq files. Results Analyses indicated high FASLG CN associated with worse survival outcomes. Interestingly, higher FASLG gene expression was found to be associated with better survival outcomes, which led to a determination of whether this result was due to FASLG expression from tumor-infiltrating lymphocytes (TILs) instead of cancer cells demonstrating the higher CNs. In fact, T-cell markers CD4 and CD8A highly correlated with FASLG expression, consistent with the hypothesis that the high FASLG expression was associated with the TILs. Subsequent analyses confirmed that CN increases led to increased gene expression in the genomic region of the FASLG gene, particularly with an assessment of the expression of the neighboring PRRC2C gene. In sum, FASLG CN assessments, even independently of a corresponding gene expression correlation, may provide important characterizations of tumor cells. Conclusion This study indicates that FASLG CN increases could represent a mechanism of tumor escape from TILs and a prognostic indicator; and tumor FASLG may be a suitable drug target for reducing tumor evasion of T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0153f78ee8bea9bd0c80db1c2166e2902441e45" target='_blank'>
              Programmatically Efficient Separation of Immune Infiltrate and Tumor Gene Expression Overlap Potentials in a Big Data Setting: FASLG Gene Expression-related Survival Distinctions
              </a>
            </td>
          <td>
            Elizabeth A. Fletcher, Toriana R. Dabkowski, Mallika Varkhedi, George Blanck
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The genetic code, long viewed as a frozen relic of early evolution, is now being systematically reprogrammed. This article examines the chemical, evolutionary, and technological foundations of its expansion, focusing on plasticity, codon architecture, and amino acid selection. The triplet codon structure reflects an evolutionary compromise between metabolic cost and information capacity, while proteins ‐ the essential catalytic polymers ‐ are built from α‐L‐amino acids derived from L‐alanine. Introducing non‐triplet codons or non‐α amino acids would demand new metabolic pathways to supply precursors and energy for synthetic life. Current methods for incorporating noncanonical amino acids ‐ stop codon suppression, sense codon reassignment, and quadruplet recoding ‐ are transient and inefficient, consistent with the “ambiguous intermediate” model. Achieving stable expansion requires redesigning tRNA identity sets, improving ribosomal fidelity, rewiring metabolism, and potentially creating new foldamer scaffolds. Orthogonal translation systems and codon box deconstruction have become key tools, while permanent codon reassignment and engineered orthogonal systems mark the likely future. Core technologies include genome‐scale codon swapping, directed evolution, cellular compartmentalization, and metabolic integration. Framed by evolutionary models such as Wong's coevolution theory, the WesthoffGrosjean model, and the Alanine World hypothesis, genetic code expansion emerges as a radical extension of life's chemical vocabulary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab678cc963c8f654a81a0f61b3b8b7111c6c171e" target='_blank'>
              Essential Logic and Facts Behind the Expansion of the Genetic Code: A Critical Assessment
              </a>
            </td>
          <td>
            Huy Tran Le Luu, H. Karbalaei-Heidari, N. Budisa
          </td>
          <td>2025-09-27</td>
          <td>ChemCatChem</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd9218b7e444fde0659e2b4793dd03d323ad206c" target='_blank'>
              Single-cell genetics identifies cell type-specific causal mechanisms in complex traits and diseases
              </a>
            </td>
          <td>
            A. Henry, A. Senabouth, R. Tyebally, B. Bowen, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, A. Cuomo, J. Fan, H. L. Huang, H. Tanudisastro, A. Xue, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0987fea6e48fd7cd3a2b34fbb805d81b9c4a798" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Nancy R Zhang, Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew Pardoll, Christina Jackson
          </td>
          <td>2025-09-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Treatments for high-grade serous ovarian cancer (HGSOC) are initially effective but most invariably fail. Although they can successfully suppress the bulk of the tumour cell population, residual cancer cells can enter alternative therapy-resistant cell fates highlighted by proliferative arrest. Understanding the nature of these fates and how cells may resume uncontrolled proliferation will lead to the development of new treatments for HGSOC. In this study, we examine the response of HGSOC cells to standard of care cisplatin chemotherapy and to the RNA Polymerase I transcription inhibitor CX-5461/Pidnarulex, two drugs that elicit a potent DNA damage response and growth arrest. Here, we identify that HGSOC cells exposed to these therapies show multiple hallmarks of therapy-induced senescence (TIS) and derive a core TIS gene expression signature irrespective of genetic background or senescence trigger. Given that TIS is a potentially escapable state, we have performed a focussed drug screen to identify drugs that eradicate senescent HGSOC cells. We identify that therapy-induced senescent HGSOC cells, including those with decreased sensitivity to senolytic drugs that inhibit the pro-survival protein BCL-XL, can be eliminated using drugs that induce ferroptosis, an iron-dependent form of cell death. Mechanistically, we demonstrate that senescent HGSOC cells have altered expression of regulators of iron metabolism leading to intracellular iron overload that underpins this targetable vulnerability. Together, we highlight elevated levels of iron as a TIS biomarker in HGSOC and the potential of inducing ferroptosis to eradicate residual HGSOC cells following initial therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3787e084ffe3e100d6ec83768df6b1bb600d9e4a" target='_blank'>
              Exploiting dysregulated iron homeostasis to eradicate persistent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Carmelo Cerra, Madeleine R. C. Tancock, N. Thio, Ada Koo, AnnRann Wong, Karla J Cowley, Swati Varshney, Madelynne O. Willis, Kaylene Simpson, D. Bowtell, E. Sanij, Elizabeth L Christie, Richard B. Pearson, Jian Kang, Keefe T. Chan
          </td>
          <td>2025-09-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a major global health concern, ranking among the most prevalent cancer in men worldwide. Despite the availability of various therapeutic options, the clinical efficacy of current anti-PCa agents is often compromised by drug resistance and adverse effects. DNA minor groove binders offer a potential therapeutic alternative, owing to their selective mechanism of action and favorable safety profiles. In the present study, we utilized a multiomics strategy to investigate the molecular impact of novel compound MGB4. LNCaP cells were treated with doxorubicin, MGB4, or a combination of both, followed by LC-MS/MS-based untargeted proteomics and metabolomics analyses. One-way ANOVA (p-value <0.05) revealed 55 significantly dysregulated proteins and 57 altered metabolites across treatments. Our findings indicate that both MGB4 and doxorubicin impacted key cellular pathways, including inhibition of translation and alterations in sphingolipid and amino acid metabolism, while doxorubicin and the combination therapy also reduced spermine and spermidine metabolism. Notably, the combined treatment exhibited synergistic effects, significantly impacting purine metabolism and reducing metabolite levels more than individual therapies. This study provides key molecular insights into MGB4 and doxorubicin's mechanisms, supporting MGB4 as a potential prostate cancer drug candidate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0baef9cfc704d2fe10207a5529ebf62ac3bfbc81" target='_blank'>
              Untargeted Multiomics of LNCaP Cell Line Treated with a Novel DNA Minor Groove Binder and/or Doxorubicin Using Mass Spectrometry.
              </a>
            </td>
          <td>
            Ruba A Zenati, Nelson C. Soares, Hasan Y. Alniss, Hamza M. Al-Hroub, Raafat El-Awady, A. Abuhelwa, Wafaa S Ramadan, Shereen M. Aleidi, W. El‐Huneidi, Eman Abu-Gharbieh, K. Alzoubi, Yasser K. Bustanji, M. Semreen
          </td>
          <td>2025-09-04</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="


 Cisplatin remains a standard first-line therapy for epithelial ovarian cancer; however, the development of chemoresistance leads to poor prognosis and high recurrence rates. Kaplan–Meier analysis of patient data (TCGA) revealed that cisplatin-sensitive tumors are associated with significantly improved overall survival, underscoring the need for strategies to overcome resistance in clinical settings. To investigate the underlying mechanisms, we conducted a comprehensive bioinformatics analysis of publicly available GEO datasets from cisplatin-treated ovarian cancer patients (n = 255). This analysis identified six key molecular drivers of resistance: ZBTB33 (Kaiso), pregnane X receptor (PXR), NF-κB, HER2 (ERBB2), P-glycoprotein (P-gp/ABCB1), and HIF1A. These targets were validated in ovarian tumor specimens via immunohistochemistry, confirming elevated expression in chemo-resistant disease. Additionally, the quantitative real-time PCR analysis confirms the transcriptional upregulation of the six resistance-associated genes in cisplatin-resistant SKOV3 and OVCAR-5 ovarian cancer cells, consistent with the IHC findings. The average fold change in mRNA transcripts ranged from 2.4 for P-glycoprotein (P-gp) to 5 for both NF-kB and Kaiso (ZBTB33). Although less well studied in ovarian cancer, Kaiso (ZBTB33) is known to regulate EMT and tumor invasion in other solid tumors. Functional studies using SKOV3 and OVCAR-5 cell lines demonstrated that knockdown of Kaiso (ZBTB33) via RNA interference significantly increased cisplatin-induced cell death, indicating a direct role in therapeutic resistance. Furthermore, we investigated the synergistic effects of combining SDA, a plant-based omega-3 fatty acid known to inhibit NF-κB, and cisplatin on the cell death of SKOV3 and OVCAR-5 cell lines, comparing the results to those of each compound used individually. Interestingly, co-treatment with stearidonic acid (SDA) synergistically enhanced the cytotoxicity of cisplatin at a lower dose in both cell models. These findings reveal a clinically relevant resistance signature and highlight the therapeutic potential of combinatorial strategies that target both transcriptional regulators (e.g., Kaiso) and inflammatory signaling (e.g., NF-κB). Dual targeting of these pathways may resensitize tumors to cisplatin and improve outcomes for patients with advanced ovarian cancer.



 Mahmoud Mansour, Sabrina Van Ginkel, Maaz Kamal Alata, Ibrahim Bani, Isra Elhussin. Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a654db1a318364cee4d66d47d1f28ac0d3c685d" target='_blank'>
              Abstract A025: Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization
              </a>
            </td>
          <td>
            Mahmoud Mansour, Sabrina Van Ginkel, M. Alata, Ibrahim Bani, Isra Elhussin
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Methods Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type-tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Results Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. Conclusion: These findings may provide guidance for development of clinical trial combination regimens including cirtuvivint, CC-671 or iadademstat. Full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd17fd73c746ccc133073188d85cd1b611794f0" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H. Doroshow
          </td>
          <td>2025-09-12</td>
          <td>Cancer Chemotherapy and Pharmacology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Whole-genome duplication (WGD) represents an evolutionarily conserved process occurring in prokaryotes, eukaryotes, and somatic mammalian tissues. While developmentally programmed WGD supports normal tissue regeneration, unscheduled WGD drives chromosomal instability and oncogenic progression in cancer. Recent studies have clarified dual roles of WGD across physiological homeostasis and disease pathogenesis. Here, we review the prevalence of WGD, the molecular mechanisms driving its major causes and its biological consequences. In addition, we highlight recent advancements in WGD detection, including both conventional cytogenetic techniques and newly developed high-throughput sequencing approaches. The integration of multi-omics and machine learning further improves ploidy analysis, particularly in cancer research. Together, these insights establish WGD as a critical regulator of development, regeneration, and disease and underscore the importance of emerging computational and sequencing tools for its precise characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ebbe235d853294a695ca1852f4aab18a1bed1ef" target='_blank'>
              The dual role of whole-genome duplication: biological mechanisms, functional consequences, and detection advances.
              </a>
            </td>
          <td>
            Yawei Song, Jiajie Yang, Shuheng Wu, Wei Wu
          </td>
          <td>2025-09-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew M Pardoll, Christina Jackson, Nancy R Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults, with poor prognosis due to its high recurrence rate and resistance to therapy. Despite standard of care treatment, surgical resection followed by radiotherapy and temozolomide chemotherapy, relapse occurs in 80-90% of patients. This is largely attributed to the extensive genetic and cellular heterogeneity of GBM, including a subpopulation of tumor-initiating cells (TICs) that drive therapeutic resistance and tumor regrowth.



 To identify therapeutic strategies that bypass tumor-specific mutations, we aimed to repurpose FDA-approved drugs to target common vulnerabilities across diverse TIC populations. We employed patient-derived 3D TIC cultures and orthotopic xenograft (PDX) models to focuse on essential cellular processes such as apoptosis, cell cycle control, protein synthesis, and the DNA damage response. A high-throughput screen of 2,702 FDA-approved compounds was conducted on three heterogeneous TIC models: two derived from matched primary and recurrent GBMs, and one from an independent recurrent tumor.



 Of the 2,702 FDA-approved compounds screened, only 66 (approximately 2%) demonstrated consistent and significant anti-tumor activity across all TIC models. From this subset, we prioritized three candidates: Homoharringtonine, a protein synthesis inhibitor; Ixazomib Citrate, a proteasome inhibitor; and Panobinostat, a histone deacetylase inhibitor. Notably, Homoharringtonine and Ixazomib have not been previously evaluated in GBM, highlighting their potential as novel therapeutic agents. Although Panobinostat has been explored in prior studies, it exhibited robust efficacy across all tested models.



 These results highlight our TICs panel as a valuable screening platform for real-time therapy monitoring and for identifying promising TIC-targeting drugs for GBM treatment. Ongoing studies will investigate resistance mechanisms through single-cell transcriptomic and epigenomic profiling, with the goal of informing the design of rational, synergistic combination therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bef61d6e7408f7fec265fc541b325c0d4e191e84" target='_blank'>
              P01.05.A UNRAVELING DRUG EFFICACY IN GLIOBLASTOMA: TARGETING TUMOR-INITIATING CELLS TO OVERCOME INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            A. Marinaro, D. Osti, L. Fuzio, S. Ghouse, M. Del Bene, F. DiMeco, G. Pelicci
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Cell division cycle 6 (CDC6) is a key licensing factor for DNA replication in the G1 and S phases. Besides initiating replication, CDC6 also helps establish and maintain the S-M checkpoint, ensuring genomic stability. Emerging evidence highlights its dysregulation in various cancers, implicating CDC6 in tumor progression and therapy resistance. However, a comprehensive pan-cancer analysis evaluating its diagnostic, prognostic and immunomodulatory potential remains lacking, underscoring the need for further investigation. Methods By integrating multi-omics datasets from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression Project (GTEx), cBioPortal, Human Protein Atlas (HPA), UALCAN and SangerBox, we employed systematic bioinformatics approaches to investigate the oncogenic role of CDC6 across multiple cancer types. Our analysis encompassed prognostic associations, mutational landscapes, tumor immune microenvironment (TIME) infiltration patterns and epigenetic regulation via DNA methylation, providing a pan-cancer perspective on CDC6’s potential role in tumorigenesis. In addition, CDC6’s role in melanoma cell proliferation, invasion and migration was experimentally assessed. Results Pan-cancer analysis demonstrated CDC6 as a consistently upregulated oncogene across multiple malignancies, exhibiting significantly elevated expression compared to normal tissues. Notably, CDC6 is closely associated with prognosis across various cancer types. Our investigation further revealed robust correlations between CDC6 expression and immune cell infiltration patterns. Epigenetic profiling identified significant associations between CDC6 expression and DNA methylation alterations in nine cancer types. Functional studies validated CDC6’s oncogenic role, where its overexpression significantly promoted cellular proliferation, migration and invasion in melanoma. Conclusions Our study demonstrates that CDC6 serves as a crucial oncogenic driver across diverse tumor types, establishing its dual utility as a diagnostic biomarker and independent prognostic indicator. Importantly, we identified a significant correlation between elevated CDC6 expression and specific immune microenvironment alterations, suggesting its potential as a predictive biomarker for immunotherapy response. These findings demonstrate CDC6’s dual role in cancer development and immune regulation, warranting further investigation into its mechanisms and therapeutic potential. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14782-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b86557cdc5c9f375fa0289acd10e271c4227fce8" target='_blank'>
              CDC6 as a pan-cancer immunological and prognostic biomarker and its role in suppressing melanoma malignancy
              </a>
            </td>
          <td>
            Lijun Mo, Min Jia, Qian Wu, Qingqing Xu, Z. Mai, Bo Fu, Zhengqiang He, Mingchang Huang, Yinyuan Lan, Qiaozhen Xu, Zhenmou Xie, Xiaoni He, Yaohua Xue
          </td>
          <td>2025-09-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer affects women at an increasingly younger age, with genetic predispositions and other factors contributing to its second-highest cancer mortality rate. The diversity of pharmacological treatment stems from its heterogeneity, which favors a more precise approach to each subtype. Despite the extensive advances in medicine in recent decades, the problem of treating cancer patients remains significant. The problem with modern therapeutic methods is low effectiveness, emerging side effects, difficulty in eliminating all cancer cells, and the quite common use of monotherapy and the associated drug resistance, which may lead to disease progression. The aim of this review is to present the latest therapeutic strategies (combination therapies) used in the treatment of breast cancer. PubMed databases and clinical data from ClinicalTrials.gov were used for this purpose. The review included characteristics of the latest clinical trials from the last year (2024–2025), which present currently recruiting studies of breast cancer treatment with immunotherapy. The review also presented characteristics of clinical trials from the last 5 years (2020–2025) using nanoparticles as an adjunct to breast cancer treatment. Articles published between 2016 and August 2025 (excluding articles that describe the first use of a given drug) were included in the review. The review analyzed drugs targeting molecular targets, including intracellular pathways responsible for cell cycle regulation, as well as new directions such as nanotechnology in treatment breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed040e9e74acc963a86d9067aa624d7cb2b27cc7" target='_blank'>
              Breast Cancer Treatments: Drugs Targeting the PI3K/AKT/mTOR Pathway, TNBC Therapy and Future Directions: A Review
              </a>
            </td>
          <td>
            Klaudia Dynarowicz, D. Bartusik-Aebisher, Katarzyna Koszarska, Aleksandra Kotlińska, David Aebisher
          </td>
          <td>2025-10-10</td>
          <td>Life</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction Drug discovery and development is a complex, multi-stage process that often spans over a decade and involves high costs and low success rates. Preclinical testing, particularly the assessment of drug-induced liver injury, plays a crucial role in identifying safe and effective therapeutics before clinical trials. In vitro models based on hepatic cell lines are commonly used to study hepatotoxicity, yet their physiological relevance varies significantly. This study aimed to compare the expression of key liver-specific genes and proteins in four widely used hepatic cancer cell lines—HepG2, C3A, SNU449, and SNU475—with those in primary human hepatocytes. Methods Using RT-qPCR, protein analysis and metabolic tests, we assessed the ability of these cell lines to perform liver-specific functions, especially drug metabolism. Results We found significant differences in liver-related gene expression and metabolic profiles among tested cell lines, especially the most striking differences were found between tumor cells of divergent origin: hepatoblastomas and hepatocellular carcinomas. Discussion Our findings emphasize the importance of careful selection and validation of in vitro models in hepatotoxicity testing, as significant differences exist in their gene expression profiles and functional characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78bcc6246608be71a7900f976920735f2ad06f7" target='_blank'>
              Variable expression of hepatic genes in different liver tumor cell lines: conclusions for drug testing
              </a>
            </td>
          <td>
            Monika Joanna Wisniewska, Agnieszka Wencel, Malgorzata Jakubowska, J. A. Motyl, Krzysztof Dudek, Beata Burzynska, Dorota Genowefa Pijanowska, K. Pluta
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The clonal evolution model provides a framework for understanding the evolution of cancer cells. According to this model, cancer cells accumulate genetic mutations over time, and these mutations are passed down to their descendants, leading to genetic diversity within the tumor. Some of these mutations confer selective advantages, causing certain lineages of cancer cells (clones) to dominate and expand. However, this model is rooted in certain conceptual assumptions, which we propose to revisit by considering the potential involvement of retrotransposons in cancer initiation and progression. In recent years, it has become evident that transposable elements, particularly retrotransposons, play a significant role in driving cancer transformation and progression. We first review how current knowledge about retrotransposon activity aligns with the clonal evolution model by highlighting its ability to modulate cancer cell fitness. We then take a forward-looking perspective to explore additional ways retrotransposons may also influence clonal dynamics beyond the current model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb2629152d1c164e115163464d0e7599e1333a7" target='_blank'>
              Revisiting Clonal Evolution Through the Light of Retrotransposons.
              </a>
            </td>
          <td>
            Anaïs Lamoureux, Emilie Elvira-Matelot, Françoise Porteu, L. Laplane
          </td>
          <td>2025-10-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation can lead to breaks in the double-stranded structure of DNA. Although cells have mechanisms that can repair such breaks, misrejoining events can occur that lead to mutations in an individual’s genome. When mutations affect certain genes, such as oncogenes and tumor-suppressor genes, cancer can result. Although ionizing radiation is a known cause of cancer, it has been difficult to determine how much of an excess cancer risk results from low-dose radiation exposure. Radiation signatures that distinguish radiation-induced cancers from spontaneous cancers are thus useful for assessing cancer risks after exposure to low-dose radiation. Recent advances in high-throughput sequencing technologies have enabled the comprehensive profiling of mutations in individual cancer cells. Analyses of radiation-associated cancer genomes have identified an increase in small deletions and chromosome rearrangements, including large deletions, inversions, and translocations, which are likely to be formed by the misrejoining of DNA double-strand breaks. Among these, chromosome rearrangements leading to cancer-related mutational events are potentially useful as biomarkers and as therapeutic targets for treating radiation-induced cancer. This review summarizes research on radiation-induced mutational signatures found in animal and human cancers, consolidates current knowledge, and discusses the future perspectives in this research field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Abstract LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is an aggressive ovarian cancer with a heterogeneous tumor microenvironment (TME). Advances in single-cell RNA sequencing (scRNA-seq) and spatially-resolved transcriptomics have enabled the study of complex TME. This study explores connections between molecular subtypes described from bulk transcriptomes and spatial domains characterized by distinct gene expression in HGSC and their variability between patients. We explored both intra- and inter-tumor heterogeneity across 2D space and identified differing spatial patterns of gene expression pertaining to immune pathways and vasculature development. Functional characterization of tumor spaces revealed potentially shared cell states across molecular subtypes, while correlation analysis underscored subtype-specific spatial anti-colocalization between spots exhibiting antigen-presenting functions and B cell-mediated immunity. Lastly, we performed spatially-aware cell-cell communication analysis on the spatial samples and identified a molecular subtype specific difference in total signaling activity and heterogeneity in Midikine signaling between the differentiated subtype. Our results suggest that generating multiple tissue slices per patient might be necessary to enable comprehensive characterization of HGSC spatial transcriptomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12564a0c02e3362085c018e2a380154da3a01959" target='_blank'>
              Characterizing intra- and inter-tumor heterogeneity in Ovarian high-grade serous carcinoma subtypes using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Weishan Li, Laurie Grieshober, Jason Gertz, Adriana Ivich, J. Doherty, Casey S. Greene, SC Hicks
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Method Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Results Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). Conclusion This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly aggressive malignancy with poor prognosis and significant molecular heterogeneity. This study investigates the role of tumor-specific enhancers in metabolic reprogramming, focusing on glucose-6-phosphate dehydrogenase (G6PD) and the pentose phosphate pathway (PPP). Using native elongating transcript-cap analysis of gene expression and single-cell RNA sequencing, tumor-specific enhancers driving G6PD overexpression were identified in ICC tumor epithelial cells. Functional assays demonstrated that G6PD promotes tumor proliferation by enhancing PPP activity and maintaining redox homeostasis, which provides NADPH to counter oxidative stress. Enhancer knockdown disrupted G6PD expression and PPP activity, increasing reactive oxygen species levels and reducing the NADPH/NADP+ ratio. These metabolic changes impaired tumor cell proliferation and sensitized ICC cells to cisplatin, emphasizing the dual therapeutic potential of targeting G6PD to inhibit tumor growth and overcome chemoresistance. Survival analyses showed that high G6PD expression correlates strongly with poor overall survival in ICC patients. While previous studies have recognized the roles of G6PD and PPP in cancer metabolism, this study uniquely links enhancer-mediated regulation to these processes in ICC, offering novel insights into epigenetic mechanisms driving metabolic reprogramming. Moreover, the findings highlight tumor-specific enhancers as critical epigenetic drivers of ICC progression, with potential as therapeutic targets. Future research should explore the integration of enhancer profiling into precision medicine frameworks and the development of novel enhancer-targeting strategies. These efforts could uncover additional metabolic vulnerabilities and provide effective treatments for this highly aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f09ce5004e00e224dd82b9d53133eeef55290e" target='_blank'>
              Epigenetic Regulation of G6PD Drives Metabolic Reprogramming in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Yusuke Nakano, Miwa Tanaka, Takeharu Sakamoto, M. Hashimoto, Taro Tobo, Hideyuki Saito, Tadashi Abe, Tomohiko Ikehara, Takashi Ofuchi, Koto Kawata, Takaaki Masuda, T. Ogino, Mamoru Uemura, H. Eguchi, Y. Doki, K. Mimori
          </td>
          <td>2025-10-07</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="BACKGROUND
Therapies for diffuse glioma induce DNA damage response (DDR), and strategies to exploit DDR defects are active areas of investigation. While global DNA methylation profiling effectively classifies gliomas into subtypes, the epigenetic and gene expression patterns of DDR genes, and their contribution to tumor classification and outcomes, have yet to be fully elucidated. Thus, dissecting the DDR epigenetics, gene expression, and single-cell heterogeneity may reveal key molecular characteristics, refine prognosis, and identify novel treatment strategies and resistance mechanisms.


METHODS
We characterized DDR epigenetics and gene expression of TCGA glioblastomas (GBM) and low-grade gliomas (LGG). Single-cell protein analysis by imaging mass cytometry (IMC) was performed on a separate cohort of 118 diffuse gliomas.


RESULTS
Analysis of TCGA cohorts revealed two DDR methylation groups that correlated with IDH mutation status and previously reported molecular groups. DDR transcription profiling further classified tumors into four groups. Those with high DDR transcription across pathways were linked to poor survival independent of IDH or MGMT status, and potentially improved prognostication beyond established biomarkers. Single-cell characterization of a separate cohort revealed intratumoral DDR diversity and identified proliferative tumor cells with high DDR protein expression across pathways that are associated with unfavorable grade and survival.


CONCLUSIONS
Tumor-level epigenetic and transcriptional DDR signatures alone can distinguish molecular-defined diagnosis and outcomes of gliomas beyond established biomarkers. A higher abundance of glioma cells with high DDR effector expression across pathways is associated with poor survival. Thus, clinical assessment of pan-DDR expression may inform prognosis and identify potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b0fbeaa6fa3878bda82552131f1d944fc87086" target='_blank'>
              DNA damage response profile distinguishes poor-acting gliomas with shared methylome signatures.
              </a>
            </td>
          <td>
            Nalin Leelatian, Charu Singh, Richard Bouffard, R. Sundaram, Kirsten E Diggins, William Sullivan, Sateja Paradkar, Zeynep E. Erson Omay, B. Mobley, Susan E. Gueble, Juan C Vasquez, R. Bindra
          </td>
          <td>2025-08-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>