<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-17 09:41:26 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>65</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1830846c3b9289add1524e2178f4fc69c53b21d" target='_blank'>
              Deciphering the transcriptomic landscape of pyrazinamide resistance in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Ananthi Rajendran, Ahmed Kabir Refaya, K. Palaniyandi
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies. This study introduces single-cell transcription factor (TF) sequencing, a single-cell barcoded and doxycycline-inducible TF overexpression approach that reveals dose-sensitive functional classes of TFs and cellular heterogeneity by mapping TF dose-dependent transcriptomic changes during the reprogramming of mouse embryonic multipotent stromal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793bb57e9ee8be0a292f9fe822d8b2b3a9781a8c" target='_blank'>
              Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq
              </a>
            </td>
          <td>
            Wangjie Liu, Wouter Saelens, Pernille Y Rainer, Marjan Biočanin, V. Gardeux, Antoni J. Gralak, Guido van Mierlo, Angelika Gebhart, Julie Russeil, Tingdang Liu, Wanze Chen, Bart Deplancke
          </td>
          <td>2025-10-01</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Despite the outstanding achievements of precision medicine in hematology, many targeted therapies eventually fail due to the emergence of resistance mechanisms. Traditionally, a genocentric approach has been adopted to uncover the molecular underpinnings of treatment resistance. This has contributed to identifying resistance gene mutations and designing novel therapeutic molecules with increased potency for the mutant target. However, over the last five years, additional non-genetic adaptations have become increasingly recognized as crucial promoters of treatment resistance. In parallel, emerging works in the field of evolutionary biology suggest that advantageous phenotypic traits appear most often due to cell-intrinsic phenotypic plasticity and can arise independently of gene mutations. In selected cases, single genetic abnormalities such as those involving TP53 can prime human cells for plasticity and facilitate phenotypic variability. In this narrative review, we retrace the resistance mechanisms to targeted therapies in the framework of these novel evolutionary concepts. We highlight the dichotomy between genes-first and phenotypes-first pathways of treatment adaptation, with the former being driven by traditional single-point mutations and the latter initiated by the phenotypic diversity and the high-level plasticity of cancer cells. Focusing on resistance mechanisms to kinase inhibitors and BH3 mimetics in leukemias and lymphomas, we describe how each drug can trigger both escape routes, which may even coexist within the tumor bulk of individual patients. Lastly, we provide a three-step translational perspective on how to counteract phenotypes-first resistance mechanisms, with the aim of prolonging disease control in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/326e4b504dccc956ee299598bcd494d4433bc387" target='_blank'>
              Genes-first and phenotypes-first paths to treatment resistance in hematological malignancies
              </a>
            </td>
          <td>
            Edoardo Tamellini, Cristina Frusteri, Isacco Ferrarini
          </td>
          <td>2025-11-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>None</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd8a87b52ce435726d0713cb35d9987d7862c73" target='_blank'>
              The two sides of resistance: aggressiveness and mitotic instability as the Achilles’ heel of Osimertinib-resistant NSCLC
              </a>
            </td>
          <td>
            P. Armanetti, E. Cabrera San Millán, R. Mercatelli, Chiara Cavallini, Alice Chiodi, E. Mosca, Arianna Consiglio, Nicola Losito, Luca Menichetti, E. Bramanti, Azhar Ali, Elena Levantini, Giorgia Maroni
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b28a4cea8f32f994db559bdedf94de2b973cd5" target='_blank'>
              Clonal Evolution of Pediatric Acute Myeloid Leukemia and Its Contribution to Disease Relapse
              </a>
            </td>
          <td>
            M. Umeda, P. Kolekar, A. L. Huskey, S. Suryaprakash, J. Ma, H. Li, T. Westover, M. P. Walsh, G. Song, Y. Liu, Q. Tran, V. Balagopal, L. Wang, S. W. Brady, T. A. Gruber, S. Pounds, P. E. Mead, S. E. Karol, H. Inaba, J. Rubnitz, R. Ribeiro, X. Ma, J. Klco
          </td>
          <td>2025-11-06</td>
          <td>None</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and aggressive malignancy, characterized by a lack of targeted therapies and limited clinical benefits. Here, we conducted an optimized whole-genome CRISPR screen across five HNSCC cell lines aimed at identifying actionable genetic vulnerabilities for rapid preclinical evaluation as novel targeted therapies. Given their critical role in cancer, cyclin-dependent kinases (CDKs) were prioritized for further investigation. Among these, CDK7 was identified as an essential and targetable gene across all five cell lines, prompting its selection for in-depth functional and molecular characterization. Genetic and pharmacological inhibition of CDK7 significantly and consistently reduced tumor cell proliferation due to generalized cell cycle arrest and apoptosis induction. Additionally, CDK7 knockout (KO) and selective inhibitors (YKL-5-124 and samuraciclib) demonstrated potent antitumor activity, effectively suppressing tumor growth in HNSCC patient-derived organoids (PDOs), as well as in both cell line- and patient-derived xenograft (PDX) mouse models with minimal toxicity. Mechanistically, CDK7 inhibition led to a broad downregulation of gene sets related to cell cycle progression and DNA repair, and significantly reduced the transcription of essential genes and untargetable vulnerabilities identified by our CRISPR screen. These findings highlight CDK7 as a promising therapeutic target for HNSCC. Our study provides strong evidence of the robust antitumor activity of CDK7-selective inhibition in disease-relevant preclinical models, strongly supporting its progression to clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4085230d5db63fd39c16115dd67e53fe6113079" target='_blank'>
              CDK7-targeted therapy effectively disrupts cell cycle progression and oncogenic signaling in head and neck cancer
              </a>
            </td>
          <td>
            María Otero-Rosales, M. Álvarez-González, Irene Pazos, Beatriz de Luxán-Delgado, Sonia Del Marro, Esperanza Pozo-Agundo, Mar Rodríguez-Santamaría, Ana López-Fernández, D. Corte-Torres, R. Granda-Díaz, S. Álvarez-Teijeiro, Iván Fernández-Vega, Corina Lorz, Ramón García-Escudero, J. P. Rodrigo, Konstantinos Tzelepis, G. Vassiliou, Irene Ferrer, Mónica Álvarez-Fernández, J. García-Pedrero, F. Hermida-Prado
          </td>
          <td>2025-11-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, A. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, A. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Treatments for high-grade serous ovarian cancer (HGSOC) are initially effective but most invariably fail. Although they can successfully suppress the bulk of the tumour cell population, residual cancer cells can enter alternative therapy-resistant cell fates highlighted by proliferative arrest. Understanding the nature of these fates and how cells may resume uncontrolled proliferation will lead to the development of new treatments for HGSOC. In this study, we examine the response of HGSOC cells to standard of care cisplatin chemotherapy and to the RNA Polymerase I transcription inhibitor CX-5461/Pidnarulex, two drugs that elicit a potent DNA damage response and growth arrest. Here, we identify that HGSOC cells exposed to these therapies show multiple hallmarks of therapy-induced senescence (TIS) and derive a core TIS gene expression signature irrespective of genetic background or senescence trigger. Given that TIS is a potentially escapable state, we have performed a focussed drug screen to identify drugs that eradicate senescent HGSOC cells. We identify that therapy-induced senescent HGSOC cells, including those with decreased sensitivity to senolytic drugs that inhibit the pro-survival protein BCL-XL, can be eliminated using drugs that induce ferroptosis, an iron-dependent form of cell death. Mechanistically, we demonstrate that senescent HGSOC cells have altered expression of regulators of iron metabolism leading to intracellular iron overload that underpins this targetable vulnerability. Together, we highlight elevated levels of iron as a TIS biomarker in HGSOC and the potential of inducing ferroptosis to eradicate residual HGSOC cells following initial therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3787e084ffe3e100d6ec83768df6b1bb600d9e4a" target='_blank'>
              Exploiting dysregulated iron homeostasis to eradicate persistent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Carmelo Cerra, Madeleine R. C. Tancock, N. Thio, Ada Koo, AnnRann Wong, Karla J Cowley, Swati Varshney, Madelynne O. Willis, Kaylene Simpson, D. Bowtell, E. Sanij, Elizabeth L Christie, Richard B. Pearson, Jian Kang, Keefe T. Chan
          </td>
          <td>2025-09-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d57534bc0d9d0ab660dd7d7cddbf910c13f4aa" target='_blank'>
              Strain-resolved CRISPRi-Seq reveals conserved antibiotic vulnerabilities in Staphylococcus aureus
              </a>
            </td>
          <td>
            Maria-Vittoria Mazzuoli, Jessica Burnier, Albane Schmid, Louise Martin, Axel B. Janssen, Clément Gallay, A. Zinkernagel, J. Veening
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f13f3a42ad0960ba2f1336cd288a90d6e8ad4e1f" target='_blank'>
              Opposing Copy Number Variation Dynamics Shape Adaptation to Glucose and Galactose in Diploid Yeast
              </a>
            </td>
          <td>
            Prachitha Nagendra, Saket Choudhary, S. Saini
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, C. Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c573e7d40354c3158261048e704da74d1dc94a5" target='_blank'>
              Whole-chromosome duplications drive antimicrobial resistance in Aspergillus fumigatus
              </a>
            </td>
          <td>
            Anna E. Lehmann, Enrique Aguilar Ramírez, Nancy P. Keller, Joseph Heitman
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Antimycin A, an antifungal agent that inhibits mitochondrial respiration, provides a useful model for studying resistance mechanisms. Antifungal resistance is an escalating clinical concern with limited treatment options available. To understand the molecular mechanisms of antimycin A resistance, a genetically stable, antimycin A-resistant Saccharomyces cerevisiae strain was successfully developed for the first time through an evolutionary engineering strategy, based on long-term systematic application of gradually increasing antimycin A stress in repetitive batch cultures without prior chemical mutagenesis. Comparative whole genome resequencing analysis of the evolved strain ant905-9 revealed two missense mutations in PDR1 and PRP8 genes involved in pleiotropic drug resistance and RNA splicing, respectively. Using CRISPR/Cas9 genome editing tools, the identified mutations were introduced individually and together into the reference strain, and it was confirmed that the Pdr1p.M732R mutation alone confers antimycin A-resistance in S. cerevisiae. Comparative transcriptomic analysis of the reverse-engineered Pdr1p.M732R strain showed alterations in PDR (pleiotropic drug resistance), transmembrane transport, vesicular trafficking, and autophagy pathways. Our results highlight the potential key role of PDR1 in antifungal drug resistance. This study provides new insights into mitochondrial drug resistance and the adaptive potential of yeast under respiratory stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b896127a479ec689ce1d1e9d511ec8bcaa5efae" target='_blank'>
              Evolutionary engineering and molecular characterization of an antimycin A-resistant Saccharomyces cerevisiae strain: the key role of pleiotropic drug resistance (PDR1)
              </a>
            </td>
          <td>
            A. Topaloğlu, Can Holyavkin, Ömer Esen, Ogün Morkoç, Karl Persson, Cecilia Geijer, Z. P. Çakar
          </td>
          <td>2025-10-17</td>
          <td>FEMS Yeast Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a200a976abe18d2ada20b9002895dd27ac2f5" target='_blank'>
              Prognostic and Therapeutic Implications of BRAF Mutations in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Darren Lee, Yazan Abu-Shihab, Kate Plas, D. Nicolet, K. Mrózek, M. Routbort, K. Patel, C. Walker, Jill Buss, Andrew Stiff, Andrea Laganson, C. Dinardo, N. Daver, T. Kadia, F. Ravandi, Andrew J Carroll, J. Kolitz, B. Powell, William Blum, Maria R. Baer, G. Marcucci, Geoffrey L. Uy, Wendy Stock, Richard M. Stone, L. J. Medeiros, John C. Byrd, J. Blachly, Robert L. Bowman, J. Tyner, S. Loghavi, Ann-Kathrin Eisfeld, Linde A. Miles
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, Wesley S. Lu, Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Fleur Chapus, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oxaliplatin resistance remains a critical barrier to effective colorectal cancer treatment. The molecular mechanisms underlying this resistance are not fully understood, highlighting the need to define the transcriptional alterations that contribute to therapeutic failure. Accordingly, a comparative transcriptome analysis was performed on oxaliplatin-resistant colorectal cancer cells (HCT-116-ROx) and their parental counterparts (HCT-116) using RNA sequencing in this study. Differentially expressed gene (DEG) analysis was conducted using a quasi-likelihood negative binomial model. Gene Ontology (GO) enrichment and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment were carried out using the topGO and clusterProfiler packages, respectively. To confirm the robustness of the transcriptomic data, the genes with the most significant expression changes, based on false discovery rate-adjusted P value less than 0.05 and a |logFC| > 2 thresholds, were selected for validation by quantitative real-time PCR (qRT-PCR). A total of 313 DEGs were identified, including ALDH3A1 and TACSTD2 (upregulated) and IFITM1 (downregulated); these three genes were chosen for validation by qRT-PCR. Gene Ontology enrichment revealed significant changes in cell motility, redox regulation, and extracellular matrix remodeling. KEGG analysis indicated upregulation of ferroptosis, glutathione metabolism, and lysosome-related pathways, and downregulation of p53 signaling, oxidative phosphorylation, and cancer-specific pathways. Oxaliplatin-resistant colorectal cancer cells undergo multifaceted transcriptional reprogramming that promotes redox homeostasis, metabolic adaptation, and structural plasticity while suppressing apoptotic and mitochondrial functions. These changes support chemoresistance and may represent potential therapeutic targets to restore drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5ab501b314c92e8ff167ffcb8251950e164cb1" target='_blank'>
              Multifaceted transcriptional reprogramming supports oxaliplatin chemoresistance in colorectal cancer cells.
              </a>
            </td>
          <td>
            Gizem Calibasi-Kocal, Hasan Kurter, Zerrin Isik
          </td>
          <td>2025-10-09</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The vast majority of recurrent somatic mutations arising in tumors affect protein-coding genes in the nuclear genome. Here, through population-scale analysis of 14,106 whole tumor genomes, we report the discovery of highly recurrent mutations affecting both the small (12S, MT-RNR1) and large (16S, MT-RNR2) mitochondrial RNA subunits of the mitochondrial ribosome encoded within mitochondrial DNA (mtDNA). Compared to non-hotspot positions, mitochondrial rRNA hotspots preferentially affected positions under purifying selection in the germline and demonstrated structural clustering within the mitoribosome at mRNA and tRNA interacting positions. Using precision mtDNA base editing, we engineered models of an exemplar MT-RNR1 hotspot mutation, m.1227G>A. Multimodal profiling revealed a heteroplasmy-dependent decrease in mitochondrial function and loss of respiratory chain subunits from a heteroplasmic dosage of ~10%. Mutation of conserved positions in ribosomal RNA that disrupt mitochondrial translation therefore represent a class of functionally dominant, pathogenic mtDNA mutations that are under positive selection in cancer genomes. Analysis of 14,106 tumor genomes highlights recurrent mutations in mitochondrial ribosomal RNA encoded within the mitochondrial genome. Mutations occur at hotspot positions and are under strong purifying selection in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eda054c049bfbd95fe327306b7323841c22683" target='_blank'>
              Functionally dominant hotspot mutations of mitochondrial ribosomal RNA genes in cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Jacqueline Tait-Mulder, Minsoo Kim, Cerise Tang, Tricia Park, Flora McNulty, S. Lilla, S. Zanivan, Alejandro Huerta-Uribe, Benan N. Nalbant, Mark Zucker, D. Sumpton, Geoffray Monteuuis, Christopher B Jackson, Wei Wei, P. Chinnery, R. Chaligné, Caleb A. Lareau, Ed Reznik, P. Gammage
          </td>
          <td>2025-11-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4375fd9e9cdc4563ab3b711dfac8983eeba0ca" target='_blank'>
              Genome-Wide CRISPR/Cas9 Screening Reveals Lipid Metabolism and Inflammatory Signalling as Modulators of Ganoderic Acid DM Cytotoxicity
              </a>
            </td>
          <td>
            Nimo Abdullah, Jacqueline Lewis, Prakash Arumugam
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573448152ca34657743158301dc6d5b323113cd1" target='_blank'>
              TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition
              </a>
            </td>
          <td>
            Jean-François Spinella, J. Chagraoui, I. Boivin, Guillaume Richard Carpentier, Céline Moison, N. Mayotte, François Béliveau, Vincent-Philippe Lavallée, J. Hébert, Guy Sauvageau
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT The DNA damage response (DDR), an evolutionarily conserved surveillance mechanism enabling cells to detect genomic lesions and cell cycle arrest for repair, coordinates genomic stability through key effector kinases. In Candida albicans, our prior work characterized Rad53-mediated transcriptional reprogramming based on the RAD53 deletion strain, establishing its canonical DDR functions. However, Rad53 activation dynamically orchestrates cell cycle synchronization and repair coordination during genotoxic challenge. Investigating cellular modifications resulting from increased Rad53 functionality is essential for a deeper understanding of its regulatory response. To this end, we overexpressed Rad53 revealing its dual regulatory capacity in stress adaptation and developmental plasticity. Global transcriptomic profiling demonstrated that RAD53 overexpression enhances methyl methanesulfonate (MMS) resistance while paradoxically stimulating filamentous growth, accompanied by upregulation of DNA repair/replication genes (RFA3, POL30) and non-canonical DDR targets including cell wall remodeling factors. Comparative analysis of RAD53 deletion and overexpression transcriptomes identified enriched transcription factors, notably Sfl1, Yox1, and Rfx1. In particular, Rfx1 mediated the resistance to genotoxic stress resulting from RAD53 overexpression or deletion. Surprisingly, Rad53 overexpression substantially hindered hyphal formation and virulence, whereas its kinase-dead mutation restored these deficiencies. Notably, Rad53’s role in mediating morphogenesis was independent of Rfx1. This multi-modal functionality positions Rad53 as a critical node interfacing genomic stability, developmental plasticity, and pathogenicity in C. albicans. Our study enhances the understanding of the DDR pathway in C. albicans, providing a potential target for treating this fungal pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618ff870ecbc422dc2701275aa24d5074e975660" target='_blank'>
              Rad53 orchestrates divergent pathways governing genotoxic resistance, morphogenesis, and virulence in Candida albicans: Mechanistic decoupling through overexpression analysis
              </a>
            </td>
          <td>
            Jia Wang, Shaling Yang, Yi Zhou, Huaxin Cai, Jinrong Feng
          </td>
          <td>2025-10-06</td>
          <td>Virulence</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, A. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, M. Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation. Radiation-induced malignantly transformed cells undergo multi-scale 3D genome reorganization. TP63 acts as a key mediator, creating accessible chromatin sites that form tumour-specific loops activating the MYC oncogene during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The regulatory mechanisms governing transcriptional programs in the cancer genome remain elusive, particularly those concerning cell-type specificity. We carefully curated single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) and single-cell RNA sequencing (scRNA-seq) data from eight distinct carcinoma tissues, including breast, skin, colon, endometrium, lung, ovary, liver, and kidney. Using single-cell multi-omics analysis, we identified extensive open chromatin regions and constructed peak-gene link networks, which can reveal distinct cancer gene regulation and genetic risks. We further explored conserved epigenetic regulation across cell types within cancer and elucidated their functional implications. Moreover, we identified cell-type-associated transcription factors (TFs) that regulate key cellular functions, such as the TEAD family of TFs, which widely control cancer-related signaling pathways in tumor cells. In colon cancer, we further identified tumor-specific TFs that are more highly activated in tumor cells than in normal epithelial cells, including CEBPG, LEF1, SOX4, TCF7, and TEAD4, which are pivotal in driving malignant transcriptional programs and represent potential therapeutic targets, as corroborated by single-cell sequencing data from multiple sources and in vitro experiments. Our findings provide a comprehensive understanding of the regulatory dynamics underlying carcinomas and offer valuable insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdeeefa6fa9c95e73d3a569b4614cf1e9a4ee17" target='_blank'>
              Single-cell multi-omics analysis reveals cancer regulatory elements of transcriptional programs and clinical implications
              </a>
            </td>
          <td>
            Xiaowei Tang, Qiaoling Zhang, Zichu Shen, Jian Xiao, Minghao Li, Xiangyan Meng, Chenyi Wang, Guangze Zhang, Anhang Liu, Yuxin Yin
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Effective cancer immunotherapy relies on the clonal proliferation and expansion of CD8+ T cells in the tumor. However, our insights into clonal expansions are limited, owing to an inability to track the same clones in tumors over time. Here, we develop a multi-site tumor mouse model system to track hundreds of expanding and contracting CD8+ T cell clones over multiple timepoints in tumors of the same individual. Through coupling of clonal expansion dynamics and single-cell RNA/TCR-seq data, we identify a transcriptomic signature in PD-1+Ly108+ precursor exhausted cells that strongly predicts rates of intratumoral clone expansion. The signature correlates with expansion in mice, both with and without immunotherapies, and in patients undergoing PD-1 blockade therapy. Expression of the signature during treatment corresponds with positive clinical outcomes. Downregulation of the signature precedes clone contraction—a phase in which clones contract but maintain revivable precursor exhausted cells in the tumor. LAG-3 blockade re-activates the expansion signature, re-expanding pre-existing clones, including previously contracted clones. These findings reveal how the study of clonal expansion dynamics provide a powerful ‘pan-immunotherapy’ signature for monitoring immunotherapies with implications for their future development. Effective response to immune checkpoint inhibitors depends on the proliferation and expansion of tumor-reactive CD8+ T cells. By combining analysis of CD8+ T cell clones in a preclinical multi-site tumor model with clinical datasets, the authors track the expansion dynamics of hundreds of T cell clones over time and define a signature to predict intratumoral CD8+ T cell expansion in response to immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b741acd274fcbe8ee1c316efd33c130b4d3fe4" target='_blank'>
              A pan-immunotherapy signature to predict intratumoral CD8+ T cell expansions
              </a>
            </td>
          <td>
            Munetomo Takahashi, Mikiya Tsunoda, Hiroyasu Aoki, Masaki Kurosu, H. Ogiwara, Shigeyuki Shichino, D. Bending, S. Ishikawa, James E. D. Thaventhiran, Kouji Matsushima, S. Ueha
          </td>
          <td>2025-10-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="ABSTRACT The Proline-Glutamate/Proline-Proline-Glutamate (PE/PPE) gene family comprises approximately 10% of the Mycobacterium tuberculosis (Mtb) genome and is characterized by GC-rich, highly repetitive sequences. As a result, these genes are usually excluded from short-read-based whole-genome sequencing analyses, leaving their sequence diversity and evolutionary dynamics poorly characterized. Recently, a genome masking approach demonstrated that roughly 54% of PE/PPE sequences are recoverable from short-read data, providing an opportunity to examine the evolution of this gene family at a population level. Here, we analyzed 51,229 Mtb genomes to characterize sequence diversity and selection pressures across the PE/PPE gene family. Overall, we observed that PE/PPE genes are under relaxed purifying selection compared to other gene categories, as evidenced by higher ratios of nonsynonymous to synonymous polymorphisms (pNpS) and greater mutation burdens. We identified 12 PE/PPE genes with signatures of positive selection and 7 with selective pressure associated with antibiotic resistance. Among these genes, PPE51 exhibited selection favoring loss-of-function mutations, which occurred only in Mtb strains that were already multidrug-resistant (MDR). This pattern suggests either compensatory evolution or adaptation related to resistance against second-line or newly introduced drugs. Additionally, we identified T-cell epitopes in six PE/PPE genes that were subject to diversifying selection, suggesting immune-driven adaptation. Collectively, this work provides a baseline characterization of genetic diversity in PE/PPE genes and highlights specific genes that may be involved in adaptation to host immunity and antibiotic pressure and represent candidates for further investigation. IMPORTANCE Tuberculosis remains a significant global health challenge, partly due to Mycobacterium tuberculosis (Mtb)’s remarkable evolutionary adaptation to antibiotics and human immune responses. Around 10% of its genome comprises PE/PPE genes, whose functions and evolutionary dynamics are poorly understood due to their repetitive sequences and high GC content. In this study, we analyzed 51,229 global Mtb genomes using an advanced genome-masking method, revealing numerous PE/PPE genes under positive selection, potentially facilitating antibiotic resistance and immune evasion. Notably, PPE51 often loses its function in strains resistant to multiple antibiotics, suggesting a role in bacterial survival during drug treatment. Additionally, we identified mutation-prone regions within six PE/PPE genes, highlighting potential targets for future vaccine development. Collectively, our findings underscore the crucial role of PE/PPE genes in Mtb evolution and drug resistance, providing valuable insights to inform novel therapeutic and vaccine strategies. Tuberculosis remains a significant global health challenge, partly due to Mycobacterium tuberculosis (Mtb)’s remarkable evolutionary adaptation to antibiotics and human immune responses. Around 10% of its genome comprises PE/PPE genes, whose functions and evolutionary dynamics are poorly understood due to their repetitive sequences and high GC content. In this study, we analyzed 51,229 global Mtb genomes using an advanced genome-masking method, revealing numerous PE/PPE genes under positive selection, potentially facilitating antibiotic resistance and immune evasion. Notably, PPE51 often loses its function in strains resistant to multiple antibiotics, suggesting a role in bacterial survival during drug treatment. Additionally, we identified mutation-prone regions within six PE/PPE genes, highlighting potential targets for future vaccine development. Collectively, our findings underscore the crucial role of PE/PPE genes in Mtb evolution and drug resistance, providing valuable insights to inform novel therapeutic and vaccine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9b29bb129ce08affc17b0455ca54e5010e1604c" target='_blank'>
              Ongoing evolution of PE/PPE genes in Mycobacterium tuberculosis associated with drug resistance and host immune response
              </a>
            </td>
          <td>
            Mingyu Gan, Dan Wang, Suqing Li, Qinglan Wang, Qingyun Liu
          </td>
          <td>2025-09-22</td>
          <td>mSystems</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="The recurrence of clinically advanced cancers is an evolutionary consequence of standard-of-care chemotherapies generally administered at maximum tolerated doses to kill as many cancer cells as possible. The inevitable appearance of resistance raises the possibility of shifting treatment goals from complete tumor eradication to long-term disease control. The latter approach is employed by adaptive therapy, which aims to inhibit the evolutionary dynamics governing the spread of resistant tumor phenotypes. Adaptive therapy changes focus from the cancer cells that are responsive to therapy to those that are resistant and ultimately govern outcome. This therapeutic approach retains a pool of sensitive cancer cells to compete with the therapy-resistant ones through dynamic dose modulation and/or timing. Thus, fluctuations of treatment-sensitive cells are used to control the resistant population and prolong tumor control with existing therapy agents. Here, we explore non-genetic mechanisms of resistance, including the protective role of the tumor stroma, the epithelial-to-mesenchymal transition, the overexpression of drug efflux pumps, and the extracellular vesicle-mediated transfer of them. These mechanisms can increase the size of the resistant population at the expense of the sensitive one, reducing the ability of adaptive therapy to force tumor evolution into controllable cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a73ca4c852453e99e4af1b8ebd8a14a56d16cbe0" target='_blank'>
              Sensitivity to ATR–CHK1 pathway inhibition in AML/MDS is enhanced by SRSF2 mutations and reduced by RUNX1 loss
              </a>
            </td>
          <td>
            S. Eldfors, Sumit Rai, Vineet Sharma, Tareq Hossan, Claudia Cabrera Pastrana, Amy Bertino, Angelique Gilbert, K. Porkka, Matthew Walter, Timothy A. Graubert
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="In Mycobacterium tuberculosis, a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            Anna G Green, Mahbuba Tasmin, R. Vargas, M. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Multiplatform mutational and gene expression profiling complemented with proteomic and metabolomic spatial mapping were used on the whole‐organ scale to identify the molecular profile of bladder cancer evolution from field effects. Analysis of the mutational landscape identified three types of mutations, referred to as α, β, and γ. Time modeling of the mutations revealed that carcinogenesis may span 30 years and can be divided into dormant and progressive phases. The α mutations developed in the dormant phase. The progressive phase lasted 5 years and was signified by expanding β mutations, but it was driven to invasive cancer by γ mutations. The mutational landscape emerged on a background of disorganized urothelial differentiation, activated epithelial‐mesenchymal transition, and enhanced immune infiltration with T‐cell exhaustion. Complex dysregulation of mitochondrial energy metabolism with downregulation of oxidative phosphorylation emerged as the leading mechanism driving the progression of mucosal field effects to invasive cancer. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8262a5c744425164951210468902345eaeccb98d" target='_blank'>
              Dysregulated mitochondrial energy metabolism drives the progression of mucosal field effects to invasive bladder cancer
              </a>
            </td>
          <td>
            Sangkyou Lee, Sung Yun Jung, P. Kuś, J. Bondaruk, J. Lee, R. Jaksik, N. Putluri, K. Dinh, David Cogdell, Huiqin Chen, Yishan Wang, Jiansong Chen, N. Navai, C. Dinney, Cathy L. Mendelsohn, David McConkey, Richard R. Behringer, Charles C Guo, Peng Wei, Marek Kimmel, Bogdan A. Czerniak
          </td>
          <td>2025-09-25</td>
          <td>The Journal of Pathology</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The treatment of infections by Acinetobacter baumannii, a clinically significant nosocomial gram-negative bacterial pathogen, is hampered by antibiotic resistance, which is exacerbated by its exceptional genetic plasticity. Knowledge of the dynamics and mechanisms underlying the acquisition of antibiotic resistance is essential for the proper stewardship of their utilization. Here, we used a continuous culture device (morbidostat) to characterize the evolutionary trajectories of two A. baumannii strains in response to the increasing pressure of last-resort drugs, tigecycline and colistin. This approach allows us to confidently and comprehensively map resistance-driving mutations while circumventing both the “driver vs passenger” uncertainty and “selection bottleneck” limitations characteristic of clinical isolate analysis and conventional laboratory evolution, respectively. Tigecycline resistance predominantly occurred through the combination of missense mutations in the adeSR two-component system and disruptive events in the S-adenosyl methionine (SAM)-dependent methyltransferase, trm, while colistin resistance predominantly occurred through missense mutations in the gene cluster responsible for lipid A phosphoethanolamine modification, pmrCAB. Mapping of these mutational events over numerous publicly available A. baumannii genomes identified a relatively low prevalence of resistance to these two drugs. This work represents an initial step toward predictive resistomics of A. baumannii, leveraging gene-level genomic variations in addition to the conventional approaches based on the presence or absence of antibiotic resistance genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6157696474e55a2be83838d634d994f26454b5" target='_blank'>
              Mutations in two-component signaling systems drive experimental evolution of tigecycline and colistin resistance in Acinetobacter baumannii
              </a>
            </td>
          <td>
            J. E. Kent, M. Elane, S. Leyn, J. Zlamal, N. Wong, M. Aizin, C. Zampaloni, S. Louvel, A. Haldimann, M. Vercruysse, A. Osterman
          </td>
          <td>2025-10-29</td>
          <td>Antimicrobial Agents and Chemotherapy</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            S. Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df66d9d10b79290100e939f48a6a4eda6aef615a" target='_blank'>
              NR0B1 alters the 9-cis-retinoic acid response in Ewing Sarcoma cells
              </a>
            </td>
          <td>
            Benjamin D. Sunkel, Meng Wang, Rachel D Dreher, M. Sammons, Lindsay Ryan, A. Gross, Scott Friedland, Ryan D. Roberts, Emily R. Theisen, Elaine R. Mardis, Richard K. Wilson
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb9b509dc6cd6165a216f7f2331a61922945369" target='_blank'>
              Pooled CRISPR screens identify genes and non-coding genomic regions that regulate red blood cell density
              </a>
            </td>
          <td>
            Nicolas Brosseau, Thomas Pincez, Ken Sin Lo, M. Beaudoin, Guillaume Lettre
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Shillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane E. Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, A. Stergachis, Rachel J. O’Neill, K. Miga, Steven L. Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d88265df3748494692babcea1f6ced027e33fc6b" target='_blank'>
              Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver
              </a>
            </td>
          <td>
            Rémi Tellier, Marie Fernandes, Audrey Vinchent, Sonia Paget, Agathe Laratte, Céline Vuillier, Elisabeth Werkmeister, C. Villenet, Jean-Pascal Meneboo, A. Chotteau-Lelièvre, Eric Wasielewski, Z. Kherrouche, Marie-José Truong, C. Descarpentries, Luca Grumolato, M. Figeac, Alexis B. Cortot, Laurent Poulain, Andréa Paradisi, David Tulasne
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Glioma, the most common primary brain tumor, shows poor prognosis and resistance to first-line therapies, like temozolomide (TMZ), lomustine (LOM) and panobinostat-marizomib (PM), complicating treatment and diminishing disease-free survival. It is critical to investigate the molecular mechanisms that underlie chemoresistance to develop effective therapies. This study examines common transcriptional networks that facilitate chemoresistance to identify novel therapeutic targets that may overcome resistance.



 The pediatric high-grade glioma KNS42 cell line was chronically treated with increasing doses of TMZ, LOM, or PM to induce drug resistance (RES); control cells received vehicle (DMSO) treatment. Bulk-RNA sequencing was performed followed by differential gene expression analyses (PartekFlow-ANOVA/DESeq2) followed by pathway and comparison analysis (QIAGEN IPA, |FDR|<0.05).



 RNA-sequencing analysis followed by comparison-analysis revealed differential gene expression in treatment-resistant glioma cells as compared to control cells, including 3,839 differentially expressed genes (DEGs) in LOM RES, 3,641 in TMZ RES, and 3,445 in PM RES groups compared to control group. Among these, 374, 417, and 360 DEGs were downregulated in LOM, TMZ, and PM RES cells, respectively, while 265 (LOM and TMZ RES) and 216 (PM RES) DEGs were upregulated (log2FC > 2; p < 0.05). Interestingly, when the top 10 upregulated genes from each resistant group were compared, six genes (VCAN, FOXA1, SOX2, PDGFRA, ERBB4, and COL4A) were found to be commonly upregulated in all 3 resistant groups. We further confirmed that SOX2, PDGFRA, and ERBB4 are upregulated at the protein level. Ongoing studies will test whether these 6 DEGs are required for mediating resistance against diverse drugs.



 RNA-sequencing identified common genes linked to TMZ, LOM, and PM chemoresistance, revealing key genetic networks involved in acquiring and maintaining drug resistance. Targeting these pathways, along with epigenetic silencing and/or combinatorial therapies, may resensitize chemoresistant cells and allow for effective treatment of glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5824551e4b1f9c0b4ac4e374b69dbac7316d823" target='_blank'>
              DDDR-78. Identifying common transcriptional mechanisms in treatment-resistant glioma
              </a>
            </td>
          <td>
            Namratha B Rao, Aaron R Johnson, Subrahmanya D. Vallabhapurapu, Tracy A Miller, Ian F Pollack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The mismatch repair system is critical for correcting base–base mismatches and insertion-deletion loops during DNA replication. Deficiencies in MMR (due to mutations in MLH1, MSH2, MSH6, or PMS2) lead to microsatellite instability and contribute to the development of various cancers, such as Lynch syndrome-related colorectal cancer and sporadic tumors. This review will delve into the molecular basis of MMR deficiency. Additionally, the review will cover diagnostic approaches for detecting MSI and MMR deficiency, including next-generation sequencing and PCR-based methods. The implications for treatment will be discussed, emphasizing immune checkpoint inhibitors (e.g., pembrolizumab) that target tumors with high mutational burdens due to MMR deficiency, as well as novel therapeutic approaches like synthetic lethality exploiting DNA repair vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c3fe776b0060c3cc36839d638f2791e2a3b03a" target='_blank'>
              Mismatch Repair Deficiency and the Role of Non-Canonical Functions in Cancer: Diagnosis and Therapeutic Implications
              </a>
            </td>
          <td>
            Alicja Dąbrowska, Jakub Mastalerz, Zofia Łapińska, I. Deszcz, Agnieszka Chwiłkowska, Nina Rembiałkowska
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR, in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer. Prostate cancer (PCa) evaluation remains challenging due to its heterogenous and multifocal nature. Here, the authors reconstruct the nuclear chromatin compartmentalization of PCa patients-derived biopsies distinguishing two epigenetic subtypes and deriving an 18-gene signature with prognostic value on TCGA samples that is further validated with multiple independent PCa cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transcriptional regulatory elements (TREs) orchestrate gene expression programs fundamental to cellular identity and transitions between physiological and pathological states. Decoding the regulatory logic of human biology requires resolving where, when, and how these elements are transcriptionally engaged. Here, we profiled the active transcriptional regulatory landscape across all major organ systems and a broad spectrum of developmental and disease states using PRO-cap, a high-resolution method that captures nascent transcription start sites with unprecedented sensitivity and specificity. This atlas of active TREs highlights elements shaped by their cellular contexts and evolutionary constraints, sheds light on the genetic architecture of human traits and diseases, and reveals how patterns of transcription initiation and pausing encode regulatory logic. In cancer, nascent transcription enables the delineation of lineage-specific regulatory states, metastatic adaptations, and the co-option of pre-existing programs. Together, these findings establish nascent transcription as a core dimension of gene regulation, illuminating principles that govern development, physiology, and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa390b89a37848a8f83c759acd4c4c93e92f26e" target='_blank'>
              The regulatory landscape of nascent transcription in human health and disease
              </a>
            </td>
          <td>
            Sagar R. Shah, You Chen, A. K. Leung, Priscila V. Castilho Navarro, Mauricio I. Paramo, Juhi Gupta, Aishwarya Gurumurthy, Rebuma Firdessa Fite, Annika K. Weimer, Qian Du, Ahmed M. Mohyeldin, Dieter Egli, Remi J. Creusot, Russell J H Ryan, Michael P. Snyder, Andrew G. Clark, John T. Lis, Haiyuan Yu
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b535784b00dc4f23ce87d1f10a7f009abae40d" target='_blank'>
              Extensive and differential platinum chemotherapy mutagenesis in children
              </a>
            </td>
          <td>
            Anna Wenger, H. Lee-Six, Manas Dave, M. Layeghifard, A. Lawson, Federico Abascal, Pantelis A. Nicola, T. D. Treger, Toochi Ogbonnah, Conor Parks, Thomas R. W. Oliver, Jonathan Kennedy, A. Hodder, Nathaniel D. Anderson, Felipe Luz Torres Silva, Mi K. Trinh, Thomas Dowe, Marwo Habarwaa, James J. Sun, Sergio Assia-Zamora, M. Cortes-Cerisuelo, W. Jassem, Charlotte Town, Anil Dhawan, Vandana Jain, Karin Straathof, M. Deheragoda, I. Martincorena, Liina Palm, J. C. Hutchinson, Tim H H Coorens, Claire Trayers, Nigel Heaton, A. Shlien, Yoh Zen, Foad J. Rouhani, Sam Behjati
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Environmental exposures during early life are increasingly recognized as key determinants of health and disease in adulthood (1) but how they durably shape disease risk across the lifecourse remains poorly understood. Here, we show early-life toxicants reprogram the epigenome and redirect age-associated transcriptional trajectories—polarizing cell-specific gene expression and predisposing to liver disease. The TaRGET II Consortium (2) exposed mice to diverse toxicants from pre-conception through weaning and followed individual animals though adulthood with multi-omic profiling. Analysis of >800 liver epigenomic and transcriptomic profiles from male and female mice revealed that despite differing chemical classes and mechanisms of action, multiple toxicants — BPA, TBT, TCDD, and PM2.5 — produced exposure signatures that converged on genes normally differentially expressed in the liver as animals aged. Histone modifications at enhancers emerged as key targets for epigenomic reprogramming of these liver aging-associated plasticity genes (LAAsP genes). Reprogrammed LAAsP genes exhibited a striking, bidirectional signature. In hepatocytes LAAsP genes that typically increase with age, such as those involved in metabolism, were repressed. Conversely, in non-parenchymal cells, LAAsP genes that normally decline with age, including those for extracellular matrix production, remained elevated. An attenuated LAAsP gene signature and polarized transcriptional states were mirrored in human liver disease and hepatocellular carcinoma and could effectively distinguished healthy from diseased human liver transcriptomes. Together, these findings demonstrate that early-life environmental exposures can hijack the plasticity of epigenomic aging, durably reprogram expression trajectories, and lock in polarized states that foreshadow chronic liver disease and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e50202ac8b33f8ab48ea3ab4ef30e279c45916f2" target='_blank'>
              Plasticity of Epigenomic and Transcriptomic Aging Reveals Common Targets for Reprogramming by Environmental Exposures
              </a>
            </td>
          <td>
            Cheryl Walker, Sandra L. Grimm, Rahul K. Jangid, M. Bartolomei, Dana C. Dolinoy, D. Aylor, Gokhan Mutlu, Shyam Biswal, Bo A. Zhang, Robert Hamanaka, Justin A. Colacino, Maureen Sartor, Laurie Svoboda, Ting Wang, Cristian Coarfa
          </td>
          <td>2025-09-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains a clinical challenge with suboptimal long-term survival. While circular RNAs derived from the RNF220 host gene have been implicated in AML pathogenesis, the functional role and regulatory mechanisms of RNF220 itself in AML are poorly understood. Methods We integrated bioinformatics analyses of public databases (TCGA-LAML, TARGET-LAML) and local cohort with in vitro functional assays. RNF220 was knocked down and overexpressed in AML cell lines using lentivirus. Transcriptomic profiling (RNA-seq), metabolic pathway enrichment (GSVA, GSEA), and immune microenvironment deconvolution (xCELL, CIBERSORT, MCP-counter) were performed. Transcription factor binding sites were predicted across five databases (JASPAR, ENCODE, GTRD, etc.). Validation of transcriptional regulation was performed using ChIP-PCR and luciferase reporter assays. Results RNF220 overexpression correlated with poor prognosis in AML, drove an immunosuppressive microenvironment characterized by reduced CD8+ T cells, and inhibited NK activity and M2 polarization of macrophage. RNF220 promoted tumor proliferation by suppressing apoptosis and preventing G1 arrest. Knockdown of RNF220 dysregulated metabolic pathways, notably suppressing glycolysis and phenylalanine metabolism. Mechanistically, FOXA1 was identified as an upstream negative regulator of RNF220, where high FOXA1 predicted favorable survival and inversely correlated with RNF220-associated metabolic reprogramming. Conclusion NF220 acts as an oncogenic ubiquitin ligase in AML by coordinating dual pro-leukemic mechanisms: cell-intrinsic metabolic rewiring (glycolysis/phenylalanine) and immune evasion via microenvironment suppression. Targeting the FOXA1–RNF220 axis may offer novel therapeutic strategies for high-risk AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ff73f3a35554c15311058c2d83eb2628a0504d" target='_blank'>
              Dual oncogenic role of RNF220 in AML: linking metabolic rewiring to cell proliferation and immune evasion
              </a>
            </td>
          <td>
            Bi-xia Li, Shi Jiang, Yao Xu, Xiao Yan, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, D. D. Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e33fb7e42f73e65912640c7f54217de318d7753d" target='_blank'>
              Base-editing a single missense mutation in A20 enhances CAR-T cell efficacy
              </a>
            </td>
          <td>
            Adam Blaisdell, Stefanie Bachl, Luis R. Sandoval, Carter Ching, C. Bowman, Nupura Kale, Manu Prabandham, Morgan Diolaiti, Claire Havig, Rommel Advincula, Nika Lenci, Zhongmei Li, Emily Yamashita, Charlotte H. Wang, Shimin Zhang, Qi Liu, Philip Achacoso, Dorothea Stibor, Inger Øynebråten, Jin Seo, Alan Ashworth, Alex Marson, C. Ye, B. Malynn, Justin Eyquem, J. Carnevale, Averil Ma
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genome instability, tumour-promoting inflammation, and immune escape are three distinct hallmarks of cancer. However, accumulating scientific and clinical evidence over the past decade have uncovered a multifaceted interplay of complex dynamic network of interactions between genome instability, the DNA damage response (DDR), and tumour immunogenicity. Fuelled by the clinical successes of immune checkpoint blockers (ICB), growing interest for immuno-oncology and recent cancer biology discoveries have allowed a better understanding of the underlying biology and clinical opportunities brought by this interplay—which is yet, still only in its infancy. The cooperative nature of tumour cell-intrinsic and -extrinsic mechanisms involved suggests that harnessing genomic instability in cancer does not only hamper cancer cells fitness but also stimulate the anti-tumour immune response, thereby paving the way to the development of DDR-based immunomodulatory therapeutic strategies applicable to a variety of molecular and histological cancer types. Here, we review the various aspects of this crosstalk between genome instability and tumour immunogenicity, including feedforward and feedback mechanisms affecting either side of this interplay, as well as the specific consequences of chromosomal instability. We further discuss emerging DDR-based predictive biomarkers of response to ICB therapies, and finally examine the latest clinical developments of therapeutic combinations that exploit the DDR–immunity interplay in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd59247f293c0351f5d4a4afa84e86e0d39c9e9" target='_blank'>
              Genome instability and crosstalk with the immune response
              </a>
            </td>
          <td>
            Roman M. Chabanon, F. Danlos, Kaïssa Ouali, Sophie Postel-Vinay
          </td>
          <td>2025-11-04</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Most mutant adenomatous polyposis coli (APC) gene produced truncated APC protein (Trunc-APC), which has been shown to function as an oncogene in colorectal cancer (CRC) pathogenesis; however, its role in modulating innate immune responses within tumor cells remains unexplored. We utilized CRISPR-Cas9 to knockout mutant APC and performed transcriptome sequencing across multiple CRC cell lines to investigate the immunomodulatory function of Trunc-APC. Subcellular fractionation, proteinase K protection assays, and immunofluorescence were employed to determine Trunc-APC subcellular localization. Protein interaction studies, ubiquitination assays, and aggregation analyses were conducted to elucidate Trunc-APC binding to MAVS and its impact on MAVS ubiquitination and RIG-I association. The effects of Trunc-APC deletion, alone or in combination with 5-azacytidine and trichostatin A, were evaluated on type I interferon activation, apoptosis, and tumor growth both in vitro and in vivo. We found that Trunc-APC partially localizes to the mitochondrial outer membrane and attenuates type I interferon signaling by binding to MAVS, suppressing its K63-linked polyubiquitination, and disrupting MAVS-RIG-I interactions. Deletion of Trunc-APC, particularly when combined with 5-azacytidine and trichostatin A, enhanced innate immune activation, promoted tumor cell apoptosis, and significantly inhibited CRC tumor growth both in vitro and in vivo. Our study reveals a previously unrecognized role of Trunc-APC in dampening tumor-intrinsic innate immunity and suggests that co-targeting Trunc-APC with epigenetic therapy may offer a promising strategy to enhance anti-tumor immune responses in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0e1308090a0906847a31cc6ff0f2170860b1e8" target='_blank'>
              Truncated APC impairs innate immune response by targeting MAVS on mitochondria in colorectal cancer
              </a>
            </td>
          <td>
            Si Li, Xin-Yi Wang, Jing Wang, Jinghua Cao, Yong-rui Lv, Dan Xie, Fengwei Wang
          </td>
          <td>2025-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e643a6feb31e1193c4fe51251e45e7c7b63296ed" target='_blank'>
              Cross-Species Transcriptomic Integration Reveals a Conserved, MIRO1-Mediated Macrophage-to-T Cell Signaling Axis Driving Immunosuppression in Glioma
              </a>
            </td>
          <td>
            Zehui Du, Menghan Li, Brandon Bergsneider, Andy P. Tsai, Kwang Bog Cho, Lily H Kim, John Choi, Gordon Li, T. Wyss-Coray, Michael Lim, Xinnan Wang
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Purpose FOS-like antigen 1 (FRA1), encoded by FOSL1, is an inducible subunit of the AP-1 transcription factor complex and regulates gene expression in response to proliferative and environmental cues. Although FRA1 has been linked to cancer progression, its role in early transformation and radiation responses remains unclear. Methods CRISPR-engineered human CGL1 cells—a hybrid of HeLa and normal fibroblasts—were used to evaluate the impact of FRA1 overexpression and knockout on neoplastic transformation. Transformation frequency, clonogenic survival, DNA damage recognition and repair, and cell cycle distribution were assessed following irradiation. Transcriptomic profiling was performed under baseline and serum-stimulated conditions. Results FRA1 loss markedly increased both spontaneous and radiation-induced transformation frequency, while overexpression suppressed transformation under both conditions. FRA1-deficient cells were sensitized to radiation-induced cell killing, despite intact DNA damage recognition and repair. In contrast, FRA1 overexpression promoted G2/M accumulation post-irradiation, suggesting enhanced checkpoint activation. Transcriptomic profiling revealed that FRA1 remodels AP-1 complex composition and functions as a transcriptional repressor of mitogen- and stress-responsive genes. FRA1-mediated repression was observed across gene networks involved in extracellular matrix remodeling, hypoxia signaling, inflammation, and proliferation, under both baseline and serum-stimulated conditions. Conclusion These findings establish FRA1 as a key modulator of neoplastic transformation and radiation response, acting primarily through transcriptional repression of pro-tumorigenic signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb780bfa4e21042a750aefe2ef72a09f6e8a78f" target='_blank'>
              FRA1 (FOSL1) suppresses neoplastic transformation and modulates radiation responses via transcriptional control of mitogenic and stress-responsive networks
              </a>
            </td>
          <td>
            W. Al-Khayyat, T. Laframboise, Jessica Dougherty, Marc S. Mendonca, D. Boreham, T. C. Tai, Christopher Thome, S. Tharmalingam
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PARP inhibitors exploit synthetic lethality in BRCA1/2-mutated ovarian cancers but are limited by emerging therapeutic resistance. Therefore, novel biomarkers predicting PARP inhibitor response are urgently needed. In this study, we performed integrative analysis using drug sensitivity, patient survival, gene dependency, and expression data to identify biomarkers associated with PARP inhibitor response in ovarian cancer. Mutations in BRCA1, MLL2, NF1, and SMARCA4 were associated with increased sensitivity to PARP inhibitors, suggesting potential synthetic lethality with PARP1. In contrast, SMAD4 mutations were linked to PARP inhibitor resistance, and low SMAD4 expression was associated with poor overall survival in patients with ovarian cancer. Further gene dependency score (GDS)-based screening revealed 51 candidate genes potentially involved in SMAD4-mediated resistance. Functional enrichment revealed associations with stress response, tumor-associated signaling pathways, and additional processes. Subsequent correlation and survival analyses nominated ACACA, PRPF4B, and TUBD1 as potential therapeutic targets. Notably, low ACACA expression in patients with low SMAD4 expression was associated with improved survival, indicating its relevance in overcoming PARP inhibitor resistance. This study contributes to predicting clinical outcomes in ovarian cancer and developing personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80b49672c28e84e41d8bee90691985b721fb554" target='_blank'>
              Identification of Anticancer Targets in Ovarian Cancer Using Genomic Drug Sensitivity Data
              </a>
            </td>
          <td>
            Yebin Son, Jae Yong Ryu
          </td>
          <td>2025-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f1059bdaf05d7d3243514db8119194d9b788af" target='_blank'>
              In host mutational adaptation of Mycobacterium tuberculosis complex strains
              </a>
            </td>
          <td>
            Helen Zhang, Naomi Medina-Jaudes, Alicia Forcada-Nadal, E. Harrison, Francesc Coll
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Simple Summary Cancer develops through a complex series of genetic and molecular changes. Modern sequencing technologies now allow scientists and physicians to study these changes in great detail, helping them understand why each person’s cancer behaves differently. By examining the entire genome, exome, or transcriptome of a tumor, researchers can uncover genetic alterations that guide personalized treatment decisions. This article explains how these sequencing methods, together with data analysis tools and artificial intelligence, help identify the best targeted and immune-based therapies for different cancer types. By linking genetic findings to approved drugs, the study aims to make precision cancer care more effective and accessible for patients while also guiding future research directions in oncology. Abstract The rapid evolution of sequencing technologies has profoundly advanced precision oncology. Whole-exome sequencing (WES), whole-genome sequencing (WGS), and whole-transcriptome sequencing (RNA-Seq) enable comprehensive characterization of tumor biology by detecting actionable mutations, gene fusions, splice variants, copy number alterations, and pathway dysregulation. These approaches also provide critical insights into biomarkers such as homologous recombination deficiency (HRD), tumor mutational burden (TMB), and microsatellite instability (MSI), which are increasingly essential for guiding therapeutic decisions. Importantly, comprehensive genomic profiling not only refines patient stratification for targeted therapies but also sheds light on tumor–immune interactions and the tumor microenvironment, paving the way for more effective immunotherapeutic combinations. WGS is considered the gold standard for detecting germline mutations and complex structural variants, while WES remains central for detecting somatic driver mutations that guide targeted therapies. RNA-Seq complements these methods by capturing gene expression dynamics, identifying clinically relevant fusions, and revealing mechanisms of resistance. Together with advances in bioinformatics and artificial intelligence, these tools translate molecular data into actionable strategies for patient care. This review integrates insights from WGS, WES, and RNA-Seq with an overview of FDA- and EMA-approved targeted therapies, organized by tumor type, and highlights the molecular signaling pathways that drive cancer development and treatment. By bridging genomic profiling with regulatory-approved therapies, we outline current advances and future perspectives in delivering personalized cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dfbd87c701593d0e68f1b2d6aa4002ef2eff85" target='_blank'>
              Advances in Precision Oncology: From Molecular Profiling to Regulatory-Approved Targeted Therapies
              </a>
            </td>
          <td>
            P. Brlek, V. Škaro, Nenad Hrvatin, Luka Bulić, Ana Petrović, Petar Projić, Martina Smolić, Parth Shah, D. Primorac
          </td>
          <td>2025-10-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600bc28120bc59b1b37b42c37c760d59334a28c6" target='_blank'>
              CRISPR/Cas9-mediated generation of a de novo C9ORF72 knock-in isogenic iPSC cell bank to model ALS and FTD
              </a>
            </td>
          <td>
            Niamh L. O’Brien, Erin C. Hedges, R. Nair, AJ Cammack, M. Carcolé, Deniz Vaizoglu, Sara Tacconelli, Katie Sutherland, Chloe L. Fisher-Ward, A. R. Awan, Simon Topp, A. Iacoangeli, A. Isaacs, Thomas J. Cunningham, M. Ruepp, Sarah Mizielinska
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) remains the most aggressive primary brain tumour in adults, marked by pronounced cellular heterogeneity, diffuse infiltration, and resistance to conventional treatment. In recent years, transcriptomic profiling has provided valuable insights into the molecular mechanisms that govern the progression of glioblastoma. This systematic review aims to synthesise the current literature on dysregulated gene expression in GBM, focusing on gene signatures associated with stemness, immune modulation, extracellular matrix remodelling, metabolic adaptation, and therapeutic resistance. Methods We conducted a systematic search of PubMed, The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and the GlioVis portal for studies published between January 2005 and April 2025, limited to English-language reports. Studies were eligible if they included adult glioblastoma tissue or patient-derived datasets and reported gene-level expression or clinical associations. Reviews, commentaries, and studies on non-GBM gliomas were excluded. Screening followed the PRISMA 2020 checklist, with 410 records initially identified, 90 duplicates removed, and 125 studies retained after full-text review. Data were synthesised descriptively, and findings were validated against TCGA/CGGA expression datasets to ensure consistency across cohorts. Results We categorised recurrently dysregulated genes by their biological function, including transcription factors (SOX2, ZEB2), growth factor receptors (EGFR, PDGFRA), immune-related markers (PD-L1, TAP1, B2M), extracellular matrix regulators (MMP2, LAMC1, HAS2), and metabolic genes (SLC7A11, PRMT5, NRF2). For each group, we examine the functional consequences of transcriptional alterations and their role in driving key glioblastoma phenotypes, including angiogenesis, immunosuppression, invasiveness, and recurrence. Conclusion We further discuss the prognostic implications of these gene signatures and evaluate their potential utility in precision medicine, including current clinical trials that target molecular pathways identified through transcriptomic data. This review highlights the power of gene expression profiling to stratify glioblastoma subtypes and improve personalised therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f178b970f37a5487e44c1e368727e8a6a513f43" target='_blank'>
              RNA Expression Signatures in Glioblastoma: A Systematic Review of Tumour Biology and Therapeutic Targets
              </a>
            </td>
          <td>
            Amber Hassan, Badr Hafiz, Taghreed A. Alsinani, Rakan Bokhari, D. Mirdad, Awab T. Tayyib, Alaa Alkhotani, Ahmad Fallata, Iman Mirza, Eyad Faizo, Saleh Baeesa, Huda Alghefari, Maher Kurdi
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="This study investigates the expression and functional significance of mitochondrial ribosomal protein S12 (MRPS12) within NSCLC pathogenesis. The Cancer Genome Atlas dataset reveals a significant association between MRPS12 overexpression and unfavorable clinical outcomes in NSCLC patients. Single-cell RNA sequencing data unequivocally confirm MRPS12 overexpression across diverse NSCLC cancer cell populations. MRPS12 is also upregulated in locally-treated NSCLC tissues. shRNA-mediated gene silencing or CRISPR-Cas9-mediated gene knockout (KO) of MRPS12 impaired mitochondrial function, resulting in a reduction in oxygen consumption rate, marked declines of complex I activity and ATP levels, mitochondrial membrane depolarization, and an increase in reactive oxygen species production in NSCLC cells. Moreover, MRPS12 silencing or knockout attenuated cell viability, proliferation, and migratory capacity, while inducing apoptotic cell death in various NSCLC cell types. Conversely, ectopic MRPS12 overexpression conferred a pro-tumorigenic phenotype to NSCLC cells. In vivo studies demonstrated that MRPS12 silencing markedly inhibited the growth of subcutaneous xenografts derived from primary NSCLC cells in nude mice. The MRPS12-silenced NSCLC xenografts exhibited decreased ATP levels, elevated oxidative injury, diminished proliferation, inhibited Akt-mTOR activation and an increase in apoptosis. These findings support the important role of MRPS12 in supporting mitochondrial function and driving the malignant progression of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83f9722c929e54c00c45abc4f6d59820f25b7d72" target='_blank'>
              Exploring mitochondrial ribosomal protein S12 as a novel target for non-small cell lung cancer
              </a>
            </td>
          <td>
            Zihao Chen, Yu Jin, Shiyou Wei, Rongqiang Wei, Xinyu Ding
          </td>
          <td>2025-11-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, Céline Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Cancer cells exhibit reprogrammed metabolic pathways to sustain aggressive phenotypes, including continuous cell division, stemness, invasion, and metastasis. Emerging evidence suggests that these metabolic adaptations profoundly impact DNA repair pathways, which contribute to the responses to therapy and influence overall outcomes. Metabolic processes such as the Warburg effect, nicotinamide adenine dinucleotide (NAD) metabolism, glutamine metabolism, and one-carbon metabolism support DNA repair by expanding the metabolite pool and facilitating post-translational modifications. Conversely, oncometabolites impair DNA repair pathways through epigenetic reprogramming, thereby promoting genomic instability. This review highlights recent discoveries that elucidate the intricate connections between metabolic hallmarks in cancer cells and DNA repair mechanisms, offering insights into potential therapeutic targets for future cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0683240e558e0cfd022d73899ecd3cc85f65ee47" target='_blank'>
              Cancer Metabolism Meets DNA Repair: The Hidden Link to Therapy Resistance.
              </a>
            </td>
          <td>
            Chunzhang Yang
          </td>
          <td>2025-09-28</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Colon cancer is a common disease, treated with few chemotherapeutic agents with similar treatment sequencing despite its heterogeneity. A significant proportion of patients are diagnosed with metastasis, and resistance to antineoplastic drugs is associated with disease progression and therapeutic failure. It is known that the tumor microenvironment plays an essential role in cancer progression, contributing to processes that may be associated with therapeutic resistance mechanisms in colon cancer. In this study, we aim to identify a gene expression signature and its relationship with immune cell infiltration in colon cancer, contributing to the identification of potential resistance biomarkers. Methods An in silico study was conducted using RNA-seq data from The Cancer Genome Atlas Program (TCGA) samples, subdivided into two groups (treatment-resistant and non-resistant), taking into account the molecular subgroups (CMS1, CMS2, CMS3, and CMS4). The following algorithms were used: i. Limma was applied to identify differentially expressed genes; ii. WGCNA was applied to construct co-expression networks; iii. CIBERSORT was applied to estimate the proportion of infiltrating immune cells; and iv. TIMER was applied to explore the relationship between core genes and immune cell content. Results Twenty differentially expressed genes (DEGs) were found, with 18 related to the group considered resistant to oncologic treatment and presenting poorer overall survival. T CD4 memory resting cells and M0 and M2 macrophages were found in more significant proportions in the analyzed samples and more infiltrated in the tumor microenvironment, the higher the expression of some of these resistance DEGs. Additionally, these genes correlate with biological aspects of neuronal differentiation, axogenesis, and synaptic transmission. Conclusion The gene expression signature suggests the presence of differentially expressed synaptic membrane genes, which may be involved in neuronal pathways that influence the tumor microenvironment, potentially serving as future biomarkers. Furthermore, the presence of M0 and M2 macrophages and T CD4 memory resting cells suggests a potential interaction that may play a role in therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d04147fc5526a1e2f859a73c8cdc7c99f7a9f" target='_blank'>
              Gene expression profile in colon cancer therapeutic resistance and its relationship with the tumor microenvironment
              </a>
            </td>
          <td>
            P. Doria, G. Rocha, Vanessa Dybal Bertoni, Roberto de Souza Batista dos Santos, Mariana Araújo-Pereira, Clarissa A Gurgel
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a genetically and clonally heterogeneous blood malignancy characterized by somatic gene mutations influencing disease biology. Despite advances in genomic analyses, splicing alterations associated with AML mutations remain underexplored. In this study, we systematically analyzed 914 single nucleotide variants previously found in 1540 AML patients and integrated DNA and RNA sequencing data from an additional 639 cases obtained from the BeatAML and TCGA-LAML cohorts. We identified 125 somatic variants in 32 genes with predicted splicing impact, including traditionally categorized missense/nonsense mutations. Among the 36 variants tested experimentally, 17 were validated through RNA-seq and/or minigene assays, with the latter providing support when RNA-seq evidence was limited. Validated splicing-associated variants (SAVs) frequently induced exon skipping or cryptic splice site activation in AML-associated genes such as EZH2, FLT3, ASXL1, IDH1, and U2AF1. In several cases, these splicing changes altered conserved protein domains or were predicted to trigger nonsense-mediated decay. Our findings reveal a previously underappreciated contribution of cis-acting SAVs to AML pathogenesis, highlighting the need for integrative prediction–validation strategies to uncover their functional and potential therapeutic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b18e9c22a27efbea5c26d4eafcb5e36e040af33" target='_blank'>
              Unveiling splicing disruptions due to common somatic variants in acute myeloid leukemia
              </a>
            </td>
          <td>
            Mireya Morote-Faubel, María Guaita-Céspedes, Beatriz Fernández-Blanco, C. Martínez-Valiente, Mariam Ibáñez, Marta Santiago-Balsera, E. Such, Javier de la Rubia, A. Liquori, José Cervera
          </td>
          <td>2025-11-04</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background & Objective: Gastric cancer (GC) remains one of the most prevalent and lethal malignancies worldwide, with cisplatin serving as a cornerstone in its chemotherapeutic regimen. However, the frequent and often rapid development of cisplatin resistance significantly compromises therapeutic efficacy. MicroRNAs (miRNAs) are key post-transcriptional regulators that modulate critical cellular mechanisms underlying chemoresistance, including apoptosis, DNA repair, drug efflux, and proteostasis. This study aimed to identify candidate miRNAs and molecular pathways associated with cisplatin resistance through an integrated bioinformatics approach. 
Materials & Methods: miRNA expression profiles from the GEO dataset GSE86195, comprising cisplatin-sensitive and cisplatin-resistant GC cell lines, were analyzed. Differential expression analysis was conducted using limma, followed by functional enrichment analysis of validated miRNA targets via clusterProfiler. Family-level aggregation, Weighted Gene Co-expression Network Analysis (WGCNA), and Random Forest feature ranking were subsequently applied to identify potential hub and predictive miRNAs. 
Results: Although no miRNAs survived false discovery rate (FDR) correction, an exploratory nominal p-value threshold of < 0.05 revealed 957 candidate differentially expressed miRNAs (538 upregulated and 416 downregulated). Enrichment analysis indicated the involvement of pathways related to nucleocytoplasmic transport, RNA splicing, ubiquitin-mediated proteolysis, and platinum drug resistance. A coordinated dysregulation of the miR-346, miR-421, and miR-139-5p families was identified. Machine learning further highlighted hsa-let-7e and hsa-miR-20a-star as top-ranked predictive candidates, although both WGCNA and Random Forest findings should be interpreted cautiously due to the limited sample size (n = 4). 
Conclusion: This exploratory bioinformatics analysis identifies candidate miRNAs and signaling pathways that may underlie cisplatin resistance in GC. The findings should be considered hypothesis-generating and warrant validation in larger cohorts (e.g., TCGA) as well as experimental confirmation through functional assays prior to clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9dcf90b8f9e5b34f8d4781f2a3c4043851f82e" target='_blank'>
              An Exploratory Bioinformatics Investigation of microRNA Signatures in Cisplatin- Resistant Gastric Cancer Cell Lines
              </a>
            </td>
          <td>
            Kianoush Mohammadi, R. Safaralizadeh, Asadollah Asadi
          </td>
          <td>2025-09-22</td>
          <td>Journal of Advanced Biomedical Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd02f04b4b0ffce18f7e7041b90b9a413f147da5" target='_blank'>
              Divergent spontaneous antibiotic-resistance evolution confers reciprocal and exploitable collateral sensitivity effects
              </a>
            </td>
          <td>
            A. Liakopoulos, S. T. Tandar, Irene V. Hoogendijk, Efthymia Fragkiskou, Zheren Zhang, Bastienne Vriesendorp, L. Aulin, J. V. van Hasselt, Daniel E. Rozen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b68316c5391afa0d45aa42e226e37099bf84d45" target='_blank'>
              Tissue specificity and chromosomal alterations shape divergent immune programs in HRD tumors
              </a>
            </td>
          <td>
            D. Gulhan, David Barras, Marco Mina, Eleonora Ghisoni, Yoo-Na Kim, V. Viswanadham, Hu Jin, Florian Huber, K. Homicsko, M. Bassani-Sternberg, G. Ciriello, Peter J. Park, Denarda Dangaj Laniti
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks that causes DNA damage and tumor cell death. While these drugs are front-line treatments for multiple human cancers – including high-grade serous ovarian cancer (HGSOC) – platinum resistance and tumor recurrence remains a major clinical challenge. Here, we report findings from both a genome-wide CRISPRi screen and an RNA-binding protein (RBP)-focused CRISPRi screen aimed at identifying modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, as well as to an microtubule-targeting taxane. Using an in vitro treatment regimen designed to mimics clinical treatment protocols, we found that loss of genes in the Fanconi anemia (FA) and homologous recombination (HR) DNA repair pathways markedly increased the killing of p53-deficient HGSOC cells following treatment with cisplatin or oxaliplatin. In contrast, knockdown of genes in the nucleotide excision repair (NER) pathway—despite its known role in repairing platinum-induced intrastrand crosslinks, particularly in G1—had a significantly lesser effect. These results suggest that platinum-induced cytotoxicity in HGSOC is primarily driven by replication stress, which is exacerbated by disruption of error-free DNA repair pathways active during the S- and G2 phases of the cell cycle. Our screens also identified several known and novel RBPs as potent sensitizers to both platinum agents and taxanes. Depletion of these RBPs resulted in substantial activation of interferon signaling, a hallmark of innate immune activation and cellular stress responses. Among them, the RNA-editing enzyme ADAR1 emerged as a key modulator of drug resistance. ADAR1 catalyzes adenosine-to-inosine RNA editing, thereby altering mRNA coding potential, secondary structure, and subcellular localization. Additionally, ADAR1 has been implicated in regulating the integrated stress response (ISR), a signaling network that enables cellular adaptation to environmental stress. As part of this stress adaptation, we observed that platinum and taxol treatment caused a translational shift from canonical cap-dependent translation mediated by eIF4G1 to a cap-independent mechanism reliant on eIF4G2, supporting selective translation of stress-responsive mRNAs. Together, our findings suggest that ADAR1-dependent suppression of interferon signaling and activation of the ISR contributes to resistance to both platinum-based chemotherapy and microtubule-targeting agents, which together comprise the current standard of care for HGSOC.



 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912428b4f14100051b63d5819642408cb6578a72" target='_blank'>
              Abstract PR009: CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Thyroid carcinomas are phenotypically heterogeneous malignancies. Advances in molecular and cellular technologies have revealed genetic, epigenetic, and nongenetic factors underlying this heterogeneity. Our study aimed to assess the impact of single and combined treatments with anticancer agents (Carboplatin, Doxorubicin, Paclitaxel, Avastin), natural compounds (Quercetin), and epigenetic modulators (suberoylanilide hydroxamic acid and 5-Azacytidine) on the expression of long noncoding RNAs, methylation regulators, and functional features in the human thyroid cancer cell line K1. Methods: Treated and untreated K1 cells were used throughout experiments to evaluate the drug-induced cytotoxicity, apoptosis, cell cycle distribution, cytokine release, gene expression, and global DNA methylation levels. Results: Some single- and combined-drug treatments modulated both cell cycle progression and apoptotic events, demonstrating anti-tumor activity of the tested compounds. Gene expression analysis showed treatment-specific regulation of target genes and lncRNAs, including both upregulation and downregulation across different drug combinations. All treatments resulted in increased global DNA methylation levels compared to the untreated controls. Several combinations significantly upregulated DNMT1 and DNMT3B, while concomitantly decreased EZH2 levels. Conclusions: These coordinated epigenetic changes highlight the therapeutic potential of combining epigenetic modulators with chemotherapeutic agents, suggesting a strategy to prevent or reverse treatment resistance and improve outcomes in thyroid cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1eba8f910f8930ce4647663ca6adba36af18" target='_blank'>
              Epigenetic Remodeling in Thyroid Cancer: New Dimensions of Targeted Therapy Through lncRNA Modulation
              </a>
            </td>
          <td>
            A. Albulescu, A. Fudulu, M. Mihăilă, I. Iancu, Adriana Pleșa, M. Bostan, A. Botezatu, L. Brasoveanu, C. Hotnog
          </td>
          <td>2025-10-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d6f72ebd8a3733c60e8ee75cfbcd9b5da5d9c6" target='_blank'>
              Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes
              </a>
            </td>
          <td>
            Zhe Liu, Yihang Bao, An Gu, Weichen Song, Guan Ning Lin
          </td>
          <td>2025-09-26</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3642f23461379e5a0adfe4119834fff5e5e8de7" target='_blank'>
              Pan-cancer analysis reveals context-dependent roles of LINE-1 ORF1p in immune regulation and copy number alterations
              </a>
            </td>
          <td>
            E. Kaparos, Wenjing Zhang, Ryusei Miyanaga, Wilson H. McKerrow, J. Boeke, Teresa Davoli, P. Mita, Kelly Ruggles, David Fenyö
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Regorafenib, a multikinase inhibitor, is widely used to treat hepatocellular carcinoma. However, chemoresistance poses a significant challenge to its long-term efficacy. This study conducted a genome-wide CRISPR/Cas9 knockout screen in liver cancer cell lines to identify key regulators of regorafenib resistance and elucidate the underlying molecular mechanisms. The screen identified ARHGAP35 as a critical negative regulator of regorafenib resistance. ARHGAP35 depletion conferred resistance in HepG2 and Huh7 cells, while regorafenib-resistant variants (HepG2-R and Huh7-R) exhibited decreased ARHGAP35 expression. Reintroducing ARHGAP35 restored drug sensitivity. Further analysis revealed that reduced ARHGAP35 expression facilitated epithelial-mesenchymal transition (EMT) by activating the RhoA signaling pathway. Notably, RhoA inhibition reversed EMT and restored regorafenib sensitivity. These findings highlight ARHGAP35 as a key modulator of regorafenib resistance through RhoA suppression, offering potential therapeutic targets to combat chemoresistance in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b3a4892777e73e2cf92f82b25659fdd83d51c4" target='_blank'>
              A genome-wide screen identified ARHGAP35 as a regulator of regorafenib resistance in liver cancer.
              </a>
            </td>
          <td>
            Kun Chen, Miaomiao Zhang, Yuexin Liu, Zhengnan Dong, Yaoji Liang, Jinzhang Zeng, Jie Liu
          </td>
          <td>2025-10-16</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Genomic instability and the accumulation of DNA damage are hallmarks of cancer, often resulting from defects in DNA repair pathways. While normal cells rely on highly coordinated DNA damage response (DDR) mechanisms to maintain genomic integrity, cancer cells exploit aberrant DDR regulation to sustain uncontrolled proliferation and survival. Despite significant advancements in chemotherapy, targeted therapy, and immunotherapy, the emergence of resistance remains a major challenge in cancer treatment. Small molecule inhibitors targeting key DDR proteins have emerged as promising therapeutic agents, not only as direct anticancer drugs but also as indispensable tools for dissecting the molecular intricacies of DNA repair. Recent therapeutic approaches leverage synthetic lethality and DDR pathway vulnerabilities to selectively eradicate tumor cells while minimizing damage to normal tissues. These inhibitors provide insights into mechanisms of tumor resistance, facilitating the rational design of combination therapies to enhance treatment efficacy. This review examines the latest advancements in DNA repair-targeted therapeutics, with a focus on small molecule inhibitors currently under clinical investigation. Additionally, we discuss emerging strategies for optimizing DDR-targeted interventions, including biomarker-driven patient selection and rational drug combinations. Understanding these molecular interactions will contribute to the development of novel, more effective treatment paradigms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62086e2b8dc746fe8a2545d0353e820f59f91c95" target='_blank'>
              Targeting DNA repair mechanisms in cancer therapy: the role of small molecule DNA repair inhibitors
              </a>
            </td>
          <td>
            Seula Jeong, Y. Chung, Soomin Heo, Kyungjae Myung
          </td>
          <td>2025-11-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ee3d709c3a53cc71a54980118c28a4596ab59b" target='_blank'>
              HyperdCas12a-Based Multiplexed Genetic Regulation in Candida albicans
              </a>
            </td>
          <td>
            Nicholas C. Gervais, R. K. Rogers, Madeleine R. Robin, Rebecca S. Shapiro
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 DHX9, a multi-functional RNA helicase, plays important roles in maintaining genome stability by unwinding various nucleic acid secondary structures that may lead to replication stress and DNA damage if left unresolved. DHX9 is overexpressed in multiple cancers, with selective vulnerability to DHX9 inhibition noted in tumors characterized by genomic instability and high replicative stress. ATX-559 is an orally bioavailable, small-molecule inhibitor of DHX9 helicase activity currently in Phase 1 clinical development. ATX-559 induces robust anti-tumor activity in a variety of preclinical models with genomic instability, including those with microsatellite instability-high (MSI-H) and/or deficient mismatch repair (dMMR), BRCA alterations, and homologous recombination repair deficiency (HRD).



 ATX-559 anti-proliferative activity was tested in a multi-indication cell panel screen. Triple negative breast cancer (TNBC) and colorectal cancer (CRC) cells were used to evaluate ATX-559 mechanistic activity using immunofluorescence imaging, dot blot, and western blot assays. ATX-559 was dosed orally BID to evaluate its anti-tumor activity in CRC and TNBC cell line-derived xenograft (CDX) models, as well as a panel of 64 patient-derived xenograft (PDX) models including TNBC, Ovarian, CRC, endometrial, and gastric cancers. qPCR was used to evaluate the target engagement PD biomarker, circBRIP1.



 ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress, subsequent DNA damage, and cell death selectively in cancer models with genomic instability and replicative stress. ATX-559 sensitivity was enriched in dMMR/MSI-H cell lines including CRC. Sensitivity was also enriched in BRCA altered TNBC and HRD ovarian cancer cell lines. ATX-559 is well tolerated in vivo, resulting in robust dose-dependent tumor growth inhibition and regression selectively observed in CRC dMMR/MSI-H and BRCA altered TNBC CDX models, along with a well correlated PK/PD/efficacy relationship. Additionally, ATX-559 demonstrated enriched sensitivity in a large panel of BRCA altered TNBC (75%), dMMR/MSI-H (68%), and HRD ovarian (55%) PDX models.



 Together, these findings represent preclinical validation of DHX9 as a selective therapeutic in cancers with genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1/2, open-label, dose-escalation and expansion study to evaluate the safety profile of ATX-559 and to determine the recommended phase 2 dose (RP2D) and preliminary antitumor activity in patients with locally advanced or metastatic solid tumors, and molecularly defined cancers (NCT06625515).



 Sunaina Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver. ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2a8e9dfeed1b9e094182590e0697b62580f888" target='_blank'>
              Abstract B045: ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            S. Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen M. Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Mitochondria are essential organelles involved in energy production, cellular metabolism, and signal transduction. They have important impacts on tumorigenesis and cancer progression. Nevertheless, the associations between mitochondrial metabolic processes and chemotherapy resistance in colorectal cancer (CRC) are not well understood.



 We generated a chemotherapy-resistant colorectal cancer cell line, HCT-15/DOX, via doxorubicin (DOX) induction. We then performed proteomic and metabolomic analyses via LC-MS/MS technology on both the parental and the DOX-resistant cell lines. Additionally, transmission electron microscopy was used to examine changes in mitochondrial morphology between the two cell lines.



 The results revealed significant dysregulation of 185 proteins and 1099 metabolites in HCT-15/DOX cells relative to parental cells, highlighting the impact of chemotherapy resistance on cellular processes. The key functional proteins that were identified included upregulated SDHA, BCKDHB, CRYZ, NUDT6, CPT1A, and POLG, and downregulated CRAT, FDPS, SFXN1, and ATAD3B. Additionally, through combined multiomics pathway enrichment analysis, pyrimidine metabolism, purine metabolism, ascorbate and aldarate metabolism, propanoate metabolism, and the citrate cycle (TCA cycle) were identified as important metabolic processes associated with CRC chemotherapy resistance. Transmission electron microscopy analysis revealed that HCT-15/DOX cells had increased mitochondrial number, length, and area.



 This research highlights notable differences in mitochondrial morphology and diverse mitochondrial metabolic functions between parental and DOX-resistant HCT-15 CRC cells. The findings of the present study provide insights into the mitochondrial metabolic changes associated with CRC chemotherapy resistance, offering valuable insights into the mechanisms underlying these changes and identifying potential therapeutic targets for addressing CRC chemotherapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47edb9f4c564452e7f562c5c1875b89a970a0bb4" target='_blank'>
              Multiomics analyses of human colorectal cancer reveal changes in mitochondrial metabolism associated with chemotherapy resistance
              </a>
            </td>
          <td>
            Shiyi Chen, Qian Li, Wei Zheng
          </td>
          <td>2025-11-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany B. Johnson, Christopher Markovic, C. Fronick, Robert S. Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="This article provides a comprehensive overview of targeted and epigenetic therapies for acute myeloid leukemia (AML), highlighting their role in advancing treatment strategies. Recognizing that AML arises from complex genetic and epigenetic alterations, a deeper understanding of its pathogenesis has paved the way for precision medicine. The article reviews key targeted therapies, including IDH inhibitors (e.g., ivosidenib, enasidenib) for IDH mutations, FLT3 inhibitors (e.g., midostaurin, gilteritinib) for FLT3 alterations, BCL2 inhibitors (e.g., venetoclax) impacting apoptosis, and emerging Menin inhibitors for specific subtypes like KMT2A-rearranged or NPM1-mutant AML. It also discusses epigenetic therapies that target reversible changes, such as DNMT inhibitors (e.g., azacitidine, decitabine) for DNA methylation and HDAC inhibitors for histone modifications, along with other promising agents under development targeting epigenetic regulators. Furthermore, the article explores gene therapy as a significant future direction, detailing advancements in gene editing (CRISPR/Cas9) and strategies designed to shield healthy hematopoietic cells from targeted toxicity (e.g., CD33, CD123 editing) or to enhance the anti-leukemic function of immune cells (e.g., CAR-T, TCR therapy). The future perspective emphasizes the importance of developing synergistic combination therapies and leveraging molecular insights for truly personalized treatment approaches to overcome therapeutic challenges and improve long-term outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f1a005625d793e87f3fac27421531cd23f4756" target='_blank'>
              Targeted and epigenetic therapies for acute myeloid leukemia treatment
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Mobin Ghaderi, Milad Mokarami, Homaira Arian, Sara Mohammadnabi, Arastoo Farokhi
          </td>
          <td>2025-09-26</td>
          <td>Discover Oncology</td>
          <td>2</td>
          <td>3</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28967441eb1f46f1a9e983b8df32cfb979b71b23" target='_blank'>
              Chromatin and gene-regulatory dynamics of human pulmogenesis by single cell multiomic sequencing
              </a>
            </td>
          <td>
            Samuel H. Kim, Soon Il Higashino, Lea C. Steffes, Li Li, Anca M. Pașca, Maya Kumar, Christin S. Kuo, Mark A. Krasnow, William J. Greenleaf
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, A. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c038a3eb980cd4c3376d3d5fe3b978d2d8815e95" target='_blank'>
              A Machine Learning Approach Reveals CRISPR-Cas I-F as a Genomic Marker of Antibiotic Susceptibility in Uropathogenic E. coli
              </a>
            </td>
          <td>
            Alexandra M Young, P. Humburg, Fang Liu, Michael C. Wehrhahn, A. Tay, Stephen M. Riordan, Li Zhang
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Loss of the tumor suppressor PTEN is strongly associated with a lack of response to immune checkpoint blockade therapies in cancer patients, but the underlying mechanisms are not fully understood. We have developed a transformation model where knocking out PTEN in human mammary epithelial cells drives dependence on the p110β subunit of PI3K and, notably, elicits a robust induction of endogenous retroviral elements (ERVs) and activation of the interferon signaling. This constitutive cell-intrinsic interferon response, hallmarked by hyperactivated STAT1, is also observed in human tumors with a PTEN-low status. We further find that PTEN deficiency renders cancer cells resistant to the cytotoxic effects of immune cells and interferon-γ. Notably, PTEN loss also results in a dependency on an activated DNA damage response pathway, leading to an exquisite vulnerability to CDK12 inhibition. Our study suggests an interferon adaptation model in which tumors driven by PTEN deficiency inherently activate the interferon response, enabling them to adapt to interferon cytotoxicity and gain resistance to immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e20dd8e7190008586d0ab467c90ab994ab9aef9" target='_blank'>
              Loss of the tumor suppressor PTEN activates cell-intrinsic interferon signaling to drive immune resistance
              </a>
            </td>
          <td>
            Yubao Wang, Cherubin Manokaran, Jiajia Chen, Hao Gu, David Liu, Philippe Bousso, Jean J. Zhao, Thomas M Roberts
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="RNA repair is critical for cellular function. The Rtc system maintains RNA integrity within the translational machinery of bacteria. In E. coli, Rtc expression enables cells to rescue growth and survive treatment by conferring transient resistance to ribosome-targeting antibiotics, yet the mechanisms underpinning this resistance remain obscure. Here, we present a computational model of Rtc-regulated repair of translational RNAs. Integrating model predictions with experimental validations, we uncover notable cell-to-cell heterogeneity in rtc expression that impacts on translational capacity, indicating that rtc may induce a form of heteroresistance. We moreover identify Rtc targets that may reduce the translational capacity of cells and so potentiate antibiotic effects. Our findings elucidate a complex response underpinning resistance conferred by Rtc, offering alternate routes for addressing resistance in E. coli and other relevant pathogens. RNA repair helps bacteria survive antibiotic stress. Here, authors show that Rtc-driven repair causes cell-to-cell variation in resistance levels, revealing a potential form of heteroresistance, and identify key Rtc targets to enhance antibiotic effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd19ca4b27ee23689628c6e23e1bb197f23a5f1c" target='_blank'>
              Heterogeneity in responses to ribosome-targeting antibiotics mediated by bacterial RNA repair
              </a>
            </td>
          <td>
            Hollie J Hindley, Zechuan Gong, S. Moradian, M. G. Giuliano, Andrei V. Sapelkin, I. Kotta-Loizou, Martin Buck, C. Engl, Andrea Y Weiße
          </td>
          <td>2025-11-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 IDH-mutant gliomas are diffusely infiltrating, slow-growing brain tumors that inevitably undergo malignant progression to grade IV astrocytoma - a highly aggressive and uniformly fatal disease with dismal prognosis. Despite extensive research, the molecular mechanisms underlying this transformation remain poorly understood. While alterations in canonical driver genes such as CDKN2A, PTEN, and PDGFRA have been implicated, the broader landscape of genetic drivers involved in IDH-mutant glioma progression is largely uncharacterized. In contrast to many other malignancies, IDH-mutant gliomas acquire relatively few somatic point mutations during progression - typically in the range of 30 to 40 - suggesting that alternative mechanisms may be driving tumor evolution. Notably, these tumors exhibit widespread chromosomal instability, resulting in a heterogeneous and highly prevalent pattern of copy number alterations (CNAs). This observation raises the possibility that gene dosage changes due to CNAs play a critical role in tumor progression.



 To investigate this hypothesis, we performed integrative analysis of IDH-mutant glioma cohorts from the Mayo Clinic and TCGA datasets (~1,500 patients), identifying twelve highly recurrent CNAs significantly associated with patient outcomes. We refer to these as malignant IDH-associated CNAs (mIDH CNAs). We hypothesized that these recurrent CNAs harbor driver genes that confer selective growth advantages, thereby contributing to their recurrence and prognostic relevance. To systematically assess the functional impact of genes within these regions, we developed a novel in vivo CRISPR-based screening platform, termed CRISPR-Knock Out and Activation Linked Assay (CRISPR-KOALA). This approach utilizes stereotaxic delivery of lentiviral gRNA libraries - targeting mouse orthologs of ~1,800 genes within recurrently deleted regions and ~1,200 genes within amplified regions - into the brains of a genetically engineered mouse model of IDH-mutant glioma (rs557mut;IDHmut;P53KO;AtrxKO;Cas9GFP: Yanchus et. al Science 2022).



 Mice injected with mIDH CNA-targeting libraries developed tumors significantly more rapidly than control cohorts. Deconvolution of the CRISPR-KOALA screen identified several putative driver genes, including regulators of the NOTCH signaling pathway, lipid metabolism modulators such as THEM6, and the NMDA receptor modulator GRINA - implicating dysregulation of metabolism, immune response, and glutamatergic signaling in the malignant transformation of IDH-mutant gliomas.



 Together, these findings provide critical insights into the genetic mechanisms driving the progression of IDH-mutant gliomas and identify new candidate genes and pathways for potential therapeutic targeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174bd23b8ba2dba117306ce08fc0c554d76de475" target='_blank'>
              OS09.5.A RECURRENT COPY NUMBER ALTERATIONS DRIVE IDH-MUTANT ASTROCYTOMA DEVELOPMENT AND PROGRESSION
              </a>
            </td>
          <td>
            S. Haider, K. Drucker, R. Tsai, K. Al-Zahrani, C. Chiu, T. Kollmeyer, G. Zadeh, R. Jenkins, D. Schramek
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is among the deadliest human cancers, characterized by high heterogeneity that renders standardized treatments largely ineffective. The identification of biomarkers predictive of therapy response, and elucidating their underlying mechanisms, could unveil novel therapeutic opportunities and advance precision medicine strategies. We recently demonstrated that WNT6, a key WNT signaling activator, acts as an oncogenic driver and has prognostic value in GBM, but the molecular mechanisms by which WNT6 influences GBM aggressiveness and its impact on therapy response remain elusive.



 RNA sequencing of two matched GBM in vitro models with differential WNT6 expression provided a comprehensive transcriptomic profile. Bioinformatics analyses, including functional annotation, gene-set enrichment, and ingenuity pathways analysis, identified WNT6-driven biological processes. Functional and molecular assays, along with an in vivo orthotopic experiment, evaluated WNT6’s impact on temozolomide (TMZ) sensitivity, the standard GBM chemotherapy. Clinical data analyses assessed the prognostic significance of WNT6 in TMZ-treated GBM patients.



 The WNT6 transcriptome was enriched in processes and pathways that suggest a potential role for WNT6 in therapy resistance. In vitro, WNT6 expression reduced TMZ efficacy across multiple oncogenic features, which was further supported in vivo, where WNT6-high tumors showed decreased responsiveness to TMZ in comparison to WNT6-low tumors. WNT6 expression was also associated with the differential activation of DNA repair pathways critical for TMZ response, including base excision repair, mismatch repair, and MGMT-mediated repair. Among TMZ-treated GBM patients, WNT6 was found to be an independent predictor of reduced survival.



 This study suggests that WNT6 is predictive of TMZ response in GBM, and sheds light on the molecular mechanisms by which WNT6 may be operating. Collectively, these findings position WNT6 as a promising candidate for future therapeutic strategies in precision medicine, including combination therapies targeting WNT6 and its associated resistance pathways to improve TMZ efficacy.



 Financial support was provided by grants and projects from FCT (https: //doi.org/10.54499/2022.04859.PTDC to B.M.C. and E.P.M.; https: //doi.org/10.54499/2021.02600.CEECIND/CP1664/CT0015 to C.S.G; and 2024.00785.BD to J.M.F.) and by National funds https: //doi.org/10.54499/UIDB/50026/2020, and https: //doi.org/10.54499/UIDP/50026/2020, https: //doi.org/10.54499/LA/P/0050/2020, and by the project NORTE-01-0145-FEDER-000055, supported by Norte Portugal Regional Operational Programme (NORTE 2020), under the PORTUGAL 2020 Partnership Agreement, through the European Regional Development Fund (ERDF).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93dd14a38dce213f951800a3a65a7c23b2f067e1" target='_blank'>
              P12.08.A DECIPHERING WNT6-DRIVEN MECHANISMS OF AGGRESSIVENESS AND RESISTANCE TO CHEMOTHERAPY IN HUMAN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. M. Ferreira, C. Gonçalves, E. Martins, B. Costa
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancer is a highly lethal malignancy, with treatment efficacy frequently compromised by the development of cisplatin resistance. This study investigates the molecular mechanisms driving cisplatin resistance, focusing on interferon-induced protein with tetratricopeptide repeats 1 (IFIT1), a key interferon-stimulated gene. Through comprehensive transcriptome profiling, we systematically compared gene expression patterns between cisplatin-sensitive and cisplatin-resistant ovarian cancer cells, identifying a consistent upregulation of IFIT1 in resistant cells. Functional assays provided evidence that IFIT1 plays a central role in mediating cisplatin resistance. Mechanistically, IFIT1 was found to be associated with activation of inflammatory signaling pathways, suggesting its involvement in resistance-related immune regulation. Clinically, elevated IFIT1 expression significantly correlated with an unfavorable prognosis in ovarian cancer patients. Moreover, we demonstrated that hypoxia-inducible factor-1 alpha transcriptionally regulates IFIT1, linking its expression to hypoxia response. Overall, these findings identify IFIT1 as a key driver of cisplatin resistance in ovarian cancer, shedding light on its clinical significance and the regulatory networks underpinning therapy resistance. IFIT1 holds promise both as a prognostic indicator and as a target for therapeutic intervention. However, its clinical translation will depend on rigorous validation and in-depth exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3e993b5a5faff584d9a192dda2fe082dc88120" target='_blank'>
              Overcoming cisplatin resistance in ovarian cancer by targeting IFIT1-mediated inflammation responses
              </a>
            </td>
          <td>
            Min Hu, Xuejiao Zhao, Xiuming Li, Bin Liu, Chunluan Yuan
          </td>
          <td>2025-11-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Clear cell renal cell carcinoma (ccRCC) represents the most aggressive form of renal cell carcinoma (RCC), distinguished by pronounced intratumoral heterogeneity, extensive metabolic reprogramming, and marked resistance to conventional therapeutic approaches. This study aimed to comprehensively characterize the cellular heterogeneity, epigenetic regulation, and transcription factor (TF) networks in ccRCC by integrating multi-omics data, and to identify functional key genes with prognostic and therapeutic significance. Methods Single-cell RNA sequencing (scRNA-seq), single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq), and spatial transcriptomics (ST) were integrated to comprehensively explore cellular heterogeneity, epigenetic regulation, and TF networks in ccRCC. To uncover dynamic alterations in gene expression during cellular differentiation, single-cell pseudotime analysis and gene set enrichment analysis (GSEA) were performed. Furthermore, the functional significance of Y-box binding protein 3 (YBX3) in ccRCC cells was experimentally validated. Results Single-cell transcriptomic profiling revealed 16 distinct cell populations within the ccRCC tumor microenvironment (TME), including ccRCC tumor cells, exhausted CD8+ T cells (Exhau CD8+ T cells), and macrophages. The scATAC-seq analysis demonstrated cell type-specific chromatin accessibility in immune cells, whereas ccRCC tumor cells exhibited reduced accessibility at immune-related genes, such as cluster of differentiation 2 (CD2). Epigenetic profiling further indicated that differentially accessible chromatin peaks in ccRCC cells were primarily enriched within intronic and exonic regions, implicating key TFs, including hepatocyte nuclear factor 1-beta (HNF1B) and the FOS-JUNB complex. An integrated analysis of scRNA-seq and scATAC-seq datasets identified five critical genes, YBX3, cubilin (CUBN), small nucleolar RNA host gene 8 (SNHG8), acetyl-CoA acyltransferase 2 (ACAA2), and protein kinase AMP-activated catalytic subunit α2 (PRKAA2), that were significantly associated with ccRCC prognosis. Pathway enrichment analysis revealed their involvement in metabolic reprogramming and tumor progression. Functional assays further confirmed that YBX3 knockdown inhibited ccRCC cell proliferation and migration. Conclusions This study elucidates cellular heterogeneity, the epigenetic regulatory landscape, and the key genes driving ccRCC progression. The integration of multi-omics data offers novel insights into precise diagnostic strategies and therapeutic interventions, highlighting the pivotal role of genes such as YBX3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ee8baff74e64150e50eefda9fdb1e11ec3dfcb" target='_blank'>
              Single-cell multi-omics and spatial transcriptomics reveal the transcriptional regulatory landscape of clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Juan Duan, Peifeng Ke, Bangqi Wang, Xiaofu Qiu, Yifeng He, Zongtai Zheng, Zemin Wan, Chuling Wu, Zhuoyun Lv
          </td>
          <td>2025-10-01</td>
          <td>Translational Andrology and Urology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f239a0389d1c04e5b0bbbc12b98886b6a5e770dd" target='_blank'>
              Alanine dependence of trans-translation contributes to riboregulation of mycobacterial antibiotic recalcitrance genes
              </a>
            </td>
          <td>
            Andrea Majstorovic, Helene Botella, Sarah M. Schrader, Yaroslav Lavrynchuk, Xinguo Chen, Enkai Jin, Jason Taslim, R. Antoine, Alain R. Baulard, C. Guilhot, Przemysław Płociński, Sandra L. Wolin, Julien Vaubourgeix
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/487c4f78dbf061ef150408c6cab4f54351fa39a1" target='_blank'>
              Forecasting multi-trait resistance evolution under antibiotic stress
              </a>
            </td>
          <td>
            Suvam Roy, E. Libby, Peter Lind
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db525943b0ddbcca73f6e3d8fd119fef42f8fb23" target='_blank'>
              Aberrant inheritance of extrachromosomal DNA amplifications promotes cancer evolution
              </a>
            </td>
          <td>
            Shir Marom, Inbar Lifshits Dayan, Venkata Narasimha Kadali, Mădălina Giurgiu-Kraljič, Gabriela Koifman, Karen Hakeny, Madhuri Chaurasia, Orléna Benamozig, R. Nevo, I. Azuri, Julia Ryvkin, R. Rotkopf, Gil Stelzer, Nino Oniashvili, Jacques Mardoukh, Sarah Pollock, Nika Iremadze, Z. Shipony, Meital Kupervaser, N. Wigoda, Dena Leshkowitz, Z. Elazar, Esther Berko, A. Henssen, Ofer Shoshani
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="DNA double-strand break (DSB) clusters are a hallmark of high-linear energy transfer (high-LET) radiation and are associated with pronounced biological effects, including reduced cell survival and elevated genomic instability. Our previous work in Chinese hamster cells, engineered with variably designed clusters of I-SceI recognition sites, integrated at multiple genomic locations, revealed that DSB clusters suppress classical non-homologous end-joining (c-NHEJ) and induce chromosomal translocations that ultimately increase cell lethality. Here, we extend this line of investigation to human cell lines and generate DSB clusters using alternative approaches that do not require prior genetic manipulation of the test cell lines. We employ CRISPR/Cas9-technology to generate DSB clusters of specific design at a selected genomic locus and examine their consequences on locus integrity. We target Exon 3 of the human HPRT (hHPRT) gene and introduce single DSBs or DSB clusters of varying numbers and inter-DSB distances. Alterations at the locus reflecting hHPRT gene inactivation, are quantified as mutations causing resistance to 6-thioguanine (6TG). Our results show that DSB clusters are markedly more potent inducers of mutations than single DSBs and that DSBs spaced within ~ 600 base pairs synergize in mutation induction. Mechanistic analyses using small-molecule inhibitors and engineered gene knockout cell lines reveal that the increased mutagenicity of clustered DSBs is primarily mediated by DNA end resection and PARP1-dependent alternative end-joining (alt-EJ) pathways. These findings reinforce the biological relevance of DSB clusters as a severe form of complex DNA damage and provide mechanistic insights into high-LET radiation-induced increased cell killing and genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82859607e3d79ce6936e2d753c4009b872b5c5d9" target='_blank'>
              CRISPR/Cas9 generated DSB clusters mimic complex lesions induced by high-LET radiation and shift repair from c-NHEJ to mutagenic repair pathways
              </a>
            </td>
          <td>
            E. Mladenov, M. Kallies, M. Stuschke, Eleni Gkika, G. Iliakis
          </td>
          <td>2025-10-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3850c36c459b0acd857f870c20ecd2d5e8ac8aed" target='_blank'>
              Activation of the TP53 pathway is a therapeutic vulnerability in NUP98::KDM5A+ pediatric AML
              </a>
            </td>
          <td>
            L. N. Cifarelli, Hasan Issa, L. Proietti, Konstantin Schuschel, L. Gack, K. Menge, C. Ihling, Meike Vogler, A. Sinz, J. Klusmann, Dirk Heckl
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive and lethal form of primary brain tumor, often characterized by resistance to conventional therapies and a poor clinical prognosis. Tumor Necrosis Factor-Related Apoptosis-Inducing Ligand (TRAIL) selectively induces apoptosis in cancer cells; however, TRAIL resistance remains a significant obstacle in GBM treatment. SETD5, a histone methyl transferase with emerging roles in chromatin remodeling and gene regulation, has been implicated in neurodevelopmental disorders and cancer, but remains poorly characterized in glioma biology. This study investigates the role of SETD5 in modulating TRAIL resistance and explores its potential as a therapeutic target in GBM. Publicly available datasets (TCGA, GTEx, GEPIA2, UALCAN, Gliovis, and GSCA) were used to assess SETD5 expression, mutation, and promoter methylation in GBM. Associations with clinical parameters, molecular subtypes, immune infiltration, and functional states were analyzed using tools like cBioPortal, CancerSEA, MEXPRESS, and STRING. siRNA-mediated knockdown of SETD5 was performed in U87 and LN229 GBM cell lines, followed by TRAIL treatment. Western blotting assessed changes in apoptotic and survival pathways, while wound healing assays evaluated cell migration. Expression of deubiquitinases (USP5, USP8, and USP10) was also measured. SETD5 was significantly upregulated in GBM compared to normal brain tissues across multiple databases and was especially elevated in the Mesenchymal and Classical subtypes. CNV analyses revealed alterations in SETD5 that correlated with immunosuppressive cell infiltration. SETD5 expression was positively correlated with survival pathways and negatively correlated with cytotoxic immune infiltration. TRAIL treatment induced SETD5 expression in a dose-dependent manner. Knockdown of SETD5 enhanced TRAIL-mediated apoptosis, reduced expression of survival markers (p-AKT, P-ERK, NF-κB), and increased pro-apoptotic proteins (Cytochrome c, SMAC). SETD5 depletion also downregulated USP5, USP8, and USP10, suggesting its role in stabilizing oncogenic proteins via deubiquitination. SETD5 plays a critical role in promoting TRAIL resistance and GBM survival by regulating apoptotic pathways, immune evasion, and deubiquitinase expression. Moreover knockdown of SETD along with TRAIL treatment leads to down regulation NFκB which a major promoter for cell survival. Its inhibition sensitizes glioma cells to TRAIL-induced apoptosis, identifying SETD5 as a potential therapeutic target. Targeting SETD5 could represent a novel strategy to overcome TRAIL resistance and enhance the efficacy of GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3902ac90e3299e5f399e27e82b6fce1864afc3" target='_blank'>
              SETD5 in glioma cells conferred TRAIL resistance induction
              </a>
            </td>
          <td>
            Lakshay Taneja, Sachin Bhardwaj, Ajay Kumar Yadav
          </td>
          <td>2025-09-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739921fbe29bda5d659d1ced9f84aedc8ae3f8d2" target='_blank'>
              Next-generation oncology: integrative therapeutic frontiers at the crossroads of precision genomics, immuno-engineering, and tumor microenvironment modulation
              </a>
            </td>
          <td>
            Ahmad M Alamri, Abdullah A. Assiri, Bushra Khan, Najeeb Ullah Khan
          </td>
          <td>2025-09-20</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, Sameh Youssef, Alain de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Glioblastoma (GBM), as one of the most common and aggressive primary brain tumors, pose significant challenges in diagnosis and treatment, highlighting the urgent need for the identification of novel biomarkers, NFKBIE, which may play a critical role in tumor progression, immune modulation, and therapeutic response. We investigated the role of NFKBIE in GBM through both bioinformatics analysis and experimental validation. Initially, bioinformatics analysis based on public databases revealed significant differential expression of NFKBIE across various cancer types, particularly in GBM, where elevated expression was correlated with poor patient prognosis, including overall survival, disease-specific survival, and progression-free survival. Furthermore, genetic alterations in NFKBIE, such as copy number variations (CNVs) and tumor mutational burden (TMB), were significantly associated with tumor progression. In the experimental validation phase, we utilized small interfering RNA (siRNA) technology to silence NFKBIE expression in GBM cell lines, including U87 and T98G. Functional assays were then performed to assess the impact of NFKBIE knockdown. Cell proliferation assays, including CCK-8 and clonogenic assays, demonstrated that silencing NFKBIE significantly inhibited the proliferation of GBM cells. Migration and invasion assays, including wound healing and Transwell assays, further confirmed that NFKBIE knockdown markedly suppressed the migratory and invasive abilities of GBM cells. Additionally, apoptosis assays, such as TUNEL staining, revealed that NFKBIE silencing significantly promoted apoptosis in GBM cells. Western blot analysis confirmed the changes in the expression of key markers associated with proliferation, stemness, migration, invasion, and apoptosis. Furthermore, in a mouse xenograft model, NFKBIE knockdown significantly slowed tumor growth. Histological analysis of the excised tumors confirmed that reduced NFKBIE expression was closely associated with the inhibition of tumor growth. Notably, NFKBIE silencing also led to the downregulation of components of the Hedgehog signaling pathway, suggesting that NFKBIE may regulate tumor progression through modulation of this critical pathway. In conclusion, high NFKBIE expression is strongly associated with GBM progression and poor prognosis, and targeting its expression may offer a promising therapeutic strategy for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567a01090acb470e17dfe52d5931751c0e81fc63" target='_blank'>
              NFKBIE as a prognostic biomarker and therapeutic target for GBM: role in Hedgehog signaling activation
              </a>
            </td>
          <td>
            Feng Mo, Yang Li, Ce Dong, Lijun Yang, Xiaoli Ma, Baogen Pan, Z. Fan
          </td>
          <td>2025-10-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Recurrent GBM poses significant treatment challenges due to limited therapeutic options and reliance on primary tumor molecular profiles, often overlooking treatment-induced molecular and phenotypic changes in recurrent tumors. This study investigates genomic, DNA methylation, and transcriptomic differences between a primary GBM (HF3016) and its matched recurrent tumor post-temozolomide and radiotherapy (HF3016R), using neurosphere cultures and patient-derived xenografts (PDXs) to assess radiotherapy (RT) response. Genomic DNA was analyzed via whole-genome sequencing, whole-exome sequencing, and EPIC DNA methylation arrays. Bulk RNA sequencing was performed on HF3016 and HF3016R cancer stem cell (CSC) cultures (n=4) and orthotopic PDX models (n=3). Differential gene expression was analyzed using NOISeq (q=0.95, fold change >2) on TMM-normalized data, with pathway enrichment via Metascape 3.1. Tumor growth and RT sensitivity were evaluated in NCRNU athymic nude mice implanted intracranially with HF3016 or HF3016R cells (3x10^5 cells/mouse, N=9-10 per group), receiving 5 Gy RT daily for 3 days or no treatment. Survival was assessed using Kaplan-Meier curves and Log-rank tests. No genomic driver alterations (CDK4, EGFR, MYC ecDNA amplification, PTEN, p53 mutations) or DNA methylation mesenchymal subgroup changes were observed between primary and recurrent models. However, a transcriptional shift from proneural to mesenchymal subtype occurred in recurrent models, with enriched epithelial-mesenchymal transition and interferon response pathways in HF3016R, while nervous system development pathways were enriched in HF3016. HF3016R PDXs exhibited faster growth (p=0.002) and greater RT resistance (p=0.0021 for HF3016, p=0.061 for HF3016R) compared to HF3016. These findings highlight the importance of considering treatment-induced molecular changes in recurrent GBM for optimizing therapeutic strategies. This study reveals that while genomic and DNA methylation profiles remain stable between primary and recurrent GBM models, significant transcriptional reprogramming occurs. The enrichment of epithelial-mesenchymal transition and interferon response pathways in recurrent models is associated with their aggressive growth and increased radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec6b57a9c052c7471268a4341bec91fcb4af93" target='_blank'>
              EXTH-73. Identifying therapeutic targets for recurrent glioblastoma using patient-derived longitudinal models
              </a>
            </td>
          <td>
            Indrani Datta, O. Nuga, A. Berezovsky, Laila M. Poisson, Ana deCarvalho
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pathways of intrinsic resistance in bacteria are promising targets for novel antibiotics and resistance breakers. Here, we used a genome-wide screen to identify single gene knockouts of Escherichia coli that were hypersusceptible to trimethoprim and chloramphenicol, two chemically diverse broad-spectrum antibiotics. Among the hits from our screen, knockouts of acrB, an efflux pump, and rfaG or lpxM, both involved in cell envelope biogenesis, were hypersensitive to multiple antimicrobials and could sensitize genetically resistant E. coli strains to antibiotics. Using experimental evolution under trimethoprim pressure, we show that high drug selection regimes drove these knockouts to extinction more frequently than wild type. Among them, ΔacrB was most compromised in its ability to evolve resistance, establishing it as a promising target for “resistance proofing.” At a sub-inhibitory trimethoprim concentration, however, all three knockouts adapted to the antibiotic and consequently recovered from hypersensitivity, albeit to different extents. This recovery was driven by mutations in drug-specific resistance pathways, rather than compensatory evolution, frequently involving upregulation of the drug target. Notably, resistance-conferring mutations could by-pass defects in cell wall biosynthesis more effectively than efflux even though resistant mutations did not directly engage either pathway. Since inhibiting drug-efflux emerged as a better strategy, we tested the ability of chlorpromazine, an efflux pump inhibitor (EPI), to resistance proof E. coli against trimethoprim. While qualitatively similar in the short term, genetic and pharmacological inhibition differed dramatically on an evolutionary time scale due to evolution of resistance to the EPI. Further, adaptation to the EPI-antibiotic pair also led to multidrug adaptation. The lack of concordance between genetic and pharmacological inhibition revealed a crucial lacuna in our understanding of the mutational repertoires that facilitate adaptation to antibiotics in bacteria. We propose that while intrinsic resistance mechanisms are effective targets for antibiotic sensitization, rapid evolutionary recovery may significantly limit their utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfa86a0d6ec827f3a247e4e17cf602f5ec115f1d" target='_blank'>
              Impeding pathways of intrinsic resistance in Escherichia coli confers antibiotic sensitization and resistance proofing
              </a>
            </td>
          <td>
            Manasvi Balachandran, Rohini Chatterjee, Ishaan Chaudhary, Chinmaya Jena, N. Matange
          </td>
          <td>2025-10-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="We have investigated changes in chromosome conformation, nuclear organization, and transcription during differentiation and maturation of control and mutant motor neurons harboring hexanucleotide expansions in the C9orf72 gene that cause amyotrophic lateral sclerosis (ALS). Using an in vitro reprogramming, differentiation and neural maturation protocol, we obtained highly purified populations of post-mitotic motor neurons for both normal and diseased cells. As expected, as fibroblasts are reprogrammed into iPSCs, and as iPSCs differentiate into motor neurons, chromatin accessibility, chromosome conformation, and nuclear organization change along with large-scale alterations in transcriptional profiles. We find that the transcriptome changes extensively during the first three weeks of post-mitotic neuronal maturation, with thousands of genes changing expression, but then is relatively stable for the next three weeks. In contrast, chromosome conformation and nuclear organization continue to change over the entire 6-week maturation period: chromosome territoriality increases, long-range interactions along chromosomes decrease, compartmentalization strength increases, and centromeres and telomeres increasingly cluster. In motor neurons derived from ALS patients such changes in chromosome conformation were much reduced. Chromatin accessibility changes also showed delayed maturation. The transcriptome in these cells matured relatively normally but with notable changes in expression of genes involved in lipid, sterol and mitochondrial function. We conclude that neural maturation is associated with large scale post-mitotic changes in gene expression, chromosome conformation and nuclear organization, and that these processes are defective in motor neurons derived from ALS patients carrying C9orf72 hexanucleotide repeat expansions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ebb664b6f1b21244e0cfd38a3721d45d012b3f" target='_blank'>
              Dynamic changes in chromosome and nuclear architecture during maturation of normal and ALS C9orf72 motor neurons
              </a>
            </td>
          <td>
            Özgün Uyan, Snehal Sambare, Marlies E. Oomen, Nicholas Wightman, A. Schooley, Joseph R. Klim, Houda Belaghzal, Özkan Aydemir, Betul Akgol-Oksuz, Zeynep Sena Agim Uslu, Kevin Eggan, Robert H. Brown, Job Dekker
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Fms-like tyrosine kinase 3 (FLT3), a class III receptor tyrosine kinase essential for hematopoiesis, is a well-established oncogenic driver in acute myeloid leukemia (AML). Canonical internal tandem duplications (ITD) and tyrosine kinase domain (TKD) mutations inform prognosis and guide targeted therapy. Recent evidence highlights FLT3 as a critical oncogenic hub in acute lymphoblastic leukemia (ALL), where its alterations extend beyond ITD/TKD mutations to include non-canonical mutations with only partially explored functional implications. Moreover, recently discovered regulatory mechanisms, mostly acting on the FLT3 locus, drive FLT3 overexpression in ALL, including transcriptional regulation by rearranged ZNF384, epigenetic modifications, novel circular-RNA URAD::FLT3 fusions, and 13q12.2 deletions leading to enhancer hijacking and topologically associated domain (TAD)-boundary disruptions. The impact of these alterations on leukemogenesis and the possibility to target them in ALL subtypes is discussed here. Data from the Functional Omics Resource of Acute Lymphoblastic Leukemia (FORALL) across B- and T-ALL cell line subtypes drug screening, and from preclinical and clinical evidence reveals a variable efficacy in FLT3-mutated and FLT3-overexpressing ALL subtypes, supporting a molecularly guided treatment approach. Building on the success of FLT3 inhibitors in mutated AML and in light of the emerging results in patients lacking FLT3-ITD and in FLT3-like AML cases, presenting with a gene expression pattern similar to FLT3-mutated ones despite the absence of mutations, we discuss their potential in ALL and we consider novel therapeutic strategies, including new FLT3 inhibitors, antibody-based approaches, FLT3 CAR-T therapy, and synergistic drug combinations, such as FLT3 and BCL2 inhibition. These new insights reviewed here may redefine FLT3 as a pan-leukemic target, with ALL-specific activation mechanisms offering unique therapeutic windows. The implementation of FLT3 expression profiling and full-coding mutation screening in ALL (and in AML) diagnostics could unlock precision medicine approaches. By bridging the AML experience with ALL innovations, this review outlines a roadmap for FLT3-targeted therapies and combination strategies, underscoring the urgency of biomarker-driven clinical trials to optimize FLT3-directed interventions in acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b724e5e3a02c4639c7dca6522dee54c5f66974d4" target='_blank'>
              Novel molecular mechanisms of FLT3 deregulation: from the acute myeloid leukemia experience to therapeutic insights in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            F. Lo Schiavo, C. Salvesi, M. Jandoubi, F. Pirini, J. Garbetta, G. Martinelli, G. Simonetti, A. Ferrari
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33632e56269b92001ac89763ee4bc728806996ab" target='_blank'>
              Molecular mechanisms of drug-induced mutagenesis: a narrative review
              </a>
            </td>
          <td>
            Tamene Bekele, A. Tsige, K. Ayenew
          </td>
          <td>2025-09-23</td>
          <td>Discover Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Pseudomonas aeruginosa causes severe healthcare-associated infections. High-risk clones are defined by global dissemination and multidrug resistance, yet virulence is heterogeneous. We sought to map accessory-genome determinants associated with high-risk clones by integrating whole-genome sequencing (WGS) with a Caenorhabditis elegans infection model. Methods We analyzed 84 clinical isolates plus publicly available genomes using WGS, phylogenomics, and resistome/virulome profiling. Virulence was measured by C. elegans slow-killing (SK). A GWAS of accessory-genome subelements (AGEs) identified loci with high- (HVA) or low-virulence association (LVA). Coding sequences were annotated with Prokka and InterPro. Results Although high-risk and sporadic clones carried a similar total number of antimicrobial-resistance genes, 15/67 (22.38%) genes/variants were enriched in high-risk clones, producing class-level enrichment (p < 0.002) for aminoglycosides, phenicols, trimethoprim, sulfonamides, and tetracyclines, but not β-lactams or fosfomycin. Many resistance determinants are recognized mobile-element cargo such as integron cassettes or plasmid/ICE-borne genes (e.g., aadA, dfrB, blaVIM-2, crpP, cmlA, floR), indicating a mobility-linked resistome in high-risk clones. GWAS identified 113 AGEs linked to SK virulence (42 HVA, 71 LVA). HVA regions were enriched for pyoverdine (fpvA, pvdE, pvdD) and LPS O-antigen (wbpA/B/D) loci, whereas LVA regions were enriched for ICE/conjugation/integrase motifs. cdsA and clpP were newly associated with P. aeruginosa virulence. Phenotypically, high-risk clones were more often strong biofilm producers and none were non-producers. High-risk clones were not consistently more virulent in SK, suggesting success reflects persistence traits (mobile DNA and biofilm under antibiotic pressure). Conclusion Accessory-genome GWAS revealed two risk dimensions: acute-virulence programs (HVA) versus mobility functions (LVA) favoring persistence and spread. Because SK measures acute virulence, readouts did not align with high-risk designations. Genomic reports should combine high-risk assignment with accessory-genome and effector profiling to support earlier containment and mechanism-aware, biofilm-focused care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8045c333041165303ba0060a8e6a13199737d2ff" target='_blank'>
              Genetic and virulence factors behind the success of high-risk Pseudomonas aeruginosa clones: insights from comparative genomics and an experimental infection model
              </a>
            </td>
          <td>
            R. O. de Sales, Laura Leaden, Dayblegschwel Santos Martins, Paula Koga, A. R. Toniolo, Fernando Gatti de Menezes, M. A. Mori, M. Martino, Patricia Severino
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aim: This study aimed to elucidate the role of regulator of chromosome condensation 1 (RCC1) in colorectal cancer (CRC) progression, as well as its involvement in chemoresistance. We specifically examined how RCC1 knockdown modulates cellular responses, including cell cycle, apoptosis, and senescence induced by 5-fluorouracil (5-FU) or doxorubicin (Doxo) in both parental and drug-resistant CRC cell lines. Additionally, we assessed the potential of RCC1 inhibition as an adjuvant therapeutic strategy to enhance the efficacy of chemoradiotherapy in CRC.
 Methods: The expression of RCC1 in colon cancer tissues and corresponding adjacent non-cancerous tissues was evaluated through tissue microarrays, and its correlation with characteristics and patient prognosis was also examined. Subsequently, a series of in vivo and in vitro experiments based on parental and drug-resistant CRC cell lines were conducted to assess the impact of RCC1 knockdown on sensitivity to 5-FU or Doxo. Finally, transcriptomic analysis and subsequent validation assays were performed to explore the underlying molecular mechanisms.
 Results: RCC1 knockdown significantly enhanced the antitumor efficacy of 5-FU and Doxo in both CRC and drug-resistant CRC cells. In xenograft models, RCC1 knockdown in combination with 5-FU or Doxo suppressed tumor growth with no evident systemic toxicity observed. Transcriptomic profiling and experimental verification revealed that RCC1 knockdown may impair DNA repair by downregulating key repair proteins, thereby leading to more severe and sustained DNA damage.
 Conclusion: Our results indicate that RCC1 downregulation enhances the responsiveness of both parental and drug-resistant CRC cells to 5-FU and Doxo, highlighting its potential as a therapeutic adjunct to improve clinical outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8afbd5b9e059c22aa8009a12c6871018f16b21" target='_blank'>
              RCC1 knockdown sensitizes drug-resistant colorectal cancer to 5-fluorouracil or doxorubicin by impairing DNA repair
              </a>
            </td>
          <td>
            Jing Li, Ya Meng, Xumei Ouyang, Xiaowen Lin, Yangzhe Wu, Hang Fai Kwok
          </td>
          <td>2025-11-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa756c05989530f16b2f4c4778c0bfc833ec875f" target='_blank'>
              Deep Learning links TP53 genotype to expression-defined transcriptional program in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            M. Tsesmelis, Rebecca Andersson, Alba Ferrer Perez, Sara Montserrat Vazquez, M. Carolina Florian, M. Mulaw
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768ee203fcd258706f2498c5fcf4ab4120a2eda" target='_blank'>
              Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages
              </a>
            </td>
          <td>
            Moein Rajaei, Alexander Yang, Kira A. Glasmacher, Christopher N. Cross, Nic Fisk, Elizabeth B. Perry, Jeff Mandell, S. Gaffney, Takafumi N. Yamaguchi, J. Livingstone, Jose Costa, Peter Humphrey, Vincent L. Cannataro, P. Boutros, Jeffrey P. Townsend
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb4e3ab30047458c58fb459af469d22c1112544" target='_blank'>
              CRISPR/Cas9 library screening uncovered CCT2 as a critical driver of acquired resistance to EGFR-targeted therapy by stabilizing TMX1 in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Zihao Ke, Qi Zhang, Xingyu Chen, Rongrong Jin, Gaohua Han, Qianhua Cao, Ke Zhu, Shihui Wei, Jiajin Chen, Qian Wang, Meiling Zhang, Weina Huang, Kaimin Li, Kunlong Xiong, Kaihua Lu, Ying Chen, Chao Cao
          </td>
          <td>2025-10-30</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53813423a746c2c5a950b8780d63e2566846931" target='_blank'>
              Modeling response to the KRAS-G12C inhibitor AZD4625 in KRASG12C NSCLC patient-derived xenografts reveals insights into primary resistance mechanisms.
              </a>
            </td>
          <td>
            J. Rosen, P. Cao, Nhu-An Pham, Matthew Waas, Q. Li, Katrina Hueniken, Mutian Wang, R. Navab, Leanne Wybenga-Groot, N. Radulovich, Michael Niedbala, A. Koers, Sarah Ross, Michael F Moran, A. Sacher, M. Tsao
          </td>
          <td>2025-10-11</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d27cc9b475222eaa7f4e83e26c9db176ee78f1" target='_blank'>
              Novel mutations in transducin-like enhancer of split 6 cause infertility in women with embryonic arrest.
              </a>
            </td>
          <td>
            Jifan Tan, Yue Jiang, Ching Wan, Kehan Zhu, Yan Xu, Yanwen Xu, Jia Huang
          </td>
          <td>2025-09-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Phosphatidylinositol 3-kinases (PI3Ks) are heterodimeric lipid kinases that are involved in a diverse array of cellular functions such as growth, metabolism, and migration. Mutations in PIK3CD, which encodes an immune-specific catalytic subunit of PI3K, cause both dominant (activating) and recessive (loss of function) immune deficiencies in humans. Here we report a family with three affected children carrying a novel bi-allelic, truncating mutation in PIK3CD. All three patients exhibited chronic diarrhea and recurrent sinopulmonary infections. Immunoblot confirmed loss of protein along with reduced expression of the associated p85α regulatory subunit. Immune phenotyping showed B cell dysregulation with abnormally high levels of naïve cells. In vitro functional testing of CD19 + and enriched naïve B cells revealed impaired proliferation, and reduction in class-switch recombination upon CD40L and IL-21 stimulation. Our data raise the possibility that PI3K-related dysregulation in human B cells may be broader than in mouse models, where class-switch recombination can still occur with external T cell help. Our study substantially increases the limited number of patients known to have immune deficiency due to loss of PIK3CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fc13ab6649c69647506d7b2a8ea278a43388d95" target='_blank'>
              Homozygous Loss of Function PIK3CD Mutation in Multiple Siblings Leading To B Cell Dysregulation and Autoimmunity
              </a>
            </td>
          <td>
            Huda Alajlan, Amer Al-Mazrou, Hibah Alruwaili, S. Sumayli, A. Almehaidib, Khalid Alsaleem, Sawsan Abu Awwad, H. Ghebeh, M. Al-Alwan, A. Alazami, H. Al-Mousa
          </td>
          <td>2025-09-30</td>
          <td>Journal of Clinical Immunology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Tunneling nanotubes (TNTs) are dynamic membranous protrusions that are critical for intercellular communication in cancer, yet their transient nature and lack of definitive molecular markers pose challenges for systematic study. To address this issue gap in knowledge, we used bulk and single-cell transcriptomic sequencing approaches to identify differentially expressed genes in ovarian cancer cells enriched for TNTs to identify molecular pathways and genes potentially implicated in TNT formation, maintenance and function. We used edgeR and Seurat to perform differential expression analysis on ovarian cancer cells grown in conditions favoring upregulation of TNT formation and further performed single-cell sequencing of the same cell type isolated via spatiotemporal genomic analysis (SaGA) and fluorescence-activated cell sorting (FACS). We found 771, 32 and 279 genes upregulated in bulk sequenced TNT forming SKOV3, COV362 and ES-2 ovarian carcinoma cell lines, respectively. Gene families that repeatedly came up across all three of the cell lines included carbonic anhydrases, matrix metalloproteinases, SERPIN (Serine Protease Inhibitor-30), and SNORA (Small Nucleolar RNAs). We narrowed down candidate genes of interest to 23 differentially expressed genes (DEGs), many of which are involved in cellular stress response, organization of the extracellular matrix or intracellular cytoskeleton, cell adhesion, and other pathways. Further analyzing the single cell sequenced ES-2 cells, we found the differential expression of KISS1 gene, a metastasis suppressor implicated in cytoskeletal organization, significantly upregulated in TNT-enriched cells, highlighting its potential role in TNT formation. Other genes, such as EPHB1, MMP3, and WNT7B, were also upregulated in TNT-enriched cells, suggesting their involvement in regulating cell adhesion, migration, and differentiation. Conversely, we observed downregulation of BGN, a gene associated with promoting metastatic potential and NFkB signaling in cancer cells, as well as SYNGR1, a gene traditionally linked to synaptic transmission but also involved in vesicle exocytosis, which may influence TNT formation.



 Rojina Sapkota. RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5389ea72947e09856895a8de975c87a724ed10" target='_blank'>
              Abstract B015: RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication
              </a>
            </td>
          <td>
            Rojina Sapkota
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40935d544b5f896144e035981022bfa9b8389a70" target='_blank'>
              Investigation of Double-Stranded DNA Donors and CRISPR-Cas9 RNP for Universal Correction of Mutations Causing Cystic Fibrosis in Human Airway Cells
              </a>
            </td>
          <td>
            Vrishti Sinha, P. Ayoub, C. Juett, Lindsay E. Lathrop, Ruth A. Foley, Ruby A. Sims, Joseph D. Long, Emily C. Duggan, Neil Fernandes, Beate Illek, B. Gomperts, S. Jonas, Donald B. Kohn
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cervical cancer (CC) remains among the top causes of death for women worldwide, especially in low-income countries, where screening strategies are less widespread. Treatment strategies are mainly based on DNA-damaging agents, though resistance mechanisms still pose a substantial challenge. Among the cellular components that mediate treatment resistance, long non-coding RNAs (lncRNAs) stand out because of their broad regulatory effects. They are involved in virtually all drug resistance mechanisms, such as drug efflux, DNA repair, evasion of cell death, and aberrant epigenetic modifications. Although resistance mechanisms are fundamentally similar in most cancers, the underlying regulatory networks vary substantially. Here, we review the literature for lncRNAs involved in treatment resistance mechanisms in general, and then focus on lncRNAs that mediate resistance in CC. We found a broad area of opportunity in lncRNA research in resistant CC, as the lncRNAs involved are still to be described. These master regulators are promising candidates for response markers and therapeutic targets. May this compilation serve as the basis for further descriptions of the regulatory roles of lncRNA in CC treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38374134a894f6b5db043d7b741d396bd485cc0d" target='_blank'>
              Long noncoding RNAs involved in therapeutic response: implications for cervical cancer drug resistance
              </a>
            </td>
          <td>
            Samuel Trujano-Camacho, Carlos Contreras-Romero, Verónica García-Castillo, David Sánchez-Marín, Mercedes Olvera-Valencia, M. Rodríguez-Dorantes, O. Peralta-Zaragoza, David Cantú- De León, E. López-Urrutia, C. Pérez-Plasencia
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39534227ca18263af64802892a94b5211231b6b" target='_blank'>
              Failure of DNA repair leads to the tumorigenic transformation of a regenerating blastema in Drosophila imaginal discs
              </a>
            </td>
          <td>
            Tanroop Kaur, Ada Repiso, Ying Liu, Paula Doria, Joan Urgell, Luis Rodriguez-Escudero, Katerina Karkali, Yanhui Hu, Norbert Perrimon, E. Martín-Blanco
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy, with five-year survival rates below 50% largely due to late-stage diagnosis and the development of drug resistance. The advent of precision medicine offers new opportunities to improve patient outcomes. To date, molecular profiling has identified key therapeutic targets in ovarian cancer, including homologous recombination deficiency (HRD), BRCA mutations, and mismatch repair defects. PARP inhibitors have demonstrated significant clinical benefit in patients with HRD-positive tumors. Also, folate receptor alpha (FRa) targeting antibody drug conjugate, mirvetuximab soravtansine has demonstrated improved overall survival in platinum resistant ovarian cancer expressing high levels of FRa. To exploit DNA replication vulnerabilities in ovarian cancer cells, inhibitors of cell cycle regulators ATR and CHK1, have been studied in ovarian cancer preclinical and clinical settings. DHX9 helicase inhibition represents another novel approach, targeting RNA/DNA helicase activity critical for transcriptional regulation and DNA repair processes in cancer cells. Additional emerging targets include KIF18A that regulates mitotic progression and microtubule dynamics, which have demonstrated selective activity in chromosomally instable high-grade serous ovarian cancer cells. Novel therapeutic targets such as DHX9 helicase and KIF18A, offer new opportunities for patients with limited treatment options. Future directions include expanding biomarker testing, developing combination therapeutic strategies that exploit multiple vulnerabilities simultaneously, and addressing resistance mechanisms.



 Jung-Min Lee. Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c03eea4f891eb2bca142fb8a480f45ccc5bf2a" target='_blank'>
              Abstract IA018: Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Viruses are traditionally viewed as intracellular parasites that exploit host resources to propagate, often at the host’s expense. However, emerging evidence suggests more nuanced interactions, including potential mutualism. The L-A double-stranded RNA (dsRNA) virus, a non-lytic, cytoplasmic virus commonly found in Saccharomyces cerevisiae, presents an intriguing case, as it lacks clear pathogenic effects in the absence of its M1 satellite. To address how and why L-A persistently resides in its yeast host, we conducted a genome-wide screen using yeast deletion and temperature-sensitive (ts) mutant collections, covering approximately 93% of annotated genes, to identify host factors required for efficient L-A maintenance. This screen revealed 96 genes spanning diverse biological processes. Transcriptomic profiling indicated that L-A presence alters the host stress-response gene expression. Furthermore, competitive fitness assays under stress conditions demonstrated that L-A can enhance host resilience, suggesting a mutualistic relationship. Our findings uncover a previously unrecognized virus-host mutualism, wherein L-A benefits its host under environmental stress. These results not only expand our understanding of persistent viral infections in eukaryotes but also challenge the canonical view of viruses as purely parasitic, offering new insights into virus-host co-evolution and endogenous viral persistence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfce1fd2a8016b8a2e6031157a47fddaf4965a3c" target='_blank'>
              Genome-wide screen uncovers novel host factors for L-A virus maintenance and a potential mutualistic-symbiosis relationship in yeast
              </a>
            </td>
          <td>
            Wan-Yi Hsiao, Chung-Shu Yeh, Hsin-I Liu, Luh Tung, 
          </td>
          <td>2025-11-11</td>
          <td>BMC Microbiology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="SUMMARY Bacteroides fragilis occupies a dynamic position within the human gut. Though it comprises a relatively minor fraction of the gut microbiota, it is disproportionately enriched at extraintestinal sites of infection. This ability to survive in contrasting host environments pivots on a regulatory framework that is both modular and highly plastic. Rather than deploying a suite of hierarchical global regulators, B. fragilis employs numerous operon-embedded transcriptional switches, including site-specific DNA inversions, phase-variable epigenetic systems, extracytoplasmic function sigma/anti-sigma factor pairs, and hybrid two-component systems. These networks are further complemented by cis-regulatory elongation checkpoints and post-transcriptional control by small RNAs. This review explores the full spectrum of these regulatory mechanisms, highlighting how they facilitate niche adaptation, surface variation, immune evasion, and metabolic prioritization. It also explores intraspecies variation focusing on glycan metabolism, antibiotic resistance, and virulence. Additionally, it outlines recombination-driven regulation, alongside extracytoplasmic function sigma factor diversification, flexible promoter architecture, and elongation checkpoints, each contributing to the evolution of transcriptional control in B. fragilis. Finally, it outlines unanswered questions, including the largely unexplored sRNA regulon, the coordination of DNA inversions, elongation control, and phase-variable methylation, and proposes experimental strategies to investigate the integration of these regulatory systems during environmental transitions. Taken together, B. fragilis emerges as a model bacterium for studying decentralized gene regulation in complex microbial ecosystems, with implications for both microbial ecology and therapeutic targeting of the gut microbiota.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f20bfd3b213b02ddffd61d220bca702ce67a3e4a" target='_blank'>
              Gene regulation in Bacteroides fragilis: adaptive control in a dynamic host environment
              </a>
            </td>
          <td>
            Daniel Ryan
          </td>
          <td>2025-10-17</td>
          <td>Microbiology and Molecular Biology Reviews</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background/Aim: Cancer remains a leading cause of mortality globally, driven by complex molecular mechanisms and characterized by significant biological heterogeneity across cancer types. We aimed to discover mitogen-activated protein kinases (MAPKs) family members as both biomarkers and therapeutic targets in different cancer types. MAPKs are key signaling molecules regulating cell proliferation, differentiation, stress response, and apoptosis. Dysregulation of MAPK pathways has been implicated in the onset and progression of multiple cancers, contributing to tumor growth, metastasis, and therapeutic resistance. Given their diverse roles across cancer types, systematic analysis of MAPK gene expression, mutations, and interactions with tumor microenvironment is essential. Materials and Methods: The present study undertakes a comprehensive transcriptomic analysis of MAPKs in nine major cancer types using RNA-Seq datasets from The Cancer Genome Atlas (TCGA). RNA-Seq data were analyzed to identify differentially expressed MAPKs across nine cancer types using DESeq2, NOIseq, and limma. Significant genes (adjusted p<0.05) were subjected to GO and KEGG enrichment (EnrichR), mutational profiling (cBioPortal), and Kaplan-Meier survival analysis. Tumor immune infiltration was further assessed using TIMER2.0 to explore immune–gene interactions. Results: Through rigorous differential expression analysis, we identified key MAPKs that are significantly altered in liver hepatocellular carcinoma (LIHC) and lung adenocarcinoma (LUAD). Specifically, six MAPKs (MAPK3, MAPK7, MAPK9, MAPK10, MAPK12, and MAPK13) were found to be differentially expressed in LIHC, while MAPK6 emerged as the sole significant candidate in LUAD. Functional enrichment and pathway analysis revealed that these MAPKs are involved in critical oncogenic pathways, including MAPK-mediated transcriptional regulation and stress-activated signaling cascades. Mutational profiling and survival analysis further validated the prognostic significance of these genes, with several MAPKs showing strong associations with reduced patient survival. Tumor immune infiltration analysis indicated potential roles of these MAPKs in modulating immune responses within the tumor microenvironment. Also, MAPKs identified in this study are structurally related, suggesting that targeting them collectively may enhance therapeutic efficacy and overcome resistance mechanisms. Conclusion: Our integrated approach underscores the value of MAPK family members as both biomarkers and therapeutic targets in LIHC and LUAD. This study contributes important insights into MAPK-related oncogenic processes and supports the development of targeted therapies under the framework of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b14065bbb6005ef268d03b15d61fb336481cae" target='_blank'>
              Investigating the Roles of MAPKs as Precision Oncology Targets: A Multi-cancer Expression and Survival Study
              </a>
            </td>
          <td>
            Md Nayab Sulaimani, Prithvi Singh, Mohammad Umar Saeed, Sobia Zaidi, Afzal Hussain, M. Alajmi, Anas Shamsi, Ravins Dohare, M. Hassan
          </td>
          <td>2025-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, Anoop Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 IDH-mutant gliomas are typically slow growing on initial presentation, however selective pressure during conventional upfront treatment with genotoxic radiation and chemotherapy can fuel the evolution of sub-populations of tumor cells. These subclones promote recurrence of IDH-mutant gliomas that are less likely to benefit from standard treatments. Therefore, novel tumor-specific therapeutic strategies with non-overlapping mechanisms are needed for durable tumor control. IDH mutant cancer cells are known to reprogram their metabolism to sustain rapid proliferation, which generate potentially targetable vulnerabilities in canonical metabolic pathways. Using the patient-derived, WHO grade 3 oligodendroglioma line TS603, we screened a small molecule library of 240 metabolism-related compounds, including both FDA-approved and non- approved drugs, over a range of 10 concentrations (1nM-20µM). Drug efficacy was assessed after 5-day incubation via cell viability assay (Cell-Titer Glo). We identified 53 compounds with IC50<2µM, including drugs that target previously reported IDH-mutant glioma targets, like NAMPT and DHODH. Notable target pathways include folate metabolism and de novo thymidylate synthesis (methotrexate, raltitrexed, 5-FU), p97 AAA ATPase/VCP chaperone (CB-5083, ML240, eeyarestatin I), mevalonate pathway (Ro 48-8071 fumarate, atorvastatin calcium), and the BCL-2 family (Navitoclax, ABT-737). We are validating these drugs across a panel of patient-derived IDH-mutant gliomasphere models, and examining the combinatorial effects of these agents with vorasidenib and CDK4/6 inhibitors. Our results provide insight into important metabolic pathway dependencies in recurrent IDH1-R132H mutant gliomas that have targeting potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99af98af377416756584e34e3b8b2129cefb73a2" target='_blank'>
              DDDR-40. Identifying metabolic vulnerabilities of recurrent IDH1-R132H mutant glioma by high throughput screening
              </a>
            </td>
          <td>
            Yutong Wang, Sayali Daware, D. Cahill, Julie Miller
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f73f73f3e37aa93d53554d82182a30e140df7f94" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identifies CLK1 inhibition as a strategy to restore PARP inhibitor sensitivity via ERCC1 isoform switching.
              </a>
            </td>
          <td>
            Chaohua Liu, Xu Fei, Yutuan Wu, Li Jiana, Mengdong Ni, Xia Siyu, Chen Lihua, Haiyun Zhao, Yu Min, Zhou Yuqi, Meiqin Zhang, Jiajia Li, Xiaohua Wu, Huang Yan, Zhu Tao, Xiaojun Chen
          </td>
          <td>2025-11-04</td>
          <td>Protein & cell</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J. Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) is traditionally discussed in the context of horizontally acquired resistance genes and point mutations at target loci. However, this gene-centred model fails to account for a large number of clinically important modalities of resistance. There is now substantial evidence implicating bacteria in the ability to escape the effects of antibiotics in a variety of non-canonical ways, which are not considered in traditional diagnostic and surveillance pipelines. Among these factors, we can list those arising from global regulatory networks, phase variability, epigenetic tuning, small RNAs, genome structural variability, and phenotypic states like tolerance and persistence. This review will blend the current knowledge on these alternative pathways of resistance and underscore how they intersect with canonical genetic determinants. We will highlight cases where resistance emerges in the absence of known resistance genes, analyse the role of regulatory plasticity in efflux pump expression and membrane remodelling, and examine the contributions of bacterial stress responses and post-transcriptional control. Additionally, we will address methodological gaps in the detection of these mechanisms and their implications for clinical treatment failure, resistance surveillance, and drug development. By integrating insights from molecular microbiology, systems biology, and genomics, this review aims to offer a framework for understanding AMR as a multifaceted, context-dependent phenotype, not merely a genotype. We conclude by identifying knowledge gaps and suggesting priorities for research and diagnostic innovation in this evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cfbc4cd9dd576d2ed3bfaa746258988e167067" target='_blank'>
              Beyond Genes: Non-Canonical Mechanisms Driving Antimicrobial Resistance in Bacteria
              </a>
            </td>
          <td>
            Leonard Koolman, C. Emenike, Debasis Mitra, S. Chattaraj
          </td>
          <td>2025-10-01</td>
          <td>Bacteria</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, O. Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Antibiotic resistance remains one of the most formidable challenges to modern medicine, threatening to outpace therapeutic innovation and undermine decades of clinical progress. While resistance was once viewed narrowly as a clinical phenomenon, it is now understood as the outcome of complex ecological and molecular interactions that span soil, water, agriculture, animals, and humans. Environmental reservoirs act as silent incubators of resistance genes, with horizontal gene transfer and stress-induced mutagenesis fueling their evolution and dissemination. At the molecular level, advances in genomics, structural biology, and systems microbiology have revealed intricate networks involving plasmid-mediated resistance, efflux pump regulation, integron dynamics, and CRISPR-Cas interactions, providing new insights into the adaptability of pathogens. Simultaneously, the environmental dimensions of resistance, from wastewater treatment plants and aquaculture to airborne dissemination, highlight the urgency of adopting a One Health framework. Yet, alongside this growing threat, novel therapeutic avenues are emerging. Innovative β-lactamase inhibitors, bacteriophage-based therapies, engineered lysins, antimicrobial peptides, and CRISPR-driven antimicrobials are redefining what constitutes an “antibiotic” in the twenty-first century. Furthermore, artificial intelligence and machine learning now accelerate drug discovery and resistance prediction, raising the possibility of precision-guided antimicrobial stewardship. This review synthesizes molecular insights, environmental drivers, and therapeutic innovations to present a comprehensive landscape of antibiotic resistance. By bridging ecological microbiology, molecular biology, and translational medicine, it outlines a roadmap for surveillance, prevention, and drug development while emphasizing the need for integrative policies to safeguard global health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2665fac6c17a4051c07ada66840ae2833282cd2b" target='_blank'>
              Beyond the Resistome: Molecular Insights, Emerging Therapies, and Environmental Drivers of Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-10-01</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA mismatch repair (MMR) preserves genomic integrity by correcting replication errors. Deficiency in MMR (MMRd) results in microsatellite instability, increased tumor mutational burden, neoantigen generation, and activation of immune response. In this review, we first outlined how MMR loss promotes immune activation and responsiveness to immune checkpoint blockade (ICB), establishing MMRd status as the first tumor-agnostic biomarker for ICB therapy. Subsequently, we summarized the compelling evidence that defines MMR status as a dynamic, context-dependent process influenced by environmental and therapeutic pressures, rather than a fixed, binary trait. Accordingly, we discussed the implications of spatial and temporal heterogeneity of MMR status for both the diagnosis and treatment of cancer, the differential response of MMRd tumors to ICB, as well as the occasional benefits observed in MMR-proficient (MMRp) immune-cold cancers. We then explored strategies to exploit MMR dynamics and mimic MMRd-like phenotypes through alkylating agents, pharmacological MMR inhibition, and stress-mediated modulation, with the aim of sensitizing refractory tumors to immunotherapy. Finally, we reported emerging therapeutic opportunities in MMRd tumors, including Werner helicase inhibition, nonsense-mediated decay blockade, and neoantigen-targeted vaccines. Altogether, reframing MMR as a dynamic and targetable axis may broaden immunotherapy applicability and advance precision immune oncology across different tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6face00bfd040e4e182ff419f940662d44199ca5" target='_blank'>
              Mismatch Repair as a Dynamic and Clinically Actionable Vulnerability in Cancer.
              </a>
            </td>
          <td>
            Eleonora Piumatti, P. P. Vitiello, V. Amodio, A. Bardelli, Giovanni Germano
          </td>
          <td>2025-09-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Identification of tumor driver genes in cancer has led to the discovery of specific allelic variants that are useful as predictive or prognostic markers. In colorectal cancer (CRC), the K-Ras GTPase is one of the most mutated oncoproteins; therefore, understanding the functional consequences of its mutations will allow us to exploit allele-specific vulnerabilities and propose potential mechanisms of resistance in cancer therapeutics. In this study, cellular phenotypic consequences of the K-Ras mutants K117I, K117N, A146T, E168K, and K172T identified in Filipino young-onset CRC (YO-CRC) patients were assessed in HCT116, Caco-2, and NIH3T3 cells. In assays assessing DNA doubling in 2D cultures and viability in 3D multicellular spheroids, only K117I, K117N, and A146T exhibited enhanced proliferation and supported a tumor-like property of increased size and viability. All mutants exhibited increased cellular migration in HCT116 and NIH3T3 but not in Caco-2 cells. None of them caused resistance to apoptosis. Compared to wild type, K117I, K117N, and A146T increased Elk1 reporter activation as well as phospho-Erk and phospho-Akt levels. All mutants induced extensive cytoskeletal remodeling including formation of migratory and invasive structures. Finally, GDP docking simulations suggested increased nucleotide exchange as a mechanism of activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d480b5f5aee202dd438c3fe6977210301750ccd6" target='_blank'>
              Differential effects of KRAS G-domain and hypervariable region mutants on cancer phenotypes
              </a>
            </td>
          <td>
            J. A. D. de Borja, Kim Denyse J. Hao Lin, Daniel Angelo R. Mirador, Alexandra Isabelle D. Ang, Sidney Allen A. Chua, Joanne Marie D. Sytangco, D. Sacdalan, Reynaldo L. Garcia
          </td>
          <td>2025-11-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary malignant tumor of bone and occurs predominantly in children and adolescents during periods of accelerated skeletal growth. Within the tumor microenvironment (TME), vascular smooth muscle cells (VSMCs) may acquire osteogenic characteristics. Nuclear factor of activated T cells c1 (NFATc1) is central to skeletal biology, yet its behavior in VSMCs in osteosarcoma remains insufficiently defined. We profiled fresh osteosarcoma tissues by single-cell RNA sequencing on the 10x Genomics Chromium platform. Mechanical and enzymatic dissociation yielded single-cell suspensions; viable cells (> 85%) were enriched; libraries were prepared with Single Cell 3′ v3 reagents and sequenced on NovaSeq 6000. Reads were processed with Cell Ranger (GRCh38). Following log-normalization, 2,000 highly variable genes were retained. Harmony integration reduced batch structure. We computed 50 principal components, generated UMAP/t-SNE embeddings, and applied k-nearest neighbor/Louvain clustering (k = 20; resolution = 0.5). Cell types were assigned from differentially expressed markers cross-checked with CellMarker and PanglaoDB. NFATc1 distribution and co-expression patterns were assessed by Pearson correlation. Nine major cell classes were identified, including VSMCs, fibroblasts, macrophages, T/NK subsets, B cells, neutrophils, mast cells, and inflammatory myeloid cells. NFATc1 expression was most prominent in VSMCs, with a bimodal distribution within this lineage. In VSMCs, NFATc1 co-varied with SERGEF, APOE, and CCS, pointing to coordinated cytoskeletal remodeling, lipid handling, and copper homeostasis during osteogenic reprogramming. At single-cell resolution, NFATc1 marks a VSMC fraction with osteogenic features in osteosarcoma. The associated network suggests testable links to metabolism and matrix remodeling; functional perturbation will be required to establish causality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3638bf4253781f5bd9cf49aea68da6335438a6" target='_blank'>
              Single-cell RNA sequencing reveals NFATc1-mediated regulatory mechanisms of vascular smooth muscle cell osteogenic transformation in the osteosarcoma microenvironment
              </a>
            </td>
          <td>
            Xuejiao Yi, Shilin Fu, Chunliang Zhang, Tianle Zhao, Huan Feng, Dehui Qian
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms. Cancer cell plasticity is often linked to treatment failure. Here, the authors discuss the role of epigenome in DNA repair and, how DNA repair-induced chromatin changes influence cancer cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) are emerging as important regulators of gene expression in both normal biological systems and disease. Among them, metastasis-associated lung adenocarcinoma transcript 1 (MALAT1) has attracted considerable attention due to its high nuclear expression and evolutionary conservation. It was first identified as a biomarker for lung cancer metastasis, but since then, its involvement has been reported in a wide range of cancers. MALAT1 influences several key cancer-related processes, including cell proliferation, migration, angiogenesis, apoptosis, epithelial–mesenchymal transition, and the behavior of cancer stem cells. In this review, we take a closer look at how MALAT1 functions through different mechanisms, such as regulating RNA splicing, altering chromatin states, acting as a sponge for microRNAs, or modifying protein interactions, to drive these changes. We also discuss its growing role in shaping the immune landscape of tumors, particularly its involvement in immune evasion, inflammatory signaling, and regulation of immunogenic cell death. In addition, MALAT1 contributes to resistance against therapy, rewiring of cellular metabolism, and communication between tumor and stromal cells via exosomes. While many studies describe MALAT1 as an oncogene, others suggest it may also play a suppressive role depending on the cancer type and biological context. Here, we aim to offer a comprehensive overview of MALAT1’s diverse actions and evaluate its potential as a diagnostic marker and therapeutic target in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73cb56fb6ad82b5646d4f05c255a6ee44614bc12" target='_blank'>
              MALAT1 as a molecular driver of tumor progression, immune evasion, and resistance to therapy
              </a>
            </td>
          <td>
            A. Bitaraf, Alireza Zafarani, Pardis Jahandideh, Benyamin Hakak-Zargar, A. Haghi, Golareh Asgaritarghi, S. Babashah
          </td>
          <td>2025-10-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
This review explores the relationship between genotypes and resistant phenotypes in Gram-negative pathogens. We analyse to what extent conventional phenotypic methods predict genetic mechanisms of resistance, the reliability of genotypic approaches, and how integrated strategies may improve diagnostic accuracy and clinical utility.


RECENT FINDINGS
Traditional AST remains the clinical reference standard due to its correlation with therapeutic outcomes, yet it often fails to identify the molecular basis of resistance. Molecular methods such as PCR, microarrays, and targeted sequencing allow rapid detection of known genes but cannot reliably predict expression or functionality. Whole-genome sequencing provides the most comprehensive overview, capturing both known and novel resistance determinants as well as mobile genetic elements. Nevertheless, genotype-phenotype discordance persists, driven by regulatory mutations, inducible expression, or synergistic mechanisms. Emerging technologies - including real-time sequencing, metagenomics, and machine learning-based predictive models - are enhancing our ability to infer phenotypes from genomic data. Still, these approaches face challenges of standardization, validation, and integration into clinical workflows.


SUMMARY
Linking genotypes to resistant phenotypes in Gram-negative pathogens remains complex. While phenotypic AST ensures reliability for therapy, genotypic methods provide unprecedented insight into resistance mechanisms and epidemiology. Discrepancies between the two highlight the need for integrated diagnostic platforms that combine functional and genomic perspectives. Artificial intelligence-driven predictive models and curated resistance databases hold promise for improving accuracy, but widespread adoption requires robust datasets, clinical validation, and harmonized interpretative frameworks. Ultimately, integrating phenotypic and genotypic data represents the most effective strategy to provide mechanism-informed, clinically actionable diagnostics for antimicrobial resistance management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f34c2b73b9d73ae3113ba2142013f1f7e8cf759a" target='_blank'>
              Is it possible to identify genotypes underlying resistant phenotypes in Gram-negative pathogens?
              </a>
            </td>
          <td>
            A. Burillo, Julia Serrano-Lobo, Emilio Bouza, Patricia Muñoz
          </td>
          <td>2025-10-14</td>
          <td>Current opinion in infectious diseases</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background and Objectives: Changes in the environment and physiology may be associated with an increased or decreased risk of cancer. Our study aims to evaluate the strength and the direction of the selection acting on oncosuppressor genes in association with phenotypic changes. Methods: We calculated the relative evolutionary rate (RER) using the converge method and linear regression on branches of phylogenetic trees. The association between changes in the evolutionary rate of oncosuppressors (DNA repair and cell cycle control genes) and trait selection was studied. The evolutionary rates of single oncosuppressor genes and pathways were evaluated. We studied two types of traits: those that are characteristic of vertebrates, such as homeothermy (the ability to maintain a constant body temperature), flight, and amnions; and those that are characteristic of mammals, such as high body mass and lifespan, an underground lifestyle, and hibernation. The analysis included 19,445 genes; 100 vertebrates and 46 mammalian species. We studied ancestral branches individually and all the clades having a trait. Results: Oncosuppressor genes accelerated in association with the ability to fly; p-value = 0.03 (positive or relaxed negative selection) and decelerated in homeothermic species; p-value = 0.04 (stabilizing selection). DNA repair genes were significantly accelerated in ancestral branches and in all clades of amniotic, homeothermic, and high-body-mass mammals (p-value < 0.05, FDR correction). Cell cycle control genes were under stabilizing selection in homeothermic animals, high-body-mass, long-lived, and underground mammals (p-value < 0.05, FDR correction). Data on the evolution of oncosuppressors are crucial for understanding the origin of cancer and will be important for future studies of tumor pathogenesis, pathomorphosis, and microevolution. Conclusions: The selection of traits associated with changes in cancer risk leads to positive/relaxed negative and stabilizing selection of oncosuppressor genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/109805551d7170219d8ed5f7db0bf73cea1a8aba" target='_blank'>
              Evolutionary Dynamics of Oncosuppression Under Selection Pressure
              </a>
            </td>
          <td>
            M. Potievskiy, P. Shatalov, I. Klabukov, D. Atiakshin, Anna O. Yakimova, Denis S. Baranovskii, Peter V. Shegai, Andrey D. Kaprin
          </td>
          <td>2025-10-01</td>
          <td>Life</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9e3d7e8367b6848517e2b91b0fabe3f4c124a0" target='_blank'>
              Identification of a novel PML exon 6 splice variant in atypical PML::RARα transcripts in acute promyelocytic leukemia
              </a>
            </td>
          <td>
            Haimin Chen, Meihong Chen, Zhongjie Yang, Shuzhen Liao, Yun Lin, Linlin Yan
          </td>
          <td>2025-09-26</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The endosomal pathway plays a crucial role in eukaryotes, and its conservation across evolution underscores its importance in fundamental processes, such as endocytosis, cytokinesis, and autophagy. Dysregulation of the Notch signaling system, essential for growth and development, contributes to cancers like leukemia. In Drosophila melanogaster (DM), specific endosomal genes function as tumor suppressors, regulating Notch signaling, either through promoting Notch degradation or directing its recycling. Nevertheless, it is still unanswered how endosomal genes specifically, lethal giant discs (lgd/CC2D1A), affect Notch-1-mediated transduction in human malignancies, such as T-cell acute lymphoblastic (T-ALL). This study aimed to comprehensively characterize the (lgd/CC2D1A) CC2D1A/lgd in human T-ALL by performing mutational analysis and sequencing and evaluating its gene and protein expression levels. These analyses were conducted to elucidate better the role of (lgd/CC2D1A)CC2D1A in the pathophysiology of T-ALL and to explore its potential association with the Notch-1 signaling pathway. In this study, we sequenced the lgd and investigated its expression in peripheral blood samples of human T-ALL samples. We used molecular approaches to analyse the Lgd/CC2D1A gene for mutations and characterize its gene and protein expression across genomic and proteomic datasets. Sequencing data revealed the presence of the lgd variant (CC2D1A) in T-ALL patients and healthy individuals. Lgd and Notch-1 expression were significantly upregulated in T-ALL patients, demonstrating a strong positive correlation at both the genomic and proteomic levels. Through this analysis, we reported the presence of Lgd with an altered nucleotide sequence in T-ALL patients. We also found its elevated expression compared with healthy controls, which might affect disease progression. Our investigation results also lay the foundation for the possible development of Lgd as a predictive indication for the early identification of T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b8e3e49ce5c7f6dd3ebfcedabaf637bb4da449" target='_blank'>
              Characterization, expression, and mutational analysis of endosomal protein lethal giant disc (Lgd/CC2D1A) as a biomarker in patients with T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sandeep Pandey, Ranjana Singh, Nimra Habib, Rashmi Kushwaha, A. A. Mahdi
          </td>
          <td>2025-11-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bb8c6ba0429e042acd5b6b051c6318cb9fb300" target='_blank'>
              Identification, functional insights and therapeutic targeting of EMT tumour states.
              </a>
            </td>
          <td>
            Anqi Dong, Cédric Blanpain
          </td>
          <td>2025-09-25</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mismatch repair (MMR) system alterations can trigger transient hypermutation, promoting adaptive mutations under stress, such as antibiotic exposure. While most organisms use MutS and MutL protein families for MMR, many archaea and actinobacteria, including the major human pathogen Mycobacterium tuberculosis, lack these components and instead rely on NucS, a structurally distinct enzyme driving a non-canonical MMR pathway. Given the role of MMR in mutation control, understanding how nucS expression is regulated could be essential for uncovering the molecular basis of antibiotic resistance development in mycobacteria. In this study, we characterized the nucS promoter and transcription start site in Mycobacterium smegmatis. We found that nucS expression declines during the stationary phase in both M. smegmatis and M. tuberculosis, paralleling replication activity and canonical MMR downregulation. Our data suggest that the alternative sigma factor σB may negatively regulate nucS expression during this phase. Additionally, we identified candidate compounds that may modulate nucS expression, underscoring its responsiveness to environmental cues. These findings enhance our understanding of mycobacterial stress responses and lay the groundwork for exploring antibiotic resistance mechanisms. Strikingly, our work reveals a case of double convergent evolution: both canonical (MutS/MutL) and non-canonical (NucS) pathways have independently evolved not only the same DNA repair function, but also similar regulatory frameworks for genome integrity preservation under stress conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7d5cf01753d5a2e85e0726bcedfc7ae6cce138b" target='_blank'>
              Regulation of the Expression of nucS, a Key Component of the Mismatch Repair System in Mycobacteria
              </a>
            </td>
          <td>
            E. Cebrián-Sastre, Ángel Ruiz-Enamorado, A. Castañeda-García, Susanne Gola, Pablo García-Bravo, Leonor Kremer, Jesús Blázquez
          </td>
          <td>2025-10-24</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c3621c150005a31c64ba8bfb331db8c9802b362" target='_blank'>
              Assessing the molecular and phenotypic contribution of a horizontally acquired region to yeast adaptation
              </a>
            </td>
          <td>
            Andres Romero, Camila Bastías, Matteo De Chiara, Xanita Saayman, Hajar Cherkaoui, Benjamin P. Barré, Francisco A Cubillos, C. Martínez, Eduardo I. Kessi-Pérez, Gianni Liti, Francisco Salinas
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90dbf82e0a483cd704102a3b6a055d355e22f224" target='_blank'>
              A mathematical model of drug-resistant cancer with cell cycle structure: stability, optimal control, and cost-effectiveness analysis
              </a>
            </td>
          <td>
            Tawfik Jaber, O. Balatif
          </td>
          <td>2025-09-29</td>
          <td>International Journal of Dynamics and Control</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b73fae020e6752c501ebd1de56b61a2010abe7" target='_blank'>
              Cancer metabolism: bridging tumorigenesis mechanisms to treatment susceptibility.
              </a>
            </td>
          <td>
            Jian Liu, Dan Shan, Zhaokai Zhou, Xutao Wen, Rao Fu, Bo Xu, Peng Luo, Zhengrui Li, Ling Zhang
          </td>
          <td>2025-09-28</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mutations in RNA-binding proteins are increasingly identified in cancers through tumor sequencing and are correlated with disease progression, therapy response, and overall patient outcomes, underscoring the need to study them. Here, we focus on the RNA-binding protein Poly-C binding protein 1 (PCBP1), which binds target RNAs through K-homology (KH) domains to regulate RNA fate. PCBP1 is a tumor suppressor gene and hotspot missense mutations at leucine residues 100 and 102 are observed in colorectal cancer (CRC). PCBP1 mutations have been recurrently reported in CRC genome-wide mutation studies and are associated with poor clinical outcomes; however, their effects on PCBP1 expression and function remain largely unexplored. We show that cancer-associated mutations substituting leucine 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover. The L100/L102 residues occur at the interface of the RNA-binding KH1 and KH2 domains, and our molecular dynamics simulations show that mutations at these residues disrupt the secondary structure of PCBP1. Additionally, these mutants display increased cytoplasmic localization. Importantly, mutant PCBP1 physically interacts with wild type PCBP1 and suppresses its expression through a dominant-negative mechanism. Together, our data demonstrate that CRC-associated PCBP1 mutations destabilize the protein and act as dominant-negative variants, revealing a novel mechanism of tumor suppressor inactivation in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a7a507c9272e7ca6986d3ffe683c348245fff39" target='_blank'>
              Colorectal cancer-associated PCBP1 mutations disrupt protein stability in a dominant negative manner
              </a>
            </td>
          <td>
            Paige V. Blinkiewicz, Nicole C. Hoenes, Myra X. Afzal, Yihong Liu, Ulysses J. Hill, B. Bjarnadóttir, T. E. Burgin, P. Malaney
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The biological mechanisms driving the long survival in glioblastoma (GBM). Five‐year long‐term survival (LTS) and 10‐year survival very long‐term survival (VLTS) remain significantly understudied. Here we molecularly detailed two cases. AR10‐046 (VLTS) was affected by a giant cell‐GBM, classified as the pedHGG_RTK1a subtype according to the v12.5 Heidelberg brain tumor methylation classifier. Somatic and germline MSH6 mutations, typically in Lynch syndrome, and high tumour mutational burden were detected. The copy number variation plots showed chromosome 1q gain and chromosome 13 loss with no other typical GBM alterations. AR10‐037 (LTS) suffered from a classical GBM, identified as pedHGG_MYCN subclass. Apart from the canonical chromosome 7 gain and chromosome 10q loss, we observed MDM2 gene amplification and possible rearrangements on chromosome 12 and 18 with the typical aspect of chromothripsis, harbouring two putative new gene fusions: CPSF6::CPM and PTPRR::RAB3IP. We described two patients with peculiar tumour molecular profile, widening the scenario of clinical and molecular variability in such patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8551ad58743a80e7a359a8ec1bcd9af76ef1c7" target='_blank'>
              Unique Genetic and Epigenetic Alterations in Glioblastoma Long‐Term Survivors: Insights From Two Clinical Cases
              </a>
            </td>
          <td>
            E. Anghileri, E. Miele, S. Patrizi, S. Barresi, E. Lazzarini, L. Maddaloni, M. Patanè, L. Pedace, R. Paterra, A. Silvani, Franco Locatelli, S. Indraccolo, B. Pollo
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Despite significant advances in cancer research, the complexity of signaling pathways remains a major challenge in precision oncology. Tumors harbor a diverse array of genetic alterations; however, these often converge onto a limited set of core signaling pathways, notably RAS/RAF/MEK/ERK (MAPK), PI3K/AKT/mTOR (PAM), and Wnt/β-catenin. Concurrently, these pathways diverge extensively downstream, driving therapeutic resistance through mechanisms such as epithelial-mesenchymal transition, immune evasion, and metabolic reprogramming. Unlike previous literature that largely provides descriptive accounts of pathway alterations, this review uniquely synthesizes convergent and divergent signaling into a clinically actionable diagnostic and therapeutic framework. It critically assesses current precision oncology strategies, identifies gaps, and proposes a dual-axis model that integrates static genomic profiling with dynamic signaling evolution to inform precision therapy. By highlighting opportunities for combination therapy informed by pathway interdependencies and adaptive resistance mechanisms, this perspective provides novel clinical insights and tangible directions for future research in overcoming resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f096cbac949c4b15346a048f653260da1f8e6" target='_blank'>
              Convergent and Divergent Signaling Pathways in Cancer: A Dual-Axis Model for Adaptive Precision Oncology.
              </a>
            </td>
          <td>
            Aliasgar Shahiwala PhD
          </td>
          <td>2025-10-08</td>
          <td>Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA replication, as the foundation of genetic information transmission and the continuity of life, relies on high fidelity and dynamic regulatory mechanisms that are central to maintaining genome stability. Through literature analysis, this review systematically summarizes the core molecular mechanisms of DNA replication initiation, elongation, proofreading, and repair, while comparing the commonalities and differences between prokaryotic and eukaryotic systems. Comparative analysis demonstrates that while prokaryotic and eukaryotic DNA replication share conserved high-fidelity principles, they exhibit fundamental mechanistic divergences. Advances in technologies such as cryo-electron microscopy have uncovered the structure and dynamics of the replication machinery and its coupling with epigenetic regulation. Future research should employ integrative multi-omics approaches to decipher the spatiotemporal regulation of replication, facilitating therapeutic interventions for replication-associated pathologies including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951febf5def0fd8aa3519a39b3eb248cfca4ae46" target='_blank'>
              DNA Replication and the Transmission of Genetic Information: A Molecular Mechanism Perspective
              </a>
            </td>
          <td>
            Ziying Wang
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Aim: Cisplatin resistance in head and neck squamous cell carcinoma (HNSCC) is thought to involve both reduced drug uptake and altered molecular responses. However, the relative contribution of these mechanisms remains unclear. Methods: Two HNSCC cell lines with differing sensitivity (HNO97 and HNO41) were analyzed using cytotoxicity assays, atomic absorption spectroscopy-based quantification of intracellular cisplatin, caspase 3/7 assays, Western blotting, polymerase chain reaction (PCR)-based transcriptomic analysis of DNA damage response and cell cycle arrest pathways, and RNA-seq data from The Cancer Genome Atlas (TCGA) to characterize the resistance phenotype. Results: HNO97 (IC50 = 440 µM) was 7.6-fold more resistant to cisplatin than HNO41 (IC50 = 57.8 µM; P = 0.0286). After quantifying intracellular uptake (pg Pt/µg protein) and normalizing cytotoxicity to intracellular drug levels, HNO97 (IC50 = 778.9 pg Pt/µg protein) remained 5-fold more resistant than HNO41 (IC50 = 153.5 pg Pt/µg protein), indicating only a partial reduction in resistance (33% decrease, from 7.6-fold to 5-fold; P = 0.0286). At cisplatin concentrations yielding comparable intracellular exposure (HNO97: 440 µM; HNO41: 196 µM; both ≈ 725 pg Pt/µg protein), caspase 3/7 activation and induction of CDKN1A, GADD45A, GADD45G, and PPP1R15A were weaker in HNO97 than in HNO41. Notably, baseline expression of these genes was significantly higher in HNO97. In the TCGA cohort, multivariate analysis showed that high FANCD2 expression was associated with unfavorable recurrence-free survival in platinum-treated patients (hazard ratio = 4.0; P = 0.011), but not in those who did not receive platinum chemotherapy. Conclusion: Cisplatin resistance in HNSCC appears to be driven primarily by molecular mechanisms involving DNA damage response and cell cycle arrest pathways, rather than poor drug uptake.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba27336a14cf094e2bc0e409273ca8370a971da1" target='_blank'>
              Cisplatin resistance in head and neck squamous cell carcinoma is linked to DNA damage response and cell cycle arrest transcriptomics rather than poor drug uptake
              </a>
            </td>
          <td>
            Ketaki Sandu, R. Warta, Uddipta Biswas, Wang Zhang, P. Michl, Christel Herold-Mende, Johanna Weiss, Dirk Theile
          </td>
          <td>2025-09-19</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Mutational activation of the KRAS oncogene in pancreatic ductal adenocarcinoma (PDAC) plays a crucial role in tumor initiation, progression, and alterations of cellular programs that support PDAC growth. Mutant KRAS alters the metabolic state of PDAC and has been linked to upregulation of glycolysis and alterations in the nutrient scavenging pathways autophagy and macropinocytosis. Our lab determined that genetic suppression or inhibition of the RAS pathway unexpectedly resulted in upregulation of autophagy. Additionally, concurrent inhibition of the RAS-MAPK pathway and autophagy synergistically improved anti-tumor efficacy in both PDAC cell lines, organoids, and PDX tumor models. To begin to address the temporal regulation of metabolic signaling following RAS inhibition, we examined the sustainability of upregulated autophagic flux following prolonged (168 h) RAS inhibition. Preliminary data indicates that autophagy upregulation following KRAS inhibition is transient and returns to basal levels following prolonged inhibitor treatment. Given the limited durability of autophagic induction upon RAS inhibition, we sought to determine whether other nutrient scavenging pathways that culminate in the lysosome were altered following prolonged RAS inhibitor treatment. Macropinocytosis is upregulated in PDAC and is RAS-dependent. We monitored macropinocytic uptake following direct inhibition of RAS, and found that following an initial decrease in macropinocytosis, PDAC cells exhibited a compensatory increase in macropinocytotic uptake and utilization at later timepoints. Based on this data, we have begun to interrogate the metabolic phenotypes associated with cells subjected to acute (24-72 h) and prolonged (168 h) RAS inhibitor treatment. Furthermore, we have also derived PDAC cell lines that are resistant to RAS inhibition via prolonged inhibitor dose escalation. We found that RAS inhibitor resistant cell lines have heterogenous capacities for autophagic flux and retain similar sensitivity to autophagy inhibitors. Conversely, resistant cells display persistently increased macropinocytosis, suggesting that that scavenging pathway compensates for metabolic requirements of growth under sustained RAS inhibition. Future studies are aimed at investigating the molecular underpinnings driving upregulated macropinocytosis and more broadly profiling the metabolic landscape of RAS-inhibitor resistant PDAC models.



 Sarah E. Ackermann, Ryan Robb, Khalilah E. Taylor, Runying Yang, Kirsten L. Bryant. Assessing metabolic requirements for RAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f9b16a108ea5a5b93f12330dee22b394bce19" target='_blank'>
              Abstract A006: Assessing metabolic requirements for RAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Sarah E Ackermann, Ryan Robb, Khalilah E Taylor, Run-Wei Yang, Kirsten L. Bryant
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary Many efforts have been made to detect pathogenic mutations in cell-free circulating DNA (cfDNA) derived from blood, which offers a less invasive and more practical alternative to tissue biopsy. However, the low abundance of mutated DNA in blood samples makes detection by high-throughput sequencing challenging. In this study, we developed a highly sensitive CRISPR/Cas12a-based diagnostic system that selectively removes wild-type (WT) DNA to enrich rare mutant DNA, enabling reliable mutation detection through PCR. Applying this approach to cfDNA from 48 non-small-cell lung cancer (NSCLC) patients, we targeted two clinically meaningful EGFR mutations: L858R and exon 19 deletion. Our system successfully detected all 7 L858R-positive cases and 6 of 11 exon 19 deletion-positive cases, consistent with prior tissue biopsy results. These findings demonstrate that our CRISPR/Cas12a-based system enables sensitive and selective detection of low-frequency mutations in cfDNA, supporting its potential use in non-invasive cancer diagnostics. Abstract Background/Objectives: Liquid biopsy using cfDNA has emerged as a promising, minimally invasive alternative to traditional tissue biopsy for detecting cancer-associated mutations. However, the extremely low proportion of mutant DNA in cfDNA poses a major challenge for accurate detection, especially when using conventional sequencing methods. To address this limitation, we sought to develop a highly sensitive diagnostic strategy to selectively enrich rare mutant sequences and improve the detection of clinically important mutations in patients with NSCLC. Methods: We established a CRISPR/Cas12a-based diagnostic system designed to selectively cleave WT DNA, thereby increasing the relative abundance of mutant DNA in cfDNA samples. Following Cas12a-mediated WT cleavage, the remaining DNA was subjected to PCR amplification for mutation identification. The system was applied to plasma cfDNA from blood samples of 48 NSCLC patients to evaluate its ability to detect two major EGFR mutations: L858R and exon 19 deletion. Results: The CRISPR/Cas12a-based diagnostic system effectively identified low-frequency EGFR mutations in cfDNA. Specifically, all 7 L858R-positive samples and 6 out of 11 samples harboring exon 19 deletions—previously validated through tissue biopsy—were successfully detected. This demonstrated a high degree of concordance between our liquid biopsy approach and conventional diagnostic methods. Conclusions: Our findings highlight the potential of the CRISPR/Cas12a-based mutation enrichment system as a powerful tool for detecting rare oncogenic mutations in liquid biopsy samples. This technique enhances diagnostic sensitivity and could be broadly applicable for the non-invasive detection of various genetic alterations in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/068b8935c94cdd53865ee3f28471c9d10867ede9" target='_blank'>
              CRISPR with a Double Mismatch Guide RNA Enhances Detection Sensitivity for Low-Frequency Single-Base EGFR Mutation in Circulating Cell-Free DNA of Lung Cancer Patients
              </a>
            </td>
          <td>
            K. W. Been, Seunghun Kang, Taegeun Bae, Sumin Hong, Garyeong Kim, J. Hur, Woochang Hwang, Boksoon Chang
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the second leading cause of cancer deaths, with a five-year survival rate of 65%. Oxaliplatin was the first platinum drug shown to improve CRC patient outcomes and is now a common adjuvant therapy for advanced disease, yet 90% of patients develop resistance. Oxaliplatin was developed as a third-generation derivative of cisplatin, but recent evidence points to divergent modes of action. Genome-wide CRISPR activation and knockout screens were conducted to identify genetic changes that confer resistance to oxaliplatin in two CRC cell lines with distinct molecular backgrounds (SW620 and RKO). Guide RNAs corresponding to the neutral amino acid transporter SLC43A1 (LAT3) were the most significantly enriched in knockout screens and depleted in activation screens, suggesting a potential role for LAT3 in modulating oxaliplatin resistance. In vitro CRISPR knockout and overexpression of LAT3 in SW620 and RKO cell lines confirm increased resistance or sensitivity to oxaliplatin, respectively, and this correlated with levels of intracellular accumulation of oxaliplatin and DNA damage, demonstrating that enhanced LAT3-mediated uptake of oxaliplatin can exert its expected mechanism of action and induce cytotoxicity. Further studies to investigate the specificity of this mechanism of resistance to oxaliplatin in CRC and other cancers are ongoing to clarify the molecular basis of oxaliplatin resistance. These findings suggest a novel use for LAT3 as a potential biomarker and predictor of oxaliplatin sensitivity and can provide insights into the interplay between essential nutrient uptake and drug transport.



 Nisha R. Pawar, Heidi M. Wade, Allison V. Mitchell, Connor P. Jewell, Lisa M. Jenkins, Robert W. Robey, Michael M. Gottesman. A whole genomic CRISPR-Cas9 screen identifies the LAT3 (SLC43A1) neutral amino acid transporter as a major determinant of oxaliplatin resistance in colorectal cancer cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb9e1d72d03ca8da071d5c00ee84e7d75c353cf" target='_blank'>
              Abstract A039: A whole genomic CRISPR-Cas9 screen identifies the LAT3 (SLC43A1) neutral amino acid transporter as a major determinant of oxaliplatin resistance in colorectal cancer cells
              </a>
            </td>
          <td>
            Nisha R Pawar, Heidi M Wade, Allison V. Mitchell, Connor P. Jewell, Lisa M. Jenkins, Robert W. Robey, Michael M. Gottesman
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-10</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Stenotrophomonas maltophilia (S. maltophilia) is a multidrug-resistant opportunistic pathogen. There are an increasing number of case reports on S. maltophilia infections in recent years, and the species is becoming a public health concern. Many studies have focused on profiling and pangenome of the species, particularly on their antibiotic resistance and virulence genes. However, there is a lack of studies on mobile genetic elements (MGEs), a subset of pangenome that significantly contributes to the diversity, stability, and plasticity of a population. In this study, 20 genomes of S. maltophilia were downloaded from the NCBI Genome database. The genomes were subjected to profiling of MGEs, their impact on the population structures, and the evaluation of evolutionary trends of the core genomes. The cataloguing of MGEs indicated active horizontal gene transfer events in the S. maltophilia’s population. Multiple virulence and drug resistance genes were predicted within and outside of the MGEs. We observed multiple chromosomal rearrangements in the genomes, most likely caused by MGEs, affecting up to approximately 50% of a single genome sequence. A high number of linkage disequilibrium sites were also predicted in the core genomes. This study provides insights into stability in the core and plasticity in the accessory regions in the S. maltophilia population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e411d02f608f82241ac3e80c32d842463977ee" target='_blank'>
              Comparative Analyses Suggest Genome Stability and Plasticity in Stenotrophomonas maltophilia
              </a>
            </td>
          <td>
            Danny Khar Chen Sum, Yee Yee Chong, Joon Liang Tan
          </td>
          <td>2025-10-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematopoietic stem cell (HSC) therapy remains essential in treating blood disorders, autoimmune diseases, neurodegenerative conditions, and cancers. Despite its potential, challenges arise from the inherent heterogeneity of HSCs and the complexity of their regulatory niche. Recent advancements in single-cell RNA sequencing and chromatin accessibility sequencing have provided deeper insights into HSC markers and chromatin dynamics, highlighting the intricate balance between intrinsic and extrinsic regulatory mechanisms. Zebrafish models have emerged as valuable tools in HSC research, particularly through live imaging and cellular barcoding techniques. These models have allowed us to describe critical interactions between HSCs and embryonic macrophages, involving reactive oxygen species and calreticulin signaling. These are essential for ensuring HSC quality and proper differentiation, with implications for improving HSC transplant outcomes. Furthermore, the review examines clonal hematopoiesis, with a focus on mutations in epigenetic regulators such as DNMT3A, TET2, and ASXL1, which elevate the risk of myelodysplastic syndromes and acute myeloid leukemia. Emerging technologies, including in vivo cellular barcoding and CRISPR-Cas9 gene editing, are being investigated to enhance clonal diversity and target specific mutations, offering potential strategies to mitigate these risks. Additionally, macrophages play a pivotal role in maintaining HSC clonality and ensuring niche localization. Interactions mediated by factors such as VCAM-1 and CXCL12/CXCR4 signaling are crucial for HSC homing and the stress response, opening new therapeutic avenues for enhancing HSC transplantation success and addressing clonal hematopoiesis. This review synthesizes findings from zebrafish models, cutting-edge sequencing technologies, and novel therapeutic strategies, offering a comprehensive framework for advancing HSC biology and improving clinical outcomes in stem cell therapy and the treatment of hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6985cdf565a94b5e4ce65dadd2a9ba4efa0f62f7" target='_blank'>
              Advancements in Hematopoietic Stem Cell Therapy: From Biological Pathways to Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Viviana Cortiana, H. Chorya, R. H. Abbas, Jade Gambill, Adhith Theyver, Chandler Park, Yan Leyfman
          </td>
          <td>2025-10-15</td>
          <td>Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Mutations may cause changes in the structure and function of immune-related proteins, thereby affecting the operation of the immune system. Deep mutational scanning combines saturation mutagenesis, functional selection, and high-throughput sequencing to evaluate the effects of mutations on a large scale and with high resolution. By systematically and comprehensively analyzing the impact of mutations on the functions of immune-related proteins, the immune response mechanism can be better understood. However, each stage in deep mutation scanning has its limits, and the approach remains constrained in several ways. These include data and selection biases that affect the robustness of effect estimates, insufficient library coverage and editability leading to uneven representation of sites and alleles, system-induced biased signals that deviate phenotypes from their true physiological state, and imperfect models and statistical processing that limit extrapolation capabilities. Therefore, this technology still needs further development. Herein, we summarize the principles and methods of deep mutational scanning and discuss its application in immunological research. The aim is to provide insights into the broader application prospects of deep mutational scanning technology in immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fab3ce7657843d2dcbc129fae5b583a583f6457" target='_blank'>
              Deep Mutational Scanning in Immunology: Techniques and Applications
              </a>
            </td>
          <td>
            Chengwei Shao, Siyue Jia, Yue Li, Jingxin Li
          </td>
          <td>2025-10-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cisplatin remains a standard first-line therapy for epithelial ovarian cancer; however, the development of chemoresistance leads to poor prognosis and high recurrence rates. Kaplan–Meier analysis of patient data (TCGA) revealed that cisplatin-sensitive tumors are associated with significantly improved overall survival, underscoring the need for strategies to overcome resistance in clinical settings. To investigate the underlying mechanisms, we conducted a comprehensive bioinformatics analysis of publicly available GEO datasets from cisplatin-treated ovarian cancer patients (n = 255). This analysis identified six key molecular drivers of resistance: ZBTB33 (Kaiso), pregnane X receptor (PXR), NF-κB, HER2 (ERBB2), P-glycoprotein (P-gp/ABCB1), and HIF1A. These targets were validated in ovarian tumor specimens via immunohistochemistry, confirming elevated expression in chemo-resistant disease. Additionally, the quantitative real-time PCR analysis confirms the transcriptional upregulation of the six resistance-associated genes in cisplatin-resistant SKOV3 and OVCAR-5 ovarian cancer cells, consistent with the IHC findings. The average fold change in mRNA transcripts ranged from 2.4 for P-glycoprotein (P-gp) to 5 for both NF-kB and Kaiso (ZBTB33). Although less well studied in ovarian cancer, Kaiso (ZBTB33) is known to regulate EMT and tumor invasion in other solid tumors. Functional studies using SKOV3 and OVCAR-5 cell lines demonstrated that knockdown of Kaiso (ZBTB33) via RNA interference significantly increased cisplatin-induced cell death, indicating a direct role in therapeutic resistance. Furthermore, we investigated the synergistic effects of combining SDA, a plant-based omega-3 fatty acid known to inhibit NF-κB, and cisplatin on the cell death of SKOV3 and OVCAR-5 cell lines, comparing the results to those of each compound used individually. Interestingly, co-treatment with stearidonic acid (SDA) synergistically enhanced the cytotoxicity of cisplatin at a lower dose in both cell models. These findings reveal a clinically relevant resistance signature and highlight the therapeutic potential of combinatorial strategies that target both transcriptional regulators (e.g., Kaiso) and inflammatory signaling (e.g., NF-κB). Dual targeting of these pathways may resensitize tumors to cisplatin and improve outcomes for patients with advanced ovarian cancer.



 Mahmoud Mansour, Sabrina Van Ginkel, Maaz Kamal Alata, Ibrahim Bani, Isra Elhussin. Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a654db1a318364cee4d66d47d1f28ac0d3c685d" target='_blank'>
              Abstract A025: Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization
              </a>
            </td>
          <td>
            Mahmoud Mansour, Sabrina Van Ginkel, M. Alata, Ibrahim Bani, Isra Elhussin
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Kyinno provides TMD8 cell lines with endogenous BTK resistance mutations (e.g., C481S, L528W, V416L, T474I, A428D) under native regulation, enabling detailed studies of covalent and non-covalent inhibitor responses. These models faithfully recapitulate clinical escape alleles, supporting both mechanistic investigations and drug-resistance profiling.



 High-fidelity CRISPR/Cas9 RNP electroporation with 200-bp ssODN donors and proprietary HDR enhancers routinely yields >80 % BTK knock-in efficiency. Following optimized limiting-dilution cloning, each clone undergoes genomic PCR, Sanger sequencing, and RT-PCR to confirm on-target integration, allelic zygosity.



 Mutant TMD8 lines display broad IC50 shifts: A428D and L528W confer >100-fold resistance to both ibrutinib and pirtobrutinib, C481S remains nanomolar to pirtobrutinib, whereas T474I/V416L stay responsive to zanubrutinib; all variants expect A428D retain low-nanomolar BGB-16673 and NX-5948 susceptibility. In vivo, C481F tumors show modest ibrutinib delay, whereas L528W xenografts resist ibrutinib yet regress under daily NX-5948 without overt weight loss. Proliferation assays mirror these drug-response patterns across the panel.



 Kyinno’s BTK-mutant lines accelerate high-throughput screening of small-molecule inhibitors and targeted degraders, delineate resistance circuitry via isogenic comparisons, and improve translational relevance by faithfully modeling patient-derived escape variants. This platform supports rapid go/no-go decision-making in early drug discovery and biomarker identification for next-generation BTK therapies.



 Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong. BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B099.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723b954bdd0f1f0d06162f08a8fe3aac141048aa" target='_blank'>
              Abstract B099: BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery
              </a>
            </td>
          <td>
            Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Melanoma is a highly heterogeneous disease, with unique genetic subtypes that influence clinical behavior and treatment response. While targeted therapies and immunotherapy have transformed care for more common types of melanoma, optimal strategies for KIT-mutant melanoma are less well-defined. In this review, we summarize the associations between KIT mutations and specific clinico-pathologic patterns, including their enrichment in acral, mucosal, and chronically sun-damaged melanomas. We detail the spectrum of KIT mutations across relevant exons, noting how mutation subtype influences sensitivity to targeted therapies. We will then analyze several therapeutic trials and case reports that describe the use of c-KIT inhibitors as well as immune checkpoint inhibitors in melanoma treatment. We also discuss early findings supporting combination strategies that may enhance therapeutic outcomes. Finally, we identify critical gaps in understanding the mechanisms of resistance, CNS progression, and the immunogenic landscape of KIT-driven melanoma. Deeper insight into the function of KIT and its interaction with therapeutic pathways is essential to optimizing treatment sequencing and tailoring personalized strategies that improve outcomes in this patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a3b1783130151a4e725cc98816d0f672f99e5" target='_blank'>
              KIT-Mutant Melanoma: Understanding the Pathway to Personalized Therapy
              </a>
            </td>
          <td>
            Aditi Kaveti, Ryan J. Sullivan, Hensin Tsao
          </td>
          <td>2025-11-13</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Persister cells in Escherichia coli (E. coli) represent a transient, non-mutational form of antibiotic tolerance that contributes to treatment failure and chronic infections. This study investigated the genetic and epigenetic responses of E. coli persister cells exposed to ciprofloxacin-induced stress, with a focus on the role of nucleoid-associated proteins (NAPs) in promoting DNA protection and persistence. Fifty clinical E. coli isolates were screened for persister cells using the Rapid Killing and Replica Plating Tolerance Isolation System (REPTIS). The expression of selected NAP genes (hns, hupA, hupB, and dps) was analyzed using real-time PCR (RT-qPCR), while hns gene methylation was assessed via qMethyl PCR. DNA integrity was evaluated by the neutral comet assay, and Dps protein expression was measured by Western blotting. Persister cells were detected in 4% of isolates, with isolate E3 confirmed by REPTIS. RT-qPCR showed significant upregulation of NAP genes in persisters, with hupB showing the highest fold increase (685-fold). qMethyl PCR revealed elevated hns methylation levels (61.55%), suggesting epigenetic regulation. The comet assay showed reduced DNA damage in persister cells, while Western blotting indicated a 1.2-fold increase in Dps expression under ciprofloxacin stress. These findings suggest that E. coli persistence involves a multifaceted mechanism, including transcriptional activation, epigenetic modifications, and protein-level regulation of NAPs. The enhanced expression of Dps under antibiotic stress highlights its protective role and supports the potential of targeting NAPs in developing novel therapeutic strategies against persistent and recurrent E. coli infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77531baaae0bf08af5e5d25a617bafc8cffc2938" target='_blank'>
              EPIGENETIC AND GENETIC RESPONSES OF ESCHERICHIA COLI PERSISTER CELLS TO CIPROFLOXACIN STRESS: INSIGHTS INTO THE ROLE OF NUCLEOID-ASSOCIATED PROTEINS
              </a>
            </td>
          <td>
            Ahmed Abdulwahhab, Mohammed Al Marjani, Zayneb Abdulghany
          </td>
          <td>2025-09-29</td>
          <td>Journal of microbiology, biotechnology and food sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, , David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Radiotherapy (RT) is a crucial treatment for hepatocellular carcinoma (HCC); however, resistance to radiation remains a major challenge to its effectiveness. Circular RNAs (circRNAs), traditionally considered stable non-coding RNAs, have recently gained attention for their potential to encode proteins or peptides that play crucial roles in HCC radioresistance. In this study, circSNX25 (has_circ_0004874) was identified as a novel circRNA with coding potential related to radiation resistance in HCC through circRNA sequencing, Northern blotting, and mass spectrometry (MS). We demonstrated that SNX25-215, a novel protein encoded by circSNX25, rather than the parental circSNX25 itself, promotes resistance to radiotherapy in HCC cells both in vitro and in vivo. Co-immunoprecipitation (co-IP) and molecular docking revealed that amino acids H207 and E214 of SNX25-215 are critical for binding to Golgi to ER traffic protein 4 homolog (GET4). This interaction inhibits the binding of BCL2-associated athanogene 6 (BAG6) to GET4, thereby exposing the nuclear localization sequence (NLS) of BAG6 and facilitating its nuclear translocation. This, in turn, enhances DNA damage repair, ultimately increasing resistance to ionizing radiation (IR) in HCC cells. Importantly, elevated levels of SNX25-215 lead to nuclear localization of BAG6, endowing HCC cells with radioresistant activity, which is further supported by clinical evidence. Our findings highlight the potential of circSNX25 as a prognostic biomarker and a therapeutic target for overcoming radioresistance in HCC. This study provides deeper insights into the roles that circRNA-encoded proteins play in HCC radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16daa4d6e57dc11c1f72d49cb0e599242e4afab2" target='_blank'>
              Circular SNX25 encoded radioresistance augmenter facilitates DNA damage repair in hepatocellular carcinoma by targeting BAG6-GET4 interaction
              </a>
            </td>
          <td>
            Shuping Li, Ying Gao, Yuhao Tang, Aoran Dong, Shuifang Hu, Ruizhi Wang, Xiaoyong Gong, Zhenwei Peng
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbb0b650183e1ab55b2143b3779619fab2e24dd" target='_blank'>
              aiAtlas: High-Fidelity Cell Simulations of Genetic Perturbations in Rare Diseases and Cancers
              </a>
            </td>
          <td>
            WR Danter
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Cyclin-dependent kinase 12 (CDK12) regulates general gene transcription elongation, and plays multiple roles in RNA splicing, DNA damage-response, cell cycle and genomic stability. However, transcriptional partners that guide CDK12-specific gene programs have not been identified. Genomic alterations in CDK12 have been observed in multiple cancers, exhibiting both pro-tumorigenic and tumor-suppressive functions, suggesting a context-dependent mechanism of action.


METHODS
CDK12 copy number alterations and gene expression levels were analyzed in matched primary and brain metastatic patient tumors. Clinical significance was assessed by immunohistochemistry in a large cohort of primary breast cancer patient tumors. RNA-sequencing, ChIP-sequencing and molecular studies were conducted to explore CDK12's mechanism of action, and pharmacological studies were performed both in vitro and in vivo using models of advanced (endocrine-resistant and metastatic) estrogen receptor positive (ER+) disease.


RESULTS
CDK12 amplifications and gene overexpression were observed in brain metastatic tumors. In ER+ primary patient tumors, high CDK12 protein expression significantly associated with poor overall survival, particularly within the ER+/HER2-negative group. In ER+ endocrine resistant models, CDK12 regulated estrogen signaling pathways, with ER/MED1 identified as the master transcriptional complex directing CDK12-specific pro-tumorigenic gene programs. Pharmacological inhibition of CDK12 significantly reduced viability in endocrine resistant and metastatic cell and organoid models in vitro, and decreased metastatic spread in vivo.


CONCLUSION
This work describes a novel mechanism for CDK12, suggesting a potential vulnerability in ER+ breast cancer. These findings provide a basis for further investigation into the role of CDK12 inhibition as a therapeutic approach, particularly in advanced disease settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af116245324e2c76800300a12fb01f632badb6d3" target='_blank'>
              Targeting CDK12 disrupts estrogen-receptor chromatin recruitment and ER-MED1 transcription in advanced ER+ breast cancer.
              </a>
            </td>
          <td>
            Daniela Ottaviani, Mihaela Ola, A. Allotta, Yasmine Maati Chaibi, Seán Hickey, Petra Jagust, Nicola Cosgrove, S. Cocchiglia, F. Bane, Ramón Fallon, Gordon Daly, A. Hegarty, Lance Hudson, Katherine Sheehan, Shannon Kalsi, Stephen Shovlin, Aoibhín M Powell, Ashwani K. Bahl, Ed Ainscow, S. Oesterreich, Adrian V. Lee, F. Couch, Arnold D K Hill, D. Varešlija, Leonie S Young
          </td>
          <td>2025-10-15</td>
          <td>Journal of the National Cancer Institute</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Malaria remains one of the most significant public health challenges globally, particularly in tropical and subtropical regions. Throughout evolutionary history, malaria-induced natural selection has profoundly influenced human genetic evolution, leading to the emergence of numerous genetic variations that confer resistance to the disease. These adaptations highlight the complicated interplay between pathogens and human genetics. This review focuses on key genetic variations associated with malaria resistance, including hemoglobinopathies (such as sickle cell trait and thalassemia), glucose-6-phosphate dehydrogenase deficiency, blood group polymorphisms and genetic variants related to inflammation and immune regulation. The prevalence of these genetic adaptations varies widely across different geographic regions, reflecting the historical burden of malaria in those areas. Despite significant advancements in genetic research, the precise mechanisms by which these mutations confer protection against malaria remain incompletely understood. Furthermore, the interactions between these genetic factors and environmental influences add to another layer of complexity. A comprehensive understanding of these genetic variations and their functional implications is crucial for advancing malaria epidemiology, improving diagnostic tools, and developing targeted prevention and control strategies, ultimately contributing to global efforts to eradicate malaria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18b17879dc0c6117b588d9876603f22ae600545d" target='_blank'>
              Human genetic variations conferring resistance to malaria
              </a>
            </td>
          <td>
            Xiaokun Zhang, Jie Wu, Yunxing Peng, Lan Luo, Lu Zhang, Xi Huang, Guoying Chen, Yirong Li, Haoan Yi
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA is continuously exposed to endogenous and exogenous factors that induce oxidative modifications leading to mutations and genomic instability. Oxidative DNA damage plays a dual role, contributing to physiological signaling at low levels while promoting mutagenesis, carcinogenesis and degenerative diseases when unpaired. Among various lesions, an oxidized base, such as 8-oxo-2′-deoxyguanosine (8-oxodG), is one of the major biomarkers of oxidative stress and genomic damage. Cells have evolved sophisticated repair processes, including base excision repair (BER), nucleotide excision repair (NER), and mismatch repair (MMR), to maintain genomic integrity. Dysregulation or polymorphism of these repair genes has been linked with cancer, neurologic, and cardiovascular disorders. This review discusses an overview of what is presently known concerning oxidative DNA damage and repair mechanisms, particularly emphasizing their molecular players, signaling routes, and human disease implications. It further refers to the latest advances in CRISPR-based technologies and multi-omics approaches that are redefining our understanding of DNA damage response (DDR) networks and creating new frontiers for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f647de037f61e5dbeca9c5a033dbdfb5daa6ba" target='_blank'>
              Cellular and Molecular Mechanisms of Oxidative DNA Damage and Repair
              </a>
            </td>
          <td>
            A. Ayna, Cuneyt Caglayan, S. Taysi
          </td>
          <td>2025-11-11</td>
          <td>Medicina</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="OBJECTIVES
Antimicrobial resistance (AMR) is an escalating global health concern, driven by multifactorial biological processes not yet fully understood. This study employed network diffusion analysis to dissect the molecular mechanisms driving AMR in Escherichia coli, aiming to identify novel potential drug targets for therapeutic development.


METHODS
A systems biology approach was used to identify genes and biological pathways associated with AMR, by mapping known AMR-related genes from the Comprehensive Antibiotic Resistance Database (CARD) and PointFinder database into the E. coli protein interactome. Through a network diffusion algorithm, several network modules were identified, i.e. genes and pathways, in part already known to be involved in AMR mechanisms. We selected gene candidates for performing an in vitro susceptibility validation test, consisting of 13 knockout mutants against nine different antibiotics.


RESULTS
Compared with the WT E. coli BW25113, the AMR of some mutants showed significant shifts of biological relevance: ΔuhpB (S/I) and ΔmdaB (S/R) against ampicillin, ΔrpmG (I/S) and ΔrplA (I/S) against ciprofloxacin, and ΔrplA (I/S) against streptomycin (S, susceptible; I, intermediate; R, resistant). In other cases, only a significant change in inhibition disc diameter was observed, probably deserving further studies.


CONCLUSIONS
Network diffusion is an effective tool to infer relevant biological insights related to AMR from microbial biological networks. Our results contribute to a better understanding and characterization of AMR in E. coli. Furthermore, the in vitro validated genes could be considered as new putative drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1855f0a6e4ce4deeef7bbc3d3b9fa04f99bcd8" target='_blank'>
              Identification of antimicrobial resistance genes in Escherichia coli through network diffusion.
              </a>
            </td>
          <td>
            Anis Mansouri, F. Durazzi, Muhammad Ahmed Ihsan, Sholeem Griffin, G. Manfreda, V. Valdramidis, F. Pasquali, D. Remondini
          </td>
          <td>2025-11-11</td>
          <td>The Journal of antimicrobial chemotherapy</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cardiac myxoma, the most common primary heart tumor, remains poorly understood at the molecular level. Here, we combined single-nucleus RNA sequencing, third-generation transcriptomics, and untargeted metabolomics to dissect its origin and pathology. Single-cell analyses demonstrate an endothelial origin driven by aberrant endothelial-to-mesenchymal transition (EndMT), with pseudotime and RNA-velocity tracing a continuum from endothelial-like to mesenchymal-like and metabolically active states. We identify two distinct myxoma subtypes: Subtype 1, marked by MAPK/WNT/EGFR pathway activation, and Subtype 2, characterized by ribosomal and oxidative phosphorylation signatures alongside immune-evasive programs. Third-generation data highlight extracellular matrix remodeling and endothelial signaling, while metabolomics reveal dysregulated purine, nicotinic acid, and nicotinamide metabolism. Notably, MET–PTK2 signaling emerges as a potential driver of tumor initiation and progression. These integrated findings define the cellular architecture and metabolic adaptations of cardiac myxoma and lay the foundation for future interventions. The effect of zingerone on postmenopausal osteoporosis, as modelled in mouse, was assessed The authors combine single-cell transcriptomics and bulk metabolomics to investigate cardiac myxoma. They identify Subtype 1 marked by MAPK/WNT/EGFR pathway activation and Subtype 2, characterized by ribosomal and oxidative phosphorylation signatures">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0702495e091268eccb0b45edf83576207a5dbb0c" target='_blank'>
              Multi-omics data reveal the origin of cardiac myxoma
              </a>
            </td>
          <td>
            Shengzhong Liu, Wanfeng Zhang, Huajun Sun, Chenqing Zheng, Keli Huang, Chengming Fan, Rensheng Lai, Mingzhu Yin, Jie Lan, Xiushan Wu, Longke Ran, Xiaoping Li
          </td>
          <td>2025-09-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Conventional antibiotics are becoming less effective as the global health crisis of multidrug-resistant (MDR) bacterial infections worsens. The function of bacterial DNA repair systems in promoting MDR is a quickly developing paradigm, despite the well-established nature of conventional resistance mechanisms such as efflux pumps and drug-inactivating enzymes. The present understanding of how DNA repair pathways, which are necessary for the stability of the genome, paradoxically promote adaptive mutagenesis and horizontal gene transfer under antibiotic stress is summarized in this review. In important MDR pathogens, we investigate the complex interactions among repair mechanisms, stress reactions, and resistance evolution. The regulatory crosstalk with other bacterial systems and the potential of DNA repair inhibitors as novel therapeutic adjuvants are two examples of significant knowledge gaps that are highlighted. We wrap up by going over potential future directions for focusing on DNA repair to re-sensitize MDR bacteria and prolong the effectiveness of current antibiotics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eab813759b15d787a511efda36aa4a661434f2" target='_blank'>
              DNA Repair Mechanisms in Multi Drug Resistant Bacteria: Impact on Genome Stability and Antibiotic Resistance
              </a>
            </td>
          <td>
            Faiqa Shakeel, Iman Mustafa, Ruba Shahid, Amna Noor, Mishaal Fatima, Kiran Ghafoor, AminaTariq, Samra Zulfiqar
          </td>
          <td>2025-09-30</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Anaplastic lymphoma kinase (ALK) gene fusions are potent oncogenic drivers across diverse malignancies, including non-small cell lung cancer (NSCLC), anaplastic large cell lymphoma (ALCL), breast cancer, and glioma. These fusions lead to constitutive activation of the ALK tyrosine kinase domain (TKD), rendering tumors initially responsive to ALK tyrosine kinase inhibitors (TKIs)—primarily in NSCLC, but also in rare cases such as infant-type hemispheric glioma. However, resistance inevitably emerges, frequently due to acquired single nucleotide variants (SNVs) within the TKD. A significant proportion of these SNVs are classified as variants of uncertain significance (VUSs), highlighting the need for systematic functional annotation to inform precision therapy.



 We employed prime editing to introduce all possible SNVs across the ALK TKD (exons 20-28). High-throughput drug resistance screening was conducted against three generations of ALK TKIs: alectinib (second-generation), lorlatinib (third-generation), and zotizalkib (fourth-generation, previously evaluated in clinical trials). Resistance classification data were further assessed using Kaplan-Meier survival analyses to explore potential prognostic implications.



 A total of 2,953 SNVs (99.2% of all possible variants in exons 20-28) were successfully introduced and profiled. Resistance phenotypes were comprehensively mapped for each variant across the three TKIs. Canonical resistance mutations such as G1202R and L1196M were accurately recapitulated, validating our screening platform. Moreover, several previously uncharacterized VUSs were identified as conferring differential resistance profiles. Structural modeling revealed mechanistic insights into altered TKI binding. These findings aligned with existing clinical and preclinical datasets and expanded the known spectrum of clinically actionable ALK resistance mutations.



 This study presents the most comprehensive functional landscape to date of ALK TKD variants in the context of TKI resistance. Our data provide an invaluable resource for interpreting VUSs, optimizing TKI selection, and advancing personalized treatment in ALK fusion-positive cancers. Furthermore, the approach and insights are broadly applicable to other ALK-rearranged malignancies, including glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6b913fdc6e574ca69abb2b1d6922fc64f2b203" target='_blank'>
              JS04.5.A HIGH RESOLUTION DRUG-RESISTANT MAPPING OF ALK FUSION VARIANTS USING PRIME EDITING
              </a>
            </td>
          <td>
            H. Oh
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56a3d728c459f75967661fea1d9baa92b4ef2796" target='_blank'>
              Age distinguishes selection from causation in cancer genomes
              </a>
            </td>
          <td>
            David Cheek, Martin Blohmer, Martin A. Nowak, Tibor Antal, Kamila Naxerova
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Highlights What are the main findings? ALT functions as a context-dependent double-edged sword: it drives aggressiveness in rapidly proliferating cancers but imposes replicative and metabolic stress in slow-growing, immune-cold tumors such as glioblastoma and chondrosarcoma. Loss of ATRX/DAXX and ALT activation creates unique DNA repair vulnerabilities, including hypersensitivity to ATR and PARP inhibition, opening avenues for targeted therapy. What is the implication of the main finding? Refines prognostic interpretation of ALT positivity—transforming it from a uniformly negative marker into a tumor-type-specific indicator of clinical outcome. Positions ALT as a precision oncology target, where integrated biomarkers (C-circle as say, APB detection, ATRX/DAXX mutation profiling) enable patient stratification and therapeutic decision-making. Abstract Telomere maintenance enables unlimited cell proliferation by counteracting telomere erosion. While the majority of tumors activate telomerase, a significant subset—approximately 10–15%—utilizes alternative lengthening of telomeres (ALT), a recombination-based mechanism. ALT-positive cancers are classically associated with genomic instability, anaphase bridges, chromosomal rearrangements, and resistance to DNA-damaging therapies. This process is closely associated with genetic instability, which contributes to chromosomal rearrangements and tumor evolution. Consequently, ALT has traditionally been considered an adverse prognostic marker in aggressive malignancies such as osteosarcoma, pancreatic neuroendocrine tumors, and high-grade sarcomas. Paradoxically, recent evidence demonstrates that ALT positivity correlates with improved survival in glioblastoma (GBM) and chondrosarcoma, two tumor types that have historically been regarded as immune-cold and therapeutically intractable. This favorable outcome likely reflects a convergence of factors, including replication stress and DNA damage that impose a fitness cost in slow-growing or metabolically constrained tumors. Loss of ATRX/DAXX, while enabling ALT, further amplifies chromatin fragility, and ALT-mediated instability may paradoxically enhance immunogenicity within immune-quiescent microenvironments. Moreover, ALT-positive cells exhibit unique therapeutic vulnerabilities, particularly to ATR and PARP inhibitors. Together, these observations support a context-dependent model in which ALT functions as a double-edged sword, acting as a driver of malignant aggressiveness in rapidly proliferating cancers while serving as a relative liability in slower-growing, immune-cold tumors. Understanding this duality not only refines prognostic stratification but also opens opportunities for precision oncology. By integrating ALT-specific biomarkers into clinical workflows and exploiting ALT-related DNA repair dependencies, clinicians may transform a once uniformly negative prognostic factor into an actionable therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6ed85bfa1a7b22ce5afc4a049e72d82de6bf4" target='_blank'>
              Alternative Lengthening of Telomeres: A Prognostic Paradox in Cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Advanced prostate cancer (PrCa) remains a leading cause of cancer-related death among men due to disease progression in nearly all patients on standard of care therapy targeting the androgen receptor. An important mechanism driving therapeutic resistance is lineage plasticity, which enables PrCa cells to reprogram into lineage variants no longer dependent on androgen receptor signaling. As inhibitors of the histone methyltransferase EZH2 are being evaluated clinically for the treatment of advanced PrCa, we investigated here how EZH2 affects PrCa lineage plasticity. Data from genetically engineered mice and human clinical samples demonstrated that genetic or pharmacological suppression of EZH2 altered chromatin to expand active transcription factor programs. These changes in gene expression during PrCa progression increased the diversity of PrCa lineage variants that arose. EZH2 suppression did not inhibit disease progression or therapeutic resistance in this context. These findings advance the current understanding of PrCa lineage plasticity and suggest EZH2 inhibitors may be less effective in treating PrCa prone to lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee66b00f883047cd2ed8c292d9f0f040e78b6ffe" target='_blank'>
              EZH2 Suppression Diversifies Prostate Cancer Lineage Variant Evolution and Lacks Efficacy in Inhibiting Disease Progression.
              </a>
            </td>
          <td>
            Justine J Jacobi, K. Wadosky, Neha Jaiswal, Xiaojing Zhang, Yanqing Wang, Prashant K Singh, Jie Wang, Eduardo Cortes Gomez, Jianmin Wang, Bo Xu, Mark D Long, Paloma Cejas, Shweta Kukreja, H. Long, David W. Goodrich
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Epigenetic regulation, encompassing DNA methylation, histone modifications, and non-coding RNA activities, is a crucial mechanism through which gene expression is modified without corresponding changes in genomic DNA sequences. Dysregulation of these mechanisms can lead to histone and DNA modifications that either suppress or enhance the expression of disease progression-related genes. Among these regulatory processes, histone modifications are particularly significant, as they contribute to genomic stability, DNA repair, and chromatin dynamics, all of which influence breast cancer initiation and progression. This review offers a detailed analysis of the current state of research centered on epigenetic regulatory factors, with a particular focus on the role that histone modifications play in the treatment of breast cancer. It also examines the interplay between epigenetic modifications and the effectiveness of radiotherapy and chemotherapy when treating breast cancer. Lastly, this article explores the potential of epigenetic regulatory factors as viable targets for the future design of new anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379da6773863c74043489d2ccffa9dd0b5e23c2c" target='_blank'>
              Epigenetic regulation espeically histone modifications in breast cancer: A viable and emerging targeted therapeutic strategy
              </a>
            </td>
          <td>
            Yibing Zhou, Haotian Liu, Weimin Hong, Haotian Su, Yuxiao Mu, Yijie Cheng, Chaoshen Wu, Xuli Meng, Da Qian
          </td>
          <td>2025-09-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) containing TP53 (p53) mutations are routinely treated with decitabine or 5-azacytidine which deplete DNA methyltransferase 1 (DNMT1)('hypomethylating agents', HMA). Unfortunately, resistance/relapse, characterized by preserved DNMT1, is rapid. HMA are pyrimidine analogs, and to deplete DNMT1, must compete with endogenous pyrimidines. These were substantially increased in HMA-resistant versus parental AML cells, together with upregulation of carbamoyl-phosphate-synthetase-2/aspartate transcarbamylase/dihydroorotase (CAD) that rate-limits de novo pyrimidine synthesis. Moreover, TP53-mutated AMLs appeared primed for such resistance with baseline higher CAD. Pyrimidine synthesis can be depowered by using the BCL2-inhibitor venetoclax to release BAX to depolarize mitochondrial membranes. However, BAX, a p53-target gene, was ~2-fold less expressed in TP53-mutated vs TP53-wildtype cells, and venetoclax impacts were correspondingly limited. Alternatively, pyrimidine synthesis can be inhibited directly at dihydroorotate dehydrogenase (DHODH) using the clinical drug teriflunomide. Contrasting with venetoclax, teriflunomide decreased pyrimidines several-fold, restored DNMT1-depletion, and cytoreduced HMA-resistant TP53-mutated AML cells via p53/apoptosis-independent terminal lineage-maturation. This non-cytotoxic pathway preserved viability and proliferation of normal hematopoietic stem/progenitor cells (HSPC). Inhibiting pyrimidine synthesis triggered automatic increases in pyrimidine salvage, such that schedules for teriflunomide combination with HMA, taken-up by salvage, mattered: in mice with TP53-mutated AML, teriflunomide scheduled day-before HMA was more efficacious than same-day or day-after. Chronic teriflunomide exposure paradoxically increased pyrimidines via sustained compensatory pyrimidine salvage, conferring resistance rather than sensitivity to HMA. In sum, DNMT1- and DHODH-targeting, administered by timed-intermittent (metronomic) schedules, can circumvent genetic-resistance caused by TP53-mutations, and adaptive-resistance caused by metabolic homeostasis, without cytotoxicity to normal HSPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b648fdff4610f9244486c913d8f3bc8335aa5ac" target='_blank'>
              Combining drugs that bypass p53 to treat TP53-mutated leukemias.
              </a>
            </td>
          <td>
            Sudipta Biswas, Zeinab Zahran, Xiaorong Gu, Lisa Cardone, Remuna Marti, Nour Mouannes, Maximillian Stich, Akriti G Jain, Kateryna Fedorov, Benjamin K Tomlinson, M. Goldfinger, Amit Verma, Yogenthiran Saunthararajah
          </td>
          <td>2025-10-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 SETD2 is a tumor suppressor gene responsible for trimethylation of H3K36, crucial for DNA mismatch repair and homologous recombination. Novel synthetic lethality agents, ATR inhibitors (ATRi), have demonstrated anti-tumor activity in preclinical models and early clinical trials for pan-tumors. SETD2 and H3K36 deficient cells have been demonstrated to display Micro Satellite Instability (MSI) with increased tumor mutational burden (TMB). Deleterious SETD2 alterations occur in 5-10% of brain tumor patients with glioblastoma, IDH wild type, WHO grade 4, (GBM). These patients have a poor prognosis. This study aims to evaluate targeted treatment strategies in central nervous system (CNS) tumors, by investigating the potential of ATRi for tumors harboring deleterious SETD2 alterations and immuno-therapy for tumors with increased mutational burden, with the objective of improving patient outcomes.



 This study analyzes genomic profiles from 434 patients diagnosed with central nervous system (CNS) tumors between January 1st, 2019, and February 28th, 2025, enrolled in the prospective Neurogenome and CoPPO studies. Rigshospitalet has one of the largest single-institution datasets with comprehensive data on SETD2 alterations derived from whole exome/genome sequencing including structural variants, copy number variations, and RNA expression.



 The study cohort included 434 patients with CNS tumors (166 female, 268 male) with a median age at diagnosis of 59 years (interquartile range: 49-66 years). GBM was the predominant tumor type (86%), followed by astrocytoma, IDH mutated (mut), WHO grade 2-4 (12%) and oligodendroglioma, IDH-mut, 1p/19q co-deletion, WHO-grade 2-3 (1.2%). Other CNS tumors were diagnosed in 6 (1.4%) patients. Deleterious SETD2 alterations were found in 13 (3%) patients: 11 with GBM, 1 astrocytoma IDH mut, and 1 oligodendroglioma. Twelve patients had mutations: 6 premature stop codons, 5 frameshift mutations and 1 splice site variant, and one case with biallelic SETD2 loss. Data on tumor mutational burden (TMB) was assessable for 12 SETD2-deficient patients, none exceeding 10mut/mb; mean TMB was 3.3 mut/mb vs. 6.2 mut/mb in the SETD2-wildtype (wt) group. No SETD2-deficient patients had received temozolomide (TMZ) treatment prior to sampling. By contrast, some samples in the SETD2-wt cohort were obtained at relapse, which may have inflated TMB due to TMZ treatment. COSMIC signatures were available for 10 SETD2-deficient patients with none having indications of MSI based on COSMIC signature evaluation.



 Loss of SETD2 function in primary CNS-tumors is relatively rare and was predominantly driven by mutations, with copy number analysis revealing only 1 biallelic deletion in 434 patients. SETD2 alterations was primarily seen in GBM patients. The SETD2 deficient patients did not exhibit indications of MSI or high TMB. However, the impact of patient relapse samples with TMZ induced mutations in the SETD2-wt cohort is being explored further together with pan-tumor markers for DNA damage repair.



 Tine Østergaard, Christoffer L. Halvorsen, Dorte S. Nørøxe, Vincent N. Fougner, Iben Spanggaard, Kristoffer S. Rohrberg, Martin Højgaard. Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in SETD2 without associated increase in mutational burden [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5615e3697959292b283838db14154a56c4e82cf6" target='_blank'>
              Abstract C037: Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in
 SETD2
 without associated increase in mutational burden
              </a>
            </td>
          <td>
            Tine Østergaard, Christoffer L. Halvorsen, D. Nørøxe, V. Fougner, I. Spanggaard, K. S. Rohrberg, M. Højgaard
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Traditional pooled genetic dependency screens, such as DepMap, have proven powerful for identifying essential genes in cancer cell lines. However, these pooled formats are prone to the bystander effect, where secreted factors from wild-type cells support the survival of knockout (KO) cells within the same culture, potentially masking dependencies on specific secreted proteins. To address this limitation, we performed a well-by-well CRISPR screen targeting genes encoding secreted factors in the pancreatic ductal adenocarcinoma (PDAC) cell line HPAFII. This arrayed format eliminates cross-feeding and enables precise assessment of cell-intrinsic dependencies on targeted proteins and could thus expose hidden vulnerabilities. CellTiter-Glo (CTG) viability assays were used to assess gene lethality following target gene KOs. Furthermore, RNA sequencing verified effective disruption of target genes, establishing the specificity and reliability of the screening workflow. We observed strong growth inhibition in EFNA1 (Ephrin-A1) KO samples compared to lipofectamine-treated controls. However, this inhibitory effect was missed and underrepresented in the conventional pooled format. EFNA1 is a ligand for EPH receptors, involved in cell-cell communication, migration, and proliferation signaling. Differential gene expression analysis comparing EFNA1 KO to lipofectamine-treated controls revealed enriched pathways of oxidative phosphorylation, E2F targets, MYC targets, and DNA repair in EFNA1 KO samples. This result supports our observation of lethality in EFNA1 KO samples. It suggests that EFNA1 contributes to cell-cell communication and is critical for HPAFII cell survival. Its KO triggers stress pathways, including E2F, MYC, OXPHOS, and DNA repair, and acts as a suppressor of excessive proliferative programs. In conclusion, we identified EFNA1 as a critical non-cell-autonomous essential gene through an arrayed CRISPR screen, with its loss showing lethality to HPAFII cells exclusively in isolated settings, likely as a result of disrupted cell-cell interactions. Defining such dependencies may facilitate the discovery of directly druggable and more tumor-selective therapeutic strategies.



 Xingyi Shi, David Ruddy, Millicent Gabriel, Michelle Piquet, Kris Muskiewicz, Joel Wagner, Joshua Korn, Cory Johannessen. Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd6ce939e1194e9cb9b823d2bb2e7b39fb1e35f" target='_blank'>
              Abstract C035: Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells
              </a>
            </td>
          <td>
            Xingyi Shi, David A Ruddy, Millicent Gabriel, M. Piquet, Kris Muskiewicz, Joel P Wagner, Joshua Korn, Cory Johannessen
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase inhibitors (PARPis) have improved the prognosis of patients with high-grade serous ovarian carcinoma (HGSOC). However, PARPis are not effective for all HGSOC patients. Spatial transcriptomics is a powerful tool for characterizing the tumor microenvironment. We used Visium to analyze eight tumor samples from HGSOC patients with clinical information on PARPi sensitivity. Two complementary analyses were performed: an integrated analysis across all samples, without considering spatial information, and an analysis with spatial information within each sample. Both approaches indicated that midkine (MDK) signaling is involved in PARPi resistance. Furthermore, we identified receptors that enhance MDK signaling in cancer cells. To assess the generalizability of this finding, we deconvolved bulk RNA-sequencing data using single-cell RNA-sequencing data as a reference to examine the relationship between receptor expression levels and overall survival (OS). This analysis revealed that high SDC4 expression in cancer cells is associated with poor OS in HGSOC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6e6aeb1b5b6668556eb27188c5496dc383c124" target='_blank'>
              Exploring drug resistance via intercellular crosstalk using spatial transcriptomics in high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Hironori Suzuki, Kosuke Yoshida, Akira Yokoi, Kazuhiro Suzuki, Yuta Hirano, Masami Kitagawa, Eri Asano-Inami, M. Yoshihara, Yukari Nagao, S. Tamauchi, N. Yoshikawa, Hiroaki Kajiyama, Yusuke Yamamoto
          </td>
          <td>2025-11-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by the expansion of hematopoietic stem and progenitor cells harboring somatic mutations, which confers an increased risk of hematologic malignancies and cardiovascular disease. Among CH-associated mutations, mutations affecting splicing factors (SFs), including splicing factor 3b subunit 1 (SF3B1), serine/arginine-rich splicing factor 2 (SRSF2), U2 small nuclear RNA auxiliary factor 1 (U2AF1), and zinc finger CCCH-type, RNA binding motif and serine/arginine rich 2 (ZRSR2), play a unique role in promoting clonal expansion and leukemogenesis. In this review, we summarize recent findings on the role of SF mutations in CH progression, their interplay with other mutations (e.g., DNA methyltransferase 3 alpha (DNMT3A), ten-eleven translocation methylcytosine dioxygenase 2 (TET2) and isocitrate dehydrogenase 2 (IDH2)), and their impact on hematopoietic homeostasis. Epidemiological studies have demonstrated that SF-mutant CH exhibits an accelerated clonal expansion compared to other CH clones. Furthermore, murine models suggest that SF mutations alone do not inherently confer a growth advantage for clonal expansion but rather enhance disease phenotypes when co-existing with epigenetic mutations, such as IDH2 and TET2. These findings suggest that SF mutations contribute to CH expansion and malignant transformation through a synergistic interplay with other mutations and external factors such as inflammation. Given the clinical significance of SF mutations, ongoing research is focused on developing targeted therapies that modulate aberrant RNA splicing and prevent CH-driven leukemogenesis. Understanding the mechanisms underlying mutant spliceosome-mediated CH expansion may provide novel insights into early detection, risk stratification, and therapeutic interventions in hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8d9014adbb748e6a4c3fdbe95052f49e04f07e8" target='_blank'>
              The Impact of Splicing Factor Mutations on Clonal Hematopoiesis and Myeloid Neoplasm Progression.
              </a>
            </td>
          <td>
            Takuya Izumi-Tamura, A. Kawachi, Akihide Yoshimi
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Targeted therapy has been established as a therapeutic option for the treatment of metastatic melanoma. Despite initially being very efficient, many tumors develop resistance to targeted therapy, leading to its failure. We previously demonstrated that the neural crest (NC)-associated gene ERRFI1 is highly expressed in metastatic melanoma and correlates with a bad prognosis. Here, we show that the expression of ERRFI1 was upregulated in melanoma and negatively correlated with the expression of melanocytic differentiation markers, such as MITF and TYR. Downregulation of ERRFI1 with the help of siRNA increased the susceptibility of melanoma cells toward BRAF inhibition (BRAFi) and resensitized BRAFi-resistant melanoma cells to BRAFi. Mass spectrometry-based proteomic analysis revealed that ERRFI1 silencing diminished the activation of the mitogen-activated protein kinase (MAPK) and AKT signaling pathways, which usually contribute to drug resistance. Furthermore, we show that miR-200c targeted the 3'UTR of ERRFI1 and reduced its expression, resulting in the resensitization of BRAFi-resistant melanoma cells to BRAFi. Our study results suggest that ERRFI1 could be a potential therapeutic target for the treatment of metastatic melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a645774843921b1750028307066a9681c4de577" target='_blank'>
              The neural crest-associated gene ERRFI1 is involved in melanoma progression and resistance toward targeted therapy.
              </a>
            </td>
          <td>
            Nina Wang, Qian Sun, Daniel Novák, Lei Zhu, Juliane Poelchen, Tamara Steinfass, Yiman Wang, V. Umansky, Jochen Utikal
          </td>
          <td>2025-10-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="

 Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored.



 We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls.




 AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g.,
 FLT3
 ,
 RUNX1
 , and
 CD33
 ) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML.




 This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgen Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA repair mechanisms in human primary cells, including error-free repair, and, recurrent nuclease cleavage events, remain largely uncharacterised. We elucidate gene-editing related repair processes using Cleavage and Lesion Evaluation via Absolute Real-time dPCR (CLEAR-time dPCR), an ensemble of multiplexed dPCR assays that quantifies genome integrity at targeted sites. Utilising CLEAR-time dPCR we track active DSBs, small indels, large deletions, and other aberrations in absolute terms in clinically relevant edited cells, including HSPCs, iPSCs, and T-cells. By quantifying up to 90% of loci with unresolved DSBs, CLEAR-time dPCR reveals biases inherent to conventional mutation screening assays. Furthermore, we accurately quantify DNA repair precision, revealing prevalent scarless repair after blunt and staggered end DSBs and recurrent nucleases cleavage. This work provides one of the most precise analyses of DNA repair and mutation dynamics, paving the way for mechanistic studies to advance gene therapy, designer editors, and small molecule discovery. Quantifying genomic aberrations resulting from designer nucleases activity is essential for gene therapy clinical translation. Here, the authors present a modular digital PCR technique that profiles DNA repair precision and cut-repair cycles at the edited loci, exposing current evaluation biases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347cbf692d67c07b13519f5e18f058f81e33be4d" target='_blank'>
              Unveiling the cut-and-repair cycle of designer nucleases in human stem and T cells via CLEAR-time dPCR
              </a>
            </td>
          <td>
            Nathan White, John Alexander Chalk, Yi-Ting Hu, Samuel Mark Pins, Chinnu Rose Joseph, Panagiotis Antoniou, Sandra Wimberger, Stina Svensson, S. Caetano-Silva, Anne Charlotte Adriane Mudde, R. Rai, Sridhar Selvaraj, William N. Feist, Marianna Romito, Grzegorz Sienski, Roberto Nitsch, Claire Booth, G. Santilli, Alessia Cavazza, Matthew H. Porteus, Marcello Maresca, A. Thrasher, G. Turchiano
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ataxia-Telangiectasia (A-T) is caused by biallelic Ataxia-Telangiectasia Mutated (ATM) protein mutations and is hallmarked by neurodegeneration, immunodeficiency and cancer susceptibility. Emerging work suggests that non-coding RNAs are integral to the ATM driven DNA damage response, yet the scope of non-coding RNA perturbation in A-T has not systematically mapped. This study is done to chart and synthesize all primary evidence on ncRNA dysregulation in A-T and to identify mechanistic gaps and research priorities. The evidence base comprises a total of five studies focused exclusively on microRNAs and long non-coding RNAs. Baseline sequencing of 20 patients revealed consistent downregulation of miR-195-5p, miR-30a-5p and miR-342-3p. Low dose γ-irradiation of ATM-null lymphoblastoid cell lines produced an exaggerated bidirectional shift (8 microRNAs upregulated, 6 downregulated) versus wild-type, whereas primary T-cells showed largely ATM independent rises of miR-34a-5p and miR-182-5p. A lymphoblastoid cell lines study distinguished 22 recessive or dominant radiation-responsive microRNAs. Long non-coding RNAs profiling identified 149 transcripts that require functional ATM for induction and three uniquely upregulated in A-T cells. No circular RNA study in A-T is observed till date. The ncRNA research in A-T is nascent with five studies present a miRNA dysregulation and an ATM-dependent lncRNA surge that is lost in A-T, but leave circRNAs, neuronal tissues and functional validation unexplored. Intermediate dose stress models, total and small RNA-seq under DNA damage and oxidative stress, and CRISPR rescue/knockout screens in patient derived organoids are immediate priorities for exploring the role of ncRNAs as therapeutic targets in A-T.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347f924a4e62687ed1a1c526f37bf32430e75d71" target='_blank'>
              Mapping the non-coding RNA landscape in ataxia telangiectasia: a scoping review of ATM dependent miRNA and lncRNA dysregulation
              </a>
            </td>
          <td>
            Muhammad Junaid Iqbal, Usman Ameen, Hamza Tanveer, Laraib Israr, Anastasia Ricci, Gianluca Morganti, Muhammad Jawad Khan, M. Menotta
          </td>
          <td>2025-10-09</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>